<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-09 09:30:19 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>40</td>
          <td>43</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancers develop and evolve by overcoming numerous selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. Specifically, when disease progression occurs after targeted therapy treatment, genomically-defined mechanisms of resistance are only identified in approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cancer cell states. We have successfully established a high-throughput screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and therapy resistance; 3) perform single-cell RNA sequencing to define families of TFs that drive specific cell state transitions and resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors, suggesting that cell state adaption may contribute to multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and potential predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for patients.


 Srivatsan Raghavan, Evelyn Y. Tong, Aswanth H. Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andrew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter. Systematic interrogation of cell states that drive inflammation tolerance and therapy resistance in cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4399.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d685ee84bfeabbf621f354633519c29df6bcad68" target='_blank'>
              Abstract 4399: Systematic interrogation of cell states that drive inflammation tolerance and therapy resistance in cancer
              </a>
            </td>
          <td>
            Srivatsan Raghavan, Evelyn Y. Tong, Aswanth H. Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andrew W. Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 In oncology, patients treated with targeted therapies often have strong and durable responses early in treatment; however, the development of resistance hinders long-term therapeutic benefit. A subset of tumor cells can develop resistance, often from both genetic and non-genetic mechanisms through gradual, multi-step resistance. Nonetheless, predicting which cells will evolve to become resistant, and strategies to suppress adaptability contributing to resistance remain unknown. While resistance may emerge in cells harboring driver mutations, the adaptation process necessitates substantial modifications to the phenotype facilitated by cellular plasticity. Therefore, plasticity serves as a crucial facilitator of resistance by enhancing adaptability, rather than functioning as a direct cause of resistance. We hypothesize that the direct selection for therapy resistance leads to indirect selection for enhanced plasticity. We propose that as treatment selects for drug-resistant phenotypes, concurrent selection for cells exhibiting greater adaptability occurs. Thus, enhanced plasticity is indirectly favored during selection for drug resistance. We have found strong support for this hypothesis through in silico modeling, with subsequent validation through functional assays. In our functional assays, we utilize the H3122 cell line derived from a patient with ALK+ non-small cell lung cancer (NSCLC) to model targeted therapy resistance. This well-studied model effectively mimics the initial responsiveness and subsequent resistance observed in clinical settings. Utilizing single-cell sorting and the expansion of clonal isolates from the parental H3122 population, we identified heterogeneity in the ability to adapt where subgroups exhibited a range of high adaptation potential (HAP) to low adaptation potential (LAP) based on their response to numerous independent stressors, including targeted ALK inhibitors. We have found that therapeutic stressors lead to selection of subpopulations with higher adaptability potential, enriching for a more lethal subgroup than the original tumor. HAP cells display increased multi-drug resistance and metastatic potential compared to LAP cells. HAP cells are significantly enriched in plasticity-related pathways overlapping with those found in cells that have developed strong resistance to ALK inhibitors. In subcutaneous tumor models, LAP cells show a 1, 000-fold reduction in tumor burden, while HAP cells develop resistance. Our findings suggest that disrupting cellular adaptability can prevent resistance, maintaining cells in a therapy-sensitive state. This approach shifts the focus from killing tumor cells to blocking their ability to adapt, opening up a novel angle for overcoming resistance. Addressing adaptability reveals intervention points that conventional strategies might overlook, offering new, proactive strategies in cancer treatment.


 Alicia Bjornberg, Xierali Aobuli, Matthew Froid, Rowan Barker-Clarke, Jeff Maltas, Jacob Scott, Berkley Gryder, David Bassanta, Alexander Anderson, Virginia Turati, Andriy Marusyk. Cellular plasticity facilitates therapy resistance through enhancing adaptability. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2674.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4abf519afad0178ba59d0cd17f3f428071bf92cb" target='_blank'>
              Abstract 2674: Cellular plasticity facilitates therapy resistance through enhancing adaptability
              </a>
            </td>
          <td>
            Alicia Bjornberg, Xierali Aobuli, M. Froid, R. Barker-Clarke, Jeffrey Maltas, Jacob G. Scott, Berkley Gryder, David Bassanta, Alexander Anderson, Virginia Turati, A. Marusyk
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cancer cells that survive therapeutic drug pressure are a significant cause of disease relapse and progression, impeding curative cancer treatment. Drug-triggered Darwinian selection and the emergence of subclones harbouring specific mutations that confer resistance have been well documented and extensively studied. However, these genetic alterations, while important, do not fully explain clinical observations where some patients, after a drug holiday, regain sensitivity to the same treatment despite previous disease progression. This phenomenon highlights the possibility that drug resistance may not solely rely on genetic mutations but could also involve reversible, non-genetic mechanisms. Recent studies have highlighted the existence of drug-tolerant persister cells (DTPs), a subpopulation of cancer cells that can survive short-term therapeutic pressure without acquiring resistance-associated genetic alterations. These cells exhibit a temporary yet reversible tolerance to the initial treatment while also acquiring cross-tolerance to other anti-cancer therapies. The presence of DTPs underscores a dynamic and complex plasticity in tumours, wherein cancer cells can utilise epigenetic rewiring, metabolic reprogramming, and specific signalling pathways to transit between drug-tolerant and drug-sensitive states to adapt to environmental pressures. Furthermore, this adaptive resilience enables DTPs to act as a reservoir for the development of genetically stable resistance, resulting in cancer therapy failure and eventual relapse. In this mini-review, we examine recent evidence on DTPs to provide an overview of their characteristics, development, and survival mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f885f194fb8254e0480e9608eccdddbd6818a86e" target='_blank'>
              Targeting Cancer Drug-Tolerant Persister Cells in Minimal Residual Disease
              </a>
            </td>
          <td>
            Wei Zhang, Sorayut Chattrakarn, Fuhui Chen, Haobin Chai, Mariana Maranga, Jingwei Zhang
          </td>
          <td>2025-05-27</td>
          <td>International Journal of Drug Discovery and Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The advent of single-cell RNA-seq has permitted a global view of the transcriptomic variation across cancer cells in individual tumors and revealed functionally-related sets of genes - a gene module - underpin a cancer cell state. While single-cell transcriptomic atlases have provided a descriptive characterization of cancer cell states across tumor types, we lack a mechanistic framework on the constraints of tumor heterogeneity. Therefore, we propose a ‘developmental constraint’ model - where cancer cell state transitions are restricted by the organism’s developmental map, wherein from the cell-of-origin the malignant cells traverse states that are accessible by its developmental trajectory, both toward progenitor-like states and differentiated-like states of adjacent lineages. Our model posits a constraint on the emergence of states, even under evolutionary pressures such as therapeutic intervention. To test this framework, we modeled non-genetic mechanisms of acquired resistance to KRAS inhibitors - histological adeno-to-squamous transition (AST). Multi-omic analysis of 68 patients enrolled in the KRYSTAL-1 clinical trial revealed a squamous gene signature correlated with a poor response specifically in patients with co-occurring KRASG12C;STK11 (KCL) mutations. We recapitulated AST upon resistance to KRASG12C inhibition in KCL genetically engineered mouse models and tumor-derived organoids. Single-cell transcriptomic and epigenomic of tumor-derived organoid revealed that AST is mediated by a two-step transition - from adeno-like states to an intermediate highly plastic cell state expressing markers of mixed-identity to enriched for squamous-like states. These findings support the developmental constraint model where cell state transitions are mediated by progenitor states in contrast to a trans-differentiation model. Furthermore, we hypothesized that cancer cell state transitions are restricted by the cell’s chromatin context and specific chromatin modifiers are required for cell state transitions analogous to development. Therefore, we leveraged a time-course Perturb-seq approach against 191 chromatin modifiers to identify functional requirements for the adeno-to-intermediate and intermediate-to-squamous transition. Using a module-based method, we found cells with perturbations of the SWI/SNF complex (Arid1a, Smarca4 and Smarce1) did not transition to the squamous cell state but were restricted to the intermediate highly plastic cell state, suggesting that the SWI/SNF complex is required for the intermediate-to-squamous transition. Together, we have showed evidence that cell state transitions in cancer may be explained by the developmental constraint model and revealed that these constraints may act at the chromatin level where chromatin modifiers and chromatin states enable AST plasticity during adaptation to KRAS inhibitors.


 Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, Kwok-Kin Wong, Itai Yanai. Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4426.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Triple negative breast cancer (TNBC) is a breast cancer subtype that manifests fast disease progression and very poor overall survival. Despite initial response to standard of care, development of therapy resistance remains the major challenge in TNBC clinical management. Delineating the mechanisms underlying resistance development has been impeded by the still unknown cell population of origin and lack of longitudinal molecular profiling. To investigate this question, we applied a CRISPR based lineage tracing system into TNBC cell lines. Longitudinal samples from an Antibody Drug Conjugate (ADC), Sacituzumab Govitican sensitive to resistance were collected in vitro via stepwise drug dose escalation and in vivo via xnenograft serial transplantation. We leveraged cellular barcoding with single cell multiome from 10X genomics to simultaneously capture clonal identity, accessible chromatin and gene expression and hence, directly linked lineage with functional cell states. Preliminary results suggest that TNBC cells adopted distinct cell states shortly after drug exposure with characteristics including upregulation of epithelial-to-mesenchymal transition (EMT) genes, stemness markers and downregulation of immune related pathways. Clonal level analysis revealed only a subset of clones survived initial treatment and are enriched in specific “primed” cell states. Next, we will construct single cell lineage trees and quantify clonal dynamics and cell state transition rates. We will identify and validate regulators of cellular plasticity and explore if potential drug combinations will prevent emergence of resistance cell state. Overall, this study will build a comprehensive map of the phenotypic trajectory towards chemotherapy resistance and pinpoints biomarkers of therapy response in triple negative breast cancer.


 Chufan Zhang, Chu Pan, Dave Cescon, Mathieu Lupien. Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5541.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96dda01278fdb738773a58b49aca47aa0308c83d" target='_blank'>
              Abstract 5541: Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing
              </a>
            </td>
          <td>
            Chufan Zhang, Chu Pan, D. Cescon, M. Lupien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 KRAS G12C inhibitors (G12Ci), including the FDA-approved Sotorasib and Adagrasib, represent a significant advancement in targeted cancer therapies. As their use in non-small cell lung cancer patients expands, addressing the pervasive challenge of drug resistance is critical. Resistance mechanisms can be innate, acquired, or both, often coexisting within complex scenarios. Recent studies have highlighted various resistance mechanisms to G12Ci, such as oncogene amplification, activation of alternative signaling pathways, additional mutations in KRAS or its associated networks, and adaptive processes like epithelial-to-mesenchymal transition. Identifying the primary drivers and predominant mechanisms of resistance is crucial for developing effective countermeasures. To tackle these challenges, we developed Oligo-CALL, an innovative cellular barcoding technology that employs antisense oligonucleotide-inducible CRISPRa for lineage labeling. This technique allows for precise clonal tracking and the retrospective isolation of clones. Using Oligo-CALL, we analyzed tumor clonal heterogeneity and traced the evolution of resistance in lung cancer cells treated with G12Ci. Treatment with Sotorasib resulted in reduced clonal diversity, indicative of strong selective pressure. Similar clonal compositions across different replicates suggest that intrinsic mechanisms significantly contribute to G12Ci resistance. Notably, about 30% of resistant clones showed increased expression of the Kras G12C allele, primarily due to genomic gains, pointing to chromosomal instability (CIN) as a key factor in resistance development. Comprehensive analyses including fluorescent karyotyping, whole-genome, and transcriptome sequencing of isolated tumor clone pairs, revealed extensive genomic complexity and increased heterogeneity under G12Ci treatment. Resistant clones exhibited specific increases in copy number variations, chromosomal structure variations, and additional single nucleotide mutations, further underscoring CIN as a driving force behind G12Ci resistance. Clones that underwent drug treatment displayed higher CIN scores compared to their treatment-naïve counterparts. Intriguingly, cells resistant to G12Ci demonstrated increased sensitivity to PARP inhibitors, suggesting that targeting DNA repair pathways could induce synthetic lethality, offering a potential strategy to overcome resistance in lung cancer cells.


 Dingcheng Gao, Yingzhuo Liu, Yi Ban. Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 419.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bca1af35819fcfb6f9d7674ff4443096f1ec742" target='_blank'>
              Single-Cell Pharmacogenomic Landscapes of Epigenetic Therapy in Gastric Cancer
              </a>
            </td>
          <td>
            Chang Xu, Haoran Ma, T. Sheng, M. Lee, S. A. A. Abdul Ghani, Laura Perlaza-Jiménez, K. Huang, Shen Kiat Lim, Supriya Srivastava, Xuewen Ong, S. Tay, S. Ho, Angie Tan, Feng Zhu, Vincenzo Nasca, G. Randon, Hassan Ashktorab, A. S. Cheng, A. Jeyasekharan, Shang Li, Ming Teh, Raghav Sundar, David R. Powell, Joseph Rosenbluh, F. Pietrantonio, W. Yong, J. So, Patrick Tan
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Drug resistance in cancer therapy remains a significant challenge, driven by the ability of cancer cells to evade treatment through complex epigenetic adaptations. Identifying distinct molecular and structural signatures in drug-resistant cells is crucial for uncovering predictive biomarkers and understanding the mechanisms underlying resistance. In this study, we combine highly multiplexed fluorescence imaging and label-free quantitative phase imaging technologies with genomic analyses to investigate the morphological and molecular alterations associated with drug resistance. Using the colon cancer cell line SW480, we developed a drug-resistant phenotype by gradually increasing concentration of 5-fluorouracil (5-FU). Our approach includes morphological analysis, profiling 26 proteins and chromatin modifications, and transcriptomic evaluation. Genomic analyses—including CUT&RUN, RNA-seq, and ATAC-seq—further characterize histone modifications, gene expression, and chromatin accessibility in drug-resistant and parental cells. Our analysis reveals distinct morphological features in drug-resistant (DR) cells compared to their parental counterparts, such as increased morphological plasticity, an elongated shape and pronounced protrusions characteristic of cells undergoing epithelial-to-mesenchymal transition (EMT). A deep neural network analysis of these morphological features effectively distinguishes drug-resistant cells from parental cells, achieving a high area under the receiver operating characteristic curve (AUC) of 0.95. Biomarker expression analysis identifies significant changes in protein expression in drug-resistant cells, notably H3K27me3, H3K27ac, CD44, and EZH2, indicating substantial epigenetic reprogramming and increased stemness. Gene ontology analysis highlights the upregulation of Wnt and EMT pathways, alongside the downregulation of cell adhesion and tight junction pathways, reflecting enhanced invasiveness in drug-resistant cells. Additionally, ATAC-seq analysis reveals a more compact chromatin landscape in drug-resistant cells, consistent with epigenetic modifications that drive transcriptional repression of specific pathways. In conclusion, our multiscale high-content imaging platform proves to be a powerful tool for uncovering the morphological and molecular characteristics of drug-resistant cancer cells. We demonstrate that drug-resistant cells adopt mesenchymal-like morphology, coupled with significant epigenetic reprogramming, chromatin compaction, and enhanced stemness. These results underscore the interplay between morphological adaptations and epigenetic alterations in the development of drug resistance. Our study provides critical insights into the mechanisms driving resistance and lays the groundwork for identifying novel biomarkers and therapeutic targets.


 Yaxin Yang, Jianquan Xu, Chaojie Zhang, Hongqiang Ma, Sarah J. Hainer, Yang Liu. Multiscale high-content imaging to identify molecular and morphological signatures in drug-resistant cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6356.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eed8fdb3c30cedec701229d2292e4e2a7ab7ba3" target='_blank'>
              Abstract 6356: Multiscale high-content imaging to identify molecular and morphological signatures in drug-resistant cancer cells
              </a>
            </td>
          <td>
            Yaxin Yang, Jianquan Xu, Chaojie Zhang, Hongqiang Ma, Sarah J. Hainer, Yang Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Resistance to cancer treatment remains a significant obstacle in curing cancer patients. Despite recent advancements in novel cancer drugs, patients often experience treatment resistance even after initially responding to therapy, leading to tumor relapse. To understand how resistance develops against standard cancer therapies, it is essential to study the gradual changes leading to full resistance. Recent research shows that cancer cells can evade the effects of chemotherapy and targeted therapies by entering a reversible, slow-growing phase called the drug-tolerant persister (DTP) state. In this DTP state, cancer cells survive drug treatment long enough to begin accumulating mutations and eventually develop additional mechanisms of irreversible resistance. Patient-derived organoid (PDO) models, also known as HUB Organoids®, are directly derived from patient tissues, making them more physiologically relevant than traditional cell lines. This study highlights the value of PDO models in studying drug persistence and resistance, as they are more likely to reflect in-vivo conditions. Here, we demonstrate the successful development of colorectal cancer (CRC) PDO models that can persist and resist standard therapies. These models were characterized using whole exome sequencing and bulk RNA sequencing to assess their persistent or resistant state and to identify potential pathways and new drug targets involved in resistance. In conclusion, developing persistent and resistant PDO models is essential for understanding the mechanisms behind drug resistance. This research contributes to developing patient-relevant preclinical models to develop more effective and personalized cancer treatments and expands our overall understanding of how resistance develops in cancer patients.


 Yasmine Abouleila, Timo Voskuilen, Mayke Doorn, Roel Verkerk, Joris Maas, Carla Verissimo, Sylvia F. Boj. From persistence to resistance: insights into the mechanisms of drug tolerance in cancer therapy using patient-derived organoids [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5551.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9151033e1d6669563430e41bef627c6f841c5064" target='_blank'>
              Abstract 5551: From persistence to resistance: insights into the mechanisms of drug tolerance in cancer therapy using patient-derived organoids
              </a>
            </td>
          <td>
            Yasmine Abouleila, Timo Voskuilen, M. Doorn, Roel Verkerk, Joris Maas, C. Veríssimo, Sylvia F. Boj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chemotherapy remains a prevalent strategy in cancer therapy; however, the emergence of drug resistance poses a considerable challenge to its efficacy. Most drug resistance arises from the accumulation of genetic mutations in a minority of resistant cells. The mechanisms underlying the emergence and progression of cancer resistance from these minority-resistant cells (MRCs) remain poorly understood. This study employs force-induced remnant magnetization spectroscopy (FIRMS) alongside various biological investigations to reveal the mechanical pathways for MRCs fostering drug resistance and tumor progression. The findings show that minority Paclitaxel-resistant cancer cells have enhanced mechanical properties. These cells can transmit high-intensity forces to surrounding sensitive cells (SCs) through the force transducer, Merlin. This force transmission facilitates the assimilation of surrounding SCs, subsequently strengthening the contraction and adhesion of tumor cells. This process is termed "mechano-assimilation," which accelerates the development of drug resistance and tumor progression. Interestingly, disturbances and reductions of mechano-assimilation within tumors can restore sensitivity to Paclitaxel both in vitro and in vivo. This study provides preliminary evidence highlighting the contribution of MRCs to the development of drug resistance and malignancy, mediated through mechanical interactions. It also establishes a foundation for future research focused on integrating mechanical factors into innovative cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f46cb96deeea2cf4ed683e45346444a026b4be6" target='_blank'>
              Mechanisms and Therapeutic Strategies for Minority Cell-Induced Paclitaxel Resistance and Tumor Progression Mediated by Mechanical Forces.
              </a>
            </td>
          <td>
            Xueyan Feng, Di Zhang, Guoxun Wang, Liwei Lu, Feng Feng, Xiuyu Wang, Chanchan Yu, Yahong Chai, Jin Zhang, Wenchao Li, Jing Liu, Hongxia Sun, Li Yao
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer remains a significant global health burden due to its high morbidity and mortality. Oncogene-targeted therapy and immunotherapy have markedly improved the 5-year survival rate in the patients with advanced or metastatic tumors compared to outcomes in the era of chemotherapy/radiation. Nevertheless, the majority of patients remain incurable. Initial therapies eliminate the bulk of tumor cells, yet residual populations termed drug-tolerant persister cells (DTPs) survive, regenerate tumor and even drive distant metastases. Notably, DTPs frequently render tumor cross-resistance, a detrimental phenomenon observed in the patients with suboptimal responses to subsequent therapies. Analogous to species evolution, DTPs emerge as adaptative products at the cellular level, instigated by integrated intracellular stress responses to therapeutic pressures. These cells exhibit profound heterogeneity and adaptability shaped by the intricate feedforward loops among tumor cells, surrounding microenvironments and host ecology, which vary across tumor types and therapeutic regimens. In this review, we revisit the concept of DTPs, with a focus on their generation process upon targeted therapy or immunotherapy. We dissect the critical phenotypes and molecule mechanisms underlying DTPs to therapy from multiple aspects, including intracellular events, intercellular crosstalk and the distant ecologic pre-metastatic niches. We further spotlight therapeutic strategies to target DTP vulnerabilities, including synthetic lethality approaches, adaptive dosing regimens informed by mathematical modeling, and immune-mediated eradication. Additionally, we highlight synergistic interventions such as lifestyle modifications (e.g., exercise, stress reduction) to suppress pro-tumorigenic inflammation. By integrating mechanistic insights with translational perspectives, this work bridges the gap between DTP biology and clinical strategies, aiming for optimal efficacy and preventing relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a66d9e3af1fc508d0ed83941eccffc587ef2ea" target='_blank'>
              Drug-induced tolerant persisters in tumor: mechanism, vulnerability and perspective implication for clinical treatment
              </a>
            </td>
          <td>
            Shujie Liu, Anfeng Jiang, Faqing Tang, Minghao Duan, Bin Li
          </td>
          <td>2025-05-24</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Melanoma, a highly aggressive form of skin cancer, presents a significant clinical challenge. The identification of BRAFV600E mutations as a driver in roughly half of all melanoma cases has led to the development of targeted therapies, exemplified by Vemurafenib (Vem), which inhibits overactivated MAPK signaling. However, frequent relapses remain a major obstacle. To investigate the complex mechanisms driving drug resistance in melanoma, we developed PerturbFate, a scalable single-cell combinatorial-indexing strategy that integrates multi-modal profiling with CRISPR perturbations. This approach simultaneously captures newly synthesized and pre-existing transcriptomes, chromatin accessibility, and CRISPR sgRNA identity from individual cells. Using this platform, we profiled BRAFV600E melanoma cells (A375) after knocking down each of 143 potential drug resistance candidate genes, both with and without Vem treatment, in a pooled manner. We identified extensive phenotypic state transitions in melanoma cells under genetic and chemical perturbations, revealing that an undifferentiated state is associated with enhanced drug resistance, while a more benign, Vem-responsive melanocytic state exists at the opposite end of the phenotypic spectrum. Temporal gene regulatory network analyses revealed that AP-1 transcription factors (TFs), particularly FOSL1, pioneer other diverse TFs to drive the melanoma dedifferentiation. In contrast, SOX10 is the key determinant of the melanocytic state. We identified MAPK pathway, which primarily promotes proliferation, and Hippo/YAP signaling, which predominantly drives the undifferentiated state transition, as convergent key pathways mediating drug resistance. Their synergistic interplay, executed through the collaborative activation of downstream TFs (e.g., FOSL1, RREB1, SMADs, KLF5)-mediated gene expression, is a critical driver of Vem resistance. To elucidate the mechanisms underlying the observation that perturbation of genes encoding the Mediator Complex components (e.g., MED12, CCNC, MED13, MED19, MED15, MED24) induced autonomous dedifferentiation and/or resistance to Vem in melanoma, we established linkages between their distinct biochemical properties and heterogeneous perturbation responses. We also identified the convergence of dysregulated inflammation, potentially mediated by VEGFC and interleukins (ILs), with Hippo/YAP-mediated transcriptional responses in driving melanoma dedifferentiation and Vem resistance following perturbation of mediator complex genes. Altogether, our study introduces a novel single-cell multi-modal CRISPR perturbation platform, which we applied to comprehensively chart the responses of melanoma cells to the perturbation of all potential Vem resistance candidate genes. These findings pave the way for the identification of precise targets for combination therapies with enhanced efficacy against melanoma drug resistance.


 Zihan Xu, Ziyu Lu, Aileen Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao. Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Karyotyping, SNP arrays, fluorescence in situ hybridisation and next generation sequencing techniques have greatly improved our understanding of the genetic drivers of acute lymphoblastic leukemia (ALL). In the past years, another layer of genetic data has been added by the study of mutational signatures, patterns of somatic mutations that represent specific mutational mechanisms. Mutational signatures can give insight in tumor development, but also reveal mutagenic side-effects of treatment in relapse samples. Multiple treatment-related mutational signatures have been detected in relapsed ALL that could play a role in therapy resistance and relapse development. In fact, multiple pathogenic driver mutations have been attributed to these treatment-related mutational processes, including a recurrent TP53 mutation. Studies in childhood ALL revealed that thiopurine exposure is the most common source of therapy-related mutagenicity in ALL and presents differently when patients are DNA mismatch repair deficient. Thiopurine-induced DNA damage indicates that leukemic cells were able to survive thiopurine exposure. This could be due to metabolic defects, acquired mutations that induce thiopurine resistance during treatment or resistance to drugs synergizing with thiopurines. In this review, we discuss the types and prevalence of treatment-related mutational signatures in ALL, and explore mechanisms of thiopurine cytotoxicity and mutagenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f4deac881b3b7a4007820d8b07d9cc82f4007b" target='_blank'>
              Treatment-related mutagenic processes in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Cédric G van der Ham, F. V. van Leeuwen, Roland P Kuiper
          </td>
          <td>2025-05-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Resistance to EGFR and ALK tyrosine kinase inhibitors (TKIs) in lung adenocarcinoma (LUAD) remains a major barrier to effective treatment. Tumor adaptation frequently involves dynamic cellular state transitions, altered signaling networks, and interactions within the tumor microenvironment (TME). Understanding the mechanisms driving resistance in EGFR- and ALK-mutant tumors is critical to developing strategies to improve patient outcomes. This study investigates transcriptomic changes underlying TKI resistance and persistance, combining patient data with functional insights from PDX models. Tumor biopsies from 11 LUAD patients were collected across three treatment stages: baseline (n=6), minimal residual disease (MRD, n=4), and relapse (n=1). Patient-derived xenograft (PDX) models were generated from pretreated tumors with EGFR and ALK mutations. Tumors were grafted into five mice per model, with three receiving treatment until stable residual disease emerged and two serving as untreated controls. Single-nucleus RNA sequencing was performed on both patient tumor samples and PDX models to comprehensively profile transcriptomic changes. Transcriptomic changes were observed in both patient tumor samples and PDX models during TKI treatment. In patients, at MRD, a novel subpopulation expressing mesenchymal and neuronal markers was identified, accompanied by altered cell-cell communication involving neuronal signaling (RELN), stromal remodeling (FGFRs), and survival pathways (MAPK). At relapse, signatures of epithelial-mesenchymal transition (EMT), ECM remodeling, and immune evasion were enriched. In PDX models, most persister cells were arrested in the G1 phase and showed suppression of canonical cancer pathways, while notable upregulation of the ciliated pathway was observed across models. This study provides a high-resolution transcriptomic map of TKI-treated LUAD tumors. Resistance mechanisms in EGFR- and ALK-mutant tumors include neuronal, EMT, MAPK signaling, ECM remodeling, and activation of the ciliated pathway. These findings offer novel therapeutic targets to combat resistance, leveraging the integration of patient-derived data and functional insights from PDX models.


 Naomi Mfonfu, Floriane Brayé, Galina Boldina, Matteo Cesaroni, Luc Fibroulet, Sergey Nikolaev, Andrey Yurchenko. Single-cell transcriptomic profiling of longitudinal patient tumors and PDX models reveals resistance mechanisms in ALK- and EGFR-mutant LUAD [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df82979a5771df69f4d6ba63039b4660f6ec17f" target='_blank'>
              Abstract 5065: Single-cell transcriptomic profiling of longitudinal patient tumors and PDX models reveals resistance mechanisms in ALK- and EGFR-mutant LUAD
              </a>
            </td>
          <td>
            Naomi Mfonfu, F. Braye, Galina Boldina, Matteo Cesaroni, Luc Fibroulet, Sergey I. Nikolaev, Andrey A. Yurchenko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Chemotherapy resistance is a major contributor of cancer-related deaths. Resistance can either be acquired during treatments or pre-exist at the time of the diagnosis. Previous studies on high-grade serous carcinoma (HGSC) have shown contradictory findings on the role of genetic causes explaining emergence of chemotherapy resistance at relapses, for example selection of resistant clones during chemotherapy. HGSC is often diagnosed at an advanced stage, showing multiple genetically heterogeneous clones present before therapeutic intervention. While most patients respond well to primary therapy, cancer relapses within few years, leading to treatment resistance. To understand genetic impact on emergence of resistance, we used matched pretreatment and treated samples from 214 unselected HGSC patients collected at DECIDER trial (NCT04846933). We constructed phylogenetic trees based on whole-genome sequencing of 804 tumor samples and whole-exome sequencing of 63 circulating tumor DNA (ctDNA) samples from plasma. Cancer evolution analysis identified patients with and without selection during treatments: patients with remained heterogeneity with stalled evolution, and others dominated by subclones originating from single resistant clone. Patients with better primary therapy outcome were linked to re-expansion of a single major clone at relapse, while multiple ancestral clones founded relapse populations in poorer outcome patients. The single-clone expansion pattern also emerged in later relapses, after multiple lines of therapy. Majority of the relapsed cancers evolved further from the ancestral clones already present at pretreatment. The relapse emerging subclonal mutations affected pathways such as carbohydrate metabolism and cAMP signaling. In contrast, sensitive clones eliminated by treatment carried mutations affecting extracellular matrix and NTRK3 signaling. Interestingly, some cancers did not acquire new mutations at relapse, despite becoming chemoresistant, suggesting that non-genetic mechanisms, such as phenotypic plasticity or microenvironmental factors, might underlie their resistance. In summary, we discovered heterogeneous cancer evolution trajectories during treatment which were linked to mutations in key pathways and prior therapy responses. While chemotherapy often selects persistent cancer subclones that seed later relapses, some patients acquire resistance through non-genetic mechanisms, highlighting the complexity of cancer evolution and resistance dynamics. Understanding these dynamics is crucial to define better therapeutic interventions in these poor prognosis patients with advanced HGSC.


 Jaana Oikkonen, Giulia Micoli, Susanna Holmström, Kari Lavikka, Mai T. Nguyen, Yilin Li, Giovanni Marchi, Anna Rajavuori, Heidi Koskela, Johanna Hynninen, Sampsa Hautaniemi. Response to primary therapy predicts tumor evolution dynamics at relapse in ovarian cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1257.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7191d1be8112623ab8fc75b326b24bf447563f16" target='_blank'>
              Abstract 1257: Response to primary therapy predicts tumor evolution dynamics at relapse in ovarian cancer
              </a>
            </td>
          <td>
            J. Oikkonen, Giulia Micoli, Susanna Holmström, Kari Lavikka, M. T. Nguyen, Yilin Li, G. Marchi, A. Rajavuori, Heidi Koskela, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a highly aggressive primary brain tumor with a median survival of less than two years. Despite standard therapy of surgery, radiation, and chemotherapy, GBMs recur. These recurrences may partially be due to glioblastoma stem cells (GSCs). Various subpopulations of GSCs exist, likely attributing to different resistant niches. The mesenchymal (MES) subpopulation may drive radiation (RT) resistance. The transcriptomic shifts that define GSC adaptation to RT remain unknown. This study explores the transcriptomic reprogramming of MES GSCs following RT to identify resistance mechanisms.



 Using dataset GSE162931 from the GEO database and MES gene signatures from PMID:31327527, we analyzed two patient-derived GBM cell lines pre- and post-10 Gy RT. Data for the two cell lines were preprocessed and integrated with Seurat, followed by clustering and UMAP visualization with 2D kernel module-score weighted density plots. Cluster proportion changes between conditions were statistically examined using Chi-squared or Fisher’s tests with adjusted p-values. Differential expression analysis revealed conserved and unique gene markers distinguishing treated from naïve conditions, visualized in volcano plots and heatmaps. A final score integrating module scores and cluster proportions quantified MES traits at the single-cell level. All analyses were conducted in R studio version 4.2.3.



 UMAP, density plots, and cluster proportion changes indicate significant shifts in MES GSC distributions post-RT, suggesting RT selection for specific transcriptomic profiles. Highly MES subclusters exhibit distinctive RT-responsive gene expression patterns. The final scoring revealed enhanced MES traits in post-RT clusters. Distinct clustering patterns in MES GSCs highlight diverse adaptive strategies. Differential expression analysis reveals upregulation of DNA repair genes (SAT1, NDRG1, GADD45A), enhancing genomic stability post-RT, and stemness markers (NES, GDF15), emphasizing self-renewal capacity. Genes supporting oxidative stress management (SQSTM1, DDIT4) were upregulated, while downregulated metabolic genes (PGK1, LDHA) indicate a shift towards oxidative phosphorylation. Gene Set Enrichment Analysis identified enriched pathways related to DNA repair, cell migration, and differentiation regulation.



 These findings underline the transcriptomic plasticity in MES GSCs that drive RT resistance. Potential targets like AKAP12 and S100A4, essential for maintaining stem-like traits and MES phenotypes, present new therapeutic targets that could be used to combat resistant GSC populations, aiming to reduce tumor recurrence and enhance GBM patient outcomes.



 Grace Rosner, Abigail Goen, Godwin Peasah-Darkwah, Jonathan Rodgers Gochicoa, Alos Diallo, Gokul Srinivasan, Elisa Bu Sha, Joshua Levy, Kelli B. Pointer. Radiation-induced transcriptomic reprogramming of mesenchymal glioblastoma stem cells reveals key pathways and potential targets [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7502.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b60f1dd8ce9984467648ff9cf2c70d381e498a6" target='_blank'>
              Abstract 7502: Radiation-induced transcriptomic reprogramming of mesenchymal glioblastoma stem cells reveals key pathways and potential targets
              </a>
            </td>
          <td>
            Grace M. Rosner, Abigail Goen, Godwin Peasah-Darkwah, Jonathan Rodgers Gochicoa, Alos B. Diallo, Gokul Srinivasan, Elisa Bu Sha, Joshua Levy, Kelli B. Pointer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Poly ADP-ribose polymerase (PARP) inhibition effectively treats prostate cancer (PCa) by inducing replication stress and cell death through synthetic lethality. However, PARP inhibitor (PARPi) resistance inevitably develops, necessitating the study of short- and long-term effects. Responses to PARP inhibition in PCa vary from cytostasis to cell death, with the impact of different dosages and durations underexplored. This study examines how temporal and dosage variations of Olaparib (Ola) influence PCa cell phenotypes and identifies mechanisms underlying treatment response and putative therapeutic strategies to address vulnerabilities following PARP inhibition.



 C4-2B cells were treated with 0, 1, and 5 µM Olaparib for 1 and 5 days. RNA sequencing (RNA-seq), gene set enrichment analysis (GSEA), cell growth assays, microscopy, and western blotting were used to identify pathway alterations and putative vulnerabilities.



 C4-2B cells respond well to clinically relevant doses of olaparib. Morphological assessment at 1 and 5 days suggests significant differences in response based on duration and intensity of treatment. After 1 day, RNA-seq and western blots support a dominant p53-associated cell cycle checkpoint response which may be driven by ATM. By day 5, surviving cells exhibited significant morphology changes indicating epithelial-mesenchymal transition (EMT) and suggesting acquisition of a stress-induced phenotype. RNA-seq and GSEA support phenotypic shifts including EMT and significant changes in fatty acid metabolism. These findings suggest that prolonged PARP inhibition leads to adaptations in surviving cells. Strategies to either co-target cell cycle checkpoint activation or putative survival adaptations show promise in combination with olaparib.



 PARP inhibition induces distinct cellular and metabolic changes based on dosage and duration. Acute exposure may trigger p53-mediated cell cycle arrest, while prolonged exposure results in phenotypic shifts which may promote cell survival. Our study underscores the need for the design of rational combination therapies to address both immediate and long-term effects of treatment.



 Love A. Moore, Akshaya Karthikeyan, Bryan Correa Gonzalez, Anamitra Bhaumik, Ethan Sandoval, Alan Paul Lombard. Mechanistic and phenotypic insights into temporal and dosage-dependent responses to PARP inhibition in advanced prostate cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 377.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02475b4b90f0316a8b2e0b8f1c3f51f20ea0bcc8" target='_blank'>
              Abstract 377: Mechanistic and phenotypic insights into temporal and dosage-dependent responses to PARP inhibition in advanced prostate cancer
              </a>
            </td>
          <td>
            Love A. Moore, Akshaya Karthikeyan, Bryan Correa Gonzalez, Anamitra Bhaumik, Ethan Sandoval, A. P. Lombard
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Both intrinsic and acquired resistance limit the depth and duration of response to all clinically evaluated anti-KRAS therapies. Genetic resistance mechanisms, identified in nearly half of relapsed patients, include acquisition of mutations that effectively restore KRAS-dependent activities. Reported non-genetic mechanisms include signaling and transcriptional activities that support epithelial-to-mesenchymal transition (EMT) or transition to cancer subtypes less responsive to KRAS inhibition. We recently reported MYC amplification and TEAD-dependent transcriptional activation by YAP as drivers of resistance to the multi-RAS inhibitor RMC-7977 (RASmulti), the preclinical analog of RMC-6236. Importantly, we showed that cells resistant to RASmulti that harbor amplified MYC could be re-sensitized to RAS inhibition by co-treatment with TEAD inhibitors. These studies motivated us to define the MYC- and TEAD-dependent transcriptional profiles in KRAS-mutant pancreatic adenocarcinoma (PDAC) cells. We knocked down MYC using siRNA in a panel of eight human PDAC cell lines to derive the MYC-dependent transcriptome. We found 3,010 MYC-dependent genes, 36% of which support oncogenic KRAS signaling through ERK. To determine the TEAD-dependent transcriptome, we treated a panel of three sets of paired naïve and RASmulti-resistant KPC cell lines, derived from an autochthonous mouse model of PDAC, and of six human KRAS-mutant PDAC cell lines with the TEAD inhibitor IAG933, with RMC-7977, or with the combination. We found 3,865 TEAD-dependent genes in mouse cells and 3,590 TEAD-dependent genes in human cells. We found co-regulated RAS, MYC, and TEAD gene transcriptional networks enriched for essential genes that support cell cycle progression. We also found distinct gene clusters regulated by each protein that support distinct molecular functions, such as protein biosynthesis, RHO GTPases, and immune signaling. Importantly, combined inhibition of TEAD and RAS led to a more robust downregulation of essential genes than either single treatment. Proteomic profiling of PDAC cell lines with acquired resistance to RMC-7977 showed upregulation of TEAD-dependent gene products, supporting TEAD activation as a non-genetic mechanism of acquired resistance. Together, our findings expand the role of MYC and TEAD in supporting PDAC cell growth and resistance to RAS inhibition and provide rationale for further evaluation of combination therapeutic strategies to limit resistance.


 Brandon L. Mouery, Priya S. Hibshman, A Cole Edwards, Ryan D. Mouery, Runying Yang, Jeff A. Klomp, Angie L. Mordant, Thomas S. Webb, Aurora Cabrera, Laura E. Herring, Adrienne D. Cox, Timour Baslan, Channing J. Der, Clint A. Stalnecker. Determination of the MYC- and TEAD-dependent transcriptome in pancreatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB288.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0847dafec64111106a227406998f89b030109af3" target='_blank'>
              Abstract LB288: Determination of the MYC- and TEAD-dependent transcriptome in pancreatic cancer
              </a>
            </td>
          <td>
            Brandon L. Mouery, Priya S. Hibshman, A. C. Edwards, Ryan D. Mouery, Run-Wei Yang, Jeffrey A. Klomp, Angie L Mordant, Thomas S. Webb, Aurora Cabrera, Laura E. Herring, Adrienne D. Cox, T. Baslan, C. Der, Clint A. Stalnecker
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15e9d70c7ccc9d91ae61ea8b1915d90983e21823" target='_blank'>
              Genomic Insights into Oral Cancer Highlight Mutant SIGMAR1 as a Critical Target to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            P. S. Chagas, C. B. Garcia, A. M. Leopoldino
          </td>
          <td>2025-04-21</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Uncontrolled proliferation is one of the most important hallmarks of cancer. Consequently, cancer cells exhibit a high demand for oxygen and nutrients from the tumor microenvironment. Nonetheless, cancer cells in the core region of a solid tumor often undergo nutrient and oxygen deficiency owing to the restricted vascularization, ultimately leading to the occurrence of metabolic stress. To survive under stress, series of stress response will be activated, providing opportunities for cancer cells to confront unfavorable circumstances and further developing phenotypic plasticity. Without altering genetic sequence, phenotypic plasticity enables cancer cells to adjust their behavior to survive from stress. This critical ability of cancer cells is orchestrated by multiple stress response under different stress stimuli, but its regulatory mechanism upon metabolic stress still requires further investigation. We identified a transcription factor EGR1 that is rapidly induced upon metabolic stress in breast cancer (BC). Known as an immediate early gene, EGR1 can be upregulated by multiple types of stimulation and further regulates tumor progression. Through cell viability and anoikis assay, we found that EGR1 could promote BC cell survival upon metabolic stress under both attach and detach condition. Furthermore, through xenograft tumor model, we noticed the inhibitory effect of EGR1 on tumor growth was more obvious in the late phase of tumor development, when metabolic stress mostly occurred. These results demonstrated the role of EGR1 in regulating phenotypic plasticity to promote BC cell growth upon metabolic stress. To identified genes regulated by EGR1 upon metabolic stress, EGR1-based ChIP-Seq was performed, and further compared with glucose deprivation RNA-Seq data to identify overlapping genes. Being the most upregulated overlapping gene upon glucose deprivation, HERPUD1 was chosen for further validation. HERPUD1 has been found to be important for degrading unfolded or misfolded protein in unfolded protein response (UPR). However, the role of HERPUD1 upon metabolic stress remains elusive. We found HERPUD1 expression significantly increased upon metabolic stress and was higher in EGR1 knockdown (KD) cells, suggesting EGR1 was a transcriptional suppressor upon metabolic stress. As a critical stress response to promote phenotypic plasticity, UPR was diminished in EGR1 KD cells due to higher expression of HERPUD1, potentially explaining the increased vulnerability of these cells to metabolic stress. Collectively, EGR1 was identified as a transcription factor significantly upregulated upon metabolic stress. By suppressing HERPUD1 expression, EGR1 maintained UPR at an optimal level, which promoted the development of phenotypic plasticity of BC cells and assisted cells to survive and thrive from metabolic stress.


 Yi-Zhen Wu, Ge-Jyun Liu, Shan-Ting Lei, Wei-Huan Zeng, Mei-Ju Juan, Yu-Shu Liu, Yong-Han Su, David K. Ann, Ching-Ying Kuo. EGR1 regulates phenotypic plasticity to promote breast cancer progression upon metabolic stress through elaborating HERPUD1-mediated unfolded protein response [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1512.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a113f78cc11aa5cfca10446523fbea0eb55d404" target='_blank'>
              Abstract 1512: EGR1 regulates phenotypic plasticity to promote breast cancer progression upon metabolic stress through elaborating HERPUD1-mediated unfolded protein response
              </a>
            </td>
          <td>
            Yi-Zhen Wu, Ge-Jyun Liu, Shan-Ting Lei, Wei-Huan Zeng, Mei-Ju Juan, Yu-Shu Liu, Yong-Han Su, David K. Ann, Ching-Ying Kuo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are propelled by the remarkable plasticity of signaling networks, which dynamically rewire under selective pressures to maintain proliferation, enable immune evasion and promote metastasis. Despite advances in precision oncology, the dynamic crosstalk between tumor cells, non-coding genomes and the microenvironment continues to undermine treatment efficacy. This call for submissions, Revolutionizing Cancer Treatment: Navigating the Intricate Landscape of Cellular Signaling Networks, seeks cutting-edge research that dissects these adaptive mechanisms through innovative technologies - from single-cell multi-omics and spatial transcriptomics to AI-powered network modeling. We welcome studies leveraging physiomimetic models (e.g., organoids, 3D-bioprinted ecosystems) to decode tumor heterogeneity, as well as translational work targeting emergent vulnerabilities at the intersection of epigenetics, metabolic reprogramming and stromal interactions. By integrating systems biology with computational and experimental approaches, this collection aims to catalyze the design of adaptive therapies that outmaneuver cancer's evolutionary resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba13faeaf95516c4bb5cbd68aeea6df92e61f127" target='_blank'>
              Revolutionizing cancer treatment: Navigating the intricate landscape of cellular signaling networks.
              </a>
            </td>
          <td>
            Hao Zhang
          </td>
          <td>2025-05-26</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e20582


 Background:
 Despite advances in therapies, the development of drug resistance in non-small cell lung cancer (NSCLC) poses a significant challenge. The emergence of neoantigen-based therapies offer a promising new frontier which enhances treatment efficacy and reduces associated costs.
 Methods:
 A total of 24 patients diagnosed with NSCLC were enrolled in this study. Primary tumor tissues and peripheral blood samples were collected from each patient. Paired whole exome and transcriptomic sequencing data were performed to identify somatic mutations and to quantify their respective expression levels. Somatic single nucleotide variants (SNVs) and small insertions or deletions (inDels) were detected using Strelka algorithm from whole exome sequencing (WES) data. Neoantigen prediction focused on human leukocyte antigens (HLA) type 1. HLA alleles were predicted using WES data encoding adjacent non-cancerous tissue samples, identifying four alleles that were present in at least 50% of the patients. Neoantigens were considered potentially immunogenic if their predicted median IC50 binding scores were ≤500nM while expressed transcripts per million (TPM) exceeds 1 in tumor samples.
 Results:
 Using systematic bioinformatics pipeline, we analyzed 24 NSCLC samples with matched tumor tissues and peripheral blood, and detected the 4677 mutations involving 2442 genes, including 2819 missense SNVs, 941 InDels, 229 gene fusion and 688 RNA alternative splicing events, 94.7% of which (4428 mutations involving in 2247 genes) were common with those reported in the TCGA-LUAD dataset. The average number of mutations was 194.8 in NSCLCs. We found that C>T/G>A transitions/ transversions were dominant. Missense mutations were the most frequent types of somatic mutation in the coding sequence regions. 98% of mutated genes were not detected simultaneously with DNA and RNA mutations.
 ADCK5
 ,
 CD151
 and
 EIF3K
 were identified as potential new driver genes. Genomic variation profiling identified 2213 potential neoantigens with a median IC50 binding score of ≤500nM and a transcript expression level of >1 TPM. Among these, 41 were listed in databases TSNAdb, IEDB, and CTDatabase. A total of 14 genes harboring identical mutations were detected, with the identical mutations observed in over 50% of the patient cohort. Notably, mutations in 14 genes were either gene fusions or RNA alternative splicing events, and core antigenic sequences generated were found to be identical in the majority of patients. Of particular interest is the RPRD2 gene, where an RNA exitrons site was detected in all patient samples, along with the identification of the same neoantigen core sequence.
 Conclusions:
 Our analysis suggests that gene fusions and RNA splicing variations are more likely than point mutations to generate shared antigens, underscoring their importance for developing targeted immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1725e99e0357b9a0ce8d33b25d7c68d81214eed0" target='_blank'>
              Comprehensive genomic alterations identification and potential shared neoantigens for non-small cell lung cancer immunotherapy.
              </a>
            </td>
          <td>
            Muchun Zhu, Qing Zhao Ruan, Boyue Zhang, Weiqiang Yin, Run Sheng Ruan
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Therapy-induced senescence (TIS) is considered a permanent cell cycle arrest following DNA-damaging treatments; however, its irreversibility has recently been challenged. Here, we demonstrate that escape from TIS is universal across breast cancer cells. Moreover, TIS provides a reversible drug resistance mechanism that ensures the survival of the population, and could contribute to relapse. Methods TIS was induced in four different breast cancer cell line with high-dose chemotherapy and cultured until cells escaped TIS. Parental, TIS and repopulating cells were analyzed by bulk and single-cell RNA sequencing and surface proteomics. A genetically engineered mouse model of triple-negative breast cancer was used to prove why current senolytics cannot overcome TIS in tumors. Results Screening the toxicity of a diverse panel of FDA-approved anticancer drugs revealed that TIS meditates resistance to half of these compounds, despite their distinct mechanism of action. Bulk and single-cell RNA sequencing, along with surface proteome analysis, showed that while parental and repopulating cells are almost identical, TIS cells are significantly different from both, highlighting their transient nature. Furthermore, investigating dozens of known drug resistance mechanisms offered no explanation for this unique drug resistance pattern. Additionally, TIS cells expressed a gene set associated with immune evasion and a potential KRAS-driven escape mechanism from TIS. Conclusion Our results reveal that TIS, as a transient drug resistance mechanism, could contribute to overcome the immune response and to relapse by reverting to a proliferative stage. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02310-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838695afba129f01c7b6f860f5a9601961169be" target='_blank'>
              Therapy-induced senescence is a transient drug resistance mechanism in breast cancer
              </a>
            </td>
          <td>
            Eszter Bajtai, Csaba Kiss, Éva Bakos, Tamás Langó, Anna Lovrics, É. Schád, V. Tisza, Károly Hegedűs, Péter Fürjes, Zoltán Szabó, Gábor E. Tusnády, G. Szakács, Á. Tantos, S. Spisák, J. Tóvári, András Füredi
          </td>
          <td>2025-05-01</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Mismatch repair (MMR) status plays a critical role in colorectal cancer (CRC), influencing tumor biology and response to therapies. MMR-deficient (MMR-d) tumors, characterized by microsatellite instability (MSI), exhibit high mutation burdens and increased T cell infiltration, whereas MMR-proficient (MMR-p) tumors typically demonstrate low T cell density and poor immune checkpoint inhibitor response. Despite well-documented clinical implications, the molecular and cellular differences of T cells in the tumor microenvironment between MMR-d and MMR-p CRC remain incompletely understood.



 Single-cell RNA sequencing (scRNA-seq) data were analyzed using the CellChat R package to identify ligand-receptor interactions enriched in MMR-p colorectal cancer. Differentially enriched pathways involving T cells were identified and subjected to Gene Ontology (GO) enrichment analysis to uncover biological processes specific to these interactions. Validation was performed using spatial transcriptomic data from the Human Tumor Atlas Network (HTAN) to confirm ligand-receptor co-expression in a spatial context.



 CellChat analysis revealed significant enrichment of TNFRSF14-associated signaling pathways (BTLA-TNFRSF14, LTA-TNFRSF14, and TNFS14-TNFRSF14) in MMR-p CRC. Functional analysis of TNFRSF14+ T cells in MMR-p tumors demonstrated upregulation of pathways related to leukocyte migration, negative regulation of T cell activation, and immune effector process regulation, as identified through GO enrichment analysis. Validation using spatial transcriptomic data confirmed co-expression of the BTLA-TNFRSF14 ligand-receptor pair in a single MSS sample but did not support broader enrichment across MMR-p samples.



 These findings highlight distinct ligand-receptor interactions and immune regulatory processes in MMR-p CRC, particularly involving TNFRSF14 signaling. While functional analysis underscores the role of these interactions in shaping T cell activity, spatial transcriptomic validation suggests heterogeneity in their enrichment across samples. This underscores the complexity of immune microenvironment regulation in MMR-p CRC tumors.



 Dimitri Joseph, Alexandra Guillaume. Characterization of mismatch repair status in colorectal cancer using single cell and spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5092.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217101cddf06b5ca15708fdd50f15de9f63e9eb9" target='_blank'>
              Abstract 5092: Characterization of mismatch repair status in colorectal cancer using single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Dimitri Joseph, Alexandra Guillaume
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Glioblastoma (GBM), the most frequent and aggressive primary brain tumor, almost invariably recurs as an incurable disease after standard therapy. Molecular heterogeneity underlies the failure of current treatment and drives GBM progression. Although significant efforts have been directed at resolving the subclonal architecture, identifying and characterizing the genetic subpopulation in GBM has proven challenging. Recent advances in single-cell genomics (scWGS) have provided new opportunities for exploring the genomic complexity of tumors at unprecedented resolution. However, earlier methods of scWGS faced limitations due to poor amplification uniformity resulting in high number of artefacts. The introduction of primary template-directed amplification (PTA) has overcome this challenge allowing for more accurate detection of genomic alterations at single-cell resolution. In this study, we combined PTA scWGS of 5,689 single cells with single-nucleus RNA sequencing (snRNAseq) of 41,071 single nuclei from 25 GBM samples to explore the clonal architecture and the dynamics of genetic and non-genetic subclonal programs activated in GBM. PTA-based scWGS data showed optimal performance in terms of overdispersion and coverage breadth when compared to other single cell methods for measuring somatic copy number alterations (SCNA). The SCNA substructure of GBM was reconstructed by applying a graph-based clustering to scWGS data, which identified a median of 4 subclones per tumor. GBM samples showed a heterogenous distribution of genetic clones constituted by variable numbers of cells. For each sample we computed the consensus SCNA profiles to reconstruct the genetic phylogeny of GBM. The predicted evolutionary model showed a first trajectory from the diploid cell to the most recent common ancestor (MRCA) with the highest number of SCNA (median 14), including frequent GBM driver alterations (chr 7 gain, chr 10 loss, EGFR amp, CDKN2A del). Conversely, number and frequency of SCNA decreased in the clades diversifying the MRCA into the later ancestors. By aligning the scWGS clonal lineages with snRNAseq gene expression profiles from the same tumors, we found significant coordination of genetic and non-genetic mechanisms that diversified tumor subpopulations and led to divergent phenotypes recurring across the patients. The functional phylogeny of GBM identified two main branches of tumor evolution that emerged from the MRCA with the activation of neuronal functions and differentiation in one clade, and the acquisition of cell cycle programs, stem cell activities, and epithelial to mesenchymal transition in the opposite clade, respectively. This multi-omics approach enabled the reconstruction of the evolutionary dynamics of GBM at both genetic and transcriptional levels, revealing critical insights into clonal evolution and therapeutic resistance.


 Fulvio D'Angelo, Aram Ko, Luciano Garofano, Pedro Davila, Luigi Ferraro, Benjamin Currall, Sion Williams, Sakir Gultekin, Ashish Shah, Ricardo Komotar, Michael Ivan, Corneliu Sologon, Alice Laurenge, Alberto Picca, Amel Dridi-Aloulou, Bertrand Mathon, Franck Bielle, Stephan Zuchner, Macarena de la Fuente, Marica Eoli, Chul-Kee Park, Michele Ceccarelli, Marc Sanson, Anna Lasorella, Antonio Iavarone. Exploring genetic and non-genetic evolution of glioblastoma by integrating single cell DNA and RNA sequencing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6350.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a60e85c34b84d3ee7bcc4641260225bbd7766477" target='_blank'>
              Abstract 6350: Exploring genetic and non-genetic evolution of glioblastoma by integrating single cell DNA and RNA sequencing
              </a>
            </td>
          <td>
            F. D’Angelo, A. Ko, L. Garofano, Pedro Davila, Luigi Ferraro, Benjamin Currall, S. Williams, S. Gultekin, Ashish Shah, Ricardo J Komotar, Michael Ivan, Corneliu Sologon, A. Laurenge, A. Picca, Amel Dridi-Aloulou, B. Mathon, Franck Bielle, Stephan Zuchner, M. D. L. Fuente, M. Eoli, Chul-Kee Park, Michele Ceccarelli, Marc Sanson, A. Lasorella, A. Iavarone
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27234afaddab888f1a989f8c718a47f9f017a789" target='_blank'>
              Role of the tumor microenvironment in cancer therapy: unveiling new targets to overcome drug resistance.
              </a>
            </td>
          <td>
            Sri Sathya Sandilya Garemilla, Siri Chandana Gampa, S. Garimella
          </td>
          <td>2025-05-07</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Colon cancer (CC) is one of the most common and lethal cancers worldwide, with rising incidence rates in both developed and developing countries. Although advances in treatments such as surgery, chemotherapy, and targeted therapies have been made, prognosis for advanced colon cancer, particularly with metastasis, remains poor. Recent studies highlight the significant role of post-transcriptional modifications like acetylation in cancer biology, affecting processes like gene transcription, metabolism, and tumor progression. Methods This study applied multi-omics analyses, including single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and Mendelian randomization. Data were obtained from public datasets like GSE132465, UCSC Xena, and GeneCards. We focused on acetylation-related genes, specifically NAT10 and GNE, using scoring methods, cell–cell interaction models, and survival analyses to investigate their role in colon cancer development, metastasis, and immune evasion. Results This study identifies that NAT10 is highly expressed in epithelial cells of colorectal cancer (CC) and is closely associated with tumor progression and metastasis. Single-cell RNA sequencing analysis revealed that NAT10-positive epithelial cells exhibited strong interactions with myeloid cells and T cells, with significant differences in cell–cell communication (p < 0.05). Based-on-summary-data Mendelian randomization (SMR) analysis further supports a causal relationship between NAT10 and colorectal cancer. In the MR analysis, a significant positive correlation was observed between NAT10 and colorectal cancer risk using summary data from genome-wide association studies (GWAS) and expression quantitative trait loci (eQTL) studies (β_SMR = 0.004, p_SMR = 0.041, p_HEIDI = 0.737). These findings suggest that NAT10 may serve as a pathogenic factor in colorectal cancer development, providing additional genetic evidence that links this acetylation-related gene to colorectal cancer. Survival analysis further demonstrated that NAT10-positive epithelial cells are associated with poorer prognosis. In the TCGA dataset, patients with NAT10-positive epithelial cells exhibited a significantly shorter disease-free survival (DFS) (p = 0.012). Unlike GNE-positive cells, NAT10-positive epithelial cells exhibited immune escape characteristics, and TIDE analysis indicated that NAT10-positive epithelial cells were associated with a lower response to immune checkpoint blockade therapy (p = 1.3e−5), suggesting that they may impair the efficacy of immunotherapy by promoting immune evasion. In contrast, GNE was also significantly expressed in epithelial cells of colorectal cancer, but its role differs from that of NAT10. GNE-positive epithelial cells demonstrated strong communication with immune cells, particularly in interactions between myeloid cells and T cells through receptor-ligand pairs. Despite the important role of GNE-positive epithelial cells in the tumor microenvironment, their association with immune escape is weaker compared to NAT10. Survival analysis revealed that GNE-positive epithelial cells were associated with a better prognosis (p = 0.015). In the TCGA dataset, patients with GNE-positive epithelial cells displayed longer disease-free survival (DFS), contrary to the results from the SMR analysis. Conclusions Leveraging SMR and multi-omics analysis, this study highlights the significant role of acetylation-related genes, particularly NAT10, in colon cancer. The findings suggest that acetylation modifications in epithelial cells contribute to immune evasion and cancer progression. NAT10 could serve as a promising biomarker and therapeutic target for early diagnosis and targeted therapy, offering new avenues for improving colon cancer treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37348c834755949c2631ebe0e9fc050634d0c0ce" target='_blank'>
              Deciphering the role of acetylation-related gene NAT10 in colon cancer progression and immune evasion: implications for overcoming drug resistance
              </a>
            </td>
          <td>
            Xuancheng Zhou, Xun Sang, Lai Jiang, Shengke Zhang, Chenglu Jiang, Yuheng Gu, Yiping Fu, Guanhu Yang, Jieying Zhang, Hao Chi, Binbin Wang, Xiaolin Zhong
          </td>
          <td>2025-05-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) has emerged as a pivotal technology in the field of oncology, transforming the approach to cancer diagnosis and treatment. This paper provides a comprehensive overview of the integration of NGS into clinical settings, emphasizing its significant contributions to precision medicine. NGS enables detailed genomic profiling of tumors, identifying genetic alterations that drive cancer progression and facilitating personalized treatment plans targeting specific mutations, thereby improving patient outcomes. This capability facilitates the development of personalized treatment plans targeting specific mutations, leading to improved patient outcomes and the potential for better prognosis. The application of NGS extends beyond identifying actionable mutations; it is instrumental in detecting hereditary cancer syndromes, thus aiding in early diagnosis and preventive strategies. Furthermore, NGS plays a crucial role in monitoring minimal residual disease, offering a sensitive method to detect cancer recurrence at an early stage. Its use in guiding immunotherapy by identifying biomarkers that predict response to treatment is also highlighted. Ethical issues related to genetic testing, such as concerns around patient consent and data privacy, are also important considerations that need to be addressed for the broader implementation of NGS. These include the complexities of data interpretation, the need for robust bioinformatics support, cost considerations, and ethical issues related to genetic testing. Addressing these challenges is essential for the widespread adoption of NGS. Looking forward, advancements such as single-cell sequencing and liquid biopsies promise to further enhance the precision of cancer diagnostics and treatment. This review emphasizes the transformative impact of NGS in oncology and advocates for its incorporation into routine clinical practice to promote molecularly driven cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f589e1477d90e5debe146a1fa230847b1a203d" target='_blank'>
              Next-generation sequencing in cancer diagnosis and treatment: clinical applications and future directions
              </a>
            </td>
          <td>
            Nima Ghoreyshi, Reza Heidari, A. Farhadi, Mohsen Chamanara, Nastaran Farahani, Mahmood Vahidi, Javad Behroozi
          </td>
          <td>2025-04-20</td>
          <td>Discover Oncology</td>
          <td>2</td>
          <td>4</td>
        </tr>

        <tr id="


 Small cell lung cancer (SCLC) represents a major medical challenge due to its aggressive progression and a dismal 5-year overall survival rate of only 7%. While SCLC initially responds well to chemotherapy, the rapid relapse and emergence of therapy-resistant tumors remain critical barriers to effective treatment. We hypothesized that therapy response is associated with genome evolution, which, in turn, influences transcriptional plasticity. This study aims to decipher the clonal evolution and transcriptomic adaptations in response to chemotherapy in SCLC.



 We established over 100 patient-derived xenotransplant models from clinical SCLC specimens, capturing diverse genetic features and clinical characteristics. A representative subset of 33 models was selected to replicate clinical scenarios in mice, modeling the effects of platinum-based chemotherapy and evaluating tumor responses under controlled conditions. Whole-exome sequencing and single-cell transcriptome profiling were performed to investigate genomic evolutionary trajectories and transcriptional plasticity in therapy-naïve and relapsed tumors.



 Xenotransplant models accurately reflect patient responses to chemotherapy. To our surprise, several models from relapsed patients retained sensitivity to treatment. Chemotherapy-resistant models maintain their tumor clones, whereas sensitive models show dynamic shifts from a common ancestral clone with emerging subclones shaped by mutations associated with platinum damage, APOBEC activity, and defective DNA damage repair. The genomic landscape, defined by subclonal mutations and copy number alterations, strongly impacts transcriptomic patterns. Specifically sensitive models post-chemotherapy reveal a strong shift of transcriptional programs reflecting adaptive plasticity as a consequence of genome evolution.



 Here we provide an in-depth functional in vivo characterization of clonal adaptions in SCLC under therapy. Chemotherapy drives the selection of subclones with favorable genomic features, which in turn impact the broader transcriptomic landscape. Our data provides critical insights into the molecular mechanisms of therapy resistance.



 Laura Constanze Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, Martin Peifer, Roman K. Thomas, Julie George. Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1258.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f680cbd3c45701162ce05be4c984e82cf3cb95" target='_blank'>
              Abstract 1258: Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, M. Peifer, Roman K. Thomas, Julie George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Bone metastases represent frequent and severe complications in various cancers, notably impacting prognosis and quality of life. This review article delves into the genetic and epigenetic mechanisms underpinning drug resistance in bone metastases, a key challenge in effective cancer treatment. The development of drug resistance in cancer can manifest as either intrinsic or acquired, with genetic heterogeneity playing a pivotal role. Intrinsic resistance is often due to pre-existing mutations, while acquired resistance evolves through genetic and epigenetic alterations during treatment. These alterations include mutations in driver genes like TP53 and RB1, epigenetic modifications such as DNA methylation and histone changes, and pathway alterations, notably involving RANK-RANKL signaling and the PI3K/AKT/mTOR cascade. Recent studies underline the significance of the tumor microenvironment in fostering drug resistance, with components such as cancer-associated fibroblasts and hypoxia playing crucial roles. The interactions between metastatic cancer cells and the bone microenvironment facilitate survival and the proliferation of drug-resistant clones. This review highlights the necessity of understanding these complex interactions to develop targeted therapies that can overcome resistance and improve treatment outcomes. Current therapeutic strategies and future directions are discussed, emphasizing the integration of genomic profiling and targeted interventions in managing bone metastases. The evolving landscape of genetic research, including the application of next-generation sequencing and CRISPR technology, offers promising avenues for novel and more effective therapeutic strategies. This comprehensive exploration aims to provide insights into the molecular intricacies of drug resistance in bone metastases, paving the way for improved clinical management and patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed3b250a10d84b32555ed5529bedbf646475e86e" target='_blank'>
              The genetic architecture of bone metastases: unveiling the role of epigenetic and genetic modifications in drug resistance
              </a>
            </td>
          <td>
            Ahmad Dawalibi, Mohamad Bakir, Khalid S. Mohammad
          </td>
          <td>2025-04-22</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Efficacy of targeted therapies in metastatic colorectal cancer (mCRC) is hindered by onset of resistance, therefore novel strategies are needed. A subpopulation of drug-tolerant persister cells (DTPs) survives to targeted therapies and initiates adaptive mutability response. DTPs experience an increased DNA damage and a shift to error-prone DNA damage repair, thus promoting mutability and resistance.



 The purpose of this project is to explore key players modulating adaptive mutability to prevent DTPs’ evolution to resistance, thus preventing tumor recurrence.



 CAS9-expressing CRC cells were infected with a custom CRISPR library targeting over 500 DNA damage response (DDR) genes, divided in 12 subpopulations, and grown either for 4 weeks in normal cell culture conditions (untreated) or treated with specific oncogenic inhibitors until the development of resistance in a Time-To-Progression (TTP) assay. CAS9-expressing clones infected with empty library were grown in parallel, in absence or presence of targeted therapies, as control. Next Generation Sequencing-based analysis of sgRNA (single guides RNA) was then performed on resistant subpopulations grown in the presence of targeted therapies.



 We initially isolated CAS9-expressing clones from a microsatellite stable (MSS) CRC cell line and selected clones with growth rate and drug-sensitivity comparable to that of parental population. The selected CAS9-expressing clone was then infected with the CRISPR DDR library or empty library and, after the selection with puromycin, the selected populations were divided in 12 subpopulations, to exclude the possibility that pre-existing resistant cells might outgrow and interfere with the effect of DDR genes knock-outs (KO) on the development of drug-resistance. The subpopulations were then grown in absence or presence of oncogenic inhibitors for several weeks until the onset of resistance. Notably, while the 12 subpopulations infected with the empty library simultaneously developed resistance, the development of resistance in the DDR library-infected cells was spread over multiple weeks, thus highlighting that KO of DDR genes might either accelerate or delay the onset of resistance. sgRNA of resistant population were then analyzed by next-generation sequencing. A negative selection analysis will be applied to identify sgRNA that are lost in the resistant populations, thus targeting DDR genes whose depletion is potentially synthetical lethal with targeted therapies in mCRC.



 A DDR CRISPR screening identified novel vulnerabilities of CRC DTPs able to improve efficacy of targeted therapies and prevent tumor relapse in mCRC.



 Martina Miotto, Simona Lamba, Gaia Grasso, Giorgio Corti, Julie Bonetto, Alberto Sogari, Alberto Bardelli, Mariangela Russo. CRISPR-CAS9 library screening identifies novel vulnerabilities in CRC drug-tolerant persister cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4190.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a3bde6f2ebc33e7239f55d9b0ddb6732733f31" target='_blank'>
              Abstract 4190: CRISPR-CAS9 library screening identifies novel vulnerabilities in CRC drug-tolerant persister cells
              </a>
            </td>
          <td>
            Martina Miotto, S. Lamba, G. Grasso, G. Corti, Julie Bonetto, A. Sogari, A. Bardelli, Mariangela Russo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6992085eb1aec63c000f8c350d79dda9f01e87" target='_blank'>
              A computational strategy to uncover fusion genes in prostate cancer cell lines
              </a>
            </td>
          <td>
            Rebecca A. Morgan, Gary Hardiman
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) has a poor prognosis, since most PDAC tumors develop resistance to the standard of care (SoC) treatments like chemotherapy, radiation, and targeted therapies. All tumors consist of multiple, genetically related subpopulations of cancer cells that evolve in parallel and display heterogeneity at genomic, epigenetic, or phenotypic levels. As cancer develops, some subpopulations of cancer cells may show faster growth, increased metastatic potential, and resistance toward SoC treatment. A key challenge in PDAC management is understanding chemoresistance driven by cancer subpopulation dynamics. This would broaden our understanding of tumor adaptation to treatments and guide targeting resistant subpopulations to improve therapeutic outcomes. In this study, we identified PDAC cell subpopulations resistant to chemotherapy using our DNA barcoding system B-GLI (barcode-guide lineage isolation). The B-GLI system leverages a highly complex DNA barcode library and CRISPR activation (CRISPRa) to trace and isolate sub-lineages within heterogeneous cell populations by their DNA barcodes. PDAC cell lines PANC-1 and Mia-PaCa-2 were barcoded with the B-GLI barcode library, which consist of two optimized guide RNA binding sites in each barcode. After DNA barcoding, we performed treatments with two SoC chemotherapeutics, gemcitabine + paclitaxel and FOLFIRINOX, to induce a selection pressure in the PDAC cell lines, and then identified the resistant subpopulations through differentially represented barcodes and sequencing. Furthermore, we designed barcode-specific single guide RNAs (sgRNAs) targeting the resistant subpopulations and activated the expression of the puromycin resistance gene by CRISPRa. This allowed us to enrich and isolate chemoresistant subpopulations by puromycin treatment. After having isolated the resistant PDAC subpopulations, we performed ATAC-seq and RNA-seq for molecular characterization of the resistant and parental cell populations, along with phenotypic drug screening with 384 compounds to identify novel treatment vulnerabilities. The compound testing confirmed that the resistant cells were less sensitive to the two SoC treatments, as well as identified selective sensitivity to specific compounds. ATAC-seq and RNA-seq data will allow us to decode whether and how chromatin structure and transcriptomic changes are associated with the treatment resistance, and the confirmation screen of the drug sensitivities will identify potential compounds for future pre-clinical and clinical testing in PDAC models and patients with resistant disease. In summary, this study will identify targeted treatment alternatives, accompanied by molecular biomarkers for chemotherapy resistant PDAC.


 Subhendu Roy Choudhury, Shixiong Wang, Yevhen Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, Tero Aittokallio. Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5544.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc3b5fa458a00b7ac6aa3d0d25d390376657ca" target='_blank'>
              Abstract 5544: Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells
              </a>
            </td>
          <td>
            S. Choudhury, Shixiong Wang, Y. Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, T. Aittokallio
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Reversible alterations at DNA sequence or epigenetic levels can result in phenotypes that are unstably inherited. The reversibility of these inheritable changes might be uniquely beneficial for adaption to possible fluctuations in environment. However, unstable changes are always ignored for the genetic instability in traditional studies, especially in the cause of drug resistance. In this study, we conduct a specific genetic screen in fission yeast using rapamycin (+caffeine) and obtain 173 resistant isolates. In contrast to the common strategy of isolating stable genetic mutants, we passage the cell culture with rapamycin resistance on drug free condition and test the resistance of offspring every five days, and obtain 14 strains that exhibit unstable resistance to rapamycin (the drug resistance is lost randomly among the cell progenies without drug selection pressure). Further studies show that the unstable genetic resistance of some strains is regulated by reversible DNA sequence alterationat the ssp1 gene locus. This study provides new insights and relevant scientific basis for the regulatory mechanism of unstable drug resistance in the process of rapamycin as a clinical anti-tumor drug, and a new possible target for solving the problem of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076e8ebfb600d70faa36a964146ab32ad3cd766a" target='_blank'>
              Screening and application of unstable genetically resistant strains in fission yeast.
              </a>
            </td>
          <td>
            Lu-Feng Dan, Yi-Wen Chu, Xin-Rong Wang, Xiang-Wei He
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Ovarian cancer, characterized by its heterogeneity and resistance to conventional therapies such as cisplatin (CDDP), presents a significant clinical challenge. To address these challenges, we combined genome-scale CRISPR/Cas9 knockout screening, single-cell-CRISPR Purterb RNA sequencing (scRNA-seq), and AI-driven virtual screening to identify novel chemoresistance mechanisms and evaluate potential therapeutic strategies. This integrated approach aims to reveal genetic determinants and actionable therapeutic targets to improve treatment outcomes.



 We performed genome-scale CRISPR/Cas9 knockout screening in ovarian cancer cell lines treated with CDDP, erastin, and a combination of elesclomol and copper (ES+Cu), identifying genes involved in drug resistance mechanisms. Differential gene candidates will be prioritized based on its strong association with resistance across the drug treatments. AI-based virtual screening will be employed to identify novel drugs targeting the candidates. Additionally, surviving cells from drug treatments will be subjected to scRNA-seq to reveal distinct cell cluster harboring resistance-related gene signatures. We also established ovarian cancer cell lines resistant to CDDP, erastin, and ES+Cu to validate the therapeutic targets and efficacy of the repurposed drugs in both resistant and wild-type cell lines.



 CRISPR screening identified several key genes associated with drug resistance, with YTHDC2 emerging as a central target. Pathway analysis highlighted its role in regulating survival and proliferation in the context of drug-induced stress. AI-driven virtual screening pinpointed Proguanil, Calcipotriol, and Sildenafil as high-affinity binders to YTHDC2, suggesting their potential as targeted therapeutics. scRNA-seq analysis of resistant cancer cells identified key gene cluster with GADD45A and ZFAS1 associated with cisplatin resistance. Gene Ontology and STRING pathway analyses revealed their involvement in DNA damage response and apoptosis regulation. Drug-response assays demonstrated that Proguanil, Calcipotriol, and Sildenafil effectively suppressed cell proliferation in the resistant cell lines. These findings support their potential to overcome the chemotherapy resistance.



 This study integrates CRISPR-based screening, scRNA-seq, and virtual screening to provide the first comprehensive framework for addressing drug resistance in ovarian cancer. The identification of YTHDC2 as a key resistance-related gene and the validation of Proguanil, Calcipotriol, and Sildenafil as potential therapeutics underscore the translational potential. These findings offer new avenues for precision oncology, with the goal of improving outcomes for ovarian cancer patients.



 Jinjiang Wang, Jiaxi Li, Jiao Liu, Kit-Ying Chan, Ho-Sze Lee, Zhewei Zhang, Joseph Kwong, Tat-San Lau, Chi-Chiu Wang. CRISPR/Cas9 knockout and AI-based drug screening for novel gene targets in ovarian cancer chemoresistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4408.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5713e3952828abd074f75de5f64c2e281aa291d9" target='_blank'>
              Abstract 4408: CRISPR/Cas9 knockout and AI-based drug screening for novel gene targets in ovarian cancer chemoresistance
              </a>
            </td>
          <td>
            Jinjiang Wang, Jiaxi Li, Jiao Liu, Kit-Ying Chan, Ho-Sze Lee, Zhewei Zhang, Joseph Kwong, T. Lau, C. Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Synthetic lethality (SL) has emerged as a highly promising therapeutic strategy, gaining significant attention in the field of cancer treatment in recent years. The principle behind SL is that the simultaneous loss or inhibition of two genes results in cell death, while the inhibition of only one gene does not. SL drugs exploit this mechanism by targeting the partner gene of an already inactivated or mutated gene in cancer cells, inducing their death. This approach allows for the precise targeting of genetic vulnerabilities specific to cancer cells, sparing normal tissues from damage. Additionally, SL therapies expand the range of druggable targets, including proteins that are difficult to inhibit with traditional small molecules. By affecting both enzymatic and non-enzymatic protein functions, SL therapies reduce the risk of developing drug resistance, providing an effective option for treating cancers that are resistant to conventional therapies.Kyinno offers an extensive collection of knockout cell lines targeting a range of genes associated with synthetic lethality, including SHLD2, P53, T53BP1, AQR, ATRX, SETD2, ATM, WRN, BRCA1, and BRCA2. These cell lines, developed using the CRISPR/Cas9 genome editing technique and validated through sequencing and western blot analysis, are crucial for investigating the complex mechanisms underlying synthetic lethality and for identifying novel therapeutic strategies in cancer treatment. For example, our knockout lines for BRCA1 and BRCA2 enable researchers to investigate vulnerabilities in homologous recombination repair, paving the way for the development of innovative PARP inhibitors and targeted therapies. Additionally, lines with deletions in ATM and WRN provide essential insights into DNA damage response pathways, which can enhance the effectiveness of DNA-damaging agents. The diverse range of gene knockouts, including those involved in cell cycle regulation, DNA repair, and apoptosis, allows for comprehensive studies into the synergistic effects of dual gene inhibition. By utilizing our extensive library of synthetic lethality cell lines, researchers can effectively validate hypotheses, screen novel drug candidates, and optimize combination therapies to improve outcomes for patients with resistant cancers. We highly recommend these resources for advancing research in synthetic lethality and targeted cancer therapies.


 Yue Huang, Yunpeng Zhai, Yan Feng, Xinyue Jia, Xiaohuan Zhang, Jinying Ning, Feng Hao. CRISPR-edited cell lines targeting key molecular pathways: Accelerating the development of synthetic lethal therapeutics beyond PARP inhibitors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4189.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea7a11f970c3515d6da1945980a0668b04d9e68" target='_blank'>
              Abstract 4189: CRISPR-edited cell lines targeting key molecular pathways: Accelerating the development of synthetic lethal therapeutics beyond PARP inhibitors
              </a>
            </td>
          <td>
            Yue Huang, Yunpeng Zhai, Yan Feng, Xinyue Jia, Xiaohuan Zhang, Jinying Ning, F‐J Hao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Adoptive T-cell therapy has emerged as a promising approach for the treatment of several malignancies, including melanoma. Despite promising outcomes, therapeutic resistance remains a significant challenge, with limited or transient responses observed in many patients. The molecular mechanisms driving resistance pathways have not been thoroughly studied in vivo. Systematic characterization of the genetic regulators of adoptive T-cell therapeutic resistance is thus critical for designing next-generation of synergistic combination therapies and treatment regimens to enhance efficacy and durability.



 We developed an in vivo CRISPR knockout screening system to identify genetic factors mediating resistance to adoptive T-cell therapy in a melanoma mouse model. To explore potential synergistic therapeutic combinations, we opted to target the easily druggable subset of the human genome, encompassing all known kinases. CRISPR-edited B16-GFP-OVA cells transduced with a mouse CRISPR KO library (Brie: comprising 2, 852 sgRNAs targeting 713 mouse kinase genes) were engrafted into C57BL/6J mice. The mice were divided into two groups: a PBS control group (n=6) and a T-cell treatment group (n=14). Tumors were harvested either at an early (Day X) or later time points (Day Y) for genomic DNA extraction and next-generation sequencing (NGS)-based guide RNA enrichment analysis. Deep sequencing data were analyzed using the MAGeCK pipeline. Functional validation of our top candidates was performed by generating single clones of single-gene CRISPR knockouts and conducting functional profiling using orthogonal assays, including flow cytometry, LDH cytotoxicity assay, and pharmacological inhibition studies to assess target engagement and efficacy.



 The CRISPR screen identified two key regulators of adoptive T-cell resistance, termed NU-ATCTr-101, NU-ATCTr-102. Single-gene knockouts of these targets significantly enhanced T-cell-mediated killing in vitro. Pharmacological inhibition of NU-ATCTr-101 demonstrated a dose-dependent increase in cytotoxicity and enhanced T-cell response. Mechanistic studies revealed alterations in antigen presentation, PD-L1 expression, and T-cell activation pathways. Bulk RNA sequencing and additional pathway analyses are ongoing to further elucidate precise molecular mechanisms underpinning these enhanced therapeutic outcomes.



 Our findings identify NU-ATCTr-101 and NU-ATCTr-102 as potential therapeutic targets to overcome resistance to adoptive T-cell therapy. Synergistic pharmacological interventions targeting these kinases may serve as promising adjuvants towards the development of effective immunotherapeutic strategies. Future studies will focus on validating these findings across additional human cancer model systems to evaluate and expedite their translational potential.



 Chenlin Zhao, Sharif Ahmed, Xiyue Hu, Ranjit S. Atwal, Zongjie Wang, Sophie Xiao, Shana O. Kelley. Systematic discovery of regulators of adoptive T-cell therapeutic resistance using in vivo CRISPR screening [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2991.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd0035313cd95f13c0ad0b1db10655d699eb842" target='_blank'>
              Abstract 2991: Systematic discovery of regulators of adoptive T-cell therapeutic resistance using in vivo CRISPR screening
              </a>
            </td>
          <td>
            Chenlin Zhao, Sharif U. Ahmed, Xiyue Hu, R. S. Atwal, Zongjie Wang, Sophie Xiao, S. Kelley
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="The tumorigenesis of small intestinal neuroendocrine tumors (siNETs) is not understood and comprehensive genomic and transcriptomic data sets are limited. Therefore, we performed whole genome and transcriptome analysis of 39 well differentiated siNET samples. Our genomic data revealed a lack of recurrent driver mutations and demonstrated that multifocal siNETs from individual patients can arise genetically independently. We detected germline mutations in Fanconi anemia DNA repair pathway (FANC) genes, involved in homologous recombination (HR) DNA repair, in 9% of patients and found mutational signatures of defective HR DNA repair in late-stage tumor evolution. Furthermore, transcriptomic analysis revealed low expression of the transcriptional repressor REST. Summarizing, we identify a novel common transcriptomic signature of siNETs and demonstrate that genomic alterations alone do not explain initial tumor formation, while impaired DNA repair likely contributes to tumor evolution and represents a potential pharmaceutical target in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44f7cb964c906b47270fb2a184506e3a4a4d940" target='_blank'>
              Small intestinal neuroendocrine tumors lack early genomic drivers, acquire DNA repair defects and harbor hallmarks of low REST expression
              </a>
            </td>
          <td>
            Felix Bolduan, Niklas Müller-Bötticher, Olivia Debnath, I. Eichhorn, Yvonne Giesecke, Alexandra Wetzel, S. Sahay, Tomasz Zemojtel, Marten Jaeger, Ute Ungethuem, Christoph Roderburg, C. A. Kunze, Annika Lehmann, David Horst, Frank Tacke, Roland Eils, Bertram Wiedenmann, Michael Sigal, N. Ishaque
          </td>
          <td>2025-05-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Development of next generation sequencing technologies allows to identify a large number of genetic landscape types in various cancers including breast cancer. Frequent genetic abnormalities identified using whole genome sequencing are point mutations (missense, nonsense mutations), deletions, insertions, which usually lead to activation of protooncogenes and inactivation of tumor suppressor genes. Genome sequencing of malignant tumors allowed, on one hand, to identify driver mutations in carcinogenic genes in different organs, and on the other – to use mutated genes for targeted therapy. Study of biological functions of these genes from the point of view of their contribution to carcinogenesis allows to better understand its mechanism. In this review, signaling cascades of breast cancer with identified mutated genes – targets for therapy – are analyzed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d647ad1e76680b2e0c017bae0688ab2ee0a334" target='_blank'>
              Signaling cascades are targets for breast cancer therapy in the light of genome – wide sequencing data
              </a>
            </td>
          <td>
            L. F. Gulyaeva, M. Filipenko, N. Kushlinskii
          </td>
          <td>2025-04-13</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Mutational processes such as chromosomal instability and replication stress are important molecular features of head and neck squamous cell carcinoma (HNSCC) and non-small cell lung cancer (NSCLC), contributing to poor patient survival. Their changing influences during cancer development creates challenges in identifying response biomarkers. Patient-derived organoid 3D cancer models (PDOs) have been shown to match their primary source material and to reflect tumour biology more faithfully than cell lines. In our companion abstract, we detail how we have built the Digital Biological Twin lab to study PDOs for drug testing and biomarker discovery. In this work, we used PDOs to investigate resistance to PARP inhibitors (PARPi), which are mechanistically linked to DNA repair and replication. 10 PDOs (6 HNSCC, 4 NSCLC) generated from untreated tumor resections were bulk RNA-sequenced and treated with Niraparib. IC50, cell viability measured by flow cytometry and protein expression levels were used to classify 3 PDOs as sensitive to treatment. Expression analysis, stratifying by PARPi sensitivity, resulted in 287 differentially expressed genes (DEGs). Enrichment analyses found involvement of the endothelial-to-mesenchymal transition (EMT) and small molecule transport pathways. Knowing that EMT is linked to multi-drug resistance, we treated the PDOs with Cisplatin and found a significant correlation to PARPi IC50 values. By examining the biology of the DEGs, we categorized them using MSigDB into five gene sets, each with 1 to 14 genes. We functionally validated the gene sets by assessing PARPi susceptibility in 45 cell lines pre and post a genome-wide CRISPR knockout, finding four significantly enriched in PARPi gene dependencies. For two replication stress-derived gene sets, we demonstrated significant concordance between PARPi sensitivity status of PDOs and cell lines from DepMap, and with progression free interval of HNSCC and NSCLC patients from TCGA. Leveraging Cox analysis on these patients, one replication stress gene set stratified the patients by showing a significant decrease in relapse probability. In summary, our results indicate that PARPi resistance is associated with the deregulation of genes related to small molecule transport, multi-drug resistance and EMT. The consistent findings around replication stress across disease models strongly suggests a conserved and central role in resistance mechanisms. We also identify a replication stress gene set that significantly correlates with resistance in cancer models and relapse in patients. Further work is needed to translate our findings into a predictive model for drugs linked to replication stress. Here we showed that PDOs can be successfully used for furthering our understanding of resistance mechanisms while improving molecular stratification of patients with direct clinical implications.


 Ruben M. Drews, Finnian Firth, Paul R. Barber, Anna Pasto, Tony T. Ng. Exploring resistance mechanisms in cancer using patient-derived organoids reveals replication stress and EMT as potential biomarkers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4605.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf51075c6ced2d719b16f81522dd0e475d5f9454" target='_blank'>
              Abstract 4605: Exploring resistance mechanisms in cancer using patient-derived organoids reveals replication stress and EMT as potential biomarkers
              </a>
            </td>
          <td>
            Ruben M. Drews, Finnian Firth, Paul R. Barber, Anna Pasto, Tony T. Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Phenotypic plasticity is emerging as a key hallmark of cancer progression, yet the molecular mechanisms of cancer cell reprogramming remain poorly understood. Using single cell RNA sequencing of synchronously surgically resected matched trios of normal colon, primary and metastatic tumors from colorectal cancer (CRC) patients, we recently demonstrated that CRC undergoes highly conserved sequential phenotypic transitions during tumor progression: (1) differentiated epithelial cells dedifferentiate into an LGR5+ intestinal stem cell (ISC) state during primary tumor initiation (2) disseminating cancer cells adopt an injury-induced fetal-like state and (3) at the metastatic site again dedifferentiate into LGR5+ ISC states, or alternatively undergo lineage plasticity into non-canonical neuroendocrine and squamous states, associated with aggressive behavior and poor clinical outcomes. Cancer cells thus dedifferentiate into LGR5+ ISC states at two steps during tumor progression. Seeking regulators of dedifferentiation, computational analysis identified the RNA binding protein ZFP36L2 as a stress-associated ISC state marker in normal intestinal crypts and in primary and metastatic tumors. In mouse models, we demonstrate that ZFP36L2 is dispensable for intestinal homeostasis, but is required for epithelial regeneration after DSS induced colitis or ISC ablation. Using human tissue samples, organoids and orthotopic mouse models, we show that loss of ZFP36L2 impairs progenitor regeneration and metastatic seeding—both of which require dedifferentiation into an intestinal stem cell state—while promoting proliferation and lineage plasticity into non-canonical states. Multiplex immunofluorescence staining of clinical samples demonstrates that loss of function mutations in ZFP36L2, found in 5-10% of human CRC, are associated with non-canonical differentiation in metastasis. Mechanistically, using HyperTRIBE and mRNA decay analysis, we show that ZFP36L2 binds to stress-associated mRNAs containing AU-rich 3’UTRs, in turn inducing the formation of dynamic phase-separated biomolecular condensates. Unexpectedly, IP-MS identifies the intestinal differentiation transcription factor CDX2, and the NCOR nuclear corepressor complex as lead ZFP36L2 interactors. Nuclear/cytoplasmic fractionation and super resolution live cell microscopy demonstrate that while ZFP36L2 is largely cytoplasmic, it can shuttle into the nucleus, where it induces transcriptional repression of the intestinal differentiation program through CDX2-NCOR complex formation and chromatin compaction. Together, our work identifies ZFP36L2 as a key orchestrator of stress-induced phenotypic plasticity, driving cellular dedifferentiation and epigenetic reprogramming essential for re-establishing the ISC state during metastatic seeding and wound healing.


 Qingwen Jiang, Manisha Raghavan, Britney Forsyth, Aileen Rodriguez, Cyrus Tam, Sasha Balkaran, Saskia Hartner, Morgan Lallo, Ahmed Mahmoud, Andrew Moorman, Jura Pintar, Dana Pe’er, Richard Koche, Quaid Morris, Joe Chan, Karuna Ganesh. ZFP36L2 orchestrates stress adaptive plasticity during injury repair and metastasis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2648.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6afa5982640193cfc266d2068be1bacf5082da2a" target='_blank'>
              Abstract 2648: ZFP36L2 orchestrates stress adaptive plasticity during injury repair and metastasis
              </a>
            </td>
          <td>
            Qingwen Jiang, Manisha Raghavan, Britney Forsyth, Aileen Rodriguez, Cyrus Tam, Sasha Balkaran, S. Hartner, Morgan Lallo, Ahmed Mahmoud, Andrew Moorman, Jura Pintar, D. Pe’er, R. Koche, Quaid Morris, Joseph Chan, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Abstract APOBEC mutagenesis is one of the most common endogenous sources of mutations in human cancer and is a major source of genetic intratumor heterogeneity. High levels of APOBEC mutagenesis are associated with poor prognosis and aggressive disease across diverse cancers, but the mechanistic and functional impacts of APOBEC mutagenesis on tumor evolution and therapy resistance remain relatively unexplored. To address this, we investigated the contribution of APOBEC mutagenesis to acquired therapy resistance in a model of EGFR-mutant non–small cell lung cancer. We find that inhibition of EGFR in lung cancer cells leads to a rapid and pronounced induction of APOBEC3 expression and activity. Functionally, APOBEC expression promotes the survival of drug-tolerant persister cells (DTP) following EGFR inhibition. Constitutive expression of APOBEC3B alters the evolutionary trajectory of acquired resistance to the EGFR inhibitor gefitinib, making it more likely that resistance arises through de novo acquisition of the T790M gatekeeper mutation and squamous transdifferentiation during the DTP state. APOBEC3B expression is associated with increased expression of the squamous cell transcription factor ΔNp63 and squamous cell transdifferentiation in gefitinib-resistant cells. Knockout of p63 in gefitinib-resistant cells reduces the expression of the ΔNp63 target genes IL-1α/β and sensitizes these cells to the third-generation EGFR inhibitor osimertinib. These results suggest that APOBEC activity promotes acquired resistance by facilitating evolution and transdifferentiation in DTPs and that approaches to target ΔNp63 in gefitinib-resistant lung cancers may have therapeutic benefit. Significance: APOBEC mutagenesis is a common source of genetic heterogeneity in cancer, and APOBEC mutational signatures are enriched in metastatic and drug-resistant tumors. However, the mechanisms through which APOBEC3 enzymes promote tumor evolution remain unknown. In this study, we show that APOBEC3 activity contributes to the development of therapy-resistant cancer cells by promoting evolution of DTP cells. These findings offer insights into the role of APOBEC mutagenesis in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7a11c8b2dc4731f11b9133f359e7ac89041b4a" target='_blank'>
              APOBEC3 Activity Promotes the Survival and Evolution of Drug-Tolerant Persister Cells during EGFR Inhibitor Resistance in Lung Cancer
              </a>
            </td>
          <td>
            N. M. G. Garcia, Jessica N Becerra, Sharan Srinivasan, Brock J McKinney, Ashley V DiMarco, Feinan Wu, Matthew Fitzgibbon, James V. Alvarez
          </td>
          <td>2025-05-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Primary mouse renal proximal tubule epithelial cells (moRPTECs) were immortalized by lentivirus transduction to create hTERT or SV40LgT (LgT) cell lines. Prior work showed a more pronounced injury and repair response in LgT versus hTERT cells after chemical challenge. We hypothesized that unique genomic changes occurred after immortalization, altering critical genes and pathways. RNA-seq profiling and whole-genome sequencing (WGS) of parent, hTERT, and LgT cells showed that 92.5% of the annotated transcripts were shared, suggesting a conserved proximal tubule expression pattern. However, the cell lines exhibited unique transcriptomic and genomic profiles different from the parent cells. Three transcript classes were quite relevant for chemical challenge response—Cyps, ion channels, and metabolic transporters—each important for renal function. A pathway analysis of the hTERT cells suggested alterations in intermediary and energy metabolism. LgT cells exhibited pathway activation in cell cycle and DNA repair that was consistent with replication stress. Genomic karyotyping by combining WGS and RNA-seq data showed increased gene copy numbers in chromosome 5 for LgT cells, while hTERT cells displayed gene copy losses in chromosomes 4 and 9. These data suggest that the exaggerated transcriptional responses of LgT cells versus hTERT cells result from differences in gene copy numbers, replication stress, and the unique selection processes underlying LgT or hTERT immortalization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5fc2423810ee9671b81c235e2952419eb5bbc3d" target='_blank'>
              hTERT and SV40LgT Renal Cell Lines Adjust Their Transcriptional Responses After Copy Number Changes from the Parent Proximal Tubule Cells
              </a>
            </td>
          <td>
            B. A. Merrick, Ashley M Brooks, Julie F. Foley, N. Martin, Rick D. Fannin, Wesley Gladwell, Kevin E. Gerrish
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Genetic instability is a hallmark of cancer, often arising from mutations in DNA damage repair and response (DDR) proteins. It is our goal to map the repair dependencies associated with different DNA-damaging agents (DDA) and DDR inhibitors (DDRi). To this end, we developed a DDR-targeted, CRISPR knockout screening approach to assess relationships among 355 DNA repair genes and 30 DDA and DDRi in LN229 glioma cells.
 Systems-level analysis of over 11, 000 interactions revealed that DDA cluster into distinct classes. In contrast, DDRi including ATR, Pol-θ, DNA-PK, WRN, Wee1, USP1, ATM and PKMYT1 inhibitors exhibited a variety of unique dependency profiles. Pathway analysis revealed clear associations between the knockout of Fanconi anemia pathway genes and sensitivity to bifunctional alkylating agents (e.g. chlorambucil), nucleotide excision repair and monofunctional alkylating agents (e.g. illudin S), homologous recombination and PARP inhibitors (e.g. talazoparib), and non-homologous end-joining and radiomimetics (e.g. bleomycin).
 Furthermore, this platform identified differences in repair dependencies of compounds with subtle variations in chemical structure, highlighted by the imidazotetrazines temozolomide (TMZ) and KL50. In contrast to the methyl adducts deposited by TMZ, DNA modifications deposited by KL50 slowly evolve into DNA interstrand crosslinks (ICLs), circumventing mismatch repair (MMR)-driven resistance observed with TMZ. Indeed, screening revealed that resistance to TMZ, but not KL50, is driven by loss of MMR genes. Furthermore, we observed that BRIP1, a helicase known to play a key role in repair of ICLs, emerged as a unique sensitizing hit for KL50, but not TMZ, providing biological support for the formation of ICLs by KL50.
 Finally, we sought to identify the most clinically actionable chemo-genomic interactions from our screens. Bi-allelic inactivating mutations (BI) of DNA repair genes are often necessary for manifestation of a DDR deficiency phenotype. Thus, we subjected our gene list to a custom bioinformatic pipeline to quantify rates of BI in samples from The Cancer Genome Atlas (TCGA). Excluding genes listed as Level 1 biomarkers on OncoKB, we found that 4.73% of TCGA samples harbored putative BI of at least one of the 138 genes that emerged as a hit (FDR < 0.01) in our screens.
 Overall, our screens validated established chemo-genomic interactions while also identifying genetic dependencies associated with DDA of previously unknown mechanisms of action. Furthermore, we elucidated subtle differences in repair dependencies between structurally similar compounds. Finally, we identified novel genetic biomarkers with frequent BI in human tumor samples that sensitized to specific therapeutic agents upon loss, nominating chemo-genomic interactions with potential clinical relevance.


 Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, Ranjini Sundaram, Vijay Menon, Joseph Edmonds, Karlie Lucas, Nadeem Riaz, Susan E. Gueble, Ranjit S. Bindra. Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2902.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b90e0ffa740ed3554dce2f43e94e83c85289e48" target='_blank'>
              Abstract 2902: Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening
              </a>
            </td>
          <td>
            Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, R. Sundaram, Vijay Menon, Joseph Edmonds, Karlie N Lucas, N. Riaz, Susan E. Gueble, R. Bindra
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The development of chemoresistance in many cancer patients poses a significant clinical challenge, while the underlying causes for heterogeneous drug responses remain largely elusive. Increasing evidence suggests that tumors exhibit extensive heterogeneity and harbor cancer subpopulations in less differentiated states that are linked to treatment resistance. Therefore, dissecting immature cancer cell states pre-existing in tumors is essential for identifying biomarkers associated with chemoresistance and establishing targeted therapeutic strategies, particularly in breast cancer, where specific biomarkers are largely lacking.



 We utilized CytoTRACE, a robust computational framework for predicting the cellular differentiation status from single-cell RNA sequencing data, to unveil the immature cancer cell state and its associated markers in single-cell atlases of human breast cancer. The association between the identified immature cell state and patient survival was determined by inferring the proportions of epithelial cancer cells expressing the marker gene in bulk gene expression datasets. The MMTV-PyMT mouse breast cancer model was used to experimentally investigate the role of the immature cancer cell state in conferring chemoresistance. Single-cell profiling was conducted on tumor samples before and after treatment to examine the inherent characteristics of immature cancer cells and their response to drug pressure. The results obtained from mouse tumors were subsequently confirmed using human breast cancer models.



 We revealed that BCL11B, a zinc finger transcription factor, defines a distinct immature cancer cell state in human breast cancer that displays basal cell features and demonstrates a heterogeneous expression pattern across breast tumors. Breast cancer patients with higher levels of BCL11B+ cancer cells face a significantly increased risk of relapse following chemotherapy. This pre-existing tumor subpopulation exhibits intrinsic fitness advantages and demonstrates a remarkable ability to evolve into a drug-resistant state through various BCL11B-dependent pre-existing and transcriptional adaptation mechanisms. Notably, TNF-α was pinpointed as a natural inhibitor of BCL11B, and treatment with TNF-α significantly increases chemotherapy efficacy by inducing the exit of the drug-resistant cell state. In conclusion, our findings identify BCL11B as a valuable biomarker for predicting chemotherapy resistance and suggest TNF-α combination treatment for breast cancer patients at high risk of developing chemoresistance.



 Zhen Qi,1 Gunsagar S. Gulati,2 Angera H. Kuo,1 Shaheen S. Sikandar,3 William Hai Dang Ho,1 Dalong Qian,1 Frederick M. Dirbas,1 Aaron M. Newman,1 Shang Cai,4 Michael F. Clarke1. BCL11B predetermines an immature drug-resistant cell state in breast cancer that is vulnerable to TNF-α treatment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1256.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b6836817685e87c48dea5b327743f821ab98f7f" target='_blank'>
              Abstract 1256: BCL11B predetermines an immature drug-resistant cell state in breast cancer that is vulnerable to TNF-α treatment
              </a>
            </td>
          <td>
            Zhen Qi, Gunsagar S. Gulati, Angera H. Kuo, Shaheen S. Sikandar, William Hai Dang Ho, D. Qian, F. Dirbas, Aaron M. Newman, Shang Cai, Michael F. Clarke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Chromatin bivalency plays an important role in cell lineage specification during development. Cancer therapy resistance involves an extensive reprogramming of gene expression resulting in tumor lineage plasticity (LP). However, the underlying mechanisms are poorly understood. Rev-erb-alpha, a member of the nuclear receptor transcription factor family, plays a key role in regulation of circadian rhythm and metabolism primarily through repression of its target genes. We performed sequential ChIP-seq epigenome profiling of tumor tissues from the clinic and PDX models of prostate cancer and integrated analysis of tumor epigenome and transcriptome. We also treated the PDX models with small-molecule inhibitors of the receptor. We found that most of the tumor LP genes, including those in programs of neurodevelopment, neural signaling, stemness and EMT, are controlled by bivalent promoters with concurrent marking of H3K27me3 and H3K4me3. Our further studies revealed that the bivalent chromatin state resolves during the development of cancer therapy resistance. Interestingly, Rev-erb-alpha plays a critical role in promoting the bivalency resolution and activation of the LP programs. Its pharmacological targeting effectively reverses the bivalency resolution and diminishes the LP in anti-AR therapy-resistant tumors. Therefore, our study establishes that resolution of chromatin bivalency is a major epigenetic mechanism of tumor LP and demonstrates that therapeutic targeting of Rev-erb-alpha represents a novel treatment of advanced prostate cancer.
 This work was supported by NIH and UC Davis comprehensive cancer center and U.S. Department of Defense.


 Yatian Yang, Xiong zhang, hongye zou, Eva Corey, Ronald M. Evans, Amina Zoubeidi, Hongwu Chen. A circadian regulator drives tumor lineage plasticity through chromatin bivalency resolution [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2726.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728e53382096a103c9861e4f20d349955371bb" target='_blank'>
              Abstract 2726: A circadian regulator drives tumor lineage plasticity through chromatin bivalency resolution
              </a>
            </td>
          <td>
            Yatian Yang, Xiong zhang, hongye zou, E. Corey, Ronald M. Evans, Amina Zoubeidi, Hongwu Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third leading cause of cancer-related deaths in the US, with rectal cancer (RC) accounting for one-third of cases. Only 20% of RC patients have a complete response to chemoradiation therapy (CRT) and may be able to avoid surgery, therefore, therapeutic resistance remains a major barrier to improving outcomes. The genetic and cellular heterogeneity of RC, particularly in treatment-resistant sub-clones, suggests that this diversity could be a harbinger of adverse outcomes. A detailed characterization of treatment responses at the subclonal level is critical to advancing targeted therapeutic strategies.



 Patient-derived xenograft (PDX) models and single-cell RNA sequencing were employed to investigate the cellular and molecular landscape of RC tumors. Two PDX models were established from the primary tumors of pre-treatment RC patients. Mice bearing tumors were subjected to a standard CRT regimen, consisting of 2 Gy daily radiation and 100mg/kg capecitabine daily for two weeks, followed by a week-long rest period. Tumor tissues were collected at baseline and at various post-treatment intervals for sequencing.



 Our scRNA-seq analysis of one PDX model identified five distinct tumor clusters with varying responses to treatment. Two clusters demonstrated initial sensitivity to CRT, followed by a resurgence, indicating potential therapy resistance. Pathway analysis revealed cluster-specific activation of critical oncogenic pathways, including HIF-1, p53, and FoxO signaling pathways. These pathways are implicated in tumor progression and therapeutic resistance, and the associated marker genes present potential targets for intervention. Lineage tracing of tumor cells uncovered two differentiation trajectories, with one showing increased resistance. Cell-cell communication analysis highlighted extensive interactions among tumor sub-populations, particularly mediated by CLDN3-CLDN3 adhesion, which may contribute to therapeutic resistance. Additionally, our investigation into copy number variations identified alterations in the chr6p21.31-p21.1 region, which harbors several oncogenes with copy number changes directly implicated in tumor progression. In the second PDX model, scRNA-seq identified two therapy-resistant tumor clusters among five detected clusters. The resistant tumor clusters exhibited enrichment of HIF-1, Hippo, and p53 signaling pathways, similar to the first PDX model but also unique activation of pathways like TGF-β and Hippo signaling.



 Our study highlights the dynamic, heterogeneity of RC tumors to CRT, identifying both shared and patient-specific resistance mechanisms in two PDX models. These findings underscore the complexity of RC tumor biology and offer a foundation for developing precision therapies targeting treatment-resistant subclones.



 Yuwei song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong. Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1248.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9d728476d907142dd31d1c87ffc2d2c3f782be" target='_blank'>
              Abstract 1248: Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment
              </a>
            </td>
          <td>
            Yuwei Song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Despite advancements in medical treatment, hepatocellular carcinoma (HCC) remains a leading cause of cancer-related mortality globally. Combination therapies involving multiple targeted agents are increasingly recognized as effective strategies to improve treatment outcomes and overcome the limitations of single-agent therapies. While single-target approaches have shown specificity and potency, they face challenges such as drug resistance, limited pharmacokinetics, and patient adherence. Given the complex nature of HCC, targeting multiple molecular pathways through combination strategies holds promise for more robust therapeutic outcomes.
 Epigenetic therapies offer a distinct advantage in treating HCC by targeting reversible modifications to reprogram aberrant gene expression and address tumor heterogeneity. These therapies can enhance the efficacy of existing treatments, overcome resistance mechanisms, selectively target cancer cells, and modulate the immune microenvironment, making them a promising avenue for improving HCC outcomes.
 EZH2 (Enhancer of Zeste Homolog 2) is a histone methyltransferase that catalyzes the trimethylation of histone H3 at lysine 27 (H3K27me3), leading to the silencing of tumor suppressor genes and the promotion of stem cell-like features. LSD1 (Lysine-specific demethylase 1) is a histone demethylase that modulates gene expression to promote tumor progression, cancer stemness, drug resistance, and epithelial-mesenchymal transition (EMT). These epigenetic enzymes form a corepressor complex to inhibit tumor suppressor gene expression. Both EZH2 and LSD1 are overexpressed in HCC and are strongly associated with poor clinical outcomes. Patients with high expression levels of both EZH2 and LSD1 exhibit the poorest survival, suggesting that dual inhibition of these enzymes may offer a more effective therapeutic strategy than monotherapy in HCC.
 Dual inhibition of EZH2 and LSD1 using GSK126 (an EZH2 inhibitor) and SP2509 (an LSD1 inhibitor) more effectively suppresses liver cancer cell proliferation compared to single-agent treatments. This combination reduces cell proliferation markers, including PCNA and Cyclin D1, and is associated with enhanced programmed cell death. Additionally, we observed decreased cell migration, invasion potential, cancer stem cell characteristics, and drug resistance. Gene Set Enrichment Analysis (GSEA) revealed that high expression of either EZH2 or LSD1 is linked to crucial processes such as cell proliferation, migration, invasion, cancer stemness, and EMT, reinforcing the potential of dual epigenetic targeting. Although EZH2 and LSD1 inhibitors are currently undergoing clinical trials for various cancers, their combination has not yet been explored in HCC. Therefore, evaluating this dual epigenetic therapy for HCC treatment represents a promising direction for future research with immediate translational potential.


 Kyounghyun Kim, Ponmari Guruvaiya. Dual epigenetic targeting of hepatocellular carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 440.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca6f30d3b9e64a4de714f44e69384d7201b4a8d" target='_blank'>
              Abstract 440: Dual epigenetic targeting of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kyounghyun Kim, Ponmari Guruvaiya
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Numerical chromosomal instability (n-CIN) is a defining characteristic of most human cancers with serious implications of large-scale alterations of the genome - such as chromosome arm level copy number changes. These kinds of somatic alterations are frequently targeting oncogenes and tumor suppressors, enriched in cancer driver events in order to shape the transcriptome in favor of tumor progression and proliferation. While oncogenes are mostly associated with copy number gains and tumor suppressors are more frequent in chromosome arm losses, neighboring genes are also affected in both cases. This collateral damage may be able to create synthetic lethal relationships and context-specific dependencies. Cancer cells might be able to avoid partial loss and gain-related lethal effects by triggering different dose compensation mechanisms, occurring at the genomic, epigenetic and post-translational levels, which leads to deviations across copy number and expression profiles.
 To propose a metric for dose compensation potential, we analyzed copy number and gene expression profiles of the Cancer Cell Line Encyclopedia in order to characterize cancer cell lines with the ability to compensate dependencies arising from chromosomal losses and gains. Our metric consists of the sum of gene expression residuals from linear model fits, describing how much a cell line tends to under or overexpress genes compared to what we expect from copy number counts. By comparing other publicly available datasets with our results, such as perturbation transcriptomics data from the LINCS-L1000 project and the DepMap genome-wide gene essentiality measurements, we aim to identify CIN-related therapeutic vulnerabilities associated with chromosome arm losses and gains. Besides finding potential target candidates, we show that our method is capable of finding well-known pathway-specific compensation mechanisms. As an example, we show a significant correlation between the activity of the Nonsense mediated decay (NMD) pathway and our proposed compensation potential score (CPS) of cancer cell lines.
 We propose our method to be used to detect context-specific dependencies associated with gene dosage compensation, and also able to provide pathway-level evidence - both novel and previously known from the literature.


 Péter Szikora, Ivan Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel Veres. Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3718.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Increased ribosome biogenesis supports the need for high translation capacity and is a hallmark of cancer. Targeting this process presents a promising strategy for cancer therapy. We have developed first-in-class inhibitors of RNA polymerase I (Pol I) transcription that block the first and rate-limiting step of ribosome biogenesis, rRNA synthesis. We find that Pol I inhibition by BMH-21 leads to a significant decrease in active ribosomes. We found a broad, but heterogenous efficacy in 18 cancer cell lineages. To identify genes that drive potential resistance to Pol I inhibition, we performed genome-wide CRISPR-Cas9 screens in human colorectal carcinoma cells. These positive selection screens identified high-confidence hits leading to drug resistance and included all key positive regulators of the mTORC1 complex. These findings are striking and counterintuitive given that mTOR is a major driver of ribosome biogenesis and cellular translational programs. The findings were validated using chemical inhibition of mTOR by Torin-1, a catalytic mTOR inhibitor and genetic knockout of an activator of mTOR signaling. Consistently, compromised mTOR activity led to resistance to BMH-21. However, loss of mTOR activity did not abrogate Pol I transcription inhibition by BMH-21. To assess the impact on protein translation, we used polysome profiling and found that BMH-21 treatment caused a severe defect in ribosome biogenesis. Surprisingly, mTOR inactivation partially rescued the translation repressed by Pol I inhibition, suggesting that this rescue could be pivotal for cell survival. To profile translational events, we performed Ribo-seq and RNA-seq in the BMH-21 sensitive and resistant cells. These multi-omics results revealed that mTOR inactivation decreased polysome-based translation on 5’TOP mRNAs encoding ribosomal proteins, suggesting that mTOR inactivation reduces ribosome occupation on ribosomal mRNAs. We propose that mTOR inactivation switches off polysome translation to rearrange active ribosomes for translation of survival-essential mRNAs. We further quantified newly synthesized proteins by AHA-labeling combined with TMT-mass spectrometry. These data show that distinct proteins continue to be translated when both Pol I transcription and mTOR activity are blocked. Next, we will examine preferentially translated proteins in resistant cells, and explore combination therapies to circumvent the drug resistance. In summary, we uncover a new non-genetic model of cancer drug resistance termed as translational fitness that enables cells to survive severe translational repression. These findings reveal an unexpected complication by mTOR inhibitory strategies and have implications for exploring effective drug combinations in cancer therapy.


 Wenjun Fan, Lijing Yang, Hester Liu, Steffie Pitts, Niladri Sinha, Rajeshkumar N.V., Hariharan Easwaran, Rachel Green, Marikki Laiho. Inactivation of mTOR is a strategy for tumor resistance to ribosome biogenesis inhibition [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5506.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/094980891c477076ac9ee63d92403d7dd49a9f93" target='_blank'>
              Abstract 5506: Inactivation of mTOR is a strategy for tumor resistance to ribosome biogenesis inhibition
              </a>
            </td>
          <td>
            Wenjun Fan, Lijing Yang, Hester Liu, Steffie Pitts, Niladri Sinha, N.V. Rajeshkumar, H. Easwaran, Rachel Green, M. Laiho
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background/Objectives: TIGD1 (Trigger Transposable Element Derived 1) is a recently identified oncogene with largely unexplored biological functions. Emerging evidence suggests its involvement in multiple cellular processes across cancer types. This study aimed to perform a comprehensive pan-cancer analysis of TIGD1 to evaluate its expression patterns, diagnostic utility, prognostic value, and association with immunotherapy response and drug resistance. Methods: Transcriptomic and clinical data from TCGA and GTEx were analyzed using various bioinformatic tools. Expression profiling, survival analysis, immune correlation studies, gene set enrichment, single-cell sequencing, and drug sensitivity assessments were performed. Results: TIGD1 was found to be significantly upregulated in various tumor types, with notably high expression in colon adenocarcinoma. Elevated TIGD1 expression was associated with poor prognosis in several cancers. TIGD1 levels correlated with key features of the tumor immune microenvironment, including immune checkpoint gene expression, TMB, and MSI, suggesting a role in modulating anti-tumor immunity. GSEA and single-cell analyses implicated TIGD1 in oncogenic signaling pathways. Furthermore, high TIGD1 expression was linked to resistance to several therapeutic agents, including Zoledronate, Dasatinib, and BLU-667. Conclusions: TIGD1 may serve as a promising diagnostic and prognostic biomarker, particularly in colon, gastric, liver, and lung cancers. Its strong associations with immune modulation and therapy resistance highlight its potential as a novel target for precision oncology and immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1163400219b2ecb1878714587256ba612b794cd1" target='_blank'>
              Beyond Transposons: TIGD1 as a Pan-Cancer Biomarker and Immune Modulator
              </a>
            </td>
          <td>
            Merve Gülşen Bal Albayrak, Tuğcan Korak, Gurler Akpinar, Murat Kasap
          </td>
          <td>2025-05-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Molecularly targeted therapies have significantly improved clinical outcomes by directly addressing oncogenic driver mutations in genes such as EGFR. However, despite their initial success, long-term patient survival remains limited due to the emergence of drug tolerance and resistance. This study investigates the molecular mechanisms underlying drug tolerance in response to targeted therapies in non-small cell lung cancer (NSCLC). Using single-cell RNA sequencing of oncogene-driven human NSCLCs at the residual disease (RD) state during active targeted therapy, we observed an enrichment of an alveolar type I/II (AT1/2) mixed lineage program in surviving cancer cells, resembling an alveolar cell-like state. This transition is characterized by increased expression of WNT3A/4 ligands, FZD2 receptors, β-catenin signaling, and AT1/2 signatures in RD cells. Consistent with these clinical observations, preclinical models of drug-tolerant persister (DTP) cells exhibited elevated AT1/2-like transcriptional signatures and target gene expression associated with β-catenin and YAP signaling. Further analysis revealed the formation of a stable nuclear protein complex between β-catenin and YAP in DTP cells, which appears to play a critical role in sustaining drug tolerance. Disrupting the expression and stability of the β-catenin/YAP complex, either through knockdown of these genes or by using Tankyrase inhibitors (e.g. XAV-939 or IWR1) to regulate the β-catenin destruction complex, re-sensitized DTP cells to the EGFR inhibitor Osimertinib. In xenograft mouse models, the combination of Osimertinib with a Tankyrase inhibitor significantly improved tumor growth inhibition compared to Osimertinib alone. This therapeutic strategy also led to a notable reduction in AT1/2 gene signatures, a decrease in nuclear β-catenin and YAP protein expression, reduced stability of these proteins, and suppression of their interaction, further supporting its potential in overcoming drug tolerance. Additionally, we identified specific domains within both β-catenin and YAP proteins that are essential for their interaction. This interdependence underscores the critical roles of β-catenin and YAP in maintaining DTP cell survival and driving alveolar cell-like lineage plasticity. Our findings shed light on the interplay between β-catenin/YAP signaling and alveolar cell-like lineage signatures in residual cancer states. By targeting this axis, we propose a novel therapeutic approach to overcome drug tolerance and improve clinical outcomes for patients with NSCLC undergoing targeted therapy.


 Yu-Ting Chou, Wei Wu, D. Lucas Kerr, Victor Olivas, Macey Slota, Trever G. Bivona. Deciphering the interplay of β-catenin/YAP signaling and alveolar lineage plasticity during targeted therapy in non-small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5557.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc2b803c7ac7b99f1e6df1e4aac1c92682211a5" target='_blank'>
              Abstract 5557: Deciphering the interplay of β-catenin/YAP signaling and alveolar lineage plasticity during targeted therapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Yu-Ting Chou, Wei Wu, D. Kerr, V. Olivas, Macey Slota, T. Bivona
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background Integration of immune checkpoint inhibitors (ICIs) with non-immune therapies relies on identifying combinatorial biomarkers, which are essential for patient stratification and personalized treatment. Methods We analyzed genomic and transcriptomic data from pretreatment tumor samples of 342 melanoma patients treated with ICIs to identify mutations and expression signatures associated with ICI response and survival. External validation and mechanistic exploratory analyses were conducted in two additional datasets to assess generalizability. Results Responders were more likely to have received anti-PD-1 therapy rather than anti-CTLA-4 and exhibited a higher tumor mutation burden (both P < 0.001). Mutations in the dynein axonemal heavy chain (DNAH) family genes, specifically DNAH2 (P = 0.03), DNAH6 (P < 0.001), and DNAH9 (P < 0.01), were enriched in responders. The combined mutational status of DNAH 2/6/9 effectively stratified patients by progression-free survival (hazard ratio [HR]: 0.69; 95% confidence interval [CI] 0.51–0.92; P = 0.013) and overall survival (HR: 0.58; 95% CI 0.43–0.78; P < 0.001), with consistent association observed in the validation cohort (HR: 0.28; 95% CI 0.12–0.61; P < 0.001). DNAH-altered melanomas exhibited upregulation of chemokine signaling, cytokine-cytokine receptor interaction, and cell cycle-related pathways, along with elevated expression of immune-related signatures in interferon signaling, cytolytic activity, T cell function, and immune checkpoints. Using LASSO logistic regression, we identified a 26-gene composite signature predictive of clinical response, achieving an area under the curve (AUC) of 0.880 (95% CI 0.825–0.936) in the training dataset and 0.725 (95% CI 0.595–0.856) in the testing dataset. High-risk patients, stratified by the expression levels of a 13-gene signature, demonstrated significantly shorter overall survival in both datasets (HR: 3.35; P < 0.001; HR: 2.93; P = 0.002). Conclusions This analysis identified potential molecular determinants of response and survival to ICI treatment. Insights from melanoma biomarker research hold significant promise for translation into other malignancies, guiding individualized anti-tumor immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06467-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92479e3bbfea3311336630e41672d9439b0a6863" target='_blank'>
              Genetic and transcriptional insights into immune checkpoint blockade response and survival: lessons from melanoma and beyond
              </a>
            </td>
          <td>
            Jiaxin Wen, Yanfeng Wang, Song Wang, Yuxin Liang, Xiaozhen Hu, Qiuxiang Ou, H. Bao, Kuo Zhao, Youyu Wang
          </td>
          <td>2025-04-23</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Bladder cancer often harbors driver mutations in genes regulating epigenetic processes, yet their influence on therapeutic perturbations is understudied. One key example is KDM6A, a gene frequently mutated in bladder cancer, which encodes a histone demethylase that confers transcription permissive chromatin state. Retrospective analysis of patient cohorts demonstrated that KDM6A mutations correlate with improved response to immune checkpoint therapy (ICT) and decreased sensitivity to cisplatin-based chemotherapy. Mechanistic studies reveal that CRISPR-Cas9 deletion of Kdm6a reduced expression of critical genes including Msh2, Msh6, and Exo1 involved in DNA Mismatch repair (MMR) and double-stranded base repair (DSBR) in tumor cells with improved response to anti-PD-1 therapy. Further, KDM6A deficiency downregulated genes involved in cisplatin resistance and attenuated sensitivity to cisplatin-based chemotherapy in vitro and in vivo. Additionally, loss of KDM6A was associated with metabolic shift from glycolysis to oxidative phosphorylation. CyTOF analyses show decreased abundance regulatory T cells (Tregs) in Kdm6a-knockout tumors. Importantly, reduced lactate in the Kdm6a-knockout tumor microenvironment impaired histone lactylation (H3K9la/H3K18la) in Tregs with concurrent attenuation of expression of key immunosuppressive genes including Foxp3, Tgfß1 and Pdcd1. Collectively, this study provided insights into the role of KDM6A-mediated epigenetic regulation of DNA repair and metabolic reprogramming which could potentially govern response to chemotherapy and ICT aiding patient stratification and development of personalized treatment algorithms.


 Pratishtha Singh, Deblina Raychaudhuri, Bidisha Chakraborty, Aminah J. Tannir, Swadhin Meher, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, Patrick G. Pilie, Sangeeta Goswami. Genomic and metabolic effects of KDM6A loss regulates differential responses to immune checkpoint therapy and chemotherapy in bladder cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4834.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2254442f541b7850b7eb9c6ffd437b3ddb785eca" target='_blank'>
              Abstract 4834: Genomic and metabolic effects of KDM6A loss regulates differential responses to immune checkpoint therapy and chemotherapy in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, D. Raychaudhuri, Bidisha Chakraborty, Aminah J. Tannir, Swadhin Meher, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, P. Pilié, Sangeeta Goswami
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="
 The formation of cancer and responsiveness to therapy is governed by a constantly evolving balance between somatic variation and dynamic epigenetic regulation. This evolution has been recognized as a significant obstacle confronting the development of durable cancer therapies. Thus, targeting epigenetic reprogramming in human cancers is a key focus for the design of new drugs to potentially deliver robust and durable anti-tumor activity. Lysine acetyltransferases (KATs) are a promising class of epigenetic targets that operate within interchangeable multi-subunit complexes to regulate cell cycle, pluripotency and lineage determination. Within this class, KAT6A and KAT7 have been recognized as mechanistically intertwined modulators of stem cell identity and lineage commitment through partially overlapping control of chromatin organization specified by histone H3 acetylation of lysine 23 (H3K23) and lysine 14 (H3K14). We noted multiple lines of orthogonal genetic evidence for a “paralog” relationship between these enzymes to support cancer cell autonomous survival in lineage-enriched subsets of the CCLE associated with predictive molecular features. These observations prompted evaluation of concomitant inhibition of KAT6A and KAT7 as a therapeutic approach in biomarker-defined patient populations. Here, we report an equipotent KAT6/KAT7 dual inhibitor with high selectivity versus the structurally similar KAT family members KAT5 and KAT8. We observed synergistic impact on cell viability and pharmacodynamic (PD) modulation of H3K23/H3K14 acetylation in biomarker positive breast, lung, and esophageal cancers with dual KAT6/7 inhibition versus selective inhibition of KAT6 or KAT7. In vitro transcriptomic evaluation of dual KAT6/7 drug effects in ER+ metastatic breast cancer models recapitulated convergent paralog behavior observed using genetic suppression with RNAi. Quantitative transcriptomic and epigenomic profiling of cancer cell responses to dual KAT6/7 inhibition confirmed downregulation of gene expression programs supporting lineage specification, stemness, and WNT signaling. In vivo efficacy studies in 8p11-amplified breast and lung xenografts confirmed enhanced tumor growth inhibition and PD modulation responses for dual KAT6/7 versus selective KAT6 inhibition. Notably, dual KAT6/7 inhibition also promoted improved durable and stable anti-tumor responses in endocrine therapy resistant and ESR1-mutant breast cancer patient derived xenografts that could not be achieved with selective KAT6 inhibition alone. Taken together, these observations support evaluation of dual KAT6/7 inhibition in biomarker-selected patient populations.


 Manav Gupta, Pranav Gupta, Scott R. Tyler, Nour Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, Ivan G. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, Jay Prakash Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca Choy, Richard Zang, Xin Linghu, Yuchen Bai, Peter Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, Josh Taygerly, Jacob Burch-Konda, Paul A. Barsanti, Michael A. White, Brian T. Jones. Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 442.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01f5cfa8e398963dd5157942872b501bf3ce27d8" target='_blank'>
              Abstract 442: Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance
              </a>
            </td>
          <td>
            Manav Gupta, Pranav Gupta, Scott R. Tyler, N. Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, I. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel L. Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, J. Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca M. Choy, Richard Zang, Linghu Xin, Yuchen Bai, P. Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, J. Taygerly, Jacob Burch-Konda, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Breast cancer, characterized by considerable heterogeneity, showcases genetic and epigenetic variability among individual tumors and their constituent cells. Notably, this intratumoral heterogeneity (ITH) can drive clonal evolution and therapy resistance, particularly in Human Epidermal Growth Factor Receptor 2 (HER2)-positive cancers. Despite the efficacy of HER2-targeted therapies, many patients develop resistance, resulting in poor prognosis. This study aims to elucidate the dynamic interplay between genetic alterations and epigenetic modifications during anti-HER2 therapy using advanced single-cell resolution techniques. Biopsy samples were collected at three time points from patients undergoing neoadjuvant HER2-targeted therapy as part of the TRIO-US B07 clinical trial. Utilizing methods such as low-pass whole-genome sequencing (WGS, n=47), single-nuclei WGS (snWGS, n=21), and matched single-nuclei RNA/ATAC-seq (n=22), we explored copy-number variations (CNVs), subclonal tumor dynamics, and alterations in the tumor microenvironment (TME). We also analyzed additional deep WGS (n= 205) to reconstruct the structure of HER2 amplicons. Our analysis revealed distinct CNV dynamics in HER2-positive breast cancers during treatment. We found that while broad-level CNVs remained stable, focal amplifications exhibited temporal variability, suggesting the presence of extrachromosomal DNA (ecDNA). Specifically, the structure of HER2 amplificons emerged as a pivotal determinant of aggressiveness, with higher numbers of co-amplified oncogenes correlating with non-pathological complete response (pCR) and metastasis. Single-cell resolution analyses unraveled significant subclonal diversity in a subset of patients exhibiting complex tumor architectures. Integration of transcriptomic and epigenomic profiling highlighted that pCR patients exhibited higher lymphocyte infiltration and dynamic immune shifts post-treatment. Conversely, non-pCR tumors displayed increased regulatory T cells. Furthermore, fibroblast state transitions characterized by altered extracellular matrix (ECM) gene expression suggested a potential role in modulating immune responses. Our findings provide novel insights into the genetic and epigenetic factors driving therapy resistance in HER2-positive breast cancer. We emphasize the importance of longitudinal, multimodal profiling to refine patient stratification and inform personalized treatment strategies. Overall, this study offers new avenues for understanding tumor evolution in response to therapy and highlights the need for comprehensive assessments of ITH and microenvironmental dynamics for improved clinical outcomes.


 Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis. Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6355.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Hepatocellular carcinoma (HCC), the most common type of primary liver cancer, is the sixth most prevalent tumor worldwide and the fourth leading cause of cancer-related deaths. Lenvatinib, a multi-targeted tyrosine kinase inhibitor (TKI), is a first-line therapy for HCC patients who are ineligible for immunotherapy. However, like other targeted therapies, its efficacy is often compromised by the development of resistance. Understanding the molecular alterations driving this resistance is critical to improving therapeutic outcomes in HCC. In this study, lenvatinib-resistant human HCC cell lines were established by exposing parental sensitive cells to increasing concentrations of lenvatinib. Resistant Hep3B and Huh7 cell lines were successfully developed, exhibiting higher lenvatinib-specific IC50 values compared to their sensitive counterparts. To investigate the mechanisms underlying drug resistance, whole-genome sequencing (WGS, average depth: 30X) was performed, identifying resistance-specific genomic alterations, including copy number variations (CNVs), mutations, and structural variations (SVs). Emerging evidence suggests that chromosomal instability (CIN) plays a critical role in resistance evolution through structural rearrangements and focal amplifications. To reconstruct these amplified regions, we utilized AmpliconArchitect, followed by classification into chromosomal amplification (ChrAmp) and extrachromosomal DNA (ecDNA) using AmpliconClassifier. The number of focal amplifications increased from 4 to 7 in resistant Hep3B cells and from 14 to 18 in resistant Huh7 cells. Notably, newly identified oncogenes were detected within these amplicons, including ZIC2 in Hep3B and EGFR in Huh7. Despite the emergence of new amplicon profiles in resistant cells, most retained their original classifications as ecDNAs or ChrAmps, suggesting that clones carrying these amplicons exhibit an intrinsic resistance phenotype. Ongoing studies involving single-clone isolation and further genomic characterization aim to refine and expand these findings. In conclusion, we successfully established lenvatinib-resistant HCC cell lines and characterized their resistance-specific genomic alterations. Our results highlight the role of CIN in driving resistance under the selective pressure of targeted therapies, offering potential insights for improving therapeutic strategies in HCC.


 JungHyun Kim, Ji Young Kim, Gamin Han, BoYoon Kim, Hoon Kim, Yeup Yoon, Wonseok Kang. Detection of genomic alterations in lenvatinib-resistant liver cancer cell lines [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7503.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aecf1e2b76e187b7e7357a8c7f6d61cd00b2bcd" target='_blank'>
              Abstract 7503: Detection of genomic alterations in lenvatinib-resistant liver cancer cell lines
              </a>
            </td>
          <td>
            Junghyun Kim, Ji Young Kim, Gamin Han, BoYoon Kim, Hoon Kim, Yeup Yoon, Wonseok Kang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the “basal” signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the basal state is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the basal state. Taken together, our data indicates that PDAC harbors a striking dependency on the basal state and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.


 Anupriya Singhal, Hannah Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Zeynep C. Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional dissection of the highly plastic basal cell state in pancreatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3880.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9d10ba962b31f50d67da52456440cdb7f8eb0c" target='_blank'>
              Abstract 3880: Functional dissection of the highly plastic basal cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G. Christensen, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="
 Dynamic adaptations in cancer cell-state due to therapy is a growing concern. Despite multimodal clinical therapy, tumor cells surpass treatment resulting in frequent relapses in rapidly decreasing timeline. We have shown that, in accordance with therapy pressure associated tumor microenvironment, progressive disease (PD) cells exhibit reversible lineage transformation and unparalleled plasticity. However, our understanding of the genetic determinant(s) and/ or mechanism(s) that coordinate cancer cell plasticity is a far reach. Herein, we investigated the crucial cancer cell phenotypic switching in response to therapy pressure. Using a panel of 14 well characterized human neuroblastoma (NB) cells derived during diagnosis (Dx) and PD, we investigated the association of retinal degeneration protein 3 (RD3) status (ELISA) to their migratory and invasive capabilities (transwell assay), clonal expansion (BrDu), EMT (E-CAD, N-CAD, Vimentin, SLUG), metastatic state (Ron-β, MMP-9, MMP-2), stemness maintenance (CD133, CD117, CD114, Oct3/4, Sox2, NANOG, CD54, NOTCH1) and cellular dedifferentiation (β-3 tubulin, NF-L, GFAP, GAP43) (immunoblotting). Further, to investigate the function of RD3 in coordinating cancer cell plasticity, we utilized six select NB cells, three derived during Dx and three derived during PD, representing both primary and metastatic tumors. These patient derived cells were reverse engineered for stable expression of RD3 in RD3- PD cell lines; stable silencing of RD3 in RD3+ Dx cell lines and were subjected to end-point readouts as above. RD3 loss in PD correlated with the increased cellular migration, invasion, metastatic potential, EMT and stemness maintenance. Directed simulation experiments with stable restoration and silencing systems clearly identified the function of RD3-loss in cancer cell plasticity by regulating stemness (ALDH, CD133), EMT (Vimentin, E-CAD, N-CAD), metastasis (Ron-β, MMP-9), pluripotency maintenance (Oct3/4, Sox2, NANOG), dedifferentiation (β-3 tubulin, NF-H, GFAP, GAP43) and clonal expansion (BrDu). Our findings for the first time identified the defined function and relevance of RD3 in regulating cancer cell EMT, stemness and pluripotency maintenance, metastatic state and prompting differentiation. Crucially, our findings recognized therapy pressure directed RD3 loss dictates cancer cell lineage transformation to cancer stem cells. These outcomes together pave way to develop novel and effective molecular-targeted maintenance therapeutic strategy for patients presented with therapy defying progressive disease.


 This work was partially or in full, funded by Department of Defense CA-210339; OCAST-HR19-04; NIH-P20GM103639 to Dr. Aravindan; and by the NIH-NCI-Cancer center support Grant P30CA225520, NIGMS P30GM154635, Oklahoma Tobacco Settlement Endowment Trust [R23-03] to the OU Health Stephenson Cancer Center.
 Citation Format: Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, Natarajan Aravindan. Driver and signaling cascade in therapy pressure driven cancer cell plasticity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1246.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37bbf9a9279b9ea882191a6ab180a2f1195c66ee" target='_blank'>
              Abstract 1246: Driver and signaling cascade in therapy pressure driven cancer cell plasticity
              </a>
            </td>
          <td>
            Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, N. Aravindan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 SCLC is the deadliest type of lung cancer, which is mostly diagnosed at late stages and accounts for 15% of pulmonary tumors. Chemotherapy is initially effective, but SCLC patients relapse quickly and develop resistant tumors. Large-scale studies uncovered key molecular patterns in SCLC, which include universal loss of TP53 and RB1, as well as transcriptional patterns based on the expression of 4 lineage transcription factors (TF). We aimed to dissect how genomic cues impact transcriptional phenotypes, and the dynamics of tumor and immune cell interactions, to thus decipher molecular mechanisms of phenotypic divergence in SCLC.
 We performed genome and single cell transcriptome profiling on 56 SCLC patient tumors, including primary tumors, metastases and paired relapse tumors, including a patient sample after treatment with DLL3-targeted immunotherapy. Single cell RNA-seq was performed with the 10x Genomics platform to study cancer cells and the tumor micro-environment (TME), and tumor cell intrinsic heterogeneity. Data were processed with an in-house pipeline.
 Single cell expression profiles were determined for > 200000 cells from 56 tumors. Transcriptome profiling at single cell levels revealed co-expression of at least two of the lineage TF ASCL1, NEUROD1 or POU2F3 for each SCLC tumor. Transcriptional profiles pointed to a marked intra- and interpatient, as well as to inter-metastatic heterogeneity. Cell-cell-interaction analysis across all samples uncovered recurrent patterns of intra-tumoral receptor-ligand interactions. The integrative analysis of genome and single cell transcriptome data pointed to somatic alterations and copy number alterations shaping transcriptional heterogeneity. Specifically, tumors with gene amplifications of MYC family members share similar expression profiles. We performed multi-regional and longitudinal studies of matched patient cases to project genomic patterns of clonal evolution to transcriptional profiles at single cell level, thus analyzing the effect of genome diversity on the transcriptional landscape. Additionally, tumor intrinsic profiles of heterogeneity were mapped to single cell data of patient tumors to elucidate the effects on tumor-immune cell interactions.
 Altogether we provide a first comprehensive framework for the study of underlying genetic alterations that shape the transcriptional landscape and phenotypic plasticity in SCLC. Our data points to a concerted regulation of a multitude of factors including lineage TF, transcriptional programs and interaction networks in SCLC.


 Marcel Schmiel, Laura Kaiser, Julie George, Roman K. Thomas. Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC) [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3891.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 DNA Mismatch Repair (MMR) genes’ (including MSH3) alterations through silencing and/or mutations plays a role in colorectal cancer (CRC) carcinogenesis. MSH3 interacts with MSH2 to form MutSβ, which recognizes and directs repair of insertion/deletion loops during in DNA replication; deficiencies lead to elevated mutation rates. Msh3 alterations are associated with instability at tetranucleotide repeats (termed Elevated Microsatellite Alterations at Selected Tetranucleotide repeats, EMAST), a phenotype elevated frequency among African American (AA). CRCs and associated with poor patient prognosis. Aim: To assess functional impact of 3 out of 4 Exomes (E) E20, E-21, E22, E23 novel deleterious AA variants within the ATPase domain of MSH3.



 We used CRISPR-Cas9 knock-in and successfully introduced 3 of these mutations (confirmed by Sanger sequencing) in Msh3 ATPase domain (c.G2926A, c.C3028T, c.G3241A) in SW620 colon cancer cell line (carry Msh3 gene wild-type). In Silico and wet-lab MSH3 functional analysis was performed. We used MTT for cell proliferation, tetranucleotide repeats instability assays, immunofluorescence for MSH3 localization, Western blot for protein expression, co-immunoprecipitation for MSH3 interaction with MSH2 MMR protein, and thin layer chromatography ATPase assays with purified MutSβ complexes.



 In silico analysis showed deleterious effect of the 4 novel msh3 mutants. However, biological phenotypic assays revealed no observable changes in cell morphology or proliferation between wild-type and MSH3-variant knocked-in cells, with no EMAST after 7 to 10 passages. Subcellular localization of variant MSH3 was unaffected compared to wild type MSH3, and interaction between mutant MSH3 and MSH2 proteins was not impacted, However, the ATPase activity of two of the variant MutSβ complexes was affected. MSH3 c.G2759T (E20) exhibited overall reduced ATPase activity compared to wildtype and was not affected by the presence of DNA. MSH3 c.C3028T (E22) exhibited elevated ATPase activity compared to wildtype in the presence of specific (loop) or non-specific DNA (homoduplex).



 Despite computational deleterious predictions, novel MSH3 variants identified from AA CRCs appeared to show no biological or genetic evidence of pathogenicity, but did reveal altered in vitro biochemical properties. Altered MutSβ ATPase activity, both decreased and increased, has been shown to disrupt proper MMR activity. We propose that disruptions in the ATP hydrolytic cycle contribute to MMR deficiency, resulting in elevated mutation rates with contribution to colorectal cancer pathogenesis. This effect may be enhanced in distinct genetic backgrounds, and directly affect cancer cells’ proliferation and cell division.



 Mudasir Rashid, Hassan Brim, Katherine Casazza, Jennifer A. Surtees, Shaolei Teng, Adebiyi Sobitan, Adeyinka O. Laiyemo, Zaki Sherif, Koi Minoru, John Carethers, Hassan Ashktorab. Do MSH3 mutations really affect colorectal cells neoplastic transformation: an evolution of three African Americans novel mutations within the MSH3 ATPase domains. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1477.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648d311b893d674bef8fa9845b19283b6062d3e3" target='_blank'>
              Abstract 1477: Do MSH3 mutations really affect colorectal cells neoplastic transformation: an evolution of three African Americans novel mutations within the MSH3 ATPase domains
              </a>
            </td>
          <td>
            Mudasir Rashid, H. Brim, Katherine M Casazza, Jennifer A. Surtees, Shaolei Teng, Adebiyi Sobitan, A. Laiyemo, Z. Sherif, Koi Minoru, John Carethers, H. Ashktorab
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Genetically identical bacterial cells often exhibit heterogeneous responses to antibiotics − some survive, others die. Here, we show that this heterogeneity propagates across generations to give rise to phenotypic resistance. Using real-time single-cell tracking, we exposed Escherichia coli to the β-lactam cefsulodin at its clinical breakpoint concentration and analyzed cell fate within genealogical trees statistically. Cell survival was strongly correlated among family members, driving the selective enrichment of robust lineages within an otherwise susceptible population. Our genealogical population model identified heritable phenotypic resistance as a key factor underlying this enrichment, which was validated experimentally. Comparing enrichment dynamics between the wild-type and a tolC knock-out strain, deficient in multidrug efflux, uncovered nuanced changes that increased the intergenerational memory of phenotypic resistance. Our findings provide evidence for heritable phenotypic resistance and demonstrate how its propagation through cell-to-cell heterogeneity enables the survival of minority cells within isogenic populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6715bbe58fe26a64981336caf37053a9a8306b82" target='_blank'>
              Lineage-dependent variations in single-cell antibiotic susceptibility reveal the selective inheritance of phenotypic resistance in bacteria
              </a>
            </td>
          <td>
            Wesley Stine, Tatsuya Akiyama, David S. Weiss, Minsu Kim
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Clear Cell Sarcoma (CCSA) is a rare soft tissue sarcoma characterized by specific gene fusions EWSR1::ATF1 or EWSR1::CREB1, leading to the expression of an oncogenic fusion protein. Metastasized CCSA is resistant to conventional chemotherapy, targeted therapy or immunotherapy. This study aims to uncover new therapeutic strategies for CCSA by utilizing a whole-genome CRISPR/Cas9 knock-out screen combined with functional drug screening.



 We performed a genome-wide knock-out screen in the CCSA cell line model KAS using the pooled Brunello sgRNA library (Addgene # 73178, ∼4 guides/gene, 19'114 genes). In parallel, three CCSA patient samples and multiple CCSA cell lines (SU-CCS-1, DTC-1, KAS) were treated with a library of 84 solid tumor pan-cancer drugs, and a custom drug library tailored to the hits of the CRISPR screen (70 compounds). Genes identified as critical vulnerabilities in CCSA were validated through single-gene knockouts, and promising therapeutic targets were further mechanistically interrogated and evaluated in a CCSA xenograft model.



 Results revealed 1'552 significantly depleted genes (log fold change <-1 and FDR < 0.01) after 21 days, with enriched pathways in proteasome activity, mismatch repair, DNA replication, and, mTORC1 signaling, revealing critical vulnerabilities. Notable highly ranked depleted genes include the transcription factor and CCSA marker SOX10, as well as MDM2, MCL1, CHEK1, HSPA5, PSMA5 and mTOR. Through integration of genetic knock-out and pharmacological inhibition data identified in our dual screening, we were able to identify several overlapping vulnerabilities. Initial validation of the therapeutic potential of PI3K/mTOR inhibition in CCSA cell models in vitro revealed an induction of apoptosis upon treatment. We then corroborated those findings in a CCSA xenograft mouse model in vivo. Treatment with the two combined PI3K/mTOR inhibitors omipalisib and copanlisib both resulted in significantly improved survival (P = 0.027 and P = 0.036, respectively) of tumor bearing mice, and reduced tumor growth, each compared to vehicle treatment.



 These findings offer new insights into actionable vulnerabilities in CCSA, with PI3K/mTOR inhibitors showing potential for therapeutic development. Future studies will investigate synergistic combinations to further optimize treatment efficacy and validate further vulnerabilities identified by our dual screening approach.



 Laura Leuenberger, Larissa Isenegger, Yanjiang Chen, Miles Burri, Sarah Kollar, Lara Planas-Paz, Peter Karl Bode, Agnieszka Wozniak, Sebastian Bauer, Patrick Schöffski, Chantal Pauli, Christian Britschgi. Identifying novel vulnerabilities in clear cell sarcoma through functional and genomic screening [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5664.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1582331698256f81dcbd70eb9ab64e969d89900e" target='_blank'>
              Abstract 5664: Identifying novel vulnerabilities in clear cell sarcoma through functional and genomic screening
              </a>
            </td>
          <td>
            Laura Leuenberger, Larissa Isenegger, Yanjiang Chen, Miles Burri, Sarah Kollar, Lara Planas-Paz, Peter Karl Bode, A. Wozniak, S. Bauer, Patrick Schöffski, Chantal Pauli, Christian Britschgi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The deregulation of BCL2 family proteins is a key contributor to the development of T-cell acute lymphoblastic leukemia (T-ALL). Pharmacological inhibition of these proteins, particularly through the BCL2-specific inhibitor venetoclax, has emerged as a promising therapeutic approach. However, the clinical efficacy of venetoclax is often compromised due to primary and acquired resistance mechanisms that remain incompletely understood. We employed a deep tabular learning algorithm to develop a drug sensitivity prediction model using T-ALL patient samples. Our findings revealed distinct expression profiles between drug-sensitive and drug-resistant samples. Specifically, in the context of venetoclax sensitivity and resistance phenotypes, we observed associations with several key proteins regulating the cell cycle, cytokine signaling, and mitochondrial apoptosis. Our data suggest a key link between lipid metabolism and beta-catenin signaling, which may cooperate with transcription factors regulating the transcriptional activation of key apoptosis regulatory genes. Together, our results underscore the complexity of therapy resistance and highlight the therapeutic potential of targeting multiple pathways for growth inhibition. Future research should focus on clinical trials exploring this combinatorial approach to overcome resistance and improve patient outcomes in T-ALL.


 hmad Nasimian, Kinjal Shah, Julhash U. Kazi. Understanding therapy resistance in T-cell acute lymphoblastic leukemia [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5550.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369cd649004283a085be1edef8035acdd6d9bbcf" target='_blank'>
              Abstract 5550: Understanding therapy resistance in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Ahmad Nasimian, Kinjal Shah, Julhash U. Kazi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Inhibition of the mutant BRAFV600E protein in melanoma via targeted therapy causes many cells to die, but resistant populations emerge that are refractory to further treatments. Strategies to target resistance have proven difficult to find, which could be explained by the potential heterogeneity of the population. It is assumed that genetic mutations alone cause resistance, but targeting such mutations has met with limited success in melanoma. Work from our lab and others has now established that non-genetic variability also leads to resistance in vitro and in vivo: we have shown that rare clonal cells, marked by differences in gene expression, survive and form resistant colonies. We recently demonstrated in vitro and in vivo that individual resistant colonies have distinct morphological and transcriptional differences, even with a clonal parent population. These results provide a potential explanation for the difficulty in identifying effective second-line therapies: individual resistant types may have different drug susceptibilities, so an effective second-line drug would have to kill all resistant types at once.
 These findings raise the possibility that there may be therapeutic vulnerabilities to exploit that are invisible in the conventional view of homogeneous resistance. For instance, drug A could kill half of the resistant population while drug B kills the other half. These drugs alone would not be viable second-line treatments, but together they could be highly effective, greatly expanding the pool of drugs that could combat resistance. However, screens to evaluate all combinations of potential drugs would be prohibitively large. This search space could be simplified by determining the drug sensitivities of individual resistant types and using those drugs in combination to eliminate a larger proportion of the total resistant population. However, such per-type classification has not yet been performed in melanoma.
 We generated clonal cell lines from individual resistant colonies, screening four and the drug-naive line using a library of ∼2200 drugs and small molecules with known pathways and targets in cancers. These genetically identical resistant lines indeed had distinct drug sensitivity profiles, and therapy-resistant lines as a whole had novel sensitivities and resistances compared to the drug-naive line. Treating drug-naive cells with targeted therapy in combination with drugs that kill resistant lines can reduce or eliminate resistance, even when the candidate second-line drugs alone do not kill drug-naive cells. Using a panel of 24 drugs identified as of interest in our initial screens, we screened 13 resistant lines, including the original four and four resistant to a different targeted therapy. We found potential links between the transcriptional and toxicity profiles of resistant colonies, and are continuing to test pairs of drugs that kill distinct fractions of the resistant population.


 Gianna T. Busch, Ryan H. Boe, Jingxin Li, Pavithran T. Ravindran, Arjun Raj. Heterogeneity in sensitivity to second-line inhibitors of therapy-resistant melanoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6385.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11b087a0a9fc0566d0ad6c1ba6f886b3c8f91ce" target='_blank'>
              Abstract 6385: Heterogeneity in sensitivity to second-line inhibitors of therapy-resistant melanoma
              </a>
            </td>
          <td>
            Gianna T. Busch, Ryan H. Boe, Jingxin Li, Pavithran T. Ravindran, Arjun Raj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Breast cancer (BC), the most common cancer globally and a leading cause of death in women, often utilizes radiation therapy (RT) as part of treatment, with over 85% of patients receiving RT after breast-conserving surgery. Still, over 15% of women experience local recurrence despite RT, underscoring the need to better understand the molecular mechanisms driving RT response and resistance. We hypothesized that transcriptomic changes that occur after ionizing radiation in intrinsically radiosensitive and resistant BC models would offer mechanistic insight into mediators of this differential response. To test this, we conducted in vitro and in vivo RNA-seq experiments to characterize transcriptomic changes following RT across a panel of BC cell lines of varying levels of intrinsic radiation sensitivity. Furthermore, we conducted a genome-wide CRISPRi screen to identify genes which are critical to radiation survival and drive radioresistance.



 Eight BC cell lines representing all intrinsic subtypes were treated with 4 Gy RT and RNA was collected 24 hours after treatment for RNA-seq. For 2 cell lines, we also conducted in vivo mouse xenograft experiments in which mice received fractionated daily RT (6 fractions of 2 Gy each). Differential gene expression analysis with DeSeq2 was performed, followed by pathway analysis with Advaita Bioinformatics’ iPathwayGuide. For the CRISPRi experiments, we used the Dolcetto library in two TNBC cell lines: BT-549 and HCC-1937. Cells were transfected with the library and then treated with 3 fractions of 3 Gy RT. DNA was isolated for sequencing after 7 days and analyzed with MaGeCK to determine sgRNA abundance.



 An average of 935 differentially expressed genes were noted in the in vitro models (S.D. 906); 142 and 324 were found in the in vivo models. Among the most highly enriched pathways from the transcriptomic data in both resistant and sensitive cell lines after RT, we found processes including cell cycle, cellular senescence, and estrogen signaling. Pathways uniquely differentially expressed in the in vivo samples for both cell lines include IL-17 signaling and transcriptional misregulation in cancer. From the CRISPRi screen, we identified genes involved in similar pathways, as well as canonical regulators of DNA damage response.



 Ionizing radiation induces transcriptomic expression changes that differ between intrinsically radiosensitive and resistant BC models. These data, coupled with results from the CRISPRi radiosensitivity screen, identify pathways that offer potential insight into the mechanisms underlying intrinsic radioresistance and suggest biologic vulnerabilities that may be targeted to more effectively treat women at a high risk of local BC recurrence. Ongoing studies are exploring the potential of therapeutic targeting of these pathways when combined with RT in breast cancer.



 Breanna McBean Priyanka S. Rana Benjamin Hauk Anna R. Michmerhuizen Samuel Lichtman-Mikol Reine Abou Zeidane Vesna Markovic Kari Wilder-Romans Mingfang Tao Alan P. Boyle Corey Speers. Deciphering mechanisms of intrinsic radioresistance in breast cancer: Insights from transcriptomics and CRISPRi screens [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1469.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2d4cdd26b00d23c9f2af55b08e343fcfbd17a4" target='_blank'>
              Abstract 1469: Deciphering mechanisms of intrinsic radioresistance in breast cancer: Insights from transcriptomics and CRISPRi screens
              </a>
            </td>
          <td>
            B. McBean, Priyanka S. Rana, Benjamin Hauk, A. Michmerhuizen, S. Lichtman-Mikol, Reine Abou Zeidane, Vesna Markovic, K. Wilder-Romans, Mingfang Tao, Alan P. Boyle, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65dfedea9c7ed384b972c98e352341985ea58fa" target='_blank'>
              APOBEC3 mutagenesis drives therapy resistance in breast cancer.
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N. Brown, Hong Shao, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Yanjun Chen, Isabella Vegas, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, G. Curigliano, B. Weigelt, N. Riaz, Simon N. Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-05-16</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 PAX5 is known to be frequently mutated in childhood B-cell precursor acute lymphoblastic leukemia (B-ALL) via several molecular mechanisms, including chimeric fusion, focal deletion, and point mutations. Among these, patients with PAX5 internal tandem duplication (PAX5-ITD) are reported to have a poor outcome. Unlike other common ITDs (e.g., FLT3), PAX5-ITD often involves gain of more than one copy of involved exons, and the exact nature of these alterations remain elusive. Here we aim to investigate the following questions: 1) what are the typical copy number gains in DNA and RNA among PAX5-ITD cases? 2) are PAX5-ITD always in-frame? 3) are these aberrations a result of multi-step or single-step events? Short-read whole genome sequencing (WGS) and RNA sequencing (RNA-seq) and PacBio ISO-seq, a state-of-the-art long-read sequencing method, are used in this study. By integrative analysis of WGS and RNA-seq data from a relapse cohort (n=584) and a non-relapse cohort (n=1055), we identified PAX5 mutation rate of ∼30% in relapsed patients versus that of ∼20% in non-relapsed patients. Among the various PAX5 mutation types, the PAX5-ITD mutation shows the greatest degree of enrichment (P=3e-6) in relapsed patients (n=21), compared with those who remain in remission (n=5). Copy number gains in PAX5-ITD cases ranged from 3 to 25 and involved four exon combinations within the gene: E5, E3-E5, E2-E5 and E1-E5 and concordant copy number gains were detected between diagnosis tumor and relapse tumor. We observed a negative correlation between the number of copy gains and the length of duplicated genomic regions. Furthermore, 11 out of the 26 PAX5-ITD cases exhibited multiple structural variants (SVs) and 1 case harbored 2 somatic SNV in ITD region. To investigate the transcriptional consequences of these SV events, we performed ISO-seq on 7 PAX5-ITD cases with available specimen. By mapping SVs and SNVs to full-length RNA transcripts, we observed complex sequences of events driving the formation of ultimate PAX5-ITD in 3 cases. Besides, copy number analysis on full-length RNA transcripts revealed variable copy number gains within tumors due to alternative splicing. Additionally, we identified neo-splice sites in 2 cases, indicating that the splicing process was influenced by the altered DNA context created by SVs. Finally, our data indicated that all PAX5-ITDs can transcribe in-frame mRNAs, suggesting activating nature of these mutations. These findings provide valuable insights into the genomic complexity of PAX5-ITD and highlight the diverse transcriptional and splicing outcomes due to these alterations.


 Yanling Liu, Bensheng Ju, Li Dong, Melanie Loyd, Yuan Feng, Heather Mulder, Pandurang Kolekar, Li Fan, Hanxia Li, Scott Foy, Geoff Neale, Patrick A. Brown, William L. Carroll, Mignon L. Loh, John Easton, Xiaotu Ma. Uncovering genomic complexity of PAX5 internal tandem duplication using short-read and long-read sequencing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2617.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e44c370bd7521d24afccebc5dd604dc8d4b6c7b" target='_blank'>
              Abstract 2617: Uncovering genomic complexity of PAX5 internal tandem duplication using short-read and long-read sequencing
              </a>
            </td>
          <td>
            Yanling Liu, B. Ju, Li Dong, Melanie Loyd, Yuan Feng, H. Mulder, P. Kolekar, Li Fan, Hanxia Li, S. Foy, Geoffrey Neale, Patrick A. Brown, William L Carroll, M. Loh, John Easton, Xiaotu Ma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6350a9e7edaa4d3aae5b8855bca2a2b934a43915" target='_blank'>
              P53 deficiency triggers hypertranscription, inducing nucleotide insufficiency causing replication stress, and genomic instability
              </a>
            </td>
          <td>
            Wisam Zaatra, George Philippos, Petr Smirnov, Shira Milo, Jan Otonicar, Michelle Chan, Michal Harel, Frauke Devens, Tchelet Goldberg, A. Eliassaf, Karen Grimes, M. Sinai, Gianluca Sigismondo, Tamar Geiger, Jan O. Korbel, Jeroen Krijgsveld, O. Shalev, Batsheva Kerem, Aurélie Ernst
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c8de13c6fd974bf26d03a422a6ee56b54180c3" target='_blank'>
              Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, F. Cidre-Aranaz, A. Wenninger-Weinzierl, M. Zylka, Karla Queiroz, D. Kurek, A. Sastre, Javier Alonso, Martin Distel, Anna Obenauf, T. G. Grünewald, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Osteosarcoma is the most common primary malignant bone tumor in childhood. Advancements in therapy have been stagnant for over 40 years and the prognosis for osteosarcoma remains challenging. The 5-year relative survival rate of localized osteosarcoma is 75%. If disease is metastatic upon diagnosis or first line therapy fails, survival rate falls to approximately 25%. First line therapy consists of alternating cycles of methotrexate with doxorubicin and cisplatin (MAP). In contrast, second line therapy is not standardized, and outcomes after salvage therapy remain poor. Identifying better second line agents may be an avenue to successful salvage therapy. Cancer is a dynamic state and can be viewed through an evolutionary framework. When cancer cells are exposed to chemotherapy, resistant cells may be selected for and proliferate. These resistant cells may exhibit a different resistance/sensitivity profile to other therapy options, termed collateral changes. By correlating these collateral changes with genomic and transcriptomic data, we can theoretically develop a library of gene/gene expression markers that correlate with resistance or sensitivity to a panel of drugs.



 We have exposed an osteosarcoma cell line, MG63.3, in cell culture to a clinically relevant sequential therapy that models MAP. 5 evolutionary replicates were exposed to combination cisplatin and doxorubicin, followed by methotrexate. 3 control replicates were exposed to solvent without drug. Once cells returned to confluency after each exposure to chemotherapy, we performed drug assays to a panel of 16 second-line drugs and drug combinations, in addition to radiation therapy, and extracted DNA/RNA from each replicate.



 As a result of this work, we created a heatmap displaying the development of sensitivity and resistance to this treatment panel, based on the change in half maximal effective concentration (EC50) over time and have found variable collateral changes indicating timepoints in which cells are significantly more sensitive or resistant to second line agents as our evolutionary replicates evolved resistance to frontline therapy.



 This heat map can serve as a springboard for transcriptomic and genomic analyses to correlate the changes with resistance and sensitivity. This may allow future researchers to correlate our findings with patient derived genomics and transcriptomics to ultimately choose better second line therapeutics for patients who do not respond to MAP therapy.



 Gabriel Mandel, Elizabeth Nowak, Jarrel Imamur, Kristi Lin-Rahrdja, Mina Dinh, Masahiro Hitomi, Jacob Scott, Zachary Burke. Collateral drug screening throughout the evolution of osteosarcoma drug resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4446.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3aaeeac848f38939d26de5e26a501762000386d" target='_blank'>
              Abstract 4446: Collateral drug screening throughout the evolution of osteosarcoma drug resistance
              </a>
            </td>
          <td>
            Gabriel Mandel, Elizabeth Nowak, Jarrel Imamur, Kristi Lin-Rahrdja, Mina N. Dinh, Masahiro Hitomi, Jacob G. Scott, Zachary Burke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In recent years, synthetic lethality has become an important theme in the field of targeted cancer therapy. Synthetic lethality refers to simultaneous defects in two or more genes leading to cell death, whereas defects in any single gene do not lead to cell death. Taking advantage of the genetic vulnerability that exists within cancer cells, it theoretically has no negative impact on healthy cells and has fewer side effects than non-specific chemotherapy. Currently, targeted cancer therapies focus on inhibiting key pathways in cancer. However, it has been found that over-activation of oncogenic-related signaling pathways can also induce cancer cell death, which is a major breakthrough in the new field of targeted therapies. In this review, we summarize the conventional gene targets in synthetic lethality (PARP, ATR, ATM, WEE1, PRMT) and provide an in-depth analysis of their latest potential mechanisms. We explore the impact of over-activation of pathways such as PI3K/AKT, MAPK, and WNT on cancer cell survival, and present the technical challenges of current research. Important theoretical foundations and insights are provided for the application of synthetic lethal strategies in cancer therapy, as well as future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a12057bb33fb82ecea65af89cddf52c4a9e3cd" target='_blank'>
              Perspectives on cancer therapy—synthetic lethal precision medicine strategies, molecular mechanisms, therapeutic targets and current technical challenges
              </a>
            </td>
          <td>
            Shixuan Peng, Mengle Long, Qisheng Chen, Zhijian Yin, Chang Zeng, Wanyong Zhang, Qingyang Wen, Xinwen Zhang, Weiqi Ke, Yongjun Wu
          </td>
          <td>2025-04-16</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6f92731e4153548f5adf8b4c57b2dcf0276aa2" target='_blank'>
              Single-cell transcriptomics uncovers that cell-to-cell communication controls p53 activation
              </a>
            </td>
          <td>
            L. Castellano, Mark E Robinson, Aleksandra Dabrowska, Niklas Feldhahn, S. Ottaviani, Justin Stebbing
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="RET-fusion-driven lung adenocarcinomas (LUADs) are a rare and particularly aggressive subtype of lung cancer, often affecting young, non-smoker populations. While selective RET-TKI therapies are initially highly effective, they often fail to provide durable responses, with resistance mechanisms emerging over time. Notably, approximately 40% of patients exhibit resistance mechanisms that remain unidentified. To uncover these unknown mechanisms, we performed a comprehensive multi-omic analysis of clinical samples from patients with RET-fusion lung cancers before and after RET-TKI treatment. RNA sequencing analysis revealed inactivation of RB1 and upregulation of DNA damage repair (DDR) pathways in resistant tumors relative to TKI-sensitive samples. Accordingly, methylome analysis confirmed hypomethylation of E2F targets and DDR pathways, suggesting that their transcriptomic upregulation is mainly driven by epigenetic changes. We hypothesized that DDR pathway upregulation was a result of inhibition of Rb1 during TKI treatment. To validate this, we knocked down Rb1 or overexpressed of E2F, and confirmed upregulation of key DDR genes such as BRCA1, MSH6, and Rad18 in RET cell lines. To validate the role of DDR pathways in resistance, we established isogenic cell lines by overexpressing or knocking out BRCA1, MSH6, and Rad18, and conducted drug-tolerant persister (DTP) assays. Genetic inhibition of each of these DDR proteins significantly reduced the generation of DTPs after high dose treatment with the RET-TKI selpercatinib (4.5 uM) while overexpression increased the number of DTPs. We next evaluated the inhibition of XPO1, an exportin protein linked to DDR pathway activation, as a strategy to target DTPs. Treatment with selinexor, an FDA-approved XPO1 inhibitor, in combination with selpercatinib strongly reduced the number of DTPs in vitro, and overexpression of the DDR genes BRCA1, MSH6, and Rad18 rescued this effect. In vivo experiments using RET-driven patient-derived xenograft (PDX) models confirmed that treatment with combination of selpercatinib with selinexor significantly delayed tumor relapse in TKI-sensitive PDX models (n=4) and resensitized resistant PDX tumors to selpercatinib (n=2). Our research highlights DDR pathways as a critical mechanism of resistance via Rb1 inactivation in TKI-treated RET-driven lung cancers and identifies XPO1 inhibition as a promising therapeutic strategy to overcome this resistance. Finally, we also found upregulation of DDR proteins upon treatment with lorlatinib and osimertinib in ALK and EGFR driven LUAD PDXs, respectively, suggesting that targeting DDR pathways could be an effective strategy against resistance in a pan-TKI setting across other RTK-driven LUADs.


 Fathema Z. Uddin, Samuel Tischfield, Romel Somwar, Alexander Lim, Hyung Jun Woo, Christina Falcon, Barbara P. Mello, Dennis Kinyua, Harsha Sridhar, Parvathy Manoj, Hong Zhong, Juan Qiu, Elisa de Stanchina, Mark Donoghue, Mark Ladanyi, Esther Redin, Charles M. Rudin, Alvaro Quintanal-Villalonga, Alexander Drilon. DNA damage repair pathways enable a drug-tolerant persister state in RET-fusion NSCLC and precedes TKI resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6382.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0da8bbe26d7b85bc1bd170b1562ddebfb5b49ba9" target='_blank'>
              Abstract 6382: DNA damage repair pathways enable a drug-tolerant persister state in RET-fusion NSCLC and precedes TKI resistance
              </a>
            </td>
          <td>
            F. Uddin, Samuel Tischfield, R. Somwar, Alexander Lim, Hyung Jun Woo, Christina J Falcon, Barbara P Mello, Dennis Kinyua, Harsha Sridhar, P. Manoj, Hong Zhong, Juan Qiu, E. de Stanchina, M. Donoghue, Marc Ladanyi, E. Redin, Charles M. Rudin, Á. Quintanal-Villalonga, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="In recent years, targeted therapies have become the standard of care for refractory/relapsed mantle cell lymphoma (MCL). Although the mutational profile of MCL has been extensively studied, there is a lack of understanding of resistance mechanisms and genetic factors that impact the response to novel treatments. Since patients relapsing on targeted treatment experience poor clinical outcomes, understanding the genetic foundation of resistance mechanisms in MCL is essential. In this study, we aimed to scrutinize the copy number profile and clonal dynamics of double-resistant MCL patients treated sequentially with Bruton's tyrosine kinase inhibitor (BTKi) and venetoclax using low-coverage whole genome sequencing (lcWGS). Samples obtained after systemic therapy showed more copy number alterations (CNAs) (p = 0.039; Wilcoxon) compared to samples collected before treatment initiation. Patients showing early progression on BTKi demonstrated CNAs affecting cytobands encompassing the coding regions of NOTCH1, TRAF2, BIRC2, BIRC3, and ATM. A deletion in chromosome 9p21.3 was identified in two out of three venetoclax-resistant patients. For patient MCL2, progressing on ibrutinib but showing venetoclax resistance, a 9p21.3 deletion was found throughout the disease course, with acquired SMARCA4-del(19)(p13.3-q13.11) and DLC1-del(8)(p23.2-q11.1) observed at relapse, highlighting their role in disease progression and therapy resistance. Using lcWGS, an innovative genome-wide approach, this study revealed novel putative primary and acquired resistance mechanisms in BTKi and venetoclax double-resistant MCL patients. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99df806a910f464aab37d45027b5e7fa26c9d503" target='_blank'>
              Resistance mechanisms and clonal dynamics in mantle cell lymphoma treated with sequential BTKi and venetoclax therapy.
              </a>
            </td>
          <td>
            Tamás László, L. Pinczés, B. Bátai, Luca Varga, B. Timár, Anita Gulyás, Ilona Tárkányi, M. Plander, Z. Nagy, P. Rajnics, M. Egyed, Zsuzsa Molnár, János Rottek, A. Masszi, P. Tamáska, Róbert Szász, Árpád Illés, D. Alpár, F. Magyari, Csaba Bödör
          </td>
          <td>2025-05-15</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58e72b48369a6d6279bf45ff8835cc869b1ce39" target='_blank'>
              CRISPR screens in the context of immune selection identify CHD1 and MAP3K7 as mediators of cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Alex Watterson, G. Picco, Vivien Veninga, Youhani Samarakoon, C. M. Cattaneo, Sara F Vieira, Emre Karakoc, S. Bhosle, Thomas W Battaglia, Sarah Consonni, Timotheus Y. F. Halim, E. Voest, Mathew J. Garnett, Matthew A. Coelho
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Over 50% of melanomas are driven by BRAFV600 mutations. Clinically, targeted inhibitors of mutant BRAF and MEK are the primary treatment for more than 50% of melanoma patients that are refractory to standard-of-care immunotherapy. While targeted therapies are initially clinically effective, a subset of residual cancer cells persist and inevitably drive tumor recurrence and drug resistance. While some cells persist due to pre-existing mechanisms (e.g. genetic mutations), others can non-genetically adapt to tolerate drug by rewiring their internal signaling cascades. Importantly, a key feature of these non-genetically adapted persister cells is that their drug-tolerant phenotype is acquired in response to drug pressure, making them extremely challenging to identify and characterize.
 Using cutting-edge long-term time-lapse microscopy and an automated single-cell tracking pipeline, we studied the behaviors of persister cells in drug-treated BRAFV600E melanoma populations. Notably, we found significant heterogeneity within persister populations. While all persister cells initially enter a state of drug-induced quiescence, a subset of these cells—termed “escapees”—can escape drug action and resume proliferation. By computationally reconstructing single-cell lineages from long-term time-lapse imaging experiments, we monitored key signaling pathways before, during, and after escape from drug. Using this approach, we identified a significant increase in mTorC1 signaling activity in persisters that would go on to escape drug. Additionally, mTorC1 signaling caused increases in protein translation and cell growth rates during the quiescent periods of these cells.
 The abundance of Cyclin D1, a key driver of proliferation, is extremely sensitive to cellular translation rates due to its short half-life. In order to determine if the observed increase in mTorC1-mediated growth and translation is driving the upregulation of Cyclin D1 levels past a threshold necessary for escape, we used CRISPR to tag Cyclin D1 at the endogenous locus. Importantly, we found that escapees significantly upregulate Cyclin D1 levels in the hours prior to their escape, compared to non-escapees. These findings have important clinical implications because they suggest that Cyclin D1 protein levels are a key bottleneck for cell-cycle re-entry under MAPK inhibition. Additionally, these findings implicate Cyclin D1 as one of the first markers that can distinguish future escapees from non-escapees in the quiescence period before these distinct cell-cycle fates emerge.


 Varuna Nangia, Humza Ashraf, Nasreen Marikar, Victor Passanisi, Christopher Ryland Ill, Timothy Hoffman, Sabrina Leigh Spencer. Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6418.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13bcea3579836571808ac96de5e2e45ad09f6441" target='_blank'>
              Abstract 6418: Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma
              </a>
            </td>
          <td>
            Varuna Nangia, Humza M. Ashraf, Nasreen Marikar, Victor J. Passanisi, Christopher Ryland Ill, Timothy E. Hoffman, Sabrina L. Spencer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Lung cancer is the leading cause of cancer mortality worldwide. Mutations in the epidermal growth factor receptor (EGFR) account for approximately 27% of all lung adenocarcinoma (LUAD) cases. Osimertinib is a third generation EGFR tyrosine kinase inhibitor (TKI) that is, initially, remarkably effective at treating patients with EGFR mutant LUAD. Acquired resistance to Osimertinib develops in most patients leaving few therapeutic options. This highlights an unmet clinical need to prevent/combat Osimertinib resistance. Lineage plasticity, an epigenetic process characterized by transcriptional heterogeneity and molecular adaptation, has been identified as a mechanism of Osimertinib resistance in the clinic. As more potent targeted therapeutics are deployed for treating cancer patients, plasticity-based resistance will likely increase. The molecular basis of lineage plasticity is not well understood, and greater characterization of lineage plasticity is imperative for developing new therapies aimed at preventing/reversing plasticity-associated resistance. Loss of the major tumor suppressor gene Rb1 has previously been implicated in lineage plasticity of several different cancers including LUAD. How loss of Rb1 contributes towards lineage plasticity remains poorly understood. Rb1 is a known regulator of chromatin accessibility, playing a key role in recruiting epigenetic writers to silence gene expression through repressive methylation of histones and DNA. Whether altered chromatin regulation in the absence of Rb1 contributes to LUAD lineage plasticity has not been examined. To begin examining the chromatin accessibility/plasticity axis in vitro, we are developing a PROTAC-based, rapidly inducible Rb1 protein degradation system utilizing the dTAG degron system to characterize the dynamics of molecular changes subsequent to Rb1 loss of function. Changes in chromatin accessibility will be assessed by micrococcal nuclease digestion and a novel nuclear propidium iodide staining technique. Plasticity will be assessed indirectly by in vitro colony formation and time to resistance assays. The timing of changes in transcriptional plasticity and chromatin accessibility will be compared. The goal of this project is to gain a greater understanding of how Rb1 loss impacts chromatin structure and subsequently, whether increased chromatin accessibility promotes plasticity-associated Osimertinib resistance.


 Bradley Balk, David W. Goodrich. Studying the role of chromatin accessibility in lineage plasticity of Rb1-deficient EGFR-mutant lung adenocarcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1259.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dead0a4e6712a90fb575b926eb0a4af6b736d577" target='_blank'>
              Abstract 1259: Studying the role of chromatin accessibility in lineage plasticity of Rb1-deficient EGFR-mutant lung adenocarcinoma
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 The early detection and characterization of rare cellular populations in preneoplastic lesions are crucial for understanding cancer initiation and progression, as well as for identifying novel therapeutic targets. Rare cell populations, often overlooked due to their low abundance, play pivotal roles in shaping the tumor microenvironment and driving disease development. In this study, we employed a lineage tracing system based on tracing vectors introduced into a 2D cell line model of esophageal preneoplasia. This approach allowed us to track genetic perturbations that accumulate during cell proliferation, enabling precise identification of distinct cellular lineages. By coupling this lineage tracing method with single-cell RNA sequencing (scRNA-seq), we achieved high-resolution transcriptomic profiling, uncovering rare cellular populations with unique molecular and lineage-specific characteristics. The rare populations identified through this integrated approach exhibited a quiescent phenotype, suggesting their involvement in lesion persistence and resistance to therapies targeting proliferative cells. Transcriptomic analysis revealed distinct genetic signatures and key pathways associated with these populations, offering potential avenues for therapeutic intervention and biomarker development. Our findings demonstrate the value of combining genetic perturbation-based lineage tracing with scRNA-seq to uncover rare and functionally significant cellular populations. This study provides new insights into the cellular dynamics underlying esophageal cancer initiation and progression, laying the groundwork for future research aimed at understanding the roles of these populations and developing targeted therapies.


 Jinho Jang, Kyung-Pil Ko, Jie Zhang, Jae-Il Park. Uncovering rare cellular populations in esophageal preneoplasia through genetic perturbation tracking and scRNA-seq [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7490.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43978659ce38ef19fbe0b48f8cd99b848f7e9b3" target='_blank'>
              Abstract 7490: Uncovering rare cellular populations in esophageal preneoplasia through genetic perturbation tracking and scRNA-seq
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Jae-Il Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Adenosine base editors (ABEs) facilitate A·T to G·C base pair conversion with significant therapeutic potential for correcting pathogenic point mutations in human genetic diseases, such as sickle cell anemia and β-thalassemia. Unlike CRISPR–Cas9 systems that induce double-strand breaks, ABEs operate through precise deamination, avoiding chromosomal instability. However, the off-target editing effects of ABEs remain inadequately characterized. In this study, we present a biochemical method Selict-seq, designed to evaluate genome-wide off-target editing by ABEs. Selict-seq specifically captures deoxyinosine-containing single-stranded DNA and precisely identifies deoxyadenosine-to-deoxyinosine (dA-to-dI) mutation sites, elucidating the off-target effects induced by ABEs. Through investigations involving three single-guide RNAs, we identified numerous unexpected off-target edits both within and outside the protospacer regions. Notably, ABE8e(V106W) exhibited distinct off-target characteristics, including high editing rates (>10%) at previously unreported sites (e.g. RNF2 and EMX1) and out-of-protospacer mutations. These findings significantly advance our understanding of the off-target landscape associated with ABEs. In summary, our approach enables an unbiased analysis of the ABE editome and provides a widely applicable tool for specificity evaluation of various emerging genome editing technologies that produce intermediate products as deoxyinosine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7f86cbbb409d5234a83d13231df0efe7d59590" target='_blank'>
              Selict-seq profiles genome-wide off-target effects in adenosine base editing
              </a>
            </td>
          <td>
            Kexin Yuan, Xin Xi, Shaoqing Han, Jingyu Han, Bin Zhao, Qi Wei, Xiang Zhou
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="
 The presence of oncogenic driver mutations such as EGFR, ALK, or KRAS renders non-small cell lung adenocarcinomas sensitive to molecularly targeted therapies, however the inevitable emergence of acquired drug resistance remains an unsolved clinical problem. Osimertinib, a third-generation EGFR inhibitor that was initially developed to overcome the EGFRT790M resistance mutation, is now the standard first-line therapy. Although various mechanisms of acquired resistance to osimertinib have been identified, the specific drivers of resistance remain unexplained in over half of patients. Recent work from our group and others has suggested that cancer-associated fibroblasts may play a role in drug resistance to tyrosine kinase inhibitors (TKIs). Notably, we have observed that patient-derived cancer-associated fibroblast (PDF) cell lines established from TKI-resistant tumors display heterogeneous phenotypes with ability to rescue cancer cells from TKIs. To enable parallel discovery of both tumor cell-intrinsic and stroma-driven resistance mechanisms to osimertinib, we developed a genome-wide CRISPR sensitizer screening approach that incorporates osimertinib-resistant patient-derived EGFR-mutant NSCLC cell lines and conditioned media from matched immortalized PDFs. These studies aim to uncover intrinsic and extrinsic resistance pathways and nominate targets for next-generation therapies to overcome acquired resistance to osimertinib


 Radhika Koranne, Aimee Gleason, Emily Hensley, Xiaodong Wang, Alba Font-Tello, Brian Rabe, Fayaz Seifuddin, Eugene Drokhlyansky, Kartik Subramanian, Martino Colombo, Jason Dobson, Mavrakis J. Konstantinos, Leila Dardaei, Aaron Hata. Decoding tumor intrinsic and stroma-related drug resistance in EGFR mutant NSCLC [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b24fd08026b81ce4d7ce4581da39b975952905" target='_blank'>
              Abstract 5108: Decoding tumor intrinsic and stroma-related drug resistance in EGFR mutant NSCLC
              </a>
            </td>
          <td>
            Radhika Koranne, Aimee Gleason, Emily Hensley, Xiaodong Wang, Alba Font-Tello, Brian Rabe, Fayaz Seifuddin, Eugene Drokhlyansky, Kartik Subramanian, Martino Colombo, Jason R. Dobson, Mavrakis J. Konstantinos, Leila Dardaei, Aaron N. Hata
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Establishing DNA as the genetic material, deciphering the genetic code and defining the double helical structure of the genome provided a foundation for pursuing a mechanistic understanding of gene expression that is operative for biological control and pathology. Equally important are a series of discoveries that have established epigenetic mechanisms, defined as the control of gene expression by non-DNA-encoded regulation, mediated by an evolving histone code that supports chromatin states, as complementary and obligatory determinants for biological activity. Evidence is accruing for pivotal contributions by epigenetic parameters of control to gene expression that is required for development and differentiation as well as for transient and sustained responsiveness to physiological regulatory cues that support cell structure, function, cell survival, tissue repair and remodeling. Epigenetic control is emerging as a decisive contributor to the regulatory compliance for the highly integrative processes that balance phenotypic function and tumor suppression with aberrant gene expression that accompanies the onset and progression of disease, strikingly illustrated with tumorigenesis and skeletal disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4c2c01eadaece3705383df803332330abc2445" target='_blank'>
              Reflections and perspectives on epigenetically mediated biological control: compromises in cancer and skeletal pathology
              </a>
            </td>
          <td>
            Gary S. Stein
          </td>
          <td>2025-04-11</td>
          <td>Academia Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Alterations in the genome that drive the transformation of normal cells into malignant cells program cancer initiation and progression. This rewiring also induces unique dependencies for genes and pathways that can be targeted therapeutically. Even though we have a clearer view of the spectrum of these molecular alterations, we still lack a complete understanding of how these alterations affect biological processes and create specific vulnerabilities in cancer cells. To address this, we have created the Cancer Dependency Map (DepMap) to systematically identify and map cancer vulnerabilities. Here, we provide an overview of the history and development of the current DepMap. We also highlight biological insights enabled by DepMap. Findings from DepMap will provide insights into new targets suitable for drug discovery efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a21c97e108e69ab358170dee175b96d9645c472" target='_blank'>
              Cancer Dependency Map: Past and Present
              </a>
            </td>
          <td>
            Rand Arafeh, T. Shibue, Francisca Vazquez, William C Hahn
          </td>
          <td>2025-04-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, J. Khandare, G. Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 The combination of sitravatinib, nivolumab, and ipilimumab offers a novel approach to enhance immunotherapy outcomes in advanced clear cell renal cell carcinoma (ccRCC). To explore the immunomodulatory and resistance mechanisms underlying this triplet therapy, we performed single-cell RNA sequencing (scRNA-seq) on tumor samples collected longitudinally from 12 patients across treatment timepoints. Transcriptomic data from 70,526 cells, including 58,331 tumor microenvironment (TME) cells, revealed dynamic shifts in cellular composition and transcriptional states during therapy. Analysis of TME cells identified eight major cell types, with notable increases in T cells and reductions in myeloid and endothelial cells post-treatment. Baseline T cell abundance correlated with improved clinical response, while fibroblast enrichment predicted resistance. Tumor cells were classified into ten transcriptionally distinct clusters, including a treatment-resistant population (C8) enriched in epithelial-mesenchymal transition (EMT), hypoxia, and glycolysis signatures. These signatures correlated with shorter progression-free survival in the trial cohort and worse survival outcomes in an external TCGA ccRCC dataset. T cell states showed a transition from cytotoxic to exhausted phenotypes in non-responders and end-of-treatment samples, suggesting a role for T cell dysfunction in resistance. Similarly, myeloid cells exhibited a shift toward immunosuppressive phenotypes, characterized by M2-like macrophages and inhibitory signaling pathways. These findings elucidate resistance mechanisms to the triplet therapy, highlighting EMT-driven tumor cell states and immunosuppressive TME dynamics as key contributors. The identification of a tumor-specific C8 signature provides a potential biomarker for predicting poor responses and tailoring therapeutic strategies. Further validation in larger cohorts is needed to optimize patient selection and therapeutic outcomes.


 Kai Yu, Ying Yuan, Jianfeng Chen, Xinmiao Yan, Menuka Karki, Fei Duan, Rahul Annil Sheth, Priya Rao, Kanishka Sircar, Amishi Yogesh Shah, Amado Zurita-Saavedra, Giannincola Genovesa, Min Li, Chih-Chen Yeh, Minghao Dang, Guangchun Han, Yanshuo Chu, Max Hallin, Peter Olson, Rui Yang, Daniela Slavin, Hirak Der-Torossian, Curtis Chin, Nizar M Tannir, Linghua Wang, Jianjun Gao, Pavlos Msaouel. Single-cell transcriptomic insights into tumor and immune dynamics driving resistance to sitravatinib, nivolumab, and ipilimumab in advanced clear cell renal cell carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3971.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86c2e0fe302cf21b65e16778aa8ab9c33ba4b41f" target='_blank'>
              Abstract 3971: Single-cell transcriptomic insights into tumor and immune dynamics driving resistance to sitravatinib, nivolumab, and ipilimumab in advanced clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Kai Yu, Ying Yuan, Jianfeng Chen, Xinmiao Yan, Menuka Karki, Fei Duan, Rahul A Sheth, Priya Rao, K. Sircar, A. Shah, A. Zurita-Saavedra, Giannincola Genovesa, Min Li, Chih-Chen Yeh, M. Dang, Guangchun Han, Yanshuo Chu, Max Hallin, Peter Olson, Rui Yang, Daniela Slavin, H. Der-Torossian, Curtis Chin, N. Tannir, Linghua Wang, Jianjun Gao, P. Msaouel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="
 8107


 Background:
 Relapsed small cell lung cancer (R-SCLC) is characterized by poor outcomes and treatment resistance. The mechanisms driving treatment resistance in R-SCLC remain poorly understood. We comprehensively profiled R-SCLC samples along with patient-matched treatment-naive samples (TN-SCLC) using whole-exome (WES), whole-genome (WGS), and RNA-sequencing (RNA-seq) to catalog mechanisms of treatment resistance and identify actionable alterations.
 Methods:
 We analyzed 54 R-SCLC and 27 TN-SCLC samples (with 26 patient-matched TN-R pairs) using WES. At presentation, 31 patients were diagnosed with extensive and 23 with limited stage SCLC. R-SCLC samples were also analyzed by WGS (n = 28, including 18 TN-R pairs) and RNA-seq (n = 31, including 12 TN-R pairs). Differences in mutational signatures, structural variations, gene expression, alternative splicing, and neoantigen profiles were investigated between TN and R-SCLC samples.
 Results:
 R-SCLC samples contained mutation signatures characteristic of platinum exposure and APOBEC mutagenesis, which were absent in TN-SCLC. MYC family genes (MYC, MYCL, MYCN) were frequently amplified at relapse. Transcriptomic analyses revealed dysregulation of WNT signaling and IDO1. Extensive differences in alternative splicing, especially intron retention (IR), were observed (96% of all IR events were in TN-SCLC). IR in TN-SCLC affected genes involved in DNA repair, RNA metabolism, WNT and MYC pathways. TN-SCLC showed a median of 86 neoantigens, and R-SCLC 90 neoantigens per sample.
 TP53
 was

 the most frequently altered gene to result in a neo-antigen (48% of analyzed samples). Immune evasion mechanisms, including upregulation of CD24 and downregulation of MHC-I, were observed in R-SCLC samples.
 Conclusions:
 This study highlights the heterogeneity of treatment resistance in SCLC, driven by genomic instability, WNT and MYC dysregulation, and splicing aberrations. Potential therapeutic strategies for R-SCLC include targeting splicing machinery, WNT signaling, and immune evasion pathways. These findings advance our understanding of SCLC biology and provide a foundation for biomarker-driven drug development.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/692ba79a3de8247d26cc5aaf83a2d2b738adb23f" target='_blank'>
              Cataloging genomic and transcriptomic features of relapsed SCLC.
              </a>
            </td>
          <td>
            Sid Devarakonda, S. Sankararaman, My H Hoang, Zach L. Skidmore, Brett H. Herzog, K. Pepin, Joshua F. McMichael, Jeffrey Ward, D. Morgensztern, S. Waqar, M. Griffith, O. Griffith, T. Oliver, R. Govindan
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="


 Lung cancer is the second most prevalent cancer with the highest mortality rate in the world. Paclitaxel (PTX) is one of the most effective clinical chemotherapeutic agents for the treatment of lung cancer, but a variety of mechanisms induce its chemoresistance including alterations in microtubule-associated proteins, up-regulation of drug efflux systems. Lipid metabolism was reported to impact chemosensitivity, but the underlying mechanisms are to be clarified. Therefore, this study aims to exploring the molecular mechanisms of PTX resistance regulated by lipid metabolism, and to discover novel PTX resistant targets and therapeutic agents.



 Lipid metabolism related sgRNA library was used for CRISPR screen. PTX-resistant lung cancer cells were constructed for transcriptome sequencing. The key regulators were identified via integrating negative enriched genes in CRISPR screen and the differential ones in transcriptome sequencing. The regulation of PTX sensitivity by the key regulators was confirmed in deficient lung cancer cells. Transcriptome sequencing was applied to find downstream pathways, and immunoblotting and RT-qPCR were used to confirm the molecular mechanisms. Mouse models were used to investigate the safety and effectiveness of its inhibitors combined with PTX for lung cancer treatments in vivo.



 DGKA is highly expressed in PTX-resistant lung cancer cells, and its deficiency significantly induced PTX sensitivity in lung cancer cells. DGKA knockdown significantly inhibited lung cancer cell proliferation, invasion and migration. In addition, DGKA depletion promoted PYX-induced cell apoptosis and G2/M-phase arrest in lung cancer cells. Transcriptomics showed that the MAPK signaling pathway was significantly enriched after DGKA knockdown, and further experiments confirmed that DGKA inhibition increased lung cancer cell sensitivity to PTX via inhibiting ERK phosphorylation. DGKA inhibitor (R59022) could induce the PTX sensitivity of lung cancer cells safely and effectively in vivo.



 DGKA promotes PTX sensitivity of lung cancer cells and alleviates PTX resistance through the MAPK pathway, providing preclinical evidence for targeting DGKA as a novel strategy for lung cancer therapy against PTX resistance.



 Jiang Luo, Mini Zhang, Jiawen Tang, Qingwei Wang, Yu Yuan, Min Xia, Yufei Yan, Yushuang Tan, Conghua Xie, Yan Gong. Identification of DGKA as a regulatory factor for paclitaxel resistance in lung cancer based on CRISPR screen and transcriptome sequencing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6789.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874bff63660c812dc6306123212ce70b86ccda05" target='_blank'>
              Abstract 6789: Identification of DGKA as a regulatory factor for paclitaxel resistance in lung cancer based on CRISPR screen and transcriptome sequencing
              </a>
            </td>
          <td>
            Jiang Luo, Mini Zhang, Jiawen Tang, Qingwei Wang, Yu Yuan, Min Xia, Yufei Yan, Yushuang Tan, Conghua Xie, Yan Gong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 4116


 Background:
 Biliary tract cancers (BTC) are rare, highly aggressive malignancies with limited treatment options, leading to consistently poor outcomes. An improved understanding of BTC tumor evolution could inform enhanced screening strategies, identify useful prognostic markers, and discover novel therapeutic targets.
 Methods:
 We performed whole genome and transcriptome sequencing (WGTS) at high depth ( > 80X) on a prospective cohort of BTC tumors. After detecting mutations and mutational signatures, we developed two novel methods for driver identification and evolutionary reconstruction. First, we created new oncogene and tumor suppressor-specific models that integrate copy number profiles, structural variant breakends, and expression changes to distinguish structural drivers from neutral chromosomal rearrangements. Second, we applied population genetics techniques to fit demographic models to tumor allele frequencies across 20 paired primary and metastatic samples.
 Results:
 We analyzed 130 tumor samples from 110 patients, representing the largest BTC whole-genome cohort to date. We identified hypermutated tumors (50-150 mutations/mb) with distinct etiologies, including mismatch repair deficiency, platinum exposure, tobacco use, and aristolochic acid-related damage, the latter of which was associated with response to immunotherapy. Ourintegrated-driver approach identified an association between selection on
 RAD23A
 and an increased structural variant load, resulting in a tandem-duplicator-like mutational phenotype. This also revealed an underappreciated impact of
 SMAD4
 in BTC, which is inactivated through multiple mutation types in 12%. Notably,
 BAP1
 mutations occurred in 24% of cases, including 4% that were inactivated through deletion of the
 BAP1
 promoter that lowered transcript expression, a previously undescribed mechanism. By pairing clinical data to the genomics, we observed a co-occurrence of
 BAP1
 mutations and
 FGFR2
 fusions in small-duct, mass-forming intrahepatic cholangiocarcinomas. These mutations were mutually exclusive with
 TP53
 mutations, which were enriched in patients with primary sclerosing cholangitis. Finally, our novel method for subclonal population reconstruction on paired samples illuminated recent tumor evolutionary dynamics and identified an
 ARID1B
 fusion with a potential role in metastasis.
 Conclusions:
 This study uncovered novel genomic mechanisms underlying the evolutionary origins of BTC beyond those previously identified with exome and panel sequencing, highlighting the value of WGTS. These results highlight the complex genomic heterogeneity of BTC, with potential implications for precision therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bbdd5f85cf47a722dc40e8dbc7f432d73cb4b4" target='_blank'>
              Whole-genome sequencing of biliary tract cancer: Uncovering the genomic origins of evolutionary trajectories.
              </a>
            </td>
          <td>
            Felix Beaudry, Nicholas Light, Duhan Yendi, A. Zorigtbaatar, Maggie Hildebrand, A. Dodd, Roxana Bucur, Julie M. Wilson, Enrique Sanz Garcia, Xin Wang, Erica S. Tsang, G. O’Kane, Steven Gallinger, Arndt Vogel, Jennifer J Knox, F. Notta, G. Sapisochin, Robert C Grant
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background About 40% of relapsed or non-responder tumors exhibit therapeutic resistance in the absence of a clear genetic cause, suggesting a pivotal role of intracellular communication. A deeper understanding of signaling pathways rewiring occurring in resistant cells is crucial to propose alternative effective strategies for cancer patients. Methods To achieve this goal, we developed a novel multi-step strategy, which integrates high sensitive mass spectrometry-based phosphoproteomics with network-based analysis. This strategy builds context-specific networks recapitulating the signaling rewiring upon drug treatment in therapy-resistant and sensitive cells. Results We applied this strategy to elucidate the BCR::ABL1-independent mechanisms that drive relapse upon therapy discontinuation in chronic myeloid leukemia (CML) patients. We built a signaling map, detailing - from receptor to key phenotypes - the molecular mechanisms implicated in the control of proliferation, DNA damage response and inflammation of therapy-resistant cells. In-depth analysis of this map uncovered novel therapeutic vulnerabilities. Functional validation in patient-derived leukemic stem cells revealed a crucial role of acquired FLT3-dependency and its underlying molecular mechanism. Conclusions In conclusion, our study presents a novel generally applicable strategy and the reposition of FLT3, one of the most frequently mutated drivers of acute leukemia, as a potential therapeutic target for CML relapsed patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02185-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff332b1ad425972fa586920b8d83716a55d5548" target='_blank'>
              A network-based approach to overcome BCR::ABL1-independent resistance in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Valeria Bica, Veronica Venafra, Giorgia Massacci, Simone Graziosi, Sara Gualdi, G. Minnella, F. Sorà, P. Chiusolo, Maria Elsa Brunetti, G. Napolitano, Massimo Breccia, Dimitrios Mougiakakos, Martin Böttcher, Thomas Fischer, Livia Perfetto, Francesca Sacco
          </td>
          <td>2025-04-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="The DNA damage response (DDR) is a multifaceted network of pathways that preserves genome stability1,2. Unravelling the complementary interplay between these pathways remains a challenge3,4. Here we used CRISPR interference (CRISPRi) screening to comprehensively map the genetic interactions required for survival during normal human cell homeostasis across all core DDR genes. We captured known interactions and discovered myriad new connections that are available online. We defined the molecular mechanism of two of the strongest interactions. First, we found that WDR48 works with USP1 to restrain PCNA degradation in FEN1/LIG1-deficient cells. Second, we found that SMARCAL1 and FANCM directly unwind TA-rich DNA cruciforms, preventing catastrophic chromosome breakage by the ERCC1–ERCC4 complex. Our data yield fundamental insights into genome maintenance, provide a springboard for mechanistic investigations into new connections between DDR factors and pinpoint synthetic vulnerabilities that could be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="
 The rapid emergence of resistance to targeted therapies poses a substantial challenge in managing various advanced cancers, including prostate cancer (PCa). The extent to which PCa cells can autonomously manipulate the tumor microenvironment (TME) and swiftly adapt to therapeutic challenges remains inadequately understood. We have elucidated that PCa cells can exploit neurolysin, a crucial regulator of neuronal signaling, to counter therapeutic interventions. The frequent absence of neurolysin results in increased WNT3A secretion into the TME and activates aberrant WNT signaling in adjacent cancer cells, thereby conferring AR targeted therapy resistance. This escalated WNT secretion and subsequent resistance are dependent on dysregulated cellular trafficking governed by the motor protein KIF11, which can be abolished by targeting either WNT3A or KIF11. These findings reveal an uncharted mechanism where cancer cells divert neuronal signaling molecules to redirect cellular trafficking machinery, thereby autonomously altering the TME and promoting resistance.


 Our study reveals an as-yet-undescribed molecular mechanism by which cancer cells can divert key neuronal molecules to regulate cellular trafficking and thereby autonomously modify the TME and promote resistance. It not only filled a crucial knowledge gap by elucidating how cancer cells autonomously modify cellular trafficking and the TME to resist therapy, but also suggest a novel therapeutic approach to address this challenge in contemporary targeted therapies.



 P.M. served as a scientific consultant to Accutar Biotechnology, Inc. T.W. is a scientific co-founder of NightStar Biotechnologies, Inc. H.L. is a shareholder and scientific advisor of Precision Scientific Ltd. All other authors declare that they have no competing interests.



 Ping Mu, Choushi Wang, Xiaoling Li, Su Deng. Prostate cancer exploits neurolysin to modulate the tumor microenvironment and therapy resistance via KIF11-mediated WNT secretion [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6417.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b01f38da37b78292846f1ed57bc5a2eeac89d1d" target='_blank'>
              Abstract 6417: Prostate cancer exploits neurolysin to modulate the tumor microenvironment and therapy resistance via KIF11-mediated WNT secretion
              </a>
            </td>
          <td>
            Ping Mu, Choushi Wang, Xiaoling Li, Su Deng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Up to 10% of all cancers may be caused by inherited genetic changes. Consequently, the increasing prevalence of cancer is driving the growth of the hereditary cancer testing as it is important to implement surveillance and risk-reduction strategies, detect cancer at an early, more treatable stage, and provide tailored treatment and therapies. Panels are often designed to cover hundreds of genes and performing validation and assessing performance requires samples with common mutations in each of those genes. Biosynethic reference materials can address these challenges. Here we describe the development and multisite evaluation of Seraseq® Inherited Cancer DNA Mix v2, which contains genes recommended for testing by the American College of Medical Genetics, the National Comprehensive Cancer Network, and the American Society of Clinical Oncology. Genomic DNA from the GM24385 cell line was blended with 61 biosynthetic DNA variants in 55 different genes including 21 deletions, 16 duplications, 3 insertions, 5 INDELs, 15 SNVs and 1 Inversion. Variant allele frequency (VAF) was targeted to 50% and verified using the Bio-Rad QX-200 Droplet Digital PCR (ddPCR) System. The mix was analyzed with the Axen Hereditary Cancer Panel v2 (Macrogen), the ViennaLab Hereditary Cancer Assay (ViennaLab) on the MiSeq System (Illumina), the Twist Library Preparation and AVITI 2x150 Sequencing Kit Cloudbreak Low Output on the Element Biosciences AVITI (Halo Precision Diagnostics), the GALEAS Hereditary Plus v2 Panel (Nonacus) on the NextSeq 2000 System (Illumina), and the Celemics Library Preparation Kit and Celemics Target Enrichment Kit and the NextSeq 500/550 Kits v2.5 (Illumina) on the NextSeq 500 System (Illumina).All biosynthetic variants were detected within 40-60% by ddPCR. Most NGS assays were not able to detect all variants as many that were included are difficult to detect. Identification of these complex variants required modifications to alignment, variant calling, and annotation settings. For example, RAD51D c.694_715delins is detected as two separate c.694C>T and c.715C>T variants (e.g. at ViennaLab), unless the variant caller is set to merge. At Nonacus, the Boland inversion and BRCA2 and STK11 large deletions were detected by a combination of short-range and long-range paired-end mapping and split-read analysis. An Alu-specific module using repeat masking allowed successful calling of the BRCA2 AluY insertion; both increased detection accuracy from 85.5% to 95.1%.
 The performance of the Seraseq Inherited Cancer DNA Mix v2 across NGS assays, supported by robust digital PCR data, demonstrated that these materials have broad compatibility and will allow for assessment of the ability of variant callers to detect complex variants.


 Dana Ruminski Lowe, Deepika Philkana, Krystyna Nahlik, Melissa Berenger, Andrew Anfora, Yves Konigshofer, Catherine Huang, Russell Garlick, Bharathi Anekella. Interlab study supporting hereditary cancer testing in an NGS-focused landscape [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6199.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d329440f52845fe5e04ed56db29d5167fcf05d" target='_blank'>
              Abstract 6199: Interlab study supporting hereditary cancer testing in an NGS-focused landscape
              </a>
            </td>
          <td>
            Dana J. Ruminski Lowe, Deepika Philkana, K. Nahlik, Melissa Berenger, Andrew Anfora, Yves Konigshofer, Catherine Huang, R. Garlick, B. Anekella
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="

 Intra-tumor heterogeneity (ITH) refers to the presence of diverse cell populations within a single tumor, each with distinct genetic, epigenetic, and phenotypic characteristics. This complexity presents significant challenges in treatment, as different subpopulations may respond variably to therapies, yet the non-genetic component of ITH and its contribution to treatment response remains poorly understood. We mapped transcriptomic and epigenomic ITH to explore evolution of non-genetic ITH in response to therapy in high-grade serous ovarian cancer (HGSOC).



 The SCANDARE study (NCT03017573) included patients with HGSOC that are treated with neo-adjuvant chemotherapy (NAC). Biopsies were performed at baseline, after NAC surgery and at recurrence for single-cell RNA sequencing (scRNA-seq, n=17 samples, 111,946 with 1,773 median gene coverage), single-nuclei RNA sequencing (snRNA-seq, n=35 samples, 151,719 nuclei with 2,843 median gene coverage), and single-cell epigenomics (snCUT&Tag, n=24 samples, 58,892 nuclei with 1,409 unique fragments median coverage). Our snCUT&Tag dataset is one of the first single-cell epigenomic map of cancer patients under treatment, focusing on the histone modification H3K4me1, which accumulates at primed and active enhancers and promoters. We integrated these clinically annotated single-cell/nuclei datasets with publicly available scRNA-seq data of HGSOC.



 We constructed a consensus map of functional tumor states in HGSOC, based on scRNA-seq analysis of over 200,000 tumor cells. This analysis revealed 15 recurrent tumor cell phenotypes, ranging from stressed to cycling or inflammatory states. We identified the transcription factors and cell-cell communications specific to each tumor cell state. These tumor states are encoded at the epigenomic level, we could identify in each tumor epigenetic clones - clusters of cells with the same epigenome - each displaying a H3K4me1 landscape characteristic of one tumor state. These results show that tumor states correspond to tumor populations with distinct non-genetic features. Longitudinal sampling by snRNA-seq further showed that tumor state composition consistently evolves upon chemotherapy exposure in patients: tumors lose their cycling, hypoxic and interferon cells while gaining cells with partial mesenchymal and inflammatory characteristics.



 We propose a consensus map of transcriptomic and epigenomic tumor states in HGSOC. Our work is a proof of concept that we can monitor functional ITH with high resolution snRNA-seq from frozen biopsies of the standard of care. We show that these consensus tumor states consistently evolve upon treatment in all patients, pinpointing tumor cell states that are not successfully targeted by NAC, that would need to be targeted by adjuvant therapies.



 Yuna Landais, Juliette Bertorello, Marta Puerto, Baptiste Simon, Amélie Roehrig, Adeline Durand, Marceau Quatredeniers, Constance Lamy, Fabrice Lecuru, Christophe Le Tourneau, Celine Vallot. Evolution of transcriptomic and epigenomic intra-tumor heterogeneity in high-grade serous ovarian cancer with chemotherapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1252.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daad54b0fcb7c5c704adea069e134ecf7c7a856b" target='_blank'>
              Abstract 1252: Evolution of transcriptomic and epigenomic intra-tumor heterogeneity in high-grade serous ovarian cancer with chemotherapy
              </a>
            </td>
          <td>
            Yuna Landais, Juliette Bertorello, Marta Puerto, Baptiste Simon, Amélie Roehrig, Adeline Durand, Marceau Quatredeniers, C. Lamy, Fabrice Lécuru, Christophe Le Tourneau, Céline Vallot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Glioblastoma (GBM), the most lethal primary brain tumor, remains resistant to current treatment modalities that include surgical resection, chemotherapy, and radiotherapy (RT). A critical challenge in GBM therapy is the presence of severe hypoxic tumor regions caused by the rapid growth of solid tumors and poor vascularization. Hypoxia drives therapeutic resistance by impairing DNA damage induction and repair mechanisms, particularly double-strand breaks (DSBs), which are the most cytotoxic lesions induced by RT and chemotherapeutic agents.Cellular deoxyribonucleoside triphosphate (dNTP) pools, tightly regulated by ribonucleotide reductase (RNR) and SAM- and HD-domain-containing protein (SAMHD1), play a pivotal role in DNA replication and repair. Hypoxia disrupts this balance by reducing dNTP synthesis due to the oxygen dependency of RNR, a rate-limiting enzyme, leading to diminished replication and cell cycle arrest. These metabolic adaptations hinder the efficacy of DSB-inducing agents in hypoxic GBM cells. Notably, rapidly replicating cells are more sensitive to cancer therapeutic agents, partly due to replication-induced conversion of single-strand breaks and other manageable damages into highly lethal DSBs.To address this resistance mechanism, we investigated the role of dNTP modulation in enhancing DNA damage and therapeutic outcomes in hypoxic GBM cells. Our study reveals that pharmacologically increasing dNTP levels in hypoxic GBM cells restored cell cycle progression and DNA replication, thereby sensitizing cells to DSB-inducing therapies. Specifically, we found that:
 1. Elevated dNTP pools reversed hypoxia-induced cell cycle arrest: Flow cytometry analysis demonstrated a marked increase in S-phase entry upon dNTP modulation under hypoxic conditions.
 2. Increased DSB formation following therapy: Single-cell microscopic analysis revealed enhanced DSB induction in hypoxic GBM cells treated with ionizing radiation (IR) or bleomycin after dNTP elevation.
 3. Enhanced apoptotic response: Pharmacological dNTP modulation amplified apoptosis activation following DSB induction, indicating improved therapeutic efficacy.
 This study establishes the critical link between dNTP availability, DNA replication, and hypoxia-induced therapeutic resistance in GBM. By leveraging dNTP modulation, we provide a novel preclinical strategy to overcome hypoxia-mediated resistance to DSB-inducing therapies. Our findings support further exploration of this approach as an adjunctive treatment to improve outcomes in patients with hypoxic GBM.


 Edidiong R. Usoro, Dominique Monroe, Kristen Carver, Emily Steinbeck, Arilyn Williams, Waaqo Daddacha. Enhancing double-strand break induction in hypoxic glioblastoma cells through dNTP modulation and its potential therapeutic implications [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1818.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5a495cc4677d82bf911302787442a5796efaf4" target='_blank'>
              Abstract 1818: Enhancing double-strand break induction in hypoxic glioblastoma cells through dNTP modulation and its potential therapeutic implications
              </a>
            </td>
          <td>
            E. Usoro, Dominique Monroe, Kristen Carver, Emily Steinbeck, Arilyn Williams, Waaqo Daddacha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide. Despite therapeutic advances, there is a critical need to identify novel, effective, and safe drug targets to improve precision treatment strategies.We developed a multi-layered framework integrating Mendelian randomization (MR), colocalization analysis, genome-wide association study (GWAS) data, and expression quantitative trait loci (eQTLs) to prioritize causal and druggable genes in CRC. Single-cell and bulk RNA sequencing were used to characterize gene expression within the tumor microenvironment. Phenome-wide association studies (PheWAS) assessed off-target effects, and drug repurposing potential was evaluated using OpenTargets, DrugBank, and DGIdb. Validation of key targets was performed through RT-qPCR and immunohistochemistry (IHC) in CRC patient samples.Out of 4,479 druggable genes, MR analysis identified 47 candidates significantly associated with CRC risk. Six genes (TFRC, TNFSF14, LAMC1, PLK1, TYMS, and TSSK6) demonstrated strong colocalization signals and were further validated across replication datasets and subtype-stratified analyses. PheWAS analysis revealed minimal off-target effects for these genes. Notably, several of these genes have already been targeted by existing or investigational drugs, suggesting potential for repurposing. These genes exhibited distinct expression patterns in tumor and stromal cell types and were differentially expressed in CRC versus normal tissues. Among them, TNFSF14, an immune modulator, is particularly involved in regulating T cell activation within the tumor microenvironment.This study identifies and validates six promising druggable targets for CRC, providing a strong foundation for future preclinical studies. These findings open avenues for advancing precision oncology and drug repurposing strategies in CRC treatment, contributing to the development of more effective and personalized therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2fdfb0fd7198a877424d14cc7cd5c612aeedd20" target='_blank'>
              Integrative genomic and single-cell framework identifies druggable targets for colorectal cancer precision therapy
              </a>
            </td>
          <td>
            Yanggang Hong, Jiajun Li, Nuo Xu, Wanyi Shu, Feng Chen, Yuze Mi, Haigang Geng, Qian Li
          </td>
          <td>2025-05-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Adenosine deaminases acting on RNA (ADAR) enzymes constitute a natural cellular mechanism that induces A-to-I(G) editing, introducing genetic changes at the RNA level. Recently, interest in the endogenous-ADAR editor has emerged for correcting genetic mutations, consisting of a programmed oligonucleotide that attracts the native ADAR, thereby offering opportunities for medical therapy. Here, we systematically chart the scope of cancer mutations that endogenous-ADAR can correct. First, analyzing germline single nucleotide variants in cancer predisposition genes, we find that endogenous-ADAR can revert a fifth of them, reducing the risk of cancer development later in life. Second, examining somatic mutations across various cancer types, we find that it has the potential to correct at least one driver mutation in over a third of the samples, suggesting a promising future treatment strategy. We also highlight key driver mutations that are amenable to endogenous-ADAR, and are thus of special clinical interest. As using endogenous-ADAR entails delivering relatively small payloads, the prospects of delivering endogenous-ADAR to various cancers seem promising. We expect that the large scope of correctable mutations that are systematically charted here for the first time will pave the way for a new era of cancer treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a024daf32faffd9eaf4adfd22fdca5d40c18746f" target='_blank'>
              A systematic evaluation of the therapeutic potential of endogenous-ADAR editors in cancer prevention and treatment
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Ariel Dadush, Sumeet Patiyal, P. Rajagopal, Gulzar Daya, Shay Ben-Aroya, A. Schäffer, Eli Eisenberg, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-02</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="


 Chromosomal instability (CIN) is the predominant form of genomic instability in cancers, characterized by both numerical and structural alterations of chromosomes. CIN is crucial for the most threatening aspect of cancers: their ability to evolve, allowing them to circumvent immune detection, metastasize, develop resistance to therapies, and lead to treatment failure. At the same time, the CIN phenotype could provide valuable targets for cancer therapy. We developed a multiplex FISH (miFISH) method to study heterogeneity and ongoing CIN in cancer cells. The objective of our research was to determine whether high-grade aneuploidy develops through sequential or abrupt changes in chromosomal content, a question previously unexplored due to the absence of comprehensive methods for simultaneously detecting different forms of ongoing CIN on a single-cell level.



 Sequential hybridizations of miFISH panels allow the assessment of up to 35 probes per single cell. Each panel was designed to include the centromeric probe and gene-specific probes on the p- and q-arm for a given chromosome. This approach was applied to colorectal cancer cell lines (with and without microsatellite instability) and colorectal cancer samples. A total of 10000-25000 targets were collected from each slide. Analysis was done by counting of signals in mononucleated cells, binucleated/dividing cells and metaphase spreads.



 The simultaneous evaluation of centromeric and gene-specific sets of miFISH probes led to unprecedented insights into the patterns of distribution of chromosomal content among individual cells, and, especially, in dividing cells. We compared different forms of ongoing CIN: structural rearrangements, formation of micronuclei with whole chromosomes or chromosomal fragments, non-disjunctions of chromosomes, grossly asymmetrical cell divisions, including divisions with ploidy changes and multipolar mitoses. We observed that gradual changes in chromosomal/genomic content occur primarily due to frequent structural rearrangements, less frequent chromosome or fragment losses in micronuclei, and very rare single chromosome non-disjunctions. Of note, grossly abnormal cell divisions, characterized by massive non-disjunctions and structural rearrangements, occur with relatively high frequency. The same patterns of chromosomal heterogeneity and instability have been observed in cancer cell lines and in tumor samples.



 The novel miFISH approach applied to studies of chromosomal instability in colorectal cancers and cell lines revealed multiple paths leading to chromosomal diversification of cancer cell populations. Grossly abnormal cell divisions are likely to contribute to high-level aneuploidies and ploidy changes, and could underlie the rapid evolution of cancer under selective pressure.



 Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, Kerstin Heselmeyer-Haddad, Paul Meltzer. Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3890.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Forward genetic screens have revolutionized the field of target discovery, target deconvolution, and target validation, particularly in the oncology field. However, most approaches are limited to the interrogation of non-essential factors and identify gene networks rather than direct target structures. Therefore, the identification of druggable target structures and understanding an active molecules target space remains a challenge in drug discovery and development. To address this, we established a chemical mutagenesis approach in relevant cell types that allows entirely unbiased identification of small molecule targets at amino acid resolution, literally mapping compound-target interaction surfaces. Chemical mutagenesis randomly introduces single nucleotide variants into the entire genome resulting in cell populations which carry substitutions in statistically more than 90% of all amino acids. Challenging such mutagenized cell populations with a drug candidate of interest, followed by next generation sequencing and analysis, reveals direct drug target interactions, drug resistance mechanism, and potential off-targets or concealed efficacies. The alteration of single amino acids thereby allows interrogation of non-essential as well as most essential factors and increases screening resolution to the amino acid level. Several case studies will be shown.


 Moritz Horn, Georg Michlits, Andreia Rocha, Christoph Stelzer, Josef M. Penninger. High resolution target identification by unbiased forward genetics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5547.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aba9f5d2cf8bd1341acacd8ce88421b1a228c70" target='_blank'>
              Abstract 5547: High resolution target identification by unbiased forward genetics
              </a>
            </td>
          <td>
            Moritz Horn, Georg Michlits, Andreia Rocha, Christoph Stelzer, Josef M. Penninger
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PC) is the most commonly diagnosed cancer in men worldwide. While androgen deprivation therapy (ADT) is initially effective, many patients develop resistance, progressing to castration-resistant prostate cancer (CRPC). Recent studies have identified the interaction between PRMT5 (protein arginine methyltransferase 5) and pICLn as a promising therapeutic target, as it promotes the transcription of double-strand break (DSB) repair genes that contribute to therapy resistance. To target this pathway, a screening campaign identified J021-0199 as a potential hit compound that disrupts the PRMT5/pICLn interaction. Biochemical assays demonstrated that J021-0199 binds to the N-terminal TIM barrel domain of PRMT5. In CRPC cell lines (LNCaP and 22Rv1), J021-0199 selectively inhibited cancer cell growth. qPCR analysis further revealed downregulation of DNA damage response (DDR) genes involved in homologous recombination, nonhomologous end joining, and G2 arrest. These results support J021-0199 as a promising lead compound for overcoming resistance in CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a497a66cc78c68ab5bd249fd907b3924354d1f68" target='_blank'>
              Virtual High-Throughput Screening of Ligands for Disrupting PRMT5/pICLn Interaction in Prostate Cancer Cells.
              </a>
            </td>
          <td>
            Zhihang Shen, Gustavo Seabra, Chenglong Li
          </td>
          <td>2025-04-24</td>
          <td>ACS medicinal chemistry letters</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335901b6f2e363bcd4c769050615cefdadb15cb6" target='_blank'>
              Mechanistic Insights into Proteomic Mutation-Phenotype Linkages from Tiling Mutagenesis Screens
              </a>
            </td>
          <td>
            Wei He, Jen-Wei Huang, Yalong Wang, Samuel B. Hayward, G. Leuzzi, Rongjie Fu, Shuyue Wang, Alina Vaitsiankova, Mark T. Bedford, Raphaël Guérois, Alberto Ciccia, Han Xu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Radiotherapy is a cornerstone of colorectal cancer (CRC) treatment; however, its therapeutic efficacy is often compromised by both intrinsic and acquired resistance in CRC cells. This study employed small interfering RNA (siRNA) technology to elucidate the functional roles of BAMBI, GADD34, NFKBIA, and NFKBID in CRC cell lines SW480 and HCT116. We assessed their impact on key cellular processes and radiation sensitivity. Gene silencing of all four target genes significantly suppressed CRC cell proliferation, migration, and invasion. Moreover, siRNA-mediated knockdown enhanced radiation sensitivity, as evidenced by a substantial increase in apoptosis and a marked reduction in cell viability compared with controls. These findings suggest that BAMBI, GADD34, NFKBIA, and NFKBID serve as critical regulators of CRC progression and radiation resistance. Overall, this study provides a mechanistic foundation for further exploration into the pathways underlying radiation resistance and underscores the potential for developing personalized radiotherapy strategies guided by molecular profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b02786eb68733b852aeed62e7695e4393e02cbb" target='_blank'>
              Mechanistic Insights into Radiation Resistance in Colorectal Cancer: Gene Exploration Study
              </a>
            </td>
          <td>
            Beamjun Park, Soohyeon Lee, Inyoung Jo, Donghyun Kang, Taewan Kim, Jaesung Ryu, Hyejeong Kong, Moojun Baek, Taesung Ahn
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Transposons are DNA sequences able to move or copy themselves to other genomic locations leading to insertional mutagenesis. Although transposon-derived sequences account for half of the human genome, most elements are no longer transposition competent. Moreover, transposons are normally repressed through epigenetic silencing in healthy adult tissues but become derepressed in several human cancers, with high activity detected in colorectal cancer. Their impact on non-malignant and malignant tissue as well as the differences between somatic and germline retrotransposition remain poorly understood. With new sequencing technologies, including long read sequencing, we can access intricacies of retrotransposition, such as insertion sequence details and nested repeats, that have been previously challenging to characterize. Results In this study, we investigate somatic and germline retrotransposition by analyzing long read sequencing from 56 colorectal cancers and 112 uterine leiomyomas. We identified 1495 somatic insertions in colorectal samples, while striking lack of insertions was detected in uterine leiomyomas. Our findings highlight differences between somatic and germline events, such as transposon type distribution, insertion length, and target site preference. Leveraging long-read sequencing, we provide an in-depth analysis of the twin-priming phenomenon, detecting it across transposable element types that remain active in humans, including Alus. Additionally, we detect an abundance of germline transposons in repetitive DNA, along with a relationship between replication timing and insertion target site. Conclusions Our study reveals a stark contrast in somatic transposon activity between colorectal cancers and uterine leiomyomas, and highlights differences between somatic and germline transposition. This suggests potentially different conditions in malignant and non-malignant tissues, as well as in germline and somatic tissues, which could be involved in the transposition process. Long-read sequencing provided important insights into transposon behavior, allowing detailed examination of structural features such as twin priming and nested elements. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00357-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6e6a5d11920586374ddcd1ad544515162f4af5" target='_blank'>
              Structural features of somatic and germline retrotransposition events in humans
              </a>
            </td>
          <td>
            Päivi Nummi, Tatiana Cajuso, T. Norri, Aurora Taira, Heli Kuisma, Niko Välimäki, A. Lepistö, L. Renkonen-Sinisalo, S. Koskensalo, T. Seppälä, Ari Ristimäki, Kyösti Tahkola, Anne Mattila, J. Böhm, J. Mecklin, Emma Siili, A. Pasanen, O. Heikinheimo, Ralf Bützow, A. Karhu, K. Burns, Kimmo Palin, Lauri A Aaltonen
          </td>
          <td>2025-04-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Genomic instability has been increasingly recognized over the past decade as a fundamental driver of cancer initiation and progression, largely owing to its association with specific genes and cellular mechanisms that offer therapeutic potential. However, a comprehensive molecular framework that captures the interconnected processes underlying this phenomenon remains elusive. In this study, we focused on polo-like kinase 1 (PLK1), a key cell cycle regulator frequently overexpressed in diverse human tumors, to reconstruct a regulatory network that consolidates pre-existing biological knowledge exclusively related to pathways involved in genome stability maintenance and cancer. The resulting model integrates nine biological processes, 1030 reactions, and 716 molecular species to form a literature-supported network in which PLK1 serves as a central regulatory node. However, rather than depicting an isolated PLK1-centric system, this network reflects a broader and more complex architecture of interrelated genomic instability mechanisms. As expected, the simulations reproduced known behaviors associated with PLK1 dysregulation, reinforcing the well-established role of the kinase in genome destabilization. Importantly, this model also enables the exploration of additional, less-characterized dynamics, including the potential involvement of genes such as kif2c, incenp, and other regulators of chromosomal segregation and DNA repair, which appear to contribute to instability events downstream of PLK1. While these findings are grounded in mechanistic simulations and require further experimental validation, gene expression and survival analyses across tumor types support their clinical relevance by linking them to poor prognosis in specific cancers. Overall, the model provides a systemic and adaptable foundation for studying PLK1-related genomic instability, enabling both the reinforcement of known mechanisms and discovery of candidate genes and circuits that may drive tumorigenesis through compromised genome integrity across distinct cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 In the present study we illustrate how, using the concept of synthetic lethality (SL), we developed cellular assays suitable for the screening of new drugs targeting cancer-specific weaknesses. SL is a promising strategy in cancer therapy that exploits the genetic vulnerabilities of cancer cells. This approach targets pairs of genes where the mutation or changes of one gene does not affect cell viability, but the simultaneous occurrence of these events in both leads to cell death. This concept is particularly valuable in cancer treatment, as it allows for the selective targeting of cancer cells while sparing normal cells. In this study we coupled powerful molecular approaches such as CRISPR/Cas9 or RNA interference (siRNA and shRNA) with cell viability tests (CellTiter-Glo®), to generate cellular assays suitable for assessing SL of two different genes of interest (GOIs) in proof-of-concept studies and perform drug screening to identify novel inhibitors able to mimic loss or mutations of the GOIs, inducing specific cell death of tumor cells. First, we induced SL by silencing VPS4A in cells already exhibiting loss or reduced function of VPS4B, ultimately triggering selective tumor cell death while sparing normal cells. Indeed, the homozygous loss of VPS4B is seen in up to 4% of colorectal, ovarian, and pancreatic cancers. The SL was then assessed by using siRNA to silence VPS4A and VPS4B in various cancer cell lines expressing different levels of the 2 GOIs and evaluating cell viability. Conversely, to generate stable cellular assays, CRISPR/Cas9 technology was applied to develop both VPS4B knockout (KO) and heterozygous (HET) cell lines, validated through Sanger sequencing and protein expression analysis. These genetically modified lines were critical for understanding the molecular mechanisms driving SL, as they were further engineered to obtain an inducible VPS4A silencing (shRNA technology), allowing for controlled VPS4A depletion. These experiments demonstrated that VPS4A silencing in VPS4B KO cells significantly reduced cell viability, confirming the synthetic lethal relationship between the two GOIs. In conclusion, our survival assay approach could be applied to validate the SL between 2 GOIs, identify a suitable genetic context for applying customized therapeutic options, and new drugs to treat cancer. Proof of principle studies could be conducted in transient using siRNA to silence GOIs in various cancer cell lines expressing different levels of GOIs; while stable modulation of the expression of GOI might be obtained using ad hoc engineered cell lines, thus allowing long-term studies and screening campaigns. Cellular assays able to elucidate SL mechanisms may have a significant impact on the cancer drug discovery identifying therapies that specifically target cancer cells with known genetic alterations, while minimizing the impact on normal cells and therefore reducing potential side effects.


 Giovanna Maresca, Fiorella Scagnoli, Francesca Sassone, Alessandro Poli, Paola Bonetti, Laura Curti, Silvia Goldoni, Helena Batoulis, Ekaterina Nevedomskaya, Jean-Francois Rolland, Elisabeth Pook, Loredana Redaelli, Tom Grammatopoulos. Novel synthetic lethality-based cellular assays for cancer drug discovery [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3157.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6658643c3cff50e2f74694ebc1a1ac3759cf2352" target='_blank'>
              Abstract 3157: Novel synthetic lethality-based cellular assays for cancer drug discovery
              </a>
            </td>
          <td>
            Giovanna Maresca, Fiorella Scagnoli, Francesca Sassone, Alessandro Poli, Paola Bonetti, Laura Curti, Silvia Goldoni, Helena Batoulis, Ekaterina Nevedomskaya, Jean-Francois Rolland, Elisabeth Pook, L. Redaelli, Tom Grammatopoulos
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1225ffe2d0c6c775628165cdb9bef1963448726" target='_blank'>
              A First-in-Class Therapy Targeting Transcription-Replication Conflicts Drives Synthetic Lethality in Homologous Recombination Deficient Cancers
              </a>
            </td>
          <td>
            Yi Chieh Lim, Soak-Kuan Lai, Ru Han Yang, Yong Kang Chua, Rémi Patouret, Vaishnervi Manoj, Colin Chun Xuan Soh, Benjamin Lebeau, Ning Cham, Wei Meng, Sze Keong Tey, Szu Ling Yeap, Songjing Zhang, Jessie Yong Xing Woon, Zahin Ali, Marek Mutwil, Pritpal Singh, M. Fullwood, Linda LD Zhong, Xue Ming Dong, Chiba Shunsuke, Cheng-Gee Koh, H. Li
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Resistance to radiation therapy is a common challenge in the field of oncology. Cancer cells with an increased ability to effectively repair DNA or cells with higher levels of antioxidants are more resistant to radiation. As cancer cells rely on these traits for survival, they may offer vulnerabilities that could be exploited. Methods In the current study, rectal cancer organoids that showed different responses to radiation treatment were identified. RNA sequencing was used to compare radioresistant and radiosensitive organoids. In vitro combination drug screens were performed. The selection of drugs was guided by the RNA sequencing results. Results Radioresistant organoids exhibited superior transcriptional adaptability and activated more DNA repair pathways when irradiated. Additionally, radioresistant organoids displayed enhanced antioxidant metabolism, including pathways related to the detoxification of reactive oxygen species and the synthesis of glutathione. Combinatorial drug screens identified the combination of RRx-001 (an inducer of oxidative stress) with GCLC inhibitor BSO as a highly effective and synergistic drug combination in killing radioresistant organoids. CRISPR-CAS-mediated knockout of GCLC sensitised organoids to RRx-001. Conclusion Combining RRx-001 with the inhibition of GCLC may be a promising alternative treatment strategy in radioresistant rectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc4a68bb9eb32114e3b611e03801c7a052fee55" target='_blank'>
              Rational design of alternative treatment options for radioresistant rectal cancer using patient-derived organoids
              </a>
            </td>
          <td>
            D. Andel, A. J. Nouwens, S. Klaassen, J. Laoukili, B. Viergever, A. Verheem, M. P. W. Intven, M. Zandvliet, J. Hagendoorn, I. Borel Rinkes, O. Kranenburg
          </td>
          <td>2025-04-10</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Hematologic malignancies (HMs) encompass a diverse spectrum of cancers originating from the blood, bone marrow, and lymphatic systems, with myeloid malignancies representing a significant and complex subset. This review provides a focused analysis of their classification, prevalence, and incidence, highlighting the persistent challenges posed by their intricate genetic and epigenetic landscapes in clinical diagnostics and therapeutics. The genetic basis of myeloid malignancies, including chromosomal translocations, somatic mutations, and copy number variations, is examined in detail, alongside epigenetic modifications with a specific emphasis on DNA methylation. We explore the dynamic interplay between genetic and epigenetic factors, demonstrating how these mechanisms collectively shape disease progression, therapeutic resistance, and clinical outcomes. Advances in diagnostic modalities, particularly those integrating epigenomic insights, are revolutionizing the precision diagnosis of HMs. Key approaches such as nano-based contrast agents, optical imaging, flow cytometry, circulating tumor DNA analysis, and somatic mutation testing are discussed, with particular attention to the transformative role of machine learning in epigenetic data analysis. DNA methylation episignatures have emerged as a pivotal tool, enabling the development of highly sensitive and specific diagnostic and prognostic assays that are now being adopted in clinical practice. We also review the impact of computational advancements and data integration in refining diagnostic and therapeutic strategies. By combining genomic and epigenomic profiling techniques, these innovations are accelerating biomarker discovery and clinical translation, with applications in precision oncology becoming increasingly evident. Comprehensive genomic datasets, coupled with artificial intelligence, are driving actionable insights into the biology of myeloid malignancies and facilitating the optimization of patient management strategies. Finally, this review emphasizes the translational potential of these advancements, focusing on their tangible benefits for patient care and outcomes. By synthesizing current knowledge and recent innovations, we underscore the critical role of precision medicine and epigenomic research in transforming the diagnosis and treatment of myeloid malignancies, setting the stage for ongoing advancements and broader clinical implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1633c8ceb2748c3ad6ca551f7099ac2f4cee0994" target='_blank'>
              Epigenomic insights and computational advances in hematologic malignancies
              </a>
            </td>
          <td>
            Carolyn Lauzon-Young, Ananília Silva, B. Sadikovic
          </td>
          <td>2025-04-12</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Lineage plasticity has been widely recognized in various cancers as a driver of therapeutic resistance, but the molecular basis of this plasticity remains incompletely defined. Evidence from the literature reveals that the inactivation of RB1 and TP53 in prostate cancer (PC) results in the emergence of alternative lineage states that are resistant to androgen receptor (AR)-targeted treatment. How RB1 and TP53 loss cause this lineage plasticity is not clear. This study proposes that RB1 and TP53 help maintain heterochromatin in order to restrict chromatin accessibility and maintain the fidelity of lineage specific gene expression. To test this hypothesis, changes in chromatin accessibility (ATAC-seq) and lineage gene expression (RNA-seq) are monitored in single prostate organoid cells over time following deletion of PTEN (SKO) or PTEN/RB1/TP53 (TKO). Analysis of resulting data indicates that TKO organoids have increased transcriptional diversity and elevated expression pathways like JAK-STAT that have been functionally linked to lineage plasticity and resistance to AR-targeted therapies. These gene expression changes are preceded by increased chromatin accessibility at relevant gene promoters. Enrichment of neuroendocrine PC gene signatures is found in both SKO and TKO organoids likely driven by histone methyltransferase EZH2 as well as DNA methyltransferase DNMT1. TKO organoids also exhibit increased ability to adapt to changing growth conditions, likely facilitated by their increased transcriptional plasticity. Future directions for this study will assess the influence of EZH2 and DNMT1 on chromatin accessibility, gene expression, lineage plasticity, and therapeutic resistance following tumor suppressor deletion. Overall, results are consistent with the hypothesis that RB1 and TP53 loss in PC cells creates a more accessible chromatin state permissive for transcription. Findings may also reveal approaches to suppress therapeutic resistance in PC by therapeutic targeting of lineage plasticity.


 Mauricio Rodrigo Flores, David W. Goodrich, Justine Jacobi. RB1 and TP53 restrict lineage plasticity in prostate cancer by regulating chromatin accessibility [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6713.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc24f6cab28b67012c2dbe001093787b08c40ec" target='_blank'>
              Abstract 6713: RB1 and TP53 restrict lineage plasticity in prostate cancer by regulating chromatin accessibility
              </a>
            </td>
          <td>
            Mauricio Rodrigo Flores, David W. Goodrich, Justine J Jacobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Advances in cancer biology have been achieved by the identification of oncogenes and tumor suppressor genes through the remarkable progression of next-generation sequencing. New techniques, such as single-cell analysis, help uncover cancer progression and heterogeneity. Reverse genetic screenings, including methods like random mutagenesis via retroviruses, transposons, RNA interference, and CRISPR, are useful for exploring gene functions and their roles in cancer. Especially in random mutagenesis, CRISPR screening and its modifications have recently emerged as powerful tools due to their comprehensiveness and simplicity in inducing genetic mutations. Initially, CRISPR screening focused on analyzing biological phenotypes in a cell population. It has since evolved to incorporate advanced techniques, such as combining single-cell and spatial analyses. These developments enable the investigation of cell-cell and spatial interactions, which more closely mimic In Vivo microenvironments, offering deeper insights into complex biological processes. These approaches allow for the identification of essential genes involved in cancer survival, drug resistance, and tumorigenesis. Together, these technologies are advancing cancer research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Radiotherapy (RT) is an integral modality for the treatment of cancer, with >70% patients requiring RT. However, tumor responses to RT are heterogeneous and influenced by several intrinsic and extrinsic factors. Clonal dynamics have identified that in some cancers, radioresistance is dominated by a process of clonal selection and adaptation, thereby suggesting that tumor cells harbor an adaptive pro-survival mechanism to RT. To understand the mechanisms underpinning this process, we generated four radioresistant (RR) cancer models - prostate (22Rv1, DU145) and head and neck (FaDu, HK1) cancer cell lines by exposing them to 90 Gy over 45 fractions delivered over 12 weeks, which mimicked the fractionated radiotherapy schema employed in the clinic. First, WES revealed an enrichment of mismatch repair mutational signatures (SBS6, 14, 15, 20) across our RR models and identified several non-synonymous single nucleotide variants in RAD51AP1, SMC1A, and APC2 genes that are associated with pro-survival pathways. At the transcriptome level, we observed several up- and down-regulated genes across the 4 models, but were able to identify five common overexpressed genes - ATP8A1, BAHD1, CTSD, JAK1, and MYLIP. Over-representation pathway analyses highlighted the association of the BAHD1 gene with an activated heterochromatin formation pathway (rich factor=0.01, adjusted P-value=0.02). This corroborated WB and IF analyses showing an increased expression of heterochromatin markers (H3K9me3 and H3K27me3) in the RR cells post-irradiation, but not in their respective wild-type (WT) parentals. Knockdown of BAHD1 by siBAHD1 treatment consequently reduced the expression of H3K9me3 and H3K27me3 and reversed the RR phenotypes (Clonogeic SF2Gy siBAHD1:control=1.72 [FaDu-RR], SF4Gy siBAHD1:control=1.63 [22Rv1-RR]). Collectively, these findings point to the model that BAHD1-dependent heterochromatin formation is an acquired phenomenon integral to radioresistance in our models. Simultaneously, parallel high-throughput drug screening of our RR and WT cancer cells identified susceptibility to Panobinostat (Pano) - a pan-HDAC inhibitor in vitro and validated in vivo. Combinatorial Pano and AZD7762 - a Chk1 inhibitor exhibited stronger anti-tumor activity against the RR than WT cells (IC50: 0.45 nm [RR] vs 0.60 nm [WT]). Downstream analyses of Pano-treated RR compared with WT tumors further indicated higher apoptosis and reduced proliferation in the former. Finally, knockdown of class I and II HDACs pointed to reversal of HDAC6 expression as the primary target of Pano. Herein, our systems approach of investigations of our RR cancer models uncovered the importance of the heterochromatin in acquired cancer radioresistance, and presented a potential new molecular vulnerability of RR cancer cells to targeting by epigenetic drugs.


 Chaw Yee Beh, Sharon Pei Yi Chan, Celestia Pei Xuan Yeo, Boon Hao Hong, Evelyn Mui Cheng Tan, Dewi Susanti, Pei Ling Tai, Eugenia Li Ling Yeo, Khee Chee Soo, Edward Kai-Hua Chow, Melvin Lee Kiang Chua. System biological approach identifies heterochromatin modification as an important mechanism of acquired cancer radioresistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1338.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9a2666f037bf98ff78850cacfb5af10e4de653" target='_blank'>
              Abstract 1338: System biological approach identifies heterochromatin modification as an important mechanism of acquired cancer radioresistance
              </a>
            </td>
          <td>
            Chaw Yee Beh, Sharon Pei Yi Chan, Celestia Pei Xuan Yeo, Boon Hao Hong, Evelyn Mui Cheng Tan, Dewi Susanti, Pei Ling Tai, Eugenia Li Ling Yeo, Khee Chee Soo, Edward Kai-Hua Chow, Melvin Lee Kiang Chua
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Colorectal cancer (CRC), the third most common cancer worldwide, is one of the deadliest cancers. CRC is known as a cold tumor, characterized by a low immune response that makes it difficult for immune cells to infiltrate and exhibits strong resistance to immunotherapy with checkpoint inhibition. This restricted response is largely attributed to signature gene mutations including mismatch repair (MMR) genes, KRAS, BRAF, APC, and TP53, which are also the main oncogenes in CRC. Mutated signature genes continuously upregulate abnormal signaling pathways, leading to excessive proliferation, cancer progression, and metastasis. Furthermore, it reorganizes the tumor microenvironment (TME) by recruiting immunosuppressive cells. However, the mutation can produce neoantigens that can provoke an immune response, making it a potential target for immunotherapy. In particular, cancer vaccines that leverage the strong neoantigenic properties of these mutations are considered promising for overcoming immune resistance and eliciting anti-tumor responses. In this review, we will describe signature gene mutations in CRC and focus on cancer vaccines targeting these mutations as potential therapies for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24e80e0a03a79befa516d29031109ab14c8ea3b4" target='_blank'>
              Signature Gene Mutations in Colorectal Cancer: Potential Neoantigens for Cancer Vaccines
              </a>
            </td>
          <td>
            Jaegoo Yoon, Haeun Moon, Yuna Jeon, Soo-Cheon Choe, Hyunho Yoon
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3847b0cc02af732174f2aba89f84c63f9e5ea11" target='_blank'>
              Proof-of-Concept of A DNA-Based Recording System for High-Throughput Functional Gene Screening
              </a>
            </td>
          <td>
            Shoichi Kato, Atsushi Ikemoto, Jun Isayama, Tetsuya Takimoto, Hideyuki Saya, Ken-ichi Hamada
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a characteristic feature of esophageal adenocarcinoma (EAC), drives tumor aggressiveness and therapy resistance, presenting an intractable problem in cancer treatment. CIN leads to constitutive stimulation of the innate immune cGAS–STING pathway, which has been typically linked to anti-tumor immunity. However, despite the high CIN burden in EAC, the cGAS– STING pathway remains largely intact. To address this paradox, we developed novel esophageal cancer models, including a CIN-isogenic model, discovering myeloid-attracting chemokines – with the chemokine CXCL8 (IL-8) as a prominent hit – as conserved CIN-driven targets in EAC. Using high-resolution multiplexed immunofluorescence microscopy, we quantified the extent of ongoing cGAS-activating CIN in human EAC tumors by measuring cGAS-positive micronuclei in tumor cells, validated by orthogonal whole-genome sequencing-based CIN metrics. By coupling in situ CIN assessment with single-nucleus RNA sequencing and multiplex immunophenotypic profiling, we found tumor cell-intrinsic innate immune activation and intratumoral myeloid cell inflammation as phenotypic consequences of CIN in EAC. Additionally, we identified increased tumor cell-intrinsic CXCL8 expression in CINhigh EAC, accounting for the inflammatory tumor microenvironment. Using a novel signature of CIN, termed CINMN, which captures ongoing CIN-associated gene expression, we confirm poor patient outcomes in CINhigh tumors with signs of aberrantly rewired cGAS–STING pathway signaling. Together, our findings help explain the counterintuitive maintenance and expression of cGAS–STING pathway components in aggressive, CINhigh tumors and emphasize the need to understand the contribution of CIN to the shaping of a pro-tumor immune landscape. Therapeutic strategies aimed at disrupting the cGAS-driven inflammation axis may be instrumental in improving patient outcomes in this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dfd8088150f447fa336f3b6ae36838000571864" target='_blank'>
              Chromosomal instability shapes the tumor microenvironment of esophageal adenocarcinoma via a cGAS–chemokine–myeloid axis
              </a>
            </td>
          <td>
            Bruno Beernaert, Rose L Jady-Clark, Parin Shah, Erik Ramon‐Gil, Nora M Lawson, Zack D. Brodtman, Somnath Tagore, Frederik Stihler, Alfie S Carter, Shannique Clarke, Tong Liu, Winston Zhu, Erkin Erdal, Alistair Easton, Leticia Campo, Molly Browne, Stephen Ash, Nicola Waddell, Thomas Crosby, Simon R. Lord, Derek A Mann, Ignacio Melero, Carlos E. De Andrea, Andrea E. Tijhuis, F. Foijer, Ester M. Hammond, Kadir C Akdemir, J. Leslie, Benjamin Izar, Eileen E Parkes
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Therapy resistance is one of the most common underlying causes of poor prognosis in cancer patients, ultimately driving fatal outcome. We have previously demonstrated that following chemotherapeutic stress, a subset of cancer cells undergo a mitotic skip and enter an endocycle, causing whole-genome duplication without division. This endocycling cell state is resistant to cytotoxic therapies and thus is an actuator of therapy resistance. Cells in the endocycling state eventually undergo depolyploidization and repopulate the tumor, observed clinically as a recurrence. A common feature of cancers is centrosome amplification (CA) that includes aberrations in centrosome number (>2 centrosomes/cell), size, shape, and/or positioning. Our preliminary data has shown that endocycling cancer cells exhibit CA following treatment across multiple types of chemotherapies. Based upon this data, we hypothesize that studying centrosome dynamics and abnormalities presents a window into better understanding endocycle biology and this novel mechanism of therapy resistance.We have found that the population of prostate cancer and breast cancer cells possessing CA following treatment with LD50 dose of cisplatin or docetaxel increased as a function of recovery time to over 95% by day 10, with nearly all cells clustering extra centrosomes together. Of note, the number of mitotic skips an endocycling cell undergoes has not been found to be associated with the number of centrosomes the cell has. PLK4 is a kinase essential for centriole duplication and its overexpression causes CA. Immunofluorescence analysis has revealed that endocycling cancer cells exhibit an increase in PLK4 protein expression. Endocycling cancer cells also exhibit increased protein expression of KIFC1, a mediator of centrosome clustering. Both PLK4 and KIFC1 protein expression have been found to positively correlate with DNA content. We hypothesize that endocycling cells may use clustered, supernumerary centrosomes as platforms to locally concentrate signaling molecules, lowering the activation threshold for downstream effectors. Published data from bioID proteomics of centrosomal proteins has revealed an enormous unexplored protein landscape around these organelles. Because of this, CA may be a critical mechanism for endocycling cancer cells to survive and maintain this resistant cell state.


 Madison M. Purkerson, Sarah R. Amend, Kenneth J. Pienta. Cancer cells may utilize centrosome amplification to promote survival as a response to chemotherapeutic stress [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 375.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fbe089f3b9ae2dd8bdd29ef8533760180387d9e" target='_blank'>
              Abstract 375: Cancer cells may utilize centrosome amplification to promote survival as a response to chemotherapeutic stress
              </a>
            </td>
          <td>
            Madison M. Purkerson, Sarah R Amend, Kenneth J. Pienta
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Advancements in pharmacogenomics, artificial intelligence (AI), and CRISPR gene-editing technology are revolutionizing precision medicine by enabling highly individualized therapeutic strategies. Artificial intelligence-driven computational techniques improve biomarker discovery and drug optimization while pharmacogenomics helps to identify genetic polymorphisms affecting medicine metabolism, efficacy, and toxicity. Genetically editing based on CRISPR presents a precise method for changing gene expression and repairing damaging mutations. This review explores the convergence of these three fields to enhance improved precision medicine. Method: A methodical study of the current literature was performed on the effects of pharmacogenomics on drug response variability, artificial intelligence, and CRISPR in predictive modeling and gene-editing applications. Results: Driven by artificial intelligence, pharmacogenomics allows clinicians to classify patients and select the appropriate medications depending on their DNA profiles. This reduces the side effect risk and increases the therapeutic efficacy. Precision genetic modifications made feasible by CRISPR technology improve therapy outcomes in oncology, metabolic illnesses, neurological diseases, and other fields. The integration of artificial intelligence streamlines genome-editing applications, lowers off-target effects, and increases CRISPR specificity. Notwithstanding these advances, issues including computational biases, moral dilemmas, and legal constraints still arise. Conclusions: The synergy of artificial intelligence, pharmacogenomics, and CRISPR alters precision medicine by letting customized therapeutic interventions. Clinically translating, however, hinges on resolving data privacy concerns, assuring equitable access, and strengthening legal systems. Future research should focus on refining CRISPR gene-editing technologies, enhancing AI-driven pharmacogenomics, and developing moral guidelines for applying these tools in individualized medicine going forward.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8236a26eed620c341dd9cb62cad5fed494f016e7" target='_blank'>
              Transforming Pharmacogenomics and CRISPR Gene Editing with the Power of Artificial Intelligence for Precision Medicine
              </a>
            </td>
          <td>
            Amit Kumar Srivastav, M. Mishra, J. Lillard, Rajesh Singh
          </td>
          <td>2025-04-24</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Gastrointestinal adenocarcinoma is a major cancer type for the digestive system, ranking as the top cause of cancer-related deaths worldwide. While there has been extensive research on mutations in protein-coding regions, the knowledge of the landscape of its non-coding regulatory elements is still insufficient. Combining the analysis of active enhancer profiles and genomic structural variation, we discovered and validated a lineage-specific super-enhancer for MYB in gastrointestinal adenocarcinoma. This super-enhancer is composed of a predominant enhancer e4 and several additional enhancers, whose transcriptional activity is regulated by the direct binding of HNF4A and MYB itself. Suppression of the super-enhancer downregulated the expression of MYB, inhibited downstream Notch signaling and prevented the development of gastrointestinal adenocarcinoma both in vitro and in vivo. Our study uncovers a mechanism driven by non-coding variations that regulate MYB expression in a lineage-specific manner, offering new insights into the carcinogenic mechanism and potential therapeutic strategies for gastrointestinal adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f994b853e5612dbe78f049706800d9c2a5981d" target='_blank'>
              Systemic genome-epigenome analysis captures a lineage-specific super-enhancer for MYB in gastrointestinal adenocarcinoma
              </a>
            </td>
          <td>
            Fuyuan Li, Shangzi Wang, Lian Chen, Ning Jiang, Xingdong Chen, Jin Li
          </td>
          <td>2025-04-15</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cyclophosphamide (CTX) is a widely used chemotherapeutic agent in breast cancer (BC) treatment regimens. Despite its frequent use, patient responses vary significantly. While several studies have identified clinically relevant genetic variants associated with CTX metabolism to phosphoramide mustard (PM)—the active cellular anti-tumor metabolite—the factors influencing sensitivity at the cellular level remain poorly characterized. We previously performed a genome-wide association study (GWAS) of lymphoblastoid cell lines identified the SNP rs12408401 as associated with PM resistance. Employing the 15-state chromatin prediction model, ChromHMM, and other in silico tools, we identified an association between this SNP and the disruption of a repressive CTCF binding domain in multiple tissue types, including breast. This leads to an upregulation of several nearby genes regulated by this motif, including RFX5. Analysis of TCGA BC patient data revealed that increased RFX5 expression correlates with reduced disease-free interval in breast cancer patients treated with CTX (HR = 5.32, P = 0.028), but not in other BC patients. RFX5 is a transcription factor that typically regulates the expression of MHC II-related genes. However, RFX5 has been reported to exhibit upregulation in several malignancies and off-target transcriptional activity. Herein, we aim to functionally validate the role of RFX5 in mediating CTX resistance, hypothesizing that decreased RFX5 expression will decrease drug resistance in vitro. To test this hypothesis, we achieved gene knockdown through the administration of silencing RNA (siRNA) targeting RFX5 then assessed sensitivity by treating cells with PM, determining cell viability to quantify IC50. As determined by qPCR, knockdown of RFX5 expression in the cell line MDA-MB-468 resulted in a 91% reduction in RFX5 expression compared to cells treated with scrambled siRNA (p = 0.004). At a PM concentration of 0.4 mM, RFX5 knockdown cells showed significantly reduced viability compared to scramble control cells (33.3 ± 4.5% vs. 49.6 ± 5.4%; p = 0.031). Furthermore, there was a 15% reduction in the IC50 of knockdown cells compared to control cells (0.273 ± 0.016 mM versus 0.346 ± 0.020 mM; P = 0.004). Together, these results provide strong evidence that RFX5 significantly attenuates CTX resistance. This work aims to link SNP genotype, gene expression, and drug sensitivity to improve predictive models of patient response and support targeted therapies enhancing CTX efficacy. Ongoing studies aim to delineate the pathways mediating aberrant RFX5 activity and validate its role in PM sensitivity across additional TNBC cell lines, with a focus on using CRISPRi to investigate the impact of genetic variation on the regulation RFX5 expression and RFX5 target genes.


 Elizabeth Katherine Molchan, Kathleen Streeks, Alyssa Ohaegbulam, Mohammed Gbadamosi. RFX5 knockdown restores sensitivity to cellular cyclophosphamide in triple-negative breast cancer cell lines [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 545.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b575dc8351428b4d22b9c3382531e97fc82f1bcf" target='_blank'>
              Abstract 545: RFX5 knockdown restores sensitivity to cellular cyclophosphamide in triple-negative breast cancer cell lines
              </a>
            </td>
          <td>
            Elizabeth K Molchan, Kathleen Streeks, Alyssa Ohaegbulam, Mohammed Gbadamosi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 e15533


 Background:
 Colorectal cancer (CRC) remains a major global health concern due to its high prevalence and mortality, especially in advanced stages. Despite advancements such as immune checkpoint inhibitors, resistance and tumor heterogeneity remain significant obstacles. Recent research highlights the β2-adrenergic receptor (ADRB2) signaling pathway as a crucial factor in T cell exhaustion, which weakens the immune system’s anti-tumor response. Inhibiting this pathway could restore immune function against CRC. Single-cell RNA sequencing (scRNA-seq) has enhanced understanding of the tumor microenvironment, offering new opportunities for discovering biomarkers and therapeutic targets for personalized treatment strategies.
 Methods:
 We applied scRNA-seq and bulk RNA sequencing to examine adrenergic receptor signaling in CRC, particularly before and after neoadjuvant therapy. Bioinformatics analyses identified T cells expressing ADRB2, confirmed by immunofluorescence and flow cytometry. Copy number variation (CNV) analysis detected malignant cells in the tumor microenvironment, and intercellular communication studies were conducted to explore interactions between ADRB2+ T cells and other immune cells. RNA scope was employed to map ADRB2 distribution in tissue samples. Additionally, deconvolution of TCGA data revealed important immune cell populations, helping to build a prognostic model based on adrenergic receptor signaling and therapy-induced gene expression changes.
 Results:
 The adrenergic receptor signaling pathway was notably enriched in immune cells, especially T cells. After neoadjuvant therapy, scRNA-seq revealed a significant increase in ADRB2 expression in T cells. RNA scope validated ADRB2 co-localization with T cells in CRC tissues, while flow cytometry showed increased ADRB2 expression alongside markers of T cell exhaustion and reduced immune function. CNV analysis identified malignant populations within the tumor microenvironment. Studies of intercellular interactions suggested that ADRB2+ T cells contribute to immune suppression. TCGA data deconvolution indicated that ADRB2+ T cells and CD44+ tumor cells are critical for prognosis. The resulting prognostic model, integrating adrenergic signaling and post-treatment gene expression changes, demonstrated strong predictive power for patient outcomes.
 Conclusions:
 Increased ADRB2 expression in T cells after neoadjuvant therapy in CRC correlates with poor prognosis. This positions ADRB2 as a promising prognostic biomarker and a potential therapeutic target to enhance immunotherapy effectiveness following neoadjuvant treatment. Targeting ADRB2 could offer novel approaches to alleviate immune exhaustion and improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65e2be39dd18963a3eba9d28b1c8ce6324042919" target='_blank'>
              ADRB2-driven immunosuppression as a novel prognostic indicator and therapeutic avenue following neoadjuvant treatment in colorectal cancer.
              </a>
            </td>
          <td>
            Songbing He, Tianliang Liu, Yingyi Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10fbe2acee93fcc04d4cf719859f00b1fc38a341" target='_blank'>
              Megabase-scale human genome rearrangement with programmable bridge recombinases
              </a>
            </td>
          <td>
            Nicholas T. Perry, Liam J. Bartie, Dhruva Katrekar, Gabriel A. Gonzalez, Matthew G. Durrant, James J. Pai, Alison Fanton, Masahiro Hiraizumi, Chiara Ricci-Tam, Hiroshi Nishimasu, Silvana Konermann, Patrick D. Hsu
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Resistance to cytotoxic cancer drugs is considered a key cause of cancer deaths worldwide. Although extensive understanding of cancer drug mechanisms of action (MoA) and resistance has been derived through focused research, our understanding of these process still remains fragmented.
 The Cellular Thermal Shift Assay (CETSA) was originally developed in our lab for accessing drug target engagement in intact cells (Martinez Molina, 2013), and is now widely used for this application in academia and industry. However, CETSA can also detect modulations of cellular biochemistry (Dai, 2018), e.g. effects on downstream-of-target pathways, although this application of CETSA remains relatively unexplored. Using a comprehensive and highly sensitive mass-spectrometry implementation of CETSA, we have studied the cellular biochemistry induced by more than 50 clinical cancer drugs. By applying a systematic approach, including time dependent responses to dissect sequential pathway activation, we can confirm components of previously known MoAs for these drugs but also provide comprehensive new insights into their MoA (Ramos 2024). Moreover, parallel studies in sensitive and resistant cancer cell lines provide differential pathway activation in sensitive and resistant cells (Liang, 2021), giving new insights into how cancer cells acquire drug resistance at the biochemical level. Together our data illustrate that CETSA can yield valuable information on modulation of downstream-of-target biochemistry during cancer drug action, information that is not easily accessed with other omics methods. Proteins discovered in some specific pathways also constitute new candidate drug targets for attenuate resistance responses, as well as potential clinical biomarkers.


 Martinez Molina D, Jafari R, Ignatushchenko M, Seki T, Larsson EA, Dan C, Sreekumar L, Cao Y and Nordlund Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay. Science, 2013 341:84.- Dai L, Zhao T, Bisteau X, Sun W, Prabhu N, Lim YT, Sobota RM, Kaldis P, Nordlund P. Modulation of Protein-Interaction States through the Cell Cycle. Cell, 2018 173(6):1481-1494. - Liang YY, Bacanu S, Sreekumar L, Ramos AR, Dai L, Michaelis M, Cinatl J, Seki T, Cao Y, Coffill CR, Lane DP, Prabhu N, Nordlund P, CETSA interaction proteomics define specific RNA modification pathways as key components of fluorouracil-based cancer drug toxicity. Cell Chemical Biology, 2021, S2451-9456, 00306.-Ramos AD, Liang YY, Surova O, Bacanu S, Gerault M, Mandal T, Ceder S, Langebäck A, Österroos A, Ward GA, Bergh J, Wiman KG, Lehmann S, Prabhu N, Lööf S, Nordlund P. Proteome-wide CETSA reveals diverse apoptosis-inducing mechanisms converging on an initial apoptosis effector stage focused at the peripheral region of the nucleus. Cell Reports, 2024 Oct 22;43:114784.



 Par Nordlund, Sara Lööf, Anderson Ramos, smaranda Bacanu, Marc Antoine Gerault, Ying Yu Liang, Wai Leong TAM, Nayana Prabhu. How cancer drugs work, and fail: proteome-wide cellular thermal shift assay (CETSA) to define cellular mechanism of action and resistance of clinical cancer drugs [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5763.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294979244322fee861b6d9b92034605b4639b574" target='_blank'>
              Abstract 5763: How cancer drugs work, and fail: proteome-wide cellular thermal shift assay (CETSA) to define cellular mechanism of action and resistance of clinical cancer drugs
              </a>
            </td>
          <td>
            Pär Nordlund, Sara Lööf, A. Ramos, smaranda Bacanu, Marc-Antoine Gerault, Ying Yu Liang, Wai Leong TAM, N. Prabhu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, M. Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Melanomas have poor prognosis at the metastatic stage despite impressive progress in targeted- and immuno-therapies. Therefore, identification of novel critical genes involved in melanoma biology with targeting potential is desirable.
 We and others have previously demonstrated the transcription factor interferon regulatory factor 4 (IRF4) as a dependency factor in various lymphocyte-derived cancers. These findings align with the therapeutic efficacy attained by immunomodulatory drugs which indirectly target IRF4 expression in these cancers.
 IRF4 also takes part in pigmentation in melanocytes, and its overexpression is prevalent in melanomas. Recently, we showed that high expression of IRF4 is linked to dependency in melanoma cells and mortality in patients. Mechanistically, we demonstrated that IRF4 regulates epigenetic silencing factors involved in DNA and histone H3K27 methylation, and consequently, controls the expression of melanoma-critical tumor suppressor genes and key downstream oncogenic pathways.
 In order to identify novel genes that melanoma cells rely on for survival or proliferation, we performed CRISPR/Cas9 drop-out screens. One set of screens was aimed at identifying single individual genes necessary for cell fitness. The other set of screens was aimed at uncovering genes, when knocked-out, compromised cell fitness synthetically with IRF4 deficiency. For each set, we screened in replicate two publicly available sub-genomic guide RNA libraries, one targeting genes involved in gene expression and the other targeting genes involved in apoptosis and cancer processes. In order to detect drop-outs between the start and the end of the experiments, we analyzed samples quantitatively via next generation sequencing of guides and the MAGeCK algorithm. Depletion of guides targeting core essential genes verified the screens’ ability to detect dependency genes. We then defined all significant dependency genes in their respective screens. We observed that, in addition to core essential genes, several genes that are deemed druggable show up as essential for melanoma cell fitness, some of which not previously reported. Several additional genes show up only in IRF4-knock out state, suggesting synthetic dependency. We followed up with pathway enrichment analyses to identify the prominent cellular pathways with dependency genes. Here, we observed gene transcription and translation, and cell cycle/mitosis as pathways enriched for melanoma dependency genes, among others. In addition, RNA processing and protein degradation are among the pathways enriched for dependency genes in the IRF4-knockout setting.
 The dependency genes uncovered in these CRISPR/Cas9 drop-out screens are expected to pave the way for novel targeting approaches in melanoma by themselves or in combination with IRF4.


 Ulduz Sobhiafshar, N. C. Tolga Emre. CRISPR/Cas9 essentiality screens identify stand-alone and IRF4-linked genetic vulnerabilities in melanoma cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2771.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/919d6be56b49f799484f0a056c28802836f4c288" target='_blank'>
              Abstract 2771: CRISPR/Cas9 essentiality screens identify stand-alone and IRF4-linked genetic vulnerabilities in melanoma cells
              </a>
            </td>
          <td>
            U. Sobhiafshar, N. T. Emre
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Breast cancer affects over 300, 000 individuals annually in the United States, with survival rates improving due to advances in chemotherapy, radiation, hormonal, and targeted therapies. However, these treatments may inadvertently accelerate biological aging through epigenetic modifications, including DNA methylation, histone alterations, and changes in non-coding RNAs. This review explores how breast cancer therapies impact aging-related epigenetic pathways. Chemotherapy induces cellular senescence by altering DNA methylation patterns and histone modifications, potentially leading to the early onset of age-related diseases. Targeted therapies also affect epigenetic regulation, influencing gene expression related to cell cycle control, apoptosis, and DNA repair. Non-coding RNAs, such as microRNAs and long non-coding RNAs, are essential gene expression regulators affected by cancer treatments. They play significant roles in cellular senescence and therapeutic resistance, adding complexity to the aging phenotype observed in survivors. Clinical manifestations like cardiovascular dysfunction, endocrine disorders, and systemic inflammation are linked to therapy-induced epigenetic changes and in turn accelerated aging. Emerging epigenetic therapies targeting DNA methylation and histone modifications offer potential strategies to mitigate treatment-induced aging effects. DNA methyltransferase inhibitors and histone deacetylase inhibitors show promise in reactivating silenced genes and enhancing therapy sensitivity. Non-pharmacological approaches, including exercise, dietary modifications rich in bioactive compounds, and gut microbiota modulation, may also counteract therapy-exacerbated aging processes. In conclusion, existing evidence shows that cancer therapies are associated with decreased cardiorespiratory fitness, increased risk of cardiomyopathy, osteoporosis, and metabolic dysregulation, contributing to accelerated biological aging via diverse epigenetic mechanisms. Recognizing these changes is crucial for developing interventions that survivor’s health and quality of life. Further research is needed to fully understand therapy-induced epigenetic alterations and their long-term impacts, facilitating personalized survivorship care that addresses this population’s aging complexities.


 Nikita Nikita, Zhengyang Sun, Swapnil Sharma, Amy Shaver, Victoria Seewaldt, Ana Maria Lopez, Grace Lu Yao. Accelerated aging in breast cancer survivors: the role of therapy-induced epigenetic modifications [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b49562d5cbbdfc03f5130abae6836f4cec03143" target='_blank'>
              Abstract 4109: Accelerated aging in breast cancer survivors: the role of therapy-induced epigenetic modifications
              </a>
            </td>
          <td>
            Nikita Nikita, Zhengyang Sun, Swapnil Sharma, Amy Shaver, Victoria L Seewaldt, Ana María López, Grace Lu Yao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Synthetic lethality (SL) offers a promising avenue for cancer therapy by exploiting vulnerabilities unique to tumor cells and minimizing off-target effects on normal tissue. However, the application of SL-based therapies remains constrained by a limited repertoire of validated targets and biomarkers. Using data from the Cancer Dependency Map (DepMap) and advanced in silico modeling, Evariste’s Frobenius Target platform identified 50 potential SL gene pairs with druggable nodes and rational mechanisms linking targets and biomarkers. From this set, 24 gene pairs were prioritized for validation based on translational relevance and impact on the patient population.
 Validation was achieved through a collaborative high-throughput workflow with EditCo Bio, leveraging their CRISPR-based cell line engineering platform to generate 72 knockout pools in under three weeks. Validation studies in multiple cell lines demonstrated >10 high-confidence SL pairs, including a novel SL relationship involving focal adhesion kinase (FAK). To further substantiate this relationship, novel inhibitors were developed with best-in-class selectivity over the SL partner gene. Functional assays confirmed these inhibitors’ effectiveness in biomarker-low cell lines, with promising implications for clinical translation.
 This study highlights the potential of AI-driven approaches to uncover the next generation of SL targets, expanding the therapeutic landscape in cancer research. By focusing on biomarker-low gene expression as a predictor of SL, our work establishes a robust framework for the discovery and validation of clinically actionable SL relationships.


 Amalia Soenens, Daniel Miller, Oliver Vipond, Florencia Skuras, Alfie Brennan, Anna Hercot. AI driven discovery of novel synthetic lethal gene pairs for targeted cancer therapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3643.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b05e86f87ff9d548ca0d97272bc2384f88e53a2" target='_blank'>
              Abstract 3643: AI driven discovery of novel synthetic lethal gene pairs for targeted cancer therapy
              </a>
            </td>
          <td>
            Amalia Soenens, Daniel S. Miller, Oliver Vipond, Florencia Skuras, Alfie Brennan, Anna Hercot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Osimertinib is a 3rd generation EGFR inhibitor and the newest FDA approved adjuvant therapy for non-small cell lung cancer (NSCLC) with EGFR exon 19 deletions or exon 21 L858R mutation. Although the drug can yield impressive initial responses, the majority of patients eventually progress due to the evolution of resistance. While the molecular mechanisms underlying resistance have been well studied, little is known about the ecological and evolutionary laws that govern the creation and subsequent expansion of these mutant populations. How frequently are different evolutionary routes taken and to what degree is evolution repeatable? To study the repeatability of osimertinib resistance evolution, we conducted a large-scale evolution experiment aimed at characterizing the distribution of resistant states that emerge, along with the pathways and vulnerabilities associated with their development. In our experimental design, we evolve PC-9, an EGFR-mutated NSCLC cell line to osimertinib independently 10 times (evolutionary replicates) for 10 cycles (serial passages). We also evolved 6 control replicates that undergo no treatment. At the end of the experiment, we will employ a multi-faceted approach to studying the resistance phenotype: 1) IC50 against osimertinib; 2) bulk RNA-seq; 3) DNA/mutational profiling; 4) a collateral sensitivity screening against 20 FDA approved drugs; and 5) a radiation sensitivity assay against 6 doses. The initial pretreatment dose response curve (DRC) against osimertinib showed a double plateau, suggesting the pre-existence of tolerant or even resistant subpopulation. In our control arms, we see consistent persistence of our double plateau throughout all 10 cycles, indicating this population is maintained in the absence of treatment but cannot expand. Conversely, across all 8 treated replicates we see a large shift in IC50 within the 1st treatment cycle, consistent with rapid selection and outgrowth of this population. More detailed analysis using time-lapse microscopy shows while these cells have a higher IC50, they also have a reduced growth rate, even if treatment is completely removed again (a “resistance cost”). This suggests potential evolutionary vulnerabilities which could be leveraged through strategic treatment breaks or drug sequences.


 Mina N. Dinh, Jinling Wu, Masahiro Hitomi, Maximilian Strobl, Jacob G. Scott. Characterizing the evolution of osimertinib resistance in EGFR mutated non-small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1247.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ab08c733369ba98ccceec430e609487bfe090d5" target='_blank'>
              Abstract 1247: Characterizing the evolution of osimertinib resistance in EGFR mutated non-small cell lung cancer
              </a>
            </td>
          <td>
            Mina N. Dinh, Jinling Wu, Masahiro Hitomi, Maximilian Strobl, Jacob G. Scott
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892d32d1ea696e84aa4f250bf0edeb5b5584501c" target='_blank'>
              CRISPRi perturbation screens and eQTLs provide complementary and distinct insights into GWAS target genes
              </a>
            </td>
          <td>
            Sam Ghatan, Jasper Panten, Winona Oliveros, Neville E. Sanjana, John A Morris, Tuuli Lappalainen
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Small cell lung cancer (SCLC) is an aggressive disease with poor treatment outcomes, in part due to epigenetic mechanisms driving tumor growth and resistance. Cell-free DNA methylome profiles in untreated SCLC can identify prognostic sub-groups, making it a useful biomarker. In this study, we hypothesize that changes in the cell free methylome of SCLC influence disease response to therapy and drive treatment resistance.



 Cell free methylated DNA immunoprecipitation followed by sequencing (cfMeDIP-seq) was performed on 34 baseline-relapse paired samples belonging to a cohort of SCLC patients treated at the Princess Margaret Cancer Centre (Toronto, Canada). Matched leukocyte DNA methylation profiles were incorporated to exclude the contribution of non-cancer methylation to the cfMeDIP signal and allow focused profiling of changes in the SCLC methylome through first-line treatment. The plasticity of the SCLC methylome was determined by subtracting the reads per kilobase of transcript per million mapped reads (RPKM) at the time of progression compared to the pre-treatment timepoint. A methylome demonstrated plasticity if it showed an increase or decrease in reads compared to the median RPKM across all samples in the cohort. Associations between methylation groups and relevant clinical data were identified. Kaplan-Meier and Cox regression analysis were performed to determine if methylome changes were associated with overall survival and progression-free survival, anchored from the time of SCLC diagnosis. KEGG pathways corresponding to the top differentially methylated windows between baseline and relapse pairs were identified and characterized.



 Plasticity of the SCLC methylome were seen in 67% of patients (n=23/34). Methylome plasticity was found to be associated with a shorter time to progression from the end of first-line treatment (mean difference = 70 days; 95% CI 25-115, p = 0.0031). Accordingly, patients with greater methylome plasticity were more likely to be platinum resistant (57% vs. 17%; χ-squared p = 0.033). Cox regression showed that methylome plasticity was significantly associated with worse progression free survival (PFS) (adjusted hazard ratio [aHR] = 5.7, p = 0.0010) and overall survival (OS) (aHR = 1.9, p = 0.11), after adjusting for VA stage at diagnosis. Pathway analysis of differentially methylated windows mapped to genes related to neuronal polarity and axonal guidance, as well as Wnt signaling.



 Changes in cell-free DNA methylomes serve as a biomarker for treatment response in SCLC. Increased plasticity of the methylome is associated with shorter PFS. The underlying biology of this relationship may involve changes in pathways that govern neuronal change and established cancer-associated pathways.



 Danielle Benedict Sacdalan, Sami Ul Haq, Luna Jia Zhan, Janice J. Li, Vivek Philip, Scott V. Bratman, Geoffrey Liu, Benjamin H. Lok. Methylome plasticity as a biomarker of treatment response in small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2758.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532359eb783a00c7e5a0af7f6a8c8009cbbb7c55" target='_blank'>
              Abstract 2758: Methylome plasticity as a biomarker of treatment response in small cell lung cancer
              </a>
            </td>
          <td>
            D. Sacdalan, Sami Ul Haq, L. Zhan, J. J. Li, V. Philip, Scott V. Bratman, Geoffrey Liu, Benjamin H. Lok
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Adoptive cell therapy using engineered natural killer (NK) cells has emerged as a promising new treatment, with gene editing offering opportunities to further increase antitumor potency. However, strategies to address both intrinsic and extrinsic therapeutic barriers remain largely unexplored. To address this, we developed a pooled CRISPR discovery platform to identify genetic perturbations that can be exploited for CAR-NK cellular engineering strategies.
 Using primary human NK cells, we performed multiple genome-wide CRISPR screens under conditions of repeated tumor challenge, relevant to the tumor microenvironment (TME). These screens uncovered novel NK cell checkpoints that regulate resistance to tumor-induced and TME-driven immunosuppression. High-content screening in NK cells with enhanced degranulation capacity validated these findings, identifying phenotypic correlates of how reprogrammed NK cells acquire enhanced potency. Cross-validation of top screen hits in a large patient cohort across 17 diverse cancer types demonstrated transcriptional upregulation of top hits compared to healthy donors, underscoring their clinical relevance.
 Key pathways enriched among the hits included components of the Cul5-RING ubiquitin ligase, the Mediator and the SAGA complexes, as well as genes involved in chromatin remodeling and negative regulation of NFκB. These findings highlight transcriptional reprogramming and post- translational modification (PTM) components as means to optimize NK cell effector functions for enhanced anticancer immunity.
 Targeted ablation of top hits MED12, ARIH2 and CCNC markedly improved the antitumor potency of both non-transduced and CAR/IL-15-engineered NK cells against multiple hard-to-treat cancers using two different IL-15-armored CAR receptors targeting CD70 and TROP2. ARIH2 and CCNC deficient TROP2/IL-15-CAR-NK cells displayed enhanced metabolic fitness and a marked increase in activated/cytotoxic clusters (DNAM-1high, Granzyme Bhigh, Perforinhigh, CD69high), with a concomitant decrease in inhibitory subsets (NKG2Ahigh, Granzyme Blow, CD69low), as shown by mass cytometry.
 Multiplexed ARIH2/CCNC editing of TROP2/IL-15-CAR-NK cells significantly increased NK cell expansion, tumor bed infiltration, release of cytotoxic granules and antitumor efficacy in an orthotopic in vivo mouse model of pancreatic cancer. This was accompanied by the induction of cytotoxic (CD16high, Granzyme Bhigh, Perforinhigh) and activated (DNAM-1high, CD69high, NKG2Dhigh) NK cell subsets.
 In conclusion, by applying high-content genome-wide CRISPR screening in primary human NK cells, this study revealed novel regulators of NK cell function, including MED12, ARIH2 and CCNC. Targeting these regulators enhances CAR-NK cell metabolic fitness, cytotoxicity and in vivo antitumor efficacy, offering new opportunities for optimizing CAR-NK therapies.


 Alexander Biederstädt, Rafet Basar, Jeong-Min Park, Merve Dede, John G. Watts, Nadima Uprety, Rejeena Shrestha, Francia Reyes-Silva, May Daher, Bin Liu, Hind Rafei, Pinaki Prosad Banerjee, Ping Li, Sunil Acharya, Sanjida T. Islam, Huihui Fan, Donghai Xiong, Vernikka C. Woods, Paul Lin, Silvia Tiberti, Ana K. Nunez Cortes, Xin Ru Jiang, Inci Biederstädt, Patrick C. Zhang, Ye Li, Seema Rawal, Enli Liu, Luis H. Muniz-Feliciano, Gary M. Deyter, Elizabeth J. Shpall, Natalie W. Fowlkes, Ken Chen, Katayoun Rezvani. Genome-wide CRISPR screens identify critical targets to enhance CAR-NK cell antitumor potency [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3763.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b23788a6f827c0b72ecb2ed6905b6d5b0e6258" target='_blank'>
              Abstract 3763: Genome-wide CRISPR screens identify critical targets to enhance CAR-NK cell antitumor potency
              </a>
            </td>
          <td>
            Alexander Biederstädt, R. Başar, J. Park, Merve Dede, John G. Watts, N. Uprety, R. Shrestha, Francia Reyes-Silva, May Daher, Bin Liu, H. Rafei, P. Banerjee, Ping Li, S. Acharya, S. Islam, Huihui Fan, Donghai Xiong, Vernikka Woods, Paul Lin, Silvia Tiberti, A. N. Nunez Cortes, X. Jiang, Inci Biederstädt, Patrick Zhang, Ye Li, S. Rawal, Enli Liu, Luis Muniz-Feliciano, Gary M. Deyter, Elizabeth J. Shpall, N. Fowlkes, Ken Chen, Katayoun Rezvani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 In cancer cells, chromatin replication is often misregulated in part due to mutations in oncogenes and tumor suppressor genes, leading to increased replication stress and genomic instability. While this instability drives cancer progression, it also makes cancer cells more dependent on genome maintenance pathways, presenting a therapeutic target. A promising strategy is to exacerbate replication stress, causing lethal genome instability. We have developed a sensitive reporter to monitor replication stress via single-stranded DNA at stressed forks. Our earlier study identified Tousled-like kinases (TLKs) crucial for chromatin assembly during genome replication and discovered TLK inhibitors to treat cancers through increasing replication stress. Additionally, we recently uncovered histone deacetylase 8 (HDAC8) inhibitors that disrupt chromatin replication and compromise genome integrity, particularly when combined with checkpoint kinase inhibitors. In this presentation, I will show you how HDAC8 behaves as caretakers of genetic and epigenetic integrity and how we targeted replication stress to create cancer-specific synergistic vulnerabilities.


 Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee. HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1475.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320798304ad2c6aa4cbcc689f8af4488470a00c4" target='_blank'>
              Abstract 1475: HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications
              </a>
            </td>
          <td>
            Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Although anti-epidermal growth factor receptor (EGFR) therapy has established efficacy in RAS/BRAF wild type (WT) metastatic colorectal cancer (mCRC), a significant proportion (30-50%) of WT patients exhibit primary resistance to this treatment. Existing biomarker discovery has predominantly focused on tumor-specific DNA mutations, which suffer from variable mutation frequencies among patients. In contrast, DNA methylation represents a common and early event in carcinogenesis. For this reason, methylation markers hold great promise for detecting resistance mechanisms in primary tumors. Several studies have shown an association between DNA methylation patterns and resistance to anti-EGFR therapy in CRC. To validate and build upon these findings, we investigated the methylomes of primary tumors with resistant and responsive phenotypes.



 We isolated DNA from FFPE tissue of 40 primary CRC tumors. DNA was subjected to bisulfite conversion and Infinium MethylationEPIC v2.0 BeadChip (Illumina). Subsequently, ChAMP was used to determine the methylation values (β values) and perform differential methylation analysis. The CpG sites were ranked based on the largest Δβ values and the significance of p-values across the patient groups. To ensure the reliability of our findings, we performed a false discovery rate (FDR) evaluation using the Benjamini-Hochberg method, which led to the selection of a subset of differentially methylated CpG sites (DMCs) with the highest likelihood of being genuine associations. Additionally, a gene set enrichment analysis (GSEA) was performed with methylGSA.



 48 DMCs were identified with adjusted p-values < 0.05. The affected genes are involved in cancer progression and invasiveness, suggesting a role in a yet-to-be discovered resistance mechanism. The GSEA revealed significant enrichment in 53 KEGG and 33 Reactome pathways, including MAPK-related and Akt signaling pathways. There were 470 significant gene ontology (GO) pathways: 347 in biological processes, 71 in cellular components and 52 in molecular functions.



 We have found methylation sites associated with resistance to anti-EGFR therapy in CRC patients. In addition to identifying critical DMCs, the pathways uncovered hold potential as a rich source of predictive biomarkers. Next, we will include 24 additional patients. When increasing the sample size, we expect a higher statistical power and an enrichment of CpGs with significant p-values. Afterwards, we will independently validate the most significant CpG sites in a follow-up study using a highly sensitive multiplexed droplet digital PCR assay on additional tissue biopsies.



 Ana Regina de Abreu, Yann Heylen, Joe Ibrahim, Marc Peeters, Guy Van Camp, Ken Op de Beeck. Uncovering methylation signatures and pathways associated with resistance to anti-EGFR therapy in colorectal cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3311.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/844d851fd9820543dfb4e679c6a5eddb01a41ffd" target='_blank'>
              Abstract 3311: Uncovering methylation signatures and pathways associated with resistance to anti-EGFR therapy in colorectal cancer
              </a>
            </td>
          <td>
            Ana Regina de Abreu, Yann Heylen, J. Ibrahim, Marc Peeters, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) patients with T cell-cold tumors experience rapid cancer progression and poor survival. While T cell-enhancing immunotherapies have improved outcomes for a subset of CRC patients, most remain nonresponsive. Therefore, better understanding the mechanisms that drive T cell dysfunction in CRC is needed. We used matched gene expression and copy number profiles from TCGA to evaluate CRC tumors (N=373) based on measures of T cell infiltrate, and developed a method to identify genes with potential roles in T cell exclusion. Leveraging the intersect between copy number amplification and gene expression in the T cell-cold state, we pinpointed a region of chromosome 20q11.21 where DNA- and RNA-levels were more strongly correlated with T cell exclusion than anywhere else in the CRC genome. This locus also contained the most frequently amplified and overexpressed genes in the CRC genome. Within this region, Protein O-Fucosyltransferase 1 (POFUT1) was the most distinguished gene, and correlated with CTL inhibition across 12/24 solid tumor types annotated in TCGA (SKCM, THCA, COAD, READ, LUSC, CESC, KIRC, STAD, HNSC, KIRP, LIHC, UCEC; p<0.0001). Stable overexpression of POFUT1 (POFUT1-OE) in MC38 mouse colon cancer cells enhanced proliferation relative to MC38-Vector (RTCA p<0.0001, CellTiterGlo p=0.0011); and in co-culture with c57BL/6 spleen-derived PBMCs, CD8+ T cell populations from MC38-POFUT1-OE co-cultures were significantly reduced over the course of 7 days. This effect persisted when PBMCs were stimulated with IL-2 and anti-CD2/CD3/CD28 in 50% conditioned medium (CM) from MC38-POFUT1-OE, prompting us to profile the CM for differences in secretomes. Of 110 profiled cytokines, two were at significantly higher levels in MC38-POFUT1-OE CM: IL-6 and IL-23 (p=0.0003, p=0.0055). These results were validated via ELISA and transient knockdown (KD) of POFUT1, where POFUT1-KD resulted in reduced cancer cell proliferation, diminished secretion of IL-6, and increased CD8+ T cell abundance in our CM models. Furthermore, siRNA-mediated KD of POFUT1 in MDST8 and HCT15 human CRC cell lines resulted in significant reductions in secreted IL-6 (p=0.0003, p<0.0001). Treatment of murine PBMCs with IL-6 (5.0 pg/mL - 50 ng/mL) decreased the proportion of CD8+ T cells to an analogous level as culture with POFUT1-OE CM. The decrease in CD8+ T cell proportion was also associated with less proliferation (CFSE, p=0.0049), and reduced surface expression of activating co-receptors CD25 and CD69 (p=0.1008, p=0.0015). Dosing with anti-IL-6 IgG (10 ng/mL) reversed the decrease in CD8+ T cell proliferation induced by CM from MC38-POFUT1-OE, establishing IL-6 as the downstream mechanism of POFUT1-dependent inhibition of CD8+ T cell activity. Together, these findings link POFUT1 amplification in CRC with dysregulation of CD8+ T cells via upregulation of cancer-intrinsic IL-6 secretion.


 Calvin Jordan Wagner. Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 904.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 The Cancer Dependency Map (DepMap) is an ongoing initiative to identify and explore synthetic lethal targets, by systematically linking tumor molecular alterations to gene dependencies and drug sensitivities. The resource used by scientists worldwide has significantly contributed to both basic and translational cancer research and has provided the community with a landscape view of synthetic lethal targets. Numerous targets identified through DepMap are actively under investigation, with several already advancing into clinical trials, highlighting their clinical relevance and potential. For instance, we recently discovered a synthetic lethal interaction involving the PELO-HBS1L and SKI complex, both involved in mRNA quality control. PELO is a promising synthetic lethal target in cancers with SKI complex deficiencies, such as those with FOCAD deletions or high microsatellite instability. Despite the progress, much work remains to fully exploit cancer dependencies. We have integrated next-generation patient-derived models, including organoids and neurospheres grown in 3D to better represent various cancer subtypes. Additionally, we used di-genic screening libraries to uncover synthetic lethal dependencies previously masked due to paralog redundancy. We are also actively expanding multi-omics datasets and building computational features that will enable the identification of novel molecular links to synthetic lethal dependencies. DepMap continues to grow with the ultimate goal of creating a comprehensive and detailed landscape of cancer vulnerabilities to accelerate precision cancer medicine.


 Francisca Vazquez. Mapping synthetic lethal interactions at scale: A functional genomics approach to cancer therapeutic discovery [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY14-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a986ae0bdffed64d9a4eefeb701fd4d7309816c" target='_blank'>
              Abstract SY14-02: Mapping synthetic lethal interactions at scale: A functional genomics approach to cancer therapeutic discovery
              </a>
            </td>
          <td>
            Francisca Vazquez
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Acute myeloid leukemia (AML) is a heterogeneous malignancy characterized by genetic and epigenetic dysregulation, including aberrant activity of chromatin modifiers. The complex interactions among these modifiers and regulators in AML pathogenesis remain poorly understood, posing significant challenges to developing effective combination therapies. To address this, we used Perturb-seq, a CRISPR-based single-cell transcriptomics approach, to profile 16 key genes from the Menin-MLL, Polycomb, and histone acetyltransferase (HAT) complexes in the MOLM-13 AML cell line. We developed a machine learning framework called Composer, which predicts the combinatorial effects of perturbations by analyzing their impact on the transcriptome. We identified cooperative regulation of oncogenic pathways by Menin-MLL and KAT6A, while PRC1 components, such as PCGF1, were found to antagonize DOT1L activity. These predictions were validated using bulk RNA-seq and experiments combining perturbations through small molecule inhibitors and gene knockouts, demonstrating that dual inhibition of Menin-MLL and KAT6A, or DOT1L and KAT6A, significantly reduced AML cell viability. This study underscores the power of combining AI with single-cell Perturb-seq to provide novel insight into the epigenetic landscape of AML and identify promising therapeutic targets to overcome treatment resistance.


 Changde Cheng, Rui Lu, Sajesan Aryal, Doug A. Welsch. Uncovering synergistic and antagonistic interactions in epigenetic therapeutics using perturb-seq [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 219.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08272a17f03be6497869a3622bf7e66968e9d5bd" target='_blank'>
              Abstract 219: Uncovering synergistic and antagonistic interactions in epigenetic therapeutics using perturb-seq
              </a>
            </td>
          <td>
            Changde Cheng, Rui Lu, Sajesan Aryal, Doug A. Welsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 CRISPR screens have been successfully used to identify targetable genes for anti-cancer research and drug development. Typically, CRISPR screens are conducted in vitro on cell lines, which do not fully capture the influence of the in vivo physiological environment. It is well known that any genetic editing might trigger the adjustment of gene balance to compensate for the modifications in the target cells and tissues. To address the complexity of cellular interactions and the immunological response, in vivo CRISPR screening on a xenograft model is a compelling approach to unravel biological processes where cells are targeted within their natural physiological conditions.



 We have conducted a CRISPR screening approach using the human non-small cell lung cancer cell line A549 by knocking out tumor suppressor genes using a library of 3, 188 single guide RNAs (sgRNA). Subsequently, in vivo testing was carried out with the genetically edited A549 cells with or without treatment with the kinase inhibitor Trametinib. At the study endpoint, tumor samples were isolated and analyzed. Amplicon sequencing was used to capture residual sgRNA. MAGeCK was used to calculate the gene enrichment score.



 After CRISPR editing, 37 sgRNA corresponding to 20 genes were captured in the edited A549 cell line. In the vehicle-treated A549 xenograft tumors, 45 sgRNA corresponding to 27 genes were captured, of which 17 genes were overlapping with engineered A549 cell culture, 8 upregulated and 9 downregulated. In comparison, in the A549 xenograft tumors of the Trametinib treated group, 38 sgRNA corresponding to 23 genes were discovered, with 15 overlapping genes, 9 upregulated and 6 downregulated. The change in expression levels was abundant. Of the 3, 188 sgRNA, the majority of sgRNAs were filtered out in the A549 cell line, under in vivo evolution and the influence of Trametinib treatment. We believe that the pathways related to this small set of identified genes may reveal potential new targets for anti-cancer research and drug development.



 The in vivo CRISPR approach offers a valuable and translational tool for discovering novel targets in cancer research, providing insights for new anti-cancer therapies.



 Ce Zhong, Lu Bai, Tongtong Wang, Yiying Shao, Lingyun Zhang, Lu Zhang, Jingqi Huang. Identification of potential therapeutic targets by in vivo CRISPR screening on xenograft models [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5673.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55dbe0088f239e1966f0ecbd4169616a69ee9add" target='_blank'>
              Abstract 5673: Identification of potential therapeutic targets by in vivo CRISPR screening on xenograft models
              </a>
            </td>
          <td>
            Ce Zhong, Lu Bai, Tongtong Wang, Yiying Shao, Lingyun Zhang, Lu Zhang, Jingqi Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 We have leveraged long-read and spatial sequencing modalities to identify cancer-specific oncogenic alterations for treatment of osteosarcomas.
 Osteosarcoma is archetypal of cancers driven by aneuploidy and structural rearrangement rather than point mutations. The functional consequences of many SVs stem from their effect on multidimensional genome organization. SVs can enable enhancer hijacking, alter boundaries of topologically associated domains (TADs), and move gene loci to different regulatory compartments to affect gene expression. SVs can also lead to the formation of extrachromosomal DNA amplicons (ecDNA), which can be megabases in size and can incorporate both genes and regulatory elements to result in substantial increases in gene transcription and intratumor heterogeneity. Complex SVs may have profound implications for prognosis and treatment in OS and other cancers yet they remain poorly understood. By incorporating long-read optical genome mapping (OGM) and chromatin conformation capture sequencing (HiC), we studied SVs in osteosarcoma with the goal of elucidating their contribution to cancer development.
 We studied a unique panel of 11 OS patient-derived xenograft (PDX) cell lines which have been previously characterized regarding response to targeted therapies as well as genomically via whole genome (WGS), RNA (RNA-seq), and chromatin accessibility (ATAC-seq) sequencing. To understand the complex genomic reorganization that occurs in osteosarcoma, we integrated OGM and WGS data for all 11 cell lines to describe detailed genome-wide SVs including ecDNA formation. PDX lines assayed using OGM exhibited between 106 and 704 translocations each (versus zero detected in germline controls) and we identified dozens of potential ecDNA amplicons, including one that may explain exceptionally high expression of YAP1 and BIRC3 transcripts. Further, we use HiC to determine which ecDNA amplicons appear to be free-floating based on their genome-wide interaction scores, noting significantly greater interaction in between 33% and 67% of ecDNA detected using OGM and break point graph-based methods. We also used HiC to describe the topological environment resulting from genome-wide SVs and observed potential enhancer hijacking events affecting dozens of known oncogenes including MYC and CDK4. HiC interaction maps among six cell lines reveal genomic compartments and TAD boundaries in detail, with 4,167 unique TAD boundaries identified among six cell lines using 50kb genomic bins. Changing TAD boundaries are associated with numerous SVs and/or copy number discrepancies, and are associated with genomic regions that contain genes with dysregulated RNA transcript levels. Overall, OGM and HiC modalities add considerably to our ability to detect oncogenic SVs and the genes they affect.


 Andrew Scott Clugston. High-resolution mapping of oncogenic structural changes in osteosarcoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2775.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Recent research has identified a potential protective effect of higher numbers of circulating lymphocytes on colorectal cancer (CRC) development. However, the importance of different lymphocyte subtypes and activation states in CRC development and the biological pathways driving this relationship remain poorly understood and warrant further investigation. Specifically, CD4+ T cells -- a highly dynamic lymphocyte subtype -- undergo remodelling upon activation to induce the expression of genes critical for their effector function. Previous studies investigating their role in CRC risk have used bulk tissue, limiting our current understanding of the role of these cells to static, non-dynamic relationships only. Methods Here, we combined two genetic epidemiological methods -- Mendelian randomisation (MR) and genetic colocalisation -- to evaluate evidence for causal relationships of gene expression on CRC risk across multiple CD4+ T cell subtypes and activation stage. Genetic proxies were obtained from single-cell transcriptomic data, allowing us to investigate the causal effect of expression of 1,805 genes across five CD4+ T cell activation states on CRC risk (78,473 cases; 107,143 controls). We repeated analyses stratified by CRC anatomical subsites and sex, and performed a sensitivity analysis to evaluate whether the observed effect estimates were likely to be CD4+ T cell-specific. Results We identified six genes with evidence (FDR-P<0.05 in MR analyses and H4>0.8 in genetic colocalisation analyses) for a causal role of CD4+ T cell expression in CRC development -- FADS2, FHL3, HLA-DRB1, HLA-DRB5, RPL28, and TMEM258 . We observed differences in causal estimates of gene expression on CRC risk across different CD4+ T cell subtypes and activation timepoints, as well as CRC anatomical subsites and sex. However, our sensitivity analysis revealed that the genetic proxies used to instrument gene expression in CD4+ T cells also act as eQTLs in other tissues, highlighting the challenges of using genetic proxies to instrument tissue-specific expression changes. Conclusions Our study demonstrates the importance of capturing the dynamic nature of CD4+ T cells in understanding disease risk, and prioritises genes for further investigation in cancer prevention research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3854c3b38125ddae1dd0d632aa93ab78f073f33f" target='_blank'>
              Transcriptome-wide Mendelian randomisation exploring dynamic CD4+ T cell gene expression in colorectal cancer development
              </a>
            </td>
          <td>
            Benedita Deslandes, Xueyan Wu, Matthew A Lee, L. J. Goudswaard, Gareth W Jones, A. Gsur, A. Lindblom, Shuji Ogino, Veronika Vymetálková, A. Wolk, Anna H. Wu, J. Huyghe, Ulrike Peters, Amanda I. Phipps, Claire E. Thomas, Rish K. Pai, Robert C Grant, Daniel D Buchanan, J. Yarmolinsky, Marc J. Gunter, Jie Zheng, E. Hazelwood, E. E. Vincent
          </td>
          <td>2025-04-17</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="
 Synthetic lethality, a phenomenon in which alterations in two or more genes together induce cell death while changes in either gene alone do not, provides a promising approach for cancer treatment. Indeed, various inhibitors targeting synthetic lethal interactions are approved by the Food and Drug Administration or in clinical trials. Seeking to nominate additional synthetic lethal targets, we analyzed large-scale CRISPR knockout screening data and identified Pelota mRNA surveillance and ribosome rescue factor (PELO) as a promising therapeutic target for two independent and frequently observed molecular subtypes of cancer: biallelic deletion of chromosomal region 9p21.3 (9p21.3-/-) and microsatellite instability-high (MSI-H).
 Homozygous deletion of chromosomal region 9p21.3 is one of the most frequently observed somatic copy number alterations in human cancers, occurring in approximately 15% of all cancers. Many 9p21.3-/- cancers are associated with poor clinical outcomes including subsets of glioblastoma, mesothelioma, urothelial, pancreatic, esophageal, and non-small cell lung cancers. The primary driver of 9p21.3-/- is thought to be loss of the tumor suppressor cyclin-dependent kinase inhibitor 2A. Additionally, studies have linked the loss of the interferon gene cluster on 9p21.3 to immune evasion and primary resistance to immune checkpoint inhibitors. Extensive deletions of chromosomal regions can provide opportunities for cancer treatment. For instance, PRMT5 and MAT2A were found to be promising targets in 9p21.3-/- cancers due to synthetic lethal interactions with deletions of the 9p21.3 gene methylthioadenosine phosphorylase. While clinical trials are underway examining PRMT5 and MAT2A inhibitors, the frequency and poor outcomes of many 9p21.3−/− cancers underscores the urgent need for additional therapies to treat this diverse group of cancers. Using large-scale functional genomic datasets, we identified PELO as the top preferential dependency in 9p21.3-/- cell lines.
 We also observed that some cell lines with intact 9p21.3 (9p21.3+) were dependent on PELO for survival. Examining other genomic features, we found that these cells were characterized as MSI-H, a hypermutable state observed in subsets of colon, endometrial, gastric, and ovarian cancers. Indeed, when we compared 9p21.3+/MSI-H and 9p21.3+/microsatellite stable (MSS) cell lines, we found that PELO scored as a strong preferential dependency in MSI-H cell lines, second only to the previously described synthetic lethal target Werner helicase (WRN).
 To validate these findings, we interrogated the viability effects of PELO knockdown across a panel of cell lines representing 9p21.3+/MSS, 9p21.3-/-/MSS, or 9p21.3+/MSI-H. CRISPR interference (CRISPRi)-mediated PELO knockdown with three distinct guide RNAs impaired the viability of 9p21.3-/-/MSS cell lines and 9p21.3+/MSI-H cell lines, but not 9p21.3+/MSS cell lines. We also sought to validate PELO dependency in tumor organoid models for which pre-existing CRISPR data were unavailable, and therefore, we could only predict their dependence on PELO based on the two identified biomarkers. We demonstrated that 9p21.3-/- or MSI-H tumor organoids were preferentially dependent on PELO as compared to 9p21.3+/MSS models. We also sought to evaluate the effect of PELO knockdown in tumor maintenance in vivo. Leveraging a doxycycline-inducible CRISPRi system to suppress PELO expression, we showed that PELO knockdown significantly shrank 9p21.3-/- tumors grown on the flanks of nude mice.
 We next pursued the culprit lesion in 9p21.3-/- cancers that conferred PELO dependency. Hypothesizing that the loss of a 9p21.3 gene confers PELO dependency, we performed a focused loss-of-function CRISPR/Cas12a screen targeting 9p21.3 genes in the context of PELO knockdown or control conditions. We identified the mRNA surveillance gene FOCAD as a 9p21.3 gene whose knockout lead to preferential impairment in cells with PELO depletion. This led us to hypothesize that FOCAD deletions were conferring PELO dependency. In focused validation, we showed that FOCAD deletion sensitizes cell lines to PELO knockdown. Furthermore, we discovered that restoring FOCAD rescued 9p21.3-/- cells from PELO depletion. Together, these data demonstrated FOCAD loss was both necessary and sufficient for PELO dependency in the 9p21.3-/- context.
 Next, we sought to determine the mechanistic basis for PELO dependency in MSI-H cancers. We hypothesized that a mutation associated with MSI-H sensitizes cells to PELO loss. We observed that deletions in the splicing acceptor site at intron 29 of the superkiller complex (SKIc) member tetratricopeptide repeat domain 37 (TTC37) was the most correlated microsatellite mutation with PELO dependency. Thymidine deletions in this splicing acceptor site were predictive of decreased TTC37 protein levels in cell lines. These data led us to hypothesize that functional loss of TTC37 is the culprit lesion that renders MSI-H cancers dependent on PELO for survival. Evaluating this hypothesis, we found that TTC37 knockout sensitized cells to PELO depletion. We also showed that exogenous expression of TTC37, but not control, cDNA rescued the viability of MSI-H cell lines from PELO knockdown. Taken together, these data demonstrated that functional TTC37 impairment is sufficient and necessary for PELO dependency in the MSI-H setting and could serve as a refined predictive biomarker for PELO dependency.
 We next sought to investigate how FOCAD loss and TTC37 mutations converge to confer PELO dependency. TTC37 interacts with Ski2-like RNA helicase (SKIV2L) and WD Repeat Containing Protein 61 (WDR61) to form the SKIc, which promotes the 3’-5’ exosome degradation of mRNA from stalled ribosomes. We examined the Cancer Cell Line Encyclopedia proteomics dataset and demonstrated positive correlations between FOCAD, TTC37, and SKIV2L. Consistent with these datasets, we observed decreased TTC37 and SKIV2L protein levels in 9p21.3-/- cell lines and MSI-H cell lines. Based on these results, we asked if FOCAD was required and/or sufficient to maintain TTC37 and SKIV2L protein stability. We expressed FOCAD cDNA in 9p21.3-/- cells and observed increased TTC37 and SKIV2L protein levels. Furthermore, FOCAD knockout reduced SKIV2L and TTC37 protein levels in 9p21.3+/MSS cells. Together, these data support FOCAD as a critical regulator of SKIV2L and TTC37 stability. We hypothesized that FOCAD deletions and TTC37 mutations were capturing SKIc loss, thereby increasing reliance on PELO for survival. We reasoned that if functional loss of the SKIc was responsible for PELO dependency, SKIV2L deletion would also confer increased PELO dependency. We knocked SKIV2L out and observed that loss of SKIV2L sensitized cells to PELO loss. These data support our hypothesis that loss of SKIc function confers increased dependence on PELO for survival.
 To assess how SKIc-deficient cancer cells respond to PELO suppression, we performed gene expressing profiling. We found that SKIc-deficient cells had a robust transcriptional response with gene set enrichment analyses demonstrating upregulation of the unfolded protein response, a signaling network responding to aggregated unfolded or misfolded proteins. We validated these findings by demonstrating that PELO depletion preferentially upregulated C/ebp homologous protein (CHOP) expression and induced X-BOX binding protein (XPB1) splicing, markers of UPR activation, in the SKIc-deficient setting.
 Together, our observations reveal that MSI-H associated mutations and large 9p21.3 deletions involving TTC37 and FOCAD, respectively, independently impair the SKIc and confer a synthetic lethal relationship with PELO. Since MSI-H and large 9p21.3 deletions are frequently observed in patients, a PELO-based therapeutic could have broad implications for clinical oncology.


 Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel Lubicki, Ashir Borah, Nicholas Lofaso, Sohani Das Sharma, Riya V. Kishen, Joshua Dempster, Francisca Vazquez, Edmond M. Chan. Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY14-03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Oncogenic fusion genes and amplified oncogenes are common drivers of cancer but there are currently no therapeutic approaches available that directly target these cancer-specific genomic features. Amplified oncogenes in cancer can be found on chromosomes as homologous staining regions (HSRs) or on extrachromosomal DNA (ecDNA). Presence of oncogenes on ecDNA poses a particular challenge because they are under strong positive selection in response to anti-cancer therapies leading to treatment resistance as their copy number can rapidly change due to uncontrolled segregation during cell division. To specifically target cells with fusions or amplified oncogenes, we have developed a CRISPR-based therapeutic approach, “KLIPP, ” which is designed to target structural variants junctions (SVJs) specific to cancer genomes, with few, or no off-target effects expected in normal cell. KLIPP uses a “split enzyme” approach consisting of a dead Cas9 endonuclease (dCas9) fused to the endonuclease Fok1 where two Fok1 endonucleases need to homodimerize to become active. To “nucleate” and activate these complexes, sgRNAs are designed to bind sequences flanking cancer-specific SVJs, bringing two Fok1-dCas9 complexes together to induce double-strand breaks (DSBs). Using the KLIPP approach, we have obtained strong proof-of-concept that it can specifically target cancer-unique SVJs and induce DSBs leading to cell death in cancer cells both in culture and in vivo. We have shown this for SVJs throughout cancer genomes as well as in oncogenic fusion genes such as EWS-FLI1 in Ewing sarcoma. Furthermore, we show that we can use KLIPP to target unique SVJs on amplified oncogenes resulting in strong induction of γH2AX and loss of cell survival. We are currently investigating the efficacy of KLIPP in targeting ecDNA in cancer cells. We believe that KLIPP could be a safe and cancer-specific precision therapy for targeting cancer cells driven by fusions and amplified oncogenes.


 Huibin Yang, Radhika Suhas Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Venkata Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman. KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6371.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background
 Chromosomal instability (CIN) is pervasive during cancer evolution, particularly in non-small cell lung cancer (NSCLC) where it is associated with poor recurrence-free survival and correlates with the frequency of whole genome doubling events (WGD). By duplicating the complete set of chromosomes, WGD is a key event during cancer evolution and correlates with poor prognosis and targeted therapy resistance. Despite the importance of these events, genetic events responsible for the initiation and maintenance of CIN and WGD in NSCLC have not been systematically investigated.
 Methods
 TRAcking Cancer Evolution through Therapy/Rx (TRACERx) is a multicancer longitudinal study which utilizes multiregional sampling and whole exome sequencing (WES) to elucidate potential causes of tumor heterogeneity and cancer progression. TRACERx enables us to investigate genetic alterations and their relationship with WGD and CIN, and proceed to functionally characterize them.
 Results
 We identified that genetic alterations in 6 genes, including FAT1, results in homologous recombination (HR) repair deficiencies and CIN. Mutations in FAT1 are further associated with HR deficiency in patient cohorts. Using orthogonal genetic and experimental approaches, we demonstrate that FAT1 alterations are positively selected before WGD in lung cancer evolution. FAT1 ablation causes persistent replication stress, an elevated mitotic failure rate, nuclear deformation, and elevated structural CIN including chromosome translocations and radial chromosomes. Furthermore, FAT1 loss contributes to WGD, which we attribute to the dysregulation of YAP1. Co-depletion of the oncogene YAP1 partially reduces the elevated numerical CIN caused by FAT1 loss but does not relieve HR deficiencies nor structural CIN. Importantly, overexpression of constitutively active YAP15SA is sufficient to induce numerical CIN. HR deficiencies and high CIN caused by FAT1 loss synergizes with WGD to fuel tumor evolution, a prominent factor underpinning therapy resistance.
 Conclusions
 These data suggest that FAT1 loss in lung cancer attenuates HR repair and exacerbates structural and numerical CIN through two distinct downstream mechanisms, leading to increased tumor heterogeneity.


 Wei-ting Lu, Lykourgos-Panagiotis Zalmas, Chris Bailey, James R. Black, Carlos Martinez-Ruiz, Oriol Pich, Francisco Gimeno-Valiente, Ieva Usaite, Mariam Jamal-Hanjani, Nicolas Tapon, Kasper Fugger, Nicholas McGranahan, Kevin Litchfield, Nicolai J. Birkbak, Jiri Bartek, Nnennaya Kanu, Charles Swanton. FAT1 alterations contribute to chromosomal instability in cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1169.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Tumors deficient in homologous recombination (HRD) predominantly rely on alternative DNA repair pathways for maintaining genome stability, especially once end resection has been initiated after a double strand break (DSB). These back up repair mechanisms, especially microhomology-mediated end-joining (MMEJ) and single-strand annealing (SSA), once considered minor contributors to genome stability, have garnered increased attention as recent evidence supports their significant roles in tumorigenesis and their potential as therapeutic targets. Consequently DNA polymerase theta (POLQ), a key regulator of MMEJ, emerges as an attractive target for the selective treatment of HRD cancers, and recent efforts in the development of POLQ inhibitors have demonstrated their potential to enhance the efficacy of existing therapies and overcome resistance mechanisms. In our study, we investigated the genetic drivers of the under-investigated SSA and its connection to the related MMEJ pathway in human cells. A previously unidentified role for POLQ in regulating the frequency of SSA was observed in addition to its role in MMEJ. Beyond its established role as a gap-filling polymerase, we show that the helicase activity of POLQ is critical for homology-mediated annealing in both MMEJ and SSA. POLQ is shown to be a key facilitator of SSA through its ability to process DNA ends and promote annealing of homologous sequences with at least 80% sequence identity (homeology), resulting in a range of deletion sizes. We also provide evidence for these homeology-dependent SSA-like repair scars in tumor genomes and show that they are associated with HRD and elevated POLQ expression, suggesting the utilization of these mechanisms as a critical repair pathway in these tumors. These findings suggest POLQ is the ideal target for therapies of HRD cancer.


 Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, Nadeem Riaz, Marcin Imielinski, Simon N. Powell. POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1473.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e9ca7a5e65c5dcd9a8e77355be009d71f35f7d" target='_blank'>
              Abstract 1473: POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, N. Riaz, Marcin Imieliński, Simon N. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Adult T-cell leukemia/lymphoma (ATLL), a rare form of non-Hodgkin lymphoma, affects approximately 2-5% of individuals infected by the retrovirus Human T-cell Lymphotropic Virus- 1 (HTLV-1). The disease has a dismal prognosis and resistance to chemotherapy is commonly seen in most patients. A significant characteristic of ATLL pathogenesis involves the disruption of normal epigenetic regulation, which results in transcriptional repression of tumour suppressors, mediated by the polycomb repressor complex 2 protein. Within this protein complex, the histone methyltransferases EZH1 and EZH2 play vital roles in catalyzing the excessive trimethylation of histone 3 lysine 27 (H3K27me3). In light of these mechanisms, a phase II clinical trial evaluated the efficacy of Valemetostat (VAL), a dual EZH1/2 inhibitor, in ATLL patients. Half of the patients responded to VAL; however, the median progression-free survival was only 7 months.To address these challenges and unbiasedly identify agents that could synergize with VAL, we conducted extensive high-throughput drug screens across four ATLL cell lines (ATL1K, ATL43T-, ATL55, and ED41214+). We initially screened 3,113 FDA-approved drugs to assess their pharmacological relevance in the presence of VAL. Additionally, given the important role of epigenetic deregulation in the pathogenesis of ATLL, we also tested 68 epigenetic chemical probes (ECPs) from the Structural Genomics Consortium with and without VAL. From these screens, we narrowed down the top 98 candidates from the FDA-approved drugs, based on the average cell viability minus two standard deviations, and the top five synergistically effective ECPs based on Bliss syngery score calculated by Synergy Finder across all four cell lines. We next screened these hits from FDA-approved drugs and the synergistically effective ECPs along with the corresponding negative non-functional probes across a wider ten-point dose range(2.5nM to 1uM for FDA-approved drugs and 10nM to 5uM for ECPs), both in the presence or absence of VAL. Based on the observed synergy, we identified our top nine hits that included bromodomain and methyltransferase inhibitors. Finally, we tested these nine hits in 6x6 combination matrices to precisely determine the optimal dosing with VAL.Currently, we are expanding our validation across additional ATLL cell lines and optimizing dosage schedules for in vivo testing on our established ATLL models to ensure synergistic effective.This research aims to identify novel targets that will sensitize resistant ATLL to epigenetic therapies. These findings may contribute to developing strategies to address drug resistance and offer additional treatment opportunities for this difficult-to-treat disease.


 Noorhan Ghanem, Carman K.M. Ip, Kit Tong, Mehran Bakhtiari, Michael Y. He, Aaron Schimmer, Robert Kridel. Drug screens identify new therapeutic targets that synergize with EZH1/2 inhibition in adult T-cell leukemia/lymphoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4498.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ddc9edbf2bf1f968d70d06263a337e58c74158" target='_blank'>
              Abstract 4498: Drug screens identify new therapeutic targets that synergize with EZH1/2 inhibition in adult T-cell leukemia/lymphoma
              </a>
            </td>
          <td>
            Noor B Ghanem, C. Ip, Kit I Tong, Mehran Bakhtiari, Michael Y. He, A. Schimmer, R. Kridel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="
 Chemotherapy resistance remains a major challenge in clinical oncology. The mechanisms that regulate dynamic movement through adaptive cell states to survive through chemotherapeutic stress need to be understood to advance treatment options for patients and move towards curative therapies. In response to various stressors, cancer cells can reprogram their cell cycle, uncoupling DNA replication from mitotic division to acquire a large polyploid phenotype. This alternative cell cycle of repeated G and S phases is known as an endocycle, observed across the tree of life during both development and stress response. Cells exhibiting this phenotype exist in the transient endocycling cancer cell state. Endocycling cancer cells (ECCs) may serve as an actuator of chemotherapy resistance and disease recurrence. Understanding the mechanisms regulating the mitotic-to-endocycle switch and back are crucial to enhance treatments for patients with chemotherapy resistant cancer. A major driver of endocycling described in the literature is the transcription factor, c-Myc, due to its role in governing the G1/S transition. C-Myc is dysregulated in >70% of cancers, functioning as a critical cell cycle driver through transcriptional regulation of cyclins and CDKs, as well as essential metabolic programs necessary for cycle progression. Despite the published roles of Myc in both cancer and endocycling, we have shown through single cell RNA sequencing and western blotting that ECCs exhibit a low-Myc phenotype across multiple contexts. Myc protein levels can be rescued using the proteasome inhibitor, MG132, demonstrating that the low Myc status is driven by proteasomal degradation. To test if Myc degradation serves a mechanistic function, the small molecule inhibitor, MYCi975, was used to inhibit Myc activity at various points in the mitotic cell cycle. Our data shows that loss of Myc activity in cells beyond the restriction point induces mitotic bypass, generating a 4N G1 population. Notably, cells actively replicating their DNA in S phase show high sensitivity to Myc inhibition after skipping to G1, displaying cytotoxic effects not seen in other phases of the cell cycle. Similarly, when ECCs are challenged with the Myc inhibitor, the treatment promotes mitotic bypass and enriches for G0 and G1 populations, primed to enter the next S phase upon release from Myc inhibition. Together, these data suggest a temporal aspect of Myc regulation that has not been described in the literature. Myc activity beyond the restriction point appears to commit cells to mitotic entry, playing a functional role in coupling DNA replication to mitosis. This data supports our hypothesis that the low-Myc status of endocycling cells plays a mechanistic role in permitting mitotic bypass through premature loss of activity in G2. Preliminary data suggests that rescuing Myc activity will force mitotic entry producing mitotic progeny cells.


 Michael Loycano, Kenneth J. Pienta, Sarah R. Amend. Temporal regulation of c-Myc in endocycling cancer cells facilitates mitotic bypass in response to chemotherapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5413.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80e1849f5d2a4bf9cdd4d6129184c9b1ac64a97" target='_blank'>
              Abstract 5413: Temporal regulation of c-Myc in endocycling cancer cells facilitates mitotic bypass in response to chemotherapy
              </a>
            </td>
          <td>
            Michael A. Loycano, Kenneth J. Pienta, Sarah R Amend
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 ASXL1 is frequently mutated in myeloid malignancies, often causing a stabilization of ASXL1 and the polycomb-repressive deubiquitinase (PR-DUB) complex, that results in a global reduction in H2AK119Ub and increased mRNA transcription. Poorer prognosis of AML with mutant (mt) ASXL1 creates an unmet need to identify and test novel therapies for this AML sub-type. We generated two knock-in (KI) models of the clinically relevant mtASXL1 Y591* into the AML cell lines OCI-AML3 and UCSD-AML1. Notably, ASXL1/BAP1 stabilization caused an increased expression of c-Myc at both the RNA and protein level. We observed via ChIP-Seq reduced H2AK119Ub at H3K27Ac-marked chromatin on Wnt/β-catenin stem cell pathway genes that resulted in increased mRNA expression of FZD5, TCF7L2, and WNT5A. KI of mtASXL1 in OCI-AML3 cells also increased G0/G1 with a concomitant reduction in S and G2/M phases of the cell-cycle. RNA-Seq analysis showed reduced expression of the REACTOME cell-cycle mitotic gene-set, associated with depletion of E2F1, AURKA/B, and PLK1 mRNA. A CRISPR screen in parental and KI cells revealed SPI1, CDK9, and EP300 among the top dependencies specific to mtASXL1 AML. Notably, compared to parental cells, OCIAML3 ASXL1 Y591* cells exhibited reduced sensitivity to standard anti-AML, chemotherapeutic agents, including cytarabine, etoposide and daunorubicin. Our findings confirm the previously published discovery that the presence of mtASXL1 confers an increased sensitivity to BETi inhibitors, e.g., pelabresib or birabresib. These cells were also more sensitive to other epigenetically targeted agents, including HATi (GNE-781 or CBP-IN-1), CDK8/19 mediator kinase inhibitor (SEL120-34A/RVU120) and the dual HAT/BETi (NEO2734/EP31670). Treatment with NEO2734 reduced the permissive chromatin marks especially on c-Myc and SPI1 super enhancers. There was also a significant reduction in mRNA and protein expressions of c-Myc, c-Myb, and PU.1. RNA-seq and mass-spectrometric analyses revealed that treatment with NEO2734 negatively enriched gene sets of c-Myc targets, rRNA processing, mRNA splicing, and translation. Treatment with SEL120 globally reduced active chromatin marks in the mtASXL1 expressing AML cells. This was associated with reduced protein expression of BAP1, pSTAT1 (S727), pSTAT5 (S726), MEIS1 and HOXA9, exclusively in mtASXL1 expressing cells. Co-treatment with pelabresib and NEO2734 or SEL120 and NEO2734 synergistically induced apoptosis in mtASXL1-expressing cells. Importantly, in the NSG mice engrafted with OCIAML3 ASXL1 Y591*, monotherapy with NEO2734, pelabresib or SEL120 significantly reduced AML burden. Collectively these findings highlight previously uncharacterized biologic effects of the presence of mtASXL1 and support the rationale for further evaluating AML therapies incorporating BETi, HAT-BETi or mediator-kinase inhibitor.


 Christopher P. Mill, Warren C. Fiskus, Christine E. Birdwell, John A. Davis, Kaberia Das, Hanxi Hou, Surbhi Sharma, Koji Sasaki, Sanam Loghavi, Tapan M. Kadia, Naval Daver, Courtney D. DiNardo, Lu Wang, Kapil N. Bhalla. ASXL1 mutations in AML are associated with a distinct epigenetic state that results in vulnerabilities to epigenetic-targeted agents [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 444.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631764abf56c15adfa48bcb7f3a761557f103393" target='_blank'>
              Abstract 444: ASXL1 mutations in AML are associated with a distinct epigenetic state that results in vulnerabilities to epigenetic-targeted agents
              </a>
            </td>
          <td>
            Christopher P. Mill, W. Fiskus, Christine E. Birdwell, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, K. Sasaki, S. Loghavi, T. Kadia, Naval G Daver, C. Dinardo, Lu Wang, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Pooled CRISPR screens are vital in the unbiased interrogation of gene function and are instrumental in uncovering therapeutic targets and biological processes. However, follow-up hit validation is critical to confirm observed results. Researchers need a simple and robust approach to rapidly verify putative hits and test resulting observations. Thus, we developed a CRISPR-based method for hit validation that tests the effect of a genetic perturbation on cell fitness. By editing target loci and monitoring the indel profiles over time, we have created a Cellular Fitness (CelFi) assay that can elucidate cellular vulnerabilities and verify hits from pooled CRISPR knockout screens. Unlike traditional cellular fitness assays that evaluate viability over time, the CelFi assay correlates changes in the indel profile at the target gene with a selective growth advantage or disadvantage in individual cells over time. Moreover, the CelFi assay can be utilized to evaluate gene dependencies and test new hypotheses, regardless of variations in single guide RNA optimization, ribonucleoprotein concentration, and gene copy number. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-96095-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bfe3c00eb1f10374c9c0ac1481d5794cd3fb46b" target='_blank'>
              Rapid and robust validation of pooled CRISPR knockout screens using CelFi
              </a>
            </td>
          <td>
            Allister J Loughran, S. Narina, Jonathon Klein, Jamaica F. Siwak, Jon P Connelly, S. Pruett-Miller
          </td>
          <td>2025-04-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Squamous cell carcinomas (SCCs) arise from epithelial tissues in the lung, head and neck, esophagus, bladder, and cervix. SCCs lack canonical oncogenic mutations and urgently need targeted therapies. Aneuploidy—chromosomal losses or gains, is rare and poorly tolerated in normal cells but occurs in 90% of tumors in tissue-specific patterns. SCCs are characterized by near-universal early chromosome 3p deletion and 3q gain, which correlate with poor prognosis and metastasis. To study effects of 3p deletion in human cells, we used CRISPR-Cas9 to delete a copy of 3p in human immortalized lung epithelial cells, representing the basal cell (lung SCC cell-of-origin). In a subset of clones with 3p deletion, we observed whole chromosome duplication of the unaltered chromosome 3 copy. As a result, these cells transitioned to a state of 3q gain, providing us models for 3p loss, 3q gain, and 3 disomy.
 To uncover vulnerabilities associated with chromosome 3 arm aneuploidies, we performed drug-repurposing and CRISPRi screens in our engineered cells. In 3q gain cells, the top genetic hit was sterol regulatory element-binding factor 1 (SREBF1), a key regulator of cholesterol and fatty acid biosynthesis. Compared to cells without 3q gain, 3q gain cells were more sensitive to knockdown of SREBF1, co-factor SCAP, and downstream target HMG-CoA reductase (HMGCR), the rate-limiting enzyme in cholesterol synthesis. HMGCR inhibitors (several statins, including mevastatin) were top chemical hits, inducing higher cell death in 3q gain cells. Chemical inhibition of SREBF1 using fatostatin recapitulated effects seen with genetic knockdown. Rescue with mevalonate or geranylgeranyl pyrophosphate (GGPP, a product of the mevalonate pathway involved in protein prenylation) demonstrated the toxicity of pathway inhibition are on-target. Transcriptomic and lipidomic profiling suggest that increased statin sensitivity of 3q gain cells is due to their reduced activation of SREBF1 and HMGCR, causing increased cell death upon inhibition of the mevalonate pathway. Ongoing mechanistic studies will investigate how aneuploidy alters lipid metabolism and contributes to cancer.
 Statins have been shown to induce tumor-specific apoptosis and reduce cancer risk, though these effects can be highly variable and context-dependent. Preliminary studies in organoid cultures and SCC cancer cell lines suggest that 3q gain is a biomarker of statin response and sensitivity in SCCs. Mevastatin and fatostatin co-treatment, inhibition at two points in the mevalonate pathway, showed synergy, suggesting statin potentiation that could be beneficial in cancer therapy. Additionally, 3q gain cells showed increased toxicity in response to other cholesterol-lowering compounds. Understanding aneuploidy-induced vulnerabilities will reveal how aneuploidies promote tumorigenesis and help identify precision medicine targets for SCCs with 3q gain.


 Nadja Zhakula, Sejal Jain, Laura Byron, Joan J. Castellano, Matthew L. Meyerson, Alison M. Taylor. Uncovering metabolic dysregulation and therapeutic targets in aneuploid models of squamous cell carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2992.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce10ef6d68c6903f3dcfc0aa9cd8d7cbbe833212" target='_blank'>
              Abstract 2992: Uncovering metabolic dysregulation and therapeutic targets in aneuploid models of squamous cell carcinoma
              </a>
            </td>
          <td>
            Nadja Zhakula, Sejal Jain, Laura Byron, Joan J. Castellano, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Immune checkpoint inhibitors are cancer therapeutics that have shown remarkable success in extending lives in many cancers, including melanoma, MSI-high cancers, and other cancers. However, these therapeutics have not shown benefit for many patients with cancer, especially those with advanced cancer diagnoses. In addition, many patients develop resistance to these therapeutics and/or life-altering adverse events that can include cardiotoxicity, pneumonitis, thyroiditis, pancreatitis, and hepatitis. Extensive efforts to improve cancer care by uncovering mechanisms of resistance to immune therapy in solid tumors have led to identification of new sources of resistance and to the development of new approaches to activate or sustain antitumor immunity. Chronic stimulation of T cells by tumors and by checkpoint inhibitors can lead to a progressive state of T-cell exhaustion. Chronic T-cell activation by the tumor microenvironment (TME) or immune therapeutics can upregulate the expression and function of alternate checkpoints, including the T-cell protein LAG-3. Persistent interferon signaling in the TME can drive epigenetic changes in cancer cells that enable tumors to counter immune activation and disrupt tumor cell elimination. In addition, immune-suppressive macrophages can flood tumors in response to signals from dying tumor cells, further preventing effective immune responses. New clinical developments and/or approvals for therapies that target alternate immune checkpoints, such as the T-cell checkpoint LAG-3; myeloid cell proteins, such as the kinase phosphoinositide 3-kinase gamma isoform; and chronic interferon signaling, such as Jak 1 inhibitors, have been approved for cancer care or shown promise in recent clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0773d2ad13d0927cca67db552652c316f54537" target='_blank'>
              One Step Ahead: Preventing Tumor Adaptation to Immune Therapy.
              </a>
            </td>
          <td>
            Erica L Braverman, Giuliana P Mognol, A. Minn, D. A. Vignali, Judith A. Varner
          </td>
          <td>2025-06-01</td>
          <td>American Society of Clinical Oncology educational book. American Society of Clinical Oncology. Annual Meeting</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Colon adenocarcinoma (COAD) is the most frequently occurring type of colon cancer. Cancer-associated fibroblasts (CAFs) are pivotal in facilitating tumor growth and metastasis; however, their specific role in COAD is not yet fully understood. This research utilizes single-cell RNA sequencing (scRNA-seq) to identify and validate gene markers linked to the malignancy of CAFs. Methods ScRNA-seq data was downloaded from a database and subjected to quality control, dimensionality reduction, clustering, cell annotation, cell communication analysis, and enrichment analysis, specifically focusing on fibroblasts in tumor tissues compared to normal tissues. Fibroblast subsets were isolated, dimensionally reduced, and clustered, then combined with copy number variation (CNV) inference and pseudotime trajectory analysis to identify genes related to malignancy. A Cox regression model was constructed based on these genes, incorporating LASSO analysis, nomogram construction, and validation.Subsequently, we established two FNDC5-knockdown cell lines and utilized colony formation and transwell assays to investigate the impact of FNDC5 on cellular biological behaviors. Results Using scRNA-seq data, we analyzed 8,911 cells from normal and tumor samples, identifying six distinct cell types. Cell communication analysis highlighted interactions between these cell types mediated by ligands and receptors. CNV analysis classified CAFs into three groups based on malignancy levels. Pseudo-time analysis identified 622 pseudotime-related genes and generated a forest plot using univariate Cox regression. Lasso regression identified the independent prognostic gene FNDC5, which was visualized in a nomogram. Kaplan-Meier survival analysis confirmed the prognostic value of FNDC5, showing associations with T stage and distant metastasis. In vitro experiment results demonstrated a strong association between FNDC5 expression levels and the proliferative, migratory, and invasive abilities of colon cancer cells. Conclusion We developed a risk model for genes related to the malignancy of CAFs and identified FNDC5 as a potential therapeutic target for COAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb7d8e81c3f345d4ada707194f030cd48f14acb" target='_blank'>
              The malignant signature gene of cancer-associated fibroblasts serves as a potential prognostic biomarker for colon adenocarcinoma patients
              </a>
            </td>
          <td>
            Hao Zhang, Zirui Zhuang, Li Hong, Ruipeng Wang, Jinjing Xu, Youyuan Tang
          </td>
          <td>2025-04-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a highly heterogeneous hematological malignancy characterized by diverse genetic and epigenetic aberrations, making therapeutic interventions particularly challenging. We have identified a novel combination therapy that leverages differences in cell cycle dynamics to induce AML cell differentiation, and found AML cells which were highly sensitive to inhibitors of the histone demethylase LSD1/KDM1A depending on their cell cycle profile. Here, we aim at identifying other potential epigenetic vulnerabilities of AML cell lines exposed to the treatment with different CDK inhibitors. Given the heterogeneity of the epigenetic aberrations found in AML, targeting additional chromatin regulators could improve the clinical applicability of this combination therapy. Thus, we started an in vitro CRISPR-screening approach, targeting over 600 chromatin-related genes. Following optimization of Cas9-expressing clones and evaluation of knockout (KO) efficiency, we confirmed that the best-performing clone retained phenotypic consistency with parental cells under combination therapy. Preliminary data validated the effectiveness of our screening platform, as sgRNAs targeting LSD1 recapitulated the phenotype observed with CDK inhibitors treatment, thereby demonstrating the feasibility of the approach. In parallel, we investigated the molecular link between cell cycle manipulation and chromatin remodeling. We conducted a nuclear phospho-proteomics experiment on cells treated with CDK inhibitors. This analysis identified differential phosphorylation of several proteins, which are currently under investigation. Additionally, gene ontology analysis revealed enrichment in pathways regulating RNA metabolism, transcription, and DNA damage response. By integrating cell cycle modulation with epigenetic targeting, our work offers new insights into AML differentiation therapy, with the potential to improve therapeutic outcomes in diverse AML subtypes.


 Riccardo Valzelli, Kourosh Hayatigolkhatmi, Amir Hosseini, Elena Ceccacci, Giorgia Parodi, Angela Bachi, Saverio Minucci. Leveraging cell cycle to enhance differentiation therapy in acute myeloid leukemia [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 454.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8bc96152bbbb1bdeac5241e7bd6de510c434bdc" target='_blank'>
              Abstract 454: Leveraging cell cycle to enhance differentiation therapy in acute myeloid leukemia
              </a>
            </td>
          <td>
            Riccardo Valzelli, Kourosh Hayatigolkhatmi, Amir Hosseini, E. Ceccacci, Giorgia Parodi, A. Bachi, S. Minucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 Renal cell carcinoma (RCC) is a heterogeneous malignancy with complex genetic underpinnings. While short-read sequencing has provided valuable insights into RCC genomics, the detection of large and complex structural variants (SVs) remains limited. In this study, we analyzed 17 pairs of RCC samples and identified 555 somatic SVs in RCC tumor samples using both long-read and short-read sequencing. All somatic SVs except repeat expansions are singleton variants, indicating high inter-tumor heterogeneity. Repeat expansions are recurrent, with the GAAGGG expansion in intron 7 of ARHGAP19 occurring with the highest prevalence of 64.29%. Next, to explore the potential impact of SVs on gene expression and chromatin accessibility, we generated snMultiome and bulk transcriptome data. We found that >25% SV-related genes exhibited significant changes in either gene expression or chromatin accessibility. The expression comparisons among mutant samples, non-mutant RCCs, and normal-adjacent kidneys identified putative protumor SVs (e.g. MET exon 14-15 duplication induced expression elevation in pRCC) and protective SVs (e.g. VCAN deletion induced expression loss in ccRCC). By integrating spatial transcriptome data, we investigated the spatial expression variations of SV-related genes and linked them to the clonal status of SVs. The most representative example is the differential MET expression observed in tumor cells within vascular cores and those lining the papillae membranes, suggesting the subclonal nature of MET duplication in pRCC. Finally, we constructed an RCC-specific pangenome to enhance SV calling and genotyping. Taken together, our results provide a comprehensive landscape of somatic SVs in RCC, shedding light on their effects on epigenomic and transcriptomic alterations. This study enhances our understanding of the role of SVs in RCC development and offers new insights into their potential clinical significance.


 Yuwei Zhang, Kapur Dhami, Yize Li, Cody Weimholt, Wagma Caravan, John Herndon, Reyka Jayasinghe, Song Cao, Melissa Reimers, Russell Pachynski, Feng Chen, Li Ding. Multi-modal profiling reveals the role of somatic structural variations in renal cell carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6621.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c34b2c44db5e2413af90c06943f2d16b821f4c" target='_blank'>
              Abstract 6621: Multi-modal profiling reveals the role of somatic structural variations in renal cell carcinoma
              </a>
            </td>
          <td>
            Yuwei Zhang, Kapur B. Dhami, Yize Li, Cody Weimholt, Wagma Caravan, John M Herndon, R. Jayasinghe, Song Cao, Melissa A. Reimers, Russell Pachynski, Feng Chen, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Drug-tolerant persister cells (DTPs) are a subpopulation of cancer cells that survive initial anticancer drug treatment despite lacking genetic resistance-driving alterations. DTPs have been shown to serve as an intermediate stage in the evolution of acquired drug resistance, facilitating the adaptation and survival of cancer cells under sustained therapeutic pressure. Recent studies have shed light on the biological characteristics of DTPs during and following drug treatment, describing the various adaptations that cells undergo while transforming into a DTP state. However, little is known about the biology of persistence potential — the cellular and molecular characteristics that prime certain drug-naïve cells to survive initial drug exposure. The main challenge in studying the persistence potential of drug-naïve cells lies in the need to treat cells to determine whether they will persist. To address this issue, we demonstrated the hereditary nature of persistence potential and adopted a high-resolution lineage tracing technology, to develop an assay for the study of persistence potential. This method, utilizing CRISPR-targeted evolving barcodes to reconstruct a lineage tree from a single-cell-derived clone, allows for the placement of persister cells along drug-naïve cells on the lineage tree. Utilizing this technology to study persistence to EGFR inhibitors in EGFR-positive non-small-cell lung cancer cells, we identified events on the lineage tree that alter the persistence potential of specific clades and derived the persistence potential of drug-naïve cells. Combining single-cell RNAseq data with cell persistence potential, we developed a computational pipeline to study the molecular mechanisms that govern the probability of drug-naïve cells to persist through drug treatment. We demonstrated that upregulation of genes related to oxidative phosphorylation correlates with a significant increase in persistence potential. These results agree with previous studies that demonstrate that cells in the DTP state are less dependent on glycolysis and rely more on oxidative phosphorylation for energy production and suggests that drug-naïve cells with elevated persistence potential are primed towards a high oxidative phosphorylation state. Intriguingly, we also described a module of translation-related genes that are negatively correlated with persistence potential, in agreement with recent results that show that persister cells undergo reversible remodeling of mRNA translation. This work introduces a novel approach and computational framework to study persistence potential and allows for the identification of potential drivers of persistence. A better understanding of the unique cellular and molecular traits that drive persistence potential is crucial for the study of drug persistence and development of targeted therapies preventing persistence and enhancing the likelihood of curative treatment.


 Binyanim Zhitomirsky, Jideofor Ezike, Brian P. Danysh, Gad Getz. High resolution lineage tracing for the study of cancer drug persistence [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5543.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5520257da55eb20975e2b76db4a10f023d4be1" target='_blank'>
              Abstract 5543: High resolution lineage tracing for the study of cancer drug persistence
              </a>
            </td>
          <td>
            Binyanim Zhitomirsky, Jideofor Ezike, B. Danysh, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 Melanoma, a highly heterogeneous cancer, evolves through a complex interplay of genetic alterations, including both single nucleotide variants (SNVs) and structural variants (SVs). To study the evolutionary trajectory of melanoma, we established a model system composed of 24 single-cell-derived clonal sublines (C1-C24) from the M4 melanoma model, developed in a genetically engineered hepatocyte growth factor (HGF)-transgenic mouse. While SNVs have been extensively used to construct phylogenetic trees using Trisicell (Triple-toolkit for single-cell intratumor heterogeneity inference), a tool that analyzes intratumor heterogeneity and single-cell RNA mutations, the role and timing of SVs in melanoma evolution remain less well understood. This study integrates SV data with an SNV-driven phylogeny to investigate whether SV patterns align with SNV-based evolutionary trajectories in the mouse melanoma model, providing insights into the functional impact of SVs during tumor progression. We performed long-read sequencing on the 24 clonal sublines and detected SVs using Severus, a tool optimized for phasing in long-read sequencing. The SVs were mapped to the SNV-driven phylogeny using R and classified as either concordant (aligning with the SNV-based tree) or discordant (deviating from the SNV phylogeny). Gene ontology enrichment analysis revealed that concordant SVs were significantly enriched in genes associated with the hepatocyte growth factor receptor signaling pathway and the negative regulation of peptidyl-threonine phosphorylation, both of which represent core drivers of tumor progression. In contrast, discordant SVs were associated with a broader range of functional pathways, including the positive regulation of antigen receptor-mediated signaling and the regulation of natural killer cell-mediated cytotoxicity, though the exact mechanisms underlying these associations remain unclear. By integrating these SVs with an established SNV-driven phylogeny, this study highlights the distinct and critical roles SVs play in melanoma evolution. Concordant SVs appear to drive core oncogenic processes, while discordant SVs may contribute to other aspects of tumor evolution. These findings underscore the importance of considering SVs alongside SNVs to fully capture the complexity of melanoma evolution. Ongoing investigations will continue to explore the functional implications of these SVs and how the gene disruption patterns they cause shape the evolutionary trajectory of melanoma, offering potential targets for future therapeutic strategies.


 Xiwen Cui, Ayse G. Keskus, Salem Malikic, Yuelin Liu, Anton Goretsky, Chi-Ping Day, Farid R. Mehrabadi, Mikhail Kolmogorov, Glenn Merlino, S. Cenk Sahinalp. Integrating structural variants and single nucleotide variants to uncover evolutionary trajectories in melanoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3898.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e8e473fd0d41652458c66d5982b1c03c7dc5d7e" target='_blank'>
              Abstract 3898: Integrating structural variants and single nucleotide variants to uncover evolutionary trajectories in melanoma
              </a>
            </td>
          <td>
            Xiwen Cui, A. Keskus, S. Malikić, Yuelin Liu, Anton Goretsky, Chi-Ping Day, F. Mehrabadi, M. Kolmogorov, Glenn Merlino, S. C. Sahinalp
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="CRISPR–Cas9 is a powerful genome-editing technology that can precisely target and cleave DNA to induce double-strand breaks (DSBs) at almost any genomic locus. While this versatility holds tremendous therapeutic potential, the predominant cellular pathway for DSB repair—non-homologous end-joining (NHEJ)—often introduces small insertions or deletions that disrupt the target site. In contrast, homology-directed repair (HDR) utilizes exogenous donor templates to enable precise gene modifications, including targeted insertions, deletions, and substitutions. However, HDR remains relatively inefficient compared to NHEJ, especially in postmitotic cells where cell cycle constraints further limit HDR. To address this challenge, numerous methodologies have been explored, ranging from inhibiting key NHEJ factors and optimizing donor templates to synchronizing cells in HDR-permissive phases and engineering HDR-enhancing fusion proteins. These strategies collectively aim to boost HDR efficiency and expand the clinical and research utility of CRISPR–Cas9. In this review, we discuss recent advances in manipulating the balance between NHEJ and HDR, examine the trade-offs and practical considerations of these approaches, and highlight promising directions for achieving high-fidelity genome editing in diverse cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ccb815e547c69027d6ce30c697c1185bdc578f" target='_blank'>
              Fine-Tuning Homology-Directed Repair (HDR) for Precision Genome Editing: Current Strategies and Future Directions
              </a>
            </td>
          <td>
            Sibtain Haider, C. Mussolino
          </td>
          <td>2025-04-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758af464be2f5fe707681ad668cca7d960076b75" target='_blank'>
              Targetable BIRC5 dependency in therapy-resistant TP53 mutated acute myeloid leukemia
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, Fang Qi Lim, Yang Mi, E. Olesinski, Cheryl Gek Teng Chan, S. Jasdanwala, Xiao Xian Lin, Yichen Wang, Jordan Yong Ming Tan, K. Bhatia, Valeriia Sapozhnikova, Chuqi Wang, Aarthi Nivasini Mahesh, Daniel Tan En Liang, Nishtha Chitkara, P. Mertins, L. Hogdal, Brandon D Brown, Torsten Haferlach, Camille Lobry, Coleman Lindsley, Alexandre Puissant, Han Kiat Ho, Sanjukta Das, A. Letai, S. Kornblau, J. Krönke, Edward Ayoub, Koji Itahana, Michael Andreeff, Shruti Bhatt
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Chromothripsis, a phenomenon of massive genomic rearrangements, introduces extensive mutations in critical genes, affecting cell survival and apoptosis. Among these genes, the BCL2 (B-cell lymphoma 2) gene, which plays a crucial antiapoptotic role in cancer cells, is often subjected to significant alterations. Here, we present a computational pipeline to model and analyze the structural and functional impacts of chromothripsis-induced Single-Nucleotide Polymorphisms (SNPs) within the BCL2 gene. This pipeline integrates mutation simulation, homology modeling, and protein inter-action analysis to evaluate the stability and apoptotic potential of BCL2 mutations. These results indicate that chromothripsis-induced mutations can destabilize the Bcl-2 protein, thereby disrupting its binding affinity with apoptotic regulators, such as Bax. These findings support the potential of ergodic anticancer therapy to exploit such mutations, facilitating the apoptosis of cancer cells. Our computational model offers a novel in silico approach for understanding mutation-driven alterations in cancer biology, aiding the development of therapeutic strategies targeting apoptotic pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8abc547c111e4d9c763f84a8f9da96c16d5a06c5" target='_blank'>
              Breaking Boundaries in Cancer Therapy: Harnessing Chromothripsis- Induced Mutations for Targeted BCL2 Protein Destabilization.
              </a>
            </td>
          <td>
            Sergey Shityakov, Michael Nosonovsky, Ekaterina V. Skorb, Viacheslav Kravtsov
          </td>
          <td>2025-06-02</td>
          <td>Current drug discovery technologies</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Nonsense mutations in gene coding regions introduce an in‐frame premature termination codon (PTC) in the mRNA transcript, resulting in the early termination of translation and the production of a truncated, nonfunctional protein. The absence of protein expression and the consequent loss of essential cellular functions are responsible for the severe phenotypes in the so‐called genetic nonsense‐related diseases (NRDs), such as cystic fibrosis, hemophilia, Duchenne muscular dystrophy, Fabry disease, Choroideremia, Usher syndrome, Shwachman–Diamond syndrome, and even certain types of cancer. Nonsense mutations pose a significant challenge in the treatment of NRDs, as a specific approach directly addressing this genetic defect is currently unavailable. Developing new therapeutic strategies for nonsense suppression is a crucial goal of precision medicine. This review describes some of the most promising therapeutic approaches and emerging strategies for treating NRDs. It considered both the use of small molecules to interfere with molecular mechanisms related to nonsense mutations, such as translational readthrough‐inducing drugs (TRIDs) or inhibitors of the nonsense‐mediated decay (NMD) pathway, and also innovative approaches involving nucleic acids, such as gene editing, anticodon engineered‐tRNA (ACE‐tRNA), or mRNA‐based therapy. Future research should focus on refining these approaches and exploring integrated and personalized treatments to enhance therapeutic outcomes and ensure continuous improvement in the quality of care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a20c54becc5991f4ba9e63eae1c1a43ac7c817" target='_blank'>
              Advancing Therapeutic Strategies for Nonsense‐Related Diseases: From Small Molecules to Nucleic Acid‐Based Innovations
              </a>
            </td>
          <td>
            Davide Ricci, I. Cruciata, I. Fiduccia, Emanuele Vitale, Federica Corrao, A. Branchini, P. Carollo, I. Pibiri, L. Lentini
          </td>
          <td>2025-05-01</td>
          <td>Iubmb Life</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Patient derived xenografts (PDX) are important preclinical models of cancer as they faithfully recapitulate many aspects of human tumors. Currently, models derived from patients of non-European genetic ancestry are underrepresented, limiting the potential of precision medicine advancements to equitably improve patient outcomes. The United for Health Equity Living PDX Program (U4HELPP) seeks to address this need by generating new PDX models representative of the diverse population served by the VCU Health System. This study aims to 1) characterize the histologic, genomic, and transcriptomic features of a new cohort of ancestrally diverse PDX models of breast, pancreatic, colon, and lung cancer, and 2) identify drug sensitivities and response markers that will be targeted in ongoing studies utilizing these models.



 Tumor fragments were implanted in the bilateral mammary glands or subcutaneous flank of NSG mice, and growth rates were defined. Tissue was fixed and embedded for histological staining to identify the cancer type and subtype. Whole genome and bulk RNA sequencing were performed to assess the mutational profile, genetic ancestry, and global transcriptomic landscape. Single cell RNA sequencing was utilized to capture the transcriptomic heterogeneity within each model. Finally, high throughput drug screening with the NCI NExT Oncology Interrogation Tools Set of 555 agents was performed in vitro to assess the drug sensitivities of each PDX.



 Whole genome sequencing and bulk RNA sequencing positively identified distinct genetic ancestries within the cohort. Histopathologic analysis revealed a rare metaplastic breast carcinoma, and immunohistochemical staining stratified breast cancer models into estrogen receptor positive and triple negative subtypes. Bulk RNA sequencing was used to subtype pancreatic ductal adenocarcinoma models as basal-like, classical, or intermediate, and single-cell RNA sequencing confirmed the presence of mixed subtype populations within these samples. High throughput drug screening identified sensitivities that correlated with mutations detected through whole genome sequencing; overall, several classes of drugs including proteasome and cyclin-dependent kinase inhibitors were found to be highly cytotoxic in all cancer types tested.



 This study integrates multiple modalities to characterize the histological features, molecular signatures, and cellular heterogeneity of this new cohort of PDXs, which will provide researchers with extensive data on which to base future studies utilizing these models. Furthermore, it identifies promising therapeutic agents and explores genomic markers of drug sensitivity which could be used to predict patient response to novel treatments.



 Nina M. Dashti-Gibson, Amy L. Olex, Katarzyna M. Tyc, Carson J. Walker, Oniya M. Smith, Rachel K. Myrick, Bin Hu, Praveen Bhoopathi, Madhavi Puchalapalli, Li Wang, Yue Zhang, Brydon P. Wall, Valentina Robila, Mikhail G. Dozmorov, Jinze Liu, Jennifer E. Koblinski, Jose G. Trevino, J Chuck Harrell. Genetic, genomic, and pharmacologic characterization of a new cohort of ancestrally diverse patient-derived xenograft models of cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 49.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5507e82127e26407851957911f722cad0e285824" target='_blank'>
              Abstract 49: Genetic, genomic, and pharmacologic characterization of a new cohort of ancestrally diverse patient-derived xenograft models of cancer
              </a>
            </td>
          <td>
            Nina M. Dashti-Gibson, Amy Olex, Katarzyna M Tyc, Carson J Walker, Oniya M. Smith, Rachel K. Myrick, Bin Hu, Praveen Bhoopathi, Madhavi Puchalapalli, Li Wang, Yue Zhang, Brydon P. G. Wall, Valentina Robila, Mikhail G. Dozmorov, Jinze Liu, Jennifer E Koblinski, Jose G. Trevino, J. Harrell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Aneuploidy, a hallmark of cancer, leads to widespread changes in chromosome copy number, altering the abundance of hundreds or thousands of proteins. How-ever, evidence suggests that levels of proteins encoded on affected chromosomes are often buffered toward their abundances observed in diploid cells. Despite its preval-ence, the molecular mechanisms driving this protein dosage compensation remain largely unknown. It is unclear whether all proteins are buffered to a similar degree, what factors determine buffering, and whether dosage compensation varies across different cell lines or tumor types. Moreover, its potential adaptive advantage and therapeutic relevance remain unexplored. Here, we established a novel approach to quantify protein dosage buffering in a gene copy number-dependent manner, show-ing that dosage compensation is widespread but variable in cancer cell lines and in vivo tumor samples. By developing multifactorial machine learning models, we identify mean gene dependency, protein complex participation, haploinsufficiency, and mRNA decay as key predictors of buffering. We also show that dosage com-pensation can affect oncogenic potential and that higher buffering correlates with reduced proteotoxic stress and increased drug resistance. These findings highlight protein dosage compensation as a crucial regulatory mechanism and a potential therapeutic target in aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 p53 and Rb co-mutations are common in several human cancers and are associated with the onset of aggressive tumors with complex karyotypes. Canonically, these tumor suppressors respond to DNA damage and cellular stress by inducing cell cycle arrest, activating pro-apoptotic and anti-proliferative signalling, and promoting cellular senescence. While these pathways are sufficient to partially explain the tumor suppressive abilities of p53 and Rb, recent studies in genetically engineered mouse models suggest additional, non-canonical means of tumor suppression, including modulation of cellular differentiation and DNA topology, and interestingly, influencing cellular response to chromosomal instability. Chromosomal instability (CIN) is a hallmark of aggressive, therapy-resistant cancers and is associated with mitotic dysfunction, faulty DNA damage responses, immune evasion, and cellular stress response pathways. While it is known that p53 and Rb maintain the integrity of the genomic landscape, the mechanisms by which they prevent CIN remain understudied. Here, we use mouse embryonic fibroblasts (MEFs) to show that p53 + Rb double knock-out (DKO) causes a dramatic increase in ploidy at very early passages, which stabilizes at tetraploid levels. These cells also harbour high levels of CIN, indicated by the presence of micronuclei, DNA double-stranded breaks, and lagging chromosomes. Additionally, we observed CIN in p5325, 26 transactivation domain (TAD) mutant MEFs that are unable to activate most canonical p53 targets yet retain the ability to suppress tumorigenesis in vivo, raising the possibility that the maintenance of chromosomal stability is dispensable to the tumor suppressive functions of p53. Using a combination of bulk and scRNA sequencing of aneuploid and diploid DKO MEFs, in future experiments we will identify p53- and Rb-dependent changes in gene expression associated with the onset and progression of CIN. Overall, the results of this study are expected to provide critical insights into the molecular mechanisms by which tumor suppressors p53 and Rb work together to repress CIN.


 Risha Banerjee, David G. Kirsch. Mechanisms promoting chromosomal instability following the loss of p53 and Rb [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2837.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T. Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 e15180


 Background:
 Pancreatic cancer has apoor prognosis and limited therapeutic options. While
 K-RAS
 mutations are prevalent in this malignancy, a subset of patients harboring wild-type
 K-RAS
 presents distinct clinical behavior. Gene fusions have been discerned in pancreatic cancer and are promising candidates for administering targeted therapies.
 Methods:
 We conducted a retrospective analysis of 926 pancreatic cancer patients of which 165 (18%) carried a wild-type
 K-RAS
 gene, by next generation sequencing (NGS) or polymerase chain reaction (PCR) testing. Tumors were analyzed with a clinical-grade hybrid capture-based RNA-seq clinical fusionome test, evaluating gene fusions between 1104 genes or to other known or novel partners as well as gene expression transcript analysis. Single nucleotide variants/insertions deletions (SNV/Indels) were tested with a clinical-grade NGS assay targeting > 300 cancer genes.
 Results:
 Gene fusions were detected in 32% of
 K-RAS
 wild-type pancreatic cancers, with 75% involving known oncogenic, clinically actionable genes including
 FGFR
 (two
 FGFR2
 and one
 FGFR1
 fusion), two
 NTRK3
 and one each of
 RAF1, RET, ROS1
 and
 BRAF
 fusions. Deleterious SNVs were frequently found in both fusion-positive and fusion-negative
 K-RAS
 wild-type pancreatic cancers. Transcriptomic expression revealed additional potentially druggable targets, e.g.,
 ROS1
 and
 FGFR3
 overexpression in a tumor harboring an
 NTRK3
 fusion,
 BRAF
 overexpression in the context of an otherwise non-actionable
 ETV6
 fusion, and
 MET
 overexpression in a
 TES::TFEC
 -fusion bearing cancer.
 Conclusions:
 Druggable fusions are frequent in
 K-RAS
 wild-type pancreatic cancers. Moreover, RNA fusion assays as well as transcriptomic expression testing can identify candidate personalized therapies for patients with
 K-RAS
 wild-type pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723fc53e1c67e373fda744cec67f0720b9c9b122" target='_blank'>
              The cancer fusionome and overexpressed druggable oncogenic signals in
 K-RAS
 wild-type pancreatic cancer.
              </a>
            </td>
          <td>
            Michal Krawczyk, Roisin Puentes, Nathan D Montgomery, Derek D Lyle, R. Kurzrock, F. Lopez-Diaz
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lynch syndrome (LS), also known as hereditary non-polyposis colorectal cancer (HNPCC), is caused by monoallelic germline mutations in mismatch repair (MMR) genes (e.g., MLH1, MSH2, MSH6, and PMS2), which predispose individuals to an increased risk of developing various cancers (mainly colorectal cancer and endometrial cancer). The hallmark of LS-associated tumors is high microsatellite instability (MSI-H) characterized by increased mutations at coding mononucleotide repeats (cMNR), which usually lead to frameshift mutations (FSMs) in coding exons. These recurrent FSMs are thought to play a central role in the increased cancer risk. Vaccines targeting neoantigens produced by FSMs can elicit immune responses, which may prevent or suppress LS tumor formation and progression. Indeed, frameshift neoantigen-based vaccines have been developed and are being tested in LS or MSI-H patients in clinical trials (e.g., NOUS-209). To accelerate vaccine development and optimization, we have developed and characterized an intra-cecal tumor implantation model using tumor fragments derived from a LS mouse model (Msh2LoxP/LoxP;Villin-Cre; VCMsh2). The model was examined for its suitability for preclinical vaccine evaluation in prevention (vaccination followed by fragment implantation) and treatment (fragment implantation followed by vaccination) settings. Primary cecal tumors were adenocarcinoma mixed with mucinous feature. In contrast to subcutaneous tumor implant models that never develop metastasis, metastatic lesions were detected in the lung, liver, and lymph nodes within two months post intra-cecal tumor implantation. In both settings, vaccinated animals had intra-cecal tumors that were smaller in size and a lower rate of metastasis and lower number of metastatic nodules as compared to unvaccinated controls (2 vs. 13 nodules in preventive setting and 1 vs. 4 nodules in therapeutic setting, respectively). FSM analysis by targeted next generation sequencing of a panel of 50 genes revealed that some loci had lower mutation rates and variant allele frequencies, but not at all vaccine targets, in tumors from vaccinated groups compared to controls, indicating that tumor cells harboring certain FSMs targeted by the vaccine were not eliminated. Vaccine target composition and formulation need to be further optimized. Interestingly, plasma cell free DNA (cfDNA) showed lower FSM rates in vaccinated groups, which correlated with a lower metastatic rate. This is consistent with data from VCMsh2 model vaccinated with frameshift peptide vaccines. Additional studies are needed to confirm whether this is neoantigen vaccine specific. In summary, this intra-cecal implantation LS model appears suitable for preclinical neoantigen-based vaccine testing and may help elucidate aspects of tumor cell/DNA shedding and distant metastasis.


 Yurong Song, Brandon Somerville, Shaneen S. Baxter, Jason D. Marshall, Lei Wei, Chelsea Sanders, Qiang Hu, Song Liu, Alan Hutson, Baktiar Karim, Simone Difilippantonio, Ligia Pinto, Shizuko Sei, Robert H. Shoemaker. Frameshift neoantigen-based vaccine testing in an intra-cecal implantation mouse model of Lynch syndrome [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6343.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d85b02574f80309e49880132063bfa30fa50f5b" target='_blank'>
              Abstract 6343: Frameshift neoantigen-based vaccine testing in an intra-cecal implantation mouse model of Lynch syndrome
              </a>
            </td>
          <td>
            Yurong Song, Brandon Somerville, Shaneen S Baxter, Jason D. Marshall, Lei Wei, Chelsea Sanders, Qiang Hu, Song Liu, Alan Hutson, Baktiar Karim, Simone Difilippantonio, Ligia A. Pinto, Shizuko Sei, Robert H. Shoemaker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary The presence of certain DNA damage/repair errors together can cause cancer cells to die, offering hope for new treatments. Mutational signatures show how DNA damage/repair errors happened by looking at the types and amounts of mutations. We studied mutational signatures that occur together or separately, called mutational signature interactions, to find which combinations of DNA repair errors help cancer cells thrive and which ones harm them. We analyzed mutational signatures and their interactions in 1014 breast cancer genomes. HRd and NHEJd were found together with APOBEC, while they did not overlap with the ROS mutational signature. Mutational signature interactions were distinct in various breast cancer types. Non-HRd TNBC tumors showed a unique ROS signature, while HRd TNBC tumors did not have an ROS signature. These analyses show that mutational signatures and their interactions could help identify promising synthetic lethal targets in DNA repair systems, improve how patients are grouped and point to potential treatments. Abstract Background: Defective DNA repair systems result in the accumulation of mutations, loss of genomic integrity, and eventually cancer. Following initial malignant transformation due to specific DNA damage and defective DNA repair, cancer cells become reliant upon other DNA repair pathways for their survival. The co-occurrence of specific repair deficiencies brings catastrophic outcomes such as cell death for cancer cells and thus holds promise as a potential therapeutic strategy. Exploring the co-occurrence and mutual exclusivity of mutational signatures provides valuable knowledge regarding combinations of defective repair pathways that are cooperative and confer selective advantage to cancer cells and those that are detrimental and cannot be tolerated by them. Methods: Taking advantage of mutational signature profiling, we analyzed whole-genome sequences of 1014 breast cancers to reveal the underlying mutational processes and their interrelationships. Results: We found an inverse relationship between deficiencies of homologous recombination (HRd) and non-homologous end joining (NHEJd) with reactive oxygen species (ROS). Moreover, HRd and NHEJd co-occurred with APOBEC but were mutually exclusive with mismatch repair deficiency (MMRd) and ROS. Our analysis revealed that SBS8 and SBS39 signatures of currently unknown etiology correlate with NHEJd. ID1 and ID2 signatures co-occur with ROS and have mutual exclusivity with HRd, SBS8, SBS39 and NHEJd. The ID4 signature, with currently unknown etiology, has mutual exclusivity with HRd and NHEJd and co-occurred with ROS. On the other hand, the ID15 signature, with currently unknown etiology, co-occurred with SBS8, SBS39, HRd, NHEJd and DBS2, while having an inverse relationship with MMRd and ROS. Comparing the mutational signatures of HRd and non-HRd TNBC genomes reveals the unique presence of ROS signatures in non-HRd tumors and the lack of ROS signature in HRd tumors. Conclusion: Taken together, these analyses indicate the possible application of mutation signatures and their interactions in advancing patient stratification and suggest appropriate therapies targeting the make-up of individual tumors’ mutational processes. Ultimately, this information provides the opportunity to discover promising synthetic lethal candidates targeting DNA repair systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575ec445e704ca863d5507eaa78e755caa175d07" target='_blank'>
              The Use of Mutational Signatures to Decipher the Inter-Relationship of Reactive Oxygen Species with Homologous Recombination and Non-Homologous End-Joining Deficiencies as Well as Their Effects on APOBEC Mutagenesis in Breast Cancer
              </a>
            </td>
          <td>
            Amir Farmanbar, Robert Kneller, Sanaz Firouzi
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20d1d5ff1c3f82036ab1040950e6af671f747e51" target='_blank'>
              Efficient installation of heterozygous mutations in human pluripotent stem cells using prime editing
              </a>
            </td>
          <td>
            Annabelle Suter, Alison Graham, Jia Yi Kuah, Jason Crisologo, Chathuni Gunatilake, K. Sourris, Michael See, F. J. Rossello, M. Ramialison, Katerina Vlahos, Sara E. Howden
          </td>
          <td>2025-05-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The molecular continuum of premalignant-to-malignant transition remains elusive in carcinogenesis, primarily due to the difficulty in sampling transitional lesions. Here, we present comprehensive genomic and immunological profiling of 21 very early hepatocellular carcinomas (veHCCs) arising within 17 cancer-prone DNs. Surprisingly, 82% of cancer-prone DNs harbored TERT alterations, suggesting a predisposing rather than causative role in malignancy transition. A substantial increase in CNA burden, rather than SNV, was noted, indicating a role of chromosome instability in malignancy transition. Additionally, veHCC-originating DNs showed immune inactivity, not falling into the prevailing paradigm that HCCs develop in a context of chronic inflammation. 43% of veHCCs showed an inflamed phenotype with relatively mild CNA burden but already exhibited immune evasion features. Two major evolutionary scenarios were thus proposed: 1) CNA-dominant progression and 2) early immune evasion of those with mild CNA burden. Collectively, our findings illustrate previously unexplored molecular paradigms in HCC initiation, highlighting the therapeutic potential of immunotherapy for early intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/931220b668a9cfbdeab70ee80f3f11b3d5f599f4" target='_blank'>
              Molecular insights into early malignant transition of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Zhengtao Zhang, Hong Li, Lingli Chen, Tao lu, Xinyi Shentu, Yuanhua Liu, Shuyi Ji, Zhixin Qiu, Yining Zou, Hong Wen, Jing Han, Zhengzeng Jiang, Jie Fan, Dianfan Li, A. Mesropian, R. Pinyol, J. M. Llovet, Hui Dong, Yuan Ji, Lijian Hui
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 e15664


 Background:
 Adjuvant Therapy (AT) is recommended in colon cancer (CC) to enhance surgical cures, by few percentages, typically in node positive stages. Many patients don’t derive benefits from AT but are difficult to identify currently. Ten-year data from the MOSAIC study indicates that AT’s impact on recurrence is limited to 30 months post-surgery. This timeframe aligns with the predicted period between stem cell activation and clinical recurrence. Chemotherapy, therefore, primarily targets the proliferative progenitors facilitating the innate immune effectors (e.g. NK cells) interception of recurrence events (optimizing the ratio). By identifying proliferative capacities through DNA findings could optimize patient selection. We hypothesized that an intrinsic DNA events drive early recurrence and denote AT beneficiaries. To test this hypothesis, we focused on a set of genes relevant to the cell cycle and report on copy number alterations (CNA).
 Methods:
 Patients in stages 2 and 3 CC in the cancer genome atlas (TCGA) were categorized into those who recurred within the first 30 months (R); or the remaining population (N). Cyclins and Cyclin dependent kinases (CDK) would be relevant if amplified in R and cell cycle inhibitors would be relevant if amplified in N. Here we report on gene amplification of E2F1, RB, p21, CDKs, CCNE1, CCNB1 CCNA1, MCM2, MCM5, MCM7, and RFC2.
 Results:
 Cyclins E, A and B are amplified in R substantially more than in N (17% v 2%; 11% v 0% and 6% v 3%, respectively). Similarly, p21 is amplified in 6% in N v 0% in R. E3F amplification is similar in both R and N groups and RB is amplified, paradoxically, in the R (28%) v 9% in N.
 Conclusions:
 The findings highlight the potential role of intrinsic DNA events mediating proliferation in predicting recurrence following surgery in a time frame that is relevant to AT. Identifying such markers may enhance patient selection and reduce unnecessary toxicity from ineffective treatments. Relevant DNA events are certainly not limited to CNAs studied in this work and need to expand to include relevant mutational and rearrangement events (gain- or loss of function, based on the physiologic roles). Given the stability of DNA confirmation of these findings is feasible even in archival material. Moreover, unsupervised analysis of the mediators of recurrence in relevant postoperative times is valid and promising. Further studies with independent larger datasets are needed to validate these observations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7de95d853e19df76b22738b76b4dac56c1f4d29f" target='_blank'>
              Role of proliferation-related genes as predictive factors for adjuvant therapy in colon cancer.
              </a>
            </td>
          <td>
            A. Trikannad, Sruthi Vellanki, Jim Z Chen, Nishanth Thalambedu, A. M. Safar
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Immune checkpoint inhibitors (ICI) improve survival in patients with metastatic urothelial cancers (mUC) but most patients experience disease progression. The mechanisms driving this progression are mostly unknown. We longitudinally assessed mUC microenvironment at the single-cell level in ICI-treated patients to explore the determinants of the response and adaptive resistance mechanisms on therapy.



 The prospective MATCH-R study (NCT02517892) included patients with mUC treated with single-agent PD-(L)1 inhibitors. Biopsies of metastatic sites were performed at baseline, on therapy and at progression for single-nuclei RNA sequencing (10X genomics, 5’ v1.1). After stringent quality control and cell type annotation, we compared the proportions and transcriptomic signatures of tumor and immune cell subsets between responders and non-responders, and explored their evolution during treatment. Pseudobulk analyses were used to classify whole samples according to the consensus molecular classification (Kamoun et al. 2020).



 We enrolled 32 patients among whom eight (25%) achieved objective response (OR). All patients underwent biopsies at baseline, 6 (19%) on therapy, 17 (53%) at progression. Tumor cells showed high transcriptomic heterogeneity within the consensus groups, with the coexistence of basal and luminal cell states within the same samples. The proportion of basal cells at baseline, but not the basal sample classification, was associated with an increased OR (p=0.02). Immune features associated with resistance to ICI at baseline involved both myeloid and lymphoid compartments : within myeloid cells, an abundance of pro-tumoral HES1 macrophages and higher expression of immunosuppressive genes CD163, MS4A4A and STAB1 ; within T cells, a higher proportion of exhausted CD8+ lymphocytes, and higher expression of immune checkpoint genes including PDCD1. Conversely, the baseline abundance of CD4+ naive T helpers was associated with improved OR. Longitudinal sampling showed potential adaptive resistance mechanisms at progression, including downregulation of HLA genes and IFN signaling in tumor cells ; shift from M1 to M2 macrophage polarization within myeloid cells; increased expression of immune checkpoints, downregulation of type-I interferon induced genes and cytotoxicity markers such as GZMA in T cells. Individual patient analyses have shown that these mechanisms may coexist with discrete patterns.



 We identified potential mechanisms of resistance to ICI in patients with mUC related to both tumor and immune compartments, including specific T cell and macrophage subpopulations. Longitudinal assessments in patients show individual molecular patterns related to progression that may allow for dynamic and tailored therapeutic strategies.



 Ronan Flippot, Amélie Roehrig, Julien Vibert, Nicolas Stransky, Luc Cabel, Kevin Mulder, Benjamin Besse, Claudio Nicotra, Maud Ngo Camus, Christophe Massard, Etienne Rouleau, Gerome Jules-Clement, Alicia Tran-Dien, Lambros Tselikas, Fabrice Andre, Jean-Yves Scoazec, Céline Vallot, Maud Kamal, Eric Letouze, Yohann Loriot. Single-cell trajectories of metastatic urothelial cancer and individual patterns of resistance to immune checkpoint inhibitors. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 738.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/058fbe88bdb4a0b6edaf627e6a91c600aafe91d0" target='_blank'>
              Abstract 738: Single-cell trajectories of metastatic urothelial cancer and individual patterns of resistance to immune checkpoint inhibitors
              </a>
            </td>
          <td>
            R. Flippot, Amélie Roehrig, Julien Vibert, N. Stransky, Luc Cabel, Kevin Mulder, B. Besse, C. Nicotra, M. N. Camus, Christophe Massard, E. Rouleau, Gerome Jules-Clement, A. Tran-Dien, L. Tselikas, Fabrice André, J. Scoazec, Céline Vallot, Maud Kamal, Eric Letouze, Yohann Loriot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 c-MYC (MYC) is a central regulatory protein that is dysregulated in >50% of all human cancers and is linked to aggressive disease. Developing MYC inhibitors would revolutionize cancer treatment; however, efforts to target MYC directly using small molecular inhibitors have historically failed. A promising approach is to identify and inhibit critical MYC partner proteins to inactivate MYC and trigger cancer cell death. Inhibiting these targets therapeutically can result in synthetic lethality (MYC-SL), which can be exploited in MYC-dysregulated cancers. To identify MYC-SL targets, we performed a genome-wide CRISPR knock-out screen using an isogenic pair of non-transformed and MYC-driven breast cancer cells. In contrast to other screens, this model is both dependent on MYC and recapitulates human disease at pathological and molecular levels in vivo. Finally, these hits were cross-referenced with our MYC protein-interactome data to reveal putative MYC partner proteins that are critical for MYC activity.
 From our screen results, we performed gene set enrichment analysis to identify biological activities that may represent core functional dependencies in MYC dysregulated cancer cells. Using this approach, we identified and validated topoisomerase 1 (TOP1) as an actionable vulnerability that can be targeted with clinically approved inhibitors. Genetic and pharmacological inhibition of TOP1 in multiple orthogonal assays resulted in MYC-driven cell death. Finally, drug response to TOP1 inhibitors correlated with MYC levels and activity across panels of breast cancer cell lines and patient-derived organoids, highlighting TOP1 as a promising target for MYC-driven cancers.
 The recent accessibility of large-scale datasets detailing functional dependencies across hundreds of cancer cell lines offers an unprecedented opportunity to prioritize targets with greater translational relevance, which is a major limitation of the synthetic-lethal approach for target discovery. As a secondary analysis of our hits, we utilized DEPMAP data to classify cancer cell lines as relatively MYC-dependent or MYC-independent, enabling the in silico evaluation of each MYC-SL hit’s differential essentiality. MYC-SLs that exhibited selective essentiality in MYC-dependent cell lines were prioritized for further investigation. Results from this strategy revealed critical MYC cofactors that have been validated by us and others (e.g., CDK9), providing confidence in this approach. Excitingly, previously underexplored targets were identified with promising validation to-date. Together, this work features two successful strategies to prioritize hits from hundreds of synthetic-lethal genome-wide CRISPR screens to identify novel MYC-driven vulnerabilities in cancer.


 Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H. Tong, Katherine Chan, Samah El Ghamrasni, Miquel Pujana, David W. Cescon, Jason Moffat, Linda Z. Penn. An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc94f977a238b5f1df2211d9c38889ceb4f0e79" target='_blank'>
              Abstract 4039: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H Y Tong, Katherine Chan, S. E. Ghamrasni, M. Pujana, D. Cescon, Jason Moffat, Linda Z. Penn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686514f6a242593983edc4eedbd2ea02d17cd369" target='_blank'>
              REpair of heterozygous Mutations independent of Exogenous Donor template with high efficiency (REMEDY) using allele specific CRISPR targeting and HDR enhancers
              </a>
            </td>
          <td>
            Y. Sezgin, Genesis Snyder, Noushin Saljoughian, Colin Maguire, Ebru Erzurumluoglu Gokalp, Devi Jaganathan, Eleonora S D'Ambrosio, B. Ozes, Gregory Wheeler, Ben Kelly, Mark E. Hester, Zarife Sahenk, S. Vaidyanathan, Afrooz Rashnonejad, Jerry Mendell, Nizar Y. Saad, Roshini S. Abraham, Juhi Bagaitkar, Susan D. Reynolds, Allison Bradbury, M. N. Kararoudi
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 In acute myeloid leukemia (AML), genetic mutations distort hematopoietic differentiation, resulting in the accumulation of leukemic blasts. However, it remains unclear how these mutations intersect with the cellular origins of each patient's disease, and whether distinct sets of mutations converge upon similar differentiation patterns. Single-cell RNA sequencing has enabled high-resolution mapping of the relationship between leukemia and normal hematopoietic cell states. Yet, this application has been hampered by imprecise reference maps of normal hematopoiesis, or by small patient cohort sizes which do not adequately capture the extent of inter-patient heterogeneity in AML. To resolve this, we constructed a reference atlas of human bone marrow hematopoiesis from 263,519 single-cell transcriptomes enriched for rare hematopoietic stem and progenitor cells (HSPCs). The resulting reference spans 55 cellular states and has been benchmarked against independent datasets of purified HSPCs. Using this comprehensive reference atlas, we confidently mapped over 1.2 million single-cell transcriptomes from 318 patient samples spanning AML, mixed-phenotype acute leukemia (MPAL), and acute erythroid leukemia (AEL) diagnoses. Single-cell composition analysis revealed twelve patient subgroups, each reflecting distinct patterns of aberrant differentiation in AML. Strikingly, some AML samples exhibited virtually no overlap in cell state involvement with one another, likely reflecting their disparate cellular origins. One subgroup was enriched for early lymphoid progenitors and featured co-clustering of AML and MPAL samples. Notably, this early lymphoid subgroup included an AML sample which eventually relapsed with lymphoid disease. To understand the genetic determinants of aberrant differentiation in AML, we quantified leukemia cell state abundance in >1,200 patient samples and evaluated genotype-to-phenotype associations to link >40 genetic driver alterations with their specific impacts on AML differentiation. This identified genetic drivers for unconventional lineage phenotypes in AML, including erythroid lineage priming associated with the co-occurrence of complex cytogenetics with TP53 mutations as well as lymphoid lineage priming associated with bi-allelic RUNX1 mutations. We also identified non-genetic determinants of AML differentiation. For example, we identified two subgroups of KMT2A-rearranged AML with distinct cellular origins reflecting either HSPCs or committed myeloid precursors. Last, we show that distinct leukemia cell hierarchies can co-exist within individual patients, providing insights into AML evolution. Together, single-cell reference mapping of malignant cell states provides a framework for understanding the impact of genetic alterations on hematopoietic differentiation across hundreds of individual AML patients.



 Andy G. Zeng, Ilaria Iacobucci, Sayyam Shah, Amanda Mitchell, Gordon Wong, Suraj Bansal, David Chen, Qingsong Gao, Hyerin Kim, James A. Kennedy, Andrea Arruda, Mark D. Minden, Torsten Haferlach, Charles G. Mullighan, John E. Dick. Single-cell transcriptional mapping reveals genetic and non-genetic determinants of aberrant differentiation in AML [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3806.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37654945a8b87aa1dc552938a4ce7cfc7067cb7a" target='_blank'>
              Abstract 3806: Single-cell transcriptional mapping reveals genetic and non-genetic determinants of aberrant differentiation in AML
              </a>
            </td>
          <td>
            Andy G. X. Zeng, Ilaria Iacobucci, Sayyam Shah, Amanda Mitchell, Gordon Wong, Suraj Bansal, David Chen, Qingsong Gao, Hyerin Kim, James A Kennedy, Andrea Arruda, Mark D. Minden, Torsten Haferlach, C. Mullighan, J. Dick
          </td>
          <td>2025-05-22</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a deadly brain tumor that often develops resistance to DSB-inducing therapeutics due to dysregulated DNA repair. Ionizing radiation (IR) targets these cancer cells by causing DNA double-stranded breaks (DSBs). However, GBM often has overactive genes involved in the error-free DSB-repair process known as Homologous Recombination (HR). This results in therapy resistance and poor patient outcomes. Here, we provide an alternative approach to target DSB-resistant GBM by modulating the dNTP pool instead of solely targeting DNA-repair proteins. Through a comet assay, we found that increasing the dNTP pool prior to IR resulted in impaired DSB-repair. Immunofluorescence analysis of key HR proteins revealed significant disruptions in end resection, a necessary phase of HR. Our RNA-seq results show that increasing the dNTP pool prior to IR treatment caused significant changes in gene expression when compared to IR or dN treatment alone. Pathways most affected include apoptosis and cell cycle arrest. Flow cytometry analysis confirmed the significant increase in apoptosis in irradiated cancer cells with elevated dNTP pools. Not only do these data suggest that HR efficiency is reduced due to end resection impairment, but they also show that as a result, cell viability is destabilized due to prolonged DNA damage, resulting in apoptosis. The identification of this impairment may be essential in finding novel therapeutic approaches to sensitize therapy-resistant GBM.


 Dominique Monroe, Mercy Kehinde-Ige, Apeksha Anand, Arilyn Williams, Edidiong Usoro, Matthew Kededa, Aman Kalsi, Christian Fahmy, Ramsha Khanam, Huidong Shi, Waaqo Daddacha. Elevated dNTP pools impair homologous recombination-mediated repair, promoting apoptosis in therapy-resistant glioblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4200.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4631b336ea353f2fab308e1332f9b36313977696" target='_blank'>
              Abstract 4200: Elevated dNTP pools impair homologous recombination-mediated repair, promoting apoptosis in therapy-resistant glioblastoma
              </a>
            </td>
          <td>
            Dominique Monroe, Mercy Kehinde-Ige, Apeksha Anand, Arilyn Williams, E. Usoro, Matthew Kededa, Aman Kalsi, Christian Fahmy, Ramsha Khanam, Huidong Shi, Waaqo Daddacha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Lung squamous cell carcinoma (LUSC) is an aggressive tumor type accounting for 30% of all lung cancers, with a roughly 30% 5-year survival. The genetic nature of these tumors precludes the use of targeted therapies and the standard immunotherapy, either alone or in combination with chemotherapy, results in response rates below 50% after biomarker selection. Better patient selection for immunotherapy regimens requires a better definition of the molecular and immune landscape of LUSC. The purpose of this study is to comprehensively characterize the molecular, clinical and immune context of LUSC to identify novel biomarkers that can guide treatment decisions and to find novel therapeutic targets.



 A multiparametric study including a targeted DNA and RNA sequencing of genes related to tumor-immune system crosstalk was conducted in a cohort of 98 early-stage LUSC patients. Histopathological features including immune infiltrate and metabolic traits were defined by in the LUSC tissues. Consensus clustering algorithm classified the sample transcriptomes, and all this data was cross-analyzed with clinical, molecular and immune annotations.



 Computational analysis of immuno-oncology gene expression data from 98 LUSC tumors led to the classification in 4 patient subgroups with specific molecular and immune characteristics. Interestingly, we found a genomic amplification of the chromosome region 3q, present in almost 30% of the patients, strongly associated with the low immunogenic, as previously shown, and high tumorigenic subgroup. To know the mechanism behind this robust association and given the nature of relevant genes in this region, we analyzed iron accumulation in the tumor samples. We found that iron-positive tumors showed a reduced infiltration of CD8+ T cells compared to iron-negative tumours. The same tendency was observed in CD4, CD20 and CD68 analyses, further supporting an association of this amplification and iron accumulation with a low immunogenic tumor profile. These results were validated in an independent cohort of early-stage LUSC.



 Taken together, our results support the possible connection between iron accumulation and 3q amplification as a mechanism for immune exclusion in LUSC. We believe that this molecular alteration can be clinically relevant as a biomarker for immunotherapy selection in LUSC, at least in a neoadjuvant setting. Whether this association applies to other squamous carcinomas or might represent novel therapeutic vulnerabilities ensures further research



 María Gutiérrez Pérez, Javier Ramos Paradas, David Gómez Sánchez, Ricardo García Luján, Jon Zugazagoitia, Nuria Carrizo, Roxolyana Khoroz, Inés Díaz Cano, Laura García Redondo, Ana B Enguitta, Esther Conde, Eva M Garrido Martín, Álvaro C. Ucero Herrería, Luis Paz-Ares. Chromosome 3q amplification in lung squamous cell carcinoma is associated to iron metabolism and immune cell exclusion [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 910.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9488336a355d0c8f13d6aba1e3fdc05af9aece24" target='_blank'>
              Abstract 910: Chromosome 3q amplification in lung squamous cell carcinoma is associated to iron metabolism and immune cell exclusion
              </a>
            </td>
          <td>
            María Gutiérrez Pérez, J. R. Paradas, David Gómez Sánchez, R. G. Luján, Jon Zugazagoitia, N. Carrizo, Roxolyana Khoroz, Inés Díaz Cano, Laura García Redondo, Ana B Enguitta, Esther Conde, Eva M Garrido Martín, Álvaro C. Ucero Herrería, Luis Paz-Ares
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The diploid genome of the fungal pathogen Candida albicans is highly heterozygous, with most allele pairs diverging at either the coding or regulatory level. When faced with selection pressure like antifungal exposure, this hidden genetic diversity can provide a reservoir of adaptive mutations through loss of heterozygosity (LOH) events. Validating the potential phenotypic impact of LOH events observed in clinical or experimentally evolved strains can be difficult due to the challenge of precisely targeting one allele over the other. Here, we show that a CRISPR-Cas9 system can be used to overcome this challenge. By designing allele-specific guide RNA sequences, we can induce targeted, directed LOH events, which we validate by whole-genome long-read sequencing. Using this approach, we efficiently recapitulate a recently described LOH event that increases resistance to the antifungal fluconazole. Additionally, we find that the recombination tracts of these induced LOH events have similar lengths to those observed naturally. To facilitate future use of this method, we provide a database of allele-specific sgRNA sequences for Cas9 that provide near genome-wide coverage of heterozygous sites through either direct or indirect targeting. This approach will be useful in probing the adaptive role of LOH events in this important human pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/161077a2ca34ace846023a0e504cab47b241a6a0" target='_blank'>
              Targeted loss of heterozygosity in Candida albicans using CRISPR-Cas9
              </a>
            </td>
          <td>
            Philippe C. Després, Nicholas C. Gervais, Meea Fogal, Ruby KJ Rogers, Christina A. Cuomo, Rebecca S. Shapiro
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The clustered regularly interspaced short palindromic repeats (CRISPR)/Cas system is a powerful genome editing tool that has revolutionized research. Single nucleotide polymorphisms (SNPs) are the most common form of genetic variation in humans. Only a subset of these SNPs has been shown to be linked to genetic diseases, while the biological relevance of the majority remains unclear. Investigating these variants of unknown significance could provide valuable insights into their roles in biological processes, disease susceptibility, and treatment responses. While CRISPR/Cas has emerged as a transformative technology, its ability to make single nucleotide substitutions remains a significant limitation. Other techniques in single nucleotide editing, such as base editing and prime editing, offer promising possibilities to complement CRISPR/Cas systems, though they also have their own limitations. Hence, alternative approaches are necessary to overcome the limitations of CRISPR. Here, to improve the feasibility of generating single base edits in the genome, we provide a protocol that introduces a multiple expression and dual selection (MEDS) system, which, alongside CRISPR, utilizes the opposing roles of cytosine deaminase/uracil phosphoribosyltransferase (CD/UPRT) for negative selection and neomycin phosphotransferase II (NPT II) for positive selection. As a proof of concept and to demonstrate feasibility of the method, we used MEDS, along with traditional CRISPR-Cas9, to generate sickle hemoglobin by introducing a point mutation (A → T) in the sixth codon of the hemoglobin beta gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea49db880f24cd1e00dae98c95c6cafbd8936b9" target='_blank'>
              CRISPR genome editing using a combined positive and negative selection system
              </a>
            </td>
          <td>
            I. Sharma, Kerisa Hall, Shannon Moonah
          </td>
          <td>2025-05-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) progression is closely linked to alterations in key signaling pathways and metabolic reprogramming. The WNT signaling pathway plays a critical role in regulating cell development and proliferation, and its abnormal activation is a hallmark of CRC. Plant Homeodomain 6 (PHF6), a chromatin-associated factor, is known for its roles in clonal hematopoiesis, ribosome biogenesis, DNA repair, and aging, and is frequently mutated in lymphoid cancers and leukemia. In this study, RNA-seq and ATAC-seq were conducted to investigate the function of PHF6, we identified PHF6 as a novel regulator of the WNT signaling pathway. We found that PHF6 depletion activate WNT signaling and proliferation of colon cancer cells. In the AOM/DSS model, PHF6 knockout mice exhibited larger polyp formation compared to wild type mice, with polyps extending into the proximal region. Proximal CRC is more prevalent and aggressive and has distinct genetic features compared to distal CRC. Alterations in proximal CRC include the PIK3CA mutation and CpG island methylator phenotype (CIMP)-high frequency, the development of CIMP is followed by global gene expression alteration and metabolic changes. However, the molecular mechanisms of this type of cancer are not fully understood. We observed a reduced levels of LEO1, the transcriptional elongation partner of PHF6 within the PAF1 complex, in the proximal region of PHF6 KO mice. We identified LEO1 and SETD1A as epigenetic regulators of lipid homeostasis in colon cancer cells and patient samples with matched normal. Dysregulation of lipid metabolism in cancer cells impacts tumor initiation, progression, and metastasis. LEO1 KO cells exhibited reduced expression and activation of SREBP1, a master regulator of lipid metabolism, under glucose starvation. These effects were further enhanced by SETD1A knockdown. LEO1 was found to localize to the nucleus and is required for the nuclear localization of SETD1A under nutrient stress. These cells showed an altered cell cycle, resulting in reduced proliferation but increased metastatic potential. These findings suggest that the PHF6-LEO1 axis plays a crucial role in the regional development and metastasis, underscoring its importance in advancing our understanding of tumor biology and guiding targeted therapy development.


 Su Chan Park, Shinae Park, Ji-Yeon Lee, Taehan Yoon, So Hyun Kwon, Jung-Shin Lee, Eun Jung Park, Ji Min Lee. PHF6-LEO1 axis depletion drives aggressive colorectal cancer through WNT activation and metabolic reprogramming with therapeutic implications [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 184.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eebb48dbe13fca3330112d2c1d24f34c4814602f" target='_blank'>
              Abstract 184: PHF6-LEO1 axis depletion drives aggressive colorectal cancer through WNT activation and metabolic reprogramming with therapeutic implications
              </a>
            </td>
          <td>
            Su Chan Park, Shinae Park, Ji-yeon Lee, Taehan Yoon, So Hyun Kwon, Jung-Shin Lee, Eun Jung Park, Ji Min Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Gastroesophageal adenocarcinomas (GEA) are characterized by chromosomal instability (CIN) predominantly consisting of high amplitude focal amplifications of oncogenes, including Cyclin E1. An initial pan-cancer analysis revealed that CCNE1 is one of the top genes whose expression correlates with focal CIN.



 We proposed a causal role for excess CCNE1 as a driver of CIN and gastroesophageal tumorigenesis, and investigated the therapeutic vulnerabilities of CCNE1 amplification/overexpression.



 We engineered a novel CCNE1-overexpressing mouse gastric organoid model with Trp53 deletion and studied the functional effects on tumor initiation and progression. Whole genome sequencing of tumors was performed to characterize the effects of CCNE1 overexpression on genomic instability. To identify dependencies of CCNE1-amplified GEA, we executed a CRISPRi functional genomic screen. Top genetic dependencies were translated into potential pharmacologic vulnerabilities and tested in monotherapy and combination.



 CCNE1 overexpression accelerated tumor initiation and progression of mouse gastric organoids with Trp53 deletion. These tumors demonstrated increased chromosomal complexity, mainly focal amplifications. Our functional genomic screen identified specific genetic dependencies of CCNE1-amplified GEA, including CCNE1/CDK2 as the strongest hits, and SKP2 as a novel dependency likely through its function as a p27 degrader. Pharmacologic inhibition of CDK2 was insufficient and antagonistic with concurrent inhibition of the DNA damage response via targeting the ATR/CHEK1 axis. However, staggered inhibition of these distinct vulnerabilities demonstrated efficacy.



 Our study identifies CCNE1 as a driver of CIN in GEA and builds a foundation for further exploration of therapies for CCNE1-amplified GEA characterized by genomic instability.



 Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, Adam Bass. CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2834.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0c88288c8c7bf7b80395b40fbfcb9b8e990950" target='_blank'>
              Abstract 2834: CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, A. Bass
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutations in genome maintenance factors drive sporadic and hereditary breast cancers. Here, we searched for potential drivers based on germline DNA analysis from a cohort consisting of early-onset breast cancer patients negative for BRCA1/BRCA2 mutations. This revealed candidate genes that subsequently were subjected to RNAi-based phenotype screens to reveal genome integrity impacts. We identified several genes with functional roles in genome maintenance, including Glucose-6-Phosphatase Catalytic Subunit 3 (G6PC3), SMC4, and CCDC108. Notably, G6PC3-deficient cells exhibited increased levels of γH2AX and micronuclei formation, along with defects in homologous recombination (HR) repair. Consistent with these observations, G6PC3 was required for the efficient recruitment of BRCA1 to sites of DNA double-strand breaks (DSBs). RNA sequencing analysis revealed that G6PC3 promotes the expression of multiple homologous recombination repair genes, including BRCA1. Through CRISPR-Select functional-genetic phenotype analysis of G6PC3 germline mutations, we identified two germline G6PC3 variants displaying partial loss-of-function. Furthermore, our study demonstrated that G6pc3 deficiency accelerates mammary tumor formation induced by Trp53 loss in mice. In conclusion, our cohort-based functional analysis has unveiled genome maintenance factors and identified G6PC3 as a potential candidate tumor suppressor in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c29e60f63dffafbff9d001a2d3645ac87336a0cf" target='_blank'>
              G6PC3 promotes genome maintenance and is a candidate mammary tumor suppressor.
              </a>
            </td>
          <td>
            Xin Li, Maria Rossing, Ana Moisés da Silva, Muthiah Bose, Thorkell Gudjónsson, Jan Benada, J. Thatte, Jens Vilstrup Johansen, J. Börcsök, H. van der Gulden, Ji-Ying Song, Renée Menezes, Asma Tajik, Lucía Sena, Zoltan Szallasi, Morten Frödin, J. Jonkers, F. C. Nielsen, C. S. Sørensen
          </td>
          <td>2025-04-22</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) has been classified into molecular subtypes using gene-level data from both bulk and single-cell RNA sequencing (scRNA-seq), leading to the development of the consensus molecular subtypes (CMS) and intrinsic CMS (iCMS). In addition to CRC studies, scRNA-seq and spatial transcriptomics characterization of human polyps, aligned with the conventional adenoma and serrated polyps, identified distinct driver mutations, cells-of-origin and iCMS traits associated with each polyp type. These gene-level transcriptional classification approaches were both strongly aligned to mutations underpinning the Vogelstein model of CRC development.
 However, when viewed using pathway-level information, the most dominant mechanistic signaling cascades in CRC were completely independent of these same mutational drivers. Therefore, to deliver a holistic landscape-view of the tumor ecosystem, we proposed that pathway-level subtyping could offer a broader understanding of non-genetic phenotypic signaling in CRC, leading to more informative stratification and therapeutic decision-making.
 Using this new approach in combination with bulk transcriptional data from n=1200+ tumors, alongside n=29, 452 scRNA-seq profiles derived from murine organoids, and n=49, 155 epithelial cells derived from n=63 human tumor samples, we identified three pathway-derived subtypes (PDS1-3), revealing previously overlooked changes in epithelial cell identities and phenotypic states. PDS2 represents tumors with stroma/immune-dense, regenerative stem-like features. PDS1 were associated with elevated MYC signaling, proliferation and canonical stem-cells with good prognosis, while PDS3 displayed a differentiated-like phenotype associated with reduced polycomb repressive complex 2 (PRC)-mediated gene repression and poorer prognosis.
 The inverse relationship between MYC/PRC targets within PDS1/PDS3 tumors supported the development of stem maturation index, which was directly related to cellular entropy and differentiation status of colonic cells allowing our tumor data to be projected onto a Waddington’s landscape. Given these associations with developmental patterns in established tumors, we next utilized the same pathway-level approach in combination with transcriptional data derived from n=2641 human polyps, where we observe that PDS traits can be established early in precancer development. Using a combination of trajectory analyzes, lineage identities and cellular phenotyping, these approaches have again revealed that, even in very early precancers, there is no association between Vogelstein-related mutational drivers and prominent phenotypic signaling cascades.
 While work is ongoing, the use of pathway-level information provides a unique insight into the dynamic nature of phenotypic evolution across early-to-late stages of colorectal precancers and tumors.


 Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, Joanne Edwards, Owen Sansom, Philip D. Dunne. Tracking evolution of transcriptional landscapes during colorectal cancer development and progression [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 503.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5fa7d80af5b43992f24ebd211378eba4c94dea" target='_blank'>
              Abstract 503: Tracking evolution of transcriptional landscapes during colorectal cancer development and progression
              </a>
            </td>
          <td>
            Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, J. Edwards, Owen J. Sansom, Philip D Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Medulloblastoma or MB is the most common malignant brain tumor in children. Tumor recurrence and therapy resistance remain critical obstacles for the treatment of MB. Although surgical resection followed by high doses of chemotherapies and ionizing radiation IR has considerably improved the survival rates. Resistance of residual tumor to IR and chemotherapies often leads to tumor recurrence. Despite advances in standard therapies, survival rates for relapsed MB patients have remained stagnant for over three decades, leaving clinicians and families with limited options. Previous reports have demonstrated that APE1, a key protein involved in repairing AP sites and single-strand breaks SSBs in DNA, is overexpressed in MB, and its activity is associated with tumor progression. Knockdown of APE1 enhances sensitivity of MB cells to radiation and chemotherapy, suggesting that APE1 mediates IR/chemo resistance. We have recently demonstrated that IR or cisplatin treatment induced APE1 interaction which Facilitates Chromatin transcription FACT complex, and treatment with CBL0137, a FACT targeted small molecule, improves the potency of cisplatin and radiation in vitro and in intracranial orthotopic MB xenograft model. Our current study discovered that recombinant APE1 binds to non-canonical four stranded DNA secondary structure called G quadruplexes or G4 DNA in the genome of MB cells and interacts with G4 in cells. Inhibition of APE1 binding to G4 by small molecules affects damage repair in cells. Treatment with these small molecules dose-dependently affects MB tumor cell growth in vitro. Further, we have delineated a mechanistic link of G4 with APE1 mediated drug resistance. Our project unraveled a novel mechanism by which APE1 and G4 promote therapy resistance and identify a promising strategy to improve the efficacy of IR and cisplatin and prevent tumor recurrence.


 Ritesh Dey, Yingling Chen, Sutapa Ray, Kishor Bhakat. Exploring APE1-G4 interactions and FACT complex inhibition to overcome therapy resistance in pediatric medulloblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4493.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/810a20889322c24ae633443bf24e59f0a26d11b0" target='_blank'>
              Abstract 4493: Exploring APE1-G4 interactions and FACT complex inhibition to overcome therapy resistance in pediatric medulloblastoma
              </a>
            </td>
          <td>
            Ritesh Dey, Yingling Chen, Sutapa Ray, K. Bhakat
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Lung adenocarcinoma in never smokers (NS-LUAD) has a high mortality rate. Compared to LUAD from patients with smoking history, NS-LUAD are less sensitive to immune checkpoint blockade (ICB) potentially due to differences in tumor mutational burden and immune microenvironment. Knowledge of NS-LUAD phenotypic plasticity and cell composition can provide guidance for prognosis and treatment. However, previous studies of LUAD gene expression landscape were predominantly conducted in smokers and full transcriptomic sequencing (RNA-seq) was only examined in a few dozens of NS-LUAD. By investigating cell dynamics through RNA-seq data from 684 NS-LUAD, we identified three gene expression-based subtypes that encapsulate tumors’ clinical, morphological, and genomic features. The ‘steady’ subtype features low proliferation markers and high fraction of alveolar cells, while it is depleted of TP53 mutations and ALK fusions. With its moderate-to-well differentiated morphological features, this subtype is associated with prolonged overall survival and predicted favorable response to immune checkpoint inhibitors, independent of the PD1/PD-L1 expression levels. The ‘proliferative’ subtype features increased proliferation, and enrichment of TP53 mutations and ALK and other gene fusions. The ‘chaotic’ subtype is characterized by elevated levels of mesenchymal cell state, increased proportions of cancer associated fibroblasts and histologic features of mixed-lineage, and is associated with worst overall survival, even within stage I tumors. We derived a signature of 60 genes’ expression sufficient to recapitulate the transcriptomic classification and validated it in an independent NS-LUAD dataset. This 60-gene signature strongly predicted survival even within subgroups based on tumor stage and beyond known molecular or morphological features, confirming its importance for NS-LUAD prognostication in clinical settings.


 Wei Zhao, Tongwu Zhang, Xing Hua, Phuc H. Hoang, Mona Miraftab, Monjoy Saha, John P. McElderry, Jian Sang, Olivia Lee, Caleb Hartman, Azhar Khandekar, Sunandini Sharma, Frank J. Colón-Matos, Samuel Anyaso-Samuel, Defei Wang, Kristine Jones, Amy Hutchinson, Belynda Hicks, Jennifer Rosenbaum, Xiaoming Zhong, Yang Yang, Angela Pesatori, Dario Consonni, Karun Mutreja, Scott Lawrence, Nathaniel Rothman, Ludmil B. Alexandrov, Charles Leduc, Marina K. Baine, Philippe Joubert, Lynette M. Sholl, William D. Travis, Robert Homer, Qing Lan, Stephen J. Chanock, Lixing Yang, Soo-Ryum Yang, Jianxin Shi, Maria Teresa Landi. A transcriptomic signature predicts mortality risk and immune checkpoint inhibitor response beyond molecular and morphological features in lung adenocarcinoma from never smokers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB263.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc7f554e3c033394f0dca864fed36b87b63713d" target='_blank'>
              Abstract LB263: A transcriptomic signature predicts mortality risk and immune checkpoint inhibitor response beyond molecular and morphological features in lung adenocarcinoma from never smokers
              </a>
            </td>
          <td>
            Wei Zhao, Tongwu Zhang, Xing Hua, P. Hoang, Mona Miraftab, Monjoy Saha, John P. McElderry, Jian Sang, Olivia Lee, Caleb Hartman, Azhar Khandekar, Sunandini Sharma, Frank J. Colón-Matos, Samuel Anyaso-Samuel, Defei Wang, Kristine Jones, A. Hutchinson, B. Hicks, J. Rosenbaum, Xiaoming Zhong, Yang Yang, A. Pesatori, Dario Consonni, Karun Mutreja, Scott M Lawrence, Nathaniel Rothman, Ludmil B. Alexandrov, Charles Leduc, M. Baine, Philippe Joubert, L. Sholl, William D Travis, Robert J Homer, Qing Lan, S. Chanock, Lixing Yang, S. Yang, Jianxin Shi, Marion Landi
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="


 Advancements in drug design have resulted in the development of approved and investigational agents targeting KRAS-driven tumors. RIT1 is a GTPase protein that shares homology and oncogenic activation mechanisms with KRAS and plays a role in cell division and resistance to apoptosis. In this study, we aimed to characterize the landscape of RIT1 alterations across a pan-cancer cohort to assess its frequency and potential as an oncogenic driver.



 Tumor content, ploidy and allelic specific copy number of 86, 039 tumor-normal matched samples sequenced clinically with a targeted sequencing panel (MSK-IMPACT) were analyzed with FACETS. Low purity (<20%) and indetermined copy number samples were excluded. Presence of RIT1 amplifications (Amp), structural variants, oncogenic mutations (Mut), and concurrent OncoKb oncogenic/likely-oncogenic alterations in other genes were evaluated. Co-occurrence and mutual-exclusivity (Co-Me) analyses were performed using DISCOVER. RIT1-altered samples were evaluated for genomic complexity compared to wild-type (WT) samples. False discovery rate correction was applied when appropriate.



 After removing low-quality samples, 63, 270 cancer samples were included, and 1.32% (n=838) were found to harbor RIT1 oncogenic alterations. Most RIT1-alterations observed were Amps (735/838) followed by somatic mutations (Muts) (83/838), concomitant Mut and Amp (2/838) and structural variants (18/838). The most frequent RIT1 Mut was p.M90I (n=40), followed by p.F82C (n=17), p.A77T (n=11) and R122Q (n=9).Uterine carcinosarcomas (5.5%), hormone receptor-positive breast cancer (4.9%) and triple-negative breast cancer (4.7%) had the highest prevalence of RIT1 alterations. Restricting the analysis only to Muts, lung adenocarcinoma (LUAD) (n=15; 0.22%) and bladder cancers (n=13; 0.51%) had the highest number of oncogenic RIT1 Muts. RIT1-mutant samples had higher TMBs than RIT1-WT samples (p<0.01). This association was conserved when accounting for histology in bladder cancer and LUAD (p<0.01). RIT1 Amps were associated with higher fraction of genome altered (p<0.01) and co-occurred with amps of neighboring genes NTRK1 (p<0.01) and AKT3 (p<0.01) across most tumor types. DISCOVER assisted Co-Me analysis identified RIT1 Muts as mutually exclusive with EGFR Muts in LUAD (p<0.01). Among RIT1-mutant samples, 10/15 (66%) LUADs harbored RIT1 as the sole identifiable oncogene driver alteration, whereas only 3/13 (23%) bladder cancers were otherwise driver negative.



 RIT1 alterations appear to be rare across most tumor types. Higher TMB and rarity of RIT1 Muts and the observation of RIT1 Amps frequently co-amplified with other chromosome 1 arm q genes, suggest that these alterations might be primarily passengers. These results have implications for stakeholders interested in developing potential therapies for RIT1 as the landscape of these alterations has not been previously explored.



 Matteo Repetto, Pier Selenica, Soo-Ryum Yang, Britta Weigelt, Alexander Drilon. The pan-cancer landscape of RIT1 genetic alterations [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7145.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0db92d7678fd17061debda8f9cf53b805a41693" target='_blank'>
              Abstract 7145: The pan-cancer landscape of RIT1 genetic alterations
              </a>
            </td>
          <td>
            Matteo Repetto, P. Selenica, S. Yang, B. Weigelt, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Background Head and neck squamous cell carcinoma (HNSCC) is a lethal malignancy with a high recurrence and distant metastasis rate, posing significant challenges to patient prognosis. Recent studies suggest that tumor-associated neutrophils (TANs) can modulate immune cell infiltration and influence tumor initiation and progression. However, the potential clinical significance of TANs in HNSCC remains insufficiently explored. Methods TANs-specific marker genes were identified via single-cell sequencing data from HNSCC. Based on data from The Cancer Genome Atlas (TCGA), a prognostic risk model was constructed using TANs cell marker genes, and the model was validated with data from the Gene Expression Omnibus (GEO) database. The associations between the TANs signature and clinical characteristics, functional pathways, immune cell infiltration, immune checkpoint expression, and responses to immunotherapy and chemotherapy, were then investigated. Cell counting kit‐8(CCK-8), Transwell, and wound healing assays were conducted to assess the functional role of TANs marker molecules. Results TANs characteristic genes were identified from single-cell sequencing data from HNSCC patients. On the basis of these characteristic genes, a tumor-associated neutrophils-associated signature (NRS) was developed and validated across internal and external cross-platform cohorts through comprehensive procedures. The NRS demonstrated robust and reliable performance in predicting overall survival. Additionally, patients with a low NRS showed enhanced immune cell infiltration, active lipid metabolism, and increased sensitivity to immunotherapy. In contrast, patients with a high NRS exhibited poor prognostic outcomes, advanced clinical stages, and significant associations with HNSCC metastasis and progression. Furthermore, we identified a TANs-associated biomarker, OLR1, and validated that OLR1 promotes HNSCC proliferation, invasion, and migration through CCK-8, Transwell invasion, and wound healing assays. Conclusion This study has developed a promising TANs-based tool that may aid in personalized treatment and prognostic management for patients with HNSCC. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14179-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f49873abbdf8d985e4439dead6d8844316f9eb" target='_blank'>
              Integrated analysis of single-cell RNA-seq and bulk RNA-seq unravels the molecular feature of tumor-associated neutrophils of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Haiyang Cui, Zhikai Li, Yukun Liu, Yuchuan Yang, Linkuan Huangfu, Jinglin Kong, Xiaocong Sun, Shibo Gao, Daoke Yang, Yingjuan Zheng
          </td>
          <td>2025-05-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Autophagy suppresses tumorigenesis in normal cells, but in established tumors, it can promote tumor progression, particularly by enhancing resistance to stress. However, the mechanism underlying this tumor-promoting function remains unclear. To investigate this, we adopted an interdisciplinary approach combining database analysis with experimental validation. Specifically, by classifying the autophagy-related genes using AutoML analysis on their expression patterns in the COSMIC database, we identified an autophagy subnetwork that correlated with the PLK1-RAD9A axis, a pathway we had previously linked to genotoxic resistance. Cell-based experiments confirmed that autophagy enhanced PLK1 expression at both the transcriptional and translational levels, facilitating genotoxic resistance. Notably, in stressed S-phase cells, we found that PLK1 expression levels varied among individual cells, yet overall cell population acquired genotoxin resistance. The genotoxin resistance in the cell population with heterogeneous PLK1 expression was driven by autophagy by facilitating the secretion of currently unidentified factors, likely by switching funtion of RAD9A from DNA checkpoint to substance secretion. Together our data demonstrate that intra-tumor heterogeneity contributes to the malignant features of tumors through an autophagy-PLK-RAD9A axis that promotes intercellular communication via secretion. (180 words).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c385cb8f12e65d2ac980560afdb3e17f79dfea" target='_blank'>
              Autophagy promotes the acquisition of genotoxic resistance in cancer cells in a paracrine manner via upregulation of PLK1-RAD9A axis.
              </a>
            </td>
          <td>
            M. Ikura, Takuma Shiraki, T. Ikura, Kanji Furuya
          </td>
          <td>2025-05-29</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) poses a significant global health burden, highlighting the urgent need for innovative and effective treatment options. This study focuses on the discovery and validation of novel therapeutic targets, particularly synthetic lethal (SL) interactions, to exploit specific genetic vulnerabilities in CRC cells and enable the development of safer and more effective therapies. We employed a comprehensive strategy integrating healthy cells, genetically engineered isogenic cancer models derived from human intestinal stem cells (hISCs) with patient-derived xenografts (PDXs) and primary cell cultures. Using CRISPR/Cas9 technology, we introduced key CRC-associated mutations into the hISCs to replicate disease-specific contexts. Transcriptome profiling, coupled with machine learning, validated these models as robust, clinically relevant representations of tumor biology by aligning them with clinical datasets. Genome-wide CRISPR/Cas9 screenings conducted across the isogenic and patient-derived models revealed actionable vulnerabilities, including the identification of genes that are synthetically lethal with APC and KRAS mutations. Discovered SL targets were rigorously validated in vitro and in vivo using patient-derived cells with matching mutational profiles. This integrative approach bridges preclinical and clinical research, ensuring translational relevance. By identifying and validating SL targets tailored to CRC mutational landscapes, we provide a robust foundation for the development of novel treatment options for CRC patients.


 Andrzej Mazan, Marcin Duleba, Eliza Zimolag, Joanna Szuszkiewicz, Marcin Serocki, Joanna Krawczyk, Oleksii Bryzghalov, Agata Stachowicz-Walaszek, Izabella Wieckowska, Marcelina Chmiel, Alexander Kurzejamski, Barbara Lipert, Katarzyna Sarad, Michal Mikula, Rafal Dziadziuszko, Tomasz Rzymski, Andrew Thomason, Krzysztof Brzozka. Discovery of novel synthetic lethal targets for effective and safe colorectal cancer therapies [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2973.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ab372dd4cae4e9e87fbdd2a61f62f73ea8595e" target='_blank'>
              Abstract 2973: Discovery of novel synthetic lethal targets for effective and safe colorectal cancer therapies
              </a>
            </td>
          <td>
            Andrzej Mazan, M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Izabella Wieckowska, M. Chmiel, A. Kurzejamski, B. Lipert, Katarzyna Sarad, M. Mikula, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Molecular profiling of DNA and RNA from pediatric cancers by next-generation sequencing has been demonstrated to improve diagnosis and prognosis and to identify somatic alterations indicating vulnerability to targeted therapies. Hence, much like in the treatment of adult cancers, molecular profiling is now routinely utilized in clinical workflows for pediatric cancers as a companion to conventional pathology diagnosis. Many variants of unknown significance identified through DNA profiling are being characterized by saturation genome editing, enabled by CRISPR editing technology and clever functional assays. Newer technologies and analytics are revealing additional structural complexity around cancer drivers and gene fusions in pediatric cancer DNA. Similarly, computational methods such as rare variant association studies and polygenic risk scoring are being used to identify novel cancer susceptibility. Together, these advances are expanding our understanding of pediatric cancer's complexity and fueling the development of emerging methods such as liquid biopsy-based monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae78e85702756670907089a42238ceb27fd06f7" target='_blank'>
              Pediatric Cancer Genetics and Genomics.
              </a>
            </td>
          <td>
            Elaine R. Mardis
          </td>
          <td>2025-05-28</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a crucial role in the progression of tumors, resistance to immunotherapy, and the development of thyroid carcinoma (THCA). However, the precise mechanisms involved remain poorly understood. In this study, 26 mRNAs linked to cancer stemness were identified through an analysis of 13 publicly available THCA transcriptomic datasets alongside a CRISPR dataset for thyroid cancer cell lines. Notably, we revealed an inverse relationship between the stemness of cancer cells and the efficacy of immunotherapy. Utilizing multiple machine learning approaches, we created and confirmed a model for tumor stemness cells (TSC), which included genes associated with cancer stemness. This model successfully forecasted patient outcomes and responses to treatment with Cytotoxic T-Lymphocyte Associated Protein 4 (CTLA4) immune checkpoint inhibitors (ICIs) across different types of cancer. Additionally, chromosomal amplifications in regions 1q and 8p were identified as intrinsic contributors to the onset of thyroid cancer. Specifically, based on single cell dataset, the amplification of CKS1B on chromosome 1q was revealed to advance the progression of thyroid cancer cells by boosting their proliferation. Furthermore, experimental results revealed that the heightened expression of CKS1B considerably enhanced the proliferation and invasion capabilities of THCA, suggesting it could serve as a possible therapeutic target for this condition. In summary, our study sheds light on the relationship between the traits of cancer stem cells and the efficacy of immunotherapy, providing a novel model for predicting the prognosis and individual responses to immunotherapy in patients with THCA. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02813-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865efc03cf551caa6a2f623a0978529554bdabd9" target='_blank'>
              Multi-omics analysis reveals cancer stemness function and establishes a predictive model for thyroid cancer prognosis and immunotherapy
              </a>
            </td>
          <td>
            Zhanrong Feng, Yue-Chao Xu, Ying Ding, Xiaoxiao Sun, Qian Zhao, Jingjing Zhu, Qiang Chen, Yang Zhang, Yilai Zhang
          </td>
          <td>2025-06-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Cancer interception represents a new approach, aimed at targeting precancerous lesions and therefore preventing cancer occurrence. Lynch syndrome (LS) is one of the most prevalent hereditary cancer syndromes, with an estimated prevalence of one in 300, and high-risk predisposition to several types of cancer, with up to 80% lifetime risk to develop CRC. LS is caused by germline mutations in one of the DNA mismatch repair (MMR) genes. Loss of MMR proteins causes an accumulation of mutations in the microsatellite sequences, known as Microsatellite instability (MSI), that can lead to shared frameshift mutations (FSP). Nous-209 is a neoantigen directed immunotherapy encoding 209 FSP shared across sporadic and hereditary MSI tumors and precancer lesions in clinical development for interception and treatment of MSI tumors (Leoni et al., 2020; NCT04041310). Here, we report the full set of safety and immunogenicity results of a Phase Ib/II study of Nous-209 monotherapy in LS carriers.
 NCT05078866 is a Phase Ib/II single-arm, open-label, trial of Nous-209 for cancer interception in LS carriers assessing as co-primary endpoints the safety and immunogenicity against the FSP neoantigens encoded by Nous-209. Nous-209 is administered intramuscularly: one prime with a Great Ape Adenovirus (GAd20-209-FSPs) on day 1, followed by a boost with a Modified Vaccinia Ankara (MVA-209-FSPs) at week 8. Immunogenicity was measured by ex-vivo IFN-γ ELISpot assay utilizing pre- and post-vaccination peripheral blood mononuclear cells (PMBCs) samples collected at different timepoints.
 The trial fully enrolled 45 LS carriers, all receiving GAd20-209-FSPs and MVA-209-FSPs. Most pts (21 of 45, 46.7%) harbored MSH2 pathogenic germline variants. 19 of 45 (42.2%) pts had a prior history of cancer. Treatment was safe and well tolerated with no treatment-related serious adverse events (AEs) and no unexpected reactogenicity. The most common AEs were injection site reactions (any grade: 84% post-prime, 73% post-boost; grade 3: 0%) and fatigue (any grade: 78% post-prime, 51% post-boost; grade 3: 4%). Neoantigen specific immune responses post Nous-209 were observed in 100% of evaluable participants (37/37), with induction of robust T cell immunity reaching a mean of T cell response at peak of ∼ 1100 IFN-γ spot forming cells (SFC) per million of PBMC, polyfunctional and long lasting CD8 and CD4 T cell responses broadly recognizing multiple FSPs. The use of peptide binding HLA predictions allowed the precise identification of several immunogenic FSPs, shown to be present in independent datasets of MSI premalignant lesions and cancers in LS carriers.
 The complete safety and immunogenicity results from this Phase Ib/II trial support the further development of Nous-209 monotherapy, highlighting its potential to efficiently stimulate the immune system and intercept cancer in LS carriers.


 Jason Willis, Anna Morena D'Alise, Michael J. Hall, Marcia Cruz-Correa, Gregory E. Idos, Selvi Thirumurthi, Veroushka Ballester, Guido Leoni, Irene Garzia, Laura Antonucci, Lorenzo De Marco, Elisa Micarelli, Sven Gogov, Wenli Dong, J. Jack Lee, Lana A. Vornik, Araceli Garcia-Gonzalez, Laura Reyes-Uribe, Ellen Richmond, Asad Umar, Powel H. Brown, Luz Maria Rodriguez, Elisa Scarselli, Eduardo Vilar. Nous-209 off-the-shelf neoantigen immunotherapy induces robust neoantigen T cell response with the potential to intercept cancer in Lynch syndrome carriers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6427.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0960349fa1d9c3399276e6b3bfa98752934b018" target='_blank'>
              Abstract 6427: Nous-209 off-the-shelf neoantigen immunotherapy induces robust neoantigen T cell response with the potential to intercept cancer in Lynch syndrome carriers
              </a>
            </td>
          <td>
            Jason Willis, A. D’Alise, Michael J. Hall, Marcia Cruz-Correa, Gregory E. Idos, S. Thirumurthi, Veroushka Ballester, Guido Leoni, I. Garzia, Laura Antonucci, Lorenzo De Marco, E. Micarelli, Sven Gogov, Wenli Dong, J. J. Lee, L. Vornik, Araceli Garcia-Gonzalez, Laura Reyes-Uribe, Ellen Richmond, Asad Umar, Powel H Brown, Luz María Rodríguez, Elisa Scarselli, Eduardo Vilar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Ovarian cancer is a lethal heterogenous disease that is typically diagnosed at late stages. In high-grade serous ovarian carcinoma (HGSOC), the most common subtype, PARP inhibitors and chemotherapy are standard of care. Although HGSOC patients are responsive to these therapies, resistance develops and patients invariably relapse. Alterations in genes within the phosphoinositide 3-kinase (PI3K) and AKT network are highly prevalent. Although specific PI3K and AKT small molecule inhibitors have been developed and clinically evaluated, to date none have been approved for clinical use in ovarian cancer patients. To identify collateral druggable vulnerabilities to AKT hyperactivation and develop novel combination therapies in HGSOC, we have performed a CRISPR-based genome-wide synthetic lethal screen. The goal is to identify genes that upon genetic knockout transform a cytostatic cellular response to PI3Ki/AKTi to a cytotoxic one. Using this platform, we have identified genes that regulate integrin signaling as a major mechanism of synthetic lethality against the PI3Ki pictilisib and the AKTi capivasertib. ITGAV, encoding the integrin subunit αV, was a significant dropout against both kinase inhibitors. Pharmacological targeting of integrin αV synergized with pictilisib and capivasertib to induce apoptotic cell death in multiple cell line models of ovarian cancer, as well as patient-derived organoids. Combined PI3K/AKT and integrin αV inhibition induced DNA damage and cell cycle arrest. The mechanisms underlying induction of DNA damage and cell death in response to combination treatment are currently under investigation. The ultimate goal is to bring this combination treatment approach to HGSOC patients and shed new light on mechanisms of resistance to PI3K/AKT pathway inhibitors in ovarian cancer.


 Gilbert K. Salloum. ITGAV is a targetable vulnerability in epithelial ovarian cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 829.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2db16c30d79726e181227a7646a42de7a4c218" target='_blank'>
              Abstract 829: ITGAV is a targetable vulnerability in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Gilbert K. Salloum
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transcription-coupled repair (TCR) and R-loops are two interrelated processes critical to the maintenance of genome stability during transcription. TCR, a specialized sub-pathway of nucleotide excision repair, rapidly removes transcription-blocking lesions from the transcribed strand of active genes, thereby safeguarding transcription fidelity and cellular homeostasis. In contrast, R-loops, RNA–DNA hybrid structures formed co-transcriptionally, play not only regulatory roles in gene expression and replication but can also contribute to genome instability when persistently accumulated. Recent experimental evidence has revealed dynamic crosstalk between TCR and R-loop resolution pathways. This review highlights current molecular and cellular insights into TCR and R-loop biology, discusses the impact of their crosstalk, and explores emerging therapeutic strategies aimed at optimizing DNA repair and reducing disease risk in conditions such as cancer and neurodegenerative disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb58ac22e35569ba8451a29e77e0d78908989716" target='_blank'>
              Transcription-Coupled Repair and R-Loop Crosstalk in Genome Stability
              </a>
            </td>
          <td>
            Jeseok Jeon, Tae-Hong Kang
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Early onset colorectal cancer (EOCRC) is increasing yet the mechanism for the accelerated shift to a younger age is unknown. We hypothesize that sporadic gene mutations that are necessary for colon carcinogenesis but not sufficient to cause cancer, create a preneoplastic cell state for increased risk of transformation. To identify transitional cell phenotypes, we used Spatial Transcriptomics (ST) on distal colon samples from Car1-Cre (CAC) mice with Cre expression limited to ∼6% of the distal colon/rectum epithelium, where EOCRC is more prevalent. CAC;ROSAmT/mG (CR) mice (where eGFP marks cells expressing Cre) were crossed to those with floxed oncogene alleles to generate mice with one (Apc+/-, ACR) or two (Apc+/-;KrasG12D/wt, AKCR) non-transforming mutations. ACR mice had normal crypt morphology and low adenoma incidence (<5% of mice), while 4-week-old AKCR mice exhibited crypt elongation, and 8-10 wk-old AKCR developed 3-8 advanced adenomas/mouse. Distal colon Swiss rolls (n=9) and tumors (n=3) with phenotypes ranging from normal to malignant were collected and processed using the 10X Visium Spatial Gene Expression System. Sequencing data was processed, and sample data were pooled for analysis using Seurat. Spatial counts were normalized, dimensionality was reduced with PCA, then clustered and visualized as UMAP projections. Clusters were annotated to known cell types, and differential gene expression (DEG) analysis was followed by functional annotation using GO and Pathway terms. In normal tissue, clusters for the crypt base, middle, and top were identified. These spatially restricted clusters were replaced in early AKCR crypts and again in late AKCR crypts. The clusters from late AKCR crypts were distinct from those in ACR tumors, which were different from clusters in AKCR tumors. This indicates the existence of transition states from normal to “not normal, not cancer” to early adenoma to late adenoma. Normal epithelial clusters were enriched for genes controlling typical colonic functions, e.g. metabolism, and transmembrane transport pathways. AKCR crypt clusters lost these functions and gained others, e.g. immune and antimicrobial genes normally associated with Paneth cells (e.g., Defa22, Defa24, Defa30), gene markers associated with CRC tumors (e.g. Egfr, Hamp, Plet1, Wfdc18), genes for xenobiotic inactivation (Maoa, various Cyp, Gst, Ugt, and Sult genes). Compared to late AKCR crypts, AKCR tumor clusters lost even more colonic functions (e.g. downregulation of differentiation marker Krt20 and xenobiotic metabolism markers) and expressed genes related to mitosis, DNA replication, and DNA repair, highlighting a transition to uncontrolled proliferation and genomic instability. Our findings delineate key early molecular transitions in epithelial cells during colon carcinogenesis that may increase the risk of transformation.


 Yujin Lee, James C. Fleet. Spatial transcriptomic profiling of the mouse distal colon during early carcinogenesis reveals transitional cell states that precede cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6602.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2792f89ad7f680f56297c16cdb7083ab9c731d1f" target='_blank'>
              Abstract 6602: Spatial transcriptomic profiling of the mouse distal colon during early carcinogenesis reveals transitional cell states that precede cancer
              </a>
            </td>
          <td>
            Yujin Lee, James C. Fleet
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 The most common subtype of ovarian cancer, High-Grade Serous Ovarian Carcinoma (HGSC), bears an exceptionally high level of chromosomal instability including chromosomal recombination and copy number variation. Many oncogenes are amplified in HGSC, of which some of the most frequently examined ones include MYC, PAX8 and CCNE1. Although downstream pathways of those genes have been repeatedly studied, epigenetic features of many of them are relatively unknown, as they could involve mechanisms that additively and continuously promote oncogene expression that may contribute to the aggressiveness of HGSCs. In our study, we specifically focus on super-enhancers (SEs), large transcriptional hubs known to drive oncogene expression. We aim to assess whether SEs are hijacked coordinately with copy number alteration in HGSCs. We were able to apply a multi-omic approach that includes bulk RNA sequencing, histone post-translational modification CUT&Tag, and Hi-C analysis to not only identify SE regions in HGSC cell lines but link them to the genes they regulate. With integration of these layers of data, we show that there is a positive correlation between HGSC SEs and gene expression changes in HGSCs, in the context of both cancer growth as well as drug resistance acquisition. We observed that SEs are almost always statistically associated with higher gene expression of their target genes compared to enhancers. We also explored changes in SEs in the context of platinum resistance acquisition in our HGSC cell lines, with the observation that gain of new SEs are associated with increase in gene expression and vice versa. Together, these findings support the possibility that SEs could be critical for essential oncogene expression and changes that lead to drug resistance. We then explored if the copy number amplification (CNA) known associated with cancer genome also plays a role in SE regulation of oncogene expression. We found in almost all HGSC cell lines tested that compared to non-oncogene-related SEs, oncogene-related SEs are significantly more likely to be in copy number amplified regions of the genome (p < 0.001), rendering it a potential mechanism contributing to SE associated oncogene upregulation. Based on this result, we are launching screens for specific SE-oncogene interaction in locating functionally important epigenetic marks with CRISPRi, as well as to leverage our existing Hi-C contact data in combination with long-read sequencing to understand the chromatin architecture in the context of CNV.In summary, in this study we observed promising trends supporting SE hijacking in oncogene upregulation and CNA as a potential factor that contributes to SE regulation. We hope that understanding the mechanisms and context behind epigenetic regulation may lead to rational use of epigenetic inhibitors for cancers driven by copy number alterations to otherwise untargetable oncogenes.


 Zhongmou Jin, Kristin Adams, Kendra Wendt, Josie Wood, Sydney Olson, Jessica D. Lang. Super-enhancers in oncogene upregulation in high-grade ovarian serous carcinoma proliferation and platinum-resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2733.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe56ec63c27c1afc432bcd0d7820f4cd76de13e" target='_blank'>
              Abstract 2733: Super-enhancers in oncogene upregulation in high-grade ovarian serous carcinoma proliferation and platinum-resistance
              </a>
            </td>
          <td>
            Zhongmou Jin, Kristin Adams, Kendra Wendt, Josie Wood, Sydney Olson, Jessica D. Lang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cervical and other anogenital malignancies are largely caused by E6 and E7 oncogenes of high-risk human papillomaviruses (HPVs), which inhibit important tumor suppressors like p53 and pRb when they are persistently activated. The main goal of traditional treatments is to physically or chemically kill cancer cells, but they frequently only offer temporary relief, have serious side effects, and have a high risk of recurrence. Exploring the efficacy and accuracy of CRISPR-Cas9 gene editing in both inducing death in HPV-infected cancer cells and restoring the activity of tumor suppressors is our main goal. In this study, we propose a novel precision oncology strategy that targets and inhibits the detrimental effects of the E6 and E7 oncogenes using the CRISPR-Cas9 gene editing system. In order to do this, we create unique guide RNAs that target the integrated HPV DNA and reactivate p53 and pRb. Reactivation is meant to halt aberrant cell development and restart the cell’s natural dying pathways. This review discusses the potential of CRISPR/Cas9 in targeting HPV oncogenes, with a focus on studies that have demonstrated its promise in cancer treatment. Given the absence of a definitive treatment for papillomavirus infection and its subsequent association with various cancers, future clinical trials and experimental investigations appear essential to establish and evaluate the therapeutic potential of CRISPR-based approaches. This approach provides a less invasive alternative to conventional treatments and opens the door to personalized care that considers the genetic makeup of each patient’s tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612cd6ffb8df66c436b36650924c944f8c2ee3b0" target='_blank'>
              HPV-driven cancers: a looming threat and the potential of CRISPR/Cas9 for targeted therapy
              </a>
            </td>
          <td>
            Atefeh Zamani Kermanshahi, Fatemeh Ebrahimi, Ahmad Taherpoor, Narges Eslami, H. B. Baghi
          </td>
          <td>2025-05-22</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Chromosomal translocations are complex biological processes that require the formation of DNA double-strand breaks (DSBs) at two different loci, followed by the joining of the broken ends by DNA repair mechanisms. In B-cell lymphoma, chromosomal translocations are directed by the on- or off-target activity of activation-induced cytidine deaminase (AID) that converts cytosine to uracil in single-stranded DNA (ssDNA), facilitating DSBs when two unpaired nicks are close together. However, no such enzymes have been discovered in solid tumors. While AID, a B-cell-specific APOBEC enzyme, is responsible for somatic hypermutations and chromosomal translocations in B-cell malignancies, apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like 3 (APOBEC3; A3) enzymes have been primarily studied for their mutagenic potential in solid cancers. In this study, we present detailed characterizations of A3-induced chromosomal translocations in multiple cell models. While shared features of AID- and A3-driven translocations are expected because they deaminate cytosine at similar consensus sequences which should result in similar patterns of DNA DSB hotspots, their translocation patterns are partially distinct. As expected, AID generated focal DSB clusters (FDCs) near transcription start site (TSS) regions, but AID also induced new types of FDCs in the gene body and in both TSS and gene body, as well as dispersed DSB cluster (DDC)-type translocations. Interestingly, A3s enhanced FDCs and DDCs in fibroblasts with the similar level of AID, but A3s were only able to induce FDCs in the gene body, not in the TSS region. Both AID- and A3-mediated chromosomal translocations were enriched in genomic regions associated with early-replicating constant timing regions (CTRs), A compartments, and topologically associated domain (TAD) boundaries. Moreover, we revealed an enrichment of A3-attributed mutational signatures at the vicinity of A3-induced chromosomal rearrangements and the oncogene amplification boundaries in patient samples. Finally, we found that A3 expression significantly increased the number of oncogenic tyrosine kinase (TK) fusions, such as ALK, RET, ROS1, and NTRK1 fusions that are recurrently found in non-small cell lung cancer (NSCLC). Furthermore, deletion of APOBEC3 genes significantly decreased the number of oncogenic TK fusions. This study highlights that A3 enzymes induce chromosomal translocations and the formation oncogenic TK fusions in human cancers, with their translocation patterns distinct from but partially shared with those induced by AID, suggesting A3s as promising therapeutic targets for reducing genomic instability in cancer patients.


 Taek-Chin Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle. Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2847.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 3577


 Background:
 HRD is linked to sensitivity to platinum-based chemotherapy and poly (ADP-ribose) polymerase inhibitors across various tumor types. However, the presence of HRD, driver events (such as pathogenic mutations in HR genes or other genomic alterations), and its clinical relevance in mCRC remain underexplored.
 Methods:
 We performed the VHIO-300 test, an ISO15189 accredited custom NGS panel profiling over 450 genes (including HR-related genes:
 ATM, BRCA1, BRCA2, BRIP1, CHEK2 or PALB2
 ) on 356 Stage IV mCRC patients (corresponding to 247 primary colorectal and 109 to metastatic samples) enrolled in the Vall d’Hebron Institute of Oncology’s Molecular Prescreening Program from June 2021 to December 2024. All samples had a tumor cellularity > 40% as per pathologist evaluation
 .
 An HR score (sHR) based on genome-wide copy number alterations (CNA) and loss of heterozygosity (LOH) patterns is generated. After cross-validation with Myriad MyChoice, sHR ≥ 56 was established based on a cohort of ovarian tumors and used to identify HRD in mCRC tumor samples.
 Results:
 HRD prevalence was 3.4% in our mCRC cohort (12/356), but, much higher in metastatic lesions, (6.4%, 7/109) than in primary samples (2.03%, 5/247) (p = 0.05). In fact, the median sHR between primary (21) vs. metastases (30) in CRC was significantly different (p < 0.01). Regarding HR gene status, HR-mutated samples were not significantly within the HRD group (p = 0.27) and only 6.5% (2/31) were HRD. Noteworthy,
 BRCA2
 exhibits a frameshift deletion in a homopolymer stretch, that is a frequent hotspot in microsatellite instable (MSI) tumors, but this event was not found to be associated with HRD. In fact, all HRD tumors (n = 12) were microsatellite stable (MSS). Other frequent alterations in mCRC were studied and
 BRAF
 mutations were found to be present in 42% of HRD tumors (p < 0.01). Inversely, HRD was rare in
 KRAS
 -mutated samples (0.7%; 1/140) and, in fact, highly correlated with non-HRD status (p = 0.02), especially the G12 mutation (p < 0.01). Interestingly, CNA profiles also revealed a strong association between the
 BCL2L1
 loci gain and HRD (p < 0.01). Clinically, HRD was not significantly associated with prognostic value nor clinical benefit to oxaliplatin-based combinations. However, the limited sample size and heterogeneous treatment lines restricted robust statistical analysis.
 Conclusions:
 This study identified a small, yet significant subset of mCRC that displays HRD. sHR and HRD rates were higher in metastatic lesions vs primary tumors, indicating HR scarring could be accumulating over time in some mCRC patients. No clear association between pathogenic HR gene mutations and HRD were found, suggesting the involvement of alternative molecular mechanisms in this process. Frequent co-occurring events, such as
 BRAF
 mutations or
 BCL2L1
 gains could be drivers in CRC HRD, and shape, eventually, new therapeutic options for these patients in the metastatic setting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14cea3ee627505991df2a2a9cb9438dafa79fb0" target='_blank'>
              Drivers of homologous recombination deficiency (HRD) in metastatic colorectal cancer (mCRC).
              </a>
            </td>
          <td>
            Paula Romero Lozano, Javier Ros Montañá, Eduardo García-Galea, Maria Vila-Casadesús, R. Comas, Ágatha Martín, Jenifer González-Zorelle, Paolo Nuciforo, H. G. Pálmer, J. Tabernero, E. Élez, Ana Vivancos-Prellezo
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Focal gene amplification serves as an oncogenic driver during tumorigenesis and is a hallmark of many forms of cancer. Oncogene amplifications promote genomic instability, which is integral to cancer cell survival and evolution. However, focal gene amplification potentially affords an opportunity for therapeutic exploitation. As a proof-of-concept, we leverage CRISPR-Cas9 nickase to selectively promote cancer cell death in MYCN-amplified neuroblastoma in a gene amplification-dependent manner. Our analysis demonstrates that CRISPR-Cas9 nickase can generate a lethal number of highly toxic, replication-dependent double-strand breaks in cells harboring amplified loci. Furthermore, we demonstrate that Cas9 nickase—mediated toxicity can be modulated in combination with small molecule inhibitors targeting key regulators of the DNA-damage response or cell death pathways. Importantly, our findings in MYCN-amplified neuroblastoma translate to other cancer types with distinct oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426fa411ebfe0306ef5fb3727f439d058ec49944" target='_blank'>
              Selective targeting of genome amplifications and repeat elements by CRISPR-Cas9 nickases to promote cancer cell death
              </a>
            </td>
          <td>
            Matthew B Hanlon, Jason M Shohet, S. Wolfe
          </td>
          <td>2025-06-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="
 Glioblastoma multiforme (GBM) is the most common and lethal brain cancer. Standard of care treatments (SOC) are limited to surgery, radiation and chemotherapies, providing an average overall survival of only ∼16 months. Tumor heterogeneity and cellular plasticity are major contributions to treatment resistance in this disease. We constructed an integrated patient-level and cohort-level framework and analyzed single-nucleotide RNA sequencing data in matched primary and recurrence specimens to understand tumor heterogeneity, evolution, and drug resistance in SOC treated GBM patients (N=23). Patient-level tumor analysis revealed distinct tumor cell transcriptional subclones representing widespread intra-patient heterogeneity. Tumor evolution analysis from primary to recurrence revealed diverse treatment responses representing sensitive, intrinsic resistant, and acquired resistant subclones. Further characterization of tumor subclones and clustering analyses indicated that tumor heterogeneity under SOC treatments was linked to diverse biological mechanisms including neural development and oncogenic pathways, contributing to varying levels of subclonal resistance. Our analysis also highlighted intrinsic resistance was prominent as primary tumors often harbored substantial proportion of resistant tumor cells. This insight on resistant status at cell level then allowed us to compare resistant to sensitive cells and identify both established and potentially novel mechanisms of SOC resistance that were not always obvious in the comparison between primary and recurrence samples. These included activation of PI3K/AKT, MAPK, TGF-β, EMT, and WNT signaling pathways. Taken together, our novel analytical strategy revealed complex tumor heterogeneity and evolution landscape in SOC treated GBM, offering insights on potential mechanisms driving these phenomena and providing a resource for further exploration and development of therapeutic strategies.


 Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, Celia Fontanillo, Kai Wang, Ha Dang. Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1253.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339780e1e052f73bd7a155d38cbe11ed7059d53c" target='_blank'>
              Abstract 1253: Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma
              </a>
            </td>
          <td>
            Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, C. Fontanillo, Kai Wang, Ha Dang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Understanding cellular behavior within the context of the tumor-immune microenvironment is essential to developing next-generation cancer therapies and advancing precision medicine. The inherent challenges of this problem reflect the complexity of human biology - patient and tissue heterogeneity, a multitude of interacting signaling pathways, dynamic short- and long-range interactions between the tumor and the immune system, and the intrinsic limitations of measurement techniques. Machine Learning foundation models trained on multimodal patient data present an opportunity to grapple with this complexity and push the field forward leveraging recent advances in spatial biology.



 A custom multimodal transformer was trained on 1399 primary resections from lung cancer patients profiled via H&E, CosMx spatial transcriptomics, whole-exome sequencing, and a custom multiplex immunofluorescence panel. To our knowledge this is the largest extant spatial transcriptomics dataset, comprising more than 40 million cells. The transformer was trained via self-supervised learning to predict expression of each CosMx panel gene for a single cell, conditioned on both spatially proximal and patient-level data from all modalities. This training task induces the model to learn fundamental rules that govern cell state and cell-cell interactions within the context of disease. The resulting model, Celleporter, can generate gene expression in a “virtual” cell at a particular location within real or simulated patient tissue. Counterfactual simulations with modified patient data can be used to predict the effects of genetic alterations, gene expression changes, or external interventions on the tumor-immune microenvironment.



 Celleporter accurately predicted spatial gene expression patterns from sparsely sampled data, resolving the limitations of traditional experimental approaches. Virtual cell simulations reproduced distinct biological states, such as cytotoxic and naïve transitions of CD8+ T cells within and outside tumor regions and reproduced foundational immunology, including the relationship between MHC-I and T cell activation. Comparative analyses across patient cohorts identified immune-suppressive mechanisms in STK11-mutant tumors resistant to immunotherapy. Perturbation simulations highlighted therapeutic targets predicted to restore cytotoxic activity in STK11-mutant tumor microenvironments.



 This study demonstrates that a self-supervised foundation model trained on large-scale multimodal patient data can learn fundamental aspects of cancer immunology, and accurately reproduce the impact of the tumor-immune microenvironment on cell state in a patient-specific manner. This flexible system for interrogating cell and tissue biology has direct application to patient stratification and target discovery.



 Yubin Xie, Eshed Margalit, Tyler Van Hensbergen, Dexter Antonio, Jake Schmidt, Yu Phoebe Guo, Jérémie Decalf, Maxime Dhainaut, Lucas Cavalcante, Hargita Kaplan, Rodney Collins, Francis Fernandez, Rob Schiemann, Eric Siefkas, Michela Meister, Joy Tea, Carl Ebeling, Anastasia Mavropoulos, Nicole Snell, Shafique Virani, Ron Alfa, Lacey Padron, Jacob Rinaldi, Daniel Bear. Celleporter: a foundation model of cell and tissue biology with application to patient stratification and target discovery [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3652.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/700ea58a4c1d36566c271fc146a1aaff828a43d8" target='_blank'>
              Abstract 3652: Celleporter: a foundation model of cell and tissue biology with application to patient stratification and target discovery
              </a>
            </td>
          <td>
            Yubin Xie, Eshed Margalit, Tyler Van Hensbergen, Dexter Antonio, Jake Schmidt, Yu (Phoebe) Guo, Jérémie Decalf, Maxime Dhainaut, Lucas Cavalcante, Hargita Kaplan, Rodney Collins, Francis Fernandez, Rob Schiemann, Eric Siefkas, Michela Meister, Joy Tea, Carl Ebeling, Anastasia Mavropoulos, Nicole Snell, Shafique Virani, Ronald Alfa, Lacey J Padrón, Jacob Rinaldi, Daniel Bear
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Half of of colorectal cancer (CRC) patients develop metastatic CRC (mCRC) with a low survival rate. Epigenetic abnormalities in promoters and enhancers play a significant role in CRC advancement and hold promise for diagnosis and therapy and over 5, 000 variable enhancer loci (VELs) have been shown to be gained/lost in mCRC. Here, the functional impact of these VELs on driving CRC progression was explored. We designed a CRISPRi library including 1, 524 VELs gained or lost specifically in CRC cell lines, the top 500 essential genes for the cell line as positive controls, 1, 000 non-targeting guides as negative controls and 6 sgRNAs per site, cloned into a lentiviral plasmid. CRISPR screen was applied to the HCT116-dCas9-KRAB-MeCP2 cells for NGS. Anticancer potentials of silenced VELs were evaluated by in vitro assays (i.e MTT assay, apoptosis, cell cycle assay, 3D bioprinting). Chromosomal interactions and epigenetic state were analyzed by Chromosome conformation capture (3C) coupled with sequencing (Hi-C), chromatin immunoprecipitation (ChIP) with massively parallel DNA sequencing (ChIP-seq) and RNA sequencing. We identified 12 depleted VELs causing a significant decrease in HCT116 proliferation after CRISPR-mediated silencing. VEL_1243 and VEL_1084 caused the highest level of cell growth inhibition as well as increased apoptosis and cell cycle arrest in HCT116 cells. After silencing both enhancers, 3D bioprinting showed a reduction in key markers of metastasis. HiC-Seq and transcription factor binding data showed long-range interaction of both enhancers with genes such as MYO3B, UBR3 and GINS3 and enrichment for disease-associated TF including RFX7, and ELF3. RNA seq results showed significant transcriptome-wide expression alterations regulated by VEL_1243 and VEL_1084, exemplified by their impact on disease-associated signaling pathways including PI3K-Akt and MAPK. IFNγ response pathway -which is a critical aspect of anti-tumor immunity - was downregulated following both VEL_1243 and VEL_1084 silencing. Given that immune/inflammatory cells (e.g., tumor-associated macrophages (TAMs), T-cells, natural killer (NK) and cancer-associated fibroblasts (CAFs) are the important components of the tumor microenvironment (TME), we further examined if VELs mediated regulation of TAMs or immune cells controls the TME in a way that promotes tumour growth. M1 were co-cultured with CRC cells knock-down of VELs. Both VELs repression could induce M1 polarization in HCT116 and SW620 cells. Considering M1 macrophages have the capability to promote the Th1 response and engulf and kill target tumor cells, VEL_1084 and VEL_1243 could be potential therapeutic targets for CRC by affecting the polarization of macrophages. VEL_1243 and VEL_1054 regulate CRC-associated phenotypes and warrant further exploration to determine their regulatory role and potential as a novel therapeutic target in CRC.


 Flavia Genua, Xinxin Shao, Ellen Finnegan, Christina Caragine, Sarah Maguire, Paulien Van Minsel, Grainne Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das. Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7181.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Rectal cancer (RC) remains a significant clinical challenge, in part due to chemotherapy resistance encountered in neoadjuvant treatment approaches. To address this, we utilized a unique platform integrating gene expression data from RC tumor tissues and matched patient-derived organoids (PDO). Gene expression data were analyzed using the computational tool Moonlight to investigate the molecular mechanisms underlying chemotherapy response. We hypothesized that combining genomic data from RC PDOs and matched tumors would implicate molecular targets predicting chemotherapy response and clinical outcomes better than targets identified using RC tumor tissue or RC PDO input alone. We analyzed 18 tissue samples and 32 matched PDOs, capturing a comprehensive representation of tumor biology. Our integrative approach involved differential expression analyses (DEAs) and gene regulatory network (GRN) studies, identifying 5,199 genes regulating at least one potential regulon. Focusing on cancer-associated biological processes curated in Moonlight, we highlighted 2,118 regulator-regulon pairs with potential roles in oncogenic processes. Further integration with DEA results revealed 334 regulator-regulon pairs significantly enriched in both tissue and PDO samples, designating these regulators as oncogenic mediators (OMs). Among them, five genes—BTK, CDK1, LAX1, NAT2, and RAD51AP1—were associated with FOLFOX response. Survival analysis across four independent cohorts further validated CDK1, LAX1, NAT2, and RAD51AP1 as indicators of prolonged recurrence-free survival (RFS), demonstrating consistent patterns associated with good prognosis. These findings provide new insights into the molecular mechanisms of chemotherapy response in RC, and validation of these targets is ongoing. Our integrated approach underscores the translational relevance of PDOs and demonstrates the utility of Moonlight in identifying potential therapeutic targets, setting a new benchmark for precision medicine in rectal cancer.


 Hanchen Huang, Wini Zambare, Chao Wu, Antonio Colaprico, Janet Alvarez, Min Jung Kim, Aron Bercz, Paulina Bleu, Lily Wang, Philip B. Paty, Paul B. Romesser, J. Joshua Smith, X. Steven Chen. Discovery of oncogenic mediator genes in rectal cancer chemotherapy response using gene expression data from matched tumor and patient-derived organoids [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a8992e140899a62dade76814a52a30a531da19" target='_blank'>
              Abstract 5026: Discovery of oncogenic mediator genes in rectal cancer chemotherapy response using gene expression data from matched tumor and patient-derived organoids
              </a>
            </td>
          <td>
            Hanchen Huang, W. Zambare, Chao Wu, A. Colaprico, J. Alvarez, Min Jung Kim, A. Bercz, P. Bleu, Lily Wang, P. Paty, P. Romesser, J. J. Smith, X. Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Epigenetic status-altering mutations in chromatin-modifying enzymes are a feature of human diseases, including many cancers. We investigated cellular dependencies, or vulnerabilities, that arise when enhancer function is compromised by loss of the frequently mutated COMPASS family members MLL3 (KMT2C) and MLL4 (KMT2D). We performed a CRISPR dropout screen that revealed a targetable metabolic dependency arising from epigenetic factor deficiency, identifying potential therapies for cancers with epigenetic alterations secondary to MLL3/4-COMPASS dysfunction. We identified strong evidence that there is an internal balance between promoter and enhancer usage dictated by MLL1- versus MLL4-COMPASS, and this equilibrium is subject to disruption during cancer. Using an autochthonous carcinogen model of bladder cancer, we demonstrate that truncated, cytoplasmic MLL4 predicts response to targeted metabolic inhibition therapy for bladder cancer and could be developed as a biomarker for KMT2D-mutated cancers. We also highlight the broader potential for prognosis, patient stratification and treatment decision-making based on KMT2D mutation status in MLL4 truncation-relevant diseases, including human cancers and Kabuki Syndrome. Due to the high prevalence of loss-of-function mutations affecting MLL4 or its COMPASS partner UTX (KDM6A) in bladder cancer, we hypothesized that KMT2D mutation status may predict responsiveness to purine-targeting therapy, including methotrexate and pemetrexed. Our study found that KMT2D mutant bladder cancer cells were selectively dependent on TYMS and showed heightened sensitivity to pemetrexed, a drug that inhibits TYMS, DHFR, and GART. This suggests that targeting multiple enzymes within the one-carbon metabolism and de novo purine synthesis pathway could be promising for the treatment of MLL4/UTX-COMPASS mutant bladder cancer. Based on our previous and current findings, a clinical trial using pemetrexed in MLL4/UTX-COMPASS mutant solid tumors is being launched at Northwestern Medicine.


 Zibo Zhao, Sarah Gold, Yuki Aoi, Khyati A. Meghani, Luke St John, Rukkia Liaqat, Carolyn Moloney, Yanni Yu, Jun Qian, Issam Ben-Sahra, Rintaro Hashizume, Devalingam Mahalingam, Joshua Meeks, Ali Shilatifard. MLL4/COMPASS dysfunction in cancer and treatment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1d772bceb37d658c289fadfd2de9d117e82a8a" target='_blank'>
              Abstract 4103: MLL4/COMPASS dysfunction in cancer and treatment
              </a>
            </td>
          <td>
            Zibo Zhao, Sarah Gold, Yuki Aoi, K. Meghani, Luke St John, Rukkia Liaqat, Carolyn Moloney, Yanni Yu, Jun Qian, I. Ben-Sahra, Rintaro Hashizume, D. Mahalingam, Joshua Meeks, A. Shilatifard
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Aneuploidy, prevalent in most solid tumors, significantly alters cellular phenotype and fitness. Despite its critical role in tumor evolution, a detailed quantitative understanding of its evolutionary dynamics remains elusive. To address this gap, we developed ALFA-K, a novel method to infer the fitness landscape of chromosome-level karyotypes. Utilizing longitudinal single-cell karyotype data from evolving cell populations, our method estimates the fitness of thousands of karyotypes proximal to the input data in karyotype space, enabling predictions of unseen karyotypes. We validated our approach using both synthetic data from an agent-based model (ABM) and empirical data from in vitro and in vivo passaged cell lines. The empirical data comprised over 40, 000 sequenced cells across five cell lines, each sequenced longitudinally at 5-10 timepoints per lineage. Analysis of the fitted landscapes yielded several key insights: 1) WGD alters the distribution of fitness effects of copy number alterations (CNAs) (K.S. test: P<10-). Specifically, karyotypes with WGD exhibit more mutations with positive selection coefficients compared to diploid karyotypes; 2) Mutations in cisplatin-treated lineages showed changes in fitness effects on average 2.3 times larger than those in untreated lineages (t-test: P<2.2×10-1); 3) The fitness impact of a CNA varies depending on the parental karyotype; 4) The rate of chromosome missegregation can dictate the predominant karyotypes in an evolving population.
 To further investigate how the tumor microenvironment shapes karyotype evolution, we are integrating these inferred fitness landscapes into an existing spatial ABM. This modeling simulates the spatially heterogeneous tumor microenvironment and its influence on karyotypic selection and adaptation. By leveraging this ABM, we aim to capture the effects of spatial constraints—such as resource availability, competition, and cellular interactions—on the fitness and distribution of karyotypes. Complementing these simulations, we are developing a bioinformatics pipeline for spatial transcriptomics data from glioblastoma clinical samples. This dataset includes samples from various regions (normal, tumor, and tumor-infiltrating) across 31 samples from 19 patients, using 10X Visium with an average of 3, 100 spots per sample. The pipeline is designed to provide detailed information about the spatial organization of karyotypic diversity, elucidating how aneuploid karyotypes are distributed and interact within the tumor microenvironment. By integrating this spatially resolved data with the ABM, we aim to generate a comprehensive framework to explore how microenvironmental pressures in glioblastoma tumors shape the evolution of karyotypic diversity and drive the emergence of therapeutic resistance in patient-specific contexts.


 Richard J. Beck, Noemi Andor. Decoding tumor evolution through fitness landscapes of aneuploid cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2486.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) are known to be involved in the manifestation of aggressive and therapy-resistant phenotypes in solid tumors. Nevertheless, the effects of dynamic intervention by TAMs on the DNA damage response of cancer cells are largely unexplored. Herein, we report that TAMs modulate the DNA damage repair pathways of ovarian cancer cells in response to platinum-(Pt) based therapeutic regimen. We demonstrate that coculture of TAMs with cancer cells directly upregulate Pol η, along with RAD18 and REV1 of the Translesion DNA synthesis (TLS) pathway, while concurrently downregulating components of the high-fidelity nucleotide excision repair (NER) mechanism. Consequently, we observed a better survival probability, DNA repair capacity, and enrichment of stemness properties in ovarian cancer cells. DNA bulky adducts produced by cisplatin are resolved through differential activation NER and TLS pathways. However, we elucidated that TAMs provide favorable conditions for activating the error-prone TLS pathway for lesion bypass over damage resolution. Furthermore, cellular crosstalk in cocultured cancer cells stimulates the nuclear translocation and expression of RelA, which recruits Pol η by acting as a potent transcription factor. In fact, with pristimerin-mediated disruption of p65 (RelA) translocation, the cancer cells become more prone to DNA damage-induced cell death and compromised regenerative potential. In both in vitro cell cultures and in vivo mouse xenograft models, cocultured macrophages exhibited predominantly M2-like phenotype with prevalence in the invasive zone of xenograft tumor margins. Taken together, our investigation revealed multifaceted crosstalk-mediated regulation of DNA damage repair between TAMs and ovarian cancer cells. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05731-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df732d214e8a9fdb550d6752112eedb8ef7093af" target='_blank'>
              Tumor-associated macrophages contribute to cisplatin resistance via regulating Pol η-mediated translesion DNA synthesis in ovarian cancer
              </a>
            </td>
          <td>
            Bilash Chatterjee, Mrinmoy Sarkar, Debanjana Ghosh, Sangita Mishra, Subhankar Bose, Md Maqsood Ahamad Khan, Senthil Kumar Ganesan, Nabanita Chatterjee, A. K. Srivastava
          </td>
          <td>2025-05-29</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract The molecular control of epigenetic information relies on hundreds of proteins of diverse function, which cooperate in defining chromatin structure and DNA methylation landscapes. While many individual pathways have been characterized, how different classes of epigenetic regulators interact to build a resilient epigenetic regulatory network (ERN) remains poorly understood. Here, we show that most individual regulators are dispensable for somatic cell fitness, and that robustness emerges from multiple layers of functional cooperation and degeneracy among network components. By disrupting 200 epigenetic regulator genes, individually or in combination, we generated network-wide maps of functional interactions for representative regulators. We found that paralogues represent only a first layer of functional compensation within the ERN, with intra- or inter-class interactions buffering the effects of perturbation in a gene-specific manner: while CREBBP cooperates with multiple acetyltransferases to form a subnetwork that ensures robust chromatin acetylation, ARID1A interacts with regulators from across all functional classes. When combined with oncogene activation, the accumulated epigenetic disorder exposes a synthetic fragility and broadly sensitizes ARID1A-deficient cells to further perturbation. Our findings reveal homeostatic mechanisms through which the ERN sustains somatic cell fitness and uncover how the network remodels as the epigenome is progressively deregulated in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74c759f71b52bc8e0f4b18c555786227753fc4c1" target='_blank'>
              Systematic genetic perturbation reveals principles underpinning robustness of the epigenetic regulatory network
              </a>
            </td>
          <td>
            Thomas Stuart Wilson, R. Noberini, Eirini Moysidou, Ifeyinwa Ojukwu, Marta Milan, Ming Jiang, Gavin Kelly, Michael Howell, Tiziana Bonaldi, Paola Scaffidi
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) modification, one of the most prevalent RNA epigenetic modifications in eukaryotes, constitutes over 60% of all RNA methylation modifications. This dynamic modification regulates RNA processing, maturation, nucleocytoplasmic transport, translation efficiency, phase separation, and stability, thereby linking its dysregulation to diverse physiological and pathological processes. METTL3 , a core catalytic component of the methyltransferase complex responsible for m6A deposition, is frequently dysregulated in diseases, including colorectal cancer (CRC). Although METTL3 ’s involvement in CRC pathogenesis has been documented, its precise molecular mechanisms and functional roles remain incompletely understood. METTL3 mediates CRC progression-encompassing proliferation, invasion, drug resistance, and metabolic reprogramming-through m6A-dependent modulation of both coding RNAs and noncoding RNAs. Its regulatory effects are primarily attributed to interactions with key signaling pathways at multiple stages of CRC development. Emerging evidence highlights METTL3 as a promising biomarker for CRC diagnosis and prognosis, as well as a potential therapeutic target. By synthesizing recent advances in METTL3 research within CRC, this review provides critical insights into novel strategies for clinical diagnosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ccdd33e7855e192e72bc49bfec42dd0313d9f3f" target='_blank'>
              Switching from messenger RNAs to noncoding RNAs, METTL3 is a novel colorectal cancer diagnosis and treatment target
              </a>
            </td>
          <td>
            Junnan Liao, Wen-Juan Ni, Ping-Hui Wu, Yadong Yang, Ying Yang, Wen Long, Mei-Zhen Xie, Xiu-Zhi Zhu, Fuhua Xie, Xiao-Min Leng
          </td>
          <td>2025-05-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 The development of targeted therapies, including monoclonal antibodies, CAR-T cells, nanobodies, and combination treatments, has significantly advanced cancer management, particularly for HER2-positive tumors. Despite these advancements, challenges such as instability, high toxicity, immune responses, rapid clearance, drug resistance, and exorbitant manufacturing costs persist. A major contributor to these costs is the reliance on mammalian cell systems for production. Additionally, minimizing off-target effects remains a critical concern for targeted therapies.In this study, we engineered virus-like particles (VLPs) to selectively target HER2-positive tumors with minimal off-target effects. These VLPs were further designed to carry genetically expressed RNA interference (RNAi) systems for in situ gene silencing. Cryo-electron microscopy (Cryo-EM) was employed to characterize the functionalization of the VLP exterior and verify the packaging of the RNAi cargo. This novel single-cell approach combines targeted tumor homing with siRNA delivery, enabling simultaneous tumor targeting and silencing of the PDL1 gene. The multivalent nature of VLPs allows for the incorporation of multiple peptides, enhancing tumor affinity and specificity. With a size of approximately 35 nm, VLPs exhibit favorable pharmacokinetics, including reduced clearance and effective tumor homing. A key advantage of our approach is the ability to express VLPs in Escherichia coli systems, significantly reducing production costs compared to traditional mammalian cell-based systems, making the therapy more accessible.Our findings demonstrate that intravenously administered HER2-targeting VLPs achieve high specificity and affinity for HER2-positive tumors. Once localized, the VLPs penetrate deeply into tumor tissue, delivering RNAi cargo to silence PDL1 expression. Light sheet microscopy will be used to visualize tumor architecture in three dimensions to further investigate tumor penetration, offering a more comprehensive view than traditional cryosections. By inhibiting tumor growth and recruiting immune cells, this approach offers a dual mechanism of action: direct tumor suppression and enhanced immune-mediated clearance. Furthermore, VLPs provide unique advantages such as efficient cell penetration, scalable production, and robust cargo delivery, positioning them as a promising platform for cost-effective and effective cancer therapies.


 Ikeda Trashi, Jeremiah Gassensmith. Engineering viruses for the next generation targeted cancer therapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 478.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5750dddf572726400a2fad0203c2ec7fc9717955" target='_blank'>
              Abstract 478: Engineering viruses for the next generation targeted cancer therapy
              </a>
            </td>
          <td>
            Ikeda Trashi, Jeremiah Gassensmith
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Non-replicative herpes simplex virus type 1 (nrHSV-1) vectors are promising delivery vehicles for gene therapy due to their large DNA payload capacity and ability to infect a broad range of cell types. However, the genomic deletions made to generate such nrHSV-1 vectors can result in undersized genomes that trigger genomic instability—including rearrangements and size extensions—compromising their therapeutic potential. This study investigates the stabilization of undersized nrHSV-1 vectors through the insertion of stuffer DNA segments. We assess genomic stability, productivity, toxicity, and transgene expression in vitro and in vivo. Our findings demonstrate that nrHSV-1 can accommodate variations in genome size up to 5–6% and highlight the importance of maintaining a genome size close to that of the wild-type HSV-1 for enhanced genomic stability and sustained transgene expression without adverse effects. This strategy offers a promising approach for optimizing nrHSV-1 vectors for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63984927847424424cc3892babc095b69c28460a" target='_blank'>
              Genomic Size Is Critical to Guarantee the Genomic Stability of Non-Replicative HSV1 Vectors
              </a>
            </td>
          <td>
            Justine Basset, Alexandra Seraffin, Julien Ratelade, Yohann Dickx, Tomasz Benedyk, Grzegorz Sarek, Teddy Jégu, Alberto L. Epstein
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Triple Negative Breast Cancer (TNBC) is an aggressive and therapeutically challenging subtype of breast cancer, characterized by significant intra-tumoral heterogeneity and the absence of actionable molecular markers. This study addresses the urgent need for targeted therapies by investigating the role of non-coding regulatory elements, specifically super enhancers (SEs), in TNBC pathogenesis. SEs are large clusters of enhancers with high transcription factor occupancy and enhancer RNA (eRNA) production, regulating key genes involved in cell identity and cancer progression. Using Chromatin Immunoprecipitation sequencing (ChIP-seq) and Global Run-On sequencing (GRO-seq) across 13 cancer cell lines, we identified a novel TNBC-specific, transcriptionally active SE on chromosome 17, located upstream of the ITGA3 gene. To elucidate its functional role, we conducted CRISPR-interference (CRISPRi) assays coupled with RNA sequencing in TNBC cells, revealing genome-wide SE target genes. Integration with high-throughput chromosome conformation capture (Hi-C) data established direct chromatin interactions linking the SE to its target genes, which collectively form a gene signature predictive of poor clinical outcomes. Perturbation of this SE via CRISPR significantly impaired tumor cell proliferation, highlighting its role as a potential driver of oncogenic phenotypes. These findings provide new insights into enhancer-target gene interactions in TNBC and identify this SE and its associated gene network as promising therapeutic targets.


 Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector Franco. Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1428.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Potassium ions (K+) within the tumor microenvironment, along with dysregulation of K+ channels, play critical roles in supporting cancer cell survival and preventing their elimination. Directly monitoring changes in K+ homeostasis within cancer cells is invaluable for understanding these processes. However, achieving high selectivity over other biological metal ions, a detection dynamic range that aligns with intracellular K+ levels, and broad accessibility to research laboratories remain technically challenging for current K+ imaging probes. In this study, we report the in vitro selection of the first K+-specific RNA-cleaving DNAzyme and the development of a K+-specific DNAzyme fluorescent sensor with exceptional selectivity, achieving over 1000-fold selectivity against Na+ and more than 100-fold selectivity over other major biologically relevant metal ions. This sensor has an apparent dissociation constant (105 mM) that is close to the intracellular level of K+, and it has a broad detection range from 21 to 200 mM K+. Using this tool, we reveal a progressive decline in intracellular K+ levels in breast cancer cells with more advanced progression states. Moreover, we demonstrate that elevated extracellular K+ levels interfere with the efficacy of anticancer compounds like ML133 and Amiodarone, suggesting an underappreciated role of microenvironmental K+ in chemoresistance. Notably, blocking the Kir2.1 channel activity restored treatment sensitivity, presenting a potential strategy to overcome chemoresistance in aggressive cancers. These findings underscore the role of K+ homeostasis in tumor progression and support further exploration of ion-channel-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15a5b2af9bd939cc77bcb6b3ee460a1e0556b12" target='_blank'>
              Decoding Potassium Homeostasis in Cancer Metastasis and Drug Resistance: Insights from a Highly Selective DNAzyme-Based Intracellular K+ Sensor.
              </a>
            </td>
          <td>
            Zhenglin Yang, Xiangli Shao, Yuting Wu, Aritra Roy, Elijah Garcia, Annie Farrell, Shreestika Pradhan, Weijie Guo, Heather Gan, Zeynep Korkmaz, Emily Adams, Yi Lu
          </td>
          <td>2025-05-14</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 The current standard of care for glioblastoma (GBM) is surgical intervention followed by a combination of temozolomide (TMZ) and radiation therapy. Only 5% GBM patients survive more than five years despite aggressive therapeutic treatment. A significant hurdle to develop target-based therapies for GBM is the complexity and diversity of the disease. Genomic and epigenetic profiling has found a broad range mutational and transcriptional events across this histologically diverse disease. A hallmark of GBM is genomic instability, which is a key factor in its progression and complexity. The fast growing and aggressive GBM cells demand rapid DNA replication and hypertranscription, leading to enhanced transcription-replication conflicts (TRCs). A major source of genomic instability, TCR causes lethal double stranded DNA breaks (DSBs) if not resolved properly. We hypothesize that enhanced TRC is a weak link and a promising target for treating GBM.
 Proliferating cell nuclear antigen (PCNA) is a key component of DNA replication machinery and plays an essential role in the DNA replication and repair. Through a rational drug design, we identified a small molecule PCNA inhibitor, AOH1996, which selectively kills cancer cells. Mechanistically, AOH1996 enhances the interaction between PCNA and the largest subunit of RNA polymerase II, RPB1, and dissociates PCNA from actively transcribed chromatin regions. By interfering with TRC resolution as evidenced by induction of R-loop levels in cancer cells, AOH1996 induces DSBs in a transcription-dependent manner. Orally administrable and metabolically stable, AOH1996 can freely pass the blood brain barrier and enter brain tumors. Here, we report that it selectively kills GBM cells as a monotherapy or in combination with TMZ but causes no discernable side effects. Importantly, AOH1996 targets a cancer specific region of PCNA that is a highly conserved through evolution and influences the activity of broad cellular processes that are essential to the growth and survival of cancer cells. GBM cancer cells are, therefore, less likely to develop resistance to AOH1996 through mutagenesis in PCNA. If combined with the existing chemotherapeutic regime and inhibitors of DNA repair pathways, AOH1996 may provide a new and unique therapeutic avenue for exploiting this cancer-selective vulnerability to deliver a durable response in GBM.


 Long Gu, Robert J. Hickey, Linda H. Malkas. Targeting transcription-replication conflict for selective chemotherapy in glioblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5645.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca665dc53860039a134586f0a4361eb2c643769" target='_blank'>
              Abstract 5645: Targeting transcription-replication conflict for selective chemotherapy in glioblastoma
              </a>
            </td>
          <td>
            Long Gu, Robert J. Hickey, L. Malkas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="

 Due to the lack of molecular-targeted therapies for triple-negative breast cancer (TNBC), there have been efforts to use multimodal strategies to enhance the efficacy of existing therapies such as radiation. The androgen receptor (AR) has been identified as a radiosensitizing target and is expressed in 25-50% of TNBC tumors. The combination of AR inhibitors, such as enzalutamide (enza), and radiation therapy (RT), leads to radiosensitization in AR+ TNBC models but the mechanism by which AR mediates radioresistance is not known, and targeting this radioresistance could provide a novel strategy to improve outcomes in this aggressive breast cancer subtype. Following RT, AR is translocated to the nucleus suggesting that AR may drive transcriptional changes in response to RT.



 Bulk RNA-seq with and without R1881 stimulation and RT was used to assess transcriptional changes at 24 hours. RPPA analysis evaluated protein and phophoprotein changes. AR ChIP-seq was used to identify changes in AR binding and Bru-seq to profile changes in nascent RNA. These experiments were performed in two AR+ TNBC models, MDA-MB-453 and MFM-223 cells, with and without enza and RT. Clonogenic survival assays were performed to assess for radiosensitivity.



 RNA-seq and RPPA analysis demonstrated numerous changes in gene and protein expression after enza-, RT-, and combination treatment. Gene set enrichment analysis (GSEA) of these changes nominated alterations of MAPK pathway genes with R1881 and R1881+RT compared to controls. To assess the MAPK/ERK pathway, constitutively active ERK was overexpressed in AR+ TNBC models which induced radioresistance, confirming this relationship. As expected, AR ChIP-seq data showed enza treatment induced a significant reduction in AR peaks compared to DMSO-treated controls. Radiation induced fewer changes in AR binding than enza treatment. Bru-seq GSEA showed significant upregulation of MYC target gene sets in response to RT in control conditions. Conversely, with enza treatment, there was no significant MYC target gene set upregulation. While there were no changes in MYC expression or AR binding to known MYC enhancers, there was increased binding of AR to known MYC binding sites in control samples in response to RT. This RT-induced binding to MYC sites was not seen in the enza-treated samples suggesting that AR inhibition may dampen the RT induction of MYC signaling leading to radiosensitization.



 These findings provide new insights into the molecular underpinnings of AR-mediated radioresistance and identify MAPK and MYC pathways as potential therapeutic targets to enhance radiosensitivity in AR+ TNBC. Ongoing studies aim to translate these findings into clinically actionable strategies. This work expands our understanding of AR-mediated radioresistance which may uncover resistance mechanisms to combination AR inhibition and RT.



 Benjamin Hauk, Breanna McBean, Anna Michmerhuizen, Douglas Gurdak, Savannah Tidmore, Michelle Paulsen, Lynn Lerner, Connor Ward, Kassidy Jungles, Daniel Spratt, Lori Pierce, Mats Ljungman, Corey Speers. Multiomic insights into mechanisms of androgen receptor-driven radioresistance in triple-negative breast cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4681.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2116c40f11199361c9cb93d0a62e0a019f2b44da" target='_blank'>
              Abstract 4681: Multiomic insights into mechanisms of androgen receptor-driven radioresistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Benjamin Hauk, B. McBean, A. Michmerhuizen, Douglas Gurdak, Savannah Tidmore, Michelle Paulsen, L. Lerner, C. Ward, Kassidy M. Jungles, Daniel Spratt, Lori J. Pierce, Mats Ljungman, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Chromatin architecture is frequently altered in cancers, including one of the most prevalent cancers, hepatocellular carcinoma (HCC). Emerging evidence suggests that liquid-liquid phase separation plays a crucial role in shaping chromatin architecture and regulating gene transcription. Immune checkpoint blockade (ICB) therapy has become a new standard of care for HCC. However, the majority of HCC patients exhibit either primary or adaptive resistance to this immunotherapy, highlighting the urgent need for effective combination therapies. This study aims to uncover the differences in chromatin architecture between ICB-sensitive and resistant HCC tumors and dissect the underlying regulatory mechanisms.



 Hi-C, ATAC-seq, CUT&Tag, and RNA-seq were performed on RIL175 cells from a well-established adaptive ICB-resistant mouse model to generate the multi-omics dataset. To explore the functional role of F1 in ICB resistance of HCC, RIL175-PD1-R cells with F1 knockdown were constructed using the lenti-virus infection. To explore the phase separation ability of F1, F1 protein tagged with mCherry was expressed in E. coli and purified. Droplet formation assay and fluorescence recovery after photobleaching (FRAP) assay were applied.



 Three-dimensional genomic analysis by Hi-C reveals comparable chromatin interactions between ICB-sensitive and resistant cells at both compartment and topologically associated domain (TAD) levels but an increase in loops within ICB-resistant cells. In accord with this, ATAC-seq results indicate upregulated chromatin accessibility in a subset of chromatin regions in ICB-resistant cells compared to ICB-sensitive cells. Motif analysis identifies F1 as the top transcription factor enriched for the open chromatin regions. CUT&Tag shows that F1 binds to those accessible regions and alters chromatin looping in ICB-resistant cells. Modulating F1 expression in HCC tumor cells significantly increases the therapeutic efficacy of ICB treatment. We also found that F1 undergoes phase separation, which may mediate the changes in chromatin architecture between ICB-sensitive and resistant cells.



 F1 undergoes phase separation in HCC, particularly in ICB resistant cells. This characteristic of F1 may shape specific three-dimensional chromatin architecture in HCC cells, contributing to their resistance to ICB treatment. Targeting F1 condensates may provide a novel therapeutic strategy to reprogram the genome architecture to improve ICB efficacy for patients with HCC.



 chromatin architecture, phase separation, hepatocellular carcinoma, immune checkpoint blockade (ICB) therapies



 This work is supported by NSFC/RGC (460795048) & Li Ka Shing Foundation.



 Siyun CHEN, Yingnan LIN, Chengpeng Zhong, Yalin TU, Haoran WU, Yiling ZHANG, Jia WANG, Alfred Sze Lok CHENG. Role of transcriptional regulation in chromatin architecture and immunotherapy resistance of hepatocellular carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2751.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa313239bf094607775440d72807390cbdf49546" target='_blank'>
              Abstract 2751: Role of transcriptional regulation in chromatin architecture and immunotherapy resistance of hepatocellular carcinoma
              </a>
            </td>
          <td>
            S‐I. Chen, Yingnan Lin, Chengpeng Zhong, Yalin Tu, Haoran Wu, Yiling Zhang, Jia Wang, Alfred Sze Lok CHENG
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce54fab673636a1009faf91581b1433f439bd51f" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, P. Bachireddy
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The adaptive response of cancer cells to hypoxia, a key microenvironmental factor in solid tumors, is intricately orchestrated by Hypoxia-inducible factor 1 (HIF1). Evidence is accumulating on oxygen dynamics in tumor cores, showing that hypoxia is frequently unstable, or cycling, associated with specific phenotypic outcomes for the cancer. Transcriptomic analysis show that most gene expression changes in cycling hypoxia lie in between the change caused by stable hypoxia, suggesting multi-cycle averaging of dosage in the oxygen tension, and likely HIF-1 induced transcription. However, small subset of genes show an oscillation/cycling hypoxia specific response, suggesting that the transcriptional machinery of these genes may interpret cycling HIF-1 activity differently from stably high HIF-1 activity. Here, we model a gene regulatory circuit, the incoherent feed-forward loops (IFFLs) to explore parameter regimes where oscillatory specific transcription is plausible. In these IFFL models, HIF-1 regulates gene transcription of a target gene directly, as well indirectly via another transcription factor with an opposite effect on gene transcription. We have identified several plausible parameter regimes where oscillation specific gene expression can occur, with potentially significant effect on cancer growth and progression. Finally, we experimentally confirmed that HIF-1 can form IFFLs with two key transcription factors p53, and Notch1, resulting in cycling hypoxia specific gene expression linked to breast cancer progression and poor prognosis. Our models mechanistically reveal how temporal fluctuations in the tumor microenvironment may directly inform downstream transcription, identify hitherto unknown HIF-1 driven mechanism of cancer progression contributing to emergent tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b65afa29d37135628627c1ec1ee0a105f3db46eb" target='_blank'>
              Cancer Cell Adaptation to Cycling Hypoxia: Integrating Mathematical Modeling with Experimental Validation of HIF-1 Dynamics
              </a>
            </td>
          <td>
            Xihua Qiu, Yamin Liu, Paola Vera-Nicola, E. Agmon, Kshitiz, Yasir Suhail
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 KRAS-G12C inhibitors are promising in targeted therapeutic strategies. However, it remains uncertain to search for biological reminders of clinical outcomes and early resistance.



 Systematic review has been performed up to September, 2024 to evaluate treatment efficacy and beneficial patients. We constructed KEAP1 and STK11 knockout mutants with CRISPR/Cas9 technology and PD-L1 overexpression mutants with lentiviral vectors. The sensitivity of cells to different KRAS-G12C inhibitors was assessed using CCK-8 assays. Potential pathways of resistance were identified through comprehensive bioinformatics analyses.



 13 clinical trials with 1, 132 enrolled patients indicated remarkable clinical efficacy and benefit the senior and the female. Patients with liver and brain metastases also tends to benefit from KRAS-G12C inhibitors. KEAP1 mutation serves as a critical negative biomarker for early progression and resistance. Validation studies indicated a notable increase in IC50 values for different KRAS-G12C inhibitors after KEAP1 knockout (P < 0.0001). Differentially expressed genes in KEAP1 knockout cells versus contrast groups were conducted with GO, KEGG, and Reactome pathway analyses, suggested evident enrichment in extracellular matrix organization and cell adhesion processes. In addition, STK11 mutations and an increasing PD-L1 expression level reminds of worse prognosis, despite absence of significant difference.



 This research considered KRAS-G12C inhibitors as optional therapy and selected some subgroups who are more likely gain benefits from targeted therapy. As a key adverse biomarker, KEAP1 mutation tends to affect extracellular matrix organization and cell adhesion and promotes early resistance.



 Yan Wang, Linyan Tian, Chengming Liu, Sufei Zheng, Huiyang Shi, Fang Wei, Wenxin Jiang, Yucheng Dong, Haiyan Xu, Enzhi Yin, Nan Sun, Jie He. Crucial prognostic biomarkers for resistance to KRAS-G12C inhibitors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4722.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/801d1a28b5036fffab863a5f513c202209cf4c12" target='_blank'>
              Abstract 4722: Crucial prognostic biomarkers for resistance to KRAS-G12C inhibitors
              </a>
            </td>
          <td>
            Yan Wang, Linyan Tian, Chengming Liu, Sufei Zheng, Huiyang Shi, Fang Wei, Wenxin Jiang, Yucheng Dong, Haiyan Xu, Enzhi Yin, Nan Sun, Jie He
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 The emerging field of comparative oncology may provide a novel solution to rare human cancer studies, as genomic resources, reference genomes, and non-human model cancer sequencing data are proliferating rapidly. Studying cancer across a diverse array of species provides a unique opportunity to interrogate factors underpinning cancer, facilitating disease modeling in the setting of spontaneous tumors complicated by comorbidities and metastases. Unfortunately, spontaneously developed tumors in non-human mammalian models have been largely overlooked, leaving a foundational resource underutilized. Several key barriers prevent the adoption of these critical data. Extensive reporting of the frequency and diversity of animal tumors has not been systematically linked to the plethora of genomic resources available. Additionally, complications in integrating data from hundreds of genomes have prevented large-scale analysis. Fortunately, recent advances in comparative genomics allowing for joint alignments of hundreds of species have bridged this gap, enabling us to integrate the rapidly expanding genetic resources for non-traditional model species and presenting an enormous potential resource for oncology.
 To facilitate the transition of comparative oncology studies from human plus one or two other species to pan-mammalian analyses, we developed the Paipu computational pipeline to systematically identify and process data from hundreds of mammalian species, creating a pan-mammalian tumor compendium. Paipu consists of three phases: reference genome preparation, SRA metadata retrieval and harmonization, and FREYA for RNA-seq data processing and harmonization. Paipu identified and harmonized data from over 100, 000 non-human mammalian cancer RNA-seq samples from NCBI, spanning 52 of the 241 mammalian species in the Zoonomia multi-species genome alignment. We demonstrate Paipu’s potential via a comparative analysis of human cancers and a rapidly emerging model system, the dog. Using the Paipu cancer data compendium, we analyzed ten cancers in human and dog. We identified a subset of the cancers as strong models of human cancer, which we validated through literature analysis. Additionally, our comparative cancer analysis results parallel the original TCGA pan-cancer analysis findings.
 Paipu’s harmonization of data from numerous mammalian species facilitates downstream comparative genomics analysis. Increasing the diversity of genomes used in comparative studies can reveal shared and lineage-specific variation across species, leading to the identification of novel mechanisms of cancer risks and resistance. Our approach provides a unique opportunity to harness significant advances in the rapidly developing fields of machine learning and genomics for human cancer research by identifying reciprocal links between evolution and cancer resistance.


 Bria S. Smith, Geesa V. Daluwatumulle, Leslie A. Smith, James A. Cahill, Kiley Graim. The Paipu pipeline for pan-mammalian cancer genomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5042.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f17818e89e418eef53189bf0e536334d7fee6f" target='_blank'>
              Abstract 5042: The Paipu pipeline for pan-mammalian cancer genomics
              </a>
            </td>
          <td>
            Bria S. Smith, Geesa Daluwatumulle, Leslie A. Smith, James A. Cahill, Kiley Graim
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 In cancer research, perturbing gene function is crucial for understanding disease mechanisms and developing treatments. Techniques such as RNA interference and CRISPR-knockout, -interference, or -activation are commonly used. In this recent study we compared the phenotypic effects of cellular morphology by knocking out or knocking down key cell cycle regulators, Aurora kinase B (AURKB), Geminin (GMNN), and Polo-like kinase 1 (PLK1), which are known key critical proteins involved in cell cycle regulation, and their dysfunction linked to cancer progression. In this in vitro osteosarcoma cell line (U 2-OS) model, we inhibited these proteins using CRISPR-Cas9 for gene knockout or siRNA for mRNA knockdown in a comparative study not previous reported. Employing Cell Painting methodology (6 fluorescent probes, high content screening (HCS) confocal imaging, and multivariant analysis) we imaged and profiled the morphological phenotypes following gene perturbations. Principal component analysis (PCA) analysis of hundreds of HCS image phenotypic feature measurements revealed significant morphological changes and reduced cell numbers from AURKB, GMNN, and PLK1 inhibition in U 2-OS cells caused significant phenotypic changes and reduced cell numbers suggesting alteration of cell cycle progression. Interestingly, in the GMNN inhibition, the siRNA-treated cells showed a higher percentage of large nuclei compared to small nuclei from CRISPR-treated cells. These findings suggest siRNA-mediated knockdown has a faster inhibitory effect on GMNN than CRISPR-treated conditions. This study demonstrates the complementary nature of CRISPR-Cas9 and siRNA gene modulation and underscores the importance of orthogonal validation for robust genetic studies with key cell cycle regulators.


 Alexander Schreiner, Angelika Foitzik, Josien Levenga, Karin Boettcher, Barbara Sonnenberg, Žaklina Strezoska, Joe Trask. Unlocking cell cycle dynamics: Gene regulation of CRISPR-Cas9 and siRNA phenotypes using cell painting [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB443.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ca3a1f08d426b3a13684391aa11a6fb1c8556a" target='_blank'>
              Abstract LB443: Unlocking cell cycle dynamics: Gene regulation of CRISPR-Cas9 and siRNA phenotypes using cell painting
              </a>
            </td>
          <td>
            Alexander Schreiner, A. Foitzik, J. Levenga, Karin Boettcher, Barbara Sonnenberg, Z. Strezoska, Joe Trask
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) is a heterogeneous disease showing significant variability in clinical aggressiveness and treatment response. Primary and acquired resistance limit the efficacy of available treatments, and identification of new therapeutic targets is needed to further improve patients’ outcomes. To identify actionable targets, we performed a “drop-out” loss-of-function screening with an shRNA library covering 200 drug-target genes in nine different CRC cell lines, covering the whole spectrum of molecular subtypes of CRC. Multiple screening hits were found. Among them RRM1, encoding a member of the ribonucleotide reductase complex and targeted by the FDA-approved drug clofarabine, was found to be non-dispensable in all cell lines. In vitro drug efficacy studies confirmed sensitivity to clofarabine of most CRC cell lines. In vivo testing on a CRC cell line xenograft confirmed substantial growth inhibition by clofarabine. These results unveil a possible therapeutic opportunity in colorectal cancer.


 Federica Invrea, Consalvo Petti, Claudio Isella, Enzo Medico. Ribonucleotide reductase is a potential therapeutic target in colorectal cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6892.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a9e6faf45fc758543cd3dfb6d7c7bf6cfe5d5f" target='_blank'>
              Abstract 6892: Ribonucleotide reductase is a potential therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Federica Invrea, Consalvo Petti, C. Isella, E. Médico
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4f9af4dcbefbfc2af9850cbacc2d2628f2e0e1e" target='_blank'>
              TEITbase: a database for transposable element (TE)-initiated transcripts in human cancers
              </a>
            </td>
          <td>
            Yun Zhang, Jianqi She, Xueyan Hu, Yueqi Jin, Changyu Tao, Minghao Du, Ence Yang
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Large scale structural variation in the genome is known to be linked to cancer pathways. Breast cancer progression from normal epithelium to localized, and metastatic cancer involves genomic perturbations such as mutations, deletions, and duplications. Emerging research has shown that the structure of the 3D genome, specifically how chromosomes and genes are folded and organized in the nucleus, directly impacts gene expression and thus, may contribute to disease progression. In fact, local folding around critical genes, especially those involved in important biological pathways, may have direct impact on cancer initiation and progression. Limits in technology has restricted a more complete understanding linking 3D genome to disease. Biochemical methods require many cells and are unable to preserve single cell and sub-population level information. Previous imaging methods are limited by the number of analyzable targets, reducing the generalizability of results across other genomic loci and limited a comprehensive genome-wide understanding.
 Here, we use jebFISHTM and the PaintScapeTM platform to resolve 3D structural features of genomic loci at high fidelity. With our Biological Pathways Panel, we focus on genes that are known to play critical roles in DNA repair, cell cycle, apoptosis, chromatin modifications, and transcriptional regulation. Using breast cancer progression as the model, we apply jebFISHTM to cell lines from normal epithelium, localized cancer, and metastatic cancer. We observe dramatic changes in 3D genome at both single cell and population levels. At each stage of breast cancer progression, we identify specific alterations to 3D genome features including changes in structure and switching of A/B chromatin compartments. Structural alterations are also present at whole chromosome level. We see disruption and repositioning of chromosome territories as well as interchromosomal interactions. jebFISHTM single-cell, single-homolog resolution enables identification of cellular sub-populations at each stage of breast cancer progression. We observe changes in the frequency of sub-populations with disease progression. Our results suggest that genome organization, at the local gene as well as chromosome level is modulating the expression of genes important in biological pathways. The presence of sub-populations suggests that the heterogeneous nature of 3D genome organization within the population is an important feature, impacting our understanding and treatment of diseases. While this study focuses on breast cancer, we envisage that these results are generalizable and further the understanding of genome organization and gene expression.


 Huy Nguyen, Shyamtanu Chattoraj, Jude Dunne. 3D genome structural alterations in biological pathway genes drive the progression of cells from normal epithelium to metastatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2746.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f032fd3c43cdde86d384472ad3a3c6263ba1644" target='_blank'>
              Abstract 2746: 3D genome structural alterations in biological pathway genes drive the progression of cells from normal epithelium to metastatic cancer
              </a>
            </td>
          <td>
            Huy Nguyen, Shyamtanu Chattoraj, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Cancer heterogeneity poses a significant medical challenge, necessitating therapies tailored not only to individual patients but also to the unique cancer clones that exist within diverse microenvironmental contexts. Despite this diversity, nearly all cancer cells share a reliance on telomeres to maintain genomic stability. The telomerase (hTERT) gene is hyperactive in many cancers and has been identified as a potential target for treatment. While this has led to the development of multiple telomerase-targeting approaches, disappointingly, none have been approved in clinical applications yet. To circumvent this concern, our team has applied a genetic approach called synthetic dosage lethality (SDL), to exploit hTERT overexpression to identify potential targets to treat cancer. SDL is a genetic concept, where a normally non-lethal gene inactivation kills cells only in the context of overexpression of another gene like hTERT. Our laboratories have used lentiviral-based pooled CRISPR/Cas9 and pooled shRNA-screening platforms to systematically query the genome and recently identified several SDL partners of hTERT. Through extensive validation using pooled in vivo CRISPR/Cas9 screens across multiple cell lines, tumor xenografts, and patient-derived organoids, we identified RMT1, an RNA 2'-O-methyltransferase, as a prominent SDL partner of hTERT. In collaboration with the AtomWise and Thoth Biosimulations companies, we have developed novel inhibitors for RMT1 that preferentially suppress hTERT-overexpressing cell lines growth and viability. We have also performed a large-scale screen of FDA-approved small molecule inhibitors with the goal of repurposing these drugs to target hTERT-overexpressing cancers. Subsequently, we plan to apply these therapies in combination to amplify the efficiency of treatment against cancers, yielding preclinical evidence to support the development of novel cancer therapies. Since hTERT activity is essential in all cancer cells, our findings introduce a transformative approach to overcoming cancer cell heterogeneity by targeting the SDL interactions of hTERT. This strategy also capitalizes on the ubiquitous overexpression of hTERT in all cancer types, enabling the selective eradication of hTERT-dependent malignancies.


 Vincent Maranda, Frederick S. Vizeacoumar, Yue Zhang, Liliia Kyrylenko, Andrew Freywald, Franco J. Vizeacoumar. Developing combination therapies for telomerase-overexpressing cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5412.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, F. Vizeacoumar, Yue Zhang, L. Kyrylenko, A. Freywald, Franco J Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Genotoxic stress-induced stem cell maldifferentiation (GSMD) integrates DNA damage responses with loss of stemness and lineage-specific differentiation to prevent damaged stem cell propagation. However, molecular mechanisms governing GSMD remain unclear. Here, we identify the p53-induced long non-coding RNA LOC644656 as a key regulator of GSMD in human embryonic stem cells. LOC644656 accumulates in the nucleus upon DNA damage, disrupting pluripotency by interacting directly with POU5F1 and KDM1A/LSD1-NuRD complexes, repressing stemness genes, and activating TGF-β signaling. Additionally, LOC644656 mitigates DNA damage by binding DNA-PKcs and modulating the DNA damage response. In cancer, elevated LOC644656 correlates with poor patient survival and enhanced chemoresistance. Our findings demonstrate that LOC644656 mediates stemness suppression and resistance to genotoxic stress by coordinating DNA damage signaling and differentiation pathways. Thus, LOC644656 represents a potential therapeutic target for overcoming chemoresistance and advancing stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a2bd396273c74fb23a4471cceb78f9c794e8d" target='_blank'>
              p53-inducible lncRNA LOC644656 causes genotoxic stress-induced stem cell maldifferentiation and cancer chemoresistance
              </a>
            </td>
          <td>
            Ai Tamura, Kazuyuki Yamagata, Takashi Kono, Masanori Fujimoto, Takahiro Fuchigami, Motoi Nishimura, Masataka Yokoyama, Akitoshi Nakayama, N. Hashimoto, I. Sakuma, Nobuyuki Mitsukawa, Yusuke Kawashima, O. Ohara, S. Motohashi, Eiryo Kawakami, Takashi Miki, Atsushi Onodera, Tomoaki Tanaka
          </td>
          <td>2025-05-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are short non-coding RNAs, which perform a key role in cellular differentiation and development. Most human diseases, particularly cancer, are linked to miRNA functional dysregulation implicated in the expression of tumor-suppressive or oncogenic targets. Cancer hallmarks such as continued proliferative signaling, dodging growth suppressors, invasion and metastasis, triggering angiogenesis, and avoiding cell death have all been demonstrated to be affected by dysregulated miRNAs. Thus, for the treatment of different cancer types, the detection and quantification of this type of RNA is significant. The classical and current methods of RNA detection, including northern blotting, reverse transcription-quantitative PCR, rolling circle amplification and next-generation sequencing, may be effective but differ in efficiency and accuracy. Furthermore, these approaches are expensive, and require special instrumentation and expertise. Thus, researchers are constantly looking for more innovative approaches for miRNA detection, which can be advantageous in all aspects. In this regard, an RNA manipulation tool known as the CRISPR and CRISPR-associated sequence 13 (CRISPR/Cas13) system has been found to be more advantageous in miRNA detection. The Cas13-based miRNA detection approach is cost effective and requires no special instrumentation or expertise. However, more research and validation are required to confirm the growing body of CRISPR/Cas13-based research that has identified miRNAs as possible cancer biomarkers for diagnosis and prognosis, and as targets for treatment. In the present review, current updates regarding miRNA biogenesis, structural and functional aspects, and miRNA dysregulation during cancer are described. In addition, novel approaches using the CRISPR/Cas13 system as a next-generation tool for miRNA detection are discussed. Furthermore, challenges and prospects of CRISPR/Cas13-based miRNA detection approaches are described.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e7cad934b860ca66c2574dc37104aa0e67005c2" target='_blank'>
              Current updates regarding biogenesis, functions and dysregulation of microRNAs in cancer: Innovative approaches for detection using CRISPR/Cas13-based platforms (Review)
              </a>
            </td>
          <td>
            Abdulaziz A. Aloliqi, Abdullah M. Alnuqaydan, Aqel Albutti, Basmah F. Alharbi, A. Rahmani, Amjad Ali Khan
          </td>
          <td>2025-04-11</td>
          <td>International Journal of Molecular Medicine</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="


 Small cell lung cancer (SCLC) is the most lethal form of lung cancer. This poor prognosis is due in large part to SCLC’s recalcitrance to therapy. The standard of care for extensive-stage SCLC (ES-SCLC) is chemotherapy with immunotherapy. While most tumors initially respond, responses are transient and >80% of tumors progress within one year of treatment. Seminal studies have shown that sequencing aggressive tumors before and after therapy identifies drivers of resistance. Unfortunately, the aggressive clinical course of ES-SCLC has precluded collection of patient-matched naïve and recurrent tumor tissue traditionally required to study evolution. In lieu of this, other groups have used circulating tumor DNA (ctDNA) to study SCLC evolution. However, these studies used sequencing panels that analyze <0.004% of the SCLC genome. Using new methods to comprehensively trace how SCLCs evolve to therapy resistance is a significant unmet need.



 Medical records were reviewed to identify treatment naïve ES-SCLC patients with longitudinal liquid biopsies collected pre- and post-therapy using IRB-approved protocols. ctDNA was isolated from plasma using NucleoSnap cfDNA kits. Germline DNA was extracted from buffy coats using NucleoSpin Blood kits. Whole genome sequencing (WGS) libraries were prepared using Twist cfDNA Library Preparation with UMI Adapter kits. Deep WGS of ctDNA (>100X) and germline DNA (>40X) was performed using an Illumina NovaSeqX sequencer. Reads were aligned to the hg38 reference genome with BWA-mem. Somatic mutations were called using a two-of-three consensus approach and Mutect2, Strelka2, and MuSE. Structural variants were called using a consensus approach with GRIDSS, SvABA, and Manta. Subclonal copy number alterations were identified with Battenberg. Pyclone was used to reconstruct clonal populations.



 As expected, we identified deleterious genomic alterations impacting TP53 and RB1 tumor suppressor genes, known drivers of SCLC development. COSMIC mutational signature analysis identified high activity of tobacco carcinogen, APOBEC, and replicative-based processes across tumors. We found highly varied patterns of genomic evolution following frontline chemoimmunotherapy across patients. Most ES-SCLC tumors exhibited genomic evolution at recurrence, with evidence of focal amplifications and deletions or whole genome duplication. Some tumors exhibited minimal copy number evolution at relapse, suggesting non-genomic mechanisms driving resistance. We also found that second line therapies forced additional genomic evolution following initial recurrence.



 Our data demonstrates that deep WGS of ctDNA recapitulates known genetics of ES-SCLC and enables sensitive characterization of resistant clones. Our data highlights the under-appreciated role of genomic evolution in ES-SCLC frontline therapy resistance.



 Benjamin B. Morris, Zhihui Zhang, Andrew R. Lynch, Simon Heeke, Kyle Concannon, C. Allison Stewart, Runsheng Wang, Alexa Halliday, Waree Rinsurongkawong, Vadeerat Rinsurongkawong, J. Jack Lee, Jianjun Zhang, Don L. Gibbons, Ara A. Vaporciyan, Carl M. Gay, Hai T. Tran, John V. Heymach, Lauren A. Byers, Peter Van Loo. Reconstructing small cell lung cancer evolution using deep whole genome sequencing of circulating tumor DNA [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1263.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a605a688e58842e081be2262f0d61629173ea95c" target='_blank'>
              Abstract 1263: Reconstructing small cell lung cancer evolution using deep whole genome sequencing of circulating tumor DNA
              </a>
            </td>
          <td>
            Benjamin B. Morris, Zhihui Zhang, Andrew R. Lynch, Simon Heeke, Kyle Concannon, C. Stewart, Runsheng Wang, Alexa J. Halliday, W. Rinsurongkawong, Vadeerat Rinsurongkawong, J. J. Lee, Jianjun Zhang, Don L Gibbons, A. Vaporciyan, C. Gay, Hai T. Tran, J. Heymach, L. Byers, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) can be divided into 2 genetically different subtypes: Microsatellite instable CRC (MSI-CRC; ∼15% of CRC), and microsatellite stable (MSS-CRC; ∼85%). While MSI-CRC develops point mutations that increase neoantigen burden and enhanced response to immunotherapy, MSS-CRC develops chromosomal instability (CIN) and resistance to immunotherapy. Current mouse models do not recapitulate the key genetic defect of MSS-CRC: CIN. This limits our understanding of CIN‘s role in regulating tumor immunity in CRC, and therefore our capacity to generate new immunotherapies for MSS-CRC.
 To address this gap, we developed the first genetically engineered mouse model of MSS-CRC with human-like CIN. To recapitulate human CIN caused by telomere erosion, we incorporated the conditional null Apc and Trp53, and inducible Lox-Stop-Lox (LSL) mouse telomerase reverse transcriptase (mTert) alleles, referred to as iTAP. Due to LSL-mTert, mice lack mTert expression and therefore telomerase activity, which leads to telomere erosion upon successive generational intercrosses, evolving from G0 mice (LSL-mTert+/-) that have no CIN (CIN- iTAP) to G3 mice after 3 breeding generations with the mTert+/+ allele that develop CIN (CIN+ iTAP). Colonoscopy-guided submucosal administration of 4-OHT activates Villin-driven Cre-ER in colonocytes leading to Apc and Trp53 deletion and deletion of the stop cassette of LSL-mTert causing activation of telomerase in cancer cells. Oncogenesis in cells with telomere erosion leads to CIN.
 Consistent with worse OS in human MSS CINhigh, CIN+ iTAP tumors were more aggressive than CIN- iTAP. Single-cell RNA sequencing (scRNA-seq) confirmed that CIN+ iTAP tumors developed more copy number variations. Subclustering of the infiltrating immune populations revealed that γδ T cells expanded and changed their phenotype in CIN+ iTAP tumors, which was confirmed by flow cytometry. Treatment with a blocking antibody towards γδ TCR in mice with established tumors lead to increased survival in CIN+ iTAP mice but not in CIN- iTAP mice. ScRNA-seq of primary tumors of patients with MSS-CRC revealed that γδ T cells also exhibit phenotypic changes upon different levels of CIN in human MSS CRC.


 Paulino Tallón de Lara, Aviad Ben-Shmuel, Wen-Hao Hsu, Chaohao Li, Kyle Labella, Chi Wut Wong, Yonghong Liu, Shan Jiang, Xiaoying Shang, Scott Kopetz, Ronald DePinho. Chromosomal instability promotes colorectal cancer progression through manipulation of gamma delta T cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5246.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c350dc98b4d3786f2ffe26744dbe7c5702a670" target='_blank'>
              Abstract 5246: Chromosomal instability promotes colorectal cancer progression through manipulation of gamma delta T cells
              </a>
            </td>
          <td>
            Paulino Tallón de Lara, Aviad Ben-Shmuel, Wen-Hao Hsu, Chaohao Li, Kyle Labella, Chi Wut Wong, Yonghong Liu, Shan Jiang, Xiaoying Shang, S. Kopetz, Ronald A. DePinho
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Colorectal cancer (CRC), the third most common cancer worldwide, often shows limited responsiveness to immunotherapy due to its predominantly immune-excluded phenotype. Despite increasing insights into the complex tumor microenvironment (TME), the metabolic heterogeneity of CRC cells and their interactions with tumor-infiltrating immune cells remain poorly understood.We analyzed 46,374 epithelial cells from 17 CRC patients treated with PD-1 blockade to develop an amino acid (AA) metabolism score using the AUCell algorithm. This score was applied to a separate single-cell RNA sequencing (scRNA-seq) dataset from 23 CRC patients to investigate cell-cell interactions and functions of tumor-infiltrating immune cells, revealing distinct immune TME landscapes shaped by tumor metabolism. An in vitro co-culture assay of CRC cells and CD8+ T cells was performed to validate the findings. Additionally, LASSO and Cox regression analyses were conducted to construct an AA metabolism-related risk score for predicting prognosis and drug sensitivity across multiple bulk transcriptome cohorts.This study identified a link between elevated amino acid metabolism in CRC epithelial cells and resistance to PD-1 blockade therapy. A 31-gene AA score was developed by intersecting differentially expressed genes between responders and non-responders to PD-1 blockade with amino acid metabolism-related genes from the Molecular Signature Database (MSigDB). Using this score, 23 additional CRC samples were classified into high and low AA score groups. Comparative analysis revealed that the low AA group exhibited a more robust immune response, characterized by a greater number and stronger cell-cell interactions. Tumor-infiltrating immune cells in this group demonstrated enhanced activation and anti-tumor functions. Furthermore, CD8+ T cells showed increased Granzyme B levels when co-cultured with CRC cells in which Psat1 or Shmt2 was knocked down. Finally, a machine learning-derived risk score based on six genes was established to translate single-cell findings to bulk transcriptomes. This risk score was found to correlate with immune checkpoint expression and immune cell infiltration, with potential implications for predicting prognosis and drug sensitivity.Our findings highlight the role of elevated epithelial amino acid metabolism in shaping an immune-suppressive microenvironment, offering insights for patient stratification and therapeutic decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b9bda18d8cb73c7c5646b13e67e456908ec19a2" target='_blank'>
              Single-cell and bulk transcriptome analyses reveal elevated amino acid metabolism promoting tumor-directed immune evasion in colorectal cancer
              </a>
            </td>
          <td>
            Tianyue Sun, Yan Chen, Ying Chen
          </td>
          <td>2025-05-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Chromosomal instability (CIN), characterized by frequent chromosomal alterations, plays a crucial role in tumorigenesis, significantly influencing cancer progression and therapeutic resistance. Targeting vulnerabilities associated with CIN, particularly in DNA repair pathways, presents promising therapeutic strategies. However, existing CIN detection methods are often costly and labor-intensive. Thus, the development of efficient and cost-effective CIN detection panels is essential for optimizing cancer treatments and enhancing patient outcomes while reducing healthcare costs.



 We conducted a retrospective study analyzing 4, 941 tissue samples from patients diagnosed with 14 distinct cancer types, including ovarian, uterine, breast, pancreatic, prostate, cervical, liver, lung, colorectal, and gastric cancers. A targeted sequencing panel, GENESEEQ GENAVISTA, assessing 437 cancer-associated genes with an approximate depth of coverage of 1, 000X, facilitated detailed mutational profiling. CIN assessment utilized read depth and allele frequency evaluation, alongside variant calling algorithms to identify gene duplications and genomic rearrangements. Additionally, a prospective cohort of 33 cancer patients was analyzed to compare the efficacy of our sequencing panel against the gold standard of whole genome sequencing (WGS, ∼30X).



 Our analysis revealed significant CIN prevalence across various cancer types, particularly elevated in ovarian cancer (73.26%) and breast cancer (76.80%). Lung cancer exhibited a CIN rate of 78.57%, while melanoma demonstrated an 80.00% prevalence. Endometrial cancer presented a moderate CIN rate of 40.62%, whereas pancreatic cancer displayed an equal distribution between CIN and genomically stable tumors, each accounting for 50.00%. Prostate cancer showed a CIN incidence of 45.16%, and cervical cancer recorded a rate of 62.30%. The findings illustrate the heterogeneous nature of CIN among malignancies, suggesting its utility as a prognostic marker and therapeutic target. The panel demonstrated a high correlation with WGS results in the prospective cohort (Spearman rank correlation = 0.91), confirming its effectiveness in identifying CINs in cancer patients.



 Our study establishes the GENESEEQ GENAVISTA targeted sequencing panel as an effective tool for accurately detecting CIN across various cancer types. The strong correlation with whole genome sequencing highlights its potential as a reliable, cost-effective alternative for CIN assessment, facilitating optimized cancer treatment strategies and improving patient outcomes while minimizing healthcare costs.



 Hua Bao, Rui Liu, Haimeng Tang, Ningyou Li, Min Wu, Xunbiao Liu, Zhili Chang, Xue Wu. Efficient detection of chromosomal instability in diverse cancers using a targeted sequencing approach [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5055.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3d68234ed3b90b54b46e31e83216b5a77d6d67" target='_blank'>
              Abstract 5055: Efficient detection of chromosomal instability in diverse cancers using a targeted sequencing approach
              </a>
            </td>
          <td>
            H. Bao, Rui Liu, Haimeng Tang, Ningyou Li, Min Wu, Xunbiao Liu, Zhili Chang, Xue Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Chromatin dysregulation has been implicated in the development of a wide range of diseases, including cancer, as well as therapeutic resistance. Genetic alterations in chromatin regulatory factors and aberrant activity of transcriptional regulators alter chromatin states and drive tumor growth in cancer. One example of a cancer that is driven by alterations in chromatin landscape is Ewing sarcoma, a highly aggressive pediatric cancer of bones and soft tissues. Ewing sarcoma is driven by an oncoprotein generated by the translocation of the EWSR1 and FLI1 genes, resulting in the fusion oncoprotein (EWSR1::FLI1), which activates noncanonical enhancers and alters the expression of critical genes associated with the disease. EWSR1::FL1 is challenging, if not impossible, to target directly with small molecules, and viable therapeutic targets for Ewing sarcoma have been slow to emerge. To broaden the search for druggable targets we developed a target agnostic screening approach that exploits oncoprotein-specific chromatin accessibility states as a relevant and direct functional readout for identifying novel therapeutics. We developed and optimized a 384-well, high throughput (HT) version of the Assay for Transpose-Accessible Chromatin assay (HT-ATAC-seq) that was applied to 500 EWSR1::FLI1-regulated chromatin regions. Using Ewing sarcoma-derived A673 cells, we conducted a pilot screen of 12,000 small-molecule compounds comprising approved drugs, investigational agents, and bioactive compounds. The screen identified 285 compounds that partially reversed Ewing sarcoma chromatin abnormalities. The set of active compounds included Panobinostat, which is currently in preclinical development for Ewing sarcoma. This pilot screen supports the HT-ATAC-seq assay as a robust approach to identify novel compounds that reverse the chromatin alterations associated with Ewing sarcoma. This approach and assay can be adapted to investigate other cancers or pathologies associated with chromatin state alterations.


 Darian Williams, Andrew McFadden, Austin Hepperla, Min Shen, Tino Sanchez, Gordon Stott, Mark Henderson, Ian Davis, Samantha Pattenden. Identifying novel Ewing sarcoma therapeutics using a high-throughput chromatin accessibility assay [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5488.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d431bff55b178eb615284d91d43a85ede147802" target='_blank'>
              Abstract 5488: Identifying novel Ewing sarcoma therapeutics using a high-throughput chromatin accessibility assay
              </a>
            </td>
          <td>
            Darian Williams, Andrew McFadden, Austin J. Hepperla, Min Shen, Tino W Sanchez, Gordon M. Stott, M. Henderson, Ian Davis, Samantha Pattenden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90139c8e3ae8545f259aeac6842fc6fa1b255360" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Genomic instability is a hallmark of cancer where the acquisition of numerous genetic alterations leads to transformation and carcinogenesis. These alterations result in the generation of potentially immunogenic neoantigens, and the clinical success of checkpoint inhibitor therapies has highlighted the importance of neoantigens in cancer immunotherapy. Evaluation of the next generation of neoantigen therapies is complicated by the difficulty in sourcing primary patient material. During tumor resection and pathology review, the tissue is fixed, which prevents the ability to test in vitro model systems for therapy efficacy. While fresh tissue is amendable to in vitro culture, it is not possible to perform large scale genomic and proteomic screens in a timeframe for the culture to occur. By leveraging viably cryopreserved dissociated tissue, the tumor can be analyzed for genetic alterations and then, when novel neoantigens are determined, cultured at a later date. To test this workflow, we profiled dissociated tissues from cancer indications with a high degree of genomic instability for microsatellite instability (MSI). Consistent with previous studies, 17-55% of endometrial, colorectal, and gastric cancer samples were MSI-high. MSI-high and microsatellite stable (MSS) samples were further analyzed for peptide identification using LC-MS/MS (Orbitrap Astral) and over 10,000 protein groups and 80,000 peptides were identified. Samples were also amendable to T cell receptor (TCR) repertoire sequencing to uncover tumor-reactive T cell clones. These deep molecular analyses provided the foundation for the development of short-term spheroid and long-term tumoroid culture systems to understand immune responses using autologous tumor-infiltrating and peripheral lymphocytes. Collectively, this workflow provides a scalable system to identify neoantigens and evaluate the immune response to these antigens in a robust and reproducible pipeline.


 Shawn P. Fahl, Casey Smith, Kerri Colwell, Jessica Moore, Jaison Arivalagan, Danielle Gutierrez. Neoantigen evaluation in viably cryopreserved dissociated tissue via large scale proteomic profiling [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2843.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/242255fea6b3955bb0a9ccc76de9eb6a66d735ca" target='_blank'>
              Abstract 2843: Neoantigen evaluation in viably cryopreserved dissociated tissue via large scale proteomic profiling
              </a>
            </td>
          <td>
            Shawn P. Fahl, Casey Smith, Kerri Colwell, Jessica L. Moore, Jaison Arivalagan, Danielle B. Gutierrez
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Genes involved in centrosome function, microtubule dynamics, and mitotic regulation are critical for normal cell division. In cancer, dysregulation of these processes can lead to chromosomal instability and tumor progression. The genes CEP72, HAUS4, TUBGCP4, HAUS2, PLK1, and OFD1 play essential roles in cell cycle regulation, mitosis, and microtubule dynamics, particularly in spindle assembly and centrosome function. These genes are involved in interconnected cellular processes related to mitotic progression, and their altered expression may be linked to cancer progression and survival outcomes.



 This study analyzes the expression profiles for these six genes across thirteen cancer types, aiming to identify associations between dysregulation of these genes and cancer progression, as well as their impact on survival outcomes.



 We analyzed the expression of these six genes and the tumor suppressor genes APC and PTEN using RNA sequencing data from TCGA cancerous tissues and normal control tissues from GTEx for the thirteen cancer types. Differential gene expression was assessed using the DESeq2 R package, which models RNA sequencing data using a negative binomial distribution.



 Our analysis identified significant changes in gene expression across cancers, including bladder (BLCA), colon (COAD, READ), esophagus (ESCA), kidney (KICH, KIRC, KIRP), liver (LIHC), prostate (PRAD), stomach (STAD), and thyroid (THCA). Elevated CEP72 expression was associated with lower progression-free survival (PFS). We observed consistent downregulation of CEP72, HAUS2, and PLK1 across several cancers, with significantly adjusted p-values (padj < 1E-5). OFD1, HAUS4, and PTEN showed tissue-specific expression patterns, with upregulation in cancers such as prostate (padj = 1.1E-11) and thyroid (padj = 8.5E-5). We further identified cancer-specific expression profiles for these six genes involved in cell division. These findings suggest complex regulatory mechanisms influencing tumor progression and metastasis. Further analysis of these mitotic and centrosomal proteins revealed significant cancer-specific correlations between gene expression levels of CEP72, HAUS4, TUBGCP4, HAUS2, PLK1, OFD1 and patient outcomes, including PFS, disease-specific survival, and overall survival.



 This study underscores the interconnectedness of centrosomal and mitotic genes, suggesting that downregulation of key proteins like CEP72, HAUS2, and PLK1 disrupts essential cell migration, proliferation, and adhesion pathways. Conversely, upregulation of HAUS4 may enhance tumor suppressor functions and immune modulation. Analyzing these genes within their broader biological networks provides valuable insights into their roles in cancer progression, offering potential as cancer-specific biomarkers or therapeutic targets.



 Christopher L. Farrell, Amy Turner, Alex Feltus, Victoria Cipollino. Centrosomal and mitotic gene expression profiles across cancer types: Implications for tumor progression and survival outcomes. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3343.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb84576556f09b1b3e9082fc0a15bf2dd37e8f04" target='_blank'>
              Abstract 3343: Centrosomal and mitotic gene expression profiles across cancer types: Implications for tumor progression and survival outcomes
              </a>
            </td>
          <td>
            Christopher L. Farrell, Amy Turner, Alex Feltus, Victoria Cipollino
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 9520


 Background:
 HRDsig can aid in identifying tumors with DNA repair deficiencies, guiding PARP inhibitor (PARPi) use. It expands treatment options beyond breast cancer gene (BRCA) mutations enabling personalized therapy. Combining PARPi with targeted therapy or immunotherapy shows promise in overcoming resistance and improving outcomes for treatment-resistant ACM.
 Methods:
 Formalin-fixed paraffin-embedded (FFPE) primary tumors and metastatic biopsies from 9,576 ACM cases were analyzed using hybrid capture-based comprehensive genomic profiling (CGP), evaluating all classes of genomic alterations (GA). Central pathology review was conducted for all cases. Microsatellite instability-high (MSIH) status, tumor mutational burden (TMB), genomic ancestry, and mutational signatures were derived from sequencing data. HRDsig was assessed using copy number changes and genomic scars. PD-L1 expression was determined via immunohistochemistry [Dako 22C3; tumor proportion score (TPS)]. Statistical comparisons were made using Fisher’s exact test with Benjamini-Hochberg correction.
 Results:
 Among 9,576 ACM cases, 198 (2.1%) were HRDsig positive (HRDsig+). HRDsig+ cases when compared to HRDsig negative (HRDsig-) were older (median age: 69 vs. 67 years; p=0.034), more often female (49.5% vs. 36.2%; p=0.001), and had more GA per tumor (median: 7 vs. 6; p=0.026). HRDsig+ cases were more often of African (5.6% vs. 1.2%; p<0.0001) or American ancestry (8.1% vs. 4.1%; p=0.026) and less often European (84.3% vs. 94.0%; p<0.0001). GA more common in HRDsig+ included
 IGF1R
 (5.1% vs. 1.3%; p=0.003),
 KIT
 (11.6% vs. 5.6%; p=0.004),
 KRAS
 (7.1% vs. 2.7%; p=0.005),
 NF1
 (38.4% vs. 21.4%; p<0.0001),
 RAD21
 (6.7% vs. 3.0%; p=0.037), and
 TP53
 (40.4% vs. 24.3%; p<0.0001). HRDsig- cases exhibited higher TMB (median: 13.8 vs. 6.1; p<0.0001), more frequent TMB >10 mutations/Mb (60.9% vs. 39.9%; p<0.0001), and more UV light exposure trinucleotide signatures (57.3% vs. 36.4%; p<0.0001). GA more common in HRDsig- cases included
 BRAF
 (including V600E) (44.6% vs. 23.2%; p<0.0001),
 CDKN2A
 (49.2% vs. 36.9%; p=0.003),
 NRAS
 (28.0% vs. 11.6%; p<0.0001), and
 TERT
 (74.4% vs. 29.8%; p<0.0001). In both positive and negative groups, PD-L1 low-level expression (1-49% TPS) was comparable (43.3%-39.7%), while BRCA1/2 alterations (<2.0%) and MSIH status (0.0%-0.1%) remained rare.
 Conclusions:
 HRDsig+ status is rare in ACM but more frequent in non-white genomic ancestries. HRDsig+ ACM cases are less likely to have BRAF GA, more likely to have KIT GA and lower TMB levels. These findings may guide the future development of clinical trials employing combination therapies and PARPi in ACM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162f9c9205bbddc1331c24e0a0b790056a2c6e2c" target='_blank'>
              Homologous recombination deficiency signature (HRDsig) in advanced cutaneous melanoma (ACM): A genomic landscape study.
              </a>
            </td>
          <td>
            Nimisha Srivastava, D. Pavlick, E. Sokol, O. Gjoerup, J. Elvin, Ryon P. Graf, Gerald Li, J. Quintanilha, Sanchit Panda, J. Ross, Jade Homsi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 The Cancer Genome Atlas (TCGA) provides comprehensive genomic and transcriptomic data from over 10,000 cancer patients, along with metadata including viral infection status and survival information. We investigated the role of viral infections in dysregulating the expression of A3 enzymes (A3A/B/C/D/F/G/H). These enzymes primarily function to restrict viral infections by introducing C-to-U mutations in viral genomes. However, certain members (notably A3A and A3B) can also mutate cellular DNA, contributing to tumor initiation and progression. Cancers such as breast, bladder, cervical, head & neck, and skin exhibit characteristic C-to-T mutations attributed to the aberrant expression/activity of A3 enzymes. Accurate quantification of A3 dysregulation in tumor cells using TCGA bulk RNAseq data is complicated by the ubiquitous expression of these enzymes in infiltrating immune cells. This highlights a broader challenge in analyzing heterogeneous tumor biopsies to uncover virus-specific dysregulated processes in cancer. To address this challenge, we employed Nonnegative Matrix Factorization (NMF) to deconvolute A3 expression profiles from TCGA bulk RNA-seq data, i.e. to separate tumor cell profiles from those of the tumor microenvironment. We validated our NMF-extracted profiles by analyzing publicly available single-cell RNA-seq datasets from approximately 3,000 cancer projects in the Gene Expression Omnibus (GEO). Additionally, to identify cell types represented by these profiles, we compared them with experimentally obtained A3 expressions from purified cell types. Our analyses identified up to four distinct A3 profiles: two associated with T cells and NK cells, one linked to dendritic cells and macrophages, and one corresponding to cancerous epithelial cells. Notably, the A3 profiles derived from our analysis of single-cell RNA-seq data closely matched those deconvoluted from TCGA bulk RNA-seq data, demonstrating that our bulk RNA-seq deconvolution effectively captures single-cell population-level profiles. We found that both tumor-specific and immune cell-specific A3 profiles correlate with viral infection; however, significant variations were observed across different viral types and tumor types. None of these profiles were correlated with the extent of A3 mutational signatures SBS2 and SBS13.In conclusion, our study demonstrates that deconvolution of bulk RNA-seq datasets can reveal single-cell population-level insights into the cellular composition of tumor biopsies. Moreover, this approach offers the resolution needed to pinpoint virus-specific dysregulated processes within the complex and heterogeneous tumor microenvironment.


 Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi. Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 990.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Since its advent about ten years ago, the CRISPR-Cas9 system has been frequently used in biomedical applications. It has advanced various fields, and CRISPR-Cas9-based therapeutics have shown promising results in the treatment of specific hematological diseases. Furthermore, CRISPR gene editing technologies have revolutionized cancer research by enabling a broad range of genetic perturbations, including genetic knockouts and precise single nucleotide changes. This perspective focuses on the state-of-the-art methodology of CRISPR knock-ins to engineer immune cells. Since this technique relies on homology-directed repair (HDR) of double-strand breaks (DSBs) induced by the Cas9 enzyme, it can be used to introduce specific mutations into the target genome. Therefore, this methodology offers a valuable opportunity to functionally study specific mutations and to uncover their impacts not only on overall cell functions but also on the mechanisms behind cancer-related alterations in common signaling pathways. This article highlights CRISPR knock-in strategies, protocols, and applications in cancer and immune research, with a focus on diffuse large B cell lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15e1b343be2ebd40cfad2b5fa562b6f9cb4962e" target='_blank'>
              Strategies for CRISPR-based knock-ins in primary human B cells and lymphoma cell lines
              </a>
            </td>
          <td>
            Sophie Lund, Chun Gong, Xin Yu, Louis M. Staudt, Daniel J Hodson, Sebastian Scheich
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Novel therapeutic protocols are desperately needed to treat non-small cell lung cancer (NSCLC), the deadliest cancer world-wide. Recently, utilizing a patient’s tumor-infiltrating lymphocytes (TIL) is proving to be a viable and highly personalized approach. Despite its promise, many obstacles remain, such as refining expansion protocols for better in vivo TIL proliferative potential, improving T cell ability for homing-to and targeting tumor cells upon re-infusion, and avoiding IL-2/lymphodepletion related side effects. We hypothesize that inhibiting epigenetic enzyme EZH2 (EZH2i) will improve both TIL expansion and infusion outcomes in NSCLC patients. Yet, there are likely mechanisms of T cell suppression that EZH2i may not overcome. To address and identify these mechanisms, stochastic mathematical modeling of gene signaling in cancer has risen as a useful tool. Thus, we further hypothesize that modern sequencing data and mathematical biology can discover alternative mechanisms of T cell suppression that will suggest novel targets for TIL-combination treatment.
 To test these hypotheses, we have tested the EZH2 inhibitor valemetostat with anti-PD1 in a murine model of non-small cell lung cancer. We observed tumor regression and strong IFN-gamma T cell responses in combination treated animals. We also observed up-regulation of both major histocompatibility complexes, greater pro-T cell cytokine signaling, and myeloid populations became more tumor-eliminating. To translate these findings to TILs, we are currently securing NSCLC samples from our Biospecimen Core, establishing patient-derived tumoroids, and expanding TILs ex vivo. TILs will be expanded with/without EZH2 inhibitor valemetostat (Val) under “young” protocols. At ex vivo endpoint TILs will be assessed for Val-driven differences in viability, differentiation state, and reactivity via flow cytometry. To complement or studies with an in silico approach, we are utilizing libraries of publicly available NSCLC data to perform bioinformatic analysis and implemented equation learning techniques to infer NSCLC tumor signaling networks from clinical data between immunotherapy responders and non-responders. Differentiation patterns will be assessed as a function of pseudo-time-course data to inform predictions of signal transduction pathways. Lastly, mathematical phenotype control theory will be used to uncover potential targets within the tumor to boost TIL response in NSCLC.
 Given that advanced stage lung cancers remains refractory to most treatments, determining methods to improve precision medicine options such as TIL therapy is an important goal for the field. Our blending of classic bench science with bio-informatics has the potential to deepen our understanding of TIL therapy and to identify new way to improve therapeutic responses. Funding: T32 CA165990 (DRP), R01 CA237643 (CFB), R01 HL170193 (CFB), P30 CA177558 (Markey Shared Resources).


 Daniel R. Plaugher, Avery R. Childress, Christian M. Gosser, Dave-Preston Esoe, Xuilong Song, Jinpeng Liu, Christine F. Brainson. Enhancing TIL efficacy in NSCLC through epigenetic reprogramming and computational modeling [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7258.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4407199c782b232b5e9ca054b0b3617a834a373" target='_blank'>
              Abstract 7258: Enhancing TIL efficacy in NSCLC through epigenetic reprogramming and computational modeling
              </a>
            </td>
          <td>
            Daniel Plaugher, Avery R Childress, Christian M. Gosser, Dave-Preston Esoe, Xuilong Song, Jinpeng Liu, C. F. Brainson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb25978847c5553a988089ffa28339e8dff51f1" target='_blank'>
              Single-Cell profiling reveals targetable malignant T-cell subtypes and immune evasion pathways in Sezary Syndrome
              </a>
            </td>
          <td>
            M.S.F Beth A. Childs, Ph.D Eslam A. Elghonaimy, Praveen, Ramakrishnan Geethakumari, Kiran A. Kumar, Joseph F. Merola, M. D. Heather W. Goff, M. Todd A Aguilera, Ph.D Todd
          </td>
          <td>2025-05-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Platinum-based chemotherapeutic drugs induce intra- and interstrand purine crosslinks causing DNA damage and tumor cell death. Although these drugs are widely used as front-line treatment for multiple human cancers, particularly high-grade serous ovarian cancer (HGSOC), platinum resistance and tumor recurrence remains a common clinical problem. Here we report results from both a genome-wide CRISPRi screen and an RNA-binding protein-focused CRISPRi screen for modulators of sensitivity and resistance of HGSOC cells to two widely used platinum agents, cisplatin and oxaliplatin, and an microtubule-targeting taxane. Using an in vitro approach that mimics clinical treatment protocols, we found that cell lethality after cisplatin or oxaliplatin treatment in p53-defective ovarian cancer cells is markedly enhanced by suppressing genes in the Fanconi anemia (FA) and homologous recombination (HR) pathways, but considerably less enhanced by suppressing genes in the nucleotide excision repair (NER) pathway, despite the well-established importance of NER in reversing platinum-induced intrastrand crosslinks, particularly for cells in G1. Our findings suggest that cell death induced by platinum-DNA crosslinks results from replication stress, which can be further exacerbated by interference with error-free DNA repair mechanisms known to be particularly active during the S- and G2 phases of the cell cycle. These screens further implicated several known and novel RNA-binding proteins (RBPs) as potent sensitizers of HGSOC to both platinum agents and taxanes, including the RNA-modifying enzyme ADAR1, that catalyzes the deamination of adenosine-to-inosine in RNA, altering mRNA coding potential, modifying RNA structure, and regulating RNA localization. Additionally, ADAR1 has been implicated in controlling the integrated stress response (ISR), a signaling network that allows cells to adapt to changes in environment. These findings suggest that activation of an ADAR1-dependent ISR underlies the resistance of HGSOC cells to both platinum-based agents and anti-microtubule drugs that form the current clinical standard of care for HGSOC treatment.


 Yi Wen Kong, Erika D. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B. Yaffe. CRISPRi screening reveals RNA editing as a major determinant of platinum sensitivity in high-grade serous ovarian cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1514.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b4e4b812f53d84fe2b91abe7b8dd8fa8cfa234" target='_blank'>
              Abstract 1514: CRISPRi screening reveals RNA editing as a major determinant of platinum sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6086eff81e816ca3205d5bed8a0f1a0d7841fa7" target='_blank'>
              Oncogene aberrations drive medulloblastoma progression, not initiation.
              </a>
            </td>
          <td>
            K. Okonechnikov, P. Joshi, Verena Körber, Anne Rademacher, Michele Bortolomeazzi, Jan-Philipp Mallm, Jan Vaillant, P. B. da Silva, Britta Statz, Mari Sepp, Ioannis Sarropoulos, Tetsuya Yamada, A. Wittmann, Kathrin Schramm, Mirjam Blattner-Johnson, Petra Fiesel, Barbara Jones, N. Jäger, T. Milde, K. Pajtler, Cornelis M. van Tilburg, Olaf Witt, Konrad Bochennek, Katharina Johanna Weber, Lisa Nonnenmacher, Christian Reimann, David R. Ghasemi, U. Schüller, M. Mynarek, S. Rutkowski, David T. W. Jones, A. Korshunov, K. Rippe, Frank Westermann, S. Thongjuea, Thomas Höfer, Henrick Kaessmann, Lena M. Kutscher, Stefan M. Pfister
          </td>
          <td>2025-05-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Next generation cancer models, including organoids, neurospheres and cell lines, are significantly advancing our ability to study individual human tumors in vitro and to develop novel cancer therapeutics in settings that resemble in vivo conditions. However, recent questions have emerged as to the degree to which in vitro models may faithfully represent intra- and inter-patient cell state heterogeneity at the transcriptional level of single cells. The Human Cancer Models Initiative (HCMI), a global collaboration involving the NCI (NIH), Cancer Research UK, the Wellcome Sanger Institute and the Hubrecht Organoid Technology foundation, has generated a large repertoire of 665 patient-derived models of human cancers, including organoids, neurospheres and conditionally reprogrammed cells, with matching parental tumor and clinical annotations. We conducted a tumor-model fidelity analysis using bulk profiles from Whole Genome Sequencing (WGS) and RNA-seq and confirmed strong cell-state concordance between each model and its parental tumor across most HCMI tumor-model pairs, thus showcasing models’ ability to retain key genetic, transcriptional and epigenetic attributes of their parental tumors. However, our analysis also identified divergent features in a subset (<8%) of models, potentially resulting in a loss of fidelity and challenging their translational use. To identify whether this discordance might be driven by ex vivo adaptations or selective evolutionary pressures from in vitro culture, we profiled a representative subset (n=13 pairs) of glioblastoma (GBM, n=7) and pancreatic cancer (PAAD, n=6) tumor-model pairs using single-nucleus RNA-seq (snRNA-seq). Analysis of 132,593 nuclei (tumors: 66,054; models: 66,539) identified outlier GBM models undergoing proneural-to-mesenchymal (PTM) transition and highlighted serum culture media as a major associated factor of this process. Additionally, we found that divergences in selected PAAD (72,688 nuclei: tumors: 30,254; models: 42,434) pairs were linked to the complete loss of the rich tumor microenvironment cells in the original tumors and the in vitro expansion of select malignant subpopulations, as identified by network-based analysis of VIPER-inferred protein activity profiles. The observed divergences suggest that in vitro culture conditions can occasionally drive cellular reprogramming and alter phenotypic fidelity, resulting in varying degrees of state-specific transitions. However, the majority of the models (even those classified as outliers) are preserved for state-frequency in cellular subpopulations between tumors and models, and most if not all of the cellular states found in the parental tumor, thus faithfully representing intra-tumor heterogeneity. Taken together, these findings highlight the translational potential of the HCMI model repository and its translational relevance as an invaluable tool to support precision oncology.


 Dina ElHarouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, Toshinori Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Perez-Mayoral, Megan J. Stine, Eva Tonsing-Carter, James M. Clinton, The HCMI Network, Peter W. Laird, Calvin J. Kuo, Olivier Elemento, David L. Spector, Andrew D. Cherniack, Kyle Elrott, Martin L. Ferguson, Rameen Beroukhim, Katherine A. Hoadley, Nicolas Robine, Andrew McPherson, Matthew J. Garnett, David A. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse Boehm. Single cell transcriptional dynamics in the HCMI cancer model collection [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6358.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c16f756ddb575cdf26b92c703a6e6c80ae8425f" target='_blank'>
              Abstract 6358: Single cell transcriptional dynamics in the HCMI cancer model collection
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Pérez-Mayoral, Megan J. Stine, Eva Y. Tonsing-Carter, James M. Clinton, The Hcmi Network, Peter W. Laird, Calvin J. Kuo, O. Elemento, David L. Spector, Andrew D Cherniack, Kyle Elrott, M. Ferguson, R. Beroukhim, K. Hoadley, N. Robine, Andrew McPherson, M. Garnett, D. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse S. Boehm
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="Objectives The intestine exhibits high radiosensitivity and is susceptible to damage during radiotherapy for abdominal or pelvic tumors, especially at radiation doses exceeding 5 Gy, which can lead to severe gastrointestinal complications. This study aims to elucidate the mechanisms of DNA repair and immune responses in radiation-induced intestinal injury, with the goal of enhancing radiotherapy efficacy. Methods We employed weighted correlation network analysis (WGCNA) to analyze RNA sequencing data from intestinal tissues collected at various time points following irradiation, as well as from mice with targeted disruption of the NFE2-like bZIP transcription factor 2 (Nrf2) gene and their wild-type counterparts. Results Our analysis revealed enhanced DNA repair capacity and attenuated immune response 3.5 days after irradiation. Histone variants and ribosomal proteins were identified as potential contributors to DNA repair processes. We also constructed a competing endogenous RNAs (ceRNA) network including genes from both the DNA repair and immune modules and identified Akr1b8, Gsta3, and Nqo1 as radiation-effect genes regulated by Nrf2. Conclusions These findings provide valuable insights into the molecular mechanisms of radiation-induced intestinal injury and suggest potential targets for protecting normal intestinal tissue during radiotherapy. This knowledge may contribute to improved treatment outcomes and enhanced patient well-being.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35b6d07080e47e5026d5a66f95a6290e3e083b33" target='_blank'>
              Alterations of DNA Repair and Immune Infiltration in Radiation-Induced Intestinal Injury
              </a>
            </td>
          <td>
            Jiale Li, Huanteng Zhang, Xiaoxiao Jia, Jiebing Guan, Xumin Zong, Shiqi Li, Xinran Lu, Ningning He, Huijuan Song, Kaihua Ji, Manman Zhang, Jianguo Li, Qiang Liu, Jinhan Wang, Chang Xu
          </td>
          <td>2025-04-17</td>
          <td>Dose-Response</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Lung TRACERx is a prominent study employing multi-region and longitudinal multi-omics sequencing to unravel the evolutionary trajectories of cancer. While aberrant DNA methylation is described in most cancers, its interplay with genomic alterations and chromatin remodeling in non-small cell lung cancer (NSCLC) is less understood. We propose Allosteric Chromatin Activity Transition (AllChAT) as a framework to elucidate epigenetic dosage compensation mechanisms, focusing on essential genes impacted by oncogene-driven DNA methylation changes. The study utilized multi-region sampling from 59 NSCLC patients, encompassing 217 tumor regions and 59 adjacent normal tissue samples. RRBS was performed to profile DNA methylation patterns, with tumor-specific methylation rates determined using the CAMDAC tool. WES was conducted to identify somatic mutations and copy number alterations via ASCAT. Additionally, ChIP-seq for H3K4me3 and H3K27me3 was carried out on TRACERx patient-derived cell lines (PDCs) to investigate chromatin state changes associated with epigenetic transitions.We identified oncogenes such as RAC1, EGFR, and CDK4, whose expression scales with amplification and exhibited lower methylation levels in tumour regions where amplified compared to non-amplified tumor regions. We also discovered oncogene-proximal genes under dosage compensation by DNA methylation associated with amplification of CCND1 in LUSC and LUAD, as well as CDK4, KRAS, and GNAS, specifically in LUAD. These dosage-compensated genes were enriched for essential genes. Using chromatin immunoprecipitation on TRACERx patient derived cell lines (PDCs) from 2 tumors and one normal sample, we observed that oncogenes such as CDKN1B, FGFR1, and JAK2 were co-amplified and associated with chromatin opening and hypomethylation at amplified loci compared to non-amplified loci. Additionally, we found essential passenger genes, including NOP2, DCTN6 and FOXD4, were associated with closed chromatin and increased DNA methylation at the respective loci when amplified compared to when not amplified in tumor PDCs compared to the normal PDC. As observed in TRACERx tumor tissues, we observed evidence for AllChAT around LRRC34 when co-amplified with the PI3KCA oncogene in both tumor PDCs. Conversely, as expected, no evidence of AllChAT was observed in non-amplified control regions such as for the essential passenger genes TOMM40 and RPS9 located proximal to the ERCC2 oncogene when not amplified nor for the essential gene ECT2 located proximal to PIK3CA. Taken together, these data suggest that copy number alterations around amplified oncogenes might trigger an allosteric chromatin transition and DNA hypermethylation of co-amplified passenger genes, which might be crucial to the regulation of essential genes during tumor evolution.


 Francisco Gimeno-Valiente, Carla Castignani, Xiaohong Liu, Georgia Stavrou, Nana E. Mensah, Elizabeth Larose Cadieux, Özgen Deniz, Mariam Jamal-Hanjani, Stephan Beck, Jonas Demeulemeester, Miljana Tanic, Charles Swanton, Peter Van Loo, Nnennaya Kanu. Allosteric chromatin activity transition mechanisms drive epigenetic dosage compensation in essential genes triggered by oncogene activation in TRACERx [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4094.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c882aec893125b1655a88dcecde1b32dd7df1e2c" target='_blank'>
              Abstract 4094: Allosteric chromatin activity transition mechanisms drive epigenetic dosage compensation in essential genes triggered by oncogene activation in TRACERx
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, Xiaohong Liu, Georgia Stavrou, Nana E. Mensah, E. L. Cadieux, Özgen Deniz, M. Jamal-Hanjani, Stephan Beck, Jonas Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="
 Our goal is to develop a cell culture model of cancer cells that persist under all challenges including therapies, mostly in quiescence, and evolve further to cause recurrence. The rationale for the choice of model as cell culture is that it is ideal of evaluating therapeutic strategies that may inhibit highly abnormal but adaptable cells. Body imposes a variety of bottlenecks ranging from different components of immunity to metabolic challenges to enforce quiescence and influence cancer evolution. As it is not possible to replicate all such bottlenecks in cell culture, our approach is to apply as severe a bottleneck as possible, thus eliminating all proliferative (non-adaptable) cells, leaving behind rare cells surviving in deep quiescence. We hypothesize that if the quiescent cell state being modeled can survive a severe bottleneck, it can also survive other bottlenecks of similar or lower severity. We have arrived at a prolonged lack of glutamine in cell culture medium as a bottleneck for selecting evolutionarily fit rare cells after trying several alternatives, including a lack of glucose, a partial lack of oxygen (hypoxia), or a lack of serum on surface that is unsuitable for cell attachment. While investigating glutamine dependence in aggressive triple-negative breast cancer cell lines, we serendipitously discovered that some rare cells could survive in deep quiescence not only this but variety of other metabolic and therapeutic challenges. Since glutamine and its metabolites are important in protein synthesis, nucleic acid synthesis, epigenome and epitranscriptome regulation, and in maintaining redox status, its sudden disappearance creates a catastrophe killing 99.99% cells; only rare cells survive in deep quiescence and evolve further to proliferate (an important feature of cancer recurrence/metastases). The adaptable cells selected in this manner have many desirable functional and molecular features that are relevant in resistant cancers with an elevated risk of recurrence. We will discuss our observations with rare cells (0.01% in population) surviving in quiescence for several weeks in aggressive breast cancer and melanoma cell lines and their functional characteristics. Our results obtained with adaptable cells from SUM149 TNBC cell line suggest that they are stem-like quiescent cells. Our gene expression analyses and phenotypic changes during quiescence suggest that like a model proposed for stem cell quiescence, cancer stem cells are not dormant; instead, they represent states of poised potential and active restraint as cells “idle” in anticipation of activation, proliferation, and differentiation. We will also discuss how we choose and evaluate therapies that would likely inhibit quiescent and slow-growing cancer cells. This framework may significantly improve anti-cancer drug discovery.


 Balraj Singh, Anthony Lucci. A strategic framework for modeling abnormal quiescence that drives cancer evolution, and for preclinical development of therapies to prevent recurrence and metastasis in high-risk patients [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5280.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f4f8607796131836a1e01cc0a05131a1e222c3" target='_blank'>
              Abstract 5280: A strategic framework for modeling abnormal quiescence that drives cancer evolution, and for preclinical development of therapies to prevent recurrence and metastasis in high-risk patients
              </a>
            </td>
          <td>
            Balraj Singh, Anthony Lucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa2044a37bf408a983f6463cafdba0b09b74c94" target='_blank'>
              A resource and computational approach for quantifying gene editing allelism at single-cell resolution
              </a>
            </td>
          <td>
            Matthew Ung, Gabriella Angelini, Ruijia Wang, Alyssa Pyclik, J. Lydeard, J. Ferrucio, Michelle Lin, Huanying Gary Ge
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction CRISPR/Cas9-edited induced pluripotent stem cells (iPSCs) are valuable research models for mechanistic studies. However, gene conversion between a gene-pseudogene pair that share high sequence identity and form direct repeats in proximity on the same chromosome can interfere with the precision of gene editing. Mutations in the human beta-glucocerebrosidase gene (GBA1) are associated with Gaucher disease, Parkinson’s disease, and Lewy body dementia. During the creation of a GBA1 KO iPSC line, we detected about 70% gene conversion from its pseudogene GBAP1. These events maintained the reading frame and resulted from GBA1-specific cleavage by CRISPR/Cas9, without disrupting the GBA1 gene. Method To increase the percentage of alleles with out-of-frame indels for triggering nonsense-mediated decay of the GBA1 mRNA, we supplied the cells with two single-stranded oligodeoxynucleotide (ssODN) donors as homology-directed repair (HDR) templates. Results We demonstrate that HDR using the ssODN templates effectively competes with gene conversion and enabled biallelic KO clone isolation, whereas the nonallelic homologous recombination (NAHR)-based deletion rate remained the same. Discussion Here, we report a generalizable method to direct cellular DNA repair of double strand breaks at a target gene towards the HDR pathway using exogenous ssODN templates, allowing specific editing of one gene in a gene-pseudogene pair without disturbing the other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/522033033b99ee07b46d656d628a06d2bb8f2da5" target='_blank'>
              Efficient GBA1 editing via HDR with ssODNs by outcompeting pseudogene-mediated gene conversion upon CRISPR/Cas9 cleavage
              </a>
            </td>
          <td>
            Joseph S. Lagas, M. Sentmanat, Xiaoxia Cui
          </td>
          <td>2025-04-30</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 DHX9, a multi-functional RNA helicase, contributes to genome stability by unwinding nucleic acid secondary structures, including DNA/RNA hybrids (R-loops), circular RNA and G-quadruplexes. Multiple cancer types exhibit increased expression of DHX9, and selective dependance to DHX9 has been observed in molecular defined tumors that exhibit genomic instability. These tumors often have elevated replicative stress due to rapid unregulated proliferation and defective DNA damage repair (DDRD) pathways; DHX9 inhibition in these tumors further increases replicative stress and DNA damage to catastrophic levels. We previously demonstrated that DHX9 genetic loss or inhibition by the DHX9 tool compound ATX968 was efficacious in tumor types that exhibit DDRD and/or genomic instability; these include tumors with defective mismatch repair and/or high microsatellite instability (dMMR/MSI-H) or alterations in the DDR genes BRCA1 and/or BRCA2 (BRCA).
 Here we describe ATX-559, a potent, selective, orally bioavailable, small-molecule inhibitor of DHX9 helicase activity that is currently in clinical development. In preclinical studies, ATX-559 treatment leads to increased accumulation of R-loops and G-quadruplexes resulting in increased replicative stress and stalling, subsequent DNA damage, and cell death selectively in cancer models with high genomic instability. In a panel of 36 colorectal (CRC) and endometrial cancer (EC) cell lines, approximately 55% were sensitive (IC50 ≤ 1 μM) to ATX-559 treatment, with enriched antiproliferation activity (80%) in dMMR/MSI-H cells. ATX-559 sensitivity was also enriched in breast cancer cell lines, including 76% of BRCA altered triple negative breast cancer (TNBC) models.
 ATX-559 is well tolerated in vivo across multiple cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models, when dosed orally BID for periods out to 28 days or more. Robust dose-dependent tumor growth inhibition (TGI) and regression are selectively observed in CRC and EC dMMR/MSI-H and BRCA altered TNBC CDX models. TGI is correlative with plasma and tumor drug exposure, as well as with induction of the target engagement PD biomarker, circBRIP1, in both tumors and mouse PBMCs. ATX-559 was also profiled in a PDX mouse clinical trial design in vivo study of 46 models including TNBC, CRC, EC, and gastric cancers. Oral treatment of 45 mg/kg BID demonstrated enriched sensitivity in BRCA altered TNBC (75%) and dMMR/MSI-H (68%) PDX models.
 Together, these findings represent preclinical pharmacological validation of DHX9 as a selective therapeutic target in cancers that exhibit genomic instability and replication stress. ATX-559 is currently in a first-in-human, Phase 1, open-label, dose-escalation and expansion study in patients with locally advanced or metastatic solid tumors and molecularly defined cancers (NCT06625515).


 Jennifer Castro, Matthew H. Daniels, Sunaina Nayak, David Brennan, Cindy Collins, Sophie A. Shen, Monique Laidlaw, Jie Wu, Anugraha Raman, Deepali Gotur, Kevin Knockenhauer, Shihua Yao, Simina Grigoriu, Gordon J. Lockbaum, Kate Newberry, Stephen J. Blakemore, P. Ann Boriack-Sjodin, Kenneth W. Duncan, Stuart Ince, Jason A. Sager, Robert A. Copeland, Serena J. Silver. ATX-559, a first in class DHX9 inhibitor, and targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1758.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c4901cd0ac19548c13b53daf9ce1ecb7db4d98" target='_blank'>
              Abstract 1758: ATX-559, a first in class DHX9 inhibitor, and targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            Jennifer Castro, Matthew H Daniels, S. Nayak, David Brennan, Cindy Collins, Sophie A. Shen, Monique Laidlaw, Jie Wu, Anugraha Raman, D. Gotur, Kevin E Knockenhauer, Shihua Yao, Simina Grigoriu, Gordon J Lockbaum, Kate Newberry, Stephen J Blakemore, P. Boriack-Sjodin, K. Duncan, Stuart Ince, Jason A. Sager, Robert A Copeland, Serena J Silver
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0631bffe60efc9e321a6ea4a06deb1f550391138" target='_blank'>
              Clonal effects of the Ras oncogene revealed by somatic mutagenesis in a Drosophila cancer model
              </a>
            </td>
          <td>
            Takuya Akiyama, M. Gibson
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) is still one of the most commonly diagnosed cancers worldwide. Treatment strategies include chemotherapy, radiotherapy, targeted therapy, and immunotherapy. Despite the availability of such a diverse set of therapeutic strategies, around a million people still die from this disease each year worldwide. Phenotypic intra-tumor heterogeneity, arising from both genetic and non-genetic origins, has become a significant focus of study due to its impact on treatment response. Among the various subpopulations within a tumor, the stem-like cells have been extensively investigated, as they are recognized as tumor initiators. In contrast, the subpopulations identified by the expression of genes associated with differentiation has received comparatively less attention. An interesting hypothesis is that differentiated populations might plastically give rise to stem cell populations fueling tumor regrowth. Here we examine phenotypic intra-tumor heterogeneity in patient-derived metastatic CRC (mCRC) tumoroids. Our goal was to decipher the unknown role of differentiated cells and to relate it to growth and therapeutic response. In particular, we sought to determine whether manipulating exogenously the proportions of differentiated cells within CRC tumoroids could alter the drug response. We analysed one KRAS wild type (wt) model and one mutant (kras) model by scRNAseq. Our data confirmed that tumoroids present two main subpopulations, one associated to stemness (with high expression of LGR5), and the other associated to differentiation genes (with high expression of KRT20), coherently with reported studies. First we identified two novel cell surface markers associated to the KRT20+ subpopulation: GABRA2 and GPRC5C. Using FACS, we confirmed that our models present a double-positive subpopulation at the protein level. RT-qPCR confirmed that the expression of both genes increased over time. We next studied the effect of different cytokines on the modulation of this subpopulation. We found that exogenously adding IL6 and TGF-b1 alters the proportion of our markers, with a stronger effect in the wt sample. For both models, ELISA assays did not indicate any significant secretion into the culture medium of these cytokines. Interestingly, we found a release of IL6 receptor (IL6R) in the medium, particularly from the wt sample, likely as a results of the previously characterized cleavage of its membrane-bound form. Furthermore, RT-qPCR showed that TGF-b1 induced its own expression, a phenomenon known as autoinduction. Our preliminary results using the wt sample indicate that pre-treatment with these cytokines affected the response to Oxaliplatin, making it more sensitive to the chemotherapeutic reagent. Overall, our results indicate that cytokines released in the tumor microenvironment might reshape the intra-tumor phenotypic heterogeneity impacting drug-response.


 Sabrina Johanna Fletcher, Irene Catalano, Elena Grassi, Sofia Borgato, Barbara Peracino, Luca Primo, Livio Trusolino, Andrea Bertotti, Alberto Puliafito. Single-cell analysis and cytokines modulation of intra-tumor subpopulations in colorectal cancer tumoroids [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2665.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de37ada758a8c9054780c41940fecac62f6f55b3" target='_blank'>
              Abstract 2665: Single-cell analysis and cytokines modulation of intra-tumor subpopulations in colorectal cancer tumoroids
              </a>
            </td>
          <td>
            S. J. Fletcher, Irene Catalano, Elena Grassi, Sofia Borgato, Barbara Peracino, Luca Primo, L. Trusolino, A. Bertotti, A. Puliafito
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d980820189344192705dd2c34f2a4d48d18931d" target='_blank'>
              An atlas of gene programs in non-small cell lung cancer delineates novel patterns of lineage vacillation, plasticity, and aggressiveness
              </a>
            </td>
          <td>
            Rui Hong, Pascal Klöckner, Sarah A. Mazzilli, Avrum E. Spira, J. Beane, Masanao Yajima, Joshua D. Campbell
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Neoadjuvant chemotherapy (NAC) is vital in treating primary breast cancer, yet the mechanisms behind its success are not fully understood. Understanding transcriptomic changes associated with NAC resistance is crucial for improving treatment. This study sought to identify transcriptomic alterations linked to NAC resistance, enhance comprehension of treatment resistance, and lay the groundwork for developing targeted strategies for patients unresponsive to standard NAC protocols.We analyzed the transcriptomes of 44 matched breast cancer specimens from nonresponder patients to NAC at the Colombian National Cancer Institute. Samples were collected before and after NAC, representing luminalA, luminalB/HER2-positive, luminalB/HER2-negative, and TNBC subtypes. We performed differential gene expression and enrichment analyses and compared hallmark gene sets. Furthermore, we used deconvolution algorithms to estimate tumor microenvironment (TME) infiltration and assess immune population activity within the TME. Our findings were compared with publicly available GEO database data.Gene expression patterns before and after treatment were distinct across subtypes, with FOS and NR4A1 common to all, and ZBTB16 unique to luminalB. FOSB was found in all luminal samples. ATF3, DUSP1, EGR1, and JUN were shared among HER2-negative subtypes. GEM was common to luminalA and LuminalB/HER2-negative subtypes. Pathway variations included complement and coagulation cascades, and IL-17, TNF, MAPK, and NF-κB signaling pathways among subtypes and post-NAC specimens. Post-treatment LuminalA samples had increased CD4 memory cells, Tregs, and neutrophils. The LuminalB/HER2+ group showed higher CD8+ and CD4+ memory-activated cells post-treatment. In LuminalB/HER2- post-treatment samples, B cell CD4 memory resting, Macrophages M2, and neutrophils were elevated. TNBC post-treatment samples had higher resting mast cells and CD4+ memory cells. MSIGBD comparison of gene sets linked to cisplatin, docetaxel, and doxorubicin resistance identified several shared genes. GeoDatabases confirmed FOS and NR4A1 across all groups, with other DEGs shared among subtypes.Our research indicates that NAC affects gene expression, cell function, and interactions within the TME, potentially contributing to treatment resistance. Assessing TME components is crucial for predicting NAC effectiveness and reducing the risk of recurrence. These results highlight the importance of studying diverse patient populations to gain fresh perspectives and address disparities in treatment outcomes. This investigation emphasizes the need to understand how TME influences chemotherapy resistance, paving the way for the development of more precise and efficient therapeutic strategies.


 H Michelle Guevara-Nieto, Carlos A. Orozco, Rafael S. Parra-Medina, Jovanny Zabaleta, Liliana Lopez-Kleine, Alba L. Combita. Molecular determinants to neoadjuvant chemotherapy resistance in breast cancer patients: Insights from a Latin-American cohort [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4632.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8df54e46e7183e7945f80187fc6745c6cb3b628" target='_blank'>
              Abstract 4632: Molecular determinants to neoadjuvant chemotherapy resistance in breast cancer patients: Insights from a Latin-American cohort
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos A. Orozco, R. Parra-Medina, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Combita
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gene expression during cell development and differentiation is orchestrated by distal regulatory elements that precisely modulate cell selective gene activity. Gene therapy vectors leverage these elements for precise spatiotemporal transgene expression. Here, we develop a one-shot approach to screen candidate regulatory sequences from large-scale epigenomics data for programmable transgene expression within gene therapy viral vectors. We assess a library of 15,000 short sequences derived from developmentally active elements during erythropoiesis using a clinically relevant reporter vector. These elements display a gradient of transcriptional enhancer activity in erythroid cells, with high cell type restriction and developmental stage specificity. Finally, replacing the canonical β-globin μLCR with a compact enhancer in a β-thalassemia lentiviral vector successfully corrects the thalassemic phenotype in patient-derived hematopoietic and stem and progenitor cells (HSPCs), while increasing viral titers and cell transducibility. Our approach provides further insights into enhancer biology with wider implications for human gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca3f35c5addd6ce9a5ce45bc98d0104eddf947b" target='_blank'>
              Large-scale discovery of potent, compact and erythroid specific enhancers for gene therapy vectors
              </a>
            </td>
          <td>
            N. Psatha, P. Sova, G. Georgolopoulos, K. Paschoudi, Mineo Iwata, Jordan Bloom, Tatyana Ulyanova, Hao Wang, A. Kirtsou, Ninos-Ioannis Vasiloudis, Matthew S Wilken, J. Stamatoyannopoulos, Evangelia Yannaki, Thalia Papayanopoulou, G. Stamatoyannopoulos, J. Vierstra
          </td>
          <td>2025-05-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Endogenous Retroviruses (ERVs) are remnants of ancient retroviral infections that have permanently integrated into host genomes, including humans. Human Endogenous Retroviruses (HERVs) are ‘fossil’ viral sequences, having lost their ability to generate infectious viruses due to mutations and deletions accumulated over millions of years. Although most HERVs are deemed non-functional, they constitute up to 8% of the human genome. Some HERVs have roles in gene expression regulation, and others are transcribed under specific conditions, such as in cancer. HERV-K is the most recently integrated class within our genome, with some members retaining relatively intact open reading frames (ORFs), suggesting their potential to encode proteins. In cancer, reactivation of HERV expression may impact gene expression, genomic stability, and cell signaling pathways.In this study, we investigated the expression of HERVs in breast cancer (BC) and non-tumor breast tissue using RNA sequencing (RNA-Seq) data from The Cancer Genome Atlas (TCGA). Specifically, we analyzed the expression profiles of HERVs across BC subgroups and their association with protein-coding gene expression profiles. We identified differentially expressed HERVs in BC subgroups (HR-/HER2-, HR+/HER2+, HR-/HER2+, HR+/HER2-) in comparison to non-tumor samples. Our analysis revealed 98 differentially expressed HERVs in HR-/HER2-, 51 in HR+/HER2+, 81 in HR-/HER2+, and 48 in HR+/HER2-. Distinct expression profiles across all subtypes were evident. Furthermore, we identified 45 differentially expressed protein-coding genes with transcription start sites within 5 kb of differentially expressed HERVs. This proximity suggests a potential regulatory influence of HERVs on these genes, indicating the need for further research to clarify the underlying mechanisms. In conclusion, the unique HERV expression profiles across BC subtypes suggest specific molecular characteristics and gene regulation patterns. These findings may aid in identifying new therapeutic targets and treatment strategies for breast cancer, emphasizing HERVs' importance in tumor biology.


 Nathália Da Roz D'Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, Pedro Alexandre Favoretto Galante. Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2387.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cf64c5137ce4675e3a801e6fb302671875782c" target='_blank'>
              Using Cancer Profiles to Identify Synthetic Lethal Therapeutic Targets and Predictive Biomarkers in Cancer Gene Dependency Data
              </a>
            </td>
          <td>
            Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2025-04-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Neuroblastoma (NB) is a highly metastatic pediatric cancer originating from neural crest cells. Progression is frequently driven by MYCN amplified (MA) disease, a marker of poor prognosis presented in approximately 20% of cases. While MA NB is associated with increased MYCN protein expression, recent results suggest that non-coding RNA (ncRNA) elements within the MYCN mRNA independently contribute to its oncogenic potential. The let-7 microRNA (miRNA) family, which targets the 3′ UTR of MYCN, is sequestered by MYCN mRNA itself in MA disease, thus impairing its function and establishing that ncRNA elements can contribute to disease pathology. To further explore the role of these ncRNA elements, we engineered cells expressing either MYCN-ORF (MYCN protein-coding region only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (where EGFP replaces MYCN protein coding sequence but retains all ncRNA elements). In vitro growth analyses showed that MYCN-GL-expressing cells exhibited the most robust growth, followed by Null-GL and MYCN-ORF. In vivo, the Null-GL construct induced the highest incidence and largest tumors of all constructs, followed by MYCN-GL and then MYCN-ORF. Next-generation sequencing revealed that MYCN-GL and Null-GL cells were enriched for targets of microRNAs that also target MYCN, including let-7, miR-101, and miR-34a. This suggests that the MYCN 3’UTR may interact with these microRNAs, thus protecting other mRNA targets. Additionally, MYCN-GL and Null-GL constructs exhibited more dynamic differential gene expression than MYCN-ORF and EGFP-expressing cells. Gene ontology analysis showed that MYCN-GL and Null-GL cells were more enriched in cancer-related pathways, such as RNA metabolism and microRNA processing. Non-coding MYCN mRNA elements may thus regulate broader cellular functions that enhance oncogenesis. Exon usage analysis revealed that MYCN-GL and Null-GL constructs influenced splicing patterns, with increased exon inclusion and exclusion events observed across multiple chromosomes. KEGG pathway analysis of differentially spliced genes further confirmed non-coding RNA element influence on critical oncogenic pathways, including PI3K signaling, TNF signaling, viral carcinogenesis, cell survival, proliferation, and immune evasion. Whole genome sequencing revealed significant similarities in copy number variation (CNV) patterns between MYCN-GL and Null-GL constructs. These results demonstrate that non-coding elements within MYCN mRNA are potent drivers of tumorigenesis, influencing microRNA interactions, splicing patterns, genetic patterning, and overall activation of oncogenic pathways. Our findings provide new insights into the oncogenic mechanisms of MA NB, with broad implications for the genetic contribution of noncoding RNA elements in other oncogene-driven human malignancies.


 Vishwaben Patel, Lorraine-Rana E. Benhamou, Dr. John T. Powers. Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3952.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9fe2ca640f1cfacf9753f7fd79adbb41fae4fa" target='_blank'>
              Abstract 3952: Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            V. Patel, Lorraine-Rana E. Benhamou, Dr. John T. Powers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is a heterogeneous disease both within and across individuals. Genomic alterations, epigenetic changes, and interactions with cells in the tumor microenvironment (TME) contribute to different states in tumor cells. Single-cell RNAseq (scRNAseq) can provide insights into the transcriptional heterogeneity of tumor cells and cell types in the TME. However, most scRNAseq studies are limited by small sample sizes. Here, we performed the first gene program association study in PCa to identify gene modules and immune cell types associated with intra-tumoral heterogeneity and metastasis.



 14 datasets were merged to create a harmonized scRNAseq atlas. Copy number inference and boosted gradient tree models were trained to identify the tumor cells. Celda was used to identify co-expression modules. Linear mixed effect models were used to associate modules with clinical phenotypes. Finally, a transformer model similar to RoBERTa for annotation of PCa datasets was developed and validated in 2 independent datasets.



 We compiled an atlas of 560, 492 cells from 128 PCa patients with localized primary PCa, Castration resistant PCa (CRPC), and metastatic PCa from 7 different sites. 166 co-expression modules were identified across 106, 998 tumor cells. 2 cell-cycle modules were consistently upregulated across all metastatic sites while prostate lineage and antigen presentation modules were consistently downregulated. Different metastatic sites exhibited unique module expression patterns. For example, a module of FOLH1 and SPON2 was upregulated specifically in brain metastasis while erythroid modules were found in liver metastasis. 4 different modules were significantly higher in neuroendocrine prostate cancers (NEPC). Each NE tumor was not monolithic but expressed a unique combination of these modules. A novel subtype of NEPC tumors was identified which predominantly expresses the HES1 module instead of the canonical NE module. We developed a novel scoring system to identify NEPC tumors in bulk RNA-seq data using a combination of NE-associated modules which performed better than previously established gene set signatures (AUC = 0.938). Within the TME, we detected a T-cell cluster signified by heat shock protein genes enriched in African American PCa patients than European American patients. Furthermore, a transformer neural network was developed for automated annotation of scRNAseq PCa datasets which achieved a 91% accuracy in validation datasets.



 We generated a comprehensive atlas of cells from PCa, characterized novel gene programs that defined tumors cells in localized and metastatic samples, identified a novel subtype of NEPC, and developed a robust computational tool to predict these states in new studies. These findings can potentially be used clinically to stratify high-risk patients with localized disease.



 Hanbing Song, Junxiang Xu, Paul Allegakoen, Yih-An Chen, Hannah N. Weinstein, Arda Demirci, Isaac Salzman, Keila Velazquez-Arcelay, Margaret Tsui, Jon Akutagawa, Julia H. Pham, Peter R. Carroll, Matthew R. Cooperberg, Ezequiel Goldschmidt, Nancy Greenland, Joshua D. Campbell, Franklin W. Huang. A gene program association study (GPAS) in prostate cancer reveals novel gene modules associated with plasticity and metastasis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7506.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8ac6134f8fc2dc9bc5d1f05d773b9d8559310fb" target='_blank'>
              Abstract 7506: A gene program association study (GPAS) in prostate cancer reveals novel gene modules associated with plasticity and metastasis
              </a>
            </td>
          <td>
            Hanbing Song, Junxiang Xu, Paul Allegakoen, Yih-An Chen, H. N. Weinstein, Arda Demirci, Isaac Salzman, Keila Velazquez-Arcelay, Margaret Tsui, Jon Akutagawa, Julia H. Pham, Peter R. Carroll, M. Cooperberg, Ezequiel Goldschmidt, Nancy Y. Greenland, Joshua D. Campbell, Franklin W. Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Introduction Immune checkpoint blockade (ICB) therapy has shown promise in treating advanced colorectal cancer, particularly in patients with microsatellite instability-high (MSI-H) tumors. However, only a subset of these patients responds favorably, highlighting the need for strategies to improve immunotherapy efficacy. Methods To identify potential regulators of immunotherapy response, we conducted a comprehensive analysis of colorectal cancer datasets from The Cancer Genome Atlas (TCGA). We performed multi-omics analyses and functional assays in both human and murine colorectal cancer cell lines. Additionally, we evaluated tumor growth and immune cell infiltration using syngeneic mouse models. Results Our analysis revealed that RNA binding motif protein 15 (RBM15) is highly expressed in colorectal cancer and correlates with poor patient prognosis. Functional studies demonstrated that RBM15 loss led to increased expression of fumarate hydratase (FH). This led to decreased levels of fumarate, a metabolite known to suppress anti-tumor immune responses. In vivo, RBM15 depletion significantly delayed tumor progression and enhanced CD8⁺ T cell infiltration and activation in the tumor microenvironment. Discussion These findings identify RBM15 as a negative regulator of anti-tumor immunity in colorectal cancer. Targeting RBM15 may represent a novel strategy to boost immune responsiveness and improve outcomes for patients undergoing immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6fd5bb65d4751738b1ded96c2f1862a85aeb651" target='_blank'>
              RBM15-mediated metabolic reprogramming boosts immune response in colorectal cancer
              </a>
            </td>
          <td>
            Chen Wang, Mengyan Chen, Panyu Chen, Jinlu Han, Hong Hu, Jiong Chen, Qiong Wu, De Zhao, Tongshuai Wang, Jingyi Zhou, Qi Li, Runkai Zhou, Yugang Wen, Jing Yang, Min Shi, Yugang Wang
          </td>
          <td>2025-04-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Cyclin E1 (CCNE1) expression has been associated with replicative stress and defects in double-strand break repair in multiple tumor types; however, this has yet to be demonstrated in ovarian cancer. Herein, we describe CCNE1 DNA amplification (AMP), messenger RNA (mRNA) expression, and cyclin E1 protein expression in 2 large cohorts of high-grade serous ovarian cancer (HGSOC) (The Cancer Genome Atlas [TCGA] and Caris Life Sciences, Phoenix, AZ, USA).



 8127 HGSOC samples were analyzed by next-generation sequencing (NextSeq, 592 genes or NovaSeq, whole exome sequencing) and RNA (NovaSeq, whole transcriptome sequencing) (Caris Life Sciences). For genomic loss of heterozygosity (LOH), 22 autosomal chromosomes were split into 552 segments and the LOH of single nucleotide polymorphisms within each segment was calculated (LOH-H ≥16%). Homologous repair deficient (HRD+/−) status was determined by high genomic scar score (GSS) or BRCA1/2 mutant (MT) [HRD+], or low GSS and BRCA1/2 wild-type (WT) [HRD−], respectively. GSS was calculated from large-scale state transitions (LSTs) and LOH (GSS-H ≥46). 1177 samples from TCGA were also included in this analysis.



 In the Caris data set, HGSOC CCNE1 AMP was significantly more frequent in BRCA1/2 WT HGSOC tumors (CCNE1 AMP: 4.88% BRCA1/2 MT vs 21% BRCA1/2 WT; P<0.0001), as reported in other studies. Additionally, median expression of CCNE1 mRNA was higher in BRCA1/2 WT samples when compared with BRCA1/2 MT samples (1.11-fc; P<0.0001). Median CCNE1 mRNA was higher in HRD− HGSOC compared with HRD+ HGSOC (Caris: 1.11-fc, P<0.0001; TCGA: P<0.0001). In both data sets, CCNE1 AMP was associated with lower global LOH prevalence (CCNE1-AMP %: 22.6% in LOH-L vs 13.9% LOH-H in Caris data, P<0.0001; TCGA, P<0.0001). However, there was a significant increase in CCNE1 mRNA expression in LOH-H compared with LOH-L HGSOC Caris samples (1.27-fc, P<0.0001). There were no associations between CCNE1 AMP or RNA/protein expression and LST in either data set, or with telomeric allelic imbalance score in TCGA. CCNE1 AMP was associated with aneuploidy in TCGA (P<0.001). We also examined an independent set of commercial ovarian cancer samples for CCNE1 protein and pH2AX expression by immunohistochemistry to evaluate for an association with double-strand breaks, which will be included in the presentation.



 Together, these data demonstrated that CCNE1 AMP and RNA/protein expression were associated with various types of replicative stress in HGSOC. These data warrant further investigation on the impact of CCNE1 AMP and CCNE1 expression on clinical outcomes.



 Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, Ming Poi, Sharon Wu, Edoardo Missiglia, Bettina Bisig, Krisztian Homicsko. Association of cyclin E1 expression with genomic instability in ovarian cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2846.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a24afa888bf591284938fd750888dc7dfa3335e" target='_blank'>
              Abstract 2846: Association of cyclin E1 expression with genomic instability in ovarian cancer
              </a>
            </td>
          <td>
            Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, M. Poi, Sharon Wu, E. Missiglia, Bettina Bisig, K. Homicsko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Adult stem cells possess a unique ability for asymmetric self-renewal, enabling the simultaneous production of non-stem cell progenitors and self-renewing stem cells, distinguishing them from symmetrical cell division. In cancer cells, the regulation of cell division is often disrupted to sustain proliferation. However, specific patterns might be essential for evading cell death and adapting to the tumor microenvironment. In this study, transcriptomic data from engineered p53-expressing and p53-null mouse fibroblasts were re-analyzed to identify expression signatures of asymmetric self-renewal associated (ASRA) and p53/IMPDH2-related self-renewal patterns associated (SRPA). Pathway analysis revealed that ASRA signatures were enriched in immune regulation and angiogenesis pathways, in addition to cell cycle regulation. Genomic analysis using The Cancer Genome Atlas (TCGA) database demonstrated correlations between ASRA signatures and survival outcomes, as well as molecular properties across various cancer types, including stomach adenocarcinoma (STAD). Additionally, public single-cell RNA sequencing (scRNA-seq) and clinical dataset analyses revealed the role of ASRA genes in the interactions between tumor cells and surrounding immune and stromal cells. These findings suggested that targeting ASRA signatures is a promising strategy for developing novel cancer therapies.


 Seokyoung Hwang, Minju Lee, Jayhyun Cho, Minkyu Kim, Sang Kook Lee, Minsoo Noh. Bioinformatics analysis of the role of asymmetric self renewal associated genes in the cancer progression [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5090.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9bbc590d0b38c9e40c5b483743c16479ab91ce" target='_blank'>
              Abstract 5090: Bioinformatics analysis of the role of asymmetric self renewal associated genes in the cancer progression
              </a>
            </td>
          <td>
            Seokyoung Hwang, Minju Lee, Jayhyun Cho, Minkyu Kim, Sang Kook Lee, Minsoo Noh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Loss of heterozygosity (LOH) is a common event in cancer development, disrupting gene and protein translation and resulting in loss of function (LOF). In terms of additional vulnerability, recent studies have suggested that investigating LOF due to LOH can be beneficial for cancer treatment. Additionally, genetic regions, including single nucleotide polymorphisms (SNP) linked to LOF caused by LOH, vary across cancer types. From this perspective, we conducted an in-silico analysis of Korean genomic data, cancer cell line data, and public genomic databases to identify genes associated with SNPs leading to LOF due to LOH. We analyzed a total of 10 million SNPs from a large cohort of 1, 496 Koreans using whole genome sequencing (WGS) data, as well as up to 9 million SNPs from 1, 007 cancer cell lines (including 129 lung, 21 stomach, and 28 colon samples) based on whole exome sequencing (WES). After filtering out rare variants, we identified 453 functional SNPs affecting protein translation in both Korean and cancer cell line data. Of these, 194 SNPs with heterozygous genotype frequencies between 0.1 and 0.9 were selected. Using copy number loss regions from the COSMIC database, we identified 39 genes associated with potential LOH-related SNPs. Finally, through the analysis of gene expression data from The Human Protein Atlas and previous studies, six LOH-associated genes linked to LOF in Koreans were identified: ADAMTS1, ANXA2, IL17RB, LPL, TUBA8, and ZNF117. Based on these six genes, we propose a list of drugs that, when used in combination, enhance the efficacy of conventional anticancer treatments. We plan to validate the effectiveness of these drugs in addressing additional vulnerabilities through in vitro experiments. While experimental validation is ongoing, we anticipate that our study will serve as a valuable resource for researchers aiming to identify population-specific LOH-associated genes using in-silico approaches.


 Sun-Woo Lee, Jae-Yoon Kim, Hyojin Heo, Seung-Woo Baek, Jee-Woo Seo, Jong-Lyul Park, Seon-Young Kim. Identification of loss of heterozygosity-associated genes linked to loss of function in Korean population through in-silico analysis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2411.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b985272e8a9e4b7bc99e5a2a4b3235fd2b5e2af6" target='_blank'>
              Abstract 2411: Identification of loss of heterozygosity-associated genes linked to loss of function in Korean population through in-silico analysis
              </a>
            </td>
          <td>
            Sun-Woo Lee, Jae-Yoon Kim, Hyojin Heo, Seung-Woo Baek, Jee-Woo Seo, Jong-Lyul Park, Seon-Young Kim
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Androgen receptor (AR) signaling is critical for the survival and proliferation of prostate cancer (PCa) cells, thus making androgen deprivation therapy (ADT; e.g., castration) the mainstay for treatment. Notably, the oncogenic transcriptional functions of AR rely on a host of chromatin-binding regulatory proteins, which includes a pioneer transcription factor called FOXA1. FOXA1 de-compacts chromatin to enable DNA binding of AR and activation of target gene expression. Our lab found FOXA1 alterations to recur within three distinct structural classes in over 35% of metastatic castration-resistant PCa (mCRPC) cases in Caucasian White men. Subsequent studies reported FOXA1 mutations to be prevalent in over 40% of primary PCa in Chinese patients, thus positioning FOXA1 as a principal oncogene in this disease. Yet, the tumorigenic potential and pathobiology of FOXA1 alterations remain unexplored in vivo. Here, we have developed and characterized the first in-field transgenic mouse models that conditionally overexpress FOXA1 mutants in the prostate luminal epithelia. Our findings reveal that FOXA1 class1 mutations (i.e., wing 2 alterations) in a Trp53-null background drive high-grade, invasive luminal prostate adenocarcinoma with complete penetrance by 40 weeks. These lesions retain cardinal luminal features of the human disease, including regression upon androgen withdrawal. Mechanistically, the Class1 mutants reprogram AR and activate the PI3K/mTORC1 pathway to drive transformation. In contrast, FOXA1 class 2 mutations (i.e., C-terminal truncations) that are acquired in human mCRPC do not drive prostate luminal transformation in mice. Instead, these mutations induce intra-luminal plasticity by reprogramming differentiated cells into a progenitor stem-like state within androgen-replete normal tissues—a phenomenon otherwise induced upon castration. Class2 mutants lead to a 20-fold expansion of Ar+/Ck8+ luminal epithelia that gain expression of stemness markers such as Trop2, Ck4, and Psca. Mechanistically, we found the cistromically-dominant Class2 mutants to pioneer over 40, 000 neo-enhancer elements, bound by stemness-associated transcription factors, which instructed an androgen-insensitive luminal progenitor cell fate. Consistently, we found Class2-mutant mouse prostates to show minimal atrophy upon castration, with immunohistological assessment uncovering a higher density of Ki67+ luminal epithelial cells relative to both wild-type and Class1-mutant tissues. Collectively, our data establishes FOXA1 as a multifaceted oncogene in AR-dependent prostate cancers, wherein divergent evolution of FOXA1 mutational classes distinctly drive either cancer formation or therapy-resistant, intra-luminal plasticity during disease progression. These findings also represent the first report of FOXA1-driven prostate adenocarcinoma in mice.


 Sanjana Eyunni, Rahul Mannan, Yuping Zhang, Eleanor Young, Jie Luo, Jean Ching-Yi Tien, Qiuyang Zhang, Matthew Pang, Somnath Mahapatra, James George, Mustapha Jaber, Hamzah Hakkani, Sandra Carson, Abbey Todd, Noshad Hosseini, Ryan Rebernick, Xuhong Cao, Rui Wang, Fengyun Su, Rohit Mehra, Marcin Cieslik, Arul Chinnaiyan, Abhijit Parolia. Divergent evolution of FOXA1 mutations drive prostate tumorigenesis or therapy-resistant intra-luminal plasticity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3825.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b1ea31e530ff890dfc3d45aab43f96f9c81ec4" target='_blank'>
              Abstract 3825: Divergent evolution of FOXA1 mutations drive prostate tumorigenesis or therapy-resistant intra-luminal plasticity
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Jie Luo, Jean Ching-Yi Tien, Qiuyang Zhang, Matthew Pang, Somnath Mahapatra, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J. Todd, Noshad Hosseini, Ryan J. Rebernick, Xuhong Cao, Rui Wang, Fengyun Su, Rohit Mehra, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>168</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb9f7e3e34a8c78dbc563d40d2ad93cdb90edab" target='_blank'>
              Engineered protein circuits for cancer therapy
              </a>
            </td>
          <td>
            Andrew C. Lu, Lukas Moeller, Stephen Moore, Shiyu Xia, Kevin Ho, Evan Zhang, Mark W. Budde, Haley Larson, Ali Ahmed Diaz, Bo Gu, James M. Linton, Leslie Klock, Michael J. Flynn, Xiaojing J. Gao, D. Siegwart, Hao Zhu, M. Elowitz
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, A. Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Pancreatic cancer (PC) is the third leading cause of cancer deaths in US with a 5-year survival rate of 13%. Most patients are diagnosed at advanced stages, and conventional treatments remain ineffective and highly toxic to these patients as both chemotherapy and radiation affect normal, healthy cells, leading to adverse side effects. We have adapted CRISPR-Cas9 as a cancer-specific killing strategy that spares normal cells. CRISPR-Cas9 is a molecular scissor that induces DNA double strand breaks (DSBs) at its targeted genomic locations. We use CRISPR-Cas9 to target the subset of somatic mutations that create protospacer adjacent motifs (PAMs). Using S. pyogenes Cas9 that recognizes a 5’-NGG-3’ PAM, we identified somatic single base substitutions (SBSs) that give rise to novel PAMs (e.g. NCG→NGG), and use the upstream sequence to design sgRNAs for cancer-targeting. We have discovered that ∼13% of SBSs in PC form novel PAMs, and these cancer-specific sgRNAs exhibited on-target activity and lack of off-target events. Notably, while examining WGS data from rapid autopsy samples, we found that 90% of PAMs were maintained among primary carcinomas and metastases. Using multi-target sgRNAs targeting 2-16 sites in the non-coding regions of human genome, we demonstrate a dose-response relationship in which >10 target sites are required to achieve >99% cell elimination. Moreover, we established that simultaneous induction of multiple DSBs by CRISPR-Cas9 causes extreme chromosomal instability that ultimately leads to cell death. We detected massive chromosomal aberrations, including dicentrics, ring chromosomes, and polyploidization. Surprisingly, CRISPR-Cas9-induced DSBs are significantly more cytotoxic in PCs than a comparable number of radiation-induced DSBs, in which the number of DSBs required to achieve equitoxic effects was ∼3 times higher for radiation than CRISPR-Cas9. Finally, we have developed a pipeline to screen for the most toxic sgRNAs in a given PC for selective killing. Using our PAM-based approach, we identified >100 Panc10.05-specific sgRNAs that have no off-target activity at 0- and 1-mismatch sites. We cloned them into lentiviral vectors and transduced in bulk, followed by NGS to discover sgRNAs that decreased in representation in the targeted cell line but not in the controls. We selected the top sgRNAs that have 100% variant allele frequency (VAF), cloned 4 of them into a multi-sgRNA expression vector, and transduced it into co-cultures consisting of two PC cell lines. Greater than 85% selective reduction of the targeted cell line was detected. Then, we selected nine top sgRNAs, transduced them as a pool into Panc10.05 cell line, and injected these cells into athymic, nude mice for tumor growth. We observed >90% tumor growth inhibition with these cancer-specific sgRNAs. Overall, our study highlights the therapeutic potential of CRISPR-Cas9 as an anti-cancer strategy and supports the genetic targeting of PCs.


 Selina Shiqing K. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Laura Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, Michael Goldstein, Ying S. Zou, James R. Eshleman. CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 471.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Human papillomaviruses (HPVs), like many other viruses, are able to integrate their genomes into the host cellular genome. This integration can activate viral oncogenes or alter the function of cellular oncogenes and tumor suppressor genes, thereby increasing the likelihood of HPV-associated tumor development. In particular, HPV types 16 and 18 are responsible for over 70% of all cervical, anal, and oropharyngeal cancers worldwide, with rising incidence. Even more, high-resolution mapping of preferred integration sites using LR-Seq technologies offers deep insights into the molecular mechanisms of HPV integration. LR-Seq enables the detection of complex integration patterns, where the viral genome can be replicated and amplified into virus–host concatemers, including events within large structural variations or highly repetitive genomic regions. Furthermore, aligning LR-Seq data to the latest T2T reference genome (hs1) is necessary to provide new information about viral integration in genomic regions that were previously inaccessible, such as centromeres and other structurally complex repeat-rich loci. In this review, we provide insights into HPV genomic integration revealed by LR-Seq technologies, with a particular focus on how the use of the complete T2T reference genome enhances the detection of integration events in previously uncharacterized, repeat-rich regions of the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bd3d524ef4cdc3c87b3f69c07be4376d8e44c3" target='_blank'>
              Novel Avenues for the Detection of Cancer-Associated Viral Genome Integrations Using Long-Read Sequencing Technologies
              </a>
            </td>
          <td>
            Larissa-Anna Bergmann, Alicja Pacholewska, Michal R. Schweiger
          </td>
          <td>2025-05-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Chromatin remodeling proteins contribute to DNA replication, transcription, repair, and recombination. The chromodomain helicase DNA-binding (CHD) family of remodelers plays crucial roles in embryonic development, neurogenesis, and carcinogenesis. As the founding member, CHD1 is capable of assembling nucleosomes, remodeling chromatin structure, and regulating gene transcription. Dysregulation of CHD1 at genetic, epigenetic, and post-translational levels is common in prostate cancer and other malignancies. Through interacting with different genetic alterations, CHD1 possesses the capabilities to exert oncogenic or tumor-suppressive functions in context-dependent manners. In the past decade, my study uncovered the novel cell-intrinsic and -extrinsic roles of CHD1 during prostate cancer development, progression, and therapy resistance (Nature 2017, Cancer Discover 2020, Cancer Research 2022, Nature Cancer 2024). Cancer genome studies have uncovered a unique subtype of prostate cancer characterized by speckle-type BTB/POZ protein (SPOP) mutations, which stabilize AR and its co-activators and have been associated with prolonged responses to androgen deprivation therapies and favorable prognoses. Loss of CHD1 frequently co-occurs with SPOP mutations; however, how concurrent CHD1 loss impacts SPOP-mutated prostate cancer remains unclear. Most recently, by generating the first-in-the-field genetically engineered mouse models, we provided genetic evidence supporting that CHD1 loss accelerates SPOP-mutated prostate cancer progression and confers resistance to castration in this context. We uncover a non-canonical epigenetic function of CHD1 and demonstrate that CHD1 reprograms lipid metabolism by governing SREBPs. CHD1 loss-induced lipid reprogramming supplies intratumoral androgen biosynthesis and retains AR transcriptional activity in SPOP-mutated prostate tumors upon castration. Furthermore, the mechanistic studies facilitate the development of a novel biomarker-driven combinatorial therapy for castration-resistant prostate cancer harboring CHD1 loss and SPOP mutations. These findings advance our understanding of an emerging prostate cancer subtype and mechanism of action and provide novel therapeutic strategies for men with advanced prostate cancer.


 Feiyu Chen, Di Zhao. Decipher actionable tumor suppressor defects in prostate cancer for precision medicine [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 273.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262a87481193ac96806ab02306fdc71e5a95bc29" target='_blank'>
              Abstract 273: Decipher actionable tumor suppressor defects in prostate cancer for precision medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The efficacy of immunotherapy fundamentally depends on effective T cell-tumor cell interactions, yet the molecular determinants orchestrating this engagement are not fully understood. Here, we systematically mapped and perturbed key regulators of the T cell-tumor cell interactome using innovative cell-cell interaction assays. We performed genome-wide CRISPR screens to identify genes critically required for physical and functional interactions between T cells and tumor cells. Using TurboID proximity labeling and T cell-tumor cell conjugates, we identified the complex N-glycan biosynthesis pathway as a crucial regulator of T cell-tumor cell interactions. Recognizing the role of N-glycans in protein trafficking and surface retention, we subsequently performed cell surface proteomics and TurboID-based interactome profiling on complex N-glycan-deficient T cells, revealing increased expression of cell adhesion and co-stimulatory molecules. T cells with perturbed N-glycans show a shift towards an effector memory phenotype with elevated granzyme and integrin expression. In vivo, both genetic and pharmacological interventions that decrease the number of complex N-glycans on the surface of T cells enhanced intra-tumoral infiltration. Our findings provide a comprehensive functional atlas of molecules modulating T cell-tumor cell interactions and position the complex N-glycan biosynthesis pathway as a potential therapeutic target to strengthen T cell-mediated cytotoxicity in cancer immunotherapy.


 Adriaan F. de Groot, Zowi R. Huinen, Juan Simon Nieto, Daniel S. Peeper. Genome-wide CRISPR screens for genes inhibiting T cell-tumor cell interactions identify complex N-glycans [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3960.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6a5926479af2b1d3bad59b8a321348441349405" target='_blank'>
              Abstract 3960: Genome-wide CRISPR screens for genes inhibiting T cell-tumor cell interactions identify complex N-glycans
              </a>
            </td>
          <td>
            A. F. de Groot, Zowi R. Huinen, Juan Simon Nieto, D. Peeper
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 Recently we and others reported high prevalence of clonal hematopoiesis (CH) in long-term survivors of pediatric cancers by performing ultra-deep panel sequencing of peripheral blood samples, a phenomena significantly associated with exposures to radiotherapy and chemotherapeutic agents. Motivated by a preliminary finding that a therapy-related STAT3 CH mutations were highly enriched in T-cells of 3 Hodgkin’s Lymphoma (HL) survivors, we initiated a systematic investigation to map the CH mutations across different immune cells populations. Blood samples of 108 survivors of 13 pediatric cancer types (4 hematological, 8 solid and 1 central nervous system (CNS) tumor types), known to harbor CH mutations discovered by prior bulk sample analysis, were selected for multi-omics single-cell profiling with Mission Bio Tapestri assay. For mutation screening, we designed a panel of 29 amplicons across 12 genes, covering 52 CH mutations detected in bulk samples. Amongst the 23 completed samples, we identified 99 mutations in CH-genes (DNMT3A, STAT3, TP53, KRAS, TET2, and ASXL1), and 67.7% of those were found in all cell types and the rest were present in at least two cell types. We discovered that a subset of CH mutations was enriched (FDR < 0.05) in certain cell types across several samples. For example, STAT3 Y640F mutations were consistently enriched in CD8 T-cells in 6 out of 7 HL samples with a mean fold change of 18.5. DNMT3A mutations were enriched in cells from both myeloid (monocytes) and lymphoid (CD4/8 T-cells and NK-cells) lineages and affected samples from five cancer types. Specifically, we found that DNMT3A mutations were enriched in monocytes in 2 HL and 2 acute lymphoblastic leukemia (ALL) cases with an average fold change of 2.3; and in CD8 T-cells in 5 samples from Wilms tumor, soft tissue sarcoma, ALL, HL, and CNS with an average fold change of 4.9. KRAS L19F was found to be enriched in monocytes and NK-cells in an ALL sample. By contrast, we observed a uniform distribution of some TP53 mutations across all cell types, suggesting that they confer comparable selective advantage across all immune cell types, consistent with the biological function of TP53 in cell cycle. To our knowledge, this is the first large-scale multi-omics single-cell profiling of CH mutations in long-term survivors of pediatric cancer. The differential clonal expansion in immune cell populations demonstrates the potential impact of CH mutations on immune cell dysfunction. This suggests that CH mutations may contribute to long-term adverse health outcomes related to the immune system in long-term survivors beyond the direct cause of secondary neoplasms, which will warrant future investigations using the valuable data resource established through this study.


 Sivaraman Natarajan, Nadezhda V. Terekhanova, Kohei Hagiwara, Bensheng Ju, Li Dong, Zhaoming Wang, Jinghui Zhang, John Easton. Mapping therapy-related clonal hematopoiesis across immune cell populations in long-term survivors of pediatric cancer by single-cell multi-omics profiling [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 782.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a161ae578a2ee3c1d5173bcd900fe19406b46227" target='_blank'>
              Abstract 782: Mapping therapy-related clonal hematopoiesis across immune cell populations in long-term survivors of pediatric cancer by single-cell multi-omics profiling
              </a>
            </td>
          <td>
            Sivaraman Natarajan, Nadezhda V. Terekhanova, Kohei Hagiwara, B. Ju, Li Dong, Zhaoming Wang, Jinghui Zhang, John Easton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Neoadjuvant chemoradiotherapy is the stand of care for patients with locally advanced rectal cancer (LARC). For microsatellite-stable (MSS) LARC patients, combining immune checkpoint inhibitors (ICIs) with radiotherapy has shown improved tumor regression, though the mechanisms behind tumor response remain unclear. Here, we analyzed 60 pre-, mid- and post-treatment tumor samples from 39 LARC patients in the TORCH clinical trial, utilizing single-cell sequencing and spatial transcriptomics to explore cell dynamics and their association with therapeutic efficacy. In well responders, combination therapy expanded the CD8+ and CD4+ memory T cell pool, which interacted with type 2 dendritic cells and clustered around non-malignant epithelial cells, forming part of tertiary lymphoid structures, as validated by spatial transcriptomics. In poor responders, the tumor microenvironment was enriched with immunosuppressive cells clustering around cancer-associated fibroblasts. This study uncovers mechanisms underlying the efficacy of radiotherapy combined with ICIs and provides valuable resources for identifying targets in MSS LARC patients.


 Jingwen Wang, Zhen Zhang. Spatiotemporally deciphering the response dynamics of micro-satellite stable rectal cancer to neoadjuvant short-course radiotherapy and PD-1 blockade. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 655.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba50a4897b94e23b9a56ce785c4accfcfef0e23" target='_blank'>
              Abstract 655: Spatiotemporally deciphering the response dynamics of micro-satellite stable rectal cancer to neoadjuvant short-course radiotherapy and PD-1 blockade
              </a>
            </td>
          <td>
            Jingwen Wang, Zhen Zhang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Despite increasing incorporation of gene panel testing into radiotherapeutic decision-making, there is no established method to tailor prescribed radiation regimens to individual tumor radiosensitivity. We propose a novel approach to assessing evolutionary determinants of radioresistance from patient-derived tumor samples. We hypothesized that pre-existing resistance to oxidative damage in tumor microenvironment (TME) can define a cell's evolutionary trajectory and its radiosensitivity. In contrast with the current genetic panels, we use a panel catered toward heterogeneity within the tumor and the TME niches, which may harbor radioresistant populations from collateral TME mechanisms of resistance. The aim is to develop an AI-based algorithm using these eco-evolutionary results to individualize radiation dose in patients, thus minimizing the percentage of non-responders. 3D spheroids of several breast cancer cell lines were grown and collected at multiple time points after 10 Gy of radiation. MALDI data was acquired in negative and positive ionization mode for spatial lipidomics and metabolomics analysis. Additionally, spatial single cell multiplexed immunofluorescence was performed on the sequential cut to MALDI using markers associated with hypoxia, acidosis, ROS, and epithelial-to-mesenchymal transition. Then we grew patient derived organoids (PDO) of breast cancer and irradiated them and analyzed them as described above. Based on a previously validated oxygen diffusion model, spheroids were categorized according to size, with those <240 micrometers in diameter designated as O-spheroids (oxygenated) and those ≥ 240 microns designated as H-spheroids (hypoxic core). While the irradiated O-spheroid growth curves plateaued upon irradiation, H-spheroids grew at the same rate as the non-irradiated controls. To investigate this phenomenon, habitats and cell phenotypes were defined using spatial multi-omics data from MALDI and multiplexed-IF, respectively. We found differential distributions of habitats in radiated spheroids as determined by the cluster size and abundance spatio-temporally. We discovered a switch in the sphingolipid metabolism pathway. Using principles of ecology and evolution, we have developed a novel approach to identifying radiosensitivity markers. Detecting these cell states and ecological niches in patients’ pre-treatment biopsies can inform a more individualized approach to radiotherapy.


 Naheel Khatri, Raafat Chalar, Saiful Samad, Alexander Stessin, Mehdi Damaghi. (R)evolutionary approach to predicting individual response to radiation therapy in breast cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4694.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7188b7f43eeddcafd2d8d2c5bfae450202c1e2b1" target='_blank'>
              Abstract 4694: (R)evolutionary approach to predicting individual response to radiation therapy in breast cancer
              </a>
            </td>
          <td>
            Naheel Khatri, Raafat Chalar, Saiful Samad, Alexander Stessin, Mehdi Damaghi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Aneuploidy, a hallmark of cancer, drives chromosomal instability and antineoplastic drug resistance. Common in hematologic and solid tumors, centrosome amplification leads to supernumerary centrosomes, causing chromosomal missegregation in dividing cancer cells. This imbalance produces progeny with abnormal chromosome counts, leading to cell death, resistance to antineoplastics, and clinically-aggressive tumors. This study examined the effects of CDK2 inhibition on aneuploid lung cancer cells and tumors. It revealed that CDK2 antagonism triggers anaphase catastrophe through centrosome clustering disruption and multipolar mitosis. Unexpectedly, this also leads to the formation of a persistent population of polyploid, apoptosis-resistant cells. Using independent CDK2 inhibitors and shRNAs, we observed that some cancer cells underwent multipolar division and subsequent cell death. Others exhibited cytokinesis failure, resulting in multinucleated, and polyploid cancer cells that continued to survive and proliferate despite continuous CDK2 inhibition. Through time-lapse imaging with transfected FUCCI cell cycle probes and RNA-seq analysis of polyploid versus diploid populations, we identified upregulated CDK1-related pathways and kinesin (KIF) family members in the polyploid cellular population. This likely contributes to the persistent polyploid population, despite CDK2 antagonism. These findings were independently validated in vivo using intravital microscopy and patient-derived xenograft (PDX) lung cancer models. Intriguingly, polyploid and multinucleated cancer cells were found to proliferate post-CDK2 inhibitor treatments. In marked contrast, a polyploid population was not found in CDK2 inhibited primary human alveolar epithelial cells, indicating that this is a pathway that is preferentially activated in aneuploid cancer cells. Analysis of gene expression in TCGA lung cancer datasets linked CDK1 and KIF family overexpression to an unfavorable patient survival, implicating this persistent polyploid population as contributing to poor survival outcomes in lung cancer. These findings are relevant to the clinical development of CDK2 inhibitors. They indicate that despite the pro-apoptotic effects of CDK2 antagonism in aneuploid lung cancer cells, a population of drug-resistant, polyploid cells persists. We propose that a combinatorial therapy targeting CDK2 alongside polo-like kinase 4 (PLK4), CDK1 or kinesin family members would further disrupt centrosome clustering, eliminate polyploid cancer cells, and enhance tumor cell apoptosis and antineoplastic effects. This dual approach would enhance the anti-tumor efficacy of CDK2 inhibition, potentially overcoming resistance mechanisms. This provides a promising strategy to eradicate persistent lung cancer cells following CDK2 inhibition.


 Liliya Tyutyunyk-Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky. CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5327.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fc58e0b6388ab5fc82e889af73612c384df0b6" target='_blank'>
              Abstract 5327: CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Werner syndrome helicase (WRN) is a RecQ helicase family member that plays an important role in genome stability, primarily through its involvement in DNA repair processes such as nonhomologous end-joining and homologous recombination. Upregulation of WRN in mismatch repair (MMR)-deficient or microsatellite instability-high (MSI-H) tumor cells facilitates DNA damage repair and, thus, cell survival. In these cells, WRN inhibition induces double-strand DNA breaks, leading to cell death. Therefore, WRN has emerged as a novel, promising synthetic lethality target in MMR-deficient/MSI-H solid tumors, particularly endometrial and colorectal cancers, which exhibit MMR-deficiency/MSI-H in approximately 20%-30% and 5%-15% of cases, respectively. GSK4418959, a WRNi, is a distinct, oral small molecule that uniquely binds to an allosteric site in the WRN helicase domain, exhibiting synthetic lethality in MMR-deficient/MSI-H tumor cells. It showed potent antitumor activity in in vitro and in vivo preclinical models by selectively inducing DNA damage and, consequently, death of MSI-H cancer cells.



 The SYLVER first-in-human, open-label, multicenter, phase 1/2 trial (NCT06710847) opened on December 9, 2024 and aims to assess safety, tolerability, pharmacokinetics, pharmacodynamics, and preliminary efficacy of GSK4418959, as monotherapy or in combination with a programmed cell death protein-1 (PD-1) inhibitor. The trial comprises WRNi monotherapy dose escalation (part 1), WRNi monotherapy dose expansion (part 2), and dose escalation of WRNi + PD-1 inhibitor (part 3). Key eligibility criteria include age ≥18 years, an advanced MMR-deficient/MSI-H solid tumor, an ECOG PS score of 0-1, and having exhausted standard of care (parts 1 and 3) or received 1-3 lines of systemic therapy, including ≥1 immune checkpoint inhibitor therapy (part 2). Patients who have not recovered from treatment-associated adverse events, have uncontrolled hypertension, or have received prior treatment with a WRNi are ineligible. Primary endpoints are safety during the dose-limiting toxicity observation period (ie, first 21 days; parts 1 and 3) and objective response rate (part 2). Secondary endpoints include overall safety (parts 1-3), pharmacokinetics (parts 1-3), progression-free survival (part 2), and duration of response (part 2). Tumor responses will be assessed per RECIST 1.1. Planned enrollment is approximately 133 patients in the US, Australia, Japan, South Korea, Spain, Belgium and the Netherlands, with potential to expand to additional countries for later stages of the study.



 This study (NCT06710847) is sponsored by GSK.



 NCT06710847
 Third-party medical writing support: Writing and editorial support, funded and coordinated by GSK, was provided by Tafara T.R. Kunota, PhD, and Mary C. Wiggin of Ashfield MedComms, an Inizio company.



 Vivek Samnotra, Natalia Ramos-Hernandez, Thomas Faitg, Meenakshi Srinivasan, Tania Roy, Andrew Liu, Tomasz Zaremba, Clare Holford, Rachel Smith, Jason T. Henry, Jeanne Tie, Noboru Yamamoto, Gul Serbest. First-in-human, phase 1/2 study of GSK4418959, an oral Werner DNA helicase inhibitor, alone or combined with other anticancer agents in adults with advanced mismatch repair-deficient or microsatellite instability-high solid tumors (SYLVER) [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr CT194.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d8faa11ff0e47828029568767559cbff4cf0c6" target='_blank'>
              Abstract CT194: First-in-human, phase 1/2 study of GSK4418959, an oral Werner DNA helicase inhibitor, alone or combined with other anticancer agents in adults with advanced mismatch repair-deficient or microsatellite instability-high solid tumors (SYLVER)
              </a>
            </td>
          <td>
            Vivek Samnotra, Natalia Ramos-Hernandez, Thomas Faitg, Meenakshi Srinivasan, Tania Roy, Andrew Liu, Tomasz Zaremba, Clare Holford, Rachel Smith, Jason T Henry, Jeanne Tie, N. Yamamoto, Gul Y. Serbest
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e13538


 Background:
 Understanding the molecular profile of cancer-associated genes is crucial in oncology, as it informs targeted therapies and improves patient outcomes. Tumor suppressor genes inhibit the activation of oncogenic signals, while oncogenes promote them. Ras proteins, encoded by KRAS, NRAS, and HRAS genes, are involved in cell growth, differentiation, and survival pathways. This study aims to investigate the frequency of KRAS, NRAS, and HRAS mutations in solid tumors treated at a single academic institution between January 2017 and May 2024. By analyzing these mutations, we seek to enhance our understanding of their distribution across different cancer types in Venezuela, providing valuable insights for personalized cancer treatment strategies.
 Methods:
 We conducted a retrospective, cross-sectional, analytical cohort study. The study included 175 patients, with 179 molecular genetic sequencing studies and 20 RT-PCR analyses. We examined mutations in codons 12, 13, 59, 61, 117, and 146 of the KRAS, NRAS, and HRAS genes.
 Results:
 We identified 43 mutations in the RAS gene family: 37 in KRAS, 5 in NRAS, and 1 in HRAS. Lung cancer accounted for 14 of those mutations, with 13 in KRAS and 1 in HRAS. Colorectal cancer showed 19 mutations, with 16 in KRAS and 3 in NRAS. Other KRAS mutations (13.9%) were found in breast, endometrial, pancreatic cancers, and thymoma. We also recorded 2 NRAS mutations in melanoma and 1 in pancreatic cancer.
 Conclusions:
 This study represents the largest collection of KRAS, NRAS, and HRAS mutations reported in Venezuela.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d60941cc22a3a39b9e5850c1cc1b772e65a8cda" target='_blank'>
              Frequency of RAS family gene mutations in metastatic solid tumors in Venezuela: Experience at a large academic center.
              </a>
            </td>
          <td>
            Guillermo Borga, Oscar A. Sucre Astorga, Santiago Sucre, C. Sucre
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Cancer immunotherapy is a promising treatment for many forms of cancer, however only a small subset of patients benefit, and relapse/recurrence frequently occurs due to various resistance mechanisms. To overcome this challenge combinatorial therapeutic strategies are needed targeting multiple steps of the cancer-immunity cycle to achieve long-term efficacy in a large population of cancer patients. It is now known that DNA damage activates the innate immune response and improves immune checkpoint blockade (ICB) efficacy. Combination immunotherapy can target several immunosuppressive elements in the tumor microenvironment and activate multiple steps of the cancer-immunity cycle. For developing an effective combination treatment strategy, including ICB with various other therapeutic modalities, we need to first understand the diverse expression pattern of various factors within tumor immune microenvironment. Multiplex immunofluorescence (mIF) is an important tool for profiling protein expression patterns in tumor tissues. This can provide a framework to understand the sensitivity to a particular combination of therapies and identify potential predictive biomarkers. Here we show spatial proteomics profiling of a diverse range of biomarkers in human colon adenocarcinoma tissue. Our findings demonstrate that spatial proteomics can be tailored to precision oncology using artificial intelligence to predict a patient’s response to combination treatment strategies. The Cell DIVE Multiplex Imaging Solution in combination with AI-based image processing software, AIVIA delivers a comprehensive solution for exploring immune landscape of tumor tissue. The broad portfolio of robust IHC- validated antibodies from Cell Signaling Technology (CST) enables the detection of key proteins in the tumor microenvironment (TME). Here, we demonstrate multiplexed Cell DIVE imaging using a novel CST panel to probe human colon adenocarcinoma. The availability of cell type specific biomarkers, combined with the ability to interrogate using multiplexed tissue imaging, provides unprecedented and novel insights and spatial resolution of diverse cell populations in the TME.


 Arindam Bose, Susan Keezer, Emily Quann Alonzo, Sophie Struble, Jeremy Fisher, Matthew Norton, Richard A. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal. Spatial mapping of the tumor microenvironment for combination therapy design [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6464.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258df61f2a4c963aeb2c62a2f4585ccce0f62b1b" target='_blank'>
              Abstract 6464: Spatial mapping of the tumor microenvironment for combination therapy design
              </a>
            </td>
          <td>
            Arindam Bose, Susan Keezer, E. Alonzo, Sophie Struble, Jeremy Fisher, Matthew Norton, R. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Constitutional mismatch repair deficiency (CMMRD), caused by bi-allelic germline variants in mismatch repair (MMR) genes, is associated with high cancer incidence early in life. A better understanding of mutational processes driving sequential CMMRD tumors can advance optimal treatment. Here, we describe a genomic characterization on a representative collection of CMMRD-associated tumors consisting of 41 tumors from 17 individuals. Mutational patterns in these tumors appear to be influenced by multiple factors, including the affected MMR gene and tumor type. Somatic polymerase proofreading mutations, commonly present in brain tumors, are also found in a T-cell lymphoblastic lymphoma displaying associated mutational patterns. We show prominent mutational patterns in two second primary hematological malignancies after temozolomide treatment. Furthermore, an indel signature, characterized by one-base pair cytosine insertions in cytosine homopolymers, is found in 54% of tumors. In conclusion, analysis of sequential CMMRD tumors reveals diverse mutational patterns influenced by the affected MMR gene, tumor type and treatment history.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c61f9eb1036d6b9dbdcb4d99fcc534a6fcd3f8" target='_blank'>
              Unraveling mutagenic processes influencing the tumor mutational patterns of individuals with constitutional mismatch repair deficiency
              </a>
            </td>
          <td>
            Dilys D Weijers, Snežana Hinić, E. Kroeze, M. Gorris, G. Schreibelt, Sjors Middelkamp, Arjen R Mensenkamp, R. Bladergroen, K. Verrijp, N. Hoogerbrugge, Pieter Wesseling, Rachel S van der Post, Jan L C Loeffen, C. Gidding, M. V. van Kouwen, I. J. D. de Vries, R. van Boxtel, R. D. de Voer, M. Jongmans, Roland P Kuiper
          </td>
          <td>2025-05-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Human exonuclease 1 (EXO1), a member of the structure-specific nuclease family, plays a critical role in maintaining genome stability by processing DNA double-strand breaks (DSBs), nicks, and replication intermediates during DNA replication and repair. As its exonuclease activity is essential for homologous recombination (HR) and replication fork processing, EXO1 has emerged as a compelling therapeutic target, especially in cancers marked by heightened DNA damage and replication stress. Through high-throughput screening of 45,000 compounds, we identified seven distinct chemical scaffolds that demonstrated effective and selective inhibition of EXO1. Representative compounds from two of the most potent scaffolds, C200 and F684, underwent a comprehensive docking analysis and subsequent site-directed mutagenesis studies to evaluate their binding mechanisms. Biochemical assays further validated their potent and selective inhibition of the EXO1 nuclease activity. Tumor cell profiling experiments revealed that these inhibitors exploit synthetic lethality in BRCA1-deficient cells, emphasizing their specificity and therapeutic potential for targeting genetically HR-deficient (HRD) cancers driven by deleterious mutations in HR genes like BRCA1/2. Mechanistically, EXO1 inhibition suppressed DNA end resection, stimulated the accumulation of DNA double-strand breaks, and triggered S-phase PARylation, effectively disrupting DNA repair pathways that are essential for cancer cell survival. These findings establish EXO1 inhibitors as promising candidates for the treatment of HRD cancers and lay the groundwork for the further optimization and development of these compounds as targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bbdf0d10f263cc553f41d2d920cc6308756913" target='_blank'>
              Discovery and Characterization of Small Molecule Inhibitors Targeting Exonuclease 1 for Homologous Recombination-Deficient Cancer Therapy.
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Chen Wang, Lingzi Ma, Megan Luo, J. Jossart, John J. Perry, David Kwon, Zhe Wang, Xinyu Pei, Changxian Shen, Yingying Wang, Mian Zhou, Holly Yin, David Horne, A. Nussenzweig, Li Zheng, Binghui Shen
          </td>
          <td>2025-05-16</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Estrogen receptor-positive breast cancer frequently develops resistance to tamoxifen in one third of patients, driven by various mechanisms. Our work has linked 3D chromatin alterations to this resistance. However, most of the work in 3D chromatin regulation was mostly conducted in in vitro cell culture system. It is becoming very clear that to expand the characterization of 3D genome organization in patient samples is an emerging task.



 We hypothesize that 3D chromatin alterations, including compartment switching, TAD reorganization, and dysregulated chromatin looping, drive tamoxifen resistance (TR) in breast cancer. Looping-mediated genes may play critical roles in resistance and serve as potential therapeutic targets.



 To investigate the role of 3D chromatin changes in tamoxifen resistance, we performed Hi-C profiling on normal tissues (NTs), primary tumors (PTs), and tamoxifen-treated recurrent tumors (RTs), uncovering significant chromatin alterations. These included 46.8% A-to-B and 32.8% B-to-A compartment switching from NTs to PTs, and 58.3% B-to-A flipping during PT-to-RT progression. TAD analysis identified RT-specific Neo and Fuse alterations as well as tumor-specific chromatin loops. CA2 was identified as a enhancer-promoter looping (EPL)-mediated gene with its higher expression linked to poor relapse-free survival in endocrine-treated patients. Functional studies using the CA2 inhibitor brinzolamide demonstrated its ability to reduce cell viability, loop intensity, and CA2 expression in MCF7TR and T47DTR cells. In vivo, brinzolamide significantly reduced tumor growth in tamoxifen-resistant cell-derived xenograft (CDX) models. Additionally, CRISPR deletion of CA2 EPL enhancer region in MCF7TR cells resulted in a marked reduction in cellular viability, gene expression and protein expression.



 Our findings suggest CA2 as a key looping-mediated gene play an oncogenic role in driving tamoxifen resistance in breast cancer. Our study yields significant mechanistic insights into the role and clinical relevance of 3D chromatin architecture in tamoxifen resistant breast cancer.



 Lavanya Choppavarapu, Kun Fang, Tianxiang Liu, Aigbe G. Ohihoin, Victor X. Jin. The role of ca2 in regulating 3d chromatin structure in tamoxifen resistant breast cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2735.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb67a4446cf1d38e1dfaa6a69287fa69917e1c7" target='_blank'>
              Abstract 2735: The role of ca2 in regulating 3d chromatin structure in tamoxifen resistant breast cancer
              </a>
            </td>
          <td>
            Lavanya Choppavarapu, Kun Fang, Tianxiang Liu, A. Ohihoin, Victor X. Jin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genome maintenance is of the utmost importance in stem cells, as mutations can be propagated and cause defects in derivative tissues. Many stem cell types display low mutation rates, with embryonic stem cells (ESCs) being a notable example. The bases for this property are unclear but may be achieved by optimization of various processes including high-fidelity DNA repair, cell cycle checkpoint controls, and hypersensitivity to genotoxic insults that trigger cell death. Here, we investigate the mechanisms underlying the unique responses of mouse ESCs (mESCs) to replication stress (RS) using an array of small molecule inhibitors and genotoxins. We find that whereas mESCs survive under acute RS in an ATR- and CHK1-dependent manner similar to somatic cells, they lack a strong G2/M checkpoint and fail to repair DNA crosslinks in the absence of ATR signaling. Despite the lack of a strong G2/M checkpoint, mESCs maintain a spindle assembly checkpoint (SAC). We posit that mESCs preferentially repair DNA crosslinks in S phase via homology-directed mechanisms, and cells that fail to complete repair before mitosis undergo mitotic catastrophe and cell death. These findings shed light on mutation avoidance mechanisms in ESCs that may extend to other stem cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca127bc92b2f97069c67210cae368db947e036c" target='_blank'>
              Embryonic Stem Cell-Specific Responses to DNA Replication Stress
              </a>
            </td>
          <td>
            Ryan C. James, Jerry K. Wang, Siddhanth R. Bhatt, Sophie E. Shadid, Daryl J. Phuong, J. Schimenti
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Tumor suppressor genes are frequently targeted by mutations introducing premature termination codons (PTC) in the protein coding sequence, both in sporadic cancers and in the germline of patients with cancer predisposition syndromes. These mutations have a high pathogenic impact since they generate C‐terminal truncated proteins with altered stability and function. In addition, PTC mutations trigger transcript degradation by nonsense‐mediated mRNA decay. Suppression of PTC by translational readthrough restores protein biosynthesis and stabilizes the PTC‐targeted mRNA, making a suitable therapeutic approach the reconstitution of active full‐length tumor suppressor proteins by pharmacologically‐induced translational readthrough. Here, we review the recent advances in small molecule pharmacological induction of translational readthrough of disease‐associated PTC from tumor suppressor genes, and discuss the therapeutic potential of translational readthrough in specific groups of patients with hereditary syndromic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c0c11615ee99ac8075d00fb3fd428aa4a1973d7" target='_blank'>
              Therapeutic Potential of Translational Readthrough at Disease‐Associated Premature Termination Codons From Tumor Suppressor Genes
              </a>
            </td>
          <td>
            Leire Torices, C. Nunes-Xavier, R. Pulido
          </td>
          <td>2025-05-01</td>
          <td>Iubmb Life</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Blood cancers are one of the most common cancers worldwide. These diseases stem from defects in blood components having cytogenetic aberrations and genetic mutations. There have been vast improvements in terms of treatment options and survival outcomes. Nevertheless, due to the clonal nature and heterogeneity of the diseases, the number of cases reported exhibit a rising pattern due to chemoresistance and disease relapse thus posing a healthcare burden. Therefore, the need for more specific forms of targeted therapies is ever-present.


RECENT FINDINGS
CRISPR has emerged as a key player and is the epitome of gene editing technology in this post genomic era. In line with the current trend, numerous studies in blood cancer research have extensively utilized CRISPR-based applications to understand the functional genomics of hematologic malignancies and identify potential therapeutic targets for development of novel therapeutic applications.


SUMMARY
The importance of comprehending the utilities of state-of-the-art technologies such as CRISPR for studying hematologic malignancies has never been more apparent and timelier. Therefore, this review attempts to scrutinize the versatility of CRISPR applications which range from functional genomics to immunotherapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215dcf4713fb2244aa818902ca452469fccb308c" target='_blank'>
              Multifaceted roles of CRISPR technology in blood cancer research.
              </a>
            </td>
          <td>
            Maheswaran Solayappan, Adam Azlan, Kang Zi Khor, Mot Yee Yik, Aswani Jaishanker, Thevendran Ramesh, Mohamed Saleem, N. Yusoff, E. Moses
          </td>
          <td>2025-06-04</td>
          <td>Current opinion in hematology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The relentless pursuit of understanding and combating glioblastoma (GBM), one of the most formidable foes in the realm of cancer, requires a deeper exploration of its intricate dynamics. Gliomas, particularly GBM, are known for their lethal nature, and a significant aspect of their pathogenesis lies in their ability to manipulate the blood vessels that sustain them. This complex relationship is governed by a multitude of molecular mechanisms involving a diverse array of cell types within the tumor microenvironment. Central to this intricate web of regulation are non-coding RNAs (ncRNAs), enigmatic molecules that have recently emerged as key players in cancer biology. These ncRNAs wield a remarkable influence on gene expression, often via epigenetic modifications and intricate control over angiogenesis-related molecules. Their role in GBM angiogenesis adds another layer of complexity to our understanding of this disease. In the realm of cancer therapeutics, targeting angiogenesis has become a prominent strategy. However, the efficacy of current antiangiogenic treatments against GBM is often transient, as these tumors can rapidly develop resistance, becoming even more aggressive. GBM employs a diverse set of strategies to foster its abnormal vasculature, which, in turn, holds the key to understanding why anti-angiogenic therapies often fall short of expectations. This review aims to shed light on potential strategies and novel perspectives to overcome GBM 's resistance to anti-angiogenic therapy. By exploring innovative approaches, including those centered on ncRNAs, we strive to chart a course toward more effective treatments. This journey into the depths of GBM 's complexities offers not only hope but also a blueprint for future research and therapeutic development. As we uncover the intricate mechanisms at play, we inch closer to the day when GBM is no longer an insurmountable adversary in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23d8a8688a2f6cfce6a37bfe094112f447e2a5cf" target='_blank'>
              Non-coding RNAs-based Therapy and Angiogenesis: A New Era for the Management of Gliomas.
              </a>
            </td>
          <td>
            O. Beylerli, I. Gareev, Elmar Musaev, Tatiana Ilyasova, Sergey Roumiantsev, Vladimir Chekhonin
          </td>
          <td>2025-04-11</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="This study analyzed the genetics of classical Hodgkin lymphoma (cHL) by using circulating tumor DNA (ctDNA). Two genetic subtypes were identified, differing in genetic instability mechanisms: one subtype (64% of cases) showed a higher mutation load and a higher fraction of mutations associated with activation-induced cytidine deaminase and microsatellite instability signatures, while the other subtype (36% of cases) exhibited chromosomal instability with more somatic copy number alterations. Whole-genome duplication was more common in cHL compared to other B-cell tumors and emerged as a prognostic biomarker for patients undergoing ABVD-based therapy. Non-coding regulatory mutations, similar to those in diffuse large B-cell lymphoma, were highly prevalent in 86% of cHL. A recurrent somatic expression quantitative trait locus (seQTL) involving the BCL6 gene was found in 30% of cases. The seQTL of BCL6 aligned with accessible chromatin and increased H3K27 acetylation in cHL, disrupted PRDM1 binding, and co-occurred with BCL6 expression in cHL cells. Weak to strong expression of BCL6 was observed in 68% of cases and BCL6 expression associated with gene repression similarly in cHL and germinal center B cells. After BCL6 degradation, the core set of genes directly bound and regulated by BCL6 was derepressed in cHL and proliferation was impaired. The number and clonality of neoantigens was associated with tumor microenvironment type and response to checkpoint blockade. Finally, ctDNA analysis was suggested as a tool to distinguish ambiguous PET/CT-positive lesions post-treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc33ff26bbe2b930345064a0a4545ff56f194772" target='_blank'>
              A Comprehensive Genetic Study of Classical Hodgkin Lymphoma Using Circulating Tumor DNA.
              </a>
            </td>
          <td>
            M. Pirosa, A. Bruscaggin, L. Terzi di Bergamo, M. Salehi, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, G. A. Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Ilaria Romano, S. Annunziata, F. Bergesio, R. Boldorini, E. Borsatti, P. Bulian, Eleonora Calabretta, S. Chauvie, Francesco Corrado, S. Crisci, M. Cuzzocrea, R. De Filippi, Bernhard Gerber, M. Kurlapski, L. M. Larocca, Elisabetta Merlo, A. Rinaldi, M. Rodari, G. Romanowicz, G. Sacchetti, A. Stathis, G. Stussi, I. Zangrilli, A. Pinto, L. Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, A. Tzankov, Carmelo Carlo-Stella, G. Gaidano, L. Ceriani, Emanuele Zucca, Davide Rossi
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 Ewing’s sarcoma (ES) is a highly aggressive cancer that primarily originates in the bones or soft tissues and is predominantly diagnosed in adolescents and young adults. The EWS-FLI1 fusion protein is considered a hallmark of ES, being expressed in approximately 85% of cases. Prognosis for advanced ES remains poor, with survival rates around 30%, and current chemotherapy regimens are associated with severe long-term side effects. Mithramycin has emerged as a promising inhibitor of EWS-FLI1; however, its clinical application is limited by dose-dependent toxicities. We hypothesize that combining mithramycin with standard chemotherapy could enhance therapeutic outcomes through synergistic effects, potentially allowing for reduced doses of both agents to maximize efficacy while minimizing toxicity. To investigate this, we evaluated the combined impact of mithramycin and etoposide on ES cell lines. Using TC205 cells, we further explored the underlying mechanisms through proteomics and integrative pathway analysis (IPA). In this study, we assessed the dose- and time-dependent effects of mithramycin and etoposide in ES cell lines (CHLA10 and TC205) and non-cancerous cardiomyocytes (H9C2) to optimize treatment conditions. In vivo assays demonstrated a significant reduction in tumor growth with combination therapy compared to single-agent treatment. Using TC205 cells, we assessed the impact of mithramycin and etoposide both as monotherapies and in combination. Cells were plated in triplicate for each group (control, mithramycin, etoposide, and mithramycin + etoposide) and treated with optimized doses. After 48 hours, cells were harvested, and protein samples were subjected to proteomics and IPA to elucidate the mechanisms of action of both mono- and combination therapies. The combination of mithramycin and etoposide significantly inhibited cell viability compared to individual treatments. Proteomics analysis identified 517 altered proteins (t-test with BH correction), with 142 upregulated and 375 downregulated. IPA revealed significant changes in markers associated with disease and physiological function networks, including homologous recombination, cytostasis of sarcoma cell lines, cell movement of sarcoma cell lines, and osteoclastogenesis of limb bones. More than 70 disease and function networks and over 200 canonical pathways were significantly affected. Key pathways included eukaryotic translation initiation, cell cycle checkpoints, neutrophil degranulation, and cell cycle control of chromosomal replication. Overall, these findings reveal that the combination of mithramycin and etoposide alters critical networks and pathways, enhancing therapeutic efficacy in ES cell lines.


 Christoffer B. Lambring, Khadiza Zaman, Elin Stone, Natalie Gierat, Ameya Bhargava, Laszlo Prokai, Riyaz Basha. Proteomics and integrative pathway analysis to understand the mechanism of action of mithramycin and etoposide combination induced cell death in Ewing sarcoma cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7053.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca4cdf73484b348317c78d3c6466b5fa0ebab2b" target='_blank'>
              Abstract 7053: Proteomics and integrative pathway analysis to understand the mechanism of action of mithramycin and etoposide combination induced cell death in Ewing sarcoma cells
              </a>
            </td>
          <td>
            C. Lambring, K. Zaman, Elin Stone, Natalie Gierat, Ameya Bhargava, Laszlo Prokai, Riyaz Basha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>