<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-09 10:10:47 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>93</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Mutations in epigenetic regulators are common in bladder cancer, yet their impact on therapeutic responses remains unclear. Here, we identify that loss-of-function mutations in KDM6A, a histone demethylase altered in about 26% of advanced bladder cancers, are associated with poor survival after cisplatin chemotherapy, whereas they correlate with improved outcomes with anti-PD-1 therapy. Using CRISPR-Cas9-engineered murine and human bladder cancer models, we show that KDM6A deficiency increases formation of extrachromosomal circular DNA carrying chemoresistance loci, promoting cisplatin resistance. In parallel, KDM6A loss impairs DNA repair and rewires tumor metabolism, reducing glycolysis and lactate output. This metabolic shift diminishes histone lactylation in regulatory T cells, suppressing immunoregulatory genes and limiting expansion of PD-1hi regulatory T cells. Collectively, our findings establish KDM6A mutation as a key regulator of therapeutic responses, providing a foundation for its use in guiding precision therapy in advanced bladder cancer. Genes encoding key epigenetic regulators, including Lysine Demethylase 6A (KDM6A), are frequently mutated in bladder cancer. Here, the authors show that loss of KDM6A promotes formation of extrachromosomal circular DNA (eccDNA), genomic instability, and metabolic reprogramming, driving resistance to cisplatin chemotherapy while simultaneously enhancing sensitivity to immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b55c41a895a4562f500627e7c41a6885170b27" target='_blank'>
              Clonal morphology-guided combination therapies overcome heterogeneity-driven drug tolerance
              </a>
            </td>
          <td>
            Xiaoxi Li, Ling Liu, Yuanyuan Yan, Mengfang Yang, Lingli Luo, Yong Jiang, Weijun Ren
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Molecular targeted therapies targeting KRAS signaling have significantly improved patient outcomes, but they have not achieved sufficient therapeutic efficacy in colorectal cancer (CRC). Here, we demonstrate that a subset of KRAS-mutant CRC cells transitions to a cellular state characterized by enhanced ribosome biogenesis upon KRAS signaling inhibition. The mitogen-activated protein kinase kinase inhibitor, trametinib, and AMG510 induce a cellular state characterized by a gene expression profile highly enriched for ribosome biogenesis. We find that they are vulnerable to the inhibition of RNA polymerase I, and they exhibit synergistic anti-tumor effects with trametinib in an autochthonous mouse model of intestinal tumors and human patient-derived organoids (PDOs). These observations demonstrate that high ribosome biogenesis induced by KRAS inhibition is indispensable to maintain this cellular state and is a potential therapeutic target. Overall, this study reveals novel mechanisms of drug tolerance to KRAS inhibition, thereby facilitating the development of new therapeutic strategies. Drug tolerance to KRAS inhibition remains a major challenge in cancer therapy. Here the authors find that KRAS inhibition induces a cellular state characterized by enhanced ribosome biogenesis in KRAS-mutant colorectal cancer cells and these cells are vulnerable to inhibition of RNA polymerase I.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b7de6d8aeceaad9c8d57eb1d9c3d308c5a80e2" target='_blank'>
              Ribosome biogenesis as a potential therapeutic target in KRAS mutant colorectal cancer
              </a>
            </td>
          <td>
            Yui Tanaka, M. Sakahara, H. Yamanaka, Yasuko Natsume, Daisuke Kusama, Kohei Kumegawa, Harunori Yoshikawa, Yuich Abe, K. Okabayashi, Shimpei Matui, Y. Kitagawa, Naohiko Koshikawa, H. Osumi, E. Shinozaki, Satoshi Nagayama, Jun Adachi, R. Maruyama, Ryoji Yao
          </td>
          <td>2025-12-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d7c38058905c3f246edc0fb45f9435b0439528" target='_blank'>
              Efflux pump regulation and metabolic rewiring define the ceiling of efflux-mediated drug resistance in Mycobacteria
              </a>
            </td>
          <td>
            Shiksha Sharma, Sarika Mehra
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce1dd81fa69ca7596c6e2f6203643899d73e28c1" target='_blank'>
              In vivo isogenic modelling unveils a TP53-mediated relapse phenotype in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Stéphanie Gachet, S. Quentin, L. Hernandez, Loïc Maillard, M. Passet, R. Kim, Hugo Bergugnat, Melha Benlebna, Maxime Boy, Véronique Parietti, P. Fenaux, André Baruchel, Hervé Dombret, N. Boissel, François Sigaux, Hugues de Thé, E. Clappier, J. Soulier
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Oncogenic KRAS mutations drive metabolic reprogramming in pancreatic ductal adenocarcinoma (PDAC). Src-homology 2 domain-containing phosphatase 2 (SHP2) is essential for full KRAS activity, and promising dual SHP2/mitogen-activated protein kinase (MAPK) inhibition is currently being tested in clinical trials. Exploitable metabolic adaptations may contribute to invariably evolving resistance. To understand the metabolic changes induced by dual inhibition, we comprehensively tested human and murine PDAC cell lines, endogenous tumor models, and patient-derived organoids, which are representative of the full spectrum of PDAC molecular subtypes. We found that dual SHP2/mitogen-activated protein kinase kinase (MEK1/2) inhibition induces major alterations in mitochondrial mass and function, impacts reactive oxygen species (ROS) homeostasis and triggers lipid peroxidase dependency. Anabolic pathways, autophagy and glycolysis were also profoundly altered. However, most strikingly, mitochondrial remodeling was evident, persisting into a therapy-resistant state. The resulting vulnerability to the induction of ferroptotic cell death via the combination of vertical SHP2/MEK1/2 with glutathione peroxidase (GPX4) inhibition was largely independent of the PDAC molecular subtype and was confirmed with direct targeting of RAS. The triple combination of SHP2/MEK1/2 inhibition and the ferroptosis-inducing natural compound withaferin A suppressed tumor progression in an endogenous PDAC tumor model in vivo. Our study offers a metabolic leverage point to reinforce RAS pathway interference for targeted PDAC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ece64da9d6dcfbff96e0c83595844cf2b2f5547" target='_blank'>
              Vertical RAS pathway inhibition in pancreatic cancer drives therapeutically exploitable mitochondrial alterations
              </a>
            </td>
          <td>
            P. Hafner, S. Keller, Xun Chen, Asma Alrawashdeh, H. Jumaa, Friederike I. Nollmann, S. Besson, Judith Kemming, Oliver Gorka, T. Das, Bismark Appiah, Ariane Lehmann, M. Li, Petya Apostolova, Bertram Bengsch, R. Zeiser, S. Tholen, Oliver Schilling, Olaf Groß, Andreas Vlachos, U. Wittel, Dominik von Elverfeldt, W. Reichardt, M. Boerries, G. Andrieux, Guus Heynen, S. Fichtner-Feigl, Luciana Hannibal, D. Ruess
          </td>
          <td>2026-01-16</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e404655f34f2f760a22785dfe7969af3c340a1ca" target='_blank'>
              Genome-wide CRISPR screens identified C18orf32 as a novel regulator of lipid metabolism that mediates PFOA-induced toxicity
              </a>
            </td>
          <td>
            Chanhee Kim, A. Tagmount, Zhaohan Zhu, W. B. Barbazuk, Rhonda L. Bacher, Christopher D. Vulpe
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, Darya A Yurchenko, Zhanna G Markova, Tatiana V Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT High-quality reference genomes are essential for comparative genomics and accurate genotype-phenotype mapping. Here, we corrected the Mycobacterium tuberculosis Erdman strain reference genome (ErdmanTI) using ultra-deep HiFi sequencing. Among the small variants (n = 275) between ErdmanTI and the current Erdman reference NC_020559.1 (ErdmanSTJ), numerous are likely errors in ErdmanSTJ. We identified a novel bias toward in-frame structural variations (SVs) in pe/ppe genes and 28 SVs between ErdmanTI and ErdmanSTJ, half representing likely errors in ErdmanSTJ. Other SVs were consistent with in vitro evolution, including copy number variation (CNV) of promoter tandem repeats (PTRs). PTR CNVs were polyphyletic and within isogenic populations (10−2–10−3 CNVs/chromosome), demonstrating the impact of phase-variable CNV across evolutionary timescales. These hypervariable PTRs pinpoint a genomic basis for rapidly switching nitric oxide resistance (Dop), biofilm formation (LpdA), drug tolerance (EfpA), and glycerol utilization (GlpD2) phenotypes. This work uncovers a common phase variation mechanism obscured by short-read sequencing limitations and provides an improved reference for comparative studies. IMPORTANCE Mycobacterium tuberculosis (Mtb), the pathogen responsible for tuberculosis, is often described as genetically stable. Our findings reveal an overlooked evolutionary adaptation mechanism: phase variation driven by tandem repeat copy number changes in gene promoters. Enabled by ultra-deep, long-read sequencing, we corrected errors in the Erdman reference genome and uncovered frequent, spontaneous expansions and contractions of promoter repeats upstream of genes linked to nitric oxide resistance, drug efflux, and biofilm formation. Through altering promoter strength, these dynamic promoter variants may generate phenotypic diversity within subpopulations and across diverse clinical lineages, suggesting a conserved evolutionary advantage for navigating host-imposed stress. This reframes Mtb’s evolutionary potential, highlighting how adaptive flexibility has been underestimated due to reliance on short-read sequencing and limited resolution of subpopulations at standard genomic depths. Our findings underscore the need to integrate structural variation-aware approaches into studies of Mtb pathogenesis, evolution, and drug response. Mycobacterium tuberculosis (Mtb), the pathogen responsible for tuberculosis, is often described as genetically stable. Our findings reveal an overlooked evolutionary adaptation mechanism: phase variation driven by tandem repeat copy number changes in gene promoters. Enabled by ultra-deep, long-read sequencing, we corrected errors in the Erdman reference genome and uncovered frequent, spontaneous expansions and contractions of promoter repeats upstream of genes linked to nitric oxide resistance, drug efflux, and biofilm formation. Through altering promoter strength, these dynamic promoter variants may generate phenotypic diversity within subpopulations and across diverse clinical lineages, suggesting a conserved evolutionary advantage for navigating host-imposed stress. This reframes Mtb’s evolutionary potential, highlighting how adaptive flexibility has been underestimated due to reliance on short-read sequencing and limited resolution of subpopulations at standard genomic depths. Our findings underscore the need to integrate structural variation-aware approaches into studies of Mtb pathogenesis, evolution, and drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b608d990d2aa84315a169548ac17ba8eb0b7b1e2" target='_blank'>
              Updated Erdman reveals tandem repeat copy number is phase-variable and impacts M. tuberculosis adaptation across evolutionary timescales
              </a>
            </td>
          <td>
            Samuel J. Modlin, Nachiket Thosar, Paulina M. Mejía-Ponce, Raegan L. Lunceford, Gaëlle Guiewi Makafe, Brian Weinrick, F. Valafar
          </td>
          <td>2025-12-17</td>
          <td>mSystems</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite advances in cancer therapies, treatment failure from resistance and recurrence remains a major clinical challenge. Therapy-induced senescence (TIS), a state of stable cell cycle arrest with sustained metabolic activity, has emerged as a driver of inflammation, tumor persistence, and relapse. However, the heterogeneity of TIS complicates its detection and targeting. Here, we applied a multi-modal strategy to characterize metabolic alterations in senescent cancer cells induced by doxorubicin or γ-irradiation across three tumor cell lines: MCF7, HeLa, and TPC-1. Mitochondrial dysfunction was assessed using MitoTracker and JC-1 staining, while two-photon excitation fluorescence (TPEF) microscopy enabled label-free visualization of intracellular NAD(P)H and FAD distribution. Lipid remodeling was evaluated by MALDI mass spectrometry imaging, and RNA sequencing was performed on control, senescent, and engulfing-senescent MCF7 cells to identify differentially expressed genes and enriched pathways. Senescent cells displayed mitochondrial dysfunction, with altered NAD(P)H/FAD distribution and decreased membrane potential. TPEF confirmed redistribution of coenzymes, reflecting redox changes. Lipidomics revealed consistent remodeling, notably involving cardiolipin precursors. Transcriptomic profiling showed engulfing-senescent MCF7 cells possess a distinct signature marked by increased lipid metabolism, endocrine signaling, and cell–cell communication. Overall, our findings reveal conserved and cell type–specific metabolic traits of TIS, highlighting metabolic vulnerabilities for senolytic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Immune aging is being increasingly recognized as a critical barrier to effective cancer immunotherapy, as the aged tumor microenvironment (TME) drives T cell dysfunction and impairs immune control of cancer1–4. However, the key molecular drivers of this process as well as potential targets to rescue T cell dysfunction in aged tumors remain incompletely understood. Therefore, we performed in vivo single-cell CRISPR screens in CD8⁺ T cells within aged tumors and tumor-draining lymph nodes (tdLNs). We identified Dusp5 and Zfp219 as key regulators of T cell persistence and effector differentiation in aged hosts. Loss of Dusp5, a negative regulator of ERK signaling, increased ERK1/2 phosphorylation and enhanced T cell proliferation in both young and aged tumors. In contrast, loss of Zfp219, a transcriptional repressor, induced epigenetic reprogramming of cytotoxic gene programs, thereby increasing granzyme secretion and enhancing antitumor immunity. Moreover, expression of the human ortholog gene ZNF219 is increased within intratumoral CD8⁺ T cells in older cancer patients. High ZNF219 expression correlates with poorer survival following immune checkpoint blockade (ICB) and reduces persistence of human intratumoral T cells. Notably, Zfp219 ablation synergized with anti-PD-1 blockade in mice to expand effector-like CD8⁺ T cells, leading to significantly enhanced anti-tumor immunity and tumor clearance in aged hosts. Together, these findings highlight Dusp5 and Zfp219 as critical drivers of age-related T cell dysfunction and as potential therapeutic targets to rejuvenate T cell antitumor immunity in older cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017b5a7413706a4c2c7fbdc538a3a98539127c42" target='_blank'>
              CRISPR screens identify targets to rescue age-related T cell dysfunction in cancer
              </a>
            </td>
          <td>
            Alex C. Y. Chen, Keely Ji, Cansu Yerinde, Nelson H. Knudsen, Kevin Bi, Daniela Martinez, Thomas J Carmona-LaSalle, Kazuhiro Taguchi, Katherine H Xu, Elizabeth M. Seider, Marc A. Schwartz, Maria Zschummel, Linda T. Nieman, Kathleen B. Yates, T. R. Mempel, R. Manguso, N. Hacohen, Debattama R. Sen
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Oncogene amplifications fuel some of the most lethal, therapy‑refractory cancers, yet remain clinically untargeted. We report a single‑guide CRISPR/Cas9 strategy that converts the sheer copy‑number excess of oncogene amplicons into an Achilles’ heel. A solitary intronic double‑strand break is innocuous in diploid genomes but collapses oncogene amplification‑positive cells across neuroblastoma, small‑cell lung and colorectal carcinoma models, driving > 90% loss of viability, G₂/M blockade and catastrophic DNA‑damage signalling. Amplified‑locus cleavage rewires transcription toward cell death activation, necroptosis and cGAS–STING–mediated immunogenic cell death, enabling dendritic‑cell cross‑priming and T‑cell activation and proliferation. In xenografts, delivery of the intronic sgRNA shrinks tumours by 90%, prolongs survival and remodels the innate tumour microenvironment. Deep sequencing confirms negligible off‑target editing, and combination with doxorubicin achieves supra‑additive killing. These findings establish amplification density, not sequence content, as a tractable, tumour‑exclusive target and unveil a dual‑action platform that is simultaneously cytotoxic and immunostimulatory. Editing of tumor amplifications therefore offers a blueprint for translating copy‑number aberrations into precision genome‑editing therapies for treatment‑resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V Semenov, Irina S Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5ef1091f544d26b38d75e4b4f01629d3fcc603" target='_blank'>
              VARS1 fuels prostate cancer evolution via codon-selective translational rewiring
              </a>
            </td>
          <td>
            Qinju He, Rongrong He, Yingying Li, Miaomiao Xu, Shuchao Ren, C. Zou, Zhenyu Li, Wenchao Li, Yuanzhen Zhang, Lanxin Hu, Bin Xu, Baobing Zhao, Zhipeng Zhou, Dingxiao Zhang
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9cd3d853d5fc4e5d9af6268cc197a4347824161" target='_blank'>
              Epigenetic plasticity is a driver of heritable pollution tolerance in Atlantic killifish
              </a>
            </td>
          <td>
            Mathia Colwell, Melissa K Drown, Nicole Flack, Carrie Walls, Christopher Faulk, Samantha Carrothers, Emma Weeks, Caren Weinhouse
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4fd78fdc6e7168376ba52f3c22a1f28b8371667" target='_blank'>
              Transcriptomic profiling of epigenetic regulators and metabolic reprogramming in human cholangiocarcinoma
              </a>
            </td>
          <td>
            Amaya López-Pascual, Jasmin Elurbide, Emiliana Valbuena-Goiricelaya, M. Latasa, Elena Anaya, Elena Adán-Villaescusa, Borja Castelló-Uribe, L. A. Martínez-Pérez, I. Uriarte, M. Arechederra, Sergio Ciordia, Fernando Corrales, Sona Frankova, Eva Sticová, Ondřej Fabián, L. Colyn, Patricia Inacio, Robert Arnes-Benito, J. Bayo, Meritxell Huch, C. Berasain, M. G. Fernández-Barrena, Matías A. Avila
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The efficacy of anti-cancer therapies is severely limited by the emergence of drug resistance. While genetic drivers are well-characterized, growing evidence suggests that non-genetic mechanisms, particularly those involving drug-tolerant persisters (DTPs), play a pivotal role in solid tumor relapse. To elucidate the evolutionary dynamics of DTP-induced resistance, we develop a stochastic agent-based model (ABM) of solid tumor evolution that couples macroscopic population dynamics with microscopic epigenetic state inheritance during the cell cycle. Our simulations accurately reproduce the temporal progression of relapse observed in experimental studies, capturing the dynamic transition from sensitive cells to DTPs, and ultimately to stable resistant phenotypes under prolonged therapy. By explicitly modeling the epigenetic plasticity of individual cells, our model bridges the gap between cellular heterogeneity and population-level tumor evolution. Furthermore, we performed \textit{in silico} clinical trials using virtual patient cohorts to evaluate therapeutic outcomes, demonstrating that optimized adaptive treatment strategies can significantly delay tumor relapse compared to standard dosing. This study provides a quantitative framework for dissecting DTP-driven resistance mechanisms and designing more effective, biologically informed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5335957f9dd7ef1ab2ffbe1285dc0a351204f56d" target='_blank'>
              Epigenetic state inheritance drivers drug-tolerant persister-induced resistance in solid tumors: A stochastic agent-based model
              </a>
            </td>
          <td>
            Xiyin Liang, Jinzhi Lei
          </td>
          <td>2026-01-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed06fb384ee497b6337be9253bfa34f2d9608de" target='_blank'>
              The sensitivity of acute myeloid leukemia to CDK8/19 inhibitors is determined by their metabolic profile
              </a>
            </td>
          <td>
            Varlamova Ekaterina Antonovna, Andreeva Daria Vladislavovna, Dalina Alexandra Alexandrovna, Misnik Elena Vladimirovna, Pavlenko Natalia Germanovna, Bruter Alexandra Vladimirovna, Tatarskiy Victor Vyacheslavovich
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b11efdad5d60344c302ed6d2c56a7d648ae4da7" target='_blank'>
              Targeted therapy-induced chromosomal instability dictates mitotic dependency on Aurora Kinase A
              </a>
            </td>
          <td>
            Chendi Li, Varuna Nangia, Melissa D. Vieira, Anahita Nimbalkar, Christopher J. Graser, Jeremy B Chang, Mohammad U. Syed, Yi Shen, Radhika Koranne, Lee Zou, Xueqian Gong, Sabrina L Spencer, Aaron N. Hata
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cisplatin-based chemotherapy remains a mainstay for the treatment of bladder cancer (BLCA); however, its clinical efficacy is frequently compromised by the emergence of chemoresistance, which leads to poor patient outcomes. Although known mechanisms—such as alterations in drug efflux, DNA repair, and key signaling pathways—have been implicated, they fail to fully explain the clinical complexity of cisplatin resistance, indicating that additional molecular drivers remain undiscovered. Keratin 14 (KRT14), an intermediate filament protein associated with aggressive BLCA subtypes, is consistently upregulated in cisplatin-resistant tumors, yet its precise mechanistic role in resistance remains unclear. In this study, we elucidate the functional contribution of KRT14 to cisplatin resistance in BLCA using patient-derived tissues, established cell lines, xenograft mouse models, and a suite of molecular interaction assays. Our results demonstrate that KRT14 is significantly upregulated in BLCA tissues, correlates with poor clinical prognosis, and functionally drives cisplatin resistance both in vitro and in vivo. Mechanistically, we identify a novel and direct interaction between KRT14 and the translation initiation factor eIF4H, specifically through the N-terminal Head domain of KRT14. This interaction modulates the association of eIF4H with the core eIF4F complex, thereby selectively promoting the translation of Acyl-CoA Oxidase 2 (ACOX2) mRNA through its 5’ untranslated region. We further show that ACOX2 is essential for mediating the effects of KRT14 on lipid metabolism, cell proliferation, survival, and ultimately, cisplatin resistance. Collectively, our findings reveal that KRT14 contributes to chemoresistance in BLCA not only via its structural roles but also by directly regulating translational machinery through eIF4H, leading to upregulation of the metabolic enzyme ACOX2. The newly defined KRT14–eIF4H–ACOX2 axis orchestrates lipid metabolic reprogramming and cell survival, underscoring the KRT14–eIF4H interface as a promising therapeutic target for overcoming cisplatin resistance in BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec144141be685aac475058d09e8da4ccea2f5711" target='_blank'>
              Cytoskeletal protein KRT14 governs cisplatin resistance by modulating eIF4H-dependent ACOX2 translation and lipid metabolism in bladder cancer
              </a>
            </td>
          <td>
            Shenghua Liu, Tingting Liu, Chen Yang, Z. Mou, L. Zou, Haowen Jiang
          </td>
          <td>2025-12-24</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven in part by metabolic reprogramming and the persistence of cancer stem-like cells (CSCs). The seven in absentia homolog 2 (SIAH2)/with-no-lysine kinase 1 (WNK1) signaling axis has emerged as a potential regulator of these processes, yet its functional role in CRC metabolism and tumor–stroma crosstalk remains incompletely understood. Integrated analyses of The Cancer Genome Atlas–Colon Adenocarcinoma (TCGA-COAD) and Gene Expression Omnibus (GEO, GSE17538) datasets revealed significant upregulation of SIAH2 and WNK1 in CRC tissues, with strong positive correlations to glycolysis- and hypoxia-associated genes, including PFKP, LDHA, BPGM, ADH1A, ADH1B, and HIF-1α. Single-cell and clinical profiling further demonstrated preferential enrichment of SIAH2 in undifferentiated, stem-like tumor cell populations. Functional studies across multiple CRC cell lines showed that SIAH2 silencing suppressed proliferation, clonogenic growth, tumor sphere formation, and cell-cycle progression, whereas SIAH2 overexpression exerted opposite effects. Seahorse extracellular flux analyses established that SIAH2 promotes glycolytic capacity and metabolic flexibility. At the protein level, SIAH2 regulated glycolytic enzymes and WNK1/hypoxia-inducible factor-1α (HIF-1α) signaling, effects that were amplified by cancer-associated fibroblast (CAF)-derived conditioned medium. CAF exposure enhanced SIAH2 expression, CSC spheroid growth, and resistance to fluorouracil, leucovorin, and oxaliplatin (FOLFOX) chemotherapy, whereas SIAH2 depletion effectively abrogated these effects. Collectively, these findings identify the SIAH2/WNK1 axis as a central metabolic regulator linking glycolysis, CSC maintenance, and microenvironment-driven therapy resistance in CRC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb269b1f5618b40199d4b3cc75e161d29b2f08d9" target='_blank'>
              SIAH2–WNK1 Signaling Drives Glycolytic Metabolism and Therapeutic Resistance in Colorectal Cancer
              </a>
            </td>
          <td>
            Kee-Thai Kiu, Cheng-Ying Chu, Yi-Chiao Cheng, Min-Hsuan Yen, Ying-Wei Chen, N. W. Pikatan, Vijesh Kumar Yadav, Tung-Cheng Chang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Aims Mitochondrial DNA copy number (mtDNA-CN) is associated with several age-related chronic diseases and is a predictor of all-cause mortality. Here, we examine site-specific differential nuclear DNA (nDNA) methylation and differential gene expression resulting from in vitro reduction of mtDNA-CN to uncover shared genes and biological pathways mediating the effect of mtDNA-CN on disease. Materials and methods Epigenome and transcriptome profiles were generated for three independent human embryonic kidney (HEK293T) cell lines harboring a mitochondrial transcription factor A (TFAM) knockout generated via CRISPR-Cas9, and matched control lines. Results We identified 2924 differentially methylated sites, 67 differentially methylated regions, and 102 differentially expressed genes associated with mtDNA-CN. Integrated analysis uncovered 24 Gene-CpG pairs. GABAA receptor genes and related pathways, the neuroactive ligand signaling pathway, ABCD1/2 gene activity, and cell signaling processes were overrepresented, providing insight into the underlying biological mechanisms facilitating these associations. We also report evidence implicating chromatin state regulatory mechanisms as modulators of mtDNA-CN effect on gene expression. Conclusions We demonstrate that mitochondrial DNA variation signals to the nuclear DNA epigenome and transcriptome and may lead to nuclear remodeling relevant to development, aging, and complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcbea560ce049511e5c017437908e0cfbb595232" target='_blank'>
              Mitochondrial DNA copy number reduction via in vitro TFAM knockout remodels the nuclear epigenome and transcriptome
              </a>
            </td>
          <td>
            Phyo W. Win, Julia Nguyen, Elly H. Shin, T. S. Nagano, Brent Selimi, Katie Hong, A. Meybodi, Bradley P Yates, Emma V Burke, David E Carter, Gregory A Newby, Charles E Newcomb, D. Arking, C. Castellani
          </td>
          <td>2025-12-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="To probe drug mechanism of action (MOA) and interrogate the genetic architecture of human cells, we carried out isogenic genome-wide CRISPR/Cas9 knockout screens against 310 diverse drugs, bioactive compounds, and stress conditions. Stringent statistical correction for gene knockout fitness defects yielded a large-scale matrix of >12,000 high confidence chemical-genetic interactions (CGIs). This dataset revealed many previously unappreciated off-target effects for well-characterized compounds and novel MOAs for uncharacterized compounds. The CGI matrix uncovered dense genetic modules that yielded new biological insights into phospholipidosis, mitotic regulation, metabolism, the DNA damage response, and mTOR signaling. The dataset allowed identification of multi-drug sensitization and resistance mechanisms, inference of gene function, elaboration of cross-process connectivity, evaluation of the cell type specificity of CGIs, prediction of chemical synergism, and extensive annotation of understudied genes. This resource provides a map of the genetic landscape in human cells and a framework to help guide drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ea6f9c264b3644d442991366676c1b8529f455" target='_blank'>
              A Chemical-Genetic Interaction Matrix Reveals Drug Mechanism and Genetic Architecture
              </a>
            </td>
          <td>
            J. Coulombe-Huntington, Thierry Bertomeu, Caroline Huard, Andrew Chatr-aryamontri, Daniel J. St-Cyr, María Sánchez-Osuna, David Papadopoli, Karine Normandin, Mohammadjavad Paydar, Shannon Mclaughlan, C. St-Denis, Li Zhang, Henry Say, Roger Palou, Chris Stark, Bobby-Joe Breitkreutz, Almer M. van der Sloot, Sandhya Manohar, H. Lavoie, Katherine L. B. Borden, B. Raught, Damien D’Amours, Frank Sicheri, Alain Verreault, S. Mader, Sylvain Meloche, M. Therrien, Pierre Thibault, Brian T. Wilhelm, Peter B. Dirks, J. Aitchison, E. Patton, Randall W. King, Philippe P. Roux, Guy Sauvageau, Trang Hoang, A. Marinier, Lea Harrington, Benjamin Kwok, Vincent Archambault, I. Topisirovic, Mike Tyers
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) identified over 600 loci containing single-nucleotide polymorphisms (SNPs) associated with type 2 diabetes (T2D), most of which reside in non-coding regions. Among the set of T2D SNPs, linking causal genome variants to disease risk experimentally has remained a challenge; however, advances in synthetic mammalian genome writing techniques now enable the delivery of multiple haplotypes to human induced pluripotent stem cells (hiPSCs) to create a series of isogenic cell lines that can be differentiated and phenotyped in vitro. Here, to begin efforts in dissecting a T2D GWAS locus, we engineered an NKX6-3/ANK1 gene cluster knockout hiPSC line and introduced a landing pad facilitating the delivery of synthetic haplotype payloads. We built four haplotypes, including several that are not observed in nature, containing risk SNPs spanning the NKX6-3/ANK1 gene cluster using a method called “variant Switching Auxotrophic markers for Integration” (vSwAP-In), and integrated them precisely into hiPSCs. NKX6-3/ANK1 deletion blocked pancreatic progenitor and skeletal muscle differentiation, suggesting that NKX6-3 and ANK1 are required for early pancreatic and skeletal muscle development, and perhaps related to the existence of two nonoverlapping sets of SNPs in linkage disequilibrium that associate with the expression of the two adjacent genes. When NKX6-3/ANK1 T2D “Risk” haplotypes were reintroduced, skeletal muscle and pancreatic progenitor differentiation capabilities were restored. ANK1 expression was elevated in the ANK1 Risk and All-Risk haplotypes compared to the NKX6-3 Risk and Non-Risk haplotypes, establishing a functional experimental platform to examine risk SNP clusters in their native contexts. Overall, this work establishes a platform for the dissection of GWAS loci using synthetic haplotype genomics in hiPSCs. Significance Statement Genome-wide association studies have been used to identify disease-associated SNPs; however, most SNPs lie in non-coding regions, making functional experimentation difficult to perform. Using vSwAP-In, a yeast-based DNA variant-building method, and mSwAP-In, a mammalian genome engineering approach, we establish a platform for functional GWAS dissection in hiPSCs. This platform allows us to build DNA harboring virtually any combination of disease-risk SNPs, allowing for functional characterization of SNPs without the limitations of linkage disequilibrium. We demonstrate this approach using a Type 2 diabetes GWAS gene cluster, NKX6-3/ANK1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f19a7bbc6f3616a799aaa84ab7eb946f7cda112e" target='_blank'>
              Genome writing and Targeted Delivery of the NKX6-3/ANK1 gene cluster and its Type 2 Diabetes GWAS Variants to Human iPSCs
              </a>
            </td>
          <td>
            Noor Chalhoub, Arushi Varshney, Weimin Zhang, Skyler Uhl, J. M. Laurent, Colleen McLoughlin, Hannah J. Ashe, Xingrui Mou, Nathan Dale, Kiran Ramnarine, Dan Paull, Jordan Goldberg, Matthew T. Maurano, Ran Brosh, David Fenyö, Filippo Cipriani, Stephen C. J. Parker, J. Boeke
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Abstract A major challenge in treating AML with the BCL-2 inhibitor venetoclax is the frequent development of drug resistance, which diminishes therapeutic efficacy and leads to patient death. The fundamental mechanisms underlying this resistance are not fully understood. Here, we established venetoclax-resistant cell models of AML that propagate even when the levels of BCL-2, MCL-1, cleaved PARP, and cleaved caspase-9 are reduced, suggesting a BCL-2-independent resistance mechanism. Compared to sensitive cells, resistant Kasumi-1 (VENK) and MV4-11 (VENM) cells exhibit enhanced proliferation both in vitro and in vivo , forming larger and more numerous spheroids and colonies, and displaying higher tumorigenicity in mice. RNA sequencing and KEGG pathway analysis identified the neuroactive ligand–receptor interaction (NLRI) pathway as a key vulnerability in both resistant cell lines. While the NLRI pathway contains numerous altered genes, CHRNB4 is the only gene commonly shared and significantly downregulated in both VENK and VENM cells and tumors. Enforced expression of CHRNB4 in resistant cells with low basal expression impaired cell adhesion and colony formation. Clinically, CHRNB4 downregulation is associated with poor AML patient overall survival and predicts a diminished response to venetoclax treatment. This study identifies the NLRI pathway as a crucial vulnerability in venetoclax resistance and unveils CHRNB4 as a promising predictive biomarker for treatment response. These results suggest that targeting the NLRI pathway represents a novel strategy for developing next-generation therapies to improve the poor outcomes of current combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4499cae427f9bfc7940382b20e5c02e2a96abc" target='_blank'>
              Rewired Neuroactive Ligand-Receptor Signaling Confers Adaptive Resistance to BCL-2 Inhibition in AML
              </a>
            </td>
          <td>
            Hiroaki Koyama, Sachiko Seo, William Tse, Sicheng Bian, Shujun Liu
          </td>
          <td>2026-01-12</td>
          <td>Research Square</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The tumour suppressor TP53 (tumor protein p53) is a master regulator of cell cycle, DNA repair, and apoptosis, and its mutation is a hallmark of cancer, with individual mutations exerting distinct effects on tumour biology. Despite accounting for ~7% of all TP53 variants, splice site mutations remain the least studied class, and their functional and clinical consequences are poorly understood. We analyzed 25,058 TP53 variants (18,562 somatic; 6496 germline) to characterize the frequency, molecular impact, transcriptional effects, genomic instability, and clinical outcomes of splice mutations. These alterations showed distinct distributions and substitution patterns between germline and somatic contexts and were frequently associated with copy number alterations, reduced TP53 mRNA, and variable protein expression. Transcriptomic profiling identified two transcriptional phenotypes: one with global suppression of canonical p53 target genes and another with mixed activation and repression independent of tumour type. Genomic instability was elevated in a subset of splice-mutant tumours, correlating with increased relapse risk, while other splice mutations showed lower instability but divergent clinical outcomes, including unexpectedly poor prognoses. Our findings fill a critical knowledge gap, defining the biological and clinical spectrum of TP53 splice site mutations and highlighting their potential as prognostic biomarkers and therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df743368c74812062cf79c30615a68f562fc982" target='_blank'>
              TP53 Splice Mutations Have Tumour-Independent Effects on Genomic Stability and Prognosis: An In Silico Study
              </a>
            </td>
          <td>
            Apeksha Arun Bhandarkar, Noah Ethan Kelly-Foleni, D. Sarkar, Aaron Jeffs, T. Slatter, Antony Braithwaite, Sunali Y Mehta
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Amino acid metabolism plays a crucial role in cancer progression and invasion by modulating immune responses within the tumor microenvironment and tumor immune response. Transmembrane transport of cellular amino acids is mediated by members of the cationic amino acid transporters SLC7 family, a group of cationic amino acid transporters. Dysregulation of SLC7 family genes can disrupt systemic homeostasis, contributing to a variety of diseases, including multiple cancer types. Additionally, SLC7 family members can alter amino acid metabolic plasticity, further influencing tumor adaptation and progression. However, the precise impact of SLC7 family genes within the cancer microenvironment remains largely unexplored. This study analyzed the expression patterns, genomic alterations (single nucleotide variants, copy number variations, methylation levels), pathway activations, and clinical and subtype associations of SLC7 family genes across The Cancer Genome Atlas (TCGA) pan-cancer cohort. SLC7 activity was quantified using the ssGSEA function of the “GSVA” package to generate the SLC7 score, which was found to correlate with oncogenic and immune-related pathways, including mTORC1 signaling, interferon-gamma response, and inflammatory pathways. In vitro validation was conducted using multiple cancer cell lines (A549, H1299, HCT116, FaDu, Panc-1) and counterparts (BxPC-3, SW480, H460, BEAS-2B, FHC, HaCaT). Gene knockdown was achieved by siRNA transfection, followed by quantitative PCR, Western blotting, and functional assays assessing proliferation, invasion, and ferroptosis (iron and GSH levels) under erastin treatment. SLC7 family genes showed significant associations with immune infiltration and poor prognosis across various cancers. The SLC7 score demonstrated strong positive correlations with malignant and immune-related pathways, such as mTORC1 signaling, interferon-gamma response, and inflammatory response across the pan-cancer cohort. Elevated SLC7 scores were also associated with enhanced immune-related characteristics, and SLC7 genes, particularly SLC7A8 and SLC7A9, emerged as promising predictive markers for response to immune checkpoint inhibitor therapy (Atezolizumab) in cancer patients. Among SLC7 family genes, SLC7A5 expression was markedly elevated in KRAS-mutant tumor cell lines and was required for KRAS-driven proliferation and invasion. SLC7A11 was highly expressed in multiple tumor types and protected cancer cells from ferroptosis by maintaining intracellular cystine uptake and redox homeostasis. Individuals exhibiting elevated SLC7 scores and aberrant expression of SLC7 family genes demonstrate significant immune-inflammatory tumor immune microenvironment (TIME) characteristics and predictive potential for immunotherapy. Thus, the SLC7 family genes have the potential to serve as valuable biomarkers and therapeutic targets for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15f37f25792e120e4c9d88c14c57b67424b4e09" target='_blank'>
              Integrative pan-cancer analysis reveals the prognostic and immunological significance of SLC7 family cationic amino acid transporters
              </a>
            </td>
          <td>
            Gang Peng, Hao Peng, Zhe Shao, Liangqi Jiang, Mingrui Li, Yang Li
          </td>
          <td>2026-01-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Loss of isozyme diversity (LID) refers to the selective dependency on a single isozyme following the functional collapse of its redundant counterparts, uncovering a metabolic vulnerability. This metabolic liability establishes LID as a novel framework for precision targeting strategy in cancer therapy. We integrated a large-scale breast cancer multi-omics cohort to systematically delineate isozyme expression patterns and identify therapeutically promising LID targets across intrinsic molecular subtypes. Genomic and epigenetic data were employed to decipher the underlying causes of LID. Subsequent experiments were conducted to validate the collateral metabolic vulnerabilities arising from identified LID targets and investigate the associated mechanisms. Our analysis delineated the isozyme utilization patterns for metabolic reactions in breast cancer, which revealed widespread and significant LID phenomena. We further decoded the enrichment characteristics of these LID reactions and identified a set of potential clinically relevant LID targets across different subtypes. Further investigation revealed that copy number loss and epigenetic silencing predominantly drive these LIDs, as exemplified by luminal subtype-specific N-myristoyltransferase 2 (NMT2) deficiency caused by hypermethylation. Mechanistic studies demonstrated that NMT2 deficiency confers synthetic lethality with NMT1 inhibition, potentially through impaired myristoylation of the oncogenic effector FMR1 autosomal homolog 2 (FXR2). These findings underscored the potential of NMT inhibitors as viable therapeutic strategies for NMT2-deficient tumors. In conclusion, this study constructed a comprehensive LID landscape across all breast cancer subtypes, advancing precision oncology through innovative metabolic targeting approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48d7510a6c9d424472c6f680b4af6eada3b51b02" target='_blank'>
              Collateral metabolic vulnerabilities unveiled by loss of isozyme diversity in breast cancer
              </a>
            </td>
          <td>
            Rui Ding, Tianjian Yu, Yizhou Jiang, Yi Xiao, Zhi-Ming Shao
          </td>
          <td>2025-12-12</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Genetic factors driving the development of pediatric Acute Lymphoblastic Leukemia (ALL) remain incompletely understood. While epigenetic dysregulation through histone methyltransferases and demethylases has emerged as a critical oncogenic mechanism, their specific contributions to pediatric ALL require systematic investigation. In this study, we systematically investigated genetic variants in histone methyltransferases and demethylase found in top 50 mutated genes from TARGET ALL Phase 2 cohort. Single nucleotide variations were functionally annotated using PredictSNP and I-Mutant2.0, followed by survival analysis and pathway enrichment studies comparing mutated versus wild-type patients. KMT2D, NSD2, SETD2, EZH2 (methyltransferases), and KDM6A (demethylase), ranked among the most frequently mutated genes. Multiple frameshift and nonsense mutations were identified in these genes, likely resulting in truncated proteins and loss of function. Critically, missense variants in EZH2 (F145C, R679C, P132T, D652G, A576D, S280C, Y728H, C566S), KDM6A (G1242D), NSD2 (E1099K, D1125H), and SETD2 (G2170D, R1592Q) were computationally predicted as deleterious and destabilizing. Although survival analysis revealed no statistically significant differences (p = 0.37), mutation carriers exhibited a 44% increased mortality risk, suggesting clinical relevance despite limited sample size (n = 35). Gene set enrichment analysis of differentially expressed genes revealed significant activation of hematopoietic cell lineage pathways in mutated patients. Our findings underscore the importance of histone methyltransferase and demethylase gene variants in pediatric ALL, particularly novel mutations affecting protein function and stability that remain poorly characterized in the literature. This study identifies candidate mutations that warrant further functional and clinical investigation for their role in disease progression and therapeutic targeting.
HIGHLIGHTS

KMT2D, NSD2, SETD2, EZH2, and KDM6A are identified among the top 50 most frequently mutated genes in pediatric ALL
KMT2D exhibits the highest mutational burden in pediatric ALL, predominantly harboring frameshift and nonsense alterations
KMT2D, NSD2, SETD2, EZH2, and KDM6A genes show consensus deleterious predictions across multiple in silico tools, with 18 mutations representing high-priority candidates for experimental validation
Structural stability analysis reveals critical destabilizing mutations in EZH2 (F145C, R679C, P132T, D652G, A576D, S280C, Y728H, C566S), KDM6A (G1242D), NSD2 (E1099K, D1125H), and SETD2 (G2170D, R1592Q)

GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15b6568cbd53c364a5d95d54d4bb823b4c767ac" target='_blank'>
              Histone Methyltransferase and Demethylase Gene in Pediatric Acute Lymphoblastic Leukemia: A Molecular Insights
              </a>
            </td>
          <td>
            Nur Aziz, Y. N. Patria, A. Wahdi, Eddy Supriyadi, Dewajani Purnomosari
          </td>
          <td>2026-01-01</td>
          <td>Trends in Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fab0829fa668d3f68087f51374ae0be2ba943151" target='_blank'>
              Perturbational single-cell profiling of patient tumors defines lineage- and context-specific programs of innate immune resistance
              </a>
            </td>
          <td>
            C. Perry, A. Frey, Y. Fei, J. Wang, P. Costa, H. N. Amin, S. Ghadermarzi, D. Levine, W. Tong, A. Koda, M. Mackie, M. He, Y. Nie, K. Clulo, F. Ouerghi, J. Wei, T. Cordero Dumit, M. Yaskolko, M. Ding, A. Caldera, O. Kyrysyuk, L. Lum, J. W. Allen, W. Guo, A. Naqash, A. Elliot, A. Vanderwalde, M. Capelletti, T. Adeyelu, S. Barry, M. Hugaboom, A. Bacchiocchi, H. Kluger, M. Bosenberg, A. Iwasaki, D. Braun, J. Clune, D. van Dijk, K. Olino, J. Ishizuka
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf612b75850cf10e9bb17c10ec3f99a040c4e2d" target='_blank'>
              Lineage plasticity: a new dilemma in lung cancer treatment
              </a>
            </td>
          <td>
            Cainan Huo, Yongxuan Wang, Rujia Li, Xiaofei Xie, Weiqi Lyu, Dingke Chen, Yanqin Sun
          </td>
          <td>2026-02-04</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Neurodevelopmental disorders (NDDs) are highly diverse conditions often associated with genetic abnormalities. However, a large number of NDD cases remain undiagnosed despite thorough genetic testing using short-read sequencing and chromosomal microarray analysis. Emerging evidence indicates that structural variants (SVs) in evolutionarily new and human-specific genomic regions may be responsible for these unresolved cases. We used Oxford Nanopore long-read DNA sequencing in six patients with unexplained NDDs who had previously tested negative for genetic mutations. Structural variants were identified and filtered based on their genomic location, regulatory potential, and expression in the central nervous system. Confirmatory fluorescence in situ hybridization (FISH) was performed. Additionally, immunohistochemical analysis of the candidate gene ANKRD20A1 was carried out to assess its spatiotemporal expression in the developing human brain. Interactome analysis was also performed to evaluate the functional connections of genes impacted by SVs. A total of twenty-six candidate SVs were found, including deletions, duplications, insertions, and inversions. Many of these SVs affect rapidly evolving gene families such as NBPF, TBC1D3, and RGPD, as well as regulatory elements in brain-expressed genes, suggesting their potential to disrupt brain development and lead to NDD. FISH analysis confirmed several large SVs. Patient-specific interactome maps showed that most RGPD-disrupted genes create extensive interactions. Immunohistochemical analysis revealed that ANKRD20A1, a candidate gene, is dynamically expressed during midfetal human cortical development, suggesting its involvement in neurodevelopmental events. Our findings highlight the crucial role of long-read DNA sequencing in uncovering concealed structural variants within recently evolved genomic regions. These SVs may contribute to the development of NDDs by disrupting coding sequences, regulatory elements, or complex gene networks. This study supports integrating long-read sequencing into research workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c916e8e3cf253df90c9f61c26b96d9c2ab970305" target='_blank'>
              Structural variations in evolutionary novel genomic regions: new insights into neurodevelopmental disorders by long-read DNA Sequencing
              </a>
            </td>
          <td>
            Martina Rinčic, Janja Kopić, Valentina Klein, Ž. Krsnik, Thomas Liehr, Sebastian Giesselmann, I. Kurth, F. Kraft
          </td>
          <td>2026-01-09</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the second most frequently diagnosed cancer worldwide and represents a major challenge for public health. Despite advances in molecular profiling, important gaps remain in our understanding of tumorigenesis and the regulatory mechanisms underlying CRC progression. The most widely adopted classification system is the Consensus Molecular Subtypes (CMS), which stratifies CRC into four biologically distinct subtypes. We investigated the role of A-to-I RNA editing across CMS subtypes in a cohort of 100 CRC patients at various disease stages. Bulk RNA-seq data were analyzed using REDItools to detect editing events, focusing on both recoding sites and edits within repetitive elements, such as Alu sequences. Furthermore, expression levels of the ADAR enzyme family were assessed, and deconvolution analyses were performed on single-cell RNA-seq data from an independent cohort of stage II CRC patients to characterize editing activity within the tumor microenvironment (TME). Competitive endogenous RNA (ceRNA) networks, specific to each CMS subtype, were constructed based on editing events in repetitive elements. A multivariate Cox proportional hazards model was applied to evaluate associations with overall survival (OS). We observed statistically significant differences in ADARB1 expression across CMS subtypes. Single-cell RNA-seq data revealed subtype-specific distribution patterns of ADAR enzymes within the TME. Analysis of editing events showed subtype-specific signatures in both known cancer-related genes (e.g., COPA, CADPS, IGFBP7) and novel candidates (ZNF552, RALGPS1). Editing in repetitive elements informed the construction of distinct ceRNA networks for each CMS subtype, suggesting different post-transcriptional regulatory mechanisms. Survival analysis identified three variables significantly associated with OS, independent of CMS classification and clinical stage: ADARB1 expression, and editing events in NOP14-AS1 (chr4:2960236; p = 0.036; HR = 0.0069), previously linked to 5-FU sensitivity, and ST7-AS2 (chr4:117120557). This study underscores the biological relevance of RNA editing in CRC, highlighting its impact on chemoresistance, the tumor microenvironment, and subtype-specific gene regulation. Our findings suggest that RNA editing represents a critical post-transcriptional regulatory layer in CRC and holds potential as a biomarker and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/289a3e685e9081e2037f2e82b68ea443c3971d6f" target='_blank'>
              Biological and prognostic relevance of A-to-I RNA editing across consensus molecular subtypes of colon cancer
              </a>
            </td>
          <td>
            Dario Monaco, D. Traversa, E. Mattioli, F. A. Zito, Grazia Cristiani, Francesca Buono, Sabina Delcuratolo, Tiziana Guarino, R. Pinto, Antonia Lasorella, R. Lacalamita, Ernesto Picardi, Stefania Tommasi, S. De Summa
          </td>
          <td>2026-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy where traditional DNA sequence-based classification fails to fully explain clinical outcomes. This review explores the dynamic and multifaceted role of the epigenome as a critical driver of leukemogenesis, progression, and therapeutic resistance. We detail how disruptions in DNA methylation, histone modifications, chromatin architecture, and non-coding RNAs create a complex regulatory landscape that transcends genetic alterations. We highlight the emergence of single-cell multi-omics technologies, which are resolving intratumoral heterogeneity and uncovering the epigenetic signatures of leukemic stem cells. Furthermore, we discuss the significant translational potential of these discoveries, including the development of epigenetic biomarkers for refined risk stratification, minimal residual disease (MRD) monitoring, and predicting response to novel epigenetic therapies like menin and LSD1 inhibitors. Integrating these dynamic epigenetic layers into clinical decision-making promises to usher in a new era of precision medicine, ultimately improving prognostic accuracy and therapeutic outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3853f768df9c1d2f10b0fd3237fd7635474d26" target='_blank'>
              Beyond the DNA sequence: mapping the dynamic epigenetic landscape for risk stratification and therapeutic intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jiqian Xie, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-12-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains incurable despite advances in treatment options. Although tumor subtypes and specific DNA abnormalities are linked to worse prognosis, the impact of immune dysfunction on disease emergence and/or treatment sensitivity remains unclear. We developed an Immune Atlas of MM by generating profiles of 1,397,272 single cells from the bone marrow (BM) of 337 newly diagnosed participants and characterized immune and hematopoietic cell populations. Cytogenetic risk-based analysis revealed heterogeneous associations with T cells of BM, with 17p13 deletion showing distinct enrichment of a type 1 interferon signature. The disease progression-based analysis revealed the presence of a proinflammatory immune senescence-associated secretory phenotype in rapidly progressing participants. Furthermore, signaling analyses suggested active intercellular communication involving a proliferation-inducing ligand and B cell maturation antigen, potentially promoting tumor growth and survival. Lastly, using independent discovery and validation cohorts, we demonstrated that integrating immune cell signatures with known tumor cytogenetics and individual characteristics significantly improves stratification for the prediction of survival. Pilcher et al. present a single-cell transcriptomics-based immune atlas of participants with newly diagnosed multiple myeloma, reporting on the association of cell types, gene expression and intercellular interactions with disease progression phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f88dd8011aa8686433c4d02b3a0e4dfd59a9255" target='_blank'>
              A single-cell atlas characterizes dysregulation of the bone marrow immune microenvironment associated with outcomes in multiple myeloma
              </a>
            </td>
          <td>
            William C. Pilcher, Lijun Yao, Edgar Gonzalez-Kozlova, Yered H. Pita-Juárez, Dimitra Karagkouni, Chaitanya R. Acharya, Marina E. Michaud, Mark Hamilton, Shivani Nanda, Yizhe Song, Kazuhito Sato, Julia Wang, Sarthak Satpathy, Yuling Ma, Jessica Schulman, D. D’souza, R. Jayasinghe, Denis J Ohlstrom, Katherine E. Ferguson, Giulia Cheloni, Mojtaba Bakhtiari, Nick Pabustan, K. Nie, J. Foltz, Isabella Saldarriaga, Rania Alaaeldin, Eva Lepisto, Rachel Chen, M. Fiala, Beena E. Thomas, April Cook, Junia Vieira Dos Santos, Chiang I-ling, Igor Figueiredo, J. Fortier, Michael Slade, Stephen T. Oh, Michael P. Rettig, Emilie I. Anderson, Ying Li, Surendra Dasari, Michael A. Strausbauch, Vernadette A. Simon, , I-ling Chiang, N. Kalavros, Jennifer Rogers, T. Dawson, Brian H. Lee, G. Kelly, Laura A Walker, Nicolas F. Fernandez, John Leech, Jarod Morgenroth-Rebin, K. Angeliadis, Matthew A. Wyczalkowski, Song Cao, Omar Ibrahim, Roderick Lin, T. Fehniger, Andrew J. Houston, E. Radkevich, Adeeb H Rahman, Zhihong Chen, A. Laganà, John F. DiPersio, J. Rosenblatt, S. Kim-Schulze, S. Lonial, Shaji K. Kumar, Swati S. Bhasin, T. Kourelis, M. Dhodapkar, Ravi Vij, D. Avigan, H. Cho, George Mulligan, Li Ding, Sacha Gnjatic, Ioannis S. Vlachos, Manoj K. Bhasin
          </td>
          <td>2026-01-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Background: Long INterspersed Element-1 (LINE-1) retrotransposons comprise 17–20% of the human genome. These retroelements are normally silenced early in embryonic development through epigenetic mechanisms and reawakened during oncogenesis, leading to transcriptional dysregulation, genomic instability, and immune evasion. Methods: In the present study, we categorized LINE-1 transcripts across 121 non-small cell lung cancer (NSCLC) cell lines from the Cancer Cell Line Encyclopedia (CCLE) by subfamily, length, orientation, chromosomal origin, and distribution. In addition, high-prevalence insertions were mapped to nearby genes to assess potential functional interactions. Results: LINE-1 transcript abundance and length in NSCLC were dominated by evolutionarily young subfamilies, particularly L1HS and L1PA2 through L1PA5. Chromosomal patterns were conserved across NSCLC subtypes, with modest enrichment of L1HS activity on Chromosome 4 and the X Chromosome. The lung squamous cell carcinoma (LSQCC) subtype exhibited the highest total levels of L1HS expression relative to other NSCLC subtypes. Race modestly influenced LINE-1 transcript abundance, with cell lines derived from self-identified African American individuals showing elevated overall LINE-1 and L1HS expression. Age showed a weak positive correlation with total LINE-1 abundance. Integrative analysis revealed recurrent hotspots at 22q12.1 and 20p11.21 that were transcriptionally active across subtypes and coincided with previously reported intact LINE-1 elements active in epithelial cancers. Recurrent insertions were located near cancer-associated genes, including RB1, NEDD4, FTO, LAMA2, NOD1, and KCNB2, implicating LINE-1 activity in cis-regulatory remodeling of oncogenic pathways. Conclusions: Together, these findings indicate that LINE-1 transcript heterogeneity in NSCLC is shaped by host genomic architecture and conserved functional hotspots, providing new insights into the mechanisms of genetic and epigenetic dysregulation associated with LINE-1 retroelements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76a2793974308abd948a4e791ee9224ae6f4f7f" target='_blank'>
              LINE-1 Transcript Heterogeneity in Non-Small Cell Lung Cancers Is Driven by Host Genomic Context and Conserved Functional Hotspots
              </a>
            </td>
          <td>
            Yingshan Wang, Kenneth S. Ramos
          </td>
          <td>2026-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Highlights What are the main findings? The anti-tumor efficacy of nucleoside analogs is dependent on transport, enzymatic activation, nucleotide pools and metabolic responses, which are often aberrant in cancer cells, rendering these chemotherapeutic drugs ineffective. What are the implications of the main findings? Small molecule inhibitors are being tested/developed to target or downregulate specific mechanisms responsible for resistance to nucleoside analogs. The combination of small molecule inhibitors with nucleoside analogs has enhanced the efficacy of nucleoside analogs and improved patient outcomes. Abstract Nucleoside analogs remain central to the treatment of hematologic malignancies and solid tumors, yet resistance frequently occurs, contributing to relapse and disease-related mortality. Rather than arising from a single mechanism, effective nucleoside analog activity requires successful navigation of multiple biological barriers, including cellular uptake, intracellular activation, nucleotide pool balance, genome surveillance, and mitochondrial stress responses. This review integrates recent advances describing how alterations at each of these levels contribute to resistance to nucleoside analog therapies. We further highlight emerging therapeutic strategies centered on small-molecule inhibitors that exploit these vulnerabilities to enhance the efficacy of nucleoside analogs. Together, this integrative perspective supports the need for development of small molecule inhibitors and design of combination approaches aimed at restoring apoptotic competence and improving the use of nucleoside analog-based therapies for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f48f64e40e1b94edba907600f2e063e159b5fa3" target='_blank'>
              Enhancing the Nucleoside Analog Response with Translational Therapeutic Approaches to Overcome Resistance
              </a>
            </td>
          <td>
            Jenna Thibodeau, Kian K Hershberger, Sai Samanvitha M. Ramakrishna, Yongwei Su, Lauren Timmer, Bryce Brophy, Katherine K Zhang, Holly Edwards, Jeffrey W. Taub, Yubin Ge
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd45c31488bc26e7ec09d0ff349b87ce4062cd48" target='_blank'>
              Predicting mutation-rate variation across the genome using epigenetic data
              </a>
            </td>
          <td>
            Machiko Katori, Tetsuya J. Kobayashi, Magnus Nordborg, Shoi Shi
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The pathogenesis of bladder cancer (BCa) is driven in part by aberrant epigenetic regulation, most notably the dysregulated expression of histone deacetylases (HDACs). As a class I HDAC, HDAC2 is often overexpressed in cancers and promotes malignancy through diverse mechanisms. Given its broad oncogenic role, an in-depth investigation of its specific functions in epigenetic and post-translational regulation within BCa holds significant promise for developing novel precision therapies. In vitro functional assays, including CCK-8, colony formation, transwell and apoptotic assays, as well as in vivo assays in a nude mouse subcutaneous tumor model, were performed to assess the oncogenic and drug-resistant effects of HDAC2. RNA-seq and ATAC-seq were employed to analyze the epigenetic regulatory mechanisms of HDAC2. Combined proteome, lactylome and acetylome analysis of control and HDAC2-overexpressing BCa cells were conducted to map the global profiling of protein lysine acetylation (Kac) and lactylation (Kla). In vitro and in vivo experiments confirmed that HDAC2 overexpression significantly promoted proliferation, metastasis and chemoresistance of BCa. Integrated RNA-seq and ATAC-seq analysis revealed that HDAC2 overexpression led to significant epigenetic alternations, and knockdown of its downstream GRIK2 significantly reversed the oncogenic effects of HDAC2. We screened class I HDACs for their impact on Kac and Kla in BCa cells and found that HDAC2 most significantly reduced global Kla levels. Subsequent proteomic analysis of HDAC2-overexpressing cells identified 528 differentially regulated Kla proteins (encompassing 683 sites) and 1,129 differentially regulated Kac proteins (encompassing 1,458 sites). Notably, DHX15 in the splicesome pathway emerged as the most prominent HDAC2-regulated lactylated protein in the absence of concurrent Kac alterations. Moreover, HDAC2 promoted BCa malignancy through the downregulation of DHX15 Kla and the subsequent modulation of RPL9 splicing. Collectively, these findings suggest the pivotal role of HDAC2 in epigenetic modulation and lysine lactylation, and underscore HDAC2 as a promising therapeutic target in BCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d5a62ac3780c5fc198eca0d57d65e4b1dc3fc0d" target='_blank'>
              HDAC2 as a therapeutic target in bladder cancer: insights into the altered epigenetic regulation and lysine lactylation
              </a>
            </td>
          <td>
            Guanghui Xu, Shuo Liang, Ganlin Hu, Wei Zhao, Yuqin Li, Minghao Zheng, Zhigang Wu, Tianlei Xie, Shuting Fang, Shan Peng, Yongming Deng, Yihua Zhou, Hongqian Guo, J. Zhuang, W. Diao
          </td>
          <td>2025-12-17</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Background Primary pulmonary sarcomas (PPS) and pulmonary sarcomatoid carcinoma (PSC) are rare and aggressive diseases that pose significant diagnostic challenges, requiring extensive sampling and comprehensive evaluation. To date, the single‐cell characteristics and distinctions between these two conditions have not been thoroughly investigated. Methods In this study, we employed single‐nucleus RNA sequencing (snRNA‐seq) to characterise the cellular heterogeneity of PSC and PPS. Our analysis included 20 PSC samples, seven PPS samples and two non‐malignant control samples obtained from adjacent normal tissue. Results Our results revealed that the majority of cells in PSC were of epithelial origin, while fibroblasts predominated in PPS. Specifically, AT2 cells, a major source of epithelial cells in PSC, underwent malignant transformation primarily through epithelial–mesenchymal transition, suggesting AT2 cells may serve as the origin of PSC. High Mobility Group AT‐Hook 2 (HMGA2) expression was elevated in malignant AT2 cells of PSC and correlated with an unfavourable prognosis. Moreover, MET‐mutated patients have a significantly higher expression level of HMGA2 (p < .001). In PPS, fibroblasts constituted the majority, only lipofibroblasts exhibited malignant features. A direct comparison between PSC and PPS lipofibroblasts revealed largely similar expression profiles, with the exception of an enrichment in DNA repair pathways specifically observed in PPS lipofibroblasts. Conclusion These findings provide novel insights of PSC and PPS at the single‐cell level. Key points Distinct Cellular Origins: PSC arises primarily from epithelial (AT2) cells via EMT, whereas PPS is predominantly fibroblast‐derived. Prognostic Driver HMGA2: Elevated HMGA2 in malignant AT2 cells correlates with poor prognosis and is significantly higher in MET‐mutated PSC. Pathway Divergence in Malignant Cells: Malignant lipofibroblasts in PPS share a similar profile with PSC but uniquely enrich DNA repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914312002105a075ce979102043e4694713bf91a" target='_blank'>
              Single‐nucleus RNA sequencing reveals the distinct heterogeneity of primary pulmonary sarcomas (PPS) and pulmonary sarcomatoid carcinoma (PSC)
              </a>
            </td>
          <td>
            Jianfei Zhu, Yanlu Xiong, Shouzheng Ma, Jun Wei, Jia-kuan Chen, Wen-chuan Wang, Q. Duan, Qin Zhang, Dongsheng Chen, Wanglong Deng, Tao Jiang, Jie Lei
          </td>
          <td>2025-12-30</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Most colorectal cancers are resistant to immune checkpoint inhibitors due to an immunosuppressive tumor microenvironment. LINC00673, previously classified as a long non-coding RNA, has been implicated in tumor progression, but its role in antitumor immunity remains poorly understood. We used single-cell RNA sequencing, flow cytometry, and functional assays in Linc00673 knockout and wild-type mouse models treated with programmed death 1 (PD-1) blockade to dissect changes in immune landscape and tumor cell behavior. Linc00673 deletion enhanced the efficacy of PD-1 therapy, reducing tumor burden and prolonging survival. KO tumors showed increased infiltration of cytotoxic CD8⁺ T cells, reduced exhaustion, and improved antigen presentation. The tumor microenvironment shifted toward a pro-inflammatory state, with elevated dendritic cell function, reduced immunosuppressive macrophages, and improved T cell–tumor interactions. Linc00673 also promoted tumor cell migration and immune evasion independently of immune cells, suggesting dual tumor-intrinsic and -extrinsic roles. LINC00673 suppresses antitumor immunity and promotes tumor aggressiveness. Its deletion synergizes with PD-1 blockade, revealing a promising therapeutic target to overcome immune resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9ef0a9a3f2685b8b34be389ae9c671f82a1666" target='_blank'>
              Single-cell RNA sequencing unveils enhanced antitumor immunity in colorectal cancer with PD-1 blockade and LINC00673 deletion
              </a>
            </td>
          <td>
            Yifei Zhu, Yanxi Yao, Yaxian Wang, Zhibing Qiu, Dingpei Zhou, Huixia Huang, Keji Chen, Qingyang Sun, Jiayu Chen, Yuxue Li, Jianqiang Tang, Dawei Li, Ping Wei
          </td>
          <td>2026-01-03</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is a precursor condition characterized by the expansion of blood cell clones harboring somatic mutations originating in hematopoietic stem cells (HSCs). Since individuals with CHIP face a high risk of developing myeloid malignancies, targeting CHIP clones could provide a viable strategy for leukemia prevention. Despite its clinical significance, the mechanisms underlying CHIP predisposition and progression remain poorly understood. Recent genome wide association studies (GWAS) have identified several non-coding genetic loci that are strongly associated with CHIP; however, their underlying mechanisms still remain unknown. We hypothesize that risk variants in these non-coding loci modulate enhancer elements active in HSCs. To test this, we selected 1,374 non-coding variants from 51 loci associated for CHIP risk in the UK Biobank and screened them for regulatory activity using a Massively Parallel Reporter Assay (MPRA). We performed our lentiviral MPRA screen in MUTZ-3 cells, a human hematopoietic cell line relevant to HSCs, which express CD34 surface marker and are dependent on HSC-specific transcription factors. Using a MPRA library of ∼73,000 constructs in CD34+ fraction of MUTZ-3 cells, we identified 87 variants representing 32 GWAS loci. We used targeted genome editing to demonstrate endogenous enhancer activity across 3 MPRA variants that affect the transcription of NKD2, FLT3, and MSI2. Our functional studies on MSI2 indicate that presence of higher levels of MSI2 mediated by CHIP risk allele enhances the clonal expansion of TET2 knockout hematopoietic stem and progenitor cells, providing a mechanistic link whereby non-coding genetic variants can influence the expansion of mutant CHIP clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103a0a6127003db01ce70e503d0d9d03c9b83f1d" target='_blank'>
              Systematic functional dissection of germline noncoding risk variants impacting clonal hematopoiesis
              </a>
            </td>
          <td>
            Trieu Nguyen, Jessica Jeejan, Takeshi Iwasaki, S. Kales, Joyeeta Chakraborty, Chie Yanase, Aruna Shekhar, Dominika Kwasniak, Aditi Hegde, Richard Voit, Kristy Stengel, Keisuke Ito, Ryan Tewhey, Satish K. Nandakumar
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="In placental XX females, one X chromosome is silenced during a narrow developmental time window by X-chromosome inactivation, which is mediated by Xist noncoding RNA. Although most X-linked genes are silenced during X-chromosome inactivation, some genes can escape. Here, by increasing its endogenous level, we show that Xist RNA can silence escapees well beyond early embryogenesis both in vitro, in differentiated cells, as well as in vivo, in mouse pre- and post-implantation embryos. We further demonstrate that Xist RNA plays a role in eliminating topologically associating domain-like structures spanning clusters of escapees, and this is dependent on SPEN. The function of Xist in silencing escapees and eliminating topological domains is initially fully reversible, but sustained Xist upregulation leads to irreversible silencing and CpG island DNA methylation of escapees. Thus, gene activity and three-dimensional topology of the inactive X chromosome are directly controlled by Xist, well beyond an early developmental time window. The authors show that increased Xist RNA levels can induce de novo silencing of genes that normally escape X inactivation. SPEN depletion prevents the silencing of escape genes upon Xist RNA overexpression in neural progenitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b274cefc089887937597429bef5c9660ece124a" target='_blank'>
              Escape from X inactivation is directly modulated by Xist noncoding RNA
              </a>
            </td>
          <td>
            Antonia Hauth, Jasper Panten, Emma Kneuss, Christel Picard, N. Servant, Isabell Rall, Yuvia A. Pérez-Rico, Lena Clerquin, N. Servaas, L. Villacorta, Ferris Jung, Christy Luong, Howard Y. Chang, Judith B. Zaugg, O. Stegle, D. T. Odom, Edith Heard, Agnese Loda
          </td>
          <td>2025-12-15</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79941e547de1ddb3fb2c22aa561602d507f6cf5a" target='_blank'>
              CRISPR-enhanced assessment of variants of unknown significance nominates oncology therapeutic targets and drug repositioning opportunities
              </a>
            </td>
          <td>
            A. Savino, Athanasios Oikonomou, Riccardo R. De Lucia, Miquel L Grau, Kathrina McCarten, H. Najgebauer, Umberto Perron, Luca Azzolin, Alexandra Livanova, Paolo Cremaschi, Núria López-Bigas, A. Sottoriva, F. Iorio
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a330bae0129f86adaee3ff8e6b953e0c449188da" target='_blank'>
              Virus-like particles enable targeted gene engineering and pooled CRISPR screening in primary human myeloid cells
              </a>
            </td>
          <td>
            Hyuncheol Jung, Pascal Devant, Carter Ching, Mineto Ota, J. Hamilton, Zachary Steinhart, Wayne Ngo, Luis R. Sandoval, Jae Hyung Jung, Da Xu, Meirui An, Esha Urs, P. Chen, Vincent Allain, Takuya Tada, James K. Nuñez, Nathaniel R Landau, David R. Liu, Justin Eyquem, Jennifer A. Doudna, Alex Marson, J. Carnevale
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Although two-thirds of cancers arise from loss-of-function mutations in tumor suppressor genes, there are few approved targeted therapies linked to these alterations. Synthetic lethality offers a promising strategy to treat such cancers by targeting vulnerabilities unique to cancer cells with these mutations. To identify clinically relevant synthetic lethal interactions, we analyzed genome-wide CRISPR/Cas9 knock-out (KO) viability screens from the Cancer Dependency Map and evaluated their clinical relevance in patient tumors through mutual exclusivity, a pattern indicative of synthetic lethality. Indeed, we found significant enrichment of mutual exclusivity for interactions involving cancer driver genes compared to non-driver mutations. To identify therapeutic opportunities, we integrated drug sensitivity data to identify inhibitors that mimic the effects of CRISPR-mediated KO. This approach revealed potential drug repurposing opportunities, including BRD2 inhibitors for bladder cancers with ARID1A mutations and SIN3A-mutated cell lines showing sensitivity to nicotinamide phosphoribosyltransferase (NAMPT) inhibitors. However, we discovered that pharmacological inhibitors often fail to phenocopy KO of matched drug targets, with only a small fraction of drugs inducing similar effects. This discrepancy reveals fundamental differences between pharmacological and genetic perturbations, emphasizing the need for approaches that directly assess the interplay of loss-of-function mutations and drug activity in cancer models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4924d51007baa9dac5b3c8eec35c04f94156ddd0" target='_blank'>
              Challenges and opportunities for oncology drug repurposing informed by synthetic lethality
              </a>
            </td>
          <td>
            Michael Vermeulen, Andrew W. Craig, Tomas Babak
          </td>
          <td>2025-12-01</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25944c10a9197f587ec4417ecb3ba79c9c511b5c" target='_blank'>
              Genome-scale perturb-seq in primary human CD4+ T cells maps context-specific regulators of T cell programs and human immune traits
              </a>
            </td>
          <td>
            Ronghui Zhu, Emma Dann, Jun Yan, Justine Reyes Retana, Ryunosuke Goto, Reese C. Guitche, Lillian K. Petersen, Mineto Ota, Jonathan K. Pritchard, Alex Marson
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35607d62497aa71c83442fc00cbc1c011c30044b" target='_blank'>
              DICER1 syndrome mutations lead to a gain of 3p-miRNA function, HERVH activity and increased metastatic potential
              </a>
            </td>
          <td>
            K. Gordon, N. Bellora, J. Witteveldt, Joanna K. Wojtus, Felix Mueller, Katharina Kases, Pilar G. Marchante, G. Peris, Shelagh Boyle, Sara R. Heras, Atlanta G. Cook, Cei Abreu-Goodger, S. Macias
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b65d9dfad5be63f412b4572537ea1e29f8d88766" target='_blank'>
              In-Depth Characterization of Stem Cell Potency and Genotoxicity for Clinical-Scale Ex Vivo CRISPR/Cas9 Gene Editing
              </a>
            </td>
          <td>
            A. Naseem, W. Vetharoy, T. E. Whittaker, F. Zinghirino, R. Ghosh, E. Gomez Castaneda, H. Ali, C. Cipriani, E. Montini, A. J. Thrasher, G. Santilli, D. Cesana, G. Turchiano, A. Cavazza
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cisplatin remains a cornerstone of chemotherapy for numerous cancers, despite the persistent challenges of toxicity and the development of drug resistance. Therefore, a deeper understanding of the mechanisms behind cisplatin resistance and the development of strategies to counter it are of critical importance. This review systematically examines the pivotal role of mitochondrial dynamics in cisplatin resistance and discusses emerging therapeutic strategies that target these processes. Mitochondrial dynamics regulate the structure and function of the mitochondrial network through a balance of fusion and fission. Dysregulation of this process directly contributes to cisplatin resistance by maintaining cellular energy homeostasis, inhibiting apoptosis, and enhancing oxidative phosphorylation (OXPHOS). Furthermore, mitophagy, metabolic reprogramming, and the tumor immune microenvironment converge on mitochondrial dynamics to drive the acquisition of drug resistance. Consequently, targeting mitochondrial dynamics presents a promising approach to overcome cisplatin resistance. Potential strategies include restoring the balance of fusion and fission, intervening in mitophagy, disrupting OXPHOS metabolism, and developing mitochondrial-targeted nanodrug delivery systems. However, despite this promising outlook, significant challenges remain, including the heterogeneity of resistance mechanisms, a lack of reliable biomarkers, and the need for selective targeting to minimize off-target effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db056eb8c3db5d2ebbae460e0446276957e89344" target='_blank'>
              Mitochondrial dynamics in cisplatin resistance: molecular mechanisms and therapeutic targeting
              </a>
            </td>
          <td>
            Wei Huang, Wei Sun, Zhiyuan Yang, Yiwen Li, Ziliang Wang
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Deregulation of proteins involved in chromatin regulation is common in pancreatic ductal adenocarcinoma (PDAC). Lysine demethylase 4C (KDM4C) is one of the chromatin-modifying proteins frequently overexpressed across multiple solid cancers and is linked to chromatin instability, increased cell proliferation, and enhanced stem cell–like behavior. We observed upregulation of KDM4C protein in a panel of human PDAC cell lines and patient samples compared with nonneoplastic controls. CRISPR/Cas9-mediated deletion of KDM4C in human and murine PDAC cells reduced proliferation, clonogenicity, and increased survival of orthotopically implanted murine PDAC allografts. Transcriptomic and proteomic analyses revealed that loss of KDM4C in both human and murine PDAC cell lines was associated with the reduction of activated phospho-ERK, a pivotal effector downstream of mutant RAS. Using proximity labeling, we identified the histone deacetylase SIRT1 as a novel interacting protein with KDM4C via the latter’s Tudor reader domain. SIRT1-mediated deacetylation leads to repression of downstream targets, including the dual specificity phosphatase DUSP2, which is known to inactivate ERK via dephosphorylation. In vitro propagation of KDM4C-null PDAC lines eventually led to adaptation and restitution of ERK signaling, with rescue of the KDM4C loss induced growth suppression. To bypass this adaptive phenomenon, we tested a preclinical pan-KDM4 inhibitor TACH107 and confirmed its efficacy in in vitro and in vivo PDAC models. Our studies identify KDM4C as an oncogenic molecule that sustains ERK signaling in KRAS-mutant PDAC and can be broadly targeted via small-molecule inhibitors. Significance: Our data suggests that KDM4C is a novel regulator of ERK signaling, the main effector pathway downstream of mutant RAS. This is the first demonstration linking the requirement of sustained KDM4 activity to ERK signaling in cancer and presents an opportunity to leverage this oncogenic pathway for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9ce02e321041b11b3d0c59cd917566ed5c1973" target='_blank'>
              The Lysine Demethylase KDM4C Is an Oncogenic Driver and Regulates ERK Activity in KRAS-Mutant Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Menna-T-Allah Shaheen, Sarah Dhebat, K. Rajapakshe, Bidyut Ghosh, Benson Chellakkan Selvanesan, Shariq S Ansari, C. Haymaker, Dorsay Sadeghian, Huamin Wang, Ching-Fei Li, H. Ying, Anirban Maitra
          </td>
          <td>2026-01-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Cellular plasticity is crucially important in cancer-induced cell reprogramming, as well as in regeneration therapies in diabetes, Alzheimer's, and Parkinson's diseases. Protein-protein interaction, signaling, and gene regulatory networks are increasingly used to describe plasticity-induced cellular adaptation in disease progression. This review delineates how network analysis of cell plasticity leads to novel therapy options against 1) cancer progression; 2) epithelial-mesenchymal transition-induced metastases; 3) cancer stem cells, and 4) pre-existent drug-resistant cells. Network plasticity-designed sequential and differentiation therapies are also outlined. 55 plasticity-related cancer drug targets are listed, where 20 have already approved drugs, 9 have investigational drugs, and 26 are drug target candidates. The recent expansion of plastic network-driven pancreatic beta cell and neuron regeneration therapies is described in diabetes, as well as in Alzheimer's and Parkinson's diseases, respectively. Finally, six major network-related research gaps and promising future research areas are outlined, including the discovery of plasticity-related cancer signaling pathways and cross-talks, cancer resensitization therapies, and the use of recently available proteome-wide network data and models to find novel cancer cell differentiation cocktails, drug targets, proper timing, and biomarkers of sequential therapies, as well as to perform in silico drug combination screens and in silico clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0feedb6dd227338600bf4a18ec50e34de570d13c" target='_blank'>
              Modulation of Network Plasticity Opens Novel Therapeutic Possibilities in Cancer, Diabetes, and Neurodegeneration.
              </a>
            </td>
          <td>
            Márk Kerestély, István Narozsny, Levente Szarka, D. Veres, Peter Csermely, D. Keresztes
          </td>
          <td>2026-01-31</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary Uveal melanoma is an aggressive ocular malignancy with high mortality rates. Although localized interventions such as surgical resection or radiotherapy effectively control the primary tumor, many patients ultimately develop metastatic disease. Current therapies targeting principal oncogenic drivers often fail to provide durable responses, underscoring the need to elucidate alternative survival pathways that confer resistance. In this study, we systematically evaluated gene function in laboratory models of uveal melanoma, identifying critical contributions from signaling pathways including JAK/STAT, BCL2/BCL-XL, PI3K/mTOR, and Hippo. These results highlight novel molecular mechanisms underpinning therapeutic resistance and support the development of combinatorial treatment strategies. Such approaches may offer improved clinical outcomes for patients with advanced uveal melanoma by effectively targeting both primary oncogenic drivers and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4821b9179d95993f752756f5859601f462e83c03" target='_blank'>
              Unraveling Resistance Mechanisms to Gαq Pathway Inhibition in Uveal Melanoma: Insights from Signaling-Activation Library Screening
              </a>
            </td>
          <td>
            Simone Lubrano, Rodolfo Daniel Cervantes-Villagrana, Nadia Arang, Elena Sofia Cardenas-Alcoser, Kuniaki Sato, Gabriela Cuesta-Margolles, J. Paradis, Monica Acosta, J. Gutkind
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the most commonly diagnosed cancer and the second leading cause of cancer-related death in American men. Men with germline BRCA2 mutations are at especially high risk (19%-61% by age 80) of developing PCa. Loss of BRCA2 impairs homologous recombination repair (HRR), resulting in genomic instability and enhanced tumor progression. Our laboratory previously performed a whole-genome CRISPR-Cas9 screen in a BRCA2 mutant cell line under cisplatin induced stress. This screen identified Chromodomain Helicase DNA-binding protein 1 (CHD1) as the top gene whose loss promoted cell survival under DNA damage stress. CHD1 is a chromatin remodeler that is deleted in approximately 15% PCas and is strongly linked to metastatic disease. However, how its loss drives tumor progression, particularly in BRCA2-deficient PCas, remains poorly understood. To address this, we carried out RNA-seq analysis in castration-resistant prostate cancer (CRPC) cell line PC-3 following knockdown of CHD1 and BRCA2, individually and in combination. Notably, depletion of CHD1 in BRCA2-deficient cells resulted in the upregulation of gene signatures associated with hypoxia and epithelial-mesenchymal-transition (EMT). Functional validation using wound healing and migration assays demonstrated that CHD1 loss significantly enhanced the migration ability of PC-3 cells (including BRCA2-deficient) compared to controls. In keeping with increased EMT, we also see high vimentin expression in CHD1-depleted PC3. Furthermore, Causal Inference Enrichment (CIE) analysis of the transcriptomic data pointed to significant activation of transcription factors (TFs) FOSL1 and SOX9 after CHD1 knockdown compared to control cells. To assess clinical relevance, we analyzed RNA-seq data from the Stand Up To Cancer (SU2C) database. This analysis revealed that CHD1-low tumors exhibited upregulation of multiple oncogenic pathways, especially EMT, angiogenesis, apical junction, and IL2-STAT5 signaling, consistent with our in vitro observations. Finally, genetically engineered mouse model (GEMM)-derived prostate organoids from B2PPt (Brca2flox/flox;P53flox/flox;Ptenflox/flox) and B2CPPt (Brca2flox/flox; Chd1flox/flox;P53flox/flox;Ptenflox/flox) mice, after Adeno-Cre infection, revealed higher Fosl1, along with EMT marker vimentin in B2CPPt organoids compared to B2PPt, supporting FOSL1-dependent lineage plasticity and activation of EMT programs upon CHD1 loss in BRCA2-mutant prostate cells. Collectively, these findings indicate that CHD1 loss, either independently or in the context of BRCA2 deficiency, promotes oncogenic phenotypes through activation of FOSL1-dependent lineage plasticity and EMT programs that favor metastasis. Ongoing work dissects how CHD1 loss reshapes chromatin accessibility to reprogram transcriptional networks and drives oncogenic signaling and metastatic pathways in PCa, with particular relevance to BRCA2-deficient PCa.



 Jingzhu Hao, Shailja Pathania, Miklos Diossy, Kourosh Zarringhalam, Changmeng Cai, Zoltan Szallasi. Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2ea6307c1138357462e3893b183854dd498126" target='_blank'>
              Abstract PR017: Dissecting the role of CHD1 as a key regulator of pro-metastatic transcriptional reprogramming in BRCA2-mutant prostate cancer
              </a>
            </td>
          <td>
            Jingzhu Hao, S. Pathania, M. Dióssy, Kourosh Zarringhalam, Changmeng Cai, Zoltán Szállási
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Patients with congenital heart disease are identified in 1% of live births. Improved surgical intervention means many patients now survive to adulthood, the corollary of which is increased mortality in the over-65-year-old congenital heart disease (CHD) population. In the clinic, genetic sequencing increasingly identifies novel genetic variants in genes related to CHD. Traditional assays for interpreting novel genetic variants are often limited by gene-specificity, whereas animal models are cumbersome and may not accurately reflect human disease. This study investigates CRISPR gene editing in induced pluripotent stem cells and cardiomyocyte-directed differentiation as a human disease model to investigate novel genetic variants identified in association with CHD. We identified a GATA4 p.Arg284His genetic variant in a paediatric patient. This genetic variant was introduced into induced pluripotent stem cells (iPSCs) using CRISPR gene editing with homology-directed-repair. GATA4 genetic variant and isogenic control iPSCs were selected and differentiated into cardiomyocytes. Expression of the GATA4 p.Arg284His variant resulted in altered calcium transients, indicative of CHD and consistent with the patient’s clinical phenotype. Transcriptomics revealed cellular pathway changes in cardiac development, calcium handling, and energy metabolism that contribute to disease aetiology, mechanism and identification of potential treatments. Directed differentiation of iPSCs harbouring the GATA4 p.Arg284His genetic variant recapitulated the CHD phenotype, indicated disease mechanisms, and pointed to potential sites for targeting with therapy. The study highlights the utility of transcriptomics for the functional interpretation of cardiac genetic variants and is an exemplar for precision medicine approaches for the investigation of CHD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7301261a9b5117cfbb952246e659818f7c79a156" target='_blank'>
              A precision medicine approach to interpret a GATA4 genetic variant in a paediatric patient with congenital heart disease
              </a>
            </td>
          <td>
            Catherine A. Forbes, Nicole C. Shaw, Kevin Chen, Mitchell Hedges, Teagan S. Er, Livia Hool, Michelle Ward, C. Poulton, Gareth Baynam, Timo Lassmann, Vanessa S. Fear
          </td>
          <td>2026-01-03</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cb28bdcaf79e7d74e6f5d2dbc03edf13bb5c93" target='_blank'>
              SMAD4 loss drives cancer chromosomal instability through transcription-replication conflict-induced replication stress
              </a>
            </td>
          <td>
            Atmika Paul, Gargee Joshi, Philipp Jungk, Simranjeet Kaur, Shengyu Yao, Ioannis Tsirkas, N. Böhly, Maren Sitte, Gabriela Salinas, Stephan Hamperl, Maik Kschischo, Holger Bastians
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Breast cancer remains a leading cause of cancer-related mortality worldwide, with radiotherapy serving as a cornerstone of treatment. However, the development of radioresistance significantly compromises therapeutic efficacy and patient outcomes. Through integrative analysis of TCGA and GEO datasets combined with quantitative proteomics, we identified acetyl-CoA synthetase 1 (ACSS1) as a key driver of radioresistance in breast cancer. ACSS1 is aberrantly overexpressed in breast cancer and correlates with poor prognosis following radiotherapy. Functional studies revealed that overexpressed ACSS1 is able to enhance radioresistance both in vitro and in vivo. Mechanistically, ACSS1 amplifies the ionizing radiation (IR)-induced metabolic coupling of pyruvate with ROS for acetate synthesis, which fuels energy production and expands the nuclear acetyl-CoA pool, enabling histone acetylation at DNA damage sites. Such acetylation promotes chromatin relaxation at damage sites, facilitating the recruitment of homologous recombination (HR) repair machinery and ultimately leading to radioresistance. Our findings reveal that ACSS1 orchestrates acetyl-CoA-driven histone acetylation to enhance DNA repair efficiency, highlighting a metabolic-epigenetic crosstalk that sustains radioresistance in breast cancer. Targeting ACSS1 represents a promising therapeutic strategy to sensitize tumours to radiotherapy and improve clinical outcomes in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/838ec4e17eb487298059e675586feb53fff4f6af" target='_blank'>
              ACSS1 co-opts acetyl-CoA metabolism to drive DNA repair and undermine radiotherapy efficacy in breast cancer
              </a>
            </td>
          <td>
            Xiao Fu, Yingyu Zhu, Xin Lu, Xingjie Gao, L. Xin, Yuanyuan Ren, Xin Liu, Lin Ge, Jihui Hao, Zhi Yao, Lei Shi, Jie Yang
          </td>
          <td>2025-12-18</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a387e7c68c6390622f2c2d1c86f9db7afbe8e8ce" target='_blank'>
              miRISC inhibition causes mitotic defects and synergizes with genotoxic agents in cancers
              </a>
            </td>
          <td>
            Minsi Zhang, Ziqi Jiao, Ylenia Cendon Florez, Ivana Lessel, Xiaoyi Li, Melissa A. Yao, Moritz Weigl, Mateusz M. Ozimek, Turgut Dogruluk, Kevin Chen, Keith Conrad Fernandez, Vinagolu K. Rajasekhar, Olesja Popow, João A. Paulo, Davide Pradella, Tanmay Mishra, C. Mastroleo, Juliana I. Delgado, Francesco Enrico D’Amico, Vincenzo Cavalieri, Carol D. Morris, M. B. Tsou, Kevin M. Haigis, Jayanta Chaudhuri, R. Benezra, Joana A. Vidigal, Davor Lessel, Andrea Ventura, Gaspare La Rocca
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Genomic instability is a hallmark of nearly all human cancers, yet the genetic regulators that govern this instability, their functional roles, and their therapeutic potential remain incompletely understood in breast cancer. In this study, we conducted a comprehensive analysis of the expression profiles of genomic instability-maintaining genes (GIMGs) in breast cancer and revealed that elevated expression of GIMGs was associated with adverse clinical outcomes and an altered tumor immune microenvironment. Through further analysis, we identified a core set of GIMGs and pinpointed RPGRIP1L as a critical driver of genomic instability in breast cancer. Notably, RPGRIP1L expression was significantly upregulated in breast cancer tissues and strongly correlated with poor patient prognosis. Functional studies demonstrated that RPGRIP1L knockdown reduced genomic instability in tumor cells, as evidenced by decreased Phosphorylated Histone H2AX (γH2AX) expression, and suppressed tumor growth in mouse models. Additionally, high RPGRIP1L expression was linked to elevated expression of immune checkpoint Programmed Death Ligand 1(PD-L1) in human breast cancer samples. Mechanistically, we found that RPGRIP1L promoted the activation of the Hedgehog signaling pathway, which in turn drived tumor proliferation and upregulated PD-L1 expression. Collectively, these findings highlight RPGRIP1L as a key genomic instability-maintaining gene in human breast cancer, offering critical insights into the molecular mechanisms underlying disease progression. Furthermore, targeting RPGRIP1L may represent a promising therapeutic strategy for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c08de5eb400d357e336428d7f7a366a3a038cb3" target='_blank'>
              Identification of RPGRIP1L as an instability-maintaining gene to drive tumor growth and PD-L1 expression via Hedgehog signaling in breast cancer
              </a>
            </td>
          <td>
            Meng Lv, Yan’e Liu, Weihong Cao, Yongmei Wang, Qi Wang, Xue-Qiang Gao, Baowei Peng, Hai-bo Wang, Yan Mao
          </td>
          <td>2025-12-30</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly effective utilization of PARP inhibitors. However, the variability of different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to use the whole genome landscape to better define HRD in a pan-cancer cohort. We employed a whole genome sequencing HRD classifier that includes genome-wide signatures associated with HRD to analyze 580 tumor/normal paired samples. The HRD phenotype was correlated with genomic variants in BRCA1/2 and other homologous recombination repair genes. In this paper we show that the HRD phenotype is identified in various cancers including breast (21%), pancreaticobiliary (20%), gynecological (17%), prostate (9%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases are not confined to BRCA1/2 mutations; 24% of HRD cases are BRCA1/2 wild-type. A diverse range of gene alterations involved in HRD are elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. In a subset of cases, the whole genome sequencing-based classifier offers more insights and a better correlation to treatment response when compared to other assays. Although HRD is a biomarker used to determine which cancer patients would benefit from PARP inhibitors, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive whole genome sequencing analysis to better predict HRD and elucidates genomic mechanisms associated with this phenotype. Homologous recombination deficiency is a condition in which a cancer cell cannot repair certain types of DNA damage. It causes genetic instability and is often due to changes in parts of the DNA called genes, such as BRCA1 and BRCA2. Cancers with this deficiency can be more readily killed by certain drugs that prevent DNA repair. Some of these drugs are approved for the treatment of several types of cancer, including ovarian, breast, pancreatic, and prostate cancers. To better identify tumors with this deficiency, we characterize the whole genome of cancer samples. We find that a comprehensive analysis of the entire genome improves the detection of homologous recombination deficiency. This type of analysis may provide a more accurate way to guide treatment decisions for people with cancer. Assaad, Hadi and Levine et al. develop a whole-genome sequencing classifier to improve the detection of homologous recombination deficiency (HRD) across a pan cancer cohort. The classifier detects HRD beyond BRCA1/2 mutations, reveals HRD-related genomic events, and correlates with treatment response in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/108bfd65e5fc2e36aa5d310bd5ebd99b171e9621" target='_blank'>
              Whole genome sequencing approach to assess homologous recombination deficiency in a pan-cancer cohort
              </a>
            </td>
          <td>
            Majd Al Assaad, Kevin Hadi, Max F. Levine, Daniela Guevara, Minal Patel, Marvel Tranquille, Abigail King, John Otilano, Alissa Semaan, G. Gundem, J. Medina-Martinez, M. Sigouros, J. Manohar, Hui-Hsuan Kuo, D. C. Wilkes, Eleni Andreopoulou, E. Chapman-Davis, S. Tagawa, A. Sboner, Allyson J. Ocean, Manish A. Shah, E. Papaemmanuil, Cora N. Sternberg, Kevin M. Holcomb, D. Nanus, O. Elemento, Juan Miguel Mosquera
          </td>
          <td>2026-01-12</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d8e985c420ff8b3595d58e817f6b4a4d574874" target='_blank'>
              ButterflyVI: enabling high-throughput variant interpretation and biomarker discovery with functional genomics
              </a>
            </td>
          <td>
            Debora Sesia, András Hatos, P. Nittis, G. Ciriello
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite significant therapeutic advances, multiple myeloma (MM) remains incurable in most patients due to frequent tumor relapse. A major contributor to relapse is clonal heterogeneity, where subclones exhibit distinct mechanisms of therapy resistance, along with the presence of drug-resistant dormant cells. Eliminating these distinct populations, which often coexist in the tumor niche, is clinically challenging. Identifying survival mechanisms shared by drug-resistant proliferating and dormant cells holds potential for the simultaneous elimination of different tumor-repopulating clones. To identify shared mechanisms of therapeutic resistance, we analyzed clinical databases and drug-resistant myeloma cell lines. We employed pharmacologic approaches to target common candidates identified in our analysis and assessed their impact on tumor progression and survival in preclinical mouse models containing both therapy-resistant and dormant cells. We identified upregulation of several components of the Notch signaling pathway in both dormant and drug-resistant MM cells, which correlated with poor clinical outcomes in newly diagnosed MM patients. Selective blockade of NOTCH3 with a neutralizing antibody or pan-Notch inhibition with a bone-targeted inhibitor reduced tumor burden and eliminated coexisting dormant and bortezomib-resistant cells in clinically relevant models of MM disease. Our findings reveal that NOTCH3-dependent survival programs represent a shared vulnerability in both cells refractory to therapy and dormant cells. These programs can be exploited to overcome the diverse mechanisms by which cancer cells evade therapy, potentially preventing disease relapse and extending remission in patients with MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f4eb8f9fdb1cae4aedbf1364db151a77768c687" target='_blank'>
              Targeting NOTCH3 to eradicate dormant and therapy-resistant multiple myeloma cells
              </a>
            </td>
          <td>
            H. Sabol, Bethany C. Paxton, A. Anloague, Japneet Kaur, Mattie R. Nester, Sharmin Khan, James Smith, Peter I Croucher, Michelle M McDonald, Corey O Montgomery, J.B. Stambough, C. L. Barnes, E. Ambrogini, F. H. Ebetino, C. Schinke, Cody Ashby, J. Delgado-Calle
          </td>
          <td>2026-01-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/504ab75add2450e8f7389e1bac7e5c84edbac572" target='_blank'>
              System-wide profiling reveals metabolic alterations in a G2019S LRRK2 knockin mouse model of Parkinson’s disease: systemic depletion of pyrimidine nucleosides
              </a>
            </td>
          <td>
            Yue Ma, Madalynn L. Erb, Kayla Sipple, Alina V. Offerman, Darren J. Moore
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The development of resistance to chemotherapy is one of the main causes of treatment failure in malignant tumors. Despite advances in the creation of new anticancer agents, many tumors remain refractory to standard regimens. Key mechanisms of resistance include active drug efflux via ABC transporters, enhanced DNA repair, evasion of apoptosis, therapy-induced autophagy, epigenetic alterations, metabolic reprogramming, the presence of cancer stem cells, and the influence of the tumor microenvironment. This review discusses the molecular basis of these processes and contemporary approaches to overcoming them: ABC transporter inhibitors, targeted and immunotherapeutic agents, DNA repair inhibitors, autophagy blockers, epigenetic drugs, metabolic targets, strategies for eliminating cancer stem cells, nanotechnology-based delivery systems, as well as novel forms of programmed cell death (ferroptosis, cuproptosis, pyroptosis).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a9ae02e92096be80f406876ce9572393fe3bd5" target='_blank'>
              Mechanisms of chemotherapy resistance and strategies to overcome it
              </a>
            </td>
          <td>
            R. A. Abramenko, A. R. Gasanov, B. M. Babatova, D. A. Abdramanova, A. D. Abdullatipov
          </td>
          <td>2025-12-18</td>
          <td>International journal of Innovative Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study aimed to investigate the molecular heterogeneity, accurately evaluate the prognosis, and identify those patients who would benefit most from immunotherapy in the hypermutated colorectal cancer (CRC) cohort. The unsupervised clustering analysis of CRC somatic mutation data from the Cancer Genome Atlas database was performed by combining tumor mutational burden (TMB) and indel burden to identify the subtypes within the hypermutated CRC cohort. Then, the somatic mutations, transcriptome, and gene mutations in the DNA damage repair signaling pathway of these subtypes were compared to obtain the corresponding molecular features. The proportion of 22 immune cells infiltrated in the samples was calculated using the CIBERSORT algorithm, and immunotherapeutic sensitivity was predicted for the patients using immunophenoscores (IPS). Five distinct clusters were identified from the hypermutated CRC sample, each exhibiting different characteristics regarding somatic mutations, gene expression, immune infiltration, and prognosis. The proportion of M2 macrophage infiltrated in Cluster 6 was significantly higher than in Cluster 1 (P = .018) and Cluster 3 (P = .003). The proportion of infiltrating M1 macrophages was significantly higher in Cluster 3 than in Cluster 1 (P = .026) and Cluster 6 (P = .016). Cluster 1 had a longer OS time compared to Cluster 3 (P = .015) and Cluster 6 (P = .005). In all 4 IPS types, Cluster 3 demonstrated higher IPS than Cluster 6 (P < .05), Cluster 2 demonstrated lower IPS than Cluster 5 (P < .05). In the IPS-CTLA4 type, the IPS of Cluster 1 was significantly higher than that of Cluster 6 (P = .038). In the non-POLE mutant cohort, the patients with the lowest TMB and indel burden had longer overall survival. The patients with higher levels of M1 macrophage infiltration showed higher sensitivity to anti-PD-1/PD-L1 immunotherapy or combined anti-PD-1/PD-L1 and anti-CTLA4 immunotherapy. In the POLE mutant cohort, the patients from the low TMB cohort may be better candidates for immunotherapy than those from the higher TMB cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec135f15fc8160351aa36efc3ceee1240577672" target='_blank'>
              Comprehensive analysis reveals the molecular heterogeneity and the differences in prognosis and immunotherapy sensitivity in hypermutated colorectal cancer
              </a>
            </td>
          <td>
            Gaobo Bai, Xiaojun Yang, Liang Xiao, Chunhui Xiong
          </td>
          <td>2026-01-09</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lynch syndrome (LS) is a genetic predisposition to mainly colorectal and endometrial cancer due to heterozygous disruptive germline mutations in the DNA mismatch-repair (MMR) genes MSH2, MSH6, MLH1, or PMS2. Beyond clearly pathogenic mutations, germline sequencing often reveals variants of uncertain significance (VUS), predominantly single base-pair alterations in coding or noncoding regions. These uncertain variants obstruct LS diagnosis, hampering personalized surveillance. To address this challenge, we developed a highly accurate functional assay that interrogates VUS pathogenicity in human cells. Building on a mouse-based cellular assay, we adapted oligonucleotide-directed mutation screening (ODMS) for human cells and introduced a refined approach named "coselection ODMS." To ensure physiological expression, the variant is introduced into the endogenous MMR gene by replication-coupled gene editing. Coselection ODMS demonstrated 100% accuracy in classifying 50 benign and 86 pathogenic variants spanning coding and noncoding regions in all four MMR genes. Among 109 patient-derived VUS, 51 were identified as deleterious for MMR function. Importantly, coselection ODMS delivered 100% concordant results in a clinical diagnostic laboratory. With >93% sensitivity and >92% specificity, coselection ODMS provides a highly reliable functional assay in the diagnosis of enigmatic LS variants, enabling risk assessment and personalized surveillance or treatment for affected families.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04046716ba1ab699ab383d6687b388d0e98d095b" target='_blank'>
              An accurate cellular assay to determine pathogenicity of coding and noncoding variants in Lynch syndrome genes.
              </a>
            </td>
          <td>
            Iris E. Glykofridis, Marleen Dekker, Chantal Stoepker, Thomas W. van Ravesteyn, Y. Tiersma, Cédric G van der Ham, Beaunelle de Bruijn, Salma Ebrahim, Renée X de Menezes, E. Kasteleijn, Frans Verheijen, T. V. van Ham, H. te Riele
          </td>
          <td>2026-01-16</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT Aging is a dominant risk factor for chronic diseases characterized by the functional decline of tissues and organs. During aging, the hematopoietic system declines in regenerative capacity—seemingly attributable to increases in DNA damage, replicative stress, and autophagic flux—resulting in skewing towards a myeloid lineage and away from a lymphoid lineage. Here, we characterized the transcriptomic and cellular landscape of the aged C57Bl/6J mouse hematopoietic system using a combination of bulk RNAseq and single cell RNAseq (scRNAseq). We show that aging leads to global transcriptional alterations in bulk peripheral blood mononuclear cells (PBMCs), lineage marker‐depleted bone marrow cells (Lin‐BM), and in hematopoietic stem and progenitor cells (HSPCs), immunophenotypically lineage marker negative (Lin‐) Sca1+ cKit+ (LSK+). These changes indicate widespread activation of inflammatory processes, namely in PBMCs and Lin‐BM cells. Interestingly, there is also a downregulation of cell cycle genes in HSPCs during aging. ScRNAseq across 39 hematopoietic cell types revealed age‐related skewing in cell composition. Aged PBMCs showed significant decreases in CD4 and CD8 naïve cells concomitant with increases in CD4/8 memory and CD8 exhausted T cell populations. Lin‐BM cells showed significant myeloid skewing in common myeloid progenitor (CMP) cells, as well as in the HSC population. We also identified a unique HSC population marked by increased Vwf, Wwtr1, and Clca3a1 expression that does not exist in young HSCs, thus likely marking true aged HSCs. Collectively, this work should serve as a useful resource for understanding and therapeutically targeting the aged hematopoietic system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19cc88f7b5c2f02b7b252356ac23e9d207486aa" target='_blank'>
              A Cellular and Transcriptomic Atlas of the Aged Mouse Hematopoietic System
              </a>
            </td>
          <td>
            Ryan R. White, Kun Xiong, Matthew H. Wakai, Allison Surian, Christina Adler, Nicole Negron, Min Ni, T. Shavlakadze, Yu Bai, David J. Glass
          </td>
          <td>2026-01-28</td>
          <td>Aging Cell</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The contribution of mitochondrial genetics to heart failure is thought to be reciprocal. Its complex traits are influenced by genetic and environmental factors. Genetically diverse mouse strains form a vital repository to uncover the interaction between the mitochondrial and nuclear genomes underlying heart failure due to their traceable genetic origins, maternal lineages and controlled genetic variation. Using systems genetics, we studied this cross-talk in the Collaborative Cross (CC) mice challenged with heart failure (HF). We used 63 strains of CC-mice that grouped into 8 mitochondrial haplotypes and subjected them to HF using isoproterenol (Iso), a beta-adrenergic stimulant that mimics progressive stress induced HF in humans. The Alzet osmotic pumps delivered a consistent dose of the drug for 21 days following which the mice were euthanized for organ collection. A group of mice with saline loaded pumps acted as control (Ctrl). Baseline and end of study echocardiography were recorded for these mice. Bulk RNA sequencing was carried out on the left ventricles and data analyzed on R-studio. The CC-strains showed differences across the haplotypes for organ weights and heart function. Noticeable treatment specific gene expression differences were observed for 34 nuclear encoded genes from MitoCarta3.0 unlike mt-DNA encoded genes that were insignificant after correcting for sex and haplotypes. These genes were associated with multiple metabolic pathways in the cell. Trait-GWAS associations with markers from the mitochondrial genome were observed only for mice from Iso group with cell surface area (p = 6.19e-06) and change in ejection fraction (p = 9.22e-05) as top hits under FWER threshold of 0.01. Cyfip2 was the top candidate gene (p = 2.66e-07) among the 831 hits (47-MitoCarta & 784 other nuclear) based on our trans-eQTL analysis. The eQTL genes influenced critical pathways of OXPHOS, myogenesis, apoptosis etc. Our approach uncovered 24 gene candidates associated with mt-DNA and HF that overlapped with our previous mi-eQTL reports. CC-mice revealed ancestry dependent effects underlying HF for studying mito-nuclear interactions. Despite establishing differences, modeling these interaction needs development of complex cybrid systems to evaluate the bidirectional impact on HF. Author Summary The Collaborative Cross (CC) mouse is a genetically diverse population that has been used to study complex diseases. Recently, our group has comprehensively characterized the heart from 63 strains of the CC and reported genetic associations with heart failure (HF). Traditionally, genetic abnormalities underlying a disease are related to the nuclear genome but there exists an alternate genome within the cells, the mitochondrial DNA (mt-DNA), whose contribution is understudied. The mt-DNA of CC mice is maternally derived from the 8 founder strains, and our sequencing data holds this information, allowing us the ability to use them to study the contribution of mitochondrial ancestry (haplotypes) to HF. We found haplotype differences in terms of cardiac function and gene expression (both nuclear and mitochondrial) that were associated with HF. It revealed stress induced changes in the heart (trait and gene expression) linked with regions in the mt-DNA that encode genes involved in oxidative phosphorylation and metabolism. We uncovered 24 high-confidence nuclear gene candidates that agreed with our previous analyses and were associated with regions on the mt-DNA. These findings open up new opportunities to study the CC and advance the understanding of Mito-nuclear interactions in HF pathophysiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b11197c1c5783c21dda29005834d7d9a3d7da82" target='_blank'>
              A systems genetics approach to uncover mitochondrial drivers of heart failure reveals mitochondria-nuclear cross talk in genetically diverse mouse strains
              </a>
            </td>
          <td>
            Sriram Ravindran, Todd Kimball, Brian Gural, Caitlin Lahue, Christoph D. Rau
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99817029794cbefbdaf247b4f46b15275fd31679" target='_blank'>
              Metabolic reprogramming in non-small cell lung cancer: unraveling therapeutic vulnerabilities and resistance mechanisms
              </a>
            </td>
          <td>
            T. Addissouky
          </td>
          <td>2026-01-17</td>
          <td>Journal of Rare Diseases</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The regenerative response of retinal cells to injury and aging depends on the epigenomic plasticity that enables the dedifferentiation and neuronal differentiation capacities of Müller glial cells (MG). In mammals, this regenerative ability is extremely limited, and disruptions in epigenetic mechanisms, particularly those involving DNA methylation and demethylation, may underlie this restricted potential. To explore this possibility, we aimed to develop DNA methylation-targeting molecular tools to enhance the dedifferentiation and neurogenic capacity of primary MG cultures derived from mouse retina. Using CRISPR/dCas9-based gene regulation technology, we selectively and transiently inhibited Dnmt3a, a de novo DNA methyltransferase previously implicated in maintaining transcriptional repression, for approximately 3–5 days. Our results show that Dnmt3a knockdown leads to sustained upregulation of pluripotency-associated genes, including Ascl1, Lin28, and Nestin, as measured by RT-qPCR and immunofluorescence. This epigenetic modulation also promoted increased cell proliferation and migration, both hallmarks of a regenerative response. Furthermore, Dnmt3a knockdown, either alone or in combination with neurogenic stimuli, induced MG to acquire neuronal-like morphologies and express the early neuronal marker βIII-tubulin. These findings suggest that Dnmt3a acts as a repressive regulator of MG plasticity, likely serving as an epigenetic barrier that counteracts injury-induced demethylation events. Overall, our study identifies Dnmt3a as a critical modulator of MG fate and highlights the potential of its targeted downregulation to facilitate reprogramming. By prolonging the transient progenitor-like state of MG, DNMT3a inhibition may serve as a complementary approach to unlock the neurogenic and regenerative potential of the mammalian retina, offering promising avenues for future therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3571121818a753d9291146efaf83359b38f99a" target='_blank'>
              Targeted knockdown of DNA methyltransferase 3a (DNMT3a) unlocks dedifferentiation and neurogenic potential in mouse retinal Müller glia
              </a>
            </td>
          <td>
            Rebeca Victoria-Chavez, Mónica Lamas
          </td>
          <td>2025-12-17</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bladder urothelial carcinoma (BUC) is one of the most common malignant tumors of the urinary system, and its drug resistance often leads to treatment failure and high mortality. Given the crucial role of tumor aerobic glycolysis in regulating drug resistance in BUC, elucidating its mechanisms is promising for developing novel therapeutic strategies against this clinical challenge. In this review, we provide a systematic description of the specific mechanisms through which tumor aerobic glycolysis regulates drug resistance of BUC, including improved DNA damage repair mechanisms, regulation of epigenetic changes, induction of tumor immune evasion, activation of resistance-promoting downstream mechanisms and increased drug efflux. Additionally, we discuss the potential of tumor aerobic glycolysis for clinical diagnosis and treatment of BUC. Tumor aerobic glycolysis regulates drug resistance in BUC through multiple pathways. Future research should prioritize the development of biomarkers and targeted therapies against key enzymes and metabolites within these pathways to improve the diagnosis, treatment, and prognosis of drug-resistant BUC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50df8a8623f30bb1b92c7628e084c46bfe9c8fdf" target='_blank'>
              Regulation of drug resistance in bladder urothelial carcinoma by tumor aerobic glycolysis
              </a>
            </td>
          <td>
            Can Weng, Hao Deng, Ziyang Yang, Xi Zeng, Haoxuan Xu, Jiuru Li, Chunping Yu
          </td>
          <td>2025-12-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) continues to be a representative malignancy globally, with its inherent heterogeneity and chemotherapy resistance posing significant therapeutic obstacles. Doxorubicin (Dox), a commonly utilized chemotherapeutic agent, is often constrained by dose‐dependent toxicities and acquired resistance, which ultimately lead to suboptimal clinical outcomes in numerous BC patients. Co‐treatment of Dox with RSL3, a small‐molecule compound that inactivates glutathione peroxidase 4 (GPX4), was shown in this study to synergistically inhibit BC cell viability and proliferation at low doses in vitro and in vivo. In vitro tests showed that RSL3 exacerbated Dox‐induced cell cycle arrest and apoptosis, markedly potentiating the cytotoxicity of Dox against BC cells. Mechanistically, RSL3 triggered ferroptosis‐dependent ROS accumulation, leading to glycolytic impairment characterized by reduced intracellular ATP and pyruvate concentrations levels and downregulation of HKII, PFKP, and PKM2. Supplementation with exogenous sodium pyruvate ameliorated RSL3‐induced glycolytic dysfunction, restored the changes in apoptosis‐related protein expression caused by RSL3/Dox co‐treatment, and reduced RSL3/Dox‐induced cell apoptosis. Besides, according to in vivo studies, the combination of RSL3 and Dox further suppressed BC tumor growth without inducing significant adverse effects. On these accounts, RSL3 makes BC cells more sensitive to Dox by inducing glycolytic dysfunction, implying that RSL3/Dox co‐treatment could be a viable therapeutic approach for BC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747146a4b35fc36042a81b955c0139570c0040b9" target='_blank'>
              RSL3 Mitigates Doxorubicin Resistance in Breast Cancer via Targeted Disruption of Glycolytic Flux
              </a>
            </td>
          <td>
            Mengxin Li, Xuanzhong Wang, Jinghui Hong, Xin Guan, Yichang Chen, Hongxu Zhou, Xin Wang, Baoling Liang, Xiaochuan Gao, Yuheng Wu, Dong Song
          </td>
          <td>2025-12-17</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chemoresistance remains a major barrier in cancer therapy, frequently resulting in treatment failure and reduced patient survival. This multifaceted phenomenon arises from the interplay of well-established mechanisms such as genetic mutations, non-genetic adaptations, and tumor microenvironment (TME) mediated influences as well as newly emerging findings from recent research (2020-present). Key biochemical contributors include diminished intracellular drug accumulation through altered uptake or efflux, dysregulation of drug metabolism and bioactivation involving multiple Phase I and Phase II enzymes, genomic instability affecting DNA repair pathways, disruption of cell cycle control, and evasion of apoptosis. In addition, recent evidence highlights the roles of epigenetic reprogramming, metabolic reconfiguration, and TME-derived signaling in amplifying chemoresistance. This review integrates both foundational concepts and recent advancements in understanding drug resistance, with particular emphasis on updated insights into drug-metabolizing enzymes and their impact on therapeutic failure. It also evaluates current and emerging strategies to overcome resistance including targeting metabolic enzymes, modulating the TME, and implementing polytherapy's that address multiple resistance pathways. By synthesizing established knowledge with recent discoveries, this review highlights promising directions for improving the efficacy of cancer treatments and enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92ce9c249ee2d1807438a427a4bdf1ac44ab032" target='_blank'>
              Molecular basis of cancer chemoresistance: biochemical insights.
              </a>
            </td>
          <td>
            R. Guneidy
          </td>
          <td>2026-01-26</td>
          <td>Drug metabolism reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/433ea7c0b515392b57a3190c314827cc2576b6c4" target='_blank'>
              A Novel Patient-Derived Mutation in Glucocorticoid Receptor Reveals an Essential Role for DNA binding in Development and Endocrine Regulation
              </a>
            </td>
          <td>
            T. Tettey, Qizong Lao, Sonal Vaid, Sohyoung Kim, L. Varticovski, Vipula Kolli, Deborah P. Merke, Gordon L. Hager
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd77a79644efdb57765d420fb7be18f39a993472" target='_blank'>
              Cancer-associated synonymous mutations reveal stress signal-dependent mRNA folding that selectively modulates protein function
              </a>
            </td>
          <td>
            Sivakumar Vadivel Gnanasundram, Lixiao Wang, Sa Chen, Ondrej Bonczek, B. Vojtěšek, Robin Fåhraeus
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Detecting somatic mutations in normal tissues is challenging due to sequencing errors and the low allele fractions of post-zygotic variants. Duplex sequencing greatly reduces errors and can detect mutations at any allele fraction, but systematic, cross-platform comparisons are lacking. We present a comprehensive benchmarking of six duplex sequencing technologies used by the SMaHT Network: CODEC, CompDuplex-seq, HiDEF-seq, NanoSeq, ppmSeq, and VISTA-seq. We evaluated their performance using cord blood DNA, a tumor-normal cell line mixture, and homogenates from six human tissues. Each method shows distinct profiles in genomic footprint, sensitivity, and cost. Despite differences in library construction and sequencing platforms, estimates of mutation rates and mutational signatures are highly concordant. Integration with ultra-deep whole-genome sequencing shows that duplex approaches sensitively capture mutations and signatures beyond embryonic or clonally expanded variants. These results provide a foundation for selecting duplex methods and interpreting their data, enabling scalable single-molecule analyses of somatic mutation landscapes. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d72f7e31a7995a205874cfe4a0b03ef948514059" target='_blank'>
              Benchmarking of duplex sequencing approaches to reveal somatic mutation landscapes
              </a>
            </td>
          <td>
            Yang Zhang, Vinay Viswanadham, Michail Andreopoulos, Dominik Glodzik, Ruolin Liu, Lovelace J. Luquette, Se-Young Jo, Azeet Narayan, Muchun Niu, Lisa Anderson, Joseph A. Brew, H. Chao, C. Cibulskis, Guanlan Dong, U. S. Evani, William C. Feng, Marta Grońska-Pęski, Adrienne Helland, N. Hilal, Nisrine Jabara, Hu Jin, Ning Li, Monica D. Manam, Shayna Mallett, Alexi M. Runnels, Constantijn Scharlee, Carter Smith, Diane D. Shao, Christopher A. Walsh, V. Adalsteinsson, E. Lee, Peter J. Park, K. Ardlie, S. Germer, Richard A. Gibbs, S. Choudhury, Harsha Doddapaneni, Gilad D. Evrony, Chenghang Zong, Tim H. H. Coorens
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>37</td>
        </tr>

        <tr id="Chemoresistance remains a major obstacle to effective cancer treatment, often driven by enhanced DNA repair mechanisms that enable tumor cells to withstand genotoxic therapies. One such pathway involves the atypical DNA damage repair complex ALKBH3–ASCC, activated by the E3 ligase RNF113A in response to alkylation damage. We previously showed that SMYD3-dependent methylation of RNF113A stimulates this pathway, enhancing DNA repair and promoting resistance. Here, we identify KDM7B/PHF8 as the bona fide RNF113A demethylase, establishing one of the first functional examples of a dynamic, reversible non-histone methylation event regulating genome integrity. KDM7B antagonizes SMYD3 activity by maintaining low levels of methylated RNF113A, thereby limiting ASCC activation and sensitizing cancer cells to alkylating agents. To dissect this regulation in depth, we focused on small cell lung cancer (SCLC), a particularly aggressive malignancy characterized by limited therapeutic options and rapid acquisition of resistance. In SCLC, high KDM7B levels correlate with improved patient prognosis, whereas xenografts with reduced expression exhibit diminished responses to alkylating treatment. Moreover, CRISPR-based on/off modulation of KDM7B in genetically engineered SCLC mouse models demonstrates its central role in determining tumor response to chemotherapy. Our findings position the RNF113A–ASCC axis as a central modulator of chemoresistance, regulated through a post-translational methylation switch representing an innovative therapeutic vulnerability that could be exploited to enhance the efficacy of alkylating agents. Targeting this pathway may provide new opportunities to overcome chemoresistance, with KDM7B levels serving as a predictive biomarker to guide treatment in SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fadb48eb1c5f8acecabf2f1b43e895eccc0dda31" target='_blank'>
              KDM7B-mediated demethylation of RNF113A regulates small cell lung cancer sensitivity to alkylation damage
              </a>
            </td>
          <td>
            Tanveer Ahmad, Xiaojie Yang, A. Foucher, Lingnan Ren, Ning Tsao, Natasha M. Flores, Jinkai Wan, L. Belmudes, E. Dubiez, Monika Chandan Bhowmik, Jessica Vayr, Simone Hausmann, Florent Chuffart, Xiaoyin Lu, Sandrine Blanchet, Tourkian Chasan, F. Boussouar, Yohann Couté, N. Mosammaparast, Fei Lan, Jan Kadlec, Pawel K. Mazur, N. Reynoird
          </td>
          <td>2026-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Cervical cancer remains a major global health burden. Despite standard-of-care therapies, 30–50% of locally advanced-stage patients develop treatment resistance, leading to recurrence and mortality. While tumour-intrinsic mechanisms (e.g., DNA methylation, cancer-associated fibroblasts) explain only partial resistance heterogeneity, emerging evidence identifies the microbiome as a critical modulator of therapeutic efficacy. This review synthesizes recent advances demonstrating that vaginal microbial dysbiosis, characterized by Lactobacillus iners enrichment and L. crispatus depletion, drives resistance through lactate-mediated metabolic rewiring, immune checkpoint stabilization and drug metabolism alteration. Longitudinal studies reveal dynamic microbiome trajectories during therapy, with geographic variations (notably HIV co-infection in sub-Saharan Africa) further modulating treatment responses. We critically evaluate microbiome-based interventions, from probiotics to engineered bacteria, including synthetic biology-driven precision microbiome therapies, and establishing standardized multi-centre trial protocols. Bridging mechanistic insights with clinical application represents a paradigm shift towards microbiome-informed cervical cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a822ae4e52835707e7a0923e3a8688d6920b131" target='_blank'>
              Microbiome-driven resistance in cervical cancer therapy: from mechanistic dissection to clinical translation
              </a>
            </td>
          <td>
            Xiaoqi Wu, Dingjie Wang, Xiaodan Mao, Binhua Dong, Yue Wang
          </td>
          <td>2025-12-29</td>
          <td>Expert Reviews in Molecular Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Intratumor heterogeneity (ITH), a critical driver of tumor evolution and immune evasion, remains inadequately characterized at the transcriptomic level in colorectal cancer (CRC), and its clinical implications are not yet fully understood. Methods We integrated transcriptomic datasets from TCGA-COAD/READ and two independent GEO cohorts (GSE40967 and GSE87211) to develop an RNA-seq–based ITH score using the DEPTH2 algorithm and to con0struct an ITH-related gene (ITRG) prognostic model. A unified cutoff value of 0.64 was established to stratify patients into high- and low-ITH groups. Using 52 survival-associated ITRGs, we generated a nine-gene prognostic signature selected from 101 distinct combinations of feature selection techniques and modeling algorithms and validated its performance in two external datasets. The tumor microenvironment and potential responsiveness to immune checkpoint inhibitors were evaluated using ssGSEA, ESTIMATE, and TIDE algorithms. SHAP analysis, together with in vitro and in vivo experiments, was employed to identify and functionally validate key regulatory genes. Results Patients in the high-ITH group had markedly poorer overall survival (OS) than those in the low-ITH group. The ITH score correlated strongly with aggressive clinical features, including T3/4 invasion depth, N1/2 nodal status, and AJCC stage III. The nine-gene prognostic signature demonstrated consistent predictive capability across the TCGA training cohort and both GEO validation cohorts. In TCGA, the model yielded time-dependent AUCs of 0.669, 0.664, and 0.645 for 1-, 3-, and 5-year OS, respectively, and retained its status as an independent prognostic indicator in multivariate Cox regression analysis; in GSE40967 and GSE87211, the corresponding AUCs ranged from 0.544–0.573 and 0.648–0.744. The high-risk subgroup was characterized by a stromal immune phenotype enriched with cancer-associated fibroblasts and macrophages, elevated expression of multiple immune checkpoints, increased TIDE scores, and higher tumor mutational burden. SHAP analysis identified IL20RB as the top risk-associated gene, whose knockdown significantly suppressed CRC cell proliferation, migration, invasion, and tumorigenicity in vitro and in vivo. Conclusion This study introduces and validates a transcriptomic ITH score and a nine-gene ITRG-based prognostic model that delineate the immune landscape and enables effective survival stratification in CRC, complementing the limitations of current staging systems. Additionally, IL20RB is highlighted as a promising therapeutic target, supporting the development of personalized immuno-targeted combination therapies in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/385eed67dbbc79210270a5630b0149ac9e500c8e" target='_blank'>
              Intratumor heterogeneity score reveals immune landscape and survival stratification in colorectal cancer
              </a>
            </td>
          <td>
            Zijing Wang, Liyuan Ma, Jinzhong Cao, Xi Chen, Xin Chang, Ruxue Ma, Hengyi Lv, Zixin Zhang, Hai Li, Tao Jiang
          </td>
          <td>2026-01-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In a recent study published in Nature, Cosenza et al. 1 introduced MAGIC (machine-learning-assisted genomics and imaging convergence), a versatile platform that photolabels cells with micronuclei and other nuclear atypia, enabling direct quanti ﬁ cation of de novo chromosomal abnormality (CA) formation and ongoing chromosomal instability (CIN) at single-cell resolution. Using this framework, the authors show how spontaneous and induced lesions from pre-mitotic DNA damage and mitotic errors drive distinct CIN trajectories, with TP53 inactivation markedly amplifying CA burden and complexity, highlighting how CIN emergence and propagation prior to clonal expansion may inform tumour initiation, progression and the identi ﬁ cation of actionable biomarkers and therapeutic vulnerabilities. CIN is the ongoing process by which cells acquire new CAs through repeated chromosome missegregation and structural rearrangements, altering chromosome number and structure across divisions (Fig. 1a). CAs are pervasive across cancers and typically affect more genes than substitutions or small insertion/ deletion alterations, fuelling clonal diversi ﬁ cation and tumour adaptation. 2 – 5 Accurate chromosome segregation and genome integrity rely on coordinated processes such as DNA replication and repair, telomere maintenance, spindle assembly and mitotic exit. 2 When these are compromised and cells bypass senescence and apoptosis, concurrent pre-mitotic and mitotic defects drive ongoing CIN. 2,5 However, most CIN">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c5eb4ff0382eff914e4d728cb5754b2a5b73b7" target='_blank'>
              It’s a kind of MAGIC: uncovering the origins of chromosomal instability
              </a>
            </td>
          <td>
            D. Gómez-Peregrina, César Serrano
          </td>
          <td>2026-02-05</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            Sara B. Costa, Joana P. Gonçalves
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Identifying the mutant genes that are selected during carcinogenesis is key to identifying candidates for intervention and understanding the processes that promote transformation. Here we applied two selection metrics to study the dynamics of mutational selection in a mouse model of ultraviolet light driven skin carcinogenesis in which multiple synchronous tumors develop in each animal. Sequencing normal skin and tumors over a time course revealed two genetic routes to squamous carcinoma. Nonsynonymous Trp53 mutants were positively selected in both epidermis and tumors and present in 90% of tumors. The remaining tumors carried other oncogenic mutants, including activating Kras mutations. However, other positively selected mutant genes lost their competitive advantage in heavily mutated epidermis and in tumors. We found ten mutant genes under negative selection in normal skin, one of which was also negatively selected in tumors. In addition one gene was negatively selected in tumors but not normal skin. We conclude that analysing selection in normal tissue alongside tumors may resolve the dynamics of selection in carcinogenesis and refine the identification of cancer drivers. Sequencing mutants in both normal skin and tumors that arise from it in a mouse model of ultraviolet light driven carcinogenesis reveals mutant selection changes as cancers develop. Only p53 mutants are selected throughout squamous carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4638b6cb2b56a741aa25d0049e47e04146d5e21" target='_blank'>
              The dynamics of mutational selection in cutaneous squamous carcinogenesis
              </a>
            </td>
          <td>
            G. Skrupskelyte, J. Fowler, S. Dentro, Carine Winkler, Irina Abnizova, Niklaus Beumer, R. Sood, Thomas Quarrell, C. King, J. Kamarashev, E. Guenova, M. Gerstung, Benjamin A. Hall, Liliane Michalik, P. Jones
          </td>
          <td>2026-01-12</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The B‐cell lymphoma 2 (BCL‐2) inhibitor venetoclax (VEN) in combination with hypomethylating agents (HMAs) has improved treatment outcomes for acute myeloid leukemia (AML) patients unfit for intensive chemotherapy and is increasingly used in the relapsed/refractory setting. However, primary resistance remains a significant challenge, affecting 20%–35% of treatment‐naïve and around 50% of previously treated AML patients. To investigate the mechanisms driving primary resistance to VEN–HMA therapy, we analyzed genetic, transcriptomic, BCL‐2 family protein expression, and ex vivo drug sensitivity data from 101 AML patients and correlated these profiles with clinical outcomes to VEN–HMA. Our study found that blasts from refractory patients exhibit an elevated BCL‐XL/BCL‐2 protein expression ratio, an immature CD34+CD38− phenotype, and frequent TP53 mutations. Consistent with the high ratio of BCL‐XL/BCL‐2, resistant samples showed increased ex vivo sensitivity to the dual BCL‐2/BCL‐XL inhibitor navitoclax. In addition, SMAC mimetics were effective in refractory blasts, which correlated with high TNF gene expression in these cells. Ex vivo treatment with the combination of navitoclax and SMAC mimetics further enhanced the eradication of VEN–HMA refractory blasts, although toxicity was also observed in healthy CD34+ cells. In conclusion, our integrative analysis identifies molecular signatures associated with primary VEN–HMA resistance and highlights BCL‐2/BCL‐XL inhibition and SMAC mimetics as therapeutic strategies to target resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f404303a3f5b827792610f3342a27ad893ee2ef5" target='_blank'>
              Drivers of clinical resistance to venetoclax and hypomethylating agents in acute myeloid leukemia and strategies for improving efficacy
              </a>
            </td>
          <td>
            Ida Vänttinen, Joseph Saad, Tanja Ruokoranta, S. Kytölä, Guangrong Qin, Bahar Tercan, Pia Ettala, A. Partanen, M. Pyörälä, J. Rimpiläinen, T. Siitonen, Mikko Manninen, P. Valk, G. Huls, V. Thorsson, C. Heckman, Mika Kontro, H. Kuusanmäki
          </td>
          <td>2026-01-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="This study aimed to explore the potential genetic underpinnings of childhood acute lymphoblastic leukemia (ALL). Initially, an exome-wide association analysis of 89 ALL survivors and 356 controls was conducted to identify candidate genes, incorporating single-variant tests and annotation-weighted aggregation tests. Single-cell analyses characterized the candidate genes’ biological context, while aggregate testing linked these genes to clinical traits in survivors, along with bulk tissue transcriptomes profiling their expression patterns in relation to secondary adverse events (e.g., progression, relapse, secondary malignant neoplasms or death). Results from single-variant tests showed that rs57421986 of hematology-related gene SPTB exhibited a suggestive association with ALL [minor allele frequency = 0.029, odds ratio (OR) = 21.7, P = 5.86 × 10− 5]. Aggregation tests of low-frequency and rare variants identified MFAP3L (P = 3.97 × 10− 8), CARD11 (P = 4.18 × 10− 8), NOL8 (P = 2.46 × 10− 10), and OR7G2 (P = 2.20 × 10− 7) as ALL-associated genes. Notably, CARD11 and NOL8 demonstrated differential expression across diverse immune cells between incident cases and controls. Clinical phenome analyses linked CARD11 to mean corpuscular hemoglobin (P = 4.32 × 10− 2), alkaline phosphatase (P = 4.33 × 10− 2), and free triiodothyronine (P = 4.85 × 10− 2), NOL8 to red blood cell count (P = 2.83 × 10− 4) and mean corpuscular volume (P = 2.04 × 10− 3), and OR7G2 to thyroid-stimulating hormone (P = 9.56 × 10− 3) and alkaline phosphatase levels (P = 3.07 × 10− 2). Furthermore, a higher expression of OR7G2 at diagnosis showed a suggestive association with the occurrence of secondary adverse events (P = 0.034). Our findings provide initial clues regarding the shared genetic factors underlying both ALL onset and prognosis. These insights may offer potential guidance for refined risk stratification and personalized survivorship care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed68046e0425c98697fcedfc5d6cbfdc047e6e06" target='_blank'>
              Low frequency and rare coding variants affect susceptibility and progression of childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xiao Liu, Ru Zhang, Heng Zhang, Junyi Xin, Huiqin Li, Yongchu Pan, Liwen Zhu, Meiyun Kang, Hua Jiang, Yongjun Fang, Yao Xue, Mulong Du
          </td>
          <td>2026-01-02</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Double-Negative T (DNT) cells, lacking both CD4 and CD8 expression, play critical roles in cancer immunology, and have garnered increasing attention in cancer research. However, their heterogeneity and functional diversity within the tumor microenvironment (TME) remain underexplored. In-house and publicly available single-cell RNA sequencing (scRNA-seq) data for different cancer types were integrated after quality control and batch effect correction, followed by DNT cells separation from CD3+ T cells subtypes. Functional characteristics, intercellular communication, differentiation trajectories, regulatory networks, and clinical relevance were analyzed among different DNT subsets. Key findings were validated using multiplex immunofluorescence and spatial transcriptomics to investigate the spatial localization of DNT subsets and their interactions within the TME. Impact of γδ T cells on immunotherapy response was also assessed using MC38-based murine tumor model. By integrating scRNA-seq data from 2,369 samples across 23 cancer types, we established a comprehensive single-cell atlas of 157,025 high-quality DNT cells. Fourteen distinct DNT subsets (6 αβ DNT and 8 γδ T cell subsets) were identified, demonstrating tumor both type-specific and shared distribution patterns, as well as unique cell-cell interaction network within the TME. These subsets displayed specialized functional profiles, including cytotoxicity, antigen presentation, and immune modulation, indicating that the functional diversity of DNT cells is largely subset-specific rather than a manifestation of multifunctionality within a single population. We also delineated divergent trajectories for αβ DNT and γδ T cell subsets, including the functional plasticity of gut-resident γδ T cells transitioning between cytotoxic and immunosuppressive states. Notably, several DNT subsets were significantly associated with favorable clinical treatment outcomes, including improved responses to cancer immunotherapy. Consistently, depletion of γδ T cells in the murine tumor model significantly decreased the efficacy of PD-1 blockade, underscoring their critical role in therapeutic response. Our study uncovers the previously underappreciated heterogeneity and functional diversity of DNT cells in the TME and demonstrates their profound impact on tumor progression and immunotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf4d4eea61e2f155251ae235dac3fc5c5c43d18" target='_blank'>
              A pan-cancer single cell landscape reveals heterogeneity and functional diversity of double-negative T cells
              </a>
            </td>
          <td>
            Qing Hao, Tao Zhou, Huayun Yan, Zhixiang Ren, Wei Mao, Li-Bin Huang, Yue-Yun Chen, Yuelan Wang, Minqin Xiao, Yajiao He, Xuyang Xia, Jing Wu, Wei-Han Zhang, Hai-Ning Chen, Maoxiang Qian, Feng Zhang, Li Yang, Lunzhi Dai, Huiyuan Zhang, Canhua Huang, Yang Shu, Yu Xu, Yang Luo, Yong Peng, Heng Xu
          </td>
          <td>2026-01-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Therapy resistance remains a major cause of relapse and poor outcomes in acute lymphoblastic leukemia (ALL). Recent multi-omics studies in ALL have revealed that resistance arises from a combination of leukemia-specific genetic lesions, treatment-driven clonal evolution, and adaptive non-genetic programs. Genomic analyses have identified recurrent alterations associated with resistance to chemotherapy, tyrosine kinase inhibitors, and immunotherapies, while single-cell profiling has uncovered heterogeneous cell states that persist during treatment and contribute to minimal residual disease. Emerging epigenetic, proteomic, and metabolic data further indicate that reversible regulatory and signaling changes play a central role in leukemic persistence. Integrative analyses are beginning to define convergent resistance pathways and clinically relevant biomarkers, although longitudinal sampling and clinical translation remain limited. This review summarizes the current multi-omics landscape of therapy resistance in ALL and discusses opportunities to improve risk stratification and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98ef9b63ea903857bf083205a34581a4f1ebb5b7" target='_blank'>
              Multi-Omics Approaches to Understanding Therapy Resistance in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Xiuyun Wu, Jingxin Zhang
          </td>
          <td>2026-01-29</td>
          <td>Lymphatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68a7f2d65b2bcc7ad531577f95f9da5073382215" target='_blank'>
              A rapid ONT-based sequencing approach to capture complete Ataxia-Mutomes (AtaxiaMutSeq)
              </a>
            </td>
          <td>
            T. De, M. Faruq
          </td>
          <td>2025-12-19</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We present a large-scale study of structural variation (SV) in the Qatari population, based on short-read whole-genome sequencing (WGS) of 6,141 individuals, identifying 153,946 variants across 5 classes reflecting the region’s diversity and evolutionary history. Leveraging consanguinity and biobank phenotypes, we identify >180 putative gene knockouts, and use proteomics to show functional consequences in homozygotes. Conversely, 52 genes show significant depletion of homozygous deletions, eight of which cause severe pediatric disease or murine embryonic lethality. Examining phenotypic extremes uncovers several non-exonic homozygous deletions with large effect, including in SPIRE2 (creatinine), MAGI2 (leanness) and a chr19 microRNA cluster (extreme obesity). Further, SV-GWAS reveals gene-trait associations independent of SNPs, including at ACY1 (acetylation), SLC2A9 (uric acid), UGT1A8 (bilirubin) and ZNF251 (alanine aminotransferase). Notably, 3.2% of Qataris carry findings in medically actionable genes, one-third attributable to SVs. Our findings offer a rich SV reference for a globally understudied population, and demonstrate the utility of consanguineous biobanks for studying SVs in health and disease. All common SVs and tag-SNPs are provided as imputation resource. From 6,141 Qatari genomes the authors chart structural variants tied to large-scale protein data, biobank traits and everyday health measures. When combined with single nucleotide variants, they find that 1 in 30 participants have a medically actionable genomic finding. The work builds a detailed structural variation reference for Arab ancestries worldwide.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76785aa0cb84326d3654e25b7c82c65fe6408667" target='_blank'>
              The biomedical landscape of genomic structural variation in the qatari population
              </a>
            </td>
          <td>
            E. Aliyev, Najeeb Syed, A. Visconti, Taghi Aliyev, A. Belkadi, Mohammadmersad Ghorbani, Niccolò Rossi, Haroon Naeem, Geethanjali Devadoss Gandhi, G. Thareja, A. Al-Maraghi, W. Aamer, Amal Abdulsalam Ibrahim, R. Shaath, F. Al-Ajli, Rozaimi Razali, F. Sedlazeck, Sonia Davila, A. Akil, K. Suhre, Younes Mokrab, Mario Falchi, Khalid A. Fakhro
          </td>
          <td>2026-01-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb14a3c17daae13a9f8ee1cd3546f3c3832ec32" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>2</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb493837934e32a1361cf64825c6efc8b0b26882" target='_blank'>
              Pan-cancer RNA editing activity reveals complex editing functions and cancer immunotherapy biomarkers
              </a>
            </td>
          <td>
            M. Guo, Yuanyan Xiong
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Aim Breast cancer is the most prevalent form of cancer among females and carries a substantial societal impact. DSN1, a component of the MIS12 complex, plays a critical role in centromere assembly, distribution, and stability. While DSN1’s role in tumors has been investigated, its specific function in breast cancer remains unclear. Methods First, we utilized bioinformatics techniques to explore DSN1 expression in breast cancer and conducted functional enrichment and correlation analyses. Subsequently, we assessed the clinical relevance of DSN1 through immunohistochemistry. Furthermore, we examined how DSN1 affects the growth of breast cancer cells by conducting CCK8 and colony formation tests. Cell cycle and apoptosis changes were assessed using flow cytometry. Moreover, we examined key genes related to cell cycle and apoptosis to further elucidate the underlying mechanisms. Finally, we screened potential drugs targeting DSN1 by drug sensitivity and molecular docking analyses. Results Bioinformatics analysis revealed that DSN1 is highly expressed in breast cancer, making it a potential diagnostic marker. Functional enrichment analysis indicated that the DSN1- overexpressed group was enriched in cell proliferation-related pathways. Cellular experiments confirmed that DSN1 promotes breast cancer proliferation by affecting cell cycle pathways, involving key molecules such as CCNB1, CCND1, CKD1, CDK4, and CDK6. Drug sensitivity analysis showed that the DSN1 high expression group was resistant to drugs such as Epirubicin, Cyclophosphamide, Ribociclib, and Palbociclib, but relatively sensitive to tamoxifen and lapatinib. Conclusions DSN1 contributes to breast cancer progression by modulating cell cycle pathways, making it a potential diagnostic and therapeutic target with clinical applicability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3515d6bfa96b38426d6222c03e6a7fb688833cda" target='_blank'>
              DSN1 drives breast cancer progression via cell cycle regulation: diagnostic and therapeutic implications
              </a>
            </td>
          <td>
            Dongyang Liu, M. Luis, Djojomoenawi Sherilyn H. G., Xiaoxuan Zhu, Xiuqin Zhang, Yuan Fang, Xiaoyan Lin, Shasha Tang, Fengfeng Cai
          </td>
          <td>2026-01-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29ad767de6b1b60629bc324b66e333d5c7c4a2b" target='_blank'>
              A Human Genetics Framework for De-risking Gene Editing Targets for Hematopoietic Cell and Gene Therapy
              </a>
            </td>
          <td>
            A. Dhall, Alyssa Pyclik, Matthew H. Ung, Julia Etchin, Y. Keschner, Huanying Gary Ge, Tirtha Chakraborty, J. Lydeard
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Mitochondria are highly dynamic organelles that integrate metabolic regulation, signal transduction, and programmed cell death with their canonical role in adenosine triphosphate (ATP) production. Their ability to undergo continuous remodeling through the opposing processes of fusion and fission is essential for maintaining cellular homeostasis, preserving organelle quality control, and enabling adaptive responses to metabolic and oxidative stress. Among the core regulators of mitochondrial dynamics, the dynamin-related guanosine triphosphatase (GTPase) OPA1 plays a central role in inner membrane fusion, cristae architecture maintenance, bioenergetic efficiency, and the modulation of redox balance and apoptotic signaling. Accumulating evidence indicates that dysregulation of OPA1 expression or activity contributes to the initiation and progression of multiple malignancies, underscoring its importance in tumor cell survival, proliferation, metabolic adaptation, and resistance to stress. Here, we summarize current knowledge on OPA1 dysregulation in cancer and, based on preliminary, unpublished in silico analyses, we highlight the growing relevance of OPA1 as a therapeutic target, particularly through its GTPase domain and the still understudied Interface 7. Overall, these findings outline how integrated computational approaches could potentially guide the identification of novel OPA1 modulators, offering a conceptual framework that highlights OPA1 as a promising, yet still largely underexplored, target in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bbc85b07d21254bc1200e427724eda318625e7" target='_blank'>
              OPA1 as a Cancer Target: Molecular Mechanisms, Structural Insights, and Strategies for Drug Development
              </a>
            </td>
          <td>
            Antonio Curcio, Ludovica Ganino, Ilenia Valentino, Massimo Gentile, Stefano Alcaro, R. Rocca, A. Artese, N. Amodio
          </td>
          <td>2026-01-01</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062e9996a43c3e3095b1126f2396b4c755a5c6ba" target='_blank'>
              Expanding the genetic landscape of inherited metabolic diseases using long-read sequencing and transcriptomic profiling.
              </a>
            </td>
          <td>
            Alejandro Soriano-Sexto, Obdulia Sánchez-Lijarcio, Leonardo Beccari, Natalia Castejón-Fernández, F. Leal, P. Alcaide, Belén de la Morena-Barrio, M. Bahíllo‐Curieses, Patricia Correcher, Rafael Hencke-Tresbach, Laura López, Elena Martín-Hernández, Raquel Yahyaoui, Magdalena Ugarte, P. Rodríguez-Pombo, Belén Pérez
          </td>
          <td>2026-01-26</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abedd58342641982b4fa326716bab2f81320dd" target='_blank'>
              Coordinate Up-regulation of Both TK1 and TS Expression in Cycling Cells: Mechanistic Basis and Implications for Dual-Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Wanxing Wang, Jiawei Zhang, Guangyou Duan, Yiyao Zhang, Guofeng Xu, Zhi Cheng, Shunmei Chen, Xue Yao, Xin Li, Shan Gao
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Replication stress (RS) is a hallmark of cancer, largely driven by oncogene activation. Due to high levels of RS, cancer cells depend heavily on the RS response mechanisms to avoid DNA damage. This dependency creates a therapeutic opportunity that can be exploited for more effective cancer treatment. This review synthesizes current mechanistic understanding of RS and RS response and further describes how targeted disruption of RS response proteins (ATR, Chk1, Wee1, PARP, RPA) has been used in preclinical and clinical studies. We summarize preclinical and emerging clinical evidence for exploiting RS for radiosensitization, and outline candidate biomarkers and functional assays for patient selection. We also highlight the links between RS, therapy-induced senescence and innate immune activation via the cGAS–STING (cyclic GMP-AMP synthase—Stimulator of Interferon Genes) pathway, and address current challenges and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/290040ba46caa91c5606fcee3d46699f73704969" target='_blank'>
              Replication Stress in Cancer: Mechanistic Insights and Therapeutic Opportunities for Radiosensitization
              </a>
            </td>
          <td>
            Spyridon N. Vasilopoulos, Ioanna Tremi, I. Kotta-Loizou, A. Gkikoudi, O. Tsitsilonis, Sophia Havaki, A. Georgakilas
          </td>
          <td>2026-01-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Analysis of coding areas has long been used to study monogenic illnesses, but despite the extensive use of whole-exome sequencing (WES), up to half of suspected cases remain genetically unexplained. Variants outside coding areas can alter splicing, transcript stability, or gene regulation, compromising normal gene activity. These include mutations in noncoding RNAs, promoters, enhancers, deep intronic sequences, and untranslated regions (UTRs). Several well-known disorders have been linked to these mechanisms, including β-thalassemia caused by deep intronic mutations leading to aberrant splicing, familial hypercholesterolemia caused by promoter defects affecting LDLR expression, and inherited retinal diseases driven by noncoding variants influencing retinal gene regulation. These instances show that pathogenic variation is not limited to the exome and can have significant clinical implications. This review summarizes current understanding of noncoding and regulatory variants in monogenic diseases, discusses how they influence diagnosis and therapy, and highlights integrative approaches combining genomic, transcriptomic, and epigenomic data. Multi-layered research has increased diagnostic accuracy and unveiled new therapeutic potentials, although noncoding variations make the connection between genotype and phenotype more complex. Noncoding regions will need to be incorporated into standard diagnostic procedures to convert molecular insights into concrete therapeutic applications in the future. Predictive algorithms, patient-derived model systems, and functional validation testing will all help to simplify this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffd879f4fd9d85153dc94e05643890e26eb43d7" target='_blank'>
              Beyond the Exome: The Role of Noncoding and Regulatory Variants in Monogenic Diseases
              </a>
            </td>
          <td>
            Efthalia Moustakli, Nektaria Zagorianakou, Stylianos Makrydimas, Andreas Miltiadous, Alexandros T. Tzallas, G. Makrydimas
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cancer is the second leading cause of death worldwide. While significant progress has been made in early detection and treatment, metastasis remains the major cause of cancer-related morbidity and mortality. In the last decade the rate of long-term survivorship of metastatic cancer has continued to improve and overcoming resistance to therapy has now become a challenge. Developing strategies to prevent and treat metastatic disease is a priority for public health and requires a thorough understanding of the mechanisms driving progression of a specific patient's tumor and the rapid identification of targetable cancer drivers and drug resistance genes. Custom bioprinted tumors, which recreate the interactions between tumors and surrounding tissues, can be integrated into organ-on-chip platforms, and leveraging molecular pathology and OMICS data, can provide highly realistic patient-specific models. These biomimetic tools enable the investigation of metastasis organotropism, the identification of therapeutic targets and the design of drug administration protocols to prevent metastasis and to overcome resistance. Benefits, limitations, and challenges to address for an efficient and routine application of this cutting-edge approach, together with the role of Artificial-Intelligence (AI) in managing the complex datasets generated by OMICS technologies will be highlighted in this review, as well as their real-life implications and evolutionary prospects. Applying patient-derived bioprinted tumors and organs for clinical purpose and developing standardized 4D and 5D bioprinting protocols would allow assessment of cancer response to treatments in a dynamic and faithfully reconstructed microenvironment. Integration of advanced molecular diagnostics and multi-OMICS data, with customized small-scale tumor models, assisted by AI-powered tools, requires a multidisciplinary framework. This integrated approach can upgrade clinical management of metastatic diseases, by accelerating the identification of actionable biomarkers and resistance mechanisms for timely therapy adjustments, thus enabling tailored treatment regimens based on individual tumor behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edae83d1a5d8f7f09e850f3c6b17b8282edb0c53" target='_blank'>
              Tumor-on-chip’s alliance with molecular pathology against metastatic disease
              </a>
            </td>
          <td>
            E. Di Carlo
          </td>
          <td>2026-01-06</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Breast cancer is a significant global health issue, with resistance to doxorubicin (DOX) posing a major challenge to effective treatment. SUMOylation, a post-translational modification process, is linked to cancer progression and therapy resistance. PIAS4, a SUMO E3 ligase involved in maintaining genome stability and stress response, may play a role in these mechanisms. However, its function in breast cancer progression and DOX resistance remains uncertain. This study investigates the potential role of PIAS4 in mediating DOX resistance in breast cancer. Naked mole-rats (NMRs) are cancer-resistant rodents with improved genome maintenance, yet the role of SUMOylation in this trait remains unclear. SUMOylation machinery gene expression levels are investigated using qPCR in NMR tissue in comparison with carcinogenic breast cancer (MCF-7) cell line. Functional studies are performed in MCF-7 cells overexpressing PIAS4 to demonstrate effects on proliferation, invasion, drug sensitivity, and protein expression in the presence and absence of DOX treatment. While most SUMOylation genes were expressed at low levels in NMR intestinal tissues, PIAS4 showed higher expression compared to MCF-7 cells. PIAS4 overexpression in MCF-7 cells significantly decreases colony formation, invasiveness, and resistance to DOX. Western blot analysis showed downregulated Bcl-2 protein levels after DOX treatment, indicating a potential role in apoptosis evasion. PIAS4 expression level plays a role in breast cancer cell survival, invasiveness, and chemoresistance, partly by altering anti-apoptotic pathways. These findings position PIAS4 as a potential biomarker and therapeutic target for overcoming resistance to anthracycline-based therapies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c44a6f66fc39ce2adfec6f2cb758f8f029d8969" target='_blank'>
              SUMOylation machinery protein, PIAS4 role in breast cancer cell proliferation and drug sensitivity
              </a>
            </td>
          <td>
            M. A. M. Salih, Mohamed M. A. E. L. Salem, Muhammad Ali Shahid, Hussein A. S. Elrewey, Ritchie Williamson, Sriharsha Kantamneni
          </td>
          <td>2026-01-30</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Neuroblastoma (NB) is a childhood cancer that arises when certain nerve-related cells fail to mature properly and become malignant. Our study focuses on a gene called INSM1, which is normally active during early cell development but is abnormally elevated in NB tumors. We aimed to understand how INSM1 contributes to cancer’s aggressiveness and poor differentiation. We discovered that INSM1 keeps tumor cells in an immature, stem-like state and is closely linked to a chemical pathway called the methionine cycle, which affects gene regulation through epigenetic changes. Disrupting this cycle alters INSM1 activity and may help push cancer cells toward normal development. We also found that retinoic acid, a compound known to promote cell differentiation, reduces INSM1 and another cancer-driving gene, N-Myc. These findings suggest that targeting INSM1 and its metabolic regulation could be a promising strategy to treat NB by encouraging tumor cells to mature and lose their malignant properties. This research provides new insights into the biology of NB and may lead to more effective therapies for children affected by this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb4304aef088d414eab3a9ced76da697e7e9f2" target='_blank'>
              Regulation of INSM1 Gene Expression and Neuroendocrine Differentiation in High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Chiachen Chen, Siyuan Cheng, Xiuping Yu, Yisheng Lee, Michael S. Lan
          </td>
          <td>2025-12-22</td>
          <td>Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02053574b70df78fae143f9f004a2f833452c35d" target='_blank'>
              AlphaGenome-enabled analysis of non-coding regulatory variants underlying RHD expression with wet-lab validation
              </a>
            </td>
          <td>
            Miao Liu, Ziyang Shen, You Kyeong Jeong, Na Yu, Shang-Chuen Wu, Andre Wittig, Danielle Tenen, Yanjing Liu, Jun Liu, Li Chai
          </td>
          <td>2026-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor adaptation and therapeutic resistance represent major limitations to sustained treatment efficacy in modern oncology. Despite advances in targeted therapies and personalized approaches, disease progression and relapse remain frequent, reflecting the intrinsic biological complexity of malignant tumors. This review synthesizes high-impact evidence addressing the mechanisms that drive tumor adaptation under therapeutic pressure, with particular emphasis on intratumoral heterogeneity, evolutionary dynamics, and cellular plasticity. The analyzed literature supports the view of cancer as a dynamic and evolving ecosystem composed of genetically and phenotypically diverse subpopulations that respond heterogeneously to treatment. Resistance emerges as a predictable consequence of Darwinian selection, shaped by both genetic alterations and non-genetic adaptive processes, including reversible drug-tolerant states and microenvironmental influences. The findings further highlight the limitations of static, single-time-point molecular profiling in capturing tumor complexity and guiding long-term therapeutic decisions. Evolution-informed strategies, such as adaptive therapy and rational treatment sequencing, are discussed as biologically grounded approaches that may improve disease control by managing, rather than attempting to eradicate, heterogeneous tumor populations. This integrative perspective is particularly relevant for diverse healthcare settings, including those in Latin America, where conceptual frameworks can support clinical decision-making in variable resource contexts. Overall, understanding tumor adaptation and resistance as emergent evolutionary phenomena provides a coherent foundation for advancing personalized oncology, improving clinical reasoning, and strengthening oncology education">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ab93e274f88a42001d0b9860cca9a59037481" target='_blank'>
              Evolutionary Dynamics of Tumor Adaptation and Therapeutic Resistance in Personalized Oncology
              </a>
            </td>
          <td>
            Alexis Ramírez Hernández, José Arnulfo Perez Carrillo, Juan Diego Paloma Meza, Adrián Jesús Santoyo Rojas, Patricio Sebastian Ordóñez Montalvo, Wilmer Rene Cujilema Guillin, Jorge Che Enseñat, Zuleika Morales
          </td>
          <td>2026-01-26</td>
          <td>IECCMEXICO</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="How endogenous retroviral elements (ERVs), a family of transposable elements, may promote tumor progression is not well understood. Tripartite motif-containing 28 (TRIM28/TIF1b/KAP1) is a key transcriptional co-repressor protein that represses ERV expression in many cell types including embryonic stem cells, neural progenitor cells, differentiated adult cells, and cancer cells. In this study, we investigated the effect of Trim28 deletion on the expression of ERVs using an immune competent genetically engineered mouse model for prostate cancer. We found Trim28 deletion in prostate tumors led to the expression of ERVs in prostates from both hormonally intact and castrated mice. ERVs can regulate the expression of neighboring genes, and we detected increased expression of several protein-coding genes near overexpressed ERVs. Our data suggest that Trim28 deletion in prostate tumor epithelial cells may promote an innate immune response. However, Trim28 deletion also led to excessive deposition of tumor extracellular matrix (ECM). Our findings suggest that ECM alterations downstream of ERV derepression could affect immune cells in the tumor microenvironment and may promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0f3ac90397a0b7a69b88c44582035578d15374" target='_blank'>
              Expression of endogenous retroviral elements is associated with extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            Emily C. Williams, Dewanga R. Mayarata, Anelia Horvath, Katherine B. Chiappinelli, Maho Shibata
          </td>
          <td>2026-01-08</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Rapid advances in biotechnology provide researchers with the opportunity to integrate omics profiles (genomics, epigenomics, transcriptomics, proteomics, etc.) with multiple phenotypes or experimental conditions. In cancers such as acute myeloid leukemia (AML), where combination therapies are standard of care, identifying genetic drivers of drug resistance requires evaluating how genes are associated with multiple drug response phenotypes. Statistical analyses associating omics profiles with multiple phenotypes yield multiple significance values and rankings for each of many genes. There is a great need to consolidate these multiple rankings into a consensus ranking to prioritize specific genes for detailed follow-up wet-lab or clinical studies. Methods/Results: Here, we evaluate the well-known Fisher’s method, the sum of squared z-statistics (SSz), and the recently published cellMCD method as tools for gene prioritization. In simulation studies, cellMCD showed very similar or highly superior performance to the widely used Fisher’s and SSz methods. These advantages were also observed in an example application involving a CRISPR drug screen of an acute myeloid leukemia cell line. Conclusions: In summary, our results indicate that cellMCD should be more widely used for prioritizing discoveries from multiple omic association studies. These methods are available as an R package on github.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936ca737a6856489d929417a1a5e1ec23e0e8b36" target='_blank'>
              cellMCD Effectively Discovers Drug Resistance and Sensitivity Genes for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Dora Obodo, Nam H. K. Nguyen, Xueyuan Cao, P. K. Parcha, Christopher D. Vulpe, J. Lamba, S. Pounds
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbc11170b8c3915b19ca1579bf9e3cf87692a5a" target='_blank'>
              Spatio-temporal resource landscapes govern the confinement and escape of therapy-resistant mutants in structured populations
              </a>
            </td>
          <td>
            Nico Appold, Timon Citak, Auguste Palm, Jona Kayser
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The phenomenon of transgene silencing was first observed shortly after the generation of the initial transgenic plants. The vast body of experimental data accumulated since then constitutes an invaluable resource for dissecting the mechanisms of epigenetic gene regulation. Silencing operates at either the transcriptional (TGS) or post-transcriptional (PTGS) level and is predominantly mediated by small interfering RNAs (siRNAs). Although these two epigenetic pathways involve distinct sets of proteins and enzymes, they share fundamental mechanistic features: the generation of double-stranded RNA (dsRNA), its processing into siRNAs by DICER-LIKE (DCL) enzymes, and the assembly of an Argonaute-centered effector ribonucleoprotein complex (RISC). Guided by sequence-specific siRNAs, this complex identifies complementary target sequences with high precision. A comprehensive understanding of these regulatory pathways enables the targeted induction or suppression of specific plant genes. This review traces the history of experimental findings regarding the loss of recombinant gene activity in transformants and their progeny, which collectively established the foundation for elucidating the molecular mechanisms of transgene silencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ca1645109162b7bd31d62f8f845655fb6172721" target='_blank'>
              Gene Inactivation in Transgenic Plants—A Unique Model for Studying Epigenetic Regulation of Gene Expression
              </a>
            </td>
          <td>
            T. Marenkova, A. Zagorskaya, I. Deyneko, Elena V. Deineko
          </td>
          <td>2026-01-01</td>
          <td>Plants</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multidrug resistance (MDR) is a central cause of chemotherapy failure and tumor recurrence and metastasis, and its mechanism involves enhanced drug efflux, target mutation, upregulation of DNA repair and remodeling of the tumor microenvironment. ABC transporter protein (P-gp, MRP, and BCRP)-mediated efflux of drugs is the most intensively researched aspect of the study, but the first three generations of small-molecule reversal agents were stopped in the clinic because of toxicity or pharmacokinetic defects. Natural products are considered as the fourth generation of MDR reversal agents due to their structural diversity, multi-targeting and low toxicity. In this paper, we systematically summarize the inhibitory activities of monoterpenes, sesquiterpenes, diterpenes and triterpenes against ABC transporter proteins in in vitro and in vivo models and focus on the new mechanism of reversing drug resistance by blocking efflux pumps, modulating signaling pathways such as PI3K-AKT, Nrf2, NF-κB and remodeling the tumor microenvironment. For example, Terpenoids possess irreplaceable core advantages over traditional multidrug resistance (MDR) reversers: Compared with the first three generations of synthetic reversers, natural/semisynthetic terpenoids integrate low toxicity (mostly derived from edible medicinal plants, half-maximal inhibitory concentration IC50 > 50 μM), high target specificity (e.g., oleanolic acid specifically inhibits the ATP-binding cassette (ABC) transporter subtype ABCC1 without cross-reactivity with ABCB1), and multi-mechanistic synergistic effects (e.g., β-caryophyllene simultaneously mediates the dual effects of “ABCB1 efflux inhibition + apoptotic pathway activation”). These unique characteristics enable terpenoids to effectively circumvent key limitations of traditional synthetic reversers, such as high toxicity and severe drug–drug interactions. Among them, lupane-type derivative BBA and euphane-type sooneuphanone D (triterpenoids), as well as dihydro-β-agarofuran-type compounds and sesquiterpene lactone Conferone (sesquiterpenoids), have emerged as the core lead compounds with the greatest translational potential in current MDR reverser research, attributed to their potent in vitro and in vivo MDR reversal activity, low toxicity, and excellent druggable modifiability. At the same time, we point out bottlenecks, such as low bioavailability, insufficient in vivo evidence, and unclear structure–activity relationship and put forward a proposal to address these bottlenecks. At the same time, the bottlenecks of low bioavailability, insufficient vivo evidence and unclear structure–activity relationship have been pointed out, and future research directions such as nano-delivery, structural optimization and combination strategies have been proposed to provide theoretical foundations and potential practical pathways for the clinical translation research of terpenoid compounds, whose clinical application still requires further in vivo validation and translational research support.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc8e568cf68a8569a80453b9e89afe2909550d62" target='_blank'>
              Terpenoids: Emerging Natural Modulators for Reversing ABC Transporter-Mediated Multidrug Resistance in Cancer Chemotherapy
              </a>
            </td>
          <td>
            Lanfei Ma, Dina Mahemuti, Yuanhong Lan, Jianxiong Xu, Wenfang Li, Zhengding Su, Jinyao Li, Aytursun Abuduwaili, Ayitila Maimaitijiang
          </td>
          <td>2026-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Full-length gene-humanized mice engineered by completely replacing mouse loci with human counterparts, including untranslated and regulatory regions, provide a robust in vivo platform for human gene function studies. However, reliably humanizing large genomic regions remains challenging due to limited DNA insert sizes, complex protocols, and specialized material requirements. This study introduces a streamlined approach that enables full-length gene humanization through two sequential CRISPR-assisted homologous recombination steps in embryonic stem cells. This method supports targeted knock-in of genomic fragments ( > 200 kbp) and is applicable across multiple mouse strains. Humanized alleles generated using the developed method recapitulate human-like splicing isoforms and organ-specific gene expression while restoring essential functions in hematopoiesis, spermatogenesis, and survival. Furthermore, disease-associated mutations can be engineered into humanized alleles to model human genetic disorders in vivo. This versatile platform enables the creation of physiologically relevant, fully gene-humanized mouse models for broad applications in biomedical research. Humanized mice bearing full-length human genes recapitulate human biology but are challenging to generate. Here, the authors develop TECHNO, a twostep CRISPR–ES cell method that efficiently replaces mouse loci with >200-kbp human orthologs, enabling versatile in vivo disease modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ed2ef004857ad6af4b2c26c7a523b20d1affec" target='_blank'>
              A scalable two-step genome editing strategy for generating full-length gene-humanized mice at diverse genomic loci
              </a>
            </td>
          <td>
            Jumpei Taguchi, Mio Kikuchi, Hyojung Jeon, Ryo Shimizu, Hideto Mori, Masahito Ikawa, Yasuhiro Yamada, Kei Sato, Terumasa Ikeda, Satoshi Yamazaki, Manabu Ozawa
          </td>
          <td>2026-01-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Ewing sarcoma is an aggressive bone tumor of adolescence characterized by a hallmark EWSR1::ETS fusion oncogene. The resulting chimeric oncoprotein drives tumorigenesis by reshaping transcriptional and epigenetic landscapes. However, how it is transcriptionally regulated and whether additional master transcription factors (MTFs) form a core regulatory circuit (CRC) in Ewing sarcoma remain unclear. Using an extensive panel of Ewing sarcoma cell lines and primary tumors, we mapped super-enhancers and identified enrichment of GGAA microsatellites, confirming their specificity to Ewing sarcoma as compared to other pediatric cancers and normal tissues. Integrating transcriptomic, epigenetic, 3D chromatin conformation, and dependency data, we predicted a set of MTFs potentially forming a CRC. However, functional validation demonstrated that these MTFs neither establish auto-regulatory loops nor confer robust proliferative dependencies typical of CRCs in other pediatric tumors. Instead, EWSR1::FLI1 emerged as an “hegemonic” oncoprotein, regulating expression of these MTFs without reciprocal regulation. Knockdown of EWSR1::FLI1 strongly shifted H3K27ac profiles toward mesenchymal states, whereas silencing individual or combined MTFs did not alter cell growth or EWSR1::FLI1 expression. These findings highlight the absence of a classical CRC in Ewing sarcoma and emphasize EWSR1::FLI1 as the dominant oncoprotein and a major vulnerability in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/941b3d130e41a819129bbb3b8a4a261f87cf3d37" target='_blank'>
              The hegemonic EWSR1::ETS oncoprotein overrules core regulatory circuitry principles in Ewing sarcoma
              </a>
            </td>
          <td>
            Sandrine Grossetête, Sakina Zaidi, M. Orth, Sarah Morice, Karine Laud, Caroline Louis-Brennetot, Anna Sole Ferre, Virginie Perrin, F. Cidre‐Aranaz, Virginie Raynal, M. Aynaud, Sylvain Baulande, Marco Wachtel, I. Janoueix-Lerosey, Erika Brunet, Thomas G P Grünewald, Olivier Delattre, Didier Surdez
          </td>
          <td>2025-12-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Esophageal carcinoma (EC), comprising esophageal squamous cell carcinoma (ESCC) and adenocarcinoma (EAC), remains a deadly malignancy predominantly affecting individuals over 60 years of age. Although advances in diagnosis and treatment have improved outcomes for some cancers, EC still carries a poor prognosis, particularly in the elderly. Aging contributes significantly to EC pathogenesis via accumulated genetic mutations, metabolic deregulation, immunosenescence, and a pro-tumorigenic microenvironment. In this review, we synthesize current understanding of the molecular alterations that drive EC, with particular emphasis on age-related factors. We examine key genomic mutations (e.g., TP53, chromosomal instability), metabolic reprogramming (including glycolysis, glutaminolysis, and lipid metabolism), and their interplay with the tumor microenvironment (TME). We also highlight the influence of aging processes such as clonal hematopoiesis, senescence-associated secretory phenotype (SASP), and immune exhaustion in modulating tumor behavior and therapeutic resistance. We propose that aging-associated metabolic and immunologic alterations represent promising targets for therapeutic intervention. Greater integration of aging biology into EC research and clinical strategy is needed to advance personalized care for elderly patients. This review provides a conceptual foundation for future translational and clinical studies at the intersection of oncology and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdcaff8428584b8741ff99971d31c4c36a210c4" target='_blank'>
              Aging-associated mechanisms and metabolic vulnerabilities in esophageal carcinoma: an integrative review
              </a>
            </td>
          <td>
            Qiwei Liu, Zhiyu Xie, Wei Li, Mengxiang Li, Yijun Qi
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06b83eddceaa6549ab81ac78629425aa46eb6aed" target='_blank'>
              MSH2 Prevents Liver Tumorigenesis by Regulating Cell Cycle Checkpoints under Chronic Inflammation.
              </a>
            </td>
          <td>
            Shigeharu Nakano, Atsushi Takai, Eriko Iguchi, Yosuke Fujii, Haruka Amino, Masayuki Ueno, Takahiko Ito, Mari Teramura, M. Mishima, Ken Kumagai, Tadashi Inuzuka, Haruhiko Takeda, Y. Eso, Takahiro Shimizu, T. Maruno, Hiroshi Seno
          </td>
          <td>2026-01-29</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Targeting cell cycle checkpoints has emerged as a promising strategy in cancer therapy, yet single-agent inhibitors often fail because of compensatory mechanisms. Here, we demonstrate that coinhibition of ATR (RP-3500) and PKMYT1 (RP-6306) induces synthetic lethality in Rb1-deficient breast cancers by disrupting both S-G2 and G2-M checkpoints. This leads to replication stress, premature mitotic entry, and accumulation of DNA damage. In vitro, Rb1-deficient breast cancer cells exhibited marked apoptosis and loss of clonogenic survival, whereas Rb1-proficient models remained resistant to combination treatment. Genetic manipulation confirmed this dependency: Rb1 knockdown sensitized resistant models, whereas reexpression conferred protection. In vivo, patient-derived xenograft (PDX) models recapitulated these findings. Rb1-deficient tumors regressed after ATR/PKMYT1 coinhibition, whereas Rb1-proficient tumors showed only modest responses. Combination therapy was well tolerated without weight loss or measurable toxicity. Biomarker analysis revealed increased γH2AX and reduced Ki67 staining exclusively in Rb1-deficient PDX models, underscoring the specificity of this response. Mechanistically, Rb1 loss impaired double-strand DNA repair by attenuating homologous recombination and nonhomologous end joining, leading to replication fork collapse, chromosomal instability, and mitotic catastrophe. Proteogenomic analysis identified JNK/p38 stress response pathway activation as a key driver of apoptosis after ATR/PKMYT1 inhibition in Rb1-deficient cells. Clinically, a retrospective analysis of stage IV breast cancer datasets revealed that Rb1-low tumors display reduced DNA repair pathway activity in triple-negative and CDK4/6 inhibitor-resistant luminal breast cancers. These results identify Rb1 loss as a predictive biomarker for ATR/PKMYT1-targeted therapy, offering a potential precision treatment strategy for advanced breast cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd18d92004042bbcc1904fd2be3bc1b5cf617210" target='_blank'>
              Rb1 deficiency induces synthetic lethality with ATR and PKMYT1 coinhibition in breast cancer cell lines and patient-derived xenografts.
              </a>
            </td>
          <td>
            Xiao-Ting Jiang, Mei-Kuang Chen, Jiwang Lee, X. Rao, Yan Wang, Linjie Luo, Jing Wang, Kelly K. Hunt, K. Keyomarsi
          </td>
          <td>2025-12-24</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Highlights What are the main findings? Only 2.3% (52 total) of all human antisense genes have been examined in head and neck cancer (HNC) thus far. HOXA10-AS, LEF1-AS1, MSC-AS1, and ZEB2-AS1 have clinical promise for future biomarker development. What are the implications of the main findings? Robust cross-validations on antisense gene aberrations will promote biomarker development and implementation in the HNC field, as genetic biomarkers are lacking. Antisense gene aberrations and treatment responses are underexplored and warrant future studies. Abstract Antisense genes (usually suffixed by -AS) represent a class of long non-coding RNAs (lncRNAs) transcribed from the opposite strand of annotated human genes or exon(s). A total of ~2236 human antisense genes exist in the human genome. Their genomic locations with respect to the corresponding sense genes, their dysregulated expression patterns in cancer specimens, and clinical associations with patient outcomes reveal their potential importance in clinical settings. As of today, there lacks a comprehensive review of HNC-associated antisense genes/transcripts to help move forward the antisense field for genetic biomarker development or future drug research. In total, 2.3% (52/2236 antisense genes) of all known human antisense genes have been investigated in head and neck cancer (HNC). Thus, we perform a comprehensive review of the genomic aberrations (mutations, copy number changes, RNA-expression dysregulation, and single nucleotide polymorphisms) associated with HNC patient prognosis, disease progression, cancer cell signaling, drug sensitivity, and radio-resistance. Four antisense genes, namely HOXA10-AS, LEF1-AS1, MSC-AS1, and ZEB2-AS1, have been clinically cross-validated and have consistently demonstrated to be associated with patient outcomes in multiple independent cohorts by different research teams, with clear evidence for the prioritization of clinical biomarker development in HNC. Single nucleotide polymorphisms (SNPs) of antisense genes with evidence for HNC risk or outcomes should be further validated in different ethnic groups, for potential global HNC applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb801fbeee9cde00558b26b3b11efef6455c2950" target='_blank'>
              Genomic Aberrations of Antisense Gene Transcripts in Head and Neck Cancer
              </a>
            </td>
          <td>
            Jishi Ye, S. M. Abbang, Yuen-Keng Ng, V. Lui
          </td>
          <td>2025-12-19</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Epstein‐Barr virus (EBV) infection has been studied at single‐cell resolution for six decades and counting. Such investigations can reveal virus‐host interactions and their dependence on viral strain, cellular niche, infection program, immune response regulation, and time. Understanding these factors is paramount to treating EBV‐associated cancers and autoimmune diseases. This review examines the state of the field in EBV single‐cell and spatial‐omics spanning experimental models and clinical samples. Topics of primary interest include the growing adoption and emerging biological themes from single‐cell assays and analyses, the shift from characterization toward functional single‐cell studies, and strategies to maximize clinically relevant insights from dense single‐cell and spatial datasets. Ancillary topics include the historical evolution of the single‐cell EBV field and end‐to‐end single‐cell sequencing workflows. Special attention is given to open questions in molecular mechanisms of EBV pathogenesis and how they might be resolved by future studies utilizing single‐cell techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0066a9dbc4d3e08b9d29c0f78fd36900086ca38" target='_blank'>
              Epstein‐Barr Virus Infection at Single‐Cell Resolution
              </a>
            </td>
          <td>
            Elliott D. SoRelle
          </td>
          <td>2026-02-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer driver mutations alone are often insufficient to fully explain tumorigenesis. We demonstrate that these mutations cooperate with somatic copy number variations (CNVs) in a tissue-specific pattern of genomic epistasis. Analyzing 93,462 tumors, we identified 54 gene-cancer type pairs with significant co-occurrence of somatic mutations and CNVs. Our new Binoculars algorithm, which resolved phased DNA/RNA reads, revealed frequent preferential amplification in oncogenic mutation alleles, including AKT1 p.E17K, BRAF p.V600E, KRAS p.G12C/D/V, NRAS p.Q61K, and a fraction of gain-of-function TP53 p.R175H. Conversely, deletions selectively targeted the reference alleles, leading to loss of heterozygosity of IDH1 p.R132H and tumor suppressor mutations, including CDKN2A and TP53 truncations. Lung cancer patients carrying co-occurrences of somatic mutation-CNVs in TP53 and KRAS showed poorer survival than those carrying the same gene mutations. These findings reveal epistasis of cancer mutations and CNVs at an allelic resolution, suggesting specific genomic events to enhance patient stratification and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c67a7d5e6bdabbcb4e758c260b7827b1f0b38f4" target='_blank'>
              Widespread Epistasis between Cancer Driver Mutations and Allele-Specific Copy Number Variations
              </a>
            </td>
          <td>
            Serge Merzliakov, Guanlan Dong, Andrea Castro, Mahad Bihie, Yve Nichols-Evans, Hanna Carter, Kuan-lin Huang
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer therapy resistance emerges from highly integrated molecular systems that enable tumor cells to evade cell death and survive cytotoxic therapeutic stress. High Mobility Group Box 1 (HMGB1) is increasingly gaining recognition as a central coordinator of these resistance programs. This review delineates how HMGB1 functions as a molecular switch that dynamically redistributes between cellular compartments in response to stress, with each localization enabling a distinct layer of resistance. In the nucleus, HMGB1 enhances chromatin accessibility and facilitates the recruitment of DNA repair machinery, strengthening resistance to radio- and chemotherapeutic damage. Cytosolic HMGB1 drives pro-survival autophagy, maintains redox stability, and modulates multiple regulated cell death pathways, including apoptosis, ferroptosis, and necroptosis, thereby predominantly shifting cell-fate decisions toward survival under therapeutic pressure. Once released into the extracellular space, HMGB1 acts as a damage-associated molecular pattern (DAMP) that activates key pro-survival and inflammatory signaling pathways, establishing microenvironmental circuits that reinforce malignant progression and therapy escape. HMGB1 further intensifies resistance through upregulation of multidrug resistance transporters, amplifying drug efflux. Together, these compartmentalized functions position HMGB1 as a central node in the networks of cancer therapy resistance. Emerging HMGB1-targeted agents, ranging from peptides and small molecules to receptor antagonists and nanoformulations, show promise in reversing resistance, but clinical translation will require precise, context- and redox-informed HMGB1 targeting to overcome multifactorial resistance program in refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcd9023ae13f0952b85c73d6ba319988bfa2cf22" target='_blank'>
              HMGB1: A Central Node in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Bashar A. Alhasan, Boris A. Margulis, Irina V. Guzhova
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="
 Tumour heterogeneity, encompassing genetic, epigenetic, and microenvironmental diversity, remains a fundamental obstacle in precision oncology. Traditional bulk sequencing captures only averaged molecular profiles, thereby masking rare yet functionally critical subpopulations that drive malignant progression and therapeutic resistance. Recently, the emergence of single‐cell sequencing technologies has overcome the limitations of bulk approaches, enabling high‐resolution analyses of the genome, transcriptome, epigenome and proteome at the single‐cell level. These advances have enabled detailed mapping of tumour ecosystems, identification of key cellular subtypes, reconstruction of evolutionary trajectories and elucidation of intercellular communication networks within the tumour microenvironment. Accumulating evidence demonstrates that single‐cell technologies elucidate fundamental aspects of tumour biology and reveal potential diagnostic and therapeutic targets. This review systematically summarises the recent advances and applications of single‐cell sequencing in the field of precision oncology, with particular emphasis on its applications in mechanistic discovery, diagnosis, therapy, and prognosis. Furthermore, we discuss current challenges related to technology, data analysis, and clinical translation, and outline future research directions. In summary, single‐cell sequencing has profoundly reshaped our understanding of tumour biology and is propelling oncology into a new era of precision, prediction, and personalisation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cbf7015a15504645fc0387a8b9ed997f9b6eef" target='_blank'>
              Single‐cell sequencing for cancer precision medicine: From mechanism discovery to diagnosis and therapeutics
              </a>
            </td>
          <td>
            Yue Zhao, Hancong Li, Huanzuo Yang, Gongshuang Zhang, Xinyue Fu, Peiheng Li, Puxing He, Shuang Wu, Han Luo
          </td>
          <td>2026-01-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The discovery of lactylation, a post-translational modification derived from lactate, has fundamentally altered the perception of cancer metabolism. Once regarded as a metabolic waste product, lactate is now recognized as a central fuel source, a signaling molecule, and an epigenetic substrate capable of reprogramming gene expression and cellular function. Lactylation integrates metabolic reprogramming, tumor plasticity, and immune suppression, thereby orchestrating cancer initiation, progression, and resistance to therapy. This review provides a critical and integrative commentary on recent advances in lactylation biology, drawing from biochemical, epigenetic, and immunological perspectives. It synthesizes mechanistic insights into lactylation, highlights its role in tumorigenesis and the tumor microenvironment (TME), and evaluates therapeutic strategies that target lactate production, transport, and lactylation machinery. By dissecting consensus, controversies, and unresolved questions, we argue that lactylation represents both a hallmark of tumor adaptation and a potential Achilles’ heel for intervention. We further discuss future research directions, including comprehensive lactylome mapping, structural biology of lactylated proteins, microbiome-derived lactate, and clinical translation. Ultimately, lactylation is not merely a byproduct of glycolysis but a metabolic language that tumors employ to communicate, adapt, and thrive. Decoding this language may open new frontiers in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f11df3d617ec11389a48bfdc33dfa42dd8ab08b2" target='_blank'>
              Lactylation in cancer: mechanistic insights, tumor microenvironment, and therapeutic horizons
              </a>
            </td>
          <td>
            Qiang Yang, Zhibo Yang, Hai Zhao
          </td>
          <td>2026-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d478805bf09978bcfb14d9f820b94dc85c8881" target='_blank'>
              Precision CRISPR annotation of the functional enhancer landscape in primary human T cells
              </a>
            </td>
          <td>
            Ping Wang, Josephine R. Giles, Hua Huang, Sasikanth Manne, Martha S. Jordan, Alexander C. Huang, Junwei Shi, E. Wherry
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac5062560ab2a4d1c7e4c81040ac2034b6307511" target='_blank'>
              Synthetic Hybrid Receptors for Safer and Programmable T Cell Therapy
              </a>
            </td>
          <td>
            Maxwell G. Foisey, Julie M. Garcia, Xun Li, Xiaoyu Yang, Claire Hilburger, Chloë Thienpont, Alejandro Chaves-Martinez, Serkan Belkaya, Tina Truong, Iowis Zhu, Raymond Liu, Axel Hyrenius-Wittsten, Ricardo Almeida, Brian R. Shy, Greg M. Allen, Sarah Wyman, Kole T. Roybal
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Breast cancer progression is increasingly linked to disturbances in circadian rhythm genes, although the underlying molecular mechanisms remain poorly understood. Circadian rhythm genes help maintain normal biological processes and their disruption contributes to breast cancer development. Transcriptomic data from breast cancer (MCF-7) and normal breast (MCF-10A) cell lines from the GSE76370 dataset were analyzed using the limma R package to identify differentially expressed genes. Functional enrichment and network analyses using GO, KEGG, STRING and Cytoscape revealed 1,788 DEGs, including 1,008 upregulated genes involved in DNA replication, chromatin remodeling and PI3K-Akt signaling and 780 downregulated genes associated with cell adhesion and apoptosis. Disrupted expression of core circadian genes (BMAL1, CLOCK and PER3) and hub genes such as ACTB, GAPDH and CDK1 suggests that circadian gene dysregulation promotes breast cancer progression and represents a potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f54cf298483ced6cca5212ac65f53b37db09bc31" target='_blank'>
              Disrupted circadian control promotes oncogenesis in breast cancer
              </a>
            </td>
          <td>
            Y. Fatima, Mohammad Kashif, Prashant Ankur Jain
          </td>
          <td>2025-12-15</td>
          <td>Bioinformation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality among women, with hormone receptor-positive (HR + ) disease representing the majority of cases. Despite advances in targeted therapies and antibody-drug conjugates, resistance to treatment remains a significant challenge, necessitating innovative therapeutic strategies. Epigenetic dysregulation, particularly involving histone acetylation, plays a crucial role in tumorigenesis and therapy resistance. Lysine acetyltransferase 6 (KAT6A/B), key regulators of chromatin structure and gene expression, have emerged as promising therapeutic targets in HR+ breast cancer. This review highlights the molecular mechanisms by which KAT6A/B modulate histone acetylation and gene regulation, emphasizing their roles in oncogenic pathways and cellular processes such as transcriptional activation, chromatin remodeling, and regulation of cellular senescence. We discuss preclinical evidence supporting the therapeutic potential of KAT6 inhibition and the challenges faced in epigenetic drug development, including the need for robust biomarker-driven approaches. Finally, we explore the prospects for integrating KAT6 inhibitors with existing therapies to overcome resistance and improve patient outcomes in breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/882962d718736529869b6bb59db5c43e6a4e7b49" target='_blank'>
              Exploring KAT6 as a therapeutic target in breast cancer: epigenetic approaches for precision medicine
              </a>
            </td>
          <td>
            Cyril Roussel-Simonin, Aranzazu Fernandez-Martinez, S. Postel-Vinay, F. Mosele, Alessandro Viansone, C. Massard, Adrien Mouren, Julien Vibert, B. Pistilli
          </td>
          <td>2025-12-01</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="As a type of cell with self-renewal ability and multi-directional differentiation potential, stem cells are closely related to their functions, such as reprogramming transcription factors, histone modifications, and energy metabolism. m6A (N6-methyladenosine modification) is one of the most abundant modifications in RNA, and dynamic reversible m6A modification plays an important role in regulating stem cell function. This review moves beyond listing isolated functions and instead adopts an integrated perspective, viewing m6A as a temporal regulator of cellular state transitions. We discuss how m6A dynamically regulates stem cell pluripotency, coordinates epigenetic and metabolic reprogramming, and serves as a central hub integrating key signaling pathways (Wnt, PI3K-AKT, JAK-STAT, and Hippo). Finally, using somatic reprogramming as an example, we elucidate the stage-specific role of m6A in complex fate transitions. This comprehensive exposition not only clarifies the context-dependent logic of m6A regulation but also provides a precise framework for targeting the m6A axis in regenerative medicine and cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e397129d10e01c5414b01cf61047665a5c7f45a" target='_blank'>
              The Regulatory Role of m6A Modification in the Function and Signaling Pathways of Animal Stem Cells
              </a>
            </td>
          <td>
            Xiaoguang Yang, Yongjie Xu, Suaipeng Zhu, Mengru Wang, Hongguo Cao, Lizhi Lu
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b30edb27f07cff97af34aacf8166230cf5b8614" target='_blank'>
              Unsupervised Identification of Cancer Attractor States through the Lens of Embryonic Origin and Cancer Hallmarks
              </a>
            </td>
          <td>
            P. Deshpande, Prabal Deb, Ibrahim Al Haddabi, B. Itkin
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, and although immunotherapy has achieved remarkable breakthroughs, its efficacy is often limited by tumor-induced immunosuppression within the tumor microenvironment (TME). Emerging evidence indicates that metabolic reprogramming plays a pivotal role in shaping the TME and regulating antitumor immune responses. Targeting tumor and immune cell metabolism has therefore become a promising strategy to enhance the effectiveness of cancer immunotherapy. This review first summarizes the metabolic reprogramming that occurs within the TME, including alterations in glucose, lipid, and amino acid metabolism in tumor cells, as well as the metabolic adaptation of immune cells. We then highlight recent advances in natural products that modulate key metabolic pathways and their potential to reshape the immunosuppressive TME. Special emphasis is placed on natural compounds that not only inhibit tumor cell metabolism but also restore the metabolic fitness of immune cells, thereby improving antitumor immunity. In addition, advances in delivery strategies, including nanocarrier-based and stimuli-responsive systems, are reviewed for their roles in improving the bioavailability, stability, and tumor targeting of natural metabolism-regulating agents. Finally, we discuss the current status and challenges of translating natural metabolism-regulating agents into clinical applications, including issues of dose optimization, safety evaluation, and patient selection. Despite these hurdles, precision targeting of metabolic pathways, interdisciplinary collaboration, and the discovery of novel compounds—particularly immune-sensitizing agents derived from traditional medicine—are expected to accelerate progress. Collectively, natural products represent promising adjuvant strategies for cancer immunotherapy, with great potential to overcome current therapeutic limitations and improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c77bd01b532b4a1633bcd32fd11764fdfa763e" target='_blank'>
              Natural products as metabolic modulators to enhance cancer immunotherapy: reprogramming the tumor microenvironment
              </a>
            </td>
          <td>
            Xiaoyu Fan, Shushu Guo, Wanfang Li, Yingyin Wang, Jie Bao, Jiuming He, Hongtao Jin
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The stable and safe integration of exogenous DNA into the genome is crucial to both genetic engineering and gene therapy. Traditional transgenesis approaches, such as those using retroviral vectors, result in random genomic integration, posing the risk of insertional mutagenesis and transcriptional dysregulation. Safe harbor sites (SHSs), genomic loci that support reliable transgene expression without compromising endogenous gene function, genomic integrity, or cellular physiology, have been identified and characterized across various model organisms. Well-established SHSs such as AAVS1, ROSA26, and CLYBL are routinely utilized for targeted transgene integration in human cells. Recent advances in genome architecture, gene regulation, and genome editing technologies are driving the discovery of novel SHSs for precise and safe genetic modification. This review aims to provide a comprehensive overview of SHSs and their applications that will guide investigators in the choice of SHS, especially when complementary sites are needed for more than one transgene integration. First, it outlines safety and functional criteria that qualify a genomic site as a safe harbor site. It then discusses the two primary strategies for identifying SHSs: i) traditional lentiviral-based random transgenesis, and ii) modern genome-wide in silico screening followed by CRISPR-based validation. This review also provides an updated catalogue of currently known SHSs in the human genome, detailing their characteristics, uses, and limitations. Additionally, it discusses the diverse applications of SHSs in basic research, gene therapy, CAR T cell-based therapy, and biotechnological production systems. Finally, it concludes by highlighting challenges in identifying universally applicable SHSs and outlines future directions for their refinement and validation across biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/600741ca6d853466b695798a17050f6f9f6dbe8e" target='_blank'>
              Human Genome Safe Harbor Sites: A Comprehensive Review of Criteria, Discovery, Features, and Applications
              </a>
            </td>
          <td>
            Amer Ahmed, D. Di Molfetta, G. N. Iaconisi, Antonello Caponio, Ansu Singh, A. Bibi, V. Dolce, Luigi Palmieri, Vincenzo Coppola, G. Fiermonte
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2f5392a871706eceb6c7ad5228a2907ac09eeb" target='_blank'>
              Association of copy number alterations with the immune transcriptomic landscape in cancer.
              </a>
            </td>
          <td>
            S. Loipfinger, A. Bhattacharya, C. G. Urzúa-Traslaviña, M. V. van Vugt, M. de Bruyn, Rudolf S. Fehrmann
          </td>
          <td>2026-01-17</td>
          <td>NPJ systems biology and applications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3ae1ba137c707051b533b78e7831f38e7fb1ac" target='_blank'>
              Metabolic reprogramming-driven resistance to multi-kinase inhibitors in hepatocellular carcinoma: molecular mechanisms and therapeutic opportunities.
              </a>
            </td>
          <td>
            Junxin Li, Yu Huang, Jiawei Li, Ming Shi, Yi Xiao, Fei Du, Gongli Hu
          </td>
          <td>2026-01-27</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Blood cancers such as leukemia, lymphoma, and myeloma remain refractory in many patients due to immune escape, antigen heterogeneity, and therapy‑related toxicities. To address these challenges, we review recent strategies that harness CRISPR‑engineered gut commensals as precision “living therapeutics” to modulate host immunity and directly target malignant clones. We frame this review around three principal themes: (1) mechanistic strategies whereby CRISPR-engineered commensals modulate host immunity and directly antagonize malignant clones; (2) the enabling technologies and delivery/containment platforms, CRISPR variants, phage/LNP delivery, genetic circuits and biocontainment, that make living therapeutics feasible; and (3) translational progress, outstanding technical and safety barriers, and ethical/regulatory challenges that must be addressed for clinical deployment. To illustrate these themes, we discuss three concrete therapeutic modalities: engineered microbial secretion of immunomodulators, targeted delivery of tumor-lytic payloads, and engineered production of anticancer metabolites, and how these are enabled by contemporary CRISPR and synthetic-biology toolkits. Selected preclinical models report substantial antitumor effects, often >60% tumor reduction in rodent studies, and restoration of CAR-T cell function in controlled settings; however, effect sizes vary across models, and human translation remains unproven. We also analyze key technical barriers, strain stability, biocontainment, off‑target effects, and propose solutions, including auxotrophic kill-switches and AI‑guided strain optimization. Finally, we outline future directions, from in situ phage delivery to multi‑omics–driven patient stratification. CRISPR‑microbiome editing represents a paradigm shift in hematologic oncology, offering localized, sustained therapy with reduced systemic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e49361664a6596c9e706abe9f5db117dba2b21fd" target='_blank'>
              CRISPR-engineered microbiome: living therapeutics revolutionize blood cancer immunotherapy
              </a>
            </td>
          <td>
            Fang Cheng, Hamed Soleimani Samarkhazan, Yeganeh Khazaei
          </td>
          <td>2025-12-13</td>
          <td>NPJ Biofilms and Microbiomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59601a1a811d20ae0b8430613e168e0ec52452b" target='_blank'>
              Multimodal antigenic escape to GPRC5D-targeted T cell engagers in multiple myeloma.
              </a>
            </td>
          <td>
            Holly Lee, Sungwoo Ahn, Gerone A. Gonzales, Noémie Leblay, E. Barakat, Dylan Greening, Pina Colarusso, Johnathan Canton, S. Benaoudia, Elham Hasheminasabgorji, Mansour Poorebrahim, David Jung, M. Truger, Jihong Jeong, C. Riedhammer, H. Einsele, K. M. Kortüm, Jan Eckmann, Jitka Somandin, Sara-Sheena Engel, L. Boise, J. Frenking, Niels Weinhold, Konstantina Taouxi, E. Kastritis, Sheri Skerget, D. Vishwamitra, Yunje Cho, F. Maura, Marc-Steffen Raab, J. Corre, L. Rasche, Hervé Avet-Loiseau, Paola Neri, Nizar J. Bahlis
          </td>
          <td>2026-01-15</td>
          <td>Nature medicine</td>
          <td>1</td>
          <td>76</td>
        </tr>

        <tr id="Background

Immune checkpoint inhibitors have transformed the treatment landscape of non-small cell lung cancer (NSCLC); however, primary and acquired resistance remain major clinical challenges. Immune evasion in NSCLC is increasingly recognized as a complex, dynamic process driven by tumor heterogeneity and adaptive remodeling of the tumor microenvironment. Conventional bulk profiling approaches are limited in resolving this complexity, underscoring the need for higher-resolution methodologies.

Objective
This review aims to synthesize current evidence on how single-cell transcriptomic approaches have advanced the understanding of immune evasion mechanisms in treatment-resistant NSCLC, with a focus on tumor-intrinsic programs, immune cell dysfunction, and intercellular communication networks.

Methods

A structured narrative review with systematic elements was conducted, analyzing peer-reviewed studies employing single-cell RNA sequencing and related single-cell technologies in human NSCLC. Studies were evaluated for their insights into immune escape pathways, resistance to immunotherapy, and translational relevance.

Results

Single-cell transcriptomic analyses reveal profound intratumoral and immune heterogeneity in treatment-resistant NSCLC. Resistant tumors harbor distinct malignant cell states characterized by impaired antigen presentation, dysregulated interferon signaling, and lineage plasticity. Within the immune compartment, hierarchical T-cell exhaustion, depletion of progenitor-like exhausted T cells, and expansion of immunosuppressive myeloid populations—particularly tumor-associated macrophages—emerge as dominant features of resistance. Ligand–receptor interaction analyses further demonstrate coordinated immunosuppressive communication networks that sustain immune escape at a systems level. Several single-cell–defined cellular states correlate with poor response to immune checkpoint blockade and adverse clinical outcomes.

Conclusion

Single-cell transcriptomics has fundamentally reshaped the conceptual framework of immune resistance in NSCLC, redefining it as a multicellular, dynamic ecosystem rather than a single-pathway phenomenon. These insights provide a strong biological rationale for developing biomarker strategies and rational combination therapies that target both tumor-intrinsic adaptations and the immunosuppressive microenvironment. Continued integration of single-cell technologies into translational and clinical research is essential to overcome immunotherapy resistance and improve patient outcomes in NSCLC.

Keywords: Non-small cell lung cancer; Single-cell RNA sequencing; Immune evasion; Immunotherapy resistance; Tumor microenvironment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7495ff2bd82ff3331af5929ec1f5ca0f1ed50c9" target='_blank'>
              Single-Cell Transcriptomic Mapping of Immune Evasion Pathways in Treatment-Resistant Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Marco Rossi, Giulia Bianchi, Alessandro Conti
          </td>
          <td>2026-01-19</td>
          <td>Advanced Journal of Biomedicine &amp; Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background/Objectives: Comprehensive genomic profiling (CGP) is a cornerstone of personalized oncology. However, large-scale, systematic data on the somatic mutation spectrum in Russian cancer patients are scarce. This study aimed to characterize the genomic landscape and assess the potential for matched therapy in a Russian cohort of patients with solid tumors. Methods: This retrospective study included 204 patients with various solid tumors. CGP was performed using the FoundationOne®CDx (FFPE tissue) and FoundationOne®Liquid CDx (cfDNA) platforms. The analysis assessed single-nucleotide variants, indels, copy number alterations, gene fusions, tumor mutational burden (TMB), microsatellite instability (MSI), and PD-L1 expression. Results: The most frequently mutated genes were TP53 (61.5%) and KRAS. The median TMB was 4.0 mut/Mb and was significantly lower in stage IV tumors. Significant co-occurrence was observed between KRAS and TP53 mutations, as well as between APC and KRAS mutations, which were particularly characteristic of colorectal cancer. KRAS mutations were associated with higher combined positive score (CPS) values in cases with lung cancer. Based on the CGP results, 44% of patients had findings that supported the use of an approved matched targeted therapy or immunotherapy for their tumor type. An additional 36% of patients had alterations indicating potential benefit from off-label targeted therapy. Conclusions: This study reveals the distinct genomic characteristics of solid tumors in a Russian cohort and confirms the high clinical utility of CGP for identifying actionable targets. Implementing CGP early in the diagnostic process is a necessary step towards realizing personalized treatment strategies for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66aca90a2bac69b97335224b751826ae6cb88014" target='_blank'>
              Genetic Landscape of Solid Malignant Tumors in a Russian Cohort of Patients
              </a>
            </td>
          <td>
            Iurii K. Slepov, E.D. Kopylov, Anton A. Turchin, D. Khmelkova, V. Kaimonov, A. Isaev, R. Deev
          </td>
          <td>2025-12-19</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Malignant transformation of mature cystic teratoma (MTMCT) of the ovary is a rare but aggressive malignancy for which no standardized chemotherapy or effective targeted therapies currently exist. To identify therapeutic vulnerabilities in MTMCT, we performed a genome-wide CRISPR-Cas9 knockout screen using the MTMCT-derived NOSCC1 cell line. Two parallel selective pressures were applied: in vivo tumorigenicity in immunodeficient mice and cisplatin exposure in vitro. From this screen, 67 negatively selected genes were identified, among which SOD1 and NDUFB4 emerged as top candidates based on high basal expression levels and clinical relevance. Integration with spatial transcriptomic data from three independent MTMCT patient tumors further supported the prioritization of these targets. SOD1 was selected for further investigation due to the availability of known pharmacological inhibitors. Both siRNA-mediated knockdown and small-molecule inhibition of SOD1 using LCS-1 significantly suppressed MTMCT cell proliferation in vitro by inducing oxidative stress and impairing cell cycle progression. This antiproliferative effect was reversed by co-treatment with N-acetylcysteine, a reactive oxygen species scavenger. In vivo validation using patient-derived xenograft models demonstrated that oral administration of LCS-1 led to significant tumor growth suppression and increased expression of apoptotic and DNA damage markers, including cleaved caspase-3 and γH2AX. These findings establish SOD1 as a critical vulnerability in MTMCT and provide preclinical evidence supporting redox modulation as a therapeutic strategy for this highly chemoresistant and understudied ovarian cancer subtype. Our integrative approach combining functional genomics, spatial transcriptomics, and pharmacologic validation offers a framework for the discovery of novel targets in rare gynecologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e67036000a59a86b2e3dd28f5c317d050f6d45c" target='_blank'>
              In-Tumor CRISPR-Cas9 Knockout Screening and Novel Therapy Development for Malignant Transformation of Ovarian Teratoma.
              </a>
            </td>
          <td>
            S. Tamauchi, Kosuke Yoshida, Xinyuan Wang, Atsushi Nakagawa, Akira Yokoi, N. Yoshikawa, K. Niimi, Yusuke Yamamoto, Hiroaki Kajiyama
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Simple Summary MELK is a serine/threonine kinase that regulates cell cycle progression, proliferation, apoptosis, migration, and stemness. MELK is overexpressed in many solid and hematologic malignancies. Furthermore, high MELK levels correlate with aggressive disease, poor survival, treatment resistance, and cancer stem-like features. Several small-molecule inhibitors and RNAi-targeting MELK have suppressed the development and progression of various cancers. OTSSP167 has shown promising results in phase I and II clinical trials. However, a study employing CRISPR/Cas9 knockout in breast cancer cell lines reported that loss of MELK did not impair proliferation. The review concludes that MELK is a robust prognostic biomarker of proliferation and aggressive behavior across multiple cancers but is not a universal driver of tumor growth. Discrepancies may arise from off-target effects of RNAi or inhibitors, context differences across cancer types, and differences in study settings. MELK is proposed as a conditional oncogenic vulnerability, especially in p53-deficient triple-negative breast cancer (TNBC) and glioma stem cells. Further research is needed to clarify the target selectivity of inhibitors, context-specific dependencies, and the optimal use of MELK-focused agents in combination therapies rather than as standalone treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0e62df49d62d42a4c4ea852f26c65300bd99df5" target='_blank'>
              Maternal Embryonic Leucine Zipper Kinase (MELK) in Cancer: Biological Functions, Therapeutic Potential, and Controversies
              </a>
            </td>
          <td>
            Alaeddin M. Alzeer, Saad Al-Lahham
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosome 3p (chr3p) is frequently deleted in multiple cancers, indicating the presence of shared tumor suppressors. Analysis of genomic alterations in 33 different cancer types implicates the deletion or deleterious mutations of SET-domain-containing 2 (SETD2) at chr3p21 in significantly facilitating the formation of isochromosomes, consisting of two identical mirror-imaged arms, thereby promoting genomic instability conducive to large-scale chromosomal rearrangements and rapid cancer genome evolution. Fracturing of dicentric isochromosomes during cell division is pervasive and follows the dynamic fragmentation pattern of solids under impulse. Across cancers, isochromosomes form most frequently on chr8 to amplify the MYC-containing q-arm and chr17 to delete the TP53-containing p-arm. In the most aggressive uveal melanoma (UVM) subtype, chr3 deletion also includes MITF, a critical melanocyte differentiation and survival factor, and co-occurs with chr8q amplification. We demonstrate that MITF is a master transcriptional regulator of GNAQ/GNA11, mutated in 90% of UVM patients, and the associated synthetic-lethal genes identified by recent CRISPR screening studies. MITF maintains MAPK and calcium homeostasis in UVM, and its deletion is thus accidental, creating an early crisis during oncogenesis. We further show that MITF, MYC, and GNAQ/GNA11 form coupled regulatory feedback loops in the melanocyte lineage, and MITF deletion in UVM creates acute dependency on MYC-mediated rescue via chr8q amplification, often as a consequence of isochromosome formation. The discovered feedback loops predict both overall and relapse-free patient survival within the most aggressive UVM subtype, explain sensitivity to therapeutic gene perturbations, and inform effective combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f31378489089eb1239e625480e1522a21f193cf" target='_blank'>
              Chromosome 3p deletion leads to extensive genomic alterations in diverse cancers and confers synthetic lethality in uveal melanoma
              </a>
            </td>
          <td>
            Mitchell C. Cutler, P. B. Howland, M. Hejna, Jun S. Song
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Introduction Chromosomal instability (CIN), a hallmark of over 90% of solid tumors, presents both a vulnerability and an opportunity in cancer therapy. KIF18A, a kinesin motor protein essential for mitotic fidelity, becomes critical for CIN cancer cells, where it suppresses lethal mitotic errors. This selective dependency makes KIF18A an attractive therapeutic target. Areas covered This review explores the structural and functional biology of KIF18A, its role in maintaining chromosomal alignment, and how its inhibition leads to mitotic failure specifically in cancer cells exhibiting high CIN. We summarize the development of small-molecule inhibitors, from early compounds like BTB-1 to second-generation agents including Sovilnesib and VLS-1488, the latter currently in Phase I/II trials. Clinical data show manageable toxicity and promising antitumor activity, particularly in high-grade serous ovarian cancer. New entrants like ATX-295 and GH2616 further expand the landscape. We also highlight efforts to refine KIF18A targeting through structural studies and emerging AI-driven drug discovery. Expert opinion KIF18A inhibition represents a mechanistically grounded, tumor-selective strategy for treating CIN-driven cancers. As clinical trials progress, optimizing patient stratification and exploring synergistic combinations will be crucial. Broadening the range of druggable sites on KIF18A, especially beyond the motor domain, could mitigate resistance and help expand therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b172472dda432c5dc196b15477d3bfabc81ddaf1" target='_blank'>
              KIF18A inhibition and its rising role in cancer therapy trials: from bench to bedside
              </a>
            </td>
          <td>
            Jia Q Truong, Tal Galper, Jessica K. Holien
          </td>
          <td>2026-01-02</td>
          <td>Expert Opinion on Investigational Drugs</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The mechanisms underlying cancer cell drug resistance have not yet been fully elucidated. Numerous processes hinder effective tumor treatment, such as cancer cell epigenetic alterations, DNA repair, drug target modification, and epithelial–mesenchymal transition. Recent studies have highlighted the ability of cancer cells to transiently enter the resting phase, namely, the G0 phase of the cell cycle under treatment conditions; this is associated with an elevated level of cell death. Cells in the G0 phase are characterized by a specific phenotype, the exact features of which remain to be fully elucidated. Cell division and cell cycle dynamics are closely associated with cell adhesion, i.e., interaction with the extracellular matrix in normal cells, whereas cancer cells are characterized by the dys-coordination of these processes. However, the precise patterns of cell adhesion alterations in association with the cell cycle remain unclear. Knowledge on the interactions of cells in the G0 phase with the extracellular matrix during the entry, the exit, and the sustenance during the resting phase is limited. At the same time, focal adhesion-mediated molecules may be an attractive goal for cancer cells in the G0 phase, targeting to overcome tumor dissemination and drug resistance. Drug resistance and immune system response evasion are specific features of nonproliferating cancer cells that call for more detailed and accurate characterization of this population of malignant cells to optimize anticancer treatment and control tumor dissemination. The present review represents the recent data on the association of focal adhesion and cancer cell surveillance due to cell cycle alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16149b6070228042071985c6c4e07c2cd03bc235" target='_blank'>
              Focal Adhesion and Cancer Cell Cycle-Associated Resistance
              </a>
            </td>
          <td>
            T. Ruksha, N. Palkina, Ekaterina Z. Lapkina, Kangdong Liu
          </td>
          <td>2025-12-25</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cellular memory, or epigenetic memory, represents the capacity for cells to retain information beyond the underlying DNA sequence. This heritable characteristic is primarily governed by epigenetic mechanisms which enable cells to maintain specialized characteristics across divisions. This persistent cellular state is essential for fundamental biological processes, such as maintaining tissue identity and facilitating cell differentiation, especially embryonic cells. Early-stage perturbations such as assisted reproductive technologies (ART) and nutritional stress links embryonic exposures to adult health and disease within the Developmental Origins of Health and Disease (DOHaD) framework. Crucially, memory established during early embryogenesis links these epigenetic modifications to adult long-term phenotypes related to metabolic disorders. These modifications—including DNA methylation, histone modifications, and non-coding RNAs—support cellular memory transmission across cell divisions, and in certain organisms, can be transmitted across generations without alterations to the DNA sequence. This review synthesizes recent advances in epigenetic pathways that mediate cellular memory, highlights critical preimplantation windows of vulnerability and outlines gaps necessary for mammalian developing interventions that safeguard future generations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d6f5c2aa2e2cb595481c0a26f1959e68f5a807" target='_blank'>
              How Early-Life Programming During Embryogenesis Imprints Cellular Memory
              </a>
            </td>
          <td>
            Norermi Firzana Alfian, Kei Uechi, Yoshiya Morishita, Kaname Sato, Maruhashi Yui, Jannatul Ferdous Jharna, Md Wasim Bari, Shiori Ishiyama, Kazuki Mochizuki, Satoshi Kishigami
          </td>
          <td>2025-12-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mitochondria function as the primary energy hubs of cells and possess semi-autonomous genetic characteristic. Mutations in mitochondrial DNA (mtDNA) frequently lead to severe illness and even premature death. The rapid advancement of gene therapy offers promising potential for correcting such disorders. This review first aims to delineate the mechanisms of gene therapy strategies applicable to mitochondrial diseases, including the allotopic expression of mtDNA in the nucleus, mitochondrial-targeted nuclease cleavage, and mtDNA-targeted base editing. It also discusses in detail the clinical efficacy of mtDNA allotopic expression and the preclinical progress of other strategies. Furthermore, the unique physiological features of mitochondria, such as heteroplasmy and independent molecular transport mechanisms, pose distinct challenges for the clinical implementation of mitochondrial gene therapy strategies. Accordingly, this review elaborates on the current limitations of each approach. Finally, it highlights potential optimization directions to address these challenges, emphasizing that understanding heteroplasmy dynamics and their corresponding phenotypes, ensuring the safe delivery and tissue-specific expression of therapeutic elements, and maintaining long-term therapeutic specificity and efficiency are essential for the clinical translation of mitochondrial gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19af52e15b8608097010e37fd7cc07ef155c45a1" target='_blank'>
              Advances in gene therapy for mitochondrial genetic disorders: current status and clinical implementation challenges
              </a>
            </td>
          <td>
            Lei Lyu, Beibei Qie, Yanjie He, Feilong Chen, Bao Liu
          </td>
          <td>2025-12-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) is escalating worldwide, posing a serious threat to global public health by driving infections that are no longer treatable with conventional antibiotics. CRISPR–Cas technology offers a programmable and highly specific therapeutic alternative by directly targeting the genetic determinants responsible for resistance. Various CRISPR systems can restore antibiotic susceptibility and induce selective bactericidal effects by eliminating resistance genes, disrupting biofilm formation, and inhibiting virulence pathways. Moreover, CRISPR can suppress horizontal gene transfer (HGT) by removing mobile genetic elements such as plasmids, thereby limiting the ecological spread of AMR across humans, animals, and the environment. Advances in delivery platforms—including conjugative plasmids, phagemids, and nanoparticle-based carriers—are expanding the translational potential of CRISPR-based antimicrobial strategies. Concurrent progress in Cas protein engineering, spatiotemporal activity regulation, and AI-driven optimization is expected to overcome current technical barriers. Collectively, these developments position CRISPR-based antimicrobials as next-generation precision therapeutics capable of treating refractory bacterial infections while simultaneously suppressing the dissemination of antibiotic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f0f21606a1f66b09cc62a141dac0135cf1b347" target='_blank'>
              CRISPR–Cas-Mediated Reprogramming Strategies to Overcome Antimicrobial Resistance
              </a>
            </td>
          <td>
            Byeol Yoon, Jang Ah Kim, Yoo Kyung Kang
          </td>
          <td>2026-01-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genomic imprinting disorders (IDs) arise from the disruption of parent-of-origin-specific gene expression, a process governed by heritable epigenetic marks. Conventional therapies are largely symptomatic and fail to correct the underlying defect in dynamic gene regulation. Epigenome editing, using programmable DNA-targeting tools to rewrite epigenetic information, has emerged as a powerful therapeutic strategy that directly addresses this molecular pathology. This review synthesizes the transformative progress of the past decade, arguing that epigenome editing is poised to revolutionize the treatment of these conditions. We detail key breakthroughs in both editor design and in vivo delivery. These include the development of next-generation editors, such as compact “hit-and-run” systems that establish durable epigenetic memory after only transient expression—a critical feature for clinical safety. We then examine parallel advances in delivery platforms, including engineered adeno-associated virus (AAV) vectors with enhanced central nervous system (CNS) tropism and programmable lipid nanoparticles (LNPs) for precise extra-hepatic targeting. The therapeutic potential of combining these technologies is underscored by convincing preclinical data in models of Angelman (AS) and Prader-Willi syndromes (PWS), where the targeted rewriting of epigenetic marks has successfully reactivated silenced alleles and rescued disease-relevant phenotypes. Finally, we discuss the remaining challenges—including long-term durability and off-target safety—that must be overcome to translate this scientific potential into transformative medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a215ce891005fe8fac9034ff5a7f3671a19208" target='_blank'>
              Advances and challenges of precision epigenetic therapy in treating genomic imprinting diseases
              </a>
            </td>
          <td>
            Ying Lv, Mingyan Li, Chai Ji
          </td>
          <td>2026-01-01</td>
          <td>Translational Pediatrics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mechanisms driving the increase in cell growth in developing leukemia are not fully understood. We focused on epigenomic regulation of this process by analyzing the changes of chromatin marks and gene expression in leukemic cell clones as they progressed toward increased proliferation in a mouse model of acute myeloid leukemia (AML). This progression was characterized by gradual modulation of chromatin states and gene expression across the genome, with a surprising preferential trend of reversing the prior changes associated with the origins of leukemia. Our analyses of this modulation in independently developing clones predicted a small set of potential growth regulators whose transcriptomic and epigenomic progression was consistent between clones and maintained both in vivo and ex vivo. We selected three of these genes as candidates (Irx5 and Plag1 as growth suppressors and Smad1 as a driver) and successfully validated their causal growth effects by overexpression in mouse leukemic cells. Overexpression of the IRX5 gene in human MOLM13 leukemic cells suppressed cell growth both in vitro and in mouse xenografts. Public patient data confirmed expression levels of PLAG1 and SMAD1 as markers of AML status and survival, suggesting that multiomic analysis of evolving clones in a mouse model is a valuable predictive approach relevant to human AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e782878493e2646d7e50204cd904395a449780e" target='_blank'>
              Multiomic analysis of clonal development reveals new regulators of leukemic cell growth.
              </a>
            </td>
          <td>
            Gracia Bonilla, Alexander Morris, Sharmistha Kundu, Anthony Ducasse, Grace Kirkpatrick, Jelena Milosevic, N. E. Jeffries, Kashish Chetal, Emma E Yvanovich, Ting Zhao, Jun Xia, Rana Barghout, David T. Scadden, Michael K. Mansour, Robert E. Kingston, David B. Sykes, François E. Mercier, Ruslan I. Sadreyev
          </td>
          <td>2026-02-04</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Elucidating the evolution of cancers allows us to understand their key events, and the order in which they occur. To chart and interpret these evolutionary trajectories, we leverage whole-genome sequencing of lung tumours, including those from the largest cohort to date of lung cancers in subjects who have never smoked. Through ordering frequent genomic alterations, we discover three distinct evolutionary paths taken by lung adenocarcinomas; two dominated by tumours from people who have never smoked (NS-LUAD), and one followed by the vast majority of those who have smoked (S-LUAD). However, one in six NS-LUAD follow the smoking-dominant trajectory. These tumours, surprisingly, have fewer somatic alterations than the other NS-LUAD, and have shorter latency. They are strongly enriched for KRAS mutations. Our results suggest that gaining KRAS mutations allows these tumours to evolve more rapidly, acquiring a set of smoking-associated key alterations, with less need for genomic instability to progress. These tumours are three times more frequent in subjects of European vs. East Asian ancestry. These findings could shape clinical management strategies for lung adenocarcinoma patients, particularly for tumours driven by smoking-like evolutionary trajectories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b7e5dadb5cdb44bd67cb1c0260a2c8d918874c1" target='_blank'>
              Revealing the Drivers Underlying Distinct Evolutionary Trajectories in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Christopher Wirth, Tongwu Zhang, M. Díaz-Gay, Christopher D. Steele, Phuc H. Hoang, Yang Yang, Azhar Khandekar, Wei Zhao, Jian Sang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, P. Joubert, R. Homer, S. Yang, Thi-Van-Trinh Tran, John P McElderry, Caleb Hartman, M. Miraftab, Olivia W. Lee, Kristine M. Jones, Bin Zhu, Jacobo Martínez Santamaría, M. Schabath, Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, V. Janout, D. Mates, Simona Ognjanovic, Milan Savic, M. Kontić, Y. Bossé, Bonnie E. Gould Rothberg, D.C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, M. Wong, Kin Chung Leung, Chih-Yi Chen, I-Shou Chang, C. A. Hsiung, A. Pesatori, Dario Consonni, Nathaniel Rothman, Qing Lan, Martin A. Nowak, S. Chanock, Jianxin Shi, Lixing Yang, Ludmil B. Alexandrov, D. Wedge, M. Landi
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) has become a cornerstone technology in modern genomics, fundamentally changing how genetic information is generated, analyzed, and applied. This review explores the expanding role of NGS across both research and clinical settings, emphasizing its transformative impact on precision medicine while also addressing existing technical and analytical challenges. An extensive review of contemporary literature was undertaken to evaluate recent progress in sequencing platforms, library preparation strategies, data processing workflows, and downstream analytical methods. Particular attention is given to the application of NGS in the study of rare genetic diseases, cancer genomics, infectious disease surveillance, microbiome analysis, population genetics, and functional genomics. By enabling the parallel sequencing of millions of DNA and RNA fragments, NGS provides unparalleled resolution into genomic architecture, sequence variation, gene expression dynamics, and epigenetic regulation. These capabilities have significantly advanced diagnostic accuracy, facilitated the development of targeted and personalized therapeutic strategies, and improved disease risk assessment. Emerging innovations, including long-read sequencing technologies, single-cell sequencing approaches, and integrative epigenomic profiling, are further enhancing the analytical power and clinical relevance of NGS. Despite these advancements, several limitations continue to hinder the full potential of NGS, such as challenges related to sequencing accuracy, data interpretation, cost constraints, and the scalability of bioinformatics infrastructures required to manage large and complex datasets. Addressing these issues through improvements in sequencing chemistry, computational algorithms, and cost-efficiency will be critical for broader clinical adoption. Overall, NGS continues to bridge the gap between genomic research and clinical implementation, and ongoing technological and analytical innovations are expected to expand its accessibility and impact across global healthcare and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d824a324e3bb7609410298c882f9ddc4fa49330b" target='_blank'>
              Next-Generation Sequencing (NGS) Technologies: Transformative Impact on Genomics and Personalized Medicine
              </a>
            </td>
          <td>
            Uday Kiran Bathini, Vaishnavi Burra, Lahari Vuddanti, Seetharam Gude
          </td>
          <td>2026-01-02</td>
          <td>Journal of International Research in Medical and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2879e795e7efcebd514454af1fc1a736f45a29d8" target='_blank'>
              Somatic and germline mutational processes across the tree of life
              </a>
            </td>
          <td>
            Sangjin Lee, Yichen Wang, Haynes Heaton, Emily Mitchell, M. Maddison, Liam M. Crowley, Patrick Adkins, N. Mieszkowska, Mark Blaxter, R. Durbin, P. Campbell
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="BACKGROUND
Cancer cells are heterogeneous, each harboring distinct molecular aberrations and being dependent on different genes for their survival and proliferation. While targeted therapies based on driver DNA mutations have shown success, many tumors lack druggable mutations, limiting treatment options. We hypothesize that new precision oncology targets may be identified through "expression-driven dependency," where cancer cells with high expression of specific genes are more vulnerable to the knockout of those same genes.


RESULTS
We developed BEACON, a Bayesian approach to identify expression-driven dependency targets by analyzing global transcriptomic and proteomic profiles alongside genetic dependency data from cancer cell lines across 17 tissue lineages. BEACON successfully identified known druggable genes, including BCL2, ERBB2, EGFR, ESR1, and MYC, while revealing novel targets confirmed by both mRNA and protein-expression driven dependency. The identified genes showed a 3.8-fold enrichment for approved drug targets and a 7 to 10-fold enrichment for druggable oncology targets. Experimental validation demonstrated that depletion of GRHL2, TP63, and PAX5 effectively reduced tumor cell growth and survival in their dependent cells.


CONCLUSIONS
Our approach provides a systematic method to identify precision oncology targets based on expression-driven dependency patterns. By integrating multi-omics data with genetic dependency screens, we've created a comprehensive catalog of potential therapeutic targets that may expand treatment options for cancer patients lacking druggable mutations. This resource offers new opportunities for precision oncology target discovery beyond mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d9df8d8c54b9f6f78829d05c5029fee1f72554" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Oncology.
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Zishan Wang, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2026-01-29</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Activating mutations in KRAS occur in approximately 30% of lung adenocarcinomas. Despite advances in RAS-targeted therapies, intrinsic resistance limits their long-term efficacy. Here, we identify elevated levels of wild-type KRAS (WT-KRAS) protein as a key driver of intrinsic resistance in KRAS-mutant lung tumors. KRAS accumulation results from impaired LZTR1-mediated degradation, triggered either by LZTR1 loss or pharmacological RAS inhibition. Stabilized WT-KRAS activates the mTOR/HIF1α pathway by promoting lysosomal recruitment of the SLC3A2/SLC7A5 amino acid transporter complex, reprogramming lysosomal amino acid sensing. Shallow deletions of LZTR1, present in up to 40% of KRAS-mutant lung adenocarcinomas, are associated with increased mTOR activity and may contribute to therapeutic resistance to RAS inhibitors. Co-inhibition of mTOR or the SLC3A2/SLC7A5 complex using dactolisib or JPH203 restores sensitivity to KRAS inhibitors in vitro and in vivo. These findings support combinatorial targeting of mTOR signaling or amino acid transport to overcome intrinsic resistance in KRAS-mutant lung cancer. Despite recent advances in targeting RAS, resistance to anti-RAS therapies limits their effectiveness in KRAS-mutant lung cancer. Here, the authors show that RAS inhibitors impair wild-type KRAS degradation, leading to its accumulation and resistance through mTOR, and demonstrate that targeting mTOR or amino acid transport can overcome this resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af1dd8cb117bbc9bc976fa5b962d536a9f69059a" target='_blank'>
              Intrinsic resistance to RAS inhibitors is driven by dysregulation of KRAS degradation
              </a>
            </td>
          <td>
            Tonci Ivanisevic, Yan Ma, Emiel Van Boxel, Wout Magits, Benoit Lechat, Zeynep Koçberber, Phillipp Willnow, S. Fendt, G. Velde, R. Sewduth, A. Sablina
          </td>
          <td>2025-12-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c4664b6580edc429e59cb7fc3bb0cee1ddce94" target='_blank'>
              LISA: A Comprehensive R Package for Lentiviral Integration Site Analysis in Gene Therapy Safety Assessment
              </a>
            </td>
          <td>
            Shuai Ni, Ning Wu, Yun Kang, Fengjiao Zhu, Jueyu Wang, Lina Wang, Wei Wang, Heng Mei, Kejia Kan
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer has widely been described by the classic adenoma-carcinoma sequence, where sequential mutations in APC, KRAS, and TP53 drive the transition from normal epithelium to carcinoma. However, recent high-resolution genomic analyses have expanded this framework by revealing the presence of multiple driver mutations within morphologically normal colonic crypts. In this review, we summarize the emerging landscape of several somatic driver mutations in normal crypts, highlighting genes that were reported to be under positive selection, such as FBXW7, STAG2, AXIN2, PIK3CA, ERBB2, and ERBB3, and discuss their known functions in both normal epithelium and colorectal cancer. We also postulate the functional consequences of these mutations in the overall priming of colonic crypts to potential tumorigenesis. Despite their presence, the frequencies of these mutations in both normal epithelial and colorectal cancers are low, and most of them do not progress to malignancy. We therefore discuss the intrinsic and extrinsic factors that dictate the fate of these crypts. Together, these findings underscore that tumor-initiating events in colorectal cancer may occur much earlier than previously recognized. A deeper understanding of these early driver events may help to better inform strategies for early detection, risk stratification, and prevention before these morphologically normal crypts transform into neoplasia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a28e266bc4daef9955008a254e7922bfd9547bc8" target='_blank'>
              Early mutational events and clonal dynamics in normal crypts: implications for colorectal tumorigenesis
              </a>
            </td>
          <td>
            Charlie Marvalim, Dedrick Kok Hong Chan
          </td>
          <td>2025-12-01</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Rashedunnabi Akanda, Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4773775e6ecfd7cb45043c77dfe433bc00bd2ae5" target='_blank'>
              Epigenetic Intelligence: How Organisms Track Their Environment Through Molecular Memory
              </a>
            </td>
          <td>
            Holly V. Moeller, H. Putnam, Ross Cunning, Steven B. Roberts, J. Eirín-López, R. Nisbet
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="

 IEIs are a heterogeneous group of primarily inherited genetic disorders characterized by impaired immune system function. While most genetic alterations are single nucleotide variants (SNVs), some cases are caused by larger genomic changes such as insertions or deletions that lead to copy number variations (CNVs). To identify potential large deletions in the WAS gene, we developed a molecular strategy combining a novel own design multiplex ligation-dependent probe amplification (MLPA) assay specific to WAS with a droplet digital PCR (ddPCR) assay.



 Three patients with suspected Wiskott–Aldrich syndrome (WAS) were studied. Initial analysis involved Sanger sequencing. When a CNV was suspected, additional methodologies were incorporated: a WAS-specific MLPA test (covering exons 1, 2, 3, 4, 5, 6, and 8) designed according to the manufacturer’s recommendations and a ddPCR assay targeting designed exons 7, 8, 9, and 12.



 None of the patients showed amplification of individual WAS exons by PCR. MLPA analysis revealed a complete absence of all exons in two patients, while the third patient showed absence of exon 8 by MLPA and absence of exons 7, 8, 9, and 12 by ddPCR. Both methodologies allowed for the analysis of women carriers of this X-linked hereditary pathology, leading to the identification of a carrier.



 These patients clearly demonstrate the importance of having more than one screening strategy for efficient diagnosis and family genetic counseling, mostly in cases where the clinical suspicion is very strong but conventional diagnostic strategies yield negative results. In many IEIs, bioinformatic algorithms in next-generation sequencing studies detect potential CNVs that need to be confirmed. Both MLPA and ddPCR are complementary techniques that would allow for a definitive diagnosis in these cases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ac509b654f89f2624ce5daf8a49bbf716d811e" target='_blank'>
              MLPA+Ddpcr: Dual Molecular Strategy to Detect Copy Number Variations (CNVs) in Inborn Error of Immunity (IEI) Genes: A Case Study of the WAS Gene
              </a>
            </td>
          <td>
            V. Goris, E. Prieto, Mariana Villa, M. Oleastro, J. Yancoski
          </td>
          <td>2025-12-22</td>
          <td>Journal of Human Immunity</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Comprehensive genetic characterization and screening for congenital adrenal hyperplasia (CAH) have not yet been achieved at the population level because of the complexity of the CYP21A2 locus. This prospective study incorporated long-read sequencing (LRS) into the current first-tier biochemical newborn screening (NBS) to comprehensively characterize the variant spectrum of CYP21A2, fully investigate the carrier frequency and expected incidence of classic and non-classic CAH (NCCAH), and evaluate the clinical feasibility of genetic NBS for CAH. A total of 21,239 newborns were consecutively recruited from 11 centers across China between June 2023 and May 2024. All the participants underwent biochemical and genetic NBS. In vitro enzymatic activity and minigene assays were performed to determine the pathogenicity of novel variants. A 30.8-kb long amplicon, followed by LRS, was performed to determine the phasing of duplication chimera and single-nucleotide variations (SNVs) and indels in CYP21A2. Eligible genetic screening results were obtained for 21,234 (99.98%) newborns. The allele frequencies of duplications and deletions at the CYP21A2 locus were 4.51% and 0.15%, respectively. In vitro functional analysis and LRS-based phasing were performed to precisely determine carrier alleles, setting an overall frequency of 1.67% (711/42468, 95% confidence interval (CI): 1.55–1.80%), with 0.75% (320/424268, 95% CI: 0.67–0.84%) and 0.92% (391/42468, 95% CI: 0.83–1.01%) for classic and NC carriers, respectively. Notably, hotspot variants including SNVs/indels caused by microgene conversion and 30-kb deletions caused by unequal crossover accounted for 84.0% (597/711), whereas rare variants comprised as high as 16.0% (114/711) of all variants. The expected incidence of classic CAH according to allele frequency was 1/17613. The expected incidence of NCCAH in Chinese population (1/4474) was significantly lower than that in US Ashkenazi Jews (1/133) and Caucasians (1/337), mainly owing to the different allele frequencies of the NC variant CYP21A2:c.844G > T. Biochemical NBS identified 106 (0.50%) positive samples with a positive predictive value of 0.94% (1/106). LRS accurately identified the one case of classic CAH, with no false positives. Our findings provide a population-level carrier frequency and incidence estimates with a comprehensive landscape of the CYP21A2 locus, and demonstrate the effectiveness of first-tier LRS-based genetic NBS for CAH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557caa75d8cd7db35fb409f5e86110581845aa44" target='_blank'>
              Genetic characterization and screening of congenital adrenal hyperplasia by long-read sequencing in a cohort of 21,239 newborns
              </a>
            </td>
          <td>
            Desheng Liang, Min Zhu, Qiaowei Liang, Rong Qiang, Lei Yu, Shiyi Xu, Menglin Li, Jieping Song, Yulin Zhou, Xiaoyan He, Yonglan Huang, Hua Jin, Jianqiang Tan, Hui Liu, Aihua Xia, Yingdi Liu, Peisen Liu, Zhuo Li, Ruifang Wang, Dongjuan Wang, Ruixue Zhang, Qian Pu, Jinfu Zhou, R. Xu, Xudong Wang, M. Tan, Dayu Chen, Chaoyan Wu, Di Cui, Aiping Mao, Wenhao Zhou, Wenjuan Qiu, Lingqian Wu
          </td>
          <td>2025-12-30</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background: The profound heterogeneity of glioblastoma and the often-limited efficacy of conventional treatments, including arsenic trioxide (ATO), underscore the urgent and critical demand for innovative combination strategies specifically designed to overcome treatment resistance. Methods: We evaluated the therapeutic effects of ATO as a single agent and in combination with the MNK1 inhibitor AUM001 across patient-derived xenograft (PDX) models and investigated molecular determinants of sensitivity and synergy. Our results demonstrated that GBM models resistant to ATO, particularly those of the mesenchymal subtype, are more likely to show synergistic cytotoxicity when AUM001 is added. The combination significantly reduces the frequency of glioblastoma stem cells (GSCs) compared to either drug alone, especially in ATO-resistant models. Results: These observations suggest that targeting the MNK1 pathway in conjunction with ATO is a promising strategy to specifically eradicate GSCs, which are major drivers of GBM recurrence and therapeutic failure. Transcriptomic analyses revealed that ATO sensitivity correlated with activated translation-related pathways and cell cycle processes, while synergistic responses to the combination were driven by distinct molecular signatures in different GBM subtypes. Overall, synergistic response to the combination therapy is more associated with cellular organization, amino acid transmembrane transporter activity, ion channels, extracellular matrix organization and collagen formation. Conclusions: Our findings highlight that specific molecular pathways and their activities, including those involving translation, cell cycle and ion transport, appear to modulate the synergistic efficacy of the ATO and AUM001 combination, thereby offering potential biomarkers for improved patient stratification in future GBM clinical trials of such ATO-based treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed3a011ab153f12fa9045338da4a46a0d67f877a" target='_blank'>
              Arsenic Trioxide and the MNK1 Inhibitor AUM001 Exert Synergistic Anti-Glioblastoma Effects by Modulating Key Translational, Cell Cycle, and Transmembrane Transport Pathways
              </a>
            </td>
          <td>
            Yue Hao, Charles Shaffer, N. Tang, Valerie DeLuca, Angela Baker, Michael E Berens
          </td>
          <td>2026-01-23</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 MicroRNAs (miRNAs) are central regulators of gene expression, and rare pathogenic mutations in miRNA genes can cause Mendelian disease. However, the contribution of germline miRNA variants to prostate cancer (PCa) risk and clinical outcomes remains unclear, in part due to limited coverage of miRNA genes in previous exome, GWAS, and population sequencing datasets.



 We performed ultra-deep, targeted germline sequencing and rigorous variant calling of over 500 miRNA genes (including flanking regions) in 1,502 PCa patients from hereditary and sporadic cohorts, representing both high- and low-grade disease. Variants were systematically annotated for functional region (stem, seed, mature, loop) and predicted impact on pre-miRNA folding (ΔG). Frequencies of recurrent variants (>2 carriers) were compared to gnomAD whole-genome sequencing (WGS) and whole-exome sequencing (WES) populations using Fisher’s exact test within ancestry-matched groups. Results were cross-referenced with miRNASNP-v4, miRNA-seq data from prostate cancer cell lines, and TCGA-PRAD expression/outcome data. Per-gene miRNA burdens and individual variants were assessed for association with clinical features (family history, age, grade, metastasis, progression, PSA, cause of death).



 We generated a comprehensive, high-confidence annotation resource of miRNA gene variants in germline DNA from prostate cancer patients, identifying 48 recurrent miRNA gene variants significantly enriched in PCa cases versus gnomAD (p < 1×10^−8), including both known and novel polymorphisms. A subset displayed significant pre-miRNA folding change (ΔG > 2 or > 5 kcal/mol) and/or mapped to seed or mature regions. Several variants overlapped miRNA genes robustly detected in PCa cell lines and differentially expressed in TCGA-PRAD, with recurrent hits in known risk regions such as 8q24 (notably enriched among African/African American patients).



 This study establishes the deepest and most extensive annotation of miRNA gene variants in prostate cancer to date, integrating germline miRNA variation, functional prediction, and population ancestry. We observed significant discrepancies in miRNA variant allele frequencies when comparing gnomAD WGS to WES data, underscoring the critical need to use WGS datasets for reliable estimation of miRNA gene variant frequencies. Our results identify multiple miRNA gene variants that are enriched in our PCa cohort compared to gnomAD WGS reference populations and are predicted to impact miRNA biogenesis, function, and/or PCa cell biology. While these findings are novel and present strong candidates for further study, they remain preliminary and should be validated in independent cohorts for association with PCa risk and biological effects.



 Shawn E. Lupold, Jun Wei, Siqun Zheng, Jun Luo, William Isaacs, Jianfeng Xu. Comprehensive Evaluation of Recurrent microRNA Gene Variants in Prostate Cancer: Integrative Association with Risk, Predicted Function, and Clinical Outcome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23c6330923e6f2e746abce64dc2fe13161f56d5e" target='_blank'>
              Abstract A041: Comprehensive Evaluation of Recurrent microRNA Gene Variants in Prostate Cancer: Integrative Association with Risk, Predicted Function, and Clinical Outcome
              </a>
            </td>
          <td>
            Shawn E. Lupold, Jun Wei, S. Zheng, Jun Luo, W. Isaacs, Jianfeng Xu
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="CRISPR/Cas systems have transformed molecular medicine, yet the field still lacks principled guidance on when transient editing suffices versus when sustained exposure through in vivo viral delivery is necessary and how to keep prolonged exposure safe. Notably, EDIT-101 was designed for a permanent edit in post-mitotic photoreceptors with lifelong Cas9 persistence. This review addresses this gap by defining the biological and therapeutic conditions that drive benefit from extended Cas activity while minimizing risk. We will (i) examine relationships between expression window and efficacy across Cas9/Cas12/Cas13 modalities, (ii) identify genome-wide off-target liabilities alongside orthogonal assays, and (iii) discuss controllable, self-limiting, and recallable editor platforms. By separating durable edits from persistent nuclease exposure, and by providing validated control levers, this work establishes a generalizable framework for safe, higher-efficacy CRISPR medicines. Furthermore, we highlight key studies in cell lines, murine models, non-human primates, and humans that examine the long-term effects of sustained expression of CRISPR/Cas systems and discuss the safety and efficacy of such approaches. Current evidence demonstrates promising therapeutic outcomes with manageable safety profiles, although there is a need for continued monitoring as CRISPR/Cas therapies are increasingly applied in clinical contexts and therapies are developed for broader clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ffced530e1e86e12ebc967a34622935497f148" target='_blank'>
              Chronic In Vivo CRISPR-Cas Genome Editing: Challenges, Long-Term Safety, and Outlook
              </a>
            </td>
          <td>
            Caroline Bao, Catherine I. Channell, Yi Hsuan Tseng, J. Bailey, Naeem Sbaiti, A. Demirkol, Stephen H. Tsang
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/704dba0681951d72abca65e0d6b244073c98cbe4" target='_blank'>
              Protocol for Non-viral HDR-based CRISPR/Cas9 platform for small custom editing in primary T cells
              </a>
            </td>
          <td>
            Katariina Mamia, Anniken Solveig Matheson Sollano, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2026-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lung cancer persists as the leading cause of cancer-related mortality globally, largely due to late detection, genomic complexity, and limited durability of existing therapeutic interventions. The integration of CRISPR/Cas9 gene-editing systems with biomarker-driven therapeutic strategies represents a transformative advance in precision oncology. Biomarkers—including genetic mutations, epigenetic alterations, protein signatures, and circulating analytes—enable early detection, patient stratification, and dynamic monitoring of therapeutic response. CRISPR/Cas9 offers a unique opportunity to directly reprogram these biomarkers or the pathways regulating them, enhancing tumor immunogenicity, reversing immune evasion mechanisms, and strengthening anti-tumor immune responses. Preclinical models demonstrate that CRISPR-mediated biomarker editing can restore antigen presentation, augment T-cell cytotoxicity, sensitize resistant tumors to immunotherapy, and improve tumor regression. Early clinical trials further validate the feasibility and safety of CRISPR-engineered immune cells in patients. 
However, major challenges persist, including off-target editing, inefficient delivery to solid tumors, tumor microenvironment–mediated suppression, and ethical considerations linked to genome manipulation. Rapid advancements in editing fidelity, lipid nanoparticle systems, viral vectors, engineered vesicles, high-throughput biomarker discovery, and artificial intelligence–assisted CRISPR design are expected to accelerate clinical translation. This review synthesizes the current landscape, mechanistic underpinnings, emerging applications, and future directions of CRISPR/Cas9-enabled biomarker engineering for lung cancer immunotherapy, positioning this technology as a cornerstone of next-generation personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d80a98d71ca868cfed559df1a6401f5e2e8b0702" target='_blank'>
              Precision Reprogramming of Lung Cancer Biomarkers via CRISPR/Cas9: A Paradigm Shift in Personalized Immunotherapy
              </a>
            </td>
          <td>
            M. R. Jameel, Amar Arora, Prem Shankar Mishra, Hina Jameel, M. Makhdoomi, Abid Ali Sheikh
          </td>
          <td>2025-12-30</td>
          <td>Journal of Analytical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Bladder cancer is a common malignancy, and the insensitivity of advanced bladder cancer to cisplatin poses an imminent challenge to treatment. Our study aims to identify novel targets that mediate cisplatin responsiveness in bladder cancer. Accordingly, overexpression of the histone demethylase KDM4A in clinical cohorts was found in association with poor prognosis. Tissue culture and animal tests showed that KDM4A pis ro-proliferative in bladder cancer cells. Using co-immunoprecipitation and mass spectrometry methods, we identified that USP7 is an interacting partners in KDM4A protein complex, in which USP7 catalyzes KDM4A proteins deubiquitination that uncouples the proteasome-dependent degradation. In accordance, a positive correlation between USP7 and KDM4A protein expression was noted in bladder cancer clinical samples. Functional validation tests confirmed that USP7 and KDM4A act complementarily to drive bladder cancer cell proliferation. Importantly, cell and animal assays all evidenced that antagonizing the USP7-KDM4A axis would aggravate cisplatin-induced DNA damage and sensitize cisplatin responsiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09990c377284116741fc0bd433926843134e7edd" target='_blank'>
              USP7 promotes chemotherapy resistance and DNA damage response through stabilizing and deubiquitinating KDM4A in bladder cancer
              </a>
            </td>
          <td>
            Hailang Yang, Xiaoqiang Liu, Jianqiang Nie, Shuwei Wu, Li Ma, Yi Jiang, Lizhi Zhou, Wen Deng, Qianxi Dong, S. Xiong, Sheng Li, Fuchun Zheng, An Xie, Songhui Xu, Bin Fu
          </td>
          <td>2025-12-23</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b89d2858599b4236f22769f3a4af65ec9fd4eb7" target='_blank'>
              Deciphering a common code: Unitary gene profile factor enabling Spontaneous Tumor Regression
              </a>
            </td>
          <td>
            Arushi Misra, B. Kumari, Prasun K. Roy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oncofetal reprogramming has recently emerged as a critical concept in translational cancer research, particularly for its role in driving therapeutic resistance across a variety of malignancies. This biological process refers to a pattern of gene expression that is restricted to embryogenesis, but becomes expressed again in a subpopulation of cancer cells. These genes are typically suppressed after embryogenesis, and their aberrant re-expression in tumors endows cancer cells with stem-like properties and enhanced adaptability. The goal of this review is the following: (i) comprehensively examine the multifaceted nature of oncofetal reprogramming; (ii) elucidate its underlying molecular mechanisms, including its regulators and effectors; and (iii) evaluate its consequences for the therapeutic response in different cancer types. We comprehensively integrate the latest findings from colorectal, breast, lung, liver, and other cancers to provide a detailed understanding of how oncofetal programs interfere with tumor response to treatment. Among the candidates, YAP1 and AP-1 have emerged as central transcriptional drivers of this reprogramming process, especially in colorectal and breast cancers. We also explore the distinct expression patterns of oncofetal genes across different tumor types and how these patterns correlate with treatment outcomes and patient survival. Lastly, we propose a dual-targeting therapeutic strategy that simultaneously targets both cancer stem cells and oncofetal-reprogrammed populations as a more effective approach to overcome resistance and limit recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a262b7681553fb103b5d82c8b95c6fdfada4bf5" target='_blank'>
              Oncofetal Reprogramming: A New Frontier in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Anh L Nguyen, Molly Lausten, Bruce M. Boman
          </td>
          <td>2026-01-29</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b7b9d009213c7ffc5a98ac8b16a699f1f4455" target='_blank'>
              Anchored random reverse primer sequencing for quantitative detection of novel gene fusions.
              </a>
            </td>
          <td>
            Xuehao Xiu, Yi Wu, Jiangxue Li, Dongfa Lin, Xiao Sun, Xinglei Su, Zhi Weng, Xiaolei Zuo, Xiurong Yang, Chunhai Fan, Yudong Wang, David Yu Zhang, Ping Song
          </td>
          <td>2026-01-02</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML) are clonal stem cell disorders in which disrupted post-transcriptional regulation contributes to aberrant hematopoiesis and leukemic transformation. The miRNA biogenesis machinery, which comprises Drosha, DGCR8, Dicer, TARBP2, and AGO1, ensures the precise maturation of miRNAs that control lineage commitment and proliferation. However, the extent to which alterations in this pathway reshape hematopoietic gene networks during myeloid disease evolution remains largely unexplored. Methods: Bone marrow samples from newly diagnosed, untreated MDS and AML patients and matched healthy controls were analyzed for the expression of five key miRNA biogenesis genes using quantitative real-time PCR. Statistical comparisons, correlation matrices, and ROC analyses were performed to characterize gene-expression differences. These results were integrated with multigene logistic modeling, decision-curve analysis, and exploratory random forest/SHAP approaches to evaluate molecular interactions and diagnostic relevance. Results: DROSHA, DICER1, and TARBP2 were significantly downregulated in both MDS and AML, suggesting impaired miRNA maturation and a loss of global post-transcriptional control. DGCR8 expression increased across higher-risk MDS groups, suggesting compensatory activation of the Microprocessor complex, whereas AGO1 levels remained relatively stable, consistent with partial maintenance of RISC function. Correlation analyses revealed a co-regulated DROSHA–TARBP2–AGO1 module. ROC, logistic, and machine learning models identified DGCR8 and DICER1 as the strongest diagnostic discriminators. The integrated five-gene signature achieved high discriminative performance (AUC ≈ 0.98) and showed promise but remains preliminary potential for clinical application. Conclusions: Our findings suggest that defects in miRNA biogenesis disrupt hematopoietic homeostasis, reflecting common mechanisms in MDS and AML. The dysregulation of DICER1, DGCR8, and TARBP2 offers insights into miRNA-driven leukemogenesis and may pave the way for miRNA-based diagnostic and therapeutic strategies, pending validation in larger cohorts. Although transcript-level data are provided, future studies should include functional validation to determine the impact on downstream miRNA processing and hematopoietic pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d47acba4f826476e2de9cf5f5fedcb821bc1d6f0" target='_blank'>
              Disrupted miRNA Biogenesis Machinery Reveals Common Molecular Pathways and Diagnostic Potential in MDS and AML
              </a>
            </td>
          <td>
            Kenan Çevik, M. Ay, Anıl Tombak, Özlem İzci Ay, Ümit Karakaş, Mehmet Emin Erdal
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c6776876b0ba015273baad473a4e391867ab9a" target='_blank'>
              CRISPR screens identify PRMT7 as a therapeutic target to enhance T cell-mediated killing in breast cancer.
              </a>
            </td>
          <td>
            Wei Shi, Yi Luo, Yizhuo Wang, Jacqueline M. Burrows, Debra Black, Andrew Civitarese, Laura Perlaza-Jiménez, Ping Zhang, Murray Manning, Natasha Tuano, Miguel E Rentería, Christos Xiao, Siok-Keen Tey, Joseph Rosenbluh, Corey Smith, G. Chenevix-Trench, Jonathan Beesley
          </td>
          <td>2026-01-21</td>
          <td>NPJ breast cancer</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Hematologic malignancies remain among the most difficult cancers to treat, challenged by profound heterogeneity, treatment-induced immune dysfunction, and the frequent emergence of drug resistance. Beyond tumor-intrinsic mechanisms, dysbiosis of the gut microbiome is increasingly recognized as a critical determinant of therapeutic outcomes, shaping hematopoiesis, immune responses, and drug metabolism. Bacteriophage (phage) therapy has re-emerged as a precision tool capable of selectively eradicating pathogenic taxa while preserving commensal short-chain fatty acid–producing communities. Preclinical and early human studies demonstrate that phages can recalibrate microbial ecosystems, disrupt antibiotic-tolerant biofilms, and enrich metabolites such as butyrate that support mucosal integrity and immune balance. Mechanistically, phage DNA enriched with CpG motifs engages Toll-like receptor 9, activating dendritic cells and enhancing cytotoxic T lymphocyte responses, suggesting dual benefits in infection control and anti-tumor immunity. Emerging applications extend further, with engineered phages serving as vectors for CRISPR–Cas gene editing, targeted cytokine delivery, and nanocarrier platforms for leukemia therapy. Despite translational promise, major hurdles persist, including immunogenicity, horizontal gene transfer, resistance evolution, and regulatory uncertainty. Addressing these challenges through GMP-compliant manufacturing, metagenomics-guided personalization, and AI-optimized cocktail design could establish phage therapy as a microbiome-informed adjunct to overcome drug resistance in blood cancers. However, direct clinical evidence of phage therapy efficacy in hematologic malignancies remains limited, and current data are largely derived from preclinical and compassionate-use contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9ece11723039e03ff5895ac5fce518e0f2ca3a" target='_blank'>
              Phage therapy and the microbiome in hematologic malignancies: opportunities, mechanisms, and early evidence
              </a>
            </td>
          <td>
            Juanwen Zhang, Jin Liu, Alireza Bayani
          </td>
          <td>2025-12-12</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Precancers, defined as normal-appearing or morphologically altered tissues with a risk of oncogenesis, exhibit various detectable manifestations across anatomical sites, including epithelial dysplasia, metaplasia, hyperplasia, and stromal fibrosis. Considering the prevailing assumption that most cancers arise from precancers, early intervention at the precancerous stage has immense potential to reduce cancer-related morbidity and mortality. However, the complex signaling networks governing precancer initiation and progression remain elusive, hampering the development of effective targeted interventions. This review synthesizes three critical dimensions of precancer biology: historical foundations tracing the conceptual evolution of precancer research over the past century; mechanisms underlying the multistep progression of precancer biology, encompassing epithelial and macro/microenvironmental remodeling; and signaling networks cataloging dysregulated pathways and their therapeutic potential. Over 10 signaling pathways, including the transforming growth factor-β (TGF-β), p53, Wnt, phosphatidylinositol 3-kinase (PI3K), and mitogen-activated protein kinase (MAPK) pathways, drive multistep malignant transformation. We further synthesize emerging evidence supporting microenvironmental dominance, proposing the novel “soil degeneration” hypothesis. This paradigm shift underscores the necessity for dual-window intervention in which early-phase microenvironmental normalization prevents the establishment of precancerous lesions and advanced-phase treatment concurrently addresses epithelial malignancy and stromal degeneration. This review bridges foundational molecular discoveries with translational clinical potential and advocates for precision intervention frameworks that extend from biomarker-guided risk assessment to synergistic remodeling of the precancer microenvironment, thereby redefining precancer intervention in the molecularly targeted era.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67dd61437ebeffa1b547c7834fa24572f4d7328c" target='_blank'>
              Signaling pathways and targeted interventions for precancers
              </a>
            </td>
          <td>
            Jin Yang, Shimeng Wang, Xin Li, Hongdan Xu, Tongxu Sun, Tao Hu, Jingjing Luo, Hongmei Zhou
          </td>
          <td>2026-01-07</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="This paper delves into the role of epigenetics in complementing and challenging Darwinian evolution theory within the field of evolution. Epigenetics has revealed new sources of genetic variation, with epigenetic variations such as DNA methylation and histone modifications capable of influencing gene expression without altering DNA sequences. These variations can spread and accumulate within populations, exhibiting reversibility and dynamism, enabling species to rapidly adapt to environmental changes. Additionally, epigenetics expands the scope of natural selection, revises the patterns of genetic information flow, and highlights the need for updated evolutionary predictive models. Researchers have proposed theoretical integration pathways such as the Extended Evolutionary Synthesis framework, multilevel selection theory, and gene-epigenome coevolution models, offering new perspectives and directions for evolutionary biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b730d3e4497777825de2252ebd00338b352de17" target='_blank'>
              Epigenetics Complements and Challenges Darwinian Evolution: From Gene Regulation to the Study of Species Adaptation
              </a>
            </td>
          <td>
            Meichen Zhou
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drosophila melanogaster, a well-established model organism in biomedical research, has significantly advanced our understanding of genetic mechanisms underlying human diseases. Its genetic similarity to humans, with approximately 75% of human disease-related genes having orthologues in the fly genome, renders it a valuable tool for studying complex conditions like cancer. The organism’s low cost, short life cycle, and manipulable genome make it ideal for high-throughput studies. Recent advances have enabled the use of Drosophila as a model for different cancer types, including colorectal, thyroid, lung, and brain cancers. These models replicate key tumor characteristics such as cell proliferation, invasion, and metastasis. Drosophila’s utility in drug screening has been exemplified through heterochromatin-promoting assays and personalized medicine approaches using patient-specific gene expression data. Its role in testing therapeutic combinations, such as MEK inhibitors with statins for lung cancer, underscores its translational potential. Despite its utility, the model has notable limitations. Differences in tumor biology between flies and humans, especially regarding organ systems and immune responses, limit its applicability in certain cancer types and metastatic studies. Furthermore, environmental influences and the simplified genetic background of Drosophila may not accurately represent human cancer complexity. Ethical concerns and translational challenges also exist, warranting cautious interpretation of results. The recent advancements in CRISPR-based genome editing, particularly cytosine and adenine base editors, have improved gene-editing efficiency in Drosophila, supporting its ongoing use in precision oncology. Future directions point toward integrating Drosophila models with other in vivo and in silico systems to optimize personalized drug discovery and better emulate human tumor heterogeneity. While the use of Drosophila melanogaster has intrinsic limitations, its genetic tractability and compatibility with cutting-edge genomic tools position it as a complementary model to mammalian systems. Continued innovations in genetic engineering and model integration will further strengthen its role in translational oncology and precision medicine. This review explores the applications of the model in human cancer research and highlights the major challenges with its use. This is crucial for the fight against cancer, as it helps inform researchers of the potential areas that require further research, especially with the rapid evolution of personalised medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b860b652fc843a37123c4a5f30311be7939a37a1" target='_blank'>
              Applications and challenges of Drosophila melanogaster as a laboratory model in human cancer research: a narrative review
              </a>
            </td>
          <td>
            R. Abdullateef, Jonas Paul Ibekwe, Henry Oyoyo, Samuel Ogbodo
          </td>
          <td>2025-12-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="TP53 mutation-driven gene expression programs define oncogenic phenotypes. While extensive studies have concentrated on the transcriptome and proteome, post-transcriptional processes, particularly translational variation, remain underexplored. This study presents a comprehensive analysis of the transcriptomics, translatiomics, and proteomics dynamics in the ovarian cancer cell line SKOV3, with a focus on the effects of p53 missense mutations (R175H, R273H, and Y220C) on gene dosage fluctuations. Despite clear transcriptional differences between wild-type and mutant p53, we find that extensive translational and post-translational buffering processes attenuate these discrepancies, yielding comparatively stable protein abundances. Moreover, we delineate that the relative contributions of transcription output, translation engagement, and protein stability collectively shape the final protein abundance in the context of p53 mutations. Clinical proteomic analysis of platinum-resistant ovarian cancer tissues reveals tumor-specific factors and acquired resistance pathways linked to p53 mutations. Our findings elucidate the multilayered regulatory landscape of p53 mutations and identify potential risk factors for platinum resistance associated with these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/765c815e9b25f95d1edceba753e2ba1a94641e62" target='_blank'>
              Multilayered Regulatory Dynamics of p53 Mutations and Platinum Resistance in Ovarian Cancer.
              </a>
            </td>
          <td>
            Liling Hu, Hanchen Zou, LvYing Peng, Fan Li, Danya Liu, Jiangli Lu, Yuying Li, Chris Zhiyi Zhang, Qiu-Hong Tian
          </td>
          <td>2025-12-16</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Oncofusions, cancer-causing chimeric proteins formed by chromosomal rearrangements, are key drivers in approximately 16% of all cancer cases. These fusion proteins are associated with a wide range of malignancies, including varieties of leukemia, lung cancer, and sarcoma. Thousands of unique oncofusions have been identified in recent years, yet most of these fusion proteins lack mechanistic annotation. Many oncofusions dysregulate chromatin architecture in complex ways that are not yet fully elucidated. Investigating these chromatin alterations is both labor-intensive and costly, limiting the feasibility of large-scale studies. This lack of mechanistic insight has hindered the development of targeted therapies, with only a handful of efficacious small-molecule inhibitors directed toward oncofusions currently in clinical trials or approved for patient use. To address this critical gap, we adapted our recently developed PROD-ATAC method into a high-throughput, scalable platform to systematically elucidate the effects of dozens of candidate inhibitory small molecules on the chromatin dysregulation profiles induced by a pooled library of diverse oncofusions. In the PROD-ATAC scheme, a pooled protein-coding variant library is expressed in a single disease-agnostic cell line, and scATAC-seq (single-cell assay for transposase accessible chromatin) catalogs each variant’s effect on the chromatin landscape. We applied PROD-ATAC to simultaneously assay the chromatin regulatory impact of >40 small molecule inhibitors against a panel of tens of oncofusions (including the majority of patient-identified fusion sequences causal for Ewing sarcoma) in a single assay, generating genome-wide chromatin accessibility datasets representative of >1,000 unique drug - fusion interactions. In this manner, we characterized several small molecules as previously unidentified inhibitors of multiple oncofusions. Certain inhibitors were found to abrogate the chromatin dysregulation caused by an oncofusion either globally or in a targeted cell signaling pathway-specific manner. By profiling multiple rare fusions causal for the same cancers, we identified separate instances of homogenous and heterogenous modes of chromatin dysregulation associated with distinct fusion proteins. In this work, PROD-ATAC chromatin accessibility readouts coupled the identification of novel oncofusion inhibitors with the characterization of each inhibitor’s impact on the epigenetic landscape of oncofusion-expressing cells.



 James E. Corban, Max Frenkel, Srivatsan Raman. High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff99b547f5b85ca4c657ff5fd20f9c13db3f972" target='_blank'>
              Abstract A024: High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions
              </a>
            </td>
          <td>
            James E. Corban, Max Frenkel, Srivatsan Raman
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Advances in single cell multi-omics technologies have allowed for investigation into genotype-immunophenotype relationships and dynamic clonal changes in cancer patient samples. These technologies provide rich insights into genetic profiling, drivers of disease progression, and subclonal dynamics. Increased adoption and throughput of these technologies, necessitate accessible computational tools for processing and analysis. Toward developing easy to use computational tools, we introduce the Single Cell DNA (scDNA) package that allows for rapid analysis of single-cell molecular profiling. Our platform aims to provide a diagnostic summary tool for sample quality control, representative subclonal architecture within a sample, demultiplexing functionality for multi-sample processing, copy number variation, and mutation trajectory analysis to identify order of subclonal mutation acquisition. We showcase a series of vignettes on hematopoiesis datasets for each aim that reflects recently deployed uses of scDNA. Additionally, we show scDNA provides a modular, user-friendly framework that readily feeds into other standard software pipelines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e9603df019d6ce21c68f981f038fde96385eec2" target='_blank'>
              scDNA: Single Cell DNA analysis software toolkit for subclonality discovery and assessment
              </a>
            </td>
          <td>
            Michael Bowman, Shreeya Gounder, Varsha Singh, Olga Shestova, Troy M Robinson, Amy Zhang, Anushka Gandhi, Roopsha Bandopadhyay, Sheng F Cai, R. L. Levine, Saar I. Gill, Linde A. Miles, Robert L. Bowman
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immunotherapies have transformed cancer treatment by harnessing the immune system to recognize and eliminate malignant cells, offering durable clinical benefit across diverse tumor types. Despite successes with immune checkpoint inhibitors (ICIs) and other strategies like cytokines, oncolytic viruses, cancer vaccines, bispecific antibodies, and adoptive cell therapies, substantial fractions of patients still fail to respond or develop resistance. The oncogene MYC, deregulated in ~70% of human cancers, has emerged as a central driver of immune evasion and a key contributor to immunotherapy failure. MYC regulates broad transcriptional networks controlling proliferation, metabolism, angiogenesis, and cell survival, while also orchestrating profound remodeling of the tumor microenvironment (TME). Mechanistically, oncogenic MYC suppresses antigen processing and presentation, attenuates interferon signaling, and upregulates immune checkpoints such as PD-L1 and CD47. Concurrently, MYC stimulates secretion of immunosuppressive cytokines and chemokines that recruit regulatory T cells, myeloid-derived suppressor cells, and M2 macrophages, while driving metabolic reprogramming that fosters nutrient competition, hypoxia, and acidosis, impairing effector T- and NK-cell function. Through these pathways, MYC promotes primary, adaptive, and acquired resistance to immunotherapy. Targeting MYC, directly or indirectly, holds promise to restore immune surveillance and potentiate immunotherapeutic efficacy. This review highlights MYC as a master regulator of tumor–immune interactions and underscores the therapeutic potential of MYC inhibition to overcome resistance and expand the clinical impact of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437ab7ab32dc49430fcd2b151a4005d089060a97" target='_blank'>
              MYC at the tumor–immune interface: mechanisms of immune escape and immunotherapy resistance
              </a>
            </td>
          <td>
            Íñigo González-Larreategui, Manrique Valdés-Bango Martín, S. Casacuberta-Serra, Laura Soucek
          </td>
          <td>2026-01-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The advent of next-generation high-throughput sequencing and precision medicine has allowed the identification of recurrent genetic events of vital interest for leukemic cells and for the management of patients with leukemia, since it has a direct impact on prognosis and therapeutic stratification. The precise cytogenetic characterization of these diseases has rapidly enabled the development of innovative therapeutic strategies to improve outcomes and limit toxicities. New targeted therapies, including small molecule inhibitors, monoclonal antibodies, bispecific T cell engagers, antibody–drug conjugates, recombinant immunotoxins, and chimeric antigen receptor T cells (CAR-T cells) represent a broad therapeutic arsenal now available in everyday practice. The growing number of treatments means that many possibilities of drugs and combinations can be used to offer new lines of therapy in cases of refractory disease or relapse, but also in frontline treatment. This review provides a broad overview of pathogenic mechanisms and therapies (including targeted therapies, immunotherapy and RNA-based therapy) and gathers the main research progress in the field of leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9ad84d56b4eca59a338789eb4fc5c1bd3ecb571" target='_blank'>
              Molecularly targeted therapy and immunotherapy in leukemias
              </a>
            </td>
          <td>
            Elie Cousin, Agathe Picard, Maria Oundjian, Nina Tardif, Anaïs M. Quéméner, Anne-Gaëlle Rio, M. Galibert, Virginie Gandemer, Frédéric Mazurier
          </td>
          <td>2026-01-11</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Transposons (TEs) are genetic elements that can change their positions within genome. They cause transcriptional activation or repression by inserting into regulatory elements of related genes or by modulating epigenetic modifications on regulatory elements. In addition, some transposons encode and express peptides or proteins that affect or disrupt the cellular biological functions. Recent studies have demonstrated that while host cells effectively control most transposons, certain harmful insertions can disrupt normal gene expression processes, potentially leading to diseases. In this mini-review, we discuss recent advances in understanding how transposons contribute to genomic instability. We hope to provide researchers and clinical practitioners with new insights into transposon biology and its potential implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e746fb5f9c54cbee0068471cbc97060459d5187a" target='_blank'>
              Transposons disrupt genomic stability and trigger cancers
              </a>
            </td>
          <td>
            Weixia Dong, Menghui Li, Ping Li, Xiaoli Hou, Fei He, Shaoping Ji
          </td>
          <td>2025-12-12</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are tiny, non-coding RNA molecules that are pivotal in regulating cell growth, development, cell cycle, and programmed cell death. Alterations in miRNA expression, caused by various mechanisms such as amplification, deletion, mutation, or epigenetic changes, can lead to either an increase or decrease in their levels in cancerous cells. Since their initial discovery in 1993, a wealth of research has established miRNAs as key players in oncogenesis and tumor suppression across various human cancers. The study of miRNAs is critical for predicting cancer outcomes and aiding in its detection. Numerous miRNAs have demonstrated potential as markers for cancer prognosis and diagnosis, as well as targets for therapy. Their profound influence on tumor development and spread is becoming increasingly acknowledged. This article presents a summary of how miRNAs are synthesized and generated, and explores their dual function as tumor suppressors and oncogenes. Additionally, we delve into the promising applications of miRNAs as instruments for cancer diagnosis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874dc41eef7a91dbf277f54075fa7796a49bfc9e" target='_blank'>
              MicroRNA Regulation in Cancer: Insights into Diagnosis, Prognosis, and Treatment
              </a>
            </td>
          <td>
            Ibtihaj Alrehaili, Norah Alrashoud, Rawaeb Almutairi, Lama Alqahtani, Nour Alhatm, A. Almutairi, Asma Alghamdi, Azhar Barnawi, Kharsan Alyami, H. Alghamdi, Jamilah Algasem, Reham Alshahrani, Lubna Aljaddua, M. Alahmari
          </td>
          <td>2026-02-04</td>
          <td>Pharmacy Practice</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d21384971b0ced62ebc3c05bc7a513b76dd2fb8" target='_blank'>
              Five dominant amino acid substitution signatures shape tumour immunity.
              </a>
            </td>
          <td>
            Szilvia Juhász, B. Papp, Anna Tácia Fülöp, Z. Farkas, Dávid Kókai, Dóra Alexandra Gyémánt, Franciska Tóth, Zsófia Nacsa, Dóra Spekhardt, Balázs Koncz, Péter Burkovics, Csaba Pál, Máté Manczinger
          </td>
          <td>2026-01-28</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract DNA enzymes (DNAzymes) offer an attractive therapeutic approach for targeting disease-associated mutations in mRNA transcripts, but face limitations in development due to unintended engagement by RNase H1. Although chemical optimization has led to designs with improved catalytic activity, strategies to mitigate RNase H1 recognition remain underexplored. Here, we report the incorporation of threose nucleic acid (TNA) into the backbone architecture of the 10-23 DNAzyme variant known as Dz46. Substitution of the dC3 position in the catalytic loop with TNA increases activity, whereas installation of two TNA residues in the binding arm abrogates competition by RNase H1. The resulting enzyme enables allele-specific knockdown of an oncogenic KRAS mutation in mammalian cells and facilitates general knockdown of PCSK9 and GATA3 targets. Together, these results demonstrate the utility of TNA as a chemical tool for enhancing DNAzyme performance and evading RNase H1 activity in cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e469750e80515262619643ab4087e9b5fc3043" target='_blank'>
              Allele-specific knockdown by an engineered DNAzyme capable of RNase H1 evasion
              </a>
            </td>
          <td>
            Erica M. Lee, Kim Nguyen, Noah A Setterholm, Turnee N. Malik, John C. Chaput
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Most intracellular proteins undergo degradation by proteasomes, which exist in constitutive, immune, and intermediate forms. The diversity arises from the presence of different catalytic β subunits incorporated into the 20S core particle. Constitutive proteasomes contain β1, β2 and β5 subunits, whereas immunoproteasomes integrate β1i, β2i and β5i proteins. Intermediate proteasomes comprise combinations of constitutive and immune subunits, but typically lack the β2i subunit. Distinct proteasome configurations have widespread impact on cellular metabolism, gene expression, immune signaling, stress adaptation and tumorigenesis. However, the biological functions of proteasome subtypes remain incompletely defined, underscoring the need for refined experimental systems to uncover their specific activities. Methods Previously, we generated cancer cell lines expressing mCherry-tagged β5i subunit under the control of endogenous regulatory elements. Using CRISPR/Cas9 nickase technology, we further modified the genomes of the colon adenocarcinoma SW620B8-mCherry and cervical adenocarcinoma TZM-blB8-mCherry cells. In SW620B8-mCherryB5-GFP cells, the eGFP gene was inserted at the 3’ end of the PSMB5 gene, which encodes the β5 subunit. Similarly, the β2i-encoding gene (PSMB10) was fused with the photo-switchable cyan fluorescent protein gene (PS-CFP2) in TZM-blB8-mCherryB10-CFP cells. Results Efficient integration of fluorescent protein-encoding sequences into target genomes provided robust expression and integration of chimeric subunits into proteasomes. In TZM-blB8-mCherryB10-CFP cells, photo-conversion of PS-CFP2 allowed visualization of both intermediate and immunoproteasomes within the cellular nuclei following treatment with IFN-γ/TNF. In SW620B8-mCherryB5-GFP cells, analysis of intracellular proteasome localization revealed discrete regions enriched in β5i-containing proteasomes. Moreover, IFN-γ/TNF exposure induced a fluorescence shift from green to red, accompanied by redistribution of intracellular proteasomes in SW620B8-mCherryB5-GFP cells. Multiphoton microscopy of tumors grafted into immunocompromised mice showed the formation of β5i-containing, proteasome-enriched inclusions and suggested their potential release from cancer cells. Collectively, the TZM-blB8-mCherryB10-CFP cell line provides a tool to interrogate immunoproteasome functions independently of intermediate proteasomes, whereas SW620B8-mCherryB5-GFP cells enable visual discrimination between constitutive and β5i-containing proteasomes both in vitro and in vivo. Conclusions Generated cell lines provide a novel platform for dissecting proteasome functions, addressing the distinct properties of individual proteasome forms, and assessing the impact of diverse compounds on the proteasome repertoire in cultured cells and in tumor xenografts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f185dc9839ab4abec820b173f8f7dd6f377d1a" target='_blank'>
              Next-generation cell lines for profiling different proteasome forms and their implications in cancer
              </a>
            </td>
          <td>
            A. Burov, Anastasia Yakovleva, Georgy Linovskiy, Andrei Fedotov, V. Popenko, Olga Leonova, Aleksandr Lanin, Marina Drutskaya, Vadim Karpov, Alexey Morozov
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer progression involves dynamic crosstalk between tumor-intrinsic pathways and microenvironmental remodeling, and identifying pan-cancer biomarkers is critical for precision oncology. CDKN2AIPNL exhibits a paradoxical role in cancer, acting as a tumor suppressor in myeloid malignancies but promoting solid tumor progression, yet its systematic pan-cancer characteristics remain unelucidated. This study aimed to comprehensively analyze CDKN2AIPNL’s expression patterns, prognostic value, genetic alterations, and molecular mechanisms across multiple tumor types using public datasets including TCGA, GTEx, HPA, and tools such as GEPIA2, cBioPortal, TIMER2, STRING, and BioGRID. We performed expression difference analysis, survival analysis (overall survival, disease-free survival, progression-free survival), genetic alteration analysis, cancer-associated fibroblast (CAF) infiltration analysis, and gene/protein interaction enrichment analysis. Results showed that CDKN2AIPNL was significantly upregulated in multiple tumors (e.g., LIHC, UVM, BRCA, LUAD) and downregulated in others (e.g., KICH, KIRP, THCA), with high tumor specificity. Elevated CDKN2AIPNL expression correlated with poor overall survival in LIHC (HR = 1.7, p = 0.0026), UVM (HR = 26, p = 2.2e-6), BRCA (HR = 26, p = 2.2e-6), LUAD (HR = 1.36, p = 0.049), PCPG (HR = 1.71, p = 0.0012), and TGCT (HR = 0.37, p=0.023), and was associated with advanced tumor stages in metabolically active cancers. Genetic alterations (amplifications and mutations) were frequent in KIRC (>5%) and ACC (>4%), with all mutations localized to the XTBD region, and amplification predicted poor prognosis in PRAD (p = 0.008) while mutations conferred favorable outcomes in BLCA. CDKN2AIPNL expression positively correlated with CAF infiltration in ESCA, KICH, UVM, and other tumors, and interacted with MYC, XRN2, and CHAMP1 to regulate metabolic reprogramming, cell cycle, and immune suppression. Our findings systematically reveal CDKN2AIPNL’s dual role in tumorigenesis and validate it as a potential pan-cancer prognostic biomarker, providing novel insights for cancer diagnosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1804517cd011102843d1d1b41c7599a28bf25c60" target='_blank'>
              CDKN2AIPNL: a potential pan-cancer biomarker
              </a>
            </td>
          <td>
            Yulin Yuan, Shengjie Ma, Heshi Liu, Yang Gong, Xuan Sun, Quan Wang, Weifu Zhang
          </td>
          <td>2026-01-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Congenital Anomalies of the Kidney and Urinary Tract (CAKUT) are among the most common congenital malformations and the leading cause of chronic kidney disease in children. They arise when key steps in kidney development are disrupted, including ureteric bud induction, branching morphogenesis and nephron progenitor differentiation. These processes depend on coordinated transcriptional programs, signaling pathways, ciliary function and proper extracellular matrix (ECM) organization. Advances in whole exome and whole genome sequencing, as well as copy number variation analysis, have expanded the spectrum of known monogenic causes. Pathogenic variants have now been identified in major transcriptional regulators and multiple ciliopathy-related genes. Evidence also points to defects in central signaling pathways and changes in ECM composition as contributors to CAKUT pathogenesis. Clinical presentations vary widely, shaped by modifying effects of genetic background, epigenetic regulation and environmental influences such as maternal diabetes and fetal hypoxia. Emerging tools, including human kidney organoids, gene-editing approaches and single-cell or spatial transcriptomics, allow detailed exploration of developmental mechanisms and validation of candidate pathways. Overall, CAKUT reflects a multifactorial condition shaped by interacting genetic, epigenetic and environmental determinants. Integrating genomic data with experimental models is essential for improving diagnosis, deepening biological insight and supporting the development of targeted therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49984d1e5fe55c45e295c376d5264763f38c60ef" target='_blank'>
              From Genes to Malformations: Molecular Mechanisms Driving the Pathogenesis of Congenital Anomalies of the Kidney and Urinary Tract
              </a>
            </td>
          <td>
            Maria Fourikou, J. Dotis
          </td>
          <td>2025-12-19</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background: A significant challenge in oncology is the development of therapies that selectively eliminate cancer cells while engaging the immune system for durable protection. Oncogene-driven transformation has been shown to rewire cellular metabolism and organelle function, creating unique dependencies. Lysosomes, traditionally considered mere degradative centers, are emerging as critical regulators of cell survival and death. We hypothesized that oncogenic transformation, particularly by drivers like Her2, creates a lysosomal "Achilles' heel," priming these organelles for permeabilization and offering a novel therapeutic target. Methods: Using the MCF-10A-neoT model of oncogene-driven transformation, we first characterized lysosomal alterations. We then conducted a high-throughput screen for compounds inducing Lysosomal Membrane Permeabilization (LMP), focusing on Reactive Oxygen Species (ROS) inducers. Lead compounds were evaluated for their ability to trigger established markers of Immunogenic Cell Death (ICD), including surface calreticulin (CRT), ATP release, and HMGB1 release. Selectivity and efficacy were further validated in co-culture models and immunocompetent mouse models. Results: Oncogene-transformed cells exhibited significantly enlarged lysosomes, altered subcellular distribution, and increased cathepsin B/L activity compared to their non-transformed counterparts. Our screen identified several ROS-generating compounds that selectively induced LMP in cancer cells. Treatment with these lead compounds resulted in the robust exposure of CRT, and the extracellular release of ATP and HMGB1, confirming the induction of ICD. In co-culture, the compounds demonstrated a clear therapeutic window, preferentially killing cancer cells. In vivo studies in immunocompetent mice confirmed tumor regression and the establishment of a protective anti-tumor immune memory. Conclusion: Our findings validate that targeted LMP induction is a viable strategy for selectively killing oncogene-addicted cancer cells. Moreover, by triggering ICD, this approach not only directly eliminates tumor cells but also stimulates a systemic immune response, addressing the critical need for therapies that provide long-term anti-cancer immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cde312e8a563ef0269b386c82e442a967bac8f91" target='_blank'>
              Exploiting Lysosomal Vulnerabilities: A Novel Therapeutic Strategy to Induce Immunogenic Cell Death in Oncogene-Addicted Cancers
              </a>
            </td>
          <td>
            Hafiz Muhammadismail, Azher Syed
          </td>
          <td>2025-12-30</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Genomic imprinting is an epigenetic occurrence that results in the expression of alleles specific to the parent of origin, plays pivotal roles in plant development, stress adaptation, and agronomic trait regulation. While imprinting has been intensively investigated in model plants (e.g. Arabidopsis, maize, and rice), its dynamic regulatory mechanisms and evolutionary implications remain enigmatic. Recent advances in bioinformatics—including single-cell omics, machine learning, and deep learning—have revolutionized the identification, functional annotation, and network modeling of imprinted genes. This review not only provides a detailed summary of the identification, functions and regulatory mechanisms of plant imprinted genes, but also systematically summarizes methodologies for studying plant genomic imprinting, highlights challenges in multi-omics data integration, and envisions artificial intelligence–driven strategies for epigenetic breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11ce49ffeb91e29e8463f4f11f40151db2d73d90" target='_blank'>
              Bioinformatics insights into plant genomic imprinting: approaches, challenges, and future perspectives
              </a>
            </td>
          <td>
            Xiaotong Jing, Xi Su, Quan Zou, Mengting Niu
          </td>
          <td>2026-01-09</td>
          <td>Briefings in Functional Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Whole-genome sequencing (WGS) has transformed microbial genomics since the first bacterial genome was published in 1995. Advances in sequencing technology, together with decreasing costs, now enable high-resolution investigation of bacterial pathogens for epidemiological surveillance and infection control. A major breakthrough has been the advent of third-generation long-read sequencing (LRS) platforms, such as Pacific Biosciences and Oxford Nanopore Technologies, which overcome the limitations of short-read sequencing (SRS) by producing long continuous reads. LRS facilitates accurate de novo genome assembly, resolution of repetitive and structurally complex regions, and precise characterization of plasmids and other mobile genetic elements (MGEs) that frequently harbor antimicrobial resistance genes (ARGs). A particular strength of LRS lies in its ability to reveal the complete genomic architecture of ARGs, including their localization, copy number, and surrounding genetic environment. Such contextual information is essential, since e.g. the interpretation of antimicrobial resistance (AMR) depends not only on the presence of specific genes but also on their structural organization, mobility potential, and genomic integration. By contrast, LRS provides a reliable foundation for understanding AMR evolution and dissemination through both clonal expansion and horizontal gene transfer. Recent developments in bioinformatics, including dedicated tools for plasmid reconstruction, typing, and annotation, further enhance the analytical value of LRS and hybrid approaches. Beyond isolate-level analyses, LRS enables plasmid surveillance and the tracing of ARG transmission across strains, hosts, and healthcare settings. This review sets out to give readers a brief overview of LRS technology and its capabilities and outlines current approaches and tools to analyze bacterial plasmids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4146e810eb38fefea1bf36ca663b1f0430e8995f" target='_blank'>
              Long-read sequencing for bacterial plasmid analysis: a brief overview.
              </a>
            </td>
          <td>
            V. van Almsick, A. Sobkowiak, V. Schwierzeck
          </td>
          <td>2026-01-30</td>
          <td>FEMS microbiology letters</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer reversion therapy represents a paradigm shift in oncology, focusing on reprogramming malignant cells to a non-malignant state rather than destroying them. This narrative review synthesizes current evidence, emerging technologies, and future directions in this promising field. Cancer reversion is founded on key biological observations: somatic cell reprogramming, spontaneous cancer regression, and microenvironmental influences on malignant behavior. Current approaches include epigenetic reprogramming using HDAC inhibitors and DNA methyltransferase inhibitors; microenvironmental modulation through extracellular matrix manipulation and vascular normalization; differentiation therapy exemplified by all-trans retinoic acid in acute promyelocytic leukemia; and targeting oncogene addiction as demonstrated in BCR-ABL-driven leukemias. Emerging technologies accelerating progress include single-cell analyses that reveal cancer heterogeneity and cellular state transitions; CRISPR-based approaches enabling precise genetic and epigenetic manipulation; patient-derived organoids that model tumor complexity; and artificial intelligence applications that identify novel reversion-inducing agents. Critical evaluation reveals that many reported “reversion” phenomena represent stimulus-dependent plasticity or transient growth arrest rather than stable phenotypic normalization. True cancer reversion requires durable, heritable phenotypic changes that persist after treatment withdrawal, with evidence of epigenetic consolidation and functional restoration. Despite promising advances, significant challenges remain: cancer cell plasticity facilitating therapeutic escape, difficulties in establishing stable reversion states, delivery challenges for solid tumors, and the need for combination approaches to address tumor heterogeneity. Future directions include integrated multi-omics analyses to comprehensively map cellular state transitions, studies of natural regression phenomena to identify reversion mechanisms, advanced nanodelivery systems for targeted therapy, and synthetic biology approaches creating intelligent therapeutic systems. By redirecting rather than destroying cancer cells, reversion therapy offers the potential for reduced toxicity and resistance, potentially transforming cancer from a deadly disease to a manageable condition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68ef16378f672c1c6324bdae132a2624ad963a92" target='_blank'>
              Cancer Reversion Therapy: Prospects, Progress and Future Directions
              </a>
            </td>
          <td>
            Emmanuel O. Oisakede, D. Olawade, Oluwakemi Jumoke Bello, Claret Chinenyenwa Analikwu, Eghosasere Egbon, Oluwaseun Fapohunda, S. Boussios
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>1</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Background Chordoma is a rare bone tumor that is considered to have originated from the remnants of notochord by the general consensus in the field. Chordoma patients suffer from local invasion and recurrence of the disease, along with high resistance to conventional therapies. Despite the scarcity of the disease, new biomarkers must be discovered to develop targeted therapeutics. Methods This study aimed at analyzing the whole exome sequencing data of a chordoma cell line, CH22, which held more than 3,000 missense variants in the genome. These missense-carrying genes were investigated in terms of their association with chordoma pathogenesis. Results Out of them, neurotrophic tyrosine kinase receptor type 3 (NTRK3) and nuclear factor erythroid 2-related factor 2 were found to be associated with the altered pathways involved in chordoma. Conclusion Further examination through structural modeling, molecular docking, and phylogenetic analyses related to the missense variants of NTRK3 was conducted in the scope of this study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b2fe9308814058a063733e1cf9abe4023d9644d" target='_blank'>
              Deciphering the Structural and Molecular Effects of a Novel Genetic Mutation by In Silico Analysis in Chordoma Cell Line
              </a>
            </td>
          <td>
            Muhammed Ümeyr Fettah, Nur Ekimci-Gürcan, Emre Yorulmaz, Batuhan Oğuz, A. Köseoğlu, Furkan Ayaz, Esra Aydemir
          </td>
          <td>2025-09-30</td>
          <td>Journal of Neurological Surgery Part B: Skull Base</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Growing knowledge around disease states has led to opportunities within research to make designer molecules with improved specificity and broader efficacy. These next-generation molecules frequently take advantage of multispecific targeting and controlled mechanisms of action by utilizing four unique peptide chains as seen in many bispecific or trispecific antibodies. However, with all the opportunities these multispecifics offer, their increased biological complexities come with increased challenges during expression and purification to produce high-quality material. Lower yields accompanied with a high degree of mispairing after the initial capture purification step are often limiting factors. Developing new methods for stable pool expression can offer a strong advantage for progressing these molecules through research toward development. Here, we implemented optimized stable cell pools using targeted dual selection (TDS), a novel approach that combines specified selective pressure with transposon-guided semi-targeted gene integration. By utilizing key analytical data obtained during early-stage high-throughput transient productions, we can predict improved vector configurations for the generation of optimized TDS stable pools. We demonstrate that this design can improve molecule quality at the initial capture purification step in two Y-shaped bispecific molecules and two cross-over dual variable trispecific molecules by achieving up to four-fold increase in protein of interest yields while maintaining product quality. Use of this strategy in research can enable simplified purification strategies as well as increased production yields required for successful and timely project progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4ead5e389c831e279ea452ea421a912ad9538e" target='_blank'>
              Targeted dual selection to optimize transposon stable pool generation of multispecifics
              </a>
            </td>
          <td>
            Julie Johnston, Sonja Tierson, Yuyan Xu, Kalie Mix, Yj Jane Guo, Serhan Zenger, David Reczek, Dietmar Hoffmann, Brian Hall, Virginia Brophy
          </td>
          <td>2025-12-14</td>
          <td>mAbs</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The Japanese medaka, Oryzias latipes, has become an important vertebrate model organism for addressing research questions across a broad range of disciplines, including developmental and evolutionary biology, stem cells, gene–environment interactions, behavioral neuroscience, disease modeling, and drug discovery. The medaka community took advantage of the successful completion of the ERC Synergy Grant project IndiGene to gather once again in the beautiful Heidelberg (July 22–25, 2025). Building on the opportunities created by the IndiGene project, which leverages the medaka inbreed panel Medaka Inbred Kiyosu–Karlsruhe as a unique resource for dissecting complex phenotype–genotype relationships, the meeting offered an outstanding update on emerging concepts, technologies, and community resources. By summarizing the content of each session, this report provides an overview of a vibrant and highly productive event that highlights the continued growth and vitality of medaka research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03d233ac5889c75a856a0044d1f780508929da83" target='_blank'>
              Insights and emerging perspectives from the 2025 Medaka PI/ERC IndiGene meeting: Heidelberg July 22–24, 2025
              </a>
            </td>
          <td>
            R. D. Acemel, Julien Bobe, Lázaro Centanin, Cielo Centola, Sapna Chhabra, F. Loosli, R. Bhandari, Javier Vázquez-Marín, L. Zilova, E. Birney, Jochen Wittbrodt, J. Martínez-Morales
          </td>
          <td>2026-01-21</td>
          <td>Developmental Dynamics</td>
          <td>0</td>
          <td>130</td>
        </tr>

        <tr id="


 Prostate adenocarcinomas are the most common form of the disease. Targeting the androgen receptor (AR) is the principal treatment strategy. However, resistance is common. Lineage plasticity (LP)—change in differentiation state—is increasingly recognized as a mechanism of resistance to AR-targeting therapies. LP is a continuum ranging from amphicrine to neuroendocrine prostate cancer (NEPC). Despite the increasing incidence of LP due to more widespread use of new and potent AR inhibitors, mechanisms by which LP emerges after therapy have been understudied due to paucity of matched patient tumors. To address this deficit, we performed multi-omic profiling on matched biopsies from 16 patients, including seven patients that underwent LP after treatment.



 We used DNA sequencing to identify mutations and copy number alterations present in tumors at baseline and at progression. We also performed RNA sequencing and transcriptional subtyping of all tumors.



 Analysis of RNA sequencing revealed two distinct subtypes of LP in our cohort—most consistent with amphicrine or NEPC. DNA sequencing identified TP53 and RB1 alteration patterns in baseline tumors that eventually underwent specific LP trajectories. Pathways linked to proliferation and inflammatory signaling were highly activated in baseline tumors that eventually underwent LP.



 Our analysis identified distinct LP trajectories linked with specific genomic and transcriptomic features at baseline. Profiling tumors with this approach may provide clues about which patients may be at greatest risk for undergoing specific LP trajectories after therapy.



 Anbarasu Kumaraswamy, Visweswaran Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Zachery R. Reichert, Arul M. Chinnaiyan, Joshi J. Alumkal. Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d201d414071181c06c4abf5de951c2f23846da37" target='_blank'>
              Abstract A036: Multi-Omic Profiling Clarifies Mechanisms of Therapy-Induced Prostate Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            A. Kumaraswamy, V. Ravikumar, Ryan J. Rebernick, Eva Rodansky, Amina Tanweer, Joel A. Yates, Aaron M. Udager, Marcin P. Ceislik, Arvind Rao, Z. Reichert, A. Chinnaiyan, J. Alumkal
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>171</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is a heterogeneous malignancy characterized by various genomic alterations playing a crucial role in disease classification, prognosis, and response to treatment. However, molecular diagnosis and effective management of this hematological malignancy remain a major challenge, particularly in developing countries, including Tunisia. In this study, we aimed to conduct a detailed analysis of copy number alterations (CNAs) associated with ALL in a cohort of 60 primary samples from Tunisian patients. Using multiplex ligation-dependent probe amplification (MLPA), major genetic lesions, including IKZF1, CDKN2A/2B, PAX5, ETV6, BTG1, and genes located in the PAR1 region, were analyzed and their associations with clinical and laboratory features, as well as survival outcomes, were also evaluated. Our analysis revealed that 70% of patients had deletions and/or amplifications in at least one gene. The most frequently observed deletions were in CDKN2A/2B (33.3%, n = 20), IKZF1 (30%, n = 18), and PAX5 genes (25%, n = 15). BTG1 deletions were significantly associated with female gender, IKZF1 deletions were more frequent in adult patients, in those with elevated white blood cell (WBC) counts, and in cases involving the BCR::ABL1 translocation, while duplications of the PAR1 region were significantly associated with hyperdiploïdy. Regarding treatment response, cases of IKZF1 deletions showed a significant association with poor glucocorticoid response (GC) at day 8 of treatment and positive minimal residual disease (MRD) rates at days 33 and 63, particularly in B-ALL cases. Furthermore, patients with IKZF1 deletions were associated with significantly lower survival rates in both univariate and multivariate analyses compared to those without these deletions. Additionally, the integration of IKZF1 deletion status into risk stratification models revealed markedly different survival outcomes, highlighting its potential interest in developing new stratification algorithms. These results underscore the critical importance of molecular profiling, particularly IKZF1 status, for improving outcomes in ALL patients in Tunisia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd7521499fd8c33c0af8b7b43eac8773db943a64" target='_blank'>
              Genomic characterization and prognostic significance of copy number alterations in Tunisian patients with acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Ameni Bedoui, W. Ayadi, Nour Louati, Imen Frikha, Yosra Fakhfakh, Fahmi Smaoui, Ali Gargouri, Ikram Ben Amor, Moez Elloumi, Moez Mdhaffar, Raja Mokdad Gargouri
          </td>
          <td>2026-02-03</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T cell therapies have demonstrated remarkable efficacy in several hematological malignancies, yet their success has not been fully replicated in solid tumors. Moreover, even in hematological cancers, relapse after CAR T cell infusion continues to compromise long-term outcomes. These challenges highlight the urgent need to develop strategies that enhance CAR T cell efficacy, persistence, overcoming tumor and microenvironment-mediated resistance. Clustered regularly interspaced short palindromic repeats (CRISPR)-Cas9-based screening platforms provide a powerful approach to systematically identify genes that regulate CAR T cell function. By linking genetic perturbations to phenotypic outcomes, these assays enable the discovery of pathways controlling activation, proliferation, memory formation, and cytotoxicity. Standard workflows involve transduction of substantial numbers of cells with a single guide RNA (sgRNA) library, Cas9-mediated editing, selection of edited cells, and PCR amplification of sgRNA cassettes from genomic DNA (gDNA) prior to sequencing. However, PCR amplification using large amounts of gDNA poses significant challenges and often fails to selectively amplify and retrieve sgRNAs. Here, we describe an optimized CRISPR-Cas9 knockout screening protocol, which we have tested on primary human CAR T cells. The method here incorporates an intermediate step during sgRNA library preparation that reduces gDNA carryover through enzymatic digestion and selective pulldown of the sgRNA cassette, thereby increasing the efficiency of the first PCR amplification. This modification allowed us to retrieve sgRNA information across our CAR T cell screens, which had remained elusive in our previous attempts using traditional 1 and 2-step PCR amplification protocols. In conclusion, this optimized workflow facilitates CRISPR screening library preparation in challenging samples and enables the identification of key genetic determinants that can be targeted to improve therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecca94f275145ee359153517eba4093f85ededd3" target='_blank'>
              Improving CRISPR-Cas9 Screens in CAR T Cells: A Refined Method for Library Preparation.
              </a>
            </td>
          <td>
            Maider Garnica, Patxi San Martín-Úriz, Paula Rodriguez-Marquez, M. E. Calleja-Cervantes, Saray Rodríguez-Díaz, Rebeca Martinez-Turrillas, Mikel Hernaez, Felipe Prósper, J. R. Rodríguez-Madoz
          </td>
          <td>2026-01-02</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Pediatric acute leukemia is the most common childhood cancer, and acute myeloid leukemia (AML) represents the second most frequent subtype, with overall survival rates of approximately 70% and relapse rates near 30%. The purpose of this study is to define molecular mechanisms essential for leukemogenesis and uncover potential vulnerabilities in a high-risk subtype of pediatric AML driven by the fusion oncoprotein PICALM::MLLT10. Our laboratory recently performed an integrated transcriptomic and genomic analysis of 887 pediatric AML cases, identifying 23 distinct molecular categories that differ in HOXA/HOXB expression, co-occurring mutations, and clinical outcomes. Among these, PICALM::MLLT10 AML is notable for its HOXA expression pattern similar to KMT2A-rearranged cases, yet it forms a distinct cluster with other immature AMLs on a UMAP projection, suggesting a unique transcriptional program. Leveraging genome-wide CRISPR screening data from the Cancer Dependency Map (DepMap) across 64 AML cell lines, we identified the RNA helicase DDX19A, but not its paralog DDX19B, as a selective genetic dependency in PICALM::MLLT10 models (U937, P31/FUJ) with an average Chronos score of -1.753 compared to -0.278 in all other AML models. Using a dTAG-induced degradation system, acute DDX19A depletion significantly impaired proliferation in PICALM::MLLT10-expressing cells and led to increased apoptosis as measured by Annexin V staining, and overexpression of DDX19A rescued the proliferation defect induced by dTAG13-mediated degradation. Domain-specific mutagenesis revealed that the DDX19A helicase and RNA-binding domains are essential for proliferation, while the NUP214-binding domains are dispensable in the PICALM::MLLT10 context. Next, we sought to define the transcriptional consequences of DDX19A loss in PICALM::MLLT10 cells. RNA-seq was performed on U937 cells expressing a FKBP12F36V-DDX19A allele after treatment with either DMSO or dTAG-13 for 24 hours, and pathway enrichment analysis revealed upregulation of cytoplasmic translation in DDX19A depleted cells. To functionally assess this effect, we measured nascent protein synthesis using O-propargyl-puromycin (OPP) incorporation after 24 hours of either DMSO or dTAG-13 treatment and observed a global reduction of protein synthesis in DDX19A depleted cells. These data establish DDX19A as a selective vulnerability and uncover the role of DDX19A in regulating protein synthesis in PICALM::MLLT10 driven pediatric AML. This work highlights a novel mechanistic vulnerability that may be leveraged for the development of targeted therapeutic strategies for this high-risk pediatric AML subtype.



 Lauren J. Ezzell, Ryan Hiltenbrand, Elizabeth Arnold, Michael P. Walsh, Melvin E. Thomas III, Tamara Westover, Masayuki Umeda, Jing Ma, Shondra M. Pruett-Miller, Jeffery M. Klco. Uncovering novel molecular vulnerabilities in PICALM::MLLT10 driven pediatric acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT005.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f08fe0a493aa37ad7908724e233549343842d3e" target='_blank'>
              Abstract LT005: Uncovering novel molecular vulnerabilities in PICALM::MLLT10 driven pediatric acute myeloid leukemia
              </a>
            </td>
          <td>
            Lauren Ezzell, R. Hiltenbrand, Elizabeth D Arnold, Michael P Walsh, Melvin E Thomas, Tamara Westover, Masayuki Umeda, Jing Ma, S. Pruett-Miller, J. Klco
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J K Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Metastasis is still the leading cause of cancer-related death. It happens when disseminated tumor cells (DTCs) successfully navigate a series of steps and adapt to the unique conditions of distant organs. In this review, key molecular and immune mechanisms that shape metastatic spread, long-term survival, and eventual outgrowth are examined, with a focus on how tumor-intrinsic programs interact with extracellular matrix (ECM) remodeling, angiogenesis, and immune regulation. Gene networks that sustain tumor-cell plasticity and invasion are described, including EMT-linked transcription factors such as SNAIL and TWIST, as well as broader transcriptional regulators like SP1. Also, how epigenetic mechanisms, such as EZH2 activity, DNA methylation, chromatin remodeling, and noncoding RNAs, lock in pro-metastatic states and support adaptation under therapeutic pressure. Finally, proteases and matrix-modifying enzymes that physically and biochemically reshape tissues, including MMPs, uPA, cathepsins, LOX/LOXL2, and heparinase, are discussed for their roles in releasing stored growth signals and building permissive niches that enable seeding and colonization. In parallel, immune-evasion strategies that protect circulating and newly seeded tumor cells are discussed, including platelet-mediated shielding, suppressive myeloid populations, checkpoint signaling, and stromal barriers that exclude effector lymphocytes. A major focus is metastatic dormancy, cellular, angiogenic, and immune-mediated, framed as a reversible survival state regulated by stress signaling, adhesion cues, metabolic rewiring, and niche constraints, and as a key determinant of late relapse. Tumor-specific metastatic programs across mesenchymal malignancies (osteosarcoma, chondrosarcoma, and liposarcoma) and selected high-burden cancers (melanoma, hepatocellular carcinoma, glioblastoma, and breast cancer) are highlighted, emphasizing shared principles and divergent organotropisms. Emerging therapeutic strategies that target both the “seed” and the “soil” are also discussed, including immunotherapy combinations, stromal/ECM normalization, chemokine-axis inhibition, epigenetic reprogramming, and liquid-biopsy-enabled minimal residual disease monitoring, to prevent reactivation and improve durable control of metastatic disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5ad40c8765430fb4e35e8df4878d707b5a6318" target='_blank'>
              Molecular and Immune Mechanisms Governing Cancer Metastasis, Including Dormancy, Microenvironmental Niches, and Tumor-Specific Programs
              </a>
            </td>
          <td>
            Dae Joong Kim
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="KRAS is one of the most frequently mutated genes in all human cancers, and its oncogenic mutation hotspots are glycine 12 (G12), glycine 13 (G13), glutamine 61 (Q61) and alanine 146 (A146). Among these hotspot mutations, A146 substitution mutations (A146X) occur relatively infrequently, except for haematopoietic and lymphoid cancers, suggesting that A146X causes intrinsically distinct KRAS signalling compared to other KRAS oncogenic alleles. However, due to the absence of model A146X cell lines derived from haematopoietic sources, the cellular mechanisms that cause the differences between KRAS.A146X and other common KRAS mutants, such as KRAS.G12X, remain largely unexplored. In this study, we developed a set of isogenic model haematopoietic cell lines expressing KRAS.A146T, KRAS.G12V and KRAS.G12G (non-mutated) from the endogenous locus by genetically modifying the human lymphoblastoid TK6 cell line. We found that TK6 cells carrying KRASA146T/+ or KRASG12V/+ exhibited increased replication stress compared to KRAS wild-type cells. Strikingly, KRASA146T/+ cells strongly rely on PrimPol for maintaining cellular survival upon replication stress. In contrast, KRASG12V/+ cells exhibited hypersensitivity to inhibitors for the ATR-Chk1 checkpoint signalling axis and to nucleoside analogues commonly used to treat cancers and viral infections. Our findings demonstrate that the endogenously expressed oncogenic KRAS mutations exacerbate the replication stress and reveal KRAS allele-specific replication phenotypes, facilitating the development of effective chemotherapies tailored to specific oncogenic KRAS mutation alleles and types of cancer. Moreover, our study offers valuable model cell lines for investigating mechanisms underlying replication vulnerability in cancers harbouring oncogenic KRAS mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5113da5b492954d75a6c3e0082f785f965a4200" target='_blank'>
              Generation of Human Haematopoietic Model Cell Lines Revealed Distinct Replication Stress Tolerance Between Two Oncogenic KRAS Mutations, G12V and A146T
              </a>
            </td>
          <td>
            Mone Okuda, Ryotaro Kawasumi, Kayoko Tanaka, Kouji Hirota
          </td>
          <td>2026-01-28</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Wellcome Trust scientists have shown that “mutational signatures” in specific nucleotide contexts accumulate in genomes of mammalian tissues, providing clues to underlying causes of specific signatures. Analysis of cancer genomes has identified more than 50 single‐base substitution (SBS) signatures, with SBS1, SBS5, and SBS40 linked to aging and present in normal tissues. SBS1 results from cytosine demethylation, whereas SBS5 and SBS40 arise from unknown endogenous mechanisms. We hypothesized that loss of fragile‐site genes drives these two signatures. FHIT, located at FRA3B, is frequently deleted in cancers, and Fhit‐deficient mouse tissues exhibit a mutation profile resembling human SBS5. Data mining of cancer genome sequences showed that FHIT/Fra3B was the gene loss most significantly correlated with human SBS5 mutations, likely representing the endogenous molecular process determining SBS5 mutations in soma, germ cells, and cancers, and inversely associated with lifespan in mammals. Exome sequences of Fhit knockout tissues revealed ~1000 SBS1 and ~4000 SBS5 mutations per sample, compared to markedly fewer in wild‐type controls. Using SigProfilerAssignment, similar analyses performed with Fhit ko samples where Wwox is also frequently lost identified SBS40c. These findings implicate fragile‐site gene inactivation as a major source of clock‐like mutational signatures, contributing to lifelong mutation accumulation and potentially influencing aging, evolution, and speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b576f2a56d83e49e328a800a1ee42f5ac37825f" target='_blank'>
              Endogenous Processes Underlying Clock‐Like Mutational Signatures
              </a>
            </td>
          <td>
            T. Druck, Rami I. Aqeilan, C. Marcelo Aldaz, N. Zanesi, Kay Huebner
          </td>
          <td>2026-01-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cervical cancer remains a major global public health concern, with persistent infection by high-risk human papillomavirus (hrHPV) types recognized as the primary etiological factor. This review explores the multifactorial nature of the disease, focusing on the complex interplay between host genetic susceptibility and epigenetic alterations that drive cervical carcinogenesis. Evidence from genome-wide association studies (GWAS) is discussed, highlighting the contribution of specific genetic loci, predominantly within the HLA region, to susceptibility to HPV infection and disease progression. In parallel, the review examines the molecular mechanisms by which the viral oncoproteins E6 and E7 promote genetic instability and epigenetic reprogramming, including histone modifications and dysregulation of non-coding RNAs. Particular emphasis is placed on DNA methylation, affecting both the viral genome and host genes such as FAM19A4, CADM1, PAX1, and MAL, as a promising biomarker for triage and detection of high-grade intraepithelial lesions (CIN2+). Finally, the review evaluates currently available methylation-based assays and self-sampling devices, highlighting their potential to enhance diagnostic accuracy and increase participation in cervical cancer screening programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91df97fd2742e76a0b3c768e22567f076cd3a39f" target='_blank'>
              HPV-Driven Cervical Carcinogenesis: Genetic and Epigenetic Mechanisms and Diagnostic Approaches
              </a>
            </td>
          <td>
            E. Legaki, Theofania Lappa, Konstantina-Lida Prasoula, Zoi Kardasi, E. Kalampokas, Theodoros Kalampokas, M. Roubelakis, Ekaterina Charvalos, M. Gazouli
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer is one of the most common malignant tumors among women worldwide, and its high heterogeneity poses significant challenges for early diagnosis and precise treatment. Despite the continuous development of molecular typing and targeted therapy, how to precisely distinguish the benign and malignant regions of tumors at the spatial level and analyze their molecular characteristics remains an urgent problem to be solved. This study is based on spatial transcriptomics technology on the 10x Genomics Visium platform. Systematic transcriptional analyses were conducted on human breast cancer tissue sections. Through differential expression analysis of DCIS/LCIS and IDC regions, a total of 1,143 significantly differentially expressed genes were identified, among which genes such as S100A6, STC2, and TFF3 were significantly upregulated in breast cancer progression, suggesting a key role in malignant transformation of tumors. Gene ontology enrichment analysis revealed that the differentially expressed genes were significantly aggregated in key functional modules such as metabolic pathways, MAPK signaling pathways, cell cycle and cancer-related pathways. in particular, the significant enrichment of the Rap1 signaling pathway and Pathways in cancer revealed the synergistic activation mechanism of multi-level biological processes such as cell adhesion, migration, and matrix degradation during the transformation of breast cancer from benign to malignant. In summary, this study revealed the molecular heterogeneity in the benign and malignant regions of breast cancer and the spatial distribution characteristics of key signaling pathways at the tissue level through spatial transcriptomics, providing new theoretical basis and spatial omics evidence for spatial molecular typing of breast cancer, targeted drug development, and optimization of individualized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c838369620db12e859841cf0ebbe90e1c62e11f" target='_blank'>
              Unraveling Benign and Malignant Tumor Heterogeneity Through Spatial Transcriptomics
              </a>
            </td>
          <td>
            Haosheng Lyu
          </td>
          <td>2026-01-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22515b198019d64378e7353a52adee0b1a72d0ba" target='_blank'>
              Querying functional drivers in primary murine naïve and memory T cells using RNP-mediated CRISPR-Cas9 Gene Deletion
              </a>
            </td>
          <td>
            Surojit Sarkar, Yevgeniy Yuzefpolskiy, V. Kalia
          </td>
          <td>2026-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) remains a highly prevalent and aggressive malignancy with limited improvements in survival rates. One of the major obstacles to successful treatment is the development of chemoresistance, which contributes to recurrence, metastasis, and treatment failure. This narrative review aims to integrate current evidence on the molecular and cellular mechanisms that drive chemoresistance in OSCC and to delineate how these processes converge under therapeutic pressure. A structured search was performed to identify relevant studies addressing chemoresistance in OSCC, focusing on preclinical and translational evidence. Multiple interconnected mechanisms have been implicated in driving resistance in OSCC, including epigenetic alterations, deregulated signaling pathways, cancer stem cell plasticity, epithelial–mesenchymal transition (EMT), interactions with the tumor microenvironment (TME), drug efflux mediated by ATP-binding cassette (ABC) transporters, and enhanced DNA damage response. In combination, these mechanisms support tumor persistence and limit effective antitumor immunity. Emerging strategies such as epigenetic modulators, signaling pathway inhibitors, immunomodulation, and nanomedicine-based delivery systems have shown promising results in preclinical models. By highlighting convergent resistance networks, this integrative perspective supports the rational design of combination therapies and biomarker-guided strategies aimed at overcoming chemoresistance in OSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25af512c23cbe217030a9a034876fead83f7c218" target='_blank'>
              Molecular Mechanisms of Chemoresistance in Oral Squamous Cell Carcinoma: A Narrative Review with Present and Future Perspectives
              </a>
            </td>
          <td>
            E. Morais, L. Q. R. de Oliveira, Cintia Eliza Marques, H. Morais, Deborah Gondim Lambert Moreira, Lucas de Araújo Albuquerque, J. R. V. Silva, R. Freitas, Ricardo Della Coletta
          </td>
          <td>2026-01-05</td>
          <td>Applied Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3333bb3dc7aa46ec7dea00023d761c89be2478c2" target='_blank'>
              Large-scale single-cell analysis and in silico perturbation reveal dynamic evolution of HCC: from initiation to therapeutic targeting.
              </a>
            </td>
          <td>
            Peng Xia, Shuang Si, Daosen Fu, Luyao Liu, Dongliang Yang, Yanan Guo, Yixiao Tian, Pengfei Ji, Xinyi Yuan, Yingxia Tian, Rong Shen, Degui Wang
          </td>
          <td>2026-01-30</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Prostate cancer depends on androgen receptor (AR) signaling for growth, which is why androgen deprivation (castration) therapy is effective at early stages. However, many tumors eventually progress to a lethal form known as castration-resistant prostate cancer (CRPC). A subset of CRPC tumors bypass dependency on AR signaling by acquiring lineage plasticity, where prostate cancer cells transdifferentiate into alternate cellular states through epigenetic reprogramming. Neuroendocrine (NE) prostate cancer represents one well-known lineage plasticity phenotype. Nevertheless, most AR-independent tumors do not exhibit NE features and are defined as AR-negative/NE-negative or “double-negative prostate cancer” (DNPC). In a collaboration with Dr. Ekta Khurana’s computational genomics lab at Weill Cornell Medicine, we recently classified CRPC into four epigenetic subtypes, including the well-established 1) AR and 2) NE, as well as the novel DNPC subgroups 3) WNT and 4) stem cell-like (SCL) (PMID: 35617398). We focused on the SCL subtype as it is the second most common group in CRPC patients and lacks therapeutic targets. Using functional genomic approaches, we found that YAP/TAZ/TEAD cooperates with FOSL1 to drive the SCL lineage and growth of SCL models. We therefore hypothesize that the heightened dependency on the YAP/TAZ/TEAD/FOSL1 transcriptional program represents a therapeutic vulnerability in CRPC-SCL. To test this, we exposed CRPC models to TEAD inhibitors and found robust growth suppression in SCL cells compared to non-SCL cells in vitro. To evaluate whether the TEAD inhibitors are on-target, we performed transcriptomic profiling in SCL models and observed downregulation of YAP/TAZ gene signature as well as FOSL1 expression, which phenocopies the effects of YAP/TAZ double knockdown. To define the cistromes of these factors upon TEAD inhibition, we performed ChIP-seq and observed reduced co-occupancy at consensus sites, suggesting the disruption of the YAP/TAZ/TEAD/FOSL1 transcriptional circuit by the small molecule compound. To determine whether these phenotypes are recapitulated in vivo, we will treat mice harboring CRPC-SCL xenografts with TEAD inhibitors to assess growth response and evaluate epigenetic and transcriptional response using single-nucleus Multiome (ATAC+RNA). These studies will establish whether small molecule inhibition of TEAD is a promising strategy for the treatment of CRPC-SCL and allow high-resolution analysis of cell state transitions, with a focus on loss of SCL-specific signatures and potential emergence of AR/NE programs as adaptive resistance mechanisms.



 Chen Khuan Wong, Dan Li, Hongsu Wang, Marjorie Roskes, Weiling Li, Shipra Shukla, Dana Schoeps, Ekta Khurana, Yu Chen. Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A074.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912fdd784be0fd097e91b40d9e4c6933dd2c1b14" target='_blank'>
              Abstract A074: Defining and targeting drivers of lineage plasticity in stem cell-like prostate cancer
              </a>
            </td>
          <td>
            C. K. Wong, Dan Li, Hongsu Wang, M. Roskes, Weiling Li, Shipra Shukla, Dana M. Schoeps, E. Khurana, Yu Chen
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Somatic mosaicism is pervasively observed in human aging, with clonal expansions of cells harboring mutations in recurrently mutated driver genes. Bulk sequencing of tissues captures mutation frequencies, but cannot reconstruct clonal architectures nor delineate how driver mutations impact cellular phenotypes. We developed single-cell Genotype-to-Phenotype sequencing (scG2P) for high-throughput, highly-multiplexed, joint capture of genotyping of mutation hotspots and mRNA markers. We applied scG2P to aged esophagus samples from six individuals and observed large numbers of clones with a single driver event, accompanied by rare clones with two driver mutations. NOTCH1 mutants dominate the clonal landscape and are linked to stunted epithelial differentiation, while TP53 mutants promote clonal expansion through both differentiation biases and increased cell cycling. Thus, joint single-cell highly multiplexed capture of somatic mutations and mRNA transcripts enables high resolution reconstruction of clonal architecture and associated phenotypes in solid tissue somatic mosaicism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e453d2ebb8e890d0f12017626a25469ee8fcb34" target='_blank'>
              Genotype-to-phenotype mapping of somatic clonal mosaicism via single-cell co-capture of DNA mutations and mRNA transcripts.
              </a>
            </td>
          <td>
            Dennis J. Yuan, John Zinno, Theo Botella, Dalia Dhingra, Shu Wang, Allegra G Hawkins, Ariel D Swett, Jesus Sotelo, Ramya Raviram, Clayton Hughes, Catherine Potenski, Katharine D Godfrey, Kara M Ainsworth, Shuzhen Xu, Jianwen Que, Julian A Abrams, Akira Yokoyama, N. Kakiuchi, Seishi Ogawa, D. Landau
          </td>
          <td>2025-12-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18e77f40b4649c94b742354f1776a0273d7ae6d5" target='_blank'>
              A Hallmark-Integrated, Agent-Based Framework for Intratumor Heterogeneity in Melanoma Evolution
              </a>
            </td>
          <td>
            Khola Jamshad, Trachette L. Jackson
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The effects of cis-regulatory alterations in prostate cancer (PCa) are insufficiently characterized, presenting an opportunity to discover driver genes and therapeutic targets. To comprehensively study these effects, we identify genes undergoing allele-specific expression (ASE) in localized PCa and metastatic castration-resistant PCa (mCRPC) samples. By defining recurrent ASE events across prostate tissue and tumor-enriched ASE, we develop CASEDI, a computational framework for prioritizing cancer drivers by integrating ASE and clinical data. CASEDI reveals genes showing recurrent ASE and altered expression in PCa, including AR-regulated and oncogenic ACSM1. mCRPC samples show enrichment of ASE in DNA repair, resistance pathways, and oncogenes and increased frequency of monoallelic expression (MAE) compared to localized tumors. We define an mCRPC gene signature based on MAE status that identifies a subgroup of localized patients with worse prognosis. Using ASE analysis, we expand the landscape of cis-regulatory events in PCa to inform the identification of additional therapeutic targets. Implications: This study develops a framework for identifying cancer drivers using prostate cancer allele-specific expression (ASE) data, generates a comprehensive dataset of ASE in prostate cancer and highlights candidate targets in tumorigenesis and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/616d957dd08b5a8335edc5a1298ef89197981ba1" target='_blank'>
              Landscape of allele-specific expression in prostate cancer reveals recurrent, stage-specific events in AR signaling and resistance pathways.
              </a>
            </td>
          <td>
            Maggie Tsui, Kevin Hu, Hanbing Song, Sarah C Hsu, Yih-An Chen, Lorraine Nuniz, Julia H Pham, Chih-Hao Chang, Keliana S F Hui, David A. Quigley, Jingjing Li, Franklin W. Huang
          </td>
          <td>2026-01-16</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9627eeea0e72cf88ceb4ce602d15c8920df0c909" target='_blank'>
              Splicing-driven post-translational dysregulation: a new frontier for precision cancer medicine and immunotherapy.
              </a>
            </td>
          <td>
            Sael Alatawi
          </td>
          <td>2026-01-24</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Leukemia is a heterogeneous group of hematologic malignancies characterized by the dysfunctional proliferation of white blood cells in the bone marrow. Genetic alterations are important risk factors for the development and progression of leukemia, and their detection is crucial. Although many genetic techniques, including karyotyping, fluorescence in situ hybridization (FISH), and polymerase chain reaction (PCR), have provided valuable information, they all have the limitation of incomplete genomic coverage. The evolution of genomic technologies, including Next Generation Sequencing (NGS) and Third Generation Sequencing (TGS), has enabled a more comprehensive and detailed characterization of the genetic landscape of leukemia. NGS technology has revolutionized leukemia diagnosis, treatment, and minimal residual disease monitoring, and its integration into routine leukemia care will enhance patient outcomes and pave the way for truly personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f067f109a8a6e2615bdfa692706a81a96d161ae" target='_blank'>
              The Application of Next-Generation Sequencing in Leukemia
              </a>
            </td>
          <td>
            Konstantinos Agiannitopoulos, E. Kouvidi
          </td>
          <td>2025-12-25</td>
          <td>OBM Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The CRISPR-Cas9 gene editing system represents a powerful approach for precision targeting of genetic determinants involved in cancer cell survival. This study evaluated the ability of CRISPR-Cas9 to induce apoptosis in cancer cells through targeted disruption of anti-apoptotic genes. Acute myeloid leukemia (HL-60 and KG-1) and anaplastic thyroid cancer (8505C and C643) cell lines were used to knockout BIRC5 (survivin) and MADD using specific single-guide RNAs. Efficient genome editing was achieved, with indel formation exceeding 50% across all models. Targeting BIRC5 and MADD significantly reduced cell viability to 39.8–54.6% of control levels and induced substantial apoptosis. Total apoptotic populations reached 52.8–60.1% following BIRC5 knockout and 45.3–50.6% following MADD knockout. Apoptosis induction was confirmed by caspase-3/7 activation, increased Sub-G1 accumulation, and cleavage of apoptosis-related proteins, indicating activation of the intrinsic apoptotic pathway.Notably, BIRC5 disruption consistently produced stronger pro-apoptotic effects than MADD across all cancer models. Overall, these findings highlight the therapeutic potential of CRISPR-Cas9–mediated targeting of anti-apoptotic genes as a promising strategy for precision oncology and for overcoming resistance to conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c6b9d8f35f7365dc8f3a9d540781b27c8b647d" target='_blank'>
              Exploration of CRISPR-Cas9 Gene Editing System Utilization for Targeted Induction of Apoptosis in Cancer Cells
              </a>
            </td>
          <td>
            Resti Ariani, Eka Cahya Muliawati, Zola Efa Harnis
          </td>
          <td>2026-01-31</td>
          <td>Science Get Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT RNA‐targeted therapy is reshaping molecular medicine by shifting the traditional “protein‐centric” view toward an “RNA‐regulatory network” paradigm. Beyond carrying genetic information, RNA plays essential roles in posttranscriptional regulation, signaling pathways, and epigenetic modulation. Advances in high‐throughput sequencing, structural biology, and delivery technologies have accelerated the development of diverse RNA therapeutics, including antisense oligonucleotides (ASOs), small interfering RNA (siRNA), microRNA (miRNA) modulators, messenger RNA (mRNA) therapeutics, aptamers, short hairpin RNA, and CRISPR/Cas‐guided single‐guide RNAs. However, a concise comparison of these major RNA modalities and the translational barriers that limit their broader clinical application is still lacking. This review outlines the mechanisms and representative applications of these RNA‐based strategies in gene silencing, editing, protein replacement, immune activation, and targeted drug delivery. Special emphasis is placed on ASOs and siRNAs for neurological, metabolic, and infectious diseases, as well as mRNA therapeutics that are transforming vaccine development. Common challenges‐such as in vivo stability, delivery efficiency, and immune activation‐are also discussed. Finally, we highlight how chemical modification, nanotechnology, and artificial intelligence‐assisted design are enhancing the specificity, stability, and safety of RNA therapeutics, providing a framework for advancing next‐generation precision RNA medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8a098f4a76e1439134aee84974e5e6a557a0fc" target='_blank'>
              Progress in RNA‐Targeted Therapeutics for Human Diseases
              </a>
            </td>
          <td>
            Wangzheqi Zhang, Aimin Jiang, Bin‐Kui Jia, Yuming Jin, Yinghu Chen, Zhaoyu Li, Yan Liao, Haoling Zhang, Zhiheng Lin, Xiao Fang, Linhui Wang
          </td>
          <td>2026-02-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Bone sarcomas remain lethal despite multimodal therapy, primarily because the mineralized, immunosuppressive tumor microenvironment (TME) promotes chemo- and immune-resistance. Integrating single-cell and spatial omics across osteosarcoma, Ewing sarcoma and chondrosarcoma delineates subtype-specific TME archetypes dominated by M2 macrophages, exhausted T cells and a stiff extracellular matrix. Mechanistic dissection reveals tractable vulnerabilities, myeloid reprogramming, extracellular matrix modulation and metabolic and epigenetic checkpoints, that can be targeted with bone-selective delivery systems and biomarker-driven combination trials to convert therapeutic failure into durable remission. Therefore, the aim of the present review is to synthesize the latest single-cell, spatial and functional data to map bone-sarcoma TME heterogeneity, dissect resistance mechanisms and propose integrated, biomarker-guided therapeutic strategies that can be translated into treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42d68e5855e4884a22e5af558a0428be6a50022" target='_blank'>
              Tumor microenvironment in bone sarcomas: Implications for immunotherapy and emerging therapeutic vulnerabilities (Review)
              </a>
            </td>
          <td>
            Wentao Li, Lijun Lv, Yibin Jin, Xin Yuan
          </td>
          <td>2026-01-16</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions.
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="In the rapidly evolving field of hematology, the diagnosis of leukemias and lymphomas poses major challenges, despite significant genetic advancements. Although established diagnostic methods comprise a multidisciplinary approach and are considered gold standard, in some cases they fall short in conclusively identifying hematological neoplasms. In this context, the current SIRIUS study (NCT05046444) delves into the potential of whole genome sequencing (WGS) and whole transcriptome sequencing (WTS) to bridge diagnostic gaps. By analyzing 106 patients with an unclear diagnosis or clinical condition following gold standard diagnostics, our study demonstrates that WGS and WTS can uncover a broader range of somatic alterations, including rare single-nucleotide variants (SNVs), small copy number variations (CNVs), and aberrant gene expression patterns not detected by conventional diagnostics. WGS and WTS provided additional diagnostic insights in 25% of cases, suggesting their value not only in enhancing diagnostic accuracy but also in contributing to more informed prognostic assessments and personalized treatment strategies. Therefore, our study underscores the importance of integrating WGS and WTS into the diagnostic toolbox for hematological neoplasms. This approach promises not only to improve patient outcomes but also to do so in a manner that is both financially sustainable and ethically sound.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8e0109a6619c55f8f417adf50a2b9d3386b8ef0" target='_blank'>
              Solving Riddles Through Sequencing (SIRIUS): unlocking hematologic diagnoses by whole genome and transcriptome sequencing
              </a>
            </td>
          <td>
            M. Truger, M. Meggendorfer, W. Walter, S. Hutter, Winfried Alsdorf, Gero Massenkeil, Uwe M. Martens, W. Kern, Katharina Hörst, Constanze Kühn, A. Reiter, A. Hochhaus, T. Haferlach
          </td>
          <td>2025-12-19</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="BACKGROUND
Mutant IDH1 (mIDH1) defines a therapeutically-targetable subtype of intrahepatic cholangiocarcinoma (ICC), with the mIDH1 inhibitor ivosidenib approved for advanced disease. A subset of patients experiences prolonged disease stabilization, although the molecular basis for eventual progression remains poorly defined.


EXPERIMENTAL DESIGN
We performed molecular profiling of matched baseline and post-progression circulating tumor DNA (ctDNA) samples from patients with mIDH1 ICC enrolled in the ClarIDHy phase III trial. Functional studies were conducted to characterize candidate resistance mechanisms.


RESULTS
Longitudinal ctDNA analysis of 18 patients treated with ivosidenib for >6 months revealed emergent genomic alterations in multiple cases. Acquired mutations in MAPK-pathway genes (KRAS, NRAS, MAP2K1, NF1) were identified in five cases, with instances of concurrent alterations and/or high variant allele fractions (VAF). Additional candidate resistance events included a secondary IDH1 mutation and a hotspot IDH2 mutation, detected at low VAF in the same patient. Functional studies demonstrated that these IDH mutations conferred sustained 2-hydroxyglutarate (2HG) production and ivosidenib resistance, whereas MAPK activation blunted gene expression induced by ivosidenib plus interferon-γ, a key therapeutic output of mIDH1 inhibition. In parallel, baseline ctDNA profiling of 81 patients linked ARID1A mutations and elevated mIDH1 VAF to reduced clinical benefit.


CONCLUSIONS
MAPK-pathway alterations represent a recurrent mechanism of resistance to mIDH1 inhibition in ICC, while emergent IDH1/IDH2 mutations appear infrequent. Functional data suggest that MAPK-mediated resistance may involve impaired IFN signaling. These results support MAPK-directed combination strategies and highlight the utility of ctDNA profiling to identify predictive and resistance biomarkers in mIDH1-driven ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aefbd8f840b00705379daf10a2c077d305123a7" target='_blank'>
              MAPK Pathway Mutations Emerge in Mutant-IDH1 Inhibitor-Resistant Cholangiocarcinoma and Attenuate the Interferon Response.
              </a>
            </td>
          <td>
            J. Wan, H. Saatcioglu, H. Ellis, E. Aguado-Fraile, Qin Xu, Hiroshi Kondo, R. Manguso, I. Gritti, B. Norden, Carolina Noble, R. Corcoran, Sam J Lubner, M. Javle, G. Abou-Alfa, James M Cleary, R. Kelley, M. Borad, T. Macarulla, Do-Youn Oh, Scott Daigle, C. Gliser, Susan Pandya, Sung Choe, A. Tron, N. Bardeesy
          </td>
          <td>2025-12-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Sickle cell anemia (SCA) is a monogenic inherited disease caused by a single point mutation in the HBB gene. Traditional treatments are ineffective in fundamentally correcting the genetic defect. In recent years, CRISPR-Cas9, an emerging gene editing technology, has shown great potential in the treatment of SCA. This article systematically reviews the mechanism of action of CRISPR-Cas9 and compares it with other editing tools. We focus on its application in SCA treatment, including strategies such as repairing the HBB gene mutation or targeting the HBG1/2 promoter to activate fetal hemoglobin. Furthermore, based on data from cell-based experiments, animal models, and clinical trials, we discuss the efficacy and limitations of representative therapies such as CTX001, Casgevy, and Lyfgenia. Results have shown that CRISPR-Cas9 can significantly improve red blood cell function in patients, and some therapies have received clinical approval, marking a significant breakthrough in its transition from laboratory research to clinical application. However, challenges remain, such as off-target effects, insufficient delivery efficiency, high treatment costs, and transplant-related complications, which need to be addressed. In the future, emerging strategies such as RNA editing are expected to expand into broader areas, including tumor treatment, while improving safety, providing safer and more accessible treatment options for monogenic diseases such as SCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732f63e17e30e7bab825eb7a3cacdb558c0d1f25" target='_blank'>
              CRISPR-Cas9 Technology in the Treatment of Sickle Cell Anemia (SCA)
              </a>
            </td>
          <td>
            Zihao Wang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Accurate ABO typing is essential for transfusion safety, yet weak A/B subgroups continue to present diagnostic challenges. Although most weakened ABO antigen expressions are linked to missense mutations or small indels within coding regions, the contribution of non-coding regulatory variants −especially large structural alterations−remains insufficiently characterized. Standard genotyping methods, including Sanger sequencing and targeted next-generation sequencing, often fail to detect large upstream deletions, leaving some serological-genotypic discrepancies unresolved in donors and patients. Such gaps hinder accurate transfusion risk assessment. To clarify these mechanisms, this study performed detailed ABO blood typing in a donor with an A subgroup and his family and explored the molecular basis for weakened A antigen expression. Materials and methods ABO phenotypes of the proband and his parents, spouse, two daughters, and son were determined using serological testing. Full-length ABO gene sequencing was then performed using third-generation single-molecule sequencing. Results The proband, his father, and his son displayed the Aweak phenotype. All three carried a heterozygous 12,309-bp deletion located upstream of the ABO gene promoter, along with a normal ABOA1.02 allele. The remaining family members carried genotypes ABOO.01.01/ABOO.01.01 (mother), ABOB.01/ABOO.01.02 (wife), ABOB.01/ABOO.01.01 (daughter 1), and ABOO.01.01/ABOO.01.02 (daughter 2). No variants were detected in coding regions, introns, or splice sites. Pedigree analysis confirmed that the proband inherited the ABOA1.02 allele with the 12.3-kb deletion from his father and passed it to his son. Discussion This study identifies, for the first time, a large 12.3-kb upstream deletion of the ABO promoter responsible for weakened erythrocyte A antigen expression. The finding expands current understanding of regulatory mechanisms underlying weak ABO phenotypes and highlights the value of advanced sequencing for transfusion safety.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f4146b0f082d3ba147e0860c65e35ef41dbacf" target='_blank'>
              Molecular mechanism of a novel Aweak subtype in Chinese pedigrees: identification and analysis of a large genomic deletion in the promoter upstream distal region of the ABO gene
              </a>
            </td>
          <td>
            Jian-Cheng Liu, Feng Shao, Jie Yang, Xiaoyin Mao, Xiaoyun Bu, Jing Hai
          </td>
          <td>2025-12-17</td>
          <td>Blood Transfusion</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The progression of prostate cancer (PCa) to metastatic disease remains a critical clinical challenge, making it essential to gain a detailed understanding of the factors that enable certain subclones to invade and spread. Building on our previous study, which examined genomic variations in benign and malignant prostate tissues by inferring copy number alterations (CNA) from spatial transcriptomics data1, this study aims to trace the spatial evolutionary trajectories of PCa subclones from the primary tumor to matched lymph node metastases and to identify alterations in associated microenvironments and early diagnostic markers. To achieve this, we performed spatial transcriptomics on entire prostate axial disks and patient-matched lymph node metastases from 10 individuals. Standard spatial transcriptomics (55 µm) was used for tissue profiling; from the spatial transcriptomics data, copy number alterations (CNAs) were inferred to identify distinct tumor subclones, and a clone-tree was constructed to describe their evolutionary relationships. Additionally, a high-resolution spatial transcriptomics (2 µm) technology was employed to gain a cellular-level view of the immediate tumor-microenvironment (TME) surrounding specific subclones. Analysis of over 1,000,000 barcoded regions identified many distinct tumor subclones, enabling us to trace their evolutionary trajectories spatially. We observed notable subclonal events within the lymph nodes, including polyclonal colonization, indicating multiple dissemination events during the evolution of the primary disease. Exploration of the immediate TME revealed significant cellular heterogeneity and upregulation of genes related to antigen presentation and inflammatory pathways concentrated near ancestral tumor clones, specifically at the tumor border. By focusing on the inferred CNA profiles of the metastasizing clone across patients, we identified several common features. Additionally, we were able to identify potential lethal disease as early as the diagnostic core needle biopsy stage by detecting features of metastasis and pinpointing the metastasizing clone. In summary, our study provides a detailed spatial map of PCa clonal evolution and dissemination, linking primary tumors to nodal metastases and revealing altered cell composition and gene expression around tumor clone borders. Importantly, it demonstrates subclonal events within lymph nodes, polyclonal colonization, and the potential to identify metastasizing clones as early as the diagnostic phase, which holds strong implications for precision oncology and early intervention.
 1. Erickson, A., He, M., Berglund, E. et al. Spatially resolved clonal copy number alterations in benign and malignant tissue. Nature 608, 360–367 (2022). https://doi.org/10.1038/s41586-022-05023-2



 Mengxiao He, Sandy Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, Ian G. Mills, Joakim Lundeberg, Alastair D. Lamb. Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604426087ba878274c47234a7ce00263acf256a4" target='_blank'>
              Abstract PR013: Comprehensive spatial profiling of prostate cancer metastasis: Mapping clonal evolution, microenvironment dynamics, and early metastatic markers
              </a>
            </td>
          <td>
            Mengxiao He, S. Figiel, Max Beesley, Jintong Shi, Wencheng Yin, Emmanouela Perisynaki, Richard Colling, I. Mills, J. Lundeberg, Alastair D. Lamb
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background/Objectives: The further application of high-coverage whole genome sequencing in fields such as paleogenomics, forensic investigations, and conservation genomics is impeded by two major barriers: extremely high costs and stringent sample requirements. Utilizing low-coverage sequencing offers a practical solution to these constraints; however, this approach introduces a primary challenge—the necessity to reconstruct distorted genomic information for downstream analysis. Methods: Analytical experiments conducted on low- to medium-coverage sequencing data confirmed the accuracy of several existing methods for inferring relationships up to the third degree and distinguishing unrelated individuals. Subsequently, efforts were made to evaluate allele-frequency-independent methods within animal genomics, where analyses are likely to encounter challenges such as uncertain allele frequencies, diverse sample types, and suboptimal sample quality. Kinship inference was performed on a total of 33 pairs of animal samples across three species, comprising nine parent–offspring pairs and four full-sibling pairs. Results: The analysis revealed that two efficient algorithm implementations (READ and KIN) successfully identified all unrelated pairs. Notably, among the various algorithms utilized, only KIN exhibited confusion between first- and second-degree relationships when subjected to. Conclusions: This study has filled a critical gap in the existing literature by conducting a comprehensive evaluation of various algorithms on low-coverage sequencing data derived from authentic human and animal samples, accompanied by detailed ground truth—a vital task that has been overlooked.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3096df84b32400375ed5e14d8a6c6380a5094d6e" target='_blank'>
              Predicting Genetic Relatedness from Low-Coverage Sequencing Data of Human and Animal Genomes Using Various Algorithms
              </a>
            </td>
          <td>
            Xinyi Lin, Shuang Han, Qifan Sun, Yuting Lei, Zhen Liu, Xueling Ou
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chronic myeloid leukemia is a hematopoietic stem cell malignancy characterized by excessive proliferation of white blood cells, primarily driven by the Philadelphia chromosome (t 9;22). Despite the availability of tyrosine kinase inhibitors, disease progression and therapy resistance remain major challenges, often linked to additional somatic mutations. Objectives: To find out the mutational status and expression variations of the ASXL1 gene hotspot region in CML patients treated by TKIs. Methods: This retrospective cross-sectional study was conducted at the Department of Zoology, Division of Science and Technology, University of Education, Lahore, Pakistan. DNA and mRNA of 50 CML patients were analyzed alongside 10 healthy controls and statistically assessed using SPSS. Results: Results showed that sequence analysis of exon 13 identified hotspot mutations, including two novel missense alterations, G659S and D667E, within the coding region of ASXL1. In-silico analysis also suggested potential structural and functional relevance of these variants. The ASXL1 gene expression showed a progressive but statistically non-significant down-regulation in CML patients across disease phases (p=0.662), disease duration (p=0.42), treatment with second-line versus first-line TKIs (p=0.412), and wild-type and mutant (p=0.544). Conclusions:  It is concluded that two novel missense alterations, G6559S and D667E, were identified in the hotspot region, and ASXL1 gene expression was downregulated non-significantly in CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ee16b9a16245af7fb856280afba55f65850e27" target='_blank'>
              Identification of Novel Alterations in the ASXL1 Gene and Its Expression Profile in CML Patients
              </a>
            </td>
          <td>
            R. Mehmood, Khuram Shehzad Khan, A. Akram, Sikandar Hayat, Amjad Zafar
          </td>
          <td>2025-12-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46cb1762556f83822b8fcd8773e27d5e3004fd71" target='_blank'>
              Circular RNAs: key regulators in cancer stem cell dynamics
              </a>
            </td>
          <td>
            Puneet Jain, Varsha Kaushik, Ravindresh Chhabra
          </td>
          <td>2026-02-03</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung cancer is a highly malignant disease, posing a significant threat to global health. The presence of tumor heterogeneity results in substantial variations in prognosis and therapeutic responses among patients. Advances in bulk RNA sequencing and single-cell RNA sequencing have facilitated the identification of driver gene mutations and the exploration of cellular diversity within tumors. However, tumors are complex ecosystems comprising both tumor cells and their microenvironment, where interactions among different cell types give rise to specific functional structural units that collectively drive tumorigenesis and progression. The emergence of spatial omics technologies has allowed for the analysis of tumor ecosystems, providing unprecedented insights into tumor heterogeneity. This review aims to present updates on spatial omics technologies and data analysis algorithms, discuss current technical limitations, and explore potential future developments. Furthermore, we summarize the latest applications of spatial omics in elucidating lung cancer heterogeneity, investigating mechanisms of lung cancer progression and drug resistance, and identifying novel biomarkers. Based on these findings, we propose strategies for integrating spatial omics into lung cancer research, offering new perspectives for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca909ba170b3407f8c473dadaa8f67180cd2619" target='_blank'>
              The Evolution of Spatial Omics Technologies Introduces A Novel Avenue for Lung Cancer Research.
              </a>
            </td>
          <td>
            Yue He, Zifan Li, Wenxiang Wang, Xu Liu, Shanshan Lu, Jing Bai, Lin Weng, Qingna Zhang, Jun Wang, Kezhong Chen
          </td>
          <td>2026-01-30</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNMT1 variants are linked to complex neurodegenerative syndromes, yet their variant-specific functional and epigenomic consequences remain poorly defined. DNMT1 variants were identified in eight patients using gene-panel or whole-exome sequencing. Functional effects were assessed by site-directed mutagenesis and transient expression in HEK293T cells. Genome-wide methylation profiling of peripheral blood leukocyte DNA was performed using Nanopore sequencing, enabling direct quantification of 5-methylcytosine (5mC). CpG island-level differential methylation and gene set enrichment analysis (GSEA) were conducted. Variants in the replication foci targeting sequence (RFTS) domain (p.Y511H, p.Y540C, p.H569R) exhibited reduced DNMT1 protein expression, decreased enzymatic activity, and cytosolic aggregation. Variants in the C-terminal catalytic domain (p.A1334V and p.P1546S) showed reduced protein expression with relatively mild enzymatic impairment. Patients carrying the p.Y511H variant demonstrated a significant reduction in global 5mC levels compared with controls. Principal component analysis revealed distinct methylomic profiles separating most patients from controls, with marked intra- and inter-familial heterogeneity. CpG island-level analysis identified a single significantly hypomethylated region in p.Y511H carriers, and GSEA revealed differential enrichment of multiple Gene Ontology biological pathways. This study defines domain-dependent functional effects of DNMT1 variants and provides the first nanopore-based methylome analysis, revealing variant-specific and heterogeneous epigenomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08782723cf1505854fccf87fe2c66f251748e64" target='_blank'>
              Functional and Epigenomic Consequences of DNMT1 Variants in Inherited Neurological Disorders
              </a>
            </td>
          <td>
            Junhui Yuan, Y. Higuchi, M. Ando, A. Yoshimura, S. Nozuma, Y. Sakiyama, Takashi Kanda, Masahiro Nomoto, Takeshi Nakamura, Yasuyuki Nobuhara, Hiroshi Takashima
          </td>
          <td>2026-01-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that orchestrate spatiotemporal gene expression patterns in response to developmental cues and environmental stimuli. Genetic and epigenetic alterations in enhancers are associated with the initiation and progression of human diseases, including cancers. Over the past few decades, accumulating evidence has revealed that a class of nascent RNA transcripts, known as enhancer RNAs (eRNAs), is broadly transcribed from active enhancers. These eRNA species contribute to complex and dynamic gene regulatory networks under both physiological and pathological conditions through diverse mechanisms. Notably, dysregulated eRNA expression has been reported across various cancer types and is often correlated with patient survival outcomes. Consequently, eRNAs are emerging as promising biomarkers and therapeutic targets for cancer treatment. This review provides a comprehensive summary of the current understanding of eRNAs and their mechanisms of action in gene regulation. We discuss the critical roles of eRNAs in both health and disease and highlight their diagnostic and prognostic value, as well as their therapeutic potential in cancer. Additionally, we review current strategies for targeting RNA transcripts, including eRNAs, and discuss the major challenges in developing eRNA-targeted therapies. Finally, we propose future directions for advancing eRNA-based interventions in the treatment of human diseases, including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9397eec4991d568fddc2093214f80298c06bfee4" target='_blank'>
              Deciphering the regulatory landscape of enhancer RNAs in health and disease
              </a>
            </td>
          <td>
            Qian Wang, P. ten Dijke, Chuannan Fan
          </td>
          <td>2026-01-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="S i c k l e C e l l D i s e a s e ( S C D ) i s a n i n h e r i t e d hemoglobinopathy caused by a point mutation (GAG→GTG) at the 6th codon of the β-globin gene (HBB). Among traditional treatments, hydroxyurea requires lifelong administration and some patients show poor response; although hematopoietic stem cell transplantation (HSCT) can cure the disease, only 18% of patients can find HLA-matched donors, and there is also the risk of graft-versus-host disease (GVHD). The CRISPR-Cas9 gene editing technology provides a potential curative option for SCD by precisely modifying hematopoietic stem/progenitor cells (HSPCs). This article systematically reviews the clinical application strategies, experimental evidence, and existing challenges of this technology, and discusses its translational prospects in combination with the latest research findings. It covers the clinical effects of strategies such as targeting the BCL11A gene enhancer to activate fetal hemoglobin (HbF), directly correcting the E6V mutation of the HBB gene via homology-directed repair (HDR), and mimicking hereditary persistence of fetal hemoglobin (HPFH), as well as safety and accessibility challenges (e.g., off-target effects, preconditioning toxicity, and high treatment costs) and their corresponding solutions, aiming to provide comprehensive references for the research and application of gene therapy in SCD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/890a8f128bb576259bd63b951a1f15ea5bf4e0b1" target='_blank'>
              Application and Breakthrough of CRISPR-Cas9 Gene Editing Technology in the Clinical Treatment of Sickle Cell Disease
              </a>
            </td>
          <td>
            Boxun Sun
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Genetic mutations have proven to be the epicenters of cancer and disease progression. Traditional WXS sequencing and BLOSUM scoring can be used to infer the evolutionary conservation of amino acid substitutions, though these approaches are not informed by probable base pair sequence changes. Within gene mutation analysis, most tools focus on amino acid conservation or codon switching independently, limiting their ability to contextualize observed mutations against stochastic mutational processes. In the clinical setting, variants of unspecified significance remain difficult to interpret, as clinicians are often unable to determine whether observed mutations arise from oncogenic selection or from stochastic mutational degradation. Methods: We analyzed mutation sequences from the TCGA BRCA cohort for TP53 and PIK3CA and developed a model that integrates BLOSUM scoring with statistical modeling of base pair changes to evaluate deviation from codon-aware neutral expectations. Observed mutational distributions were compared against a stochastic neutral model to assess statistical significance. Results: Within the TCGA BRCA cohort, TP53 mutations were significantly more evolutionarily radical than expected under the codon-aware neutral model, while PIK3CA mutations were significantly more evolutionarily conservative, as determined using chi-square testing. These opposing patterns are consistent with the distinct functional roles of TP53 and PIK3CA in oncogenesis, where TP53 is inhibited through disruptive loss-of-function mutations, whereas PIK3CA is recurrently mutated in a manner that preserves protein structure and promotes constitutive pathway activation. This contrast reflects selective pressure toward disabling tumor suppressor function while maintaining persistent oncogenic signaling. Conclusions: Codon-aware neutral modeling provides a statistical framework for distinguishing mutations that deviate from stochastic expectations and may aid in the interpretation of variants of unspecified significance. By contextualizing mutational severity relative to neutral processes, this approach offers insight into tumor evolution and may support prognostic assessment without relying on predefined gene-level neutrality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ede46a7215880d43a910fedecc21269d21b24516" target='_blank'>
              GenBlosum: On Determining Whether Cancer Mutations Are Functional or Random
              </a>
            </td>
          <td>
            Alejandro Leyva, Muhammad Khalid Khan Niazi
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Large-scale single-cell genomics projects have revolutionized our understanding of human immune variation. Yet most studies to date have been Eurocentric, limited in cell-type resolution, or restricted to a single data modality. The newly published Chinese Immune Multi-Omics Atlas helps address these gaps by profiling 428 healthy Chinese adults using a multiomics single-cell approach that combines single-cell RNA sequencing and single-cell chromatin accessibility sequencing across over 10 million immune cells. This integrated strategy enabled the identification of 73 distinct immune cell subsets and the construction of cell-type–specific gene regulatory networks linking noncoding enhancers to target genes. The atlas delineated hundreds of enhancer modules (eRegulons), highlighting both established and novel regulators of immune cell identity. By aligning transcriptomic and epigenomic maps, Yin et al. show how expanding both the ancestral diversity and data modalities of immune cell genomics can reveal new biology and provide a valuable addition to global reference cell atlases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95dd47b2f0f754d8765c9bb6a6fd440e0d251aa3" target='_blank'>
              Charting immune variation through genetics and single-cell genomics
              </a>
            </td>
          <td>
            J. E. Powell
          </td>
          <td>2025-12-30</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb07356d2343226e0789b6d32493c4e60b77098" target='_blank'>
              Identification of 5' untranslated region variants in genes involved in neurodevelopmental disorders.
              </a>
            </td>
          <td>
            Taiju Hayashi, Sachiko Miyamoto, Yusaku Endo, Kenji Shimizu, Y. Ohkubo, Kazuyuki Komatsu, Shogo Furukawa, M. Nakashima, Tokiko Fukuda, Tsutomu Ogata, Takuya Hiraide, H. Saitsu
          </td>
          <td>2026-01-05</td>
          <td>Journal of human genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Most deaths from cancer are caused by their metastases. Along with the ability to evade the immune system and to invade and disrupt very different tissue environments, metastases usually develop resistance to whatever therapeutic approaches are tried. Their extreme adaptability requires a diversity of traits from which natural selection can choose, but current models, such as the epithelial-to-mesenchymal transition (EMT), do not directly address how this diversity is generated. We observed that single cell clones of Panc1 human pancreas cancer cells that had had crispR knock outs (KO) of the gene for activin-like kinase 4 (ALK4) had developed a markedly diverse morphology. Time-lapse and fluorescence microscopy, FACS analysis and mitotic chromosome squashes provided evidence that the cells diversify profoundly in size, behavior and chromosome number. This diversity appears to develop through cytokinesis failure, the generation of very large multinuclear cells that exhibit a high range of nuclear configurations, coupled with continued cell divisions that sometimes generate three, four, and more daughter cells in one cell division. This astounding activity often resulted in cells that were much smaller than the normal Panc1 cells, and indirect evidence suggests they were significantly sub-ploidy, yet underwent regular cell divisions. A subset of the smaller cells were highly motile, and were observed to sometimes merge together or even into larger cells. This phenomenon could provide an unappreciated venue for transferring mutated genes, chromosome fragments, whole chromosomes or sets of chromosomes into an invaded cell. We suggest that these observations may be a serendipitous illustration of the missing link between the genetic mutations implicated in the development of primary carcinoma, and the extreme genomic diversity that characterizes malignant metastases. Graphical Summary Single cell clones of Panc 1 ALK4 KO cultures changed from largely uniformly epithelioid in character (A) to cultures composed of an unexpectedly wide variety of cells, and that this variety is not explained by the traditional “EMT” (epithelial-to-mesenchymal-transition). These cells range from very small to very large, have varying degrees of motility, chromosome number, and cell division behavior. We describe the new morphologies and behaviors, and present time lapse movies and microscopic evidence that help explain the evolution of this variety. These new cell types integrate into colonies where the smallest cells gather into spherical masses (D). We believe that the heterogeneity in genomic content and behavior provides the variety needed for natural selection to select cells that are resistant to most treatments for metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6b16627321b67c2cf57a151fe0aa1bc2c09235" target='_blank'>
              A Model for the Diversity Explosion Fundamental to Metastasis
              </a>
            </td>
          <td>
            Edward Tim O’Brien
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Fluoropyrimidine anticancer agents, exemplified by 5-fluorouracil (5-FU), induce severe adverse effects-such as myelosuppression, emesis, diarrhea, and hand-foot syndrome-in approximately 10-30% of patients. As such toxicities can result in delays or discontinuation of therapy, accurate prediction of drug response prior to treatment initiation is of critical importance. The metabolic degradation of 5-FU is primarily mediated by the drug-metabolizing enzymes dihydropyrimidine dehydrogenase (DPD) and dihydropyrimidinase (DHPase). These enzymes are encoded by the DPYD and DPYS genes, respectively; polymorphisms in these genes that reduce or abolish enzymatic activity lead to elevated systemic concentrations of 5-FU, thereby increasing the risk of severe toxicity. In Caucasian populations, four DPYD polymorphisms have been identified as predictive markers of adverse drug reactions. However, these variants are rarely observed in Japanese individuals, and reliable pharmacogenomic biomarkers remain largely unreported in this population. To address this gap, we conducted a comprehensive in vitro functional analysis of DPD and DHPase activity of DPYD and DPYS variants identified through large-scale whole-genome sequencing databases. This review summarizes our findings and elucidates the underlying mechanisms affecting the function of 5-FU-metabolizing enzymes, as revealed by our prior research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42d05830affe95465f834cdd71289f75658fb239" target='_blank'>
              [Characterization of Genetic Polymorphisms Related to 5-FU Metabolizing Enzymes in Japanese Populations].
              </a>
            </td>
          <td>
            Eiji Hishinuma
          </td>
          <td>2026-01-01</td>
          <td>Yakugaku zasshi : Journal of the Pharmaceutical Society of Japan</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Micronuclei (MN) are small extranuclear chromosomal fragments that arise from genomic instability and serve as established biomarkers for genotoxicity and disease susceptibility. Once considered only markers of disease, they are now recognized as active, causative drivers of disease progression, driven by emerging evidence of epigenetic regulation within these structures. The micronucleus assay is recognized as a cost-effective and minimally invasive method for monitoring genotoxicity resulting from both chronic and early exposure to environmental factors such as arsenic and lead, as well as from genetic instability associated with cancer progression. This review critically examines the expanding role of MN beyond traditional cytogenetic endpoints, with particular emphasis on recent insights into their epigenetic landscape. Mass spectrometry-based studies have demonstrated that MN possess distinct histone posttranslational modification signatures compared to primary nuclei, including alterations in H3K27ac, H3K9ac, and H3K18ac. These modifications affect chromatin structure, gene expression, and DNA repair mechanisms. In the context of xenobiotic exposures, MN-associated epigenetic changes may function as early indicators of disease progression. Additionally, rupture of the MN envelope can activate innate immune responses through the cGAS-STING pathway or result in chromothripsis, both of which contribute to cancer progression. The concept of "Micronuclear Epigenetics" is highlighted, with its potential application in high-throughput diagnostic platforms, particularly liquid biopsy, discussed. This approach may enhance early detection and risk stratification in exposure-induced toxicity and diseases such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a963e7666be290f5a7f2a006e9788007c4d9584f" target='_blank'>
              Potential of Micronuclear Epigenetic Signatures in Analyses of Toxicity and Genomic Instability.
              </a>
            </td>
          <td>
            Somnath Paul, Ankita Das, A. Bandyopadhyay, Ashok K. Giri
          </td>
          <td>2026-01-27</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Expanded carrier screening (ECS) has emerged as a cornerstone of reproductive genetics, enabling the identification of couples at risk of transmitting autosomal recessive and X-linked disorders. Advances in next-generation sequencing and the increasing adoption of exome- and genome-based strategies have greatly expanded its clinical scope. However, despite existing national and international recommendations, substantial heterogeneity remains in gene panel composition, inclusion criteria, and interpretation frameworks. A key novelty of the current genomic era is the availability of large, publicly accessible human genome datasets encompassing broader ancestral diversity. These resources are transforming our understanding of variant frequencies and disease penetrance across populations, supporting the development of evidence-based and ancestry-informed gene panels. In parallel, recent studies investigating the genetic contribution of pathogenic variants to euploid pregnancy losses are opening new perspectives for ECS. Incorporating genes associated with lethal conditions in utero may enhance the predictive power of screening and deepen our understanding of reproductive outcomes, while also demanding careful consideration of clinical validity and counseling implications. This mini-review synthesizes current evidence on ECS, examines ongoing interpretive and implementation challenges, and discusses how population-scale genomic resources and emerging data on reproductive lethality are shaping the next generation of evidence-based carrier screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9fc3d491964f756b96075baec5b1227f73f9c1" target='_blank'>
              Expanded Carrier Screening: Current Evidence and Future Directions in the Era of Population Genomics
              </a>
            </td>
          <td>
            Lucrezia Pilenzi, Vincenzo Scorrano, Sara Di Rado, Carlotta Buccolini, Roberta Giansante, L. Siciliani, L. Stuppia, Valentina Gatta, A. Capalbo
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Two non-synonymous single-nucleotide polymorphisms (SNPs) rs61752561 (D95N substitution) and rs17632542 (I163T substitution) in the KLK3 gene encoding prostate-specific antigen (PSA), a chymotrypsin-like serine protease, are associated with prostate cancer risk and have been shown to reduce the activity of PSA. However, the structural impact of these SNPs on PSA, which may underlie the observed risk associations and functional alterations, has not been fully explored. Computational modelling predicted that the variants D95N and I163T do not cause drastic structural changes in PSA. However, molecular dynamics simulations suggested that while the two prominent loops of wild-type PSA remain tethered to their initial conformations over 500 ns of simulation, they are disrupted in both variants, leading to increased loop dynamics. Frustration analysis, normal mode analysis (NMA) and perturbation response scanning identified dynamic links between mutation sites and increased loop dynamics that trigger long-range conformational changes, disrupting the active site and potentially hindering catalytic activity. Thermal denaturation stability assays using recombinant protein show the impact of D95N and I163T substitution on the protein stability. These data show that KLK3 SNPs disrupt dynamic communication of the key loops required for proteolytic activity of PSA, which may explain the association of these SNPs with prostate cancer risk and/or progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387e91e1f338cbf07e62b9545288e8b8401fb288" target='_blank'>
              Prostate cancer risk-associated single-nucleotide polymorphisms impact the conformational dynamics of prostate-specific antigen
              </a>
            </td>
          <td>
            Srilakshmi Srinivasan, Brooke K Hayes, Mauricio G S Costa, Blake T Riley, Emily Wilson, Emilia M. Marijanovic, Itamar Kass, Hannu Koistinen, D. Hoke, Judith A. Clements, Ashley M Buckle, J. Batra
          </td>
          <td>2025-12-17</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Autoimmune diseases (AIDs) are chronic disorders driven by the breakdown of immune tolerance, where genetic susceptibility and environmental triggers converge to promote aberrant T and B cell activation, inflammatory cytokine release, and progressive tissue damage. Current therapeutic strategies, including corticosteroids, immunosuppressants, and biologics, can alleviate symptoms but remain limited by adverse effects, incomplete remission, and variable patient responses. CRISPR interference (CRISPRi), a novel gene regulation platform based on catalytically inactive Cas9 and guide RNAs, achieves precise, reversible suppression of target genes without inducing DNA double-strand breaks, thereby offering improved safety and controllability over conventional genome editing. Recent studies demonstrate that CRISPRi can effectively silence multiple inflammatory mediators and immune checkpoint molecules, enabling immune microenvironment remodeling and providing a feasible strategy for complex autoimmune conditions. Advances in multiplex CRISPRi, including the design of multi-sgRNA constructs and delivery via viral or non-viral systems, expand its capacity to simultaneously repress synergistic cytokine networks. Despite ongoing challenges in delivery efficiency, off-target risk, and long-term safety, CRISPRi holds considerable promise for network-level therapeutic intervention. This review aims to summarize the principles, technological progress, and potential applications of CRISPRi in autoimmune diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1138bca3d3877bd7932621af6651b3b14d329c9b" target='_blank'>
              Potential Applications and Research Exploration of CRISPRi in the Treatment of Autoimmune Diseases
              </a>
            </td>
          <td>
            Shaopu Huang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary The incidence of neuroendocrine neoplasms is increasing over the years, and therapy effectiveness is still relatively limited by the development of resistance mechanisms. Given the low rate mutation of these neoplasia, there is evidence of an emerging role of epigenetic modifications in neuroendocrine tumor pathogenesis and therapy resistance, leading to a need to investigate more deeply how these mechanisms are altered in these tumor types, how they can affect therapeutic efficacy, and if they can mediate the crosstalk between tumor cells, tumor microenvironment components, and tumor-stroma crosstalk that support tumoral development and progression. In this review we aim to describe all the recent evidence involving epigenetic mechanisms and the tumor microenvironment components that support tumor onset, development and resistance to treatment in the context of neuroendocrine neoplasm. The goal is to highlight potential new path to improve treatment response and outcomes in patients affected by neuroendocrine neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a66ba50c801aa6fd2102a018fc8123d71e68a47" target='_blank'>
              Epigenetics and the Tumor Microenvironment in Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Alice Castenetto, Teresa Gagliano
          </td>
          <td>2025-12-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0b876781bf210db756c35db260709c192ea56b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A McIntyre, Akimasa Hayashi, Nicolas Lecomte, Marc Hilmi, Wungki Park, Nan Pang, Eileen M. O’Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-01-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92529c21b1d903977db7b6f99cc016a10cc689ad" target='_blank'>
              Molecular analysis of tumor recurrence using established cancer stem cell-line and drug discovery.
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, Y. Zaitsu, Chiyoko Nishime, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, T. Kawazoe, Kentaro Nonaka, Keita Natsugoe, Sachie Omori, H. Saeki, Hiroyuki Aburatani, Tatsumi Yamazaki, Yoshihiko Maehara
          </td>
          <td>2026-01-20</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Immune dysregulation arises from a complex interplay between genetic predisposition, environmental exposures, and dynamic epigenetic modifications that collectively shape immune cell development, signaling, and responsiveness. Over the past two decades, rapid advances in genomics and epigenomics have elucidated key mechanisms by which inherited variants, somatic mutations, DNA methylation, histone modifications, microRNAs, and chromatin remodeling contribute to abnormal immune activation or suppression. These mechanisms underlie a wide spectrum of disorders, including autoimmunity, chronic inflammation, immunodeficiency, cancer-associated immune escape, and aberrant responses to infection. This review synthesizes current knowledge on major genetic and epigenetic determinants of immune dysregulation, highlighting the molecular pathways involved, their clinical significance, and the interplay between developmental and environmental influences. Emerging insights into gene-environment interactions, including the effects of diet, stress, xenobiotics, and the microbiome on immune epigenetics, reveal new therapeutic opportunities. Understanding how diverse layers of regulation converge on immune homeostasis provides a foundation for precision medicine approaches aimed at diagnosing, predicting, and treating diseases associated with immune imbalance.

Keywords: Immune dysregulation, genetic variants, epigenetic regulation, DNA methylation, microRNAs">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe02e272c25cf60f72783170e93ac5975c2b37f8" target='_blank'>
              Genetic and Epigenetic Modifiers of Immune Dysregulation: A Comprehensive Review
              </a>
            </td>
          <td>
            Bwanbale Geoffrey David
          </td>
          <td>2026-01-07</td>
          <td>IDOSR JOURNAL OF SCIENCE AND TECHNOLOGY</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ade226a59a479c368236a01782e76d70485a45e" target='_blank'>
              Overcoming barriers in CAR-NK immunotherapy: CRISPR-Driven advances in checkpoint editing and allogeneic design
              </a>
            </td>
          <td>
            Mohammed J Mansoor, Sarah F. Al-Taie, Z. Al-Khafaji, Ali G. Alkhathami, Jyothi S Renuka, Rajashree Panigrahi, Hansi Negi, Pushkar Jassal, Yasser Fakri Mustafa, Hamza Fadhel Hamzah
          </td>
          <td>2025-12-17</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has raised considerable health concerns worldwide, with increasing incidence rates, specifically among younger populations. Despite remarkable progress in diagnosing and treating various diseases, the genetic basis of CRC remains only partially understood. This paper aims to examine novel genetic variants associated with CRC in Iranian patients. We performed whole-exome sequencing (WES) in two Iranian families with a history of early-onset CRC. Candidate variants were validated by Sanger sequencing and assessed for segregation. Pathogenicity was evaluated through comprehensive in silico analysis and the application of ACMG/AMP guidelines. Analysis revealed two clinically significant variants. In Family 1, we identified a heterozygous stop-gain variant in MLH1 (c.1043T>A, p.Leu348), a known pathogenic mutation consistent with Lynch syndrome. In Family 2, we discovered a previously undocumented heterozygous missense variant in ERBB2 (c.2268G>T, p.Arg756Ser). Through a detailed ACMG assessment, this ERBB2 variant was classified as Likely Pathogenic based on its location in a critical tyrosine kinase domain, absence from population databases, and concordant deleterious in silico predictions. WES offers a deeper understanding of CRC genetics, suggesting potential biomarkers with promising applications for early diagnosis and targeted treatments, eventually improving patient-related outcomes. The results of this study underscore the significant contribution of genetic screening to the well-being of high-risk families and offer valuable insights for targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24298c3854c9017f625e9688093a76a2972d11f" target='_blank'>
              Exome Sequencing Identifies Variants in MLH1 and ERBB2 as Potential Cancer-Predisposing Factors in Familial Early-Onset Colorectal Cancer.
              </a>
            </td>
          <td>
            Behnaz Bagheri, N. Mohsen-Pour, Najmeh Salehi, P. K. Moghadam, Adel Zeinalpour, Amir Sadeghi, Samira Kalayinia, N. Fatemi
          </td>
          <td>2026-01-02</td>
          <td>The Kaohsiung journal of medical sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rituximab has profoundly improved outcomes for patients with Diffuse large B-cell lymphoma, yet the development of drug resistance continues to pose a serious clinical challenge, leading to treatment failure in 30–40% of cases. Previous studies have revealed several key resistance mechanisms underlying rituximab resistance, such as CD20 antigen expression and conformation, functional impairments in immune effector mechanisms such as antibody-dependent cellular cytotoxicity (ADCC) and complement-dependent cytotoxicity (CDC), the establishment of an immunosuppressive tumor microenvironment (TME), and the activation of intrinsic tumor cell survival pathways. Despite these advances, the resistance landscape of rituximab in diffuse large B-cell lymphoma is not fully mapped; for instance, the roles of ferroptosis and non-coding RNAs remain to be elucidated. This review will summarize these mechanisms to help provide insights that will aid in the development of novel therapeutic strategies and facilitate more personalized approaches to overcome resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/463291dad96bbffedffd98a9639e5b0b55368969" target='_blank'>
              Unraveling the Landscape of Rituximab Resistance in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            Xiaoting Liao, Ting Wei, Qingshan Li
          </td>
          <td>2025-12-25</td>
          <td>International Journal of Biology and Life Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colitis-associated colorectal cancer (CAC) represents a distinct subtype of colorectal malignancy that arises in the setting of chronic inflammatory bowel disease (IBD). Unlike sporadic colorectal cancer, CAC develops through inflammation-driven molecular pathways, in which epigenetic alterations play a pivotal role in tumor initiation and progression. This review highlights the major epigenetic mechanisms implicated in CAC, including DNA methylation, histone modifications, and microRNA (miRNA) dysregulation. Aberrant DNA methylation patterns, such as promoter hypermethylation of tumor suppressor genes and global hypomethylation, contribute to genomic instability and altered gene expression. In parallel, inflammation-induced changes in histone configuration modulate chromatin accessibility and transcriptional activity of key oncogenic and tumor-suppressive pathways. Furthermore, deregulated miRNAs influence multiple aspects of CAC pathogenesis by targeting genes involved in inflammation and tumor progression. Understanding these epigenetic processes provides valuable insights into the development of colorectal malignancy and identifies potential biomarkers for early detection and intervention in colitis-associated colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a48d72d8d5159f1dafbf9dad8f63ec3ff7ed8efe" target='_blank'>
              Epigenetic Alterations in Colitis-Associated Colorectal Cancer
              </a>
            </td>
          <td>
            N. Ganesh, William M. Grady, A. Kaz
          </td>
          <td>2026-01-16</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="

 Imatinib mesylate is a small molecule inhibitor targeting the BCR‐ABL tyrosine kinase. However, some CML patients develop resistance to imatinib. This resistance is commonly associated with mutations in the kinase domain of BCR‐ABL, notably the prevalent T315I and E255K mutations. We evaluated the frequency of E255K and T315I mutations in CML patients who do not respond to imatinib therapy.



 Twenty imatinib‐resistant CML patients were selected from a total cohort of 100 CML patients undergoing regular follow‐up. Genomic screening for the T315I and E255K mutations was carried out using a two‐step molecular approach: initial reverse transcription‐PCR (RT‐PCR) amplification of the ABL kinase domain, followed by restriction fragment length polymorphism (RFLP) analysis. The subsequent clinical management and outcomes for patients harboring these mutations were meticulously tracked and analyzed.



 The T315I and E255K mutations were present in 20% (4/20) and 10% (2/20) of the imatinib‐resistant patients, respectively. All patients required an immediate switch to second‐line tyrosine kinase inhibitors. Notably, one patient with the T315I mutation secured ponatinib and achieved remission, while three other patients underwent allogeneic stem cell transplantation. Two patients carrying the E255K mutation ultimately progressed to blast crisis and died, highlighting the particularly dire prognosis associated with this mutation.



 The T315I and E255K mutations are significant contributors to imatinib resistance in our patient population and are linked to an aggressive clinical course. These findings underscore the critical importance of integrating routine mutation screening into clinical practice to enable early treatment modification, facilitate access to advanced therapies, and ultimately improve patient outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec7df8b8e8284c1381d55698dc70e162de37d23" target='_blank'>
              Genetic Analysis of Imatinib Resistance in CML: The Role of T315I and E255K
              </a>
            </td>
          <td>
            Saba Yari, Reza Masoomi Jahandizi, M. Payandeh, Kimiya Dadashizadeh
          </td>
          <td>2026-01-01</td>
          <td>Advances in Hematology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="A large number of reports have suggested that the cancer is associated with the gene mutations and chromosomal aberrations, and thus it has been a focus of research. Today new insights are proposed by analyzing relevant literature, suggesting that the cancer is not caused by the gene mutations and chromosomal aberrations, but by the regulatory dysfunction of gene expression resulted from the nuclear damage, among which, the gene mutations and chromosomal aberrations are only an accompany phenomena also resulting from the nuclear damage. The gene mutations generally do not cause diseases, but if they do, they mainly cause the single-gene diseases. The chromosomal aberrations usually refer to the abnormalities in chromosome structure and number, often leading to cell death or inducing various chromosomal diseases. Some chronic refractory diseases, such as the cancer, Alzheimer's disease, degenerative diseases, are probably not caused by the gene mutations and chromosomal abnormalities, but rather by the abnormal regulation of gene expression and the dysfunction of DNA transcription-protein synthesis resulted from the nuclear damage. These diseases are all the diseases of nuclear dysfunction and collapse of differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4826bbb55a85cfdd2750ad0771ebe3b7d95aef3d" target='_blank'>
              Differences and Pathogenicity Between the Gene Mutation and Chromosomal Aberration
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-12-29</td>
          <td>Medical Research Frontiers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Sickle cell disease (SCD) and β-thalassemia are autosomal recessive disorders of erythroid cells due to gene mutations occurring at the level of the β-globin gene. The severe forms of these hemoglobinopathies observed in individuals homozygous for these defective genes need intensive treatments, are associated with a poor quality of life, and allogeneic hematopoietic stem cell represents the only curative treatment option that can be offered to a limited proportion of patients. Methods: This work is a narrative review supported by a systematic literature search and analysis. Results: To bypass this limitation, autologous hematopoietic stem cell transplantation has been developed in these patients, in which patients’ HSCs are harvested and genetically modified ex vivo, then transplanted back into patients after conditioning for stem cell transplantation. There are two different approaches for gene therapy of hemoglobinopathies, one based on gene addition or gene silencing using lentiviruses as vectors and the other based on gene editing strategies using CRISPR-Caspase 9 technology or base editing. Several gene therapy products have been successfully evaluated in these patients, achieving transfusion independence and correction of hematological abnormalities durable over time. Conclusions: Several gene therapy products have been approved for the treatment of SCD and β-thalassemic patients and offer potentially curative treatment for these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28863dbc2a8942bf5d2a11710259a99e9f3bf5dc" target='_blank'>
              Gene Therapy of Beta Hemoglobinopathies
              </a>
            </td>
          <td>
            Ugo Testa, E. Pelosi, Germana Castelli
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Sickle cell anemia represents a cornerstone in molecular medicine, where a single gene mutation gives rise to complex pathophysiological cascades encompassing hemolysis, vaso-occlusion, and organ dysfunction. Although conventional therapies such as hydroxyurea and blood transfusions have improved survival, they remain largely palliative. The advent of molecular diagnostics and therapeutic innovation has transformed the landscape of SCA care, ushering in an era of precision medicine. This narrative review explores the expanding repertoire of emergingbiomarkers—including inflammatory mediators, oxidative stress markers, and genetic modifiers—that provide insights into disease heterogeneity and therapeutic responsiveness. Concurrently, moleculartherapies, ranging from gene addition and CRISPR-based gene editing to RNA-targeted interventions and pharmacologic fetal hemoglobin inducers, are redefining curative potential. These advancements signal a shift from symptomatic management toward molecular correction and individualized treatment. Integrating biomarker discovery with gene-directed therapies holds the promise of achieving long-term remission and functional cure, positioning SCA at the forefront of translational hematology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d582de2f5097a85f5ccdea63221a5417404ee855" target='_blank'>
              EMERGING BIOMARKERS AND MOLECULAR THERAPIES: THE NEXT FRONTIER IN SICKLE CELL ANEMIA CARE
              </a>
            </td>
          <td>

          </td>
          <td>2026-01-01</td>
          <td>Journal of Bio Innovation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e3adf53024be934b16593cb6c83e94395958adb" target='_blank'>
              System-level regulation of hierarchical transitions in a tumour lineage
              </a>
            </td>
          <td>
            Emma Legait, Charlotte Rulquin, Lauranne Bouteille, Raphaël Clément, C. Maurange
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a prevalent and aggressive malignancy with a persistently high mortality rate, largely attributable to therapy resistance and tumor recurrence. This review comprehensively explores the critical interplay between epigenetic dysregulation and the tumor microenvironment (TME) in driving OSCC progression. We detail how key epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNAs (ncRNAs), intrinsically transform cancer cells and actively orchestrate pro-tumorigenic TME. These alterations substantially contribute to resistance against conventional therapies. Furthermore, we discuss the therapeutic potential of targeting these pathways using epigenetic drugs (epi-drugs), such as DNA methyltransferase (DNMT) inhibitors and histone deacetylase (HDAC) inhibitors, as well as engineered extracellular vesicles (EVs). The primary objective of this review is to synthesize current knowledge on the epigenetic-TME axis, thereby providing a mechanistic foundation for developing novel therapeutic strategies. We emphasize that rational combinations of epigenetic-targeting agents with conventional treatments or immunotherapy hold significant promise for overcoming drug resistance and improving clinical outcomes in OSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aaa3d5aaab2d3886a4c338f86161bbd9cbefef" target='_blank'>
              Dissecting the role of epigenetic regulation in oral squamous cell carcinoma microenvironment: mechanisms and therapeutics
              </a>
            </td>
          <td>
            Xuechao Li, Yifei Ren, Shenghua Pei, Kai Zhao, Guanyu Chen, Zhenglin He
          </td>
          <td>2026-01-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Transdifferentiation, also known as direct reprogramming, is the transformation of one terminally differentiated cell type into another mature cell type, while bypassing the stage of pluripotency. In leukemia, this phenomenon has a dual significance: on the one hand, it is an adaptive mechanism driving tumor survival and resistance to treatment, and on the other, it offers a potential opportunity for innovative therapies. Of particular interest is the directional transdifferentiation (mostly partial) toward dendritic cell-like phenotypes, which increases the immunogenicity of cancer cells. Mastering this process could define a new generation of immunotherapies that leverage the inherent plasticity of leukemic cells to achieve therapeutic benefits. In this brief review, we attempt to gather information concerning the molecular mechanism of this process and point to the role of dendritic cells as a crucial element of anticancer, particularly anti-leukemia innate and acquired, immunity. Thus, in vitro and in vivo techniques of inducing transformation of the leukemia cells into cancer antigen-presenting cells and the application of these technologies in current and future therapies are discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4860b12b5b1e05e4c6dca4a179939d6e410810" target='_blank'>
              Leukemic Transdifferentiation: From Pathological Plasticity to Dendritic Cell-Based Immunotherapy
              </a>
            </td>
          <td>
            Joanna Dubis, Aleksander Czogalla, Kazimierz Kuliczkowski, Aleksander F. Sikorski
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Epigenetic editing has emerged as a promising approach in the treatment of neurological and neuropsychiatric disorders, enabling the precise and enduring modification of genes associated with these conditions. Interventions that focus on chromatin, such as programmable systems like CRISPR/dCas9, zinc-finger proteins, and TALEs linked to epigenetic effector domains, enable the modification of DNA methylation, histone modifications, and noncoding RNA control at specific loci. This work integrates current progress in understanding the epigenetic landscape of neurological neuropsychiatric disorders, highlighting the functions of DNA methylation (de novo vs maintenance, active versus passive demethylation), histone remodeling, and context-dependent gene regulation. We emphasize that the dysregulation of these processes is essential to diseases such as Alzheimer’s disease (AD), Parkinson’s disease (PD), Huntington’s disease (HD), and major psychiatric disorders. Innovative therapeutic approaches, including KRAB- and TET-based repressors, “hit-and-run” epigenome editing, and targeted noncoding RNA regulation, are analyzed alongside translational methodologies that utilize gene therapy vectors, nanoparticle delivery systems, and inducible expression mechanisms. We also examine proof-of-concept studies that demonstrate how to prevent gene expression and alter the transcriptional networks of diseased cells in living organisms. We identify current challenges, including off-target effects, delivery issues, inadequate understanding of long-term stability, and the need for reliable diagnostics, while highlighting the translational promise of combining epigenetic clearance with biogenesis and repair. This review is aimed at providing a comprehensive and critical examination of the molecular principles, therapeutic strategies, and translational obstacles associated with epigenetic editing in neurological and neuropsychiatric disorders, thereby facilitating the development of next-generation precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8da6771a9dde7d1eca945df355bc0ba5bae7ba5" target='_blank'>
              Epigenetic Editing in Neurological and Neuropsychiatric Disorders: Pioneering Next-Gen Therapeutics for Precision Gene Control
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-12-28</td>
          <td>Molecular Neurobiology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Cystic fibrosis is an autosomal recessive genetic disorder caused by mutations in the cystic fibrosis transmembrane conductance regulator (CFTR) gene, leading to CFTR dysfunction and impaired ion transport, which subsequently results in progressive dysfunction of the digestive and respiratory systems and reduced life expectancy. Although current therapies, including airway clearance techniques, antibiotics, pancreatic enzyme replacement therapy, CFTR modulators, and RNA-based approaches, have significantly improved survival and the quality of life, most of these therapies primarily focus on symptom management and are limited by genotype specificity, transient efficacy, and the inability to permanently correct the underlying genetic defect. As a result, patients may require lifelong treatment. Genome engineering technologies appeared as promising causative therapeutic strategies to address cystic fibrosis by directly repairing or replacing defective CFTR alleles. Recently, the latest progress in base editing, prime editing, and gene addition technology has successfully repaired the CFTR function of common and rare mutations in patient-derived airway epithelial cells and organoids, highlighting the potential for long-lasting and mutation-specific repair. However, significant challenges remain, including the effective delivery to airway stem cells and progenitor cells, achieving sufficient in vivo editing efficiency, minimizing the off-target effects, as well as solving ethical, regulatory, and manufacturing barriers. Overall, genome editing provides a potentially transformative method for the treatment of cystic fibrosis, which is expected to achieve long-term disease correction and surpass lifelong symptomatic treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ccac98c3a901be38223edfc2eedb667b1410b7f" target='_blank'>
              Applications and Challenges of Genome Engineering Technologies in the Treatment of Cystic Fibrosis
              </a>
            </td>
          <td>
            Ching Li
          </td>
          <td>2026-01-20</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Accurate detection of gene fusions is critical for the diagnosis and management of sarcomas. However, occasional false-negative results may occur due to factors such as low tumor cellularity, limited assay coverage of relevant genomic regions, or mis-annotation by bioinformatic pipelines. We recently encountered a case in which our clinical SARCP fusion detection assay failed to identify an HMGA2::NCOR2 (exon 3–exon 16) fusion.



 The SARCP assay is a targeted, amplicon-based RNA-seq next-generation sequencing (NGS) assay designed to detect gene fusions across 138 genes, including HMGA2. Although NCOR2 is not among the targeted genes, the assay is capable of detecting fusions when only one gene partner is represented.



 Suspicion of a false-negative result arose when the pathologist noted that the tumor’s histologic features—keratin-positive, giant cell-rich soft tissue—were consistent with a subtype previously associated with the HMGA2::NCOR2 fusion. This prompted a review of the bioinformatic output, which had erroneously reported an HMGA2::MAML2 fusion. The misannotation was traced to sequence homology involving a CAA/CAG repeat at the 5′ end of NCOR2 exon 16, likely resulting from DNA polymerase slippage and misalignment in low-complexity, partially complementary repetitive regions. This erroneously lead to the target genomic sequence getting mapped to MAML2 exon 2 as opposed to the NCOR2 exon 16.



 This case highlights how bioinformatic mis-annotation can lead to occasional false-negative fusion calls. It underscores the importance of manual review in cases with strong clinical or pathologic suspicion and calls attention to the need for fusion-calling algorithms that better handle repetitive and homopolymeric sequences.



 Ganesh P. Pujari, Mitch L. Klebig, Kurt S. Willkom, Montha Chhunn, Stefan W. Nelson, Neiladri Saha, Kevin C. Halling. Mis-Annotation of an HMGA2::NCOR2 Fusion in FFPE Sarcoma Samples: A Case Illustrating Bioinformatic Limitations in Fusion Detection [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b640522fda2138d045848d46734cd774a873c147" target='_blank'>
              Abstract A012: Mis-Annotation of an HMGA2::NCOR2 Fusion in FFPE Sarcoma Samples: A Case Illustrating Bioinformatic Limitations in Fusion Detection
              </a>
            </td>
          <td>
            Ganesh P. Pujari, Mitch L. Klebig, Kurt S. Willkom, Montha Chhunn, Stefan W. Nelson, Neiladri Saha, K.C. Halling
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="WHIM syndrome is an immunodeficiency caused by autosomal dominant hyperfunctional mutations in chemokine receptor CXCR4 that promote panleukopenia due to BM retention. We previously reported a preclinical gene therapy protocol involving allele-nonspecific Cxcr4 CRISPR/Cas9 inactivation, leveraging the known in vivo dominance of Cxcr4+/o (+, WT; o, inactivated) hematopoietic stem cells (HSCs) for autologous BM engraftment and leukocyte reconstitution over HSCs with other Cxcr4 genotypes. Here, we show that without BM conditioning, this approach is not able to correct leukopenia in WHIM mice. We therefore modified the protocol by adding conditioning with a non-genotoxic CD117-targeted immunotoxin, CD117-antibody-saporin-conjugate (CD117-ASC). With this change, donor-derived blood cells rapidly reached ~95% chimerism after transplantation, which was stable without adverse events for more than 400 days. Mice receiving edited HSCs showed rapid normalization of absolute myeloid cell counts, the key blood subset responsible for WHIM syndrome. In competitive transplants using equal numbers of edited and unedited donor HSCs, over 80% of blood cells originated from the edited population, predominantly with the Cxcr4+/o genotype. These results provide proof of principle that CRISPR/Cas9-mediated inactivation of the Cxcr4 disease allele, combined with non-genotoxic HSC-targeted conditioning, may offer a safe and effective gene therapy strategy generalizable to all WHIM mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4ee1a529fa02ac6fcb74ceeb332ab0f6c4ee8bf" target='_blank'>
              Gene therapy via CRISPR/Cas9-mediated Cxcr4 disease allele inactivation reverses leukopenia in WHIM mice.
              </a>
            </td>
          <td>
            Ji-Liang Gao, Zhanzhuo Li, Rafael Calderon-Perez, Antonia Pavek, Lina Kim, David H. McDermott, Philip M. Murphy
          </td>
          <td>2026-01-08</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The occurrence and progression of cancer are generally considered to result from the accumulation of driver gene mutations. Accurately identifying these genes is crucial for elucidating tumor mechanisms and developing targeted therapies. We propose IMDG, a method for identifying cancer driver genes based on modular dysregulated gene networks that integrates multi-layer scoring. IMDG first constructs a weighted proteinprotein interaction (PPI) network based on the gene-miRNA network. It then extracts features using node embedding to build a relationship network for dysregulated genes and scores genes based on the association between potential driver genes and modules of the dysregulated gene network. Subsequently, it reassesses gene mutations through random walks on the weighted PPI network and integrates network connectivity to derive the final gene scores. We compared IMDG with six existing driver gene prioritization methods on two real-world cancer datasets. Results demonstrate that IMDG exhibits optimal identification performance in most cases. Genes prioritized by IMDG not only show higher concordance with benchmark databases compared to those identified by other methods, but some also exhibit strong relevance to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0fbd1a76102b82b9bbd69990021a208bee8001b" target='_blank'>
              Identification of Potential Cancer Driver Genes Based on Modular Dysregulated Genes
              </a>
            </td>
          <td>
            Zheng Deng, Jingli Wu, Xiaorong Chen, Gaoshi Li
          </td>
          <td>2025-12-15</td>
          <td>2025 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="This review aims to provide a broad, multidisciplinary perspective on how dynamic genomics and systems biology are transforming modern healthcare, with a focus on cancer especially liver cancer (HCC). It explains how integrating multi-omics technologies such as genomics, transcriptomics, proteomics, interactomics, metabolomics, and spatial transcriptomics deepens our understanding of the complex tumor environment. These innovations enable precise patient stratification based on molecular, spatial, and functional tumor characteristics, allowing for personalized treatment plans. Emphasizing the role of regulatory networks and cell-specific pathways, the review shows how mapping these networks using multi-omics data can predict resistance, identify therapeutic targets, and aid in the development of targeted therapies. The approach shifts from standard, uniform treatments to flexible, real-time strategies guided by technologies such as liquid biopsies and wearable biosensors. A case study showcases the benefits of personalized therapy, which integrates epigenetic modifications, checkpoint inhibitors, and ongoing multi-omics monitoring in a patient with HCC. Future innovations, such as cloud-based genomic ecosystems, federated learning for privacy, and AI-driven data analysis, are also discussed to enhance decision-making and outcomes. The review underscores a move toward predictive and preventive healthcare by integrating layered data into clinical workflows. It reviews ongoing clinical trials using advanced molecular and immunological techniques for HCC. Overall, it promotes a systemic, technological, and spatial approach to cancer treatment, emphasizing the importance of experimental, biochemical–functional, and biophysical data-driven insights in personalizing medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36072da0d316cc3d93e645a709d89a9b90398c65" target='_blank'>
              Advancing Liver Cancer Treatment Through Dynamic Genomics and Systems Biology: A Path Toward Personalized Oncology
              </a>
            </td>
          <td>
            Giovanni Colonna
          </td>
          <td>2026-01-21</td>
          <td>DNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The Tracking Cancer Evolution Through Therapy (TRACERx) program represents the most comprehensive effort to characterize tumor evolution in real time. Through longitudinal, multiregion, and multiomic profiling of tumors—and particularly of non-small-cell lung cancer and clear cell renal cell carcinoma—TRACERx has illuminated the dynamic interplay between genetic, nongenetic, and (micro)environmental factors that drive cancer progression, immune evasion, and therapeutic resistance. A central insight from TRACERx has been that not all tumor evolution is genomic: Transcriptomic diversity, epigenetic alterations, RNA editing, and changes in cell–cell interactions also drive adaptation. Methodological innovations—including tumor-informed and ultrasensitive circulating tumor DNA assays, representative sequencing, and integrative immune–genomic analyses—have yielded biomarkers resistant to sampling bias and/or predictive of recurrence, metastasis, and treatment response. By demonstrating that intratumor heterogeneity is a key determinant of clinical outcome and revealing its molecular, transcriptional, and ecosystem-level drivers, TRACERx has established a framework for linking evolutionary dynamics to patient care. As both a scientific framework and a clinical paradigm, TRACERx demonstrates how adaptive, iterative research can refine evolutionary models, improve patient risk stratification, and inspire next-generation cancer evolution studies across malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a59431b9259c4928178579f6c460cf8e369f4f1" target='_blank'>
              TRACERx as an Optimized Paradigm for Understanding Cancer Evolution
              </a>
            </td>
          <td>
            Naomi Iris van den Berg, C. Swanton, S. Turajlic
          </td>
          <td>2025-12-24</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Simple Summary Cancer treatment is increasingly guided by tumor DNA alterations. Changes in fibroblast growth factor receptors, a family of four related proteins, occur across many solid tumors and drive growth, spread, and treatment response by activating multiple signaling pathways. This review examines how these receptor changes—amplifications, mutations, and fusions—differ in their mechanisms and distribution across cancer types, and how they reshape the tumor environment, blood supply, and immune landscape. We summarize the clinical benefits and limitations of approved targeted drugs, explain why resistance develops through secondary mutations or alternative signaling routes, and describe how blood-based tests may detect resistance early. We also discuss next-generation strategies including combination with immunotherapy or anti-vessel therapy, antibody–drug conjugates, and nanotechnology-based delivery systems. By integrating molecular, translational, and clinical evidence, this review aims to guide patient selection, treatment monitoring, and the design of durable precision therapies targeting these receptor abnormalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59bbdc0ef7fd9f5e7a981804b0e4e8f11594434a" target='_blank'>
              FGFR Aberrations in Solid Tumors: Mechanistic Insights and Clinical Translation of Targeted Therapies
              </a>
            </td>
          <td>
            Zijie He, Yizhen Chen, Genglin Li, Jintao Wang, Yuxin Wang, Pengjie Tu, Yangyun Huang, Lilan Zhao, Xiaojie Pan, Hengrui Liu, Wenshu Chen
          </td>
          <td>2025-12-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 225


 Background:
 Colorectal cancer (CRC) is the second leading cause of cancer-related deaths in the United States. Although immune checkpoint blockade benefits ~15% of patients with high mutational burden and microsatellite instability, the majority of CRC remains refractory, with cytotoxic chemotherapy as the standard of care. Identifying mechanisms that sustain cancer cell survival and proliferation is therefore critical to uncovering vulnerabilities that may enhance therapeutic efficacy. The complement peptide C5a, a potent effector generated by proteolytic activation of the complement cascade, signals through its receptor C5AR1 to drive cell survival, cytokine production, and chemotaxis. While historically regarded as a systemic pathway, recent evidence suggests that complement activation can also occur locally. We hypothesized that CRC tumors locally generate C5a, with cancer cell-intrinsic C5AR1 driving survival and chemoresistance.
 Methods:
 CRC tissue microarrays (TMAs) were generated from de-identified specimens (Hackensack Meridian
 Health
 ,
 N
 = 28; commercial TMA,
 N
 = 17). Preliminary spatial transcriptomics was performed on the commercial TMA using the Nanostring CosMx Spatial Molecular Imager. Publicly available single-cell transcriptomes from two independent CRC cohorts (
 N
 = 63 and
 N
 = 62) were reanalyzed. Multiplexed immunofluorescence (24-plex Cyclic IF) was applied to detect complement fragments (C3, C3b/iC3b, C5a) and C5AR1, and to map their cell type-specific expression in situ. Cancer cell-intrinsic C5AR1 expression was validated in two human CRC cell lines and four patient-derived organoids. Functional relevance of C5AR1 was assessed using in vitro cytotoxicity assays combining chemotherapy with pharmacologic C5AR1 inhibition.
 Results:
 Spatial transcriptomics revealed induction of complement pathway transcripts in CRC tumors, further confirmed by reanalysis of two independent CRC single-cell datasets. Multiplexed CycIF revealed an enriched localization of complement ligands (C3, C3b/iC3b, C5a) to tumor-adjacent regions, whereas C5AR1 was enriched in the tumor core. Furthermore, CycIF demonstrated robust C5AR1 protein expression in cytokeratin-positive cancer cells and infiltrating myeloid phagocytes; cancer cell-specific C5AR1 expression was further validated in two human CRC cell-lines and patient-derived organoids. Functionally, cancer cell-intrinsic C5AR1 proved essential for survival, and its pharmacologic inhibition synergistically enhanced 5-fluorouracil cytotoxicity.
 Conclusions:
 Complement activation is spatially organized in CRC, with cancer cell-intrinsic C5AR1 supporting survival. Pharmacologic C5AR1 inhibition synergizes with chemotherapy, positioning the C5a-C5AR1 axis as a tractable therapeutic vulnerability. Ongoing studies aim to define how C5AR1 supports cancer cell survival and mediates chemoresistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a950393c39eca04ec13c0ab1c5dcbbbb94f802" target='_blank'>
              Role of cell-intrinsic C5AR1 in colorectal cancer survival and chemoresistance.
              </a>
            </td>
          <td>
            Ashley Carlo, Raju Shivarathri, Adela Karuli, Richa Mandrekar, Anastasia Zenkevich, Chen-Hua Ma, Kar Chow, Alvin Makohon-Moore, Kevin Tong, Jigar V. Desai
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Due to the non-renewable nature of central nervous system cells, genetic heterogeneity and the mechanism of dominant negative mutations, neurogenetic diseases have long lacked effective radical treatment methods. Traditional gene editing techniques such as CRISPR-Cas9 rely on DNA double-strand breaks (DSB), which are inefficient and pose safety risks in non-dividing neurons. Prime Editing (PE), as a disruptive "search - replace" type of precise gene editing technology, realizes all 12 base conversions, small fragment insertions and deletions without introducing DSB, bringing unprecedented opportunities for the treatment of neurogenetic diseases. This article will elaborate on the technical principles of PE and its unique advantages in the treatment of neurogenetic diseases by combining experimental research and literature reports in recent years. It will focus on summarizing the latest breakthrough progress of PE in the preclinical research of alternating hemiplegia (AHC) in children, especially its symptom reversal and functional recovery ability demonstrated in in vivo treatment. Meanwhile, this article delves deeply into the core challenges faced by the clinical transformation of PE, such as in vivo delivery efficiency and interference with mismatch repair (MMR) systems, and looks forward to the future directions of addressing these challenges through carrier engineering innovation. PE technology is expected to drive the transformation of neurogenetic diseases from symptomatic treatment to one-time radical cure in a new era.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad791c11a8114d313fbf81479901314fde72aab" target='_blank'>
              The Progress of Prime Editing in The Treatment of Neurogenetic Diseases
              </a>
            </td>
          <td>
            Xuannan Zhuang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated stromal cells (TASCs) are key architects of the tumor microenvironment (TME), playing a vital role in tumor development, metastasis, and therapeutic response. Their spatiotemporal heterogeneity, characterized by dynamic phenotypic plasticity, diverse cellular subtypes, and distinct spatial distributions, offers profound insights into tumor behavior and paves the way for innovative therapy development. In particular, stromal–immune interactions reveal the powerful capacity of TASCs to shape the immune landscape, highlighting their potential as targets in immunotherapy. Despite growing evidence in functional diversity, precise mechanisms underlying the temporal evolution and spatial organization of TASCs remain elusive, impeding clinical translation. This review delved into the molecular signatures and functional states of TASCs, emphasizing their roles in tumor dynamics and therapeutic resistance. We also discussed innovative strategies targeting the plasticity of TASCs to reverse immune evasion and potentiate immune-mediated tumor eradication. Future studies should prioritize identifying spatially resolved and mechanically defined biomarkers with multi-omics and machine learning approaches, enabling a comprehensive understanding of TASCs to bridge the gap from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc16bf824dc473946d5d3958736301da12edf085" target='_blank'>
              The Spatiotemporal Heterogeneity of Tumor-Associated Stromal Cells: Reprogramming Plasticity to Unlock Precision Cancer Immunotherapy
              </a>
            </td>
          <td>
            Yingying Lv, Tingfei Duan, Jinling Song, Shutong Liu, Zhaokai Zhou, Yuhao Ba, Siyuan Weng, Anning Zuo, Hui-Xian Xu, Peng Luo, Quan Cheng, Chuhan Zhang, Jingyuan Ning, Yukang Chen, Yuyuan Zhang, Zaoqu Liu, Xinwei Han
          </td>
          <td>2025-12-12</td>
          <td>Cancer Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [9, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>