<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-21 06:33:12 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>8</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41df9adbcb86e982509871382b3eba588480f4fb" target='_blank'>
              Base editing screens define the genetic landscape of cancer drug resistance mechanisms.
              </a>
            </td>
          <td>
            Matthew A Coelho, Magdalena E Strauss, Alex Watterson, Sarah Cooper, S. Bhosle, G. Illuzzi, Emre Karakoc, Cansu Dincer, Sara F Vieira, Mamta Sharma, Marie Moullet, Daniela Conticelli, Jonas Koeppel, Katrina McCarten, C. M. Cattaneo, Vivien Veninga, Gabriele Picco, Leopold Parts, J. Forment, E. Voest, J. Marioni, Andrew Bassett, Mathew J. Garnett
          </td>
          <td>2024-10-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33f7b85ef899c9135146a0af70ef75b42923dd2f" target='_blank'>
              Optical genome mapping of structural variants in Parkinson’s disease-related induced pluripotent stem cells
              </a>
            </td>
          <td>
            Joanne Trinh, Susen Schaake, C. Gabbert, Theresa Lüth, Sally A Cowley, André Fienemann, Kristian K Ullrich, Christine Klein, Philip Seibler
          </td>
          <td>2024-10-19</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf5407072a36736a7b97ab6c58b0a5238e234714" target='_blank'>
              Characterizing the allele-specific gene expression landscape in high hyperdiploid acute lymphoblastic leukemia with BASE
              </a>
            </td>
          <td>
            Jonas Andersson, Efe Aydın, Rebeqa Gunnarsson, H. Lilljebjörn, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2024-10-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Tumors consist of cancer cells with different genetic, epigenetic, and phenotypic properties. Cancer stem cells are an important subpopulation of heterogeneous cancer cells and are capable of initiating and propagating tumors. The term cancer stem cells has become broader in efforts to understand their phenotypic plasticity to switch fates between self-renewal and differentiation. Cancer stem cell plasticity is significantly associated with the initiation of metastasis, resistance to therapy, and tumor recurrence. With our broadened knowledge of epigenetic regulation and metabolic reprogramming as key elements enabling such capabilities, an expansive body of literature has demonstrated the functional importance of each element in contributing to cancer stem cell characteristics. Recently, the direct interplay between epigenetic regulation and metabolic reprogramming has begun to be appreciated in the context of cancer stem cells with growing interest. In this review, we discuss the mechanisms by which cancer stem cells orchestrate the reciprocal regulation of cellular metabolism and epigenetic alterations. In the discussion, compelling, unanswered questions on this topic have been elaborated for the interest of the research community and how recent technological developments help tackle such research ideas. A comprehensive understanding of cancer stem cell attributes that are largely governed by epigenetic and metabolic reprogramming would enable the advancement of precise therapeutic options and the prediction of better responses to drugs, holding great promise in cancer treatment and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a352f514fc793046331539c8fc8b91b79542075" target='_blank'>
              Interplay between epigenetics and metabolism controls cancer stem cell plasticity
              </a>
            </td>
          <td>
            Jee-Eun Choi, Inwha Baek
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f62e543f9ec01d9f73f5eef49615f95671e727" target='_blank'>
              Cancer drug-tolerant persister cells: from biological questions to clinical opportunities.
              </a>
            </td>
          <td>
            Mariangela Russo, Mengnuo Chen, Elisa Mariella, H. Peng, Sumaiyah K Rehman, E. Sancho, Alberto Sogari, Tzen S Toh, Nathalie Q Balaban, Eduard Batlle, R. Bernards, Mathew J. Garnett, Matthew J Hangauer, E. Leucci, J. Marine, Catherine A O'Brien, Yaara Oren, E. Patton, C. Robert, Susan M Rosenberg, Shensi Shen, A. Bardelli
          </td>
          <td>2024-09-02</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e459f679b760cbc6c5f4d15f5d578e006d0e8b9" target='_blank'>
              Establishment and characterization of 18 Sarcoma Cell Lines: Unraveling the Molecular Mechanisms of Doxorubicin Resistance in Sarcoma Cell Lines
              </a>
            </td>
          <td>
            Young-Eun Cho, Soon-Chan Kim, Ha Jeong Kim, Ilkyu Han, Ja-Lok Ku
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401dfb6b1e61a2b7931608ee7b632746b618204c" target='_blank'>
              Potential promising of synthetic lethality in cancer research and treatment.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Behnam Najafiyan, Reza Morovatshoar, Mahdieh Khorsandi, Adib Dashtizadeh, Arash Kiani, Farnoosh Farzam, Kimia Sadat Kazemi, Mohsen Nabi Afjadi
          </td>
          <td>2024-09-21</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="
 We aim to offer new insights on the regulation of metabolic plasticity in renal cell carcinoma (RCC) and identify novel metabolic regulatory targets that can be used for potential treatment therapies of the disease. RCC affects an estimated 400,000 people worldwide each year with over 100,00 deaths annually that disproportionately affects racial/ethnic minorities in the United States. RCC is a term that covers an array of kidney cancer subtypes differing in incidence, histopathology, genetic and molecular alterations, as well as in clinical outcomes and prognoses. However, a common characteristic of many RCCs is that they are driven by metabolic rewiring, due to a high frequency of mutations in genes that regulate major metabolic processes in the cell. It was recently discovered that patient overall survival and prognosis has been linked to the expression of certain metabolic signatures in RCC tumor cells. Metabolic plasticity allows cancer cells to survive and adapt to different environments and metabolic stress conditions, such as nutrient deprivation. Therefore, exploring and exploiting metabolic rewiring in these cells may reveal genetic vulnerabilities and dependencies. This could lead to discovery of novel biomarkers and molecular targets that can be implicated as future therapeutic interventions for this disease. To this end, we applied functional genomic screening to expose genetic dependencies and vulnerabilities under different physiologic and metabolic conditions in patient derived medullary (PDM) RCC cells. The medullary RCC is a rare and aggressive RCC subtype that disproportionately affects the African American/Black population. We applied a CRISPR- based combinatorial screening platform that is based on co-expression of Cas9 and Cas12a nucleases and libraries of hybrid guide RNAs (hgRNA) to achieve ultra-efficient gene knockout (Aregger et al., 2021; Gonatopoulos-Pournatzis et al., 2020) in PDM RCC cells. Additionally, cells were screened under different media conditions that mimic human physiological nutrient conditions and a media deprived of lipids, one of the major metabolic substrates of the cell. By utilizing a genome-wide pooled CRISPR gene knockout screen we were able to systematically uncover genes required for cell survival and proliferation in a panel of patient-derived RCCs. We have begun to uncover genetic dependencies and major metabolic pathways that can be elucidated and exploited for future studies, such as those involving ferroptosis which is a targetable pathway for the development of cancer treatment therapies. Our findings demonstrate some of the key genetic drivers underlying cellular adaptations to different physiological environments and nutrient deprivation. In future studies we will further explore the relationship between different key metabolic regulatory genes in RCC and map for genetic interactions in major metabolic pathways.
 Citation Format: Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger. Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd9b94600307109e3720c07a3b1a13c91abd8ae" target='_blank'>
              Abstract C091: Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma
              </a>
            </td>
          <td>
            Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Intragenomic mutation rates can vary dramatically due to transcription-associated mutagenesis or transcription-coupled repair, which vary based on local epigenomic modifications that are nonuniformly distributed across genomes. One feature associated with decreased mutation is higher expression level, which depends on environmental cues. To understand the magnitude of expression-dependent mutation rate variation, we perturbed expression through a heat treatment in Arabidopsis thaliana. We quantified gene expression to identify differentially expressed genes, which we then targeted for mutation detection using duplex sequencing. This approach provided a highly accurate measurement of the frequency of rare somatic mutations in vegetative plant tissues, which has been a recent source of uncertainty. Somatic mutations in plants may be useful for understanding drivers of DNA damage and repair in the germline since plants experience late germline segregation and both somatic and germline cells share common repair machinery. We included mutant lines lacking mismatch repair (MMR) and base excision repair (BER) capabilities to understand how repair mechanisms may drive biased mutation accumulation. We found wild-type (WT) and BER mutant mutation frequencies to be very low (mean variant frequency 1.8 × 10−8 and 2.6 × 10−8, respectively), while MMR mutant frequencies were significantly elevated (1.13 × 10−6). Interestingly, in the MMR mutant lines, there was no difference in the somatic mutation frequencies between temperature treatments or between highly versus lowly expressed genes. The extremely low somatic variant frequencies in WT plants indicate that larger datasets will be needed to address fundamental evolutionary questions about whether environmental change leads to gene-specific changes in mutation rate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c6a9ca240c4acb07b722d7f1e2c985281e1839" target='_blank'>
              Exploring the Relationship Between Gene Expression and Low-Frequency Somatic Mutations in Arabidopsis with Duplex Sequencing
              </a>
            </td>
          <td>
            Gus Waneka, Braden Pate, J. G. Monroe, Daniel B. Sloan
          </td>
          <td>2024-10-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The ability of cancer cells to change and adapt poses a critical challenge to identifying curative solutions. Tumor evolution has been extensively studied from a genetic perspective, to guide clinicians in selecting the most appropriate therapeutic option based on a patient's mutational profile. However, several studies reported that tumors can evolve toward more aggressive stages or become resistant to therapies without changing their genetic makeup. Indeed, several cell-intrinsic and cell-extrinsic mechanisms contribute to tumor evolution. In this viewpoint, I focus on how chromatin, epigenetic, and transcriptional changes contribute to tumor evolution, allowing cancer cells to transition to different cell states and bypass response to therapies. Although tumor nongenetic evolution is harder to trace and predict, understanding its principles might open new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0869302e828b2c81eac62f89984d51de011a3b74" target='_blank'>
              Nongenetic evolution of the tumor: from challenges to new therapeutic opportunities.
              </a>
            </td>
          <td>
            E. Oricchio
          </td>
          <td>2024-10-18</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Translocations have largely been implicated in tumor development. However, beyond the consequences of aberrant gene expression near the breakpoint, their effects remain unexplored. In this work, we characterize the interplay between translocations, chromatin organization and gene expression using mantle cell lymphoma (MCL) as a model. We show that in vitro induced MCL-associated translocations can drive transcriptional changes at entire chromosome arms affecting multiple genes in a regulon-like fashion. Additionally, overexpressed genes in MCL patients are enriched in the exact same genomic regions, further underlining its potential relevance for lymphomagenesis. Moreover, we demonstrate a clear link between the translocation-induced transcriptional alterations and genome organization, with genes most susceptible to change expression residing in pre-existing long-range interacting loops spanning 50 megabases. The translocation places the strong immunoglobulin enhancer into this loop, allowing the spread of its regulatory potential over the entire affected chromosome arm. Finally, we show that translocation-induced effects mainly represent expression enhancement of genes already active prior to translocation formation, highlighting the importance of the epigenetic state of the cell in which this initial hit occurs. In summary, we show that translocations can induce ample, simultaneous gene expression changes affecting entire chromosome arms, representing an important new mechanism for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdbb3468215c566e38e0c9ba35afc86f4ee62c1" target='_blank'>
              Translocations can drive expression changes of multiple genes in regulons covering entire chromosome arms
              </a>
            </td>
          <td>
            Anna Oncins, Roser Zaurin, Houyem Toukabri, Kimberly Quililan, José R Hernández Mora, Magdalena A. Karpinska, Erik Wernersson, Alastair Smith, Agostina Bianchi, Leone Albinati, Andrea Rivero, François Serra, Raúl Gómez, Cristina López, Sílvia Beà, Nadia Halidi, A. Valencia, Magda Bienko, A. M. Oudelaar, Renée Beekman
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole-genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2,726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirically testing all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated two compounds as potent PGCC inhibitors. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f39f65e370bc9aafe05599fe5e71c2036a3f471" target='_blank'>
              High-Throughput Empirical and Virtual Screening to Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a hematological malignancy associated with poor prognosis and relapses. MM has a strong inherited component, and while the mutational and transcriptomic landscapes of MM have been characterized extensively, the molecular consequences of germline genetic variation in MM have not been previously explored. Here, we have leveraged matched genotype and expression data from 607 patients to map the germline and somatic regulatory variation in MM. We find that non-coding variation shapes the transcriptional landscape of MM, contributing to overall survival. These regulatory variants alter the binding of transcription factors with known importance in MM, such as IRF4, providing insights into the underlying mechanisms. These data contribute to the increasing understanding of the genetic, molecular, and cellular forces shaping MM risk and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5c5bba32932ad19d6d4a4784baebd131dde155a" target='_blank'>
              Germline and somatic genetic effects on gene expression and outcome in patients with Multiple Myeloma
              </a>
            </td>
          <td>
            H. Natri, Linh Bui, L. Peter, Bianca Argente, A. Federico, A. Gutierrez, Mei-i Chung, Jonathan Keats, Nicholas E Banovich
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Effective treatment of pediatric acute leukemia is dependent on accurate genomic classification, typically derived from a combination of multiple time-consuming and costly techniques such as flow cytometry, fluorescence in situ hybridization (FISH), karyotype analysis, targeted PCR, and microarrays (Arber et al., 2016; Iacobucci & Mullighan, 2017; Narayanan & Weinberg, 2020). We investigated the feasibility of a comprehensive single-assay classification approach using long-read sequencing, with real-time genome target enrichment, to classify chromosomal abnormalities and structural variants characteristic of acute leukemia. We performed whole genome sequencing on DNA from diagnostic peripheral blood or bone marrow for 54 pediatric acute leukemia cases with diverse genomic subtypes. We demonstrated the characterization of known, clinically relevant karyotype abnormalities and structural variants concordant with standard-of-care clinical testing. Subtype-defining genomic alterations were identified in all cases following a maximum of forty-eight hours of sequencing. In 18 cases, we performed real-time analysis – concurrent with sequencing – and identified the driving alteration in as little as fifteen minutes (for karyotype) or up to six hours (for complex structural variants). Whole genome nanopore sequencing with adaptive sampling has the potential to provide detailed genomic classification of acute leukemia specimens with reduced cost and turnaround time compared to the current standard of care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa083d181be0ae2fc8596d1c7ad6534ea54258c0" target='_blank'>
              Real-time genomic characterization of pediatric acute leukemia using adaptive sampling
              </a>
            </td>
          <td>
            Julie Geyer, K. Opoku, John Lin, Lori Ramkissoon, C. Mullighan, N. Bhakta, Thomas B. Alexander, Jeremy R. Wang
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Despite development of several effective therapies for multiple myeloma (MM), the prognosis of patients with partial deletion of chromosome 17 (del(17p)) and TP53 aberrations remains poor. By applying comprehensive multi-omics profiling analyses (whole exome and transcriptome sequencing plus proteomics) and functional ex vivo drug screening to samples from 167 patients with MM, we uncovered novel therapeutic vulnerabilities specific to TP53 mutated MM. Our findings revealed a distinct sensitivity profile to a range of inhibitors (mitotic, topoisomerase, HDAC, HSP90, IGF1R and PI3K/AKT/mTOR inhibitors) irrespective of 17p deletion status. Conversely, no increase in sensitivity was observed for monoallelic TP53 (del(17p) with WT TP53) when compared to other samples, highlighting the remaining unmet clinical need. Notably, plicamycin, an RNA synthesis inhibitor linked to modulation of chromatin structure and increased transcription, emerged as particularly efficacious for TP53 mutated MM. The increased sensitivity correlated with higher protein expression of the drug targets: HDAC2, HSP90AA1 and multiple ribosomal subunits. Additionally, we observed increased RNA expression of G2M checkpoint, E2F targets and mTORC1 signaling in our cohort and the MMRF-CoMMpass (NCT01454297) study in TP53 mutated MM. Harmonization of multi-omics data with ex vivo drug screening results revealed that TP53 mutated MM is functionally distinct from MM with monoallelic TP53, and demonstrate that MM with mutated TP53, with and without del(17p), may be targetable by approved drugs, and further indicates the need for regular monitoring by sequencing to identify these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a52d0d33d267850e10246d4013cb5921df1165" target='_blank'>
              Multi-omic and functional screening reveal targetable vulnerabilities in TP53 mutated multiple myeloma
              </a>
            </td>
          <td>
            Dimitrios Tsallos, Nemo Ikonen, Juho J. Mienen, M. M. Majumder, S. Eldfors, I. Västrik, A. Parsons, M. Suvela, Katie Dunphy, Paul Dowling, D. Bazou, Peter O’Gorman, J. Lievonen, R. Silvennoinen, Pekka Anla, C. Heckman
          </td>
          <td>2024-08-23</td>
          <td>None</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Prostate cancer (PCa) is the second-leading cause of cancer mortality in U.S. men, with African American (AA) men showing higher incidence and mortality compared to European American men. In addition, AA men are more likely to be diagnosed with more aggressive PCa resulting in reduction of treatment options, especially as tumors develop therapy resistance. Thus, understanding the mechanisms underlying PCa therapy resistance is crucial for developing new therapeutic options. The upregulation of the Lens Epithelium Derived Growth Factor p75 (LEDGF/p75) by glucocorticoid receptor (GR) is a novel mechanism of PCa therapy resistance recently identified by our group. GR upregulation in PCa bypasses the blockade of androgen receptor by inhibitors such as enzalutamide and abiraterone. LEDGF/p75 is an oncogenic transcription co-activator that promotes DNA damage repair and cancer cell survival against environmental stressors including chemotherapeutic drugs such as docetaxel (DTX). An emerging concern in PCa treatment is therapy cross-resistance, in which the molecular mechanisms of resistance to a particular drug confer resistance to another drug. Our recent studies showed that GR upregulates and interacts with LEDGF/p75 forming a transcriptional complex that includes other oncoproteins such as Menin, JPO2, c-Myc and HRP2, and promotes DTX resistance (DTX-R) in PCa cells. In addition, we reported that both GR and LEDGF/p75 are upregulated in enzalutamide resistant (ENZ-R) PCa cells. We hypothesized that the GR- LEDGF/p75 transcriptional network plays a role in PCa therapy cross-resistance to both ENZ and DTX and is a promising therapeutic target. Cell viability assays revealed that ENZ-R in PCa cells confers cross-resistance to DTX. Western blot analysis revealed upregulation of members of the GR-LEDGF/p75 network such as GR, LEDGF/p75, Menin and c-Myc in ENZ-R PCa cells, consistent with similar observations in DTX-R PCa cells. Upon knockdown of GR and LEDGF/p75 in DTX-R PCa cells, we identified by RNA-seq analysis overlapping DEGs associated with both ENZ-R and DTX-R, suggesting that the GR-LEDGF/p75 transcriptional network may contribute to therapy cross-resistance. Consistent with this, mining of publicly available RNA-seq databases identified 46 overlapping differentially expressed genes (DEGs) between ENZ-R and DTX-R PCa cells. Pathway analysis of DEGs regulated by both GR and LEDGF/p75 revealed significant enrichment of Apoptosis, p53, and DNA repair pathways, which are associated with therapy resistance. Ongoing studies are evaluating the anti-cancer properties and synergistic/additive potential of various inhibitors of the GR-LEDGF/p75 transcriptional network in cellular models of ENZ-R and DTX-R. These studies provide novel mechanistic insights into PCa therapy cross-resistance and may lead to the development of novel therapeutic strategies to reduce PCa mortality disparities.
 Citation Format: Pedro T. Ochoa, Evelyn S. Sanchez-Hernandez, Kai Wen Cheng, Isaac Kremsky, Charles Wang, Carlos A. Casiano. The GR-LEDGF/p75 transcriptional network as a promising target to reduce therapy cross-resistance in prostate cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C096.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0c6225006bf01cdfdfbdc7e8efd4e751393fc1" target='_blank'>
              Abstract C096: The GR-LEDGF/p75 transcriptional network as a promising target to reduce therapy cross-resistance in prostate cancer
              </a>
            </td>
          <td>
            P. Ochoa, Evelyn S. Sanchez-Hernandez, Kai Wen Cheng, Isaac Kremsky, Charles Wang, C. Casiano
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Long INterspersed Element-1 (LINE-1; L1) and Alu are two families of transposable elements (TEs) occupying ∼17% and ∼11% of the human genome, respectively. Though only a small fraction of L1 copies is able to produce the machinery to mobilize autonomously, Alu elements and degenerate L1 copies can hijack their functional machinery and mobilize in trans. The expression and subsequent copy number expansion of L1 and Alu can exert pathological effects on their hosts, promoting genome instability, inflammation, and cell cycle alterations. These features have made L1 and Alu promising focus subjects in studies of aging and aging diseases where they can become active. However, the mechanisms regulating variation in their expression and copy number remain incompletely characterized. Moreover, the relevance of known mechanisms to diverse human populations remains unclear, as mechanisms are often characterized in isogenic cell culture models. To address these gaps, we leveraged genomic data from the 1000 Genomes Project to carry out a trans-ethnic GWAS of L1 and Alu insertion global singletons. These singletons are rare insertions observed only once in a population, potentially reflecting recently acquired L1 and Alu integrants or structural variants, and which we used as proxies for L1/Alu-associated copy number variation. Our computational approach identified single nucleotide variants in genomic regions containing genes with potential and known TE regulatory properties, and it enriched for single nucleotide variants in regions containing known regulators of L1 expression. Moreover, we identified many reference TE copies and polymorphic structural variants that were associated with L1/Alu singletons, suggesting their potential contribution to TE copy number variation through transposition-dependent or transposition-independent mechanisms. Finally, a transcriptional analysis of lymphoblastoid cells highlighted potential cell cycle alterations in a subset of samples harboring L1/Alu singletons. Collectively, our results (i) suggest that known TE regulatory mechanisms may also play regulatory roles in diverse human populations, (ii) expand the list of genic and repetitive genomic loci implicated in TE copy number variation, and (iii) reinforce the links between TEs and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c547e837e626947d2c017f057b521d189f2c4fd3" target='_blank'>
              Multi-ancestry GWAS reveals loci linked to human variation in LINE-1- and Alu-copy numbers
              </a>
            </td>
          <td>
            Juan I. Bravo, Lucia Zhang, BA Benayoun
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff04d632e2b6ac550f2911954ea295c050c33ca2" target='_blank'>
              Development of a polygenic score predicting drug resistance and patient outcome in breast cancer
              </a>
            </td>
          <td>
            Divya Sahu, Jeffrey Shi, Isaac Andres Segura Rueda, Ajay Chatrath, Anindya Dutta
          </td>
          <td>2024-10-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24869ef91b9d9f698a0aa1c1c25527b7385dd280" target='_blank'>
              Deciphering resistance mechanisms in cancer: final report of MATCH-R study with a focus on molecular drivers and PDX development
              </a>
            </td>
          <td>
            Damien Vasseur, L. Bigot, Kristi Beshiri, Juan-David Florez-Arango, F. Facchinetti, A. Hollebecque, L. Tselikas, M. Aldea, F. Blanc-Durand, A. Gazzah, D. Planchard, Ludovic Lacroix, N. Pata-Merci, Catline Nobre, Alice Da Silva, C. Nicotra, M. Ngo-Camus, F. Braye, Sergey Nikolaev, S. Michiels, Gerome Jules-Clement, K. Olaussen, Fabrice André, J. Scoazec, F. Barlesi, Santiago Ponce, Jean-Charles Soria, Benjamin Besse, Yohann Loriot, L. Friboulet
          </td>
          <td>2024-10-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Hydroxychloroquine (HCQ) and chloroquine are repurposed drugs known to disrupt autophagy, a molecular recycling pathway essential for tumor cell survival, chemotherapeutic resistance, and stemness. We pursued a multi-omic strategy in OVCAR3 ovarian cancer and CCL218 colorectal cancer cells. Two genome-scale screens were performed. In the forward genetic screen, cell populations were passaged for 15 drug pulse-chases with HCQ or vehicle control. Evolved cells were collected and processed for bulk RNA-seq, exome-seq, and single-cell RNA-seq (scRNA-seq). In the reverse genetic screen, a pooled CRISPR-Cas9 library was used in cells over three pulse-chases of HCQ or vehicle control treatments. HCQ evolved cells displayed remarkably few mutational differences, but substantial transcriptional differences. Transcriptomes revealed multiple pathways associated with resistance to HCQ, including upregulation of glycolysis, exocytosis, and chromosome condensation/segregation, or downregulation of translation and apoptosis. The Cas9 screen identified only one autophagy gene. Chromosome condensation and segregation were confirmed to be disrupted by HCQ in live cells and organelle-free in vitro extracts. Transcriptional plasticity was the primary mechanism by which cells evolved resistance to HCQ. Neither autophagy nor the lysosome were substantive hits. Our analysis may serve as a model for how to better position repurposed drugs in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebde0f2f8551b6143901c8c35aae825bc7889573" target='_blank'>
              Autophagy unrelated transcriptional mechanisms of hydroxychloroquine resistance revealed by integrated multi-omics of evolved cancer cells.
              </a>
            </td>
          <td>
            Silvia G Vaena, Martin J. Romeo, Mirna Mina-Abouda, Emma C Funk, G. Fullbright, David T. Long, Joe R Delaney
          </td>
          <td>2024-09-19</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8719f1e85495bf521f29d9969611ef5b8435eeb3" target='_blank'>
              The evolutionary theory of cancer: challenges and potential solutions.
              </a>
            </td>
          <td>
            L. Laplane, Carlo C. Maley
          </td>
          <td>2024-09-10</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background/Objectives: The rat osteosarcoma cell line UMR-106 is widely used for the study of bone cancer biology but it has not been well characterized with modern genomic methods. Methods: To better understand the biology of UMR-106 cells we used a combination of optical genome mapping (OGM), long-read sequencing nanopore sequencing and RNA sequencing.The UMR-106 genome was compared to a strain-matched Sprague-Dawley rat for variants associated with human osteosarcoma while expression data were contrasted with a public osteoblast dataset. Results: Using the COSMIC database to identify the most affected genes in human osteosarcomas we found somatic mutations in Tp53 and H3f3a. OGM identified a relatively small number of differences between the cell line and a strain-matched control animal but did detect a ~45 Mb block of amplification that included Myc on chromosome 7 which was confirmed by long-read sequencing. The amplified region showed several blocks of non-contiguous rearranged sequence implying complex rearrangements during their formation and included 14 genes reported as biomarkers in human osteosarcoma, many of which also showed increased transcription. A comparison of 5mC methylation from the nanopore reads of tumor and control samples identified genes with distinct differences including the OS marker Cdkn2a. Conclusions: This dataset illustrates the value of long DNA methods for the characterization of cell lines and how inter-species analysis can inform us about the genetic nature underlying mutations that underpin specific tumor types. The data should be a valuable resource for investigators studying osteosarcoma, in general, and specifically the UMR-106 model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0957ae8779454bb6e3009d9a818af23c55e01e" target='_blank'>
              Characterization of the Rat Osteosarcoma Cell Line UMR-106 by Long-Read Technologies Identifies a Large Block of Amplified Genes Associated with Human Disease
              </a>
            </td>
          <td>
            Alan F. Scott, David W. Mohr, William A. Littrell, Reshma Babu, Michelle Kokosinski, V. Stinnett, Janvi Madhiwala, John Anderson, Ying S. Zou, Kathleen L. Gabrielson
          </td>
          <td>2024-09-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic cancer is an aggressive disease with limited therapeutic options. Gemcitabine is a clinically used chemotherapy for pancreatic cancer treatment but has limited efficacy. The molecular mechanisms responsible for gemcitabine resistance in pancreatic cancer are still unclear. Here, we analyzed the genomics of drug sensitivity in cancer dataset and identified various pancreatic cancer cells with distinct sensitivity to gemcitabine. Validation with proliferation and clonogenic assays confirmed that while CFPAC1 and PANC0327 are highly gemcitabine-sensitive, ASPC1 and PANC1 display intrinsic gemcitabine resistance. Analysis of the gene expression profile of these cells identified several genes that are differentially upregulated in resistant versus sensitive cells. Pathway enrichment analysis using the differential genes revealed the upregulation of Hippo, Wnt, MAPK signaling pathways as well as autophagy and glutathione metabolism in the resistant cells. Using quantitative PCR, immunoblotting and biochemical assays, and extracellular flux analysis methods, we have characterized the molecular distinctions between gemcitabine sensitive and resistant cells. Our results provide insight on the molecular mechanisms driving gemcitabine resistance in pancreatic cancer and could lead to novel therapeutic strategies to overcome resistance and improve patients’ treatment.
 Citation Format: Amina D Bilalbegovic, Bowen Fu, Zeribe Nwosu. Identification of novel drivers of gemcitabine resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52052963f11a91114395dd33df22ba52dd0b89f" target='_blank'>
              Abstract B089: Identification of novel drivers of gemcitabine resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Amina D Bilalbegovic, Bowen Fu, Z. Nwosu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Resistance to endocrine therapies (ET) is common in estrogen receptor (ER) positive breast cancer, and most relapsed patients die with ET-resistant disease. While genetic mutations provide explanations for some relapses, mechanisms of resistance remain undefined in many cases. Drug-induced epigenetic reprogramming has been shown to provide possible routes to resistance. By analyzing histone H3 lysine 27 acetylation (H3K27ac) profiles and transcriptional reprogramming in models of ET resistance, we discovered that selective ER degraders (SERDs), such as fulvestrant, promote expression of VGLL1, a co-activator for TEAD transcription factors. VGLL1, acting via TEADs, promoted expression of genes that drive growth of fulvestrant-resistant breast cancer cells. Pharmacological disruption of VGLL1/TEAD4 interaction inhibited VGLL1/TEAD-induced transcriptional programs to prevent growth of resistant cells. EGFR was among the VGLL1/TEAD-regulated genes, and VGLL1-directed EGFR upregulation sensitized fulvestrant-resistant breast cancer cells to EGFR inhibitors. Taken together, these findings identify VGLL1 as a transcriptional driver in ET resistance and advance therapeutic possibilities for relapsed ER+ breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24a869ddbb92dd50c0974c10582d19a6e18e2d9" target='_blank'>
              Induction of the TEAD Co-activator VGLL1 by Estrogen Receptor-Targeted Therapy Drives Resistance in Breast Cancer.
              </a>
            </td>
          <td>
            Carolina Gemma, Chun-Fui Lai, Anup K Singh, A. Belfiore, N. Portman, Heloisa Z Milioli, M. Periyasamy, Sara Raafat, Alyssa J Nicholls, Claire M Davies, Naina R Patel, Georgia M Simmons, Hailing Fan, Van T M Nguyen, Luca Magnani, Emad A. Rakha, Lesley-Ann Martin, Elgene Lim, R. C. Coombes, G. Pruneri, L. Buluwela, Simak Ali
          </td>
          <td>2024-10-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT New and improved drugs are required for the treatment and ultimate eradication of malaria. The efficacy of front-line therapies is now threatened by emerging drug resistance; thus, new tools to support the development of drugs with a lower propensity for resistance are needed. Here, we describe the development of a RESistance Mapping And Profiling (ResMAP) platform for the identification of resistance-conferring mutations in Plasmodium drug targets. Proof-of-concept studies focused on interrogating the antimalarial drug target, Plasmodium falciparum lysyl tRNA synthetase (PfKRS). Saturation mutagenesis was used to construct a plasmid library encoding all conceivable mutations within a 20-residue span at the base of the PfKRS ATP-binding pocket. The superior transfection efficiency of Plasmodium knowlesi was exploited to generate a high coverage parasite library expressing PfKRS bearing all possible amino acid changes within this region of the enzyme. The selection of the library with PfKRS inhibitors, cladosporin and DDD01510706, successfully identified multiple resistance-conferring substitutions. Genetic validation of a subset of these mutations confirmed their direct role in resistance, with computational modeling used to dissect the structural basis of resistance. The application of ResMAP to inform the development of resistance-resilient antimalarials of the future is discussed. IMPORTANCE An increase in treatment failures for malaria highlights an urgent need for new tools to understand and minimize the spread of drug resistance. We describe the development of a RESistance Mapping And Profiling (ResMAP) platform for the identification of resistance-conferring mutations in Plasmodium spp, the causative agent of malaria. Saturation mutagenesis was used to generate a mutation library containing all conceivable mutations for a region of the antimalarial-binding site of a promising drug target, Plasmodium falciparum lysyl tRNA synthetase (PfKRS). Screening of this high-coverage library with characterized PfKRS inhibitors revealed multiple resistance-conferring substitutions including several clinically relevant mutations. Genetic validation of these mutations confirmed resistance of up to 100-fold and computational modeling dissected their role in drug resistance. We discuss potential applications of this data including the potential to design compounds that can bypass the most serious resistance mutations and future resistance surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d7dc04ef86759e728eca0280100918ebb17c149" target='_blank'>
              ResMAP—a saturation mutagenesis platform enabling parallel profiling of target-specific resistance-conferring mutations in Plasmodium
              </a>
            </td>
          <td>
            Richard J. Wall, Stuart A MacGowan, Irene Hallyburton, Aisha J Syed, Sowmya Ajay Castro, Gourav Dey, Rachel Milne, Stephen Patterson, J. Phelan, Natalie Wiedemar, Susan Wyllie
          </td>
          <td>2024-08-27</td>
          <td>mBio</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d317e0704a11a531980e863a514ed7e3ab276870" target='_blank'>
              Advances in pathogenesis research and challenges in treatment development for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hiroki Yamaguchi
          </td>
          <td>2024-09-03</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genomic studies have identified frequent mutations in subunits of the SWI/SNF chromatin remodeling complex including SMARCA4 and ARID1A in non-small cell lung cancer. Previously, we and others have identified that SMARCA4-mutant lung cancers are highly dependent on oxidative phosphorylation (OXPHOS). Despite initial excitements, therapeutics targeting metabolic pathways such as OXPHOS have largely been disappointing due to rapid adaptation of cancer cells to inhibition of single metabolic enzymes or pathways, suggesting novel combination strategies to overcome adaptive responses are urgently needed. Here, we performed a functional genomics screen using CRISPR-Cas9 library targeting genes with available FDA approved therapeutics and identified ROCK1/2 as a top hit that sensitizes cancer cells to OXPHOS inhibition. We validate these results by orthogonal genetic and pharmacologic approaches by demonstrating that KD025 (Belumosudil), an FDA approved ROCK inhibitor, has highly synergistic anti-cancer activity in vitro and in vivo in combination with OXPHOS inhibition. Mechanistically, we showed that this combination induced a rapid, profound energetic stress and cell cycle arrest that was in part due to ROCK inhibition-mediated suppression of the adaptive increase in glycolysis normally seen by OXPHOS inhibition. Furthermore, we applied global phosphoproteomics and kinase-motif enrichment analysis to uncover a dynamic regulatory kinome upon combination of OXPHOS and ROCK inhibition. Importantly, we found converging phosphorylation-dependent regulatory cross-talk by AMPK and ROCK kinases on key RHO GTPase signaling/ROCK-dependent substrates such as PPP1R12A, NUMA1 and PKMYT1 that are known regulators of cell cycle progression. Taken together, our study identified ROCK kinases as critical mediators of metabolic adaptation of cancer cells to OXPHOS inhibition and provides a strong rationale for pursuing ROCK inhibitors as novel combination partners to OXPHOS inhibitors in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa35b0eb45529ff5e6512abe7e0fede05865b618" target='_blank'>
              Therapeutic modulation of ROCK overcomes metabolic adaptation of cancer cells to OXPHOS inhibition and drives synergistic anti-tumor activity
              </a>
            </td>
          <td>
            Nicholas Blazanin, Xiaobing Liang, Iqbal Mahmud, Eiru Kim, Sara A Martinez, Lin Tan, Waikin Chan, N. Anvar, M. Ha, Md Qudratullah, R. Minelli, M. Peoples, Philip L. Lorenzi, Traver Hart, Yonathan Lissanu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56281f9e2c04f45376f07cc22699136fff5305aa" target='_blank'>
              Exploring the interaction between immune cells in the prostate cancer microenvironment combining weighted correlation gene network analysis and single-cell sequencing: An integrated bioinformatics analysis
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Sobhan Bavandi, Melika Djamali, Ali Shakeri Abroudi
          </td>
          <td>2024-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Glioblastoma is an aggressive primary malignant brain tumour with poor survival. Unsuccessful improvement of treatment could be due to transcriptional plasticity of the cancer cells including stem-like cancer cells. Short-term cultured patient-derived neurospheres consisting of cancer cells and stem-like cancer cells are used in this project to find individual drug sensitivities via drug screening. Drug resistance mechanisms on transcriptome level is further explored at high resolution using single nucleus RNA sequencing (snRNA-seq) on drug- and mock-treated neurospheres to find drug vulnerabilities.



 Neurospheres were established from patients included in Protarget, a Phase II, prospective, non-randomized clinical trial. A library including BET-inhibitors was used for high-throughput drug screening to evaluate ex vivo patient drug sensitivities. DMSO-treated and drug-treated neurospheres were prepared for snRNA-seq using 10X genomics. We analyzed snRNA-seq using state-of-the art bioinformatic methods to derive e.g. differentially expressed genes, transcriptional states, and large-scale somatic copy number alterations.



 Through integrative analysis of drug- and DMSO-treated neurosphere cultures, we identified drug-specific differences in both transcriptional state, cell cycle, and stem-like signatures. We found that major copy number changes were preserved between the primary tumour surgical sample and neurosphere culture, exhibiting low average divergence. We found tumour-cell specific transcriptomic differences with clusters exhibiting more extreme drug-responses, implying that neurosphere cultures either sustain or develop sub-clonal, intrinsic cell-type specific drug-responses. One glioblastoma patient underwent treatment with a BET-inhibitor as part of a clinical trial and had initial good clinical response. The transcriptional response following treatment with the same BET-inhibitor on neurospheres from the same patient revealed an increased block in cell cycle, induced transcriptional state changes towards MES2- and OPC-like states, and revealed small clusters of tumour cells showing reduced response to the BET-inhibitor, indicating a pattern of drug resistance.



 We find overall high concordance between the initial tumour surgical sample, assessed by whole-genome sequencing, and patient-derived neurospheres at the level of large-scale copy number changes. Transcriptional subtypes change during treatment including response to a BET-inhibitor on sensitive patient-derived neurospheres. We conclude that combining high-throughput drug screening with snRNA-seq on DMSO-treated and drug-treated neurospheres is a feasible way to access response and resistance mechanisms in glioblastoma, and targeting transcriptional states could be a promising way to treat glioblastoma patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35ffeb471ac7dd263a23a8284dfbe8557f014c84" target='_blank'>
              P01.05.A DRUG-RESPONSE ANALYSIS AND SINGLE NUCLEUS RNA SEQUENCING ON PATIENT-DERIVED NEUROSPHERES TO REVEAL DRUG RESPONSE AND RESISTANCE MECHANISMS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. J. S. Lü, J. Hendriksen, J. Ariey-Bonnet, A. Anand, S. R. Michaelsen, F. G. R. González, A. Locallo, A. Azam, J. Skjøth-Rasmussen, P. Hamerlik, D. Nørøxe, B. Hasselbalch, E. Vik-Mo, B. Kristensen, T. Urup, U. Lassen, H. Poulsen, K. Wennerberg, J. Weischenfeldt
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP‐ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell's plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. Key points/Highlights Tumours are ecosystems in which cancer and non‐cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco‐evolutionarily designed therapies should consider cancer cell plasticity and patient‐specific characteristics to improve clinical outcome and prevent relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f584cd6ae71db959a312c8365ae2a9ada15cf5" target='_blank'>
              Ecological and evolutionary dynamics to design and improve ovarian cancer treatment
              </a>
            </td>
          <td>
            Grace Y Q Han, Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, Mehdi Damaghi
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>2</td>
          <td>1</td>
        </tr>

        <tr id="Genomic instability is a core characteristic of cancer, often stemming from defects in DNA damage response (DDR) or increased replication stress. DDR defects can lead to significant genetic alterations, including changes in gene copy numbers, gene rearrangements, and mutations, which accumulate over time and drive the clonal evolution of cancer cells. However, these vulnerabilities also present opportunities for targeted therapies that exploit DDR deficiencies, potentially improving treatment efficacy and patient outcomes. The development of PARP inhibitors like Olaparib has significantly improved the treatment of cancers with DDR defects (e.g., BRCA1 or BRCA2 mutations) based on synthetic lethality. This achievement has spurred further research into identifying additional therapeutic targets within the DDR pathway. Recent progress includes the development of inhibitors targeting other key DDR components such as DNA-PK, ATM, ATR, Chk1, Chk2, and Wee1 kinases. Current research is focused on optimizing these therapies by developing predictive biomarkers for treatment response, analyzing mechanisms of resistance (both intrinsic and acquired), and exploring the potential for combining DDR-targeted therapies with chemotherapy, radiotherapy, and immunotherapy. This article provides an overview of the latest advancements in targeted anti-tumor therapies based on DDR and their implications for future cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdf72d2a9fa59533e1f8a82e5fc32e8e54f1de8" target='_blank'>
              Advancing cancer therapy: new frontiers in targeting DNA damage response
              </a>
            </td>
          <td>
            Jiekun Qian, Guoliang Liao, Maohui Chen, Ren-Wang Peng, Xin Yan, Jianting Du, Renjie Huang, Maojie Pan, Yuxing Lin, Xian Gong, Guo-bing Xu, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most prevalent malignant brain tumor in children that demonstrates clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), and particularly the 3gsubtype, exhibit high levels of MYC, high rates of metastasis, and have the worst prognosis. Despite treatment with surgery, radiation and chemotherapy, Group 3 medulloblastoma patients are more likely to develop recurrent tumors which are far more aggressive resulting in a 5-year survival rate after recurrence of 12%. Thus, there is an imperative need to examine the underlying mechanisms mediating treatment resistance. Recent evidence has identified cellular plasticity, as a major driver of therapy resistance. To investigate whether epigenetic mediated cell plasticity contributes to Myc-MB therapy response we performed Single Nuclei Multiome ATAC + Gene Expression analysis of paired primary and relapsed tumors. We identified the emergence of novel progenitor populations in resistant tumors with stem-like gene expression networks and altered chromatin accessibility. These emergent popuoations demostrated new enhancer-promoer interactions and exhibited increased metabolic reprogramming with potential for theraputic targeting. Because radiation is one of the key treatment modalities for Myc-MB, we chose to model radiation resistance in further studies. Single-cell RNA sequencing of outgrown in vivo irradiated orthotopic xenograft MB tumors showed an overall upregulation of metabolic pathways with the enhancement of genes that mediate oxidative phosphorylation and glutamate metabolism centering on wild-type IDH1 (wtIDH1), consistent with our patient data. IR-resistant cells showed altered chromatin occupancy of H3K3me3 and H3K27Ac. Disruption of IDH1 activity in IR resistant cells in vitro altered the epigenomic landscape, facilitating reversion to the pre-irradiated state. Interestingly, we observed that pluripotent genes with bivalent regions (H3K4me3 and H3K27me3) were enhanced in radiation-resistant cells and suppressed by IDH1 inhibition. Metabolic mass spectrometry reveled that metabolites from the citric acid cycle and synthesis of unsaturated fatty acids were among the main metabolically elevated molecules in radiation-resistant cells consistnat with the observed epigenomic reprograming. IDH1 inhibition reversed the metabolic reprograming and resensitized MB cells to radiation. In total our data identify a novel mechanism of radiation resistance in medulloblastoma with potential theraputic implications.
 Citation Format: Bethany Veo, Dong Wang, John Desisto, Rajeev Vibhakar. Mechanisms of radiation resistance in medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaed4654cb7c14fb325db4bb243c211a30ba0383" target='_blank'>
              Abstract A071 Mechanisms of radiation resistance in medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Investigating mutations, including single nucleotide variations (SNVs), gene fusions, alternative splicing and copy number variations (CNVs), is fundamental to cancer study. Recent computational methods and biological research have demonstrated the reliability and biological significance of detecting mutations from single-cell transcriptomic data. However, there is a lack of a single-cell-level database containing comprehensive mutation information in all types of cancer. Establishing a single-cell mutation landscape from the huge emerging single-cell transcriptomic data can provide a critical resource for elucidating the mechanisms of tumorigenesis and evolution. Here, we developed scTML (http://sctml.xglab.tech/), the first database offering a pan-cancer single-cell landscape of multiple mutation types. It includes SNVs, insertions/deletions, gene fusions, alternative splicing and CNVs, along with gene expression, cell states and other phenotype information. The data are from 74 datasets with 2 582 633 cells, including 35 full-length (Smart-seq2) transcriptomic single-cell datasets (all publicly available data with raw sequencing files), 23 datasets from 10X technology and 16 spatial transcriptomic datasets. scTML enables users to interactively explore multiple mutation landscapes across tumors or cell types, analyze single-cell-level mutation-phenotype associations and detect cell subclusters of interest. scTML is an important resource that will significantly advance deciphering intra-tumor and inter-tumor heterogeneity, and how mutations shape cell phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44540801ce810af79670418ea8247b9a3e3dfa5" target='_blank'>
              scTML: a pan-cancer single-cell landscape of multiple mutation types.
              </a>
            </td>
          <td>
            Haochen Li, Tianxing Ma, Zetong Zhao, Yixin Chen, Xi Xi, Xiaofei Zhao, Xiaoxiang Zhou, Yibo Gao, Lei Wei, Xuegong Zhang
          </td>
          <td>2024-10-18</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, J. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Microsatellite unstable colorectal cancer (MSI-CRC) can arise through germline mutations in mismatch repair (MMR) genes in individuals with Lynch syndrome (LS), or sporadically through promoter methylation of the MMR gene MLH1. Despite the different origins of hereditary and sporadic MSI tumours, their genomic features have not been extensively compared. A prominent feature of MMR-deficient genomes is the occurrence of many indels in short repeat sequences, an understudied mutation type due to the technical challenges of variant calling in these regions. In this study, we performed whole genome sequencing and RNA-sequencing on 29 sporadic and 14 hereditary MSI-CRCs. We compared the tumour groups by analysing genome-wide mutation densities, microsatellite repeat indels, recurrent protein-coding variants, signatures of single base, doublet base, and indel mutations, and changes in gene expression. We show that the mutational landscapes of hereditary and sporadic MSI-CRCs, including mutational signatures and mutation densities genome-wide and in microsatellites, are highly similar. Only a low number of differentially expressed genes were found, enriched to interferon-γ regulated immune response pathways. Analysis of the variance in allelic fractions of somatic variants in each tumour group revealed higher clonal heterogeneity in sporadic MSI-CRCs. Our results suggest that the differing molecular origins of MMR deficiency in hereditary and sporadic MSI-CRCs do not result in substantial differences in the mutational landscapes of these tumours. The divergent patterns of clonal evolution between the tumour groups may have clinical implications, as high clonal heterogeneity has been associated with decreased tumour immunosurveillance and reduced responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c6128f5465fd71306e14170da497710701fc42" target='_blank'>
              Lynch syndrome-associated and sporadic microsatellite unstable colorectal cancers: different patterns of clonal evolution yield highly similar tumours.
              </a>
            </td>
          <td>
            Samantha Martin, Riku Katainen, Aurora Taira, Niko Välimäki, Ari Ristimäki, T. Seppälä, L. Renkonen-Sinisalo, A. Lepistö, Kyösti Tahkola, Anne Mattila, S. Koskensalo, J. Mecklin, K. Rajamäki, Kimmo Palin, L. Aaltonen
          </td>
          <td>2024-08-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="The integration of genomics into pharmacology has revolutionized the prediction of drug responses, enabling the personalization of medicine. Advances in next-generation sequencing and pharmacogenomics allow for the identification of genetic variants that influence drug efficacy and toxicity. By analyzing genetic data, researchers can develop predictive models that optimize drug selection and dosing based on individual patient genomes. This approach is particularly relevant in cancer treatment, where targeting specific genetic mutations enhances therapeutic outcomes. However, challenges remain in the annotation of non-coding regions and ethical concerns related to personalized medicine. This paper examines the current advancements in genomic technologies, key genetic variants affecting drug response, and their clinical applications, along with the potential future directions in this field. Keywords: Genomics, Pharmacogenomics, Personalized Medicine, Next-Generation Sequencing, Drug Response Prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd02d803d7b684652b4aec052531a8ab2115eefa" target='_blank'>
              The Role of Genomics in Predicting Drug Response
              </a>
            </td>
          <td>
            Mukamurera P. Nyiramana
          </td>
          <td>2024-09-01</td>
          <td>Research Output Journal of Public Health and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, P. Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="Leukemias are a group of heterogeneous hematological malignancies driven by diverse genetic variations, and the advent of genomic sequencing technologies facilitates the investigation of genetic abnormalities in leukemia. However, these sequencing-based studies mainly focus on nuclear DNAs. Increasing evidence indicates that mitochondrial dysfunction is an important mechanism of leukemia pathogenesis, which is closely related to the mitochondrial genome variations. Here, we provide an overview of current research progress concerning mitochondrial genetic variations in leukemia, encompassing gene mutations and copy number variations. We also summarize currently accessible mitochondrial DNA (mtDNA) sequencing methods. Notably, somatic mtDNA mutations may serve as natural genetic barcodes for lineage tracing and longitudinal assessment of clonal dynamics. Collectively, these findings enhance our understanding of leukemia pathogenesis and foster the identification of novel therapeutic targets and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a51e65e882f1837d1adb0460175f694592b51dd7" target='_blank'>
              Mitochondrial genetic variations in leukemia: a comprehensive overview
              </a>
            </td>
          <td>
            Ao Zhang, Wenbing Liu, S. Qiu
          </td>
          <td>2024-09-05</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Cancer is a complex disease characterized by genetic and molecular diversity, often involving dysregulation of critical cellular pathways. Recent advances in pan-cancer research have highlighted the importance of shared oncogenic mechanisms across different cancer types, providing new avenues for therapeutic exploration. Protein kinases, particularly those involved in phagocytosis, play pivotal roles in cellular homeostasis and immune response. This study systematically examines the genetic alterations and expression profiles of protein kinases associated with phagocytosis across various cancer types, using data from The Cancer Genome Atlas (TCGA) and other publicly available resources. We analyzed single nucleotide variations (SNVs), copy number variations (CNVs), methylation patterns, and mRNA expression to identify recurring alterations and their associations with survival outcomes. Our findings reveal that MET and MERTK are the most frequently mutated genes, with missense mutations dominating across cancers. CNV analysis shows significant correlations with survival in cancers like UCEC, KIRP, and KIRC, while methylation analysis indicates cancer-specific regulatory patterns affecting gene expression. Differential expression analysis highlights distinct cancer-type-specific expression profiles, with genes like MET and BTK displaying significant variation. Crosstalk pathway analysis further reveals the involvement of these kinases in key cancer-related pathways, such as epithelial-mesenchymal transition (EMT) and apoptosis. Drug sensitivity analysis identifies potential therapeutic targets, with gene expression correlating significantly with cancer cell line responsiveness to specific compounds. These findings underscore the importance of the phagocytotic kinome in cancer biology and suggest potential therapeutic strategies targeting protein kinases to enhance immune response and improve treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec7e3ff62485c6898d190779703562a070a2f0e" target='_blank'>
              Protein Kinases in Phagocytosis: Promising Genetic Biomarkers for Cancer
              </a>
            </td>
          <td>
            Sadhika Arumilli, Hengrui Liu
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The study of chromosomal shape, characteristics, and behavior in somatic cell division (mitosis) during growth and development and in germ cell division (meiosis) during reproduction is known as cytogenetics. Many techniques can be used for cytogenetics, including fluorescent in situ hybridization (FISH), spectral karyotyping (SKY), multicolor FISH (M-FISH), microarray, and optical genome mapping (OGM). OGM is a novel genome-wide method that can identify structural variants (SVs) and copy number variants (CNVs) with only one test. Genomic structural information that is difficult to obtain with DNA sequencing can be promptly obtained with OGM, in which large molecule lengths can be mapped at a reasonable cost. OGM is increasingly being used to investigate chromosome abnormalities in genetic disorders and human cancer, but it was first utilized in genome assembly and research. According to recent research, OGM is capable of identifying every clinically significant variation seen in trials using conventional care. OGM is being utilized to identify genomic abnormalities in patients with malignancies and constitutional illnesses. It is regarded as a revolution in the field of cytogenetics. Rather than sequencing DNA, OGM relies on DNA labeling. Currently, the OGM technique with the Saphyr system from Bionano Genomics is a widely utilized platform for cytogenetic analysis. In conclusion, OGM can now be considered a highly reliable method for the identification of chromosomal abnormalities in the diagnosis of tumors and hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e6e0a2264e6747c604c9c870d801b4fcbe0131" target='_blank'>
              Cytogenetics and the Revolution of Optical Genome Mapping in the Diagnosis of Diseases.
              </a>
            </td>
          <td>
            Osama M Al-Amer, Yahya Khubrani
          </td>
          <td>2024-09-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="TP53 functions as a central regulator in response to DNA damage and other cell stressors by inducing the expression of many protective pathways such as cell cycle arrest and apoptosis. Consequently, this gene is often found disrupted in human cancers. Elephants are a particularly interesting species for the study of cancer, by virtue of their large number of cell divisions and long lives yet low incidence of cancer. Elephants also possess multiple retrogene copies of TP53, which have previously been shown to induce strong cellular responses to DNA damage. However, most previous studies have largely focused only on African elephant TP53 retrogenes and often in non-native backgrounds. Here we generated CRISPR-Cas9 knockouts of TP53, all 29 TP53 retrogenes, or both in combination in Asian elephant fibroblasts. We find that while there is considerable overlap in the DNA damage responses of the TP53 and retrogene knockouts, there are also many unique pathways enriched in both. In particular, the retrogene knockouts exhibit strong enrichment of many extracellular pathways suggesting they may play a large role in the tumor microenvironment and mitigating metastatic growth. We also find that only a small fraction of these 29 retrogenes appear to be expressed across a variety of tissues and identify three loci that are likely driving this response. This work shows for the first time the transcriptomic effect of these retrogenes within their native background and establishes a foundation for future research into the relative contributions of these genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01cd22e9caf6290532e5ca297543c409aee6bcfc" target='_blank'>
              Engineering Asian elephant TP53: TP53 retrogene knockouts activate common and unique cancer-relevant pathways
              </a>
            </td>
          <td>
            Emil Karpinski, Nikil Badey, Esther Mintzer, Asaf Ashkenazy-Titelman, George M. Church
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC), a leading cause of cancer mortality in the United States, presents significant treatment challenges due to its late diagnosis and poor prognosis. Despite advances, the five-year survival rates remain dismally low, with only a fraction of patients eligible for potentially curative surgical interventions. This review aims to comprehensively examine the current landscape of targeted therapies in PDAC, focusing on recent developments in precision medicine approaches. We explore various molecular targets, including KRAS mutations, DNA damage repair deficiencies, mismatch repair pathway alterations, and rare genetic fusions. The review discusses emerging therapies, such as PARP inhibitors, immune checkpoint inhibitors, and novel targeted agents, like RET and NTRK inhibitors. We analyze the results of key clinical trials and highlight the potential of these targeted approaches in specific patient subgroups. Recent developments in PDAC research have emphasized precision oncology, facilitated by next-generation sequencing and the identification of genetic and epigenetic alterations. This approach tailors treatments to individual genetic profiles, improving outcomes and reducing side effects. Significant strides have been made in classifying PDAC into various subtypes, enhancing therapeutic precision. The identification of specific mutations in genes like KRAS, along with advancements in targeted therapies, including small molecule inhibitors, offers new hope. Furthermore, emerging therapies targeting DNA repair pathways and immunotherapeutic strategies also show promising results. As research evolves, integrating these targeted therapies with conventional treatments might improve survival rates and quality of life for PDAC patients, underscoring the shift towards a more personalized treatment paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0623bca1b0876fdbb0324db3ab4221bfe60d5eb8" target='_blank'>
              Targeted Therapies in Pancreatic Cancer: A New Era of Precision Medicine
              </a>
            </td>
          <td>
            Bingyu Li, Qiong Zhang, Claire Castaneda, Shelly Cook
          </td>
          <td>2024-09-25</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, weisen fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Cancer remains a global health challenge, characterized not just by uncontrolled cell proliferation but also by the complex metabolic reprogramming that underlies its development and progression. Objectives: This review delves into the intricate relationship between cancer and its metabolic alterations, drawing an innovative comparison with the cosmological concepts of dark matter and dark energy to highlight the pivotal yet often overlooked role of metabolic reprogramming in tumor evolution. Methods: It scrutinizes the Warburg effect and other metabolic adaptations, such as shifts in lipid synthesis, amino acid turnover, and mitochondrial function, driven by mutations in key regulatory genes. Results: This review emphasizes the significance of targeting these metabolic pathways for therapeutic intervention, outlining the potential to disrupt cancer’s energy supply and signaling mechanisms. It calls for an interdisciplinary research approach to fully understand and exploit the intricacies of cancer metabolism, pointing toward metabolic reprogramming as a promising frontier for developing more effective cancer treatments. Conclusion: By equating cancer’s metabolic complexity with the enigmatic nature of dark matter and energy, this review underscores the critical need for innovative strategies in oncology, highlighting the importance of unveiling and targeting the “dark energy” within cancer cells to revolutionize future therapy and research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31079ef8c6b8d99084156d8e6e74525f2dc79cf1" target='_blank'>
              Insights into Metabolic Reprogramming in Tumor Evolution and Therapy
              </a>
            </td>
          <td>
            Ching-Feng Chiu, Jonathan Jaime G. Guerrero, R. Regalado, Ma. Joy B. Zamora, Jiayan Zhou, K. Notarte, Yu-Wei Lu, Paolo C. Encarnacion, Cidne Danielle D. Carles, Edrian M. Octavo, Dan Christopher I. Limbaroc, Charupong Saengboonmee, Shih-Yi Huang
          </td>
          <td>2024-10-17</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genetic mechanisms have been at the forefront of our exploration into the substrate of adaptive evolution and phenotypic diversification. However, genetic variation only accounts for a fraction of phenotypic variation. In the last decade, the significance of RNA modification mechanisms has become more apparent in the context of organismal adaptation to rapidly changing environments. RNA m6A methylation, the most abundant form of RNA modification, is emerging as a potentially significant player in various biological processes. Despite its fundamental function to regulate other major post-transcriptional mechanisms such as microRNA and alternative splicing, its role in ecology and evolution has been understudied. This review highlights the potential importance of m6A RNA methylation in ecological adaptation, emphasising the need for further research, especially in natural systems. We focus on how m6A not only affects mRNA fate but also influences miRNA-mediated gene regulation and alternative splicing, potentially contributing to organismal adaptation. The aim of this review is to synthesise key background information to enhance our understanding of m6A mechanisms driving species survival in dynamic environments and motivate future research into the dynamics of adaptive RNA methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf43da54f3833e8823168fb9ac8931ee33e9a607" target='_blank'>
              Emerging Orchestrator of Ecological Adaptation: m6A Regulation of Post-Transcriptional Mechanisms.
              </a>
            </td>
          <td>
            E. Ahi, Pooja Singh
          </td>
          <td>2024-10-05</td>
          <td>Molecular ecology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite the crucial role of T cell clones in the anti-tumor activity, their characterization and association with clinical outcome following immune checkpoint inhibitors (ICI) is lacking. Here we analyzed paired single-cell RNA-sequencing/T-cell receptor sequencing of 767,606 T cells from 460 samples spanning 6 cancer types. We found a robust signature of response based on expanded CD8+ clones that differentiates between responders and non-responders. Analysis of persistent clones showed transcriptional changes that are differentially induced by therapy in the different response groups, suggesting an improved reinvigoration capacity in responding patients. Moreover, a gene trajectory analysis revealed changes in the pseudo-temporal state of de-novo clones that are associated with response to therapy. Lastly, we found that clones shared between tumor and blood are more abundant in non-responders and execute distinct transcriptional programs. Overall, our results highlight differences in clonal transcriptional states that are linked to patient response, offering valuable insights into the mechanisms driving effective anti-tumor immunity. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3504c21c4fc5c6f9ee1a5a18b7688458b8f59956" target='_blank'>
              Single-cell meta-analysis of T cells reveals clonal dynamics of response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            Ofir Shorer, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28b08db9038d5c3400d445c5aadfc149da32c65" target='_blank'>
              Centromeres in cancer: Unraveling the link between chromosomal instability and tumorigenesis.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Ahmad Nazari, Noushin Parsania, Paria Behboodi, Seyedeh Sara Ketabi, Pegah Razmjouei, Farnoosh Farzam, Seyyed-Ghavam Shafagh, Mohsen Nabi Afjadi
          </td>
          <td>2024-10-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d61b6a296f0f4f8539e1a19d49c6d27ddf36337d" target='_blank'>
              Deciphering the role of sphingolipid metabolism in the immune microenvironment and prognosis of esophageal cancer via single-cell sequencing and bulk data analysis
              </a>
            </td>
          <td>
            Rongzhang He, Jing Tang, Haotian Lai, Tianchi Zhang, Linjuan Du, Siqi Wei, Ping Zhao, Guobin Tang, Jie Liu, Xiufang Luo
          </td>
          <td>2024-09-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Even though epigenetic factors contribute to oncogenesis, most human cancer models still assume that disease originates from driver DNA mutations. Thus, it is still unclear if non-genetic mechanisms are sufficient to trigger malignant transformation. Special AT-rich binding protein 2 (SATB2) is a chromatin organizer that brings distal DNA elements into close proximity, thus remodeling chromatin structures to reprogram cell-specific and/or developmentally-sensitive gene networks. Here, we discover that SATB2 generates a co-expressed circ3915 RNA that is translated into a peptide and co-locates with SATB2 in the cell nucleus. Ectopic SATB2 or circ3915 over- expression rearranges global chromatin accessibility, generates KRAS- and NFE2L2-like oncogenic gene expression patterns, and transforms lung epithelial cells independent of driver mutations. Thus, oncogenic pathways can be activated in mammalian cells without pre-disposing mutations in oncogenes or epigenetic regulators. Graphical abstract:">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2809e3fd65e4f1849899e9487a4c550596b35a99" target='_blank'>
              SATB2 and circ3915 RNA chromatin dysregulation drive KRAS-like oncogenic transformation
              </a>
            </td>
          <td>
            Rebekah Eleazer, Smitha George, Luke Shoemaker, Wesley N. Saintilnord, Kin H. Lau, Darrell Chandler, Yvonne Fondufe-Mittendorf
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a14bfe5f9c18048663459b9f007d9027d3e70b" target='_blank'>
              Gene editing of NCF1 loci is associated with homologous recombination and chromosomal rearrangements
              </a>
            </td>
          <td>
            Federica Raimondi, Kah Mun Siow, Dominik Wrona, Carla Fuster-García, Oleksandr Pastukhov, Michael Schmitz, Katja Bargsten, Lucas Kissling, D. C. Swarts, G. Andrieux, Toni Cathomen, Ute Modlich, M. Jinek, Ulrich Siler, Janine Reichenbach
          </td>
          <td>2024-10-09</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="T cells recognize an immense spectrum of pathogens to initiate immune responses by means of a large repertoire of T cell receptors (TCRs) that arise from somatic rearrangements of variable, diversity and joining gene segments at the TCR loci. These gene segments have emerged from a limited number of ancestral genes through a series of gene duplication events, resulting in a greatly variable number of such genes across different species. Apart from the complete V(D)J gene annotations in the human and mouse reference assemblies, little is known about the structure of TCR loci in other species. Here, we performed a comprehensive comparison of the TCRα and TCRβ gene segment clusters in mice and three of its closely related sister species. We show that the TCRα variable gene cluster is frequently rearranged, leading to deletions and sequence inversions in this region. The resulting complexity of TCR loci severely complicates the assembly of these loci and the annotation of gene segments. By jointly utilizing genomic and transcriptomic data, we show that in Mus musculus castaneus the variable gene cluster at the α locus has undergone a recent major locus contraction, leading to the loss of 74 variable gene segments. Additionally, we validated the expression of functional variable genes, including atypical ones with inverted orientation relative to other such segments. Disentangling the fine-scale structure of TCR loci in different species can provide valuable insights in the evolution and diversity of TCR repertoires.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/050dd2ec4e870352e83af91f6ed80d0e6588c1e2" target='_blank'>
              Distinct evolution at TCRα and TCRβ loci in the genus Mus
              </a>
            </td>
          <td>
            Moritz Peters, Volker Soltys, Dingwen Su, Y. F. Chan
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The Lucena 1 cell line, derived from the human chronic myeloid leukemia cell line K562 under selective pressure of vincristine supplementation, exhibits multidrug resistance (MDR). This study aims to explore and elucidate the underlying mechanisms driving MDR in the Lucena 1 cell line. A proteomic analysis comparing K562 and Lucena 1 revealed qualitative differences, with a focus on the ATP-dependent efflux pump, Translocase ABCB1, a key contributor to drug resistance. Tubulin analysis identified two unique isoforms, Tubulin beta 8B and alpha chain-like 3, exclusive to Lucena 1, potentially influencing resistance mechanisms. Additionally, the association of Rap1A and Krit1 in cytoskeletal regulation and the presence of STAT1, linked to the urea cycle and tumor development, offered insights into Lucena 1’s distinctive biology. The increased expression of carbonic anhydrase I suggested a role in pH regulation. The discovery of COP9, a tumor suppressor targeting p53, further highlighted the Lucena 1 complex molecular landscape. This study offers new insights into the MDR phenotype and its multifactorial consequences in cellular pathways. Thus, unraveling the mechanisms of MDR holds promise for innovating cancer models and antitumor targeted strategies, since inhibiting the P-glycoprotein (P-gp)/ABCB1 protein is not always an effective approach given the associated treatment toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222b6fc76a9342b783ed19fca22fd98cd9e78a53" target='_blank'>
              Proteomic Dynamics of Multidrug Resistance Mechanisms in Lucena 1 Cell Line
              </a>
            </td>
          <td>
            Emidio Beraldo-Neto, Fernanda Cardoso Amador, Karolina Rosa Fernandes, Giselle Zenker Justo, José Thalles Lacerda, M. A. Juliano
          </td>
          <td>2024-08-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric malignancy. Based on gene expression profiling, B-ALL can be classified into distinct transcriptional subtypes with differing disease outcomes. Many of these transcriptional subtypes are defined by mutations in transcription factors and chromatin-modifying enzymes, but how such diverse mutations lead to distinct transcriptional subtypes remains unclear. To illuminate the chromatin regulatory landscape in B-ALL, we analyzed 3D genome organization, open chromatin, and gene expression in 53 primary patient samples. At the level of 3D genome organization, we identified chromatin interactions that vary across transcriptional subtypes. These sites of variable 3D chromatin interactions correlate with local gene expression changes and are enriched for core drivers of B-ALL observed in genome-wide CRISPR knock-out screens. Sites of variable 3D genome interactions are frequently shared across multiple transcriptional subtypes and are enriched for open chromatin sites found in normal B-cell development but repressed in mature B-cells. Within an individual patient sample, the chromatin landscape can resemble progenitor chromatin states at some loci and mature B-cell chromatin at others, suggesting that the chromatin in B-ALL patient tumor cells is in a partially arrested immature state. By analyzing transcriptomic data from large cohorts of B-ALL patients, we identify gene expression programs that are shared across transcriptional subtypes, associated with B-cell developmental stages, and predictive of patient survival. In combination, these results show that the 3D genome organization of B-ALL reflects B-cell developmental stages and helps illustrate how B-cell developmental arrest interacts with transcriptional subtypes to drive B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1e7461bf425fea64b8e5097538780b0bffaf" target='_blank'>
              Heterogeneity in chromatin structure drives core regulatory pathways in B-cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Arko Sen, Zhichao Xu, Sélène T. Tyndale, Jean Yasis, Chae Yun Cho, Rosalind Bump, Sahaana Chandran, Linda Luo, Yi Fu, Lillian Kay Petersen, Max Shokhirev, D. J. Kuo, G. McVicker, Jesse R. Dixon
          </td>
          <td>2024-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Background: Immune checkpoint blockades (ICBs) lack clinical efficacy in all-coming pediatric solid tumors. Gene expression-based classification of the tumor immune environment (TiME) can identify a subset of pediatric tumors with a rich immune environment, potentially actionable by ICBs. Methylation reprogramming is a major player in TiME modeling and ICB resistance, its role in pediatric tumors has been little explored. Hypothesis: dual-omic classification, combining DNA-methylation and RNA-sequencing, can identify tumors with epigenetically altered TiME, potentially responsible for ICB resistance. Objective: identify and characterize pediatric solid tumors with epigenetically altered immune phenotypes. Methods: We studied RNA-sequencing and DNA-methylation quantitative data from pediatric extra-cranial solid tumors. We used similarity network fusion (SNF) to individualize immune phenotypes. A preselected customized panel of 1226 immune genes, and their corresponding probes, were used for dual-omics clustering. Differential expression (DE) and methylation (DM) were performed between clusters using volcano3D. Scatter plots visualized the cis-regulation of probe-gene pairs with significant DE-DM. Gene set enrichment analyses (GSEA) were performed with clusterProfiler and gometh packages for DE and DM, respectively. Adjusted p values <0.01 were retained. Results: 184 samples were included. SNF clustering identified 3 phenotypes regrouping 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 was characterized by a low expression of immune genes (cold phenotype), cl2 by overexpression in immune genes (hot phenotype), and cl3 by global hypermethylation (epigenetically altered phenotype). Both cl2 and cl3 overexpressed genes of immune checkpoints (CD274, PDCD1) and T-cell activator chemokines (CXCL9, CXCL10), central for ICB sensitivity. However, only cl2 overexpressed major histocompatibility complex (MHC) class I-II genes and their regulators (CIITA, TAP2), essential for antigen (Ag)-presenting machinery and immune recognition. Cl2 and 3 were also enriched for T-, B-cell and pro-inflammatory interferon gamma signaling pathways, known as ICB sensitivity biomarkers. Only cl2 was enriched in Ag processing and presentation pathway, confirming prior observation. DE-DM correlation showed that DNA-methylation programming induced overexpression of most immune genes (78%) in cl2, including CIITA, CXCR3 and MHC genes, when most immune genes (63%) were repressed by methylation in cl3. Methylation-based GSEA confirmed the epigenetic regulation of T- and B-cell signaling in cl2 and cl3, and for Ag presentation in cl2 only. Conclusion We demonstrated that DNA-methylation can reshape the TiME of pediatric solid tumors. We identified a subset of tumors with epigenetically altered immune phenotype characterized by inflamed immune environment but altered for Ag presentation by methylation reprogramming. Future studies should investigate methylation modulators to reverse these mechanisms and enable ICB sensitivity.
 Citation Format: Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-Cazavieille, Charles Joly-Beauparlant, Lara Herrmann, Emeric Texeraud, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Vincent-Philippe Lavallée, Thai Hoa Tran, Sonia Cellot, Daniel Sinnett, Arnaud Droit, Raoul Santiago. Dual-omic characterization of pediatric solid tumors identified a subset of tumors with epigenetically altered immune phenotype [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B057.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a9f318a4bf2de77816f3008a669473ec30397b" target='_blank'>
              Abstract B057: Dual-omic characterization of pediatric solid tumors identified a subset of tumors with epigenetically altered immune phenotype
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Lara Herrmann, Emeric Texeraud, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Vincent-Philippe Lavallée, T. Tran, Sonia Cellot, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Identification of somatic driver mutations in the noncoding genome remains challenging. To comprehensively characterize noncoding driver mutations for pancreatic ductal adenocarcinoma (PDAC), we first created genome-scale maps of accessible chromatin regions (ACRs) and histone modification marks (HMMs) in pancreatic cell lines and purified pancreatic acinar and duct cells. Integration with whole-genome mutation calls from 506 PDACs revealed 314 ACRs/HMMs significantly enriched with 3,614 noncoding somatic mutations (NCSMs). Functional assessment using massively parallel reporter assays (MPRA) identified 178 NCSMs impacting reporter activity (19.45% of those tested). Focused luciferase validation confirmed negative effects on gene regulatory activity for NCSMs near CDKN2A and ZFP36L2. For the latter, CRISPR interference (CRISPRi) further identified ZFP36L2 as a target gene (16.0 - 24.0% reduced expression, P = 0.023 - 0.0047) with disrupted KLF9 binding likely mediating the effect. Our integrative approach provides a catalog of potentially functional noncoding driver mutations and nominates ZFP36L2 as a PDAC driver gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db992992dda460b3aa1dd358fc40078a768ec148" target='_blank'>
              Large-scale multi-omic analysis identifies noncoding somatic driver mutations and nominates ZFP36L2 as a driver gene for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jun Zhong, Aidan O'Brien, Minal Patel, Daina Eiser, Michael Mobaraki, I. Collins, Li Wang, Konnie Guo, Thucnhi Truongvo, Ashley A. Jermusyk, Maura J. O’Neill, Courtney D. Dill, A. Wells, M. Leonard, J. Pippin, Struan F. A. Grant, Tongwu Zhang, T. Andresson, Katelyn E. Connelly, Jianxin Shi, H. E. Arda, Jason W. Hoskins, L. Amundadottir
          </td>
          <td>2024-09-24</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827407d1ee135266c2bdec8a7b4b96ce85a8df0f" target='_blank'>
              A genome-wide CRISPR/Cas9 knockout screen identifies SEMA3F gene for resistance to cyclin-dependent kinase 4 and 6 inhibitors in breast cancer.
              </a>
            </td>
          <td>
            Yuko Kawai, Aiko Nagayama, Kazuhiro Miyao, Makoto Takeuchi, Takamichi Yokoe, Tomoe Kameyama, Xinyue Wang, T. Seki, Maiko Takahashi, T. Hayashida, Yuko Kitagawa
          </td>
          <td>2024-10-01</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Genetic variants associated with autoimmune diseases are highly enriched within putative cis-regulatory regions of CD4+ T cells, suggesting that they alter disease risk via changes in gene regulation. However, very few genetic variants have been shown to affect T cell gene expression or function. We tested >18,000 autoimmune disease-associated variants for allele-specific expression using massively parallel reporter assays in primary human CD4+ T cells. The 545 expression-modulating variants (emVars) identified greatly enrich for likely causal variants. We provide evidence that many emVars are mediated by common upstream regulatory conduits, and that putative target genes of primary T cell emVars are highly enriched within a lymphocyte activation network. Using bulk and single-cell CRISPR-interference screens, we confirm that emVar-containing T cell cis-regulatory elements modulate both known and novel target genes that regulate T cell proliferation, providing plausible mechanisms by which these variants alter autoimmune disease risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4697d558dbf5487b1c63a4626bb50cef2fee4eae" target='_blank'>
              Linking candidate causal autoimmune variants to T cell networks using genetic and epigenetic screens in primary human T cells
              </a>
            </td>
          <td>
            Ching-Huang Ho, Maxwell A. Dippel, Meghan S. McQuade, Arpit Mishra, Stephan Pribitzer, LeAnn P. Nguyen, Samantha Hardy, Harshpreet Chandok, Florence M. Chardon, Troy A. McDiarmid, Hannah A. DeBerg, Jane H. Buckner, Jay Shendure, Carl G. de Boer, Michael H. Guo, Ryan Tewhey, John P. Ray
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the deadliest and most common form of brain tumour. The average length of survival for GBM patients remains low, in part, due to a lack of targeted therapeutics. Identiﬁcation and validation of novel therapeutic targets is therefore crucial. Chomobox 2 (CBX2; a component of polycomb repressive complex 1 (PRC1)), represses gene expression and has a pro-oncogenic role in several aggressive cancer subtypes. CBX2 expression is elevated in GBM, however, very little is known about its role in this deadly disease. We aim to assess the phenotypic and gene expression effects of preventing CBX2-chromatin interaction in patient-derived GBM cell models, and to determine the functional mechanisms by which CBX2 promotes GBM progression; thereby determining its potential as a novel therapeutic target.



 Crucial to impactful target validation is the use of translationally relevant models, this study utilises a 3D spheroid model, using patient-derived cells, which more closely recapitulates the in vivo tumour microenvironment. GBM cell monolayers and spheroids were treated with SW2_152F, a selective CBX2 inhibitor. The phenotypic effects of CBX2 inhibition were assessed by cell count, CellTiter-Glo viability assays, ﬂuorescence microscopy, and ﬂow cytometry. Genes and biological pathways differentially regulated following CBX2 inhibition were assessed by RNA-sequencing and Gene Set Enrichment Analysis (GSEA).



 Inhibition of CBX2 induced cell death and reduced cell viability in both 2D and 3D GBM cultures. RNA-seq and GSEA of SW2_152F-treated cells identiﬁed dysregulation of gene signatures involved in pro-oncogenic MYC and E2F signalling pathways, and dysregulation of genes associated with G2/M cell cycle progression.



 Preventing CBX2-chromatin interaction induces GBM cell death. Further research will identify CBX2- chromatin binding sites to characterise the activity and direct regulatory role of CBX2 in GBM. Furthermore, we will assess the phenotypic and gene expression effects of CBX2 inhibition and depletion on ex vivo maintained GBM tissue biopsies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41607adec8f62477f0d798d3212dbec12fcc049c" target='_blank'>
              THE EPIGENETIC REGULATORY PROTEIN CBX2 AS A NOVEL THERAPEUTIC TARGET FOR GLIOBLASTOMA
              </a>
            </td>
          <td>
            Ms Danielle Marsh, Prof John Greenman, Dr Mark Wade
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chemical screens across hundreds of cell lines have shown that the drug sensitivities of human cancers can vary by genotype or lineage. However, most drug discovery studies have relied on culture media that poorly reflect metabolite levels in human blood. Here, we perform drug screens in traditional and Human Plasma–Like Medium (HPLM). Sets of compounds that show conditional anticancer activity span different phases of global development and include non-oncology drugs. Comparisons of the synthetic and serum-derived components that comprise typical media trace sets of conditional phenotypes to nucleotide synthesis substrates. We also characterize a unique dual mechanism for brivudine, a compound approved for antiviral use. Brivudine selectively impairs cell growth in low folate conditions by targeting two enzymes involved in one-carbon metabolism. Cataloged gene essentiality data further suggest that conditional phenotypes for other compounds are linked to off-target effects. Our findings establish general strategies for identifying drug-nutrient interactions and mechanisms of action by exploiting conditional lethality in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52075b84c4effc26f41a203f90514fe9f3c95c9f" target='_blank'>
              Conditional lethality profiling reveals anticancer mechanisms of action and drug-nutrient interactions
              </a>
            </td>
          <td>
            K. Flickinger, Kelli M. Wilson, Nicholas J. Rossiter, Andrea L. Hunger, Paresh V Vishwasrao, Tobie D. Lee, Carlos A. Mellado Fritz, Rebecca M Richards, Matthew D. Hall, Jason R. Cantor
          </td>
          <td>2024-10-04</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a593c69280acd8e1ddd89d1994ea815f977cea" target='_blank'>
              Chromosomal instability: a key driver in glioma pathogenesis and progression
              </a>
            </td>
          <td>
            Adele Mazzoleni, W. Awuah, Vivek Sanker, H. Bharadwaj, N. Aderinto, J. K. Tan, H. Huang, Jeisun Poornaselvan, Muhammad Hamza Shah, O. Atallah, Aya Tawfik, Mohamed Elmanzalawi, Sama Hesham Ghozlan, T. Abdul-Rahman, Jeremiah Adepoju Moyondafoluwa, Athanasios Alexiou, Marios Papadakis
          </td>
          <td>2024-09-04</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Background: Hispanic American (HA) men with prostate cancer (PC) are more likely to have more aggressive and advanced diseases and greater mortality rates compared to non-Hispanic white (NHW) men. Resistance to the current anticancer therapy is the major hurdle that leads to poor clinical outcomes. Hedgehog, Notch, and Wnt signaling pathways play a crucial role in development, progression, and resistance to the various anticancer therapies. Here, we characterized the enrichment of Hedgehog, Notch, and Wnt signaling pathways and molecular tumors differences between HA and NHW patients. Methods: Our data consist of 88 PC tumors samples (HA=34, NHW=54) obtained from treatment-naive metastatic hormone-sensitive prostate cancer (mHSPC) that were analyzed by next-generation sequencing (592, NextSeq; Whole Exome Sequencing, NovaSeq) and Whole Transcriptome Sequencing (WTS; NovaSeq) (Caris Life Sciences, Phoenix, AZ). Pathway enrichment was determined by Gene Set Enrichment Analysis (GSEA, Broad Institute) by measurable TPM (transcripts per kilobase million). The medical records were reviewed in a deidentified fashion for clinical features. Statistical significance was determined using chi-square and Mann-Whitney U (p<0.05). Results: HA PC had enrichment of Hedgehog signaling (NES: 1.45, FDR=0.04), Notch signaling (NES: 1.41, FDR=0.07), and Wnt/β-catenin signaling (NES: 1.38, FDR=0.11) pathways compared to NHW tumors. HA tumors had higher expression of genes (HES1, Notch1, WNT1, WNT2; FC:1.1-2.5) associated with crosstalk between Hedgehog, Notch, and Wnt/β- catenin signaling pathways (all p<0.05). Moreover, HA tumors had higher expression of stemness (ALDH1A2, ALDH1A1, PROM1; FC: 1.7-3.0), drug efflux (ABCC1, ABCB1; FC: 1.1- 1.5), epithelial to mesenchymal transition (EMT) (TWIST1, MMP2; FC: 1.1-1.5), cell cycle (CCND2; FC: 1.2), inhibition of apoptosis (BIRC2, XIAP; FC: 1.1) and epigenetic (EP300, TET1, TET2; FC: 1.3-1.4) related genes compared to NHW tumors (all p<0.05). HA PC had higher frequency of Hedgehog and Wnt/β-catenin associated SPOP (16% vs 6.6%, p=0.2) and CTNNB1 (12% vs 2.2%, p=0.1) mutation compared to NHW PC. Conclusion: These results demonstrate that tumors from HA patients had enrichment of oncogenic Hedgehog, Notch, and Wnt/β-catenin signaling pathways, higher expression of stemness, drug efflux, EMT, cell cycle, inhibition of apoptosis and epigenetic-related genes, and mutations directly involved in activating the Hedgehog and Wnt/β-catenin signaling pathways. Together, these findings suggest that tumors from HA patients are enriched for gene signatures that promote cancer progression and therapy resistance. Additional research on collaboration and crosstalk between Hedgehog, Notch, and Wnt/β-catenin signaling pathways may help in designing better therapeutic strategies for HA men with mHSPC.
 Citation Format: Alejandro Recio-Boiles, Sachin Kumar Deshmukh, Ricardo J. Estrada-Mendizabal, Kenneth Barker, Sharon Wu, Joanne Xiu, Milan Radovich, Elisabeth I. Heath, Rana McKay, Juan Chipollini, Cynthia K. Miranti. Defining the role of Hedgehog, Notch, and Wnt signaling pathways in aggressive lethal prostate cancer of Hispanic men [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C100.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084e712d5ece7637137ae910e70858f7d21562a6" target='_blank'>
              Abstract C100: Defining the role of Hedgehog, Notch, and Wnt signaling pathways in aggressive lethal prostate cancer of Hispanic men
              </a>
            </td>
          <td>
            Alejandro Recio-Boiles, S. Deshmukh, Ricardo J. Estrada-Mendizabal, Kenneth Barker, Sharon Wu, Joanne Xiu, M. Radovich, Elisabeth I. Heath, Rana Mckay, Juan Chipollini, Cynthia K. Miranti
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K C Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K J Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44ac4365f3d537cc727149bd2f0c2b4a261aed5" target='_blank'>
              The neuroendocrine transition in prostate cancer is dynamic and dependent on ASCL1.
              </a>
            </td>
          <td>
            Rodrigo Romero, Tinyi Chu, Tania J González Robles, Perianne Smith, Yubin Xie, Harmanpreet Kaur, Sara Yoder, Huiyong Zhao, Chenyi Mao, Wenfei Kang, Maria V. Pulina, K. Lawrence, Anuradha Gopalan, S. Zaidi, K. Yoo, Jungmin Choi, Ning Fan, Olivia Gerstner, W. Karthaus, E. deStanchina, Kelly V. Ruggles, P. Westcott, R. Chaligné, D. Pe’er, C. Sawyers
          </td>
          <td>2024-10-11</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The emergence of novel cell types fuels evolutionary innovations and contributes to the diversity of life forms and their morphological and functional traits. Cell types are fundamental functional units of multicellular organisms defined by their specific gene expression programs. The evolution of these transcriptional programs is driven by genetic changes, such as gene co-option and cis- regulatory evolution, known to facilitate the assembly or rewiring of molecular networks and give rise to new cell types with specialized functions. However, the role of novel genes in this complex evolutionary process is underexplored. Here, we examine the trends in skeletal cell type evolution with a focus on lineage-specific genes. We find that immature chondrocytes express the oldest transcriptome and resemble ancestral skeletogenic cell type, supporting the existence of a conserved genetic program for cartilage development in bilaterians. The subsequent acquisition of lineage-restricted genes led to the individuation of the ancient gene expression program and powered the emergence of osteoblasts and hypertrophic chondrocytes. We found a significant enrichment of Vertebrate-specific genes in osteoblasts and Gnathostome-specific genes in hypertrophic chondrocytes. By identifying the functional properties of the recruited genes, coupled with the recently discovered fossil evidence, our findings challenge the long-standing view on the evolution of vertebrate skeletal structures and suggest that endochondral ossification and chondrocyte hypertrophy evolved already in the last common ancestors of gnathostomes. Finally, our findings highlight the critical role of novel genes in shaping cellular diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/677ea0e1eca83b3b7673677deb5e3a705075d5bc" target='_blank'>
              Evolutionary trends in the emergence of skeletal cell types
              </a>
            </td>
          <td>
            Amor M. Damatac, Sara Koska, Kristian K Ullrich, Tomislav Domazet-Lošo, Alexander Klimovich, M. Kaucká
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="TP53, the most frequently mutated gene in human cancer, encodes a transcriptional activator that induces myriad downstream target genes. Despite the importance of p53 in tumor suppression, the specific p53 target genes important for tumor suppression remain unclear. Recent studies have identified the p53-inducible gene Zmat3 as a critical effector of tumor suppression, but many questions remain regarding its p53-dependence, activity across contexts, and mechanism of tumor suppression alone and in cooperation with other p53-inducible genes. To address these questions, we used Tuba-seqUltra somatic genome editing and tumor barcoding in a mouse lung adenocarcinoma model, combinatorial in vivo CRISPR/Cas9 screens, meta-analyses of gene expression and Cancer Dependency Map data, and integrative RNA-sequencing and shotgun proteomic analyses. We established Zmat3 as a core component of p53-mediated tumor suppression and identified Cdkn1a as the most potent cooperating p53-induced gene in tumor suppression. We discovered that ZMAT3/CDKN1A serve as near-universal effectors of p53-mediated tumor suppression that regulate cell division, migration, and extracellular matrix organization. Accordingly, combined Zmat3-Cdkn1a inactivation dramatically enhanced cell proliferation and migration compared to controls, akin to p53 inactivation. Together, our findings place ZMAT3 and CDKN1A as hubs of a p53-induced gene program that opposes tumorigenesis across various cellular and genetic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5afaeddf5354538c432a3c29c86495b8e7485c" target='_blank'>
              Integrative multiomic approaches reveal ZMAT3 and p21 as conserved hubs in the p53 tumor suppression network
              </a>
            </td>
          <td>
            A. Boutelle, Aicha R. Mabene, David Yao, Haiqing Xu, Mengxiong Wang, Yuning J. Tang, Steven S. Lopez, Sauradeep Sinha, J. Demeter, Ran Cheng, Brooks A. Benard, Liz J. Valente, Alexandros P. Drainas, Martin Fischer, Ravi Majeti, D. Petrov, Peter K. Jackson, Fan Yang, M. Winslow, Mike C Bassik, L. Attardi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Simple Summary Phenotypic diversity in tumors makes treatment difficult and leads to poor prognosis. A better understanding of what causes intratumoral heterogeneity and differences in cells that are within the same tumor will advance therapeutic options and improve patient outcomes. This review will summarize recent studies that have examined the various cancer cell phenotypes along the epithelial–mesenchymal spectrum present in tumors and how epigenetic factors and the mechanical properties of tumor tissue mediate these phenotypes. This review will also discuss the therapeutic implications of intratumoral heterogeneity, such as drug resistance and tumor recurrence, and highlight opportunities to address current limitations in cancer treatments. Abstract The tumor microenvironment comprises various cell types and experiences dynamic alterations in physical and mechanical properties as cancer progresses. Intratumoral heterogeneity is associated with poor prognosis and poses therapeutic challenges, and recent studies have begun to identify the cellular mechanisms that contribute to phenotypic diversity within tumors. This review will describe epithelial–mesenchymal (E/M) plasticity and its contribution to phenotypic heterogeneity in tumors as well as how epigenetic factors, such as histone modifications, histone modifying enzymes, DNA methylation, and chromatin remodeling, regulate and maintain E/M phenotypes. This review will also report how mechanical properties vary across tumors and regulate epigenetic modifications and E/M plasticity. Finally, it highlights how intratumoral heterogeneity impacts therapeutic efficacy and provides potential therapeutic targets to improve cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b710fff2d564553c94ffb93440614a32c0f2e22e" target='_blank'>
              Epithelial–Mesenchymal Plasticity and Epigenetic Heterogeneity in Cancer
              </a>
            </td>
          <td>
            Jessica L. Sacco, Esther W. Gomez
          </td>
          <td>2024-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) stands as the most aggressive tumor of the CNS. Current treatment typically involves surgical intervention followed by cycles of Temozolomide (TMZ) chemotherapy in combination with radiotherapy. However, the efficacy of these treatments remains limited due to GBM’s ability to adapt and resist therapy. TMZ treatment leads to significant alterations in the cell cycle of GBM cells, with a decrease in cells in the G0-G1 phase and an increase in cells in the S or G2 phases. This shift in cell cycle distribution suggests a dual response: re-entry of tumor cells into the cell cycle and concurrent arrest induced by TMZ, potentially contributing to tumor resistance and disease progression. Here, we leverage the variable patterns of Ki-67, a marker of cellular proliferation, which expression fluctuates throughout the cell cycle, peaking during S, G2, and M phases to evaluate cellular response to TMZ treatment across various primary cell lines and tissue samples.



 We developed an image-based approach to investigate and reconstruct the cell cycle using advanced machine learning for pattern recognition to identify the detailed cell cycle status on single cell level. To characterized primary cell lines (n=6), we performed whole-genome DNA sequencing, whole-genome methylation sequencing and RNA-sequencing. For functional read-outs, we performed large-scale TMZ treatment in a time- and concentration dependent manner in cell culture and GBM-tissue slice models.



 Our machine learning model of cell cycle reconstructions demonstrated a robust reconstruction of detailed cell cycles stages across all cell lines with the majority of cells assigned to G0-G1 stated without treatment. TMZ treatment resulted in a significant shift towards enhanced S/G2 phase (p < 0.02) in 3 of 6 tested cell lines. Pseudo-time reconstruction revealed a dose-dependent alteration of the cell cycle showing significant changes with TMZ-doses below 50µmol. Cell lines which enhanced stem-like phenotypes based on epigenetically and transcriptionally characterisation demonstrated no cell cycle alterations even in high-dose treatment conditions (>150µmol) regardless their MGMT promotor methylation status. This heterogeneity underscores the complexity of GBM and highlights the importance of personalized treatment strategies.



 Our findings provide valuable insights into GBM’s cell cycle alterations and adaptive mechanisms of resistance independent of the MGMT methylation status as a known biomarker for TMZ response. TMZ-induced cell cycle alterations can provide a novel biomarker to detect treatment response and offer a foundation for developing personalized therapeutic approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61831fb8a82ccbd2689745661a37a7f98e1edd0f" target='_blank'>
              P10.05.B ADVANCED MACHINE LEARNING TO DETECT HETEROGENEITY OF CELL CYCLE ALTERATIONS DURING TMZ CHEMOTHERAPY AT SINGLE CELL LEVEL
              </a>
            </td>
          <td>
            G. Villa, J. Kückelhaus, Y. Yabo, O. Schnell, D. Delev, J. Benotmane, H. D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The chromatins are folded into three-dimensional (3D) structures inside cells, which coordinates the regulation of gene transcription by the non-coding regulatory elements. Aberrant chromatin 3D folding has been shown in many diseases, such as acute myeloid leukemia (AML), and may contribute to tumorigenesis. The anthracycline topoisomerase II inhibitors can induce histone eviction and DNA damage. We performed genome-wide high-resolution mapping of the chemotherapeutic effects of various clinically used anthracycline drugs. ATAC-seq was used to profile the histone eviction effects of different anthracyclines. TOP2A ChIP-seq was used to profile the potential DNA damage regions. Integrated analyses show that different anthracyclines have distinct target selectivity on epigenomic regions, based on their respective ATAC-seq and ChIP-seq profiles. We identified the underlying molecular mechanism that unique anthracycline variants selectively target chromatin looping anchors via disrupting CTCF binding, suggesting an additional potential therapeutic effect on the 3D genome. We further performed Hi-C experiments, and data from K562 cells treated with the selective anthracycline drugs indicate that the 3D chromatin organization is disrupted. Furthermore, AML patients receiving anthracycline drugs showed altered chromatin structures around potential looping anchors, which linked to distinct clinical outcomes. Our data indicate that anthracyclines are potent and selective epigenomic targeting drugs and can target the 3D genome for anticancer therapy, which could be used for personalized medicine to treat tumors with aberrant 3D chromatin structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760450c07be781a1d46120b324cb67687f8de734" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A R Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f3792b41778f947365ed0dac38ff748d4c49600" target='_blank'>
              L3MBTL1, a polycomb protein, promotes Osimertinib acquired resistance through epigenetic regulation of DNA damage response in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zihe Zhang, Yongwen Li, Ruifeng Shi, Chaoyi Jia, Songlin Xu, Guangsheng Zhu, P. Cao, Hua Huang, Xuanguang Li, Hongbing Zhang, Minghui Liu, Chen Chen, Hongyu Liu, Chunsheng Kang, Jun Chen
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Ewing sarcoma is a highly aggressive pediatric cancer driven by the EWS::FLI1 (EF) fusion oncogene. Evidence suggests that EF expression levels dictate tumor cell plasticity contributing to treatment resistance and relapse. Our study aims to elucidate the transcriptional programs and phenotypes associated with distinct EF thresholds. To dynamically fine-tune EF levels, we engineered Ewing sarcoma cell line models by C-terminally tagging endogenous EF with a fluorescent (mNG) and a degron tag (dTAG). Using mNG as a quantitative proxy for EF levels, we confirmed that increasing concentrations of the dTAG ligand led to a gradual loss of the fusion protein. Already subtle modulation of EF thresholds significantly enhanced tumor cell migration. Furthermore, a 25% EF reduction was sufficient to significantly lower anchorage-independent growth in soft agar, which was completely abolished with further EF depletion. Supported by zebrafish xenograft experiments, these findings suggest that minor EF fluctuations enhance migratory properties while still retaining significant in vitro tumor-forming potential. To characterize acute transcriptional responses to variations in EF thresholds, we performed SLAM-sequencing after 3 hours of treatment with increasing dTAG concentrations. While minor reduction of the fusion protein to 76% led to immediate downregulation of EF-activated genes enriched in GGAA microsatellites, most EF-repressed targets were upregulated only when EF levels were reduced by 58% to 96%. These de-repressed targets were significantly enriched for epithelial-to-mesenchymal transition (EMT)-related pathways. Testing EF chromatin occupancy by CUT&RUN followed by motif enrichment analysis, identified several transcription factor binding motifs linked to EF-repressed genes. Promising candidates include SP1, EGR1, and TFAP2A, all of which have previously been implicated in the regulation of metastasis-related pathways. Long-term dTAG treatment (1 to 3 weeks), followed by EF rescue through dTAG ligand washout for 7 days revealed a set of metastasis-related genes that notably remained upregulated despite a full recovery of EF levels already within 24 hours of ligand washout. This suggests that transient oncogene fluctuations may cause a sustained increase in metastatic potential of Ewing sarcoma cells. Importantly, about thirty of these genes negatively correlated with overall survival in a cohort of 196 Ewing sarcoma patients and were anti-correlated with the GGAA microsatellite-driven transcriptional repressor BCL11B, which was highly sensitive to minor changes in EF thresholds. Collectively, our results highlight the critical role of EF oncogene fluctuations in Ewing sarcoma cell plasticity and patient prognosis providing new insights into the gene regulatory mechanisms involved. Further, our findings have important implications for therapeutic approaches aiming at directly targeting EF. Understanding EF dynamics in Ewing sarcoma serves a paradigmatic approach to the identification of therapeutic vulnerabilities in fusion oncogene-driven cancers.
 Citation Format: Veveeyan Suresh, Christoph Hafemeister, Sarah Grissenberger, Caterina Sturtzel, Florencia Cidre Aranaz, Thomas Grünewald, Ana Sastre, Javier Alonso, Martin Distel, Florian Halbritter, Heinrich Kovar, Valerie Fock. Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/533254be9587cdde4641f2edde320cdf2280f903" target='_blank'>
              Abstract A017 Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, S. Grissenberger, C. Sturtzel, F. C. Aranaz, Thomas Grünewald, A. Sastre, Javier Alonso, Martin Distel, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="CRISPR gene editing strategies are shaping cell therapies through precise and tunable control over gene expression. However, limitations in safely delivering high quantities of CRISPR machinery demand careful target gene selection to achieve reliable therapeutic effects. Informed target gene selection requires a thorough understanding of the involvement of target genes in gene regulatory networks (GRNs) and thus their impact on cell phenotype. Effective decoding of these complex networks has been achieved using machine learning models, but current techniques are limited to single cell types and focus mainly on transcription factors, limiting their applicability to CRISPR strategies. To address this, we present CRISPR-GEM, a multilayer perceptron (MLP) based synthetic GRN constructed to accurately predict the downstream effects of CRISPR gene editing. First, input and output nodes are identified as differentially expressed genes between defined experimental and target cell/tissue types, respectively. Then, MLP training learns regulatory relationships in a black-box approach allowing accurate prediction of output gene expression using only input gene expression. Finally, CRISPR-mimetic perturbations are made to each input gene individually, and the resulting model predictions are compared to those for the target group to score and assess each input gene as a CRISPR candidate. The top scoring genes provided by CRISPR-GEM therefore best modulate experimental group GRNs to motivate transcriptomic shifts toward a target group phenotype. This machine learning model is the first of its kind for predicting optimal CRISPR target genes and serves as a powerful tool for enhanced CRISPR strategies across a range of cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b614857317f506b1c1387c17ae0939d3839f0837" target='_blank'>
              CRISPR-GEM: A Novel Machine Learning Model for CRISPR Genetic Target Discovery and Evaluation.
              </a>
            </td>
          <td>
            Josh P. Graham, Yu Zhang, Lifang He, Tomas Gonzalez-Fernandez
          </td>
          <td>2024-10-07</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d35e0bab7dec79077bc3332fba4617308d5113" target='_blank'>
              Comprehensive evaluation of immunological attributes and immunotherapy responses of positive T cell function regulators in colorectal cancer
              </a>
            </td>
          <td>
            Ke Pu, Jingyuan Gao, Yang Feng, Jian Hu, Shunli Tang, Guodong Yang, Chuan Xu
          </td>
          <td>2024-10-01</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Disruption of mechanisms that maintain genome stability is an essential factor of tumor progression. Accordingly, predisposition to the development of neoplasms is often associated with germline mutations in genes involved in DNA damage detection and repair. At the same time, impairment of DNA repair systems may be a predictor of antitumor treatment efficacy while overexpression of genes involved in DNA repair is a frequent event in various types of malignancies that can lead to development of tumor cells’ resistance to chemo- and radiotherapy. NBN (nibrin) gene encodes the subunit of the MRN complex which acts as a sensor of double-strand DNA breaks and participates in their repair by homologous recombination. Germline variants in NBN which are associated with increased risk of tumor development are generally represented by frameshift mutations that lead to the synthesis of truncated protein as well as by nonsense and some missense mutations which occur in functionally significant domains. These germline mutations result in partial loss of nibrin function and in increased frequency of spontaneous and induced chromosomal aberrations in the cells of the carriers. On the contrary, amplification of NBN locus is a predominant type of somatic mutations affecting this gene, which indicates a dual role of NBN protein in tumor progression. The results of several studies demonstrate the influence of NBN expression level and its mutational status on anti-tumor drug resistance in particular types of tumor cells and on the survival rate of patients. These data indicate that an in-depth study of different variants and their functional significance is necessary since NBN status may be essential for the choice of treatment tactics for some types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c608441ac7c7406f3e5d539484e420368a9c4c9" target='_blank'>
              Germline and somatic alterations in NBN and their putative impact on the pathogenesis of malignant neoplasms
              </a>
            </td>
          <td>
            O. Krivtsova, D. D. Ozerova, N. L. Lazarevich
          </td>
          <td>2024-10-10</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The fusion expression of DNA replication-related proteins with nucleotide deaminase enzymes promotes random mutations in bacterial genomes, thereby increasing genetic diversity among population. Most previous studies have focused on cytosine deaminase, which produces only C→T mutations, significantly limiting the variety of mutation types. In this study, we developed a fusion expression system by combining DnaG (RNA primase) with adenine deaminase TadA-8e (DnaG-TadA) in Escherichia coli, which is capable of rapidly introducing A→G mutations into the E. coli genome, resulting in a 664-fold increase in terms of mutation rate. Additionally, we engineered a dual-functional TadA variant, TadAD, and then fused it with DnaG. This construct introduced both C→T and A→G mutations into the E. coli genome, with the mutation rate further increased by 370-fold upon co-expression with an uracil glycosylase inhibitor (DnaG-TadAD-UGI). We applied DnaG-TadA and DnaG-TadAD-UGI systems to the adaptive laboratory evolution for Cd2+ and kanamycin resistance, achieving an 8.0 mM Cd2+ and 200 μg/mL kanamycin tolerance within just 17 days and 132 hours, respectively. Compared to conventional evolution methods, the final tolerance levels were increased by 320% and 266%, respectively. Our work offers a novel strategy for random mutagenesis in E. coli and potentially other prokaryotic species. For TOC only">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8dc3e641872269a76809a2a9f58812a1c2451c2" target='_blank'>
              Genome-wide A→G and C→T Mutations Induced by Functional TadA Variants in Escherichia coli
              </a>
            </td>
          <td>
            Hao Wang, Zhengxin Dong, Jingyi Shi, Lei Chen, Tao Sun, Weiwen Zhang
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cervical cancer (CC), one of the most prevalent and detrimental gynaecologic cancers, evolves through genetic and epigenetic alterations resulting in the promotion of oncogenic activity and dysfunction of tumour-suppressing mechanisms. Despite medical advancement, the prognosis for advanced-stage patients remains extremely low due to high recurrence rates and resistance to existing treatments. Thereby, the search for potential prognostic biomarkers is heightened to unravel new modalities of CC pathogenesis and to develop novel anti-cancer therapies. Epitranscriptomic modifications, reversible epigenetic RNA modifications, regulate various biological processes by deciding RNA fate to mediating RNA interactions. This narrative review provides insight into the cellular and molecular roles of endogenous RNA-editing proteins and their associated epitranscriptomic modifications, especially N6-methyladenosine (m6A), 5-methylcytosine (m5C) and N1-methyladenosine (m1A), in governing the development, progression and metastasis of CC. We discussed the in-depth epitranscriptomic mechanisms underlying the regulation of over 50 RNAs responsible for tumorigenesis, proliferation, migration, invasion, survival, autophagy, stemness, epithelial-mesenchymal transition, metabolism (glucose, lipid, glutamate and glutamine), resistance (drug and radiation), angiogenesis and recurrence of CC. Additionally, we provided a concise overview of the therapeutic potential of targeting the altered expression of endogenous RNA-editing proteins and aberrant deposition of RNA modifications on both coding and non-coding RNAs in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d247af5a7e732d73b316957acc73d51f868974f4" target='_blank'>
              Epitranscriptomics and cervical cancer: the emerging role of m6A, m5C and m1A RNA modifications
              </a>
            </td>
          <td>
            Akshat D. Modi, Hira Zahid, Ashlyn Chase Southerland, Dharmeshkumar M Modi
          </td>
          <td>2024-10-08</td>
          <td>Expert Reviews in Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="While T cell lymphomas are classified as mature neoplasms, emerging evidence indicates that malignant transformation may occur at an earlier stage of T cell maturation. In this study, we determined clonal architectures in a broad range of T cell lymphomas. Our multidimensional profiling indicates that a large part of these lymphomas in fact emerge from an immature lymphoid T cell precursor at a maturation stage prior to V(D)J rearrangement that undergoes branching evolution. Consequently, at single cell resolution we observed considerable clonal tiding under selective therapeutic pressure. T cell receptor next-generation sequencing suggested a highly biased usage of TRBV20-1 gene segments as part of multiple antigen receptor rearrangements per patient. The predominance of TRBV20-1 was found across all major T cell lymphoma subtypes analyzed. This suggested that this particular V gene - independently of complementarity-determining region 3 (CDR3) configuration - may represent a driver of malignant transformation. Together, our data indicate that T cell lymphomas derive from immature lymphoid precursors and display considerable intratumoral heterogeneity that may provide the basis for relapse and resistance in these hard-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5cc9e8d39a7b3561d529df56bcd87a167e0b44e" target='_blank'>
              T-cell receptor architecture and clonal tiding provide insight into the transformation trajectory of peripheral T-cell lymphomas.
              </a>
            </td>
          <td>
            E. Willscher, C. Schultheiss, L. Paschold, Franziska Lea Schümann, Paul Schmidt-Barbo, Benjamin Thiele, M. Bauer, Claudia Wickenhauser, Thomas Weber, Mascha Binder
          </td>
          <td>2024-08-29</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b39abde50129bd51799285e63a0661a0178d6716" target='_blank'>
              DNA damage response in breast cancer and its significant role in guiding novel precise therapies
              </a>
            </td>
          <td>
            Jiayi Li, Ziqi Jia, Lin Dong, Heng Cao, Yansong Huang, Hengyi Xu, Zhixuan Xie, Yiwen Jiang, Xiang Wang, Jiaqi Liu
          </td>
          <td>2024-09-27</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A fundamental challenge in biomedicine is understanding the mechanisms predisposing individuals to disease. While previous research has suggested that switch-like gene expression is crucial in driving biological variation and disease susceptibility, a systematic analysis across multiple tissues is still lacking. By analyzing transcriptomes from 943 individuals across 27 tissues, we identified 1,013 switch-like genes. We found that only 31 (3.1%) of these genes exhibit switch-like behavior across all tissues. These universally switch-like genes appear to be genetically driven, with large exonic genomic structural variants explaining five (∼18%) of them. The remaining switch-like genes exhibit tissue-specific expression patterns. Notably, tissue-specific switch-like genes tend to be switched on or off in unison within individuals, likely under the influence of tissue-specific master regulators, including hormonal signals. Among our most significant findings, we identified hundreds of concordantly switched-off genes in the stomach and vagina that are linked to gastric cancer (41-fold, p<10-4) and vaginal atrophy (44-fold, p<10-4), respectively. Experimental analysis of vaginal tissues revealed that low systemic levels of estrogen lead to a significant reduction in both the epithelial thickness and the expression of the switch-like gene ALOX12. We propose a model wherein the switching off of driver genes in basal and parabasal epithelium suppresses cell proliferation therein, leading to epithelial thinning and, therefore, vaginal atrophy. Our findings underscore the significant biomedical implications of switch-like gene expression and lay the groundwork for potential diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8ed2dab6e33efaca4d5da4963155c33d9525bc" target='_blank'>
              Switch-like Gene Expression Modulates Disease Susceptibility
              </a>
            </td>
          <td>
            Alber Aqil, Yanyan Li, Zhiliang Wang, Saiful Islam, Madison Russell, T. Kunovac Kallak, Marie Saitou, Omer Gokcumen, N. Masuda
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Successful tuberculosis therapy requires treatment with an unwieldy multidrug combination for several months. Thus, there is a growing need to identify novel genetic vulnerabilities that can be leveraged to develop new, more effective antitubercular drugs. Consequently, recent efforts to optimize TB therapy have exploited Mtb chemical genetics to identify pathways influencing antibiotic efficacy, novel mechanisms of antibiotic action, and new targets for TB drug discovery. However, the influence of the complex host environment on these interactions remains largely unknown, leaving the therapeutic potential of the identified targets unclear. In this study, we leveraged a library of conditional mutants targeting 467 essential Mtb genes to characterize the chemical-genetic interactions (CGIs) with TB drugs directly in the mouse infection model. We found that these in vivo CGIs differ significantly from those identified in vitro. Both drug-specific and drug-agnostic effects were identified, and many were preserved during treatment with a multidrug combination, suggesting numerous strategies for enhancing therapy. This work also elucidated the complex effects of pyrazinamide (PZA), a drug that relies on aspects of the infection environment for efficacy. Specifically, our work supports the importance of coenzyme A synthesis inhibition during infection, as well as the antagonistic effect of iron limitation on PZA activity. In addition, we found that inhibition of thiamine and purine synthesis increases PZA efficacy, suggesting novel therapeutically exploitable metabolic dependencies. Our findings present a map of the unique in vivo CGIs, characterizing the mechanism of PZA activity in vivo and identifying novel targets for TB drug development. Significance The inevitable rise of multi-drug-resistant tuberculosis underscores the urgent need for new TB drugs and novel drug targets while prioritizing synergistic drug combinations. Chemical-genetic interaction (CGI) studies have delineated bacterial pathways influencing antibiotic efficacy and uncovered druggable pathways that synergize with TB drugs. However, most studies are conducted in vitro, limiting our understanding of how the host environment influences drug-mutant interactions. Using an inducible mutant library targeting essential Mtb genes to characterize CGIs during infection, this study reveals that CGIs are both drug-specific and drug-agnostic and differ significantly from those observed in vitro. Synergistic CGIs comprised distinct metabolic pathways mediating antibiotic efficacy, revealing novel drug mechanisms of action, and defining potential drug targets that would synergize with frontline antitubercular drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d045b0a24f1162d0aaf03742b7cbebc2dd9fdd94" target='_blank'>
              Chemical genetic interactions elucidate pathways controlling tuberculosis antibiotic efficacy during infection
              </a>
            </td>
          <td>
            P. Oluoch, Eun-Ik Koh, Megan M Proulx, Charlotte J. Reames, K. Papavinasasundaram, Kenan C. Murphy, Matthew D. Zimmerman, Véronique Dartois, Chris Sassetti
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484b45daaba1111e0c4d69a75c275aee523f917f" target='_blank'>
              Multi-omics profiling of longitudinal samples reveals early genomic changes in follicular lymphoma
              </a>
            </td>
          <td>
            Baoyan Bai, J. F. Wise, Daniel Vodák, S. Nakken, Ankush Sharma, Y. Blaker, M. Brodtkorb, V. Hilden, G. Trøen, W. Ren, S. Lorenz, Michael S Lawrence, O. Myklebost, E. Kimby, Q. Pan-Hammarström, Chloé B Steen, L. Meza-Zepeda, Klaus Beiske, E. Smeland, E. Hovig, O. Lingjærde, H. Holte, J. Myklebust
          </td>
          <td>2024-08-27</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Gastrointestinal tumors, the second leading cause of human mortality, are characterized by their association with inflammation. Currently, progress in the early diagnosis and effective treatment of gastrointestinal tumors is limited. Recent whole‐genome analyses have underscored their profound heterogeneity and extensive genetic and epigenetic reprogramming. Epigenetic reprogramming pertains to dynamic and hereditable alterations in epigenetic patterns, devoid of concurrent modifications in the underlying DNA sequence. Common epigenetic modifications encompass DNA methylation, histone modifications, noncoding RNA, RNA modifications, and chromatin remodeling. These modifications possess the potential to invoke or suppress a multitude of genes associated with cancer, thereby governing the establishment of chromatin configurations characterized by diverse levels of accessibility. This intricate interplay assumes a pivotal and indispensable role in governing the commencement and advancement of gastrointestinal cancer. This article focuses on the impact of epigenetic reprogramming in the initiation and progression of gastric cancer, esophageal cancer, and colorectal cancer, as well as other uncommon gastrointestinal tumors. We elucidate the epigenetic landscape of gastrointestinal tumors, encompassing DNA methylation, histone modifications, chromatin remodeling, and their interrelationships. Besides, this review summarizes the potential diagnostic, therapeutic, and prognostic targets in epigenetic reprogramming, with the aim of assisting clinical treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa46852e2f3c4f502e85931e028a5b5bde3a9141" target='_blank'>
              Epigenetic reprogramming in gastrointestinal cancer: biology and translational perspectives
              </a>
            </td>
          <td>
            Yingjie Wang, Hongyu Liu, Mengsha Zhang, Jing Xu, Liuxian Zheng, Pengpeng Liu, Jingyao Chen, Hongyu Liu, Chong Chen
          </td>
          <td>2024-08-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell genomic analysis enables researchers to gain novel insights across diverse research areas, including developmental biology, tumor heterogeneity, and disease pathogenesis. Conducting single-cell genomic analysis using next-generation sequencing (NGS) methods has traditionally been challenging as the amount of genomic DNA present in a single cell is limited. Advancements in multiple displacement amplification (MDA) technologies allow the unbiased amplification of limited quantities of DNA under conditions that maintain its integrity. This method of amplification results in high yield and facilitates the generation of high-complexity NGS libraries that ensure the highest coverage to effectively allow variant calling. With the introduction of new sequencing platforms and chemistry, whole genome sequencing became a more cost-effective application, but enrichment of specific regions of interest further reduces the amount of required sequencing output and associated costs. There are two enrichment methods, polymerase chain reaction (PCR)-based and hybrid-capture-based methods. PCR-based methods are very flexible and highly effective but focus on specific loci, typically known to be associated with disease. Inherited diseases of unknown genetic origin require a more comprehensive approach to capture the genetic variation that is not yet associated with a specific disease. Hybrid capture enrichment methods require considerable amounts of DNA such that exome enrichment from single cells is only possible after preamplification of this limited material. This article describes the complete workflow from single cells and small quantities of DNA to exome-NGS libraries for Illumina sequencing instruments and includes the following protocols: © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Whole genome amplification from single cells or small amounts of gDNA Basic Protocol 2: NGS library generation of MDA-amplified material Basic Protocol 3: Exome enrichment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891307a5d438a6375e34496a127912e0af6f10c1" target='_blank'>
              Exome Sequencing Starting from Single Cells.
              </a>
            </td>
          <td>
            Ioanna Andreou, Markus Storbeck, Peter Hahn, Samuel J. Rulli, Eric Lader
          </td>
          <td>2024-09-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0914eac9368db3a26273ef88f18ab51249d37f" target='_blank'>
              Advances in targeting cancer epigenetics using CRISPR-dCas9 technology: A comprehensive review and future prospects.
              </a>
            </td>
          <td>
            Jeevitha Rajanathadurai, Elumalai Perumal, Jospin Sindya
          </td>
          <td>2024-09-18</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0fe672a884b6c86c84e083c9b2ab93ac24f65f" target='_blank'>
              Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements
              </a>
            </td>
          <td>
            Florence M. Chardon, Troy A. McDiarmid, Nicholas F. Page, R. Daza, Beth K. Martin, Silvia Domcke, Samuel Regalado, Jean-Benoît Lalanne, Diego Calderon, Xiaoyi Li, L. Starita, Stephan J. Sanders, N. Ahituv, J. Shendure
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>49</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing technology is a promising tool for correcting pathogenic variants for autologous cell therapies for Inborn Errors of Immunity (IEI). The present IEI correction strategies mainly focus on the knock-in of therapeutic cDNAs, or knockout of the disease-causing gene when feasible. These strategies address many single-gene defects but may disrupt gene expression and require significant optimization for each newly discovered IEI-causing gene, highlighting the need for complementary platforms that can precisely correct diverse pathogenic variants. Here, we present a safe and efficient T cell single nucleotide variant (SNV) correction pipeline based on homology-directed repair (HDR), suitable for diverse monogenic mutations. By using founder mutations of Deficiency of ADA2 (DADA2), Autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED) and Cartilage Hair Hypoplasia (CHH) as IEI models, we show that our pipeline can achieve up to 80% bi-allelic editing, with resultant functional correction of the disease phenotype in patient T cells. We do not find detectable pre-malignant off-target effects or karyotypic, transcriptomic or proteomic aberrations upon profiling patient T cells with GUIDE-seq, single cell RNA sequencing, PacBio based long-read whole genome sequencing, and high-throughput proteomics. This study demonstrates that HDR-based SNV editing is a safe and effective option for IEI T cell correction and that it could be developed to an autologous T cell therapy, as the presented protocol is scalable for a GMP-compatible workflow. This study is a step towards the development of gene correction platform that targets a broad number of monogenic mutations. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3514ea764ac815dfdaa1d9a93597841e9c3e8944" target='_blank'>
              T cell correction pipeline for Inborn Errors of Immunity
              </a>
            </td>
          <td>
            Katariina Mamia, Sólrún Kolbeinsdóttir, Zhuokun Li, Kornel Labun, Anna Komisarczuk, S. Keskitalo, Ganna Reint, Frida Høsøien Haugen, Britt Olaug Lindestad, T. Gjerdingen, Antti Tuhkala, Carolina Wieczorek Ervik, Pavel Kopcil, Nail Fatkhutdinov, Monika Szymańska, Eero Tölö, V. Glumoff, J. Saarela, T. Michelsen, Camilla Schalin-Jäntti, Johanna Olweus, Eira Leinonen, M. Varjosalo, Eivind Valen, Timo Hautala, Martin Enge, T. Martelius, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Precise transcriptional regulation is critical for cellular function and development, yet the mechanism of this process remains poorly understood for many genes. To gain a deeper understanding of the regulation of neuropsychiatric disease risk genes, we identified a total of 39 functional enhancers for four dosage-sensitive genes, APP, FMR1, MECP2, and SIN3A, using CRISPR tiling deletion screening in human induced pluripotent stem cell (iPSC)-induced excitatory neurons. We found that enhancer annotation provides potential pathological insights into disease-associated copy number variants. More importantly, we discovered that allelic enhancer deletions at SIN3A could be compensated by increased transcriptional activities from the other intact allele. Such allelic compensation effects (ACE) on transcription is stably maintained during differentiation and, once established, cannot be reversed by ectopic SIN3A expression. Further, ACE at SIN3A occurs through dosage sensing by the promoter. Together, our findings unravel a regulatory compensation mechanism that ensures stable and precise transcriptional output for SIN3A, and potentially other dosage-sensitive genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68c9cf4cae8ed81690d40ccc0776865b484f3141" target='_blank'>
              CRISPR tiling deletion screens reveal functional enhancers of neuropsychiatric risk genes and allelic compensation effects (ACE) on transcription
              </a>
            </td>
          <td>
            Xingjie Ren, Lina Zheng, Lenka Maliskova, Tsz Wai Tam, Yifan Sun, Hongjiang Liu, Jerry Lee, M. Takagi, Bin Li, Bing Ren, Wei Wang, Yin Shen
          </td>
          <td>2024-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0362aabb9265fe87bbca700d69f60f98a5d89595" target='_blank'>
              Steering research on mRNA splicing in cancer towards clinical translation.
              </a>
            </td>
          <td>
            Olga Anczuków, Frédéric H-T Allain, B. Angarola, Douglas L. Black, Angela N. Brooks, Chonghui Cheng, Ana Conesa, Edie I. Crosse, E. Eyras, Ernesto Guccione, Sydney X Lu, Karla M Neugebauer, Priyanka Sehgal, Xiao Song, Zuzana Tothova, Juan Valcárcel, Kevin M. Weeks, Gene W Yeo, A. Thomas-Tikhonenko
          </td>
          <td>2024-10-09</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="In recent years, large-scale sequencing efforts have identified targetable driver mutations in haematopoietic stem cells. These efforts have led to the development and approval of nine novel agents for relapsed or refractory acute myelogenous leukaemia (R/R AML). However, despite an expansion in targeted therapies, achieving a durable remission in AML and high-risk myelodysplastic syndrome (HR-MDS) remains a significant challenge, and there is an urgent need for new effective treatments. Modulation of aberrant RNA splicing has emerged as a novel therapeutic approach in myeloid diseases. Aberrant splicing drives dysregulated gene expression that promotes tumourigenesis through increased proliferation and metastatic potential, immune evasion, decreased apoptosis, and chemotherapy resistance. Mutations in spliceosomal components have been identified in numerous cancer subtypes, with mutations in RNA binding proteins SF3B1, SRSF2, U2AF1, and ZRSR2 occurring frequently in AML and in up to 60% of patients with MDS, as well as in chronic myelomonocytic leukaemia and in 10% of patients with chronic lymphocytic leukaemia. In this review, we explore therapeutic strategies targeting aberrant splicing and the potential of these approaches to drive clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17b179523c7f6d0d667d6bb250cabd47b56f2a64" target='_blank'>
              Therapeutic strategies targeting aberrant RNA splicing in myeloid malignancies.
              </a>
            </td>
          <td>
            Leora S Boussi, Jeetayu Biswas, Omar Abdel-Wahab, E. Stein
          </td>
          <td>2024-10-15</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction Biological processes such as aging, carcinogenesis, and immune response rely on the ability to maintain or rapidly expand cell populations. The fitness of the involved cells is constrained by their replicative potential, which is reflected in the cellular telomere content. Method We apply TelomereHunter to scATAC-seq data to determine telomere content on single-cell level, in a hematopoietic dataset consisting of 35,139 cells from samples of basal cell carcinoma patients receiving programmed cell death protein 1 (PD1) blockade treatment. Integrating information from open-chromatin-based signatures to assess cell identity, we characterize the heterogeneity of telomere length for individual cell populations pre- and post-immunotherapy. Results The extracted telomeric reads reflect the expected telomereome-to-genome fraction. Telomere content distributions differ significantly between cell populations, and the median telomere content in intermediate and terminal exhausted CD8+ T-cells pre-treatment is significantly correlated to response to PD-1 checkpoint blockade. Likewise, telomere content correlates with post-treatment cell proliferation in terminally exhausted and T follicular helper cells from responding patients. Conclusion Telomere content measurement from scATAC-seq data has a sufficiently high signal-to-noise ratio to detect significant differences between cell types. Furthermore, the telomere content of CD8+ exhausted T-cells pre-treatment is a putative biomarker for successful PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c0b43ae3c8ae5883f7e563f1131356d52bd5e2" target='_blank'>
              Quantifying immune cell telomere content at single-cell resolution in context of PD-1 checkpoint immunotherapy
              </a>
            </td>
          <td>
            Niklas L. Engel, Lea Herzel, Julie Surmely, Hanna Frieß, Malte Simon, B. Brors, Charles D. Imbusch, L. Feuerbach
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Therapy-related clonal hematopoiesis (t-CH) is defined as clonal hematopoiesis detected in individuals previously treated with chemotherapy and/or radiation therapy. With the increased use of genetic analysis in oncological care, the detection of t-CH among cancer patients is becoming increasingly common. t-CH arises through the selective bottleneck imposed by chemotherapies and potentially through direct mutagenesis from chemotherapies, resulting in a distinct mutational landscape enriched with mutations in DNA damage-response pathway genes such as TP53, PPM1D, and CHEK2. Emerging evidence sheds light on the mechanisms of t-CH development and potential strategies to mitigate its emergence. Due to its unique characteristics that predominantly affect cancer patients, t-CH has clinical implications distinct from those of CH in the general population. This Review discusses the potential mechanisms of t-CH development, its mutational landscape, mutant-drug relationships, and its clinical significance. We highlight the distinct nature of t-CH and call for intensified research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cad43d7ae7881b3bb897b501eceb9fbccf93c93" target='_blank'>
              Distinct landscape and clinical implications of therapy-related clonal hematopoiesis
              </a>
            </td>
          <td>
            Koichi Takahashi, Daisuke Nakada, Margaret A Goodell
          </td>
          <td>2024-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Resistance to paclitaxel poses a major obstacle in esophageal squamous cell carcinoma (ESCC) treatment. A better understanding of the mechanisms underlying paclitaxel resistance could help identify prognostic biomarkers and improved therapeutic strategies. In this study, we established a patient-derived xenograft (PDX) model of acquired paclitaxel resistance and used RNA-sequencing to identify galectin-1, encoded by LGALS1, as a key mediator of resistance. Integrative analysis of clinical data and physiological studies indicated that serum galectin-1 levels were elevated in resistant patients and correlated with treatment outcomes before and during taxane therapy. Importantly, exposing cells to serum from resistant patients resulted in increased paclitaxel resistance compared to serum from sensitive patients, which was closely associated with galectin-1 concentrations in the serum. The specific clearance of galectin-1 from resistant patient serum significantly restored paclitaxel sensitivity, and inhibiting galectin-1, through knockdown or the pharmacologic inhibitor OTX008, increased sensitivity to paclitaxel. Galectin-1 inhibition reduced the activity of β-catenin, thereby inhibiting stem cell properties induced by the Wnt/β-catenin pathway. Furthermore, galectin-1 regulated MDR1 transcription through increased nuclear accumulation of β-catenin, thus increasing resistance to paclitaxel. Combining OTX008 with clinical taxane formulations effectively reversed paclitaxel resistance in vitro and in vivo. Elevated galectin-1 levels thus serve as an indicator of response to paclitaxel therapy in ESCC, offering a therapeutic intervention strategy to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/177752d2c25514f91117f50aa85953de6bf7869d" target='_blank'>
              Targeting Galectin-1 Overcomes Paclitaxel Resistance in Esophageal Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Liting Zhou, Jie Tian, Keke Wang, Yijie Ma, Xiaojie Chen, Hui Luo, Bingbing Lu, Nan Wang, Penglei Wang, Xuejiao Liu, Ran Zhao, Simin Zhao, Jiutao Wang, Wenna Nie, Hong Ge, Wenting Liu, Tingxuan Gu, Kangdong Liu, Mee-Hyun Lee, Xiang Li, Zigang Dong
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) poses significant challenges due to its complex biological characteristics and high recurrence rate. The high recurrence rate of LUAD is closely associated with cellular dormancy, which enhances resistance to chemotherapy and evasion of immune cell destruction. Using single-cell RNA sequencing (scRNA-seq) data from LUAD patients, we categorized the cells into two subclusters: dormant and active cells. Utilizing high-density Weighted Gene Co-expression Network Analysis (hdWGCNA) and pseudo-time cell trajectory, aberrant expression of genes involved in protein O-glycosylation was detected in dormant cells, suggesting a crucial role for O-glycosylation in maintaining the dormant state. Intercellular communication analysis highlighted the interaction between fibroblasts and dormant cells, where the Insulin-like Growth Factor (IGF) signaling pathway regulated by O-glycosylation was crucial. By employing Gene Set Variation Analysis (GSVA) and machine learning, a risk score model was developed using hub genes, which showed high accuracy in determining LUAD prognosis. The model also demonstrated robust performance on the training dataset and excellent predictive capability, providing a reliable basis for predicting patient clinical outcomes. The group with a higher risk score exhibited a propensity for adverse outcomes in the tumor microenvironment (TME) and tumor mutational burden (TMB). Additionally, the 50% inhibitory concentration (IC50) values for chemotherapy exhibited significant variations among the different risk groups. In vitro experiments demonstrated that EFNB2, PTTG1IP, and TNFRSF11A were upregulated in dormant tumor cells, which also contributed greatly to the diagnosis of LUAD. In conclusion, this study highlighted the crucial role of O-glycosylation in the dormancy state of LUAD tumors and developed a predictive model for the prognosis of LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdea9ceef7f2c70dc3e70ad8f0a54632f63dceb" target='_blank'>
              Deciphering Dormant Cells of Lung Adenocarcinoma: Prognostic Insights from O-glycosylation-Related Tumor Dormancy Genes Using Machine Learning
              </a>
            </td>
          <td>
            Chenfei Dong, Yang Liu, Suli Chong, Jiayue Zeng, Ziming Bian, Xiaoming Chen, Sairong Fan
          </td>
          <td>2024-08-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Telomeres protect chromosome ends from DNA damage responses, and their dysfunction triggers genomic alterations like chromosome fusion and rearrangement, which can lead to cellular death. Certain cells, including specific cancer cells, adopt alternative lengthening of telomere (ALT) to counteract dysfunctional telomeres and proliferate indefinitely. While telomere instability and ALT activity are likely major sources of genomic alteration, the patterns and consequences of such changes at the nucleotide level in ALT cells remain unexplored. Here we generated haplotype-resolved genome assemblies for type I ALT mouse embryonic stem cells, facilitated by highly accurate or ultra-long reads and Hi-C reads. High-quality genome revealed ALT-specific complex chromosome end structures and various genomic alterations including over 1000 structural variants (SVs). The unique sequence (mTALT) used as a template for type I ALT telomeres showed traces of being recruited into the genome, with mTALT being replicated with remarkably high accuracy. Subtelomeric regions exhibited distinct characteristics: resistance to the accumulation of SVs and small variants. We genotyped SVs at allele resolution, identifying genes (Rgs6, Dpf3 and Tacc2) crucial for maintaining ALT telomere stability. Our genome assembly-based approach elucidated the unique characteristics of ALT genome, offering insights into the genome evolution of cells surviving telomere-derived crisis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d9657866100fc15991f3ce13ef281bd3238fb24" target='_blank'>
              Haplotype-resolved de novo assembly revealed unique characteristics of alternative lengthening of telomeres in mouse embryonic stem cells.
              </a>
            </td>
          <td>
            Hyunji Lee, Hiroyuki Niida, Sanghyun Sung, Junho Lee
          </td>
          <td>2024-10-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="While immune checkpoint inhibitors have revolutionized cancer therapy, many patients exhibit poor outcomes. Here, we show immunotherapy responses in bladder and non–small cell lung cancers are effectively predicted by factoring tumor mutation burden (TMB) into burdens on specific protein assemblies. This approach identifies 13 protein assemblies for which the assembly-level mutation burden (AMB) predicts treatment outcomes, which can be combined to powerfully separate responders from nonresponders in multiple cohorts (e.g., 76% versus 37% bladder cancer 1-year survival). These results are corroborated by (i) engineered disruptions in the predictive assemblies, which modulate immunotherapy response in mice, and (ii) histochemistry showing that predicted responders have elevated inflammation. The 13 assemblies have diverse roles in DNA damage checkpoints, oxidative stress, or Janus kinase/signal transducers and activators of transcription signaling and include unexpected genes (e.g., PIK3CG and FOXP1) for which mutation affects treatment response. This study provides a roadmap for using tumor cell biology to factor mutational effects on immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5f818fb592baa5f66559ad69838daf202c25a6" target='_blank'>
              Prediction of immunotherapy response using mutations to cancer protein assemblies
              </a>
            </td>
          <td>
            JungHo Kong, Xiaoyu Zhao, Akshat Singhal, Sungjoon Park, Robin Bachelder, Jeanne Shen, Haiyu Zhang, Jimin Moon, Changho Ahn, C. Ock, Hannah Carter, T. Ideker
          </td>
          <td>2024-09-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="BACKGROUND The mitotic intra-S DNA damage checkpoint signaling gene set is potentially involved in cancers where the genes play an important role. 17 total genes are involved in this gene set: ATF2, CHEK2, EME1, EME2, FANCD2, HUS1, HUS1B, MDC1, MRE11, MSH2, MUS81, NEK11, RAD17, RAD9A, RAD9B, TIPIN, XPC. The aim of this study is to complete a pan-cancer profile of each gene in the mitotic intra-S DNA damage checkpoint signaling gene set in order to determine potential diagnostic and prognostic purposes, while also determining how they could be used in a clinical setting as therapeutic targets to help patients. METHODS Multiomic data was acquired for the 17 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of CNV, mRNA expression, and pathway analysis. RESULTS The CNVs of some of these genes are associated with the survival of MESO, PCPG, BLCA, SKCM, LUAD, HNSC, LUSC, OV, and BRCA could be affected by the mRNA expression of the genes which can involve regulation of copy number. CONCLUSION With sufficient investigation, the genes involved in mitotic intra-S DNA damage checkpoint signaling may contribute to the development of cancer and may be used as biomarkers for cancer prognosis and diagnosis. To prove their clinical use for diagnosis and prognosis, however, and to create workable applications in clinical settings, further work is required. However, these pan-cancer profiles provide a more comprehensive knowledge of the mitotic intra-S DNA damage checkpoint signaling gene set in cancer as well as valuable information for future reference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f222ed09772900dec985568ac443adca2980e33" target='_blank'>
              Potential Cancer Biomarkers: Mitotic Intra-S DNA Damage Checkpoint Genes
              </a>
            </td>
          <td>
            Kashvi Agarwal, Hengrui Liu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b7a1b60f27d8f207e63fcabad98fcf75b1aff6" target='_blank'>
              Deciphering the impact of genomic variation on function.
              </a>
            </td>
          <td>
            J. Engreitz, Heather A. Lawson, Harinder Singh, Lea M. Starita, Gary C. Hon, Hannah Carter, Nidhi Sahni, Timothy E. Reddy, Xihong Lin, Yun Li, Nikhil V. Munshi, Maria H. Chahrour, Alan P. Boyle, B. Hitz, Ali Mortazavi, Mark Craven, K. Mohlke, Luca Pinello, Ting Wang, A. Kundaje, Feng Yue, Sarah Cody, Nina P. Farrell, Michael I. Love, Lara A Muffley, M. Pazin, Fairlie Reese, Eric Van Buren, Kushal K. Dey, Martin Kircher, Jian Ma, P. Radivojac, Brunilda Balliu, Brian A. Williams, Danwei Huangfu, Chong Y. Park, Thomas Quertermous, Jishnu Das, M. Calderwood, Douglas M. Fowler, Marc Vidal, Lucas Ferreira, Sean D. Mooney, V. Pejaver, Jingjing Zhao, Steven Gazal, Evan Koch, Steven K. Reilly, S. Sunyaev, Anne E. Carpenter, Jason D. Buenrostro, Christina S. Leslie, Rachel E. Savage, Stefanija Giric, Chongyuan Luo, Kathrin Plath, Alejandro Barrera, Max Schubach, A. Gschwind, Jill E. Moore, N. Ahituv, S. Yi, Ingileif B. Hallgrímsdóttir, K. Gaulton, S. Sakaue, Sina Booeshaghi, Eugenio Mattei, Surag Nair, L. Pachter, Austin T. Wang, J. Shendure, Vikram Agarwal, Andrew Blair, Theofilos Chalkiadakis, Florence M. Chardon, P. Dash, Chengyu Deng, Nobuhiko Hamazaki, Pia Keukeleire, Connor A Kubo, Jean-Benoît Lalanne, Thorben Maass, Beth Martin, Troy A. McDiarmid, Mai Nobuhara, Nicholas F. Page, Sam Regalado, Jasmine Sims, A. Ushiki, Sabrina M. Best, Gabe Boyle, Nathan Camp, Silvia Casadei, Estelle Y. Da, Moez Dawood, Samantha C. Dawson, Shawn Fayer, Audrey Hamm, Richard G. James, Gail P. Jarvik, Abbye E. McEwen, Nick Moore, Sriram Pendyala, Nicholas A Popp, Mason Post, Alan F. Rubin, Nahum T. Smith, J. Stone, Malvika Tejura, Ziyu R. Wang, Melinda K. Wheelock, Ivan Woo, Brendan D. Zapp, Dulguun Amgalan, Aradhana Aradhana, Sophia M. Arana, M. Bassik, Julia R. Bauman, Asmita Bhattacharya, Xiangmeng S. Cai, Ziwei Chen, Stephanie Conley, Salil Deshpande, Benjamin R. Doughty, Peter P. Du, James A. Galante, Casey Gifford, W.J. Greenleaf, Katherine Guo, Revant Gupta, Sarasa Isobe, Evelyn Jagoda, Nimit Jain, Hank Jones, Helen Kang, Samuel H. Kim, YeEun Kim, Sandy Klemm, Ramen Kundu, Soumya Kundu, Mauro Lago-Docampo, Yannick C. Lee-Yow, Roni Levin-Konigsberg, Daniel Y. Li, Dominik Lindenhofer, X. R. Ma, Georgi K. Marinov, Gabriella E. Martyn, Chloe V. McCreery, Eyal Metzl-Raz, João P. Monteiro, Michael T. Montgomery, Kristy S. Mualim, Chad J. Munger, Glen Munson, Tri C. Nguyen, Tri C. Nguyen, Brian T. Palmisano, Anusri Pampari, Marlene Rabinovitch, Markus Ramste, Judhajeet Ray, Kevin R. Roy, Oriane M. Rubio, Julia M. Schaepe, Gavin Schnitzler, Jacob M Schreiber, Disha Sharma, Maya U. Sheth, Huitong Shi, Vasundhara Singh, Riya Sinha, Lars M Steinmetz, Jason Tan, Anthony Tan, Josh Tycko, Raeline Valbuena, Valeh Valiollah Pour Amiri, Mariëlle J. F. M. van Kooten, Alun Vaughan-Jackson, Anthony Venida, Chad S. Weldy, Matthew D. Worssam, Fan Xia, David Yao, Tony Zeng, Quanyi Zhao, Ronghao Zhou, Zitong Sam Chen, B. Cimini, Georges Coppin, Atina G. Coté, Marzieh Haghighi, Tong Hao, David E. Hill, Jessica Lacoste, Florent Laval, Chloe Reno, Fritz Roth, Shantanu Singh, Kerstin Spirohn-Fitzgerald, Mikko Taipale, Tanisha Teelucksingh, Maxime Tixhon, Anupama Yadav, Zhipeng Yang, W. L. Kraus, Daniel A. Armendariz, Ashley E. Dederich, Ashlesha Gogate, Lauretta El Hayek, S. Goetsch, Kiran Kaur, Hyung Bum Kim, Melissa K. McCoy, Mpathi Z. Nzima, C. Pinzon-Arteaga, Bruce A. Posner, Daniel A. Schmitz, Sushama Sivakumar, Anjana Sundarrajan, Lei Wang, Yihan Wang, Jun Wu, Lin Xu, Jian Xu, Leqian Yu, Yanfeng Zhang, Huan Zhao, Qinbo Zhou, Hyejung Won, Jessica L Bell, K. A. Broadaway, Katherine N. Degner, A. Etheridge, Beverly H. Koller, Won Mah, Wancen Mu, Kimberly D. Ritola, Jonathan D. Rosen, Sarah A. Schoenrock, Rachel A. Sharp, Daniel Bauer, G. Lettre, Richard Sherwood, Basheer Becerra, Logan J. Blaine, Eric Che, M. Francoeur, Ellie N. Gibbs, Nahye Kim, Emily M. King, B. Kleinstiver, Estelle Lecluze, Zhijian Li, Zain M. Patel, Quang Vinh Phan, Jayoung Ryu, Marlena L. Starr, Ting Wu, C. Gersbach, Gregory E. Crawford, Andrew S. Allen, William H. Majoros, Nahid Iglesias, Ruhi Rai, Revathy Venukuttan, Boxun Li, Taylor Anglen, Lexi R. Bounds, Marisa C. Hamilton, Siyan Liu, Sean R. McCutcheon, Christian McRoberts Amador, Samuel J. Reisman, Maria A. ter Weele, Josephine C. Bodle, Helen L Streff, Keith Siklenka, Kari Strouse, Bradley E. Bernstein, Juliana Babu, Guillermo Barreto Corona, Kevin Dong, Fabiana M. Duarte, Neva C. Durand, Charles B. Epstein, K. Fan, Elizabeth A. Gaskell, Amelia W. Hall, Alexandra M. Ham, Mei K. Knudson, N. Shoresh, Siddarth Wekhande, Cassandra M. White, Wang Xi, Ansuman T. Satpathy, M. Corces, Serena H. Chang, Iris M. Chin, James M Gardner, Zachary A. Gardell, Jacob C. Gutierrez, Alia W. Johnson, Lucas Kampman, Maya Kasowski, Caleb A. Lareau, Vincent Liu, Leif S. Ludwig, Christopher S. McGinnis, Shreya Menon, Anita Qualls, Katalin D. Sandor, Adam W. Turner, C. J. Ye, Yajie Yin, Wenxi Zhang, Barbara J. Wold, Maria Carilli, Dayeon Cheong, Ghassan Filibam, Kim Green, S. Kawauchi, Charlene Kim, Heidi Liang, Rebekah Loving, L. Luebbert, Grant MacGregor, Angel G. Merchan, Elisabeth Rebboah, Narges Rezaie, Jasmine Sakr, Delaney K. Sullivan, Nikki Swarna, Diane Trout, S. Upchurch, Ryan Weber, Christopher P. Castro, Elysia Chou, Fan Feng, Andre Guerra, Yuanhao Huang, Linghua Jiang, Jie Liu, Ryan E. Mills, Weizhou Qian, Tingting Qin, Maureen A. Sartor, Rintsen N. Sherpa, Jinhao Wang, Yiqun Wang, Joshua D. Welch, Zhenhao Zhang, Nanxiang Zhao, Sayan Mukherjee, C. D. Page, Shannon Clarke, Richard W. Doty, Yuncheng Duan, Raluca Gordân, Kuei-Yueh Ko, Shengyu Li, Boyao Li, Alexander Thomson, Soumya Raychaudhuri, Alkes Price, Thahmina A. Ali, Arun Durvasula, M. Kellis, L. Iakoucheva, Tulika Kakati, Yile Chen, Mariam Benazouz, Shantanu Jain, Daniel Zeiberg, M. C. De Paolis Kaluza, Michelle Velyunskiy, Audrey Gasch, Kunling Huang, Yiyang Jin, Qiongshi Lu, Jiacheng Miao, Michael Ohtake, Eduardo Scopel, Robert D. Steiner, Yuriy Sverchkov, Zhiping Weng, Manuel Garber, Yu Fu, Natalie Haas, Xihao Li, Nishigandha Phalke, Shuo C. Shan, Nicole Shedd, Tianxiong Yu, Yi Zhang, Hufeng Zhou, Alexis Battle, Livnat Jerby, Eran Kotler, Andrew R. Marderstein, Stephen B. Montgomery, AkshatKumar Nigam, Evin M. Padhi, Aman Patel, Jonathan K. Pritchard, Ivy Raine, Vivekanandan Ramalingam, Kameron B. Rodrigues, Jacob M Schreiber, Arpita Singhal, Marisa Abundis, Deepa Bisht, Trirupa Chakraborty, Jingyu Fan, David R. Hall, Zarifeh H. Rarani, Abhinav K. Jain, Babita Kaundal, Swapnil Keshari, Daniel McGrail, Nicholas A. Pease, Vivian F. Yi, Hao Wu, Sreeram Kannan, Hongjun Song, Jingli Cai, Ziyue Gao, Ronni Kurzion, Julia I. Leu, Fan Li, Dongming Liang, Guo-Li Ming, K. Musunuru, Qi Qiu, Junwei Shi, Yijing Su, Sarah Tishkoff, Ning Xie, Qian Yang, Wenli Yang, Hongjie Zhang, Zhijian Zhang, Michael A Beer, A. Hadjantonakis, Sharon Adeniyi, Hyein Cho, Ronald Cutler, Rachel A. Glenn, David Godovich, Nan Hu, Svetlana Jovanic, Renhe Luo, Jin Woo Oh, Milad Razavi-Mohseni, Dustin Shigaki, Simone Sidoli, Thomas Vierbuchen, Xianming Wang, Breanna Williams, Jielin Yan, Dapeng Yang, Yunxiao Yang, Maike Sander, Bing Ren, W. Bartosik, Hannah S Indralingam, A. Klie, Hannah M. Mummey, Mei-Lin Okino, Gaowei Wang, Nathan R. Zemke, Kai Zhang, Han Zhu, Noah Zaitlen, Jason Ernst, J. Langerman, Terence Li, Yu Sun, Alexander Y. Rudensky, P. Periyakoil, Vianne R. Gao, Melanie H. Smith, Norman M. Thomas, L. Donlin, Amit Lakhanpal, Kaden M. Southard, Rico C. Ardy, J. M. Cherry, Mark B. Gerstein, Kalina Andreeva, Pedro Assis, Beatrice Borsari, Eric Douglass, Shengcheng Dong, I. Gabdank, Keenan Graham, Otto Jolanki, Jennifer Jou, Meenakshi S. Kagda
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>4</td>
          <td>110</td>
        </tr>

        <tr id="Children with Down syndrome (DS) and B-cell acute lymphoblastic leukemia (B-ALL) are at increased risk for treatment-related mortality and relapse, highlighting the need for new therapies. Leukemia cell lines (CLs) have been fundamental to understanding therapeutic responses to pharmacological agents. We generated three DS B-ALL CLs characterized with diverse genomic alterations, including IGH::CRLF2 rearrangement (BCR::ABL1 like), mutations in FLT3 and TP53, and a novel ERG::CEBPD rearrangement. DS CLs had diminished proliferation, metabolism, and mitochondrial function when compared to non-DS (NDS) CLs and interestingly, these findings were similar to NDS Philadelphia chromosome-like (Ph-like) B-ALL CLs. Based on similar mitochondrial defects and prior preclinical data using Venetoclax for Ph-like B-ALL, we hypothesized that Venetoclax would be effective in DS. Intriguingly, Venetoclax was more effective in DS when compared to both NDS and Ph-like CLs. Efficacy was observed in DS patient derived xenografts (PDXs) and diagnostic/relapsed patient samples treated with Venetoclax, which synergized with Trametinib and Vincristine. Mass spectrometry-based multiomics analyses in DS and NDS B-ALL patient samples revealed an enriched metabolite profile in DS, particularly in the hubs of glucose metabolism and polyunsaturated phosphatidylcholines and phosphatidylinositols. Transcriptome analyses in DS B-ALL patients (n=249) supported enhanced glucose and fatty acid metabolism; along with a reliance on BCL2 family members for apoptosis resistance. Venetoclax synergized with inhibition of glucose metabolism in DS B-ALL CLs. In summary, we have generated novel tools for studying DS B-ALL and identify altered metabolism in DS that responds to Venetoclax.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f404242cc6274ce2ef0f772a4cb26039fa167f" target='_blank'>
              Metabolic vulnerabilities in Down syndrome B-cell acute lymphoblastic leukemia can be targeted using Venetoclax
              </a>
            </td>
          <td>
            Dallas Jones, Elena Woods, Zhenhua Li, Kristin L. Schaller, Eric Hoffmeyer, Ben Kooiman, Joseph Fernandez, Tegan Wharton, Dejene M. Tufa, Spencer C. Hall, George Trahan, Kelly W. Maloney, Holly L. Pacenta, Kelly Sullivan, Joaquin Espinosa, James R. Roede, John O. Marentette, Brett M Stevens, Anagha Inguva, D. Stephenson, J. Reisz, Angelo D’Alessandro, Craig T. Jordan, Jun J Yang, Michael R Verneris
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Mitochondrial dysfunction is an important factor in the pathogenesis of schizophrenia. However, the relationship between mitophagy and schizophrenia remains to be elucidated. Methods Single-cell RNA sequencing datasets of peripheral blood and brain organoids from SCZ patients and healthy controls were retrieved. Mitophagy-related genes that were differentially expressed between the two groups were screened. The diagnostic model based on key mitophagy genes was constructed using two machine learning methods, and the relationship between mitophagy and immune cells was analyzed. Single-cell RNA sequencing data of brain organoids was used to calculate the mitophagy score (Mitoscore). Results We found 7 key mitophagy genes to construct a diagnostic model. The mitophagy genes were related to the infiltration of neutrophils, activated dendritic cells, resting NK cells, regulatory T cells, resting memory T cells, and CD8 T cells. In addition, we identified 12 cell clusters based on the Mitoscore, and the most abundant neurons were further divided into three subgroups. Results at the single-cell level showed that Mitohigh_Neuron established a novel interaction with endothelial cells via SPP1 signaling pathway, suggesting their distinct roles in SCZ pathogenesis. Conclusion We identified a mitophagy signature for schizophrenia that provides new insights into disease pathogenesis and new possibilities for its diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212bca9cf10404ea1cfa1f21a9877804c45832e8" target='_blank'>
              Analyses of single-cell and bulk RNA sequencing combined with machine learning reveal the expression patterns of disrupted mitophagy in schizophrenia
              </a>
            </td>
          <td>
            Wei Yang, Kun Lian, Jing Ye, Yuqi Cheng, Xiufeng Xu
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Psychiatry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background: Melanoma remains one of the most challenging cancers to treat effectively with drug resistant remaining a constant concern, primarily with activating BRAF mutations. Mutations in the BRAF gene appear in approximately 50% of patients, 90% of which are V600E. Two frontline BRAF inhibitors (BRAFi), vemurafenib and dabrafenib, are frequently used to treat unresectable or metastatic BRAF V600E melanoma. Initial response rates are high, but soon thereafter, 70-80% of patients develop resistance to treatment within a year. A major mechanism of resistance is the generation of a secondary Q61K mutation in the NRAS gene. Methods: We have developed an approach in which a CRISPR-Cas complex can be designed to distinguish between mutant genes enabling resistance to standard care in tumor cells and normal genomes of healthy cells. For the first time, we demonstrated the utility of two CRISPR-directed mutation-specific editing approaches to restore BRAFi sensitivity in BRAFV600E/NRASQ61K resistant A375 cells. Results: We utilize an AsCas12a protospacer adjacent motif site created by the NRAS Q61K mutation and the Q61K mutation in the critical seed region of an SaCas9 sgRNA for Q61K-selective targeting. We show here that both approaches allow for effective NRAS targeting of only mutated-Q61K and after CRISPR-directed Q61K-targeting, previously resistant A375 cells are re-sensitized to BRAFi treatment. Conclusion: Our data support the feasibility of the development of CRISPR-Cas therapeutic approaches to the treatment of melanoma. Successful therapeutic CRISPR-directed gene editing would enable both specific and efficient editing of a mutation-specific targeting approach eliminate concern for on- and off-target damage to the genomes of healthy cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0aea0b595d066144e478150c4bdebe3df9d61c5" target='_blank'>
              Mutation-Specific CRISPR Targeting with SaCas9 and AsCas12a Restores Therapeutic Sensitivity in Treatment-Resistant Melanoma.
              </a>
            </td>
          <td>
            Brett M. Sansbury, Sophia B. Masciarelli, Salma Kaouser, Olivia M. Tharp, Kelly Banas, Eric B. Kmiec
          </td>
          <td>2024-10-10</td>
          <td>The CRISPR journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Recent advancements in spatial biology have revolutionized our understanding of the organization and functional dynamics of cells and tissues. In this study, we applied Molecular Pixelation (MPX), a single-cell spatial proteomics assay, to investigate the modulation of the cell surface proteome in an in vitro drug screening model using the ETV6::RUNX1 acute lymphoblastic leukemia (ALL) cell line, Reh. Specifically, we focused on the in vitro response to fludarabine, a chemotherapeutic agent used prior to allogenic stem cell transplantation and chimeric antigen receptor (CAR)-T cell therapy in high-risk, refractory, or relapsed ALL patients. Using MPX, we quantified changes in protein abundance, spatial distribution, and colocalization of 76 targeted cell surface proteins in Reh cells before and after fludarabine treatment. Our analysis revealed 25 proteins with altered abundance, 24 proteins with increased polarity, and 138 protein pairs with modified colocalization following treatment. Notably, the tetraspanins CD82 and CD53, which are known for their roles in chemotherapy resistance, exhibited increased abundance, polarization, and colocalization post-treatment, suggesting their potential as a therapeutic scaffold. These findings underscore the unique ability of spatially resolved single-cell proteomics to uncover nuanced cellular responses that would otherwise remain undetected.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5a352c544200f32c43afeccf0c1c6f723f3d81" target='_blank'>
              Mapping the Spatial Proteome of Leukemia Cells Undergoing Fludarabine Treatment
              </a>
            </td>
          <td>
            , Henrik Gezelius, Anna Pia Enblad, Anders Lundmark, Mariya Lysenkova Wiklander, O. Krali, Martin Åberg, Arja Harila, A. Raine, Claes Andersson, J. Nordlund
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D’Amato, Yi Xue, Kristina Fontanez, Aaron A. May-Zhang, Trinity Smithers, Y. Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The complexity of cancer requires a comprehensive approach to understand its diverse manifestations and underlying mechanisms. Initially outlined by Hanahan and Weinberg in 2000 and updated in 2010, the hallmarks of cancer provide a conceptual basis for understanding inherent variability in cancer biology. Recent expansions have further elucidated additional hallmarks, including phenotypic plasticity and senescent cells. The International Agency for Research on Cancer (IARC) has identified the key characteristics of carcinogens (KCCs) to evaluate their carcinogenic potential. We analyzed chemicals of concern for environmental exposure that interact with specific receptors to induce genomic instability, epigenetic alterations, immune suppression, and receptor-mediated effects, thereby contributing to chronic inflammation. Despite their varying degrees of carcinogenicity, these chemicals have similar KCC profiles. Our analysis highlights the pivotal role of receptor binding in activating most other KCCs, underscoring their significance in cancer initiation. Although KCCs are associated with early molecular or cellular events, they do not encompass processes directly linked to full cellular malignancy. Thus, there is a need to integrate clear endpoints that anchor KCCs to the acquisition of a complete malignant phenotype into chemical testing. From the perspective of toxicology and cancer research, an all-encompassing strategy that incorporates both existing and novel KCCs and cancer hallmarks is essential to enable the targeted identification of prevalent carcinogens and facilitate zone-specific prevention strategies. To achieve this goal, collaboration between the KCC and cancer hallmarks communities becomes essential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a462cc73a8d245ed18a298dc440b46e953accd9" target='_blank'>
              Key characteristics of carcinogens meet hallmarks for prevention-cutting the Gordian knot
              </a>
            </td>
          <td>
            S. Senga, William H. Bisson, Annamaria Colacci
          </td>
          <td>2024-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetic intra-tumour heterogeneity (gITH) is a universal property of all cancers. It emerges from the interplay of cell division, mutation accumulation and selection with important implications for the evolution of treatment resistance. Theoretical and data-driven approaches extensively studied gITH in ageing somatic tissues or cancers at detection. Yet, the expected patterns of gITH during and after treatment are less well understood. Here, we use stochastic birth-death processes to investigate the expected patterns of gITH across different treatment scenarios. We consider homogeneous treatment response with shrinking, growing and stable disease, and follow up investigating heterogeneous treatment response with sensitive and resistant cell types. We derive analytic expressions for the site frequency spectrum, the total mutational burden and the single-cell mutational burden distribution that we validate with computer simulations. We find that the SFS after homogeneous treatment response retains its characteristic power-law tail, while emergent resistant clones cause peaks corresponding to their sizes. The frequency of the largest resistant clone is subdominant and independent of the population size at detection, whereas the relative total number of resistant cells increases with detection size. Furthermore, the growth dynamics under treatment determine whether the total mutational burden is dominated by preexisting or newly acquired mutations, suggesting different possible treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b068763b28793a776bfaf3fe0647aae0b724cad" target='_blank'>
              On the patterns of genetic intra-tumour heterogeneity before and after treatment
              </a>
            </td>
          <td>
            Alexander Stein, Benjamin Werner
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 At the Gene Editing Institute, we are advancing a novel strategy for the treatment of pancreatic ductile adenocarcinoma by employing CRISPR-directed gene editing to disable the functions of key genes that block effective drug application including chemotherapy, radiation and immunotherapy. No matter how careful one is at designing a specific drug against a specific protein, ultimately, the tumor cell develops a workaround. A paradigm shift is required to truly overcome this formidable barrier. We’ve accomplished this by leveraging the effectiveness of standard care and augmenting it, thereby reducing patient suffering, increasing treatment tolerance and, most importantly, improving outcomes. With an enthusiastic response for this concept from the FDA- INTERFACT meeting, we have demonstrated that a carefully selected CRISPR/Cas complex disables the NRF2 gene in the tumor cell and significantly reduces the dose of chemotherapy required to halt pancreatic tumor cell growth, and consequently promoting cell death. The efficiency of gene editing approaches 70 % routinely, which destroys the function of NRF2 and enables additional killing of residual tumor cells by chemotherapeutic agents. As a consequence of CRISPR/Cas activity, we effectively reduce the amount of chemotherapy needed to kill tumor cells by over 20-fold. This level of reduction should enable patients to complete their treatment regimens more successfully, at lower dosages and with far fewer adverse side effects. In addition to reduced suffering, this will invariably prolong survival as we are augmenting already proven treatments. This approach is agnostic to the haplotype of the tumor cell and does not rely on specific biomarker composition; radically expanding the patient population that would be suitable for this type of treatment. We will discuss the impact of CRISPR/Cas-gene editing to disable specific genes, including NRF2 and mutant KRAS, to enable current therapies to work at greater efficacy and at lower dosages.
 Citation Format: London P McGill, Kelly H Banas, Gregory Tiesi, Eric B Kmiec. A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/159a6092b196d29302afbdfaca37a6f4ece01dff" target='_blank'>
              Abstract A010: A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma
              </a>
            </td>
          <td>
            L. McGill, Kelly Banas, Gregory Tiesi, Eric B. Kmiec
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Cell morphologies represent a phenotypic integration of intrinsic cellular processes, changes in cell state, and interactions with neighboring cells. While analyses of large cell line collections such as the Cancer Dependency Map (DepMap) have greatly enhanced discovery of novel molecular biomarkers and therapeutic targets, morphological analysis of these models has not yet been performed. Hence, there is a rich opportunity to discover novel regulators and functional properties of cell morphologies by linking image-based profiling to -omics based assays. Here, we studied the transcriptional and functional properties of 16 human pancreatic cancer cell lines, associations with distinct cell morphologies, and treatment-induced remodeling. At baseline, we observed extensive morphological heterogeneity across and within cell lines, as well as differences in multicellular organization. We categorized cell line morphologies into epithelial, mesenchymal, and spheroid, and organizational patterns into tightly aggregated, multilayered, and dispersed. We performed differential expression analysis across morphological subtypes utilizing DepMap transcriptomics data and queried functional correlates including CRISPR dependency, drug sensitivity, and proclivity for metastasis. For example, we discovered that the mesenchymal morphology and multilayered organizational pattern [WH3] were associated with metastatic potential, while expressing reduced levels of tight junction and cell adhesion genes. We then explored treatment associated morphological changes and their potential as cost-effective biomarkers for cell-intrinsic resistance to chemotherapy (e.g., 5-FU and gemcitabine) and KRAS inhibitors (e.g., MRTX1133 and RMC- 6236). Cells were treated continuously and monitored with live cell imaging (Incucyte SX5) followed by Cell Painting at the three-day endpoint. We observed conserved morphological responses to therapy across cell lines including a shift towards spindle-like cell shapes with neurite-like projections in chemotherapy-treated conditions, and a cell bloating phenomena characterized by an increase in cell area in KRAS inhibitor treated conditions. To directly link transcriptional state to morphology, we performed 1000-plex spatial molecular imaging (Nanostring CosMx) at subcellular resolution to pools of cell lines, demonstrating that morphological diversity within cell lines corresponded to distinct transcriptional states. In conclusion, this study highlights the potential of harnessing cell morphological information in a rapid, cost-effective phenotyping assay to aid precision oncology efforts leveraging patient- derived in vitro models.
 Citation Format: Dennis Gong, William L Hwang. Morphological diversity predicts functional traits in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B076.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ee8c48f6e344128f81cda79e9fefc7cbd605a4f" target='_blank'>
              Abstract B076: Morphological diversity predicts functional traits in pancreatic cancer
              </a>
            </td>
          <td>
            Dennis Gong, W. L. Hwang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background and aims: Colorectal carcinomas (CRCs) in patients with Lynch syndrome (LS) exhibit heightened immunogenicity due to mismatch repair deficiency (MMR-d), often resulting in favorable responses to T cell immune checkpoint therapies. Recent studies indicate that the phenotype and genotype of LS-associated CRCs vary depending on the specific MMR gene mutated. Here, we investigated whether the immune profiles of LS-associated CRCs differ based on the MMR gene defects. Methods: Tissue material from 18 MLH1-, 16 MSH2-, 40 MSH6-, and 23 PMS2-mutated CRCs and 35 sporadic MMR-d CRCs were included in the study. Imaging mass cytometry (IMC) analysis, along with targeted multiplex immunofluorescence imaging (mIF) and immunohistochemistry, were applied to examine the tumor immune microenvironment, including Human Leukocyte Antigen (HLA) class I and programmed death-ligand 1 (PD-L1) expression. Results: Unsupervised hierarchical clustering of cell phenotypes identified by IMC, followed by mIF validation, revealed comparable lymphoid and myeloid cell infiltration levels across CRCs from all MMR groups. Infiltrating T cell levels negatively correlated with the number of mutations at coding microsatellite sequences, particularly in MLH1-mutated CRCs. HLA class I defects were observed in 76% of all CRCs. These defects were more frequently accompanied by {beta}2M defects in hereditary MMR-d CRCs (67%) compared to sporadic MMR-d CRCs (37%), and did not associate with the number of {gamma}{delta} T cells, which were present in CRCs from all MMR groups. PD-L1 expression in tumor cells was only detected in 8% of all CRCs. Conclusion: Our findings illustrate that, from an immunological perspective, there is no evidence of differing immunogenic features across MMR defects. This is important to consider when developing preventive vaccine strategies and evaluating immunotherapy for LS patients and those with MMR-d CRCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0cdd08de3e8c7b38b9b131001cb730c9f4dade" target='_blank'>
              Immunological profiles in Lynch syndrome colorectal cancers are not specific to mismatch repair gene defects
              </a>
            </td>
          <td>
            Noah C. Helderman, M. Ijsselsteijn, Madalina Cabuta, Manon van der Ploeg, T. Wezel, A. Ahadova, Matthias Kloor, H. Morreau, Maartje Nielsen, N. F. D. Miranda
          </td>
          <td>2024-08-28</td>
          <td>None</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="In the fields of medicine and bioscience, gene editing is increasingly recognized as a promising therapeutic approach for treating pathogenic variants in humans and other living organisms. With advancements in technology and knowledge, it is now understood that most genetic defects are caused by single-base pair variants. The ability to substitute genes using genome editing tools enables scientists and doctors to cure genetic diseases and disorders. Starting with CRISPR (clustered regularly interspaced short palindromic repeats)/Cas, the technology has evolved to become more efficient and safer, leading to the development of base and prime editors. Furthermore, various approaches are used to treat genetic disorders such as hemophilia, cystic fibrosis, and Duchenne muscular dystrophy. As previously mentioned, most genetic defects leading to specific diseases are caused by single-base pair variants, which can occur at many locations in corresponding gene, potentially causing the same disease. This means that, even when using the same genome editing tool, results in terms of editing efficiency or treatment effectiveness may differ. Therefore, different approaches may need to be applied to different types of diseases. Prevalently, due to the safety of adeno-associated virus (AAV) vectors in gene therapy, most clinical trials of gene therapy are based on AAV delivery methods. However, despite their safety and nonintegration into the host genome, their limitations, such as confined capacity, dosage-dependent viral toxicity, and immunogenicity, necessitate the development of new approaches to enhance treatment effects. This review provides the structure and function of each CRISPR-based gene editing tool and focuses on introducing new approaches in gene therapy associated with improving treatment efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175c7ca99490d5250dd5cadc163e90b91db26af6" target='_blank'>
              Mechanism of Genome Editing Tools and Their Application on Genetic Inheritance Disorders
              </a>
            </td>
          <td>
            Dae Hwan Oh
          </td>
          <td>2024-09-16</td>
          <td>Global Medical Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Signaling pathways induce stereotyped transcriptional changes as stem cells progress into mature cell types during embryogenesis. Signaling perturbations are necessary to discover which genes are responsive or insensitive to pathway activity. However, gene regulation is additionally dependent on cell state-specific factors like chromatin modifications or transcription factor binding. Thus, transcriptional profiles need to be assayed in single cells to identify potentially multiple, distinct perturbation responses among heterogeneous cell states in an embryo. In perturbation studies, comparing heterogeneous transcriptional states among experimental conditions often requires samples to be collected over multiple independent experiments. Datasets produced in such complex experimental designs can be confounded by batch effects. We present Design-Aware Integration of Single Cell ExpEriments (DAISEE), a new algorithm that models perturbation responses in single-cell datasets with a complex experimental design. We demonstrate that DAISEE improves upon a previously available integrative non-negative matrix factorization framework, more efficiently separating perturbation responses from confounding variation. We use DAISEE to integrate newly collected single-cell RNA-sequencing datasets from 5-hour old zebrafish embryos expressing optimized photoswitchable MEK (psMEK), which globally activates the extracellular signal-regulated kinase (ERK), a signaling molecule involved in many cell specification events. psMEK drives some cells that are normally not exposed to ERK signals towards other wild type states and induces novel states expressing a mixture of transcriptional programs, including precociously activated endothelial genes. ERK signaling is therefore capable of introducing profoundly new gene expression states in developing embryos. Significance Statement Signaling perturbations produce heterogeneous transcriptional responses that must be measured at the single-cell level. Data integration techniques allow us to model these responses which, however, can be confounded by batch effects. We present a computational tool (DAISEE) for extracting the common and perturbation-specific features of single-cell datasets representing multiple experimental conditions while achieving efficient batch effect correction. DAISEE outperforms its predecessor and will enable accurate analysis of a broad range of single-cell datasets. DAISEE applied to new single-cell RNA sequencing data from zebrafish embryos shows that gain-of-function signaling perturbations can induce novel states. Our analysis suggests that a wild type endothelial cell-specification program can be activated in abnormal developmental contexts when the extracellular signal-regulated kinase (ERK) pathway is deregulated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9843baa0dda4952c851210ee811a954b61353c56" target='_blank'>
              Disrupted developmental signaling induces novel transcriptional states
              </a>
            </td>
          <td>
            Aleena Patel, Vanessa Gonzalez, Triveni Menon, Stanislav Y. Shvartsman, R. Burdine, M. Avdeeva
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Prostate cancer (PC) is a leading cause of death in men because of the high incidence and long-term inefficacy of the existing treatment options. Furthermore, it exhibits significant health disparities that affect African-American (AA) men more adversely than others do. Previously, we established CYP3A5, a highly expressed protein in AAs PC, as a positive regulator of androgen receptor (AR) signaling. We examined the impact of CYP3A5 depletion on genome-wide transcriptional output using RNA sequencing to gain deeper mechanistic insights. The data revealed that 561 genes were downregulated and 263 were upregulated upon silencing of CYP3A5 in PC cells. Furthermore, in silico pathway analyses of differentially expressed genes suggested that the cell cycle regulation pathway was most significantly affected by CYP3A5 inhibition. Cell cycle analysis of CYP3A5-silenced cells and those treated with clobetasol, a specific CYP3A5 pharmacological inhibitor, showed G1/S phase blockade. Both CYP3A5-depletion and pharmacological inhibition resulted in the downregulation of cyclin D, cyclin B, and CDK2, along with the upregulation of p27kip1 but had minimal effects on CDK4/6 levels. Combination treatment with clobetasol and the CDK4/6 inhibitor palbociclib exhibited synergy with combination index (CI) values ranging from 0.28-0.78. Our findings support the utility of CYP3A5 as a druggable therapeutic target that works more effectively in combination with CDK4/6 inhibition to limit the progression of PC, especially for AA patients with AA. This combination addresses CDK4/6 inhibitor resistance, which is often linked to CDK2 overexpression, and can potentially be useful in reducing disparities in the clinical outcomes of PC. Significance Our study highlights CYP3A5 as a key regulator of the cell cycle in prostate cancer (PC). Its overexpression in African American (AA) patients may be a key molecular driver of disparities in outcomes. The combination of CYP3A5 and CDK4/6 inhibitors shows a synergistic effect on therapeutic outcomes and addresses CDK2-mediated resistance. Thus, targeting both CYP3A5 and CDK4/6 could improve treatment outcomes, especially in AA PC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04bad07ffad0a8a7cbaae50a2fcf79091b02b95e" target='_blank'>
              CYP3A5 inhibition causes G1/S blockade and synergizes with CDK4/6 inhibitor to suppress prostate cancer cell growth: Implications in reducing health disparity
              </a>
            </td>
          <td>
            Jeetesh Sharma, Imran K. Mohammed, R. Tillett, Jake McLean, Shirley Shen, Ajay P. Singh, Oscar B. Goodman, Edwin C. Oh, R. Mitra
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Tumorigenesis is commonly attributed to Darwinian processes involving natural selection among cells and groups of cells. However, progressing tumors are those that also achieve an appropriate group phenotypic composition (GPC). Yet, the selective processes acting on tumor GPCs are distinct from that associated with classical Darwinian evolution (i.e. natural selection based on differential reproductive success) as tumors are not genuine evolutionary individuals and do not exhibit heritable variation in fitness. This complex evolutionary scenario is analogous to the recently proposed concept of ‘selection for function’ invoked for the evolution of both living and non-living systems. Therefore, we argue that it is inaccurate to assert that Darwinian processes alone account for all the aspects characterizing tumorigenesis and cancer progression; rather, by producing the genetic and phenotypic diversity required for creating novel GPCs, these processes fuel the evolutionary success of tumors that is dependent on selection for function at the tumor level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed9ff2b803aab7c7484cdca4ce05128b6e07445" target='_blank'>
              A new perspective on tumor progression
              </a>
            </td>
          <td>
            Frédéric Thomas, James DeGregori, A. Marusyk, Antoine M. Dujon, Beata Ujvari, Jean-Pascal Capp, Robert Gatenby, A. Nedelcu
          </td>
          <td>2024-09-19</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Global healthcare systems have a great challenge in the form of inherited hematological diseases, which necessitates the development of new remedial strategies. By precisely targeting inherited abnormalities, Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-mediated ex vivo gene editing has surfaced as a promising approach to treat these diseases. This review offers a comprehensive examination of the advancements, challenges, and clinical eventuality of CRISPR-Cas9-intermediated ex vivo gene editing for inherited hematological diseases. With advancements in CRISPR-Cas9 technology, the eventuality to correct inheritable mutations responsible for inherited hematological diseases is within reach. However, challenges such as off-target effects, immune responses, and ethical considerations need to be addressed for the safe and effective perpetration of this technology. A promising understanding of how CRISPR-Cas9-intermediated gene editing functions in practice is handed by ongoing clinical studies, giving rise to the possibility of advanced remedial approaches and bettered patient issues. By addressing these complications in a human-readable format, this review attempts to provide greater understanding and appreciation for the eventuality of CRISPR-Cas9 technology in revolutionizing the treatment landscape for these challenging disorders and contribute to the ongoing discussion in the field and facilitate further exploration towards effective treatments for these challenging disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d496cce2c3ddc8c2b97b6be801d229f168c15" target='_blank'>
              CRISPR-CAS9-MEDIATED EX VIVO GENE EDITING FOR INHERITED HEMATOLOGICAL DISORDERS: ADVANCEMENTS, CHALLENGES, AND CLINICAL POTENTIAL
              </a>
            </td>
          <td>
            Atasi Ranjan Panda, Shreeya Das
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Pharmacy and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary This study aimed to determine the gene expression profiles associated with chemotherapeutic responses in conventional osteosarcomas (COS) within South Africa. We observed a significant downregulation in the ATP binding cassette subfamily C members (ABCC3 and ABCB1-p-glycoprotein), excision repair cross-complimenting group 1 (ERCC 1), replication factor C subunit 1 (RFC1), and tumour protein 53 (p53) genes in the COS tumours compared to the healthy donors. Furthermore, an upregulated ERCC1 gene expression level predicted a poor chemotherapeutic response. Additionally, the predictors of COS chemotherapeutic response comprised age, chondroblastic and osteoblastic histological subtypes, and ABCC3, ERCC1, and RFC1 gene expression. Abstract Background: We determined the predictive gene expression profiles associated with chemo-response in conventional osteosarcomas (COS) within South Africa. Materials and methods: In 28 patients, we performed an RNA extraction, cDNA synthesis, and quantitative analysis using the RT-PCR 2−∆∆CT method to determine the fold change in gene expression alongside GAPDH (housekeeping gene). Results: We observed a significant downregulation in the mRNA expression profiles of ABCB1-p-glycoprotein (p = 0.0007), ABCC3 (p = 0.002), ERCC1 (p = 0.007), p-53 (p = 0.007), and RFC1 (p = 0.003) in the COS patients compared to the healthy donors. Furthermore, ABCB1-p-glycoprotein (p = 0.008) and ABCC3 (p = 0.020) exhibited a significant downregulation in the COS tumour tissues when compared to the healthy donors. In our univariate logistic regression, the predictors of chemotherapeutic response comprised ERCC1 [restricted cubic spline (RCS) knot: OR −0.27; CI −0.504 to −0.032; p = 0.036]; osteoblastic subtype [OR −0.36; CI −0.652 to −0.092; p = 0.026); fibroblastic subtype [OR 0.91; CI 0.569 to 1.248; p < 0.001]; and mixed subtype [OR 0.53; CI 0.232 to 0.032; p = 0.032]. In our multivariable logistic regression, the significant predictors of chemotherapeutic response comprised age [RCS knot: OR −2.5; CI −3.616 to −1.378; p = 0.022]; ABCC3 [RCS knot: OR 0.67; CI 0.407 to 0.936, p = 0.016]; ERCC1 [RCS knot: OR 0.57; CI 0.235 to 0.901; p = 0.044]; RFC1 [RCS knot: OR −1.04; CI −1.592 to −0.487; p = 0.035]; chondroblastic subtype [OR −0.83; CI −1.106 to −0.520; p = 0.012]; and osteoblastic subtype [OR −1.28; CI −1.664 to −0.901; p = 0.007]. Conclusions: In this South African cohort, we observed the unique gene expression profiles of osteosarcoma tumourigenesis and chemotherapeutic responses. These may serve as prognostication and therapeutic targets. Larger-scale research is needed on the African continent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66cda9efe1f101a9fdd82c5518590f36a3653f2a" target='_blank'>
              Unique Gene Expression Profiles within South Africa Are Associated with Varied Chemotherapeutic Responses in Conventional Osteosarcoma
              </a>
            </td>
          <td>
            P. G. Mthethwa, Thilona Arumugam, V. Ramsuran, Anmol Gokul, R. Rodseth, Leonard Marais
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Background Various epigenetic regulations systematically govern gene expression in cells involving various biological processes. Dysregulation of the epigenome leads to aberrant transcriptional programs and subsequently results in diseases, such as cancer. Therefore, comprehensive profiling epigenomics is essential for exploring the mechanisms underlying gene expression regulation during development and disease. Methods In this study, we developed single‐cell chromatin proteins and accessibility tagmentation (scCPA‐Tag), a multi‐modal single‐cell epigenetic profile capturing technique based on barcoded Tn5 transposases and a droplet microfluidics platform. scCPA‐Tag enables the simultaneous capture of DNA profiles of histone modification and chromatin accessibility in the same cell. Results By applying scCPA‐Tag to K562 cells and a hepatocellular carcinoma (HCC) sample, we found that the silence of several chromatin‐accessible genes can be attributed to lysine‐27‐trimethylation of the histone H3 tail (H3K27me3) modification. We characterized the epigenetic features of the tumour cells and different immune cell types in the HCC tumour tissue by scCPA‐Tag. Besides, a tumour cell subtype (C2) with more aggressive features was identified and characterized by high chromatin accessibility and a lower abundance of H3K27me3 on tumour‐promoting genes. Conclusions Our multi‐modal scCPA‐Tag provides a comprehensive approach for exploring the epigenetic landscapes of heterogeneous cell types and revealing the mechanisms of gene expression regulation during developmental and pathological processes at the single‐cell level. Highlights scCPA‐Tag offers a highly efficient and high throughput technique to simultaneously profile histone modification and chromatin accessibility within a single cell. scCPA‐Tag enables to uncover multiple epigenetic modification features of cellular compositions within tumor tissues. scCPA‐Tag facilitates the exploration of the epigenetic landscapes of heterogeneous cell types and provides the mechanisms governing gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1d48ebd31bec69ee99957ebc5de9bec997e3ad" target='_blank'>
              Single‐cell multi‐modal chromatin profiles revealing epigenetic regulations of cells in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Chunqing Wang, Waidong Huang, Yu Zhong, Xuanxuan Zou, Shang Liu, Jie Li, Yunfan Sun, Kaiqian Zhou, Xi Chen, Zihao Li, Shanshan Wang, Yaling Huang, Yinqi Bai, Jianhua Yin, Xin Jin, Shiping Liu, Yue Yuan, Q. Deng, Miaomiao Jiang, Chuanyu Liu, Longqi Liu, Xun Xu, Liang Wu
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Understanding the molecular mechanisms underlying tumour progression and resistance to treatment is crucial to the development of new treatment approaches. In high risk pediatric cancers, somatic DNA alterations such as genomic amplifications, copy number alterations, translocations or mutations play an important role as molecular diagnostic, prognostic and predictive biomarkers. However it is now known that in many high risk pediatric cancers, clonal evolution most likely plays an important role in tumor progression and treatment resistance. Circulating tumour DNA, a fraction of cell free DNA can be readily isolated from plasma and now provides an important tool and surrogate for tumor molecular analyses at diagnosis, during treatment and follow-up. At diagnosis, the prospective clinical trials MICCHADO (NCT03496402) enrolled 599 patients, including high-risk pediatric cancer (Neuroblastoma, Rhabdomyosarcoma, Ewing sarcoma, other high-risk cancers). Whole -Exome-Sequencing (WES) was performed on tumor, germline DNA and cfDNA extracted from plasma at diagnosis, during treatment and follow up. Whereas all cfDNA samples obtained at follow-up in patients without evidence of disease revealed no or few tumor cell-specific SNVs, cfDNA samples obtained at relapse harbored additional, new relapse-specific SNVs in all cases, targeting genes of interest. Deep sequencing capture techniques enable to develop models of clonal evolution. In pediatric embryonal brain tumours, ctDNA isolated from CSF enables detection of tumor cell specific genetic alterations with a high sensitivity. At relapse, ctDNA studies can provide complementary information to molecular analyses of tumour samples performed within programs such as MAPPYACTS (NCT02613962), with 76% of actionable alterations detected in tumor also identified in ctDNA, while also highlighting the importance of spatial heterogeneity. MONALISA, a SIOPEN pragmatic clinical trial to MOnitor NeuroblastomA relapse with LIquid biopsy Sensitive Analysis aims to establish liquid biopsies as standard-of-care to monitor relapsed/refractory neuroblastoma. Reliable, early assessment of molecular progression or relapse determined by mRNA and/or ctDNA analysis is the main aim of this randomized clinical trial. ctDNA also enables to infer expression profiles. Gene expression levels are reflected by nucleosome positioning, and differences in nucleosome organization at transcription start sites (TSS) lead to differential clipping of fragments upon ctDNA release and distinct nucleosome footprints depending on the expression of a given gene in the originating cells. Altogether the presence of tumor genetic and epigenetic abnormalities in ctDNA can be documented in most patients with high risk pediatric cancer and frequently suggest spatial and temporal heterogeneity. Sequential studies will further elucidate mechanisms of clonal evolution, tumor progression and therapy resistance. Thus, sequential studies based on liquid biopsies are now integrated into the development and optimization of targeted treatment strategies.
 Citation Format: Gudrun Schleiermacher. Opportunities and challenges for liquid biopsies in pediatric oncology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d5d9b1f8524120f05332a3107841ee2c1bb8ff8" target='_blank'>
              Abstract IA010: Opportunities and challenges for liquid biopsies in pediatric oncology
              </a>
            </td>
          <td>
            G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d7559338158829d62b6b4147b9bfe3e99b724d9" target='_blank'>
              HLA-G neo-expression modifies genetic programs governing tumor cell lines
              </a>
            </td>
          <td>
            D. Tronik-Le Roux, M. Daouya, I. Poras, François Desgrandchamps, E. Carosella
          </td>
          <td>2024-10-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="To comprehensively unravel the temporal relationship between initiating and driver events and its impact on clinical outcomes, we analyzed 421 whole-genome sequencing profiles from 382 patients. Using clock-like mutational signatures, we estimated a time lag of 2-4 decades between initiating events and diagnosis. In patients with hyperdiploidy, we demonstrate that trisomies of odd-numbered chromosomes can be acquired simultaneously with other chromosomal gains, such as 1q gain. We provide evidence that hyperdiploidy is acquired after canonical IGH translocation when both events are present. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra-copies), but faring worse than those with late 1q gain. This underscores that the prognostic impact of 1q gain/amp depends more on the timing of acquisition than on the number of extra copies gained. Overall, this study contributes to a better understanding of the life history of MM and may have prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98e16f03f4610ad7609f6b41fac703153b2f8eb" target='_blank'>
              Temporal Genomic Dynamics Shape Clinical Trajectory in Multiple Myeloma
              </a>
            </td>
          <td>
            Francesco Maura, Marcella Kaddoura, A. Poos, L. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, Hartmut Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth Morgan, S. Rajkumar, Shaji Kumar, S. Usmani, O. Landgren, Marc-Steffen Raab, Niels Weinhold
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d175f8dd810d13a4f1256bdc9f679caa427edf2" target='_blank'>
              Systematic screening of protein-coding gene expression identified VWF as a potential key regulator in anthracycline-based chemotherapy-exacerbated metastasis of breast cancer
              </a>
            </td>
          <td>
            Yawei Zhao, Meihui He, Lianzhi Cui, Min Zhang, Tianyu Zhao, Xuehan Yang, Yang Xu, Jianhua Dong, Kan He, Hansi Zhang, Li Chen
          </td>
          <td>2024-10-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal primary brain tumor in adults and is driven by self-renewing glioblastoma stem cells (GSCs) that persist after therapy and seed treatment refractory recurrent tumors. GBM tumors display a high degree of intra- and inter-tumoral heterogeneity that is a prominent barrier to targeted treatment strategies. This heterogeneity extends to GSCs that exist on a gradient between two transcriptional states or subtypes termed developmental and injury-response. Drug targets for each subtype are needed to effectively target GBM. To identify conserved and subtype-specific genetic dependencies across a large and heterogeneous panel of GSCs, we designed the GBM5K targeted gRNA library and performed fitness screens in a total of 30 patient-derived GSC cultures. The focused CRISPR screens identified the most conserved subtype-specific vulnerabilities in GSCs and elucidated the functional dependency gradient existing between the developmental and injury-response states. Developmental-specific fitness genes were enriched for transcriptional regulators of neurodevelopment, whereas injury-response-specific fitness genes were highlighted by several genes implicated in integrin and focal adhesion signaling. These context-specific vulnerabilities conferred differential sensitivity to inhibitors of β1 integrin, FAK, MEK and OLIG2. Interestingly, the screens revealed that the subtype-specific signaling pathways drive differential cyclin D (CCND1 vs. CCND2) dependencies between subtypes. These data provide biological insight and mechanistic understanding of GBM heterogeneity and point to opportunities for precision targeting of defined GBM and GSC subtypes to tackle heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43d6812955301efd762e7b039babebcb9586d37" target='_blank'>
              Fitness Screens Map State-Specific Glioblastoma Stem Cell Vulnerabilities.
              </a>
            </td>
          <td>
            G. Macleod, Fatemeh Molaei, Shahan Haider, Maira P Almeida, Sichun Lin, M. Kushida, Haresh Sureshkumar, Jasmine K Bhatti, Jack Q Lu, Daniel Schramek, Peter B Dirks, Stéphane Angers
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer. Highlights Sinabro, the sequential mutation simulator, facilitates measuring the robustness of human protein-coding sequences under all COSMIC mutational signatures. Robustness under APOBEC mutational signatures showed the largest mean and standard deviation in the human genome. Robustness to mutational signatures analysis reveals the role of APOBECs is complementary to cancer in the evolvability of cancer cells in later stages. Principal component analysis indicates that the GC content at the codon’s wobble position significantly influences robustness. A long-lived bat (Myotis myotis) has higher robustness to APOBECs than a short-lived one (Molossus molossus) than humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011df1029d192af53d074e293c75b5f6a3b83940" target='_blank'>
              Human and bats genome robustness under COSMIC mutational signatures
              </a>
            </td>
          <td>
            Joon-Hyun Song, Ying Zeng, Liliana M. Dávalos, Thomas MacCarthy, Mani Larijani, Mehdi Damaghi
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Recent studies demonstrate growing roles for genetic mosaicism in neurodevelopmental and neuropsychiatric disorders, with the paradigm being drug-resistant pediatric focal epilepsy related to activating somatic variants in the PI3K-mTOR pathway. While identifying the genotype-associated changes at the single-cell level is fundamental to understanding disease pathophysiology, this remains technically challenging in human tissue samples with existing methods. Here, we performed single-nucleus RNA-sequencing (snRNA-seq) of 20 focal cortical dysplasia (FCD) samples removed surgically for treatment of drug-resistant epilepsy, and 10 non-FCD controls, and we developed a new approach, Genotyping Of Transcriptomes Enhanced with Nanopore sequencing (GO-TEN), that combines targeted complementary (c)DNA sequencing with snRNA-seq to perform concurrent single-nucleus genotyping and transcriptional analysis. We find that mosaic pathogenic variants in FCD do not produce a detectable novel cell identity, but instead we observe conserved cell types present both in FCD cases and non-FCD control specimens. Similarly, GO-TEN analysis shows that most pathogenic variant-carrying cells have well-differentiated neuronal or glial identities and are enriched for layer II-III excitatory neurons. We identify cell-intrinsic disruption of glutamate and GABA-A signaling pathways in variant-carrying neurons and altered intercellular signaling, making potential mechanisms for epileptogenesis in FCD. In summary, by addressing genotype-specific changes in mosaic epilepsy-associated lesions, our study highlights new potential disease mechanisms and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc181f7065f3cb1a379b3d4368b2d029b87b1b3b" target='_blank'>
              Cell-type-informed genotyping of mosaic focal epilepsies reveals cell-autonomous and non-cell-autonomous disease-associated transcriptional programs
              </a>
            </td>
          <td>
            S. Bizzotto, E. Stronge, Maya Talukdar, Qiwen Hu, Zinan Zhou, A. Huang, B. Chhouk, Alissa M. D’Gama, Edward Yang, T. E. Green, David C. Reutens, Saul A. Mullen, Michael S. Hildebrand, Russell J Buono, A. Poduri, S. Khoshkhoo, Christopher A. Walsh
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Most phenotype-associated genetic variants map to non-coding regulatory regions of the human genome. Moreover, variants associated with blood cell phenotypes are enriched in regulatory regions active during hematopoiesis. To systematically explore the nature of these regions, we developed a highly efficient strategy, Perturb-multiome, that makes it possible to simultaneously profile both chromatin accessibility and gene expression in single cells with CRISPR-mediated perturbation of a range of master transcription factors (TFs). This approach allowed us to examine the connection between TFs, accessible regions, and gene expression across the genome throughout hematopoietic differentiation. We discovered that variants within the TF-sensitive accessible chromatin regions, while representing less than 0.3% of the genome, show a ∼100-fold enrichment in heritability across certain blood cell phenotypes; this enrichment is strikingly higher than for other accessible chromatin regions. Our approach facilitates large-scale mechanistic understanding of phenotype-associated genetic variants by connecting key cis-regulatory elements and their target genes within gene regulatory networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd75703a3e0196006c63fecef421b6d1dc25cee5" target='_blank'>
              Transcription factor networks disproportionately enrich for heritability of blood cell phenotypes
              </a>
            </td>
          <td>
            Jorge D. Martin-Rufino, Alexis Caulier, Seayoung Lee, Nicole Castano, Emily King, Samantha Joubran, Marcus Jones, Seth R Goldman, Uma P. Arora, Lara Wahlster, Eric S. Lander, V. Sankaran
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 Pediatric cancers are frequently driven by genomic alterations that result in aberrant transcription factor activity. While direct targeting of transcription factors is difficult, one method to circumvent this problem is to target co-factors and chromatin complexes that are necessary for their oncogenic function. To evaluate protein complex dependencies enriched in MYCN-amplified neuroblastoma, a disease of dysregulated development driven by aberrant expression of an oncogenic expression program, we curated a list of protein complexes using the CORUM database and mined the Dependency Map using single sample gene set enrichment analysis. This analysis identified the KAT module of the transcriptional coactivator Spt-Ada-Gcn5-acetyltransferase (SAGA) complex as a selective dependency in MYCN-amplified neuroblastoma. Loss of SAGA complex activity in neuroblastoma through both genetic knockout and induced protein degradation of the lysine acetyltransferase (KAT) module subunit TADA2B resulted in a global reduction of histone lysine acetylation and impaired neuroblastoma cell growth. Genetic knockout of TADA2B did not impair the growth of pediatric sarcoma cell lines, suggesting that SAGA loss-of-function is not broadly deleterious. Furthermore, SAGA loss-of-function reduced MYCN binding to chromatin, resulting in suppression of the MYCN-driven gene expression program and impaired cell cycle progression. Importantly, the SAGA complex is pharmacologically targetable in vitro and in vivo with GSK-699, a PROTAC that induces proteolysis of both KAT2A and KAT2B paralogs. As observed with genetic knockout studies, loss of SAGA activity through KAT2A/B degradation reduced the MYCN-driven transcriptional signature and impaired cell cycle progression. Our findings expand our understanding of the chromatin complexes that maintain the oncogenic transcriptional state in MYCN-amplified neuroblastoma and suggest therapeutic potential for inhibitors of SAGA KAT activity in MYCN-amplified neuroblastoma.
 Citation Format: Nathaniel W. Mabe, Clare F. Malone, Alexandra B. Forman, Gabriela Alexe, Kathleen L. Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, Sharon YR Dent, Kimberly Stegmaier. Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be30c933e33904854e0410865fe3ee874fab512e" target='_blank'>
              Abstract A062 Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program
              </a>
            </td>
          <td>
            Nathaniel W. Mabe, Clare F Malone, Alexandra B Forman, G. Alexe, Kathleen Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, S. Y. Dent, K. Stegmaier
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea593405bbe5868b6d752054463ac22a293113d" target='_blank'>
              Ex vivo disease modelling of Rett syndrome: the transcriptomic and metabolomic implications of direct neuronal conversion.
              </a>
            </td>
          <td>
            B. Karaosmanoğlu, Gozde Imren, M. S. Ozisin, T. Reçber, P. Ö. Şimşek Kiper, G. Haliloglu, Mehmet Alikaşifoğlu, E. Nemutlu, E. Taşkıran, G. Utine
          </td>
          <td>2024-09-13</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract The p53 tumor suppressor gene governs a multitude of complex cellular processes that are essential for anti-cancer function and whose dysregulation leads to aberrant gene transcription, activation of oncogenic signaling and cancer development. Although mutations can occur at any point in the genetic sequence, missense mutations comprise the majority of observed p53 mutations in cancers regardless of whether the mutation is germline or somatic. One biological process involved in both mutant and wild-type p53 signaling is the N6-methyladenosine (m6A) epitranscriptomic network, a type of post-transcriptional modification involved in over half of all eukaryotic mRNAs. Recently, a significant number of findings have demonstrated unique interactions between p53 and the m6A epitranscriptomic network in a variety of cancer types, shedding light on a previously uncharacterized connection that causes significant dysregulation. Cross-talk between wild-type or mutant p53 and the m6A readers, writers and erasers has been shown to impact cellular function and induce cancer formation by influencing various cancer hallmarks. Here, this review aims to summarize the complex interplay between the m6A epitranscriptome and p53 signaling pathway, highlighting its effects on tumorigenesis and other hallmarks of cancer, as well as identifying its therapeutic implications for the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e91fbc5b77b6bf7efe4bda3817610c3bb0dedde" target='_blank'>
              Decoding the molecular symphony: interactions between the m6A and p53 signaling pathways in cancer
              </a>
            </td>
          <td>
            R. Shoemaker, Mo-Fan Huang, Ying-Si Wu, Cheng-Shuo Huang, Dung-Fang Lee
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Preclinical models such as cell lines and mice are the backbone of drug development and experimental-mechanistic oncology. However, we currently lack a detailed understanding of the direct clinical relevance of data collected in most preclinical models, hampering the development of new treatments. Despite this, few formal approaches have been proposed to determine how the various preclinical models represent/resemble primary patient tumors. Here, we present the first comprehensive single-cell RNA-seq analysis of neuroblastoma across an extensive cohort of patient tumors and a variety of preclinical model systems (n = 126 total samples assembled – the largest cohort of its kind). By developing an unsupervised machine learning method, which we term “automatic consensus nonnegative matrix factorization” (acNMF), we have integrated and contrasted the transcriptional landscapes of patient tumors with those of cell lines, patient-derived xenografts (PDX), and genetic mouse models (GEMM). We discovered that the dominant adrenergic gene expression programs commonly found in neuroblastoma patient tumors were generally preserved across all preclinical models. However, the presumptive chemo-resistant mesenchymal-like programs, while identifiable in cell lines, were primarily restricted to subpopulations of cancer-associated fibroblasts and Schwann-like cells in vivo. Surprisingly however, a mesenchymal-like program could be acutely chemotherapy-induced in GEMM and was evident in pre-treated patient and PDX samples, suggesting a previously uncharacterized mechanism of therapy escape resulting from an acute shift in cell state. In addition, our approach could further delineate the classical neuroblastoma adrenergic and mesenchymal gene expression programs, discovering for example, novel subpopulations of cancer associated fibroblasts and reproducible subtypes of adrenergic programs. These behaviors were conserved across tumors and preclinical models, which we validated by RNA in situ hybridization, an ultra-sensitive, high resolution, spatial transcriptomics technology. Overall, we offer a nuanced, high-resolution view of neuroblastoma pre-clinical systems for advancing therapeutic development, as well as a generalizable set of computational tools, which can be applied in other diseases. We have created an open-source web resource, featuring this integrated map to aid the scientific community in further exploration of these integrated data (available at http://pscb.stjude.org).
 Citation Format: Richard H. Chapple, Xueying Liu, Sivaraman Natarajan, Margaret I.M. Alexander, Yuna Kim, Anand G. Patel, Christy W. LaFlamme, Min Pan, William C. Wright, Hyeong-Min Lee, Yinwen Zhang, Meifen Lu, Selene C. Koo, Courtney Long, John Harper, Chandra Savage, Melissa D. Johnson, Thomas Confer, Walter J. Akers, Michael A. Dyer, Heather Sheppard, John Easton, Paul Geeleher. An integrated single-cell RNA-seq map of human neuroblastoma tumors and preclinical models uncovers divergent mesenchymal-like gene expression programs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b38392d6531b277e9f9ee236ca213de0c124f13" target='_blank'>
              Abstract B006: An integrated single-cell RNA-seq map of human neuroblastoma tumors and preclinical models uncovers divergent mesenchymal-like gene expression programs
              </a>
            </td>
          <td>
            Richard H. Chapple, Xueying Liu, Sivaraman Natarajan, M. I. Alexander, Yuna Kim, Anand G. Patel, Christy W. LaFlamme, Min Pan, William C. Wright, Hyeong-Min Lee, Yinwen Zhang, Meifen Lu, Selene C. Koo, Courtney Long, John Harper, Chandra Savage, Melissa D. Johnson, Thomas Confer, Walter J. Akers, Michael A. Dyer, Heather Sheppard, John Easton, P. Geeleher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eddab72a32f5e67003082a5b17959fbc71f348a1" target='_blank'>
              Gene therapy for polygenic or complex diseases
              </a>
            </td>
          <td>
            Tingting Wu, Yu Hu, Liang V Tang
          </td>
          <td>2024-09-04</td>
          <td>Biomarker Research</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Base editing shows promise for the correction of human mutations at a higher efficiency than other repair methods and is especially attractive for mutations in large genes that are not amenable to gene augmentation therapy. Here, we demonstrate a comprehensive workflow for in vitro screening of potential therapeutic base editing targets for the USH2A gene and empirically validate the efficiency of adenine and cytosine base editor/guide combinations for correcting 35 USH2A mutations. Editing efficiency and bystander edits are compared between different target templates (plasmids versus transgenes) and assays (Next generation sequencing versus Sanger), as well as comparisons between unbiased empirical results and computational predictions. Based on these observations, practical assay recommendations are discussed. Finally, a humanized knock-in mouse model was created with the best-performing target, the nonsense mutation c.11864G>A p.(Trp3955*). Split-intein AAV9 delivery of editing reagents resulted in the restoration of USH2A protein and a correction rate of 65 ± 3% at the mutant base pair and of 52 ± 3% excluding bystander amino acid changes. This efficiency compares favorably to a prior genome editing strategy tested in the retina that completed a clinical trial and demonstrates the effectiveness of this overall strategy to identify and test base editing reagents with the potential for human therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b11e55d59e66e72cd37b7653ddc5c10089e197" target='_blank'>
              Systematic empirical evaluation of individual base editing targets: validating therapeutic targets in USH2A and comparison of methods
              </a>
            </td>
          <td>
            Yuki Tachida, K. Manian, Rossano Butcher, Jonathan M. Levy, Nachiket C. Pendse, E. Hennessey, David R. Liu, Eric A. Pierce, Qin Liu, J. Comander
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="If one must prioritize among the vast array of contributing factors to cancer evolution, environmental-stress-mediated chromosome instability (CIN) should easily surpass individual gene mutations. CIN leads to the emergence of genomically unstable life forms, enabling them to grow dominantly within the stable life form of the host. In contrast, stochastic gene mutations play a role in aiding the growth of the cancer population, with their importance depending on the initial emergence of the new system. Furthermore, many specific gene mutations among the many available can perform this function, decreasing the clinical value of any specific gene mutation. Since these unstable life forms can respond to treatment differently than stable ones, cancer often escapes from drug treatment by forming new systems, which leads to problems during the treatment for patients. To understand how diverse factors impact CIN-mediated macroevolution and genome integrity–ensured microevolution, the concept of two-phased cancer evolution is used to reconcile some major characteristics of cancer, such as bioenergetic, unicellular, and multicellular evolution. Specifically, the spiral of life function model is proposed, which integrates major historical evolutionary innovations and conservation with information management. Unlike normal organismal evolution in the microevolutionary phase, where a given species occupies a specific location within the spiral, cancer populations are highly heterogenous at multiple levels, including epigenetic levels. Individual cells occupy different levels and positions within the spiral, leading to supersystems of mixed cellular populations that exhibit both macro and microevolution. This analysis, utilizing karyotype to define the genetic networks of the cellular system and CIN to determine the instability of the system, as well as considering gene mutation and epigenetics as modifiers of the system for information amplification and usage, explores the high evolutionary potential of cancer. It provides a new, unified understanding of cancer as a supersystem, encouraging efforts to leverage the dynamics of CIN to develop improved treatment options. Moreover, it offers a historically contingent model for organismal evolution that reconciles the roles of both evolutionary innovation and conservation through macroevolution and microevolution, respectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b4dc885b10926269c63b9c0c31f322d9ccdff48" target='_blank'>
              The Spiral Model of Evolution: Stable Life Forms of Organisms and Unstable Life Forms of Cancers
              </a>
            </td>
          <td>
            Andrzej Kasperski, Henry H. Heng
          </td>
          <td>2024-08-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56963ef797b360ca0fd377e89bb8feeef832b223" target='_blank'>
              Genome-wide characterization of dynamic DNA 5-hydroxymethylcytosine and TET2-related DNA demethylation during breast tumorigenesis
              </a>
            </td>
          <td>
            Shuang-Ling Wu, Lin Yang, Changcai Huang, Qing Li, Chunhui Ma, Fang Yuan, Yinglin Zhou, Xiaoyue Wang, Wei-Min Tong, Yamei Niu, Feng Jin
          </td>
          <td>2024-09-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is characterized by its aggressive behavior and complex molecular heterogeneity, posing significant challenges for treatment and prognostication. This study offers a comprehensive analysis of ccRCC by leveraging both bulk and single-cell RNA sequencing data, with a specific aim to unravel the complexities of sphingolipid metabolism and the intricate dynamics within the tumor microenvironment (TME). By examining ccRCC samples sourced from public databases, our investigation delves deep into the genetic and transcriptomic landscape of this cancer type. Employing advanced analytical techniques, we have identified pivotal patterns in gene expression and cellular heterogeneity, with a special focus on the roles and interactions of various immune cells within the TME. Significantly, our research has unearthed insights into the dynamics of sphingolipid metabolism in ccRCC, shedding light on its potential implications for tumor progression and strategies for immune evasion. A novel aspect of this study is the development of a risk score model designed to enhance prognostic predictions for ccRCC patients, which is currently pending external validation to ascertain its clinical utility. Despite its contributions, the study is mindful of its limitations, including a reliance on observational data from public sources and a primary focus on RNA sequencing data, which may constrain the depth and generalizability of the findings. The study does not encompass critical aspects, such as protein expression, posttranslational modifications, and comprehensive metabolic profiles. Moreover, its retrospective design underscores the necessity for future prospective studies to solidify these preliminary conclusions. Our findings illuminate the intricate interplay between genetic alterations, sphingolipid metabolism, and immune responses in ccRCC. This research not only enhances our understanding of the molecular foundations of ccRCC but also paves the way for the development of targeted therapies and personalized treatment modalities. The study underlines the importance of cautious interpretation of results and champions ongoing research using diverse methodologies to thoroughly comprehend and effectively combat this formidable cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d290ea11babde7963cf17fade3be71816bf90d65" target='_blank'>
              Integrative analysis of bulk and single-cell RNA sequencing reveals sphingolipid metabolism and immune landscape in clear cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Dongdong Xie, Zhitao Han, Yu Wang, Haoyu Shi, Xiang Wu, Jiaqing Wu, Y. Dai
          </td>
          <td>2024-09-04</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6bcaf46c997cceee1e5ce02e4d68cb36cec179" target='_blank'>
              Targeting mRNA-coding genes in prostate cancer using CRISPR/Cas9 technology with a special focus on androgen receptor signaling
              </a>
            </td>
          <td>
            Mobina Tabibian, Fahimeh Salasar Moghaddam, E. Motevaseli, S. Ghafouri-Fard
          </td>
          <td>2024-10-17</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 High-grade gliomas (HGGs) are the most aggressive type of gliomas and have the poorest outcomes. Chromatin remodeling (CR) genes have been implicated in multiple oncogenic pathways in numerous cancer types. In gliomagenesis, CR genes have been implicated in regulating stemness of glioma cells, the tumor microenvironment (TME), and resistance to therapies.



 We performed molecular profiling of 4244 HGGs and evaluated associations of CR mutations with other cancer related biomarkers, infiltration by immune cells, and immune gene expression. We also evaluated the association between CR mutations and survival in IDH WT HGG patients.



 Nearly 10% of HGGs carry mutations in CR genes, with higher prevalence (15%) in HGGs with IDH mutations. Analysis of co-occurrence with other biomarkers revealed that CR-mutated HGGs possess favorable genetic alterations which may have prognostic value. CR-mutated HGGs with wild type IDH demonstrated colder TME and worse OS overall compared to the CR-wild type HGGs.



 Our study reveals the prognostic effects of CR mutations in HGG and points to several biomarker candidates that could suggest sensitivity to emerging therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a6ec214f04da421e41c3579984b7e7f5d8d94d" target='_blank'>
              Biological and prognostic relevance of epigenetic regulatory genes in high-grade gliomas (HGGs)
              </a>
            </td>
          <td>
            Sonikpreet Aulakh, Joanne Xiu, Andrew Hinton, S. Darabi, M. Demeure, S. Sengupta, Santosh Kesari, David M Ashley, Ashley Love Sumrall, M. Glantz, D. Spetzler
          </td>
          <td>2024-10-04</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the frequent tumors that seriously endanger the physical and mental well-being in women with strong heterogeneity, and its pathogenesis involves multiple risk factors. Depending on the type of BC, hormonal therapy, targeted therapy, and immunotherapy are the current systemic treatment options along with conventional chemotherapy. Despite significant progress in understanding BC pathogenesis and therapeutic options, there is still a need to identify new therapeutic targets and develop more effective treatments. According to recent sequencing and profiling studies, non-coding (nc) RNAs genes are deregulated in human cancers via deletion, amplification, abnormal epigenetic, or transcriptional regulation, and similarly, the expression of many ncRNAs is altered in breast cancer cell lines and tissues. The ability of single ncRNAs to regulate the expression of multiple downstream gene targets and related pathways provides a theoretical basis for studying them for cancer therapeutic drug development and targeted delivery. Therefore, it is far-reaching to explore the role of ncRNAs in tumor development and their potential as therapeutic targets. Here, our review outlines the potential of two major ncRNAs, long non-coding RNAs (lncRNAs) and microRNAs (miRNAs) as diagnostic and prognostic biomarkers as well as targets for new therapeutic strategies in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c75f06853600c6ed1247db1e2526cd7f187129" target='_blank'>
              Potential therapies for non-coding RNAs in breast cancer
              </a>
            </td>
          <td>
            Ruonan Li, Yuxin Ji, Ruyin Ye, Guohui Tang, Wenrui Wang, Changjie Chen, Qingling Yang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The detection of mitochondrial DNA (mtDNA) mutations in single cells holds considerable potential to define clonal relationships coupled with information on cell state in humans. Previous methods focused on higher heteroplasmy mutations that are limited in number and can be influenced by functional selection, introducing biases for lineage tracing. Although more challenging to detect, intermediate to low heteroplasmy mtDNA mutations are valuable due to their high diversity, abundance, and lower propensity to selection. To enhance mtDNA mutation detection and facilitate fine-scale lineage tracing, we developed the single-cell Regulatory multi-omics with Deep Mitochondrial mutation profiling (ReDeeM) approach, an integrated experimental and computational framework. Recently, some concerns have been raised about the analytical workflow in the ReDeeM framework. Specifically, it was noted that the mutations detected in a single molecule per cell are enriched on edges of mtDNA molecules, suggesting they resemble artifacts reported in other sequencing approaches. It was then proposed that all mutations found in one molecule per cell should be removed. We detail our error correction method, demonstrating that the observed edge mutations are distinct from previously reported sequencing artifacts. We further show that the proposed removal leads to massive elimination of bona fide and informative mutations. Indeed, mutations accumulating on edges impact a minority of all mutation calls (for example, in hematopoietic stem cells, the excess mutations on the edge account for only 4.3%-7.6% of the total). Recognizing the value of addressing edge mutations even after applying consensus correction, we provide an additional filtering option in the ReDeeM-R package. This approach effectively eliminates the position biases, leads to a mutational signature indistinguishable from bona fide mitochondrial mutations, and removes excess low molecule high connectedness mutations. Importantly, this option preserves the large majority of unique mutations identified by ReDeeM, maintaining the ability of ReDeeM to provide a more than 10-fold increase in variant detection compared to previous methods. Additionally, the cells remain well-connected. While there is room for further refinement in mutation calling strategies, the significant advances and biological insights provided by the ReDeeM framework are unique and remain intact. We hope that this detailed discussion and analysis enables the community to employ this approach and contribute to its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94914bd4d265668ea63fab9b4a1aa10d1231c91c" target='_blank'>
              Robustness and reliability of single-cell regulatory multi-omics with deep mitochondrial mutation profiling
              </a>
            </td>
          <td>
            Chen Weng, Jonathan S. Weissman, V. Sankaran
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Despite a critical role for tumor-initiating cancer stem cells (CSCs) in breast cancer progression, major questions remain about the properties and signaling pathways essential for their function. Recent discoveries highlighting mechanisms of CSC-resistance to the stress caused by chromosomal instability (CIN) may provide valuable new insight into the underlying forces driving stemness properties. While stress tolerance is a well-known attribute of CSCs, CIN-induced stress is distinctive since levels appear to increase during tumor initiation and metastasis. These dynamic changes in CIN levels may serve as a barrier constraining the effects of non-CSCs and shaping the stemness landscape during the early stages of disease progression. In contrast to most other stresses, CIN can also paradoxically activate pro-tumorigenic antiviral signaling. Though seemingly contradictory, this may indicate that mechanisms of CIN tolerance and pro-tumorigenic inflammatory signaling closely collaborate to define the CSC state. Together, these unique features may form the basis for a critical relationship between CIN and stemness properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d491eba9730d9558923c4e53a3264dd576b3117a" target='_blank'>
              Chromosomal instability as an architect of the cancer stemness landscape
              </a>
            </td>
          <td>
            Shahnawaz A. Baba, Aran Zakeri, J. Desgrosellier
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Hepatocellular adenomas (HAs) are tumors that can develop under different conditions, including in patients harboring a germline mutation in HNF1A. However, little is known about the pathogenesis of such disease. This work aims to better define what mechanisms lie under the development of this condition. Six HAs were sampled from the liver of a 17-year-old male affected by diabetes and multiple hepatic adenomatosis harboring the heterozygous pathogenic germline variant c.815G>A, p.(Arg272His) in HNF1A, which has a dominant negative effect. All HAs were molecularly characterized. Four of them were shown to harbor a second somatic HNF1A variant and one had a mutation in the ARID1A gene, while no additional somatic changes were found in the remaining HA and normal parenchyma. A transcriptomic profile of the same HA samples was also performed. HNF1A biallelic mutations were associated with the up-regulation of several pathways including the tricarboxylic acid cycle, the metabolism of fatty acids, and mTOR signaling while angiogenesis, endothelial and vascular proliferation, cell migration/adhesion, and immune response were down-regulated. Contrariwise, in the tumor harboring the ARID1A variant, angiogenesis was up-modulated while fatty acid metabolism was down-modulated. Histological analyses confirmed the molecular data. Independently of the second mutation, energetic processes and cholesterol metabolism were up-modulated, while the immune response was down-modulated. This work provides a complete molecular signature of HNF1A-associated HAs, analyzing the association between specific HNF1A variants and the development of HA while identifying potential new therapeutic targets for non-surgical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26474672d542f89d0fa38cb2c1556ad93882621f" target='_blank'>
              Genomic and Transcriptomic Profile of HNF1A-Mutated Liver Adenomas Highlights Molecular Signature and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            A. Faini, F. Arruga, M. Pinon, Valeria Bracciamà, Francesco Edoardo Vallone, Fiorenza Mioli, M. Sorbini, Martina Migliorero, A. Gambella, Damiano Carota, Isaac Giraudo, P. Cassoni, Silvia Catalano, Renato Romagnoli, Antonio Amoroso, P. L. Calvo, T. Vaisitti, Silvia Deaglio
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Maintaining engineered cell populations’ genetic stability is a key challenge in synthetic biology. Synthetic genetic constructs compete with a host cell’s native genes for expression resources, burdening the cell and impairing its growth. This creates a selective pressure favouring mutations which alleviate this growth defect by removing synthetic gene expression. Non-functional mutants thus spread in cell populations, eventually making them lose engineered functions. Past work has attempted to limit mutation spread by coupling synthetic gene expression to survival. However, these approaches are highly context-dependent and must be tailor-made for each particular synthetic gene circuit to be retained. In contrast, we develop and analyse a biomolecular controller which depresses mutant cell growth independently of the mutated synthetic gene’s identity. Modelling shows how our design can be deployed alongside various synthetic circuits without any re-engineering of its genetic components, outperforming extant gene-specific mutation spread mitigation strategies. Our controller’s performance is evaluated using a novel simulation approach which leverages resource-aware cell modelling to directly link a circuit’s design parameters to its population-level behaviour. Our design’s adaptability promises to mitigate mutation spread in an expanded range of applications, whilst our analyses provide a blueprint for using resource-aware cell models in circuit design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2420835e594d4a65b9dfcd0a4c88ad37ec47058" target='_blank'>
              Model-guided gene circuit design for engineering genetically stable cell populations in diverse applications
              </a>
            </td>
          <td>
            Kirill Sechkar, Harrison Steel
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The transcriptional state of a cell provides a multi-parameter representation of gene expression programs that reflect its identity and phenotype. However, current transcriptomic profiling technologies result in destruction of the biological sample, preventing direct analysis of transcriptional dynamics in the same living cells over time. Here, we developed a synthetic RNA export system called cellular ‘self-reporting’ to address this fundamental technological limitation. We repurposed the murine leukemia virus retroviral protein Gag to enable diverse types of immortalized and primary mammalian cells to package cellular RNA molecules in virus-like particles (VLPs) for export into the extracellular environment. We applied self-reporting to interrogate the transcriptome-wide dynamics that occur during neuronal differentiation from induced pluripotent stem cells and detected gene expression profiles from individual live cells. Leveraging this genetically encodable approach, we expanded the capabilities of self-reporting through molecular engineering of VLP components. Pseudotyping VLPs with epitope-tagged envelope proteins enabled multiplexed selective live-cell readout of transcriptional states from heterogeneous co-cultures. Furthermore, structure-guided protein engineering of Gag fusions with human RNA binding domains improved the mRNA representation in self-reporting readouts and enabled the directed export of libraries of synthetic barcode transcripts. Taken together, this work establishes self-reporting as a facile and broadly enabling technology for live-cell, transcriptome-scale profiling of dynamic processes across diverse cell types and biological applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd9363ab1dcbb456eb0b3e0944d84bfdb375d0c" target='_blank'>
              Live-cell transcriptomics with engineered virus-like particles
              </a>
            </td>
          <td>
            M. Najia, Jacob Borrajo, Anna Le, FuNien Tsai, Jeremy Y. Huang, Linda G. Griffith, George Q Daley, P. Blainey
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Resistance to therapy remains a significant challenge in cancer treatment, often due to the presence of a stem-like cell population that drives tumor recurrence post-treatment. Moreover, many anticancer drugs inadvertently induce plasticity, reverting differentiated cancer cells to drug-resistant stem-like states. Addressing this phenomenon is crucial for advancing cancer therapeutics. In this study, we introduce a robust statistical framework based on multi-type branching processes that can dissect tumor dynamics and drug effects from high throughput drug screening data. Through comprehensive in silico experiments, we show the efficacy of our framework in estimating parameters governing population dynamics and drug responses in a heterogeneous tumor population where cell state transitions are influenced by the drug. Finally, using recent in vitro data involving AGS-SORE6+/- cells treated with ciclopirox olamine, we show how our framework can be used to confirm the presence of drug-induced cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85013788a6ef1ec3c633fabb86b8c0544b0624bd" target='_blank'>
              Inferring Drug-induced Plasticity via Drug Screen Data
              </a>
            </td>
          <td>
            Chenyu Wu, E. B. Gunnarsson, J. Foo, K. Leder
          </td>
          <td>2024-09-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f34d93ff1b305ad73729aa0a43e0b0a160605f" target='_blank'>
              Characterization of MYC Rearrangements in Multiple Myeloma: an Optical Genome Mapping Approach
              </a>
            </td>
          <td>
            Jung Yoon, Taesung Jeon, J. Kwon, Soo-Young Yoon
          </td>
          <td>2024-09-20</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Alterations in Fibroblast growth factor receptor (FGFR)-family proteins frequently occur as oncogenes in many cancers, including a subset of pediatric gliomas. Here, we performed a genomic analysis of 11,635 gliomas across ages and found that 4.5% of all gliomas harbor FGFR alterations including structural variants (SV) and single nucleotide variants (SNV), with an incidence of almost 10% in pediatric gliomas. FGFR family members are differentially enriched by age, tumor grade, and histological subtype, with FGFR1-alterations associated with glioneuronal histologies and pediatric low-grade gliomas. Across development, we find FGFR1 expression in both neuronal and glial precursors, while FGFR3 expression is largely restricted to astrocytic lineages. Leveraging novel isogenic model systems, we confirm FGFR1 alterations to be sufficient to activate MAPK and mTOR signaling, drive gliomagenesis, activate neuronal transcriptional programs and exhibit sensitivity to MAPK pathway inhibitors, including pan-FGFR inhibitors. Models driven by FGFR1 SVs exhibited different patterns of sensitivity compared to those driven by SNVs. Finally, we performed a retrospective analysis of clinical responses in children diagnosed with FGFR-driven gliomas and found that targeted MAPK or FGFR-inhibition with currently available inhibitors is largely associated with stability of disease. This study provides key insights into the biology of FGFR1-altered gliomas, therapeutic strategies to target them and associated challenges that still need to be overcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badc242f4fb57f142e32352b142311d05135cc1f" target='_blank'>
              A diverse landscape of FGFR alterations and co-mutations defines novel therapeutic strategies in pediatric low-grade gliomas
              </a>
            </td>
          <td>
            Eric Morin, April Apfelbaum, D. Sturm, Georges Ayoub, Jeromy J. Digiacomo, Sher Bahadur, Bhavyaa Chandarana, Phoebe C Power, Margaret Cusick, Dana Novikov, Robert Jones, J. Vogelzang, Connor Bossi, Seth W. Malinowski, John Jeang, Jared Collins, Sehee Oh, Hyesung Jeon, A. Cameron, Patrick Rechter, Angela Deleon, K. Murugesan, M. Montesion, L. Albacker, S. Ramkissoon, C. M. Tilburg, Emily C Hardin, P. Sievers, Felix Sahm, K. Yeo, Tom Rosenberg, Susan N Chi, Karen D Wright, S. Hébert, Sydney Peck, A. Picca, Valérie Larouche, Samuele Renzi, T. Bale, Amy A. Smith, M. Touat, Nada Jabado, Eric S Fischer, Michael Eck, Lissa Baird, Olaf Witt, Claudia L. Kleinman, Quang-De Nguyen, S. Alexandrescu, D. Jones, Keith L. Ligon, P. Bandopadhayay
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) poses significant challenges in veterinary medicine, with limited treatment options and poor survival rates. While substantial progress has been made in characterizing human AML, translating these advancements to veterinary practice has been hindered by limited molecular understanding and diagnostic tools. The case study presented illustrates the application of whole genome sequencing in diagnosing AML in a dog, showcasing its potential in veterinary oncology. Our approach facilitated comprehensive genomic analysis, identifying mutations in genes that may be associated with AML pathogenesis in dogs, such as KRAS, IKZF1, and RUNX1. However, without supportive evidence of its clinical utility (eg, association with response to treatment or prognosis), the information is limited to exploration. This article reviews the comparative features of canine AML with human AML and discusses strategies to shrink the knowledge gap between human and veterinary medicine with cost-effective next-generation sequencing (NGS) techniques. By utilizing these approaches, the unique and shared molecular features with human AML can be identified, aiding in molecular classification and therapeutic development for both species. Despite the promise of NGS, challenges exist in implementing it into routine veterinary diagnostics. Cost considerations, turnaround times, and the need for robust bioinformatics pipelines and quality control measures must be addressed. Most importantly, analytical and clinical validation processes are essential to ensure the reliability and clinical utility of NGS-based assays. Overall, integrating NGS technologies into veterinary oncology holds great potential for advancing our understanding of AML and improving disease stratification, in hopes of improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8bd6fdd9b3631dd5e67d1617f989917515a9291" target='_blank'>
              Advancements in genetic analysis: Insights from a case study and review of next-generation sequencing techniques for veterinary oncology applications.
              </a>
            </td>
          <td>
            R. Harris, Jillian Nolan, Dylan Ammons, Samantha Beeson, Douglas Thamm, Anne Avery
          </td>
          <td>2024-10-04</td>
          <td>Veterinary clinical pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA repair genomic aberrations in the Homologous Recombination pathway are identifiable in up to 25% of patients with advanced prostate cancer, making them more likely to benefit from treatment with poly (ADP-ribose) polymerase inhibitors (PARPi) alone or in combination with other therapies, particularly when BRCA driver genomic aberrations are documented. Although several clinical trials have demonstrated the efficacy of this approach, the validation of reliable biomarkers predictive of response still needs further improvement to refine patient selection. In this setting, the characterization of resistance mechanisms and the validation of novel biomarkers are critical to maximize clinical benefit and to develop novel treatment combinations to improve outcomes. In this review, we summarize the development of PARPi in prostate cancer as single agent as well as the efficacy of their combination with other drugs, and the future directions for their implementation in the management of advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332cf66eb0d0e0669852e95223ac366cb113738f" target='_blank'>
              PARP inhibitors alone or in combination for prostate cancer
              </a>
            </td>
          <td>
            M. D. Fenor de la Maza, J. L. Pérez Gracia, B. Miñana, Elena Castro
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Urology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Froese, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel Goulet, Michael T. Hemann. Elucidating the tumor-driven mechanisms of resistance to immunotherapies in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr B039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81348949cfc84943625452ad98cf593484f76b30" target='_blank'>
              Abstract B039: Elucidating the tumor-driven mechanisms of resistance to immunotherapies in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Froese, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel Goulet, Michael Hemann
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVES
The TP53 mutation, a prevalent tumor suppressor gene alteration, is linked to chemotherapy resistance, increased relapse rates and diminished overall survival (OS) in acute myeloid leukemia (AML) patients.


METHODS
In this study, we characterize the TP53 mutation phenotypes across various AML cohorts utilizing The Cancer Genome Atlas (TCGA) data. We devised a TP53-related prognostic signature derived from differentially expressed genes between mutated and wild-type TP53 AML specimens. In-depth analyses were conducted, encompassing genetic variation, immune cell infiltration and prognostic stratification.


RESULTS
A six-gene TP53-related signature was established using least absolute shrinkage and selection operator (LASSO)-Cox regression, demonstrating robust prognostic predictability. This signature exhibited strong performance in both the OHSU validation cohorts, an independent Gene Expression Omnibus (GEO) validation cohort (GSE71014) and proved by results of the in vivo experiment. Finally, we used single cell database (GSE198681) to observe the characteristics of these six genes.


DISCUSSION
Our study may facilitate the development of efficacious therapeutic approaches and provide a novel idea for future research. Conclusion: The TP53-related signature and pattern hold the potential to refine prognostic stratification and underscore emerging targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57809ff1c4369159931bf563a0e583afd0ffd0c" target='_blank'>
              Elucidating the immune landscape and potential prognostic model in acute myeloid leukemia with TP53 mutation.
              </a>
            </td>
          <td>
            Gelan Zhu, J. Cai, Wanbin Fu, Yue Sun, Ting Wang, Hua Zhong
          </td>
          <td>2024-09-27</td>
          <td>Hematology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Neuroendocrine transdifferentiation (NEtD), also commonly referred to as lineage plasticity, emerges as an acquired resistance mechanism to molecular targeted therapies in multiple cancer types, predominately occurs in metastatic epidermal growth factor receptor (EGFR)‐mutant non‐small cell lung cancer treated with EGFR tyrosine kinase inhibitors and metastatic castration‐resistant prostate cancer treated with androgen receptor targeting therapies. NEtD tumors are the lethal cancer histologic subtype with unfavorable prognosis and limited treatment. A comprehensive understanding of molecular mechanism underlying targeted‐induced plasticity could greatly facilitate the development of novel therapies. In the past few years, increasingly elegant studies indicated that NEtD tumors share key the convergent genomic and phenotypic characteristics irrespective of their site of origin, but also embrace distinct change and function of molecular mechanisms. In this review, we provide a comprehensive overview of the current understanding of molecular mechanism in regulating the NEtD, including genetic alterations, DNA methylation, histone modifications, dysregulated noncoding RNA, lineage‐specific transcription factors regulation, and other proteomic alterations. We also provide the current management of targeted therapies in clinical and preclinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7693a38c0d6b0d0a5a68e9cf1e1598b353f12ed8" target='_blank'>
              Neuroendocrine transdifferentiation in human cancer: molecular mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Jun Jiang, Donghui Han, Jiawei Wang, Weihong Wen, Rui Zhang, Weijun Qin
          </td>
          <td>2024-10-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Multiple myeloma is a plasma cell malignancy that accounts for 1-2% of all cancer diagnoses and 10-15% of hematopoietic neoplasms. Clonal rearrangement of the IG locus suggests a monoclonal model for pathogenesis. The presence of cytogenetic abnormalities in >90% of cases suggests transformation by chromosomal instability and rearrangement. Classification and consensus guidelines by The International Myeloma Working Group (IMWG) and others reflect this by categorizing myeloma based on ploidy, translocations, and chromosomal abnormalities.
 Interestingly, a portion of Emory’s test volume for cancer gene mutation profiling by second generation short-read sequencing is for multiple myeloma cases. This suggests our 75-gene myeloid neoplasm panel (MMP75) is being utilized for medical purposes that are not met by cytogenetic and microarray characterizations performed during the diagnostic workup of multiple myeloma. Inspired by this observation, this study investigates providers motivations for ordering MMP75 and assesses the adequacy of our current implementation for their stated purposes.
 Anecdotes from ordering physicians indicate interest in MMP75 results when informing prognosis, selecting treatments, and evaluating for treatment associated secondary neoplasms. A more formal survey is underway that will statistically capture how the results of MMP75 testing were ultimately used: to inform diagnosis, determine prognosis, guide treatment, explain progression, identify/rule-out secondary neoplasia, or no effect on management.
 We also explore the variants called by MMP75 when performed using bone marrow from patients who received a cytogenetic workup for multiple myeloma at some point. A retrospective analysis of the 5-year interval between Jan 2019 and Jan 2024 identified 156 cases meeting this criterion. The number of pathogenic variants per case ranged between 1 and 8, with mutations detected across 42 genes. The genes most frequently identified with pathogenic mutations were TP53 (22.4%), DNMT3A (19.2%), TET2 (14.7%), and ASXL1 (12.2%). All four have been associated with adverse outcomes in patients with multiple myeloma. Activating events in therapeutic targets included mutations in the RAS/RAF pathway (16.7%), JAK/STAT signaling (2.6%), and the kinase KIT (0.6%).
 While these findings support the use of MMP75 for detection of relevant mutations, the genetic variants identified are not unique to multiple myeloma. Mutations detected in DNMT3A, TET2, and ASXL1 are common in clonal hematopoiesis of indeterminant potential (CHIP) and myeloid neoplasms. Hotspot mutations in TP53 and activating events in the RAS/RAF pathway also occur in various hematologic malignancies. This test therefore detects events that occur in both the plasma cell compartment and other cells of origin. This encumbers distinguishing variants that inform on a patient’s multiple myeloma from events that suggest a secondary neoplasm. Future directions include determining whether the effect of plasma cell enrichment on a mutations variant allele frequency (VAF) in repeat testing can help distinguish true multiple myeloma variants from events in other cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/987b88a74ed80e2065d8c8a3c7edc9ab6f63f8af" target='_blank'>
              Improving the Utility of a Cancer Gene Mutation Panel for Multiple Myeloma Patients
              </a>
            </td>
          <td>
            Lalit Patel, Linsheng Zhang, Geoffrey Smith, David L. Jaye, Thomas Schneider
          </td>
          <td>2024-10-01</td>
          <td>American Journal of Clinical Pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44f2554beac8f2504f8636ae14089de94d4df8bc" target='_blank'>
              Deciphering the impact of aggregated autophagy-related genes TUBA1B and HSP90AA1 on colorectal cancer evolution: a single-cell sequencing study of the tumor microenvironment
              </a>
            </td>
          <td>
            Qianping Xu, Chao Liu, Hailin Wang, Shujuan Li, Hanshen Yan, Ziyang Liu, Kexin Chen, Yaoqin Xu, Runqin Yang, Jingfang Zhou, Xiaolin Yang, Jie Liu, Lexin Wang
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The use and publication of research utilising CRISPR/cas9 in gene editing and knock outs (KOs) within cell lines is now widespread and has proved extremely powerful in the interrogation of gene function. However, the potential of experimental artefacts resulting from the need to generate single cell clones post gene-manipulation has been somewhat overlooked. In this study, we show that the commonly used pulmonary cell line A549 displays significant heterogeneity in terms of their gene expression, and that individual cells from a population exhibit dramatically different susceptibility to viral infection from a range of viruses including respiratory syncytial virus (RSV), influenza A virus (IAV), Hazara virus (HAZV), lymphocytic choriomeningitis virus (LCMV) and Bunyamwera virus (BUNV). Moreover, we demonstrate the rapid re-emergence of phenotypic heterogeneity even after cloning. These results demonstrate the need for caution in interpreting results from CRISPR screens and CRISPR-KO validation studies, especially in the study of viral infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0e26c9049ed60692f4658ae9936a4fcc5067296" target='_blank'>
              Single cell variation and rapid emergence of phenotypic heterogeneity in cell lines- a cautionary tale for devotees of CRISPR-Cas9
              </a>
            </td>
          <td>
            Hayley Bradley, John N Barr, Martin Stacey
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dysfunction of epigenetic modulators, such as the SWI/SNF complex, is a wide-spread but relatively ill-defined feature of a broad spectrum of cancer entities. Among SWI/SNF-mutant entities, SMARCB1-deficient cancers, such as the highly aggressive Epithelioid Sarcoma (EpS), are characterized by this genetic event in an otherwise rather silent mutational landscape. This renders EpS an ideal model to study how epigenetic reprogramming by a single mutation can contribute to tumorigenesis. Hence, to characterize and compare the function of the SMARCB1-deficient, residual and the physiological SWI/SNF complex in cancer, we generated a panel of SMARCB1 re-expressing EpS cell lines and employed a functional multi-omics approach. Here, we show that SWI/SNF holds canonical characteristics of both tumor-suppressors and proto-oncogenes due to its multi-faceted role in the regulation of the epigenome. Our data indicates that the loss of SMARCB1 causes an overall loss of SWI/SNF chromatin affinity at cis-regulatory enhancer elements, inducing a preference for uncontrolled proliferation and cell cycle progression as opposed to development and differentiation. We further demonstrate that EpS cell lines depend on residual SWI/SNF action to maintain clonogenicity and proliferation. Consequently, EpS cell lines exhibit markedly increased sensitivity to pharmacological inhibition of the residual SWI/SNF when compared with SWI/SNF-proficient cancer entities. Collectively, our results from the EpS model shed new light on how a single mutation can rewire the pleiotropic effects of an epigenetic master regulator and provide inroads for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba81f1fa7d5881ab59ab9ddb07d9a1d9290af17" target='_blank'>
              Loss of SMARCB1 evokes targetable epigenetic vulnerabilities in Epithelioid Sarcoma
              </a>
            </td>
          <td>
            Jia Xiang Jin, Fabia Fuchslocher, Martha J. Carreño-Gonzalez, Felina Zahnow, A. K. Ceranski, Rainer Will, Dominic Helm, Felix Bestvater, A. Banito, R. Imle, S. Ohmura, F. Cidre-Aranaz, T. G. Grünewald
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Advances in preclinical models that recapitulate chemorefractory and relapsed disease are needed to better predict the efficacy of an expanding and promising armamentarium of drug candidates being tested in early-phase pediatric clinical trials. Here, we used longitudinal magnetic resonance imaging to design an individualized, dose-escalating treatment regimen that induces evolution of neuroblastoma in the Th-MYCN genetically-engineered mouse model, concomitant with the acquisition of resistance to temozolomide, a standard chemotherapy used in treatment of refractory, relapsed neuroblastoma patients within European early-phase clinical trials. MRI longitudinally identified the development of intra-tumoral heterogeneity. Molecular profiling of expanding, treatment-refractory regions identified prominent up-regulation of the nor-adrenergic core regulatory signature and deregulation of the CDK2 pathway. Treatment with a CDK2/9 inhibitor led to significant response and an overall survival benefit in temozolomide-resistant Th-MYCN tumors and allografts generated from these resistant tumours. These findings demonstrate the utility of genetically-engineered mouse models as platforms to dissect the evolution of chemoresistance in neuroblastoma and they provide a mechanistic rationale to support the evaluation of CDK2/9 in ongoing paediatric phase I studies of chemotherapy combined with temozolomide in relapsed, treatment refractory neuroblastoma patients.
 Citation Format: Evon Poon, Tong Liang, Giuseppe Barone, Barbara M. Costa, Sara Heuss, Simon P. Robinson, Charles Lin, Paul Workman, Yann Jamin, Louis Chesler. Acquired resistance to temozolomide in the Th-MYCN mouse as a clinically-relevant platform to evaluate novel therapeutic strategies against high-risk neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cbf60b6dfa50cf3144b983799fd4b6d993e0d1e" target='_blank'>
              Abstract A066 Acquired resistance to temozolomide in the Th-MYCN mouse as a clinically-relevant platform to evaluate novel therapeutic strategies against high-risk neuroblastoma
              </a>
            </td>
          <td>
            E. Poon, Tong Liang, Giuseppe Barone, Barbara M. Costa, Sara Heuss, Simon P. Robinson, Charles Lin, Paul Workman, Y. Jamin, Louis Chesler
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The effectiveness of drug treatments is profoundly influenced by individual responses, which are shaped by gene expression variability, particularly within pharmacogenes. Leveraging single-cell RNA sequencing (scRNA-seq) data, our study explores the extent of expression variability among pharmacogenes in a wide array of cell types across eight different human tissues, shedding light on their impact on drug responses. Our findings broaden the established link between variability in pharmacogene expression and drug efficacy to encompass variability at the cellular level. Moreover, we unveil a promising approach to enhance drug efficacy prediction. This is achieved by leveraging a combination of cross-cell and cross-individual pharmacogene expression variation measurements. Our study opens avenues for more precise forecasting of drug performance, facilitating tailored and more effective treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b12a7bcbec6921ec291037e369d1c317c0110c" target='_blank'>
              Deciphering single-cell gene expression variability and its role in drug response.
              </a>
            </td>
          <td>
            Sizhe Liu, Liang Chen
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Nonsense variations, characterized by premature termination codons, play a major role in human genetic diseases as well as in cancer susceptibility. Despite their high prevalence, effective therapeutic strategies targeting premature termination codons remain a challenge. To understand and explore the intricate mechanisms involved, we developed StopKB, a comprehensive knowledgebase aggregating data from multiple sources on nonsense variations, associated genes, diseases, and phenotypes. StopKB identifies 637 317 unique nonsense variations, distributed across 18 022 human genes and linked to 3206 diseases and 7765 phenotypes. Notably, ∼32% of these variations are classified as nonsense-mediated mRNA decay-insensitive, potentially representing suitable targets for nonsense suppression therapies. We also provide an interactive web interface to facilitate efficient and intuitive data exploration, enabling researchers and clinicians to navigate the complex landscape of nonsense variations. StopKB represents a valuable resource for advancing research in precision medicine and more specifically, the development of targeted therapeutic interventions for genetic diseases associated with nonsense variations. Database URL: https://lbgi.fr/stopkb/">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c08327c52fb4147c847231c518c5ad046b88f2c2" target='_blank'>
              StopKB: a comprehensive knowledgebase for nonsense suppression therapies
              </a>
            </td>
          <td>
            Nicolas Haas, Julie Dawn Thompson, Jean-Paul Renaud, K. Chennen, Olivier Poch
          </td>
          <td>2024-10-12</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In the ever-evolving landscape of oncology, the battle against prostate cancer (PCa) stands at a transformative juncture, propelled by the integration of molecular diagnostics into traditional cytopathological frameworks. This synthesis not only heralds a new epoch of precision medicine but also significantly enhances our understanding of the disease’s genetic intricacies. Our comprehensive review navigates through the latest advancements in molecular biomarkers and their detection technologies, illuminating the potential these innovations hold for the clinical realm. With PCa persisting as one of the most common malignancies among men globally, the quest for early and precise diagnostic methods has never been more critical. The spotlight in this endeavor shines on the molecular diagnostics that reveal the genetic underpinnings of PCa, offering insights into its onset, progression, and resistance to conventional therapies. Among the genetic aberrations, the TMPRSS2-ERG fusion and mutations in genes such as phosphatase and tensin homolog (PTEN) and myelocytomatosis viral oncogene homolog (MYC) are identified as significant players in the disease’s pathology, providing not only diagnostic markers but also potential therapeutic targets. This review underscores a multimodal diagnostic approach, merging molecular diagnostics with cytopathology, as a cornerstone in managing PCa effectively. This strategy promises a future where treatment is not only tailored to the individual’s genetic makeup but also anticipates the disease’s trajectory, offering hope for improved prognosis and quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/040ce4884f4e6bdcb8041b33d1c1868319f07888" target='_blank'>
              From microscopes to molecules: The evolution of prostate cancer diagnostics
              </a>
            </td>
          <td>
            Junyue Tao, Xiaokang Bian, Jun Zhou, Meng Zhang
          </td>
          <td>2024-08-29</td>
          <td>CytoJournal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Amino-terminal (Nt-) acetylation (NTA) is a common protein modification, affecting 80% of cytosolic proteins in humans. The human essential gene, NAA10, encodes the enzyme NAA10, as the catalytic subunit for the N-terminal acetyltransferase A (NatA) complex, including the accessory protein, NAA15. The first human disease directly involving NAA10 was discovered in 2011, and it was named Ogden syndrome (OS), after the location of the first affected family residing in Ogden, Utah, USA. Since that time, other variants have been found in NAA10 and NAA15. Here we describe the generation of 31 iPSC lines, with 16 from females and 15 from males. This cohort includes CRISPR-mediated correction to the wild-type genotype in 4 male lines, along with editing one female line to generate homozygous wild-type or mutant clones. Following the monoclonalizaiton and screening for X-chromosome activation status in female lines, 3 additional pairs of female lines, in which either the wild type allele is on the active X chromosome (Xa) or the pathogenic variant allele is on Xa, have been generated. Subsets of this cohort have been successfully used to make cardiomyocytes and neural progenitor cells (NPCs). These cell lines are made available to the community via the NYSCF Repository.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1adf8656f9aef233d0cf25ef9cb89566ccecd" target='_blank'>
              A repository of Ogden syndrome patient derived iPSC lines and isogenic pairs by X-chromosome screening and genome-editing.
              </a>
            </td>
          <td>
            Josephine Wesely, Tom Rusielewicz, Yu-Ren Chen, Brigham Hartley, Dayna McKenzie, Matthew K. Yim, Colin Maguire, Ryan Bia, Sarah Franklin, Rikhil Makwana, E. Marchi, Manali Nikte, Soha Patil, Maria Sapar, Dorota Moroziewicz, Lauren Bauer, Jeannie T. Lee, Frederick J. Monsma, Dan Paull, G. Lyon
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 Background: Dynamic regulation of gene expression is fundamental for cellular adaptation to exogenous stressors. PTEFb-mediated promoter proximal pause-release of Pol II is a conserved regulatory mechanism for synchronous transcriptional induction best described in response to heat shock, but this pro-survival role has not been examined in the applied context of cancer therapy. Design/Method: In order to examine the dynamics of chromatin reorganization following radiotherapy, we performed a combination of ChIP-, ATAC-, and RNA-seq in model systems of diffuse intrinsic pontine glioma (DIPG) and other pediatric high-grade gliomas (pHGG) following IR exposure. We interrogated IR-induced gene expression in the presence or absence of PTEFb blockade, including both mechanistic and functional consequences of concurrent inhibition or genetic depletion. We utilized culture models with live cell imaging to assess the therapeutic synergy of PTEFb inhibition with IR, as well as the therapeutic index of this intervention relative to normal controls. Finally, we employed orthotopic models of pHGG treated with conformal radiotherapy and CNS-penetrant PTEFb inhibitors in order to assess tolerability and anti-tumor effect in vivo. Results: Rapid genome-wide redistribution of active chromatin features and PTEFb facilitates Pol II pause-release to drive nascent transcriptional induction within hours of exposure to therapeutic ionizing radiation. Concurrent inhibition of PTEFb imparts a transcription elongation defect, abrogating canonical adaptive programs such as DNA damage repair and cell cycle regulation. This combination demonstrates a potent, synergistic therapeutic potential agnostic of glioma subtype, leading to a marked induction of tumor cell apoptosis and prolongation of xenograft survival.Conclusion: These studies reveal a central role for PTEFb underpinning the early adaptive response to radiotherapy, opening new avenues for combinatorial treatment in these lethal malignancies.
 Citation Format: Faye M. Walker, Lays Martin Sobral, Etienne Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela Pierce, Sana Karam, Natalie J. Serkova, Nicholas K. Foreman, Sujatha Venkataraman, Robin Dowell, Rajeev Vibhakar, Nathan A. Dahl. Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cbc389c63c005639d0ccef6f75bb33d60a2a03" target='_blank'>
              Abstract B078: Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy
              </a>
            </td>
          <td>
            Faye Walker, L. M. Sobral, Etienne P. Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Sana D Karam, Natalie J Serkova, Nicholas K. Foreman, S. Venkataraman, Robin Dowell, Rajeev Vibhakar, N. Dahl
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Saccharomyces cerevisiae is an excellent model to study the effect of external cues on cell division and stress response. 5-Fluorocuracil (5-FU) has been used to treat solid tumors since several decades. The drug was initially designed to interfere with DNA replication but was later found to exert its antiproliferative effect also via RNA-dependent processes. Since 5-FU inhibits the activity of the 3'-5'-exoribonuclease Rrp6 in yeast and mammals, earlier work has compared the effect of 5-FU treatment and RRP6 deletion at the transcriptome level in diploid synchronized yeast cells. To facilitate interpreting the expression data we have developed an improved 5-Fluorouracil RNA (5-FUR) expression viewer. Users can access information via genome coordinates and systematic or standard names for mRNAs and Xrn1-dependent-, stable-, cryptic-, and meiotic unannotated transcripts (XUTs, SUTs, CUTs, and MUTs). Normalized log2-transformed or linear data can be displayed as filled diagrams, line graphs or color-coded heatmaps. The expression data are useful for researchers interested in processes such as cell cycle regulation, mitotic repression of meiotic genes, the effect of 5-FU treatment and Rrp6 deficiency on the transcriptome and expression profiles of sense/antisense loci that encode overlapping transcripts. The viewer is accessible at http://5fur.genouest.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad442f4d43c5032d0d97fedb33e587d2cd06478" target='_blank'>
              The 5-Fluorouracil RNA Expression Viewer (5-FUR) Facilitates Interpreting the Effects of Drug Treatment and RRP6 Deletion on the Transcriptional Landscape in Yeast.
              </a>
            </td>
          <td>
            Ugo Szachnowski, Oliver Sallou, Matéo Boudet, Anthony Bretaudeau, M. Wéry, Antonin Morillon, Michael Primig
          </td>
          <td>2024-09-30</td>
          <td>Yeast</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e37b1314ac250746d64ceb8f5f7327f203eb9bb1" target='_blank'>
              Whole genome and reverse protein phase array landscapes of patient derived osteosarcoma xenograft models
              </a>
            </td>
          <td>
            Chia-Chin Wu, Licai Huang, Zhong-Fa Zhang, Z. Ju, Xingzhi Song, E. A. Kolb, Wendong Zhang, Jonathan D Gill, Min Ha, Malcolm A. Smith, Peter Houghton, Christopher L Morton, R. Kurmasheva, John Maris, Yael Mosse, Yiling Lu, R. Gorlick, P. Futreal, Hannah C. Beird
          </td>
          <td>2024-08-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Direct lineage reprogramming into dopaminergic (DA) neurons holds great promise for the more effective production of DA neurons, offering potential therapeutic benefits for conditions such as Parkinson's disease. However, the reprogramming pathway for fully reprogrammed DA neurons remains largely unclear, resulting in immature and dead-end states with low efficiency. In this study, using single-cell RNA sequencing, the trajectory of reprogramming DA neurons at multiple time points, identifying a continuous pathway for their reprogramming is analyzed. It is identified that intermediate cell populations are crucial for resetting host cell fate during early DA neuronal reprogramming. Further, longitudinal dissection uncovered two distinct trajectories: one leading to successful reprogramming and the other to a dead end. Notably, Arid4b, a histone modifier, as a crucial regulator at this branch point, essential for the successful trajectory and acquisition of mature dopaminergic neuronal identity is identified. Consistently, overexpressing Arid4b in the DA neuronal reprogramming process increases the yield of iDA neurons and effectively reverses the disease phenotypes observed in the PD mouse brain. Thus, gaining insights into the cellular trajectory holds significant importance for devising regenerative medicine strategies, particularly in the context of addressing neurodegenerative disorders like Parkinson's disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bfa61a76443945c179bc5c94c72de94ea5c5df" target='_blank'>
              Epigenetic Dynamics in Reprogramming to Dopaminergic Neurons for Parkinson's Disease.
              </a>
            </td>
          <td>
            Byounggook Cho, Junyeop Kim, Sumin Kim, Saemin An, Yerim Hwang, Yunkyung Kim, Daeyeol Kwon, Jongpil Kim
          </td>
          <td>2024-09-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genomic structural variants (SVs) play a pivotal role in driving the evolution of hematologic malignancies, particularly in leukemia, in which genetic abnormalities are crucial features. Detecting SVs is essential for achieving precise diagnosis and prognosis in these cases. Karyotyping, often complemented by fluorescence in situ hybridization and/or chromosomal microarray analysis, provides standard diagnostic outcomes for various types of SVs in front-line testing for leukemia. Recently, optical genome mapping (OGM) has emerged as a promising technique due to its ability to detect all SVs identified by other cytogenetic methods within one single assay. Furthermore, OGM has revealed additional clinically significant SVs in various clinical laboratories, underscoring its considerable potential for enhancing front-line testing in cases of leukemia. This review aims to elucidate the principles of conventional cytogenetic techniques and OGM, with a focus on the technical performance of OGM and its applications in diagnosing and prognosticating myelodysplastic syndromes, acute myeloid leukemia, acute lymphoblastic leukemia, and chronic lymphocytic leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7daafa70669857dc79dda86074368e05463db4df" target='_blank'>
              Genomic structural variants analysis in leukemia by a novel cytogenetic technique: Optical genome mapping.
              </a>
            </td>
          <td>
            Song Lu, Kefu Liu, Di Wang, Yuan Ye, Zhiping Jiang, Yunhua Gao
          </td>
          <td>2024-08-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ceac36006253317f77204fdfb7e8de57c83f85" target='_blank'>
              Population variability in X-chromosome inactivation across 10 mammalian species
              </a>
            </td>
          <td>
            Jonathan M. Werner, John Hover, Jesse Gillis
          </td>
          <td>2024-10-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal G. Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew D Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K. Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Non-coding regulatory sequences play essential roles in adjusting gene output to cellular needs and are thus critical to animal development and health. Numerous such sequences have been identified in mammalian genomes ranging from transcription factors binding motifs to recognition sites for RNA-binding proteins and non-coding RNAs. The advent of CRISPR has raised the possibility of assigning functionality to individual endogenous regulatory sites by facilitating the generation of isogenic cell lines that differ by a defined set of genetic modifications. Here we investigate the usefulness of this approach to assign function to individual miRNA binding sites. We find that the process of generating isogenic pairs of mammalian cell lines with CRISPR-mediated mutations introduces extensive molecular and phenotypic variability between biological replicates making any attempt of assigning function to the binding site essentially impossible. Our work highlights an important consideration when employing CRISPR editing to characterize non-coding regulatory sequences in cell lines and calls for the development and adoption of alternative strategies to address this question in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1d3a055f759b974342766678fe87bc5f92b153" target='_blank'>
              CRISPR-based dissection of miRNA binding sites using isogenic cell lines is hampered by pervasive noise
              </a>
            </td>
          <td>
            Mahendra Prajapat, Joana A. Vidigal
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e206396092ebd32f9a052e89e6643d141dcd6ce" target='_blank'>
              High-resolution and programmable RNA-IN and RNA-OUT genetic circuit in living mammalian cells
              </a>
            </td>
          <td>
            Min Zhang, Xue Zhang, Yongyue Xu, Yanhui Xiang, Bo Zhang, Zhen Xie, Qiong Wu, Chunbo Lou
          </td>
          <td>2024-10-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract SF3B1 is the most recurrently mutated RNA splicing gene in cancer. However, research of its pathogenic role has been hindered by a lack of disease-relevant cell line models. Here, our study compared four genome engineering platforms to establish SF3B1 mutant cell lines: CRISPR-Cas9 editing, AAV homology-directed repair editing, base editing (ABEmax, ABE8e), and prime editing (PE2, PE3, PE5max). We showed that prime editing via PE5max achieved the most efficient SF3B1 K700E editing across a wide range of cell lines. Our approach was further refined by coupling prime editing with a fluorescent reporter that leverages a SF3B1 mutation-responsive synthetic intron to mark successfully edited cells. By applying this approach, called prime editing coupled intron-assisted selection (PRECIS), we introduced the K700E hotspot mutation into two chronic lymphocytic leukemia cell lines, HG-3 and MEC-1. We demonstrated that our PRECIS-engineered cells faithfully recapitulate known mutant SF3B1 phenotypes, including altered splicing, copy number variations, and cell-growth defect. Moreover, we discovered that the SF3B1 mutation can cause the loss of Y chromosome in chronic lymphocytic leukemia. Our results showcase that PRECIS is an efficient and generalizable method for engineering genetically faithful SF3B1 mutant models. Our approach provides new insights on the role of SF3B1 mutation in cancer and enables the generation of SF3B1 mutant cell lines in relevant cellular context. Significance: This study developed an approach that can reliably and efficiently engineer SF3B1 mutation into different cellular contexts, thereby revealing novel roles of SF3B1 mutation in driving aberrant splicing, clonal evolution, and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0b3ac1820b19171ca9e6d9370b57efca6b03" target='_blank'>
              Engineering Oncogenic Hotspot Mutations on SF3B1 via CRISPR-Directed PRECIS Mutagenesis
              </a>
            </td>
          <td>
            Mikel Fernandez, Lei Yu, Qiong Jia, Xuesong Wang, Kevyn L Hart, Zhenyu Jia, Ren-Jang Lin, Lili Wang
          </td>
          <td>2024-08-28</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Patients with breast cancer, especially triple-negative breast cancer, have a poor prognosis. There is still no effective treatment for this disease. Due to resistance to traditional treatments such as chemotherapy and radiation therapy, there is a need to discover novel treatment strategies to treat this disease. Ribociclib is a selective CDK4/6 inhibitor. Approximately 20% of patients with HR+ breast cancer developed primary resistance to CDK4/6 inhibitors, and more than 30% experienced secondary resistance. Since most patients experience resistance during CDK4/6 inhibitor treatment, managing this disease is becoming more challenging. Many malignant tumors abnormally express microRNA (miR)-141, which participates in several cellular processes, including drug resistance, proliferation, epithelial–mesenchymal transition, migration, and invasion. Materials and methods In the present study, we cultured MDA-MB-231 and MCF-7 cells in DMEM-F12 medium. By performing MTT assay we determined the cytotoxic effects of ribociclib on breast cancer cells, as well as determining the IC50 of it. Then, we treated the cells with ribociclib at two time points: 24 h and 72 h. After that, RNA was isolated and reverse transcribed to cDNA. Finally, we performed qRT‒PCR to evaluate how ribociclib affects the expression level of desired genes. Results and conclusion We found that ribociclib can inhibit cell growth in a dose- and time-dependent manner. We examined the mRNA expression of 4 genes. After ribociclib treatment, the mRNA expression of CDK6 and MYH10 decreased (p < 0.01, p < 0.05). The mRNA expression of CDON increased (p<0.05), but no significant changes were observed in ZEB1 mRNA expression. Furthermore, the qRT‒PCR results for miR-141 showed that the expression of miR-141 increased (p<0.01) after 72 h of treatment with ribociclib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af2742d79589071ef6d86e863e12d574976bef5" target='_blank'>
              The effect of ribociclib on the expression levels of miR-141 and CDK4/6-USP51 signaling pathway genes in MCF-7 and MDA-MB-231 cells
              </a>
            </td>
          <td>
            Shayeste Sadat Baghermanesh, Mahmoud Barati, Arshad Hosseini
          </td>
          <td>2024-08-28</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epigenetic mechanisms often fuel the quick evolution of cancer cells from normal cells. Mutations or aberrant expressions in the enzymes of DNA methylation, histone post-translational modifications, and chromatin remodellers have been extensively investigated in cancer pathogenesis; however, cancer-associated histone mutants have gained momentum in recent decades. Next-generation sequencing of cancer cells has identified somatic recurrent mutations in all the histones (H3, H4, H2A, H2B, and H1) with different frequencies for various tumour types. Importantly, the well-characterised H3K27M, H3G34R/V, and H3K36M mutations are termed as oncohistone mutants because of their wide roles, from defects in cellular differentiation, transcriptional dysregulation, and perturbed epigenomic profiles to genomic instabilities. Mechanistically, these histone mutants impart their effects on histone modifications and/or on irregular distributions of chromatin complexes. Recent studies have identified the crucial roles of the H3K27M and H3G34R/V mutants in the DNA damage response pathway, but their impacts on chemotherapy and tumour progression remain elusive. In this review, we summarise the recent developments in their functions toward genomic instabilities and tumour progression. Finally, we discuss how such a mechanistic understanding can be harnessed toward the potential treatment of tumours harbouring the H3K27M, H3G34R/V, and H3K36M mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d93aa8724101a0a6b04c672de1eafdda67a5e3" target='_blank'>
              Emerging roles of cancer-associated histone mutations in genomic instabilities
              </a>
            </td>
          <td>
            Priyanka Yadav, Ronit Jain, Rajesh Kumar Yadav
          </td>
          <td>2024-10-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell type identity is determined by gene regulatory networks (GRNs), comprising the expression of specific transcription factors (TFs) regulating target genes (TGs) via binding to open chromatin regions (OCRs). The regulatory logic of differentiation includes factors specific to one or multiple cell types, functioning in a combinatorial fashion. Classic approaches of GRN discovery used perturbational data to elucidate TF-TG links, but are laborious and not scalable across the tree of life. Single cell transcriptomics has emerged as a revolutionary approach to study gene expression with cell type resolution, but incorporating perturbational data is challenging. Planarians, with their pluripotent neoblast stem cells continuously giving rise to all cell types, offer an ideal model to attempt this integration. Despite extensive single cell transcriptomic studies, the transcriptional and chromatin regulation at the cell type level remains unexplored. Here, we investigate the regulatory logic of planarian stem cell differentiation by obtaining an organism-level integration of single cell transcriptomics and single cell accessibility data. We identify specific open chromatin profiles for major differentiated cell types and analyse their transcriptomic landscape, revealing distinct gene modules expressed in individual types and combinations of them. Integrated analysis unveils gene networks reflecting known TF interactions in each type and identifies TFs potentially driving differentiation across multiple cell types. To validate our predictions, we combined TF knockdown RNAi experiments with single cell transcriptomics. We focus on hnf4, a TF known to be expressed in gut phagocytes, and confirm its influence on other types, including parenchymal cells. Our results demonstrate high overlap between predicted targets and experimentally-validated differentially-regulated genes. Overall, our study integrates TFs, TGs and OCRs to reveal the regulatory logic of planarian stem cell differentiation, showcasing that the combination of single cell methods and perturbational studies will be key for characterising GRNs widely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d812fbadba7ebc9a3898e7e95eae459989657669" target='_blank'>
              The Regulatory Logic of Planarian Stem Cell Differentiation
              </a>
            </td>
          <td>
            Alberto Pérez-Posada, Helena García-Castro, Elena Emili, Virginia Vanni, Cirenia Arias-Baldrich, S. Frölich, S. V. van Heeringen, N. Kenny, Jordi Solana
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2352f41e08e0411c3c2eeaf443ff948521637f4" target='_blank'>
              Deletion of exons 45 to 55 in the DMD gene: from the therapeutic perspective to the in vitro model
              </a>
            </td>
          <td>
            Javier Poyatos-García, Patricia Soblechero-Martín, A. Liquori, A. López-Martínez, Pilar Maestre, Elisa González-Romero, Rafael P. Vázquez-Manrique, Nuria Muelas, Gema García-García, Jessica Ohana, Virginia Arechavala-Gomeza, J. J. Vilchez
          </td>
          <td>2024-10-01</td>
          <td>Skeletal Muscle</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Disruptions in normal development and the emergence of health conditions often result from the malfunction of vital genes in the human body. Decades of scientific research have focused on techniques to modify or substitute defective genes with healthy alternatives, marking a new era in disease treatment, prevention and cure. Recent strides in science and technology have reshaped our understanding of disorders, medication development and treatment recommendations, with human gene and cell therapy at the forefront of this transformative shift. Its primary objective is the modification of genes or adjustment of cell behaviour for therapeutic purposes. In this review, we focus on the latest advances in gene and cell therapy for treating human genetic diseases, with a particular emphasis on FDA and EMA‐approved therapies and the evolving landscape of genome editing. We examine the current state of innovative gene editing technologies, particularly the CRISPR‐Cas systems. As we explore the progress, ethical considerations and prospects of these innovations, we gain insight into their potential to revolutionize the treatment of genetic diseases, along with a discussion of the challenges associated with their regulatory pathways. This review traces the origins and evolution of these therapies, from conceptual ideas to practical clinical applications, marking a significant milestone in the field of medical science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/605617113766e025c90c18b7b7452f93c13a0842" target='_blank'>
              Gene and cell therapy of human genetic diseases: Recent advances and future directions
              </a>
            </td>
          <td>
            Busra Cetin, Fulya Erendor, Y. E. Eksi, A. Şanlioğlu, S. Sanlioglu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0943aff1f710e095be6a9439cd65322a1fc6aa0e" target='_blank'>
              Genomic Balancing Act: deciphering DNA rearrangements in the complex chromosomal aberration involving 5p15.2, 2q31.1, and 18q21.32.
              </a>
            </td>
          <td>
            Zain Dardas, Dana Marafi, Ruizhi Duan, Jawid M Fatih, O. El-Rashidy, Christopher M. Grochowski, Claudia M B Carvalho, S. Jhangiani, W. Bi, Haowei Du, Richard A Gibbs, J. Posey, D. Calame, Maha S. Zaki, James R. Lupski
          </td>
          <td>2024-09-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2b8937040c82aaaf4474d35899e8a2ae8efb870" target='_blank'>
              Genomic, epigenomic and transcriptomic landscape of glioblastoma.
              </a>
            </td>
          <td>
            T. C. Dakal, Ganesh S. Kakde, Pawan Kumar Maurya
          </td>
          <td>2024-08-24</td>
          <td>Metabolic brain disease</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Chronic Lymphocytic Leukemia (CLL) is the most frequent type of leukemia in Western countries. In recent years, there have been important advances in the knowledge of molecular alterations that underlie the disease’s pathogenesis. Very heterogeneous prognostic subgroups have been identified by the mutational status of immunoglobulin heavy variable genes (IGVH), FISH analysis and molecular evaluation of TP53 mutations. Next-generation sequencing (NGS) technologies have provided a deeper characterization of the genomic and epigenomic landscape of CLL. New therapeutic targets have led to a progressive reduction of traditional chemoimmunotherapy in favor of specific biological agents. Furthermore, in the latest clinical trials, the minimal residual disease (MRD) has emerged as a potent marker of outcome and a guide to treatment duration. This review focuses on recent insights into the understanding of CLL biology. We also consider the translation of these findings into the development of risk-adapted and targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3c13c68ecb506ccc3f3b3f0d7d44f962adc11fb" target='_blank'>
              Recent Advances in the Molecular Biology of Chronic Lymphocytic Leukemia: How to Define Prognosis and Guide Treatment
              </a>
            </td>
          <td>
            A. Arcari, Lucia Morello, Elena Borotti, Elena Ronda, Angela Rossi, Daniele Vallisa
          </td>
          <td>2024-10-14</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9291d16699439a2d6b53e165a3605cd9a24fb" target='_blank'>
              Unraveling the spatial organization and development of human thymocytes through integration of spatial transcriptomics and single-cell multi-omics profiling
              </a>
            </td>
          <td>
            Yanchuan Li, Huamei Li, Cheng Peng, Ge Meng, Yijun Lu, Honglin Liu, Li Cui, Huan Zhou, Zhu Xu, Lingyun Sun, Lihong Liu, Qing Xiong, Beicheng Sun, Shiping Jiao
          </td>
          <td>2024-09-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="All subtypes of rhabdomyosarcoma (RMS) show skeletal muscle differentiation but each has different oncogenic mechanisms including recurrent PAX3 and PAX7 gene fusions (alveolar), a hotspot mutation in MYOD1 (spindle/sclerosing) and mutations in other oncogenes and tumor suppressors (embryonal, pleomorphic). This range of genomic findings typically requires several different DNA and RNA-based diagnostic assays. Here, we utilize a clinically validated, targeted RNA next-generation sequencing (RNAseq) panel to simultaneously detect MYOD1 transcript levels for lineage assignment, PAX3 and PAX7 translocations and mutations in MYOD1 and other RMS-related genes for definitive subtyping in a single assay. RNA-based detection of MYOD1 p.L122R and other mutations were orthogonally validated with digital droplet (dd)PCR or DNA-based NGS. Expanding the utility of clinical-grade RNAseq data beyond the detection of gene fusions is a cost-effective and time-efficient approach for more comprehensive screening of RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e47bc521700facb67966c59b363c93b774e0f8" target='_blank'>
              Integrated Molecular Profiling of Rhabdomyosarcoma Subtypes by Targeted RNAseq
              </a>
            </td>
          <td>
            Matthew R Avenarius, Ashley Patton, Nehad Mohamed, Pan, Xiaokang, Dan Jones
          </td>
          <td>2024-10-14</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts.
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            Lauren M. Brown, G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E. Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Children born with Down syndrome (DS), a genetic condition caused by the presence of an additional chromosome 21 (trisomy 21 or T21), have a substantially higher risk of developing childhood leukaemia. Approximately 30% of DS infants develop a preleukemic condition called transient abnormal myelopoiesis (TAM). TAM is strictly associated with GATA1 truncating mutations on a T21 background. Most TAM cases spontaneously resolve within the first few months of life. However, by the age of five, approximately 10% of cases progress to megakaryoblastic/erythroid leukaemia, known as myeloid leukaemia of DS (ML-DS). This final step of transformation is often driven by specific third-hit somatic mutations, most of which affects genes encoding JAK kinases, cohesin complexes, or epigenetic regulators. These well-defined stepwise genetic aberrations associated with ML-DS pathogenesis provide a unique and highly tractable system to elucidate the molecular changes underlying leukaemogenesis. Using single-cell mRNA sequencing of primary human samples, we directly interrogate the transcriptional changes underpinning the multi-step pathogenesis of ML-DS. We observe an expansion of megakaryocyte-erythrocyte progenitors (MEPs) specifically in T21 foetal liver compared to diploid or other trisomies. However, it is the GATA1 mutation that accounts for the majority of the transcriptomic changes towards that of the leukaemic blasts. Furthermore, investigation of an aggressive TAM case reveals an enrichment in expression of the leukaemogenic gene module associated with the full-blown leukaemia state ML-DS. We also demonstrate that TAM and ML-DS blasts still retain transcriptional signatures of normal haematopoiesis, showing heterogeneous features of differentiation towards erythrocytes, megakaryocytes, and mast cells. Finally, by integrating genomic and transcriptomic data, we directly defined the genomic evolution, and its corresponding transcriptional consequences, that underpins a treatment-refractory case of ML-DS. Overall, we have generated the first comprehensive single-cell mRNA atlas of TAM / ML-DS, enabling unbiased quantitative characterisation of the transcriptional transformation along ML-DS pathogenesis. Our study explores the molecular signature associated with progression of TAM to ML-DS, as well as the mechanism underlying differences in treatment response. These insights offer the possibility of more meaningful clinical interpretations and potential to explore novel therapeutic targets for these conditions.
 Citation Format: Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, Angus Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, Jan-Henning Klusmann, Jack Bartram, Sam Behjati. Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f4dde5c53348eeefdb9808ec633733f445cffc" target='_blank'>
              Abstract A018 Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome
              </a>
            </td>
          <td>
            Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, A. Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f0f651c9fac9c8c40b22fc5b21b4578646df27" target='_blank'>
              Multi-omics features of immunogenic cell death in gastric cancer identified by combining single-cell sequencing analysis and machine learning
              </a>
            </td>
          <td>
            Shu-Long Dai, Jian-Qiang Pan, Zhen-Rong Su
          </td>
          <td>2024-09-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetic silencing is essential for regulating gene expression and cellular diversity in eukaryotes. While transposable elements (TEs) are mostly silenced with DNA and H3K9 methylation, gene silencing is mediated by H3K27me3, an epigenetic mark deposited by the Polycomb repressive complex 2 (PRC2). Despite the major role epigenetic silencing plays in the development of multicellular eukaryotes, little is known about how epigenetically-controlled regulatory networks were shaped over evolutionary time. Here, we analyse epigenomes from a diverse group of species across the green lineage and infer the chronological epigenetic recruitment of genes that occurred during land plant evolution. We first reveal the nature of plant heterochromatin in the unicellular green microalga Chlorella sorokiniana and identify a substantial number of genes marked with H3K27me3, highlighting the deep origin of PRC2-regulated genes in the green lineage. By incorporating genomic phylostratigraphy, we show how genes of differing evolutionary age are partitioned into distinct epigenetic states in plants, with evolutionarily young genes incorporated into developmental programs controlled by H3K9 methylation in Arabidopsis. We further reveal a major wave of PRC2 recruitment to genes that emerged during land terrestrialisation and flowering plant evolution, and identify an ancestral PRC2 network with a shared functional topology in green algae through to land plants, providing a glimpse of the earliest types of genes regulated by PRC2 during the course of plant evolution. Finally, we analyse the potential regulation of these ancestral PRC2 target genes and find a strong enrichment of motifs bound by ancient AP2/ERF transcription factors (TFs) known to interact with PRC2, which we hypothesise were key determinants in shaping some of the first gene regulatory networks controlled by PRC2 in plants. Our data thus reveal pivotal epigenetic adaptations that occurred during a significant period in the evolutionary history of plants, which likely contributed to key regulatory innovations that influenced major morphological and developmental change into the modern-day. More broadly, our findings offer insight into the evolutionary dynamics and molecular triggers that drive the adaptation and elaboration of epigenetic regulation, laying the groundwork for its future consideration in other major eukaryotic lineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/578bce57cf922c3e4cbf1c7dcaf1f5ced20825df" target='_blank'>
              The expansion and diversification of epigenetic regulatory networks underpins major transitions in the evolution of land plants
              </a>
            </td>
          <td>
            Romy Petroll, Ranjith K. Papareddy, Rafal Krela, Alice Laigle, Quentin Rivière, Kateřina Bišová, I. Mozgová, Michael Borg
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="BACKGROUND/AIM
The emergence of novel DNA damage repair (DDR) pathways in molecular-target therapy drugs (MTTD) has shown promising outcomes in treating patients with metastatic castration-resistant prostate cancer (mCRPC). About 25% of mCRPC patients have actionable deleterious aberrations in DDR genes, primarily in the homologous recombination (HR) pathway. However, the response rate in patients with BRCA1/2 or mutations in HRR-related genes is only 45%-55%, when exposed to poly ADP ribose polymerase (PARP) inhibitor-based therapy (PARPi). A frequent characteristic feature of prostate cancer (PC) is the occurrence of genomic rearrangement that affects the transmembrane protease serine 2 (TMPRSS2) and E26 transformation-specific (ETS)- transcription factor-related gene (ERG).


MATERIALS AND METHODS
In this study, a total of 114 patients with mCRPC had their RNA and DNA sequenced using next-generation sequencing.


RESULTS
Based on their genetic profile of deleterious gene alterations of BRCA1/2 or ATM, six patients were selected for PARPi. Patients with TMPRSS2:ERG gene fusion and homozygous alteration in ATM or BRCA2 (n=2) or heterozygous alterations (BRCA1 or BRCA2) and lack of TMPRSS2:ERG gene fusion (n=2) did not show clinical benefit from PARPi (treatment duration <16 weeks). In contrast, patients (n=2) without TMPRSS2:ERG gene fusion and homozygous deleterious alterations in ATM or BRCA2 all had clinical benefit from PARPi (treatment duration ≥16 weeks).


CONCLUSION
The TMPRSS2:ERG transcript product might be used as a PARPi resistance biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb09c69e7cd8aed30d5976f567f7199cd33c56e" target='_blank'>
              TMPRSS2:ERG Gene Fusion Might Predict Resistance to PARP Inhibitors in Metastatic Castration-resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Tim S Poulsen, Anders Nygaard Lørup, Per Kongsted, R. Eefsen, M. Højgaard, Estrid Høgdall
          </td>
          <td>2024-09-30</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2e1918f5013376fd130c9f594868a7f384598c2" target='_blank'>
              HMOX1: A pivotal regulator of prognosis and immune dynamics in ovarian cancer
              </a>
            </td>
          <td>
            Jinfa Huang, Ruiwan Tan
          </td>
          <td>2024-08-30</td>
          <td>BMC Women's Health</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Synovial Sarcomas (SS) are characterized by the presence of the SS18::SSX fusion gene, which protein product induce chromatin changes through remodeling of the BAF complex. To elucidate the genomic events that drive phenotypic diversity in SS, we performed RNA and targeted DNA sequencing on 91 tumors from 55 patients. Our results were verified by proteomic analysis, public gene expression cohorts and single-cell RNA sequencing. Transcriptome profiling identified three distinct SS subtypes resembling the known histological subtypes: SS subtype I and was characterized by hyperproliferation, evasion of immune detection and a poor prognosis. SS subtype II and was dominated by a vascular-stromal component and had a significantly better outcome. SS Subtype III was characterized by biphasic differentiation, increased genomic complexity and immune suppression mediated by checkpoint inhibition, and poor prognosis despite good responses to neoadjuvant therapy. Chromosomal abnormalities were an independent significant risk factor for metastasis. KRT8 was identified as a key component for epithelial differentiation in biphasic tumors, potentially controlled by OVOL1 regulation. Our findings explain the histological grounds for SS classification and indicate that a significantly larger proportion of patients have high risk tumors (corresponding to SS subtype I) than previously believed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5e414a7a3b033cc20bf7f0e410a6830f2ad049" target='_blank'>
              Molecular Profiling Defines Three Subtypes of Synovial Sarcoma.
              </a>
            </td>
          <td>
            Yi Chen, Yanhong Su, Xiaofang Cao, I. Siavelis, I. Leo, Jianming Zeng, P. Tsagkozis, A. Hesla, A. Papakonstantinou, Xiao Liu, Wen-Kuan Huang, Binbin Zhao, Cecilia Haglund, M. Ehnman, Henrik Johansson, Yingbo Lin, Janne Lehtiö, Yifan Zhang, O. Larsson, Xuexin Li, Felix Haglund de Flon
          </td>
          <td>2024-09-10</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction: The role of genetic variants in response to chemotherapy has been investigated in several studies. This study aimed to investigate genetic variants associated with response to chemotherapy in breast cancer (BC) patients. Methods: Significant variants (p < 5 × 10-8) associated with response to chemotherapy were obtained from GWA studies. Candidate variants were identified by haplotype analysis (r2 ≥ 0.9, D'≥0.9) using 1000Genome LD data. To determine the effects of the variants on gene expression, expression quantitative trait loci (eQTL) were evaluated. To compare the expression of the identified genes in tumor samples, expression levels were compared between TCGA tumor types and adjacent normal tissues. Results: Six rs3820706, rs147451859, rs4784750, rs17587029, rs16830728, and rs16972207 variants were significantly associated with response to chemotherapy in BC patients (p < 5 × 10-8). Seven novel haplotypic structures were identified to be associated with adverse response to chemotherapy in BC patients. These haplotypes formed two genetic structures associated with neutropenia, leukopenia, chemotherapy-induced cytotoxicity (GAG-TTAT), and chemotherapy-induced alopecia (CC-CAACTCCCGTTGCGG). These variants are located on PPCDC, NLRC5, STAM2, and TNFSF13B genes, and the expression of these genes significantly changed in BC tissues than normal tissues (P ≤ 0.05), also showing gene-gene correlation (P ≤ 0.05). Conclusions: These genetic variants and their associated novel haplotypic structures can predict adverse response to chemotherapy in BC patients and could potentially form BC-associated genetic panel for adverse response to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f6f813884867edf6f2b79ca7de2ac3b63826948" target='_blank'>
              Novel genetic structures associated with adverse response to chemotherapy in breast cancer.
              </a>
            </td>
          <td>
            Morteza Gholami, Mohsen Asouri, A. Ahmadi, Mehrab Nasirikenari
          </td>
          <td>2024-08-28</td>
          <td>Journal of oncology pharmacy practice : official publication of the International Society of Oncology Pharmacy Practitioners</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="B-acute lymphoblastic leukaemia (B-ALL) is a haematological disease resulting from haematopoietic system dysfunction, leading to the unchecked growth of immature B lymphoblasts. The disease's complexity is underscored by the spectrum of genetic aberrations that underlie B-ALL entities, necessitating advanced genetic analyses for precise classification and risk determination. Prior to the adoption of next-generation sequencing into standard diagnostic practices, up to 30% of B-ALL cases were not assigned to specific entities due to the limitations of traditional diagnostic methods. The advent of comprehensive genomic analysis, especially whole-genome transcriptome sequencing, has significantly enhanced our understanding of B-ALL's molecular heterogeneity, paving the way for the exploration of novel, tailored treatment strategies. Furthermore, recent technological innovations, such as optical genome mapping, methylation profiling, and single-cell sequencing, have propelled forward the fields of cancer research and B-ALL management. These innovations introduce novel diagnostic approaches and prognostic markers, facilitating a deeper, more nuanced understanding of individual patient disease profiles. This review focuses on the latest diagnostic standards and assays for B-ALL, the importance of new technologies and biomarkers in enhancing diagnostic accuracy, and the expected role of innovative advancements in the future diagnosis and treatment of B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d034cce9741226007a94cd17126d35c356146f9" target='_blank'>
              Diagnosis of acute lymphoblastic leukaemia: an overview of the current genomic classification, diagnostic approaches, and future directions.
              </a>
            </td>
          <td>
            W. Walter, Ilaria Iacobucci, M. Meggendorfer
          </td>
          <td>2024-10-15</td>
          <td>Histopathology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a hereditary blood disorder characterized by the presence of abnormal hemoglobin molecules and thus distortion (sickling) of the red blood cells. SCD causes chronic pain and organ damage and shortens life expectancy. Gene therapy emerges as a potentially curative approach for people with SCD who lack a matched sibling donor for hematopoietic stem cell transplantation. Here, we review recent progress in gene therapy for SCD and focus on innovative technologies that target the genetic roots of the disease. We also review the challenges associated with gene therapy, including oncogenic risks, and the need for refined delivery methods. Despite these hurdles, the rapidly evolving landscape of gene therapy for SCD raises hope for a paradigm shift in the treatment of this debilitating disease. As research progresses, a deeper understanding of the molecular mechanisms involved and continuous improvements in gene-editing technologies promise to bring gene therapy for SCD closer to mainstream clinical application, offering a transformative, curative option for patients with this genetic disorder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6e785d7cbf41248336e372c5a5fb5eb1bf4e0c" target='_blank'>
              Treating Sickle Cell Disease: Gene Therapy Approaches.
              </a>
            </td>
          <td>
            M. Cavazzana, Alice Corsia, Mégane Brusson, A. Miccio, Michaela Semeraro
          </td>
          <td>2024-09-11</td>
          <td>Annual review of pharmacology and toxicology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background Alterations in DNA damage repair genes in advanced prostate cancer (PC) may impact responses to therapy and clinical outcomes. This study described homologous recombination repair (HRR) testing patterns and clinical outcomes among patients with metastatic castration-resistant prostate cancer (mCRPC) by HRR alteration status and race in the United States (US). Methods Clinical data in the nationwide (US-based) Flatiron Health-Foundation Medicine, Inc. (FMI) Metastatic PC Clinico-Genomic Database were evaluated (01/01/2011–12/31/2022). Patients initiating first-line (1L) mCRPC therapy on or after mCRPC diagnosis were included. Testing patterns, time-to-next treatment, overall survival (OS), and time-to-prostate specific antigen response were described. Results Of the 1367 patients with mCRPC and at least one HRR panel test prior to or on the date of 1L mCRPC therapy initiation, 332 (24.3%) were HRR positive (White patients: n = 219 [66.0%]; Black patients: n = 37 [11.1%]) and 1035 (75.7%) were HRR negative (White patients: n = 702 [67.8%]; Black patients: n = 84 [8.1%]). The mean time between first positive test and 1L mCRPC therapy initiation date was 588 days (White patients: 589 days; Black patients: 639 days). Among HRR positive relative to negative patients, trends for faster progression (respective 12-month rate overall: 71.1% and 63.7%; White patients: 72.5% and 64.0%; Black patients: 65.4% and 56.4%), shorter OS (respective 24-month rate overall: 46.8% and 51.9%; White patients: 48.6% and 46.2%; Black patients: 52.8% and 54.1%), and decreased treatment response (respective 12-month rate overall: 24.3% and 37.9%; White patients: 24.5% and 35.2%; Black patients: 17.0% and 43.9%) were observed. Conclusion Patients with mCRPC positive for HRR alterations tended to exhibit poorer treatment responses and clinical outcomes than those with a negative status. These findings highlight the importance of timely genetic testing in mCRPC, particularly among Black patients, and the need for improved 1L targeted therapies to address the unmet need in HRR positive mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b5b000d1e3f13e0b5a22c9691a361f7dc5edd2" target='_blank'>
              Homologous Recombination Repair Testing Patterns and Outcomes in mCRPC by Alteration Status and Race
              </a>
            </td>
          <td>
            M. Bilen, I. Khilfeh, C. Rossi, Erik Muser, L. Morrison, A. Hilts, Lilian Diaz, Patrick Lefebvre, D. Pilon, Daniel George
          </td>
          <td>2024-09-01</td>
          <td>ClinicoEconomics and Outcomes Research: CEOR</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="In recent years, it has been demonstrated that many of the pervasive genetic defects throughout cancerogenesis occur in genes encoding chromatin regulators (CRs). We analyzed the distribution and characteristics of well-studied CRs across tens of thousands of tumor samples. Our analysis revealed that tumors with mutations in CRs are associated with high tumor mutational burden (TMB). The co-occurrence of mutations in multiple CRs was linked with a further increase in TMB. Given that TMB may predict the clinical response to immune checkpoint inhibitor (ICI) treatment, we investigated the relationship between mutations in CRs and ICI response. We found that patients harboring mutations in CRs exhibited improved responses to ICI treatment, comparable to those with deficiencies in canonical DNA repair pathways. Overall, this study uncovered significant relationships between mutations in chromatin regulators and critical features of cancer, underscoring the need for further functional and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a483b13f9b49765baf5b4ffa7cd0de2126755c55" target='_blank'>
              Tumors with mutations in chromatin regulators are associated with higher mutational burden and improved response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            G. Kungulovski, M. Gjorgjievska, S. Mehandziska, D. Bukovec, M. Risteski, I. Kungulovski, Z. Mitrev
          </td>
          <td>2024-10-16</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Aim Despite therapeutic advancements, metastatic colorectal cancer is usually fatal, necessitating novel approaches based on the molecular pathogenesis to improve outcomes. Some colorectal cancers have no mutations in the extended RAS panel (KRAS, NRAS, BRAF) genes and represent a special subset, which deserves particular therapeutic considerations. Materials and Methods The genomic landscape of colorectal cancers from publicly available genomic series was interrogated, using the cBioportal platform. Colorectal cancer cohorts with cancers devoid of KRAS/NRAS or BRAF mutations were evaluated for the presence of mutations in the catalytic sub-unit alpha of kinase PI3K, encoded by the gene PIK3CA. Results PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations were observed in 3.7% to 7.6% of colorectal cancers in the different series examined. Patients with all four genes in wildtype configuration (quadruple wild type) represented 32.2% to 39.9% of cases in the different series examined. Compared with quadruple wild type cancers, triple (KRAS/NRAS/BRAF) wild type/PIK3CA mutated cancers had a higher prevalence of high TMB cases and additional mutations in colorectal cancer associated genes except for mutations in TP53. Mutations in genes encoding for epigenetic modifiers and the DNA damage response (DDR) were also more frequent in triple wild type/PIK3CA mutated cancers. The prognosis of the two groups was comparable. Conclusion Colorectal cancers with PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations have frequently mutations in epigenetic modifiers and DDR response genes, which may provide opportunities for targeting. These mutations are present in a smaller subset of quadruple wild type cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fec366ce24e54405084d6854f8aadb7604b8213" target='_blank'>
              PIK3CA Mutated Colorectal Cancers Without KRAS, NRAS and BRAF Mutations Possess Common and Potentially Targetable Mutations in Epigenetic Modifiers and DNA Damage Response Genes
              </a>
            </td>
          <td>
            I. Voutsadakis
          </td>
          <td>2024-08-27</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, J. Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Colorectal cancer (CRC) features molecular heterogeneity and differing immune cell infiltration patterns, affecting how patients respond to therapy. However, a comprehensive understanding of immune cell phenotypes across patient subgroups is missing. Here, we dissect the CRC tumor microenvironment by integrating 4,27 million single cells from 1670 samples and 650 patients across 48 studies (76 datasets) representing 7 billion expression values. These samples encompass the full spectrum of disease progression, from normal colon to polyps, primary tumors, and metastases, covering both early and advanced stages of CRC. We present a high-resolution view of CRC with 62 major cell types or states, revealing different cell-type composition patterns in CRC subtypes. The atlas allows for refined tumor classification and patient stratification into four immune phenotypes: immune desert, myeloid, B cell, and T cell subtypes. Using bulk samples with genomic and clinical information, we identify cellular components associated with tumor histology and genotypes. These findings could have important implications for developing improved cancer immunotherapy approaches in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b83f214ca7a0f8d484f556a59979b00abd51f6" target='_blank'>
              Dissecting heterogeneity of tumor microenvironment in colorectal cancer using high-resolution single-cell atlas
              </a>
            </td>
          <td>
            Valentin Marteau, Niloofar Nemati, Kristina Handler, Deeksha Raju, Erika Kvalem Soto, G. Fotakis, Sandro Carollo, Nina Böck, A. Kirchmair, A. Scheiber, Arno Ammann, Andreas Seeber, Elisabeth Gasser, Lena Horvath, Hassan Fazilaty, Tomas Valenta, Gregor Sturm, D. Rieder, S. Sopper, A. Pircher, P. Ziegler, Markus Tschurtschenthaler, Dominik Wolf, Isabelle Arnold, S. Salcher, Z. Trajanoski
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Chromatin is not randomly distributed within the nucleus, but organized in a three-dimensional structure that plays a critical role in genome functions. Сohesin and condensins are conserved multi-subunit protein complexes that participate in mammalian genome organization by extruding chromatin loops. The fine temporal regulation of these complexes is facilitated by a number of other proteins, one of which is microcephalin (Mcph1). Mcph1 prevents condensin II from associating with chromatin through interphase. Loss of Mcph1 induces chromosome hypercondensation; it is not clear to what extent this reorganization affects gene expression. In this study, we generated several mouse embryonic stem cell (mESC) lines with knockout of the Mcph1 gene and analyzed their gene expression profile. Gene Ontology analyses of differentially expressed genes (DEGs) after Mcph1 knockout revealed gene categories related to general metabolism and olfactory receptor function but not to cell cycle control previously described for Mcph1. We did not find a correlation between the DEGs and their frequency of lamina association. Thus, this evidence questions the hypothesis that Mcph1 knockout-mediated chromatin reorganization governs gene expression in mESCs. Among the negative effects of Mcph1 knockout, we observed numerous chromosomal aberrations, including micronucleus formation and chromosome fusion. This confirms the role of Mcph1 in maintaining genome integrity described previously. In our opinion, dysfunction of Mcph1 may be a kind of “Rosetta stone” for deciphering the function of condensin II in the interphase nucleus. Thus, the cell lines with knocked-out Mcph1 can be used to further study the influence of chromatin structural proteins on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d65d4d4365d60abd6b73f946f30c7616d79df9" target='_blank'>
              Generation and analysis of mouse embryonic stem cells with knockout of the Mcph1 (microcephalin) gene
              </a>
            </td>
          <td>
            A. Yunusova, A. V. Smirnov, T. Shnaider, I. Pristyazhnuk, S.Y. Korableva, N. R. Battulin
          </td>
          <td>2024-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d791af736d485e488b3ee8b58f9beef46966154" target='_blank'>
              Crossing epigenetic frontiers: the intersection of novel histone modifications and diseases
              </a>
            </td>
          <td>
            Weiyi Yao, Xinting Hu, Xin Wang
          </td>
          <td>2024-09-16</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f24a956f031769841584e7d92965e89a42014c" target='_blank'>
              p53motifDB: integration of genomic information and tumor suppressor p53 binding motifs
              </a>
            </td>
          <td>
            Gabriele Baniulyte, S. M. Hicks, Morgan A. Sammons
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4aa3f8da17c5a1e64257f76fd2c37200840abc1" target='_blank'>
              Identification of SLC22A17 DNA methylation hotspot as a potential biomarker in cutaneous melanoma
              </a>
            </td>
          <td>
            Alessandro Lavoro, L. Falzone, Giuseppe Gattuso, Giuseppe N Conti, Rosario Caltabiano, G. Madonna, M. Capone, J. McCubrey, P. Ascierto, Massimo Libra, S. Candido
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="
 Rhabdoid tumors (RTs) are rare, highly aggressive pediatric malignancies which occur in kidney, soft-parts and brain. They are characterized by a complete inactivation of the SMARCB1 tumor suppressor gene encoding a core subunit of the chromatin remodeling SWI/SNF complex. Prognosis for children with RTs is poor as, in many instances, these tumors are resistant to conventional type chemotherapy. Pharmacological inhibition of EZH2 is a promising strategy to treat some tumors with loss of function in SMARCB1. However, the clinical response rate remains low, and both primary and secondary resistance have been reported. Understanding the mechanisms of resistance to EZH2 inhibition may allow new therapeutic hypothesis. For this purpose, we realized a genome-wide CRISPR-Cas9 knockout screening on two rhabdoid cell lines treated with EZH2 inhibitor. We identified TP53 loss as the sole consistent gene knock-out able to confer some resistance to EZH2 inhibition in both cell lines. Conversely, we demonstrated that MDM2/MDM4 inhibition, in a TP53-dependant manner, strongly synergized with EZH2 inhibition to control cell lines viability in vitro. We further treated 4 rhabdoid patient-derived xenografts with UNC1999, Idasanutlin and the combination of both and observed a potent tumor growth control upon combined EZH2 and MDM2/MDM4 inhibitions. To conclude, our results strongly encourage to assess the actual efficacy of combined MDM2/MDM4 and EZH2 inhibitors in the treatment of patients with SMARCB1-deficient rhabdoid tumors.
 Citation Format: Céline Chauvin, Tiphaine Hery, Rachida Bouarich, Fariba Nemati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, Sergio Roman-Roman, Didier Decaudin, Raphaël Margueron, Franck Bourdeaut. Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A051.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85bf34da504cd580597c4be8039c1663b824b215" target='_blank'>
              Abstract A051 Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors
              </a>
            </td>
          <td>
            C. Chauvin, Tiphaine Héry, Rachida Bouarich, F. Némati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, S. Roman-Roman, Didier Decaudin, Raphaël Margueron, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a prevalent and heterogeneous tumor with limited treatment options and unfavorable prognosis. The crucial role of a disintegrin and metalloprotease (ADAM) gene family in the tumor microenvironment of HCC remains unclear. Methods This study employed a novel multi-omics integration strategy to investigate the potential roles of ADAM family signals in HCC. A series of single-cell and spatial omics algorithms were utilized to uncover the molecular characteristics of ADAM family genes within HCC. The GSVA package was utilized to compute the scores for ADAM family signals, subsequently stratified into three categories: high, medium, and low ADAM signal levels through unsupervised clustering. Furthermore, we developed and rigorously validated an innovative and robust clinical prognosis assessment model by employing 99 mainstream machine learning algorithms in conjunction with co-expression feature spectra of ADAM family genes. To validate our findings, we conducted PCR and IHC experiments to confirm differential expression patterns within the ADAM family genes. Results Gene signals from the ADAM family were notably abundant in endothelial cells, liver cells, and monocyte macrophages. Single-cell sequencing and spatial transcriptomics analyses have both revealed the molecular heterogeneity of the ADAM gene family, further emphasizing its significant impact on the development and progression of HCC. In HCC tissues, the expression levels of ADAM9, ADAM10, ADAM15, and ADAM17 were markedly elevated. Elevated ADAM family signal scores were linked to adverse clinical outcomes and disruptions in the immune microenvironment and metabolic reprogramming. An ADAM prognosis signal, developed through the utilization of 99 machine learning algorithms, could accurately forecast the survival duration of HCC, achieving an AUC value of approximately 0.9. Conclusions This study represented the inaugural report on the deleterious impact and prognostic significance of ADAM family signals within the tumor microenvironment of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/000a5d98243250c138efd38742555d600655a9e5" target='_blank'>
              Unraveling the role of ADAMs in clinical heterogeneity and the immune microenvironment of hepatocellular carcinoma: insights from single-cell, spatial transcriptomics, and bulk RNA sequencing
              </a>
            </td>
          <td>
            Junhong Chen, Qihang Yuan, Hewen Guan, Yuying Cui, Chang Fu, Tianfu Wei, Kai Liu
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The neuropathological assessment of central nervous system (CNS) tumors often necessitates extensive molecular analyses to arrive at a diagnosis consistent with the WHO CNS classification (2021). Key diagnostic indicators include the epigenetic profile of the tumor, MGMT methylation status, gene amplifications, deletions, duplications, gene fusions, and point mutations. Achieving comprehensive classification typically involves multiple distinct analyses, including NGS gene panels, fluorescent in situ hybridization (FISH), pyrosequencing, Sanger sequencing, and methylation bead arrays. The emergence of the nanopore long-read sequencing enables whole-genome sequencing (WGS) of a tumor sample, consolidating all diagnostically relevant information into a single analysis.



 DNA from fresh-frozen tissue from 52 CNS tumors was sequenced retrospectively using nanopore Promethion flow cells. Basecalling, methylation calling and mapping to reference genome was performed during sequencing over a period of 72 hours. Once sequencing was completed, copy-number variations (CNVs), structural variants (SVs) and single-nucleotide variants (SNVs) were called using the Epi2me software suite. Methylation on CpG sites overlapping the EPIC 850K array were extracted and used for classification using the MNP-flex platform. All relevant findings were compared to results from the routine diagnostic work-up.



 Nanopore WGS provided on average 35x coverage after 72 hours of sequencing on a single flow cell. The average turnaround time from obtaining a tumor biopsy to producing a report was five days. Classification based on the methylation profile was concordant with molecular pathology in 46 of 52 cases (88%). Three cases were classified as control tissue, characterized by low tumor-cell content, while four cases displayed discordance with the initial classification. All relevant gene-fusion events and amplifications were successfully identified in the nanopore data, with two additional relevant findings reported that had not been previously detected. Seventy-three clinically relevant SNVs were reported in the nanopore data matching previous findings while seven SNVs were unreported, six of which were from samples with low tumor content.



 Nanopore whole-genome sequencing showed high concordance with traditional methods for CNS tumor classification. It holds potential for improving diagnostic accuracy and shortening the turnaround time from surgery to complete molecular diagnosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9c13c79de86cc5a208f04c0338de38086b1cdd" target='_blank'>
              JS04.4.A SHORTENING THE TURNAROUND TIME FROM SURGERY TO COMPLETE MOLECULAR DIAGNOSIS
              </a>
            </td>
          <td>
            S. Halldorsson, H. Leske, R. M. Nagymihaly, A Patel, F. Sahm, E. Vik-Mo
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1dca550adace3299de2700d468a5c183e07ebc1" target='_blank'>
              Widespread mutagenesis and chromosomal instability shape somatic genomes in systemic sclerosis
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Thomas Blouin, James McCollum, Latarsha Porcher, François Virard, Jiri Zavadil, Carol Feghali-Bostwick, Natalie Saini
          </td>
          <td>2024-10-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48137767fd26c2f7051bdcf540a74692bdb8a20b" target='_blank'>
              The role of hexokinases in epigenetic regulation: altered hexokinase expression and chromatin stability in yeast
              </a>
            </td>
          <td>
            Srinivasu Karri, Quinn Dickinson, Jing Jia, Yi Yang, Haiyun Gan, Zhiquan Wang, Yibin Deng, Chuanhe Yu
          </td>
          <td>2024-08-27</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Over the last decade with the advent of new genetic technologies such as CRISPR and more recently described, seekRNA, we now have the ability to transfer genetic materials through retroviruses or other tools to directly edit the aberrant genes and repair them. We describe and categorize the genodermatoses in tabular forms devoting each genodermatosis to its own table which describes the clinical characteristics, genetic inheritance pattern, pathophysiology with molecular and genetic pathways, current treatments and future anticipated treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44072062d6572b1a7db4c4c78619b3ba1799f97" target='_blank'>
              Genodermatoses and Therapeutics on the Horizon: A Review and Table Summary
              </a>
            </td>
          <td>
            Edward M Klepper
          </td>
          <td>2024-08-31</td>
          <td>Journal of Clinical Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Precursor T-cell acute lymphoblastic leukemia (Pre-T ALL) is a malignant neoplastic disease in which T-cells proliferate in the bone marrow. Single-cell sequencing technology could identify characteristic cell types, facilitating the study of the therapeutic mechanisms in Pre-T ALL.


METHODS
The single-cell sequencing data (scRNA-seq) of Pre-T ALL were obtained from public databases. Key immune cell subpopulations involved in the progression of Pre-T ALL were identified by clustering and annotating the cellular data using AUCell. Next, pseudo-temporal analysis was performed to identify the differentiation trajectories of immune cell subpopulations using Monocle. Copy number mutation landscape of cell subpopulations was characterized by inferCNV. Finally, cellphoneDB was used to analyze intercellular communication relationships.


RESULTS
A total of 10 cellular subpopulations were classified, with Pre-T ALL showing a higher proportion of NK/T cells. NK/T cells were further clustered into two subpopulations. Stem T cells showed a high expression of marker genes related to hematopoietic stem cells, Naive T cells had a high expression of CCR7, CCR7, RCAN3, and NK cells high-expressed KLRD1, TRDC. The cell proliferation was reduced and the activation of T cell was increased during the differentiation of stem T cells to Naive T cells. We observed interaction between stem T cells with dendritic cells such as CD74-COPA, CD74-MIF as well as co-inhibition-related interactions such as LGALS9-HAVCR2, TGFB1-TGFBR3.


CONCLUSION
Stem T cells were involved in the development of Pre-T-ALL through the regulatory effects of transcription factors (TFs) KLF2 and FOS and multiple ligand-receptor pairs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bda61ec4282f1229f78a0266e24331d8eec22a53" target='_blank'>
              Single-cell sequencing technology to characterize stem T-cell subpopulations in acute T-lymphoblastic leukemia and the role of stem T-cells in the disease process.
              </a>
            </td>
          <td>
            Yan Li, Zhenwei Jia, Xiaoyan Liu, Hongbo Zhao, Guirong Cui, Jianmin Luo, Xiaoyang Kong
          </td>
          <td>2024-10-17</td>
          <td>Aging</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Serrated lesions are common precancerous pathways in colorectal cancer (CRC), but the process by which they progress to malignancy remains unclear. We aimed to elucidate this progression through a single-cell RNA landscape. We conducted single-cell RNA sequencing on three normal colonic tissues and fifteen SLs (including HPs, SSLs, SSLD, and TSAs) and integrated these data with datasets containing tumor samples. We identified three invasive malignant epithelial cell subtypes related to CRC progression: SLC1, SLC2, and tumor cell. SLC1, specific to SSLs, is involved in cell proliferation and shows a continuum of malignancy in gene expression. TSA-specific SLC2 exhibited FOXQ1 upregulation and active EMT, indicating invasiveness. The trajectory analysis showed that HPs do not progress to cancer, and different SL types are linked to the MSI status of advanced CRCs. We validated molecular drivers in premalignant lesions and later carcinogenesis. In the tumor microenvironment, CAF and pre-CAF fibroblast subtypes associated with progression were identified. During the premalignant stage, SLC1 triggered CD8+ T cell responses, while at the advanced stage, CAFs promoted tumor invasion and metastasis via FN1-CD44, influencing tumor progression and the treatment response. Our findings highlight transcriptional changes across serrated pathway stages, aiding in early CRC diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f52080cc10a19d5a436d2837e1d1e6040efb30" target='_blank'>
              Single-Cell Transcriptomics Reveals Cellular Heterogeneity and Drivers in Serrated Pathway-Driven Colorectal Cancer Progression
              </a>
            </td>
          <td>
            Jiahui Wang, Yu Zhang, Xinyi Chen, Qi Sheng, Junluo Yang, Yuyao Zhu, Yuhang Wang, Fangrong Yan, Jingya Fang
          </td>
          <td>2024-10-11</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d8d8f2f89f8ee70047ce83a13996f46168eafe" target='_blank'>
              Star wars against leukemia: attacking the clones.
              </a>
            </td>
          <td>
            Monika M. Toma, Tomasz Skorski
          </td>
          <td>2024-09-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The alteration of the glioblastoma genome by epigenetic mechanisms that share functions with normal developmental processes, such as self-renewal and fate specification of neural stem cells (NSC), is a key piece of evidence that links brain cancer pathogenesis with dysregulated stem cell functions. A patient-specific comparison of glioblastoma stem cells (GSC) with NSC, a putative cell of origin of at least a proportion of these tumours, is not feasible as patient-matched endogenous NSC are not surgically accessible and all epigenetic studies in glioblastoma have so far compared epigenetic changes of different tumours with each other or to comparators obtained from foetal brains or an unrelated donor. We reasoned that availability of syngeneic GSC and NSC pairs would allow to identify crucial epigenetic differences on a patient-specific basis and would provide essential therapeutic contrast to define disease-and patient-intrinsic biomarkers of drug response that are less confounded by germline variation.



 We have derived GSC from IDH-wildtype glioblastoma and harnessed state-of-the-art stem cell technologies to generate patient-matched fibroblast-derived EPSC, which were induced to NSC (iNSC). We generated transcriptome, DNA methylome, ChIP-Seq (H3K4me3, H3K27ac, H3K3me3, H3K27me3) and ATACSeq datasets for 10 GIC/iNSC pairs.



 Comparative integrated analysis has shown that redistribution of selected histone marks as well as shifts in chromatin states across the genome identifies differentially expressed genes, including novel druggable regulatory mechanisms in GSC, which are specific of the neoplastic context. In particular, we identified METTL7B as an essential regulator of lineage specification and plasticity in human glioblastoma, with impact on both tumour size and invasiveness in an in vivo xenograft model. Single cell transcriptomic analysis of these tumours and of cerebral organoids derived from expanded potential stem cells (EPSCs) overexpressing METTL7B reveal a regulatory role for the gene in the neural stem cells-to-astrocyte differentiation trajectory. Mechanistically, METTL7B downregulates the expression of key neuronal differentiation players, including SALL2, via post-translational modifications of histone marks.



 SYNGN is an epigenetic platform which can identify novel disease mechanisms and druggable targets in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be96bd21ac249c6fcb9083c430d82766e4fcbf5a" target='_blank'>
              P01.14.B SYNGN: A NOVEL EPIGENETIC PLATFORM TO DISCOVER PATIENT-SPECIFIC DISEASE MECHANISMS AND DRUGGABLE TARGETS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Marino, C. Vinel, M. Constantinou
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Desmoplastic small round cell tumor (DSRCT) is an aggressive sarcoma subtype that is driven by the EWS-WT1 chimeric transcription factor. The prognosis for DSRCT is poor, and major advances in treating DSCRT have not occurred for over two decades. To identify effective therapeutic approaches to target DSRCT, we conducted a high-throughput drug sensitivity screen in a DSRCT cell line assessing chemosensitivity profiles for 79 small-molecule inhibitors. DSRCT cells were sensitive to PARP and ATR inhibitors (PARPi, ATRi), as monotherapies and in combination. These effects were recapitulated using multiple clinical PARPi and ATRi in three biologically distinct, clinically-relevant models of DSRCT, including cell lines, a patient-derived xenograft (PDX)-derived organoid model, and a cell line-derived xenograft mouse model. Mechanistically, exposure to a combination of PARPi and ATRi caused increased DNA damage, G2/M checkpoint activation, micronuclei accumulation, replication stress, and R-loop formation. EWS-WT1 silencing abrogated these phenotypes and was epistatic with exogenous expression of the R-loop resolution enzyme RNase H1 in reversing the sensitivity to PARPi and ATRi monotherapies. The combination of PARPi and ATRi also induced EWS-WT1-dependent cell-autonomous activation of the cGAS/STING innate immune pathway and cell surface expression of PD-L1. Taken together, these findings point towards a role for EWS-WT1 in generating R-loop-dependent replication stress that leads to a targetable vulnerability, providing a rationale for the clinical assessment of PARPi and ATRi in DSRCT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33f00435911403846b95b540256a684bbb112555" target='_blank'>
              Replication Stress is an Actionable Genetic Vulnerability in Desmoplastic Small Round Cell Tumors.
              </a>
            </td>
          <td>
            Asuka Kawai-Kawachi, Madison M Lenormand, Clémence Astier, Noé Herbel, Meritxell B Cutrona, Carine Ngo, M. Garrido, Thomas Eychenne, N. Dorvault, Laetitia Bordelet, Fei Fei Song, Ryme Bouyakoub, Anastasia Loktev, Antonio Romo-Morales, C. Henon, Léo Colmet-Daage, Julien Vibert, Marjorie Drac, R. Brough, Etienne Schwob, Oliviano Martella, G. Pinna, Janet M Shipley, Sibylle Mittnacht, Astrid Zimmermann, Aditi Gulati, Olivier Mir, Axel Le Cesne, M. Faron, Charles Honoré, Christopher J. Lord, Roman Chabanon, S. Postel-Vinay
          </td>
          <td>2024-10-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Genome-wide association studies implicate common genetic variations in the LRP1 (low-density lipoprotein receptor-related protein 1) locus at risk for multiple vascular diseases and traits. However, the underlying biological mechanisms are unknown.


METHODS
Fine mapping analyses included Bayesian colocalization to identify the most likely causal variant. Human induced pluripotent stem cells were genome-edited using CRISPR-Cas9 to delete or modify candidate enhancer regions and generate LRP1 knockout cell lines. Cells were differentiated into smooth muscle cells through a mesodermal lineage. Transcription regulation was assessed using luciferase reporter assay, transcription factor knockdown, and chromatin immunoprecipitation. Phenotype changes in cells were conducted using cellular assays, bulk RNA sequencing, and mass spectrometry.


RESULTS
Multitrait colocalization analyses pointed at rs11172113 as the most likely causal variant in LRP1 for fibromuscular dysplasia, migraine, pulse pressure, and pulmonary function trait. We found the rs11172113-T allele to associate with higher LRP1 expression. Genomic deletion in induced pluripotent stem cell-derived smooth muscle cells supported rs11172113 to locate in an enhancer region regulating LRP1 expression. We found transcription factors MECP2 (methyl CpG binding protein 2) and SNAIL to repress LRP1 expression through an allele-specific mechanism, involving SNAIL interaction with disease risk allele. LRP1 knockout decreased induced pluripotent stem cell-derived smooth muscle cell proliferation and migration. Differentially expressed genes were enriched for collagen-containing extracellular matrix, connective tissue development, and lung development. LRP1 knockout and deletion of rs11172113 enhancer showed potentiated canonical TGF-β (transforming growth factor beta) signaling through enhanced phosphorylation of SMAD2/3. Analyses of the protein content of decellularized extracts indicated partial extracellular matrix remodeling involving enhanced secretion of CYR61, a known LRP1 ligand involved in vascular integrity and TIMP3, implicated in extracellular matrix maintenance and also known to interact with LRP1.


CONCLUSIONS
Our findings support allele-specific LRP1 gene repression by the endothelial-to-mesenchymal transition regulator SNAIL. We propose decreased LRP1 expression in smooth muscle cells to remodel the extracellular matrix enhanced by TGF-β as a potential mechanism of this pleiotropic locus for vascular diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130d9d0beb72cded358387a84a2d2c98e035e14a" target='_blank'>
              LRP1 Repression by SNAIL Results in ECM Remodeling in Genetic Risk for Vascular Diseases.
              </a>
            </td>
          <td>
            Lu Liu, Joséphine Henry, Yingwei Liu, C. Jouve, J. Hulot, A. Georges, N. Bouatia-Naji
          </td>
          <td>2024-10-02</td>
          <td>Circulation research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) maintain homeostasis in the hematopoietic ecosystem, which is tightly regulated at multiple layers. Acute myeloid leukemia (AML) is a severe hematologic malignancy driven by genetic and epigenetic changes that lead to the transformation of leukemia stem cells (LSCs). Since somatic mutations in DNA methylation-related genes frequently occur in AML, DNA methylation is widely altered and functions as a starting engine for initiating AML. Additionally, RNA modifications, especially N6-methyladenosine (m6A), also play an important role in the generation and maintenance of the hematopoietic ecosystem, and AML development requires reprogramming of m6A modifications to facilitate cells with hallmarks of cancer. Given the complex pathogenesis and poor prognosis of AML, it is important to fully understand its pathogenesis. Here, we mainly focus on DNA methylation and RNA m6A modification in hematopoiesis and AML and summarize recent advances in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92f8b80ecba345e57d84b39e3572cdbd8a3dd106" target='_blank'>
              Epigenetic modifications in hematopoietic ecosystem: a key tuner from homeostasis to acute myeloid leukemia
              </a>
            </td>
          <td>
            Shu-Qi Yao, Rongxia Guo, Wen Tian, Yanbing Zheng, Jin Hu, G-X Han, Rong Yin, Fuling Zhou, Haojian Zhang
          </td>
          <td>2024-09-12</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16f71ff68478a0f30f71105d86b66a3bf8d6b91" target='_blank'>
              Single-Cell Sequencing: High-Resolution Analysis of Cellular Heterogeneity in Autoimmune Diseases.
              </a>
            </td>
          <td>
            Xuening Tang, Yudi Zhang, Hao Zhang, Nan Zhang, Ziyu Dai, Q. Cheng, Yongzhen Li
          </td>
          <td>2024-08-26</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Olaparib and temozolomide (OT) combination therapy is in clinical trial evaluation for adolescent rhabdomyosarcoma (RMS) muscle cancers. Unfortunately, resistance to OT has been reported in other cancers, with no counterstrategies available. Using preclinical mouse xenograft experiments, we show that OT is effective at curbing RMS growth, yet a subset of tumors develop resistance that is associated with transcriptomic changes that occur in the absence of recurrent genomic mutation. Importantly, a vast majority of resistant RMS models upregulate the PIK3CA/AKT pathway, which in turn activates NRF2 transcription factor phosphorylation and subsequent transcriptional expression of multidrug resistance ABC transport proteins that rapidly efflux drugs from cells. Using dynamic real-time imaging of tumor growth and assessing drug responses at single-cell resolution in human cell line models, we found that the PIK3CA inhibitor alpelisib re-sensitized resistant RMS cells to OT therapy by suppressing the expression of ABC transport proteins. Excitingly, RMS uses the same PIK3CA/AKT pathway activation to drive resistance to standard-of-care combination therapy vincristine, actinomycin D, and cyclophosphamide (VAC) and the combination of OT + alpelisib effectively killed VAC-resistant RMS. Alpelisib also re-sensitized resistant RMS to chemotherapy in preclinical xenograft mouse models resulting in reduced tumor burden and extended disease-free survival. Our work defines a common resistance pathway in RMS and has credentialled a new preclinical strategy to kill therapy-resistant RMS.
 Citation Format: Yueyang Wang, Qiqi Yang, Chuan Yan, David M. Langenau. The PIK3CA/AKT pathway drives therapy resistance in rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de779864c638c41dc5737016f4c37a97b2fb0f5c" target='_blank'>
              Abstract A072 The PIK3CA/AKT pathway drives therapy resistance in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Yueyang Wang, Qiqi Yang, Chuan Yan, D. Langenau
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 Since the approval of the first checkpoint inhibitor, ipilimumab (α-CTLA4), there continues to be enthusiasm around leveraging the immune system to eradicate solid tumors. However, unknowns about the cellular mechanism(s) that cause some patients to durably respond while others relapse, continue to stymie the field. To support ongoing efforts elucidating cell subsets responsible for differences in patient responses, we aim to evaluate the functional heterogeneity of T-cell subsets responding to checkpoint inhibitors in mouse models of solid tumors. Single-cell RNA-seq has enabled discoveries of transcriptionally defined cell subsets, but understanding their function require inference from transcription, flow-based validation biased by pre-selection of subsets with specific markers, or reliance on surrogate markers for cell-extrinsic functions (e.g., cytotoxicity). Thus, functional heterogeneity could be obscured. To address this, we leveraged a novel technology that converges live, cellular functions with transcriptomics at the resolution of single cell-cell interactions. This technology captures tens of thousands of individual cells or defined combinations of groups of cells in bio-compatible hydrogel compartments, enabling longitudinal imaging over days to weeks. Throughout this period, functions are characterized via bright-field or immunofluorescence readouts, including antibodies, cytokine capture beads, and dyes. Cells are then lysed within compartments to generate barcoded libraries for mRNA-sequencing and gene signatures from individual cells computationally deconvoluted. With this technology, we developed a novel assay that enables longitudinal, unbiased functional characterization of individual T cells mapped to single-cell transcriptomics. Single mouse primary T cells were encapsulated with one or more MC38 target cells and imaged over time. We validated combinations of different readouts can be measured from the same T-cell, including cell-counts for proliferation, CD8/3 expression to differentiate T cells from target cells, activation markers, and / or target cell death via Sytox. Thus, single T cells can be multi-functionally evaluated over time, enabling the study of attributes inaccessible at high scale with prior technologies, including kinetics of killing or the number of target cells killed by a single T-cell. For assays combining cytotoxicity and surface receptors, we generated barcoded libraries from functionally characterized T cells for sequencing and developed an analytical pipeline that deconvolutes mRNA from single T cells mapped to their multi-functional profiles, to identify novel pathways that regulate anti-tumor functions. We will leverage this workflow to functionally characterize single T cells in tumor-bearing mice treated with checkpoint inhibitors and map the data to underlying transcriptomics. In doing so, we aim to discover functional subsets, potentially with identical transcriptional profiles, to support hypothesis generation around mechanisms governing anti-tumor efficacy of checkpoint inhibitors.
 Citation Format: Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer. Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b89db5bb53ca20d5ac128fa18040b839f5041ca9" target='_blank'>
              Abstract A025: Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors
              </a>
            </td>
          <td>
            Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
The identification of the most appropriate targeted therapies for advanced cancers is challenging. We performed a molecular profiling of metastatic solid tumors utilizing a comprehensive next-generation sequencing (NGS) assay to determine genomic alterations' type, frequency, actionability, and potential correlations with PD-L1 expression.


METHODS
A total of 304 adult patients with heavily pretreated metastatic cancers treated between January 2019 and March 2021 were recruited. The CLIA-/UKAS-accredit Oncofocus assay targeting 505 genes was used on newly obtained or archived biopsies. Chi-square, Kruskal-Wallis, and Wilcoxon rank-sum tests were used where appropriate. Results were significant for P < .05.


RESULTS
A total of 237 tumors (78%) harbored potentially actionable genomic alterations. Tumors were positive for PD-L1 in 68.9% of cases. The median number of mutant genes/tumor was 2.0 (IQR: 1.0-3.0). Only 34.5% were actionable ESCAT Tier I-II with different prevalence according to cancer type. The DNA damage repair (14%), the PI3K/AKT/mTOR (14%), and the RAS/RAF/MAPK (12%) pathways were the most frequently altered. No association was found among PD-L1, ESCAT, age, sex, and tumor mutational status. Overall, 62 patients underwent targeted treatment, with 37.1% obtaining objective responses. The same molecular-driven treatment for different cancer types could be associated with opposite clinical outcomes.


CONCLUSIONS
We highlight the clinical value of molecular profiling in metastatic solid tumors using comprehensive NGS-based panels to improve treatment algorithms in situations of uncertainty and facilitate clinical trial recruitment. However, interpreting genomic alterations in a tumor type-specific manner is critical.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b13a3de4d8a4a44c9264bbeddda708851f093875" target='_blank'>
              Next-generation sequencing-based evaluation of the actionable landscape of genomic alterations in solid tumors: the "MOZART" prospective observational study.
              </a>
            </td>
          <td>
            F. Schettini, Marianna Sirico, M. Loddo, Gareth H Williams, Keeda-Marie Hardisty, Paul Scorer, R. Thatcher, Pablo Rivera, Manuela Milani, C. Strina, G. Ferrero, Marco Ungari, C. Bottin, Fabrizio Zanconati, Nicolò de Manzini, S. Aguggini, Richard Tancredi, E. Fiorio, Antonio Fioravanti, Maurizio Scaltriti, Daniele Generali
          </td>
          <td>2024-08-23</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2b01750179731a3467c6191264b3a3625ebdeb" target='_blank'>
              Altered methylation of imprinted genes in neuroblastoma: implications for prognostic refinement
              </a>
            </td>
          <td>
            Medha Suman, Maja Löfgren, S. Fransson, Jewahri Idris Yousuf, Johanna Svensson, A. Djos, Tommy Martinsson, P. Kogner, T. Kling, Helena Carén
          </td>
          <td>2024-08-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="TRIM44, a tripartite motif (TRIM) family member, is pivotal in linking the ubiquitin-proteasome system (UPS) to autophagy in multiple myeloma (MM). However, its prognostic impact and therapeutic potential remain underexplored. Here, we report that TRIM44 overexpression is associated with poor prognosis in a Multiple Myeloma Research Foundation (MMRF) cohort of 858 patients, persisting across primary and recurrent MM cases. TRIM44 expression notably increases in advanced MM stages, indicating its potential role in disease progression. Single-cell RNA sequencing across MM stages showed significant TRIM44 upregulation in smoldering MM (SMM) and MM compared to normal bone marrow, especially in patients with t(4;14) cytogenetic abnormalities. This analysis further identified high TRIM44 expression as predictive of lower responsiveness to proteasome inhibitor (PI) treatments, underscoring its critical function in the unfolded protein response (UPR) in TRIM44-high MM cells. Our findings also demonstrate that TRIM44 facilitates SQSTM1 oligomerization under oxidative stress, essential for its phosphorylation and subsequent autophagic degradation. This process supports the survival of PI-resistant MM cells by activating the NRF2 pathway, which is crucial for oxidative stress response and, potentially, other chemotherapy-induced stressors. Additionally, TRIM44 counters the TRIM21-mediated suppression of the antioxidant response, enhancing MM cell survival under oxidative stress. Collectively, our discoveries highlight TRIM44’s significant role in MM progression and resistance to therapy, suggesting its potential value as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016505694bfc0a30037baa711cb807808634a1de" target='_blank'>
              TRIM44, a Novel Prognostic Marker, Supports the Survival of Proteasome-Resistant Multiple Myeloma Cells
              </a>
            </td>
          <td>
            Trung Vu, Yuqin Wang, Annaliese Fowler, Anton Simieou, Nami McCarty
          </td>
          <td>2024-08-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a disease characterized by spatiotemporal heterogeneity of tumor clones. Different genetic aberrations can be observed simultaneously in tumor cells from different loci, and as the disease progresses, new subclones may appear. The role of liquid biopsy, which is based on the analysis of tumor DNA circulating in the blood plasma, continues to be explored in MM. Here, we present an analysis of the STR profiles and mutation status of the KRAS, NRAS, and BRAF genes, evaluated in plasma free circulating tumor DNA (ctDNA), CD138+ bone marrow cells, and plasmacytomas. The prospective single-center study included 97 patients, with a median age of 55 years. Of these, 94 had newly diagnosed symptomatic MM, and three had primary plasma cell leukemia. It should be noted that if mutations were detected only in ctDNA, “non-classical” codons were more often affected. A variety of adverse laboratory and clinical factors have been associated with the detection of rare KRAS or NRAS gene mutations in bone marrow or ctDNA, suggesting that these mutations may be factors of an unfavorable prognosis for MM. Liquid biopsy studies provide undeniable fundamental information about tumor heterogeneity and clonal evolution in MM. Moreover, we focus on using liquid biopsy to identify new high-risk factors for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef6c42652b61bb294400045b1d3bc2249d75f9b" target='_blank'>
              Loss of Heterozygosity and Mutations in the RAS-ERK Pathway Genes in Tumor Cells of Various Loci in Multiple Myeloma
              </a>
            </td>
          <td>
            M. Soloveva, M. Solovev, N. Risinskaya, E. Nikulina, I. Yakutik, B. Biderman, Tatiana Obukhova, Y. Chabaeva, Sergej Kulikov, Andrey Sudarikov, L. Mendeleeva
          </td>
          <td>2024-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Drug resistance in tumors constitutes a significant obstacle to tumor therapy. Head and neck squamous cell carcinoma (HNSCC) presents a major challenge due to its deep anatomical location, limited space, and complex structure. These factors complicate surgical procedures and hinder the effectiveness of chemoradiotherapy, leading to poor prognosis and reduced quality of life. However, there is hope in the form of circular RNAs (circRNAs), non-coding RNA molecules with a closed-loop structure that exhibits superior stability and resistance to degradation compared to linear RNAs. Recent advances in high-throughput sequencing and bioinformatics technology revealed that circRNAs participate in tumor proliferation, invasion, migration, and drug resistance. This review aims to summarize current research progress on the involvement of circRNAs in drug resistance of HNSCC and provide valuable insights for the prevention and mitigation of drug resistance in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79955e57ba8f160fd4d0536e1d8ad58c88c872c" target='_blank'>
              Research progress on the role and mechanism of circular RNA in drug resistance of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Hongli Zeng, Junshang Ge, Yi Meng, Qian Wang, Mei Yang, Zhaoyang Zeng, Wei Xiong, Xuyu Zu
          </td>
          <td>2024-09-02</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Melanoma is the deadliest form of skin cancer with a 5-year survival rate of less than 20%. While significant strides have been made in the field of kinase-targeted and immune-based therapies for melanoma, the development of resistance to these therapeutic agents has hindered the success of treatment. Drug-resistant melanoma is particularly reliant on enhanced cap-dependent translation to drive the production of oncoproteins that promote growth and survival. The m7GpppX cap-binding protein eukaryotic translation initiation factor 4E (eIF4E) is the rate-limiting factor of cap-dependent translation initiation, and its overexpression in melanoma tumors has been shown to drive resistance to BRAFV600E kinase-targeted inhibitors. These findings point to eIF4E-targeted therapies as a promising strategy to overcome drug resistance in melanoma. Herein, we build upon our previous work of developing cell-permeable cap analogue inhibitors to design second-generation cap analogues that inhibit eIF4E-mediated cap-dependent translation in drug-resistant melanoma cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30a3a2f8096ad4422aaf3a1a0bdabfac09f97075" target='_blank'>
              Second-Generation Cap Analogue Prodrugs for Targeting Aberrant Eukaryotic Translation Initiation Factor 4E (eIF4E) Activity in Drug-Resistant Melanoma
              </a>
            </td>
          <td>
            Emilio L. Cárdenas, Rachel L. O’Rourke, Arya Menon, Gabriela Vega-Hernández, Jennifer Meagher, Jeanne Stuckey, Amanda L. Garner
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8b3281c13d1ffc75f1261f03703dc8638c833" target='_blank'>
              A protein expression atlas on tissue samples and cell lines from cancer patients provides insights into tumor heterogeneity and dependencies.
              </a>
            </td>
          <td>
            Jun Li, Wei Liu, K. Mojumdar, Hong Kim, Zhicheng Zhou, Z. Ju, Shweth V. Kumar, P. Ng, Han Chen, Michael A. Davies, Yiling Lu, Rehan Akbani, Gordon B. Mills, Han Liang
          </td>
          <td>2024-09-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="TET2 mutations (mTET2) are common genetic events in myeloid malignancies and clonal hematopoiesis (CH). These mutations arise in the founding clone and are implicated in many clinical sequelae associated with oncogenic feedforward inflammatory circuits. However, the direct downstream effector of mTET2 responsible for the potentiation of this inflammatory circuit is unknown. To address this, we performed scRNA and scATAC-seq in COVID-19 patients with and without TET2-mutated CH reasoning that the inflammation from COVID-19 may highlight critical downstream transcriptional targets of mTET2. Using this approach, we identified MALAT1, a therapeutically tractable lncRNA, as a central downstream effector of mTET2 that is both necessary and sufficient to induce the oncogenic pro-inflammatory features of mTET2 in vivo. We also elucidate the mechanism by which mTET2 upregulate MALAT1 and describe an interaction between MALAT1 and P65 which leads to RNA "shielding" from PP2A dephosphorylation thus preventing resolution of inflammatory signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e647627f6805fe19dbf59a6bc50dfd7de16f1c5" target='_blank'>
              RNA shielding of P65 is required to potentiate oncogenic inflammation in TET2 mutated clonal hematopoiesis.
              </a>
            </td>
          <td>
            N. Ben-Crentsil, Wazim Mohammed Ismail, M. Balasis, Hannah Newman, Ariel Quintana, M. Binder, Traci L. Kruer, Surendra Neupane, M. Ferrall-Fairbanks, Jenna A. Fernandez, T. Lasho, C. Finke, Mohammed L Ibrahim, K. McGraw, Michael Wysota, Amy L Aldrich, Christopher B Ryder, Christopher T Letson, Joshua Traina, Amy F. McLemore, N. Droin, A. Shastri, Seongseok Yun, E. Solary, D. Sallman, Amer A Beg, Li Ma, Alexandre Gaspar-Maia, Mirinal S Patnaik, E. Padron
          </td>
          <td>2024-08-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune evasion is not only critical for tumor initiation and progression, but also determines the efficacy of immunotherapies. Through iterative in vivo CRISPR screens with seven syngeneic tumor models, we identified core and context-dependent immune evasion pathways across cancer types. This valuable high-confidence dataset is available for the further understanding of tumor intrinsic immunomodulators, which may lead to the discovery of effective anticancer therapeutic targets. With a focus on triple-negative breast cancer (TNBC), we found that Mga knock-out significantly enhances antitumor immunity and inhibits tumor growth. Transcriptomics and single-cell RNA sequencing analyses revealed that Mga influences various immune-related pathways in the tumor microenvironment. Our findings suggest that Mga may play a role in modulating the tumor immune landscape, though the precise mechanisms require further investigation. Interestingly, we observed that low MGA expression in breast cancer patients correlates with a favorable prognosis, particularly in those with active interferon-γ signaling. These observations provide insights into tumor immune escape mechanisms and suggest that further exploration of MGA's function could potentially lead to effective therapeutic strategies in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a148d1222506ce8171cce7b1609e8c01da8e2a" target='_blank'>
              In vivo CRISPR screens identify Mga as an immunotherapy target in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Xu Feng, Chang Yang, Yuanjian Huang, Dan Su, Chao Wang, L. L. Wilson, Ling Yin, Mengfan Tang, Siting Li, Zhen Chen, Dandan Zhu, Shimin Wang, Shengzhe Zhang, Jie Zhang, Huimin Zhang, Litong Nie, Min Huang, Jae-Ll Park, Traver Hart, Dadi Jiang, Kuirong Jiang, Junjie Chen
          </td>
          <td>2024-09-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Background: Significant progress has been made in identifying mutations and defining biological heterogeneity in neuroblastoma (NB) using high-throughput sequencing approaches. However, functional roles for many specific genetic alterations in NB initiation, growth, and metastasis remain to be fully defined. Pathogenic and/or likely pathogenic (P/LP) alterations in genes involved in DNA Damage Response (DDR) pathways were identified in approximately 30% of patients enrolled in the SickKids Cancer Sequencing Program (KiCS). Despite being well studied in many adult cancers, roles for DDR disruption and potential therapeutic implications remains poorly understood for NB and other pediatric solid tumors. Aims: We aimed to define roles for DDR-deficiency in NB progression and assess the therapeutic potential of targeted agents using patient-specific DDR loss-of-function NB models. Methods: We incorporated loss-of-function mutations in atm, brca2, palb2, and bard1 into an established zebrafish MYCN-driven model of human NB and characterized tumor progression, transcriptional signatures, and response to candidate therapeutics in vivo. Human cell lines with DDR protein knock-down were used to assess candidate drug combinations in vitro. Results: Mutations in DDR genes were found to enhance tumor penetrance and metastasis in MYCN-driven zebrafish NB, and resulted in upregulation of proliferation, cell cycle checkpoint, and DNA damage repair transcriptional signatures. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Conclusions: Our data supports an in vivo functional role for DDR-deficiency in NB, as well as potential molecular vulnerabilities. Pre-clinical assessments in vitro and in vivo suggest therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes.
 Citation Format: Madeline N. Hayes, Sarah Cohen-Gogo, Lynn Kee, Mehdi Layeghifard, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, Anita Villani, Adam Shlien, Daniel A. Morgenstern, Meredith S Irwin. DNA damage response deficiency enhances neuroblastoma progression and sensitivity to combination PARP and ATR inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cc0578406ca9a79efb4aae714485b02b1ea3da3" target='_blank'>
              Abstract A007 DNA damage response deficiency enhances neuroblastoma progression and sensitivity to combination PARP and ATR inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, M. Layeghifard, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f52f7f8ee82929f6197e0c94074a6bfe46589e7" target='_blank'>
              Derivation of human primary prostate epithelial cell lines by differentially targeting the CDKN2A locus along with expression of hTERT
              </a>
            </td>
          <td>
            Jason S Wasserman, Holly Fowle, Rumesa Hashmi, Diba Atar, Kishan R Patel, Amir Yarmahmoodi, Alexander W. Macfarlane, Yinfei Tan, Edna Cukierman, Bojana Gligorijevic, Adam Karami, Kelly A. Whelan, Kerry S. Campbell, Xavier Graña
          </td>
          <td>2024-09-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Overall survival for patients with advanced PDAC remains less than 1 year. Biomarkers are needed to select chemotherapy regimens, either alone or in combination with novel agents. PASS-01 compared mFOLFIRINOX and Gemcitabine/nab-Paclitaxel in a phase II randomised control trial while prospectively exploring tissue, liquid and organoid biospecimens to interrogate putative biomarkers of chemotherapy response and resistance. Methods PASS-01 was conducted across 6 sites in North America. Patients with de novo metastatic disease and a lesion amenable to core biopsy were included. Cases with germline pathogenic variants in BRCA1/2 or PALB2 were excluded. Up to 6 core biopsies were obtained with priorities for clinical diagnosis, patient derived organoids (PDOs), and whole genome and transcriptome sequencing (WGTS). WGTS were performed at a single site following laser capture microdissection. WGS reports were clinically (CAP) accredited and annotated mutations, copy number alterations, mutational signatures, TMB and structural variant patterns. KRAS allelic states were documented with major imbalances in mutant KRAS (KRASmaj) defined as mutant: WT copy number >3. Transcriptomic subtypes (classical vs. basal-like) were classified as previously described (Moffitt et al., 2015). WGTS results were discussed at a molecular tumor board with PDO pharmacotyping to determine best second line options. Results: 160 patients were included in the intention to treat population, of these WGS was available in 127 (79%) and additional NGS reports in 20 (13%). RNA-seq was successful in 119 (74%) and the basal-like subtype accounted for 29% and was associated with inferior outcomes. KRASm were evident in 91% and KRASG12D represented 45%. Tumor suppressors TP53, CDKN2A and SMAD4 were lost in 81%, 76% and 39% respectively. MTAP deletion was present in 40%, largely synonymous with CDKN2A deletions. Commonly co-altered genes in the dataset included epigenetic regulators (ARID1A, KMT2C, KDM6A) and TGFBR2. KRASmaj was present in 30% of cases and polyploid tumors in 50%. 50% of basal-like PDAC harboured KRASmaj compared to 22% of classical PDAC, p<0.01. Dominant signatures (SBS) included age related SBS1, 5 and 8. Additionally, we saw a prevalence of SBS17 and APOBEC SBS2/13, signatures which occurred more frequently compared to a similarly characterised resected cohort. Unstable genotypes were present in 24% with somatic-HRD evident in 3%. Of KRASWT PDAC, 58% harboured class I-II BRAF variants. KRAS WT PDAC had fewer co-mutations and in one case where an NGS report did not find a driver, a novel fusion in SLC4A4-RASGRF2 was identified. 28 molecular tumor boards took place throughout the trial influencing 50% obtaining a 2nd-line treatment. Conclusion: The PASS-01 cohort exhibits a heterogeneous group of genotypes and molecular subgroups. Certain mutational processes are more evident in advanced compared to resected PDAC. The impact of subgroups on survival and chemotherapy response is ongoing with a planned data lock on July 15th. Results will be presented at the meeting.
 Citation Format: Grainne M O'Kane, Gunho Jang, Trevor J Pugh, Julie Wilson, Stephanie Ramotar, Daniel A King, Dan Laheru, Ken H Yu, Kimberly J Perez, Amber Habowski, Erica S Tsang, Robert C Grant, Andrew J Aguirre, Raymond W-J Jang, Craig E Devoe, Eileen M O'Reilly, Anna Dodd, Brian M Wolpin, Xiang Y Ye, Elizabeth M Jaffee, David Tuveson, Steven Gallinger, Jennifer J Knox, Faiyaz Notta. Genomic and transcriptomic characterization of pancreatic cancer patients on the PASS-01 trial [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65f655824e6bccfd53e0df1cf2553965b9da5ae0" target='_blank'>
              Abstract PR-17: Genomic and transcriptomic characterization of pancreatic cancer patients on the PASS-01 trial
              </a>
            </td>
          <td>
            G. O’Kane, G. Jang, T. J. Pugh, Julie M. Wilson, S. Ramotar, Daniel A King, Daniel A Laheru, Kenneth H Yu, Kimberly Perez, Amber N Habowski, Erica S Tsang, Robert C Grant, A. Aguirre, Raymond W-J Jang, Craig E. Devoe, E. O'Reilly, A. Dodd, B. Wolpin, X. Y. Ye, Elizabeth M. Jaffee, D. Tuveson, Stephen Gallinger, Jennifer J Knox, F. Notta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>105</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [17],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>