<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-29 09:30:43 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>58</td>
          <td>45</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Platinum-based chemotherapy resistance remains a critical barrier in colorectal cancer (CRC) treatment. In this study, cytogenetic karyotyping was combined with transcriptomic profiling (RNA-seq) to elucidate resistance mechanisms by comparing two CRC cell lines: oxaliplatin-sensitive HCT116 and its resistant derivative HCT116 oxpl-R. Karyotyping unveiled tetraploidization and extensive genomic rearrangements in resistant cells, accompanied by prominent transcriptomic changes: 1807 differentially expressed genes (1216 upregulated and 519 downregulated). Pathway enrichment highlighted altered redox homeostasis and metabolic adaptation. Specifically, HCT116 oxpl-R cells exhibited elevated reactive oxygen species (ROS) production and enhanced energy metabolism. These findings establish a direct association between structural genomic alterations, transcriptional rewiring, and functional phenotypes in platinum resistance, providing a framework for targeting metabolic vulnerabilities in refractory CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1c5397ea3bf691ad1c8a6e4e4cf327eb00a1d" target='_blank'>
              Transcriptomic and Cytogenetic Analysis of Oxaliplatin-Resistant Colorectal Adenocarcinoma HCT116 Cells to Identify Markers Associated with Platinum Resistance
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, Ksenia Fedotova, Natalija Yartseva, Nikolay Aksenov, M. Igotti
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd5596bbafd3812efcd1d8c0a4bfdf21b8096e0" target='_blank'>
              Adaptive cancer therapy: can non-genetic factors become its achilles heel?
              </a>
            </td>
          <td>
            Gábor Valcz, Robert A. Gatenby, B. Újvári, E. Buzás, Béla Molnár
          </td>
          <td>2025-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The development of drug-resistant cell lines is essential for understanding the mechanisms of drug resistance and identifying strategies to overcome treatment failure in cancer therapy. Resistance models enable preclinical evaluation of novel compounds, repurposed drugs, and combination therapies. To generate resistant cells, parental cancer cell lines are repeatedly exposed to incrementally increasing concentrations of the target drug over several weeks. Cells that survive and proliferate at each stage are selected, expanded, and exposed to higher drug doses. The development of resistance is confirmed by quantifying and comparing the half-maximal inhibitory concentration (IC50) values between parental and resistant cells using cell viability assays and nonlinear regression analysis. Significantly increased IC50 values indicate successful adaptation to drug pressure and the development of resistance. These drug-resistant cell lines are available for comprehensive analysis, such as microarray and single-cell sequencing, as well as various in vitro or in vivo experiments. These models provide valuable tools for investigating potential therapeutic strategies to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb7439fafb8c53daf4f72d4753cdd3f7c5b1b9f" target='_blank'>
              Development of Drug-resistant Cell Lines for Experimental Procedures.
              </a>
            </td>
          <td>
            S. Kadomoto, G. Shelley, Atsushi Mizokami, Evan T. Keller
          </td>
          <td>2025-08-12</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="KRAS is mutationally activated in 45–50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRASG12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, J. Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Drug-tolerant persister (DTP) cells are a subpopulation of cancer cells capable of surviving therapeutic stress through reversible, non-genetic adaptations. These cells contribute to minimal residual disease and eventual tumor relapse. Understanding the mechanisms that govern the entry into and exit from the DTP state—such as epigenetic remodeling, metabolic rewiring, and transcriptional plasticity—reveals actionable vulnerabilities. This article reviews the biological basis of DTP reversibility, outlines the major challenges in targeting these cells, and proposes innovative therapeutic strategies including epigenetic inhibitors, metabolic disruptors, and adaptive dosing regimens. We also highlight the importance of biomarker development and dynamic monitoring. Targeting DTP cells at their reversible stage may prevent permanent resistance, offering a promising avenue to improve treatment durability and patient outcomes in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1eb48d2762dc332cbef357e887ed9a593167f02" target='_blank'>
              Drug-tolerant persister cell in cancer: reversibility, microenvironmental interplay, and therapeutic strategies
              </a>
            </td>
          <td>
            Haifeng Li, Wenlong Xu, Wenqi Cheng, Guanxiao Yu, Dong Tang
          </td>
          <td>2025-08-14</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, Gregory Schuyler Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Aim: Growing evidence points to non-genetic mechanisms underlying long-term resistance to cancer therapies. These mechanisms involve pre-existing or therapy-induced transcriptional cell states that confer resistance. However, the relationship between early transcriptional responses to treatment and the eventual emergence of resistant states remains poorly understood. Furthermore, it is unclear whether such early resistance-associated transcriptional responses are evolutionarily conserved. In this study, we examine the similarity between early transcriptional responses and long-term resistant states, assess their clinical relevance, and explore their evolutionary conservation across species. Methods: We integrated datasets on early drug responses and long-term resistance from multiple cancer cell lines, bacteria, and yeast to identify early transcriptional changes predictive of long-term resistance and assess their evolutionary conservation. Using genome-wide CRISPR-Cas9 knockout screens, we evaluated the impact of genes associated with resistant transcriptional states on drug sensitivity. Clinical datasets were analyzed to explore the prognostic value of the identified resistance-associated gene signatures. Results: We found that transcriptional states observed in drug-naive cells and shortly after treatment overlapped with those seen in fully resistant populations. Some of these shared features appear to be evolutionarily conserved. Knockout of genes marking resistant states sensitized ovarian cancer cells to Prexasertib. Moreover, early resistance gene signatures effectively distinguished therapy responders from non-responders in multiple clinical cancer trials and differentiated premalignant breast lesions that progressed to malignancy from those that remained benign. Conclusion: Early cellular transcriptional responses to therapy exhibit key similarities to fully resistant states across different drugs, cancer types, and species. Gene signatures defining these early resistance states have prognostic value in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4c9b8cf4452549a749a1c9bb16bff08ecee299" target='_blank'>
              Resistance signatures manifested in early drug response across cancer types and species
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Arashdeep Singh, MM Gottesman, Sridhar Hannenhalli
          </td>
          <td>2025-08-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            Saeid Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Cancer genomic profiling has revolutionized cancer research and clinical practice by facilitating personalized medicine approaches. Next-generation sequencing technologies have played a pivotal role in expanding the possibilities of cancer genomics, providing a comprehensive view of the genomic landscape of tumors and identifying potential therapeutic targets. In Japan, the approval of several cancer-related gene panel tests has accelerated the implementation of cancer genomic medicine. However, there remains a need for evidence-based selection criteria for the different panel tests available. Among these tests, ACTOnco+® stands out as a comprehensive cancer genomic profiling tool. In this study, we compared 110 samples from 29 different tumor types using FoundationOne® CDx (324 genes for DNA analysis) and ACTOnco+® (440 genes for DNA and 31 genes for RNA analysis). Overall, the mutation profiles between the 2 assays exhibited 82.8% positive agreement in reported sequence alterations in clinically actionable genes, including single nucleotide variants and insertions–deletions. Copy number gains showed concordance of 76.9%, and copy number losses in 66.7%. In the case of KIAA1549-BRAF fusion-positive astrocytoma, the fusion event was not detected by DNA only test, but RNA sequencing identified the rearrangement. The patient exhibited clinical benefit from MEK inhibitor treatment. Tumor mutational burden and microsatellite instability (MSI) demonstrated high concordance across various cancer types. ACTOnco+® identified a total of 329 heterozygous deletions, which were subsequently validated for reliability using FISH analysis. Gene profiling with ACTOnco+® exhibited comparable results to FoundationOne® CDx, thereby contributing to future personalized medicine endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/584447fccf78026ffcd29a7adc016edfe5f11d6e" target='_blank'>
              Characterization of a novel comprehensive genomic profiling test with better detection of heterozygous deletions and RNA-based gene fusion analysis
              </a>
            </td>
          <td>
            Ryouta Kakuta, Akira Takemoto, Kousuke Tanimoto, Rika Noji, Ryo Kudo, Yoshihito Kano, Yayoi Ando, Tatsunori Simoi, Yusuke Kinugasa, Sadakatsu Ikeda
          </td>
          <td>2025-08-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da08fc2c51aefcac002abe7af73df18eb83fa23" target='_blank'>
              Integrative proteo-genomic profiling uncovers key biomarkers of lapatinib resistance in HER2-positive breast cancer.
              </a>
            </td>
          <td>
            J. Steggall, V. Rajeeve, N. Al-Subaie, A. Naeem, A. Ikram, A. Naeem, A. Hayat
          </td>
          <td>2025-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Single-cell transcriptomics has significantly advanced our ability to uncover the cellular heterogeneity of tumors. A key challenge in single-cell transcriptomics is identifying cancer cells and, in particular, distinguishing them from non-malignant cells of the same cell lineage. Focusing on features that can be measured by single-cell transcriptomics, this review explores the molecular aberrations of cancer cells and their observable readouts at the RNA level. Identification of bona fide cancer cells typically relies on three main features, alone or in combination: i) expression of cell-of-origin marker genes; ii) inter-patient tumor heterogeneity; iii) inferred copy-number alterations. Depending on the cancer type, however, alternative or additional features may be necessary for accurate classification, such as single-nucleotide mutations, gene fusions, increased cell proliferation, and altered activation of signaling pathways. We summarize computational approaches commonly applied in single-cell analysis of tumoral samples, as well as less explored features that may aid the identification of malignant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86abedac67c16f4a560937fab77dcbb2e53869a5" target='_blank'>
              Identification of malignant cells in single-cell transcriptomics data
              </a>
            </td>
          <td>
            M. Andreatta, J. Garnica, Santiago J. Carmona
          </td>
          <td>2025-08-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background/Objectives: Next-Generation Sequencing (NGS) has transformed cancer diagnostics and treatment by enabling comprehensive genomic profiling of tumors. This review aims to summarize the current applications of NGS in oncology, highlighting its role in early detection, precision therapy, and disease monitoring. Methods: We conducted a comprehensive review of the recent literature, focusing on the application of NGS in cancer care. Results: NGS enables high-resolution genomic profiling, identifying actionable mutations (e.g., EGFR, KRAS, and ALK) and immunotherapy biomarkers (e.g., PD-L1, TMB, and MSI), guiding personalized treatment selection and improving outcomes in advanced malignancies. Liquid biopsy enhances diagnostic accessibility and enables real-time monitoring of minimal residual disease and treatment resistance. Despite these advances, widespread clinical adoption remains constrained by technical limitations (e.g., coverage uniformity and sample quality), economic challenges (high costs and complex reimbursement), and interpretative issues, including the management of variants of uncertain significance (VUSs). Conclusions: NGS is central to precision oncology, enabling molecularly driven cancer care. Integration with artificial intelligence, single-cell sequencing, spatial transcriptomics, multi-omics, and nanotechnology promises to overcome current limitations, advancing personalized treatment strategies. Standardization of workflows, cost reduction, and improved bioinformatics expertise are critical for its full clinical integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29924fb52e872fdb917a78ad329fe64a89de4bc1" target='_blank'>
              Next-Generation Sequencing: A Review of Its Transformative Impact on Cancer Diagnosis, Treatment, and Resistance Management
              </a>
            </td>
          <td>
            Alexandru Isaic, Nadica Motofelea, Teodora Hoinoiu, A. Motofelea, Ioan Cristian Leancu, Emanuela Stan, Simona R. Gheorghe, A. Duțu, Andreea M. Crintea
          </td>
          <td>2025-09-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chemotherapy is a foundational element in cancer therapy; however, the development of resistance to antineoplastic agents presents a significant challenge to achieving enduring therapeutic triumphs. The acquisition of drug resistance by malignant cells is the result of a complex interplay between inherent and acquired mechanisms. This phenomenon leads to a reduction in pharmacological effectiveness and contributes to treatment failure. This review investigates the primary mechanisms contributing to chemoresistance in specific malignancies, namely lung, pancreatic, and thyroid cancers, along with non-Hodgkin lymphoma. The primary focus of this research is on genetic modifications, interactions within the tumor microenvironment, and cellular adaptations at the molecular level. The present study places particular emphasis on the contributions of drug efflux pumps, DNA repair pathways, epithelial-mesenchymal transition (EMT), microRNAs, and the suppression of apoptosis in mediating this resistance. In addition, we examine promising approaches to counteract drug resistance, encompassing multi-agent regimens, inhibitors targeting specific molecular pathways, and innovative therapeutic modalities. A comprehensive understanding of these fundamental mechanisms is imperative for the development of sophisticated therapeutic interventions that not only improve patient outcomes but also effectively address resistance. The resistance pathways and prospective translational approaches discussed are drawn from both preclinical and clinical investigations, providing a thorough perspective. Keywords: The following terms are relevant to the study: chemoresistance, tumor microenvironment, molecular pathways, targeted therapy, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec567f4501baf205432eef42dce8eb9f17d9a8c" target='_blank'>
              Mechanisms of Chemoresistance in Solid and Hematologic Malignancies: Challenges and Future Perspectives
              </a>
            </td>
          <td>
            Badiaa Batlamous, Boutaina Elgharbaoui, Imane Bensalim, Mohamed Khalis
          </td>
          <td>2025-09-24</td>
          <td>Middle Eastern Cancer and Oncology Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2 to 7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, Andrew B. Stergachis, Rachel J. O’Neill, K. Miga, Steven L Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="The molecular basis of therapy resistance in multiple myeloma (MM) remains poorly understood. Here, we performed single-cell RNA sequencing coupled with VDJ-targeted sequencing of highly purified primary MM cells from patient bone marrow. This approach uncovered cellular heterogeneity and phenotypic plasticity along the CD138 axis, accompanied by drastic epigenetic alterations. Notably, therapy-resistant subpopulations were identified within a minor fraction of CD138- MM cells, and were shown via CRISPR/Cas9 screening to be vulnerable to splicing pathway inhibition. Consistently, this fraction of CD138- MM cells showed increased differential splicing associated with overexpression of SR protein family splicing factors. Among these splicing factors, RNA-binding protein 39 (RBM39) was specifically overexpressed in therapy-resistant cells and involved in aberrant splicing. Both genetic and pharmacological RBM39 inhibition exhibited a significant selective lethal effect on therapy-resistant CD138- MM cells. Collectively, our findings identify distinct therapy-resistant MM subpopulations and highlight the splicing pathway as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dfb101a52f0fa47f40e1c355892c0f32082d1b" target='_blank'>
              Identification of a CD138-negative therapy-resistant subpopulation in multiple myeloma with vulnerability to splicing factor inhibition.
              </a>
            </td>
          <td>
            Takahiro Kamiya, Masahiko Ajiro, M. Oshima, S. Koide, Yaeko Nakajima-Takagi, Kazumasa Aoyama, Akiho Tsuchiya, Satoshi Kaito, Naoki Itokawa, Ryoji Ito, Kiyoshi Yamaguchi, Yoichi Furukawa, Bahityar Rahmutulla, Atsushi Kaneda, Takayuki Shimizu, N. Doki, Taku Kikuchi, Nobuhiro Tsukada, Masayuki Yamashita, Shinichiro Okamoto, Akihide Yoshimi, Keisuke Kataoka, Atsushi Iwama
          </td>
          <td>2025-09-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The human genome is partitioned into functional compartments that replicate at specific times during the S-phase. This temporal program, referred to as replication timing (RT), is co-regulated with the 3D genome organization, is cell type-specific, and changes during development in coordination with gene expression. Moreover, RT alterations are linked to abnormal gene expression, genome instability, and structural variation in multiple diseases, including cancer. However, mechanistic links between RT, large-scale 3D genome architecture, and transcriptional regulation remain poorly understood. A major limitation is that current approaches require the separate profiling of RT and transcriptomes from independent batches of samples, obscuring the complex co-regulation between the epigenome and transcriptome. Here, we developed PARTAGE, a multiomics approach that enables joint profiling of copy number variation (CNV), RT, and gene expression from the same sample, providing a more accurate integrative view of the complex relationships between RT and gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957eda64f84a18622679de092756bc343443ffa1" target='_blank'>
              PARTAGE: Parallel analysis of replication timing and gene expression
              </a>
            </td>
          <td>
            Lakshana Sruthi Sadu Murari, Quinn Dickinson, Silvia Meyer-Nava, J. Rivera-Mulia
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/739921fbe29bda5d659d1ced9f84aedc8ae3f8d2" target='_blank'>
              Next-generation oncology: integrative therapeutic frontiers at the crossroads of precision genomics, immuno-engineering, and tumor microenvironment modulation.
              </a>
            </td>
          <td>
            Ahmad M Alamri, Abdullah A Assiri, Bushra Khan, Najeeb Ullah Khan
          </td>
          <td>2025-09-20</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Multiple Myeloma (MM) is a malignant plasma cell dyscrasia that progresses through the consecutive asymptomatic, often undiagnosed, precancerous stages of Monoclonal Gammopathy of Undetermined Significance (MGUS) and Asymptomatic Multiple Myeloma (SMM). MM is characterized by low survival rates, severe complications and drug resistance; therefore, understanding the molecular mechanisms of progression is crucial. This study aims to detect genetic mutations, both germline and somatic, that contribute to disease progression and drive tumorigenesis at the final stage of MM, using samples from patients presenting MGUS or SMM, and newly diagnosed MM patients. Methods: Mutations were identified through a fully computational pipeline, implemented in a Linux and RStudio environment, applied to each patient sequence, obtained through single-cell RNA-sequencing (scRNA-seq), separately. Structural and functional mutation types were identified by stage, along with the affected genes. The analysis included quality control, removal of the Unique Molecular Identifiers (UMIs), trimming, genome mapping and result visualization. Results: The findings revealed frequent germline and somatic mutations, with distinct structural and functional patterns across disease stages. Mutations in key genes were identified, pointing to molecules that may play a central role in carcinogenesis and disease progression. Notable examples include the HLA-A, HLA-B and HLA-C genes, as well as the KIF, EP400 and KDM gene families, with the first four already confirmed. Comparative analysis between the stages highlighted molecular transition events from one stage to another. Emphasis was given to novel genes discovered in newly diagnosed MM patients, that might contribute to the tumorigenesis that takes place. Conclusions: This study contributes to the understanding of the genetic basis of plasma cell dyscrasias and the transition events between the stages, offering insights that could aid in early detection and diagnosis, guide the development of personalized therapeutic strategies, and improve the understanding of mechanisms responsible for resistance to existing therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db8c2ca600e5f8d213e747b7e35b473639690b95" target='_blank'>
              Genomic Insights into Tumorigenesis in Newly Diagnosed Multiple Myeloma
              </a>
            </td>
          <td>
            Marina Kyriakou, C. Papaloukas
          </td>
          <td>2025-08-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Through most of medical history, treatments for metastatic cancers were ineffective, and rapid patient death was inevitable. Over the past five decades, a worldwide drug development effort has introduced a remarkable range of new cancer treatment strategies and agents so that virtually all metastatic cancers have one or more effective therapeutic options to prolong life. Yet most metastatic cancers remain fatal, and increasingly, the proximate cause of death is evolution. Local or systemic therapies applied to large, heterogeneous cancer populations elicit complex short- and long-term adaptive responses. Cells already possessing the molecular machinery of resistance obtain a stepwise fitness benefit relative to treatment-sensitive cells, allowing increased proliferation. Cells, otherwise sensitive to the treatment, may survive when in epigenetic states resistant to the treatment-induced death pathway or microenvironmental conditions that reduce drug delivery/efficacy, followed by a transition to "hard-wired" resistance allowing proliferation. These dynamics, enabled by the vast information content of the human genome, can produce diverse adaptive strategies in response to virtually all available treatments. Thus, oncology is rapidly approaching an era in which patient death is caused not by the absence of effective therapies but rather by eco-evolutionary dynamics that defeat initially successful treatments. Emerging evidence suggests that explicit integration of evolutionary principles to control or eliminate resistant populations can improve outcomes. In this issue of Cancer Research, Hockings and colleagues present an important evolutionary strategy to delay or prevent the evolution of resistance in ovarian cancer, with broad potential application. See related article by Hockings et al., p. 3503.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa08f9c64d9497ce73be1df571e19d588bc63780" target='_blank'>
              The Approaching Darwinian Era in Oncology.
              </a>
            </td>
          <td>
            Robert A. Gatenby
          </td>
          <td>2025-09-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Current cancer therapies often fail due to tumor heterogeneity and rapid resistance evolution. A new evolutionary framework, ‘selection for function,’ proposes that tumor progression is driven by group phenotypic composition (GPC) and its interaction with the microenvironment, not by individual cell traits. This perspective opens new therapeutic avenues: targeting the tumor’s functional networks rather than individual cells. Real-time tracking of GPC changes could inform adaptive treatments, delaying progression and resistance. By integrating evolutionary and ecological principles with conventional therapies, this strategy aims to transform cancer from a fatal to a manageable chronic disease. Crucially, it does not necessarily require new drugs but offers a way to repurpose existing therapies to impair a tumor’s evolutionary potential. By steering tumor evolution toward less aggressive states, this approach could improve prognosis and long-term patient survival compared to current methods. We argue that leveraging GPC dynamics represents a critical, yet underexplored, opportunity in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53fd3b887f1a92efe08b517e2a13e55225513a1" target='_blank'>
              Leveraging selection for function in tumor evolution: System-level cancer therapies
              </a>
            </td>
          <td>
            F. Thomas, Jean-Pascal Capp, A. Dujon, A. Marusyk, K. Asselin, M. Campone, Pascal Pujol, Catherine Alix-Panabières, Benjamin Roche, B. Újvári, Robert Gatenby, A. Nedelcu
          </td>
          <td>2025-08-18</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ffde0bbfbd90bb0bf5e018161ce107f39f7740f" target='_blank'>
              Assessing the current molecular understanding of therapeutic targets in osteosarcoma.
              </a>
            </td>
          <td>
            Chao Zhang, Ji-Lan Yang
          </td>
          <td>2025-08-07</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The discovery of genomics has redefined oncology into precision rather than standardized treatment. But
therapeutic resistance of either kind, intrinsic or acquired, is a serious impediment to sustained success. The present review
outlines the domains on the crossroad between genomic intelligence and resistance evolution, where multi-omics profiling,
high-throughput sequencing, and AI-based analytics are explaining the challenging complexity of the tumor and predicting
resistance pathways. We review the contribution of clonal evolution, tumor plasticity and adaptive signaling in resistance to
therapy, and we promote moving towards active molecular monitoring rather than snapshot ting genome. To preempt and
target therapeutic escape we suggest a framework of adaptive precision oncology that comprehends real-time biomarkers,
liquid biopsy follow-up and resistance-predictive algorithms. New approaches like combination therapy, recalibration of
treatment with the help of AI, and longitudinal monitoring of genomes are mentioned. We also deal with moral and logistical
obstacles of adopting these strategies, and especially in low-resource environments. The new era of precision oncology We
now find ourselves in the next era of precision oncology that aims to personalization extend to real-time responsiveness, so
that cancer treatment can increasingly be more adaptable, more resilient and more durable, achieved through personalizing
the rapidly changing tumor biology with real-time dynamically changing genomic insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7508321bc4abb3899cb6fc37344230d9a86147e" target='_blank'>
              Genomic Intelligence and Resistance Evolution: Redefining Oncotherapeutic Strategies in Precision Oncology
              </a>
            </td>
          <td>
            Muchukota Sushma, Bharathi Bhogenahalli Venkatappa, Gowthami V., Nigel Viju Thomas, Souman Samanta
          </td>
          <td>2025-08-28</td>
          <td>International Journal of Innovative Science and Research Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chemotherapy response variability in cancer patients necessitates novel strategies targeting chemoresistant populations. While combinatorial regimens show promise through synergistic pharmacological interactions, traditional pathway enrichment methods relying on static gene sets fail to capture drug-induced dynamic transcriptional perturbations.To address this challenge, we developed the Pathway-Responsive Gene Sets (PRGS) framework to systematically identify chemoresistance-associated pathways and guide therapeutic intervention. Comparative evaluation of three computational strategies (GSEA-like method, Hypergeometric test-based method, Bates test-based method) revealed that the GSEA-like methodology exhibited superior performance, enabling precise identification of drug-induced pathway dysregulation.Key experimental findings demonstrated PRGS’s superiority over conventional Pathway Member Gene Sets (PMGS), exhibiting statistical independence (p < 0.0001) and enhanced detection of chemotherapy-driven pathway dysregulation. Application of PRGS to the GDSC dataset identified 8 resistance-associated pathways. Screening of agents targeting these pathways yielded candidates with predicted anti-resistance activity. An in vitro cellular experiment demonstrated that the bortezomib-bleomycin combination exhibited synergistic cytotoxicity (IDAcomboScore = 0.014) in T47D cells, highlighting the potential of PRGS-guided therapeutic strategies.This study establishes a PRGS-based methodological framework that integrates genomic perturbations with precision oncology, demonstrating its capacity to decode resistance mechanisms and guide therapeutic development through dynamic pathway analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/814bf51ab320b8d56ddbcfefe1cd9a9aa4f449e3" target='_blank'>
              Drug discovery for chemotherapeutic resistance based on pathway-responsive gene sets and its application in breast cancer
              </a>
            </td>
          <td>
            Dehua Feng, Jingwen Hao, Lingxu Li, Jian Chen, Xinying Liu, Ruijie Zhang, Huirui Han, Tianyi Li, Xuefeng Wang, Xia Li, Lei Yu, Bing Li, Jin Li, Limei Wang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) is a genetically and clinically heterogeneous malignancy marked by poor prognosis and limited therapeutic options, especially in older patients. While conventional treatments such as the “7 + 3” chemotherapy regimen and allogeneic stem cell transplantation remain standard care options, the advent of next-generation sequencing (NGS) has transformed our understanding of AML’s molecular complexity. Among the emerging hallmarks of AML, metabolic reprogramming has gained increasing attention for its role in supporting leukemic cell proliferation, survival, and therapy resistance. Distinct AML subtypes—shaped by specific genetic alterations, including FLT3, NPM1, and IDH mutations—exhibit unique metabolic phenotypes that reflect their underlying molecular landscapes. Notably, FLT3-ITD mutations are associated with enhanced reactive oxygen species (ROS) production and altered energy metabolism, contributing to disease aggressiveness and poor clinical outcomes. This review highlights the interplay between metabolic plasticity and genetic heterogeneity in AML, with a particular focus on FLT3-driven metabolic rewiring. We discuss recent insights into how these metabolic dependencies may be exploited therapeutically, offering a rationale for the development of metabolism-targeted strategies in the treatment of FLT3-mutated AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdd39e68513bba88417b983cb0b3114bb567964" target='_blank'>
              Metabolic Signature of FLT3-Mutated AML: Clinical and Therapeutic Implications
              </a>
            </td>
          <td>
            C. Banella, Gianfranco Catalano, Maura Calvani, Eleonora Candi, N. Noguera, S. Travaglini
          </td>
          <td>2025-09-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Although precision-targeted therapies and tyrosine kinase inhibitors (TKIs) have significantly improved outcomes in non-small-cell lung cancer (NSCLC), patients with EGFR-mutant NSCLC with concurrent TP53 mutations often develop drug resistance and experience poor clinical outcomes. This study aims to investigate the molecular mechanisms underlying this aggressive subtype using single-cell RNA sequencing. Methods Formalin-fixed paraffin-embedded (FFPE) tumor samples were obtained from 40 hospitalized NSCLC patients. Somatic mutation profiles were determined using a targeted 23-gene next-generation sequencing (NGS) panel. Four samples harboring concurrent EGFR and TP53 mutations were selected for single-cell transcriptomic profiling using the 10x Genomics platform. Results Two dominant malignant epithelial cell populations were identified: C1_EGFR+, associated with proliferation and invasion, and C2_STAT1+, linked to immunosuppression and drug resistance. These tumor subtypes cooperatively drive CD8+ T cell exhaustion through the MDK–(ITGA4+ITGB1), MIF–(CD74+CXCR4), and TGF-β signaling pathways. In addition, antigen-presenting cancer-associated fibroblasts (apCAFs) recruit regulatory T cells via the CCL5–CCR4 axis, collectively establishing an immune-excluded tumor microenvironment. Mechanistically, a STAT1/ETS1-centered transcriptional program regulates the expression of key immunosuppressive (e.g., MDK, MIF, TGFB1) and resistance-associated genes (e.g., ERBB2, JAK2). Conclusion These findings reveal a coordinated transcriptional network that promotes immune evasion and therapeutic resistance in EGFR/TP53 co-mutated NSCLC. Targeting the STAT1/ETS1 axis, in combination with EGFR-TKIs or immune checkpoint inhibitors, may provide a novel strategy to overcome resistance and improve patient outcomes. Further validation in larger patient cohorts and functional studies is warranted to confirm these observations and support clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dfb7de9c158ecd7f633c8ed6b345d4fa18555ab" target='_blank'>
              Dissecting the immune evasion and therapeutic resistance mechanisms in EGFR/TP53 co-mutated non-small cell lung cancer: implications for targeted and immunotherapy strategies
              </a>
            </td>
          <td>
            Haiyan Shi, Kun Xu, Xueying Kong, Weining Xie, Yingying Chen, Ding He, Zufu Cheng, Xianshan Huo, Ke Gao, Mingshuang Song, Ning Tian
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hematologic malignancies, including leukemia, lymphoma, and multiple myeloma, represent a diverse group of blood cancers driven by complex oncogenic pathways and genetic aberrations. Despite advances in treatment, resistance and relapse remain significant clinical challenges. This review presents an integrative approach to blood cancer therapy, highlighting the convergence of precision medicine, CRISPR-based gene editing, immunotherapy (including CAR-T cells and immune checkpoint inhibitors), and targeted modulation of amino acid metabolism. We investigate the molecular pathways driving tumour development and resistance, highlighting how multi-omic profiling facilitates personalised therapy approaches. Recent breakthroughs in reprogramming immune responses and editing driver mutations have revolutionized therapeutic paradigms. Additionally, the metabolic dependencies of cancer cells, particularly in amino acid biosynthesis and catabolism, offer novel vulnerabilities. This meticulous synthesis outlines a framework for translational techniques that integrate molecular targeting with immunological and metabolic regulation to improve treatment effectiveness and surmount therapeutic resistance in haematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602acd2f85cdfeb967bf671ca25b0e041a71b90f" target='_blank'>
              Deciphering and targeting oncogenic pathways through integrated approaches and amino acid metabolism in hematologic malignancies
              </a>
            </td>
          <td>
            Farhan Ikhtiar, Adil Jamal, Syed M Safeer Mehdi Bokhari
          </td>
          <td>2025-09-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A complex and gradual process, the epithelial–mesenchymal transition (EMT) occurs both during embryonic development and tumor progression. Cells undergo a transition from an epithelial to a mesenchymal state throughout this process. More and more evidence points to EMT as a cause of increased metastatic spread of prostate cancer (PCa), along with stemness enhancement and therapy resistance. Here, we used bioinformatic methods to analyze gene expression microarray data, single-cell RNA sequencing, oncogenes, and tumor suppressor genes (TSGs) in order to reconstruct the network of differentially expressed genes (DEGs) involved in the epithelial–mesenchymal transition with PCa. No prior study has documented this sort of analysis. We next validated our results using data from the Cancer Genome Atlas (TCGA), which included microarray and single-cell RNA sequencing. Potentially useful in PCa diagnosis and treatment are extracellular matrix in epithelial–mesenchymal transition genes, including ITGBL1, DSC3, COL4A6, ANGPT1, ARMCX1, MICAL2, and EPHA5. In this study, we aimed to shed light on the molecular characteristics and pathways of DEGs in PCa, as well as to identify possible biomarkers that are important in the development and advancement of this cancer. These insights have important implications for understanding prostate cancer progression and for the development of therapeutic strategies targeting ECM-mediated pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15775ad90d8cab3fed2c521129b0f7f8c9fd8d19" target='_blank'>
              Analysis of Microarray and Single-Cell RNA-Seq Finds Gene Co-Expression and Tumor Environment Associated with Extracellular Matrix in Epithelial–Mesenchymal Transition in Prostate Cancer
              </a>
            </td>
          <td>
            Ali Shakeri Abroudi, Mahtab Mashhouri Moghaddam, Danial Hashemi Karoii, Melika Djamali, Hossein Azizi, Thomas Skutella
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background/Objectives: Myelodysplastic syndrome (MDS) is a heterogeneous clonal hematopoietic disorder characterized by ineffective hematopoiesis and leukemic transformation risk. Current therapies show limited efficacy, with ~50% of patients failing hypomethylating agents. This review aims to synthesize recent discoveries through an integrated network model and examine translation into precision therapeutic approaches. Methods: We reviewed breakthrough discoveries from the past three years, analyzing single-cell multi-omics technologies, epitranscriptomics, stem cell architecture analysis, and precision medicine approaches. We examined cell-type-specific splicing aberrations, distinct stem cell architectures, epitranscriptomic modifications, and microenvironmental alterations in MDS pathogenesis. Results: Four interconnected mechanisms drive MDS: genetic alterations (splicing factor mutations), aberrant stem cell architecture (CMP-pattern vs. GMP-pattern), epitranscriptomic dysregulation involving pseudouridine-modified tRNA-derived fragments, and microenvironmental changes. Splicing aberrations show cell-type specificity, with SF3B1 mutations preferentially affecting erythroid lineages. Stem cell architectures predict therapeutic responses, with CMP-pattern MDS achieving superior venetoclax response rates (>70%) versus GMP-pattern MDS (<30%). Epitranscriptomic alterations provide independent prognostic information, while microenvironmental changes mediate treatment resistance. Conclusions: These advances represent a paradigm shift toward personalized MDS medicine, moving from single-biomarker to comprehensive molecular profiling guiding multi-target strategies. While challenges remain in standardizing molecular profiling and developing clinical decision algorithms, this systems-level understanding provides a foundation for precision oncology implementation and overcoming current therapeutic limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e501707362379bd5b49ca68342c5a79da7724f2" target='_blank'>
              A Genetically-Informed Network Model of Myelodysplastic Syndrome: From Splicing Aberrations to Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Sanghyeon Yu, Junghyun Kim, Man S. Kim
          </td>
          <td>2025-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Phenotypic plasticity is a key mechanism of metastatic progression and cancer therapy resistance. This hallmark of human malignancies is enabled by highly conserved epigenetic mechanisms that control gene expression. Functional alterations in DNA methylation and histone post-translational modifications have been extensively described as drivers of metastatic dissemination and therapy resistance. Pharmacological inhibitors of epigenetic enzymes can revert these alterations, thereby stopping cancer progression and counteracting the emergence of resistant clones. Despite promising pre-clinical evidence, the clinical implementation of epigenetic therapies in solid cancers has led to disappointing results. Several factors can explain these challenges, including the lack of rational combinations. Notably, response to epigenetic treatments can be heterogeneous and short-lived. A liquid biopsy technology that allows the measure of specific epigenetic alterations enables patient selection and therapy monitoring, leading to the development of precision epigenetic therapies. In this review, we discuss the state of the art of this emerging treatment modality, and we identify key challenges that need to be overcome to reach the full potential of this new therapeutic concept.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876ad36ab7595e359c53da7e87448ed401afa714" target='_blank'>
              Precision epigenetic therapies in oncology
              </a>
            </td>
          <td>
            Arundhati Chaudhary, Kayleigh J A Orchard, F. Salani, Theodora Partsou, Mark Eccleston, Guido Bocci, A. Italiano, Francesco Crea
          </td>
          <td>2025-09-19</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="KRAS mutations are among the most prevalent oncogenic drivers in non-small cell lung cancer (NSCLC), yet the mechanisms of therapeutic resistance to KRAS inhibitors in these cancers remains poorly understood. Here, we deploy high-throughput CRISPR base editing screens to systematically map resistance mutations to three mechanistically distinct KRAS-targeted therapies, including KRAS-G12C(OFF) inhibitor (adagrasib), RAS(ON) G12C-selective tri-complex inhibitor (RMC-4998), and RAS(ON) multi-selective tri-complex inhibitor (RMC-7977). Using both a saturation Kras tiling approach and cancer-associated mutation library, we identify common and compound-selective second-site resistance mutations in Kras, as well as gain-of-function and loss-of-function variants across cancer-associated genes that rewire signaling networks in a context-dependent manner. Notably, we identify a recurrent missense mutation in capicua (Cic), that promotes resistance to RMC-7977 in vitro and in vivo. Moreover, we show that targeting NFκB signaling in CIC-mutant cells can resensitize them to RAS pathway inhibition and overcome resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597a0301e9cbbccb635fb6b4893ce93089ff3713" target='_blank'>
              Genetic mechanisms of resistance to targeted KRAS inhibition
              </a>
            </td>
          <td>
            Bianca J. Diaz, Max Kops, Sara Bernardo, Henri Schmidt, Elizabeth Grankowsky, Adrian Vega, Chen Zhang, Matthew Bott, Maria Skamagki, Aidan C A Tomlinson, Nicole A Vita, Alyna Katti, Mark Labrecque, Ida Aronchik, Mallika Singh, Lukas E. Dow
          </td>
          <td>2025-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chemotherapy is the most widely used and effective treatment method in today's cancer treatment. However, the rise of cancer drug resistance has limited chemotherapy effectiveness. The molecular mechanisms and related genes by which many cancer cells acquire resistance to chemotherapy drugs remain to be identified. Recent advancements of CRISPR-Cas9 genome-editing techniques have drawn rigorous exploration of precision oncology applications. CRISPR-Cas9 technology can knock out, inhibit or activate the expression of specific genes. By observing the phenotype of gene-edited cells, drug-resistant genes in carcinoma cells can be identified. This paper will decipher the molecular basis of CRISPR-Cas9 technology in targeted gene modification and give examples to illustrate its application in the screening of drug resistance genes in three types of human cancers. Additionally, the application limitations and emerging horizons of CRISPR-Cas9 gene modification technique in strategies for overcoming anticancer drug resistance are discussed. The continuous development and optimization of this technology are expected to provide strong support for breaking through the bottlenecks in tumor treatment and pushing precision cancer medicine to new heights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0492def30740e765a8ca58409631a4cc538a87f" target='_blank'>
              CRISPR-Cas9 Gene-Editing Technology for the Screening of Drug Resistance Genes in Tumor Cells
              </a>
            </td>
          <td>
            Zisheng Zeng
          </td>
          <td>2025-09-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74a710dbabf1414500dcedb29ef09db85188c8e" target='_blank'>
              Gene editing in cancer therapy: overcoming drug resistance and enhancing precision medicine.
              </a>
            </td>
          <td>
            Hyeonjeong Park, Suyeun Yu, Taeyoung Koo
          </td>
          <td>2025-09-18</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Multiple myeloma (MM) is a malignancy marked by uncontrolled plasma cell proliferation, immune evasion, and drug resistance. Despite advances in treatment, the disease remains incurable due to relapses and drug resistance. This study aims to investigate the molecular and cellular interactions within the myeloma microenvironment using single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and pathway analysis to uncover therapeutic targets.


METHODS
We used scRNA-seq to analyze tumor, immune, and stromal cell interactions in MM. Data were processed using Seurat for clustering, dimensionality reduction, and cell-type annotation. Gene Set Variation Analysis (GSVA) and cell-cell interaction analysis were performed to identify signaling pathways involved in disease progression. Two-sample MR was applied to explore causal relationships between genetic variants and gene expression. Finally, molecular docking simulations were used to identify potential small molecule modulators of key proteins involved in MM.


RESULTS
We identified significant immune-related signaling pathways, including PI3K-AKT-mTOR, WNT-β-catenin, and TGF-β, upregulated in immune cells within the MM microenvironment. Genes such as HLA-C, CTSS, and LRRFIP1 showed positive causal relationships with MM, while SHISA5 and ISG15 exhibited protective roles. Cell communication analysis revealed key ligand-receptor interactions between immune and tumor cells. Molecular docking identified promising small molecules like actein and aflatoxin B1 targeting ISG15 and TAGLN2.


CONCLUSIONS
This study reveals key genetic drivers and immune modulation mechanisms in MM. Targeting immune-related pathways, such as PI3K-AKT-mTOR and WNT-β-catenin, and small molecules targeting ISG15 and TAGLN2 could offer new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ddcfc1e5a8f4f32f9fd598c516ebb1c5e5f270" target='_blank'>
              Single-cell RNA sequencing reveals immune regulatory mechanisms and molecular therapeutic strategies in the microenvironment of multiple myeloma.
              </a>
            </td>
          <td>
            Qiang Su, Kehan Long, Mozili Adu, Meijun Jiang, Qiaochu Li, Xufeng Wan, Jian Cao, Yan Yue, Shuoyuan Li, Zhendong Ying, Qibin Liu, Chaoqun You, Zhuang Zhang, Duan Wang
          </td>
          <td>2025-09-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51250a57c51180b7973c2f5e6b3bbb56fc3acbc6" target='_blank'>
              Personalizing cancer therapy: the role of pharmacogenetics in overcoming drug resistance and toxicity.
              </a>
            </td>
          <td>
            Hossein Mardani -Nafchi, Hormoz Solatzadeh, Elnaz Hajimaghsoudi, Emad Babakhanzadeh
          </td>
          <td>2025-08-02</td>
          <td>Molecular biology reports</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer drug resistance remains a major barrier to durable treatment success, often leading to relapse despite advances in precision oncology. While combination therapies are being increasingly investigated, such as chemotherapy with small molecule inhibitors, predicting drug response and identifying rational drug combinations based on resistance mechanisms remain major challenges. Therefore, a proteome-wide, single-gene overexpression screening platform is essential for guiding rational therapy selection. Here, we present BOGO (Bxb1-landing pad human ORFeome-integrated system for a proteome-wide Gene Overexpression), a robust, scalable, and reproducible screening platform that enables single-copy, site-specific integration and overexpression of ~19,000 human ORFs across cancer cell models. Using BOGO, we identified drug-specific response drivers for 16 chemotherapeutic agents and integrated clinical datasets to uncover proliferation and resistance-associated genes with prognostic potential. Drug response similarity networks revealed both shared and unique mechanisms, highlighting key pathways such as autophagy, apoptosis, and Wnt signaling, and notable resistance-associated genes including BCL2, POLD2, and TRADD. In particular, we proposed a synergistic combination of the BCL2 family inhibitor ABT-263 (Navitoclax®) and the DNA analog TAS-102 (Lonsurf®), which revealed that lysosomal modulation is a key mechanism driving DNA analog resistance. This combination therapy selectively enhanced cytotoxicity in colorectal and pancreatic cancer cells in vitro, and demonstrated therapeutic benefit in vivo in both cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models. Together, these findings establish BOGO as a powerful gene overexpression perturbation platform for systematically identifying chemoresistance and chemosensitization drivers, and for discovering rational combination therapies. Its scalability and reproducibility position BOGO as a broadly applicable tool for functional genomics and therapeutic discovery beyond cancer resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb42d8c647bdd3aef172a4ebd8dbd580fe7fe538" target='_blank'>
              BOGO: A Proteome-Wide Gene Overexpression Platform for Discovering Rational Cancer Combination Therapies
              </a>
            </td>
          <td>
            Kyeong Beom Jo, M. Alruwaili, Da-Eun Kim, Yongjun Koh, Hyeyeon Kim, Kwontae You, Ji-Sun Kim, Saba Sane, Yanqi Guo, Jacob P. Wright, H. Lim, Maricris N. Naranjo, Atina G. Coté, Frederick P. Roth, David E. Hill, Jung-Hyun Choi, Hunsang Lee, Kenneth A. Matreyek, Kyle K.-H. Farh, Jong-Eun Park, Hyunkyung Kim, Andrei V. Bakin, Dae-Kyum Kim
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The addition of CDK4/6 inhibitors to endocrine therapy has significantly improved outcomes in HR+/HER2-breast cancer. However, variable patient responses and acquired resistance remain a clinical challenge. We therefore defined the comprehensive molecular response to palbociclib, the most clinically used CDK4/6 inhibitor. Global analyses of gene expression, protein abundance, splicing, and chromatin accessibility revealed broad patterns and specific changes that result from CDK4/6-inhibition in breast cancer cells. We uncovered unexpected feedback between CDK4/6 and estrogen-response signaling, which has clear clinical implications. We also revealed a widespread alternative splicing program that partially overlapped with genes whose expression is regulated, and which is expected to impact protein function. These molecular changes nominated combination therapies that interfere with the activation of CDKs or ERα. Accordingly, co-targeting CDK7, which regulates CDK2, CDK4/6 and ERα, additively impacted cell fitness. Collectively, these data reveal a complex, multi-tiered response to CDK4/6 inhibition, with implications for therapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ceb1151f3964ee516b8b4bec2c0e19c51711b54" target='_blank'>
              Charting the Multi-level Molecular Response to Palbociclib in ER-Positive Breast Cancer
              </a>
            </td>
          <td>
            Archishma Kavalipati, Amy Aponte, Michael E Sullivan, Sarah L Whittington, José C Martínez, Grant A. Goda, Maria M. Aleman, Michael J Emanuele, Daniel Dominguez
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Melanoma, the deadliest human skin cancer, frequently harbors activating BRAF mutations, with V600E being the most prevalent. These alterations drive constitutive activation of the MAPK pathway, promoting uncontrolled cell proliferation, survival, and dissemination. The advent of BRAFi and MEKi has significantly improved outcomes in BRAF V600-mutant melanoma. However, therapeutic resistance remains a major clinical barrier. Methods: This review integrates recent findings from preclinical and clinical studies to delineate resistance mechanisms to BRAF-targeted therapy. It categorizes resistance into primary (intrinsic), adaptive, and acquired forms, and analyzes their molecular underpinnings, including genetic and epigenetic alterations, pathway reactivation, and microenvironmental interactions. Results: Primary resistance is linked to pre-existing genetic and epigenetic changes that activate alternative signaling pathways, such as PI3K-AKT. Adaptive and acquired resistance includes secondary BRAF mutations, pathway redundancy, phenotype switching, and immune and stromal interactions. High-throughput sequencing has revealed novel mutations, including NRAS, NF1, and PTEN alterations, that contribute to resistance. Discussion: Understanding the multifaceted nature of resistance is critical to improving outcomes in advanced melanoma. This review highlights emerging strategies to overcome resistance, including combinatorial therapies, metabolic targeting, and biomarker-driven approaches, aiming to inform future therapeutic development and precision oncology strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fe52d09f6a6cd013c45f83f652a44de19330f7" target='_blank'>
              Molecular Basis of BRAF Inhibitor Resistance in Melanoma: A Systematic Review
              </a>
            </td>
          <td>
            Ilaria Cosci, V. Salizzato, P. Del Fiore, J. Pigozzo, Valentina Guarneri, Simone Mocellin, A. Ferlin, Sara Mathlouthi, L. Piccin, Mariangela Garofalo
          </td>
          <td>2025-08-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a9aba01e0cb2e8c3213b471e7d8f34414b59cc" target='_blank'>
              Functional phenotyping of genomic variants using joint multiomic single-cell DNA-RNA sequencing.
              </a>
            </td>
          <td>
            Dominik Lindenhofer, Julia R. Bauman, John A. Hawkins, Donnacha Fitzgerald, Umut Yildiz, Haeyeon Jung, Anastasiia Korosteleva, Mikael Marttinen, M. Kueblbeck, Judith B. Zaugg, K. Noh, Sascha Dietrich, W. Huber, O. Stegle, Lars M. Steinmetz
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a highly heterogeneous and aggressive hematologic malignancy characterized by clonal proliferation of myeloid precursors. Despite significant advancements in genomic profiling and targeted therapies, patient outcomes remain suboptimal due to disease complexity, resistance mechanisms, and high relapse rates. The integration of multi-omics approaches—spanning genomics, epigenomics, transcriptomics, proteomics, and metabolomics—has revolutionized AML research, offering a comprehensive understanding of leukemogenesis, tumor heterogeneity, and therapeutic vulnerabilities. Recent studies leveraging high-throughput sequencing, mass spectrometry, and advanced computational tools have uncovered novel biomarkers, clonal evolution dynamics, and microenvironmental interactions that drive AML progression and resistance. For instance, single-cell multi-omics has revealed chemotherapy-resistant leukemic stem cell populations, while proteogenomic analyses have identified actionable targets such as MCL1 and metabolic dependencies like OXPHOS. Clinically, integrated omics platforms are refining risk stratification, minimal residual disease (MRD) monitoring, and personalized therapy selection. However, challenges such as data integration complexity, cost barriers, and ethical considerations remain. This review highlights the transformative potential of multi-omics in AML, emphasizing recent advancements in technology, biomarker discovery, and therapeutic innovation. By bridging the gap between molecular insights and clinical practice, multi-omics integration promises to redefine AML management, paving the way for precision oncology and improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5a212633a58eb354f5e13985e9ea262565cab98" target='_blank'>
              Integrating multi-omics approaches in acute myeloid leukemia (AML): Advancements and clinical implications
              </a>
            </td>
          <td>
            Hamed Soleimani Samarkhazan
          </td>
          <td>2025-08-31</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95f957c61adab36c7dec0eefa4bae427dbcc6b3" target='_blank'>
              CRISPR-Edited Cell Lines: A New Era in Functional Oncology Research.
              </a>
            </td>
          <td>
            Amita Joshi Rana, M.D.S.A.D.I.Q.U.E. Hussain, Ali Hanbashi, Faroq Kamli, Gyas Khan, Marwa Qadri, Saeed A Al-Qahtani, Mohammad Gayoor Khan, Sushil S. Burle, Vikas Jakhmola, Gaurav Gupta
          </td>
          <td>2025-08-13</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Microsatellite-stable metastatic colorectal cancer (MSS mCRC) is currently treated with chemotherapy and targeted agents based on RAS and BRAF mutational status. Although these therapies offer initial benefit, most patients rapidly develop resistance, with fewer than 20% remaining progression-free at two years. This review aims to synthesize emerging evidence on the metabolic mechanisms driving treatment resistance in MSS mCRC, with a particular focus on the immune-metabolic signature (IMMETCOLS) classification. We conducted a comprehensive review of preclinical models, transcriptomic datasets, and clinical trial results addressing metabolic adaptations to chemotherapy and targeted therapies in MSS mCRC. The IMMETCOLS framework defines three metabolic subtypes—IMC1, IMC2, and IMC3—each associated with distinct resistance mechanisms. IMC1 exhibits glycolysis and transforming growth factor-β (TGF-β)-dependent signaling enriched in inflammatory fibroblasts, conferring resistance to chemotherapy. IMC2 relies on oxidative phosphorylation and glutamine metabolism, supporting antioxidant defenses and resistance to both cytotoxic agents and anti-EGFR therapies. IMC3 demonstrates lactate-fueled respiration and pentose phosphate pathway activation, contributing to redox balance, DNA repair, and resistance to targeted therapies such as anti-BRAF or KRAS inhibitors. All subtypes display metabolic plasticity under therapeutic pressure. Emerging clinical data support tailoring targeted therapy combinations based on IMMETCOLS subtype, particularly in BRAF- and HER2-positive populations. Understanding subtype-specific metabolic rewiring in MSS mCRC offers novel opportunities to overcome drug resistance. Targeting the metabolic vulnerabilities defined by the IMMETCOLS signature may improve response durability and inform precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662ed873ba87c81960ffe96e54fa075a4697d04f" target='_blank'>
              Metabolic plasticity drives specific mechanisms of chemotherapy and targeted therapy resistance in metastatic colorectal cancer
              </a>
            </td>
          <td>
            Mariam Rojas, Malena Manzi, S. Madurga, Fernando Enrique García Velásquez, Maira Alejandra Romero, Silvia Marín, Marta Cascante, Joan Maurel
          </td>
          <td>2025-09-23</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a cornerstone in the treatment of early-stage high-risk breast cancers (BC), particularly in triple-negative, HER2-positive, and selected hormone receptor-positive subtypes. However, its effectiveness is frequently hindered by intrinsic or acquired resistance, resulting in a significant residual cancer burden (RCB) in more than half of patients. Despite extensive genomic profiling, reliable predictive biomarkers for treatment response remain limited, impeding the development of personalized therapeutic strategies. Emerging evidence highlights tumor metabolic reprogramming as a key non-genetic mechanism contributing to NAC resistance. In this review, we critically examine current advances in metabolic imaging and metabolomics as tools to predict NAC response in BC. We also discuss the role of the tumor microenvironment (TME), including hypoxia and acidosis, in shaping metabolic plasticity and fostering treatment resistance. Furthermore, we explore novel therapeutic strategies targeting metabolic pathways, ranging from enzyme inhibition to dietary interventions, and the use of advanced preclinical models. Together, these insights offer a metabolic framework for overcoming NAC resistance and advancing precision oncology in early-stage BC. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03500-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1de78332c9b34ec581f5a78aef2b8dfd8a219776" target='_blank'>
              Resistance to neoadjuvant chemotherapy in breast cancers: a metabolic perspective
              </a>
            </td>
          <td>
            Manon Desgres, Melis Poyraz, Buse Sari, Francois P. Duhoux, Cédric van Marcke, C. Corbet
          </td>
          <td>2025-08-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is driven by a complex spectrum of somatic mutations and structural variants that contribute to tumor heterogeneity and therapy resistance. In this study, we performed a comparative analysis of short-read Illumina and long-read Nanopore sequencing technologies across multiple CRC sample groups, encompassing diverse tissue morphologies. Our evaluation included general base-level metrics—such as nucleotide ratios, sequence match rates, and coverage—as well as variant calling performance, including variant allele frequency (VAF) distributions and pathogenic mutation detection rates. Focusing on clinically relevant genes (KRAS, BRAF, TP53, APC, PIK3CA, and others), we characterized platform-specific detection profiles and completed the ground truth validation of somatic KRAS and BRAF mutations. Structural variant (SV) analysis revealed Nanopore’s enhanced ability to resolve large and complex rearrangements, with consistently high precision across SV types, though recall varied by variant class and size. To enable direct comparison with the Illumina exome panel, we applied an exonic position reference file. To assess the impact of depth and PCR amplification, we completed an additional high-coverage Nanopore sequencing run. This analysis confirmed that PCR-free protocols preserve methylation signals more accurately, reinforcing Nanopore’s utility for integrated genomic and epigenomic profiling. Together, these findings underscore the complementary strengths of short- and long-read sequencing platforms in high-resolution cancer genomics, and we highlight the importance of coverage normalization, epigenetic fidelity, and rigorous benchmarking in variant discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4037869e56706b3aeec547e05839c0679ce3d95b" target='_blank'>
              Methodological Comparison of Short-Read and Long-Read Sequencing Methods on Colorectal Cancer Samples
              </a>
            </td>
          <td>
            Nikolett Szakállas, Alexandra Kalmár, K. Rada, M. Kucarov, Tamás Richárd Linkner, B. Barták, István Takács, Béla Molnár
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Kidney cancer, particularly clear cell renal cell carcinoma (ccRCC), has emerged as a paradigm for cancer metabolic reprogramming, exhibiting distinctive alterations that drive tumor progression and therapeutic resistance. This comprehensive review synthesizes current knowledge of the molecular mechanisms underlying metabolic dysregulation in kidney cancer, with emphasis on the central role of VHL/HIF pathway activation and its downstream consequences on glycolysis, lipid metabolism, and glutamine utilization. We systematically analyze how pseudohypoxia-driven metabolic rewiring not only supports tumor bioenergetics and biosynthesis but also shapes an immunosuppressive microenvironment through metabolite-mediated crosstalk with stromal and immune cells. The review highlights groundbreaking therapeutic advances, including FDA-approved HIF-2α inhibitors and emerging agents targeting glycolytic enzymes, glutaminase, and lipid metabolism, while addressing the challenges of metabolic plasticity and acquired resistance. Special attention is given to innovative combination strategies that pair metabolic modulators with immunotherapy or tyrosine kinase inhibitors, supported by preclinical rationale and clinical trial data. We further discuss cutting-edge technologies transforming the field - from hyperpolarized MRI for real-time metabolic imaging to AI-driven analysis of multi-omics datasets for patient stratification. By integrating fundamental science with translational applications, this review provides a framework for understanding kidney cancer as a metabolic disease and outlines future directions for targeted therapies, biomarker development, and personalized treatment approaches. The synthesis of these insights offers both a conceptual foundation and practical guidance for researchers and clinicians working to exploit metabolic vulnerabilities in kidney cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db2d25efcb582bfcc2f7cef446966ad2850d13c0" target='_blank'>
              Metabolic Reprogramming in Kidney Cancer: Implications for Therapy
              </a>
            </td>
          <td>
            Elena Tena Edo
          </td>
          <td>2025-08-07</td>
          <td>annals of urologic oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

Cancer remains one of the most complex diseases globally, driven by genetic and epigenetic
alterations that disrupt normal cellular processes. This complexity is further amplified by
mechanisms such as uncontrolled cell proliferation, inhibition of apoptosis, angiogenesis, immune
evasion, and metabolic reprogramming, all of which contribute to tumor growth and metastasis.
While advancements in diagnostic technologies, including next-generation sequencing and liquid
biopsies, have facilitated early detection and the development of personalized therapeutic strategies,
traditional treatments such as chemotherapy, radiotherapy, and immunotherapy continue to
face significant limitations, including drug resistance, adverse side effects, and tumor heterogeneity.
Recent studies have emphasized the potential of bacteriophages in cancer treatment. As viruses
that specifically target bacteria, phages offer unique advantages due to their genetic modifiability,
roles in immunotherapy, and capabilities in targeted drug delivery. They can reshape the tumor microenvironment,
enhance immune responses, and deliver therapeutic agents directly to cancer
cells. Moreover, phages hold promise in addressing antibiotic resistance in cancer patients and improving
the efficacy of traditional therapies through combination strategies. However, challenges
remain regarding the optimization of phage stability, bioavailability, and tumor-specific targeting.
Research and clinical trials on the therapeutic application of phages are critical to unlocking their
full potential as innovative approaches to overcome the limitations of current cancer treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f176f476b0e17d22a339146ab7610aed3ab80193" target='_blank'>
              Bacteriophages in Cancer Treatment
              </a>
            </td>
          <td>
            Mustafa Ustundag, Berrin Ustundag
          </td>
          <td>2025-08-22</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genetic mechanisms regulating gene expression encompass complex processes such as transcription, translation, epigenetic modifications, and interactions of regulatory elements. These mechanisms play a crucial role in shaping phenotypic diversity in humans. High-throughput technologies, such as expression microarrays and next-generation sequencing (NGS), have enabled precise analysis of transcripts for thousands of genes genome-wide. These methods have enabled researchers to measure gene expression levels in various tissues and cells and to gain deeper insights into previously inaccessible biological processes. Numerous studies show that gene expression varies significantly among individuals. However, there are also notable differences between populations from different continental groups, driven by genetic, epigenetic, environmental factors, and natural selection. Furthermore, disease states represent an important factor influencing gene activity, as they can significantly alter the transcriptomic profiles of individual cells. In this context, comparative population genetic studies help uncover the molecular mechanisms underlying complex phenotypic traits and identify population-specific features of transcriptomic profiles in both health and disease. However, despite significant progress in this field, many aspects remain underexplored. Specifically, the distribution of gene expression variability among populations, the degree of research coverage for specific ethnic groups, the spectrum of biological materials used, and the contribution of population affiliation to observed differences in gene expression during pathological conditions require further investigation. This review presents an overview of contemporary research focused on analyzing variability in expression profiles across different human populations. It summarizes findings from individual studies, outlines the advantages and limitations of the methods employed, highlights key research directions in population transcriptomics, and discusses potential practical applications of the data obtained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e020705578f481ead7a3caa26fc484de5385fb09" target='_blank'>
              Population transcriptomics: a novel tool for studying genetic diversity in human populations under normal and pathological conditions
              </a>
            </td>
          <td>
            A. A. Babovskaya, E. A. Trifonova, V. A. Stepanov
          </td>
          <td>2025-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Psoriasis is a chronic inflammatory skin disease involving dysregulated immune responses and complex genetic factors. This study combines single-cell RNA sequencing (scRNA-seq), gene expression profiling, and genetic analysis to explore cellular and molecular contributors to psoriasis. Methods Single-cell RNA-seq data (n = 3 psoriasis, n = 2 control; GSE228421) were used for cell-type annotation and functional characterization. T cell subsets were analyzed for differentiation trajectories and cell-cell communication. Differentially expressed genes in mucosal-associated invariant T (MAIT) cells were evaluated by enrichment analysis. Candidate gene causality was tested via eQTL-based Mendelian randomization (MR) and supported by bulk RNA-seq validation. Results MAIT cells were enriched in psoriatic lesions and exhibited strong intercellular interactions. Functional analyses revealed activation of IL6-JAK-STAT3 signaling, TNF-NFκB pathway, and glycolysis in MAIT cells. MR identified RPS20 as a protective factor (OR = 0.5994, p = 0.011) and PFN1 as a potential risk gene (OR = 1.7229, p = 0.037), with PFN1 highly expressed in MAIT cells. Colocalization analysis showed no significant genetic overlap between PFN1 expression and psoriasis risk. Metabolic profiling revealed differential pathway involvement in PFN1+ and PFN1− MAIT cells. Conclusion Our integrative analysis highlights MAIT cells and PFN1 as likely contributors to psoriasis pathogenesis. These findings offer insights into immune and metabolic alterations, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48055881b868991adfc41d78b968062c6c8aae6e" target='_blank'>
              MAIT Cells with High PFN1 Expression Mediate Immune Activation and Metabolic Reprogramming in Psoriasis
              </a>
            </td>
          <td>
            Man-Ning Wu, , Xiang-Nan Zhou, Sangwon Hong, Lei Wang, Yan-Ping Bai
          </td>
          <td>2025-09-09</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer research has traditionally focused on identifying driver genes, those with mutations that initiate tumorigenesis. The Cancer Driver Gene (CDG) paradigm, further supported by the observation of oncogene addiction in tumors, has successfully guided the development of targeted therapies. However, the limitations of this driver-centric view, highlighted by the broad emergence of frequent therapeutic resistance, the presence of driver mutations in healthy tissues or individuals, and the lack of identifiable drivers in many tumors, call for a shift in perspective and clinical practice. The latest network controllability perspective on cancer cells introduced the concept of Cancer Keeper Genes (CKGs) and a CKG-based paradigm for cancer therapeutics. The new concept encompasses the concept of non-oncogene addiction, emphasizing reliance on non-mutated pathways crucial for maintaining oncogenic cellular states. Here, we explore the transition towards a system-level understanding of cancer based on the CKG paradigm, emphasizing the essential role of genes required for tumor maintenance, irrespective of their initiating function or mutational capacity. We discuss clinical implications of this paradigm shift, highlighting the progress made so far and potential of targeting non-driver CKGs, genes involved in processes like DNA damage response, proteostasis, and metabolism, as a promising strategy to overcome therapeutic challenges and achieve more durable cancer control. Targeting these maintenance vulnerabilities represents a critical evolution in precision oncology, moving towards therapies designed to dismantle the networks sustaining malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e80123ffc1524bf35e125c1b2645e60b9bc7703" target='_blank'>
              From Cancer Drivers to Cancer Keepers: Paradigm Shift and Clinical Implications
              </a>
            </td>
          <td>
            Xizhe Zhang, Weixiong Zhang
          </td>
          <td>2025-08-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Targeted therapies have significantly improved the prognosis and productivity of non-small-cell lung cancer (NSCLC) patients carrying driver mutations, but drug resistance is inevitable. Histological transformation, an important resistance mechanism, is often manifested as transformation into small-cell lung cancer, large-cell neuroendocrine carcinoma, squamous cell carcinoma, and sarcomatoid carcinoma. The mechanisms involved are complex, including RB1/TP53 inactivation, epithelial–mesenchymal transition, and microenvironmental changes. Post-transformation tumors are often more aggressive and drug-resistant, with limited therapeutic options and poorer prognosis. In this paper, we systematically review the histological transformation types, molecular mechanisms, and therapeutic strategies of NSCLC after resistance to targeted therapy, with the aim of providing a reference for clinical decision-making and promoting the development of individualized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71911ca0beebb701fd3deee5a9b4ea23c86ca3e7" target='_blank'>
              Histologic transformation after targeted therapy resistance in driver gene-positive NSCLC: mechanisms and therapeutic challenges
              </a>
            </td>
          <td>
            Xinyue Li, K. Ding, Dujiang Liu, Yuxia Wang, Yanjun Xu
          </td>
          <td>2025-08-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Recent advancements in transcriptomic analysis, combined with single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, have deepened our understanding of the tumor microenvironment and cellular heterogeneity, laying the groundwork for personalized therapies. The aim of this research is to explore the heterogeneity of tumor cells in colorectal cancer (CRC) and evaluate their prognostic value in different therapeutic contexts, emphasizing the impact of tumor cell heterogeneity on disease progression. Methods scRNA-seq alongside spatial transcriptomics was employed to analyze the heterogeneity of tumor cells in CRC, the spatial distribution of tumor cells, and their interactions with the microenvironment. Results We identified nine distinct tumor cell subtypes, with MLXIPL + neoplasm prevalent in advanced CRC, while ADH1C + and MUC2 + neoplasms were more common in early-stage CRC. MLXIPL + neoplasm was significantly associated with chemotherapy and targeted therapy efficacy. Analysis of spatial transcriptomics indicated that MLXIPL + neoplasm is located in the core area of the tumor cells. We developed a 13-gene prognostic signature (PS) using machine learning algorithm (StepCox backward), which predicts the prognosis of CRC patients. Furthermore, the patients with low PS score demonstrated higher immune cell infiltration and immune regulatory factors, suggesting enhanced immune surveillance and treatment response. Conclusions The findings highlight the critical role of tumor cell heterogeneity in CRC progression and treatment response, suggesting the need for personalized therapeutic strategies targeting different subpopulations. The constructed PS demonstrates significant clinical application value in predicting patient prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00805-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065bcb7e9315d28e942e5a0a0b33a35e0d6f402" target='_blank'>
              Spatial transcriptomics and scRNA-seq: decoding tumor complexity and constructing prognostic models in colorectal cancer
              </a>
            </td>
          <td>
            Wei Song, Yatao Wang, Min Zhou, Fengqin Guo, Yanliang Liu
          </td>
          <td>2025-08-13</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that their continued expression is required for maintaining their leukemic phenotype and that fusion genes therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5469845f04d542fe9db7d0ec4a2180ccb6b7c697" target='_blank'>
              Fusion Gene Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. D. Mata Casimiro, Anita van Oort, M. Du Plessis, Luca van den Brink, Minoo Ashtiani, C. Zwaan, A. Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, Josef Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-09-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) encompasses tumors arising in the colon (CC) and rectum (RC), often treated as a single disease despite emerging evidence of biological divergence. Understanding the molecular differences between CC and RC is critical for improving diagnosis, prognosis, and therapeutic strategies. Methods We performed an integrated genomic and transcriptomic analysis of CC and RC data from The Cancer Genome Atlas (TCGA) to investigate their degree of similarity and observed that these tumors present distinct molecular profiles, which suggest an evolution through divergent pathways. Comparative analyses included copy number alterations (CNAs), somatic mutations, driver gene prediction, differential gene expression, pathway enrichment, and survival analysis. Results Chromosomal analyses revealed that 43% of focal and 77% of large-scale CNAs were specific of CC, while 10.5% and 57% were specific of RC with 8% of mutant genes unique to CC and 0.18% to RC. CC and RC presented distinct profiles of gene mutations, with CC showing significantly higher tumor mutational burden (0.51 muts/Mb vs 0.28 muts/Mb in RC). Distinct mutational signatures were identified, with CC characterized by a higher frequency of PIK3CA, BRAF, and DNAH1 mutations, while RC showed enrichment for TP53 and NRAS mutations. Importantly, analysis of predicted non-canonical driver genes identified ACVR1B, LTBP4, SETD1A as CC-specific drivers and C4BPA, EHD1 as RC-specific drivers, underscoring divergent oncogenic mechanisms. However, the most substantial divergence was observed in transcriptomic profiling, with 56% and 33% of DEGs (in CC and RC, respectively) that were tumor-type specific. Notably, RC tumors segregated into two distinct transcriptional subtypes (Cluster 1 and Cluster 2), with Cluster 1 showed a more heterogeneous Consensus Molecular Subtypes (CMS) distribution, while Cluster 2 enriched in CMS4 (mesenchymal) and CMS3 (metabolic) consensus molecular subtypes. Accordingly, Gene Set Enrichment Analysis revealed CC-specific upregulation of Wnt, MYC, and mTOR signaling pathways, and RC-specific enrichment of GPCR and neuronal development pathways. On the other hand, pseudogene expression was significantly higher in CC, suggesting differential mechanisms of transcriptional dysregulation. Finally, we identified an RC-specific multigene survival signature as a prognostic model involving upregulation of C2CD4B, HSPD1P1, LINC01356, CBX3P9, GATA2-AS1 and downregulation of ATP5F1EP2, HSP90AB3P and SNRPFP1. Conclusions Collectively, our findings provide robust molecular evidence that CC and RC follow divergent oncogenic pathways, emphasizing the need for site-specific biomarker development and therapeutic targeting in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06908-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab7228806c5ae698ac668178d1fa7e354b1b537" target='_blank'>
              Distinctive chromosomal, mutational and transcriptional profiling in colon versus rectal cancers
              </a>
            </td>
          <td>
            Maria Teresa De Angelis, Antonia Rizzuto, Angela Amaddeo, Carlo Sagnelli, Niccolò Vono, Michela Reda, Valentina Lise, Luca Parrillo, Carmela de Marco, Donatella Malanga, Gianluca Santamaria, Giuseppe Viglietto
          </td>
          <td>2025-08-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Multiple myeloma (MM) is highly heterogeneous, with relapse occurring in the majority of cases, and recent advancements in single-cell RNA sequencing (scRNA-seq), sc-metabolism profiling, and bulk RNA-seq have facilitated the identification of cell subpopulations and metabolic reprogramming at the single-cell level, uncovering novel molecular mechanisms. This study aims to establish a multi-omics atlas of MM, characterizing the cell subpopulations and signaling pathways that drive immune evasion and disease progression. Additionally, sc-metabolic profiling identifies reprogramming patterns and informs therapeutic screening. We integrated scRNA-seq and bulk RNA-seq data using R to analyze immune and non-immune cell features and pathways in MM. Metabolic reprogramming was assessed via sc-metabolic profiling, and drug candidates were screened through multi-omics integration, with efficacy evaluated in vitro using CCK-8 assays, flow cytometry, Western blotting, and CalcuSyn software. Novel MM subpopulations were identified, including myeloma-activated hematopoietic stem cells and ISG15+ B cells, which correlated with survival and were validated by multiplex immunofluorescence. IFN-γ is primarily secreted by effector memory CD8+T cells, and IFN-α is primarily secreted by non-classical monocytes, driving an IFN-γ/α-B2M feedback loop. Multi-omics identified four drug candidates, each demonstrating anti-tumor effects against myeloma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed7155f8bb3f2fadcad9b5654646e878c15255d" target='_blank'>
              Multi-omics reveals immune features in immune and non-immune cells, an IFN-γ/IFN-α-B2M positive feedback loop, and targeted metabolic therapy in multiple myeloma
              </a>
            </td>
          <td>
            Chen Li, Yaping Liao, Ling Xu, Yan Chen
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related morbidity and mortality, with African-ancestry populations experiencing higher incidence and poorer outcomes. Although socioeconomic factors contribute, growing evidence suggests biological differences also play a role. However, ancestry-associated molecular features of CRC remain insufficiently defined, largely due to the underrepresentation of non-European populations in genomic studies. In this study, transcriptomic profiles of African-Black and European-White CRC patients from The Cancer Genome Atlas (TCGA) were compared to identify ancestry-associated molecular programs. Differential expression analysis identified dysregulated genes, and pathway enrichment revealed distinct biological processes. White patients exhibited coordinated immune–metabolic pathways, whereas Black patients showed fragmented modules centered on CRYBB2 and MAP9. Immune profiling using single-sample gene set enrichment analysis (ssGSEA) further demonstrated higher enrichment of interferon alpha response, allograft rejection, and complement activation in White patients, complemented by increased expression of immune checkpoint genes. Bootstrap resampling confirmed the robustness of these findings despite sample imbalance. These results provide novel evidence of ancestry-linked transcriptomic differences in CRC and underscore the importance of including diverse populations in cancer genomics to develop equitable biomarkers and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c54690c9d78f17a34e09a1ce349638b6ab3f2c1" target='_blank'>
              Divergent Molecular Networks Drive Ancestry-Related Colorectal Cancer Disparities
              </a>
            </td>
          <td>
            Salma Fareed
          </td>
          <td>2025-09-15</td>
          <td>Scholarly Review Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a highly heterogeneous and aggressive hematologic malignancy, with the activated B-cell-like (ABC) subtype displaying particularly poor prognosis due to inherent treatment resistance and elevated recurrence rates. Despite advances in targeted therapies and immunotherapies, a significant proportion of patients experience relapse or refractory disease, highlighting the urgent need for novel biomarkers and innovative therapeutic strategies to improve clinical outcomes. Methods A multi-dimensional analysis of SH3BP5 expression was performed across DLBCL subtypes, integrating transcriptomic, proteomic, and clinical datasets to assess its correlation with immune infiltration, tumor metabolism, and patient prognosis. Single-cell RNA sequencing data were employed to examine the tumor microenvironment (TME) with higher resolution. Further analysis of the association between SH3BP5 and immune checkpoint gene expression was conducted to explore its potential role in immunotherapy response. Functional in vitro assays were carried out to assess the impact of SH3BP5 knockdown on DLBCL cell proliferation and apoptosis. Results The analysis revealed that SH3BP5 is preferentially overexpressed in the ABC subtype of DLBCL across multiple datasets and validated cohorts, and its high expression is significantly associated with poor overall survival. Single-cell transcriptomic profiling demonstrated that SH3BP5 is mainly expressed in malignant B cells and inversely correlated with immune cell infiltration, particularly CD8 + T cells. Mechanistically, pathway enrichment and metabolic assays indicated that SH3BP5 is linked to mitochondrial metabolic reprogramming, promoting oxidative phosphorylation (OXPHOS) and potentially contributing to reduced responsiveness to immune checkpoint inhibitors (ICIs). Functional studies showed that SH3BP5 knockdown significantly suppressed DLBCL cell proliferation, induced apoptosis, and reduced tumor cell viability in vitro. Conclusion This study suggests that SH3BP5 may serve as a prognostic biomarker and a potential therapeutic target in DLBCL, particularly within the ABC subtype. By delineating its associations with immune evasion and metabolic reprogramming, these findings provide a mechanistic basis for further exploration of SH3BP5-targeted interventions to help overcome therapy resistance. Future studies in larger clinical cohorts and functional models are warranted to validate these results and assess the potential of integrating SH3BP5 expression profiling into precision medicine strategies for DLBCL. Trial registration The study was registered in the Chinese Clinical Trial Registry (ChiCTR2200060430; http://www.chictr.org.cn/) on June 1, 2022. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06951-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27222190fd1301e27cfd9869b08b07fc58aa185" target='_blank'>
              SH3BP5-driven metabolic-immune crosstalk in DLBCL: a prognostic biomarker and therapeutic target for reshaping immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Tong Wu, Yi Yang, Yuan Zong, Jiawen Zhao, Xiaoyu Zhao, Lei Li, Yiming Gao, Ning Li, Liting Jiang, Yinyin Xie
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6adfd64895599aa6bae6a1f8896e93b0d56bc5ac" target='_blank'>
              ADAR1-mediated RNA editing in breast cancer: molecular mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Si-Yu Chen, Si-Ying Chen, Shu Yang, Yi Li, Shengyong Yang
          </td>
          <td>2025-08-09</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: The number one cause of cancer-related deaths across the globe is metastatic spread of malignant tumors. To determine therapeutic targets, it is important to understand cellular and genetic changes that facilitate invasion, dissemination, and colonization. Although this field can be improved through genomics, the interactions between tumor-intrinsic alterations and the microenvironment are not fully outlined, and thus, current metastasis-preventive approaches are limited. 
Objective: The purpose of the study was to examine cellular changes and genetic mutations related to tumor metastasis, to examine individual patient features, and to examine clinical correlations that point to possible areas of therapeutic intervention. 
Study Design: A retrospective observational study. 
Place and Duration of study: Pakistan Institute of Medical Sciences, PIMS, Islamabad, Pakistan. Jan 2024 to December 2024 
Methods: Review of 100 patients with histologically proven malignant tumors was a retrospective observational study. Institutional records were used to collect demographic and clinical data. Recurrent mutations in TP53, KRAS, and PIK3CA of tumor tissue samples and epithelial-mesenchymal transition markers were assessed by immunohistochemistry. Statistical analyses comprised descriptive statistics, calculation of mean ages and standard deviation and comparison of groups using Student t-test with a significance level set to p < 0.05. 
Results: 100 patients (52 males and 48 females) were used as the cohort. The mean age was 59.4 years +- 11.2 SD. Genomic findings identified TP53 mutations in 45 percent of patients, KRAS mutations in 28 percent and PIK3CA mutations in 18 percent. Loss of E-cadherin expression was also found in 36 percent of samples and was associated with greater invasiveness. Analysis of the metastatic burden revealed that patients who had both TP53 and KRAS mutations had a much greater metastatic burden than patients without (p = 0.018). Clinical associations suggested that genetic changes and epithelial-mesenchymal transition signatures operated as a combined effect on progression, which underscored a suggestion that molecular heterogeneity is the basis of metastatic potential. 
Conclusion: We have shown that genetic mutations, together with cellular reprogramming (epithelial-mesenchymal transition) contribute to metastasis. Combined TP53 and KRAS mutations and their relationship with metastatic progression are correlated, which is important to identify potential risk stratification biomarkers. The combination of molecular and cellular analysis provides informative tools to develop therapies that will help intercept metastatic progression, which ultimately will raise the prognosis and survival rates in patients with progressive malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a13e9e21cb70713f15ecb05360a37152fe398f" target='_blank'>
              Analyzing The Cellular and Genetic Alterations That Contribute to The Metastatic Spread of Malignant Tumors
              </a>
            </td>
          <td>
            Rakhshanda Shamim, Zaid Waqar, Adeel Haidar, Ammar Niazi, Muhammad Mustafa Sadiq, Bushra Anam Ali
          </td>
          <td>2025-09-23</td>
          <td>Journal of Neonatal Surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background A key to understanding cancer is to determine the impact on the cellular pathways caused by the repertoire of DNA changes accrued in a cancer cell. Exploring the interactions between genomic aberrations and the expressed transcriptome can not only improve our understanding of the disease but also identify potential therapeutic approaches. Results Using random forest models, we successfully identified transcriptional patterns associated with the loss of wild-type activity in cancer-related genes across various tumour types. While genes like TP53 and CDKN2A exhibited unique pan-cancer transcriptional patterns, others like ATRX, BRAF, and NRAS showed tumour-type-specific expression patterns. We also observed that genes like AR and ERBB4 did not lead to strong detectable patterns in the transcriptome when disrupted. Our investigation has also led to the identification of genes highly associated with transcriptional patterns. For instance, DRG2 emerged as the top contributor in classification of ATRX alterations in lower-grade gliomas and was significantly downregulated in ATRX mutant tumours. Additionally, transcriptional features important in classification of PTEN aberrations, such as CDCA8, AURKA, and CDC20, were found to be closely related to PTEN function. Conclusions Our findings demonstrate the utility of machine learning in interpretation of cancer genomic data and provide new avenues for development of targeted therapies tailored to individual patients with cancer. Our analysis on the transcriptome revealed genes with expression levels strongly correlated with alterations in cancer-related genes. Additionally, we identified AURKA inhibitors as potential therapeutic option for tumours with alterations in tumour suppressors like FBXW7 or NSD1. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02339-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e1f512af72f3e98e7eac17d6531fcc9f244b2e0" target='_blank'>
              Transcriptional patterns of cancer-related genes in primary and metastatic tumours revealed by machine learning
              </a>
            </td>
          <td>
            Faeze Keshavarz-Rahaghi, E. Pleasance, Steven J M Jones
          </td>
          <td>2025-08-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most common malignancies worldwide. Its complex molecular mechanisms make traditional treatments ineffective in some patients. In recent years, advances in genetic engineering technologies have opened up new possibilities for precision medicine in BC. Researchers have attempted to intervene in cancer cell proliferation and induce apoptosis through various genetic strategies. Preclinical studies are increasing, and therapeutic targets and delivery technologies are expanding. However, current research has largely focused on specific cases or single technologies, lacking a systematic integration and analysis of various genetic strategies from a mechanistic perspective. This paper reviews three major genetic engineering strategies for BC based on their therapeutic mechanisms: RNA interference, gene compensatory expression, and gene repair and deletion. Representative studies, key technologies, challenges, and development trends are analyzed for each. Furthermore, this paper summarizes common challenges across these different mechanisms regarding delivery efficiency, stability, and targeting, and proposes potential directions for future synergistic strategies. This paper helps clarify current research approaches in BC gene therapy from a mechanistic perspective, providing a reference for the design of subsequent cross-mechanism integration and precision intervention strategies. However, issues such as mechanism overlap and diversification of delivery platforms still need further in-depth exploration. Future research can focus more on multi-mechanism combination strategies, individualized treatment pathways, and feasibility assessment of clinical transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38bb75ae7dd66792d49d2225441735e8c43eeb18" target='_blank'>
              Applications of Three Gene Engineering Mechanisms in Breast Cancer Therapy
              </a>
            </td>
          <td>
            Yifan Tian
          </td>
          <td>2025-09-03</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481b8053e41e58296473fbe96b5f5b8ab91cf715" target='_blank'>
              Reversible therapeutic resistance in EGFR mutant lung cancer caused by RB1 loss induced lineage plasticity.
              </a>
            </td>
          <td>
            Letian Zhang, M. Chinnam, Bradley Balk, Xiaojing Zhang, Yanqing Wang, Tatiana Shaurova, Shamshad Alam, Saraswati Pokharel, Prashant Singh, Justine Jacobi, Jianming Wang, Eduardo Cortes Gomez, Michael Habitzruther, Mukund Seshadri, B. Atanassov, Wei Wu, E. Benevolenskaya, C. Blakely, Pamela Hershberger, Mark Long, T. Bivona, David W. Goodrich
          </td>
          <td>2025-08-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background Thyroid cancer pathogenesis involves complex interactions between genetic predisposition and alterations in the tumor microenvironment. The causal relationships between inflammatory gene variants and thyroid cancer risk remain poorly understood, as does the cellular heterogeneity within the tumor ecosystem. This study aimed to investigate the causal associations between inflammatory protein genes and thyroid cancer risk, and to characterize the cellular composition and differentiation trajectories within the thyroid cancer microenvironment. Methods We employed a two-pronged approach combining Mendelian randomization (MR) analysis and single-cell RNA sequencing (scRNA-seq). MR analyses were conducted using genetic variants associated with the expression of inflammatory proteins (4EBP1_EIF4EBP1, ADA_ADA, ARTN_ARTN, AXIN1_AXIN1, and Beta-NGF_NGF) as instrumental variables to assess their causal effects on thyroid cancer risk. Multiple MR methods (MR Egger, weighted median, inverse variance weighted, simple mode, and weighted mode) were used to enhance robustness. For the cellular characterization, scRNA-seq was performed on thyroid cancer samples, followed by dimensionality reduction, clustering analysis, cell type annotation, and pseudotime trajectory inference. Results MR analyses revealed a significant positive causal association between AXIN1_AXIN1 expression and thyroid cancer risk (weighted median: OR = 1.396, p < 0.05; inverse variance weighted: OR = 1.291, p < 0.05), while ADA_ADA showed protective effects (simple mode: OR = 0.731, p < 0.05). The scRNA-seq analysis identified six major cell populations within the thyroid cancer microenvironment: epithelial cells, T cells, natural killer cells, fibroblasts, stromal cells, and macrophages. Pseudotime analysis revealed distinct differentiation trajectories with natural killer cells and macrophages appearing in early pseudotime, while epithelial cells and fibroblasts demonstrated multiple developmental states. Gene expression profiling identified four distinct cellular states with unique molecular signatures, including immune/inflammatory, stromal, and vascular components. Conclusion Our findings suggest that inflammatory protein genes, particularly AXIN1, have causal effects on thyroid cancer risk, providing potential targets for risk prediction and intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbfdc7a38a023cb6812db2d719e9e3e9bf3c6626" target='_blank'>
              The role of cellular heterogeneity in thyroid cancer: a multi-modal analysis approach with Mendelian randomization
              </a>
            </td>
          <td>
            Yichun Qian, Wei Chen, Xinyuan Chen, Shuai Cheng, Fangzhou Liu
          </td>
          <td>2025-08-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cutaneous squamous cell carcinoma (CSCC) is a prevalent skin cancer with aggressive progression that poses significant challenges, especially in metastatic cases. Single-cell DNA sequencing (scDNA-seq) has become an advanced technology for elucidating tumor heterogeneity and clonal evolution. However, comprehensive scDNA-seq studies and tailored mutation panels for CSCC are lacking. Methods We analyzed the genomic landscape of Chinese CSCC patients via a Multi-Patient-Targeted (MPT) scDNA-seq approach. This method combined bulk exome sequencing with Tapestri scDNA-seq. Mutations identified through bulk sequencing were used to design a targeted panel for scDNA-seq. Comparative analysis was conducted to explore the associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis was performed to understand the evolutionary trajectories of the tumors. Results Bulk sequencing revealed a diverse spectrum of somatic mutations in CSCC tumors, with missense mutations being predominant. The top tumor mutations, such as those in NOTCH1, TP53, NOTCH2, TTN, MUC16, RYR2, PRUNE2, DMD, HRAS, and CDKN2A, presented similar frequencies to those reported in studies in Korean and Caucasian populations. However, the mutation frequencies of HRAS, TTN, MUC16 and MUC4 were significantly different from the Korean and Caucasian populations. Comparative analysis revealed associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis via scDNA-seq revealed distinct evolutionary trajectories and their potential correlation with tumor development and patient prognosis. Furthermore, scDNA-seq identified two low-frequency mutation clones, NLRP5 and HMMR, which play important roles in the clonal evolution of CSCC. Conclusions Our study introduced a novel MPT sequencing approach for CSCC, providing insights into tumor heterogeneity and clonal evolution. We identified novel mutations and their potential associations with patient survival and tumor characteristics. Overall, our study layed the groundwork for personalized treatment strategies and provides a reference for future applications of the MPT panel in CSCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14585-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667c56c3222da6ac371627cb03ea1bf3e1f96087" target='_blank'>
              Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing
              </a>
            </td>
          <td>
            Wen Chen, Jiawei Xu, Chengdong Yu, Meng Zhou, Yong Ai, Wenbing Rao, Qingchuan Wang, Siyi Xu, Lei Tang, Si Gong, Jun Rao
          </td>
          <td>2025-08-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 In contrast to prevailing wisdom that cancer is a genetic disease that evolves via Darwinian selection, it is increasingly recognized that in addition to genetic mutations, intratumoral heterogeneity also results through non-genetic mechanisms. Furthermore, it is now evident that the malignant phenotype results from complex interactions between genetic and non-genetic mechanisms. These interactions contribute to functional changes across multiple spatiotemporal scales, from molecular to cellular to tissue level, creating a heterogeneous cancer cell population. Cancer Systems Biology is a comprehensive, up-to-date treatise that addresses these issues. The main intent of this compendium is to encourage cross-pollination of ideas from experts in multiple disciplines including basic and translational science researchers, systems/mathematical biologists, and biophysicists and clinicians interested in decoding a systems-level emergent view of cancer so that a deeper understanding of disease progression can be gained and more effective treatment strategies can be designed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03788cb0b91aa174c4a7535dcca74621b417df9" target='_blank'>
              Cancer Systems Biology
              </a>
            </td>
          <td>
            Ravi Salgia, M. Jolly, Prakash Kulkarni, Govindan Rangarajanm
          </td>
          <td>2025-09-01</td>
          <td>None</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Background: Based on our previous study, we have identified ZNF180, a zinc finger protein, as a pro-tumorigenic regulator in primary melanoma and a marker for poor prognosis. Herein, we report that ZNF180-regulated pathway, hence ZNF180-regulome, underlies resistance towards immune checkpoint inhibitions (ICIs). Methods: To investigate regulatory roles of ZNF180 to confer these immune suppressive phenotypes, we performed ZNF180 knock-down in melanoma cells in vitro with different genetic backgrounds, namely A375 (BRAF-mutant) and SKMEL147 (NRAS-mutant) cells, and performed RNA- and ATAC-sequencing. We performed integrative analysis of RNA- and ATAC-sequencing data with publicly available sequencing data from ICI-treated cohorts to construct comprehensive model of ZNF180-regulome and its impacts on immune microenvironment. Further, we performed ZNF180 silencing in immune competent Yumm1.7 murine model to confirm the changes in immune microenvironments. Results: ZNF180-regulome was predictive of ICI responses in independent bulk sequencing cohorts, and ZNF180+ tumors persisted after the therapy with immune-suppressive features such as MHC-I loss and CD155 expressions, the primary ligand to TIGIT inhibitory receptor. Further, ZNF180 silencing revealed its regulations on AP-1 transcription factors to drive melanoma reprogramming towards de-differentiated MITFlowAXLhigh cells, an established melanoma subtypes associated with recurrence and ICI resistance. In tandem, we observed that ZNF180+ tumor neighborhood significantly excluded with CD4 T-cells in metastatic tumor, and its silencing in immune competent murine model increased CD4 helper T-cell infiltrations with significant tumor regression in vivo. Conclusion: Collectively, these results indicate ZNF180 is a tumor intrinsic regulator of melanoma plasticity to drive de-differentiated phenotypes with immune-suppressive features including loss of immunogenicity, T-cell inhibitory signals through TIGIT/CD155 checkpoint and exclusion of CD4 helper T-cells. As ZNF180-regulome manifests in non-metastatic melanoma in contrast to the current focus of standard-of-care ICI on the metastatic disease, these results establish ZNF180-regulome as a biomarker and novel therapeutic avenue for early-stage, non-metastatic melanoma to intervene ICI resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31baf3f08584db6bc1b15c0af412ba0fb9282ea1" target='_blank'>
              ZNF180 modulates tumor intrinsic immunotherapy resistance in melanoma through driving plasticity
              </a>
            </td>
          <td>
            Won-Min Song, Sree Karani Kondapuram, Xianxiao Zhou, Shu-hsia Chen, Praveen Agrawal
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background/Aim Cell lines serve as valuable in vitro models to study altered cellular signaling pathways, to identify mutations in key oncogenic genes, and to test potential antitumor drugs. The Jurkat cell line, for example, has provided important information about various signaling pathways in lymphoblastic leukemia, establishing most of what is currently known about T-cell receptor (TCR) signaling. However, many aspects of the genome modification of this cell line have not yet been analyzed. To identify genes of potential biological and clinical relevance in acute T-lymphoblastic leukemia (T-ALL), we performed an array comparative genomic hybridization (aCGH) approach on the widely used Jurkat cell line and examined the association of the detected copy number alterations (CNAs) with cancer hallmarks and T-ALL pathogenesis. Materials and Methods Cells were harvested by using trypsin/EDTA from culture flasks to extract genomic DNA. aCGH experiments were performed on an Agilent microarray platform using the SurePrint G3 Cancer CGH + SNP Microarray 4×180 K. Functional enrichment analysis of all CNAs was performed with the R package g:Profiler2. The association of these alterations with key cancer hallmarks was analyzed using the Cancer Hallmarks web-tool. Results Our analysis revealed several novel CNAs, including losses at 5p15.2, 6q27, 10q22.2, 14q11.2, 18q11.2-q12.1, and Xp22.33, as well as gains at 2p11.2, 7p21.2, 7q21.2 and 18p11.32. Genes within these regions were associated with important oncogenic pathways, including ‘sustained proliferative signaling’, ‘tumor suppressor evasion’, and ‘angiogenesis promotion’. Conclusion These findings suggest that Jurkat cells may serve as a valuable model for identifying new targets for cancer research. Further studies are required to confirm the phenotypic implications of these variants, which may open new avenues for exploring the functional impact of these alterations and their potential role in the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da88c0ba8ab646537d572f0801ee5d18aa14a5e" target='_blank'>
              High-Resolution aCGH Analysis of the Jurkat Cell Line: Copy Number Alterations and Their Association With Cancer Hallmarks
              </a>
            </td>
          <td>
            Layssa Mota DA COSTA, Wallax Augusto SILVA FERREIRA, C. B. Machado, Carolina Koury NASSAR AMORIM, Ana Virgínia VAN DEN BERG, M. O. de Moraes Filho, M. E. D. de Moraes, Lucas Eduardo Botelho de Souza, A. Khayat, E. H. C. de Oliveira, C. A. Moreira-Nunes
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0ce2d3aba4ba6a29f1b308846500c4a3d7ec7" target='_blank'>
              Beyond inversions and deletions: the evolutionary and functional insights from translocations, fissions, and fusions in animal genomes.
              </a>
            </td>
          <td>
            Célian Diblasi, M. Saitou
          </td>
          <td>2025-08-01</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Cancer can recur when a subset of tumor cells, termed persister cells, survive therapy and re-enter the cell cycle. The cellular lineages that give rise to persister cells and the mechanisms that confer the persister state remain poorly understood. Through single-cell multiomic profiling (snRNA-seq and snATAC-seq) on a cohort of (1) non-malignant fallopian tube, (2) treatment-naïve, and (3) neoadjuvant chemotherapy (NACT)-treated high-grade serous ovarian cancer (HGSOC) patient samples, we identified an epigenetic signature that defines the chemotherapy-tolerant persister state. The changes in chromatin accessibility characterizing the signature were identified in residual NACT tumors but are also present in treatment-naive samples from patients who later developed resistance. Furthermore, this epigenetic signature independently predicted chemotherapy response in patient-derived xenograft models of HGSOC and in a separate patient cohort. Cells enriched in the persister state arose from multiple lineages and displayed activation of oncogenic pathways, including altered stress responses, epithelial to mesenchymal transition, and changes to the cell cycle promoting quiescence. Finally, we identified a subset of genes that are epigenetically primed for expression before treatment and are upregulated after treatment. These findings suggest that an intrinsic epigenetic program primes tumor cells towards chemotherapy tolerance and reveal new vulnerabilities that could be exploited to delay or prevent cancer recurrence.



 Mihai G. Dumbrava, Wazim M. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. John Weroha, Alexandre Gaspar-Maia. Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a450af46a8e3f0b0c1a2127ce5282e5d4428e5a8" target='_blank'>
              Abstract PR008: Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells
              </a>
            </td>
          <td>
            Mihai G. Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Pancreatic cancer is marked by a poor prognosis and an exceptionally high mortality rate, with its aggressive nature contributing to its classification as a highly malignant disease. For effective therapeutic strategies, the development of sophisticated and regulated DNA manipulation methods is essential. Originally part of the prokaryotic immune system, CRISPR/Cas has emerged as a pivotal genome-editing tool with promising applications in pancreatic cancer research and therapy. This gene editing method is known for simplicity, rapid advancement, and superior precision compared to earlier techniques. Its adaptability allows precise gene editing for therapeutic purposes, including oncogene silencing and correction of pathogenic mutations. Additionally, CRISPR-driven gene editing has facilitated the development of pancreatic cancer models, which serve as valuable platforms for drug discovery and personalized treatment strategies, offering deeper insights into the genetic landscape of pancreatic tumors. This article provides an overview of the current applications of CRISPR technology in gene therapy and cancer research, particularly in the context of pancreatic cancer, and lays the foundation for future studies. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ab39b4071496f7653becd6d74d5e06cbbcdd5a1" target='_blank'>
              CRISPR/Cas technologies in pancreatic cancer research and therapeutics: recent advances and future outlook
              </a>
            </td>
          <td>
            Hanieh Noormohamadi, Hamed Soleimani Samarkhazan, Maryam Kargar, F. Maroufi, Nazli Servatian, Fatemeh Davami
          </td>
          <td>2025-08-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are cells shed from tumors into the bloodstream, providing a unique source of information on tumor biology. CTCs can be collected through liquid biopsy, offering a noninvasive option to track cancer progression and treatment responses. Whereas ctDNA gives valuable insights into genomic alterations, CTCs allow for the direct analysis of cellular protein and RNA expression, essential for identifying targetable markers and resistance mechanisms. Resistance to cancer therapies is often due to genetic and phenotypic adaptations within tumor cells. CTC analysis can help track these adaptations, providing real-time insights into evolving resistance mechanisms and suggesting alternative therapies.


SIGNIFICANCE
Enumeration and molecular characterization of CTCs provide unique insights into the biology of cancer metastasis as well as clinically relevant information on tumor evolution, risk assessment, and therapy response in individual patients with solid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80509a0d0dd4086ca6871a4474226159831e914e" target='_blank'>
              From Discovery to Diagnosis: A Perspective for Circulating Tumor Cells in Personalized Oncology.
              </a>
            </td>
          <td>
            C. Alix-Panabières, Klaus Pantel
          </td>
          <td>2025-08-18</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Cancer cells utilize codon-biased translation to fuel tumorigenesis and drug resistance, but underlying mechanisms remain poorly understood. Here, we show ALKBH1 is overexpressed in acute myeloid leukemia (AML) and essential for leukemia stem/initiating cell (LSC/LIC) self-renewal and AML development/maintenance, whereas dispensable for normal hematopoiesis. ALKBH1 enhances mitochondrial assembly/function and oxidative phosphorylation (OXPHOS), crucial for AML survival/proliferation and resistance to venetoclax, a potent BCL2 inhibitor and widely-used first-line targeted therapy for AML in clinic. Mechanistically, ALKBH1 catalyzes 5-formylcytosine (f5C) at tRNA wobble positions, reprograming decoding and facilitating codon-biased translation, a mechanism we term "Epitranslatomic Midas touch", which in turn drives leukemogenesis and drug resistance by promoting synthesis of key oncogenic proteins like WDR43. Targeting ALKBH1, particularly together with venetoclax, exhibited potent anti-leukemia efficacy in preclinical models with favorable safety profiles. Collectively, our findings elucidate ALKBH1's pivotal role in codon-biased translation and tumorigenesis, and propose a novel therapeutic strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3e11cf9a76aba0c7266c1034a7a9bfa25e9f8a" target='_blank'>
              ALKBH1 drives tumorigenesis and drug resistance via tRNA decoding reprogramming and codon-biased translation.
              </a>
            </td>
          <td>
            Chao Shen, Yuan Che, Keren Zhou, Kitty Wang, Wei Li, Diyuan Xue, Tong Wu, Lu Yang, Meiling Chen, Yue Sheng, Chengwan Zhang, S. Robinson, Huiying Chen, Lillian Sau, Zhenhua Chen, M. Wunderlich, Li Han, Tingting Tang, Y. Qing, Dong Wu, Miao Sun, Keith Leung, Katarzyna Dąbrowska, P. Pirrotte, Yueh-Chiang Hu, Lucy Y Ghoda, Bin Zhang, J. Mulloy, Minjie Wei, Guido Marcucci, Chuan He, Xiaolan Deng, Jianjun Chen
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Musculin antisense RNA 1 (MSC-AS1) is a long non-coding RNA (lncRNA) located on human chromosome 8q13.3-q21.11. Emerging evidence shows that MSC-AS1 is either upregulated or downregulated in 16 types of human cancers, and is associated with clinical pathological features and patient prognosis in 12 of these cancers. It is widely believed that the dysregulation of MSCAS1 contributes to tumor cell growth, metastasis, epithelial-mesenchymal transition (EMT) progression, metabolic reprogramming, and drug resistance formation. Mechanistically, MSC-AS1 can act as a competing endogenous RNA (ceRNA) by sponging 14 miRNAs to affect the expression of downstream mRNAs, or it may directly interact with proteins, both of which contribute to the activation of the PI3K/AKT and Wnt/β-catenin signaling pathways. Our review study suggests that MSC-AS1 is a potential cancer biomarker and therapeutic target. In summary, we have explained the research on MSC-AS1 related to cancer treatment, its expression patterns, functional characteristics, and molecular mechanisms in malignant tumors. We have further emphasized its significance in clinical prognosis and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be7dcd5f668c3a79f584b07dd072351fb0f20264" target='_blank'>
              The Roles, Mechanisms, and Clinical Significance of Long Non-coding RNA MSC-AS1 in Cancer.
              </a>
            </td>
          <td>
            Jingjie Yang, Fan Peng, Kexing Liu, Haodong He, Haoran Liu, Li Li, Qianqian Yao, Ning Yang, Gang Zhou, Chengfu Yuan
          </td>
          <td>2025-09-15</td>
          <td>Mini reviews in medicinal chemistry</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Circulating tumour DNA (ctDNA) in liquid biopsies has emerged as a powerful biomarker in cancer patients. Its relative abundance in cell-free DNA serves as a proxy for the overall tumour burden. Here we present GeneBits, a method for cancer therapy monitoring and relapse detection. GeneBits employs tumour-informed enrichment panels targeting 20–100 somatic single-nucleotide variants (SNVs) in plasma-derived DNA, combined with ultra-deep sequencing and unique molecular barcoding. In conjunction with the newly developed computational method umiVar, GeneBits enables accurate detection of molecular residual disease and early relapse identification. Results To assess the performance of GeneBits and umiVar, we conducted benchmarking experiments using three different commercial cell-free DNA reference standards. These standards were tested with targeted next-generation sequencing (NGS) workflows from both IDT and Twist, allowing us to evaluate the consistency and accuracy of our approach across different oligo-enrichment strategies. GeneBits achieved comparable depth of coverage across all target sites, demonstrating robust performance independent of the enrichment kit used. For duplex reads with ≥ 4x UMI-family size, umiVar achieved exceptionally low error rates, ranging from 7.4×10-7 to 7.5×10-5. Even when including mixed consensus reads (duplex & simplex), error rates remained low, between 6.1×10-6 and 9×10-5. Furthermore, umiVar enabled variant detection at a limit of detection as low as 0.0017%, with no false positive calls in mutation-free reference samples. In a reanalysed melanoma cohort, variant allele frequency kinetics closely mirrored imaging results, confirming the clinical relevance of our method. Conclusion GeneBits and umiVar enable highly accurate therapy and relapse monitoring in plasma as well as identification of molecular residual disease within four weeks of tumour surgery or biopsy. By leveraging small, tumour-informed sequencing panels, GeneBits provides a targeted, cost-effective, and scalable approach for ctDNA-based cancer monitoring. The benchmarking experiments using multiple commercial cell-free DNA reference standards confirmed the high sensitivity and specificity of GeneBits and umiVar, making them valuable tools for precision oncology. UmiVar is available at https://github.com/imgag/umiVar. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06993-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599400583a25eed13254e7af73c49e834e021490" target='_blank'>
              GeneBits: ultra-sensitive tumour-informed ctDNA monitoring of treatment response and relapse in cancer patients
              </a>
            </td>
          <td>
            Julian Broche, Olga Kelemen, Aishwarya Sekar, Leon Schütz, Francesc Muyas, A. Forschner, Christopher Schroeder, Stephan Ossowski
          </td>
          <td>2025-08-27</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Streptococcus pneumoniae (pneumococcus) is a genetically diverse opportunistic bacterial pathogen that expresses two phase-variable loci encoding restriction-modification systems. Comparisons of two genetically-distinct pairs of epigenetically-distinct variants, each distinguished by a stabilised arrangement of one of these phase-variable loci, found the consequent changes in genome-wide DNA methylation patterns were associated with differential expression of mobile genetic elements (MGEs). This relationship was hypothesised to be mediated through changes in xenogenic silencing (XS) or nucleoid organisation. Therefore the chromosomal conformation of both variants of each pair were characterised using Illumina Hi-C, and Nanopore Pore-C, sequencing. Both methods concurred that the organisation of the pneumococcal chromosome was dominated by small-scale structures, with most pairwise interactions between loci <25 kb apart. Neither found substantial evidence for higher-order structure or XS in the pneumococcal genome, with more complex contact patterns only evident around the replication origin. Comparisons between the variants identified phage-related chromosomal islands (PRCIs) as the foci of differential contact densities between the variants. This was driven by copy number variation, resulting from variable excision and replication of the episomal PRCIs. However, the methods were discordant in their identification of the variant in which the PRCI was more actively replicating in both pairs. Validatory experiments demonstrated that the prevalence of circular PRCIs was not determined by DNA modification, but instead varied stochastically between colonies in both backgrounds, and was metastable during vegetative growth. PRCI excision was inducible by mitomycin C, but independent of the presence of a prophage. Yet transcriptional activation of these elements was affected by both signals, indicating transcription and replication are separately regulated. Therefore pneumococcal MGEs do not appear to be subject to XS, resulting in heterogeneity being generated within these bacterial populations through the frequent local disruption of chromosome conformation resulting from the stochastic excision and reintegration of episomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e055c4fe6d0e438363a006251436d6f88a77020" target='_blank'>
              Pore-C sequencing identifies episome-driven chromosome conformation perturbations differentiating pneumococcal epigenetic variants
              </a>
            </td>
          <td>
            Tze Yee Lim, Samuel T. Horsfield, C. Troman, Stephen D. Bentley, M. Kwun, N. Croucher
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Trachea squamous cell carcinoma (TSCC) is a subtype of lung cancer. A thorough investigation of the tumor microenvironment of TSCC is crucial for the development of cancer therapeutics and predicting clinical responses. In this study, we utilized single-cell RNA sequencing to analyze seven TSCC samples (including five malignant and two non-malignant samples) and obtained 70,682 high-quality cells. Based on the expression levels of marker genes, we identified 7 major cell types within the samples. By comparing malignant samples that received chemotherapy with those that did not, we identified critical transcriptional regulators responsible for T cell state transition in response to chemotherapy. Additionally, we found specific transcriptional regulators and differentially expressed genes between malignant and non-malignant groups. We identified more particularly abundant specifical intercellular communication in the malignant sample group and that may significantly influence the progression and spread of cancerous cells. Overall, our study provides the first single-cell atlas that comprehensively explains TSCC development and chemotherapy effects, thereby laying a new molecular foundation for therapeutic research in TSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f4d5fafc26e824adc29e95477c2a4bfd6ebf074" target='_blank'>
              Comprehensive single-cell sequencing reveals the tumor microenvironment and tumor-specific characteristics in trachea squamous cell carcinoma
              </a>
            </td>
          <td>
            HongWu Wang, Hongli Li, Heng Zou, Han Meng, Yan Liu, Chengjun Ban, Weixia Yu, Miao Cheng, Jun Teng
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) was a major cause of cancer-related mortality globally. Despite advancements in immunotherapy and targeted therapies, clinical outcomes were still limited by tumor heterogeneity and treatment resistance. The transcription factor (TF) FOS, a key component of the AP-1 complex, was linked to tumor progression and therapy resistance in various cancers, but its precise mechanisms remained unclear, and its role in lung adenocarcinoma (LUAD) was unknown. We investigated the tumor microenvironment (TME) of LUAD using single-cell RNA sequencing (scRNA-seq) to identify potential therapeutic vulnerabilities and FOS-driven mechanisms. Methods We identified fourteen cell types by analyzing scRNA-seq data from LUAD samples (GSE164789) using Seurat (v4.4.0) and Harmony for batch correction. InferCNV was utilized to characterize the tumor cell subtypes after they were clustered using marker genes. CytoTRACE and Monocle were used to create pseudotime trajectories in order to map differentiation states. CellChat revealed intercellular communication networks, while SCENIC identified TF regulatory modules. The CCK-8, Edu, Transwell, and wound healing assays showed that FOS knockdown functionally validated A549 and NCI-H1975 cells. Furthermore, a prognostic model was developed. Results We discovered that invasive LUAD was dominated by a highly stem-like C0 MAFF+ tumor cell subtype that produced chemokines and activated lipid metabolism. These cells stimulated immunosuppression and tumor-associated macrophage (TAM) differentiation by interacting with macrophages via MIF-(CD74+CD44) signaling. Experiments using FOS knockdown demonstrated its role in maintaining invasion, migration, and proliferation. Using the MTRS model, patients were categorized into high- and low-risk cohorts, high-risk patients exhibited unique drug sensitivities. Immunoprofile analysis revealed higher M1 macrophages in high-risk patients, suggesting that FOS inhibition could repolarize TAMs and enhance immunotherapies. Conclusion Our studies show that FOS is a main regulator of C0 MAFF+ TCs in LUAD, polarizing macrophages via MIF and rewiring lipid metabolism to support cancer. The MTRS model offers clinical value for risk assessment even if FOS inhibition shows promise as a therapeutic approach to raise immunotherapy efficacy. Targeting the FOS could cause TME immunosuppression to be disrupted, thus LUAD presents a fresh precision oncology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2ef348cf0a48b9b9c178cc598309bdb56456d8" target='_blank'>
              Targeted and personalized immunotherapy in lung adenocarcinoma: single-cell RNA sequencing of MAFF+ tumor cells and the therapeutic potential of FOS
              </a>
            </td>
          <td>
            Xiangsong Cheng, Shu Chen, Yilong Fu, Runze Jiang, Yanlong Jing, Bizhu Zhao, Dong Guo, Liangyu Wang, Zi Ye, Yumeng Li, Xianliang Chen
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Olaparib and temozolomide (OT) combination therapy is in clinical trial evaluation for rhabdomyosarcoma (RMS). Unfortunately, OT resistance has been reported in other cancers. Using preclinical mouse xenograft experiments, we show that OT effectively suppresses RMS growth, yet over half of RMS tumors develop resistance associated with transcriptomic changes that occur in the absence of recurrent genomic mutation. To date, our mechanistic studies have uncovered that OT resistance is driven by the PIK3CA/AKT pathway which in turn regulates expression of ABC transporters that are well-known to efflux drugs from cells and lead to therapy resistance in RMS. PIK3CA inhibitor alpelisib re-sensitizes resistant cells to OT by suppressing expression of ABC transporters. The combination of OT + alpelisib also kills RMS cells which are resistant to standard-of-care combination chemotherapy [vincristine, actinomycin D, and cyclophosphamide (VAC)] and was effective in preclinical xenograft mouse models at curbing tumor growth. We also observed that cells can transit from therapy resistant to sensitive states after long-term, serial passaging in the absence of drug in most therapy-resistant RMS cell models, which indicates the therapy-resistance may be commonly regulated by non-genetic mechanisms. Our work defines a common resistance pathway in RMS and has credentialed PIK3CA/AKT inhibition as a preclinical strategy to kill therapy resistant RMS.



 Yueyang Wang, Qiqi Yang, Luis A. Corchete Sanchez, Sabateeshan Mathavarajah, Yun Wei, Chuan Yan, Stephanie Strom, David Langenau. The PIK3CA/AKT pathway drives therapy resistance in rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B042.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb32d15d2c8b959f6afceb326a51ba4c88af24f" target='_blank'>
              Abstract B042: The PIK3CA/AKT pathway drives therapy resistance in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Yueyang Wang, Qiqi Yang, L. A. Corchete Sanchez, Sabateeshan Mathavarajah, Yun Wei, Chuan Yan, Stephanie Strom, David M Langenau
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor gene alterations can serve as predictive biomarkers for therapy response. The nucleotide excision repair (NER) helicase ERCC2 carries heterozygous missense mutations in approximately 10% of bladder tumors, and these may predict sensitivity to cisplatin treatment. To explore the clinical actionability of ERCC2 mutations, we assembled a multinational cohort of 2,012 individuals with bladder cancer and applied the highly quantitative CRISPR-Select assay to functionally profile recurrent ERCC2 mutations. We also developed a single-allele editing version of CRISPR-Select to assess heterozygous missense variants in their native context. From the cohort, 506 ERCC2 mutations were identified, with 93% being heterozygous missense variants. CRISPR-Select pinpointed deleterious, cisplatin-sensitizing mutations, particularly within the conserved helicase domains. Importantly, single-allele editing revealed that heterozygous helicase-domain mutations markedly increased cisplatin sensitivity. Integration with clinical data confirmed that these mutations were associated with improved response to platinum-based neoadjuvant chemotherapy. Comparison with computational algorithms showed substantial discrepancies, highlighting the importance of precision functional assays for interpreting mutation effects in clinically relevant contexts. Our results demonstrate that CRISPR-Select provides a robust platform to advance biomarker-driven therapy in bladder cancer and supports its potential integration into precision oncology workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298c9b125eade81e30854dcc4c068fdf43533872" target='_blank'>
              Quantitative functional profiling of ERCC2 mutations deciphers cisplatin sensitivity in bladder cancer
              </a>
            </td>
          <td>
            J. Börcsök, Diyavarshini Gopaul, Daphne Devesa-Serrano, Clémence Mooser, Nicolas Jonsson, M. Cagiada, D. Stormoen, M. N. Ataya, B. Guercio, H. Kaimakliotis, G. Iyer, Kresten Lindorff-Larsen, L. Dyrskjøt, Kent W. Mouw, Z. Szallasi, C. S. Sørensen
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="DNA mismatch repair (MMR) preserves genomic integrity by correcting replication errors. Deficiency in MMR (MMRd) results in microsatellite instability, increased tumor mutational burden, neoantigen generation, and activation of immune response. In this review, we first outlined how MMR loss promotes immune activation and responsiveness to immune checkpoint blockade (ICB), establishing MMRd status as the first tumor-agnostic biomarker for ICB therapy. Subsequently, we summarized the compelling evidence that defines MMR status as a dynamic, context-dependent process influenced by environmental and therapeutic pressures, rather than a fixed, binary trait. Accordingly, we discussed the implications of spatial and temporal heterogeneity of MMR status for both the diagnosis and treatment of cancer, the differential response of MMRd tumors to ICB, as well as the occasional benefits observed in MMR-proficient (MMRp) immune-cold cancers. We then explored strategies to exploit MMR dynamics and mimic MMRd-like phenotypes through alkylating agents, pharmacological MMR inhibition, and stress-mediated modulation, with the aim of sensitizing refractory tumors to immunotherapy. Finally, we reported emerging therapeutic opportunities in MMRd tumors, including Werner helicase inhibition, nonsense-mediated decay blockade, and neoantigen-targeted vaccines. Altogether, reframing MMR as a dynamic and targetable axis may broaden immunotherapy applicability and advance precision immune oncology across different tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6face00bfd040e4e182ff419f940662d44199ca5" target='_blank'>
              Mismatch Repair as a Dynamic and Clinically Actionable Vulnerability in Cancer.
              </a>
            </td>
          <td>
            Eleonora Piumatti, P. P. Vitiello, V. Amodio, A. Bardelli, Giovanni Germano
          </td>
          <td>2025-09-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background/Objectives: Medulloblastoma is the most common malignant brain tumor in children and comprises four molecular subtypes—WNT, SHH, Group 3, and Group 4—each with distinct genetic, epigenetic, and metabolic features. Increasing evidence highlights the critical role of metabolic reprogramming and epigenetic alterations in driving tumor progression, therapy resistance, and clinical outcomes. This review aims to explore the interplay between metabolic and epigenetic mechanisms in medulloblastoma, with a focus on their functional roles and therapeutic implications. Methods: A comprehensive literature review was conducted using PubMed and relevant databases, focusing on recent studies examining metabolic pathways and epigenetic regulation in medulloblastoma subtypes. Particular attention was given to experimental findings from in vitro and in vivo models, as well as emerging preclinical therapeutic strategies targeting these pathways. Results: Medulloblastoma exhibits metabolic adaptations such as increased glycolysis, lipid biosynthesis, and altered amino acid metabolism. These changes support rapid cell proliferation and interact with the tumor microenvironment. Concurrently, epigenetic mechanisms—including DNA methylation, histone modification, chromatin remodeling, and non-coding RNA regulation—contribute to tumor aggressiveness and treatment resistance. Notably, metabolic intermediates often serve as cofactors for epigenetic enzymes, creating feedback loops that reinforce oncogenic states. Preclinical studies suggest that targeting metabolic vulnerabilities or epigenetic regulators—and particularly their combination—can suppress tumor growth and overcome resistance mechanisms. Conclusions: The metabolic–epigenetic crosstalk in medulloblastoma represents a promising area for therapeutic innovation. Understanding subtype-specific dependencies and integrating biomarkers for patient stratification could facilitate the development of precision medicine approaches that improve outcomes and reduce long-term treatment-related toxicity in pediatric patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/733445d34f0ee82899b45eeeaeabd1e9ff5c7ba8" target='_blank'>
              Deciphering Medulloblastoma: Epigenetic and Metabolic Changes Driving Tumorigenesis and Treatment Outcomes
              </a>
            </td>
          <td>
            Jenny Bonifacio-Mundaca, Sandro Casavilca-Zambrano, Christophe Desterke, Íñigo Casafont, Jorge Mata-Garrido
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is a major challenge to public health in China. Its incidence and mortality rates are increasing year by year, and traditional treatment methods are facing limitations such as drug resistance and side effects. This year's research has found that circRNA plays an important role in tumorigenesis, metastasis and drug resistance by regulating mechanisms such as miRNA interaction networks, gene expression and cell cycles. Its stability and specificity make it a novel biomarker and therapeutic target. However, the standardization of circRNA detection techniques is insufficient, and its role in epigenetic regulation and the tumor microenvironment remains unclear. This article systematically analyzes the generation mechanism, functional diversity (such as molecular sponges, protein interactions, and encoded peptides) of circRNAs, as well as their application strategies in cancer treatment (such as exosome detection) and therapy (such as mesenchymal stem cell delivery), revealing the dynamic network by which they regulate tumor drug resistance and cellular metabolism. The research results provide a theoretical basis for the development of non-invasive diagnostic tools and targeted therapies. In the future, it is necessary to combine multi-group masking techniques and engineered delivery systems to break through the bottleneck of detection technology and explore the synergistic mechanism between circRNA and other ncRNAs, so as to promote the clinical transformation of precision cancer medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cfef4283107d45fd6a939a00a655bc04b6cb7fb" target='_blank'>
              Evaluation of the Regulatory Role of Circular RNA in Tumors and Its Application Value
              </a>
            </td>
          <td>
            Yingjie Xin
          </td>
          <td>2025-08-26</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. First, mapping the proteins onto pathways assists combinatorial drug selections and helps to detect metastases. Second, it provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients’ genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Simple Summary Breast cancer (BC) remains a significant global health challenge, partly due to its ability to develop resistance to treatment. BC resistance arises from various biological mechanisms, including cancer cells actively expelling drugs, altering their internal metabolic and signaling pathways, evading the immune system, or persisting as drug-tolerant cancer stem cells. These changes occur in the context of a tumor microenvironment that further shields the cancer, limiting treatment access. As the disease advances, particularly when it spreads to other body parts, effective treatment becomes increasingly difficult. Recent advancements in artificial intelligence (AI) offer promising avenues for improving early detection, refining diagnoses, enhancing clinical decision-making and thereby personalizing treatment strategies. This review examines the biological mechanisms that cause BC resistance and its treatment. It further explores AI’s potential to address critical challenges, transforming BC therapy to provide improved survival rates and enhanced quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb5357bf49f1fab994fbcce36bcc1738760cec3" target='_blank'>
              The Underlying Mechanisms and Emerging Strategies to Overcome Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Krishnaswamy Kannan, Alagarsamy Srinivasan, Aarthi Kannan, Nawab Ali
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4e4fc33eb2e8814d38d33c359427c463ea1f89" target='_blank'>
              Resistance to CDK7 inhibitors directed by acquired mutation of a conserved residue in cancer cells.
              </a>
            </td>
          <td>
            Chun-Fui Lai, Victoria I. Cushing, Ellen Olden, C. Bevan, R. C. Coombes, Basil J. Greber, L. Buluwela, Simak Ali
          </td>
          <td>2025-09-08</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hormone receptor-positive (HR + )/HER2−negative (HER2 − ) breast cancer is the most common subtype, with biomarker-driven therapies improving outcomes. Circulating tumor DNA (ctDNA) analysis enables minimally invasive assessment of somatic alterations to guide therapy. However, assay choice impacts clinical utility, and access remains inconsistent. This study compares single-gene and panel-based sequencing for assessing PIK3CA mutations and broader genomic profiling. We conducted a prospective, multicenter study analyzing 161 plasma samples from 146 patients before initiating a new line of palliative therapy using the SiMSen-Seq (SSS) assay for PIK3CA hotspot mutations, the AVENIO ctDNA Expanded assay (77 genes) and mFAST-SeqS for tumor fraction estimation. High concordance (92.6%) was observed between SSS and AVENIO for PIK3CA mutations. AVENIO identified additional actionable alterations, including ESR1 (17.5%) and PI3K pathway alterations (40.6%), and together with tumor fraction estimation, improved interpretation of negative liquid biopsy findings. These findings support broader ctDNA profiling in clinical practice while highlighting accessibility challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e5152da00e15846285751e550fbaaef5313e05" target='_blank'>
              Clinical impact of single-gene vs. panel sequencing in advanced HR + /HER2− breast cancer: insights and implications
              </a>
            </td>
          <td>
            E. Klocker, N. Dobrić, Ricarda Graf, Christine Beichler, D. Hlauschek, C. Suppan, L. Pancheri, D. Egle, Carmen Albertini, R. Bartsch, A. Starzer, P. Jost, G. Rinnerthaler, Ellen Heitzer, N. Dandachi, M. Balic
          </td>
          <td>2025-08-07</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT Introduction Next‐generation sequencing‐based comprehensive cancer genomic profiling (CGP) tests are beneficial for refining diagnosis and personalized treatment of various cancers. However, the clinical impact of CGP, as covered by public health insurance in the management of sarcomas, remains unknown. Especially, the data on the utility of the newly emerging dual DNA–RNA panel compared to the conventional DNA‐only panel in clinical settings is lacking. Therefore, we evaluated the utility of CGP in routine clinical practice for sarcoma treatment. Patients and Methods In this study, three types of DNA panel and one DNA–RNA panel, reimbursed by Japanese public health insurance, were utilized. We detected oncogenic and druggable gene mutations and genotype‐matched therapies. Results One hundred and thirty‐six patients were included in this study. Based on the detection of highly histology‐specific translocations in the sequencing results, 2.2% of patients were re‐classified. In patients with translocation‐related sarcomas, a DNA–RNA panel identified more histology‐specific fusion genes than DNA panels (p = 0.0035). Specifically, 86.8% and 39.0% of patients had oncogenic and druggable genomic alterations, respectively. Of these, 9.6% underwent genotype‐matched therapy, with a 36.3% response rate and an 81.8% disease control rate. Patients who were administered genomically matched therapy had better overall survival (OS) than those who did not in patients with metastatic or advanced sarcoma with no prior chemotherapy (3‐year OS: 83.3% vs. 48.0%, p = 0.42). Patients with TP53 and RB1 mutations had worse OS than those without. Germline findings were detected in 11.0% of the patients, one of whom had a truly germline origin. Conclusions This study suggests that publicly reimbursed CGP tests, particularly the dual DNA–RNA panel, could be beneficial for refining diagnostic precision in selected sarcoma subtypes, treatment decisions, detecting the germline findings, and prognosis prediction in routine clinical settings for sarcoma. The implementation of genotype‐matched therapies showed favorable clinical outcomes and improved the prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f63a84e1907839e26eef4d1f8d8db48869e5a3" target='_blank'>
              Real‐World Data of Comprehensive Cancer Genomic Profiling Tests Performed in the Routine Clinical Setting in Sarcoma
              </a>
            </td>
          <td>
            E. Nakata, D. Ennishi, Tatsunori Osone, K. Ninomiya, Shuta Tomida, Takuto Itano, T. Fujiwara, T. Kunisada, Naoyuki Ida, Hideki Yamamoto, Mashu Futagawa, T. Shimoi, Hiroyuki Yanai, Akira Hirasawa, S. Toyooka, Masahiro Tabata, Toshifumi Ozaki
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            , 
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background DNA methylation is an epigenetic mechanism involved in gene regulation and cellular differentiation. Accurate and comprehensive assessment of DNA methylation patterns is thus essential for understanding their role in various biological processes and disease mechanisms. Bisulfite sequencing has long been the default method for analyzing methylation marks due to its single-base resolution, but the associated DNA degradation poses a concern. Although several methods have been proposed to circumvent this issue, there is no clear consensus on which method might be better suited for specific study designs. Results We conducted a comparative evaluation of four DNA methylation detection approaches: whole-genome bisulfite sequencing (WGBS), Illumina methylation microarray (EPIC), enzymatic methyl-sequencing (EM-seq) and third-generation sequencing by Oxford Nanopore Technologies (ONT). DNA methylation profiles were assessed across three human genome samples derived from tissue, cell line, and whole blood. We systematically compared these methods in terms of resolution, genomic coverage, methylation calling accuracy, cost, time, and practical implementation. EM-seq showed the highest concordance with WGBS, indicating strong reliability due to their similar sequencing chemistry. ONT sequencing, while showing lower agreement with WGBS and EM-seq, captured certain loci uniquely and enabled methylation detection in challenging genomic regions. Despite a substantial overlap in CpG detection among methods, each method identified unique CpG sites, emphasizing their complementary nature. Conclusions Our findings underscore the strengths and limitations of current DNA methylation detection methods. EM-seq and ONT emerge as robust alternatives to WGBS and EPIC, offering unique advantages: EM-seq delivers consistent and uniform coverage, while ONT excels in long-range methylation profiling and access to challenging genomic regions. These insights provide practical guidance for method selection based on specific experimental goals. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00616-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115f5c6b6dae937448bce4077f0951e9ac581f3" target='_blank'>
              Comparison of current methods for genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Ana Regina de Abreu, J. Ibrahim, Vasileios Lemonidis, Ligia Mateiu, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer remains a significant global health challenge, with varying incidence and mortality rates across the globe. According to the Surveillance, Epidemiology and End Results (SEER) registry [1] and the American Cancer Society (ACS) [2], estimated new cancer cases in the US will be over 2 million in 2025. Understanding the burden of disease and advancements in tailored treatments, also known as precision or personalized medicine, is crucial for improving patient outcomes. Pharmacogenomics is the study of how an individual's genetic makeup influences their response to treatment [3]. This field seeks to identify genetic variations among individuals (pharmacogenomic variants) in order to predict drug efficacy and the likelihood of adverse drug reactions, thereby optimizing and personalizing treatment outcomes while minimizing side effects to the patient [4]. In the last decade, increasing attention has shifted beyond the traditional coding genome to noncoding RNAs (ncRNAs), which serve as key regulators of gene expression and biomarkers in cancer, adding complexity to tumor signaling networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098a88f252b1f1d56afe7934e6fcc3acf0eeef5b" target='_blank'>
              Current Insights into Pharmacogenomics of Noncoding RNAs for Cancer Therapy
              </a>
            </td>
          <td>
            Anand Bhaskar, Yundong Zhou, Sujit Nair
          </td>
          <td>2025-09-26</td>
          <td>GenoMed Connect</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c96cb29c7c4ee18f843780f85e8c5d424d7abe77" target='_blank'>
              Epigenetic modulation in cancer drug discovery: promising targets and clinical applications.
              </a>
            </td>
          <td>
            S. Vickram, Shofia Saghya Infant, M. Sivasubramanian, Saravanan Anbalagan, Mathan Muthu Chinnakannu Marimuthu, Hitesh Chopra
          </td>
          <td>2025-09-02</td>
          <td>Pharmacological reports : PR</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM) is one of the most aggressive malignant brain tumors. The Stupp protocol is associated with a short median survival and high recurrence rate, referring to limited effective therapeutic alternatives. Various immunotherapies are therefore under active investigation. However, to date, their clinical application has been hindered by an unfavorable risk-benefit profile, with drug resistance posing a major barrier to therapeutic success. This review focuses on an immune checkpoint PD-1/PD-L1 inhibitors, which have shown limited efficacy as monotherapy. Key mechanisms of resistance include the immunosuppressive tumor microenvironment, aberrant genetic and epigenetic regulation, and impaired T-cell function and infiltration. To address these challenges, combination strategies targeting these resistance mechanisms have been developed, including combination of dual or triple drug regimens, targeting multiple pathways to reverse the immunosuppressive tumor microenvironment or correct underlying genetic abnormalities, along with the optimization of drug delivery routes to fundamentally enhance the drug efficacy. These approaches offer a promising theoretical framework for enhancing the therapeutic efficacy of immune checkpoint inhibitors in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d85d013983ef4b334056fbe10d67cea23b2ca903" target='_blank'>
              Decoding PD-1 Resistance: From Microenvironment to Combination Therapies in Glioblastoma
              </a>
            </td>
          <td>
            Wenqin Ding
          </td>
          <td>2025-09-03</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioma is an aggressive brain tumor that requires challenging treatments. Tumor Treating Fields (TTFields), an FDA-approved therapy for glioblastoma (GBM), pleural mesothelioma, and platinum-refractory metastatic nonsmall cell lung cancer (in combination with PD-1/PD-L1 inhibitors or docetaxel), employs specific frequency electric fields to disrupt cell division and enhance treatment efficacy. However, their molecular mechanisms remain unclear. This study aimed to elucidate these mechanisms and optimize the therapeutic potential of TTFields through quantitative proteomics, phosphoproteomics, and glycoproteomics. Pathway analysis of the proteomics revealed that TTFields impact the cell cycle, DNA repair, autophagy, and DNA replication. Phosphoproteomic studies further demonstrated a marked decline in the activity of key kinases ABL1 and PDK1, while glycoproteomics highlighted disruptions in cell adhesion and ECM-receptor interactions. Notably, proteomic analysis identified an upregulation of PARP1 and BRD4 protein levels, suggesting a previously unrecognized resistance mechanism. Consistently, combining TTFields with inhibitors targeting these proteins significantly enhanced the treatment efficacy in U87 cells. Thus, this study uncovers comprehensive molecular mechanisms underlying TTFields' effects on GBM cells and supports the development of concomitant therapies to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23e36a333192c5464239bf92b14bd9554e97f70e" target='_blank'>
              Multiomics Insights into the Mechanism and Enhanced Efficacy of Tumor Treating Fields (TTFields) Therapy in Glioblastoma.
              </a>
            </td>
          <td>
            Yanmei Gao, Chenxu Wang, Guobin Liu, Ruilin Zhang, Xuelian Ren, Guangyuan Hu, Qi Mei, He Huang
          </td>
          <td>2025-09-01</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Paclitaxel remains a cornerstone of cancer therapy, yet resistance significantly limits its clinical efficacy. Emerging evidence implicates circular RNAs (circRNAs) as key regulators of paclitaxel resistance, offering potential biomarkers and therapeutic targets. This review synthesizes current knowledge on circRNA-mediated resistance mechanisms, categorizing their roles as promoters or suppressors of chemoresistance. We highlight two major pathways—miRNA sponging and protein interactions—through which circRNAs alter drug sensitivity. Moreover, we focus on enhancement of epithelial–mesenchymal transition and modulation of autophagy as two significant downstream pathways influenced by the established primary mechanisms. Notably, our analysis reveals context-dependent dualities, where certain circRNAs exert opposing effects in different cancers, underscoring the need for tissue-specific therapeutic strategies. Furthermore, we identify critical gaps in understanding circRNA regulatory networks and their clinical translatability. By evaluating circRNA signatures associated with paclitaxel response, this review proposes a framework for biomarker development and combination therapies to circumvent resistance. Our findings emphasize the urgency of functional studies and standardized profiling methods to harness circRNAs for personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6cc3f7c3626a0076f8fb95907d4419e7ad6fdb" target='_blank'>
              Circular RNAs: key regulators of paclitaxel sensitivity and resistance in cancer
              </a>
            </td>
          <td>
            Zahra Tajik, S. Ghafouri-Fard
          </td>
          <td>2025-08-18</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573d73f70f9bc8245cb6e46c84579fa69539af01" target='_blank'>
              Role of tumor microenvironment in cancer promotion, development of drug resistance and cancer treatment.
              </a>
            </td>
          <td>
            Duaa e Fathah, S. Ejaz
          </td>
          <td>2025-09-15</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d27cc9b475222eaa7f4e83e26c9db176ee78f1" target='_blank'>
              Novel mutations in transducin-like enhancer of split 6 cause infertility in women with embryonic arrest.
              </a>
            </td>
          <td>
            Jifan Tan, Yue Jiang, Ching Wan, Kehan Zhu, Yan Xu, Yanwen Xu, Jia Huang
          </td>
          <td>2025-09-27</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Recent research suggests higher circulating lymphocyte counts may protect against colorectal cancer (CRC). However, the role of specific lymphocyte subtypes and activation states remain unclear. CD4+ T cells - a highly dynamic lymphocyte subtype - undergo gene expression changes upon activation that are critical to their effector function. Previous studies using bulk tissue have limited our understanding of their role in CRC risk to static associations. We applied Mendelian randomization (MR) and genetic colocalisation to evaluate causal relationships of gene expression on CRC risk across multiple CD4+ T cell subtypes and activation states. Genetic proxies were obtained from single-cell transcriptomic data, allowing us to investigate the causal effect of expression of 1,805 genes across CD4+ T cell activation states on CRC risk (78,473 cases; 107,143 controls). Analyses were stratified by CRC anatomical subsites and sex, with sensitivity analyses assessing whether the observed effect estimates were likely to be CD4+ T cell-specific. We identified six genes - FADS2, FHL3, HLA-DRB1, HLA-DRB5, RPL28, and TMEM258 - with strong evidence for a causal role in CRC development (FDR-P<0.05; colocalisation H4>0.8). Causal estimates varied by CD4+ T cell subtype, activation state, CRC subsite and sex. However, many of genetic proxies used to instrument gene expression in CD4+ T cells also act as eQTLs in other tissues, highlighting the challenges of using genetic proxies to instrument tissue-specific expression changes. We demonstrate the importance of capturing the dynamic nature of CD4+ T cells in understanding CRC risk, and prioritize genes for further investigation in cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7263a903e4588231be1bcb20d71d5c5ee5801de5" target='_blank'>
              Transcriptome-wide Mendelian randomization exploring dynamic CD4+ T cell gene expression in colorectal cancer development.
              </a>
            </td>
          <td>
            Benedita Deslandes, Xueyan Wu, Matthew A Lee, L. J. Goudswaard, Gareth W Jones, A. Gsur, Annika Lindblom, S. Ogino, Veronika Vymetálková, A. Wolk, Anna H Wu, J. R. Huyghe, Ulrike Peters, Amanda I. Phipps, Claire E Thomas, Rish K. Pai, Robert C Grant, Daniel D Buchanan, J. Yarmolinsky, Marc J. Gunter, Jie Zheng, E. Hazelwood, E. E. Vincent
          </td>
          <td>2025-09-17</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third leading cause of cancer-related deaths. Integrating cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present a comparative mRNA-seq analysis of tissue samples from 32 CRC patients, pairing tumors with adjacent healthy tissues. Differential expression gene (DEG) analysis revealed dysregulated metabolic programs. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2, which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potential of the Xc- system, we analyzed gene perturbations from CRISPR screens across various cell types and used functional assays in five primary patient-derived organoid models. We identified a previously uncharacterized cell surface protein signature predicting chemotherapy resistance and highlighted the causality and potential of pharmacological blockage of ferroptosis as a promising avenue for cancer therapy. Redox homeostasis, ion/amino acid transporters, and regulators of neuronal survival and differentiation were pathways associated with these co-dependent genes in patient specimens. This study highlights several potential clinical targets for CRC therapy and promotes the use of patient-derived organoids oids to functionally validate in silico predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af3aa92ab85625c6d369fc2c084fc1ee49ca6118" target='_blank'>
              Patient-specific pharmacogenomics demonstrates xCT as predictive therapeutic target in colon cancer with possible implications in tumor connectivity.
              </a>
            </td>
          <td>
            M. Strecker, Keren Zohar, Martin Böttcher, T. Wartmann, H. Freudenstein, Maximilian Doelling, Mihailo Andric, Wenjie Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, D. Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. Kahlert
          </td>
          <td>2025-09-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) is one of the leading cause of cancer-associated mortality globally. Its tumor microenvironment consists of various types of cells, which have a great impact on patient response and prognosis. The present study used scRNA-seq technology to investigate gene expression pattern of various cell types in NSCLC and their correlation with immune microenvironment. Methods We acquired NSCLC single-cell RNA sequencing data from GSE117570 and employed UMAP clustering analysis to explore the major cell subpopulations. Additionally, we explored the differential expression of important genes among various cell types and their associations with immune cell infiltration, scores of tumor microenvironment. Results Our results exhibited the heterogenicity of cell types in NSCLC and also indicated more than 60 genes whose expression displayed significant differences between cell groups, such as AP1S1, BTK, FUCA1, NDEL14, TMEM106B and UNC13D. Expression of these genes correlated with immunocyte infiltration, tumor microenvironment scores, indicating their potential roles in tumor progression and therapeutics. Conclusion Our single-cell transcriptomic dissection of NSCLC highlights cellular heterogeneity as well as potential biomarkers associated with immune microenvironment. The findings provide a foundation for the rational development of biomarker-driven personalized treatment and therapy by extending the cell-type specificity of current designs of NSCLC therapeutics. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03494-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfe159d40376ac64bc4970c70d60376979b77ad" target='_blank'>
              Single-cell RNA-seq reveals gene expression heterogeneity in NSCLC and its link to the immune microenvironment
              </a>
            </td>
          <td>
            Jinzhuo Ren, Hua Zhang, Haiyong Wu
          </td>
          <td>2025-09-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A substantial portion of lung cancer-associated genetic elements in East Asian populations remains unidentified, underscoring the need for large-scale genome-wide studies, particularly on non-coding regulation. We conducted a whole genome sequencing (WGS)-based genome-wide scan in 13,722 Chinese individuals to identify regulatory elements associated with lung cancer. We verified common-variant-based loci by meta-analysis across the available East Asian studies. Integrating a genome-transcriptome reference panel of lung tissue in 297 Chinese, we bridged the variant-lung cancer associations, highlighting genes including TP63 and DCBLD1. Implementing the STAAR pipeline for rare variant aggregate analysis, we identified and replicated novel genes, including PARPBP, PLA2G4C, and RITA1 in the context of non-coding regulation. Adapting a deep learning-based approach, potential upstream regulators such as TP53, MYC, ZEB1, and NFKB1 were revealed for the lung cancer-associated genes. These findings offered crucial insights into the non-coding regulation for the etiology of lung cancer, providing additional potential targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/218dccec773052bdb16c39b8293ccbf37c8ec89f" target='_blank'>
              Non-coding genetic elements of lung cancer identified using whole genome sequencing in 13,722 Chinese
              </a>
            </td>
          <td>
            Dan Zhou, Ming Wu, Qilong Tan, Liyang Sun, Yuanxing Tu, Weifang Zheng, Yun Zhu, Min Yang, Kejia Hu, Fang Hu, Xiaohang Xu, Hanyi Zhou, Tian Luo, Fangming Yang, Fuqiang Li, Xin Jin, Huakang Tu, Wenyuan Li, Kui Wu, Xifeng Wu
          </td>
          <td>2025-08-09</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32a97dbee7d52daeda75142b34c4c8f28f256d2" target='_blank'>
              ZBTB11 depletion targets metabolic vulnerabilities in KRAS inhibitor-resistant PDAC.
              </a>
            </td>
          <td>
            Nathan L Tran, Jiewei Jiang, Min Ma, Gillian E. Gadbois, K. Gulay, Alyssa L Verano, Cara R. Schiavon, Elena Rebollo, Haowen Zhou, Chun-Teng Huang, Rabi Murad, David A Scott, Uri Manor, Anne G. Bang, Hervé Tiriac, A. Lowy, Eric S Wang, F. Ferguson
          </td>
          <td>2025-08-11</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Background ADTKD- MUC1 is caused by frameshift mutations in MUC1 gene that produce a frameshifted protein (MUC1fs) toxic to kidney cells. The gene’s variable number of tandem repeats (VNTR), with high GC content, makes it largely inaccessible to standard sequencing. As a result, both the reference sequence and natural variation in this region remain poorly defined, complicating mutation detection and data interpretation. Standard methods also fail to pinpoint the exact VNTR unit affected, limiting insight into mutation mechanisms and genotype–phenotype correlations. Methods We employed Single Molecule, Real-Time (SMRT) sequencing and characterized the genomic sequence of MUC1 in 300 individuals including 279 individuals from 143 families suspected of having ADTKD- MUC1 . We compared these results to those obtained using the CLIA-approved mass spectrometry-based probe extension (PE) assay, which specifically detect the most prevalent 59dupC mutation. We correlated the structural features of the MUC1 VNTR with the rate of kidney function decline in affected individuals. Results We identified MUC1 consensus sequences for 205 unique VNTR alleles, with 9 distinct types of frameshift mutations present on 52 distinct mutated VNTR alleles. MUC1 frameshift mutations were identified in 71 of 143 families (50%) with suspected ADTKD, comprising 135 genetically affected individuals (48%). The SMRT assay exhibited complete concordance and revealed that the PE assay is capable of detecting frameshift mutations in approximately 85% of affected families. The constellation of VNTR structures supports a genotype–progression model, in which fast progressors exhibit a significantly lower number of repeat units on the wild-type allele and a higher number of repeats on the mutation-bearing allele, including an increased number of frameshifted repeat units. Conclusions SMRT sequencing outperforms current diagnostic methods for ADTKD- MUC1 and reveals the prognostic value of VNTR structures. Although their contribution to disease progression is modest (∼6% variance explained), it remains biologically and clinically meaningful. Key Points 3 Single Molecule, Real-Time (SMRT) sequencing of MUC1 outperforms existing genetic and immunohistochemical methods for ADTKD- MUC1 diagnosis. CLIA-approved mass spectrometry-based genotyping assay, complemented by SMRT sequencing, can detect nearly all ADTKD- MUC1 patients. Disease progression correlates with VNTR pattern: fast progressors have fewer WT repeats and more mutant/frameshifted repeats">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcff66f92581ab116e48866b395087d3b037c6c9" target='_blank'>
              Long-Read Sequencing of the MUC1 VNTR: Genomic Variation, Mutational Landscape, and Its Impact on ADTKD Diagnosis and Progression
              </a>
            </td>
          <td>
            Alena Vrbacká, A. Přistoupilová, K. Kidd, Václav Janoušek, M. Radina, Petr Vyleťal, Ibrahim Bitar, V. Stránecký, Lenka Steiner-Mrázová, Helena Trešlová, J. Sovová, K. Hodaňová, H. Hartmannová, D. Mušálková, Klára Svojšová, Tereza Kmochová, V. Barešová, Abby Taylor, Lauren Martin, Antonio Sanchez, Romana Ryšavá, Innet Lajtmanová, Silvie Rajnochova-Bloudickova, O. Viklický, Gregorius Papagregorius, Constantinos Deltas, C. Stavrou, Sofia Jorge, , Márcia Rodrigues, E. Elhassan, Michelle Clince, Colm Rowan, Peter J. Conlon, Omri Teltsh, Gianpiero L Cavalleri, B. Blumenstiel, Diana Toledo, Marina DiStefano, M. DeFelice, Martina Živná, Anthony J. Bleyer, Stanislav Kmoch
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematological malignancy driven by diverse genetic aberrations, with the Philadelphia chromosome and its resultant BCR‑ABL1 fusion gene constituting key pathogenic drivers. Atypical BCR‑ABL1 fusion transcripts have distinctive structural and functional properties. Structural divergence in these variants leads to functional alterations of encoded oncoproteins, potentially influencing disease progression and therapeutic responsiveness. Conventional diagnostic modalities, including reverse transcription‑PCR and fluorescence in situ hybridization, may fail to detect rare variants, necessitating complementary high‑sensitivity techniques such as next‑generation sequencing). Tyrosine kinase inhibitors (TKIs), including imatinib and dasatinib, remain cornerstone treatments; however, marked inter‑variant heterogeneity in TKI responsiveness is observed: Patients harboring e13a3/e14a3 transcripts generally show favorable prognoses, while those with e1a3/e6a2 variants demonstrate an increased risk of relapse and/or TKI resistance, often requiring multimodal strategies combining chemotherapy or allogeneic hematopoietic stem cell transplantation. Although Chimeric Antigen Receptor)‑T cell therapy has shown promise in treating (Philadelphia chromosome‑positive B‑cell Acute Lymphoblastic Leukemia, its application in CML, particularly in variants such as e1a3 or e6a2, is not currently recommended as a first‑line treatment. Despite advances in elucidating the clinical implications of fusion gene heterogeneity in leukemogenesis, the prognostic value of atypical BCR‑ABL1 isoforms requires further validation through multicenter studies with extended cohorts. This review aimed to summarize cases of atypical fusion genes in CML, with analysis of clinical characteristics, therapeutic interventions, and prognostic outcomes, to provide clinicians with enhanced reference material for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec3c00c917da742543a693d4c05a294ce191255" target='_blank'>
              The silent players: Atypical BCR‑ABL isoforms as biomarkers and therapeutic hurdles in CML pathogenesis (Review).
              </a>
            </td>
          <td>
            Xin Zhou, Ai Li, Dexiao Kong, Yuqi Shi, Peipei Zhang, Ning-Ning Shan
          </td>
          <td>2025-09-24</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The mechanism underlying the pathogenic impact of mutations within intrinsically disordered regions of proteins remains enigmatic, and the mechanisms behind compensatory responses to these perturbations lie within an even deeper veil of obscurity. This study focuses on the compensatory mechanisms of single nucleotide variants within the disordered C-terminal tail of the BTG2 protein, a crucial regulator in cell cycle control. Here, we develop a novel approach by combining molecular dynamics simulations with time-dependent linear response theory to accurately compute the long-distance coupling dynamics between the tail and the structured domain. Using this approach, we reveal how specific mutations can counteract the functional disruptions caused by a known disease-associated mutation, V141M. Our findings demonstrate that the disordered tail regulates critical binding sites allosterically, and a weakening of this modulation may contribute to disease manifestation. However, compensatory mutations restore lost interactions between the disordered region and binding sites, exerting long-distance dynamic control over both critical binding sites and the mutation site 141M. This secondhand allosteric control could be a general mechanism for compensatory mutations to rescue function. These insights not only illuminate the pathogenic mechanisms at play but also offer a framework for identifying potential therapeutic targets in diseases associated with disordered protein regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44953c05dc0147543f5bd00f5cc928d283c8664e" target='_blank'>
              Second-order allosteric control as a mechanism for compensatory mutations in B-cell translocation gene 2.
              </a>
            </td>
          <td>
            Nicholas J. Ose, P. Campitelli, Tushar Modi, S. Ozkan
          </td>
          <td>2025-08-25</td>
          <td>Protein science : a publication of the Protein Society</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4026bb201d062046e64e9ee6c696085e559cd08a" target='_blank'>
              Treatment of NSCLC after chemoimmunotherapy - are we making headway?
              </a>
            </td>
          <td>
            M. Reck, N. Frost, Solange Peters, Bernard A. Fox, Roberto Ferrara, R. Savai, F. Barlesi
          </td>
          <td>2025-08-14</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="ABSTRACT Glioblastoma is a highly aggressive and devastating brain malignancy with dismal prognosis and extremely limited therapeutic options. Identification of prognostic biomarkers and therapeutic targets from multi‐omics data is critical for improving patient outcomes. In this study, we investigated the clinical significance of cellular heterogeneity and super‐enhancer‐driven regulatory networks, which are critically implicated in glioblastoma progression and treatment resistance. We first performed scRNA‐seq to dissect tumour microenvironment heterogeneity, identifying 16 distinct cell clusters, including astrocytes, macrophages, and CD8+ T cells. CellChat analysis revealed key intercellular signalling pathways, with astrocytes and macrophages acting as central communication hubs. To integrate bulk RNA sequencing data, we applied the Scissor algorithm to identify survival‐associated cell states. By combining single‐cell and bulk transcriptomic data, we uncovered 642 survival‐related genes, including QKI and RBM47, which robustly predicted patient survival and immunotherapy response. Furthermore, WGCNA analysis identified seven co‐expression modules and super enhancer‐regulated networks orchestrated by transcription factors (RFX2, RFX4) and hub genes (NEAT1, CFLAR). These networks stratified patients into high‐ and low‐risk groups with significant survival differences. Collectively, our findings elucidate the intricate interplay between cellular heterogeneity and super enhancer‐driven gene regulation in glioblastoma, providing a translational framework for targeting oncogenic hubs and modulating microenvironment interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7616587b7549f42937faeb12de6e3dcf343225e7" target='_blank'>
              Integration of Single‐Cell RNA and Bulk RNA Sequencing Reveals Cellular Heterogeneity and Identifies Survival‐Associated Regulatory Networks in Glioblastoma
              </a>
            </td>
          <td>
            Zijun Xu, Bohan Xi, Jiaming Huang, Liqiang Zhang, Sifu Cui, Xianwei Wang, Dong Chen, Shupeng Li
          </td>
          <td>2025-08-13</td>
          <td>IET Systems Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite extensive prior research on prostate cancer (PCa) transcriptomics, the molecular mechanisms underlying the disease’s progression, particularly in the castration-resistant or metastatic stages, remain incompletely understood. The majority of recent research has concentrated on bulk RNA sequencing, which could mask the variation found in tumor microenvironments. This study aims to address this gap by integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing with weighted gene co-expression network analysis (WGCNA) to investigate the molecular mechanisms of PCa at a higher resolution. In order to further individualized treatment plans for PCa, we aim to discover important genes and signaling pathways that could be used as therapeutic targets. We first preprocessed expression profile data from prostate cancer tissue samples, selecting 9,809 high-quality cells from a dataset. Following batch correction with Harmony and dimensionality reduction with principal component analysis (PCA), we used the Louvain clustering algorithm to divide the cells into discrete subtypes. The clusters were then visualized using t-SNE. This resulted in 16 cellular subtypes categorized into five major cell types: epithelial cells, monocytes, endothelial cells, CD8 + T-cells, and fibroblasts. Analysis of receptor-ligand pairs uncovered significant interactions between monocytes and both tumor cells and endothelial cells. Applying the high-dimensional WGCNA (hdWGCNA) method to construct a gene co-expression network, we detected seven gene modules, four of which were highly expressed in tumor cell subtypes and contained 380 key genes. Combining pathway analysis, we ultimately screened six key genes: CNPY2, CPE, DPP4, IDH1, NIPSNAP3A, and WNK4. We used Cox univariate regression and least absolute shrinkage and selection operator (lasso) regression techniques to build a prognostic prediction model that included these six important genes based on clinical data gathered from PCa patients. The prognostic prediction model constructed in this study demonstrated excellent predictive performance in both the training set and an external validation set, with the high-risk group showing significantly lower overall survival (OS) than the low-risk group. Furthermore, there was a substantial correlation found between risk scores and several immune-related gene sets, chemotherapeutic drug sensitivity, and tumor immune infiltration. High- and low-risk groups exhibited significant differences in immune cell content, immune factor levels, and immune dysfunction. Further analysis revealed significant correlations between the expression levels of model genes and multiple disease-related genes. Through Gene Set Variation Analysis (GSVA) and Gene Set Enrichment Analysis (GSEA), we uncovered perturbations in multiple signaling pathways in high- and low-risk groups, potentially impacting the prognosis of PCa patients. This study uncovers key genes and signaling pathways in the prostate cancer tumor microenvironment, particularly genes such as CNPY2, CPE, DPP4, IDH1, NIPSNAP3A and WNK4, which have potential as therapeutic targets. Our findings provide new insights into personalized treatment strategies for PCa and warrant further clinical validation in the future. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15682-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779fa642ce8a3880c524211b95657271b91e6425" target='_blank'>
              Integrative analysis of molecular mechanisms in prostate cancer via single-cell RNA sequencing and weighted gene co-expression network analysis
              </a>
            </td>
          <td>
            Jing Zhai, Yizhou Wang, Yu Zhang, Wenhui Zhu, Xinyu Xu, Yu Peng, Guanxiong Ding
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19d6f72ebd8a3733c60e8ee75cfbcd9b5da5d9c6" target='_blank'>
              Predicting the regulatory impacts of noncoding variants on gene expression through epigenomic integration across tissues and single-cell landscapes.
              </a>
            </td>
          <td>
            Zhe Liu, Yihang Bao, An Gu, Weichen Song, Guan Ning Lin
          </td>
          <td>2025-09-26</td>
          <td>Nature computational science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Cancer cell evasion of therapy is a highly adaptive process that undermines the efficacy of many treatment strategies. A significant milestone in the study of these mechanisms has been the advent of pooled CRISPR knockout screens, which enable high-throughput, genome-wide interrogations of tumor dependencies and synthetic lethal interactions, advancing our understanding of how cancer cells adapt to and evade therapies. However, the utility of this approach diminishes when applied to dynamic biological contexts, where processes are transient and sensitivity to routine cell culture manipulations that introduce noise and limit meaningful discoveries. To overcome these limitations, we present RESTRICT-seq, a next-generation pooled screening methodology that restricts Cas9 nuclear activation in controlled, repeated cycles. By confining Cas9 catalytic activity to strict temporal windows, RESTRICT-seq mitigates undesired fitness penalties that routinely accumulate throughout pooled screens. When benchmarked against conventional pooled screens and standard inducible protocols, RESTRICT-seq revealed significantly fewer divergent cell clones and increased signal-to-noise ratio, overcoming a key limitation of traditional methods. Leveraging RESTRICT-seq, we conducted a comprehensive functional survey of the druggable mammalian epigenome, uncovering several elusive epigenetic drivers of treatment resistance in cutaneous squamous cell carcinoma (cSCC). This revealed PAK1 as a previously unrecognized mediator of cSCC resistance in human and mouse SCC, offering new insights into a prognostic marker and therapeutic target of high clinical significance. Our findings establish RESTRICT-seq as a powerful tool for extending the applicability of pooled CRISPR screens to dynamic and previously intractable biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ccdefb21a27947026277d5009da2011b09160aa" target='_blank'>
              RESTRICT-seq enables time-gated CRISPR screens and uncovers novel epigenetic dependencies of SCC resistance
              </a>
            </td>
          <td>
            Selahattin Can Ozcan, Dreyton G Amador, Justin Anthony Powers, Ashley G. Njiru, Zahra Ansari, Yvon Woappi
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite the development of various effective anaplastic lymphoma kinase tyrosine kinase inhibitors (ALK-TKIs), therapeutic resistance remains a major challenge. Both on-target and off-target mechanisms have been identified as key contributors to resistance. With the popularization of genetic testing and the development of precision therapies, the prognosis and survival of patients with ALK-positive non-small cell lung cancer (NSCLC) have improved. However, even with second- and third-generation ALK-TKIs, overcoming resistance remains difficult. Resistance frequently arises during approved treatments, underscoring the need for further research to elucidate the molecular events and resistance mechanisms associated with ALK-positive lung cancer. The discovery of anaplastic lymphoma kinase (ALK) rearrangement as an actionable oncogenic driver in NSCLC has established a biomarker-driven treatment paradigm for advanced disease. This article summarizes current knowledge of the mechanisms of resistance to ALK-targeted therapy in lung cancer, including both primary and acquired mechanisms, treatment strategies following resistance, recent therapeutic advances, and the impact of the immune system and tumor microenvironment. A deeper understanding of ALK-targeted therapy resistance is critical for developing new treatment strategies and may provide important insights to guide the diagnosis, treatment, and management of patients with resistant ALK+ lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880963ca5d4f45d24c1ab8fad996bba2e1305930" target='_blank'>
              New advances in understanding the mechanisms and treatment challenges of ALK-targeted therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Mengle Long, Shixuan Peng, Qingyang Wen, Zhijian Yin, Xinwen Zhang, Haoyu Tan, Yun Xu, Yongjun Wu
          </td>
          <td>2025-08-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is an aggressive and heterogeneous liver cancer with restricted therapy selections and poor diagnosis. Although there have been great advances in genomics, the molecular mechanisms essential to HCC progression are not yet fully implicit, particularly at the single-cell stage. This research utilized single-cell RNA sequencing technology to evaluate transcriptional heterogeneity, immune cell infiltration, and potential therapeutic targets in HCC. A detailed bioinformatics pipeline used in the experiment included quality control, feature selection, dimensionality reduction using Principal Component Analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), and t-distributed stochastic neighbor embedding (t-SNE), clustering, differential gene expression, pseudotime trajectory inference, and immune cell profiling with GSEA and survival analysis examining potential biomarkers of survival. Key findings include the identification of 1178 differentially expressed genes (DEGs), with macrophage infiltration contributing to immune evasion. Notably, APOE and ALB are linked to a better prognosis, while XIST and FTL are associated with poor survival. The potential drug candidates include IGMESINE in the case of SERPINA1 and PKR-A/MITZ for APOA2 in the gene-drug interaction analysis. Graph Neural Network (GNN) is used to predict drug-gene interactions and rank potential therapeutic candidates. The model shows robust predictive performance (R²: 0.9867, MSE: 0.0581) and identifies important drug candidates, such as Gadobenate Dimeglumine and Fluvastatin, and describes repurposing opportunities in network analysis, enhancing computational drug discovery for novel treatments. This research sheds new light on HCC tumor evolution, immune suppression, and the potential drug target based on the viewpoint of the importance of single-cell approaches in liver cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e827b59658da5d3c8017bf844efafdcd1183fd" target='_blank'>
              Integrating single-cell RNA sequencing and artificial intelligence for multitargeted drug design for combating resistance in liver cancer
              </a>
            </td>
          <td>
            Houhong Wang, Youyuan Yang, Junfeng Zhang, Wenli Chen, Jingyou Dai, Changquan Li, Qing Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Single base substitution (SBS) mutations, particularly C > T and T > C, are increased owing to unrepaired DNA replication errors in mismatch repair-deficient (MMRd) cancers. Excess CpG > TpG mutations have been reported in MMRd cancers defective in mismatch detection (dMutSα), but not in mismatch correction (dMutLα). Somatic CpG > TpG mutations conventionally result from unrepaired spontaneous deamination of 5’-methylcytosine throughout the cell cycle, causing T:G mismatches and signature SBS1. It has been proposed that MutSα detects those mismatches, prior to error correction by base excision repair (BER). However, other evidence appears inconsistent with that hypothesis: for example, MutSα is specifically expressed in S/G2 phases of the cell cycle, and defects in replicative DNA polymerase proofreading specifically cause excess CpG > TpG mutations in signature SBS10b. Methods We analysed mutation spectra and COSMIC mutation signatures in whole-genome sequencing data from 1803 colorectal cancers (164 dMutLα, 20 dMutSα) and 596 endometrial cancers (103 dMutLα, 9 dMutSα) from the UK 100,000 Genomes Project. We mapped each C > T mutation to its genomic features, including normal DNA methylation state, replication timing, transcription strand, and replication strand, to investigate the mechanism(s) by which these mutations arise. Results We confirmed that dMutSα tumours specifically had higher CpG > TpG burdens than dMutLα tumours. We could fully reconstitute the observed dMutSα CpG > TpG mutation spectrum by adding CpG > TpG mutations in proportion to their SBS1 activity to the dMutLα spectrum. However, other evidence indicated that the SBS1 excess in dMutSα cancers did not come from 5’-methylcytosine deamination alone: non-CpG C > T mutations were also increased in dMutSα cancers; and, in contrast to tumours deficient in BER, CpG > TpG mutations were biased to the leading DNA replication strand, at similar levels in dMutSα and dMutLα cancers, suggesting an origin in DNA replication. Other substitution mutations usually corrected by BER were not increased in dMutSα tumours. Conclusions There is a CpG > TpG and SBS1 excess specific to dMutSα MMRd tumours, consistent with previous reports, and we find a general increase in somatic C > T mutations. Contrary to some other studies, the similar leading replication strand bias in both dMutSα and dMutLα tumours indicates that at least some of the excess CpG > TpG mutations arise via DNA replication errors, and not primarily via the replication-independent deamination of 5’-methylcytosine. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01525-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f0be2946a1331e8205341cd8d8b7fbe7da7996" target='_blank'>
              Replication-associated mechanisms contribute to an increased CpG > TpG mutation burden in mismatch repair-deficient cancers
              </a>
            </td>
          <td>
            Joseph C Ward, Ignacio Soriano, Steve Thorn, Juan Fernández-Tajes, Kitty Sherwood, Güler Gül, Joost Scheffers, A. Frangou, Ben Kinnersley, Ioannis Kafetzopoulos, Duncan Sproul, Sara Galavotti, C. Palles, Andreas J. Gruber, David N. Church, Ian Tomlinson
          </td>
          <td>2025-08-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin W. Włodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="


 Platinum-based chemotherapeutic drugs induce intra- and interstrand purine crosslinks that causes DNA damage and tumor cell death. While these drugs are front-line treatments for multiple human cancers – including high-grade serous ovarian cancer (HGSOC) – platinum resistance and tumor recurrence remains a major clinical challenge. Here, we report findings from both a genome-wide CRISPRi screen and an RNA-binding protein (RBP)-focused CRISPRi screen aimed at identifying modulators of sensitivity and resistance of HGSOC cells to two widely used platinum agents, cisplatin and oxaliplatin, as well as to an microtubule-targeting taxane. Using an in vitro treatment regimen designed to mimics clinical treatment protocols, we found that loss of genes in the Fanconi anemia (FA) and homologous recombination (HR) DNA repair pathways markedly increased the killing of p53-deficient HGSOC cells following treatment with cisplatin or oxaliplatin. In contrast, knockdown of genes in the nucleotide excision repair (NER) pathway—despite its known role in repairing platinum-induced intrastrand crosslinks, particularly in G1—had a significantly lesser effect. These results suggest that platinum-induced cytotoxicity in HGSOC is primarily driven by replication stress, which is exacerbated by disruption of error-free DNA repair pathways active during the S- and G2 phases of the cell cycle. Our screens also identified several known and novel RBPs as potent sensitizers to both platinum agents and taxanes. Depletion of these RBPs resulted in substantial activation of interferon signaling, a hallmark of innate immune activation and cellular stress responses. Among them, the RNA-editing enzyme ADAR1 emerged as a key modulator of drug resistance. ADAR1 catalyzes adenosine-to-inosine RNA editing, thereby altering mRNA coding potential, secondary structure, and subcellular localization. Additionally, ADAR1 has been implicated in regulating the integrated stress response (ISR), a signaling network that enables cellular adaptation to environmental stress. As part of this stress adaptation, we observed that platinum and taxol treatment caused a translational shift from canonical cap-dependent translation mediated by eIF4G1 to a cap-independent mechanism reliant on eIF4G2, supporting selective translation of stress-responsive mRNAs. Together, our findings suggest that ADAR1-dependent suppression of interferon signaling and activation of the ISR contributes to resistance to both platinum-based chemotherapy and microtubule-targeting agents, which together comprise the current standard of care for HGSOC.



 Yi Wen Kong, Erika D. Handly, Robin A. Lu, Molly A. Bird, Jack Schaeffer, Samkyu P. Yaffe, Michael B. Yaffe. CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912428b4f14100051b63d5819642408cb6578a72" target='_blank'>
              Abstract PR009: CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2μm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The fusion of nanotechnology with gene editing promises a revolutionary strategy in combating cancer, providing the possibility of precise and focused treatments. This review examines the synergistic integration of these two potent technologies, specifically emphasising their combined effectiveness in oncological therapies. Nanotechnology offers a flexible framework for administering gene-editing tools, improving their accuracy, and reducing unintended side effects, all of which are significant obstacles in existing cancer treatments. Nanoparticles can improve the effectiveness of therapies, lower the risk of systemic toxicity, and allow the simultaneous manipulation of many genetic pathways involved in cancer growth by delivering CRISPR-Cas9 and other gene-editing systems directly to tumour sites. We conduct a thorough analysis of recent progress in this burgeoning field, emphasising significant advancements in the design of nanoparticles and gene-editing techniques that propel the development of next-generation cancer medicines. In addition, we address the present obstacles and constraints, such as the effectiveness of delivery, apprehensions over safety, and regulatory obstacles, while suggesting potential areas of future research to surmount these barriers. This study thoroughly examines the promise of nano-precision gene editing as a transformative approach to cancer treatment by incorporating findings from recent clinical trials and case studies. By highlighting recent clinical advancements and emerging innovations, this review underscores the potential of nano-precision gene editing as a groundbreaking approach in next-generation cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a93a0064afa2aa4000c185a6eb3a8730afcf15" target='_blank'>
              Harnessing Nanotechnology and Gene Editing for Cancer Therapy: A Synergistic Approach to Precision Medicine.
              </a>
            </td>
          <td>
            A. Goel, Istuti Saraswat, Sahaj Sharma, Rijul Joshi
          </td>
          <td>2025-08-01</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Located on chromosome 10, the RET gene encodes a transmembrane receptor tyrosine kinase that, when activated by a ligand, is essential for cell differentiation, proliferation, and survival. RET specifically supports the development of neuronal tissues and neural crest-derived cell types. However, RET is a proto-oncogene and mutations can lead to aggressive cancers like thyroid and lung tumours, or endocrine syndromes. Treatments for RET-driven cancers remain limited, relying on therapies that use RET-specific inhibitors. However, they show limited success in clinical settings due to adverse side effects and acquired resistance over time. To find alternative treatments, integrating the principles of synthetic dosage lethality (a genetic interaction causing a lethal effect in cancer cells) with machine learning may identify novel gene targets to complement existing therapies. In previous studies, we sought candidate genes that induce cell death when depleted in cells overexpressing oncogenic RET, but not in normal cells. A previously identified candidate, NCAM1, which is involved in neuronal cell migration, has been found to contribute to the differentiation of neural cell types. Using the SH-SY5Y neuroblastoma cell line as a model, multiple techniques and assays were performed to validate the synthetic lethal relationship between NCAM1 and RET. NCAM1 knockdown cell lines were generated using lentiviral transduction, and a live/dead cell assay was used to assess potential cytotoxic effects of gene knockdown. Additionally, a cycloheximide assay was performed to examine RET half-life by tracking its degradation over twelve hours. Over the summer, the project enabled us to reassess gene priority rankings using synthetic dosage lethal logic, characterize phenotypes of NCAM1 knockdown SH-SY5Y cells, optimize various assays under in-vitro conditions, and consolidate our understanding of NCAM1’s relationship to RET. The project thus demonstrates the value of combining bioinformatic and in-lab techniques as a productive approach to finding precise therapeutic targets against RET-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08538d4a42ece74adec0afb573a8e0619996891d" target='_blank'>
              Identifying Novel Gene Targets in RET-Driven Cancers Using Synthetic Dosage Lethality Models
              </a>
            </td>
          <td>
            Zahra Simpson-Stairs
          </td>
          <td>2025-08-29</td>
          <td>Inquiry@Queen's Undergraduate Research Conference Proceedings</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="T cell receptors (TCRs) orchestrate adaptive immunity, yet the complex, repetitive architecture of the TCR loci has impeded systematic characterization of human genetic variation in the genes encoding the TCR. Using public long-read sequencing data from 2,668 donors, we build a near-complete map of common alleles in TCR V, D, and J genes, revealing amino acid variation at almost every position within V genes. We discover pervasive evidence of natural selection on TCR genes, including balancing selection on a TRAJ gene recognizing an immunodominant influenza epitope and positive selection on a TRAV gene. We find TCR allelic polymorphism alters core functional properties of T cells, including thymic fate commitment, phenotypes in diseased tissues, and cell-surface receptor abundance. Collectively, our findings position inherited variation in TCR genes as a key axis of immunological diversity that may shape interindividual differences in immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d398484bfbdbd87bb2aee53bb4ed8e33eec07466" target='_blank'>
              TCR germline diversity reveals evidence of natural selection on variable and joining alpha chain genes
              </a>
            </td>
          <td>
            S. Mantena, Ali Akbari, Soumya Raychaudhuri
          </td>
          <td>2025-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer is still a huge problem because of the complication of genetic heterogeneity and emergence of drug resistance which leads to poor therapeutic responses. Conventional drug development is usually restricted in their capacity to discover the exact molecular target and anticipate the drug responses. This is because the integration of CRISPR-based genome editing technologies into cancer drug screening does two main services: it offers a new direction of discovering gene-drug interactions and targeting against drugs. The study has used A549, MCF-7, and HCT116 cancer cell lines to examine how the consequences of knocking out and activating certain genes (KRAS, TP53, MYC, PTEN, and EGFR) affect a cell in terms of response to chemotherapeutic agents ( d o x rubicin, paclitaxel and c is platinum ). The design of the experiment was the inclusion of drug treatments at different concentrations, cell viability assays, measurement of gene expression and quantifying the protein. The statistical tests, such as ANOVA and post-hoc testing, were used in order to determine the statistical significance of the changes. The data showed that KRAS knockout sensitized the A549 cells to doxorubicin (IC50 = 0.35 10 9 ) and TP53 knockout rendered the cells resistant (IC50 = 1.50 10 9 ). MYC CRISPR interference decreased sensitivity to paclitaxel whereas MYC activation increased it. PTEN knockout (in MCF- 7) rendered cells more susceptible to cisplatin and activation induced resistance. Pearson correlation analysis demonstrated that there were significant correlations between the change in expression of the genes and cell viability indicating the functional relevance of the CRISPR alterations. The presented study also notes that CRISPR-based functional screening can be a promising field of precision oncology and can help to understand the molecular underpinning of drug resistance and discover new targets of therapeutics against cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6621891a1e50692e473d10afd771d57415d90fde" target='_blank'>
              CRISPR-BASED GENOME EDITING TECHNOLOGIES IN CANCER RESEARCH AND THEIR APPLICATIONS IN FUNCTIONAL DRUG SCREENING
              </a>
            </td>
          <td>
            Aiman Nishat, Ahmad Ashraf, Tahir Hasan, Abrar Hussain, Muhammad Afaq, Haseeb Khaliq, Muhammad Kamran Tahir, Abul Hassan Khan, Gul Sika
          </td>
          <td>2025-08-14</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematologic malignancy originating from hematopoietic stem cells and driven by the BCR-ABL fusion oncogene. Imatinib (IM), a tyrosine kinase inhibitor, is commonly used as a frontline therapy for CML. However, some patients exhibit primary resistance or show persistent molecular evidence of disease despite treatment. Emerging studies indicate that exosome-derived microRNAs (miRNAs) play a role in mediating drug resistance and may serve as promising biomarkers for cancer diagnosis and predicting therapeutic response. This study aimed to investigate the plasma exosomal miRNA expression profiles in CML patients to identify potential biomarkers associated with IM resistance. Exosomes were isolated from plasma samples of both IM-sensitive and IM-resistant CML patients. The exosomal miRNA content was analyzed using RNA sequencing, followed by differential expression analysis, which revealed 13 upregulated and 21 downregulated miRNAs in IM-resistant patients. Subsequent bioinformatics analysis indicated significant enrichment in pathways related to autophagy and PI3K-Akt signaling. Notably, miR-451a and miR-16–2-3p were among the most significantly upregulated miRNAs in exosomes from IM-resistant individuals. Interestingly, miR-16–2-3p expression showed a strong inverse correlation with clinical laboratory results, specifically blood urea nitrogen and creatinine levels. This pilot study identified plasma exosomal miRNAs, particularly miR-451a and miR-16–2-3p, as potential biomarkers for imatinib resistance in chronic myeloid leukemia. Target gene prediction was performed to explore their regulatory roles. Despite the limited sample size, these findings enhance our understanding of drug resistance mechanisms and warrant further validation in larger cohorts to assess their clinical relevance and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ffa66d21071c796372f54b926f9da678ac08d63" target='_blank'>
              Exosomal miRNA expression profiling in patients with imatinib resistant Chronic myeloid leukemia: A pilot study
              </a>
            </td>
          <td>
            R. Navakanitworakul, Pirun Saelue, Tipparat Penglong, P. Molika, Natakorn Nokchan, Natta Tansila, H. Buncherd, Supinya Thanapongpichat, Kanitta Srinoun
          </td>
          <td>2025-08-29</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Understanding how genes influence drug responses is critical for advancing personalized cancer treatments. However, identifying these gene-drug interactions in a physiologically relevant human system remains a challenge, as it requires a model that reflects the complexity and heterogeneity among individuals. Here we show that large-scale CRISPR-based genetic screens, including knockout, interference (CRISPRi), activation (CRISPRa), and single-cell approaches, can be applied in primary human 3D gastric organoids to systematically identify genes that affect sensitivity to cisplatin. Our screens uncover genes that modulate cisplatin response. By combining CRISPR perturbations with single-cell transcriptomics, we resolve how genetic alterations interact with cisplatin at the level of individual cells and uncover an unexpected link between fucosylation and cisplatin sensitivity. We identify TAF6L as a regulator of cell recovery from cisplatin-induced cytotoxicity. These results highlight the utility of human organoid models for dissecting gene-drug interactions and offer insights into therapeutic vulnerabilities in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/306fe56d4d5b9a4de12be73a14dcdbee9c2f3137" target='_blank'>
              Large-scale CRISPR screening in primary human 3D gastric organoids enables comprehensive dissection of gene-drug interactions
              </a>
            </td>
          <td>
            Yuan-Hung Lo, Hudson T Horn, Mo-Fan Huang, Wei-Chieh Yu, Chia-Mei Young, Qing Liu, Madeline Tomaske, Martina Towers, Antonia Dominguez, M. Bassik, Dung-Fang Lee, L. S. Qi, Jonathan S. Weissman, Jin Chen, Calvin J. Kuo
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>64</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease characterized by a broad spectrum of molecular alterations that influence clinical outcomes. TP53 mutations define one of the most lethal subtypes of acute myeloid leukemia (AML), driving resistance to nearly all available treatment modalities, including venetoclax plus azacitidine (VenAza). Yet, the molecular basis of this resistance, beyond affecting transactivation of BCL-2 family genes, has remained elusive. Here, we demonstrate that VenAza treatment leads to reduced transcriptional upregulation of the p53 signaling pathway in TP53 mutant/deficient AML compared to wild-type AML. Functionally, TP53 mutant/deficient AML exhibits selective failure in apoptosis induction rather than impaired G1 arrest or senescence. Despite inhibition of pro-apoptotic BAX and selective enrichment for MCL-1 in TP53 mutant isogenic AML cells, compensatory upregulation of BIM preserved functional mitochondrial outer membrane permeabilization (MOMP). TP53 mutant primary AML tumors at baseline also had retained capacity for MOMP. Instead, TP53 mutant AML exhibited disruption in caspase-3/7 activation to evade apoptosis after VenAza therapy – decoupling the mitochondrial and executioner phases of apoptosis. Importantly, this “post-MOMP brake” is not a bystander effect but itself a driver of VenAza and chemotherapy resistance in TP53 mutant/deficient AML. This previously unrecognized mechanistic insight shifts the focus from mitochondrial priming to terminal caspase blockade in TP53 mutant AML and opens the door for urgently needed therapeutic strategies that reignite apoptosis at its execution point.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56075976a6297045972be1b8a77a5e2796abdc68" target='_blank'>
              TP53 mutations drive therapy resistance via post-mitochondrial caspase blockade
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, E. Olesinski, Fang Qi Lim, S. Jasdanwala, Yang Mi, Nicole Si, En Lin, Daniel Tan, En Liang, Nishtha Chitkara, L. Hogdal, R. C. Lindsley, A. Letai, Koji Itahana, Shruti Bhatt
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e6c9eab699f52e66672938b0f0e3b24667455ef" target='_blank'>
              From cell cycle control to cancer therapy: exploring the role of CDK1 and CDK2 in tumorigenesis.
              </a>
            </td>
          <td>
            Jitendra Gupta, Bahaa Ibrahim Saeed, A. K. Bishoyi, A. Alkhathami, Shodiyev Asliddin, Deepak Nathiya, M. Ravi Kumar, Deepak Bhanot, A. Rashed, Yasser Fakri Mustafa
          </td>
          <td>2025-08-09</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background While the RNA-binding protein HuR is implicated in individual cancers, its comprehensive diagnostic, prognostic, and immunological roles across diverse cancer types remain unexplored. Methods We performed an integrated pan-cancer analysis of HuR using public datasets. This encompassed expression profiling, survival analysis, diagnostic accuracy assessment, immune microenvironment characterization, and drug sensitivity prediction. We investigated HuR’s regulatory mechanisms through pathway correlation and differential gene expression analyses. Results HuR expression was consistently elevated across multiple cancers and correlated with poor patient prognosis. It demonstrated high diagnostic accuracy (>85%) via TMB/PD-L1 biomarkers. High HuR expression was associated with an immunosuppressive tumor microenvironment and reduced efficacy of immune checkpoint inhibitors, establishing it as a key immunoregulatory biomarker. HuR also predicted sensitivity to cell cycle inhibitors and other pathway-targeted drugs. Mechanistically, HuR drives malignancy by dysregulating core processes: cell cycle progression, immune evasion, and cellular metabolism. Conclusions Our pan-cancer analysis establishes HuR as a consistently upregulated oncogenic driver across malignancies, functioning as a potential universal biomarker for prognosis and diagnosis. Its critical roles in modulating the immune response and predicting therapeutic sensitivity highlight its importance for personalized cancer treatment strategies. HuR orchestrates tumorigenesis and malignant progression by integrally regulating vital cellular processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee36dd4154d3b1fc037a47fa32d256e8c8ba2af" target='_blank'>
              Integrated pan-cancer analysis of RNA binding protein HuR investigates its biomarker potential in prognosis, immunotherapy, and drug sensitivity
              </a>
            </td>
          <td>
            Jian Peng, Jichuan Quan, Xiaoru Wang
          </td>
          <td>2025-08-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary DNA repair mechanisms are crucial for maintanence of DNA and cell integrity. When disrupted, they can lead to increased mutation rate and contribute to the cancerogenesis. On the other hand, cancer cells that bear mutations in genes involved in DNA repair are more prone to DNA damaging insults which opens opportunities for treatments involving synthetic lethality. microRNAs are short non-coding RNAs that regulate expression by binding to gene transcripts and inducing mRNA degradation or inhibition of translation. Here, we review the miRNA-mediated dysregulation of genes involved in DNA damage response (DDR) and DNA repair pathways in ovarian cancer (OvCa), one of the deadliest female malignancy. We also discuss miRNAs, which affect response to OvCa therapy by regulating PARP1 (Poly(ADP-Ribose) Polymerase-1), a central gene of the OvCa synthetic lethality treatments. Finally, we address the limitations of miRNAs as diagnostic biomarkers and potential targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd4db6c3d00cf07b80c8f608dd0b112d9dfce41" target='_blank'>
              DNA Damage and Repair in Ovarian Cancer: Focus on MicroRNAs
              </a>
            </td>
          <td>
            Katarzyna D. Arczewska, A. Piekiełko-Witkowska
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary Meningiomas are common brain tumors, but a subset behaves aggressively and resists current treatments. This review proposes a developmentally informed framework showing that where a meningioma arises in the skull mirrors its embryologic origin (neural crest vs. mesoderm) and imprints distinct genetic, epigenetic, and immune programs that shape tumor behavior and treatment response. This lineage “stamp” helps explain why convexity and skull-base tumors differ and highlights actionable circuits, NF2/YAP–TAZ, PI3K/AKT, TRAF7-linked pathways. We integrate spatial genomics and single-cell data to map tumor location and molecular class to rational therapies, including CDK4/6, PI3K/AKT, YAP/TEAD, and epigenetic inhibitors, with emerging strategies such as synthetic-lethal combinations, CRISPR-guided target discovery, and degrader (PROTAC) approaches for high-grade and recurrent disease. Our goal is a practical roadmap that links anatomy and biology to precision treatment and trial design, improving outcomes by matching the right therapy to the right meningioma at the right site.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3274f2ff22ffc0fdf55125037e12d854830b1a18" target='_blank'>
              Spatially Encoded Oncogenesis and Transcriptional Plasticity in Meningioma: Drivers of Therapeutic Resistance and Opportunities for Targeted Intervention
              </a>
            </td>
          <td>
            Matthew A Abikenari, Amit Regev, Brandon Bergsneider, Vratko Himič, Shreyas Annagiri, Lily H Kim, Ravi Medikonda, John Choi, S. Jeyaretna, Daniel M. Fountain, Michael Lim
          </td>
          <td>2025-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee7ec76a900e6d0548a03ab74a2803cf82d9035" target='_blank'>
              Genomic Divergence Between Matched Primary and Metastatic Tumors Across Cancer Types: A Pan-Cancer Analysis of 5,692 Samples
              </a>
            </td>
          <td>
            Y. Ergun
          </td>
          <td>2025-08-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT

 Epigenetic changes are inherited but repairable genetic DNA modifications that affect gene expression regardless of the underlying sequence. Epigenetic abnormalities have been associated with a wide range of human diseases, including cancer. Malignancy can be caused by abnormal epigenetic changes that incorrectly activate oncogenes. Epigenetic marks are appealing therapeutic targets because they are reversible and susceptible to exogenous changes. Significant progress has been made in developing treatments that target epigenetic regulation in the past few years, with several pharmaceuticals being evaluated. this review describes the epigenetic drugs currently undergoing clinical trials. It also discusses the pros and cons of using them to treat cancer and the role of epigenetics in cancer diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6ed463ac635d77e490db012f7136926b14458e8" target='_blank'>
              The role of epigenetics in cancer: From pathways to the clinic
              </a>
            </td>
          <td>
            Abdulraheem A Almalki
          </td>
          <td>2025-08-01</td>
          <td>Journal of Family Medicine and Primary Care</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861fd3ffe0c8c1bb6ed63c62ce2660ccaa3e238b" target='_blank'>
              miRNA-mediated resistance mechanisms in prostate cancer: implications for targeted therapy and metastatic progression.
              </a>
            </td>
          <td>
            Mostafa M. Mostafa, M. K. A. El-Aziz, D. Ellakwa
          </td>
          <td>2025-08-29</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Background Colorectal cancer (CRC) remains a major challenge in oncology, necessitating the identification of new therapeutic targets. This study aimed to explore the CRC microenvironment and uncover potential targets using single‐cell RNA sequencing (scRNA‐seq). Methods Single‐cell RNA sequencing data from GEO (GSE164522, GSE132465, and GSE144735) were integrated and stratified based on CRC clinical stages and tumor grades. Cell clustering analysis identified distinct cellular subsets, while survival and functional enrichment analyses evaluated the prognostic significance and biological activity of differentially expressed genes. Additionally, transcription factor activity and pseudotime trajectory analyses elucidated cellular dynamics during CRC progression. Cell–cell communication analysis identified key ligand‐receptor interactions involved in immune regulation. Finally, multiplex immunofluorescence was used to validate the expression of key markers and cells in CRC tissues from patients. Results In the low differentiated colorectal adenocarcinoma (LDCA) grade of CRC, the proportion of T cells significantly increased, accompanied by the enrichment of Treg cells and the upregulation of genes associated with immunosuppression, proliferation, and differentiation. Notably, TGFβ1+ Tregs significantly expanded in LDCA, accumulating in late tumor grades with enhanced immunosuppressive capacity and self‐proliferation. Pseudotime analysis confirmed their dominance in late CRC, reinforcing immunosuppression through signal amplification and sustained expansion. Concurrently, CD8+ T cells in the LDCA grade exhibited a progressive loss of cytotoxic function, transitioning into an exhausted state while concurrently activating apoptotic pathways, leading to a profound impairment of antitumor immunity. Cell–cell communication analysis further demonstrated that TGFβ1+ Treg exhibits the strongest interactions with CD8+ T cells, with KLRB1‐CLEC2D, LGALS9‐HAVCR2, and TNFSF10‐TNFRSF10B emerging as pivotal ligand‐receptor pairs displaying significantly enhanced signaling in the LDCA grade. Conclusion Collectively, TGFβ1+ Treg may mediate CD8+ T cell dysfunction through these ligand‐receptor interactions, accelerating T cell exhaustion and apoptosis, thereby fostering a profoundly immunosuppressive tumor microenvironment that ultimately drives CRC immune evasion and malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b126f27787b7f75c899e3f95b02d5186b9c098e5" target='_blank'>
              Integrated Single‐Cell RNA‐Seq Reveals Immunosuppressive Mechanisms of Treg Cell Differentiation and Tumor Microenvironment Interactions in Colorectal Cancer
              </a>
            </td>
          <td>
            Dongwei Li, Jingya Liu, Lixian Yu, Xina Luo, Xiaomei Wu, Ronghui Chen
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

This review provides an in-depth analysis of KRASG12D, one of the most prevalent
oncogenic mutations found in pancreatic, colorectal, and lung cancers. Despite its historically regarded
"undruggable" status, the success of KRASG12C inhibitors has spurred significant interest
in targeting KRASG12D. The non-covalent inhibitor MRTX1133 has emerged as a key breakthrough,
demonstrating potent anti-tumor efficacy via the Switch II pocket, highlighting the potential
of non-covalent strategies for KRASG12D inhibition. In addition, we explore other promising
inhibitors such as BI-2852, KD2, TH-Z835, and KD-8, which leverage structural optimization and
computer-aided drug design (CADD) to enhance selectivity and binding affinity. Innovative therapeutic
approaches, including PROTAC-based SOS1 degraders (e.g., ZZ151) and combination therapies
(e.g., MRTX1133 with pan-ERBB inhibitors), are being explored to overcome the challenges
of drug resistance. Despite these advancements, issues of selectivity, chemical diversity, and overcoming
adaptive resistance remain, necessitating ongoing multidisciplinary efforts to develop novel
and effective KRASG12D-targeted therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1909e1a02307682be1e5453365c44f3844f70f59" target='_blank'>
              KRASG12D in Cancer: Structural Insights, Signaling Pathways, and Progress in Targeted Inhibitor Design
              </a>
            </td>
          <td>
            Jian Zhou, Zhou Fang, Wen Zhang, Guo-Wu Rao, Quan Zheng
          </td>
          <td>2025-08-22</td>
          <td>Mini-Reviews in Organic Chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most diagnosed cancer worldwide and the leading cause of cancer-related mortality. Despite advancements in detection and therapy, current diagnostic and treatment strategies are primarily based on evidence derived from limited cohorts, which are then applied to a diverse and genetically heterogeneous global population, often resulting in suboptimal outcomes. This one-size-fits-all model overlooks individual variability in genetic, environmental, and lifestyle factors, which have a significant impact on disease progression, therapeutic response, and overall prognosis. Disparities in incidence, progression, and treatment outcomes may arise from population-specific variations in gene expression. We investigated gene expression profiles of tumor and adjacent non-tumor tissues from people in Ibadan, Nigeria, West Africa, using transcriptomics analysis of bulk RNA sequencing data. Differentially expressed genes (DEGs) were defined by a log2 fold change (log2FC) ≥ 1.0 for upregulated genes and ≤ -1.0 for downregulated genes, with a false discovery rate (FDR) adjusted p-value < 0.05 using the Benjamini–Hochberg correction. We compared DEGs to publicly available CRC bulk RNA sequencing datasets retrieved from the NCBI-GEO database and The Cancer Genome Atlas (TCGA) database. We identified 148 upregulated and 1,769 downregulated DEGs in CRC tumor samples from the Ibadan cohort. Comparative analysis revealed substantial variation in DEG profiles across populations, with many DEGs being unique to specific datasets and no single DEG consistently shared among all datasets. Notably, individuals of African ancestry exhibited distinct DEG signatures across datasets. Among these, the tumor-associated calcium signal transducer 2 (TACSTD2) gene was the only DEG consistently upregulated across all datasets, highlighting its potential relevance in CRC among individuals of African ancestry.



 Seidu Adams, Aiden Amin, Rachel Bonacci, Udekwu Chinedum, Oluwabusola Ajagbe, Maya Wilson, Dimitri Joseph Brian Thiesen, Ellen Li, David Irabor, Akinfemi Akingboye, Olorunseun O. Ogunwobi. Tumor-associated calcium signal transducer 2 is consistently upregulated in colorectal cancer in people of African ancestry [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A030.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/556a649592f5adc67c16f721ff85fb13fe69edcc" target='_blank'>
              Abstract A030: Tumor-associated calcium signal transducer 2 is consistently upregulated in colorectal cancer in people of African ancestry
              </a>
            </td>
          <td>
            Seidu Adams, Aiden Amin, Rachel E. Bonacci, Udekwu Chinedum, Oluwabusola Ajagbe, Maya Wilson, Dimitri F. Joseph, Brian Thiesen, Ellen Li, David Irabor, A. Akingboye, O. Ogunwobi
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54525feaa0ad7b16a4eefd8a46c2d6fd3ccc01e5" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of Single-Nucleotide Variant and Copy Number Aberration Evolution from Single-Cell DNA Sequencing of Tumors.
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2025-09-15</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d108c58970b821d1c2170e04d8c4196841969e5f" target='_blank'>
              Phage therapy and its role in cancer treatment and control.
              </a>
            </td>
          <td>
            H. A. Abd El-Moaty, A. Doghish, Hebatallah Ahmed Mohamed Moustafa, Wagiha S. Elkalla, Ghadir A Sayed, Nourhan H Elshami, S. S. A. Mageed, Osama A Mohammed, Mahmoud A. Elrebehy, Kareem AlFarsi, Amr M. Abdelfatah, Aly M Waseem, Ahmed O Abdulkader, Rabab S. Hamad, Akram N Salah
          </td>
          <td>2025-09-17</td>
          <td>Folia microbiologica</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma (GB) is the most common and aggressive brain derived tumor. It often shows genetic alterations in kinase signaling pathways, such as the Pi3K/mTOR and RAS/MAPK pathways, that frequently converge onto oncogenic processes. However, it is unknown to what extend co-vulnerabilities exist within this network and which kinase drug targets are promising for GB treatment. We investigated the drug sensitivity of GB cell line models to monotherapy and synergy effects in dual combination therapy to targeting components of Pi3K/mTOR and RAS/MAPK pathways. In addition, we examined cell line drug sensitivities in relation to their individual genetic tumor driving lesions (i.e. NF1 alterations as well as transcriptomic defined GB subtypes). Synergy levels were correlated to in-lab generated phospoproteomic data. Lastly, serial or simultaneous addition of MEK and mTOR inhibitors were investigated in longitudinal experiments. Dual inhibition of MEK and mTOR resulted in synergistic effects, which associated with NF1-deficiency. Strong synergy effects was also associated with the mesenchymal subtype. Dual inhibition of MEK and mTOR led to prolonged growth inhibition in GB spheroids. In addition, sequential drug treatment resulted in similar growth inhibitory effects compared to simultaneous combination therapies. Our findings highlight the potential of dual inhibition strategies targeting multiple kinases for the treatment of GB, particularly in NF1-deficient and mesenchymal tumors, the most lethal subtype of GB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/049087437ba8fac6737126641a3cf7ff0b99a6f7" target='_blank'>
              Combined inactivation of MEK and mTOR can lead to synergistic cell death in glioblastoma models and associates with NF1-deficiency and a mesenchymal subtype.
              </a>
            </td>
          <td>
            F. Cornelissen, Yoran Broersma, R. Narayan, Rogier Dik, S. Piersma, Richard de Goeij-de Haas, T. Pham, D. Noske, W. Vandertop, Connie R. Jimenez, Bart A Westerman
          </td>
          <td>2025-08-01</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Microsatellite instability (MSI) serves as an important therapeutic and prognostic marker. Immunohistochemistry (IHC) and polymerase chain reaction (PCR) have long been regarded as the “gold standard” for MSI detection. In recent years, next-generation sequencing (NGS)-based methods, offering expanded target coverage of microsatellite (MS) loci and improved analytical performance, have gained widespread acceptance. However, discrepancies between NGS and traditional methods have been occasionally reported. In this study, we conducted a large-scale retrospective analysis of MSI results of 35,563 Chinese pan-cancer cases which underwent a NGS test in a central clinical laboratory. Our work introduced a novel algorithm for NGS-based MSI detection, examined MSI-H prevalence and value distribution across cancer types, identifies MSI-associated genes and variants, evaluates the MSI detection discordance between PCR and NGS in a pan-cancer context, and distilled 7 MS loci suitable for pan-cancer MSI detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c293d8863f4abc1b0f16238e217e4bad714bd24" target='_blank'>
              Applying next-generation sequencing to detect microsatellite instability in pan-cancer patients: a retrospective study of 35,563 Chinese cases
              </a>
            </td>
          <td>
            Yun Shang, Longfeng Yang, Pengtao Hu, Sihui Zhao, Jianbiao Xu, Yanwei Zhao, Xing Ren, Ding Zhang, Qingzhong He, Xuefei Liu
          </td>
          <td>2025-08-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4704796a9cf8557ad095d79824927f1f6b68e0" target='_blank'>
              Revealing genetic drivers of ovarian cancer and chemoresistance: insights from whole-genome CRISPR-knockout library screens.
              </a>
            </td>
          <td>
            Tali S. Skipper, Kristie-Ann Dickson, Christopher E. Denes, Matthew A. Waller, Tian Y. Du, G. Neely, N. Bowden, Alen Faiz, Deborah J. Marsh
          </td>
          <td>2025-08-28</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aim This study aims to explore the dynamic tumor microenvironment of hepatocellular carcinoma (HCC) through deep transcriptomic analysis and to identify key regulatory genes, among which MRE11 was further validated for its immunomodulatory and prognostic significance. Methods We performed Summary-data-based Mendelian Randomization (SMR) analysis to identify genes causally associated with HCC and intersected these with DNA damage repair (DDR) genes, leading to the identification of MRE11. A comprehensive evaluation of MRE11 expression in HCC was conducted using transcriptomic data analysis. We collected data from 92 HCC patient samples and validated MRE11 expression differences in HCC tissues through qPCR, immunohistochemistry, and Western blotting. Publicly available single-cell RNA sequencing (scRNA-seq) data and spatial transcriptomics were utilized to explore MRE11’s dynamic mechanisms in the tumor microenvironment (TME) of both primary and post-immunotherapy cases. We also screened for differentially expressed genes and constructed a robust HCC prognosis model using 101 machine-learning algorithms. Results Our results demonstrated that high MRE11 expression is strongly associated with poor prognosis in HCC. In the primary TME, MRE11 regulates immune responses, facilitating immune evasion. Single-cell analysis revealed significant tumor heterogeneity in MRE11 high-expression groups, particularly in macrophages and malignant cells, where MRE11 regulates immune evasion and tumor progression via the cGAS-STING pathway and HGF-MET axis. Under immunotherapy, high MRE11 expression facilitated epithelial-mesenchymal transition (EMT) and extensive remodeling of the TME. Furthermore, MRE11 dynamically enhanced macrophage regulation, exhibiting immunosuppressive and tumor-invasive features. Finally, our prognostic model exhibited strong predictive accuracy across multiple datasets. Conclusion High MRE11 expression is crucial in regulating the immune microenvironment in HCC, fostering immune evasion and driving tumor progression. MRE11 emerges as a promising biomarker for HCC diagnosis and a potential target for personalized immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03931-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60428c69be5bed6fbcd8373c293bad782a86b9ab" target='_blank'>
              Elucidating the dynamic tumor microenvironment through deep transcriptomic analysis and therapeutic implication of MRE11 expression patterns in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruiqiu Chen, Chaohui Xiao, Zizheng Wang, Guineng Zeng, Shaoming Song, Gong Zhang, Lin Zhu, Penghui Yang, Rong Liu
          </td>
          <td>2025-08-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Human endogenous retroviruses constitute a significant portion of the human genome and play complex roles in gene regulation and disease processes. However, the expression pattern and disease associations of specific retroviral loci remain pooly understood. This study examines the expression and regulatory mechanisms of these retroviral elements in immune cells. Utilizing single-cell RNA sequencing data from peripheral blood mononuclear cells, we identify 41,460 expressed retroviral loci, with 1936 showing cell type-specific expression. We further detect 3463 conditionally independent expression quantitative trait loci linked to retroviral elements, highlighting their potential role in mediating genetic variants and disease associations. Notably, these retroviral sequences associate significantly with autoimmune diseases, with specific loci demonstrating pleiotropic associations with disease-related genes. Our findings suggest that these elements are not merely genomic remnants but active participants in cellular regulation and disease progression. This work advances understanding of human-retroviral coevolution and highlights potential therapeutic targets in immune disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11df69e317870e5c63c88d7648ee572723112ba" target='_blank'>
              Single-cell eQTL mapping of human endogenous retroviruses reveals cell type-specific genetic regulation in autoimmune diseases
              </a>
            </td>
          <td>
            Fan Zhu, Yi Liu, Jiehao Lei, Xinxing Li, Zexu Jiang, Xuan Dong, Ying Gu, Young Li
          </td>
          <td>2025-08-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Breast cancer remains the most frequently diagnosed malignancy among women worldwide, with significant morbidity and mortality rates despite advances in early detection and treatment. This review provides a comprehensive overview of recent developments in breast cancer research, spanning genetic and molecular insights, novel diagnostic techniques, and emerging therapeutic strategies. The advent of next-generation sequencing and multi-omics approaches has deepened our understanding of tumor heterogeneity, revealing key genetic drivers, epigenetic regulators, and the role of cancer stem cells in disease progression. Early detection strategies have also evolved with digital breast tomosynthesis and molecular breast imaging, offering improved sensitivity and specificity. On the therapeutic front, breakthroughs in targeted treatments—including cyclin-dependent kinase 4/6 and phosphoinositide 3-kinase inhibitors, antibody–drug conjugates, and immune checkpoint inhibitors—have transformed patient outcomes. The integration of chimeric antigen receptor T-cell therapy and mRNA-based therapeutics holds great promise in overcoming treatment resistance and improving long-term survival. However, challenges such as treatment accessibility, drug resistance, and disparities in healthcare persist, particularly in low- and middle-income regions. Emerging technologies, including artificial intelligence-driven diagnostics and risk-adapted screening, are paving the way for more precise and personalized interventions. This review highlights the latest innovations and ongoing challenges in breast cancer research, emphasizing the need for continued efforts to translate scientific advancements into clinical practice to improve patient outcomes globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c3c761954bdae167d685c04a34eda5ce49f0c7" target='_blank'>
              Clinical advancements in breast cancer research: A comprehensive review
              </a>
            </td>
          <td>
            P. Puvvula, Ria Sahasra Puvvula
          </td>
          <td>2025-09-23</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="TP53 mutations drive oncogenesis and therapeutic resistance in myelodysplastic syndromes (MDSs) and acute myeloid leukemia (AML), impairing p53-regulated functions such as apoptosis, immune surveillance, and genomic stability, leading to immune evasion and metabolic reprogramming. The tumor microenvironment in TP53-mutated MDS and AML fosters leukemic progression through cytokine dysregulation, altered metabolism, and immune suppression. Current therapies, including chemotherapy and hypomethylating agents, offer limited efficacy, resulting in poor overall survival rates for these high-risk patients. However, novel therapeutic approaches provide promising avenues, including MDM2 inhibitors, p53-reactivating agents, pathway-targeted inhibitors (Hedgehog, Wnt, NF-κB), immune modulation (checkpoint inhibitors, CAR-T therapy), metabolic interventions (fatty acid metabolism, glycolysis), and gene-editing technologies (CRISPR/Cas9, base editing). This review explores the mechanisms of immune dysfunction in TP53-mutated MDS and AML while highlighting emerging therapeutic strategies, emphasizing the integration of targeted, metabolic, and immune-modulating therapies as a transformative approach to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd676eb4edce0f1928207f8f6cec1ec2c9dedb2" target='_blank'>
              TP53-mutated MDS and AML: immune dysregulation, tumor microenvironment, and emerging therapeutic strategies
              </a>
            </td>
          <td>
            Marwah Albakri
          </td>
          <td>2025-08-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Aberrant pre-mRNA splicing is a hallmark of many cancers, yet the underlying genetic mechanisms driving these alterations remain incompletely understood. While splice-altering variants can explain a subset of events, many tumor-specific splicing changes arise in the absence of nearby sequence variants. This study aims to systematically dissect the contribution of proximal splice-altering variants to tumor-specific splicing in pediatric brain tumors and to define the residual cohort likely driven by regulatory mechanisms such as mutations affecting splicing factors or the spliceosome machinery. We hypothesized that some fraction of tumor-specific splice events in pediatric brain tumors are not associated with nearby genetic variants, and that these may highlight broader disruptions to splicing regulation.



 We applied replicate Multivariate Analysis of Transcript Splicing (rMATS-turbo) to stranded total RNA-seq data (N=635) from the Pediatric Brain Tumor Atlas (PBTA). Tumor-specific single exon (SE) splicing events were defined as a change in percent spliced in (| ΔPSI |) > 0.3 between tumors and healthy brain samples sourced from GTEx (<40 years, n=2,642) and pediatric normals (n=7). Tumor-normal matched whole genome sequencing data were used to identify somatic and rare (<0.1% Allele Frequency in gnomad) germline SNVs/Indels proximal (<250nt) to rMATS-defined splice junctions.



 Across the cohort (N=635), we identified 107,827 tumor-specific SE alternative splicing events. Only 44 tumors harbored a variant proximal to the affected splice site (11 germline, 39 somatic), leaving most events without a clear cause. We are now investigating this variant-negative subgroup as a preliminary cohort. Current efforts focus on excluding structural drivers before turning to potential regulatory mechanisms, including the disruption of spliceosome components. Preliminary findings suggest some tumors show altered expression or mutations in spliceosome components, though further validation is needed.



 This dual-ended analysis of tumor-specific splicing events reveals that while a subset can be attributed to local genetic variation, the majority likely arise from broader disruptions in splicing regulation. These findings underscore the importance of integrating RNA- and DNA-based analyses to fully capture the molecular basis of splicing dysregulation in cancer. Moreover, the variant-negative cohort may serve as a rich resource to uncover novel regulatory mechanisms and potential therapeutic targets in splicing-driven pediatric brain tumors.



 Patricia J Sullivan, Ryan J Corbett, Ammar S Naqvi, Alexander Sickler, Rebecca Kaufman, Bo Zhang, Chuwei Zhong, Sharon J Diskin, Jo Lynne Rokita. Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0c6326494c3f8ecc5b90e1fc51e9fc24433d6c" target='_blank'>
              Abstract B031: Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors
              </a>
            </td>
          <td>
            Patricia J Sullivan, Ryan J. Corbett, Ammar S. Naqvi, Alexander Sickler, Rebecca S. Kaufman, Bo Zhang, Chuwei Zhong, S. Diskin, J. L. Rokita
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Leiomyosarcoma (LMS) is a rare and aggressive soft tissue sarcoma with limited therapeutic options and poor prognosis. Identifying reliable prognostic markers and therapeutic targets is critical for improving personalized treatment strategies. Methods We integrated single-cell RNA sequencing (scRNA-seq) data of LMS from the Gene Expression Omnibus (GEO) with bulk RNA-seq and clinical data from The Cancer Genome Atlas (TCGA). Malignant cells were identified using machine learning approaches, and their stemness index was calculated. Cells were stratified into high and low stemness index groups, and differential gene expression analysis was performed. Prognostic markers were identified through a sequential pipeline: univariate Cox regression to screen candidate genes, Lasso regression for feature selection, and multivariate Cox regression for model construction and survival analysis. Results Cells with a high stemness index exhibited a more complex tumor immune microenvironment (TIME) and enhanced intercellular interactions compared to those with a low stemness index. Differential expression analysis identified genes distinguishing high versus low stemness cells. Through the regression pipeline, six prognostic markers were identified: BOP1, CTBP1, DSE, PMSD10, SRPK1, and HACD4. These markers were significantly associated with tumor cell proliferation and patient survival outcomes. Discussion Our findings suggest that stemness-related heterogeneity in LMS shapes the tumor immune microenvironment and contributes to disease progression. The six identified prognostic markers not only provide insights into the molecular mechanisms of LMS but also represent potential therapeutic targets for personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8914a51d2e0d775a0ec016871e9455ef5edac1b" target='_blank'>
              Integrated single-cell and bulk RNA-seq analysis reveals prognostic stemness genes in leiomyosarcoma
              </a>
            </td>
          <td>
            Aiju Lou, Yu Cai, Tuquan Zheng, Long Zhang, Yupeng Hu, Huachi Li, Lang Ying
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) are a class of functional transcripts that are not translated into proteins and primarily include microRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs). In recent years, as therapeutic challenges such as tumor heterogeneity and drug resistance have become increasingly prominent, ncRNAs have emerged as pivotal targets in cancer mechanistic research and precision intervention strategies due to their central roles in tumor initiation, progression, metastasis, and therapeutic resistance. Given the limitations of conventional treatments, inadequate early detection methods, and significant inter-individual variability, elucidating the regulatory functions of ncRNA networks have become an urgent imperative for advancing cancer diagnosis and treatment. This review systematically integrates the regulatory mechanisms of miRNAs, lncRNAs, and circRNAs in key oncogenic processes from a pan-cancer perspective, including cell proliferation, cell cycle control, apoptosis evasion, metastatic activation, and resistance reprogramming, while highlighting their hub-like roles in multi-pathway crosstalk,the application of ncRNA in cancer diagnosis and its role in treatment and prognosis. Furthermore, future research must strive for breakthroughs in areas such as the optimization of nanodelivery systems, AI-driven target identification, and dynamic multi-omics integration, with the ultimate goal of achieving the systematic translation of ncRNAs into actionable strategies for personalized precision medicine-transforming "functional transcriptional regulation" into "targetable therapeutic intervention."">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba1326eb3e6789f0e0413308a504eb2eb558493b" target='_blank'>
              Regulatory mechanisms and emerging diagnostic and therapeutic opportunities of non-coding RNAs in tumorigenesis: a pan-cancer perspective.
              </a>
            </td>
          <td>
            D. Sandai, Zengkan Du, Haoling Zhang, Qi Sun
          </td>
          <td>2025-09-16</td>
          <td>Critical reviews in clinical laboratory sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Randomized trials in adult patients have shown that rituximab, a monoclonal antibody targeting CD20 (MS4A1 gene), can improve treatment response in de novo Acute Lymphoblastic Leukemia (ALL). These findings have led to clinical trials incorporating rituximab into first-line chemotherapy for pediatric patients with high- or intermediate-risk CD20+ ALL. While immunotherapy in this context might improve survival, resistance to antibody-based therapies remains a major challenge, and its interaction with chemotherapy has not been studied in large cohorts of childhood ALL (cALL). Given the potential toxicities of immunotherapies, identifying predictive markers is critical for precision medicine. We hypothesized that the expression and relative abundance of MS4A1 isoforms influence the response of leukemic cells to rituximab in combination with standard chemotherapy. To test this, we studied MS4A1 transcriptional variability at cALL diagnosis using short-read (Illumina, n=31) and long-read (Oxford Nanopore Technologies ONT, n=2) RNA-seq. We obtained bone marrow aspirates of patients enrolled in the ALLIC-GATLA 2010 clinical protocol. ONT RNA-seq was performed at the Genomics Center of Facultad de Ciencias Exactas y Naturales, Universidad de Buenos Aires, while Illumina RNA-seq was conducted at Macrogen, Korea. We used Salmon, FLAIR and IsoQuant to annotate and quantify known and novel MS4A1 transcripts. Our results showed high heterogeneity among patients, both in transcriptional diversity and total transcript abundance. Notably, the MS4A1-204 isoform, which lacks the rituximab-binding domain, was expressed at diagnosis in >50% of patients. Isoform quantification differed significantly between ONT and Illumina, despite similar relative sequencing coverage of exons. Illumina predicted similar proportions of isoforms 201, 212, and 206, whereas ONT predicted dominance of the 201 isoform. Nevertheless, despite ONT sequencing depth was 6 to 11-fold lower than Illumina, molecular subtype predictions were concordant between both platforms. The discovery of novel MS4A1 transcripts using FLAIR and IsoQuant in ONT-sequenced samples showed differences in transcript identities. Structural analysis of two novel isoforms suggested the coding of putative proteins lacking extracellular or transmembrane domains, which are important for rituximab binding or may interfere with protein function. In conclusion, we successfully implemented ONT transcriptome sequencing in Argentina for ALL molecular subtyping, isoform annotation, quantification and novel transcript discovery. However, we detected high variability between sequencing platforms and bioinformatic tools, highlighting the need for further studies to improve reproducibility. These results are crucial for designing prospective studies evaluating patients treated with rituximab plus chemotherapy, currently underway in a multicentric clinical trial in Argentina. Additionally, this analysis could be extended to whole-transcriptome isoform diversity, complementing the characterization of MS4A1.



 Maria Sol Ruiz, Lucila Viappiani, Daniel Avendaño, Ignacio Gomez Mercado, Marina L Ingravidi, Geraldine Gueron, Javier Cotignola. Characterization of CD20 transcriptional diversity using short- and long-read sequencing in pediatric acute lymphoblastic leukemia: potential implications for rituximab plus chemotherapy treatment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9570d8b058543a8f0614390dc61e70e4dcf1f779" target='_blank'>
              Abstract B003: Characterization of CD20 transcriptional diversity using short- and long-read sequencing in pediatric acute lymphoblastic leukemia: potential implications for rituximab plus chemotherapy treatment
              </a>
            </td>
          <td>
            Maria Sol Ruiz, Lucila Viappiani, Daniel Avendaño, Ignacio Gomez Mercado, Marina L. Ingravidi, G. Gueron, J. Cotignola
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Despite advances in systemic therapeutic approaches, metastatic colorectal cancer (mCRC) patients harboring BRAF or RAS mutations have poor outcomes. Cancer stem cells (CSCs) play central roles in drug resistance and CRC recurrence. Therefore, targeting the epigenetic mechanisms that sustain CSC properties is a promising therapeutic approach. In this study, we report the efficacy of a treatment strategy with the potential to overcome chemotherapy resistance that involves administering the well-known antiepileptic drug and epigenetic agent valproic acid (VPA) and the standard chemotherapy regimen of oxaliplatin/fluoropyrimidine to wild-type CSCs and CSCs with BRAF and RAS mutations in enriched primary spheroid cultures. Notably, we demonstrated that VPA plus chemotherapy was more effective than other epigenetic drug-chemotherapy combinations by inhibiting cell proliferation and clonogenic growth and by inducing apoptosis and DNA damage. Mechanistically, proteomic analysis demonstrated that VPA induced CSC differentiation through the critical target of VPA, β-Catenin. Indeed, VPA promoted the proteasome-dependent degradation of β-Catenin by enhancing its binding to the E2 ubiquitin-conjugating enzyme UBE2a, leading to marked reductions in nuclear and cytoplasmic β-Catenin levels and subsequently decreasing β-Catenin/TCF-LEF target promoter activation. These effects were confirmed in three in vivo CRC xenograft models, including a syngeneic CT26 immunocompetent mouse model, where VPA combined with oxaliplatin/capecitabine chemotherapy and anti-VEGF therapy, a standard first-line treatment for mCRC, significantly suppressed tumor growth and prolonged survival with minimal toxicity. Proteomic analysis of tumor tissues from in vivo CRC models confirmed the VPA-mediated downregulation of CSC markers and β-Catenin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcea9839b49b167a072f5098d6aef8221a661f2c" target='_blank'>
              Valproic acid improves the efficacy of oxaliplatin/fluoropyrimidine-based chemotherapy by targeting cancer stem cell via β-Catenin modulation in colorectal cancer
              </a>
            </td>
          <td>
            M. Roca, Rita Lombardi, Cristina Testa, Federica Iannelli, Laura Grumetti, T. Moccia, Veronica Barile, L. Addi, Domenico Memoli, A. Leone, S. Di Franco, G. Stassi, A. Avallone, F. Bruzzese, B. Pucci, Alfredo Budillon, E. Di Gennaro
          </td>
          <td>2025-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 The androgen receptor (AR) plays a critical role in the development and progression of prostate cancer by regulating key cellular processes such as cell proliferation and apoptosis. Although traditional AR-targeted therapies have shown initial success, acquired resistance remains a significant clinical challenge, often driven by AR alterations and somatic gene mutations associated with homologous recombination deficiency (HRD). Approximately 20% of advanced prostate cancer cases exhibit HRD, resulting in substantial genomic instability and complicating treatment. Fortunately, Food and Drug Administration–approved poly(ADP-ribose) polymerase inhibitors, including olaparib and rucaparib, exploit synthetic lethality to target prostate cancer with HRD, and additional drugs targeting DNA damage response (DDR) proteins are under development. Emerging evidence suggests that AR activity enhances DDR gene expression, with multiple DDR proteins localized near androgen-regulated regions, highlighting a close interaction between AR and DDR pathways. Consequently, recent preclinical and clinical studies have investigated combining AR-targeted therapies with treatments that induce DNA damage, such as radiation therapy, or inhibit DNA repair mechanisms. This review discusses AR's role in cellular processes, the interplay between AR and DDR, and recent advances in prostate cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bcf547c4cdb4775dcc9944086243c67a7115b7" target='_blank'>
              The interconnection between AR and DNA damage response pathways in prostate cancer
              </a>
            </td>
          <td>
            Mallory Sands, Sam Adams, Jihaeng Lee, Michael Li, Mae Wang, Thomas Walsh, Leo Leon, Adriana Zablah, Matthew Haerens, Zhichao Liu, Jianneng Li
          </td>
          <td>2025-08-19</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c0e77a0dc71f8f861306d03d6b2410b93243bdc" target='_blank'>
              Next-generation T cell immunotherapies engineered with CRISPR base and prime editing: challenges and opportunities.
              </a>
            </td>
          <td>
            Karl Petri, E. D’Ippolito, A. Künkele, Ulrike Köhl, Dirk H. Busch, H. Einsele, M. Hudecek
          </td>
          <td>2025-09-19</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="


 The access to next-generation sequencing, primarily performed for therapeutic purposes, has considerably increased in the real-time clinical practice in pediatric malignancies. Using germline DNA as a matched reference, this approach is also a unique opportunity to detect alterations in medically actionable genes (named "additional data"). The main objective of this project was to estimate the frequency of such genetic predisposition discoveries among children and adolescents with relapsing or refractory cancer.



 Among 791 children included in MAPPYACTS “MoleculAr Profiling for Pediatric and Young Adult Cancer Treatment Stratification” study, germline DNA exome sequencing was available for 674 patients with many distinct cancer types. Informed consent was obtained to disclose genetic findings if an actionable predisposition was detected. A multidisciplinary germline molecular board established the list of genes to be looked at, including 184 cancer predisposition genes with consensual surveillance guidelines, and 49 non cancer genes belonging to the ACMG Secondary Findings v3.1 list. Bioinformatic analysis included two different indel variant calling (Varscan and Haplotypecaller), quality controls and filters to ensure high-confidence variant calls, ClinVar annotations for pathological classification and in-silico function assessment by computational prediction (with several tools including CADD, SpliceAI and SPiP). Expert biologists have evaluated the pathogenicity of genetic variations with the use of genetic databases, computational predictions and medical literature.



 Among 184 cancer genes, 16 607 genetic variants have emerged. Twenty-five per cent of them were classified as unknown significance corresponding to a median of 7 (0;47) variants/patient and 8% of variants were retained as likely pathogenic or pathogenic variant (LPV/PV). Thus, 132 patients (19.6%) carried one LPV/PV variant among 53 cancer genes. According to inheritance patterns (we retained heterozygous variant for dominant disease and only biallelic events for recessive disease), genetic counselling was recommended for 58 patients (8.6%). Only two patients had two different cancer predisposition. The most frequently involved genes were TP53 (n=16), DICER1 (n=5), NF1 (n=4) and BRCA1 (n=4). Those identified genetic predispositions to cancer corresponded to the expected tumor spectrum in 50% of cases and were previously known in 38% of families. Furthermore, 10 patients (1.5%) were germline carriers of LPV/PV in genes involved in other genetic conditions (n=6 for cardiopathy; n=2 familial hypercholesterolemia; n=2 transthyretin amyloidosis).



 A specific evaluation of exome sequenced for therapeutic purposes provides germline information that could be actionable for 10.1% of families to improve cancer prevention or manage some other genetic condition. The psychological impact induced by the return of secondary findings to families will be investigated.



 Tiphaine Adam de Beaumais, Yahia Adnani, Léa Guerrini-Rousseau, Samuel Abbou, Cécile Acquaviva-Bourdain, Pablo Berlanga, Adeline Bonnard, Gaelle Bougeard, Franck Bourdeaut, Nelly Burnichon, Sandrine Caputo, Alain Carrié, Olivier Caron, Hélène Cavé, Albain Chansavang, Nadège Corradini, Sophie Cotteret, Philippe Denizeau, Alice Fievet, Mathilde Filser, Marion Gauthier-Villars, Birgit Geoerger, Nadim Hamzaoui, Edwige Kasper, Florence Kyndt, Ludovic Lacroix, Jessica Le Gall, Julien Masliah-Planchon, Laurence Pacot, Cécile Pagan, Mélanie Pagès, Béatrice Parfait, Eric Pasmant, Gaelle Pierron, Pascale Richard, Nathalie Roux-Buisson, Cécile Saint-Martin, Hela Sassi, Gudrun Schleiermacher, Renaud Touraine, Nancy Uhrhammer, Dominique Vidaud, Laurence Brugières, Gilles Vassal, Etienne Rouleau, Yoann Vial, Lisa Golmard, Odile Cabaret. Genetic predisposition discoveries of clinical utility by exome sequencing performed for therapeutic purposes in children with relapsing or refractory cancer in the MAPPYACTS study [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c04f956567ca633c6e7649afb07bcfab43f50a08" target='_blank'>
              Abstract A006: Genetic predisposition discoveries of clinical utility by exome sequencing performed for therapeutic purposes in children with relapsing or refractory cancer in the MAPPYACTS study
              </a>
            </td>
          <td>
            Tiphaine Adam de Beaumais, Yahia Adnani, L. Guerrini-Rousseau, S. Abbou, Cécile Acquaviva-Bourdain, P. Berlanga, A. Bonnard, G. Bougeard, Franck Bourdeaut, Nelly Burnichon, Sandrine Caputo, Alain Carrié, Olivier Caron, Hélène Cavé, A. Chansavang, Nadége Corradini, S. Cotteret, Philippe Denizeau, Alice Fiévet, M. Filser, M. Gauthier‐Villars, Birgit Geoerger, N. Hamzaoui, E. Kasper, Florence Kyndt, Ludovic Lacroix, J. Le Gall, J. Masliah-Planchon, Laurence Pacot, Cécile Pagan, Mélanie Pagès, B. Parfait, Eric Pasmant, G. Pierron, Pascale Richard, N. Roux-Buisson, C. Saint-Martin, Hela Sassi, G. Schleiermacher, R. Touraine, N. Uhrhammer, D. Vidaud, L. Brugières, Gilles Vassal, E. Rouleau, Y. Vial, L. Golmard, O. Cabaret
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study explores the molecular impact of cisplatin, Actemra (tocilizumab), remdesivir, and SARS-CoV-2 infection on colorectal cancer (CRC) SW-480 cells using RNA sequencing (RNA-Seq). Differential gene expression analysis revealed treatment-specific transcriptional changes. Cisplatin led to the downregulation of genes involved in lipid metabolism and focal adhesion, while remdesivir upregulated chromatin remodeling pathways. SARS-CoV-2 infection altered cytokine signaling, particularly IL-17 and TNF-α. Principal component and hierarchical clustering analyses confirmed distinct gene expression profiles across treatments. ELISA assays validated the reduction of ACE2 and CD147 protein levels following drug treatment. MTT assays demonstrated remdesivir’s cytotoxicity at high doses. These findings highlight potential drug-gene interactions and suggest that combining transcriptomic profiling with functional assays may guide personalized therapeutic strategies for CRC patients, especially those co-infected with SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eae066da47093f217085c9697d40cc942624f2a" target='_blank'>
              RNA sequencing reveals transcriptional signatures of drug response and SARS-CoV-2 interaction in colorectal cancer cells
              </a>
            </td>
          <td>
            Alaa K. Hameed, Ifad K. Abd Al-Shibly, Rana A. Ghaleb
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gastric cancer (GC) remains one of the leading causes of cancer-related mortality worldwide, necessitating innovative approaches for its diagnosis and treatment. Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9), a revolutionary gene-editing technology, has emerged as a powerful tool for unraveling the molecular mechanisms underlying GC and for advancing precision medicine strategies. This review explores the current applications of CRISPR/Cas9 in GC research, including the identification of oncogenes and tumor suppressors, modeling tumor microenvironment interactions, and developing gene-based therapies. We highlight recent breakthroughs in genome editing that have enhanced our understanding of GC pathogenesis and resistance mechanisms to conventional therapies. Additionally, we discuss the potential of CRISPR/Cas9 for therapeutic gene editing in GC, addressing challenges such as off-target effects, delivery methods, and ethical considerations. By summarizing the progress and limitations of CRISPR/Cas9 in GC, this review aims to provide a comprehensive perspective on how this transformative technology could shape future strategies for the prevention, diagnosis, and treatment of GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb38e84cdf12c7752ea82097cec6e46d413aca2" target='_blank'>
              CRISPR/Cas9 gene editing in gastric cancer: Mechanisms, advances, and therapeutic potential
              </a>
            </td>
          <td>
            Grigorios Christodoulidis, Dimitra Bartzi, Kyriaki Tsagkidou, Alexandra Dimaki, Lydia Lazaridou, Kalliopi Vakalou, K. Koumarelas, Dimitrios Schizas
          </td>
          <td>2025-09-22</td>
          <td>World Journal of Gastrointestinal Pathophysiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immunotherapy has emerged as a promising treatment option for triple-negative breast cancer (TNBC); however, the pronounced heterogeneity of the tumor immune microenvironment significantly hinders the prediction of therapeutic efficacy, with effective biomarkers also remaining limited. This study utilizes single-cell RNA sequencing (scRNA-seq) and transcriptome analysis to dissect the immune landscapes of TNBC, emphasizing CD8 effector T cells (CD8Teff) as potential predictors of immune checkpoint inhibitors (ICIs) treatment response. Immune profiling revealed that CD8Teff cells were predominantly enriched in "hot" tumors strongly correlating with improved progression-free and overall survival. Notably, the cytokine C-X-C Motif Chemokine Ligand 13 (CXCL13) emerged as a key regulator, with its high expression marking an immune-active tumor microenvironment favorable to ICI efficacy. Comprehensive analysis uncovered critical interactions between CD8Teff cells and other immune and stromal components, supporting the formation of an immunologically active tumor microenvironment (TME) conducive to tumor control. Additionally, metabolic reprogramming of CD8Teff cells in responsive tumors highlighted pathways that may influence ICIs efficacy. CD8Teff cells significantly influence the TME and predict ICI success in TNBC, with CXCL13 as a pivotal modulator. A pathology-based artificial intelligence model for CD8Teff recognition was developed, achieving an area under the curve (AUC) of 0.823 in the training cohort and 0.805 in the validation cohort. This study provides an in-depth characterization of the TNBC immune microenvironment, identifying CD8Teff functionality, CXCL13 signaling, and immune-metabolic pathways as critical determinants of immunotherapy success. These findings support the development of personalized treatment strategies based on immune profiling to enhance therapeutic efficacy in TNBC. Supplementary Information The online version contains supplementary material available at 10.1186/s43556-025-00306-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bfd680374201a021e9ef78818435e2a0634a514" target='_blank'>
              Single-cell RNA sequencing identifies CD8Teff cell activation as a predictive biomarker in triple-negative breast cancer immunotherapy
              </a>
            </td>
          <td>
            L. Mao, Zebang Zhang, Yongjian Chen, Qing Peng, Zhenjun Huang, Wenhao Ouyang, D. Zeng, W. Ren, Zifan He, Tang Li, Zehua Wang, Ruichong Lin, Jianli Zhao, Jiannan Wu, Herui Yao, Yunfang Yu
          </td>
          <td>2025-09-19</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is the most common type of leukemia in Western populations and remains incurable despite significant therapeutic advancements. Over the past decade, the treatment landscape has evolved from traditional chemoimmunotherapy to targeted oral agents, including Bruton’s tyrosine kinase inhibitors (BTKis) and BCL2 inhibitors (BCL2is), which have demonstrated superior efficacy and tolerability, especially in elderly patients. Venetoclax, a BCL2i, induces apoptosis in CLL cells through selective inhibition of the anti-apoptotic BCL2 protein, while BTKis, such as ibrutinib and its next-generation analogs, disrupt B-cell receptor signaling critical to CLL cell survival. However, resistance to both drug classes has emerged, including mutations in BTK and BCL2, prompting the exploration of novel therapeutic strategies. This review outlines the molecular basis and clinical implications of these resistance mechanisms, as well as emerging therapeutic solutions, including non-covalent BTKis like pirtobrutinib and BTK-targeting PROTAC degraders such as BGB-16673 and NX-2127. Additionally, we discuss promising combination therapies incorporating BTKis, BCL2is, and anti-CD20 monoclonal antibodies. Finally, we highlight the growing role of measurable residual disease (MRD) as a biomarker to guide treatment duration and evaluate therapeutic success. As resistance mechanisms continue to emerge, tailoring therapy based on underlying biology will be critical to sustaining disease control and enhancing outcomes in patients with CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0956883756565c425ce66e36bee2fa5445ca9e94" target='_blank'>
              Resistance Mutations in CLL: Genetic Mechanisms Shaping the Future of Targeted Therapy
              </a>
            </td>
          <td>
            Samantha Sekeres, Erica N. Lamkin, Eduardo Bravo, Allison Cool, Justin Taylor
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite advances in medicine, cancer remains one of the foremost global health concerns. Conventional treatments like surgery, radiotherapy, and chemotherapy have advanced with the emergence of targeted and immunotherapy approaches. However, therapeutic resistance and relapse remain major barriers to long-term success in cancer treatment, often driven by cancer stem cells (CSCs). These rare, resilient cells can survive therapy and drive tumour regrowth, urging deeper investigation into the mechanisms underlying their persistence. CSCs express ion channels typical of excitable tissues, which, beyond electrophysiology, critically regulate CSC fate. However, the underlying regulatory mechanisms of these channels in CSCs remain largely unexplored and poorly understood. Nevertheless, the therapeutic potential of targeting CSC ion channels is immense, as it offers a powerful strategy to disrupt vital signalling pathways involved in numerous pathological conditions. In this review, we explore the diverse repertoire of ion channels expressed in CSCs and highlight recent mechanistic insights into how these channels modulate CSC behaviours, dynamics, and functions. We present a concise overview of ion channel-mediated CSC regulation, emphasizing their potential as novel diagnostic markers and therapeutic targets, and identifying key areas for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74bc88d91ef29f0fa1f654c524e84092e0b4e373" target='_blank'>
              Deciphering the Role of Functional Ion Channels in Cancer Stem Cells (CSCs) and Their Therapeutic Implications
              </a>
            </td>
          <td>
            Krishna Samanta, Gali Sri Venkata Sai Rishma Reddy, Neeraj Kumar Sharma, Pulak Kar
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c98b6aa7585e4608adcb87c526d2b9ca18744b8" target='_blank'>
              Missense but mis-spliced: germline TP53 variant c.671A>C (p.E224A) and the path from uncertainty to pathogenicity
              </a>
            </td>
          <td>
            Irena Velkova, Serena Cappato, Daniela Rivera, Ferruccio Romano, Deborah Schonneger, Renata Bocciardi, P. Hainaut, P. Marco, Viviana Gismondi, Gabriella Cirmena, Ludovica Menta, M. Ognibene, Alberto Garaventa, Carla Manzitti, Samuele Brugnara, Y. Ciribilli, Alessandra Bisio, Elisa Marcaccini, Paolo Malatesta, F. Faravelli, P. Menichini, P. Monti, V. Capra
          </td>
          <td>2025-08-01</td>
          <td>None</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Ultra-accurate detection of rare somatic mutations is critical for understanding mutational processes in human disease, aging, and environmental exposures, yet current methods are limited by error rates, restricted genome coverage, and high DNA input. We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA. Benchmarking in human sperm estimates a UDSeq error rate of ~2.5x10^-9 per base pair. UDSeq captures mutational signatures from heterogeneous populations without clonal expansion, reproduces exposure-specific patterns in cell lines and rodent models, and enables cross-species profiling. Compared with prior duplex methods, UDSeq yields up to fourfold more usable duplex molecules, improves library conversion, and remains cost-effective. We include a step-by-step protocol with quality-control checkpoints for fragment size, ligation yield, library conversion, and duplication rate. UDSeq provides a scalable, low-input platform for accurate profiling of somatic mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25aa13b255931a4c667fbbd3b80aa9966be011" target='_blank'>
              A Universal Duplex Sequencing Approach for Accurate Detection of Somatic Mutations
              </a>
            </td>
          <td>
            Shuvro P. Nandi, Yuhe Cheng, Shams Al-azzam, Safa Saeed, Audrey Kristin, Nadia Sunico, Isabella Stuewe, Zichen Jiang, Luka Culibrk, Maria Zhivagui, Xiaoxu Yang, Rachel Wise, Foster C. Jacobs, Bérénice Chavanel, M. Korenjak, Mia Petljak, S. Balbo, Laurie G Hudson, Ke Jian Liu, Jiri Zavadil, Joseph G Gleeson, Ludmil B. Alexandrov
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Increasing evidence underscores the driving role of coding and non-coding variants in cancer development. Analyzing gene sets in biological processes offers deeper insights into the molecular mechanisms of carcinogenesis. Here, we developed geMER to identify candidate driver genes genome-wide by detecting mutation enrichment regions within coding and non-coding elements. We subsequently designed a pipeline to identify a core driver gene set (CDGS) that broadly promotes carcinogenesis across multiple cancers. CDGS comprising 25 genes for 25 cancers displayed instability in DNA aberrations. Variants within the TTN enrichment region may influence the folding of the I-set domain by altering local polarity or side-chain chemistry properties of amino acids, potentially disrupting its antigen-binding capacity in LUAD. Multi-omics analysis revealed that APOB emerged as a candidate oncogene in LIHC, whose genetic alterations within the enrichment region may activate key TFs, upregulate DNA methylation levels, modulate critical histone modifications, and enhance transcriptional activity in the HepG2 and A549 cell lines compared to Panc1. Additionally, CDGS mutation status was an independent prognostic factor for the pan-cancer cohort. High-risk patients tended to develop an immunosuppressive microenvironment and demonstrated a higher likelihood of responding to ICI therapy. Finally, we provided a user-friendly web interface to explore candidate driver genes using geMER (http://bio-bigdata.hrbmu.edu.cn/geMER/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe1ede7baa4480b19bb1878aa07140d59ffd789" target='_blank'>
              A core driver gene set identified based on geMER reveals its potential driver mechanism in pan-cancer
              </a>
            </td>
          <td>
            Jing Gan, Yuncong Wang, Zhuoran Shi, Haoyu Hu, Manyi Xu, Xinrong Li, Wenbo Dong, Jiaheng He, Yusen Zhao, Yakun Zhang, Yue Sun, Caiyu Zhang, Qianyi Lu, Shangwei Ning, Yan Jin, Hui Zhi
          </td>
          <td>2025-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="BACKGROUND
Ageing is a general, intrinsic and progressively deleterious process that affects all cells, tissues and organs albeit at different extent and rate in each individual. The complexity and universality of its phenotypic manifestations suggest a multifactorial origin. The autosomal recessive disorder Werner syndrome likely represents a segmental progeroid disorder since patients show several, but not all phenotypes of premature ageing.


SUMMARY
Proliferative senescence of diploid cells in culture provided a model system in which ageing can be studied experimentally. Cultures of cells from patients with Werner syndrome experienced an extreme form of proliferative senescence and a clonal succession of translocations, known as variegated translocation mosaicism. In addition, Werner syndrome cells showed spontaneous deletion formation and a prolongation of and arrest in the S phase of the cell cycle. The WRN protein harbors a helicase, an exonuclease and a RecQ interaction domain. With the latter the WRN protein may interact with NBS1, RPA, MRE11, TREX1, MUTYH, POT1, TRF1, FEN-1, PAPRP-1, p97/VCP, TRF2, DNA polymerase(beta), Ku76/80, EXO-1, NEIL1, and p53, which are key to DNA damage response pathways including canonical NHEJ, homologous recombination, base excision repair and telomere maintenance. The WRN exonuclease domain is a target of WRNIP1 binding, which links WRN to resolution of stalled replication due to collision with transcription and the ATM-mediated cell cycle checkpoint. . Patients with an incomplete complement of Werner syndrome phenotypes, called atypical Werner syndrome patients, were found to carry variants in LMNA, POLD1, SPRTN, MDM2, CTC1, SAMHD1.


KEY MESSAGES
These findings broaden the genotypic landscape and the phenotypic spectrum of Werner syndrome. In this review potential molecular mechanisms underlying genomic instability in Werner syndrome, including chromothripsis due to asynchronous S phase traverse and telomere crises followed by bridge fusion breakage cycles are discussed. The participation of WRN in multiple gene networks is consistent with the multifactorial nature of ageing in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c57a230c575d7a7f034afedf23d1a5bec7bbac" target='_blank'>
              Molecular mechanisms of proliferative senescence and genomic instability in Werner syndrome and the WRN gene network.
              </a>
            </td>
          <td>
            Martin Poot
          </td>
          <td>2025-09-19</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Tumor-associated macrophages (TAMs) are a type of tumor-infiltrating immune cell that play a crucial role in tumor progression. However, the roles of TAMs and their regulatory mechanisms in lung adenocarcinoma (LUAD) remain poorly understood. Therefore, we aimed to develop a novel TAM-related prognostic signature to predict survival outcomes and constructed a lncRNA-miRNA-mRNA network based on these genes. Methods Transcriptomic data, clinical data, and single-cell RNA-sequencing (scRNA-seq) data were obtained from LUAD patients were obtained from the Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases. Differentially expressed-lncRNAs (DE-lncRNAs), miRNAs (DE-miRNAs), and mRNAs (DEGs) were identified in LUAD. Differentially expressed TAM-related genes (DE-TAMGs) were selected and used to construct prognostic signatures. A TAM-related risk score was calculated, and patients were stratified into high- and low-risk groups based on the median risk score. Then, biological functions, immune characteristics, and responses to immunotherapy and chemotherapy were assessed across the risk groups. A TAM-related lncRNA-miRNA-mRNA network was constructed based on DE-lncRNAs, DE-miRNAs, and TAM-related signatures. Quantitative polymerase chain reaction (qPCR) was used to validate the expression of TAM-related genes, and scRNA-seq analysis was used to examine cell-type-specific expression of these genes. Results A total of 316 DE-lncRNAs, 162 DE-miRNAs, and 2,601 DEGs were screened in LUAD. Among these, 147 DE-TAMGs were selected. KLF4, GAPDH PDGFB, TIMP1, CD74, and CCL20 were identified as the key prognostic markers in LUAD. Patients were divided into high- and low-risk groups based on the median risk score. Enrichment analysis revealed several cancer-related pathways associated with the high-risk group, and significant differences in terms of immune cell infiltration, HLA-related gene expression, immune checkpoints expression, and therapeutic responses were observed between high- and low-risk groups. We also constructed a lncRNA-miRNA-mRNA network, which included 36 DE-lncRNAs, 23 shared miRNA, and four TAMGs (PDGFB, CD74, KLF4, and CCL20). The qPCR results indicated the increased expression of PDGFB, CD74, and KLF4 but decreased expression of CCL20 in LUAD tumor tissues compared with adjacent normal tissues. scRNA-seq analysis revealed that CD74, KLF4, CCL20, PDGFB were specifically expressed in macrophages. Conclusions In conclusion, we identify the TAM-related prognostic signature that predicts the survival outcome in patients with LUAD. This signature may offer a novel effective therapeutic strategy for LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2589968144872dc461581884ea2b60391b27fe7" target='_blank'>
              Integrative analyses of single-cell and bulk RNA sequencing to construct the tumor-associated macrophage-related prognostic signature in lung adenocarcinoma
              </a>
            </td>
          <td>
            Chaoqun Yu, Jun Chen, Jianming Deng, Hui Li, Qianru Zhuang, Bingqing Luo, Hua Ye, Hui Tian
          </td>
          <td>2025-09-01</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The combination of 5-fluorouracil and irinotecan (FOLFIRI) remains a standard-of-care treatment for metastatic colorectal cancer (mCRC), yet benefits only about half of patients. Using patient-derived xenografts (PDXs), we investigated the biological underpinnings of this heterogeneous response. FOLFIRI-resistant models showed transcriptional upregulation of innate immunity and mitochondrial metabolism genes, together with reduced expression of the DNA polymerase POLD1. Sensitive counterparts exhibited a BRCAness-like phenotype with genomic scars of homologous recombination (HR) deficiency, not caused by genetic or epigenetic loss of HR genes but by low abundance of the RAD51 recombinase. In tumoroids, forced RAD51 overexpression attenuated HR deficiency-related scars and chemotherapy-induced damage, while HR inhibition through ATM blockade enhanced drug sensitivity. The predictive relevance of key response determinants was validated in clinical samples. This work illuminates functional, non-genetic facets of BRCAness in mCRC and introduces actionable biomarkers and targets, offering prospects to improve clinical decision-making and broaden therapeutic options for chemorefractory patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb783603c147c9c964fbfcc1a8e05ff7e8e4445" target='_blank'>
              The molecular and functional landscape of resistance to FOLFIRI chemotherapy in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marco Avolio, S. Leto, F. Sassi, B. Lupo, Elena Grassi, Irene Catalano, E. Zanella, V. Vurchio, F. Cottino, Petros K Tsantoulis, L. Lazzari, Paolo Luraghi, Martina Ferri, Francesco Galimi, E. Berrino, S. Bellomo, Marco Viviani, A. Sogari, G. Mauri, F. Tosi, Federica Cruciani, A. Sartore-Bianchi, S. Siena, Felice Borghi, V. Torri, E. Élez, J. Tabernero, Maria Nieva, C. Montagut, N. Tarazona, Andrés Cervantes, Sabine Tejpar, A. Bardelli, C. Marchiò, S. Marsoni, Andrea Bertotti, L. Trusolino
          </td>
          <td>2025-09-22</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The development of de novo resistance is a major disadvantage in molecularly targeted therapies. While much focus is on cell-intrinsic mechanisms, the microenvironment is also known to play a crucial role. This study examines interactions between cancer cells and cancer associated fibroblasts (CAFs) to understand the local crosstalk facilitating residual disease. Using a hybrid-discrete-continuum model, we explore how treatment-induced stress responses can elicit CAF activation and how breaks in treatment allow microenvironment normalisation. We investigate how fluctuating environmental conditions shape the local crosstalk and ultimately drive residual disease. Our experimentally calibrated model identifies environmental and treatment conditions that allow tumour eradication and those that enable survival. We find two distinct mechanisms that underpin residual disease: vasculature-limited drug delivery and CAF-mediated rescue. This work provides a better understanding of the mechanisms that drive the creation of localised residual disease, crucial to informing the development of more effective treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e202efa4d7c859cf189d67f82a4d897793861f" target='_blank'>
              The role of environmentally mediated drug resistance in facilitating the spatial distribution of residual disease
              </a>
            </td>
          <td>
            Amy Milne, A. Marusyk, P. Maini, Alexander Anderson, Noemi Picco
          </td>
          <td>2025-08-09</td>
          <td>Communications Biology</td>
          <td>3</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="The SS18::SSX oncogene is the driver of synovial sarcoma, an aggressive cancer presenting in young adults that has poor long-term outcomes. Over the past five years, significant progress has been made in understanding the molecular, genomic, and epigenetic mechanisms underlying synovial sarcoma. This review synthesizes recent advancements in synovial sarcoma, including diagnostic pathology, genomic profiling, SS18::SSX biology, epigenetic dysregulation, proteomics, targetable pathways and immunotherapy. Key findings include the identification of rare but instructive alternative gene fusions, the roles of PRC1 and of liquid-liquid phase separation in SS18::SSX-mediated oncogenesis, and the development of epigenetic and engineered T-cell therapies. These advances offer new hope for improved treatments and outcomes in synovial sarcoma patients, though challenges remain in overcoming resistance and ensuring equitable access to emerging therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9194070a3830c48d1eb33aa8623f50b92cbe3b84" target='_blank'>
              Molecular and epigenetic oncogenesis in synovial sarcoma: implications for cancer biology, diagnosis and treatment
              </a>
            </td>
          <td>
            Amy Xueqi Wang, Kevin B. Jones, Torsten O Nielsen
          </td>
          <td>2025-08-23</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA-binding molecules regulate gene expression, replication, repair, and transcription, making their research crucial to molecular biology. This review investigated their architectures, processes, and impacts on cancer research and therapy to determine their biological and therapeutic potential. A thorough examination of published data on DNA-binding agents, including intercalators, groove binders, and metal complexes, focused on their chemical properties, biological activity, and therapeutic significance. Doxorubicin intercalated between base pairs to inhibit replication and transcription, whereas cisplatin produced covalent cross-links with guanine bases to cause tumor cell death. Multiple derivatives of metal complexes reduced tumor development by over 70% in leukemia models as DNA probes and therapeutics. Also reviewed, Dps proteins in *Escherichia coli* showed that their non-specific DNA binding offered up to 65% oxidative stress resistance compared to control cultures, validating DNA-protein protection as a survival strategy. In Phase II studies, amsacrine caused substantial remission in acute leukemia patients, whereas doxorubicin was more effective across many cancer types but had greater cardiotoxicity concerns. Selectivity, toxicity, and resistance limit DNA-targeting medicines, although they are successful. Chemical modifications like hydrophobic tailoring and sequence-specific binding have enhanced binding affinity and therapeutic index, yet only 10% of candidate compounds get clinical approval. Recent studies show that AI-driven design has expedited screening, lowering development costs by 30% and durations by 3–5 years. These data suggest that DNA-binding medicines have great potential in cancer, but safer, more selective, and resistance-free therapy are still needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfaa0d838d07a9aad5a057a0a95be735af4700a" target='_blank'>
              Chemistry of DNA-binding Molecules
              </a>
            </td>
          <td>
            Evon Akram, Dina A. Najeeb, Asmaa A. Jawad, N. H. Bedair, A. M. Hussein, Saba R. Jaafar, Ruaa H. Ali, R. F. Shaher, Marwa A. Hussein, Daniah M. Hamid, Reem H. Al-Tabra, Alyaa K. Abood, Salam Mohammed
          </td>
          <td>2025-08-30</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 TP53, often referred to as the guardian of the genome, is one of the most frequently mutated genes in human cancers. High grade serous ovarian cancer (HGSOC) has a TP53 mutation rate of up to 96%. Given its multi-faceted role in human cancers, there have been many efforts to therapeutically target mutant p53. APR-246 is a small molecule drug that is proposed to bind mutant p53 and promote its thermodynamic stabilization, allowing the tumor suppressor to regain wild-type-like conformation and functionality. However, the mechanism of this drug has remained a contentious topic, as it has been shown to act via various p53-independent mechanisms, including modulation of the redox system. To address this, we have performed a systematic functional genetic screening for the identification of APR-246 rescuable p53 mutants. Our approach leverages a pooled TP53 mutagenesis library containing ∼8,000 amino acid variants, spanning the entirety of the TP53 coding sequence. Cell based screenings against sub-lethal doses of APR-246 in two ovarian cancer cell lines, SKOV3 and OVCAR5, has revealed that sensitivity to this drug is cell-line dependent rather than mutant p53-dependent. Our results suggest that p53-independent factors such as intrinsic resistance to ferroptosis may play a more significant role in response to APR-246. We propose that resistance to ferroptosis provides cellular conditions which enable identification of p53 mutants that are sensitive to APR-246. This is evidenced by our results which show that genetic screen in OVCAR5, which is more intrinsically resistant to ferroptosis, uncovers a greater number of potentially sensitive mutants, including R175H, a mutant largely recognized as rescuable by APR-246. Further, we have identified GPX4, an anti-ferroptosis gene, as a key determinant of APR-246 sensitivity. Indeed, we demonstrate synergy between APR-246 and GPX4 inhibitor, ML-210. These studies support the hypothesis that APR-246 efficacy is largely attributable to p53-independent mechanisms and that attenuation of these off-target effect may reveal potentially sensitive p53 mutants.



 Anais Saunders, Caili Tong, Anthony N. Karnezis, Gary S. Leiserowitz, Jeremy R. Chien. Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7beb6941bcf2a1d1e3d25e6917a37bc8794a91d" target='_blank'>
              Abstract A046: Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects
              </a>
            </td>
          <td>
            Anais Saunders, Caili Tong, A. Karnezis, G. Leiserowitz, Jeremy Chien
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Breast cancer is one of the most common malignancies among women globally, with an annually increasing incidence rate. Its complex pathogenesis and high heterogeneity pose significant challenges to clinical diagnosis and treatment. Traditional diagnostic methods and therapeutic approaches have limitations in improving patient survival rates and prognosis, thus urgently necessitating the identification of novel biomarkers and therapeutic targets. MicroRNA (miRNA), a class of endogenous non-coding small RNA molecules with a length of approximately 20–24 nucleotides, finely regulates gene expression by binding to the 3’ untranslated region of target mRNAs, inhibiting gene translation, or promoting mRNA degradation. In 2024, the Nobel Prize in Physiology or Medicine was awarded for research related to miRNA. Numerous studies have demonstrated that miRNAs play pivotal roles in the initiation, progression, invasion, metastasis, and drug resistance of breast cancer. Aberrant expression of specific miRNAs is closely associated with the molecular subtypes, prognosis, and treatment response of breast cancer, suggesting their potential as diagnostic and prognostic biomarkers. To explore the potential value of miRNA in the diagnosis and treatment of breast cancer, this article systematically reviews the latest research progress on the role of miRNAs in the diagnosis and treatment of breast cancer, with a focus on their application as tumor markers in early diagnosis, molecular subtyping, and therapeutic response monitoring. It elucidates the possibilities of miRNAs as therapeutic targets and tools in targeted therapy, including the current research status of miRNA mimics and inhibitors in breast cancer treatment. Furthermore, it analyzes the role of miRNAs in prognosis assessment, exploring their correlation with patient survival rates, recurrence risks, and treatment responsiveness. Additionally, this article discusses the challenges faced by miRNA research in precision medicine for breast cancer and future directions, providing new insights and strategies for early diagnosis, individualized treatment, and prognosis assessment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302a8130c8ecf1046d266c188e11099501cd566d" target='_blank'>
              MicroRNAs in breast cancer—new frontiers in diagnosis, targeted therapy, and prognosis assessment
              </a>
            </td>
          <td>
            Jinsuo Xiao, Lanhui Zhang, Ruifan Su, Bo Zhao, Yuanyuan Dang, Chuanlin Zhao, Sujin Wang, Teng Qi, Fuqing Ji
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer has recently overtaken other conditions as the foremost cause of death worldwide. While numerous
conventional treatment modalities and cytotoxic immunotherapies are available, the intricate nature of tumor biology—
driven by a multitude of genetic and cellular factors underlying tumorigenesis and metastasis—necessitates the
development of advanced treatment approaches capable of acting on cellular as well as genetic fronts. Amid these
advancements, Chimeric Antigen Receptor (CAR) T-cell therapy has risen as a pioneering frontier in the domain of T-cell
engineering. Here, T cells obtained from the patient are engineered ex vivo to carry synthetic receptors designed to
recognize tumor-associated antigens. A key feature is that these modified receptors can identify tumor antigens without
requiring MHC presentation. Recently, CAR-T cell therapy has achieved significant clinical milestones, inducing
remission in up to 80% of patients with hematologic malignancies, particularly those with acute lymphoblastic leukemia
(ALL) and certain subtypes of non-Hodgkin lymphoma, such as large B-cell lymphoma. The anti-CD19 CAR design,
UCART19, has demonstrated remarkable therapeutic potency in managing relapsed or refractory hematologic
malignancies. Furthermore, other surface antigens, including CD20 and CD22, prevalent in various leukemias and
lymphomas, are being actively explored as therapeutic targets, with multiple clinical trials underway. Although its
application is presently concentrated on blood cancers, the integration of advanced modalities—such as bispecific CARs,
tandem CARs, inhibitory CARs, multi-antigen targeting, CRISPR-based gene editing, and nanoparticle-mediated
delivery—holds the promise of significantly enhancing its efficacy. These advancements may extend its utility to both
hematologic and solid tumors, offering a treatment paradigm that is not only rapid and precise but also safer compared to
traditional modalities. The purpose of this review is to offer an in-depth analysis of the benefits and emerging
developments in CAR T-cell immunotherapy, while underscoring its rising advantage over conventional treatments like
chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5c74588893737227ed28d56c33635129d1547e" target='_blank'>
              Targeting Cancer at its Core: Cellular and Genetic Approaches Via CAR-T Cell Therapy CAR-T Cell Therapy of Cancer
              </a>
            </td>
          <td>
            Bandaru Aashritha, Ambati Akshara, Nandini Kongara, P. V. Babu
          </td>
          <td>2025-09-04</td>
          <td>International Journal of Innovative Science and Research Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established.We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG.Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown.This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda63458271c1d78bb1529cdfe4855855ca64701" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer.
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The ataxia–telangiectasia-mutated (ATM) protein plays a crucial role in the DNA damage response, particularly in the homologous recombination (HR) pathway. This study aimed to assess the impact of deleterious ATM variants on homologous recombination deficiency (HRD) and response to PARP inhibitors (PARPi) in melanoma patients, using a cell line established from melanoma tissue of a patient carrying the c.5979_5983del germline ATM variant. Despite proven loss of heterozygosity, lack of ATM activation, and HRD, our model did not show sensitivity to PARPi. We assessed the potential contribution of the Schlafen family member 11 (SLFN11) helicase, whose expression is inversely correlated with PARPi sensitivity in other cancers, to the observed resistance. The ATM mutant cell line lacked SLFN11 expression and featured hypermethylation-mediated silencing of the SLFN11 promoter. While sensitive to the ATR inhibitor (ATRi), the addition of ATRi to PARPi was unable to overcome the resistance. Our findings suggest that ATM mutational status and HRD alone do not adequately account for variations in sensitivity to PARPi in our model. A comprehensive approach is essential for optimizing the exploitation of DNA repair defects and ultimately improving clinical outcomes for melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f01ff8501beaba82fe3e6fd4e0b7083d80c6308" target='_blank'>
              Assessing Determinants of Response to PARP Inhibition in Germline ATM Mutant Melanoma
              </a>
            </td>
          <td>
            E. Allavena, Michela Croce, B. Dalmasso, Cecilia Profumo, Valentina Rigo, V. Andreotti, I. Vanni, B. Pellegrino, A. Musolino, N. Campanini, W. Bruno, L. Mastracci, G. Zoppoli, E. Tanda, F. Spagnolo, P. Ghiorzo, L. Pastorino
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcaf09592f5e3c1f92fc05933cdb4d9765107436" target='_blank'>
              Evolutionary Overview and Future Perspectives: ESR1 Mutations, Liquid Biopsy, and Artificial Intelligence for a New Era of Personalized Medicine in ER+ Breast Cancer.
              </a>
            </td>
          <td>
            Serafina Martella, Giacomo Cusumano, T. H. Senevirathne, Dimitrios Stylianakis, Enrico Palmas, N. Denaro, Chiara Tommasi, M. Scartozzi, Lorenzo Gerratana, C. Solinas
          </td>
          <td>2025-08-30</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Breast cancer, a major health concern worldwide, involves diverse molecular subtypes and complex gene expression patterns. This study conducted a comprehensive bioinformatics analysis of breast cancer, analyzing 10 gene expression datasets from the Gene Expression Omnibus archive to find common genes that exhibit differential expression (DEGs). Then, we conducted pan-cancer and functional enrichment analyses, including single-cell level investigations of DEGs. To identify potential hub genes, we built protein–protein interaction networks, which were further analyzed for methylation patterns and expression in various cancer stages. The study also explored interactions between these hub genes with microRNAs and transcription factors, along with their correlation with cytokine expression and infiltration of immune cells in the tumor microenvironment. The TCGA BRCA dataset was used for hub gene survival analysis. Key findings include the identification of 36 shared DEGs, with distinct expression patterns in principal component analysis, suggesting their potential as molecular signatures for diagnosis and prognosis. Investigation at the single-cell level revealed upregulated DEGs mainly expressed by tumor-associated fibroblast cells. Gene ontology unveiled upregulated genes mainly involved in the extracellular matrix, but most of the downregulated genes are tumor suppressors. The protein–protein interaction network analysis highlighted 7 hub genes: CAV1, FN1, BGN, CXCL12, SPTBN1, COL11A1, and INHBA. Methylation analysis revealed intricate epigenetic regulation of these genes. MicroRNA and transcription factor interaction analysis underscored the complex regulatory networks influencing gene expression. Cytokine profile analysis in tumor microenvironment and its correlation with hub gene expression provided insights into the immunological landscape of breast cancer. Survival analysis indicated that both upregulated and downregulated hub genes are linked with patient overall survival outcomes, although it is not statistically significant. This study provides a detailed view of the underlying molecular mechanisms in breast cancer, suggesting potential biomarkers and therapeutic targets. Its findings can contribute significantly to understanding the complexity of the breast cancer microenvironment, leading to the development of more sophisticated and targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf4528f9aeae9acc9b474eba2a50c1c17c8b995" target='_blank'>
              Identification of potential hub genes and molecular mechanisms in breast cancer microenvironment: A comprehensive transcriptomics approach
              </a>
            </td>
          <td>
            MS Abdullah Al Noman, MS a Abdullah Al Saba, MS a Mohammad Sayem, PhD Tahirah Yasmin, PhD A. H. M. Nurun Nabi, A. N. Nabi
          </td>
          <td>2025-08-29</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Accounting for the highest cancer-related death burden worldwide, NSCLC, presents a significant public health challenge due to its high incidence and persistently low survival rates. Despite progress in low-dose spiral CT screening and smoking cessation campaigns, the five-year survival rate for patients diagnosed at advanced stages remains below 20%, underscoring the urgent need for optimized treatment strategies. This article synthesizes recent research advances concerning pathogenic mechanisms, diagnostic classification, and therapeutic innovations. It begins by dissecting the interaction networks between driver gene mutations and the tumor immune microenvironment, elucidating the critical role of epigenetic dysregulation in tumor progression. It subsequently reviews how the integration of liquid biopsy with artificial intelligence (AI)-enhanced imaging techniques is refining molecular subtyping. The article then provides a systematic evaluation of breakthrough clinical developments, including the iterative evolution of targeted agents (such as fourth-generation EGFR inhibitors), the expansion of immunotherapy across all treatment lines (from neoadjuvant settings to advanced disease), and the emergence of antibody-drug conjugates (ADCs) targeting HER3 and TROP2. These transformative approaches have propelled the median survival of advanced patients beyond 30 months and increased the pathological complete response rate in early-stage patients by tenfold. Studies demonstrate that biomarker-directed precision therapy is profoundly reshaping clinical practice. Future efforts must prioritize unraveling complex resistance mechanisms to further extend therapeutic benefits to all patient populations through multi-omics-guided personalized combination strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67b8c6e9f6adaa36c36105fd4cd3a5e640e941ac" target='_blank'>
              Comprehensive Advances in Non-Small Cell Lung Cancer: From Molecular Mechanisms to Precision Therapy
              </a>
            </td>
          <td>
            Jiayi Sun
          </td>
          <td>2025-08-26</td>
          <td>MedScien</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648ee47786050552f00a0c31f0d21f6a61f09642" target='_blank'>
              LEF1 confers resistance to DNA-damaging chemotherapies through upregulation of PARP1 and NUMA1 in ovarian cancer.
              </a>
            </td>
          <td>
            Lin Huang, Mengna Zhu, Mengqing Chen, F. Ying, Qiulei Wu, Pan Liu, Wenhan Li, Guoqing Li, Yuewen Gao, Shuyan Yi, Wenwen Wang, Yiping Wen, Si Sun, Jing Cai, Man Xiao
          </td>
          <td>2025-09-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Molecular subtyping of pediatric B-cell acute lymphoblastic leukemia (B-ALL) has improved patient outcomes through stratification and selection of targeted therapies. Despite extensive genomic and transcriptomic profiling of this cancer, few studies to date have characterized the proteomic landscape, although proteins are the direct targets of many therapeutic agents. In this study, we demonstrate the utility of multi-omic integration of global transcriptomic, proteomic, and phosphoproteomic profiles of samples from patients diagnosed with either of two B-ALL subtypes – Ph-like (BCR::ABL1-like) and ETV6::RUNX1. Through individual and multi-omic analysis, we recapitulate known transcriptomic findings and identify novel subtype-specific proteomic and phosphoproteomic biomarkers. Our findings suggest a previously undescribed role for calcium-dependent signaling processes in Ph-like B-ALL, which has the potential to serve as a novel avenue for targeted treatments. By integrating multiple ‘omics modalities, we identify not only features of interest but also begin to unravel the regulatory interactions driving subtype-specific mechanisms of leukemogenesis. This integrated analytic approach paves the way for enhanced precision medicine for precise subtyping and treatment selection for pediatric leukemia patients. Mass spectrometry data generated in this study have been deposited in MassIVE under accession MSV000097955.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c9873abb2ee33fbad496278e06ca849fb38658" target='_blank'>
              Integrated multi-omic analysis reveals novel subtype-specific regulatory interactions in pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            I. Pushel, Zachary Clark, L. Lansdon, B. Yoo, M. Rekowski, Nicole M. Wood, Michael P. Washburn, M. Farooqi
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Chemoresistance is a major obstacle in cancer treatment, and it often results in treatment failure and disease progression. Among the plethora of factors contributing to chemoresistance, the transcription factor Krüppel-like factor 4 (KLF4) has emerged as a pivotal player. This review discusses the role of KLF4 in orchestrating various mechanisms underlying cancer chemoresistance. KLF4, originally identified as a regulator of cell differentiation and proliferation, has recently gained attention for its role in modulating cellular responses to chemotherapeutic agents. Through complex regulatory networks, KLF4 modulates the process of drug efflux, DNA repair, apoptotic signaling, tumor heterogeneity, and cancer cell stemness, leading to the development of cancer chemoresistance. Additionally, tissue or cell types specific post-translational modification (PTM) of KLF4 plays a significant role in the development of cancer chemoresistance. The review explores emerging possibilities and available information that can be utilised to understand the mechanism of chemoresistance mediated by KLF4 in cancer. In conclusion, understanding the complex mechanisms through which KLF4 orchestrates cancer chemoresistance opens promising avenues for developing more effective therapeutic interventions to combat treatment-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e91a9cb65a0f14dcdac5b83f903eda917dba7f" target='_blank'>
              KLF4 in cancer chemoresistance: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Suresh Singh Yadav, Punita Kalia, Navneet Kaur, Sourav Sharma, Shambhavi Thakur, Soni Kumari, R. R. Nair
          </td>
          <td>2025-09-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Acute lymphoblastic leukemia (ALL) is the leading cause of cancer-related mortality in pediatrics. Between 15-30% of patients relapse, and current treatments are associated with acute and long-term toxicity. The immune component of the tumor microenvironment is an under-explored factor that could play a role in the establishment or progression of the disease, as well as modulate response to chemo- and immunotherapies. We characterized the molecular profile of leukemias from 32 patients with pediatric ALL treated under the ALLIC-GATLA 2010 clinical protocol in Argentina by transcriptome sequencing (Illumina). We estimated the immune component through digital cytometry (MIXTURE) and a cytolytic score based on the expression of five genes. We searched for single nucleotide variants (SNV) in the transcriptome (CTAT-Mutations) that could alter the interaction with immune cells in the tumor microenvironment. For this, we analyzed 843 genes from Reactome collections associated with the immune system. Variants were filtered and annotated using publicly available germline and somatic variant databases. The group of patients with higher abundance of cytotoxic cells (Z-score>1) was associated with an increased risk of relapse/death (HR=4.5; p=0.05), and higher levels of CX3CR1 (q-val<0.05) and TIM-3 (q-val=0.06). Analysis of an independent high-risk cohort (TARGET B-ALL phase II, N=88), confirmed that a higher cytolytic score was associated with the risk of relapse in an univariate (HR=2.66, p=0.001) and multivariate model including cytolytic score (p<0.006), protocol (p=0,07), ETV6::RUNX1 (p:>0.05) and TCF3::PBX1 status (p<0.001). Cytolytic score was not associated with molecular subtype, age, sex or early response to treatment (p>0.05). For SNV analysis, we developed a threshold value for the QUAL parameter to select for high-confidence variants, establishing a specificity of 95%. Eleven variants were found in nine patients, being 8/11 classified by Franklin as TIER3 in CEP290, UBE3B, CPNE3, DUSP4, DAPP1, RNF135, TBC1D10C and TLR1; 2/11 as TIER1 in TP53, KRAS; and 1/11 as TIER4 in BTN3A3. Patient harboring variants in these genes were significantly associated with a lower GSVA score in the Reactome dataset “MHC_Class_II_Antigen_Presentation” (q-val<0.01). We found a variant in UBE3B, located in the active site of the E3 ubiquitin ligase, and associated with a low GSVA score in the Reactome “Antigen presentation through MHC class I” pathway. A novel variant in CPNE3 gene was found, located in a highly conserved residue and predicted to be deleterious by multiple tools. Conclusion: the abundance of cytotoxic cells was independent of molecular subtype and associated with a worse prognosis in independent cohorts. Further analysis of the variants found in this study has the potential to identify novel mechanisms modulating the interaction between the immune system and leukemic cells in ALL.



 Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, Pablo Sanchis, Maria Cecilia Riccheri, Geraldine Gueron, Javier Cotignola, Maria Sol Ruiz. Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A040-PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8f83b4eeb02fdcb7483db9e8c77e60a6f1fd7e" target='_blank'>
              Abstract A040-PR008: Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients
              </a>
            </td>
          <td>
            Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, P. Sanchis, M. C. Riccheri, G. Gueron, J. Cotignola, Maria Sol Ruiz
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Classical Hodgkin lymphoma (cHL) is a biologically and clinically unique malignancy characterized by rare Hodgkin and Reed–Sternberg (HRS) cells surrounded by a dense and diverse inflammatory infiltrate. These malignant cells actively reshape the tumor microenvironment (TME) through metabolic reprogramming and immune evasion strategies. This review synthesizes current knowledge on how metabolic alterations contribute to tumor survival, immune dysfunction, and therapeutic resistance in cHL. We discuss novel therapeutic approaches aimed at disrupting these processes and examine the potential of combining metabolic interventions with immune-based strategies—such as immune checkpoint inhibitors (CPIs), epigenetic modulators, bispecific antibodies, and CAR-T/CAR-NK cell therapies—which may help overcome resistance and enhance anti-tumor responses. Several agents are currently under investigation for their ability to modulate immune cell metabolism and restore effective immune surveillance. Altogether, targeting metabolic vulnerabilities within both tumor and immune compartments offers a promising, multifaceted strategy to improve clinical outcomes in patients with relapsed or refractory cHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84b70765d93f8c830e92294e485ce37dc955d43b" target='_blank'>
              Metabolic Interactions in the Tumor Microenvironment of Classical Hodgkin Lymphoma: Implications for Targeted Therapy
              </a>
            </td>
          <td>
            M. Kurlapski, Alicja Braczko, Paweł Dubiela, Iga Walczak, B. Kutryb-Zając, J. Zaucha
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P Ji, Nancy R. Zhang
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality worldwide with KRAS mutations present in nearly 45% of cases. Compared to KRAS wild-type (WT) CRC, KRAS-mutant CRC is associated with poorer prognosis and fewer effective treatment options. Protein Arginine Methyltransferase 5 (PRMT5), an epigenetic regulator involved in diverse cellular processes, is currently under investigation as a therapeutic target in multiple cancer types. Our previous work demonstrated that PRMT5 inhibition produces stronger therapeutic effects in KRAS-mutant CRC cells than in KRAS WT cells, suggesting potential crosstalk between PRMT5 and KRAS. In this study, we aimed to identify key downstream proteins that may mediate this interaction. Through a literature review, protein–protein interaction analysis (STRING database), gene expression analysis (GEPIA database), and correlation analysis (GEPIA database), we identified MYC, E2F1, and EIF4E as critical candidates. These proteins are shown to interact with both PRMT5 and KRAS in STRING, are overexpressed in CRC tumor samples, and show positive gene expression correlations with PRMT5 and KRAS in patient data. These findings are significant, as they provide new insights into the PRMT5–KRAS crosstalk and suggest potential targets for novel and combination therapies in KRAS-mutant CRC. Further research and biological experiments are needed to verify and outline the exact molecular processes behind MYC, E2F1, and EIF4E’s interactions with both PRMT5 and KRAS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d713c1881674e6d304d22fb6424025303ae112" target='_blank'>
              PRMT5 Inhibition as a Potential Strategy for KRAS Mutant CRC: Downstream Mediators of the PRMT5–KRAS Crosstalk
              </a>
            </td>
          <td>
            Mark Spivak, Moshe Pahmer, Dorna Delrahimnia, Tzuriel Sapir, David Shifteh
          </td>
          <td>2025-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Mycosis fungoides (MF) is a primary T-cell lymphoma that manifests on the skin. Using the American Association for Cancer Research (AACR) Project Genomics, Evidence, Neoplasia, Information, Exchange (GENIE), a publicly accessible genomic repository, this study explores the mutational profile of MF in a large, diverse patient cohort. This research uncovers the most common somatic mutations in FAT1, KMT2D, TP53, JAK3, and SETBP1, which disrupt key pathways controlling cell adhesion, chromatin structure, p53 activity, and cytokine signaling. These alterations provide insight into disease mechanisms and reveal potential diagnostic, prognostic, and therapeutic targets. Significant co-mutation patterns suggest cooperation between signaling and epigenetic pathways, while the identification of mutation enrichments across racial groups provides new insight into potential genetic and environmental influences on disease biology. These findings highlight the potential for the development of biomarker- and patient-targeted therapeutic strategies in MF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8db9113c13bdcc4501fff2341302e49c2ec642f" target='_blank'>
              Integrative Mutational Landscape of Mycosis Fungoides Using a National Genomics Repository
              </a>
            </td>
          <td>
            Grace S. Saglimbeni, Beau Hsia, Peter T. Silberstein, Abubakar Tauseef
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background/Objectives: Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with poor prognosis. T cell exhaustion (TEX) is a key factor in tumor immune evasion and therapeutic resistance. In this study, we integrated single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data to characterize TEX-related transcriptional features in HCC. Methods: We first computed TEX scores for each sample using a curated 65-gene signature and classified them into high-TEX and low-TEX groups by the median score. Differentially expressed genes were identified separately in scRNA-seq and bulk RNA-seq data, then intersected to retain shared candidates. A 26-gene prognostic signature was derived from these candidates via univariate Cox and LASSO regression analysis. Results: The high-TEX group exhibited increased expression of immune checkpoint molecules and antigen presentation molecules, suggesting a tumor microenvironment that is more immunosuppressive but potentially more responsive to immunotherapy. Functional enrichment analysis and protein–protein interaction (PPI) network construction further validated the roles of these genes in immune regulation and tumor progression. Conclusions: This study provides a comprehensive characterization of the TEX landscape in HCC and identifies a robust gene signature associated with prognosis and immune infiltration. These findings highlight the potential of targeting TEX-related genes for personalized immunotherapeutic strategies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a1103280627c731b735d281df63ab3e636c5d0" target='_blank'>
              Bulk and Single-Cell Transcriptomes Reveal Exhausted Signature in Prognosis of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Ruixin Chun, Haisen Ni, Ziyi Zhao, Chunlong Zhang
          </td>
          <td>2025-08-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a major global challenge, with growing attention to its pathogenesis as mediated by the gut microbiome and epigenetic regulation. Despite therapeutic progress, clinical management remains difficult. CRC accounts for ~10% of cancers and is the second leading cause of cancer death worldwide. Romania bears a substantial burden, with many diagnoses at advanced stages. Etiology—Integrated Genetic, Environmental, and Microbial Determinants. Hereditary syndromes explain 10–15% of cases; most are sporadic, with hypermutated MSI/POLE (~15%), non-hypermutated chromosomal instability (~85%), and a CpG island methylator phenotype (~20%). GWAS implicate loci near SMAD7, TCF7L2, and CDH1; in Romania, SMAD7 rs4939827 associates with risk. Lifestyle exposures—high red/processed meat, low fiber, adiposity, alcohol, and smoking—shape susceptibility. Microbiome–Epigenome Interactions. Dysbiosis promotes carcinogenesis via genotoxins (e.g., colibactin), hydrogen sulfide, activation of NF-κB/STAT3, barrier disruption, and epigenetic remodeling of DNA methylation and microRNAs. Fusobacterium nucleatum, enterotoxigenic Bacteroides fragilis, and pks+ Escherichia coli exemplifies these links. Population-Specific Risk—Romania within Lifestyle–Microbiome Evidence. Incidence is rising, including early-onset disease. Romania lacks CRC-specific microbiome datasets. However, metabolic cohorts show loss of butyrate producers, enrichment of pathobionts, and SCFA imbalance—patterns that mirror European CRC cohorts—and exhibit regional heterogeneity. Beyond Fusobacterium nucleatum. Additional oncobacteria shape tumor biology. Peptostreptococcus stomatis activates integrin α6/β4→ERBB2–MAPK and can bypass targeted inhibitors, while Parvimonas micra enhances WNT/β-catenin programs and Th17-skewed immunity. Together, these data support a systems view in which microbial cues and host epigenetic control jointly drive CRC initiation, progression, metastasis, and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b63cfd4181c0a26226fd793b68b2268313fd047" target='_blank'>
              Gut Microbiome-Mediated Genetic and Epigenetic Alterations in Colorectal Cancer: Population-Specific Insights
              </a>
            </td>
          <td>
            Simona Turcu, F. Grama, M. Gazouli
          </td>
          <td>2025-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92a0745094dde86c826e8220fdb2990989d2b040" target='_blank'>
              Artificial Intelligence-Guided Molecular Determinants of PI3K Pathway Alterations in Early-Onset Colorectal Cancer Among High-Risk Groups Receiving FOLFOX
              </a>
            </td>
          <td>
            M. D. Fernando C. Diaz, B. Waldrup, Ph.D Francisco G. Carranza, Sophia Manjarrez, M. Enrique Velazquez-Villarreal
          </td>
          <td>2025-08-21</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer risk increases with age, and cellular senescence may be a major contributor to cellular carcinogenesis. Enormous efforts have been made to investigate the interrelation between aging and tumors, but little is known about the comparative features of normal aging, cellular senescence, and cancer at single-cell resolution. By integrating analyses of genomics, epigenomics, and bulk and single-cell transcriptomics, we revealed a directionally opposite transcriptional profile between cellular senescence and tumorigenesis at the single-cell level, which may be affected by epigenomic regulations. A total of 648 aging-dependent senescence-associated coregulated modules (SACMs), disproportionately affecting the reproductive systems of both females and males, were initially defined across 17 tissues. Single-cell analysis revealed that aging primarily affects endothelial cells, followed by T cells, epithelial cells, macrophages, and fibroblasts. Opposite directions of change in gene expression between aging and cancer can commonly be observed in endothelial, fibroblast, and epithelial cells, which may prompt the opposing patterns of gene expression between tissue aging and epithelial carcinoma at the bulk level. A similar pattern of expression can be observed in immune cells, which are characterized by decreased self-renewal with aging, but this pattern is reversed in epithelial carcinoma. Our study highlighted the role of senescence as a natural barrier against tumor formation and supported the idea that aging-related systemic environment changes create a protumorigenic milieu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4073352af61f280e6d1b0c8dc5f8cd7b466fe5" target='_blank'>
              Integrative Transcriptomic Analysis Decodes the Interplay Between Aging, Senescence, and Cancer.
              </a>
            </td>
          <td>
            Jiale Cai, Deng Wu, Dahua Xu, Fujin Liu, Peihu Li, Dehua Zheng, Zhizhou Xu, Meng Cao, Yutong Shen, Nihui Zhang, Guanghui Tian, Chengyi Fan, Bo Wang, Kongning Li, Xiaoman Bi
          </td>
          <td>2025-09-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/527c49eaf6701e153dab5ae1e99ef449da850bc8" target='_blank'>
              In vivo CRISPR screening links NFKB1 to endocrine resistance in ER⁺ breast cancer.
              </a>
            </td>
          <td>
            Cancan Lyu, Sam Hall, Mark A. Stamnes, Songhai Chen
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Over the last decade, immunotherapy has revolutionized lung cancer treatments, particularly in non-small cell lung cancer, where immune checkpoint inhibitors have achieved significant clinical success. However, high percentages of patients do not respond initially or eventually develop a resistance to these therapies. This review explores the evolution and challenges of immunotherapy in lung cancer, highlighting its clinical milestones and intrinsic and extrinsic resistance mechanisms. We investigate tumor-intrinsic resistance factors, including alterations in antigen presentation, the loss of Beta-2 microglobulin function, impaired interferon signaling, immune editing, epigenetic modifications, and tumor-extrinsic resistance, such as an immunosuppressive lung tumor microenvironment, dysregulated cytokine profiles, and the upregulation of immune checkpoints. Then, we focus on the emerging role of resistance biomarkers and the development of personalized treatment strategies to overcome these challenges. The complex interplay between tumor biology and immune modulation in lung cancer paves the way for novel approaches for improving the effectiveness of immunotherapeutic treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be521ad9dbd5fb1b3f963d8423b3bfabe7d4614d" target='_blank'>
              Unraveling Resistance in Lung Cancer Immunotherapy: Clinical Milestones, Mechanistic Insights, and Future Strategies
              </a>
            </td>
          <td>
            Maria Vitale, Raffaella Pagliaro, G. Viscardi, Lucio Pastore, Giuseppe Castaldo, F. Perrotta, Susan F M Campbell, Andrea Bianco, Filippo Scialò
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Systems biology approaches have been applied to prostate cancer to model how individual cellular and molecular components interact to influence cancer development, progression, and treatment responses. The integration of multi-omic experimental data with computational models has provided insights into the molecular characteristics of prostate cancer and emerging treatment strategies that have the potential to improve patient outcomes. Here, we highlight recent advancements that have emerged from systems modeling in prostate cancer. These include descriptions of the molecular landscape of prostate cancer and how genomic alterations inform computational models of disease progression, how evolutionary processes give rise to mechanisms of therapeutic resistance, and the development of innovative treatment strategies such as adaptive therapy. We also highlight current challenges in prostate cancer that can be addressed through systems biology approaches. These include tumor heterogeneity, poor immunotherapy response, a paucity of experimental model systems, and the ongoing translation of computational models for clinical decision making. Leveraging systems biology approaches has the potential to lead to a better understanding of the disease and better patient outcomes in the treatment of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aeffc4990ae5d3cf0820d80475621089e3b441" target='_blank'>
              Systems biology successes and areas for opportunity in prostate cancer
              </a>
            </td>
          <td>
            Michael V Orman, Laura S Graham, Scott D. Cramer, James C. Costello
          </td>
          <td>2025-08-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), the most prevalent primary liver cancer, continues to pose a significant global health burden due to its high mortality rate. In addition to genetic alterations, epigenetic aberrations, including DNA methylation, histone modifications, chromatin remodeling, and noncoding RNA (ncRNA) dysregulation, play critical roles in HCC initiation and progression. Notably, miR-375 and miR-483-5p are among the most dysregulated miRNAs in HCC, with their altered expression levels closely associated with tumor stage and patient survival. These epigenetic modifications offer promising therapeutic avenues due to their reversibility and dynamic nature. Furthermore, specific epigenetic signatures such as CDH1 promoter hypermethylation and HOTAIR overexpression are being explored as potential biomarkers for early detection and treatment response. In this chapter, we review recent advances in the epigenetic landscape of HCC and discuss their diagnostic and therapeutic implications, highlighting their potential to improve patient outcomes through personalized medicine approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e32b3133cdd46cb1e5dedf967c8b74c8ee431edc" target='_blank'>
              Epigenetic Alterations in Hepatocellular Carcinoma: Mechanisms, Biomarkers, and Therapeutic Implications
              </a>
            </td>
          <td>
            A. Wali, Abid Reza Ansari, P. Mir, Mohamed El-Tanani, Rasha Babiker, M.D.S.A.D.I.Q.U.E. Hussain, Jasreen Uppal, Asma Ishrat Zargar, R. H. Mir
          </td>
          <td>2025-08-27</td>
          <td>Pharmaceuticals</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Extrachromosomal DNA amplification is associated with poor cancer prognoses1. Large numbers of excised signal circles (ESCs) are produced as by-products of antigen receptor rearrangement during V(D)J recombination2,3. However, current dogma states that ESCs are progressively lost through cell division4. Here we show that ESCs replicate and persist through many cell generations and share many properties in common with circular extrachromosomal DNAs. Increased ESC copy numbers at diagnosis of B cell precursor acute lymphoblastic leukaemia were highly correlated with subsequent relapse. By taking advantage of the matching recombination footprint that is formed upon the generation of each ESC, we measured ESC persistence and replication and found increased ESC replication in patients who later relapsed. This increased replication is controlled by cell-intrinsic factors and corresponds to increased expression of DNA replication- and repair-associated genes. Consistent with high ESC levels having a role in disease progression, the number of mutations typical of those caused by the V(D)J recombinase–ESC complex was significantly increased at diagnosis in patients who later relapsed. The number of such mutations in genes associated with relapse increased between diagnosis and relapse, and corresponded to clonal expansion of cells with high ESC copy numbers. These data demonstrate that the by-product of V(D)J recombination, when increased in abundance, potently associates with the V(D)J recombinase to cause adverse disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7238ca759d86a62210ebf70bb830d0369e9ff71e" target='_blank'>
              Excised DNA circles from V(D)J recombination promote relapsed leukaemia
              </a>
            </td>
          <td>
            Zeqian Gao, James N. F. Scott, Matthew P Edwards, Dylan Casey, Xiaoling Wang, Andrew D Gillen, S. Ryan, Lisa J Russell, Anthony V Moorman, Ruth de Tute, Catherine Cargo, Anthony M Ford, D.R. Westhead, J. Boyes
          </td>
          <td>2025-08-06</td>
          <td>Nature</td>
          <td>2</td>
          <td>18</td>
        </tr>

        <tr id="Identifying the factors that contribute to individual susceptibility to cancer is essential for both prevention and treatment. The advancement of biotechnologies, particularly next-generation sequencing, has accelerated the discovery of genetic variants linked to cancer susceptibility. While hundreds of cancer-susceptibility genes have been identified, they only explain a small fraction of the overall cancer risk, a phenomenon known as "missing heritability". Despite progress, even considering factors such as epistasis, epigenetics, and gene-environment interactions, the missing heritability remains unresolved. Recent research has revealed that an individual's microbiome composition plays a significant role in cancer susceptibility through several mechanisms, such as modulating immune cell activity and influencing the presence or removal of environmental carcinogens. In this review, we examine the multifaceted roles of the microbiome in cancer risk and explore gene-microbiome and environment-microbiome interactions that may contribute to cancer susceptibility. Additionally, we highlight the importance of experimental models, such as collaborative cross mice, and advanced analytical tools, like artificial intelligence, in identifying microbial factors associated with cancer risk. Understanding these microbial determinants can open new avenues for interventions aimed at reducing cancer risk and guide the development of more effective cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec7fdd77d7c90fbf86b583872501b6376ea77e5" target='_blank'>
              Emerging multifaceted roles of the microbiome in cancer susceptibility
              </a>
            </td>
          <td>
            Hang Chang, J. Pérez-Losada, J. Mao
          </td>
          <td>2025-09-24</td>
          <td>World Journal of Clinical Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Intratumor heterogeneity is a hallmark of cancer, enabling subpopulations of cells to evade therapy, adapt to immune attack, and thrive in diverse microenvironments. While retrospective genomic and epigenomic analyses have mapped the large-scale histories of tumor evolution, they cannot capture the rapid, dynamic changes in cell state that occur as individual cells divide. Panagopoulos and colleagues recently developed a cellular platform to monitor the role of transient replication stress in real time, tracking sister cells as they divide and replicate. The authors use these techniques to show that daughter and granddaughter cells can inherit very different states, often leading to further cellular instability. This work broadens our understanding of how diverse cell states arise from oncogenic stress and how cellular heterogeneity emerges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beadadba41d03604ec2c3cef748b8a0844e1482c" target='_blank'>
              Generations in Flux: Tracking the Birth of Cellular Heterogeneity in Cancer.
              </a>
            </td>
          <td>
            Aaron McKenna
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b348bc9a3e4abf3cb6086b89e0d2182bceec0bd" target='_blank'>
              NEK9-mediated Wnt signalling repressor TLE3 rewires Docetaxel resistance in cancer cells by inducing pyroptosis.
              </a>
            </td>
          <td>
            S. Ansari, Sibasish Mohanty, P. Mohapatra, Sudeshna Datta, Mamuni Swain, Rachna Rath, D. Muduly, Saroj K D Majumdar, R. Swain, Sunil K Raghav, Rupesh Dash
          </td>
          <td>2025-09-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genomic antigen loss is a recurring mechanism of resistance to chimeric antigen receptor T-cell (CAR-T) and T-cell engagers (TCE) in relapsed/refractory multiple myeloma (RRMM). Yet, it remains unclear whether these events are acquired under treatment or merely selected from pre-existing, undetectable clones. By leveraging chemotherapy mutational signatures as temporal barcodes within whole genome sequencing data, we could time genomic antigen escape in 4 out of 11 RRMM patients. In all cases, the biallelic loss was driven by genomic events acquired after exposure to BCMA- and GPRC5D-targeted CAR-T/TCE, and not present at baseline. Longitudinal digital PCR analysis corroborated that resistance mutations were undetectable at therapy initiation but emerged preceding relapse. Among 752 newly diagnosed patients only 2.7% and 9% had monoallelic inactivation of TNFRSF17 and GPRC5D, respectively, with no biallelic loss. Our findings suggest limited utility of mutational screening prior to CAR-T/TCE, while underscoring the importance of dynamic surveillance during therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481837333886a5830de7147acdc462180a9ceabf" target='_blank'>
              Timing genomic antigen loss in multiple myeloma treated with T-cell redirecting immunotherapies.
              </a>
            </td>
          <td>
            M. Papadimitriou, Sungwoo Ahn, B. Diamond, Holly Lee, J. McIntyre, M. Truger, Michael A Durante, B. Ziccheddu, Katalin Osz, O. Landgren, L. Rasche, Nizar J. Bahlis, Paola Neri, F. Maura
          </td>
          <td>2025-09-04</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81bcddaf13d081e97df3be63dd2c84368f07c4" target='_blank'>
              mTOR activity and metabolic reprogramming of CD8+ T cells is impaired under hypoxia and within the multiple myeloma bone marrow.
              </a>
            </td>
          <td>
            Taylor Fulton-Ward, N. Gudgeon, Isaac Thirlwell, E. Bishop, Bryan Marzullo, Hannah Giles, Graham McIlroy, Paul Ferguson, B. Kishore, Kate Rogers, Nuri Nuri Alfasi, Timothy Wong, Satnam Aytain, Daniel A. Tennant, G. Pratt, Sarah Dimeloe
          </td>
          <td>2025-09-12</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="INTRODUCTION
Metastatic colorectal cancer (mCRC) remains a leading cause of cancer mortality worldwide, with limited long-term survival despite therapeutic advances. The increasing understanding of its molecular heterogeneity has paved the way for precision medicine approaches aiming to optimize treatment efficacy and reduce unnecessary toxicity.


AREAS COVERED
This review provides an in-depth analysis of the current and emerging molecular targets in mCRC, including RAS, BRAF, HER2, and microsatellite instability. We discuss the clinical relevance of tumor sidedness, hyperselection panels, EGFR ligand expression, and rare alterations such as NTRK, RET, and ALK fusions. The review also explores the evolving role of KRAS G12C inhibitors, HER2-targeted therapies, and the application of liquid biopsy - particularly circulating tumor DNA (ctDNA) - in treatment monitoring, rechallenge strategies, and resistance detection. Literature was selected through a comprehensive review of recent clinical trials, consensus guidelines, and translational studies.


EXPERT OPINION
Personalized treatment is now an attainable goal in mCRC. While promising, broader implementation of molecular-driven strategies requires overcoming challenges such as resistance mechanisms, assay standardization, and equitable access. The integration of innovative agents with real-time molecular monitoring holds the key to a more dynamic and effective management of mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d84552ed9fbf737b1d558e20dc110dcd8268ee" target='_blank'>
              Personalized therapy in metastatic colorectal cancer: biomarker-driven use of biologics.
              </a>
            </td>
          <td>
            F. Sullo, Chiara Gallio, Laura Matteucci, Alessandro Bittoni, M. Muratore, Luca Esposito, Bianca Ceredi, Graziana Gallo, Paola Ulivi, I. Rapposelli, A. Passardi
          </td>
          <td>2025-09-04</td>
          <td>Expert opinion on biological therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="This review provides a comprehensive synthesis of current knowledge on immunotherapy resistance in non-small cell lung cancer (NSCLC), a disease that accounts for approximately 85% of all lung cancer cases and remains the leading cause of cancer-related death worldwide. Although immune checkpoint inhibitors (ICIs) have significantly improved survival for a subset of patients with advanced NSCLC, over 70% of cases ultimately exhibit primary or acquired resistance, underscoring the urgent need to understand the underlying mechanisms. The review categorizes resistance into tumor-intrinsic and tumor-extrinsic processes and provides an in-depth mechanistic analysis of how factors such as tumor antigen loss, impaired antigen presentation, cGAS-STING pathway dysregulation, metabolic reprogramming in tumor microenvironment (TME), immune cell exhaustion, and microbiomes collectively contribute to immune escape. In parallel, the influence of the lung and gut microbiome on shaping immunotherapy responses is discussed, with emphasis on microbial dysbiosis, immunosuppressive metabolite production, and TME remodeling. Therapeutic strategies to overcome resistance are also discussed, including combination approaches involving chemotherapy, radiotherapy, and antiangiogenic agents, as well as epigenetic modulators (HDAC and BET inhibitors). Moreover, the review explores bispecific antibodies, antibody-drug conjugates, and small-molecule agents that enhance T cell function or disrupt immunosuppressive signaling networks. By integrating insights from preclinical models and clinical trials, the review underscores the necessity of biomarker-guided patient stratification, combination immunotherapy approaches, and interventions that restore tumor immunogenicity. It concludes that a multipronged therapeutic strategy, one that addresses both immune evasion and TME-induced suppression, holds the greatest promise for improving response durability and advancing personalized immunotherapy for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df76f6889b4c3ea7f132198d5c6301465fae771" target='_blank'>
              Immunotherapy resistance in non-small cell lung cancer: from mechanisms to therapeutic opportunities
              </a>
            </td>
          <td>
            Huiyu Wang, Xiaomin Niu, Zhenning Jin, Shaoxing Zhang, Rong Fan, Hua Xiao, Shen S. Hu
          </td>
          <td>2025-08-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Single‐cell RNA sequencing (scRNA‐seq) has transformed our understanding of tumours by enabling high‐resolution profiling of their cellular composition. Traditionally perceived as masses of homogeneous cancer cells, tumours are now recognised as complex ecosystems shaped by the tumour microenvironment (TME), which includes diverse immune cells, cancer‐associated fibroblasts and extracellular matrix components. scRNA‐seq has revealed remarkable heterogeneity within the TME, identifying novel or rare immune cell subsets and delineating their dynamic functional states. In particular, it has illuminated intercellular signalling networks and temporal cell‐state transitions that drive tumour progression and immune evasion. Moreover, the integration of scRNA‐seq data with clinical information has highlighted its potential in improving patient stratification, biomarker discovery and therapeutic target identification. Here, we systematically summarise recent advances in applying scRNA‐seq to dissect the TME, discuss the implications of these findings for immunotherapy resistance and precision oncology, and outline future opportunities for integrating scRNA‐seq with emerging technologies to develop more effective and personalised cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f7bf7b9a7f8397821baf87f91e3e07f64fb492a" target='_blank'>
              Tumour Single‐Cell Bioinformatics: From Immune Profiling to Molecular Dynamics
              </a>
            </td>
          <td>
            Jane Siu Fan Li, Zoey Zeyuan Ji, Aaron Qi Zhang, C. Ng, Guibin Qiao, Dongmei Zhang, Chunjie Li, Qing Zhang, Ka-Fai To, P. M. Tang
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The purpose of the study was a comprehensive analysis of molecular genetic changes in tumor and normal tissues of patients with colorectal cancer using whole-genome sequencing to identify de novo mutations, microsatellite instability patterns, and mutational signatures associated with tumor development.Material and Methods. A single-time cross-sectional study was conducted in the Kaliningrad region. Colorectal cancer samples were analyzed using whole-genome next-generation sequencing (WGS). The human reference genome GRCh38.p14 was used for read mapping. Bioinformatic analysis of signatures was conducted separately for each patient. 71 pairs of samples were sequenced: complete genomes of tumor biopsies and DNA isolated from blood samples for 71 patients. The average age among the analyzed patients was 69 years, with a median of 71 years.Results. Genetic alterations associated with tumor development and progression, as well as potential therapeutic targets were identified. Using the Oncovar database, 151 driver mutations were detected in 62 patients. Using the CIVIC database, 70 marker mutations were found in 45 patients. It was found that the most common driver mutations in the study cohort were mutations in the KRAS gene (KRAS:G35T, KRAS:G35A, KRAS:G38A), associated with tumor resistance to Panitumumab and Cetuximab. The frequency of the most common driver mutations in the study cohort was low, indicating high heterogeneity of colorectal cancer. Analysis of the tumor mutational burden (TMB) revealed a high average value of 5.5, suggesting a significant potential of immunotherapy in colorectal cancer treatment. In the studied population, the mutational signatures SBS1, SBS5, and SBS39 were the most common.Conclusion. The identified diversity of genomic changes may be associated with DNA replication errors, exposure to external factors, and genetic predisposition. The pattern of mutational signatures was very similar among all patients in the studied samples, suggesting a single mutagenic factor is involved in the development of colorectal cancer in all patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/410fb5988f2bc832f5b1b887492d03eded83e97b" target='_blank'>
              Genetic landscape of colorectal cancer. A pilot study in the Russian Federation
              </a>
            </td>
          <td>
            E. V. Semina, K. A. Kubasova, V. Kakotkin, S. V. Rodimov, M. Agapov
          </td>
          <td>2025-08-04</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive subtype of breast cancer, lacking the expression of estrogen receptor, progesterone receptor, and HER2, which leads to poor prognosis and limited treatment options. Despite advances in targeted therapies, TNBC patients often fail to benefit due to its heterogeneity, drug resistance, and immune evasion. Gene editing technologies such as CRISPR/Cas9 and RNA interference offer promising strategies for precise gene modulation, yet their clinical translation is challenged by delivery efficiency, off-target effects, and safety concerns. This review systematically summarizes key regulatory genes implicated in TNBC progression, including those involved in invasion, proliferation, and chemoresistance, such as BRCA1/2, TP53, MUC1, EGFR, MYC, and others. We further discuss advances in gene editing tools and their combination with targeted delivery materialsranging from viral vectors (AAV, lentivirus) to nonviral platforms (lipid nanoparticles, polymer-based systems, and bioderived vesicles)highlighting their potential to enhance editing specificity, minimize immune response, and overcome tumor microenvironment barriers. Finally, we address biosafety, ethical concerns, and mitigation strategies such as high-fidelity Cas9 variants and AI-assisted off-target prediction. Collectively, this review provides a comprehensive framework for future research and clinical application of targeted gene editing in TNBC, aiming to develop safer, more effective, and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01340f159e29a0963fbf18ee30b93bcf3d8bf597" target='_blank'>
              Application of Gene Editing Technology Based on Targeted Delivery Materials in TNBC
              </a>
            </td>
          <td>
            Peng Qu, Xue Li, Jun Liu, Xiaohan Su, Shiqi Han, Yali Wang, Yunbo Luo, Jinsui Li, Cui Ma, Shishan Deng, Lingmi Hou, Qi Liang, Panke Cheng
          </td>
          <td>2025-09-08</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transfer RNA-derived fragments (tRFs) are a class of small non-coding RNAs that have exhibited several functions in cancer. Recent studies have shown that mutations in tRNA genes can lead to global changes in tRF expression levels and may affect tRF function, highlighting the need to further elucidate the regulation and functions of tRFs in cancer. Here, we conducted a pan-cancer analysis of tRF quantitative trait loci (tRFQTLs), encompassing 16,703 genetic variants associated with tRF expression across 31 cancer types. A joint analysis of GWAS data revealed that tRFQTLs were preferentially enriched in cancer risk loci and colocalized with 106 GWAS variants, explaining a substantial portion of cancer heritability. Moreover, tRFs regulated by tRFQTLs were enriched in cancer-related pathways and correlated with drug response and immune infiltration. Notably, polygenic risk score models incorporating tRFQTLs improved high-risk population identification. Investigation of large-scale population cohorts revealed a tRFQTL, rs9461276, associated with colorectal cancer (CRC) risk. In biological assays, the rs9461276-C allele increased tRF-18-HSQS52D2 expression, which suppressed CRC malignant phenotypes. Mechanistically, tRF-18-HSQS52D2 bound to the 3'UTR of POU2F1, destabilizing the oncogenic transcript. Integrated RNA sequencing and ChIP-seq assays indicated that POU2F1 enhanced CRC cell proliferation by activating various pathological pathways associated with oxidative and glycolytic metabolism, mitotic stability, and cell cycle regulation. Finally, a database (Cancer-tRFQTL) was generated as a resource to support investigation into tRF-mediated mechanisms and genetic basis of tRF expression in human cancers. Overall, this study helps advance the understanding of tRFs in cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/888d9f82216c90d7528814f40ba0327b18c81f36" target='_blank'>
              Genetic Control of tRNA-Derived Fragments Contributes to Cancer Risk.
              </a>
            </td>
          <td>
            Bin Li, Hanting Li, Yan Li, Zhirui Chen, Danyi Zou, Chunyi He, Songqiang Li, Kexin Hu, Caibo Ning, Yanmin Li, Hui Geng, Shuhui Yang, Chaoqun Huang, Xiaojun Yang, Qiuhong Wang, Xu Zhu, Xiangpan Li, Ying Zhu, Meng Jin, Xiaoping Miao, Jianbo Tian
          </td>
          <td>2025-08-07</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Nasopharyngeal carcinoma (NPC) is an epithelial malignancy closely associated with Epstein-Barr virus (EBV) infection. Although patients with early-stage NPC can achieve a high cure rate through radiotherapy, recurrence and distant metastasis remain the primary causes of treatment failure in patients with advanced-stage NPC. Circular RNA (circRNA) is a class of covalently closed non-coding RNAs involved in multiple aspects of tumor biology. Recent evidence has shown that certain circRNAs can encode functional peptides, which participate in the regulation of tumor-related signaling pathways. In NPC, circRNAs have been implicated in the modulation of signaling pathways, including NF-κB and JAK/STAT, both of which are activated in the EBV-infected microenvironment. Furthermore, frequently mutated genes in NPC, such as TNF receptor-associated factor 3 and cylindromatosis lysine 63 deubiquitinase, are known regulators of the NF-κB pathway, suggesting a potential link between genetic alterations and circRNA-related mechanisms. This article systematically reviews the biological mechanisms of circRNA-encoded peptides, summarizes the expression and function of circRNA in NPC and focuses on discussing the potential roles of circRNA-encoded peptides in tumor microenvironment regulation, immune escape and clinical application prospects. By integrating existing research results, this article aims to provide a new perspective and theoretical basis for the in-depth exploration of circRNA-encoded peptides in the field of NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026498398de031d10f9b113d1987dfe09fb352cd" target='_blank'>
              Functional mechanisms of circular RNA-encoded peptides and future research strategies and directions in nasopharyngeal carcinoma (Review)
              </a>
            </td>
          <td>
            Wei-hua Xu, Zhichao Ma, Wei Gong, Shengmiao Fu, Xinping Chen
          </td>
          <td>2025-08-18</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the most aggressive and treatment-resistant brain tumors, largely due to its profound intratumoral heterogeneity and immunosuppressive microenvironment. Various classifications of GBM subtypes were created based on transcriptional and methylation profiles. This effort, followed by the development of new technology such as single-nuclei sequencing (snRNAseq) and spatial transcriptomics, led to a better understanding of the glioma cells’ plasticity and their ability to transition between diverse cellular states. GBM cells can mimic neurodevelopmental programs to resemble oligodendrocyte or neural progenitor behavior and hitchhike the local neuronal network to support their growth. The tumor microenvironment, especially under hypoxic conditions, drives the tumor cell clonal selection, which then reshapes the immune cells’ functions. These adaptations contribute to immune evasion by progressively disabling T cell and myeloid cell functions, ultimately establishing a highly immunosuppressive tumor milieu. This complex and metabolically constrained environment poses a major barrier to effective antitumor immunity and limits the success of conventional therapies. Understanding the dynamic interactions between glioma cells and their microenvironment is essential for the development of more effective immunotherapies and rational combination strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0dda0986360e9c65e565d1dc755ad16cfd3932d" target='_blank'>
              Unraveling Glioblastoma Heterogeneity: Advancing Immunological Insights and Therapeutic Innovations
              </a>
            </td>
          <td>
            Joshua H. Liu, Maksym Horiachok, Santosh Guru, Cecile L. Maire
          </td>
          <td>2025-08-01</td>
          <td>Brain Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07528eee7ff3d627e8787aab243a7521b7075a4" target='_blank'>
              Targeting p53-MDM2 pathway with novel triazole-oxazole hybrids: a fragment-based drug discovery approach for next-generation cancer therapies.
              </a>
            </td>
          <td>
            Apurva Prajapati, Hitesh Patel
          </td>
          <td>2025-09-23</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Cancer remains the most lethal disease globally, despite the significant progress made in early screening, surgery, and therapeutic development in recent decades. Programmed cell death (PCD) is a genetically regulated process essential for eliminating aberrant cells, yet its dysregulation drives tumorigenesis and therapy resistance. In this review, we present a complete discovery timeline of them and comprehensively synthesize the roles and mechanisms of major PCD forms, such as apoptosis, necroptosis, autophagy, pyroptosis, ferroptosis, and cuproptosis, across diverse cancer types. We not only detail the molecular mechanisms, dual functions, and alterations of these PCD modalities in cancers, but also summarize their interconnections and intrinsic crosstalk. Furthermore, we comprehensively discuss how diverse therapies, including chemotherapy, radiotherapy, immunotherapy, targeted agents, and hormone therapy, engage and manipulate specific PCD pathways, revealing the involvement of PCD in cancer treatment mechanisms. This review integrates extensive preclinical and clinical evidence on PCD‐targeted therapies with an in‐depth focus on ferroptosis, including its regulatory networks and therapeutic relevance. Special emphasis is placed on prostate cancer, highlighting the PCD‐based translational opportunities in this common malignancy. Taken together, we provide novel insights into the complex interplay between PCD and cancer biology and offer a framework for developing precision oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c45427c1d2eb5ce79e949eeb0b16e4625cc2c1" target='_blank'>
              Programmed Cell Death in Cancer
              </a>
            </td>
          <td>
            Yuang Wei, William Hankey, Dongliang Xu, Fuwen Yuan
          </td>
          <td>2025-08-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Immune checkpoint blockade and cancer immunotherapy have revised the cancer treatment therapeutic environment, but patient reactions are excessively difference. The targeted population in this systematic review is animal trials in order to understand any genetic markers effecting the efficacy of the immunotherapy. The research tries using murine models like syngeneic, genetic engineered mice and humanized mice to define key genes that are determinants of therapeutic response (PD-L1, IFN-gamma, JAK1/2, CTNNB1 and EZH2). High expression of PD-L1 and IFN-γ reported positive correlation with response and survival whereas mutation in JAK1/2 and CTNNB1 activation reported resistance because of ineffective immune infiltration and stimulation events. Inhibition of EZH2 increased immune activity partly which indicates the role of epigenetic control. The predictive value of these markers was confirmed by the analysis of data via ANOVA, Kaplan-Meier survival curves, correlation, and bioinformatics tools. Animal models will continue to be essential to mechanisms, despite species-related and methodological limitation. These results confirm the use of genetic profiling in personalized immunotherapy strategies and point to the importance of additional translational data in models more humanized as well as multi-omics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/845932ec61e6b82c7d0c7ccc9ea937f5a7aad54f" target='_blank'>
              Genetic Markers for Predicting Cancer Immunotherapy Response
              </a>
            </td>
          <td>
            Mamidisetti Sai Ganesh, Nallani Venkata Rama Rao, Nadendla. Rama Rao, Banavathu Tulasya Naik, Doddigaralla Salman Raju
          </td>
          <td>2025-08-04</td>
          <td>Journal of Pharmacology, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id=" 
Background: Accurate and early identification of lung cancer biomarkers is crucial for effective management yet remains a clinical challenge. CRISPR-based technologies provide highly sensitive and specific tools that can be adapted for both diagnostic and therapeutic use in oncology.
Methods: A prospective study was conducted including 120 patients with confirmed lung cancer and 60 healthy individuals as controls at a tertiary care center. A CRISPR-Cas12a diagnostic platform was developed to detect EGFR exon 19 deletions, KRAS G12C mutations, and ALK rearrangements in tumor tissue and plasma samples. In parallel, CRISPR-Cas9 editing was applied to patient-derived lung cancer cell cultures to selectively disrupt EGFR mutant alleles. The diagnostic performance of the CRISPR assay was compared against conventional PCR and next-generation sequencing (NGS).
Results: The CRISPR-Cas12a assay successfully detected as few as 10 DNA copies/µL within one hour. In tissue samples, the assay achieved 96.7% sensitivity and 98.3% specificity relative to NGS. In circulating tumor DNA extracted from plasma, sensitivity reached 92.1%. Functional assays demonstrated that CRISPR-Cas9-mediated disruption of EGFR mutations reduced cell viability by approximately 65% and suppressed downstream oncogenic signaling pathways.
Conclusion: CRISPR-based strategies demonstrate considerable promise for both precise detection and targeted intervention in lung cancer. The integration of CRISPR diagnostics with gene-editing applications could significantly advance precision oncology by enabling earlier diagnosis and patient-tailored therapeutic options.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b66e666da19106418ca11e5bfbe956b66a8dd301" target='_blank'>
              CRISPR-BASED APPROACHES FOR THE DETECTION AND TARGETING OF LUNG CANCER BIOMARKERS: A NEW ERA IN PRECISION ONCOLOGY
              </a>
            </td>
          <td>
            Ibtasam Mazhar, Maimona Sadia, Muhammad Hassam Saleem, Ateeqah Siddique, Areeba Musferah, Areej Safdar, Anood Malik
          </td>
          <td>2025-08-31</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cebb7efc870646ab91862d4a77c390cd760bc112" target='_blank'>
              Dual targeting of CDK6 and LSD1 is synergistic and overcomes differentiation blockade in AML.
              </a>
            </td>
          <td>
            Lise M Brault, E. Voisset, Mathieu Desaunay, Antonia Boudet, Paraskevi Kousteridou, S. Letard, Nadine Carbuccia, A. Goubard, Rémy Castellano, Yves Collette, J. Vernerey, Isabelle Vigon, J. Pasquet, Patrice Dubreuil, Sophie Lopez, P. de Sepulveda
          </td>
          <td>2025-08-29</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract BACKGROUND A major barrier in our understanding of glioblastoma (GBM) is the difficulty in tracking tumour development in real-time. We adopted a novel approach that combines the principles of classic cellular barcoding with CRISPR/Cas-9 technology and single-cell RNA sequencing known as continuous lineage tracing to apply a phylogenetic approach to studying tumour development. METHODS Patient derived glioma initiating cell lines were engineered with expressed DNA barcodes with CRISPR/Cas-9 targets and subsequently implanted to create an intracranial xenograft model. Tumors were sent for single cell RNA sequencing; clonal relationships were surmised through identification of expressed barcodes, and cells were characterized by their transcriptional profiles. Phylogenetic lineage trees were created utilizing lineage reconstructive algorithms to define cell fitness and expansion. RESULTS Our work has revealed a significant amount of intra-clonal cell state heterogeneity, suggesting that tumour cells engage in phenotype switching prior to therapeutic intervention. We defined a consistent transcriptional pattern for tumour engraftment and in vivo clonal advantage. Phylogenetic lineage trees allowed us to define gene signatures of both cell fitness and expansion, which correlate strongly with published neural-mesenchymal and developmental-injury response phenotypes. GBMs exist along a transcriptional gradient between undifferentiated but “high-fit” cells and terminally differentiated, “low-fit” cells: cells with highly fitness appear to represent cells undergoing cell state transition. CONCLUSION We successfully employed a novel lineage tracing technique in GBM, creating a powerful tool for real-time tracing of tumour growth through the analysis of a unique set of highly detailed transcriptional data with associated clonal and phylogenetic relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51825ee39f0efc909dc234d3cc75f48f8c3aa8f6" target='_blank'>
              56 SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS AND EXPANSION AXES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Ajisebutu, F. Fan, Chloe Gui, Y. Ellenbogen, A. Landry, Jeff Liu, Q. Wei, G. Zadeh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The prevalence and death rates of early-onset colorectal cancer (EOCRC) have been increasing at an alarming rate since 1994. Compared with late-onset colorectal cancer (LOCRC), EOCRC is more aggressive and resistant to treatment. Despite
 KRAS
 mutations and the mitogen-activated protein kinase/extracellular signal-related kinase (MAPK/ERK) pathway's significant role in colorectal cancer tumorigenesis and progression, these mutations are less frequent in patients with EOCRC.




 In this study, we conducted proteotranscriptomic profiling of two independent cohorts (MD Anderson for testing and The Cancer Genome Atlas for validation), comparing EOCRC and LOCRC tumors. Our focus was on sporadic, microsatellite-stable, treatment-naïve patients with early-onset (<50 years) and late-onset (>60 years) colorectal adenocarcinomas.




 Our analysis revealed consistent molecular differences between EOCRC and LOCRC, particularly in pathway enrichment profiles. The two pathways with the highest levels of significant gene set enrichment analysis in EOCRC tumors were the downregulation of epithelial-mesenchymal transition and upregulation of Wnt/β-catenin signaling pathways. Both testing and validation cohorts showed a consistent enrichment in hallmark KRAS signaling. However, using the novel KRAS-ERK gene signatures to better capture the pathway signaling functionality, we identified a consistent upregulation of MAPK signaling specifically in EOCRC tumors. Further analysis of this enrichment in wild-type RAS GTPase (
 KRAS
 ,
 NRAS
 , and
 HRAS
 ) tumors confirmed significant MAPK upregulation in EOCRC wild-type RAS tumors (
 P
 < .001). Proteomic analysis further supported these transcriptomic findings, reinforcing the molecular distinctions in EOCRC.




 This study not only elucidates the unique molecular attributes of EOCRC but also lays the groundwork for exploring MAPK-targeted therapies specifically for this age group. Given the increasing prevalence of EOCRC, our findings highlight the need for larger investigations into MAPK upregulation in EOCRC and its potential as a target for precision therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5365b8787a6860734caebe70b8eefa54d1dc39aa" target='_blank'>
              Proteotranscriptomic Analyses of Microsatellite-Stable Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            J. Alshenaifi, Giulia Maddalena, Jessica C. Lal, Oluwadara Coker, Hey Min Lee, O. Villarreal, G. Manyam, Alaa M. Mohamed, G. Vetere, Micheal White, N. Ajami, Preeti Kanikarla Marie, Alexey Sorokin, David Menter, Y. N. You, Andy Futreal, S. Kopetz
          </td>
          <td>2025-08-01</td>
          <td>JCO Oncology Advances</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The RAS signaling pathway, particularly through mutations in KRAS, NRAS, and HRAS, plays a pivotal role in driving oncogenesis in a wide range of cancers. For years, RAS proteins were deemed "undruggable" due to their smooth surface and lack of deep binding pockets. However, recent breakthroughs in targeting specific RAS mutations, particularly KRASG12C, have revolutionized the field. The discovery of covalent inhibitors that bind to an allosteric pocket near the cysteine residue of KRASG12C has led to the development of FDA-approved drugs, marking a significant milestone in RAS-targeted therapy. This review provides a comprehensive overview of the evolution of direct RAS inhibitors, focusing on the chemical development of small molecule inhibitors, molecular glues, protein degraders, and other emerging strategies. We highlight the structural evolution of KRAS inhibitors, from covalent fragment-based approaches to non-covalent inhibitors and pan-RAS targeting strategies. Additionally, we discuss the clinical progress of key inhibitors, including their efficacy, resistance mechanisms, and combination treatment options. Finally, this review explores other innovative approaches such as cyclopeptide inhibitors and outlines future directions of RAS-targeting strategies. The success of RAS-targeted therapies underscores the transformative potential of overcoming the "undruggable" nature of RAS, offering new hope for patients with RAS-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5391dac665b788fff815474bfa40f19a2c086765" target='_blank'>
              Evolution of direct RAS inhibitors: from undruggable target to clinical breakthroughs
              </a>
            </td>
          <td>
            Xia Wang, Jing Wu, Aotian Xiao, Jie Wang, Jun Tian
          </td>
          <td>2025-09-24</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human papillomavirus (HPV) is a major etiological factor in cervical, anal, and oropharyngeal cancers. Although prophylactic vaccines have substantially reduced infection rates, effective therapeutic options for established HPV-associated malignancies remain limited. This review provides an up-to-date overview of emerging strategies to treat HPV-driven tumours. Key approaches include immune checkpoint inhibitors, therapeutic vaccines such as VGX-3100 and PRGN-2012, and gene-editing tools like CRISPR/Cas9. Epigenetic drugs, particularly histone deacetylase inhibitors, show promise in reactivating silenced tumour suppressor genes and enhancing antitumour immunity. In addition, natural bioactive compounds and plant-derived molecules are being explored as complementary anti-HPV agents, while drug repurposing and combination therapies offer cost-effective opportunities to broaden treatment options. We also highlight the role of patient-derived organoid models as powerful platforms for personalized drug screening and functional assessment. By integrating these therapeutic innovations with precision oncology approaches, this review outlines a multidimensional framework aimed at improving clinical outcomes and quality of life for patients with HPV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c470c11ec7ad7e2174f0a27714b34305b11d10e" target='_blank'>
              Emerging Therapeutic Strategies for HPV-Related Cancers: From Gene Editing to Precision Oncology
              </a>
            </td>
          <td>
            M. O. Cakir, Guldide Kayhan, B. Yilmaz, Mustafa Ozdogan, G. H. Ashrafi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies, with limited treatment options and poor prognosis. Recent advances in cancer genomic analysis enable the identification of actionable gene alterations, opening new opportunities for personalized therapy. Among these, homologous recombination DNA repair (HRR) gene alterations are associated with distinct biological behavior, favorable prognosis, and increased sensitivity to platinum-based chemotherapy. However, the prognostic impact of coexisting mutations in key driver genes—KRAS, TP53, CDKN2A, and SMAD4—within HRR-altered PDAC remains poorly understood. Methods We retrospectively analyzed PDAC patients who underwent genomic profiling testing with FoundationOne® CDx between June 2019 and December 2021 through the Center for Cancer Genomics and Advanced Therapeutics (C-CAT) database. We compared the prevalence and prognostic significance of key gene alterations between HRR-altered and HRR–wild-type (WT) tumors. Results Of 2,381 PDAC patients, 274 (11.5%) harbored HRR alterations. These patients showed significantly longer overall survival (OS) than those with HRR-WT tumors (HR = 0.66, p = 0.002). The frequencies of KRAS, TP53, and CDKN2A mutations were less frequent in HRR-altered tumors. TP53 mutation was independently associated with poorer OS across both HRR subgroups, while CDKN2A alteration was a poor prognostic factor in HRR-WT tumors. Interestingly, SMAD4 alteration was linked to improved survival in the HRR-altered group. Conclusion HRR-altered PDAC has a distinct genomic profile and is associated with a favorable prognosis. Our findings demonstrate that coexisting alterations are significant prognostic factors in both HRR-altered and HRR–wild-type tumors. These results highlight the clinical relevance of incorporating comprehensive genomic profiling into routine care to stratify patient prognosis better and inform individualized treatment strategies in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada42c90fd2263fd4d9bb1466e612ccff4b0a39f" target='_blank'>
              Prognostic impact of gene alterations via homologous recombination DNA repair gene alteration status in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yusuke Kawanaka, Chiaki Inagaki, Masaki Okura, S. Mitani, T. Takahama, K. Yonesaka, Y. Chiba, Kazuhiko Nakagawa, Hisato Kawakami, H. Hayashi
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="


 Cisplatin remains a standard first-line therapy for epithelial ovarian cancer; however, the development of chemoresistance leads to poor prognosis and high recurrence rates. Kaplan–Meier analysis of patient data (TCGA) revealed that cisplatin-sensitive tumors are associated with significantly improved overall survival, underscoring the need for strategies to overcome resistance in clinical settings. To investigate the underlying mechanisms, we conducted a comprehensive bioinformatics analysis of publicly available GEO datasets from cisplatin-treated ovarian cancer patients (n = 255). This analysis identified six key molecular drivers of resistance: ZBTB33 (Kaiso), pregnane X receptor (PXR), NF-κB, HER2 (ERBB2), P-glycoprotein (P-gp/ABCB1), and HIF1A. These targets were validated in ovarian tumor specimens via immunohistochemistry, confirming elevated expression in chemo-resistant disease. Additionally, the quantitative real-time PCR analysis confirms the transcriptional upregulation of the six resistance-associated genes in cisplatin-resistant SKOV3 and OVCAR-5 ovarian cancer cells, consistent with the IHC findings. The average fold change in mRNA transcripts ranged from 2.4 for P-glycoprotein (P-gp) to 5 for both NF-kB and Kaiso (ZBTB33). Although less well studied in ovarian cancer, Kaiso (ZBTB33) is known to regulate EMT and tumor invasion in other solid tumors. Functional studies using SKOV3 and OVCAR-5 cell lines demonstrated that knockdown of Kaiso (ZBTB33) via RNA interference significantly increased cisplatin-induced cell death, indicating a direct role in therapeutic resistance. Furthermore, we investigated the synergistic effects of combining SDA, a plant-based omega-3 fatty acid known to inhibit NF-κB, and cisplatin on the cell death of SKOV3 and OVCAR-5 cell lines, comparing the results to those of each compound used individually. Interestingly, co-treatment with stearidonic acid (SDA) synergistically enhanced the cytotoxicity of cisplatin at a lower dose in both cell models. These findings reveal a clinically relevant resistance signature and highlight the therapeutic potential of combinatorial strategies that target both transcriptional regulators (e.g., Kaiso) and inflammatory signaling (e.g., NF-κB). Dual targeting of these pathways may resensitize tumors to cisplatin and improve outcomes for patients with advanced ovarian cancer.



 Mahmoud Mansour, Sabrina Van Ginkel, Maaz Kamal Alata, Ibrahim Bani, Isra Elhussin. Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a654db1a318364cee4d66d47d1f28ac0d3c685d" target='_blank'>
              Abstract A025: Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization
              </a>
            </td>
          <td>
            Mahmoud Mansour, Sabrina Van Ginkel, M. Alata, Ibrahim Bani, Isra Elhussin
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="All forms of leukemia, including AML, ALL, CML, and CLL, are estimated to make up 2.5% of new cancer diagnoses and 3.1% of all cancer fatalities worldwide. Leukemia is mainly propelled by LSCs that possess remarkable self-renewal and differentiation abilities, comparable to those of regular HSCs. The presence of LSCs is crucial for the progress and amplification of leukemia, as they enable evasion of the immune system and resilience to therapy. ncRNAs have emerged as a potential means of regulating various forms of cancer, owing to their role in diverse cellular processes. Long non-coding RNAs, microRNAs, and Circular RNAs are among the most studied ncRNAs and exert control over the expression of numerous proteins related to the advancement of malignancies. Given the importance of LSCs in leukemia and the potential of ncRNAs, we aimed to explore the ncRNAs abnormally expressed in leukemia and investigate them as therapeutic targets. In addition, in our review, we thoroughly examined the ncRNAs responsible for conferring chemo-resistance to LSCs. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02836-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1432d88b3fafb495184caaf567eb5a5ef26b6d53" target='_blank'>
              The roles of non-coding RNAs (ncRNAs) in the function of leukemic stem cells (LSCs): a comprehensive review
              </a>
            </td>
          <td>
            Mehran Sharifi, Shahram Nekoeian, Reza Eshraghi, Ehsan Behzadi, Ali Rajabi
          </td>
          <td>2025-08-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Gene products from the highly variable major histocompatibility locus, including HLA, are essential for self-recognition and immune surveillance of malignancy. Following allogeneic hematopoietic cell transplantation (alloHCT), genetic and epigenetic alterations in HLA can drive disease recurrence, making precise HLA assessment critical for determining future therapy. However, current methods lack the sensitivity to quantify HLA transcripts at the single-cell level, limiting their clinical utility. We introduce scrHLA-typing , a novel technique that accurately identifies and quantifies HLA transcripts in single cells using long-read sequencing. When applied to samples from patients with post-transplant relapse, scrHLA-typing successfully detected HLA allele-specific expression, across a range of levels of donor-recipient chimerism, at clinically actionable levels. By characterizing allele expression in residual leukemia cells, our assay identified differences in expression patterns among patients. This capability highlights scrHLA-typing’s potential to improve risk stratification and guide the selection of appropriate salvage therapies, enhancing personalized treatment strategies after relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/291c4c43ac598982cc22e914879048e23b09a038" target='_blank'>
              Quantifying HLA transcripts by genotype in chimeric mixtures at single-cell resolution
              </a>
            </td>
          <td>
            Sami B. Kanaan, Jason G. Underwood, Rula Green Gladden, Everett Fan, Shruti S Bhise, Monica Thakar, Carla A. Jaeger-Ruckstuhl, Jeffrey Stevens, Ashley N. Gray, Stanley R. Riddell, Marie Bleakley, S. Meshinchi, Scott N. Furlan
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Breast cancer (BC) is among the most prevalent malignancies globally. It is progressively acknowledged as a diverse type of cancer, exhibiting considerable differences in its genomic and transcriptomic characteristics. Its growing evidence highlights the substantial role of epigenetic modification in pathogenesis, prognosis and treatment. Cancer and epigenetics are closely linked; abnormal epigenetic changes can influence numerous aspects of cancer biology, including unusual transcription patterns, initiation of cancer, its progression, resistance to drugs, and metastasis. Epigenetic drugs (epi-drugs), including DNA methyltransferase (DNMT) and histone deacetylase (HDAC) inhibitors, serve as promising therapeutic agents, particularly in combination with conventional therapies. Additionally, nanotechnology-assisted epi-drug delivery systems are emerging as innovative approaches to enhance treatment efficacy and reduce systemic toxicity. While several epigenetic biomarkers have shown potential in liquid and tissue biopsies, their clinical validation remains a challenge. The integration of epigenetic insights into personalized medicine could revolutionize BC management, offering more targeted and effective treatment strategies. This systematic review aims to evaluate recent advancement in epigenetic research related to BC, focusing on diagnostic and prognostic biomarkers, epigenetic-based therapies and ongoing clinical trials. A comprehensive literature search was carried out in databases like PubMed, Scopus, and Google Scholar up to January 2025, following PRISMA guidelines. Seventy two (72) studies were included, addressing key aspects of DNA methylation, histone modification, and non-coding RNAs as potential biomarkers for early detection and disease progression monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f25e3d13419223a1552ed3674dcac4a437328a1" target='_blank'>
              A systematic review of progress toward unlocking the power of epigenetics in breast cancer: latest updates and perspectives
              </a>
            </td>
          <td>
            Ghallab Alotaibi
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/705a9b21b7c445d9d6a328fd2aa7da43797bf85e" target='_blank'>
              Strain-Level Genetic Heterogeneity and Colonization Dynamics Drive Microbiome Therapeutic Efficacy
              </a>
            </td>
          <td>
            K. Chen, Y. Liu, J. Rong, N. Dai, C. Xu, H. Li, L. Zhong, B. Wang, Z. Ji, S. Xie, Y. Xu, F. Yang, J. Wang, D. Li, Y. Gu, X. Zhou, Y. Li, M. Chen, Y. Chen, W. Li, Z. Tang, J. Cai, J. Xu, S. Xia, Q. Zhan, Z. Zhou
          </td>
          <td>2025-08-19</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z. Lin, Dylan Schaff, Sydney M. Shaffer, Drew M Pardoll, Christina Jackson, Nancy R. Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7188c988056f790ffe3551531682c3604b9aecad" target='_blank'>
              Integrative Gene-Centric Analysis of Breast Cancer: High-Confidence Germline Predisposition Genes from Population-Scale Cohorts
              </a>
            </td>
          <td>
            R. Zucker, S. Schreiber, A. Stern, M. Linial
          </td>
          <td>2025-09-12</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Malignant tumors remain a major threat to global human health. This study aimed to systematically integrate multi-omics data to identify a candidate gene with biomarker potential across diverse cancer types and to evaluate its possible clinical applications in oncology. Methods We first performed Mendelian randomization based on summary statistics to integrate blood expression quantitative trait loci data with genome-wide association study results from esophageal adenocarcinoma, stomach cancer, and clear cell renal cell carcinoma. A comprehensive series of multi-omics bioinformatics analyses was subsequently conducted to assess the gene’s expression patterns, genomic alterations, prognostic relevance, and associations with the tumor microenvironment (TME) across various cancer types. In addition, single-cell transcriptome data were analyzed to explore the gene’s functional roles in the TME. The key findings were further validated through in vitro experiments. Results Mendelian randomization identified peptidylprolyl isomerase H (PPIH) as a potential biomarker across multiple malignancies. Single-cell transcriptome analysis suggested that this gene may enhance the proliferative ability of malignant cells and participate in communication between immune and stromal components in the TME. Multi-omics analyses revealed that the gene is abnormally expressed and significantly correlated with patient prognosis in several cancer types. Consistently, in vitro assays demonstrated that increased expression of PPIH promotes the proliferation, migration, and invasion of hepatocellular carcinoma (HCC) cells. Conclusion This study highlights PPIH as a candidate biomarker with pan-cancer relevance and potential clinical value. These findings offer new directions for cancer diagnosis and provide a foundation for further development of targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f155a791f569f5293a783f76610a44c07c11b98" target='_blank'>
              eQTL and multi-omics integration reveal PPIH as a prognostic and immunotherapeutic biomarker
              </a>
            </td>
          <td>
            Fenglin Lv, Xinlu Zhang, Yanmei Wu, Zhipeng Li, Xiaomen Zheng, Huaxin Zhou, Wei Wang
          </td>
          <td>2025-08-14</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. M. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease characterized by various genetic and epigenetic alterations. Early diagnosis, accurate subtyping, and staging are essential for effective, personalized treatment and improved survival rates. Traditional diagnostic methods, such as biopsies, are invasive and carry operational risks that hinder repeated use, underscoring the need for noninvasive and personalized alternatives. In response, this study integrates transcriptomic data into human genome‐scale metabolic models (GSMMs) to derive patient‐specific flux distributions, which are then combined with genomic, proteomic, and fluxomic (JX) data to develop a robust multi‐omic classifier for lung cancer subtyping and early diagnosis. The JX classifier is further enhanced by analyzing heterogeneous datasets from RNA sequencing and microarray analyses derived from both tissue samples and cell culture experiments, thereby enabling the identification of key marker features and enriched pathways such as lipid metabolism and energy production. This integrated approach not only demonstrates high performance in distinguishing lung cancer subtypes and early‐stage disease but also proves robust when applied to limited pancreatic cancer data. By linking genotype to phenotype, GSMM‐driven flux analysis overcomes challenges related to metabolome data scarcity and platform variability by proposing marker processes and reactions for further investigation, ultimately facilitating noninvasive diagnostics and the identification of actionable biomarkers for targeted therapeutic intervention. These findings offer significant promise for streamlining clinical workflows and enabling personalized therapeutic strategies, and they highlight the potential of our versatile workflow for unveiling novel biomarker landscapes in less studied diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c867c0d4475941417554e4edd0273dd0301a176c" target='_blank'>
              Multi‐omic data integration and exploiting metabolic models using systems biology approach increase precision in subtyping and early diagnosis of cancer
              </a>
            </td>
          <td>
            Ezgi Tanıl, E. Nikerel
          </td>
          <td>2025-08-05</td>
          <td>Quantitative Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metastasis is the main cause of failure in anticancer therapies, and is frequently related to poor prognosis for patients. The true challenge in extending cancer patient life expectancy, eventually managing cancer as a chronic disease with periodic but controllable relapses, relies on the development of effective therapeutic strategies specifically targeting key mechanisms involved in the metastatic cascade. Traditional chemotherapy with alkylating agents, microtubule inhibitors, and antimetabolites has shown limited efficacy against metastatic cells, largely due to the emergence of chemoresistant populations that undergo epithelial-to-mesenchymal transition (EMT), promoting the colonization of distant organs and sustaining metastatic progression. This scenario has spurred significant efforts to identify small molecules and biologics capable of interfering with specific steps in the metastatic process. In this review, we provide an overview of recent advances involving small interfering RNAs (siRNAs) and microRNAs (miRNAs) in cancer therapy. Although most of these agents are still under investigation and have not yet been approved for clinical use, insights into their development stage offer valuable information to identify new targets in the ongoing fight against metastasis. Particular emphasis is placed on the role of chemical modifications applied to siRNAs, such as backbone, sugar, terminal, base, and conjugation changes, and how these factors influence their stability, immunogenicity, and targeting precision. By integrating these aspects into the discussion, this review provides a focused and up-to-date resource for researchers in medicinal chemistry, drug delivery, and pharmaceutical formulation, where molecular design plays a critical role in therapeutic success.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0dbbc8021fb135eb5717ae3c5e587b5f4b68b62c" target='_blank'>
              Knocking on Cells’ Door: Strategic Approaches for miRNA and siRNA in Anticancer Therapy
              </a>
            </td>
          <td>
            Massimo Serra, Alessia Buccellini, M. Paolillo
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9473a008423522556ea8e825486e97240efc7ead" target='_blank'>
              Transcriptome Analysis Identifies Functional and Prognostic Hypoxia-Associated Genes in Multiple Myeloma.
              </a>
            </td>
          <td>
            Lijia Hou, Jing Zhang, Q. Ran, Zhongjun Li, Maoshan Chen
          </td>
          <td>2025-09-26</td>
          <td>International journal of laboratory hematology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Ionizing radiation can lead to breaks in the double-stranded structure of DNA. Although cells have mechanisms that can repair such breaks, misrejoining events can occur that lead to mutations in an individual’s genome. When mutations affect certain genes, such as oncogenes and tumor-suppressor genes, cancer can result. Although ionizing radiation is a known cause of cancer, it has been difficult to determine how much of an excess cancer risk results from low-dose radiation exposure. Radiation signatures that distinguish radiation-induced cancers from spontaneous cancers are thus useful for assessing cancer risks after exposure to low-dose radiation. Recent advances in high-throughput sequencing technologies have enabled the comprehensive profiling of mutations in individual cancer cells. Analyses of radiation-associated cancer genomes have identified an increase in small deletions and chromosome rearrangements, including large deletions, inversions, and translocations, which are likely to be formed by the misrejoining of DNA double-strand breaks. Among these, chromosome rearrangements leading to cancer-related mutational events are potentially useful as biomarkers and as therapeutic targets for treating radiation-induced cancer. This review summarizes research on radiation-induced mutational signatures found in animal and human cancers, consolidates current knowledge, and discusses the future perspectives in this research field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33c28820d79dcb3eabee840cf5431bbdeaec073" target='_blank'>
              Mutational Signatures in Radiation-Induced Cancer: A Review of Experimental Animal and Human Studies
              </a>
            </td>
          <td>
            K. Daino, C. Tsuruoka, Atsuko Ishikawa, S. Kakinuma, T. Imaoka
          </td>
          <td>2025-08-29</td>
          <td>Biology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) remains a formidable therapeutic challenge, particularly within high-risk cohorts. Advances in next-generation sequencing have elucidated critical mutations that significantly influence prognosis and therapeutic decision-making. Tyrosine kinase inhibitors (TKIs) have significantly improved treatment outcomes in Philadelphia chromosome-positive (Ph+) ALL. Meanwhile, emerging therapies such as monoclonal antibodies and chimeric antigen receptor (CAR) T-cell therapies show promise for B-cell ALL, although they are associated with considerable toxicities. These developments underscore the persistent need for alternative therapeutic strategies that can benefit a wider range of patients. In this study, human ALL cell lines—characterized by either wild-type or mutant tumor protein p53 (TP53) status—were treated with RG7388 (an MDM2 (mouse double minute 2 homolog) inhibitor) and BBI608 (a STAT3 (signal transducer and activator of transcription 3) inhibitor), both as single agents and in combination. Cell viability was quantified using XTT assays, while apoptosis was assessed via flow cytometry. Additionally, immunoblotting and qRT-PCR were employed to evaluate changes in protein and gene expression, respectively. RG7388 demonstrated potent growth inhibition in the majority of ALL cell lines, with p53-mutant cell lines exhibiting resistance. BBI608 reduced cell viability across all tested cell lines, though with variable sensitivity. Notably, the combination of RG7388 and BBI608 elicited synergistic anti-proliferative effects in p53 wild-type and partially functional p53-mutant cells, enhancing apoptosis and stabilizing p53 protein levels. In contrast, MOLT-4 cells, which harbor concurrent TP53 and STAT3 mutations, did not benefit from the combination treatment, indicating an inherent resistance phenotype within this subset. Collectively, these findings highlight the therapeutic potential of combined MDM2 and STAT3 inhibition in ALL, particularly in p53 wild-type and partially functional p53-mutant contexts. This combinatorial approach augments apoptosis and tumor growth suppression, offering a promising avenue for expanding treatment options for a broader patient population. Further investigation is warranted to validate these preclinical findings and to explore translational implications in genetically diverse ALL subsets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175bf78e5a77155f44b8ab2779e6277c64a69ad7" target='_blank'>
              Synergistic MDM2-STAT3 Inhibition Demonstrates Strong Anti-Leukemic Efficacy in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            E. Aptullahoğlu, Emrah Kaygusuz
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="It has become evident from decades of clinical trials that multimodal therapeutic approaches with focus on cell intrinsic and microenvironmental cues are needed to improve understanding and treat the rare, inoperable, and ultimately fatal diffuse intrinsic pontine glioma (DIPG), now categorized as a diffuse midline glioma. In this study we report the development and characterization of an in vitro system utilizing 3D Tumor Tissue Analogs (TTA), designed to replicate the intricate DIPG microenvironment. The innate ability of fluorescently labeled human brain endothelial cells, microglia, and patient-derived DIPG cell lines to self-assemble has been exploited to generate multicellular 3D TTAs that mimic tissue-like microstructures, enabling an in- depth exploration of the spatio-temporal dynamics between neoplastic and stromal cells. The 3D-TTA model recapitulates clinical patterns of DIPG growth, evidenced by resistance to chemotherapy, HDAC and proteasome inhibitors, as well as sensitization to the antibody-activated innate immune microenvironment including complement proteins and surrounding microglia. Multimodal fluorescence imaging platforms integrated with high-throughput omics revealed that alterations in tumor cell motility and growth in the 3D-TTA model compared to tumor cell only spheroids correlated with specific transcriptomic and proteomic changes. STAT3, ITGA5, LGALS1, SOD2, MVP, and CLIC1, associated with microenvironment signaling, DNA replication, and immune regulation, were identified as potential novel targets in the 3D model. The results indicate that the 3D TTA platform developed here represents a powerful tool for preclinical studies, paving the way for identification/validation of tissue specific biomarkers and novel drug targets, thus advancing disease management strategies for DIPG in children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="BACKGROUND
Head and neck squamous cell carcinoma (HNSCC) is a malignancy with poor prognosis and survival. While epidermal growth factor receptor (EGFR) is a known driver of HNSCC, EGFR inhibitors show limited efficacy as monotherapies, suggesting that effective combination therapies are needed.


METHODS
We performed GeCKO and Kinase CRISPR library screens to identify candidate knockouts that increase EGFR inhibitor sensitivity in resistant HNSCC cells. We validated a candidate using the CellTiter-Glo viability assay following RNA interference and investigated the mechanisms using apoptosis and cell cycle flow cytometry analysis.


RESULTS
Kinase CRISPR library screens identified that PIK3C2A gene downregulation enhanced EGFR inhibitor sensitivity in resistant HNSCC cells. Viability assays further validated that PIK3C2A knockdown overcame EGFR inhibitor resistance in HNSCC cells.


CONCLUSIONS
Our data suggests that PIK3C2A is a novel mediator of EGFR inhibitor resistance in HNSCC. Future studies are required to determine the mechanisms by which PIK3C2A drives EGFR inhibitor resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7734f4490eb7420a85bafc6685737b3d4a1310e4" target='_blank'>
              CRISPR Screens Identify PIK3C2A as a Novel Mediator of EGFR Inhibitor Resistance in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Jiayu Wang, M. Ludwig, Aditi Kulkarni, Andrew C Birkeland, N. Michmerhuizen, E. Gensterblum-Miller, Jingyi Zhai, Hui Jiang, P. Swiecicki, Marisa Buchakjian, M. H. Heft Neal, S. Chinn, M. Spector, J. C. Brenner
          </td>
          <td>2025-09-23</td>
          <td>Head & neck</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The RAS family of small GTPases is among the most frequently mutated gene families in human cancer. In pancreatic ductal adenocarcinoma (PDAC), ~95% of cases harbor an activating KRAS mutation, primarily at codon 12, 13, or 61, with G12D the most common overall (40%). In contrast, the KRASQ61L mutation, though constitutively active, is virtually absent in PDAC patient tumors. This suggests that KRASQ61L may engage in distinct, allele-specific signaling that limits its ability to drive tumorigenesis. Determining the mechanisms that limit the occurrence of this mutation will aid in our understanding of the critical KRAS effectors and pathways that drive tumorigenesis. To investigate these mechanisms, we utilized a tightly controlled doxycycline-inducible KRAS expression system in an isogenic, immortalized pancreatic cell line, enabling direct comparison of KRASQ61L to the common PDAC mutant KRASG12D. Using TurboID proximity labeling alongside RNA sequencing, we mapped early effector interactions and transcriptional responses, revealing that KRASQ61L induces greater hyperactivation of the ERK/MAPK pathway, resulting in increased nuclear translocation of ERK1/2. Finally, pancreatic cells are highly tolerant to overexpression of KRASG12D, but KRASQ61L overexpression leads to impaired proliferation and increased apoptosis. These findings provide experimental support for the long-standing "Goldilocks" model of oncogenic signaling, where too much ERK/MAPK pathway activation is detrimental to tumorigenesis. Our work offers a mechanistic explanation for the relative absence of KRASQ61L in PDAC and contributes to our understanding of KRAS allele-specific vulnerabilities, which can inform future therapeutic strategies targeting KRAS-driven pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7014e20c6c94dbffda8bc275f228979c75801203" target='_blank'>
              Pancreatic cells are resistant to KRAS(Q61L) expression due to hyperactive ERK/MAPK signaling and apoptosis induction.
              </a>
            </td>
          <td>
            Rachel A Burge, Lucas Bialousow, Thomas McFall, Logan Bamonte, Grayson Johnson, Merissa Smith, Silvia G Vaena, Susana Comte-Walters, Lauren E Ball, S. Berto, John P O'Bryan, G. A. Hobbs
          </td>
          <td>2025-09-15</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mutations that initiate AML can cause clonal expansion without transformation ("clonal hematopoiesis"). Cooperating mutations, usually in signaling genes, are needed to cause overt disease, but these may require a specific "fitness state" to be tolerated. Here, we show that nearly all AMLs arising in a mouse model expressing two common AML initiating mutations (Dnmt3aR878H and Npm1cA) acquire a single copy amplification of chromosome 7, followed by activating mutations in signaling genes. We show that overexpression of a single gene on chromosome 7 (Gab2, which coordinates signaling pathways) is tolerated in the presence of the Npm1cA mutation, can accelerate the development of AML, and is important for survival of fully transformed AML cells. GAB2 is likewise overexpressed in many human AMLs with mutations in NPM1 and/or signaling genes, and also in Acute Promyelocytic Leukemia initiated by PML::RARA; the PML::RARA fusion protein may activate GAB2 by directly binding to its 5' flanking region. A similar pattern of GAB2 overexpression preceding mutations in signaling genes has been described in other human malignancies. GAB2 overexpression may represent an oncogene-driven adaptation that facilitates the action of signaling mutations, suggesting an important (and potentially targetable) "missing link" between the initiating and progression mutations associated with AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6f3ad6142f75e1cb2d2a0d8414ccd605f8e467" target='_blank'>
              Overexpression of the signaling-coordinator GAB2 can play an important role in Acute Myeloid Leukemia progression.
              </a>
            </td>
          <td>
            Michael H. Kramer, Stephanie N Richardson, Yang Li, Tiankai Yin, Nichole M. Helton, Daniel R. George, Michelle Cai, S. M. Ramakrishnan, C. Katerndahl, Christopher A. Miller, T. Ley
          </td>
          <td>2025-08-07</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Tumor immunity has emerged as a focal point for cancer research. Although tumor immunotherapy represents a promising approach to cancer treatment, its effectiveness is often hindered by the heterogeneity of the tumor microenvironment (TME) and immune escape mechanisms. In recent years, small non-coding RNAs (sncRNAs) have attracted increasing attention in tumor immunology due to their essential role in gene regulation. This review systematically examines the multifaceted regulatory functions of sncRNAs in tumor immunity, with a focus on six major subtypes: microRNA, siRNA, piRNA, snoRNA, tsRNA, and snRNA. The molecular mechanisms by which these sncRNAs reshape the TME are discussed, including their roles in modulating immune cell differentiation (e.g., T cell polarization, macrophage phenotype transition), regulating immune checkpoint expression (PD-1/PD-L1, CTLA-4, Tim-3, LAG-3), and influencing tumor antigen presentation. This review also explores the dynamic network through which sncRNAs contribute to tumor immune escape. Furthermore, this study highlights the clinical potential of sncRNAs as liquid biopsy biomarkers and their application prospects in therapeutic strategies, such as targeted silencing of immunosuppressive molecules via nano-delivery systems, combination treatments with radiotherapy and chemotherapy, and Chimeric Antigen Receptor T-cell (CAR-T) therapy. Despite current challenges, including limited delivery efficiency and off-target effects, emerging technologies like AI-assisted sequence design and organ-on-a-chip models present new opportunities for clinical translation. This comprehensive review provides a theoretical foundation and translational insights for elucidating the functional network of sncRNAs in tumor immunology and advancing precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7dcee9c888595b1a7bd3be0ebd8ff37c9ea4926" target='_blank'>
              Small non-coding RNAs: key regulatory factors and potential therapeutic targets in tumor immunity
              </a>
            </td>
          <td>
            Zihan Liu, Haotian Dong, Chengyuan Ye, Jianing Yan, Min Miao, Y. Shao
          </td>
          <td>2025-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Breast cancer remains the most prevalent malignancy among women, and radiotherapy plays a pivotal role in reducing local recurrence and improving prognosis. However, the emergence of radioresistance in a subset of patients significantly compromises treatment efficacy, underscoring the need for a deeper understanding of the underlying molecular mechanisms. In recent years, non-coding RNAs (ncRNAs) have emerged as key regulators of gene expression and have garnered increasing attention for their roles in mediating radioresistance in breast cancer. This review systematically summarises the major molecular mechanisms by which ncRNAs contribute to breast cancer radioresistance, including cell cycle regulation, DNA damage repair, programmed cell death (e.g., apoptosis, autophagy and ferroptosis), oxidative stress response, tumour microenvironment remodelling and maintenance of cancer stem cell properties. On the translational front, RNA-based therapeutic approaches-including antisense oligonucleotides (ASOs), small interfering RNAs (siRNAs), miRNA mimics and CRISPR/Cas9-offer promising avenues for radiosensitisation, yet face substantial clinical hurdles. These include immune activation, poor delivery specificity, intracellular trafficking barriers and limited stability. Advances in chemical modifications and nanoparticle-based delivery systems-such as redox-responsive nanocarriers-have shown potential in enhancing the efficacy and safety of ncRNA-targeted therapies. Despite encouraging progress, clinical translation remains constrained by a lack of methodological standardisation, insufficient high-quality clinical data, limited biomarker reliability, suboptimal target selection and unresolved safety concerns. Future efforts should prioritise optimisation of delivery platforms, validation of multi-ncRNA biomarker panels in large, multicentre cohorts and integration of multi-omics data to reconstruct comprehensive regulatory networks, ultimately accelerating the clinical deployment of ncRNA-based radiosensitisation strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e7200463162269c84a998ecdd48960e551fbf5f" target='_blank'>
              Non-Coding RNAs in Breast Cancer Radioresistance: Mechanisms, Functional Roles and Translational Potentials.
              </a>
            </td>
          <td>
            Xiaohui Zhao, Yuting Qiu, Jie Chen, Danni Wang, Zairui Wang, Shuang Ma, Yimin Liu, Guoying Liu, Zhuofei Bi
          </td>
          <td>2025-09-14</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The incidence and mortality rates of tumors continue to show an annual upward trend. Since its discovery in 1986, nuclear receptor subfamily 2, group F, member 2 (NR2F2) has been shown to serve a key role in the transcriptional regulation of various genes. Furthermore, NR2F2 has a notable impact on the progression of tumors; however, its role in tumor progression it not yet fully understood. Multi-omics can illuminate the paradoxical roles of NR2F2 across various malignancies, where it functions as either an oncogenic driver or tumor suppressor in a context-dependent manner. The present review discusses the determinant influence of NR2F2 in modulating disease advancement and mechanisms in heterogeneous types of cancer, including its effects on tumor proliferation, apoptosis and epithelial-mesenchymal transition. The aim was to assess the role and mechanisms of NR2F2 in various tumors, identify its clinical utility in targeted cancer therapy and thereby provide a novel therapeutic option for overcoming cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/133411ae131c0e0936a8531e075498590fc78c99" target='_blank'>
              The role of NR2F2 in cancer (Review)
              </a>
            </td>
          <td>
            Zi-Ning Liu, Xiao Yang
          </td>
          <td>2025-08-21</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ce4746633f2d4d42f9a7aafe673ea0e10428f4" target='_blank'>
              Genome-wide profiling the integration patterns with T7-PCR.
              </a>
            </td>
          <td>
            Ang Li, Chichen Xiao, Jinwen Wang, Xiao Huang, Xionglei He, Kehui Liu, Fan-Nv Xia
          </td>
          <td>2025-08-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disorder marked by irregular expansion and maturation, giving rise to the aggregation of immature myeloid precursor cells. Although most patients achieve remission with initial treatment, the majority of relapses lead to poorer overall survival. The bone marrow (BM) immune microenvironment has been proven to significantly affect the progression of AML. However, the mechanisms that cause the imbalance of immune cell subsets and phenotypes remain partially obscure. Therefore, this research sought to explore the immune-regulatory genes and to determine their role in AML.Differentially expressed genes (DEGs) were obtained through differential analysis of the AML cohort. Enrichment analyses were applied to explore their biological functions. Weighted Gene Co-expression Network Analysis (WGCNA) was performed to identify the key module of AML. ROC curve analysis was performed to identify hub genes with good predictive power. CIBERSORT and the ESTIMATE algorithm were used to assess the correlation between hub genes and the immune microenvironment of AML. The impact of hub gene expression on the prognosis of AML was verified through prognostic traits and clinical samples.Through differential analysis and WGCNA, seven genes were identified as markedly related to the development of AML. By mapping ROC curves, three hub genes were verified: CCR7, SLC16A6, and MS4A1, which have high diagnostic value for AML. Additionally, an imbalanced immune microenvironment was found to be common in AML. Three hub genes were significantly associated with immune components, including immune cells and immunomodulatory factors. Ultimately, through the validation of clinical samples and the analysis of prognostic characteristics, three genes were confirmed to be reduced in AML patients, and their high expression suggested a favorable prognosis.Our study identified and validated the efficacy of SLC16A6, CCR7, and MS4A1 as tumor suppressors implicated in AML progression and related to immune cell infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8072861bf4b6696259633b8c455f0a59b5d5338a" target='_blank'>
              Identification and validation of three tumor suppressors associated with the immune response of acute myeloid leukemia
              </a>
            </td>
          <td>
            Yueyuan Pan, Guocai Wu, Chenchen Liu, Minggui Chen, Tian Xia, Yonghua Ma, Zhigang Yang, Ruiting Wen
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Alterations in the RAS-RAF-MEK (MAPK) pathway through oncogenic RAS mutations or NF1 inactivation are frequently observed in rhabdomyosarcoma (RMS), the most common soft tissue sarcoma in children. Inhibiting the MAPK pathway by targeting CRAF and MEK has shown promising activity in H/NRAS Q61-mutated RMS. In this study, we evaluated the preclinical efficacy of the paralog selective B/CRAF inhibitor naporafenib in combination with the MEK inhibitor trametinib in a panel of 9 RMS tumors xenografts spanning diverse H/NRAS codon mutations beyond Q61X, as well as NF1-inactivation, and sought to identify markers of response. Precilinical objective response was 67% (6/9). We performed comparative RNA sequencing analysis and in vivo pharmacodynamic assays in progressive versus sensitive tumor models and determined that neither potency of MAPK pathway inhibition, nor the induction of myogenic differentiation correlated with response, while lack of MYC and S6 ribosomal protein (rpS6) inhibition and increase of oxidative phosphorylation correlated with progressive disease. Using genetic and pharmacologic manipulations, we show that both rpS6 and MYC are required for the growth of rhabdomyosarcoma cells and that suppression of rpS6 and MYC, or inhibition of oxidative phosphorylation, can sensitize cells and tumor xenografts to MAPK inhibition. Further, we show that rhabdomyosarcoma-associated NRAS mutations determine sensitivity to B/CARF + MEK inhibitors combination, with NRAS Q61X, NRAS G13R and NRAS G12D being preferentially sensitive, but NRAS G13D being resistant. In addition, we find that concurrent NF-1 inactivation and RAS mutation could predict resistance. These results highlight the importance of considering genetic makeup and markers of response when selecting RAS-RAF-MEK targeted agents for the treatment of RAS-driven rhabdomyosarcoma.



 Shreetama Bandyopadhayaya, Natalia Garcia, Funan He, Arnab Sarkar, Katia Campos, Teresa Marple, Paulomi Modi, Siyuan Zheng, Myron Ignatius, Vesselina G. Cooke, Angelina V. Vaseva. Identification of biomarkers of response to RAS-RAF-MEK pathway inhibition in RAS-altered rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A028.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de6e0d8c5aad8ab106359aebb4e827f87486bc79" target='_blank'>
              Abstract A028: Identification of biomarkers of response to RAS-RAF-MEK pathway inhibition in RAS-altered rhabdomyosarcoma
              </a>
            </td>
          <td>
            Shreetama Bandyopadhayaya, Natalia Garcia, Funan He, Arnab Sarkar, Katia Campos, Teresa Marple, Paulomi S. Modi, Siyuan Zheng, Myron S. Ignatius, V. Cooke, Angelina V. Vaseva
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chromosomal diagnostics have evolved significantly over the past several decades, from classical karyotyping to high-resolution genomic technologies. This review traces the development and integration of cytogenetic toolskaryotyping, fluorescence in situ hybridization (FISH), chromosomal microarray analysis (CMA), and whole genome sequencing (WGS)in modern clinical diagnostics. We examine each technologys diagnostic utility, limitations, and role in clinical workflows. Importantly, insights from a practicing clinical cytogeneticist highlight the increasing automation of laboratory processes and the continued relevance of karyotyping in routine diagnostics due to its cost-effectiveness and structural resolution. As genome-wide testing becomes more accessible, we explore future directions including the rise of artificial intelligence (AI), non-invasive screening technologies, and emerging ethical challenges surrounding incidental findings, data privacy, and equitable access. Together, these perspectives underscore the importance of combining technological innovation with clinical expertise and ethical foresight to guide the future of precision diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d4b14a05606752d08d57c2ff217d9ddb756ec" target='_blank'>
              From Chromosome Karyotyping to Genomic Precision: The Evolution of Diagnostic Technologies in Medicine
              </a>
            </td>
          <td>
            Run Liu
          </td>
          <td>2025-08-13</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of “signatures”, one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines [1–3]. The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a “funnel”, through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0289ae32bcbca4304d80ec26b288975aaa0f5e9" target='_blank'>
              Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature
              </a>
            </td>
          <td>
            Natanael Spisak, Marc de Manuel, Molly Przeworski
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39afb12b1cf53d01e3a00f3383b9ed1def1b93f7" target='_blank'>
              PARP Inhibitors in Gastric Cancer: Unlocking Precision Oncology.
              </a>
            </td>
          <td>
            Derek Tai, Vitor Goes, Sharanya Kumar, Pranati Shah, Farris Al-Manaseer, Daniel Park, Christiana Crook, Sofia Guzman, Xiaolin Zhu, Daneng Li, D. Castillo
          </td>
          <td>2025-09-17</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aim. To identify clinical, molecular, genetic, and immunological predictors of response to PARP inhibitor olaparib therapy in patients with metastatic castration-resistant prostate cancer (mCRPC) harboring germline and somatic mutations in homologous recombination repair (HRR) genes.Materials and methods. A prospective analysis was conducted on data from 39 patients with mCRPC and HRR gene mutations (BRCA1/2, ATM, CHEK2, CDK12, among others) receiving оlaparib therapy after prior treatment with androgen receptor signaling inhibitors. Diagnoses were histologically verified according to the current WHO classification (5th edition). Genetic status was determined by next-generation sequencing (NGS). The tumor immune microenvironment was evaluated by immunohistochemical analysis assessing expression of CD4, CD8, CD68, CD163 markers and tumor-infiltrating lymphocyte (TILs) levels. Statistical analysis involved Cox proportional-hazards regression and Kaplan–Meier survival analysis.Results. The BRCA2 mutation was the most frequently detected alteration (61 %), correlating with increased expression of CD163 and decreased CD68 expression. The most significant clinical predictors associated with improved progressionfree survival during оlaparib therapy included BRCA2 mutation positivity, patient age under 64 years, initially metastatic disease presentation, and ≥90 % reduction in prostate-specific antigen levels at 3 months post-treatment initiation. Immunological markers indicative of favorable therapeutic response included elevated levels of M2-polarized macrophages (CD163 ≥30 %) and a high CD163/CD68 ratio (≥2.83). Conversely, robust prior response to еnzalutamide therapy and administration of docetaxel during hormone-sensitive stages significantly worsened the prognosis for olaparib efficacy.Conclusion. Olaparib therapy is effective in patients with HRRm mCRPC, with particularly effectiveness in BRCA2m tumors.  Clinical (age, disease dissemination pattern, prostate-specific antigen dynamics), immunological (CD163 expression and CD163/CD68 ratio), and molecular and genetic tumor characteristics should be considered when selecting patients for olaparib treatment, enhancing personalized therapeutic strategies in mCRPC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bedca89d6cf003aa836f25f27df6fe20a943841" target='_blank'>
              HRR gene mutations and characteristics of the tumor microenvironment as predictors of response to olaparib therapy in metastatic castration-resistant prostate cancer
              </a>
            </td>
          <td>
            A. I. Stukan, S. Vtorushin, A. P. Bogdan, Т. Y. Semiglazova, Kh. R. Tovbulatova, V. Bodnya, V. Porkhanov, A. A. Dovlatbekyan, M. A. Chagiev, D. V. Khoreva
          </td>
          <td>2025-08-06</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Polymerase theta (POLQ) is essential for theta-mediated end joining (TMEJ), an error-prone DNA repair pathway that is hyperactive in homologous recombination-deficient (HRD) cancers such as high-grade serous ovarian cancer (HGSOC). Inhibiting POLQ is a promising therapeutic strategy, but the molecular determinants of sensitivity and downstream mechanisms remain incompletely defined.



 We assessed the growth-inhibitory activity of two structurally distinct POLQ inhibitors, novobiocin and ART558, across 18 well-characterized HGSOC cell lines. Genomic profiling data were integrated with drug response values (GR50s) to identify predictive biomarkers, including mutational status, HRD scores, and copy number signatures (CX1–CX17). Mechanistic assays included POLQ ATPase activity, cell cycle analysis (FUCCI), immunofluorescence for γH2AX and POLQ, apoptosis markers and DNA fiber assays.



 POLQ inhibitor sensitivity varied independently from olaparib or carboplatin response, indicating a distinct mode of action. Novobiocin sensitivity significantly correlated with CX3 and CX5 copy number signatures, which reflect impaired homologous recombination and replication stress. In contrast, resistant lines were enriched for CX14 and CX6, signatures linked to mitotic errors and telomere shortening. Sensitive cell lines showed increased DNA damage (γH2AX foci), POLQ accumulation on chromatin, and apoptosis, particularly in BRCA1/2-mutant backgrounds. Cell cycle profiling revealed impaired G1/S progression and S-phase depletion. DNA fiber assays demonstrated replication fork degradation upon POLQ inhibition, consistent with POLQ’s role in fork protection.



 We identify genomic features, specifically CX3 and CX5 copy number signatures as candidate biomarkers of POLQ inhibitor sensitivity in HGSOC. Mechanistically, POLQ inhibition induces replication stress, persistent DNA damage, and apoptosis in HR-deficient settings. These findings support the clinical development of POLQ inhibitors and rational patient stratification based on genome instability signatures.



 Aleksandra Kruglikov. POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe824b1195032b879fa6dbdb3ed12cc08d24058" target='_blank'>
              Abstract A037: POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response
              </a>
            </td>
          <td>
            Aleksandra Kruglikov
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Purpose Next-generation sequencing (NGS) has revolutionized cancer treatment by enabling comprehensive cancer genomic profiling (CGP) to guide genotype-directed therapies. While several prospective trials have demonstrated varying outcomes with CGP in patients with advanced solid tumors, its clinical utility in colorectal cancer (CRC) remains to be evaluated. Methods We conducted a prospective observational study of CGP in our hospital between September 2019 and March 2024. Overall survival (OS) of the patients who received CGP-based therapy and those did not was compared, and genomic variables associated with OS were evaluated. Results A total of 100 patients with CRC underwent CGP using four platforms. The median patient age was 67 years, and most had a good performance status. The most frequent genomic alterations were TP53 (82%), APC (82%), and KRAS (55%). Actionable mutations such as ERBB2 amplification and BRAF V600E were identified in some patients, and 9% received CGP-based therapy, including immune checkpoint inhibitors for tumor mutational burden-high or microsatellite instability-high tumors. Patients receiving CGP-based therapy had longer OS from expert panel discussion (16.0 vs. 10.8 months) compared to those who did not. Alterations in TP53, SMAD4, and NF1 were associated with worse OS. Interestingly, PTEN mutations were linked to improved survival. TP53 alterations were more common in left-sided CRC. Conclusion Although some patients with CRC received CGP-guided therapy, a statistically significant survival benefit was not observed. However, TP53 and SMAD4 mutations were identified as negative prognostic markers, indicating their potential as targets for future drug development. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06295-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26f03483e8e9f29b9ac2b6a966e10869cf85e5a3" target='_blank'>
              Clinical utility of comprehensive genomic profiling test for colorectal cancer: a single institution prospective observational study
              </a>
            </td>
          <td>
            Hiroki Tanabe, Katsuyoshi Ando, Keitaro Takahashi, Tomomi Kamanaka, S. Yuzawa, Junko Kikuchi, Y. Ohhara, Shin Ariga, Tatsuya Shonaka, C. Tani, Shin Otake, Takaaki Sasaki, Kenji Takahashi, Nobuhiro Ueno, K. Moriichi, M. Tanino, Ichiro Kinoshita, Yusuke Mizukami, M. Fujiya
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="CRISPR/Cas9 gene-editing technology has revolutionized biomedical research by enabling precise, efficient, and cost-effective genome manipulation. This paper focuses on the application of CRISPR/Cas9 in cancer research, with an emphasis on triple-negative breast cancer (TNBC), a particularly aggressive and treatment-resistant subtype of breast cancer that lacks expression of estrogen, progesterone, and HER2 receptors. Breast cancer is the most commonly diagnosed cancer among women worldwide, and TNBC represents one of its most challenging forms due to the absence of effective targeted therapies. While hormone receptor-positive and HER2-positive subtypes benefit from endocrine and targeted treatments, TNBC patients are often limited to chemotherapy, which is associated with high toxicity and limited efficacy. This review synthesizes recent CRISPR-based studies that identify oncogenes and tumor suppressors relevant to TNBC, explores advanced delivery systems such as non-viral nanoparticles, and evaluates the therapeutic promise of this technology. Despite current limitations—including off-target effects, delivery challenges, and ethical concerns—CRISPR/Cas9 offers a promising path toward developing personalized, gene-targeted treatments for TNBC and other hard-to-treat cancers.

Key words: CRISPR/Cas9, gene editing, breast cancer, triple negative breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c17f19abfb2d0fe3b181a39613fc90c46bf2f21" target='_blank'>
              Review of Current Applications of CRISPR Gene Editing Techniques in Breast Cancer Therapeutics Development
              </a>
            </td>
          <td>
            Yichen Zhang
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Health Sciences and Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND/AIM
Comprehensive genomic profiling (CGP) with tissue- and blood-based next-generation sequencing (NGS) is integral to the delivery of personalized medicine for targeted cancer therapy. This study aimed to evaluate the variant concordance for somatic variants using two clinical NGS systems for conducting both tissue- and blood-based analyses: Genexus-OCA v3 (OCA) vs. FoundationOne CDx (F1) for tissues and Genexus OPA (OPA) vs. FoundationOne CDx Liquid (F1L) for blood.


PATIENTS AND METHODS
The concordance of genomic alterations between the two NGS analyses was compared in six patients with breast, head, and neck cancers using tissue and circulating tumor DNA biopsies.


RESULTS
A total of 130 genes were common between F1 and OCA, and 41 between F1L and OPA. When comparing FoundationOne to Genexus for common genes, the sensitivity and specificity of OCA and OPA were 55% and 99%, respectively. Nine single-nucleotide variants (SNVs), one copy number alteration (CNA), and one fusion were detected by both Genexus and FoundationOne. However, one SNV (MAP2K1 F53V), two CNAs (AKT3 and MYC), and one fusion (ESR-CCDC170) were detected only in Genexus, whereas two SNVs (TP53 Q331* and KRAS G12V) were detected only in FoundationOne.


CONCLUSION
The two cancer genome panels were equivalent but not perfect in terms of the detection of variants using tissue and blood, indicating that different assays and analytical methods may have influenced the results. When performing CGPs, it is important to consider the characteristics of each NGS-based CGP test and the genetic variants associated with each disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbe1f8fa1288486304d422d7baf60accd4be21e6" target='_blank'>
              Comparison of Comprehensive Genomic Profiling Testing "Ion Torrent Genexus Sequencer" With FoundationOne.
              </a>
            </td>
          <td>
            Kenji Fujiyoshi, Rie Sugihara, Naoki Miyamoto, Yoriko Watanabe, Tomoya Sudo, Sanae Numata, Jun Akiba, H. Abe, Yuka Ichinose, Kenji Inoue, Shuich Ozono, T. Ono, Kentaro Orioka, Masaki Kashihara, Ryousuke Kajiwara, Hiroyuki Kawano, Akihiko Kawahara, Ryuta Takase, Uhi Toh, Kazuaki Hashimoto, T. Hisaka, Shingo Hirai, M. Mitsuoka, Daiki Miyazaki, Fumi Yoshitomi, Ken Yamamoto, H. Umeno, Masahisa Nomura, Yoshiki Naito
          </td>
          <td>2025-08-01</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258c585d858690cd52a8f16499c3f52ebcd32f02" target='_blank'>
              Genetic regulation of cell type-specific chromatin accessibility shapes immune function and disease risk
              </a>
            </td>
          <td>
            A. Xue, J. Fan, O. A. Dong, H. L. Huang, P. C. Allen, E. Spenceley, E. Sagi-Zsigmond, B. Bowen, A. Cuomo, A. Henry, H. Tanudisastro, Z. Qiao, L. Chen, E. Ben-David, K. Farh, D. Neavin, A. S. Lee, A. Senabouth, C. Bartie, R. A. McCloy, V. Chin, K. M. de Lange, G. Figtree, A. W. Hewitt, D. G. MacArthur, J. E. Powell
          </td>
          <td>2025-08-29</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Missense mutations in the 12th codon of KRAS are key drivers of lung cancer, with glycine-to-cysteine (G12C) and glycine-to-aspartic acid (G12D) substitutions being among the most prevalent. These mutations are strongly associated with poor survival outcomes. Given the critical role of KRAS in lung cancer and other cancers, it remains as a major target for the development of new and complementary treatments. We have developed a CRISPR-High Fidelity (HiFi)-Cas9-based therapy strategy that can effectively and specifically target KRASG12C and KRASG12D mutants, avoiding KRASWT off-targeting and affecting KRAS downstream pathways, thereby significantly reducing tumorgenicity. The delivery of HiFiCas9 components via ribonucleoprotein particles (RNPs) and adenovirus (AdV) effectively abrogates cell viability in KRAS-mutant Non-Small Cell Lung Cancer (NSCLC) preclinical models, including 2D and 3D cell cultures, cell-derived xenografts (CDX), and patient-derived xenograft organoids (PDXO). Our in vitro studies demonstrate that HiFiCas9-based therapy achieves superior KRAS inhibition compared to Sotorasib and effectively circumvents certain resistance mechanisms associated with Sotorasib treatment. Moreover, in vivo delivery using adenoviral particles significantly suppresses tumor growth in preclinical NSCLC models. Collectively, our findings establish HiFiCas9 as an effective therapeutic strategy with promising clinical applications, especially if in vivo delivery methods are further optimized.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/593330427b7e38641ea2b0aa9b194318fd13184b" target='_blank'>
              High-fidelity Cas9-mediated targeting of KRAS driver mutations restrains lung cancer in preclinical models
              </a>
            </td>
          <td>
            J. Álvarez-Pérez, Juan Sanjuan-Hidalgo, Alberto M. Arenas, Ivan Hernández-Navas, Maria S. Benitez‑Cantos, Á. Andrades, S. Calabuig-Fariñas, E. Jantus-Lewintre, Luis Paz-Ares, Irene Ferrer, Pedro P Medina
          </td>
          <td>2025-09-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Acute myeloid leukemia (AML) characterized by NPM1 mutations or KMT2A rearrangements depends on abnormal epigenetic programs mediated by menin, a critical scaffold protein for sustaining the expression of oncogenic HOX/MEIS1 genes. Therefore, menin inhibitors have become a promising class of AML treatments.


RECENT FINDINGS
Early-phase trials have shown that agents such as revumenib, ziftomenib, bleximenib, and enzomenib are active, particularly in relapsed/refractory disease, with 23-48% of patients achieving composite complete remission. However, the durability of single-agent treatment remains limited, and resistance mechanisms, such as MEN1 mutations or transcriptional reprogramming, have been identified. Consequently, combination approaches involving venetoclax, hypomethylating agents, or chemotherapy are being investigated to improve response depth and duration. Recent SAVE, KOMET-007, and cAMeLot-2 trials have demonstrated high overall response rates (68%-100%) and encouraging MRD-negative complete remissions when combining menin inhibitors with venetoclax-based regimens. This has been observed even in patients who have previously received venetoclax. Combinations with intensive chemotherapy (e.g., 7 + 3) in the frontline setting have also yielded high CRc rates (up to 94% in NPM1-mutated AML) without exacerbating toxicity.


SUMMARY
These findings justify the integration of menin inhibitors into the AML therapeutic landscape, and support ongoing randomized trials to confirm their benefit in both frontline and relapse or refractory settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30efc9044b0dac7eb9b858bad0224206b5fa44af" target='_blank'>
              The role of menin inhibitors in acute myeloid leukemia.
              </a>
            </td>
          <td>
            A. Isidori, Giovanni Marconi
          </td>
          <td>2025-08-08</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background/Objectives: FAM72B (Family with sequence similarity 72 member B) is a gene whose function is not yet fully elucidated and which belongs to the FAM72 gene family. Recent studies have indicated that it is involved in the regulation of stem cell proliferation and DNA repair and serves as a valuable prognostic biomarker for a few types of cancer. This study aimed to systematically investigate the expression profile of FAM72B in pan-cancer, its role in the tumor immune microenvironment, and its potential as a prognostic and immunotherapeutic biomarker. Methods: Using bioinformatics tools such as SangerBox3.0, GEPIA2.0, Kaplan–Meier Plotter, and cBioPortal, we systematically analyzed the correlation of FAM72B expression levels with various cancer types, clinical pathological parameters, prognostic value, genetic mutations, genomic heterogeneity, immune checkpoint genes, immune cell infiltration levels, and single-cell-level characteristics. Results:FAM72B was found to be overexpressed in most cancers and significantly associated with poor prognosis, although it may exert a protective effect in some cancers like thymoma (THYM). Its expression level was positively correlated with tumor mutation burden (TMB), microsatellite instability (MSI), neoantigen (NEO) levels, and expression of immune checkpoint genes in most cancers, suggesting that patients with high FAM72B expression may respond better to immune checkpoint inhibitors. Moreover, FAM72B expression was significantly correlated with the infiltration levels of various immune cells in the tumor immune microenvironment across pan-cancer. Single-cell sequencing results also demonstrated a significant correlation between FAM72B and the biological functional states of multiple cancers. Conclusions:FAM72B holds promise as a potential pan-cancer prognostic biomarker and therapeutic target, providing a novel basis for the development of personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/402a246b12f127e82a21e1ee4883073b0a45f68c" target='_blank'>
              Spotlight on FAM72B: Pan-Cancer Expression Profiles and Its Potential as a Prognostic and Immunotherapeutic Biomarker
              </a>
            </td>
          <td>
            Anran Chu, Yuchan Wang
          </td>
          <td>2025-09-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 BRCA1, BRCA2, and PALB2 mutations frequently involve shifting of the gene reading frame, resulting in the production of premature stop codons and truncated proteins. Frequently, the latter are subject to misfolding and proteasomal degradation. We have found that in response to prolonged PARP inhibitor (PARPi) treatment, multiple common pathogenic BRCA1 mutations use post-transcriptional mechanisms to produce truncated and hypomorphic protein isoforms. BRCA1 protein isoforms could induce PARPi resistance when expressed at sufficiently high levels. Moreover, we discovered several BRCA1 genetic alterations, such as secondary splice site mutations, and Alu-mediated gene rearrangements, which result in the production of protein isoforms that avoid misfolding and promote PARPi resistance. Finally, we are currently assessing the impact of PALB2 hypomorphic proteins on cancer initiation and will discuss recent results.



 Neil Johnson. BRCA mutations and protein products in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b875969d2d788ca87a548ad50a77f9264de3b1" target='_blank'>
              Abstract IA022: BRCA mutations and protein products in cancer
              </a>
            </td>
          <td>
            Neil Johnson
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This review article explores the role of immuno-radiotherapy in the context of genome instability and tumour evolution. Genomic changes in tumours exist in a delicate balance with the immune system, offering evolutionary pathways to adapt and grow but risking provoking an immune response. Rapid developments across both immunotherapy and radiotherapy have raised questions about the potential benefits combination therapy, and how best to identify ideal treatment populations. Here we discuss foundational studies of genomic instability and tumour evolution, how these paradigms translate into immune surveillance and evasion, and subsequently go on to explore recent preclinical and clinical studies of both treatment modalities. Understanding how cancers evolve in the context of the immune system could provide a key insight in delivering better therapies that could overcome treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4d64cec361f889e2be65100e62ef8d285da2f26" target='_blank'>
              Tumour Evolution Driving Genome Instability, Immune Interactions, and Response to Radiotherapy
              </a>
            </td>
          <td>
            G. Seed, France Truong, Rahma Riahi, Ben O’Leary
          </td>
          <td>2025-06-09</td>
          <td>Cancer journal (Sudbury, Mass.)</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 The introduction of poly (ADP-ribose) polymerase (PARP) inhibitors has transformed the therapeutic landscape for patients with epithelial ovarian, fallopian tube, and primary peritoneal cancers (EOC), particularly those harboring BRCA1/2 mutations or tumors characterized by homologous recombination deficiency (HRD). Identifying HRD status is critical to inform the selection of maintenance therapy in patients with advanced EOC because it is associated with improved clinical outcomes. Here, we present the analytical performance of Labcorp’s expanded OmniSeq® INSIGHT (OSI+HRD) test to determine HRD and inform treatment decisions.



 The OSI+HRD test utilizes the Illumina TSO500 High-Throughput (HT) and HRD assays, NovaSeq 6000 sequencers, and DRAGEN TSO500 HRD pipeline. The HRD component includes ∼25K genome-wide probes to assess genomic scars, including loss of heterozygosity (LOH), telomeric-allelic imbalance (TAI), and large-scale state transitions (LST). To determine HRD status, a proprietary pipeline generates a genomic instability score (GIS) score, which is used with BRCA1/2 mutational status to determine HRD status. The validation study consisted of accuracy, precision, limit of detection (LOD), limit of blank (LOB), and sample input experiments, utilizing 169 patient samples across 13 different tumor types.



 The analytical accuracy of OSI+HRD assay to determine GIS and HRD status was evaluated by using 66 ovarian cancer specimens, orthogonally tested using a validated HRD assay. The HRD GIS values were highly concordant between the two tests (Pearson’s correlation R=0.97). For 53 samples with a definitive HRD status determined by both assays, the observed PPA and NPA were 89.47% and 97.06%, respectively. The majority of the discordant cases were either near the GIS cutoff of 42, or due to different BRCA1/2 variant classification. Three commercial HRD contrived control samples with verified HRD status were also assessed, yielding a PPA and NPA of 100%. Assay precision was assessed using 8 HRD-positive samples and 2 HRD-negative samples, to assess variability within runs, between runs and across operators, days, instruments, reagent lots, and DNA input amounts. The pooled standard deviation of GIS values for these studies for HRD-positive samples was 2.51. For HRD-negative samples, the mean absolute deviation was 1.13. The Limit of Detection (LOD) was determined using the minimum percentage of tumor nuclei needed to successfully call HRD positivity and was established to be 30%.



 This study supports the clinical validity of this HRD assay as a reliable tool for identifying patients with EOC who may benefit from PARP inhibitors. The assay successfully detected HRD status, including BRCA1/2 mutations and GIS score. This assay demonstrated 94.34% (50/53) concordance with orthogonal validated assays and high reproducibility, supporting it as a sensitive, specific and robust diagnostic option for patients with advanced EOC.



 Jie An, Hardik I. Parikh, Michael J. Collins, Erin A. DeBlasi, Erik Van Roey, Yamuna Pulivendula, Angela Stout, Hanchun T. DeFedericis, Stacey Gabrielli, Natalie Church, Nicholas Perez, Heather Boyanich, Daniel Metzger, Collin Parrow, Yvette Rodriguez, Julianna Du Hart, Jennifer Meyn, David Starks, Rachel J. Elsey, Tobias Meissner, Stephanie B. Hastings, Kyle C. Strickland, Michelle Green, Eric A. Severson, Pratheesh Sathyan, Shakti Ramkissoon, Rebecca A. Previs, Sarabjot Pabla, Taylor J. Jensen. Analytical validation of homologous recombination deficiency (HRD) status in patients with ovarian cancer using the OmniSeq® INSIGHT (OSI) test [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6691751d944dfbadddc6d08f486063b630fde36e" target='_blank'>
              Abstract A019: Analytical validation of homologous recombination deficiency (HRD) status in patients with ovarian cancer using the OmniSeq® INSIGHT (OSI) test
              </a>
            </td>
          <td>
            Jie An, Hardik I. Parikh, Michael J. Collins, Erin Deblasi, Erik Van Roey, Yamuna Pulivendula, Angela Stout, Hanchun T. DeFedericis, Stacey Gabrielli, Natalie Church, Nicholas Perez, Heather Boyanich, Daniel Metzger, Collin Parrow, Yvette Rodriguez, Julianna Du Hart, Jennifer Meyn, David Starks, Rachel J Elsey, Tobias Meissner, S. Hastings, Kyle C. Strickland, Michelle F Green, Eric A Severson, P. Sathyan, Shakti H. Ramkissoon, R. Previs, S. Pabla, Taylor J. Jensen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract R-loops or DNA–RNA hybrids are prominent nucleic acid structures that commonly arise during transcription. These structures play important biological functions, such as regulating gene expression and DNA repair. However, when unresolved by nucleic acid processing factors, pathological R-loops can be harmful and lead to genome instability. N6-Methyladenosine (m6A), the most prevalent modification in messenger RNA, has been recently identified to be crucial for regulating R-loop balance and maintaining genome stability. Strikingly, m6A-modified R-loop formation can have opposing consequences, either stabilization or resolution, depending on the biological context. In this review, we discuss the current knowledge of the regulatory roles of m6A on R-loops across various processes, including gene transcription, DNA repair, and centromere and telomere stability. Additionally, we explore other m6A-mediated processes, such as nascent transcription and chromatin landscape, that potentially affect R-loop dynamics. Finally, we discuss the current limitations and future directions of studying the m6A–R-loop axis, as well as the opportunities to target this pathway as a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5d8ee286e956882ce3a5149e41ece752c38e0d" target='_blank'>
              m6A modification in R-loop homeostasis: a potential target for cancer therapeutics
              </a>
            </td>
          <td>
            Minh-Anh Vu, Manuela Spagnuolo, Chun-Long Chen
          </td>
          <td>2025-08-11</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5af84640a910b5c7e72c72fa38163e38d00a66c4" target='_blank'>
              Morphological map of under- and overexpression of genes in human cells.
              </a>
            </td>
          <td>
            S. Chandrasekaran, Eric Alix, John Arevalo, Adriana Borowa, Patrick J. Byrne, William G. Charles, Zitong S. Chen, B. Cimini, Boxiong Deng, John G Doench, Jessica D. Ewald, Briana Fritchman, Colin J. Fuller, Jedidiah Gaetz, Amy Goodale, Marzieh Haghighi, Yu Han, Zahra Hanifehlou, Holger Hennig, Desiree Hernandez, Christina B. Jacob, Tim James, Tomasz Jetka, Alexandr A. Kalinin, B. Komalo, Maria Kost-Alimova, Tomasz Krawiec, Brittany A. Marion, Glynn Martin, Nicola Jane McCarthy, Lisa Miller, Arne Monsees, Nikita Moshkov, Al'an F. Munoz, Arnaud Ogier, Magdalena Otrocka, Krzysztof Rataj, D. Root, Francesco Rubbo, Simon Scrace, Douglas W Selinger, Rebecca Senft, Peter Sommer, A. Thibaudeau, Sarah Trisorus, Rahul Valiya Veettil, William J. Van Trump, Sui Wang, Michał Warchoł, Erin Weisbart, A. Weiss, Michael Wiest, Agata Zaremba, Andrei Zinovyev, Shantanu Singh, A. Carpenter
          </td>
          <td>2025-08-01</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Translocation renal cell carcinoma (tRCC) is an aggressive kidney cancer driven by gene fusions of the TFE3 transcription factor. TFE3 is essential in tRCC but dispensable in normal cells, presenting an attractive but pharmacologically challenging therapeutic target. We show that the basic helix-loop-helix (bHLH) domain of TFE3 is crucial for chromatin binding and transcriptional function. Via a phenotypic screen of 25,000 compounds, we identified molecules that either displace or retain chromatin-bound TFE3. BRD6866, a compound trapping TFE3 on chromatin, emerged as a pan-CDK inhibitor. Mechanistically, its inhibition of CDK9 – a key regulator of transcriptional elongation – was linked to impaired TFE3 fusion activity. These effects were recapitulated by the CDK9-selective inhibitor enitociclib, which downregulated TFE3 targets and suppressed tRCC cell growth. Our findings nominate CDK9 inhibition as a therapeutic strategy in tRCC and demonstrate the utility of mechanism-informed phenotypic screening for challenging targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723d2379f40a96d89ca22542e09ca70c344fbe88" target='_blank'>
              Phenotypic screening converges on CDK9 inhibition as a therapeutic strategy in translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Dharma R. Thapa, Riva Deodhar, Adam Skepner, Prathyusha Konda, Meng Zhou, Joseph Bluck, Christian Pichlo, Yantong Cui, Paige Stocker, Prateek Khanna, Jiao Li, Jonathan M. Tsai, Alison Leed, Sandy Gould, Maria Alimova, Dhana Friedrich, Jaime H. Cheah, R. Rupaimoole, O. Guicherit, S. Viswanathan
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8f50562c127ac79c0cdedb311535fc46568766" target='_blank'>
              Longitudinal single-cell analysis reveals treatment-resistant stem and mast cells with potential treatments for pediatric AML.
              </a>
            </td>
          <td>
            Denis Ohlstrom, Mojtaba Bakhtia, Hope L. Mumme, Marina E. Michaud, Alejandro De Janon Gutierrez, D. DeRyckere, Katherine E. Ferguson, Franklin Chien, William C. Pilcher, Sarthak Satpathy, Sean C Jordan, Douglas K Graham, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The advent of CRISPR-Cas9 has revolutionized gene editing, offering vast possibilities for disease treatment. This review dissects its mechanisms, structural aspects, and classification, underlining its superiority to preceding technologies such as ZFNs and TALENs. The evolution of CRISPR’s applications is traced from bacterial immunity to the forefront of genome editing. Delivery methods—both viral and non-viral—are assessed, with detailed exploration of emerging solutions like lipid nanoparticles and DNA nanocarriers. CRISPR/Cas9 demonstrates significant therapeutic value in oncology by inactivating oncogenes, restoring tumor suppressors, and enhancing immunotherapeutic strategies. Furthermore, promising results have emerged in treating inherited disorders such as beta-thalassemia and sickle cell anaemia, as well as in clinical studies focused on T-cell therapies and viral diseases. Although these developments are encouraging, obstacles persist, including off-target genetic modifications and the intricacies of delivery. Innovations, notably base editing and prime editing, show promise to resolve these barriers. This review highlights the immense therapeutic prospects of CRISPR/Cas9 and stresses the importance of sustained research to fully unlock its clinical benefits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937dc42e57cb7d44775b157991f2406ddf45a511" target='_blank'>
              CRISPR-Cas9 PROTEIN: PHARMACEUTICAL FRONTIERS IN GENETIC RESTORATION
              </a>
            </td>
          <td>
            Senthil Prabhu R, Gunaseelan M, J. R.
          </td>
          <td>2025-08-30</td>
          <td>International Journal of Pharmaceutical Sciences and Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonally established tumor cell lines often do not recapitulate the behavior of cells in tumors. The sequencing of a whole tumor tissue may not uncover transcriptome profiles induced by the interactions of all different cell types within a tumor. Interferons for instance have a vast number of binding sites in their target genes. Access to the DNA binding sites is determined by the epigenomic state of each different cell type within a tumor mass. To understand how genes such as interferons appear to have both tumor-promoting and tumor-inhibiting functions, single-cell transcript analysis was performed in the breast cancer tissue of HER2+ (epidermal growth factor receptor 2) patients. We identified that potential antagonistic oncogenic activities of cells can be due to diverse expression patterns of genes with pleiotropic functions. Molecular pathways both known and novel were identified and were similar with those previously identified for patients with rheumatoid arthritis. Our study demonstrates the efficacy in using single-cell transcript analysis to gain insight into genes with apparent contradictory or paradoxical roles in oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261a79861db7f4bc06df15b1b4671db2e425c476" target='_blank'>
              Single-Cell mRNA Analysis for the Identification of Molecular Pathways of IRF1 in HER2+ Breast Cancer
              </a>
            </td>
          <td>
            Laura Vilardo, P. Pelucchi, Antonia Brindisi, E. Abeni, E. Piscitelli, E. Mosca, G. Bertalot, Mira Palizban, Theodoros Karnavas, A. Gritzapis, Ioannis Misitzis, Martin Götte, Ileana Zucchi, R. Reinbold
          </td>
          <td>2025-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Alkyladenine DNA glycosylase (AAG) is a critical enzyme in the base excision repair (BER) pathway, responsible for removing a broad spectrum of alkylated DNA lesions. While AAG maintains genomic stability, dysregulated activity has been implicated in cancer development, drug resistance, and neurodegenerative diseases. This review synthesizes the current knowledge on AAG’s structure, catalytic mechanism, and polymorphic variants, highlighting their potential roles in disease pathogenesis. A comprehensive bioinformatics analysis of over 370 AAG single-nucleotide polymorphisms (SNPs) is presented, identifying ~40% as high-risk variants likely to impair enzymatic function. Notably, 151 SNPs were predicted to be damaging by multiple algorithms, including substitutions at catalytic residues and non-conserved sites with unknown functional consequences. Analysis of cancer databases (COSMIC, cBioPortal, NCBI) revealed 93 tumor-associated AAG variants, with 18 classified as high-impact mutations. This work underscores the need for mechanistic studies of AAG variants using structural biology, cellular models, and clinical correlation analyses. Deciphering AAG’s polymorphic landscape may unlock personalized strategies for cancer prevention and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35650f59a2a5d69d14473a41169b47ec06f0170" target='_blank'>
              The Link Between Human Alkyladenine DNA Glycosylase and Cancer Development
              </a>
            </td>
          <td>
            O. A. Kladova, A. A. Kuznetsova
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Whole-genome analyses have revealed that large-scale structural variations (SVs) such as whole-genome duplication (WGD) occur early in the development of many cancers. However, the diversity of chromosomal abnormalities within tumors before and after WGD remains poorly understood. Here, we analyzed various types of Japanese tumor genomes via whole-genome sequencing and examined the diversity of WGD by focusing on large SVs at the chromosomal level. WGD was detected in 52% of cases, while the frequency of chromothripsis (CT) was 20%. Although aneuploidy via deletion of chromosome arms was common in many cancers, in rare ovarian cancers, all chromosomes were near-haploidy before WGD. Minor allele analysis revealed that many non-mutated ohnolog genes drifted down chromosome arms after WGD and returned to normal ploidy, but only 17p, including TP53, which is also an ohnolog, underwent loss of heterozygosity due to arm deletion before WGD in most cancers. TP53 mutations were frequently detected in WGD and CT-positive tumors, and these SVs strongly correlated with homologous recombination deficiency scores. Furthermore, these tumors had many mutations that continued to generate neoantigens and resulted in worse survival outcomes. Diversity analysis of tumors with WGD will provide a new perspective on structural abnormalities in tumor genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Recently, cancer has become a leading cause of death worldwide, prompting increased research to understand the pathways involved in cancer development and to identify solutions for its treatment. The PI3K/AKT pathway has garnered significant attention because of its involvement in promoting cell proliferation and inhibiting programmed cell death. The protein phosphatase and tensin homolog on chromosome 10 (PTEN) plays a crucial role in inhibiting this pathway, thereby limiting uncontrolled cell proliferation. PTEN gene expression is strictly regulated at the transcriptional, posttranscriptional, and posttranslational levels, and recent research has focused on PTEN due to its reduced levels in cancer cells. This review aims to provide a deep understanding of the PTEN protein from structural and regulatory perspectives, its mutated forms, and its interactions with the occurrence of various malignant tumors to summarize the recent work performed to combat cancers via molecular strategies to enhance PTEN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006440d6b28c83ed5863d6ee8a11b4692d78170c" target='_blank'>
              PTEN defects in cancer, from gene to protein molecular causes and therapeutic targets
              </a>
            </td>
          <td>
            Ghaith Khasarah, Darina Al Nabilsi, Odai Madani, Farah Khazem, Ranwa Alsayed
          </td>
          <td>2025-08-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>