<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-11-11 06:31:14 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>10</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41df9adbcb86e982509871382b3eba588480f4fb" target='_blank'>
              Base editing screens define the genetic landscape of cancer drug resistance mechanisms
              </a>
            </td>
          <td>
            Matthew A Coelho, Magdalena E Strauss, Alex Watterson, Sarah Cooper, S. Bhosle, G. Illuzzi, Emre Karakoc, Cansu Dincer, Sara F Vieira, Mamta Sharma, Marie Moullet, Daniela Conticelli, Jonas Koeppel, Katrina McCarten, C. M. Cattaneo, Vivien Veninga, Gabriele Picco, Leopold Parts, J. Forment, E. Voest, J. Marioni, Andrew Bassett, Mathew J. Garnett
          </td>
          <td>2024-10-18</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Despite advances in targeted cancer therapy, the promise of precision medicine has been limited by resistance to these treatments. In this study, we propose a mathematical modelling framework incorporating cellular heterogeneity, genetic evolutionary dynamics, and non-genetic plasticity, accounting for both irreversible and reversible drug resistance. Previously we proposed Dynamic Precision Medicine (DPM), a personalized treatment strategy that designed individualized treatment sequences by simulations of irreversible genetic evolutionary dynamics in a heterogeneous tumor. Here we apply DPM to the joint model of reversible and irreversible drug resistance mechanisms, analyze the simulation results and compare the efficacy of various treatment strategies. The results indicate that this enhanced version of DPM results in superior patient outcomes compared with current personalized medicine treatment approaches. Our results provide insights into cancer treatment strategies for heterogeneous tumors with genetic evolutionary dynamics and non-genetic cellular plasticity, potentially leading to improvements in survival time for cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77dda28f0e2ff4035e49a44126de0e17b1ff9547" target='_blank'>
              Personalized cancer treatment strategies incorporating irreversible and reversible drug resistance mechanisms
              </a>
            </td>
          <td>
            Wei He, Matthew D. McCoy, R. Riggins, Robert A Beckman, Chen-Hsiang Yeang
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Understanding the mechanisms by which cancer cells switch between different adaptive states and evade therapeutic interventions is essential for clinical management. In this study, the in vivo cellular dynamics of a new chronic myeloid leukaemia cell line displaying altered phenotype and resistance to tyrosine kinase inhibitors were investigated in correlation with their parental cells for invasiveness/metastasis, angiogenic potential and population kinetics. We showed that the cells exhibiting drug resistance and plastic phenotype possess an increased capacity for invasion compared to their parental cells, that exposure to imatinib mesylate has the potential to enhance cellular motility and that in a leukaemic cell population, even a minority of plastic cells exhibit improved migratory ability. Furthermore, we show that these plastic cells have angiogenic and extravasation potential. The present study provides significant insights into the cellular dynamics displayed by a TKI‐resistant, phenotypically plastic CML cell line, using a zebrafish (Danio rerio) xenograft model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8debdaa7aac5be72ea83fbcd5d951e3b97abea7b" target='_blank'>
              Phenotypically plastic drug‐resistant chronic myeloid leukaemia cell line displays enhanced cellular dynamics in a zebrafish xenograft model
              </a>
            </td>
          <td>
            Seda Baykal, Zeynep Yuce, G. Ozhan
          </td>
          <td>2024-10-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9291404ff76427f0917fd53c4b67029ebf03d548" target='_blank'>
              Role of non-coding RNA in lineage plasticity of prostate cancer.
              </a>
            </td>
          <td>
            Wenhui Tan, Changkai Xiao, Min Ma, Youhan Cao, Zhenguo Huang, Xiaolan Wang, Ran Kang, Zhenfa Li, Ermao Li
          </td>
          <td>2024-11-04</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8682cef5538b95d1f2e68873fc143c3785f2658e" target='_blank'>
              Reciprocal inhibition of NOTCH and SOX2 shapes tumor cell plasticity and therapeutic escape in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Morgane Fournier, Joaquim Javary, Vincent Roh, Nadine Fournier, Freddy Radtke
          </td>
          <td>2024-10-30</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Glioblastoma stands as the most prevalent and aggressive primary brain tumor. The current treatment regimen involves a combination of radiotherapy and chemotherapy post-surgical tumor resection. However, for most patients, tumor recurrence transpires with a more aggressive phenotype. Tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) holds promise as it selectively triggers apoptosis in tumor cells, rendering it an attractive therapeutic option. Nonetheless, numerous tumor cells, glioblastoma included, can develop acquired resistance against TRAIL therapy, yet the underlying mechanisms remain poorly elucidated. Apart from resistance to apoptosis, these mechanisms may also encompass non-genetic factors stemming from intracellular dynamics. In our previous research on generating models resistant to TRAIL, we observed differences in the expression of metabolic pathway genes in TRAIL-resistant glioblastoma cells. Our transcriptomic analysis revealed a positive correlation among pathways related to cellular metabolism in these resistant cells. Further examination of this gene family revealed common alterations across various cell lines. Additionally, to delve deeper into the role of cellular metabolism, we conducted in vivo CRISPR/Cas9 screening using a focused library targeting metabolism. We also compared mitochondria densities using MitoTracker dye between sensitive and resistant GBM cells, noting a significant increase in resistant cells. Using the Seahorse Extracellular Flux Analyzer, we measured cellular metabolism activity and observed higher oxygen consumption in resistant cells, indicative of elevated energy levels. Based on these preliminary findings, our study will undergo thorough genetic and metabolomic screening, promising significant insights into cancer metabolism. The identified target genes and metabolites could potentially serve as therapeutic targets for developing novel cancer treatments, particularly glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/518a0308c77c26c6504aaa96145e6989538e7b80" target='_blank'>
              P04.02.A DISCOVERY OF NEW METABOLIC PATHWAYS IMPLICATED IN THERAPY RESISTANCE IN GLIOBLASTOMA (GBM)
              </a>
            </td>
          <td>
            A. Cingöz
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, T. Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e459f679b760cbc6c5f4d15f5d578e006d0e8b9" target='_blank'>
              Establishment and characterization of 18 Sarcoma Cell Lines: Unraveling the Molecular Mechanisms of Doxorubicin Resistance in Sarcoma Cell Lines
              </a>
            </td>
          <td>
            Young-Eun Cho, Soon-Chan Kim, Ha Jeong Kim, Ilkyu Han, Ja-Lok Ku
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401dfb6b1e61a2b7931608ee7b632746b618204c" target='_blank'>
              Potential promising of synthetic lethality in cancer research and treatment.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Behnam Najafiyan, Reza Morovatshoar, Mahdieh Khorsandi, Adib Dashtizadeh, Arash Kiani, Farnoosh Farzam, Kimia Sadat Kazemi, Mohsen Nabi Afjadi
          </td>
          <td>2024-09-21</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>3</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematologic neoplasia with a complex polyclonal architecture. Among driver lesions, those involving the FLT3 gene represent the most frequent mutations identified at diagnosis. The development of tyrosine kinase inhibitors (TKIs) has improved the clinical outcomes of FLT3-mutated patients (Pt). However, overcoming resistance to these drugs remains a challenge. To unravel the molecular mechanisms underlying therapy resistance and clonal selection, we conducted a longitudinal analysis using a single-cell DNA sequencing approach (MissionBioTapestri® platform, San Francisco, CA, USA) in two patients with FLT3-mutated AML. To this end, samples were collected at the time of diagnosis, during TKI therapy, and at relapse or complete remission. For Pt #1, disease resistance was associated with clonal expansion of minor clones, and 2nd line TKI therapy with gilteritinib provided a proliferative advantage to the clones carrying NRAS and KIT mutations, thereby responsible for relapse. In Pt #2, clonal architecture was less complex, and 1st line TKI therapy with midostaurin was able to eradicate the leukemic clones. Our results corroborate previous findings about clonal selection driven by TKIs, highlighting the importance of a deeper characterization of individual clonal architectures for choosing the best treatment plan for personalized approaches aimed at optimizing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4202a0b52113f492de1c5d891aefedd2f36b44c" target='_blank'>
              Tracking Response and Resistance in Acute Myeloid Leukemia through Single-Cell DNA Sequencing Helps Uncover New Therapeutic Targets
              </a>
            </td>
          <td>
            S. Bruno, E. Borsi, A. Patuelli, L. Bandini, M. Mancini, D. Forte, J. Nanni, Martina Barone, Alessandra Grassi, G. Cristiano, C. Venturi, V. Robustelli, Giulia Atzeni, Cristina Mosca, S. De Santis, C. Monaldi, A. Poletti, C. Terragna, A. Curti, M. Cavo, Simona Soverini, E. Ottaviani
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="
 We aim to offer new insights on the regulation of metabolic plasticity in renal cell carcinoma (RCC) and identify novel metabolic regulatory targets that can be used for potential treatment therapies of the disease. RCC affects an estimated 400,000 people worldwide each year with over 100,00 deaths annually that disproportionately affects racial/ethnic minorities in the United States. RCC is a term that covers an array of kidney cancer subtypes differing in incidence, histopathology, genetic and molecular alterations, as well as in clinical outcomes and prognoses. However, a common characteristic of many RCCs is that they are driven by metabolic rewiring, due to a high frequency of mutations in genes that regulate major metabolic processes in the cell. It was recently discovered that patient overall survival and prognosis has been linked to the expression of certain metabolic signatures in RCC tumor cells. Metabolic plasticity allows cancer cells to survive and adapt to different environments and metabolic stress conditions, such as nutrient deprivation. Therefore, exploring and exploiting metabolic rewiring in these cells may reveal genetic vulnerabilities and dependencies. This could lead to discovery of novel biomarkers and molecular targets that can be implicated as future therapeutic interventions for this disease. To this end, we applied functional genomic screening to expose genetic dependencies and vulnerabilities under different physiologic and metabolic conditions in patient derived medullary (PDM) RCC cells. The medullary RCC is a rare and aggressive RCC subtype that disproportionately affects the African American/Black population. We applied a CRISPR- based combinatorial screening platform that is based on co-expression of Cas9 and Cas12a nucleases and libraries of hybrid guide RNAs (hgRNA) to achieve ultra-efficient gene knockout (Aregger et al., 2021; Gonatopoulos-Pournatzis et al., 2020) in PDM RCC cells. Additionally, cells were screened under different media conditions that mimic human physiological nutrient conditions and a media deprived of lipids, one of the major metabolic substrates of the cell. By utilizing a genome-wide pooled CRISPR gene knockout screen we were able to systematically uncover genes required for cell survival and proliferation in a panel of patient-derived RCCs. We have begun to uncover genetic dependencies and major metabolic pathways that can be elucidated and exploited for future studies, such as those involving ferroptosis which is a targetable pathway for the development of cancer treatment therapies. Our findings demonstrate some of the key genetic drivers underlying cellular adaptations to different physiological environments and nutrient deprivation. In future studies we will further explore the relationship between different key metabolic regulatory genes in RCC and map for genetic interactions in major metabolic pathways.
 Citation Format: Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger. Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd9b94600307109e3720c07a3b1a13c91abd8ae" target='_blank'>
              Abstract C091: Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma
              </a>
            </td>
          <td>
            Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The ability of cancer cells to change and adapt poses a critical challenge to identifying curative solutions. Tumor evolution has been extensively studied from a genetic perspective, to guide clinicians in selecting the most appropriate therapeutic option based on a patient's mutational profile. However, several studies reported that tumors can evolve toward more aggressive stages or become resistant to therapies without changing their genetic makeup. Indeed, several cell-intrinsic and cell-extrinsic mechanisms contribute to tumor evolution. In this viewpoint, I focus on how chromatin, epigenetic, and transcriptional changes contribute to tumor evolution, allowing cancer cells to transition to different cell states and bypass response to therapies. Although tumor nongenetic evolution is harder to trace and predict, understanding its principles might open new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0869302e828b2c81eac62f89984d51de011a3b74" target='_blank'>
              Nongenetic evolution of the tumor: from challenges to new therapeutic opportunities.
              </a>
            </td>
          <td>
            E. Oricchio
          </td>
          <td>2024-10-18</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole-genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2,726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirically testing all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated two compounds as potent PGCC inhibitors. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f39f65e370bc9aafe05599fe5e71c2036a3f471" target='_blank'>
              High-Throughput Empirical and Virtual Screening to Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a hematological malignancy associated with poor prognosis and relapses. MM has a strong inherited component, and while the mutational and transcriptomic landscapes of MM have been characterized extensively, the molecular consequences of germline genetic variation in MM have not been previously explored. Here, we have leveraged matched genotype and expression data from 607 patients to map the germline and somatic regulatory variation in MM. We find that non-coding variation shapes the transcriptional landscape of MM, contributing to overall survival. These regulatory variants alter the binding of transcription factors with known importance in MM, such as IRF4, providing insights into the underlying mechanisms. These data contribute to the increasing understanding of the genetic, molecular, and cellular forces shaping MM risk and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5c5bba32932ad19d6d4a4784baebd131dde155a" target='_blank'>
              Germline and somatic genetic effects on gene expression and outcome in patients with Multiple Myeloma
              </a>
            </td>
          <td>
            H. Natri, Linh Bui, L. Peter, Bianca Argente, A. Federico, A. Gutierrez, Mei-i Chung, Jonathan Keats, Nicholas E Banovich
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK) rearrangement is a major oncogenic driver in non-small cell lung cancer (NSCLC). While ALK tyrosine kinase inhibitors have shown promising therapeutic effects, overcoming resistance with immunotherapy becomes necessary when resistance develops. However, various clinical trials have revealed that their efficacies remain limited. To investigate the tumor microenvironment (TME) factors contributing to poor immune checkpoint blockade responses in ALK-positive patients, we performed single-cell RNA and ATAC sequencing on lung adenocarcinoma (LUAD) tumors with and without ALK rearrangements. Integrative analysis with additional public LUAD cohorts revealed distinct immune landscapes in ALK-positive tumors, marked by enriched innate immunity and depleted adaptive immunity. ALK-positive malignant cells exhibit higher stemness and aggressive phenotype. Tumor-associated macrophages (TAMs) in these tumors predominantly maintain pro-tumoral M2-like states, reinforcing immune suppression. B cells show reduced immune reactivity and impaired tertiary lymphoid structure formation, while CD8+ T cells display bystander-like signatures and reduced tumor reactivity. Single-cell chromatin accessibility profiles combined with regulatory network analysis suggest that differences in transcription factor activities, rather than chromatin accessibility, may underlie T cell dysfunction. These findings provide insights into the immunosuppressive TME of ALK-positive LUAD, potentially explaining the failure of recent immunotherapy trials and highlighting targets for improving efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb7277d768d3078fb9b8e8f161ef723f32126" target='_blank'>
              Single-cell multi-omics reveals tumor microenvironment factors underlying poor immunotherapy responses in ALK-positive lung cancer
              </a>
            </td>
          <td>
            S. Baek, Euijeong Sung, Gamin Kim, M. Hong, Chang Young Lee, H. Shim, Seong Yong Park, Hye Ryun Kim, Insuk Lee
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Evolutionary repair refers to the compensatory evolution that follows perturbations in cellular processes. While evolutionary trajectories are often reproducible, other studies suggest they are shaped by genotype-by-environment (GxE) interactions. Here, we test the predictability of evolutionary repair in response to DNA replication stress—a severe perturbation impairing the conserved mechanisms of DNA synthesis, resulting in genetic instability. We conducted high-throughput experimental evolution on Saccharomyces cerevisiae experiencing constitutive replication stress, grown under different glucose availabilities. We found that glucose levels impact the physiology and adaptation rate of replication stress mutants. However, the genetics of adaptation show remarkable robustness across environments. Recurrent mutations collectively recapitulated the fitness of evolved lines and are advantageous across macronutrient availability. We also identified a novel role of the mediator complex of RNA polymerase II in adaptation to replicative stress. Our results highlight the robustness and predictability of evolutionary repair mechanisms to DNA replication stress and provide new insights into the evolutionary aspects of genome stability, with potential implications for understanding cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d644d70495ca5cd494d5397eda100129033764cf" target='_blank'>
              Compensatory Evolution to DNA Replication Stress is Robust to Nutrient Availability
              </a>
            </td>
          <td>
            Mariana Natalino, M. Fumasoni
          </td>
          <td>2024-11-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="While a subset of patients with metastatic melanoma achieves durable responses to immune checkpoint blockade (ICB) therapies, the majority ultimately exhibit either innate or acquired resistance to these treatments. However, the molecular mechanisms underlying resistance to ICB therapies remain elusive and are warranted to elucidate. Here, we comprehensively investigated the tumor and tumor immune microenvironment (TIME) of paired pre- and post-treatment tumor specimens from metastatic melanoma patients who were primary or secondary resistance to anti-CTLA-4 and/or anti-PD-1/PD-L1 therapies. Differentially expressed gene (DEG) analysis and single-sample gene set enrichment analysis (ssGSEA) with transcriptomic data identified cell cycle and c-MYC signaling as pathway-based resistance signatures. And weighted gene co-expression network analysis (WGCNA) revealed the activation of a cross-resistance meta-program involving key signaling pathways related to tumor progression in ICB resistant melanoma. Moreover, spatially-resolved, image-based immune monitoring analysis by using NanoString’s digital spatial profiling (DSP) and Cyclic Immunofluorescence (CyCIF) showed infiltration of suppressive immune cells in the tumor microenvironment of melanoma with resistance to ICB therapies. Our study reveals the molecular mechanisms underlying resistance to ICB therapies in patients with metastatic melanoma by conducting such integrated analyses of multi-dimensional data, and provides rationale for salvage therapies that will potentially overcome resistance to ICB therapies. Statement of translational relevance This study paves the way for the creation of innovative therapeutic strategies, aimed at subverting resistance to immune checkpoint blockade (ICB) therapies in metastatic melanoma patients. By unraveling the specific molecular mechanisms underlying resistance, scientists can design effective alternative treatments that target pathways such as pathways associated with cell cycle dysregulation and c-MYC signaling. Furthermore, through the application of advanced immune monitoring techniques such as NanoString Digital Spatial Profiling (DSP) and Cyclic Immunofluorescence (CyCIF), this study has significantly enriched our understanding of the tumor microenvironment. This enhanced characterization facilitates the discovery of potential biomarkers that may forecast a patient’s response to ICB treatment. Ultimately, these advancements could potentially refine patient outcomes and foster the development of more personalized cancer treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c359686d6b4f4f12c03d81c082e61089dfa6d8" target='_blank'>
              A Comprehensive Proteogenomic and Spatial Analysis of Innate and Acquired Resistance of Metastatic Melanoma to Immune Checkpoint Blockade Therapies
              </a>
            </td>
          <td>
            Shiyou Wei, Kuang Du, Hongbin Lan, Zhenyu Yang, Yulan Deng, Zhi Wei, Dennie T. Frederick, Jinho Lee, Marilyne Labrie, Tian Tian, Tabea Moll, Yeqing Chen, Ryan J. Sullivan, Gordon B. Mills, Genevieve M. Boland, Keith T. Flaherty, Lunxu Liu, M. Herlyn, Gao Zhang
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Lung cancer remains one of the leading causes of cancer-related deaths globally, with its complexity driven by intricate and intertwined genetic, epigenetic, and environmental factors. Despite advances in genomics, transcriptomics, and proteomics, understanding the phenotypic diversity of lung cancer has lagged behind. Next-generation phenomics, which integrates high-throughput phenotypic data with multiomics approaches and digital technologies such as artificial intelligence (AI), offers a transformative strategy for unraveling the complexity of lung cancer. This approach leverages advanced imaging, single-cell technologies, and AI to capture dynamic phenotypic variations at cellular, tissue, and whole organism levels and in ways resolved in temporal and spatial contexts. By mapping the high-throughput and spatially and temporally resolved phenotypic profiles onto molecular alterations, next-generation phenomics provides deeper insights into the tumor microenvironment, cancer heterogeneity, and drug efficacy, safety, and resistance mechanisms. Furthermore, integrating phenotypic data with genomic and proteomic networks allows for the identification of novel biomarkers and therapeutic targets in ways informed by biological structure and function, fostering precision medicine in lung cancer treatment. This expert review examines and places into context the current advances in next-generation phenomics and its potential to redefine lung cancer diagnosis, prognosis, and therapy. It highlights the emerging role of AI and machine learning in analyzing complex phenotypic datasets, enabling personalized therapeutic interventions. Ultimately, next-generation phenomics holds the promise of bridging the gap between molecular alterations and clinical and population health outcomes, providing a holistic understanding of lung cancer biology that could revolutionize its management and improve patient survival rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff2978937e2ef89767871d93305b0ec9138e48" target='_blank'>
              Next-Generation Cancer Phenomics: A Transformative Approach to Unraveling Lung Cancer Complexity and Advancing Precision Medicine.
              </a>
            </td>
          <td>
            Sanjukta Dasgupta
          </td>
          <td>2024-10-22</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Precision gene editing enables massively parallel identification of cancer-promoting genes Since its discovery in 2012 as a gene-editing tool, CRISPR-Cas9 has been touted as a turning point for drug discovery. Indeed, CRISPR-based technologies have an unprecedented power to interrogate the mechanistic underpinnings of disease by systematically testing the cellular effects of targeted genetic alterations (1). Disease hallmarks specific to cancer, such as unchecked cell proliferation and drug resistance, readily lend themselves to massively parallel CRISPR screens. In these screens, individual cells harboring a guide RNA that also acts as a barcode for a DNA edit of interest may acquire a competitive or diminished proliferative advantage relative to other cells in the same test pool. This allows the perturbation of thousands of genes in a single experiment. The resulting genetic maps generated in cancer cell lines (CCLs) and in cancer-derived tissues now support exceptional speed and economy in the rationalization of new drug development and may shorten bench-to-bedside timelines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2013426359d6875b2659e5a9c78a646561186a" target='_blank'>
              Impact of CRISPR in cancer drug discovery
              </a>
            </td>
          <td>
            Jason Moffat, Alexis C. Komor, Lawrence Lum
          </td>
          <td>2024-10-25</td>
          <td>Science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187812e12825019ab421661fdd39125bff16e32a" target='_blank'>
              Potential role of heteroplasmic mitochondrial DNA mutations in modulating the subtype-specific adaptation of oral squamous cell carcinoma to cisplatin therapy
              </a>
            </td>
          <td>
            Amnani Aminuddin, Pei Yuen Ng, Chee Onn Leong, S. Makpol, E. Chua
          </td>
          <td>2024-10-19</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Prostate cancer (PCa) is the second-leading cause of cancer mortality in U.S. men, with African American (AA) men showing higher incidence and mortality compared to European American men. In addition, AA men are more likely to be diagnosed with more aggressive PCa resulting in reduction of treatment options, especially as tumors develop therapy resistance. Thus, understanding the mechanisms underlying PCa therapy resistance is crucial for developing new therapeutic options. The upregulation of the Lens Epithelium Derived Growth Factor p75 (LEDGF/p75) by glucocorticoid receptor (GR) is a novel mechanism of PCa therapy resistance recently identified by our group. GR upregulation in PCa bypasses the blockade of androgen receptor by inhibitors such as enzalutamide and abiraterone. LEDGF/p75 is an oncogenic transcription co-activator that promotes DNA damage repair and cancer cell survival against environmental stressors including chemotherapeutic drugs such as docetaxel (DTX). An emerging concern in PCa treatment is therapy cross-resistance, in which the molecular mechanisms of resistance to a particular drug confer resistance to another drug. Our recent studies showed that GR upregulates and interacts with LEDGF/p75 forming a transcriptional complex that includes other oncoproteins such as Menin, JPO2, c-Myc and HRP2, and promotes DTX resistance (DTX-R) in PCa cells. In addition, we reported that both GR and LEDGF/p75 are upregulated in enzalutamide resistant (ENZ-R) PCa cells. We hypothesized that the GR- LEDGF/p75 transcriptional network plays a role in PCa therapy cross-resistance to both ENZ and DTX and is a promising therapeutic target. Cell viability assays revealed that ENZ-R in PCa cells confers cross-resistance to DTX. Western blot analysis revealed upregulation of members of the GR-LEDGF/p75 network such as GR, LEDGF/p75, Menin and c-Myc in ENZ-R PCa cells, consistent with similar observations in DTX-R PCa cells. Upon knockdown of GR and LEDGF/p75 in DTX-R PCa cells, we identified by RNA-seq analysis overlapping DEGs associated with both ENZ-R and DTX-R, suggesting that the GR-LEDGF/p75 transcriptional network may contribute to therapy cross-resistance. Consistent with this, mining of publicly available RNA-seq databases identified 46 overlapping differentially expressed genes (DEGs) between ENZ-R and DTX-R PCa cells. Pathway analysis of DEGs regulated by both GR and LEDGF/p75 revealed significant enrichment of Apoptosis, p53, and DNA repair pathways, which are associated with therapy resistance. Ongoing studies are evaluating the anti-cancer properties and synergistic/additive potential of various inhibitors of the GR-LEDGF/p75 transcriptional network in cellular models of ENZ-R and DTX-R. These studies provide novel mechanistic insights into PCa therapy cross-resistance and may lead to the development of novel therapeutic strategies to reduce PCa mortality disparities.
 Citation Format: Pedro T. Ochoa, Evelyn S. Sanchez-Hernandez, Kai Wen Cheng, Isaac Kremsky, Charles Wang, Carlos A. Casiano. The GR-LEDGF/p75 transcriptional network as a promising target to reduce therapy cross-resistance in prostate cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C096.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d0c6225006bf01cdfdfbdc7e8efd4e751393fc1" target='_blank'>
              Abstract C096: The GR-LEDGF/p75 transcriptional network as a promising target to reduce therapy cross-resistance in prostate cancer
              </a>
            </td>
          <td>
            P. Ochoa, Evelyn S. Sanchez-Hernandez, Kai Wen Cheng, Isaac Kremsky, Charles Wang, C. Casiano
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Diverse sets of complete human genomes are required to construct a pangenome reference and to understand the extent of complex structural variation. Here, we sequence 65 diverse human genomes and build 130 haplotype-resolved assemblies (130 Mbp median continuity), closing 92% of all previous assembly gaps1,2 and reaching telomere-to-telomere (T2T) status for 39% of the chromosomes. We highlight complete sequence continuity of complex loci, including the major histocompatibility complex (MHC), SMN1/SMN2, NBPF8, and AMY1/AMY2, and fully resolve 1,852 complex structural variants (SVs). In addition, we completely assemble and validate 1,246 human centromeres. We find up to 30-fold variation in α-satellite high-order repeat (HOR) array length and characterize the pattern of mobile element insertions into α-satellite HOR arrays. While most centromeres predict a single site of kinetochore attachment, epigenetic analysis suggests the presence of two hypomethylated regions for 7% of centromeres. Combining our data with the draft pangenome reference1 significantly enhances genotyping accuracy from short-read data, enabling whole-genome inference3 to a median quality value (QV) of 45. Using this approach, 26,115 SVs per sample are detected, substantially increasing the number of SVs now amenable to downstream disease association studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6654cdcb0238112d367eba4f362c39559b46653" target='_blank'>
              Complex genetic variation in nearly complete human genomes
              </a>
            </td>
          <td>
            Glennis A. Logsdon, Peter Ebert, P. Audano, Mark Loftus, David Porubsky, Jana Ebler, F. Yilmaz, P. Hallast, Timofey Prodanov, DongAhn Yoo, Carolyn A. Paisie, William T. Harvey, Xuefang Zhao, Gianni V. Martino, Mir Henglin, Katherine M. Munson, Keon Rabbani, Chen-Shan Chin, Bida Gu, Hufsah Ashraf, Olanrewaju Austine-Orimoloye, Parithi Balachandran, M. Bonder, Haoyu Cheng, Zechen Chong, J. Crabtree, Mark B. Gerstein, L. Guethlein, Patrick Hasenfeld, G. Hickey, Kendra Hoekzema, Sarah E. Hunt, Matthew Jensen, Yunzhe Jiang, S. Koren, Youngjun Kwon, Chong Li, Heng Li, Jiaqi Li, Paul J. Norman, Keisuke K. Oshima, B. Paten, A. Phillippy, Nicholas R Pollock, Tobias Rausch, M. Rautiainen, Stephan Scholz, Yuwei Song, Arda Söylev, Arvis Sulovari, Likhitha Surapaneni, Vasiliki Tsapalou, Weichen Zhou, Ying Zhou, Qihui Zhu, Michael C Zody, Ryan E. Mills, S. Devine, Xinghua Shi, M. Talkowski, Mark J. P. Chaisson, Alexander T Dilthey, Miriam K. Konkel, J. Korbel, Charles Lee, Christine R. Beck, Evan E. Eichler, T. Marschall
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>3</td>
          <td>97</td>
        </tr>

        <tr id="Mutations in genes encoding critical epigenetic regulators are frequently noted in bladder cancer, however, the impact of these mutations on therapeutic efficacy is unclear. One of the most common driver mutations in bladder cancer occurs in the KDM6A gene, which encodes a histone demethylase that promotes gene transcription. Retrospective analyses of patients with bladder cancer demonstrated that KDM6A mutations correlate with improved overall survival (OS) with immune checkpoint therapy (ICT), while they are associated with lower OS in patients undergoing cisplatin-based chemotherapy. Mechanistic studies utilizing CRISPR-Cas9 mediated deletion of Kdm6a showed reduced expression of DNA mismatch repair (MMR) and DNA double-stranded base repair (DSBR) genes in tumor cells with improved response to anti-PD-1 therapy and attenuated sensitivity to cisplatin-based chemotherapy in preclinical models of bladder cancer. Additionally, the loss of Kdm6a-mediated reduction in glycolysis and intratumoral lactate accumulation impaired histone 3 lysine 9 lactylation (H3K9la) and histone 3 lysine 18 lactylation (H3K18la) in Tregs with concurrent decrease in the expression of key genes including Foxp3, Tgfb and Pdcd1 and their immune-suppressive function. Further, reduced expansion of PD-1hi Tregs improved the ratio of cytotoxic T cells to Tregs and response to anti-PD-1 therapy in Kdm6a deficient tumor-bearing mice. Collectively, this study provided key insights into the role of KDM6A-mediated epigenetic regulation of DNA repair and metabolic reprogramming which potentially govern response to chemotherapy and ICT thus highlighting the utility of KDM6A mutation status for patient stratification and development of personalized treatment algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825bb07017841ff9cc12c29d05779f53d82d0047" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming differentially regulates responses to immune checkpoint therapy and chemotherapy in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, D. Raychaudhuri, Bidisha Chakraborty, Swadhin Meher, Aminah J. Tannir, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, P. Pilié, Sangeeta Goswami
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Epimutations modify gene expression and lead to phenotypic variation while the encoding DNA sequence remains unchanged. Epimutations mediated by RNA interference (RNAi) and/or chromatin modifications can confer antifungal drug resistance and may impact virulence traits in fungi (1–5). However, whether these epigenetic modifications can be transmitted across generations following sexual reproduction was unclear. This study demonstrates that RNAi epimutations conferring antifungal drug resistance are transgenerationally inherited in the human fungal pathogen Mucor circinelloides. Our research revealed that RNAi-based antifungal resistance follows a DNA sequence-independent, non-Mendelian inheritance pattern. Small RNAs (sRNAs) are the exclusive determinants of inheritance, transmitting drug resistance independently of other known repressive epigenetic modifications. Unique sRNA signature patterns can be traced through inheritance from parent to progeny, further supporting RNA as an alternative molecule for transmitting information across generations. This study marks a significant advance in understanding epigenetic inheritance, highlighting RNAi-exclusive epimutations as a widespread phenomenon in the pathogenic M. circinelloides species complex. Understanding how epimutations occur, propagate, and confer resistance may enable their detection in other eukaryotic pathogens, provide solutions for challenges posed by rising antimicrobial drug resistance (AMR), and also advance research on phenotypic adaptability and its evolutionary implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df7aab0fd6f057abc3a0de9ddf4f9920c418d202" target='_blank'>
              RNAi epimutations conferring antifungal drug resistance are inheritable
              </a>
            </td>
          <td>
            Carlos Pérez-Arques, M. Navarro-Mendoza, Ziyan Xu, Grit Walther, Joseph Heitman
          </td>
          <td>2024-10-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="While FLT3 inhibitors have significantly improved the treatment of aggressive FLT3-mutated acute myeloid leukemia (AML), the emergence of resistance remains as a major challenge. Here, we applied our recently developed single-cell lineage-tracing method ReSisTrace to identify cells that are pre-resistant or pre-sensitive to FLT3 inhibitors midostaurin and quizartinib in FLT3-ITD-positive AML. By comparing the gene expression profiles of these cells, we unraveled the transcriptional pre-resistance signatures, including G1 to S phase transition 1 (GSPT1) gene. Targeting GSPT1 with the small molecule CC-90009 exhibited strong synergistic effect when combined with FLT3 inhibitors in the FLT3-ITD-mutated MOLM-13 and MV4-11 cell lines and primary AML patient samples. Further, we identified novel compounds that induced transcriptomic changes opposite to the pre-resistance signatures, thereby driving cells to FLT3 inhibitor-sensitive states. Vistusertib (mTOR inhibitor), linsitinib (IGF1R and insulin receptor inhibitor), and meisoindigo (IGF1R and Src family kinase inhibitor), all inhibiting pathways parallel to or downstream of oncogenic FLT3 signaling, were predicted and validated to pre-sensitize the FLT3-ITD-mutated cell lines and primary cells to FLT3 inhibitors. Collectively, these findings demonstrate the validity of our lineage-tracing method in unveiling pre-existing transcriptional features of treatment vulnerability in hematological cancers, and elucidate novel strategies for enhancing FLT3 inhibitor treatment efficacy in FLT3-ITD-positive AML by preventing the emergence of treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0428ff40d99003abc28b0d9e495047e8ef959354" target='_blank'>
              Uncovering pre-sensitizing agents to FLT3 inhibitors in acute myeloid leukemia with ReSisTrace lineage tracing
              </a>
            </td>
          <td>
            Johanna Eriksson, Shuyu Zheng, Jie Bao, Jun Dai, Wenyu Wang, Anna Vähärautio, Jing Tang
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24869ef91b9d9f698a0aa1c1c25527b7385dd280" target='_blank'>
              Deciphering resistance mechanisms in cancer: final report of MATCH-R study with a focus on molecular drivers and PDX development
              </a>
            </td>
          <td>
            Damien Vasseur, L. Bigot, Kristi Beshiri, Juan-David Florez-Arango, F. Facchinetti, A. Hollebecque, L. Tselikas, M. Aldea, F. Blanc-Durand, A. Gazzah, D. Planchard, Ludovic Lacroix, N. Pata-Merci, Catline Nobre, Alice Da Silva, C. Nicotra, M. Ngo-Camus, F. Braye, Sergey Nikolaev, S. Michiels, Gerome Jules-Clement, K. Olaussen, Fabrice André, J. Scoazec, F. Barlesi, Santiago Ponce, Jean-Charles Soria, Benjamin Besse, Yohann Loriot, L. Friboulet
          </td>
          <td>2024-10-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Combining epigenetic alterations with cancer immunotherapy offers a promising approach to improve therapeutic outcomes by targeting the complex biology of tumors. Epigenetic mechanisms such as DNA methylation, histone modifications, and non-coding RNAs (ncRNAs) play a dual role in maintaining immune homeostasis and promoting cancer progression. Changes like hypomethylation in tumor cells can contribute to immune evasion and treatment resistance. Advances in epigenetic tools, particularly clustered regularly interspaced short palindromic repeat (CRISPR) and their associated protein (Cas9)-based epigenetic editing, allow for precise gene expression control, opening new avenues for research and therapy. The improved accuracy of CRISPR/dCas9 systems, when paired with appropriate delivery methods such as viral vectors or nanoparticles, has facilitated innovative combination therapies involving immune checkpoint inhibitors and epigenetic drugs. These combinations enhance the immune system's ability to recognize and destroy cancer cells. Despite these advancements, challenges like off-target effects, delivery issues, and resistance remain. Current research is focused on identifying new therapeutic targets, improving delivery systems, and using real-time feedback to refine treatment. Overall, combining immunotherapy with epigenetic modifications holds significant potential for personalized cancer treatment, paving the way for more effective and individualized therapeutic strategies that target both the immune system and the tumor microenvironment. Further development in this area is expected to revolutionize cancer therapy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b341f5edae61e3df7a281ad746dc6490bf615c55" target='_blank'>
              Harnessing Epigenetic Mechanisms to Overcome Immune Evasion in Cancer: The Current Strategies and Future Directions
              </a>
            </td>
          <td>
            Rajabikramaditya Panda, Sumithra Mohan, Chitra Vellapandian
          </td>
          <td>2024-10-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Approximately 50% of patients diagnosed with ovarian cancer harbor tumors with mutations in BRCA1, BRCA2, or other genes involved in homologous recombination repair (HR). The presence of homologous recombination deficiency (HRD) is an approved biomarker for poly-ADP-ribose polymerase inhibitors (PARPis) as a maintenance treatment following a positive response to initial platinum-based chemotherapy. Despite this treatment option, the development of resistance to PARPis is common among recurrent disease patients, leading to a poor prognosis. In this study, we conducted a comprehensive analysis using publicly available datasets to elucidate the molecular mechanisms driving PARPi resistance in BRCA1-deficient ovarian cancer. Our findings reveal a central role for the interferon (IFN) pathway in mediating resistance in the context of BRCA1 deficiency. Through integrative bioinformatics approaches, we identified LY6E, an interferon-stimulated gene, as a key mediator of PARPi resistance, with its expression linked to an immunosuppressive tumor microenvironment (TME) encouraging tumor progression and invasion. LY6E amplification correlates with poor prognosis and increased expression of immune-related gene signatures, which is predictive of immunotherapy response. Interestingly, LY6E expression upon PARPi treatment resistance was found to be dependent on BRCA1 status. Gene expression analysis in the Orien/cBioPortal database revealed an association between LY6E and genes involved in DNA repair, such as Rad21 and PUF60, emphasizing the interplay between DNA repair pathways and immune modulation. Moreover, PUF60, Rad21, and LY6E are located on chromosome 8q24, a locus often amplified and associated with the progression of ovarian cancer. Overall, our study provides novel insights into the molecular determinants of PARPi resistance and highlights LY6E as a promising prognostic biomarker in the management of HRD ovarian cancer. Future studies are needed to fully elucidate the molecular mechanisms underlying the role of LY6E in PARPi resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4052305f8854485a3b85613abf68315aff09bfc" target='_blank'>
              Exploring Molecular Drivers of PARPi Resistance in BRCA1-Deficient Ovarian Cancer: The Role of LY6E and Immunomodulation
              </a>
            </td>
          <td>
            T. Petta, Joseph Carlson
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d68a011913dc4e071fed1241ce9416cb60688a" target='_blank'>
              A novel ITGB8 transcript variant sustains ovarian cancer cell survival through genomic instability and altered ploidy on a mutant p53 background
              </a>
            </td>
          <td>
            Aravindan Narayanan, Ankita S. More, Muskan Talreja, Avinash M. Mali, Sannannagari Boya Vinay, S. Bapat
          </td>
          <td>2024-11-06</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: The rat osteosarcoma cell line UMR-106 is widely used for the study of bone cancer biology but it has not been well characterized with modern genomic methods. Methods: To better understand the biology of UMR-106 cells we used a combination of optical genome mapping (OGM), long-read sequencing nanopore sequencing and RNA sequencing.The UMR-106 genome was compared to a strain-matched Sprague-Dawley rat for variants associated with human osteosarcoma while expression data were contrasted with a public osteoblast dataset. Results: Using the COSMIC database to identify the most affected genes in human osteosarcomas we found somatic mutations in Tp53 and H3f3a. OGM identified a relatively small number of differences between the cell line and a strain-matched control animal but did detect a ~45 Mb block of amplification that included Myc on chromosome 7 which was confirmed by long-read sequencing. The amplified region showed several blocks of non-contiguous rearranged sequence implying complex rearrangements during their formation and included 14 genes reported as biomarkers in human osteosarcoma, many of which also showed increased transcription. A comparison of 5mC methylation from the nanopore reads of tumor and control samples identified genes with distinct differences including the OS marker Cdkn2a. Conclusions: This dataset illustrates the value of long DNA methods for the characterization of cell lines and how inter-species analysis can inform us about the genetic nature underlying mutations that underpin specific tumor types. The data should be a valuable resource for investigators studying osteosarcoma, in general, and specifically the UMR-106 model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0957ae8779454bb6e3009d9a818af23c55e01e" target='_blank'>
              Characterization of the Rat Osteosarcoma Cell Line UMR-106 by Long-Read Technologies Identifies a Large Block of Amplified Genes Associated with Human Disease
              </a>
            </td>
          <td>
            Alan F. Scott, David W. Mohr, William A. Littrell, Reshma Babu, Michelle Kokosinski, Victoria Stinnett, Janvi Madhiwala, John Anderson, Ying S. Zou, Kathleen L. Gabrielson
          </td>
          <td>2024-09-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer-related mortality worldwide, primarily driven by genetic mutations. The most common genetic alterations implicated in lung cancer include mutations in TP53, KRAS, KEAP1, NF1, EGFR, NRF2, ATM, ALK, Rb1, BRAF, MET, and ERBB2. Targeted therapies have been developed to inhibit cancer growth by focusing on these specific genetic mutations. However, either the mutations are undruggable or the efficacy of these therapies is often compromised over time due to the emergence of drug resistance, which can occur through additional mutations in the targeted protein or alternative growth signaling pathways. In recent years, immunotherapy has emerged as a promising approach to enhance the effectiveness of cancer treatment by leveraging the body’s immune system. Notable advancements include immune checkpoint inhibitors, monoclonal antibodies targeting cell surface receptors, antibody–drug conjugates, and bispecific antibodies. This review provides an overview of the mechanisms of FDA-approved immunotherapeutic drugs, offering an updated perspective on the current state and future developments in lung cancer therapy. More importantly, the factors that positively and negatively impact the immunotherapy’s efficacy will also be discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/085b4866f89074fa926fadcafb60ecbab395ec3f" target='_blank'>
              The Impact of Genetic Mutations on the Efficacy of Immunotherapies in Lung Cancer
              </a>
            </td>
          <td>
            Ki Lui, Kwok-Kuen Cheung, Winnie Wing Man Ng, Yanping Wang, Doreen W. H. Au, W. C. Cho
          </td>
          <td>2024-11-07</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic cancer is an aggressive disease with limited therapeutic options. Gemcitabine is a clinically used chemotherapy for pancreatic cancer treatment but has limited efficacy. The molecular mechanisms responsible for gemcitabine resistance in pancreatic cancer are still unclear. Here, we analyzed the genomics of drug sensitivity in cancer dataset and identified various pancreatic cancer cells with distinct sensitivity to gemcitabine. Validation with proliferation and clonogenic assays confirmed that while CFPAC1 and PANC0327 are highly gemcitabine-sensitive, ASPC1 and PANC1 display intrinsic gemcitabine resistance. Analysis of the gene expression profile of these cells identified several genes that are differentially upregulated in resistant versus sensitive cells. Pathway enrichment analysis using the differential genes revealed the upregulation of Hippo, Wnt, MAPK signaling pathways as well as autophagy and glutathione metabolism in the resistant cells. Using quantitative PCR, immunoblotting and biochemical assays, and extracellular flux analysis methods, we have characterized the molecular distinctions between gemcitabine sensitive and resistant cells. Our results provide insight on the molecular mechanisms driving gemcitabine resistance in pancreatic cancer and could lead to novel therapeutic strategies to overcome resistance and improve patients’ treatment.
 Citation Format: Amina D Bilalbegovic, Bowen Fu, Zeribe Nwosu. Identification of novel drivers of gemcitabine resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52052963f11a91114395dd33df22ba52dd0b89f" target='_blank'>
              Abstract B089: Identification of novel drivers of gemcitabine resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Amina D Bilalbegovic, Bowen Fu, Z. Nwosu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Resistance to endocrine therapies (ET) is common in estrogen receptor (ER) positive breast cancer, and most relapsed patients die with ET-resistant disease. While genetic mutations provide explanations for some relapses, mechanisms of resistance remain undefined in many cases. Drug-induced epigenetic reprogramming has been shown to provide possible routes to resistance. By analyzing histone H3 lysine 27 acetylation (H3K27ac) profiles and transcriptional reprogramming in models of ET resistance, we discovered that selective ER degraders (SERDs), such as fulvestrant, promote expression of VGLL1, a co-activator for TEAD transcription factors. VGLL1, acting via TEADs, promoted expression of genes that drive growth of fulvestrant-resistant breast cancer cells. Pharmacological disruption of VGLL1/TEAD4 interaction inhibited VGLL1/TEAD-induced transcriptional programs to prevent growth of resistant cells. EGFR was among the VGLL1/TEAD-regulated genes, and VGLL1-directed EGFR upregulation sensitized fulvestrant-resistant breast cancer cells to EGFR inhibitors. Taken together, these findings identify VGLL1 as a transcriptional driver in ET resistance and advance therapeutic possibilities for relapsed ER+ breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24a869ddbb92dd50c0974c10582d19a6e18e2d9" target='_blank'>
              Induction of the TEAD Co-activator VGLL1 by Estrogen Receptor-Targeted Therapy Drives Resistance in Breast Cancer.
              </a>
            </td>
          <td>
            Carolina Gemma, Chun-Fui Lai, Anup K Singh, A. Belfiore, N. Portman, Heloisa Z Milioli, M. Periyasamy, Sara Raafat, Alyssa J Nicholls, Claire M Davies, Naina R Patel, Georgia M Simmons, Hailing Fan, Van T M Nguyen, Luca Magnani, Emad A. Rakha, Lesley-Ann Martin, Elgene Lim, R. C. Coombes, G. Pruneri, L. Buluwela, Simak Ali
          </td>
          <td>2024-10-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Genomic studies have identified frequent mutations in subunits of the SWI/SNF chromatin remodeling complex including SMARCA4 and ARID1A in non-small cell lung cancer. Previously, we and others have identified that SMARCA4-mutant lung cancers are highly dependent on oxidative phosphorylation (OXPHOS). Despite initial excitements, therapeutics targeting metabolic pathways such as OXPHOS have largely been disappointing due to rapid adaptation of cancer cells to inhibition of single metabolic enzymes or pathways, suggesting novel combination strategies to overcome adaptive responses are urgently needed. Here, we performed a functional genomics screen using CRISPR-Cas9 library targeting genes with available FDA approved therapeutics and identified ROCK1/2 as a top hit that sensitizes cancer cells to OXPHOS inhibition. We validate these results by orthogonal genetic and pharmacologic approaches by demonstrating that KD025 (Belumosudil), an FDA approved ROCK inhibitor, has highly synergistic anti-cancer activity in vitro and in vivo in combination with OXPHOS inhibition. Mechanistically, we showed that this combination induced a rapid, profound energetic stress and cell cycle arrest that was in part due to ROCK inhibition-mediated suppression of the adaptive increase in glycolysis normally seen by OXPHOS inhibition. Furthermore, we applied global phosphoproteomics and kinase-motif enrichment analysis to uncover a dynamic regulatory kinome upon combination of OXPHOS and ROCK inhibition. Importantly, we found converging phosphorylation-dependent regulatory cross-talk by AMPK and ROCK kinases on key RHO GTPase signaling/ROCK-dependent substrates such as PPP1R12A, NUMA1 and PKMYT1 that are known regulators of cell cycle progression. Taken together, our study identified ROCK kinases as critical mediators of metabolic adaptation of cancer cells to OXPHOS inhibition and provides a strong rationale for pursuing ROCK inhibitors as novel combination partners to OXPHOS inhibitors in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa35b0eb45529ff5e6512abe7e0fede05865b618" target='_blank'>
              Therapeutic modulation of ROCK overcomes metabolic adaptation of cancer cells to OXPHOS inhibition and drives synergistic anti-tumor activity
              </a>
            </td>
          <td>
            Nicholas Blazanin, Xiaobing Liang, Iqbal Mahmud, Eiru Kim, Sara A Martinez, Lin Tan, Waikin Chan, N. Anvar, M. Ha, Md Qudratullah, R. Minelli, M. Peoples, Philip L. Lorenzi, Traver Hart, Yonathan Lissanu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb72e1941a1af14f302d1d6e0b549c2a4a1aa635" target='_blank'>
              Defining the transcriptome of PIK3CA-altered cells in a human capillary malformation using single cell long-read sequencing
              </a>
            </td>
          <td>
            Michelle A. Wedemeyer, Tianli Ding, Elizabeth A. R. Garfinkle, Jesse J. Westfall, Jaye B. Navarro, Maria Elena Hernandez Gonzalez, Elizabeth A. Varga, Patricia Witman, Elaine R. Mardis, Catherine E. Cottrell, Anthony R Miller, Katherine E. Miller
          </td>
          <td>2024-10-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7f6d0809acd049c0dda666004c8622cf27bd784" target='_blank'>
              Dissecting L-glutamine metabolism in acute myeloid leukemia: single-cell insights and therapeutic implications
              </a>
            </td>
          <td>
            Yanli Chen
          </td>
          <td>2024-11-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Glioblastoma is an aggressive primary malignant brain tumour with poor survival. Unsuccessful improvement of treatment could be due to transcriptional plasticity of the cancer cells including stem-like cancer cells. Short-term cultured patient-derived neurospheres consisting of cancer cells and stem-like cancer cells are used in this project to find individual drug sensitivities via drug screening. Drug resistance mechanisms on transcriptome level is further explored at high resolution using single nucleus RNA sequencing (snRNA-seq) on drug- and mock-treated neurospheres to find drug vulnerabilities.



 Neurospheres were established from patients included in Protarget, a Phase II, prospective, non-randomized clinical trial. A library including BET-inhibitors was used for high-throughput drug screening to evaluate ex vivo patient drug sensitivities. DMSO-treated and drug-treated neurospheres were prepared for snRNA-seq using 10X genomics. We analyzed snRNA-seq using state-of-the art bioinformatic methods to derive e.g. differentially expressed genes, transcriptional states, and large-scale somatic copy number alterations.



 Through integrative analysis of drug- and DMSO-treated neurosphere cultures, we identified drug-specific differences in both transcriptional state, cell cycle, and stem-like signatures. We found that major copy number changes were preserved between the primary tumour surgical sample and neurosphere culture, exhibiting low average divergence. We found tumour-cell specific transcriptomic differences with clusters exhibiting more extreme drug-responses, implying that neurosphere cultures either sustain or develop sub-clonal, intrinsic cell-type specific drug-responses. One glioblastoma patient underwent treatment with a BET-inhibitor as part of a clinical trial and had initial good clinical response. The transcriptional response following treatment with the same BET-inhibitor on neurospheres from the same patient revealed an increased block in cell cycle, induced transcriptional state changes towards MES2- and OPC-like states, and revealed small clusters of tumour cells showing reduced response to the BET-inhibitor, indicating a pattern of drug resistance.



 We find overall high concordance between the initial tumour surgical sample, assessed by whole-genome sequencing, and patient-derived neurospheres at the level of large-scale copy number changes. Transcriptional subtypes change during treatment including response to a BET-inhibitor on sensitive patient-derived neurospheres. We conclude that combining high-throughput drug screening with snRNA-seq on DMSO-treated and drug-treated neurospheres is a feasible way to access response and resistance mechanisms in glioblastoma, and targeting transcriptional states could be a promising way to treat glioblastoma patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35ffeb471ac7dd263a23a8284dfbe8557f014c84" target='_blank'>
              P01.05.A DRUG-RESPONSE ANALYSIS AND SINGLE NUCLEUS RNA SEQUENCING ON PATIENT-DERIVED NEUROSPHERES TO REVEAL DRUG RESPONSE AND RESISTANCE MECHANISMS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            M. J. S. Lü, J. Hendriksen, J. Ariey-Bonnet, A. Anand, S. R. Michaelsen, F. G. R. González, A. Locallo, A. Azam, J. Skjøth-Rasmussen, P. Hamerlik, D. Nørøxe, B. Hasselbalch, E. Vik-Mo, B. Kristensen, T. Urup, U. Lassen, H. Poulsen, K. Wennerberg, J. Weischenfeldt
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f44ac4365f3d537cc727149bd2f0c2b4a261aed5" target='_blank'>
              The neuroendocrine transition in prostate cancer is dynamic and dependent on ASCL1.
              </a>
            </td>
          <td>
            Rodrigo Romero, Tinyi Chu, Tania J González Robles, Perianne Smith, Yubin Xie, Harmanpreet Kaur, Sara Yoder, Huiyong Zhao, Chenyi Mao, Wenfei Kang, Maria V. Pulina, K. Lawrence, Anuradha Gopalan, S. Zaidi, K. Yoo, Jungmin Choi, Ning Fan, Olivia Gerstner, W. Karthaus, E. deStanchina, Kelly V. Ruggles, P. Westcott, R. Chaligné, D. Pe’er, C. Sawyers
          </td>
          <td>2024-10-11</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="
 The inter- and intra-tumor heterogeneity of triple negative breast cancers (TNBC), which is reflected in diverse drug responses, interplays with tumor evolution. Here, we developed a preclinical experimental and analytical framework using treatment-naive TNBC patient-derived tumor xenografts (PDTX) to test their predictive value in personalized cancer treatment approaches. Patients and their matched PDTX exhibited concordant drug responses to neoadjuvant therapy using two trial designs and dosing schedules. This platform enabled analysis of non-genetic mechanisms involved in relapse dynamics. Treatment resulted in permanent phenotypic changes with functional and therapeutic consequences. High throughput drug screening methods in ex vivo patient derived tumor xenograft cells (PDTCs) revealed patient-specific drug response changes dependent on first-line therapy. This was validated in vivo, as exemplified by a change in olaparib sensitivity in tumors previously treated with clinically relevant cycles of standard-of-care chemotherapy. In summary, PDTXs provide a robust tool to test patient drug responses and therapeutic regimens and to model evolutionary trajectories. However, high inter-model variability and permanent non-genomic transcriptional changes constrain their use for personalized cancer therapy. This work highlights important considerations associated with preclinical drug response modeling and potential uses of the platform to identify efficacious and preferential sequential therapeutic regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8c0bbe2cdb3aa66895b4352d80e8131c8d6526" target='_blank'>
              Modeling Drug Responses and Evolutionary Dynamics using Patient-Derived Xenografts Reveals Precision Medicine Strategies for Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            A. Shea, Y. Eyal-Lubling, Daniel Guerrero-Romero, Raquel Manzano Garcia, W. Greenwood, Martin O’Reilly, D. Georgopoulou, Maurizio Callari, G. Lerda, Sophia Wix, A. Giovannetti, Riccardo Masina, E. Esmaeilishirazifard, Wei Cope, , Ai Nagano, Lisa Young, Steven Kupczak, Yi Cheng, H. Bardwell, E. Provenzano, J. Kane, Jonathan Lay, L. Grybowicz, K. McAdam, Carlos Caldas, Jean Abraham, O. Rueda, Alejandra Bruna
          </td>
          <td>2024-11-08</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background: Kidney renal clear cell carcinoma (KIRC) is the most prevalent subtype of malignant renal cell carcinoma and is well known as a common genitourinary cancer. Stratifying tumors based on heterogeneity is essential for better treatment options. Methods: In this study, consensus clusters were constructed based on gene expression, DNA methylation, and gene mutation data, which were combined with multiple clustering algorithms. After identifying two heterogeneous subtypes, we analyzed the molecular characteristics, immunotherapy response, and drug sensitivity differences of each subtype. And we further integrated bulk data and single-cell RNA sequencing (scRNA-Seq) data to infer the immune cell composition and malignant tumor cell proportion of subtype-related cell subpopulations. Results: Among the two identified consensus subtypes (CS1 and CS2), CS1 was enriched in more inflammation-related and oncogenic pathways than CS2. Simultaneously, CS1 showed a worse prognosis and we found more copy number variations and BAP1 mutations in CS1. Although CS1 had a high immune infiltration score, it exhibited high expression of suppressive immune features. Based on the prediction of immunotherapy and drug sensitivity, we inferred that CS1 may respond poorly to immunotherapy and be less sensitive to targeted drugs. The analysis of bulk data integrated with single-cell data further reflected the high expression of inhibitory immune features in CS1 and the high proportion of malignant tumor cells. And CS2 contained a large number of plasmacytoid B cells, presenting an activated immune microenvironment. Finally, the robustness of our subtypes was successfully validated in four external datasets. Conclusion: In summary, we conducted a comprehensive analysis of multi-omics data with 10 clustering algorithms to reveal the molecular characteristics of KIRC patients and validated the relevant conclusions by single-cell analysis and external data. Our findings discovered new KIRC subtypes and may further guide personalized and precision treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/203996654d7983016b85becfbf4c5b9e1aed4bef" target='_blank'>
              Integrative analysis of single-cell and bulk multi-omics data to reveal subtype-specific characteristics and therapeutic strategies in clear cell renal cell carcinoma patients
              </a>
            </td>
          <td>
            Xinjia Ruan, Chong Lai, Leqi Li, Bei Wang, Xiaofan Lu, Dandan Zhang, Jingya Fang, Maode Lai, Fangrong Yan
          </td>
          <td>2024-10-18</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. Guertin, J. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Isocitrate dehydrogenase (IDH) mutant gliomas are progressive, invariably fatal brain tumors affecting young and middle-age adult patients. Previous single-cell transcriptomic characterization of IDH-mutant gliomas has uncovered a heterogeneity of cell states, with a developmental hierarchy that starts with neural progenitor-like stem cells that transition to more differentiated malignant cell states. Despite this understanding, the dynamics of these cell state transitions as tumors undergo grade progression, and the factors that facilitate or restrict cell state transitions remain poorly understood.



 Here, we characterize the cell states of IDH-mutant gliomas via single-cell gene expression (scRNA-seq) and single-cell analyses of accessible chromatin (scATAC-seq) in tumors spanning all grades using fresh patient samples and compare them with transcriptional programs and chromatin accessibility profiles of normal brain cells. We perform mechanism validation experiments using IDH-mutant glioma organoids.



 We find that tumor transcriptional cell states experience a marked shift as tumors undergo progression. In low-grade tumors, predominant cell states mimic slow-cycling oligodendrocyte progenitor-like cells (OPCs). With grade progression, predominant cell states transition to resemble highly proliferative neural progenitor-like cells (NPCs). Additionally, scATAC-seq analyses revealed novel permissive chromatin states, which could facilitate this transition. Genetic analyses identified several molecular alterations in high-grade tumors, including loss of DNA methylation, oncogene amplification, and the deletion of genes associated with the interferon (IFN) response. Those IFN genes, as well as tumor suppressors, are initially repressed by DNA hypermethylation in low-grade tumors. Lastly, we validate the role of DNA hypomethylation in favoring immune evasion with grade progression using an IDH-mutant glioma organoid model.



 Overall, our findings provide evidence that tumor progression in IDH-mutant gliomas is driven by an interplay of epigenetic and genetic mechanisms, with the former being determinant in low-grade tumors and the latter in high-grade tumors. The emergence of genetically-driven mechanisms promotes more mitotically active cell states as well as decreased anti-tumoral immune activation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13fd62dc7724c0acd329992464a530f1dbcd46aa" target='_blank'>
              KS02.6.A SINGLE-CELL MULTIMODAL ANALYSIS OF IDH-MUTANT GLIOMAS UNCOVERS SHIFT FROM EPIGENETIC TO GENETIC DRIVERS WITH TUMOR PROGRESSION
              </a>
            </td>
          <td>
            L. G. Gonzalez Castro, J. Wu, S. Battaglia, C. El Farran, J. D’Antonio, T. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The majority of malignant tumors exhibit an altered metabolic phenotype that ultimately provides the required energy and molecular precursors necessary for unregulated cell division. Within this, phosphoserine aminotransferase 1 (PSAT1) is involved in de novo serine biosynthesis and its activity promotes various biochemical processes, including one-carbon metabolism. It also directly generates α-ketoglutarate (α-KG), a Kreb cycle intermediate and epigenetic-regulating metabolite. Prior studies examining PSAT1 depletion have identified individual affected downstream pathways, such as GSK3β and E2F, in several cancer types, including non-small-cell lung cancer (NSCLC). However, global gene expression examination in response to PSAT1 loss, particularly in EGFR mutant NSCLC, has not been unexplored. Transcriptional profiling of EGFR mutant NSCLC cells with or without stable knock-down of PSAT1 identified differentially expressed genes (DEGs) enriched in several metabolic pathways required for cell division, including amino acid and nucleotide biosynthesis. Supplementation studies involving non-essential amino acids, nucleosides and α-KG partially restored defects in anchorage-independent growth due to the knockdown of PSAT1. Kyoto Encyclopedia of Genes and Genomes and Gene Ontology enrichment analysis identified potential impacts on actin cytoskeleton arrangement and β-catenin activity, which were rescued by PSAT1 re-expression. Finally, a comparative analysis of PSAT1 DEGs against transcripts enriched in patient EGFR mutant lung tumors identified a gene signature that is associated with overall and relapse-free survival (RFS) and was able to distinguish low or high-risk populations for RFS in early-stage EGFR mutant NSCLC. Overall, investigating genes altered by PSAT1 loss confirmed known PSAT1-regulated cellular pathways, identified a previously unknown role in the mediation of cytoskeleton arrangement in EGFR mutant NSCLC cells and allowed for the characterization of a gene signature with putative predictive potential for RFS in early-stage disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50181c5f02cda5725c20d7feae248e7edda179a9" target='_blank'>
              Bioinformatics analysis of PSAT1 loss identifies downstream pathways regulated in EGFR mutant NSCLC and a selective gene signature for predicting the risk of relapse
              </a>
            </td>
          <td>
            Rumeysa Biyik‐Sit, Sabine Waigel, Kalina Andreeva, Eric Rouchka, Brian Clem
          </td>
          <td>2024-10-17</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Drug resistance remains a major clinical challenge in the treatment of colorectal cancer (CRC) with conventional chemotherapy. Analyzing changes within tumor cells and tumor microenvironment (TME) after treatment and in metastases is essential to understanding how resistance develops. In this study, we analyzed scRNA-seq data from 56 CRCs including treatment-naïve tumors and tumors treated with standard chemotherapy with the known response status (18 responders and 6 progressors). In our cohort, primary left-sided CRCs were associated with metastatic potential mesenchymal phenotype and with depleted B cells. In the post-treatment CRC, there was a high prevalence of dendritic cells (DC) in the TME in the response group. The DC-derived signature was associated with better survival in a large CRC cohort from the TCGA. In progressors there was an enrichment of pericyte-like fibroblasts, which appeared to be associated with poor survival in a CRC-TCGA cohort. Progressors also showed elevated fractions of exhausted CD8+ T memory cells suggesting a pro-inflammatory TME. In tumor cells of progressors group, we identified specific expression of chemo-protective markers MTRNR2L1 and CDX1; and their co-expression with stemness-related immune-checkpoint CD24. In summary, scRNA-seq provides a valuable information for the discovery of prognostic markers, and reveals distinct features potentially underlying response to chemotherapy or disease progression in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d907795ada63036c087467c94e2007531b414d" target='_blank'>
              Single-cell Transcriptome Profiling of Post-treatment and Treatment-naïve Colorectal Cancer: Insights into Putative Mechanisms of Chemoresistance
              </a>
            </td>
          <td>
            Grigory A. Puzanov, Clémence Astier, Andrey A. Yurchenko, Gerome Jules-Clement, Fabrice André, A. Marabelle, A. Hollebecque, Sergey Nikolaev
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc178a768ee0b40dc9d03feaaf470b3550cae348" target='_blank'>
              Microarray analysis points to LMNB1 and JUN as potential target genes for predicting metastasis promotion by etoposide in colorectal cancer
              </a>
            </td>
          <td>
            Jiafei Liu, Hongjie Yang, Peng Li, Yuanda Zhou, Zhichun Zhang, Q. Zeng, Xipeng Zhang, Yi Sun
          </td>
          <td>2024-10-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d11267e3052403e1a0f2c358857a352b19807b" target='_blank'>
              RAS-mutant leukaemia stem cells drive clinical resistance to venetoclax.
              </a>
            </td>
          <td>
            Junya Sango, Saul Carcamo, M. Sirenko, Abhishek Maiti, Hager Mansour, Gulay Ulukaya, Lewis E Tomalin, Nataly Cruz-Rodriguez, Tiansu Wang, Malgorzata Olszewska, Emmanuel Olivier, Manon Jaud, B. Nadorp, B. Kroger, Feng Hu, Lewis Silverman, Stephen S Chung, Elvin Wagenblast, R. Chaligné, Ann-Kathrin Eisfeld, Deniz Demircioglu, D. Landau, P. Lito, E. Papaemmanuil, Courtney D. DiNardo, Dan Hasson, Marina Y Konopleva, E. Papapetrou
          </td>
          <td>2024-10-30</td>
          <td>Nature</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaf0847e7afd3b17c26839196bb1a05ffa786834" target='_blank'>
              Aneuploidy as a driver of human cancer.
              </a>
            </td>
          <td>
            Eran Sdeor, Hajime Okada, Ron Saad, Tal Ben-Yishay, Uri Ben-David
          </td>
          <td>2024-10-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite the crucial role of T cell clones in the anti-tumor activity, their characterization and association with clinical outcome following immune checkpoint inhibitors (ICI) is lacking. Here we analyzed paired single-cell RNA-sequencing/T-cell receptor sequencing of 767,606 T cells from 460 samples spanning 6 cancer types. We found a robust signature of response based on expanded CD8+ clones that differentiates between responders and non-responders. Analysis of persistent clones showed transcriptional changes that are differentially induced by therapy in the different response groups, suggesting an improved reinvigoration capacity in responding patients. Moreover, a gene trajectory analysis revealed changes in the pseudo-temporal state of de-novo clones that are associated with response to therapy. Lastly, we found that clones shared between tumor and blood are more abundant in non-responders and execute distinct transcriptional programs. Overall, our results highlight differences in clonal transcriptional states that are linked to patient response, offering valuable insights into the mechanisms driving effective anti-tumor immunity. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3504c21c4fc5c6f9ee1a5a18b7688458b8f59956" target='_blank'>
              Single-cell meta-analysis of T cells reveals clonal dynamics of response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            Ofir Shorer, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Chromosomal rearrangements and recurrent gene fusions were previously presumed to be the primary oncogenic mechanisms of hematological malignancies. However, the discovery of gene fusions in different cancers has opened new horizons to comprehensively investigate how cell type-specific fusion oncoproteins modulate signaling cascades. Prostate cancer (PCa) is a multifaceted and therapeutically challenging disease, and functional genomics have helped us develop a better understanding of the mechanisms underlying prostate carcinogenesis, castration-resistant PCa, and metastasis. Keeping in mind the fact that gene fusions have also been discovered in PCa, there has been rapid expansion in the field of molecular oncology and researchers are uncovering new facets regarding the mechanistic regulation of signaling pathways by fusion oncoproteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cde2b544758ddbdbf4e1ace08e37821205b4df7" target='_blank'>
              Mechanistic Insights and Molecular Diagnostics of TMPRSS2-ERG: Overview of the Journey from Regulation of Signaling Landscape in Fusion Positive Prostate Cancer to Appraisal as a Diagnostic Marker.
              </a>
            </td>
          <td>
            A. Farooqi, A. Turgambayeva, A. Almabayeva, M. Zhanaliyeva, Lyazat Orakbay, Zhanara Shabanbayeva, Oryngul Narmanova, Marat Syzdykbayev
          </td>
          <td>2024-09-26</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract Cancer stem cells (CSCs) are widely acknowledged as the drivers of tumor initiation, epithelial‐mesenchymal transition (EMT) progression, and metastasis. Originating from both hematologic and solid malignancies, CSCs exhibit quiescence, pluripotency, and self‐renewal akin to normal stem cells, thus orchestrating tumor heterogeneity and growth. Through a dynamic interplay with the tumor microenvironment (TME) and intricate signaling cascades, CSCs undergo transitions from differentiated cancer cells, culminating in therapy resistance and disease recurrence. This review undertakes an in‐depth analysis of the multifaceted mechanisms underlying cancer stemness and CSC‐mediated resistance to therapy. Intrinsic factors encompassing the TME, hypoxic conditions, and oxidative stress, alongside extrinsic processes such as drug efflux mechanisms, collectively contribute to therapeutic resistance. An exploration into key signaling pathways, including JAK/STAT, WNT, NOTCH, and HEDGEHOG, sheds light on their pivotal roles in sustaining CSCs phenotypes. Insights gleaned from preclinical and clinical studies hold promise in refining drug discovery efforts and optimizing therapeutic interventions, especially chimeric antigen receptor (CAR)‐T cell therapy, cytokine‐induced killer (CIK) cell therapy, natural killer (NK) cell‐mediated CSC‐targeting and others. Ultimately use of cell sorting and single cell sequencing approaches for elucidating the fundamental characteristics and resistance mechanisms inherent in CSCs will enhance our comprehension of CSC and intratumor heterogeneity, which ultimately would inform about tailored and personalized interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d170930289915a8a14d31763d5c1c28676522808" target='_blank'>
              Intricate relationship between cancer stemness, metastasis, and drug resistance
              </a>
            </td>
          <td>
            T. C. Dakal, Ravi Bhushan, Caiming Xu, Bhana Ram Gadi, S. Cameotra, Vikas Yadav, Jarek Maciaczyk, Ingo G H Schmidt-Wolf, Abhishek Kumar, Amit Sharma
          </td>
          <td>2024-09-21</td>
          <td>MedComm</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecologic malignancies, primarily due to its late-stage diagnosis and frequent relapse. Poly (ADP-ribose) polymerase (PARP) inhibitors have emerged as promising agents in the management of ovarian neoplasms, particularly among patients with BRCA1/2 mutations or homologous recombination deficiency (HRD). Despite initial therapeutic successes, the development of drug resistance poses a significant challenge, limiting the long-term efficacy of these agents. This review analyzes the molecular mechanisms underlying PARP inhibitor resistance, including genetic reversion mutations in BRCA1/2, restoration of homologous recombination repair pathways, and alterations in drug efflux mechanisms. Furthermore, we explore strategies to overcome PARP inhibitor resistance, such as combination therapies with other targeted agents, using ATR inhibitors, and optimizing the timing of maintenance chemotherapy. Current advances in identifying predictive biomarkers for PARP inhibitor response are also discussed, offering potential pathways for personalized treatment approaches. Understanding the complex interplay of resistance mechanisms and the evolving therapeutic landscape is crucial to improving outcomes in patients with ovarian cancer. This review aims to summarize current knowledge on PARP inhibitor resistance and emerging strategies to enhance their therapeutic potential in ovarian neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63ca18244a7a646c70cdc8d8fc4942c8493b1d0" target='_blank'>
              Decoding PARP Inhibitor Resistance in Ovarian Cancer: Molecular Insights and Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Amália Cinthia Meneses Rêgo, I. Araújo-Filho
          </td>
          <td>2024-11-03</td>
          <td>International Journal of Innovative Research in Medical Science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment, particularly in advanced non-small cell lung cancer (NSCLC) and muscle-invasive bladder cancer (MIBC). However, identifying reliable predictive biomarkers for ICI response remains a significant challenge. In this study, we analyzed real-world cohorts of advanced NSCLC and MIBC patients treated with ICIs as first-line therapy. Tumor samples underwent Whole Genome Sequencing (WGS) to identify specific somatic variants and assess tumor mutational burden (TMB). Additionally, mutational signature extraction and pathway enrichment analyses were performed to uncover the underlying mechanisms of ICI response. We also characterized HLA-I haplotypes and investigated LINE-1 retrotransposition. Distinct mutation patterns were identified in patients who responded to treatment, suggesting potential biomarkers for predicting ICI effectiveness. In NSCLC, tumor mutational burden (TMB) did not differ significantly between responders and non-responders, while in MIBC, higher TMB was linked to better responses. Specific mutational signatures and HLA haplotypes were associated with ICI response in both cancers. Pathway analysis showed that NSCLC responders had active inflammatory and immune pathways, while non-responders had pathways such related to FGFR3 and neural crest differentiation associated to resistance mechanisms. In MIBC, responders had alterations in DNA repair, leading to more neoantigens and a stronger ICI response. Importantly, for the first time, we found that LINE-1 activation was positively linked to ICI response, especially in MIBC. These findings reveal promising biomarkers and mechanistic insights, offering a new perspective on predicting ICI response and opening up exciting possibilities for more personalized immunotherapy strategies in NSCLC and MIBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b70492d02c2b0d60bcbf3b1913b9cfd98468b" target='_blank'>
              Genomic Insights Guiding Personalized First-Line Immunotherapy Response in Lung and Bladder Tumors
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, María Gallardo-Gómez, Ana Oitaben, Martin E. Lázaro-Quintela, Luis León, Joao M Alves, L. Juaneda-Magdalena, Carme García-Benito, I. Abdulkader, L. Muinelo, J. Paramio, Mónica Martínez-Fernández
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1167f70aefb28c2bd36dcb2ed6f87d34742f535" target='_blank'>
              Single-cell landscape of innate and acquired drug resistance in acute myeloid leukemia
              </a>
            </td>
          <td>
            R. Wegmann, Ximena Bonilla, Ruben Casanova, Stéphane Chevrier, Ricardo Coelho, Cinzia Esposito, Joanna Ficek-Pascual, Sandra Goetze, Gabriele Gut, Francis Jacob, Andrea Jacobs, Jack Kuipers, Ulrike Lischetti, J. Mena, Emanuela S Milani, M. Prummer, Jacobo Sarabia del Castillo, Franziska Singer, Sujana Sivapatham, Nora C Toussaint, Oliver Vilinovszki, M. H. Wildschut, T. Thavayogarajah, D. Malani, R. Aebersold, M. Bacac, N. Beerenwinkel, C. Beisel, B. Bodenmiller, V. Heinzelmann-Schwarz, V. Koelzer, Mitchell P Levesque, H. Moch, L. Pelkmans, Gunnar Rätsch, Markus Tolnay, Andreas Wicki, Bernd Wollscheid, M. Manz, B. Snijder, A. Theocharides
          </td>
          <td>2024-10-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="MicroRNAs are a conserved class of small, tissue-specific, non-coding RNAs that regulate gene expression to preserve cellular homeostasis. Proper miRNA expression is crucial for physiological balance because it affects numerous genetic pathways, including cell cycle control, proliferation, and apoptosis, through gene expression targeting. Deregulated miRNA expression has been implicated in several cancer types, including prostate cancer (PC), acting as tumor suppressors or oncogenes. Despite the availability of promising therapies to control tumor growth and progression, effective diagnostic and therapeutic strategies for different types of cancer are still lacking. PC continues to be a significant health challenge, particularly its castration-resistant (CRPC) form, which presents major therapeutic obstacles because of its resistance to conventional androgen deprivation treatments. This review explores miRNAs’ critical roles in gene regulation and cancer biology, as well as various miRNA delivery systems, highlighting their potential and the challenges in effectively targeting cancer cells. It aims to provide a comprehensive overview of the status of miRNA research in the fight against CRPC, summarizing miRNA-based therapies’ successes and limitations. It also highlights the promise of miRNAs as therapeutic agents for CRPC, underlining the need for further research to overcome existing challenges and move these therapies toward clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90859ee13ab8e70d2bc0534d1943825261dccf29" target='_blank'>
              MicroRNAs as Promising Therapeutic Agents Against Prostate Cancer Resistant to Castration—Where Are We Now?
              </a>
            </td>
          <td>
            Mariana Ferreira, Mariana Morais, Rui Medeiros, A. Teixeira
          </td>
          <td>2024-10-22</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ed0ba58e5446ac108522198d795c2809c0f59f" target='_blank'>
              Current Drug Resistance Mechanisms and Treatment Options in Gastrointestinal Stromal Tumors: Summary and Update
              </a>
            </td>
          <td>
            Chunxiao He, Zilong Wang, Jiaying Yu, Shuang Mao, Xi Xiang
          </td>
          <td>2024-10-23</td>
          <td>Current Treatment Options in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE
The recurrence of esophageal squamous cell carcinoma (ESCC) in radiation therapy treatment presents a complex challenge due to its resistance to radiation. However, the mechanism underlying the development of radioresistance in ESCC remains unclear. In this study, we aim to uncover the mechanisms underlying radioresistance in ESCC cells and identify potential targets for radiosensitization.


METHODS
We established two radio-resistant cell lines, TE-1R and KYSE-150R, from the parental ESCC cell lines TE-1 and KYSE-150 through fractionated irradiation. A TMT-based quantitative proteomic profiling approach was applied to identify changes in protein expression patterns. Cell Counting Kit-8, colony formation, γH2AX foci immunofluorescence and comet assays were utilized to validate our findings. The downstream effectors of the DNA repair pathway were confirmed using an HR/NHEJ reporter assay and Western blot analysis. Furthermore, we evaluated the expression of potential targets in ESCC tissues through immunohistochemistry combined with mass spectrometry.


RESULTS
Over 2,000 proteins were quantitatively identified in the ESCC cell lysates. A comparison with radio-sensitive cells revealed 61 up-regulated and 14 down-regulated proteins in the radio-resistant cells. Additionally, radiation treatment induced 24 up-regulated and 12 down-regulated proteins in the radio-sensitive ESCC cells. Among the differentially expressed proteins, S100 calcium binding protein A6 (S100A6), glutamine gamma-glutamyltransferase 2 (TGM2), glycogen phosphorylase, brain form (PYGB), and Thymosin Beta 10 (TMSB10) were selected for further validation studies as they were found to be over-expressed in the accumulated radio-resistant ESCC cells and radio-resistant cells. Importantly, high S100A6 expression showed a positive correlation with cancer recurrence in ESCC patients. Our results suggest that several key proteins, including S100A6, TGM2, and PYGB, play a role in the development of radioresistance in ESCC.


CONCLUSIONS
Our results revealed that several proteins including Protein S100-A6 (S100A6), Protein-glutamine gamma-glutamyltransferase 2 (TGM2), Glycogen phosphorylase, brain form (PYGB) were involved in radio-resistance development. These proteins could potentially serve as biomarkers for ESCC radio-resistance and as therapeutic targets to treat radio-resistant ESCC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405f18b1fbd588e250b885e57f58458a45366252" target='_blank'>
              TMT-Based Quantitative Proteomic Profiling of Human Esophageal Cancer Cells Reveals the Potential Mechanism and Potential Therapeutic Targets Associated With Radioresistance.
              </a>
            </td>
          <td>
            A. Gao, Chao He, Hengrui Chen, Qianlin Liu, Yin Chen, Jianying Sun, Chuanfeng Wu, Ya Pan, Sonia Rocha, Mu Wang, Jundong Zhou
          </td>
          <td>2024-10-07</td>
          <td>Proteomics. Clinical applications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4200779de8ffeea14186de4448271c5a8d0e9f0" target='_blank'>
              Developmental-status-aware transcriptional decomposition establishes a cell state panorama of human cancers
              </a>
            </td>
          <td>
            Yikai Luo, Han Liang
          </td>
          <td>2024-10-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of CTC-IGC was significantly associated with poorer progression-free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13c77b6b25150646e97a913d6046b3f971cd07d" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            James B. Hicks, Michael J. Schmidt, Amin Nahgdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, Isla Garraway, Michael Lewis, Ana Aparicio, A. Zurita, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah Amend
          </td>
          <td>2024-10-14</td>
          <td>Research Square</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Background: Acute myeloid leukemia (AML) is characterized by uncontrolled proliferation of myeloid progenitor cells. This study aims to elucidate genetic and transcriptomic alterations in AML through whole exome sequencing (WES) and transcriptome analysis, linking these findings with regulatory networks in hematopoiesis and immunology. Methods: We performed WES and transcriptome analysis on 10 AML samples to identify genetic mutations and expression changes. Bioinformatics tools, including differential expression analysis, gene set enrichment analysis (GSEA), and pathway analysis, were used to identify critical genes and regulatory networks. Comparative analyses with hematopoietic stem cells (HSCs) and immune cell subsets were conducted. Single-cell RNA sequencing (scRNA-seq) data were analyzed to delineate cellular heterogeneity and lineage differentiation pathways affected in AML. Results: Integrative analysis revealed key transcription factors and epigenetic regulators, such as RUNX1, FLT3, CEBPA, TP53, WT-1, GATA2, and TET2, frequently mutated or dysregulated in AML. Differential expression analysis identified distinct gene expression signatures in AML cells compared to normal HSCs, highlighting disruptions in cell cycle regulation, differentiation, and apoptosis. Altered immune regulatory pathways, including IL-7R and PD-1, indicated disruptions in immune cell activation and homeostasis. Single-cell analysis highlighted the cellular diversity and plasticity in AML. Conclusion: Our integrative bioinformatics approach links genetic and transcriptomic alterations in AML with disruptions in hematopoiesis and immune function. Key regulators and pathways identified offer potential therapeutic targets and deepen understanding of AML pathogenesis.
 Citation Format: HARSH GOEL, Pranay Tanwar. Unraveling Immune Dysregulation in Acute Myeloid Leukemia: Insights from Whole Exome and Transcriptome Analysis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A050.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e62ed562f4b43b5d97771716821ca21ed4a9b7" target='_blank'>
              Abstract A050: Unraveling Immune Dysregulation in Acute Myeloid Leukemia: Insights from Whole Exome and Transcriptome Analysis
              </a>
            </td>
          <td>
            Harsh Goel, P. Tanwar
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Towards comprehensively investigating the genotype-phenotype relationships governing the human pluripotent stem cell state, we generated an expressed genome-scale CRISPRi Perturbation Cell Atlas in KOLF2.1J human induced pluripotent stem cells (hiPSCs) mapping transcriptional and fitness phenotypes associated with 11,739 targeted genes. Using the transcriptional phenotypes, we created a minimum distortion embedding map of the pluripotent state, demonstrating rich recapitulation of protein complexes, such as strong co-clustering of MRPL, BAF, SAGA, and Ragulator family members. Additionally, we uncovered transcriptional regulators that are uncoupled from cell fitness, discovering potential novel pluripotency (JOSD1, RNF7) and metabolic factors (ZBTB41). We validated these findings via phenotypic, protein-interaction, and metabolic tracing assays. Finally, we propose a contrastive human-cell engineering framework (CHEF), a machine learning architecture that learns from perturbation cell atlases to predict perturbation recipes that achieve desired transcriptional states. Taken together, our study presents a comprehensive resource for interrogating the regulatory networks governing pluripotency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf0f66e8e51806f9ebf2ea82ca2dd611e0145032" target='_blank'>
              A Perturbation Cell Atlas of Human Induced Pluripotent Stem Cells
              </a>
            </td>
          <td>
            Sami Nourreddine, Yesh Doctor, Amir Dailamy, Antoine Forget, Yi-Hung Lee, Becky Chinn, Hammza Khaliq, Benjamin J. Polacco, Monita Muralidharan, Emily Pan, Yifan Zhang, Alina Sigaeva, Jan Niklas Hansen, Jiahao Gao, Jillian A. Parker, Kirsten Obernier, Timothy Clark, Jake Y. Chen, Christian Metallo, Emma Lundberg, T. Ideker, N. Krogan, Prashant Mali
          </td>
          <td>2024-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="As we age, many tissues become colonised by microscopic clones carrying somatic driver mutations. Some of these clones represent a first step towards cancer whereas others may contribute to ageing and other diseases. However, our understanding of the clonal landscapes of human tissues, and their impact on cancer risk, ageing and disease, remains limited due to the challenge of detecting somatic mutations present in small numbers of cells. Here, we introduce a new version of nanorate sequencing (NanoSeq), a duplex sequencing method with error rates <5 errors per billion base pairs, which is compatible with whole-exome and targeted gene sequencing. Deep sequencing of polyclonal samples with single-molecule sensitivity enables the simultaneous detection of mutations in large numbers of clones, yielding accurate somatic mutation rates, mutational signatures and driver mutation frequencies in any tissue. Applying targeted NanoSeq to 1,042 non-invasive samples of oral epithelium and 371 samples of blood from a twin cohort, we found an unprecedentedly rich landscape of selection, with 49 genes under positive selection driving clonal expansions in the oral epithelium, over 62,000 driver mutations, and evidence of negative selection in some genes. The high number of positively selected mutations in multiple genes provides high-resolution maps of selection across coding and non-coding sites, a form of in vivo saturation mutagenesis. Multivariate regression models enable mutational epidemiology studies on how carcinogenic exposures and cancer risk factors, such as age, tobacco or alcohol, alter the acquisition and selection of somatic mutations. Accurate single-molecule sequencing has the potential to unveil the polyclonal landscape of any tissue, providing a powerful tool to study early carcinogenesis, cancer prevention and the role of somatic mutations in ageing and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb1ed2352282f2639f8bc59358f511e72d35ab26" target='_blank'>
              Somatic mutation and selection at epidemiological scale
              </a>
            </td>
          <td>
            A. R. Lawson, F. Abascal, P. A. Nicola, S. Lensing, Amy L Roberts, Georgios Kalantzis, A. Baez-Ortega, Natalia Brzozowska, Julia S. El-Sayed Moustafa, Dovile Vaitkute, Belma Jakupovic, Ayrun Nessa, S. Wadge, Anna L. Paterson, Doris M. Rassl, R. E. Alcantara, L. O’Neill, S. Widaa, Siobhan Austin-Guest, M. Neville, M. Przybilla, Wei Cheng, M. Morra, Lucy Sykes, Matthew Mayho, Nicole Müller-Sienerth, Nick Williams, Diana Alexander, Luke M. R. Harvey, Thomas Clarke, Alex Byrne, Jamie R Blundell, Matthew D. Young, A. KrishnaaT., Mahbubani, K. Saeb-Parsy, Hilary C. Martin, M. R. Stratton, P. J. Campbell, R. Rahbari, K. Small, I. Martincorena
          </td>
          <td>2024-11-01</td>
          <td>None</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Malignant melanoma is an aggressive cancer, with a high risk of metastasis and mortality rates, characterized by cancer cell heterogeneity and complex tumor microenvironment (TME). Single cell biology is an ideal and powerful tool to address these features at a molecular level. However, this approach requires enzymatic cell dissociation that can influence cellular coverage. By contrast, single nucleus RNA sequencing (snRNA-seq) has substantial advantages including compatibility with frozen samples and the elimination of a dissociation-induced, transcriptional stress response. To better profile and understand the functional diversity of different cellular components in melanoma progression, we performed snRNA-seq of 16,839 nuclei obtained from tumor samples along the growth of murine syngeneic melanoma model carrying a BRAFV600E mutation and collected 9 days or 23 days after subcutaneous cell injection. We defined 11 different subtypes of functional cell clusters among malignant cells and 5 different subsets of myeloid cells that display distinct global transcriptional program and different enrichment in early or advanced stage of tumor growth, confirming that this approach was useful to accurately identify intratumor heterogeneity and dynamics during tumor evolution. The current study offers a deep insight into the biology of melanoma highlighting TME reprogramming through tumor initiation and progression, underlying further discovery of new TME biomarkers which may be potentially druggable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d37af9fbd7e323bb1c7c0e6480590eefcaf22d16" target='_blank'>
              Single-Nuclei Transcriptome Profiling Reveals Intra-Tumoral Heterogeneity and Characterizes Tumor Microenvironment Architecture in a Murine Melanoma Model
              </a>
            </td>
          <td>
            S. Parab, Valery Sarlo, S. Capellero, Luca Palmiotto, A. Bartolini, Daniela Cantarella, M. Turi, Annamaria Gullà, Elena Grassi, Chiara Lazzari, Marco Rubatto, Vanesa Gregorc, Fabrizio Carnevale-Schianca, Martina Olivero, Federico Bussolino, V. Comunanza
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric malignancy. Based on gene expression profiling, B-ALL can be classified into distinct transcriptional subtypes with differing disease outcomes. Many of these transcriptional subtypes are defined by mutations in transcription factors and chromatin-modifying enzymes, but how such diverse mutations lead to distinct transcriptional subtypes remains unclear. To illuminate the chromatin regulatory landscape in B-ALL, we analyzed 3D genome organization, open chromatin, and gene expression in 53 primary patient samples. At the level of 3D genome organization, we identified chromatin interactions that vary across transcriptional subtypes. These sites of variable 3D chromatin interactions correlate with local gene expression changes and are enriched for core drivers of B-ALL observed in genome-wide CRISPR knock-out screens. Sites of variable 3D genome interactions are frequently shared across multiple transcriptional subtypes and are enriched for open chromatin sites found in normal B-cell development but repressed in mature B-cells. Within an individual patient sample, the chromatin landscape can resemble progenitor chromatin states at some loci and mature B-cell chromatin at others, suggesting that the chromatin in B-ALL patient tumor cells is in a partially arrested immature state. By analyzing transcriptomic data from large cohorts of B-ALL patients, we identify gene expression programs that are shared across transcriptional subtypes, associated with B-cell developmental stages, and predictive of patient survival. In combination, these results show that the 3D genome organization of B-ALL reflects B-cell developmental stages and helps illustrate how B-cell developmental arrest interacts with transcriptional subtypes to drive B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85aa1e7461bf425fea64b8e5097538780b0bffaf" target='_blank'>
              Heterogeneity in chromatin structure drives core regulatory pathways in B-cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Arko Sen, Zhichao Xu, Sélène T. Tyndale, Jean Yasis, Chae Yun Cho, Rosalind Bump, Sahaana Chandran, Linda Luo, Yi Fu, Lillian Kay Petersen, Max Shokhirev, D. J. Kuo, G. McVicker, Jesse R. Dixon
          </td>
          <td>2024-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Identification of somatic driver mutations in the noncoding genome remains challenging. To comprehensively characterize noncoding driver mutations for pancreatic ductal adenocarcinoma (PDAC), we first created genome-scale maps of accessible chromatin regions (ACRs) and histone modification marks (HMMs) in pancreatic cell lines and purified pancreatic acinar and duct cells. Integration with whole-genome mutation calls from 506 PDACs revealed 314 ACRs/HMMs significantly enriched with 3,614 noncoding somatic mutations (NCSMs). Functional assessment using massively parallel reporter assays (MPRA) identified 178 NCSMs impacting reporter activity (19.45% of those tested). Focused luciferase validation confirmed negative effects on gene regulatory activity for NCSMs near CDKN2A and ZFP36L2. For the latter, CRISPR interference (CRISPRi) further identified ZFP36L2 as a target gene (16.0 - 24.0% reduced expression, P = 0.023 - 0.0047) with disrupted KLF9 binding likely mediating the effect. Our integrative approach provides a catalog of potentially functional noncoding driver mutations and nominates ZFP36L2 as a PDAC driver gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db992992dda460b3aa1dd358fc40078a768ec148" target='_blank'>
              Large-scale multi-omic analysis identifies noncoding somatic driver mutations and nominates ZFP36L2 as a driver gene for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Jun Zhong, Aidan O'Brien, Minal Patel, Daina Eiser, Michael Mobaraki, I. Collins, Li Wang, Konnie Guo, Thucnhi Truongvo, Ashley A. Jermusyk, Maura J. O’Neill, Courtney D. Dill, A. Wells, M. Leonard, J. Pippin, Struan F. A. Grant, Tongwu Zhang, T. Andresson, Katelyn E. Connelly, Jianxin Shi, H. E. Arda, Jason W. Hoskins, L. Amundadottir
          </td>
          <td>2024-09-24</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) has previously been considered as a single disease. However, it is currently characterized as a heterogeneous group of tumors that differ in histological features, genetic abnormalities, and variable clinical course. In normal cells, energy is produced by the cleavage of chemical bonds in nutrients through the oxidation of fats, proteins, or carbohydrates. Mutational alterations in genes associated with RCC, including VHL, FLCN, PTEN and SDH, lead to abnormal cellular adaptation to changes in oxygen status, iron metabolism and nutrients. The present paper reviews the known genetic abnormalities observed in RCC and their impact on metabolic alterations. Understanding the genetic and metabolic mechanisms underlying RCC is crucial for the development of effective therapies. Targeting specific genetic abnormalities or metabolic pathways represents a promising approach to the RCC treatment. In addition, studies into the metabolic basis of RCC contribute to the development of new biomarkers for early diagnosis and monitoring of the disease. Moreover, investigating the role of VHL, FLCN, PTEN, and SDH genes in the development of RCC provides valuable information on the molecular mechanisms behind the disease. As a result, it may lead to the development of new treatment strategies aimed at restoring the normal function of these genes or compensating for their abnormalities. Overall, an integrated approach to the study of RCC that considers genetic, metabolic, and clinical aspects will ensure that more effective treatments are developed and prognosis for patients with this disease are improved.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1601eb66db07531257237b1f84d2ed09ae8e4e" target='_blank'>
              Genetic Variation and Metabolic Basis of Kidney Cancer: New Opportunities for Targeted Therapy
              </a>
            </td>
          <td>
            M. Urmantsev, R. I. Tavabilov, M. R. Bakeev
          </td>
          <td>2024-10-14</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1436facb9b04a543d668f4862e82c8d1f590873" target='_blank'>
              FBXO2 as a switch guides a special fate of tumor clones evolving into a highly malignant transcriptional subtype in oral squamous cell carcinoma.
              </a>
            </td>
          <td>
            Jingyi Cheng, O. Liu, Xin Bin, Zhangui Tang
          </td>
          <td>2024-11-02</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827407d1ee135266c2bdec8a7b4b96ce85a8df0f" target='_blank'>
              A genome-wide CRISPR/Cas9 knockout screen identifies SEMA3F gene for resistance to cyclin-dependent kinase 4 and 6 inhibitors in breast cancer.
              </a>
            </td>
          <td>
            Yuko Kawai, Aiko Nagayama, Kazuhiro Miyao, Makoto Takeuchi, Takamichi Yokoe, Tomoe Kameyama, Xinyue Wang, T. Seki, Maiko Takahashi, T. Hayashida, Yuko Kitagawa
          </td>
          <td>2024-10-01</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND/AIM
Biomarkers indicating sensitivity to poly ADP-ribose polymerase (PARP) inhibitors have not yet been identified in gastric cancer. PARP inhibitors target homologous recombination deficiency (HRD); however, homologous recombination (HR) induces complex changes in gene expression, which makes it difficult to identify reliable biomarkers. In this study, we identified a multi-gene expression signature as a marker of PARP inhibitor sensitivity in gastric cancer.


MATERIALS AND METHODS
Seven gastric cancer cell lines were evaluated for susceptibility to PARP inhibitors using a growth inhibition assay. Gene expression profiling (GEP) was used to identify differentially expressed genes between PARP inhibitor-sensitive and -resistant cell lines. The resulting gene set was subjected to cluster analysis using tumor samples from 250 patients who underwent gastrectomy for primary gastroesophageal junction and gastric adenocarcinoma. HRD was defined as a truncating mutation in one or more of 22 HR-related genes and HRD scores were calculated using whole-exome sequencing data.


RESULTS
In the growth inhibition assays, the HGC27 and HSC39 cell lines were sensitive to the PARP inhibitors, olaparib, and rucaparib, and were significantly correlated with the GEP results. Seven (2.8%) patients harbored truncating mutations in HR-related genes. A gene expression signature based on the top 100 high and low differentially expressed genes between sensitive and resistant cell lines revealed a patient cluster with a high prevalence of HR-related gene mutations and high HRD scores.


CONCLUSION
The 100-gene expression signature identified in this study may serve as a valuable predictive biomarker for PARP inhibitor sensitivity in gastric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28788afa14a601dcbc3b636a50a525919ea5413e" target='_blank'>
              A Gene Expression Signature that Predicts Gastric Cancer Sensitivity to PARP Inhibitor Therapy.
              </a>
            </td>
          <td>
            Keiichi Fujiya, M. Serizawa, K. Ohshima, R. Umehara, Yuko Watanabe, T. Nagashima, E. Bando, K. Urakami, Y. Akiyama, Yasuhiro Tsubosa, Takashi Sugino, Ken Yamaguchi, Masanori Terashima
          </td>
          <td>2024-10-29</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea66538fc66995f8a97d32c51497b62496c4728f" target='_blank'>
              Single-cell dissection reveals promotive role of ENO1 in leukemia stem cell self-renewal and chemoresistance in acute myeloid leukemia
              </a>
            </td>
          <td>
            Yun Tian, Jiafan Guo, Lipeng Mao, Zhixi Chen, Xingwei Zhang, Yangqiu Li, Yikai Zhang, Xianfeng Zha, Oscar Junhong Luo
          </td>
          <td>2024-10-08</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chemical screens across hundreds of cell lines have shown that the drug sensitivities of human cancers can vary by genotype or lineage. However, most drug discovery studies have relied on culture media that poorly reflect metabolite levels in human blood. Here, we perform drug screens in traditional and Human Plasma–Like Medium (HPLM). Sets of compounds that show conditional anticancer activity span different phases of global development and include non-oncology drugs. Comparisons of the synthetic and serum-derived components that comprise typical media trace sets of conditional phenotypes to nucleotide synthesis substrates. We also characterize a unique dual mechanism for brivudine, a compound approved for antiviral use. Brivudine selectively impairs cell growth in low folate conditions by targeting two enzymes involved in one-carbon metabolism. Cataloged gene essentiality data further suggest that conditional phenotypes for other compounds are linked to off-target effects. Our findings establish general strategies for identifying drug-nutrient interactions and mechanisms of action by exploiting conditional lethality in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52075b84c4effc26f41a203f90514fe9f3c95c9f" target='_blank'>
              Conditional lethality profiling reveals anticancer mechanisms of action and drug-nutrient interactions
              </a>
            </td>
          <td>
            K. Flickinger, Kelli M. Wilson, Nicholas J. Rossiter, Andrea L. Hunger, Paresh V Vishwasrao, Tobie D. Lee, Carlos A. Mellado Fritz, Rebecca M Richards, Matthew D. Hall, Jason R. Cantor
          </td>
          <td>2024-10-04</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="


 Hotspot mutations in the IDH genes serve as tumor-initiating events that define a subset of adult diffuse glioma that can further be divided into either oligodendroglioma or astrocytoma subtypes. Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood.



 To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing.



 Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles.



 Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d952023b3d2c6da79f53a908951847332e2e66b4" target='_blank'>
              KS02.7.A LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, A. Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. J. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Z. Zhao, Q. Wang, T. Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Simple Summary Cancer is a global health problem caused by uncontrolled cell growth and changes in how cells get and use energy. This review compares cancer’s hidden metabolic changes to dark matter and dark energy in the universe, which are mysterious and often ignored. It looks at how cancer cells alter their energy use, such as through the Warburg effect and changes in fat and protein production, driven by genetic mutations. These changes help cancer grow and survive. The review suggests that targeting these metabolic pathways could be a new way to treat cancer. It calls for more research to better understand these changes and develop new therapies by focusing on the "dark energy" that fuels cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31079ef8c6b8d99084156d8e6e74525f2dc79cf1" target='_blank'>
              Insights into Metabolic Reprogramming in Tumor Evolution and Therapy
              </a>
            </td>
          <td>
            Ching-Feng Chiu, Jonathan Jaime G. Guerrero, R. Regalado, Ma. Joy B. Zamora, Jiayan Zhou, K. Notarte, Yu-Wei Lu, Paolo C. Encarnacion, Cidne Danielle D. Carles, Edrian M. Octavo, Dan Christopher I. Limbaroc, Charupong Saengboonmee, Shih-Yi Huang
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is a potent target for cancer therapy because it acts as a signaling hub, engaging in various signaling pathways and regulating a number of cellular functions like cell differentiation, proliferation, and survival. Recently, an emergency approval from the US-FDA has been issued for KRASG12C inhibitors (sotorasib and adagrasib) for metastatic lung cancer treatment. However, clinical studies on covalent KRASG12C inhibitors have rapidly confronted resistance in patients. Many methods are being assessed to overcome this resistance, along with various combinatorial clinical studies that are in process. Moreover, because KRASG12D and KRASG12V are more common than KRASG12C, focus must be placed on the therapeutic strategies for this type of patient, along with sustained efforts in research on these targets. In the present review, we try to focus on various strategies to overcome rapid resistance through the use of combinational treatments to improve the activity of KRASG12C inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a7d869ac2287e7b6ce7ff416a2192b4c748daa" target='_blank'>
              KRAS inhibitors in drug resistance and potential for combination therapy.
              </a>
            </td>
          <td>
            Lala S Rathod, N. Sakle, S. Mokale
          </td>
          <td>2024-11-06</td>
          <td>Tumori</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background Combinatory therapeutic strategy containing immunochemotherapy as part of induction therapy components is one of the current trends in the treatment of high‐risk metastatic locally advanced nasopharyngeal carcinoma (NPC). However, the mechanism underlying the heterogeneity of response at the single‐cell level has not been underexplored. Methods 18 bulks and 11 single‐cell RNA sequencing from paired before‐treatment and on‐treatment samples in patients with treatment‐naive high‐risk metastatic locally advanced NPCs were obtained. Following quality control, a total of 87 191 cells were included in the subsequence bioinformatics analysis. Results Immunochemotherapy was associated with on‐treatment tumour microenvironment (TME) remodelling, including upregulation of anti‐TMEs signatures, downregulation of pro‐TMEs signatures, reversing CD8+ T exhaustion, and repolarizing proinflammatory TAMs. For the patients achieving a complete response, the cytotoxic activity of CD8+ T cells was stimulated and more interferon‐gamma was provided, which would be the key for TAMs proinflammatory repolarization and eventually promote the CD8+ T cells maturation in turn. Among patients who did not reach complete response, differentiation and hypoxia signatures for endothelial cells were elevated after therapy. These patients exhibited higher levels of immune checkpoint genes in malignant cells at the baseline (before treatment), and decreased tumour antigen presentation activity, which may underlie the resistance mechanism to therapy. Conclusions This study pictures a map of TME modulation following immunochemotherapy‐based combination induction therapy and provides potential future approaches. Highlights Immunochemotherapy remodeled T cell phenotypes. For the patients achieving complete response, more interferon gamma was provided by CD8+ T cells after therapy, which would be the key for TAMs pro‐inflammatory repolarization and eventually promote the CD8+ T cells maturation in turns. Among patients who did not reach complete response, malignant cells exhibited higher level of immune checkpoint genes before therapy, and decreased tumor antigen presentation activity, which may underlie the resistance mechanism to therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044286a61943d724e4fdf3e451f5967f64243037" target='_blank'>
              Single‐cell transcriptome analysis reveals evolving tumour microenvironment induced by immunochemotherapy in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Yaofei Jiang, Weixin Bei, Wangzhong Li, Ying Huang, Shuiqing He, Xiaobin Zhu, Lisheng Zheng, Weixiong Xia, Shuhui Dong, Qin Liu, Chuanrun Zhang, Shuhui Lv, Changqing Xie, Yanqun Xiang, Guoying Liu
          </td>
          <td>2024-10-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas (DLBCL) are highly aggressive tumors. Their genetic complexity and heterogeneity have hampered the development of novel approaches for precision medicine. Our study aimed to develop a personalized therapy for DLBCL by utilizing the CRISPR/Cas system to induce knockouts (KO) of driver genes, thereby causing cancer cell death while minimizing side effects. We focused on OCI-LY3 cells, modeling DLBCL, and compared them with BJAB cells as controls. Analysis of whole exome sequencing revealed significant mutations in genes like PAX5, CD79B, and MYC in OCI-LY3 cells. CRISPR/Cas9-mediated KO of these genes resulted in reduced cancer cell viability. Subsequent single and dual gRNA targeting of PAX5 mutations inhibited proliferation specifically in OCI-LY3 cells. Moreover, dual gRNA targeting of PAX5 and MYC induced chromosomal rearrangements, reducing cell proliferation substantially. However, targeting single intronic mutations did not affect cell viability, highlighting the importance of disrupting protein function. Targeting multiple mutations simultaneously addresses intra-tumoral heterogeneity, and the transient delivery of CRISPR/Cas9 allows for permanent gene disruption. While challenges such as incomplete editing efficiency and delivery limitations exist, further optimization may enhance therapeutic efficacy. Overall, our findings demonstrate the efficacy of CRISPR/Cas9 in targeting oncogenic mutations, opening avenues for precision medicine in DLBCL treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6638da8f9e731128d18c1e54c6964f784156f835" target='_blank'>
              Targeting DLBCL by mutation-specific disruption of cancer-driving oncogenes
              </a>
            </td>
          <td>
            Najmeh Heshmatpour, S. M. Kazemi, Niklas D. Schmidt, Sarita R. Patnaik, Patrick Korus, Bodo G. C. Wilkens, Arturo Macarrón Palacios, Lei Peng, Luojia Yang
          </td>
          <td>2024-10-14</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b681524478cf528a97b828064b23970db22097a" target='_blank'>
              Epigenomic heterogeneity as a source of tumour evolution.
              </a>
            </td>
          <td>
            Marthe Laisné, Mathieu Lupien, C. Vallot
          </td>
          <td>2024-10-16</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="Chimeric antigen receptor (CAR) T-cells and bispecific antibodies (BsAb) targeting B-cell maturation antigen (BCMA) have significantly advanced the treatment of relapsed and refractory multiple myeloma (MM). Resistance to BCMA-targeting therapies, nonetheless, remains a significant challenge. BCMA shedding by gamma-secretase is a known resistance mechanism, and preclinical studies suggest that inhibition may improve anti-BCMA therapy. Leveraging a phase I clinical trial of the gamma-secretase inhibitor (GSI), crenigacestat, with anti-BCMA CAR T-cells (FCARH143), we utilized single-nuclei RNA sequencing and Assay for Transposase-Accessible Chromatin (ATAC) sequencing to characterize the effects of GSI on the tumor microenvironment. The most significant impacts of GSI involved effects on monocytes, which are known to promote tumor growth. In addition to observing a reduction in the frequency of non-classical monocytes, we also detected significant changes in gene expression, chromatin accessibility, and inferred cell-cell interactions following exposure to GSI. Although many genes with altered expression are associated with gamma-secretase-dependent signaling, such as Notch, other pathways were affected, indicating GSI has far-reaching effects. Finally, we detected monoallelic deletion of the BCMA locus in some patients with prior exposure to anti-BCMA therapy, which significantly correlated with reduced progression-free survival (median PFS 57 days versus 861 days). GSIs are being explored in combination with the full spectrum of BCMA targeting agents, and our results reveal widespread effects of GSI on both tumor and immune cell populations, providing insight into mechanisms for enhancing BCMA-directed therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0270c45ce4ccff6ee8256dcb44e77e0a31404f58" target='_blank'>
              Single-cell analysis of the multiple myeloma microenvironment after gamma-secretase inhibition and CAR T-cell therapy.
              </a>
            </td>
          <td>
            D. Coffey, Pinar Ataca Atilla, Erden Atilla, O. Landgren, Andrew J Cowan, Sylvain Simon, Margot Pont, Melissa L. Comstock, Geoffrey R. Hill, Stanley R. Riddell, Damian J Green
          </td>
          <td>2024-10-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Patients with triple-negative breast cancer (TNBC) have limited targeted therapeutic options. PARP inhibitors (PARPi) have demonstrated an important role for BRCA-mutant patients with early TNBC. Combination approaches with PARPi can broaden the use of PARPi to a larger cohort of TNBC patients. We selected six genes from our previously identified 63-gene signature that was associated with PARPi response. siFEN1 increased cells in G2/M arrest, DNA damage and particularly apoptosis. Targeting FEN1 with a chemical inhibitor enhanced the efficacy of PARPi in 7/10 cell lines, and synergy was demonstrated mainly in PARPi-resistant TNBC cell lines. A BRCA2-mutant cell line with acquired resistance to olaparib (HCC1395-OlaR) was strongly synergistic, with a combination index value of 0.20. The combination of PARPi and FEN1 inhibition also showed synergy in a PARPi-resistant xenograft-derived organoid model. Two mechanisms which explain the underlying efficacy are rapid progression in DNA replication fork speed and enhancement of DNA damage. The combination induced the highest fork speed (47% difference in comparison to control, P<0.0001) when FEN1 inhibition and PARPi equally increased fork speed individually in a cell line with a pre-existing increase in replication stress. The combination also increased DNA damage at lower drug concentrations, driving response in most of the synergistic cell lines. Gene expression analysis suggested that the sensitizing role of FEN1 inhibition in PARPi-resistant cell lines may be due to downregulation of pathways including mismatch repair. Therefore, targeting FEN1 shows great therapeutic potential as a targeted combination approach, particularly in the context of PARPi-resistant TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51da609111db5e5375ca86bde9a72eef75e23b2b" target='_blank'>
              Targeting FEN1 to enhance efficacy of PARP inhibition in triple-negative breast cancer
              </a>
            </td>
          <td>
            Mallory I. Frederick, Elicia Fyle, Anna Clouvel, Djihane Abdesselam, Saima Hassan
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Non-melanoma skin cancer (NMSC) mainly includes basal cell carcinoma, cutaneous squamous cell carcinoma, and Merkel cell carcinoma, showing a low mortality rate but the highest incidence worldwide. In recent decades, research has focused on understanding the pathogenesis and clinical treatments of NMSC, leading to significant advances in our knowledge of these diseases and the development of novel therapies, including immunotherapy. Nevertheless, the low to moderate objective response rate, high recurrence, and therapeutic resistance remain persistent challenges, which are partly attributable to the intratumoral heterogeneity. This heterogeneity indicates that tumor cells, immune cells, and stromal cells in the tumor microenvironment can be reshaped to a series of phenotypic and transcriptional cell states that vary in invasiveness and treatment responsiveness. The advent of single-cell RNA sequencing (scRNA-seq) has enabled the comprehensive profiling of gene expression heterogeneity at the single-cell level, which has been applied to NMSC to quantify cell compositions, define states, understand tumor evolution, and discern drug resistance. In this review, we highlight the key findings, with a focus on intratumoral heterogeneity and the mechanism of drug resistance in NMSC, as revealed by scRNA-seq. Furthermore, we propose potential avenues for future research in NMSC using scRNA-seq.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a5efda06af7a4c8d9343615ef50b870bd5ba887" target='_blank'>
              Unraveling the landscape of non-melanoma skin cancer through single-cell RNA sequencing technology
              </a>
            </td>
          <td>
            G. Yan, Xiuli Wang, Guolong Zhang
          </td>
          <td>2024-11-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="TP53, the most frequently mutated gene in human cancer, encodes a transcriptional activator that induces myriad downstream target genes. Despite the importance of p53 in tumor suppression, the specific p53 target genes important for tumor suppression remain unclear. Recent studies have identified the p53-inducible gene Zmat3 as a critical effector of tumor suppression, but many questions remain regarding its p53-dependence, activity across contexts, and mechanism of tumor suppression alone and in cooperation with other p53-inducible genes. To address these questions, we used Tuba-seqUltra somatic genome editing and tumor barcoding in a mouse lung adenocarcinoma model, combinatorial in vivo CRISPR/Cas9 screens, meta-analyses of gene expression and Cancer Dependency Map data, and integrative RNA-sequencing and shotgun proteomic analyses. We established Zmat3 as a core component of p53-mediated tumor suppression and identified Cdkn1a as the most potent cooperating p53-induced gene in tumor suppression. We discovered that ZMAT3/CDKN1A serve as near-universal effectors of p53-mediated tumor suppression that regulate cell division, migration, and extracellular matrix organization. Accordingly, combined Zmat3-Cdkn1a inactivation dramatically enhanced cell proliferation and migration compared to controls, akin to p53 inactivation. Together, our findings place ZMAT3 and CDKN1A as hubs of a p53-induced gene program that opposes tumorigenesis across various cellular and genetic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5afaeddf5354538c432a3c29c86495b8e7485c" target='_blank'>
              Integrative multiomic approaches reveal ZMAT3 and p21 as conserved hubs in the p53 tumor suppression network
              </a>
            </td>
          <td>
            A. Boutelle, Aicha R. Mabene, David Yao, Haiqing Xu, Mengxiong Wang, Yuning J. Tang, Steven S. Lopez, Sauradeep Sinha, J. Demeter, Ran Cheng, Brooks A. Benard, Liz J. Valente, Alexandros P. Drainas, Martin Fischer, Ravi Majeti, D. Petrov, Peter K. Jackson, Fan Yang, M. Winslow, Mike C Bassik, L. Attardi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Fibroblast Growth Factor Receptor 2 inhibition presents a promising therapeutic approach for restraining the growth and survival of cancer cells, particularly in breast tumours. However, the emergence of resistance to FGFR2 inhibitors like PD173074 highlights the importance of understanding the molecular mechanisms driving resistance and identifying effective therapeutic strategies. In this study, we employed temporal quantitative proteomics and phosphoproteomics, complemented by computational clustering, PTM-SEA analysis, and kinase activity prediction, to monitor the response of MFM223 triple negative breast cancer cells to FGFR2 inhibition. Strikingly, we observed a marked enrichment of ribosome biogenesis function modules immediately following treatment, a phenomenon not observed with other inhibitors. Additionally, we discovered that CX-5461, an RNA polymerase I inhibitor, synergistically enhanced the growth inhibition induced by PD173074, mechanistically attributed to its significant suppression of rDNA transcription stimulated by PD173074. Moreover, our phosphoproteomics dynamic profiling identified the clustering of kinases within MAPK and ErbB signalling pathways, indicative of their reactivation in response to FGFR2 inhibition. Experimentally validating this finding, we observed a notable rebound in phosphorylation levels of key kinases such as ERK1/2 and ErbB3, and demonstrated a substantial synergistic effect of PD173074 in combination with Trametinib, a MEK inhibitor, in suppressing cancer cell growth. Collectively, our findings provide critical insights into the network rewiring triggered by FGFR2 inhibition and offer a foundation for the rational design of combinatorial therapeutic strategies to overcome resistance mechanisms associated with FGFR2 inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc089ba7e1dffd9ff2361b087cdde2575bba907" target='_blank'>
              Ribosomal Biogenesis Hyperactivation and ErbB signalling Mediated Network Rewiring Causes Adaptive Resistance to FGFR2 Inhibition
              </a>
            </td>
          <td>
            Tao Zang, Sung-young Shin, Callan McCrimmon, Mandy Theocharous, Ralf B. Schittenhelm, Thierry Jarde, Roger J. Daly, Lan K. Nguyen
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Phenotypic diversity in tumors makes treatment difficult and leads to poor prognosis. A better understanding of what causes intratumoral heterogeneity and differences in cells that are within the same tumor will advance therapeutic options and improve patient outcomes. This review will summarize recent studies that have examined the various cancer cell phenotypes along the epithelial–mesenchymal spectrum present in tumors and how epigenetic factors and the mechanical properties of tumor tissue mediate these phenotypes. This review will also discuss the therapeutic implications of intratumoral heterogeneity, such as drug resistance and tumor recurrence, and highlight opportunities to address current limitations in cancer treatments. Abstract The tumor microenvironment comprises various cell types and experiences dynamic alterations in physical and mechanical properties as cancer progresses. Intratumoral heterogeneity is associated with poor prognosis and poses therapeutic challenges, and recent studies have begun to identify the cellular mechanisms that contribute to phenotypic diversity within tumors. This review will describe epithelial–mesenchymal (E/M) plasticity and its contribution to phenotypic heterogeneity in tumors as well as how epigenetic factors, such as histone modifications, histone modifying enzymes, DNA methylation, and chromatin remodeling, regulate and maintain E/M phenotypes. This review will also report how mechanical properties vary across tumors and regulate epigenetic modifications and E/M plasticity. Finally, it highlights how intratumoral heterogeneity impacts therapeutic efficacy and provides potential therapeutic targets to improve cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b710fff2d564553c94ffb93440614a32c0f2e22e" target='_blank'>
              Epithelial–Mesenchymal Plasticity and Epigenetic Heterogeneity in Cancer
              </a>
            </td>
          <td>
            Jessica L Sacco, Esther W. Gomez
          </td>
          <td>2024-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epithelial-mesenchymal transition (EMT) is a developmental cell state transition co-opted by cancer to drive metastasis and resistance. Stable EMT intermediate states play a particularly important role in cell state plasticity and confer metastatic potential. To explore the dynamics of EMT and identify marker genes of highly metastatic intermediate cells, we analyzed EMT across multiple tumor types and stimuli via mathematical modeling with single-cell RNA-sequencing (scRNA-seq) data. We identified pan-cancer genes consistently expressed or upregulated in EMT intermediate states, most of which were not previously annotated as markers of EMT. Using Bayesian parameter inference, we fit a simple mathematical model to scRNA-seq data, revealing tumor-specific transition rates. This mathematical model offers a framework to quantify EMT progression. A consensus analysis of differential intermediate expression, regulation, and model-derived dynamics identified marker genes associated with persistence of the intermediate EMT state. SFN and NRG1 emerged as genes with the strongest evidence for their role influencing intermediate EMT dynamics. Through analysis of an independent cell line, we verified the role of SFN as a marker intermediate EMT transition. Modeling and inference of genes associated with EMT dynamics offer means to find biomarkers and to identify therapeutic approaches to harness or reverse tumor-promoting cell state transitions driven by EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07f06e1f24747672056bab504533d2d4e89992ef" target='_blank'>
              Modeling the dynamics of EMT reveals genes associated with pan-cancer intermediate states and plasticity
              </a>
            </td>
          <td>
            MeiLu McDermott, Riddhee Mehta, E. R. Roussos Torres, Adam L. Maclean
          </td>
          <td>2024-10-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Melanoma is an aggressive cancer characterized by rapid growth, early metastasis, and poor prognosis, with resistance to current therapies being a significant issue. BRAF mutations drive uncontrolled cell division by activating the MAPK pathway. In this study, A375 and FO-1, BRAF-mutated melanoma cell lines, were treated for 4–5 months with RAF inhibitor dabrafenib or AZ628, leading to drug resistance over time. The resistant cells showed altered molecular signatures, with differences in cell cycle regulation and the propensity of cell death. Dabrafenib-resistant cells maintained high proliferative activity, while AZ628-resistant cells, especially A375 cells, exhibited slow-cycling, and a senescent-like phenotype with high susceptibility to ferroptosis, a form of cell death driven by iron. Antiretroviral drugs doravirine and cabotegravir, known for their effects on human endogenous retroviruses, were tested for their impact on these resistant melanoma cells. Both drugs reduced cell viability and colony formation in resistant cell lines. Doravirine was particularly effective in reactivating apoptosis and reducing cell growth in highly proliferative resistant cells by increasing tumor-suppressor proteins p16Ink4a and p27Kip1. These findings suggest that antiretroviral drugs can influence apoptosis and cell proliferation in RAF-inhibitor-resistant melanoma cells, offering potential therapeutic strategies for overcoming drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891570a4b6dcb28fdafca7435d6f3b715c4ac28f" target='_blank'>
              BRAF-Mutated Melanoma Cell Lines Develop Distinct Molecular Signatures After Prolonged Exposure to AZ628 or Dabrafenib: Potential Benefits of the Antiretroviral Treatments Cabotegravir or Doravirine on BRAF-Inhibitor-Resistant Cells
              </a>
            </td>
          <td>
            Valentina Zanrè, F. Bellinato, Alessia Cardile, Carlotta Passarini, Stefano Di Bella, Marta Menegazzi
          </td>
          <td>2024-11-06</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Adrenocortical carcinoma (ACC) is a rare endocrine malignancy with a poor prognosis and limited treatment options. Bulk genomic characterization of ACC has not yielded obvious therapeutic or immunotherapeutic targets, yet novel therapies are needed. We hypothesized that elucidating the intratumoral cellular heterogeneity by single nuclei RNA sequencing analyses would yield insights into potential therapeutic vulnerabilities of this disease. In addition to characterizing the immune cell and fibroblast landscape, our analyses of single nuclei gene expression profiles identified an adrenal cortex cell cluster exhibiting a program of replication stress and DNA damage response in primary and metastatic ACC. In vitro assessment of replication stress and DNA damage response using an ACC cell line and a series of newly-derived hormonally active patient-derived tumor organoids revealed ATR sensitivity. These findings provide novel mechanistic insight into ACC biology and suggest that an underlying dependency on ATR may be leveraged therapeutically in advanced ACC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1469f57c44a466002613ce12061fcafe073cc550" target='_blank'>
              Single Nuclei Sequencing Reveals Intratumoral Cellular Heterogeneity and Replication Stress in Adrenocortical Carcinoma
              </a>
            </td>
          <td>
            Liudmila V. Popova, Elizabeth A. R. Garfinkle, Daniel M. Chopyk, Jaye B. Navarro, Adithe Rivaldi, Yaoling Shu, Elena Lomonosova, J. Phay, Barbra S. Miller, Swati Sattuwar, Mary Mullen, Elaine R. Mardis, Katherine E. Miller, Priya H. Dedhia
          </td>
          <td>2024-10-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Over 90% of treatment failures in cancer therapy can be attributed to multidrug resistance (MDR), which can develop intracellularly or through various routes. Numerous pathways contribute to treatment resistance in cancer, but one of the most significant pathways is intracellular drug efflux and reduced drug concentrations within cells, which are controlled by overexpressed drug efflux pumps. As a member of the family of ABC transporter proteins, ABCC5 (ATP Binding Cassette Subfamily C Member 5) reduces the intracellular concentration of a drug and its subsequent effectiveness using an ATP-dependent method to pump the drug out of the cell. Numerous studies have demonstrated that ABCC5 is strongly linked to both poor prognosis and poor treatment response. In addition, elevated ABCC5 expression is noted in a wide variety of malignancies. Given that ABCC5 is regulated by several pathways in a broad range of cancer types, it is a prospective target for cancer treatment. This review examined the expression, structure, function, and role of ABCC5 in various cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0764c7131feb3889bb64e294bca3ca7ccb4b5c4b" target='_blank'>
              Role of ABCC5 in cancer drug resistance and its potential as a therapeutic target
              </a>
            </td>
          <td>
            Yinlong Pan, Mengmeng Wu, Huazhong Cai
          </td>
          <td>2024-11-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5aeebfc014f88f36ffbdee87dcef5097f55ec3d" target='_blank'>
              Development of a novel treatment based on PKMYT1 inhibition for cisplatin-resistant bladder cancer with miR-424-5p-dependent cyclin E1 amplification
              </a>
            </td>
          <td>
            Wataru Fukumoto, S. Okamura, M. Tamai, Junya Arima, Ichiro Kawahara, Ikumi Fukuda, Akihiko Mitsuke, Takashi Sakaguchi, S. Sugita, R. Matsushita, S. Tatarano, Yasutoshi Yamada, M. Nakagawa, H. Enokida, H. Yoshino
          </td>
          <td>2024-10-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Cellular diversity is a product of evolution acting to drive divergent regulatory programs from a common genome. Here, we use cross-cell-type epigenetic conservation to gain insight into the impact of selective constraints on genome function and phenotypic variation. By comparing chromatin accessibility across hundreds of diverse cell-types, we identify 1.4% of the human genome safeguarded by conserved domains of facultative heterochromatin, which we term regions under “cellular constraint”. We calculate single-base resolution cellular constraint scores and demonstrate robust prediction of functionally important coding and non-coding loci in a cell-type-, trait-, and disease-agnostic manner. Cellular constraint annotation enhances causal variant identification, drug discovery, and clinical diagnostic predictions. Furthermore, cell-constrained sequences share paradoxical evolutionary signals of positive and negative selection, suggesting a dynamic role in driving human adaptation. Overall, this study demonstrates that evolutionary chromatin dynamics can be leveraged to inform the translation of genetic discoveries into effective biological, therapeutic, and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0bc98afde8c329e9647a592102463c5d074b6c9" target='_blank'>
              Epigenetic constraint of cellular genomes evolutionarily links genetic variation to function
              </a>
            </td>
          <td>
            Enakshi Sinniah, Dalia Mizikovsky, Woo Jun Shim, Chris Siu Yeung Chow, Yassine Souilmi, Fei-Fei Cheng, Zhili Zeng, Jordan Laurie, Matthew Foster, Sonia Shah, Mikael Bodén, Jian Zeng, B. Llamas, N. Palpant
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b953135fad61d9fae48d6431dd2b14304a5d3" target='_blank'>
              Aberrant FAM135B attenuates the efficacy of chemotherapy in colorectal cancer by modulating SRSF1-mediated alternative splicing.
              </a>
            </td>
          <td>
            Wanmei Lin, Lijun Xu, Yaoying Li, Junze Li, Liang Zhao
          </td>
          <td>2024-10-13</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancers develop resistance to inhibitors of oncogenes mainly due to target-centric mechanisms such as mutations and splicing. While inhibitors or antagonists force targets to unnatural conformation contributing to protein instability and resistance, activating tumor suppressors may maintain the protein in an agonistic conformation to elicit sustainable growth inhibition. Due to the lack of tumor suppressor agonists, this hypothesis and the mechanisms underlying resistance are not understood. In estrogen receptor (ER)-positive breast cancer (BC), androgen receptor (AR) is a druggable tumor suppressor offering a promising avenue for this investigation. Spatial genomics suggests that the molecular portrait of AR-expressing BC cells in tumor microenvironment corresponds to better overall patient survival, clinically confirming AR's role as a tumor suppressor. Ligand activation of AR in ER-positive BC xenografts reprograms cistromes, inhibits oncogenic pathways, and promotes cellular elasticity toward a more differentiated state. Sustained AR activation results in cistrome rearrangement toward transcription factor PROP paired-like homeobox 1, transformation of AR into oncogene, and activation of the Janus kinase/signal transducer (JAK/STAT) pathway, all culminating in lineage plasticity to an aggressive resistant subtype. While the molecular profile of AR agonist-sensitive tumors corresponds to better patient survival, the profile represented in the resistant phenotype corresponds to shorter survival. Inhibition of activated oncogenes in resistant tumors reduces growth and resensitizes them to AR agonists. These findings indicate that persistent activation of a context-dependent tumor suppressor may lead to resistance through lineage plasticity-driven tumor metamorphosis. Our work provides a framework to explore the above phenomenon across multiple cancer types and underscores the importance of factoring sensitization of tumor suppressor targets while developing agonist-like drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6c58f43e2870ced4e1b54180f1d8d95c299ca56" target='_blank'>
              A molecular switch from tumor suppressor to oncogene in ER+ve breast cancer: Role of androgen receptor, JAK-STAT, and lineage plasticity.
              </a>
            </td>
          <td>
            Sarah Asemota, W. Effah, J. Holt, Daniel Johnson, Linnea Cripe, S. Ponnusamy, T. Thiyagarajan, Yekta Khosrosereshki, Dong-Jin Hwang, Yali He, B. Grimes, Martin D. Fleming, Frances E. Pritchard, Ashley Hendrix, M. Fan, Abhinav K. Jain, Hyo Young Choi, Liza Makowski, D. N. Hayes, Duane D Miller, Lawrence M. Pfeffer, Balaji Santhanam, Ramesh Narayanan
          </td>
          <td>2024-09-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Biological processes are fundamentally dynamic, yet existing methods for capturing these temporal changes are limited. We present scDynaBar, a novel approach that integrates CRISPR-Cas9 dynamic barcoding with single-cell sequencing to enable the recording of temporal cellular events. In this system, genetic barcodes accumulate mutations over a 4-week timeframe and then are sequenced together with the transcriptome of each single cell. We propose that this gradual accumulation of genetic diversity can be exploited to create a time-ordered record of cellular events. To demonstrate this, we apply the system to track the transition from a pluripotent state to a two-cell (2C)–like state in mouse embryonic stem cells (mESCs). The results provide compelling evidence for the transient nature of the 2C-like state. Additionally, our system shows consistent mutation rates across diverse cell types in a mouse gastruloid model, underscoring its robustness and versatility across various biological contexts. This technique not only improves our ability to study cellular dynamics but also creates exciting new opportunities for future applications based on recording temporal signals at the single-cell level—in other words, using dynamic barcoding as a molecular clock.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0a4bc0e689a5dd87c70b2f28e78dbbf5756625" target='_blank'>
              Using CRISPR barcoding as a molecular clock to capture dynamic processes at single-cell resolution
              </a>
            </td>
          <td>
            Yolanda Andres-Lopez, Alice Santambrogio, Ioannis Kafetzopoulos, Christopher D Todd, Celia Alda-Catalinas, Stephen J. Clark, Wolf Reik, I. Hernando-Herraez
          </td>
          <td>2024-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) often exhibit stem-like attributes that depend on an intricate stemness-promoting cellular ecosystem within their niche. The interplay between CSCs and their niche has been implicated in tumor heterogeneity and therapeutic resistance. Normal stem cells (NSCs) and CSCs share stemness features and common microenvironmental components, displaying significant phenotypic and functional plasticity. Investigating these properties across diverse organs during normal development and tumorigenesis is of paramount research interest and translational potential. Advancements in next-generation sequencing (NGS), single-cell transcriptomics, and spatial transcriptomics have ushered in a new era in cancer research, providing high-resolution and comprehensive molecular maps of diseased tissues. Various spatial technologies, with their unique ability to measure the location and molecular profile of a cell within tissue, have enabled studies on intratumoral architecture and cellular cross-talk within the specific niches. Moreover, delineation of spatial patterns for niche-specific properties such as hypoxia, glucose deprivation, and other microenvironmental remodeling are revealed through multilevel spatial sequencing. This tremendous progress in technology has also been paired with the advent of computational tools to mitigate technology-specific bottlenecks. Here we discuss how different spatial technologies are used to identify NSCs and CSCs, as well as their associated niches. Additionally, by exploring related public data sets, we review the current challenges in characterizing such niches, which are often hindered by technological limitations, and the computational solutions used to address them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548e0b0064280f307e085a25804779bf2b1c655a" target='_blank'>
              Deciphering normal and cancer stem cell niches by spatial transcriptomics: opportunities and challenges.
              </a>
            </td>
          <td>
            Hirak Sarkar, Eunmi Lee, Sereno Lopez-Darwin, Yibin Kang
          </td>
          <td>2024-11-04</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Metabolic reprogramming within tumor cells involves a shift towards either glycolysis or mitochondrial respiration, depending on the stage of tumor progression. Consequently, irreversible dysfunction of the mitochondria is considered a crucial mechanism driving the progression mechanism. While numerous mutations in mitochondrial DNA (mtDNA) have been identified across various tumor types, including glioblastoma, many studies have been limited in the scope, focusing on small segments of mtDNA or utilizing sequencing methods with restricted sensitivity. As a result, several potentially significant mtDNA mutations may have been underestimated, along with their heteroplasmic states, which play a crucial role in determining the phenotypic impact of mtDNA mutation. Although both somatic and germline mtDNA mutations have been observed in different tumor types, research on the mtDNA mutations linked to glioblastoma remains scarce. The mitochondrial genome encodes thirteen protein-coding genes that are essential for the proper functioning of respiratory complex chains. Alterations in mitochondrial function manifest at various levels, including structural and functional changes, impacting mitogenic, hemodynamic, bioenergetic, and apoptotic signaling pathways. These alterations often signify a reduced efficiency of the oxidative phosphorylation system and energy production in tumor cells. As the crucial role of mitochondrial dysfunction in glioma development grows, mitochondria have emerged as promising targets for therapy aimed at overcoming chemoresistance and eliminating cancer cells. This brief review outlines the association between mtDNA alteration and glioblastoma, as well as the current advancements in therapeutic strategies targeting mtDNA alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee99789d0cd4b8fc434072886db2b5f88f4e6d3" target='_blank'>
              Mitochondrial DNA Alterations in Glioblastoma and Current Therapeutic Targets.
              </a>
            </td>
          <td>
            Maher Kurdi, Ahmed K. Bamaga, Alaa Alkhotani, Thamer Alsharif, Ghada A Abdel-Hamid, Mohamed E Selim, Taghreed A. Alsinani, Ahmed Albeshri, Adnan Badahdah, Mazen A Basheikh, Saleh Baeesa
          </td>
          <td>2024-10-23</td>
          <td>Frontiers in bioscience</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8e33c5b973a4103516f253546861d556660e98" target='_blank'>
              Integrated immunogenomic analyses of single-cell and bulk profiling construct a T cell-related signature for predicting prognosis and treatment response in osteosarcoma
              </a>
            </td>
          <td>
            Chicheng Niu, Weiwei Wang, Qingyuan Xu, Zhao Tian, Hao Li, Qiang Ding, Liang Guo, Ping Zeng
          </td>
          <td>2024-10-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genetic mechanisms have been at the forefront of our exploration into the substrate of adaptive evolution and phenotypic diversification. However, genetic variation only accounts for a fraction of phenotypic variation. In the last decade, the significance of RNA modification mechanisms has become more apparent in the context of organismal adaptation to rapidly changing environments. RNA m6A methylation, the most abundant form of RNA modification, is emerging as a potentially significant player in various biological processes. Despite its fundamental function to regulate other major post-transcriptional mechanisms such as microRNA and alternative splicing, its role in ecology and evolution has been understudied. This review highlights the potential importance of m6A RNA methylation in ecological adaptation, emphasising the need for further research, especially in natural systems. We focus on how m6A not only affects mRNA fate but also influences miRNA-mediated gene regulation and alternative splicing, potentially contributing to organismal adaptation. The aim of this review is to synthesise key background information to enhance our understanding of m6A mechanisms driving species survival in dynamic environments and motivate future research into the dynamics of adaptive RNA methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf43da54f3833e8823168fb9ac8931ee33e9a607" target='_blank'>
              Emerging Orchestrator of Ecological Adaptation: m6A Regulation of Post-Transcriptional Mechanisms.
              </a>
            </td>
          <td>
            E. Ahi, Pooja Singh
          </td>
          <td>2024-10-05</td>
          <td>Molecular ecology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfcbcf7fbffafb54867bd0e1ae67ad8c3f72b49a" target='_blank'>
              Longitudinal analysis of genetic and epigenetic changes in human pluripotent stem cells in the landscape of culture-induced abnormality.
              </a>
            </td>
          <td>
            Yunsoo Kim, Byunghee Kang, Solbi Kweon, Sejin Oh, Dayeon Kim, Dayeon Gil, Hyeonji Lee, Jung-Hyun Kim, Ji Hyeon Ju, Tae-Young Roh, Chang Pyo Hong, Hyuk-Jin Cha
          </td>
          <td>2024-11-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d35e0bab7dec79077bc3332fba4617308d5113" target='_blank'>
              Comprehensive evaluation of immunological attributes and immunotherapy responses of positive T cell function regulators in colorectal cancer
              </a>
            </td>
          <td>
            Ke Pu, Jingyuan Gao, Yang Feng, Jian Hu, Shunli Tang, Guodong Yang, Chuan Xu
          </td>
          <td>2024-10-01</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations that occur in the cell lineages of sperm or eggs can be transmitted to offspring. In humans, positive selection of driver mutations during spermatogenesis is known to increase the birth prevalence of certain developmental disorders. Until recently, characterising the extent of this selection in sperm has been limited by the error rates of sequencing technologies. Using the duplex sequencing method NanoSeq, we sequenced 81 bulk sperm samples from individuals aged 24 to 75 years. Our findings revealed a linear accumulation of 1.67 (95% CI = 1.41-1.92) mutations per year per haploid genome, driven by two mutational signatures associated with human ageing. Deep targeted and exome NanoSeq of sperm samples identified over 35,000 germline coding mutations. We detected 40 genes (31 novel) under significant positive selection in the male germline, implicating both activating and loss-of-function mechanisms and diverse cellular pathways. Most positively selected genes are associated with developmental or cancer predisposition disorders in children, while four genes that exhibit elevated frequencies of protein-truncating variants in healthy populations. We find that positive selection during spermatogenesis drives a 2-3 fold elevated risk of known disease-causing mutations in sperm, resulting in 3-5% of sperm from middle-aged to elderly individuals carrying a pathogenic mutation across the exome. These findings shed light on the dynamics of germline mutations and highlight a broader increased disease risk for children born to fathers of advanced age than previously appreciated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c80dbe9f5280309a71b10ce99d847c79d577e695" target='_blank'>
              Sperm sequencing reveals extensive positive selection in the male germline
              </a>
            </td>
          <td>
            M. Neville, A. R. Lawson, R. Sanghvi, F. Abascal, M. H. Pham, A. Cagan, P. A. Nicola, T. Bayzetinova, A. Baez-Ortega, K. Roberts, S. Lensing, S. Widaa, R. E. Alcantara, M. P. Garcia, S. Wadge, M. R. Stratton, P. J. Campbell, K. Small, I. Martincorena, M. Hurles, R. Rahbari
          </td>
          <td>2024-11-01</td>
          <td>None</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Disruption of mechanisms that maintain genome stability is an essential factor of tumor progression. Accordingly, predisposition to the development of neoplasms is often associated with germline mutations in genes involved in DNA damage detection and repair. At the same time, impairment of DNA repair systems may be a predictor of antitumor treatment efficacy while overexpression of genes involved in DNA repair is a frequent event in various types of malignancies that can lead to development of tumor cells’ resistance to chemo- and radiotherapy. NBN (nibrin) gene encodes the subunit of the MRN complex which acts as a sensor of double-strand DNA breaks and participates in their repair by homologous recombination. Germline variants in NBN which are associated with increased risk of tumor development are generally represented by frameshift mutations that lead to the synthesis of truncated protein as well as by nonsense and some missense mutations which occur in functionally significant domains. These germline mutations result in partial loss of nibrin function and in increased frequency of spontaneous and induced chromosomal aberrations in the cells of the carriers. On the contrary, amplification of NBN locus is a predominant type of somatic mutations affecting this gene, which indicates a dual role of NBN protein in tumor progression. The results of several studies demonstrate the influence of NBN expression level and its mutational status on anti-tumor drug resistance in particular types of tumor cells and on the survival rate of patients. These data indicate that an in-depth study of different variants and their functional significance is necessary since NBN status may be essential for the choice of treatment tactics for some types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c608441ac7c7406f3e5d539484e420368a9c4c9" target='_blank'>
              Germline and somatic alterations in NBN and their putative impact on the pathogenesis of malignant neoplasms
              </a>
            </td>
          <td>
            O. Krivtsova, D. D. Ozerova, N. L. Lazarevich
          </td>
          <td>2024-10-10</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia with complex karyotype (ckAML) is characterized by high genomic complexity, including frequent TP53 mutations and chromothripsis. We hypothesized that the numerous genomic rearrangements could reposition active enhancers near proto-oncogenes, leading to their aberrant expression. We developed pyjacker, a computational tool for the detection of enhancer hijacking events, and applied it to a cohort of 39 ckAML samples. Pyjacker identified motor neuron and pancreas homeobox 1 (MNX1), a gene aberrantly expressed in 1.4% of AML patients, often as a result of del(7)(q22q36) associated with hijacking of a CDK6 enhancer. MNX1-activated cases show significant co-occurrence with BCOR mutations and a gene signature shared with t(7;12)(q36;p13) pediatric AML. We demonstrated that MNX1 is a dependency gene, as its knockdown in a xenograft model reduces leukemia cell fitness. In conclusion, enhancer hijacking is a frequent mechanism for oncogene activation in AML. Statement of significance This study examines the consequences of structural alterations and demonstrates that proto-oncogene activation by enhancer hijacking is an overlooked pathomechanism in AML. MNX1 overexpression demonstrates that deletions on chromosome 7q can not only lead to haploinsufficiency, but also to activation of oncogenes by enhancer hijacking, providing a novel leukemogenic mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3e7eaaf436e861083291af3f4d69f9c709309" target='_blank'>
              Pyjacker identifies enhancer hijacking events in acute myeloid leukemia including MNX1 activation via deletion 7q
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, A. Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, Mei-Ju May Chen, Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nádia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Matthias Schlesner, S. Vosberg, T. Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. H. Heidel, C. Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, K. Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia Erpelinck, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92346b55940f48c9910eb89c8a2199b9be5b6633" target='_blank'>
              KRAS G12C mutation in NSCLC in a small genetic center: insights into sotorasib therapy response potential
              </a>
            </td>
          <td>
            Metin Eser, Gulam Hekimoğlu, Murat Hakkı Yarar, Sezin Canbek, Melike Ozcelik
          </td>
          <td>2024-11-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a0addcf1581a1ad2c8de2a4d0d4092969062b23" target='_blank'>
              The interaction between UBR7 and PRMT5 drives PDAC resistance to gemcitabine by regulating glycolysis and immune microenvironment
              </a>
            </td>
          <td>
            Maoxiao Feng, Qinlian Jiao, Yidan Ren, Xiaoyan Liu, Zihan Gao, Zhengjun Li, Yun Wang, Miaoqing Zhao, Lei Bi
          </td>
          <td>2024-10-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Recent breakthroughs in the genetic manipulation of mitochondrial DNA (mtDNA) have enabled the precise introduction of base substitutions and the effective removal of genomes carrying harmful mutations. However, the reconstitution of mtDNA deletions responsible for severe mitochondrial myopathies and age-related diseases has not yet been achieved in human cells. Here, we developed a method to engineer specific mtDNA deletions in human cells by co-expressing end-joining (EJ) machinery and targeted endonucleases. As a proof-of-concept, we used mito-EJ and mito-ScaI to generate a panel of clonal cell lines harboring a ∼3.5 kb mtDNA deletion with the full spectrum of heteroplasmy. Investigating these isogenic cells revealed a critical threshold of ∼75% deleted genomes, beyond which cells exhibited depletion of OXPHOS proteins, severe metabolic disruption, and impaired growth in galactose-containing media. Single-cell multiomic analysis revealed two distinct patterns of nuclear gene deregulation in response to mtDNA deletion accumulation; one triggered at the deletion threshold and another progressively responding to increasing heteroplasmy. In summary, the co-expression of mito-EJ and programable nucleases provides a powerful tool to model disease-associated mtDNA deletions in different cell types. Establishing a panel of cell lines with a large-scale deletion at varying levels of heteroplasmy is a valuable resource for understanding the impact of mtDNA deletions on diseases and guiding the development of potential therapeutic strategies. Graphical Abstract Highlights Combining prokaryotic end-joining with targeted endonucleases generates specific mtDNA deletions in human cells Engineering a panel of cell lines with a large-scale deletion that spans the full spectrum of heteroplasmy 75% heteroplasmy is the threshold that triggers mitochondrial and cellular dysfunction Two distinct nuclear transcriptional programs in response to mtDNA deletions: threshold-triggered and heteroplasmy-sensing">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c60438e8f46fecefc3ea7cd6e713fab4e42d857" target='_blank'>
              Engineering mtDNA Deletions by Reconstituting End-Joining in Human Mitochondria
              </a>
            </td>
          <td>
            Yinxu Fu, Max Land, Ruobing Cui, Tamar Kavlashvili, Minsoo Kim, Toby Lieber, Keunwoo Ryu, Emily DeBitetto, I. Masilionis, Rahul Saha, Meril Takizawa, Daphne C Baker, Marco Tigano, Ed Reznik, Roshan Sharma, R. Chaligné, Craig B. Thompson, D. Pe’er, Agnel Sfeir
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cervical cancer (CC), one of the most prevalent and detrimental gynaecologic cancers, evolves through genetic and epigenetic alterations resulting in the promotion of oncogenic activity and dysfunction of tumour-suppressing mechanisms. Despite medical advancement, the prognosis for advanced-stage patients remains extremely low due to high recurrence rates and resistance to existing treatments. Thereby, the search for potential prognostic biomarkers is heightened to unravel new modalities of CC pathogenesis and to develop novel anti-cancer therapies. Epitranscriptomic modifications, reversible epigenetic RNA modifications, regulate various biological processes by deciding RNA fate to mediating RNA interactions. This narrative review provides insight into the cellular and molecular roles of endogenous RNA-editing proteins and their associated epitranscriptomic modifications, especially N6-methyladenosine (m6A), 5-methylcytosine (m5C) and N1-methyladenosine (m1A), in governing the development, progression and metastasis of CC. We discussed the in-depth epitranscriptomic mechanisms underlying the regulation of over 50 RNAs responsible for tumorigenesis, proliferation, migration, invasion, survival, autophagy, stemness, epithelial-mesenchymal transition, metabolism (glucose, lipid, glutamate and glutamine), resistance (drug and radiation), angiogenesis and recurrence of CC. Additionally, we provided a concise overview of the therapeutic potential of targeting the altered expression of endogenous RNA-editing proteins and aberrant deposition of RNA modifications on both coding and non-coding RNAs in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d247af5a7e732d73b316957acc73d51f868974f4" target='_blank'>
              Epitranscriptomics and cervical cancer: the emerging role of m6A, m5C and m1A RNA modifications
              </a>
            </td>
          <td>
            Akshat D. Modi, Hira Zahid, Ashlyn Chase Southerland, Dharmeshkumar M Modi
          </td>
          <td>2024-10-08</td>
          <td>Expert Reviews in Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd7b87d57c150bf5b9368e6def7dc5609b418052" target='_blank'>
              siRNA-based strategies to combat drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Abdulrahman Qais Khaleel, Mohammad Y Alshahrani, J. Rizaev, H. Malathi, Seema Devi, Atreyi Pramanik, Yasser Fakri Mustafa, Ahmed Hjazi, Ismoilova Muazzamxon, Beneen Husseen
          </td>
          <td>2024-10-21</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/494f9f3d6168b6080a07c2c82fa2e4922184b0d9" target='_blank'>
              miR-128-3p suppresses tumor growth and enhances chemosensitivity in tongue squamous cell carcinoma through MAP2K7 targeting.
              </a>
            </td>
          <td>
            Pooja Gupta, Bibekanand Mallick
          </td>
          <td>2024-10-30</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ed8af38ff728f1fd91a0f56c0da64fa1f39b833" target='_blank'>
              Clinical relevance of protein-truncating variants of germline DNA repair genes in prostate cancer
              </a>
            </td>
          <td>
            Yu-Hsuan Joni Shao, Cai-Sian Liao, Yu-Ching Hsu, Yu-Chiao Chiu, Tsai-Jung Lu, Yen-Chuan Ou, Tzu-Hung Hsiao
          </td>
          <td>2024-10-25</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Elucidating the mechanisms underlying cancer development and proliferation is important for the development of therapeutic methods for the complete cure of cancer. In particular, the identification of diagnostic markers for early detection and new therapeutic strategies for refractory gastrointestinal cancers are needed. Various abnormal phenomena occur in cancer cells, such as functional changes of proteins, led by genomic mutations, and changes in gene expression due to dysregulation of epigenetic regulation. This is no exception for noncoding RNA (ncRNA), which do not encode proteins. Recent reports have revealed that microRNA (miRNA), long noncoding RNA (lncRNA), and circular RNA (circRNA) are deeply involved in cancer progression. These ncRNAs have attracted attention as gene expression regulatory molecules. Recent advances in technology have made it possible not only to read DNA and RNA sequences but also to study the modification state of each base. In particular, comprehensive analysis of N6-methyladenosine (m6A) has been performed by many research groups, with multiple studies reporting that m6A modifications of specific genes are associated with cancer progression. Based on the above, this review examines how ncRNA modifications are related to cancer progression in gastrointestinal cancers such as colorectal and pancreatic cancer. We also discuss enzyme inhibitors that have been reported to have drug discovery potential targeting m6A modifications. By utilizing the new perspective of ncRNA modification, we may be able to accumulate knowledge on the molecular biology of cancer and contribute to human health through diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24e2ef3283a2f1c095d3a859b796d319aaa73ec" target='_blank'>
              Recent advances in noncoding RNA modifications of gastrointestinal cancer.
              </a>
            </td>
          <td>
            T. Hara, Sikun Meng, Yasuko Arao, Yoshiko Saito, Kana Inoue, Sarah Rennie, Ken Ofusa, Y. Doki, H. Eguchi, Toru Kitagawa, Hideshi Ishii
          </td>
          <td>2024-11-01</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ed2057a00d451d2750eb4c6690ca3020a9e3a6" target='_blank'>
              The prominent pervasive oncogenic role and tissue specific permissiveness of RAS gene mutations
              </a>
            </td>
          <td>
            Ming Yi, Daniel Soppet, Frank McCormick, D. Nissley
          </td>
          <td>2024-10-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Emerging evidence suggests that cancer stemness plays a crucial role in tumor progression, metastasis, and chemoresistance. Upon exposure to internal or external stress, ribosomes stand sentinel and facilitate diverse biological processes, including oncological responses. In the present study, ribosome-inactivating stress (RIS) was evaluated for its modulation of cancer cell stemness as a pivotal factor of tumor cell reprogramming. Based on the concept of stress-responsive cancer cell stemness, we addressed human intestinal cancer cell line-based off-the-shelf spheroid cultures. Intestinal cancer cell line-based spheroids exhibited heightened levels of CD44+CD133+ cancer stemness, which was improved by chemical-induced RIS. Further evaluations revealed the potential of these stress-imprinted spheroids as a platform for chemoresistance screening. Compared to adherent cells, stemness-improved spheroid cultures displayed reduced apoptosis in response to 5-fluorouracil (5-FU), a frontline chemotherapeutic agent against colorectal cancer. Moreover, serial subcultures with repeated RIS exposure maintained and even enhanced cancer stemness and chemoresistance patterns. In particular, isolated CD44+CD133+ cancer stem cells exhibited higher chemoresistance compared to unsorted cells. To elucidate the mechanisms underlying RIS-induced stemness, RNA-seq analysis identified Wnt signaling pathways and stemness-associated signals as notable features in spheroids exposed to RIS. Loss-of-function studies targeting connective tissue growth factor (CTGF), a negative regulator of Wnt signaling, revealed that CTGF-deficient spheroids exhibited improved cancer stemness and resistance to 5-FU, with RIS further enhancing these effects. In conclusion, this proof-of-concept study demonstrates the feasibility of leveraging stress-responsive cancer stemness for the development of spheroid-based platforms for chemoresistance evaluation and elucidation of pathophysiological processes of colorectal tumorigenesis under environmental stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca77eecde88a9c437174d5b9fe9de069a499f734" target='_blank'>
              Leveraging Xenobiotic-Responsive Cancer Stemness in Cell Line-Based Tumoroids for Evaluating Chemoresistance: A Proof-of-Concept Study on Environmental Susceptibility
              </a>
            </td>
          <td>
            Ki-Hyung Kim, Seung Joon Lee, Juil Kim, Yuseok Moon
          </td>
          <td>2024-10-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Vitamin C plays a significant role in various physiological functions. Humans depend on external sources of vitamin C due to the loss of the L-gulono-γ-lactone oxidase (GULO) gene that contributes to the synthesis of vitamin C. During the evolutionary loss of the GULO gene, physical, chemical, and biological factors were different from the present environmental settings. Besides the evolutionary genetic loss of the GULO gene, there is a gap in the insightful discussion on the potential implications of the non-functional GULO gene towards the predisposition of humans to cancer that faces hostile and carcinogenic environments. Various methods by which vitamin C modulates cellular processes related to cancer, including DNA repair, epigenetic changes, and redox balance, are discussed. Furthermore, we present experimental and clinical evidence indicating that vitamin C deficiency promotes tumor growth, metastasis, and therapy resistance, emphasizing its potential as a cancer phenotypic modulator. Therapeutic implications of restoring vitamin C levels in cancer treatment range from improving the efficacy of conventional medicines to exploiting metabolic vulnerabilities in tumors. The relevance of assessing vitamin C status in cancer patients and the basis for additional research into vitamin C supplementation as an adjuvant therapy is emphasized. This paper presents a comprehensive overview of the implications associated with the functional deficiency of the GULO gene in human subjects exhibiting diverse tumor hallmarks, encompassing ECM remodeling, hypoxia, epigenetic reprogramming, oxidative stress, and drug responsiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b5b27fd6a0a394480679fcc72328139bc69dd8" target='_blank'>
              Potentiation of Tumor Hallmarks by the Loss of GULO, a Vitamin C Biosynthesis Gene in Humans.
              </a>
            </td>
          <td>
            Gopinath Sekar, Anjali Bahot, Mahima Bansode, Anushka Phadnis, S. Sarode, Nilesh Kumar Sharma
          </td>
          <td>2024-10-14</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is characterized by cellular and genetic heterogeneity, which correlates with clinical course. Although single-cell RNA sequencing (scRNA-seq) reflect this diversity to some extent, the low sample numbers in individual studies limit the analytic potential of comparisons of specific patient groups. Results We performed large scale integration of published scRNA-seq datasets to create a unique single-cell transcriptomic atlas for AML (AML scAtlas), totaling 748,679 cells, from 159 AML patients and 44 healthy donors from 20 different studies. This is the largest single-cell data resource for AML to our knowledge, publicly available at https://cellxgene.bmh.manchester.ac.uk/AML/. This AML scAtlas, allowed exploration of the clinical importance of age in t(8;21) AML to an unprecedented level, given the in-utero origin of pediatric disease. We uncovered age-associated gene regulatory network (GRN) signatures, which we validated using bulk RNA sequencing data to delineate distinct groups with divergent biological characteristics. Furthermore, using an additional multiomic dataset (scRNA-seq and scATAC-seq), we created a de-noised GRN reflecting the previously defined age-related signatures. Conclusions Applying integrated data analysis of the AML scAtlas, we reveal age-dependent gene regulation in t(8;21), perhaps reflecting immature/fetal HSC origin in prenatal origin disease vs postnatal origin. Our analysis revealed that BCLAF1, which is particularly enriched in t(8;21) pediatric AML of inferred in-utero origin, is a promising prognostic indicator. The AML scAtlas provides a powerful resource to investigate molecular mechanisms underlying different AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e4db9179e7bc109d5a55d66fc1882eee7ad503" target='_blank'>
              Single-Cell Atlas of AML Reveals Age-Related Gene Regulatory Networks in t(8;21) AML
              </a>
            </td>
          <td>
            Jessica Whittle, Stefan Meyer, G. Lacaud, Syed Murtuza Baker, Mudassar Iqbal
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The discovery of growth factors and their involvement in cancer represents the foundation of precision oncology. The preclinical and clinical development of agents targeting epidermal growth factor receptor (EGFR) in colorectal cancer (CRC) were accompanied by big hype and hopes, though the clinical testing of such agents clashed with intrinsic and acquired resistance, greatly limiting their therapeutic value. However, a better understanding of the biology of the EGFR signaling pathway in CRC, coupled with the development of liquid biopsy methodologies to study cancer evolution in real time, fostered the clinical refinement of anti-EGFR treatment in CRC. Such a workflow, based on the co-evolution of biology knowledge and clinical development, allowed to couple the discovery of relevant therapy resistance mechanisms to the development of strategies to bypass this resistance. A broader application of this paradigm could prove successful and create an effective shortcut between the bench and the bedside for treatment strategies other than targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a608cc6a9618f352b70a4c24f44bb5a5875ce0fc" target='_blank'>
              When molecular biology transforms clinical oncology: the EGFR journey in colorectal cancer.
              </a>
            </td>
          <td>
            P. P. Vitiello, N. Saoudi González, A. Bardelli
          </td>
          <td>2024-10-29</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioma remains a significant clinical challenge and poses a dismal patient prognosis. This study focused on the long noncoding ribonucleic acid growth arrest-specific transcript 5 (GAS5) and explored the role of GAS5 and GAS5-related m6A genes in glioma. We explored the mechanisms of GAS5 expression in glioma using bioinformatic analysis based on glioma data from the Cancer Genome Atlas, GSE1142, and Chinese Glioma Genome Atlas databases. Kaplan–Meier curve analysis, nomogram construction, immune cell infiltration, drug sensitivity, mutations, and pathway analyses were performed to determine the GAS5 mechanism in glioma. Spearman correlation and weighted gene co-expression analyses were used to identify the GAS5-related m6A gene. Furthermore, we explored the correlation between GAS5, GAS5-related m6A gene, and clinical traits using analysis of variance. The Kaplan–Meier curve analysis suggested that patients with high expressions of GAS5 had better survival. The nomogram constructed indicated that GAS5 was an independent prognostic factor. Furthermore, GAS5 significantly correlated with plasma cells. GAS5 expression was significantly associated with biological processes, including oxidative phosphorylation, proteasome, and ribosome mitotic spindle. GAS5 expression was associated with sensitivity to erlotinib and gemcitabine. Differentially expressed GAS5 was significant in histology (P = 2.8e−09), grade (P = 3.7e−05), isocitrate dehydrogenase (IDH) mutation (P = 3.4e−17), 1p/19q co-deletion (Codel) status (P = 1.7e−08), and IDH mutation status and 1p/19q Codel status (P = 2.9e−18). Heterogeneous nuclear ribonucleoproteins C1/C2 (HNRNPC) gene was significant in IDH mutation (P = .008) and IDH mutation status and 1p/19q Codel status (P = 2.1e−05). GAS5 and HNRNPC expressions reflected the malignant grade of glioma and are associated with prognosis. The abnormal expression of GAS5 could be an important biomarker for guiding erlotinib and gemcitabine use in glioma treatment. GAS5 and heterogeneous nuclear ribonucleoproteins C1/C2 are potential diagnostic and prognostic markers for glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617fae713c8e7dfe9f42028feb516bfb3a3360fb" target='_blank'>
              Characterization of growth arrest-specific transcript 5 and growth arrest-specific transcript 5-related m6A gene signature in glioma: An observational study
              </a>
            </td>
          <td>
            Yutian Liao, Li Du, Eryue Qiu, Yiqian Zeng
          </td>
          <td>2024-09-27</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abbdc3050d16ddd79081ef3d007abf9825dc94d" target='_blank'>
              Leveraging CRISPR gene editing technology to optimize the efficacy, safety and accessibility of CAR T-cell therapy.
              </a>
            </td>
          <td>
            Tao Lei, Yazhuo Wang, Yuchen Zhang, Yufei Yang, Jiaying Cao, Jiansong Huang, Jiali Chen, Huajing Chen, Jiayi Zhang, Luzheng Wang, Xinjie Xu, Robert Peter Gale, Liang Wang
          </td>
          <td>2024-10-25</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetic dysregulation is a hallmark of many human malignancies, with DNA methylation being a primary mechanism influencing gene expression and maintaining genomic stability. Genome-wide hypomethylation, characteristic of many cancers, is partly attributed to the demethylation of repetitive elements, including LINE-1, a prevalent non-LTR retrotransposon. The methylation status of LINE-1 is closely associated with overall genomic methylation levels in tumors. cfDNA comprises extracellular DNA fragments found in bodily fluids such as plasma, serum, and urine, offering a dynamic snapshot of the genetic and epigenetic landscape of tumors. This real-time sampling provides a minimally invasive avenue for cancer diagnostics, prognostics, and monitoring. The methylation status of LINE-1 in cfDNA has emerged as a promising biomarker, with several studies highlighting its potential in diagnosing and predicting outcomes in cancer patients. Recent research also suggests that cfDNA-based LINE-1 methylation analysis could serve as a valuable tool in evaluating the efficacy of cancer therapies, including immunotherapy. The growing clinical significance of cfDNA calls for a closer examination of its components, particularly repetitive elements like LINE-1. Despite their importance, the role of LINE-1 elements in cfDNA has not been thoroughly gauged. We aim to address this gap by reviewing the current literature on LINE-1 cfDNA assays, focusing on their potential applications in diagnostics and disease monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/341df9b432e212e6ae303b9a96f7ea2e9a0b4140" target='_blank'>
              LINE-1 cfDNA Methylation as an Emerging Biomarker in Solid Cancers
              </a>
            </td>
          <td>
            U. Gezer, E. Özgür, E. E. Yörüker, Eleni Polatoglou, S. Holdenrieder, A. Bronkhorst
          </td>
          <td>2024-11-05</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Direct lineage reprogramming into dopaminergic (DA) neurons holds great promise for the more effective production of DA neurons, offering potential therapeutic benefits for conditions such as Parkinson's disease. However, the reprogramming pathway for fully reprogrammed DA neurons remains largely unclear, resulting in immature and dead‐end states with low efficiency. In this study, using single‐cell RNA sequencing, the trajectory of reprogramming DA neurons at multiple time points, identifying a continuous pathway for their reprogramming is analyzed. It is identified that intermediate cell populations are crucial for resetting host cell fate during early DA neuronal reprogramming. Further, longitudinal dissection uncovered two distinct trajectories: one leading to successful reprogramming and the other to a dead end. Notably, Arid4b, a histone modifier, as a crucial regulator at this branch point, essential for the successful trajectory and acquisition of mature dopaminergic neuronal identity is identified. Consistently, overexpressing Arid4b in the DA neuronal reprogramming process increases the yield of iDA neurons and effectively reverses the disease phenotypes observed in the PD mouse brain. Thus, gaining insights into the cellular trajectory holds significant importance for devising regenerative medicine strategies, particularly in the context of addressing neurodegenerative disorders like Parkinson's disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bfa61a76443945c179bc5c94c72de94ea5c5df" target='_blank'>
              Epigenetic Dynamics in Reprogramming to Dopaminergic Neurons for Parkinson's Disease
              </a>
            </td>
          <td>
            Byounggook Cho, Junyeop Kim, Sumin Kim, Saemin An, Yerim Hwang, Yunkyung Kim, Daeyeol Kwon, Jongpil Kim
          </td>
          <td>2024-09-16</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma (DLBCL) is an aggressive non-Hodgkin lymphoma and the most frequently diagnosed hematologic malignancy in the United States. DLBCL exhibits significant molecular and clinical heterogeneity, and at least a third of patients are left uncured with standard frontline chemoimmunotherapy. As such, there is a critical need to identify novel targeted therapies to improve outcomes. We conducted a phenotypic screen of kinase inhibitors against DLBCL cell lines and non-malignant controls. We identified the cyclin G-associated kinase (GAK) as a tumor-selective, readily druggable target whose inhibition killed DLBCL cell lines, while sparing non-malignant blood cells. Upon investigation of GAK’s cellular function, we discovered that inhibition results in G2/M-phase cell cycle arrest. Immunofluorescent confocal microscopy revealed significant chromosome misalignment and spindle distortion in DLBCL cells following GAK-inhibition, disrupting progression through mitosis. Analysis of RNA-seq data from clinical samples showed increased GAK expression associates strongly with RB1 deficiency in DLBCL cases, suggesting dependency on GAK for proper mitotic progression linked to retinoblastoma associated protein (RB) loss of function, a common DLBCL driver. In cell-cycle analyses and under microscopy, RB-deficient DLBCL cells treated with an exquisitely selective GAK inhibitor showed complete arrest at G2/M, pronounced distortion of mitotic spindles, and widespread chromosomal damage. Finally, in vivo studies of DLBCL xenograft-bearing NSG mice achieved a dramatic tumor-burden reduction in response to targeted GAK inhibition. These results reveal a novel cell cycle kinase suitable for therapeutic exploitation in DLBCL patients and linked to the common, undruggable biomarker of RB loss of function. KEY POINTS Phenotypic screening and target deconvolution revealed GAK as a novel therapeutic target in diffuse large B-cell lymphoma GAK kinase activity is a key requirement for successful mitosis in DLBCL linked to deficiency of the retinoblastoma (RB) tumor supressor">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dbd2dded9e4b2613776188deface431c8db78e0" target='_blank'>
              The cyclin-G associated kinase (GAK) is a novel mitotic kinase and therapeutic target in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            O. Lightfuss, Preet Kumar, Lingxiao Li, Toral Shastri, Austin D. Newsam, Paola Manara, A. Y. Alaoui, Diego E. Hernandez Parets, Pierre A. Younes, Dhanvantri Chahar, Artavazd Arumov, C. Coughlin, Indigo P. Williams, Nikolai Fattakhov, Alicia Bilbao Martinez, O. Landgren, Stephan C. Schürer, Vance Lemmon, Francesco Maura, Daniel Bilbao, Yangbo Feng, Hassan Al-Ali, Jonathan H. Schatz
          </td>
          <td>2024-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cells are heterogeneous, each harboring distinct molecular aberrations and are dependent on different genes for their survival and proliferation. While successful targeted therapies have been developed based on driver DNA mutations, many patient tumors lack druggable mutations and have limited treatment options. Here, we hypothesize that new precision oncology targets may be identified through “expression-driven dependency”, whereby cancer cells with high expression of a targeted gene are more vulnerable to the knockout of that gene. We introduce a Bayesian approach, BEACON, to identify such targets by jointly analyzing global transcriptomic and proteomic profiles with genetic dependency data of cancer cell lines across 17 tissue lineages. BEACON identifies known druggable genes, e.g., BCL2, ERBB2, EGFR, ESR1, MYC, while revealing new targets confirmed by both mRNA- and protein-expression driven dependency. Notably, the identified genes show an overall 3.8-fold enrichment for approved drug targets and enrich for druggable oncology targets by 7 to 10-fold. We experimentally validate that the depletion of GRHL2, TP63, and PAX5 effectively reduce tumor cell growth and survival in their dependent cells. Overall, we present the catalog of express-driven dependency targets as a resource for identifying novel therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/328c4c5a157ff3ba7e44391b07b799fdfa4b7395" target='_blank'>
              Expression-Driven Genetic Dependency Reveals Targets for Precision Medicine
              </a>
            </td>
          <td>
            Abdulkadir Elmas, Hillary M. Layden, Jacob D. Ellis, Luke N. Bartlett, Xian Zhao, Reika Kawabata-Iwakawa, Hideru Obinata, Scott W. Hiebert, Kuan-lin Huang
          </td>
          <td>2024-10-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0914eac9368db3a26273ef88f18ab51249d37f" target='_blank'>
              Advances in targeting cancer epigenetics using CRISPR-dCas9 technology: A comprehensive review and future prospects.
              </a>
            </td>
          <td>
            Jeevitha Rajanathadurai, Elumalai Perumal, Jospin Sindya
          </td>
          <td>2024-09-18</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Nearly a quarter century ago, Hanahan and Weinberg conceived six unifying principles explaining how normal cells transform into malignant tumors. Their provisional set of biological capabilities acquired during tumor development  cancer hallmarks  would evolve to fourteen tenets as knowledge of cancer genomes, molecular mechanisms, and the tumor microenvironment expanded, most recently adding four emerging enabling characteristics: phenotypic plasticity, epigenetic reprogramming, polymorphic microbiomes, and senescent cells. AKT kinases are critical signaling molecules that regulate cellular physiology upon receptor tyrosine kinases and phosphatidylinositol 3-kinase activation. The complex branching of the AKT signaling network involves several critical downstream nodes that significantly magnify its functional impact, such that nearly every organ system and cell in the body may be affected by AKT activity. Conversely, tumor intrinsic dysregulation of AKT can have numerous adverse cellular and pathological ramifications, particularly in oncogenesis, as multiple tumor suppressors and oncogenic proteins regulate AKT signaling. Herein, we review the mounting evidence implicating the AKT pathway in the aggregate of currently recognized hallmarks of cancer underlying the complexities of human malignant diseases. The challenges, recent successes, and likely areas for exciting future advances in targeting this complex pathway are also discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc996efb642ec81ce6e4555cd7fa9142e0c17d3" target='_blank'>
              AKT and the Hallmarks of Cancer.
              </a>
            </td>
          <td>
            Eleonora Sementino, Dalal Hassan, Alfonso Bellacosa, Joseph R Testa
          </td>
          <td>2024-10-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0fe672a884b6c86c84e083c9b2ab93ac24f65f" target='_blank'>
              Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements
              </a>
            </td>
          <td>
            Florence M. Chardon, Troy A. McDiarmid, Nicholas F. Page, R. Daza, Beth K. Martin, Silvia Domcke, Samuel Regalado, Jean-Benoît Lalanne, Diego Calderon, Xiaoyi Li, L. Starita, Stephan J. Sanders, N. Ahituv, J. Shendure
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4229ea75fc458fb4ee692eac3c38175128262ee2" target='_blank'>
              Effect of fibroblast heterogeneity on prognosis and drug resistance in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Tingjie Wang, Lingxi Tian, Bing Wei, Jun Li, Cuiyun Zhang, Ruitao Long, Xiaofei Zhu, Yougai Zhang, Bo Wang, Guangbo Tang, Jun Yang, Yongjun Guo
          </td>
          <td>2024-11-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c899c38678aa8e86ec7ec4d5de561bdad4078c7d" target='_blank'>
              CDK4/6 inhibition initiates cell cycle arrest by nuclear translocation of RB and induces a multistep molecular response
              </a>
            </td>
          <td>
            Ting Hong, Anna C Hogger, Dongbiao Wang, Qi Pan, Julie Gansel, T. Engleitner, R. Öllinger, Jürgen E Gschwend, R. Rad, R. Nawroth
          </td>
          <td>2024-10-26</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8944ad87f93a33edd3ce6c09cbae53782b87ab0f" target='_blank'>
              Single-cell transcriptomics of pediatric Burkitt lymphoma reveals intra-tumor heterogeneity and markers of therapy resistance.
              </a>
            </td>
          <td>
            Clarissa Corinaldesi, A. Holmes, Gaia Martire, A. Tosato, Domenico Rizzato, F. Lovisa, I. Gallingani, Qiong Shen, Lavinia Ferrone, Marian Harris, Kimberly Davies, Luca Molinaro, Umberto Mortara, Angelo Paolo dei Tos, Kenneth Ofori, E. d'Amore, Roberto Chiarle, Bo Ngan, E. Carraro, Marta Pillon, Shafinaz Hussein, Govind Bhagat, Marco Pizzi, L. Mussolin, Katia Basso
          </td>
          <td>2024-10-18</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Introduction: The worldwide challenge presented by human immunodeficiency virus type 1, affecting 39 million individuals, endures despite the highly active antiretroviral therapy. Limitations of it, including insufficiently addressing viral reservoirs leading to prolonged drug toxicity, resistance concerns, and economic burdens, have fueled exploration into alternative approaches. Gene editing and gene repression techniques have emerged as potential solutions. Objective: Conduct a comprehensive analysis of recent research following Preferred Reporting Items for Systematic Reviews and Meta-Analyses guidelines, focusing on genetic modification and gene repression in HIV-1-infected samples. Methods: A meticulous search identified 413 articles, of which 76 met strict criteria. Inclusion criteria involved studies on gene editing and HIV-1 expression (2019–2023), emphasizing CCR5 repression. Exclusions targeted weak evidence, lacking practical efficacy, and non-relevance to HIV-1 or genetic tools. Duplicates and non-open-access articles were also excluded. Results: The identified strategies show promise in their ability to mitigate the chronicity of the infection. However, challenges such as off-target effects, ethical considerations surrounding in vivo, ex vivo, and germline editing, and variable efficacy across studies are evident. The imperative for continued evaluation, refinement of techniques, and the development of effective administration methods is underscored. Conclusions: This topic review provides a nuanced exploration of recent advancements in gene editing and gene repression techniques for managing HIV-1. While the findings suggest promising avenues, the identified challenges necessitate ongoing research and innovation in order to harness the full potential of these strategies in the quest for an effective HIV-1 solution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53fae1f8a23cc0b10ee128f34eb4680a5249d869" target='_blank'>
              Novel Approaches in HIV-1 Cure: Genetic Editing and Gene Repression Techniques. Topic Review
              </a>
            </td>
          <td>
            Nicolas Alejandro Castilla Salcedo
          </td>
          <td>2024-10-23</td>
          <td>Salud &amp; Sociedad Uptc</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recent advancements in glioma treatment are largely driven by the identification of genetic alterations, which enhance diagnostic precision and prognostic assessments, and unveil potential therapeutic targets. TERT promoter mutations, in particular, are associated with a poorer prognosis and aggressive clinical behavior.This study explores the genetic interplay between TERT and other genes (ntrk, pdl1, alk, idh, p53, egfr, her2) in brain tumors through an integrative literature review. This method synthesizes evidence from selected articles spanning 2014 to 2023.The review identified 65 articles based on defined inclusion criteria, out of which 14 were analyzed in depth. Findings reveal that TERT, TP53, and IDH1 are the most frequently mutated genes in gliomas. The prognosis of glioma patients can be refined through the combined analysis of IDH and TERT mutations. Additionally, PD‐L1 expression levels are associated with prognosis and may influence treatment responses, particularly, in immunotherapy.The study underscores the importance of molecular diagnostics, such as Next‐Generation Sequencing (NGS), in detecting key genetic mutations. These advancements have paved the way for new therapeutic strategies and better patient outcomes. The findings highlight the crucial role of genetic markers in glioma treatment and prognosis, advocating for continued research to enhance clinical applications and patient care.The use of NGS is indispensable in identifying biomarkers associated with mutations in the TERT gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c51adbeed476d28eee62551a9aa91bd193de639" target='_blank'>
              TERT Gene Mutation in Gliomas Cross‐Linked With (NTRK, PDL1, ALK, IDH, P53, EGFR, HER2): A Integrative Review TERT Gene Mutation in Gliomas
              </a>
            </td>
          <td>
            Gunter Gerson Santos, Guilherme Nobre Nogueira, Iasmin Maria Rodrigues Saldanha, Ana Gabriela Ponte Farias, Cauan Miranda Mateus, Osvaldo Mariano Viana Neto, Maria Jânia Teixeira
          </td>
          <td>2024-11-07</td>
          <td>Journal of Surgical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49efc881c885ab1ba2d61433704ad1711808a206" target='_blank'>
              Dysregulation of R-loop homeostasis shapes the immunosuppressive microenvironment and induces malignant progression in melanoma.
              </a>
            </td>
          <td>
            Ouyang Yan, Yan Gu, Shuqin Li, Xianpeng Wei, Yang Liu, Zejun Wang, Fuzhou Tang, Shichao Zhang
          </td>
          <td>2024-11-02</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99b084e2208e5b0e67e8e029bea90456f03863d" target='_blank'>
              A multiplex single-cell RNA-Seq pharmacotranscriptomics pipeline for drug discovery.
              </a>
            </td>
          <td>
            Alice Dini, Harlan Barker, Emilia Piki, Subodh Sharma, Juuli Raivola, A. Murumägi, Daniela Ungureanu
          </td>
          <td>2024-10-31</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="In recent years, large-scale sequencing efforts have identified targetable driver mutations in haematopoietic stem cells. These efforts have led to the development and approval of nine novel agents for relapsed or refractory acute myelogenous leukaemia (R/R AML). However, despite an expansion in targeted therapies, achieving a durable remission in AML and high-risk myelodysplastic syndrome (HR-MDS) remains a significant challenge, and there is an urgent need for new effective treatments. Modulation of aberrant RNA splicing has emerged as a novel therapeutic approach in myeloid diseases. Aberrant splicing drives dysregulated gene expression that promotes tumourigenesis through increased proliferation and metastatic potential, immune evasion, decreased apoptosis, and chemotherapy resistance. Mutations in spliceosomal components have been identified in numerous cancer subtypes, with mutations in RNA binding proteins SF3B1, SRSF2, U2AF1, and ZRSR2 occurring frequently in AML and in up to 60% of patients with MDS, as well as in chronic myelomonocytic leukaemia and in 10% of patients with chronic lymphocytic leukaemia. In this review, we explore therapeutic strategies targeting aberrant splicing and the potential of these approaches to drive clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17b179523c7f6d0d667d6bb250cabd47b56f2a64" target='_blank'>
              Therapeutic strategies targeting aberrant RNA splicing in myeloid malignancies.
              </a>
            </td>
          <td>
            Leora S Boussi, Jeetayu Biswas, Omar Abdel-Wahab, E. Stein
          </td>
          <td>2024-10-15</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Mutations are not always phenotypically active or show different effects in different individuals. While the mechanisms underlying this variable relationship between mutations and phenotypes are largely elusive, some specific genes may influence the phenotypic effects of cryptic variation. We employ the toolbox of Saccharomyces cerevisiae to perform a genome-wide screen aimed at identifying these so-called genetic buffer genes. Measuring the fitness of 1.8 million mutated strains identified a small set of evolutionary conserved buffer genes involved in protein folding and chromatin organization, including GIM3, SSA2, HOG1 and FKH2. Deletion of these genes increases the fitness effect of de novo mutations as well as standing genetic variation, with some mutations becoming adaptive. Moreover, losing a buffer gene results in a decline of standing genetic variation. Together, these results reveal a set of conserved genes that influence the phenotypic outcome of mutations and play a role in complex genetics and incomplete penetrance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23f77a1888d72233722ce01119fc36d042aeaae" target='_blank'>
              Genes involved in protein folding and chromatin organization buffer genetic variation
              </a>
            </td>
          <td>
            Jens Frickel, Mohammed T Tawfeeq, Emily Baker, Sara Baco, Jonas Rombout, K. Voordeckers, Daniel F. Jarosz, Sibylle C. Vonesch, K. Verstrepen
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Simple Summary In this study, we applied translational informatics for intelligent medicine of acute myeloid leukemia, a type of cancer characterized by disease relapses even after seemingly successful treatments. Treatment failure is associated, at least in part, with the fact that targeting individual proteins often promotes rewiring of relevant networks and re-organization of interactions of, among others, non-targeted proteins to eventually evade single-target therapies. To develop efficient therapies, these dynamics should be taken into account and target whole network modules instead of singleton genes in order to prevent the establishment of compensating signaling circuits. Therefore, we integrated network-based methods, structural pharmacology, and molecular modeling to establish two complementary multitargeting strategies, one in the form of repurposable drug combinations and the other as a de novo synthesized triple-targeting agent. Of note, our study exploits, for the first time, a greedy algorithm to identify optimal combinations of drugs and therapeutic protein targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5dfe547a00d472fa7518f8c4c154b7df0f1b6a4" target='_blank'>
              Computer-Aided Identification and Design of Ligands for Multi-Targeting Inhibition of a Molecular Acute Myeloid Leukemia Network
              </a>
            </td>
          <td>
            Seyedehsadaf Asfa, Reza Arshinchi Bonab, Onur Önder, Merve Uça Apaydın, Hatice Döşeme, Can Küçük, A. Georgakilas, Bernhard M. Stadler, Stella Logotheti, Seyit Kale, Athanasia Pavlopoulou
          </td>
          <td>2024-10-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Epigenetic mechanisms often fuel the quick evolution of cancer cells from normal cells. Mutations or aberrant expressions in the enzymes of DNA methylation, histone post-translational modifications, and chromatin remodellers have been extensively investigated in cancer pathogenesis; however, cancer-associated histone mutants have gained momentum in recent decades. Next-generation sequencing of cancer cells has identified somatic recurrent mutations in all the histones (H3, H4, H2A, H2B, and H1) with different frequencies for various tumour types. Importantly, the well-characterised H3K27M, H3G34R/V, and H3K36M mutations are termed as oncohistone mutants because of their wide roles, from defects in cellular differentiation, transcriptional dysregulation, and perturbed epigenomic profiles to genomic instabilities. Mechanistically, these histone mutants impart their effects on histone modifications and/or on irregular distributions of chromatin complexes. Recent studies have identified the crucial roles of the H3K27M and H3G34R/V mutants in the DNA damage response pathway, but their impacts on chemotherapy and tumour progression remain elusive. In this review, we summarise the recent developments in their functions toward genomic instabilities and tumour progression. Finally, we discuss how such a mechanistic understanding can be harnessed toward the potential treatment of tumours harbouring the H3K27M, H3G34R/V, and H3K36M mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d93aa8724101a0a6b04c672de1eafdda67a5e3" target='_blank'>
              Emerging roles of cancer-associated histone mutations in genomic instabilities
              </a>
            </td>
          <td>
            Priyanka Yadav, Ronit Jain, Rajesh Kumar Yadav
          </td>
          <td>2024-10-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Therapy-related clonal hematopoiesis (t-CH) is defined as clonal hematopoiesis detected in individuals previously treated with chemotherapy and/or radiation therapy. With the increased use of genetic analysis in oncological care, the detection of t-CH among cancer patients is becoming increasingly common. t-CH arises through the selective bottleneck imposed by chemotherapies and potentially through direct mutagenesis from chemotherapies, resulting in a distinct mutational landscape enriched with mutations in DNA damage-response pathway genes such as TP53, PPM1D, and CHEK2. Emerging evidence sheds light on the mechanisms of t-CH development and potential strategies to mitigate its emergence. Due to its unique characteristics that predominantly affect cancer patients, t-CH has clinical implications distinct from those of CH in the general population. This Review discusses the potential mechanisms of t-CH development, its mutational landscape, mutant-drug relationships, and its clinical significance. We highlight the distinct nature of t-CH and call for intensified research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cad43d7ae7881b3bb897b501eceb9fbccf93c93" target='_blank'>
              Distinct landscape and clinical implications of therapy-related clonal hematopoiesis
              </a>
            </td>
          <td>
            Koichi Takahashi, Daisuke Nakada, Margaret A Goodell
          </td>
          <td>2024-10-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a2cdc51c75cca03becfcc1f33a36724cbb508a0" target='_blank'>
              Targeting mitochondria: a novel approach for treating platinum-resistant ovarian cancer
              </a>
            </td>
          <td>
            Xin Cui, Juan Xu, X. Jia
          </td>
          <td>2024-10-25</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) is a heterogeneous disease where multiple mutations in genes encoding transcriptional and growth regulators lead to an extensive rewiring of the gene regulatory network (GRN), thereby changing the identity of hematopoietic stem and progenitor cells in a way that blocks myeloid differentiation [1]. One hallmark of AML is the acquisition of mutations in growth factor receptor and growth signalling genes such as FMS-like tyrosine kinase 3 ( FLT3 ), the receptor for stem cell factor ( KIT ) and RAS . FLT3 is one of the most frequently mutated genes in AML. 25% of AMLs display an internal tandem duplication (ITD) which renders this receptor constitutively active. The prognosis for FLT3-ITD AML patients is dire as even after complete remission after chemotherapy, many patients relapse. Hence, inhibitors that also target the aberrant FLT3 receptor have been introduced but even then most patients relapse and become resistant, by either acquiring additional FLT3 mutations or mutations in downstream signalling molecules. However, many patients do not show additional mutations and rewire their signalling pathways to evade FLT3 inhibition by becoming dependent on other cytokines [2]. Significantly, AML cells can become also become resistant in a non-genetic way by changing their gene expression programs [3] but the role of aberrant signalling in this process remained unclear.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7212e2ffaa1e48f58fa237164fd146932496cb53" target='_blank'>
              Gene regulatory network and signalling pathway rewiring: How blood cancer cells shift their shapes to evade drug treatment
              </a>
            </td>
          <td>
            C. Bonifer, P. Cockerill
          </td>
          <td>2024-01-06</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Improving the selectivity and effectiveness of drugs represents a crucial issue for future therapeutic developments in immuno-oncology. Traditional bulk transcriptomics faces limitations in this context for the early phase of target discovery as resulting gene expression levels represent the average measure from multiple cell populations. Alternatively, single cell RNA sequencing can dive into unique cell populations transcriptome, facilitating the identification of specific targets. Here, we generated Tumor-Infiltrating regulatory T cells (TI-Tregs) and exhausted T cells (Tex) gene signatures from a single cell RNA-seq pan-cancer T cell atlas. To overcome noise and sparsity inherent to single cell transcriptomics, we then propagated the gene signatures by diffusion in a protein-protein interaction network using the Patrimony high-throughput computing platform. This methodology enabled the refining of signatures by rescoring genes based on their biological connectivity and shed light not only on processes characteristics of TI-Treg and Tex development and functions but also on their immunometabolic specificities. The combined use of single cell transcriptomics and network propagation may thus represent an innovative and effective methodology for the characterization of cell populations of interest and eventually the development of new therapeutic strategies in immuno-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/060798152c5e47a1cd66c8ffdb21f2a43f037a14" target='_blank'>
              The combined use of scRNA-seq and network propagation highlights key features of pan-cancer Tumor-Infiltrating T cells
              </a>
            </td>
          <td>
            Adèle Mangelinck, Elodie Molitor, Ibtissam Marchiq, Lamine Alaoui, Matthieu Bouaziz, Renan Andrade-Pereira, Hélène Darville, Etienne Becht, Céline Lefebvre
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The tumour microenvironment (TME) is a dynamic nexus where cancer cell metabolism and the immune system intricately converge, with nucleotide metabolism (NM) playing a pivotal role. This review explores the critical function of NM in cancer cell proliferation and its profound influence on the TME and immune landscape. NM is essential for DNA and RNA synthesis and is markedly upregulated in cancer cells to meet the demands of rapid growth. This metabolic rewiring fuels cancer progression, but also shapes the TME, impacting the function and viability of immune cells. The altered nucleotide milieu in the TME can suppress immune response, aiding cancer cell evasion from immune surveillance. Drug discoveries in the field of NM have revealed different therapeutic strategies, including inhibitors of nucleotide synthesis and drugs targeting salvage pathways, which are discussed thoroughly in this review. Furthermore, the emerging strategy of combining NM-targeted therapies with immunotherapies is emphasised, particularly their effect on sensitising tumours to immune checkpoint inhibitors and enhancing overall treatment efficacy. The Human Genome Project paved the way for personalised medicine, countering the established 'one size fits all' approach to cancer treatment. Advances in understanding the TME and NM have spurred interest in personalised therapeutic strategies. This review highlights the potential of leveraging individual tumour metabolic profiles to guide treatment selection, aiming to optimise efficacy and minimise adverse effects. The strategic importance of targeting NM in cancer therapy and its synergistic potential with immunotherapies offers a path towards more effective and personalised cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bc17614b60cc3e87bf59862d1663d401029e4eb" target='_blank'>
              Harnessing nucleotide metabolism and immunity in cancer: a tumour microenvironment perspective.
              </a>
            </td>
          <td>
            Hadil Sulieman, Alexandra J Emerson, Peter M. Wilson, Karl A Mulligan, R. Ladner, Melissa J LaBonte
          </td>
          <td>2024-09-22</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Breast cancer (BC) is marked by significant genetic, morphological and clinical heterogeneity. To capture this heterogeneity and unravel the molecular mechanisms driving tumor progression and drug resistance, we established a comprehensive patient-derived xenograft (PDX) biobank, focusing particularly on luminal (estrogen receptor, ER+) and young premenopausal patients, for whom PDX models are currently scarce. Across all BC subtypes, our efforts resulted in an overall success rate of 17% (26 established PDX lines out of 151 total attempts), specifically 15% in luminal, 12% in human epidermal growth factor receptor 2 positive (HER2+) and 35% in triple negative BC. These PDX mirrored morphologic and genetic features of BC from which they originated, serving as a reliable tool to investigate drug resistance and test therapeutic strategies. We focused on understanding resistance to CDK4/6 inhibitors (CDK4/6i), which are crucial in the treatment of patients with advanced luminal BC. Treating a sensitive luminal BC PDX with the CDK4/6i palbociclib revealed that, despite initial tumor shrinkage, some tumors might eventually regrow under drug treatment. RNA sequencing, followed by gene set enrichment analyses, unveiled that these PDXs have become refractory to CDK4/6i, both at biological and molecular levels, displaying significant enrichment in proliferation pathways, such as MTORC1, E2F and MYC. Using organoids derived from these PDX (PDxO), we observed that acquisition of CDK4/6i resistance conferred cross-resistance to endocrine therapy and that targeting MTORC1 was a successful strategy to overcome CDK4/6i resistance. Considered together, these results indicate that our PDX models may serve as robust tools to elucidate the molecular basis of BC disease progression and, by providing the possibility to simultaneously test different therapies on the same tumor, to surmount treatment resistance. While this approach is of course not feasible in the clinic, its exploitation in PDX may expedite the identification and development of more successful therapies for patients with advanced luminal BC. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e518aa62c8ccd88fccbc075513337a7ece3d380" target='_blank'>
              A comprehensive luminal breast cancer patient-derived xenografts (PDX) library to capture tumor heterogeneity and explore the mechanisms of resistance to CDK4/6 inhibitors.
              </a>
            </td>
          <td>
            I. Segatto, M. Mattevi, Gian Luca Rampioni Vinciguerra, Nicole Crestan, L. Musco, Andrea Favero, Alessandra Dall'Acqua, Gabriele Di Giustino, Giorgia Mungo, Sara D'Andrea, Chiara Gava, Federica Ruggiero, Matteo Dugo, Lorenzo Gerratana, Fabio Puglisi, S. Massarut, Riccardo Bomben, Maurizio Callari, Tiziana Perin, Gustavo Baldassarre, B. Belletti
          </td>
          <td>2024-10-25</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Treatment with the hypomethylating agent 5-azacytidine (AZA) increases survival in high-risk (HR) myelodysplastic syndrome (MDS) patients, but predicting patient response and overall survival remains challenging. To address these issues, we analyzed mutational and transcriptional profiles in CD34+ hematopoietic stem/progenitor cells (HSPCs) before and following AZA therapy in MDS patients. AZA treatment led to a greater reduction in the mutational burden in both blast and hematological responders than non-responders. Blast and hematological responders showed transcriptional evidence of pre-treatment enrichment for pathways such as oxidative phosphorylation, MYC targets, and mTORC1 signaling. While blast non-response was associated with TNFa signaling and leukemia stem cell signature, hematological non-response was associated with cell-cycle related pathways. AZA induced similar transcriptional responses in MDS patients regardless of response type. Comparison of blast responders and non-responders to normal controls, allowed us to generate a transcriptional classifier that could predict AZA response and survival. This classifier outperformed a previously developed gene signature in a second MDS patient cohort, but signatures of hematological responses were unable to predict survival. Overall, these studies characterize the molecular consequences of AZA treatment in MDS HSPCs and identify a potential tool for predicting AZA therapy responses and overall survival prior to initiation of therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f64e8b321545ad08389931dee01148ac0d9c897" target='_blank'>
              Molecular profiling of pre- and post- 5-azacytidine myelodysplastic syndrome samples identifies predictors of response
              </a>
            </td>
          <td>
            Mónica del Rey González, Sohini Chakraborty, J. Hernández-Sánchez, M. Díez Campelo, Christopher Y. Park, J. H. Hernández Rivas
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5d9e40df709b51d001bd72cce2707582e44efe8" target='_blank'>
              Loss of miR-200c-3p promotes resistance to radiation therapy via the DNA repair pathway in prostate cancer
              </a>
            </td>
          <td>
            Maureen Labbé, Manon Chang, Benjamin Saintpierre, Franck Letourneur, Laurence de Beaurepaire, J. Veziers, S. Deshayes, Marine Cotinat, J. Fonteneau, C. Blanquart, Vincent Potiron, S. Supiot, D. Fradin
          </td>
          <td>2024-10-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Aim This study was to explore role of immune landscape and the immune cells in acute lymphoblastic leukemia (ALL) progression. Background The most prevalent genetic alteration in childhood ALL is the ETV6-RUNX1 fusion. The increased proliferation of B progenitor cells could expedite the disease’s progression due to irregularities in the cell cycle. Nevertheless, the mechanisms by which particular cell clusters influence the cell cycle and promote the advancement of ALL are still not well understood. Objective This study was to explore role of immune landscape and the immune cells in ALL progression. Methods Single-cell RNA sequencing (scRNA-seq) data of ETV6-RUNX1 and healthy pediatric samples obtained from GSE132509 were clustered and annotated using the Seurat package, and differentially highly expressed genes identified in each cluster were analyzed using DAVID for pathway annotation. Chromosome amplification and deletion were analyzed using the inferCNV package. SCENIC evaluated the regulation of transcription factors and target gene formation in cells. cellphoneDB and CellChat were served to infer ligand-receptor pairs that mediate interactions between subpopulations. The role of the target gene in regulating ALL progression was assessed using RT-qPCR, Transwell and scratch healing assays. Results The bone marrow mononuclear cells (BMMCs) from ETV6-RUNX1 and healthy pediatric samples in GSE132509 were divided into 11 clusters, and B cell cluster 1 was identified as B progenitor cell, which was amplified on chromosome 6p. B progenitor cells were divided into seven clusters. Expression levels of amplified genes in chromosome 6p of B progenitor cell cluster 5 were the highest, and its specific highly expressed genes were annotated to pathways promoting cell cycle progression. Regulons formed in B progenitor cell cluster 5 were all involved in promoting cell cycle progression, so it was regarded as the B progenitor cell cluster that drives cell cycle progression. The key regulator of the B progenitor cell is E2F1, which promotes the migration and invasion ability of the cell line HAP1. The major ligand-receptor pairs that mediate the communication of B progenitor cell cluster 5 with cytotoxic NK/T cells or naive T cells included FAM3C−CLEC2D, CD47−SIRPG, HLAE−KLRC2, and CD47−KLRC2. HLAE−KLRC1 and TGFB1−(TGFBR1+TGFBR2). Conclusion This study outlined the immune cell landscape of ETV6-RUNX1 ALL and identified chromosome 6p amplification in B progenitor cells, described the major B progenitor cell cluster driving cell cycle progression and its potential regulatory mechanisms on NK cells and T cells, providing cellular and molecular insights into ETV6-RUNX1 ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de3198365fdda74c6d3b8026f58bc539920ce2b9" target='_blank'>
              Potential molecular mechanisms of ETV6-RUNX1-positive B progenitor cell cluster in acute lymphoblastic leukemia revealed by single-cell RNA sequencing
              </a>
            </td>
          <td>
            Ning Qu, Yue Wan, Xin Sui, Tianyi Sui, Yang Yang
          </td>
          <td>2024-11-01</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Caspase activated DNase (CAD) induced DNA breaks promote cell differentiation and therapy-induced cancer cell resistance. CAD targeting activity is assumed to be unique to each condition, as differentiation and cancer genesis are divergent cell fates. Here, we made the surprising discovery that a subset of CAD-bound targets in differentiating muscle cells are the same genes involved in the genesis of cancer-causing translocations. In muscle cells, a prominent CAD-bound gene pair is Pax7 and Foxo1a, the mismatched reciprocal loci that give rise to alveolar rhabdomyosarcoma. We show that CAD-targeted breaks in the Pax7 gene are physiologic to reduce Pax7 expression, a prerequisite for muscle cell differentiation. A cohort of these CAD gene targets are also conserved in early differentiating T cells and include genes that spur leukemia/lymphoma translocations. Our results suggest the CAD targeting of translocation prone oncogenic genes is non-pathologic biology and aligns with initiation of cell fate transitions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95535f8b5c0fda938ec705d36c58cef6067d4ac1" target='_blank'>
              Caspase-Activated DNase localizes to cancer causing translocation breakpoints during cell differentiation
              </a>
            </td>
          <td>
            Dalal Alsowaida, Brian D. Larsen, Sarah Hachmer, M. Azimi, Eric Arezza, Steve Brunette, Steven Tur, C. Palii, Bassam Albraidy, Claus S. Sorensen, Marjorie Brand, F. J. Dilworth, L. Megeney, Senior Scientist
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Although the concept of cancer stem cells is still controversial, previous studies have shown that blood cancers, as well as specific types of solid cancers such as colorectal cancer, rely on stem cells during the onset of tumor growth and further tumor development. Moreover, resistance to therapeutic treatment in leukemias such as acute myeloid leukemia and in colorectal cancer can be attributed to a small population of cells with stemness properties known as minimal residual disease. In this review, we look back on the discovery of cancer stem cells and the contribution of the findings in blood cancer to a parallel discovery in solid cancers. We focus on CD44 as a stem cell marker, both in blood cancers and in several types of solid cancers, particularly of the gastro-intestinal tract. This review highlights newly discovered molecular mechanisms of action of CD44 which indicate that CD44 has indeed a function in stemness, stem cell maintenance and drug resistance. We attempt here to make the link between the functions of CD44 isoforms in stemness and their involvement in specific steps of tumor growth and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8580e7867b570560c73223a09cea13edca2587fc" target='_blank'>
              CD44: A Stemness Driver, Regulator and Marker - all in one?
              </a>
            </td>
          <td>
            S. J. Sonnentag, Nagwa S M Ibrahim, V. Orian-Rousseau
          </td>
          <td>2024-10-04</td>
          <td>Stem cells</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Recent models of infection and experimental inflammation reveal that hematopoietic stem and progenitor cells (HSPCs) can generate a memory of the exposure that heightens the response to subsequent stimulation, a process termed central trained immunity. Inflammation is also a constitutive feature of cancer, including hematologic malignancies. Here, we adapt a translationally relevant model of acute myeloid leukemia (AML) to determine if inflammation in the bone marrow (BM) niche durably reprograms resident healthy HSPCs. To simulate the onset of malignancy along with the associated inflammatory surge as well as the subsequent remission, we generated hematopoietic chimera with healthy HSPCs and HSPCs bearing a doxycycline-responsive oncogene (hMLL-AF9) expression cassette, a validated model of AML. Results show that the exposure to AML blasts in the BM leaves healthy HSPCs during experimental remission with broad transcriptomic, epigenetic changes and enhanced reliance on glycolysis. A heterologous secondary challenge of AML-experienced animals resulted in pronounced gene expression changes in inflammatory and metabolic pathways. These augmented responses coincided with altered chromatin accessibility in AML-experienced HSPCs. Motif analysis of the epigenome in AML trained HSPC points to the involvement of core hematopoietic transcription factors. Altogether, these observations provide first evidence for the durable inflammatory reprogramming of healthy HSPCs in the cancer microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77f62b237aad9d93fc1b1c079c79f60b62a3df71" target='_blank'>
              Leukemia confers a durable imprint on healthy hematopoietic stem and progenitor cells
              </a>
            </td>
          <td>
            Ding-Wen Chen, Julie M. Schrey, Jian-Meng Fan, Sarah E. Adams, , Eric K. Wafula, Peter Kurre
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4c265b900d210c2847c325efc074e2d026f85f2" target='_blank'>
              The BIM deletion polymorphism potentiates the survival of leukemia stem and progenitor cells and impairs response to targeted therapies.
              </a>
            </td>
          <td>
            Mengge Yu, G. Nah, Vaidehi Krishnan, Fatin Nasha Bte Sulaimi, K. Ng, Chuqi Wang, Shruti Bhatt, Charles T.H. Chuah, David E Bergstrom, S. Ong
          </td>
          <td>2024-10-22</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background: Melanoma remains one of the most challenging cancers to treat effectively with drug resistant remaining a constant concern, primarily with activating BRAF mutations. Mutations in the BRAF gene appear in approximately 50% of patients, 90% of which are V600E. Two frontline BRAF inhibitors (BRAFi), vemurafenib and dabrafenib, are frequently used to treat unresectable or metastatic BRAF V600E melanoma. Initial response rates are high, but soon thereafter, 70-80% of patients develop resistance to treatment within a year. A major mechanism of resistance is the generation of a secondary Q61K mutation in the NRAS gene. Methods: We have developed an approach in which a CRISPR-Cas complex can be designed to distinguish between mutant genes enabling resistance to standard care in tumor cells and normal genomes of healthy cells. For the first time, we demonstrated the utility of two CRISPR-directed mutation-specific editing approaches to restore BRAFi sensitivity in BRAFV600E/NRASQ61K resistant A375 cells. Results: We utilize an AsCas12a protospacer adjacent motif site created by the NRAS Q61K mutation and the Q61K mutation in the critical seed region of an SaCas9 sgRNA for Q61K-selective targeting. We show here that both approaches allow for effective NRAS targeting of only mutated-Q61K and after CRISPR-directed Q61K-targeting, previously resistant A375 cells are re-sensitized to BRAFi treatment. Conclusion: Our data support the feasibility of the development of CRISPR-Cas therapeutic approaches to the treatment of melanoma. Successful therapeutic CRISPR-directed gene editing would enable both specific and efficient editing of a mutation-specific targeting approach eliminate concern for on- and off-target damage to the genomes of healthy cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0aea0b595d066144e478150c4bdebe3df9d61c5" target='_blank'>
              Mutation-Specific CRISPR Targeting with SaCas9 and AsCas12a Restores Therapeutic Sensitivity in Treatment-Resistant Melanoma.
              </a>
            </td>
          <td>
            Brett M. Sansbury, Sophia B. Masciarelli, Salma Kaouser, Olivia M. Tharp, Kelly Banas, Eric B Kmiec
          </td>
          <td>2024-10-10</td>
          <td>The CRISPR journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D’Amato, Yi Xue, Kristina Fontanez, Aaron A. May-Zhang, Trinity Smithers, Y. Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="While immune checkpoint inhibitors have revolutionized cancer therapy, many patients exhibit poor outcomes. Here, we show immunotherapy responses in bladder and non–small cell lung cancers are effectively predicted by factoring tumor mutation burden (TMB) into burdens on specific protein assemblies. This approach identifies 13 protein assemblies for which the assembly-level mutation burden (AMB) predicts treatment outcomes, which can be combined to powerfully separate responders from nonresponders in multiple cohorts (e.g., 76% versus 37% bladder cancer 1-year survival). These results are corroborated by (i) engineered disruptions in the predictive assemblies, which modulate immunotherapy response in mice, and (ii) histochemistry showing that predicted responders have elevated inflammation. The 13 assemblies have diverse roles in DNA damage checkpoints, oxidative stress, or Janus kinase/signal transducers and activators of transcription signaling and include unexpected genes (e.g., PIK3CG and FOXP1) for which mutation affects treatment response. This study provides a roadmap for using tumor cell biology to factor mutational effects on immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5f818fb592baa5f66559ad69838daf202c25a6" target='_blank'>
              Prediction of immunotherapy response using mutations to cancer protein assemblies
              </a>
            </td>
          <td>
            JungHo Kong, Xiaoyu Zhao, Akshat Singhal, Sungjoon Park, Robin Bachelder, Jeanne Shen, Haiyu Zhang, Jimin Moon, Changho Ahn, C. Ock, Hannah Carter, T. Ideker
          </td>
          <td>2024-09-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Introduction Single-cell RNA sequencing (scRNA-seq) has become an essential tool for uncovering the complexities of various physiological and immunopathological conditions in veterinary medicine. However, there is currently limited information on immune-suppressive cancer subsets in canine breast cancers. In this study, we aimed to identify and characterize immune-suppressive subsets of triple-negative canine breast cancer (TNBC) by utilizing integrated scRNA-seq data from published datasets. Methods Published scRNA-seq datasets, including data from six groups of 30 dogs, were subjected to integrated bioinformatic analysis. Results Immune modulatory TNBC subsets were identified through functional enrichment analysis using immune-suppressive gene sets, including those associated with anti-inflammatory and M2-like macrophages. Key immune-suppressive signaling, such as viral infection, angiogenesis, and leukocyte chemotaxis, was found to play a role in enabling TNBC to evade immune surveillance. In addition, interactome analysis revealed significant interactions between distinct subsets of cancer cells and effector T cells, suggesting potential T-cell suppression. Discussion The present study demonstrates a versatile and scalable approach to integrating and analyzing scRNA-seq data, which successfully identified immune-modulatory subsets of canine TNBC. It also revealed potential mechanisms through which TNBC promotes immune evasion in dogs. These findings are crucial for advancing the understanding of the immune pathogenesis of canine TNBC and may aid in the development of new immune-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc211a7b60743c5c7bf82a5e5216ed89760ca205" target='_blank'>
              Leveraging single-cell transcriptomic data to uncover immune suppressive cancer cell subsets in triple-negative canine breast cancers
              </a>
            </td>
          <td>
            Myung-Chul Kim, N. Borcherding, Woojin Song, R. Kolb, Weizhou Zhang
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Veterinary Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Despite efforts to understand breast cancer biology, metastatic disease remains a clinical challenge. Identifying suppressors of breast cancer progression and mechanisms of transition to more invasive phenotypes could provide game changing therapeutic opportunities. Transcriptional deregulation is central to all malignancies, highlighted by the extensive reprogramming of regulatory elements that underlie oncogenic programs. Among these, super-enhancers (SEs) stand out due to their enrichment in genes controlling cancer hallmarks. To reveal novel breast cancer dependencies, we integrated the analysis of the SE landscape with master regulator activity inference for a series of breast cancer cell lines. As a result, we identified T-helper-inducing Poxviruses and Zinc-finger (POZ)/Krüppel-like factor (ThPOK, ZBTB7B), a CD4+ cell lineage commitment factor, as a breast cancer master regulator that is recurrently associated with a SE. ThPOK expression is highest in luminal breast cancer but is significantly reduced in the basal subtype. Manipulation of ThPOK levels in cell lines shows that its repressive function restricts breast cancer cells to an epithelial phenotype by suppressing the expression of genes involved in the epithelial-mesenchymal transition (EMT), WNT/β-catenin target genes, and the pro-metastatic TGFβ pathway. Our study reveals ThPOK as a master transcription factor that restricts the acquisition of metastatic features in breast cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bcac35bb061bf9ab262a029b66da149e294b402" target='_blank'>
              Super-enhancer profiling reveals ThPOK/ZBTB7B, a CD4+ cell lineage commitment factor, as a master regulator that restricts breast cancer cells to a luminal non-migratory phenotype
              </a>
            </td>
          <td>
            Camila D. Arcuschin, Kamin Kahrizi, Rosalyn W. Sayaman, Carolina DiBenedetto, Yizhuo Shen, Pedro J Salaberry, Ons Zakroui, Cecilia Schwarzer, Alessandro Scapozza, Paola Betancur, Julie D. Saba, Jean-Philippe Coppé, M. Barcellos-Hoff, Dietmar Kappes, L. J. van ’t Veer, Ignacio E Schor, Denise P Muñoz
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains an intractable cancer characterized by significant inter- and intra-patient heterogeneity. Recent research has established connections between key genetic drivers and distinct transcriptional profiles in GBM, yet the interactions between genetic changes and the transcriptional and spatial heterogeneity remain unclear. This study used genetically engineered mouse models, coupled with spatially resolved and single-cell RNA sequencing, to explore how specific genetic alterations influence the tumor microenvironment (TME) and cellular responses spatially.



 We used nestin-Tva C57BL/6 mice and generated de novo GBM using overexpression of oncogenes in combinations with short hairpin RNAs (shRNAs) or deletion to various tumor suppressors, including Tp53, Cdkn2A, and Pten. We modeled key driver mutations for GBM: PDGFB, NF1, and EGFR. We conducted single-cell RNA sequencing, combined with spatially resolved transcriptomics and proteomics. Data integration techniques, including horizontal (mutual nearest neighbors) and vertical (weighted nearest neighbors) approaches, were used and analyzed with the SPATA2 software. Advanced computational methods, such as graph neural networks and single-cell deconvolution, were applied to examine GBM genotypes and the spatial variability within tumors and their TMEs.



 Our findings revealed persistent cellular heterogeneity among tumor cells under various genetic conditions. However, specific genetic alterations influenced the cells toward dominant transcriptional states. NF1 and EGFR mutations resulted in shifts towards MES and AC-like states, respectively, unlike PDGFB, which promoted a NPC/OPC-like state. These transcriptional shifts were strongly associated with distinct microenvironmental changes, such as enhanced neuronal signaling in PDGFB-driven GBMs versus pronounced immune infiltration in NF1 and EGFR models. Utilizing weighted co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA) alongside spatial-niche deconvolution, we identified a mesenchymal transition leading to an immunosuppressive TME in NF1-driven GBMs and increased lymphocyte presence in EGFR-amplified GBMs, especially around blood vessels. Similar spatial patterns were observed in human spatially resolved transcriptomic data, highlighting the complexity and variability of genetic drivers in human GBMs.



 Our genetically engineered mouse models provide a valuable platform for investigating GBM with defined driver mutations. By leveraging sophisticated analytical techniques and computational modeling, this study underscores the potential to elucidate the interactions between genetic drivers and the tumor microenvironment. Our approach paves the way for developing targeted therapies and precision medicine strategies in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0056b4191c0de2f537409a208d3f06f5659352" target='_blank'>
              OS07.7.A DISSECTING SPATIAL DIVERSITY OF GLIOBLASTOMA GENOTYPES
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Hambardzumyan, D. Heiland
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetic intra-tumour heterogeneity (gITH) is a universal property of all cancers. It emerges from the interplay of cell division, mutation accumulation and selection with important implications for the evolution of treatment resistance. Theoretical and data-driven approaches extensively studied gITH in ageing somatic tissues or cancers at detection. Yet, the expected patterns of gITH during and after treatment are less well understood. Here, we use stochastic birth-death processes to investigate the expected patterns of gITH across different treatment scenarios. We consider homogeneous treatment response with shrinking, growing and stable disease, and follow up investigating heterogeneous treatment response with sensitive and resistant cell types. We derive analytic expressions for the site frequency spectrum, the total mutational burden and the single-cell mutational burden distribution that we validate with computer simulations. We find that the SFS after homogeneous treatment response retains its characteristic power-law tail, while emergent resistant clones cause peaks corresponding to their sizes. The frequency of the largest resistant clone is subdominant and independent of the population size at detection, whereas the relative total number of resistant cells increases with detection size. Furthermore, the growth dynamics under treatment determine whether the total mutational burden is dominated by preexisting or newly acquired mutations, suggesting different possible treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b068763b28793a776bfaf3fe0647aae0b724cad" target='_blank'>
              On the patterns of genetic intra-tumour heterogeneity before and after treatment
              </a>
            </td>
          <td>
            Alexander Stein, Benjamin Werner
          </td>
          <td>2024-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 At the Gene Editing Institute, we are advancing a novel strategy for the treatment of pancreatic ductile adenocarcinoma by employing CRISPR-directed gene editing to disable the functions of key genes that block effective drug application including chemotherapy, radiation and immunotherapy. No matter how careful one is at designing a specific drug against a specific protein, ultimately, the tumor cell develops a workaround. A paradigm shift is required to truly overcome this formidable barrier. We’ve accomplished this by leveraging the effectiveness of standard care and augmenting it, thereby reducing patient suffering, increasing treatment tolerance and, most importantly, improving outcomes. With an enthusiastic response for this concept from the FDA- INTERFACT meeting, we have demonstrated that a carefully selected CRISPR/Cas complex disables the NRF2 gene in the tumor cell and significantly reduces the dose of chemotherapy required to halt pancreatic tumor cell growth, and consequently promoting cell death. The efficiency of gene editing approaches 70 % routinely, which destroys the function of NRF2 and enables additional killing of residual tumor cells by chemotherapeutic agents. As a consequence of CRISPR/Cas activity, we effectively reduce the amount of chemotherapy needed to kill tumor cells by over 20-fold. This level of reduction should enable patients to complete their treatment regimens more successfully, at lower dosages and with far fewer adverse side effects. In addition to reduced suffering, this will invariably prolong survival as we are augmenting already proven treatments. This approach is agnostic to the haplotype of the tumor cell and does not rely on specific biomarker composition; radically expanding the patient population that would be suitable for this type of treatment. We will discuss the impact of CRISPR/Cas-gene editing to disable specific genes, including NRF2 and mutant KRAS, to enable current therapies to work at greater efficacy and at lower dosages.
 Citation Format: London P McGill, Kelly H Banas, Gregory Tiesi, Eric B Kmiec. A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/159a6092b196d29302afbdfaca37a6f4ece01dff" target='_blank'>
              Abstract A010: A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma
              </a>
            </td>
          <td>
            L. McGill, Kelly Banas, Gregory Tiesi, Eric B Kmiec
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Somatic mutations in mitochondrial genomes (mtDNA) accumulate exponentially during aging. Using single cell sequencing, we characterize the spectrum of age-accumulated mtDNA mutations in mouse and human liver and identify directional forces that accelerate the accumulation of mutations beyond the rate predicted by a neutral model. “Driver” mutations that give genomes a replicative advantage rose to high cellular abundance and carried along “passenger” mutations, some of which are deleterious. In addition, alleles that alter mtDNA-encoded proteins selectively increased in abundance overtime, strongly supporting the idea of a “destructive” selection that favors genomes lacking function. Overall, this combination of selective forces acting in hepatocytes promotes somatic accumulation of mutations in coding regions of mtDNA that are otherwise conserved in evolution. We propose that these selective processes could contribute to the population prevalence of mtDNA mutations, accelerate the course of heteroplasmic mitochondrial diseases and promote age-associated erosion of the mitochondrial genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b79d25beebe82ec541628bbaaa070025330213" target='_blank'>
              Selection promotes age-dependent degeneration of the mitochondrial genome
              </a>
            </td>
          <td>
            Ekaterina Korotkevich, Daniel N. Conrad, Zev J. Gartner, Patrick H. O’Farrell
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Understanding gene function on a genome-wide scale is a fundamental goal in biology. With the advent of next-generation sequencing, high-throughput methods such as transposon sequencing (Tn- seq) were developed to measure fitness of single deletion mutants en masse. Nevertheless, gene redundancy complicates these approaches, as inactivating genes individually may not produce discernable phenotypes. Here, we report Dual Tn-seq, a technique for simultaneously assaying fitness of a comprehensive pool of double mutants. Dual Tn-seq couples random barcode transposon-site sequencing (RB Tn-seq) with the Cre-lox system, allowing us to determine the fitness of ≈68% of all possible double mutant combinations in the human pathogen Streptococcus pneumoniae. Genetic interactions identified from ≈1.4 billion double mutants uncovered new avenues in widely conserved biochemical pathways, exemplified by the discovery of a new CTP synthase and a regulator of peptidoglycan biosynthesis. Since Dual Tn-seq has very few requirements, it can be readily adapted to diverse organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1138531d07519f5ffb747fb41884aaea9e2bed44" target='_blank'>
              Dual transposon sequencing (Dual Tn-seq) to probe genome-wide genetic interactions
              </a>
            </td>
          <td>
            J. Zik, M. Price, Adam P. Arkin, A. Deutschbauer, Lok-To Sham
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Gliomas, originating from the most common non-neuronal cells in the brain (glial cells), are the most common brain tumors and are associated with high mortality and poor prognosis. Glioma cells exhibit a tendency to disrupt normal cell-cycle regulation, leading to abnormal proliferation and malignant growth. This study investigated the predictive potential of GJC1 in gliomas and explored its relationship with the cell cycle. Methods Retrospective analysis of RNA-seq and single-cell sequencing data was conducted using the Chinese Glioma Genome Atlas (CGGA) and The Cancer Genome Atlas (TCGA) databases. The differential expression of GJC1 in gliomas with various pathological features and in different non-neuronal cell groups was analyzed. Functional data were examined using gene set variation analysis (GSVA). Furthermore, CellMiner was used to evaluate the relationship between GJC1 expression and predicted treatment response across these databases. Results GJC1 expression was enriched in high-grade gliomas and 1p/19q non-codeletion gliomas. GJC1 enrichment was observed in classical and mesenchymal subtypes within the TCGA glioma subtype group. In single-cell subgroup analysis, GJC1 expression was higher in glioma tissues compared to other non-neuronal cells. Additionally, the TCGA classical subtype of glioma cells exhibited more GJC1 expression than the other subgroups. GJC1 emerged as an independent prognostic factor for overall survival in glioma. GSVA unveiled potential mechanisms by which GJC1 may impact cell-cycle regulation in glioma. Finally, a significant correlation was observed between GJC1 expression and the sensitivity of multiple anti-cancer drugs. Conclusion These findings confirmed GJC1 as a novel biomarker and provided insights into the differential gene expression in non-neuronal cells and the impact of the cell cycle on gliomas. Consequently, GJC1 may be used to predict glioma prognosis and has potential therapeutic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24a5a8d74969f90d7980595b424382f99d6cd110" target='_blank'>
              Discovery of GJC1 as a prognostic biomarker in glioma cells: insights into its cell-cycle relationship and differential expression in non-neuronal cells
              </a>
            </td>
          <td>
            Xiangtian Ji, Xin Chen, Guozhong Lin, Kai-Li Ma, Jun Yang, Xiaofang Zhao, Su-hua Chen, Jun Yang
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In the fields of medicine and bioscience, gene editing is increasingly recognized as a promising therapeutic approach for treating pathogenic variants in humans and other living organisms. With advancements in technology and knowledge, it is now understood that most genetic defects are caused by single-base pair variants. The ability to substitute genes using genome editing tools enables scientists and doctors to cure genetic diseases and disorders. Starting with CRISPR (clustered regularly interspaced short palindromic repeats)/Cas, the technology has evolved to become more efficient and safer, leading to the development of base and prime editors. Furthermore, various approaches are used to treat genetic disorders such as hemophilia, cystic fibrosis, and Duchenne muscular dystrophy. As previously mentioned, most genetic defects leading to specific diseases are caused by single-base pair variants, which can occur at many locations in corresponding gene, potentially causing the same disease. This means that, even when using the same genome editing tool, results in terms of editing efficiency or treatment effectiveness may differ. Therefore, different approaches may need to be applied to different types of diseases. Prevalently, due to the safety of adeno-associated virus (AAV) vectors in gene therapy, most clinical trials of gene therapy are based on AAV delivery methods. However, despite their safety and nonintegration into the host genome, their limitations, such as confined capacity, dosage-dependent viral toxicity, and immunogenicity, necessitate the development of new approaches to enhance treatment effects. This review provides the structure and function of each CRISPR-based gene editing tool and focuses on introducing new approaches in gene therapy associated with improving treatment efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175c7ca99490d5250dd5cadc163e90b91db26af6" target='_blank'>
              Mechanism of Genome Editing Tools and Their Application on Genetic Inheritance Disorders
              </a>
            </td>
          <td>
            Dae Hwan Oh
          </td>
          <td>2024-09-16</td>
          <td>Global Medical Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Prostate cancer (PC) is a leading cause of death in men because of the high incidence and long-term inefficacy of the existing treatment options. Furthermore, it exhibits significant health disparities that affect African-American (AA) men more adversely than others do. Previously, we established CYP3A5, a highly expressed protein in AAs PC, as a positive regulator of androgen receptor (AR) signaling. We examined the impact of CYP3A5 depletion on genome-wide transcriptional output using RNA sequencing to gain deeper mechanistic insights. The data revealed that 561 genes were downregulated and 263 were upregulated upon silencing of CYP3A5 in PC cells. Furthermore, in silico pathway analyses of differentially expressed genes suggested that the cell cycle regulation pathway was most significantly affected by CYP3A5 inhibition. Cell cycle analysis of CYP3A5-silenced cells and those treated with clobetasol, a specific CYP3A5 pharmacological inhibitor, showed G1/S phase blockade. Both CYP3A5-depletion and pharmacological inhibition resulted in the downregulation of cyclin D, cyclin B, and CDK2, along with the upregulation of p27kip1 but had minimal effects on CDK4/6 levels. Combination treatment with clobetasol and the CDK4/6 inhibitor palbociclib exhibited synergy with combination index (CI) values ranging from 0.28-0.78. Our findings support the utility of CYP3A5 as a druggable therapeutic target that works more effectively in combination with CDK4/6 inhibition to limit the progression of PC, especially for AA patients with AA. This combination addresses CDK4/6 inhibitor resistance, which is often linked to CDK2 overexpression, and can potentially be useful in reducing disparities in the clinical outcomes of PC. Significance Our study highlights CYP3A5 as a key regulator of the cell cycle in prostate cancer (PC). Its overexpression in African American (AA) patients may be a key molecular driver of disparities in outcomes. The combination of CYP3A5 and CDK4/6 inhibitors shows a synergistic effect on therapeutic outcomes and addresses CDK2-mediated resistance. Thus, targeting both CYP3A5 and CDK4/6 could improve treatment outcomes, especially in AA PC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04bad07ffad0a8a7cbaae50a2fcf79091b02b95e" target='_blank'>
              CYP3A5 inhibition causes G1/S blockade and synergizes with CDK4/6 inhibitor to suppress prostate cancer cell growth: Implications in reducing health disparity
              </a>
            </td>
          <td>
            Jeetesh Sharma, Imran K. Mohammed, R. Tillett, Jake McLean, Shirley Shen, Ajay P. Singh, Oscar B. Goodman, Edwin C. Oh, R. Mitra
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Tumorigenesis is commonly attributed to Darwinian processes involving natural selection among cells and groups of cells. However, progressing tumors are those that also achieve an appropriate group phenotypic composition (GPC). Yet, the selective processes acting on tumor GPCs are distinct from that associated with classical Darwinian evolution (i.e. natural selection based on differential reproductive success) as tumors are not genuine evolutionary individuals and do not exhibit heritable variation in fitness. This complex evolutionary scenario is analogous to the recently proposed concept of ‘selection for function’ invoked for the evolution of both living and non-living systems. Therefore, we argue that it is inaccurate to assert that Darwinian processes alone account for all the aspects characterizing tumorigenesis and cancer progression; rather, by producing the genetic and phenotypic diversity required for creating novel GPCs, these processes fuel the evolutionary success of tumors that is dependent on selection for function at the tumor level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed9ff2b803aab7c7484cdca4ce05128b6e07445" target='_blank'>
              A new perspective on tumor progression
              </a>
            </td>
          <td>
            Frédéric Thomas, James DeGregori, A. Marusyk, Antoine M. Dujon, Beata Ujvari, Jean-Pascal Capp, Robert Gatenby, A. Nedelcu
          </td>
          <td>2024-09-19</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9918c8bdd0094035be3fb2b57ac7bbf09b9c90ee" target='_blank'>
              Comprehensive analysis of bulk, single-cell RNA sequencing, and spatial transcriptomics revealed IER3 for predicting malignant progression and immunotherapy efficacy in glioma
              </a>
            </td>
          <td>
            Qi Wang, Chunyu Zhang, Ying Pang, Meng Cheng, Rui Wang, Xu Chen, Tongjie Ji, Yuntong Yang, Jing Zhang, Chunlong Zhong
          </td>
          <td>2024-10-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8f6ed0c99a8d797b23d1e3852690ab89f48967" target='_blank'>
              Nanotechnology and CRISPR/Cas-Mediated Gene Therapy Strategies: Potential Role for Treating Genetic Disorders.
              </a>
            </td>
          <td>
            Guneet Kaur, Jasnoor Arora, Abhinashi Singh Sodhi, Sonu Bhatia, Navneet Batra
          </td>
          <td>2024-10-24</td>
          <td>Molecular biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a839b7ca2ef04f3b61acc9f2cdd2aeccc65b7e8c" target='_blank'>
              Cells and signals of the leukemic microenvironment that support progression of T-cell acute lymphoblastic leukemia (T-ALL).
              </a>
            </td>
          <td>
            Aram Lyu, Seo Hee Nam, Ryan S. Humphrey, T. Horton, Lauren I R Ehrlich
          </td>
          <td>2024-11-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Therapy-related myeloid neoplasm (t-MN), characterized by its association with prior exposure to cytotoxic therapy, remains poorly understood and is a major impediment to long-term survival even in the era of novel targeted therapies due to its aggressive nature and treatment resistance. Previously, cytotoxic therapy–induced genomic changes in hematopoietic stem cells were considered sine qua non in pathogenesis; however, recent research demonstrates a complex interaction between acquired and hereditary genetic predispositions, along with a profoundly senescent bone marrow (BM) microenvironment. We review emerging data on t-MN risk factors and explore the intricate interplay among clonal hematopoiesis, genetic predisposition, and the abnormal BM microenvironment. Significance: t-MN represents a poorly understood blood cancer with extremely poor survival and no effective therapies. We provide a comprehensive review of recent preclinical research highlighting complex interaction among emerging therapies, hereditary and acquired genetic factors, and BM microenvironment. Understanding the risk factors associated with t-MN is crucial for clinicians, molecular pathologists, and cancer biologists to anticipate and potentially reduce its incidence in the future. Moreover, better understanding of the molecular pathogenesis of t-MN may enable preemptive screening and even intervention in high-risk patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a381c438c0051da067518ff9a38cd40bb1928975" target='_blank'>
              Therapy-Related Myeloid Neoplasms: Complex Interactions among Cytotoxic Therapies, Genetic Factors, and Aberrant Microenvironment
              </a>
            </td>
          <td>
            D. Singhal, M. Kutyna, Christopher C Hahn, M. Shah, D. Hiwase
          </td>
          <td>2024-10-18</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d7559338158829d62b6b4147b9bfe3e99b724d9" target='_blank'>
              HLA-G neo-expression modifies genetic programs governing tumor cell lines
              </a>
            </td>
          <td>
            D. Tronik-Le Roux, M. Daouya, I. Poras, François Desgrandchamps, E. Carosella
          </td>
          <td>2024-10-03</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d175f8dd810d13a4f1256bdc9f679caa427edf2" target='_blank'>
              Systematic screening of protein-coding gene expression identified VWF as a potential key regulator in anthracycline-based chemotherapy-exacerbated metastasis of breast cancer
              </a>
            </td>
          <td>
            Yawei Zhao, Meihui He, Lianzhi Cui, Min Zhang, Tianyu Zhao, Xuehan Yang, Yang Xu, Jianhua Dong, Kan He, Hansi Zhang, Li Chen
          </td>
          <td>2024-10-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d437e23d05d9c118d0d36a3629e8b0d7efbc77c" target='_blank'>
              Parallel genome-scale CRISPR-Cas9 screens uncouple human pluripotent stem cell identity versus fitness
              </a>
            </td>
          <td>
            Bess P. Rosen, Qing V. Li, Hyein S. Cho, Dingyu Liu, Dapeng Yang, Sarah Graff, Jielin Yan, Renhe Luo, Nipun Verma, Jeyaram R Damodaran, Hanuman T. Kale, Samuel J Kaplan, Michael A Beer, Simone Sidoli, Danwei Huangfu
          </td>
          <td>2024-10-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Stem cells have the potential to replace defective cells in several human diseases by depending on their self-renewal and differentiation capacities that are controlled by genes. Currently, exploring the regulation mechanism for stem cell capacities from the perspective of methyltransferase-like 3 (METTL3)-mediated N6-methyladenosine modification has obtained great advance, which functions by regulating target genes post-transcriptionally. However, reviews that interpret the regulatory network of METTL3 in stem cells are still lacking. In this review, we systematically analyze the available publications that report the role and mechanisms of METTL3 in stem cells, including embryonic stem cells, pluripotent stem cells, mesenchymal stem cells, and cancer stem cells. The analysis of such publications suggests that METTL3 controls stem cell fates and is indispensable for maintaining its normal capacities. However, its dysfunction induces various pathologies, particularly cancers. To sum up, this review suggests METTL3 as a key regulator for stem cell capacities, with further exploration potential in translational and clinical fields. In conclusion, this review promotes the understanding of how METTL3 functions in stem cells, which provides a valuable reference for further fundamental studies and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89ed73aafb376e2732c137150f513994ca26c14b" target='_blank'>
              Regulatory Network of Methyltransferase-Like 3 in Stem Cells: Mechanisms and Medical Implications
              </a>
            </td>
          <td>
            Yan Zeng, Fengyang Wang, Silu Li, Bin Song
          </td>
          <td>2024-01-01</td>
          <td>Cell Transplantation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) and Blast Crisis Chronic Myeloid Leukemia (BC CML) share several clinical characteristics, including the accumulation of immature myeloid cells and disrupted hematopoiesis. Despite the similarities, the two diseases exhibit different genetic profiles, contributing to distinct disease progressions and therapeutic outcomes. In this study, we perform a comparative analysis using bioinformatics and systems biology approaches to explore shared and unique mutational signatures, transcriptional networks, and molecular regulators in AML and BC CML. Our analysis identifies 29 genes common to both diseases, which are involved in key biological processes such as cell proliferation, cell death regulation, and transcriptional control. A subsequent pathway enrichment highlights common pathways, including the PI3K-AKT signaling and cytokine-mediated signaling, that may contribute to disease progression and resistance to therapy. Additionally, we construct a protein-protein interaction network and a transcription factor-miRNA coregulatory network, revealing key hub genes such as TP53, RUNX1, and BCL2. These genes could be potential shared therapeutic targets for treating AML and BC CML. Therefore, our analysis offers insights into the molecular underpinnings of these aggressive hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1aff1a0450bf81f4feefcd2ff55f60cc322c2c" target='_blank'>
              Shared Genetic and Regulatory Mechanisms in Acute Myeloid Leukaemia and Blast Crisis Chronic Myeloid Leukaemia
              </a>
            </td>
          <td>
            Sulagna Basu, Rakesh Pandey
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3305dbdb10abe125333dc69b5540a59698a94ae3" target='_blank'>
              Signaling pathways involved in colorectal cancer: pathogenesis and targeted therapy
              </a>
            </td>
          <td>
            Qing Li, Shan Geng, Hao Luo, Wei Wang, Ya-Qi Mo, Qing Luo, Lu Wang, Guan-Bin Song, Jian-Peng Sheng, Bo Xu
          </td>
          <td>2024-10-07</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transposable elements (TEs) are key contributors to genetic novelty. Despite increasing evidence of their importance, their roles in shaping the regulatory landscape of diverse immune cell populations remain largely unclear. Using single-cell multiome data from human peripheral blood mononuclear cells, we annotated the cell-specific cis-regulatory elements for major immune cell populations and identified a highly cell-specific signature of the overrepresented TE families. Focusing on monocytes that bear fast-evolving transcriptomes, we found that high proportions of their enhancers are TE-derived and bound by multiple pioneer transcription factors. Among them, we confirmed that the core myeloid regulator SPI1 can bind and regulate hundreds of TE-derived enhancers, which further affect the expression of adjacent immune genes. Additionally, interspecies comparison reveals that non-conserved monocyte enhancers are frequently generated by lineage-specific TE insertions, and correlate with the evolved gene expression between human and mouse. Overall, our study supports the importance of TEs in shaping the regulatory landscape of diverse immune cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec49246bea325aa9a378a750ef1fb9175032369" target='_blank'>
              Unraveling transposable element-mediated regulatory landscape in diverse human immune cells
              </a>
            </td>
          <td>
            Cui-Qiong Du, Hairui Fan, Jiayao Jiang, Juan Yang, Shuai Chen, Nadezhda E. Vorobyeva, Congrui Zhu, Liming Mao, Chenxi Li, Yanhua Li, Wen-hua Bao, Ming-an Sun
          </td>
          <td>2024-10-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background While immune checkpoint inhibitors (ICIs) are adopted as standard therapy in non-small cell lung cancer (NSCLC) patients, factors that influence variable prognosis still remain elusive. Therefore, a deeper understanding is needed of how germline variants regulate the transcriptomes of circulating immune cells in metastasis, and ultimately influence immunotherapy outcomes. Methods We collected peripheral blood mononuclear cells (PBMCs) from 73 ICI-treated NSCLC patients, conducted single-cell RNA sequencing, and called germline variants via SNP microarray. Determination of expression quantitative trait loci (eQTL) allows elucidating genetic interactions between germline variants and gene expression. Utilizing aggregation-based eQTL mapping and network analysis across eight blood cell types, we sought cell-type-specific and ICI-prognosis-dependent gene regulatory signatures. Results Our sc-eQTL analysis identified 3,616 blood- and 702 lung-cancer-specific eGenes across eight major clusters and treatment conditions, highlighting involvement of immune-related pathways. Network analysis revealed TBX21-EOMES regulons activity in CD8+ T cells and the enrichment of eQTLs in higher-centrality genes as predictive factors of ICI response. Conclusions Our findings suggest that in the circulating immune cells of NSCLC patients, transcriptomic regulation differs in a cell type- and treatment-specific manner. They further highlight the role of eQTL loci as broad controllers of ICI-prognosis-predicting gene networks. The predictive networks and identification of eQTL contributions can lead to deeper understanding and personalized ICI therapy response prediction based on germline variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70aa2fedf9166d7283b9f0e6d961d358ca399349" target='_blank'>
              Systemic CD8+ T cell effector signature predicts prognosis of lung cancer immunotherapy
              </a>
            </td>
          <td>
            Hyungtai Sim, Geun-Ho Park, Woong-Yang Park, Se-Hoon Lee, Murim Choi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction. The TP53 gene encodes the p53 protein, crucial for DNA damage response, apoptosis, and cell cycle regulation. In chronic lymphocytic leukemia (CLL), TP53 loss due to 17p deletion or mutation leads to poor chemoimmunotherapy response and shorter survival. 
Patients with unmutated immunoglobulin heavy chain variable (IGHV) status exhibit a more aggressive disease course, poorer outcomes, and reduced response to standard chemoimmunotherapy. These genetic variations significantly affect disease progression, treatment choice, and response to therapy. 
Case Report. In November 2023, a 61-year-old man presented to our clinic with complaints of generalized weakness and rapid fatigue since October 2023. CLL was diagnosed in 2018 through pathohistological and immunohistochemical analysis of the bone marrow. The patient received no therapy until 2023 due to the absence of active disease signs. From March 28, 2023, to July 25, 2023, the patient received chemoimmunotherapy with the bendamustine and rituximab (BR) regimen. Five courses of therapy were administered; however, further treatment was discontinued due to side effects that occurred during the fifth course of bendamustine administration. A subsequent diagnostic evaluation aimed at assessing treatment efficacy, prognosis, and genetic profiling included a cytomorphological analysis, which revealed that lymphocytes comprised 80% of the cellular composition. Immunocytological analysis confirmed the presence of monoclonal atypical lymphocytes, accounting for 70% of the cell population. Next-generation sequencing identified a pathogenic TP53 mutation, and fragment analysis confirmed an unmutated IGHV status. The presence of the TP53 mutation and unmutated IGHV status categorizes the patient as being in the very high-risk group, for which the use of Bruton’s tyrosine kinase inhibitors (BTKIs) and/or B-cell lymphoma-2 (BCL-2) inhibitors is recommended. 
Conclusions. This case highlights the significance of comprehensive genetic profiling in CLL patients using techniques such as fluorescence in situ hybridization, polymerase chain reaction, and next-generation sequencing. Such profiling detects molecular genetic alterations, facilitating personalized and effective treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a9a770a1a98466f2dfc1c687a0f49ae58b9afa5" target='_blank'>
              Genetic Profiling in Chronic Lymphocytic Leukemia: The Role of TP53 Mutations and IGHV Status in Treatment Decision-Making – A Case Report
              </a>
            </td>
          <td>
            Kateryna Khurdepa, Oksana Karnabeda
          </td>
          <td>2024-10-16</td>
          <td>Galician Medical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genetic alterations trigger tumorigenesis in a cell type-specific manner, but the biological basis for this, remains unclear. Here we define reinforced original cell identity established by interaction between genetic alterations and original cell type is a pivotal determinant of tumor development. Using AML-M5MLL-AF9 as a model system, we demonstrated that MLL-AF9 enhanced key transcriptional factors (TFs) expression of promonocyte and cooperated with them to amplify their downstream programs for AML hallmarks. Interactions between MLL-AF9 and promonocyte endowed cancer cells with a reinforced promonocyte identity. Reinforced intrinsic cell identity also sustained cancer hallmarks across cancer types. Furthermore, we identified narciclasine, as an effective disruptor of reinforced promonocyte identity, effectively treating AML-M5MLL-AF9. This study not only identified reinforced cell identity as a fundamental tumor driver beyond genetic mutations but also offers a proof-of-concept for reinforced identity-targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641096df61080db2589b3baf959c31f742003e7c" target='_blank'>
              Reinforced identity confers cells with cancer hallmarks
              </a>
            </td>
          <td>
            Haixia Yang, Pengqi Wang, Pengcheng Tan, Qiaomei Xue, Hua Wu, Kezhang He, Bowen Wang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The response to targeted therapies and immune checkpoint inhibitors for patients suffering from metastatic clear cell renal cell carcinoma (ccRCC) is heterogeneous and currently not predictable in clinic. In this work, a comprehensive integrated study of 700 ccRCCs profiled by DNA methylation and RNA sequencing showed that the hyper-methylated tumors exhibited a worse prognosis, a higher fraction of cycling tumor cells and a lower activity of homeobox transcription factors. To translate the use of DNA methylation information into a clinical setting, we developed a simple model accurately predicting the ccRCC methylation subtypes (AUC-ROCs of 0.91) from two gene expression ratios (IGF2BP3/PCCA, TNNT1/TMEM88). In addition, these methylation subtypes were significantly associated with the therapeutic outcome of patients to anti-PD-1, mTOR inhibitor or tyrosine kinase inhibitor therapies. Overall, our framework for predicting the ccRCC DNA methylation subtypes from targeted gene expression data is easy to translate in clinic and contributes to better personalization of ccRCC therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0521226e2f11211160c8cebde1cb06c9530046ba" target='_blank'>
              DNA methylation is a key determinant of response to targeted and immune checkpoint therapies in metastatic renal cell carcinoma
              </a>
            </td>
          <td>
            Florian Jeanneret, Sarah Schoch, Pedro Ballester, Stefan N Symeonides, Alexander Laird, Håkan Axelson, Delphine Pflieger, Christophe Battail
          </td>
          <td>2024-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a45bbf69cda262c393e7312258b75ce154e5c8c" target='_blank'>
              Beyond endocrine resistance: estrogen receptor (ESR1) activating mutations mediate chemotherapy resistance through the JNK/c-Jun MDR1 pathway in breast cancer.
              </a>
            </td>
          <td>
            Marwa Taya, Keren Merenbakh-Lamin, A. Zubkov, Z.G. Honig, Alina Kurolap, Ori Mayer, N. Shomron, Ido Wolf, T. Rubinek
          </td>
          <td>2024-10-29</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e419ef20d77c35c04caa3dddc4e91b5a8579b30f" target='_blank'>
              Recurrent somatic mutations of FAT family cadherins induce an aggressive phenotype and poor prognosis in anaplastic large cell lymphoma.
              </a>
            </td>
          <td>
            M. Villa, Geeta G Sharma, Federica Malighetti, M. Mauri, Giulia Arosio, N. Cordani, C. Lobello, Hugo Larose, A. Pirola, D. D’Aliberti, L. Massimino, Lucrezia Criscuolo, Lisa Pagani, C. Chinello, Cristina Mastini, Diletta Fontana, S. Bombelli, Raffaella Meneveri, F. Lovisa, L. Mussolin, A. Janíková, Š. Pospíšilová, S. Turner, G. Inghirami, Fulvio Magni, Mario Urso, Fabio Pagni, D. Ramazzotti, R. Piazza, Roberto Chiarle, C. Gambacorti-Passerini, Luca Mologni
          </td>
          <td>2024-10-30</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Base editing shows promise for the correction of human mutations at a higher efficiency than other repair methods and is especially attractive for mutations in large genes that are not amenable to gene augmentation therapy. Here, we demonstrate a comprehensive workflow for in vitro screening of potential therapeutic base editing targets for the USH2A gene and empirically validate the efficiency of adenine and cytosine base editor/guide combinations for correcting 35 USH2A mutations. Editing efficiency and bystander edits are compared between different target templates (plasmids versus transgenes) and assays (Next generation sequencing versus Sanger), as well as comparisons between unbiased empirical results and computational predictions. Based on these observations, practical assay recommendations are discussed. Finally, a humanized knock-in mouse model was created with the best-performing target, the nonsense mutation c.11864G>A p.(Trp3955*). Split-intein AAV9 delivery of editing reagents resulted in the restoration of USH2A protein and a correction rate of 65 ± 3% at the mutant base pair and of 52 ± 3% excluding bystander amino acid changes. This efficiency compares favorably to a prior genome editing strategy tested in the retina that completed a clinical trial and demonstrates the effectiveness of this overall strategy to identify and test base editing reagents with the potential for human therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b11e55d59e66e72cd37b7653ddc5c10089e197" target='_blank'>
              Systematic empirical evaluation of individual base editing targets: validating therapeutic targets in USH2A and comparison of methods
              </a>
            </td>
          <td>
            Yuki Tachida, K. Manian, Rossano Butcher, Jonathan M. Levy, Nachiket C. Pendse, E. Hennessey, David R. Liu, Eric A. Pierce, Qin Liu, J. Comander
          </td>
          <td>2024-10-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa5e27e413e8d0f42c28b9018464ee24128a7aa" target='_blank'>
              Unraveling the mechanisms of glioblastoma's resistance: investigating the influence of tumor suppressor p53 and non-coding RNAs.
              </a>
            </td>
          <td>
            Qamar A. Alshammari, Saud O. Alshammari, Abdulkarim Alshammari, Moaddey Alfarhan, Fahad Hassan Baali
          </td>
          <td>2024-10-30</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumour suppressor factor p53 plays an essential role in regulating numerous cellular processes, including the cell cycle, DNA repair, apoptosis, autophagy, cell metabolism and immune response. TP53 is the most commonly mutated gene in human cancers. These mutations are primarily non‑synonymous changes that produce mutant p53 proteins characterized by loss of function, a dominant negative effect on p53 tetramerisation and gain of function (GOF). GOF mutations not only disrupt the tumour‑suppressive activities of p53 but also endow the mutant proteins with new oncogenic properties. Recent studies analysing different pathogenic features of mutant p53 in cancer‑derived cell lines have demonstrated that restoring wild‑type p53, rather than removing GOF mutations, reduces cancer cell growth. These findings suggest that therapeutic strategies for reactivating wild‑type p53 function in cancer cells may bring a greater benefit than approaches halting mutant p53. This approach could involve the use of small molecules, gene therapy and other methods to re‑establish wild‑type p53 activity. This review describes the complexity of the biological activities of different p53 mutants and summarizes the current therapeutic approaches to restore p53 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48833d89250389a497918ef135eacb5f7e0504d" target='_blank'>
              TP53 mutations in cancer: Molecular features and therapeutic opportunities (Review).
              </a>
            </td>
          <td>
            M. Tornesello
          </td>
          <td>2024-10-24</td>
          <td>International journal of molecular medicine</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="DNA that is found outside the main chromosomes in a cell's nucleus is known as extrachromosomal DNA. This type of DNA is not part of the standard 23 pairs of chromosomes in humans and instead exists as standalone circular or linear DNA structures. Unlike bacterial plasmids, extrachromosomal DNA in human cells contains important genetic material and holds particular significance in cancer cells. Critical characteristics of ecDNA in Cancer is that they frequently harbour amplified versions of oncogenes such as MYC, EGFR, and CCND1. These oncogenes propel the growth and survival of tumors. Cancer cells utilize ecDNA to generate multiple copies of these oncogenes, amplifying their expression without requiring alterations to the chromosomal structure. This amplification grants the cancer cells a competitive edge in growth. ecDNA plays a crucial role in the advancement and growth of cancer, as it amplifies oncogenes, genetic diversity, and resistance to treatments. Its adaptable and ever-changing characteristics empower cancer cells to adjust to external influences such as medication. Gaining insights into ecDNA's operations and impacts could lead to new possibilities for diagnosing and treating cancer, as well as developing therapies aimed at this distinct DNA structure.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b548c36ec02cb85af82ad09081289b0df56847d0" target='_blank'>
              Extrachromosomal DNA : New Players in Oncology
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, S. Karmakar
          </td>
          <td>2024-10-01</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6bcaf46c997cceee1e5ce02e4d68cb36cec179" target='_blank'>
              Targeting mRNA-coding genes in prostate cancer using CRISPR/Cas9 technology with a special focus on androgen receptor signaling
              </a>
            </td>
          <td>
            Mobina Tabibian, Fahimeh Salasar Moghaddam, E. Motevaseli, S. Ghafouri-Fard
          </td>
          <td>2024-10-17</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a258e30c79cfe98cd4c2597aa4e57162f07486" target='_blank'>
              Emerging CRISPR Therapies for Precision Gene Editing and Modulation in the Cardiovascular Clinic.
              </a>
            </td>
          <td>
            Nicholas J Legere, J. T. Hinson
          </td>
          <td>2024-09-17</td>
          <td>Current cardiology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome 1q copy number variations, collectively termed +1q, are 1 of the most common cytogenetic abnormalities in multiple myeloma. 1q abnormalities are associated with overexpression of a high-risk gene signature promoting cell proliferation, apoptosis resistance, genomic instability, and treatment resistance, and acquisition or expansion of +1q subclones mediate disease development and relapse. While there remains significant controversy as to whether the presence of +1q is itself an independent driver of poor prognosis or is simply a marker of other high-risk features, +1q has recently been incorporated into multiple prognostic scoring models as a new high-risk cytogenetic abnormality. In this review, we present possible underlying genetic mechanisms of high-risk disease in +1q myeloma, implications for subclonal development, its role in modifying the tumor microenvironment, current evidence for clinical significance in newly-diagnosed and relapsed patients, and current controversies in +1q classification and prognostication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83d04b16ebbddbed7bed25e1883e91238060b83" target='_blank'>
              The role of 1q abnormalities in multiple myeloma: Genomic insights, clinical implications, and therapeutic challenges.
              </a>
            </td>
          <td>
            Zachary M. Avigan, C. Mitsiades, A. Laganà
          </td>
          <td>2024-10-01</td>
          <td>Seminars in hematology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Synovial Sarcoma (SySa) is an aggressive soft tissue sarcoma that accounts for 5 – 10% of all soft tissue sarcomas. Current treatment involves radiation and radical surgery including limb amputation, highlighting the urgent need to develop targeted therapies. We reasoned that transcriptional rewiring by the fusion protein SS18-SSX, the sole oncogenic driver in SySa, creates specific vulnerabilities that can be exploited for treatment. To uncover genes that are selectively essential for SySa, we mined The Cancer Dependency Map (DepMap) data to identify genes that specifically impact the fitness of SySa compared to other tumor cell lines. Targeted CRISPR library screening of SySa-selective candidates revealed that the small ubiquitin-like modifier 2 (SUMO2) was one of the strongest dependencies both in vitro as well as in vivo. TAK-981, a clinical-stage small molecule SUMO2 inhibitor potently inhibited growth and colony-forming ability. Strikingly, transcriptomic studies showed that pharmacological SUMO2 inhibition with TAK-981 treatment elicited a profound reversal of a gene expression program orchestrated by SS18-SSX fusions. Of note, genetic or pharmacological SUMO2 inhibition reduced global and chromatin levels of the SS18-SSX fusion protein with a concomitant reduction in histone 2A lysine 119 ubiquitination (H2AK119ub), an epigenetic mark that plays an important role in SySa pathogenesis. Taken together, our studies identify SUMO2 as a novel, selective vulnerability in SySa. Since SUMO2 inhibitors are currently in Phase 1/2 clinical trials for other cancers, our findings present a novel avenue for targeted treatment of synovial sarcoma. SIGNIFICANCE Our study identifies SUMO2 as a selective dependency in synovial sarcoma. We demonstrate that the SUMO2/3 inhibitor TAK-981 impairs sarcomagenesis and reverses the SS18-SSX fusion-driven oncotranscriptome. Our study indicates that SUMO2 inhibition may be an attractive therapeutic option in synovial sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1863d8b2a5acaf7a4dc2a730005a178b4452ec" target='_blank'>
              SUMO2 Inhibition Reverses Aberrant Epigenetic Rewiring Driven by Synovial Sarcoma Fusion Oncoproteins and Impairs Sarcomagenesis
              </a>
            </td>
          <td>
            Rema Iyer, Anagha Deshpande, Aditi Pedgaonkar, Pramod Akula Bala, Taehee Kim, Gerard L. Brien, D. Finlay, K. Vuori, A. Soragni, Rabi Murad, Aniruddha J. Deshpande
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="BACKGROUND
Cancer is regarded as one of the main causes of death globally. Future predictions indicate that the death rate from cancer will keep rising, which may reach 11.4 million in 2030. Carcinogenesis refers to the phenomenon of transforming a normal cell into a cancer cell. Cancer is characterized by unregulated and uncontrolled cell division due to alterations at the molecular and genetic levels. Gene mutations can speed up the rate of cell division, which leads to cancer. Metastasis entails the dissemination of cancer cells from the primary site to distant regions of the body via the circulatory or lymphatic systems.


OBJECTIVE
This review is mainly focusing on the anticancer properties of terpenoids. In the case of human beings, several types of cancers can be treated clinically based on the form and phase of the cancer. So, there are different types of treatment regimens available for the management of cancer, such as immunotherapy, hormonal therapy, radiation therapy, and chemotherapy.


METHODS
Several problems are associated with cancer therapy, including chemoresistance, severe toxicity, relapse, and metastasis. To minimize these complications, natural products like terpenoids seem to be beneficial for the effective management of cancer.


RESULTS
Experimental results revealed that the anticancer potential of terpenoids is due to activation of apoptosis and stimulation of cell cycle arrest. Some of the terpenoids exhibit anticancer effects by inhibiting angiogenesis and metastasis via the regulation of several signaling pathways intracellularly. Certain terpenoids have been shown to work in concert with anticancer medications (doxorubicin, cisplatin, paclitaxel, and 5-fluorouracil) to provide synergistic effects. These terpenoids have also been shown to be effective against cancer cells that are resistant to several drug therapies.


CONCLUSION
The current study will focus on signaling pathways and mode of action of several types of terpenoids as anticancer agents. Further, it will provide insights into the ongoing clinical trials and prospective pathways for the advancement of terpenoids as possible anti-cancer agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e8bfa768606774f1a7a769cf7804a33f08d22f" target='_blank'>
              Current Insights into Therapeutic Potential of Terpenoids as Anticancer Agents.
              </a>
            </td>
          <td>
            B. Sahoo, B. Banik, Shikha Sharma, Bhupendra Singh
          </td>
          <td>2024-10-22</td>
          <td>Anti-cancer agents in medicinal chemistry</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Despite a critical role for tumor-initiating cancer stem cells (CSCs) in breast cancer progression, major questions remain about the properties and signaling pathways essential for their function. Recent discoveries highlighting mechanisms of CSC-resistance to the stress caused by chromosomal instability (CIN) may provide valuable new insight into the underlying forces driving stemness properties. While stress tolerance is a well-known attribute of CSCs, CIN-induced stress is distinctive since levels appear to increase during tumor initiation and metastasis. These dynamic changes in CIN levels may serve as a barrier constraining the effects of non-CSCs and shaping the stemness landscape during the early stages of disease progression. In contrast to most other stresses, CIN can also paradoxically activate pro-tumorigenic antiviral signaling. Though seemingly contradictory, this may indicate that mechanisms of CIN tolerance and pro-tumorigenic inflammatory signaling closely collaborate to define the CSC state. Together, these unique features may form the basis for a critical relationship between CIN and stemness properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d491eba9730d9558923c4e53a3264dd576b3117a" target='_blank'>
              Chromosomal instability as an architect of the cancer stemness landscape
              </a>
            </td>
          <td>
            Shahnawaz A. Baba, Aran Zakeri, J. Desgrosellier
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Metabolic reprogramming is one of the characteristics of tumor cell metabolism. In tumor cells, there are multiple metabolic enzymes and membrane proteins to regulate metabolic reprogramming, and hypoxia inducible factor-1α (HIF-1α) can be regulated in transcription, translation, posttranslational modification and other aspects through multiple pathways, and HIF-1α affects multiple metabolic enzymes and membrane proteins during metabolic reprogramming, thus playing a central role in the metabolic reprogramming process, and thus has some implications for tumor therapy and understanding chemotherapy drug resistance. HIF-1α affects a number of metabolic enzymes and membrane proteins in the metabolic reprogramming process, thus playing a central role in the metabolic reprogramming process, which has certain significance for the treatment of tumors and the understanding of chemotherapeutic drug resistance. In this paper, we review the central role of HIF-1α in metabolic reprogramming, chemotherapeutic agents targeting HIF-1α, and chemotherapeutic drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23d4a3651269b2495c2043daecd406a09fde2e85" target='_blank'>
              Central role of hypoxia-inducible factor-1α in metabolic reprogramming of cancer cells: A review
              </a>
            </td>
          <td>
            Bing Zhu, Lichao Cheng, Baosu Huang, Runzhi Liu, Bin Ren
          </td>
          <td>2024-11-01</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatocellular adenomas (HAs) are tumors that can develop under different conditions, including in patients harboring a germline mutation in HNF1A. However, little is known about the pathogenesis of such disease. This work aims to better define what mechanisms lie under the development of this condition. Six HAs were sampled from the liver of a 17-year-old male affected by diabetes and multiple hepatic adenomatosis harboring the heterozygous pathogenic germline variant c.815G>A, p.(Arg272His) in HNF1A, which has a dominant negative effect. All HAs were molecularly characterized. Four of them were shown to harbor a second somatic HNF1A variant and one had a mutation in the ARID1A gene, while no additional somatic changes were found in the remaining HA and normal parenchyma. A transcriptomic profile of the same HA samples was also performed. HNF1A biallelic mutations were associated with the up-regulation of several pathways including the tricarboxylic acid cycle, the metabolism of fatty acids, and mTOR signaling while angiogenesis, endothelial and vascular proliferation, cell migration/adhesion, and immune response were down-regulated. Contrariwise, in the tumor harboring the ARID1A variant, angiogenesis was up-modulated while fatty acid metabolism was down-modulated. Histological analyses confirmed the molecular data. Independently of the second mutation, energetic processes and cholesterol metabolism were up-modulated, while the immune response was down-modulated. This work provides a complete molecular signature of HNF1A-associated HAs, analyzing the association between specific HNF1A variants and the development of HA while identifying potential new therapeutic targets for non-surgical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26474672d542f89d0fa38cb2c1556ad93882621f" target='_blank'>
              Genomic and Transcriptomic Profile of HNF1A-Mutated Liver Adenomas Highlights Molecular Signature and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            A. Faini, F. Arruga, M. Pinon, Valeria Bracciamà, Francesco Edoardo Vallone, Fiorenza Mioli, M. Sorbini, Martina Migliorero, A. Gambella, Damiano Carota, Isaac Giraudo, P. Cassoni, Silvia Catalano, Renato Romagnoli, Antonio Amoroso, P. L. Calvo, T. Vaisitti, Silvia Deaglio
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Genome editing technology is an emerging toolbox that can specifically target genes. At present, the existing gene editing technologies include ZFN, TALEN, and CRISPR system editing technology with clusters of regularly spaced short palindromic repeats. However, ZFN and TALEN are no longer the preferred tools for gene editing due to some of their problems, and their complex protein design, high cost, and high difficulty are the main reasons that limit their use. The CRISPR system, on the other hand, is able to achieve specific binding by base complementary pairing of sgRNA to the target sequence. Traditional cancer treatment methods have limited efficacy and high recurrence rates, and the disease can be fundamentally treated by inactivating oncogenes or activating tumor suppressor genes through gene editing technology and then changing downstream signaling pathways for cancer treatment. With CRISPR-Cas9 technology, it is possible to have gene knockouts, insertions, and mutations. Here, we will investigate and explore the feasibility of the CRISPR system for tumor treatment, including the following three topics: oncogene knockout, augmentation of tumor suppressor gene expression, and enhancement of engineered T cell viability for effective tumor Through these methods, the CRISPR system is of great help and promising tumor therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa5e9e8d18667744c4c9905968635b74973ccacc" target='_blank'>
              Advances in CRISPR-Cas9 Technology for Tumor Therapy
              </a>
            </td>
          <td>
            Haochuan Xu
          </td>
          <td>2024-10-29</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to therapy remains a significant challenge in cancer treatment, often due to the presence of a stem-like cell population that drives tumor recurrence post-treatment. Moreover, many anticancer drugs inadvertently induce plasticity, reverting differentiated cancer cells to drug-resistant stem-like states. Addressing this phenomenon is crucial for advancing cancer therapeutics. In this study, we introduce a robust statistical framework based on multi-type branching processes that can dissect tumor dynamics and drug effects from high throughput drug screening data. Through comprehensive in silico experiments, we show the efficacy of our framework in estimating parameters governing population dynamics and drug responses in a heterogeneous tumor population where cell state transitions are influenced by the drug. Finally, using recent in vitro data involving AGS-SORE6+/- cells treated with ciclopirox olamine, we show how our framework can be used to confirm the presence of drug-induced cell plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85013788a6ef1ec3c633fabb86b8c0544b0624bd" target='_blank'>
              Inferring Drug-induced Plasticity via Drug Screen Data
              </a>
            </td>
          <td>
            Chenyu Wu, E. B. Gunnarsson, J. Foo, K. Leder
          </td>
          <td>2024-09-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f34d93ff1b305ad73729aa0a43e0b0a160605f" target='_blank'>
              Characterization of MYC Rearrangements in Multiple Myeloma: an Optical Genome Mapping Approach
              </a>
            </td>
          <td>
            Jung Yoon, Taesung Jeon, J. Kwon, Soo-Young Yoon
          </td>
          <td>2024-09-20</td>
          <td>Blood Cancer Journal</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) constitute a prominent immune cell population within various solid cancers, playing a pivotal role in disease progression. Their increased numbers and frequencies often strongly correlate with resistance to therapy and reduced overall survival rates. Within the complex ecosystem of the tumor microenvironment (TME), activated cancer-associated fibroblasts (CAFs) are expanded and contribute significantly to tumor growth and metastasis, and to chemo- or immune-therapy resistance. CAFs exert a critical influence on TAM phenotype and functions by orchestrating the reprogramming of tissue-infiltrating monocytes, thereby modulating their survival and differentiation. This reciprocal interaction between TAMs and CAFs forms a crucial axis in fostering a suppressive crosstalk within the TME, mediated by a diverse array of signals exchanged between these cell types. Recent advancements in single-cell RNA sequencing (sc-RNA seq) technologies and spatial transcriptomics have enhanced our comprehension of the signaling dynamics at the interface between TAMs and CAFs, including their spatial distribution within the tissue. In this review, we delve into the latest discoveries elucidating the biology of TAM and CAF crosstalk. We examine the complexity of TAM-CAF and CAF-TAM interactions within the TME of solid cancers, with particular focus on ligand-receptor interactions and clinically significant targets for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba817d130671717874a36642fd784b7dbabebd41" target='_blank'>
              At the interface of tumor-associated macrophages and fibroblasts: immune-suppressive networks and emerging exploitable targets.
              </a>
            </td>
          <td>
            E. Timperi, Hugo Croizer, Darya Khantakova, Mhaidly Rana, Martina Molgora, Jennifer L. Guerriero, F. Mechta-Grigoriou, Emanuela Romano
          </td>
          <td>2024-09-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) poses significant challenges in veterinary medicine, with limited treatment options and poor survival rates. While substantial progress has been made in characterizing human AML, translating these advancements to veterinary practice has been hindered by limited molecular understanding and diagnostic tools. The case study presented illustrates the application of whole genome sequencing in diagnosing AML in a dog, showcasing its potential in veterinary oncology. Our approach facilitated comprehensive genomic analysis, identifying mutations in genes that may be associated with AML pathogenesis in dogs, such as KRAS, IKZF1, and RUNX1. However, without supportive evidence of its clinical utility (eg, association with response to treatment or prognosis), the information is limited to exploration. This article reviews the comparative features of canine AML with human AML and discusses strategies to shrink the knowledge gap between human and veterinary medicine with cost-effective next-generation sequencing (NGS) techniques. By utilizing these approaches, the unique and shared molecular features with human AML can be identified, aiding in molecular classification and therapeutic development for both species. Despite the promise of NGS, challenges exist in implementing it into routine veterinary diagnostics. Cost considerations, turnaround times, and the need for robust bioinformatics pipelines and quality control measures must be addressed. Most importantly, analytical and clinical validation processes are essential to ensure the reliability and clinical utility of NGS-based assays. Overall, integrating NGS technologies into veterinary oncology holds great potential for advancing our understanding of AML and improving disease stratification, in hopes of improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8bd6fdd9b3631dd5e67d1617f989917515a9291" target='_blank'>
              Advancements in genetic analysis: Insights from a case study and review of next-generation sequencing techniques for veterinary oncology applications.
              </a>
            </td>
          <td>
            R. Harris, Jillian Nolan, Dylan Ammons, Samantha Beeson, Douglas Thamm, Anne Avery
          </td>
          <td>2024-10-04</td>
          <td>Veterinary clinical pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BTK inhibitor therapy induces peripheral blood lymphocytosis in chronic lymphocytic leukemia (CLL) lasting for several months. It remains unclear whether non-genetic adaptation mechanisms exist, allowing CLL cells' survival during BTK inhibitor-induced lymphocytosis and/or playing a role in therapy resistance. We show that in approximately 70 % of CLL cases, ibrutinib treatment in vivo increases Akt activity above pre-therapy levels within several weeks, leading to compensatory CLL cell survival and a more prominent lymphocytosis on therapy. Ibrutinib-induced Akt phosphorylation (pAktS473) is caused by the upregulation of FoxO1 transcription factor, which induces expression of Rictor, an assembly protein for mTORC2 protein complex that directly phosphorylates Akt at serine 473 (S473). Knock-out or inhibition of FoxO1 or Rictor led to a dramatic decrease in Akt phosphorylation and growth disadvantage for malignant B cells in the presence of ibrutinib (or PI3K inhibitor idelalisib) in vitro and in vivo. FoxO1/Rictor/pAktS473 axis represents an early non-genetic adaptation to BCR inhibitor therapy not requiring PI3Kδ or BTK kinase activity. We further demonstrate that FoxO1 can be targeted therapeutically, and its inhibition induces CLL cells' apoptosis alone or in combination with BTK inhibitors (ibrutinib, acalabrutinib, pirtobrutinib) and blocks their proliferation triggered by T-cell factors (CD40L, IL-4, and IL-21).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9196e36f3efd04dcada6ed6f664575c51279c08" target='_blank'>
              FoxO1/Rictor axis induces a non-genetic adaptation to Ibrutinib via Akt activation in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            L. Ondrisova, V. Seda, K. Hlavac, Petra Pavelkova, E. Hoferkova, G. Chiodin, L. Košťálová, Gabriela Mladonicka Pavlasova, D. Filip, Josef Vecera, P. F. Zeni, J. Oppelt, Z. Kahounová, Ráchel Víchová, Karel Souček, A. Panovska, K. Plevová, Š. Pospíšilová, M. Šimkovič, F. Vrbacký, Daniel Lysák, Stacey M. Fernandes, M. Davids, A. Maiques-Diaz, S. Charalampopoulou, J. Martín-Subero, Jennifer R. Brown, M. Doubek, Francesco Forconi, J. Mayer, M. Mraz
          </td>
          <td>2024-10-22</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Froese, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel Goulet, Michael T. Hemann. Elucidating the tumor-driven mechanisms of resistance to immunotherapies in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr B039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81348949cfc84943625452ad98cf593484f76b30" target='_blank'>
              Abstract B039: Elucidating the tumor-driven mechanisms of resistance to immunotherapies in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Froese, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel Goulet, Michael Hemann
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Several therapeutic agents have been approved for treating multiple myeloma (MM), a cancer of bone marrow resident plasma cells. Predictive biomarkers for drug response could help guide clinical strategies to optimize outcomes. Here, we present an integrated functional genomic analysis of tumor samples from MM patients that were assessed for their ex vivo drug sensitivity to 37 drugs, clinical variables, cytogenetics, mutational profiles, and transcriptomes. This analysis revealed a MM transcriptomic topology that generates "footprints" in association with ex vivo drug sensitivity that have both predictive and mechanistic applications. Validation of the transcriptomic footprints for the anti-CD38 monoclonal antibody daratumumab and the nuclear export inhibitor selinexor demonstrated that these footprints can accurately classify clinical responses. The analysis further revealed that daratumumab and selinexor have anti-correlated mechanisms of resistance, and treatment with a selinexor-based regimen immediately after a daratumumab-containing regimen was associated with improved survival in three independent clinical trials, supporting an evolutionary-based strategy involving sequential therapy. These findings suggest that this unique repository and computational framework can be leveraged to inform underlying biology and to identify therapeutic strategies to improve treatment of MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b324ac9181f2db70d574620d1a3e7ec5783d8859" target='_blank'>
              The Functional Transcriptomic Landscape Informs Therapeutic Strategies in Multiple Myeloma.
              </a>
            </td>
          <td>
            P. Sudalagunta, R. Canevarolo, M. Meads, Maria Silva, Xiaohong Zhao, Christopher L Cubitt, S. Sansil, Gabriel Deavila, R. Alugubelli, Ryan T Bishop, Alexandre Tungesvik, Qi Zhang, Oliver Hampton, J. Teer, Eric A Welsh, Sean J Yoder, Bijal D Shah, Lori Hazlehurst, Robert A Gatenby, Dane R. Van Domelen, Yi Chai, Feng Wang, Andrew DeCastro, Amanda M Bloomer, E. Siegel, Conor C Lynch, Daniel M Sullivan, Melissa Alsina, T. Nishihori, J. Brayer, John L Cleveland, William S. Dalton, C. Walker, Y. Landesman, Rachid Baz, A. Silva, K. Shain
          </td>
          <td>2024-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="OBJECTIVES
The TP53 mutation, a prevalent tumor suppressor gene alteration, is linked to chemotherapy resistance, increased relapse rates and diminished overall survival (OS) in acute myeloid leukemia (AML) patients.


METHODS
In this study, we characterize the TP53 mutation phenotypes across various AML cohorts utilizing The Cancer Genome Atlas (TCGA) data. We devised a TP53-related prognostic signature derived from differentially expressed genes between mutated and wild-type TP53 AML specimens. In-depth analyses were conducted, encompassing genetic variation, immune cell infiltration and prognostic stratification.


RESULTS
A six-gene TP53-related signature was established using least absolute shrinkage and selection operator (LASSO)-Cox regression, demonstrating robust prognostic predictability. This signature exhibited strong performance in both the OHSU validation cohorts, an independent Gene Expression Omnibus (GEO) validation cohort (GSE71014) and proved by results of the in vivo experiment. Finally, we used single cell database (GSE198681) to observe the characteristics of these six genes.


DISCUSSION
Our study may facilitate the development of efficacious therapeutic approaches and provide a novel idea for future research. Conclusion: The TP53-related signature and pattern hold the potential to refine prognostic stratification and underscore emerging targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57809ff1c4369159931bf563a0e583afd0ffd0c" target='_blank'>
              Elucidating the immune landscape and potential prognostic model in acute myeloid leukemia with TP53 mutation.
              </a>
            </td>
          <td>
            Gelan Zhu, J. Cai, Wanbin Fu, Yue Sun, Ting Wang, Hua Zhong
          </td>
          <td>2024-09-27</td>
          <td>Hematology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Neuroendocrine transdifferentiation (NEtD), also commonly referred to as lineage plasticity, emerges as an acquired resistance mechanism to molecular targeted therapies in multiple cancer types, predominately occurs in metastatic epidermal growth factor receptor (EGFR)‐mutant non‐small cell lung cancer treated with EGFR tyrosine kinase inhibitors and metastatic castration‐resistant prostate cancer treated with androgen receptor targeting therapies. NEtD tumors are the lethal cancer histologic subtype with unfavorable prognosis and limited treatment. A comprehensive understanding of molecular mechanism underlying targeted‐induced plasticity could greatly facilitate the development of novel therapies. In the past few years, increasingly elegant studies indicated that NEtD tumors share key the convergent genomic and phenotypic characteristics irrespective of their site of origin, but also embrace distinct change and function of molecular mechanisms. In this review, we provide a comprehensive overview of the current understanding of molecular mechanism in regulating the NEtD, including genetic alterations, DNA methylation, histone modifications, dysregulated noncoding RNA, lineage‐specific transcription factors regulation, and other proteomic alterations. We also provide the current management of targeted therapies in clinical and preclinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7693a38c0d6b0d0a5a68e9cf1e1598b353f12ed8" target='_blank'>
              Neuroendocrine transdifferentiation in human cancer: molecular mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Jun Jiang, Donghui Han, Jiawei Wang, Weihong Wen, Rui Zhang, Weijun Qin
          </td>
          <td>2024-10-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Multiple myeloma is a plasma cell malignancy that accounts for 1-2% of all cancer diagnoses and 10-15% of hematopoietic neoplasms. Clonal rearrangement of the IG locus suggests a monoclonal model for pathogenesis. The presence of cytogenetic abnormalities in >90% of cases suggests transformation by chromosomal instability and rearrangement. Classification and consensus guidelines by The International Myeloma Working Group (IMWG) and others reflect this by categorizing myeloma based on ploidy, translocations, and chromosomal abnormalities.
 Interestingly, a portion of Emory’s test volume for cancer gene mutation profiling by second generation short-read sequencing is for multiple myeloma cases. This suggests our 75-gene myeloid neoplasm panel (MMP75) is being utilized for medical purposes that are not met by cytogenetic and microarray characterizations performed during the diagnostic workup of multiple myeloma. Inspired by this observation, this study investigates providers motivations for ordering MMP75 and assesses the adequacy of our current implementation for their stated purposes.
 Anecdotes from ordering physicians indicate interest in MMP75 results when informing prognosis, selecting treatments, and evaluating for treatment associated secondary neoplasms. A more formal survey is underway that will statistically capture how the results of MMP75 testing were ultimately used: to inform diagnosis, determine prognosis, guide treatment, explain progression, identify/rule-out secondary neoplasia, or no effect on management.
 We also explore the variants called by MMP75 when performed using bone marrow from patients who received a cytogenetic workup for multiple myeloma at some point. A retrospective analysis of the 5-year interval between Jan 2019 and Jan 2024 identified 156 cases meeting this criterion. The number of pathogenic variants per case ranged between 1 and 8, with mutations detected across 42 genes. The genes most frequently identified with pathogenic mutations were TP53 (22.4%), DNMT3A (19.2%), TET2 (14.7%), and ASXL1 (12.2%). All four have been associated with adverse outcomes in patients with multiple myeloma. Activating events in therapeutic targets included mutations in the RAS/RAF pathway (16.7%), JAK/STAT signaling (2.6%), and the kinase KIT (0.6%).
 While these findings support the use of MMP75 for detection of relevant mutations, the genetic variants identified are not unique to multiple myeloma. Mutations detected in DNMT3A, TET2, and ASXL1 are common in clonal hematopoiesis of indeterminant potential (CHIP) and myeloid neoplasms. Hotspot mutations in TP53 and activating events in the RAS/RAF pathway also occur in various hematologic malignancies. This test therefore detects events that occur in both the plasma cell compartment and other cells of origin. This encumbers distinguishing variants that inform on a patient’s multiple myeloma from events that suggest a secondary neoplasm. Future directions include determining whether the effect of plasma cell enrichment on a mutations variant allele frequency (VAF) in repeat testing can help distinguish true multiple myeloma variants from events in other cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/987b88a74ed80e2065d8c8a3c7edc9ab6f63f8af" target='_blank'>
              Improving the Utility of a Cancer Gene Mutation Panel for Multiple Myeloma Patients
              </a>
            </td>
          <td>
            Lalit Patel, Linsheng Zhang, Geoffrey Smith, David L. Jaye, Thomas Schneider
          </td>
          <td>2024-10-01</td>
          <td>American Journal of Clinical Pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a93850c21d9d18806483d79ec8a05dda4a648d" target='_blank'>
              Treatment resistance to melanoma therapeutics on a single cell level
              </a>
            </td>
          <td>
            Lijun Yao, B. Krasnick, Ye Bi, Sunantha Sethuraman, S. Goedegebuure, A. Weerasinghe, Christopher Wetzel, Qingsong Gao, A. Oyedeji, Jacqueline L Mudd, Matthew A. Wyczalkowski, M. Wendl, Li Ding, Ryan C. Fields
          </td>
          <td>2024-09-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Dysfunction of epigenetic modulators, such as the SWI/SNF complex, is a wide-spread but relatively ill-defined feature of a broad spectrum of cancer entities. Among SWI/SNF-mutant entities, SMARCB1-deficient cancers, such as the highly aggressive Epithelioid Sarcoma (EpS), are characterized by this genetic event in an otherwise rather silent mutational landscape. This renders EpS an ideal model to study how epigenetic reprogramming by a single mutation can contribute to tumorigenesis. Hence, to characterize and compare the function of the SMARCB1-deficient, residual and the physiological SWI/SNF complex in cancer, we generated a panel of SMARCB1 re-expressing EpS cell lines and employed a functional multi-omics approach. Here, we show that SWI/SNF holds canonical characteristics of both tumor-suppressors and proto-oncogenes due to its multi-faceted role in the regulation of the epigenome. Our data indicates that the loss of SMARCB1 causes an overall loss of SWI/SNF chromatin affinity at cis-regulatory enhancer elements, inducing a preference for uncontrolled proliferation and cell cycle progression as opposed to development and differentiation. We further demonstrate that EpS cell lines depend on residual SWI/SNF action to maintain clonogenicity and proliferation. Consequently, EpS cell lines exhibit markedly increased sensitivity to pharmacological inhibition of the residual SWI/SNF when compared with SWI/SNF-proficient cancer entities. Collectively, our results from the EpS model shed new light on how a single mutation can rewire the pleiotropic effects of an epigenetic master regulator and provide inroads for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba81f1fa7d5881ab59ab9ddb07d9a1d9290af17" target='_blank'>
              Loss of SMARCB1 evokes targetable epigenetic vulnerabilities in Epithelioid Sarcoma
              </a>
            </td>
          <td>
            Jia Xiang Jin, Fabia Fuchslocher, Martha J. Carreño-Gonzalez, Felina Zahnow, A. K. Ceranski, Rainer Will, Dominic Helm, Felix Bestvater, A. Banito, R. Imle, S. Ohmura, F. Cidre-Aranaz, T. G. Grünewald
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="In this review article we will highlight the evidences that how oncogenes are formed due to the physical genetic variations in proto-oncogenes and tumor suppressor genes and various planned immunotherapies which will include- The immune checkpoint inhibitor-opposing antibodies, adoptive cell treatments, and biologic modifiers (cytokines and vaccines). We will make an effort to provide guidance and potential fixes for these issues, along with pertinent sources for foundational research. For suitable studies, a literature search was undertaken from various database sources such as PubMed, EMBASE, and Google Scholar. One type of gene known as an oncogene-a cellular gene that becomes dysfunctional owing to mutation and overexpression-is the cause of cancer. Certain oncogenes seem to inhibit the homeostatic mechanism by limiting the single cell lineage of leukemia stem cells. According to the clonal theory of oncogenes, tumors are thought to begin in a single cell, Moreover, the growth of tumors is closely linked to the prevention of apoptosis, or programmed cell death. These activities of oncogene can be minimized by some immunological therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77360e601c608aabfa1825689be8b174a0154240" target='_blank'>
              Oncogenetics: Unraveling the Genetic Underpinnings of Cancer for Improved Immunotherapeutic Outcomes.
              </a>
            </td>
          <td>
            Stuti Dwivedi, Praveencumar R, T. Sivakumar, M. K. Posa, R. Dhakar, Ruchi Tiwari
          </td>
          <td>2024-10-29</td>
          <td>Zhongguo ying yong sheng li xue za zhi = Zhongguo yingyong shenglixue zazhi = Chinese journal of applied physiology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The effectiveness of drug treatments is profoundly influenced by individual responses, which are shaped by gene expression variability, particularly within pharmacogenes. Leveraging single-cell RNA sequencing (scRNA-seq) data, our study explores the extent of expression variability among pharmacogenes in a wide array of cell types across eight different human tissues, shedding light on their impact on drug responses. Our findings broaden the established link between variability in pharmacogene expression and drug efficacy to encompass variability at the cellular level. Moreover, we unveil a promising approach to enhance drug efficacy prediction. This is achieved by leveraging a combination of cross-cell and cross-individual pharmacogene expression variation measurements. Our study opens avenues for more precise forecasting of drug performance, facilitating tailored and more effective treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b12a7bcbec6921ec291037e369d1c317c0110c" target='_blank'>
              Deciphering single-cell gene expression variability and its role in drug response.
              </a>
            </td>
          <td>
            Sizhe Liu, Liang Chen
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/926bbd8c2f19d372355266fd304f5335584a732e" target='_blank'>
              Identification and genetic dissection of convergent persister cell states.
              </a>
            </td>
          <td>
            S. Blattman, Wenyan Jiang, E. R. McGarrigle, Menghan Liu, P. Oikonomou, Saeed Tavazoie
          </td>
          <td>2024-11-06</td>
          <td>Nature</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Epigenetic reinforcement of T cell exhaustion is known to be a major barrier limiting T cell responses during immunotherapy. However, the core epigenetic regulators restricting antitumor immunity during prolonged antigen exposure are not clear. We investigated three commonly mutated epigenetic regulators that promote clonal hematopoiesis to determine whether they affect T cell stemness and response to checkpoint blockade immunotherapy. CD8 T cells lacking Dnmt3a, Tet2, or Asxl1 preserved a progenitor-exhausted (Tpex) population for more than 1 year during chronic antigen exposure without undergoing malignant transformation. Asxl1 controlled the self-renewal capacity of T cells and reduced CD8 T cell differentiation through H2AK119 ubiquitination and epigenetic modification of the polycomb group–repressive deubiquitinase pathway. Asxl1-deficient T cells synergized with anti–PD-L1 immunotherapy to improve tumor control in experimental models and conferred a survival advantage to mutated T cells from treated patients. Editor’s summary T cell immunotherapies have been successful in treating some intractable cancers, but there are limitations caused by the inability of T cells to persist after prolonged stimulation. Kang et al. undertook a reverse-translation approach to investigate the molecular mechanisms regulating the durability of T cell responsiveness to checkpoint blockade immunotherapy (see the Perspective by Tsui and Kallies). Inspired by clinical observations from individuals with myelodysplastic syndrome, the epigenetic regulators (Dnmt3a, Tet2, and Asxl1) associated with clonal hematopoiesis were identified using experimental models of T cell exhaustion. Epigenetic disruption of the polycomb group repressive deubiquitinase complex resulted in preservation of a stem cell–like T cell population that was responsive to immunotherapy. Extending this mechanism to cancer adoptive cell therapy, Asxl1 disruption was found to endow T cells with superior therapeutic efficacy and an ability to synergize with checkpoint blockade immunotherapy —Priscilla N. Kelly INTRODUCTION Sustained stimulation of CD8 T cells promotes the development of functional exhaustion, a terminal cell fate that has been broadly recognized to limit the durability of T cell–based immunotherapies. This state of exhaustion arises though a progressive transition of multipotent progenitor T cells (Tpex) into a terminally differentiated population (Tex) that has been defined as being nonresponsive to immune checkpoint blockade (ICB). The molecular determinants dictating the developmental transition are now being investigated as engineering targets to block the progression of T cell exhaustion and preserve the durability of immunotherapy approaches. RATIONALE We recently observed that the long-term survival of a small cohort of myelodysplastic syndrome (MDS) patients treated with anti–PD-L1 therapy was coupled to mutation of the ASXL1 gene among their T cells. Mutations in ASXL1, in addition to DNMT3A and TET2, are commonly associated with conferring a survival advantage among hematopoietic stem cells, resulting in subclonal outgrowth referred to as clonal hematopoiesis. Because of their general link to cellular stemness, we investigated the role of these regulators in the developmental transition of Tpex into Tex and the impact of deletion of these regulators on therapeutic durability during anti–PD-L1 treatment. RESULTS To investigate the role of Dnmt3a, Tet2, and Asxl1 in the development and maintenance of an ICB-responsive T cell population, we engineered T cells to contain mutations within each of these genes and then exposed them to a chronic source of antigen by adoptively transferring them into mice infected with the clone 13 strain of lymphocytic choriomeningitis virus (LCMV). Disruption of these regulators in a canonical model of T cell exhaustion allowed them to maintain their quantity and ICB-responsive state for >1 year during chronic antigen exposure. Despite being stimulated and undergoing antigen-driven proliferation for 1 year, these T cells did not exhibit any signs of malignant outgrowth. Moreover, the total pool of T cells containing each of these mutations was enriched for a large quantity of stem-like Tcf1+ T cells. Fate-tracking adoptive transfer experiments revealed that this Tcf1+ subset of ASXL1 KO T cells could both self-renew and give rise to potent effectors. Transcriptional and epigenetic profiling of the knockout (KO) T cells identified Asxl1 as a regulator of the polycomb group–repressive deubiquitinase (PR-DUB) complex, controlling the deubiquitination of histone 2A lysine 119, which serves as a molecular checkpoint for the developmental transition of Tpex into Tex. Extension of this mechanistic insight into tumor models revealed that adoptively transferred T cells with Asxl1 disruption resisted tumor microenvironment–induced exhaustion. Moreover, the adoptively transferred Asxl1 KO T cells synergized with anti–PD-L1 blockade, resulting in heightened tumor control. Finally, we show that Asxl1 disruption of tumor-specific T cells correlates with increased expansion of T cells in MDS patients after treatment with anti–PD-L1. CONCLUSION Results from these studies show that DNMT3A, TET2, and ASXL1 control the developmental checkpoint between Tpex and terminally exhausted T cells. Specific investigation into Asxl1 revealed its role in T cell differentiation through epigenetic modification of the PR-DUB pathway. These data articulate a reverse-translation approach for defining molecular mechanisms that restrict maintenance of the ICB-responsive stem-like T cells within a suppressive tumor microenvironment. Through our mechanistic investigation of genes associated with patient survival, we have established a scientific rationale for specific genetic modification of discrete epigenetic regulators for future engineering to enhance the durability of cancer immunotherapy. Core epigenetic regulators in maintaining therapeutic durable multipotent Tpex populations. Using the canonical murine model system of exhaustion, disruption of Dnmt3a, Tet2, and Asxl1 in CD8 T cells preserved an ICB-responsive Tpex population for >1 year during chronic antigen exposure. Specific investigation into the new epigenetic regulator Asxl1 revealed that this unprecedented maintenance of Tpex is mediated by presence of monoubiqutinated histone 2A lysine 119 (H2AK119Ub) through disruption of the PR-DUB complex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58e5351a39641c751005cc32a06480cddb05203" target='_blank'>
              Epigenetic regulators of clonal hematopoiesis control CD8 T cell stemness during immunotherapy
              </a>
            </td>
          <td>
            Tae Gun Kang, Xin Lan, Tian Mi, Hongfeng Chen, Shanta Alli, Song-Eun Lim, Sheetal Bhatara, A. Vasandan, Grace Ward, Sofia Bentivegna, Josh Jang, Marianne L. Spatz, Jin-Hwan Han, B. Schlotmann, J. S. Jespersen, Christopher Derenzo, Peter Vogel, Jiyang Yu, Stephen B Baylin, P. Jones, Casey O’Connell, Kirsten Grønbæk, Ben Youngblood, Caitlin C. Zebley
          </td>
          <td>2024-10-11</td>
          <td>Science</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="High expression of the long non-coding RNA (lncRNA) FAM30A has been previously associated with leukemic stem cell (LSC) activity and poor prognosis in both adult and paediatric acute myeloid leukaemia (AML) patients, yet it has not been functionally studied. This study provides the first cellular characterization of FAM30A focussing on an internal tandemly organised region, referred to as FAM30A repeats. FAM30A levels correlated with canonical AML LSC signatures and FAM30A depletion decreased cell viability as well as increased sensitivity to chemotherapeutics. It also inhibited colony formation, promoted granulocytic differentiation and abrogated leukemic engraftment in murine bone marrow in vivo. Overexpression of FAM30A repeats in this setting enhanced stemness, proliferation, chemoresistance, and engraftment thus highlighting the biological relevance of this region for LSC biology. On the molecular level, FAM30A repeats interact with the pro-LSC regulator Musashi-2 (MSI2), positively influencing expression of its targets including RUNX1 isoforms. We herein uncover that this FAM30A-MSI2-RUNX1 regulatory loop is of potential relevance for LSC maintenance in AML. These findings provide valuable insights into FAM30A’s cellular role and highlight its targeting potential for eliminating LSCs and improving treatment outcomes in AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e944155596f6ba606bda7e6372bd8e29a902812" target='_blank'>
              The long non-coding RNA FAM30A regulates the Musashi2-RUNX1 axis and is required for LSC function in AML cells
              </a>
            </td>
          <td>
            Jaime Calvo Sánchez, Mary T Scott, Athina Varnava, Tala Alakhras, Alice Wedler, Danny Misiak, N. Bley, C. Ihling, A. Sinz, Karen Keeshan, S. Hüttelmaier, David Vetrie, Marcel Köhn
          </td>
          <td>2024-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="ABSTRACT Viruses display large variability across all stages of their life cycle, including entry, gene expression, replication, assembly, and egress. We previously reported that the immediate early adenovirus (AdV) E1A transcripts accumulate in human lung epithelial A549 cancer cells with high variability, mostly independent of the number of incoming viral genomes, but somewhat correlated to the cell cycle state at the time of inoculation. Here, we leveraged the classical Luria–Delbrück fluctuation analysis to address whether infection variability primarily arises from the cell state or stochastic noise. The E1A expression was measured by the expression of green fluorescent protein (GFP) from the endogenous E1A promoter in AdV-C5_E1A-FS2A-GFP and found to be highly correlated with the viral plaque formation, indicating reliability of the reporter virus. As an ensemble, randomly picked clonal A549 cell isolates displayed significantly higher coefficients of variation in the E1A expression than technical noise, indicating a phenotypic variability larger than noise. The underlying cell state determining infection variability was maintained for at least 9 weeks of cell cultivation. Our results indicate that preexisting cell states tune adenovirus infection in favor of the cell or the virus. These findings have implications for antiviral strategies and gene therapy applications. IMPORTANCE Viral infections are known for their variability. Underlying mechanisms are still incompletely understood but have been associated with particular cell states, for example, the eukaryotic cell division cycle in DNA virus infections. A cell state is the collective of biochemical, morphological, and contextual features owing to particular conditions or at random. It affects how intrinsic or extrinsic cues trigger a response, such as cell division or anti-viral state. Here, we provide evidence that cell states with a built-in memory confer high or low susceptibility of clonal human epithelial cells to adenovirus infection. Results are reminiscent of the Luria–Delbrück fluctuation test with bacteriophage infections back in 1943, which demonstrated that mutations, in the absence of selective pressure prior to infection, cause infection resistance rather than being a consequence of infection. Our findings of dynamic cell states conferring adenovirus infection susceptibility uncover new challenges for the prediction and treatment of viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94415b8ef9824e94b917cda81460003345123e08" target='_blank'>
              Preexisting cell state rather than stochastic noise confers high or low infection susceptibility of human lung epithelial cells to adenovirus
              </a>
            </td>
          <td>
            Anthony Petkidis, M. Suomalainen, V. Andriasyan, Abhyudai Singh, U. Greber
          </td>
          <td>2024-09-24</td>
          <td>mSphere</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Adding aPD1 to 5-FU/platinum in advanced gastroesophageal adenocarcinomas (GEA) has yielded modest and heterogeneous results. Understanding cooperativity between these two treatment modalities will inform novel combination treatments for GEA. Towards this end, we conducted a trial (n = 47) sequentially adding pembrolizumab to 5-FU/platinum in previously untreated advanced GEA. Using serial biopsy of the primary tumor at baseline, after one cycle of 5-FU/platinum, and after the addition of pembrolizumab we profiled 66,816 tumor-resident T cells with matched single-cell RNA-seq and single-cell TCR-seq. Stratifying patients into slow- (> 6 months) and fast-progressing (< 6 months) groups based on progression-free survival allowed us to quantify clonal and transcriptional differences in T cell subpopulations across three time points. We annotated CD4 and CD8 T cell phenotypes according to canonical marker genes and found evidence of 17 different subpopulations along various stages of memory and exhaustion trajectories. We found differences in the abundance of T cell subtypes between slow- and fast-progressors at various treatment time points. T cells with exhausted phenotypes were more abundant in slow progressors in the post-chemo and post-immunotherapy time points. Additionally, we found changes in phenotypic abundances between clones shared between time points. For example, clonotypes that were predominantly of the progenitor exhausted phenotype pre-immunotherapy had a notable proportion of shared post-ICB clones exhibiting an activated phenotype. Previous studies have highlighted exhausted T cells as having important roles in maintaining tumor immunity as these tend to be subject to chronic antigen stimulation due to their specificity to tumor antigen. For this reason, we identified tumor-reactive T cell clonotypes among our data using literature-defined proxies such as CXCL13 expression, exhaustion status and degree of post-therapeutic clonal expansion, in addition to transcription signature-based predictions using machine learning tools such as predicTCR and TRTpred. We found that tumor-reactive T cell clonotypes tended to be present in slow progressors across time points. CXCL13 had variable patterns of upregulation and downregulation depending on whether a particular tumor-reactive clonotype was expanding or contracting at a specific treatment time point. Moreover, we quantified whether these tumor reactive clonotypes shared clones with non-exhausted T cell subsets prior to treatment or whether they were newly present in the tumor microenvironment (TME) post-treatment to determine the relative contributions of phenotypic transition and peripheral replenishment to tumor reactive T cells in the gastric cancer tumor microenvironment. Our data provide an invaluable resource to understand T cell behavior in the context of chemoimmunotherapy and propose strategies to drive anti-tumor immunity to expand the portion of patients benefiting from immunotherapy approaches.
 Citation Format: Samuel J. Wright, Minae An, Byung Hoon Min, You Jeong Heo, Milan Parikh, Lynn Bi, Hyuk Lee, Taejun Kim, Song-Yi Lee, Jeonghyeon Moon, Matthew R. Strickland, Woong Yang Park, Won Ki Kang, Kyoung-Mee Kim, Seung Tae Kim, Ryan J. Park, Arnav Mehta, Samuel Klempner, Jeeyun Lee. Tumor-reactive T cell clonal dynamics across treatment time points drive resistance and progression to frontline chemoimmunotherapy in advanced gastric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41a98abd5d238514b71c6450ac3a4ce0efd3962" target='_blank'>
              Abstract A010: Tumor-reactive T cell clonal dynamics across treatment time points drive resistance and progression to frontline chemoimmunotherapy in advanced gastric cancer
              </a>
            </td>
          <td>
            Samuel J Wright, M. An, B. Min, Y. Heo, Milan Parikh, Lynn Bi, Hyuk Lee, Taejun Kim, Song-Yi Lee, Jeonghyeon Moon, Matthew R Strickland, Woong Yang Park, Wonseok Kang, Kyoung-Mee Kim, S. Kim, Ryan J. Park, Arnav Mehta, S. Klempner, Jeeyun Lee
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f59acd0b5bfd043cfd54293ab6200061c06ac4" target='_blank'>
              Integrated analysis of single-cell RNA-seq and bulk RNA-seq revealed key genes for bone metastasis and chemoresistance in prostate cancer.
              </a>
            </td>
          <td>
            Hongai Bai, Zhenyue Li, Yueyue Weng, Facai Cui, Wenpu Chen
          </td>
          <td>2024-10-12</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Amino-terminal (Nt-) acetylation (NTA) is a common protein modification, affecting 80% of cytosolic proteins in humans. The human essential gene, NAA10, encodes the enzyme NAA10, as the catalytic subunit for the N-terminal acetyltransferase A (NatA) complex, including the accessory protein, NAA15. The first human disease directly involving NAA10 was discovered in 2011, and it was named Ogden syndrome (OS), after the location of the first affected family residing in Ogden, Utah, USA. Since that time, other variants have been found in NAA10 and NAA15. Here we describe the generation of 31 iPSC lines, with 16 from females and 15 from males. This cohort includes CRISPR-mediated correction to the wild-type genotype in 4 male lines, along with editing one female line to generate homozygous wild-type or mutant clones. Following the monoclonalizaiton and screening for X-chromosome activation status in female lines, 3 additional pairs of female lines, in which either the wild type allele is on the active X chromosome (Xa) or the pathogenic variant allele is on Xa, have been generated. Subsets of this cohort have been successfully used to make cardiomyocytes and neural progenitor cells (NPCs). These cell lines are made available to the community via the NYSCF Repository.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1adf8656f9aef233d0cf25ef9cb89566ccecd" target='_blank'>
              A repository of Ogden syndrome patient derived iPSC lines and isogenic pairs by X-chromosome screening and genome-editing.
              </a>
            </td>
          <td>
            Josephine Wesely, Tom Rusielewicz, Yu-Ren Chen, Brigham Hartley, Dayna McKenzie, Matthew K. Yim, Colin Maguire, Ryan Bia, Sarah Franklin, Rikhil Makwana, E. Marchi, Manali Nikte, Soha Patil, Maria Sapar, Dorota Moroziewicz, Lauren Bauer, Jeannie T. Lee, Frederick J. Monsma, Dan Paull, G. Lyon
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are a small population of less differentiated cells with robust self-renewal ability. CSCs have been recognized as the root cause of tumor initiation, progression, relapse, and drug resistance. Recent studies from us and others have highlighted that N6-methyladenosine (m6A), the most prevalent modification in mRNA, plays a crucial role in carcinogenesis and CSC homeostasis. Dysregulation of the m6A modification machinery has been implicated in CSC survival and self-renewal, thereby regulating cancer progression and therapeutic resistance. In this review, we provide an overview of the roles and molecular mechanisms of the RNA m6A modification machinery in CSC survival and self-renewal. Additionally, we summarize the currently known small-molecule inhibitors targeting the dysregulated m6A modification machinery and discuss proof-of-concept studies focusing on the efficacy of these compounds in eliminating CSCs and cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc3a2c463331686ce335a111580d49921b4543bf" target='_blank'>
              Combating cancer stem cells: RNA m6A methylation and small-molecule drug discovery
              </a>
            </td>
          <td>
            Honghai Zhang, Xueer Wang, Jianjun Chen, R. Su
          </td>
          <td>2024-09-30</td>
          <td>Frontiers in Drug Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The nature of standing genetic variation remains a central debate in population genetics, with differing perspectives on whether common variants are mostly neutral or have functional effects. We address this question by directly mapping the fitness effects of over 9,000 natural variants in the Ras/PKA and TOR/Sch9 pathways—key regulators of cell proliferation in eukaryotes—across four conditions in Saccharomyces cerevisiae. While many variants are neutral in our assay, on the order of 3,500 exhibited significant fitness effects. These non-neutral variants tend to be missense and affect conserved, more densely packed, and less solvent-exposed protein regions. They are also typically younger, occur at lower frequencies, and more often found in heterozygous states, suggesting they are subject to purifying selection. A substantial fraction of non-neutral variants showing strong fitness effects in our experiments, however, is present at high frequencies in the population. These variants show signs of local adaptation as they tend to be found specifically in domesticated strains adapted to human-made environments. Our findings support the view that while common variants are often neutral, a significant proportion have adaptive functional consequences and are driven into the population by local positive selection. This study highlights the potential to explore the functional effects of natural genetic variation on a genome scale with quantitative fitness measurements in the laboratory, bridging the gap between population genetics and functional genomics to understand evolutionary dynamics in the wild.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ac4b3d1ad23cba5c317f47581f589736666612" target='_blank'>
              Massively parallel experimental interrogation of natural variants in ancient signaling pathways reveals both purifying selection and local adaptation
              </a>
            </td>
          <td>
            José Aguilar-Rodríguez, Jean Vila, Shi-An A. Chen, Manuel Razo-Mejia, Olivia Ghosh, Hunter B. Fraser, Dan Jarosz, D. Petrov
          </td>
          <td>2024-10-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Epigenetic regulation involves the coordinated interplay of diverse proteins. To systematically explore these combinations, we present COMBINE (combinatorial interaction exploration), a high-throughput platform that tests over 50,000 pairs of epigenetic effector domains up to 2,094 amino acids in length for their ability to modulate endogenous human gene transcription. COMBINE revealed diverse synergistic and antagonistic interactions between epigenetic protein domains, including a potent KRAB-L3MBTL3 fusion that enhanced gene silencing up to 34-fold in dose-limited conditions and enabled robust bidirectional CRISPR perturbation. Inducible screening showed DNA methylation modifiers are essential for epigenetic memory, with distinct combinations driving long-term silencing, repression, or activation. Notably, we identified TET1-based combinations that induce hit-and-run upregulation for over 50 days, demonstrating long-term transcriptional activation. This systematic analysis of pairwise domain interactions provides a rich resource for understanding epigenetic crosstalk and developing next-generation epigenome editing tools. More broadly, COMBINE offers a generalizable platform to functionally characterize combinatorial biological processes at scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a367580d618e43d754a424814c605aa4d48914d" target='_blank'>
              A combinatorial domain screening platform reveals epigenetic effector interactions for transcriptional perturbation
              </a>
            </td>
          <td>
            Hyungseok C. Moon, Michael H. Herschl, April Pawluk, Silvana Konermann, Patrick D. Hsu
          </td>
          <td>2024-11-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rationale: Both bulk RNA-sequencing and GEO database upon chemotherapy to non-small cell lung cancer (NSCLC) cells reveal that ZNF131 (Zinc Finger Protein 131) maybe a crucial transcriptional factor involved. However, it is a recently discovered protein with largely unexplored expression patterns and biological functions. Methods: Bioinformatics analyses and immunohistochemistry staining were assessed to detect both mRNA and protein levels of ZNF131 in NSCLC specimens and cell lines. Next, colony formation assay, MTT assay, EdU assay, transwell assay, flow cytometric analysis, sphere formation assay, western blotting analysis, mouse xenograft model analysis, immunofluorescence assay, and reverse transcriptase ‐ polymerase chain reaction were performed to investigate the effect of ZNF131 interaction on proliferation, invasion, stemness, chemotherapy sensitivity. RNA-sequencing assay, RNA-microarray, and ChIP-sequencing assay were used to identify candidate downstream target genes. Further, liquid chromatography-tandem mass spectrometry analysis, GST pull-down assay, and immunoprecipitation assays were performed to evaluate the interactions between ZNF131, BACH1, and CUL3. Results: ZNF131 was elevated in NSCLC specimens and cell lines, which significantly correlates with advanced TNM stage and poor prognosis in NSCLC patients. ZNF131 overexpression promotes NSCLC cell proliferation, invasion, and stemness both in vitro and in vivo . ZNF131 appears to target the RAD51 gene within a well-defined region (-668bp to -403bp) of the RAD51 promoter. ZNF131 contributes to RAD51-dependent homologous recombination (HR), primarily through its Zinc Finger and BTB domains. ZNF131-BACH1 interaction, mediated by their respective BTB domains, enhances the stability of both proteins, effectively preventing their ubiquitin-mediated degradation by CUL3. The ZNF131-BACH1 partnership significantly amplifies RAD51-dependent HR, resulting in expedited resistance to both radiotherapy and chemotherapy in NSCLC patients. Desoxyrhaponticin was shown to halt NSCLC progression and orchestrate a synergistic effect together with chemotherapy at least partially by targeting ZNF131.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0b84d4dd3e29e2628ded0f7bfec71ddec9627fb" target='_blank'>
              ZNF131-BACH1 transcriptionally accelerates RAD51-dependent homologous recombination repair and therapy-resistance of non-small-lung cancer cells by preventing their degradation from CUL3
              </a>
            </td>
          <td>
            Mingwei Fan, Quanbo Liu, Xiaowen Ma, Yufeng Jiang, Yilong Wang, Shuting Jia, Yingtong Nie, Ruoyi Deng, Pengchong Zhou, Shuyu Zhang, Siyu Jiang, Mengyao Guan, Yuekang Hou, Yuan Miao, Yong Zhang, Xiupeng Zhang
          </td>
          <td>2024-10-28</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="A major challenge in radiotherapy is enhancing tumor cell sensitivity while minimizing damage to healthy cells. Ionising radiation (IR) induces mitochondrial DNA (mtDNA) alterations that can impair mitochondrial function and cell survival. Since mitochondria play a key role in tumor cell proliferation, they present a promising therapeutic target for cancer treatment. In this study, we characterized the impact of different IR sources on mitochondrial function in radioresistant cancer cells. Our findings revealed several adaptive responses that may contribute to radioresistance, including increased mtDNA content, mitochondrial mass, enhanced activity, and hyperfusion of the mitochondrial network. Notably, the use of mitochondrial-targeted G-quadruplex (G4) ligands, which block mtDNA replication and transcription, disrupted these adaptations, reducing cancer cell survival in an mtDNA-dependent manner. These results demonstrate that mitochondrial adaptations contribute to radioresistance and highlight mitochondria as a novel target for the radiosensitizing effects of G4-ligands, extending their potential beyond telomere destabilization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea314d6106916e6fee9f1c76901743b60a11856" target='_blank'>
              G-quadruplex ligand RHPS4 compromises cellular radio-resistance by blocking the increase in mitochondrial mass and activity induced by ionising irradiation
              </a>
            </td>
          <td>
            S. Tricot, C. Siberchicot, I. Bontemps, C. Desmaze, G. Kratassiouk, M. Vandamme, G. Pinna, J. Radicella, G. Lenaers, J. Lebeau, A. Campalans
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Genetic mutations are being thoroughly mapped in human cancers, yet a fundamental question in cancer biology is whether such mutations are functionally required for cancer initiation, maintenance of established cancer, or both. Here, we study this question in the context of human acute myeloid leukemia (AML), where DNMT3AR882 missense mutations often arise early, in pre-leukemic clonal hematopoiesis, and corrupt the DNA methylation landscape to initiate leukemia. We developed CRISPR-based methods to directly correct DNMT3AR882 mutations in leukemic cells obtained from patients. Surprisingly, DNMT3AR882 mutations were largely dispensable for disease maintenance. Replacing DNMT3AR882 mutants with wild-type DNMT3A did not impair the ability of AML cells to engraft in vivo, and minimally altered DNA methylation. Taken together, DNMT3AR882 mutations are initially necessary for AML initiation, but are largely dispensable for disease maintenance. The notion that initiating oncogenes differ from those that maintain cancer has important implications for cancer evolution and therapy. STATEMENT OF SIGNIFICANCE Understanding which driver mutations are required for cancer initiation, maintenance, or both phases remains poorly understood. Here, we uncover that highly prevalent pre-leukemic DNMT3A mutations are only required during disease initiation, but become dispensable after leukemic transformation, uncovering the context-specific role of this driver mutation with important therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94506bb3eb524303455ffe91b550e1d6fd5d7a8d" target='_blank'>
              DNMT3AR882H Is Not Required for Disease Maintenance in Primary Human AML, but Is Associated With Increased Leukemia Stem Cell Frequency
              </a>
            </td>
          <td>
            Thomas Köhnke, Daiki Karigane, Eleanor Hilgart, Amy C. Fan, Kensuke Kayamori, Masashi Miyauchi, Cailin T Collins, Fabian P. Suchy, Athreya S Rangavajhula, Yang Feng, Yusuke Nakauchi, Eduardo Martinez-Montes, J. Fowler, Kyle M. Loh, Hiromitsu Nakauchi, Michael A Koldobskiy, Andy Feinberg, Ravi Majeti
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Predicting how genetic variation affects phenotypic outcomes at the organismal, cellular, and molecular levels requires deciphering the cis-regulatory code, the sequence rules by which non-coding regions regulate genes. In this perspective, we discuss recent computational progress and challenges towards solving this fundamental problem. We describe how cis-regulatory elements are mapped and how their sequence rules can be learned and interpreted with sequence-to-function neural networks, with the goal of identifying genetic variants in human disease. We also discuss how studies of the 3D chromatin organization could help identifying long-range regulatory effects and how current methods for mapping gene regulatory networks could better describe biological processes. We point out current gaps in knowledge along with technical limitations and benchmarking challenges of computational methods. Finally, we discuss newly emerging technologies, such as spatial transcriptomics, and outline strategies for creating a more general model of the cis-regulatory code that is more broadly applicable across cell types and individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b9c7ac6b4abee50c1ee672caf88f62471744e3c" target='_blank'>
              Perspective on recent developments and challenges in regulatory and systems genomics
              </a>
            </td>
          <td>
            Julia Zeiltinger, Sushmita Roy, Ferhat Ay, Anthony Mathelier, Alejandra Medina-Rivera, Shaun Mahony, Saurabh Sinha, Jason Ernst
          </td>
          <td>2024-11-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Gliomas, the most frequent malignant primary brain tumors, lack curative treatments. Understanding glioma-specific molecular alterations is crucial to develop novel therapies. Among them, the biological consequences of the isocitrate dehydrogenase 1 gene mutation (IDH1R132H) remain inconclusive despite its early occurrence and widespread expression.



 We thus employed CRISPR/Cas adenine base editors, which allow precise base pair alterations with minimal undesirable effects, to correct the IDH1R132H mutation.



 Successful correction of the IDH1R132H mutation in primary patient-derived cell models led to reduced IDH1R132H protein levels and decreased production of 2-hydroxyglutarate, but increased proliferation. A dual adeno-associated virus split intein system was used to successfully deliver the base editor in vitro and in vivo.



 Taken together, our study provides a strategy for a precise genetic intervention to target the IDH1R132H mutation, enabling the development of accurate models to study its impact on glioma biology and serving as a framework for an in vivo gene therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/392336c2a72052298f6dba93e1588fedfbacb08a" target='_blank'>
              Targeting the IDH1R132H mutation in gliomas by CRISPR/Cas precision base editing
              </a>
            </td>
          <td>
            Remi Weber, F. Vasella, Artsiom Klimko, M. Silginer, M. Lamfers, M. Neidert, Luca Regli, Gerald Schwank, M. Weller
          </td>
          <td>2024-11-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive neoplasm. Although most patients respond to induction therapy, they commonly relapse due to recurrent disease in the bone marrow microenvironment (BMME). So, disruption of the BMME, releasing tumour cells into the peripheral circulation, has therapeutic potential. Using both primary donor AML cells and cell lines, we developed an in-vitro co-culture model of the AML BMME. We used this model to identify the most effective agent(s) to block AML cell adherence and reverse adhesion-mediated treatment resistanc E. We identified anti-CD44 treatment significantly increased the efficacy of cytarabine. However, some AML cells remained adhered, and transcriptional analysis identified focal adhesion kinase (FAK) signalling as a contributing factor; adhered cells showed elevated FAK phosphorylation that was reduced by the FAK inhibitor, defactinib. Importantly, we demonstrated that anti-CD44 and defactinib were highly synergistic at diminishing adhesion of the most primitive CD34high AML cells in primary autologous co-cultures. Taken together, we identified anti-CD44 and defactinib as a promising therapeutic combination to release AML cells from the chemoprotective AML BMME. As anti-CD44 is already available as a recombinant humanised monoclonal antibody, the combination of this agent with defactinib could be rapidly tested in AML clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d636733a55fb2234c803bb8f4874252b8ff49e2d" target='_blank'>
              A novel in-vitro model of the bone marrow microenvironment in AML identifies CD44 and Focal Adhesion Kinase as therapeutic targets to reverse cell adhesion-mediated drug resistance
              </a>
            </td>
          <td>
            Eleni E Ladikou, Kim Sharp, F. A. Simoes, John Jones, Thomas Burley, Lauren Stott, Aimilia Vareli, Emma Kennedy, Sophie Vause, Timothy J T Chevassut, Amarpreet Devi, Iona Ashworth, David M. Ross, T. N. Hartmann, Simon Mitchell, Chris Pepper, Giles Best, A. Pepper
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary The therapeutic landscape of Chronic Lymphocytic Leukemia (CLL) has dramatically changed in recent years, with a shift from chemoimmunotherapy towards many new targeted agents, such as Bruton Tyrosine Kinase inhibitors (BTKi) and anti-BCL-2. We now need an accurate re-definition of prognostic/predictive parameters to support clinicians in choosing the more appropriate option. This review explores recent advances in the molecular biology of chronic lymphocytic leukemia, with the aim of identifying the most important biomarkers to define prognosis and to guide first-line and second-line treatment. Abstract Chronic Lymphocytic Leukemia (CLL) is the most frequent type of leukemia in Western countries. In recent years, there have been important advances in the knowledge of molecular alterations that underlie the disease’s pathogenesis. Very heterogeneous prognostic subgroups have been identified by the mutational status of immunoglobulin heavy variable genes (IGVH), FISH analysis and molecular evaluation of TP53 mutations. Next-generation sequencing (NGS) technologies have provided a deeper characterization of the genomic and epigenomic landscape of CLL. New therapeutic targets have led to a progressive reduction of traditional chemoimmunotherapy in favor of specific biological agents. Furthermore, in the latest clinical trials, the minimal residual disease (MRD) has emerged as a potent marker of outcome and a guide to treatment duration. This review focuses on recent insights into the understanding of CLL biology. We also consider the translation of these findings into the development of risk-adapted and targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3c13c68ecb506ccc3f3b3f0d7d44f962adc11fb" target='_blank'>
              Recent Advances in the Molecular Biology of Chronic Lymphocytic Leukemia: How to Define Prognosis and Guide Treatment
              </a>
            </td>
          <td>
            A. Arcari, Lucia Morello, Elena Borotti, Elena Ronda, Angela Rossi, Daniele Vallisa
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e206396092ebd32f9a052e89e6643d141dcd6ce" target='_blank'>
              High-resolution and programmable RNA-IN and RNA-OUT genetic circuit in living mammalian cells
              </a>
            </td>
          <td>
            Min Zhang, Xue Zhang, Yongyue Xu, Yanhui Xiang, Bo Zhang, Zhenchi Xie, Qiong Wu, Chunbo Lou
          </td>
          <td>2024-10-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epigenetic dysregulation has long been recognized as a significant contributor to tumorigenesis and tumor maintenance, impacting all recognized cancer hallmarks. Although some epigenetic drugs have received regulatory approval for certain hematological malignancies, their efficacy in treating solid tumors has so far been largely disappointing. However, recent advancements in developing new compounds and a deeper understanding of cancer biology have led to success in specific solid tumor subtypes through precision medicine approaches. Moreover, epigenetic drugs may play a crucial role in synergizing with other anticancer treatments, enhancing the sensitivity of cancer cells to various anticancer therapies, including chemotherapy, radiation therapy, hormone therapy, targeted therapy, and immunotherapy. In this review, we critically evaluate the evolution of epigenetic drugs, tracing their development from initial use as monotherapies to their current application in combination therapies. We explore the preclinical rationale, completed clinical studies, and ongoing clinical trials. Finally, we discuss trial design strategies and drug scheduling to optimize the development of possible combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/357590ca2c8c3ea804892d4f86c25aabfdeb5726" target='_blank'>
              Progresses and Pitfalls of Epigenetics in Solid Tumors Clinical Trials
              </a>
            </td>
          <td>
            Alice Rossi, F. Zacchi, Anna Reni, Michele Rota, Silvia Palmerio, Jessica Menis, Andrea Zivi, Stefano Milleri, Michele Milella
          </td>
          <td>2024-10-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Overexpression of the MYC oncogene, encoding c-MYC protein, contributes to the pathogenesis and drug resistance of acute myeloid leukemia (AML) and many other hematopoietic malignancies. Although standard chemotherapy has predominated in AML therapy over the past five decades, the clinical outcomes and patient response to treatment remain suboptimal. Deeper insight into the molecular basis of this disease should facilitate the development of novel therapeutics targeting specific molecules and pathways that are dysregulated in AML, including fms-like tyrosine kinase 3 (FLT3) gene mutation and cluster of differentiation 33 (CD33) protein expression. Elevated expression of c-MYC is one of the molecular features of AML that determines the clinical prognosis in patients. Increased expression of c-MYC is also one of the cytogenetic characteristics of drug resistance in AML. However, direct targeting of c-MYC has been challenging due to its lack of binding sites for small molecules. In this review, we focused on the mechanisms involving the bromodomain and extra-terminal (BET) and cyclin-dependent kinase 9 (CDK9) proteins, phosphoinositide-Akt-mammalian target of rapamycin (PI3K/AKT/mTOR) and Janus kinase-signal transduction and activation of transcription (JAK/STAT) pathways, as well as various inflammatory cytokines, as an indirect means of regulating MYC overexpression in AML. Furthermore, we highlight Food and Drug Administration (FDA)-approved drugs for AML, and the results of preclinical and clinical studies on novel agents that have been or are currently being tested for efficacy and tolerability in AML therapy. Overall, this review summarizes our current knowledge of the molecular processes that promote leukemogenesis, as well as the various agents that intervene in specific pathways and directly or indirectly modulate c-MYC to disrupt AML pathogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1135bf881c820d3eca90352e7790a726703df16e" target='_blank'>
              Targeting Molecular Signaling Pathways and Cytokine Responses to Modulate c-MYC in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Kyle Gu, Harry A. May, Min H. Kang
          </td>
          <td>2024-09-14</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapeutic resistance to androgen receptor (AR)-targeting agents remains a significant clinical problem during the treatment of prostate cancer, with the incidence rate of resistant disease increasing as more men are treated with next-generation AR-targeted therapies. Lineage plasticity and progression to neuroendocrine prostate cancer (NEPC) are mechanisms by which prostate tumors lose dependence on androgen signaling and escape treatment. Although many known genetic alterations can predispose tumors to acquiring the NEPC phenotype, it remains unclear what, if any, drivers are essential to this progression. In this issue of Cancer Research, Rodarte and colleagues identified ASCL1 as one such essential regulator. Through the use of genetically engineered mouse models, the authors demonstrated that whereas ASCL1 was dispensable for tumor formation and growth, ASCL1 loss nearly completely abrogated the development of NEPC and instead redirected lineage trajectories toward a basal-like phenotype. This study provides an important new model for the study of NEPC, reveals the ability of ASCL1+ NEPC cells to also assume a NEUROD1+ state, and demonstrates the changes to tumor cell phenotypes following ASCL1 loss. See related article by Rodarte et al., p. 3522.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa9febe0aaec1649e5c618f9a9d1ccaf586fbd5c" target='_blank'>
              ASCL1 Drives the Development of Neuroendocrine Prostate Cancer.
              </a>
            </td>
          <td>
            Caden N McQuillen, Nicholas J Brady
          </td>
          <td>2024-11-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b80c87e540615a9cfc5879e1a24f75c5dd8d1b" target='_blank'>
              The interplay of mutagenesis and ecDNA shapes urothelial cancer evolution
              </a>
            </td>
          <td>
            Duy D. Nguyen, William Hooper, Weisi Liu, Timothy R Chu, Heather Geiger, Jennifer Shelton, Minita J. Shah, Zoe R. Goldstein, L. Winterkorn, Adrienne Helland, M. Sigouros, J. Manohar, Jenna Moyer, Majd Al Assaad, Alissa Semaan, Sandra Cohen, Florencia P. Madorsky Rowdo, David Wilkes, Mohamed Osman, Rahul R. Singh, A. Sboner, Henkel Valentine, Phillip Abbosh, Scott T. Tagawa, D. Nanus, J. Nauseef, Cora N. Sternberg, Ana M. Molina, Douglas Scherr, G. Inghirami, Juan Miguel Mosquera, O. Elemento, N. Robine, B. Faltas
          </td>
          <td>2024-10-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Abstract Transcriptional enhancers orchestrate cell type- and time point-specific gene expression programs. Genetic variation within enhancer sequences is an important contributor to phenotypic variation including evolutionary adaptations and human disease. Certain genes and pathways may be more prone to regulatory evolution than others, with different patterns across diverse organisms, but whether such patterns exist has not been investigated at a sufficient scale. To address this question, we identified signatures of accelerated sequence evolution in conserved enhancer elements throughout the mammalian phylogeny at an unprecedented scale. While different genes and pathways were enriched for regulatory evolution in different parts of the tree, we found a striking overall pattern of pleiotropic genes involved in gene regulatory and developmental processes being enriched for accelerated enhancer evolution. These genes were connected to more enhancers than other genes, which was the basis for having an increased amount of sequence acceleration over all their enhancers combined. We provide evidence that sequence acceleration is associated with turnover of regulatory function. Detailed study of one acceleration event in an enhancer of HES1 revealed that sequence evolution led to a new activity domain in the developing limb that emerged concurrently with the evolution of digit reduction in hoofed mammals. Our results provide evidence that enhancer evolution has been a frequent contributor to regulatory innovation at conserved developmental signaling genes in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f787749d36ce9eb176cc01c130701746e5627c" target='_blank'>
              Evolutionary Innovations in Conserved Regulatory Elements Associate With Developmental Genes in Mammals
              </a>
            </td>
          <td>
            Severin Uebbing, Acadia A. Kocher, Marybeth Baumgartner, Yu Ji, Suxia Bai, Xiaojun Xing, Timothy Nottoli, James P Noonan
          </td>
          <td>2024-09-20</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer is a global challenge, and genetics play a major role in onsets and in designing the next generation of cancer therapies. One of the key approaches is the downregulation of genes through RNA interference (RNAi) which has been proposed as a promising tool for controlling cancer-causing genes by employing a complementary base-pairing mechanism. Crucial tools in RNAi; small interfering RNA (siRNA), and microRNA (miRNA) have been extensively utilized in cancer therapy. Despite their discovery nearly two decades ago, only a handful of RNAi-based products have recently gained approval from regulatory authorities like the FDA, principally due to inherent limitations. The efficacy of RNAi therapies is significantly hindered by barriers related to absorption, distribution, metabolism, and excretion, leading to rapid elimination from the systemic circulation before reaching the cytosol of targeted cells. Nevertheless, RNAi therapeutics hold immense promise in various diseases, especially in various types of cancer. Overcoming these challenges demands the design of suitable drug delivery systems and the implementation of strategies aimed at enhancing pharmacokinetic parameters associated with RNAi therapies. Nanotechnology-based vehicles such as polymer-based nanoparticles, lipid nanoparticles, liposomes, dendrimers, and inorganic nanoparticles may serve as efficient carriers for targeting RNAi therapies to their desired sites. This review discusses the recent advances in nanotechnology strategies for RNAi delivery, with the overarching objective of facilitating effective targeting and gene silencing for the advancement of RNAi in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d3ac4d02e434d3ba852ff719ff6ff9da7bfae5" target='_blank'>
              Recent Advancements in Cancer RNA-interference Therapy with Nanotechnology Strategies
              </a>
            </td>
          <td>
            A. Abosalha, P. Islam, J. Boyajian, A. Kassab, S. Makhlouf, Madison Santos, Satya Prakash
          </td>
          <td>2024-10-09</td>
          <td>Journal of Cancer Biomoleculars and Therapeutics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Deletions in Exon-19 of the epidermal growth factor receptor (EGFR) play a pivotal role in the pathogenesis of non-small cell lung cancer (NSCLC), influencing patient response to tyrosine kinase inhibitors (TKIs). Although these mutations are known to affect treatment efficacy, the precise molecular mechanisms have been unclear. Building upon recent insights from the study [DOI: 10.1038/s41467-022-34398-z], which identified two distinct mutation profiles associated with differential drug sensitivity and clinical outcomes, our research delves into the molecular dynamics that drive these variances. We employed molecular dynamics simulations, enhanced sampling methods, and machine learning to classify Exon-19 deletion mutations into two profiles based on their conformational dynamics. Profile 1 mutations display only localized motions in key subdomains in their fluctuations about the equilibrium state, and a high affinity for ATP and consequent resistance to TKIs, while profile 2 mutations show reduced ATP binding affinity due to delocalized motion characterized by an increased flexibility between the N-and C-lobes of the EGFR kinase domain. This structural flexibility perturbs the ATP binding site, leading to decreased affinity and, heightened sensitivity to TKIs. Our use of the INDUS technique has shed light on the collective solvent dynamics, further elucidating the coupling between long timescale solvent fluctuations and protein conformational dynamics, that likely contributes to the observations in HDX-MS studies. Our free energy analysis, covering timescales relevant to both HDX-MS and ligand interaction, provides a deeper understanding of the relationship between protein and solvent dynamics and their collective impact on drug efficacy in NSCLC with EGFR Exon-19 deletions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe52f0dbf2d6a789337ce21c7f22177f760d945" target='_blank'>
              Coupled Solvent Dynamics and Protein Dynamics Help Drive Functional Differences in Exon-19 Deletion Mutants in the Epidermal Growth Factor Receptor (EGFR/ErbB1/HER1) Kinase Domain
              </a>
            </td>
          <td>
            Keshav Patil, Kumar D. Ashtekar, Yuko Tsutsui, Debdas Dhabal, Krishna Suresh, M. Lemmon, Ravi Radhakrishnan
          </td>
          <td>2024-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a29762d25b9a77c8fd5e2a5d7d19911938e8b1" target='_blank'>
              Integrate and conquer: pan-cancer proteogenomics uncovers cancer vulnerabilities and therapeutic opportunities
              </a>
            </td>
          <td>
            Debomita Chakraborty, Rossana Romero, K. Rajalingam
          </td>
          <td>2024-10-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R. Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E. Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d26a037aa19a7d0e8eba4d251f11045e3132706b" target='_blank'>
              A single-cell transcriptomic map of the murine and human multiple myeloma immune microenvironment across disease stages
              </a>
            </td>
          <td>
            Emma Verheye, Daliya Kancheva, Hatice Satilmis, N. Vandewalle, Rong Fan, P. Bardet, Emile Clappaert, Kevin Verstaen, A. De Becker, Karin Vanderkerken, K. De Veirman, Damya Laoui
          </td>
          <td>2024-11-07</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Epigenetic silencing is essential for regulating gene expression and cellular diversity in eukaryotes. While transposable elements (TEs) are mostly silenced with DNA and H3K9 methylation, gene silencing is mediated by H3K27me3, an epigenetic mark deposited by the Polycomb repressive complex 2 (PRC2). Despite the major role epigenetic silencing plays in the development of multicellular eukaryotes, little is known about how epigenetically-controlled regulatory networks were shaped over evolutionary time. Here, we analyse epigenomes from a diverse group of species across the green lineage and infer the chronological epigenetic recruitment of genes that occurred during land plant evolution. We first reveal the nature of plant heterochromatin in the unicellular green microalga Chlorella sorokiniana and identify a substantial number of genes marked with H3K27me3, highlighting the deep origin of PRC2-regulated genes in the green lineage. By incorporating genomic phylostratigraphy, we show how genes of differing evolutionary age are partitioned into distinct epigenetic states in plants, with evolutionarily young genes incorporated into developmental programs controlled by H3K9 methylation in Arabidopsis. We further reveal a major wave of PRC2 recruitment to genes that emerged during land terrestrialisation and flowering plant evolution, and identify an ancestral PRC2 network with a shared functional topology in green algae through to land plants, providing a glimpse of the earliest types of genes regulated by PRC2 during the course of plant evolution. Finally, we analyse the potential regulation of these ancestral PRC2 target genes and find a strong enrichment of motifs bound by ancient AP2/ERF transcription factors (TFs) known to interact with PRC2, which we hypothesise were key determinants in shaping some of the first gene regulatory networks controlled by PRC2 in plants. Our data thus reveal pivotal epigenetic adaptations that occurred during a significant period in the evolutionary history of plants, which likely contributed to key regulatory innovations that influenced major morphological and developmental change into the modern-day. More broadly, our findings offer insight into the evolutionary dynamics and molecular triggers that drive the adaptation and elaboration of epigenetic regulation, laying the groundwork for its future consideration in other major eukaryotic lineages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/578bce57cf922c3e4cbf1c7dcaf1f5ced20825df" target='_blank'>
              The expansion and diversification of epigenetic regulatory networks underpins major transitions in the evolution of land plants
              </a>
            </td>
          <td>
            Romy Petroll, Ranjith K. Papareddy, Rafal Krela, Alice Laigle, Quentin Rivière, Kateřina Bišová, I. Mozgová, Michael Borg
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Dosage-sensitive genes have characteristic patterns of evolution that include being refractory to small-scale duplication, depleted on human benign copy number variants (CNVs) and enriched on pathogenic CNVs. This intolerance to copy number change is likely due to an expression constraint that exists in one or more tissues. While genomic copy number changes alter the encompassed genes’ expression across all tissues, expression quantitative trait loci (eQTLs) –genomic regions harbouring sequence variants that influence the expression level of one or more genes– can act in a tissue-specific manner. In this work we examine expression variation of presumed dosage-sensitive and non-dosage-sensitive genes to discover how the locus duplicability constraints translate into gene expression constraints. Here we test the hypothesis that expression changes due to the presence of eQTLs acting in unconstrained tissues will not be deleterious and thus allow dosage-sensitive genes to vary expression while obeying constraints in other tissues. Using eQTLs across 48 human tissues from The Genotype-Tissue Expression (GTEx) project, we find that dosage-sensitive genes are enriched for being affected by eQTLs and that the eQTLs affecting dosage-sensitive genes are biased towards having narrow tissue-specificity with these genes having fewer eQTL-affected tissues than non-dosage-sensitive genes. Additionally, we find that dosage-sensitive genes are depleted for being affected by broad tissue breadth eQTLs, likely due to the increased chance of these eQTLs conflicting with expression constraints and being removed by purifying selection. These patterns suggest that dosage-sensitivity shapes the evolution of these genes by precluding copy number evolution and restricting their evolutionary trajectories to changes in expression regulation compatible with their functional constraints. Thus deeper interpretation of the patterns of constraints can be informative of the temporal or spatial location of the gene dosage sensitivity and contribute to our understanding of functional genomics. Author summary Gene duplication is an important and powerful evolutionary force that is responsible for the expansion of the coding capacity of genomes ultimately resulting in great genetic novelty. However, the opportunity for this evolutionary change can be limited by dosage constraints on some genes, meaning they are not normally duplicable, except in a balanced, whole genome event. This results in important, biologically relevant, differences between genes that are retained from whole genome duplication events versus those retained from small scale duplications, especially in terms of dosage sensitivity. We explored how the different dosage sensitivity in these sets of genes relates to quantitative expression variation present in populations. We found that while dosage-sensitive genes are more likely to have their expression levels influenced by genetic variation, these changes are often specific a small number of tissues. In contrast, genes that are less sensitive to dosage changes show greater variation in expression levels across multiple tissues. Our findings suggest that dosage-sensitive genes evolve through fine-tuned adjustments in their expression levels in specific tissues, thus bypassing constraints operating on other tissues. This understanding sheds light on how dosage-sensitive genes evolve and could have implications for understanding human diseases caused by these genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d05cd1cac23ae98b3aac115ebfd59b3d377a2304" target='_blank'>
              Evolution of dosage-sensitive genes by tissue-restricted expression changes
              </a>
            </td>
          <td>
            Alan M. Rice, Yuanshuo Li, Pauric Donnelly, A. McLysaght
          </td>
          <td>2024-10-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is an immensely complex ecosystem1,2. This complexity underlies difficulties in elucidating principles of spatial organization and using molecular profiling of the TME for clinical use3. Through statistical analysis of 96 spatial transcriptomic (ST-seq) datasets spanning twelve diverse tumor types, we found a conserved distribution of multicellular, transcriptionally covarying units termed ‘Spatial Groups’ (SGs). SGs were either dependent on a hierarchical local spatial context – enriched for cell-extrinsic processes such as immune regulation and signal transduction – or independent from local spatial context – enriched for cell-intrinsic processes such as protein and RNA metabolism, DNA repair, and cell cycle regulation. We used SGs to define a measure of gene spatial heterogeneity – ‘spatial lability’ – and categorized all 96 tumors by their TME spatial lability profiles. The resulting classification captured spatial variation in cell-extrinsic versus cell-intrinsic biology and motivated class-specific strategies for therapeutic intervention. Using this classification to characterize pre-treatment biopsy samples of 16 non-small cell lung cancer (NSCLC) patients outside our database distinguished responders and non-responders to immune checkpoint blockade while programmed death-ligand 1 (PD-L1) status and spatially unaware bulk transcriptional markers did not. Our findings show conserved principles of TME spatial biology that are both biologically and clinically significant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a400ff77ff3e3d5b4307dd66b458357a7b860383" target='_blank'>
              Conserved principles of spatial biology define tumor heterogeneity and response to immunotherapy
              </a>
            </td>
          <td>
            Vivek Behera, Hannah Giba, Ue-yu Pen, Anna Di Lello, Benjamin A. Doran, Alessandra Esposito, Apameh Pezeshk, C. Bestvina, Justin Kline, M. Garassino, Arjun S. Raman
          </td>
          <td>2024-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gliomas are aggressive brain tumors associated with a poor prognosis. Cancer stem cells (CSCs) play a significant role in tumor recurrence and resistance to therapy. This study aimed to identify and characterize glioma stem cells (GSCs), analyze their interactions with various cell types, and develop a prognostic signature. Methods Single-cell RNA sequencing data from 44 primary glioma samples were analyzed to identify GSC populations. Spatial transcriptomics and gene regulatory network analyses were performed to investigate GSC localization and transcription factor activity. CellChat analysis was conducted to infer cell-cell communication patterns. A GSC signature (GSCS) was developed using machine learning algorithms applied to bulk RNA sequencing data from multiple cohorts. In vitro and in vivo experiments were conducted to validate the role of TUBA1C, a key gene within the signature. Results A distinct GSC population was identified, characterized by high proliferative potential and an enrichment of E2F1, E2F2, E2F7, and BRCA1 regulons. GSCs exhibited spatial proximity to myeloid-derived suppressor cells (MDSCs). CellChat analysis revealed an active MIF signaling pathway between GSCs and MDSCs. A 26-gene GSCS demonstrated superior performance compared to existing prognostic models. Knockdown of TUBA1C significantly inhibited glioma cell migration, and invasion in vitro, and reduced tumor growth in vivo. Conclusion This study offers a comprehensive characterization of GSCs and their interactions with MDSCs, while presenting a robust GSCS. The findings offer new insights into glioma biology and identify potential therapeutic targets, particularly TUBA1C, aimed at improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c2780868fec467184ab80544d5944cba2384cf" target='_blank'>
              From single-cell to spatial transcriptomics: decoding the glioma stem cell niche and its clinical implications
              </a>
            </td>
          <td>
            Lei Cao, Xu Lu, Xia Wang, Hao Wu, Xiaye Miao
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Background: With recent advances in single cell technology, high-throughput methods provide unique insight into disease mechanisms and more importantly, cell type origin. Here, we used multi-omics data to understand how genetic variants from genome-wide association studies influence development of disease. We show in principle how to use genetic algorithms with normal, matching pairs of single-nucleus RNA- and ATAC-seq, genome annotations, and protein-protein interaction data to describe the genes and cell types collectively and their contribution to increased risk. Results: We used genetic algorithms to measure fitness of gene-cell set proposals against a series of objective functions that capture data and annotations. The highest information objective function captured protein-protein interactions. We observed significantly greater fitness scores and subgraph sizes in foreground vs.matching sets of control variants. Furthermore, our model reliably identified known targets and ligand-receptor pairs, consistent with prior studies. Conclusions: Our findings suggested that application of genetic algorithms to association studies can generate a coherent cellular model of risk from a set of susceptibility variants. Further, we showed, using breast cancer as an example, that such variants have a greater number of physical interactions than expected due to chance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f84348da737a235e58f38e58a0d84c93468aee54" target='_blank'>
              Genome wide association studies are enriched for interacting genes
              </a>
            </td>
          <td>
            Peter T. Nguyen, Simon G. Coetzee, Irina Silacheva, Dennis J. Hazelett
          </td>
          <td>2024-10-22</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="An important challenge for mammalian cell engineering is the unexpected response of transgenes to native transcriptional regulation pathways. One transgene can show different levels of expression at different genomic sites, in different cell types, and under different growth conditions. Collisions between transcription and DNA replication, heterochromatin encroachment, and viral defense have been linked to transgene silencing. In this study we identify fatty acid metabolism as another mediator of transgene behavior. Adipocyte secretome-induced lipogenesis in epithelial breast cancer cells was accompanied by the loss of expression from a Tet-TA regulated pCMV-AmCyan fluorescent protein (CFP) transgene. Transcription profiling verified the activation of lipid droplet biosynthesis genes, and revealed repression of loci at every chromosome, consistent with the idea that lipogenesis affects the availability of substrates and cofactors for global chromatin remodeling. Pre-induction of pCMV prevented full transgene silencing during lipogenesis. Our results provide new insights into the influence of the lipogenic epigenetic state on transgene behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523cfb6147ed3575e854d91fb51a5db9d67d279c" target='_blank'>
              Tet Transgene Activation is Disrupted in Lipogenic Triple Negative Breast Cancer Cells
              </a>
            </td>
          <td>
            Ashley Townsel, Yifei Wu, Maya Jaffe, Cara E. Shields, Karmella A. Haynes
          </td>
          <td>2024-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer stem cells (CSCs), or tumor-initiating cells (TICs), are small subpopulations (0.0001–0.1%) of cancer cells that are crucial for cancer relapse and therapy resistance. The elimination of each CSC is essential for achieving long-term remission. Metabolic reprogramming, particularly lipids, has a significant impact on drug efficacy by influencing drug diffusion, altering membrane permeability, modifying mitochondrial function, and adjusting the lipid composition within CSCs. These changes contribute to the development of chemoresistance in various cancers. The intricate relationship between lipid metabolism and drug resistance in CSCs is an emerging area of research, as different lipid species play essential roles in multiple stages of autophagy. However, the link between autophagy and lipid metabolism in the context of CSC regulation remains unclear. Understanding the interplay between autophagy and lipid reprogramming in CSCs could lead to the development of new approaches for enhancing therapies and reducing tumorigenicity in these cells. In this review, we explore the latest findings on lipid metabolism in CSCs, including the role of key regulatory enzymes, inhibitors, and the contribution of autophagy in maintaining lipid homeostasis. These recent findings may provide critical insights for identifying novel pharmacological targets for effective anticancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019c40b593a19f95535724601d6673fa7d7b7739" target='_blank'>
              Targeting Lipid Metabolism in Cancer Stem Cells for Anticancer Treatment
              </a>
            </td>
          <td>
            Manish Kumar Singh, Sunhee Han, Sungsoo Kim, Insug Kang
          </td>
          <td>2024-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite adjuvant treatment with endocrine therapies, estrogen receptor-positive (ER+) breast cancers recur in a significant proportion of patients. Recurrences are attributable to clinically undetectable endocrine-tolerant persister cancer cells that retain tumor-forming potential. Therefore, strategies targeting such persister cells may prevent recurrent disease. Using CRISPR-Cas9 genome-wide knockout screening in ER+ breast cancer cells, we identified a survival mechanism involving metabolic reprogramming with reliance upon mitochondrial respiration in endocrine-tolerant persister cells. Quantitative proteomic profiling showed reduced levels of glycolytic proteins in persisters. Metabolic tracing of glucose revealed an energy-depleted state in persisters where oxidative phosphorylation was required to generate ATP. A phase II clinical trial was conducted to evaluate changes in mitochondrial markers in primary ER+/HER2-breast tumors induced by neoadjuvant endocrine therapy (NCT04568616). In an analysis of tumor specimens from 32 patients, tumors exhibiting residual cell proliferation after aromatase inhibitor-induced estrogen deprivation with letrozole showed increased mitochondrial content. Genetic profiling and barcode lineage tracing showed that endocrine-tolerant persistence occurred stochastically without genetic predisposition. Mice bearing cell line- and patient-derived xenografts were used to measure the anti-tumor effects of mitochondrial complex I inhibition in the context of endocrine therapy. Pharmacological inhibition of complex I suppressed the tumor-forming potential of persisters and synergized with the anti-estrogen fulvestrant to induce regression of patient-derived xenografts. These findings indicate that mitochondrial metabolism is essential in endocrine-tolerant persister ER+ breast cancer cells and warrant the development of treatment strategies to leverage this vulnerability in the context of endocrine-sensitive disease. Statement of Significance Endocrine-tolerant persister cancer cells that survive endocrine therapy can cause recurrent disease. Persister cells exhibit increased energetic dependence upon mitochondria for survival and tumor re-growth potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084cea4bb7d59ba91f39cff9e7cec6f0ffec0615" target='_blank'>
              Endocrine persistence in ER+ breast cancer is accompanied by metabolic vulnerability in oxidative phosphorylation
              </a>
            </td>
          <td>
            Steven Tau, Mary D. Chamberlin, Huijuan Yang, Jonathan D Marotti, Alyssa M Roberts, Melissa M. Carmichael, Lauren E. Cressey, Christo Dragnev, Eugene Demidenko, Riley A. Hampsch, Shannon M. Soucy, Fred W. Kolling, Kimberley S. Samkoe, James V. Alvarez, A. Kettenbach, Todd W. Miller
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The relationship between BRAF-V600E mutations, mitochondrial DNA amplification and microsatellite instability-high (MSI-H) in colorectal cancer (CRC) has yet to be fully elucidated. The aim of the present study was to assess the association between the MSI status and BRAF-V600E gene mutations/clinicopathological features/mitochondrial DNA amplification in CRC. A non-interventional study analysis was performed using the clinicopathological features of 455 patients with CRC. Immunohistochemistry was used to evaluate four mismatch repair proteins (MutS homolog 2, MutS homolog 6, MutL homolog 1 and postmeiotic segregation increased 2), Ki-67 index, and programmed cell death protein 1 (PD-1) and programmed cell death-ligand 1 (PD-L1) expression. Additionally, PCR coupled with capillary electrophoresis were used to ascertain the MSI status. Moreover, amplification refractory mutation system-PCR was used to detect BRAF-V600E gene mutation and fluorescence in situ hybridization analysis was used to assess mitochondrial DNA. A total of 455 patients were divided into the MSI high (MSI-H) group (n=52) and microsatellite stability (MSS) group (n=403) based on their MSI status. Compared with the results of immunohistochemistry of four mismatch repair proteins, the consistency rate between mismatch repair protein deficiency and MSI was 94.23%. There were significant differences in PD-L1, primary tumor site, clinical stage, degree of differentiation, tumor size, lymph node metastasis and the occurrence of multiple primary tumors between the MSI-H group and MSS group (P<0.05 or P<0.001). However, there were no significant differences for sex, age, PD-1, Ki-67 expression and BRAF-V600E. The 24–60-month survival rate of the patients in the MSI-H group was significantly higher than that of those in the MSS group (P<0.05). Furthermore, the number of mitochondrial DNA was significantly amplified in the MSI-H group. In conclusion, the present study demonstrated that the combined detection of PD-L1 and MSI in patients with CRC can provide more accurate and effective guidance for personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5feaa7383704c34d700372d523136046568f74f8" target='_blank'>
              Association between microsatellite instability status, clinicopathological features and mitochondrial DNA amplification in patients with colorectal cancer
              </a>
            </td>
          <td>
            Junmi Lu, Hong Tan, Tao Guo, Xi Chen, Zhongyi Tong
          </td>
          <td>2024-09-26</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The role of somatic mitochondrial DNA (mtDNA) mutations in leukemogenesis remains poorly characterized. To determine the impact of somatic mtDNA mutations on the process, we assessed the leukemogenic potential of hematopoietic progenitor cells (HPCs) from mtDNA mutator mice (Polg D257A) with or without NMyc overexpression. We observed a higher incidence of spontaneous leukemogenesis in recipients transplanted with heterozygous Polg HPCs and a lower incidence of NMyc-driven leukemia in those with homozygous Polg HPCs compared to controls. Although mtDNA mutations in heterozygous and homozygous HPCs caused similar baseline impairments in mitochondrial function, only heterozygous HPCs responded to and supported altered metabolic demands associated with NMyc overexpression. Homozygous HPCs showed altered glucose utilization with pyruvate dehydrogenase inhibition due to increased phosphorylation, exacerbated by NMyc overexpression. The impaired growth of NMyc-expressing homozygous HPCs was partially rescued by inhibiting pyruvate dehydrogenase kinase, highlighting a relationship between mtDNA mutation burden and metabolic plasticity in leukemogenesis. TEASER Somatic mtDNA mutations as drivers of metabolic change in the development of leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e148b02e76091e36c8c39ac3fcae3b6950d1338" target='_blank'>
              Somatic mtDNA Mutation Burden Shapes Metabolic Plasticity in Leukemogenesis
              </a>
            </td>
          <td>
            Xiujie Li-Harms, Jingjun Lu, Yu Fukuda, John Lynch, Aditya Sheth, Gautam Pareek, Marcin Kaminski, Hailey Ross, Christopher W. Wright, Huiyun Wu, Yong-Dong Wang, Geoff Neal, Amber Smith, Peter Vogel, S. Pounds, John Schuetz, Min Ni, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="In recent years, it has been demonstrated that many of the pervasive genetic defects throughout cancerogenesis occur in genes encoding chromatin regulators (CRs). We analyzed the distribution and characteristics of well-studied CRs across tens of thousands of tumor samples. Our analysis revealed that tumors with mutations in CRs are associated with high tumor mutational burden (TMB). The co-occurrence of mutations in multiple CRs was linked with a further increase in TMB. Given that TMB may predict the clinical response to immune checkpoint inhibitor (ICI) treatment, we investigated the relationship between mutations in CRs and ICI response. We found that patients harboring mutations in CRs exhibited improved responses to ICI treatment, comparable to those with deficiencies in canonical DNA repair pathways. Overall, this study uncovered significant relationships between mutations in chromatin regulators and critical features of cancer, underscoring the need for further functional and clinical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a483b13f9b49765baf5b4ffa7cd0de2126755c55" target='_blank'>
              Tumors with mutations in chromatin regulators are associated with higher mutational burden and improved response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            G. Kungulovski, M. Gjorgjievska, S. Mehandziska, D. Bukovec, M. Risteski, I. Kungulovski, Z. Mitrev
          </td>
          <td>2024-10-16</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genomic copy-number variations (CNVs) that can cause neurodevelopmental disorders often encompass many genes, which complicates our understanding of how individual genes within a CNV contribute to pathology. MECP2 duplication syndrome (MDS or MRXSL in OMIM; OMIM#300260) is one such CNV disorder caused by duplications spanning methyl CpG-binding protein 2 (MECP2) and other genes on Xq28. Using an antisense oligonucleotide (ASO) to normalize MECP2 dosage is sufficient to rescue abnormal neurological phenotypes in mouse models overexpressing MECP2 alone, implicating the importance of increased MECP2 dosage within CNVs of Xq28. However, because MDS CNVs span MECP2 and additional genes, we generated human neurons from multiple MDS patient-derived induced pluripotent cells (iPSCs) to evaluate the benefit of using an ASO against MECP2 in a MDS human neuronal context. Importantly, we identified a signature of genes that is partially and qualitatively modulated upon ASO treatment, pinpointed genes sensitive to MeCP2 function, and altered in a model of Rett syndrome, a neurological disorder caused by loss of MeCP2 function. Furthermore, the signature contained genes that are aberrantly altered in unaffected control human neurons upon MeCP2 depletion, revealing gene expression programs qualitatively sensitive to MeCP2 levels in human neurons. Lastly, ASO treatment led to a partial rescue of abnormal neuronal morphology in MDS neurons. All together, these data demonstrate that ASOs targeting MECP2 benefit human MDS neurons. Moreover, our study establishes a paradigm by which to evaluate the contribution of individual genes within a CNV to pathogenesis and to assess their potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/204ab0081ef3960da9757eb616c690c4273ee78b" target='_blank'>
              Modeling antisense oligonucleotide therapy in MECP2 duplication syndrome human iPSC-derived neurons reveals gene expression programs responsive to MeCP2 levels.
              </a>
            </td>
          <td>
            Sameer S. Bajikar, Yehezkel Sztainberg, Alexander J. Trostle, Harini P Tirumala, Ying-Wooi Wan, Caroline L Harrop, J. Bengtsson, Claudia M B Carvalho, D. Pehlivan, Bernhard Suter, J. Neul, Zhandong Liu, P. Jafar‐nejad, Frank Rigo, H. Zoghbi
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74133d155a5e2d58ae93fb17f6009e99fe00ed5e" target='_blank'>
              A comprehensive investigation of associations between cell death pathways and molecular and clinical features in pan-cancer.
              </a>
            </td>
          <td>
            Yin A He, Xiaosheng Wang
          </td>
          <td>2024-11-02</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Natural killer (NK) cells exhibit potent cytotoxic activity against various cancer cell types. Over the past five decades, numerous methodologies have been employed to elucidate the intricate molecular mechanisms underlying NK cell-mediated tumor control. While significant progress has been made in elucidating the interactions between NK cells and tumor cells, the regulatory factors governing NK cell-mediated tumor cell destruction are not yet fully understood. This includes the diverse array of tumor ligands recognized by NK cells and the mechanisms that NK cells employ to eliminate tumor cells. Methods In this study, we employed a genome-wide CRISPR/Cas9 screening approach in conjunction with functional cytotoxicity assays to delineate the proteins modulating the susceptibility of colon adenocarcinoma HCT-116 cells to NK cell-mediated cytotoxicity. Results Analysis of guide RNA (gRNA) distribution in HCT-116 cells that survived co-incubation with NK cells identified ICAM-1 as a pivotal player in the NKp44-mediated immune synapse, with NKp44 serving as an activating receptor crucial for the elimination of HCT-116 tumor cells by NK cells. Furthermore, disruption of genes involved in the apoptosis or IFN-γ signaling pathways conferred resistance to NK cell attack. We further dissected that NK cell-derived IFN-γ promotes mitochondrial apoptosis in vitro and exerts control over B16-F10 lung metastases in vivo. Conclusion Monitoring ICAM-1 levels on the surface of tumor cells or modulating its expression should be considered in the context of NK cell-based therapy. Additionally, considering the diffusion properties of IFN-γ, our findings highlight the potential of leveraging NK cell-derived IFN-γ to enhance direct tumor cell killing and facilitate bystander effects via cytokine diffusion, warranting further investigation. WHAT IS ALREADY KNOWN ON THIS TOPIC NK cells play a crucial role in identifying and eliminating various cancer cell types. However, the mechanisms that regulate NK cell-mediated destruction of tumor cells are not yet fully understood. This involves the array of tumor ligands that NK cells recognize and the processes they utilize to carry out tumor cell elimination. WHAT THIS STUDY ADDS Our research emphasizes the critical role of ICAM-1 in NKp44-mediated destruction of HCT-116 tumor cells. Additionally, we found that interfering with genes related to apoptosis or IFN-γ signaling pathways increased resistance to NK cell attack. We showed that IFN-γ produced by NK cells induces mitochondrial apoptosis in vitro and helps regulate B16-F10 lung metastases in vivo. HOW THIS STUDY MIGHT AFFECT RESEARCH, PRACTICE OR POLICY Given the ability of IFN-γ to diffuse, our findings suggest that NK cell-derived IFN-γ can be harnessed to directly kill tumor cells and trigger bystander effects through cytokine spread. This approach holds promise for further exploration. Additionally, assessing or manipulating ICAM-1 levels on tumor cell surfaces could enhance the effectiveness of NK cell-based therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7312fc3a41477554893f985736277783c1d1ccb1" target='_blank'>
              Genome-wide CRISPR/Cas9 screen reveals factors that influence the susceptibility of tumor cells to NK cell-mediated killing
              </a>
            </td>
          <td>
            S. Guia, A. Fenis, Justine Galluso, Hakim Medjouel, Bertrand Escalière, Angelica Modelska, M. Vienne, Noella Lopes, Amélie Pouchin, Benjamin Rossi, Laurent Gauthier, Sandrine Roulland, Eric Vivier, Émilie Narni-Mancinelli
          </td>
          <td>2024-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="While cell fate determination and maintenance are important in establishing and preserving tissue identity and function during development, aberrant cell fate transition leads to cancer cell heterogeneity and resistance to treatment. Here, we report an unexpected role for the transcription factor p63 (Trp63/TP63) in the fate choice of the squamous versus neuroendocrine lineage in esophageal development and malignancy. Deletion of p63 results in extensive neuroendocrine differentiation in the developing mouse esophagus and esophageal progenitors derived from human embryonic stem cells. In human esophageal neuroendocrine carcinoma (eNEC) cells, p63 is transcriptionally silenced by EZH2-mediated H3K27 trimethylation (H3K27me3). Up-regulation of the major p63 isoform ΔNp63α, through either ectopic expression or EZH2 inhibition, promotes squamous transdifferentiation of eNEC cells. Together, these findings uncover p63 as a rheostat in coordinating the transition between squamous and neuroendocrine cell fates during esophageal development and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d8a507bf29b53a6d31ddde60eea0283fa6f1ef" target='_blank'>
              Epigenetic regulation of p63 blocks squamous-to-neuroendocrine transdifferentiation in esophageal development and malignancy
              </a>
            </td>
          <td>
            Yongchun Zhang, Dimitris Karagiannis, Helu Liu, Mi Lin, Yinshan Fang, Ming Jiang, Xiao Chen, Supriya Suresh, Haidi Huang, Junjun She, F. Shi, Jiangying Liu, Dan Luo, J. C. Angel, Guangtan Lin, Patrick Yang, W. El-Rifai, A. Zaika, A. Oro, Kuancan Liu, Anil K. Rustgi, Timothy C Wang, Chao Lu, Jianwen Que
          </td>
          <td>2024-10-09</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="To dissect variant-function relationships in the KRAS oncoprotein, we performed deep mutational scanning (DMS) screens for both wild-type and KRASG12D mutant alleles. We defined the spectrum of oncogenic potential for nearly all possible KRAS variants, identifying several novel transforming alleles and elucidating a model to describe the frequency of KRAS mutations in human cancer as a function of transforming potential, mutational probability, and tissue-specific mutational signatures. Biochemical and structural analyses of variants identified in a KRASG12D second-site suppressor DMS screen revealed that attenuation of oncogenic KRAS can be mediated by protein instability and conformational rigidity, resulting in reduced binding affinity to effector proteins, such as RAF and PI3-kinases, or reduced SOS-mediated nucleotide exchange activity. These studies define the landscape of single amino acid alterations that modulate the function of KRAS, providing a resource for the clinical interpretation of KRAS variants and elucidating mechanisms of oncogenic KRAS inactivation for therapeutic exploitation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a23afebcd7cdf77fb0cc51e7cfdf5ed4086508" target='_blank'>
              Comprehensive structure-function analysis reveals gain- and loss-of-function mechanisms impacting oncogenic KRAS activity
              </a>
            </td>
          <td>
            Jason J. Kwon, Julien Dilly, Shengwu Liu, Eejung Kim, Yuemin Bian, S. Dharmaiah, Timothy H Tran, Kevin S. Kapner, S. H. Ly, Xiaoping Yang, Dana Rabara, Timothy Waybright, Andrew O. Giacomelli, Andrew L. Hong, Sean Misek, Belinda Wang, Arvind Ravi, John G Doench, R. Beroukhim, Christopher T. Lemke, Kevin M. Haigis, Dominic Esposito, David E. Root, D. Nissley, Andrew G Stephen, Frank McCormick, D. Simanshu, William C. Hahn, Andrew J. Aguirre
          </td>
          <td>2024-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>146</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/145eeb292c624be8f86f64a77055ade9f8ec3328" target='_blank'>
              Tumor microenvironment as a complex milieu driving cancer progression: a mini review.
              </a>
            </td>
          <td>
            Zheng Li, Jing Li, Xiaolei Bai, Xufeng Huang, Qi Wang
          </td>
          <td>2024-09-28</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="The underlying gene regulatory networks (GRN) that govern leukemia stem cells (LSC) in acute myeloid leukemia (AML) and hematopoietic stem cells (HSC) are not well understood. Here, we identified GRNs by integrating gene expression (GE) and chromatin accessibility data derived from functionally defined cell populations enriched for HSC and LSC. We analyzed n=32 LSC+ and n=32 LSC-cell fractions from n=22 AML patients, along with n=7 stem and n=10 progenitor enriched cell populations sorted from human umbilical cord blood (hUCB), producing a database of n≈17,000 transcription factor (TF) regulatory interactions for hUCB-HSPC and AML. We developed an iterative algorithm that associates the degree of chromatin openness with TF binding preferences, and the GE of candidate TF and target genes within 100kb upstream of transcription start sites. A putative regulatory structure was found to be enriched in HSC-enriched cell populations, comprising TF-target gene interactions between ETS1, EGR1, RUNX2, and ZNF683 oriented in a self-reinforcing configuration. A regulatory loop comprising FOXK1 and MEIS1, rather than the 4-factor HSC subnetwork, was detected in the LSC-specific GRN. The core HSC and LSC TF networks were extended using protein-protein interaction (PPI) data to determine connectivity with interacting genes whose expression strongly associated with LSC/HSC frequency estimates, producing a database of n=103,516 PPI target pathways. The effect of perturbing genes along the identified pathways on functional HSC and LSC frequency was predicted based on statistical regression analyses. To validate GRN predictions, we used pharmacologic and CRISPR targeting, in addition to re-examining published functional data associated with several network nodes that were predicted to impact stemness. Notably, we found that inhibition of CDK6 in AML samples markedly reduced LSC numbers as assessed in de novo serial xenotransplantation studies (fold change ≈ 10), as predicted by the LSC GRN model. Additionally, in-house CRISPR-based knockdown of ETS1 resulted in a significant decrease in HSC quiescence-associated microRNA-126 expression, and increased HSC frequency. Taken together, our models provide a comprehensive view of the underlying regulatory structures governing functional human HSC and LSC. This approach has translational potential as it can be used as a high-throughput in-silico screening tool for the systematic identification of gene targets for LSC elimination and HSC expansion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41e7b578bd41a7f8073095de491190663b9cecb5" target='_blank'>
              Distinct gene regulatory networks govern hematopoietic and leukemia stem cells
              </a>
            </td>
          <td>
            Boyang Zhang, Alexander Murison, Angelica Varesi, Naoya Takayama, Liqing Jin, Nathan Mbong, Erwin M. Schoof, Stephanie Z. Xie, Amanda Mitchell, J. Etchin, A. T. Look, Mathieu Lupien, Mark D. Minden, J. C. Wang, Peter W. Zandstra, Elvin Wagenblast, John E. Dick, Stanley W.K. Ng
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Telomeres—special DNA–protein structures at the ends of linear eukaryotic chromosomes—define the proliferation potential of cells. Extremely short telomeres promote a DNA damage response and cell death to eliminate cells that may have accumulated mutations after multiple divisions. However, telomere elongation is associated with the increased proliferative potential of specific cell types, such as stem and germ cells. This elongation can be permanent in these cells and is activated temporally during immune response activation and regeneration processes. The activation of telomere lengthening mechanisms is coupled with increased proliferation and the cells’ need for energy and building resources. To obtain the necessary nutrients, cells are capable of finely regulating energy production and consumption, switching between catabolic and anabolic processes. In this review, we focused on the interconnection between metabolism programs and telomere lengthening mechanisms during programmed activation of proliferation, such as in germ cell maturation, early embryonic development, neoplastic lesion growth, and immune response activation. It is generally accepted that telomere disturbance influences biological processes and promotes dysfunctionality. Here, we propose that metabolic conditions within proliferating cells should be involved in regulating telomere lengthening mechanisms, and telomere length may serve as a marker of defects in cellular functionality. We propose that it is possible to reprogram metabolism in order to regulate the telomere length and proliferative activity of cells, which may be important for the development of approaches to regeneration, immune response modulation, and cancer therapy. However, further investigations in this area are necessary to improve the understanding and manipulation of the molecular mechanisms involved in the regulation of proliferation, metabolism, and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ab06d09eb5b7b71fe7be1c13b67665bab6c417" target='_blank'>
              Telomere Reprogramming and Cellular Metabolism: Is There a Link?
              </a>
            </td>
          <td>
            M. Rubtsova, Denis A. Nikishin, M. Y. Vyssokikh, Maria S. Koriagina, Andrey V. Vasiliev, O. A. Dontsova
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary When oncogenes, which are genes that can cause cancer, become active, they disrupt normal cell processes, especially DNA replication. This disruption is known as replication stress (RS), in which the DNA copying process is stalled or damaged. To prevent cancer, cells usually have a defense system called the DNA damage response (DDR), which can prevent damaged cells from turning into cancer. However, some cells manage to survive this stress by developing replication stress tolerance (RST). These cells can continue to grow, leading to genomic instability (GIN), which is a key feature of cancer. GIN causes various genetic changes that make cells more likely to become cancerous and more difficult to treat. This review explains how oncogenes cause RS and how cells cope with it, leading to the development of cancer. Understanding these processes can help in developing new cancer treatments. Abstract Activation of oncogenes disturbs a wide variety of cellular processes and induces physiological dysregulation of DNA replication, widely referred to as replication stress (RS). Oncogene-induced RS can cause replication forks to stall or collapse, thereby leading to DNA damage. While the DNA damage response (DDR) can provoke an anti-tumor barrier to prevent the development of cancer, a small subset of cells triggers replication stress tolerance (RST), allowing precancerous cells to survive, thereby promoting clonal expansion and genomic instability (GIN). Genomic instability (GIN) is a hallmark of cancer, driving genetic alterations ranging from nucleotide changes to aneuploidy. These alterations increase the probability of oncogenic events and create a heterogeneous cell population with an enhanced ability to evolve. This review explores how major oncogenes such as RAS, cyclin E, and MYC induce RS through diverse mechanisms. Additionally, we delve into the strategies employed by normal and cancer cells to tolerate RS and promote GIN. Understanding the intricate relationship between oncogene activation, RS, and GIN is crucial to better understand how cancer cells emerge and to develop potential cancer therapies that target these vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c929d9fb4e293ab97bee74d87f261f4dc36600" target='_blank'>
              Tolerance of Oncogene-Induced Replication Stress: A Fuel for Genomic Instability
              </a>
            </td>
          <td>
            Taichi Igarashi, K. Yano, Syoju Endo, B. Shiotani
          </td>
          <td>2024-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d791af736d485e488b3ee8b58f9beef46966154" target='_blank'>
              Crossing epigenetic frontiers: the intersection of novel histone modifications and diseases
              </a>
            </td>
          <td>
            Weiyi Yao, Xinting Hu, Xin Wang
          </td>
          <td>2024-09-16</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103c83e1829743ff7261d3403d3aa51e48f6908e" target='_blank'>
              Histone marks identify novel transcription factors that parse CAR-T subset-of-origin, clinical potential and expansion
              </a>
            </td>
          <td>
            S. Fiorenza, Y. Zheng, J. Purushe, T. J. Bock, J. Sarthy, D. Janssens, A. S. Sheih, E. L. Kimble, D. Kirchmeier, T. Phi, J. Gauthier, A. Hirayama, S. R. Riddell, Q. Wu, R. Gottardo, D. G. Maloney, J. Y. H. Yang, S. Henikoff, C. J. Turtle
          </td>
          <td>2024-09-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/626d60f7daa063f57ba8931b4a5377577d91c573" target='_blank'>
              Oncogenic EML4-ALK assemblies suppress growth factor perception and modulate drug tolerance
              </a>
            </td>
          <td>
            D. Gonzalez-Martinez, Lee Roth, T. Mumford, Juan Guan, Anh Le, R. Doebele, Bo Huang, Asmin Tulpule, Magdalena Niewiadomska-Bugaj, T. Bivona, Lukasz J. Bugaj
          </td>
          <td>2024-11-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumor in adults, with a median survival of just over 1 year. The failure of available treatments to achieve remission in patients with glioblastoma (GBM) has been attributed to the presence of cancer stem cells (CSCs), which are thought to play a central role in tumor development and progression and serve as a treatment-resistant cell repository capable of driving tumor recurrence. In fact, the property of "stemness" itself may be responsible for treatment resistance. In this study, we identify a novel long noncoding RNA (lncRNA), cancer stem cell-associated distal enhancer of SOX2 (CASCADES), that functions as an epigenetic regulator in glioma CSCs (GSCs). CASCADES is expressed in isocitrate dehydrogenase (IDH)-wild-type GBM and is significantly enriched in GSCs. Knockdown of CASCADES in GSCs results in differentiation towards a neuronal lineage in a cell- and cancer-specific manner. Bioinformatics analysis reveals that CASCADES functions as a super-enhancer-associated lncRNA epigenetic regulator of SOX2. Our findings identify CASCADES as a critical regulator of stemness in GSCs that represents a novel epigenetic and therapeutic target for disrupting the CSC compartment in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5be2f4cc8ff228e53f5dc871cb0a020d6528f65a" target='_blank'>
              CASCADES, a novel SOX2 super-enhancer-associated long noncoding RNA, regulates cancer stem cell specification and differentiation in glioblastoma.
              </a>
            </td>
          <td>
            Uswa Shahzad, Marina Nikolopoulos, Christopher Li, Michael J Johnston, J. Wang, N. Sabha, F. Varn, Alexandra N Riemenschneider, Stacey L. Krumholtz, P. M. Krishnamurthy, Christian A. Smith, Jason Karamchandani, Jonathan K Watts, R. Verhaak, Marco Gallo, James T. Rutka, Sunit Das
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous group of malignancies with poor prognosis. AML result from the proliferation of immature myeloid cells blocked at a variable stage of differentiation. Beyond inter-patient heterogeneity, AMLs are characterized by genetic and phenotypic intra-patient heterogeneity. Despite major advances in deciphering AML biology with bulk sequencing studies, pivotal questions remain unanswered. Analyses at the single-cell level could thus transform our understanding of these neoplasms. We review recent progresses in single-cell sequencing technologies from cell processing to bioinformatic pipelines. We next discuss how single-cell applications have helped understand the genetic and functional intra-leukemic heterogeneity, emphasizing aspects related to leukemic stem cells, clonal evolution and measurable residual disease (MRD) monitoring. We finally delineate how single-cell technologies could be implemented in routine clinical practice to improve patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cfa5029eba68a02f01a208ed355c8b33eea3644" target='_blank'>
              Single-cell sequencing applications in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Nicolas Lecornec, M. Duchmann, Raphael Itzykson
          </td>
          <td>2024-11-04</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 The neuropathological assessment of central nervous system (CNS) tumors often necessitates extensive molecular analyses to arrive at a diagnosis consistent with the WHO CNS classification (2021). Key diagnostic indicators include the epigenetic profile of the tumor, MGMT methylation status, gene amplifications, deletions, duplications, gene fusions, and point mutations. Achieving comprehensive classification typically involves multiple distinct analyses, including NGS gene panels, fluorescent in situ hybridization (FISH), pyrosequencing, Sanger sequencing, and methylation bead arrays. The emergence of the nanopore long-read sequencing enables whole-genome sequencing (WGS) of a tumor sample, consolidating all diagnostically relevant information into a single analysis.



 DNA from fresh-frozen tissue from 52 CNS tumors was sequenced retrospectively using nanopore Promethion flow cells. Basecalling, methylation calling and mapping to reference genome was performed during sequencing over a period of 72 hours. Once sequencing was completed, copy-number variations (CNVs), structural variants (SVs) and single-nucleotide variants (SNVs) were called using the Epi2me software suite. Methylation on CpG sites overlapping the EPIC 850K array were extracted and used for classification using the MNP-flex platform. All relevant findings were compared to results from the routine diagnostic work-up.



 Nanopore WGS provided on average 35x coverage after 72 hours of sequencing on a single flow cell. The average turnaround time from obtaining a tumor biopsy to producing a report was five days. Classification based on the methylation profile was concordant with molecular pathology in 46 of 52 cases (88%). Three cases were classified as control tissue, characterized by low tumor-cell content, while four cases displayed discordance with the initial classification. All relevant gene-fusion events and amplifications were successfully identified in the nanopore data, with two additional relevant findings reported that had not been previously detected. Seventy-three clinically relevant SNVs were reported in the nanopore data matching previous findings while seven SNVs were unreported, six of which were from samples with low tumor content.



 Nanopore whole-genome sequencing showed high concordance with traditional methods for CNS tumor classification. It holds potential for improving diagnostic accuracy and shortening the turnaround time from surgery to complete molecular diagnosis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9c13c79de86cc5a208f04c0338de38086b1cdd" target='_blank'>
              JS04.4.A SHORTENING THE TURNAROUND TIME FROM SURGERY TO COMPLETE MOLECULAR DIAGNOSIS
              </a>
            </td>
          <td>
            S. Halldorsson, H. Leske, R. M. Nagymihaly, A. Patel, F. Sahm, E. Vik-Mo
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite the advances in the understanding and treatment of myeloproliferative neoplasm (MPN), the disease remains incurable with the risk of evolution to AML or myelofibrosis (MF). Unfortunately, the evolution of the disease to MF remains still poorly understood impeding preventive and therapeutic options. Recent studies in solid tumor microenvironment and organ fibrosis have shed instrumental insights on their respective pathogenesis and drug resistance, yet such precise data are lacking in MPN. In this study, through a patient-sample driven transcriptomic and epigenetic description of the MF microenvironment landscape and cell-based analyses, we identify HOXB7 overexpression and more precisely a novel TGFβ-Wnt-HOXB7 pathway as associated to a pro-fibrotic and pro-osteoblastic biased differentiation of mesenchymal stromal cells (MSCs). Using gene-based and chemical inhibition of this pathway we reverse the abnormal phenotype of MSCs from myelofibrosis patients, providing the MPN field with a potential novel target to prevent and manage evolution to MF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f48e1042ddaa6b3683b1258ba1c11b165593dd33" target='_blank'>
              Comprehensive analysis of mesenchymal cells reveals a dysregulated TGF-β/Wnt/HOXB7 axis in patients with myelofibrosis.
              </a>
            </td>
          <td>
            Saravanan Ganesan, Sarah Awan-Toor, F. Guidez, N. Maslah, Rifkath Rahimy, Céline Aoun, P. Gou, Chloé Guiguen, J. Soret, Odonchimeg Ravdan, Valeria Bisio, Nicolas Dulphy, Camille Lobry, M. Schlageter, Michèle Souyri, Stephane Giraudier, J. Kiladjian, Christine Chomienne, B. Cassinat
          </td>
          <td>2024-10-29</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Currently, Cancer is the number one killer of human health. The death rate from cancer is increasing year over year as the population ages and grows, particularly since 2019, when there were almost 10 million cases globally recorded by the World Health Organization every year. In light of this trend, it is imperative to continue developing medications and to update and iterate on anti-cancer therapeutic technologies. CAR-T cell therapy, microphage-based therapy, oncolytic virotherapy, and checkpoint blockade therapy are the most popular cancer treatments, but a few patients still cannot tolerate these therapies. A post-optimized strategy is to use gene editing to alter the characteristics of cancer treatment. New therapeutic options for the treatment of complex disorders have become available with the development of CRISPR/Cas9 gene editing technology in recent years. This method is widely used in market research for gene knockdown, endogenous gene expression, and chromosome locus markers because of its accuracy and efficiency in processing harmful gene fragments. Also, it is extremely valuable from a medical standpoint when researching immunomodulator resistance. Despite its great potential, there remain concerns over the technology’s usefulness and efficacy and negative prognostic reactions. This review primarily focuses on the current use of CRISPR/Cas9 in cancer immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cb7ad71a9b9241a33354f318dab5664b26cb85a" target='_blank'>
              The Application of CRISPR/Cas9 in Cancer Immunotherapy
              </a>
            </td>
          <td>
            Jiale Yu
          </td>
          <td>2024-10-29</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromate has been shown to dysregulate epigenetic mechanisms such as DNA methylation, leading to changes in gene expression and genomic instability. However, most in vitro studies are limited to short incubation periods, although chronic exposure may be more relevant for both environmental and occupational exposure. In this study, human adenocarcinoma A549 cells were treated with 1, 2 or 5 µM chromate for 24 h and compared with incubations with 0.2, 0.5 or 1 µM chromate for 1 to 5 weeks. Chromium accumulated in a pronounced time- and concentration-dependent manner after short-term treatment, whereas a plateau of intracellular chromium content was observed after long-term treatment. While short-term treatment induced a G2 arrest of the cell cycle, this effect was not observed after long-term treatment at lower concentrations. The opposite was observed for global DNA methylation: while short-term treatment showed no effect of chromate, significant dose-dependent hypomethylation was observed in the long-term experiments. Time-dependent effects were also observed in a high-throughput RT-qPCR gene expression analysis, particularly in genes related to the inflammatory response and DNA damage response. Taken together, the results suggest specific differences in toxicity profiles when comparing short-term and long-term exposure to chromate in A549 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdefb7e574e70e9cd635b090972e4a01321ac941" target='_blank'>
              Chromate Affects Gene Expression and DNA Methylation in Long-Term In Vitro Experiments in A549 Cells
              </a>
            </td>
          <td>
            Franziska Fischer, Sandra Stößer, Lisa Wegmann, Eva Veh, Tatjana Lumpp, Marlene Parsdorfer, Paul Schumacher, Andrea Hartwig
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f8a43614dcaaf12716b1614411bf7e1f17fff04" target='_blank'>
              AXL as immune regulator and therapeutic target in Acute Myeloid Leukemia: from current progress to novel strategies
              </a>
            </td>
          <td>
            N. Vandewalle, Nathan De Beule, A. De Becker, E. De Bruyne, E. Menu, K. Vanderkerken, Karine Breckpot, N. Devoogdt, K. De Veirman
          </td>
          <td>2024-10-04</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 The alteration of the glioblastoma genome by epigenetic mechanisms that share functions with normal developmental processes, such as self-renewal and fate specification of neural stem cells (NSC), is a key piece of evidence that links brain cancer pathogenesis with dysregulated stem cell functions. A patient-specific comparison of glioblastoma stem cells (GSC) with NSC, a putative cell of origin of at least a proportion of these tumours, is not feasible as patient-matched endogenous NSC are not surgically accessible and all epigenetic studies in glioblastoma have so far compared epigenetic changes of different tumours with each other or to comparators obtained from foetal brains or an unrelated donor. We reasoned that availability of syngeneic GSC and NSC pairs would allow to identify crucial epigenetic differences on a patient-specific basis and would provide essential therapeutic contrast to define disease-and patient-intrinsic biomarkers of drug response that are less confounded by germline variation.



 We have derived GSC from IDH-wildtype glioblastoma and harnessed state-of-the-art stem cell technologies to generate patient-matched fibroblast-derived EPSC, which were induced to NSC (iNSC). We generated transcriptome, DNA methylome, ChIP-Seq (H3K4me3, H3K27ac, H3K3me3, H3K27me3) and ATACSeq datasets for 10 GIC/iNSC pairs.



 Comparative integrated analysis has shown that redistribution of selected histone marks as well as shifts in chromatin states across the genome identifies differentially expressed genes, including novel druggable regulatory mechanisms in GSC, which are specific of the neoplastic context. In particular, we identified METTL7B as an essential regulator of lineage specification and plasticity in human glioblastoma, with impact on both tumour size and invasiveness in an in vivo xenograft model. Single cell transcriptomic analysis of these tumours and of cerebral organoids derived from expanded potential stem cells (EPSCs) overexpressing METTL7B reveal a regulatory role for the gene in the neural stem cells-to-astrocyte differentiation trajectory. Mechanistically, METTL7B downregulates the expression of key neuronal differentiation players, including SALL2, via post-translational modifications of histone marks.



 SYNGN is an epigenetic platform which can identify novel disease mechanisms and druggable targets in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be96bd21ac249c6fcb9083c430d82766e4fcbf5a" target='_blank'>
              P01.14.B SYNGN: A NOVEL EPIGENETIC PLATFORM TO DISCOVER PATIENT-SPECIFIC DISEASE MECHANISMS AND DRUGGABLE TARGETS IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            S. Marino, C. Vinel, M. Constantinou
          </td>
          <td>2024-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Desmoplastic small round cell tumor (DSRCT) is an aggressive sarcoma subtype that is driven by the EWS-WT1 chimeric transcription factor. The prognosis for DSRCT is poor, and major advances in treating DSCRT have not occurred for over two decades. To identify effective therapeutic approaches to target DSRCT, we conducted a high-throughput drug sensitivity screen in a DSRCT cell line assessing chemosensitivity profiles for 79 small-molecule inhibitors. DSRCT cells were sensitive to PARP and ATR inhibitors (PARPi, ATRi), as monotherapies and in combination. These effects were recapitulated using multiple clinical PARPi and ATRi in three biologically distinct, clinically-relevant models of DSRCT, including cell lines, a patient-derived xenograft (PDX)-derived organoid model, and a cell line-derived xenograft mouse model. Mechanistically, exposure to a combination of PARPi and ATRi caused increased DNA damage, G2/M checkpoint activation, micronuclei accumulation, replication stress, and R-loop formation. EWS-WT1 silencing abrogated these phenotypes and was epistatic with exogenous expression of the R-loop resolution enzyme RNase H1 in reversing the sensitivity to PARPi and ATRi monotherapies. The combination of PARPi and ATRi also induced EWS-WT1-dependent cell-autonomous activation of the cGAS/STING innate immune pathway and cell surface expression of PD-L1. Taken together, these findings point towards a role for EWS-WT1 in generating R-loop-dependent replication stress that leads to a targetable vulnerability, providing a rationale for the clinical assessment of PARPi and ATRi in DSRCT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33f00435911403846b95b540256a684bbb112555" target='_blank'>
              Replication Stress is an Actionable Genetic Vulnerability in Desmoplastic Small Round Cell Tumors.
              </a>
            </td>
          <td>
            Asuka Kawai-Kawachi, Madison M Lenormand, Clémence Astier, Noé Herbel, Meritxell B Cutrona, Carine Ngo, M. Garrido, Thomas Eychenne, N. Dorvault, Laetitia Bordelet, Fei Fei Song, Ryme Bouyakoub, Anastasia Loktev, Antonio Romo-Morales, C. Henon, Léo Colmet-Daage, Julien Vibert, Marjorie Drac, R. Brough, Etienne Schwob, Oliviano Martella, G. Pinna, Janet M Shipley, Sibylle Mittnacht, Astrid Zimmermann, Aditi Gulati, Olivier Mir, Axel Le Cesne, M. Faron, Charles Honoré, C. J. Lord, Roman Chabanon, S. Postel-Vinay
          </td>
          <td>2024-10-16</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Errors in mitosis can contribute to aneuploidy and CIN and play a pivotal role in cancer. So the identification of altered mitotic regulators can contribute to the understanding of the development and progression of breast cancer. In the present study we used an in vitro model of disease progression (the MCF10A series of BC continuum) and analyzed the errors of chromosome segregation that occur during the progression of the disease. Our findings indicated that the MCF10A series exhibited several abnormalities in chromosome segregation and its frequency increased with the disease progression. These errors included anaphase lagging chromosomes, micronuclei, nuclear buds, nucleoplasmic bridges, errors of chromosome alignment, and centrosome loss/amplification. Moreover, the presence of centrosome amplification disrupted the proper orientation of the mitotic spindle, resulting in the generation asymmetrical cell lines and aneuploidy in the MCF10A series. Hyper stable kinetochore-microtubule (kt-MT) attachment was also found in premalignant, preinvasive, and invasive cell lines, which can also explain the presence of errors of chromosome alignment. The human transcriptome array also determined possible negative regulators of ciliogenesis that can explain the mechanism of chromosome missegregation that lead to CIN found in the MCF10A series. Collectively, these findings highlight the importance of mitotic defects in the progression of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acf9391e3d0717b055bd26e1631bad45d557802" target='_blank'>
              A characterization of mitotic and centrosomal defects in a continuum model of Breast Cancer
              </a>
            </td>
          <td>
            Alexsandro dos Santos, C. Diorio, Francine Durocher, S. Elowe
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="T cells engineered to express chimeric-antigen receptors (CAR-T cells) can effectively control relapsed and refractory haematological malignancies in the clinic. However, the successes of CAR-T cell therapy have not been recapitulated in solid tumours due to a range of barriers such as immunosuppression, poor infiltration, and tumour heterogeneity. Numerous strategies are being developed to overcome these barriers, which include improving culture conditions and manufacturing protocols, implementing novel CAR designs, and novel approaches to engineering the T cell phenotype. In this review, we describe the various emerging strategies to improve CAR T cell therapy for solid tumours. We specifically focus on new strategies to modulate cell function and fate that have precipitated from the growing knowledge of transcriptional circuits driving T cell differentiation, with the ultimate goal of driving more productive anti-tumour T cell immunity. Evidence shows that enrichment of particular phenotypic subsets of T cells in the initial cell product correlates to improved therapeutic responses and clinical outcomes. Furthermore, T cell exhaustion and poor persistence are major factors limiting therapeutic efficacy. The latest preclinical work shows that targeting specific master regulators and transcription factors can overcome these key barriers, resulting in superior T cell therapeutic products. This can be achieved by targeting key transcriptional circuits promoting memory-like phenotypes or sustaining key effector functions within the hostile tumour microenvironment. Additional discussion points include emerging considerations for the field such as (i) targeting permutations of transcription factors, (ii) transient expression systems, (iii) tissue specificity, and (iv) expanding this strategy beyond CAR-T cell therapy and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9ed896d01fae17546126ff879b8774824a0e5ef" target='_blank'>
              Transcriptional rewiring in CD8+ T cells: implications for CAR-T cell therapy against solid tumours
              </a>
            </td>
          <td>
            Shamini Srinivasan, Jesse Armitage, J.A. Nilsson, Jason Waithman
          </td>
          <td>2024-09-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bba8ffa37402689d1b771c33d5b55e131826ac5" target='_blank'>
              A narrative review: exploring viral-induced malignancies through the lens of dysregulated cellular metabolism and glucose transporters
              </a>
            </td>
          <td>
            Amirhossein Shahpar, Vahideh Hamidi Sofiani, Nazanin Zeinali Nezhad, Marzieh Charostad, Reza Ghaderi, Niloofar Farsiu, Amin Karimzadeh Kiskani, Sara Pezeshki, M. Nakhaie
          </td>
          <td>2024-10-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Since the approval of the first checkpoint inhibitor, ipilimumab (α-CTLA4), there continues to be enthusiasm around leveraging the immune system to eradicate solid tumors. However, unknowns about the cellular mechanism(s) that cause some patients to durably respond while others relapse, continue to stymie the field. To support ongoing efforts elucidating cell subsets responsible for differences in patient responses, we aim to evaluate the functional heterogeneity of T-cell subsets responding to checkpoint inhibitors in mouse models of solid tumors. Single-cell RNA-seq has enabled discoveries of transcriptionally defined cell subsets, but understanding their function require inference from transcription, flow-based validation biased by pre-selection of subsets with specific markers, or reliance on surrogate markers for cell-extrinsic functions (e.g., cytotoxicity). Thus, functional heterogeneity could be obscured. To address this, we leveraged a novel technology that converges live, cellular functions with transcriptomics at the resolution of single cell-cell interactions. This technology captures tens of thousands of individual cells or defined combinations of groups of cells in bio-compatible hydrogel compartments, enabling longitudinal imaging over days to weeks. Throughout this period, functions are characterized via bright-field or immunofluorescence readouts, including antibodies, cytokine capture beads, and dyes. Cells are then lysed within compartments to generate barcoded libraries for mRNA-sequencing and gene signatures from individual cells computationally deconvoluted. With this technology, we developed a novel assay that enables longitudinal, unbiased functional characterization of individual T cells mapped to single-cell transcriptomics. Single mouse primary T cells were encapsulated with one or more MC38 target cells and imaged over time. We validated combinations of different readouts can be measured from the same T-cell, including cell-counts for proliferation, CD8/3 expression to differentiate T cells from target cells, activation markers, and / or target cell death via Sytox. Thus, single T cells can be multi-functionally evaluated over time, enabling the study of attributes inaccessible at high scale with prior technologies, including kinetics of killing or the number of target cells killed by a single T-cell. For assays combining cytotoxicity and surface receptors, we generated barcoded libraries from functionally characterized T cells for sequencing and developed an analytical pipeline that deconvolutes mRNA from single T cells mapped to their multi-functional profiles, to identify novel pathways that regulate anti-tumor functions. We will leverage this workflow to functionally characterize single T cells in tumor-bearing mice treated with checkpoint inhibitors and map the data to underlying transcriptomics. In doing so, we aim to discover functional subsets, potentially with identical transcriptional profiles, to support hypothesis generation around mechanisms governing anti-tumor efficacy of checkpoint inhibitors.
 Citation Format: Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, Pier F Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer. Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor Immunology and Immunotherapy; 2024 Oct 18-21; Boston, MA. Philadelphia (PA): AACR; Cancer Immunol Res 2024;12(10 Suppl):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b89db5bb53ca20d5ac128fa18040b839f5041ca9" target='_blank'>
              Abstract A025: Novel workflow for characterizing T-cell functional heterogeneity in response to checkpoint inhibitors
              </a>
            </td>
          <td>
            Jacqueline Yee, Praveer Sharma, Yunmin Li, Isaac Thomas, P. F. Gherardini, Teresa Ai, Mostafa Ronaghi, Gary Schroth, Matthew H Spitzer
          </td>
          <td>2024-10-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa97a0ec1c0279a03e3ac7f307fe7b7f9d3560ca" target='_blank'>
              The SRC-family serves as a therapeutic target in triple negative breast cancer with acquired resistance to chemotherapy.
              </a>
            </td>
          <td>
            E. V. Egeland, Kotryna Seip, Eleni Skourti, G. F. Øy, S. Pettersen, Abhilash D Pandya, M. A. Dahle, M. H. Haugen, Alexander Kristian, S. Nakken, O. Engebraaten, G. Mælandsmo, L. Prasmickaite
          </td>
          <td>2024-10-10</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Background: Patients with hormone receptor-positive (HR+)/HER2- metastatic breast cancer (mBC) generally receive hormonal therapy (HT) combined with CDK4/6 inhibitors (CDK4/6i). Despite this treatment, resistance mechanisms to CDK4/6i emerge and the majority of these patients experience disease progression (PD). This highlight the necessity to uncover the resistance mechanism to CDK4/6i through the identification of specific biomarkers. The primary objective is to assess the accuracy and feasibility of a novel multi-gene target panel NGS assay on circulating tumor DNA (ctDNA) to detect molecular alterations of AKT1, ERBB2, ESR1, KRAS, PIK3CA, and TP53 genes in women with BC undergoing HT plus CDK4/6i treatment. Secondarily, the study aims to explore the relationship between genomic profiling and clinical outcomes. Materials and Methods: Plasma samples were collected from 16 patients diagnosed with advanced/locally advanced HR+/HER2- BC at 2 time points: T0 (baseline) and at T1 (3 months after CDK4/6i treatment). Starting from 2 mL of plasma, ctDNA was isolated and libraries were set up using the Plasma-SeqSensei (PQS)® Breast Cancer IVD Kit, sequenced on Nextseq 550 and analyzed using the Plasma-SeqSensei™ IVD Software®. Results: Among the five patients who presented PD, three had PIK3CA mutations and, of these, two showed a higher mutant allele frequency (MAF) at T1. In three patients with stable disease and in eight patients with partial response, the MAF of the detected alterations decreased dramatically or disappeared during CDK4/6i treatment. Conclusions: Based on our findings, the liquid biopsy analysis using the PQS panel seems to be both feasible and accurate, demonstrating a strong sensitivity in detecting mutations. This exploratory analysis of the clinical outcome associated to the mutational status of patients highlights the potential of molecular analysis on liquid biopsy for disease monitoring, although further validation with a larger patient cohort is necessary to confirm these preliminary observations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56cafee5ca185d21631ed12255567f16d464d5e4" target='_blank'>
              A Novel AKT1, ERBB2, ESR1, KRAS, PIK3CA, and TP53 NGS Assay: A Non-Invasive Tool to Monitor Resistance Mechanisms to Hormonal Therapy and CDK4/6 Inhibitors
              </a>
            </td>
          <td>

          </td>
          <td>2024-09-26</td>
          <td>Biomedicines</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9ece1f1388440529e828df574be9eda5a88aff" target='_blank'>
              Network modeling links kidney developmental programs and the cancer type-specificity of VHL mutations
              </a>
            </td>
          <td>
            Xiaobao Dong, Donglei Zhang, Xian Zhang, Yun Liu, Yuanyuan Liu
          </td>
          <td>2024-10-03</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND: Genome-wide association studies implicate common genetic variations in the LRP1 (low-density lipoprotein receptor-related protein 1 gene) locus at risk for multiple vascular diseases and traits. However, the underlying biological mechanisms are unknown. METHODS: Fine mapping analyses included Bayesian colocalization to identify the most likely causal variant. Human induced pluripotent stem cells were genome-edited using CRISPR-Cas9 (Clustered Regularly Interspaced Short Palindromic Repeats-CRISPR associated protein 9) to delete or modify candidate enhancer regions and generate LRP1 knockout cell lines. Cells were differentiated into smooth muscle cells through a mesodermal lineage. Transcription regulation was assessed using luciferase reporter assay, transcription factor knockdown, and chromatin immunoprecipitation. Phenotype changes in cells were conducted using cellular assays, bulk RNA sequencing, and mass spectrometry. RESULTS: Multitrait colocalization analyses pointed at rs11172113 as the most likely causal variant in LRP1 for fibromuscular dysplasia, migraine, pulse pressure, and spontaneous coronary artery dissection. We found the rs11172113-T allele to associate with higher LRP1 expression. Genomic deletion in induced pluripotent stem cell–derived smooth muscle cells supported rs11172113 to locate in an enhancer region regulating LRP1 expression. We found transcription factors MECP2 (methyl CpG binding protein 2) and SNAIL (Zinc Finger Protein SNAI1) to repress LRP1 expression through an allele-specific mechanism, involving SNAIL interaction with disease risk allele. LRP1 knockout decreased induced pluripotent stem cell–derived smooth muscle cell proliferation and migration. Differentially expressed genes were enriched for collagen-containing extracellular matrix and connective tissue development. LRP1 knockout and deletion of rs11172113 enhancer showed potentiated canonical TGF-β (transforming growth factor beta) signaling through enhanced phosphorylation of SMAD2/3 (Mothers against decapentaplegic homolog 2/3). Analyses of the protein content of decellularized extracts indicated partial extracellular matrix remodeling involving enhanced secretion of CYR61 (cystein rich angiogenic protein 61), a known LRP1 ligand involved in vascular integrity and TIMP3 (Metalloproteinase inhibitor 3), implicated in extracellular matrix maintenance and also known to interact with LRP1. CONCLUSIONS: Our findings support allele-specific LRP1 expression repression by the endothelial-to-mesenchymal transition regulator SNAIL. We propose decreased LRP1 expression in smooth muscle cells to remodel the extracellular matrix enhanced by TGF-β as a potential mechanism of this pleiotropic locus for vascular diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130d9d0beb72cded358387a84a2d2c98e035e14a" target='_blank'>
              LRP1 Repression by SNAIL Results in ECM Remodeling in Genetic Risk for Vascular Diseases
              </a>
            </td>
          <td>
            Lu Liu, Joséphine Henry, Yingwei Liu, C. Jouve, J. Hulot, A. Georges, N. Bouatia-Naji
          </td>
          <td>2024-10-02</td>
          <td>Circulation Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [23],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>