<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-12-25 12:38:15 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>13</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Phenotypic plasticity is a hallmark of cancer and increasingly realized as a mechanism of resistance to androgen receptor (AR)-targeted therapy. Now that many prostate cancer (PCa) patients are treated upfront with AR-targeted agents, it's critical to identify actionable mechanisms that drive phenotypic plasticity, to prevent the emergence of resistance. We showed that loss of tristetraprolin (TTP, gene ZFP36) increased NF-κB activation, and was associated with higher rates of aggressive disease and early recurrence in primary PCa. We also examined the clinical and biological impact of ZFP36 loss with co-loss of PTEN, a known driver of PCa. Analysis of multiple independent primary PCa cohorts demonstrated that PTEN and ZFP36 co-loss was associated with increased recurrence risk. Engineering prostate-specific Zfp36 deletion in vivo, induced prostatic intraepithelial neoplasia, and, with Pten co-deletion, resulted in rapid progression to castration-resistant adenocarcinoma. Zfp36 loss altered the cell state driven by Pten loss, demonstrated by enrichment of EMT, inflammation, TNFα/NF-κB, IL6-JAK/STAT3 gene sets. Additionally, our work revealed that ZFP36 loss also induced enrichment of multiple gene sets involved in mononuclear cell migration, chemotaxis, and proliferation. Use of the NF-κB inhibitor, dimethylaminoparthenolide (DMAPT) induced marked therapeutic responses in tumors with PTEN and ZFP36 co-loss and reversed castration resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a8debd5a52234a8e6e2eecf0d0e1cd5849b25a" target='_blank'>
              Low tristetraprolin expression activates phenotypic plasticity and primes transition to lethal prostate cancer in mice.
              </a>
            </td>
          <td>
            Katherine L. Morel, Beatriz German, Anis A Hamid, Jagpreet S Nanda, S. Linder, A. Bergman, Henk van der Poel, Ingrid Hofland, E. Bekers, S. Trostel, Deborah L. Burkhart, S. Wilkinson, Anson T. Ku, Minhyung Kim, Jina Kim, Duanduan Ma, Jasmine T. Plummer, Sungyong You, Xiaofeng A. Su, W. Zwart, A. Sowalsky, Christopher J. Sweeney, Leigh Ellis
          </td>
          <td>2024-11-19</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Aim: Resistance to hormonal and targeted therapies in breast cancer limits treatment efficacy. Epigenetic alterations, including changes mediated by DNA methyltransferases, play a key role in this process. Previously, we identified that resistance to tamoxifen and rapamycin is associated with the suppression of DNMT3A. This study aims to further explore the mechanisms underlying this suppression, with a focus on identifying NR6A1 as a novel regulatory factor. Methods: Acquisition of resistant breast cancer cell sublines, MTT-test, immunoblotting, transient transfection and reporter analysis, lentiviral infection, qRT-PCR, and analysis of methylation using bisulfite pyrosequencing. Results: Our findings indicate that the development of cross-resistance in breast cancer cells to hormonal and targeted therapies involves a shift in cell signaling to alternative AKT pathways, marked by a localized suppression of the NR6A1/DNMT3A axis and associated DNA methylation changes. We demonstrated the critical role of NR6A1 downregulation in resistance development. Additionally, we observed activation of Snail - a key regulator in the epithelial-mesenchymal transition - as a mediator of the effects of NR6A1 depletion, establishing a direct link between Snail expression and resistance formation. Conclusion: The coordinated suppression of NR6A1 and DNMT3A may contribute to sustaining the resistant phenotype in breast cancer cells. This pathway could serve as a predictive marker, helping guide the selection of optimal therapeutic strategies for breast cancer treatment in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d24584557816dae92ae33446c23b68005521f3e" target='_blank'>
              Breast cancer cell resistance to hormonal and targeted therapeutics is correlated with the inactivation of the NR6A1 axis
              </a>
            </td>
          <td>
            O. Andreeva, D. Sorokin, S. Vinokurova, P. Kopnin, N. V. Elkina, A. Katargin, R. S. Faskhutdinov, D. Salnikova, A. M. Scherbakov, M. Krasil'nikov
          </td>
          <td>2024-11-23</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a complex clonal disorder characterized by clinical, genetic, metabolomic, and epigenetic heterogeneity resulting in the uncontrolled proliferation of aberrant blood-forming precursor cells. Despite advancements in the understanding of the genetic, metabolic, and epigenetic landscape of AML, it remains a significant therapeutic challenge. Functional profiling techniques, such as BH3 profiling (BP), gene expression profiling (GEP), proteomics, metabolomics, drug sensitivity/resistance testing (DSRT), CRISPR/Cas9, and RNAi screens offer valuable insights into the functional behavior of leukemia cells. BP evaluates the mitochondrial response to pro-apoptotic BH3 peptides, determining a cell’s apoptotic threshold and its reliance on specific anti-apoptotic proteins. This knowledge can pinpoint vulnerabilities in the mitochondria-mediated apoptotic pathway in leukemia cells, potentially informing treatment strategies and predicting therapeutic responses. GEP, particularly RNA sequencing, evaluates the transcriptomic landscape and identifies gene expression alterations specific to AML subtypes. Proteomics and metabolomics, utilizing mass spectrometry and nuclear magnetic resonance (NMR), provide a detailed view of the active proteins and metabolic pathways in leukemia cells. DSRT involves exposing leukemia cells to a panel of chemotherapeutic and targeted agents to assess their sensitivity or resistance profiles and potentially guide personalized treatment strategies. CRISPR/Cas9 and RNAi screens enable systematic disruption of genes to ascertain their roles in leukemia cell survival and proliferation. These techniques facilitate precise disease subtyping, uncover novel biomarkers and therapeutic targets, and provide a deeper understanding of drug-resistance mechanisms. Recent studies utilizing functional profiling have identified specific mutations and gene signatures associated with aggressive AML subtypes, aberrant signaling pathways, and potential opportunities for drug repurposing. The integration of multi-omics approaches, advances in single-cell sequencing, and artificial intelligence is expected to refine the precision of functional profiling and ultimately improve patient outcomes in AML. This review highlights the diverse landscape of functional profiling methods and emphasizes their respective advantages and limitations. It highlights select successes in how these methods have further advanced our understanding of AML biology, identifies druggable targets that have improved outcomes, delineates challenges associated with these techniques, and provides a prospective view of the future where these techniques are likely to be increasingly incorporated into the routine care of patients with AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d60f51fb21da6cf836930cd187d2acd3d680d13" target='_blank'>
              Decoding Acute Myeloid Leukemia: A Clinician’s Guide to Functional Profiling
              </a>
            </td>
          <td>
            Prasad Iyer, S. Jasdanwala, Yuhan Wang, K. Bhatia, Shruti Bhatt
          </td>
          <td>2024-11-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BCR–ABL, the pioneering gene fusion resulting from chromosomal translocation, has marked a major milestone in understanding genetic alterations in cancer. Initially, gene fusions were linked solely to chromosomal rearrangements, serving as diagnostic markers and cancer drivers. However, advancements in high-throughput sequencing and bioinformatics have revealed additional mechanisms underlying gene fusion. The term “gene fusion” primarily refers to fusion events at the DNA level, whereas “chimeric RNA” encompasses a wide range of transcripts containing exons from different parental genes, including gene fusion transcripts. Recent developments have identified numerous chimeric RNAs in various cancer types, extending even to non-cancerous tissues. Chimeric RNAs, originating from events such as trans-splicing, read-through, and intergenic splicing, form a complex landscape with varied functions. While some chimeric RNAs have defined roles and therapeutic potential, a comprehensive understanding of their diverse functions remains a priority. Exploring the full spectrum of chimeric RNA activities is crucial for revealing their clinical and therapeutic implications. In addition, chimeric RNAs are key players in tumorigenesis, affecting cellular processes, and driving cancer progression. Understanding their intricate interactions with cellular pathways is essential for developing targeted therapies and precision medicine approaches. The dynamic nature of chimeric RNAs highlights the need for ongoing research to fully harness their diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0ece1a1fed50713b0199fed280d597f09ffa316" target='_blank'>
              Beyond chromosomal rearrangements: The expanding landscape of gene fusions and chimeric RNAs
              </a>
            </td>
          <td>
            Sangeen Khan, Yue Tang, Zhenguo Song, Xi Chen, Lijun Wang, Chengjuan Zhang, Fujun Qin
          </td>
          <td>2024-10-28</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Mantle cell lymphoma (MCL) is an incurable B-cell malignancy with current front-line treatment regimens. Dysregulated B-cell receptor signaling and resistance to apoptosis are major hallmarks in MCL pathobiology. Bruton's tyrosine kinase inhibitors (BTKi), such as ibrutinib, and BCL-2 inhibitors, such as venetoclax, have been used to treat refractory and relapsed (R/R) MCL. While these agents show clinical efficacy, patients frequently develop drug resistance with consequent poor clinical outcomes. We previously characterized the molecular mechanisms of drug response and resistance to BTKi and venetoclax single-agent therapies in MCL and identified the combination of BTKi and venetoclax as a novel approach for overcoming drug resistance and providing a more durable and synergistic response in MCL. Translation of our preclinical model was achieved through the establishment of a randomized multicenter phase 3 clinical trial with BTKi-venetoclax vs BTKi-placebo. Although this trial showed improved responses to BTKi-venetoclax therapy, the majority of patients ultimately developed resistance to this regimen. Thus, there is an urgent need to identify new therapeutic targets and to develop newer therapeutic agents and strategies to overcome and prevent the emergence of therapy resistance. With the goal of eradicating residual resistant tumor subclones and sustaining complete remissions, we here characterize the molecular genetic and epigenetic resistance mechanisms to BTKi-venetoclax combination therapy using multi-omics approaches including scRNA-seq and scATAC-seq in MCL. Our integrative multi-omics single-cell analysis reveals that epigenetic chromatin accessibility alterations underpin BTKi-venetoclax combination treatment resistance-associated transcriptome reprogramming and resistance evolution. We also validate the functional role of epigenetic regulation and transcription activation determinants (CDK7, CDK9) in BTKi-venetoclax resistance evolution. Furthermore, we establish that BTKi-venetoclax treatment reshapes the tumor microenvironment (TME) in MCL in favor of immune-responsiveness, which is evidenced by an enhanced susceptibility to immunotherapy. Taken together, this work demonstrates that the adaptive transcriptional machinery activation and immune-responsive TME state transition induced by BTKi-venetoclax are therapeutic vulnerabilities that may be effectively exploited by both targeted- (e.g., CDK9 inhibitors) and immune-therapies (e.g., CD19 CAR-T) to overcome MCL therapy resistance. This highly translational work has the potential to have an immediate impact on MCL treatment approaches through the rational design of new combination regimens.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cb5bd01eb150f523f9387e5b78b2274ee7100a9" target='_blank'>
              Dysregulation of Transcription Machinery Drives the Acquisition of BTK Inhibitor and Venetoclax Treatment Resistance in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Jing Gao, Kevin Qiu, Kallesh Danappa Jayappa, Qing Yin, Michelle Wang, Allen Hu, Danielle C Llaneza, Loryn L Blackburn, Jeffrey W Craig, Bijal D Shah, C. Portell, Michael E Williams, Chongzhi Zang, Jianguo Tao
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="

 Cancer entities and the presence or absence of genetic driver mutations often determine therapeutic trajectories for patients. However, the efficacy of chemotherapeutics and targeted therapies varies significantly and rarely induces long-term remissions due to the rapid emergence of therapy resistance. Consequently, there is an urgent need to identify response-predictive biomarkers that enable effective precision medicine.
 We hypothesized that, similar to genetic drivers that qualify patient subgroups for specific targeted therapeutics, genetic aberrations exist within cancer cells that confer intrinsic sensitivity to chemotherapeutics and small-molecule inhibitors lacking response-predictive biomarkers. To test this, we developed an advanced CRISPR/Cas9-based high-throughput screening platform to systematically interrogate gene-drug interactions of 18 clinically established and emerging therapeutics used for treating hematopoietic malignancies. These included antimetabolites, anthracyclines, hypomethylating agents, and various selective small-molecule inhibitors. Utilizing optimized sgRNA prediction algorithms and a dual-sgRNA design, we constructed low-complexity genome-wide sgRNA libraries that enhance the technical robustness and throughput of drug-modifier screens in culture and enable in vivo screens.
 We uncovered 180 genes including various established and previously unknown factors that upon knockout result in selective synergy or resistance to one of the 18 therapeutics. We functionally validated various gene-drug interactions in multiple leukemia and solid cancer cell lines, suggesting that the loss of these genes confers predictable drug response to specific compounds independent of the tissue context. As the mechanistic relationship of many gene-drug interactions was uncharted, we assessed their impact on drug metabolism using targeted liquid chromatography-tandem mass spectrometry (LC-MS/MS). While several genes modified cellular drug influx/efflux or metabolism, many others altered drug sensitivity by distinct means that remain to be explored. However, integrating publicly available whole-exome sequencing data pinpointed patients across diverse cancer entities harboring homozygous deletions within selected genes that modify drug response, suggesting these patients could display predictable drug sensitivity. To test this in vivo, we probed three drug modifier genes (SAMHD1, XRCC4, and USP48) that are recurrently deleted in cancer patients using colorectal cancer cell line-derived xenotransplants and examined their response to Cytarabine, Doxorubicin, and Decitabine treatment, respectively. Strikingly, while wildtype colorectal cancer transplants were unresponsive, tumors with deletions in these genes were amenable to treatment, resulting in significant tumor control.
 Together, our data show that our scalable screening pipeline can systematically interrogate gene-drug interactions for a broad panel of drugs lacking response-predictive biomarkers. They suggest that integrating drug modifier genes in targeted exome sequencing panels of cancer patients may pave the way for discovering unexplored treatment opportunities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596fd52391935223a262c418bf05217719567e11" target='_blank'>
              Systematic Identification of Gene-Drug Interactions Using an Advanced CRISPR Screening Platform to Predict Therapy Response across Cancer Types
              </a>
            </td>
          <td>
            Johannes Schmoellerl, Romana Maerschalk, Martina Weissenboeck, Jasmin Ali, Thomas Köcher, Aleksandra Bundalo, F. Andersch, Vivien Vogt, Bettina Sajtos, Jonas Bayerl, Martina Minnich, Michaela Fellner, Johannes Zuber
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Background: Osimertinib is currently the most common precision therapy for patients with non- small cell lung cancer (NSCLC) with EGFR mutations. While it prolongs progression-free survival, resistance eventually develops in many, leading to disease progression. Somatic NGS has identified resistance mutations in the EGFR pathway and other oncogenes, but these account for only 40-45% of cases, suggesting non-genetic factors, such as epigenetic changes, may be involved. This study investigates the epigenetic landscape at diagnosis and progression on first- line osimertinib, focusing on cases with and without canonical resistance mutations. Methods: We defined two cohorts from GuardantINFORM, a clinico-genomics database, restricting to only patients with a test on the Guardant Infinity platform, a comprehensive blood-based NGS assay, that analyzes DNA sequence variants and performs extensive methylation profiling (∼15Mb) in >20,000 differentially methylated regions (DMRs) assessed for sample-specific methylation patterns. (A) Diagnosis Cohort: Comprised of 34 NSCLC patients with blood samples collected within six days prior to the initiation of first-line osimertinib therapy. (B) Progression Cohort: Comprised of 36 patients who were on first-line osimertinib therapy, with blood samples collected within 60 days of discontinuation. The progression cohort was further stratified based on the presence or absence of known resistance mutations identified through somatic variant analysis by Guardant360. Gene set enrichment analysis was conducted on these methylation profiles using the Reactome database to define pathway-level differences between the cohorts. Results: Pathway enrichment analysis in the diagnosis and progression cohorts found expected biological processes in NSCLC such as extracellular matrix organization, G- protein coupled receptor (GPCR) signaling, and FGFR pathways. Additionally, the progression cohort lacking canonical resistance mutations (n=23) exhibited unique enrichment in two specific pathways: Glucose-dependent Insulinotropic Polypeptide and Physiological Factors. These pathways include genes such as GATA4, which has been implicated in the epithelial- mesenchymal transition (EMT)—a process associated with transforming to a small cell phenotype, a known mechanism of resistance to EGFR inhibition, including osimertinib. Conclusion: This study demonstrates the utility of epigenetic profiling via liquid biopsies in uncovering non-genetic mechanisms underlying osimertinib resistance in advanced NSCLC. Identification of unique epigenetic pathways in progressing patients without canonical resistance mutations highlights the potential for discovering novel biomarkers of resistance. Further investigation into these findings may advance our understanding of resistance mechanisms across various cancer therapies, particularly for patients without identified genomic resistance, and inform clinical decision-making and trial design.
 Citation Format: Aaron Hardin, Leslie Bucheit, Amar Das, Craig Eagel, Helmy Eltoukhy. Pathway repression in progression on osimertinib therapy in NSCLC via a comprehensive genomic and epigenomic circulating tumor DNA (ctDNA) assay [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B036.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9810af5aadd760f6c2c32f19cb5c026e7ca93" target='_blank'>
              Abstract B036: Pathway repression in progression on osimertinib therapy in NSCLC via a comprehensive genomic and epigenomic circulating tumor DNA (ctDNA) assay
              </a>
            </td>
          <td>
            Aaron Hardin, L. Bucheit, Amar Das, Craig Eagel, H. Eltoukhy
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Evolutionary repair refers to the compensatory evolution that follows perturbations in cellular processes. While evolutionary trajectories are often reproducible, other studies suggest they are shaped by genotype-by-environment (GxE) interactions. Here, we test the predictability of evolutionary repair in response to DNA replication stress—a severe perturbation impairing the conserved mechanisms of DNA synthesis, resulting in genetic instability. We conducted high-throughput experimental evolution on Saccharomyces cerevisiae experiencing constitutive replication stress, grown under different glucose availabilities. We found that glucose levels impact the physiology and adaptation rate of replication stress mutants. However, the genetics of adaptation show remarkable robustness across environments. Recurrent mutations collectively recapitulated the fitness of evolved lines and are advantageous across macronutrient availability. We also identified a novel role of the mediator complex of RNA polymerase II in adaptation to replicative stress. Our results highlight the robustness and predictability of evolutionary repair mechanisms to DNA replication stress and provide new insights into the evolutionary aspects of genome stability, with potential implications for understanding cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d644d70495ca5cd494d5397eda100129033764cf" target='_blank'>
              Compensatory Evolution to DNA Replication Stress is Robust to Nutrient Availability
              </a>
            </td>
          <td>
            Mariana Natalino, M. Fumasoni
          </td>
          <td>2024-11-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer cells exploit a mesenchymal-like transcriptional state (MLS) to survive drug treatments. Although the MLS is well characterized, few therapeutic vulnerabilities targeting this program have been identified. Here, we systematically identify the dependency network of mesenchymal-like cancers through an analysis of gene essentiality scores in ~800 cancer cell lines, nominating a poorly studied kinase, PKN2, as a top therapeutic target of the MLS. Co-essentiality relationships, biochemical experiments, and genomic analyses of patient tumors revealed that PKN2 promotes mesenchymal-like cancer growth through a PKN2-SAV1-TAZ signaling mechanism. Notably, pairing genetic PKN2 inhibition with clinically relevant targeted therapies against EGFR, KRAS, and BRAF oncogenes suppresses drug resistance by depleting mesenchymal-like drug-tolerant persister cells. These findings provide evidence that PKN2 is a core regulator of the Hippo tumor suppressor pathway and highlight the potential of PKN2 inhibition as a generalizable therapeutic strategy to overcome drug resistance driven by the MLS across cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc877056e57a8e2b137d870c6f49bfae37eaa22e" target='_blank'>
              PKN2 is a dependency of the mesenchymal-like cancer cell state.
              </a>
            </td>
          <td>
            Shane T. Killarney, Gabriel Mesa, Rachel Washart, Benjamin Mayro, Kerry Dillon, S. Wardell, M. Newlin, Min Lu, Areej Abu Rmaileh, Nicky Liu, Donald P McDonnell, A. Pendergast, Kris C Wood
          </td>
          <td>2024-11-19</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="

 Recent advancements in cell and gene therapy are transforming the treatment landscape for diseases like cancer and rare genetic disorders. Precision gene therapies targeting conditions such as sickle cell disease, beta-thalassemia, and B-cell malignancies have shown remarkable efficacy in correcting genetic defects at their source, paving the way for long-term cures.
 Despite these breakthroughs, the gene editing process can result in heterogeneous cell populations, some of which may carry undesirable outcomes that pose risks of genomic toxicity and potential malignancies. These adverse effects may include off-target edits, copy number variations, and chromosomal aberrations such as translocations or large indels. Thus, the success of gene therapies hinges on the ability to accurately measure and understand these events. Given that “cells” are the fundamental units of gene editing products, it is crucial to assess the co-occurrence of editing outcomes and potential genotoxic events at the single-cell level.
 In this study, we introduce an advanced single-cell technology that integrates microfluidics and multiplex PCR to concurrently measure several critical aspects of gene editing. This assay evaluates the co-occurrence and zygosity of on-target and off-target edits, quantitatively detects translocations between predicted edit sites, and maps the genomic copy number variation (CNV) landscape, analyzing over 10,000 cells in parallel. With single-cell resolution, this method provides a detailed and comprehensive view of the heterogeneous editing profiles in gene-edited products. It enables precise and rapid assessment of editing outcomes and overall genomic integrity, facilitating the identification of potential malignancies and enhancing the safety and efficacy of gene therapies.
 To validate the robustness of this workflow, we employed various orthogonally validated samples with different levels of adverse effects from multiple gene editing experiments. Using a targeted sequencing panel, we measured on- and off-target editing activity, detected structural variations, and assessed genome-wide CNVs at the single-cell level in a single assay. The resulting data were analyzed with a novel bioinformatics pipeline specifically designed to accurately measure all these modalities simultaneously. Additionally, for locus-level CNV detection, we demonstrated the performance of the single-cell CNV assay at a focal level. This was exemplified by confirming a Chr20q copy number alteration in PGP-1 iPSC cell lines, which conventional methods such as G-banding and CMA failed to detect.
 This high-throughput single-cell NGS assay marks a significant advancement in gene editing analysis by effectively capturing both intended and unintended consequences of genome editing at various resolutions. Its dual functionality not only enhances the optimization of gene editing protocols but also provides a comprehensive and efficient approach for evaluating the safety aspects of gene-modified therapies. By enabling detailed assessments of genomic integrity and potential malignancies, this method significantly contributes to key CQA that could guide the development of safer and more effective gene therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22439f30220ba9f3184a7820a44b6cb0483273f5" target='_blank'>
              Measuring the Mosaicism of Genome Integrity and Toxicity Events Associated with Gene and Cell Therapies at a Single-Cell Level
              </a>
            </td>
          <td>
            Shu Wang, Saurabh Parikh, Dongjo Ban, Saurabh Gulati, Qawer Ayaz, Joanne Nguyen, Benjamin Miltz, Yang Li, Ben Schroeder
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ex vivo drug response profiling is emerging as a valuable tool for identifying drug resistance mechanisms and advancing precision medicine in hematological cancers. However, the functional impact of dysregulation of the epigenome and transcriptome in this context remains poorly understood. In this study, we combined ex vivo drug sensitivity profiling with transcriptomic and epigenomic analyses in diagnostic samples from 597 pediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) patients. Ex vivo resistance to antimetabolites (e.g., cytarabine, thioguanine), glucocorticoids (e.g., dexamethasone, prednisolone), and doxorubicin was independently associated with reduced relapse-free survival (RFS, p < 0.05). Molecular profiling identified pre-treatment DNA methylation and gene expression patterns distinguishing resistant from sensitive cases, revealing key drug resistance signatures. These included aberrant expression of genes related to heme metabolism (e.g. ATPV06A) and KRAS signaling (e.g. ABCB1). Notably, we also observed atypical expression of genes usually restricted to T-cells and other immune cells (e.g., ITK) in resistant BCP-ALL cells. Our findings demonstrate that ex vivo drug response patterns are predictive of clinical outcomes and reflect intrinsic molecular states associated with drug tolerance. This integrative multi-omics approach highlights potential therapeutic targets and underscores the value of functional precision medicine in identifying treatment vulnerabilities in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b0bb7a98b1f284795f68f2172436eb0a2df655" target='_blank'>
              Ex Vivo Drug Responses and Molecular Profiles of 597 Pediatric Acute Lymphoblastic Leukemia Patients
              </a>
            </td>
          <td>
            Anna Pia Enblad, O. Krali, Henrik Gezelius, Anders Lundmark, Kristin Blom, Claes Andersson, J. Palle, B. Frost, S. Ryhänen, Trond Flægstad, Ó. Jónsson, Kjeld Schmiegelow, Arja Harila, Peter Nygren, Rolf Larsson, Gudmar Lönnerholm, J. Nordlund, Anna Pia
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8682cef5538b95d1f2e68873fc143c3785f2658e" target='_blank'>
              Reciprocal inhibition of NOTCH and SOX2 shapes tumor cell plasticity and therapeutic escape in triple-negative breast cancer
              </a>
            </td>
          <td>
            Morgane Fournier, Joaquim Javary, Vincent Roh, Nadine Fournier, Freddy Radtke
          </td>
          <td>2024-10-30</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bcc9e0c8d78b7e4a0d8c81e52724a7077db80c6" target='_blank'>
              Saturation profiling of drug-resistant genetic variants using prime editing.
              </a>
            </td>
          <td>
            Younggwang Kim, Hyeong-Cheol Oh, Seungho Lee, H. Kim
          </td>
          <td>2024-11-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The study investigates the impact of targeted chromosome engineering on telomere dynamics, chromatin structure, gene expression, and phenotypic stability in Arabidopsis thaliana. Using precise CRISPR/Cas-based engineering, reciprocal translocations of chromosome arms were introduced between non-homologous chromosomes. The subsequent homozygous generations of plants were assessed for phenotype, transcriptomic changes and chromatin modifications near translocation breakpoints, and telomere length maintenance. Phenotypically, translocated lines were indistinguishable from wild-type plants, as confirmed through morphological assessments and principal component analysis. Gene expression profiling detected minimal differential expression, with affected genes dispersed across the genome, indicating negligible transcriptional impact. Similarly, ChIPseq analysis showed no substantial alterations in the enrichment of key histone marks (H3K27me3, H3K4me1, H3K56ac) near junction sites or across the genome. Finally, bulk and arm-specific telomere lengths remained stable across multiple generations, except for minor variations in one translocation line. These findings highlight the remarkable genomic and phenotypic robustness of A. thaliana despite large-scale chromosomal rearrangements. The study offers insights into the cis-acting mechanisms underlying chromosome arm-specific telomere length setting and establishes the feasibility of chromosome engineering for studies of plant genome evolution and crop improvement strategies. Significance statement This study demonstrates the robustness of Arabidopsis thaliana in maintaining telomere stability, chromatin integrity, and wild-type phenotype despite large-scale chromosomal translocations. It underscores the potential of chromosome engineering in advancing genome evolution research and crop improvement, contributes to a deeper understanding of genome dynamics, and opens new avenues for precision breeding in plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c4a5f7dc0b1cbe9e13ab94b7dc2055482d7e96c" target='_blank'>
              Chromosome engineering points to the cis-acting mechanism of chromosome arm-specific telomere length setting and robustness of plant phenotype, chromatin structure and gene expression
              </a>
            </td>
          <td>
            Ondřej Helia, Barbora Matúšová, Kateřina Havlová, Anna Hýsková, Martin Lyčka, Natalja Beying, Holger Puchta, Jíří Fajkus, M. Fojtová
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 The cell cycle is tightly regulated by checkpoints, which play a vital role in controlling its progression and timing. Cancer cells exploit the G2/M checkpoint, which serves as a resistance mechanism against genotoxic anticancer treatments, allowing for DNA repair prior to cell division. Manipulating cell cycle timing has emerged as a potential strategy to augment the effectiveness of DNA damage-based therapies.



 In this study, we conducted a forward genome-wide CRISPR/Cas9 screening with repeated exposure to the alkylating agent temozolomide (TMZ) to investigate the mechanisms underlying tumor cell survival under genotoxic stress. The screening hits were validated through in patient derived glioma cells and xenografts.



 Our findings revealed that canonical DNA repair pathways, including the Ataxia–telangiectasia mutated (ATM)/Fanconi and mismatch repair, determine cell fate under genotoxic stress. Notably, we identified the critical role of PKMYT1, in ensuring cell survival. Depletion of PKMYT1 led to overwhelming TMZ-induced cytotoxicity in cancer cells. Isobologram analysis demonstrated potent drug synergy between alkylating agents and a Myt1 kinase inhibitor, RP-6306. Mechanistically, inhibiting Myt1 forced G2/M-arrested cells into an unscheduled transition to the mitotic phase without complete resolution of DNA damage. This forced entry into mitosis, along with persistent DNA damage, resulted in severe mitotic abnormalities. Ultimately, these aberrations led to mitotic exit with substantial apoptosis. Preclinical animal studies demonstrated that the combination regimen involving TMZ and RP-6306 prolonged the overall survival of glioma-bearing mice.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7ee903678ba371418738559dde776df52cc802e" target='_blank'>
              CCRG-01. MYT1 BLOCKADE AS A NOVEL GLIOMA THERAPEUTICS THROUGH ABROGATIING OF THE G2/M CELL CYCLE CHECKPOINT
              </a>
            </td>
          <td>
            Fengchao Lang, James A Cornwell, Wei Zhang, Meili Zhang, Hua Song, Mirit Aladjem, Michael Aregger, S. Cappell, Chunzhang Yang
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53d714dd5d6b619d9b1b3610a6e67560b7be4c02" target='_blank'>
              RAS signaling in carcinogenesis, cancer therapy and resistance mechanisms
              </a>
            </td>
          <td>
            Xiaojuan Yang, Hong Wu
          </td>
          <td>2024-11-09</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f996582dd66fed5afd8cec13e0097379571b0d4e" target='_blank'>
              Correlative studies reveal factors contributing to successful CAR-T cell therapies in cancer.
              </a>
            </td>
          <td>
            Catherine D Yao, K. Davis
          </td>
          <td>2024-12-03</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK; also known as ALK tyrosine kinase receptor) inhibitors (ALKi) are effective in treating lung cancer patients with chromosomal rearrangement of ALK. However, continuous treatment with ALKis invariably leads to acquired resistance in cancer cells. In this study, we propose an efficient strategy to suppress ALKi resistance through a meta-analysis of transcriptome data from various cell models of acquired resistance to ALKis. We systematically identified gene signatures that consistently showed altered expression during the development of resistance and conducted computational drug screening using these signatures. We identified emetine as a promising candidate compound to inhibit the growth of ALKi-resistant cells. We demonstrated that emetine exhibited effectiveness in inhibiting the growth of ALKi-resistant cells, and further interpreted its impact on the resistant signatures through drug-induced RNA-sequencing data. Our transcriptome-guided systematic approach paves the way for efficient drug discovery to overcome acquired resistance to cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb27f5b375afaa6d1b172695af6dc21d8ad944" target='_blank'>
              Integrative transcriptomic analysis identifies emetine as a promising candidate for overcoming acquired resistance to ALK inhibitors in lung cancer.
              </a>
            </td>
          <td>
            Sang-Min Park, Keeok Haam, Haejeong Heo, Doyeong Kim, Min-Ju Kim, Hyo-Jung Jung, Seongwon Cha, Mirang Kim, Haeseung Lee
          </td>
          <td>2024-11-14</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Introduction
 Genome-wide transposon mutagenesis screens in mice are powerful tools for identifying genomically, transcriptionally, and epigenetically deregulated cancer genes in a single experimental approach. A major limitation, however, is the need for large cohort sizes and substantial sequencing and bioinformatic resources to differentiate causal genetic changes from silent transposon insertions. We address these challenges with single cell PiggyBac Transposon and Transcriptome Sequencing (scPB&T-seq), a novel dual-assay that enables the identification of disrupted molecular networks at the single cell level, thus improving precision and efficacy of cancer gene studies.
 Methods
 A murine PiggyBac (PB) transposition screen was performed to generate Asxl1-mutated (Asxl1mut) acute myeloid leukemias (AMLs), a subgroup with particularly poor prognosis. For scPB&T-seq, single cells (sc) from three AMLs were isolated, and genomic DNA was physically separated from mRNA. Sc transposon insertions were amplified via multiple displacement followed by a custom PCR amplification and sequenced alongside sc transcriptomes derived by Smart-Seq2. The data was then integrated with high-throughput droplet-based scRNAseq from the same samples.
 Results
 Activating transposon mutagenesis in an Asxl1mut background led to the identification of a large candidate gene catalogue for Asxl1mut AML pathogenesis. Using scPB&T-seq we identified an average of 10-25 transposons per cell, aligning with expected frequencies. All insertions were confirmed in concurrent bulk insertion analyses. About 60% of the insertions impacted gene expression and were retained for further analysis. By integrating scRNAseq data, the insertions were assigned to specific cellular clusters, allowing the reconstruction of clonal evolution and identification of intratumoral heterogeneity. This approach confirmed known AML drivers co-occurring with ASXL1 mutations, such as PRDM16 and MECOM, and revealed novel genomic loci linked to AML pathogenesis. These findings were validated in whole genome and whole transcriptome sequencing data from 774 human AML patients generated within the Munich Leukemia Laboratory 5K Genomes Project. Interestingly, 45 (5.9%) patients from this dataset exhibited a transcriptional profile similar to the main clone of an Asxl1mut PB-AML, showcasing an upregulation of e.g., DOCK5, TCF4, and CSF1. Furthermore, this subgroup of patients displayed a significant enrichment for ASXL1 (31.1% vs. 14.5%, P = 0.005), NRAS (28.9% vs. 15.6%, P = 0.035) and concurrent ASXL1 and NRAS (13.3% vs. 3%, P = 0.004) mutations compared to the remaining 724 AML patients. Of note, an insertion in Nras was an activating insertion in a monocytic-like subclone of this Asxl1mut PB-AML.
 Conclusion
 In this study we developed scPB&T-seq, an innovative method to analyze transposon screens that uncovers functional mechanisms behind individual transposon insertions in single cells of heterogenous tumors. This enables the assessment of clonal evolution and oncogenic networks in unprecedented detail, thereby enhancing our understanding of AML pathogenesis and potentially guiding the development of targeted therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9469d3f7ae87d98530a8f7fd8406ec0dd55fb8ef" target='_blank'>
              Parallel Sequencing of Transposons and Transcriptomes in Single Cells for Genome-Wide Cancer Gene Discovery
              </a>
            </td>
          <td>
            Maximilian Holz, B. Klinger, Felix Stuckenbrok, Stefanos A. Bamopoulos, Sanket Gosavi, Caroline Bräuning, Janine Altmüller, Roland Rad, Torsten Haferlach, Nils Blüthgen, Ulrich Keller, Thomas Conrad, Francis Baumgartner
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Introduction: Since DNA strand breaks are required for genomic rearrangements, genes involved in DNA breaks can potentially drive genomic instability and clonal evolution. Therefore, we developed a novel genome-wide functional phenotyping screen to identify genes whose modulation impacts DNA breaks in multiple myeloma (MM).
 Methods: Myeloma cell lines (JIM3 and U266) were transduced (in triplicate) with a whole genome lentiviral expression library comprised of 17,255 open reading frames (ORFs) representing 12,728 genes. Cells selected in puromycin were cultured and stained using a fluorescently labeled antibody against H2AX, a marker of DNA breaks. Fluorescence-activated cell sorting was used to isolate cells with increased H2AX expression representing presence of damaged DNA; and those cells were sequenced to identify the ORFs associated with increased DNA breaks. Selected genes were further investigated in gain as well as loss of function studies.
 Results: Genome wide ORF expression screen identified, in JIM3 and U266 MM cell lines, 789 and 302 genes associated with increased DNA breaks, respectively. We further analyzed 226 genes associated with increased DNA breaks that are common in both MM cell lines by functional enrichment analysis and report 6 distinct groups. These included 1) cell cycle cyclin-dependent kinases and phosphatases; 2) serine/threonine-protein kinases including calcium/calmodulin-dependent protein kinases; 3) DNA repair and recombination proteins; 4) cytoskeleton, chromosome maintenance and mitosis/cytokinesis related proteins; 5) monooxygenases involved in metabolic detoxification of xenobiotics; and 6) RNA helicases, and proteins involved in RNA processing, splicing and degradation. We next utilized 2 large datasets to evaluate clinical impact of high expression of these genes on patient outcome in MM. Elevated expression of 20 and 23 of these genes in MM patients conferred poor overall (OS) and event free (EFS) survival, respectively (GSE24080; n=559). In second MM dataset elevated expression of 14 and 8 of these genes correlated with poor OS and EFS, respectively (IFM70; n=170). Overall, the expression of three genes correlated with both the PFS and OS in both datasets. These included; 1) DTYMK; deoxythymidylate kinase; a kinase, regulating nucleotide biosynthesis and contributes to DNA replication and repair; 2) CDC25A; cell division cycle 25A, a phosphatase involved in G1/S progression and whose degradation in response to DNA damage is essential to prevent cells with chromosomal abnormalities from progressing through cell division; and 3) EXO1; Exonuclease 1, a nuclease with roles in DNA repair and replication.
 We further functionally validated the role of these select genes for their impact on genomic instability. Overexpression of all three genes (DTYMK, CDC25A and EXO1) in non-cancerous bone marrow stromal (HS5) and/or cancer cells led to increase in genomic instability, as assessed by micronucleus assay, by ∼7-fold, 3.6-fold and > 2-fold respectively. For EXO1 and two other genes also identified in the screen (HDAC8 and TPX2) we have confirmed their role in causing spontaneous and chemotherapy-induced DNA breaks, acquisition of newer genomic changes overtime as well as proliferation of MM and/or other cancer cells. Validation of all identified hits in DNA damage and assays to measure genomic instability is currently in progress.
 Conclusion: We report a novel genome wide high throughput screen that can uniquely utilize protein staining properties and FACS to assess impact of individual gene manipulation at functional and phenotypic level. Using this screen, we have identified genes and pathways contributing to increased DNA breaks in MM and their function is further validated in gain and loss-of function studies in spontaneous and chemotherapy-induced DNA breaks, and genomic instability in myeloma cells. These genes have potential to serve as promising new targets to make MM cells genomically static and the functional screen provides a tool to investigate impact of genome wide perturbations on functional and phenotypic changes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2d1194cb644b42ff02bfccb0592ee04f93e978" target='_blank'>
              Deciphering Drivers of Functional and Phenotypic Changes through Genome-Wide Screen (DRIVE-Genome) to Identify Genes Inducing Genomic Instability in Myeloma
              </a>
            </td>
          <td>
            S. Talluri, Leutz Buon, Jiangning Zhao, Daniel Meglino, C. Chakraborty, Chengcheng Liao, M. Samur, David E. Root, M. Shammas, Nikhil C Munshi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="

 Introduction
 Chromosomal rearrangements (CR) initiate leukemogenesis in approximately 50% of acute myeloid leukemia (AML) patients. Chromosomal abnormalities can give rise to fusion proteins, significantly contributing to the disease's ontogenesis and its recurrence or refractory nature. However, limited targeted therapies exist due to a lack of accurate molecular and genetic biomarkers of refractory mechanisms during treatment.
 Methods
 We investigated the pathological landscape of treatment resistance and relapse in 16 CR-AML patients by monitoring cytogenetic, RNA sequencing, and genome-wide changes among newly diagnosed, refractory, and relapsed AML cases. Bone marrow samples from seven patients were collected for chromosome analysis using GTG-banding, and a total of twenty metaphase cells were analyzed for each case using the ASI imaging system. The KMT2A gene rearrangements were tested on the bone marrow samples with FISH probes, analyzing a total of 200-500 interphase nuclei. Bone marrow mononuclear cells were isolated from blood samples of 2 patients, and high-quality total RNA was isolated. RNA sequencing was performed and after filtering, multiple candidate gene fusions were acquired for further analysis.
 Additionally, RNA was isolated from patient samples for quantitative real-time PCR analysis, to detect DNA breakpoints of t(6;11) translocations. Sequence alignments were analyzed with BLAST and BLAT. Statistical analyses were performed using GraphPad Prism.
 Results
 FISH analysis revealed KMT2A (MLL gene, 11q23)/AFDN (AF6, 6q27) rearrangements in AML patients. RNA sequencing identified an unknown gene fusion, CCDC32 (15q15.1)/CBX3 (7p15.2), in both newly diagnosed and relapsed samples, a finding not previously reported in KMT2A/AFDN-rearranged AML. Quantitative qPCR analyses demonstrated that this gene fusion significantly affected the expression of the wild-type CCDC32, essential for embryonic development, and CBX3, an oncogene for solid tumors, particularly during relapse.
 Further investigation confirmed the existence of triple biomarkers-KMT2A/AFDN rearrangement, CCDC32/CBX3 gene fusion, and chimeric RNA variants-in a 21-year-old male patient with rapid relapsed/refractory AML. Notably, among the 16 patients studied, two patients initially exhibited the KMT2A/AFDN gene fusion. However, one patient did not show this fusion upon relapse, whereas the other retained the KMT2A/AFDN fusion during both diagnosis and relapse, identifiable only by PCR and Sanger sequencing. Interestingly, the CCDC32/CBX3 gene fusion persisted in both patients throughout the diagnostic and relapse phases, suggesting a potential role in differential treatment resistance.
 Conclusion
 Our findings highlight the dynamic nature of genetic alterations in AML, particularly the persistence and evolution of prognostic and predictive biomarkers. The study identified triple biomarkers-KRAS mutation, dual fusions (KMT2A/AFDN and CCDC32/CBX3), and chimeric RNA variants-that may contribute to subclonal evolution and poor clinical outcomes. These results underscore the importance of continuous genetic monitoring to understand the mechanisms underlying treatment resistance and relapse in AML patients, potentially guiding more effective therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d3080989904ba1a7e05b4ec6ed5d03dfcb6dcc" target='_blank'>
              Exploring Treatment-Driven Subclonal Evolution of Prognostic and Predictive Triple Biomarkers: Dual Gene Fusions and Chimeric RNA Variants in Subtypes of Acute Myeloid Leukemia Patients with KMT2A Rearrangement
              </a>
            </td>
          <td>
            Homa Dadrastoussi, Yi Xu, S. Li, Jeffrey Xiao, Qian Liu, Durga Cherukuri, Yan Liu, S. Mirshahidi, Jane Xu, Xuelian Chen, J. Olea, Kaijin Wu, Kevin R Kelly, Xi Zhang, Fengzhu Sun, C. Ghiuzeli, Esther G Chong, Hisham Abdel-Azim, Mark E. Reeves, David Baylink, H. Cao, Jiang Zhong
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Aim: Mutations in the mitochondrial (mt) genome contribute to metabolic dysfunction and their accumulation relates to disease progression and resistance development in cancer cells. This study explores the mutational status of the mt genome of cisplatin-resistant vs. -sensitive testicular germ cell tumor (TGCT) cells and explores its association with their respiration parameters, expression of respiratory genes, and preferences for metabolic pathways to reveal new markers of therapy resistance in TGCTs.
 Methods: Using Illumina sequencing with Twist Enrichment Panel, the mutations of mt genomes of sensitive 2102EP, H12.1, NTERA-2, T-cam and resistant 2102EP Cis, H12.1 ODM, 1411HP, 1777NRpmet, NTERA-2 Cis and T-cam Cis cell lines were identified. The mt respiration of the cells was assessed using high-resolution respirometry method (O2k-respirometer Oroboros) and the differential expression profiles of mt respiratory genes were determined using RT-qPCR. Associated preferences for metabolic pathways were compared using Glycolysis/OXPHOS assay.
 Results: In resistant TGCT cells, new mutations in mt genes MT-ND1-6, MT-RNR, MT-CO1-3, MT-ATP6 , and MT-CYB were recognized. The respiratory rates of the 1777NRpmet cell line were the highest, while those of the 1411HP line the lowest; rates of the control and all other TGCT cell lines fell between these two lines. The statistically significant differences in gene expression of the respiratory genes were recorded only in NTERA-2 Cis and T-cam Cis cell lines. Sensitive cell lines NTERA-2 and 2102EP preferred oxidative phosphorylation (OXPHOS), while glycolysis was typical for resistant NTERA-2 Cis, 2102EP Cis and 1411HP cell lines. Metastatic 1777NRpmet cells seem to utilize both. An isogenic pair of cell lines H12.1 and H12.1ODM showed the opposite dependence, sensitive H12.1 preferring glycolysis, while resistant H12.1ODM OXPHOS.
 Conclusion: In summary, our study identified new mutations in mt genes of resistant TGCT cell lines that are associated with different mt respiration parameters, gene expression patterns and preferences for metabolic pathways, providing potential novel molecular biomarkers that distinguish the resistant TGCT phenotype or specify its histological classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec03801d589beba72991767df646f63f72e7a8" target='_blank'>
              Mitochondrial genome variability and metabolic alterations reveal new biomarkers of resistance in testicular germ cell tumors
              </a>
            </td>
          <td>
            Pavlina Kabelikova, Danica Ivovic, Zuzana Sumbalova, M. Karhánek, Lucia Tatayova, Martina Skopkova, Michal Cagalinec, V. Bruderová, J. Roska, Dana Jurkovicova
          </td>
          <td>2024-12-18</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tandem repeat disorders (TRDs) are genetic conditions characterized by the abnormal expansion of repetitive DNA sequences within specific genes. The growing number of identified TRDs highlights their complexity, with varied molecular mechanisms ranging from toxic protein production and repeat-associated non-AUG translation to RNA toxicity and epigenetic modifications. TRDs also exhibit unique clinical features such as reduced penetrance, anticipation, and repeat motif changes. Advances in molecular diagnostics such as long-read sequencing have significantly improved the detection of TRDs, especially for large or complex repeat expansions. Additionally, emerging therapeutic strategies, particularly antisense oligonucleotides (ASOs) and gene editing technologies, are showing great promise. ASOs in particular have demonstrated success through mechanisms like allele-specific knockdown and splice modulation. In this review, we explore the classification of TRDs, advances in diagnostics, molecular mechanisms, clinical features, and innovative therapeutic strategies, highlighting the need for further research to refine treatments and improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce97c16b3dbb715874ac1ab07bee3aa1f2cd3a4a" target='_blank'>
              Tandem repeat disorders: from diagnosis to emerging therapeutic strategies.
              </a>
            </td>
          <td>
            Jangsup Moon
          </td>
          <td>2024-12-10</td>
          <td>Encephalitis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gemcitabine-based monotherapy or combination therapy has become the standard treatment for locally advanced and metastatic pancreatic cancer. However, the emergence of resistance within weeks of treatment severely compromises therapeutic efficacy. The intricate biological process of gemcitabine resistance in pancreatic cancer presents a complex challenge, as the underlying mechanisms remain unclear. Identifying the target protein of gemcitabine is crucial for studying its drug-resistance mechanism. An activity-based probe is a powerful tool for studying drug target proteins, but the current lack of activity-based gemcitabine probes with robust biological activity hinders research on gemcitabine. In this study, we developed three active probes based on gemcitabine, among which Gem-3 demonstrated excellent stability and labeling efficacy. We utilized Gem-3 in conjunction with chemical proteomics to identify intracellular target proteins. We identified 79 proteins that interact with gemcitabine, most of which were previously unknown and represented various functional classes. Additionally, we validated the increased expression of IFIT3 and MARCKS in drug-resistant cells, along with the activation of the NF-κB signaling pathway. These findings substantially contribute to our comprehension of gemcitabine's target proteins and further our understanding of the mechanisms driving gemcitabine resistance in pancreatic cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ac59d987afe9d30b0e83196487e44b2e63262f1" target='_blank'>
              Identification of Aberrant Expression of Gemcitabine-Targeting Proteins in Drug-Resistant Cells Using an Activity-Based Gemcitabine Probe.
              </a>
            </td>
          <td>
            Xiaomei Zhu, Yuqing Yuan, Kai Wang, Wei Shen, Qing Zhu
          </td>
          <td>2024-10-28</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy.
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="INTRODUCTION
The introduction of venetoclax has revolutionized the treatment landscape of acute myeloid leukemia, offering new therapeutic opportunities. However, the clinical response to venetoclax varies significantly between patients, with many experiencing limited duration of response.


AREAS COVERED
Identified resistance mechanisms include both intrinsic and acquired resistance to VEN. The former is associated with cell lineage and differentiation state. The latter includes dependency on alternative BCL-2 family anti-apoptotic protein(s) mediated by genetic, epigenetic, or post-translational mechanisms, mitochondrial and metabolic involvement, as well as microenvironment. Understanding these mechanisms is crucial for optimizing venetoclax-based therapies and enhancing treatment outcomes for patients with acute myeloid leukemia. This review aims to elucidate the primary mechanisms underlying resistance to venetoclax and explore current therapeutic strategies to overcome this challenge.


EXPERT OPINION
In patients with venetoclax resistance, alternative options include targeted combination therapies tailored to individual cases based on cytogenetics and prior treatments. Many of these therapies require further clinical investigation to validate their safety and efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c30c2a61a5f4af70a1f4bd56b17cae94beeebe" target='_blank'>
              BCL-2 inhibition in acute myeloid leukemia: resistance and combinations.
              </a>
            </td>
          <td>
            Qi Zhang Tatarata, Zhe Wang, Marina Konopleva
          </td>
          <td>2024-11-18</td>
          <td>Expert review of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Among the spectrum of mutations driving malignancy, mutation in the TP53 gene is perhaps the most important event leading to therapy resistance. Only ~25% of patients with TP53-mutated acute myeloid leukemia (AML) respond to standard induction therapy, with dismal median survival of 4.2 months. This underscores a pressing need to characterize therapy resistance mechanisms for TP53-mutated AML and to identify novel therapeutic approaches.
 To investigate TP53 mutation-driven chemoresistance, we generated isogenic AML cells harboring TP53 WT, mutant, or knockout (KO) variants. TP53 WT cells exhibited the highest sensitive to Venetoclax+Azacytidine (VenAza), while TP53-/mut cells displayed varying sensitivities, with TP53-/R248Q cells being the least responsive. This variability suggests that different mutations confer distinct levels of TP53 function loss. TP53-/mut and KO AML cells showed reduced sensitivity to cytarabine, etoposide, and VenAza-induced apoptosis without impairment of G1 arrest or cell cycle. To pinpoint the dysfunction in intrinsic apoptotic signaling allowing TP53 mutant cells to evade apoptosis, we performed Gene Set Enrichment Analysis (GSEA) on RNA-seq data. This analysis demonstrated significant enrichment of p53 signaling pathways in WT cells treated with VenAza, but not in TP53-/R248Q and KO cells. Further we observed that despite the loss of key pro-apoptotic regulators (BAX, PUMA, NOXA), upregulation inactivator protein BIM compensated for proapoptotic stimuli in TP53-/R248Q and KO after VenAza. While mitochondrial outer membrane permeabilization (MOMP) is typically considered a crucial step in apoptosis, BH3 profiling analysis revealed comparable MOMP (priming) at baseline between TP53-mutant and TP53-WT AML primary tumors (n=37) and cell lines. This suggests that MOMP is not the definitive “point-of-no-return” for TP53 mutants. Instead, we found a blockade in executioner caspase-3/7 activity in TP53-/mut and KO AML cells, following VenAza or cytotoxic chemotherapy, contrasting with WT cells. This finding was also recapitulated in TP53 null isogenic solid cancer cell lines, revealing a novel post-MOMP resistance mechanism in TP53-deficient cells across cancer types.
 To identify targetable dependencies driving caspase blockade in TP53 defective cells, we conducted whole-genome CRISPR knockout screen in WT, TP53-/R248Q, and KO cells following VenAza treatment. Our findings confirmed that knockout of pro-apoptotic genes (BAX, PMAIP1, BCL2L11) conferred resistance across all genotypes. Notably, we discovered a novel functional selective dependence on BIRC5, a member of the inhibitor of apoptosis (IAP) family, in TP53-/R248Q cells exposed to VenAza. Mechanistically, BIRC5 deletion strongly sensitized TP53-mutant/KO cells to VenAza and other chemotherapy drugs by restoring caspase 3/7 activation. RNA-seq and global proteomic analyses of isogenic cells revealed upregulation of IAP family genes in TP53-mutant/KO cells. We validated translational implications of these findings by differential RNA-seq analysis comparing TP53-mutants (n = 63) and WT (n = 668) AML patient samples, and reverse phase protein array (RPPA) in TP53-mutants (n=153) as compared to TP53-WT (n=501) primary tumors. Both analyses identified BIRC5, encoding survivin, as one of the top overexpressed targets in TP53-mutant primary tumors. Extending beyond AML, TCGA data shows BIRC5 upregulation in TP53-mutant tumors across 19/25 cancers, indicating a pan-cancer dependency in p53-inactivated tumors.
 To uncover targetable vulnerabilities, we conducted high-throughput drug screening (n=250 agents) in isogenic AML cells, with IAP inhibitors emerging as top hits against TP53-mutant/KO cells. In isogenic mice (CDX) models, a combination of VenAza with IAP inhibitor demonstrated strikingly durable leukemia blast inhibition in the bone marrow and spleen, with triple therapy showing superior efficacy in TP53-/R248Q mice compared to WT treated with VenAza alone (median survival 44.5 vs 26 days). Our research unveils a critical post-mitochondrial defect in TP53 mutant cells as a key driver of chemoresistance and shows IAP inhibition can reinstate the apoptotic potential of both cytotoxic chemotherapy and targeted therapies. This insight offers a potential therapeutic target for TP53-mutated cancers, spanning both hematological and solid tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0919464d8033f4a4af2bbdfd96f46aefd72e0ff5" target='_blank'>
              Failure to Execute Caspase Activation Drives Therapy Resistance in TP53 Mutated AML
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, Fang Qi Lim, Yang Mi, Elyse A Olesinski, Yichen Wang, S. Jasdanwala, K. Bhatia, Xiao Xian Lin, Valeriia Sapozhnikova, P. Mertins, Brandon D Brown, Y. Qiu, A. Letai, Alexandre Puissant, S. Kornblau, J. Krönke, Edward Ayoub, Koji Itahana, Michael Andreeff, Shruti Bhatt
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="
 Background: Prostate cancer is a prevalent malignancy in men, with treatment resistance posing a significant challenge to successful cure. Acquired Resistance to Therapy (ART) arises from two main components: (1) Tumor Microenvironment (TME) and (2) mutations or epigenetic changes at gene level within the tumor cells themselves. While most cancer therapies target the intrinsic biology of cancer cells, growing evidence indicates that these treatments can also harm the stromal cells within TME, potentially driving ART. The biology of ART may vary based on the types of treatments, cell types, and organ locations. Thus, this study aims to dissect ART components driven by TME-mediated cellular processes in a comprehensive manner. Hypothesis: We propose that cancer therapies induce stress responses in non-cancerous cells within the TME. These stress responses involve the secretion of cytokines, mediators, and growth factors, which we term Stress Response Secretory Programs (SRSPs). We further hypothesize that targeting SRSPs could overcome ART that helps to improve treatment specificity and efficacy. Materials and Methods: We employed comprehensive single-cell profiling of common metastatic sites included liver, prostate, lung, bone, lymph node and spleen in preclinical models to identify secreted proteins and dysregulated pathways caused by genotoxic therapies. In vitro, we evaluated treatment resistance in murine prostate epithelial cell lines (Myc-CAP) by exposing them to conditioned media (CM) from irradiated murine fibroblasts (F4M2), using cell growth assays. Results: Single-cell RNA sequencing of organ samples revealed significant expressions of SRSPs and increased inflammatory responses in liver and prostate fibroblasts as well as other stromal cells treated with carboplatin. In vitro, CM from irradiated F4M2 cells enhanced cell proliferation and survival of Myc-CAP cells in response to docetaxel treatment. Conclusion: Our findings highlight the crucial role of SRSPs in mediating ART through interactions within the TME, which alter tumor cell phenotypes. Targeting these SRSPs holds potential for overcoming treatment resistance and improving therapeutic outcomes, underscoring the need to consider TME complexity in cancer treatment strategies.
 Citation Format: Tony Lok Heng Chu, Armand Bankhead, Ilsa Coleman, Sander Frank, Peter S. Nelson, Tarana Arman. Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c940709273c954da8cb4bd8042d98f46649ce8ee" target='_blank'>
              Abstract C033: Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment
              </a>
            </td>
          <td>
            Tony Lok Heng Chu, Armand Bankhead, Ilsa M Coleman, Sander Frank, Peter S. Nelson, Tarana Arman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC), a type of lung cancer, has long been one of the leading causes of cancer-related deaths globally. Advances in understanding the genetic mutations associated with NSCLC, such as EGFR, and KRAS mutations, have led to significant progress in developing treatments tailored to them. However, despite these developments, challenges like drug resistance and variability in patient-specific responses still persist. This paper delves into the current research on genetic mutations in NSCLC, and the effectiveness of related treatments, focusing on the EGFR and KRAS gene mutations. While treatments have improved patient outcomes, they are often limited by various factors such as the development of resistance and off-target effects. This research highlights the importance of personalized treatments and emerging therapies, such as advanced drug delivery systems and new immunotherapies in overcoming these pre-existing limitations. The findings provide insight for future research and the need for continued innovation in treatments. The current unresolved issues such as patient-specific responses, secondary mutations, and the off-target effects suggest that developments in the near future might need to focus on enhancing personalization, the cost, and mechanisms regarding overcoming resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fbd2f35ccfc8cce2c77597e66db8b61e788c8aa" target='_blank'>
              Advancements in Targeted Therapies Based on the EGFR, and KRAS Genetic Mutations in Lung Cancer
              </a>
            </td>
          <td>
            Weiting Feng
          </td>
          <td>2024-12-06</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations in genes encoding critical epigenetic regulators are frequently noted in bladder cancer, however, the impact of these mutations on therapeutic efficacy is unclear. One of the most common driver mutations in bladder cancer occurs in the KDM6A gene, which encodes a histone demethylase that promotes gene transcription. Retrospective analyses of patients with bladder cancer demonstrated that KDM6A mutations correlate with improved overall survival (OS) with immune checkpoint therapy (ICT), while they are associated with lower OS in patients undergoing cisplatin-based chemotherapy. Mechanistic studies utilizing CRISPR-Cas9 mediated deletion of Kdm6a showed reduced expression of DNA mismatch repair (MMR) and DNA double-stranded base repair (DSBR) genes in tumor cells with improved response to anti-PD-1 therapy and attenuated sensitivity to cisplatin-based chemotherapy in preclinical models of bladder cancer. Additionally, the loss of Kdm6a-mediated reduction in glycolysis and intratumoral lactate accumulation impaired histone 3 lysine 9 lactylation (H3K9la) and histone 3 lysine 18 lactylation (H3K18la) in Tregs with concurrent decrease in the expression of key genes including Foxp3, Tgfb and Pdcd1 and their immune-suppressive function. Further, reduced expansion of PD-1hi Tregs improved the ratio of cytotoxic T cells to Tregs and response to anti-PD-1 therapy in Kdm6a deficient tumor-bearing mice. Collectively, this study provided key insights into the role of KDM6A-mediated epigenetic regulation of DNA repair and metabolic reprogramming which potentially govern response to chemotherapy and ICT thus highlighting the utility of KDM6A mutation status for patient stratification and development of personalized treatment algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825bb07017841ff9cc12c29d05779f53d82d0047" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming differentially regulates responses to immune checkpoint therapy and chemotherapy in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, D. Raychaudhuri, Bidisha Chakraborty, Swadhin Meher, Aminah J. Tannir, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, P. Pilié, Sangeeta Goswami
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Therapy resistance and tumor recurrence are barriers to achieving long-term survival in cancer patients, particularly evident in glioblastoma (GBM), known for its dismal 5-year survival rate below 10%. Despite aggressive treatment regimens, including maximal resection, alkylating chemotherapy, and radiation, the majority of GBM patients experience relapse within 7-9 months post-initial intervention. Recurrent GBM poses surgical challenges and resists subsequent treatments, leading to an almost universally fatal outcome and an average survival of 12-18 months. Metabolic networks can serve as ancient stress-protection mechanisms that operate independently of genomic changes, offering an initial line of defense against exogenous pressures. Although the role of metabolism in cancer has gained prominence, our understanding of the metabolic adaptations following chemoradiotherapy and their contributions to therapy resistance and tumor recurrence in highly aggressive GBM tumors remains incomplete. We postulate that metabolic adaptations play a pivotal role in facilitating therapy resistance and GBM recurrence. Exploring the metabolic dependencies of these persistent cells may unveil vulnerabilities for therapeutic exploitation to counteract chemoradiotherapy resistance. We established clinically relevant patient-derived in vitro and in vivo models mirroring the standard GBM treatment protocol. Using state-of-the-art single-cell and bulk multi-omics technologies, we profiled the temporal transcriptomic and metabolomic adaptations from therapy-naïve primary GBM to chemoradiotherapy-resistant recurrent tumors. Our findings revealed unique metabolic transitions that occurred during chemoradiotherapy treatment in GBM cells that were conserved in vitro and in vivo across ten genetically diverse GBM patients. In a pre-clinical animal trial, a novel metabolism-targeting combinatorial treatment regimen prevented the emergence of recurrent tumors and significantly prolonged survival compared to chemoradiotherapy alone. Altogether, our results uncover distinct metabolic adaptations mediating therapy resistance and tumor recurrence in GBM. These findings emphasize the need to consider metabolic flexibility and dependencies for effective therapeutic strategies, offering potential avenues to overcome chemoresistance in recurrent GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4fcfc27c4c675a70aa8ea56a48322545c93ae2b" target='_blank'>
              TMET-24. MULTI-OMIC CHARACTERIZATION OF THE TEMPORAL METABOLIC ADAPTATIONS UNDERPINNING THERAPY RESISTANCE AND TUMOR RECURRENCE IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            E. Martell, H. Kuzmychova, H. Senthil, E. Kaul, Ujala Chawla, V. Banerji, C. M. Anderson, C. Venugopal, Donald Miller, T. Werbowetski-Ogilvie, Sheila K. Singh, Tanveer Sharif
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="

 Acute myeloid leukemia (AML) is a hematological malignancy that is associated with uncontrolled clonal expansion and the accumulation of myeloid blasts in the bone marrow. A major challenge of the treatment of AML is the high relapse rates that are often observed in patients despite advances in treatment options. Consequently, there remains a need to identify potential therapeutic candidates that can improve patient survival outcomes and reduce the risk of relapse in AML patients. The objective of this study was to utilize RNA sequencing and bioinformatic analyses to identify candidate targeted therapies for use in combination with Duocarmycin SA (DSA), a natural product antitumor antibiotic, for the treatment of AML. We hypothesized that RNA sequencing of DSA treated cells will identify novel genes that can be potentially targeted for AML drug combination studies. To test our hypothesis, human AML cell lines (Molm-14 and HL-60) were treated with high and low concentrations of DSA for 36 hours. Molm-14 cells were cultured with 100 pM (high) and 11.12 pM (low) of DSA and HL-60 cells were cultured with 500 pM (high) and 112.7 pM (low) of DSA. At 36 hours, the cells were harvested, RNA was isolated from the cells using Qiagen's RNeasy mini kit and RNA sequencing and gene expression profiling analysis were performed by the Hartwell Center for Biotechnology and the Center for Applied Bioinformatics at St. Jude Children's Research Hospital, respectively. Functional enrichment analysis was performed using the NIH's Database for Annotation, Visualization and Integrated Discovery (DAVID) tool to identify top DSA-induced pathways that were activated and target genes within the pathways for which targeted drug candidates can be identified. Overlapping differentially expressed genes (DEGs) for all four DSA treated conditions (p-value <0.05, Log >0.5) were visualized using Venny 2.1.0. Our results showed that the top pathway, oxidative phosphorylation, that is activated in all DSA treatment groups is related to mitochondrial induced cell-death. Genes that are involved include mitochondrially encoded cytochrome c oxidase 1, 2 and 3 (MT-CO1, MT-CO2, and MT-CO3). In conclusion, our findings suggest that DSA induced apoptosis in human AML cells is mediated by a mitochondrial mediated mechanisms. Additionally, the list of genes involved in the DAVID analysis provide novel therapeutic targets for which targeted drug candidates can be identified and evaluated in combination with DSA for the treatment of AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14350ea5f4f4614664b56926b3777e992a53071b" target='_blank'>
              Using Drug-Induced RNA Sequencing and David Analysis to Identify Targeted Therapies for Use in Combination with Duocarmycin SA for the Treatment of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            William A Chen, Valeri Filippov, Jie Fang, Pedro Ochoa, C. Casiano, Hongjian Jin, S. Dovat, Jun J Yang, Kristopher E Boyle, Olivia Francis
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="

 Introduction:
 Over recent years, the introduction and scalability of next-generation sequencing technologies have significantly expanded our knowledge of the genetic underpinnings of multiple myeloma (MM), a heterogeneous hematologic malignancy characterized by clonal plasma cell proliferation in the bone marrow. At present, cost-effective short-read technologies are extensively utilized for sequencing. Nevertheless, these platforms are limited in their ability to fully capture repeated regions and structural variants. Progress in long-read sequencing technologies, including HiFi-sequencing (HiFi), produces extended reads with increased precision, enabling more comprehensive and continuous genome assemblies. Another notable benefit of HiFi is its ability to integrate genome and methylome data, which seems increasingly needed given that epigenetic regulation has been identified as a major driver of resistance in MM.
 Methods:
 We collected CD138+ cells from the bone marrow of four newly diagnosed and nine relapsed / refractory MM patients. Sequencing of all samples was performed with a coverage of 30x using HiFi sequencing. In addition, we conducted short-read sequencing with 100x coverage to facilitate a direct comparison between the two approaches. We performed in-depth coverage analysis using mosdepth to compare the read mappability of the two sequencing technologies. Comparative downstream analysis, with a specific focus on structural variations (SVs), single nucleotide variations (SNVs), and methylome analysis was performed.
 Results:
 The goal of this study was to conduct a technical feasibility analysis to examine genome coverage as well as detection of structural variants (SVs) and single nucleotide variants (SNVs) by HiFi long-read sequencing and short-read sequencing in a cohort of MM patients with heterogeneous disease activity. As anticipated, a mean genome coverage of 30.1x was achieved by HiFi-sequencing across all samples. Read length was consistently >10,000 bp with a mean of 13,047 bp (range: 10,503-14,836). The median quality value (QV) for each base in the sequence was reassuring, with values ranging between 34 and 40. An average of 5,658,975 regions with high coverage (each region = 500 bp) were seen by HiFi sequencing as compared to 5,620,710 regions by short-read sequencing , resulting in a difference of 19.1 Mbp with improved coverage. By applying ClairS-TO, a deep-learning algorithm to uncover somatic mutations, we could observe that the HiFi technology could detect more somatic variants (n = 151,555) as compared to the short-read approach (n = 129,921). Structural variant calling and analysis are currently ongoing and will be performed by established tools for long-read vs. short-read analysis. Furthermore, we acquired healthy plasma cells from 13 individuals and sequenced with HiFiseq. The metyhlome and genome of these samples will serve as the reference for further comparison analysis with multiple myeloma patient samples through which we will provide insights into the MM cells specific genome-wide methylattion profile as well as genomics events.
 Summary:
 In summary, we here report on the feasibility of HiFi sequencing for enhanced genomic and methylome characterization in MM. Our final analysis is ongoing and updated results will be presented at the meeting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd9f73e5821c1e47d91572d35969d953aae14cc2" target='_blank'>
              Implementing Hifi Sequencing for Enhanced Genomic and Methylome Characterization in Multiple Myeloma
              </a>
            </td>
          <td>
            Seungbin Han, Nazia Afrin, B. Tolhuis, M. Truger, Peter Raab, U. Munawar, C. Vogt, Shilpa Kurian, S. Nerreter, Xiang Zhou, C. Haferlach, H. Einsele, J. Waldschmidt, L. Rasche, K. M. Kortüm
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b1219475c2c72e7d5e477e93702ece4b16c3cc" target='_blank'>
              A Genome-Wide CRISPR/Cas9 Screen Identifies Regulatory Genes for Stem Cell Aging.
              </a>
            </td>
          <td>
            Peng Su, Yi-liang Miao
          </td>
          <td>2024-12-21</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a multi-clonal disease, existing as a milieu of clones with unique but related genotypes as initiating clones acquire subsequent mutations. However, bulk sequencing cannot fully capture AML clonal architecture or the clonal evolution that occurs as patients undergo therapy. To interrogate clonal evolution, we performed simultaneous single cell molecular profiling and immunophenotyping on 43 samples from 32 NPM1-mutant AML patients at different stages of disease. Here we show that diagnosis and relapsed AML samples display similar clonal architecture patterns, but signaling mutations can drive increased clonal diversity specifically at relapse. We uncovered unique genotype-immunophenotype relationships regardless of disease state, suggesting leukemic lineage trajectories can be hard-wired by the mutations present. Analysis of longitudinal samples from patients on therapy identified dynamic clonal, transcriptomic, and immunophenotypic changes. Our studies provide resolved understanding of leukemic clonal evolution and the relationships between genotype and cell state in leukemia biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ca51b5a9370b4c12038c29c1cdd25d0324a8d0" target='_blank'>
              Genotype-immunophenotype relationships in NPM1-mutant AML clonal evolution uncovered by single cell multiomic analysis
              </a>
            </td>
          <td>
            Morgan Drucker, Darren Lee, Xuan Zhang, Bailee N. Kain, Michael Bowman, D. Nicolet, Zhe Wang, R. Stone, K. Mrózek, Andrew J Carroll, D. Starczynowski, R. L. Levine, John C. Byrd, Ann-Kathrin Eisfeld, Nathan Salomonis, H. L. Grimes, Robert L Bowman, Linde A. Miles
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="PURPOSE Combined BRAF, MEK, and EGFR inhibition can induce clinical responses in BRAF-V600E-mutant colon cancer, but rapid resistance often occurs. METHODS We use serial monitoring of circulating tumor DNA cell-free plasma DNA (cfDNA) in a patient case study in addition to organoids derived from mouse models of BRAF-V600E-mutant intestinal cancer, which emulated the patient's mutational profile to assess drug treatment efficacy. RESULTS We demonstrate dynamic evolution of resistance to combined EGFR/BRAF/MEK inhibition in a pediatric patient with metastatic BRAF-V600E-mutant, mismatch repair-stable colon cancer. Initial resistance to targeted therapy was associated with development of MET amplification. Sequential treatment with chemotherapy and targeted therapy resulted in clearing of the resistant MET-amplified clone. Rechallenge with combined BRAF/EGFR inhibition resulted in clinical and radiographic response, demonstrating these treatments may be non–cross-resistant. Tumor organoids were used to model clinical findings and demonstrated effectiveness of combined targeted therapy and chemotherapy. CONCLUSION These findings suggest rational strategies for combining sequential chemotherapy and BRAF-/EGFR-directed therapy in BRAF-V600E-mutant colon cancer to prevent resistance and improve outcome. The data demonstrate rapid clonal dynamics in response to effective therapies in BRAF-V600E-mutant colon cancer that can be monitored by serial cfDNA analysis. Moreover, in mismatch repair-proficient BRAF-V600E-mutant colon cancers, combined EGFR and BRAF/MEK therapy is not cross-resistant with standard chemotherapy, suggesting new rational combination treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c9b185f528634189bb86b982380c362c5137613" target='_blank'>
              Evolution of Rapid Clonal Dynamics and Non–Cross-Resistance in Response to Alternating Targeted Therapy and Chemotherapy in BRAF-V600E-Mutant Colon Cancer
              </a>
            </td>
          <td>
            Srilatha Simhadri, Jillian N Carrick, Susan Murphy, Om A Kothari, Husam Al-Hraishami, A. Kulkarni, Nahed Jalloul, Katarina Stefanik, Manisha Bandari, Kavya Chettur, Ming Yao, V. Ginjala, R. Groisberg, Howard S. Hochster, Janice Mehnert, Gregory Riedlinger, Hossein Khiabanian, Michael P Verzi, Kevin Tong, Shridar Ganesan
          </td>
          <td>2024-12-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="

 Background. T-cell acute lymphoblastic leukemia (T-ALL) is caused by the accumulation of genomic alterations that disrupt the development and proliferation of T-cells. These include somatic mutations in multiple genes, with NOTCH1 seen in about 50% of patients. In addition to mutations, driver chromosomal rearrangements, such as those affecting BCL11B (14q32) form the basis of sub-classification per latest WHO (5th ed) and ICC classifications and treatment decisions. In addition, ICC has proposed multiple provisional entities based on the underlying driver genomic rearrangements, often affecting T-cell receptors (TRB/TRG) with genes such as TAL1/3, TLX1/3, LMO1/2, KMT2A rearrangements, PICALM::MLLT10, SET::NUP214 among others. Consequently, clinical genomic profiling and work-up of T-ALL cases requires the use of advanced, high-throughput genome-wide techniques for detecting both somatic mutations and chromosomal structural variants. NGS offers comprehensive mutational profiling, detecting point mutations, insertions, deletions, and copy number variations at a nucleotide level. Optical genome mapping (OGM) is a DNA-based technique, that enables genome-wide structural variant analysis at a high-resolution, including those alterations often missed by conventional cytogenetics. Integrating OGM and NGS in the diagnostic workflow allows for a more detailed and accurate genomic characterization of T-ALL, facilitating better classification and diagnosis. In this study, we present the results of targeted NGS and OGM data from our clinical lab in a large cohort of T-ALL patients.
 Methods. All patients diagnosed with T-ALL and who underwent bone marrow (BM) examination were identified from the medical records. Somatic mutation analysis was performed on BM aspirate samples using a clinical-grade 81-gene panel through next-generation sequencing (NGS). All patients underwent chromosome banding analysis (CBA). A subset of cases underwent OGM using ultra-high molecular weight DNA to identify driver genomic aberrations.
 Results. Our cohort included a total of 125 patients diagnosed with T-ALL [63 (50.4%) treatment-naïve; 62 (49.6%) previously treated]; 31% also met the criteria for early T-cell precursor acute lymphoblastic leukemia (ETP-ALL). The median age of the patients was 36 years, with 94 men (75.2%) and 31 women (24.8%).
 By NGS, 98 (78.4%) patients had at least 1 mutation. The number of mutations per case ranged between 1 and 14. The most frequent mutations observed were in NOTCH1 (50.4%), PHF6 (24%), JAK3 (19%), DNMT3A (18%), NRAS (17%), and TP53 (17%). The majority of these mutations were missense, while frameshift and nonsense mutations occured less frequently. The highest median variant allele frequency (VAF) was seen in PHF6 (64%), followed by WT1, FBXW7, JAK3 and DNMT3A, all of which showed median >30%. The median NOTCH1 VAF was 22%. NOTCH1 mutations aggregated in the C-terminal PEST domain (nonsense or frameshift), and NOD/NODP domains (missense).
 CBA showed normal karyotype in 50 (40%) and abnormal karyotypes in 75 (60%) patients; complex karyotype was seen in 14 (11.2%) cases. Due to the limitations of CBA in T-ALL, primarily due to the poor growth of T-ALL cells under in vitro culture and the cryptic nature of most abnormalities, we performed OGM on 18 cases. Seventeen (94%) patients showed Tier 1/2 cytogenetic abnormalities. In 14 (78%) patient, a putative gene fusion implicated in T-ALL pathogenesis could be identified. These included known driver rearrangements involving BCL11B [TLX3::BCL11B (n=3), t(14;16) and t(2;14;10)], ETV6 [ETV6::MN1, CCDN2::ETV6, HOX transcription factors [TRB::HOXA13; fus(7;7), PICALM::MLL10, KMT2A::MLLT1, SPTAN1::NUP214], as well as novel fusions, such as NUP98::YY1, CBL [t(1;11)]. Among copy number changes, CDKN2A/B deletions were most frequent, seen in 5 patients. Four patients showed chromoanagenesis indicating a genomic catastrophe resulting in multiple chromosomal rearrangements and copy number changes.
 Conclusions. Combining data from both NGS and OGM, enabled accurate sub-classification of T-ALL cases, including those into the provisional entities defined by the ICC. This study demonstrates the feasibility and effectiveness of using genomic analysis (NGS and OGM) to fully understand the major regulators of leukemia cell growth and metabolism in treating T-ALL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35ffc52f305357d39b508b0bb6d3c6473d570b0" target='_blank'>
              Genomic Profiling of T-Cell Acute Lymphoblastic Leukemia/Lymphoma (T-ALL) for Identification of Driver Aberrations Using Combined Next Generation Sequencing and Optical Genome Mapping in Clinical Diagnostic Laboratory
              </a>
            </td>
          <td>
            Iman Sarami, Ruchi Singhal, Mark A. Routbort, K. Sasaki, Daniel Domogala III, G. Toruner, Shibu Ninan, Nitin Jain, K. Patel, E. Jabbour, H. Kantarjian, C. Bueso-Ramos, L. J. Medeiros, R. Luthra, Guilin Tang, R. Kanagal-Shamanna
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>225</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ace1a7b748e6be8dfa6d49964b342cccfe62af" target='_blank'>
              PCK1 as a target for cancer therapy: from metabolic reprogramming to immune microenvironment remodeling
              </a>
            </td>
          <td>
            Na Liu, Xiao-Ren Zhu, Chang-Ying Wu, Yuan-Yuan Liu, Minbin Chen, Jin-Hua Gu
          </td>
          <td>2024-11-22</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dac92d6b19ba0a7b037b162b322bc5d897b5e68" target='_blank'>
              Programmed cell death pathways in lung adenocarcinoma: illuminating tumor drug resistance and therapeutic opportunities through single-cell analysis.
              </a>
            </td>
          <td>
            Long Li, Shancheng He
          </td>
          <td>2024-12-23</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55b842a30320c99847f56c0cadef798436c42b45" target='_blank'>
              PFKFB3-dependent redox homeostasis and DNA repair support cell survival under EGFR-TKIs in non-small cell lung carcinoma
              </a>
            </td>
          <td>
            Nadiia Lypova, Susan M. Dougherty, Brian F. Clem, Jing Feng, Xinmin Yin, Xiang Zhang, Xiaohong Li, Jason Chesney, Y. Imbert-Fernandez
          </td>
          <td>2024-12-18</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/660334082207693e7b39f13fca8e1d97d42e1441" target='_blank'>
              Comprehensive review of drug resistance in mammalian cancer stem cells: implications for cancer therapy
              </a>
            </td>
          <td>
            B. Mengistu, Tirunesh Tsegaw, Yitayew Demessie, Kalkidan Getnet, A. Bitew, M. Kinde, Asnakew Mulaw Beirhun, Atsede solomon Mebratu, Yesuneh Tefera Mekasha, Melaku Getahun Feleke, Melkie Dagnaw Fenta
          </td>
          <td>2024-12-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f970e4e2fc5fa77ad8483f91874195b589fe0b" target='_blank'>
              Single-cell RNA-seq reveals diverse molecular signatures associated with Methotrexate resistance in primary central nervous system lymphoma cells.
              </a>
            </td>
          <td>
            Rosuke Osako, A. Hayano, Atsushi Kawaguchi, R. Yamanaka
          </td>
          <td>2024-12-05</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="

 Pre-B cell acute lymphoblastic leukemia (ALL) is the most common cancer in children. Although modern treatment protocols cure a majority of patients, relapsed disease remains a leading cause of pediatric death from cancer. Epigenetic dysregulation plays a critical role in ALL therapeutic resistance, with mutations in epigenetic regulators identified at the time of relapse. Epigenetic dysregulation and variability can propel disease evolution by driving leukemia cell heterogeneity and plasticity. To uncover epigenetic drivers of ALL progression, we sought to comprehensively map the DNA methylation landscape in matched diagnosis-relapse ALL specimens using methods that effectively capture epigenetic variability, aiming to identify the regulators and effectors of epigenetic instability in ALL progression. We aimed to construct genome-wide maps encompassing the dynamic DNA methylation landscape in paired diagnosis-relapse samples of a defined genetic subtype of ALL. We carried outwhole genome bisulfite sequencing (WGBS) on 36 samples representing paired diagnosis and relapse specimens from 18 patients with t(12;21) ETV6-RUNX1 pre-B ALL treated on modern Children's Oncology Group protocols. This represents the most common chromosome translocation in pre-B ALL which is typically associated with favorable prognosis, highlighting the importance of identifying epigentic drivers of treatment failure and relapse. We analyzed WGBS data using the informME method, an information-theoretic approach that effectively captures methylation stochasticity and has been previously applied to the identification of non-mutated epigenetically altered driver genes in ALL (Koldobskiy et al., Nature Biomedical Engineering 5: 360-76, 2021). This approach allows the use of WGBS reads to encapsulate epigenetic variability, in order to map genomic regions undergoing evolution from diagnosis to relapse. DNA methylation levels measured by WGBS are used to generate DNA methylation probability distributions, and comparisons of these distributions from diagnosis to relapse are accomplished using an information-theoretic measure of Jensen-Shannon distance (JSD) which measures the distance between two probability distributions and thereby includes stochasticity rather than conventional analyses which only evaluate mean methylation differences. Subsequently, we ranked genomic regions and genes with significant JSD between diagnosis and relapse and tested for functional enrichments. Our analysis revealed significant differences in the DNA methylation landscape between ALL diagnosis and relapse, associated with genome-wide DNA hypomethylation and globally increased methylation entropy in the majority of relapse samples compared to their corresponding diagnosis samples. Focusing on specific genomic regulatory regions, we noted that despite the global trend of genome-wide hypomethylation at relapse, bivalent promoters showed hypermethylation at relapse compared to diagnosis. Mapping methylation discordance revealed a striking enrichment among the target genes of Polycomb Repressive Complex 2 (PRC2), a core epigenetic regulator linked to suppression of gene transcription. Notably, among the 500 genes exhibiting the greatest JSD between diagnosis and relapse, the top four enrichments converged on targets of EED and SUZ12, critical components of PRC2. To further evaluate the role of PRC2 in pre-B ALL, we employed CRISPR-Cas9-mediated genome editing to silence EED, in the B-cell leukemia Reh cell line harboring t(12;21) ETV6-RUNX1 to evaluate the influence of PRC2 modulation on gene expression and the leukemic phenotype. We employed a powerful approach that comprehensively analyzes DNA methylation genome-wide and quantifies epigenetic variability to map evolution of the dynamic epigenetic landscape in a common genetic subtype of ALL from diagnosis to relapse. This approach permits the dissection of gene-regulatory mechanisms underlying leukemic progression. Identification of PRC2 targets as the predominant effectors of epigenetic instability in leukemic progression can inform novel therapeutic strategies, such as approaches aimed at restricting epigenetic variability that contributes to therapeutic resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a83f3fbb398def62708d3edde2848e84e752f91" target='_blank'>
              Mapping the Dynamic DNA Methylation Landscape in Pre-B ALL from Diagnosis to Relapse Reveals a Role for Aberrant PRC2 Regulation
              </a>
            </td>
          <td>
            Athanasia Liapodimitri, Oscar Camacho, Kayleigh Lunsford, Ashley R Tetens, Jordyn Craig-Schwartz, Kegan O. Skalitzky, K. Rabin, Andy Feinberg, Michael A Koldobskiy
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Phased telomere to telomere (T2T) genome assemblies are revolutionising our understanding of long hidden genome biology “dark matter” such as centromeres, rDNA repeats, inter-haplotype variation, and allele specific expression (ASE). Yet insights into dikaryotic fungi that separate their haploid genomes into distinct nuclei is limited. Here we explore the impact of dikaryotism on the genome biology of a long-term asexual clone of the wheat pathogenic fungus Puccinia striiformis f. sp. tritici. We use Oxford Nanopore (ONT) duplex sequencing combined with Hi-C to generate a T2T nuclear-phased assembly with >99.999% consensus accuracy. We show that this fungus has large regional centromeres enriched in LTR retrotransposons, with a single centromeric dip in methylation that suggests one kinetochore attachment site per chromosomes. The centromeres of chromosomes pairs are most often highly diverse in sequence and kinetochore attachment sites are not always positionally conserved. Each nucleus carries a unique array of rDNAs with >200 copies that harbour nucleus-specific sequence variations. The inter-haplotype diversity between the two nuclear genomes is caused by large-scale structural variations linked to transposable elements. Nanopore long-read cDNA analysis across distinct infection conditions revealed pervasive allele specific expression for nearly 20% of all heterozygous gene pairs. Genes involved in plant infection were significantly enriched in ASE genes which appears to be mediated by elevated CpG gene body methylation of the lower expressed pair. This suggests that epigenetically regulated ASE is likely a previously overlooked mechanism facilitating plant infection. Overall, our study reveals how dikaryotism uniquely shapes key eukaryotic genome features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebe895c868a2f8ac32e5a5a400c1fdc993b92b30" target='_blank'>
              Long-read genomics reveal extensive nuclear-specific evolution and allele-specific expression in a dikaryotic fungus
              </a>
            </td>
          <td>
            Rita Tam, Mareike Möller, Runpeng Luo, Zhenyan Luo, Ashley W. Jones, Sambasivam K. Periyannan, J. Rathjen, B. Schwessinger
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7f6d0809acd049c0dda666004c8622cf27bd784" target='_blank'>
              Dissecting L-glutamine metabolism in acute myeloid leukemia: single-cell insights and therapeutic implications
              </a>
            </td>
          <td>
            Yanli Chen
          </td>
          <td>2024-11-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Mutated RAS, a key oncogenic driver in multiple myeloma (MM), can be present in up to 50% of primary MM. Several novel pharmacological inhibitors developed to target specific KRAS point mutations, have been successfully incorporated in the treatment of solid cancers, but their potential impact in MM has yet to be determined. Here, we performed genome-scale CRISPR studies to decipher the functional gene perturbation landscape of KRAS-mutated MM after treatment with specific KRAS inhibitors as well as the potential mechanisms of resistance to these molecules.



 We performed a total of 11 genome-scale CRISPR gene activation or CRISPR gene editing (knockout, KO) studies in 5 MM lines with distinct KRAS point mutations (KHM-1B and XG-7 [G12C]; KARPAS-620 and KP-6 [G12D]; or MM.1S [G12A]) after treatment with specific KRAS inhibitors (MRTX-1133, MRTX-1257, BI-2865, RMC-6236) in clinically achievable concentration ranges (as used in patients with solid tumors). Potency and mutant-specific in vitro activity of these inhibitors in the respective cell lines was assessed with bioluminescence assays and flow cytometry and confirmed to be similar to KRAS-mutant lines from our own in vitro studies (eg. MIA PaCa-2 [G12C]) as well as from other groups. Functional downstream abrogation of KRAS-dependent growth and survival pathways via MEK/MAPK and PI3K/AKT was determined by immunoblotting to confirm dephosphorylation of ERK and/or AKT. The level of apoptosis induction was assessed with flow cytometry after AnnexinV/PI staining. Cell cycle analyses were performed using DAPI staining. Furthermore, we performed RNA sequencing in 6 KRAS mutant cell lines to investigate their transcriptomic response to KRAS-inhibitor treatment.



 Our genome-scale CRISPR activation and KO studies in 5 MM lines with mutant KRAS (G12A/G12C/G12D) revealed a heterogeneous and cell line-specific pattern of gene perturbations regulating response vs. resistance to KRAS inhibition. CRISPR activation screens identified positive regulators of RAS-MAPK signaling (eg. SHOC2) and upstream surface receptors (eg. EGFR) among the highest scoring genes involved in resistance to KRAS inhibitors. Interestingly, CRISPR activation of KRAS itself was identified as a promoter of resistance, possibly due to reduced target saturation. Notably, many of the top “hit” genes are not recurrently genetically altered or differentially expressed in MM nor do they represent clinical “high-risk” features. In turn, CRISPR gene KO studies pointed to a key role of regulators of cell homeostasis (eg. KEAP1, NF2L1) in conferring KRAS inhibitor sensitization vs. resistance. Moreover, genes encoding for GTPase-activating proteins (GAPs) and negative regulators of RAS/MAPK signaling (eg. LZTR1) were functionally important. Some of the genes identified here are not typically expressed in newly diagnosed MM (eg. EGFR), but their upregulation during KRAS inhibitor treatment may be a candidate biomarker for decreased response or early relapse.
 We examined the RNA sequencing profile of 6 KRAS-mutant MM lines after short-term treatment with specific KRAS inhibitors. Transcriptional responses to KRAS inhibition were also heterogeneous between cell lines, and included downregulation of RAS effector genes (eg. ETV4, ETV5) and negative regulators of RAS signaling (eg. DUSP6, SPRY4), pointing to a RAS-driven negative feedback loop. Other recurrently differentially expressed genes include cell cycle regulators (e.g., D-type cyclin) and genes involved in apoptosis and DNA repair. Upregulation of gene expression was predominant in genes involved in cell cycle arrest and stress responses.



 Our genome-scale CRISPR activation and knockout studies revealed a heterogeneous landscape of genomic perturbations regulating the response vs. resistance of MM cell to mutant-specific KRAS inhibitors. These compounds exhibit potent and specific activity against MM cells with the respective KRAS mutations, but our functional studies point to individual lines, even those harboring the same KRAS mutation, exhibiting their own distinct “resistome” against these inhibitors. These results underscore the complex functional genomics of MM cell sensitivity vs. resistance to KRAS inhibitors and have implications for the choice of potential combination partners of these inhibitors in future preclinical or clinical studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6776aa6359cf06dbe4b6f24f549ee2405f36198" target='_blank'>
              Functional Genomics Studies Decipher the Genetic Perturbations and Unravel Mechanisms of Response/Resistance upon Mutant-Specific KRAS Inhibition in Multiple Myeloma
              </a>
            </td>
          <td>
            Torsten Steinbrunn, Ryosuke Shirasaki, O. Dashevsky, Huihui Tang, S. Yamano, Phaik Ju Teoh, Lisa B Leypoldt, B. Glassner, Ricardo de Matos Simoes, James Christensen, Constantine S. Mitsiades
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-10</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="
 The concept of synthetic lethality has long held promise for the next generation of genetically targeted cancer therapies to follow the success of PARP inhibitors in the treatment of BRCA-mutant cancers. The classical definition of synthetic lethality arose in Drosophila with the observation that loss of either of two genes independently had little effect on cell viability but loss of both led to cell death. This definition has now been expanded to capture pairs of genes in which genetic alteration of one gene and pharmacological inhibition of the other leads to death of cancer cells while sparing the normal cells which lack the genetic alteration, leading to a broad therapeutic index. Functional genomics screening (using unbiased RNAi or CRISPR-based technology) has enabled the systematic discovery of novel synthetic lethal targets for drug discovery. The selective dependence of MTAP-deleted cells on PRMT5 is one of the strongest genetic interactions observed in early RNAi screens. Approximately 10-15% of all human cancer is MTAP-deleted, providing a large and diverse patient population. MTA-cooperative PRMT5 inhibitors, which bind preferentially in the presence of MTA, have been developed by scientists at Tango Therapeutics and elsewhere to leverage the synthetic lethal relationship between PRMT5 and MTAP loss. MTA-cooperative PRMT5 inhibitors inhibit PRMT5 in MTAP-deleted cancer cells and spare MTAP-expressing normal cells leading to clinically well-tolerated and efficacious therapies. MTA-cooperative PRMT5 inhibitors have broad therapeutic potential both as single agent and in combination with other therapies.
 Citation Format: Kimberly Briggs.Leveraging synthetic lethality for the development of novel cancer therapies. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539ee15239f2d3141b9b8d5ed5ea99f9aadfdf6f" target='_blank'>
              Abstract IA002: Leveraging synthetic lethality for the development of novel cancer therapies
              </a>
            </td>
          <td>
            Kimberly J Briggs
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Chimeric transcripts frequently function as oncogenic drivers and represent potential targets for tumor-specific therapies. Routine methods for detecting these transcripts include (i) approaches that can detect a limited number of pre-defined targets, such as immunohistochemistry, fluorescence in situ hybridization, and reverse transcription–quantitative polymerase chain reaction, and (ii) approaches that can simultaneously detect multiple fusions, such as NanoString assays and NGS targeted panels. Whole transcriptome sequencing enables the analysis of a wide range of cancer biomarkers, including dysregulated genes, molecular pathways, and both known and novel cancer driver fusion transcripts. However, this method is not commonly employed for fusion discovery due to inadequate coverage at the fusion breakpoint. To address this limitation, we have developed a novel bioinformatics approach that enables the highly accurate prediction of clinically significant ALK fusions from RNA sequencing data. This extends the functionality of whole transcriptome NGS and improves its cost-effectiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6be192e0efb51bb68f058262838062a19b2ac606" target='_blank'>
              A New Approach of Detecting ALK Fusion Oncogenes by RNA Sequencing Exon Coverage Analysis
              </a>
            </td>
          <td>
            G. Zakharova, M. Suntsova, Elizaveta Rabushko, Tharaa Mohammad, Alexey Drobyshev, A. Seryakov, E. Poddubskaya, Alexey Moisseev, Anastasia Smirnova, M. Sorokin, Victor Tkachev, Alexander Simonov, E.P. Guguchkin, E. Karpulevich, Anton Buzdin
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="

 Introduction
 Cure rates for pediatric B-cell acute lymphoblastic leukemia (B-ALL) have dramatically increased due to improved treatment regimens. However, patients who relapse continue to have poor outcomes. Risk of relapse is highly correlated with the level of detected minimal residual disease (MRD) after induction therapy. Despite MRD level being the strongest prognostic indicator in B-ALL, how leukemic blasts evade chemotherapy is poorly understood. In this study, we used single-cell targeted DNA sequencing (scDNA-Seq) and multiome sequencing (scRNA-Seq + scATAC-Seq) to characterize genomic, transcriptomic, and chromatin accessibility changes after treatment and identify drivers of chemoresistance in B-ALL.
 Methods
 Bone marrow aspirates were obtained from 41 pediatric patients across 4 high-risk B-ALL subtypes: KMT2A-rearranged, Ph+, Ph-Like, and iAMP21. Paired initial diagnosis (IDX) and MRD timepoint samples from these patients were used to produce simultaneous scRNA-Seq and scATAC-Seq data. A subset of 24 patients were profiled using scDNA-Seq targeted at 355 recurrently mutated sites in pediatric B-ALL.
 Results
 In the multiome dataset, we identified 357,235 leukemic blasts from 442,710 total cells using multiple lines of evidence, including marker expression and copy number variant calling. To understand how induction therapy affected arrest states, we projected cells onto a healthy pediatric bone marrow reference to identify patient-specific developmental states. Across the cohort, we found that HSPC-like and Mature-B-like leukemic blasts increased post-induction while most blasts maintained a Pre-pro-B-like state. To determine whether these changes occurred uniformly, we clustered patient arrest state changes and found four patterns defined by proportions of cell states along the hematopoietic trajectory. Surprisingly, these patterns were uncorrelated with MRD level or genetic subtype. To understand how specific cell types respond to treatment within patient clusters, we conducted differential gene expression analysis across timepoints. We found that upregulated genes at the MRD timepoint in cell states with increased abundance were enriched for cluster-specific pathways, displaying differing phenotypic responses to induction treatment. scATAC-seq analysis of the same blasts revealed that IRF and NFKB transcription factor activity corresponds with arrest states increasing after treatment, identifying potential regulators of the chemoresistance phenotype.
 Our cell state analysis also revealed that nearly half of patients displayed significant increases in myeloid-like blasts following treatment not limited to KMT2A-rearranged patients where myeloid lineage switch is well-documented. Myeloid-like blasts lowly express canonical B-cell immunophenotypic markers, including CD19, CD179b, and CD24. In contrast, these blasts express a unique set of markers with therapeutic relevance in acute myeloid leukemia, including CD83 and CD98. This increase in CD19-low, myeloid-like blasts following treatment suggests that induction therapy may promote expansion of populations that drive CD19-negative relapse after CAR-T therapy.
 To understand how mutagenesis may drive chemoresistance, we analyzed a median of 2,346 cells at IDX and 1,828 cells at MRD to identify high-confidence single nucleotide variants using targeted scDNA-Seq. Mutations were validated orthogonally through variant calling using the multiome dataset. For certain patients, mutations known to be enriched at relapse, including NRAS, NR3C1, MSH6, and PMS2, were observed at diagnosis and post-induction, demonstrating the potential for these mutations to enable chemoresistance early in treatment. However, overall mutagenesis rates across the cohort were limited which reinforces the lack of a common mutational mechanism driving chemoresistance in B-ALL.
 Conclusion
 Our study presents the first comprehensive multiomics characterization of pediatric B-ALL minimal residual disease. Complementary multiomics data indicates that while mutagenesis is uncommon after induction in B-ALL, leukemic blasts undergo drastic transcriptomic and epigenomic shifts following induction therapy. Given the patient-specific nature of these responses to treatment, our findings underscore the need for more personalized treatment regimens for high-risk B-ALL patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69a15d0c73fac875909a596f189c7063fd7e5d43" target='_blank'>
              Multimodal Single-Cell Profiling of Minimal Residual Disease in B Acute Lymphoblastic Leukemia Reveals Genetic and Non-Genetic Factors of Early Drug Resistance
              </a>
            </td>
          <td>
            Rushabh Mehta, Changya Chen, Fatemeh Alikarami, Steven M. Foltz, Avi Loren, Elizabeth Li, Jason Xu, Marilyn M. Li, Kathrin M Bernt, Kai Tan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gene therapy, which involves the delivery of genetic material into cells to correct an underlying genetic problem, has emerged as a promising approach for treating various conditions. To promote research in this rapidly evolving field, we developed the Gene Therapy Omnibus (GTO) (http://www.inbirg.com/gto/), a comprehensive resource containing detailed clinical trial data and molecular information related to gene therapy. The GTO includes 6333 clinical trial records and 3466 transcriptome profiles, with information on 614 altered genes and 22 types of gene therapy, including DNA therapies, RNA therapies and genetically-modified cell therapies. For each gene therapy product in a clinical trial, detailed information, such as altered gene name, structural components, indication, vector information, phase of the clinical trial, clinical outcomes and adverse effects, is provided when available. Additionally, 345 comparison datasets, including 29 single-cell RNA-sequencing datasets comprising information on both gene therapy and control samples, were established. Differential gene expression and downstream functional enrichment analyses were performed through standardized pipelines to elucidate the molecular alterations induced by gene therapy. The user-friendly interface of the GTO supports efficient data retrieval, visualization and analysis, making it an invaluable resource for researchers and clinicians performing clinical research on gene therapy and the underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64873a6a2d0e61325df0c910d686a855ae83c490" target='_blank'>
              GTO: a comprehensive gene therapy omnibus.
              </a>
            </td>
          <td>
            Xuehang Meng, Yujia Du, Chang Liu, Zhaoyu Zhai, Jianbo Pan
          </td>
          <td>2024-11-14</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Background:
 Acute myeloid leukemia (AML), the most common leukemia in adults, is associated with a low survival rate. Combining BCL-2 targeting, using the BH3 mimetic Venetoclax, with hypomethylating agents shows promise for AML treatment. Despite advancements in treatment, the persistence of drug resistance remains an obstacle, resulting in a discouraging 5-year survival rate of less than 30%. This underscores the critical need to deepen our understanding of the mechanisms underlying the development of therapy resistance.
 To discover the molecular basis of resistance to BH3 mimetic-based treatments, we have previously performed genome-wide loss-of-function CRISPR screens in AML cells treated with various BH3 mimetics and their combination. One of the top synthetic lethal targets in all the treatment groups was the gene encoding the mitochondrial citrate carrier (CIC), SLC25A1. CIC plays a crucial role in regulating glycolysis, fatty acid synthesis, sterol biosynthesis, and mitochondrial oxidative phosphorylation. Yet, the exact role of CIC in leukemia development and acquisition of resistance is unexplored. Based on our data, we hypothesize that augmented citrate transport from mitochondria to the cytosol rewires AML metabolism. At the same time, CIC targeting can enhance leukemia sensitivity to BH3 mimetics-based treatments, offering a potential avenue for improving AML therapy.
 Methods:
 Analysis of single-cell transcriptomics data from patients with AML and immunoblotting analyses of patient-derived xenografts and cell line models were utilized to examine CIC expression. Protein levels were also analyzed in BH3 mimetics-resistant AML cell lines compared to their isogenic, sensitive counterparts. SLC25A1 (CIC) was genetically deleted using CRISPR/Cas9 technology. Upon successful CIC ablation, we performed competition-based survival, proliferation, and apoptosis assays. Extensive metabolomics and RNA-sequencing experiments followed by pathway enrichment analyses were carried out on CIC knockout AML cell lines. Next, we pharmacologically inhibited CIC to determine its efficacy as a monotherapy or in combination with BH3 mimetics and other FDA-approved AML therapies. For this purpose, we performed CellTiter-Glo viability assays and Annexin V staining, analyzed by flow cytometry.
 Results:
 CIC is upregulated in AML and further induced upon the gain of BH3 mimetics resistance. Notably, SLC25A1 expression positively correlates with lower overall survival in AML patients and the degree of intrinsic BH3 mimetics-resistance in a panel of AML cell lines. Loss of CIC impedes AML cell growth and enhances sensitivity to cell death, even in therapy-resistant blasts. CIC ablation impairs AML bioenergetics, including mitochondrial respiration, glycolysis, as well as lipid metabolism by affecting citrate subcellular pools. Lastly, pharmacologic inhibition of CIC re-sensitizes AML cells to FDA-approved therapies.
 Conclusion:
 Our studies demonstrate that CIC is a potential prognostic marker for AML and the emergence of resistance to BH3 mimetics treatment. CIC regulates both mitochondrial and lipid metabolism, underscoring the critical role of mitochondrial function in AML biology and therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9526846f6425b1c9a335c5a98df15c82f9a7ad72" target='_blank'>
              Countering Development of Resistance to Treatment in AML By Targeting Metabolic Adaptations
              </a>
            </td>
          <td>
            Marta Romo-González, Tanima Kundu, M. Diaz‐Rubio, Xiaoyang Su, Matthew J. McBride, Christina Glytsou
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e10dcc942b3981d9a72139061b995f341cc1437" target='_blank'>
              Protacs in cancer therapy: mechanisms, design, clinical trials, and future directions.
              </a>
            </td>
          <td>
            Akash Vikal, Rashmi Maurya, Brij Bihari Patel, Rajeev Sharma, Preeti Patel, Umesh K Patil, Balak Das Kurmi
          </td>
          <td>2024-11-29</td>
          <td>Drug delivery and translational research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA-dependent protein kinase (DNA-PK) is a key effector of non-homologous end joining (NHEJ)-mediated double-strand break (DSB) repair. Since its identification, a substantial body of evidence has demonstrated that DNA-PK is frequently overexpressed in cancer, plays a critical role in tumor development and progression, and is associated with poor prognosis in cancer patients. Recent studies have also uncovered novel functions of DNA-PK, shifting the paradigm of the role of DNA-PK in oncogenesis and renewing interest in targeting DNA-PK for cancer therapy. To gain genetic insight into the cellular pathways requiring DNA-PK activity, we used a CRISPR/Cas9 screen to identify genes in which defects cause hypersensitivity to DNA-PK inhibitors. We identified over one hundred genes involved in DNA replication, cell cycle regulation, and RNA processing that promoted cell survival when DNA-PK kinase activity was suppressed. This gene set will be useful for characterizing novel biological processes that require DNA-PK activity and identifying predictive biomarkers of response to DNA-PK inhibition in the clinic. We also validated several genes from this set and reported previously undescribed genes that modulate the response to DNA-PK inhibitors. In particular, we found that compromising the mRNA splicing pathway led to marked hypersensitivity to DNA-PK inhibition, providing a possible rationale for the combined use of splicing inhibitors and DNA-PK inhibitors for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b81329cc49dba729d43bb24bfcaa92943d47c86" target='_blank'>
              Disruption of RNA Splicing Increases Vulnerability of Cells to DNA-PK Inhibitors
              </a>
            </td>
          <td>
            A. P. Kovina, Artem V. Luzhin, V. Tatarskiy, Dmitry A. Deriglazov, Natalia V Petrova, N. V. Petrova, L. Kondratyeva, O. Kantidze, S. V. Razin, A. K. Velichko
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Loss of function screens using shRNA (short hairpin RNA) and CRISPR (clustered regularly interspaced short palindromic repeats) are routinely used to identify genes that modulate responses of tumor cells to anti-cancer drugs. Here, by integrating GSEA (Gene Set Enrichment Analysis) and CMAP (Connectivity Map) analyses of multiple published shRNA screens, we identified a core set of pathways that affect responses to multiple drugs with diverse mechanisms of action. This suggests that these pathways represent “weak points” or “Achilles heels”, whose mild disturbance should make cancer cells vulnerable to a variety of treatments. These “weak points” include proteasome, protein synthesis, RNA splicing, RNA synthesis, cell cycle, Akt-mTOR, and tight junction-related pathways. Therefore, inhibitors of these pathways are expected to sensitize cancer cells to a variety of drugs. This hypothesis was tested by analyzing the diversity of drugs that synergize with FDA-approved inhibitors of the proteasome, RNA synthesis, and Akt-mTOR pathways. Indeed, the quantitative evaluation indicates that inhibitors of any of these signaling pathways can synergize with a more diverse set of pharmaceuticals, compared to compounds inhibiting targets distinct from the “weak points” pathways. Our findings described here imply that inhibitors of the “weak points” pathways should be considered as primary candidates in a search for synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68c29dcb0705107711d0d05473f868fd7dbb129" target='_blank'>
              Cancer Cell’s Achilles Heels: Considerations for Design of Anti-Cancer Drug Combinations
              </a>
            </td>
          <td>
            Valid Gahramanov, Frederick S. Vizeacoumar, Alain Morejon Morales, Keith Bonham, M. Sakharkar, Santosh Kumar, Franco J Vizeacoumar, A. Freywald, Michael Y. Sherman
          </td>
          <td>2024-12-17</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef73b59f7e05ebe6c2a352df52b047a092cb9e6" target='_blank'>
              State of the art CRISPR-based strategies for cancer diagnostics and treatment
              </a>
            </td>
          <td>
            E. Di Carlo, Carlo Sorrentino
          </td>
          <td>2024-12-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare and aggressive malignancy with limited treatment options, necessitating comprehensive molecular characterization to uncover actionable therapeutic targets. Here, we integrate bulk and single-cell transcriptomics, epigenomics, and DNA methylation profiling to construct a detailed molecular and cellular landscape of SNSCC. Our analysis identifies five transcriptionally and epigenetically distinct malignant subpopulations, including hypoxic (TC1) and proliferative (TC2) clusters, with TC1 showing significant association with poor clinical outcomes. We uncover a hypoxia-driven angiogenic axis, wherein TC1 cells secrete adrenomedullin (ADM), MIF, and VEGFA to activate endothelial tip cells, fostering tumor angiogenesis. Epigenetic reprogramming, characterized by alterations in DNA methylation and chromatin accessibility, underpins these transcriptional programs, with AP-1 transcription factors emerging as key regulators. Notably, ADM expression is epigenetically controlled, correlates with advanced clinical stage, and predicts reduced survival in head and neck cancer patients. This multi-omic study highlights the tumor heterogeneity and microenvironmental interactions driving SNSCC progression, revealing epigenetically regulated pathways as promising therapeutic targets for future intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f1bcee9ecccbc4aae5987d7312ce58acb94e80" target='_blank'>
              Multi-omic profiling identifies malignant subpopulations and a hypoxia-driven angiogenic axis as therapeutic vulnerabilities in sinonasal squamous cell carcinoma
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, Myeong S. Yu, Yoo-Sam Chung, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Innovative treatment options have greatly improved outcomes for acute lymphoblastic leukemia (ALL) patients over several decades, yet relapse and refractory disease remain a significant clinical challenge. The complex genomic heterogeneity of molecular subtypes contributes to poor prognosis, high morbidity, and recurrent drug resistance in relapse and refractory ALL patients. Identifying novel genetic interactions and molecular pathways capable of initiating ALL will help to develop new diagnostic criteria and to identify actionable drug targets.
 Here, we developed a novel in vivo screening method using transgenic zebrafish to unbiasedly identify collaborating oncogenic drivers that when co-expressed lead to T- and B-ALL. Our high-throughput approach allows for rapid screening of collaborating oncogenic drivers across hundreds of individual animals in a cost-effective manner. Specifically, we screened a transgene pool of 68 putative oncogenes identified from relapsed human ALL for synergies at inducing leukemia in a large-scale F0 transgenic screen. Using our approach, complex gene networks involved in cancer initiation in human patients can be rapidly interrogated and unraveled with in vivo resolution utilizing the zebrafish model. Low concentrations of transgene pools containing candidate cDNAs driven by a tissue specific immune cell promoter were co-injected into zebrafish along with a fluorescent reporter (rag2:mCherry). Animals that developed tumors express mCherry along with only 5 to 20 additional transgenes (median 15 genes), indicative of random concatemeric integration of transgenes into the genome. Using histopathologic analysis and bulk mRNA sequencing, we identified zebrafish with mCherry-labeled thymic T cell lymphomas, T-ALL, and B-ALL ((>140 animals). Analysis of transgene expression from bulk-RNA sequencing studies allowed us to nominate causative combinations of human transgenes that drive leukemia onset in the zebrafish. Most notably, we identified several unique leukemia and lymphoma transgenic zebrafish models, representing T-ALL, B-ALL, mixed-phenotype acute leukemia (MPAL), T- and B-cell lymphoma, and NK cell leukemia. Our analysis also elucidated novel synergies between the proto-oncogene SET and either activated NOTCH1 or IL7R mutations to initiate ALL.
 To verify nominated collaborating oncogenic drivers as causative at inducing cancer, we next performed microinjection validation studies with: 1) rag2:SET + rag2:intracellular notch1a + rag2:mCherry (n=7, mean latency 105±16 days) and 2) rag2:SET + rag2:mutationally-activated IL7R + rag2:mCherry (n=5, mean latency 120±25 days). Both combinations lead to robust ALL formation in zebrafish, validating the oncogenic role of these factors in leukemia initiation. Importantly, lymphoma and leukemia were not detected in animals injected with rag2:SET (n=19, >180 days followed), or rag2:intracellular notch1a (n=31, >180 days followed) or rag2:mutationally-activated IL7R (n=18; >180 days followed).
 SET is a multifunctional protein involved in transcriptional regulation and histone binding. While SET has been characterized in other cancers, we found that SET collaborates with known drivers to initiate T-ALL in vivo and is also highly expressed in a vast majority of human T-ALL. SET is a potent inhibitor of H3 acetylation, suggesting that epigenetic dysregulation has a critical role in ALL oncogenic transformation. Finally, to determine whether SET is also required for human ALL maintenance, we established doxycycline inducible shRNA human ALL lines (lentiviral pINDUCER10; PMID: 21307310). We found that induced knockdown of SET inhibited cell proliferation and viability in multiple T-ALL and B-ALL cell lines while also increasing apoptotic cell death and inhibiting cell cycle progression. Taken together, our work has identified a unique role for a SET in ALL initiation and maintenance. We have generated a robust transcriptome atlas of zebrafish lymphoid malignancies, representing novel transgenic zebrafish models of several acute lymphoblastic leukemia and lymphoma molecular subtypes. In addition, our unique screening approach can be further applied towards discovery of genetic factors capable of initiating pediatric leukemias, creating a molecular “roadmap” to leukemogenesis, and identify vulnerable pathways for clinical therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce81e9a8fb6d19be5390433ea8c7fa0f118d769" target='_blank'>
              In Vivo Genetic Screen Identifies Chromatin Remodeling Gene SET As a Collaborating Oncogenic Driver in Initiation and Maintenance of Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            James R. Allen, Luis Antonio Corchete Sanchez, M. N. Bakr, Miriam Fernández-Lajarín, Alexandra R Hazelwood, Nathan Ford, Alexandra Veloso, Esther Rheinbay, D. Langenau
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="

 Background : Emerging evidence suggests that viral infections play a crucial role in the pathogenesis of certain leukemias, with viral integration and its interaction with the human genome potentially contributing to disease onset and progression. Since Astolfi et al. identified the first case of TTMV::RARA-APL in a 6-year-old child, more than ten cases have been identified in the past two years. However, no other viruses have been reported to form fusion genes with human genes in leukemia. Our study leverages a vast repository of poly(A) RNA-seq data from AML patients to develop a sensitive bioinformatics workflow. This workflow aims to uncover the molecular footprints left by viruses in the leukemic transcriptome and discover novel viral-human fusion events that may have been previously overlooked.
 Methods : The cDNA library construction and sequencing were performed as previously described (Chen, Blood Cancer J 2021). Raw RNA sequencing data were initially processed using Fastp for adapter trimming, polyA/T trimming, and low-quality reads filtering. The high-quality reads were then mapped to the human reference genome (GRCH38) using the STAR aligner (version 2.7.10b). Samtools was used to sort the aligned BAM files by chromosome position and extract reads that were properly paired. A coustom Perl scripts was written to extract the reads without proper paired alignment. SPAdes was used to assemble the unmapped reads into scaffolds, which were then annotated by Blast against the NCBI Refseq Virus database, using an expected value (E-value) cutoff of less than 1e-10 and an alignment percent more than 80%. Due to the limited data available for TTV and TTMV in the NCBI RefSeq Virus database, we enhanced the accuracy of our annotations by incorporating sequences of TTV and TTMV from the NCBI Nucleotide database into our reference database. Reads with soft clipping were identified from the BAM alignment and grouped by the genome positions where soft clipping occurred. Cap3 was then used to assemble these soft clipping reads into consensus sequences (contigs) for each group. The soft-clipped contigs were aligned to the unmapped scaffolds using Blast to reconstruct the fusion transcripts.
 Results : A cohort of 712 AML cases was enrolled, including 154 children and 556 adults (divided by ≥18 years), with an age range of 0-89 years (median 42 years), and a male-to-female ratio of 375:337. Virus sequences were identified in 523 cases (73.46%), with an average of 4.25 viruses in one sample. Duplodnaviria was found in 414 cases (58.15%), Riboviria in 220 cases (30.90%), Varidnaviria in 38 (5.34%), Monodnaviria in 14 (1.97%) and other virus in 68 cases (9.55%). Alphatorquevirus (Torque Teno Viruses, TTVs) and betatorquevirus (Torque Teno mini Viruses, TTMVs) compose mostly of the Anelloviridae fraction, with TTV was found in 101 cases (14.19%) and TTMV in 30 cases (4.21%). Human herpesvirus 1 (HHV-1) was identified in 297 cases (41.71%), Cytomegalovirus (CMV) in 239 cases (33.57%), Epstein-Barr virus (EBV) in 25 cases (3.51%), and Human adenovirus C (HAdV-C) in 24 cases (3.37%), these were the most frequently identified pathogens in this cohort. Using this analytical approach, we detected one case with the TTMV::RARA fusion in this cohort. We re-analyzed the poly(A) transcriptome sequencing data from several cases of TTMV::RARA previously reported by our center using this analytical process. The re-analysis confirmed that the TTMV::RARA fusion could be successfully identified using this protocol. No other virus-human coding gene fusion was found in this cohort, underscoring the rarity of virus-human fusion genes.
 Conclusions : We present an analytical workflow for the detection of viral gene transcripts and viral-human gene fusions, providing a reference for future studies of tumor-associated viruses using poly(A) transcriptome sequencing data. This workflow facilitates the discovery of viral integration events and the identification of expressed viral transcripts in various diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef634d92d941b0552642f106fc9df1d17ba76473" target='_blank'>
              Investigation of Viral Gene Transcripts and Viral-Human Gene Fusions in Acute Leukemia Poly(A) RNA-Seq Data
              </a>
            </td>
          <td>
            Xiaosu Zhou, Jiancheng Fang, Xiaoli Ma, Fang Wang, P. Cao, Yang Zhang, Xue Chen, Jiaqi Chen, Ming Liu, Ming Liu, Hongxing Liu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="

 Introduction: Mantle Cell Lymphoma (MCL), a rare and incurable form of B-cell lymphoma, presents significant challenges in treatment, with limited median progression-free survival. MCL is characterized by the translocation t(11;14) leading to overexpression of the cyclinD1 and deregulation of the cell cycle processes and genomic instability. Molecular heterogeneity in MCL patients reveals that MCL patients with high expression of SOX11 unmutated IGHV correlate poorly with clinical outcomes. The targeted therapies targeting cell cycle and DNA Damage response pathways have shown efficacy. However, the outcomes are dismal. Recent studies have suggested that lncRNAs regulate the cancer cell cycle, and identifying the mechanistic insights of lncRNAs in regulating cell cycle factors will provide new insights into the regulation of fundamental cellular processes to develop rational therapies.
 Methods: To identify the differential expression of lncRNA in MCL patients, we conducted RNA-Sequencing (RNA-Seq) analysis on MCL patients (n=18), comparing them to CD19+ B-cells sorted from the peripheral blood of healthy donors (n=10). We trimmed the reads for quality, removed adapters using Trim Galore, and mapped them to the human genome (HG38 assembly) using HISAT 2. All the reads mapped within the genomic coordinates of lncRNAs were counted using featureCounts. We used LIMMA to analyze differential expression between the MCL and healthy donor samples. To validate our findings from the MCL cohort, we obtained publicly available RNA-Seq data from 16 MCL patients and analyzed them using the same pipeline. Finally, the top candidates were validated in the third cohort of 48 MCL patients and 10 normal CD19+ B-cells. We performed loss of Function, RNA-protein interaction studies, and complementation studies to confirm the role of identified lncRNA in activating critical molecular events in MCL using human MCL cell lines (n=3).
 Results: Our analyses revealed dysregulations of lncRNA in MCL. Specifically, we identified 385 upregulated and 387 downregulated lncRNAs (|log2FC| > 0.58 and adjusted p-value < 0.05) in our internal MCL cohort (n=18), and over 60% of them were also confirmed in the external cohort (n=16). Notably, the lncRNA called MALAT1 was among the top ten upregulated in both cohorts, further validated in our third cohort (n=48). To investigate the potential role of MALAT1 in MCL, we analyzed MCL samples with high MALAT1 expression (upper quartile) compared to those with low expression (lower quartile) across two cohorts. Differentially expressed genes identified from this comparison were used for functional network-based topological pathway analysis via MITHrIL. This analysis highlighted the KEGG pathway “Cell Cycle” as one of the top upregulated pathways in MCL samples with high MALAT1 expression. We further demonstrate that the loss of MALAT1 in MCL cell lines (n=3) downregulates the expression of Sox11, an important transcription factor in MCL. Further studies illustrated that loss of MALAT1 downregulates SOX11 expression by inhibiting the Rb/E2F1 mediated transactivation and halting MCL progression at the G1/S stage. Mechanistically, MALAT1 loss downregulates EZH2, a polycomb repressive complex protein, and activates the cell cycle-dependent kinase inhibitor p57 (CDKNIC). In MCL Patients (n=43) the expression of MALAT1 negatively correlates with p57 (r2=-0.55; pValue=1.19e-4) and positively correlate with SOX11 (r2=.38;pValue=0.0124). Reactivation of p57 inhibits the Rb/E2F1 transactivation of SOX11 by binding to E2F1 and inhibiting its transactivation function. Interestingly, the SOX11 direct targets (SMAD3, CACYBP, and MAPK8) are also downregulated upon Loss of MALAT1 in MCL.
 Conclusion: Our study is the first to describe the novel dysregulation signature of lncRNAs in MCL across two independent cohorts and dissect the role of MALAT1 in regulating the cell cycle progression by EZH2/p57/-Sox11 axis in MCL. Our study also demonstrates that loss of MALAT1 lncRNA in MCL upregulates p57 and regulates the Rb/E2F1 transactivation to downregulate SOX11 and further alter the expression of SOX11 direct target genes controlling cell cycle and proliferation in MCL. We are currently optimizing the novel LNA-Gapmers targeting MALAT1 to target SOX11-positive MCL and identify novel drug combinations to target MCL progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b893d88ec01217196517e8ac03dfdd89bd827b5" target='_blank'>
              Transcriptomic Analysis in Mantle Cell Lymphoma Patients Identified MALAT1 Long-Non-Coding RNA to Regulate Cell Cycle Progression By Modulating EZH2/p57/Sox11 Axis
              </a>
            </td>
          <td>
            S. Singh, Alessandro La Ferlita, Neeraj Jain, Maxine Berger, Sriya Sreekanth, Rosario Distefano, Mamta Gupta, Natarajan Muthusamy, Robert A Baiocchi, L. Alinari, N. Epperla, L. Sehgal
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="

 Internal tandem duplication mutations of FLT3 (FLT3-ITD) are the most frequent genetic alteration in adult acute myeloid leukemia (AML) and confer poor prognosis. The use of FLT3 inhibitors has improved outcome among FLT3-ITD mutant patients, but clinical efficacy is transient and limited, largely due to the emergence of drug resistance which can occur through activation of alternative signaling pathways or metabolic degradation of the inhibitors within the tumor microenvironment. Therefore, it is urgent to explore new intervention targets and develop novel therapeutic strategies for FLT3-ITD mutant AML.
 In this study, we use ribosome profiling sequencing from 13 healthy donors and 35 AML patients (including 17 FLT3-ITD mutant AML cases and 18 non-FLT3-ITD mutant AML cases) to identify differentially expressed genes.In FLT3-ITD mutant AML patients, 143 up-regulated genes were obtained, which are mainly involved in cell adhesion, embryonic skeletal cell formation, coping with hypoxia, and T-cell activation etc. 538 down-regulated genes are mainly associated with signal transduction, cell differentiation etc. Among these up-regulated genes, we validated the abnormally elevated expression of LPIN1 in FLT3-ITD mutant AML patients using RT-PCR. LPIN1 has been previously reported to be a divalent magnesium ion-dependent phosphatidic acid phosphatase and play a key role in lipid metabolism. Analysis of clinical data from public databases indicated that high LPIN1 expression is positively correlated with poor prognosis in FLT3-ITD mutant AML patients.
 In FLT3-ITD mutant AML cell lines, both shRNA-mediated knockdown of LPIN1 expression and pharmacological inhibition of LPIN1 activity suppressed cell proliferation and increased apoptosis, indicating that LPIN1 plays an important role in the growth and survival of FLT3-ITD mutant AML cells. The combination of the LPIN1 inhibitor Propranolol with the FLT3 inhibitor Quizartinib significantly reduced the cell proliferation of both FLT3-ITD AML cell lines and primary FLT3-ITD mutant AML patient samples. Furthermore, the combination of Propranolol with Quizartinib eradicated leukemic cells and prolonged survival in a xenograft FLT3-ITD mutant AML model.
 In summary, our study demonstrates that LPIN1 plays a pivotal role in FLT3-ITD mutant AML pathogenesis and chemotherapy resistance. LPIN1 inhibition synergizes with FLT3 inhibitor to eliminate FLT3-ITD AML cells in vitro and in vivo, providing a new therapeutic target for FLT3-ITD mutant AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d46b4fbe2fa52dbc2864a97cf62442c223b6b92c" target='_blank'>
              Ribosome Profiling Sequencing Reveals LPIN1 As a Potential Therapeutic Target for Acute Myeloid Leukemia with FLT3 -ITD Mutation
              </a>
            </td>
          <td>
            Gao Yuang, Qianyu Zhang, Zhichun Lv, Shuhong Liu, Yide Sun, Huiying Gao, Xiongwei Zhao, Xin Li, Yang Xue, Hao Li, Changyan Li, Hongmei Ning
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 To investigate the mechanisms of resistance conferred by the novel fusion gene PTPRC-CYRIB in T-cell acute lymphoblastic leukemia (T-ALL).



 Using next generation sequencing, we identified the novel fusion gene PTPRC-CYRIB in the bone marrow of a primary drug-resistant T-ALL patient, subsequently confirmed its presence by Sanger sequencing. Molecular cloning techniques were employed to construct a plasmid overexpressing the PTPRC-CYRIB gene. Stable overexpression of PTPRC-CYRIB in T-ALL cell lines was achieved via lentiviral transduction. The T-ALL cell line overexpressing PTPRC-CYRIB was designated as the overexpression (OE) group, while the control group (NC) consisted of cells transduced with empty vector. Polymerase chain reaction (PCR) and western blotting (WB) confirmed PTPRC-CYRIB gene expression in transduced cells. Growth curves were plotted to assess proliferation rates, colony formation assays were conducted to evaluate clonogenic potential, and cell counting Kit-8 (CCK8) assays were performed following treatment with Vincristine, Idarubicin, Cyclophosphamide, and Prednisone to determine drug resistance in the OE group compared to the NC group.



 We successfully constructed stable T-ALL cell lines overexpressing PTPRC-CYRIB. PCR and WB analysis confirmed PTPRC-CYRIB overexpression in the OE group compared to the NC group. Growth curve analysis revealed significantly increased proliferation rates in the OE group. Colony formation assays demonstrated enhanced clonogenic potential in OE cells. Following drug intervention, CCK8 assays indicated significantly reduced inhibitory effects of Vincristine, Idarubicin, Cyclophosphamide, and Prednisone on proliferation in the OE group compared to the NC group, with statistical significance (P < 0.05).



 Overexpression of the fusion gene PTPRC-CYRIB in T-ALL may contribute to the development of drug resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32258fb444ae9e089675472a4f80506c5534baaa" target='_blank'>
              Novel Fusion Gene PTPRC-Cyrib As a Drug Resistance Mechanism in T-ALL
              </a>
            </td>
          <td>
            Xindi Li, Le Yin, Zhihua Wang, Yue Sheng, Zhao Cheng, Hongling Peng
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1167f70aefb28c2bd36dcb2ed6f87d34742f535" target='_blank'>
              Single-cell landscape of innate and acquired drug resistance in acute myeloid leukemia
              </a>
            </td>
          <td>
            R. Wegmann, Ximena Bonilla, Ruben Casanova, Stéphane Chevrier, Ricardo Coelho, Cinzia Esposito, Joanna Ficek-Pascual, Sandra Goetze, Gabriele Gut, Francis Jacob, Andrea Jacobs, Jack Kuipers, Ulrike Lischetti, J. Mena, Emanuela S Milani, M. Prummer, Jacobo Sarabia del Castillo, Franziska Singer, Sujana Sivapatham, Nora C Toussaint, Oliver Vilinovszki, M. H. Wildschut, T. Thavayogarajah, D. Malani, R. Aebersold, M. Bacac, N. Beerenwinkel, C. Beisel, B. Bodenmiller, V. Heinzelmann-Schwarz, V. Koelzer, Mitchell P. Levesque, H. Moch, L. Pelkmans, Gunnar Rätsch, Markus Tolnay, Andreas Wicki, Bernd Wollscheid, M. Manz, B. Snijder, A. Theocharides
          </td>
          <td>2024-10-30</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>63</td>
        </tr>

        <tr id="

 Introduction: Myeloma has the most complex genomic architecture among blood cancers. Central to this complexity are structural variants (SVs), large-scale genomic changes with the capacity to disrupt chromatin organization. SVs can modify the epigenomic landscape near DNA breakpoints, leading to gene dysregulation without directly altering coding sequences or DNA copy number. Here, we elucidate SVs involving the previously hidden regions of the human genome, namely compact heterochromatin including centromeric regions. These dense and repetitive regions, traditionally dismissed as “junk DNA”, emerge as critical areas of investigation.
 Methods: Short-read whole-genome sequencing (srWGS) data from 23 CoMMpass cases were mapped to the CHM13v2 genome assembly by BWA-MEM. RNA-seq CoMMpass (N = 928) and control GSE148924 (N = 12) sets were processed by DESeq2 normalization, variance-stabilizing transformation, and limma batch correction. Gene under-/over-expression was assessed by Z-score (>2/<-2) and log2-fold change (>1/<-1). Patient-derived xenografts (PDXs, N = 13) and KMS27 and PCM6 cell lines were studied by multiomics: srWGS (Illumina), long-read HiFi WGS (PacBio), and Micro-C (CantanaBio); including cases with t(11;14), t(4;14), t(14;16), and hyperdiploidy. Raw reads were aligned to the CHM13v2 assembly. HiFi alignments were phased by WhatsHap. In-house algorithms were built for the analysis of SVs involving repetitive regions in srWGS and DNA methylation within the HiFi reads. Copy-number variants were assessed by CNVRobot. Micro-C data were processed with Juicer tools.
 Results: SVs affecting compact heterochromatin were detected in 61% of CoMMpass cases (14/23) and frequently involved (peri)centromeres of chromosomes 8, 12, 15, and 16. Breakpoints at partner loci often disrupted tumor-suppressor genes (CYLD, GAS8, UNC5D) and recombined near proto-oncogenes (MYC, NFKB1), which increased gene expression compared to control plasma cells.
 In a PDX case, we resolved the +1q as “jumping translocations” between 1q12 pericentromeric heterochromatin and 6q15, 15p11.2, and 18q11.2. The previous hg38 assembly could not resolve t(1;6) and t(1;15) due to DNA breakpoints being present in gaps of this reference genome. HiFi reads enabled breakpoint mapping at 1q (impossible by srWGS), and Micro-C data confirmed derivative chromosome der(1)t(1;18), der(6)t(1;6), and der(15)t(1;15) interactions, and were associated with loss of the partner chromosome arm. DNA methylation analysis of HiFi phased data showed significant hypomethylation of partner loci juxtaposed to the 1q pericentromere. We identified this pattern as a mark of compact heterochromatin rearrangements, confirming an “open chromatin” state, consistent with increased transcription in the CoMMpass data.
 Allele-specific DNA methylation changes were also observed in key chromosomal SVs involving super-enhancers, such as t(11;14) CCND1-IGH, t(4;14) NSD2-IGH, t(14;16) IGH-MAF, and t(6;8) TXNDC5-MYC. For example, in a t(11;14) PDX case, we detected hypomethylation (~2 kb) at the chr11 breakpoint, followed by 225 kb hypermethylation and subsequent hypomethylation spanning CCND1. Micro-C data showed interactions between these hypomethylated regions on chr11 and the IGH promoter through introduction of a de novo translocation-specific regulatory TAD loop absent in non-t(11;14) samples, allowing close interaction of the translocation breakpoint with CCND1. No methylation changes were observed in other SVs lacking super-enhancers or compact heterochromatin involvement.
 The level of compact heterochromatin breakage was not linked to global hypomethylation. However, global methylation analysis showed a higher proportion of methylated CpG sites in the t(4;14) group (71.8%, N = 4) compared to other molecular groups: t(11;14) (44.8%, N = 4), t(14;16) (46.1%, N = 2), and hyperdiploidy (34.1%, N = 4). This supports the role of NSD2 in forming H3K36me2 histone marks, essential for maintaining DNA methylation.
 Conclusions: Utilizing novel computational algorithms, complete genome assembly, and multiomics data integration, we provide the first genomic evidence of SVs affecting compact heterochromatin as common changes in the myeloma genome. These rearrangements alter the DNA topology, epigenomic code and gene expression in juxtaposed loci, suggesting a driving role in myeloma genome instability.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce365e86100cc20873a8250a59c923a1a8ca733" target='_blank'>
              Multiomics Data Integration in the Complete Myeloma Genome Reveals Frequent Centromeric Rearrangements and Their Epigenomic Consequences
              </a>
            </td>
          <td>
            A. Mikulasova, Enze Liu, Nathan Becker, Parvathi Sudha, R. Abonour, Brian A Walker
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 Lung cancer is the leading cause of cancer mortality, and despite improvements in treatment, tumors typically respond incompletely and resume growth after acquisition of drug resistance. Recent completion of a neoadjuvant osimertinib Phase II trial treating patients with surgically resectable stage I-IIIA EGFR-mutated non-small cell lung cancer (EGFRm NSCLC) (NCT03433469) (Blakely et. Al, JCO 2024) has highlighted the importance of further identifying non-genomic mechanisms of persistence and resistance to targeted therapy. Here we analyzed these patient samples to identify transcriptionally regulated signaling patterns enriched at Residual Disease (RD) after neoadjuvant osimertinib treatment compared to treatment naïve samples (TN), as well as signaling patterns enriched at RD in patients who ultimately develop disease recurrence after surgical resection compared to those who remain disease-free. Spatially resolved transcriptomic sequencing was performed, using the 10X Genomics Visium platform, on 36 tissue sections (n=9 TN, n=19 RD, n=4 Progressive Disease (PD), n=4 Tumor Adjacent Normal), from 25 patients. TN and PD samples were from standard-of-care surgical resections. 18 of 19 RD samples were from NCT03433469. The Visium array spots that tile the sequencing capture area are 55 µm in diameter, and thus can contain an estimated 1-10 cells within each array spot. After quality control, 91,582 array spots remained for downstream analysis. We identified high confidence tumor array spots annotated both as “Cancer” by a board-certified pathologist and with tumor characteristic copy number variations. Through dimensionality reduction clustering, we identified 10 major array spot clusters, including the high confidence tumor array spot cluster. These clusters were further subset into 44 high resolution clusters of array spot niches, 32 of which were shared across at least nine patients. Because array spots are not single-cell resolution, these niches represent spots with similar mixed cell type composition and transcriptional signatures. We identified significant increases in the CCL14+ Endothelial Cell Niche, CXCL2+ Endothelial Cell Niche, Lymphocytes + Endothelial Niche, and CXCL12+ Fibroblast Niche in RD samples compared to TN. We identified a cell signaling program defined by spatially co-expressed ligand-receptor pairs associated with focal adhesion, enriched in PD samples. This same pattern was enriched in RD samples from patients who had recurrent disease after surgical resection compared to RD samples from patients who remained disease free. These data describe an increase in immune trafficking in RD samples, as well as identify an EGFR- alternative signaling pathway correlating with osimertinib resistance. Identification of spatially resolved, differential architectural organization and signaling patterns in RD and PD states will expand our understanding of targeted therapy resistance in EGFRm NSCLC, with the potential to identify additional clinically actionable targets.
 Citation Format: Whitney Tamaki, Daniel L. Kerr, Wei Wu, Grant Eilers, Anatoly Urisman, Yu-Ting Chou, Philippe Gui, Shigeki Nanjo, Johannes R. Kratz, David M. Jablons, Trever G. Bivona, Collin M. Blakely. Identification of cell-cell signaling programs across the tumor-tumor microenvironment ecosystem and associated with clinical status in neoadjuvant osimertinib treated patient samples revealed by spatial profiling [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00831e7253b0b6e70e43b596e9e7ff252fbadae" target='_blank'>
              Abstract C025: Identification of cell-cell signaling programs across the tumor-tumor microenvironment ecosystem and associated with clinical status in neoadjuvant osimertinib treated patient samples revealed by spatial profiling
              </a>
            </td>
          <td>
            W. Tamaki, D. Kerr, Wei Wu, Grant Eilers, A. Urisman, Yu-Ting Chou, Philippe Gui, Shigeki Nanjo, J. Kratz, D. Jablons, T. Bivona, Collin M Blakely
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="

 Introduction
 Somatic TP53 mutations and 17p deletions resulting in the genomic loss of TP53 occur in 8% to 10% of de novo AML and are associated with chemotherapy resistance, relapse, and poor outcome. Increasingly, TP53 is being recognized as a guardian of immune integrity. We have previously demonstrated that TP53 abnormalities are linked to elevated PD-L1 and FoxP3 expression, as well as bone marrow (BM) infiltration by dysfunctional natural killer-like CD8+ T cells. In this study, we employ a systems-level approach to interrogate HL-60 TP53-isogenic AML cell lines as well as primary bone marrow (BM) samples from patients with TP53-m AML.
 Methods
 To assess the impact of the three most common mis-sense mutations within the p53 protein's DNA-binding domain on gene expression, we pursued lentiviral transduction of parental TP53null HL-60 cells with either a custom TP53-wt construct or the R175H, R248Q and R273H hotspot mutations (GenScript Biotech), followed by RNA-seq analysis. We then carried out proteomic profiling on both the secretome and whole AML cell lysates. We merged legacy sc-RNA-seq data (Penter al et. 2023; Lasry et al. 2023; van Galen et al. 2019) with newly generated sc-RNA-seq data from 4 patients with TP53-m AML, resulting in 139,610 high-quality transcriptomes from 70 patients, 13 of which had TP53-m AML. After batch correction, we re-annotated the integrated dataset based on a curated atlas of BM hematopoiesis (263,159 single-cell transcriptomes spanning 55 cellular states; BoneMarrowMap; Zeng et al. 2023).
 Results
 Upon comparing TP53null HL-60 with isogenic TP53-wt and TP53-m AML cells, we observed large transcriptional differences unique to each TP53 hotspot mutation. TP53R175H AML cells exhibited the greatest divergence from their TP53-wt counterpart, including elevated expression of inflammatory response genes, CXCL8, HSPA8 (known to interact with TP53 and associated with inferior outcomes in TCGA solid tumors), and genes involved in metabolic processes (ANXA1, DUSP1) and DNA binding (FOSB). Weighted gene coexpression network analysis (WGCNA) pinpointed 26 modules of interconnected genes. The top-ranking module by false discovery rate encompassed 3,847 genes significantly enriched in gene ontology terms related to protein and DNA metabolism. Congruent with these observations, secretome analyses of TP53R175H AML cells revealed an enrichment of aminoacid and RNA metabolism pathways. Next, we curated a compendium of gene sets capturing glycolysis, oxidative phosphorylation [OXPHOS], lipid metabolism, cellular senescence and AML stemness, and we calculated ssGSEA scores from bulk RNA-seq data. In contrast to TP53-wt and TP53R248Q AML cells, which displayed enrichment in cellular senescence, heme and fatty acid metabolism, HL-60 cells harboring the TP53R175H and TP53R273H hotspot mutations showed enhanced OXPHOS and a higher prevalence of primitive leukemia stem and progenitor states. To assess the clinical utility of these findings, we built a LASSO-based Cox regression model for feature selection from bulk RNA-seq data from 444 diagnostic Beat-AML2 BM samples. Thirteen metabolic (fatty acid and cholesterol biosynthesis; OXPHOS, production of reactive oxygen species), stemness and senescence-related scores from our compendium predicted overall survival (OS), irrespective of TP53 mutational status. Furthermore, patients with high senescence and OXPHOS scores experienced significantly worse outcomes (median OS = 0.88 months compared to median OS = not reached in patients with low gene expression scores; log-rank P = 0.00014), suggesting that the TP53 pathway may be dysfunctional in a high proportion of patients with TP53-wt AML.
 Finally, we computed single-cell gene set scores using our integrated AML atlas and we found that mitochondrial oxidative metabolism genes were expressed by cell clusters annotated as pro-erythroblasts, CFU-E and basophilic erythroblasts, which displayed high predicted pseudotime values, indicating large transcriptional changes. This finding is in keeping with recent reports supporting an erythroid-biased differentiation trajectory in TP53-m secondary AML (Rodriguez-Meira A, 2023).
 Conclusions
 Our study offers a conceptual framework for exploiting metabolic dependencies with prognostic significance in TP53-m AML. Targeting cancer metabolism could improve outcomes in this heterogeneous patient population.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0de4150ed01f0f486d2c2f28080890c0026b682" target='_blank'>
              Multi-Omic Analyses of TP53-Mutated Acute Myeloid Leukemia Identify Prognostic Metabolic Signatures
              </a>
            </td>
          <td>
            J. Vadakekolathu, Sarah Skuli, David J. Boocock, Clare Coveney, Eruchi G Ikpo, Bofei Wang, Elena M Fenu, Hussein A. Abbas, Catherine Lai, Martin P Carroll, S. Rutella
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="

 Background: Genomic, transcriptomic, and proteomic studies have significantly advanced our understanding of multiple myeloma (MM) and facilitated the development of targeted therapies. However, a challenge has been the limited correlation between transcriptome and proteome data, leaving gaps in our understanding of how gene expression relates to protein functionality. In-depth characterization of the MM translatome will provide a foundation for elucidating complex molecular mechanisms and developing translational-targeted therapeutic approaches.
 Methods: We collected bone marrow samples from 50 newly diagnosed MM (NDMM) patients and 9 healthy donors, and performed gene panel sequencing, RNA-seq, Ribo-seq and MHC-I immunopeptide sequencing on CD138+ cells.
 Results :We elucidated a coordinated regulation of cancer pathways and genes at both transcriptional and translational levels, correlated with immune function loss and heightened DNA repair processes. Additionally, certain genes were exclusively regulated at the translational level, resulting in the suppression of pathways associated with mitochondrial oxidative respiration. These findings enhance our understanding of immune function disruption and alterations in glycolytic energy metabolism in malignant plasma cells. Furthermore, the specificity of NDMM's translational regulation in response to ER stress suggests that existing transcripts are translated into proteins more rapidly than those induced de novo, indicating a potential initial adaptive response to stress. This underscores a potential therapeutic strategy that targets the essential ER stress response at the translational level, which is crucial for MM cell survival.
 Moreover, we discovered novel translatomic subtypes, among them, TE-C2 was associated with R-ISS III. This finding represents the initial indication that high-risk NDMM patients demonstrate distinctive translational characteristics. The TE-C2 group exhibited the worst clinical outcomes, independently providing prognostic information beyond R-ISS III. No significant difference in overall survival or progression-free survival between R-ISS III and non-R-ISS III patients suggested translational features-based grouping enhances prognostic precision. Another subtype associated with SF3B1 genomic mutations. Further integrating ChIP-seq and alternative splicing analyses of SF3B1-mutant cell lines' Ribo-seq data elucidated a mechanism for splicing-driven translational control. We observed that SF3B1 mutations lead to an increased number of exons with m6A modification sites, potentially regulating translational initiation of target mRNA transcripts through the recruitment of m6A reader proteins. This could be a potential regulatory mechanism to promote the translation of DNA damage response mRNAs.
 Translatomic analysis of NDMM also uncovers novel open reading frames (ORFs) with the potential to serve as cancer neoantigens.These noncanonical ORFs may play a crucial role in maintaining protein homeostasis and cell proliferation. The translation patterns of specific lncRNAs with well-established roles in cancer invasion or metastasis and those recognized as prognostic biomarkers strongly suggest their potential as promising targets. Additionally, MHC-I immunopeptidome mass spectrometry further validates the potential of noncanonical ORFs as cancer neoantigens. For instance, micropeptides derived from the LY9 dORF hold promising potential for NDMM immunotherapy.
 Conclusion: Our comprehensive analysis of the NDMM translatome not only expands our understanding of the molecular intricacies of NDMM but also paves the way for action toward personalized therapeutic strategies. By elucidating these translational features, we hope to contribute to developing more effective treatment approaches and a paradigm shift toward precision medicine in the management of NDMM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0deaef5bb423be80abab55c30379dcea57f4d16e" target='_blank'>
              Comprehensive Translatomic Analysis of Multiple Myeloma Unveils Novel Insights into Disease Pathogenesis
              </a>
            </td>
          <td>
            Lirong Liu, Le Wu, Xinyu Li, Xiaoyun Jin, Xiaoya Chen, Guangjue Tang, Huixing Zhou, Yuan Jian, Wenming Chen, Guang-Zhong Yang, Yan Huang, Fang Wang, Jia Yu, Wen Gao, Dong Wang
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb40aeff67c4162245cfc687f8adca56b59144a" target='_blank'>
              Integrated molecular and functional characterization of the intrinsic apoptotic machinery identifies therapeutic vulnerabilities in glioma
              </a>
            </td>
          <td>
            E. Fernández, Wilson X Mai, Kai Song, Nicholas A. Bayley, Jiyoon Kim, Henan Zhu, Marissa S Pioso, Pauline Young, Cassidy Andrasz, Dimitri Cadet, Linda M Liau, Gang Li, William Yong, Fausto J Rodriguez, Scott J. Dixon, Andrew J Souers, Jingyi Jessica Li, Thomas Graeber, T. Cloughesy, David A. Nathanson
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Patients with epidermal growth factor receptor mutant nonsmall cell lung cancer (NSCLC) often fail to treat gefitinib because of secondary drug resistance. The development of tumor drug resistance is closely related to variations in cancer cell metabolism. Single-cell metabolomics analysis can provide unique information about tumor drug resistance. Herein, we constructed a platform to study the secondary resistance of tumor cells based on single-cell metabolomics (sSRTC-scM). A gefitinib-resistant NSCLC cell line (PC9GR) was constructed by increasing the dose step by step. The metabolic profiles of parental PC9 cells and PC9GR cells with different drug resistance levels were detected by intact living-cell electrolaunching ionization mass spectrometry at the single-cell level. The data were analyzed by statistical methods such as t-SNE, variance, volcano plot, heat map, and metabolic pathway analysis. Using this platform, we found that the metabolic fingerprints of PC9GR cells can evaluate drug resistance degrees. The metabolic fingerprints continue to be altered with the increase of drug resistance. We revealed 19 metabolic markers of secondary resistance by variance analysis and clarified that the glycerophospholipid metabolic pathway of PC9GR cells changed significantly. In addition, we found that with the increase in drug resistance levels, the heterogeneity of single-cell metabolism became greater and the number of cells with weak drug resistance gradually decreased. This phenomenon can be utilized to illustrate the drug resistance degrees of PC9GR cells. This study provides diagnostic markers for evaluating the drug resistance of tumors and gives new insight into overcoming the secondary resistance of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84899e62659612e5b6164a21695dadcc8005bf4e" target='_blank'>
              Single-Cell Mass Spectrometry Studies of Secondary Drug Resistance of Tumor Cells.
              </a>
            </td>
          <td>
            Guizhen Zhu, Wenmei Zhang, Yaoyao Zhao, Guangyun Wang, Hanyu Yuan, Guangsheng Guo, Xiayan Wang
          </td>
          <td>2024-12-20</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88d1d3e327b2658624144e333e3ed0c37f8a284" target='_blank'>
              CRISPR/Cas9-induced double-strand breaks in the huntingtin locus lead to CAG repeat contraction through DNA end resection and homology-mediated repair
              </a>
            </td>
          <td>
            P. Śledziński, Mateusz Nowaczyk, Marianna Śmiełowska, M. Olejniczak
          </td>
          <td>2024-12-03</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Towards comprehensively investigating the genotype-phenotype relationships governing the human pluripotent stem cell state, we generated an expressed genome-scale CRISPRi Perturbation Cell Atlas in KOLF2.1J human induced pluripotent stem cells (hiPSCs) mapping transcriptional and fitness phenotypes associated with 11,739 targeted genes. Using the transcriptional phenotypes, we created a minimum distortion embedding map of the pluripotent state, demonstrating rich recapitulation of protein complexes, such as strong co-clustering of MRPL, BAF, SAGA, and Ragulator family members. Additionally, we uncovered transcriptional regulators that are uncoupled from cell fitness, discovering potential novel pluripotency (JOSD1, RNF7) and metabolic factors (ZBTB41). We validated these findings via phenotypic, protein-interaction, and metabolic tracing assays. Finally, we propose a contrastive human-cell engineering framework (CHEF), a machine learning architecture that learns from perturbation cell atlases to predict perturbation recipes that achieve desired transcriptional states. Taken together, our study presents a comprehensive resource for interrogating the regulatory networks governing pluripotency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf0f66e8e51806f9ebf2ea82ca2dd611e0145032" target='_blank'>
              A PERTURBATION CELL ATLAS OF HUMAN INDUCED PLURIPOTENT STEM CELLS
              </a>
            </td>
          <td>
            Sami Nourreddine, Yesh Doctor, Amir Dailamy, Antoine Forget, Yi-Hung Lee, Becky Chinn, Hammza Khaliq, Benjamin J. Polacco, Monita Muralidharan, Emily Pan, Yifan Zhang, Alina Sigaeva, Jan Niklas Hansen, Jiahao Gao, Jillian A. Parker, Kirsten Obernier, Timothy Clark, Jake Y. Chen, Christian Metallo, Emma Lundberg, T. Ideker, N. Krogan, Prashant Mali
          </td>
          <td>2024-11-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="As we age, many tissues become colonised by microscopic clones carrying somatic driver mutations. Some of these clones represent a first step towards cancer whereas others may contribute to ageing and other diseases. However, our understanding of the clonal landscapes of human tissues, and their impact on cancer risk, ageing and disease, remains limited due to the challenge of detecting somatic mutations present in small numbers of cells. Here, we introduce a new version of nanorate sequencing (NanoSeq), a duplex sequencing method with error rates <5 errors per billion base pairs, which is compatible with whole-exome and targeted gene sequencing. Deep sequencing of polyclonal samples with single-molecule sensitivity enables the simultaneous detection of mutations in large numbers of clones, yielding accurate somatic mutation rates, mutational signatures and driver mutation frequencies in any tissue. Applying targeted NanoSeq to 1,042 non-invasive samples of oral epithelium and 371 samples of blood from a twin cohort, we found an unprecedentedly rich landscape of selection, with 49 genes under positive selection driving clonal expansions in the oral epithelium, over 62,000 driver mutations, and evidence of negative selection in some genes. The high number of positively selected mutations in multiple genes provides high-resolution maps of selection across coding and non-coding sites, a form of in vivo saturation mutagenesis. Multivariate regression models enable mutational epidemiology studies on how carcinogenic exposures and cancer risk factors, such as age, tobacco or alcohol, alter the acquisition and selection of somatic mutations. Accurate single-molecule sequencing has the potential to unveil the polyclonal landscape of any tissue, providing a powerful tool to study early carcinogenesis, cancer prevention and the role of somatic mutations in ageing and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb1ed2352282f2639f8bc59358f511e72d35ab26" target='_blank'>
              Somatic mutation and selection at epidemiological scale
              </a>
            </td>
          <td>
            A. R. Lawson, F. Abascal, P. A. Nicola, S. Lensing, Amy L Roberts, Georgios Kalantzis, A. Baez-Ortega, Natalia Brzozowska, Julia S. El-Sayed Moustafa, Dovile Vaitkute, Belma Jakupovic, Ayrun Nessa, S. Wadge, Anna L. Paterson, Doris M. Rassl, R. E. Alcantara, L. O’Neill, S. Widaa, Siobhan Austin-Guest, M. Neville, M. Przybilla, Wei Cheng, M. Morra, Lucy Sykes, Matthew Mayho, Nicole Müller-Sienerth, Nick Williams, Diana Alexander, Luke M. R. Harvey, Thomas Clarke, Alex Byrne, Jamie R Blundell, Matthew D. Young, A. KrishnaaT., Mahbubani, K. Saeb-Parsy, Hilary C. Martin, M. R. Stratton, P. J. Campbell, R. Rahbari, K. Small, I. Martincorena
          </td>
          <td>2024-11-01</td>
          <td>None</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="

 Introduction: Therapy resistance driven by Leukemic Stem/Progenitor Cells (LSPCs) remains a challenge in Chronic Myeloid Leukemia (CML).Recent studies indicate that Tyrosine Kinase Inhibitor-resistant (TKI-R) cells are more reliant on mitochondrial (mt) Oxidative Phosphorylation (OXPHOS) for survival, highlighting a potential new therapeutic target. Mutations in OXPHOS-related genes are potentially deleterious and can be associated with reduced OXPHOS and activation of apoptosis. We hypothesize that TKI-sensitive (TKI-S) and TKI-R patients may exhibit distinct OXPHOS phenotypes, possibly involving mtDNA mutations.
 Methods: TKI-S patients were defined based on the achievement of Major Molecular Response at 12 months of TKI therapy (MMR, BCR::ABL1<=0.1%), and TKI-R included patients who did not achieve MMR at 12 months or developed blast crisis. CD34+ LSPCs cells were isolated from blood of chronic phase (CP) CML patients at diagnosis. Samples were cultured in the presence of physiological growth factors. Mt respiration (OXPHOS) was measured using the MitoStress Test kit on the Seahorse XF96 analyser (4 TKI-S, 5 TKI-R). Mt content was assessed by Mitotracker Green using flow cytometry. MitoTrace software was used to call mtDNA variants from 10X Single Cell (sc) 5' RNA sequencing data (Krishnan V et al Blood 2023) and LoFreq software to call somatic mutations in purified mtDNA from total leukocytes.
 Results: TKI-R CD34+ cells showed increased OXPHOS capacity compared with TKI-S, including increased basal oxygen consumption rate (median 82.5 vs 52 pmol/min/105 cells, p=0.043) and increased maximal respiration (median 209.9 vs 115.9 pmol/min/105 cells, p=0.0032). The spare respiratory capacity (“mt fitness”) of the TKI-R CD34+ cells was also increased, whilst TKI-S cells showed mt dysfunction (median 126.5 vs 54.9 pmol/min/105 cells, p= 0.012). There was no statistically significant difference in the mt content of CD34+ cells between TKI-R and TKI-S patients (202% vs 151%, normalised on normal CD34+ cells), suggesting that the respiratory deficiency observed in the TKI-S cells might be linked to mt function, rather than decreased mt number.
 To investigate whether mtDNA mutations were common in CML and associated with better response to TKI, we sequenced the mtDNA of purified mitochondria from total leukocytes of 80 CP-CML patients. We identified 187 mtDNA mutations in 60 (75%) CP-CML patients (median 2 per patient, range 1-21). TKI-S patients (41/80) showed more mutations at higher variant allele frequency (VAF) than TKI-R (139 vs 48 with median VAF 21% vs 7%, p<0.0001), including non-synonymous protein-coding mutations (49 vs 24 mutations at median VAF 13% vs 6%, p=0.0008). Patients with >=3 mtDNA mutations at diagnosis (n=21/61) had a higher probability of MMR (90% vs 68%, p=0.0038) and MR4.5 (BCR::ABL1<0.0032%; 63% vs 37%, p=0.011) at 24 months of imatinib treatment. A higher frequency of non-synonymous mutations was identified in the MT-COI gene, which is relevant for the assembly and function of the OXPHOS machinery. Mutations in MT-COI were observed in 15 patients. They were associated with better response to imatinib at 24 months, including better MMR (92% vs 67%, p=0.026) and MR4.5 (69% vs 43%, p=0.0413). We linked mtDNA mutation VAF and type with the available functional data, confirming that patients (4 TKI-S) carrying mutations with VAF>7% also had dysfunctional OXPHOS, whilst patients (5 TKI-R) with no mutations or mutations with VAF<=7% had higher mt function (p=0.0079).
 We next developed a bioinformatic pipeline to call mtDNA variants in LSPCs from scRNA-sequencing data. We counted the number of mtDNA variants/cell across different cell types and calculated the ratio of number of mtDNA variants in LSPCs relative to T/NK cells (the latter serving as an internal control). Our data showed that TKI-S (8 patients) had a higher ratio of cells carrying non-synonymous mtDNA mutations than TKI-R (12 patients; 3.1 vs 2.2 p=0.02), supporting the data observed in total leukocytes.
 Conclusions: This is the first study linking mtDNA mutations to OXPHOS activity and TKI response in CML. Non-synonymous mtDNA mutations may lead to decreased OXPHOS activity and enhanced response to TKI therapy. Our findings suggest that the strategy of pharmacological inhibition of OXPHOS could be exploited to sensitize LSPCs to TKI therapy and mtDNA mutations could be useful as a biomarker of response.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16fac9d55e66ba886904d845193112aa5b7f0a70" target='_blank'>
              Imatinib-Sensitive Chronic Myeloid Leukaemia Is Associated with mtDNA Mutations and Reduced Oxphos Capacity
              </a>
            </td>
          <td>
            I. S. Pagani, Vaidehi Krishnan, John Ouyang, C. Kok, BeiJun Chen, Elyse C. Page, P. Dang, V. Saunders, Jane James, Kelly Lim, N. Shanmuganathan, S. Branford, Charles Chuah, David T Yeung, Daniel Thomas, Timothy P Hughes, S. T. Ong, David M. Ross
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Disruption of the epigenetic landscape is of particular interest in acute myeloid leukemia (AML) due to its relatively low mutational burden and frequent occurrence of mutations in epigenetic regulators. Here, we applied an information-theoretic analysis of methylation potential energy landscapes, capturing changes in mean methylation level and methylation entropy, to comprehensively analyze DNA methylation stochasticity in subtypes of AML defined by mutually exclusive genetic mutations. We identified AML subtypes with CEBPA double mutation and those with IDH mutations as distinctly high-entropy subtypes, marked by methylation disruption over a convergent set of genes. We found a core program of epigenetic landscape disruption across all AML subtypes, with discordant methylation stochasticity and transcriptional dysregulation converging on functionally important leukemic signatures, suggesting a genotype-independent role of stochastic disruption of the epigenetic landscape in mediating leukemogenesis. We further established a relationship between methylation entropy and gene expression variability, connecting the disruption of the epigenetic landscape to transcription in AML. This approach identified a convergent program of epigenetic dysregulation in leukemia, clarifying the contribution of specific genetic mutations to stochastic disruption of the epigenetic and transcriptional landscapes of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4ef99fd7d1036db8f31536805f7bae25ae59017" target='_blank'>
              DNA methylation stochasticity is linked to transcriptional variability and identifies convergent epigenetic disruption across genetically-defined subtypes of AML
              </a>
            </td>
          <td>
            Eleanor Hilgart, Weiqiang Zhou, Eduardo Martinez-Montes, Adrian Idrizi, Rakel Tryggvadottir, Lukasz P. Gondek, Ravi Majeti, Hongkai Ji, Michael A Koldobskiy, Andy Feinberg
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-18</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Developmental and epileptic encephalopathies (DEEs) comprise a complex spectrum of neurological disorders characterized by neurodevelopmental delay and early‐onset seizures primarily caused by diverse genetic mutations. Traditional treatments have largely been symptomatic, focusing on seizure control without addressing the underlying genetic causes. The advent of gene therapy, particularly through antisense oligonucleotides (ASOs), offers a promising avenue toward targeted therapeutic interventions. ASOs by virtue of their ability to modulate gene expression at the mRNA level represent a sophisticated approach to counteract the effects of pathogenic mutations. This review delves into the recent advancements in ASO technology, highlighting its application in preclinical and clinical settings for DEEs. We present evidence of the efficacy of ASOs in ameliorating disease phenotypes in vitro and in vivo, alongside promising outcomes from ongoing clinical trials. The therapeutic landscape for DEEs is on the cusp of significant transformation, underscored by the potential of ASOs to offer precise, personalized, treatments that extend beyond symptomatic relief to potentially rectify the genetic underpinnings of these disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b206134247dab8146e40050bd0232b304a5bad4" target='_blank'>
              Antisense oligonucleotides as a precision therapy for developmental and epileptic encephalopathies
              </a>
            </td>
          <td>
            Paloma García Quilón, G. Volpedo, Serena Cappato, Loretta Ferrera, Federico Zara, Renata Bocciardi, Antonella Riva, P. Striano
          </td>
          <td>2024-11-01</td>
          <td>CNS Neuroscience & Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract. Lung cancer is the most aggressive and prevalent malignancies worldwide, contributing significantly to global health challenges with over 1.8 million new cases reported annually. Non-small cell lung carcinoma (NSCLC) accounts for the majority of these cases and is characterized by a poor 5-year survival rate of only 10%. Traditional therapies, including surgery, chemotherapy, and radiotherapy, face limitations such as drug resistance and systemic toxicity. The advent of Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated (Cas) systems (CRISPR/Cas) technology has transformed the landscape of gene editing, enabling precise targeting of genes involved in NSCLC pathogenesis. This review explores the therapeutic potential of this technology in NSCLC, focusing on key genes such as AIFM1, ROCK2, TSPAN4, and the PD-1 receptor protein, which play crucial roles in tumor progression and immune evasion. By elucidating the mechanisms through which the genes above help with lung cancer, we highlight how CRISPR/Cas9-mediated interventions can pave the way for innovative and effective treatment strategies. Additionally, we address the current challenges and prospects of implementing CRISPR/Cas9 in clinical settings to enhance the efficacy and specificity of lung cancer therapies, in order to improve patient outcomes and reduce the global burden of this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5695fa779ae7c0481e84826ad52efb38a86c56" target='_blank'>
              Advancements of CRISPR/Cas9 technology in non-small cell lung cancer research: Applications and current developments
              </a>
            </td>
          <td>
            Kongzhao Chen, Jingyi Chen
          </td>
          <td>2024-11-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Background. Ibrutinib is the first in class inhibitor of BTK, highly active in chronic lymphocytic leukemia (CLL) as monotherapy. The most typical response to ibrutinib is partial remission (PR) with measurable minimal residual disease (MRD) in blood, maintained until genetically driven resistance. We aim to investigate the mechanisms that allows MRD cells to survive despite BTK inhibition.
 Methods. Pre-treatment CLL cells and under treatment MRD were systematically collected at six homogeneous timepoints from 33 CLL patients who received ibrutinib in the IOSI-EMA001 or IOSI-EMA003 studies. Samples were characterized by high-dimensional flow cytometry, bulk genomic, transcriptomic, chromatin accessibility, single cell gene expression and chromatin accessibility, and spatial transcriptomics. In vivo validation was performed.
 Results. Genetic clonal evolution was observed in 33% (N=11) of the study population, characterized by the expansion of initially subclonal mutations without a consistent pattern in specific genes or pathways. At the level of somatic copy number abnormalities, no clonal shift was detected. BTK or PLCG2 mutations were not present in the MRD phase but emerged during clinical progression.
 Ibrutinib induced chromatin dynamics in MRD resulting in a predominantly closed state. Chromatin regions that became more accessible under ibrutinib (N=731) were enriched of binding sites for transcription factor (TF) that lay downstream ERK signaling. Regions that turned into a less accessible state (N=1580) were enriched for the binding sites of TF emanating from the BTK-containing proximal signalosome of the B-cell receptor (BCR) (NF-kB, JUN, NFAT).
 Consistently, at the transcriptomic level most of the differentially expressed genes between pre- and post-treatment samples were downregulated in MRD (N=1235), while only a fraction was upregulated (N=114). BCR, IL4, NF-kB, metabolism and proliferation signatures were downregulated in MRD (FDR<0.001), while MAPK and RAS signatures were upregulated (BCRlow/ERKhigh) (FDR<0.05). The BCRlow/ERKhigh phenotype was also present in residual disease within the tissue, as demonstrated by spatial transcriptomics of residual CLL in lymph nodes excised from patients receiving ibrutinib. This phenotype was further supported by transcriptomic analysis of leukemia splenocytes from TCL1 transgenic mice treated with ibrutinib.
 High-dimensional flow cytometry and transcriptomics revealed an upregulation of BCR surface molecules in the MRD. In contrast, receptor tyrosine kinases expression remained unchanged. Using phospho-flow cytometry and immunoblotting of MRD cells, we found that stimulation of the B-cell receptor (BCR) with an anti-IgM- but not stimulation of CD40 and Toll-like receptors- resulted in persistent RAS activation and ERK phosphorylation despite BTK block, while BTK and PLCG2 phosphorylation were expectedly abrogated.
 Multiomics matched single-cell RNA-seq and ATAC-seq uncovered the pre-existence of a functional BCRlow/ERKhigh clone before ibrutinib treatment with a shared gene regulatory network similar to post treatment samples.
 Administration of ulixertinib, an ERK1/2 kinase inhibitor, in combination with ibrutinib significantly delayed tumor growth in vitro and in vivo.
 Conclusion. Adaptation toBTK inhibition in CLL is mainly epigenetically driven and selects MRD cells with BCRlow/ERKhigh phenotype. RAS-RAF-MEK-ERK proved to be a vulnerability of MRD.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d04751935270912668600d42ba05c8ff5e8b1330" target='_blank'>
              Adaptation of Chronic Lymphocytic Leukemia to Ibrutinib Is Mediated By Epigenetic Plasticity of Residual Disease and By-Pass Signaling Via MAPK Pathway
              </a>
            </td>
          <td>
            L. Terzi di Bergamo, G. Forestieri, J. Loh, Amartya Singh, V. Spina, F. Jauk, A. Zucchetto, A. Condoluci, A. Bruscaggin, K. Pini, D. Piffaretti, E. Civanelli, C. Tarantelli, M. Salehi, Ilaria Romano, M. Pirosa, S. Bocchetta, Georgia Alice Galimberti, T. Bittolo, E. Tissino, L. De Paoli, C. Deambrogi, A. Frustaci, F. Autore, M. Merli, L. Scarfò, S. Rasi, M. Deodato, Jakob R. Passweg, R. Moia, C. Martines, P. Ghia, Franco Cavalli, E. Zucca, Bernhard Gerber, Georg Stüssi, Marco Montillo, Francesco Passamonti, Michael Gregor, L. Laurenti, A. Tedeschi, A. Tzankov, Francesco Bertoni, G. Gaidano, D. Efremov, V. Gattei, Hossein Khiabanian, Davide Rossi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="

 Background
 We previously demonstrated that additional genetic abnormalities (AGAs) detected at diagnosis had a negative impact on failure-free survival (FFS, as defined by ELN 2020) and molecular response for 200 frontline imatinib-treated patients (pts), despite proactive treatment intervention. The effect was evident for blood cancer related gene variants (CGVs) and aberrant structural rearrangements associated with the formation of the Philadelphia chromosome (Ph-associated). We have now sequenced the diagnosis samples of 315 pts treated with more potent frontline BCR::ABL1 inhibitors.
 Aim
 To determine if frontline more potent BCR::ABL1 inhibitors overcome the negative impact of AGAs detected at diagnosis.
 Methods
 An RNA-based targeted myeloid and lymphoid cancer-related gene next-generation sequencing method was used to detect single nucleotide variants, RNA splice altering variants, small insertions/deletions, gene fusions and focal gene deletions. Blood samples from pts enrolled in consecutive Australasian frontline CML clinical trials were investigated; 200 imatinib-treated pts (TIDEL II) and 315 of 335 pts treated with more potent BCR::ABL1 inhibitors (dasatinib: DIRECT n=76; nilotinib: PINNACLE n=59 and ENESTxtnd n=81; asciminib: ASCEND n=99). Kaplan-Meier and cumulative incidence analyses were performed to assess the effect of AGAs on FFS, kinase domain (KD) mutation acquisition (including ATP-site and myristoyl-site) and molecular response.
 Results
 The incidence of AGAs in the total cohort of 515 pts was 32%: cancer gene variants (CGVs 18%) and Ph-associated rearrangements (Ph-ass 18%). ASXL1 variants were most frequently detected: 41/515 pts (8%).
 We previously demonstrated that AGAs at diagnosis were associated with inferior FFS and molecular response for the imatinib-treated pts. Notably, Ph-associated rearrangements were strongly associated with the acquisition of KD mutations. No differences were observed for overall or transformation-free survival. For pts treated with more potent inhibitors, the negative impact of Ph-ass for FFS, KD mutations and molecular response was overcome. However, CGVs were associated with a significantly higher rate of treatment failure (2-year FFS 76 vs 92%, P<0.001), acquisition of KD mutations at 2 years (11% vs 0.3%, P<0.001), and a lower rate of major molecular response (12-month MMR 63% vs 82%, P=0.002) for pts treated with more potent inhibitors with a median follow up of 24 months. These inferior responses were observed irrespective of whether pts were treated with nilotinib/dasatinib or asciminib. Notably, ASXL1 variants at diagnosis were associated with inferior 12-month MMR (55% vs 82%, P=0.033), 2-year FFS (68% vs 93%, P<0.001) and higher 2-year KD mutation acquisition (27% vs <0.3%, P<0.001) compared to pts without CGVs. Among asciminib-treated pts with ASXL1 variants, the cumulative incidence of KD mutations at 24 months was 37%. Non-ASXL1 variants also predicted for inferior 12-month MMR.
 Irrespective of whether pts with CGVs at diagnosis were treated with frontline imatinib or more potent inhibitors, there was no significant difference in 12-month MMR (53% vs 63%, P=0.34), 24-month FFS (71% vs 76%, P=0.57) or 24-month KD mutation acquisition (12% vs 11%, P=0.95).
 Prognostic factors for pts treated with more potent inhibitors were examined in multivariable models. CGVs at diagnosis were the only predictors of all 3 outcomes: 12-month MMR [HR 0.54 (0.36-0.80), P=0.003], 2-year FFS [HR 3.57 (1.55-8.19), P=0.003] and KD mutations [HR 30.4 (3.75-246.9), P=0.001]. In a separate model, ASXL1 variants also predicted 12-month MMR [HR 0.47 (0.24-0.92), P=0.003], 2-year FFS [HR 1.65 (1.04-2.64), P=0.035] and KD mutations [HR 73.9 (8.8-619), P<0.001]. High risk ELTS also predicted 12-month MMR [HR 0.50 (0.30-0.80), P=0.009]. No difference was observed for 24-month MR4 and overall or transformation-free survival.
 Conclusion
 CGVs at diagnosis were associated with inferior outcomes, even with more potent frontline inhibitor therapy. KD mutations almost exclusively developed in pts with CGVs at diagnosis. However, the negative impact of Ph-associated rearrangements observed for imatinib-treated pts was abrogated by more potent inhibitors. The data suggest a potential beneficial role for combination therapy in this cohort of pts where the risk of treatment failure driven by KD mutations is high.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/034851afd552a463bd3139eba5c41e0140112f68" target='_blank'>
              Strong Association between Cancer Gene Variants at Diagnosis, Especially ASXL1, and Emergence of Kinase Domain Mutation-Driven Resistance in CML Patients Despite Frontline Treatment with More Potent BCR::ABL1 Inhibitors
              </a>
            </td>
          <td>
            N. Shanmuganathan, C. Wadham, David T Yeung, N. Shahrin, A. Fernandes, Muneeza Maqsood, C. Kok, V. Saunders, I. S. Pagani, Rosalie R. Kenyon, Ming Lin, David M. Ross, T. Hughes, S. Branford
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="

 RNA splicing factor SF3B1 is the most recurrently mutated gene in chronic lymphocytic leukemia (CLL), a mature B cell malignancy. Mutations on SF3B1 are associated with poorer clinical outcomes in patients and accelerated onset of CLL in murine models, highlighting an oncogenic role for mutant SF3B1 in leukemogenesis. While typically indolent, CLL can progress to an aggressive state characterized by increased copy number variations (CNVs) and genetic lesions. We previously reported that SF3B1 mutations not only impacted RNA splicing but also contributed to chromosomal instability (Yin et al, Cancer Cell 2019; Cusan et al, JCI 2023). These findings support a model in which SF3B1 mutations can potentiate downstream cytogenetic changes to drive the clonal evolution of CLL. However, to date, little evidence exists to support this model.
 To study this, we first developed a novel methodology called prime editing coupled intron-assisted selection (PRECIS) that pair prime editing with a SF3B1 mutation-responsive intron reporter to engineer SF3B1 mutant cell lines. Our PRECIS workflow leveraged the PE5max system to introduce the K700E hotspot mutation into two CLL cell lines: HG-3 and MEC-1.Through a synthetic intron reporter that fluorescently labeled SF3B1 mutant cells with GFP, we were able to derive pure mutant clones following FACS enrichment and single-cell cloning, achieving cancer-relevant allelic frequencies of 50% and 33% in HG-3 and MEC-1, respectively. Through RNA-seq, we showed that the RNA splicing profiles in our CLL SF3B1 mutant cell lines were consistent with those of SF3B1-mutated primary CLL. We further confirmed additional SF3B1 mutation phenotypes including G1-S phase cell cycle arrest, impacted cellular growth, and loss of CD5 expression.
 To study the effect of SF3B1 mutation on CNVs, we conducted whole genome sequencing (WGS) using genomic DNA derived from these cell lines. When comparing SF3B1 mutant to WT cells, WGS analyses revealed widespread CNV events including both focal and arm level deletions and amplifications. The most significant CNV event that we observed was the loss of the Y chromosome (LOY) in SF3B1 mutant cells. We confirmed our observation using RNA-seq calls and differential gene expression (DEG) analysis that showed global downregulation of Y chromosome genes such as DDX3Y, KDM5D, UTY, etc. Furthermore, leveraging genomic DNA samples retained at each step of the PRECIS engineering process, we profiled and mapped out the clonal evolutionary trajectory of a few SF3B1 mutant clones, identifying the first LOY event - a focal deletion of KDM5D - at 35 days after editing and a complete deletion of the Y chromosome at day 90. Our results support LOY as a general SF3B1 mutation phenotype.
 We next focused on validating the linkage between SF3B1 mutation and LOY in primary CLL patient samples. Using RNA-seq data (n=395) derived from CLLmap (https://cllmap.org/), we created a Y chromosome gene signature score based on the 43 most expressed Y genes to measure LOY in male CLL patients. Using this, we observed increased levels of LOY in SF3B1 mutant samples (n=84) compared to WT samples (n=311). Known tumor suppressors such as DDX3Y and KDM5D were among the most significantly affected genes, suggesting a potential impact of LOY on patient outcomes. To determine this, we utilized the Y signature score to stratify patients into Y+ and Y- groups based on high (top 30%) and low (bottom 30%) Y chromosome expression. Compared to Y+ patients (n=116), Y- patients (n=124) had a shorter overall survival (median survival: 5701 vs 3254 days, p=0.017). Compared to WT samples (n=84), SF3B1 mutation (n=40) further accelerated disease progression in Y- patients (median survival: 3902 vs 2348 days, p=0.042). Moreover, LOY was associated with an overall poorer clinical outcome (HR=1.75, 95% CI: 1.10-2.79, p=0.019, Cox univariate test). These results suggest that SF3B1 mutation not only drives LOY but also synergizes with LOY to accelerate CLL aggressiveness.
 In summary, through our innovative PRECIS gene editing strategy, we were able to study the effect of SF3B1 mutation on CLL biology, confirming previously known phenotypes such as aberrant splicing while uncovering novel genetic events such as LOY. We were able to model for the clonal evolutionary trajectory of SF3B1 mutation, discovering LOY as a novel SF3B1 mutation phenotype and providing evidence for LOY as a contributor to aggressive CLL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65d75579c72b17c80ad027b9050578ffc2945fcc" target='_blank'>
              SF3B1 Mutation Drives Loss of Y Chromosome in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            Mikel Fernandez, Lei Yu, Qiong Jia, Xuesong Wang, Kevyn L Hart, Zhenyu Jia, Ren-Jang Lin, Lili Wang
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1436facb9b04a543d668f4862e82c8d1f590873" target='_blank'>
              FBXO2 as a switch guides a special fate of tumor clones evolving into a highly malignant transcriptional subtype in oral squamous cell carcinoma.
              </a>
            </td>
          <td>
            Jingyi Cheng, O. Liu, Xin Bin, Zhangui Tang
          </td>
          <td>2024-11-02</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Epstein-Barr virus (EBV) infects over 95% of the world’s population and is tightly associated with multiple human malignant diseases. As the first discovered human oncovirus, EBV is known to induce genomic instability by promoting various types of genomic modifications in host chromosomes. However, the mechanisms through which EBV interacts with the host genome and regulates cellular gene expression in genomic modifications are not yet fully elucidated. In this study, we conducted primary EBV infection in B cells and performed the analyses of copy number variants using whole genome sequencing. After comparing to other distinct B cells with varying EBV infection backgrounds, we identified susceptible regions and unveiled critical host factors induced by EBV infection on chromosomes. Combined with the genome-wide transcriptomic analyses, we found that the mucin gene MUC19 is distinctively activated by EBV infection and MUC19 promotes oncogenesis through the activation of mTOR signaling in lymphoblastoid cell lines (LCLs). Further results indicate that EBV nuclear antigen 1 (EBNA1) binds to multiple regions within MUC19 and enhances the expression of MUC19. Moreover, sequence alignment has shown intriguing motif homology between the basic repeat unit of MUC19 and EBV genome, suggesting the potential of EBV-specific integration. To conclude, our study maps the genomic perturbations induced by EBV primary infection and offers new insights into the oncogenic roles of MUC19 during EBV latency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c49f82b38f7e292399b40face9775ffe981ab0a" target='_blank'>
              Genome-wide mapping of EBV-induced genomic variations identifies the role of MUC19 in EBV latency
              </a>
            </td>
          <td>
            Jingwen Yu, Yaohao Wang, Qirong Liu, Xiaohui Zhou, E. Robertson, Yonggang Pei
          </td>
          <td>2024-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Abstract Motivation Knowledge of the specific cell types affected by genetic alterations in rare diseases is crucial for advancing diagnostics and treatments. Despite significant progress, the cell types involved in the majority of rare disease manifestations remain largely unknown. In this study, we integrated scRNA-seq data from non-diseased samples with known genetic disorder genes and phenotypic information to predict the specific cell types disrupted by pathogenic mutations for 482 disease phenotypes. Results We found significant phenotype-cell type associations focusing on differential expression and co-expression mechanisms. Our analysis revealed that 13% of the associations documented in the literature were captured through differential expression, while 42% were elucidated through co-expression analysis, also uncovering potential new associations. These findings underscore the critical role of cellular context in disease manifestation and highlight the potential of single-cell data for the development of cell-aware diagnostics and targeted therapies for rare diseases. Availability and implementation All code generated in this work is available at https://github.com/SergioAlias/sc-coex">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d3112d21a991efd26975ba71315ec68a20ad661" target='_blank'>
              Differential expression and co-expression reveal cell types relevant to genetic disorder phenotypes
              </a>
            </td>
          <td>
            Sergio Alías-Segura, F. Pazos, M. Chagoyen
          </td>
          <td>2024-10-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="

 Introduction
 PIs form the backbone of most first-line regimens for MM, both in transplant-eligible and ineligible patients. Despite advances, MM remains incurable primarily due to the development of drug resistance. Many studies on this phenomenon in PI-resistant cells have been conducted in recent years; nevertheless, we still lack an explanation that would lead to clinically translatable findings. In this study, to achieve better insight into the process of acquiring resistance to bortezomib (BTZ) and carfilzomib (CFZ), we conducted comparative proteomic profiling of sensitive MM cell lines and their resistant counterparts at different stages of acquiring the resistant phenotype. To further enhance the generalizability of these findings, we performed the experiments on two different MM cell lines.
 Methods
 Resistant cell lines were generated by continuously culturing sensitive RPMI8226 and U266B1 MM cells with increasing concentrations of BTZ or CFZ. Resistance was defined by the ability to remain viable in continuous culture with increasing concentrations of the drugs. Comparative proteomic profiling of sensitive cells and cells resistant to increasing concentrations of both drugs was performed as described previously (Kubicki, Am J of Cancer Res 2022). The Western blot (WB) method was used to validate the proteomic results. Proteasome activity was measured by the Proteasome-Glo Assays (Promega, USA). Apoptosis was measured by BH3 profiling as in previous reports (Langedonk, Int J Mol Sci 2022).
 Results
 RPMI8226 cells included in this analysis were able to remain viable in culture with 3, 4, 6 and 13 nM of BTZ and 2, 4, 6 and 23 nM of CFZ. For U266B1 cell lines, the concentrations were 1, 2, 5 and 11 nM of BTZ and 2, 5, 6, 11 and 17 nM of CFZ. Both drugs were able to similarly suppress chymotrypsin-like proteasome activity in both sensitive and resistant RPMI8226 and U266B1 cells. Additionally, the baseline activity of all three proteasome catalytic domains was the same in all cell lines. Subsequently, we assessed the cells' global adaptations through comparative proteomic profiling. The top-ranked canonical pathway was oxidative phosphorylation inhibited in BTZ-resistant (B-H p < 10-21; z-score = 5.7), CFZ-resistant RPMI8226 (B-H p < 10-30; z-score = 5.9), BTZ-resistant (B-H p < 10-2; z-score = 5.0), and CFZ-resistant U266B1 (B-H p < 10-2; z-score = 5.3) cells. Accumulation of almost all proteins involved in this process was linearly downregulated with increasing drugs concentrations. In line with the well-described Warburg effect, cytoplasmic proteins responsible for gluconeogenesis and glycolysis were also consistently upregulated in the resistant U266B1 and RPMI8226 cells, whereas mitochondrial proteins involved in the tricarboxylic acid cycle were downregulated. These finding were further supported by the downregulation of mitochondrial proteins that may contribute to mitochondrial dysfunction in all resistant cell lines. Based on these results, we selected the mitochondrial transcription factor A (TFAM) protein as a potential marker of resistance and confirmed its lower accumulation in both resistant RPMI8226 and U266B1 cells using WB. We also identified significant differences in accumulation of proteins associated with apoptosis in BTZ-resistant RPMI8226 cells (B-H p < 10-29; z-score = -2.9). After priming during BH3 profiling, mitochondrial outer membrane permeabilization was lower in BTZ-resistant cells compared to the sensitive ones. Synergy between bortezomib and different BH3-mimetics was confirmed (5.1 for 1 µM of venetoclax, 2.1 for 1 µM of navitoclax, 2.7 for 0.1 µM of BCL-XL inhibitor, 1.5 for 0.01 µM of MCL1 inhibitor); the combination treatment was capable of partially restoring sensitivity to BTZ. Of note, RPMI8226 cells do not harbor the translocation t(11;14). Intriguingly, we did not observe similar pattern for CFZ-resistant cells and for U266B1.
 Conclusions
 CFZ- and BTZ-resistant RPMI8226 and U266B1 MM cells exhibit a Warburg effect phenotype and signs of mitochondrial dysfunction. We identified TFAM as a potential biomarker of resistance to PIs. Combining BH3-mimetics with BTZ restored sensitivityin the resistant RPMI8226 cells. Further studies are needed to confirm the clinical significance of these results, explore the potential of targeting mitochondrial metabolism and identify patients who may benefit from BH3-mimetics.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ca74b84a51dbc9cbe616a50b45ce548863dd380" target='_blank'>
              Comparative Proteomic Profiling of Multiple Myeloma (MM) Cell Lines Identifies Mitochondrial Dysfunction As a Marker of Resistance to Proteasome Inhibitors (PIs) with Potential Therapeutic Implications for BH3-Mimetics
              </a>
            </td>
          <td>
            T. Kubicki, K. Bednarek, M. Kostrzewska-Poczekaj, Magdalena Luczak, Lydia Visser, Monique Lodewijk, Krzysztof Żyłka, Lidia Gil, M. Jarmuż-Szymczak, Dominik Dytfeld
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96442cd68e41fa673e831da90c37e37f248ca02b" target='_blank'>
              A multiomic atlas identifies a treatment-resistant, bone marrow progenitor-like cell population in T cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Jason Xu, Changya Chen, Jonathan H Sussman, S. Yoshimura, T. Vincent, P. Pölönen, Jianzhong Hu, S. Bandyopadhyay, Omar Elghawy, Wenbao Yu, Joseph S. Tumulty, Chia-Hui Chen, Elizabeth Li, Caroline Diorio, R. Shraim, H. Newman, Lahari Uppuluri, A. Li, Gregory M. Chen, David W Wu, Yang-yang Ding, Jessica Xu, Damjan Karanfilovski, Tristan Lim, Miles Hsu, Anusha Thadi, Kyung Jin Ahn, Chi-Yun Wu, Jacqueline Peng, Yusha Sun, Alice Wang, Rushabh Mehta, David Frank, Lauren Meyer, M. Loh, E. Raetz, Zhiguo Chen, Brent L Wood, M. Devidas, K. Dunsmore, S. Winter, Ti-Cheng Chang, Gang Wu, S. Pounds, Nancy R. Zhang, William L Carroll, S P Hunger, Kathrin M Bernt, Jun J Yang, C. Mullighan, Kai Tan, D. Teachey
          </td>
          <td>2024-11-25</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>103</td>
        </tr>

        <tr id="Gene editing has emerged as a transformative approach in cancer immunotherapy. Several gene editing tools have been employed for precise modification of the DNA of immune cells, enhancing their ability to target and eliminate cancer cells. This review examines the evolution and applications of key gene-editing tools, such as CRISPR-Cas9, TALENs, ZFNs, and recent innovations like base and prime editing in the field of cancer immunotherapy. 
Promising results have been observed in therapies such as CAR-T and tumor-infiltrating lymphocyte (TIL) treatments, which have shown success in cancers like leukemia and lymphoma. These technologies improve immune system function by disrupting checkpoints, boosting cytokine production, and modifying tumor microenvironments. 
Significant clinical trials have demonstrated promising outcomes, such as CRISPR-Cas9-engineered T cells targeting refractory cancers, which showed improved efficacy and safety. However, despite these advancements, there have been limitations, including off-target effects, delivery inefficiencies, immunogenicity, and ethical concerns, alongside the high costs that hinder widespread adoption. 
Future directions for gene editing in cancer therapy include the integration of AI and machine learning to enhance target accuracy and guide RNA design, as well as novel gene-editing systems such as the I-C Cascade-Cas3, which facilitates large-scale genomic deletions. Furthermore, CRISPR-based epigenome editing holds promise for further advancing cancer therapies. These innovations, combined with optimized delivery methods, are expected to improve the precision, efficacy, and accessibility of gene editing in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c53d457951919724a8c5ed4f27b1693452c6fc64" target='_blank'>
              Gene Editing in Cancer Immunotherapy: Mechanisms, Advancements, Limitations and Future Directions
              </a>
            </td>
          <td>
            Lawrence John Ajutor, Okabeonye Sunday Agbo, Christian Ajiri Adobor, Blessing Aitebiereme Iyoyojie, Kingsley Ugonna Ugoagwu, Oghenevware Manawa, Osaruese Faluyi, Emmanuel Oluwasegun Ismaila, Oluchi Esther Ugwuanyi, Adepeju Matilda Adekoya
          </td>
          <td>2024-12-21</td>
          <td>Asian Journal of Biochemistry, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Multiple myeloma (MM) is a complex malignancy marked by the abnormal proliferation of plasma cells (PCs) in the bone marrow (BM). This disease progresses from asymptomatic stages, including monoclonal gammopathy of undetermined significance (MGUS) and smouldering multiple myeloma (SMM), to active MM. The transition through these different states requires the acquisition of genetic aberrations and changes in gene expression patterns, which have been well characterized. Beyond genetics, the role of epigenetic modifications has transformed our understanding of malignant cellular transformation, including MM. However, the complete regulatory mechanisms involving PCs dysregulation across disease progression are yet to be characterized. Understanding the molecular alterations driving the transitions from precursor states to symptomatic MM are essential for early therapeutic interventions.
 In this study, we performed a comprehensive analysis of gene regulatory dysregulation associated with the transitions from healthy to precursor stages and advanced MM. PCs (CD138+) were isolated from human BM samples from healthy control (HC) individuals (n=19) and newly diagnosed MM patients at various stages of disease progression (MGUS (n=16), SMM (n=33) and MM (n=146)), from both genders with a median age of 77 years old. We profiled 23,529 expressed genes (RNA-seq), 138,969 chromatin regions (ATAC-seq), and 94,852 histone mark regions (ChIP-seq).
 Initial joint multi-omics analyses revealed weak coordination between gene expression and chromatin accessibility profiles across disease transitions, prompting us to conduct separate analyses for each disease transition: HC vs. MGUS (Transition 1, T1), MGUS vs. SMM (T2), and SMM vs. MM (T3). We observed that few genes and regions linked gene expression and chromatin accessibility with clinical aberrations from FISH profiles. Differential analysis incorporating clinical aberrations as covariates indicated significant transcriptomic changes primarily at T1, revealing 9,688 differentially expressed genes (DEGs). Only 154 and 489 DEGs were significant for T2 and T3, respectively. In contrast, chromatin accessibility changes were mainly occurring at T2 consisting of 12,166 significantly differential accessible regions, characterized by the opening of chromatin regions. Hence, suggesting that regulatory elements play distinct key roles in disease progression.
 To elucidate the role of regulators in MM progression, we employed a comprehensive ranking strategy to prioritize transcription factors (TFs) critical for each transition, integrating various statistical scorings. Each scoring method addressed, respectively, the role of TFs in transcriptional changes, their association with differentially accessible regions, and TFs activity shifts. The observed chromatin accessibility shift at T2, revealed key players involved during chromatin opening, including FOX family, SMAD family (SMAD3 and SMAD5), interferon-regulatory factors (IRF5, IRF8), zinc-fingers (ZNF214, ZNF547) and IKZF2. The majority of revealed TFs were known to be essential in MM and identified previously as pioneers (Sinha, et al., 2023; Ordoñez et al., 2020; Mayran, et al., 2018), which we plan to validate using CRISPR-Cas9 in MM cell lines. Furthermore, gene regulatory networks (GRNs) analyses explore the interplay between epigenetic and transcriptomic changes, and identify crucial enhancers and promoters involved in myeloma development. Single-cell datasets, both in-house and public, are being used to generalize findings and dissect intra-tumor heterogeneity.
 In conclusion, our study underscores the importance of integrated multi-omics approaches in unraveling the complex regulatory networks driving MM pathogenesis. By identifying key regulatory elements and pathways, we aim to facilitate the development of targeted therapies and improve early screening for MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55a3e8ce81f15144ecedc43a4f92234ac15a3a66" target='_blank'>
              Mapping Disease Transitions from Premalignant, Asymptomatic to Advanced Myeloma through Integrative Epigenomic and Transcriptional Analyses
              </a>
            </td>
          <td>
            Aleksandra Kurowska, Azari Bantan, Raghad Shuwaikan, I. Cenzano, Paula Aguirre-Ruiz, Estíbaliz Miranda, L. Gárate, A. Vilas-Zornoza, Patxi San-Martin, Marta Larrayoz, Nahia Gómez-Echarte, B. Ariceta, Daniel Soto Ortiz, Turki Sobahy, J. Martín-Subero, Vincenzo Lagani, Jesper N. Tegnér, Jose-Al Martinez-Climent, X. Agirre, Mikel Hernáez Arrazola, Felipe Prósper, D. Gómez-Cabrero
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is a potent target for cancer therapy because it acts as a signaling hub, engaging in various signaling pathways and regulating a number of cellular functions like cell differentiation, proliferation, and survival. Recently, an emergency approval from the US-FDA has been issued for KRASG12C inhibitors (sotorasib and adagrasib) for metastatic lung cancer treatment. However, clinical studies on covalent KRASG12C inhibitors have rapidly confronted resistance in patients. Many methods are being assessed to overcome this resistance, along with various combinatorial clinical studies that are in process. Moreover, because KRASG12D and KRASG12V are more common than KRASG12C, focus must be placed on the therapeutic strategies for this type of patient, along with sustained efforts in research on these targets. In the present review, we try to focus on various strategies to overcome rapid resistance through the use of combinational treatments to improve the activity of KRASG12C inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a7d869ac2287e7b6ce7ff416a2192b4c748daa" target='_blank'>
              KRAS inhibitors in drug resistance and potential for combination therapy.
              </a>
            </td>
          <td>
            Lala S Rathod, N. Sakle, S. Mokale
          </td>
          <td>2024-11-06</td>
          <td>Tumori</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51f438a90e0c706269021fae3335ddefc70e9107" target='_blank'>
              YY1 downregulation underlies therapeutic response to molecular targeted agents
              </a>
            </td>
          <td>
            Shichao Zhou, J. Zang, Mei-Chun Cai, Kaiyan Ye, Jin Liu, Pengfei Ma, Jie Wu, Chenyang Dai, Haijiao Lu, Qing Zhang, Junhong Jiang, Tianqing Chu, Ying Shen, Li Tan, Guanglei Zhuang, Xiaojing Zhao, Lan Wang, Zhuang Yu, Yujie Fu
          </td>
          <td>2024-11-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="RNA-based genetic therapies have emerged as a promising approach in the treatment of rare genetic diseases, offering a novel alternative to traditional therapeutic strategies. This study reviews the current literature on the effectiveness of RNA-based therapies in treating genetic disorders, focusing on advancements in mRNA, siRNA, and antisense oligonucleotides (ASOs) therapies. A qualitative, literature-based methodology was employed to analyze recent research articles, clinical trials, and case studies from the past five years. Key findings highlight the potential of RNA-based therapies to target the root causes of genetic diseases at the molecular level, such as correcting mutations, silencing defective genes, or replacing missing proteins. RNA therapies have shown particular promise in treating rare diseases like Duchenne muscular dystrophy, spinal muscular atrophy, and certain forms of inherited retinal diseases, with several therapies already reaching clinical approval. However, challenges such as delivery mechanisms, immune response, and long-term efficacy remain significant barriers to widespread adoption. This review emphasizes the need for further research to refine RNA delivery systems, assess long-term safety, and optimize therapeutic outcomes. The study concludes by suggesting that RNA-based therapies could revolutionize the treatment of rare genetic diseases, provided these challenges are addressed through continued innovation and clinical validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4318feb40e0dfdfce825e0cca992e00b2fd570d3" target='_blank'>
              Effectiveness of RNA-Based Genetic Therapy in Treating Rare Diseases of a Genetic Nature
              </a>
            </td>
          <td>
            Suherman Jaksa, Anna Maria Nurhajati Widjaja, Neng Mira Atjo
          </td>
          <td>2024-12-05</td>
          <td>The Journal of Academic Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Introduction
 Myeloproliferative neoplasms (MPN) are the consequence of acquired mutations in hematopoietic stem and progenitor cells (HSPCs). Specific mutations and mutational burden determine a patient's risk of disease progression to lethal post-MPN AML.
 Recent technological advancements linking single-cell gene expression and genotype have facilitated the deconvolution of tumor heterogeneity. However, these pipelines are limited in their ability to genotype low-expressed transcripts (e.g., JAK2) and multiple pathogenic loci. To overcome this, we developed a novel single-cell LOng-read genotyping of TRanscripts pipeline called LOTRSeq to explore cancer heterogeneity through target enrichment and long-read sequencing.
 Methods
 LOTRSeq genotypes 10X 3' single-cell RNA sequencing (scRNASeq) barcoded transcripts by a biotin-streptavidin probe-based pull down of 30 genes frequently mutated in MPN/AML, followed by Oxford Nanopore Technology (ONT) long-read sequencing. We performed scRNASeq of 50,439 CD34+ HSPCs of n=8 patients with JAK2V617F-driven MPN (n=4 PV, n=1 ET, n=3 MF) and n=4 post-MPN AML (2 paired samples) followed by LOTRSeq. To assess transcriptional differences between mutated and wildtype HSPCs of MPN/post-MPN AML patients compared to healthy HSPCs, we integrated CD34+ HSPCs scRNASeq data of healthy subjects (n=6, Ainciburu et al. 2023).
 Results
 We utilized a pool of cDNA generated from 4 AML cell lines with 8 different 10X barcodes and ratios to validate the LOTRSeq pipeline for the deconvolution of similar barcodes and genotyping of lowly expressed genes. Target enrichment achieved up to 100X enrichment of transcripts over whole transcriptome sequencing. Read proportions assigned to deconvoluted barcodes highly correlated with cell mix proportions (R=0.93, p=0.0007). We achieved 95% sensitivity to call JAK2V617F and 80% sensitivity to call TP53M133K, with 84% and 78% accuracy, respectively. In MPN and post-MPN AML samples, expressed variant allele frequencies (VAF) of pseudo bulked long-reads highly correlated with genomic VAF (n=6, R=0.9, p=0.01). In addition, 60-80% of cells were genotyped, with genotyping % strongly correlating with Nanopore sequencing output (R=0.85, p=0.0002).
 Single-cell analysis and clustering of CD34+ HSPCs agnostic to genotype reveals that cells cluster according to a deterministic progenitor phenotype. The HSPCs of JAK2V617F-driven MPN largely maintain transcriptional heterogeneity similar to healthy but with expanded erythroid primed cluster (p=0.0047) that correlates with the percentage of JAK2V617F cells (R=0.73, p=0.039). Consistent with previous publications (Nam et al. 2019, van Egeren et al. 2021), mutant and wildtype cells did not cluster apart. However, JAK2V617F cells compared to wildtype and healthy, showed a higher probability of converging towards an erythroid cell fate.
 Sequential analysis of a patient's bone marrow with primary MF and post-MPN AML showed linear clonal evolution of a JAK2V617F and IDH1R132S mutant clone at leukemic transformation, leading to the expansion of a transcriptional HSC-like cluster. Another paired ET and post-MPN AML with a 2-year transformation period showed branching evolution where we identified at diagnosis 27% JAK2V617F cells and 22% TP53Y236C in independent clones and 8% TP53R282W within the TP53Y236C clone. Upon leukemic transformation, the JAK2V617F clone was detected only in 0.6% of cells, outcompeted by a clone with biallelic loss of TP53 through missense mutations Y236C and R282W. This clonal population overlapped transcriptionally with Granulocyte-Megakaryocyte-Progenitor like cells.
 Conclusion
 We have developed a novel scRNASeq pipeline to concurrently genotype a panel of 30 MPN/AML driver genes and examine the transcriptional impact of genetic heterogeneity. We observe a progressive loss of transcriptional heterogeneity with disease progression associated with acquiring additional gene mutations and clonal evolution. Discrete gene mutations are associated with specific effects on transcriptional identity. This knowledge will be invaluable in understanding and monitoring the impact of mutation acquisition and clonal evolution during MPN disease progression and response to therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c97c5f1c1dc16e43acdc1eab98c9dc41b372d18" target='_blank'>
              Single Cell Genotype-Phenotype Analysis Demonstrates That Post-MPN AML Is Driven By Clonal Evolution and Loss of Transcriptional Heterogeneity within Stem Cell Compartments
              </a>
            </td>
          <td>
            Jasmin Straube, Julian Grabek, Leanne T. Cooper, Rohit N. Haldar, Caroline McNamara, David M Ross, Andrew Perkins, Megan J Bywater, Steven W Lane
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chemotherapy is often combined with surgery for muscle invasive and nonmuscle invasive bladder cancer (BCa). However, 70% of the patients recur within 5 years. Metabolic reprogramming is an emerging hallmark in cancer chemoresistance. Here, we report a gemcitabine resistance mechanism that promotes cancer reprogramming via the metabolic enzyme OXCT1. This mitochondrial enzyme, responsible for the rate-limiting step in β-hydroxybutyrate (βHB) catabolism, was elevated in muscle invasive disease and in patients with chemoresistant BCa. Resistant orthotopic tumors presented an OXCT1-dependent rise in mitochondrial oxygen consumption rate, ATP, and nucleotide biosynthesis. In resistant BCa, knocking out OXCT1 restored gemcitabine sensitivity, and administering the nonmetabolizable βHB enantiomer (S-βHB) only partially restored gemcitabine sensitivity. Suggesting an extrametabolic role for OXCT1, multi-omics analysis of gemcitabine sensitive and resistant cells revealed an OXCT1-dependent signature with the transcriptional repressor OVOL1 as a master regulator of epithelial differentiation. The elevation of OVOL1 target genes was associated with its cytoplasmic translocation and poor prognosis in a cohort of patients with BCa who have been treated with chemotherapy. The KO of OXCT1 restored OVOL1 transcriptional repressive activity by its nuclear translocation. Orthotopic mouse models of BCa supported OXCT1 as a mediator of gemcitabine sensitivity through ketone metabolism and regulating cancer stem cell differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a605291aabc7183d7c2295711f5e8ae8fbb3560" target='_blank'>
              Targeting ketone body metabolism in mitigating gemcitabine resistance
              </a>
            </td>
          <td>
            Krizia Rohena-Rivera, Sungyong You, Minhyung Kim, Sandrine Billet, Johanna ten Hoeve, Gabrielle Gonzales, Chengqun Huang, Ashley Heard, Keith Syson Chan, Neil A. Bhowmick
          </td>
          <td>2024-11-07</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557911aa9625d7bd95b52abd71f3273a6c915b9d" target='_blank'>
              Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in cancer.
              </a>
            </td>
          <td>
            Yang Yang, Michelle L Badura, Patrick C O'Leary, Henry Delavan, Troy M Robinson, Emily A Egusa, Xiaoming Zhong, Jason Swinderman, Haolong Li, Meng Zhang, Minkyu Kim, Alan Ashworth, Felix Y. Feng, Jonathan Chou, Lixing Yang
          </td>
          <td>2024-11-18</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="

 Background: Ibrutinib, as a first-line treatment for Waldenstrom macroglobulinemia (WM), has demonstrated a good safety profile and efficacy, particularly in MYD88mut patients. However, clinical responses are variable and resistance to ibrutinib remains a significant clinical challenge.
 Methods: To characterize the molecular correlates of response and resistance to ibrutinib, we performed high-throughput droplet-based 5′-single-cell RNA sequencing (scRNA-seq) paired with B cell receptor sequencing (scBCR-seq) on 75 longitudinal bone marrow samples collected at multiple timepoints from WM patients treated with ibrutinib monotherapy, with a median follow-up of 50 months (NCT02604511 clinical trial). Whole-exome sequencing (WES) was performed on 34 samples collected at baseline and post-treatment in 17 patients. Bulk RNA-seq was performed on 51 samples collected at baseline patients treated with ibrutinib monotherapy.
 Results: To study the impact of ibrutinib on tumor clonal dynamics, we identified malignant cells leveraging our scBCR-seq data, inferred somatic copy number variants (CNVs) in tumor cells and constructed phylogenetic trees for each tumor separately. We then traced the frequency of each clone across timepoints and characterized evolutionary patterns in response to ibrutinib. Clonal evolution, whereby daughter subclones increased in abundance post-treatment, was observed exclusively in patients who progressed on treatment and was associated with significantly shorter progression-free survival (PFS). Remarkably, in some cases, clonal evolution was observed several months before clinical progression was noted, suggesting that genomic surveillance could help diagnose progression long before the clinical criteria are met. Notably, several patients who responded to ibrutinib demonstrated clonal de-evolution, whereby the frequency of more genomically evolved daughter subclones decreased in abundance post-treatment resulting in less complex clonal architecture. Clonal de-evolution may help explain a particular clinical phenomenon in WM, whereby patients may experience long PFS despite significant residual disease post-therapy, and indicate that ibrutinib can help prune genomically mature subclones while sparing the root clone. Both clonal evolution and de-evolution patterns were validated by WES (n=18). Notably, clonal evolution or de-evolution was not associated with a particular CNV nor with CXCR4 mutation status, suggesting that it may be non-genetic cellular states that predict whether a subclone will dominate post-treatment (i.e., clonal evolution) or recede (i.e., clonal de-evolution). To understand whether it is possible to predict these patterns prior to administering treatment, we systematically compared the transcriptional profiles of “winner” and “loser” clones within patients who exhibited clonal evolution and derived an evolution signature out of genes that were recurrently up- and down-regulated in “winner” clones. We then used bulk RNA-seq to score a larger cohort of patients for our evolution signature (n=51) and observed that patients with higher scores had significantly shorter PFS compared to patients with lower scores. Interestingly, the top upregulated gene in “winner” clones was the BCR signaling component LYN. To verify the correlation of high LYN levels with ibrutinib resistance, we knocked down LYN in BCWM.1 and MWCL-1 cells and observed significant sensitization to ibrutinib compared to LYN-WT cells. Collectively, these results suggest that it may be possible to predict how tumors will respond to ibrutinib based on their transcriptomic status at baseline, and unveil novel targets that could be therapeutically relevant in WM.
 Conclusion: We demonstrated that genomic evolution underlies and precedes clinical progression by several months in patients with WM receiving ibrutinib, supporting that active genomic surveillance may help improve monitoring of response and early identification of resistant clone emergence. Furthermore, we showed that ibrutinib may genomically down-grade WM tumors, and adopt a novel transcriptional signature that can help predict a tumor's response to ibrutinib. Lastly, we identified novel therapeutic targets, including LYN, that could help circumvent ibrutinib resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6d3d3d0d704d7c1e1ed5a7f7f4c1edebff04c7e" target='_blank'>
              Genomic Evolution By Serial Single-Cell Sequencing Identifies High Tumor Expression of LYN As a Promoter of Resistance in Waldenstrom's Macroglobulinemia Undergoing Ibrutinib Monotherapy
              </a>
            </td>
          <td>
            Hao Sun, Romanos Sklavenitis-Pistofidis, Xia Liu, N. Tsakmaklis, M. L. Guerrera, A. Kofides, Andres Felipe Ramirez Gamero, C. Flynn, C. Patterson, Mu Hao, Lugui Qiu, S. Sarosiek, J. Castillo, Z. Hunter, Shirong Liu, S. Treon
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Population-scale resources of genetic, molecular, and cellular information form the basis for understanding human genomes, charting the heritable basis of disease, and tracing the effects of mutations. Pooled perturbation assays applied to cellular models, probing the effect of many perturbations coupled with an scRNA-seq readout (Perturb-seq), are especially potent references for interpreting disease-linked mutations or gene expression changes. However, the utility of existing maps has been limited by the comprehensiveness of perturbations possible, and the relevance of their cell line context. Here, we present the first genome-scale CRISPR interference (CRISPRi) perturbation map with single-cell RNA sequencing readout across many human genetic backgrounds in human pluripotent cells. To do so, we establish large-scale CRISPRi screening in human induced pluripotent stem cells from healthy donors, using over 20,000 guide RNAs to target 7,226 genes across 34 cell lines from 26 genetic backgrounds, and gather expression data from nearly 2 million cells. We comprehensively map trans expression changes induced by the target knockdowns, which complement co-expression patterns in unperturbed cells and facilitate the functional annotation of target genes to biological processes and complexes. Consistency of targeting protein complex members point to protein complexes as a nexus for aggregating transcriptional variation, revealing novel interaction partners. We characterise variation in perturbation effects across donors, with expression quantitative trait loci linked to higher genetic modulation of perturbation effects but overall low replication of trans effects due to knockdown of their corresponding cis regulators. This study pioneers population-scale CRISPR perturbations with single cell readouts that will fuel foundation models for the future of effective modulation of cellular disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5800ff9ee76e49e133d6ead67bfba5c5d87d6239" target='_blank'>
              A genome-scale single cell CRISPRi map of trans gene regulation across human pluripotent stem cell lines
              </a>
            </td>
          <td>
            Claudia Feng, Elin Madli Peets, Yan Zhou, Luca Crepaldi, S. Usluer, Alistair S. Dunham, Jana M. Braunger, Jing Su, Magdalena E Strauss, Daniele Muraro, Kimberly Ai Xian Cheam, M. Bonder, Edgar Garriga Nogales, Sarah Cooper, Andrew Bassett, Steven Leonard, Yong Gu, Bo Fussing, David Burke, Leopold Parts, O. Stegle, B. Velten
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="This review explores articles concerning the experimental research cycle on genome instability in cell populations of highly malignant recurrent organotropic rhabdomyosarcoma RA- 2 in rats. Clonal analysis and cloning were pivotal components of this research, which relies on the frequency of cells with micronuclei and internuclear bridges to gauge the intensity of chromothripsis and break-fusion-bridge cycles. The efficacy of cloning, determined by these indicators, stemmed from the deliberate isolation of tumor stem cells, yielding clones in which chromothripsis activity and breakage-fusion-bridge cycles were sustained. Notably, it is plausible that the stem cells themselves, progenitors of these clones, harbor dicentric chromosomes and chromosomal fragments, contributing to the formation of "fatal micronuclei" in their karyotype. Cloning based on bridges and micronuclei has proven effective up to a certain threshold (15%-18%), reaffirming the predicted reproductive extinction of malignant cell populations under mutational pressure and genome chaos, as posited by the genetic theory of cell populations. Furthermore, this review highlights the potential of ergodic cancer therapy as a novel therapeutic strategy. Ergodic therapy offers promising prospects for late-stage and terminal malignant tumors, where conventional treatments may fall short due to advanced progression. Furthermore, by "enhancing chromothripsis" through the induction of additional micronuclei and bridges, ergodic cancer therapy seeks to increase genome chaos to a critical threshold, potentially halting malignant progression. This innovative approach presents opportunities to explore new drugs and targets for chromothripsis-based oncotherapy, addressing the pressing need for effective treatments in advanced stages of malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ee83cc51235034563b6307f2bcb528ce64b269" target='_blank'>
              Perspectives on Ergodic Cancer Therapy Derived from Cloning Genome Chaos via In Vivo Rhabdomyosarcoma RA-2 Models: a Narrative Review.
              </a>
            </td>
          <td>
            Sergey Shityakov, Natalia Lubinets, Viacheslav Kravtsov
          </td>
          <td>2024-11-08</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0297441e9d4a24ced84d1909df04d7019094986" target='_blank'>
              Functional screening reveals genetic dependencies and diverging cell cycle control in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Daniel J. Merk, F. Tsiami, S. Hirsch, B. Walter, L. Haeusser, Jens D. Maile, Aaron Stahl, M. Jarboui, Anna Lechado-Terradas, Franziska Klose, S. Babaei, Jakob Admard, N. Casadei, C. Roggia, Michael Spohn, J. Schittenhelm, Stephan Singer, U. Schüller, F. Piccioni, N. Persky, Manfred Claassen, M. Tatagiba, Philipp J. Kahle, D. Root, M. Templin, Ghazaleh Tabatabai
          </td>
          <td>2024-12-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="PURPOSE
PD-1 inhibition is effective in patients with mismatch repair deficient (dMMR) solid tumors in a tumor-agnostic fashion. However, dMMR testing by immunohistochemistry (IHC) is not routinely performed across tumor types. By contrast, next-generation sequencing (NGS) for somatic genomic alterations is frequently performed across tumor types. We hypothesized that NGS would identify patients with alterations in mismatch repair (MMR) genes and that these patients would have higher rates of MMR protein loss by IHC. This would support the utility of IHC reflex testing after NGS and potential matching to approved therapeutic options.


METHODS
From January 2016 to December 2021, 15,701 patients with solid tumors received NGS covering the MMR genes, and 4,994 patients had both IHC and NGS. Sequencing results were analyzed for mutations in MMR genes, tumor type distribution, and concordance with IHC results when available.


RESULTS
Six hundred and ninety-eight (4.4%) of 15,701 patients had mutations in one of the MMR genes. Mutations were found across tumor types. Three hundred and seventeen (6.3%) of 4,994 patients displayed IHC loss for at least one MMR protein. 33.8% patients (110/325) patients with MMR mutations had dMMR, compared with just 4.4% (207/4,669) patients without mutations (P < .001); dMMR rate varied by mutation type.


CONCLUSION
Mutations in MMR genes are found in multiple tumor types where IHC testing is not routine. Reflex IHC testing of patients carrying MMR gene mutations, especially those known or inferred to be inactivating, may identify more patients with dMMR and matched treatment options. However, dedicated IHC screening is needed to capture majority of the patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/680a3a0605d43a4fa80b5c34b31d298af3f6fbad" target='_blank'>
              Genomic Alterations in DNA Mismatch Repair Genes Across Different Cancer Types.
              </a>
            </td>
          <td>
            Vijaykumar R Holla, Michael P Kahle, Sun-Hee Kim, Arash Ronaghy, Richard K Yang, Keyur P Patel, M. Routbort, M. Overman, Ecaterina E. Dumbrava, K. R. Mills Shaw, Daniel D Karp, Funda Meric-Bernstam
          </td>
          <td>2024-11-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="

 Intra-tumor heterogeneity (ITH) describes the coexistence of multiple genetically distinct subclones within the tumor of a patient resulting from somatic evolution, clonal diversification, and selection. It is a main causal driver of therapy resistance and treatment failure in the clinic by already containing subclones that are resistant to therapy or by subclones acquiring resistance to therapy. Core-binding factor (CBF) acute myeloid leukemia (AML) is cytogenetically defined by a translocation of chromosome 8 and 21 [t(8;21)(q22;q22)] or an inversion/translocation of chromosome 16 [inv(16)(p13.1q22) or t(16;16)(p13.1q22)] resulting in RUNX1::RUNX1T1 and CBFB::MYH11 fusion genes, respectively. Despite being classified as AML with favorable prognosis up to 45% of patients relapse and, therefore, understanding the history of somatic events of this heterogeneous AML subtype can contribute to improve clinical outcome.
 To unravel ITH and clonal evolution for both small-scale (single nucleotide variants, insertions and deletions) and large-scale (somatic copy-number alterations (SCNAs), gene fusions) genetic alterations in CBF AML, we analyzed longitudinal samples from 9 patients with t(8;21) or inv(16) AML by whole exome, error-corrected targeted and nanopore long-read bulk sequencing (24 samples). Single-cell DNA sequencing (scDNAseq) of longitudinal samples was performed via the MissionBio Tapestri® platform using custom targeted sequencing panels including information on patient-specific somatic variants, SCNAs and CBF fusions. Samples were collected at the time of diagnosis (D), complete remission (CR), and relapse (Rel). We used a novel approach for reconstructing the order of somatic events in our patients that allowed us to detect SCNAs in phylogenetic trees inferred from somatic variants and CBF fusions.
 In total, we identified 399 variants via bulk sequencing. In the diagnosis samples (n=9) 230 variants were detected and 169 variants in the relapse samples (n=8). Additionally, 7 SCNAs in 5 patients and patient-specific gene fusion breakpoints for all patients were identified using bulk and Nanopore sequencing. We were able to detect patient specific CBF fusions in single cells for 8/9 patients and for those 8 patients we detected between 61% and 96% (mean: 86%) of the variants identified via bulk sequencing. Additionally, we identified 10 variants by scDNAseq in the regions covered by the custom amplicon panels, that have not been detected in bulk sequencing, including FLT3-ITD.
 In total, we inferred tumor phylogenies for 8 patients with detectable gene fusion at D and inferred clonal evolution under chemotherapy pressure for 3 patients. We were able to detect SCNAs by applying our method in small tumor subclones that were not detectable with bulk data nor using existing single-cell tools. By reconstructing the order of somatic evolutionary events from scDNAseq, we can demonstrate that the CBF gene fusions were the first events to occur in disease development in our patients. For patients with samples at D and Rel (n=3) we observed a persisting founding clone throughout the course of the disease. We identified remaining tumor cells in available CR samples (n=6) ranging from 4 to 34 cells (0.16%-1.54%) that harbor at least one variant or the CBF fusion. Among the 148 cells with any detectable variant/fusion, only 5 cells would have been detected when only investigating the fusion. Thus, simultaneous tracking of multiple genotypes showed a clear superiority compared to current strategies, which focus on fusion detection for MRD purposes. By assigning those cells to tumor clones inferred from samples at D and Rel, we can demonstrate that the AML clones were not completely eradicated during treatment.
 In summary, our study presents a detailed picture of ITH and clonal evolution including somatic variants, gene fusions and SCNAs at single-cell resolution of 9 CBF AML patients. We reconstructed the mutational order of somatic events within one tumor sample including acquisition of CBF gene fusions and SCNAs, which adds a new layer into clonal evolution of CBF AML. Moreover, we were able to detect and assign remaining tumor cells at CR to inferred tumor clones of each patient. This highlights the importance of improving the understanding of tumor development and, in more detail, the necessity of identifying early events during tumorigenesis in CBF AML patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27c14f92afccda3056b80626f5b41472934ea1b" target='_blank'>
              Intra-Tumor Heterogeneity in Core-Binding Factor Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Raphael Hablesreiter, Paulina Strzelecka, Natalia Estrada, Klara Kopp, A. Dolnik, Marlon Tilgner, Coral Fustero, Felicitas Thol, F. Heidel, M. Heuser, Lars Bullinger, F. Christen, Frederik Damm
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e013dd21a317acebc6bbd04a30655def3c28bb68" target='_blank'>
              Decorin as a key marker of desmoplastic cancer-associated fibroblasts mediating first-line immune checkpoint blockade resistance in metastatic gastric cancer.
              </a>
            </td>
          <td>
            Ki Tae Kim, Min Hee Lee, Su-Jin Shin, In Cho, Jung Cheol Kuk, Jina Yun, Yoon Young Choi
          </td>
          <td>2024-11-09</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Over 90% of treatment failures in cancer therapy can be attributed to multidrug resistance (MDR), which can develop intracellularly or through various routes. Numerous pathways contribute to treatment resistance in cancer, but one of the most significant pathways is intracellular drug efflux and reduced drug concentrations within cells, which are controlled by overexpressed drug efflux pumps. As a member of the family of ABC transporter proteins, ABCC5 (ATP Binding Cassette Subfamily C Member 5) reduces the intracellular concentration of a drug and its subsequent effectiveness using an ATP-dependent method to pump the drug out of the cell. Numerous studies have demonstrated that ABCC5 is strongly linked to both poor prognosis and poor treatment response. In addition, elevated ABCC5 expression is noted in a wide variety of malignancies. Given that ABCC5 is regulated by several pathways in a broad range of cancer types, it is a prospective target for cancer treatment. This review examined the expression, structure, function, and role of ABCC5 in various cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0764c7131feb3889bb64e294bca3ca7ccb4b5c4b" target='_blank'>
              Role of ABCC5 in cancer drug resistance and its potential as a therapeutic target
              </a>
            </td>
          <td>
            Yinlong Pan, Mengmeng Wu, Huazhong Cai
          </td>
          <td>2024-11-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5aeebfc014f88f36ffbdee87dcef5097f55ec3d" target='_blank'>
              Development of a novel treatment based on PKMYT1 inhibition for cisplatin-resistant bladder cancer with miR-424-5p-dependent cyclin E1 amplification
              </a>
            </td>
          <td>
            Wataru Fukumoto, S. Okamura, M. Tamai, Junya Arima, Ichiro Kawahara, Ikumi Fukuda, Akihiko Mitsuke, Takashi Sakaguchi, S. Sugita, R. Matsushita, S. Tatarano, Yasutoshi Yamada, M. Nakagawa, H. Enokida, H. Yoshino
          </td>
          <td>2024-10-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="
 Metabolic reprogramming is a hallmark of cancer, including diffuse gliomas. IDH1 and IDH2 are key enzymes linking cellular metabolism to epigenetic regulation and redox states. IDH1/IDH2 enzymes catalyze the conversion of isocitrate to α-ketoglutarate, while the mutant enzymes increase the production of D-2-hydroxyglutarate, an oncometabolite that induces epigenetic changes, leading to gene expression alterations. Understanding how epigenetic alterations affect metabolic gene expression and metabolite levels in diffuse gliomas could help identify diagnostic biomarkers and therapeutic targets. We studied the expression of metabolism-related genes, metabolites, and genome-wide methylation status, which will facilitate an integral understanding of the metabolic changes in IDH-mutant and IDH-wildtype gliomas. Study design included 47 fresh frozen tissue samples. RNA was extracted and used for gene expression analysis using the NanoString platform (nCounter® Metabolic Pathways Panel), while extracted DNA was used for targeted sequencing with a panel interrogating 600 genes and for genome-wide methylation analysis using the Illumina Epic v2. In addition, unbiased metabolomic analysis by mass spectrometry was performed in 25 tissue and CSF samples out of 47 samples. Methylation status, gene expression and metabolite levels were compared between IDH-mutant and IDH-wildtype gliomas. Results showed significant differences in DNA methylation, gene expression and metabolites between IDH-mutant and IDH-wildtype gliomas. UPP1, an enzyme involved in uracil metabolism, was studied in depth to understand the interplay between DNA methylation, gene expression, and metabolic reprogramming in diffuse glioma. Epigenetic analysis revealed that the CpG sites within the UPP1 gene displayed reduced methylation in IDH wild-type compared to IDH-mutant tumors. IDH wild-type tumors showed higher UPP1 expression compared to their IDH-mutant counterparts. Consequently, we observed higher levels of uracil in tissue and CSF samples from patients with IDH wild-type gliomas. These data provide a framework to study the relationship between mutations, methylation, gene expression and metabolism in diffuse gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a660a3dfbb1329ab34475bab6b46d65e8f2ffd0" target='_blank'>
              PATH-52. DIFFERENCES IN GENE EXPRESSION, GENOME-WIDE METHYLATION AND METABOLITES IN ADULT DIFFUSE GLIOMAS
              </a>
            </td>
          <td>
            Anjali Vinocha, Valerie Berdecia, Daniel H Wang, Srividya Arjuna, Mauli Shah, A. Dono, N. Putluri, Y. Esquenazi, L. Ballester
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The duplication of genes has long been recognized as a substrate for evolutionary novelty and adaptation, but the factors that govern fixation of paralogs soon after duplication are only partially understood. Duplication often leads to an increase in gene dosage, or the amount of functional gene product. For genes with which an increased dosage is harmful (i.e., triplosensitive genes), a dosage balancing mechanism needs to be present immediately after duplication if it is to evade negative selection. Previous research in vertebrates has demonstrated a potential role for epigenetic factors in allowing triplosensitive genes to increase in copy number by regulating their expression post-duplication. Here we expand this research by investigating the epigenetic landscape of duplicate genes in D. discoideum, a basal lineage separated from humans by over a billion years. We found that activating histone modifications are quickly lost in duplicate genes before gradually increasing in enrichment as paralogs age. For the repressive modification H3K9me3, we found it was enriched in the youngest paralogs, and that this enrichment was likely mediated by heterochromatin spread from transposable elements. We similarly found enrichment of H3K9me3 in young human duplicates, and again found transposable elements as a potential mediator. Finally, we leveraged recent genome-wide estimates of triplosensitivity in human genes to directly examine the relationship between this kind of dosage sensitivity and enrichment for repressive histone modifications. Interestingly, while we found no significant link between enrichment for the repressive mark H3K9me3 and triplosensitivity in human paralogs, we did find a significant association between triplosensitivity and transposon proximity. Our findings suggest that transposons may contribute to the epigenetic regulatory environment associated with dosage balancing of young duplicates in both protists and humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca540ba5e86e5136db5ba3fee49629837eb4d851" target='_blank'>
              The role of histone modifications and transposable elements in the epigenetic regulation of gene dosage after gene duplication
              </a>
            </td>
          <td>
            Thomas D Wolfe, Frédéric J. J. Chain
          </td>
          <td>2024-11-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Metabolic reprogramming from oxidative respiration to glycolysis is generally considered to be advantageous for tumor initiation and progression. However, we found that breast cancer cells forced to perform glycolysis acquired a vulnerability to PARP inhibitors. Small-molecule inhibition of mitochondrial respiration-using glyceollin I, metformin, or phenformin-induced overproduction of the oncometabolite lactate, which acidified the extracellular milieu and repressed the expression of homologous recombination (HR)-associated DNA repair genes. These serial events created so-called "BRCAness," in which cells exhibit an HR deficiency phenotype despite lacking germline mutations in HR genes such as BRCA1 and BRCA2, and, thus, sensitized the cancer cells to clinically available poly(ADP-ribose) polymerase inhibitors. The increase in lactate repressed HR-associated gene expression by decreasing histone acetylation. These effects were selective to breast cancer cells; normal epithelial cells retained HR proficiency and cell viability. These mechanistic insights into the BRCAness-prone properties of breast cancer cells support the therapeutic utility and cancer cell-specific potential of mitochondria-targeting drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d552f07898d1f01f3a3d3064ba56a79d2a0073d0" target='_blank'>
              Metabolically inducing defects in DNA repair sensitizes BRCA-wild-type cancer cells to replication stress.
              </a>
            </td>
          <td>
            Kenji Watanabe, Tatsuro Yamamoto, Tomoko Fujita, Shinjiro Hino, Yuko Hino, K. Yamazaki, Y. Ohashi, Shun Sakuraba, H. Kono, M. Nakao, Koji Ochiai, Shingo Dan, Noriko Saitoh
          </td>
          <td>2024-11-12</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="

 Metabolic reprogramming is a hallmark of tumors, contributing to tumor initiation, progression and drug resistance. In acute myeloid leukemia, cells usually depend on both enhanced glycolysis and oxidative phosphorylation for their energy needs. The persistence of therapy resistance in some AML patients underscores the urgency of identifying novel metabolic targets. Here, we revealed the vital role of citrate transporter SLC25A1 in maintaining AML cell survival and regulating its drug sensitivity, establishing it as a promising metabolic target for AML therapy.
 We first showed that the SLC25A1 expressions were elevated in AML patients both on mRNA and protein level, which were associated with poorer outcomes. Genetic and pharmacological inhibition of SLC25A1 significantly suppressed the proliferation and clonogenicity, promoted apoptosis of leukemic cell lines, as well as the primary AML cells from patients. The specific inhibitor of SLC25A1 had no effect on the proliferation of mononuclear cells (MNCs) from healthy bone marrow at the same concentrations. Our data indicated that SLC25A1 was a potential target for AML therapy, which was further confirmed in AML mice carrying with AE9a or MLL-AF9.
 Functionally, inhibition of SLC25A1 revealed the mitochondrial damage and ROS increasing in AML cells by transmission electron microscopy (TEM), mitochondrial membrane potential and extracellular flux analysis, as well as flow cytometry and fluorescence microscopy. The restoration of cell viability by ROS scavenger N-acetyl-L-cysteine (NAC) in the context of SLC25A1 inhibition suggested that SLC25A1 deficiency inhibited cell survival through the upregulation of ROS. Given this, we found that targeting SLC25A1 could enhance the sensitivities of AML cells to two clinically available anti-leukemic drugs, BCL-2 inhibitor venetoclax and chemotherapy drug daunorubicin (DNR).
 Mechanically, we found that SLC25A1 was closely correlated with ROS metabolic process, extrinsic apoptotic signaling pathway and oxidative phosphorylation, as well as nicotinate metabolism and fatty acid metabolism, analyzed by gene expression data and metabolic data. We next verified that a series of key enzymes involved in fatty acid metabolism were down-regulated following SLC25A1 KD, accompanied by metabolites in fatty acid synthesized pathway. We also revealed that a self-protection mechanism after stress by up-regulating phosphoglycerides (PGs) and triglycerides (TAGs) in SLC25A1 KD cells, and the effect could be reversed by inhibiting DGAT1, a key enzyme catalyzing the conversion of fatty acids to TAGs.
 In conclusion, targeting mitochondrial metabolism and fatty acid metabolism using the SLC25A1 inhibitor might be a potential novel strategy for enhancing AML therapy and overcoming drug resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e2d9ee131e6bf2829d447dffd7b1068cf452245" target='_blank'>
              SLC25A1 Reprograms Mitochondrial and Fatty Acid Metabolism to Promote the Progression of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Xiaojing Yan, Miao Chen, Yuan Tao, Sijing Kang, Pengjie Yue, Rui Ge
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Melanoma is an aggressive cancer characterized by rapid growth, early metastasis, and poor prognosis, with resistance to current therapies being a significant issue. BRAF mutations drive uncontrolled cell division by activating the MAPK pathway. In this study, A375 and FO-1, BRAF-mutated melanoma cell lines, were treated for 4–5 months with RAF inhibitor dabrafenib or AZ628, leading to drug resistance over time. The resistant cells showed altered molecular signatures, with differences in cell cycle regulation and the propensity of cell death. Dabrafenib-resistant cells maintained high proliferative activity, while AZ628-resistant cells, especially A375 cells, exhibited slow-cycling, and a senescent-like phenotype with high susceptibility to ferroptosis, a form of cell death driven by iron. Antiretroviral drugs doravirine and cabotegravir, known for their effects on human endogenous retroviruses, were tested for their impact on these resistant melanoma cells. Both drugs reduced cell viability and colony formation in resistant cell lines. Doravirine was particularly effective in reactivating apoptosis and reducing cell growth in highly proliferative resistant cells by increasing tumor-suppressor proteins p16Ink4a and p27Kip1. These findings suggest that antiretroviral drugs can influence apoptosis and cell proliferation in RAF-inhibitor-resistant melanoma cells, offering potential therapeutic strategies for overcoming drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891570a4b6dcb28fdafca7435d6f3b715c4ac28f" target='_blank'>
              BRAF-Mutated Melanoma Cell Lines Develop Distinct Molecular Signatures After Prolonged Exposure to AZ628 or Dabrafenib: Potential Benefits of the Antiretroviral Treatments Cabotegravir or Doravirine on BRAF-Inhibitor-Resistant Cells
              </a>
            </td>
          <td>
            Valentina Zanrè, F. Bellinato, Alessia Cardile, Carlotta Passarini, Stefano Di Bella, Marta Menegazzi
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="

 The ribosomal protein RPS15 is recurrently mutated in chronic lymphocytic leukemia (CLL) and confers adverse prognosis. While translational rewiring is a recognized consequence of RPS15 mutation (RPS15-mut), its impact on genome stability and the precise mechanisms governing RPS15-mut driven B-cell leukemogenesis remain unclear. To delineate the sequence of cellular events that underlie malignant transformation and progression in the context of RPS15 mutation, we functionally interrogated a conditional knock-in mouse model of the RPS15-S138F mutation established using the Cd19-Cre/loxP system (Gutierrez et al, ASH 2020). Despite enrichment for Myc target genes in pre-leukemic Rps15-mutB splenocytes, these cells displayed a hypoproliferative phenotype characterized by increased G1 cell cycle checkpoint activation and reduced ex vivo proliferative capacity accompanied by increased apoptosis, compared to Rps15-WT splenocytes.
 We hypothesized that decreased proliferation and increased apoptosis could result from a p53-mediated response to cellular stress arising from Rps15 mutation. Informed by the genomic landscape of CLL developing in older Rps15-mut mice that displayed an enrichment of A•T>C•G and C•G>T•A transversions indicative of peroxide-induced DNA damage, we interrogated the transcriptomes of Rps15-mutpre-leukemic B splenocytes, which showed upregulation of reactive oxygen species (ROS) genes (NES 1.5, p<0.05). We experimentally quantified mitochondrial ROS which confirmed increased oxidative stress in Rps15-mut compared to Rps15-WT splenocytes (p<0.001). We reasoned that unresolved oxidative stress could lead to DNA damage accumulation. Accordingly, we demonstrated increased γH2AX accumulation, a marker of DNA damage, in Rps15-mut B splenocytes compared to their Rps15-WT counterparts (p=0.01), as well as hypersensitivity of Rps15-mut splenocytes to oxidative stress overload with parthenolide (p=0.0009). RPS15 is known to stabilize p53 via MDM2 inhibition, hence Rps15-mut cells exhibited reduced p53 expression. Nevertheless, in response to DNA damage, p53 and p21 were significantly induced, the latter a p53-dependent mediator of G1 checkpoint activation, highlighting the functional importance of residual p53 activity in Rps15-mut B cells.
 Given their heightened level of DNA damage, we further hypothesized that Rps15-mut pre-leukemic B splenocytes are hyper-dependent on DNA damage response (DDR). Conversely, defective DDR could precipitate genomic instability and facilitate acquisition of additional genomic alterations, such as TP53 deletion or loss-of-function mutation, that might promote transition toward a hyperproliferative or malignant state. To determine whether Rps15-mut B splenocytes could effectively respond to further genomic insults, we assessed cellular response to hydroxyurea (HU) and ionizing radiation (IR) that induce replication stress and DNA double-strand breaks (DSBs) respectively. While the ATR-Chk1 response to HU was preserved, the ATM-Chk2 response to IR was defective with diminished phosphorylation of downstream ATM targets and impaired induction of G2/M arrest. Indeed, Rps15-mut B cells subsequently acquired a spectrum of additional mutations and chromosomal alterations (e.g. Myc/Mapk amplification, Trp53 loss) that reflect genomic instability secondary to defective DDR.
 Finally, we postulated that translational defects directly contribute to oxidative DNA damage accumulation and genomic instability in Rps15-mut B cells. We therefore performed ribosome profiling that showed differential translation efficiency (TE) in 342 genes between Rps15-mutand Rps15-WT B splenocytes, with significant enrichment of DNA replication, DDR and cell cycle genes by GSEA that was recapitulated in RPS15-mut vs WT isogenic HG3 CLL cell lines. In particular, Gpx1, a glutathione peroxidase that functions as a key cellular antioxidant, was confirmed by western blot to be downregulated in Rps15-mut B cells secondary to reduced TE (0.6-fold change vs WT, p=0.018), alongside other proteins important for the amelioration of oxidative DNA damage and DSBs (e.g. Cyb5r4, Otud4, Fance, Tlk1). Our analyses thus link RPS15-induced translational alterations with a cascade of cellular defects that compromise genome stability and highlight the critical nature of these defects in B-cell transformation and CLL progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4b6feb4689d0ca7cd852af6a8ea529fdb6b2609" target='_blank'>
              Impaired Response to Oxidative DNA Damage Underlies Genomic Instability in B-Cell Leukemia Driven By RPS15 Mutation
              </a>
            </td>
          <td>
            M. Kwok, Catherine Gutierrez, Neil Ruthen, Peyton Waddicor, Tamara Ouspenskaia, Christina Curran, Doris Fu, B. Knisbacher, Anat Biran, Adi Nagler, Aviv Liani, Sarah Chen, G. Lazarian, Elizabeth Witten, Gabriela Brunsting Hoffmann, Emma Lin, Lucas Pomerance, Shuqiang Li, Lili Wang, K. Livak, D. Neuberg, G. Getz, Elisa ten Hacken, Catherine J. Wu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 Despite standard treatment, IDH-mutant gliomas inevitably recur. Previous studies suggest that therapeutic resistance may result from a combination of intratumoral cellular heterogeneity, epigenetic evolution, and acquired genetic alterations. However, how these multilayered molecular features interact to influence the evolutionary paths of IDH-mutant gliomas remains incompletely understood. To chart this evolution, 75 glioma samples were longitudinally collected from 35 patients (n = 13 oligodendroglioma, n = 22 astrocytoma) and profiled using single nucleus RNA sequencing, ATAC sequencing, and bulk DNA sequencing. Analysis of 331,016 nuclei (snRNA) revealed eight tumor microenvironment cell types and five pan-IDH-mutant malignant cellular states. The malignant states were distributed along a cellular hierarchy of stem-like populations (neural progenitor cell-like, oligodendrocyte progenitor cell-like, and undifferentiated) and more differentiated populations (mesenchymal-like and astrocyte-like) with cycling cells enriched in the stem-like populations. Joint open chromatin accessibility data (snATAC) was available for a subset of malignant cells (71,088 nuclei) and state-specific differentially accessible peaks supported that these states are epigenetically encoded. Across both subtypes, higher tumor grade was associated with reduced malignant cell differentiation and increased cycling populations. Between the two time points, we identified that there was a longitudinal increase in the cycling, undifferentiated, and mesenchymal-like populations with a corresponding decrease in the astrocyte-like population. Longitudinal genetic analysis revealed that 19 of 35 tumors acquired at least one of the following genetic alterations: treatment-associated hypermutation, cell cycle alteration, or large changes in copy number alterations. Importantly, tumors that acquired these key genetic alterations demonstrated significant shifts towards reduced differentiation and increased cycling populations while those tumors that did not had more stable malignant profiles. Collectively, our results suggest a common cellular hierarchy across IDH-mutant gliomas with a shift towards reduced differentiation, increased cycling populations during disease progression that is driven by acquired genetic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f68ff076c678ea3a1de296cefcb3a2b2a3560d" target='_blank'>
              EPCO-26. LONGITUDINAL SINGLE-CELL ANALYSES IDENTIFY DRIVERS OF GENETIC, EPIGENOMIC, AND CELLULAR EVOLUTION IN IDH-MUTANT GLIOMA
              </a>
            </td>
          <td>
            K. Johnson, Avishay Spitzer, F. Varn, M. Nomura, L. Garofano, T. Chowdhury, K. Anderson, F. D’Angelo, L. Bussema, S. Gritsch, Y. Oh, H. Moon, S. Paek, Franck Bielle, A. Laurenge, Anna Luisa Di Stefano, B. Mathon, A. Picca, M. Sanson, A. Lipsa, F. Hertel, Zheng Zhao, Qianghu Wang, T. Jiang, B. Hermes, N. Sanai, A. Gołȩbiewska, S. Niclou, J. Huse, W. Yung, A. Lasorella, M. Suvà, A. Iavarone, I. Tirosh, R. Verhaak
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Intercellular communication between distinct transcriptional states of various cell types in the tumor microenvironment (TME) influence the progression and clinical outcome of the tumors. Details of such clinically relevant cellular state interactions (CSIs) in IDH-mutant gliomas remain obscure. Here we developed CSI-TME, a computational pipeline that deconvolves cell type-specific gene expression from bulk transcriptomic data, identifies distinct cell states, and uncovers prognostic cell state interactions by modelling the clinical data based on the joint activity of cell state pairs. We found a significantly reproducible cell state interaction network (CSIN) that is (i) predominantly pro-tumor, (ii) likely mediated through the interaction between cell surface molecules, (iii) differentially activated in IDH-mut astrocytoma and oligodendroglioma, and (iv) significantly associated with response to immune checkpoint blockade therapy. The distinct malignant cell states involved in CSIs resembled neuronal lineages such as astrocyte-like and oligodendrocyte progenitor cells-like states and captured key interactions between glioma stem cells and immune cells. Integration of CSIN with somatic mutation data suggests the anti-tumor role of CSIs in the early stages that transitions towards pro-tumor role as glioma progresses. In summary, CSI-TME provides valuable insights into the physiology of the TME in IDH-mutant glioma and provides a framework to prioritize ligand-receptor interactions or patients stratification for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f3c50514c2a24aadfd75ca63abf749d7a1de48" target='_blank'>
              Identifying Prognostic Cell State Interactions in the Tumor Microenvironment of IDH-Mutant Gliomas Using CSI-TME
              </a>
            </td>
          <td>
            Arashdeep Singh, Bharati Mehani, Vishaka Gopalan, Kenneth Aldape, Sridhar Hannenhalli
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Proteins that emerge de novo from noncoding DNA could negatively or positively influence cellular physiology in the sense of providing a possible adaptive advantage. Here, we employ two approaches to study such effects in a human cell line by expressing random sequences and mouse de novo genes that lack homologs in the human genome. We show that both approaches lead to differential growth effects of the cell clones dependent on the sequences they express. For the random sequences, 53% of the clones decreased in frequency, and about 8% increased in frequency in a joint growth experiment. Of the 14 mouse de novo genes tested in a similar joint growth experiment, 10 decreased, and 3 increased in frequency. When individually analysed, each mouse de novo gene triggers a unique transcriptomic response in the human cells, indicating mostly specific rather than generalized effects. Structural analysis of the de novo gene open reading frames (ORFs) reveals a range of intrinsic disorder scores and/or foldability into alpha-helices or beta sheets, but these do not correlate with their effects on the growth of the cells. Our results indicate that de novo evolved ORFs could easily become integrated into cellular regulatory pathways, since most interact with components of these pathways and could therefore become directly subject to positive selection if the general conditions allow this.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43878c4781a378a056438c19941008a1a26fd3df" target='_blank'>
              Expression of Random Sequences and de novo Evolved Genes From the Mouse in Human Cells Reveals Functional Diversity and Specificity
              </a>
            </td>
          <td>
            Silvia Aldrovandi, Johana Fajardo Castro, K. Ullrich, Amir Karger, Victor Luria, D. Tautz
          </td>
          <td>2024-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec5ccdfbc4363bf23d6a45f74342f6b2cc8e760a" target='_blank'>
              CRISPR/Cas9 system: a novel approach to overcome chemotherapy and radiotherapy resistance in cancer.
              </a>
            </td>
          <td>
            Somaye Noruzi, R. Mohammadi, K. Jamialahmadi
          </td>
          <td>2024-11-19</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfcbcf7fbffafb54867bd0e1ae67ad8c3f72b49a" target='_blank'>
              Longitudinal analysis of genetic and epigenetic changes in human pluripotent stem cells in the landscape of culture-induced abnormality
              </a>
            </td>
          <td>
            Yunsoo Kim, Byunghee Kang, Solbi Kweon, Sejin Oh, Dayeon Kim, Dayeon Gil, Hyeonji Lee, Jung-Hyun Kim, Ji Hyeon Ju, Tae-Young Roh, Chang Pyo Hong, Hyuk-Jin Cha
          </td>
          <td>2024-11-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Diffuse intrinsic pontine gliomas (DIPG/DMG) are devastating pediatric brain tumors with very limited treatment options and a uniformly fatal prognosis. The recurrent histone H3K27M mutation in DIPG disrupts epigenetic regulation, leading to the hypothesis that this mutation-induced dysregulation creates a vulnerability to epigenetic targeting. We conducted a screening of compounds targeting epigenetic enzymes to identify potential inhibitors of patient-derived DIPG cell growth. High-throughput screening identified chaetocin, a fungal metabolite, as a potent inhibitor of DIPG cell growth. Chaetocin treatment selectively decreased proliferation and increased apoptosis in DIPG cells, significantly extending survival in DIPG xenograft models while restoring H3K27me3 levels. The loss of the H3K9 methyltransferase SUV39H1 also inhibited DIPG cell growth. Transcriptomic and epigenomic profiling showed that SUV39H1 loss or inhibition downregulated stemness and oncogenic networks, including growth factor receptor signaling. However, D2 dopamine receptor (DRD2) signaling adaptively upregulated, conferring resistance. Combining chaetocin with the DRD2 antagonist ONC201 or radiotherapy synergistically enhanced antitumor efficacy. This study reveals a therapeutic vulnerability in DIPG cells by targeting the SUV39H1-H3K9me3 pathway and compensatory signaling loops. Combining SUV39H1-targeting chaetocin with other agents, such as ONC201, may offer a new strategy for effective DIPG treatment. The findings suggest that chaetocin, by targeting and suppressing the H3K9me3 methyltransferase SUV39H1, inhibits DIPG tumor growth by downregulating stemness-related genes and growth factor signaling pathways. Overall, the study highlights a new therapeutic vulnerability in DIPG through the SUV39H1-H3K9me3 epigenetic pathway and compensatory signaling loops, suggesting an effective treatment strategy for DIPG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8870c1e027b5fc774e0254a042e95794942584ce" target='_blank'>
              CSIG-26. TARGETING EPIGENETIC DYSREGULATION AND ADAPTIVE RESISTANCE MECHANISMS IN DIPG/DMG
              </a>
            </td>
          <td>
            Eric Xin, Xiaowen Zhong, Chris Huang, Richard Lu
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Down Syndrome (DS), caused by an extra copy of chromosome 21, leads to over-expression of 200–300 genes. This genetic imbalance contributes to various health complications, including an elevated risk of blood cancers. Modulating gene expression is vital to mitigating these risks.
Methodology: This study focused on reducing the over-expression of genes on chromosome 21 by inhibiting the proteins they encode. Potential inhibitors were identified and evaluated using electrostatic interaction scoring and protein docking simulations. The goal was to reduce gene expression by 50%, thereby mitigating the harmful effects of over-expressed proteins.
Results: Analysis confirmed significantly higher expression levels of chromosome 21 genes in DS individuals. Targeting the encoded proteins successfully reduced gene expression by 50%. Protein docking studies identified a promising inhibitor that effectively blocked over-expressed proteins, demonstrating potential to prevent blood cancer by mitigating the effects of the extra genetic material.
Conclusion: This study proposes a novel therapeutic strategy for DS by focusing on inhibiting the over-expressed proteins linked to blood cancer. The identified drug candidates hold promise for improving health outcomes and quality of life in individuals with DS by reducing the risk of blood cancers and other complications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c0c7e4f1729a11f5a7f552440d26350c34a9f78" target='_blank'>
              Targeting Chromosome 21 Gene Over-Expression: A Novel Approach to Prevent Health Complications in Individuals with Down Syndrome.
              </a>
            </td>
          <td>
            Shehnila Zardari, N. Shar, Afshan Chohan, A. Majeed
          </td>
          <td>2024-11-30</td>
          <td>International Journal of Endorsing Health Science Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="

 INTRODUCTION
 CD19-directed chimeric antigen receptor (CAR-19) T cells have significantly improved outcomes for heavily pretreated patients with aggressive large B-cell lymphoma (LBCL). However, a high proportion of patients experience treatment failure. Our previous study (Jain et al., Blood 2022) identified genomic complexity as a factor associated with treatment failure. However, the mechanisms through which genomic complexity promotes resistance to CART are largely unknown.
 METHODS
 To investigate this important topic, we expanded our cohort and analyzed 61 whole-genome sequencing (WGS) and 54 RNA sequencing samples from 54 LBCL patients treated with CAR-19 therapy. The samples included 39 collected at baseline, 15 at relapse, and 7 both before and after treatment. Longitudinal samples were collected in different anatomical sites. We conducted phylogenetic analysis on the 7 paired samples and somatic driver discovery on baseline samples, integrating single nucleotide variants, small insertions/deletions, copy number variants, and structural variants.
 RESULTS
 Phylogenetic analysis of 7 patients who relapsed within the first year revealed minimal differences and absence of clonal shift, likely reflecting spatial heterogeneity rather than post-CAR19 selection. These results suggest that tumors likely resist CAR-19 treatment due to genomic drivers already present at baseline.
 Implementing the somatic driver discovery analysis in 54 LBCL patients, we identified several new genomic drivers involved in CAR19 resistance. Overall, these driver genes can be divided in two main groups: alterations involving the i) immune-synapse and ii) CD19 pathways. In the first group HLA-A, HLA-B, HLA-C, and B2M genes were recurrently involved by somatic events. While HLA class I monoallelic loss, detectable in 38.9% of patients, did not have any impact on progression free survival (PFS), all 10 patients with biallelic loss of the HLA class I complex progressed (p=0.0056), 90% within the first year. Given that tumor cells losing the HLA class I should be cleared by natural killer (NK) cells, we investigated somatic events involving in gene that regulate the interactions between tumor cells and NK cells. We identified recurrent loss-of-function (LOF) alterations in CD58, CD48, MICA, PVR, SPPL3, and TNFSF9 genes, and gain-of-function (GOF) alterations in MUC1, GMDS and CD44, affecting 39% of responders and 67% of progressors. These genes have been previously found be associated with resistance to NK cell-mediated killing. We found that alterations involving these genes co-occurred with biallelic loss of HLA class I in 8 out of 10 patients who progressed after CAR-19, emphasizing the importance of losing both innate and adaptive defense mechanisms. Additionally, analyzing genes involved in the inhibition of T cell function, we identified one patient (CAR_130) with PD-L1 double minutes and focal multi-gain event (copy-number = 12) experiencing progression after CAR-19 treatment. In addition, we found GOF of CMTM6, associated with maintaining PD-L1 expression in the tumor surface, in 5 patients, 4 of which showed disease progression within 99 days.
 Moving our focus to genes involved in the CD19 expression, we identified 4 interesting genes recurrently involved by somatic events and associated with primary refractoriness. 8 patients with LOF alterations in ETV6 had shorter PFS (p=0.0026). PAX5 and EBF1 genes, important in the regulation of CD19 expression, were involved by LOF mutations in 13 patients, with 85% progressing after CAR-19 treatment. Finally, we confirmed the CAR-19 therapy resistance role of RHOA, with 12 out of 13 patients with LOF alterations progressed in the first year (p=0.0008). Functional work showed how reduction of RHOA causes reduction in CD19 protein density and resistance to CAR19 in vitro.
 Combining the immune escape mechanism alterations and the intrinsic tumor drivers related to CD19, we found that 16 out of 36 progressors showed at least two alterations in both groups. Having alteration from both groups associated with shorter PFS (p<0.0001), with 88% of these patients progressing within the first 6 months.
 CONCLUSION
 These findings offer evidence of how genomic instability promotes resistance to CAR19 by permitting the selection of genomic alterations that affect the critical relationship between CAR19 cell activity, the immune system, and tumor cell immunogenicity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5997b4473fdf87c7f846f2026a172f53d89e04ec" target='_blank'>
              Intrinsic Tumor Drivers and Immune Escape Mechanisms in CD19 CAR T-Cell Therapy Resistance for Aggressive Large B Cell Lymphoma
              </a>
            </td>
          <td>
            B. Ziccheddu, M. Jain, M. Chojnacka, Michael Durante, Venu Venkatarame Gowda Saralamma, A. D. Newsam, Jerald Noble, Nicholas B Figura, M. Menges, O. Landgren, Marco L Davila, Jonathan H. Schatz, F. Maura, Frederick L. Locke
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cancers exhibit a remarkable ability to develop resistance to a range of treatments, often resulting in relapse following first-line therapies and significantly worse outcomes for subsequent treatments. While our understanding of the mechanisms and dynamics of the emergence of resistance during cancer therapy continues to advance, many questions remain about which treatment strategies can minimize the probability that resistance will evolve, thereby improving long-term patient outcomes. In this study, we present an evolutionary simulation model of a clonal population of cells that can acquire resistance mutations to one or more treatments. We then leverage this model to examine the efficacy of a two-strike “extinction therapy” protocol—in which two treatments are applied sequentially in an effort to first contract the population to a vulnerable state and then push it to extinction—in comparison to that of a combination therapy protocol. We investigate the impact of parameters such as the timing of the switch between the two strikes, the rate of emergence of resistant mutations, the dose of the applied drugs, the presence of cross-resistance, and whether resistance is a binary or a quantitative trait. Our results indicate that the timing of switching from the first to the second strike has a marked effect on the likelihood of driving the population to extinction, and that extinction therapy outperforms combination therapy when cross-resistance is present. We conduct an in silico trial that reveals more detailed insight into when and why a second strike will succeed or fail. Finally, we demonstrate that modeling resistance as a quantitative rather than binary trait does not change our overall conclusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f738c3dddcab46b26ff2c97c3888fefb1e9d2858" target='_blank'>
              Evolutionary rescue model informs strategies for driving cancer cell populations to extinction
              </a>
            </td>
          <td>
            Amjad Dabi, Joel S. Brown, Robert A Gatenby, Corbin D. Jones, Daniel R. Schrider
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 The DNA mismatch repair (MMR) pathway triggers glioblastoma (GBM) apoptosis when base mismatches induced by the alkylating agent temozolomide (TMZ) cannot be repaired. GBMs often develop hypermutation and resistance to TMZ through acquired inactivation of DNA MMR enzymes (MSH2, MSH6, MLH1, PMS2). A newly developed therapeutic, KL-50, fluoro-ethylates DNA bases, leading to DNA inter-strand cross-links. This form of DNA damage triggers MMR-independent apoptosis. Thus, KL-50 can target MMR-deficient tumor cells. Previous work showed that KL-50 is effective against GBM cell lines that were rendered TMZ-resistant through shRNA knockdown of MMR genes (PMID 35901163). In those studies, KL-50 was most effective when used in combination with an MGMT enzyme inhibitor, or in MGMT-deficient cells. Here, we extend those findings with a preclinical trial of KL-50 in GBM patient-derived xenografts (PDX), including PDX models of post-TMZ GBM that acquired MMR mutations and hypermutation through serial TMZ exposure (PMID 31852831). Among TMZ-naïve PDX, KL-50 extended survival of engrafted mice by 24 days for GBM6 (partially MGMT-deficient), and by 38 days for GBM12 PDX (fully MGMT-deficient) compared to vehicle (p<0.0001, Log-Rank). Combining KL-50 with 4.0 Gy radiation further extended survival of GBM12 mice (9 days, p=0.02), but not GBM6 mice. With engraftments of GBM6R-m185, a post-TMZ, MMR-deficient derivative of GBM6, 14/14 (100%) of KL-50 treated mice are still alive 60 days post-engraftment compared to 0/13 (0%) of vehicle control mice (median survival 37 days, p<0.0001, experiment ongoing). Complementary in vitro studies showed that GBM6R-m185 cells are more sensitive to KL-50 than TMZ at matched concentrations of 50µM (p=0.001, one-way ANOVA) and 100µM (p=0.047). These data demonstrate the potential of KL-50 in treating post-TMZ MMR-deficient recurrent gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82dc9d9d810ed8baa9128dcd5ddc06cb0fb49aee" target='_blank'>
              EXTH-23. NEW DNA-CROSSLINKING CHEMOTHERAPY IS EFFECTIVE AGAINST POST-TEMOZOLOMIDE MISMATCH REPAIR-DEFICIENT PATIENT-DERIVED HYPERMUTANT GLIOMAS
              </a>
            </td>
          <td>
            Matthew McCord, Wenxia Wang, Shejuan An, Thomas Sears, J. Sarkaria, C. James, Bruce Ruggieri, R. Bindra, C. Horbinski
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="


 Lynch syndrome is caused by pathogenic germline variants in the DNA mismatch repair (MMR) genes, predisposing individuals to malignancies, including glioblastoma (GBM). MMR-deficient tumors are resistant to monofunctional alkylators such as temozolomide but not to bifunctional alkylators such as the nitrosourea lomustine. Anti-programmed cell death protein 1 (PD1) immune checkpoint inhibitor (ICI) pembrolizumab represents another treatment option, which is approved by the United States Food and Drug Administration (FDA) for the treatment of mismatch repair deficient cancers.



 We present a patient with multifocal GBM and Lynch syndrome who experienced rapid progression following standard-of-care chemoradiation, followed by durable responses to lomustine and then to pembrolizumab.



 A 58-year-old man with Lynch syndrome (MSH2 mutation) and prior history of colon cancer presented with seizures. Magnetic resonance imaging (MRI) brain identified right frontal and left parietal masses. Right frontal tumor was resected; pathology was consistent with GBM, IDH wildtype, MGMT-unmethylated. Both lesions were treated with radiation therapy (60 Gy, 30 fractions) with concurrent and adjuvant temozolomide. After two cycles of adjuvant temozolomide, his left parietal tumor progressed and was subtotally resected, followed by four cycles of lomustine (stopped due to myelotoxicity). He remained progression-free for 15 months. Surveillance MRI identified new enhancement involving the left frontal lobe, left periventricular white matter, and right internal auditory canal, concerning for leptomeningeal dissemination. MRI spine identified no additional lesions. Cerebrospinal fluid cytology was negative for tumor cells on two occasions. Patient was treated with pembrolizumab. He remains on treatment with stable disease four months after ICI initiation and 31 months after GBM diagnosis.



 Patients with MMR-deficient GBM resist standard chemoradiation. Alternatives such as nitrosourea or ICI therapy may achieve durable responses. While data is insufficient to administer these agents as upfront therapy in this specific setting, both should be considered as options for recurrent disease.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2206e03152bec09798388ba076742d60a7a64e2b" target='_blank'>
              DNAR-13. LONG TERM FOLLOW-UP OF A LYNCH SYNDROME-ASSOCIATED GLIOBLASTOMA RESISTANT TO STANDARD-OF-CARE CHEMORADIATION
              </a>
            </td>
          <td>
            Merve Hazal Ser, D. Baykal, J. White, W. Breen, I. Parney, T. Walbert, Peter Forsyth, Arnold B Etame, Robert Macaulay, Yolanda Pina, U. Sener
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1b1de4a2bc0d1d7dad1a7c12515d499e7bb94f6" target='_blank'>
              The tumor suppressor SALL2 opposes chemotherapeutic resistance in breast cancer.
              </a>
            </td>
          <td>
            Qiji Li, Chenxin Li, Yuhao Zhang, Zihan Zheng, Yun Wang, Yingqian Yang, Qingqing Zhu, Rui Wang, Wanhui Xu, Chengming Zhu, Qin Tian, Meng Wang, Liping Ye
          </td>
          <td>2024-11-22</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 The clonal architecture of acute myeloid leukemia (AML) can be highly complex and heterogeneous. This genetic diversity complicates the treatment of AML, as different clones may respond variably to therapies, contributing to treatment resistance and disease relapse. Understanding the mechanisms and implications of clonal evolution in AML is crucial for developing more effective treatment strategies that can target the diverse cellular populations.
 The generation of patient-derived xenograft (PDX) models has revolutionized the study of AML. One of the main uses of PDX models is to study how a leukemic sample might respond to different therapies. Here, we used single-cell DNA sequencing (scDNA-seq) and surface protein profiling to monitor the clonal evolution and differentiation of an isocitrate dehydrogenase 1 (IDH1)-mutated leukemic sample in response to ivosidenib (IVO), a mutant IDH1 inhibitor, either as monotherapy or in combination with venetoclax (VEN) or azacitidine (AZA). Single-cell mutation analysis of the engrafted cell population showed that the initial evolutionary path was linear characterized by the sequential acquisition of DNMT3AR882H (clone 1), WT1R467W (clone 2), IDH1R132H (clone 3), NPM1Trp288Cysfs*12 (clone 4), FLT3-ITD (clone 5), and WT1R439C (clone 6) mutations. Three subclones branched off at the end of this linear path with each acquiring either a TET2Q1548del (clone 7), DNMT3AD531del (clone 8), or KITH40QfsTer6 (clone 9) mutation. The overall clonal composition of the input primary sample and sample from vehicle-treated animals were largely similar. To quantify the differences in therapeutic response between clones, we developed a hierarchical multinomial Bayesian model that estimated the fold change in absolute cell numbers for each clone compared with vehicle treatment. This analysis revealed that both single-agent VEN and IVO+VEN were generally more effective in targeting earlier-stage clones (clones 3-5) than later-stage clones (clones 6-9). In contrast, single-agent AZA and IVO+AZA were effective in eliminating clone 6 as well as earlier-stage clones. Notably, the combination of IVO+AZA was effective in reducing all the clones by approximately the same magnitude, including clones 7 and 8 which demonstrated lower sensitivity to AZA monotherapy. Given that the WT1R439C mutation, acquired in clone 6, distinguished early-stage from late-stage clones, these findings suggest that WT1 mutations might contribute to IVO and VEN resistance and that the addition of AZA can overcome this mechanism of resistance. Consistent with these findings, the combination of IVO+AZA strongly upregulated the expression of the myeloid markers CD11b, CD14, and CD15 to a greater extent than with either single agent. These findings provide evidence that AZA and IVO could synergize to overcome the differentiation block in IDH1-mutated AML cells, reflecting the superior clinical efficacy of this regimen over single-agent IVO.
 To broaden the applicability of PDX models to study the competition between genetic clones that co-exist infrequently in an individual sample, we generated mixed PDX (mPDX) models by co-engrafting two or more leukemic samples in the same animal and used scDNA-seq to deconvolute the clonal composition and sample origin of the engrafted cells. As proof-of-principle, we generated an IDH1R132H/IDH2R140Q mPDX to model isoform switching in both directions by treating the animals with either IVO or enasidenib (ENA), a mutant IDH2 inhibitor. We observed the expected differentiation response and depletion of IDH1-mutated or IDH2-mutated clones by IVO or ENA treatment as single agents, respectively. Further studies are underway to demonstrate whether dual mutant IDH1 and IDH2 inhibition (IVO+ENA) will circumvent resistance driven by isoform switching.
 In summary, we demonstrate the utility of applying single-cell proteogenomic analysis in traditional and mixed PDX models to gain crucial insights into mechanisms of resistance and potential strategies to overcome it.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad2ac5a87afb275fd53cef27ee87a6e28e0b788" target='_blank'>
              Single-Cell Proteogenomic Analysis of Clonal Evolution in Patient-Derived Xenograft Models of AML Treated with IDH Inhibitors
              </a>
            </td>
          <td>
            Alex Liu, S. Cathelin, Yitong Yang, Farzaneh Aboualizadeh, Amina Abow, G. Basi, Lance Li, David L Dai, Abdula Maher, Eric Grignano, Mohsen Hosseini, Dhanoop Manikoth Ayyathan, Vivian Wang, Troy Ketela, Dylan M Marchione, Adriana Tron, A. Arruda, Mark D. Minden, Steven M Chan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="

 Complete or partial deletion of chromosome 7 (-7/del7q) is one of the most frequent classes of chromosomal abnormalities found in myeloid malignancies. -7/del7q is distinguished by its poor prognosis, high prevalence among pediatric cases, association with bone marrow failure syndromes, and widespread occurrence among various pathomorphological subtypes of myeloid malignancy (Mori et al.). Despite decades of study, however, the genes responsible for the leukemogenic properties of these deletions are incompletely identified and no targeted therapies matched to the lesion exist (Inaba et al.). Hypothetical candidate mechanisms of -7/del7q pathogenesis include haploinsufficiency and biallelic inactivation of tumor-suppressor genes (TSGs). Deletion may also create clonal vulnerabilities due to haploid loss of genes contained in deleted regions that render clonal cells vulnerable to synthetic-lethal gene (SLG) targeting treatment strategies while leaving normal cells unaffected. To date, these aspects of -7/del(7q) pathogenesis have been tested only on a piecemeal basis, with most studies focusing on identifying haploinsufficient TSGs. Our group has performed extensive computational analyses to query these with the intention of creating a comprehensive understanding of -7/del7q pathogenesis. We have further verified these analyses with selected knockout models (Vail et al, Durkin et al), initiated a wide scale screening effort to comprehensively assay all published -7/del7q potential leukomegenic drivers, and continue to iterate our computational analyses to evolve a detailed map of pathogenic pathways in this context.
 To these aims, we queried an extensive in-house databank of myeloid neoplasia patients, as well as external data sources. Within this combined cohort are RNA-seq datasets from 29 - 7/del7q patients and WXS/WGS datasets from 47 - 7/del7q patients.
 We conceptualized TSGs as either haploinsufficient genes or as genes that are biallelically inactivated, where one allele is lost through the deletion event and the other through germline or somatic mutation. To identify this overlooked class of TSGs, we created a biallelic inactivation test, wherein we queried chromosome 7 genes for preferential loss of either the wild-type or a previously mutated allele. We identified 84 genes with a significant preference for loss of the wild-type allele, suggesting these genes are TSGs physiologically inactivated by the second loss-of-function genetic lesion. Among these hits were four genes that are known tumor suppressors in other contexts: TRIM24, CCDC138, SFRP4, and PTPN12.
 Leukemogenesis in the -7/del7 context may also be driven through the loss of specific combinations of genes. To explore combinatory inactivation of more than one chromosome 7 TSG as a potential leukemogenic mechanism, we expanded the biallelic inactivation test to check for co-mutation amongst chromosome 7 gene pairs. From this, we identified a set of 21 chromosome 7 genes with 33 co-mutation preferences amongst the set, including RAPGEF5: MIOS, TAF6:MCM7, and PMS2:AIMP2.
 We also sought to identify non-TSG therapeutic targets in -7/del7q lesions. We analyzed expression and mutation signatures to detect compensatory genes that may represent vulnerabilities within cancer cells. We identified hemizygous mutations in our patient data and compared expression levels to diploid patients. As expected, most genes lost in a -7/del7q lesion were commensurately downregulated, but some showed no loss (or even gain) of expression. These upregulated genes may be required for cell survival in cancer cells, but not healthy cells, representing druggable SLG targets. Genes with expression unaffected (or upregulated) by deletion include those whose protein is targetable by an existing small molecule (ABCB5, ADCY1, CYP3A5, EPHA1, NOS3, NPC1L1, SMO), an existing biotherapeutic (monoclonal antibody/enzyme or other protein) (ABCB4, IL6, LAMB1), or both (ABCB1, EGFR, GPER1, MET).
 This analysis, combined with others focused on identifying haploinsufficient TSGs, comprise a comprehensive and rational atlas of -7/del7q leukemogenic drivers and therapeutic targets. These targets are currently being experimentally confirmed and characterized with the goal of developing targeted small molecule therapies via our in-house drug developmental pipeline to address the unmet clinical need these lesions present.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b967fcf35c33681f24b36588b097cfb11991e2a3" target='_blank'>
              Identification of Tumor Suppressors and Synthetic Lethal Targets in -7/del7q Myeloid Neoplasms By Iterative Computational Analysis Coupled with Experimental Validation
              </a>
            </td>
          <td>
            Daniel J Vail, V. Visconte, B. Jha, Jill Durkin, Jaroslaw P. Maciejewski, T. LaFramboise
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a hematological malignancy characterized by a block in differentiation and accelerated proliferation of myeloid progenitor cells. Genes encoding for epigenetic regulators are among the most frequent targets for mutations and structural variations in AML, giving rise to profound epigenetic heterogeneity between and within tumors. Deletions of chromosome 5q [del(5q)] are among the most common copy number alterations in AML and are associated with extremely poor clinical outcome and therapy resistance, however the mechanisms linking del(5q) to leukemic progression are not understood. Analyzing DNA methylation profiles from 477 elderly AML patients using DNA methylome deconvolution, we discovered that del(5q) AML is an epigenetically distinct subgroup characterized by a signature of DNA hypermethylation, which we propose may be linked to dysregulation of H3K9me1/2 and overexpression of the leukemic stem cell marker, DNMT3B. Interrogation of the minimally deleted 5q region highlighted the H3K9me1/2 demethylase KDM3B as a likely target for haploinsufficiency in this subgroup. Our data suggest that del(5q) AML should be reconsidered as an epigenetically dysregulated subgroup, driven by heterozygous loss of KDM3B, and that the resulting imbalance of H3K9me1/2 may contribute to the progression of these aggressive leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8284d22674e6348a422c0aa420226cc0452e3a13" target='_blank'>
              Acute Myeloid Leukemia with deletion 5q is an epigenetically distinct subgroup defined by heterozygous loss of KDM3B
              </a>
            </td>
          <td>
            Katherine Kelly, Linda Welte, Etienne Sollier, Anna Riedel, Fiona Brown-Burke, Michael Scherer, Harold Keer, Mohammad Azab, E. Jahn, H. Döhner, Konstanze Döhner, P. Lutsik, C. Plass
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Atypical teratoid/rhabdoid tumors (AT/RT) are the most common malignant brain tumors during infancy. They split into four molecular types. The major three (AT/RT-SHH, AT/RT-TYR, and AT/RT-MYC) all carry mutations in SMARCB1, the fourth smaller type is characterized by SMARCA4 mutations (AT/RT-SMARCA4). AT/RT are associated with a dismal outcome, and 50% of the patients suffer a progress or recurrence of the tumor. Molecular mechanisms of disease recurrence are currently unclear. Here, we identified molecular characteristics of AT/RT primaries and recurrences, which could help to improve the therapy for relapsed patients. Previously, we investigated tumor tissue from 26 patients revealing signatures of recurrences in comparison to matched primary tumor samples using histology, DNA methylation analysis, and bulk RNA sequencing. To further identify mechanisms regarding tumor progression and recurrence, we performed single-cell RNA (scRNA) sequencing for five of these primary-recurrence pairs so far using formalin-fixed paraffin-embedded (FFPE) tissue. Using Uniform Manifold Approximation and Projection (UMAP) for dimensionality reduction, we observed a distinct separation in the gene expression profiles of primary and recurrent tumor cells. In addition, we aimed to identify primary tumor cells which potentially persisted after therapy and developed to recurrences. Therefore, we identified primary tumor cells showing highest similarity to recurrent tumor cells using CIBERSORT and trajectory analysis. Differentially expressed genes of these specific primary tumor cells were investigated. They are involved in cell migration, -adhesion, -proliferation, and -stability, among others. Overall, molecular characteristics that occur in the course of the disease were identified, which may help to define new therapeutic targets for AT/RT recurrences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a60d1351c8cdf570aebc98d3d88c92560aa04e78" target='_blank'>
              PATH-22. SINGLE-CELL RNA (SCRNA) SEQUENCING OF PAIRED PRIMARY AND RECURRENT ATYPICAL TERATOID/RHABDOID TUMORS (AT/RT) REVEALED MOLECULAR DIFFERENCES AND THERAPY-RESISTANT PRIMARY TUMOR CELLS
              </a>
            </td>
          <td>
            L. Altendorf, Rajanya Roy, F. D. de Faria, K. Hack, A. Koch, Martin U Schuhmann, Peter Hauser, P. Johann, M. Hasselblatt, K. Kerl, M. Frühwald, U. Schüller
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is a uniformly lethal primary intracranial neoplasm that inevitably recurs despite treatment by DNA damaging agents radiation (RT) and temozolomide (TMZ). Recurrence may be driven by inherently resistant glioma stem-like cells (GSCs). GSCs have been shown to have constitutive upregulation of DNA damage response (DDR) pathways, resulting from elevated DNA replication stress. This baseline elevated DDR has consequently been shown to confer inherent resistance to DNA damaging agents, TMZ/RT. However, factors mediating elevated replication stress and upregulated DDR in GSCs are poorly defined. Previously, we identified the cell-surface protein, THY1, as a mediator of GSC phenotype and treatment resistance in GBM. To determine mechanisms mediating treatment resistance in THY1+ cells, we engineered patient-derived neurosphere and organoid models, to either overexpress or silence THY1 expression. These models were treated with RT and collected at various time points to measure DNA repair efficiency by assessing the formation and resolution of yH2AX foci and ATM/ATR activation. In parallel, patient-derived GBM tissue from over 20 patients was analyzed using single cell RNA sequencing and spatial transcriptomics to assess pathways associated with THY1 expression. Interestingly, spatial transcriptomics and single cell RNA sequencing data demonstrates significant upregulation of GSC pathways, as well as constitutive upregulation of DDR pathways (p<2e-16) in THY1-high cells. Exploiting our patient-derived lines we determined that DNA repair activation was upregulated at baseline, measured using yH2AX foci and immunoblotting. We then measured DDR in response to radiation and found that high THY1 expression mediated rapid repair of DNA. Overall, we demonstrate that THY1 mediates therapy resistance by constitutively upregulating DDR, likely by upregulating DNA replication stress. We are now working to determine the mechanisms mediating the GSC phenotype and DNA replication stress in THY1 cells in hopes of future translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fdfb60c03e046f43e9a607b25a101b9359a6482" target='_blank'>
              DNAR-08. THY1 MEDIATES TREATMENT RESISTANCE IN GLIOBLASTOMA THROUGH RAPID REPAIR OF DNA DAMAGE
              </a>
            </td>
          <td>
            Safiulla Basha Syed, Arushi Tripathy, V. Ravikumar, Sahezeel Awadia, Sravya Palavalasa, Dah-Luen Huang, Arvind Rao, Daniel R. Wahl, A. Rehemtulla, W. Al-Holou
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10915882394b88f99450ed5973df2cdfcc3bd788" target='_blank'>
              Expression quantitative trait loci influence DNA damage-induced apoptosis in cancer
              </a>
            </td>
          <td>
            Jessica Bigge, Laura L Koebbe, Ann-Sophie Giel, Dorothea Bornholdt, Benedikt Buerfent, Pouria Dasmeh, Alexander M. Zink, Carlo Maj, Johannes Schumacher
          </td>
          <td>2024-12-02</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49efc881c885ab1ba2d61433704ad1711808a206" target='_blank'>
              Dysregulation of R-loop homeostasis shapes the immunosuppressive microenvironment and induces malignant progression in melanoma.
              </a>
            </td>
          <td>
            Ouyang Yan, Yan Gu, Shuqin Li, Xianpeng Wei, Yang Liu, Zejun Wang, Fuzhou Tang, Shichao Zhang
          </td>
          <td>2024-11-02</td>
          <td>Apoptosis : an international journal on programmed cell death</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99b084e2208e5b0e67e8e029bea90456f03863d" target='_blank'>
              A multiplex single-cell RNA-Seq pharmacotranscriptomics pipeline for drug discovery.
              </a>
            </td>
          <td>
            Alice Dini, Harlan Barker, Emilia Piki, Subodh Sharma, Juuli Raivola, A. Murumägi, Daniela Ungureanu
          </td>
          <td>2024-10-31</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The development of sequencing technologies and their increased accessibility in clinical services and genetic laboratories have considerably accelerated the identification of genetic variants associated with rare diseases (RDs). Among these, Mendelian disorders of the epigenetic machinery (MDEM) are rare monogenic diseases characterized by the presence of mutations in genes encoding epigenetic regulators that play a key role in organismal development and cellular functions. Loss of function of these regulators is expected to lead to epigenetic modifications that profoundly affect genome expression and cellular identity. Disruptions in DNA methylation profiles have been documented in MDEMs, emerging as a useful diagnostic tool. The current challenge is to determine whether and how these epigenomic alterations drive the mechanisms underlying the clinical manifestations in patients suffering from this class of diseases. Studying MDEMs may therefore shed light on the important role of epigenetic information in health and disease, particularly the mechanisms involved in the development and understanding of complex pathologies, such as neurodevelopmental disorders and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b00b4062f8130453f8f77acda434d081dca0a7cd" target='_blank'>
              [Genetic diseases of epigenetic machinery].
              </a>
            </td>
          <td>
            Maud de Dieuleveult, Guillaume Velasco
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Many patients suffering from inherited diseases do not receive a genetic diagnosis and are therefore excluded as candidates for treatments, such as gene therapies. Analyzing disease-related gene transcripts from patient cells would improve detection of mutations that have been missed or misinterpreted in terms of pathogenicity during routine genome sequencing. However, the analysis of transcripts is complicated by the fact that a biopsy of the affected tissue is often not appropriate, and many disease-associated genes are not expressed in tissues or cells that can be easily obtained from patients. Here, using CRISPR/Cas-mediated transcriptional activation (CRISPRa) we developed a robust and efficient approach to activate genes in skin-derived fibroblasts and in freshly isolated peripheral blood mononuclear cells (PBMCs) from healthy individuals. This approach was successfully applied to blood samples from patients with inherited retinal dystrophies (IRD). We were able to efficiently activate several IRD-linked genes and detect the corresponding transcripts using different diagnostically relevant methods such as RT-qPCR, RT-PCR and long- and short-read RNA sequencing. The detection and analysis of known and unknown mRNA isoforms demonstrates the potential of CRISPRa-mediated transcriptional activation in PBMCs. These results will contribute to ceasing the critical gap in the genetic diagnosis of patients with IRD or other inherited diseases. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee39154429bf4aa80aaffa09c68b811f369390d0" target='_blank'>
              CRISPRa-mediated activation of genes associated with inherited retinal dystrophies in acutely isolated human cells for diagnostic purposes
              </a>
            </td>
          <td>
            Valentin J. Weber, Alice Reschigna, Maximilian-J. Gerhardt, K. S. Hinrichsmeyer, D. Y. Otify, Thomas Heigl, Frank Blaser, Isabelle Meneau, Martin Biel, S. Michalakis, E. Becirovic
          </td>
          <td>2024-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="

 Mantle Cell lymphoma (MCL) is an incurable non-Hodgkin lymphoma (NHL) and has the poorest overall prognoses. Despite intensive therapeutic approaches, the median progression-free survival is 3-5 years. Genetic alterations have been identified in MCL including chromosomal alterations and mutations. However, these changes are not solely responsible for MCL development. Recent studies highlight the biological relevance of human endogenous retroviruses (HERVs), a class of transposable elements (TE), in cancer biogenesis and their potential as novel diagnostic, prognostic, and treatment response biomarkers. However, studies on TE/HERV in lymphoid malignancies are scant and have not been examined in MCL. TEs account for 45% of the human genome, which is broken down into long-interspersed nuclear elements (LINEs), short-interspersed nuclear elements (SINEs), and long-terminal repeats (LTRs). HERV fall into the LTR family and are ancestral remnants of germline retroviral infections, accounting for 8-10% of the human genome.
 To investigate HERV expression in MCL, we performed RNA sequencing (RNA-Seq) in CD19+ B-cells taken from patients with MCL (n=18), and healthy donors (n=10). HERV characterization was conducted by developing an in-house workflow that relied on the telescope, a statistical model to quantify HERVs with a locus-specific resolution. Among the 14,968 HERV genomic loci evaluated in this study, 3345 (22%) were expressed (mean Transcript Per Million (TPM) > 10) across all our MCL samples and, therefore, considered for the differential expression analysis. Such analysis was performed by comparing the MCL samples against the healthy donors. Specifically, a total of 406 and 683 HERVs {(Log2FC) > 0.58 and FDR < 0.05} were found up- and down-regulated in MCL, respectively. From a genomic standpoint, the differentially expressed HERVs were dispersed throughout the genome. However, some chromosomes (chr) such as chr16, chr17, chr19, and chr22, had higher percentages of differentially expressed HERVs calculated by the total number of HERV loci per chromosome. In addition, we identified an increased enrichment of HERV families including HERVL, HERV3, and HERVK in MCL patients vs healthy donors. Among, all the known HERV families, the HERVK family is the most recently integrated into the human genome and the most biologically active in generating retrovirus-like particles. We validated the HERVK RNA-Seq data, by quantitative RT-PCR and confirmed the downregulation of HERVK transcription in 4 MCL cell lines compared with peripheral blood cells from healthy donors. Using RNA-Seq data, we further identified significant downregulation of HERVK subfamilies such as HML2, HML3, and HML-6 in MCL compared to normal control. Mechanistically, HERV transcription is controlled by epigenetic mechanisms including DNA and histone modifications including methylation. We thus interrogated whether DNA methyltransferase inhibitor azacytidine (Aza) or G9a methyltransferase inhibitor (G9ai) controls the HERVK transcription in the MCL cells. We observed that low doses of Aza or G9ai, exhibit variable responses in modulating the transcription of HERVK among the MCL lines. To understand the variability of HERVK upregulation among MCL cells, we examined whether TP53 status affected HERVK expression after in-vitro epigenetic drug treatment. The mutant TP53 MCL cells exhibited significantly less upregulation of HERVK with both epigenetic drugs than the wild-type (wt) TP53 MCL cells. Interestingly, nutlin-3a (MDM2 inhibitor; inhibits MDM2-p53 interaction) treatment significantly increased the HERVK expression in the TP53 wt MCL cells, however, no effect on HERVK expression was seen in TP53 mutant MCL cells. Therapeutically, nutlin-3a was highly sensitive to TP53 wt MCL cells compared to TP53 mutant cells. Combining nutlin-3a with epigenetic drugs Aza or G9ai significantly decreased the proliferation and survival of wt TP53 cells compared to single agents. This is the first study to characterize the expression of HERVs at genomic locus-specific resolution in MCL patients. Our study provides a comprehensive, genome-wide picture of HERV families in MCL along with changes in HERVK after epigenetic treatment and implicates p53 in HERV-K transcriptional regulation. Overall, we provide a novel insight into the biology of HERVs in MCL pathogenesis and epigenetic therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78b062a1bdc8a227394bdd63015402f0ed42d991" target='_blank'>
              Transcriptome Analysis at Locus-Specific Resolution Identified Alteration in Human Endogenous Retroviruses, a Class of Transposable Elements in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Alessandro La Ferlita, Nicholas Staudinger, Lara Bayraktar, Katherine Chiappinelli, L. Alinari, Robert A Baiocchi, L. Sehgal, Mamta Gupta
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="

 Introduction: In recent years, multiple myeloma (MM) research has increasingly focused on unraveling the complex interactions between malignant plasma cells and their surrounding bone marrow microenvironment (BMME). To further elucidate these interactions, we developed a comprehensive single-cell atlas of the MM BMME, integrating both coding and noncoding (ncRNA) transcriptional profiles. By correlating these profiles with specific cytogenetic abnormalities in myeloma cells, we aim to uncover how ncRNAs regulate the phenotypic and functional states of myeloma cells and, consequently, shape the cellular landscape of the BMME. A detailed understanding of how the noncoding transcriptome affects the cellular components and immune landscape of the BMME is essential for advancing our knowledge of MM pathogenesis and identifying novel biomarkers and therapeutic targets.
 Methods: Our atlas was constructed using single-cell RNA sequencing data from 481 samples of CD138neg cells sorted from bone marrow aspirates of MM patients enrolled in the MMRF CoMMpass study. This dataset includes accompanying whole-genome sequencing and survival data, enabling correlative analysis between transcriptional profiles, cytogenetic abnormalities, and patient outcomes. To generate a combined coding and non-coding transcriptome, we developed an expanded human reference genome by systematically merging mRNA and ncRNA transcripts from the LncBook2.0 and GENECODEv42 reference genomes. The sequencing data was aligned to this expanded genome, followed by quality control processing, batch correction, clustering, and supervised analysis.
 Results: The resulting coding and non-coding atlas comprises over 1.9 million cells, spanning immune, plasma, and stromal compartments. Remarkably, ncRNAs constituted 46% of the associated genes, including 22,102 long ncRNAs (lncRNAs) and 3,112 small ncRNAs. Subclustering analysis of the major immune compartments (B lymphoid, myeloid, and NK and T lymphoid) yielded 70 subclusters, with ncRNAs representing 24-38% of the top 100 differentially expressed genes within each compartment. This underscores the significant role of ncRNAs in distinguishing immune subpopulations. Moreover, examining the association between specific cytogenetic abnormalities and the immune composition of the BMME, 30 subclusters were found to be differentially abundant (P < 0.05) between cytogenetic abnormalities, supporting our hypothesis that myeloma cell genotypes significantly shape the immune microenvironment. For instance, patients with CCND1 amplification displayed an increase CD4+ central memory T cells with a decrease in CD8+ cytotoxic T cells. Similarly, patients with 1q21 gain exhibited an increase in TGFβ-stimulated monocytes, while a decrease in cytotoxic CD8+ T and CD56dim NK cells was observed in patients with 17p13 deletion.
 Next, examining which ncRNAs are associated with specific cytogenetic abnormalities, we identified 14 ncRNAs significantly differentially expressed across cytogenetic abnormalities. Increased expression of six of these ncRNAs were associated with poor overall survival both independently and in combination (P < 0.05, HR = 3.7). Notably, three of these ncRNAs have documented roles in cancer progression but have not been previously studied in the context of MM. Subsequently stratifying patients based on their enrichment of these outcome-associated ncRNAs revealed even stronger associations with immune composition, including an increase in CD4+ regulatory T cells (P < 0.05). This further supports our hypothesis that myeloma cell genotypes shape the BMME. Lastly, by constructing a gene regulatory network to predict the interactions between coding and noncoding RNAs, we illustrate the regulatory networks of ncRNAs modulating the phenotypes of myeloma and immune cell subpopulations.
 Conclusion: In summary, we present the first high-resolution transcriptomic atlas of the MM BMME, integrating both coding and noncoding RNAs. This study reveals distinct immune subpopulations associated with specific myeloma cell genotypes and highlights the central role of ncRNAs in modulating the phenotypic and functional states of both malignant and non-malignant cells in the MM BMME. Our findings emphasize the potential of ncRNAs as therapeutic targets and prognostic markers for MM, offering new avenues for research and clinical intervention.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/025f968fdac3558272e78b3c11f8f1b928e52e45" target='_blank'>
              Integrated Coding and Non-Coding Transcriptional Single-Cell Atlas of Multiple Myeloma: Unraveling the Effects of Myeloma Genotype on the Bone Marrow Microenvironment
              </a>
            </td>
          <td>
            Marina E. Michaud, Sarthak Satpathy, William Pilcher, Edgar Gonzalez-Kozlova, D. Karagkouni, Lijun Yao, Yered H Pita-Juárez, Chaitanya Acharya, Mark Hamilton, Shivani Nanda, Yizhe Song, Julia T. Wang, Mojtaba Bakhtiari, D. Avigan, T. Kourelis, M. Dhodapkar, S. Lonial, Swati S. Bhasin, Immune Network, Li Ding, Sacha Gnjatic, Ioannis S Vlachos, George Mulligan, Manoj K. Bhasin
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707a9b434c06af6bee59b88bf04f6b542c8af9a4" target='_blank'>
              Advancing the Battle against Cystic Fibrosis: Stem Cell and Gene Therapy Insights.
              </a>
            </td>
          <td>
            Disha D Shah, Mehul R. Chorawala, Aanshi J. Pandya, Nirjari R Kothari, Bhupendra G. Prajapati, Priyajeet S. Parekh
          </td>
          <td>2024-12-16</td>
          <td>Current medical science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genome-wide Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) is a powerful tool for screening essential genes (EGs) and studying gene function by systematically knocking out individual genes. EGs are essential for the survival of organisms and can be divided into core EGs (CEGs) and context-specific EGs, which are crucial for the development of drugs. Although a variety of CEGs have been identified using knockout technology, the concordance among these CEG sets is extremely low; there is a lack of studies on the EG mechanisms. Therefore, developing systematic methods to identify CEGs and reveal corresponding mechanisms are important biological issues. To address these issues, we propose a comprehensive ensembled-based model utilizing gene community-regulated pathways to decipher pan-cancer CEGs and context-specific EGs across 29 cancer types and provide insights into their regulatory abilities for each pathway. The project aims include developing a model with Systematic Identification of Essential Gene (SIEG) scores for CEGs and Context-Specific Enrichment (CSE) scores for context-specific EGs. Subsequently, we assess the regulated pathways and mechanisms of these identified EGs by integrating diverse data sources such as genome-wide CRISPR/Cas9 knockout screens, multiple omics data, KEGG pathways, and Gene Ontology. Ultimately, we aim to establish a user-friendly web service. In our preliminary results, we gathered 1,845 genome-wide CRISPR datasets and various omics datasets, including 8,941 clinical samples and 1,346 cell lines, leading to the identification of 3,213 CEGs. By employing the SIEG score, we found that 1,178 of these pan-cancer CEGs overlapped with previously defined CEGs, reflecting a 60% similarity rate. These 3,213 CEGs play crucial roles in regulating pathways associated with cell viability, expansion, and proliferation. Additionally, they exhibit characteristics typical of CEGs, showing less favorable as therapeutic targets and centralizing within protein-protein interaction networks. Moreover, we delineated six pathway signatures of pan-cancer CEGs, encompassing transcription, translation, protein folding, replication and repair, cell growth and death, as well as energy. We anticipate that these signatures will contribute to the future redefinition of CEGs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bb4c1d3a0b27d1d29e35ffe8c13d2ec1d355849" target='_blank'>
              A single-gene-based AI model to identify core and context-specific essential genes by biological interpretation from pooled genome-wide CRISPR and omics data
              </a>
            </td>
          <td>
            Chih-Yuan Chou, Jung-Yu Lee, Chia-Hwa Lee, Jinn-Moon Yang
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Oncogene-driven non-small-cell lung cancer (NSCLC) is typically treated with targeted therapies to inhibit oncogene downstream signaling pathways and reduce tumor survival. The most common subtypes, EGFR and KRAS mutations, are amenable to these therapies; however, resistance often develops, leading to oncogene-independent metastases. This study explores non-oncogene addiction (NOA) as a novel strategy, targeting essential genes like ATR, which is involved in DNA damage response, and pyruvate dehydrogenase kinases (PDKs), which play a role in energy metabolism. Experiments were conducted using sensitive PC9 and the corresponding osimertinib-resistant cells (PC9/OR), namely EGFR-mutant H1975 and KRAS-mutant A549 cells treated with TKIs, alongside ATR and DCA inhibitors. The results indicated that combining these approaches could enhance efficacy compared to TKIs alone, suggesting a tailored strategy based on tumor subtype. This research underscores the potential of new therapeutic targets to improve treatment outcomes in patients with NSCLC compared to traditional TKI therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f046003a233e91305d9f9f261ac2302b860f60ce" target='_blank'>
              Combined Therapeutic Strategies Based on the Inhibition of Non-Oncogene Addiction to Improve Tumor Response in EGFR- and KRAS-Mutant Non-Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Luisa Amato, Daniela Omodei, Caterina De Rosa, Annalisa Ariano, S. Capaldo, Camilla Carmela Tufano, Rossella Buono, Cristina Terlizzi, Anna Nardelli, Vitale Del Vecchio, Rosanna Palumbo, C. Tuccillo, F. Morgillo, Federica Papaccio, Virginia Tirino, F. Iommelli, C. D. Della Corte, Viviana De Rosa
          </td>
          <td>2024-11-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="

 Backgrounds: Therapy resistance and recurrent relapses continue to be significant challenge in the management of mantle cell lymphoma (MCL), which cannot be explained by genetic alone. Discovering drugs that can successfully correct epigenetic changes is a promising strategy to address the inadequate therapeutic response in MCL. Epigenetic regulator ubiquitin-like containing PHD and RING finger domains 1 (UHRF1) is aberrantly expressed in many human diseases and correlated with disease progression, but its function in MCL remains unclear.
 Methods: Through integrative analyses of the gene-expression profile and clinical parameters, UHRF1 was screened as a potential epigenetic therapy target in MCL patients. The pro-tumor function of UHRF1 was demonstrated by in vitro and in vivo experiment. The anti-tumor effect and safety of CM272, a functional inhibitor of UHRF1, was assessed in MCL xenograft animal model.
 Results: UHRF1 was highly expressed and positively correlated with Ki67 expression in MCL tissues samples. UHRF1-high expression predicted poor prognosis in MCL patients. Knockdown of UHRF1 inhibited MCL cell proliferation, migration and invasion whereas induced MCL cell apoptosis and cell cycle arrest. Administration of mice with CM272 significantly inhibited MCL progression and no obvious treatment-related side effects in liver and renal.
 Conclusion: Our study demonstrated that UHRF1 signaling involved in MCL tumorigenesis and progression, and provides a novel insight for therapeutic interventions in MCL by targeting UHRF1.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58be24af157be879979b457871d8d34e43d552f" target='_blank'>
              UHRF1 Is a Promising Therapy Target in Aggressive Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Yuan Yang, Ping Yang, Weilong Zhang, Jing Wang, Yingtong Chen, Chunyuan Li, Hongmei Jing
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="

 The MECOM locus (located at 3q26.2) encodes for the multi-zinc finger domain transcription factor EVI1, which regulates normal hematopoietic stem cell maintenance. EVI1 targets include other major regulators such as ERG and MYC. Aberrant EVI1 overexpression is observed in ~10% of de novo AML, of which approximately half are due to the chromosomal rearrangement involving either inv(3)(q21;q26.2) or t(3;3)(q21;q26.2). This chromosomal rearrangement (3q26.2-r) results in relocation of the distal GATA2 hematopoietic enhancer to EVI1 promoter, which now drives EVI1 expression. In 3q26.2-r AML, co-mutations in addition to GATA2 repression with increased EVI1 activity promote AML progression. This is associated with an aggressive phenotype that is relatively refractory to the available treatment options. Therefore, there is need to develop and test novel targeted therapies with improved efficacy in repressing EVI1 and improving survival of AML patients with EVI1 overexpression.
 We used two approaches to identify druggable vulnerabilities in 3q26.2-r AML. We used an unbiased high-throughput drug screen focused on mechanistically annotated drugs (NCATS Mechanism Interrogation Plates [MIPE 5.0] and domain-specific epigenetic-targeted CRISPR screen. In both screens, BRD4 was identified as an absolute dependency in the 3q26.2-r AMLs. This was consistent with our previous report showing that BET inhibitors (BETi) are effective against 3q26.2-r AML cell lines, patient-derived (PD) AML cells and their PDX models (Leukemia 2024; 38: 545-556). In addition to BRD4, the CRISPR screen identified KDM1A (LSD1), p300, and HDAC3 as dependencies in a 3q26.2-r AML cell line (UCSD-AML1), which was further evaluated here. Treatment of 3q26.2-r AML cell lines for 96-hours with ORY001 (LSD1i; 3-100 nM), RGFP966 (HDAC3i; 0.5-10 µM) or GNE-781 (CBP/p300i; 10-1000 nM) induced moderate, dose-dependent cell death, as determined by flow cytometry. However, all three agents, especially LSD1i treatment, caused a marked increase in CD11b and CD86 double-positive cells (50-80%), suggesting that these treatments also induced differentiation of 3q26.2-r AML cell lines. Following these treatments, Western analysis of 3q26.2-r AML cell lines showed a decrease in phosphorylated S6 and EVI1 expressions and an increase in CD11b protein levels. Additionally, the high-throughput NCATS MIPE 5.0 screen identified mTOR, PIK3CA, and XIAP as druggable vulnerabilities in 3q26.2-r AML cells. In follow-up experiments, mivebresib (BETi) dactolisib (BEZ235, mTOR/PI3Ki) and LCL161 (IAPi) induced dose-dependent apoptosis in 3q26.2-r cell lines. In PD AML samples, treatment with mivebresib and dactolisib for 48-hours reduced viability to a greater extent in 3q26.2-r AML, compared to non-3q26.2-r AMLs cells (p < 0.01). Tandem-tag mass spectrometry of a 3q26.2-r AML cell line with gene-set enrichment analysis (GSEA) demonstrated that treatment with mivebresib, dactolisib or LCL161 for 16 hours downregulated MYC-targets gene-set. Mivebresib and dactolisib-treated cells also showed downregulation of cell-cycle related gene-set. These findings were further corroborated by qRT-PCR, Western blot and cell-cycle analyses. Combination of mivebresib (50-250 nM) with dactolisib (50-500 nM) or LCL161 (30-1000 nM) synergistically induced in vitro apoptosis in 3q26.2-r AML cells (delta synergy scores > 1.0). Treatment of the luciferized 3q26.2-r PD AML242 model, engrafted in NSG mice, with mivebresib (0.5 mg/kg, daily 5x per week), dactolisib (20 mg/kg, daily 5x per week) or LCL161 (65 mg/kg, daily 5x per week) reduced AML burden and increased survival compared to vehicle-treated mice (p < 0.05). Importantly, in a separate luciferized 3q26.2-r AML194 PDX model, the combinations also reduced the in vivo AML burden and increased survival of the mice, as compared to treatment with the vehicle control or with single agent alone (p < 0.05). Collectively, these findings show that both CRISPR and chemical screens identified novel druggable targets, as well as helped develop treatments with their targeted agents. These treatments resulted in reduction of EVI1 and c-Myc levels, associated with preclinical differentiation and apoptotic activity in the 3q26.2-r AML cell models. Further in vivo evaluation of the efficacy of BETi or LSD1i-based combinations against 3q26.2-r AML is warranted.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5faf2caa4a5ab0b9f6bef886fb37649a2ce6a715" target='_blank'>
              Efficacy of Combinations of Targeted Agents Against Dependencies in Mecom Rearranged AML Identified By Unbiased Chemical and CRISPR Screens
              </a>
            </td>
          <td>
            Christine E. Birdwell, W. Fiskus, Christopher P. Mill, Surbhi Sharma, T. Kadia, N. Daver, C. Dinardo, K. Sasaki, John A. Davis, Kaberi Das, Hanxi Hou, M. Ceribelli, K. Bhalla
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81af71d49f35f4ca08b391cf52780be4b4e726a9" target='_blank'>
              Integrated single-cell analysis reveals distinct epigenetic-regulated cancer cell states and a heterogeneity-guided core signature in tamoxifen-resistant breast cancer
              </a>
            </td>
          <td>
            Kun Fang, A. Ohihoin, Tianxiang Liu, Lavanya Choppavarapu, Bakhtiyor Nosirov, Qianben Wang, Xue-Zhong Yu, S. Kamaraju, Gustavo W. Leone, Victor X Jin
          </td>
          <td>2024-11-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d04c42082a8c74f3b67d96a365b6e54394480b2" target='_blank'>
              Leveraging homologous recombination deficiency for sarcoma : Unravelling homologous recombination repair deficiency and therapeutic opportunities in soft tissue and bone sarcoma.
              </a>
            </td>
          <td>
            Lara Planas-Paz, Chantal Pauli
          </td>
          <td>2024-11-13</td>
          <td>Pathologie</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Drug resistance remains a formidable challenge in MCL, largely due to unrestrained proliferation of MCL cells driven by aberrant Cyclin D1 and CDK4 expression. In preclinical studies, inhibition of CDK4/6 not only induces early G1 cell cycle arrest but also reprograms MCL cells for therapy vulnerability, suggesting that CDK4/6 inhibition may deepen and prolong the clinical response to BTK inhibition. We tested this hypothesis in a phase I clinical trial for recurrent MCL and defined the resistant mechanism by integrated longitudinal genomic analysis of individual patients.
 Palbociclib (CDK4/6i) was administered on days 1-21 of a 28-day cycle; ibrutinib (BTKi) was given continuously. For longitudinal analysis, sequential tissue and blood specimens from 27 evaluable patients were collected before and during therapy, and on progression. Single cell RNA-seq (scRNA-seq) analysis of PBMCs or the monocytic fractions from bone marrow and lymph node (53 samples) was performed using a MCL-specific in-house reference library. The data were then integrated with whole transcriptome (WTS) and whole exome sequencing (WES) of MCL cells isolated from the same specimens, flowcytometry, immunoblotting, IHC, and CBC /differential to deduce the number of total MCL and immune cells in each specimen.
 CDK4/6 Inhibition appears to deepen and prolong the BTKi response, with a CR rate of 42% and 5 patients (2 CR and 3 PR) remained on therapy for ~ 10 years. Longitudinal scRNA-seq (210,000 cells) revealed that MCL cells comprise 4 major transcriptomically distinct clusters with cluster 1 (C1) resembling quiescent normal B cells. RNA velocity analysis further indicated that both C2 and C3 are in late G1, having evaded CDK4/6 inhibition; but only C2 progresses to the proliferating C4. C3 is terminally arrested in late G1 and long-lived with elevated BCL2 and IRF4 expression.
 In durable CR, all MCL clusters were depleted as expected. Primary resistance and progression on therapy correlated with a marked expansion of C2 or C3 MCL cells. Integrated longitudinal scRNA-seq analysis with WES and WTS revealed that C3 resistance was due to RB1+/del CDKN2A+/del (hemizygous loss of RB1 and CDKN2A), whereas C2 resistance was driven by CDK4amp-RB1+/del-CDKN2A+/del. Thus, compound copy number variation (cCNV) drives differential resistance to CDK4/6i and BTKi, reinforcing the critical importance of controlling the CDK4/6-RB-CDKN2A axis in MCL therapy.
 Durable CR was also associated with maintenance of CD8+ T cells and CD8+ T effector memory (TEM) cells. Conversely, progression on therapy was accompanied by rapid depletion of CD8+TEMs and proliferating (Ki67+) CD8+ T cell expressing all exhausting markers including LAG3,TIGIT and TIM3. Collectively, our data suggest that CD8+ T cell surveillance cooperates with MCL cell intrinsic cCNV to discriminate durable clinical response from resistance to targeting CDK4/6 and BTK.
 Moreover, IL10 was produced in bone marrow MCL cells before therapy regardless of the subsequent clinical response, but downregulated in early G1-arrested in vivo in a responding patient. Plasma IL10 was selectively elevated in MCL patients on progression from CR, along with sustained IL10 synthesis in myeloid cells. These novel findings suggest that IL10 promotes resistance to CDK4/6i and BTKi and a new strategy to overcome resistance. Indeed, exogenous IL10 completely abolished the killing of MCL cells from a PR patient by CDK4/6i+ BTKi and CDK4/6i+ PI3Ki ex vivo. IL10 also abrogated cCNV-guided killing of MCL cells C3 - and C2-resistant patients by BTKi+BCL2i, and C3-resistant patients by CDK2i+BCL2i ex vivo. All therapies were restored by disrupting IL10 signaling.
 In summary, by integrated longitudinal scRNA-seq analysis of a hypothesis-driven clinical trial, we have provided the first evidence in humans that 1) CDK4/6 inhibition deepens and prolongs the clinical response to BTKi; 2) cCNV of CDK4-RB1-CDKN2A drives differential expansion of resistant C2 MCL cells in transit to proliferating C4 MCL cells or long-lived non-proliferating C3 MCL cells; 3) CD8+ T cell maintenance and surveillance cooperates with MCL cell intrinsic cCNV to discriminate durable response from resistance; and 4) IL10 is a key mediator of MCL-Immune cell interaction that promotes resistance. These discoveries have profound implications for genome-guided strategy to overcome drug resistance in MCL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6f11aff634bc493f3148448673ebdfd3f7247b" target='_blank'>
              Disrupting IL10 Signaling Overcomes Compound Copy Number Variation-Driven Clinical Resistance to CDK4/6 and BTK Inhibition in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            M. Di Liberto, Yang Hu, Xiangao Huang, G. Inghirami, Kevin Wang, Christina Lee, K. Maddocks, K. Blum, J. Ruan, John P Leonard, Nancy L. Bartlett, Peter Martin, O. Elemento, S. Chen‐Kiang
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract DNA damage response (DDR) pathway is the coordinated cellular network dealing with the identification, signaling, and repair of DNA damage. It tightly regulates cell cycle progression and promotes DNA repair to minimize DNA damage to daughter cells. Key proteins involved in DDR are frequently mutated/inactivated in human cancers and promote genomic instability, a recognized hallmark of cancer. Besides being an intrinsic property of tumors, DDR also represents a unique therapeutic opportunity. Indeed, inhibition of DDR is expected to delay repair, causing persistent unrepaired breaks, to interfere with cell cycle progression, and to sensitize cancer cells to several DNA‐damaging agents, such as radiotherapy and chemotherapy. In addition, DDR defects in cancer cells have been shown to render these cells more dependent on the remaining pathways, which could be targeted very specifically (synthetic lethal approach). Research over the past two decades has led to the synthesis and testing of hundreds of small inhibitors against key DDR proteins, some of which have shown antitumor activity in human cancers. In parallel, the search for synthetic lethality interaction is broadening the use of DDR inhibitors. In this review, we discuss the state‐of‐art of ataxia‐telangiectasia mutated, ataxia‐telangiectasia‐and‐Rad3‐related protein, checkpoint kinase 1, Wee1 and Polθ inhibitors, highlighting the results obtained in the ongoing clinical trials both in monotherapy and in combination with chemotherapy and radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3effb948dd60b6e28e5a1e98f095e523b94a901" target='_blank'>
              Targeting the DNA damage response in cancer
              </a>
            </td>
          <td>
            Guffanti Federica, Chiappa Michela, Damia Giovanna
          </td>
          <td>2024-10-31</td>
          <td>MedComm</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e0e1c51f3a6b94b7b0e1a929b6b55f0369b955a" target='_blank'>
              A novel mouse model recapitulating the MMR-defective SCLC subtype uncovers an actionable sensitivity to immune checkpoint blockade
              </a>
            </td>
          <td>
            O. Ibruli, France Rose, F. Beleggia, A. Schmitt, Maria Cartolano, L. T. Fernandez, J. Saggau, D. Bonasera, M. Kiljan, Gokcen Gozum, Luca Lichius, Jiali Cai, Li-Na Niu, Manoela Iannicelli Caiaffa, Jan M Herter, H. Walczak, G. Liccardi, Holger Grüll, Reinhard Büttner, Graziella Bosco, Julie George, Roman K Thomas, Kasia Bozek, H. Christian Reinhardt, G. Herter-Sprie
          </td>
          <td>2024-11-14</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 Introduction: Despite recent innovations in thetreatment ofacute myeloid leukemia (AML), adverse risk AML continues to be associated with poor response rates and short-term survival. Mutations and epigenetic alterations are key drivers in the resistance to conventional treatment and progression of AML. In this landscape, alteration/mutation of TP53 is accompanied with poor prognosis and has been recently reported to confer resistance to CAR-T cell therapy. TP53 represents a promising target to overcome resistance to T cell-based therapies. Epigenetic modifiers, in particular hypomethylation drugs that target DNA methyltransferase (DNMT), have emerged as another promising approach, and have received FDA approval. CM-272 is a novel reversible and selective dual inhibitor of DNMT and Histone Methyltransferase (HMT) G9a with anti-tumor properties in AML cells in vitro. However, the potential role of this DNMT/HMT complex towards tumor outgrowth and immune evasion across the AML mutational spectrum remains to be fully determined. Here, we explored the therapeutic combination of CM-272 treatment with adoptive T cell immunotherapy as a synergistic approach to overcome immune resistance of AML.
 Methods: The NPM1mut OCI-AML3 cell line (expressing Luciferase) was used as a model to study the effect of CM-272 in in vitro coculture with T cells. RNA sequencing (RNA-seq) and quantitative proteomics were used to assess the effect of CM-272 treatment in OCI-AML3. Changes in the heterogeneity of peptides bound to MHC class I (MHC-I) induced by CM-272 were analyzed by immunopeptidomics. Human T cells were isolated from healthy donor buffy coats and retrovirally transduced with an HLA-A2.1-restricted p53TCR. Luminescence-based cytotoxicity assays were used to evaluate the combination of antigen-specific T cell killing and CM-272. The effect of CM-272 on the effectiveness of antigen-specific T cells was assessed by flow cytometry focusing on the T cells viability, proliferation, changes in differentiation/exhaustion markers and induction of apoptosis.
 Results: Gene expression (RNA-seq) analysis of the NPM1mut OCI-AML3 cell line demonstrated that CM-272 induces up-regulation of the TP53 gene signature. These transcriptional changes were confirmed by p21 induction, a downstream target of TP53. Combining CM-272 with p53 antigen-TCR-specific T cells resulted in improved leukemic cell elimination compared to CM-272 or T cell monotherapies. Furthermore, CM-272 treatment induced upregulation of HLA-A*02:01 and the co-stimulatory molecule CD86 on OCI-AML3 cells while inducing cell cycle arrest and apoptosis. CM-272 did not affect T cell viability and effector functions. Quantitative proteomics confirmed the upregulation of HLA-A, and immunopeptidomics detected an increase in the number of HLA-A*02:01-presented peptides on CM-272 treated OCI-AML3 cells. Importantly, this was associated with a significant increase of the HLA-A*02:01-bound p53-epitope. Globally, these results suggest that increased expression of HLA-A*02:01/p53 complex along with the co-stimulatory signal CD86 upon CM-272 treatment account for the increased antigen-specific T cell effectiveness.
 Conclusion: Our study provides first evidence of increased human T cell effector functions upon combination with a reversible epigenetic drug and a proof of principle demonstration for this novel combinatorial treatment for AML. The combinatorial epigenetic/TCR therapeutic approach lays the groundwork for further research into overcoming immune resistance in AML therapy. The immunopeptidomics results revealed new targetable tumor-associated epitopes and already provide us with predicted binding affinities to evaluate the potential of new antigen-specific T cell therapies. Deep characterization of the MHC-I bound peptides will be extended to patient-derived AML samples and validated in xenograft mouse models.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da141713a828a16c83b52f19e2641eb90b0d87" target='_blank'>
              Dual Inhibition of Histone and DNA Methylation in AML Enhances the Effectiveness of TCR-Based Cellular Therapy through Upregulation of Surface MHC Class I/p53 Antigen Complex
              </a>
            </td>
          <td>
            Ulrike Regina Schmits, Edite Antunes Ferreira, B. Guezguez, David Gomez-Zepeda, Stefan Tenzer, K. Legscha, Matthias Theobald, H. Echchannaoui
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="

 Age-related clonal hematopoiesis (ARCH) is associated with aging and can easily lead to malignant hematological or cardiovascular disease. DNA (cytosine-5)-methyltransferase 3A (DNMT3A) fulfils a variety of different functions during the aging process and is frequently mutated in ARCH and subsequent hematological malignancies. However, the specific molecular mechanism by which DNMT3A influences ARCH is unclear. This project first confirmed that DNMT3A mutations in hematopoietic stem and progenitor cells (HSPCs) were present in clinical samples from a large cohort of healthy volunteers. Secondly, we constructed a Dnmt3a R878H knock-in mouse model and found that Dnmt3a R878H mutation could induce hematopoietic cloning abnormalities in mice over a year later, mainly manifested in the active proliferation of HSPCs, deviations in the functional differentiation trajectory, and micro-environment disorder of immunity. Then, we applied high-throughput chromosomal capture (Hi-C) and single-cell RNA sequencing to explore the effects of aging-associated DNMT3A mutations in HSPCs, together with chromosome spatial structure change and transcriptional regulation and an in-depth investigation of the downstream molecular effects of the DNMT3A mutation on ARCH and the regulation of homeostasis and associated factors in immune cells, providing a new foundation for the early diagnosis and prevention of blood-related diseases induced by clonal hematopoiesis.
 Single-cell sequencing has suggested the existence of significant heterogeneity among hematopoietic stem cells, and that Dnmt3a mutations can regulate histone methylation modifications in hematopoietic stem and progenitor cells. Hi-C sequencing has revealed that in hematopoietic stem and progenitor cells, the overall chromatin conformation undergoes relatively subtle changes. However, closer examination of the inter-chromosomal interactions shows that the primary alterations occur between chromosomes 12 and 19. Utilizing advanced bioinformatic techniques to analyze differential Topologically Associating Domains (TADs) and chromatin loops, we identified significant differences in the spatial organization strength located specifically on chromosome 19. This finding suggests that, despite the overall stability of the chromatin landscape, specific regions such as those on chromosome 19 undergo more pronounced structural rearrangements, potentially impacting gene regulation and cellular function in these critical hematopoietic cell populations.
 Combining the preliminary Hi-C analysis results from ARCH mice, we identified Apobec3 as one of the significantly dysregulated genes following Dnmt3a mutation. Given the established roles of Apobec3 in DNA repair and RNA editing, our RNA sequencing GSEA analysis in mice suggests that the DNA damage repair pathway is markedly enhanced in Dnmt3a-mutated hematopoietic stem and progenitor cells as well as granulomonocytic cells. This indicates that Dnmt3a mutation may induce increased DNA damage, subsequently activating the DNA repair pathway. Within the DNA repair-related gene clusters, we observed that the expression of Atm gene and protein is significantly lower in mutant mice compared to wild-type mice, suggesting a partial blockade of cellular repair function in mutant mice.
 Further validation using patient bone marrow cells and 293 cells overexpressing mutated DNMT3A and APOBEC3F revealed that, among the DNA repair-related gene clusters, all genes except ATM showed increased expression, while ATM expression was notably decreased. Additionally, comet assay studies demonstrated a pronounced tailing phenomenon in cells harboring the mutation. Exome sequencing of mice revealed that with enhanced Apobec gene expression in mutant mice, there was a significant increase in the number of copy number variations (CNVs) and single nucleotide variations (SNVs).
 Collectively, these results suggest that Dnmt3a mutation can induce DNA/RNA damage through Apobec3, leading to secondary genetic mutations by interfering with the cellular repair machinery. The compromised repair capacity in turn facilitates the accumulation of additional genetic alterations, underscoring the complex interplay between epigenetic regulators, DNA damage, and repair pathways in the context of hematopoietic malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f2457ef93db3d746327516815011d615bb6ce3" target='_blank'>
              Exploring the Mechanism of Chromatin Spatial Structure Changes and Transcriptional Regulation in Age-Related Clonal Hematopoiesis Caused By DNMT3A Mutations Using Chromatin Capture Technology
              </a>
            </td>
          <td>
            Yujun Dai, Xue-Ping Li, Yang Liang
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 T-cell acute lymphoblastic leukemia (T-ALL) is a highly aggressive blood cancer marked by the abnormal growth of immature thymocytes. While the survival rate in pediatric cases is around 80%, there remains a significant 20% of patients who succumb to the disease due to relapse or resistance to treatment. Moreover, adult survival rates are below 50% due to higher treatment-related toxicities. This highlights the critical need for new therapeutic approaches. At the molecular level, constitutive NOTCH1 signaling, resulting from activating mutations in NOTCH1 gene observed in over 60% of patients, is a major driver of the disease. In our recent work, we had identified an extensive oncogenic NOTCH1 dependent rewiring of global tRNA biogenesis in T-ALL and demonstrated that tRNA deregulation are critical contributors to the disease pathogenesis of T-ALL. In further underlining tRNA pathway as novel therapeutic targets in T-ALL, we report pharmacological inhibition of prolyl-tRNA synthetase (ProRS) domain, that charges proline tRNAs with the cognate proline amino acid, to decrease proliferation and induces cell death in pre-clinical models of T-ALL. Expression of the bifunctionl glutamyl-prolyl-tRNA synthetase (EPRS1) and proline tRNAs is significantly upregulated in primary T-ALL samples relative to thymocytes from healthy individuals. Moreover, CRISPR knockdown of EPRS1 and proline tRNAs in T-ALL cell lines leads to reduced cell fitness, underscoring the critical role of this pathway in tumor proliferation and survival. NCP26 is a new pyrazinamide-based drug that targets the ProRS domain of EPRS1 and its efficacy in treating multiple myeloma was thoroughly evaluated in a recent study. We show that NCP26 effectively inhibits proliferation and enhances apoptosis in T-ALL cell lines and PDX samples. In contrast, T cells from healthy donors show no signs of apoptosis and exhibit only minimal proliferation inhibition, confirming the lack of toxicity on normal T cells. Our findings identify upregulated proline tRNA biogenesis as a critical adaptation in T-ALL pathogenesis and provide a molecular basis to target ProRS in T-ALL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc99bee96b3fce9747cd66255db512e1b82cad8f" target='_blank'>
              Targeting Proline Trna Biogenesis for Treatment of T Cell Acute
              </a>
            </td>
          <td>
            Valerio Ciaurro, Priyanshu Srivastava, Abdul Khalique, Jared Brottem, N. Payne, R. Mazitschek, Palaniraja Thandapani
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1f5a7ee68cb6b5638219d48a2b0e5c5d417966" target='_blank'>
              A complex systems approach to mosaic loss of the Y chromosome.
              </a>
            </td>
          <td>
            Xihan Guo, Xueqin Dai
          </td>
          <td>2024-12-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="

 Introduction: Fludarabine, a purine nucleotide analogue, is integral to preparative regimens for CAR-T cell therapy and allo-SCT in treating B-cell pediatric acute lymphoblastic leukemia (B-ALL). By disrupting DNA synthesis and repair, fludarabine induces apoptosis in proliferating leukemia cells. It also plays a vital role in immunosuppression, lymphodepletion, and tumor burden reduction, optimizing the efficacy of subsequent therapies. Despite its critical role, the molecular impact of fludarabine on leukemia cells and its efficacy factors remain inadequately understood.
 Methods: We utilized a cutting-edge single-cell spatial proteomics assay, Molecular Pixelation (MPX, Karlsson et al. Nat Methods 2024), an optics-free DNA sequencing-based method to analyze the cell surface interactome in response to fludarabine. The glucocorticoid-resistant ETV6::RUNX1 cell line REH and the BCR::ABL1 cell line SUP-B15 were incubated with and without fludarabine for 72 hours at varying concentrations, with repeated technical repetitions. Approximately 1,000 surviving single cells per condition were analyzed using the MPX assay (Pixelgen Technologies), wherein antibody-oligonucleotide conjugates generated nanometer-sized molecular pixels, read out by sequencing on a NovaSeq X Plus instrument (Illumina). Spatial proteomics networks for 76 proteins were computationally reconstructed from DNA-sequencing reads by inferring the relative locations of antibody-oligonucleotide conjugates, creating ~1,000 spatially connected neighborhoods per cell. Orthogonal immunocytochemistry (ICC) data were generated to validate the top protein interactions.
 Results: We have previously shown that ex-vivo drug screening in combination with single-cell transcriptome sequencing (scRNA-seq) readout can elucidate transcriptional effects of fludarabine (Gezelius et al. NAR Genomics and Bioinformatics 2024). This approach confirmed fludarabine's disruption of DNA synthesis and repair in dividing cells, and in addition revealed previously unknown extensive dysregulation of transcripts encoding cell surface proteins. The scRNA-seq data do not adequately resolve the spatial nor functional dynamics of the cell surface proteome, leaving this important aspect of response to fludarabine unexplored. In this study, we take advantage of recent advancements in single-cell spatial proteomics profiling to spatially elucidate how the cell surface proteome organization and function is modulated during fludarabine treatment.
 Spatial statistical analysis revealed novel patterns in cell surface organization during fludarabine treatment. The MPX assay identified 27 proteins with significantly altered abundance post-treatment (adj. Wilcoxon p < 0.01), consistent with the increased abundance of cell-surface proteins observed in our previous scRNA-seq data. Additionally, 26 proteins exhibited changes in polarity (adj. Wilcoxon p < 0.01) after fludarabine treatment, with most proteins shifting from a dispersed to a polarized state. We also detected 138 protein pairs with modified co-localization patterns. Notably, fludarabine enhanced the abundance, polarization, and co-localization of cadherin proteins CD82 and CD53, suggesting a potential therapeutic scaffold. This finding was further validated through orthogonal ICC experiments.
 Conclusions: This study advances our understanding of the spatial organization of the cell surface proteome in leukemia cells following fludarabine exposure. The findings provide new insights into fludarabine's effects at the single-cell level and may inform the development of more effective targeted therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df1ba5108871a33c10f05770da635143d8f7eec" target='_blank'>
              Mapping the Spatial Proteome of Individual Leukemia Cells Undergoing Fludarabine Treatment
              </a>
            </td>
          <td>
            J. Nordlund, M. Globisch, Henrik Gezelius, Anna Pia Enblad, O. Krali, Mariya Lysenkova-Wiklander, Martin Åberg, Arja Harila, A. Raine, Claes Andersson
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d11267e3052403e1a0f2c358857a352b19807b" target='_blank'>
              RAS-mutant leukaemia stem cells drive clinical resistance to venetoclax
              </a>
            </td>
          <td>
            Junya Sango, Saul Carcamo, M. Sirenko, Abhishek Maiti, Hager Mansour, Gulay Ulukaya, Lewis E Tomalin, Nataly Cruz-Rodriguez, Tiansu Wang, Malgorzata Olszewska, Emmanuel Olivier, Manon Jaud, B. Nadorp, B. Kroger, Feng Hu, Lewis Silverman, Stephen S. Chung, Elvin Wagenblast, R. Chaligné, Ann-Kathrin Eisfeld, Deniz Demircioglu, D. Landau, P. Lito, E. Papaemmanuil, Courtney D. DiNardo, Dan Hasson, Marina Y Konopleva, E. Papapetrou
          </td>
          <td>2024-10-30</td>
          <td>Nature</td>
          <td>1</td>
          <td>76</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is resistant to many therapies including immunotherapies. There is an urgent need to explore the role of the microenvironment in resistance to therapy with tailored pre-clinical models. Using multi-omic approaches we describe a syngeneic cancer stem cell mouse model of GBM, with a spontaneous amplification of Igf2. We investigate whether Igf2 influences tumour and microenvironment cells to promote immunosuppression in GBM. Three cell lines were previously established from spontaneous brain tumours in C57Bl/6 Trp53+/-/Nf1+/- mice and maintained under neural stem cell culture conditions. Whole genome sequencing (WGS) was used to determine single nucleotide, copy number, and structural variation, revealing loss of both copies of Trp53 and Nf1 in all three cell lines. However only one cell line, mBT0309, developed tumors when orthotopically allografted. Analysis of genomic alterations and transcriptomics revealed that mBT309 exhibits amplification of the Igf2 loci and overexpression was confirmed at the RNA and protein level. Spatial and single cell transcriptomics showed that Igf2 is overexpressed in mBT0309 allografted tumours and correlated with specific transcriptomic programs. A high-parameter Imaging Mass Cytometry (IMCTM) panel was used for spatial proteomic analysis, to monitor the development of tumours in a time-course experiment. In vitro growth characteristics and stem marker expression in both transcriptomic and spatial IMC data suggest that high levels of Igf2 may regulate GBM stemness features. Analysis of human GBM datasets revealed that Igf2 amplified tumours have reduced CXCL11 expression. These findings suggest that high levels of Igf2 may influence immunosuppression by reducing T-cell recruitment. This syngeneic immunocompetent GBM stem cell model harbouring Igf2 amplification adds to the suite of models to study the GBM microenvironment and its role in immunosuppression and resistance to therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdba6096b1411d2cfa3446430b4c3e22ff1f38c0" target='_blank'>
              IMMU-36. MULTI-OMIC CHARACTERIZATION OF A SYNGENEIC MOUSE CANCER STEM CELL MODEL OF GBM FOR PRECLINICAL APPLICATIONS
              </a>
            </td>
          <td>
            Kyle Heemskerk, Samir Assaf, Xiaguang Hao, Mathieu Meode, Smriti Kala, James Pemberton, Reza Mirzaei, J. Cairncross, Peter Forsyth, V. Yong, Samuel Weiss, F. Zemp, H. Artee Luchman
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Motivation Genomic biotechnology has rapidly advanced, allowing for the inference and modification of genetic and epigenetic information at the single-cell level. While these tools hold enormous potential for basic and clinical research, they also raise difficult issues of how to design studies to deploy them most effectively. In designing a genomic study, a modern researcher might combine many sequencing modalities and sampling protocols, each with different utility, costs, and other tradeoffs. This is especially relevant for studies of somatic variation, which may involve highly heterogeneous cell populations whose differences can be probed via an extensive set of biotechnological tools. Efficiently deploying genomic technologies in this space will require principled ways to create study designs that recover desired genomic information while minimizing various measures of cost. Results The central problem this paper attempts to address is how one might create an optimal study design for a genomic analysis, with particular focus on studies involving somatic variation that occur most often with application to cancer genomics. We pose the study design problem as a stochastic constrained nonlinear optimization problem. We introduce a Bayesian optimization framework that iteratively optimizes for an objective function using surrogate modeling combined with pattern and gradient search. We demonstrate our procedure on several test cases to derive resource and study design allocations optimized for various goals and criteria, demonstrating its ability to optimize study designs efficiently across diverse scenarios. Availability and implementation https://github.com/CMUSchwartzLab/StudyDesignOptimization">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/143147eb452ae222669c29fe9d6382ddb00144ad" target='_blank'>
              Optimizing design of genomics studies for clonal evolution analysis
              </a>
            </td>
          <td>
            Arjun Srivatsa, Russell Schwartz
          </td>
          <td>2024-12-02</td>
          <td>Bioinformatics Advances</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Treatment with genotoxic agents, such as platinum compounds, is still the mainstay therapeutical approach for the majority of cancers. Our understanding of the mechanisms of action of these drugs is, however, imperfect and continuously evolving. Recent advances highlighted single-stranded DNA (ssDNA) gap accumulation as a potential determinant underlying cisplatin chemosensitivity, at least in some genetic backgrounds, such as BRCA mutations. Cisplatin-induced ssDNA gaps form upon restart of DNA synthesis downstream of cisplatin-induced lesions through repriming catalyzed by the PRIMPOL enzyme. Here, we show that PRIMPOL overexpression in otherwise wild-type cells results in accumulation of cisplatin-induced ssDNA gaps without sensitizing cells to cisplatin, suggesting that ssDNA gap accumulation does not confer cisplatin sensitivity in BRCA-proficient cells. To understand how ssDNA gaps may cause cellular sensitivity, we employed CRISPR-mediated genome-wide genetic screening to identify factors which enable the cytotoxicity of cisplatin-induced ssDNA gaps. We found that the helicase HELQ specifically suppresses cisplatin sensitivity in PRIMPOL-overexpressing cells, and this is associated with reduced ssDNA accumulation. We moreover identify RAD52 as a mediator of this pathway. RAD52 promotes ssDNA gap accumulation through a BRCA-mediated mechanism. Our work identified the HELQ-RAD52-BRCA axis as a regulator of ssDNA gap processing and cisplatin sensitization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d8640e7a773e07a338c2a6edff813977332e73d" target='_blank'>
              CRISPR knockout genome-wide screens identify the HELQ-RAD52 axis in regulating the repair of cisplatin-induced single-stranded DNA gaps
              </a>
            </td>
          <td>
            Lindsey M Pale, Jude B. Khatib, Alexandra Nusawardhana, Joshua Straka, Claudia M. Nicolae, George-Lucian Moldovan
          </td>
          <td>2024-11-12</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="

 Gene expression profiling following targeted pathway perturbations on primary patient samples provides important information for understanding heterogeneous pathway dependencies in cancer, thereby supporting the design of personalized therapies. However, analyzing such high-dimensional data poses a challenge. In this study, we developed a novel computational framework utilizing semi-supervised factor analysis, regularized multiple linear regression, and causal inference to analyze and interpret drug-perturbed transcriptomic profiles from patient samples. We applied our framework to RNA sequencing data obtained after ex-vivo perturbation with ten different small-molecule inhibitors in primary tumor cells from 108 chronic lymphocytic leukemia (CLL) patients, yielding 1010 transcriptional drug response profiles (Do et al., ASH2023 abstract 4633).
 To capture and quantify the shared and unique dimensions of gene expression changes induced by different pathway inhibitors, we employed guided sparse factor analysis (GSFA), originally designed for analyzing single-cell CRISPR screening (Zhou et al., Nat Methods, 2023). We identified perturbations uniquely associated with single factors, such as XPO1 inhibition by selinexor, MDM2 inhibition by nutlin-3a, and MEK1/2 inhibition by trametinib, suggesting the induction of distinct downstream signatures not shared with other perturbations. Conversely, perturbations like BTK inhibition by ibrutinib, PI3K inhibition by duvelisib, and mTOR inhibition by everolimus were linked to multiple factors shared across perturbations, suggesting extensive downstream pathway cross-talks. Notably, PI3K inhibition was significantly associated with two factors, one shared with BTK inhibition, and the other with PI3K and mTOR inhibition. Pathway enrichment analysis revealed that the factor shared between BTK and PI3K was enriched for the B-cell receptor signaling pathway, while the factor shared among PI3K, ATK, and mTOR inhibitions was enriched for MAP kinase pathway genes. This highlights the complex role of PI3K signaling in CLL and provides novel insights into the hierarchical pathway structure in CLL biology.
 We further examined the heterogeneity of pathway activity changes upon perturbation among patients with different driver mutations. Based on the GSFA model, we derived a per-patient perturbation score for each factor and associated them with major molecular subgroups in CLL using regularized multiple linear regression. We observed a strong association between TP53 mutational status and the factor representing MDM2 inhibition, reflecting the known dependencies of DNA damage response signaling on TP53 mutational status. Furthermore, this approach uncovered that IGHV mutational status strongly determines the downstream MAP kinase activity change upon PI3K/mTOR/AKT inhibition but not the BCR signaling activity change upon PI3K and BTK inhibition. By testing for interaction in linear models, we further investigated the dependence of individual gene expression changes upon pathway perturbation and the presence of driver mutations. Thanks to the high-dimensional nature of gene expression profiling, we could infer causal interactions between mutations and targets of perturbation using the method developed by Fischer et al., eLife, 2015. We successfully recaptured the known signaling structure of the BCR-PI3K cascade and TP53-MDM2 interaction. Moreover, we unveiled other novel interactions, such as the activation of BTK and MEK signaling by trisomy12, explaining our previous observations of increased sensitivity to BTK and MEK inhibitors in CLL samples with trisomy12 (Dietrich et al., J Clin Invest, 2018).
 This study underscores the value of perturbed transcriptomic profiling of primary cancer patient samples for mapping detailed pathway structures and dependencies on disease drivers. Our comprehensive computational workflow serves as a guide for mining and interpreting complex high-dimensional data, advancing our understanding of disease biology, and facilitating the rational design of novel personalized therapies, including combinational treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f702d18cbeba7eb5540013f6935d03318357636" target='_blank'>
              Unraveling Molecular Subgroup-Specific Pathway Dependencies in Chronic Lymphocytic Leukemia through Drug-Perturbed Transcriptomic Profiling and Statistical Modeling
              </a>
            </td>
          <td>
            Caroline Lohoff, Thi Huong Lan Do, Ferris Jung, Sandra Kummer, Vladimír Beneš, W. Huber, Thorsten Zenz, Junyan Lu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is the leading cancer among Saudis, and mutations in BRAF, KRAS, and NRAS genes are therapeutically significant due to their association with pathways critical for cell cycle regulation. This study evaluates the prevalence and frequency of somatic mutations in these actionable genes in Saudi CRC patients and assesses their pathogenicity with bioinformatics methods. Methodology The study employed the TruSight Tumor 15 next-generation sequencing (NGS) panel on 86 colorectal cancer (CRC) samples to detect somatic mutations in BRAF, KRAS, and NRAS genes. Bioinformatic analyses of NGS sequences included variant annotation with ANNOVAR, pathogenicity prediction, variant reclassification with CancerVar, and extensive structural analysis. Additionally, molecular docking assessed the binding of Encorafenib to wild-type and mutant BRAF proteins, providing insights into the therapeutic relevance of pathogenic variants. Results Out of 86 tumor samples, 40 (46.5%) harbored somatic mutations within actionable genes (BRAF: 2.3%, KRAS: 43%, NRAS: 2.3%). Fourteen missense variants were identified (BRAF: n = 1, KRAS: n = 11, NRAS: n = 2). Variants with strong clinical significance included BRAF V600E (2.32%) and KRAS G12D (18.60%). Variants with potential clinical significance included several KRAS and an NRAS mutation, while variants of unknown significance included KRAS E49K and NRAS R102Q. One variant was novel: NRAS R102Q, and two were rare: KRAS E49K and G138E. We further extended the CancerVar prediction capability by adding new pathogenicity prediction tools. Molecular docking demonstrated that Encorafenib inhibits the V600E variant BRAF protein less effectively compared to its wild-type counterpart. Conclusion Overall, this study highlights the importance of comprehensive molecular screening and bioinformatics in understanding the mutational landscape of CRC in the Saudi population, ultimately improving targeted drug treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1eadbaa87cb208b49527f30de43b306a5abf8a" target='_blank'>
              Exploring somatic mutations in BRAF, KRAS, and NRAS as therapeutic targets in Saudi colorectal cancer patients through massive parallel sequencing and variant classification
              </a>
            </td>
          <td>
            Thamer Abdulhamid Aljuhani, N. Shaik, R. T. Alqawas, R. Y. Bokhary, Mahmood Al-Mutadares, H. B. Al Mahdi, Nuha Al-Rayes, Ashraf AbdulRahman El-Harouni, R. Elango, B. Banaganapalli, Zuhier Ahmad Awan
          </td>
          <td>2024-11-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a myeloproliferative disorder caused by the reciprocal translocation of the BCR-ABL1 oncogene, which activates tyrosine kinase resulting in uncontrolled cell proliferation and apoptosis suppression. Although Imatinib (IM) is an effective treatment, IM resistance remains a significant concern. Therefore, microRNAs (miRNAs) have emerged as potential alternative therapies. These short, non-coding RNAs (20-22 nucleotides) regulate gene expression by binding to the 3' untranslated region (3'UTR) of target genes. The study aims to identify miRNAs linked to CML, determine miRNA’s target genes, construct a protein-protein interaction (PPI) network, and analyze pathways and gene ontology. A literature search using the keywords “microRNA”, and “chronic myeloid leukemia” yielded relevant papers, which were screened and categorized into three groups: 1) miRNA associated with TKI resistance, 2) miRNA as biomarkers or leukemogenesis, and 3) miRNA as therapy. Target genes for the identified miRNAs were determined using DIANA tools and miRTarBase. Gene ontology and pathway analysis were conducted using DAVID, while PPI and network visualization were performed using STRING, Cytoscape, and ClueGO. Thirteen miRNAs were selected, targeting 782 genes and forming 16 clusters. ClueGO identified clusters associated with key biological processes, including G1/S cell cycle transition, miRNA-mediated gene silencing, hematopoietic stem cell differentiation, and apoptosis regulation. Target genes are also significant in the CML pathway and other cancer pathways, such as p53, ErbB, FoxO, autophagy, apoptosis, VEGF, TNF, and microRNA. Notably, hsa-miR-16 emerged as the most promising therapeutic and biomarker candidate for CML, highlighting its role in critical pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082a3893f78953cb45d6e871eb9d7b48eccbc043" target='_blank'>
              Role of MicroRNAs in Chronic Myeloid Leukemia (CML) Treatment, Biomarkers, and Resistance to Tyrosine Kinase Inhibitor: A Bioinformatics-Based Analysis
              </a>
            </td>
          <td>
            Syarifah Faezah binti Syed Mohamad, Rabiatul Adawiyah Ramlee, M. H. Elias
          </td>
          <td>2024-11-21</td>
          <td>Journal of Tropical Life Science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Taxanes and platinum molecules, specifically paclitaxel and carboplatin, are widely used anticancer drugs that induce cell death and serve as first-line chemotherapy for various cancer types. Despite the efficient effect of both drugs on cancer cell proliferation, many tumours have innate resistance against paclitaxel and carboplatin, which leads to inefficient treatment and poor survival rates. Haploid human embryonic stem cells (hESCs) are a novel and robust platform for genetic screening. To gain a comprehensive view of genes that affect or regulate paclitaxel and carboplatin resistance, genome-wide loss-of-function screens in haploid hESCs were performed. Both paclitaxel and carboplatin screens have yielded selected plausible gene lists and pathways relevant to resistance prediction. The effects of mutations in selected genes on the resistance to the drugs were demonstrated. Based on the results, an algorithm that can predict resistance to paclitaxel or carboplatin was developed. Applying the algorithm to the DNA mutation profile of patients' tumours enabled the separation of sensitive versus resistant patients, thus, providing a prediction tool. As the anticancer drugs arsenal can offer alternatives in case of resistance to either paclitaxel or carboplatin, an early prediction can provide a significant advantage and should improve treatment. The algorithm assists this unmet need and helps predict whether a patient will respond to the treatment and may have an immediate clinically actionable application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a5886b4a974b52231aaaccc4e3210ef163722ed" target='_blank'>
              Predicting tumour resistance to paclitaxel and carboplatin utilising genome-wide screening in haploid human embryonic stem cells.
              </a>
            </td>
          <td>
            J. Nissenbaum, Emanuel Segal, Hagit Philip, R. Cashman, T. Golan‐lev, Benjamin E. Reubinoff, Adi Turjeman, O. Yanuka, Elyad Lezmi, O. Kopper, N. Benvenisty
          </td>
          <td>2024-11-10</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Trisomy of human chromosome 21 (T21) gives rise to Down syndrome (DS), the most frequent live-born autosomal aneuploidy. T21 triggers genome-wide transcriptomic alterations that result in multiple atypical phenotypes with highly variable penetrance and expressivity in individuals with DS. Many of these phenotypes, including atypical neurodevelopment, emerge prenatally. To enable in vitro analyses of the cellular and molecular mechanisms leading to the neurological alterations associated with T21, we created and characterized a panel of genomically diverse T21 and euploid induced pluripotent stem cells (iPSCs). We subsequently differentiated these iPSCs to generate a panel of neural progenitor cells (NPCs). Alongside characterizing genotype effects from T21, we found that T21 NPCs showed inter-individual variability in growth rates, oxidative stress, senescence characteristics, and gene and protein expression. Pathway enrichment analyses of T21 NPCs identified vesicular transport, DNA repair, and cellular response to stress pathways. These results demonstrate T21-associated variability at the cellular level and suggest that cell lines from individuals with DS should not solely be analyzed as a homogenous population. Examining large cohorts of genetically diverse samples may more fully reveal the effects of aneuploidy on transcriptomic and phenotypic characteristics in T21 cell types. A panel of genomically diverse T21 and euploid induced pluripotent stem cells (iPSCs) were created and subsequently differentiated into neural progenitor cells (NPCs). T21 NPCs showed reduced growth, increased oxidative stress, and inter-individual variability in gene and protein expression. This inter-individual variability suggests that studies with large cohorts of genetically diverse T21 samples may more fully reveal the effects of aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b76c08b2fe900818339a97ad9dde96559af8a93" target='_blank'>
              Analysis of genotype effects and inter-individual variability in iPSC-derived trisomy 21 neural progenitor cells.
              </a>
            </td>
          <td>
            Sarah E Lee, Laura L Baxter, Monica I Duran, Samuel D Morris, Iman A Mosley, Kevin A Fuentes, Jeroen L. A. Pennings, F. Guedj, Diana W. Bianchi
          </td>
          <td>2024-11-13</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 Reversal of the postnatal hemoglobin switch in red blood cells is a promising approach to therapies for sickle cell disease and β-thalassemia, as reactivating fetal hemoglobin (HbF, α2γ2) in adult red blood cells ameliorates disease complications. Studies over several decades have illuminated the paradigm of this fetal-to-adult hemoglobin transition, including central roles of transcriptional repressors BCL11A, LRF, and NFIA/X, which recruit repressive chromatin modifiers to the γ-globin locus. To elucidate undescribed regulatory mechanisms of these molecules and reveal new targets for therapies, we performed the first genome-wide genetic screen to pinpoint specific pathways or protein complexes that regulate γ-globin expression in adult erythroid cells.
 Because Cas9 technology is limiting at the genome-wide scale, we created a CRISPR-Cas12a genetic screen targeting all known coding genes in the human genome using a novel custom array directed towards 18,292 genes. The single CRISPR RNA (crRNA) library was cloned into a lentivirus scaffold and transduced into the adult-type erythroid cell line, HUDEP2, engineered to stably express Cas12a. HUDEP2 cells express little HbF, allowing for positive phenotypic selection (HbF+) by fluorescence-activated cell-sorting. By isolating the top 10% and bottom 10% of HbF-expressing cells and determining the representation of crRNAs via deep sequencing, we identified a crRNA subset enriched in the high-HbF population encompassing 211 candidate genes, representing potential regulators of HbF. To prioritize these genes for follow up studies, we conducted a domain-focused, CRISPR-Cas9 screen of all 211 candidate genes. Using 2,184 sgRNAs, this validation Cas9 screen in HUDEP2 cells nominated 18 high-confidence targets.
 Our screens identified PTPA (protein phosphatase 2 phosphatase activator) as a top candidate gene. It is a known modulator of PP2A, a multi-subunit complex with diverse cellular functions as a serine/threonine phosphatase. To our knowledge, it has no previously known role on globin gene production. In validation experiments, disruption of PTPA in HUDEP2 cells significantly increased γ-globin mRNA and protein expression. Using sgRNA-Cas9 ribonucleoprotein complex nucleofection, PTPA gene editing in primary human CD34+ cell derived erythroblasts significantly increased γ-globin mRNA and protein expression without overtly affecting erythroid maturation. Notably, the degree of effect is similar to that of BCL11A +58 enhancer disruption. PTPA depletion in HUDEP2 cells markedly reduced BCL11A mRNA and protein levels. Furthermore, BCL11A overexpression in PTPA gene-edited cells reduced γ-globin mRNA commensurate to parental control cells, thereby demonstrating it mediates γ-globin silencing predominantly via BCL11A. Altogether, our studies utilizing a novel Cas12a-based, genome-wide screen nominate a new pathway mediating γ-globin silencing via regulation of BCL11A.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7ad1df85f6d20906faccd5d92c3bb61b279a40d" target='_blank'>
              A Novel Genome-Scale CRISPR-Cas12a Screen Identifies Ptpa As a Fetal Hemoglobin Regulator Via BCL11A
              </a>
            </td>
          <td>
            Elizabeth A. Traxler, Chad Komar, Megan S. Saari, Quynn Hotan, A. J. Thrasher, Eugene Khandros, Kunhua Qin, O. Abdulmalik, Claire Shao, Qingzhou Chen, Andy J. Minn, C. Keller, Junwei Shi, Gerd A. Blobel
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="

 Introduction: Acute myeloid leukemia (AML) with retinoic acid receptor γ (RARG) rearrangement presents clinical, morphological, and immunophenotypic features similar to classic acute promyelocytic leukemia (APL). However, AML with RARG rearrangement exhibits different transcriptomic features compared to APL with PML-RARA or non-APL AML using bulk-RNA sequencing. To explore the biological heterogeneity, a comprehensive single-cell RNA sequencing (scRNA-seq) is essential to elucidate the molecular and genetic landscape of this disease.
 Methods: A total of 10 participants (APL cases = 6, AML cases with RARG rearrangement = 2, healthy donors = 2) were included. Bone marrow samples were subjected to scRNA-seq. The sequencing data were processed using CellRanger (10x Genomics), and subsequent analyses were conducted with Seurat. Cell type annotation was performed utilizing SingleR, referencing the Human Primary Cell Atlas. Further analyses, including trajectory inference, pathway enrichment and score calculation, clonal evolution prediction and cellular communication, were conducted using R language.
 Results: A detailed map of the hematopoietic cell states was created on 133,730 cells. Dimensionality reduction and unsupervised graph-based clustering revealed 14 distinct cell types, including hematopoietic stem cells (HSCs), granulocyte-monocyte progenitors (GMPs), promyelocytes, neutrophils, monocytes, plasma, B cells, T cells and NK cells. Further subdivision of myeloid cell populations revealed 12 subgroups, with CD38- promyelocytes and CD16- monocytes being markedly enriched in cases with RARG rearrangement. Compared with healthy controls and APL, the myeloid subsets of RARG rearrangement variants are characterized by the upregulation of specific genes, such as H1-10, H1-2, H2AZ1, H2AZ2, H3-3A, H3-3B, H4C3, MACROH2A1, SEPTIN7, and SNHG29. Furthermore, in contrast to APL the histone deacetylases class (HDAC) III pathway was up-regulated in CD38- promyelocytes of RARG rearrangement variants, and the interleukin-6 pathway was more highly activated in GMP subsets. Additionally, the pathways involved in necroptosis, neutrophil extracellular trap formation, and aminoacyl-tRNA biosynthesis were significantly upregulated across all myeloid cell subsets of RARG rearrangement variants. Subsequently, we employed Slingshot package to analyze data, which reconstructs putative branching transcriptional trajectories to identify potential relationships across calculated states. The five inferred trajectory paths in AML with RARG rearrangement were all derived from CD34+CD38+ HSCs. Notably, the trajectory path leading to leukemic promyelocytes demonstrated a significant overexpression of key genes, including XIST, AZU1, CTSG, and ELANE, in comparison to the paths transited into normal monocytes. Using inferCNV and UPhyloplot2, a clonal evolution analysis was conducted, revealing that some of the myeloid clusters in RARG rearrangement variants were predominantly characterized by a gain of chromosome 21q. In contrast, the originating APL myeloid cells across samples exhibited a loss of chromosome 6p and 8q. However, this chromosomal loss was observed at a lower frequency or was absent in AML with RARG rearrangement. Finally, cell communication revealed the specific pathways related to RARG rearrangement variants, as opposed to APL. These pathways were predominantly implicated in the activated effector memory CD8+GZMK+ T cells involved in immune suppression. The identified pathways included the incoming pathways of APP-CD74, LCK-CD8A-CD8B1, and MHC-I, as well as the CD99-CD99 outgoing pathway from this T subset.
 Conclusions: This study elucidates the unique molecular characteristics of AML with RARG rearrangement using scRNA-seq, identifying specific gene overexpression, pathway activation and genetic alterations that may underpin therapy resistance. Notably, the HDAC III and interleukin-6 pathways emerge as potential therapeutic targets. Additionally, signaling pathways associated with immune suppression are implicated in RARG rearrangement variants. These insights can inform the development of targeted therapeutic strategies for AML with RARG rearrangement.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5182f70090c59ae97aad26afd242ae6694942e" target='_blank'>
              Single-Cell RNA Sequencing Uncovers Distinct Molecular Characteristics and Immune Suppression Profiling of AML with Retinoic Acid Receptor γ Rearrangement
              </a>
            </td>
          <td>
            Qing-Yu Xu, Shu Sun, Yebo Peng, Yingying Huo, Ting Shi, W. Gao, Honghu Zhu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Papillary thyroid cancer (PTC) is one of the fastest-growing cancers worldwide, lacking established causal factors or validated early diagnostics. Human endogenous retroviruses (HERVs), comprising 8% of human genomes, have potential as PTC biomarkers due to their comparably high baseline expression in healthy thyroid tissues, indicating homeostatic roles. However, HERV regions are often overlooked in genome-wide association studies because of their highly repetitive nature, low sequence coverage, and decreased sequencing quality. Using targeted whole-genome sequence analysis in conjunction with high sequencing depth to overcome methodological limitations, we identified associations of specific HERV variants with PTC. Analyzing WGS data from 138 patients with PTC generated through The Cancer Genome Atlas project and 2015 control samples from the 1000 Genomes Project, we examined the mutational variation in HERVs within a 20 kb radius of known cancer predisposition genes (CPGs) differentially expressed in PTC. We discovered 15 common and 13 rare germline HERV variants near or within 20 CPGs that distinguish patients with PTC from healthy controls. We identified intragenic–intronic HERV variants within RYR2, LRP1B, FN1, MET, TCRVB, UNC5D, TRPM3, CNTN5, CD70, RYR1, RUNX1, CRLF2, and PCDH1X, and three variants downstream of SERPINA1 and RUNX1T1. Sanger sequencing analyses of 20 thyroid and 5 non-thyroid cancer cell lines confirmed associations with PTC, particularly for MSTA HERV-L variant rs200077102 within the FN1 gene and HERV-L MLT1A LTR variant rs78588384 within the CNTN5 gene. Variant rs78588384, in particular, was shown in our analyses to be located within a POL2 binding site regulating an alternative transcript of CNTN5. In addition, we identified 16 variants that modified the poly(A) region in Alu elements, potentially altering the potential to retrotranspose. In conclusion, this study serves as a proof-of-concept for targeted variant analysis of HERV regions and establishes a basis for further exploration of HERVs in thyroid cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/529189957aabeea042de1fa9f7bdcd737d1637aa" target='_blank'>
              Targeted Variant Assessments of Human Endogenous Retroviral Regions in Whole Genome Sequencing Data Reveal Retroviral Variants Associated with Papillary Thyroid Cancer
              </a>
            </td>
          <td>
            Erik Stricker, E. Peckham-Gregory, Stephen Y. Lai, Vlad C. Sandulache, Michael E. Scheurer
          </td>
          <td>2024-11-27</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="

 Follicular lymphoma (FL) is the most prevalent low-grade germinal centre (GC) derived non-Hodgkin lymphoma. Despite being considered an indolent lymphoma its clinical course is heterogenous with some patients experiencing frequent relapses and eventual transformation to a high-grade disease associated with an especially poor prognosis.
 Through phylogenetic analysis of successive human FL tumors, evidence has emerged regarding the existence of common progenitor cells (CPCs). These precursor cells serve as a reservoir of cells that are able to evade killing with standard treatments for FL and give rise to relapse and transformed FL. They carry the t(14:18) translocation and are enriched for somatic mutations in chromatin-modifying genes including CREBBP and KMT2D. Developing therapies to specifically target and eliminate these common progenitor cells will be crucial in finding a cure for FL.
 To understand this population of therapy-resistant precursor cells, we engineered mouse models that faithfully replicate the early and recurrent gene mutations observed in FL, including BCL2 translocation, CREBBP and KMT2D loss-of-function (LoF) mutations (BCK mice). We then subjected mice to a standard human-like treatment regimen of R-CHOP (Rituximab, Cyclophosphamide, Doxorubicin, Vincristine, Prednisone). Administration of the mouse-adapted R-CHOP resulted in the complete depletion of B cells from control mice. However, treatment of BCK mice revealed a resistant B-cell population. Notably, these resistant cells predominantly displayed a memory B-cell phenotype.
 Single cell RNA-seq analysis of these therapy-resistant cells identifies unique transcriptional signatures and enrichment of pathways that can be targeted therapeutically. Comparison with human FL scRNAseq datasets shows that these therapy-resistant precursor cells are enriched in FL tumors of patients with early relapse.
 Finally, we have used an innovative in vitro platform that harnesses a GC B-cell culture system to amplify CPCs, facilitating subsequent high-throughput drug screening. This approach has allowed us to identify specific vulnerabilities within the CPC population, paving the way for the development of therapies tailored to overcome FL resistance mechanisms.
 Through these concerted efforts, we edge closer to the long-awaited goal of discovering a definitive cure for Follicular Lymphoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d38aaaee2711f99741edeedb01793cc234314bd" target='_blank'>
              Characterizing and Targeting Follicular Lymphoma's Common Progenitor Cells
              </a>
            </td>
          <td>
            Oscar Atkins, Lingling Zhang, Miu Shing Hung, Christian Reinhardt, D. Calado
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 Resistance to anti-cancer therapeutics remains one of major obstacles to improving survival rates in cancer; it arises in a multitude of ways including accumulation of epigenetic alterations. Recently, we discovered a new mechanism of treatment resistance that is not fully driven by epigenetic remodeling but nevertheless implies an activity switch of an epigenetic enzyme, EZH2. Despite its well-established activity as part of PRC2 for mediating gene repression by H3K27me3 deposition, new evidence points the importance of others, so-called non-canonical activities mediated by variation of PRC2 composition and kinase-related post-translational modifications (PTMs) of EZH2. We previously reported that non-canonical EZH2 activity was associated with retinoic acid (RA) resistance in acute promyelocytic leukemia (APL) (Poplineau, Blood, 2022). RA resistant cells (i) expressed genes involved in DNA repair, replication and proliferation processes, that we called ReP signature, (ii) kept the potential to develop leukemia in vivo and (iii) were marked by high EZH2 expression. Targeting pan-EZH2 activities (canonical/non-canonical) was necessary to eliminate RA resistant cells, which underlies a dependency of these cells on an EZH2 non-canonical and non-methyltransferase activity and the necessity to degrade EZH2 to overcome RA resistance in APL.
 In the present work, we questioned whether EZH2 non-canonical activity may contribute more generally to resistance in non-APL AML. To do so, we questioned non-APL public AML transcriptomic datasets (BeatAML and TCGA). At first, we classified non-APL AML patients with high and low EZH2 expression and revealed that patients with high EZH2 expression were enriched in metabolic signatures reflecting chemotherapy resistance (i-e HighOXPHOS). Next, we computed a new EZH2-related signature (i-e; 61 common genes to BeatAML and TGCA obtained by retaining the 200 most differentially expressed genes in patients with high and low EZH2 expression) and found that this signature was positively correlated with AraC low responder patients. Besides, we showed that our computed non-canonical and non-methyltransferase EZH2 and ReP signatures were associated (i) with HighOXPHOS and leukemic stem cells maintenance signatures in AML patients and (ii) relapse of patients following AraC treatment.
 Next, we investigated the relevance of targeting EZH2 in AML using an EZH2 PROTAC synthetized in our laboratory and derived from the previously published MS177 EZH2 degrader (Wang, Nat. Cell Biol., 2022). Using several leukemic cell lines, we showed that chemo-resistant cell lines, more especially AraC resistant cells, were highly sensitive to EZH2 degradation (fifty times more sensitive than AraC sensitive cells) but were not sensitive to the catalytic EZH2 inhibitor Tazemetostat. We then studied the potential synergistic effect between the EZH2 degrader and AraC and did not observe any additional beneficial effect with the combo therapy. However, we showed that EZH2 degradation has strong synergistic effect with RA in killing AML cells in vitro, suggesting that EZH2 degradation may sensitize chemotherapy resistant cells to RA.
 Finally, by coupling omics approaches on AML cell line resistant to AraC, we are currently investigating mechanisms that dictate EZH2 activities (canonical/non-canonical) and characterizing these activities at the functional level. Proteomics screening revealed specific PTMs on EZH2 and variation in its interacting partners that may contribute to AraC resistance in AML.
 Altogether, these data reinforce (i) the intimate link between EZH2 non-canonical activity and AraC resistance in non-APL AML and (ii) support the need to preclinically investigate the therapeutic value of targeting non-canonical EZH2 activities not only to sensitize non-APL AML to RA but also to overcome chemotherapy resistance in non-APL AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477e541c859f23a419d4cdd61e3b25c660a0006d" target='_blank'>
              Deciphering EZH2 Non-Canonical Activities in AML Resistance
              </a>
            </td>
          <td>
            L. N’Guyen, Emeline Boet, C. Montersino, Claire Burny, S. Audebert, Luc Camoin, R. Castellano, S. Garciaz, Atsushi Iwama, J. Sarry, E. Duprez, M. Poplineau
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia-telangiectasia-mutated) and ATR (Ataxia-telangiectasia-mutated and Rad3-related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure to UV-B, UV-C and protons induced the microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana.
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ddc2e4963aa356dbc3de332034a633ed4d7780" target='_blank'>
              Antisense oligonucleotides as a targeted therapeutic approach in model of acute myeloid leukemia.
              </a>
            </td>
          <td>
            D. Maceckova, L. Vaňková, J. Bufka, Petr Hošek, Jiří Moravec, Pavel Pitule
          </td>
          <td>2024-12-18</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="

 Introduction
 Follicular lymphoma (FL) patients usually follow an indolent clinical course but suffer recurrent relapses, after few months or after long periods of remission. Many ultimately develop resistances to therapies and a subset, with an annual incidence of 2-3%, experience transformation to DLBCL. Understanding the molecular mechanisms responsible for the clonal evolution of these tumors thus appears essential to anticipate these events. As such, two B-cells specific genetic instability processes are suspected to drive lymphomagenesis: the aberrant somatic hypermutation (ASHM) and the isotype class switch recombination (CSR). ASHM occurs when AID loses its specificity and targets multiple oncogenes while CSR, also initiated by AID, initiates double strand breaks responsible for the acquisition of chromosomal translocations. However, while it is largely recognized that ASHM usually remains active in FL cells, little is known about a possible ongoing CSR activity in these tumors and its potential impact on progression.
 Methods
 We performed a detailed analysis of the structure of the IGH loci in 16 patients: 8 with FL at diagnosis, 4 with paired FL + DLBCL (tFL) biopsies and in 5 with de novo t(14;18) positive DLBCL using high throughput IGH RNA-sequencing. Sequencing libraries were prepared using a Template Switching 5′ RACE Protocol kit (New England Biolabs) starting from total RNA extracted from fresh-frozen biopsies. Single-end 300 bp read sequencing was performed on a MiSeq or NextSeq 550Dx system.
 Results
 In agreement with previous studies, almost all FL showed genomic hallmarks of CSR activation at diagnosis (11/12). If 6 still expressed an IgM, the IGHCµ gene was deleted from the translocated allele in 9/9 t(14;18) positive cases while 2 negative biopsies had a t(3;14) and a t(8;14) involving IGH switch regions respectively. In one FL/DLBCL pair and one de novo DLBCL, we also discovered an as yet undocumented cryptic inversion on the der(18)t(14;18). This recurrent rearrangement, which is also present in the RL cell line, joins a DNA repeated region located upstream from BCL2 with an IGH switch region, resulting in a flip of the whole intervening BCL2-IGH fusion. The localization of these breakpoints and the detection of multiple mutations around the junctions, which remind the mutations introduced by AID at physiological CSR junctions, further argue for an aberrant CSR activation during lymphomagenesis.
 Unexpectedly, we also observed that the two DLBCL biopsies which harbored this inversion, a second tFL and two independent FL expressed dozens of fusion transcripts where multiple ectopic sequences were fused to the first exons of an IGH constant gene (649, 398, 284, 112 and 87 unique fusions, respectively). Most of the fusions we tested were validated by RT-PCR and Sanger sequencing (12/15), ruling out the possibility that they could correspond to RNA-seq artefacts. Additionally, we observed that the majority of the 1261 unique ectopic sequences identified aligned upstream from canonical GT splicing sites on genomic DNA (91%), at the exact end of known exons (68.8%), with a strong bias toward the first exons of their corresponding genes (51.4%). Among these genes, we identified many AID targets, some of which are implicated in recurrent IGH translocation in B-cell lymphoma (CIITA, FOXP1, BCL11A, CCND3, RHOH, CXCR4, SOCS1, IL21R, NOTCH2...), suggesting that these fusions result from a genome wide instability linked to an ongoing aberrant CSR activation. Notably, four of these five patients experienced rapid and adverse clinical evolutions, dying less than 2 years after these clinical events, suggesting that this phenomenon may be correlated to tumor aggressiveness.
 Conclusion
 We present new molecular evidence thet CSR is activated during FL lymphomagenesis and propose that a reactivation of this process at the tumoral stage can lead to a genome-wide instability which results in the emergence of dozens of subclones harboring different IGH fusion genes. This aberrant CSR reactivation may thus, like ASHM, be a major contributor to FL clinical progression by providing these cells new important selective advantages.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef894b9ff2119b0e71472d65e1e578c36d8a4339" target='_blank'>
              Aberrant Isotype Class Switch Recombination Reactivation Leads to a Genome Wide Genomic Instability in Follicular Lymphoma
              </a>
            </td>
          <td>
            M. Viennot, Marion Chanussot-Godey, V. Rainville, Sandrine Le Noir, Melody Caillot, F. Drieux, E. Veresezan, Roman Vion, P. Viailly, V. Camus, H. Tilly, Eric Pinaud, F. Jardin, P. Ruminy
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="BRCA1, a critical tumor suppressor gene, plays an essential role in maintaining genomic stability through its involvement in DNA double-strand break repair, particularly via homologous recombination. Loss or impairment of BRCA1 function disrupts this repair pathway, resulting in genomic instability and significantly increased susceptibility to breast and ovarian cancers. To elucidate the molecular mechanisms by which BRCA1 mutations contribute to tumorigenesis, we employed quantitative mass spectrometry-based proximity labeling and affinity purification to identify cancer-specific protein-protein interactions (PPIs). Our integrated omics and visualization analysis of interactors revealed that the BRCA1-Y1853ter mutant, through its interaction with BARD1, perturbs the interactome and impacts cellular processes within the cytoplasm and nucleoplasm. Structural data further corroborated these findings, showing enhanced binding between the mutant BRCA1 and specific interactors, suggesting an altered functional profile. Together, these observations raise the hypothesis that the BRCA1-Y1853ter mutant may exhibit gain-of-function characteristics, providing new insights into the molecular and cellular effects of mutations in the BRCA1 C-Terminal (BRCT) domain and their implications for the pathogenesis of breast and ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7081d3dc4a7fe94ebd83524e06e557bb05ca412" target='_blank'>
              Differential Remodeling of the BRCA1 Interactome by a Unique Minimal Clinical Truncation Mutation
              </a>
            </td>
          <td>
            Jingjing Chen, Mihaela E. Sardiu, M. Rekowski, Jules Nde, Zachary Clark, S. Anant, Michael P. Washburn, Roy A Jensen
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The mitochondrial genome, which encodes genes essential for respiration and cellular homeostasis, is the target of abundant and highly diverse somatic alterations in cancers. Somatic alterations to mitochondrial DNA (mtDNA) nearly always arise heteroplasmically, producing heterogeneous ensembles of mtDNA within single cells. Here, we review new insights derived from exponential increases in genomic sequencing data that have uncovered the nature of, selective pressure for, and functional consequences of cancer-associated mtDNA alterations. As many discoveries have been limited by their ability to determine cell-to-cell variation in mtDNA genotype, we describe a new generation of single-cell sequencing approaches that resolve otherwise indeterminate models of mtDNA heteroplasmy. In tandem with novel approaches for mtDNA editing and modeling of mutations, these advances foreshadow the quantitative dissection of dosage-dependent mtDNA phenotypes that underlie both tumor evolution and heterogeneous response to therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823dce23cbbddef91125605bfb3305bbb4cdca3d" target='_blank'>
              Single-Cell Technologies for Studying the Evolution and Function of Mitochondrial DNA Heteroplasmy in Cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Erin M. Cumming, Minsoo Kim, Caleb Lareau, Ed Reznik
          </td>
          <td>2024-11-12</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="

 Introduction
 Multiple Myeloma (MM) is a complex hematological malignancy characterized by a range of well-known cytogenetic alterations. Despite the last advances in the treatment of MM, it remains to be a challenging disease with a high variability of responses to therapy. The presence of specific genetic alterations, such as those associated with hyperdiploidy, t(11;14), 1q21 amplifications, and p53 deletions, contributes to the variability in disease progression and treatment outcomes.
 Aims:
 To elucidate the molecular alterations specific to different MM cytogenetic subtypes and to understand their implications for disease pathogenesis and progression.
 To evaluate whether mutational analysis of the CD138+ cell population is an effective approach for molecular characterization of mutations with very low Variant Allele Frequency (VAF) in MM.
 Methods:
 Next-Generation Sequencing (NGS) was performed on a cohort of 64 newly diagnosed Multiple Myeloma (MM) patients. This cohort included 39 individuals with hyperdiploidy, 25 with t(11;14), 24 with 1q21 amplifications, and 7 with p53 deletions. The percentage of aberrant plasma cells was assessed by flow cytometry using the standardized EuroFlow consortium panel for plasma cell disorders. Plasma cells were isolated from bone marrow aspirate samples using magnetic separation with anti-CD138 beads (MACs Miltenyi). Cytogenetic alterations were identified using Fluorescence in Situ Hybridization (FISH). NGS libraries were synthetized using the SOPHIA Custom Multiple Myeloma NGS panel, which targets 43 genes (1500X coverage), and analyzed in the Sophia DDM software. The minimum VAF considered was 1% in selected CD138+ population.
 Results
 The most notable alterations detected were involved in 5 major pathways: cell cycle, transcriptional regulation, signal transduction pathways, apoptosis and immune response. Interestingly, alterations were not found in the same regions, but affected different genes in each myeloma subtype. A significative proportion of mutations (17,5%) were subclonal (VAF= 1-5%). The most frequently mutated genes were p53, KRAS, BRAF, FGFR3 and FMN2. In hyperdiploid MM (HDMM), the MAPK/ERK signaling pathway was affected by BRAF mutations (18.5%), as well as KRAS (22.4%) and NRAS mutations (14.3%). Remarkably, BRAF mutations were only observed in HDMM. Transcriptional process was also affected, including mutations in EGR1 and TENT5C (13.8% and 10.3% of cases respectively). Regarding apoptosis and cell survival, we identified mutations in CYLD and BIRC2 in 14.8% of cases. Finally, alterations in SP140, implicated in the immune regulation, were found in 10.3% of HDMM cases compared to 4.5% in t(11;14), (p<0.05). On the contrary, MM with t(11;14) were characterized by alterations in cell cycle, mainly in RB1 (18.2%) and CCND1 (45.5%). Attending to signaling pathways mutations affected mostly FGFR3 (27.3%) and FAT3 (22.7%), while the MAPK/ERK pathway was altered by mutations in RASA2 (9.1%) and NF1 (10.2%). DIS3 was the most frequently altered gene (18.2%) involved in transcriptional regulation. Finally, in this group, apoptosis was impaired by mutations in PPM1D (9.1% of cases). TP53 lesions were detected in 16.3% of cases across both subtypes, with lesions occurring in cases with (29.4%) and without deletion (41.2%). Similarly, KRAS mutations were found in 22.4% of all cases, while NRAS mutations were present in 14.3%. NRAS and KRAS mutations were mutually exclusive. Notably, NRAS mutations were strongly associated with 1q21 amplification (50% vs. 18% without amplification, p<0,05), whereas KRAS mutations were absent in cases with 1q21 amplification. Additionally, we identified mutations with potential contribution to metastatic capacity in FMN2 (28.6% of all cases).
 Conclusions Comparative analysis of t(11;14) and hyperdiploid MM revealed significant differences in critical pathways. The high frequency of genetic events within the same molecular pathways (such as CCND1, RAS, or p53/RB) suggests potential synergistic effects that may contribute to drug resistance and disease progression. The sequencing of purified CD138+ cells allows the detection of low VAF mutations, offering crucial data for risk stratification and optimal management of the disease. The detection of specific mutations in the different MM subtype is the first step for future advances in the development of personalized therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db520b7b3fe2b5883ce1dec30a9a39b54d389bd5" target='_blank'>
              NGS in Purified CD138+ Plasma Cells Reveals Distinct Mutational Signatures. a Comparative Study of Newly Diagnosed Hyperdiploid Vs t(11;14) Multiple Myeloma
              </a>
            </td>
          <td>
            Paula Piñero, Rocio Gonzalez, V. Díaz, Elena Marin, Marina Gonzalez, Pablo Manresa, Javier López, F. Tarin
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chronic antigenic stimulation is central to marginal zone lymphoma (MZL) development. While the pharmacological inhibition of the B-cell receptor (BCR) signaling is initially effective, secondary resistance frequently develops. We conducted a CRISPR interference (CRISPRi) screen investigating enhancer-associated long non-coding RNAs (elncRNAs) in MZL cells to identify transcripts crucial to shape their dependence on BCR pathway activation. We identified LOC730338, an elncRNA associated with A-to-I RNA editing, which we renamed ADARreg. Silencing ADARreg re-sensitized tumor cells to BCR pathway inhibition by modulating ADAR2 nuclear translocation and altering intronic RNA editing. The process preferentially affected genes that enhance immune responses, such as STING, IRF3, and p65. ADARreg knockdown also increased lymphoma cell sensitivity to NK cell-mediated cytotoxicity. Our results indicate that targeting elncRNAs like ADARreg represents a potential strategy to overcome drug resistance in lymphoma, also opening new therapeutic opportunities for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b0f5b32756fe2cd1cc1438cd5631c98fcc3016" target='_blank'>
              Enhancer lncRNA LOC730338 limits the innate immune response against lymphoma cells, inhibiting ADAR2-dependent alternative transcription
              </a>
            </td>
          <td>
            Luciano Cascione, F. Guidetti, S. Ramnarayanan, Andrea Rinaldi, F. Spriano, Alex Zadro, C. Tarantelli, Nicolas Munz, A. Arribas, Rory Johnson, Francesco Bertoni, S. Napoli
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Acute Myeloid Leukemia (AML) and Blast Crisis Chronic Myeloid Leukemia (BC CML) share several clinical characteristics, including the accumulation of immature myeloid cells and disrupted hematopoiesis. Despite the similarities, the two diseases exhibit different genetic profiles, contributing to distinct disease progressions and therapeutic outcomes. In this study, we perform a comparative analysis using bioinformatics and systems biology approaches to explore shared and unique mutational signatures, transcriptional networks, and molecular regulators in AML and BC CML. Our analysis identifies 29 genes common to both diseases, which are involved in key biological processes such as cell proliferation, cell death regulation, and transcriptional control. A subsequent pathway enrichment highlights common pathways, including the PI3K-AKT signaling and cytokine-mediated signaling, that may contribute to disease progression and resistance to therapy. Additionally, we construct a protein-protein interaction network and a transcription factor-miRNA coregulatory network, revealing key hub genes such as TP53, RUNX1, and BCL2. These genes could be potential shared therapeutic targets for treating AML and BC CML. Therefore, our analysis offers insights into the molecular underpinnings of these aggressive hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1aff1a0450bf81f4feefcd2ff55f60cc322c2c" target='_blank'>
              Shared Genetic and Regulatory Mechanisms in Acute Myeloid Leukaemia and Blast Crisis Chronic Myeloid Leukaemia
              </a>
            </td>
          <td>
            Sulagna Basu, Rakesh Pandey
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Mutually exclusive mutations in IDH1 and IDH2 account for a total of 20% of AML cases and are commonly found genetic alterations in both de novo and treatment-related AML (t-AML). The use of mutant IDH1/2 inhibitors has demonstrated substantial activity in AML. However, these drugs are not curative, and resistance occurs mostly within the first year of treatment due to either leukemia stemness or selection of mutations in RUNX1/CEBPA or RAS-RTK pathway. Thus, novel biomarker-targeted therapies are needed to target both de novo and refractory/relapsed (r/r) cases of IDH1/2 AMLs. As per TCGA (The Cancer Genome Atlas), around 50-60% of AML exhibit silencing of O6-methylguanine methyltransferase (MGMT), a DNA repair protein that removes alkyl lesions from the O6-position of guanine resulting from alkylation DNA damage. Interestingly, patients with IDH1/2-mutant AML exhibit high MGMT promoter hypermethylation and low protein expression, thus making it a promising targetable biomarker.
 Our RNA-seq analysis on 12 AML patient samples (7 wild-type, 5 IDH1/2-mutant) and detailed CpG site-specific investigation of MGMT promoter methylation on 10 AML patient samples (2 wild-type, 8 IDH1/2-mutant) from the Yale repository independently corroborates the TCGA findings. To identify potential therapeutics employing MGMT as a potential biomarker, we first screened a panel of DNA alkylating agents that deposit different alkyl groups (methyl, chloro-ethyl, fluoro-ethyl) onto the O6-position of guanine and clinical cross-linkers (nitrogen mustards) against multiple AML cell line pairs isogenic for MGMT. Interestingly, temozolomide (TMZ), which is an FDA-approved MGMT-dependent methylating agent, and our novel first-in-class fluoro-ethylating agent KL50 emerged as top hits imparting the maximum therapeutic index.
 Previous MGMT-biomarker based clinical trials in AML using temozolomide (TMZ) highlighted a moderate correlation of MGMT promoter hypermethylation with overall treatment response. A functional mismatch repair (MMR) pathway is the key to the cytotoxic effects of methylating agents like TMZ. We have established both in vitro and in vivo that MMR loss in MGMT-silenced AML often leads to the development of resistance, and evidence suggests a significant loss of MMR in up to 50% of refractory and relapsed cases of AML. Moreover, well-characterized hematological adverse effects (HAEs) associated with TMZ which include aplastic anemia, predisposition to AML or myelodysplastic syndrome (MDS), and severe myelosuppression demand novel MGMT-dependent therapies. Here, we highlight the role of our novel fluoro-ethylating agent (KL50) in targeting MGMT-silenced AML. The time-dependent cross-linking of DNA induced by KL50 selectively activates ATR-dependent DNA damage response pathways independent of MMR. Blocking the ATR-mediated DNA repair axis with AZD6738, an ATR inhibitor (ATRi) currently in clinical trials for myeloid leukemia (NCT03770429) profoundly synergizes with the low doses of KL50 in MGMT-isogenic in vitro and in vivo preclinical AML models. We are now investigating the synergistic efficacy of this novel combination on MGMT-silenced patient-derived xenografts (PDXs) in humanized MISTRG mice models. This clinically translatable study holds tremendous potential for identifying a new class of combinatorial therapy that could target both de novo and refractory/relapsed (r/r) cases of IDH1/2 mutant AMLs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79a2e37d99cfc38c22e320284040e2b37c6a501d" target='_blank'>
              O6-Guanine Targeting Novel DNA Cross-Linker and ATR Inhibitor Combination for MGMT-Silenced IDH1/2 Mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Prateek Bhardwaj, R. Sundaram, Sam Friedman, Amro Baassiri, Martin Matthews, Jennifer VanOudenhove, Susan E. Gueble, Stephanie Halene, R. Bindra
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="

 Introduction
 The precise identification of neoantigens is critical for advancing immunotherapy in hematologic malignancies, particularly in aggressive myeloid neoplasms such as acute myeloid leukemia (AML). Neoantigens, arising from tumor-specific somatic mutations and complex RNA modifications such as alternative splicing and gene fusions, provide specific targets for immune activation. However, the genetic complexity of AML poses significant challenges for their identification. Traditional methods, dependent on paired tumor-normal samples and limited to simple variants, fall short in capturing the full spectrum of neoantigens. Here, we present a novel bioinformatics pipeline addressing this by identifying neoantigens from RNA-Seq data without needing paired samples and by analyzing a broader range of genetic alterations, including alternative splicing and gene fusions. This approach enables the discovery of a richer set of neoantigens, enhancing the potential for personalized immunotherapy strategies.
 Objectives
 To develop a novel bioinformatics tool for the precise identification of tumor neoantigens in unpaired RNA-Seq samples from patients with AML.
 Methodology
 We analyzed 104 AML diagnostic samples from bone marrow and peripheral blood using RNA-Seq. These patients were originally recruited in the Pethema Quiwi phase 2B trial. RNA was extracted, sequenced, and analyzed for transcript expression. We focused on detecting SNPs/INDELs, gene fusions, and alternative splicing events to predict neoepitopes. Predicted neoepitopes, ranging from 8 to 11 amino acids, were evaluated for their ability to bind MHC-I and TCR, crucial for immune response activation.
 Results
 Our novel bioinformatics pipeline successfully identified SNPs/INDELs, gene fusions, and splicing variants in 104 AML samples from unpaired RNA-Seq data. Neoantigens derived from SNPs/INDELs and splicing variants were consistently detected across all samples, while fusion-derived neoantigens were absent in 27 samples. On average, 23,000 neoantigens were identified from SNPs/INDELs, 19 from fusions, and 9,388 from splicing variants. Among the SNPs/INDELs, the most frequently mutated genes included BCL2 and CD58, generating neoantigens in 30 and 26 patients, respectively. Variants in BCL2, a critical regulator of apoptosis, suggest increased cell survival and resistance to apoptosis (Certo et al., 2006). Alterations in CD58, important for immune cell interactions, could contribute to immune evasion by leukemic cells (Berger et al., 1982). Splicing variants were also prominent, with alterations detected in CD27 and CD72 in 20 and 18 patients, respectively. These genes are known to play roles in immune regulation and could impair immune responses against leukemic cells (van Oers et al., 2013; Koopmans et al., 2017). Variants in GDF11 and GDF15, involved in cellular growth and senescence, were identified in 15 and 12 patients, potentially influencing cell proliferation and aging in AML (Egerman et al., 2015; Xu et al., 2006). Additionally, disruptions in NACA and NUP155, related to protein synthesis and nuclear transport, were observed in 14 and 13 patients, respectively, indicating their broad impact on cellular function and tumor biology (Garreau de Loubresse et al., 2014; Enninga et al., 1991). Fusion-derived neoantigens were identified in a subset of samples, with notable fusions involving PIM3, an oncogene, and EIF4A1, which impacts protein synthesis, detected in 10 and 8 patients, respectively. These fusions suggest enhanced oncogenic signaling and altered translational control in AML (Wang et al., 2001; Graff et al., 2008). The variability in fusion events across different leukemias underscores the need for personalized approaches in targeting these neoantigens.
 Conclusion
 Our novel bioinformatics pipeline successfully identifies a comprehensive range of neoantigens from unpaired RNA-Seq samples in AML patients. By encompassing SNPs/INDELs, gene fusions, and alternative splicing events, this approach overcomes traditional limitations and uncovers valuable targets for personalized immunotherapy. These findings highlight the potential for more precise and effective AML treatment strategies, underscoring the importance of advanced bioinformatics in cancer immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03b984268557c00f1aac2e6433e768d8c5b4a2a8" target='_blank'>
              A Cutting-Edge Bioinformatics Tool for Neoantigen Identification in Unpaired RNA-Seq Samples of AML
              </a>
            </td>
          <td>
            Carlos Pérez Míguez, A. Mosquera Orgueira, M. P. Pérez Encinas, Rebeca Rodriguez Veiga, J. B. Bergua Burgues, Jesus Lorenzo Algarra, C. Botella, J. A. Pérez Simón, T. Bernal, M. Tormo, M. Calbacho, O. Salamero, J. Serrano, V. Noriega, Juan Antonio Lopez Lopez, Susana Vives, M. Colorado, José Luis López Lorenzo, Maria Vidriales Vicente, Raimundo García Boyero, María Teresa Olave Rubio, P. Herrera Puente, O. Arce, Manuel Barrios García, Maria Jose Sayas Lloris, M. Polo, M. I. Gómez Roncero, E. Barragán, Rosa Ayala, C. Chillon, M. Calasanz, B. Boluda, A. Peleteiro Raíndo, R. Amigo, D. Martínez-Cuadrón, Jorge Labrador, P. Montesinos
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Genetic alterations trigger tumorigenesis in a cell type-specific manner, but the biological basis for this, remains unclear. Here we define reinforced original cell identity established by interaction between genetic alterations and original cell type is a pivotal determinant of tumor development. Using AML-M5MLL-AF9 as a model system, we demonstrated that MLL-AF9 enhanced key transcriptional factors (TFs) expression of promonocyte and cooperated with them to amplify their downstream programs for AML hallmarks. Interactions between MLL-AF9 and promonocyte endowed cancer cells with a reinforced promonocyte identity. Reinforced intrinsic cell identity also sustained cancer hallmarks across cancer types. Furthermore, we identified narciclasine, as an effective disruptor of reinforced promonocyte identity, effectively treating AML-M5MLL-AF9. This study not only identified reinforced cell identity as a fundamental tumor driver beyond genetic mutations but also offers a proof-of-concept for reinforced identity-targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641096df61080db2589b3baf959c31f742003e7c" target='_blank'>
              Reinforced identity confers cells with cancer hallmarks
              </a>
            </td>
          <td>
            Haixia Yang, Pengqi Wang, Pengcheng Tan, Qiaomei Xue, Hua Wu, Kezhang He, Bowen Wang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2024-11-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 [Introduction]
 Despite the enormous amount of information available due to the recent advances in DNA sequencing technologies, most genes remain elusive in terms of their biological functions. Given that the responsible gene mutations have not yet been determined in some patients with blood cancers, it is important to search for new essential genes in the development of the hematopoietic system. On the other hand, mammalian cells have diploid genes in the nucleus, and many genetic abnormalities are complemented by normal alleles, making it difficult to screen for recessive genes. To overcome this problem, we previously constructed a homozygous mutant embryonic stem cell (ESC) bank by introducing a mutation in one allele using a gene trap vector with transposon properties, followed by transient termination of the expression of the Bloom syndrome gene that suppresses homologous interchromosomal recombination.
 [Methods]
 In this study, we extend our method to screen for recessive genetic abnormalities involved in the development of hematopoietic cells, using the established method of in vitro hematopoietic differentiation by co-culture with OP9 stromal cells.
 [Results]
 To discover uncharacterized genes in the mutant ESC clones, we focused on 21 genes with uncharacterized functions in the hematopoietic system from the approximately 200 homozygous mutant clones in the bank. Of the homozygous mutant ESC clones tested, one clone was severely impaired in terms of differentiation potential toward hematopoietic cells. In this clone, the gene-trap vector was inserted in a previously uncharacterized gene in a biallelic manner. We named this gene “attenuated hematopoietic development (Ahed)”. According to the gene expression database, the expression level of Ahed was higher in tissues of hematopoietic lineages than in other tissues. To understand its specific roles in the hematopoietic system, we generated Ahed conditional knockout (cKO) mice by crossing Ahed-floxed mice with Vav1-cre transgenic mice. We found that Ahed cKO fetuses were normal in gross appearance until E12.5, however, they became anemic from E14.5, characterized by fetal liver (FL) atrophy, leading to prenatal demise. To determine the significance of Ahed in fetal hematopoiesis, we analyzed the E14.5 liver of Ahed cKO fetuses. Flow cytometric analysis revealed that the number of TER119+-erythroid cells was significantly decreased in Ahed cKO livers. By contrast, Ahed cKO fetal livers contained a substantial number of lineage−Sca1+c-kithigh (LSK) cells, representing hematopoietic stem/progenitor cells. We then assessed the hematopoietic function of these LSK cells. Transplantation experiments demonstrate the incapacity of Ahed-deficient hematopoietic cells to reconstitute hematopoiesis in vivo. Employing a tamoxifen-inducible cKO model in adult mice, Ahed cKO mice developed severe hematopoietic failure and all mice died within one week of induction, indicating that Ahed deletion impairs the intrinsic capacity of hematopoietic cells in adult bone marrow. RNA-sequencing analyze revealed that pan-hematopoietic Ahed deletion disrupts critical gene expression such as Gata2 and Runx1. Based on information from a comprehensive database of protein-protein interactions and computer analyses to infer function from protein structure, the Ahed protein was speculated to be involved in the regulation of RNA splicing. In fact, our study confirmed that Ahed protein are localized in the nuclei and mRNAs of genes involved in hematopoiesis including PU.1 are aberrantly spliced in Ahed-deficient cells. Furthermore, published databases present cancer patients with somatic mutations in Ahed human orthologue, c11orf57, especially those with hematologic malignancies such as acute myeloid leukemia.
 [Conclusion] We report the first attempt of in vitro hematopoietic differentiation screening of mutant mouse ESC lines and the successful identification of a previously uncharacterized gene, Ahed, whose perturbation disrupts early hematopoiesis both in vitro and in vivo. Our findings underscore the fundamental roles of Ahed in lifelong hematopoiesis, implicating its association with malignancies. Although precise molecular machineries by which Ahed is involved in hematopoiesis have remained to be elucidated, further analyses of the molecular mechanism underlying their function might pave the way for novel cancer therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c0e76c79e27b83317fe5efadc365ff2219a506" target='_blank'>
              A Newly Identified Gene Ahed Plays Essential Roles in Murine Hematopoiesis through RNA Splicing
              </a>
            </td>
          <td>
            R. Nakai, Takafumi Yokota, Masahiro Tokunaga, M. Takaishi, Tomomasa Yokomizo, T. Sudo, Henyun Shi, Yoshiaki Yasumizu, Daisuke Okuzaki, C. Kokubu, Sachiyo Tanaka, Katsuyoshi Takaoka, A. Yamanishi, Junko Yoshida, Hitomi Watanabe, Gen Kondoh, Kyoji Horie, Naoki Hosen, S. Sano, J. Takeda
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The response to targeted therapies and immune checkpoint inhibitors for patients suffering from metastatic clear cell renal cell carcinoma (ccRCC) is heterogeneous and currently not predictable in clinic. In this work, a comprehensive integrated study of 700 ccRCCs profiled by DNA methylation and RNA sequencing showed that the hyper-methylated tumors exhibited a worse prognosis, a higher fraction of cycling tumor cells and a lower activity of homeobox transcription factors. To translate the use of DNA methylation information into a clinical setting, we developed a simple model accurately predicting the ccRCC methylation subtypes (AUC-ROCs of 0.91) from two gene expression ratios (IGF2BP3/PCCA, TNNT1/TMEM88). In addition, these methylation subtypes were significantly associated with the therapeutic outcome of patients to anti-PD-1, mTOR inhibitor or tyrosine kinase inhibitor therapies. Overall, our framework for predicting the ccRCC DNA methylation subtypes from targeted gene expression data is easy to translate in clinic and contributes to better personalization of ccRCC therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0521226e2f11211160c8cebde1cb06c9530046ba" target='_blank'>
              DNA methylation is a key determinant of response to targeted and immune checkpoint therapies in metastatic renal cell carcinoma
              </a>
            </td>
          <td>
            Florian Jeanneret, Sarah Schoch, Pedro Ballester, Stefan N Symeonides, Alexander Laird, Håkan Axelson, Delphine Pflieger, Christophe Battail
          </td>
          <td>2024-11-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dd1105cff05b2e45c6ea02a760f9287cdb6ad6a" target='_blank'>
              Recent advances on biogenesis, functions and therapeutic potential of long noncoding RNAs in T cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            P. Kaur, Pankaj Sharma, P. Bhatia, Minu Singh
          </td>
          <td>2024-11-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/637c216e6b43ea38be0efac827cb593c26dd881e" target='_blank'>
              Genetic and epigenetic bases of long-term adverse effects of childhood cancer therapy.
              </a>
            </td>
          <td>
            Zhaoming Wang, Jinghui Zhang
          </td>
          <td>2024-11-07</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="

 Introduction
 The pathogenic role of somatic non-coding (nc) single nucleotide variants (SNVs) is underexplored in multiple myeloma (MM) and additional characterization of these regions has the potential to provide significant new insights into disease progression. To investigate the role of ncSNVs and their relationship with MM pathogenesis, we have characterized a novel series of WGS from 294 cases of MGUS, SMM, and MM using computational tools optimized to identify functional driver SNVs in these regions. We hypothesize the ncSNVs that co-localize to sites of super-enhancers (SEs) active in plasma cells (PC) are sites of high functional impact SNVs and these mutations lead to the dysregulated expression of genes that should be silent at the terminally differentiated stage of PCs, providing a selective advantage that shapes the fitness of the MM genomic landscape.
 Methods
 We analyzed 294 high-coverage matched tumor/normal pairs (80x tumor and 40x normal) derived from 21 MGUS, 86 SMM, 167 NDMM and 40 RRMM. In addition to previously sequenced samples, newly sequenced patients were obtained from NYU Langone Health, Memorial Sloan Kettering Cancer Center, University of Alabama at Birmingham and University of Miami following IRB-approved informed consent. DNA was extracted from CD138+ purified bone marrow plasma cells using standard approaches. We used the MGP1000 (https://github.com/pblaney/mgp1000) to process sequencing data which employs a union-consensus mechanism for determining the final set of variants. We collected enhancer elements (E/SEs) active in GC B-cells, ABC-DLBCL and GBC-DLBCL (Bal, et al. Nature, 2022) and PC specific E/SEs (Loven, et al. Cell 2013). FunSeq2 was used to score ncSNVs. GRAM was used to predict how variants alter TF binding and gene expression in a subset of 6 cases with paired WGS in our series and will expand this analysis on a validation set of 263 NDMM and 93 RRMM cases from University-Hospital Heidelberg enrolled in the GMMG-HD6 clinical trial.
 Results
 We identified 1,952,232 high-quality SNVs with 18,199 (0.93%) accumulating in 280 focal sites we termed “mutationally enriched regions” or MERs. To discern SNVs in MERs with the highest potential for imparting a selective advantage, we clustered mutations based on the density of FunSeq2 impact scores which uses reference data contexts on sequence conservation, modification of transcription factor (TF) binding, and gene linkage to regulatory elements and network hubs from pan-cancer studies. We show that MERs are enriched for ncSNVs with high impact scores (mean=2.57) compared to random similarly-sized regions of the genome (Fisher's exact test P<9.7e-12, OR=10.10 [3.88-37.40]). The high impact ncSNVs (hi-ncSNVs) in MERs (n=220) account for 8.09% of all hi-ncSNVs and are present in 46.3% of cases (136/294) with some patients having up to 7. The proportion of cases carrying a MER hi-ncSNVs was similar across disease stages (MGUS=42.9%, SMM=46.5%, MM=40.7%, and RR=47.5%). Analysis of the VAF of MER hi-ncSNVs shows a minor shift towards more clonal versus subclonal mutations (53.6% vs 46.4% VAF>=0.25) suggesting some high impact mutations are acquired in the GC but additional hi-ncSNV are acquired later in PC development, providing evidence that hi-ncSNVs accumulate in MERs throughout disease progression. We mapped these hi-ncSNVs to associated genes then intersected the loci with E/SEs active in GCBs and PCs to determine their functional context. This approach identified 12 genes affected by hi-ncSNVs including the previously identified nc-drivers NEAT1, MIR142, ZCCHC7, and IGLL5 as well as the PC differentiation genes IRF8, BACH2, proliferation genes RHOH, SPRED2, known coding MM drivers CCND1, ZFP36L1, and chr1q genes BTG2 and CROCC. We predicted the effect of ncSNVs on expression through modulation of TF binding at this set of putative nc drivers in MM and identified a trend towards increase in expression of SPRED2, CCND1, NEAT1, and IGLL5 compared to non-mutated cases (P=0.089). We will present data on the validation of this set of genes on a larger set of data derived from the Heidelberg group.
 Conclusions
 We identified a set of 12 putative nc driver genes in MM that modify expression and have the potential to significantly alter the pathogenesis of MM by providing a selective survival advantage. We will present the impact of these by clinical stage and on outcome.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d27e02bc3368053980f07501df92720db4c496" target='_blank'>
              Mutationally Enriched Regions of the Non-Coding Genome Harbor Functionally Impactful Mutations That Dysregulate Key Genes Involved in Multiple Myeloma Pathogenesis
              </a>
            </td>
          <td>
            P. Blaney, K. Maclachlan, M. Papadimitriou, A. Mikulasova, Dylan Gagler, Marc J Braunstein, O. Lahoud, David Kaminetzky, Yubao Wang, Sanxiong Liu, Di Zhang, Anthony Cirrincione, A. Poos, B. Ziccheddu, Marcella Kaddoura, M. Chojnacka, B. Diamond, N. Korde, A. Lesokhin, Saad Z Usmani, L. Rasche, E. Mai, M. Raab, Niels Weinhold, Elizabeth E. Brown, N. Robine, L. Bergsagel, Brian A Walker, O. Landgren, F. Maura, F. E. Davies, G. Morgan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 Autophagy induction has recently emerged as a mechanism of resistance to FLT3 inhibitors (FLT3i) in patients with FLT3-ITD mutant acute myeloid leukemia (AML). Molecularly, this resistance is facilitated by autophagy-mediated degradation of drug-targeted proteins and via activation of pro-survival pathways in FLT3-mutant cells. Chloroquine (CQ), an FDA-approved antimalarial drug, is a safe alternative to inhibit autophagy and to overcome resistance in myeloid neoplasms. By applying a multiomic approach, we assessed the molecular mechanisms by which CQ increases the efficacy of FLT3i in FLT3-ITD resistant models. Transcriptome analysis of ex vivo AML samples (n=78) treated with FLT3i (midostaurin, PKC and quizartinib, AC220) revealed enriched signatures related to mitochondrial metabolism and autophagy in resistant samples. We validated these findings in a second cohort of patients treated with PKC (n=13), where we observed an increased mitochondrial membrane potential in CD34+ cells from poor responders. Additionally, proteome profiling of CD34+/CD117+ cells revealed high expression of proteins associated with LSC-UP and LMPP programs in poor responders. In FLT3-ITD AML cell line models (MOLM13 and MV4-11), treatment with PKC and AC220 led to an increase in acidic vesicular organelles (AVOs) and reduction in the levels of autophagy-related proteins, such as LC3BI/II and p62. This indicated induction of autophagy by FLT3i. Combining FLT3i (PKC and AC220) with CQ further decreased cell survival compared to FLT3i monotherapy in both cell lines and primary patient samples. In vivo, the combination of PKC+CQ improved overall survival and preserved healthy hematopoiesis in MOLM13-transplanted mice, which was not observed with PKC (6 mg/Kg, q.d. orally) or CQ (20 mg/Kg q.d. intraperitoneally) monotherapies. To determine whether CQ induces cell death through autophagy inhibition or via other off-target effects, we performed genetic knockdown (KD) of the autophagy-related ATG5 and ATG7 genes. ATG5- and ATG7-KD increased sensitivity to both FLT3i, with no additional effects from CQ, indicating that CQ-induced cell death relies on the pharmacological inhibition of autophagy. To investigate the potential of CQ in overcoming FLT3i resistance, we generated MV4-11 cells resistant to AC220 (MV4-11 QR) through selective pressure. MV4-11 QR cells displayed higher basal levels of autophagy compared to normal MV4-11 cells. The combination CQ+AC220 demonstrated a synergistic effect in MV4-11 QR cells (ZIP score 10.8) enhancing cell death more than the monotherapies. This effect was associated with greater inhibition of FLT3, pSTAT5 and pP70S6K compared to the monotherapies. At the molecular level, CQ potentiates the efficacy of AC220 by significantly inhibiting the autophagy mediator ATG7, with this effect being more pronounced in the combination treatment than with CQ alone. To identify new targets associated with the response to CQ+FLT3i, we performed label-free quantification proteomics on MOLM13 cells treated with CQ, PKC, and their combination. We found that 46 proteins were downregulated across all 03 conditions (CQ, PKC and CQ+PKC), mostly related to cell proliferation and survival (BAX, SMARCA4 and SUMO1). RCF4 (an autophagy regulator linked to increased chemosensitivity) and GATD3 proteins were upregulated only in the CQ+PKC group. Additionally, 12 proteins were downregulated in the CQ+PKC group, which are associated with FLT3i resistance in FLT3-ITD models. Finally, we showed thatFLT3i treatment induces autophagy flux, which can limit the inhibitory effectiveness of the drug. Combining FLT3i with CQ enhances its antileukemic efficacy by overcoming this pharmacological resistance. While CQ is generally well-tolerated clinically, patient heterogeneity must be considered. Assessing the autophagic response upon FLT3i in AML could significantly improve personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43ec0dcc7d33ce8b1014b15e15480c10e72c73e8" target='_blank'>
              Molecular Mechanisms Associated with Autophagy Inhibition to Overcome Resistance to FLT3 Inhibitors
              </a>
            </td>
          <td>
            Manuela Albuquerque de Melo, D. Pereira-Martins, Brunno Gilberto Santos Macedo, Lívia Bassani Lins de Miranda, P. S. Scheucher, I. Weinhäuser, G. Huls, J. Schuringa, J. Machado-Neto, Fabíola Traina
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="

 Acute myeloid leukaemia-normal karyotype (AML-NK) comprises almost half of the AML subtypes and exhibits clinical heterogeneity in treatment response and outcome. Hitherto, the genomic landscapes of AML-NK that contribute to the clinical outcome remain veiled. Therefore, this study elucidated the genomic profiles and regulatory networks of AML-NK patients predisposed to their heterogeneous clinical outcome. In this study, 51 AML-NK samples at diagnosis (DX) and 14 paired first complete remission (CR1) were recruited for transcriptome sequencing and eight paired DX and CR1 DNA were selected for targeted DNA sequencing. The targeted DNA sequencing using the Archer HGC VariantPlex Myeloid panel that included 75 myeloid-related hotspots genes led to the ascertainment of mutations for risk stratifications, and suitable biomarkers for minimal residual disease monitoring were put forward in this analysis. The transcriptome sequencing yielded discoveries of DEG profiles and functionally enriched pathways in several subgroup analyses that included the comparison of AML-NK patients with the healthy normal groups, paired DX and CR1, FLT3/NPM1 genotypes, and overall survival (OS) of below and above five years. The DEGs between the DX and CR1 suggested their potentiality for MRD monitoring, especially in AML-NK patients who lacked genomic aberrations. The highlights of the DEG findings are the development of a prognostic scoring model based on the findings of the OS below and above five-year comparison. Six significantly upregulated genes in the (FHL1, SOCS2, IL17RC, STAT4, INHBA and TNFSF8) in the JAK-STAT signalling pathway and cytokine-cytokine receptor interaction were included in the prognostic scoring model that revealed that the gene scores were an independent prognostic marker in the AML-NK patients in this cohort. Next, fusion gene analysis disclosed several novel recurrent fusion genes, including LATS2-SAP18 and HOXA3-HOXA9 that exhibited prognostic relevance in patients with OS below five years. Clinically relevant somatic variants were discovered, including five known single nucleotide variants (SNVs) with targeted therapies. Prognostically significant frameshift insertion-deletions (InDels) were detected in the NPM1, DNMT3A and FLT3 genes. Based on established guidelines, this study incorporated the AML-NK patients' genomic findings and risk stratification. Ultimately, the findings were depicted in an oncoprint that reflected how the genomic discoveries in this study improvised the patient's risk stratification for outcome predictions and potential targeted therapies. Hence, this multifaceted study has provided new insights into the genomic profiles of AML-NK patients and shed light on their heterogeneous clinical outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dafbbf61a9252a2ba0a373f58ee0097b020fc4a" target='_blank'>
              Demystifying Genomic and Transcriptomic Landscape of Acute Myeloid Leukaemia-Normal Karyotype Using Deep Sequencing Technology
              </a>
            </td>
          <td>
            A. Ambayya, Rozaimi Razali, Tariq Mahmood Roshan, Adnan Mansoor, S. Sulong, R. Hassan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The ability to control gene expression is vital for elucidating gene functions and developing next-generation therapeutics. Current techniques are challenged by the lack of cell-specific control designs or immunogenicity risk from foreign proteins. We develop a DNA repair inducible ribozyme switch that enables cell-specific control of gene expression in cells and in vivo. This strategy designs plasmids with a DNA lesion (8-oxoG and O6-MeG) site-specifically installed within the ribozyme encoding region, generating active hammerhead ribozyme for mRNA degradation due to transcriptional mutagenesis, whereas DNA repair yields a single-base mismatch in the ribozyme to abrogate its activity. This strategy is demonstrated to allow specific control of gene expression in cancer cells with overexpressed DNA repair enzymes such as MutY DNA glycosylase and O6-methylguanine-DNA-methyltransferases. It also shows the capability of conditionally regulating the expression of different proteins for signal reporting and gene editing, enabling DNA repair monitoring and targeted gene therapy in cancer cells. This strategy is demonstrated using the inducible CRISPR/Cas9 system for in vivo editing of oncogenic Polo-like kinase 1 in a mouse model, resulting in significant tumor growth suppression. The DNA repair inducible ribozyme switch may provide a compact system for cell-specific gene expression control toward precise gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c1afdf54d3dd3272e9f3419020e47be49904e6" target='_blank'>
              Cell-Specific Control of Mammalian Gene Expression Using DNA Repair Inducible Ribozyme Switches.
              </a>
            </td>
          <td>
            Jieling Hong, Zhen Jiang, Zhenkun Wu, Jian-Hui Jiang
          </td>
          <td>2024-12-07</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Replication-repair deficiency (RRD) stemming from mismatch repair and polymerase proofreading gene defects (MMRD/PPD) lead to hypermutant childhood cancers, most frequently brain tumors. While medulloblastoma is reported, the clinical, genomic and immune landscape remains unknown.



 We analysed the genome, methylome, transcriptome (bulk, single-nuclei) and immune-microenvironment of the largest cohort of RRD-medulloblastoma patients enrolled by the International RRD Consortium and correlated these to clinical outcomes. RRD mice-models were used to preclinically assess response to immunotherapy. Results:



 RRD-medulloblastoma (n=43) were enriched for anaplasia (61%) and localised disease (77%). Methylation/nano-string-based subgrouping failed to classify 40%, while the remaining clustered with the SHH-subgroup. Copy-number changes were notably infrequent (<20%). All tumors harboured hypermutation and microsatellite instability in contrast to non-RRD controls (p<0.0001). Pathogenic variants were frequent in POLE/POLD1 (80%), TP53 (48%) and SHH-pathway genes (PTCH1, SUFU, SMO: 56%). Interestingly, some tumors additionally had glioma-driver alterations (ATRX, NF1: ~50%), similar to RRD cerebellar glioblastoma, but distinct from both non-RRD medulloblastoma (n=791) and hemispheric glioblastoma (n=733) controls. Moreover, although bulk-transcriptome matched non-RRD SHH medulloblastoma, single-nuclei analyses suggested an earlier cell-of-origin potentially explaining the heterogenous phenotypes resulting from the driver mutations. Uniquely, immune analyses (gene expression and immunohistochemistry) demonstrated high intra-tumoral CD8 T-cell infiltration. Three-year progression-free survival was 60%. Outcome was worse for RRD-medulloblastoma harbouring TP53-mutation (p=0.04) and failing subgroup-classification (p=0.004). Nestin/Cre MSH2-/- POLE;p.S459F mice developing medulloblastoma recapitulated human disease and demonstrated response to anti-PD1 monotherapy. Recurrent/progressive human tumors including those harbouring TP53-mutations exhibited radiological responses to anti-PD1 monotherapy, which was associated with prolonged ongoing survival in comparison to those not treated with checkpoint-inhibitors (p=0.02).



 Hypermutant RRD-medulloblastoma reveal heterogenous phenotypes due to unique spectrum of driver mutations and an early cell of origin. Their immune-hot microenvironment allows successful salvage treatment with checkpoint-inhibitors for those failing chemo-radiation, including the high-risk SHH/TP53-mutant and unclassified subgroups.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce04e7f8a2b9f714e7b852d9d98fb0db358fffc" target='_blank'>
              IMMU-14. TRANS-SPECIES ANALYSIS OF REPLICATION-REPAIR DEFICIENT (RRD) MEDULLOBLASTOMA AND RESPONSE TO IMMUNE CHECKPOINT INHIBITION: AN IRRDC REPORT
              </a>
            </td>
          <td>
            Anirban Das, Nick Fernandez, A. Levine, Kyle Smith, Evan Wang, M. Galati, Z. Aamir, Jill Chung, L. Negm, Hope Friedman, Katharine O’Flaherty, Owen Crump, Quang Trinh, N. Nunes, V. Bianchi, L. Stengs, M. Edwards, Lincoln Stein, E. Bouffet, Michael Taylor, P. Northcott, Vijay Ramaswamy, C. Hawkins, U. Tabori
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa5e27e413e8d0f42c28b9018464ee24128a7aa" target='_blank'>
              Unraveling the mechanisms of glioblastoma's resistance: investigating the influence of tumor suppressor p53 and non-coding RNAs.
              </a>
            </td>
          <td>
            Qamar A. Alshammari, Saud O. Alshammari, Abdulkarim Alshammari, Moaddey Alfarhan, Fahad Hassan Baali
          </td>
          <td>2024-10-30</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Fatty acid binding protein 7 (FABP7) is a multifunctional chaperone involved in lipid metabolism and signaling. It is primarily expressed in astrocytes and neural stem cells (NSCs), as well as their derived malignant glioma cells within the central nervous system. Despite growing evidence for FABP7’s tumor-intrinsic onco-metabolic functions, its mechanistic role in regulating the brain tumor immune microenvironment (TIME) and its impact on prognosis at the molecular level remain incompletely understood. Utilizing combined transcriptome profiling and pan-cancer analysis approaches, we report that FABP7 mediates the expression of multiple onco-immune drivers, collectively impacting tumor immunity and clinical outcomes across brain cancer subtypes. An analysis of a single-cell expression atlas revealed that FABP7 is predominantly expressed in the glial lineage and malignant cell populations in gliomas, with nuclear localization in their parental NSCs. Pathway and gene enrichment analysis of RNA sequencing data from wild-type (WT) and Fabp7-knockout (KO) mouse brains, alongside control (CTL) and FABP7-overexpressing (FABP7 OV) human astrocytes, revealed a more pronounced effect of FABP7 levels on multiple cancer-associated pathways. Notably, genes linked to brain cancer progression and tumor immunity (ENO1, MUC1, COL5A1, and IL11) were significantly downregulated (>2-fold) in KO brain tissue but were upregulated in FABP7 OV astrocytes. Furthermore, an analysis of data from The Cancer Genome Atlas (TCGA) showed robust correlations between the expression of these factors, as well as FABP7, and established glioma oncogenes (EGFR, BRAF, NF1, PDGFRA, IDH1), with stronger associations seen in low-grade glioma (LGG) than in glioblastoma (GBM). TIME profiling also revealed that the expression of FABP7 and the genes that it modulates was significantly associated with prognosis and survival, particularly in LGG patients, by influencing the infiltration of immunosuppressive cell populations within tumors. Overall, our findings suggest that FABP7 acts as an intracellular regulator of pro-tumor immunomodulatory genes, exerting a synergistic effect on the TIME and clinical outcomes in brain cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15939c78a79b5a83578829fc7d538cf232f24932" target='_blank'>
              Integrated Transcriptome Profiling and Pan-Cancer Analyses Reveal Oncogenic Networks and Tumor-Immune Modulatory Roles for FABP7 in Brain Cancers
              </a>
            </td>
          <td>
            Yool Lee, Carlos C. Flores, Micah Lefton, Sukanya Bhoumik, Yuji Owada, Jason R. Gerstner
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e682f9ff1366b8e5398704d90d9b219b2fb57501" target='_blank'>
              Whole genome sequencing of CRISPR/Cas9-engineered NF-κB reporter mice for validation and variant discovery
              </a>
            </td>
          <td>
            Guruswamy Mahesh, Erik W. Martin, Mohammad Aqdas, Kyu-Seon Oh, Myong-Hee Sung
          </td>
          <td>2024-11-13</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 The treatment of many solid tumors presents significant challenges with chemotherapy, radiation therapy, and the limited effectiveness of immunotherapy. Targeted therapy offers a promising approach, yet the lack of validated targets limits its applicability across the spectrum of solid tumors. Our past studies have revealed that an R-loop and mRNA-dependent DNA repair (RDDR) pathway, induced by damage at the transcribed regions of the genome, contributes to cell survival and drug resistance in cancer cells. This is particularly due to the high levels of transcription and DNA damage in these cells. We identified the RNA methyltransferase TRDMT1 as the primary modifier of mRNA methyl-5-cytosine in the RDDR pathway. Overcoming technical difficulties in monitoring RNA modifications, we developed the first TRDMT1 inhibitor (TRDMT1i) using in-house built cell-based and in vitro assays. We have identified highly potent and specific lead compounds that significantly reduce TRDMT1 activity at nanomolar concentrations. Our current lead TRDMT1 inhibitors meets most of the criteria that are required for optimal lead. Initial assessments involving the screening of normal and cancer cell lines, xenograft models, and patient specimens indicate that TRDMT1i sensitizes tumors with genomic instability when used as monotherapy. We aim to develop TRDMT1i as an investigational new drug (IND) for future clinical trials, initially targeting ovarian cancer as a monotherapy or in combination therapy for first-line maintenance and systematic therapy. Our goal is to pioneer the discovery of the RDDR pathway, leading to the development of innovative platforms and next-generation medications to revolutionize the targeting of mRNA modifications in cancer treatment.
 Citation Format: Li Lan. Targeting mRNA methyltransferase in RNA-dependent DNA repair in cancer therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5c058059942f92471fbbee99fbac2c5e418612" target='_blank'>
              Abstract PR001: Targeting mRNA methyltransferase in RNA-dependent DNA repair in cancer therapy
              </a>
            </td>
          <td>
            Li Lan
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Background: Acute myeloid leukemia (AML) is a heterogeneous disease characterized by expansion of myeloid progenitor cells and the acquisition of recurrent somatic mutations. Mutations commonly found in AML such as DNMT3A and TET2 are sometimes present in the peripheral blood of otherwise healthy adults - a phenomenon referred to as clonal hematopoiesis (CH). CH-associated mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy.
 Cohort description: We retrospectively compiled all patients diagnosed with de novo AML at our institution that had two or more next-generation sequencing (NGS) studies, performed at least 30 days apart, on blood or bone marrow specimens. The final cohort comprised 182 patients. All patients were profiled at diagnosis, with 119 sequenced at first remission and 76 sequenced at first relapse. The average age at diagnosis was 58.06 ± 1.03 (mean ± s.e.m.) years, and 53.3% of patients were female. In total, 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy (i.e. “7+3”).
 Results: To investigate the genomic changes in AML disease course, we calculated the change in variant allele fraction (VAFs) for all identified variants, evaluating matched samples from individual patients across time. Comparing VAFs at remission and diagnosis, we observed a wide spectrum of VAF changes varying across genes. CBL, IDH1, KRAS, NPM1, NF1, and NRAS variants predominantly had decreased VAFs at the time of remission. FLT3 and NPM1 variants that were initially detected at diagnosis were generally undetectable at remission (variants lost: FLT3: 35/45 NPM1: 26/29). Conversely, only one FLT3 and no NPM1 variants were newly identified at remission compared to diagnosis. Variants identified at diagnosis in the CH-associated genes ASXL1 (1/17 variants), DNMT3A (6/41 variants), and TET2 (10/40 variants) were much less likely to have been eliminated at remission. This is in contrast with FLT3 and NPM1 mutations, which rarely persist at the time of remission. Similar analyses were undertaken at first relapse: FLT3 and NPM1 mutations disappear at the time of remission and reemerge at relapse, consistent with their function as potent AML driver alterations, Meanwhile DNMT3A, TET2 and ASXL1 were generally maintained throughout the disease course in spite of chemotherapy and observed cytologic remission. Overall, these data indicate that CH-associated mutations likely exist synchronously in a preleukemic cell compartment that is not effectively eliminated by AML-directed therapy.
 We next sought to determine if early DNMT3A and TET2 mutations subsequently impact the genomic trajectories in the course of AML remission and relapse. Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. Of note, NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. In contrast, CBL variants were both gained and lost between diagnosis and relapse in TET2mut samples, but no CBL variants were observed in DNMT3Amut samples. Finally, we applied the CALDER algorithm to infer phylogenetic relationships from matched longitudinal AML sequencing data in individual patients. We observed that DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile.
 Conclusions: We demonstrate that DNMT3A and TET2 exhibit distinct constellations of co-occurring genetic alterations at the time of diagnosis and at relapse. Thus, early CH-associated mutations that precede malignant transformation can subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/474f56cb7bd9fa118218a7a3037641d6f1f25631" target='_blank'>
              Early Drivers of Clonal Hematopoiesis Shape the Subsequent Evolutionary Trajectories of De Novo AML
              </a>
            </td>
          <td>
            Ryan D Chow, Priya Velu, Safoora Deihimi, Jonathan Belman, Angela Youn, Nisargbhai Shah, Jennifer J D Morrissette, Robert L Bowman
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Genomic studies of the malignant brain tumor glioma invariably rely on aligning sequencing reads to a reference genome such as HG38, even though no single reference can capture the diversity of an individual patient or cell line. To begin addressing these biases in genomics research, we sought to construct a chromosome-scale diploid-phased assembly of a normal human astrocyte (NHA) cell line that is frequently used as a basis for developing isogenic glioma cell models. To achieve this, we generated 45x Pacific Biosciences HiFi data (N50: 16kb), 26x Oxford Nanopore Technologies ultra-long data (N50: 80kb), and 2 billion paired-end reads of Hi-C data. The assembler verkko2 was used to integrate these data and build a phased diploid assembly. We achieved an assembly size of 6.01 Gbp, with 17 gapless chromosomes. The two haplotypes contain 214 and 182 contigs, respectively. Our assembly includes 76 fully assembled telomeres and 20 fully assembled centromeres. The contiguity of our assembly is 130 Mbp, which vastly exceeds that of HG38 (N50: 68 Mbp) and approaches that of the recent telomere-to-telomere CHM13 assembly (N50: 155 Mbp). We ran BUSCO on each haplotype of the assembly using the metazoa database to quantify the single-copy genes that are expected to be present in every animal genome. The haplotypes of our assembly contain the complete sequence of 98.6% and 99.9% of these genes respectively. Finally, we used Merqury to perform a k-mer based evaluation of the two haplotypes and found quality values of 59.09 and 59.75 respectively. This diploid genome assembly of NHA will serve as a resource for researchers that wish to develop NHA-derived glioma models. It can be used to call indels or structural variants in a haplotype-specific manner which will further elucidate genotypes underlying glioma phenotypes. Most importantly, it sets the stage for personalized genome assembly and tackling the lack of diversity in genomic studies of glioma to date.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2325a1e30af3626a8d3da00529a91462ed9452ce" target='_blank'>
              EPCO-45. COMPLETE CHROMOSOME-SCALE DIPLOID-PHASED ASSEMBLY OF A NORMAL HUMAN ASTROCYTE CELL LINE
              </a>
            </td>
          <td>
            T. R. Ranallo-Benavidez, Yue Hao, Noelle Fukushima, Ogechukwu Mbegbu, Floris P Barthel
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="

 Introduction: Epigenetic alterations are known to contribute to development of T-cell lymphomas (TCL), and numerous epigenetic modifiers have shown safety and clinical efficacy in TCL leading to FDA approval of three histone deacetylase inhibiters (vorinostat, romidepsin, and belinostat). As single-agent therapies in relapsed/refractory TCLs overall response rates are ~30% with median time to response of ~2 months and duration of response typically less than 1 year. Belinostat is generally well tolerated with infrequent high-grade toxicities in relapsed TCL. It would be ideal to identify drugs that could be combined with epigenetic therapies, such as belinostat, to improve efficacy and overcome resistance. Further, identification of highly cytotoxic drugs that could provide rapid tumor debulking while preserving non-malignant T-cells and are broadly applicable across subtypes, would be beneficial to rapidly alleviate symptoms and prevent decline in a generally older population.
 Methods: We obtained a drug library containing 729 epigenetic and cytotoxic compounds tested in human clinical trials (or FDA approved) therefore ideal for Phase 2 repurposing, and we screened these compounds' cytotoxic properties against 5 TCL cell lines representative of different TCL subtypes in vitro: HH (CTCL), Hut78 (CTCL), OCI-Ly13.2 (PTCL-NOS), Karpas299 (ALCL), and HDMAR2 (LBL). We treated the five TCL cell lines for 48 hours at four concentrations (1nM, 10nM, 100nM, and 1000nM) for each of the 729 compounds. Following 48 hours of drug, the TCL cell lines were rinsed and given fresh media. After another 17 hours of incubation cell viability was assessed by flow cytometry using a viability dye validated against 7-AAD and Annexin V. From the original 729, we identified the compounds that exhibited the most cytotoxicity and further elucidated their efficacy across a range of 8 concentrations on the 5 prior TCL cell lines; in addition we evaluated cytotoxicity on 5 more TCL cell lines: MAC-1 (CTCL), MAC-2A (ALCL), SMZ-1 (PTCL), T8ML-1 (PTCL-NOS), and DERL-2 (PTCL). We tested the top compounds against 3 unique healthy donor activated T-cells at these 8 concentrations. The same protocol using 48-hour drug exposure followed by 17 hours of incubation was utilized. At the conclusion of this screen, we tested cytotoxicity of our most encouraging compound in combination with other top performing compounds.
 Results: We identified BI-847325, bortezomib, camptothecin, CUDC-907, and WAY-118959-A as the most encouraging cytotoxic compounds that are not currently being used clinically. We compared the top 5 compounds to 4 standard of care (SOC) cytotoxic drugs (belinostat, gemcitabine, romidepsin, and vincristine) that are used clinically. We also evaluated effects of the top 5 compounds (and the 4 SOC compounds) on non-malignant healthy donor T-cells. We observed that camptothecin was the most encouraging novel compound, demonstrating high cytotoxicity while sparing healthy non-malignant T-cells. We tested cytotoxicity of camptothecin in combination with 5 compounds (belinostat, bortezomib, BI-847325, romidepsin, and vincristine) that also demonstrated cytotoxicity in the malignant T-cells with relative sparing of healthy T-cells. In vitro and using Bliss modeling, we observed potential synergy when combining belinostat to camptothecin.
 Conclusions: Our preclinical studies show that when combined with belinostat, the dose of camptothecin can be decreased by 50%, while still observing increased tumor cell killing and sparing non-malignant T-cells, thereby decreasing risk of dose limiting toxicities observed in prior camptothecin clinical trials. In vivo studies are ongoing to identify the dose and administration schedule that can be studied in a clinical trial and to establish how exatecan is synergistic with belinostat.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/253d224dc03ea50ede1d8f2e5179ded3782b8ecc" target='_blank'>
              Identifying Novel Epigenetic Therapies for T-Cell Lymphomas By High Throughput Drug Screen
              </a>
            </td>
          <td>
            B. Haverkos, Xander Bradeen, Bianca Labuschagne, Mason Kragenbring, Eduardo Davila
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 Among the various subtypes of AML, each defined by distinct oncogenic mutations, those harboring mutations in the tumor suppressor gene TP53 are the most aggressive, presenting a significant clinical challenge. Notably, while TP53 mutations are present in only 10% of patients at diagnosis, their prevalence increases to 30% in secondary AML and treatment-resistant AML, further underscoring the aggressiveness of TP53-mutated (TP53-Mut) AML. Despite efforts to target TP53-Mut AML cells, no targeted therapies have resulted in significant clinical benefit for this patient population. Thus, identifying therapeutic vulnerabilities in TP53-mutated AML is a pressing need.
 To identify these vulnerabilities, we leveraged the Cancer Dependency Map (DepMap), a comprehensive repository of genome-wide loss-of-function CRISPR-Cas9 screens across over 1000 cancer cell lines. We stratified the 26 AML cell lines in DepMap into TP53-wildtype (TP53-WT) and TP53-Mut categories and calculated the differential dependency scores between these two subtypes. As expected, we found that the number of selective dependencies in the TP53-WT population exceeded the number of selective dependencies in the TP53-Mut cell lines, underscoring the more aggressive nature of the TP53-Mut cells. The strongest differential dependencies in TP53-WT cell lines were MDM2 and MDM4, both negative regulators of TP53, thereby providing support for the quality of the screening data and our approach.
 Importantly, our analysis uncovered differential dependencies in the TP53-Mut cells, including ADAR1 and PRKRA, which were two of the highest scoring dependencies. Interestingly, ADAR1 and PRKRA dependency scores are significantly correlated (r=0.507, p =0.0081), and are components of the same biological pathway, suggesting that this pathway plays a crucial role in the survival of TP53-Mut cells. We focused on ADAR1 due to its potential druggability and multiple ongoing efforts to develop ADAR1-targeting molecules.
 ADAR1 catalyzes the conversion of adenosines to inosines (A-to-I) in double-stranded RNA (dsRNA). By editing endogenous dsRNA, ADAR1 prevents the activation of cytoplasmatic dsRNA sensors that trigger the interferon (IFN) pathway, thereby preventing autoimmune related cell death. To investigate the impact of ADAR1 loss on AML cells, we generated ADAR1 knockout (KO) cells in both TP53-WT (MV4-11, MOLM13, OCIAML3) and TP53-Mut (THP1, SHI-1, OCIM2) cell lines. Consistent with our DepMap analysis, our results showed that TP53-Mut AML cells are more sensitive to ADAR1 loss. Notably, KO of MAVS, a key activator of the IFN pathway, reduced the increase of inflammatory markers observed with ADAR1 KO and rescued ADAR1 KO mediated cell death in THP1 and SHI-1 cells. Of note, TP53-KO isogenic MV4-11 cells also exhibited increased sensitivity to ADAR1 loss compared to non-targeting TP53-WT control cells.
 To elucidate the mechanism underlying the differential sensitivity of TP53-Mut AML cells to ADAR1 loss, we performed RNA sequencing (RNAseq). Our results show that ADAR1 loss increased the expression of inflammatory pathways in both TP53-WT and TP53-Mut cell lines. However, loss of TP53 enhanced the expression of inflammatory pathways in basal conditions and after ADAR1 KO. Of note, high expression of inflammatory genes is a biomarker for sensitivity to ADAR1 loss. Importantly, we found that patient AML cells with TP53 mutations are enriched for inflammatory-related pathways compared to TP53-WT patient AML cells mirroring our findings in isogenic cells.
 TP53 can repress the expression of transposable element (TE). Thus, we hypothesized that TP53 loss leads to increased expression of immunogenic double-stranded RNA (dsRNA), rendering TP53-Mut cells more reliant on ADAR1 editing. Supporting this hypothesis, we observed higher ADAR1 expression and activity (A-to-I index) in TP53-Mut cells compared to TP53-WT cell lines. These results were also recapitulated in our MV4-11 isogenic cell lines.
 In summary, this work reveals that TP53-mut cells develop a selective dependency on ADAR1 for survival, strongly suggesting that ADAR1 is a promising therapeutic target in treating the extremely aggressive TP53-Mut AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70893ffd01f8b68560bb3e45573309b13690b2d3" target='_blank'>
              ADAR1 Is a Vulnerability in TP53-Mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yara Rodriguez, G. Alexe, Audrey Taillon, Constanze Schneider, K. Stegmaier
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Therapies targeting mutated RAS, the most frequently mutated oncogene in human cancers, could benefit millions of patients. Recently approved RAS inhibitors represent a breakthrough, but are limited to a specific KRASG12C mutation and prone to resistance. Synthetic gene circuits offer a promising alternative by sensing and integrating cancer-specific biomolecular inputs, including mutated RAS, to selectively express therapeutic proteins in cancer cells. A key challenge for these circuits is achieving high cancer selectivity to prevent toxicity in healthy cells. To address this challenge, we present a novel approach combining multiple RAS sensors into RAS-targeting gene circuits, which allowed us to express an output protein in cells with mutated RAS with unprecedented selectivity. We implemented a modular design strategy and modelled the impact of individual circuit components on output expression. This enabled cell-line specific adaptation of the circuits to optimize selectivity and fine-tune expression. We further demonstrate the targeting capabilities of the circuits by employing them in different RAS-driven cancer cells. This work highlights the potential of synthetic gene circuits as a novel therapeutic strategy for RAS-driven cancers, advancing the application of synthetic biology in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51f32597c4aa4f2a9295599f20b89af450418867" target='_blank'>
              Synthetic gene circuits that selectively target RAS-driven cancers
              </a>
            </td>
          <td>
            Gabriel Senn, Leon Nissen, Y. Benenson
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="

 Introduction
 Therapy-related myeloid neoplasms (t-MN) arise from pre-malignant hematopoietic stem and progenitor cells (HSPCs) carrying mutations in leukemia driver genes. Patient outcomes are dismal, especially when harboring mutations in TP53. The molecular mechanisms underlying the progression from TP53-mutant clonal hematopoiesis (CH) to t-MN are poorly understood. Particularly, the impact of the TP53 allelic state (i.e., mono- vs. biallelic) remains controversial, reflected by inconsistencies in the clinical classification systems of myeloid neoplasms.
 Methods and Results
 To investigate the pathogenesis of t-MN derived from TP53-mutant CH, we developed novel in vitro and in vivo model systems. By enforced Hoxb8 expression in bone marrow cells derived from Trp53-floxed-R245W-GFP mice, we generated reversibly immortalized murine HSPC lines carrying inducible and traceable Trp53 missense mutations in an either mono- or biallelic configuration. This allowed us to compare the functional impact of the Trp53 allelic states in the absence of confounding genomic alterations as well as to test for their potential to promote malignant transformation by monitoring for cytotoxic therapy-induced, Hoxb8-independent immortalization.
 Upon treatment with chemotherapeutic agents, both mono- or biallelic loss of wild-type Trp53 promoted clonal expansion due to resistance to p53-dependent G1 arrest and apoptosis, but only cells with biallelic mutations showed signs of persistent genomic instability as measured by γH2AX staining. Consequently, only cells with biallelic Trp53 mutations showed continued proliferation in the absence of Hoxb8 expression after drug-induced DNA damage indicative of immortalization. These transformed cell lines uniformly exhibited multiple numerical and structural chromosomal aberrations, reminiscent of the clinical association of biallelic TP53 mutations with complex karyotypes.
 To test the importance of the Trp53 allelic state in vivo, we generated mice with 5-10% of mono- or biallelic Trp53-mutant HSPCs without prior transplantation using SCL-CreERT2;Trp53-floxed-R245W-GFP mice. After γ-irradiation to mimic anti-cancer therapy, mono- and biallelic Trp53 mutations promoted clonal expansion of HSPCs on all levels of the hematopoietic hierarchy. Subsequently, nearly 100% of mice with biallelic Trp53-mutant HSPCs progressed to t-MN. These t-MNs were characterized by cytopenias, splenomegaly, leukemic blasts with an erythroid progenitor-like immunophenotype, and an enrichment of erythroid as well as Myc target gene transcriptional signatures - compatible with an acute erythroid leukemia (AEL). Notably, mice with monoallelic Trp53 mutations similarly developed AELs, although less frequently and with a significantly longer latency. Whole genome sequencing revealed extensive structural and numerical chromosomal aberrations as well as copy number alterations, and - importantly - all cases derived from mice with monoallelic Trp53 mutations showed spontaneous loss of heterozygosity (LOH) of the Trp53 allele including copy number-neutral LOH as well as hemizygous deletions.
 To confirm that complete loss of wild-type p53 functionality is required for malignant transformation, we examined the impact of non-mutational p53 inactivation by overexpressing its negative regulator MDM2, which is often amplified in cancer. While this had no additional effect on cells carrying biallelic Trp53 mutations, combining monoallelic Trp53 mutations with MDM2 overexpression phenocopied the biallelic state regarding abrogated p53 target gene expression, resistance to apoptosis, increased genomic instability, and potential for in vitro transformation.
 Discussion and Outlook
 We established novel and well-controlled in vitro and in vivo model systems to investigate the role of the Trp53 allelic states (mono- or biallelic) during the progression from CH to t-MN. While monoallelic Trp53 mutations are sufficient for clonal expansion in the context of DNA-damaging insults, full inactivation of wild-type p53 activity, either by a second mutation or through non-mutational inactivation via MDM2 overexpression, is necessary to induce genomic instability and transformation into highly aggressive myeloid neoplasms, characterized by complex karyotypes. Together, these findings provide novel insight into TP53-driven leukemogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10484e0923dc5e7b800c8090cd3f63fae0a19cd9" target='_blank'>
              The Pathogenesis of Therapy-Related Myeloid Neoplasms from TP53-Mutant Clonal Hematopoiesis
              </a>
            </td>
          <td>
            Jonas Fullin, Ebru Topçu, R. Schimmer, Nancy Klemm, Karolina Zielinska, Christian Koch, Francisco Caiado, Cyril Doerdelmann, Melissa Lock, Joelle Tchinda, M. Manz, Massimo Lopes, S. Boettcher
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Immunodeficiency-associated primary CNS lymphoma (PCNSL) represents a distinct clinicopathological entity, which is typically Epstein-Barr virus positive (EBV+) and carries an inferior prognosis. Genetic alterations that characterize EBV related CNS lymphomagenesis remain unclear precluding molecular classification and targeted therapies. In this study, a comprehensive genetic analysis of 22 EBV+ PCNSL, integrated clinical and pathological information with exome and RNA sequencing (RNASeq) data. Deconvolution of bulk RNA sequencing data was performed using CIBERSORTx to characterize the tumor microenvironment. EBV+ PCNSL with germline controls carried a median of 55 protein-coding single nucleotide variants (SNVs; range 24-217) and 2 insertions/deletions (range 0-22). Genetic landscape was largely shaped by aberrant somatic hypermutation with a median of 41.01% (range 31.79-53.49%) of SNVs mapping to its target motifs. Tumors lacked established SNVs (MYD88, CD79B, PIM1) and copy number variants (CDKN2A, HLA loss) driving EBV negative PCNSL. Instead, EBV+ PCNSL were characterized by SOCS1 mutations (26%), predicted to disinhibit JAK/STAT signaling, and mutually exclusive gain-of-function NOTCH pathway SNVs (26%). Copy number gains were enriched on 11q23.3, a locus directly targeted for chromosomal aberrations by EBV, that includes SIK3 known to protect from cytotoxic T-cell responses. Losses covered 5q31.2 (STING), critical for sensing viral DNA, and 17q11 (NF1). Unsupervised clustering of RNASeq data revealed two distinct transcriptional groups, that shared strong expression of CD70 and IL1R2, previously linked to tolerogenic tumor microenvironments. Correspondingly, deconvolution of bulk RNASeq data revealed elevated M2-macrophage, T-regulatory cell, mast cell and monocyte fractions in EBV+ PCNSL. In addition to novel insights into the pathobiology of EBV+ PCNSL, the data provide the rationale for the exploration of targeted therapies including JAK-, NOTCH- and CD70-directed approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b593e6dd190af8cfd93ea4290bfdcf4195f81d45" target='_blank'>
              PATH-35. INTEGRATED GENOMIC ANALYSES OF IMMUNODEFICIENCY-ASSOCIATED EPSTEIN-BARR VIRUS- (EBV) POSITIVE PRIMARY CNS LYMPHOMAS
              </a>
            </td>
          <td>
            L. Kaulen, E. Denisova, F. Hinz, L. Hai, D. Friedel, O. Henegariu, D. Hoffmann, Jakob Ito, A. Kourtesakis, P. Lehnert, S. Doubrovinskaia, P. Karschnia, L. von Baumgarten, T. Kessler, J. Baehring, B. Brors, F. Sahm, Wolfgang Wick
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Anti-cancer therapies can induce cellular senescence, which is highly stable, or drug-tolerant persistence, which is efficiently reversed upon therapy termination. While approaches to target senescent cells have been extensively studied, further understanding of the processes regulating persistence is needed to develop treatment strategies to suppress persister cell survival. Here, we used mTOR/PI3K inhibition to develop and characterize a model of persistence-associated arrest in human cancer cells of various origins. Persister and senescent cancer cells shared an expanded lysosomal compartment and hypersensitivity to BCL-XL inhibition. However, persister cells lacked other features of senescence, such as loss of lamin B1, senescence-associated β-galactosidase activity, upregulation of MHC-I, and an inflammatory and secretory phenotype (SASP). Genome-wide CRISPR/Cas9 screening for genes required for the survival of persister cells revealed that they are hypersensitive to the inhibition of one-carbon (1C) metabolism, which was validated by the pharmacological inhibition of SHMT, a key enzyme that feeds methyl groups from serine into 1C metabolism. Connecting 1C metabolism with the epigenetic regulation of transcription, the repressive heterochromatic mark H4K20me3 was enriched at the promoters of SASP and interferon response genes in persister cells, while it was absent in proliferative or senescent cells. Moreover, persister cells overexpressed the H4K20 methyltransferases KMT5B/C, and their downregulation unleashed inflammatory programs and compromised the survival of persister cells. In summary, this study defined distinctive features of persister cancer cells, identified actionable vulnerabilities, and provided mechanistic insight into their low inflammatory activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abdfb65f3c9de7ecccd1a62a48f1352298560afb" target='_blank'>
              H4K20me3-Mediated Repression of Inflammatory Genes is a Characteristic and Targetable Vulnerability of Persister Cancer Cells.
              </a>
            </td>
          <td>
            Valentina Ramponi, Laia Richart, M. Kovatcheva, Camille Stephan-Otto Attolini, Jordi Capellades, Alice E Lord, O. Yanes, Gabriella Ficz, M. Serrano
          </td>
          <td>2024-10-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="

 Introduction: Loss of function (KO) mutations in TET2 (ten-eleven translocation 2) are the most common epigenetic modifier mutations in Myelodysplastic Syndromes (MDS). TET2 catalyzes conversion of 5-methylcytosine to 5-hydroxymethytosine. Its loss is hence thought to alter the epigenetic landscape in long-term hematopoietic stem cells (LT-HSCs), thereby changing gene expression and leading to improved clonal fitness. Changes to DNA methylation can impact higher-order chromatin organization, with profound effects on transcriptional output ( Shen & Laird, Cell 2013 ) . Testing this paradigm in LT-HSCs has been challenging given the large number of cells needed for unbiased studies of chromatin organization.
 Methods: To obtain sufficient numbers of purified HSC populations for Hi-C and correlative assays, we optimized an ex-vivo expansion protocol ( Wilkinson et al, Nature 2019 ). This protocol which employs polyvinyl alcohol and a defined cytokine cocktail in serum-free media resulted in a 500-fold expansion of phenotypically homogeneous murine LT-HSCs (CD150+Lin-Sca1+cKit+) over 14 days, generating ~5 million cells per adult mouse. We performed a low-input HiC protocol (tagHiC; Zhang et al, Cell Rep 2020 ) with 0.5 million ex-vivo expanded WT and Tet2 KO ( Ko et al, PNAS 2011 ) murine LT-HSC cells. TagHiC libraries were sequenced to ~700 million paired-end reads per sample, generating over 300 million valid junction reads, sufficient for a 20-40 kb Hi-C data resolution. Valid read pairs thus determined were analyzed for chromatin compartments: A (or euchromatic) and B (or heterochromatic) and Topologically Associated Domains (TADs) using HiC-Pro and HiC Explorer packages. TAD-separation scores, a measure of the strength of TAD insulation (higher scores reflecting stronger boundaries), were also calculated. Additionally, we compared the methyl-seq data between WT and Tet2 KO conditions ( Hon et al., Mol Cell 2014 ), for determining hypermethylated differentially methylated regions (hyperDMRs) and hypomethylated DMRs (hypoDMRs) in the Tet2 KO state. DMRs were intersected with A/B compartment regions and TAD data to assess differences in A/B compartment shifts and TAD-separation scores between the hypo- and hyper- DMR regions.
 Results: Marked A/B compartment shifts and changes in TAD-separation scores were evident in Tet2 KO cells. HypoDMRs in Tet2 KO exhibited 6016 A-to-B shifts and 8627 B-to-A shifts, while hyperDMRs in Tet2 KO showed 1900 A-to-B shifts and 1705 B-to-A shifts. The Fisher's Exact Test for A-to-B shifts showed an odds ratio of 0.63 (p = 3.53e-36), indicating that hypoDMRs are less likely to shift from A to B compared to hyperDMRs. Conversely, for B-to-A shifts, the odds ratio was 1.60 (p = 3.53e-36), suggesting that hypoDMRs are more likely to shift from B to A than hyperDMRs. The compartment shifts indicate that hyperDMRs are significantly more likely to transition from A-to-B compartments in TET2 KO cells, indicating reduced chromatin activity and gene repression.
 Analysis of TAD-separation scores revealed that hyperDMR intersection regions (n=16737) in WT cells had a median score of 0.12, which reduced to 0.06 in mutant cells (t-stat: 24.5, p = 2.01e-131). Conversely, in hypoDMR intersection regions (n=57855), WT cells showed a median score of -0.06, which increased to 0.0 in mutant cells (t-stat: -7.9, p = 1.50e-15). These indicate that hyperDMR-containing TAD regions exhibit reduced separation in TET2 KO, suggesting disruption of TAD boundaries, while hypoDMR-containing TADs showed increased separation, indicating stronger TAD boundaries.
 Conclusion: Our ex-vivo expansion strategy enables high-resolution HiC analysis of LT-HSCs. Using this method, we showed that Tet2 loss reorganizes chromatin compartments, making hyperDMRs more inactive and hypoDMRs more active. Our findings strongly suggest that Tet2's effect on DNA methylation influences higher-order chromatin compartmentalization. Disruption of TAD boundaries compromises gene-regulatory networks, potentially leading to pathogenic enhancer-promoter (EP) interactions. Ongoing work will integrate epigenetic marks (histone), chromatin accessibility (ATAC-seq), and methylation (Ox-BS/BS-seq) to further understand how rewiring of EP interactions across weakened TAD boundaries causes dysregulated gene expression to confer clonal advantage in Tet2 KO.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d60c040c7fa496e709256d7e683c8a8fac87db" target='_blank'>
              Tet2 Loss in Hematopoietic Stem Cells Triggers Chromatin Reorganization through DNA Methylation Shifts
              </a>
            </td>
          <td>
            Rahul Roy, Manoj M. Pillai, Prajwal Boddu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="mRNA therapy, or mRNA-based drugs that have emerged thanks to vaccines against SARS-CoV-2, have successfully passed preclinical tests and are currently at various stages of clinical trials in the treatment of many diseases, including rare metabolic disorders. In the case of rare genetic metabolic diseases, the concept of mRNA therapy can be considered as an alternative to protein replacement therapy, where exogenous mRNA leads to the production of a fully active protein instead of a non-functional one, and also delivers it to the desired cellular compartment, such as mitochondria or the cell membrane. Preclinical studies on animal models of some rare genetic diseases have fully confrmed the validity of this concept. In this mini-review, we examine and discuss the mentioned preclinical studies on efficacy and safety in several animal models. For all the diseases considered, mRNA therapy restored functional protein to therapeutically significant levels in target organs, led to stable and reproducible results after each dose of mRNA, and was well tolerated, as confirmed by functional liver tests evaluated in animal models, including non-human primates. These data convincingly confirm the potential of clinical development of mRNA therapy for the treatment of various rare metabolic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d5d10d86b98ca08364cf7e7de8ad16e7edb6b1" target='_blank'>
              mRNA therapy effective treatment of rare hereditary diseases
              </a>
            </td>
          <td>
            K. Aitbaev, I. Murkamilov, V. V. Fomin, F. Yusupov
          </td>
          <td>2024-11-07</td>
          <td>Clinical Medicine (Russian Journal)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: Multiple myeloma (MM), a B-cell neoplasm characterized by the infiltration of malignant plasma cells into the bone marrow, is the second most common blood malignancy in the United States. The molecular pathogenesis of MM involves both genetic and epigenetic alterations. While 60-75% of newly diagnosed patients achieve a complete response or better with current modern combination therapies, almost all patients eventually progress largely due to the persistence or re-emergence of minimal residual disease (MRD). Monitoring for MRD has become essential for evaluating treatment efficacy and predicting long-term outcomes in MM patients. We aimed to investigate whether epigenetic changes, particularly 5- hydroxymethylcytosines (5hmC), at diagnosis are associated with MRD status following treatment. Methods: Utilizing 5hmC-Seal, a highly sensitive chemical labeling technique, we profiled genome-wide 5hmC in circulating cell-free DNA (cfDNA) and bone marrow genomic DNA (gDNA) at baseline and after 8 cycles of treatment (C8) from newly diagnosed MM patients (n=25) enrolled in a clinical trial (NCT01816971). MRD testing was performed on bone marrow samples using the clonoSEQ next generation sequencing assay (limit of detection 10-6). Results: The 5hmC modification levels were summarized across various genomic features, revealing enrichment in gene bodies and enhancer regions, consistent with the known regulatory role of 5hmC. We ranked 5hmC-modified genes at baseline by their variability, and found that the number of overlapping genes between cfDNA and gDNA was significantly higher than expected by permutation analysis. Subsequently, we identified the top 100 most variable 5hmC- modified genes in gDNA, which exhibited a higher Pearson correlation with cfDNA within individuals (mean r=0.90) compared to between individuals (mean r=0.82; p<0.05), suggesting that cfDNA has the potential to reflect gDNA in capturing some MM-related variations. Among the 11 patients with MRD+ at C8, a genome-wide scan of 5hmC in cfDNA comparing baseline and C8 identified 1,614 differentially modified genes (p<0.05) after adjusting for age and sex. [BC1] Furthermore, we identified 459 differentially modified genes comparing baseline 5hmC levels between patients who were MRD+ (n=11) and MRD- (n= 14) at C8 (p<0.05), after adjusting for age and sex. These results highlight the differential epigenetic landscape associated with MRD status. Conclusions: Genome-wide 5hmC profiling of cfDNA from blood revealed a similar epigenetic landscape to gDNA from bone marrow in patients with MM, suggesting that cfDNA may offer a less invasive alternative for assessing epigenetic modifications. Moreover, this profiling identified distinct epigenetic changes associated with the achievement of MRD- negativity. These findings provide a foundation for further investigation into the molecular alterations linked to MM disease biology that could potentially lead to MRD negativity. Future directions will focus on identifying biomarkers that could inform personalized treatment strategies.
 Citation Format: Zhou Zhang, Bei Wang, Krissana Kowitwanich, John Cursio, Daniel Appelbaum, Chuan He, Wei Zhang, Benjamin Derman, Brian Chiu, Andrzej Jakubowiak. Genome-wide 5-hydroxymethylation mapping in cell-free DNA to characterize the epigenetic differences in minimal residual disease of multiple myeloma [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3829eae7b79b3a46c73b54d2d250e97c8b4ba07a" target='_blank'>
              Abstract B032: Genome-wide 5-hydroxymethylation mapping in cell-free DNA to characterize the epigenetic differences in minimal residual disease of multiple myeloma
              </a>
            </td>
          <td>
            Zhou Zhang, Beibei Wang, Krissana Kowitwanich, John Cursio, Daniel Appelbaum, Chuan He, Wei Zhang, B. Derman, Brian Chiu, A. Jakubowiak
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebbbd8d75155e3982ed48d0dcc72d9da7601c268" target='_blank'>
              Concomitant loss of TET2 and TET3 results in T cell expansion and genomic instability in mice
              </a>
            </td>
          <td>
            Marianthi Gioulbasani, Tarmo Äijö, Siyao Liu, Stephanie A Montgomery, Nathan D Montgomery, David Corcoran, Ageliki Tsagaratou
          </td>
          <td>2024-12-03</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 Introduction
 DIS3 mutations in multiple myeloma (MM) are associated with a poor prognosis, therefore impacting the clinical risk of patients. Mutations in DIS3 recurrently occur at three amino acid residues, known as hotspot mutations, and are found to be heterozygous. The remaining DIS3 mutations-non-hotspot mutations-are typically homozygous and span the DIS3 protein. However, the functional difference between these DIS3 mutation groups and their cellular effects is yet to be investigated. This knowledge is critical to understanding the biological role of DIS3 mutations in the progression of multiple myeloma.
 Methods
 Patient samples (SMM, n=14; NDMM, n=36; RRMM, n=54) underwent whole genome sequencing to determine DIS3 mutation rates. Endogenous copies of DIS3 were mutated in the KMS11 cell line using CRISPR-Cas9 homology directed repair and individual colonies isolated. The mutations were individually generated and included 4 non-hotspot homozygous mutations (Y106C (PIN domain), T262P, L544P, and H788Y (all RNB catalytic domain)), 3 hotspot heterozygous mutations (D479G, D488N, and R780K (all RNB domain)), and synonymous control mutations. To examine the impact of these DIS3 mutations, we performed total RNA-sequencing for differential gene expression and pathway analysis, R-loop detection, cell proliferation analysis (Incucyte S3 live cell imaging), tandem mass spectrometry, and Western blot analysis.
 Results
 We identified DIS3 mutations in SMM (7%), newly diagnosed MM (7%) and relapsed refractory MM (19%) patient samples with increasing clonal fraction, indicating an ongoing role in disease progression and relapse. To investigate their effect, mutations were engineered in the endogenous copies of DIS3 in KMS11 cells. Differential expression analysis of mutated cell lines showed that non-hotspot mutations had a greater effect on transcription factors than hotspot mutations, resulting in 49 unique differentially expressed transcription factors which were mostly downregulated. In turn, most differentially expressed genes were downregulated in non-hotspot mutations and were involved in RNA metabolic processing, transcription, and protein processing pathways; which was not seen in the hotspot mutations. Across all mutations, lncRNAs were significantly upregulated (p<0.0001), except for Y106C, possibly due to the mutation being in a different domain from the other non-hotspot mutations.
 Although DIS3 is an essential gene, Western blot analysis revealed that non-hotspot mutations had a 25-fold decrease (p<0.0001) in DIS3 protein expression while hotspot mutations had no change in expression compared to wild-type DIS3. This difference in protein abundance could not be accounted for by changes in RNA expression, implying a post-transcriptional loss. The same loss of DIS3 protein was also observed in the KMS26 cell line which harbors a V171G non-hotspot mutation in DIS3. Loss of DIS3 expression at the protein level was not due to proteasomal degradation, as seen by MG-132 inhibition. However, we observed an increase in the expression of no-go mRNA decay proteins-such as XRN1 and SKIV2L-in the non-hotspot mutations which was not seen in the hotspot mutations.
 Mass spectrometry also showed a decrease in DIS3 protein in non-hotspot mutated cells that resulted in a vast downstream difference (|log2FC|>0.49, p<0.05) in the proteome with abundance changes in a median of 1,129 proteins (range=1,112-1,219); however, this was not found in hotspot mutated cell lines (median=3; range=0-42). Non-hotspot mutations resulted in the upregulation of RNA processing genes and histone linkers, consistent with a role in RNA processing and chromatin compaction. Despite these differences we discovered, cell proliferation analysis exhibited no difference in proliferation rates between the hotspot and non-hotspot mutation groups, as well as no difference when compared to wild-type DIS3 cells.
 Conclusions
 Our findings provide insight into the cellular impact of DIS3 mutations in multiple myeloma, highlighting key differences between hotspot and non-hotspot mutations. These results suggest distinct functional roles for DIS3 mutations which may depend on the location of the mutation. Overall, we observed the molecular effects of DIS3 mutations to further understand the mechanism of pathogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f24f9cf4eef60284e5fa781171dbb1ba8bae4e" target='_blank'>
              Non-Hotspot DIS3 Mutations in Multiple Myeloma Result in Loss of Function Accompanied By Transcriptional and Proteomic Dysregulation
              </a>
            </td>
          <td>
            Ceanne Elliott, Enze Liu, Nathan Becker, Parvathi Sudha, Oumaima Jaouadi, Brian A Walker
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="

 Multiple myeloma (MM), the second most common hematologic malignancy, exhibits complex genetic and epigenetic abnormalities in its pathogenesis. Long non-coding RNA (lncRNA), defined as non-coding transcripts of more than 200 nucleotides, has been shown to play a crucial role in various critical cellular functions, including those related to malignant diseases. The lncRNA, metastasis-associated lung adenocarcinoma transcript 1 (MALAT1), was found to be upregulated in both newly diagnosed and relapsed MM, indicating its potential oncogenic role. Previous studies also revealed therapeutic targeting MALAT exhibited a significant anti-MM effect. However, the specific mechanisms by which MALAT1 contributes to MM pathogenesis remain to be elucidated.
 This study aims to investigate the role of this druggable lncRNA, MALAT1, in the development of MM. Initially, MALAT1 was overexpressed in MM cells, and its expression, along with CD38, was further upregulated in lenalidomide-resistant MM cell lines. Knocking down MALAT1 using antisense oligonucleotides (ASO) significantly decreased MM cell proliferation/viability in both lenalidomide-sensitive and resistant cells. RNAseq-based transcriptome profiles showed that genes related to the oxidative phosphorylation (OXPHOS) pathway were significantly down-regulated following MALAT1 knockdown. Downregulation of MALAT1 further reduced adenosine triphosphate (ATP) levels and elevated the generation of reactive oxygen species (ROS), suggesting those alterations in energy metabolism and the oxidative stress response in MM cells resulted in decreased cell viability. Furthermore, using a ChIP-PCR assay, MALAT1 epigenetically modulated CD38 expression via H3K27me3 modification, involving the polycomb repressive complex 2 (PRC2) complexes with H3K27 methyltransferase EZH2 and EED. Taken together, targeting the druggable lncRNA MALAT1 suppresses MM cell growth, reduces CD38 expression through epigenetic regulation by the PRC2 complexes, and has potential applications in treating lenalidomide resistance in MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb9f56bc1755eb965f303d2ac82e9316a1990a6" target='_blank'>
              Targeting Druggabble Long Non-Coding RNA MALAT1: Epigenetic Regulation of CD38 and Energy Metabolism in Multiple Myeloma
              </a>
            </td>
          <td>
            Shih-Feng Cho, Wen-Hsin Chang, W. Liao, Tsung-Jang Yeh, Hui-Ching Wang, Jeng-Shiun Du, Chin-Mu Hsu, Hui-Hua Hsiao, Yi-Chang Liu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce415c1d6cd65bcfca677d71376e494061ba037" target='_blank'>
              Genome-wide profiling of DNA repair proteins in single cells
              </a>
            </td>
          <td>
            Kim L. de Luca, Pim M. J. Rullens, Magdalena A. Karpinska, Sandra S. de Vries, Agnieszka Gacek-Matthews, L. Pongor, G. Legube, Joanna W. Jachowicz, A. M. Oudelaar, Jop Kind
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="

 Background
 Loss-of-function TET2 mutations (TET2MT) occur in up to 30% of Myelodysplastic Syndrome and Acute Myeloid Leukemia. In Chronic Myelomonocytic Leukemia it is > 66%. Additionally, TET2MT is a founding lesion in myeloid neoplasms (MN), frequently appearing in clonal hematopoiesis of indeterminate potential (CHIP), a prodromal condition that increases the risk for MN and other diseases. Therefore, targeting the founder TET2MT clone represents a promising strategy to disrupt clonal proliferation at its source. We discovered that the loss of TET2 leads to metabolic rewiring, making cancer cells vulnerable to therapeutic interventions targeting glutamine metabolism (Gu et. al. Blood 2023). CB839, a potent glutaminase inhibitor, effectively disrupts glutaminolysis. However, previous clinical attempts to target glutamine metabolism pathways using CB839 have been unsuccessful. We propose that TET2MT creates unique metabolic signatures that can be specifically targeted with CB839. Consequently, TET2MT can serve as biomarkers for glutaminolysis inhibitors in the selective treatment of AML.
 Methods and Results
 Utilizing CRISPR-Cas9 edited clonal isogenic TET2WT and TET2KO cells derived from THP1 (Guan et al., 2021), coupled with HR LC-MS, we identified ~5000 features through untargeted analysis, with significant changes in 810. Targeted analysis confirmed unique metabolic disturbances in amino acid, glutamine, methionine, nicotinamide, and glycolytic pathways in TET2KO compared to TET2WT. Pathway enrichment analysis highlighted glutamine/glutamate biosynthesis as the most disturbed pathway. Significantly lower glutamine levels were observed in TET2WT patient cells and TET2KO mice compared to wildtype controls. These changes are primarily due to intracellular effects, as reflected in the significantly upregulated expression and activity of glutaminase (GLS1) and glutamate dehydrogenase (GULD1) in TET2KO cells.
 We conducted 13C glucose and 13C/15N glutamine metabolic flux analysis with complete carbon and nitrogen atom tracing of Glucose and Glutamine. Media glucose consumption was similar in TET2KO and TET2WT cells. However, TET2KO showed a significant buildup of cellular glucose and a slower consumption rate of glucose-6-phosphate, indicating slower glycolysis. In TET2WT cells, > 60% of αKG comes from glycolysis, while in TET2KO cells, it is < 40%, consistent with higher flux from Glutamine due to increased GLS1 and GULD1 activity (Gu et al., Blood 2023). Our data suggest that TET2MT creates metabolic reprogramming, leading to heightened glutamine addiction that can be therapeutically exploited.
 Consistent with our hypothesis, loss of TET2 resulted in high sensitivity to glutamine restriction and treatment with the GLS1 inhibitor CB839 (Jiang, B., et al. Mol Cell 2022). We performed metabolic analysis to understand how CB839 limits cell growth in TET2KO cells. The most impacted pathway was glutamine metabolism in the mitochondria, where glutamine replenishes the TCA cycle. Glutamic acid levels were reduced more than two-fold with CB839 treatment in TET2KO compared to TET2WT. As a result, TET2KO cells showed a 3-5-fold reduction in αKG with CB839 treatment compared to TET2WT. Significant reductions in αKG downstream TCA cycle compounds, such as succinate, fumarate, and malate, were observed, with less impact on upstream compounds citrate and oxaloacetate. Inhibition of glutaminase reduced overall downstream metabolites, while IMP, the first product of de novo purine synthesis, increased 2-3-fold. These findings confirm that TET2 loss promotes glutamine metabolic pathway addiction, with CB839 inhibiting TET2KO cell growth by disrupting the glutamine-dependent TCA cycle.
 Given that the TET2 inhibitors Eltrombopag (Epag) and TETi76 mimic the loss of TET2 and create similar disruptions in glutaminolysis, a combination therapy approach for non-TET2 mutated AML using TETi and CB839 can be utilized.
 Conclusion.
 Our study provides a detailed mechanistic understanding of how TET2 mutations contribute to MN and CHIP. This discovery paves the way for a biomarker-driven, novel therapeutic strategy for myeloid leukemia. Furthermore, our research highlights the potential of combining TET2 inhibitors with glutaminolysis inhibitors as a pioneering treatment approach for a broad range of leukemia types.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e8fc49c926f4de8069014cced00fe783497a3c6" target='_blank'>
              Developing a Therapeutic Strategy for Targeting Metabolic Vulnerability of TET2 Mutant Leukemia
              </a>
            </td>
          <td>
            Xiaorong Gu, Yahan Zhang, Songa Bae, Dongxu Jiang, Nakisha D Williams, Daniel J Vail, Simon Schlanger, Jill Durkin, Yvonne Parker, Daniel Lindner, Jaroslaw P. Maciejewski, B. Jha
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT Jumping translocations (JT) are rare cytogenetic abnormalities associated with progression in myelodysplastic syndromes (MDS) and acute myeloid leukemia (AML). Typically, a tri–tetra‐somic 1q chromosome is translocated to two or more recipient chromosomes. In multiple myeloma JT were shown to originate after DNA demethylation and decondensation. Using epigenomics, we investigated sequential samples in an SRSF2‐mutated MDS and AML cohort with normal karyotype at diagnosis and 1qJT at disease evolution after 5′‐azacytidine (AZA). 1qJT breakpoints fell within repetitive DNA at both 1q12 and the translocation partners, namely acrocentrics n. 14, 15, 21, and 22, chromosome 16, and chromosome Y. The global methylome at diagnosis showed hypermethylation at 61% of the differentially methylated regions (DMRs), followed by hypomethylation at 80% of DMRs under AZA, mostly affecting pathways related to immune system, chromatin organization, chromosome condensation, telomere maintenance, rRNA, and DNA repair. At disease evolution, a shift toward hypermethylation, intronic enhancers enrichment and epigenetic involvement of the PI3K/AKT and MAPK signaling emerged. In particular, AKT1 phosphorylation behaved as a hallmark of the progression. Overall, we provided new insights on the characterization of 1qJT in SRSF2‐mutated myeloid neoplasms and first showed that epigenetics is a powerful tool to investigate the molecular landscape of repetitive DNA rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c67408d13a5333628e6fca2329dcdcc54f1fe5a1" target='_blank'>
              Epigenetic Modeling of Jumping Translocations of 1q Heterochromatin in Acute Myeloid Leukemia After 5'‐Azacytidine Treatment
              </a>
            </td>
          <td>
            A. G. Lema Fernandez, C. Nardelli, V. Pierini, B. Crescenzi, F. Pellanera, C. Matteucci, Maria Crocioni, S. Arniani, Valeria di Battista, M. Quintini, G. Mondanelli, C. Orabona, P. Gorello, Cristina Mecucci
          </td>
          <td>2024-11-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="

 Introduction: Despite achievement of complete remission (CR) following chemotherapy, Acute Myelogenous Leukemia (AML) relapses in the majority of adult patients. Molecular and cellular contributors to chemotherapy resistance and to AML expansion and relapse have been identified in genomic, epigenetic and proteomic aberrations, while cellular relapse reservoirs have been identified in leukemia stem cells. Here, we set out to identify and functionally validate genes with consistent expression changes between AML at Diagnosis and Relapse aiming at a better understanding of the gene drivers and biology of AML relapse.
 Methods: We employed the drop-seq 3' single cell RNA sequencing (scRNA-seq) method (Macosko et al., Cell 2015) to analyze 20 paired AML specimens procured at diagnosis and at relapse from prior CR. We compared the gene expression in pooled (pseudo bulk RNA seq) single hematopoietic stem and progenitor cells (HSPCs) at diagnosis and relapse thus nominating a gene pool with highly concordant UP or DOWN regulation at relapse. We conducted lentiviral CRISPR-Cas9 based gene inactivation screens and cell growth assays using a custom library of ~7200 guides targeting 1550 genes and 1,000 controls. The screens were conducted in three primary human CD34+ non-malignant cell samples and in six primary human AML samples using 30-40 day cultures in stem cell media. A fraction of the genes that accelerated the growth of CD34+ and AML cells were individually re-assayed. Next, we selected 12 novel genes and 2 controls for in vivo studies in AML xenografts. Primary human AML were infected with lentiviral guide pools and following two days of culture injected into irradiated NSG-SGM3 mice. The AML engraftment was monitored serially via blood flow cytometry and bone marrows harvested in weeks 12-16. Guide frequencies in AML xenografts were determined via PCR-based library generation, followed by next generation sequencing and computationally using the MAGeCK software (Li et al., Genome Biology, 2014).
 Results: The gene with the highest enrichment score (ES) following gene disruption in CD34+ cells was the known AML tumor suppressor DNMT3A (ES 1.64). The expression of DNMT3A was downregulated in AML at relapse, together supporting the conclusion that mutational as well as non-mutational DNMT3A downregulation facilitates stem and progenitor cell expansion in relapsed AML. Further, the AML tumor suppressors SETD2 (ES 1.38, rank 2), RUNX1 (ES 0.73, rank 9) and ETV6 (ES 0.79, rank 26) scored high in this assay. Remarkably, there were 150 genes with a mean enrichment score >0.5 that upon disruption resulted in cell enrichment in all three CD34+ cell samples concordantly. Serially harvested cell samples on days 27 and 40 had high concordance with a Pearson coefficient of R=0.76, p< 2 x 10-15. We measured highly similar effects in pooled day 40 colonies harvested from lentivirally infected cells grown on large MethoCultTM plates.
 In single gene validation studies, upon genetic disruption of FOXN3, NIN1, TMX1, NCOR2, ODF3B, HMGXB4, GAPT, EHBP1L1, MGAT4B, CSNK1G2, EIF4G3, PCNT and ARHGEF2, we measured significantly increased cell numbers over time. The expression of all thirteen genes were downregulated in CMP-like leukemia cells at Re, and 10/13 of these genes were downregulated in HSC/MPP-like cells. Altogether, so far we experimentally confirmed 57% (17/30) of genes that when downregulated accelerated the growth and survival of human CD34+ stem and progenitor cells. We combined the assays results derived from primary AML day 30 ex vivo growth (N=4 AML) and in vivo AML xenograft assays (N=5 AML) and identified recurrent enrichment for guides targeting the genes: HMGXB4, GAPT, TMX1, ARHGEF2, NIN, FOXN3, and EIF4G3 and additional in vivo AML xenograft experiments are ongoing.
 Conclusions: The comparative analysis of scRNA-seq data of 20 paired AML specimens procured at diagnosis and relapse, identified frequent and previously unrecognized large-scale changes in gene expression driving leukemia cell growth and relapse. Our experimental findings support a combinatorial role for reduced expression of these genes in leukemic progenitor cell fitness and expansion at relapse and revise current parsimonious models of human AML relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/833f57d58b238268a9bbf80e325eadccbe827b54" target='_blank'>
              Identification of Novel Genes Underlying Human AML Expansion at Relapse from Chemotherapy
              </a>
            </td>
          <td>
            Darren King, Kai Wu, Qianyi Ma, Josephine Dubois, Xianing Zheng, Mohamed Rizk, K. Shedden, Malathi Kandarpa, Dan Boyer, Karan Bedi, Jun Li, S. Malek
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="

 Background
 Despite recent advances, the reasons for discrepancies between ELN risk assessment and clinical outcomes in acute myeloid leukemia (AML) remain poorly understood. This creates an opportunity to explore additional predictive biomarkers using functional phenotypic analysis. Most cancer therapies induce cell death by activating the mitochondrial apoptotic pathway. Therefore, measuring the apoptotic threshold maintained between pro-survival proteins (BCL-2, BCL-XL, and MCL-1) and pro-apoptotic proteins through BH3 profiling (BP) and dynamic BH3 profiling (DBP) can provide valuable information beyond known genetic risk factors in acute leukemia. We propose that variations in apoptosis thresholds, assessed through BP and DBP, can offer insights into the risk, therapeutic response, treatment options, and outcomes of acute leukemia. Assessing patient-specific dependencies on BCL-2 family proteins can guide the selection of BH3 mimetic and chemotherapy combinations for potential therapies.
 Methods
 We exposed blasts from bone marrow aspirates or peripheral blood to various pro-apoptotic BH3 peptides and measured mitochondrial outer membrane permeabilization using BP. We examined pro-survival BCL-2 family members (BCL-2, BCL-XL, and MCL-1) and their sensitivity to BAD, HRK, and MS-1 peptides to identify factors influencing differential chemosensitivity to BH3-mimetics in these malignancies. We evaluated 30 pediatric and 43 adult AML patients, and 70 pediatric and six adult ALL patients. The samples included 96 at diagnosis, 46 at relapse, and seven during treatment. For DBP, nine pediatric AML tumors were treated with cytarabine, daunorubicin, etoposide, and fludarabine, while 12 T-ALL samples were treated with nelarabine, vincristine, dasatinib, asparaginase, and dexamethasone. We analyzed delta priming to assess chemotherapy response in vitro and conducted bulk RNA-seq of pediatric AML samples to understand differential gene expression using DESeq2 version 1.42.0. Our analysis aimed to comprehensively understand acute leukemia.
 Results
 We present an initial analysis of functional BH3 profiling on 149 primary tumor samples (73 AML, 29 T-ALL, and 47 B-ALL cases) with complete clinical and molecular details from three cancer centers in Singapore. Pediatric favorable-risk AML demonstrated significantly higher susceptibility to apoptosis than non-favorable-risk AML (p<0.05), measured by cytochrome c release when exposed to BIM peptide (mean 60% ± 20% for favorable risk vs. 27% ± 15% for non-favorable risk). Adult AML showed higher mitochondrial priming in favorable risk group compared to intermediate/adverse risk group, indicating apoptotic sensitivities correlates with ELN risk. Pediatric AML (N=30) exhibited greater priming than adults (N=43) (p<0.001), potentially explaining functional basis of improved survival outcomes in children vs adults. Direct mitochondrial sensitivity to venetoclax in lymphoblasts was higher in T-ALL (n=29) than in B-ALL (n=47). Lymphoblasts showed greater direct mitochondrial sensitivity to navitoclax, suggesting the potential use of dual inhibition of BCL2 and BCL-XL in ALL. Ex vivo apoptotic priming was enhanced after etoposide treatment in myeloblasts of 50% of pediatric AML cases refractory to standard induction. Differential gene signatures derived from RNAseq identified pathways associated with low and high apoptotic sensitivity in pediatric AML patients.
 Conclusions
 This study demonstrates the potential of integrating mitochondrial priming phenotypes with genetic data to refine AML risk assessment. Our approach facilitates the evaluation of treatment sensitivity at the cellular level to address limitations in current ELN risk stratification where favorable-risk patients may relapse, and some adverse-risk patients achieve good outcomes. Incorporating functional BH3 profiling into diagnostic assessments could enhance contemporary risk stratification strategies for both pediatric and adult AML patients. The consistent sensitivity of lymphoblasts and myeloblasts to BCL-2 inhibition across age groups and lineages suggests the broad applicability of our approach. By integrating cellular vulnerabilities to anti-apoptotic proteins into clinical assessments, we may significantly improve treatment decision-making and risk stratification in acute leukemia.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb43109a17e0697291fe8a56970aa1e4b2e3377" target='_blank'>
              Integrating Genomics and Functional Phenotypes to Refine Risk Assessment in Acute Myeloid Leukemia across the Age Spectrum: Insights from Singapore
              </a>
            </td>
          <td>
            Prasad Iyer, S. Jasdanwala, Xiao Xian Lin, K. Bhatia, Germaine Shimin Liew, Si Ting Tai, Jameelah Sheik Mohamed, W. Jen, M. Ooi, Allen Eng Juh Yeoh, Shruti Bhatt
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="

 Introduction
 Pediatric acute myeloid leukemia (pAML) involves uncontrolled proliferation of myeloid progenitor cells. Recent studies, including the TARGET initiative, show a majority of pAML to be driven by molecular abnormalities, frequently in genes regulating transcription. The homeobox domain, a highly conserved 60 amino acid DNA binding motif, present in over 300 human transcription factor genes is key in regulating early development. It is observed in all 39 HOX genes and TALE family co-factors. Understanding the global patterns of dysregulation in these master developmental genes is key to a comprehensive understanding of leukemogenesis and better outcome prognostication for patients.
 Identifying Dysregulation
 To identify which developmental transcription factors are dysregulated in pAML we employed RNA sequencing data from 1,534 diagnostic samples compared to 68 non-diseased bone marrow and 16 CD34+ peripheral blood controls. Unsupervised hierarchical clustering methods utilizing the landscape of homeobox domain containing genes revealed distinct inter and intra subtype segregation as well as separation from normal bone marrow controls. Subtypes including RUNX1-RUNX1T1, CBF-MYH11, CBFA2T3-GLIS2, FLT3+ and KMT2A-rearranged pAML differentiated themselves on expression of these early developmental transcription factors and formed homogeneous clusters of up and down-regulated homeobox containing genes. Importantly, of the 346 human homeobox genes we identified 17 which are not expressed in normal bone marrow, yet differentiated pAML. This cluster included nine members of the HOX A cluster and 4 TALE family genes (MEIS1, PBX3, MKX, IRX3) which we call our DDTF (Dysregulated Developmental Transcription Factor) cluster. Kmeans clustering of pAML using DDTF genes resulted in three groups with mean expressions of 21, 55 and 103 transcripts per million.
 Survival Analysis
 We evaluated the prognostic value of DDTF gene expression by comparing expressors vs non-expressors using the Kaplan-Meier method. Expression threshold was determined dynamically using the log-rank statistic (Survminer package, R). Cluster expression was defined as the mean value of gene expression across the DDTF cluster in transcripts per million. Outcomes were markedly different between expressors and non-expressors. 5 year overall survival (5-yr OS) for expressors of the DDTF cluster was 53% vs. 79% for non-expressors (p < 0.0001). In KMT2A rearranged pAML clusters had 5-yr OS of 31, 59 and 70% respectively. High DDTF expression consistently indicated poor outcome. To further validate that DDTF expression was associated with poor outcome and not simply identifying poor risk pAML subtypes we separated high risk from low and standard risk into two different cohorts and re-applied our survival analysis to very similar results. In 38 samples with diagnostic, remission and relapse samples we tracked DDTF expression and found a decrease in DDTF at the remission timepoint followed by a concordant increase at relapse.
 Transcriptomic Impact
 To identify the global transcriptomic impact the dysregulation of DDTF genes has in the context of KMT2A rearranged pAML we performed a differential expression analysis comparing above median expressors to below median expressors. Greater than 1,000 genes were significantly (p. adj < 0.05) differentially expressed. Gene set enrichment analysis identified significant suppression of pathways related to adaptive immune response including ‘T cell activation’ and ‘immune response mediation’ in the over DDTF expressing group. Activated pathways were enriched for early developmental pathways and included ‘embryonic morphogenesis’ and ‘pattern specific processing’.
 Conclusions
 Our data indicate that dysregulation of developmental transcription factors is both widespread in pAML and occurs in coordinated subtype specific patterns. We further find that increased expression of specific HOX and TALE family factors we call DDTFD, are prognostic of patient OS, especially in KMT2A-rearranged pAML. Dysregulation of these genes leads to global changes in gene expression impacting functional pathways including crucial immune response pathways.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd5f65ea32158d069a0e3a278374f359a5af3ca7" target='_blank'>
              Dysregulation of Developmental Transcription Factors in Acute Myeloid Leukemia Predicts Poor Outcome and Impacts Adaptive Immune Pathway Genes
              </a>
            </td>
          <td>
            Logan K. Wallace, Jack H Peplinski, R. Ries, Danielle C Kirkey, Xiaotu Ma, S. Meshinchi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Genetic mutations are being thoroughly mapped in human cancers, yet a fundamental question in cancer biology is whether such mutations are functionally required for cancer initiation, maintenance of established cancer, or both. Here, we study this question in the context of human acute myeloid leukemia (AML), where DNMT3AR882 missense mutations often arise early, in pre-leukemic clonal hematopoiesis, and corrupt the DNA methylation landscape to initiate leukemia. We developed CRISPR-based methods to directly correct DNMT3AR882 mutations in leukemic cells obtained from patients. Surprisingly, DNMT3AR882 mutations were largely dispensable for disease maintenance. Replacing DNMT3AR882 mutants with wild-type DNMT3A did not impair the ability of AML cells to engraft in vivo, and minimally altered DNA methylation. Taken together, DNMT3AR882 mutations are initially necessary for AML initiation, but are largely dispensable for disease maintenance. The notion that initiating oncogenes differ from those that maintain cancer has important implications for cancer evolution and therapy. STATEMENT OF SIGNIFICANCE Understanding which driver mutations are required for cancer initiation, maintenance, or both phases remains poorly understood. Here, we uncover that highly prevalent pre-leukemic DNMT3A mutations are only required during disease initiation, but become dispensable after leukemic transformation, uncovering the context-specific role of this driver mutation with important therapeutic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94506bb3eb524303455ffe91b550e1d6fd5d7a8d" target='_blank'>
              DNMT3AR882H Is Not Required for Disease Maintenance in Primary Human AML, but Is Associated With Increased Leukemia Stem Cell Frequency
              </a>
            </td>
          <td>
            Thomas Köhnke, Daiki Karigane, Eleanor Hilgart, Amy C. Fan, Kensuke Kayamori, Masashi Miyauchi, Cailin T Collins, Fabian P. Suchy, Athreya S Rangavajhula, Yang Feng, Yusuke Nakauchi, Eduardo Martinez-Montes, J. Fowler, Kyle M. Loh, Hiromitsu Nakauchi, Michael A Koldobskiy, Andy Feinberg, Ravi Majeti
          </td>
          <td>2024-10-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="

 Tyrosine kinase inhibitor (TKI) such as Imatinib (IM) is the preferred treatment for Chronic Myeloid Leukemia (CML). However, TKIs resistant generate a significant challenge to disease management. Activation of key signaling pathways, cell cycle disruptions, abnormal epigenetic events, aberrant stem cell niche microenvironments, and metabolic shifts had been reported to be associated with TKI resistance. We detected the increased glucose uptake and lactate production in high dose IM-induced CML resistant cells. Chiglitazar (Chi), a novel Insulin Sensitizer, reduced the glucose metabolism in stem-like cell of Acute Myeloid Leumia (AML) according to our previous study. This inspired us to test its therapeutic effect in CML management. Cell proliferation, cell cycle distribution, and apoptosis results showed the cell killing effect when combined with IM but not Chi-only in IM-resistant CML cells. RNA-Seq and western blotting showed the Chi reducing the glucose uptake, reducing lactate production, and resensitizing the IM-resistant CML cells through the PPARγ/mTOR/PKM2 pathway. Further, we introduced shPPARγ to knockdown PPARγ in IM-resistant cells after Chi treatment for 24h. Downregulation of PPARγ significantly redued the apoptosis-induction effect when IM-resistant cells treated with IM and Chi combined treatment. Moreover, mTOR activator (MHY1485) could mimic the the synergistic effect of Chi and IM combination treatment. Finally, in vivo data confirmed the anti-leukemic effect of Chi and IM combination regimen in CDX (cell derived xnograft) models. Thus, we concluded that combining Chi and IM serves as a potential thrapeutic regimen in CML clinical management.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea31cdabbd92177cf9180fe5f05341fa37c26946" target='_blank'>
              Resensitizing the Imatinib-Resistant Chronic Myeloid Leukemia through Reshaping the Glucose Mtabolism By Insulin Sensitizer
              </a>
            </td>
          <td>
            Qian Lai, Hongpeng Duan, Yuelong Jiang, Liuzhen Yang, Manman Deng, Zhijuan Lin, Weihang Shan, Mengya Zhong, Jingwei Yao, Li Zhang, Jie Zha, Bing Xu
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Relapsed T-cell Acute Lymphoblastic Leukemia (T-ALL) patients have a dismal prognosis, with 5-year overall survival rates of 10% for adults and 36% for children. Relapse is attributed to chemotherapy's inability to target Leukemia Stem Cells (LSCs), that have the potential to re-populate the cancer. LSCs exists with a unique behavior compared to the rest of the leukemia, finding targeted therapies toward LSCs is critical for disease eradication and elimination of relapse. However, investigating LSC is particularly challenging in T-ALL for two reasons 1) The lack of well-established surface markers that distinguish LSCs and 2) LSCs exist at very low frequencies in T-ALL murine models and patient samples.
 Here, we used a transgenic model of rag2-myc derived T-ALL in zebrafish that offers solutions to these challenges. Our lab had previously generated leukemia lines from this model with LSC frequency of ~ 10%, compared to <0.01% in murine models, generating an ideal environment to interrogate T-ALL LSC biology and explore specific targets. We utilized this model to screen over 770 FDA-approved drugs in vivo using >2,500 syngeneic CG1 zebrafish for their effect on self-renewal. Further secondary screening in human T-ALL cells narrowed down the potential hits into 4 compounds that specifically interfere with self-renewal. Finally, the hit drug Amiloride was confirmed by an in vivo limiting dilation assay (LDA), resulting in a more than 4-fold decrease in the LSC frequency (p=0.0026). The top hit, Amiloride, which reduced the frequency of LSCs in zebrafish and human T-ALL cells, is an inhibitor of the Sodium Hydrogen Exachanger-1 (NHE1). Notably, NHE1 has not been previously linked to self-renewal in hematologic malignancies, presenting a novel therapeutic strategy to inhibit LSCs.
 To validate the NHE1 as the target responsible for the effect on self-renewal by Amiloride, we used shRNA knock down (KD) of NHE1. We found more than 70% decrease (p< 0.0001) in self-renewal in vitro with the KD cells compared to the scrambled control. Additionally, there was a concurrent decrease in gene expression, assessed by RT-PCR, of Nanog (p< 0.0001), CD7 (p< 0.0001) and CD44 (p= 0.0079), among other self-renewal genes. Further, the inhibition of the NHE1 by KD studies and pharmacological treatment results in a significant G1 cell cycle arrest. Bulk mRNA sequencing revealed a down-regulation in the Myc Target genes, KRAS, Hedgehog and Notch signaling in KD cells compared to the control, all critical pathways in leukemia self-renewal. Interestingly, the EphA3 gene was among the top down-regulated genes (FDR=2.01*10-25), EphA3 is expressed by LSCs and involved in maintaining their leukemogenicity. Collectively, these data suggest a central role for the NHE1 in T-ALL self-renewal.
 Inhibiting the NHE1 impairs mitochondrial function in cancer cells, offering a possible mechanism for the effect on LSCs. In human T-ALL cells, we found that Amiloride significantly reduced mitochondrial basal respiration (p=0.0074) and ATP production (p=0.0008). When probing for different genes associated with mitochondrial processes, PRKN expression was constantly upregulated by more than 8-folds in Amiloride-treated cells, compared to DMSO control (p< 0.0001). Parkin is an E3 ubiquitin ligase and a well-established mediator of mitophagy, which led us to investigate the impact of Amiloride on mitochondrial morphology. We found that Amiloride treatment reduced mitochondrial footprint (p< 0.0001) and mitochondrial branching (p=0.0028) in T-ALL cells. The inhibition of the exchanger by KD studies revealed similar mitochondrial alterations including decreased ATP production, basal respiration, spare capacity, decreased mitochondrial mass, and decreased mitochondrial membrane potential. Further, Gene set enrichment analysis identified a downregulation in the OXPHOS pathway. Importantly, we found that high mitochondrial content was significantly associated with self-renewal in T-ALL cells, suggesting that LSCs may be highly sensitive to metabolic disruption.
 Overall, in this work we capitalized on a transgenic model of zebrafish T-ALL with high frequency of LSC to identify the physiologically accurate hit, Amiloride. In addition, we are the first to describe its target, the NHE1 as a vulnerability of self-renewal in hematological malignancies, potentially through regulating the cellular energy homeostasis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42d09f4fcff2d28f61280f3680ea16ef82cfbcff" target='_blank'>
              Large-Scale In Vivo Zebrafish Drug Screen Uncovers Novel Insights into Leukemia Stem Cell Biology
              </a>
            </td>
          <td>
            Majd A. Al-Hamaly, Yelena Chernyavskaya, Meghan Haney, Jeffery Jolly, Ronald Bruntz, Jessica Blackburn
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Adoptive cell therapy using Chimeric Antigen Receptor (CAR) T cells has revolutionized cancer immunotherapy for a number of hematologic diseases. However, challenges remain in Chronic Lymphocytic Leukemia (CLL) where durable remission rates are low. Previously, we found that CAR vector-mediated disruption of the epigenetic regulator TET2 led to the clonal expansion of CD8+ CAR T cells in a CLL patient (Fraietta et al., Nature 2018). TET2 is an ɑ-ketoglutarate (ɑKG)-dependent dioxygenase that catalyzes the conversion of methyl-Cytosine in DNA to hydroxy-methyl-cytosine, which culminates in the removal of methylation marks in silenced genes. The TET2-disrupted CAR T cells exhibited immunophenotypic and epigenetic characteristics of early memory T cells. Given that TET2 is a known tumor suppressor gene, complete ablation via CRISPR/Cas9 could risk malignant transformation of engineered T cells (Jain et al., Nature 2023). Isocitrate dehydrogenase 1 (IDH1) mutant R132H had previously demonstrated exhibition of neomorphic activity that producing excess amounts of D-2-hydroxyglutarate (D-2HG), which is a metabolic inhibitor of TET2 via competitor ɑKG (Figueroa et al., Cancer Cell 2010). We hypothesized that IDH1 neomorph overexpression in CAR T cells safely inhibits TET2 activity and thereby phenocopies TET2 genetic disruption as observed in our patient.
 To investigate the biological impact of IDH1 R132H neomorph overexpression (IDH1NM OE) in CAR T cells, we developed a bicistronic expression system for CD19-specific 2nd generation CAR with 4-1BB/CD3ζ to co-express IDH1 R132H neomorph (IDH1NM) and used wildtype IDH1 (IDH1WT) as a control. Western blotting confirmed IDH1WT and IDH1NM overexpression in manufactured CAR19 T cells. IDH1NM OE led to significant D-2HG accumulation in CAR19 T cells, unlike IDH1WT. The expression of IDH1NM did not affect the cytolytic activity of CAR T cells, confirming the preservation of CAR their effector function. IDH1NM OE, however, significantly augmented CAR T cell response in a repeat tumor challenge model, similar to our previously published TET2 knock-down studies in normal donor T cells (Fraietta et al., Nature 2018). Further, IDH1NM CAR T cells showed significantly increased outgrowth in the CCR7+CD45RA+ central memory phenotype during chronic CAR stimulation. Additionally, we observed that effector cytokine levels, including IFNɣ and TNFɑ, were significantly reduced along with a decrease in CD107a, a major degranulation marker, while IL-2 levels were increased which is consistent with our TET2 knock-down studies. This suggests that IDH1NM OE improves the maintenance of early memory as well as effector function in the face of repeated tumor cell encounter.
 Next, we aimed to identify the relevance between metabolic reprogramming and memory function mediated by IDH1NM OE in CAR T cells, based on our hypothesis that IDH1NM OE impact ɑKG-dependent dioxygenase activity including various histone demethylase as well as TET2 in CAR T cells. We observed an increase in methylated histone and a decrease in oxidated cytosine in IDH1NM CAR T cells which experienced repeated CAR stimulation in vitro. This suggests that metabolic reprogramming by IDH1NM OE in CAR T cells leads to epigenetic alterations toward a memory-like phenotype in CAR T cells. Our studies therefore confirm that the metabolic reprogramming of CAR T cells via IDH1 NM OE phenocopies TET2 disruption. Pre-clinical mouse studies are ongoing and will be presented at the ASH.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bced5e74ec75c24287dced3600cf0ae522fdb7ac" target='_blank'>
              Metabolic Reprogramming By IDH1 Neomorph Epigenetically Guides CAR T Cell Differentiation
              </a>
            </td>
          <td>
            Kunho Chung, Dongxu Jiang, Xiaorong Gu, Jie Xu, Céline Grégoire, Beatriz Coutinho de Oliveira, B. Jha, J. Melenhorst
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Though somatic mutations play a critical role in driving cancer initiation and progression, the systems-level functional impacts of these mutations—particularly, how they alter expression across the genome and give rise to cancer hallmarks—are not yet well-understood, even for well-studied cancer driver genes. To address this, we designed an integrative machine learning model, Dyscovr, that leverages mutation, gene expression, copy number alteration (CNA), methylation, and clinical data to uncover putative relationships between nonsynonymous mutations in key cancer driver genes and transcriptional changes across the genome. We applied Dyscovr pan-cancer and within 19 individual cancer types, finding both broadly relevant and cancer type-specific links between driver genes and putative targets, including a subset we further identify as exhibiting negative genetic relationships. Our work newly implicates–and validates in cell lines–KBTBD2 and mutant PIK3CA as putative synthetic lethals in breast cancer, suggesting a novel combinatorial treatment approach. HIGHLIGHTS Integrative framework Dyscovr links mutations within cancer drivers to downstream expression changes Dyscovr uncovers known and novel targets of cancer-driver genes Dyscovr reveals clinically important negative genetic interaction pairings Web platform to explore uncovered driver gene-target relationships eTOC BLURB An integrative computational framework, Dyscovr, links mutated cancer driver genes to expression changes in putative target genes within and across 19 TCGA cancer types. Dyscovr’s results include experimentally verifiable synthetic lethal driver-target pairings. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a48c5eb14f1890839d95663dc9da6929e284d0cf" target='_blank'>
              Integrative Computational Framework, Dyscovr, Links Mutated Driver Genes to Expression Dysregulation Across 19 Cancer Types
              </a>
            </td>
          <td>
            Sara Geraghty, Jacob A. Boyer, Mahya Fazel-Zarandi, Nibal Arzouni, Rolf-Peter Ryseck, Matthew J McBride, Lance R. Parsons, Joshua D. Rabinowitz, Mona Singh
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Introduction
 Monoclonal antibodies (MoAbs) directed against CD38 are a mainstay in multiple myeloma (MM). Downregulation of CD38 expression is a resistance mechanism of antigenic escape, however, cell surface expression may return. While this tumor cell plasticity is likely driven by epigenetic mechanisms, it is unknown if MM cells can develop permanent resistance to anti-CD38 MoAb by acquiring genomic events leading to biallelic disruption of CD38 thus precluding re-treatment. Here, we report biallelic loss of CD38 as a recurrent mechanism of resistance, often via convergent evolution where distinct subclones are advantaged towards the same selective pressure.
 Methods
 We interrogated two sets of patients (Mayo Clinic; n = 31, Calgary University; n =19) with relapse following anti-CD38 MoAbs. Two additional cases of interest were selected from Heidelberg University. Whole genome sequencing (WGS, 60-100x) or whole exome sequencing (WXS), flow cytometry (flow), and bulk RNAseq were used to validate the effect of genomic events on CD38. Missense mutations predicted to affect CD38 MoAb binding were validated with site-directed mutagenesis of a CD38 plasmid, transduction into K562 MM cells, and CD38 MoAb binding assays.
 Results
 The prevalence of biallelic inactivation of CD38 at anti-CD38 MoAb relapse was 6% (3/50). An additional selected biallelic event was noted in the Heidelberg set. Each case is described in detail: MM-19 (Calgary) received daratumumab (Dara), lenalidomide, and dexamethasone (Dex), and then had a second exposure to Dara, following relapse. At this second relapse, WGS was performed on two independent extramedullary disease localizations in the liver. Both samples had unique, large deletions (>3 Mb) on chromosome 4 coupled with additional unique focal deletions involving CD38. Flow confirmed the presence of CD138+/CD38- plasma cell populations in both sites.
 A second case of convergent evolution was seen in a patient (Mayo) after seventh-line Dara/Len/Dex. WGS revealed a common deletion encompassing CD38. On the remaining allele, one subclone had a structural variation-mediated loss of CD38(cancer cell fraction; CCF 40%) while another harbored an L153H missense mutation (CCF 60%). Flow confirmed presence of two CD138+ clones; one with diminished CD38 expression. L153H was functionally validated to abrogate Dara binding, while maintaining CD38 expression. The third sample was collected after fifth-line Dara and sixth-line Dara/Bortezomib/Dex (Heidelberg) and represents another case of convergent evolution with a common loss of chromosome 4 and two subclones detected by WGS (P98Lfs*12, CCF 39% resulting in biallelic inactivation; R140G, CCF 61%). Bulk RNAseq revealed, in comparison to NDMM from CoMMpass, CD38 expression below the first decile. Notably, the R140G variant reduced binding affinity to Dara but not to Isatuximab (Isa).
 A fourth case (Mayo) relapsing after 6 months of Dara-based therapy similarly had a focal deletion of CD38 with an L18Sfs*16 frameshift deletion on the remaining allele associated with a 4.35-fold reduction in CD38 expression by RNAseq compared to pre-treatment.
 Interestingly, in 3/4 cases the CD38-negative clone emerged after a second exposure to anti-CD38 MoAb, suggesting emergence with the first exposure and then selection during the second. One other monoallelic event (C275Y; Heidelberg) was detected post-Isa. Copy number of the alternate allele is not known given targeted sequencing methodology, but the variant was seen to reduce binding affinity to both Dara and Isa. The prevalence of monoallelic loss of CD38 in newly diagnosed MM from MMRF CoMMpass was 7% (50/701) and two patients had mutations in CD38, but there were no cases of biallelic loss suggesting these events are driven by treatment pressure.
 Conclusion
 Distinction of the mode of CD38 loss is valuable for predicting outcome of re-challenge to anti-CD38 MoAbs. We see here an estimated 5-10% of anti-CD38 MoAb relapses are characterized by genomic antigen escape. Results here are in line with antigen escape to BCMA- and GPRC5D-directed immunotherapies (Lee et al, Nat Med 2023), suggesting that acquired antigen escape resulting from prolonged exposure to the selective pressures of immunotherapy is a recurrent mechanism of resistance in MM. Moreover, in select cases, non-overlapping resistance mutations provide rationale for switching agents at second exposure.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c64994d049a14a2d6ce1fc16a3cdbd9870e439c4" target='_blank'>
              Convergent Evolution Towards CD38 Biallelic Loss Is a Recurrent Mechanism of Resistance to Anti-CD38 Antibodies in Multiple Myeloma
              </a>
            </td>
          <td>
            B. Diamond, Linda B. Baughn, Mansour Poorebrahim, A. Poos, Holly Lee, Marcella Kaddoura, J. Wiedmeier-Nutor, Michael Durante, Gregory Otteson, D. Jevremovic, Hongwei Tang, Stefan Frohling, M. Papadimitriou, B. Ziccheddu, T. Jelínek, O. Landgren, Paola Neri, P. Bergsagel, E. Braggio, Shaji K Kumar, M. Raab, Rafael Fonseca, N. Bahlis, Niels Weinhold, F. Maura
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="

 Loss of the Y chromosome (LOY) is frequently found in healthy aging men, which challenged evaluation of its clinical relevance for decades. Recently, however, LOY was identified as causative event for tumorigenesis via evasion of adaptive immunity in solid tumors. In AML, sole LOY is found in about 6% of newly diagnosed AML, comprising approximately 15% of the cytogenetic abnormalities frequently detected. Thus, we set out to characterize the possible role of LOY in AML pathogenesis and clonal evolution to delineate the clinical and translational relevance of this recurrent, yet least appreciated cytogenetic abnormality in adult patients with AML.
 Using an integrative bioinformatics methodology, we explored the transcriptomic profile of 23 bone marrow (BM) samples from male AML patients with LOY as the sole cytogenetic abnormality and a comparison group of 218 cytogenetically normal (CN) male patients with AML. As an additional control group, we also included 203 CN BM samples from female AML patients, all of which were derived from newly diagnosed, adult AML patients who were similarly treated on Cancer and Leukemia Group B/Alliance for Clinical Trials in Oncology frontline protocols. Using DESeq2, we first identified differential gene expression patterns, including 364 genes (2.3%) that were upregulated and 74 genes (0.46%) that were downregulated in the LOY compared with the CN male samples (adj p-value<0.05). Among the downregulated genes were genes known to be located on the Y chromosome, including KDM5D (adj p-value=0.0036), ZFY (adj p-value=0.0123) and DDX3Y (adj p-value=0.0480). Gene Ontology (GO) Biological Process (BP) enrichment analysis showed significant enrichment in, among others, pathways related to angiogenesis, cell-cell interaction, Wnt signaling, and regulation of ion transmembrane transport in the sole LOY male samples. Conversely, the downregulated genes showed significant enrichment in pathways related to innate immune regulation, such as inflammatory response (GO:0006954) and Toll-Like Receptor (TLR) Signaling pathways (GO:0002224, 0034135, 0034163). In line with these findings, Gene Set Enrichment Analysis showed that terms related to T cell activation, TLR signaling and NFkB activity were significantly underrepresented in the LOY male samples compared with their CN counterparts. To better understand the immune cell composition between groups, we performed deconvolution to estimate cell-type fractions within mixed cell populations, using CIBERSORTx. We found that LOY samples had higher proportions of Cytotoxic T cells (p=0.048) compared with CN male samples, while there was no difference between the LOY male and CN female group. Consistently, LOY patients had lower proportions of naïve T cells compared with CN male patients (p=0.00007).
 Lastly, we used high dimensional spectral flow cytometry to characterize the immune profile in 8 BM mononuclear (BMMC) samples from male AML patients with LOY as a sole cytogenetic abnormality, and 8 male CN-AML patients. The analysis was performed within the live CD34-CD45high single cells. While there was no difference in the proportion of CD8+ or CD4+ cells within the CD3+ T cell compartment, we found a significant increase of the CD45RA-CD45RO+ cells in the CD8+ T cell fraction in the LOY samples (p=0.0158) that was accompanied by a decrease in the CD45RA+CD45RO- CD8+ T cells (p=0.0131). Specifically, CD45RA-CD45RO+CD28+ memory-like CD8+ T cells (Tm) were significantly increased in the BM of the LOY patients (p=0.02), coupled by a sharp reduction of CD45RA+CD45RO-CD27-CD28- effector CD8+ T cells (Teff) (p=0.0001), which was validated in a second cohort of 6 LOY and 6 CN-AML BM samples. This finding suggested a potential block of Tm to Teff transition in AML with LOY.
 In summary, our results strongly suggest an immunomodulatory capacity of LOY in AML patients, thereby challenging the paradigm of LOY merely being an age-associated phenomenon without functional consequences. These results warrant additional follow-up studies to better understand the possible pathogenetic and clinical significance of LOY in AML, including amenability to immunotherapy approaches.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d882f6d9e00e71b95062963c39559e1c164343" target='_blank'>
              Characterization of Sole Loss of the Y Chromosome and Associated Effects on Adaptive Immunity in Adult Male Patients with Acute Myeloid Leukemia (AML)
              </a>
            </td>
          <td>
            M. Velegraki, Cankun Wang, D. Nicolet, K. Mrózek, Payton Weltge, Ethan Hamp, Jill Buss, Andrea Laganson, C. Oakes, Jordan Krull, John C. Byrd, Richard M. Stone, Qin Ma, Ann-Kathrin Eisfeld, Zihai Li
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is a highly aggressive and the most common brain tumor in childhood. MB presents a high intertumoral heterogeneity, with at least four molecular subgroups identified (SHH, WNT, Group 3, and Group 4). Metastatic MB, or Leptomeningeal Disease (LMD), is predominantly found in the MB Group 3 type. Although LMD represents a main clinical challenge, its molecular mechanisms remain poorly characterized. Recent research has shown that primary and MB metastasis diverge dramatically. Our work has focused on establishing therapy naïve Group 3 Patient-Derived Xenografts models of primary and metastatic Medulloblastoma to conduct transcriptomic profiling at the bulk and single-nuclei RNAseq levels to identify genetic drivers/pathways that sustain leptomeningeal disease compartment. Our results show various signaling pathways enriched across LMD models, such as MYC targets, unfolded protein response, and fatty acid metabolism. Using single-sample GSEA analysis (ssGSEA) and deconvolution approaches, we have also identified that our PDXes models retain neoplastic subpopulations previously identified in MB single-cell sequencing studies. Similarly, we have identified slight differences in cell subpopulation proportions between primary and leptomeningeal compartments. Our single-nuclei studies have confirmed these results and differentially expressed genes previously found in bulk RNAseq analyses. These results suggest the presence of cell populations enriched in the metastatic compartment with an aberrant transcription phenotype and adaptations in fatty acid metabolism to survive the leptomeningeal space. In conclusion, our work supports the notion that primary and LMD retain subpopulation clusters present in MB Group 3 tumors with different proportions in the metastatic compartment shown by bulkRNAseq deconvolution and single nuclei RNA analysis. We also show that primary and LMD are transcriptionally different, with various enriched pathways in the metastatic compartment and shared among LMD Group 3 PDX models. Through these approaches, we aim to elucidate the genetic dependencies of metastatic Medulloblastoma that will help for targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc5f85b4cb1aade499040d49a49b1625bf14d7a8" target='_blank'>
              EPCO-57. INTEGRATED ANALYSIS OF BULK AND SINGLE-CELL RNA SEQUENCING DATA OF PRIMARY AND METASTATIC PEDIATRIC MEDULLOBLASTOMA
              </a>
            </td>
          <td>
            Ana Isabel Castillo Orozco, Geoffroy Danieau, Niusha Khazaei, Masoumeh Aghababazadeh, Marjan Khatami, Livia Garzia
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background The use of FMS-like tyrosine kinase 3 (FLT3) as a crucial target for kinase inhibitors is well established, but its association with immune infiltration remains unclear. This study aimed to explore the relationship between FLT3 mutations and immune checkpoint molecules (ICMs) in patients with acute myeloid leukemia (AML). Methods The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) databases were used to identify the ICMs associated with FLT3 mutations. A Gene Ontology (GO) analysis, Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis, and gene set enrichment analysis (GSEA) were conducted to analyze the signaling pathways related to the ICMs. The single-sample GSEA (ssGSEA), Cibersort, and estimate algorithms were used to assess immune cell infiltration in AML. Results Absent in melanoma 2 (AIM2) exhibits elevated expression levels in AML patients harboring FLT3 mutation, contributing significantly to the progress of AML and establishing of an immunosuppressive microenvironment. AIM2 expression significantly correlated with sensitivity of clinically relevant drugs in ex vivo assays of AML. Additionally, AIM2 demonstrates substantial prognostic value and holds promise as a prospective immunotherapeutic target for AML. Our findings indicate a significant correlation between AIM2 and immune infiltration in AML cases, potentially affecting the presence of neutrophils, macrophages, effector memory T cells (Tem), and monocytes. Furthermore, AIM2 is closely linked to various signaling pathways, such as immune cytokine release, immune antigen presentation, and inflammasome signaling, which could play a role in immune cell enrichment in AML. Conclusions Our study identified AMI2 as an ICM linked to FLT3 mutations. AMI2 may be involved in the activation of suppressive immune cell populations, such as macrophages, neutrophils, and monocytes. AIM2 could serve as a promising immunotherapeutic target for combination therapy with FLT3 inhibitors in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d616757d933b29b3adfa5c2c167ff1c2c865a8e2" target='_blank'>
              FLT3 mutation-related immune checkpoint molecule absent in melanoma 2 (AIM2) contributes to immune infiltration in pediatric and adult acute myeloid leukemia: evidence from bioinformatics analysis
              </a>
            </td>
          <td>
            Jing Zhao, Yue Cui, Hua Zhou, Dongming Zhou, Zhiping Che, Ning Zhang, Qi Yun, J. Machado-Neto, Daniela Damiani, L. F. Al-Kzayer, Rohan Kulkarni, Meng Gu
          </td>
          <td>2024-11-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="

 Measurable residual disease (MRD) assessment is important in acute myeloid leukemia (AML), as post-remission MRD indicates higher relapse risk. Current techniques, however, lack precision in individual outcome prediction, complicated by AML's clonal heterogeneity. Novel single cell MRD (scMRD) workflows based on DNA mutations offer a potential breakthrough, simultaneously capturing genotypic and immunophenotypic changes at a single-cell level. The Tapestri® platform (Mission Bio) incorporates a microfluidic droplet technology with cell barcoding beads and gene-specific primers that enables amplification and comprehensive identification of low-frequency subclones that may lead to overt disease or resistance. With this approach, complex heterogeneity and differentiation between leukemic clones and clonal hematopoiesis can be deconstructed.
 Objective: This study represents the first large-scale scMRD analysis in AML using this novel scMRD technology to deconvolute clonal heterogeneity to identify potential relapse markers and to differentiate between leukemic clones and clonal hematopoiesis.
 Methods: We analyzed a cohort of 69 AML patients in first complete remission (CR) as assessed by bone marrow morphology from the BLAST trial (NCT02502968). For 11 patients, we analyzed paired samples from both first CR and relapse. We performed single-cell DNA + protein sequencing on a total of 250,000 cells across 23 multiplexed pools. Our methodology included 40 selected mutation hotspot genes and a 19-plex antibody oligonucleotide cocktail for AML-specific surface markers as well as patient-specific hash tag antibodies for enhanced demultiplexing. We used magnetic activated cell sorting (MACS) to enrich CD34/CD117-positive blasts to ensure high-quality samples for subsequent analysis.
 Results: Our analyses revealed a diverse genetic landscape with 25 different mutated genes. DNMT3A and NPM1:p.W228Cfs mutations were the most common alterations. The genetic distribution included a predominance of preleukemic genes (51.5%), chromatin remodelers (12.1%), splice factors (9.1%), transcription factors (6.1%), cell cycle regulators (6.1%), and other functional categories (15.2%).
 Lower MRD (and clonal hematopoiesis) levels were consistently associated with improved overall survival (OS) and relapse-free survival (RFS) across different thresholds (<5%, <10%, <20%, <30%, <40% and <50% MRD burden per patient), with patients in the <5% MRD group showing a significant difference in RFS (p = 0.0013). The median overall survival in this group was 18.0 months, while the median relapse-free survival was 11,7 months. OS and RFS remained significantly different in all MRD groups even after exclusion of DNMT3A and TET2 mutations, which most likely represented clonal hematopoiesis.
 In paired samples, TP53 mutations were notably prominent during follow-up, remaining the dominant clone, indicating a mixed composition, and the persistence of treatment resistant clones. Overall, our study highlighted the dynamic nature of clonal evolution under therapeutic pressure by demonstrating continuous clonal diversity during clinical remission and notable changes in clonal composition during relapse.
 Conclusion: In this AML cohort, our study provides insights into the genetic landscape and clonal dynamics of AML in remission. Thus, our findings emphasize the importance and the potential of single-cell DNA + protein sequencing in refining MRD assessment compared to current techniques.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db74b338da09cd55d0a1a600dcec1d4e9bacb1a1" target='_blank'>
              Clonal Dynamics of Leukemic and Clonal Hematopoiesis Mutations Predict Relapse in Single Cell MRD Analysis of AML in First Complete Remission
              </a>
            </td>
          <td>
            Enise Ceran, A. K. Merbach, Christian Rohde, Robert Durruthy-Durruthy, Adam Sciambi, Michelle Lotze, Gema Fuerte, Matteo Cattaneo, Ivan Lukic, Maxi Wass, Judith Schaffrath, Christoph Röllig, R. Schlenk, Marina Scheller, Simone Kowoll, Jörg Steighardt, B. Edemir, Beatrice Ludwig-Kraus, L. P. Mueller, Sabine Edemir, Abi Vainstein-Haras, E. Sorani, Irit Gliko Kabir, S. Kadosh, C. Gromann, Andreas Wienke, Claudia Wickenhauser, C. Baldus, U. Platzbecker, H. Serve, M. Bornhäuser, Carsten Müller-Tidow
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Killer meiotic drivers (KMDs) are selfish genetic elements that distort Mendelian inheritance by selectively killing meiotic products lacking the KMD element, thereby promoting their own propagation. Although KMDs have been found in diverse eukaryotes, only a limited number of them have been characterized at the molecular level, and their killing mechanisms remain largely unknown. In this study, we identify that a gene previously deemed essential for cell survival in the fission yeast Schizosaccharomyces pombe is a single-gene KMD. This gene, tdk1, kills nearly all tdk1Δ progeny in a tdk1+ × tdk1Δ cross. By analyzing polymorphisms of tdk1 among natural strains, we identify a resistant haplotype, HT3. This haplotype lacks killing ability yet confers resistance to killing by the wild-type tdk1. Proximity labeling experiments reveal an interaction between Tdk1, the protein product of tdk1, and the epigenetic reader Bdf1. Interestingly, the nonkilling Tdk1-HT3 variant does not interact with Bdf1. Cryoelectron microscopy further elucidated the binding interface between Tdk1 and Bdf1, pinpointing mutations within Tdk1-HT3 that disrupt this interface. During sexual reproduction, Tdk1 forms stable Bdf1-binding nuclear foci in all spores after meiosis. These foci persist in germinated tdk1Δ progeny and impede chromosome segregation during mitosis by generating aberrant chromosomal adhesions. This study identifies a KMD that masquerades as an essential gene and reveals the molecular mechanism by which this KMD hijacks cellular machinery to execute killing. Additionally, we unveil that losing the hijacking ability is an evolutionary path for this single-gene KMD to evolve into a nonkilling resistant haplotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7f5d09bb05d680c2b878734eefa4dc9aa3034f" target='_blank'>
              A meiotic driver hijacks an epigenetic reader to disrupt mitosis in noncarrier offspring.
              </a>
            </td>
          <td>
            Yu Hua, Jianxiu Zhang, Man-Yun Yang, Fan-Yi Zhang, Jing-Yi Ren, X. Lyu, Yan Ding, Fang Suo, Guangcan Shao, Jun Li, Mengqiu Dong, K. Ye, L. Du
          </td>
          <td>2024-11-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>2</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880482f92dba3e54c1a4764aaee26d7242591da7" target='_blank'>
              METTL3 confers oxaliplatin resistance through the activation of G6PD-enhanced pentose phosphate pathway in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Xiaohan Jin, Yongrui Lv, Fengjie Bie, Jinling Duan, Chao Ma, Miaomiao Dai, Jie-Wei Chen, Lianghe Lu, Shui-Dan Xu, Jie Zhou, Si Li, Jiong Bi, Fengwei Wang, Dan Xie, Muyan Cai
          </td>
          <td>2024-10-29</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="

 Introduction
 Myeloid neoplasms post cytotoxic therapy (MN-pCT) are a catastrophic complication of cancer therapy with an aggressive disease course and dismal patient outcomes. They frequently arise from preexisting clonal hematopoiesis (CH). Large sequencing studies have reported differences in the mutational spectra of age-related CH (ARCH), therapy-related CH (t-CH), de novo MN, and MN-pCT, respectively, suggesting functional differences in the capacity of the various driver gene mutations to initiate MN-pCT. Here, we utilized clinical sequencing data, meta-analytic statistical methods, newly established CRISPR/Cas9-edited mouse hematopoietic stem and progenitor cell (HSPC) lines, and murine mixed bone marrow chimeras to model the evolution of MN-pCT from CH under the selective pressure of anti-cancer therapy.
 Results
 We compared targeted next-generation sequencing data from MN-pCT and de novo MN patients collected at our institution. We found a significantly lower frequency of DNMT3A mutations in the MN-pCT group, while mutations in PPM1D and TP53 were more frequent. To determine the frequency of typical ARCH mutations (i.e., in the genes DNMT3A, TET2, ASXL1) as opposed to mutations in genes associated with t-CH (i.e., ATM, CHEK2, PPM1D, and TP53) in cohorts comprising the general population, therapy-exposed non-myeloid cancer patients, or MN-pCT patients, we performed a meta-analysis from 21 publicly available sequencing studies encompassing over 760,000 individuals. Estimation of cohort-specific mutational frequencies using a random effects logistic regression model revealed a significantly higher frequency of above mutations in therapy-exposed non-myeloid cancer patients (i.e., t-CH) as compared to the general population (i.e., ARCH) with highest odds ratios for PPM1D and TP53. Importantly, only TP53 mutations were strongly enriched in MN-pCT samples compared to patients with t-CH.
 To study the development of MN-pCT from CH in vitro, we generated CRISPR/Cas9-edited conditionally immortalized HSPC lines by reversible Hoxb8 overexpression with mutations in Dnmt3a, Tet2, Asxl1, Atm, Chek2, Ppm1d, or Trp53. First, we tested the capacity of HSPCs with these ARCH or t-CH mutations to clonally expand upon treatment with common anti-cancer drugs. Chek2- and Ppm1d-mutant HSPCs moderately outcompeted their wild-type counterparts mostly after treatment with double-strand break-inducing agents, whereas Trp53-/- HSPCs demonstrated rapid and almost complete outgrowth in the presence of a wide range of conventional cytostatic drugs as well as various targeted agents - e.g., PARP, BRAF, and BCL2 inhibitors. By contrast, cells with ARCH mutations did not expand upon treatment with anti-cancer drugs. Accordingly, Trp53-/- HSPCs displayed strongest defects in apoptosis and G1 cell cycle arrest as well as signs of persistent DNA damage as shown by γ-H2AX staining when compared to HSPCs with mutations in the other tested genes. Next, we assessed the capacity for malignant transformation after treatment with anti-cancer drugs in in vitro immortalization assays. Only Trp53-/- HSPCs were able to generate immortalized cell lines that harbored complex karyotypes reminiscent of those found in patients with TP53-mutant MN.To investigate clonal expansion and malignant transformation in vivo, we generated murine mixed bone marrow chimeras with 5-10% CRISPR/Cas9-edited HSPCs with mutations in Atm, Chek2, Ppm1d, or Trp53. After sublethal irradiation to mimic anti-cancer therapy, clonal expansion was observed in peripheral blood of Chek2-, Ppm1d- and again strongest in Trp53-mutant mice. Strikingly, only mice with Trp53-/- HSPCs displayed significantly shortened survival driven by the development of MN-pCT.
 Discussion and Outlook
 Using clinical sequencing data together with pre-clinical modelling in vitro and in vivo, we demonstrate that mutations in CHEK2, PPM1D, and TP53 but not classic ARCH mutations confer a clonal advantage to HSPCs under the selective pressure of anti-cancer drugs, thereby explaining their enrichment in t-CH. However, only TP53 mutations are sufficient to drive MN-pCT development in vitro and in vivo. Our findings further support the development of mitigation strategies to prevent MN-pCT in cancer patients at risk - in particular those with TP53 mutations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c04a0fd4f4254b49fa65a518d8e33c6826db227" target='_blank'>
              Modelling the Evolution of Clonal Hematopoiesis to Myeloid Malignancies Under the Selective Pressure of Anti-Cancer Therapy
              </a>
            </td>
          <td>
            Christian Koch, Jonas Fullin, Ebru Topçu, Nancy Klemm, Karolina Zielinska, Monika Hebeisen, Joelle Tchinda, M. Manz, S. Boettcher
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 Background: Minimal, or measurable residual disease (MRD) is the lowest sensitivity of detection of residual tumor after achieving a meaningful clinical response in multiple myeloma (MM). MRD negativity is associated with deeper response rates and longer progression free survival (PFS). Unfortunately, most patients who achieve MRD negativity eventually relapse, typically with a more aggressive and treatment-resistant malignancy. We set out to develop a preclinical model to study the biology of MRD to a standard of care (SOC) regimen and the relationship to the development of a therapy-resistant MM clone.
 Methods: To study MRD and resistance to SOC Revlimid (lenalidomide; R), Velcade (bortezomib; V), and dexamethasone (d), (collectively - RVd) in MM, we utilized genetically engineered mouse models of human-like MM which carry cMYC, and IKK2 expression from germinal-center B cells. Upon development of the disease as determined by bone marrow (BM) aspirates and peripheral gamma protein fraction, mice were treated with RVd therapy for 20 weeks. BM specimens (portional cohort sacrifice) were collected at baseline, 8 weeks (when maximal response was observed, defined as MRD) and >20 weeks (refractory/resistance, RR) and profiled using whole-genome sequencing (WGS) vs. germline control, bulk RNAseq (plasma cells alone and all remaining cells separately) and single cell RNAseq (scRNAseq) of non-sorted flushed BM.
 Results: Overall, RVd resistant plasma cells compared to baseline tumors were associated with significantly higher mutation load (median: RVd resistant 16,595; baseline 5,997, p-value = 0.0013, Wilcoxon rank-sum test) and more copy number variations (CNV) including amplifications of whole chromosomes (7, 11, 15, and 17). Further analysis identified two distinct groups of tumors resistant to RVd: one group (CNV-high) is characterized by high genetic instability (CNV-high) including a case of an acquired focal deletion of p53, while the other group (CNV-low) exhibits absence of copy number changes. These patterns were also reflected at the transcriptomic level, with each group exhibiting activation of different pathways, notably lower p53 pathway activity in CNV-high. Despite such differences, both groups showed elevated expression of genes related to TNF−alpha signaling via NF−kB (albeit much lower in the CNV-low), and hypoxia. Analysis of the scRNAseq captured at baseline, MRD and RR was able to identify GFP-expressing CD138+B220- MM cells. Focused differential expression analysis within this specific population revealed a similar increased pathway activation of TNF−alpha signaling via NF−kB, interferon alpha/gamma response, and hypoxia in tumors collected at MRD and RR compared to baseline as observed in bulk plasma cell RNAseq, confirming that these changes are intrinsic to the tumor. For correlation studies, we investigated a cohort of human newly diagnosed (NDMM) and relapse/refractory multiple myeloma (RRMM) RNAseq from CD138 isolated samples and found the upregulationof these pathways in RR compared to NDMM, underscoring the relevance of our findings from the mouse model to human disease.
 Conclusion: Our model provides a valuable preclinical tool for investigating the biology of MRD and resistance to a SOC regimen in multiple myeloma. Mice models showed an increase of CNV and mutations at RR as a key characteristic of progression, as well as the activation of TNF-alpha signaling and a more hypoxic environment, which were further validated on patient level data. This analysis enhances our understanding of the underlying mechanisms in a longitudinal manner and potentially guides the development of more effective therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54e1ed76f73a4fb277133d42d5d98e760e6aad99" target='_blank'>
              Genomic and Single Cell Investigation of MRD Clonality in Genetically Engineered Mouse Models of Multiple Myeloma
              </a>
            </td>
          <td>
            Junfei Zhao, Marta Larrayoz, C. Bjorklund, Anita K. Gandhi, Jose-Al Martinez-Climent, P. Hagner, María Ortiz
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="

 Background:
 DNAJ homolog subfamily C member 21 (DNAJC21) is a member of the DNAJ heat shock protein 40 family. Biallelic DNAJC21 mutations lead to DNAJC21 deficiency, disrupt ribosome biogenesis and cause a Schwachman Diamond like syndrome. DNAJC21 deficiency causes bone marrow (BM) failure, a risk of evolution to myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML), and extra-hematopoietic symptoms. Due to the limited published data, risk stratification for clonal evolution is currently not possible, and treatments beyond hematopoietic stem cell transplantation (HSCT) are unavailable. Understanding the molecular alterations in hematopoietic stem and progenitor cells (HSPCs) of DNAJC21 deficiency is critical for developing targeted therapies and stratifying the risk for disease progression.
 Objective:
 To collect clinical and laboratory data of DNAJC21-deficient patients.
 To elucidate the transcriptomic landscape and molecular pathways of DNAJC21 deficiency we performed scRNA seq in DNAJC21-deficient BM samples and performed shRNA-mediated DNAJC21 knock down in K562 cell model.
 Methods:
 Clinical and laboratory data were collected from 13 patients (9 unpublished and 4 published individuals, respectively). BM samples were collected from 6 DNAJC21-deficient patients and a targeted panel of genes involved in MDS/AML was performed. scRNA-seq was performed on CD34+ cells freshly isolated from the BM of 2 DNAJC21-deficient patients and compared with 2 healthy donors. The role of DNAJC21 was further investigated by reducing its expression in K562 cells using lentiviral shRNA and performing untargeted proteomic analysis. Bioinformatics analysis identified differentially expressed genes, pathways, and cell populations.
 Results:
 Data were collected from 13 patients (male:female = 8:5, median age 11 years; age range 1.5-15 years). All patients showed abnormal pancreatic imaging or function. Short stature, developmental delay, skeletal and eye abnormalities were frequently observed (11/13, 10/12, 8/13 and 7/9, respectively). Hypocellular BM was reported in 9/10 patients. Cytogenetic abnormalities were detected in 1 case (46,XY, der(15)t(1;15)(q12;p11)). Five out of 6 patients carried 1 or more somatic TP53 mutations (VAF range 5.3-33%) in the BM samples. P1 (n = 4) and P2 (n = 3) BM biopsy samples revealed progressive decrease in CD3+ T cells with a relative increase in cytotoxic CD8+ cells over time. HSCT was performed in 6 patients due to BM failure. Eleven patients were alive at the time of last follow up. Two patients died due to EBV-associated lymphoproliferative disorder (2.5 months post HSCT) or S. Aureus sepsis, respectively.
 scRNA-seq analysis revealed distinct transcriptional profiles in HSPCs from DNAJC21-deficient patients. DNAJC21 mutations affect translation and protein synthesis, immune response and metabolism. Notably, enrichment in gene sets involved in ribosome biogenesis emerged with a marked increase in the expression of ribosomal protein genes RPS17, RPL41, and RPL39, as well as the translation elongation factor EEF1G. These dysregulated genes are crucial components of the protein synthesis machinery, suggesting a compensatory response to impaired ribosome biogenesis. A marked decrease in genes involved in proliferation (MKI67, H2AFX, H3F3B, DBF4, UBB, UBC, UBE2S, FOSB, CCNL1) was also observed.
 Preliminary mass spectrometry analysis of DNAJC21-knockdown K562 cell lines identified several dysregulated pathways, including downregulation of DNA replication and galactose metabolism pathways and upregulation of hematopoietic cell lineage, purine metabolism, and SNARE interactions in vesicular transport pathways.
 Conclusion:
 Biallelic DNAJC21 mutations are associated with pancreatic abnormalities and multisystem disorders along with BM failure. The majority of patients harbors TP53 mutated clones with high VAF, whose role in disease evolution and malignant myeloid transformation remains to be elucidated.
 Biallelic DNAJC21 mutations profoundly impact the transcriptomic landscape of HSPCs, leading to dysregulated ribosome biogenesis and impaired proliferation. Our findings unveil the molecular pathways of BM failure in DNAJC21-deficient patients, highlighting critical pathways and gene sets that could be used as potential biomarkers for disease progression or targeted for therapeutic intervention.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dddc824b4783cd911f89ccadb646e274956053d5" target='_blank'>
              Decoding Transcriptomic Landscape and Molecular Pathways in DNAJC21 Deficiency
              </a>
            </td>
          <td>
            F. Saettini, Shreya Kanade, Fabiola Guerra, Benedetta E Di Majo, Francesca Vendemini, Federica Mottadelli, G. Fazio, D. D’Aliberti, S. Spinelli, Vincenzo L'Imperio, Albert Catalá, Elena Laura Gonzalez-Forster, María Solsona, Josune Zubicaray, Julián Sevilla, Eva Galvez Villa, Andrea Biondi, Yigal Dror, R. Piazza
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="

 Background
 Acute myeloid leukemia(AML)is an aggressive hematopoietic malignancy and accumulation of genetic alterations is a common mechanism in the clonal evolution of AML. Some mutations(DNMT3A, ASXL1, TET2)often persist during remission and disease progression, while relapse can either be driven by the acquisition of mutations in the founder clone or by the selection of resistant subclones (Blood,2021,137:3093-3104). However, the relationship between complete remission(CR)/relapse and the critical dynamic changes in genetic mutations is still not well covered. This study aimed to explore the dynamic changes of genetic mutations in AML patients from diagnosis to CR and relapse.
 Methods
 A total of ninety-seven AML patients with sequential serial samples were sequenced by our reported targeted exome-seq for human leukemia driver genes (Blood Cancer J,2022;12:145) from 28 May, 2019 to 7 June, 2024. Sixty-six Patients were sequenced at diagnosis and CR, twenty-four patients were sequenced at diagnosis, CR and relapse, while seven patients were sequenced at diagnosis and relapse.
 Results
 Ninety-three de novo AML patients and four secondary AML patients were analyzed with a median age of 60(15~86) years, including 53 males and 44 females. The median interval time between diagnosis to CR was 2(1~5) months, the median interval time from CR to relapse was 5(1~16) months, as well as the interval time from diagnosis to relapse was 8(2~28) months.
 The median number of mutations detected at CR was significantly less than the mutations detected at diagnosis( 1 vs. 3, P<0.001), moreover, 40 patients(44.5%)lost all of the mutations detected at diagnosis and had no mutations at CR. Although the other patients still had mutations detected at diagnosis, the variant allele frequency of these mutations significantly decreased. Furthermore, we found that patients who still had mutations detected at diagnosis had worse relapse-free survival than patients who lost all of the mutations detected at diagnosis (5 vs 11 months, P =0.042 ).
 Then, to explore the critical mutations that may promote relapse in AML patients, we found the median number of mutations detected at relapse had no difference than at the diagnosis(3 vs. 3, P= 0.401 ). However, most (20/31,64.5%)patients had newly acquired mutations at relapse, the median number of newly acquired mutations at relapse was 1(0~5).The most frequent newly acquired mutations were RAS(NRAS,KRAS,PTPN11,CBL,NF1)pathway(n=5 ,16.1 %),NPM1(n=4,12.9%) and FLT3(n= 3,9.7%) mutations.
 Conclusion
 The study clarified the dynamic changes of genetic mutation in AML patients from diagnosis to CR and relapse. Sweeping away and decreasing the genetic mutations detected at diagnosis promotes patients to CR. Newly acquired mutations and Reappeared mutations detected at the diagnosis lead to relapse. Newly acquired RAS pathway, NPM1 and FLT3 mutations may promote patients to relapse and should be targeted as potential intervention points.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43b674577881290b3192d45c38dea4c5e63c941a" target='_blank'>
              Dynamic Changes of Genetic Mutations in AML Patients from Diagnosis to Complete Remission and Relapse
              </a>
            </td>
          <td>
            Xin Yan, C. Song, Zheng Ge
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaa35d32ecdf58360fe2158aa64cfa3b17ec1726" target='_blank'>
              Evolving cell states and oncogenic drivers during the progression of IDH-mutant gliomas.
              </a>
            </td>
          <td>
            Jingyi Wu, L. G. Gonzalez Castro, S. Battaglia, C. E. El Farran, J. D’Antonio, Tyler E. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-11-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Glioblastoma (GBM) is uniformly lethal due to profound treatment resistance. Altered cellular metabolism is a key mediator of GBM treatment resistance. Uptake of the essential sulfur-containing amino acid methionine is drastically elevated in GBMs compared to normal cells, however, it is not known how this methionine is utilized or whether it relates to GBM treatment resistance.Here, we find that radiation acutely increases the levels of methionine-related metabolites in a variety of treatment-resistant GBM models. Stable isotope tracing studies further revealed that radiation acutely activates methionine to S-adenosyl methionine (SAM) conversion through an active signaling event mediated by the kinases of the DNA damage response. In vivo tumor SAM synthesis increases after radiation, while normal brain SAM production remains unchanged, indicating a tumor-specific metabolic alteration to radiation. Pharmacological and dietary strategies to block methionine to SAM conversion slowed DNA damage response and increased cell death following radiation in vitro. Mechanistically, these effects are due to depletion of DNA repair proteins and are reversed by SAM supplementation. These effects are selective to GBMs lacking the methionine salvage enzyme methylthioadenosine phosphorylase. Pharmacological inhibition of SAM synthesis hindered tumor growth in flank and orthotopic in vivo GBM models when combined with radiation. By contrast, methionine depletion does not reduce tumor SAM levels and fails to radiosensitize intracranial models, indicating depleting SAM, as opposed to simply lowering methionine, is critical for hindering tumor growth in intracranial models of GBM. These results highlight a new signaling link between DNA damage and SAM synthesis and define the metabolic fates of methionine in GBM in vivo. Inhibiting radiation-induced SAM synthesis slows DNA repair and augments radiation efficacy in GBM. Using MAT2A inhibitors to deplete SAM may selectively overcome treatment resistance in GBMs with defective methionine salvage while sparing normal brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d754c9d3d6734a75b23daa13e10505ee3a672db8" target='_blank'>
              TMET-28. RECIPROCAL LINKS BETWEEN METHIONINE METABOLISM, DNA REPAIR AND THERAPY RESISTANCE IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            Navyateja Korimerla, Kari Wilder-Romans, Jie Xu, Isra Haq, Weihua Zhou, Peter Kalev, Nathan Qi, Charles Evans, Maureen Kachman, Zitong Zhao, Angelica Lin, A. Scott, Alexandra M O'Brien, Ayesha U. Kothari, Sravya Palavalasa, Erik R Peterson, Marc L Hyer, Katya Marjon, C. Lyssiotis, Taryn Sleger, Daniel R. Wahl
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="

 Aging impacts the bone marrow microenvironment by inducing inflammation. We hypothesize that aging increases heritable epigenetic heterogeneity, leading to transcriptome heterogeneity within hematopoietic stem and progenitor cells (HSPC). This resulting heterogenous state, combined with the aged microenvironment-driven selection, shifts the functional capacity of the hematopoietic system, contributing to the functional decline of the hematopoietic system as well as the evolution of diseases such as myelodysplastic syndrome and leukemias. Our research aims to investigate clonal heterogeneity in both young and old mice and highlight the phenotypic patterns selected in each case.
 We employed CellTag indexing and single-cell RNA sequencing to track HSPC, identify the selected clones in young and old mice BM, and characterize their molecular signatures. We generated 10x Genomics single-cell RNA-seq for tagged HSPC after 15-day culturing (in vitro) and two months post-transplantation into mice (in vivo). Tracing the expansion of HSPC clones in young and old BM revealed critical changes in HSPC heterogeneity and selection in older mice.
 We observed an increase in transcriptional heterogeneity of old HSPC as compared to young samples (P<0.01, Wilcox test). Genes with higher gene expression variability in older mice coincide with pathways pivotal for HSPC fitness, e.g., ribosome synthesis, inflammation response mechanisms (JAK-STAT), hematopoietic cell lineage, AML, and cancer pathways (FDR<0.1, GSEA). At the clonal composition level, young HSPCs exhibited more oligoclonal reconstitution than old HSPC, indicated by lower Shannon entropy in young HSPCs compared to old (P<0.05 Wilcox test). This result suggests stricter clonal quality control in the young microenvironment, potentially leading to increased purifying selection that removes clones with significant transcriptional profile alterations and reduced fitness. Moreover, comparing the transcriptional profiles of clones showing robust expansion after transplantation (selected or “winner” clones) with negatively selected clones revealed several common pathways shared between young and old selected clones after expansion (FDR<0.1, GSEA), including cell cycle activity, stress response, WNT signaling, and metabolism - identifying candidate mediators of HSPC fitness in vivo. However, major differences between old and young selected clones were evident before expansion. In young, selected clones, signatures related to inflammatory response were observed.Strikingly, we observed a significant positive expression correlation (P<0.0001, Wilcox test) among the selected clones only in young mice, not in old mice. The high transcriptomic similarity among selected clones is consistent with strict clonal quality control in the young BM microenvironment. Notably, we traced the selected clones back among the tagged HSPCs in vitro (scRNA-seq) and observed higher transcriptomic similarity among selected clones from young vs. old donor mice even before transplantation (P<0.0001, Wilcox test). Collectively, these results underscore the selective nature of the young BM, contrasting with the decline of HSPC quality control in old BM, contributing to functional hematopoietic decline and fueling pre-malignant evolution.
 These studies reveal striking differences in clonal dynamics and quality control among HSPC in young and old mice. Clonal tracing of hematopoietic clones provides insight into how clonal competition and microenvironmental selection can contribute to hematopoietic decline and malignant evolution. Better understanding of the pathways that contribute to HSPC fitness and functional decline with aging could lead to approaches to track and perhaps even counter these aging-associated changes, which is expected to contribute to improved hematopoietic function and reduced risk of malignancy in the elderly.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff59a979a8b7594abbd6b4f17c49725e50fa5fe2" target='_blank'>
              Title: In Vivo Clonal Tracing of Hematopoietic Stem and Progenitor Cells Reveals Increased Clonal Heterogeneity during Aging, Alongside Critical Changes in Selection Patterns
              </a>
            </td>
          <td>
            Lamis Naddaf, Marco De Dominici, Xiaowen Chen, Parveen Kumar, James S Chavez, Travis Roeder, Shilpita Karmakar, Eric M. Pietras, Hideyuki Oguro, James DeGregori, Sheng Li
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Pancreatic ductal adenocarcinoma (PDAC) is a highly malignant tumor with a dismal prognosis. Gemcitabine‐based chemotherapy has emerged as a first‐line treatment for PDAC. However, the development of gemcitabine resistance often results in therapeutic failure. In order to uncover the underlying mechanisms of gemcitabine resistance, gemcitabine‐resistant PDAC cell lines and patient‐derived xenograft (PDX) models are established and subjected to RNA sequencing. It is found that CMTM6 is closely related to gemcitabine resistance in PDAC. Multi‐omics analysis revealed that EP300‐mediated H3K27ac modification is involved in the transcriptional activation of CMTM6, which maintains IGF2BP1 expression by preventing its ubiquitination. The m6A reader IGF2BP1 stabilizes the EP300 and MYC mRNAs by recognizing m6A modifications, forming a positive feedback loop that enhances tumor stemness and ultimately contributes to PDAC resistance. The combined application of the EP300 inhibitor inobrodib and gemcitabine exerts a synergistic effect on PDAC. Overall, these findings reveal that the EP300–CMTM6–IGF2BP1 positive feedback loop facilitates gemcitabine resistance via epigenetic reprogramming and the combined use of inobrodib and gemcitabine represents a promising strategy for overcoming chemoresistance in PDAC, warranting further investigation in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5270bc6ca7f6402a187ec472be806c9996060894" target='_blank'>
              Epigenetic Activation of the CMTM6‐IGF2BP1‐EP300 Positive Feedback Loop Drives Gemcitabine Resistance in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Ying-Qin Zhu, Yue Huang, Yin-Hao Shi, Chen-song Huang, Guang-Yin Zhao, Zhi-De Liu, Ming-Jian Ma, Jing-Yuan Ye, Xiang Xu, Qi Liu, Xi-Tai Huang, Jie-Qin Wang, Qiong-Cong Xu, Xiaoyu Yin
          </td>
          <td>2024-11-03</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genome editing enables sequence-function profiling of endogenous cis-regulatory elements, driving understanding of their mechanisms and the development of gene therapies. However, these approaches cannot be combined with direct scalable readouts of chromatin structure and accessibility across long single-molecule chromatin fibers. Here we leverage a double-stranded DNA cytosine deaminase to profile chromatin accessibility at high depth and resolution at endogenous loci of interest through targeted PCR and long-read sequencing, a method we term targeted deaminase-accessible chromatin sequencing (TDAC-seq). Powered by high sequence coverage at targeted loci of interest, TDAC-seq can be uniquely integrated with CRISPR perturbations to enable the functional dissection of cis-regulatory elements, where genetic perturbations and their effects on chromatin accessibility are superimposed on the same single chromatin fiber and resolved at single-nucleotide resolution. We employed TDAC-seq to parse CRISPR edits that activate fetal hemoglobin in human CD34+ hematopoietic stem and progenitor cells during erythroid differentiation as well as in pooled CRISPR and base editing screens tiling an enhancer controlling the globin locus. Together, TDAC-seq enables high-resolution sequence-function mapping of single-molecule chromatin fibers by genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f34b7eff8b3e483363a66c87a758f76a21f0a133" target='_blank'>
              Coupling CRISPR Scanning with Targeted Chromatin Accessibility Profiling using a Double-Stranded DNA Deaminase
              </a>
            </td>
          <td>
            Heejin Roh, Simon P. Shen, Yan Hu, H. Kwok, Allison P. Siegenfeld, Ceejay Lee, Marc Anthony Zepeda, , Shelby A. Roseman, V. Sankaran, Jason D. Buenrostro, Brian B Liau
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="

 Introduction: Despite achievement of complete remission (CR) following chemotherapy, Acute Myelogenous Leukemia (AML) relapses in the majority of adult patients. The identification of cellular and molecular mechanisms that underlie resistance, persistence, and competitive regrowth of leukemic cells is of high interest to improve the outcome of AML patients. In this study, we have investigated the consequences of sterile leukemic inflammation on acquired novel properties and therapy resistance in human AML.
 Methods: Using single cell RNA sequencing we analyzed twenty paired AML specimens procured at diagnosis and at relapse from prior CR, followed by comparative gene expression analyses of selected leukemic cell populations. In addition, we compared the single cell gene expression in leukemic monoblasts/monocytes with pooled normal monocytes of five healthy donors. We used GO analysis, pathway analysis and the database “interferome” to investigate different leukemic cell populations for enrichment of inflammatory and other signatures. We treated human non-malignant CD34+ cells with 1,000 U IFNα and IFNγ for 4h and 24h and detected gene expression changes of selected target genes via qRT-PCR. We measured the expression of IFNα, IFNb and IFNγ together with 3 retrotransposons (LINE elements) and 5 endogenous retroviruses (ERVs) in 6 immature AML (M0, M1, M2) and 8 mature AML (M4, M5a/b) via qRT-PCR. We then incubated 19 primary human AML with 1,000 U IFNα and IFNγ in 20% FBS/RPMI1640 for 4h followed by treatment with escalating doses (1 nM - 10 μM) of Venetoclax for 24 h while IFN was still present. The expression of BCL2 family members BCL2, BCL-xL, MCL1 and BCL2A1 after 4 h IFN pre-stimulation was assayed in 14 AML cases via qRT-PCR and after 24 h, 48 h and 72 h IFN treatment in 8 AML cases using immunoblotting. AML cell lines were pre-incubated with and without 500 nM of the JAK1/2 inhibitors Ruxolitinib or Baricitinib for 2 h followed by 4 h IFN pre-stimulation and measurements of Venetoclax dose responses.
 Results: We have identified cellular relapse patterns and divergent maturity states of AML at diagnosis (Dx) and at relapse (Re) and found that AML comprising monoblastic/monocytic leukemias aberrantly express many inflammatory and interferon (IFN) stimulated genes. The “inflammatory response” was the most prominent gene set when comparing leukemic monoblasts with non-malignant bone marrow resident monocytes. The correlation of expression of IFN induced genes with survival uncovered extremely poor outcomes for AML patients with the highest quartile expression of the genes IL2RA, INHBA, OPTN, EPSTI1, MX1 or BST2 demonstrating a 1,000-day survival fraction of < 20% (range 4-18%). Using human non-malignant CD34+ cells treated with 1,000 U IFNα and IFNγ, we identified IL2RA, INHBA and BCL2A1 as IFN-induced target genes. We detected blast-intrinsic types I and II IFN production in chemotherapy naive AML specimens that correlated proportionally with the expression of ERVs and retrotransposons, which likely caused the IFN production. Moreover, the mean expression of multiple retroelements was substantially higher in FAB M4/M5 AML compared to M0-M2 AML. We found that one major determinant of very low expression of LINE elements and ERVs was TP53 or TET2 mutated status. Importantly, we uncovered a substantial resistance induction to Venetoclax by IFN and measured transcriptional induction of four anti-apoptotic BCL2 family members in all AML cases tested. On the protein level, we confirmed the induction of MCL1, BCL2A1 and BCL-xL by IFN in multiple AML cases suggesting a combinatorical effect of two or three anti-apoptotic BCL2 family members resulting in IFN-mediated Venetoclax resistance. Of interest, most Venetoclax sensitive AML cases lacked BCL2A1 expression. In line with these results, AML cell lines demonstrated resistance to Venetoclax following IFN stimulation, which was partially or completely reversed after JAK-inhibitor pre-treatment.
 Conclusions: Our data revealed the importance of sterile inflammation for prognosis and therapy resistance in human AML. We demonstrated that IFN markedly reduces the sensitivity to Venetoclax induced AML cell death via combinatorial upregulation of MCL1, BCL2A1 and BCL-xL. Given that Venetoclax combinations are now major therapies for AML, our findings could improve our biological understanding and the therapy for adult AML.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d055ae446374528df986be0133a8abc5e3dd7c8" target='_blank'>
              AML Blasts Intrinsic Interferon Production Causes Resistance to Venetoclax in Human Acute Myeloid Leukemia Via Upregulation of MCL1, BCL2A1 and BCL-XL
              </a>
            </td>
          <td>
            Josephine Dubois, Kai Wu, Darren King, Xianing Zheng, Qianyi Ma, Karan Bedi, Jun Li, S. Malek
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The determination of the genetic subtypes of primary CNS lymphoma (PCNSL) and their relationship to differential chemo-immunotherapeutic response has not been established. There is a particular need for genomic biomarkers that identify patients with newly-diagnosed PCNSL at high risk of early progression and death. We applied targeted next-generation sequencing for detection of recurrent single nucleotide variants, copy number alterations and zygosity abnormalities in diagnostic specimens from 78 PCNSL patients treated with a standard methotrexate-based regimen, to identify prognostically significant molecular subgroups. All patients received induction immunochemotherapy and 44 proceeded to dose-intensive consolidation. Genomic aberrations at four loci were associated with 91% of lymphoma progression events and all 15 deaths: (1) chromosome 6p copy-neutral loss of heterozygosity (CN-LOH) or focal homozygous deletion (HD) at 6p21.3, and mutations of tumor suppressor genes (2) BTG1, (3) ETV6 and (4) TP53. Cox regression multivariate analysis demonstrated a high risk of progression in patients with aberrations at these loci. Genomic aberrations at these loci were also associated with significantly shorter survival. Lower expression of HLA-DR was associated with 6p CN-LOH/6p21.3 HD and inferior prognosis. These genomic aberrations identify a high-risk molecular subgroup that may inform risk stratification in PCNSL. Further elucidation of the mechanisms of therapeutic resistance associated with the high-risk genetic phenotype is requisite to facilitate precision medicine and progress in therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d443853490c421bb34b808e8720db40347672b" target='_blank'>
              Identification of Genomic Biomarkers of Disease Progression and Survival in Primary CNS Lymphoma.
              </a>
            </td>
          <td>
            H. Geng, Shirley Shaoyi Mo, Lingjing Chen, A. Ballapuram, Mazie Tsang, Ming Lu, A. Rauschecker, Kwun Wah Wen, W. P. Devine, D. A. Solomon, J. Rubenstein
          </td>
          <td>2024-11-13</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background/Objectives: AML is an aggressive malignant disease characterized by aberrant proliferation and accumulation of immature blast cells in the patient’s bone marrow. Chemotherapeutic treatment can effectively induce remission and re-establish functional hematopoiesis. However, many patients experience chemoresistance-associated relapse and disease progression with a poor prognosis. The identification of molecular determinants of chemoresistance that could serve as potential targets for the therapeutic restoration of chemosensitivity has proven to be challenging. Methods: To address this, we have analyzed longitudinal changes in the expression of microRNAs during disease progression in a small set of four AML patients, combined with gene ontology (GO) pathway analysis and evaluation of gene expression data in patient databases. Results: MicroRNA profiling of bone marrow samples at diagnosis and after relapse revealed significant differential expression of a large number of microRNAs between the two time points. Subsequent GO pathway analysis identified 11 signal transduction pathways likely to be affected by the differential miRNA signatures. Exemplary validation of the FoxO signaling pathway by gene expression analysis confirmed significant upregulation of FOXO1 and the target genes GADD45 and SOD2. Conclusions: Here, we show how a microRNA-based pathway prediction strategy can be used to identify differentially regulated signaling pathways that represent potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ad3edfd6af9770fc155f2d3924cf9e30fae927" target='_blank'>
              MicroRNA Screening Reveals Upregulation of FoxO-Signaling in Relapsed Acute Myeloid Leukemia Patients
              </a>
            </td>
          <td>
            Paula Reichelt, S. Bernhart, U. Platzbecker, Michael Cross
          </td>
          <td>2024-12-19</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31d8028a34805d9bd64c4990406de1eae29e1a1" target='_blank'>
              Age-related genomic alterations and chemotherapy sensitivity in osteosarcoma: insights from cancer genome profiling analyses.
              </a>
            </td>
          <td>
            Hidetatsu Outani, Masachika Ikegami, Yoshinori Imura, Shou Nakai, Haruna Takami, Yuki Kotani, A. Inoue, S. Okada
          </td>
          <td>2024-12-17</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Background: Treatment of chronic lymphocytic leukemia (CLL) has been transformed with therapies targeting Bruton's tyrosine kinase (BTK) and BCL-2, but are limited due to the development of resistance. Dual targeting of BTK and BCL-2 has been efficacious in the clinic, with many patients achieving uMRD and prolonged remission off therapy. We have previously demonstrated the pre-clinical efficacy of LP-168 (Rocbrutinib), a novel selective 4th generation BTKi with an active warhead capable of covalent interaction with WT and T474I BTK and non-covalent binding when a BTK C481 mutation is present (Gordon et al. 2023). This allows for continued BTK inhibition despite development of common resistance mutations. To explore potential combination strategies with LP-168, we utilized a genome wide CRISPR/Cas9 knockout screen with validation of identified targets.
 Methods: Genome wide CRISPR/Cas9 knockout screening was performed in HG-3 CLL cells using the Brunello library with analysis via the MAGeCK pipeline. Cells were treated with IC20 (1 uM) of LP-168 for 3 days. Following exclusion of essential genes, negatively selected sgRNAs that were differentially expressed in the LP-168 group were used for analysis. Validation of hits was performed via pharmacological inhibition utilizing CellTiter-Glo, Annexin V/PI, and western blots using HG-3, OSU-CLL, TMD8 WT BTK, TMD8 C481S BTK, and TMD8 T474I BTK cells in addition to primary patient samples. Experiments using TMD8 cells were performed following CRISPR modification to insert either C481S or T474I BTK.
 Results: The CRISPR/Cas9 screen revealed 1875 genes with at least 3 out of the 4 sgRNAs depleted in both replicates of the LP-168 treated group vs control. KEGG/GO analysis demonstrated significant enrichment of regulation of mitochondrial membrane potential (p=0.00195) and reactive oxygen species pathways (p=0.00195). As BCL-2 interacts with both pathways, and was amongst the top-ranking genes depleted, we chose to target it with either venetoclax (BCL-2i) or with LP-118, a dual BCL-2/-xL inhibitor currently under investigation for R/R hematological malignancies.
 Pharmacologic inhibition of BTK + BCL-2, with either venetoclax (ven) or LP-118, showed synergistic reductions in proliferation of HG-3 (p<0.05, n=3; p<0.05, n=3, respectively), OSU-CLL (p<0.001, n=3, p<0.0001, n=3, respectively), and TMD8 WT BTK (p<0.05, n=3; p<0.01, n=3, respectively) cells at physiologically achievable concentrations (10 nM - 1 uM LP-168 and 10 pM - 10 nM ven or LP-118). LP-168 + ven or LP-118 induced significant cytotoxic effects in OSU-CLL (11%, p=0.001, n=3; 21%, p<0.00001, n=3, respectively) and TMD8 WT BTK (36.4%, p<0.00001, n=3; 66.7%, p<0.00001, n=3, respectively) cells following 72-hour drugging. Next, we utilized primary CLL B cells from treatment-naïve patients and found after both a 24- or 48-hour exposure, the combination of 1 uM LP-168 + 4 nM LP-118 (88%, p<0.00001, n=8) or 1 uM LP-168 + 4 nM ven (73%, p<0.00001, n=8) was able to induce cytotoxicity. Following a 2-hour drugging, LP-168 inhibited both BCR and AKT/mTOR signaling in primary CLL B-cells shown by reduced phosphorylation of BTK, ERK, AKT, and GSK3β, with both combination treatments inducing further inhibition of these targets (98%, p<0.0001, n=4; 49%, p<0.05, n=4; 37%, p<0.05, n=4; 46%, p<0.01, n=4, respectively). Further, we observed retained synergistic effect with both covalent (C481S; p<0.01, n=3) and non-covalent (T474I; p<0.001, n=3) BTKi resistance mutations at physiologically relevant concentrations. Finally, to determine if these combinations retain efficacy in both BTKi and venetoclax resistant settings of CLL, we utilized patient samples who were resistant to both BTKi and ven and treated them with combination of LP-168 + ven and found both combinations retained cytotoxic effect following 24- or 48-hour exposure.
 Conclusions : These data show combined use of LP-168 with pharmacological inhibitors targeting BCL-2 and BCL-xL display synergistic activity in CLL, even in the presence of mutations that mediate resistance to BTKi and BCL-2i. These data are consistent with clinical data showing dual targeting of BTK, with previous generation inhibitors, and BCL-2 has been effective in the clinic, with many patients achieving uMRD and prolonged remission off therapy and supports continued preclinical and future clinical investigation of LP-168 with inhibitors of BCL-2 and BCL-2/-xL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf271276837cbcb773dde284829dfbf80ed7104" target='_blank'>
              Genome-Wide CRISPR/Cas9 Knockout Screen Reveals Novel LP-168 (Rocbrutinib) Combinations Targeting BCL-2 Protein Members for Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            Britten K. Gordon, E. Whipp, Lianbo Yu, Samon Benrashid, Andrew D. Mitchell, Alexander He, Jazmin Urrutia, S. Misra, Alexander R. Marr, Tzung-Huei Lai, Naina Karnati, Shanmugapriya Thangavadivel, K. Rogers, S. Bhat, N. Grieselhuber, Yi Chen, Fenlai Tan, S. Anthony, Yu Chen, Deepa Sampath, R. Lapalombella, J. Woyach
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="

 Aging is a process of systemic deterioration and the most significant risk factor for cancers. Clonal hematopoiesis (CH) commonly occurs with aging and links to higher mortality, leukemia risk, and cardiovascular diseases. Age-related CH involves the abnormal clonal expansion of hematopoietic stem cells (HSCs) bearing somatic mutations in genes frequently mutated in leukemia, including genes encoding epigenetic regulators, such as the DNA demethylase TET2. While such mutations are known to alter the HSC epigenome, the mechanisms through which these mutations drive HSC self-renewal, myeloid transformation, and inflammatory response remain elusive.
 In this study, we hypothesize that aging and CH mutations cooperatively reshape the HSC transcriptomic landscape and enhancing HSC competitive advantage that facilitates clonal expansion. Recently, aging has been closely associated with Tet2 mutation impact. Using single-cell multi-omic analyses and flow-cytometric phenotyping, we demonstrated, for the first time, that HSC aging processes at transcriptomic, epigenomic, and cellular levels are mitigated by Tet2 deficiency in mice, in an age-dependent manner. at ages greater than or equal to 16 months, but not at a young age, based on gene expression and chromatin accessibility at a single-cell resolution. This age mitigation was further validated by differential analysis of gene expression, open chromatin accessibility and DNA methylation that are elevated with aging but lowered with Tet2 deficiency in HSCs from old mice. Moreover, we found that Tet2 deficiency , a cellular hallmark of HSC aging, in an age-dependent manner. We observed increased gene expression with aging, such as for Eya4 and Lars2, which are downregulated by Tet2∆/∆ in HSCs from older mice, suggesting that Tet2 deficiency mitigates the process of aging. Also, the increased expression of Cdk6, Msi2, and Sox4 in Tet2∆/∆ HSCs, normally reduced in old HSC, promote the HSC self-renewal as shown in earlier. These gene expression changes were confirmed by single-nuclear open chromatin accessibility analysis, suggesting that Tet2 deficiency mitigates epigenome reprograming during HSC aging, as shown by increased differential single-nuclear chromatin accessibility of their motifs via ChromVar analysis. These TFs' target genes play critical roles in HSC commitment to progenitors, hemostasis, and proliferation. The methylation status of these TF binding motifs is linked to the regulation for the targets' gene expression.
 In conclusion, our findings reveal that Tet2 deficiency significantly contributes to the mitigation of HSC aging and hijacks the HSC expansion strategy during HSC aging via epigenetic reprogramming, which contributes to age-related clonal hematopoiesis. By elucidating the transcriptomic and epigenomic alterations in Tet2-deficient HSCs, our study provides novel insights into how age and somatic mutations interact to promote the pathogenesis of age-related hematological diseases. These discoveries not only enhance our understanding of HSC aging mechanisms but also offer potential biomarkers for early detection and targets for therapeutic intervention in age-associated clonal disorders.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11a53a46e4e9db18164957f387c6662bd6a099db" target='_blank'>
              Tet2 Deficiency Mitigates Epigenetic Aging in Clonal Hematopoiesis
              </a>
            </td>
          <td>
            Sheng Li, Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Eric M. Pietras, James DeGregori, Jennifer J. Trowbridge, Hideyuki Oguro
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="

 Introduction: Despite advances in therapy for pediatric B-cell acute lymphoblastic leukemia (pB-ALL), relapse remains a leading cause of death in children with cancer. The gene encoding the High Mobility Group A1 (HMGA1) chromatin regulator is highly expressed in stem cells and hematologic malignancies where high levels portend poor outcomes (Li et al. Blood 2022). We discovered that transgenic mice overexpressing Hmga1 in lymphoid cells develop leukemic transformation by amplifying transcriptional networks involved in stem cell function, proliferation, and inflammation (Hillion et al, Cancer Res 2008, Schuldenfrei et al, BMC Genomics 2011). Further, HMGA1 is overexpressed in pB-ALL with highest levels after early relapse (Roy et al, Leuk Lymphoma 2013), suggesting that HMGA1 drives relapse. We therefore sought to: 1) test the hypothesis that HMGA1 is required for relapse in pB-ALL, and, 2) identify targetable mechanisms underlying HMGA1.
 Methods: To elucidate HMGA1 function in pB-ALL, we silenced HMGA1 using CRISPR or short hairpin RNA (shRNA) in cell lines from relapsed pB-ALL, including REH, which harbor the TEL-AML1 fusion, and 697, which harbor the E2A-PBX1 fusion. We assessed leukemia phenotypes in vitro and leukemic engraftment in vivo. To dissect molecular mechanisms, we integrated multiomic sequencing (seq) approaches, including RNAseq and ATACseq. To identify pathways relevant to patients, we assessed HMGA1 gene networks and clinical outcomes in pB-ALL cohorts. Artificial intelligence (AI) was used to predict drugs that target HMGA1 networks which we tested in combination therapy in patient-derived xenograft (PDX) models.
 Results: Silencing HMGA1 expression via CRISPR or shRNA in relapsed pB-ALL cell lines (REH, 697) disrupts proliferation and clonogenicity while increasing apoptosis. To assess HMGA1 function in vivo, we compared leukemogenesis following tail vein injection of pB-ALL cells with or without HMGA1 depletion in immunodeficient mice (NOD/SCID/IL2 receptor gammanull). Strikingly, survival is prolonged in mice injected with either pB-ALL cell line (REH, 697) after HMGA1 depletion. Further, leukemic cells that ultimately engraft show increased HMGA1 expression relative to the pool of injected cells with HMGA1 silencing, suggesting that escape from HMGA1 silencing is required for engraftment. RNA/ATACseq revealed transcriptional networks governed by HMGA1 that regulate proliferation (G2M checkpoint, E2F), RAS/ERK signaling, hematopoietic stem cells, and ETS variant 5 transcription factor (ETV5) targets. We focused on ETV5 given its role in role in HSC self-renewal and association with other forms of aggressive leukemia. CRISPR-mediated ETV5 gene silencing recapitulates effects of HMGA1 silencing in pB-ALL cell lines, decreasing proliferation and clonogenicity in vitro, while delaying leukemogenesis in vivo. Further, restoring ETV5 expression in pB-ALL cell lines with HMGA1 silencing partially rescues anti-leukemogenic effects of HMGA1 depletion. Mechanistically, HMGA1 binds to AT-rich regions within the ETV5 promoter (-0.7 kb, -0.2 kb) and recruits activating histone marks (H3K27Ac, H3K4me3, H3K4me1) to induce ETV5. Moreover, HMGA1 and ETV5 are co-expressed and up-regulated in primary blasts from children with relapsed pB-ALL where high HMGA1 correlates inversely with time to relapse. AI predicted that the histone deacetylase inhibitor (HDACi), vorinostat, targets HMGA1-ETV5 gene networks. In PDX models of pB-ALL from children who relapsed, we found that vorinostat enhances responses to standard B-ALL therapy (dexamethasone, vincristine), decreasing leukemic burdens beyond that of conventional combination therapy.
 Conclusions: We discovered a previously unknown epigenetic program whereby HMGA1 up-regulates ETV5 networks by binding to chromatin and recruiting active histone marks to ETV5 promoter-enhancer regions. Both HMGA1 and ETV5 are up-regulated at relapse and high HMGA1 levels predict decreased time to relapse. Finally, HMGA1-ETV5 networks can be targeted by the HDACi, vorinostat, which enhances responses to standard pB-ALL therapy in PDX models. Our findings reveal HMGA1 as an epigenetic switch that activates ETV5 and HSC gene networks in relapsed pB-ALL and promising therapeutic target to treat or prevent relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ff096f8415c3bff1d48d90aa3cf6a5477e55b62" target='_blank'>
              Targeting HMGA1 Epigenetic Regulators and Downstream ETV5 Pathways with Histone Deacetylase Inhibitors in Combination Therapy for Relapsed, Pediatric B-ALL
              </a>
            </td>
          <td>
            Linda S. Resar, Jung-Hyun Kim, L. Luo, Katharina E. Hayer, Camilo Arenas Merizalde, Leslie Cope, A. Thomas-Tikhonenko, S. Tasian
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Hypomethylating agents (HMAs), such as decitabine and 5-azacytidine (AZA), are valuable treatment options for patients with acute myeloid leukemia that are ineligible for intensive chemotherapy. Despite providing significant extensions in survival when used alone or in combination, eventual relapse and resistance to HMAs are observed. The mechanisms leading to these outcomes are still not well defined and may, in part, be due to a focus on leukemic populations with limited information on the effects of HMAs on non-leukemic cells in the blood and other tissue compartments. In this study, we elucidated effects on immune-related gene expression in non-leukemic blood cells and the spleen during AZA treatment in leukemia-challenged mice. We observed significant changes in pathways regulating adhesion, thrombosis, and angiogenesis as well as a dichotomy in extramedullary disease sites that manifests during relapse. We also identify several genes that may contribute to the anti-leukemic activity of AZA in blood and spleen. Overall, this work has identified novel gene targets and pathways that could be further modulated to augment efficacy of HMA treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061ff6ae918890ac78976d181f8aac552d1d7fd9" target='_blank'>
              Effects of Hypomethylating Agents on Gene Modulation in the Leukemic Microenvironment and Disease Trajectory in a Mouse Model of AML
              </a>
            </td>
          <td>
            N. Ebelt, Suvithanandhini Loganathan, Lara C Avsharian, Edwin R Manuel
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Chronic Lymphocytic Leukemia (CLL) is the most common hematological malignancy in the United States, accounting for approximately 25% to 35% of all leukemias. Targeted therapies such as Bruton's Tyrosine Kinase (BTK) and B-cell lymphoma-2 (BCL-2) inhibitors ibrutinib and venetoclax have produced durable responses in CLL however, a subset of patients experience relapse and poor outcomes. This group with a poor prognosis is in urgent need of therapies that address the unique biology of treatment resistant CLL.
 To investigate mutant TP53's contribution to disease progression, we collected samples at baseline from patients entering ibrutinib treatment for front-line and relapsed/refractory CLL. Large-scale exome sequencing was performed on 270 patient samples revealing that 106 patients harbored some TP53 aberration, mostly in the DNA binding domain (DBD). Mutational analysis revealed missense mutations R175H and R248Q were among the most frequently occurring. These variants are known gain of function (GOF) mutations in solid tumors contributing to tumor progression and therapeutic resistance. Eighty-two percent of TP53 mutations in this cohort had a simultaneous deletion of the short arm of chromosome 17 (del17p) shown by fluorescence in situ hybridization (FISH). Del17p serves as an independent predictor of progression on ibrutinib as these patients had an inferior response to ibrutinib. These data lead us to hypothesize that the presence of TP53 mutations in CLL supports a shortened time to initial therapy and progression and relapse compared to patients without mutations in TP53.
 Wild type (WT) p53 is an essential transcription factor functioning as a tumor suppressor responsible for DNA damage repair, regulating apoptosis, and maintaining genomic stability. To describe mutant p53 activity, DNA damage was induced by ionizing radiation (IR) administered at 3Gy for 1 hour. WT but not mutant TP53 samples responded to DNA damage by stabilizing p53 protein expression. After stabilization, WT p53 strongly induced p21 protein, whereas p53 mutant expressing patient samples did not induce p21 protein in response to IR indicating a disrupted DNA repair pathway ex vivo. However, these studies did not show which genes are being targeted transcriptionally by mutant p53 nor how mutant p53 binds to and regulates these targets allowing these affected cells to subvert apoptosis.
 To further investigate this, we independently modeled this system in two cell lines. The HG3 and OSU-CLL lines are patient-derived lines created using CRISPR/Cas9 gene editing to introduce specific mutations at R175H and R248Q into WT p53 CLL. Homozygous knock-in of R175H and R248Q mutations on p53 was confirmed through Sanger sequencing on expanded single-cell clones. A profile of differentially expressed genes was revealed during RNA-seq analysis of mutant TP53 and WT TP53 cell lines. Genes overexpressed in relation to WT include transcription factors such as AFF3 and BCL11b and known tumor suppressors like KLHL14. Interestingly, a strong downregulation was seen in CYP4F3 across mutants whose upregulation is shown to be associated with resistance to chemotherapeutic agents such as fludarabine and targeted agents such as ibrutinib in the mutant TP53 setting. To determine whether p53 mutants in CLL exhibit a unique transcriptional signature like what is seen in solid tumors, we are conducting chromatin immunoprecipitation (ChIP) for p53 in these cell lines.
 In summary, our study reveals that TP53 mutations take advantage of mechanisms of resistance towards apoptosis-inducing therapies and impair the normal tumor-suppressive function of WT p53 which remains to be addressed in the clinic. These significantly worsen patient prognosis and accelerate relapse. Ongoing analysis will identify and characterize the action of these differentially expressed targets in CLL and highlight their role in facilitating survival in TP53 mutant CLL cells during therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd2a607318eb84b0272d94a7830a09a85dda87" target='_blank'>
              TP53 Gain-of-Function Mutants Display Unique Transcriptional Profiles in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            C. Williams, Suresh Satpati, J. Ravikrishnan, Bailey Slawin, Daisy Y. Diaz Rohena, Palaniraja Thandapani, K. Rogers, S. Bhat, Kunal Rai, J. Woyach, Deepa Sampath
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="

 Background: Tumor cell plasticity and genome instability contribute to tumor heterogeneity, leading to drug resistance and relapse in cancer patients. Despite great progress in improving 5-year survival rate in multiple myeloma (MM), relapsed and refractory MM (RRMM) remains a major challenge. Given that p53 mutations account for 8% of MM cases, we constructed 50 nonsynonymous mutations of p53 detected in MM patients which include 37 mutations in MMRF database and 13 from published studies. We aim to investigate their functions and interactions with a new tumor suppressor NR2E3, nuclear receptor subfamily 2 group E member 3. We previously demonstrated that NR2E3 stimulates p53 acetylation and its small-molecule agonist 11a represses cancer cells bearing p53 signaling in NCI-60 cancer cell panel.
 Results: Our study of TCGA and “All of US” databases uncovered the association of NR2E3 nonsynonymous mutations with 4 cancers including MM. We constructed all NR2E3 mutations detected in MM patients, and NR2E3R334W lost the capability of activating p53 transactivity. In contrast, high expression of NR2E3 correlated to superior disease-free and overall survivals of 33 cancers. Co-expressing NR2E3 with p53 mutation partially rescued a subgroup, including 4 hotspot mutations, of 50 p53 mutations in p53-responsive reporter assay. For example, p53R175H was activated by NR2E3 through stimulating the acetylation of p53R175H at K317/K386 and/or K373/K382. Activated p53R175H increased the expressions of its downstream tumor-suppressing genes, such as p21, Puma, DDIT3 and ATF3.
 NR2E3's agonist 11a enhanced p53 acetylation and transactivation of downstream tumor-suppressing genes. Disrupting NR2E3 by RNAi or overexpression modulated 11a-stimulated p53 acetylation and transactivities. Interestingly, 11a synergized with FDA-approved HDAC inhibitor Romidepsin in killing MM1S myeloma cells that express p53WT and with a new small-molecule FOXM1 inhibitor NB73 in killing OPM2 myeloma cells that carry homozygous alleles of p53R175H.
 FOXM1 is a master transcription factor of metabolism and cell cycle progression in Forkhead box family, and a critical molecular vulnerability in RRMM. Our recent study demonstrated that FOXM1 inhibitor NB73 synergizes with BCL2 inhibitor Venetoclax in killing non-t(11;14) myeloma cell lines via transcriptional inhibition of MYC pathway. The ChIP-seq study also resolved the DNA binding sites of FOXM1 on the promoters of both p53 and NR2E3 in myeloma cell lines. Degrading FOXM1 protein by NB73 decreased the binding of FOXM1 to the promoters of NR2E3 and p53, enhanced their transcriptions and protein levels in myeloma cell lines. For instance, these NB73-treated OPM2 cells expressed more NR2E3WT and p53R175H. The addition of 11a stimulated NR2E3WT which further rescued p53R175H, leading to intensive cell apoptosis.
 Ongoing experiments: Rescuing p53 mutations by NR2E3 and its agonist 11a is a very interesting observation. U266 cells bearing t(11;14) translocation are resistant to Venetoclax. We found that U266 cells express NR2E3WT, p53A161T and substantial amount of FOXM1. NR2E3 activated p53A161T, and 11a synergized with NB73 but not Venetoclax in killing U266 cells, which may provide a new therapy to t(11;14)-positive myeloma. Moreover, we are conducting the pharmacokinetics study of 11a in 3 mice using oral gavage route, with which we will be able to perform the pharmacodynamics studies of 11a alone and in combination with NB73, Romidepsin or other FDA-approved anti-cancer drugs in NSG mice engrafted with myeloma cell lines bearing various molecular signatures, including p53 mutations. Detailed mechanisms are studied with multifaceted techniques such as ChIP-Seq and molecular/cellular tools.
 Significance & conclusion: Nuclear receptors are the main targets of drug development. NR2E3 used to be a retinal photoreceptor-specific transcription factor, but the expression of NR2E3 has been detected in various tissues in human and mouse with new antibodies, which has led to interesting findings in multiple cancers over the past few years. We showed for the first time that NR2E3 is transcriptionally repressed by FOXM1 and pharmaceutically stimulated to activate p53 by its agonist in myeloma cells. Our studies will provide a new drug candidate for MM that is notoriously difficult to cure and burdened with large societal health care expenditures and high mortality rates.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a3f23393e07d3298723f2a4c9c6d0aa3068be3" target='_blank'>
              Nuclear Receptor NR2E3 Activates Both Wild-Type and Mutated p53 in Multiple Myeloma Cells
              </a>
            </td>
          <td>
            Zhi Wen, Adam Bissonnette, Luke Moat, Yidan Wang, B. Katzenellenbogen, J. Katzenellenbogen, Sung Hoon Kim, Abdul Shour, Adedayo A Onitilo
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Microtubules play essential roles in diverse cellular processes and are important pharmacological targets for treating human disease. Here, we sought to identify cellular factors that modulate the sensitivity of cells to antimicrotubule drugs. We conducted a genome-wide CRISPR/Cas9-based functional genetics screen in human cells treated with the microtubule-destabilizing drug nocodazole or the microtubule-stabilizing drug paclitaxel. We further conducted a focused secondary screen to test drug sensitivity for ∼1,400 gene targets across two distinct human cell lines and to additionally test sensitivity to the KIF11 inhibitor, STLC. These screens defined gene targets whose loss enhances or suppresses sensitivity to antimicrotubule drugs. In addition to gene targets whose loss sensitized cells to multiple compounds, we observed cases of differential sensitivity to specific compounds and differing requirements between cell lines. Our downstream molecular analysis further revealed additional roles for established microtubule-associated proteins and identified new players in microtubule function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe92d3ab4eb0f4d2bf377007f003eebe7871111" target='_blank'>
              Functional genetics reveals modulators of antimicrotubule drug sensitivity.
              </a>
            </td>
          <td>
            Kuan-Chung Su, Elena Radul, Nolan K Maier, Mary-Jane Tsang, Claire Goul, Brittania Moodie, Océane Marescal, Heather R Keys, I. Cheeseman
          </td>
          <td>2024-11-21</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204cd473c099f20975336b46bd49b14354e5b39" target='_blank'>
              Recapitulating the adenoma–carcinoma sequence by selection of four spontaneous oncogenic mutations in mismatch-repair-deficient human colon organoids
              </a>
            </td>
          <td>
            Tomohiro Mizutani, Matteo Boretto, Sangho Lim, J. Drost, D. M. González, Rurika Oka, Maarten H. Geurts, H. Begthel, J. Korving, Johan van Es, R. van Boxtel, H. Clevers
          </td>
          <td>2024-11-01</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Mutagenic processes drive evolutionary progress, with ultraviolet (UV) radiation significantly affecting evolution. Despite extensive research on SOS response-mediated mutagenesis, UV-induced repair mechanisms remain complex, and their effects on cell survival and mutagenesis are not fully understood. We previously observed a near-perfect correlation between RecA-mediated SOS response and mutation levels in Escherichia coli following UV treatment. However, prolonged UV exposure caused transient non-culturability and impaired SOS-mediated mutagenesis. Using fluorescent reporters, flow cytometry, promoter-reporter assays, single-gene deletions, knockouts, and clonogenic assays, we found that excessive UV exposure disrupts cellular translation, reducing SOS gene expression, albeit with minimal impact on membrane permeability or reactive oxygen species levels. While our findings underline the abundance of repair mechanisms in E. coli cells, enabling them to compensate when specific genes are disrupted, they also highlighted the differential impacts of gene deletions on mutagenesis versus culturability, leading to three major outcomes: (i) Disruption of proteins involved in DNA polymerase for trans-lesion synthesis (UmuC and UmuD) or Holliday junction resolution (RuvC) results in significantly decreased mutagenesis levels while maintaining a transient non-culturability pattern after UV exposure. (ii) Disruption of proteins involved in homologous recombination (RecA and RecB) and nucleotide excision repair (UvrA) leads to both significantly reduced mutagenesis and a more severe transient non-culturability pattern after UV exposure, making these cells more sensitive to UV. (iii) Disruption of DNA Helicase II (UvrD), which functions in mismatch repair, does not affect mutagenesis levels from UV radiation but results in a very pronounced transient non-culturability pattern following UV exposure. Overall, our results further advance our understanding of bacterial adaptation mechanisms and the role of DNA repair pathways in shaping mutagenesis. Author Summary Ultraviolet (UV) radiation has been a significant force in driving genetic variation and adaptation throughout billions of years of evolution. By directly damaging DNA and triggering repair mechanisms, UV radiation is a powerful tool for studying mutagenesis. This study aims to provide new insights into the complex processes behind bacterial mutagenesis, a critical topic in microbiology and public health. Although the connection between the SOS response and DNA repair in UV-treated cells is well known, two key phenomena — UV-induced bacterial cell dormancy and mutagenesis — remain poorly understood. Our findings reveal a highly unusual, SOS response-dependent transient non-culturability in Escherichia coli cells following prolonged UV exposure. However, the downstream mechanisms of this phenomenon and its links to mutagenesis remain unclear. This study seeks to thoroughly investigate these phenomena, offering new insights into their underlying molecular processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8574f751e70ca946f8a54c7557e261ab9d00bd43" target='_blank'>
              UV-Induced DNA Repair Mechanisms and Their Effects on Mutagenesis and Culturability in Escherichia coli
              </a>
            </td>
          <td>
            Sreyashi Ghosh, Mehmet A. Orman
          </td>
          <td>2024-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="

 Introduction
 Multiple myeloma (MM) is a plasma cell malignancy frequently preceded by precursor conditions, monoclonal gammopathy of undetermined significance (MGUS) and smoldering multiple myeloma (SMM). Patients with MGUS and SMM exhibit variable risk of progression to MM that cannot be fully explained by differences in genetic alterations. Characterizing both transcriptional and clonotype features in healthy and malignant plasma cells at a single-cell level may help to identify molecular mechanisms of disease progression and improve prognostication. To address this, we performed single-cell RNA and B cell receptor sequencing on bone marrow (BM) and peripheral blood (PB) CD138-positive samples collected from >250 patients with various disease stages and healthy donors, yielding ~1.7 million plasma cell transcriptional and clonal profiles. Here, we present a comprehensive investigation of their transcriptional programs including clonal expansion during tumorigenesis.
 Methods
 We performed single-cell RNA and V(D)J sequencing on 342 samples (252 BM, 90 PB) from 39 patients with MGUS, 132 patients with SMM, 51 patients with MM, and 41 healthy donors (263 individuals in total). CellRanger v6.0.1. was used to compute gene expression count matrices and identify clonotype information per single cell. Low-quality droplets (with ≥15% mitochondrial expression, or ≤400 genes covered) and potential multiplets (with ≥5000 total genes detected, or ≥50,000 total UMIs) were filtered out. Malignant plasma cells were annotated based on the matched V(D)J clonotype information. Transcriptional signatures were extracted by SignatureAnalyzer v0.0.9 at the single-cell level, and clonotype features were analyzed by Dandelion v0.3.7.
 Results
 Overall, we sequenced 1,681,085 plasma cells, including 1,256,207 malignant and 424,878 normal plasma cells with the clonotype information. Malignant plasma cells from the same patient formed a unique cluster, and these malignant clusters across different patients with the same IgH translocation were closer to each other. We derived multiple gene expression signatures that capture various programs related to dysregulated plasma cell biology as well as tumor stage by comparing healthy and malignant plasma cells between healthy donors and patients across various disease risk groups. Investigating the signature activities across disease stages at a single-cell level, we observed that: (i) the upregulation of CD47, which promotes immune evasion, was established in SMM compared to MGUS; (ii) immune regulation signature was upregulated in plasma cells and was predictive of survival (P-value < 0.0001; e.g., IFI6, a negative regulator of innate immune responses by modulating RIG-I activation); (iii) signatures related to bone biology, hypoxia, metabolic activity, and cytomobility were associated with tumor stage; and (iv) normal plasma cells are unexpectedly heterogeneous, with a clear distinction between circulating and bone marrow-resident normal plasma cells. We also found an MGUS-specific signature enriched in both malignant and normal cells, which could be useful for prognosis of disease progression. After integrating copy-number variations and B-cell receptor clonotypes per cell, we interestingly identified 4 SMM patients that had two distinct tumors with different cytogenetics. These cases may shed light on what transcriptional and genomic factors contributed to the risk, evolution and progression to MM.
 Conclusion
 We generated the largest clonotype-resolved single-cell cohort of patients with MM and its precursor stages (MGUS and SMM), and characterized novel transcriptional signatures related to the transformation of normal to malignant plasma cells and circulating and bone marrow-resident plasma cells among disease stages. Our findings in this dataset could be a useful resource to (i) score plasma cells at a single-cell level for the assessment of disease stages, and (ii) nominate novel targets or prioritize target validation for the treatment or prevention of MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/928b3480a4c485398d39f08c3235ad35723f5477" target='_blank'>
              Single-Cell Profiling of ~1.7 Million Plasma Cells across >250 Individuals Reveals Transcriptional Signatures of Progression and Clonal Expansion in Multiple Myeloma and Its Precursor Stages
              </a>
            </td>
          <td>
            Junko Tsuji, Elizabeth D Lightbody, Romanos Sklavenitis-Pistofidis, Ting Wu, Michael P. Agius, Danielle T. Firer, M. Rahmat, Hadley Barr, Y. Konishi, Ankit K. Dutta, Jean-Baptiste Alberge, N. Su, Cody J Boehner, Michelle P. Aranha, Laura Hevenor, Katherine Towle, E. Horowitz, Maya Davis, Jacqueline Perry, F. Aguet, Ruitong Li, Sungjae Kim, N. Haradhvala, Rebecca Boiarsky, G. Getz, I. Ghobrial
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Aging constitutes complex and dynamic alterations in molecular and physiological processes and is associated with numerous disorders, in part due to increased genetic instability. The aging population is projected to double by 2050, underscoring the urgent need to better understand the relationships between aging and age-related disorders. Repetitive DNA elements are intrinsic sources of genetic instability and have been found to co-localize with mutation hotspots in human cancer genomes. In this study, we explored the relationship between aging and DNA repeat-mediated genetic instability in vivo using an H-DNA-forming mirror-repeat sequence from the cancer-associated human c-MYC gene. Utilizing a unique mutation-reporter mouse model, we observed tissue-specific effects of aging on H-DNA-induced genetic instability, with mutation frequencies increasing in spleen tissues and remaining unchanged in testis tissues. Analysis of the mutation spectra revealed large deletion mutations as the primary contributor to increasing H-DNA-induced mutations, supported by increased cleavage activity of H-DNA structures in aged spleen tissues. Our findings demonstrate that aging has distinct tissue-specific effects on repeat-mediated, cancer-associated mutations, providing insights into the complex relationship between aging and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8066191467ac61ad7ea1a2f5a55dbdea73e14d68" target='_blank'>
              Tissue-Specific Effects of Aging on Repeat-Mediated Mutation Hotspots In Vivo
              </a>
            </td>
          <td>
            Alexandra M. D’Amico, Tonia T. Li, Karen M. Vasquez
          </td>
          <td>2024-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mutations in the TP53 tumor suppressor genes are prevalent in aggressive cancers. Pharmacological reactivation of dysfunctional p53 due to mutations is a promising strategy for treating such cancers. Recently, a multifunctional proline- and glutamine-rich protein, PTB-associated splicing factor (PSF), was identified as a key driver of aggressive cancers. PSF promotes the expression of numerous oncogenes by modulating epigenetic and splicing mechanisms. We previously screened a small-molecule library and discovered compound No.10-3 as a potent PSF inhibitor. Here, we report the discovery of a No.10-3 analog, C-30, as a potent PSF inhibitor. Compared to No.10-3, C-30 treatment specifically suppressed the growth and induced apoptosis of mutant p53-bearing and therapy-resistant cancer cells. Interestingly, C-30 activated a set of p53-regulated genes in therapy-resistant cancer cells. A comprehensive analysis of PSF and p53 binding regions demonstrated a higher level of PSF-binding potential in mutant p53-expressing cancer cells around genomic regions identified as p53-binding peaks in p53-wild type cancer cells. Treatment of mutant p53-expressing cancer cells with C-30 decreases PSF binding around these sites, leading to activated histone acetylation. We further demonstrated that C-30 impaired tumor growth and increased the expression of p53-target genes in vivo. These results suggested that C-30 produces tumor-suppressive effects similar to the functional reactivation of p53, providing a rationale for the inhibition of PSF activity as a promising therapy against treatment-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a883a5ac34ea9851ff495a97f5000312eee25b0f" target='_blank'>
              Inhibition of PSF activity overcomes resistance to treatment in cancers harboring mutant p53.
              </a>
            </td>
          <td>
            Ken‐ichi Takayama, Tomohiro Sato, Teruki Honma, Minoru Yoshida, Satoshi Inoue
          </td>
          <td>2024-12-03</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Metabolic reprogramming is a hallmark of cancer, including alterations in the hexosamine biosynthesis pathway (HBP). Glutamine-fructose-6-phosphate transaminase 1 (GFPT1) is the key regulatory enzyme in the HBP; however, its role in invasive breast carcinoma remains underexplored. Methods This study utilized integrated data from The Cancer Genome Atlas (TCGA) to assess GFPT1 expression in breast cancer (BRCA) patients. Functional enrichment and mutational landscape analyses were performed, along with chemosensitivity predictions. In vitro experiments were conducted by silencing GFPT1 in malignant breast epithelial cells to evaluate changes in proliferation, migration, and apoptosis. Results Elevated GFPT1 expression was linked to advanced-stage breast cancer and identified as an independent prognostic marker for overall survival (OS). High GFPT1 levels were associated with increased cytoplasmic translation, activation of oncogenic pathways, and infiltration of M2 macrophages. The GFPT1-High group also showed a higher mutational burden, with frequent TP53 mutations. Chemosensitivity analysis revealed increased IC50 values for chemotherapy drugs in this group. GFPT1 silencing led to reduced cell proliferation and migration, along with enhanced apoptosis. Conclusion These findings indicate that GFPT1 is a novel prognostic biomarker and a predictive indicator of chemotherapy response in invasive breast carcinoma. GFPT1 influences mRNA translation, cell cycle regulation, and M2 macrophage infiltration, thereby promoting cancer cell proliferation and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6655d004010dcffbbdb0a2efe7ca8624f1e53e" target='_blank'>
              Bioinformatics insights into the role of GFPT1 in breast invasive carcinoma: implications for tumor prognosis, immune modulation, and therapeutic applications
              </a>
            </td>
          <td>
            Jianghui Liang, Xiaolian Deng, Yingyi Zhang, Tianchi Fei, Muzi Ouyang, Chengjie Yu, Yang Xiang, Dongwei Jia, Fangfang Duan
          </td>
          <td>2024-11-22</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Despite advances in targeted therapies, the aggressive transition of chronic lymphocytic leukemia (CLL) to Richter's transformation (RT) remains an unmet clinical need. While oxidative phosphorylation (OXPHOS) shaped by B-cell receptor signaling is a recognized feature in CLL and RT, metabolism changes driving RT progression are poorly understood. We recently developed an RT murine model by B-cell-restricted knockout of Mga, a negative regulator of MYC, in LSK cells derived from Cd19Cre-Cas9-del(13q)-Sf3b1-K700E mice. Murine RT cells exhibit high MYC and Ki67 expression, upregulated Mga targets Myc, Nme1 (Nucleoside diphosphate kinase), and elevated OXPHOS and lipid accumulation. Targeting MYC and OXPHOS with CDK9i-inhibitor (AZ5576) and OXPHOS complex IIi (TTFA- Triflurothenoylacetone) significantly improved overall survival in murine RT, highlighting the Mga-Myc-Nme1 axis and OXPHOS as viable targets in Mga KO-driven RT (Iyer et al., Sci. Transl. Med, 2024, Iyer et al, ASH, 2023).
 To delineate the metabolite changes from CLL to RT, we performed untargeted metabolomics profiling using murine CLL and RT cells. Our results revealed that fatty acid intermediates, glycerol-3P (Gro3P) (FC=11.7) and carnitine (FC=4.3), were significantly (p<0.01) elevated in RT cells. This is coupled with increased expression of the regulatory enzyme Pgp (phosphoglycolate phosphatase) in both murine and human RT samples, suggesting a potential role of MGA deletion in driving aberrant glycerolipid metabolism in RT. Given the heightened lipid accumulation and OXPHOS dependency in RT cells, we hypothesized that linking these metabolic pathways would unveil new vulnerabilities and therapeutic targets for this aggressive disease.
 To translate murine findings to human disease and define the molecular regulation for the metabolic changes, we performed untargeted metabolomics in human B cell lines (Nalm6, MEC1) with or without MGA KO. Fatty acid intermediates, Gro3P (FC=3.5), carnitine (FC=22.1), glutathione (g-glu-cys, FC=2.8), and the pentose phosphate pathway (IMP, FC=4.9; Glucuronic acid FC=4.4), were all significantly upregulated in MGA KO Nalm6 cells. Glycerolipid metabolism pathway was highly correlated (r=0.51, p=0.02) in MEC1 and Nalm6 cells with MGA deletion. Moreover, MGA KO cells showed decreased maximal respiration when treated with etomoxir, a fatty-acid oxidation inhibitor, indicating their reliance on lipid substrates. Further analysis using a labeled 13C-glucose-based isotope tracing assay revealed that MGA KO Nalm6 cells displayed an increased labeling of TCA cycle substrate citrate (FC=12 vs. 2.6, p < 0.001) and a higher labeled fraction of Gro3P (FC=59.4 vs. 36.3, p < 0.01) compared to control cells, highlighting a strong connection between the TCA cycle and glycerol metabolism upon MGA deletion.
 Gro3P is a key metabolite linking glycolysis, glycerolipid synthesis, and OXPHOS via the mitochondrial electron transfer shuttle. To delineate the role of Gro3P in linking the TCA cycle and glycerol metabolism in MGA KO, we examined its impact on lipid synthesis and OXPHOS. Treatment of MGA KO cell lines (MEC1) with Gro3P (500μM) increased lipid droplet formation, elevated the NADH/NAD+ ratio, and enhanced the expression of MYC, NME1, PGP, and mTOR signaling, indicating Gro3P as a critical regulator for lipid metabolism and OXPHOS via the MGA-MYC-NME1-PGP axis. As PGP is a known enzyme converting excessive detrimental Gro3P to less harmful glycerol, we overexpressed PGP in MEC1 cells to examine its role in mediating the crosstalk between OXPHOS and lipid metabolism. Overexpression of PGP increased glycerol levels and lipid accumulation, elevated oxygen consumption, and activated mTOR signaling, confirming PGP as a metabolic gene target. Given the availability of the PGP inhibitor CP1, we examined the therapeutic benefits of combining CP1 with the mitochondrial OXPHOS complex II inhibitor TTFA on murine RT in vivo. The combination improved overall survival (n=8 per group, p < 0.0001) compared to single treatments by reducing mTOR signaling, MYC, and NME1, highlighting the importance of concurrently targeting mitochondrial OXPHOS and glycerolipid metabolism in RT treatment.
 Our studies reveal Gro3P as a metabolite driver of OXPHOS in MGA KO RT via the Mga-Myc-Nme1-Pgp axis. Combining glycerolipid metabolism and OXPHOS targeting may offer effective strategies to conquer RT.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92774e192be30808dd8de01df0eba650e63b19c9" target='_blank'>
              Integrating Metabolomics and Molecular Pathways to Uncover Therapeutic Vulnerabilities in Richter's Transformation
              </a>
            </td>
          <td>
            Prajish Iyer, Brian Jiang, Girish Venkataraman, Joo Y. Song, Wing-Chung Chan, Tanya Siddiqi, Steven T. Rosen, Alexey V. Danilov, Antje Gohla, Lili Wang
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Motivation Many tumours show deficiencies in DNA damage response (DDR), which influence tumorigenesis and progression, but also expose vulnerabilities with therapeutic potential. Assessing which patients might benefit from DDR-targeting therapy requires knowledge of tumour DDR deficiency status, with mutational signatures reportedly better predictors than loss of function mutations in select genes. However, signatures are identified independently using unsupervised learning, and therefore not optimised to distinguish between different pathway or gene deficiencies. Results We propose SNMF, a supervised non-negative matrix factorisation that jointly optimises the learning of signatures: (1) shared across samples, and (2) predictive of DDR deficiency. We applied SNMF to mutation profiles of human induced pluripotent cell lines carrying gene knockouts linked to three DDR pathways. The SNMF model achieved high accuracy (0.971) and learned more complete signatures of the DDR status of a sample, further discerning distinct mechanisms within a pathway. Cell line SNMF signatures recapitulated tumour-derived COSMIC signatures and predicted DDR pathway deficiency of TCGA tumours with high recall, suggesting that SNMF-like models can leverage libraries of induced DDR deficiencies to decipher intricate DDR signatures underlying patient tumours. Availability https://github.com/joanagoncalveslab/SNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afb1aa490661811f25b1b0555092a4330771d6f8" target='_blank'>
              SNMF: Integrated Learning of Mutational Signatures and Prediction of DNA Repair Deficiencies
              </a>
            </td>
          <td>
            Sander Goossens, Y. Tepeli, Colm Seale, Joana P. Gonçalves
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 The small population of cancerous cells that remain following treatment, known as measurable residual disease (MRD), is the major cause of relapse in acute myeloid leukemia (AML). Usually, these refractory cells have gained additional resistance mutations or changed their surface immunophenotypes in ways that preclude detection and phasing by current gold standard flow cytometry or bulk next-generation sequencing assays. For this reason, a multiomic single-cell MRD (scMRD) assay could offer a more comprehensive indicator of relapse and the potential for faster response. Here, we present a new scMRD assay with a 0.01% limit of detection that provides single-cell clonal architecture and immunophenotyping to not only identify residual leukemia cells, but also identify putative DNA or protein targets for novel biomarker insights. The assay enables rare-cell detection on a standard Mission Bio Tapestri® run by adding (i) an upfront enrichment for blast cells, (ii) a DNA and protein panel specifically designed for characterization of clonal architecture in AML MRD hotspots and biomarkers, (iii) an automated analysis pipeline to evaluate single-cell multiomics output, and (iv) multiplexing for simultaneous analysis of up to three patient samples in a single run via germline variant identification. By utilizing Mission Bio's technology for sequencing single cells, this pipeline can identify and correlate co-occurring de novo variants, thereby reducing false positive rates over bulk assays that do not correlate variants. Furthermore, it can create phylogenetic trees of the detected MRD cells and present their surface protein signature and arm-level copy number. To demonstrate assay features and reproducibility, surrogate MRD samples were constructed using positive control cell lines (expressing CD34 and CD117 markers) or diseased cells spiked into healthy bone marrow cells before processing them with the scMRD assay. We detected the 0.01% cell line spike-ins in 20 of 20 samples tested, with an average blast enrichment of 30.8x. We further applied the scMRD assay to bone marrow aspirate samples from AML patients and achieved 0.01% diseased cell spike-in detection. The scMRD assay resolved the genotype clonal architecture, identifying multiple leukemic clones with co-occurring mutations and readily distinguishing pre-leukemic from leukemic clones. The integration of genotype and immunophenotype further enhanced MRD detection by identifying genotype-specific protein expression patterns. By combining high reproducibility with multiomics, this assay offers a potential scalable solution for comprehensive MRD detection that could possibly guide patient stratification and therapeutic decision-making in the future.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f276edbd94144af3b5775b5bed15c5853cfd82" target='_blank'>
              A Multiomic, Single-Cell Measurable Residual Disease (scMRD) Assay for Simultaneous Assessment of DNA Mutations and Surface Immunophenotypes in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            K. Thompson, Ben Geller, L. Nousheen, Indira Krishnan, Shu Wang, Daniel Mendoza, T. Druley, Adam Sciambi
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance to temozolomide (TMZ) chemotherapy due to DNA mismatch repair (MMR) enzyme deficiency is a barrier to improving outcomes for IDH wild-type glioblastoma (GBM) patients. KL-50 is a new imidazotetrazine-based therapeutic designed to induce DNA interstrand cross links, and subsequent double-stranded breaks, in an MMR-independent manner in cells with O-6-Methylguanine-DNA Methyltransferase (MGMT) deficiency. Previous research showed its efficacy against LN229 glioma cells with MMR and MGMT knockdown. Its activity against patient-derived GBM that model post-TMZ recurrent tumors is unclear.


METHODS
We created MMR-deficient GBM patient-derived xenografts through exposure to TMZ, followed by treatment with additional TMZ or KL-50. We also generated isogenic, MSH6 knockout patient-derived GBM and tested them for sensitivity to TMZ and KL-50.


RESULTS
KL-50 extended the median survival of mice intracranially engrafted with either patient-derived TMZ-naïve GBM6 or TMZ-naïve GBM12 by 1.75-fold and 2.15-fold, respectively (p<0.0001). A low dose (4 Gy) of fractionated RT further extended the survival of KL-50 treated GBM12 mice (median survival=80 days for RT+ KL-50 vs. 71 days KL-50 alone, P=0.018). KL-50 also extended the median survival of mice engrafted with post-TMZ, MMR-deficient GBM6R-m185 (140 days for KL-50 vs. 37 days for vehicle, p<0.0001). MSH6-KO increased TMZ IC50 for GBM6 and GBM12 cultures by >5-fold and >12-fold for cell death and live cell count outputs, respectively. In contrast, MSH6-KO actually decreased KL-50 IC50 by 10-80%.


CONCLUSION
KL-50-based compounds are a promising new strategy for the treatment of MGMT-deficient, MMR-deficient GBM that recurs after frontline TMZ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3a0f339bf4219c2fc489415b10b402bba9dd1" target='_blank'>
              The novel DNA cross-linking agent KL-50 is active against patient-derived models of new and recurrent post-temozolomide mismatch repair-deficient glioblastoma.
              </a>
            </td>
          <td>
            Matthew McCord, Thomas Sears, Wenxia Wang, Rahul Chaliparambil, Shejuan An, Jann Sarkaria, C. James, Bruce Ruggeri, Susan E. Gueble, R. Bindra, C. Horbinski
          </td>
          <td>2024-12-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="

 Introduction: ASXL1-mutant Chronic Myelomonocytic Leukemia (CMML) is a high-risk myeloid neoplasm characterized by epigenetic remodeling and overexpression of leukemogenic driver genes. DLK1 (delta-like homolog 1) is a non-canonical Notch ligand, crucial for the proliferation of early hematopoietic progenitor cells and its expression is controlled by maternal imprinting. DLK1 expression is restricted to select progenitor populations, but was found to be overexpressed in several malignancies including ASXL1-mutant CMML. Here we investigate the molecular mechanisms that drive the aberrant re-expression of DLK1 and postulate potential targeted therapeutic interventions.
 Methods: We performed bulk RNA- and ChIP-seq (H3K4me1/3, H3K27ac/me3, H2AK119ub, H2BK20ac) on bone marrow mononuclear cells from 20 patients with CMML (11 ASXL1-mutant, 9 -wildtype). All libraries were sequenced on an Illumina NovaSeq instrument. Differential gene expression and histone modification occupancy between ASXL1-mutant and -wildtype patients were assessed. Single-cell RNA-seq was performed for a subset of patients to investigate the cell populations of interest for DLK1 expression. To assess DLK1 allelic expression, we identified a single nucleotide polymorphism (SNP) in the coding region of DLK1 (rs1802710) and used cases which were confirmed heterozygous by Sanger sequencing. DLK1 transcript levels were quantified by RT-PCR, and the relative abundance of alleles was measured using digital droplet PCR. We designed primers to amplify the region around our SNP of interest and utilized high mismatch discrimination locked nucleic acid (LNA) probes labeled with FAM (reference SNP allele) and HEX (alternate SNP allele) to quantify specific alleles. The number of RNA molecules originating from each allele was quantified by counting the number of droplets that fluoresced with the corresponding probe colors. Illumina 850K EPIC microarrays were used to evaluate DNA methylation. Drug studies were carried out in progenitor colony forming assays derived from ASXL1-mutant patients (bone marrow mononuclear cells) with the dual BRD4 and p300 / CBP inhibitor EP31670 (62 nM, 24-hour exposure), quantifying DLK1 expression by real-time RT-PCR.
 Results: This study included 20 patients with CMML (median age at diagnosis 74 years). All ASXL1 mutations were typical hotspot mutations (most commonly Gly646Trpfs*12) and the variant allele frequencies were compatible with heterozygosity (median 0.41). The spectrum of co-mutations was typical and included TET2 (23%), SRSF2 (16%), CBL (7%), among others. DLK1 was strongly overexpressed in ASXL1-mutant patients (log2 fold-change +4.0, FDR < 0.001). Allele-specific gene expression studies (n = 3) revealed the re-expression of the imprinted maternal DLK1 allele in ASXL1-mutant patients. Single-cell gene expression studies (n = 5) demonstrated that the DLK1 overexpression is restricted to the hematopoietic progenitor cell population. Differential methylation analysis between patients with ASXL1-mutant clonal hematopoiesis (n = 4) and matched controls (n = 8) identified two hypermethylated CpG islands in the DLK1 promoter region (cg18122767, Δß = +8.6%, p = 0.021 and cg15831296, Δß = +5.8%, p = 0.008). Histone modification analysis identified DLK1 as a p300 target gene by mapping H2BK20ac onto established ASXL1-mutant-specific enhancer regions (H3K4me1 + H3K27ac co-occupancy). The distal enhancer EH38D2760182 was predicted to bind p300 and to govern the expression of DLK1. ChIP-seq experiment in K-562 and Kasumi-1 cells showed the binding of BRD4 and p300 to EH38D2760182, respectively. Targeting BRD4 and p300 using EP31670 led to the downregulation of DLK1 expression (mean fold-change = 0.569, p < 0.001, n = 3) in progenitor colony forming assays.
 Conclusions: We observed the aberrant re-expression of the maternally imprinted DLK1 allele in patients with ASXL1-mutant CMML. Both DLK1 promoter hypermethylation and the genotype-specific activation of enhancers by p300 were identified as putative regulatory mechanisms. The distal enhancer EH38D2760182 was found to bind BRD4 and p300 and the pharmacologic disruption of BRD4 / p300 activity in model systems led to the downregulation of DLK1. The clinical-grade dual inhibitor of BRD and CBP / p300 EP31670 is currently being investigated in patients with ASXL1-mutant chronic myeloid neoplasms (NCT05488548).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb9916e8bb79dc70f198a347118cdf86bb9375b" target='_blank'>
              Targeting Aberrant DLK1 Re-Expression in ASXL1-Mutant Chronic Myelomonocytic Leukemia with BRD4 / p300-Directed Epigenetic Small Molecule Therapy
              </a>
            </td>
          <td>
            T. Lasho, Jenna A. Fernandez, Jeong-Heon Lee, Linh D. Pham, Huihuang Yan, C. Finke, A. Mangaonkar, A. Al-Kali, N. Gangat, Joseph Collard, Claes Wahlestedt, M. Patnaik, M. Binder
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="

 Background: Germline predisposition is increasingly recognized in myeloid neoplasms (MN), with a higher frequency in patients with therapy-related MN (Singhal et al., Leukemia 2021). An unequivocal diagnosis of germline predisposition carries actionable considerations for patient management, including the selection of donor stem cell source, adjustment of the conditioning regimen, and management of complications following allogeneic stem cell transplant (Saygin et al., Blood Advances 2023). However, the prevalence of pathogenic germline variants (PGVs) in different types of MN remains unknown. This study examines the frequency and type of PGVs in patients with haematological malignancy (HM) with or without prior exposure to cytotoxic therapies.
 Method: This multi-institution study assessed the prevalence of PGVs in 341 patients with HM. Genomic DNA was extracted from germline samples, including hair-root DNA, mesenchymal stromal cells, skin fibroblasts, or peripheral blood T lymphocytes.
 Results: PGVs were identified in 23.5% (n=80) of 341 HM patients compared to 4% and 3% of patients with single non-hematological cancer and age-matched controls, respectively (P <0.0001). The median age at diagnosis was not different in HM patients with or without PGVs (65.5 vs. 61 years; P=0.05), and PGV frequencies were 26.9%, 27.6%, 24.4%, and 16.5% in patients aged ≤50, 51-59, 60-69, and ≥70 years (P=0.22), respectively.
 Of the 83 PGVs identified, DDX41 (n=13, 15.7%), CHEK2 (n=9, 10.8%), BRCA1 (n=6, 7.2%) and GATA2 (n=5, 6%) were the most frequent. Half of all PGVs were in genes involved in DNA damage repair (DDR) pathways (n=43, 51.8%), followed by PGVs in DDX41 (n=13, 15.7%), and transcription factors (n=11, 13.3%). Interestingly, there were no DDX41 variants in patients diagnosed ≤50 years of age (n=85), however, this population had the highest frequency of PGVs in transcription factors (n=7, 30.4%; P=0.053). Patients ≥70 years of age had PGVs primarily in DDR pathways (n=7, 43.8%) and DDX41 (n=4, 25%). Although the majority of PGVs were distributed evenly across genders, the frequency of DDX41 PGVs was higher in men than women (25% vs. 3.1%; P=0.008).
 We further analyzed the distribution of PGVs across different HM subtypes, including primary MN (P-MN; n=92), multiple cancers including at least one HM (n=236), and bone marrow failure (BMF; n=13). Among the patients with ≥2 cancers, with at least one HM, 55 (23.3%) did not receive cytotoxic therapy (CT) prior to HM (MC-HM), while 181 patients (19%) did receive CT prior to HM diagnosis (CT-HM).
 The frequency of PGVs did not differ between patients with P-MN and those with MC-HM or CT-HM (30% vs. 19.9%, P=0.056). The relatively high frequency of P-MN in our cohort is probably due to ascertainment bias, as P-MN patients were screened if they had family history, phenotypic features, or a younger age at presentation than expected, whereas patients with ≥2 cancers including HM were screened regardless of other features.
 The PGV frequency was comparable between MC-HM and CT-HM (24% vs. 19%; P=0.43). However, the PGV pattern differed between the two groups. We observed striking enrichment of PGV in the DDR pathway of patients with CT-HM compared to those with MC-HM (70.6% vs. 38.5%; P=0.04). PGV in DDX41 (84.6% vs. 41.9%; P=0.002) and transcription factors (72.2% vs. 41.9%; P=0.43) were more prevalent in MC-HM compared to cases without PGV.
 PGVs in the DDR pathway were associated with enrichment of complex karyotype (44.4% vs. 19%; P=0.003) and monosomal karyotype (37.0% vs. 13.9%; P=0.002), in addition to lower frequency of somatic ASXL1 somatic mutations (3.8% vs. 20.5%; P=0.04) compared to patients without PGV. DDX41 PGVs were associated with somatic DDX41 mutations (66.7% vs. 0.9%; P <0.001), and these cases were less likely to harbor complex karyotype (0.0% vs. 19.0%; P=0.08) compared to those without PGV.
 Conclusion: The frequency of PGVs was higher in MN patients compared to other cancer patients and age-matched healthy controls. PGVs in the DDR pathway were highly prevalent in HM patients with prior exposure to cytotoxic therapies and were associated with poor-risk cytogenetic features. Our observation of a high frequency of PGVs in older MN patients warrants standardization of germline testing at diagnosis to guide optimal management of patients and their families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ee25454ca2e3d15eeec578ba25cb1a5124c45c" target='_blank'>
              Pathogenic Germline Variants in the DNA Damage Repair Pathway Are Enriched in Patients with Hematologic Malignancies with Prior Exposure to Cytotoxic Therapies Compared to Patients with Multiple Cancers without Prior Exposure
              </a>
            </td>
          <td>
            D. Hiwase, Zoe Price, C. Kok, M. Kutyna, Deepak Singhal, N. Shanmuganathan, Lesley Rawlings, David M. Ross, David T Yeung, Anna L Brown, C. Hahn, Lucy Godley, C. Saygin, Hamish S Scott
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is characterized by recurrent somatic mutations in epigenetic regulators, which stratify patients into clinically significant subgroups with distinct prognoses and treatment responses. However, the cell type-specific epigenetic landscape of RCC—broadly and in the context of these mutations—is incompletely understood. To investigate these open questions, we integrated single nucleus ATAC sequencing data from RCC tumors across four independent cohorts. In clear cell RCC tumors, we identified four shared malignant epigenetic programs related to angiogenesis, proximal tubule-like features, interferon (IFN) signaling, and one that lacked distinct genomic regions with increased accessibility. Among the mutated epigenetic regulators, BAP1 mutation exhibited the most significant impact on chromatin accessibility in tumor cells, and the associated epigenetic changes were linked to IFN response. We identify multiple potential sources of elevated IFN signaling in these lesions, such as increased immune infiltration and increased accessibility and expression of an IFN-associated ERV, ERV3-16A3_LTR. We find that the expression of ERV3-16A3_LTR may itself be a negative prognostic biomarker in ccRCC. Our findings highlight the convergence of malignant epigenetic programs across ccRCC tumors and suggest that BAP1 loss, potentially through ERV3-16A3_LTR dysregulation, is associated with an IFN response-high epigenetic program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f60a362700f5a511019adc2819b352bb15feca9" target='_blank'>
              Single-cell epigenetic profiling reveals an interferon response-high program associated with BAP1 deficiency in kidney cancer
              </a>
            </td>
          <td>
            Sabrina Y. Camp, M. X. He, Michael S. Cuoco, E. Saad, Erica Pimenta, Kevin Meli, Z. Bakouny, C. Labaki, Breanna M. Titchen, Yun Jee Kang, Jack Horst, Rachel Trowbridge, Erin Shannon, K. Helvie, A. Thorner, S. Vigneau, Angie Mayorga, Jahnavi Kodali, Hannah Lachmayr, Meredith Bemus, Jihye Park, T. Choueiri, Kevin Bi, E. V. Van Allen
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="


 Pediatric high-grade gliomas (pHGG) harboring H3.3 K27M/G34R mutations present formidable challenges in treatment due to their aggressiveness and resistance to standard chemotherapy. Our preliminary investigations have shown promising results, indicating that combining 2-deoxyglucose (2DG), a glycolytic inhibitor, with DNA-damaging agents synergistically inhibits pHGG growth. However, prolonged exposure to 2DG induces resistance, necessitating the exploration of alternative therapies. This study aims to investigate the potential of panobinostat, a histone deacetylase inhibitor, in sensitizing 2DG-resistant pHGG with H3.3 K27M/G34R mutations to apoptosis.



 Transcriptomic analysis via Illumina NovaSeq 6000 and metabolomic profiling employing Liquid Chromatography-High-Resolution Mass Spectrometry was used to delineate the molecular landscape of naive and 2DG-resistant pHGG cell lines. Functional assays, including the Agilent Seahorse Mito Stress Test and flow cytometry, assessed bioenergetics and mitochondrial dynamics. Intracellular NAD+ and NADH levels were quantified utilizing NAD+/NADH Glo assays, while apoptotic markers and metabolic stress were measured using the Annexin V-FITC Conjugates for Apoptosis Detection kit and western blot.



 Comparative analysis revealed increased transketolase activity in 2DG-resistant cells, augmenting the interaction between the pentose phosphate pathway and glycolysis, resulting in elevated levels of NADPH and ATP. Additionally, these resistant cells displayed enhanced reliance on mitochondrial energy metabolism, as evidenced by substantial increases in mitochondrial membrane potential and mass, coupled with elevated NAD+ levels. Subsequent treatment with panobinostat induced a significant decrease in NAD+ biosynthesis and ATP generation. Furthermore, this intervention triggered the upregulation of pro-apoptotic BCL-2 family proteins, particularly BAX and BAK, concomitant with a notable reduction in mitochondrial membrane potential and the accumulation of reactive oxygen species. Consequently, heightened caspase-3 activity ensued, leading to significantly elevated cell death rates in 2DG-resistant gliomas.



 Our findings highlight panobinostat as a promising therapy to resensitize 2DG-resistant pHGG cells to apoptosis, offering hope for treating these aggressive tumors with H3.3 K27M/G34R mutations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/266005b1b2b85197559cc753fd3ff5d1c129d197" target='_blank'>
              TMET-22. OVERCOMING RESISTANCE: PANOBINOSTAT SENSITIZES 2DG-RESISTANT PEDIATRIC HIGH-GRADE GLIOMAS WITH H3.3 K27M/G34R MUTATIONS TO APOPTOSIS
              </a>
            </td>
          <td>
            Tracy A Miller, Esther Jane, Taylor A. Gatesman, Matthew J Halbert, Chiam Sneiderman, Itay Raphael, Steven Mullett, Stacy L. Gelhaus, S. Agnihotri, Richard Steinman, Daniel R Premkumar, Ian F Pollack
          </td>
          <td>2024-11-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="

 Although genetic alterations drive carcinogenesis (PMID: 29439951), they alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy (PMID: 32807900). However, the mechanisms underpinning this heterogeneity remain under-explored. Aberrant enhancer activity is a hallmark of many cancers, including KMT2A::AFF1 acute lymphoblastic leukemia (ALL; PMID: 37626123), an aggressive leukemia subtype with a poor prognosis (PMID: 32376390) and a nearly mutationally silent genetic landscape (PMID: 25730765). Despite the low mutational burden, KMT2A-rearranged leukemias exhibit substantial heterogeneity between individuals, providing an ideal model to study transcriptional heterogeneity without the confounding effect of numerous cooperating mutations.
 To validate the extent of transcriptional heterogeneity we first compared two patient-derived KMT2A::AFF1 cell lines (RS4;11 and SEM), identifying 2,361 differentially expressed genes via RNA sequencing. Next, we compared enhancer activity using Assay for Transposase-Accessible Chromatin using sequencing (ATAC-seq), identifying 3,342 enhancers with significantly altered activity in these KMT2A::AFF1 ALL cell lines. CRISPR-Cas9-based deletion of representative enhancers confirmed cell line-specific activity on their predicted target genes. Using supervised machine learning, we developed a predictive model that identified the KMT2A-complex as a key factor in enhancer activity differences between cell lines. Our previous work showed that KMT2A::AFF1 binds enhancers and regulates them by recruiting a complex that facilitates transcriptional elongation (PMID: 37626123). In line with this, loss of KMT2A::AFF1 or components of the transcription elongation complex significantly diminishes enhancer-promoter interactions and features of active enhancers, implying that it is essential for maintaining enhancer activity (PMID: 37626123).
 To confirm differential enhancer usage in patients, we used single-cell multiomics (RNA+ATAC) on KMT2A::AFF1 patient blasts, identifying substantial transcriptional heterogeneity and 3,162 differentially active putative enhancers. As ATAC-seq identifies genomic boundary elements in addition to enhancers and promoters, we performed detailed epigenetic profiling of nine KMT2A::AFF1 patient samples (four infants and five children) with a low-cell number optimized chromatin-immunoprecipitation protocol. Employing a deep-learning-based strategy, we reduced intrinsic sample-to-sample noise and dramatically improved the stringency of differential enhancer identification. This approach revealed 2,647 differentially active enhancers, with 61 displaying activity unique to a single patient.
 We examined whether any unique enhancers were located near prognostically relevant genes and identified putative patient-specific enhancers near MEIS1 and RUNX2. Single-cell expression data revealed elevated MEIS1 and RUNX2 expression in the patient with these unique enhancers. Using Micro-Capture-C (MCC), we confirmed that these enhancers contact the MEIS1 and RUNX2 promoters, implicating these patient specific enhancers in the overexpression of these genes.
 Overall, our data suggests that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and appears to be driven by differential KMT2A-complex binding. We identify differential enhancer activity at key oncogenes such as MEIS1 and RUNX2. Analysis of published microarray data shows that overexpression of either gene is associated with significantly reduced overall survival (PMIDs: 20699438, 19880498, 17312329). Taken together, this indicates that enhancer heterogeneity may significantly contribute to the phenotypic diversity observed between patients in ALL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f0c9bbed1810a8832229fa6449924fbeebabcc" target='_blank'>
              Differential Gene Expression in KMT2A::AFF1 Leukemia Is Driven By Enhancer Heterogeneity
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, N. Elliott, T. Jackson, Joe R Harman, H. Geng, Irene Roberts, Nicholas Crump, James Oliver Davies, A. Roy, T. A. Milne
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration has been implicated in transforming HPV infection into cancer. To resolve genome dysregulation associated with HPV integration, we performed Oxford Nanopore long-read sequencing on 72 cervical cancer genomes from an Ugandan dataset that was previously characterized using short-read sequencing. We found recurrent structural rearrangement patterns at HPV integration events, which we categorized as: del(etion)-like, dup(lication)-like, translocation, multibreakpoint, or repeat region integrations. Integrations involving amplified HPV-human concatemers, particularly multibreakpoint events, frequently harbored heterogeneous forms and copy numbers of the viral genome. Transcriptionally active integrants were characterized by unmethylated regions in both the viral and human genomes downstream from the viral transcription start site, resulting in HPV-human fusion transcripts. In contrast, integrants without evidence of expression lacked consistent methylation patterns. Furthermore, whereas transcriptional dysregulation was limited to genes within 200 kilobases of an HPV integrant, dysregulation of the human epigenome in the form of allelic differentially methylated regions affected megabase expanses of the genome, irrespective of the integrant's transcriptional status. By elucidating the structural, epigenetic, and allele-specific impacts of HPV integration, we provide insight into the role of integrated HPV in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8457c68dc264c60760742af32b0cda2346039" target='_blank'>
              Rearrangements of viral and human genomes at human papillomavirus integration events and their allele-specific impacts on cancer genome regulation.
              </a>
            </td>
          <td>
            Vanessa L. Porter, Michelle Ng, Kieran O’Neill, Signe MacLennan, R. Corbett, Luka Culibrk, Zeid Hamadeh, Marissa Iden, Rachel Schmidt, Shirng-Wern Tsaih, C. Nakisige, Martin Origa, Jackson Orem, Glenn Chang, Jeremy Fan, K. Nip, Vahid Akbari, Simon K. Chan, James Hopkins, Richard A. Moore, E. Chuah, K. Mungall, A. Mungall, Inanҫ Birol, S. Jones, Janet S. Rader, M. Marra
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Metformin administration has recently emerged as a candidate strategy for the prevention of head and neck squamous cell carcinoma (HNSCC). However, the intricate relationship between genetic alterations in HNSCC and metformin sensitivity is still poorly understood, which prevents the stratifications of patients harboring oral premalignant lesions that may benefit from the chemopreventive activity of metformin. In this study, we investigate the impact of prevalent mutations in HNSCC in response to metformin. Notably, we found that the expression of oncogenic HRAS mutants confers resistance to metformin in isogenic HNSCC cell systems and that HNSCC cells harboring endogenous HRAS mutations display limited sensitivity to metformin. Remarkably, we found that metformin fails to reduce activation of the mTOR pathway in HRAS oncogene expressing HNSCC cells in vitro and in vivo, correlating with reduced tumor suppressive activity. Mechanistically, we found that this process depends on the ability of HRAS to enhance glycolytic metabolism, thereby suppressing the requirement of oxidative phosphorylation to maintain the cellular energetic balance. Overall, our study revealed that HNSCC cells with oncogenic HRAS mutations exhibit diminished metformin sensitivity, thus shedding light on a potential mechanism of treatment resistance. This finding may also help explain the limited clinical responses to metformin in cancers with RAS mutations. Ultimately, our study underscores the importance of understanding the impact of the genetic landscape in tailoring precision cancer preventive approaches in the context of HNSCC and other cancers that are characterized by the presence of a defined premalignant state and, therefore, amenable for cancer interception strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f5da85db8cc680799b131f80b79d57f7c052f9b" target='_blank'>
              Oncogenic H-RAS induces metformin resistance in head and neck cancer by promoting glycolytic metabolism.
              </a>
            </td>
          <td>
            Xingyu Wu, S. Adame-Garcia, Keiichi Koshizuka, Pham Thuy Tien Vo, Thomas S. Hoang, Kuniaki Sato, Hiroki Izumi, Yusuke Goto, M. Allevato, Kris C Wood, Scott M. Lippman, J. S. Gutkind
          </td>
          <td>2024-10-28</td>
          <td>Cancer prevention research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="

 Introduction: Molecular profiling of mature B-cell lymphomas and leukemias informs diagnosis, prognosis, and therapy. For example, WHO and NCCN guidelines designate specific genomic alterations, such as TP53 mutations in chronic lymphocytic leukemia (CLL) and mantle cell lymphoma (MCL), as essential for risk stratification. Gene rearrangements, including those involving IRF4, CCND1, BCL2, BCL6, and MYC, also have diagnostic and prognostic significance for B-cell neoplasms. We analyzed the genomic landscape of a large cohort of mature B-cell lymphomas/leukemias tested on the FoundationOne Heme (F1H) assay platform, highlighting the clinical utility of CGP in patient management.
 Methods: F1H is a hybrid-capture next-generation sequencing (NGS) assay that analyzes all classes of genomic alterations in DNA (exons from 406 genes, select introns from 31 genes) and rearrangements in RNA (265 genes) from fresh blood, bone marrow, or formalin-fixed paraffin-embedded (FFPE) tumor samples. Samples with diagnoses of CLL, diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), or MCL undergoing F1H testing between 12/2013 and 9/2023 were included in the study. Diagnoses were assigned by board-certified hematopathologists with molecular expertise who reviewed the submitted test requisition forms, hematopathology reports, and accompanying H&E or Wright Giemsa-stained slides.
 Results: In all, 3,692 unique samples were identified, including 1,188 CLL, 1,374 DLBCL, 624 FL, and 506 MCL cases. The median age was 61 years (range, 3 to >89) and 63% of the patients were male. Sixty-four percent were submitted as FFPE tissue, 26% as peripheral blood, and 10% as bone marrow aspirates. Most samples harbored pathogenic genomic alterations with diagnostic, prognostic, or therapeutic importance. Base substitution or indel variants were detected in TP53 (28% of cases), KMT2D (25%), CREBBP (14%), EZH2 (8%), MYD88 (8%), TNFRSF14 (8%), ATM (8%), NOTCH1 (7%), ARID1A (6%), B2M (6%), TNFAIP3 (6%), PIM1 (4%), CD79B (3%), BTK (2%, 97% of which were C481X ibrutinib resistance mutations in cases of CLL), MEF2B (2%), BCL2 (1%), and PLCG2 (1%). Amplifications of MYC and CD274 were both detected in 1% of cases. Overall, 57% of cases harbored a gene rearrangement, including IGH::BCL2 (24%), IGH::CCND1 (12%), IGH::BCL6 (4%), and IGH::MYC (3%), with 13% of rearrangements detectable by RNA sequencing only. One hundred twenty and 63 unique rearrangement partners were detected for IGH and BCL6, respectively. Overall, 75% of FLs harbored a canonical IGH::BCL2 rearrangement and 86% of MCLs harbored a canonical IGH::CCND1 rearrangement.
 Conclusions: This analysis of 3,692 samples demonstrated that the F1H assay platform reliably detects a broad landscape of genomic alterations across a range of mature B-cell lymphoma/leukemia subtypes. By detecting all classes of genomic alterations in a single sequencing reaction, F1H provides an important advantage over single-gene and small-panel molecular tests in an era when the diagnosis, prognosis, and treatment of hematological malignancies increasingly rely on assessing the presence, as well as the absence, of numerous genomic alterations.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8628b208ebf56f9885e117d970847f75d2223e" target='_blank'>
              Comprehensive Genomic Profiling of Mature B-Cell Lymphomas/Leukemias: Foundation One Heme Reveals Actionable Alterations and Biomarkers
              </a>
            </td>
          <td>
            N. Danziger, Chelsea Marcus, Jessica Lee, Douglas Lin, Richard Huang, Caleb Ho, Douglas Mata
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="

 Philadelphia chromosome (Ph) a hallmark of chronic myeloid leukemia (CML), is a result of reciprocal translocation t(9;22) generating aberrant persistently activate oncogenic tyrosine- kinase (BCR/ABL1) facilitating leukemogenesis. It was demonstrated that simultaneous breakages of two different chromosomes within BCR and ABL1 loci, serve as an oncogenic initiation of Ph+ leukemia. Thus, recurrency of Ph and other translocations in hematopoietic stem cells indicates that chromosomal instability (CIN) is leading pathogenetics of leukemogenesis.
 Surprisingly, despite established diagnostic methods and knowledge on CML, use of Optical Genome Mapping (OGM) identified novel pathogenetic landscape of chromosomal aberrations and generation of BCR/ABL1 p190 fusion gene without formation of Ph chromosome. We found that dominant characteristics of CIN in these leukemia cells is increase of clustered aberrations at distinct breakpoints throughout a genome with proximity to microhomology-enriched segments. We detected significant accumulation of double-strand breaks (DSBs) at near-overlapping origins of replications. OGM and chromosomal analyses strongly correlate with induced distinct expression profile and activation of microhomology-mediated break induce repair (MMBIR). MMBIR utilizes microhomology for DSB repair, producing templated insertions that promote complex genomic rearrangements detected in cancer including chromosomal illegitimate recombination, integrations and fusions. Despite growing evidence on accumulation of de novo templated insertions in several cancer types, the underlying molecular and genomic mechanisms, impact on carcinogenesis and heterogeneity, particularly in hematopoietic cells are poorly understood.
 Based on OGM, omics data mining and NGS, we developed sophisticated screening panels and molecular analyses to study BCR, ABL1, corresponding breakpoints loci, and near-overlapping microhomology-enriched elements, to identify and characterize this novel atypical p190 fusion gene. Our results indicate that in some CML patients', with unique CIN signature, induced mechanism of de novo templated insertions generating atypical p190 fusion gene instead of Ph chromosome formation.
 It's well established that different BCR/ABL1 transcripts correspond to distinct CML clinical phenotype. Furthermore, chronic, accelerated and blast phases of CML progression evaluated and measured by expression rate of each Ph+ BCR/ABL1 transcript variant, resulting in prediction of therapy response, and clinical outcomes.
 However, in few CML patients, standard assessments succeeded to detect fusion gene induced by templated insertion and monitor atypical transcripts. Thus, generally, standard tests were insufficient to assess expression rate of atypical p190 transcript initially and in MRD. Importantly, precise definition of the disease is challenging in patients with minimal CML. For example, preliminary screen detected templated insertions with atypical p190 BCR/ABL1 fusion genes in 6 patients. 3 patients were JAK2 V617F positive, while 2 subsequently treated for Essential Thrombocytosis. Third patient diagnosed and treated for p210 Ph+CML. In contrast, spontaneous recovery observed in other 3 patients, with allelic frequency of atypical p190 transcript lower than 0.035%. Thus, detection of minor clones with templated insertions, characterization of related abnormalities provide critical insights during early detection on clinical significance of atypical p190 transcript, necessity of further molecular testing and treatment evaluation. Hence, we developed robust molecular assay to measure atypical p190 transcripts to assist diagnosis and monitoring MRD. In contrast, ongoing study points on importance of MRD monitoring, detection CIN signatures, accumulation of templated insertions promoting transformation.
 In conclusion, our study indicates that distinct CIN signature orchestrates unique DNA repair landscape. Lack of precise molecular assessment, and genetic identification of seemingly similar oncogenes originated from atypical fusion gene, or Ph chromosome can be providing misleading diagnostics. Furthermore, our results suggest that increased heterogeneity observed among CML patients, reflect on clonal ability to repair DNA damage, preserve genome stability and function, affecting carcinogenesis.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8037e7571b5a68dcda78fbeaec9945cd834494" target='_blank'>
              Templated Insertion Mediates Generation of p190 BCR/ABL Facilitating CML Heterogeneity Uniquely Detected By Optical Genome Mapping
              </a>
            </td>
          <td>
            C. Ganzel, Y. Ofran, Bayan Alyan, Elina D Shnayder, Shira Vanunu, Zakaria M Sous, Victoria Vinokurov, Veronika Butin Israeli
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="

 MYB gene encodes a transcription factor that acts as a master regulator of cells self-renewal, proliferation and differentiation processes. MYB is frequently upregulated or aberrantly activated in a variety of solid cancers (e.g., adenoid cystic carcinoma [ACC], colorectal cancer [CRC], and liquid tumors). In leukemias, high MYB RNA levels are especially frequent (90%) in acute myeloid leukemia (AML). In addition, in T-cell acute lymphoblastic leukemia (T-ALL), MYB is overactivated in 20-30% of patient population due to well characterized genetic lesions (e.g., chromosomal translocations [c-MYB/AHI1; TCRB-MYB 8%], gene duplication [19%]), or somatic mutations. Importantly, a genome-wide CRISPR-Cas9 screen identified MYB as an essential gene for leukemia/lymphoma cell lines viability. Altogether, data strongly suggest MYB as a promising therapeutic target to treat blood cancers.
 RGT-61159 is a potent, selective oral small molecule inhibitor of the MYB oncogene via the inclusions of a cryptic exon into MYB RNA transcripts, resulting in the activation of the nonsense-mediated decay pathway promoting MYB mRNA depletion and sequentially MYB protein degradation. RGT-61159 Phase 1 study in adults with relapsed/ refractory ACC or CRC has been initiated (NCT 06462183)
 Here, we report RGT-61159 robust MYB RNA & protein knockdown and cell-killing activity across a broad range of different AML cell lines with different oncogenic abnormalities. Such a data set supports that RGT-61159 killing activity of leukemic cells is driven by MYB signaling inhibition and confirms MYB dependency across a broad AML patient population. In addition, we report proteomic and genomic analyses of different AML cell lines treated with RGT-61159 that provided further lights on key oncogenes (e.g., BCL2, NOTCH, MYC) directly regulated by MYB in a cell genetic context dependent.
 To further evaluate RGT-61159's potential to treat leukemia, RGT-61159 single agent was profiled in a battery of murine models of AML harboring the most common genetic lesions (e.g., AML-1-ETO, MLL-fusion and NPM1 mutation). RGT-61159 showed highly significant anti-tumor activity across the different AML tumor models tested along with robust MYB RNA depletion in tumor cells (up to 90% tumor growth inhibition [TGI]) in a dose dependent manner and at tolerated doses.
 In summary, MYB oncogene represents a therapeutic target of relevance to AML and other hematological malignancies. RGT-61159 is a potent and selective oral small molecule inhibitor of MYB RNA and protein and an attractive therapeutic avenue to treat cancers overexpressing the oncogenic MYB protein.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4463953db24fb59e6d5affeb556a6ee7827df3" target='_blank'>
              RGT-61159, Best-in-Class Small Molecule Inhibitor of MYB Via Selective RNA Splicing Alteration, Showed Robust Anti-Tumor Activity across AML Tumor Models Harboring Common Genetic Lesions
              </a>
            </td>
          <td>
            Norman Lu, Patricia Soulard, Ibrahim Kay, Kai Li, Xiubin Gu, Heather Sadlish, Jay Lee, Mariam Elhawary, Michael Rotelli, Sam Hasson, Thomas Wynn, Chris Yates, Zhiping Weng, Simon Xi, Travis Wager
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="

 Introduction
 Multiple myeloma (MM) is a complex hematological disease with high heterogeneity resulting from abnormal proliferation of plasma cells in bone marrow. The progression of the disease from Monoclonal Gammopathy of Undetermined Significance (MGUS), Smoldering Multiple Myeloma (SMM), to MM involves multiple drivers and, moderately, by dynamic interactions of cis-regulatory elements in premalignant/malignant cells. Using single-nuclei multiome technology, the same-cell readouts of nucleus RNA sequencing (snRNA-seq) and Assay for Transposase-Accessible Chromatin sequencing (snATAC-seq) provide a less biased measurement of gene expressions and chromatin accessible regions. Profiling and analyzing both omics data at different stages of MM may reveal essential molecular signatures and regulatory elements for disease progression. These insights could benefit MM patients as novel targets for early therapeutic intervention at different stages of progression.
 Methods
 Bone marrow aspirates were collected from 33 individuals: Healthy donors (HD; n=3), patients with MGUS (n=2), SMM (n=5), MM (n=21), and Plasma Cell Leukemia (PCL; n=2). Viable mononuclear cells underwent CD138+ sorting and nuclei isolation prior to library preparation. Libraries were prepared with the Chromium Multiome kit (10X Genomics) and sequenced on NovaSeq S4 flow cells at UAMS. The clonal V(D)J arrangements of each sample and expression level from HD samples were used to distinguish malignant (MM) from normal plasma cells (PC). Data of the paired modalities were integrated using Signac with the weighted nearest neighbor (WNN) analysis. All inter- and intra-tumor clusters were evaluated for markers of molecular subtype, differential gene expression, and differentially accessible regions. A variance component analysis of proximal, distal chromatin accessibility, and gene expression, was estimated by the average information restricted likelihood estimation.
 Results
 We performed the single nuclei multiomic sequencing to generate multi-stage chromatin accessibility and transcriptional profiles of plasma cell neoplasms comprising 439,134 cells across 33 samples. The malignant plasma cells for each sample were extracted by the unique clonotypic signature of the immunoglobulin gene rearrangement. Using our previous microarray data, we detected the overexpression of molecular signatures for most cells in each molecular subgroup, confirming the molecular heterogeneity in MM. The multiomics integration generated less distinct clusters between MGUS and SMM, indicating the gradual complex alterations of gene expression and chromatin accessibility in malignant transformation. We then applied a variance component model to quantify the relative contribution of accessibility to transcriptional variance globally and corrected for inter-sample and disease stage effects. The results showed that more than 70% of gene expression variance was attributed to the accessibility of promoter and enhancer. However, there were 76 genes for which >60% of the gene expression variance could be explained by the disease stage covariance. Gene enrichment analysis revealed enrichment in the cell developmental process and differentiation, suggesting the increase of disease aggressiveness during the progression. Also, we have identified domains of regulatory chromatin (DORCs), the large clusters of regions of significant peak-gene links overlapped with super-enhancers that might play critical regulatory roles in disease progression.
 Conclusion
 In this large multiomics study of CD138+ cells, we demonstrate the cis-regulatory associations between chromatin accessibility and target genes from patients with early-stage disease to highly aggressive form. Throughout the disease progression, we found that transcriptional heterogeneity is predominantly associated with the variation of chromatin accessibility. Also, we uncover unique linked peak-gene pairs for each disease stage that can further refine our understanding of cis regulation patterns in multiple myeloma. Collectively, our joint single nuclei profiling reveals additional insights into cis-regulatory interactions and investigates novel regulatory domains involved in disease progression.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feeb157a03520da1aab6c61db2138fa2b1ac1698" target='_blank'>
              Single Nuclei Multiomic Profiling of Transcriptional and Chromatin Accessibility of Tumor Cells Underlines Extensive Cis-Regulatory Interaction during Multiple Myeloma Progression
              </a>
            </td>
          <td>
            Visanu Wanchai, Hongwei Xu, C. Ashby, F. van Rhee, Guido J Tricot, John D. Shaughnessy Jr., Fenghuang Zhan
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Simple Summary Menin, encoded by the MEN1 gene, scaffolds many interacting proteins through which it regulates important gene expression critical for cellular growth and differentiation. Menin is also critical in regulating genes that drive cancer growth in acute myeloid leukemia (AML), and more recently, targeting menin’s interaction with KMT2A has emerged as a promising therapeutic strategy for treating AML, especially with KMT2A rearrangement and NPM1 mutations. The menin inhibitors currently in development demonstrate promising early efficacy and may offer a more targeted approach than traditional chemotherapy and possibly better outcomes. This review will discuss the evolution of menin as a target in AML therapeutics, the clinical development of various menin inhibitor and outcome data from the latest clinical trials, and the understanding of its resistance mechanisms and strategies to improve outcomes and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9e1716dab1ac717d8a6f6135fe51085e7caf357" target='_blank'>
              Targeting Menin in Acute Myeloid Leukemia: Therapeutic Advances and Future Directions
              </a>
            </td>
          <td>
            Sandhya Dhiman, Vikramjit Dhillon, S. Balasubramanian
          </td>
          <td>2024-11-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Introduction: The development of resistance to Imatinib (IMA) can occur over months or years, this constitutes a major problem for us due to economic limitations and the availability of 2° generation tyrosine kinase inhibitors (TKIs) in our reference centers. There are various mechanisms that lead to resistance to TKIs, especially because of specific mutations in the ABL kinase domain (KD) that impede the binding of the drug. The insertion of a 35 nucleotide between exon 8 and 9 induce a splicing product and these a truncated ABL protein that lacks the last 653 residues of KD, this transcript increased kinase inhibitors resistance sample in some chronic myeloid leukemia (CML) patients and in some cases, they respond to an increased dose of IMA, although in most cases molecular response (MR) is not achieved. However, the information of clinical value in literature is scarce. Herein we inform 20 cases with such genetic abnormality.
 Methods: A retrospective, multicenter study was conducted in 2 medical centers in Mexico City. A total of 150 patients diagnosed with CML between 2003 and 2024 were included. From these 150 patients, those who did not achieve a MR to IMA or who experienced molecular relapse were identified. Among these, 20 patients were identified with the BCR-ABL mutation with a 35 nucleotide insertion (BCR-ABLIns35bp) between ABL KD exons 8 and 9. Inclusion criteria: patients with a confirmed diagnosis of CML, availability of complete clinical and laboratory data, identification of the mutation. Exclusion criterion: incomplete data. Demographic, clinical, and laboratory data were collected from the patients' electronic medical records, including age at diagnosis, gender, date of CML diagnosis, date of identification of the mutation, treatments received, including increased doses of IMA and other TKIs such as Bosutinib, Nilotinib, Asciminib, and Dasatinib, MR prior to the increase in IMA dose or treatment change, and loss of MR. All patients started with a standard dose of IMA (400 mg), in cases of no MR, the dose was increased (600-800 mg), depending on patient tolerance. In cases of no MR to IMA, mutations were requested, and if there was no MR to the maximum dose of IMA, other TKIs were administered, depending on availability. Patients were regularly followed up with clinical visits every 3-6 months and laboratory tests to evaluate treatment response.
 Results: 20 patients (13.3%) diagnosed with CML were included, 18 adults (90%) and 2 children (10%) were identified with the BCR-ABLIns35bp, all of whom initially received standard doses of IMA. The median age at diagnosis was 33 years (4-61 years), with an equal distribution of 10 males (50%) and 10 females (50%). Among the 20 patients with the mutation: 11 patients (55%) achieved a major molecular response (MMR) but lost this response after a median time of 10 years (3-10 years) on IMA, 9 patients (45%) did not achieve an MMR at any point. Upon increasing the dose of IMA (up to 800 mg for those who tolerated it), 8 patients (40%) showed a reduction in BCR-ABL copies, but only 3 achieved an MMR. The remaining 12 patients were switched to other TKIs, all showing a reduction in BCR-ABL copies, with 5 (41%) achieving an MMR. All patients initially started on standard doses of IMA (minimum dose 400 mg, maximum dose 800 mg). 30% of patients were subsequently treated with Bosutinib, 20% Nilotinib, 5% Dasatinib, and 25% Asciminib. Up to 4 lines of treatment were required for some patients who did not achieve an MMR. No patients progressed to the blast phase during the study period.
 Conclusions: We conclude that the BCR-ABLIns35bp invariably leads to resistance to IMA and even second-generation TKIs. Due to economic limitations and a lack of resources, sometimes the only intervention we can perform is increasing the dose of IMA. While this doesn't achieve a MMR, it significantly reduces BCR-ABL copies, as do newer drugs like Nilotinib, Bosutinib, or Dasatinib. The recent use of Asciminib shows promise, but longer follow-up is needed; however, we have achieved good results so far. It is important to mention that, despite not always achieving an MR over the years, some patients with a CML diagnosis of up to 22 years have remained asymptomatic and with no progression to blast phase. This raises the question of how deep the MR truly needs to be to impact mortality and how treatment optimization can be achieved given economic limitations. The access to second-generation TKIs in Mexico is limited.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc9e12616273372a46ec28540b7ccf904ba34297" target='_blank'>
              35-Nucleotide Insertion between Exon 8 and Exon 9 Splicing Transcript, Clinical Significance in 20 Adult and Pediatric Chronic Myeloid Leukemia (CML) Ph+ Patients in Mexico City
              </a>
            </td>
          <td>
            Rafael Hurtado Monroy, Estefania Frances, Angelica S. Rios del rio, N. López-Santiago, Janet Rojas Aguilar
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="

 Acute lymphoblastic leukemia in infants (iALL) is a high-risk subtype of childhood leukemia with poor survival outcomes despite intensive therapies. Rearrangement of KMT2A (KMT2A-r) occurs in 70% of cases and is associated with chemotherapy resistance, early relapse, and rapid leukemic progression, all of which contribute to a survival rate of about 40%. The most common KMT2A-r in iALL is KMT2A::AFF1 (MLL::AF4), derived from t(4;11)(q21;23). This KMT2A-r generates a driver fusion oncogene which leads to epigenetic dysregulation of target gene transcription. Infant ALL expressing KMT2A::AFF1 is also a unique cancer in that other somatic mutations are uncommon and there are no known genetic risk factors. This leukemia's cell of origin is proposed to be a very early hematopoietic precursor, with transcriptomic studies of iALL blasts showing similarities to hematopoietic stem and progenitor cells (HSPCs), multipotent progenitors and early lymphoid progenitors (ELPs). Despite this knowledge, the factors leading to leukemogenesis remain poorly understood, with research into this devastating disease hampered by the lack of a representative model of iALL Attempts at creation of an iALL models have led to a variety of results including myeloid neoplasms, lymphomas, mixed phenotypic hyperplasia or leukemia, mature B cell neoplasias. Because of these failures, much remains unknown regarding how KMT2A::AFF1 transforms early hematopoiesis and how it alters the severity of the disease.
 To further understand the developmental state of the cell of origin and leukemogenesis of iALL, we have engineered a highly controlled human inducible pluripotent stem (iPS) cell model of KMT2A-r iALL. Using an established doxycycline (dox) regulated platform, we have generated multiple iALL driver transgene constructs, including an epitope tagged human KMT2A::AFF1 cassette and a human-mouse KMT2A::Aff1 cassette. This system targets the dox-regulated transgene and dox regulatory machinery to the endogenous AAVS1 locus into non-diseased human iPS cell lines using CRISPR mediated targeting via homology directed repair. The non-diseases human iPS cells allow for mimicking of the silent mutational landscape found in patient samples. Correct targeting of the KMT2A::Aff1 transgene was confirmed via real-time reverse transcriptase-polymerase chain reaction (RT-PCR). These cells tightly express the KMT2A::Aff1 transgene only in response to dox treatment in both undifferentiated iPS cells and after differentiation to hematopoietic cells.
 Through directed differentiation we have also produced functional HSPCs from iPS cells with multilineage differentiation capacity. Flow cytometry analysis demonstrated enrichment of CD45+CD34+ cells by day 10 of differentiation. Further flow cytometry analysis of colony forming unit assays revealed functional enrichment in lympho-myeloid (CD45RA+) differentiated cells, along with formation of monocyte (CD15+) and erythroid (CD235+) colonies. We also used the 10x Chromium system to perform single-cell RNA sequencing of the differentiated cells. The scRNAseq transcriptomic data was analyzed using Seurat in R. Transcriptomic data confirmed the cells' differentiation ability as they clustered into distinct populations representing erythrocytes, megakaryocytes, and monocytes.
 In conclusion, this model has allowed us to recapitulate hematopoietic ontogeny with the ability to control iALL induction at specific developmental stages. We are utilizing single cell genomics to investigate transcriptomic changes during hematopoietic differentiation and after expression of the KMT2A oncogenic driver. We will compare these results to published data sets of single cell RNA sequencing data from healthy fetal HSPCs and to our existing iALL and pediatric KMT2A-r ALL single cell sequencing data sets. Our iPS cell based iALL model system provides the opportunity to investigate a range of critical and outstanding questions of iALL disease initiation, progression, and treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b631981ea1355f736e4203294ab4947ce73e4273" target='_blank'>
              A Genomics Driven, Inducible Pluripotent Stem Cell Model of Infant Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Meagan Vacek, Jacqelyn Nemechek, I. Pushel, Bradley T. Thornton, Molly Leyda, Priyanka Prem Kumar, M. Farooqi, Erin Guest, Jay Vivian, John M Perry
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Gene editing with CRISPR-Cas9 technology revolutionizes modern medicine by enabling precise DNA modifications. Initially discovered as a bacterial defense mechanism, the Cas9 enzyme, guided by RNA, can target and cut specific DNA sites, allowing for gene editing. Applications include genetic engineering, functional studies, and potential treatments for genetic diseases like cancer. Notably, in 2023, CRISPR-Cas9 was approved for treating sickle cell anaemia with significant results, despite challenges like long-term security and high costs. CRISPR-Cas9 technology allows for correcting genetic defects, treating diseases, and improving agricultural crops. It can regulate gene transcription through the CRISPRi system, using an inactive Cas9 to interfere with gene expression without permanently altering DNA. This gene-editing tool shows promise in gene therapy, potentially curing diseases like HIV-1, sickle cell disease, and haemophilia B. However, challenges include off-target mutations and efficient delivery of CRISPR/Cas9. Precise target site selection and dosage control are crucial, with tools like CasOT helping identify and prevent unwanted mutations. CRISPR-Cas9 requires a PAM sequence to function, narrowing its targets in the genome but increasing specificity. Production of gRNA faces challenges due to mRNA processing, with alternatives like the artificial gene RGR showing promise. Efficient delivery methods are still needed, with current techniques involving DNA and RNA injection. Future applications include treating genetic diseases and agricultural improvements, with ongoing research essential for overcoming challenges and ensuring safety and accuracy. Global collaboration is vital for the ethical use of this technology.
Contribution to evidence-based healthcare: CRISPR therapy represents an exciting frontier in genetic medicine, allowing for precise gene editing and opening up new possibilities for treating incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0ce03e8f26830137fb7f68ada78482dfe3faf6" target='_blank'>
              CRISPR therapy: A revolutionary breakthrough in genetic medicine
              </a>
            </td>
          <td>
            Geneci Da Silva Barreto, Jair Brito do Nascimento, Karin Cristina Santos de Almeida, José Carlos Marcolino Neto, Samuel Lucas Ferreira Luz da Silva
          </td>
          <td>2024-12-16</td>
          <td>International Healthcare Review (online)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a genetic anemia caused by the production of an abnormal adult hemoglobin. The clinical severity is lessened by elevated fetal hemoglobin (HbF) production in adulthood. A promising therapy is the transplantation of autologous, hematopoietic stem/progenitor cells (HSPCs) treated with CRISPR/Cas9 to downregulate the HbF repressor BCL11A via generation of double strand breaks (DSBs) in the +58-kb erythroid-specific enhancer. Here, to further enhance HbF production without increasing the mutagenic load, we targeted both +58-kb and +55-kb BCL11A erythroid-specific enhancers using base editors. We systematically dissected DNA motifs recognized by the key transcriptional activators within these regions and identified the critical nucleotides required for activator binding. Multiplex base editing of these residues was efficient and safe and generated no or little DSBs and genomic rearrangements. We observed substantial HbF reactivation, exceeding the levels achieved using the CRISPR/Cas9 nuclease-based strategy, thus efficiently rescuing the sickling phenotype. Multiplex base editing was efficient in long-term repopulating HSPCs and resulted in potent HbF reactivation in vivo. In summary, these results show that multiplex base editing of BCL11A erythroid-specific enhancers is a safe and potent strategy for treating sickle cell disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949a30362791a74e8f68a2ef1865b96856027877" target='_blank'>
              Multiplex base editing of BCL11A regulatory elements to treat sickle cell disease
              </a>
            </td>
          <td>
            L. Fontana, P. Martinucci, S. Amistadi, T. Felix, M. Mombled, A. Tachtsidi, G. Corre, A. Chalumeau, G. Hardouin, J. Martin, O. Romano, M. Amendola, P. Antoniou, A. Miccio
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="STK11 (serine-threonine kinase 11), also known as LKB1 (liver kinase B1) is a highly conserved master kinase that regulates cellular metabolism and polarity through a complex signaling network involving AMPK and 12 other AMPK-related kinases. Germline mutations in LKB1 have been causatively linked to Peutz-Jeghers Syndrome (PJS), an autosomal dominant hereditary disease with high cancer susceptibility. The identification of inactivating somatic mutations in LKB1 in different types of cancer further supports its tumor suppressive role. Deleterious mutations in LKB1 are frequently observed in patients with epithelial ovarian cancer. However, its inconsistent effects on tumorigenesis and cancer progression suggest that its functional impact is genetic context-dependent, requiring cooperation with other oncogenic lesions. In this review, we summarize the pleiotropic functions of LKB1 and how its altered activity in cancer cells is linked to oncogenic proliferation and growth, metastasis, metabolic reprogramming, genomic instability, and immune modulation. We also review the current mechanistic understandings of this master kinase as well as therapeutic implications with particular focus on the effects of LKB1 deficiency in ovarian cancer pathogenesis. Lastly, we discuss whether LKB1 deficiency can be exploited as an Achilles heel in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b30c5a76e169ec470f9f33a782695d3af068588f" target='_blank'>
              The role of STK11/LKB1 in cancer biology: implications for ovarian tumorigenesis and progression
              </a>
            </td>
          <td>
            Jian Kang, Stefano Gallucci, Junqi Pan, Jonathan S. Oakhill, E. Sanij
          </td>
          <td>2024-10-31</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract. In the past, drug resistance in the anti-cancer process has been a major factor affecting the treatment of non-small cell lung cancer, which has greatly reduced the quality of life of patients, accompanied by greater suffering and lower survival rates. This article mainly discusses the possibility and practical feasibility of gene editing technology for curing some types of cancer under the current state of technology, and through analysis it finds that gene editing technology can greatly improve the negative effects of traditional drug treatment and chemotherapy, but the relatively immature technology and high cost often discourage many people. There are already experimented ways to cure cancer. For example, CRISPR-Cas9 gene editing technology is currently the easiest, most efficient, and most diverse way to edit genomes, and it is also one of the most attractive applications. It is hoped that it will be promoted and applied in the future while reducing costs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fd43ccfe06d5fc40eb9c4eb45122d3f40d30f1" target='_blank'>
              The treatment of lung cancer by gene editing technology and its comparison with traditional therapies
              </a>
            </td>
          <td>
            Linyue Lou
          </td>
          <td>2024-11-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Acute myeloid leukemia (AML) is the most common type of adult acute leukemia that progresses rapidly without treatment. Despite the emergence of new therapy options, most AML patients still face dire prognosis, with a 5-year survival rate of 37.5%. Venetoclax is a highly selective and effective B-cell lymphoma-2 (BCL-2) inhibitor that has demonstrated encouraging outcomes in AML patients, especially in those unfit for conventional chemotherapies. However, venetoclax-based regimens are not curative, and most patients will eventually develop resistance and relapse.
 As the rate-limiting enzyme in the nicotinamide adenine dinucleotide (NAD+) salvage pathway, which converts nicotinamide (NAM) into NAD+, nicotinamide phosphoribosyltransferase (NAMPT) plays crucial roles in cancer metabolism. NAM metabolism mediates venetoclax resistance in relapsed leukemic stem cells (LSCs), making NAMPT an attractive target for overcoming such resistance. We have previously introduced RPT1G, a novel hyperbolic NAMPT inhibitor that achieves significant efficacy in acute lymphoblastic leukemia while avoiding severe on-target toxicities that limited clinical development of previous NAMPT inhibitors. Here we further explore the use of RPT1G in combination strategies that can aid in the efforts to treat venetoclax resistance in AML patients.
 Cell health and apoptosis assessments revealed significant synergy between RPT1G and venetoclax in AML cells that are sensitive to venetoclax treatment, such as MV4;11, Molm 13, and ML-2, as well as venetoclax-resistant cells, including OCI-AML3 and THP1. Notably, synergy with venetoclax was also observed in cells that are resistant to RPT1G monotherapy, including HL-60 and Kasumi-1 cells. Moreover, we developed a venetoclax-resistant MV4;11 cell line (MV4;11-VR) showing over 50-fold increase in IC50 to venetoclax and observed strong synergy between RPT1G and venetoclax in these cells. Further analyses revealed that RPT1G and venetoclax synergized to reduce total cellular NAD+ in MV4;11 cells. While RPT1G exhibits mostly additive or modestly synergistic effects with AML standard-of-care agents, cytarabine and azacitidine, the synergy between RPT1G and venetoclax was maintained in MV4;11 cells in the presence of either agent. Moreover, RPT1G strongly synergizes with olaparib, a poly(ADP-ribose) polymerase inhibitor, and olaparib further enhanced the synergy between RPT1G and venetoclax.
 In contrast, RPT1G and venetoclax showed mostly additive or antagonistic effects in cultured human peripheral mononuclear cells from healthy donors. As RPT1G exhibits a favorable safety profile in animal toxicology studies and none of the adverse hematological toxicities of venetoclax, the use of this combination strategy could lower the dose of venetoclax required to achieve efficacy against AML cells and increase the safety margin of venetoclax-based regimens.
 B-cell lymphoma-extra large (BCL-XL) and myeloid cell leukemia 1 (MCL-1) are targets of interest to overcome venetoclax resistance, however, there are concerns over elevated on-target toxicities. We observed significant synergies between RPT1G and navitoclax, an inhibitor to multiple BCL-2 family proteins, as well as RPT1G and the MCL-1 inhibitor AMG-176. Of note, navitoclax or AMG-176 further enhanced the synergy between RPT1G and venetoclax in MV4;11-VR cells.
 The efficacy of the RPT1G and venetoclax combination was further evaluated in a MV4;11-luc mouse xenograft model. At doses of RPT1G and venetoclax that exhibited modest effects as monotherapies, the combination of the two agents significantly reduced the leukemia burden in mice, while showing no observable toxicities.
 In summary, the first-in-class hyperbolic NAMPT inhibitor RPT1G exhibited strong synergy with agents targeting anti-apoptotic BCL-2 family proteins in AML cells, both alone and in the presence of other standard-of-care agents. Given the crucial roles of NAMPT and BCL-2 family proteins in the survival and treatment-resistance of LSCs, RPT1G is emerging as an ideal candidate to be included in novel combinatorial therapies to enhance treatment efficacy of BCL-2 family inhibitors and overcome therapy resistance, improving overall outcome in a wide range of patients with AML and potentially other hematological malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a090bc68773541e37d29fef363a325878b2daba6" target='_blank'>
              The Hyperbolic NAMPT Inhibitor RPT1G Synergizes with BCL-2 Family Inhibitors and Helps Overcome Venetoclax Resistance in Acute Myeloid Leukemia Cells
              </a>
            </td>
          <td>
            Gregory Crimmins, Min Wu, Francis Roushar, Claudia F. Ramos Aguilar, Mehdi Gharghabi, Banumathi Cole, R. Lapalombella, Dennise A. de Jesús-Díaz
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="

 Multiple myeloma (MM) relies on oncogenic transcription programs that build upon transcriptional networks found in normal plasma cells. IRF4 serves as the master transcription factor (TF) to establish plasma cell identity. In myeloma cells, transcription is amplified by the recruitment of additional TFs, most prominently MYC, to drive the outgrowth and survival of malignant plasma cells. Targeted interference with this plasma/myeloma cell intrinsic transcription program would mark a precision-medicine approach with unprecedented activity but remains elusive in the absence of effective IRF4- or MYC-directed compounds. We therefore employed large-scale proteogenomic screens in combination with high-throughput combinatorial drug profiling to identify novel targets required for oncogenic transcription in MM.
 Building upon our knowledge of central TFs in MM we first aimed to define essential regulators of both, IRF4 and MYC. To this end, we performed genome-wide CRISPR screens in three myeloma cell line models (H1112, KMS12PE, SKMM1) using endogenous IRF4 and MYC protein levels as a readout. Our screens identified 645 and 576 essential genes for IRF4 and MYC expression, respectively. An integrated analysis of the dual screens resulted in only 79 common genes driving protein expression of both TFs. This included established drug targets such as EP300 and the SWI/SNF chromatin remodeling complex (SMARCA4, SMARCB1, SMARD1), as well as plasma cell specific dependencies (IRF4, XBP1), validating our screening approach.
 Gene set enrichment analysis of common screening hits prioritized general regulators of RNA polymerase II-based transcription and translation initiation factors. However, these gene sets were strongly comprised by pan-essential genes, prompting us to filter our data for common essential genes. Surprisingly, this analysis ranked the mediator complex gene MED12 at the top of the list, followed by IRF4. MED12 is involved in transcriptional regulation as part of the CDK8 subcomplex, composed of MED12, MED13, CDK8 and CCNC. We identified MED13 and CCNC as individual hits in our MYC and IRF4 screen, respectively (CDK8 was not included in the sgRNA library). Moreover, cyclin C emerged as another myeloma specific dependency. To corroborate this finding, we generated an IRF4-BioID2 fusion construct to identify proteins in close proximity (approx. 40nm) to IRF4 via mass spectrometry. Indeed, we identified not only the SWI/SNF chromatin remodeling gene family and common mediator complex members, but also CDK8 suggesting the CDK8 subcomplex as potential target to interfere with the IRF4-MYC autoregulatory loop in MM.
 We next tested three different CDK8-targeting small molecule compounds that demonstrated single-agent activity in myeloma cell line models. To reinforce this finding, we conducted high-throughput drug screens in three myeloma cell lines with different genetic background. Using recently discovered inhibitors of SWI/SNF activity as backbone drugs for interfering with IRF4/MYC activity (Bolomsky et al., Cancer Cell, 2024), we screened against 2804 mechanistically annotated compounds. Enrichment analysis for top synergy hits revealed three common drug classes that scored consistently: MEK inhibitors, mTOR inhibitors, as well as all CDK8 inhibitors. Validation experiments in multiple cell lines confirmed synergistic activity between CDK8 inhibitors and other IRF4/MYC targeting compounds (e.g. IMiDs, SWI/SNF inhibitors) independent of the genetic background (e.g. RAS mutation status). These data suggest that MED12 and the CDK8 subcomplex participate in oncogenic transcription in MM, and selective small molecule inhibition of this subcomplex in combination with established agents represents a potential novel precision medicine strategy in MM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d7c0ac1789e30d00ee49298f10bd19f9d8191f" target='_blank'>
              Proteogenomic Screens Identify Plasma Cell Specific Vulnerabilities to Halt Oncogenic Transcription in Multiple Myeloma
              </a>
            </td>
          <td>
            Arnold Bolomsky, Smriti Kanangat, M. Ceribelli, Kristina Rinaldi, Lin Zhang, Elizabeth M. Hill, J. Muppidi, Craig J. Thomas, Ryan M. Young
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff2091b650266439914b8ccf512293f71ba17a8" target='_blank'>
              YTHDC1 is a therapeutic target for B-cell acute lymphoblastic leukemia by attenuating DNA damage response through the KMT2C-H3K4me1/me3 epigenetic axis.
              </a>
            </td>
          <td>
            Xinxin Li, Minhua Zheng, Shoubao Ma, Fengze Nie, Zhiqiang Yin, Yanan Liang, Xianchun Yan, Weihong Wen, Jianhua Yu, Yingmin Liang, Siyong Huang, Hua Han
          </td>
          <td>2024-11-05</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract We developed Del-read, an algorithm targeting medium-sized deletions (6–100 bp) in short-reads, which are challenging for current variant callers relying on alignment. Our focus was on Micro-Homolog mediated End Joining deletions (MMEJ-dels), prevalent in myeloid malignancies. MMEJ-dels follow a distinct pattern, occurring between two homologies, allowing us to generate a comprehensive list of MMEJ-dels in the exome. Using Del-read, we identified numerous novel germline and somatic MMEJ-dels in BEAT-AML and TCGA-breast datasets. Validation in 672 healthy individuals confirmed their presence. These novel MMEJ-dels were linked to genomic features associated with replication stress, like G-quadruplexes and minisatellite. Additionally, we observed a new category of MMEJ-dels with an imperfect-match at the flanking sequences of the homologies, suggesting a mechanism involving mispairing in homology alignment. We demonstrated robustness of the repair system despite CRISPR/Cas9-induced mismatches in the homologies. Further analysis of the canonical ASXL1 deletion revealed a diverse array of these imperfect-matches. This suggests a potentially more flexible and error-prone MMEJ repair system than previously understood. Our findings highlight Del-read's potential in uncovering previously undetected deletions and deepen our understanding of repair mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece74b68770d9549840bdc9ed51b1bf59c99a07" target='_blank'>
              Utilizing insights of DNA repair machinery to discover MMEJ deletions and novel mechanisms
              </a>
            </td>
          <td>
            Aditee Kadam, Shay Shilo, Hadas Naor, Alexander Wainstein, Yardena Brilon, T. Feldman, Mark Minden, N. Kaushansky, N. Chapal-Ilani, L. Shlush
          </td>
          <td>2024-11-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Background: Targeted therapies (e.g., ibrutinib) have markedly improved chronic lymphocytic leukemia (CLL) management; however, ~20% of patients experience disease relapse, suggesting the inadequate depth and durability of these front-line strategies. Moreover, immunotherapeutic success in CLL has been stifled by its pro-tumor microenvironment milieu and low mutational burden, cultivating poor antigenicity and limited ability to generate anti-tumor immunity through adaptive immune cell engagement. Previously, we have demonstrated how a three-carbon-linker spirocyclic dimer (SpiD3) promotes futile activation of the unfolded protein response (UPR) in CLL cells through immense misfolded-protein mimicry, culminating in insurmountable ER stress and programmed CLL cell death. Method: Herein, we used flow cytometry and cell-based assays to capture the kinetics and magnitude of SpiD3-induced damage-associated molecular patterns (DAMPs) in CLL cell lines and primary samples. Result: SpiD3 treatment, in vitro and in vivo, demonstrated the capacity to propagate immunogenic cell death through emissions of classically immunogenic DAMPs (CALR, ATP, HMGB1) and establish a chemotactic gradient for bone marrow-derived dendritic cells. Conclusions: Thus, this study supports future investigation into the relationship between novel therapeutics, manners of cancer cell death, and their contributions to adaptive immune cell engagement as a means for improving anti-cancer therapy in CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6bb6049f43b4446b0fae7febaccea77debb8ea" target='_blank'>
              Immunogenic Cell Death Traits Emitted from Chronic Lymphocytic Leukemia Cells Following Treatment with a Novel Anti-Cancer Agent, SpiD3
              </a>
            </td>
          <td>
            Elizabeth Schmitz, Abigail Ridout, Audrey L. Smith, Alexandria P. Eiken, Sydney A. Skupa, Erin M. Drengler, Sarbjit Singh, Sandeep Rana, A. Natarajan, Dalia El-Gamal
          </td>
          <td>2024-12-16</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="

 Introduction
 The loss of chromosome 7 or 7q arm, (-7/del7q), remains one of the most common cytogenetic abnormalities in myeloid malignancies, identified in 15% of MDS, 20-25% of secondary AML, and 10-15% of primary AML patients (Ogawa et al. Blood, 2019). Despite the relatively high frequency of this cytogenetic abnormality, the current prognosis remains dismal, with no existing options for targeted therapeutics, while comprehensive oncogenic mechanisms remain elusive.
 Past attempts at identifying and characterizing pathogenic genes disrupted during the -7/del7q deletion have focused largely on identification of lost tumor suppressor genes (TSGs) within commonly deleted regions of chromosome 7 (McNerney et al. Blood, 2013). More recent studies have focused on systematic genome-wide proliferation screens utilizing CRISPR, gene-trap, as well as cDNA libraries to identify and characterize large collections of potentially targetable tumor suppressor (Baeten et al. Leukemia, 2022). A comprehensive understanding of pathogenesis remains elusive, however, due to lack of correlation between observed clinical and biological outcomes and the large number of potential false positive hits that are not relevant in the -7/del7q contexts.
 Here, we utilized a novel iterative computational analysis coupled with experimental validation to identify targets and characterized a novel TSG in the context of a large and previously unexploited clinical dataset. Our approach identified and characterized Rap guanine nucleotide exchange factor 5 (RAPGEF5), which serves as a Ras/Rap1 regulator, acts as a novel tumor suppressor gene in the -7/del7q context.
 Methods and Results
 To determine potential TSG targets in a patient-focused context, a large in-house patient dataset was combined with public genomic datasets and systematically analyzed using novel computational algorithms to generate a unique list of potential TSGs. This rigorous statistical dataset was generated utilizing RNAseq data from 29 -7/del7q patients as well as whole exome sequencing and whole genome sequencing data from 47 -7/del7q patients. From this list of putative TSG targets, the most significant hits, including RAPGEF5, were selected for in vitro characterization.
 To assess the relative protein levels of RAPGEF5 in AML cell lines, we performed a Western Blot utilizing 2 cell models including one del7q (KG1) and one -7 model (F36P) as well as a chromosome 7 diploid AML cell model (THP-1). The -7/del7q cell lines demonstrated lower levels of RAPGEF5 when compared to the diploid cell line. The relative decrease in RAPGEF5 protein levels support the in-silico findings.
 To further evaluate the effects of RAPGEF5 loss on cell proliferation in vitro, we conducted a high efficiency CRISPR knockout (CRISPRko) utilizing an individualized sgRNA targeting RAPGEF5. The cell line KG1 was selected for the CRISPRko for its chromosome 7 defect and for the lack of expression data present in DepoMap, ensuring all subsequently generated findings will be novel. RAPGEF5 knockout efficiency was 95.1% at the time of assay. We performed a cell proliferation assay and interestingly, this knockout population demonstrated a significant increase in proliferation rate (18.0%, p=0.0091) when compared to the wild-type population. This finding is noteworthy considering that KG1 is a highly proliferative AML cell model. These findings suggest that RAPGEF5 acts in a tumor suppressive manner and is lost in the -7/del7q event.
 Finally, we observed a change in the variable allele frequency (VAF) of the initial KG1 RAPGEF5 CRISPRko population from 81.0% to 95.1% in 21 days of culture. This suggests that the loss of RAPGEF5 as a tumor suppressor provides an advantage over the KG1 WT population.
 Conclusion
 The above findings comprise the first report of RAPGEF5 acting as a tumor suppressor in MN. Our initial in vitro characterization of RAPGEF5 demonstrates its potential role as a tumor suppressor in the -7/del7q lesion and offers a novel therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c30fd61451a830030dd855ce0f6ddc56e3a80ae" target='_blank'>
              Rap Guanine Nucleotide Exchange Factor RAPGEF5 Is a Novel Tumor Suppressor in 7/del7q Myeloid Neoplasia
              </a>
            </td>
          <td>
            Jill Durkin, Daniel J Vail, Dongxu Jiang, Yahan Zhang, Simon Schlanger, Jaroslaw P. Maciejewski, T. LaFramboise, B. Jha
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="

 Background:
 T-cell lymphoblastic lymphoma (T-LBL) and acute lymphoblastic leukemia (T-ALL) originate from the malignant transformation of similar subsets of immature T-cell precursors in the thymus. Unique genetic and epigenetic aberrations lead to distinct clinical presentations. Both are distinguished by bone marrow infiltration. Patient's age at diagnosis is associated with the distribution of molecular subgroups in T-ALL (Neumann, et al. Leukemia, 2024). Despite sharing many clinical features and molecular alterations, there is ongoing debate about whether T-LBL and T-ALL represent a spectrum of a disease or are distinct entities. Recent papers identified TRB::NOTCH1 fusions exclusively in T-LBL and associated with high relapse risk (te Vrugt, et al. Blood, 2024). Both diseases are characterized by poor survival in resistant or relapsed cases, highlighting a clear unmet medical need. Understanding varying therapy resistance and molecular pathogenesis across ages is crucial. Clonal evolution analysis can help reconstruct tumor development, explore treatment responses, and identify reasons for therapy failure (Sandmann, et al. Int J Environ Res Public Health, 2023). This study compares DNA samples from pediatric and adult T-LBL and T-ALL patients to uncover genetic differences and tackle therapy resistance.
 Methods:
 DNA samples include material from initial diagnosis, corresponding germline, and, if applicable, relapse. Subcohorts include 87 pediatric T-LBL cases (27 relapsed, 60 non-relapsed), 36 pediatric T-ALL (12 relapsed, 24 non-relapsed), 47 adult T-LBL (of which 3 corresponding relapse samples were available), and 41 adult T-ALL, (of which 4 corresponding relapse samples were available). Besides targeted sequencing, CNV analysis is performed by Illumina Infinium Global Screening Array v3.0.
 Results and Discussion:
 For T-LBL and T-ALL, there were no significant differences in the average number of single nucleotide variants (SNV) per sample, either within or between the groups. For CNVs, a clear increase with age was observed for both T-LBL and T-ALL, except deletions and LOH affecting chromosome 9, which were more frequent in pediatric cases and decreased with age.
 Detailed analysis revealed subgroup-specific characteristics in initial samples. In adult T-LBL vs pediatric T-LBL samples, variants in PHF6 (34% vs 13%), JAK1 (9% vs 0%), TP53 (6% vs 0%), NOTCH3 (11% vs 5%), JAK3 (11% vs 5%), STAT5B (11% vs 5%), TET2 (6% vs 1%), and NRAS (9% vs 3%) were more prevalent. NOTCH1 (60% vs 49%), PIK3CA (10% vs 2%), PTEN (15% vs 9%), PIK3R1 (8% vs 3%), CCND3 (6% vs 0%), and KMT2C (6% vs 0%) were more frequent in pediatric T-LBL than adult T-LBL patients.
 In the T-ALL cohort, some variants, such as DNM2 (17%) and RUNX1 (10%) were almost exclusively found in adult cases. The increased prevalence of TP53 mutations in pediatric relapse samples underscored the critical role of TP53 in treatment resistance and disease progression (T-ALL: 8% primary vs 29% relapse; T-LBL: 0% primary vs 13% relapse).
 Clonal evolution analysis showed that CNVs appeared early, especially deletions or LOH affecting chromosome 9. Subsequently, additional subclones with point mutations evolved. Pediatric cases revealed differential timing and frequency of mutations in T-LBL vs in T-ALL: MYB (nested level 4.6 vs 1.0), SMARCA4 (4.7 vs 1.5), or TET2 (4.0 vs 1.0). In pediatric T-LBL, deletions and LOH on chromosome 9 occurred equally in primary samples of patients who suffered/did not suffer relapse. In relapse samples, the clonal population with CNV increased, indicating a growth advantage. Frequent CNVs in chromosome 9, encompassing genes like CDKN2A/B and MTAP, underscored the importance of this region in both T-LBL and T-ALL. These CNVs, involving tumor suppressor genes such as p16INK4a and p14ARF, were early events in clonal evolution.
 Conclusion:
 Our integrated genomic analysis highlights differences and similarities between pediatric and adult T-LBL and T-ALL cases. The analysis of CNVs revealed their role in clonal evolution, particularly with frequent deletions and LOH affecting chromosome 9 in pediatric cases. This emphasize the importance of genomic profiling for each subgroup. By understanding distinct genetic landscapes and mutation patterns, we aim to discover potential novel targets to improve risk group stratification of patients with T-LBL and T-ALL.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11984b8761d6dab05602ab5214642012180efc8e" target='_blank'>
              Recurrent Molecular Variants, Clonal Evolution and Clinical Relevance in T-Cell Lymphoblastic Neoplasia Comparing Lymphoma and Leukemia Samples Derived from Children and Adults
              </a>
            </td>
          <td>
            Sarah Sandmann, Marcel Te Vrugt, Gerrit Randau, T. Beder, Martin Neumann, Amelie Alfert, Cornelia Eckert, A. Moericke, Johanna Horns, Monika Brüggemann, Birgit Burkhardt
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aim Pancreatic carcinoma is an aggressive cancer that progresses without many symptoms. The difficulty of early diagnosis and an inadequate response to traditional treatments also cause the survival rate of pancreatic cancer to be low. Current research is focusing on methods of diagnosis and treatment, such as gene therapy, to increase survival rates. Small interfering ribonucleic acid (siRNA) has emerged as a promising advanced therapeutic strategy for cancer treatment. This study sought to silence the KRAS gene in the human pancreatic carcinoma cell line using a complex of small interfering ribonucleic acid (siRNA) and gold nanoparticles (AuNP). Methods In this study, 25 nM siRNA and gold nanoparticles at 0.5 mg/ml, 0.25 mg/ml, and 0.125 mg/ml concentrations were used to silence the KRAS gene in the CAPAN-1 cell line. Real-time PCR analysis, agarose gel electrophoresis, and double staining were carried out, and xCelligence real-time cell analysis (RTCA) was used to measure proliferation. Results The PCR analysis revealed crossing point (CP) values of actin beta (ACTB) ranging from 33.04 to 35.98, which was in the expected range for all samples. The interaction between the gold nanoparticle/siRNA complex in the double staining analysis revealed that the most effective concentration of gold nanoparticle was 0.125 mg/ml. The WST-1 technique showed that siRNA/AuPEI cells in application groups had a viability rate of over 90%, indicating no toxicity or side effects. The xCELLigence RTCA® showed that at hour 72, there was a significant difference in proliferation in the 0.5 mg/mL PEI/AuNP-siRNA, 0.25 mg/mL PEI/AuNP-siRNA, and 0.125 mg/mL PEI/AuNP-siRNA application groups compared to the control and siRNA groups (p < 0.05). By hour 96, all three groups were statistically different from the control and siRNA groups in terms of proliferation (p < 0.05). Conclusions The results of this analysis suggest that the AuPEI/siRNA complex can be effectively used to silence the target gene, but more studies are needed to verify these results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ee59cc0bcf1def023f7abe057418a49f245a10" target='_blank'>
              Investigation of the efficacy of siRNA-mediated KRAS gene silencing in pancreatic cancer therapy
              </a>
            </td>
          <td>
            Büşra Küçükekmekci, Fatma Azize Budak Yıldıran
          </td>
          <td>2024-11-12</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="

 Background: Acute myelocytic leukemia (AML) is a common hematological malignancy in adults. Although several risk stratifications based on cytogenetic and molecular abnormalities are available to decide the indication of allogeneic hematopoietic cell transplantation (allo-HCT), planning treatment strategies for AML without them remains challenging. In this study, using transcriptome datasets, we investigated the association of event-free survival (EFS) of intensively treated AML cases and the aberrant expression status of endogenous retrovirus (ERV)-derived open reading frames (ORFs), which have been reported to be associated with the pathophysiology of various malignancies and have the potential to become neoantigens in specific cancers.
 Methods: The RNA-seq data were obtained for 173 AML cases from The Cancer Genome Atlas (TCGA-LAML) and the Gene Expression Omnibus (GEO; GSE49642), as well as for 29 normal hematopoietic stem cells from GEO (GSE111085 and GSE114922). All FASTQ files were mapped to the human genome (hg38), and then the reads were counted with the GTF-formatted annotation file of the human ERV ORF definition obtained from the gEVE database version 1.1. DESeq2 was used to calculate statistical differences in ERV expression between AML and normal HSCs, and those with an adjusted p value <0.05 were extracted as differentially-expressed ERVs (DE-ERVs). We conducted gene-set enrichment analysis (GSEA) of DE-ERVs based on Repbase ERV family annotations to annotate the DE ERV families in AML cells. Quantitative PCR was performed for validation of ERV-derived mRNA expression in AML-derived cell lines (THP1, KG1, HL60, and K562). Furthermore, immunopeptides presented by human leukocyte antigen (HLA) class I of these cell lines were collected using immunoprecipitation with an anti-HLA class I antibody, and their amino acid sequences were identified using mass spectrometry (immunopeptidomics) to investigate the AML specific neoantigens derived from DE-ERVs. Clinical information of 151 AML cases registered in TCGA-LAML was obtained from TCGA website for prognostic analyses.
 Results: Access three independent DE analyses (TCGA-LAML vs. GSE111085, TCGA-LAML vs. GSE114922, and GSE49642 vs. GSE111085), a total of 557 DE-ERV ORFs were annotated as AML-characterizing DE-ERV ORFs. According to GSEA of the DE-ERV ORFs, a total of 10 ERV families were annotated as DE-ERV families, with total of 155 core-enrichment DE-ERV ORFs. Immunopeptidomics suggested that immunopeptides derived from seven of those 10 DE-ERV families were presented by HLA class I as neoantigens. Among those seven DE-ERVs, the mRNA expression values of human ERV family K9 (HERVK9) ORFs were found to be associated with EFS of patients treated with 3+7-based intensive chemotherapies (median EFSs of HERVK9-high and low expressors were 32.7 and 9.6 months, respectively; p=0.00479). This survival predominance in higher HERVK9 expressors was independent of their conventional risk stratifications (NCCN2017 and ELN2017) and gene mutations known to contributing to risk stratifications according to cox proportional hazards model (p=0.01971). Furthermore, according to GSEA, it was revealed that AML cells with higher expression of HERVK9 activated antigen processing and presentation, accompanied by excess expression of genes associated with responses to adaptive immune reaction and apoptosis, indicating that aberrant expression of HERVK9 may initiate an antineoplastic immune response against themselves via excess antigen presentation.
 Conclusions: While ERVs are aberrantly expressed in AML cells, HERVK9 expression induces an anti-neoplastic immune reaction via excess antigen presentation and is associated with better EFS in cases treated with intensive chemotherapies, independent of known risk classifications. Clinically, measurement of HERVK9 expression at initial diagnosis may help the dicision-making of indication of allogeneic hematopoietic cell transplantation, especially in the cases with good- or intermediate-risk stratification. In addition, HERVK9-derived neoantigens presented by HLA class I may have the potential to be efficient target of cell therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af8953c6d2477fef646bf1bd1b91eaf6df02fffd" target='_blank'>
              Aberrant Expression of Human Endogenous Retrovirus K9-Derived Elements Is Associated with Better Clinical Outcome of Acute Myelocytic Leukemia
              </a>
            </td>
          <td>
            Ryo Yanagiya, Yuriko Minegishi, M. Onizuka, Koji Ueda, Ai Kotani, So Nakagawa
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Specific and biologically informed treatments for medulloblastoma, especially the highly lethal TP53 mutant SHH subgroup, remain elusive, where radiotherapy is the primary treatment option. Applying genome-wide CRISPR-Cas9 screening in combination with lethal doses of radiotherapy, we identified the main driver of radiation resistance in SHH medulloblastoma is loss of p53. A negative selection CRISPR-Cas9 screen across multiple models of Trp53-deficient SHH medulloblastoma revealed a strong synthetic lethal interaction between components of the non-homologous end-joining pathway and radiation, particularly DNA protein kinase (DNA-PK) and its binding partners. Both genetic and pharmacological perturbation of DNA-PK enhanced radiosensitivity in TP53-deficient SHH medulloblastoma, leading to cell death. In vivo treatment of somatic and germline TP53-mutant SHH medulloblastoma models with peposertib, a small-molecule inhibitor of DNA-PK, significantly improved survival when combined with radiotherapy, strongly supporting further clinical investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9142d0bf16c5263f737ec56456cdebe6a89b5da8" target='_blank'>
              Targeting synthetic lethality between non-homologous end joining and radiation in very high-risk SHH medulloblastoma
              </a>
            </td>
          <td>
            Alexandria DeCarlo, G. Macleod, Carolina Fernandes da Silva, Lily Shen, Lucas Aragao, Mariska Sie, Deborah Termini, Jon Magee, Brian L. Gudenas, Sneha Sukumaran, Frederic Charron, Richard Marcellus, R. Al-Awar, Ahmed Aman, U. Tabori, C. Nör, P. Northcott, Peter Dirks, Stéphane Angers, Vijay Ramaswamy
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="

 Mutations in genes that code for RAS isozymes (N- and K-) are the most common (40-50%) somatic variations in multiple myeloma (MM). While ~46% of patients with newly diagnosed MM carry a RAS/Raf mutation, >64% of patients with refractory/relapsed MM carry these mutations. RAS activation can also occur from upstream signaling through receptor tyrosine kinases (RTK) or loss-of-function mutations in GTPase-activating proteins (GAPs), which are prevalent in MM. Although covalent inhibitors targeting KRAS alleles are in various stages of clinical development, multiple mechanisms of resistance have been reported. We discovered and developed a novel, highly potent, and selective pan-RAS inhibitor, ADT-007, capable of suppressing MAPK/AKT signaling by blocking GTP activation of RAS. Here, we show that ADT-007 is effective against MM, particularly those MM cell lines harboring KRAS and NRAS mutations that are representative of relapsed/refractory myeloma (RRMM).
 For these studies, we compiled a panel of over 50 human myeloma cell lines (HMCLs) with broad biological and genetic heterogeneity that represent innate drug-response/resistance (refractory disease), as well as 10 pairs of parental and clonally-derived proteasome inhibitor (PI)- and immunomodulatory drug (IMD)- resistant HMCLs as models of acquired resistance (relapse). Using CRISPR-cas-directed gene editing, we also have generated clonal NRAS and KRAS mutants of drug-sensitive myeloma lines that initially had two wild-type alleles. All cell lines have been authenticated using the GenePrint 24 System and tested for mycoplasma contamination.
 First, we performed predictive analysis using single-cell RNA sequencing (scRNAseq; 10X Genomics) on all MM cell lines and demonstrated the potential efficacy of ADT-007 based on the subclonal expression of the target genes/pathways. Using in vitro chemosensitivity assays, ADT-007 was highly potent and effective in killing chemo-resistant and RAS-mutant myeloma cells as a single agent with IC50 values ranging from 0.76 to 12 nM with synergistic benefits if combined with PIs and IMDs. Combination indices (CI) were found to be consistently below 0.9, indicating a synergistic drug interaction. (Drug synergy was determined by CalcuSyn software based on Chou-Talalay's CI method and the isobologram algorithm). Caspase 3/7 assay and flow cytometry (Annexin V-FITC + propidium iodide) confirmed apoptosis induction by ADT-007. Furthermore, the in vitro efficacy of ADT-007 was significantly better (nM vs. µM IC50) than cyclorasin B4-27, a bicyclic peptidyl that has been reported as a potent pan-RAS inhibitor against several other cancers, and RMC-6236, a multi-selective noncovalent RAS(on) inhibitor which is in clinical trial for patients with advanced solid tumors harboring G12X, G13X, and Q61X mutations. Pre-vs-post-treatment bulk-RNA sequencing followed by pathway analysis (using Ingenuity Pathway Analysis and Gene Set Enrichment Analysis) showed that ADT-007 suppressed KRAS and NRAS in a dose-dependent manner. Immunoblotting analysis confirmed the dose-dependent reduction of protein markers implicated in cell survival and chemotherapeutic resistance in myeloma, including c-myc, Ki-67, NFkB, IRF4, IKZF1 (Ikaros) and IKZF3 (Aiolos). Finally, we performed pre-vs-post-treatment single-cell transcriptomics analysis, which revealed a distinct shift in clusters representing myeloma single-cell subpopulations following ADT-007 treatment. High-dimensional mass cytometry (cytometry by time of flight or CyTOF) analysis using heavy metal-conjugated proteins in RRMM patient bone marrow-derived primary myeloma cells is currently underway to confirm the cytotoxic effect of ADT-007 ex vivo.
 ADT-1004 is an orally bioavailable prodrug of ADT-007 designed to improve water solubility and metabolic stability. ADT-1004 has shown strong antitumor activity in highly aggressive and clinically relevant mouse models of pancreatic cancer at dosages, is well tolerated, and is being developed by the NCI Stepping Stones program. Next, we plan to evaluate the antitumor activity of ADT-1004 in mouse xenograft models of human myeloma using immunocompromised mice.
 Our research will lay the groundwork for future development efforts needed to advance ADT-1004 to clinical trials involving patients with RRMM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21fdfe3b746dc5389d4165dd76c1cc342970daf2" target='_blank'>
              Preclinical Development of a Novel Pan-RAS Inhibitor Against Relapsed/Refractory Multiple Myeloma
              </a>
            </td>
          <td>
            Jeremiah C. Pfitzer, Razan Waliagha, Linda B. Baughn, A. Keeton, Xi Chen, Y. Maxuitenko, Shaji K Kumar, Brian Van Ness, G. Piazza, Amit K Mitra
          </td>
          <td>2024-11-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>26</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [8, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>