<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-05 09:29:08 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>31</td>
          <td>43</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d685ee84bfeabbf621f354633519c29df6bcad68" target='_blank'>
              Abstract 4399: Systematic interrogation of cell states that drive inflammation tolerance and therapy resistance in cancer
              </a>
            </td>
          <td>
            Srivatsan Raghavan, Evelyn Y. Tong, Aswanth H. Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andrew W. Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4abf519afad0178ba59d0cd17f3f428071bf92cb" target='_blank'>
              Abstract 2674: Cellular plasticity facilitates therapy resistance through enhancing adaptability
              </a>
            </td>
          <td>
            Alicia Bjornberg, Xierali Aobuli, M. Froid, R. Barker-Clarke, Jeffrey Maltas, Jacob G. Scott, Berkley Gryder, David Bassanta, Alexander Anderson, Virginia Turati, A. Marusyk
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96dda01278fdb738773a58b49aca47aa0308c83d" target='_blank'>
              Abstract 5541: Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing
              </a>
            </td>
          <td>
            Chufan Zhang, Chu Pan, D. Cescon, Mathieu Lupien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bca1af35819fcfb6f9d7674ff4443096f1ec742" target='_blank'>
              Single-Cell Pharmacogenomic Landscapes of Epigenetic Therapy in Gastric Cancer
              </a>
            </td>
          <td>
            Chang Xu, Haoran Ma, T. Sheng, M. Lee, S. A. A. Abdul Ghani, Laura Perlaza-Jiménez, K. Huang, Shen Kiat Lim, Supriya Srivastava, Xuewen Ong, S. Tay, S. Ho, Angie Tan, Feng Zhu, Vincenzo Nasca, G. Randon, Hassan Ashktorab, A. S. Cheng, A. Jeyasekharan, Shang Li, Ming Teh, Raghav Sundar, David R. Powell, Joseph Rosenbluh, F. Pietrantonio, W. Yong, J. So, Patrick Tan
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eed8fdb3c30cedec701229d2292e4e2a7ab7ba3" target='_blank'>
              Abstract 6356: Multiscale high-content imaging to identify molecular and morphological signatures in drug-resistant cancer cells
              </a>
            </td>
          <td>
            Yaxin Yang, Jianquan Xu, Chaojie Zhang, Hongqiang Ma, Sarah J. Hainer, Yang Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Identifying genetic dependencies in human colon cancer could help identify effective treatment strategies. Genome-wide CRISPR-Cas9 dropout screens have the potential to reveal genetic dependencies, some of which could be exploited as therapeutic targets using existing drugs. In this study, we comprehensively characterized genetic dependencies present in a colon cancer organoid avatar, and validated tumor-specific selectivity of select pharmacologic agents. We conducted a genome-wide CRISPR dropout screen to elucidate the genetic dependencies that interacted with select driver somatic mutations. We found distinct genetic dependencies that interacted with WNT, MAPK, PI3K, TP53, and mismatch repair pathways and validated targets that could be exploited as treatments for this specific subtype of colon cancer. These findings demonstrate the utility of functional genomic screening in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f08c27268b9a856764ce30d936babdccabacfc" target='_blank'>
              Comprehensive genomic dependency landscape of a human colon cancer organoid
              </a>
            </td>
          <td>
            Sanam Khalili, Atefeh Mohseninia, Changlong Liu, Carolyn E. Banister, Paige Heine, M. Khazan, Sidney E. Morrison, P. Gokare, Glenn S Cowley, Barbara A. Weir, David Pocalyko, Kurtis E. Bachman, Phillip J. Buckhaults
          </td>
          <td>2025-03-14</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178582b0b4cac0c55049a52434a209bd8b7613e6" target='_blank'>
              Quantitative measurement of phenotype dynamics during cancer drug resistance evolution using genetic barcoding
              </a>
            </td>
          <td>
            F. J. Whiting, M. Mossner, C. Gabbutt, C. Kimberley, C. P. Barnes, A. Baker, Erika Yara, A. Sottoriva, Richard A. Nichols, Trevor A. Graham
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43efa757ce011d3db682c5cad92c0307f3b656c7" target='_blank'>
              The Role of Phenotypic Plasticity in Adaptation to Treatment and Prospective Plasticity Drivers in Hepatoblastoma
              </a>
            </td>
          <td>
            Emilia Chen, Christie English, Alejandro Allo Anido, Siân Hamer, Stefano Cairo, Alejandra Bruna
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9151033e1d6669563430e41bef627c6f841c5064" target='_blank'>
              Abstract 5551: From persistence to resistance: insights into the mechanisms of drug tolerance in cancer therapy using patient-derived organoids
              </a>
            </td>
          <td>
            Yasmine Abouleila, Timo Voskuilen, M. Doorn, Roel Verkerk, Joris Maas, C. Veríssimo, Sylvia F. Boj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chemotherapy remains a prevalent strategy in cancer therapy; however, the emergence of drug resistance poses a considerable challenge to its efficacy. Most drug resistance arises from the accumulation of genetic mutations in a minority of resistant cells. The mechanisms underlying the emergence and progression of cancer resistance from these minority-resistant cells (MRCs) remain poorly understood. This study employs force-induced remnant magnetization spectroscopy (FIRMS) alongside various biological investigations to reveal the mechanical pathways for MRCs fostering drug resistance and tumor progression. The findings show that minority Paclitaxel-resistant cancer cells have enhanced mechanical properties. These cells can transmit high-intensity forces to surrounding sensitive cells (SCs) through the force transducer, Merlin. This force transmission facilitates the assimilation of surrounding SCs, subsequently strengthening the contraction and adhesion of tumor cells. This process is termed "mechano-assimilation," which accelerates the development of drug resistance and tumor progression. Interestingly, disturbances and reductions of mechano-assimilation within tumors can restore sensitivity to Paclitaxel both in vitro and in vivo. This study provides preliminary evidence highlighting the contribution of MRCs to the development of drug resistance and malignancy, mediated through mechanical interactions. It also establishes a foundation for future research focused on integrating mechanical factors into innovative cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f46cb96deeea2cf4ed683e45346444a026b4be6" target='_blank'>
              Mechanisms and Therapeutic Strategies for Minority Cell-Induced Paclitaxel Resistance and Tumor Progression Mediated by Mechanical Forces.
              </a>
            </td>
          <td>
            Xueyan Feng, Di Zhang, Guoxun Wang, Liwei Lu, Feng Feng, Xiuyu Wang, Chanchan Yu, Yahong Chai, Jin Zhang, Wenchao Li, Jing Liu, Hongxia Sun, Li Yao
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d56e1d2945b307c87e3e5dbfa94e2c69fe76398" target='_blank'>
              Multi-Omic Profiling Reveals Epigenetic Drivers of Immunotherapy Resistance in Multiple Myeloma
              </a>
            </td>
          <td>
            L. Haertle, Natalia Buenache Cuenda, Francisco J. Villena González, Isabel Espejo Díaz, Paula Lazaro del Campo, J. M. Rosa-Rosa, Rafael Alberto, Alonso Fernandez, Maria Nieves Lopez Muñoz, J. Sánchez-Pina, A. García-Ortiz, A. Valeri, S. Barrio, Fátima Al-Shahrour, Florian Bassermann, J. Martínez-López, Di Domenico, T. Domenico
          </td>
          <td>2025-04-19</td>
          <td>None</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Cancer has consistently posed a significant challenge on a global scale, with rising incidence and mortality. Despite the tremendous advancements in cancer therapeutics throughout the years, chemotherapy remains the current standard treatment in cancer management. As a result, chemotherapy resistance among cancer patients has emerged as a major clinical challenge, causing unsatisfactory responses to the treatment. Chemotherapy resistance is closely related to the dysregulation of a normal cellular process called ‘pre-mRNA Alternative Splicing’, where one gene can generate various protein forms with different structures and functions. A comprehensive understanding of dysregulated alternative splicing mechanisms is essential to propose novel strategies in cancer research aimed at identifying new and improved methods to maximise treatment efficacy and overcome chemotherapy resistance. Therefore, this review highlights the complexity of chemotherapy resistance and the potential roles of alternative splicing in overcoming resistance to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c315ff50467ff3617156ad8cf2b1b2d1ebcf6f" target='_blank'>
              Aberrant Splicing as a Mechanism for Resistance to Cancer Therapies
              </a>
            </td>
          <td>
            Duygu Duzgun, S. Oltean
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df82979a5771df69f4d6ba63039b4660f6ec17f" target='_blank'>
              Abstract 5065: Single-cell transcriptomic profiling of longitudinal patient tumors and PDX models reveals resistance mechanisms in ALK- and EGFR-mutant LUAD
              </a>
            </td>
          <td>
            Naomi Mfonfu, F. Braye, Galina Boldina, Matteo Cesaroni, Luc Fibroulet, Sergey I. Nikolaev, Andrey A. Yurchenko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863308a92ce0800145d29b567e0b6c910bc5cae2" target='_blank'>
              Abstract 6494: Genetically characterized canine tumoral cell lines are relevant models to test the efficacy of new targeted therapies: Example of anti-MDM2 targeted therapy
              </a>
            </td>
          <td>
            P. Montagne, U. Jarry, Caroline Confais, É. Cadieu, Aline Primot, Nicolaï Hoffman, Armel Houël, Rémy Le Guével, T. Derrien, Catherine André, Jérome Abadie, Christophe Drieu La Rochelle, B. Hédan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/221d45fdda028b3fdec5da9d6b92cc4847a8b483" target='_blank'>
              Understanding the Role of Toggle Genes in Chronic Lymphocytic Leukemia Proliferation
              </a>
            </td>
          <td>
            Olga Sirbu, Gunjan Agarwal, Alessandro Giuliani, Kumar Selvarajoo
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7191d1be8112623ab8fc75b326b24bf447563f16" target='_blank'>
              Abstract 1257: Response to primary therapy predicts tumor evolution dynamics at relapse in ovarian cancer
              </a>
            </td>
          <td>
            J. Oikkonen, Giulia Micoli, Susanna Holmström, Kari Lavikka, M. T. Nguyen, Yilin Li, G. Marchi, A. Rajavuori, Heidi Koskela, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Cancer is heterogeneous and variability in drug sensitivity is widely documented across cancer types. Adaptive therapy is an emerging modality of cancer treatment that leverages this drug resistance heterogeneity to improve therapeutic outcomes. Current standard treatments typically eliminate a large fraction of drug-sensitive cells, leading to drug-resistant relapse due to competitive release. Adaptive therapy aims to retain some drug-sensitive cells, thereby limiting resistant cell growth by ecological competition. While early clinical trials of such a strategy have shown promise, optimisation of adaptive therapy is a subject of active study. Current methods largely assume cell phenotypes to remain constant, even though cell-state transitions could permit drug-sensitive and -resistant phenotypes to interchange and thus escape therapy. We address this gap using a deterministic model of population growth, in which sensitive and resistant cells grow under competition and undergo cell-state transitions. Based on the model’s steady-state behaviour and temporal dynamics, we identify distinct balances of competition and phenotypic transitions that are suitable for effective adaptive versus constant dose therapy. Our data indicate that under adaptive therapy, models with cell-state transitions show a higher frequency of fluctuations than those without, suggesting that the balance between ecological competition and phenotypic transitions could determine population-level dynamical properties. Our analyses also identify key limitations of applying phenomenological models in clinical practice for therapy design and implementation, particularly when cell-state transitions are involved. These findings provide an overall perspective on the relevance of phenotypic plasticity for emerging cancer treatment strategies using population dynamics as an investigation framework. Significance Statement Drug-sensitive and -resistant cancer cells can compete with each other within the same tumour, and adaptive therapy exploits this competition to control overall tumour growth. The fact that sensitive and resistant cell types can switch phenotypes complicates the implementation of adaptive therapy. Our ODE-based theoretical ecology framework shows that asymmetric competition favouring sensitive cells generally benefits therapy outcomes, while phenotypic plasticity is usually detrimental. Our model also provides cell population-level indicators that can help predict the underlying balance between competition and plasticity. Phenomenological models are limited by whether their parameters can be reliably identified given scarce data, and our study illuminates the scope of such models in understanding cancer population dynamics and the need for mechanistic modelling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5674ed15d6d0f458d4dbdaa153d7277526c19b0b" target='_blank'>
              Impacts of competition and phenotypic plasticity on the viability of adaptive therapy
              </a>
            </td>
          <td>
            B. Vibishan, Paras Jain, Vedant Sharma, K. Hari, Claus Kadelka, Jason T. George, M. Jolly
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b60f1dd8ce9984467648ff9cf2c70d381e498a6" target='_blank'>
              Abstract 7502: Radiation-induced transcriptomic reprogramming of mesenchymal glioblastoma stem cells reveals key pathways and potential targets
              </a>
            </td>
          <td>
            Grace M. Rosner, Abigail Goen, Godwin Peasah-Darkwah, Jonathan Rodgers Gochicoa, Alos B. Diallo, Gokul Srinivasan, Elisa Bu Sha, Joshua Levy, Kelli B. Pointer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human cell lines are fundamental tools in biomedical research and are widely used in disease modeling, drug development, and many other domains. Here, we present chromosome-level, phased diploid genome assemblies of two popular human cell lines: the BJ foreskin fibroblast line and the IMR-90 fetal lung fibroblast line. Our high-quality assemblies, generated using long-read and Hi-C sequencing data, reveal substantial structural variation, including more than 50,000 insertions, deletions, duplications, and inversions compared to the recent T2T-CHM13v2.0 reference. Our assemblies provide detailed maps of genetic variation, enabling more accurate variant calling and the ability to phase reads when using newly generated or historical sequencing data on these cell lines or their derivatives. All assemblies and associated data have been made available as a resource for the research community. We envision that diploid genome assembly will become a cornerstone approach for personalized medicine in the near future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68496fd9e9a9f934f03f0dd67116b1c071c33e67" target='_blank'>
              Diploid genome assembly of human fibroblast cell lines enables clone specific variant calling, improved read mapping and accurate phasing
              </a>
            </td>
          <td>
            T. R. Ranallo-Benavidez, Yue Hao, Emilia Volpe, Maryam Jehangir, Noelle Fukushima, Ogechukwu Mbegbu, Rebecca Reiman, Jessica Molnar, Danyael Murphy, Dorothy Marie Paredes, Shukmei Wong, Kara Karaniuk, Stephanie Buchholtz, Jonathan Keats, M. Machiela, M. Kolmogorov, B. Paten, Simona Giunta, Floris P Barthel
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Clinical analyses demonstrated treatment naive DDR status was associated with different responses to frontline chemotherapy. Using longitudinal liquid biopsies, we found that DDR Intermediate and High tumors exhibited subtype switching and coincident emergence of heterogenous phenotypes following frontline treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes and may be associated with different chemotherapy responses and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02291-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Background Therapy-induced senescence (TIS) is considered a permanent cell cycle arrest following DNA-damaging treatments; however, its irreversibility has recently been challenged. Here, we demonstrate that escape from TIS is universal across breast cancer cells. Moreover, TIS provides a reversible drug resistance mechanism that ensures the survival of the population, and could contribute to relapse. Methods TIS was induced in four different breast cancer cell line with high-dose chemotherapy and cultured until cells escaped TIS. Parental, TIS and repopulating cells were analyzed by bulk and single-cell RNA sequencing and surface proteomics. A genetically engineered mouse model of triple-negative breast cancer was used to prove why current senolytics cannot overcome TIS in tumors. Results Screening the toxicity of a diverse panel of FDA-approved anticancer drugs revealed that TIS meditates resistance to half of these compounds, despite their distinct mechanism of action. Bulk and single-cell RNA sequencing, along with surface proteome analysis, showed that while parental and repopulating cells are almost identical, TIS cells are significantly different from both, highlighting their transient nature. Furthermore, investigating dozens of known drug resistance mechanisms offered no explanation for this unique drug resistance pattern. Additionally, TIS cells expressed a gene set associated with immune evasion and a potential KRAS-driven escape mechanism from TIS. Conclusion Our results reveal that TIS, as a transient drug resistance mechanism, could contribute to overcome the immune response and to relapse by reverting to a proliferative stage. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02310-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838695afba129f01c7b6f860f5a9601961169be" target='_blank'>
              Therapy-induced senescence is a transient drug resistance mechanism in breast cancer
              </a>
            </td>
          <td>
            Eszter Bajtai, Csaba Kiss, Éva Bakos, Tamás Langó, Anna Lovrics, É. Schád, V. Tisza, Károly Hegedűs, Péter Fürjes, Zoltán Szabó, Gábor E. Tusnády, G. Szakács, Á. Tantos, S. Spisák, J. Tóvári, András Füredi
          </td>
          <td>2025-05-01</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217101cddf06b5ca15708fdd50f15de9f63e9eb9" target='_blank'>
              Abstract 5092: Characterization of mismatch repair status in colorectal cancer using single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Dimitri Joseph, Alexandra Guillaume
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Leukemia is a heterogeneous group of hematologic malignancies characterized by distinct genetic and molecular abnormalities. Advancements in genomic technologies have significantly transformed the diagnosis, prognosis, and treatment strategies for leukemia. Among these, next-generation sequencing (NGS) has emerged as a powerful tool, enabling high-resolution genomic profiling that surpasses conventional diagnostic approaches. By providing comprehensive insights into genetic mutations, clonal evolution, and resistance mechanisms, NGS has revolutionized precision medicine in leukemia management. Despite its transformative potential, the clinical integration of NGS presents challenges, including data interpretation complexities, standardization issues, and cost considerations. However, continuous advancements in sequencing platforms and bioinformatics pipelines are enhancing the reliability and accessibility of NGS in routine clinical practice. The expanding role of NGS in leukemia is paving the way for improved risk stratification, targeted therapies, and real-time disease monitoring, ultimately leading to better patient outcomes. This review highlights the impact of NGS on leukemia research and clinical applications, discussing its advantages over traditional diagnostic techniques, key sequencing approaches, and emerging challenges. As precision oncology continues to evolve, NGS is expected to play an increasingly central role in the diagnosis and management of leukemia, driving innovations in personalized medicine and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4cbce0b7fb1d47370357ef7f4f5d368e5b4543" target='_blank'>
              Genetic Profiling of Acute and Chronic Leukemia via Next-Generation Sequencing: Current Insights and Future Perspectives
              </a>
            </td>
          <td>
            Laras Pratiwi, Fawzia Hanum Mashudi, Mukti Citra Ningtyas, H. Sutanto, P. Romadhon
          </td>
          <td>2025-03-28</td>
          <td>Hematology Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c001caff1773dd2045b65bb8abc13f8680dd3d4" target='_blank'>
              Pan-Cancer PDOs Preserve Tumor Heterogeneity and Uncover Therapeutic Vulnerabilities
              </a>
            </td>
          <td>
            Hui-Hsuan Kuo, B. Bhinder, H. Gokozan, Kathryn Gorski, Pooja Chandra, J. Manohar, Daniela Guevara, John Otilano, Jenna E Moyer, Marvel Tranquille, Sarah Ackermann, Jared Capuano, Cynthia Cheung, Thomas Anthony Caiazza, Phoebe L. Reuben, Anastasia Murray Tsomides, Adriana Irizarry, M. Sigouros, David Wilkes, Abigail King, Troy Kane, Majd Al Assaad, Wael Al Zoughbi, Kentaro Ohara, Joonghoon Auh, P. Waltman, Florencia P. Madorsky Rowdo, Enrique Podaza, Valerie Gallegos, John Nguyen, Raehash Shah, Manish Shah, Allyson J. Ocean, Douglas Scherr, Nasser K. Altorki, M. Frey, Ana M. Molina, Lisa Newman, Vivan Bea, E. Chapman-Davis, Marcus D. Goncalves, Ashish Saxena, Parul J. Shukla, Kevin Holcomb, Rachel Simmons, S. Tagawa, J. Zippin, Evelyn Cantillo, R. Chandwani, Melissa B. Davis, Kelly A. Garrett, P. Kasi, Jennifer Marti, David M. Nanus, J. Nauseef, Elizabeth Popa, Joseph Ruggiero, Momin T. Siddiqui, Alicia Alonso, Cora N. Sternberg, B. M. Faltas, O. Elemento, J. Mosquera, A. Sboner, M. L. Martin
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019e491b785ffb0bdd2ceea7ba005ed8f5742762" target='_blank'>
              Relapse-specific genetic patterns in the exomic mutational landscape in acute myeloid leukemia
              </a>
            </td>
          <td>
            Wardah Qureshi, Aisha Jamal, Atia Gohar, Marium Imran Mansuri, Muhammad Irfan, I. Khan, Muhammad Shakeel
          </td>
          <td>2025-03-25</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbcd5ab9fc14637f7ac25a5d726846fbbce055ae" target='_blank'>
              Abstract 2396: Gene fusion and variant-aware isoform detection with functional prediction from long-read RNA sequencing
              </a>
            </td>
          <td>
            Colette Felton, Andrea Galvez, L. Sayles, Christopher Vollmers, Alejandro Sweet-Cordero, Angela N. Brooks
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor subclones refer to distinct cell populations within the same tumor that possess different genetic characteristics. They play a crucial role in understanding tumor heterogeneity, evolution, and therapeutic resistance. The formation of tumor subclones is driven by several key mechanisms, including the inherent genetic instability of tumor cells, which facilitates the accumulation of novel mutations; selective pressures from the tumor microenvironment and therapeutic interventions, which promote the expansion of certain subclones; and epigenetic modifications, such as DNA methylation and histone modifications, which alter gene expression patterns. Major methodologies for studying tumor subclones include single-cell sequencing, liquid biopsy, and spatial transcriptomics, which provide insights into clonal architecture and dynamic evolution. Beyond their direct involvement in tumor growth and invasion, subclones significantly contribute to tumor heterogeneity, immune evasion, and treatment resistance. Thus, an in-depth investigation of tumor subclones not only aids in guiding personalized precision therapy, overcoming drug resistance, and identifying novel therapeutic targets, but also enhances our ability to predict recurrence and metastasis risks while elucidating the mechanisms underlying tumor heterogeneity. The integration of artificial intelligence, big data analytics, and multi-omics technologies is expected to further advance research in tumor subclones, paving the way for novel strategies in cancer diagnosis and treatment. This review aims to provide a comprehensive overview of tumor subclone formation mechanisms, evolutionary models, analytical methods, and clinical implications, offering insights into precision oncology and future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1d4c21265c412fc5d4c0775f9aa0ecd36b6c8eb" target='_blank'>
              Advances in tumor subclone formation and mechanisms of growth and invasion
              </a>
            </td>
          <td>
            Yuhong Zhang, Weidong Wang
          </td>
          <td>2025-04-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive variant and androgen receptor (AR)-independent castration resistant prostate cancers (CRPC) represent the most significant diagnostic and therapeutic challenges in prostate cancer. This study examined a case of simultaneous progression of both adenocarcinoma and squamous tumors from the same common origin. Using whole-genome and transcriptome sequencing from 17 samples collected over >6 years, we established the clonal relationship of all samples, defined shared complex structural variants, and demonstrated both divergent and convergent evolution at AR. Squamous CRPC-associated circulating tumor DNA was identified at clinical progression prior to biopsy detection of any squamous differentiation. Dynamic changes in the detection rate of histology-specific clones in circulation reflected histology-specific sensitivity to treatment. This dataset serves as an illustration of non-neuroendocrine transdifferentiation and highlights the importance of serial sampling at progression in CRPC for the detection of emergent non-adenocarcinoma histologies with implications for the treatment of lineage plasticity and transdifferentiation in metastatic CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65b072299240230cb903f6cbde3bd730a8642b8" target='_blank'>
              A complex phylogeny of lineage plasticity in metastatic castration resistant prostate cancer
              </a>
            </td>
          <td>
            J. Nauseef, Timothy R. Chu, William F Hooper, Alicia Alonso, Ali Oku, Heather Geiger, Zoe R. Goldstein, Minita J. Shah, M. Sigouros, J. Manohar, Zoe Steinsnyder, Lara Winterkorn, Brian D. Robinson, A. Sboner, H. Beltran, O. Elemento, I. Hajirasouliha, Marcin Imielinski, D. Nanus, S. Tagawa, N. Robine, Juan Miguel Mosquera
          </td>
          <td>2025-03-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f680cbd3c45701162ce05be4c984e82cf3cb95" target='_blank'>
              Abstract 1258: Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, M. Peifer, Roman K. Thomas, Julie George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9567c766da45eca4af083f72109f47e2c4c4d5f4" target='_blank'>
              Abstract 5130: DNA methylation profiling reveals epigenetic dynamics across epithelial-mesenchymal transition states in the triple-negative breast tumor microenvironment
              </a>
            </td>
          <td>
            Hanxu Lu, F. Kolling, Lucas A Salas, D. Pattabiraman, Todd W Miller, Brock C. Christensen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a3bde6f2ebc33e7239f55d9b0ddb6732733f31" target='_blank'>
              Abstract 4190: CRISPR-CAS9 library screening identifies novel vulnerabilities in CRC drug-tolerant persister cells
              </a>
            </td>
          <td>
            Martina Miotto, S. Lamba, G. Grasso, G. Corti, Julie Bonetto, A. Sogari, A. Bardelli, Mariangela Russo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Triple-negative breast cancer, characterized by aggressive growth and high intratumor heterogeneity, presents a significant clinical challenge. Here, we use a lineage-tracing system, ClonMapper, which couples heritable clonal identifying tags with single-cell RNA-sequencing (scRNA-seq), to better elucidate the response to doxorubicin in a model of TNBC. We demonstrate that, while there is a dose-dependent reduction in overall clonal diversity, there is no pre-existing resistance signature among surviving clones. Separately, we found the existence of two transcriptomically distinct clonal subpopulations that remain through the course of treatment. Among clones persisting across multiple samples we identified divergent phenotypes, suggesting a response to treament independent of clonal identity. Finally, a subset of clones harbor novel changes in expression following treatment. The clone and sample specific responses to treatment identified herein highlight the need for better personalized treatment strategies to overcome tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360a99c9dba219540b793cdb094157ffa2ec6a" target='_blank'>
              Lineage Tracing Reveals Clone-Specific Responses to Doxorubicin in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Daylin Morgan, Andrea L. Gardner, Amy Brock
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc3b5fa458a00b7ac6aa3d0d25d390376657ca" target='_blank'>
              Abstract 5544: Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells
              </a>
            </td>
          <td>
            S. Choudhury, Shixiong Wang, Y. Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, T. Aittokallio
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5713e3952828abd074f75de5f64c2e281aa291d9" target='_blank'>
              Abstract 4408: CRISPR/Cas9 knockout and AI-based drug screening for novel gene targets in ovarian cancer chemoresistance
              </a>
            </td>
          <td>
            Jinjiang Wang, Jiaxi Li, Jiao Liu, Kit-Ying Chan, Ho-Sze Lee, Zhewei Zhang, Joseph Kwong, T. Lau, C. Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd635bb806c9651d9badaf4a27cb3a0268b9273e" target='_blank'>
              Synergistic Targeting of EGFR, ESR1, BCL2, and TP53 Pathways: A Multi-Pronged Approach for Advanced Breast Cancer Therapy.
              </a>
            </td>
          <td>
            Harneet Marwah, H. K. Dewangan
          </td>
          <td>2025-04-24</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John P. McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J. Colón-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea7a11f970c3515d6da1945980a0668b04d9e68" target='_blank'>
              Abstract 4189: CRISPR-edited cell lines targeting key molecular pathways: Accelerating the development of synthetic lethal therapeutics beyond PARP inhibitors
              </a>
            </td>
          <td>
            Yue Huang, Yunpeng Zhai, Yan Feng, Xinyue Jia, Xiaohuan Zhang, Jinying Ning, F‐J Hao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd0035313cd95f13c0ad0b1db10655d699eb842" target='_blank'>
              Abstract 2991: Systematic discovery of regulators of adoptive T-cell therapeutic resistance using in vivo CRISPR screening
              </a>
            </td>
          <td>
            Chenlin Zhao, Sharif U. Ahmed, Xiyue Hu, R. S. Atwal, Zongjie Wang, Sophie Xiao, S. Kelley
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a1ac1da292bdfa7cb083f4691cc613492b11ea" target='_blank'>
              Pan-cancer Analysis Identified Ectopic RUNX1T1 Associated with Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, I. Kim, Su Deng, Ping Mu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Development of next generation sequencing technologies allows to identify a large number of genetic landscape types in various cancers including breast cancer. Frequent genetic abnormalities identified using whole genome sequencing are point mutations (missense, nonsense mutations), deletions, insertions, which usually lead to activation of protooncogenes and inactivation of tumor suppressor genes. Genome sequencing of malignant tumors allowed, on one hand, to identify driver mutations in carcinogenic genes in different organs, and on the other – to use mutated genes for targeted therapy. Study of biological functions of these genes from the point of view of their contribution to carcinogenesis allows to better understand its mechanism. In this review, signaling cascades of breast cancer with identified mutated genes – targets for therapy – are analyzed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d647ad1e76680b2e0c017bae0688ab2ee0a334" target='_blank'>
              Signaling cascades are targets for breast cancer therapy in the light of genome – wide sequencing data
              </a>
            </td>
          <td>
            L. F. Gulyaeva, M. Filipenko, N. E. Kushlinskii
          </td>
          <td>2025-04-13</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf51075c6ced2d719b16f81522dd0e475d5f9454" target='_blank'>
              Abstract 4605: Exploring resistance mechanisms in cancer using patient-derived organoids reveals replication stress and EMT as potential biomarkers
              </a>
            </td>
          <td>
            Ruben M. Drews, Finnian Firth, Paul R. Barber, Anna Pasto, Tony T. Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Although two-thirds of cancers arise from loss-of-function mutations in tumor suppressor genes, there are few approved targeted therapies linked to these alterations. Synthetic lethality offers a promising strategy to treat such cancers by targeting vulnerabilities unique to cancer cells with these mutations. To identify clinically relevant synthetic lethal interactions, we analyzed genome-wide CRISPR/Cas9 knock-out (KO) viability screens from the Cancer Dependency Map and evaluated their clinical relevance in patient tumors through mutual exclusivity, a pattern indicative of synthetic lethality. Indeed, we found significant enrichment of mutual exclusivity for interactions involving cancer driver genes compared to non-driver mutations. To identify therapeutic opportunities, we integrated drug sensitivity data to identify inhibitors that mimic the effects of CRISPR-mediated KO. This approach revealed potential drug repurposing opportunities, including BRD2 inhibitors for bladder cancers with ARID1A mutations and SIN3A-mutated cell lines showing sensitivity to nicotinamide phosphoribosyltransferase (NAMPT) inhibitors. However, we discovered that pharmacological inhibitors often fail to phenocopy KO of matched drug targets, with only a small fraction of drugs inducing similar effects. This discrepancy reveals fundamental differences between pharmacological and genetic perturbations, emphasizing the need for approaches that directly assess the interplay of loss-of-function mutations and drug activity in cancer models. Author Summary Synthetic lethality is an emerging approach for targeting a biological dependency in cancer cells that does not harm normal cells. This strategy is particularly valuable for targeting loss-of-function mutations in tumor suppressor genes, which are more challenging to directly target. In an effort to accelerate treatments for cancer patients, we aimed to map out these dependencies and overlap them with responses to available drugs. We discovered different outcomes when a protein is targeted by a drug versus when that same target is disrupted genetically. Thus, if a drug is to be effectively repurposed as synthetic lethal agent, feasibility studies must capture drug biology, ideally by test the drug empirically in relevant cancer models. A second notable discovery is that in vitro synthetic lethal interactions involving cancer driver genes are significantly more likely to exhibit consistent patterns, such as mutual exclusivity in human tumor samples. This is important since selection of relevant cell lines is often critical in drug development to maximize potential for translation to clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c1469ac01d0657e51e2a370ba73f2735664c7f" target='_blank'>
              Challenges and opportunities for drug repurposing in cancers based on synthetic lethality induced by tumor suppressor gene mutations
              </a>
            </td>
          <td>
            Michael Vermeulen, Andrew W. Craig, Tomas Babak
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6afa5982640193cfc266d2068be1bacf5082da2a" target='_blank'>
              Abstract 2648: ZFP36L2 orchestrates stress adaptive plasticity during injury repair and metastasis
              </a>
            </td>
          <td>
            Qingwen Jiang, Manisha Raghavan, Britney Forsyth, Aileen Rodriguez, Cyrus Tam, Sasha Balkaran, S. Hartner, Morgan Lallo, Ahmed Mahmoud, Andrew Moorman, Jura Pintar, D. Pe’er, R. Koche, Quaid Morris, Joseph Chan, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Primary mouse renal proximal tubule epithelial cells (moRPTECs) were immortalized by lentivirus transduction to create hTERT or SV40LgT (LgT) cell lines. Prior work showed a more pronounced injury and repair response in LgT versus hTERT cells after chemical challenge. We hypothesized that unique genomic changes occurred after immortalization, altering critical genes and pathways. RNA-seq profiling and whole-genome sequencing (WGS) of parent, hTERT, and LgT cells showed that 92.5% of the annotated transcripts were shared, suggesting a conserved proximal tubule expression pattern. However, the cell lines exhibited unique transcriptomic and genomic profiles different from the parent cells. Three transcript classes were quite relevant for chemical challenge response—Cyps, ion channels, and metabolic transporters—each important for renal function. A pathway analysis of the hTERT cells suggested alterations in intermediary and energy metabolism. LgT cells exhibited pathway activation in cell cycle and DNA repair that was consistent with replication stress. Genomic karyotyping by combining WGS and RNA-seq data showed increased gene copy numbers in chromosome 5 for LgT cells, while hTERT cells displayed gene copy losses in chromosomes 4 and 9. These data suggest that the exaggerated transcriptional responses of LgT cells versus hTERT cells result from differences in gene copy numbers, replication stress, and the unique selection processes underlying LgT or hTERT immortalization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5fc2423810ee9671b81c235e2952419eb5bbc3d" target='_blank'>
              hTERT and SV40LgT Renal Cell Lines Adjust Their Transcriptional Responses After Copy Number Changes from the Parent Proximal Tubule Cells
              </a>
            </td>
          <td>
            B. A. Merrick, Ashley M Brooks, Julie F. Foley, N. Martin, Rick D. Fannin, Wesley Gladwell, Kevin E. Gerrish
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Survival of patients with metastatic castration-resistant prostate cancer (mCRPC) depends on the site of metastatic dissemination. Methods Patients with mCRPC were prospectively included in the CPCT-02 metastatic site biopsy study. We evaluated whole genome sequencing (WGS) of 378 mCRPC metastases to understand the genetic traits that affect metastatic site distribution. Results Our findings revealed that RB1, PIK3CA, JAK1, RNF43, and TP53 mutations are the most frequent genetic determinants associated with site selectivity for metastatic outgrowth. Furthermore, we explored mutations in the non-coding genome and found that androgen receptor (AR) chromatin binding sites implicated in metastatic prostate cancer differ in mutation frequencies between metastatic sites, converging on pathways that impact DNA repair. Notably, liver and visceral metastases have a higher tumor mutational load (TML) than bone and lymph node metastases, independent of genetic traits associated with neuroendocrine differentiation. We found that TML is strongly associated with DNA mismatch repair (MMR)-deficiency features in these organs. Conclusions Our results revealed gene mutations that are significantly associated with metastatic site selectivity and that frequencies of non-coding mutations at AR chromatin binding sites differ between metastatic sites. Immunotherapeutics are thus far unsuccessful in unselected mCRPC patients. We found a higher TML in liver and visceral metastases compared to bone and lymph node metastases. As immunotherapeutics response is associated with mutational burden, these findings may assist in selecting mCRPC patients for immunotherapy treatment based on organs affected by metastatic disease. Trial registration number NCT01855477. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01445-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8017838415dd6bdeb817e19ffdcf7da3197701" target='_blank'>
              Whole genome sequencing of 378 prostate cancer metastases reveals tissue selectivity for mismatch deficiency with potential therapeutic implications
              </a>
            </td>
          <td>
            Daniel J Vis, S. Palit, Marie Corradi, Edwin Cuppen, Niven Mehra, M. Lolkema, L. Wessels, M. S. van der Heijden, W. Zwart, A. Bergman
          </td>
          <td>2025-03-20</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b08a5b6171c80002da0a1517c54be1999c70c6" target='_blank'>
              MED12 dysregulation: insights into cancer and therapeutic resistance.
              </a>
            </td>
          <td>
            R. Bhole, Jagruti Shinkar, Sonali Labhade, Pawan N Karwa, Harshad S. Kapare
          </td>
          <td>2025-03-19</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728e53382096a103c9861e4f20d349955371bb" target='_blank'>
              Abstract 2726: A circadian regulator drives tumor lineage plasticity through chromatin bivalency resolution
              </a>
            </td>
          <td>
            Yatian Yang, Xiong zhang, hongye zou, E. Corey, Ronald M. Evans, Amina Zoubeidi, Hongwu Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4672987b5633b49ff2536422175c3660e10c3a" target='_blank'>
              Identification of a shared persistence program in triple-negative breast cancer across treatments and patients
              </a>
            </td>
          <td>
            Léa Baudre, Grégoire Jouault, Pacôme Prompsy, Mélissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, C. Vallot
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9d728476d907142dd31d1c87ffc2d2c3f782be" target='_blank'>
              Abstract 1248: Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment
              </a>
            </td>
          <td>
            Yuwei Song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca6f30d3b9e64a4de714f44e69384d7201b4a8d" target='_blank'>
              Abstract 440: Dual epigenetic targeting of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kyounghyun Kim, Ponmari Guruvaiya
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/094980891c477076ac9ee63d92403d7dd49a9f93" target='_blank'>
              Abstract 5506: Inactivation of mTOR is a strategy for tumor resistance to ribosome biogenesis inhibition
              </a>
            </td>
          <td>
            Wenjun Fan, Lijing Yang, Hester Liu, Steffie Pitts, Niladri Sinha, N.V. Rajeshkumar, H. Easwaran, Rachel Green, M. Laiho
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc2b803c7ac7b99f1e6df1e4aac1c92682211a5" target='_blank'>
              Abstract 5557: Deciphering the interplay of β-catenin/YAP signaling and alveolar lineage plasticity during targeted therapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Yu-Ting Chou, Wei Wu, D. Kerr, V. Olivas, Macey Slota, T. Bivona
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Lung cancer is a major cause of cancer-related deaths globally. Targeted therapies, specifically attacking cancer cells based on genetic mutations, offer promising alternatives. ALK (anaplastic lymphoma kinase) fusions result in aberrant proteins that drive cancer growth. Drugs like crizotinib and ceritinib have shown efficacy in treating ALK-positive NSCLC. Accurate detection of ALK fusions is crucial for guiding these therapies. We conducted a retrospective analysis of a Chinese cohort of 131 ALK rearrangement-positive patients detected by DNA NGS between January 2017 and December 2021. Among those 131 ALK fusions, RNA-NGS confirmed positive transcripts in 88% of canonical ALK fusions and 75% of ALK fusions with rare partners in samples sequenced by both DNA NGS and RNA NGS. The secondary classification approach increased transcript prediction accuracy to 95.4% when combining common breakpoints and inframe fusion analysis in canonical ALK fusions. Combining rare breakpoints and inframe fusion could increase transcript prediction accuracy to 100%. For ALK fusions with rare partners, combining common breakpoints and frameshift improved transcript prediction accuracy to 100%. Additionally, combining rare breakpoints with inframe or frameshift could enhance the prediction accuracy to 100%. Combining DNA NGS and RNA NGS with a secondary classification approach significantly enhances the transcript prediction accuracy at the RNA level. This method optimizes clinical diagnostic and therapeutic strategies for ALK-positive NSCLC, highlighting the importance of advanced sequencing techniques in precision oncology. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92590-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34804a6772ac12af08ec23d6c1744145feb9ff1c" target='_blank'>
              Combined utility of genomic breakpoints and frame is a reliable predictor of ALK transcript function
              </a>
            </td>
          <td>
            Qiong Yang, Yumin Guo, Weihui Guo, Dongdong Liu, Haitao Wang, Xufan Cai
          </td>
          <td>2025-03-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) play crucial roles in colorectal cancer (CRC) development and progression. Recent developments in single-cell transcriptome profiling methods have revealed surprising levels of expression variability among seemingly homogeneous cells, suggesting the existence of many more cell types than previously estimated. This review synthesizes recent advances in ncRNA research in CRC, emphasizing single-cell bioinformatics approaches for their analysis. We explore computational methods and tools used for ncRNA identification, characterization, and functional prediction in CRC, with a focus on single-cell RNA sequencing (scRNA-seq) data. The review highlights key bioinformatics strategies, including sequence-based and structure-based approaches, machine learning applications, and multi-omics data integration. We discuss how these computational techniques can be applied to analyze differential expression, perform functional enrichment, and construct regulatory networks involving ncRNAs in CRC. Additionally, we examine the role of bioinformatics in leveraging ncRNAs as diagnostic and prognostic biomarkers for CRC. We also discuss recent scRNA-seq studies revealing ncRNA heterogeneity in CRC. This review aims to provide a comprehensive overview of the current state of single-cell bioinformatics in ncRNA CRC research and outline future directions in this rapidly evolving field, emphasizing the integration of computational approaches with experimental validation to advance our understanding of ncRNA biology in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df0d488fffc7d97d7489f557a3f2dfce15740d80" target='_blank'>
              Single-Cell Transcriptomic Approaches for Decoding Non-Coding RNA Mechanisms in Colorectal Cancer
              </a>
            </td>
          <td>
            M. Gondal, H. Farooqi
          </td>
          <td>2025-03-10</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01f5cfa8e398963dd5157942872b501bf3ce27d8" target='_blank'>
              Abstract 442: Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance
              </a>
            </td>
          <td>
            Manav Gupta, Pranav Gupta, Scott R. Tyler, N. Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, I. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel L. Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, J. Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca M. Choy, Richard Zang, Linghu Xin, Yuchen Bai, P. Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, J. Taygerly, Jacob Burch-Konda, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Bone metastases represent frequent and severe complications in various cancers, notably impacting prognosis and quality of life. This review article delves into the genetic and epigenetic mechanisms underpinning drug resistance in bone metastases, a key challenge in effective cancer treatment. The development of drug resistance in cancer can manifest as either intrinsic or acquired, with genetic heterogeneity playing a pivotal role. Intrinsic resistance is often due to pre-existing mutations, while acquired resistance evolves through genetic and epigenetic alterations during treatment. These alterations include mutations in driver genes like TP53 and RB1, epigenetic modifications such as DNA methylation and histone changes, and pathway alterations, notably involving RANK-RANKL signaling and the PI3K/AKT/mTOR cascade. Recent studies underline the significance of the tumor microenvironment in fostering drug resistance, with components such as cancer-associated fibroblasts and hypoxia playing crucial roles. The interactions between metastatic cancer cells and the bone microenvironment facilitate survival and the proliferation of drug-resistant clones. This review highlights the necessity of understanding these complex interactions to develop targeted therapies that can overcome resistance and improve treatment outcomes. Current therapeutic strategies and future directions are discussed, emphasizing the integration of genomic profiling and targeted interventions in managing bone metastases. The evolving landscape of genetic research, including the application of next-generation sequencing and CRISPR technology, offers promising avenues for novel and more effective therapeutic strategies. This comprehensive exploration aims to provide insights into the molecular intricacies of drug resistance in bone metastases, paving the way for improved clinical management and patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed3b250a10d84b32555ed5529bedbf646475e86e" target='_blank'>
              The genetic architecture of bone metastases: unveiling the role of epigenetic and genetic modifications in drug resistance
              </a>
            </td>
          <td>
            Ahmad Dawalibi, Mohamad Bakir, Khalid S. Mohammad
          </td>
          <td>2025-04-22</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ac37cc5cb3a2b919750ef1ac7fd1c5b0996b8dd" target='_blank'>
              Abstract 2487: LINMAP: A framework for comprehensive cell division lineage reconstruction via molecular barcoding and machine learning
              </a>
            </td>
          <td>
            Lin Wang, Zheng Yin, Wenjuan Dong, J. Sheng, Li Yang, Kun Han, Dingcheng Gao, Stephen T. Wong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aecf1e2b76e187b7e7357a8c7f6d61cd00b2bcd" target='_blank'>
              Abstract 7503: Detection of genomic alterations in lenvatinib-resistant liver cancer cell lines
              </a>
            </td>
          <td>
            Junghyun Kim, Ji Young Kim, Gamin Han, BoYoon Kim, Hoon Kim, Yeup Yoon, Wonseok Kang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent clinical trials have suggested that solid cancers with mismatch repair (MMR) deficiency are highly responsive to immunotherapy, regardless of cancer types. Previous MMR-related studies on breast cancer have predominantly focused on germline variants. However, the somatic MMR alterations have not been comprehensively characterized in breast cancer. In this study, we integrated genomic, transcriptomic, and clinical data from over 3000 breast cancer cases across six public cohorts. Our findings revealed that 1.2% of breast cancers harbored oncogenic somatic MMR alterations, with triple-negative breast cancer (TNBC) demonstrating the highest mutation rate at 3.1%. Additionally, somatic MMR alterations were significantly associated with microsatellite instability-high (MSI-H) and MMR-related mutational signatures, indicating that somatic MMR alterations led to impaired function of the MMR system. Biallelic inactivation of MMR genes resulted in a more pronounced loss of MMR function compared to monoallelic inactivation. Importantly, these MMR alterations significantly increased the tumor mutational burden (TMB) and neoantigen load in breast cancer, regardless of MSI-H status. These findings indicate that the frequency of MMR alterations is highest in TNBC and that MMR alterations in breast cancer can lead to MMR functional deficiencies, suggesting that some patients harboring such alterations may benefit from immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6303d9f75dded8dc24ee6a6e2566fac41f5275" target='_blank'>
              Comprehensive Analysis of Oncogenic Somatic Alterations of Mismatch Repair Gene in Breast Cancer Patients
              </a>
            </td>
          <td>
            Yin Yan, Yang Wang, Junjie Tang, Xiao-ran Liu, Jichuan Wang, Guohong Song, Huiping Li
          </td>
          <td>2025-04-01</td>
          <td>Bioengineering</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9d10ba962b31f50d67da52456440cdb7f8eb0c" target='_blank'>
              Abstract 3880: Functional dissection of the highly plastic basal cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G. Christensen, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37bbf9a9279b9ea882191a6ab180a2f1195c66ee" target='_blank'>
              Abstract 1246: Driver and signaling cascade in therapy pressure driven cancer cell plasticity
              </a>
            </td>
          <td>
            Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, N. Aravindan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Somatic mutations that increase hematopoietic stem cell (HSC) fitness drive their expansion in clonal hematopoiesis (CH) and predispose to blood cancers. Although CH frequently occurs with aging, it rarely progresses to overt malignancy. Population variation in the growth rate and potential of mutant clones suggests the presence of genetic factors protecting against CH, but these remain largely undefined. Here, we identify a non-coding regulatory variant, rs17834140-T, that significantly protects against CH and myeloid malignancies by downregulating HSC-selective expression and function of the RNA-binding protein MSI2. By modeling variant effects and mapping MSI2 binding targets, we uncover an RNA network that maintains human HSCs and influences CH risk. Importantly, rs17834140-T is associated with slower CH expansion rates in humans, and stem cell MSI2 levels modify ASXL1-mutant HSC clonal dominance in experimental models. These findings leverage natural resilience to highlight a key role for post-transcriptional regulation in human HSCs, and offer genetic evidence supporting inhibition of MSI2 or its downstream targets as rational strategies for blood cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/231338d317a31d1b141aef89e6f810d3087be9ba" target='_blank'>
              Inherited resilience to clonal hematopoiesis by modifying stem cell RNA regulation
              </a>
            </td>
          <td>
            Gaurav Agarwal, Mateusz Antoszewski, Xueqin Xie, Y. Pershad, U. Arora, Chi-Lam Poon, Peng Lyu, Andrew J. Lee, Chun-Jie Guo, Tianyi Ye, Laila Barakat Norford, Anna-Lena Neehus, Lucrezia della Volpe, Lara Wahlster, Diyanath Ranasinghe, Tzu-Chieh Ho, T. Barlowe, Arthur Chow, A. Schurer, James Taggart, Benjamin H. Durham, Omar Abdel-Wahab, Kathy L. McGraw, James M Allan, Ruslan Soldatov, Alexander G. Bick, M. Kharas, V. G. Sankaran
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1712552094bb20213ab61e8b311fc4eebd77f92d" target='_blank'>
              Epigenetic Regulation of Cellular Senescence in Gastrointestinal Cancer.
              </a>
            </td>
          <td>
            Ting Ye, Hang Gao, Zheng-Rong Zhang, Yang Ge, Yi-Ke Liu, Jing-Yuan Yan, De-Yi Li, Feng-Yuan Chen, Hang Song
          </td>
          <td>2025-03-25</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef1937cc4baf82f21bbced66813abdd14ff16f89" target='_blank'>
              Application and research of genomic optical mapping technology in disease diagnosis.
              </a>
            </td>
          <td>
            Jing Quan, Yan-Qun Xiao, Da-Ru Lu, Yun Bao
          </td>
          <td>2025-04-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b444db55c3cbfb978e50dbc339c5056a87181415" target='_blank'>
              Preclinical targeting of leukemia-initiating cells in the development future biologics for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Jiaxin Dong, Marina Konopleva
          </td>
          <td>2025-04-30</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7826ec8e0e8cb2ddf7c9f777c6ebf6c3f9a237fa" target='_blank'>
              Advancing chronic myeloid leukemia research with next-generation sequencing: potential benefits, limitations, and future clinical integration.
              </a>
            </td>
          <td>
            Henry Sutanto, Laras Pratiwi, P. Romadhon, S. U. Y. Bintoro
          </td>
          <td>2025-04-21</td>
          <td>Human genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1582331698256f81dcbd70eb9ab64e969d89900e" target='_blank'>
              Abstract 5664: Identifying novel vulnerabilities in clear cell sarcoma through functional and genomic screening
              </a>
            </td>
          <td>
            Laura Leuenberger, Larissa Isenegger, Yanjiang Chen, Miles Burri, Sarah Kollar, Lara Planas-Paz, Peter Karl Bode, A. Wozniak, Sebastian Bauer, Patrick Schöffski, Chantal Pauli, Christian Britschgi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eb70f970ca5d62a053191099abf0f49d770733" target='_blank'>
              Heterogeneous therapy-resistant cancer cells have distinct and exploitable drug sensitivity profiles
              </a>
            </td>
          <td>
            Gianna T. Busch, Ryan H. Boe, Jingxin Li, Robert F. Gruener, Miles Arnett, Pavithran T. Ravindran, M. Herlyn, R. Stephanie Huang, Arjun Raj
          </td>
          <td>2025-04-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2d4cdd26b00d23c9f2af55b08e343fcfbd17a4" target='_blank'>
              Abstract 1469: Deciphering mechanisms of intrinsic radioresistance in breast cancer: Insights from transcriptomics and CRISPRi screens
              </a>
            </td>
          <td>
            B. McBean, Priyanka S. Rana, Benjamin Hauk, A. Michmerhuizen, S. Lichtman-Mikol, Reine Abou Zeidane, Vesna Markovic, K. Wilder-Romans, Mingfang Tao, Alan P. Boyle, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9213a2dd4a654c0cc05ece63777d6864d7e262c3" target='_blank'>
              Role of Histone Deacetylases in Drug-Resistant Melanoma: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            B. Kalal
          </td>
          <td>2025-04-21</td>
          <td>Kinases and Phosphatases</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acaaf06b5fb2fb1345caa571c60786539fe457bf" target='_blank'>
              Somatic TP53 Mutations Drive T and NK Cell Dysfunction in AML and Can be Rescued by Reactivating Wild Type p53
              </a>
            </td>
          <td>
            Li Li, M. Muftuoglu, Edward Ayoub, Jiangxing Lv, Mahesh Basyal, A. Bidikian, Ran Zhao, Menon Prashant, Poo Yee Mak, Rupesh K. Kesharwani, Yuki Nishida, G. Issa, N. Varadarajan, B. Carter, Michael Andreeff
          </td>
          <td>2025-04-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction: Hi-C is a widely used technique for mapping chromosomal interactions within a 3D genomic framework, however, its resolution is often constrained by sequencing depth, making it challenging to detect fine-scale interactions. To overcome this limitation, Promoter-Capture Hi-C (PCHi-C), as it selectively enriches for promoter-associated interactions, was employed. This study integrates PCHi-C and Hi-C datasets from colorectal cancer (CRC) models investigate chromosomal interaction dynamics across various regulatory levels, from cis-regulatory elements to topologically associated domains (TADs). The primary goal is to examine how genomic structural alterations shape the epigenomic landscape in CRC and to assess their potential role in colorectal cancer susceptibility. Methods: PCHi-C and Hi-C datasets from multiple colorectal cancer (CRC) studies were integrated to enhance the resolution of chromatin interaction mapping. The analysis focused on identifying fine-scale interactions within topologically associated domains (TADs) while incorporating histone modification landscapes (H3K27ac, H3K4me3) and transcriptomic signatures from CRC cell lines and the TCGA database. For experimental validation, ChIP-quantitative PCR was performed at the promoters of target genes using the highly malignant colorectal cell line HT29 and compared it to an embryonic cell line NT2D1. Results: Our integrated analysis revealed significant genomic structural instability in CRC cells, closely associated with tumor-suppressive transcriptional programs. We identified nine dysregulated genes, including long non-coding RNAs (MALAT1, NEAT1, FTX, and PVT1), small nucleolar RNAs (SNORA26 and SNORA71A), and protein-coding genes (TMPRSS11D, TSPEAR, and DSG4), all of which exhibited a substantial increase in expression in CRC cell lines compared to human embryonic stem cells (hESCs). Additionally, we observed enriched activation-associated histone modifications (H3K27ac and H3K4me3) at the potential enhancer regions of these genes, indicating possible transcriptional activation. ChIP-quantitative PCRs conducted using in the highly malignant CRC cell line HT29, compared to the embryonic cell line NT2D1, further validated these findings, reinforcing the link between altered chromosomal interactions and gene dysregulation in CRC. Discussion: This study sheds light on the dynamic 3D genome organization in CRC, highlighting critical structural changes associated with disease-associated loci. The identification of nine dysregulated genes points to potential biomarkers for colorectal cancer, with implications for diagnostic and therapeutic strategies. The combination of Hi-C and PCHi-C offers a refined approach for detecting chromosomal interactions at a higher resolution, laying the foundation for future studies on cancer-associated chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79c3094833c1e043ac48c473ef6e13b0033ea5bf" target='_blank'>
              Integrated promoter-capture Hi-C and Hi-C analysis reveals fine-tuned regulation of the 3D chromatin architecture in colorectal cancer
              </a>
            </td>
          <td>
            A. K. Saw, Ayush Madhok, Anupam Bhattacharya, Soumyadeep Nandi, Sanjeev Galande
          </td>
          <td>2025-03-28</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Heterogeneity is inherent to living organisms and it determines cell fate and phenotypic variability. Despite its ubiquity, the underlying molecular mechanisms and the genetic basis linking genotype to-phenotype heterogeneity remain a central challenge. Here we construct a yeast knockout library with a clone and genotype RNA barcoding structure suitable for genome-scale analyses to generate a high-resolution single-cell yeast transcriptome atlas of 3500 mutants under control and stress conditions. We find that transcriptional heterogeneity reflects the coordinated expression of specific gene programs, generating a continuous of cell states that can be responsive to external insults. Cell state plasticity can be genetically modulated with mutants that act as state attractors and disruption of state homeostasis results in decreased adaptive fitness. Leveraging on intra-genetic variability, we establish that regulators of transcriptional heterogeneity are functionally diverse and influenced by the environment. Our multimodal perturbation-based single-cell Genotype-to-Transcriptome Atlas in yeast provides insights into organism-level responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89465a1373d3260110cc521e62c1032cd2c67144" target='_blank'>
              A single-cell resolved genotype-phenotype map using genome-wide genetic and environmental perturbations
              </a>
            </td>
          <td>
            Mariona Nadal-Ribelles, Carme Solé, Anna Díez-Villanueva, Camille Stephan-Otto Attolini, Yaima Matas, Lars M. Steinmetz, Eulàlia de Nadal, F. Posas
          </td>
          <td>2025-03-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c8de13c6fd974bf26d03a422a6ee56b54180c3" target='_blank'>
              Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, F. Cidre-Aranaz, A. Wenninger-Weinzierl, M. Zylka, Karla Queiroz, D. Kurek, A. Sastre, Javier Alonso, Martin Distel, Anna Obenauf, T. G. Grünewald, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Drug resistance results in poor outcomes for cancer patients. Adaptive therapy is a potential strategy to address drug resistance that exploits competitive interactions between sensitive and resistant subclones. Here, we showed that adapting carboplatin dose according to tumor response (adaptive therapy) significantly prolonged survival of murine ovarian cancer models compared to standard carboplatin dosing, without increasing mean daily drug dose or toxicity. Platinum resistant ovarian cancer cells exhibited diminished fitness when drug was absent in vitro and in vivo, which caused selective decline of resistant populations due to reduced proliferation and increased apoptosis. Conversely fitter, sensitive cells re-grew when drug was withdrawn. Using a bioinformatics pipeline that exploits copy number changes to quantify the emergence of treatment resistance, analysis of cell-free DNA obtained longitudinally from ovarian cancer patients during treatment showed subclonal selection through therapy, and measurements of resistant population growth correlated strongly with disease burden. These preclinical findings pave the way for future clinical testing of personalized adaptive therapy regimens tailored to the evolution of carboplatin resistance in individual ovarian cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872bfbc7aed3e8f94862c1f84bd0f838583a3b38" target='_blank'>
              Adaptive Therapy Exploits Fitness Deficits in Chemotherapy-Resistant Ovarian Cancer to Achieve Long-Term Tumor Control.
              </a>
            </td>
          <td>
            H. Hockings, E. Lakatos, Weini Huang, M. Mossner, Mohammed A Khan, Nikolina Bakali, Jacqueline McDermott, Kane Smith, Ann-Marie Baker, Trevor A. Graham, M. Lockley
          </td>
          <td>2025-04-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0da8bbe26d7b85bc1bd170b1562ddebfb5b49ba9" target='_blank'>
              Abstract 6382: DNA damage repair pathways enable a drug-tolerant persister state in RET-fusion NSCLC and precedes TKI resistance
              </a>
            </td>
          <td>
            F. Uddin, Samuel Tischfield, R. Somwar, Alexander Lim, Hyung Jun Woo, Christina J Falcon, Barbara P. Mello, Dennis Kinyua, Harsha Sridhar, P. Manoj, Hong Zhong, Juan Qiu, E. de Stanchina, M. Donoghue, Marc Ladanyi, E. Redin, Charles M. Rudin, Á. Quintanal-Villalonga, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58e72b48369a6d6279bf45ff8835cc869b1ce39" target='_blank'>
              CRISPR screens in the context of immune selection identify CHD1 and MAP3K7 as mediators of cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Alex Watterson, G. Picco, Vivien Veninga, Youhani Samarakoon, C. M. Cattaneo, Sara F Vieira, Emre Karakoc, S. Bhosle, Thomas W Battaglia, Sarah Consonni, Timotheus Y. F. Halim, E. Voest, Mathew J. Garnett, Matthew A. Coelho
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13bcea3579836571808ac96de5e2e45ad09f6441" target='_blank'>
              Abstract 6418: Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma
              </a>
            </td>
          <td>
            Varuna Nangia, Humza Ashraf, Nasreen Marikar, Victor J. Passanisi, Christopher Ryland Ill, Timothy E. Hoffman, Sabrina L. Spencer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43978659ce38ef19fbe0b48f8cd99b848f7e9b3" target='_blank'>
              Abstract 7490: Uncovering rare cellular populations in esophageal preneoplasia through genetic perturbation tracking and scRNA-seq
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Jae-Il Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b24fd08026b81ce4d7ce4581da39b975952905" target='_blank'>
              Abstract 5108: Decoding tumor intrinsic and stroma-related drug resistance in EGFR mutant NSCLC
              </a>
            </td>
          <td>
            Radhika Koranne, Aimee Gleason, Emily Hensley, Xiaodong Wang, Alba Font-Tello, Brian Rabe, Fayaz Seifuddin, Eugene Drokhlyansky, Kartik Subramanian, Martino Colombo, Jason R. Dobson, Mavrakis J. Konstantinos, Leila Dardaei, Aaron N. Hata
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60dc48b94558a55ad5ff2dbcdf85170057c59b44" target='_blank'>
              Natural resistance to cancers in long-lived mammals: genomic mechanisms and experimental evidence to explain Peto's paradox.
              </a>
            </td>
          <td>
            Linxia Sun, Zhikang Xu, Mengqi Shuai, Chengxu Li, Guang Yang, Shixia Xu
          </td>
          <td>2025-03-21</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is the most common type of acute leukemia, accounting for 20% of cases in children and adolescents. Genome-wide studies have identified genes that are commonly mutated in AML, including many epigenetic regulators involved in either DNA methylation (DNMT3A, TET2, IDH1/2) or histone post-translational modifications (ASXL1, EZH2, MLL1). Several cell lines derived from AML patients are widely used in cancer research. Whether important differences in these cell lines exist remains poorly characterised. Results Here, we used RNA sequencing (RNA-Seq) to contrast the transcriptome of four commonly used AML-derived cell lines: THP-1, NOMO-1, MOLM-13 bearing the common initiating t(9;11) translocation, and MV4.11 bearing the t(4;11) translocation. Gene set enrichment analyses and comparison of key transcription and epigenetic regulator genes revealed important differences in the transcriptome, distinguishing these AML models. Among these, we found striking differences in the expression of clusters of genes located on chromosome 19 encoding Zinc Finger (ZNF) transcriptional repressors. Low expression of many ZNF genes within these clusters is associated with poor survival in AML patients. Conclusion The present study offers a valuable resource by providing a detailed comparative characterisation of the transcriptome of cell lines within the same AML subtype used as models for leukemia research. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11415-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a934cae686db7528797f8dea84b54367fe76eeb" target='_blank'>
              Transcriptomic characterisation of acute myeloid leukemia cell lines bearing the same t(9;11) driver mutation reveals different molecular signatures
              </a>
            </td>
          <td>
            Elise Georges, William Ho, Miren Urrutia Iturritza, L. Eory, Kamila Malysz, U. Sobhiafshar, Alan L Archibald, Daniel J. Macqueen, Barbara Shih, David Garrick, Douglas Vernimmen
          </td>
          <td>2025-03-25</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Background Lynch syndrome (LS), caused by germline pathogenic variants in the mismatch repair genes, leads to high rates of frameshift-derived neopeptide (FSDN) expression due to microsatellite instability (MSI). While colorectal cancer (CRC) prevention is effective, most LS-related tumors lack such strategies. Cancer vaccines targeting FSDNs offer a promising approach for immune interception in LS. This study aimed to identify and validate LS-related FSDNs to develop vaccines for cancer prevention. Methods We identified LS-related coding MS mutations and predicted FSDN with high coverage on common Human Leukocyte Antigen (HLA)-I and II alleles. We validated FSDN-associated mutations in colorectal adenomas (CrAD), endometrial cancers (EC), and CRC samples from patients with LS, non-LS tumors, and cell lines. Immunogenicity was assessed through interferon (IFN)-γ enzyme-linked immunospot and flow cytometry analysis of tissue-infiltrating lymphocytes (TILs) from LS carriers. Results We prioritized 53 HLA-I and 45 HLA-II FSDNs in MSI tumors using in silico predictions. Validation revealed 86.7% of FSDN-associated mutations present in LS-CRC samples, with a median of 7.67 (6.5–9) mutations in CrADs and 6.02 (2–10) in CRCs. Sequencing of CrAD and EC samples showed 95% and 77.5% of predicted FSDN-associated mutations, respectively. MSI cancer cell lines transcribed 69.8% of FSDNs. Immunogenicity assays showed that 71% of potential FSDNs elicited IFN-γ responses, with a median of 7.37 (1–10.75) HLA-I and 6 (2–5.75) HLA-II FSDNs per patient. After prioritizing 24 FSDN, in a cohort of 19 LS-derived samples (4 CrAD and 15 normal mucosa), 52% (10/19) demonstrated T-cell reactivity to an HLA-I neoantigen pool. CD8+CD137+ activation markers increased significantly (p=0.037) over time and peptide-specific cells were detected by pentamer staining. Conclusions Our predicted FSDN set has optimal coverage among LS carriers and can induce IFN-γ inflammatory responses in LS-derived TILs, offering an opportunity for vaccine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4e379b9e444406c0c4ab08355c1d7dbd836a26" target='_blank'>
              Discovery and validation of frameshift-derived neopeptides in Lynch syndrome: paving the way for novel cancer prevention strategies
              </a>
            </td>
          <td>
            Cristina Bayó, Giancarlo Castellano, Fátima Marín, J. Castillo-Iturra, T. Ocaña, Hardeep Kumari, M. Pellisé, Leticia Moreira, L. Rivero, M. Daca-Alvarez, Oswaldo Ortíz, S. Carballal, Rebeca Moreira, Júlia Canet-Hermida, Marta Pineda, Capella Gabriel, Georgina Flórez-Grau, Manel Juan, D. Benitez-Ribas, Francesc Balaguer
          </td>
          <td>2025-04-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Establishing DNA as the genetic material, deciphering the genetic code and defining the double helical structure of the genome provided a foundation for pursuing a mechanistic understanding of gene expression that is operative for biological control and pathology. Equally important are a series of discoveries that have established epigenetic mechanisms, defined as the control of gene expression by non-DNA-encoded regulation, mediated by an evolving histone code that supports chromatin states, as complementary and obligatory determinants for biological activity. Evidence is accruing for pivotal contributions by epigenetic parameters of control to gene expression that is required for development and differentiation as well as for transient and sustained responsiveness to physiological regulatory cues that support cell structure, function, cell survival, tissue repair and remodeling. Epigenetic control is emerging as a decisive contributor to the regulatory compliance for the highly integrative processes that balance phenotypic function and tumor suppression with aberrant gene expression that accompanies the onset and progression of disease, strikingly illustrated with tumorigenesis and skeletal disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4c2c01eadaece3705383df803332330abc2445" target='_blank'>
              Reflections and perspectives on epigenetically mediated biological control: compromises in cancer and skeletal pathology
              </a>
            </td>
          <td>
            Gary S. Stein
          </td>
          <td>2025-04-11</td>
          <td>Academia Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a21c97e108e69ab358170dee175b96d9645c472" target='_blank'>
              Cancer Dependency Map: Past and Present
              </a>
            </td>
          <td>
            Rand Arafeh, T. Shibue, Francisca Vazquez, William C Hahn
          </td>
          <td>2025-04-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/572fd6e1dc6d845d6923c7249ae200e2136415b1" target='_blank'>
              Genome-wide transcriptomic response of whole blood to radiation
              </a>
            </td>
          <td>
            Ahmed Salah, D. Wollschläger, Maurizio Callari, H. Schmidberger, Federico Marini, S. Zahnreich
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab0fc38af5c962f28721cc54ecd1853477def0" target='_blank'>
              Single cell multi-omic whole genome sequencing and chromatin accessibility profiling reveals genome-epigenome coevolution in colorectal cancer
              </a>
            </td>
          <td>
            M. Mossner, Chloé Colson, Qingli Guo, Freddie Whiting, Heather E. Grant, Amanda Stafford, Peter Kyle, Daniella Donato-Brown, Jamie Murphy, A. Sottoriva, Ann-Marie Baker, Trevor A. Graham
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874bff63660c812dc6306123212ce70b86ccda05" target='_blank'>
              Abstract 6789: Identification of DGKA as a regulatory factor for paclitaxel resistance in lung cancer based on CRISPR screen and transcriptome sequencing
              </a>
            </td>
          <td>
            Jiang Luo, Mini Zhang, Jiawen Tang, Qingwei Wang, Yu Yuan, Min Xia, Yufei Yan, Yushuang Tan, Conghua Xie, Yan Gong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background & Aims Patients with colorectal cancer have heterogeneous clinical responses to chemotherapy, although clinical guidelines advise little variability in treatment selection based on molecular tumor features. Precision oncology research typically utilizes patient-derived tumor organoids (PDTO) to predict clinical outcomes, but such efforts are often not directed towards identification of molecular factors underlying differential responses to therapy. Methods Bulk RNA-sequencing was performed on treatment-naive PDTOs, and gene expression data was combined to drug sensitivity data to identify transcriptomic features associated with low in vitro sensitivity to chemotherapy. Whole-exome sequencing was performed on primary tumors to infer the somatic mutations of PDTOs and used to identify somatic mutations associated with differential in vitro drug responses. Publicly available gene expression and drug sensitivity data sets were used to validate the results. RNA interference was used for functional validation. Results PDTOs with low chemosensitivity had high JAK-STAT pathway activity resulting from high expression of interferon-stimulated genes. Evidence from single-cell RNA-sequencing confirmed chemotherapy-induced expression of interferon-stimulated genes in epithelial cells of cancers with partial response. EPSTI1 knockdown decreased cancer cell viability and sensitized cells to chemotherapy. Conclusions Sustained interferon signaling in epithelial cancer cells contributes to incomplete pathologic response in colorectal cancer. The findings highlight the potential of JAK-STAT inhibition or TRAIL pathway activation to enhance chemotherapy efficacy. Future studies investigating pharmacologic modulation of these pathways in preclinical CRC models are needed to determine their viability as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ab87f4ae646ab5e983158fb38a2489ac5894d6" target='_blank'>
              Sustained epithelial interferon signaling modulates incomplete pathologic response in colorectal cancer
              </a>
            </td>
          <td>
            E. Erkan, Emmi K. Hämäläinen, Julia Kolikova, Kornelia Kuc, K. Ojala, Meiju Kukkonen, Ismaïl Hermelo, S. Koskensalo, T. Tarvainen, Alli Leppä, I. Keränen, Carola Haapamäki, E. Karjalainen, Monica Carpelan-Holmström, L. Renkonen-Sinisalo, Laura E Koskenvuo, P. Puolakkainen, J. Mecklin, Elina Virolainen, M. Nykter, L. Aaltonen, Anna H. Lepistö, Ari Ristimäki, Toni T Seppälä
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Background About 40% of relapsed or non-responder tumors exhibit therapeutic resistance in the absence of a clear genetic cause, suggesting a pivotal role of intracellular communication. A deeper understanding of signaling pathways rewiring occurring in resistant cells is crucial to propose alternative effective strategies for cancer patients. Methods To achieve this goal, we developed a novel multi-step strategy, which integrates high sensitive mass spectrometry-based phosphoproteomics with network-based analysis. This strategy builds context-specific networks recapitulating the signaling rewiring upon drug treatment in therapy-resistant and sensitive cells. Results We applied this strategy to elucidate the BCR::ABL1-independent mechanisms that drive relapse upon therapy discontinuation in chronic myeloid leukemia (CML) patients. We built a signaling map, detailing - from receptor to key phenotypes - the molecular mechanisms implicated in the control of proliferation, DNA damage response and inflammation of therapy-resistant cells. In-depth analysis of this map uncovered novel therapeutic vulnerabilities. Functional validation in patient-derived leukemic stem cells revealed a crucial role of acquired FLT3-dependency and its underlying molecular mechanism. Conclusions In conclusion, our study presents a novel generally applicable strategy and the reposition of FLT3, one of the most frequently mutated drivers of acute leukemia, as a potential therapeutic target for CML relapsed patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02185-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff332b1ad425972fa586920b8d83716a55d5548" target='_blank'>
              A network-based approach to overcome BCR::ABL1-independent resistance in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Valeria Bica, Veronica Venafra, Giorgia Massacci, Simone Graziosi, Sara Gualdi, G. Minnella, F. Sorà, P. Chiusolo, Maria Elsa Brunetti, G. Napolitano, Massimo Breccia, Dimitrios Mougiakakos, Martin Böttcher, Thomas Fischer, Livia Perfetto, Francesca Sacco
          </td>
          <td>2025-04-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a1965c162640161a9b607e38387b4fb02d284d" target='_blank'>
              Abstract 2104: Epigenetic, transcriptomic and genetic analyses of HR+/HER2- primary and metastatic breast cancer results in cohort stratification by aggressiveness and reveals heterogeneous mechanisms of endocrine therapy resistance
              </a>
            </td>
          <td>
            Jomar Sangalang, Tony Sourisseau, V. Scott, Diep Tran, Nusaibah Ibrahimi, A. Tran-Dien, Hakim Benkirane, N. Servant, Fabrice André, Sergey I. Nikolaev
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d329440f52845fe5e04ed56db29d5167fcf05d" target='_blank'>
              Abstract 6199: Interlab study supporting hereditary cancer testing in an NGS-focused landscape
              </a>
            </td>
          <td>
            Dana J. Ruminski Lowe, Deepika Philkana, K. Nahlik, Melissa Berenger, Andrew Anfora, Yves Konigshofer, Catherine Huang, R. Garlick, B. Anekella
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daad54b0fcb7c5c704adea069e134ecf7c7a856b" target='_blank'>
              Abstract 1252: Evolution of transcriptomic and epigenomic intra-tumor heterogeneity in high-grade serous ovarian cancer with chemotherapy
              </a>
            </td>
          <td>
            Yuna Landais, Juliette Bertorello, Marta Puerto, Baptiste Simon, Amélie Roehrig, Adeline Durand, Marceau Quatredeniers, C. Lamy, Fabrice Lécuru, Christophe Le Tourneau, Céline Vallot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT ALK fusions, such as the classic EML4‐ALK, are known drivers of lung cancer and effective therapeutic targets. However, variant ALK fusions, including intergenic fusions like LOC388942‐ALK (LA), have been detected in increasing numbers of patients, with their roles in tumorigenesis and ALK inhibitor resistance remaining unclear. Using CRISPR/Cas9, we generated the LA fusion in A549 and H441 cells, confirming elevated ALK expression via qRT‐PCR and immunohistochemistry (IHC) staining. Functional analyses showed that LA significantly promoted tumor growth in vitro and in vivo while conferring increased resistance to alectinib. RNA‐seq revealed upregulation of the FOS pathway in LA tumors, identifying FOS as a potential therapeutic target. Subsequently, we demonstrated that FOS disruption and inhibition sensitized LA tumors to treatment. RNA‐seq profiling demonstrated that FOS depletion in LOC388942‐ALK tumor significantly downregulated multiple oncogenic pathways related to cell cycle progression, DNA replication fidelity, and extracellular matrix remodeling, suggesting a pivotal role of FOS in maintaining tumor growth. These findings establish LOC388942‐ALK as a novel oncogenic driver in lung cancer, highlighting its role in tumor growth and ALK inhibitor resistance. Targeting FOS may provide a promising therapeutic strategy for tumors harboring this intergenic fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/459d6322ba2d77e523434dfb3f269f42f6deaabb" target='_blank'>
              Identifying the Intergenic ALK Fusion LOC388942‐ALK as a Driver of Non–Small Cell Lung Cancer
              </a>
            </td>
          <td>
            X. Zhai, Manli Wang, Qi Zhang, Donglin Li, Yanmou Wu, Zuoyu Liang, Jiewei Liu, Weiya Wang, Yu Liu, Guowei Che, Qinghua Zhou, Chong Chen
          </td>
          <td>2025-03-27</td>
          <td>MedComm</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Lynch Syndrome (LS) is an autosomal dominant disease characterized by germline heterozygous mutations in DNA mismatch repair (MMR) genes. High-risk LS patients may proceed to colorectal cancer (CRC). However, the drivers or biomarkers of LS benign colon tissue approaching malignant CRC are not completely understood. The similarity and difference between LS-related and nonLS-related CRC are also not well interrogated (LS-CRC vs. nonLS-CRC). This study aimed to understand the cellular changes during malignant transition in LS. Methods Single-cell RNA sequencing (scRNA-seq) was used to analyze paired biopsy samples from 3 patients with LS (cancer tissues vs. adjacent normal tissues, n=3). Single-nuclear RNA sequencing (snRNA-seq) was used to analyze a frozen biopsy sample from a patient with LS (n=1). scRNA-seq or snRNA-seq datasets from CRC were downloaded from the open source. Integrative computational analysis was performed to conclude the distinct pattern in the single-cell atlas. Immuno-histo-fluorescence staining (IHF) were also performed for three key markers. Results In the single-cell atlas, we observed an increase in primitive cancer stem-cell-like cells with high expression of a general cancer biomarker CarcinoEmbryonic Antigen-related Cell Adhesion Molecule 5 (CEACAM5) in the epithelium of the LS. Infiltration of immune cells and DNA repair biological activity are dramatically increased in LS carcinoma. The burden in LS is fundamentally elevated compared to that in CRC or healthy donors when the mutations are in the coding-sequence-wide range. Furthermore, T cell and macrophage-related tumor immunity is readily mobilized in carcinomas compared to paraCArcinomas. Conclusions This study provides single-cell transcriptomic resource using affected tissues from patients with Lynch Syndrome and describes an integrative profile covering the alterations (cancer stem cell markers, mutation burden, and tumor immunity) during the malignant transition from healthy to Lynch Syndrome and to colorectal cancer at the single-cell level. Highlights and Figure/Table Index (Take-Home Messages) Colon tissues from HD (n=4), patients with LS (n=6) and with CRC (n=3) were collected for snRNA-seq or scRNA-seq analysis; Diagnosis of the 6 LS patients are clearly supported by our pedigree documentations (Fig. 1 and Supp. Fig. 1) and briefed in Table 1; Among the 6 patients with LS, 1 for snRNA-seq analysis (a carcinoma tissue only), 3 for scRNA-seq analysis (3 paired para-carcinoma and carcinoma tissues) and 3 for experimental validations; (Fig. 1) Following studies are focused on the comparisons between HD, LS and CRC; In addition, comparisons between paired carcinoma and para-carcinoma tissues from patients with LS were also performed; (Fig. 2-7) When HD and LS are compared, a cancer stem cell marker CEACAM5 is readily detected by snRNA-seq analysis; (Fig. 5) When carcinoma and para-carcinoma from LS patients are compared, increased immune cell infiltration of and enhanced DNA repair activity in the malignant tissues is readily detected by scRNA-seq analysis; in addition, three upregulated markers in LS carcinoma (BACE2, GRPC5A and OLFM4) were identified and validated by immunohistoinfloresnce; (Fig. 3 and Fig. 4) SComatic algorithm-based mutation calling analysis suggests a comparable mutation burden between carcinoma and para-carcinoma from LS patients, but such mutation burdens are grossly greater than that from HD or that from patients with CRC; (Fig. 6)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863c44a229dbcaa552459faeccde9844a06701be" target='_blank'>
              Tracing the stemness and malignant transition in a heritable colorectal cancer Lynch Syndrome by single-cell RNA-seq analysis
              </a>
            </td>
          <td>
            Junfeng Xu, Yuhang Li, Zhiqin Wang, Qianru Li, Aijun Liu, Jianqiu Sheng, Ge Dong, Lang Yang, Zhigang Cai
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Germline variants in the HSD3B1 gene influence response to hormone therapies. This study examined the “adrenal-permissive” allele (c.1100 C) and its impact on various somatic features in prostate tumors. Upon inferring HSD3B1 variant status from somatic DNA sequencing data in a cohort of 6550 primary and metastatic prostate tumors, we found that tumors harboring the adrenal-permissive CC genotype, as compared to the adrenal-restrictive AA genotype, had limited somatic changes in their exomes or transcriptomes. However, significant differences were observed when we examined immune regulatory pathways and immune cell populations. This was accompanied by differential gene expression of immunoglobulins, MHC class I and II molecules, and druggable cell surface genes. These features appeared to deviate across different metastatic biopsy sites. Altogether, while tumor-intrinsic features are not regulated by germline HSD3B1 c.1100 alleles, these genotypes instead may impact cell–cell interactions and the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5953257502a6a5fc55756fab20f962c1af1666a8" target='_blank'>
              The Influence of the Germline HSD3B1 Adrenal-Permissive Allele (c.1100 C) on the Somatic Alteration Landscape, the Transcriptome, and Immune Cell Infiltration in Prostate Cancer
              </a>
            </td>
          <td>
            Samuel Kellen, Allison Makovec, Carly D Miller, Shayan Nazari, Andrew Elliott, Aiden Deacon, Emily John, Nikitha Vobugari, N. Agarwal, Rana R McKay, Pedro C. Barata, Charles J. Ryan, Nima Sharifi, Justin H. Hwang, E. S. Antonarakis
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Recent research has increasingly highlighted alterations in the proto-oncogene MET, whose abnormal activation has been implicated in multiple cancers. MET encodes c-MET, a receptor tyrosine kinase critical for cellular growth, survival, and migration. Aberrant c-MET signaling, driven by mutations or gene amplification, promotes proliferation and invasion, contributing to tumorigenesis. Scope of the Review: While MET mutations are most often observed in non-small cell lung cancer (NSCLC), they also occur in other malignancies, including breast and gastric cancers. This review highlights key MET alterations, such as gene amplification, gene fusions, and exon 14 skipping deletions, and examines their prevalence across various tumor types. Major Conclusions: We discuss the clinical significance of c-MET as a therapeutic target and identify gaps in knowledge that could inform the development of alternative treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00b01ff6f2bb032a063ee8283cbebc4ff3348662" target='_blank'>
              Targeting c-MET Alterations in Cancer: A Review of Genetic Drivers and Therapeutic Implications
              </a>
            </td>
          <td>
            Michelle Ji, Shridar Ganesan, Bing Xia, Yanying Huo
          </td>
          <td>2025-04-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32e4d3a24ff91016f9e42d7a576edaca46c0ba68" target='_blank'>
              Abstract 5339: Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer
              </a>
            </td>
          <td>
            Yuki Matsunaga, Emma Aleksandrovic, H. M. Patel, D. Sudhan, Khushi Ahuja, D. Ye, Chang-Ching Lin, Lei Guo, Cynthia Ma, Jeon Lee, Siyuan Zhang, Carlos L. Arteaga, A. Hanker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16cb06677804f48c2ac85f36bdf345c2b4df6264" target='_blank'>
              Abstract 431: Novel WRN drug resistant CDX models
              </a>
            </td>
          <td>
            Haiting Dai, Tiejun Bing, Huayi Qu, Huawei Pan, Wen-Jen Yu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654c576507b39c0969a5f79ea0daae0b983731a4" target='_blank'>
              Riding the Wave of Progress: Examining the Current Landscape and Future Potential of MicroRNAs in Cancer Gene Therapy.
              </a>
            </td>
          <td>
            Saleem Ahmad, Hanif Khan, Njoka Irene Muthoni, Mohammed Alissa, Awaji Y. Safhi, F. Y. Sabei, Osama Abdulaziz, Khadijah Hassan Khan, Essam H Ibrahim, S. Khan
          </td>
          <td>2025-03-24</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9adf9806a26a89ffa7e78b13710a5e6c2dc2cb20" target='_blank'>
              Abstract 5548: Single-cell transcriptome analysis reveals the role of myofibroblastic cancer-associated fibroblasts in chemotherapy resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Jiahao Guo, Samuele Cancellieri, Liangru Fei, T. Aittokallio, Biswajyoti Sahu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8354bb2cbdecdc87317ac3be8bce15b2494319c2" target='_blank'>
              Emerging Roles of ACTL6A as an Oncogenic Hub: Transcriptional Regulation and Beyond.
              </a>
            </td>
          <td>
            Kelvin K Tsai, Lin-Hsin Cheng, Chung-Chi Hsu, Pei-Ming Yang, Chin-Pin Chuu
          </td>
          <td>2025-04-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd1fbf6a111cb790d6d05a256a03567072e5318c" target='_blank'>
              AI-Driven Variant Annotation for Precision Oncology in Breast Cancer
              </a>
            </td>
          <td>
            Kriti Shukla, Yue Wang, Philip M. Spanheimer, Elizabeth Brunk
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aba9f5d2cf8bd1341acacd8ce88421b1a228c70" target='_blank'>
              Abstract 5547: High resolution target identification by unbiased forward genetics
              </a>
            </td>
          <td>
            Moritz Horn, Georg Michlits, Andreia Rocha, Christoph Stelzer, Josef M. Penninger
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774ca0f712c34facaa849dbeb856b7628d502178" target='_blank'>
              Precision Targeting in Metastatic Prostate Cancer: Molecular Insights to Therapeutic Frontiers
              </a>
            </td>
          <td>
            W. Kwon, Jae Young Joung
          </td>
          <td>2025-04-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/babbc9f6ed9c4e4469fe6161951a6c9c5723cbc6" target='_blank'>
              Abstract LB419: HDAC8 over-expression promotes drug resistance and alters chromatin architecture in melanoma cells
              </a>
            </td>
          <td>
            Kanchan Gupta, Richard L Bennett, M. Emmons, Yufeng Xiao, Guangrong Zheng, K. Smalley, Jonathan D. Licht
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f536aa81e01a0a6a9e8c079c861f407b92860a94" target='_blank'>
              Abstract 370: Resistance to sotorasib does not confer resistance to adagrasib: Why?
              </a>
            </td>
          <td>
            Atish Mohanty, Baiyi Quan, Supriyo Bhattacharya, Dana Do, S. Ramisetty, Tsui-Fen Chou, John Orban, Prakash Kulkarni, Ravi Salgia
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/341c46300ffe02510e6a7b891ff511cce4303ab8" target='_blank'>
              Abstract 1068: Navigating next-generation pre-clinical models of drug discovery & development with RNA-seq
              </a>
            </td>
          <td>
            Ajeet P. Singh, Rula R. Khairi, Amy L. Reese, Jade L Kirkland, James Duncan, Noah Wax, Robert Marlow, Steve King, Ana Fernandes, Jonathan L. Jacobs
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Radiotherapy is a cornerstone of colorectal cancer (CRC) treatment; however, its therapeutic efficacy is often compromised by both intrinsic and acquired resistance in CRC cells. This study employed small interfering RNA (siRNA) technology to elucidate the functional roles of BAMBI, GADD34, NFKBIA, and NFKBID in CRC cell lines SW480 and HCT116. We assessed their impact on key cellular processes and radiation sensitivity. Gene silencing of all four target genes significantly suppressed CRC cell proliferation, migration, and invasion. Moreover, siRNA-mediated knockdown enhanced radiation sensitivity, as evidenced by a substantial increase in apoptosis and a marked reduction in cell viability compared with controls. These findings suggest that BAMBI, GADD34, NFKBIA, and NFKBID serve as critical regulators of CRC progression and radiation resistance. Overall, this study provides a mechanistic foundation for further exploration into the pathways underlying radiation resistance and underscores the potential for developing personalized radiotherapy strategies guided by molecular profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b02786eb68733b852aeed62e7695e4393e02cbb" target='_blank'>
              Mechanistic Insights into Radiation Resistance in Colorectal Cancer: Gene Exploration Study
              </a>
            </td>
          <td>
            Beamjun Park, Soohyeon Lee, Inyoung Jo, Donghyun Kang, Taewan Kim, Jaesung Ryu, Hyejeong Kong, Moojun Baek, Taesung Ahn
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background Genetic testing and sequencing technologies offer a comprehensive understanding of cancer genetics, providing rapid and cost‐effective solutions. In particular, these advanced technologies play an important role in assessing the complexities of the rare cancer types affecting several systems including the bone, endocrine, digestive, vascular, and soft tissue. This review will explore how genetic testing and sequencing technologies have contributed to the identification of biomarkers across several rare cancer types in diagnostic, therapeutic, and prognostic stages, thereby advancing PM. Methods A comprehensive literature search was conducted across PubMed (MEDLINE), EMBASE, and Web of Science using keywords related to sequencing technologies, genetic testing, and cancer. There were no restrictions on language, methodology, age, or publication date. Both primary and secondary research involving humans or animals were considered. Results In practice, fluorescence in situ hybridization, karyotype, microarrays and other genetic tests are mainly applied to identify specific genetic alterations and mutations associated with cancer progression. Sequencing technologies, such as next generation sequencing, polymerase chain reaction, whole genome or exome sequencing, enable the rapid analysis of millions of DNA fragments. These techniques assess genome structure, genetic changes, gene expression profiles, and epigenetic variations. Consequently, they help detect main intrinsic markers that are crucial for personalizing diagnosis, treatment options, and prognostic assessments, leading to better patient prognosis. This highlights why these methods are now considered as primary tools in rare cancer research. However, these methods still face multiple limitations, including false positive results, limited precision, and high costs. Conclusion Genetic testing and sequencing technologies have significantly advanced the field of rare cancer research by enabling the identification of key biomarkers for precision diagnosis, treatment, and prognosis. Despite existing limitations, their integration into clinical and research fields continues to improve the development of personalized medicine strategies for rare and complex cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44e6c2844fd1beb610939541608801e2d5fb55e3" target='_blank'>
              Advancing Precision Medicine: The Role of Genetic Testing and Sequencing Technologies in Identifying Biological Markers for Rare Cancers
              </a>
            </td>
          <td>
            Joviana Farhat, Lara Alzyoud, M. Alwahsh, A. Acharjee, B. Al-Omari
          </td>
          <td>2025-04-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c1d77f1a7adf30fbae49c91b8912428daef56f" target='_blank'>
              CircRNA_103809/miR-516a/FBXL18 contributes to stemness and Gemcitabine Resistance of bladder cancer cells
              </a>
            </td>
          <td>
            Yu Yang, Feng Wang, Weiqiang Ning, Yi Ding, Wei Chen, Zhixian Yu
          </td>
          <td>2025-04-20</td>
          <td>Archives of Medical Science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc24f6cab28b67012c2dbe001093787b08c40ec" target='_blank'>
              Abstract 6713: RB1 and TP53 restrict lineage plasticity in prostate cancer by regulating chromatin accessibility
              </a>
            </td>
          <td>
            Mauricio Rodrigo Flores, David W. Goodrich, Justine J Jacobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/370d342e825336bd38d98c278a37277804ba994f" target='_blank'>
              Abstract 532: Single-cell analysis reveals neuroblastoma heterogeneity and therapeutic opportunities
              </a>
            </td>
          <td>
            Michele Tomanelli, Jennifer Hoti, Matteo Saettone, Daniele Reverberi, Adriana Bajetto, Tullio Florio, Aldo Pagano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9a2666f037bf98ff78850cacfb5af10e4de653" target='_blank'>
              Abstract 1338: System biological approach identifies heterochromatin modification as an important mechanism of acquired cancer radioresistance
              </a>
            </td>
          <td>
            Chaw Yee Beh, Sharon Pei Yi Chan, Celestia Pei Xuan Yeo, Boon Hao Hong, Evelyn Mui Cheng Tan, Dewi Susanti, Pei Ling Tai, Eugenia Li Ling Yeo, Khee Chee Soo, Edward Kai-Hua Chow, Melvin Lee Kiang Chua
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/293923f15ec319b2b53ec88ca71f468d810fc00d" target='_blank'>
              Abstract 5768: Identifying signaling changes that give rise to drug resistance in acute myeloid leukemia through multiomic modeling across diverse model systems
              </a>
            </td>
          <td>
            Juan Camilo Posso, S. Sharzehi, Marina A. Gritsenko, McKenna Finley, Chelsea Hutchinson-Bunch, Keith Beadle, Karl K Weitz, Melissa Stewart, Samantha A. Obermiller, Belinda B. Garana, Anupriya Agarwal, Cristina Tognon, J. Tyner, P. Piehowski, E. Traer, Sara J. Gosline
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Despite extensive research, the gene regulatory architecture governing mammalian cell states remains poorly understood. Here we present an integrative systems biology approach to elucidate the network architecture of primed state pluripotency. Using an unbiased methodology, we identified and experimentally confirmed 132 transcription factors as master regulators (MRs) of mouse epiblast stem cell (EpiSC) pluripotency, many of which were further validated by CRISPR-mediated functional assays. To assemble a comprehensive regulatory network, we silenced each of the 132 MRs to assess their effects on the other MRs and their transcriptional targets, yielding a network of 1273 MR → MR interactions. Network architecture analyses revealed four functionally distinct MR modules (communities), and identified key Speaker and Mediator MRs based on their hierarchical rank and centrality. Our findings elucidate the de-centralized logic of a “communal interaction” model in which the balanced activities of four MR communities maintain primed state pluripotency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7b83b212f0b2abb209a569ec434dc4a2d9e3bed" target='_blank'>
              The regulatory architecture of the primed pluripotent cell state
              </a>
            </td>
          <td>
            Bo I. Li, Mariano J. Alvarez, Hui Zhao, Napon Chirathivat, Andrea Califano, Michael M. Shen
          </td>
          <td>2025-04-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a306f36a45e59527afe8dc83f82d2317d23a26f" target='_blank'>
              Abstract A035: Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer
              </a>
            </td>
          <td>
            Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer arises as an abnormality in genomic control, either due to the activation of oncogenes or loss of functioning tumor-suppressor genes. Oncogenes result in an uncontrolled increase in cell proliferation; tumor-suppressor genes act as a brake, leading the cell cycle to stop and promote cell death when necessary. This review seeks to tease out the complex interactions between these genes, throwing light on key signaling pathways such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, all of which play a role in the progression of the tumor. We discuss major genetic theories, especially Knudson’s two-hit hypothesis, and how modern technologies like next-generation sequencing have advanced our understanding of cancer genetics. We also underscore the most current therapeutic approaches involving oncogenes and the reactivation of tumor-suppressor gene function, with challenges including drug resistance and heterogeneity of the tumor. In this regard, understanding the balance between oncogenes and tumor-suppressor genes is critical in developing effective and personalized cancer treatment, thereby bettering patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Reprogramming T cell metabolism can improve intratumoural fitness. By performing a CRISPR/Cas9 metabolic survey in CD8+ T cells, we identified 83 targets and we applied single-cell RNA sequencing to disclose transcriptome changes associated with each metabolic perturbation in the context of pancreatic cancer. This revealed elongation of very long-chain fatty acids protein 1 (Elovl1) as a metabolic target to sustain effector functions and memory phenotypes in CD8+ T cells. Accordingly, Elovl1 inactivation in adoptively transferred T cells combined with anti-PD-1 showed therapeutic efficacy in resistant pancreatic and melanoma tumours. The accumulation of saturated long-chain fatty acids in Elovl1-deficient T cells destabilized INSIG1, leading to SREBP2 activation, increased plasma membrane cholesterol and stronger T cell receptor signalling. Elovl1-deficient T cells increased mitochondrial fitness and fatty acid oxidation, thus withstanding the metabolic stress imposed by the tumour microenvironment. Finally, ELOVL1 in CD8+ T cells correlated with anti-PD-1 response in patients with melanoma. Altogether, Elovl1 targeting synergizes with anti-PD-1 to promote effective T cell responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee78135dc35b5bf79d91fde30087d13f7b5a5639" target='_blank'>
              A functional single-cell metabolic survey identifies Elovl1 as a target to enhance CD8+ T cell fitness in solid tumours
              </a>
            </td>
          <td>
            Samantha Pretto, Qian Yu, Pierre Bourdely, Sarah Trusso Cafarello, Heleen H Van Acker, Joren Verelst, Elena Richiardone, Lotte Vanheer, Amir Roshanzadeh, Franziska Schneppenheim, Charlotte Cresens, M. Sassano, J. Dehairs, Martin Carion, Shehab Ismail, P. Agostinis, S. Rocha, Tobias Bald, Johan V. Swinnen, C. Corbet, Sophia Y. Lunt, Bernard Thienpont, Mario Di Matteo, M. Mazzone
          </td>
          <td>2025-03-10</td>
          <td>Nature Metabolism</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b575dc8351428b4d22b9c3382531e97fc82f1bcf" target='_blank'>
              Abstract 545: RFX5 knockdown restores sensitivity to cellular cyclophosphamide in triple-negative breast cancer cell lines
              </a>
            </td>
          <td>
            Elizabeth Katherine Molchan, Kathleen Streeks, Alyssa Ohaegbulam, Mohammed Gbadamosi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction Thyroid cancer, a prevalent endocrine malignancy, has an age-standardized incidence rate of 9.1 per 100,000 people and a mortality rate of 0.44 per 100,000 as of 2024. Despite significant advances in precision oncology driven by large-scale international consortia, gaps persist in understanding the genomic landscape of thyroid cancer and its impact on therapeutic efficacy across diverse populations. Methods To address this gap, we performed comprehensive data mining and in silico analyses to identify pathogenic variants in thyroid cancer driver genes, calculate allele frequencies, and assess deleteriousness scores across global populations, including African, Amish, Ashkenazi Jewish, East and South Asian, Finnish and non-Finnish European, Latino, and Middle Eastern groups. Additionally, pharmacogenomic profiling, in silico drug prescription, and clinical trial data were analyzed to prioritize targeted therapeutic strategies. Results Our analysis examined 56,622 variants in 40 thyroid cancer-driver genes across 76,156 human genomes, identifying 5,001 known and predicted oncogenic variants. Enrichment analysis revealed critical pathways such as MAPK, PI3K-AKT-mTOR, and p53 signaling, underscoring their roles in thyroid cancer pathogenesis. High-throughput validation strategies confirmed actionable genomic alterations in RET, BRAF, NRAS, KRAS, and EPHA7. Ligandability assessments identified these proteins as promising therapeutic targets. Furthermore, our findings highlight the clinical potential of targeted drug inhibitors, including vandetanib, dabrafenib, and selumetinib, for improving treatment outcomes. Discussion This study underscores the significance of integrating genomic insights with pharmacogenomic strategies to address disparities in thyroid cancer treatment. The identification of population-specific oncogenic variants and actionable therapeutic targets provides a foundation for advancing precision oncology. Future efforts should focus on including underrepresented populations, developing population-specific prevention strategies, and fostering global collaboration to ensure equitable access to pharmacogenomic testing and innovative therapies. These initiatives have the potential to transform thyroid cancer care and align with the broader goals of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdb7f9eb5865d4f38b97ea959ca28a61a74b45c8" target='_blank'>
              Global analysis of actionable genomic alterations in thyroid cancer and precision-based pharmacogenomic strategies
              </a>
            </td>
          <td>
            Samantha Espinoza-Ferrao, G. Echeverría-Garcés, Sebastián Rivera-Orellana, José Bueno-Miño, Emilia Castellanos-Molina, Melanie Benítez-Núñez, A. López-Cortés
          </td>
          <td>2025-04-14</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/919d6be56b49f799484f0a056c28802836f4c288" target='_blank'>
              Abstract 2771: CRISPR/Cas9 essentiality screens identify stand-alone and IRF4-linked genetic vulnerabilities in melanoma cells
              </a>
            </td>
          <td>
            U. Sobhiafshar, N. T. Emre
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="It is well established that hematologic malignancies are often considerably radiosensitive, which enables usage of far lower doses of therapeutic radiotherapy. This review summarizes the currently known genomic landscape of hematologic malignancies, particularly as it relates to radiosensitivity and the field of radiation oncology. By tracing the historical development of the modern understanding of radiosensitivity, we focus on the discovery and implications of pivotal mutated genes in hematologic malignancies such as TP53, ATM, and other genes critical to DNA repair pathways. These genetic insights have contributed significantly to the advancement of personalized medicine, aiming to enhance treatment precision and outcomes, and there is an opportunity to extend these insights to personalized radiotherapy. We explore the transition from early discoveries to the current efforts in integrating comprehensive genomic data into clinical practice. Specific examples from Hodgkin lymphoma, non-Hodgkin lymphoma, and plasma cell neoplasms illustrate how genetic mutations could influence radiosensitivity and impact subsequent radiotherapeutic response. Despite the advancements, challenges remain in translating these genetic insights into routine clinical practice, particularly due to the heterogeneity of alterations and the complex interactions within cancer signaling pathways. We emphasize the potential of radiogenomics to address these challenges by identifying genetic markers that predict radiotherapy response and toxicity, thereby refining treatment strategies. The need for robust decision support systems, standardized protocols, and ongoing education for healthcare providers is critical to the successful integration of genomic data into radiation therapy. As research continues to validate genetic markers and explore novel therapeutic combinations, the promise of personalized radiotherapy becomes increasingly attainable, offering the potential to significantly improve outcomes for patients with hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e274aa44601f7df10924839e8b158bfac4f727a9" target='_blank'>
              The art of war: using genetic insights to understand and harness radiation sensitivity in hematologic malignancies
              </a>
            </td>
          <td>
            N. Wijetunga, J. Yahalom, B. Imber, Thomas FitzGerald, Lara Hilal, Ji Hyun Hong, Jonathan Leeman
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532359eb783a00c7e5a0af7f6a8c8009cbbb7c55" target='_blank'>
              Abstract 2758: Methylome plasticity as a biomarker of treatment response in small cell lung cancer
              </a>
            </td>
          <td>
            D. Sacdalan, Sami Ul Haq, L. Zhan, J. J. Li, V. Philip, Scott V. Bratman, Geoffrey Liu, Benjamin H. Lok
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The tumor suppressor protein, p53, which is mutated in half of human tumors, plays a critical role in cellular responses to DNA damage and maintenance of genome stability. Therefore, increasing our understanding of the p53 pathway is essential for improving cancer treatment and diagnosis. Methods This study, which aimed to identify genes and pathways that mediate resistance to p53 upregulation, used genome-wide CRISPR-Cas9 loss-of-function screening done with Nutlin-3a, which inhibits p53-MDM2 interaction, resulting in p53 accumulation and apoptotic cell death. We used bioinformatics analysis for the identification of genes and pathways that are involved in the p53 pathway and cell survival assays to validate specific genes. In addition, we used RNA-seq to identify differentially expressed p53 target genes in gene knockout (KO) cell lines. Results Our screen revealed three significantly enriched pathways: The heparan sulfate glycosaminoglycan biosynthesis, diphthamide biosynthesis and Hippo pathway. Notably, TRIP12 was significantly enriched in our screen. We found that TRIP12 is required for the p53-dependent transcription of several pro-apoptotic genes. Conclusion Our study has identified two novel pathways that play a role in p53-mediated growth restriction. Moreover, we have highlighted the interaction between the Hippo and the p53 pathways. Interestingly, we have shown that TRIP12 plays an important function in the p53 pathway by selectively affecting its role as a transcription factor. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01141-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320798304ad2c6aa4cbcc689f8af4488470a00c4" target='_blank'>
              Abstract 1475: HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications
              </a>
            </td>
          <td>
            Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most prevalent malignancies worldwide. With the increasing incidence of CRC, there is an urgent need for effective strategies for early diagnosis and treatment. Circadian rhythm, a natural biological clock, regulates various physiological processes, and its disruption has been implicated in the onset and progression of cancer. However, the specific roles of circadian rhythm-related genes (CRDGs) in CRC remain unclear. Methods In this study, we analyzed the expression patterns of CRDGs in CRC using single-cell RNA sequencing (scRNA-seq) data and bulk RNA sequencing data from the GSE178318 dataset. We constructed a CRC prognostic model based on CRD scores. Additionally, we explored the potential mechanisms of CRDGs in tumor progression through weighted gene co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA), and assessed their impact on the response to immune checkpoint inhibitors. Results The analysis revealed that CRDGs were significantly upregulated in liver metastasis samples compared to primary CRC samples and were closely associated with several metabolic and immune-related pathways. The prognostic model based on CRD scores indicated that higher CRD scores were associated with poorer outcomes in immunotherapy. These findings were further validated in multiple datasets, underscoring the potential of CRDGs as prognostic indicators in CRC. Conclusion This study systematically reveals, for the first time, the expression characteristics of CRDGs in CRC and their relationship with tumor progression and response to immunotherapy. CRDGs may serve as effective prognostic biomarkers and therapeutic targets, offering new strategies for the personalized treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85b7699b45efdb0dd0f3e56012bf04dabfd7ec73" target='_blank'>
              Integration of scRNA-Seq and Bulk RNA-Seq Identifies Circadian Rhythm Disruption-Related Genes Associated with Prognosis and Drug Resistance in Colorectal Cancer Patients
              </a>
            </td>
          <td>
            Yong Tao, Jun Li, Jianhui Pan, Qing Wang, Ru Ke, D. Yuan, Hongbiao Wu, Yuepeng Cao, Lei Zhao
          </td>
          <td>2025-04-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/844d851fd9820543dfb4e679c6a5eddb01a41ffd" target='_blank'>
              Abstract 3311: Uncovering methylation signatures and pathways associated with resistance to anti-EGFR therapy in colorectal cancer
              </a>
            </td>
          <td>
            Ana Regina de Abreu, Yann Heylen, J. Ibrahim, Marc Peeters, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73f3ee6052e03d5998ba44a27dca1b8bfd773b57" target='_blank'>
              Advancements in Cytopathology: New Diagnostic Insights via Molecular and Cellular Analysis
              </a>
            </td>
          <td>
            A. Singh, Rahul Saxena, S. Saxena
          </td>
          <td>2025-03-25</td>
          <td>Journal of Biochemistry International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="EGFR is a proto-oncogene that is mutationally activated in a variety of cancers. Small molecule inhibitors targeting EGFR can be effective in slowing the progression of disease, and in some settings these drugs even cause dramatic tumor regression. However, responses to EGFR inhibitors are rarely durable, and the mechanisms contributing to response variation remain unclear. In particular, several distinct mechanisms have been proposed for how EGFR inhibition activates cell death, and a consensus has yet to emerge. In this study, we use functional genomics with specialized analyses to infer how genetic perturbations effect the drug-induced death rate. Our data clarify that inhibition of PI3K signaling drives the lethality of EGFR inhibition. Inhibition of other pathways downstream of EGFR, including the RAS-MAPK pathway, promote growth suppression, but not the lethal effects of EGFR inhibitors. Taken together, our study reveals the first “reference map” for the genome-wide genetic dependencies of lethality for EGFR inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ecf934bb91c2c002402ace642323761657b953a" target='_blank'>
              Genome-wide profiling identifies the genetic dependencies of cell death following EGFR inhibition
              </a>
            </td>
          <td>
            Sydney A Porto, Gavin A. Birdsall, Nicholas W. Harper, Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, , Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08272a17f03be6497869a3622bf7e66968e9d5bd" target='_blank'>
              Abstract 219: Uncovering synergistic and antagonistic interactions in epigenetic therapeutics using perturb-seq
              </a>
            </td>
          <td>
            Changde Cheng, Rui Lu, Sajesan Aryal, Doug A. Welsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87c4098a87feebf01a217fad3de111af3e86862" target='_blank'>
              Abstract IA002: Functional omics for novel therapeutic combinations in myeloid malignancies
              </a>
            </td>
          <td>
            J. Tyner
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Recent research has identified a potential protective effect of higher numbers of circulating lymphocytes on colorectal cancer (CRC) development. However, the importance of different lymphocyte subtypes and activation states in CRC development and the biological pathways driving this relationship remain poorly understood and warrant further investigation. Specifically, CD4+ T cells -- a highly dynamic lymphocyte subtype -- undergo remodelling upon activation to induce the expression of genes critical for their effector function. Previous studies investigating their role in CRC risk have used bulk tissue, limiting our current understanding of the role of these cells to static, non-dynamic relationships only. Methods Here, we combined two genetic epidemiological methods -- Mendelian randomisation (MR) and genetic colocalisation -- to evaluate evidence for causal relationships of gene expression on CRC risk across multiple CD4+ T cell subtypes and activation stage. Genetic proxies were obtained from single-cell transcriptomic data, allowing us to investigate the causal effect of expression of 1,805 genes across five CD4+ T cell activation states on CRC risk (78,473 cases; 107,143 controls). We repeated analyses stratified by CRC anatomical subsites and sex, and performed a sensitivity analysis to evaluate whether the observed effect estimates were likely to be CD4+ T cell-specific. Results We identified six genes with evidence (FDR-P<0.05 in MR analyses and H4>0.8 in genetic colocalisation analyses) for a causal role of CD4+ T cell expression in CRC development -- FADS2, FHL3, HLA-DRB1, HLA-DRB5, RPL28, and TMEM258 . We observed differences in causal estimates of gene expression on CRC risk across different CD4+ T cell subtypes and activation timepoints, as well as CRC anatomical subsites and sex. However, our sensitivity analysis revealed that the genetic proxies used to instrument gene expression in CD4+ T cells also act as eQTLs in other tissues, highlighting the challenges of using genetic proxies to instrument tissue-specific expression changes. Conclusions Our study demonstrates the importance of capturing the dynamic nature of CD4+ T cells in understanding disease risk, and prioritises genes for further investigation in cancer prevention research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3854c3b38125ddae1dd0d632aa93ab78f073f33f" target='_blank'>
              Transcriptome-wide Mendelian randomisation exploring dynamic CD4+ T cell gene expression in colorectal cancer development
              </a>
            </td>
          <td>
            Benedita Deslandes, Xueyan Wu, Matthew A. Lee, L. J. Goudswaard, Gareth W Jones, A. Gsur, A. Lindblom, Shuji Ogino, Veronika Vymetálková, A. Wolk, Anna H. Wu, J. Huyghe, Ulrike Peters, Amanda I. Phipps, Claire E Thomas, Rish K Pai, Robert C Grant, Daniel D Buchanan, J. Yarmolinsky, Marc J. Gunter, Jie Zheng, E. Hazelwood, E. E. Vincent
          </td>
          <td>2025-04-17</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cfdfff54622d4227a75b1cb40c4cc408256a9de" target='_blank'>
              DNA Damage Response and Repair Genes and Anthracycline-Induced Cardiomyopathy in Childhood Cancer Survivors: A Report From the Children's Oncology Group and the Childhood Cancer Survivor Study.
              </a>
            </td>
          <td>
            Xuexia Wang, Purnima Singh, Romina B Cejas, Liting Zhou, Noha Sharafeldin, P. Trainor, Wendy Landier, Changde Cheng, L. Hageman, Fan Wang, Y. Sapkota, Yutaka Yasui, M. Hudson, Eric J Chow, S. Armenian, Joseph P Neglia, D. S. Hawkins, JillP Ginsberg, P. Burridge, Gregory T Armstrong, Smita Bhatia
          </td>
          <td>2025-03-28</td>
          <td>Circulation. Genomic and precision medicine</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72e3fdbe0004a1df71caab7b38b9a83bc3ac31c0" target='_blank'>
              Abstract 2750: Alternative enhancer usage of PER2 locus regulating 3D genome architecture in epithelial-mesenchymal states
              </a>
            </td>
          <td>
            Yi-Chia Chiu, Qing You Pang, Tuan Zea Tan, Wendy W. Hwang-Verslues, Ruby Y. J. Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef350217417b22617b2248d742f15859d0838a2a" target='_blank'>
              Abstract 752: Hypoxia-driven spatial dynamics of epigenetic and transcriptional cell states in glioblastoma
              </a>
            </td>
          <td>
            Phoebe Lombard, M. Zaidi, Ronald Wu, S. Mansouri, Gelareh Zadeh, B. Wouters
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80e1849f5d2a4bf9cdd4d6129184c9b1ac64a97" target='_blank'>
              Abstract 5413: Temporal regulation of c-Myc in endocycling cancer cells facilitates mitotic bypass in response to chemotherapy
              </a>
            </td>
          <td>
            Michael A. Loycano, Kenneth J. Pienta, Sarah R Amend
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Long Intergenic Non-Protein Coding RNA 1123 (LINC01123), located on human chromosome 2q13, is a pivotal factor in tumorigenesis, exerting multifaceted oncogenic effects. Its expression strongly correlates with clinicopathological features, patient survival, and disease progression. In vivo and in vitro experiments further demonstrate that LINC01123 influences diverse cellular processes, including proliferation, apoptosis, viability, migration, invasion, stemness, and tumor growth. Notably, it also regulates metabolic reprogramming, immune escape, and tumor cell resistance to treatment. LINC01123 is regulated by multiple transcription factors and participates in gene regulation through protein interactions and competitive endogenous RNA (ceRNA) networks, thereby modulating cancer-promoting effects. This work systematically elucidates its primary functions and molecular mechanisms driving cancer initiation and progression, suggesting that LINC01123 might serve as a novel potential oncogenic driver and biomarker in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca43f615285dd30f7033e316e2f7208e0ef3058" target='_blank'>
              A novel cancer-associated lncRNA, LINC01123, participates in tumor progression, metabolism, immune escape, and resistance
              </a>
            </td>
          <td>
            Qiang Liu, He Huang, Shuwen Zhang, Fangteng Liu, Ting Lou
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc8007e204fa82863ac047194325746e4be7c5ae" target='_blank'>
              Abstract 1441: Elevated HOXC gene expression in castration-resistant prostate cancer
              </a>
            </td>
          <td>
            Isaacson B. Adelani, Akira Kurozumi, Tobiloba Owojuyigbe, Sanchit Sanyal, Su Mi Choi, Srinivasan Yegnasubramanian, Shawn E Lupold
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631764abf56c15adfa48bcb7f3a761557f103393" target='_blank'>
              Abstract 444: ASXL1 mutations in AML are associated with a distinct epigenetic state that results in vulnerabilities to epigenetic-targeted agents
              </a>
            </td>
          <td>
            Christopher P. Mill, W. Fiskus, Christine E. Birdwell, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, K. Sasaki, S. Loghavi, T. Kadia, Naval G Daver, C. Dinardo, Lu Wang, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce10ef6d68c6903f3dcfc0aa9cd8d7cbbe833212" target='_blank'>
              Abstract 2992: Uncovering metabolic dysregulation and therapeutic targets in aneuploid models of squamous cell carcinoma
              </a>
            </td>
          <td>
            Nadja Zhakula, Sejal Jain, Laura Byron, Joan J. Castellano, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genetic diseases are serious health issues worldwide, caused by changes in gene structure due to mutations in single or multiple genes. Currently, no definitive treatments or medicines are available to cure these disorders. Gene therapy, a medical approach that utilizes genetic material, offers potential for preventing and treating diseases by addressing their underlying causes rather than relying on traditional methods like medication and surgery. This technique has shown promise in certain conditions by altering genetic material, enabling the body to produce desirable proteins or medications. However, gene therapy remains unavailable for most diseases and is still largely in the experimental stage for many conditions. This review investigates the current and future status of gene therapy not only for genetic disorders but also for life-threatening diseases such as cancer, cardiovascular disease (CVD), and diabetes. Additionally, the applications and limitations of gene therapy were evaluated. A review of recent research concluded that while rare or inherited genetic disorders could potentially be prevented or cured by precisely targeting their root causes, significant challenges remain. Rather than solely managing blood disorders, retinal diseases, and neuromuscular conditions, gene therapy can potentially eliminate these conditions. Furthermore, gene-based therapies offer expansive possibilities beyond rare genetic disorders, paving the way for broader therapeutic applications in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52a192a46bd071e41e3a160b10b231a04116a32" target='_blank'>
              Advancement in Gene therapy: Current and Future of Gene Therapy
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-03-17</td>
          <td>Kashmir Journal of Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8bc96152bbbb1bdeac5241e7bd6de510c434bdc" target='_blank'>
              Abstract 454: Leveraging cell cycle to enhance differentiation therapy in acute myeloid leukemia
              </a>
            </td>
          <td>
            Riccardo Valzelli, Kourosh Hayatigolkhatmi, Amir Hosseini, E. Ceccacci, Giorgia Parodi, A. Bachi, S. Minucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5520257da55eb20975e2b76db4a10f023d4be1" target='_blank'>
              Abstract 5543: High resolution lineage tracing for the study of cancer drug persistence
              </a>
            </td>
          <td>
            Binyanim Zhitomirsky, Jideofor Ezike, B. Danysh, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e8e473fd0d41652458c66d5982b1c03c7dc5d7e" target='_blank'>
              Abstract 3898: Integrating structural variants and single nucleotide variants to uncover evolutionary trajectories in melanoma
              </a>
            </td>
          <td>
            Xiwen Cui, A. Keskus, S. Malikić, Yuelin Liu, Anton Goretsky, Chi-Ping Day, F. Mehrabadi, M. Kolmogorov, Glenn Merlino, S. C. Sahinalp
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Recent approvals of PARP inhibitors (PARPi) for BRCA-mutant metastatic castration resistant prostate cancer (mCRPC) necessitate an understanding of the factors that shape sensitivity and resistance. Reversion mutations that restore homologous recombination (HR) repair are detected in ∼50-80% of BRCA-mutant patients who respond but subsequently relapse, but there is currently little insight into why only ∼50% of BRCA-mutant patients display upfront resistance. To address this question, we performed a genome-wide CRISPR screen to identify genomic determinants of PARPi resistance in murine Brca2Δ/Δ prostate organoids genetically engineered in a manner that precludes the development of reversion mutations. Remarkably, we recovered multiple independent sgRNAs targeting three different members (Cdt1, Cdc6, Dbf4) of the DNA pre-replication complex (pre-RC), each of which independently conferred resistance to olaparib and the next generation PARP-1 selective inhibitor AZD5305. Moreover, sensitivity to PARP inhibition was restored in Brca2Δ/Δ, Cdc6-depleted prostate cells by knockdown of geminin, a negative regulator of Cdt1, further implicating the critical role of a functional pre-RC complex in PARPi sensitivity. Furthermore, ∼50% of CRPC tumors have copy number loss of pre-RC complex genes, particularly CDT1. Mechanistically, prostate cells with impaired pre-RC activity displayed rapid resolution of olaparib-induced DNA damage as well as protection from replication fork degradation caused by Brca2 loss, providing insight into how Brca2-mutant cancer cells can escape cell death from replication stress induced by PARP inhibition in the absence of HR repair. Of note, a pharmacologic inhibitor that targets the CDT1/geminin complex (AF615) restored sensitivity to AZD5305, providing a potential translational avenue to enhance sensitivity to PARP inhibition in BRCA-mutant cancers. Significance Here, we address a major limitation in the effectiveness of PARP inhibitors in BRCA-mutant prostate cancer treatment: only ∼50% of patients respond despite clear genomic evidence of defective homologous recombination. Prior efforts to study PARP inhibitor resistance in prostate cancer have been plagued by the lack of suitable cell lines. We overcame this challenge using primary prostate organoids coupled with genome-wide CRISPR screening. The key finding is that loss of function mutations in the DNA pre-replication complex confer PARP inhibitor resistance. These genes map to chromosomal regions frequently lost in prostate cancer and could therefore serve as potential biomarkers of treatment response. Pharmacologic inhibition of geminin, a negative regulator of the pre-replication complex, can restore PARP inhibitor sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7234a92766796291cf8fafb5a6fc78a58509b240" target='_blank'>
              BRCA2 reversion mutation-independent resistance to PARP inhibition in prostate cancer through loss of function perturbations in the DNA pre-replication complex
              </a>
            </td>
          <td>
            Kyrie Pappas, Matteo Ferrari, Perianne Smith, S. Nandakumar, Zahra Khan, Serina B. Young, Justin Laclair, M. V. Russo, Emmet Huang-Hobbs, N. Schultz, W. Abida, Wouter Karthaus, M. Jasin, Charles Sawyers
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="Background Trastuzumab emtansine (T-DM1) is an antibody–drug conjugate that is an effective therapy for HER2-positive breast cancer; however, its efficacy is limited by drug resistance. While multiple mechanisms of resistance have been proposed, these are not yet well understood. Greater understanding of T-DM1 sensitivity and resistance could provide new combination strategies to overcome resistance or predictive biomarkers to guide therapy. Methods We have conducted CRISPR/Cas9 functional genomics modifier screens in HER2-positive breast cancer cell lines to allow for unbiased discovery of T-DM1 sensitivity and resistance genes. Whole-genome knockout screens were carried out in MDA-MB-361 and MDA-MB-453 cells treated with T-DM1 and its payload cytotoxin DM1. Hits were validated in secondary T-DM1 screens using a focused single-guide RNA (sgRNA) library and subsequently by individual gene knockout. Results The whole-genome CRISPR screens with T-DM1 and DM1 identified 599 genes as potential modifiers of T-DM1 sensitivity and resistance. Of these, 17 genes were significantly enriched and 3 genes depleted at P < 0.001 in either or both MDA-MB-361 and MDA-MB-453 libraries in the secondary screens. Among the top hits, were known T-DM1 sensitivity genes ERBB2 and SLC46A3, in addition to negative regulators of mTOR complex 1: TSC1 and TSC2. MDA-MB-453 clones with knockout of TSC1 or partial knockout of TSC2 were more resistant to T-DM1 than wild type cells in competition growth assays and to T-DM1 and other HER2 targeting therapies (T-DXd, lapatinib and neratinib) in growth inhibition assays, and had increased internalisation of T-DM1 at 6 h. T-DM1 and the mTOR inhibitor everolimus demonstrated synergistic activity at inhibiting cell proliferation at multiple T-DM1 concentrations across four HER2-positive breast cancer cell lines. Conclusions Our CRISPR screening approach with T-DM1 in HER2-positive breast cancer cell lines identified genes not previously implicated in T-DM1 sensitivity or resistance, including TSC1 and TSC2. These genes may inform new strategies to enhance T-DM1 therapy in the clinic. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02000-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dee2394ef1c6b351b41594d320afdc6e8fce82eb" target='_blank'>
              CRISPR screens with trastuzumab emtansine in HER2-positive breast cancer cell lines reveal new insights into drug resistance
              </a>
            </td>
          <td>
            Barbara A Lipert, K. Siemens, Aziza Khan, R. Airey, Gech Heng Dam, Man Lu, Marcella Flinterman, Queenie Yong, T. Lee, Francis W Hunter, Stephen M F Jamieson
          </td>
          <td>2025-03-31</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac8d5d5cb28b197c707cf32dafe8a0eca6770a3b" target='_blank'>
              Analysis of mutually exclusive expression in cancer cells identifies a previously unknown intergenic regulatory paradigm.
              </a>
            </td>
          <td>
            Ling Tang, Yuzhe Hu, Chao Wang, Wenling Han, Pingzhang Wang
          </td>
          <td>2025-04-04</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Epigenetic regulation plays an important role in cell fate reprogramming. Here, we found that inhibitors of epigenetic modifiers, including VPA, TSA, and 5-Aza-2'-deoxycytidine, can induce phenotypic transformation from Jurkat cells into B-cell-like cells. When Jurkat cells were treated with 5-Aza combined with VPA, B cell and stem cell marker expression was observed. These gene expression pattern changes were most remarkable in the optimized B cell induction conditions provided by the cocultured and genetically modified murine bone marrow OP9 cells. In such conditions, Jurkat cells were endowed with the ability to secrete B cell cytokines, and B lymphocyte-related genes and pathways were activated. In studying the mechanism underlying Jurkat cell reprogramming by 5-Aza and VPA, we found that PAX5, the key transcription factor regulating B cell development, was significantly upregulated. Treatment with 5-Aza and VPA inhibited the methylation of CpG islands and upregulated the acetylated H3K9 modification in the PAX5 promoter region, respectively, thus epigenetically activating the expression of PAX5 and promoting the reprogramming of Jurkat cells. Similar reprogramming results were also observed in primary CD4+T cells following treatment with 5-Aza and VPA. Our results provide a de novo paradigm for the reprogramming of T cells through epigenetic modifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7925f586974015e9c5c3ffdc07b4785039c00155" target='_blank'>
              5-Aza combined with VPA reprograms human T lineage acute leukemia Jurkat cells into B-cell-like cells by epigenetic activation of PAX5.
              </a>
            </td>
          <td>
            Wenjin Xi, Guoxu Zheng, Xu Chen, Baile Zuo, Wei Wang, Yufang Li, Chunmei Zhang, Jie Chu, Xiuli Mu, Weihong Wen, Tao Wang, An-Gang Yang
          </td>
          <td>2025-03-17</td>
          <td>Biotechnology progress</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b338d775526840d18d525abb3d293c825f24385" target='_blank'>
              Abstract B011: Harnessing epigenetic and metabolic biomarkers for targeted immunotherapy in colorectal cancer: Advances in precision medicine for solid tumors
              </a>
            </td>
          <td>
            Ji Min Lee
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39bd0ddc99a69058c299e72e0d1b4296584ccd0" target='_blank'>
              Therapeutic targeting of chromatin alterations in leukemia and solid tumors.
              </a>
            </td>
          <td>
            Florian Perner, Tobias Berg, D. Sasca, Sophie-Luise Mersiowsky, Jayant Y. Gadrey, Johanna Thomas, Michael W M Kühn, Michael Lübbert
          </td>
          <td>2025-04-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f89e30d2365e21c724d07e4e3a34981d8e7d27b4" target='_blank'>
              Beyond TKIs: Advancing Therapeutic Frontiers with Immunotherapy, Targeted Agents, and Combination Strategies in Resistant Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            I. Rangraze, Mohamed El-Tanani, A. Wali, Manfredi Rizzo
          </td>
          <td>2025-03-11</td>
          <td>Hemato</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7519cd5eee8915a40b0574f88da21fc8b5e2d5" target='_blank'>
              Therapeutic horizon of acute myeloid leukemia: Success, optimism, and challenges.
              </a>
            </td>
          <td>
            J. Senapati, T. Kadia, Naval G Daver, Courtney D. DiNardo, G. Borthakur, F. Ravandi, H. Kantarjian
          </td>
          <td>2025-03-19</td>
          <td>Cancer</td>
          <td>1</td>
          <td>226</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0460f2bfed152b0841d1d7ccdc52c08315edba3" target='_blank'>
              In vivo CRISPR screening reveals epigenetic regulators of hepatobiliary plasticity.
              </a>
            </td>
          <td>
            Jonathan H Sussman, Hector Cure, Salina Yuan, Kenji Ito, Irfan A Asangani, Benjamin A Garcia, Ben Z. Stanger, Takeshi Katsuda
          </td>
          <td>2025-04-01</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4eaf599d88dedaace0f5ff3bc70e72cf9c6e341" target='_blank'>
              Personalized Medicine Approaches for Pediatric Leukemia: Developing Targeted Therapies Based on Genetic Profiles
              </a>
            </td>
          <td>
            Sarah Hassan A Alshehri, Mohammed Abdullah Albariqi
          </td>
          <td>2025-03-24</td>
          <td>Saudi Journal of Medical and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43d3beaaac78a42fdc6dae5c6e9c46a2476c5705" target='_blank'>
              Abstract 6709: NF1 mutations in lung adenocarcinoma preclinical models and potential targeted therapies: The crucial role of the RAS-MAPK pathway
              </a>
            </td>
          <td>
            J. Giraud, Doriane Gorret, Manuela Ye, Dominique Lallemand, William C. Reinhold, Didier Decaudin, Ingrid Laurendeau, E. Pasmant, Camille Tlemsani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2951303fc3a7a8f3a52e93850fb8ede629ee404" target='_blank'>
              Abstract 4638: Investigations on association of prognostically important pan-cancer genes with disease progression and acute transformation in CML indicate involvement of diverse cellular mechanisms
              </a>
            </td>
          <td>
            S. Alanazi, S. Alfayez, Maryam Butuwaybah, N. Alanazi, A. Algarni, Kanza Adeel, Y. Taleb, A. Aleem, Aemir Mohammed, Zafar Iqbal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43e1e965f05b28c3994c4e8f0fb6a1f9448be497" target='_blank'>
              Abstract 2856: Pks+ E. coli trigger intestinal stem cell plasticity and early onset colorectal cancer
              </a>
            </td>
          <td>
            Stefanie Gerstberger, Melissa Lumish, S. Hartner, Farheen Shah, Seongmin Choi, Krystal Lum, Christopher Cowley, A. Luthra, Qingwen Jiang, Hyung Jun Woo, Ahmed Mahmoud, H. Walch, Asha Saxena, Tavis J. Reed, A. Cercek, R. Yaeger, Ileana Cristea, Andrew McPherson, F. Sánchez-Vega, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4631b336ea353f2fab308e1332f9b36313977696" target='_blank'>
              Abstract 4200: Elevated dNTP pools impair homologous recombination-mediated repair, promoting apoptosis in therapy-resistant glioblastoma
              </a>
            </td>
          <td>
            Dominique Monroe, Mercy Kehinde-Ige, Apeksha Anand, Arilyn Williams, E. Usoro, Matthew Kededa, Aman Kalsi, Christian Fahmy, Ramsha Khanam, Huidong Shi, Waaqo Daddacha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9488336a355d0c8f13d6aba1e3fdc05af9aece24" target='_blank'>
              Abstract 910: Chromosome 3q amplification in lung squamous cell carcinoma is associated to iron metabolism and immune cell exclusion
              </a>
            </td>
          <td>
            María Gutiérrez Pérez, J. R. Paradas, David Gómez Sánchez, R. G. Luján, Jon Zugazagoitia, N. Carrizo, Roxolyana Khoroz, Inés Díaz Cano, Laura García Redondo, Ana B Enguitta, Esther Conde, Eva M Garrido Martín, Álvaro C. Ucero Herrería, Luis Paz-Ares
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7430000b0cae106ff398286a8c011c0d35dbd7" target='_blank'>
              Loss-of-Function genetic Screen Unveils Synergistic Efficacy of PARG Inhibition with Combined 5-Fluorouracil and Irinotecan Treatment in Colorectal Cancer
              </a>
            </td>
          <td>
            Cristina Queralt, Cristina Moreta-Moraleda, Marta Costa, Ferran Grau-Leal, J. Diesch, Carla Vendrell-Ayats, Eva Musulén, C. Bugés, José Luis Manzano, Sara Cabrero, Johannes Zuber, M. Buschbeck, Sonia Forcales, E. Martínez-Balibrea
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387628d87300ede64a4f978b5ff12ccc1fbdb55a" target='_blank'>
              Abstract A007: Integrated molecular and functional characterization of the intrinsic apoptotic machinery identifies therapeutic vulnerabilities in glioma
              </a>
            </td>
          <td>
            E. Fernández, Wilson X Mai, Kai Song, Nicholas A. Bayley, Andrew J Souers, Jingyi Jessica Li, Thomas G. Graeber, T. Cloughesy, David A. Nathanson
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01701fc1080f2a22b516da6a31e565685ef5a558" target='_blank'>
              μSeq: Universal mutation rate quantification via deep sequencing of a single clonal expansion
              </a>
            </td>
          <td>
            Simone Pompei, Alberto Geroldi, Pietro Rivetti, Elena Grassi, V. Vurchio, Giorgio Tallarico, Giorgio Corti, Lorenzo Tattini, Gianni Liti, Andrea Bertotti, M. Lagomarsino
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8618bf3785bb983bc89d8550a35ee45cb0e4b8e9" target='_blank'>
              Current and future of targeted therapies against BCR::ABL kinases.
              </a>
            </td>
          <td>
            Sridhar Jayavel, Manasvini Subramanian, Pradeep Kumar Kesavan, Suresh Jayavel
          </td>
          <td>2025-04-07</td>
          <td>Journal of the Egyptian National Cancer Institute</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b25c599c72376da134e4b30a3445743d8254f163" target='_blank'>
              RNA hybrid-capture next-generation sequencing has high sensitivity in identifying known and less characterized oncogenic and likely oncogenic NTRK fusions in a real-world standard-of-care setting
              </a>
            </td>
          <td>
            Zachary D. Wallen, Marni B Tierno, Erica Schnettler, Alison Roos, Michelle Green, Kobina Amoah, R. Previs, S. Hastings, S. Pabla, Taylor J. Jensen, Brian Caveney, Marcia Eisenberg, P. Sathyan, Shakti H. Ramkissoon, Eric A Severson
          </td>
          <td>2025-05-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Gene fusions represent important oncogenic driver mutations resulting in aberrant cellular signaling. In up to 17% of all solid tumors at least one gene fusion can be identified. Precision therapy targeting fusion gene signaling has demonstrated effective clinical benefit. Advancements in clinically relevant next-generation sequencing and bioinformatic techniques have enabled expansion of therapeutic opportunity to subpopulations of patients with fusion gene expression. Clinically, tyrosine inhibitors have shown efficacy in treating fusion gene expressing cancers. Fusion genes are also clonal mutations, meaning it is a personal cancer target involving all cancer cells of that patient, not just a subpopulation of cancer cells within the cancer mass. Thus, both fusion signal disruption and immune signal targeting are effective therapeutic directions. This review discusses fusion gene targeting, therapeutic resistance, and molecular biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a39d77dcb6dc4121a7fa7e989f8a588d3f3ba74" target='_blank'>
              Clinical characterization and therapeutic targeting of fusion genes in oncology
              </a>
            </td>
          <td>
            Susan Morand, Lauren Rager, Daniel Craig, Alexander Nemunaitis, Khalil Choucair, Donald Rao, L. Stanbery, Richard C Phinney, Adam Walter, Maurizio Ghisoli, John Nemunaitis
          </td>
          <td>2025-03-24</td>
          <td>Future Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5fa7d80af5b43992f24ebd211378eba4c94dea" target='_blank'>
              Abstract 503: Tracking evolution of transcriptional landscapes during colorectal cancer development and progression
              </a>
            </td>
          <td>
            Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, J. Edwards, Owen J. Sansom, Philip D Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/810a20889322c24ae633443bf24e59f0a26d11b0" target='_blank'>
              Abstract 4493: Exploring APE1-G4 interactions and FACT complex inhibition to overcome therapy resistance in pediatric medulloblastoma
              </a>
            </td>
          <td>
            Ritesh Dey, Yingling Chen, Sutapa Ray, K. Bhakat
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc7f554e3c033394f0dca864fed36b87b63713d" target='_blank'>
              Abstract LB263: A transcriptomic signature predicts mortality risk and immune checkpoint inhibitor response beyond molecular and morphological features in lung adenocarcinoma from never smokers
              </a>
            </td>
          <td>
            Wei Zhao, Tongwu Zhang, Xing Hua, P. Hoang, Mona Miraftab, Monjoy Saha, John P. McElderry, Jian Sang, Olivia Lee, Caleb Hartman, Azhar Khandekar, Sunandini Sharma, Frank J. Colón-Matos, Samuel Anyaso-Samuel, Defei Wang, Kristine Jones, A. Hutchinson, B. Hicks, J. Rosenbaum, Xiaoming Zhong, Yang Yang, A. Pesatori, Dario Consonni, Karun Mutreja, Scott M. Lawrence, Nathaniel Rothman, Ludmil B. Alexandrov, Charles Leduc, M. Baine, Philippe Joubert, L. Sholl, William D Travis, Robert J Homer, Qing Lan, S. Chanock, Lixing Yang, S. Yang, Jianxin Shi, Marion Landi
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common cancers worldwide, with KRAS mutations occurring in approximately 40% of cases. These mutations drive tumorigenesis through the constitutive activation of key signaling pathways, such as RAS-RAF-MEK-ERK (MAPK) and PI3K-AKT-mTOR, contributing to therapeutic resistance and poor prognosis. Advances in molecular biology have led to significant breakthroughs, including the development of KRAS G12C inhibitors, such as sotorasib and adagrasib, which have shown promise in clinical trials. However, their efficacy is limited to a small subset of KRAS-mutant CRC, and resistance mechanisms often emerge through compensatory pathway activation. Combination strategies, including KRAS inhibitors with anti-EGFR agents, have been explored in trials like KRYSTAL-1 and CodeBreaK 300. Emerging research highlights the role of the tumor microenvironment in immune evasion and therapeutic resistance, offering opportunities for novel immunotherapy approaches, including KRAS neoantigen vaccines and adoptive T-cell therapy. Despite these advancements, challenges such as intratumoral heterogeneity, limited immune infiltration, and non-G12C KRAS mutations remain significant hurdles. This review provides a comprehensive overview of the molecular mechanisms, current advances and challenges, and future prospects in the management of KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dad5695932a1ea3976fffa19e82d922b6f2098" target='_blank'>
              Targeting the KRAS Oncogene for Patients with Metastatic Colorectal Cancer
              </a>
            </td>
          <td>
            Ruoyu Miao, James Yu, Richard D Kim
          </td>
          <td>2025-04-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0db92d7678fd17061debda8f9cf53b805a41693" target='_blank'>
              Abstract 7145: The pan-cancer landscape of RIT1 genetic alterations
              </a>
            </td>
          <td>
            Matteo Repetto, P. Selenica, S. Yang, B. Weigelt, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fe3cb02c5f8491138bcb64de94039ea1894af82" target='_blank'>
              Transcriptional reprogramming triggered by neonatal UV radiation or Lkb1 loss prevents BRAFV600E-induced growth arrest in melanocytes.
              </a>
            </td>
          <td>
            K. McGrail, Paula Granado-Martínez, Roberto Orsenigo, G. Caratù, Paula Nieto, Holger Heyn, Berta Ferrer, Javier Hernández-Losa, Eva Muñoz-Couselo, V. García-Patos, Juan A. Recio
          </td>
          <td>2025-03-08</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ab372dd4cae4e9e87fbdd2a61f62f73ea8595e" target='_blank'>
              Abstract 2973: Discovery of novel synthetic lethal targets for effective and safe colorectal cancer therapies
              </a>
            </td>
          <td>
            Andrzej Mazan, M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Izabella Wieckowska, M. Chmiel, A. Kurzejamski, B. Lipert, Katarzyna Sarad, M. Mikula, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ba721f32e0769bdd0286dcd3060ed010a1364c0" target='_blank'>
              Abstract 622: Age-dependent changes in cell cycle dynamics and hematopoietic responses to chemotherapy
              </a>
            </td>
          <td>
            Jianxing He, Yue Pan, Hao Liu, Ying Huang, H. Deng, Chao Yang, Qi Cai, Shan Xiong, Huiting Liu, Wenhua Liang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2db16c30d79726e181227a7646a42de7a4c218" target='_blank'>
              Abstract 829: ITGAV is a targetable vulnerability in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Gilbert K. Salloum
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0628902442374eea49ad28bc03ac93a39ca8fc6c" target='_blank'>
              BASELINE: A CRISPR Base Editing Platform for Mammalian-Scale Single-Cell Lineage Tracing
              </a>
            </td>
          <td>
            Evan Winter, F. Emiliani, Aidan Cook, Asma Abderrahim, Aaron McKenna
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339780e1e052f73bd7a155d38cbe11ed7059d53c" target='_blank'>
              Abstract 1253: Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma
              </a>
            </td>
          <td>
            Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, C. Fontanillo, Kai Wang, Ha Dang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland W. Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/700ea58a4c1d36566c271fc146a1aaff828a43d8" target='_blank'>
              Abstract 3652: Celleporter: a foundation model of cell and tissue biology with application to patient stratification and target discovery
              </a>
            </td>
          <td>
            Yubin Xie, Eshed Margalit, Tyler Van Hensbergen, Dexter Antonio, Jake Schmidt, Yu (Phoebe) Guo, Jérémie Decalf, Maxime Dhainaut, Lucas Cavalcante, Hargita Kaplan, Rodney Collins, Francis Fernandez, Rob Schiemann, Eric Siefkas, Michela Meister, Joy Tea, Carl Ebeling, Anastasia Mavropoulos, Nicole Snell, Shafique Virani, Ronald Alfa, Lacey J Padrón, Jacob Rinaldi, Daniel Bear
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1f9135b22221853a957c30c494e32f6d68e606" target='_blank'>
              Abstract LB016: Targeting ATM modulates oncogenic pathways and amplifies chemotherapy efficacy in small cell lung cancer
              </a>
            </td>
          <td>
            Debdatta Halder, Utsav Sen, V. Jethalia, Subhamoy Chakraborty, Yosuke Dotsu, Andrew Elliot, A. Vanderwalde, D. Demircioglu, D. Hasson, Triparna Sen
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401aff8c0f114a1e1c91ffcb779a01c90497fc5c" target='_blank'>
              Molecular insights and treatment innovations: Advancing outcomes in acute myeloid leukemia with myelodysplasia‑related changes (Review).
              </a>
            </td>
          <td>
            Hong Qiu, Chao-Wei Zhang, Xiaochen Ma, Ying Li
          </td>
          <td>2025-03-19</td>
          <td>Oncology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a8992e140899a62dade76814a52a30a531da19" target='_blank'>
              Abstract 5026: Discovery of oncogenic mediator genes in rectal cancer chemotherapy response using gene expression data from matched tumor and patient-derived organoids
              </a>
            </td>
          <td>
            Hanchen Huang, W. Zambare, Chao Wu, A. Colaprico, J. Alvarez, Min Jung Kim, A. Bercz, P. Bleu, Lily Wang, P. Paty, P. Romesser, J. J. Smith, X. Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1d772bceb37d658c289fadfd2de9d117e82a8a" target='_blank'>
              Abstract 4103: MLL4/COMPASS dysfunction in cancer and treatment
              </a>
            </td>
          <td>
            Zibo Zhao, Sarah Gold, Yuki Aoi, K. Meghani, Luke St John, Rukkia Liaqat, Carolyn Moloney, Yanni Yu, Jun Qian, I. Ben-Sahra, Rintaro Hashizume, D. Mahalingam, Joshua Meeks, A. Shilatifard
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687d3d9b838e13f050d770abff405a6b760ad3ff" target='_blank'>
              Diverse transcriptomic and mutational patterns but limited functional pathway alterations in patient-derived SS cells
              </a>
            </td>
          <td>
            F. Gallardo, Evelyn Andrades, Arnau Iglesias, Mariano R Maqueda, T. Lobo-Jarne, Jessica González, Joan Bertran, David Conde, Eva Rodriguez, Beatriz Bellosillo, Ramon M. Pujol, A. Bigas, Lluís Espinosa
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="In breast cancer related to the BRCA1 mutation, luminal progenitor cells are believed to be the cells of origin, yet how these cells transform into invasive cancer cells remain poorly understood. Here, we combine single-cell epigenomic and transcriptomic data to reconstitute sequences of events in luminal cells that lead to tumorigenesis. Upon deletion of Trp53 and Brca1, we find that luminal progenitors display an extensive epigenomic disorder associated with a loss of cell identity. These cells then progress to tumor formation through a partial epithelial-to-mesenchymal transition, orchestrated by Snail and the timely activation of immunosuppressive and FGF signaling with their microenvironment. In human samples, pre-tumoral changes can be detected in early stage, basal-like tumors, which rarely recur, as well as in normal-like mammary glands of BRCA1 mutation carriers who have had cancer. Our study fills critical gaps in our understanding of BRCA1-driven tumorigenesis, opening perspectives for the early monitoring of individuals with high cancer risk. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02331-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f69d9b899da12be47c2084b992747c9f0654fa" target='_blank'>
              Epigenomic disorder and partial EMT impair luminal progenitor integrity in Brca1-associated breast tumorigenesis
              </a>
            </td>
          <td>
            Camille Landragin, Mélissa Saichi, Marthe Laisné, Adeline Durand, Pacôme Prompsy, Renaud Leclère, Jérémy Mesple, Kyra Borgman, Amandine Trouchet, Marisa M Faraldo, A. Chiche, A. Vincent-Salomon, Hélène Salmon, Céline Vallot
          </td>
          <td>2025-04-28</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b855a13f4583cca6b74a017150a15f6334f81973" target='_blank'>
              Abstract 2773: The Gundersen Precision Oncology Cohort: a biospecimen resource for mutation characterization and target discovery
              </a>
            </td>
          <td>
            Peter Feiszt, Paraic A. Kenny
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1767b38084076cdfaf10fe2b85fb296d8eecabbf" target='_blank'>
              Abstract B021: Saturation mutational scanning uncovers druggability of all FGFR kinase point mutations
              </a>
            </td>
          <td>
            Carla Tangermann, Avantika Ghosh, Martin Ziegler, F. Facchinetti, Jannis Stappenbeck, M. Riester, Luise Carmina Viardot, Tobias Zundel, L. Friboulet, A. Hollebecque, José Naveja Romero, Angela Wanninger, M. Hess, Tilman Brummer, Melanie Börries, Sonja Loges, Yohann Loriot, Lena Illert, Sven Diedrichs
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract The molecular control of epigenetic information relies on hundreds of proteins of diverse function, which cooperate in defining chromatin structure and DNA methylation landscapes. While many individual pathways have been characterized, how different classes of epigenetic regulators interact to build a resilient epigenetic regulatory network (ERN) remains poorly understood. Here, we show that most individual regulators are dispensable for somatic cell fitness, and that robustness emerges from multiple layers of functional cooperation and degeneracy among network components. By disrupting 200 epigenetic regulator genes, individually or in combination, we generated network-wide maps of functional interactions for representative regulators. We found that paralogues represent only a first layer of functional compensation within the ERN, with intra- or inter-class interactions buffering the effects of perturbation in a gene-specific manner: while CREBBP cooperates with multiple acetyltransferases to form a subnetwork that ensures robust chromatin acetylation, ARID1A interacts with regulators from across all functional classes. When combined with oncogene activation, the accumulated epigenetic disorder exposes a synthetic fragility and broadly sensitizes ARID1A-deficient cells to further perturbation. Our findings reveal homeostatic mechanisms through which the ERN sustains somatic cell fitness and uncover how the network remodels as the epigenome is progressively deregulated in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74c759f71b52bc8e0f4b18c555786227753fc4c1" target='_blank'>
              Systematic genetic perturbation reveals principles underpinning robustness of the epigenetic regulatory network
              </a>
            </td>
          <td>
            Thomas Stuart Wilson, R. Noberini, Eirini Moysidou, Ifeyinwa Ojukwu, Marta Milan, Ming Jiang, Gavin Kelly, Michael Howell, Tiziana Bonaldi, Paola Scaffidi
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Despite advances, Glioblastoma (GBM) treatment remains challenging due to its rapid progression and resistance to therapies. Objectives This study aimed to investigate the role of anoikis—a mechanism by which cells evade programmed cell death upon detachment from the extracellular matrix—in GBM progression and prognosis. Methods Utilizing single-cell sequencing and bulk-transcriptome sequencing data from TCGA, GEO, and CGGA databases, we performed comprehensive bioinformatics analyses. We identified anoikis-related genes, constructed a prognostic model using 101 machine learning algorithms, and validated its clinical utility across multiple cohorts.Finally, we also verified the expression of model genes and the function of key gene in clinical samples and cell lines. Results Single-cell sequencing revealed heterogeneous expression of anoikis-related genes across distinct cell populations within GBM. MES-like Malignant cells and Myeloids exhibited higher enrichment of these genes, implicating their role in anoikis resistance and tumor aggressiveness. The prognostic model, based on identified genes, effectively stratified patients into high-risk and low-risk groups, demonstrating significant differences in survival outcomes. Mutation and tumor microenvironment analyses highlighted distinct genetic landscapes and immune cell infiltration patterns associated with different risk groups. SLC43A3 emerged as a key gene, showing significant upregulation in tumor tissues and correlating with poor prognosis in GBM. Conclusion This study provides insights into the molecular mechanisms of anoikis resistance in GBM, underscoring its critical role in tumor progression and patient prognosis. The developed prognostic model offers a promising tool for personalized treatment strategies and warrants further exploration of targeted therapies to improve outcomes for GBM patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03752-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90776aa9e4e44d828b89f11f0b82b3f7faa8093b" target='_blank'>
              Unraveling anoikis in glioblastoma: insights from single-cell sequencing and prognostic modeling
              </a>
            </td>
          <td>
            Q. Tang, Chenfeng Ma, Jiaheng Xie, Qixiang Zhang, Bingtao Zhang, Weiqi Bian, Qingyu Lu, Zeyu Wan, Wei Wu
          </td>
          <td>2025-03-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188faee26739e6ea96cb60a67313c6840df17757" target='_blank'>
              Abstract 3028: Tumor suppressor activation: unlocking a novel path to overcome CDK4/6 inhibitor resistance in HR-positive breast cancer
              </a>
            </td>
          <td>
            Hee-Sung Park, , Martin Pettersson, Chakrapani Subramanyam
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5750dddf572726400a2fad0203c2ec7fc9717955" target='_blank'>
              Abstract 478: Engineering viruses for the next generation targeted cancer therapy
              </a>
            </td>
          <td>
            Ikeda Trashi, Jeremiah Gassensmith
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca665dc53860039a134586f0a4361eb2c643769" target='_blank'>
              Abstract 5645: Targeting transcription-replication conflict for selective chemotherapy in glioblastoma
              </a>
            </td>
          <td>
            Long Gu, Robert J. Hickey, L. Malkas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2116c40f11199361c9cb93d0a62e0a019f2b44da" target='_blank'>
              Abstract 4681: Multiomic insights into mechanisms of androgen receptor-driven radioresistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Benjamin Hauk, B. McBean, A. Michmerhuizen, Douglas Gurdak, Savannah Tidmore, Michelle Paulsen, L. Lerner, C. Ward, Kassidy M. Jungles, Daniel Spratt, Lori J. Pierce, Mats Ljungman, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4be3aa12b8ff52b858ce8e9bed6a1d22e2b214" target='_blank'>
              Abstract 3879: Threshold dynamics of EWS::FLI1 reveal drivers of tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, F. Cidre-Aranaz, S. Grissenberger, C. Sturtzel, Karla Queiroz, D. Kurek, A. Wenninger-Weinzierl, M. Zylka, A. Sastre, Javier Alonso, Martin Distel, Anna Obenauf, Thomas G P Grünewald, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f17818e89e418eef53189bf0e536334d7fee6f" target='_blank'>
              Abstract 5042: The Paipu pipeline for pan-mammalian cancer genomics
              </a>
            </td>
          <td>
            Bria S. Smith, Geesa Daluwatumulle, Leslie A. Smith, James A. Cahill, Kiley Graim
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ca3a1f08d426b3a13684391aa11a6fb1c8556a" target='_blank'>
              Abstract LB443: Unlocking cell cycle dynamics: Gene regulation of CRISPR-Cas9 and siRNA phenotypes using cell painting
              </a>
            </td>
          <td>
            Alexander Schreiner, A. Foitzik, Josien Levenga, Karin Boettcher, Barbara Sonnenberg, Z. Strezoska, Joe Trask
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a9e6faf45fc758543cd3dfb6d7c7bf6cfe5d5f" target='_blank'>
              Abstract 6892: Ribonucleotide reductase is a potential therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Federica Invrea, Consalvo Petti, C. Isella, E. Médico
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f032fd3c43cdde86d384472ad3a3c6263ba1644" target='_blank'>
              Abstract 2746: 3D genome structural alterations in biological pathway genes drive the progression of cells from normal epithelium to metastatic cancer
              </a>
            </td>
          <td>
            Huy Nguyen, Shyamtanu Chattoraj, Jude Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, Frederick S. Vizeacoumar, Yue Zhang, Liliia Kyrylenko, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a605a688e58842e081be2262f0d61629173ea95c" target='_blank'>
              Abstract 1263: Reconstructing small cell lung cancer evolution using deep whole genome sequencing of circulating tumor DNA
              </a>
            </td>
          <td>
            Benjamin B. Morris, Zhihui Zhang, Andrew R. Lynch, Simon Heeke, Kyle Concannon, C. Stewart, Runsheng Wang, Alexa Halliday, W. Rinsurongkawong, Vadeerat Rinsurongkawong, J. J. Lee, Jianjun Zhang, Don L Gibbons, A. Vaporciyan, C. Gay, Hai T. Tran, J. Heymach, L. Byers, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c350dc98b4d3786f2ffe26744dbe7c5702a670" target='_blank'>
              Abstract 5246: Chromosomal instability promotes colorectal cancer progression through manipulation of gamma delta T cells
              </a>
            </td>
          <td>
            Paulino Tallón de Lara, Aviad Ben-Shmuel, Wen-Hao Hsu, Chaohao Li, Kyle Labella, Chi Wut Wong, Yonghong Liu, Shan Jiang, Xiaoying Shang, S. Kopetz, Ronald A. DePinho
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9365edb5d9a7621f7c2fa255adc2d1f2aebf2bfd" target='_blank'>
              Abstract 5334: Proliferative signaling through the ras-erk pathway can be inhibited by the overexpression of dual-specific phosphatase genes in collaboration with other negative regulators of cell cycle progression in the acute myeloid cell line HL-60
              </a>
            </td>
          <td>
            Michael Roberts, Amelia Harper, Kishan Mangru, Katy Meta, Sophia Kovatsis, J. Forrester, Grace Crossland, Cuong Nguyen, S. Bahadur
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3eb942377bafe8f7e43907ef0e3522eb17cee58" target='_blank'>
              1333 THY1 Upregulates Glioma Stem Cell Phenotype and Induces Treatment Resistance in Glioblastoma
              </a>
            </td>
          <td>
            Arushi Tripathy, S. Syed, V. Ravikumar, Dah-Luen Huang, A. Rehemtulla, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3d68234ed3b90b54b46e31e83216b5a77d6d67" target='_blank'>
              Abstract 5055: Efficient detection of chromosomal instability in diverse cancers using a targeted sequencing approach
              </a>
            </td>
          <td>
            H. Bao, Rui Liu, Haimeng Tang, Ningyou Li, Min Wu, Xunbiao Liu, Zhili Chang, Xue Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d431bff55b178eb615284d91d43a85ede147802" target='_blank'>
              Abstract 5488: Identifying novel Ewing sarcoma therapeutics using a high-throughput chromatin accessibility assay
              </a>
            </td>
          <td>
            Darian Williams, Andrew McFadden, Austin J. Hepperla, Min Shen, Tino W Sanchez, Gordon M. Stott, M. Henderson, Ian Davis, Samantha Pattenden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/242255fea6b3955bb0a9ccc76de9eb6a66d735ca" target='_blank'>
              Abstract 2843: Neoantigen evaluation in viably cryopreserved dissociated tissue via large scale proteomic profiling
              </a>
            </td>
          <td>
            Shawn P. Fahl, Casey Smith, Kerri Colwell, Jessica L. Moore, Jaison Arivalagan, Danielle B. Gutierrez
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4407199c782b232b5e9ca054b0b3617a834a373" target='_blank'>
              Abstract 7258: Enhancing TIL efficacy in NSCLC through epigenetic reprogramming and computational modeling
              </a>
            </td>
          <td>
            Daniel Plaugher, Avery R Childress, Christian M. Gosser, Dave-Preston Esoe, Xuilong Song, Jinpeng Liu, C. F. Brainson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c16f756ddb575cdf26b92c703a6e6c80ae8425f" target='_blank'>
              Abstract 6358: Single cell transcriptional dynamics in the HCMI cancer model collection
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Pérez-Mayoral, Megan J. Stine, Eva Y. Tonsing-Carter, James M. Clinton, The Hcmi Network, Peter W. Laird, Calvin J. Kuo, O. Elemento, David L. Spector, Andrew D Cherniack, Kyle Elrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Andrew McPherson, M. Garnett, D. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse S. Boehm
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c882aec893125b1655a88dcecde1b32dd7df1e2c" target='_blank'>
              Abstract 4094: Allosteric chromatin activity transition mechanisms drive epigenetic dosage compensation in essential genes triggered by oncogene activation in TRACERx
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, Xiaohong Liu, Georgia Stavrou, Nana E. Mensah, E. L. Cadieux, Özgen Deniz, M. Jamal-Hanjani, Stephan Beck, Jonas Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f4f8607796131836a1e01cc0a05131a1e222c3" target='_blank'>
              Abstract 5280: A strategic framework for modeling abnormal quiescence that drives cancer evolution, and for preclinical development of therapies to prevent recurrence and metastasis in high-risk patients
              </a>
            </td>
          <td>
            Balraj Singh, Anthony Lucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH), characterized by the expansion of hematopoietic stem and progenitor cells harboring somatic mutations, has emerged as a significant age-related phenomenon with profound implications for human health. While initially recognized in the 1960s, recent technological advances have revealed its complex nature and widespread prevalence, affecting up to 84% of individuals aged ≥ 70 years. The clinical significance of CH extends beyond its well-established role as a precursor to hematological malignancies, encompassing its association with cardiovascular diseases, chronic kidney disease, and other non-malignant disorders. This comprehensive review synthesizes the current understanding of CH, focusing on recent advances in genetic and molecular mechanisms, particularly the roles of commonly mutated genes such as DNMT3A, TET2, and ASXL1. We address the emerging distinction between myeloid and lymphoid CH, their differential impacts on disease progression, and the complex interplay between CH and inflammation. Special attention is given to newly identified genetic determinants of clonal expansion rates and their implications for disease progression. The review also examines the revolutionary concept of passenger-approximated clonal expansion rate and its utility in understanding CH dynamics. Furthermore, we discuss therapeutic strategies targeting inflammatory pathways and their potential in mitigating CH-associated complications. By integrating recent findings from genetic, molecular, and clinical studies, this review provides a framework for understanding CH as a systemic condition and highlights promising directions for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7375f0846811e9250a784d2b67a30ac0f6b0da37" target='_blank'>
              Clonal hematopoiesis: elements associated with clonal expansion and diseases
              </a>
            </td>
          <td>
            Gangpyo Ryu, Youngil Koh, S. Jaiswal, S. Yoon
          </td>
          <td>2025-03-13</td>
          <td>Blood Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df7c97eefb80be6ba219ad6c47ab592f0c4304c1" target='_blank'>
              Genomic profiling and personalized treatment strategies for skin malignancies: findings from the center for cancer genomics and advanced therapeutics database.
              </a>
            </td>
          <td>
            T. Hida
          </td>
          <td>2025-03-29</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee73397207289bc3e0061267436b35387c5d61ae" target='_blank'>
              Abstract 2170: S100a6-Lars2 oncogenic signaling shaping immunosuppressive environment contributes to platinum-based resistance
              </a>
            </td>
          <td>
            Yuqing Wang, Chuxia Deng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d750ecd44d29eb5bcb7a47a2c59888524520b8cd" target='_blank'>
              Context-dependent evolutionary dynamics toward bacterial survival during sequential antibiotic treatment
              </a>
            </td>
          <td>
            Adam Lyon, Mohammed Sadik Yildiz, Erdal Toprak
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/408d0e46d8aefed4371316d279839ada80b1b263" target='_blank'>
              Abstract A006: Differential therapeutic vulnerabilities and opportunities for smokers and never-smokers based on physiological and adaptive landscapes of lung adenocarcinoma
              </a>
            </td>
          <td>
            K. Dasari, Jorge A. Alfaro-Murillo, Jeffrey P. Townsend
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de37ada758a8c9054780c41940fecac62f6f55b3" target='_blank'>
              Abstract 2665: Single-cell analysis and cytokines modulation of intra-tumor subpopulations in colorectal cancer tumoroids
              </a>
            </td>
          <td>
            S. J. Fletcher, Irene Catalano, Elena Grassi, Sofia Borgato, Barbara Peracino, Luca Primo, L. Trusolino, A. Bertotti, A. Puliafito
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5392c951f2de30ee4a1335350f2dfe3bff927f11" target='_blank'>
              Circular RNAs: driving forces behind chemoresistance and immune evasion in bladder cancer.
              </a>
            </td>
          <td>
            M. Saadh, Wael Sheet Hussein, Ali Fawzi Al-Hussainy, A. K. Bishoyi, M. Rekha, Mayank Kundlas, V. Kavitha, Zafar Aminov, Sada Ghalib Taher, Mariem Alwan, M. Jawad, Hiba Mushtaq
          </td>
          <td>2025-03-25</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Exquisitely chemosensitive initially, small cell lung cancer (SCLC) exhibits dismal outcomes owing to rapid transition to chemoresistance. Elucidating the genetic underpinnings has been challenging owing to limitations with cellular models. As SCLC patient-derived xenograft (PDX) models mimic therapeutic responses, we perform genetic screens in chemosensitive PDX models to identify drivers of chemoresistance. cDNA overexpression screens identify MYC, MYCN, and MYCL, while CRISPR deletion screens identify KEAP1 loss as driving chemoresistance. Deletion of KEAP1 switched a chemosensitive SCLC PDX model to become chemoresistant and resulted in sensitivity to inhibition of glutamine metabolism. Data from the IMpower133 clinical trial revealed ~6% of patients with extensive-stage SCLC exhibit KEAP1 genetic alterations, with activation of a KEAP1/NRF2 transcriptional signature associated with reduced survival upon chemotherapy treatment. While roles for KEAP1/NRF2 have been unappreciated in SCLC, our genetic screens revealed KEAP1 loss as a driver of chemoresistance, while patient genomic analyses demonstrate clinical importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64850827a403ffc037c3b10ed964899b3dc8436a" target='_blank'>
              In vivo functional screens reveal KEAP1 loss as a driver of chemoresistance in small cell lung cancer
              </a>
            </td>
          <td>
            Lauren Brumage, Scott Best, Daniel S Hippe, Eli Grunblatt, Pritha Chanana, Feinan Wu, Myung Chang Lee, Zhe Ying, Ali H. Ibrahim, Jae Heun Chung, Anna-Lena B G Vigil, Jackson Fatherree, Slobodan Beronja, Patrick J. Paddison, Lucas B. Sullivan, B. Nabet, David MacPherson
          </td>
          <td>2025-04-23</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Overexpression of the H3K36 histone methyltransferase NSD2 in t(4;14) multiple myeloma (MM) is an early, oncogenic event, and understanding its impact on genomic organisation and expression is relevant to understanding MM biology. We performed epigenetic, transcriptional and phenotypic profiling of the t(4;14) KMS11 myeloma cell line and its isogenic translocation knock out (TKO) to characterise the sequelae of NSD2 overexpression. We found a marked global impact of NSD2 on gene expression and DNA organisation implicating cell identity genes; notably the early lymphocyte regulator, LAIR1 and MM cell surface markers, including CD38, a classical marker of plasma cells which was reduced in TKO cells. Plasma cell transcription factors such as PRDM1, IRF4 and XBP1 were unaffected, suggesting a downstream direct gene effect of NSD2 on cell identity. Changes in cell surface markers suggest an altered surface immunophenotype. Our findings suggest a role for NSD2 in maintaining MM cell identity, with potential implications for future therapeutic strategies based on targeting of NSD2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3900ea300ad687a680fc22372f0a01df92719adc" target='_blank'>
              NSD2-epigenomic reprogramming and maintenance of plasma cell phenotype in t(4;14) myeloma
              </a>
            </td>
          <td>
            Andrea Gunnell, Scott T. Kimber, R. Houlston, Martin Kaiser
          </td>
          <td>2025-03-21</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cf64c5137ce4675e3a801e6fb302671875782c" target='_blank'>
              Using Cancer Profiles to Identify Synthetic Lethal Therapeutic Targets and Predictive Biomarkers in Cancer Gene Dependency Data
              </a>
            </td>
          <td>
            Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2025-04-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BCL6 is a master transcriptional regulator of germinal center (GC) B cells. BCL6 is frequently translocated at the major translocation cluster (MTC) within intron 1 of the BCL6 locus, a hotspot commonly rearranged in diffuse large B cell lymphomas (DLBCLs). BCL6 amplifications are associated with therapeutic resistance and poor survival outcomes in hematological and solid cancers. However the mechanisms suppressing genome instability at the BCL6-MTC preventing BCL6 rearragements remain unclear. Here, transcriptome analysis and genome-wide mapping of histone H3 lysine 4 trimethylation (H3K4me3) in hydroxyurea (HU)-treated Raji cells (a Burkitt’s lymphoma model) revealed the induced expression of MBD1, encoding the DNA CpG methylation-binding protein. Functional studies using shRNA silencing and ectopic overexpression demonstrated that MBD1 suppresses BCL6 transcription whose promoter harbours conserved CpG methylation sites, suggesting a DNA methylation-dependent regulation of BCL6 trasncription by MBD1. Conversely, BCL6 repressed MBD1 expression by binding to its promoter. MBD1-depleted Raji cells exhibited increased genomic instability at the BCL6-MTC upon HU treatment, heightened sensitivity to DNA replication inhibitors (HU, gemcitabine, and etoposide), and reduced tumorigenicity in xenograft mouse models. We propose that MBD1 prevents genomic instability at the BCL6-MTC to suppress DLBCL formation. Moreover, MBD1 promotes genomic stability and cell viability during DNA replication stress. MBD1 thus represents a potential therapeutic target for cancers exhibiting resistance to chemotherapies targeting DNA replication. Key Points MBD1 suppresses BCL6 expression under the DNA replication stress MBD1 suppresses genomic instability at BCL6 translocation hotspots MBD1 depletion is associated with reduced tumorigenicity in mouse xenograft and sensitivity to DNA replication inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57727c3c919ef41a91774411963507ee9349496c" target='_blank'>
              DNA replication stress-induced transcriptome of Human Burkitt’s lymphoma identifies MBD1 as a novel suppressor of BCL6 rearrangements in germinal center derived B-lymphomagenesis
              </a>
            </td>
          <td>
            S. Gothwal, Kyoko Oichai, Jacqueline H Barlow
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Radioresistance is a major clinical challenge and the underlying mechanism has not been thoroughly elucidated. In this study, a radioresistant (RR) cell line is established to explore the transcriptomic signatures of radioresistance in colorectal cancer (CRC). KEGG enriched pathway analysis demonstrated that ferroptosis is inactivated in RR cells. Further detection confirmed that radiotherapy can promote ferroptosis, and ferroptosis inactivation is one of the hallmarks of radioresistance in CRC. What's more, induction of ferroptosis can restore the radiosensitivity of CRC cells. Then, we performed RNA sequencing to compare gene expression between parental and RR cells, and cells pretreated with or without RSL3. Via high‐throughput screening, NUPR1 was identified as a potential candidate for ferroptosis‐mediated radioresistance in CRC. CRC cells can acquire radiation resistance by NUPR1‐mediated ferroptosis suppression in the NUPR1‐overexpressing cell line. More importantly, ZZW‐115, an NUPR1 inhibitor, can sensitise RR cells to radiotherapy. Overall, our findings identify ferroptosis inactivation linked with resistance to radiotherapy. Besides, NUPR1 can promote radiation resistance by inhibiting ferroptosis, and targeting NUPR1 may be a potential strategy to relieve radioresistance associated with ferroptosis in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65b498b45232b4d8171ff61a415a9a20eb4c2e4c" target='_blank'>
              NUPR1 Promotes Radioresistance in Colorectal Cancer Cells by Inhibiting Ferroptosis
              </a>
            </td>
          <td>
            Yimin Fang, Haiyan Chen, Yunhua Liu, Kai Jiang, Yucheng Qian, Jingsun Wei, D. Fu, Hang Yang, Siqi Dai, Tian Jin, Tongtong Bu, Kefeng Ding
          </td>
          <td>2025-04-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/140dbaae467c306b2efa10ba8bbd45efee843365" target='_blank'>
              Abstract 6595: Lineage markers and driver oncogenes coordinate through gene expression subtypes of lung adenocarcinoma
              </a>
            </td>
          <td>
            Minjeong Kim, W. Lamlertthon, Heejoon Jo, Yan Cui, Hyo Young Choi, M. Wilkerson, Liza Makowski, D. N. Hayes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Chromatin modulators, like Polycomb group proteins, are key epigenetic regulators of gene expression and are frequently mutated in cancers. In adult stem cells, epigenetic regulation maintains their identity and controls their differentiation during homeostasis or aging, but its direct role in tumorigenesis remains unclear. This work explores the function of Polycomb Repressive Complex l (PRCl) in adult Drosophila intestinal stem cells (ISCs). Disrupting core PRCl components in ISCs induces the formation of small cell clusters devoid of intestinal markers, a novel phenotype linked to premature mortality under stress. Notably, transient PRCl loss is sufficient for cluster formation. These clusters overproliferate and exhibit immortality in serial transplantations, leading to their designation as tumor-initiating intestinal cells (TIICs). While JAK/STAT signaling promotes TIIC formation, the TOLL/IMD immune pathways restrict their expansion independently of cell death. Altogether, our results highlight PRCl as an epigenetic tumor suppressor in adult intestinal stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde642ec7f4f1326cc387e14ebb63c69e66e1638" target='_blank'>
              A tumor-suppressive role of the PRC1 Polycomb epigenetic complex in the maintenance of adult Drosophila intestinal stem cell identity
              </a>
            </td>
          <td>
            Aurélia Joly, Ana-Maria Popmihaylova, Corinne Rancurel, Julie Soltys, L. Fritsch, Rihab Loudhaief, Armel Gallet, Edan Foley, Anne-Marie Martinez, Raphaël Rousset
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ef83506a29a5e57d0f109af34e444759befd7da" target='_blank'>
              Single-cell RNA Sequencing Contributes to the Treatment of Acute Myeloid Leukaemia With Hematopoietic Stem Cell Transplantation, Chemotherapy, and Immunotherapy.
              </a>
            </td>
          <td>
            Raed Obaid Saleh, Ahmed Hjazi, S. O. Rab, Subasini Uthirapathy, Subbulakshmi Ganesan, Aman Shankhyan, M. Ravi Kumar, Girish Chandra Sharma, Muthena Kariem, Jawad Kadhim Ahmed
          </td>
          <td>2025-04-01</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aberrant alternative splicing, prevalent in cancer, impacts various cancer hallmarks involving proliferation, angiogenesis, and invasion. Splicing disruption often results from somatic point mutations rewiring functional pathways to support cancer cell survival. We introduce iSoMAs (iSoform expression and somatic Mutation Association), an efficient computational pipeline leveraging principal component analysis technique, to explore how somatic mutations influence transcriptome-wide gene expression at the isoform level. Applying iSoMAs to 33 cancer types comprising 9,738 tumor samples in The Cancer Genome Atlas, we identified 908 somatically mutated genes significantly associated with altered isoform expression across three or more cancer types. Mutations linked to differential isoform expression occurred through both cis- and trans-acting mechanisms, involving well-known oncogenes/suppressor genes, RNA binding protein and splicing factor genes. With wet-lab experiments, we verified direct association between TP53 mutations and differential isoform expression in cell cycle genes. Additional iSoMAs genes have been validated in the literature with independent cohorts and/or methods. Despite the complexity of cancer, iSoMAs attains computational efficiency via dimension reduction strategy and reveals critical associations between regulatory factors and transcriptional landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1e96c7e45926632d80fe6cd693fccd8a255dd7" target='_blank'>
              iSoMAs: Finding isoform expression and somatic mutation associations in human cancers.
              </a>
            </td>
          <td>
            Hua Tan, Valer Gotea, S. Jaiswal, Nancy E Seidel, David O. Holland, Kevin Fedkenheuer, Abdel Elkahloun, Sara R. Bang-Christensen, L. Elnitski
          </td>
          <td>2025-03-07</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e73b3c04c7a8aff09920f9f47c81555c9e458661" target='_blank'>
              Predicting Treatment Outcomes from Adaptive Therapy — A New Mathematical Biomarker
              </a>
            </td>
          <td>
            Kit Gallagher, Maximillian Strobl, P. Maini, Alexander R. A. Anderson
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c631feb596fea5a7b86a1eb40833189a58ee19" target='_blank'>
              Abstract 489: Unveiling markers of response to ATR inhibitor and TMZ combinations in rare tumor PDXs
              </a>
            </td>
          <td>
            Yingdong Zhao, Mariam Konaté, Ming-Chung Li, P. Wu, Li Chen, Lindsay M. Dutko, Alyssa Chapman, Nikitha Nair, M. Hollingshead, S. Jiwani, Biswajit Das, Julia Krushkal, A. Voth, Larry V Rubinstein, H. Stotler, Shannon N. Uzelac, Justine Mills, Tiffanie Chase-Miner, S. Borgel, Cindy R. Timme, Lijun Chen, Yingwei Hu, Hui Zhang, Alice P Chen, C. Karlovich, P. Williams, Yvonne A. Evrard, Lisa M. McShane, James H. Doroshow
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05067a5e4e5f9eb305cf27b302710eb5765274f3" target='_blank'>
              Abstract 3090: miRNA-221-5p-mediated epigenetic regulation promotes chemoresistance and offers therapeutic potential in ovarian cancer
              </a>
            </td>
          <td>
            T. Omy, Naresh Sah, Mark Reedy, Ganesh Acharya, Komaraiah Palle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9bbc590d0b38c9e40c5b483743c16479ab91ce" target='_blank'>
              Abstract 5090: Bioinformatics analysis of the role of asymmetric self renewal associated genes in the cancer progression
              </a>
            </td>
          <td>
            Seokyoung Hwang, Minju Lee, Jayhyun Cho, Minkyu Kim, Sang Kook Lee, Minsoo Noh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83d1334544559641fc60de6e87647100af4874" target='_blank'>
              Abstract 3454: Charting single-cell metabolism co-evolution during metastasis
              </a>
            </td>
          <td>
            Yingcheng Charles Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb9f7e3e34a8c78dbc563d40d2ad93cdb90edab" target='_blank'>
              Engineered protein circuits for cancer therapy
              </a>
            </td>
          <td>
            Andrew C. Lu, Lukas Moeller, Stephen Moore, Shiyu Xia, Kevin Ho, Evan Zhang, Mark W. Budde, Haley Larson, Ali Ahmed Diaz, Bo Gu, James M. Linton, Leslie Klock, Michael J. Flynn, Xiaojing J. Gao, D. Siegwart, Hao Zhu, M. Elowitz
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262a87481193ac96806ab02306fdc71e5a95bc29" target='_blank'>
              Abstract 273: Decipher actionable tumor suppressor defects in prostate cancer for precision medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6a5926479af2b1d3bad59b8a321348441349405" target='_blank'>
              Abstract 3960: Genome-wide CRISPR screens for genes inhibiting T cell-tumor cell interactions identify complex N-glycans
              </a>
            </td>
          <td>
            A. F. de Groot, Zowi R. Huinen, Juan Simon Nieto, D. Peeper
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f70cae7350af93e8189b0a48977a508877000d4" target='_blank'>
              Abstract SY27-02: Exploring the significance of somatic mtDNA mutations in the context of acute leukemia
              </a>
            </td>
          <td>
            M. Kundu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d478b727dab85de298dfc2ae721a0a16fcf9bb42" target='_blank'>
              Abstract 4154: Cancer cells exhibit novel metabolic vulnerabilities when exposed to low lipid environments
              </a>
            </td>
          <td>
            Diya L. Ramesh, Keene L. Abbott, Raphael Ferreira, M. V. Vander Heiden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba50a4897b94e23b9a56ce785c4accfcfef0e23" target='_blank'>
              Abstract 655: Spatiotemporally deciphering the response dynamics of micro-satellite stable rectal cancer to neoadjuvant short-course radiotherapy and PD-1 blockade
              </a>
            </td>
          <td>
            Jingwen Wang, Zhen Zhang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7188b7f43eeddcafd2d8d2c5bfae450202c1e2b1" target='_blank'>
              Abstract 4694: (R)evolutionary approach to predicting individual response to radiation therapy in breast cancer
              </a>
            </td>
          <td>
            Naheel Khatri, Raafat Chalar, Saiful Samad, Alexander Stessin, Mehdi Damaghi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Viruses are a leading cause of human morbidity and mortality. Certain viruses, including human papillomaviruses (HPVs), play a significant role in the etiology of cancer. Detection of viral DNA insertions in the human genome from next generation sequencing data defines viral associations with cancer and other diseases, identifies impacted organs and tissues, provides insights into disease mechanisms and has the potential to enhance clinical evaluations. In this study, we developed VirusIntegrationFinder (CTAT-VIF), a tool for surveying human genome insertions of various human viruses using both DNA and RNA sequencing data. We applied CTAT-VIF to analyze a dataset of over 30,000 tumor and normal samples, as well as more than 1,000 cancer cell lines. This effort resulted in the compilation of a catalog of over 30,0000 virus-human DNA or RNA junctions at more than 20,000 insertion loci and reassessed viral cancer-insertion hotspots across the human genome. Furthermore, we characterized the functional impacts of insertions with respect to human copy number alterations, effects on the expression of flanking human genes, and the identification of potentially oncogenic chimeric human and human/virus fusion transcripts at insertion loci. In addition to confirming known viral associations with specific tumor types, our study revealed both shared and virus-specific insertion hotspots in addition to variable functional impacts based on virus type. Besides some rare events of interest, we also found evidence for sequencing contamination, which underscores the need for vigilance when studying viral content or genome integrations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8c6c5dd068ea810bd4e2a265e3b19487f66e75" target='_blank'>
              Pathogenic Viruses, Genome Integrations, and Viral::Human Chimeric Transcripts Detected by VirusIntegrationFinder Across >30k Human Tumor and Normal Samples
              </a>
            </td>
          <td>
            Brian J. Haas, Anne Van Arsdale, A. Dobin, Maxwell P Brown, Joshua Gould, Christophe Georgescu, Eleanor Agosta, Sylvain Baulande, Ismail Jamail, Maud Kamal, I. Bieche, Jack Lenz, Christina Montagna, Aziz M. Al’Khafaji
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Epigenetic regulation of gene expression is essential for cellular development and differentiation processes in higher eukaryotes. Modifications of cytosine, in particular 5-methylcytosine (5mdC), in DNA play a central role through impacting chromatin structure, repressing transposons, and regulating transcription. DNA methylation is actively installed by DNA methyltransferases and reversed through Tet-dioxygenase-mediated oxidation of 5mdC to 5-hydroxylmethylcytosine (5hmdC), 5-formylcytosine (5fdC), and 5-carboxycytosine (5cadC). It is crucial to understand the role of these epigenetic DNA modifications in cellular differentiation and developmental processes, as well as in disease state mapping and tracing of 5mdC and its oxidized forms. In bisulfite sequencing, which has been the benchmark for mapping 5mdC for the last few decades, degradation of the majority of genetic material occurs through harsh chemical treatment. Alternative sequencing methods often utilize Tet-enzyme-mediated oxidation of 5mdC to locate 5mdC and 5hmdC in genomic DNA. Herein, we report the development of novel Tet3-variants for oxidation-based bisulfite-free 5mdC- sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e804da833f12d96ee4c49d305fc6d792d26acb4a" target='_blank'>
              Novel Tet3 enzymes for next-generation epigenetic sequencing.
              </a>
            </td>
          <td>
            Özge Simsir, Tobias Walter, Hanife Sahin, Thomas Carell, Sabine Schneider
          </td>
          <td>2025-03-10</td>
          <td>RSC chemical biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7175fb6cb31082cc299c7c39736754f0576dcb29" target='_blank'>
              Abstract 4662: Tracking gene expression in the tumor microenvironment throughout breast cancer progression
              </a>
            </td>
          <td>
            Ola Sayed Ahmed, heba Morsy, H. Ateya, Morkoss Medhat, Abdel Rahman Zekri
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258df61f2a4c963aeb2c62a2f4585ccce0f62b1b" target='_blank'>
              Abstract 6464: Spatial mapping of the tumor microenvironment for combination therapy design
              </a>
            </td>
          <td>
            Arindam Bose, Susan Keezer, E. Alonzo, Sophie Struble, Jeremy Fisher, Matthew Norton, R. Heil-Chapdelaine, Natasha F. Diaz Granados, Vasundhara Agrawal
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fed6e5310db01cd9e8fbb230a3891c5c1e5988e" target='_blank'>
              Abstract 44: PDX-derived cellular models for pharmacological screening and functional studies
              </a>
            </td>
          <td>
            L. Sinayen, Enora Le Ven, E. Indersie, Ester Morgado, L. Pouyet, F. Romagne, O. Deas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30bcf504297ff27c6be8dd475bedf1618228bd60" target='_blank'>
              Abstract 5052: Large-scale genomic analysis of mutation signatures reveals pathway alternations linked to breast cancer prognosis
              </a>
            </td>
          <td>
            Máté Posta, Balázs Győrffy
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Objectives/Goals: Large-scale tumor sequencing efforts have led to annotations of novel cancer hotspot mutations that may underlie driver or cooperative function. We have sought to define the molecular consequences of such hotspots associated with pediatric DICER1 syndrome cancers, with the ultimate goal of revealing novel targets that may inform new standards of care. Methods/Study Population: We have performed genomic analysis to identify tumor types (in TCGA and MSK-IMPACT patient data) for which mutations in the Dicer1 gene (encoding Dicer protein) emerge as the dominant signature of driver function. As Dicer is a critical RNA processing factor responsible for the generation of microRNAs, which are posttranscriptional gene regulatory molecules, we have modeled these mutations in human embryonic stem cells in order to study the direct effects on miRNAs and their target genes in an isogenic background. In addition to providing the required setting for unambiguous attribution of function to specific mutations, clonal human ES cells offer an opportunity for modeling of both developmental and cancer requirements associated with altered Dicer function. Results/Anticipated Results: Through generation of genomics and functional datasets from matched genotypes in Dicer mutated human ES cells, we have identified specific alterations in miRNAs and their effects on target genes. Unexpectedly, we found direct evidence for both loss of function and gain of function attributable to Dicer mutations. In addition, through integrated analysis of genomic data from tumor sequencing datasets and our human ES cell models, we have identified potential miRNA and target gene alterations that underlie tumorigenic potential, nominating gene candidates for targeted therapy in DICER1 syndrome. Direct mouse modeling of such candidate gene targets has revealed evidence for driver function of identified miRNA and their targets. Discussion/Significance of Impact: DICER1 syndrome cancers comprise a wide variety of rare pediatric tumor types. Presently, we still lack an effective standard of care. Furthermore, the previous lack of molecular profiling precluded targeted therapy opportunities. Our precise knock-in modeling of Dicer hotspots and deep profiling of relevant tumors now provide candidate targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597fce8f39319f3d39a09a21bf59ed3021208522" target='_blank'>
              518 Modeling of cancer mutations found in pediatric DICER1 syndrome informs novel therapeutic targeting strategies
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert W. Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cca4cdf73484b348317c78d3c6466b5fa0ebab2b" target='_blank'>
              Abstract 7053: Proteomics and integrative pathway analysis to understand the mechanism of action of mithramycin and etoposide combination induced cell death in Ewing sarcoma cells
              </a>
            </td>
          <td>
            C. Lambring, K. Zaman, Elin Stone, Natalie Gierat, Ameya Bhargava, Laszlo Prokai, Riyaz Basha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee700bd71bf28d3b143ba04f148030f9a950438b" target='_blank'>
              Genetics of canine cancer: a guide for the veterinary oncologist
              </a>
            </td>
          <td>
            M. Arendt, Anna Darlene van der Heiden, Raphaela Pensch, Kerstin Lindblad-Toh
          </td>
          <td>2025-04-22</td>
          <td>Veterinary Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Epigenetics and epigenomics are captivating fields of molecular biology, dedicated to the exploration of heritable alterations in gene expression and cellular phenotypes, which transpire devoid of any discernible modifications to the fundamental DNA sequence. This intricate regulatory apparatus encompasses multiple mechanisms, prominently featuring DNA methylation, histone modifications, and the involvement of non-coding RNA molecules in pivotal roles. To achieve a comprehensive grasp of these diverse mechanisms, it is imperative to conduct research employing animal models as proxies for human studies. Since experimental animal models like mice and rats struggle to replicate the diverse environmental conditions experienced by humans, this review focuses on comparing common epigenetic alterations in naturally occurring tumors in canine models, which share the human environment, with those in humans. Through this, we emphasize the importance of an epigenetic regulation in the comparative medical approach to a deeper understanding of cancers and further development of cancer treatments. Additionally, we elucidate epigenetic modifications pertinent to specific developmental stages, the ageing process, and the progression of various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bfc3c89b9e9910bb56cc9d2d52428ce2d674ca" target='_blank'>
              Comparative epigenomics to clinical trials in human breast cancer and canine mammary tumor
              </a>
            </td>
          <td>
            Su-Jin Jeong, Kang-Hoon Lee, Je-Yoel Cho
          </td>
          <td>2025-03-19</td>
          <td>Animal Cells and Systems</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea10556f7e26587dbcdc0c9ad631944d5c7f3c43" target='_blank'>
              Abstract 3824: Investigating the role of allelic imbalance in pancreatic cancer aggressiveness, survival, and therapeutic resistance
              </a>
            </td>
          <td>
            Enrico Gurreri, Li Zhang, Hania Khan, Lisa Maria Mustachio, Ningping Feng, T. Heffernan, Joseph R Marszalek, Basturk Olca, David Klimstra, Stefano Sioletic, A. Sgambato, G. Tortora, Giulio F. Draetta, Luigi Perelli, G. Genovese
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb9fb8020d3925866661e94fcf71afbc1c726241" target='_blank'>
              (A)symmetry in the Allele-Specific Chromosomal Structural Dynamics during Embryogenesis
              </a>
            </td>
          <td>
            Cibo Feng, Xiakun Chu
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5332ba21df7288f6cc769801a6e13199e6ab" target='_blank'>
              Evolution of robust cell differentiation mechanisms under epigenetic feedback
              </a>
            </td>
          <td>
            Davey Plugers, Kunihiko Kaneko
          </td>
          <td>2025-03-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed706044b2c945f86bdba6c75c9b7f8653518a59" target='_blank'>
              Abstract 3068: The clonal nature of FGFR3-TACC3 in glioma: Mechanism discovery and preclinical study
              </a>
            </td>
          <td>
            Yiming Li, Tao Li, Xiude Ren, Jianshen Liang, Jikang Fan, Xisen Wang, Sheng-ping Yu, Wei Zhang, Xuejun Yang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Poorly differentiated thyroid cancer (PDTC) and anaplastic thyroid cancer (ATC) present major challenges in treatment owing to extreme aggressiveness and high heterogeneity. In this study, deep-scale analyses spanning genomic, proteomic, and phosphoproteomic data are performed on 348 thyroid-cancer and 119 tumor-adjacent samples. TP53 (48%), TERT promoter (36.5%), and BRAF (23%) are most frequently mutated in PDTC and ATC. Ribosome biogenesis is identified as a common hallmark of ATC, and RRP9 silencing dramatically inhibits tumor growth. Proteomic clustering identified three ATC/PDTC subtypes. Pro-I subtype is characterized with aberrant insulin signaling and low immune cell infiltration, and Pro-II is featured with DNA repair signaling, while Pro-III harbors high frequency of TP53 and BRAF mutation and intensive C5AR1+ myeloid infiltration. Targeting C5AR1 synergistically improves antitumor effect of PD-1 blockade against ATC cell-derived tumors. These findings provide systematic insights into tumor biology and opportunities for drug discovery, accelerating precision therapy for virulent thyroid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618bcd39c8235e1a1c051cf80f1d7a8224b8d7f2" target='_blank'>
              Integrative proteogenomic characterization reveals therapeutic targets in poorly differentiated and anaplastic thyroid cancers
              </a>
            </td>
          <td>
            Zongfu Pan, Zhuo Tan, Ning Xu, Zhenmei Yao, Chuanming Zheng, Jinbiao Shang, Lei Xie, Jiajie Xu, Jia-feng Wang, Liehao Jiang, Xuhang Zhu, Dingyi Yu, Ying Li, Yulu Che, Yingying Gong, Zhaoyu Qin, Yiwen Zhang, Xiaozhou Zou, Tong Xu, Zhenying Guo, Tiefeng Jin, Tiannan Guo, Wei Wang, Wanyuan Chen, Yaoting Sun, Weixin Wang, Xiaojun Peng, Changtian Yin, Chen Ding, Ping Huang, Minghua Ge
          </td>
          <td>2025-04-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f7c0a88fb3654ac3464cd672c3b86897a3782a" target='_blank'>
              KPT-330, A New Candidate Drug for Targeting NOTCH1 Overexpression in T-cell Acute Lymphoblastic Leukemia, An In Vitro and Silico Study.
              </a>
            </td>
          <td>
            Fariha Naz, Saima Ejaz, Tayyaba Wali, Atiqa Nudrat, Mian Abdur Rehaman Arif, Muhammad Qandeel Waheed, Fatma Hussain, Saadiya Zia
          </td>
          <td>2025-04-11</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab96ea3da2d8da3f9378cdc0608c7c70ccd45b06" target='_blank'>
              Abstract LB101: Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis
              </a>
            </td>
          <td>
            Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Ming Xu, Eric M Pietras, James DeGregori, Jennifer J Trowbridge, Hideyuki Oguro, Sheng Li
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c687713915048ef2b6e851768a7161399616a610" target='_blank'>
              Abstract 6416: Molecular dynamics driving phenotypic divergence among KRAS mutants in pancreatic tumorigenesis
              </a>
            </td>
          <td>
            David J. Falvo, A. Grimont, R. Chandwani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72c819e1be5dc45d12cb085264654219606f64c0" target='_blank'>
              Abstract 4080: A novel integrated target capture and rapid sequencing workflow for myeloid malignancy mutation detection within 12 hours
              </a>
            </td>
          <td>
            Kyle Metcalfe, Benjamin J. Krajacich, Xiaodong Qi, Kelly N. Wiseman, Marina McCowin, Shawn Levy, Junhua Zhao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6d37955c551700a448a3cd5642f0bb308460197" target='_blank'>
              Abstract 7479: Single-cell OncoTreat: Novel therapeutic candidates by high resolution inference and validation of drug sensitivities in heterogeneous cholangiocarcinoma tumors
              </a>
            </td>
          <td>
            Aleksandar Z Obradovic, Lorenzo Tomassoni, Daoqi You, Charles G. Drake, F. D. Dela Cruz, Andrew L Kung, Andrea Califano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17eeae7cebe3a01a8802644a89b867255b13f516" target='_blank'>
              Abstract 7043: HMGA1 Epigenetic Regulators Drive Relapse in Pediatric B-Cell Leukemia By Amplifying ETV5 and Stemness Networks
              </a>
            </td>
          <td>
            Linda S. Resar, Jung-Hyun Kim, Andrei Thomas-Tikonenko, S. Tasian
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03e8bbeec84df302b9adf6fafec7b5584f0a9749" target='_blank'>
              Advancement of Gene Therapy in Cancer
              </a>
            </td>
          <td>
            Kahkashan Sultana, Ayesha Fatima
          </td>
          <td>2025-03-15</td>
          <td>Journal of Advanced Scientific Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal alterations are frequent events in lung cancer progression. Although gains and losses of chromosomal position have been reported, the association between copy number alteration and lung cancer patient survival has not been extensively investigated. In this study, we performed a meta-analysis of public cBioPortal datasets spanning 25 lung cancer studies to identify putative cancer driver genes with copy number alterations associated with overall patient survival. Ten copy-number altered genes enriched in deceased lung cancer patients were identified. Seven of these putative driver genes were located in the 7p11.2 chromosomal location, and two were in the 9p21.3 cytoband. Among these genes, the mitochondrial ribosomal protein S17 (MRPS17) amplification was significantly associated with a lower patient survival rate (P = 1.47e-7). To investigate the functional role of MRPS17, small interfering RNA-mediated knockdown was performed in two non-small cell lung cancer cell lines, A549 and NCI-H460. MRPS17 knockdown significantly reduced cell proliferation, migration, invasion, and anchorage-independent growth in both cell lines. Furthermore, knockdown of MRPS17 decreased the activation of the phosphatidylinositol 3-kinase/protein kinase B signaling pathway, suggesting its role in driving lung cancer progression through this critical oncogenic pathway. Our findings highlight MRPS17 as a potential cancer therapy target and a prognostic biomarker that may improve the survival rates of lung cancer patients. Future studies should explore its inhibition as a therapeutic strategy as well as elucidate its molecular mechanisms in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1c0b37271452f619e3684f3550b14101ddebda" target='_blank'>
              Mitochondrial Ribosomal Protein S17 Silencing Inhibits Proliferation and Invasiveness of Lung Cancer Cells
              </a>
            </td>
          <td>
            Woo Rin Lee, Kook Sun Ha
          </td>
          <td>2025-03-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BRAF mutations are critical drivers in cancers such as melanoma, colorectal cancer, and non-small-cell lung cancer. The most common mutation, BRAF V600E, is a key therapeutic target. Targeted treatments with BRAF and MEK inhibitors have significantly improved progression-free and overall survival in melanoma patients. However, in cancers like metastatic colorectal cancer, BRAF mutations are associated with poor outcomes due to aggressive disease behavior and resistance to conventional chemotherapy. Despite progress, resistance to BRAF/MEK inhibitors remains a major challenge, often driven by secondary mutations in the mitogen-activated protein kinase (MAPK) pathway, activation of alternative pathways such as phosphoinositide 3-kinases (PI3Ks)/protein kinase B (AKT), or changes in the tumor microenvironment. These challenges have motivated ongoing research into combining BRAF inhibitors with immunotherapies to enhance and prolong treatment effectiveness. Future research must also account for the role of the cancer’s tissue of origin, as the biological context significantly influences response to targeted therapies, highlighting the need for a deeper understanding of tumor biology, micro-environment, and genetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c40df95ff6814d63c0d25adecf2918b454793725" target='_blank'>
              BRAF Targeting Across Solid Tumors: Molecular Aspects and Clinical Applications
              </a>
            </td>
          <td>
            Hiba Mechahougui, James Gutmans, Roumaïssa Gouasmi, Laure Smekens, Alex Friedlaender
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/accef4dd1efc826cc2507a7542b8ca2e3603156d" target='_blank'>
              Cell-type-specific transposable element demethylation and TAD remodeling in the aging mouse brain
              </a>
            </td>
          <td>
            Qiurui Zeng, Wei Tian, Amit Klein, Anna Bartlett, Hanqing Liu, Joseph R. Nery, R. Castanon, Julia K. Osteen, Nicholas D. Johnson, Wenliang Wang, Wubin Ding, Huaming Chen, Jordan Altshul, M. Kenworthy, Cynthia Valadon, William Owens, Zhanghao Wu, Maria Luisa Amaral, Yuru Song, Cindy Tatiana Báez-Becerra, Silvia Cho, Chumo Chen, Jackson Willier, Stella Cao, Jon Rink, Jasper Lee, Ariana Barcoma, Jessica Arzavala, Nora Emerson, Yuancheng Ryan Lu, Bing Ren, M. Behrens, J. Ecker
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2050a0d48b53f00fa87bdf892317410211b4055f" target='_blank'>
              Harnessing State-of-the-Art Gene Therapy to Transform Oral Cancer Treatment.
              </a>
            </td>
          <td>
            Wei Zhang, Yan Zhang, Xiaowen Yang, Hongyan Chai
          </td>
          <td>2025-03-11</td>
          <td>Biochemical genetics</td>
          <td>1</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [5],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>