<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-13 09:31:47 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>62</td>
          <td>45</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd5596bbafd3812efcd1d8c0a4bfdf21b8096e0" target='_blank'>
              Adaptive cancer therapy: can non-genetic factors become its achilles heel?
              </a>
            </td>
          <td>
            Gábor Valcz, Robert A. Gatenby, B. Újvári, E. Buzás, Béla Molnár
          </td>
          <td>2025-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="The mismatch repair system is critical for correcting base–base mismatches and insertion-deletion loops during DNA replication. Deficiencies in MMR (due to mutations in MLH1, MSH2, MSH6, or PMS2) lead to microsatellite instability and contribute to the development of various cancers, such as Lynch syndrome-related colorectal cancer and sporadic tumors. This review will delve into the molecular basis of MMR deficiency. Additionally, the review will cover diagnostic approaches for detecting MSI and MMR deficiency, including next-generation sequencing and PCR-based methods. The implications for treatment will be discussed, emphasizing immune checkpoint inhibitors (e.g., pembrolizumab) that target tumors with high mutational burdens due to MMR deficiency, as well as novel therapeutic approaches like synthetic lethality exploiting DNA repair vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c3fe776b0060c3cc36839d638f2791e2a3b03a" target='_blank'>
              Mismatch Repair Deficiency and the Role of Non-Canonical Functions in Cancer: Diagnosis and Therapeutic Implications
              </a>
            </td>
          <td>
            Alicja Dąbrowska, Jakub Mastalerz, Zofia Łapińska, I. Deszcz, Agnieszka Chwiłkowska, Nina Rembiałkowska
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="KRAS mutations represent the most common driver genetic alterations in multiple malignant tumors, particularly exerting well-defined oncogenic driver roles in non-small cell lung cancer, colorectal cancer, and pancreatic ductal adenocarcinoma. The recent success of KRAS G12C mutation-specific allosteric inhibitors marks a therapeutic milestone in KRAS-targeted oncology. However, the rapid emergence of drug resistance in clinical applications has significantly limited the durable efficacy of these agents. The resistance mechanisms exhibit profound complexity, encompassing multidimensional pathways such as secondary/co-occurring mutations, compensatory reprogramming of signaling pathways, cellular lineage plasticity, and immune evasion within the tumor microenvironment. Studies have demonstrated that the identification of resistance-associated biomarkers is not only of critical clinical value for predicting treatment response and early warning of resistance but also provides a key entry point for dissecting resistance mechanisms. This review systematically summarizes the latest research advances in KRAS-mutant inhibitor resistance biomarkers, with a focus on the analysis of resistance molecular mechanisms, discovery of predictive biomarkers, current limitations and challenges, and exploration of biomarker-based combination therapy strategies. By integrating basic research and clinical data, we highlight the breakthroughs required to realize the application value of resistance biomarkers and prospects future research directions and priorities, aiming to provide a biomarker-related theoretical framework for precision therapy of KRAS-mutant tumors and accelerate the clinical translation of resistance-overcoming strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedd636638a7ce054ecf1cb2befdf53d765c7e0b" target='_blank'>
              Advances in biomarkers of resistance to KRAS mutation-targeted inhibitors
              </a>
            </td>
          <td>
            Lihao Wang, Dandan Wei, Shanshan Li, Shiqing Jiang
          </td>
          <td>2025-10-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells exhibit reprogrammed metabolic pathways to sustain aggressive phenotypes, including continuous cell division, stemness, invasion, and metastasis. Emerging evidence suggests that these metabolic adaptations profoundly impact DNA repair pathways, which contribute to the responses to therapy and influence overall outcomes. Metabolic processes such as the Warburg effect, nicotinamide adenine dinucleotide (NAD) metabolism, glutamine metabolism, and one-carbon metabolism support DNA repair by expanding the metabolite pool and facilitating post-translational modifications. Conversely, oncometabolites impair DNA repair pathways through epigenetic reprogramming, thereby promoting genomic instability. This review highlights recent discoveries that elucidate the intricate connections between metabolic hallmarks in cancer cells and DNA repair mechanisms, offering insights into potential therapeutic targets for future cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0683240e558e0cfd022d73899ecd3cc85f65ee47" target='_blank'>
              Cancer Metabolism Meets DNA Repair: The Hidden Link to Therapy Resistance.
              </a>
            </td>
          <td>
            Chunzhang Yang
          </td>
          <td>2025-09-28</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aim: Growing evidence points to non-genetic mechanisms underlying long-term resistance to cancer therapies. These mechanisms involve pre-existing or therapy-induced transcriptional cell states that confer resistance. However, the relationship between early transcriptional responses to treatment and the eventual emergence of resistant states remains poorly understood. Furthermore, it is unclear whether such early resistance-associated transcriptional responses are evolutionarily conserved. In this study, we examine the similarity between early transcriptional responses and long-term resistant states, assess their clinical relevance, and explore their evolutionary conservation across species. Methods: We integrated datasets on early drug responses and long-term resistance from multiple cancer cell lines, bacteria, and yeast to identify early transcriptional changes predictive of long-term resistance and assess their evolutionary conservation. Using genome-wide CRISPR-Cas9 knockout screens, we evaluated the impact of genes associated with resistant transcriptional states on drug sensitivity. Clinical datasets were analyzed to explore the prognostic value of the identified resistance-associated gene signatures. Results: We found that transcriptional states observed in drug-naive cells and shortly after treatment overlapped with those seen in fully resistant populations. Some of these shared features appear to be evolutionarily conserved. Knockout of genes marking resistant states sensitized ovarian cancer cells to Prexasertib. Moreover, early resistance gene signatures effectively distinguished therapy responders from non-responders in multiple clinical cancer trials and differentiated premalignant breast lesions that progressed to malignancy from those that remained benign. Conclusion: Early cellular transcriptional responses to therapy exhibit key similarities to fully resistant states across different drugs, cancer types, and species. Gene signatures defining these early resistance states have prognostic value in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4c9b8cf4452549a749a1c9bb16bff08ecee299" target='_blank'>
              Resistance signatures manifested in early drug response across cancer types and species
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Arashdeep Singh, MM Gottesman, Sridhar Hannenhalli
          </td>
          <td>2025-08-26</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            Saeid Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acquired therapy resistance is a dynamic process associated with early epigenetic modifications reshaping gene transcription across multiple cellular pathways, ultimately giving rise to drug-tolerant persister (DTP) cells. Unraveling the mechanisms that sustain DTP cell survival and drive their evolution into stable drug-resistant cells (DRC) is crucial for developing targeted therapies. Cisplatin-tolerant and cisplatin-resistant models were established using liver and gastric cancer cell lines for the first time to explore these mechanisms. Our investigation centered on the distinct epigenetic landscapes of DTP and DRC cells following cisplatin exposure. RNA sequencing revealed that DTP cells exhibit downregulation of pathways involved in cell cycle regulation, DNA replication, transcription, and chromatin maintenance while upregulating those associated with cell–cell communication and cytokine signaling. Interestingly, these transcriptional changes revert in the DRC state, suggesting a high degree of plasticity during the DTP phase. Furthermore, DTP cells have elevated levels of heterochromatin markers, H3K9me3, H3K27me3, and HP1α, along with their methyltransferases, G9a and Ezh2. Knockdown studies and inhibition of the enzyme activity of these modifiers showed suppression of DTP cell emergence. Valproic acid (VPA), a phase III candidate, was further assessed in vivo, where its sequential administration with cisplatin significantly reduced tumor burden versus cisplatin alone. These findings highlight the therapeutic promise of targeting epigenetic modifications to pre-sensitize cancer cells to chemotherapy, thereby restricting the survival advantage of DTP cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01966-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649db90cb654c2e41c431aeb4e6b9016419e7a71" target='_blank'>
              The synergistic action of HDAC inhibitor with cisplatin impedes survival and proliferation of drug-tolerant persister in gastric and liver cancer cells
              </a>
            </td>
          <td>
            Anjali Singh, Abhiram Natu, Flevia Anthony, Hemalatha Muthu, B. Khade, Duane T. Smoot, Hassan Ashktorab, Sanjay Gupta
          </td>
          <td>2025-10-08</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Cancer cell plasticity, defined as the ability of tumor cells to reversibly adopt distinct functional states, plays a central role in tumor heterogeneity, therapy resistance, and disease relapse. This process enables cells to enter stem-like, dormant, or drug-tolerant persister states in response to treatment or environmental stress without undergoing genetic changes. Such reversible transitions complicate and limit current treatments. Conventional cancer models often fail to capture the complexities of these adaptive states. In contrast, patient-derived tumor organoids (PDOs), which retain the cellular diversity and structure of primary tumors, provide a unique system for investigating plasticity. This review describes how PDOs can model cellular plasticity, such as the emergence of drug-tolerant persister cells and the interconversion between cancer stem cell states across multiple tumor types. We particularly focused on colorectal cancer organoids, for which research on the mechanism of plasticity is the most advanced. Combined with single-cell analysis, lineage tracing, and functional assays, PDOs can help identify the molecular pathways that control plasticity. Understanding these mechanisms is important for developing therapies to prevent treatment failure and control disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd4e7e497e61cac197e5ff80d3e09c68cd18937b" target='_blank'>
              Patient-Derived Tumor Organoids to Model Cancer Cell Plasticity and Overcome Therapeutic Resistance
              </a>
            </td>
          <td>
            R. Coppo, Masahiro Inoue
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we demonstrate that EZH2 regulates multilineage cell states dependent on the RNA binding protein Tristetraprolin (TTP) that mediates RNA stability and activation of translation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration or enzalutamide. Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1223372b66c898bfc66493d1d8cf7781f04f51" target='_blank'>
              EZH2-TTP-mTORC1 Axis Drives Phenotypic Plasticity and Therapeutic Vulnerability in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Leigh Ellis, Beatriz German, Katherine L. Morel, Teia Noel, Nadia Boufaied, Deborah L. Burkhart, Sujun Chen, Felipe Dezem, Xintao Qiu, H. Long, Stefan DiFazio, S. Baca, Ayesha Shafi, Matthew L Freedman, H. Beltran, Christopher J Sweeney, H. He, Myles Brown, Jasmine T Plummer, Simon Knott, David Labbé
          </td>
          <td>2025-08-21</td>
          <td>Research Square</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da08fc2c51aefcac002abe7af73df18eb83fa23" target='_blank'>
              Integrative proteo-genomic profiling uncovers key biomarkers of lapatinib resistance in HER2-positive breast cancer.
              </a>
            </td>
          <td>
            J. Steggall, V. Rajeeve, N. Al-Subaie, A. Naeem, A. Ikram, A. Naeem, A. Hayat
          </td>
          <td>2025-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Single-cell transcriptomics has significantly advanced our ability to uncover the cellular heterogeneity of tumors. A key challenge in single-cell transcriptomics is identifying cancer cells and, in particular, distinguishing them from non-malignant cells of the same cell lineage. Focusing on features that can be measured by single-cell transcriptomics, this review explores the molecular aberrations of cancer cells and their observable readouts at the RNA level. Identification of bona fide cancer cells typically relies on three main features, alone or in combination: i) expression of cell-of-origin marker genes; ii) inter-patient tumor heterogeneity; iii) inferred copy-number alterations. Depending on the cancer type, however, alternative or additional features may be necessary for accurate classification, such as single-nucleotide mutations, gene fusions, increased cell proliferation, and altered activation of signaling pathways. We summarize computational approaches commonly applied in single-cell analysis of tumoral samples, as well as less explored features that may aid the identification of malignant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86abedac67c16f4a560937fab77dcbb2e53869a5" target='_blank'>
              Identification of malignant cells in single-cell transcriptomics data
              </a>
            </td>
          <td>
            M. Andreatta, J. Garnica, Santiago J. Carmona
          </td>
          <td>2025-08-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, Andrew B. Stergachis, Rachel J. O’Neill, K. Miga, Steven L Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>81</td>
        </tr>

        <tr id="The molecular basis of therapy resistance in multiple myeloma (MM) remains poorly understood. Here, we performed single-cell RNA sequencing coupled with VDJ-targeted sequencing of highly purified primary MM cells from patient bone marrow. This approach uncovered cellular heterogeneity and phenotypic plasticity along the CD138 axis, accompanied by drastic epigenetic alterations. Notably, therapy-resistant subpopulations were identified within a minor fraction of CD138- MM cells, and were shown via CRISPR/Cas9 screening to be vulnerable to splicing pathway inhibition. Consistently, this fraction of CD138- MM cells showed increased differential splicing associated with overexpression of SR protein family splicing factors. Among these splicing factors, RNA-binding protein 39 (RBM39) was specifically overexpressed in therapy-resistant cells and involved in aberrant splicing. Both genetic and pharmacological RBM39 inhibition exhibited a significant selective lethal effect on therapy-resistant CD138- MM cells. Collectively, our findings identify distinct therapy-resistant MM subpopulations and highlight the splicing pathway as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dfb101a52f0fa47f40e1c355892c0f32082d1b" target='_blank'>
              Identification of a CD138-negative therapy-resistant subpopulation in multiple myeloma with vulnerability to splicing factor inhibition.
              </a>
            </td>
          <td>
            Takahiro Kamiya, Masahiko Ajiro, M. Oshima, S. Koide, Yaeko Nakajima-Takagi, Kazumasa Aoyama, Akiho Tsuchiya, Satoshi Kaito, Naoki Itokawa, Ryoji Ito, Kiyoshi Yamaguchi, Yoichi Furukawa, Bahityar Rahmutulla, Atsushi Kaneda, Takayuki Shimizu, N. Doki, Taku Kikuchi, Nobuhiro Tsukada, Masayuki Yamashita, Shinichiro Okamoto, Akihide Yoshimi, Keisuke Kataoka, Atsushi Iwama
          </td>
          <td>2025-09-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The human genome is partitioned into functional compartments that replicate at specific times during the S-phase. This temporal program, referred to as replication timing (RT), is co-regulated with the 3D genome organization, is cell type-specific, and changes during development in coordination with gene expression. Moreover, RT alterations are linked to abnormal gene expression, genome instability, and structural variation in multiple diseases, including cancer. However, mechanistic links between RT, large-scale 3D genome architecture, and transcriptional regulation remain poorly understood. A major limitation is that current approaches require the separate profiling of RT and transcriptomes from independent batches of samples, obscuring the complex co-regulation between the epigenome and transcriptome. Here, we developed PARTAGE, a multiomics approach that enables joint profiling of copy number variation (CNV), RT, and gene expression from the same sample, providing a more accurate integrative view of the complex relationships between RT and gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/957eda64f84a18622679de092756bc343443ffa1" target='_blank'>
              PARTAGE: Parallel analysis of replication timing and gene expression
              </a>
            </td>
          <td>
            Lakshana Sruthi Sadu Murari, Quinn Dickinson, Silvia Meyer-Nava, J. Rivera-Mulia
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Multiple Myeloma (MM) is a malignant plasma cell dyscrasia that progresses through the consecutive asymptomatic, often undiagnosed, precancerous stages of Monoclonal Gammopathy of Undetermined Significance (MGUS) and Asymptomatic Multiple Myeloma (SMM). MM is characterized by low survival rates, severe complications and drug resistance; therefore, understanding the molecular mechanisms of progression is crucial. This study aims to detect genetic mutations, both germline and somatic, that contribute to disease progression and drive tumorigenesis at the final stage of MM, using samples from patients presenting MGUS or SMM, and newly diagnosed MM patients. Methods: Mutations were identified through a fully computational pipeline, implemented in a Linux and RStudio environment, applied to each patient sequence, obtained through single-cell RNA-sequencing (scRNA-seq), separately. Structural and functional mutation types were identified by stage, along with the affected genes. The analysis included quality control, removal of the Unique Molecular Identifiers (UMIs), trimming, genome mapping and result visualization. Results: The findings revealed frequent germline and somatic mutations, with distinct structural and functional patterns across disease stages. Mutations in key genes were identified, pointing to molecules that may play a central role in carcinogenesis and disease progression. Notable examples include the HLA-A, HLA-B and HLA-C genes, as well as the KIF, EP400 and KDM gene families, with the first four already confirmed. Comparative analysis between the stages highlighted molecular transition events from one stage to another. Emphasis was given to novel genes discovered in newly diagnosed MM patients, that might contribute to the tumorigenesis that takes place. Conclusions: This study contributes to the understanding of the genetic basis of plasma cell dyscrasias and the transition events between the stages, offering insights that could aid in early detection and diagnosis, guide the development of personalized therapeutic strategies, and improve the understanding of mechanisms responsible for resistance to existing therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db8c2ca600e5f8d213e747b7e35b473639690b95" target='_blank'>
              Genomic Insights into Tumorigenesis in Newly Diagnosed Multiple Myeloma
              </a>
            </td>
          <td>
            Marina Kyriakou, C. Papaloukas
          </td>
          <td>2025-08-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Through most of medical history, treatments for metastatic cancers were ineffective, and rapid patient death was inevitable. Over the past five decades, a worldwide drug development effort has introduced a remarkable range of new cancer treatment strategies and agents so that virtually all metastatic cancers have one or more effective therapeutic options to prolong life. Yet most metastatic cancers remain fatal, and increasingly, the proximate cause of death is evolution. Local or systemic therapies applied to large, heterogeneous cancer populations elicit complex short- and long-term adaptive responses. Cells already possessing the molecular machinery of resistance obtain a stepwise fitness benefit relative to treatment-sensitive cells, allowing increased proliferation. Cells, otherwise sensitive to the treatment, may survive when in epigenetic states resistant to the treatment-induced death pathway or microenvironmental conditions that reduce drug delivery/efficacy, followed by a transition to "hard-wired" resistance allowing proliferation. These dynamics, enabled by the vast information content of the human genome, can produce diverse adaptive strategies in response to virtually all available treatments. Thus, oncology is rapidly approaching an era in which patient death is caused not by the absence of effective therapies but rather by eco-evolutionary dynamics that defeat initially successful treatments. Emerging evidence suggests that explicit integration of evolutionary principles to control or eliminate resistant populations can improve outcomes. In this issue of Cancer Research, Hockings and colleagues present an important evolutionary strategy to delay or prevent the evolution of resistance in ovarian cancer, with broad potential application. See related article by Hockings et al., p. 3503.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa08f9c64d9497ce73be1df571e19d588bc63780" target='_blank'>
              The Approaching Darwinian Era in Oncology.
              </a>
            </td>
          <td>
            Robert A. Gatenby
          </td>
          <td>2025-09-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Current cancer therapies often fail due to tumor heterogeneity and rapid resistance evolution. A new evolutionary framework, ‘selection for function,’ proposes that tumor progression is driven by group phenotypic composition (GPC) and its interaction with the microenvironment, not by individual cell traits. This perspective opens new therapeutic avenues: targeting the tumor’s functional networks rather than individual cells. Real-time tracking of GPC changes could inform adaptive treatments, delaying progression and resistance. By integrating evolutionary and ecological principles with conventional therapies, this strategy aims to transform cancer from a fatal to a manageable chronic disease. Crucially, it does not necessarily require new drugs but offers a way to repurpose existing therapies to impair a tumor’s evolutionary potential. By steering tumor evolution toward less aggressive states, this approach could improve prognosis and long-term patient survival compared to current methods. We argue that leveraging GPC dynamics represents a critical, yet underexplored, opportunity in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53fd3b887f1a92efe08b517e2a13e55225513a1" target='_blank'>
              Leveraging selection for function in tumor evolution: System-level cancer therapies
              </a>
            </td>
          <td>
            F. Thomas, Jean-Pascal Capp, A. Dujon, A. Marusyk, K. Asselin, M. Campone, Pascal Pujol, Catherine Alix-Panabières, Benjamin Roche, B. Újvári, Robert Gatenby, A. Nedelcu
          </td>
          <td>2025-08-18</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa96a0707b7c6c78fe5bd4b2a45727093019bee" target='_blank'>
              Targeting the crosstalk of metabolism reprogramming and replication stress: novel strategy to combat cancer.
              </a>
            </td>
          <td>
            Wenbo Liu, Xiangyan Jiang, Yong Ma, Huiguo Qing, Yingcun Bao, Zuoyi Jiao
          </td>
          <td>2025-09-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The discovery of genomics has redefined oncology into precision rather than standardized treatment. But
therapeutic resistance of either kind, intrinsic or acquired, is a serious impediment to sustained success. The present review
outlines the domains on the crossroad between genomic intelligence and resistance evolution, where multi-omics profiling,
high-throughput sequencing, and AI-based analytics are explaining the challenging complexity of the tumor and predicting
resistance pathways. We review the contribution of clonal evolution, tumor plasticity and adaptive signaling in resistance to
therapy, and we promote moving towards active molecular monitoring rather than snapshot ting genome. To preempt and
target therapeutic escape we suggest a framework of adaptive precision oncology that comprehends real-time biomarkers,
liquid biopsy follow-up and resistance-predictive algorithms. New approaches like combination therapy, recalibration of
treatment with the help of AI, and longitudinal monitoring of genomes are mentioned. We also deal with moral and logistical
obstacles of adopting these strategies, and especially in low-resource environments. The new era of precision oncology We
now find ourselves in the next era of precision oncology that aims to personalization extend to real-time responsiveness, so
that cancer treatment can increasingly be more adaptable, more resilient and more durable, achieved through personalizing
the rapidly changing tumor biology with real-time dynamically changing genomic insights.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7508321bc4abb3899cb6fc37344230d9a86147e" target='_blank'>
              Genomic Intelligence and Resistance Evolution: Redefining Oncotherapeutic Strategies in Precision Oncology
              </a>
            </td>
          <td>
            Muchukota Sushma, Bharathi Bhogenahalli Venkatappa, Gowthami V., Nigel Viju Thomas, Souman Samanta
          </td>
          <td>2025-08-28</td>
          <td>International Journal of Innovative Science and Research Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study explores the molecular impact of cisplatin, Actemra (tocilizumab), remdesivir, and SARS-CoV-2 infection on colorectal cancer (CRC) SW-480 cells using RNA sequencing (RNA-Seq). Differential gene expression analysis revealed treatment-specific transcriptional changes. Cisplatin led to the downregulation of genes involved in lipid metabolism and focal adhesion, while remdesivir upregulated chromatin remodeling pathways. SARS-CoV-2 infection altered cytokine signaling, particularly IL-17 and TNF-α. Principal component and hierarchical clustering analyses confirmed distinct gene expression profiles across treatments. ELISA assays validated the reduction of ACE2 and CD147 protein levels following drug treatment. MTT assays demonstrated remdesivir’s cytotoxicity at high doses. These findings highlight potential drug-gene interactions and suggest that combining transcriptomic profiling with functional assays may guide personalized therapeutic strategies for CRC patients, especially those co-infected with SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eae066da47093f217085c9697d40cc942624f2a" target='_blank'>
              RNA sequencing reveals transcriptional signatures of drug response and SARS-CoV-2 interaction in colorectal cancer cells
              </a>
            </td>
          <td>
            Alaa K. Hameed, Ifad K. Abd Al-Shibly, Rana A. Ghaleb
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2cfb28da058ae0d8b29e85290850287448ab79" target='_blank'>
              Transition between cell states of sensitivity reveals molecular vulnerability of drug-tolerant cells.
              </a>
            </td>
          <td>
            L. Peyre, Marielle Péré, Mickael Meyer, Benjamin Bian, Marina Moureau-Barbato, Walid Djema, B. Mari, Georges Vassaux, Jérémie Roux
          </td>
          <td>2025-10-01</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="A complex and gradual process, the epithelial–mesenchymal transition (EMT) occurs both during embryonic development and tumor progression. Cells undergo a transition from an epithelial to a mesenchymal state throughout this process. More and more evidence points to EMT as a cause of increased metastatic spread of prostate cancer (PCa), along with stemness enhancement and therapy resistance. Here, we used bioinformatic methods to analyze gene expression microarray data, single-cell RNA sequencing, oncogenes, and tumor suppressor genes (TSGs) in order to reconstruct the network of differentially expressed genes (DEGs) involved in the epithelial–mesenchymal transition with PCa. No prior study has documented this sort of analysis. We next validated our results using data from the Cancer Genome Atlas (TCGA), which included microarray and single-cell RNA sequencing. Potentially useful in PCa diagnosis and treatment are extracellular matrix in epithelial–mesenchymal transition genes, including ITGBL1, DSC3, COL4A6, ANGPT1, ARMCX1, MICAL2, and EPHA5. In this study, we aimed to shed light on the molecular characteristics and pathways of DEGs in PCa, as well as to identify possible biomarkers that are important in the development and advancement of this cancer. These insights have important implications for understanding prostate cancer progression and for the development of therapeutic strategies targeting ECM-mediated pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15775ad90d8cab3fed2c521129b0f7f8c9fd8d19" target='_blank'>
              Analysis of Microarray and Single-Cell RNA-Seq Finds Gene Co-Expression and Tumor Environment Associated with Extracellular Matrix in Epithelial–Mesenchymal Transition in Prostate Cancer
              </a>
            </td>
          <td>
            Ali Shakeri Abroudi, Mahtab Mashhouri Moghaddam, Danial Hashemi Karoii, Melika Djamali, Hossein Azizi, Thomas Skutella
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Genetic variants (both coding and noncoding) can impact gene function and expression, driving disease mechanisms such as cancer progression. The systematic study of endogenous genetic variants is hindered by inefficient precision editing tools, combined with technical limitations in confidently linking genotypes to gene expression at single-cell resolution. We developed single-cell DNA–RNA sequencing (SDR-seq) to simultaneously profile up to 480 genomic DNA loci and genes in thousands of single cells, enabling accurate determination of coding and noncoding variant zygosity alongside associated gene expression changes. Using SDR-seq, we associate coding and noncoding variants with distinct gene expression in human induced pluripotent stem cells. Furthermore, we demonstrate that in primary B cell lymphoma samples, cells with a higher mutational burden exhibit elevated B cell receptor signaling and tumorigenic gene expression. SDR-seq provides a powerful platform to dissect regulatory mechanisms encoded by genetic variants, advancing our understanding of gene expression regulation and its implications for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a9aba01e0cb2e8c3213b471e7d8f34414b59cc" target='_blank'>
              Functional phenotyping of genomic variants using joint multiomic single-cell DNA–RNA sequencing
              </a>
            </td>
          <td>
            Dominik Lindenhofer, Julia R. Bauman, John A. Hawkins, Donnacha Fitzgerald, Umut Yildiz, Haeyeon Jung, Anastasiia Korosteleva, Mikael Marttinen, M. Kueblbeck, Judith B. Zaugg, K. Noh, Sascha Dietrich, W. Huber, O. Stegle, Lars M. Steinmetz
          </td>
          <td>2025-09-01</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K J Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Phenotypic plasticity is a key mechanism of metastatic progression and cancer therapy resistance. This hallmark of human malignancies is enabled by highly conserved epigenetic mechanisms that control gene expression. Functional alterations in DNA methylation and histone post-translational modifications have been extensively described as drivers of metastatic dissemination and therapy resistance. Pharmacological inhibitors of epigenetic enzymes can revert these alterations, thereby stopping cancer progression and counteracting the emergence of resistant clones. Despite promising pre-clinical evidence, the clinical implementation of epigenetic therapies in solid cancers has led to disappointing results. Several factors can explain these challenges, including the lack of rational combinations. Notably, response to epigenetic treatments can be heterogeneous and short-lived. A liquid biopsy technology that allows the measure of specific epigenetic alterations enables patient selection and therapy monitoring, leading to the development of precision epigenetic therapies. In this review, we discuss the state of the art of this emerging treatment modality, and we identify key challenges that need to be overcome to reach the full potential of this new therapeutic concept.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876ad36ab7595e359c53da7e87448ed401afa714" target='_blank'>
              Precision epigenetic therapies in oncology
              </a>
            </td>
          <td>
            Arundhati Chaudhary, Kayleigh J A Orchard, F. Salani, Theodora Partsou, Mark Eccleston, Guido Bocci, A. Italiano, Francesco Crea
          </td>
          <td>2025-09-19</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Introduction Chemotherapy response variability in cancer patients necessitates novel strategies targeting chemoresistant populations. While combinatorial regimens show promise through synergistic pharmacological interactions, traditional pathway enrichment methods relying on static gene sets fail to capture drug-induced dynamic transcriptional perturbations. Methods To address this challenge, we developed the Pathway-Responsive Gene Sets (PRGS) framework to systematically identify chemoresistance-associated pathways and guide therapeutic intervention. Comparative evaluation of three computational strategies (GSEA-like method, Hypergeometric test-based method, Bates test-based method) revealed that the GSEA-like methodology exhibited superior performance, enabling precise identification of drug-induced pathway dysregulation. Results Key experimental findings demonstrated PRGS’s superiority over conventional Pathway Member Gene Sets (PMGS), exhibiting statistical independence (p < 0.0001) and enhanced detection of chemotherapy-driven pathway dysregulation. Application of PRGS to the GDSC dataset identified 8 resistance-associated pathways. Screening of agents targeting these pathways yielded candidates with predicted anti-resistance activity. An in vitro cellular experiment demonstrated that the bortezomib-bleomycin combination exhibited synergistic cytotoxicity (IDAcomboScore = 0.014) in T47D cells, highlighting the potential of PRGS-guided therapeutic strategies. Discussion This study establishes a PRGS-based methodological framework that integrates genomic perturbations with precision oncology, demonstrating its capacity to decode resistance mechanisms and guide therapeutic development through dynamic pathway analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/814bf51ab320b8d56ddbcfefe1cd9a9aa4f449e3" target='_blank'>
              Drug discovery for chemotherapeutic resistance based on pathway-responsive gene sets and its application in breast cancer
              </a>
            </td>
          <td>
            Dehua Feng, Jingwen Hao, Lingxu Li, Jian Chen, Xinying Liu, Ruijie Zhang, Huirui Han, Tianyi Li, Xuefeng Wang, Xia Li, Lei Yu, Bing Li, Jin Li, Limei Wang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb5a38dd74bce74d21cd64c9a442cf5e375b46f4" target='_blank'>
              Targeting Glioblastoma Cell State Plasticity for Enhanced Therapeutic Efficacy.
              </a>
            </td>
          <td>
            S. Cirigliano, R. Singhania, James G Nicholson, Isha Monga, Yushan Wan, C. Haywood, Ashlesha Muley, Skylar A. Giacobetti, Howard A. Fine
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The DNA damage response (DDR), an evolutionarily conserved surveillance mechanism enabling cells to detect genomic lesions and cell cycle arrest for repair, coordinates genomic stability through key effector kinases. In Candida albicans, our prior work characterized Rad53-mediated transcriptional reprogramming based on the RAD53 deletion strain, establishing its canonical DDR functions. However, Rad53 activation dynamically orchestrates cell cycle synchronization and repair coordination during genotoxic challenge. Investigating cellular modifications resulting from increased Rad53 functionality is essential for a deeper understanding of its regulatory response. To this end, we overexpressed Rad53 revealing its dual regulatory capacity in stress adaptation and developmental plasticity. Global transcriptomic profiling demonstrated that RAD53 overexpression enhances methyl methanesulfonate (MMS) resistance while paradoxically stimulating filamentous growth, accompanied by upregulation of DNA repair/replication genes (RFA3, POL30) and non-canonical DDR targets including cell wall remodeling factors. Comparative analysis of RAD53 deletion and overexpression transcriptomes identified enriched transcription factors, notably Sfl1, Yox1, and Rfx1. In particular, Rfx1 mediated the resistance to genotoxic stress resulting from RAD53 overexpression or deletion. Surprisingly, Rad53 overexpression substantially hindered hyphal formation and virulence, whereas its kinase-dead mutation restored these deficiencies. Notably, Rad53's role in mediating morphogenesis was independent of Rfx1. This multi-modal functionality positions Rad53 as a critical node interfacing genomic stability, developmental plasticity, and pathogenicity in C. albicans. Our study enhances the understanding of the DDR pathway in C. albicans, providing a potential target for treating this fungal pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/618ff870ecbc422dc2701275aa24d5074e975660" target='_blank'>
              Rad53 orchestrates divergent pathways governing genotoxic resistance, morphogenesis, and virulence in Candida albicans: Mechanistic decoupling through overexpression analysis.
              </a>
            </td>
          <td>
            Jia Wang, Shaling Yang, Yi Zhou, Huaxin Cai, Jinrong Feng
          </td>
          <td>2025-10-06</td>
          <td>Virulence</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolic plasticity is a hallmark of cancer, enabling tumour cells to grow and adapt to microenvironmental stress, eventually contributing to tumour heterogeneity. Although glycolysis-oxidative phosphorylation (OXPHOS) switch plays a pivotal role, emerging evidence highlights OXPHOS as an essential mechanism for cancer survival, particularly during metastasis and therapeutic stress, underscoring the complexities underlying metabolic plasticity and tumour heterogeneity. The role of glycolysis-OXPHOS switch in cancer stem cells (CSCs), a highly aggressive and drug-resistant population frequently enriched in response to the stress of tumour growth and pressure from microenvironmental or therapeutic cues, remains an open question with therapeutic interventions yielding mixed outcomes. While some strategies suppress CSC activity, others inadvertently promote resistance and tumour aggressiveness, thus contributing to treatment failure and relapse. This review critically examines the role of glycolysis-OXPHOS switch as a gatekeeper of tumorigenesis which influences CSC plasticity and resistance. By dissecting these metabolic dynamics, it aims to inform novel therapeutic strategies, emphasising tailored approaches to target CSC plasticity and improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0e3d7e96b962631a0fd41c99ab50564c947225" target='_blank'>
              Metabolic Plasticity and Cancer Stem Cell Metabolism: Exploring the Glycolysis-OXPHOS Switch as a Mechanism for Resistance and Tumorigenesis
              </a>
            </td>
          <td>
            Li Qing Keoh, Ching-Feng Chiu, T. S. Ramasamy
          </td>
          <td>2025-08-29</td>
          <td>Stem Cell Reviews and Reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="BACKGROUND
Multiple myeloma (MM) is a malignancy marked by uncontrolled plasma cell proliferation, immune evasion, and drug resistance. Despite advances in treatment, the disease remains incurable due to relapses and drug resistance. This study aims to investigate the molecular and cellular interactions within the myeloma microenvironment using single-cell RNA sequencing (scRNA-seq), Mendelian randomization (MR), and pathway analysis to uncover therapeutic targets.


METHODS
We used scRNA-seq to analyze tumor, immune, and stromal cell interactions in MM. Data were processed using Seurat for clustering, dimensionality reduction, and cell-type annotation. Gene Set Variation Analysis (GSVA) and cell-cell interaction analysis were performed to identify signaling pathways involved in disease progression. Two-sample MR was applied to explore causal relationships between genetic variants and gene expression. Finally, molecular docking simulations were used to identify potential small molecule modulators of key proteins involved in MM.


RESULTS
We identified significant immune-related signaling pathways, including PI3K-AKT-mTOR, WNT-β-catenin, and TGF-β, upregulated in immune cells within the MM microenvironment. Genes such as HLA-C, CTSS, and LRRFIP1 showed positive causal relationships with MM, while SHISA5 and ISG15 exhibited protective roles. Cell communication analysis revealed key ligand-receptor interactions between immune and tumor cells. Molecular docking identified promising small molecules like actein and aflatoxin B1 targeting ISG15 and TAGLN2.


CONCLUSIONS
This study reveals key genetic drivers and immune modulation mechanisms in MM. Targeting immune-related pathways, such as PI3K-AKT-mTOR and WNT-β-catenin, and small molecules targeting ISG15 and TAGLN2 could offer new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ddcfc1e5a8f4f32f9fd598c516ebb1c5e5f270" target='_blank'>
              Single-cell RNA sequencing reveals immune regulatory mechanisms and molecular therapeutic strategies in the microenvironment of multiple myeloma.
              </a>
            </td>
          <td>
            Qiang Su, Kehan Long, Mozili Adu, Meijun Jiang, Qiaochu Li, Xufeng Wan, Jian Cao, Yan Yue, Shuoyuan Li, Zhendong Ying, Qibin Liu, Chaoqun You, Zhuang Zhang, Duan Wang
          </td>
          <td>2025-09-03</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer drug resistance remains a major barrier to durable treatment success, often leading to relapse despite advances in precision oncology. While combination therapies are being increasingly investigated, such as chemotherapy with small molecule inhibitors, predicting drug response and identifying rational drug combinations based on resistance mechanisms remain major challenges. Therefore, a proteome-wide, single-gene overexpression screening platform is essential for guiding rational therapy selection. Here, we present BOGO (Bxb1-landing pad human ORFeome-integrated system for a proteome-wide Gene Overexpression), a robust, scalable, and reproducible screening platform that enables single-copy, site-specific integration and overexpression of ~19,000 human ORFs across cancer cell models. Using BOGO, we identified drug-specific response drivers for 16 chemotherapeutic agents and integrated clinical datasets to uncover proliferation and resistance-associated genes with prognostic potential. Drug response similarity networks revealed both shared and unique mechanisms, highlighting key pathways such as autophagy, apoptosis, and Wnt signaling, and notable resistance-associated genes including BCL2, POLD2, and TRADD. In particular, we proposed a synergistic combination of the BCL2 family inhibitor ABT-263 (Navitoclax®) and the DNA analog TAS-102 (Lonsurf®), which revealed that lysosomal modulation is a key mechanism driving DNA analog resistance. This combination therapy selectively enhanced cytotoxicity in colorectal and pancreatic cancer cells in vitro, and demonstrated therapeutic benefit in vivo in both cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models. Together, these findings establish BOGO as a powerful gene overexpression perturbation platform for systematically identifying chemoresistance and chemosensitization drivers, and for discovering rational combination therapies. Its scalability and reproducibility position BOGO as a broadly applicable tool for functional genomics and therapeutic discovery beyond cancer resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb42d8c647bdd3aef172a4ebd8dbd580fe7fe538" target='_blank'>
              BOGO: A Proteome-Wide Gene Overexpression Platform for Discovering Rational Cancer Combination Therapies
              </a>
            </td>
          <td>
            Kyeong Beom Jo, M. Alruwaili, Da-Eun Kim, Yongjun Koh, Hyeyeon Kim, Kwontae You, Ji-Sun Kim, Saba Sane, Yanqi Guo, Jacob P. Wright, H. Lim, Maricris N. Naranjo, Atina G. Coté, Frederick P. Roth, David E. Hill, Jung-Hyun Choi, Hunsang Lee, Kenneth A. Matreyek, Kyle K.-H. Farh, Jong-Eun Park, Hyunkyung Kim, Andrei V. Bakin, Dae-Kyum Kim
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In Mycobacterium tuberculosis, a prevalent and deadly pathogen, resistance to antibiotics evolves primarily through non-synonymous mutations in proteins. Sequence-based analyses are currently used to understand the genetic basis of antibiotic resistance, either via genotype-phenotype association, or via signals of convergent evolution. These methods focus on primary sequence and usually neglect other biological signals such as protein structural information. We hypothesize that integrating the structural context of mutations improves the prediction of effects on function and phenotype. We curate high confidence structural annotations for the M. tuberculosis proteome from 1,371 crystallography and 2,316 AlphaFold predictions, and combine the structures with mutations from over 31,000 clinical M. tuberculosis isolates. We demonstrate that mutations in proteins known to cause resistance are clustered in 3D space, even in proteins where inactivating mutations at any position are thought to cause resistance. We develop a statistic to search the M. tuberculosis proteome for signal of clustered non-synonymous mutations, finding over 450 proteins that display this signal, many of which have a known relationship with antibiotic resistance. Innovatively, we show that a supervised classifier trained on structure features alone has an F1 score of 94.6% at classifying mutations as resistance-conferring. This work demonstrates that protein structure provides useful information for categorizing which variants may cause antibiotic resistance, even when the majority of structures are AI-predicted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/736eca4109e1243533868a8b023c33d1b51f37d5" target='_blank'>
              The structural context of mutations in proteins predicts their effect on antibiotic resistance
              </a>
            </td>
          <td>
            Anna G Green, Mahbuba Tasmin, R. Vargas, M. Farhat
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Hypoxia is a well-recognized clinical feature of solid tumors, including lung cancer, and is associated with poor prognosis due to its role in promoting resistance to chemotherapy and radiotherapy. To investigate the cellular consequences of hypoxia, we cultured A549 lung adenocarcinoma cells under 1% O2 and examined their growth, cell cycle distribution, and redox status. Hypoxia significantly reduces cell proliferation and induced G1 phase cell cycle arrest, suggesting a cytostatic effect. Biochemical analysis showed a 2.64- and 2.04-fold increase in total and mitochondrial reactive oxygen species (ROS) levels, respectively, along with an elevated total thiol levels under hypoxic conditions compared to normoxia. To assess the reversibility of the hypoxic response, we performed a reciprocal oxygen exposure experiment where cells initially grown under hypoxia were re-exposed to normoxia, and vice versa. To explore the underlying molecular mechanism, we analyzed transcriptomic datasets (GEO accession: GSE48134 and GSE42416) which revealed that hypoxia downregulated key genes involved in energy metabolism (e.g., PDK4, G6PD), cell cycle progression (e.g., CCND1, CDK2), and redox regulation (e.g., GCLM, TXNRD1, NQO1, GCLC). Further, few of redox-related genes were validated by RT-PCR in A549 cells cultured under hypoxia and normoxia for 24 h. Importantly, cyclic hypoxia (intermittent hypoxia-reoxygenation) conditions showed partial restoration of some of these transcripts, supporting the transient nature of hypoxic stress, consistent with our in vitro observations. Furthermore, transcriptome profiles from adenocarcinoma patients (GEO accession: GSE30979) also match our cell line observations. Thus, our results clearly show that hypoxia causes a temporary cell cycle arrest in lung cancer cells, which is reversible when oxygen is restored. Supplementary Information The online version contains supplementary material available at 10.1007/s12032-025-03058-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77bbcdb23f2ddac965c8e06176945c4fad76ded" target='_blank'>
              Hypoxia-drives reversible cell cycle arrest in lung cancer cells via modulation of cellular redox and gene expression
              </a>
            </td>
          <td>
            D. Maurya, Varshita Mehta, Babita Singh
          </td>
          <td>2025-09-29</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, characterized by cells that exhibit 'monster traits' such as uncontrolled proliferation, invasiveness, resistance to apoptosis, and immune evasion. Traditional therapies often target single hallmarks of cancer, leading to limited success and eventual resistance. This paper proposes a novel, integrative therapeutic framework inspired by evolutionary adaptations in species with remarkable cancer resistance, such as elephants (Loxodonta Africana). Elephants possess enhanced TP53-mediated cancer surveillance and DNA repair mechanisms, which contribute to their extremely low cancer incidence. We hypothesize that by synthetically recapitulating these 'anti-monster' factors—such as augmented p53 activity, enhanced DNA repair, and controlled inflammation—in human cells, we can reprogram malignant phenotypes towards benignity. Our proposed strategy combines synthetic biology, gene therapy (including CRISPR/Cas systems), and multi-targeted pharmacological approaches to simultaneously address multiple cancer hallmarks. By leveraging advanced *in vitro* organoid and *in vivo* models, this research aims to pioneer a new paradigm in oncology that moves beyond cytotoxic elimination towards cellular reprogramming and durable cancer prevention, potentially yielding a significant reduction in lifetime cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c541ffbed17a7ff84c98c50ab50d257ba1c2dd4" target='_blank'>
              Engineering Cancer Resistance: A Novel Framework Inspired by Evolutionary Adaptations
              </a>
            </td>
          <td>
            Hafiz Muhammadismail Azher Syed
          </td>
          <td>2025-09-29</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression heterogeneity generates subpopulations of tumor cells that can evade therapeutic pressure. This heterogeneity has been observed in both primary Estrogen Receptor alpha positive (ERα+) breast tumors and cell lines. Therefore, understanding the mechanisms regulating expression heterogeneity is critical towards developing effective therapies. A key contributor to gene expression variability is the stochastic nature of transcription. Transcription occurs in a probabilistic, burst-like manner, in which gene activation occurs intermittently, producing RNA in pulses and interspersed with transcriptional off-periods. The estrogen-responsive gene TFF1 is expressed in the majority of ERα+ breast tumors and exemplifies such heterogeneity, with transcriptional inactivity ranging from minutes to several days. Here, we identify the molecular mechanism underlying the wide range in TFF1 expression by analyzing cells sorted based on their TFF1 activity levels. We observed that TFF1 inactive (TFF1low) cells exhibit a repressive chromatin state marked by H3K27me3 at the TFF1 promoter and enhancer. Despite global similarity in ERα binding, occupancy at the TFF1 regulatory elements was selectively reduced in TFF1low cells, resulting in fewer active alleles and diminished transcriptional bursting frequency. Conversely, TFF1high cells exhibited more active TFF1 alleles and hyperbursting. These cells also retained sensitivity to endocrine therapy, while TFF1low cells displayed reduced drug responsiveness. Genome-wide, differentially enriched H3K27me3 regions correlated with variable expression of estrogen-responsive genes, highlighting a broader regulatory mechanism that links chromatin state to expression variability. Together, our findings establish how repressive chromatin dynamics contribute to gene expression heterogeneity and endocrine resistance in ERα+ breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa89e16be6e3a720a965e4891206ee026e2fe10" target='_blank'>
              H3K27me3 chromatin heterogeneity reveals variable cell responses to estrogen and endocrine treatment
              </a>
            </td>
          <td>
            Chapus Fleur, Christopher R. Day, Laura G Kammel, Pelin Yasar, Brian D. Bennett, Erica Scappini, Charles J Tucker, Maria Sifre, Celyn Bregio, Joseph Rodriguez
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Microsatellite-stable metastatic colorectal cancer (MSS mCRC) is currently treated with chemotherapy and targeted agents based on RAS and BRAF mutational status. Although these therapies offer initial benefit, most patients rapidly develop resistance, with fewer than 20% remaining progression-free at two years. This review aims to synthesize emerging evidence on the metabolic mechanisms driving treatment resistance in MSS mCRC, with a particular focus on the immune-metabolic signature (IMMETCOLS) classification. We conducted a comprehensive review of preclinical models, transcriptomic datasets, and clinical trial results addressing metabolic adaptations to chemotherapy and targeted therapies in MSS mCRC. The IMMETCOLS framework defines three metabolic subtypes—IMC1, IMC2, and IMC3—each associated with distinct resistance mechanisms. IMC1 exhibits glycolysis and transforming growth factor-β (TGF-β)-dependent signaling enriched in inflammatory fibroblasts, conferring resistance to chemotherapy. IMC2 relies on oxidative phosphorylation and glutamine metabolism, supporting antioxidant defenses and resistance to both cytotoxic agents and anti-EGFR therapies. IMC3 demonstrates lactate-fueled respiration and pentose phosphate pathway activation, contributing to redox balance, DNA repair, and resistance to targeted therapies such as anti-BRAF or KRAS inhibitors. All subtypes display metabolic plasticity under therapeutic pressure. Emerging clinical data support tailoring targeted therapy combinations based on IMMETCOLS subtype, particularly in BRAF- and HER2-positive populations. Understanding subtype-specific metabolic rewiring in MSS mCRC offers novel opportunities to overcome drug resistance. Targeting the metabolic vulnerabilities defined by the IMMETCOLS signature may improve response durability and inform precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662ed873ba87c81960ffe96e54fa075a4697d04f" target='_blank'>
              Metabolic plasticity drives specific mechanisms of chemotherapy and targeted therapy resistance in metastatic colorectal cancer
              </a>
            </td>
          <td>
            Mariam Rojas, Malena Manzi, S. Madurga, Fernando Enrique García Velásquez, Maira Alejandra Romero, Silvia Marín, Marta Cascante, Joan Maurel
          </td>
          <td>2025-09-23</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
B-cell precursor acute lymphoblastic leukemia (B-ALL) is a hematologic malignancy characterized by the uncontrolled proliferation of immature B lymphoblasts. Despite significant advancements in treatment, chemoresistance remains a major challenge. Previous studies, including those involving Colombian patient cohorts, have identified a gene signature involving ID1, ID3, and IGJ, which correlates with poor prognosis; however, the underlying mechanisms remain unclear. This study integrates metabolic profiling with gene expression analysis of ID1, ID3, and IGJ in patient-derived lymphoblasts. It explores the role of IGJ overexpression in NALM-6 cells to assess its potential contribution to chemoresistance.


METHODS
Bone marrow samples from 33 newly diagnosed B-ALL patients were analyzed for ID1, ID3, and IGJ gene expression. Seahorse XF metabolic assays were conducted on 13 patient samples to assess mitochondrial respiration and glycolytic function. Functional studies were performed in the NALM-6 B-ALL cell line using CRISPRa-mediated IGJ overexpression, followed by metabolic and chemoresistance assays using resazurin-based viability testing with key chemotherapeutic agents.


RESULTS
Patient-derived lymphoblasts exhibited a quiescent metabolic phenotype, with two distinct metabolic subgroups based on oxygen consumption and glycolysis rates. Higher IGJ expression was significantly associated with the glycolytic subgroup and correlated with worse event-free survival (EFS, p = 0.0179) and overall survival (OS, p = 0.0205). In NALM-6 cells, IGJ overexpression led to increased metabolic activity and conferred resistance to dexamethasone, cytarabine, doxorubicin, and methotrexate, but not cyclophosphamide.


CONCLUSION
IGJ upregulation promotes metabolic reprogramming and chemoresistance in B-ALL, suggesting a potential role for IGJ as a biomarker and therapeutic target in overcoming treatment resistance. These findings provide new insights into the metabolic mechanisms underlying B-ALL progression and highlight IGJ as a candidate for future targeted interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e198116e17749ec5fbd2ea6f272323e0f4abd014" target='_blank'>
              Integration of Metabolic Profiling and Functional Genomics Suggests IGJ as a Driver of Chemoresistance in B-ALL.
              </a>
            </td>
          <td>
            Karen Lizeth Pachón-Meza, Camilo Ernesto Moreno-Cristancho, José Luis Padilla-Agudelo, Ana Isabel Ramos-Murillo, Diana Londoño-Barbosa, Gustavo Salguero, S. Sanabria-Barrera, Claudia Chica, Nataly Cruz-Rodriguez, Rubén Darío Godoy-Silva, J. A. Gutiérrez-Triana
          </td>
          <td>2025-10-06</td>
          <td>Pediatric blood & cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is driven by a complex spectrum of somatic mutations and structural variants that contribute to tumor heterogeneity and therapy resistance. In this study, we performed a comparative analysis of short-read Illumina and long-read Nanopore sequencing technologies across multiple CRC sample groups, encompassing diverse tissue morphologies. Our evaluation included general base-level metrics—such as nucleotide ratios, sequence match rates, and coverage—as well as variant calling performance, including variant allele frequency (VAF) distributions and pathogenic mutation detection rates. Focusing on clinically relevant genes (KRAS, BRAF, TP53, APC, PIK3CA, and others), we characterized platform-specific detection profiles and completed the ground truth validation of somatic KRAS and BRAF mutations. Structural variant (SV) analysis revealed Nanopore’s enhanced ability to resolve large and complex rearrangements, with consistently high precision across SV types, though recall varied by variant class and size. To enable direct comparison with the Illumina exome panel, we applied an exonic position reference file. To assess the impact of depth and PCR amplification, we completed an additional high-coverage Nanopore sequencing run. This analysis confirmed that PCR-free protocols preserve methylation signals more accurately, reinforcing Nanopore’s utility for integrated genomic and epigenomic profiling. Together, these findings underscore the complementary strengths of short- and long-read sequencing platforms in high-resolution cancer genomics, and we highlight the importance of coverage normalization, epigenetic fidelity, and rigorous benchmarking in variant discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4037869e56706b3aeec547e05839c0679ce3d95b" target='_blank'>
              Methodological Comparison of Short-Read and Long-Read Sequencing Methods on Colorectal Cancer Samples
              </a>
            </td>
          <td>
            Nikolett Szakállas, Alexandra Kalmár, K. Rada, M. Kucarov, Tamás Richárd Linkner, B. Barták, István Takács, Béla Molnár
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genome instability is a central hallmark of cancer, driven by aberrant DNA damage responses that facilitate tumor evolution and resistance to therapy. Although canonical non‑homologous end joining and homologous recombination are well‑characterized pathways for repairing DNA double‑strand breaks (DSBs), recent advances have revealed that cancer cells increasingly depend on alternative end‑joining (alt‑EJ) to survive persistent DNA damage that arises from intrinsic stresses or external therapies. Alt‑EJ, characterized by its reliance on microhomologous sequences at DSB sites, promotes mutation accumulation and chromosomal rearrangements, thereby driving genomic instability and tumor progression. Despite its pivotal role in cancer biology, the molecular regulation, contextual determinants and dualistic role of alt‑EJ in maintaining genome integrity compared with promoting instability remain incompletely understood. The present review integrated the latest mechanistic insights into alt‑EJ, elucidated its regulatory networks and interactions with canonical DSB repair pathways and discussed its consequences for cancer genome integrity and evolution. Furthermore, it highlighted the emerging potential of alt‑EJ as a therapeutic vulnerability for cancer, underscoring the urgent need to translate these discoveries into innovative treatment strategies aimed at overcoming therapy resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50b23b0e6f34e53664100d4135bd748432a49add" target='_blank'>
              Emerging insights into alternative end‑joining: Mechanisms, genome instability and therapeutic opportunities in cancer (Review).
              </a>
            </td>
          <td>
            Nasir Azam, Xiangyu Liu, Xinghan Li, Lin Ma, Qi Liu
          </td>
          <td>2025-10-09</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aim: Urinary bladder cancer (UBC) often develops chemoresistance, reducing treatment effectiveness. This study aimed to investigate diverse molecular mechanisms underlying acquired resistance by establishing and characterizing a comprehensive panel of UBC cell lines resistant to common chemotherapeutics. Methods: Fifteen UBC cell lines were examined: three parental lines (RT-112, TCC-SUP, UMUC-3) and twelve derived sublines adapted to cisplatin, vinblastine, or gemcitabine. Drug sensitivity was assessed using the SRB assay. Resistance mechanisms were explored via quantitative real-time PCR (targeting genes including ABCB1, dCK, hENT1, ECHDC1, TUBB3), Western blotting (assessing proteins such as p21, Cyclin B, and Mcl-1), and biochemical assessment of glutathione levels and redox state. Results: The adapted sublines exhibited distinct resistance profiles and cross-resistance patterns. Gene expression and protein analyses revealed drug- and lineage-specific alterations, involving factors such as p21, Cyclin B, and Mcl-1. Changes in glutathione metabolism were also associated with resistance. Notably, no single, universal mechanism accounted for resistance across the entire panel. Conclusion: UBC cells develop diverse, context-dependent adaptive strategies to resist cisplatin, vinblastine, and gemcitabine. These findings highlight the complexity of chemoresistance mechanisms. The characterized cell line panel represents a valuable resource for future studies aimed at understanding and overcoming drug resistance in bladder cancer, suggesting that personalized therapeutic approaches may be necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fbd01421411fd7eb4380d44e0e070090cf7648" target='_blank'>
              Distinct adaptive strategies to cisplatin, vinblastine and gemcitabine in a panel of chemoresistant bladder cancer cell lines
              </a>
            </td>
          <td>
            Monika Cuprych-Belter, Agnieszka Łupicka-Słowik, Artur Anisiewicz, Martin Michaelis, J. Cinatl jr., Mateusz Psurski
          </td>
          <td>2025-09-12</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="High-throughput clonal tracing of primary human samples relies on naturally occurring barcodes, such as somatic mitochondrial DNA (mtDNA) mutations detected via single-cell ATAC-seq (mtscATAC-seq). Fresh-frozen clinical specimens preserve tissue architecture but compromise cell integrity, thereby precluding their use in multiomic approaches such as mitochondrial genotyping at single-cell resolution. Here, we introduce Cryo-mtscATAC-seq, a broadly applicable method for diverse pathophysiological contexts to isolate nuclei with their associated mitochondria (“CryoCells”) from frozen samples for high-throughput clonal analysis. We applied Cryo-mtscATAC-seq to the neurodegenerated human brain, glioblastoma (GBM), pediatric neuroblastoma, and human aorta, and implemented mitobender, a computational tool to reduce ambient mtDNA in single-cell assays. Our approach revealed regional clonal gliogenesis and microglial expansions in amyotrophic lateral sclerosis (ALS), persistence of oligodendrocyte progenitor cell (OPC)-like clones in GBM recurrence, mtDNA depth heterogeneity after neuroblastoma chemotherapy, and oligoclonal proliferation of smooth muscle cells in human aorta. In conclusion, Cryo-mtscATAC-seq broadly extends mtDNA genotyping to archival frozen specimens across tissue types, opening new avenues for investigation of cell state-informed clonality in human health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c60dee046cd06ab111daf9f8ce07ff1e2d339ed7" target='_blank'>
              Cryo-mtscATAC-seq for single-cell mitochondrial DNA genotyping and clonal tracing in archived human tissues
              </a>
            </td>
          <td>
            Maren Salla, B. Obermayer, Marie Cotta, Ekaterina Friebel, Juliana Campo-Garcia, Georgia Charalambous, Roemel Jeusep Bueno, Dustin Lieu, Patryk Dabek, Ashley Helmuth, George Tellides, Roland Assi, Katrin Bankov, Marco Lodrini, H. Deubzer, D. Beule, Hattie Chung, Helena Radbruch, David Capper, Frank L. Heppner, Sarah C. Starossom, Caleb A. Lareau, Ilon Liu, Leif S. Ludwig
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Results Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Conclusions Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00376-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Immune checkpoint inhibitors (ICIs) targeting programmed death-1 (PD-1) have demonstrated significant clinical benefit in colorectal cancer (CRC) with microsatellite instability-high (MSI-H) status, yet a substantial subset of patients remains resistant to therapy. Understanding the mechanisms of resistance and identifying new therapeutic targets are urgently needed. Methods We established a murine MC38-based MSI-H CRC model to investigate the variability in PD-1 treatment response and performed single-cell RNA sequencing (scRNA-seq) on tumors from patients with PD-1-sensitive and PD-1-resistant CRC. Immune cell subsets and signaling pathways were analyzed using CellChat and AUCell, and gene expression enrichment was performed. Functional assays, including RNA-seq, ChIP-seq, qRT-PCR, and Co-IP, were employed to characterize the role of MNDA and its downstream targets. Results scRNA-seq analysis revealed a significant enrichment of polymorphonuclear neutrophil myeloid-derived suppressor cells (PMN-MDSCs) in PD-1-resistant MSI-H CRC, accompanied by increased lactylation activity and high MNDA expression. Further investigations revealed that MNDA recruits the histone acetyltransferase EP300 to the CXCR2 promoter, mediating H3K18 lactylation and enhancing CXCR2 transcription. This promotes PMN-MDSC infiltration and the establishment of an immunosuppressive microenvironment, thereby impairing PD-1 efficacy. Inhibition of the MNDA/EP300-CXCR2 axis reversed H3K18la modification, reduced PMN-MDSC infiltration, remodeled the immune microenvironment, and restored sensitivity to PD-1 therapy. Conclusions Our study reveals a novel mechanism by which MNDA-driven H3K18 lactylation of the CXCR2 promoter facilitates immune evasion and PD-1 resistance in MSI-H CRC. Targeting the MNDA/EP300-CXCR2 regulatory axis represents a promising strategy for overcoming ICI resistance and enhancing the therapeutic benefit in CRC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07097-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f63561fa4907ea76d18e3dde39a94c0132ce001" target='_blank'>
              MNDA promotes immunosuppression in microsatellite instability-high colorectal cancer by facilitating PMN-MDSC infiltration via H3K18 lactylation
              </a>
            </td>
          <td>
            Xuan Zhang, Wen-Bin Wang, Xin-Yi Cai, Xiao-Qiong Chen, Qing Feng, Quan Yang, Ji-Biao Li, Wei Xiong, Tao Wu
          </td>
          <td>2025-10-03</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genetic mechanisms regulating gene expression encompass complex processes such as transcription, translation, epigenetic modifications, and interactions of regulatory elements. These mechanisms play a crucial role in shaping phenotypic diversity in humans. High-throughput technologies, such as expression microarrays and next-generation sequencing (NGS), have enabled precise analysis of transcripts for thousands of genes genome-wide. These methods have enabled researchers to measure gene expression levels in various tissues and cells and to gain deeper insights into previously inaccessible biological processes. Numerous studies show that gene expression varies significantly among individuals. However, there are also notable differences between populations from different continental groups, driven by genetic, epigenetic, environmental factors, and natural selection. Furthermore, disease states represent an important factor influencing gene activity, as they can significantly alter the transcriptomic profiles of individual cells. In this context, comparative population genetic studies help uncover the molecular mechanisms underlying complex phenotypic traits and identify population-specific features of transcriptomic profiles in both health and disease. However, despite significant progress in this field, many aspects remain underexplored. Specifically, the distribution of gene expression variability among populations, the degree of research coverage for specific ethnic groups, the spectrum of biological materials used, and the contribution of population affiliation to observed differences in gene expression during pathological conditions require further investigation. This review presents an overview of contemporary research focused on analyzing variability in expression profiles across different human populations. It summarizes findings from individual studies, outlines the advantages and limitations of the methods employed, highlights key research directions in population transcriptomics, and discusses potential practical applications of the data obtained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e020705578f481ead7a3caa26fc484de5385fb09" target='_blank'>
              Population transcriptomics: a novel tool for studying genetic diversity in human populations under normal and pathological conditions
              </a>
            </td>
          <td>
            A. A. Babovskaya, E. A. Trifonova, V. A. Stepanov
          </td>
          <td>2025-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Psoriasis is a chronic inflammatory skin disease involving dysregulated immune responses and complex genetic factors. This study combines single-cell RNA sequencing (scRNA-seq), gene expression profiling, and genetic analysis to explore cellular and molecular contributors to psoriasis. Methods Single-cell RNA-seq data (n = 3 psoriasis, n = 2 control; GSE228421) were used for cell-type annotation and functional characterization. T cell subsets were analyzed for differentiation trajectories and cell-cell communication. Differentially expressed genes in mucosal-associated invariant T (MAIT) cells were evaluated by enrichment analysis. Candidate gene causality was tested via eQTL-based Mendelian randomization (MR) and supported by bulk RNA-seq validation. Results MAIT cells were enriched in psoriatic lesions and exhibited strong intercellular interactions. Functional analyses revealed activation of IL6-JAK-STAT3 signaling, TNF-NFκB pathway, and glycolysis in MAIT cells. MR identified RPS20 as a protective factor (OR = 0.5994, p = 0.011) and PFN1 as a potential risk gene (OR = 1.7229, p = 0.037), with PFN1 highly expressed in MAIT cells. Colocalization analysis showed no significant genetic overlap between PFN1 expression and psoriasis risk. Metabolic profiling revealed differential pathway involvement in PFN1+ and PFN1− MAIT cells. Conclusion Our integrative analysis highlights MAIT cells and PFN1 as likely contributors to psoriasis pathogenesis. These findings offer insights into immune and metabolic alterations, suggesting potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48055881b868991adfc41d78b968062c6c8aae6e" target='_blank'>
              MAIT Cells with High PFN1 Expression Mediate Immune Activation and Metabolic Reprogramming in Psoriasis
              </a>
            </td>
          <td>
            Man-Ning Wu, Yue-Min Zou, Xiang-Nan Zhou, Sangwon Hong, Lei Wang, Yan-Ping Bai
          </td>
          <td>2025-09-09</td>
          <td>Clinical, Cosmetic and Investigational Dermatology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Spontaneous mutations are the primary source of genetic variation and play a central role in shaping evolutionary processes. To investigate mutational dynamics in Daphnia obtusa, we generated a chromosome-level genome assembly spanning 129.4 Mb across 12 chromosomes, encompassing 15,321 predicted protein-coding genes. Leveraging whole-genome sequencing of eight mutation accumulation (MA) lines propagated for an average of 482 generations (spanning over 20 years), we estimated a spontaneous single nucleotide mutation (SNM) rate of 2.23 × 10−9 and an indel mutation rate of 2.75 × 10−10 per site per generation. The SNM spectrum was strongly biased toward C:G > T:A transitions. Comparative analyses with natural population data revealed that exonic mutations observed in the MA lines were significantly less likely to be present in standing variation than intronic or intergenic mutations, suggesting that purifying selection in natural populations acts to remove deleterious alleles. We also identified 48 de novo loss-of-heterozygosity (LOH) events, comprising 8 heterozygous deletions and 40 gene conversion events. The genome-wide gene conversion rate was estimated at 2.62 × 10−5 per heterozygous site per generation. These findings provide a comprehensive view of the mutation spectrum, selective pressures, and mechanisms underlying genome stability in D. obtusa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62913f009180f6bab78e3a870dd5c23fc48905e4" target='_blank'>
              The mutation landscape of Daphnia obtusa reveals evolutionary forces shaping genome stability
              </a>
            </td>
          <td>
            Ruyun Deng, Feng Guo, Wen Wei, M. Pfrender, Jeff L. Dudycha, Michael Lynch, Zhiqiang Ye
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer research has traditionally focused on identifying driver genes, those with mutations that initiate tumorigenesis. The Cancer Driver Gene (CDG) paradigm, further supported by the observation of oncogene addiction in tumors, has successfully guided the development of targeted therapies. However, the limitations of this driver-centric view, highlighted by the broad emergence of frequent therapeutic resistance, the presence of driver mutations in healthy tissues or individuals, and the lack of identifiable drivers in many tumors, call for a shift in perspective and clinical practice. The latest network controllability perspective on cancer cells introduced the concept of Cancer Keeper Genes (CKGs) and a CKG-based paradigm for cancer therapeutics. The new concept encompasses the concept of non-oncogene addiction, emphasizing reliance on non-mutated pathways crucial for maintaining oncogenic cellular states. Here, we explore the transition towards a system-level understanding of cancer based on the CKG paradigm, emphasizing the essential role of genes required for tumor maintenance, irrespective of their initiating function or mutational capacity. We discuss clinical implications of this paradigm shift, highlighting the progress made so far and potential of targeting non-driver CKGs, genes involved in processes like DNA damage response, proteostasis, and metabolism, as a promising strategy to overcome therapeutic challenges and achieve more durable cancer control. Targeting these maintenance vulnerabilities represents a critical evolution in precision oncology, moving towards therapies designed to dismantle the networks sustaining malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e80123ffc1524bf35e125c1b2645e60b9bc7703" target='_blank'>
              From Cancer Drivers to Cancer Keepers: Paradigm Shift and Clinical Implications
              </a>
            </td>
          <td>
            Xizhe Zhang, Weixiong Zhang
          </td>
          <td>2025-08-22</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Glioblastoma relapse remains inevitable, placing these tumors among the most difficult to treat. Treatment challenges include inherent drug resistance, tumour’s plastic nature, and suppressive tumour microenvironment (TME). In other cancers, treatment-induced plasticity has been associated with the emergence of drug-tolerant persister cells, which survive therapy by undergoing reversible phenotypic changes towards resistant states. The role of persisters in glioblastoma remains to be unravelled. It is unclear to what extent the responses are patient- and treatment-specific, and how much they depend on the tumor microenvironment.



 We applied a cohort of molecularly characterized patient-derived models to investigate patient-specific responses to chemotherapeutics (Temozolomide, VAL-083). Treatment efficacy and its correlation with molecular features were assessed ex vivo using organoids and stem-like cultures as well as in vivo through orthotopic xenografts. To mimic clinical treatment schedules, we assessed functional responses in time, incorporating prolonged exposure to treatment and drug holiday period over several weeks. Analysis included tumor cell viability, cell cycle, DNA damage and repair, senescence and quiescence features. Phenotypic adaptation of glioblastoma and microenvironmental cells in vivo were analysed with MRI, single-cell RNA-seq, spatial transcriptomics and IHC.



 Assessment of drug efficacy across a cohort of patient-derived models revealed resistant versus adaptive responder groups, which were treatment-specific. While resistant cells did not show functional changes in cell viability and cell cycle upon prolonged drug exposure, sensitive cells show a biphasic growth curve characteristic of persister cells. The adaptation towards the least proliferative phenotype was reversible during drug holiday period, further confirming a role of a transient drug-tolerant state. Chemotherapeutic treatment in orthotopic xenografts partially reduced tumor growth in vivo and activated apoptosis and DNA repair pathways in glioblastoma cells. These changes coincided with an increased infiltration of tumor-associated macrophages.



 Our findings highlight the critical roles of both tolerance and intrinsic resistance mechanisms. The responder groups appear to be specific to the type of treatment and the unique characteristics of each patient’s tumor, emphasizing the need to evaluate treatment-induced plasticity in a patient-specific context. Glioblastoma cells are further supported by the adaptation of the microenvironment towards a tumor-supportive crosstalk.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17ea2a43463b7a7c67d446cbd650a87fb74e2953" target='_blank'>
              P01.18.B MODELING GLIOBLASTOMA TREATMENT RESPONSES: INSIGHTS INTO TUMOR PLASTICITY AND THE ROLE OF THE TUMOR MICROENVIRONMENT
              </a>
            </td>
          <td>
            M. de Lucas Sanz, H. Allahverdi, B. Nosirov, P. Moreno Sánchez, M. Baroni Milan, A. Lipsa, M. Rezaeipour, C. Dording, D. Brown, J. M. Langlands, K. Grzyb, Y. Yabo, J. Zhang, D. Heiland, A. Michelucci, P. Nazarov, A. Gołȩbiewska
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that their continued expression is required for maintaining their leukemic phenotype and that fusion genes therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5469845f04d542fe9db7d0ec4a2180ccb6b7c697" target='_blank'>
              Fusion Gene Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. D. Mata Casimiro, Anita van Oort, M. Du Plessis, Luca van den Brink, Minoo Ashtiani, C. Zwaan, A. Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, Josef Vormoor, S. Kellaway, Olaf Heidenreich
          </td>
          <td>2025-09-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Multiple myeloma (MM) is highly heterogeneous, with relapse occurring in the majority of cases, and recent advancements in single-cell RNA sequencing (scRNA-seq), sc-metabolism profiling, and bulk RNA-seq have facilitated the identification of cell subpopulations and metabolic reprogramming at the single-cell level, uncovering novel molecular mechanisms. This study aims to establish a multi-omics atlas of MM, characterizing the cell subpopulations and signaling pathways that drive immune evasion and disease progression. Additionally, sc-metabolic profiling identifies reprogramming patterns and informs therapeutic screening. We integrated scRNA-seq and bulk RNA-seq data using R to analyze immune and non-immune cell features and pathways in MM. Metabolic reprogramming was assessed via sc-metabolic profiling, and drug candidates were screened through multi-omics integration, with efficacy evaluated in vitro using CCK-8 assays, flow cytometry, Western blotting, and CalcuSyn software. Novel MM subpopulations were identified, including myeloma-activated hematopoietic stem cells and ISG15+ B cells, which correlated with survival and were validated by multiplex immunofluorescence. IFN-γ is primarily secreted by effector memory CD8+T cells, and IFN-α is primarily secreted by non-classical monocytes, driving an IFN-γ/α-B2M feedback loop. Multi-omics identified four drug candidates, each demonstrating anti-tumor effects against myeloma cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed7155f8bb3f2fadcad9b5654646e878c15255d" target='_blank'>
              Multi-omics reveals immune features in immune and non-immune cells, an IFN-γ/IFN-α-B2M positive feedback loop, and targeted metabolic therapy in multiple myeloma
              </a>
            </td>
          <td>
            Chen Li, Yaping Liao, Ling Xu, Yan Chen
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e72f57d35ee3dac82999a8b9759c889a0508af" target='_blank'>
              Genetic and transcriptomic alterations underlying aggressiveness in 5-fluorouracil-resistant HCT116 cells.
              </a>
            </td>
          <td>
            Pornchai Sooksaen, A. Thim-uam, Ratsada Praphasawat, Chutamas Thepmalee, Kanokkarn Phromnoi, Damratsamon Surangkul, Surisak Prasarnpun, Chakkraphong Khonthun
          </td>
          <td>2025-10-10</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Diffuse large B-cell lymphoma (DLBCL) is a highly heterogeneous and aggressive hematologic malignancy, with the activated B-cell-like (ABC) subtype displaying particularly poor prognosis due to inherent treatment resistance and elevated recurrence rates. Despite advances in targeted therapies and immunotherapies, a significant proportion of patients experience relapse or refractory disease, highlighting the urgent need for novel biomarkers and innovative therapeutic strategies to improve clinical outcomes. Methods A multi-dimensional analysis of SH3BP5 expression was performed across DLBCL subtypes, integrating transcriptomic, proteomic, and clinical datasets to assess its correlation with immune infiltration, tumor metabolism, and patient prognosis. Single-cell RNA sequencing data were employed to examine the tumor microenvironment (TME) with higher resolution. Further analysis of the association between SH3BP5 and immune checkpoint gene expression was conducted to explore its potential role in immunotherapy response. Functional in vitro assays were carried out to assess the impact of SH3BP5 knockdown on DLBCL cell proliferation and apoptosis. Results The analysis revealed that SH3BP5 is preferentially overexpressed in the ABC subtype of DLBCL across multiple datasets and validated cohorts, and its high expression is significantly associated with poor overall survival. Single-cell transcriptomic profiling demonstrated that SH3BP5 is mainly expressed in malignant B cells and inversely correlated with immune cell infiltration, particularly CD8 + T cells. Mechanistically, pathway enrichment and metabolic assays indicated that SH3BP5 is linked to mitochondrial metabolic reprogramming, promoting oxidative phosphorylation (OXPHOS) and potentially contributing to reduced responsiveness to immune checkpoint inhibitors (ICIs). Functional studies showed that SH3BP5 knockdown significantly suppressed DLBCL cell proliferation, induced apoptosis, and reduced tumor cell viability in vitro. Conclusion This study suggests that SH3BP5 may serve as a prognostic biomarker and a potential therapeutic target in DLBCL, particularly within the ABC subtype. By delineating its associations with immune evasion and metabolic reprogramming, these findings provide a mechanistic basis for further exploration of SH3BP5-targeted interventions to help overcome therapy resistance. Future studies in larger clinical cohorts and functional models are warranted to validate these results and assess the potential of integrating SH3BP5 expression profiling into precision medicine strategies for DLBCL. Trial registration The study was registered in the Chinese Clinical Trial Registry (ChiCTR2200060430; http://www.chictr.org.cn/) on June 1, 2022. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06951-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27222190fd1301e27cfd9869b08b07fc58aa185" target='_blank'>
              SH3BP5-driven metabolic-immune crosstalk in DLBCL: a prognostic biomarker and therapeutic target for reshaping immunosuppressive microenvironment
              </a>
            </td>
          <td>
            Tong Wu, Yi Yang, Yuan Zong, Jiawen Zhao, Xiaoyu Zhao, Lei Li, Yiming Gao, Ning Li, Liting Jiang, Yinyin Xie
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is a highly heterogeneous tumor that lacks reliable biological markers for diagnosis and prognostic monitoring. Currently, the differentially expressed genes between paired adjacent normal tissues and ccRCC tumor tissues at single-cell resolution remained to be further discovered. To address this challenge, we performed an integrative analysis of multiple single-cell databases containing paired ccRCC samples. Using the “CopyKAT” algorithm, we accurately identified ccRCC tumor cells. Subsequently, various pseudotime algorithms were employed to identify malignant cells with tumor stem cell-like properties and high plasticity. This cell subgroup exhibited high expression of malignant features, including hypoxia, epithelial-mesenchymal transition (EMT), and proliferation/invasion phenotypes. We then performed differential analysis to identify genes highly expressed in this subgroup and constructed a reliable clinical diagnostic model for ccRCC using multiple machine learning algorithms. Furthermore, we identified AXL as a key gene with significant oncogenic activity, where high expression of AXL correlated with poor patient prognosis. Immune infiltration and spatial transcriptomics analyses further revealed that AXL promotes tumor progression interaction with M2 macrophages. Taken together, our analysis establishes a reliable 13-gene panel diagnostic model and AXL gene as reliable biological markers for ccRCC, providing valuable targets and a theoretical foundation for the development of precision-targeted therapies for ccRCC. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00563-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c5c1d87281d2808c2e2e599af16c985ed97a58" target='_blank'>
              Integrated single-cell and bulk transcriptome analysis revealed high plasticity subpopulation and promising diagnosis model for clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Zhongwen Lu, Fanyi Kong, Jiahuan Sun, Jing Ge, Jiajin Wu, Kunpeng Wang
          </td>
          <td>2025-09-30</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Cutaneous squamous cell carcinoma (CSCC) is a prevalent skin cancer with aggressive progression that poses significant challenges, especially in metastatic cases. Single-cell DNA sequencing (scDNA-seq) has become an advanced technology for elucidating tumor heterogeneity and clonal evolution. However, comprehensive scDNA-seq studies and tailored mutation panels for CSCC are lacking. Methods We analyzed the genomic landscape of Chinese CSCC patients via a Multi-Patient-Targeted (MPT) scDNA-seq approach. This method combined bulk exome sequencing with Tapestri scDNA-seq. Mutations identified through bulk sequencing were used to design a targeted panel for scDNA-seq. Comparative analysis was conducted to explore the associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis was performed to understand the evolutionary trajectories of the tumors. Results Bulk sequencing revealed a diverse spectrum of somatic mutations in CSCC tumors, with missense mutations being predominant. The top tumor mutations, such as those in NOTCH1, TP53, NOTCH2, TTN, MUC16, RYR2, PRUNE2, DMD, HRAS, and CDKN2A, presented similar frequencies to those reported in studies in Korean and Caucasian populations. However, the mutation frequencies of HRAS, TTN, MUC16 and MUC4 were significantly different from the Korean and Caucasian populations. Comparative analysis revealed associations between specific gene mutations and clinical characteristics such as tumor stage and patient sex. Clonal evolution analysis via scDNA-seq revealed distinct evolutionary trajectories and their potential correlation with tumor development and patient prognosis. Furthermore, scDNA-seq identified two low-frequency mutation clones, NLRP5 and HMMR, which play important roles in the clonal evolution of CSCC. Conclusions Our study introduced a novel MPT sequencing approach for CSCC, providing insights into tumor heterogeneity and clonal evolution. We identified novel mutations and their potential associations with patient survival and tumor characteristics. Overall, our study layed the groundwork for personalized treatment strategies and provides a reference for future applications of the MPT panel in CSCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14585-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667c56c3222da6ac371627cb03ea1bf3e1f96087" target='_blank'>
              Analysis of genomic heterogeneity and the mutational landscape in cutaneous squamous cell carcinoma through multi-patient-targeted single-cell DNA sequencing
              </a>
            </td>
          <td>
            Wen Chen, Jiawei Xu, Chengdong Yu, Meng Zhou, Yong Ai, Wenbing Rao, Qingchuan Wang, Siyi Xu, Lei Tang, Si Gong, Jun Rao
          </td>
          <td>2025-08-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 In contrast to prevailing wisdom that cancer is a genetic disease that evolves via Darwinian selection, it is increasingly recognized that in addition to genetic mutations, intratumoral heterogeneity also results through non-genetic mechanisms. Furthermore, it is now evident that the malignant phenotype results from complex interactions between genetic and non-genetic mechanisms. These interactions contribute to functional changes across multiple spatiotemporal scales, from molecular to cellular to tissue level, creating a heterogeneous cancer cell population. Cancer Systems Biology is a comprehensive, up-to-date treatise that addresses these issues. The main intent of this compendium is to encourage cross-pollination of ideas from experts in multiple disciplines including basic and translational science researchers, systems/mathematical biologists, and biophysicists and clinicians interested in decoding a systems-level emergent view of cancer so that a deeper understanding of disease progression can be gained and more effective treatment strategies can be designed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03788cb0b91aa174c4a7535dcca74621b417df9" target='_blank'>
              Cancer Systems Biology
              </a>
            </td>
          <td>
            Ravi Salgia, M. Jolly, Prakash Kulkarni, Govindan Rangarajanm
          </td>
          <td>2025-09-01</td>
          <td>None</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Background: Based on our previous study, we have identified ZNF180, a zinc finger protein, as a pro-tumorigenic regulator in primary melanoma and a marker for poor prognosis. Herein, we report that ZNF180-regulated pathway, hence ZNF180-regulome, underlies resistance towards immune checkpoint inhibitions (ICIs). Methods: To investigate regulatory roles of ZNF180 to confer these immune suppressive phenotypes, we performed ZNF180 knock-down in melanoma cells in vitro with different genetic backgrounds, namely A375 (BRAF-mutant) and SKMEL147 (NRAS-mutant) cells, and performed RNA- and ATAC-sequencing. We performed integrative analysis of RNA- and ATAC-sequencing data with publicly available sequencing data from ICI-treated cohorts to construct comprehensive model of ZNF180-regulome and its impacts on immune microenvironment. Further, we performed ZNF180 silencing in immune competent Yumm1.7 murine model to confirm the changes in immune microenvironments. Results: ZNF180-regulome was predictive of ICI responses in independent bulk sequencing cohorts, and ZNF180+ tumors persisted after the therapy with immune-suppressive features such as MHC-I loss and CD155 expressions, the primary ligand to TIGIT inhibitory receptor. Further, ZNF180 silencing revealed its regulations on AP-1 transcription factors to drive melanoma reprogramming towards de-differentiated MITFlowAXLhigh cells, an established melanoma subtypes associated with recurrence and ICI resistance. In tandem, we observed that ZNF180+ tumor neighborhood significantly excluded with CD4 T-cells in metastatic tumor, and its silencing in immune competent murine model increased CD4 helper T-cell infiltrations with significant tumor regression in vivo. Conclusion: Collectively, these results indicate ZNF180 is a tumor intrinsic regulator of melanoma plasticity to drive de-differentiated phenotypes with immune-suppressive features including loss of immunogenicity, T-cell inhibitory signals through TIGIT/CD155 checkpoint and exclusion of CD4 helper T-cells. As ZNF180-regulome manifests in non-metastatic melanoma in contrast to the current focus of standard-of-care ICI on the metastatic disease, these results establish ZNF180-regulome as a biomarker and novel therapeutic avenue for early-stage, non-metastatic melanoma to intervene ICI resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31baf3f08584db6bc1b15c0af412ba0fb9282ea1" target='_blank'>
              ZNF180 modulates tumor intrinsic immunotherapy resistance in melanoma through driving plasticity
              </a>
            </td>
          <td>
            Won-Min Song, Sree Karani Kondapuram, Xianxiao Zhou, Shu-hsia Chen, Praveen Agrawal
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3017cd20ec1bd27c4516abd6034d6436513f0d42" target='_blank'>
              Balanced chromosomal insertions as the mechanism of recurrent familial microstructural abnormalities: detailed analyses using long-read whole-genome sequencing.
              </a>
            </td>
          <td>
            Hironao Shirai, Keiko Shimojima Yamamoto, Hirokazu Arai, Yukio Sawaishi, Saori Fujita, Yoko Kuriyama, Masaki Miura, Jun Tohyama, Toshiyuki Yamamoto
          </td>
          <td>2025-09-24</td>
          <td>Journal of human genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Understanding the functional impact of bacterial genetic diversity is crucial for linking pathogen variants to clinical outcomes. Here, we introduce a high-throughput cytological profiling pipeline optimized for Mycobacterium tuberculosis (Mtb) clinical strains, integrating OD-calibrated feature analysis and high-content microscopy. Our system quantifies single-bacterium morphological and physiological traits related to DNA replication, redox state, carbon metabolism, and cell envelope dynamics. Applied to 64 Mtb clinical isolates from lineages 1, 2, and 4, the approach revealed that cytological phenotypes recapitulate genetic relationships and exhibit both lineage- and density-dependent dynamics. Notably, we identified a link between a convergent "small cell" phenotype and a convergent ino1 mutation that is associated with the presence of an antisense transcript, suggesting a potential non-canonical regulatory mechanism under selection. In summary, we present a resource-efficient approach for mapping Mtb's phenotypic landscape, uncovering cellular traits that underlie its evolution and providing new insights into the functional consequences of bacterial genetic diversity.


IMPORTANCE
Understanding how genetic variation in Mycobacterium tuberculosis (Mtb) shapes its physical traits is essential to unraveling the evolution of this global pathogen. Here, we introduce a systematically optimized, high-throughput imaging platform for the comprehensive characterization of Mtb clinical strains. We demonstrate that Mtb's phenotypic manifestation is shaped by both genetic background and culture density. By accounting for these factors, our analysis linked distinct cellular dynamics to specific lineages, sublineages, and even single nucleotide variations. Notably, we linked a recurring mutation to a unique cell-shortening phenotype, finding that it potentially acts by creating a cryptic antisense transcript. This platform provides a powerful framework for systematically dissecting the physiological dynamics underlying Mtb evolution and identifying new therapeutic vulnerabilities of this deadly pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56fa61f881d5174f374abc12d00931b99537639c" target='_blank'>
              High-throughput cytological profiling uncovers genotype-phenotype associations in Mycobacterium tuberculosis clinical isolates.
              </a>
            </td>
          <td>
            Qingyun Liu, Yue J Liu, Ruiyuan Liu, Peter H. Culviner, Xin Wang, I. D. Wolf, Ken Chen, Yiwang Chen, Yi Xiao, Guiming Zhang, Rongfeng Sun, Shoko Wakabayashi, Nicole C Howard, Mingyu Gan, Eric J. Rubin, Sarah M. Fortune, Junhao Zhu
          </td>
          <td>2025-10-09</td>
          <td>mSystems</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Oncogene targeted cancer therapies can provide deep responses but frequently suffer from acquired resistance.1 Therapeutic approaches to treat tumours which have acquired drug resistance are complicated by continual tumour evolution and multiple co-occurring resistance mechanisms.2,3 Rather than treating resistance after it emerges, it may possible to prevent it by inhibiting the adaptive processes which initiate resistance but these are poorly understood.4 Here we report that residual cancer persister cells that survive oncogene targeted therapy are growth arrested by drug stress-induced intrinsic Type I interferon (IFN) signaling. To escape growth arrest, persister cells leverage apoptotic machinery to transcriptionally suppress interferon-stimulated genes (ISGs). Mechanistically, persister cells sublethally engage apoptotic caspases to activate DNA endonuclease DNA Fragmentation Factor B (DFFB, also known as Caspase-Activated DNase (CAD)) which induces DNA damage, mutagenesis, and stress response factor Activating Transcription Factor 3 (ATF3). ATF3 limits Activator Protein-1 (AP1)-mediated ISG expression sufficiently to allow persister cell regrowth. Persister cells deficient in DFFB or ATF3 exhibit high ISG expression and are consequently unable to regrow. Therefore, sublethal apoptotic stress paradoxically promotes regrowth of residual cancer cells that survive drug treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21f810637300567b18dd70e40b7019befcd5ee94" target='_blank'>
              DFFB suppresses interferon to enable cancer persister cell regrowth
              </a>
            </td>
          <td>
            August F Williams, David A. G. Gervasio, C. Turkal, Anna E. Stuhlfire, Michael X. Wang, B.E. Mauch, Rhea Plawat, Ariel H Nguyen, Michelle H. Paw, Mehrshad Hairani, Cooper P. Lathrop, Sophie H. Harris, Jennifer L. Page, Matthew J Hangauer
          </td>
          <td>2025-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) was a major cause of cancer-related mortality globally. Despite advancements in immunotherapy and targeted therapies, clinical outcomes were still limited by tumor heterogeneity and treatment resistance. The transcription factor (TF) FOS, a key component of the AP-1 complex, was linked to tumor progression and therapy resistance in various cancers, but its precise mechanisms remained unclear, and its role in lung adenocarcinoma (LUAD) was unknown. We investigated the tumor microenvironment (TME) of LUAD using single-cell RNA sequencing (scRNA-seq) to identify potential therapeutic vulnerabilities and FOS-driven mechanisms. Methods We identified fourteen cell types by analyzing scRNA-seq data from LUAD samples (GSE164789) using Seurat (v4.4.0) and Harmony for batch correction. InferCNV was utilized to characterize the tumor cell subtypes after they were clustered using marker genes. CytoTRACE and Monocle were used to create pseudotime trajectories in order to map differentiation states. CellChat revealed intercellular communication networks, while SCENIC identified TF regulatory modules. The CCK-8, Edu, Transwell, and wound healing assays showed that FOS knockdown functionally validated A549 and NCI-H1975 cells. Furthermore, a prognostic model was developed. Results We discovered that invasive LUAD was dominated by a highly stem-like C0 MAFF+ tumor cell subtype that produced chemokines and activated lipid metabolism. These cells stimulated immunosuppression and tumor-associated macrophage (TAM) differentiation by interacting with macrophages via MIF-(CD74+CD44) signaling. Experiments using FOS knockdown demonstrated its role in maintaining invasion, migration, and proliferation. Using the MTRS model, patients were categorized into high- and low-risk cohorts, high-risk patients exhibited unique drug sensitivities. Immunoprofile analysis revealed higher M1 macrophages in high-risk patients, suggesting that FOS inhibition could repolarize TAMs and enhance immunotherapies. Conclusion Our studies show that FOS is a main regulator of C0 MAFF+ TCs in LUAD, polarizing macrophages via MIF and rewiring lipid metabolism to support cancer. The MTRS model offers clinical value for risk assessment even if FOS inhibition shows promise as a therapeutic approach to raise immunotherapy efficacy. Targeting the FOS could cause TME immunosuppression to be disrupted, thus LUAD presents a fresh precision oncology approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2ef348cf0a48b9b9c178cc598309bdb56456d8" target='_blank'>
              Targeted and personalized immunotherapy in lung adenocarcinoma: single-cell RNA sequencing of MAFF+ tumor cells and the therapeutic potential of FOS
              </a>
            </td>
          <td>
            Xiangsong Cheng, Shu Chen, Yilong Fu, Runze Jiang, Yanlong Jing, Bizhu Zhao, Dong Guo, Liangyu Wang, Zi Ye, Yumeng Li, Xianliang Chen
          </td>
          <td>2025-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor gene alterations can serve as predictive biomarkers for therapy response. The nucleotide excision repair (NER) helicase ERCC2 carries heterozygous missense mutations in approximately 10% of bladder tumors, and these may predict sensitivity to cisplatin treatment. To explore the clinical actionability of ERCC2 mutations, we assembled a multinational cohort of 2,012 individuals with bladder cancer and applied the highly quantitative CRISPR-Select assay to functionally profile recurrent ERCC2 mutations. We also developed a single-allele editing version of CRISPR-Select to assess heterozygous missense variants in their native context. From the cohort, 506 ERCC2 mutations were identified, with 93% being heterozygous missense variants. CRISPR-Select pinpointed deleterious, cisplatin-sensitizing mutations, particularly within the conserved helicase domains. Importantly, single-allele editing revealed that heterozygous helicase-domain mutations markedly increased cisplatin sensitivity. Integration with clinical data confirmed that these mutations were associated with improved response to platinum-based neoadjuvant chemotherapy. Comparison with computational algorithms showed substantial discrepancies, highlighting the importance of precision functional assays for interpreting mutation effects in clinically relevant contexts. Our results demonstrate that CRISPR-Select provides a robust platform to advance biomarker-driven therapy in bladder cancer and supports its potential integration into precision oncology workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298c9b125eade81e30854dcc4c068fdf43533872" target='_blank'>
              Quantitative functional profiling of ERCC2 mutations deciphers cisplatin sensitivity in bladder cancer
              </a>
            </td>
          <td>
            J. Börcsök, Diyavarshini Gopaul, Daphne Devesa-Serrano, Clémence Mooser, Nicolas Jonsson, M. Cagiada, D. Stormoen, M. N. Ataya, B. Guercio, H. Kaimakliotis, G. Iyer, Kresten Lindorff-Larsen, L. Dyrskjøt, Kent W. Mouw, Z. Szallasi, C. S. Sørensen
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Dysregulation of early B cell lymphopoiesis—the process guiding cellular immunity development—can lead to malignancy, making it crucial to understand its regulatory mechanisms. We generated a multiomics resource comprising paired chromatin accessibility and gene expression profiles across eight human B cell precursor populations, providing a detailed characterization of early human B cell development. Integrative analysis revealed highly cell type–specific regulatory elements and enabled the reconstruction of the gene regulatory network governing differentiation. We identified putative candidate regulons, such as ELK3, enriched in pro–B cells and potentially involved in cell cycle progression. Regulons from bulk data were projected onto single-cell data, validating their activity and refining the regulatory landscape. This resource enabled identification of active regulatory programs and transformation-associated states in B cell acute lymphoblastic leukemia. The publicly available atlas provides a valuable resource for understanding B cell development and disease, supporting future efforts to decode regulatory programs in immunity and hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72948d6bed4b92194fb9e2b0dbf1593531ef8b66" target='_blank'>
              Uncovering the regulatory landscape of early human B cell lymphopoiesis and its implications in the pathogenesis of B-ALL
              </a>
            </td>
          <td>
            Núria Planell, Xabier Martínez-de-Morentin, Daniel Mouzo, D. Lara-Astiaso, A. Vilas-Zornoza, P. S. Martín-Uriz, D. Alignani, B. Paiva, A. Maillo, Aleksandra Kurowska, Nerea Berastegui, P. García-Olloqui, Arantxa Urdangarin, Peri Noori, A. Ortega-Legarreta, Mikel Hernaez, V. Lagani, N. Kiani, Matthias Merkenschlager, T. Ezponda, J. Martín-Subero, Ricardo N. Ramirez, Jesper N. Tegnér, F. Prósper, D. Gómez-Cabrero
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 RAS inhibitors (RASi) have the potential to change the clinical landscape for pancreatic ductal adenocarcinoma (PDAC) and improve patient survival. Nevertheless, clinical resistance is likely to reduce the long-term benefits of RASi, and combinations that can bypass or overcome resistance will be needed. Here, we studied the dynamics of resistance to the RAS multi(ON) inhibitor RMC-6236 (daraxonrasib). Prior to frank drug resistance, PDAC cells persist in a drug tolerant state characterized by reduced proliferation; however, only a fraction of the drug-tolerant persister cells contributes to resistance. To identify and study the relevant, resistance-contributing cells, we developed a highly multiplexed, lentiviral DNA barcode system that is compatible with single-cell technologies, enabling paired lineage tracing and phenotypic profiling throughout RASi treatment. Our lineage tracing studies suggest that drug persistence and resistance are fueled by a subset of cells that are primed to survive RASi treatment. Ongoing work is directed at defining and tracking the cell states associated with each stage of resistance emergence.



 Jayne C. McDevitt, Connor J. Hennessey, Robert Vander Velde, Sydney M. Shaffer, Ben Z. Stanger. Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B082.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95c1bd513ed6b1024a532086d606e43f01e09bb7" target='_blank'>
              Abstract B082: Clonal dynamics and molecular mechanisms mediating resistance to RAS inhibition
              </a>
            </td>
          <td>
            Jayne C McDevitt, Connor J Hennessey, R. V. Velde, Sydney M. Shaffer, Ben Z. Stanger
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Tumor heterogeneity impacts invasive behaviors, treatment response, and clinical outcomes in triple-negative breast cancer (TNBC). However, this heterogeneity remains incompletely characterized. This study aims to utilize multi-scale data to investigate inter-tumoral heterogeneity and identify potential TNBC biomarkers. Methods Single-cell RNA expression profiles were analyzed using R packages. Specifically, the infercnv, Pyscenic, GeneNMF, SCP, Vector, CellChat, and hdWGCNA packages were employed to identify malignant cells and characterize heterogeneity in transcription factors, metaprograms, lineage evolution, developmental trajectories, cell–cell interactions, and co-expression networks. Bulk RNA datasets were incorporated to assess the prognostic value of cell clusters and candidate genes. G Protein Subunit Alpha 15 (GNA15) expression was determined via reverse transcription–quantitative PCR (RT–qPCR) and immunohistochemistry. Cell functional assays were performed to evaluate proliferation, migration, and invasion capabilities. Results A total of 14,335 malignant cells were isolated from epithelial cells across 15 single-cell RNA samples. Six tumor cell clusters were identified, which exhibited distinct prognoses, biological functions, driver transcription factors, and co-expression networks. Notably, the S2 cluster demonstrated association with multiple malignancy-related pathways and inferior survival outcomes. GNA15 emerged as the S2 cluster hub gene. In vitro experiments confirmed that GNA15 knockdown significantly attenuated proliferation, migration, and invasion in TNBC cell lines. Conclusions Our study comprehensively delineated TNBC tumor cell heterogeneity and established the critical role of GNA15 in TNBC progression. These findings enhance the understanding of TNBC heterogeneity and provide a theoretical foundation for TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e149422e4d37b7c64ca2e5cbee9d7c8957b702e6" target='_blank'>
              Multiomics profiling identifies the poor prognostic role of a tumor cluster with GNA15 overexpression in triple-negative breast cancer
              </a>
            </td>
          <td>
            Guixin Wang, Jun Cao, Chenglu Lu, Yu-Peng Cao, Shuo Wang, Ziyi Chen, Zhaohui Chen, Yingxi Li, Yue Yu, Yao Tian, Xin Wang
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Breast cancer (BC) remains a significant global health challenge, partly due to its ability to develop resistance to treatment. BC resistance arises from various biological mechanisms, including cancer cells actively expelling drugs, altering their internal metabolic and signaling pathways, evading the immune system, or persisting as drug-tolerant cancer stem cells. These changes occur in the context of a tumor microenvironment that further shields the cancer, limiting treatment access. As the disease advances, particularly when it spreads to other body parts, effective treatment becomes increasingly difficult. Recent advancements in artificial intelligence (AI) offer promising avenues for improving early detection, refining diagnoses, enhancing clinical decision-making and thereby personalizing treatment strategies. This review examines the biological mechanisms that cause BC resistance and its treatment. It further explores AI’s potential to address critical challenges, transforming BC therapy to provide improved survival rates and enhanced quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfb5357bf49f1fab994fbcce36bcc1738760cec3" target='_blank'>
              The Underlying Mechanisms and Emerging Strategies to Overcome Resistance in Breast Cancer
              </a>
            </td>
          <td>
            Krishnaswamy Kannan, Alagarsamy Srinivasan, Aarthi Kannan, Nawab Ali
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a complex disease influenced not only by genes but also by the way cancer cells generate and use energy. One rare change is the loss of a gene called fumarate hydratase (FH), which normally helps cells produce energy efficiently. We examine how this change influences how tumors grow and respond to treatment. By studying thousands of breast cancer samples, we found that tumors missing FH tend to create a special environment that helps them grow and survive, especially by encouraging new blood vessel formation. We also describe a patient with this FH change who had a remarkable and long-lasting response to a therapy that blocks blood vessel growth (known as anti-VEGF treatment). These findings suggest that identifying this rare energy-related alteration may help clinicians determine which patients are most likely to benefit from this therapy. Such insights could contribute to advancing more personalized approaches to breast cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c214b499120e8d0fc3e3396e45d4d3eb6e6c018" target='_blank'>
              Revealing the Angiogenic Signature of FH-Deficient Breast Cancer: Genomic Profiling and Clinical Implications
              </a>
            </td>
          <td>
            L. A. Sinberger, Noa Keren-Khadmy, Assaf Goldberg, T. Peretz-Yablonski, A. Sonnenblick, Mali Salmon-Divon
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Diffuse midline gliomas (DMGs) are universally fatal pediatric brain tumors with a median survival of just 11 months. Despite radiation therapy offering temporary benefit, there are no effective long-term treatments, and targeted therapies have shown minimal impact. The infiltrative nature of DMGs, combined with their critical brainstem location, underscores the urgent need to identify novel mechanisms driving tumor progression and resistance. Cellular senescence is a state of permanent cell-cycle arrest induced by stress and has traditionally been viewed as tumor-suppressive. However, recent evidence in adult cancers implicates senescent cells as active contributors to tumor growth, treatment resistance, and relapse through the secretion of pro-tumorigenic factors known as the senescence-associated secretory phenotype (SASP). While senescence has emerged as a hallmark of cancer in adults, its role in pediatric tumors such as DMGs remains largely unexplored. Using single-cell RNA sequencing of treatment-naïve DMG specimens, we identified rare populations of senescent glioma cells and microglia that would be undetectable using conventional approaches. Computational analyses revealed predicted paracrine interactions between senescent and non-senescent cells including pathways previously linked to tumor invasiveness and resistance. This study aims to define how senescent cells and their secretory programs shape the tumor microenvironment, promote DMG progression, and contribute to therapy resistance. By elucidating these mechanisms, we hope to uncover new therapeutic vulnerabilities in DMGs and expand our broader understanding of senescence as a driver of malignancy in pediatric cancers.



 Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya M. Rangaswamy, Claudia Kleinman, Pratiti Bandopadhayay. Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/785f69efbe0f5fccc4e2b6c0500f58472565b0dc" target='_blank'>
              Abstract B040: Uncovering senescence-driven pathways in diffuse midline glioma progression and treatment resistance
              </a>
            </td>
          <td>
            Timothy H. Chang, Alexander Nassar, Ria Kedia, Shriya Rangaswamy, Claudia L Kleinman, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4e4fc33eb2e8814d38d33c359427c463ea1f89" target='_blank'>
              Resistance to CDK7 inhibitors directed by acquired mutation of a conserved residue in cancer cells.
              </a>
            </td>
          <td>
            Chun-Fui Lai, Victoria I. Cushing, Ellen Olden, C. Bevan, R. C. Coombes, Basil J. Greber, L. Buluwela, Simak Ali
          </td>
          <td>2025-09-08</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 The dynamic evolution of the immune tumor microenvironment (TME) during targeted therapy is a critical yet poorly understood determinant of treatment response and resistance. While most studies compare immune states before and after treatment, temporal immune changes during therapy remain largely uncharacterized, limiting development of effective combination strategies. Here, we investigated immune dynamics throughout targeted therapy using mouse melanoma models that recapitulate human therapeutic responses. Single-cell RNA sequencing (scRNA-seq) identified an inflection point where the inflamed TME during tumor regression, characterized by robust NK cell infiltration, transitions to an immune-excluded state upon onset of drug-tolerant residual disease. We uncovered a unique macrophage subset that orchestrates NK cell recruitment during regression. Depletion of these macrophages using LysM-cre;iDTR mice significantly reduced NK cell infiltration. Extending these findings to human cancer, longitudinal scRNA-seq analysis of melanoma and lung cancer patient samples revealed dynamic NK cell infiltration during targeted therapy, establishing a direct link between innate immune remodeling and treatment outcome. Unlike prior prognostic studies assessing immune states at single time points, our results provide mechanistic evidence of a temporal relationship between NK cell infiltration and therapeutic efficacy. Together, these findings position immune evolution as a driver of acquired resistance and identify macrophage–NK cell crosstalk as a therapeutically actionable axis to overcome immune exclusion and improve targeted therapy.



 Chia-Hsin Hsu, Andrew C. White. Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/064d31f044dc7aac267bdcec71d797f7f005e82d" target='_blank'>
              Abstract A012: Cancer acquires therapy resistance by converting immune infiltration to exclusion through NK cell-macrophage interactions
              </a>
            </td>
          <td>
            Chia-Hsin Hsu, Andrew C White
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung cancer remains one of the most prevalent and lethal malignancies worldwide, responsible for nearly 1.8 million deaths annually, which accounts for approximately 18.7% of global cancer-related mortality. Cisplatin, a highly effective and widely utilized anticancer drug, is particularly effective against solid tumors and serves as a cornerstone of adjuvant chemotherapy for lung cancer. Despite continuous optimization of cisplatin-based chemotherapy regimens, the emergence of cisplatin resistance frequently results in treatment failure, significantly limiting its clinical utility and therapeutic efficacy. To address this challenge, researchers have extensively investigated the biological mechanisms underlying cisplatin resistance, including impaired DNA repair pathways and inhibition of apoptosis. Among these mechanisms, epigenetic regulation—encompassing DNA methylation, histone modifications, and noncoding RNA (ncRNA) regulation—has emerged as a critical factor in mediating cisplatin resistance by modulating gene expression and signaling pathways. This review comprehensively explores the role of epigenetic mechanisms in cisplatin resistance in lung cancer, highlighting recent research findings and their potential implications for developing strategies to overcome drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b34c6c87fbf11d6f9ef8e5eb14596e031f42344" target='_blank'>
              Regulation of cisplatin resistance in lung cancer by epigenetic mechanisms
              </a>
            </td>
          <td>
            Xiaoyan Yu, Luyu Jia, Qing Tang, Qichun Zhou, Genshu Wang, Sumei Wang
          </td>
          <td>2025-08-24</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome.
              </a>
            </td>
          <td>
            Chih-Lin Hsieh, Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Schillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Paclitaxel remains a cornerstone of cancer therapy, yet resistance significantly limits its clinical efficacy. Emerging evidence implicates circular RNAs (circRNAs) as key regulators of paclitaxel resistance, offering potential biomarkers and therapeutic targets. This review synthesizes current knowledge on circRNA-mediated resistance mechanisms, categorizing their roles as promoters or suppressors of chemoresistance. We highlight two major pathways—miRNA sponging and protein interactions—through which circRNAs alter drug sensitivity. Moreover, we focus on enhancement of epithelial–mesenchymal transition and modulation of autophagy as two significant downstream pathways influenced by the established primary mechanisms. Notably, our analysis reveals context-dependent dualities, where certain circRNAs exert opposing effects in different cancers, underscoring the need for tissue-specific therapeutic strategies. Furthermore, we identify critical gaps in understanding circRNA regulatory networks and their clinical translatability. By evaluating circRNA signatures associated with paclitaxel response, this review proposes a framework for biomarker development and combination therapies to circumvent resistance. Our findings emphasize the urgency of functional studies and standardized profiling methods to harness circRNAs for personalized oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6cc3f7c3626a0076f8fb95907d4419e7ad6fdb" target='_blank'>
              Circular RNAs: key regulators of paclitaxel sensitivity and resistance in cancer
              </a>
            </td>
          <td>
            Zahra Tajik, S. Ghafouri-Fard
          </td>
          <td>2025-08-18</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Despite the success of endocrine therapy (ET) in treating hormone receptor-positive breast cancer, a significant proportion of patients relapse during or after treatment, making ET resistance a major clinical challenge. Previously we have shown that ET-resistant breast cancer cells exhibit reduced ceramide levels and an increased sensitivity to ceramide-induced cell death. Here, we demonstrate that ceramides induce a distinct transcriptional reprogramming in ET-resistant cells, characterized by upregulation of endoplasmic reticulum stress (EnRS) pathways. Ceramide-induced EnRS is PERK-dependent and functionally linked to cell death in multiple models of ET resistance. Using a photoactivatable ceramide probe, we identify TRAM1 as a functionally important ceramide-interacting protein (CIP) in ET-resistant cells that correlates with worse relapse-free survival and a more aggressive breast cancer phenotype in luminal breast cancer patients. Additionally, knockdown of TRAM1 phenocopies ceramide action in ET resistance, thereby suggesting its role in mediating ceramide-induced lethal actions in ET resistance. Together, our findings reveal that ET-resistant breast cancer cells are more sensitive to PERK-mediated EnRS as compared to their ET-sensitive counterparts. Ceramides can exploit this dependence by interacting with CIPs such as TRAM1, leading to PERK activation and consequential cell death preferentially in the ET-resistant breast cancer models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b745d88155de10769304cbcee438fd06edfb58c0" target='_blank'>
              Ceramide-induced Endoplasmic Reticulum Stress as a Targetable Vulnerability in Endocrine Therapy-Resistant Breast Cancer
              </a>
            </td>
          <td>
            Purab Pal, S. Chitkara, Godwin K. Sarpey, Fatimah Alani, Huiping Zhao, Malik Ata, Jun Qu, Rachel Schiff, Debra A. Tonetti, Geoffrey L. Greene, Jonna Frasor, G. E. Atilla‐Gokcumen, Jonathan L. Coloff
          </td>
          <td>2025-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75865e4fe743320d35269c6c38da63e48d3392a1" target='_blank'>
              Lineage plasticity and histological transformation: tumor histology as a spectrum.
              </a>
            </td>
          <td>
            Xiaoling Li, Eric E. Gardner, Sonia Molina-Pinelo, Clare Wilhelm, Ping Mu, Á. Quintanal-Villalonga
          </td>
          <td>2025-09-30</td>
          <td>Cell research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68d27cc9b475222eaa7f4e83e26c9db176ee78f1" target='_blank'>
              Novel mutations in transducin-like enhancer of split 6 cause infertility in women with embryonic arrest.
              </a>
            </td>
          <td>
            Jifan Tan, Yue Jiang, Ching Wan, Kehan Zhu, Yan Xu, Yanwen Xu, Jia Huang
          </td>
          <td>2025-09-27</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Short tandem repeat (STR) length is a known determinant of pathogenicity in a variety of human disorders. The repeat sequence itself can modulate disease severity and penetrance; however, the broader impact of STR sequence variation on gene expression in the general population remains poorly understood. Results Here, we analyze the sequence composition of STRs across two general population cohorts of unrelated individuals (n = 3,150) and report that ~ 7% of STRs exhibit sequence variability, with distinct patterns observed among different ethnic groups. These variable repeats are more prone to expansion and are frequently found in proximity to Alu elements. Notably, STRs with variable motifs are often found near splice junctions of genes involved in brain and neuronal functions. This is supported by the differential expression of genes associated with neuron and cellular projection functions, driven by the presence of distinct STR sequences. Conclusions Our findings underscore the previously unrecognized role of STR sequence variability in modulating gene expression and contributing to human phenotypic diversity. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03754-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6083af317dff42aa59e8a9cce0eb432b06b5179" target='_blank'>
              Diverse short tandem repeat sequences influence gene regulation in human populations
              </a>
            </td>
          <td>
            Aleksandra Mitina, W. Engchuan, Brett Trost, G. Pellecchia, Stephen W. Scherer, Ryan K C Yuen
          </td>
          <td>2025-09-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third leading cause of cancer-related deaths. Integrating cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present a comparative mRNA-seq analysis of tissue samples from 32 CRC patients, pairing tumors with adjacent healthy tissues. Differential expression gene (DEG) analysis revealed dysregulated metabolic programs. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2, which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potential of the Xc- system, we analyzed gene perturbations from CRISPR screens across various cell types and used functional assays in five primary patient-derived organoid models. We identified a previously uncharacterized cell surface protein signature predicting chemotherapy resistance and highlighted the causality and potential of pharmacological blockage of ferroptosis as a promising avenue for cancer therapy. Redox homeostasis, ion/amino acid transporters, and regulators of neuronal survival and differentiation were pathways associated with these co-dependent genes in patient specimens. This study highlights several potential clinical targets for CRC therapy and promotes the use of patient-derived organoids oids to functionally validate in silico predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af3aa92ab85625c6d369fc2c084fc1ee49ca6118" target='_blank'>
              Patient-specific pharmacogenomics demonstrates xCT as predictive therapeutic target in colon cancer with possible implications in tumor connectivity.
              </a>
            </td>
          <td>
            M. Strecker, Keren Zohar, Martin Böttcher, T. Wartmann, H. Freudenstein, Maximilian Doelling, M. Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, D. Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. Kahlert
          </td>
          <td>2025-09-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) is one of the leading cause of cancer-associated mortality globally. Its tumor microenvironment consists of various types of cells, which have a great impact on patient response and prognosis. The present study used scRNA-seq technology to investigate gene expression pattern of various cell types in NSCLC and their correlation with immune microenvironment. Methods We acquired NSCLC single-cell RNA sequencing data from GSE117570 and employed UMAP clustering analysis to explore the major cell subpopulations. Additionally, we explored the differential expression of important genes among various cell types and their associations with immune cell infiltration, scores of tumor microenvironment. Results Our results exhibited the heterogenicity of cell types in NSCLC and also indicated more than 60 genes whose expression displayed significant differences between cell groups, such as AP1S1, BTK, FUCA1, NDEL14, TMEM106B and UNC13D. Expression of these genes correlated with immunocyte infiltration, tumor microenvironment scores, indicating their potential roles in tumor progression and therapeutics. Conclusion Our single-cell transcriptomic dissection of NSCLC highlights cellular heterogeneity as well as potential biomarkers associated with immune microenvironment. The findings provide a foundation for the rational development of biomarker-driven personalized treatment and therapy by extending the cell-type specificity of current designs of NSCLC therapeutics. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03494-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cfe159d40376ac64bc4970c70d60376979b77ad" target='_blank'>
              Single-cell RNA-seq reveals gene expression heterogeneity in NSCLC and its link to the immune microenvironment
              </a>
            </td>
          <td>
            Jinzhuo Ren, Hua Zhang, Haiyong Wu
          </td>
          <td>2025-09-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e99818fa12da57a725338e19c1d3d4c809e4ac" target='_blank'>
              Structural variants are enriched in deleterious visible phenotypes in Drosophila.
              </a>
            </td>
          <td>
            Alejandra Samano, Matthew Musat, Mihir Junaghare, Asad Ahmad, Mehlum Ali, Sebastian Alves, Sreeram Pasupuleti, Jelisha Perera, Omar Saada, Brady Sabido, Trevor Smith, Sophie Walz, Mahul Chakraborty
          </td>
          <td>2025-08-21</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Tumor dormancy represents a clinically significant but poorly understood state in which disseminated cancer cells persist in a quiescent, non-proliferative state, evading conventional therapies and driving late relapse. This review summarizes recent advancements in experimental models—both in vitro and in vivo—that recapitulate the full spectrum of dormancy, including its induction, maintenance, and reactivation. Crucial intrinsic pathways such as ERK/p38 signaling shifts, epigenetic remodeling, and metabolic adaptations and microenvironmental and immune-mediated cues that regulate dormant cell fate are discussed. Therapeutic strategies aimed at maintaining dormancy, reactivating dormant cells for elimination, or directly targeting their survival pathways have been highlighted. By integrating insights from model systems, molecular regulation, and therapy, this review aims to provide a comprehensive framework that informs future efforts to target dormant cancer cells and ultimately reduce recurrence and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/780c88da1c462505181b638ab331143519a6fc32" target='_blank'>
              Understanding Tumor Dormancy: from Experimental Models to Mechanisms and Therapeutic Strategies
              </a>
            </td>
          <td>
            Jaebeom Cho
          </td>
          <td>2025-08-31</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The mechanism underlying the pathogenic impact of mutations within intrinsically disordered regions of proteins remains enigmatic, and the mechanisms behind compensatory responses to these perturbations lie within an even deeper veil of obscurity. This study focuses on the compensatory mechanisms of single nucleotide variants within the disordered C-terminal tail of the BTG2 protein, a crucial regulator in cell cycle control. Here, we develop a novel approach by combining molecular dynamics simulations with time-dependent linear response theory to accurately compute the long-distance coupling dynamics between the tail and the structured domain. Using this approach, we reveal how specific mutations can counteract the functional disruptions caused by a known disease-associated mutation, V141M. Our findings demonstrate that the disordered tail regulates critical binding sites allosterically, and a weakening of this modulation may contribute to disease manifestation. However, compensatory mutations restore lost interactions between the disordered region and binding sites, exerting long-distance dynamic control over both critical binding sites and the mutation site 141M. This secondhand allosteric control could be a general mechanism for compensatory mutations to rescue function. These insights not only illuminate the pathogenic mechanisms at play but also offer a framework for identifying potential therapeutic targets in diseases associated with disordered protein regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44953c05dc0147543f5bd00f5cc928d283c8664e" target='_blank'>
              Second-order allosteric control as a mechanism for compensatory mutations in B-cell translocation gene 2.
              </a>
            </td>
          <td>
            Nicholas J. Ose, P. Campitelli, Tushar Modi, S. Ozkan
          </td>
          <td>2025-08-25</td>
          <td>Protein science : a publication of the Protein Society</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite extensive prior research on prostate cancer (PCa) transcriptomics, the molecular mechanisms underlying the disease’s progression, particularly in the castration-resistant or metastatic stages, remain incompletely understood. The majority of recent research has concentrated on bulk RNA sequencing, which could mask the variation found in tumor microenvironments. This study aims to address this gap by integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing with weighted gene co-expression network analysis (WGCNA) to investigate the molecular mechanisms of PCa at a higher resolution. In order to further individualized treatment plans for PCa, we aim to discover important genes and signaling pathways that could be used as therapeutic targets. We first preprocessed expression profile data from prostate cancer tissue samples, selecting 9,809 high-quality cells from a dataset. Following batch correction with Harmony and dimensionality reduction with principal component analysis (PCA), we used the Louvain clustering algorithm to divide the cells into discrete subtypes. The clusters were then visualized using t-SNE. This resulted in 16 cellular subtypes categorized into five major cell types: epithelial cells, monocytes, endothelial cells, CD8 + T-cells, and fibroblasts. Analysis of receptor-ligand pairs uncovered significant interactions between monocytes and both tumor cells and endothelial cells. Applying the high-dimensional WGCNA (hdWGCNA) method to construct a gene co-expression network, we detected seven gene modules, four of which were highly expressed in tumor cell subtypes and contained 380 key genes. Combining pathway analysis, we ultimately screened six key genes: CNPY2, CPE, DPP4, IDH1, NIPSNAP3A, and WNK4. We used Cox univariate regression and least absolute shrinkage and selection operator (lasso) regression techniques to build a prognostic prediction model that included these six important genes based on clinical data gathered from PCa patients. The prognostic prediction model constructed in this study demonstrated excellent predictive performance in both the training set and an external validation set, with the high-risk group showing significantly lower overall survival (OS) than the low-risk group. Furthermore, there was a substantial correlation found between risk scores and several immune-related gene sets, chemotherapeutic drug sensitivity, and tumor immune infiltration. High- and low-risk groups exhibited significant differences in immune cell content, immune factor levels, and immune dysfunction. Further analysis revealed significant correlations between the expression levels of model genes and multiple disease-related genes. Through Gene Set Variation Analysis (GSVA) and Gene Set Enrichment Analysis (GSEA), we uncovered perturbations in multiple signaling pathways in high- and low-risk groups, potentially impacting the prognosis of PCa patients. This study uncovers key genes and signaling pathways in the prostate cancer tumor microenvironment, particularly genes such as CNPY2, CPE, DPP4, IDH1, NIPSNAP3A and WNK4, which have potential as therapeutic targets. Our findings provide new insights into personalized treatment strategies for PCa and warrant further clinical validation in the future. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15682-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779fa642ce8a3880c524211b95657271b91e6425" target='_blank'>
              Integrative analysis of molecular mechanisms in prostate cancer via single-cell RNA sequencing and weighted gene co-expression network analysis
              </a>
            </td>
          <td>
            Jing Zhai, Yizhou Wang, Yu Zhang, Wenhui Zhu, Xinyu Xu, Yu Peng, Guanxiong Ding
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Eukaryotic ribosomal RNA (rRNA) genes exhibit hyper-variability at non-conserved regions known as Expansion Segments (ESs). Due to the numerous rRNA copies in the genome, editing ESs is challenging, and their significance remains unclear. In this study, we analyze rRNA variant frequencies in the UK Biobank population, revealing that highly abundant ES variations are causally linked to human health and physiology. We developed a Ribosome Variation Analysis (RiboVAn) method, identifying both heritable germline variants and a larger proportion of low-heritability, likely somatic variants. The most heritable variants cluster within four ESs of the 28S rRNA, with specific variants in es15l associated with adiposity, es39l linked to body dimensions, and es27l associated with blood-related traits and diseases. Variant-chromosome specificity is observed where functional variants are linked to certain rDNA chromosomes. These findings causally link rRNA sequence variation to human traits and establish that ESs have distinct and important functions in human physiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7471647088548ae368436bbc488b4110ad53685" target='_blank'>
              rRNA paralogs with variations, rRNA-subtypes, affect diverse human phenotypes
              </a>
            </td>
          <td>
            Daphna Rothschild, Jonathan K. Pritchard, Maria Barna
          </td>
          <td>2025-09-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematological malignancy driven by diverse genetic aberrations, with the Philadelphia chromosome and its resultant BCR-ABL1 fusion gene constituting key pathogenic drivers. Atypical BCR-ABL1 fusion transcripts have distinctive structural and functional properties. Structural divergence in these variants leads to functional alterations of encoded oncoproteins, potentially influencing disease progression and therapeutic responsiveness. Conventional diagnostic modalities, including reverse transcription-PCR and fluorescence in situ hybridization, may fail to detect rare variants, necessitating complementary high-sensitivity techniques such as next-generation sequencing). Tyrosine kinase inhibitors (TKIs), including imatinib and dasatinib, remain cornerstone treatments; however, marked inter-variant heterogeneity in TKI responsiveness is observed: Patients harboring e13a3/e14a3 transcripts generally show favorable prognoses, while those with e1a3/e6a2 variants demonstrate an increased risk of relapse and/or TKI resistance, often requiring multimodal strategies combining chemotherapy or allogeneic hematopoietic stem cell transplantation. Although Chimeric Antigen Receptor)-T cell therapy has shown promise in treating (Philadelphia chromosome-positive B-cell Acute Lymphoblastic Leukemia, its application in CML, particularly in variants such as e1a3 or e6a2, is not currently recommended as a first-line treatment. Despite advances in elucidating the clinical implications of fusion gene heterogeneity in leukemogenesis, the prognostic value of atypical BCR-ABL1 isoforms requires further validation through multicenter studies with extended cohorts. This review aimed to summarize cases of atypical fusion genes in CML, with analysis of clinical characteristics, therapeutic interventions, and prognostic outcomes, to provide clinicians with enhanced reference material for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec3c00c917da742543a693d4c05a294ce191255" target='_blank'>
              The silent players: Atypical BCR-ABL isoforms as biomarkers and therapeutic hurdles in CML pathogenesis (Review)
              </a>
            </td>
          <td>
            Xin Zhou, Ai Li, Dexiao Kong, Yuqi Shi, Peipei Zhang, Ning-Ning Shan
          </td>
          <td>2025-09-24</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite the development of various effective anaplastic lymphoma kinase tyrosine kinase inhibitors (ALK-TKIs), therapeutic resistance remains a major challenge. Both on-target and off-target mechanisms have been identified as key contributors to resistance. With the popularization of genetic testing and the development of precision therapies, the prognosis and survival of patients with ALK-positive non-small cell lung cancer (NSCLC) have improved. However, even with second- and third-generation ALK-TKIs, overcoming resistance remains difficult. Resistance frequently arises during approved treatments, underscoring the need for further research to elucidate the molecular events and resistance mechanisms associated with ALK-positive lung cancer. The discovery of anaplastic lymphoma kinase (ALK) rearrangement as an actionable oncogenic driver in NSCLC has established a biomarker-driven treatment paradigm for advanced disease. This article summarizes current knowledge of the mechanisms of resistance to ALK-targeted therapy in lung cancer, including both primary and acquired mechanisms, treatment strategies following resistance, recent therapeutic advances, and the impact of the immune system and tumor microenvironment. A deeper understanding of ALK-targeted therapy resistance is critical for developing new treatment strategies and may provide important insights to guide the diagnosis, treatment, and management of patients with resistant ALK+ lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880963ca5d4f45d24c1ab8fad996bba2e1305930" target='_blank'>
              New advances in understanding the mechanisms and treatment challenges of ALK-targeted therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Mengle Long, Shixuan Peng, Qingyang Wen, Zhijian Yin, Xinwen Zhang, Haoyu Tan, Yun Xu, Yongjun Wu
          </td>
          <td>2025-08-25</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is an aggressive and heterogeneous liver cancer with restricted therapy selections and poor diagnosis. Although there have been great advances in genomics, the molecular mechanisms essential to HCC progression are not yet fully implicit, particularly at the single-cell stage. This research utilized single-cell RNA sequencing technology to evaluate transcriptional heterogeneity, immune cell infiltration, and potential therapeutic targets in HCC. A detailed bioinformatics pipeline used in the experiment included quality control, feature selection, dimensionality reduction using Principal Component Analysis (PCA), Uniform Manifold Approximation and Projection (UMAP), and t-distributed stochastic neighbor embedding (t-SNE), clustering, differential gene expression, pseudotime trajectory inference, and immune cell profiling with GSEA and survival analysis examining potential biomarkers of survival. Key findings include the identification of 1178 differentially expressed genes (DEGs), with macrophage infiltration contributing to immune evasion. Notably, APOE and ALB are linked to a better prognosis, while XIST and FTL are associated with poor survival. The potential drug candidates include IGMESINE in the case of SERPINA1 and PKR-A/MITZ for APOA2 in the gene-drug interaction analysis. Graph Neural Network (GNN) is used to predict drug-gene interactions and rank potential therapeutic candidates. The model shows robust predictive performance (R²: 0.9867, MSE: 0.0581) and identifies important drug candidates, such as Gadobenate Dimeglumine and Fluvastatin, and describes repurposing opportunities in network analysis, enhancing computational drug discovery for novel treatments. This research sheds new light on HCC tumor evolution, immune suppression, and the potential drug target based on the viewpoint of the importance of single-cell approaches in liver cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e827b59658da5d3c8017bf844efafdcd1183fd" target='_blank'>
              Integrating single-cell RNA sequencing and artificial intelligence for multitargeted drug design for combating resistance in liver cancer
              </a>
            </td>
          <td>
            Houhong Wang, Youyuan Yang, Junfeng Zhang, Wenli Chen, Jingyou Dai, Changquan Li, Qing Li
          </td>
          <td>2025-09-02</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Single base substitution (SBS) mutations, particularly C > T and T > C, are increased owing to unrepaired DNA replication errors in mismatch repair-deficient (MMRd) cancers. Excess CpG > TpG mutations have been reported in MMRd cancers defective in mismatch detection (dMutSα), but not in mismatch correction (dMutLα). Somatic CpG > TpG mutations conventionally result from unrepaired spontaneous deamination of 5’-methylcytosine throughout the cell cycle, causing T:G mismatches and signature SBS1. It has been proposed that MutSα detects those mismatches, prior to error correction by base excision repair (BER). However, other evidence appears inconsistent with that hypothesis: for example, MutSα is specifically expressed in S/G2 phases of the cell cycle, and defects in replicative DNA polymerase proofreading specifically cause excess CpG > TpG mutations in signature SBS10b. Methods We analysed mutation spectra and COSMIC mutation signatures in whole-genome sequencing data from 1803 colorectal cancers (164 dMutLα, 20 dMutSα) and 596 endometrial cancers (103 dMutLα, 9 dMutSα) from the UK 100,000 Genomes Project. We mapped each C > T mutation to its genomic features, including normal DNA methylation state, replication timing, transcription strand, and replication strand, to investigate the mechanism(s) by which these mutations arise. Results We confirmed that dMutSα tumours specifically had higher CpG > TpG burdens than dMutLα tumours. We could fully reconstitute the observed dMutSα CpG > TpG mutation spectrum by adding CpG > TpG mutations in proportion to their SBS1 activity to the dMutLα spectrum. However, other evidence indicated that the SBS1 excess in dMutSα cancers did not come from 5’-methylcytosine deamination alone: non-CpG C > T mutations were also increased in dMutSα cancers; and, in contrast to tumours deficient in BER, CpG > TpG mutations were biased to the leading DNA replication strand, at similar levels in dMutSα and dMutLα cancers, suggesting an origin in DNA replication. Other substitution mutations usually corrected by BER were not increased in dMutSα tumours. Conclusions There is a CpG > TpG and SBS1 excess specific to dMutSα MMRd tumours, consistent with previous reports, and we find a general increase in somatic C > T mutations. Contrary to some other studies, the similar leading replication strand bias in both dMutSα and dMutLα tumours indicates that at least some of the excess CpG > TpG mutations arise via DNA replication errors, and not primarily via the replication-independent deamination of 5’-methylcytosine. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01525-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f0be2946a1331e8205341cd8d8b7fbe7da7996" target='_blank'>
              Replication-associated mechanisms contribute to an increased CpG > TpG mutation burden in mismatch repair-deficient cancers
              </a>
            </td>
          <td>
            Joseph C Ward, Ignacio Soriano, Steve Thorn, Juan Fernández-Tajes, Kitty Sherwood, Güler Gül, Joost Scheffers, A. Frangou, Ben Kinnersley, Ioannis Kafetzopoulos, Duncan Sproul, Sara Galavotti, C. Palles, Andreas J. Gruber, David N. Church, Ian Tomlinson
          </td>
          <td>2025-08-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin W. Włodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cancer cells arise from the cumulative acquisition of genetic and epigenetic alterations that affect vital cellular functions. Genomic instability results from deficiencies in protective mechanisms, such as cell cycle checkpoints, DNA replication control, or DNA repair. Claspin integrates a group of crucial proteins that maintain genome integrity. It participates in key cellular events such as DNA damage checkpoint activation, DNA replication, replication stress responses, DNA repair, epigenetic memory, and apoptosis. Given its crucial functions, a role for Claspin in cancer is not a surprise. Indeed, there is a considerable body of evidence linking Claspin deregulation with cancer. For instance, over-expression of Claspin and Tim promoted the survival of cancer cells by enabling adaptation to oncogene-induced replication stress. In addition, Claspin gene (CLSPN) mutations that affect checkpoint regulation have been identified in cancer patients, suggesting that they may contribute to cancer development. Changes in Claspin expression levels may be used as a prognostic marker in several types of cancer. Finally, several therapy-resistance signaling pathways seem to converge onto Claspin’s stabilization, turning Claspin into an attractive target for chemo- and radio-sensitization. In this review, we will focus on the role of Claspin in cancer and ways in which Claspin can be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86699ba9ba454ef5c75f676effbf8547c5ac5e0c" target='_blank'>
              Claspin and Cancer: Where Are We Now?
              </a>
            </td>
          <td>
            D. Azenha, Teresa C. Martins
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2μm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Formaldehyde (FA) is a highly reactive aldehyde that forms a variety of adducts with biomolecules, including DNA base adducts, interstrand crosslinks, intrastrand crosslinks, and DNA–protein crosslinks (DPCs). FA is produced from several exogenous and endogenous sources and has been linked to many cancer types. Persistent adducts are often bypassed by error-prone translesion synthesis (TLS) polymerases, causing mutations that contribute to the FA-induced mutational spectrum. However, it is unknown how DPC repair and TLS coordinate to bypass FA damage, and how bypass by different TLS polymerases alters FA’s mutational spectrum. Here, we use an established yeast mutational reporter system to confirm that FA preferentially mutagenizes guanine residues within single-stranded DNA (ssDNA). We find that functional TLS prevents chromosomal instability following FA exposure and that FA-induced chromosomal rearrangements are dependent on DPC repair. Finally, we find that the TLS polymerases Rev1 and Pol η function together to prevent FA mutagenesis. Overall, these results outline the mechanisms by which TLS and DPC repair coordinate to bypass FA-induced ssDNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b38028676b90bbc4fd75eb225bb7dbf166a301" target='_blank'>
              Translesion polymerases and DNA–protein crosslink repair shapes the cellular response to formaldehyde-induced DNA damage in ssDNA
              </a>
            </td>
          <td>
            Thomas Blouin, Elizabeth Ampolini, Kristina Stayer, Parameshwar Suresh, P. A. Mieczkowski, Natalie Saini
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="This study aimed to identify novel prognostic biomarkers and therapeutic strategies by focusing on drug resistance-related phenotypes. We integrated multi-omics data from various databases encompassing sequencing, clinical information, resistance-related gene sets, and genomic alteration data. Single-sample gene set enrichment analysis (ssGSEA) was employed to calculate resistance scores for individual samples, which were subsequently applied in survival analysis. Furthermore, we utilized machine learning algorithms to develop a robust prognostic model validated across multiple independent datasets. Our findings revealed 12 genes consistently linked to PAAD prognosis in diverse datasets. Pathway enrichment analysis indicated that the high-risk group was enriched in pathways associated with systemic lupus erythematosus and the cell cycle, while the low-risk group showed significant enrichment in neuroactive ligand-receptor interaction pathways. Additionally, immune cell infiltration analysis exhibited substantial differences between risk groups, with the high-risk cohort presenting lower levels of activated CD8 + T cells but higher levels of regulatory T cells. The random survival forest (RSF) model demonstrated superior predictive performance, achieving a concordance index of 0.634 and time-dependent ROC AUCs of 0.973, 0.978, and 0.996 at 1, 2, and 3 years, respectively. In conclusion, this study identifies 12 critical drug resistance genes in PAAD and highlights the associated immune differences in patient risk, paving the way for targeted immunotherapy research to improve therapeutic strategies against this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9036a9dfcaada4e83527a56e65b2bda1ef503c41" target='_blank'>
              To Develop Prognostic Markers Related to Drug Resistance in Pancreatic Cancer Patients Based on Multiple Machine Learning Methods
              </a>
            </td>
          <td>
            Song Xu, Chuanmin Deng, Z. Man, Song Yang, Ming Xu
          </td>
          <td>2025-10-10</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND
Short-read genome sequencing (sr-GS) affords efficient and accurate characterisation of apparently balanced chromosomal rearrangement (ABCR) breakpoints except in 9%-11% of cases that remain undetectable.


METHODS
Among 117 ABCR that we studied in patients with abnormal phenotype, 14 (11.9%) could not be detected by our current strategy including sr-GS, alignment against the GRCh38 reference genome and structural variant (SV) detection using Breakdancer V.1.4.5. These were all reciprocal translocations, 10 of which implicated constitutive heterochromatin, acrocentric short arms or pericentromeric regions. We re-aligned the sequencing data against the T2T-CHM13 V.2.0 reference genome and re-analysed them using five other SV callers (DELLY, GRIDSS, LUMPY, Manta and SvABA). In addition, 11 ABCRs were further characterised using FISH, linked-read sequencing, long-read sequencing or optical genome mapping, either isolated or combined.


RESULTS
We were able to characterise the breakpoints at the bp level for 12 translocations and identify specific breakpoint patterns using Integrative Genome Viewer (IGV). In each translocation, at least one breakpoint involved highly repetitive elements such as alpha-satellites, segmental duplications, satellite repeats or other poorly mapped regions. For six out of 12 patients, one of the breakpoints could explain the phenotype either by gene disruption (CAMTA1, DYRK1A, NLGN4X) or position effect (BMP2, DIAPH2, SIX3).


CONCLUSION
Failure of sr-GS is due to highly repetitive genomic regions at SV breakpoints, either absent from the reference genome or not attributed to a unique position. The resolution of ABCRs is essential to patients' care since it allowed us to conclude to a pathogenic variant in 50% of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b412e3d779e057772a957a4417d07748a1c0ca16" target='_blank'>
              Resolving structural variations missed by short-read sequencing uncovers their pathogenicity.
              </a>
            </td>
          <td>
            C. Schluth-Bolard, Laïla El Khattabi, P. Rollat-Farnier, N. Chatron, Marion Beaumont, N. Reynaud, K. Uguen, F. Diguet, A. Labalme, Claire Bardel, T. Mantere, Vérane Bard, Andreea Apetrei, A. Afenjar, F. Amblard, Jeanne Amiel, Sophie Christin-Maître, F. Devillard, M. Fradin, B. Isidor, A. Lokchine, S. Jaillard, R. Olaso, Massimiliano Rossi, S. Valence, J. Deleuze, A. Hoischen, J. Siffroi, D. Sanlaville
          </td>
          <td>2025-08-20</td>
          <td>Journal of medical genetics</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83bb8c6ba0429e042acd5b6b051c6318cb9fb300" target='_blank'>
              Identification, functional insights and therapeutic targeting of EMT tumour states.
              </a>
            </td>
          <td>
            Anqi Dong, Cédric Blanpain
          </td>
          <td>2025-09-25</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ab9d9615fda47177afd65cb9d942ab1bd6dc87" target='_blank'>
              Thymic epithelial cells amplify epigenetic noise to promote immune tolerance.
              </a>
            </td>
          <td>
            Noah Gamble, Jason A Caldwell, Joshua McKeever, Caroline Kaiser, Alexandra Bradu, Peyton J. Dooley, Sandy Klemm, William J. Greenleaf, Narutoshi Hibino, Aaron R. Dinner, Andrew S. Koh
          </td>
          <td>2025-08-20</td>
          <td>Nature</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Located on chromosome 10, the RET gene encodes a transmembrane receptor tyrosine kinase that, when activated by a ligand, is essential for cell differentiation, proliferation, and survival. RET specifically supports the development of neuronal tissues and neural crest-derived cell types. However, RET is a proto-oncogene and mutations can lead to aggressive cancers like thyroid and lung tumours, or endocrine syndromes. Treatments for RET-driven cancers remain limited, relying on therapies that use RET-specific inhibitors. However, they show limited success in clinical settings due to adverse side effects and acquired resistance over time. To find alternative treatments, integrating the principles of synthetic dosage lethality (a genetic interaction causing a lethal effect in cancer cells) with machine learning may identify novel gene targets to complement existing therapies. In previous studies, we sought candidate genes that induce cell death when depleted in cells overexpressing oncogenic RET, but not in normal cells. A previously identified candidate, NCAM1, which is involved in neuronal cell migration, has been found to contribute to the differentiation of neural cell types. Using the SH-SY5Y neuroblastoma cell line as a model, multiple techniques and assays were performed to validate the synthetic lethal relationship between NCAM1 and RET. NCAM1 knockdown cell lines were generated using lentiviral transduction, and a live/dead cell assay was used to assess potential cytotoxic effects of gene knockdown. Additionally, a cycloheximide assay was performed to examine RET half-life by tracking its degradation over twelve hours. Over the summer, the project enabled us to reassess gene priority rankings using synthetic dosage lethal logic, characterize phenotypes of NCAM1 knockdown SH-SY5Y cells, optimize various assays under in-vitro conditions, and consolidate our understanding of NCAM1’s relationship to RET. The project thus demonstrates the value of combining bioinformatic and in-lab techniques as a productive approach to finding precise therapeutic targets against RET-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08538d4a42ece74adec0afb573a8e0619996891d" target='_blank'>
              Identifying Novel Gene Targets in RET-Driven Cancers Using Synthetic Dosage Lethality Models
              </a>
            </td>
          <td>
            Zahra Simpson-Stairs
          </td>
          <td>2025-08-29</td>
          <td>Inquiry@Queen's Undergraduate Research Conference Proceedings</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Errors in alternative splicing (AS) events are recognized as contributors to the tumorigenesis and metastasis of various cancer types. However, the role of aberrant AS events in metastatic renal cell carcinoma (mRCC) remains largely unexplored. We aimed to identify aberrant AS events associated with clinical benefits from immune checkpoint inhibitors (ICIs) and targeted therapies (TTs) in mRCC. Methods We conducted a retrospective analysis on 101 patients with mRCC who received systemic therapy and underwent RNA sequencing with sufficient output quality. Patients were categorized into two cohorts based on whether they received ICIs or TTs. Patients were then further subcategorized as responders or non-responders to systemic therapy based on RECIST v1.1 criteria. Differential gene expression and splicing analyses were performed between responders and non-responders for each cohort. Paired-end bulk RNA sequencing, utilized in the analyses of splicing events, was performed as part of the OncoExTra clinical assay. Novel AS events were analyzed for their potential to generate peptide neoantigens through MHC class I binding predictions. Results Outlier splicing analysis identified 10 aberrant AS events specific to mRCC. Alternative splicing analysis revealed 409 differentially spliced events between responders and non-responders in the ICI cohort and 231 in the TT cohort, with intron retention enriched as the predominant motif of aberrant AS in responders relative to non-responders. Seven unique AS events were enriched in responders, including PTPN6, ACTN1, and SUN2. Predictive neoantigen analysis identified high MHC class I binding potential in peptides from AS events in IFFO1, ZNF692, and SUN2. A novel splice burden gene set was developed from differentially expressed genes in novel splice burden-high samples. Within both ICI and TT cohorts, this splice burden-high gene set was enriched in responders relative to non-responders. The presence of high splice burden was linked to an immunogenic tumor microenvironment, characterized by enriched antigen processing and adaptive immune responses. Nearly 50 adaptive immune cell pathways were enriched in splice burden-high patients, including CD22-mediated BCR regulation, immunoglobulin production, and humoral immune response. Conclusions This study provides a comprehensive analysis of AS events in mRCC, highlighting enrichment of intron retention as potential transcriptomic biomarkers for treatment response. Numerous adaptive immune gene pathways were enriched in mRCC patients with high splice burden, particularly humoral immune responses. Aberrant AS-derived neoantigens may serve as potential targets for adoptive cell therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4c82017f4e331576a7030ef270865526796ed7" target='_blank'>
              27Characterization of aberrant alternative splicing landscape in patients with metastatic renal cell carcinoma
              </a>
            </td>
          <td>
            Benjamin D. Mercier, Ameish Govindarajan, Nathaniel Hansen, Sara Byron, Apurva Hegde, Kristin Leskoske, N. Chawla, Luis Meza, Z. Zengin, R. Barragan-Carrillo, Hedyeh Ebrahimi, Daniela V. Castro, S. Jaime-Casas, M. Zugman, N. Dizman, J. Hsu, A. Chehrazi-Raffle, A. Tripathi, Nicholas J. Salgia, Sumanta Pal, Patrick Pirrotte
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT The human Killer cell Immunoglobulin‐like Receptor (KIR) genes, found on chromosome 19, encode for cell surface protein receptors that, through interaction with their ligand, modulate the action of Natural Killer (NK) cells and some subsets of T lymphocytes. KIR genes exhibit extensive variation through variable gene content, copy number, and allele polymorphism. The combination of KIR genes and their ligands is implicated in various clinical settings including haematopoietic stem cell and solid organ transplant, and infectious disease progression. KIR gene content has been used in the selection of optimal stem cell donors with haplotype variations in recipient and donor giving differential clinical outcomes. With the introduction of massively parallel clonal next generation sequencing and single molecule long read third generation sequencing, allele level determination of KIR genotypes has become feasible. We describe a method for amplicon‐based long read sequencing on the Oxford Nanopore Technologies platform that provides largely unambiguous allele level typing of KIR genes. The method was validated using DNA extracted from 48 10th International Histocompatibility Workshop (IHWS) cell lines with previously published allele level KIR genotypes and 176 Western Australian samples previously tested for the presence or absence of KIR genes. Our long‐read sequencing method was able to accurately determine KIR alleles with an overall concordance of 97%–99% with the published data. Importantly, phasing ambiguity caused by the inability to phase heterozygous base positions over long stretches of gene sequence was resolved in several samples. Thus, our long read PCR sequencing strategy can be used to determine KIR genotypes at allele resolution level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f07dc264b92eef3e3ce6dcccdb40f00d68f9fed" target='_blank'>
              Allele Level Sequencing of Killer Cell Immunoglobulin‐Like Receptor Genes Using Oxford Nanopore Long Read Sequencing
              </a>
            </td>
          <td>
            Jonathan Downing, Daniel Abromeit, L. Truong, F. Mobegi, Patricia Martinez, Lloyd J D'Orsogna, D. de Santis
          </td>
          <td>2025-09-01</td>
          <td>Hla</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDA) remains one of the most lethal malignancies, largely due to limited therapeutic options and the rapid development of treatment resistance. Recent progress in the development of RAS targeting agents, including mutant-specific and pan-RAS inhibitors, has expanded potential treatment strategies for tumors harboring oncogenic KRAS mutations. However, clinical and preclinical evidence indicates that resistance to these agents emerges rapidly, limiting their long-term efficacy. This study aims to investigate the mechanisms underlying resistance to RAS inhibitors and to identify rational combination strategies that enhance therapeutic durability in PDA. Using advanced organoid and in vivo models, we are characterizing resistance to clinically relevant RAS inhibitors and evaluating the therapeutic potential of combining RAS inhibition with other targeted agents. We will also examine the influence of treatment timing, drug synergy, and tumor microenvironmental factors, including immune interactions, on treatment response. Ultimately, this research seeks to inform the development of more effective combination strategies for PDA by elucidating resistance mechanisms and identifying therapeutic vulnerabilities. Insights from this work may support future translational efforts in a disease with pressing unmet clinical needs.



 Vasiliki Pantazopoulou, Casie Kubota, Satoshi Ogawa, Kassidy Curtis, Araceli Herrera Morales, Dannielle Engle. Investigating combination therapies to overcome RAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B092.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb5ac3708ad5c62f9b5c3d584781b0becdc5e90" target='_blank'>
              Abstract B092: Investigating combination therapies to overcome RAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Vasiliki I. Pantazopoulou, Casie S. Kubota, Satoshi Ogawa, Kassidy Curtis, Araceli Herrera Morales, Dannielle D. Engle
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Deoxyribonucleic acid (DNA) serves as the fundamental blueprint for life, governing both the structural development and functional regulation of living organisms. The technology explores the modern understanding of DNA role in heredity and molecular genetics that highlighted key discoveries that have shaped the field. The discovery of intricate molecular mechanisms such as epigenetic modifications, non-coding RNA and the gene environment interactions has evolved beyond Mendelian genetics. The mechanism of genetic inheritance, replication and expression illustrates how traits are passed from one generation to the next generation. This review article highlights how mutation and variations in DNA sequences influence health, evolution and disease susceptibility. A significant focus is placed on revolutionary CRISPR-Cas9 Jain editing technology which has transformed genetic research and holds immense promise for future therapeutic applications. After enabling precise modifications in the genome, CRISPR opens new avenues for the treatment of genetic disorders and advancement in biotechnology. This review emphasizes how the integration of classical molecular genetics with cutting edge gene editing techniques offers a comprehensive understanding of DNA function for addressing agricultural and global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd606e61309a0083af92e77dc5cd4dc92727de21" target='_blank'>
              MODERN INSIGHTS OF DNA FUNCTION AND HEREDITY: A REVIEW OF MOLECULAR GENETICS AND CRISPR GENE
              </a>
            </td>
          <td>
            Mariam Fatima, Mariam Fatima¹, Muhammad Haseeb, Anwar Tarar¹, Arooba Maryam², Muhammad Azhar, Ud Din³˒⁴, Ifra Muzaffar¹, H.M. Saif, UR Rahman⁵, Rida Sattar⁶
          </td>
          <td>2025-08-21</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="T cell receptors (TCRs) orchestrate adaptive immunity, yet the complex, repetitive architecture of the TCR loci has impeded systematic characterization of human genetic variation in the genes encoding the TCR. Using public long-read sequencing data from 2,668 donors, we build a near-complete map of common alleles in TCR V, D, and J genes, revealing amino acid variation at almost every position within V genes. We discover pervasive evidence of natural selection on TCR genes, including balancing selection on a TRAJ gene recognizing an immunodominant influenza epitope and positive selection on a TRAV gene. We find TCR allelic polymorphism alters core functional properties of T cells, including thymic fate commitment, phenotypes in diseased tissues, and cell-surface receptor abundance. Collectively, our findings position inherited variation in TCR genes as a key axis of immunological diversity that may shape interindividual differences in immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d398484bfbdbd87bb2aee53bb4ed8e33eec07466" target='_blank'>
              TCR germline diversity reveals evidence of natural selection on variable and joining alpha chain genes
              </a>
            </td>
          <td>
            S. Mantena, Ali Akbari, Soumya Raychaudhuri
          </td>
          <td>2025-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5b919ff4c0f1e65c48676350abe4ef162d1ad" target='_blank'>
              Pan-cancer single-cell and spatial transcriptomics implicate cancer-associated fibroblasts in neutrophil immunosuppressive phenotypic transitions and immunotherapy resistance.
              </a>
            </td>
          <td>
            Zhiyu Guo, Xujia Li, Lingli Huang, Yue Yan, Mengge Gao, Jinsheng Huang
          </td>
          <td>2025-10-10</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract The ability to interrogate changes within the tumor microenvironment (TME) before, during and following therapeutic intervention could yield important understanding of treatment response and causes for disease progression. Yet, the role of investigational tissue analysis faces key challenges in the clinical setting and the value of integrating longitudinal biopsies with emerging multimodal molecular analyses (“Multi-omics”) remains to be defined. In this study, we conducted a multicenter phase 2 clinical trial examining the effect of a novel cytotoxic T-lymphocyte-associated protein 4/programmed death-ligand 1 bispecific antibody in combination with a dual-epitope blocking anti-human epidermal growth factor receptor 2 antibody in treatment-resistant metastatic breast cancer. We performed longitudinal sampling of patient tumor tissues before and following treatment. Single-cell RNA and T cell receptor sequencing from 334,183 cells from site-matched tumors reveals significant temporal shift of various immune cell populations and phenotypes within the TME associated with treatment responses. Conversely, regulatory T cells were activated while effector T cells, natural killer cells, and dendritic cells were significantly depleted in non-responding tumors. Taken together, these results support that longitudinal analysis of TME to generate multiomics data that can lead to rich insight into disease process and to provide clinical value in evaluating treatment responses. Trial registration number NCT04521179.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d97d0a38c945abea56c7fcf08591e230c989d914" target='_blank'>
              Longitudinal tissue analysis reveals microenvironmental changes correlate with combined immunotherapy and targeted therapy response in metastatic breast cancer
              </a>
            </td>
          <td>
            , Jien Wang, Hengyu Li, Zhijun Liu, Z. Tian, Xinying Lv, Jianjian Peng, Chuangui Song, Jieqiong Liu
          </td>
          <td>2025-10-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematologic malignancy originating from hematopoietic stem cells and driven by the BCR-ABL fusion oncogene. Imatinib (IM), a tyrosine kinase inhibitor, is commonly used as a frontline therapy for CML. However, some patients exhibit primary resistance or show persistent molecular evidence of disease despite treatment. Emerging studies indicate that exosome-derived microRNAs (miRNAs) play a role in mediating drug resistance and may serve as promising biomarkers for cancer diagnosis and predicting therapeutic response. This study aimed to investigate the plasma exosomal miRNA expression profiles in CML patients to identify potential biomarkers associated with IM resistance. Exosomes were isolated from plasma samples of both IM-sensitive and IM-resistant CML patients. The exosomal miRNA content was analyzed using RNA sequencing, followed by differential expression analysis, which revealed 13 upregulated and 21 downregulated miRNAs in IM-resistant patients. Subsequent bioinformatics analysis indicated significant enrichment in pathways related to autophagy and PI3K-Akt signaling. Notably, miR-451a and miR-16–2-3p were among the most significantly upregulated miRNAs in exosomes from IM-resistant individuals. Interestingly, miR-16–2-3p expression showed a strong inverse correlation with clinical laboratory results, specifically blood urea nitrogen and creatinine levels. This pilot study identified plasma exosomal miRNAs, particularly miR-451a and miR-16–2-3p, as potential biomarkers for imatinib resistance in chronic myeloid leukemia. Target gene prediction was performed to explore their regulatory roles. Despite the limited sample size, these findings enhance our understanding of drug resistance mechanisms and warrant further validation in larger cohorts to assess their clinical relevance and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ffa66d21071c796372f54b926f9da678ac08d63" target='_blank'>
              Exosomal miRNA expression profiling in patients with imatinib resistant Chronic myeloid leukemia: A pilot study
              </a>
            </td>
          <td>
            R. Navakanitworakul, Pirun Saelue, Tipparat Penglong, P. Molika, Natakorn Nokchan, Natta Tansila, H. Buncherd, Supinya Thanapongpichat, Kanitta Srinoun
          </td>
          <td>2025-08-29</td>
          <td>PLOS One</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established. Methods We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG. Results Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown. Conclusions This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b1aa46cace7effec1d51940ecccc2bdb09fa3e" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a heterogeneous disease characterized by a broad spectrum of molecular alterations that influence clinical outcomes. TP53 mutations define one of the most lethal subtypes of acute myeloid leukemia (AML), driving resistance to nearly all available treatment modalities, including venetoclax plus azacitidine (VenAza). Yet, the molecular basis of this resistance, beyond affecting transactivation of BCL-2 family genes, has remained elusive. Here, we demonstrate that VenAza treatment leads to reduced transcriptional upregulation of the p53 signaling pathway in TP53 mutant/deficient AML compared to wild-type AML. Functionally, TP53 mutant/deficient AML exhibits selective failure in apoptosis induction rather than impaired G1 arrest or senescence. Despite inhibition of pro-apoptotic BAX and selective enrichment for MCL-1 in TP53 mutant isogenic AML cells, compensatory upregulation of BIM preserved functional mitochondrial outer membrane permeabilization (MOMP). TP53 mutant primary AML tumors at baseline also had retained capacity for MOMP. Instead, TP53 mutant AML exhibited disruption in caspase-3/7 activation to evade apoptosis after VenAza therapy – decoupling the mitochondrial and executioner phases of apoptosis. Importantly, this “post-MOMP brake” is not a bystander effect but itself a driver of VenAza and chemotherapy resistance in TP53 mutant/deficient AML. This previously unrecognized mechanistic insight shifts the focus from mitochondrial priming to terminal caspase blockade in TP53 mutant AML and opens the door for urgently needed therapeutic strategies that reignite apoptosis at its execution point.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56075976a6297045972be1b8a77a5e2796abdc68" target='_blank'>
              TP53 mutations drive therapy resistance via post-mitochondrial caspase blockade
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, E. Olesinski, Fang Qi Lim, S. Jasdanwala, Yang Mi, Nicole Si, En Lin, Daniel Tan, En Liang, Nishtha Chitkara, L. Hogdal, R. C. Lindsley, A. Letai, Koji Itahana, Shruti Bhatt
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cisplatin resistance presents a significant challenge in cancer therapy, emphasizing the necessity for identifying new regulatory elements that influence drug response. Recent research has revealed the importance of long noncoding RNAs (lncRNAs) in chemotherapy resistance, with LINC02381 identified as a potential regulatory factor. 
Through an in-depth bioinformatics analysis, we investigated the impact of LINC02381 on cisplatin resistance in ovarian cancer across various datasets. By conducting differential expression analysis, survival analysis, gene set enrichment analysis (GSEA), and constructing protein-protein interaction (PPI) networks, we identified key pathways associated with LINC02381 expression. 
The results indicated that the altered expression of LINC02381 in patients treated with cisplatin was associated with reduced survival. Functional studies and correlation analyses further demonstrated that this LncRNA influences critical pathways and genes related to apoptosis, efflux, DNA repair, and EMT. Lastly, through an examination of its interactions with microRNA and protein networks, we identified LINC02381 as a ceRNA implicated in cisplatin resistance. 
Our findings suggest that LINC02381 may influence cisplatin sensitivity in ovarian cancer and establish a basis for further experimental validation, including molecular assays or in vivo analyses, and suggest the potential therapeutic targeting of LINC02381 to combat chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a8ae366b9c19996905f6b9e1ed9f603948832f" target='_blank'>
              The Role of LINC02381 in Modulating Cisplatin Resistance in Ovarian Cancer: A Bioinformatics Approach
              </a>
            </td>
          <td>
            Zeinab Karbalaei Pazoki, B. M. Soltani, Mostafa Hosseini, Shiva Irani
          </td>
          <td>2025-09-26</td>
          <td>Iranian Journal of Allergy, Asthma and Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, , Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Transposable elements (TEs), once considered genomic “junk”, are now recognized as critical regulators of genome function and human disease. These mobile genetic elements—including retrotransposons (long interspersed nuclear elements [LINE-1], Alu, short interspersed nuclear element-variable numbers of tandem repeats-Alu [SVA], and human endogenous retrovirus [HERV]) and DNA transposons—are tightly regulated by multilayered mechanisms that operate from transcription through to genomic integration. Although typically silenced in somatic cells, TEs are transiently activated during key developmental stages—such as zygotic genome activation and cell fate determination—where they influence chromatin architecture, transcriptional networks, RNA processing, and innate immune responses. Dysregulation of TEs, however, can lead to genomic instability, chronic inflammation, and various pathologies, including cancer, neurodegeneration, and aging. Paradoxically, their reactivation also presents new opportunities for clinical applications, particularly as diagnostic biomarkers and therapeutic targets. Understanding the dual role of TEs—and balancing their contributions to normal development and disease—is essential for advancing novel therapies and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Emerging RAS inhibitors show promise in treating KRAS-mutated malignancies, but resistance mechanisms limit their clinical efficacy. Given recent clinical findings associating KRAS mutations with reduced response to neoadjuvant therapy in rectal cancer (RC), we aimed to investigate their impact on treatment outcomes and explore potential therapeutic strategies.


METHODS
We conducted a retrospective analysis of 390 rectal cancer patients to evaluate the association of KRAS mutations with disease-free survival (DFS) and response to therapy. We assessed the efficacy of KRAS inhibitors in rectal cancer cell lines, patient-derived organoids (PDOs), and patient-derived cell lines (PDCLs), and explored adaptive resistance mechanisms through transcriptomic profiling and unbiased drug screening experiments.


RESULTS
Mutant KRAS was associated with a reduced DFS and RCs harboring G12C and G12V mutations had less complete pathological responses to neo-adjuvant therapies. KRAS-mutated RC cells demonstrated adaptive resistance to KRAS inhibitors, characterized by transcriptomic restoration of oncogenic pathways, including MYC and E2F, and upregulation of ERBB2/3 expression. Consistently, drug screening identified EGFR family inhibitors as potent combinatorial partners, effectively overcoming KRAS inhibitor tolerance by inducing apoptosis. In patient-derived models, the pan-RAS inhibitor RMC-6236 combined with EGFR inhibitors demonstrated significant synergistic effects and prevented long-term tumor cell outgrowth.


CONCLUSION
Our findings point to the negative impact of KRAS mutations, particularly G12C and G12V, on RC treatment outcomes. Adaptive resistance by upregulation of ERBB genes limits the efficacy of KRAS inhibitors. Combining these with pan-ERBB inhibitors enhances anti-tumor effects in patient-derived cellular RC models, showing its potential as an alternative to the combination with anti-EGFR antibodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9be2135f6b5cf4f9e01a3b8ac31ac51a76a3401" target='_blank'>
              Pan-ERBB Inhibitors Synergize With KRAS Inhibitors in Rectal Cancer.
              </a>
            </td>
          <td>
            Jonas Buchloh, M. Spitzner, Hauke Zimmermann, Xin Fang, Constanza Tapia Contreras, Carolin Schneider, Tiago de Oliveria, Stefan Küffer, Michael Linnebacher, Felix Rühlmann, Lena-Christin Conradi, Matthias Wirth, Michael Ghadimi, Marian Grade, Jochen Gaedcke, Günter Schneider
          </td>
          <td>2025-09-16</td>
          <td>United European gastroenterology journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Emerging evidence suggests that post-translational modifications (PTMs) contribute to osteosarcoma pathogenesis, yet their exact molecular roles require further elucidation. Using the AddModuleScore method, we classified tumor cells on the basis of PTMs scores via single-cell RNA sequencing (scRNA-seq). The robust cell type decomposition (RCTD) approach was then applied to map these single-cell groupings onto spatial transcriptomics (ST), enabling the analysis of cell dependencies and the identification of distinct tumor cell subtypes. A prognostic model was constructed using bulk transcriptomic data to predict therapeutic outcomes in immunotherapy. Laboratory experiments were carried out to confirm the biological function of vimentin (VIM). PTMs scores were significantly elevated in tumor cells, stratifying osteoblastic cells (os) into two clusters: PTMs highos and PTMs lowos. The PTMs highos phenotype exhibited pronounced malignant characteristics and closely interacted with fibroblasts in both scRNA-seq and ST analyses. A set of ten hub genes was identified, forming a consensus machine learning-derived post-translational modification gene signature (CMDPTMS) with strong prognostic predictive capability. Although the high-CMDPTMS group (over the median risk score) was linked to poor outcomes and diminished benefit from immunotherapy, seven drugs were identified that may offer therapeutic promise for these patients. Finally, we confirmed that VIM can inhibit the growth and migration of OS cells. In summary, by integrating bulk RNA-seq, scRNA-seq, and ST, we introduced that CMDPTMS may serve as a powerful tool for enhancing OS prognosis prediction and optimizing immunotherapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/615e86f961c9b41f18269413e8e3b6315010a820" target='_blank'>
              Single-cell and spatial transcriptomics reveal post-translational modifications in osteosarcoma progression and tumor microenvironment
              </a>
            </td>
          <td>
            Yue Cui, Yue Wu, Dongyu Jiang, Tao Ding
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aging is associated with a progressive decline in tissue function and regenerative capacity, partly due to genomic instability, one of the hallmarks of aging1,2. Genomic instability encompasses DNA damage and the accumulation of somatic mutations in post-zygotic cells, yet the specific impact of these mutations on age-related tissue dysfunction remains poorly understood. To address this, we developed a mouse model in which genomic instability was induced specifically in muscle progenitor cells3 through targeted deletion of the Msh2 (ref. 4) and Blm5 genes. This allowed us to assess how elevated DNA damage and somatic mutations, from single-nucleotide variants (SNVs) to structural variants, affect muscle regeneration following injury. These mice exhibited impaired muscle regeneration, characterized by smaller muscle fibers, reduced muscle mass gain and decreased grip strength. Importantly, similar muscle deficits were observed in a second mouse model where somatic mutations were elevated with less substantial DNA damage. These findings provide evidence that the accumulation of somatic mutations can potentially compromise the function of somatic cells, contributing to the aging phenotype in skeletal muscle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80da5aed54842349633cd5c47d8c984f10a52ceb" target='_blank'>
              Induced somatic mutation accumulation during skeletal muscle regeneration reduces muscle strength
              </a>
            </td>
          <td>
            P. Vrtačnik, Lara G. Merino, Santhilal Subhash, Hafdís T. Helgadóttir, Matthieu Bardin, Fabiana Stefani, Depin Wang, Ping Chen, Irene Franco, Gwladys Revêchon, Maria Eriksson
          </td>
          <td>2025-08-20</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Glutamine-dependence of cancer cells reduces local glutamine availability, which hinders anti-tumor T-cell functionality and facilitates immune evasion. We thus speculated that glutamine deprivation might be limiting efficacy of CAR T-cell therapies in cancer patients. We have seen that antigen-specific T cells are unable to proliferate or produce IFN-γ in response to antigen stimulation when glutamine concentration is limited. Using multiple myeloma (MM) as a glutamine-dependent disease model, we found that murine CAR-T cells selectively targeting BCMA in MM cells were sensitive to glutamine deprivation. However, CAR-T cells engineered to increase glutamine uptake by expression of the glutamine transporter Asct2 exhibited enhanced proliferation and responsiveness to antigen stimulation, increased production of IFN-γ, and heightened cytotoxic activity, even under conditions of low glutamine concentration. Mechanistically, Asct2 overexpression reprogrammed CART cell metabolic fitness of CART cells by upregulating the mTORC1 gene signature, modifying the Solute Carrier transporter (SLC) repertoire, and improving both basal oxygen consumption rate and glycolytic function thereby enhancing CART cell persistence in vivo. Accordingly, expression of Asct2 increased the efficacy of BCMA-CART cells in syngeneic and genetically-engineered mouse models of MM, which prolonged mouse survival. In patients, reduced expression of Asct2 by MM cells predicted poor outcome to combined immunotherapy and BCMA-CAR T-cell therapy. Our results indicate that reprogramming glutamine metabolism may enhance anti-tumor CAR T-cell functionality in MM. This approach may also be effective for other cancers that depend on glutamine as a key energy source and metabolic hallmark.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/538f82c530b0189b52d7cd8d3af17c6f114cf936" target='_blank'>
              Reprogramming glutamine metabolism enhances BCMA-CART cell fitness and therapeutic efficacy in multiple myeloma.
              </a>
            </td>
          <td>
            Flor Navarro, Teresa Lozano, Andrea Fuentes-García, Inés Sánchez-Moreno, M. Larrayoz, Pedro Justicia, Beatriz Perucha, Maialen Martinez-Tabar, Rebeca Martinez-Turrillas, N. Casares, Celia Martín-Otal, M. Gorraiz, Erin W. Meermeier, M. Chesi, Douglas Lake, P. L. Bergsagel, Eva Santamaría, M. E. Calleja-Cervantes, Patxi San Martín-Úriz, Lorea Jordana-Urriza, X. Agirre, S. Hervás-Stubbs, J. R. Rodríguez-Madoz, Jose-Al Martinez-Climent, F. Prósper, J.J. Lasarte
          </td>
          <td>2025-09-19</td>
          <td>Blood</td>
          <td>1</td>
          <td>57</td>
        </tr>

        <tr id="Microscopic examination of biopsy tissues remains essential for cancer diagnosis, despite advancements in sequencing technologies. Alterations in nuclear size or the nuclear-to-cytoplasmic ratio are hallmark features of cancer cells and often correlate with disease progression. However, the mechanisms underlying nuclear size abnormalities and their impact on tumor progression remain unclear. In this study, we demonstrate that nuclear hypertrophy occurs in response to enhanced DNA replication stress, a key characteristic of cancer cells. Increased actin polymerization within the nucleus appears to be the primary mechanism driving nuclear hypertrophy downstream of the ATR-CHEK1 pathway. Replication stress-induced nuclear hypertrophy alters transcriptomic profiles and chromatin topology, while reducing the migratory and metastatic capacity of cancer cells. In addition, nuclear hypertrophy in cancer cells is associated with increased infiltration of antitumor immune cells. Our findings suggest that cell-autonomous effects of nuclear hypertrophy do not promote cellular fitness or aggressive characteristics in cancer cells. This may explain why cells with nuclear hypertrophy are not positively selected and persist as a subpopulation during tumor progression and metastasis. Furthermore, the link between replication stress and nuclear hypertrophy provides insights into why enlarged nuclei are consistently observed in advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd09203e34a2e72f651d176a77deed8567aab11" target='_blank'>
              Replication stress-induced nuclear hypertrophy alters chromatin topology and impacts cancer cell fitness.
              </a>
            </td>
          <td>
            Changgon Kim, Semyeong Hong, Soo Hyeon Ma, Joochan Lee, Hee So, Jun Young Kim, Eunbie Shin, Kippeum Lee, Sohee Choi, Juyeon Park, YongKeun Park, You-Me Kim, Ji Hun Kim, Joon Kim
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Multidrug resistance is a significant barrier in cancer therapy largely due to poorly understood regulatory mechanisms. Here we reveal that certain anticancer drugs can bind to newly synthesized proteins prior to reaching their canonical targets, resulting in various forms of protein damage. This binding disrupts protein functions, particularly those of mitochondrial proteins, resulting in substantial cytotoxicity. The protein damage is further exacerbated by mitochondrial reactive oxygen species generated as a consequence of the initial damage, creating a positive feedback loop. In response, cancer cells rapidly initiate a chain of events, which we term the Protein Damage Response (PDR). This includes damage recognition primarily mediated by protein ubiquitination and subsequent damage clearance via the proteasome system. Notably, patients with advanced, drug-resistant metastatic breast or colon cancers exhibit elevated proteasome activity. In an effort to predict drug resistance, we developed a sensitive kit for detecting proteasome levels, enabling the identification and subtyping of patients with high proteasome activity to support tailored therapeutic strategies. Using a three-dimensional tumor slice culture-based drug sensitivity assay and an investigator-initiated clinical trial, we demonstrate that three clinically approved proteasome inhibitors effectively overcome multidrug resistance in colon and breast cancer patients with elevated proteasome activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1785188ad7f7984f485f66a5779a54c95bdf57ed" target='_blank'>
              Suppressing protein damage response to overcome multidrug resistance in cancer therapy
              </a>
            </td>
          <td>
            Fangyuan Shao, Zongjie Li, Hao Xiao, Yujun Chen, Yuheng Zhang, Ling Li, Yuzhong Peng, Xinyi Li, Yuxing Hou, Bo Li, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Xi Han, Jiaxin Yao, Cuiting Zhang, Lijian Wang, Haifeng Li, N. Shao, Kai Miao, Xiaoling Xu, Yanxia Shi, Changhua Zhang, Jun Yan, Ying Lin, Chu-Xia Deng
          </td>
          <td>2025-09-30</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Carcinogenesis, while traditionally attributed to the accumulation of driver mutations in genes regulating cell proliferation and apoptosis, may also be explored as a consequence of fundamental metabolic reprogramming, an idea catalyzed by the Warburg effect, where cancer cells exhibit a paradoxical preference for glycolysis over the far more efficient oxidative phosphorylation. This implies that metabolic dysregulation may be a primary instigator of neoplastic transformation. Our hypothesis proposes that the abrupt loss of cellular energy may stimulate an atavistic response, wherein rapid proliferation and migration are triggered to enhance survival in fluctuating environments. These responses lead to pathological angiogenesis and unchecked cell growth, thereby bridging the gap between genetic and metabolic pathways of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba64bccb0f058f3f967d3066ee9360fe210d037d" target='_blank'>
              Carcinogenesis: An Alternative Hypothesis Comparing Mutagenic Versus Metabolic Models
              </a>
            </td>
          <td>
            A. Alhatem, C. Gagna, Muriel Lambert, Emily Keenan, W. Lambert
          </td>
          <td>2025-09-23</td>
          <td>Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861fd3ffe0c8c1bb6ed63c62ce2660ccaa3e238b" target='_blank'>
              miRNA-mediated resistance mechanisms in prostate cancer: implications for targeted therapy and metastatic progression.
              </a>
            </td>
          <td>
            Mostafa M. Mostafa, M. K. A. El-Aziz, D. Ellakwa
          </td>
          <td>2025-08-29</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54525feaa0ad7b16a4eefd8a46c2d6fd3ccc01e5" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of Single-Nucleotide Variant and Copy Number Aberration Evolution from Single-Cell DNA Sequencing of Tumors.
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2025-09-15</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) remains a major clinical challenge in hematologic oncology, characterized by a continuous evolution of molecular drivers that shape its heterogeneity across the age spectrum. Pediatric B-ALL is generally associated with high cure rates, while adult forms of the disease are often more aggressive and less responsive to treatment. This review examines the age-specific genetic and epigenetic landscapes that contribute to this disparity, revealing how the nature and timing of molecular alterations point to fundamentally different leukemogenic processes. Favorable genetic aberrations, such as ETV6::RUNX1 and hyperdiploidy, are predominant in children, whereas adults more frequently present with high-risk features, including BCR::ABL1 fusions and IKZF1 deletions. Epigenetic distinctions are similarly age-dependent, involving divergent patterns of DNA methylation, histone modifications, and non-coding RNA expression. For example, pediatric B-ALL frequently harbors mutations in epigenetic regulators like SETD2 and CREBBP, while adult B-ALL is more commonly affected by alterations in TET2 and IDH1/2. These molecular differences are not only prognostic but also mechanistic, reflecting distinct developmental trajectories and vulnerabilities. Understanding these age-driven transitions is essential for improving risk stratification and developing precision therapies tailored to the unique biology of B-ALL across the lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb81ee6e89e6609c04144954df008c19fbd5a755" target='_blank'>
              Age-Driven Genetic and Epigenetic Heterogeneity in B-ALL
              </a>
            </td>
          <td>
            Yoana Veselinova, M. Esteller, Gerardo Ferrer
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d108c58970b821d1c2170e04d8c4196841969e5f" target='_blank'>
              Phage therapy and its role in cancer treatment and control.
              </a>
            </td>
          <td>
            H. A. Abd El-Moaty, A. Doghish, Hebatallah Ahmed Mohamed Moustafa, Wagiha S. Elkalla, Ghadir A Sayed, Nourhan H Elshami, S. S. A. Mageed, Osama A Mohammed, Mahmoud A. Elrebehy, Kareem AlFarsi, Amr M. Abdelfatah, Aly M Waseem, Ahmed O Abdulkader, Rabab S. Hamad, Akram N Salah
          </td>
          <td>2025-09-17</td>
          <td>Folia microbiologica</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway.
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Modeling cancer recurrence and associated conditions such as cancer cell dormancy remains a significant challenge in cancer research. We developed a novel in vitro model based on platinum-resistant tumor cells treated with a platinum drug, in which cells progress through all stages of recurrence within 30–40 days post-exposure. Our results demonstrate that treatment of platinum-resistant colorectal cancer cells with oxaliplatin enriches the population with quiescent, dormant cells arrested in the G0/G1 phase. These cells exhibit increased autophagy, elevated expression of stem cell markers, reduced reactive oxygen species (ROS) levels, and heightened resistance to chemotherapy. A key advantage of our model is the high survival rate of residual cells, enabling the maintenance of a large cell population. This facilitates the application of conventional research techniques that are often limited in other recurrence models due to the small size of residual populations. Overall, this model provides opportunities for a wide range of further research for studying specific recurrence- and dormancy-associated signaling pathways and for identifying novel therapeutic targets for preventing tumor recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4b53e10213b4d15495530a1d5bcc0ce84af24bb" target='_blank'>
              An in vitro tumor recurrence model based on platinum-resistant colon cancer cells as a research tool for studying cancer cell dormancy
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, M. Igotti
          </td>
          <td>2025-10-08</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer treatment is an ongoing challenge, as directly targeting oncogenic drivers is often unfeasible in many patients due to the lack of druggable targets. This has led to the exploration of alternative strategies, such as exploiting synthetic lethality (SL) relationships between genes. SL facilitates the indirect targeting of oncogenic drivers, as exemplified by the clinical success of PARP inhibitors against BRCA-mutated tumors. Advances in high-throughput perturbation screens and multi-omics technologies have deepened our understanding of SL relationships, while computational models enhance SL predictions to better reflect biological complexity. However, while numerous experimental and computational methods have been developed to identify SL interactions, difficulties remain in translating these findings into clinical applications.This review combines recent progress on SL relationships in cancer with emerging insights into epigenetic regulation, highlighting how epigenetic drugs (epidrugs) can provide new opportunities for targeted interventions, offering a way to minimize off-target effects and enhance therapeutic precision. To advance SL-based therapies, efforts must focus not only on identifying new SL interactions but also on consolidating existing knowledge and integrating experimental and computational approaches to characterize the vulnerabilities of cancer cells. Strengthening this foundation will be critical for the effective development of SL-based cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a9988cdf76b342d040b8e183dfc1c5b3f9d7baa" target='_blank'>
              Epigenetic synthetic lethality as a cancer therapeutic strategy: synergy of experimental and computational approaches.
              </a>
            </td>
          <td>
            Maria Farina-Morillas, Laia Ollé-Monràs, Silvana Ce Maas, Isabel de Rojas-P, Miguel F. Segura, Jose A. Seoane
          </td>
          <td>2025-08-25</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4704796a9cf8557ad095d79824927f1f6b68e0" target='_blank'>
              Revealing genetic drivers of ovarian cancer and chemoresistance: insights from whole-genome CRISPR-knockout library screens.
              </a>
            </td>
          <td>
            Tali S. Skipper, Kristie-Ann Dickson, Christopher E. Denes, Matthew A. Waller, Tian Y. Du, G. Neely, N. Bowden, Alen Faiz, Deborah J. Marsh
          </td>
          <td>2025-08-28</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Aberrant pre-mRNA splicing is a hallmark of many cancers, yet the underlying genetic mechanisms driving these alterations remain incompletely understood. While splice-altering variants can explain a subset of events, many tumor-specific splicing changes arise in the absence of nearby sequence variants. This study aims to systematically dissect the contribution of proximal splice-altering variants to tumor-specific splicing in pediatric brain tumors and to define the residual cohort likely driven by regulatory mechanisms such as mutations affecting splicing factors or the spliceosome machinery. We hypothesized that some fraction of tumor-specific splice events in pediatric brain tumors are not associated with nearby genetic variants, and that these may highlight broader disruptions to splicing regulation.



 We applied replicate Multivariate Analysis of Transcript Splicing (rMATS-turbo) to stranded total RNA-seq data (N=635) from the Pediatric Brain Tumor Atlas (PBTA). Tumor-specific single exon (SE) splicing events were defined as a change in percent spliced in (| ΔPSI |) > 0.3 between tumors and healthy brain samples sourced from GTEx (<40 years, n=2,642) and pediatric normals (n=7). Tumor-normal matched whole genome sequencing data were used to identify somatic and rare (<0.1% Allele Frequency in gnomad) germline SNVs/Indels proximal (<250nt) to rMATS-defined splice junctions.



 Across the cohort (N=635), we identified 107,827 tumor-specific SE alternative splicing events. Only 44 tumors harbored a variant proximal to the affected splice site (11 germline, 39 somatic), leaving most events without a clear cause. We are now investigating this variant-negative subgroup as a preliminary cohort. Current efforts focus on excluding structural drivers before turning to potential regulatory mechanisms, including the disruption of spliceosome components. Preliminary findings suggest some tumors show altered expression or mutations in spliceosome components, though further validation is needed.



 This dual-ended analysis of tumor-specific splicing events reveals that while a subset can be attributed to local genetic variation, the majority likely arise from broader disruptions in splicing regulation. These findings underscore the importance of integrating RNA- and DNA-based analyses to fully capture the molecular basis of splicing dysregulation in cancer. Moreover, the variant-negative cohort may serve as a rich resource to uncover novel regulatory mechanisms and potential therapeutic targets in splicing-driven pediatric brain tumors.



 Patricia J Sullivan, Ryan J Corbett, Ammar S Naqvi, Alexander Sickler, Rebecca Kaufman, Bo Zhang, Chuwei Zhong, Sharon J Diskin, Jo Lynne Rokita. Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0c6326494c3f8ecc5b90e1fc51e9fc24433d6c" target='_blank'>
              Abstract B031: Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors
              </a>
            </td>
          <td>
            Patricia J Sullivan, Ryan J. Corbett, Ammar S. Naqvi, Alexander Sickler, Rebecca S. Kaufman, Bo Zhang, Chuwei Zhong, S. Diskin, J. L. Rokita
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="ABSTRACT Recent bulk analysis of Chinese hamster ovary (CHO) cell mitochondrial DNA revealed widespread heteroplasmy across cell lines and even within clones of the same parental host. To address this, we applied our previously developed single‐cell mtDNA sequencing (scmtDNAseq) method to 84 single CHO cells. We identified widespread intercellular heteroplasmy across the CHO cell population and predicted possible phenotypic impacts. 3/11 (27%) of the most variable mutations were only identified by scmtDNAseq, indicating greater resolution when compared to bulk cell analysis. Single‐cell RNAseq (scRNAseq) was also performed at the same time point and, compared to scmtDNAseq, significant differences in intercellular heteroplasmy were observed. Using an inducible mAb expression system demonstrated that short‐term additional biosynthetic burden of exogenous protein production had little impact on intercellular heteroplasmy. We additionally monitored bulk heteroplasmy over 38 days, reflecting the typical timespan from vial thaw to production vessel in a Biopharmaceutical upstream cell culture process. We observed minimal change in heteroplasmy, finding no evidence that a mAb‐producing CHO cell line develops impactful changes in heteroplasmy over that timeframe. This would suggest that for our cell line, the heteroplasmy profile established on Day 1 should be maintained throughout a full fed‐batch bioprocess run.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d8c2d19d7057b8a8f3648254122a541737446d" target='_blank'>
              Single‐Cell Mitochondrial DNA Analysis of Recombinant Chinese Hamster Ovary Cells Reveals Widespread Heteroplasmy
              </a>
            </td>
          <td>
            Alan Foley, N. Lao, Ciara McGuirk, Colin Clarke, Niall Barron
          </td>
          <td>2025-09-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Meiotic crossovers promote correct chromosome segregation and the shuffling of genetic diversity. However, the measurement of crossovers remains challenging, impeding our ability to decipher the molecular mechanisms that are necessary for their formation and regulation. Here we demonstrate a novel repurposing of the single-nucleus Assay for Transposase Accessible Chromatin with sequencing (snATAC-seq) as a simple and high-throughput method to identify and characterize meiotic crossovers from haploid testis nuclei. We first validate the feasibility of obtaining genome-wide coverage from snATAC-seq by using ATAC-seq on bulk haploid mouse testis nuclei, ensuring adequate variant detection for haplotyping. Subsequently, we adapt droplet-based snATAC-seq for crossover detection, revealing >25 000 crossovers in F1 hybrid mice. Comparison between the wild type and a hyper-recombinogenic Fancm-deficient mutant mouse model confirmed an increase in crossover rates in this genotype, however with a distribution which was unchanged. We also find that regions with the highest rate of crossover formation are enriched for PRDM9. Our findings demonstrate the utility of snATAC-seq as a robust and scalable tool for high-throughput crossover detection, offering insights into meiotic crossover dynamics and elucidating the underlying molecular mechanisms. It is possible that the research presented here with snATAC-seq of haploid post-meiotic nuclei could be extended into fertility-related diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b651b832a12539ba036217c6dc855d9399ad822" target='_blank'>
              A high-resolution meiotic crossover map from single-nucleus ATAC-seq reveals insights into the recombination landscape in mammals
              </a>
            </td>
          <td>
            S. Novakovic, Caitlin Harris, Ruijie Liu, Davis J. McCarthy, W. Crismani
          </td>
          <td>2025-09-01</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e3f8f8be28642c567355c40810567d5184f908" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Kathrin Wellach, Kiana Samimi, Ariel Galindo-Albarrán, Pierré-René Körner, Jasmine Montenegro Navarro, Andrés Elía, Fiona Megan Tilghman, Hanan Sakeer, M. Mendoza-Parra, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The persistent residual tumor cells that survive after chemotherapy are a major cause of treatment failure, but their survival mechanisms remain largely elusive. These cancer cells are typically characterized by a quiescent state with suppressed activity of MYC and MTOR. We observed that the MYC-suppressed persistent triple-negative breast cancer (TNBC) cells are metabolically flexible and can upregulate mitochondrial oxidative phosphorylation (OXPHOS) genes and respiratory function (“OXPHOS-high” cell state) in response to DNA-damaging anthracyclines such as doxorubicin, but not to taxanes. The elevated biomass and respiratory function of mitochondria in OXPHOS-high persistent cancer cells were associated with mitochondrial elongation and remodeling suggestive of increased mitochondrial fusion. A genome-wide CRISPR editing screen in doxorubicin-persistent OXPHOS-high TNBC cells revealed BCL-XL gene as the top survival dependency in these quiescent tumor cells, but not in their untreated proliferating counterparts. Quiescent OXPHOS-high TNBC cells were highly sensitive to BCL-XL inhibitors, but not to inhibitors of BCL2 and MCL1. Interestingly, inhibition of BCL-XL in doxorubicin-persistent OXPHOS-high TNBC cells rapidly abrogated mitochondrial elongation and respiratory function, followed by caspase 3/7 activation and cell death. The platelet-sparing proteolysis targeted chimera (PROTAC) BCL-XL degrader DT2216 enhanced the efficacy of doxorubicin against TNBC xenografts in vivo without induction of thrombocytopenia that is often observed with the first-generation BCL-XL inhibitors, supporting the development of this combinatorial treatment strategy for eliminating dormant tumor cells that persist after treatment with anthracycline-based chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb7ad9e0cb28b546f19e574e42e36713a6ee110" target='_blank'>
              Quiescent OXPHOS-high triple-negative breast cancer cells that persist after chemotherapy depend on BCL-XL for survival
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Marie Winter, Leandro Encarnacao Garcia, O. Akinrinmade, Francisco Madeira Marques, Emmanouil Zacharioudakis, Anna Skwarska, J. Aguirre-Ghiso, Marina Konopleva, Guangrong Zheng, Susan Fineberg, Daohong Zhou, E. Gavathiotis, Tao Wang, Eugen Dhimolea
          </td>
          <td>2025-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557ee2fb83678fffa15e9db9ad44ba30b15ed208" target='_blank'>
              Systematic decoding of functional enhancer connectomes and risk variants in human glioma.
              </a>
            </td>
          <td>
            Jinfang Bi, Weipeng Mo, Man Liu, Yingjie Song, Qian Xiao, Sibo Fan, Wenbin Wang, Tengfei Shi, Yaoqiang Zheng, Jie Lian, Rong Liu, Bohan Chen, Xiaofeng Huang, Peng Li, Zhongfang Zhao, Jiandang Shi, Lei Zhang, Guangsong Su, Nu Zhang, Wange Lu
          </td>
          <td>2025-08-19</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Non-homologous end joining (NHEJ) is a DNA repair pathway that directly ligates broken DNA ends without the need for a homologous template, whereas translesion synthesis (TLS) is a DNA damage tolerance mechanism in which specialized DNA polymerases bypass lesions on the template strand. Although both pathways play critical roles in maintaining genome integrity across organisms, they inherently introduce mutations. Here, we investigate how these two pathways contribute to spontaneous and genotoxic stress–induced genomic alterations in the yeast Yarrowia lipolytica. A NHEJ-deficient mutant (ku70) and three TLS-deficient mutants (rev1, rev3, and rad30) are subjected to mutation accumulation experiments, followed by whole-genome sequencing. Our results show that the deletion of KU70 has no significant effect on the rates of spontaneous single-nucleotide variations (SNVs), small insertions and deletions, or chromosomal rearrangements, while the deletion of REV1 and REV3 leads to significant reductions in spontaneous SNV rates. These findings indicate that TLS but not the NHEJ pathway is a major contributor to spontaneous mutagenesis in Y. lipolytica. Moreover, exposure to 0.02% methyl methanesulfonate and 80 J/m2 ultraviolet (UV) radiation resulted in 48- and 107-fold increases in SNV rates, respectively. These induced SNVs were largely dependent on DNA polymerases Rev1 and ζ, further underscoring their central roles in genotoxic stress–induced mutagenesis. We observe that DNA polymerase η can suppress C to T and C to A substitutions while promoting T to C mutations, exhibiting a dual function in regulating mutagenesis under UV treatment. Phenotypic evolution experiments reveal that TLS activity enhances the adaptive potential of Y. lipolytica under oxidative stress, underlying its broader impact on environmental fitness. Together, these findings provide new insights into the distinct roles of the NHEJ and TLS pathways in preserving genome integrity in Y. lipolytica. Key points • The NHEJ pathway has a limited role in spontaneous genomic alterations in Y. lipolytica. • DNA polymerases Rev1 and ζ contribute to most UV- and MMS-induced mutations. • The dual roles of Pol η in UV-induced mutations were revealed. • NHEJ and TLS pathways are crucial to phenotypic evolution of Y. lipolytica. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13575-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d93f494460655f7859b1e3dd9cb7f63f2e1036" target='_blank'>
              The roles of NHEJ and TLS pathways in genomic alterations and phenotypic evolution in the yeast Yarrowia lipolytica
              </a>
            </td>
          <td>
            Cen Yan, Ye-Ke Wang, Yuan-Ru Xiong, Xin-Qiu Zhou, Yuan-Chun Fang, Ruo-Tian Nie, Cunqi Ye, Ke Zhang, Dao-Qiong Zheng
          </td>
          <td>2025-08-15</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Targeting ribosome biogenesis and protein translation has emerged as a promising avenue for cancer therapy. The fat mass and obesity-associated protein (FTO), an RNA N6-methyladenosine (m6A) eraser, has been identified as an oncogenic factor in acute myeloid leukemia (AML). Here, we present the development of an FTO degrader that selectively degrades FTO in AML cells, demonstrating superior efficacy both in vitro and in vivo. We confirmed that FTO degradation increases m6A modifications on mRNAs associated with ribosome biogenesis, promoting their YTHDF2-mediated decay. This disruption of ribosome biogenesis and protein translation contributes to the inhibition of AML progression. Our findings highlight this FTO degrader as a valuable tool compound for elucidating the functional roles of FTO in cancer and as a potential foundation for the development of selective anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f811f68d8823c012bb6c09d8710afba622302ef" target='_blank'>
              FTO degrader impairs ribosome biogenesis and protein translation in acute myeloid leukemia
              </a>
            </td>
          <td>
            Wenlong Li, Yutao Zhao, Dong Wu, Zhenhua Chen, Y. Qing, Fan Yang, Fei Ji, Linda Zhang, Lillian Sau, Jianjun Chen, Chuan He
          </td>
          <td>2025-08-15</td>
          <td>Science Advances</td>
          <td>2</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase.
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immunotherapy has emerged as a promising treatment option for triple-negative breast cancer (TNBC); however, the pronounced heterogeneity of the tumor immune microenvironment significantly hinders the prediction of therapeutic efficacy, with effective biomarkers also remaining limited. This study utilizes single-cell RNA sequencing (scRNA-seq) and transcriptome analysis to dissect the immune landscapes of TNBC, emphasizing CD8 effector T cells (CD8Teff) as potential predictors of immune checkpoint inhibitors (ICIs) treatment response. Immune profiling revealed that CD8Teff cells were predominantly enriched in "hot" tumors strongly correlating with improved progression-free and overall survival. Notably, the cytokine C-X-C Motif Chemokine Ligand 13 (CXCL13) emerged as a key regulator, with its high expression marking an immune-active tumor microenvironment favorable to ICI efficacy. Comprehensive analysis uncovered critical interactions between CD8Teff cells and other immune and stromal components, supporting the formation of an immunologically active tumor microenvironment (TME) conducive to tumor control. Additionally, metabolic reprogramming of CD8Teff cells in responsive tumors highlighted pathways that may influence ICIs efficacy. CD8Teff cells significantly influence the TME and predict ICI success in TNBC, with CXCL13 as a pivotal modulator. A pathology-based artificial intelligence model for CD8Teff recognition was developed, achieving an area under the curve (AUC) of 0.823 in the training cohort and 0.805 in the validation cohort. This study provides an in-depth characterization of the TNBC immune microenvironment, identifying CD8Teff functionality, CXCL13 signaling, and immune-metabolic pathways as critical determinants of immunotherapy success. These findings support the development of personalized treatment strategies based on immune profiling to enhance therapeutic efficacy in TNBC. Supplementary Information The online version contains supplementary material available at 10.1186/s43556-025-00306-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bfd680374201a021e9ef78818435e2a0634a514" target='_blank'>
              Single-cell RNA sequencing identifies CD8Teff cell activation as a predictive biomarker in triple-negative breast cancer immunotherapy
              </a>
            </td>
          <td>
            L. Mao, Zebang Zhang, Yongjian Chen, Qing Peng, Zhenjun Huang, Wenhao Ouyang, D. Zeng, W. Ren, Zifan He, Tang Li, Zehua Wang, Ruichong Lin, Jianli Zhao, Jiannan Wu, Herui Yao, Yunfang Yu
          </td>
          <td>2025-09-19</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) are an insignificant, however enormous population of tumor cells that display capacities of self-renewal, differentiation, and tumor initiation, consequently being the core feature in cancer progression, recurrence, and drug resistance. The phosphoinositide 3-kinase/protein kinase B (PI3K/AKT) is one of the most critical signalling cascades regulating CSCs and controlling their stemness, survival, evasion of the immune system under stressful conditions, as well as metabolic reprogramming. This review provides an overview of the morphological features and functional aspects of the PI3K/AKT immune cascade and pathway, and how it essentially connects with both upstream and downstream effectors in CSC biology. The cross-communication of PI3K/AKT with other pathways, e.g., Wnt, Notch, and Hedgehog, is elaborated to emphasize the redundancy of the networks facilitating CSC maintenance and drug resistance. Additionally, we provide an in-depth scrutiny of the processes through which PI3K/AKT signalling leads to CSC resistance to chemotherapy, radiotherapy, and targeted therapy, as well as their plasticity, metastasis, and immune escape mechanisms. Current and future therapeutic approaches targeting the PI3K/AKT axis, such as small molecule inhibitors, combination therapy, and drug delivery nanotechnology, are also discussed. Finally, we present clinical issues and prospects for improving CSC-based therapy by using PI3K/AKT blockade to eliminate resistance and induce protracted, long-lasting cancer remission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f966dace17da846800f374af44f626bb75044a1c" target='_blank'>
              The Role of PI3K/AKT Signalling Pathway in Cancer Stem Cells: Emerging Therapeutic Targets and Resistance Mechanisms
              </a>
            </td>
          <td>
            Komal Arooj, Hassan Imam, Zarlish Attique, Zoha Naeem, Ali Ahmad, Hafiz Muhammad Faraz Azhar, Fariha Javaid, Zeenat Nawaz
          </td>
          <td>2025-09-30</td>
          <td>Futuristic Biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is the most common type of leukemia in Western populations and remains incurable despite significant therapeutic advancements. Over the past decade, the treatment landscape has evolved from traditional chemoimmunotherapy to targeted oral agents, including Bruton’s tyrosine kinase inhibitors (BTKis) and BCL2 inhibitors (BCL2is), which have demonstrated superior efficacy and tolerability, especially in elderly patients. Venetoclax, a BCL2i, induces apoptosis in CLL cells through selective inhibition of the anti-apoptotic BCL2 protein, while BTKis, such as ibrutinib and its next-generation analogs, disrupt B-cell receptor signaling critical to CLL cell survival. However, resistance to both drug classes has emerged, including mutations in BTK and BCL2, prompting the exploration of novel therapeutic strategies. This review outlines the molecular basis and clinical implications of these resistance mechanisms, as well as emerging therapeutic solutions, including non-covalent BTKis like pirtobrutinib and BTK-targeting PROTAC degraders such as BGB-16673 and NX-2127. Additionally, we discuss promising combination therapies incorporating BTKis, BCL2is, and anti-CD20 monoclonal antibodies. Finally, we highlight the growing role of measurable residual disease (MRD) as a biomarker to guide treatment duration and evaluate therapeutic success. As resistance mechanisms continue to emerge, tailoring therapy based on underlying biology will be critical to sustaining disease control and enhancing outcomes in patients with CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0956883756565c425ce66e36bee2fa5445ca9e94" target='_blank'>
              Resistance Mutations in CLL: Genetic Mechanisms Shaping the Future of Targeted Therapy
              </a>
            </td>
          <td>
            Samantha Sekeres, Erica N. Lamkin, Eduardo Bravo, Allison Cool, Justin Taylor
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The emerging field of epigenetics has recently unveiled a dynamic landscape in which gene expression is not determined solely by genetic sequences but also by intricate regulatory mechanisms. This review examines the interactions between these regulatory mechanisms, including DNA methylation and non-coding RNAs (ncRNAs), that orchestrate gene expression fine-tuning for cellular homeostasis and the pathogenesis of a multitude of diseases. We explore long non-coding RNAs (lncRNAs) such as telomeric repeat-containing RNA (TERRA) and Fendrr, highlighting their role in protein regulation to ensure proper gene activation or silencing. Additionally, we explain the therapeutic potential of brain-derived neurotrophic factor (BDNF)-related microRNA 132, which has shown promise in treating chronic illnesses by restoring BDNF levels. Finally, this review covers the role of DNA methyltransferases and ncRNAs in cancer, focusing on how lncRNAs contribute to X chromosome inactivation and interact with chromatin-modifying complexes and DNA methyltransferase inhibitors to reduce cancer cell aggressiveness. By amalgamating the wide array of research in this field, we aim to provide glimpses into the complex entangling of genetics and environment as they control gene expressions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7604094840dd2e1f73f9546221825fbdf25d6b74" target='_blank'>
              The Epigenetic Tapestry: A Review of DNA Methylation and Non-Coding RNA's Interplay with Genetic Threads, Weaving a Network Impacting Gene Expression and Disease Manifestations
              </a>
            </td>
          <td>
            Yu-Li He, Youshin Loh
          </td>
          <td>2025-10-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Environmental exposures during early life are increasingly recognized as key determinants of health and disease in adulthood (1) but how they durably shape disease risk across the lifecourse remains poorly understood. Here, we show early-life toxicants reprogram the epigenome and redirect age-associated transcriptional trajectories—polarizing cell-specific gene expression and predisposing to liver disease. The TaRGET II Consortium (2) exposed mice to diverse toxicants from pre-conception through weaning and followed individual animals though adulthood with multi-omic profiling. Analysis of >800 liver epigenomic and transcriptomic profiles from male and female mice revealed that despite differing chemical classes and mechanisms of action, multiple toxicants — BPA, TBT, TCDD, and PM2.5 — produced exposure signatures that converged on genes normally differentially expressed in the liver as animals aged. Histone modifications at enhancers emerged as key targets for epigenomic reprogramming of these liver aging-associated plasticity genes (LAAsP genes). Reprogrammed LAAsP genes exhibited a striking, bidirectional signature. In hepatocytes LAAsP genes that typically increase with age, such as those involved in metabolism, were repressed. Conversely, in non-parenchymal cells, LAAsP genes that normally decline with age, including those for extracellular matrix production, remained elevated. An attenuated LAAsP gene signature and polarized transcriptional states were mirrored in human liver disease and hepatocellular carcinoma and could effectively distinguished healthy from diseased human liver transcriptomes. Together, these findings demonstrate that early-life environmental exposures can hijack the plasticity of epigenomic aging, durably reprogram expression trajectories, and lock in polarized states that foreshadow chronic liver disease and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e50202ac8b33f8ab48ea3ab4ef30e279c45916f2" target='_blank'>
              Plasticity of Epigenomic and Transcriptomic Aging Reveals Common Targets for Reprogramming by Environmental Exposures
              </a>
            </td>
          <td>
            Cheryl Walker, Sandra L. Grimm, Rahul K. Jangid, Marisa S. Bartolomei, Dana C. Dolinoy, D. Aylor, Gokhan Mutlu, Shyam Biswal, Bo A Zhang, Robert Hamanaka, Justin A. Colacino, Maureen Sartor, Laurie Svoboda, Ting Wang, Cristian Coarfa
          </td>
          <td>2025-09-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background/Objectives: DNA-damaging agents can contribute to genetic instability, and such agents are often used in cancer chemotherapeutic regimens due to their cytotoxicity. Thus, understanding the mechanisms involved in DNA damage processing can not only enhance our knowledge of basic DNA repair mechanisms but may also be used to develop improved chemotherapeutic strategies to treat cancer. The high-mobility group box protein 1 (HMGB1) is a known nucleotide excision repair (NER) cofactor, and its family member HMGB3 has been implicated in chemoresistance in ovarian cancer. Here, we aim to understand the potential role(s) of HMGB3 in processing DNA damage. Methods: A potential role in NER was investigated using HMGB3 knockout human cell lines in response to UV damage. Subsequently, potential roles in DNA interstrand crosslink (ICL) and DNA double-strand break (DSB) repair were investigated using mutagenesis assays, metaphase spreads, foci formation, a variety of DNA repair assays, and TagSeq analyses in human cells. Results: Interestingly, unlike HMGB1, HMGB3 does not appear to play a role in NER. We found evidence to suggest that HMGB3 is involved in the processing of both DSBs and ICLs in human cells. Conclusions: These novel results elucidate a role for HMGB3 in DNA damage repair and, surprisingly, also indicate a distinct role of HMGB3 in DNA damage repair from that of HMGB1. These findings advance our understanding of the role of HMGB3 in chemotherapeutic drug resistance and as a target for new chemotherapeutic strategies in the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1261da2765ba41543d61e2f0da2291c8e870115b" target='_blank'>
              High-Mobility Group Box Protein 3 (HMGB3) Facilitates DNA Interstrand Crosslink Processing and Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Jillian Dangerfield, Anirban Mukherjee, W. Reh, Anna M. Battenhouse, Karen M. Vasquez
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Ageing is a general, intrinsic and progressively deleterious process that affects all cells, tissues and organs albeit at different extent and rate in each individual. The complexity and universality of its phenotypic manifestations suggest a multifactorial origin. The autosomal recessive disorder Werner syndrome likely represents a segmental progeroid disorder since patients show several, but not all phenotypes of premature ageing.


SUMMARY
Proliferative senescence of diploid cells in culture provided a model system in which ageing can be studied experimentally. Cultures of cells from patients with Werner syndrome experienced an extreme form of proliferative senescence and a clonal succession of translocations, known as variegated translocation mosaicism. In addition, Werner syndrome cells showed spontaneous deletion formation and a prolongation of and arrest in the S phase of the cell cycle. The WRN protein harbors a helicase, an exonuclease and a RecQ interaction domain. With the latter the WRN protein may interact with NBS1, RPA, MRE11, TREX1, MUTYH, POT1, TRF1, FEN-1, PAPRP-1, p97/VCP, TRF2, DNA polymerase(beta), Ku76/80, EXO-1, NEIL1, and p53, which are key to DNA damage response pathways including canonical NHEJ, homologous recombination, base excision repair and telomere maintenance. The WRN exonuclease domain is a target of WRNIP1 binding, which links WRN to resolution of stalled replication due to collision with transcription and the ATM-mediated cell cycle checkpoint. . Patients with an incomplete complement of Werner syndrome phenotypes, called atypical Werner syndrome patients, were found to carry variants in LMNA, POLD1, SPRTN, MDM2, CTC1, SAMHD1.


KEY MESSAGES
These findings broaden the genotypic landscape and the phenotypic spectrum of Werner syndrome. In this review potential molecular mechanisms underlying genomic instability in Werner syndrome, including chromothripsis due to asynchronous S phase traverse and telomere crises followed by bridge fusion breakage cycles are discussed. The participation of WRN in multiple gene networks is consistent with the multifactorial nature of ageing in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c57a230c575d7a7f034afedf23d1a5bec7bbac" target='_blank'>
              Molecular mechanisms of proliferative senescence and genomic instability in Werner syndrome and the WRN gene network.
              </a>
            </td>
          <td>
            Martin Poot
          </td>
          <td>2025-09-19</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cab8ffb00b0f66c1500dfb2ff83a6f9807f5f0" target='_blank'>
              A Mammalian Genomic Signature Shaped by Single Nucleotide Variants Controlling Transcriptome Integrity and Diversity.
              </a>
            </td>
          <td>
            Jian Yang, Samuel Ogunsola, Jason Wong, Aydan Wang, R. Joehanes, Daniel Levy, Shalini Sharma, Chunyu Liu, Jiuyong Xie
          </td>
          <td>2025-09-11</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The combination of 5-fluorouracil and irinotecan (FOLFIRI) remains a standard-of-care treatment for metastatic colorectal cancer (mCRC), yet benefits only about half of patients. Using patient-derived xenografts (PDXs), we investigated the biological underpinnings of this heterogeneous response. FOLFIRI-resistant models showed transcriptional upregulation of innate immunity and mitochondrial metabolism genes, together with reduced expression of the DNA polymerase POLD1. Sensitive counterparts exhibited a BRCAness-like phenotype with genomic scars of homologous recombination (HR) deficiency, not caused by genetic or epigenetic loss of HR genes but by low abundance of the RAD51 recombinase. In tumoroids, forced RAD51 overexpression attenuated HR deficiency-related scars and chemotherapy-induced damage, while HR inhibition through ATM blockade enhanced drug sensitivity. The predictive relevance of key response determinants was validated in clinical samples. This work illuminates functional, non-genetic facets of BRCAness in mCRC and introduces actionable biomarkers and targets, offering prospects to improve clinical decision-making and broaden therapeutic options for chemorefractory patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb783603c147c9c964fbfcc1a8e05ff7e8e4445" target='_blank'>
              The molecular and functional landscape of resistance to FOLFIRI chemotherapy in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marco Avolio, S. Leto, F. Sassi, B. Lupo, Elena Grassi, Irene Catalano, E. Zanella, V. Vurchio, F. Cottino, Petros K Tsantoulis, L. Lazzari, Paolo Luraghi, Martina Ferri, Francesco Galimi, E. Berrino, S. Bellomo, Marco Viviani, A. Sogari, G. Mauri, F. Tosi, F. Cruciani, A. Sartore-Bianchi, S. Siena, Felice Borghi, V. Torri, E. Élez, J. Tabernero, Maria Nieva, C. Montagut, N. Tarazona, Andrés Cervantes, Sabine Tejpar, A. Bardelli, C. Marchiò, S. Marsoni, Andrea Bertotti, L. Trusolino
          </td>
          <td>2025-09-22</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="


 Tumor clonal evolution represents a dynamic ecosystem underpinned by genetic alterations and Darwinian selection, posing major challenges due to intratumoral heterogeneity and therapy resistance. Although computational methods have advanced significantly, current tools often focus on single data modalities, leaving important gaps in modeling spatial and non-genetic evolution. This review systematically surveys and assesses algorithmic progress across diverse study designs to identify key limitations and future directions.



 We systematically evaluate over 20 computational tools across four study designs—cross-sectional, regional bulk, single-cell, and lineage tracing—and perform benchmarking of seven comparable tools. Multi-omics integration approaches are shown to improve phylogenetic inference, yet challenges remain in mutation ordering and polyclonal detection. A novel spatiotemporal framework is proposed to link phylogenetic branch lengths with spatial transcriptomic gradients. Future efforts should prioritize multi-modal data integration, scalable computational architectures, and clinically applicable models to translate evolutionary insights into precision oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d2b14817118e93c14f870881445bd16f81118f7" target='_blank'>
              Computational Strategies in Tumor Phylogenetics: Evaluating Multi-Modal Integration and Methodological Trade-Offs Across Study Designs
              </a>
            </td>
          <td>
            Chenghan Jiang, Zhe Wang, Ruoyu Wang, Shanshan Liang, Shuai Tao
          </td>
          <td>2025-10-01</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296f18d57dd6ad5d3ff3f75155cdc24fcd7506bf" target='_blank'>
              Cross-species analysis identifies genotype-driven vulnerabilities in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Angeliki Karatza, Maayan Pour, Jaylen M Powell, Yuan Hao, Dimitris Nasias, Yi Jer Tan, Alfonso Lopez, M. Ciampricotti, Chengwei Peng, Álvaro Quintanal Villalonga, Yeuan Ting Lee, Ruveyda Ayasun, Muhammad Elnaggar, Priyanka Sahu, Igor Dolgalev, Hai Hu, Luis Chiriboga, Cynthia Loomis, Daniel Andrussier, Magdalena A Ploszaj, A. Tsirigos, Peter Meyn, Fred P. Davis, Nicolas Stransky, Gromoslaw A. Smolen, John T. Poirier, Jonathan So, Mark Philips, Harvey I. Pass, Dafna Bar Sagi, Itai Yanai, Kwok-Kin Wong
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="DNA-binding molecules regulate gene expression, replication, repair, and transcription, making their research crucial to molecular biology. This review investigated their architectures, processes, and impacts on cancer research and therapy to determine their biological and therapeutic potential. A thorough examination of published data on DNA-binding agents, including intercalators, groove binders, and metal complexes, focused on their chemical properties, biological activity, and therapeutic significance. Doxorubicin intercalated between base pairs to inhibit replication and transcription, whereas cisplatin produced covalent cross-links with guanine bases to cause tumor cell death. Multiple derivatives of metal complexes reduced tumor development by over 70% in leukemia models as DNA probes and therapeutics. Also reviewed, Dps proteins in *Escherichia coli* showed that their non-specific DNA binding offered up to 65% oxidative stress resistance compared to control cultures, validating DNA-protein protection as a survival strategy. In Phase II studies, amsacrine caused substantial remission in acute leukemia patients, whereas doxorubicin was more effective across many cancer types but had greater cardiotoxicity concerns. Selectivity, toxicity, and resistance limit DNA-targeting medicines, although they are successful. Chemical modifications like hydrophobic tailoring and sequence-specific binding have enhanced binding affinity and therapeutic index, yet only 10% of candidate compounds get clinical approval. Recent studies show that AI-driven design has expedited screening, lowering development costs by 30% and durations by 3–5 years. These data suggest that DNA-binding medicines have great potential in cancer, but safer, more selective, and resistance-free therapy are still needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfaa0d838d07a9aad5a057a0a95be735af4700a" target='_blank'>
              Chemistry of DNA-binding Molecules
              </a>
            </td>
          <td>
            Evon Akram, Dina A. Najeeb, Asmaa A. Jawad, N. Bedair, A. M. Hussein, Saba R. Jaafar, Ruaa H. Ali, R. F. Shaher, M. A. Hussein, Daniah M. Hamid, Reem H. Al-Tabra, Alyaa K. Abood, Salam Mohammed
          </td>
          <td>2025-08-30</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 TP53, often referred to as the guardian of the genome, is one of the most frequently mutated genes in human cancers. High grade serous ovarian cancer (HGSOC) has a TP53 mutation rate of up to 96%. Given its multi-faceted role in human cancers, there have been many efforts to therapeutically target mutant p53. APR-246 is a small molecule drug that is proposed to bind mutant p53 and promote its thermodynamic stabilization, allowing the tumor suppressor to regain wild-type-like conformation and functionality. However, the mechanism of this drug has remained a contentious topic, as it has been shown to act via various p53-independent mechanisms, including modulation of the redox system. To address this, we have performed a systematic functional genetic screening for the identification of APR-246 rescuable p53 mutants. Our approach leverages a pooled TP53 mutagenesis library containing ∼8,000 amino acid variants, spanning the entirety of the TP53 coding sequence. Cell based screenings against sub-lethal doses of APR-246 in two ovarian cancer cell lines, SKOV3 and OVCAR5, has revealed that sensitivity to this drug is cell-line dependent rather than mutant p53-dependent. Our results suggest that p53-independent factors such as intrinsic resistance to ferroptosis may play a more significant role in response to APR-246. We propose that resistance to ferroptosis provides cellular conditions which enable identification of p53 mutants that are sensitive to APR-246. This is evidenced by our results which show that genetic screen in OVCAR5, which is more intrinsically resistant to ferroptosis, uncovers a greater number of potentially sensitive mutants, including R175H, a mutant largely recognized as rescuable by APR-246. Further, we have identified GPX4, an anti-ferroptosis gene, as a key determinant of APR-246 sensitivity. Indeed, we demonstrate synergy between APR-246 and GPX4 inhibitor, ML-210. These studies support the hypothesis that APR-246 efficacy is largely attributable to p53-independent mechanisms and that attenuation of these off-target effect may reveal potentially sensitive p53 mutants.



 Anais Saunders, Caili Tong, Anthony N. Karnezis, Gary S. Leiserowitz, Jeremy R. Chien. Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7beb6941bcf2a1d1e3d25e6917a37bc8794a91d" target='_blank'>
              Abstract A046: Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects
              </a>
            </td>
          <td>
            Anais Saunders, Caili Tong, A. Karnezis, G. Leiserowitz, Jeremy Chien
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Ultra-accurate detection of rare somatic mutations is critical for understanding mutational processes in human disease, aging, and environmental exposures, yet current methods are limited by error rates, restricted genome coverage, and high DNA input. We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA. Benchmarking in human sperm estimates a UDSeq error rate of ~2.5×10−9 per base pair. UDSeq captures mutational signatures from heterogeneous populations without clonal expansion, reproduces exposure-specific patterns in cell lines and rodent models, and enables cross-species profiling. Compared with prior duplex methods, UDSeq yields up to fourfold more usable duplex molecules, improves library conversion, and remains cost-effective. We include a step-by-step protocol with quality-control checkpoints for fragment size, ligation yield, library conversion, and duplication rate. UDSeq provides a scalable, low-input platform for accurate profiling of somatic mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25aa13b255931a4c667fbbd3b80aa9966be011" target='_blank'>
              A Universal Duplex Sequencing Approach for Accurate Detection of Somatic Mutations
              </a>
            </td>
          <td>
            Shuvro P. Nandi, Yuhe Cheng, Shams Al-azzam, Safa Saeed, Audrey Kristin, Nadia Sunico, Isabella Stuewe, Zichen Jiang, Luka Culibrk, Maria Zhivagui, Xiaoxu Yang, Rachel Wise, Foster C. Jacobs, Bérénice Chavanel, M. Korenjak, Mia Petljak, S. Balbo, Laurie G Hudson, Ke Jian Liu, Jiri Zavadil, Joseph G Gleeson, Ludmil B. Alexandrov
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Background Mitotic catastrophe (MC) is a well-recognized endogenous mechanism of tumor cell death, characterized as a delayed cell death process associated with aberrant mitosis. However, its prognostic significance in the context of intratumoral heterogeneity in esophageal squamous cell carcinoma (ESCC) remains largely unexplored. Methods We performed an in-depth analysis of single-cell RNA sequencing (scRNA-seq) data from ESCC obtained from the Gene Expression Omnibus (GEO) database. MC scores for individual cells were calculated using the AddModuleScore function, and T cell specific gene modules were identified via the high-dimensional weighted gene co-expression network analysis (hdWGCNA) framework. To further elucidate the developmental trajectories and intercellular interactions of T cells, pseudotime analysis and cell-cell communication inference were conducted. A prognostic risk model was then constructed using three machine learning algorithms combined with multivariate Cox regression analysis. Following risk stratification, we performed immune infiltration profiling, drug sensitivity analysis, and molecular docking to comprehensively assess the functional implications of the risk model in ESCC. Based on preliminary results from quantitative Real-time PCR (qRT-PCR) and Western blotting (WB), we selected the hub gene SLF2 for functional validation using wound healing, Cell Counting Kit-8 (CCK-8) assay, Transwell, and colony formation assays. Results Based on T cell mitotic catastrophe associated genes (MCAGs) and utilizing machine learning algorithms, we established a robust prognostic risk model for ESCC. The model demonstrated excellent stratification capability in predicting patient outcomes and effectively revealed the heterogeneity of the tumor immune microenvironment (TIME) and drug sensitivity. Furthermore, functional experiments confirmed that knockdown of the hub gene SLF2 significantly inhibited the migration, invasion, and proliferation of ESCC cells. Conclusion The prognostic model based on MCAGs we developed serves as an effective tool for predicting outcomes in ESCC.T cell-specific MCAGs drive intratumoral heterogeneity in ESCC, serving as potential prognostic biomarkers and therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00815-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4482ff5484bb99ce98963d648871eb4d8f4fb2bb" target='_blank'>
              Integrative single-cell and bulk transcriptomic analysis reveals the landscape of T cell mitotic catastrophe associated genes in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Shuang Li, Zheng Tao, Nan Wang, Yazhou Liu, Kai Xie, Haitao Ma
          </td>
          <td>2025-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Osteosarcoma is the most common primary malignant tumor of bone and occurs predominantly in children and adolescents during periods of accelerated skeletal growth. Within the tumor microenvironment (TME), vascular smooth muscle cells (VSMCs) may acquire osteogenic characteristics. Nuclear factor of activated T cells c1 (NFATc1) is central to skeletal biology, yet its behavior in VSMCs in osteosarcoma remains insufficiently defined. Methods We profiled fresh osteosarcoma tissues by single-cell RNA sequencing on the 10x Genomics Chromium platform. Mechanical and enzymatic dissociation yielded single-cell suspensions; viable cells (> 85%) were enriched; libraries were prepared with Single Cell 3′ v3 reagents and sequenced on NovaSeq 6000. Reads were processed with Cell Ranger (GRCh38). Following log-normalization, 2,000 highly variable genes were retained. Harmony integration reduced batch structure. We computed 50 principal components, generated UMAP/t-SNE embeddings, and applied k-nearest neighbor/Louvain clustering (k = 20; resolution = 0.5). Cell types were assigned from differentially expressed markers cross-checked with CellMarker and PanglaoDB. NFATc1 distribution and co-expression patterns were assessed by Pearson correlation. Results Nine major cell classes were identified, including VSMCs, fibroblasts, macrophages, T/NK subsets, B cells, neutrophils, mast cells, and inflammatory myeloid cells. NFATc1 expression was most prominent in VSMCs, with a bimodal distribution within this lineage. In VSMCs, NFATc1 co-varied with SERGEF, APOE, and CCS, pointing to coordinated cytoskeletal remodeling, lipid handling, and copper homeostasis during osteogenic reprogramming. Conclusions At single-cell resolution, NFATc1 marks a VSMC fraction with osteogenic features in osteosarcoma. The associated network suggests testable links to metabolism and matrix remodeling; functional perturbation will be required to establish causality. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03606-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3638bf4253781f5bd9cf49aea68da6335438a6" target='_blank'>
              Single-cell RNA sequencing reveals NFATc1-mediated regulatory mechanisms of vascular smooth muscle cell osteogenic transformation in the osteosarcoma microenvironment
              </a>
            </td>
          <td>
            Xuejiao Yi, Shilin Fu, Chunliang Zhang, Tianle Zhao, Huan Feng, Dehui Qian
          </td>
          <td>2025-09-29</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer, a major health concern worldwide, involves diverse molecular subtypes and complex gene expression patterns. This study conducted a comprehensive bioinformatics analysis of breast cancer, analyzing 10 gene expression datasets from the Gene Expression Omnibus archive to find common genes that exhibit differential expression (DEGs). Then, we conducted pan-cancer and functional enrichment analyses, including single-cell level investigations of DEGs. To identify potential hub genes, we built protein–protein interaction networks, which were further analyzed for methylation patterns and expression in various cancer stages. The study also explored interactions between these hub genes with microRNAs and transcription factors, along with their correlation with cytokine expression and infiltration of immune cells in the tumor microenvironment. The TCGA BRCA dataset was used for hub gene survival analysis. Key findings include the identification of 36 shared DEGs, with distinct expression patterns in principal component analysis, suggesting their potential as molecular signatures for diagnosis and prognosis. Investigation at the single-cell level revealed upregulated DEGs mainly expressed by tumor-associated fibroblast cells. Gene ontology unveiled upregulated genes mainly involved in the extracellular matrix, but most of the downregulated genes are tumor suppressors. The protein–protein interaction network analysis highlighted 7 hub genes: CAV1, FN1, BGN, CXCL12, SPTBN1, COL11A1, and INHBA. Methylation analysis revealed intricate epigenetic regulation of these genes. MicroRNA and transcription factor interaction analysis underscored the complex regulatory networks influencing gene expression. Cytokine profile analysis in tumor microenvironment and its correlation with hub gene expression provided insights into the immunological landscape of breast cancer. Survival analysis indicated that both upregulated and downregulated hub genes are linked with patient overall survival outcomes, although it is not statistically significant. This study provides a detailed view of the underlying molecular mechanisms in breast cancer, suggesting potential biomarkers and therapeutic targets. Its findings can contribute significantly to understanding the complexity of the breast cancer microenvironment, leading to the development of more sophisticated and targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdf4528f9aeae9acc9b474eba2a50c1c17c8b995" target='_blank'>
              Identification of potential hub genes and molecular mechanisms in breast cancer microenvironment: A comprehensive transcriptomics approach
              </a>
            </td>
          <td>
            MS Abdullah Al Noman, MS a Abdullah Al Saba, MS a Mohammad Sayem, PhD Tahirah Yasmin, PhD A. H. M. Nurun Nabi, A. N. Nabi
          </td>
          <td>2025-08-29</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cytotoxic chemotherapy remains the standard-of-care treatment for patients with pancreatic ductal adenocarcinoma (PDAC). However, chemotherapy only has modest effects at improving patient survival due to primary or rapidly acquired chemoresistance. The biological underpinnings of PDAC therapy resistance are incompletely defined, but the tumor microenvironment is known to be a major contributor to chemoresistance. We have found chemoresistance is imprinted on PDAC cells by the tumor microenvironment and persists for a period of days after PDAC cells are removed from tumors. However, PDAC chemoresistance is lost upon long term culture in standard laboratory conditions. Interestingly, culture of PDAC cells in Tumor Interstitial Fluid Medium (TIFM), a culture medium we developed to recapitulate the nutrient availability of the tumor microenvironment, maintains PDAC cells in a chemo- and targeted therapy resistant state even after long term culture ex vivo. These findings suggest that microenvironmental metabolic stress keeps PDAC cells in a physiologically relevant, therapy resistant cell state that standard culture models fail to maintain. Using TIFM culture, we sought to understand how PDAC cells in this state resist therapeutic challenge. We found that chemo- and targeted therapies largely retain on-target activity within TIFM medium but fail to activate cell death, enabling a “chemotolerant” cell state, which is also observed in PDAC tumors. This chemotolerant state is driven by suppression of apoptotic priming and can be overcome by targeting the anti-apoptotic regulator BCL-XL. Taken together, these findings suggest that reprogramming of cell death mechanisms by the PDAC nutrient microenvironment is a key contributor to therapy resistance in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bea129494e792fbbf0176ec1d7ce93ff6acb889" target='_blank'>
              Tumor nutrient stress gives rise to a drug tolerant cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Colin Sheehan, Lyndon Hu, Guillaume Cognet, Grace Croley, Thao Trang Nguyen, Anika Thomas-Toth, Darby Agovino, Patrick B. Jonker, Mumina Sadullozoda, Leah M. Ziolkowski, James K. Martin, Alica K Beutel, Ranya Dano, Mohammed A. Khan, Christopher J Halbrook, Kay F. Macleod, Christopher R. Weber, James L LaBelle, Alexander Muir
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Lymphoid neoplasms (LNs) are heterogeneous malignancies arising from lymphoid cells, displaying diverse clinical and molecular features. Although LNs are collectively frequent, individual subtypes are rare, posing challenges for genetic association studies. Indeed, genome-wide association studies (GWAS) explained only a fraction of the heritability. Shared genetic susceptibility and overlapping risk factors suggest a partially common etiology across subtypes. We employed a multi-trait GWAS strategy to improve discovery power by leveraging pleiotropy among LN subtypes. We defined LN phenoclusters based on cell of origin, somatic mutation profiles, and approved therapeutic agents. Using data from three large cohorts—the UK Biobank, Million Veteran Program, and FinnGen—we analyzed 31,937 LN cases and 1.2 million controls across 8 individual subtypes and 7 phenoclusters. We replicated the novel associations in two independent cohorts (All of Us and the Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial) with 2892 LN cases and 165,791 controls. We identified 76 genome-wide significant loci for individual subtypes or subtype clusters, including 20 novel associations. We identified the subtypes contributing to each locus, putative candidate causal variants, and genes underlying the associations, and found enrichment of specific cell types, biological processes, and drugs associated with LN risk genes. Overall, this study identified new LN genetic risk loci and candidate genes, providing insights that may inform novel therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8ac41083338a114514b280145d44117043358f" target='_blank'>
              Clustering of lymphoid neoplasms by cell of origin, somatic mutation and drug usage profiles: a multi-trait genome-wide association study
              </a>
            </td>
          <td>
            Murat Güler, F. Canzian
          </td>
          <td>2025-08-29</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648ee47786050552f00a0c31f0d21f6a61f09642" target='_blank'>
              LEF1 confers resistance to DNA-damaging chemotherapies through upregulation of PARP1 and NUMA1 in ovarian cancer.
              </a>
            </td>
          <td>
            Lin Huang, Mengna Zhu, Mengqing Chen, F. Ying, Qiulei Wu, Pan Liu, Wenhan Li, Guoqing Li, Yuewen Gao, Shuyan Yi, Wenwen Wang, Yiping Wen, Si Sun, Jing Cai, Man Xiao
          </td>
          <td>2025-09-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Despite occurring in an essential molecule, mutations in RNA polymerase readily emerge and elicit complex pleiotropic effects across different levels of biological organization, which are all modulated by environment. We investigated the impact of temperature on the effects of six mutations on sequence, structure, transcriptome, and organismal traits. We found temperature altered the transcriptomic response and key organismal traits such as growth rate and biofilm formation in a genotype-specific manner. Critically, mechanistic insights into the possible drivers of mutational effects emerged only when examining the relationships between different levels of organization: location of mutations in the tertiary structure and distance to key interacting molecules partly explained the observed transcriptomic differences, which in turn drove the impact of mutations on organismal traits. While falling short of capturing the full complexity of the system, our findings underscore the benefits of integrating insights across multiple biological levels to understand the relationship between environment and mutational effects in molecules with extensive pleiotropic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3646be5bb3c5019f620c6e2499c30f9214f99d4e" target='_blank'>
              Deciphering the Impact of Temperature on Pleiotropic Consequences of RNA Polymerase Mutations
              </a>
            </td>
          <td>
            D. Ozbilek, Jake K. Soley, Danna R. Gifford, Christopher G Knight, Simon C Lovell, M. Lagator
          </td>
          <td>2025-09-16</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The mutagenic translesion synthesis (TLS) pathway, which is critically dependent on REV1's ability to recruit inserter TLS polymerases and the POLζ extender polymerase, enables cancer cells to bypass DNA lesions while introducing mutations that likely contribute to the development of chemotherapy resistance and secondary malignancies. Targeting this pathway represents a promising therapeutic strategy. Here, we demonstrate that the expression of the C-terminal domain (CTD) of human REV1, a ca. 100 amino acid scaffold essential for TLS polymerase interactions, disrupts REV1/POLζ-dependent TLS in mammalian cells. Inducible expression of REV1-CTD in multiple human and murine cancer cell lines sensitizes cells to DNA-damaging agents such as cisplatin, benzo[a]pyrene diol epoxide, and methyl methanesulfonate, without intrinsic cytotoxicity. REV1-CTD expression increases genomic instability, decreases mutagenesis, and enhances G2 arrest following genotoxic stress. Mutational disruption of the CTD's interaction interfaces abrogates these effects, confirming a dominant-negative mechanism via sequestration of TLS components. In a xenograft mouse model, REV1-CTD expression markedly enhances cisplatin efficacy, significantly reducing tumor burden. These findings establish the REV1-CTD as an effective dominant-negative inhibitor of TLS and support its development as a therapeutic agent delivered to cancer cells to enhance the efficacy of genotoxic chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/119ebde0313a7f12ca9995ad4532ab0aaa8b95c3" target='_blank'>
              Sensitization of cancer cells to DNA-damaging agents by expression of the REV1 C-terminal domain: Implications for chemotherapy.
              </a>
            </td>
          <td>
            Ke Bian, Charley Gruber, Emine Byers, Paul Leclerc, Michael T. Hemann, Graham C. Walker
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chemoresistance is a major obstacle in cancer treatment, and it often results in treatment failure and disease progression. Among the plethora of factors contributing to chemoresistance, the transcription factor Krüppel-like factor 4 (KLF4) has emerged as a pivotal player. This review discusses the role of KLF4 in orchestrating various mechanisms underlying cancer chemoresistance. KLF4, originally identified as a regulator of cell differentiation and proliferation, has recently gained attention for its role in modulating cellular responses to chemotherapeutic agents. Through complex regulatory networks, KLF4 modulates the process of drug efflux, DNA repair, apoptotic signaling, tumor heterogeneity, and cancer cell stemness, leading to the development of cancer chemoresistance. Additionally, tissue or cell types specific post-translational modification (PTM) of KLF4 plays a significant role in the development of cancer chemoresistance. The review explores emerging possibilities and available information that can be utilised to understand the mechanism of chemoresistance mediated by KLF4 in cancer. In conclusion, understanding the complex mechanisms through which KLF4 orchestrates cancer chemoresistance opens promising avenues for developing more effective therapeutic interventions to combat treatment-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e91a9cb65a0f14dcdac5b83f903eda917dba7f" target='_blank'>
              KLF4 in cancer chemoresistance: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Suresh Singh Yadav, Punita Kalia, Navneet Kaur, Sourav Sharma, Shambhavi Thakur, Soni Kumari, R. R. Nair
          </td>
          <td>2025-09-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Acute lymphoblastic leukemia (ALL) is the leading cause of cancer-related mortality in pediatrics. Between 15-30% of patients relapse, and current treatments are associated with acute and long-term toxicity. The immune component of the tumor microenvironment is an under-explored factor that could play a role in the establishment or progression of the disease, as well as modulate response to chemo- and immunotherapies. We characterized the molecular profile of leukemias from 32 patients with pediatric ALL treated under the ALLIC-GATLA 2010 clinical protocol in Argentina by transcriptome sequencing (Illumina). We estimated the immune component through digital cytometry (MIXTURE) and a cytolytic score based on the expression of five genes. We searched for single nucleotide variants (SNV) in the transcriptome (CTAT-Mutations) that could alter the interaction with immune cells in the tumor microenvironment. For this, we analyzed 843 genes from Reactome collections associated with the immune system. Variants were filtered and annotated using publicly available germline and somatic variant databases. The group of patients with higher abundance of cytotoxic cells (Z-score>1) was associated with an increased risk of relapse/death (HR=4.5; p=0.05), and higher levels of CX3CR1 (q-val<0.05) and TIM-3 (q-val=0.06). Analysis of an independent high-risk cohort (TARGET B-ALL phase II, N=88), confirmed that a higher cytolytic score was associated with the risk of relapse in an univariate (HR=2.66, p=0.001) and multivariate model including cytolytic score (p<0.006), protocol (p=0,07), ETV6::RUNX1 (p:>0.05) and TCF3::PBX1 status (p<0.001). Cytolytic score was not associated with molecular subtype, age, sex or early response to treatment (p>0.05). For SNV analysis, we developed a threshold value for the QUAL parameter to select for high-confidence variants, establishing a specificity of 95%. Eleven variants were found in nine patients, being 8/11 classified by Franklin as TIER3 in CEP290, UBE3B, CPNE3, DUSP4, DAPP1, RNF135, TBC1D10C and TLR1; 2/11 as TIER1 in TP53, KRAS; and 1/11 as TIER4 in BTN3A3. Patient harboring variants in these genes were significantly associated with a lower GSVA score in the Reactome dataset “MHC_Class_II_Antigen_Presentation” (q-val<0.01). We found a variant in UBE3B, located in the active site of the E3 ubiquitin ligase, and associated with a low GSVA score in the Reactome “Antigen presentation through MHC class I” pathway. A novel variant in CPNE3 gene was found, located in a highly conserved residue and predicted to be deleterious by multiple tools. Conclusion: the abundance of cytotoxic cells was independent of molecular subtype and associated with a worse prognosis in independent cohorts. Further analysis of the variants found in this study has the potential to identify novel mechanisms modulating the interaction between the immune system and leukemic cells in ALL.



 Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, Pablo Sanchis, Maria Cecilia Riccheri, Geraldine Gueron, Javier Cotignola, Maria Sol Ruiz. Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A040-PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8f83b4eeb02fdcb7483db9e8c77e60a6f1fd7e" target='_blank'>
              Abstract A040-PR008: Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients
              </a>
            </td>
          <td>
            Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, P. Sanchis, M. C. Riccheri, G. Gueron, J. Cotignola, Maria Sol Ruiz
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Advanced sarcomas have a poor prognosis and limited therapeutic options. Disease recurrence is caused by persistent cells that survive drug treatments. The alkylating agent trabectedin, when combined with the poly (ADP-ribose) polymerase 1 (PARP1) inhibitor olaparib, exhibits variable antitumor effects in advanced sarcomas. In this study, we demonstrate that the expression of the transcription factor OCT4 is upregulated in persistent cells that survive treatment with trabectedin and olaparib, through the cGAS-STING-IRF3-IFNβ pathway. This route also leads to the upregulation of natural killer (NK) and cytokine-induced killer (CIK) lymphocyte activating ligands. These molecular events enhance the antitumor efficacy of immunotherapy with NK and CIK cells, targeting both the bulk population and residual drug-tolerant cells. In conclusion, the activation of the cGAS-STING pathway has a double-edged effect, enriching the OCT4+ persistent cell population while increasing the expression of NK/CIK ligands. The addition of olaparib to trabectedin potentiates the cGAS-STING pathway activation and the upregulation of NKG2DLs, while simultaneously counteracting the OCT4 overexpression. Therefore, sequential treatment with trabectedin and olaparib followed by NK/CIK immunotherapy represents a promising strategy against advanced sarcomas and warrants further investigation. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04141-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506de56eb87cad4e65432310520f88b9e3f761bd" target='_blank'>
              The cGAS-STING pathway is a master regulator of OCT4 expression in persistent sarcoma cells and enhances cellular immunotherapy with NK and CIK lymphocytes
              </a>
            </td>
          <td>
            Giorgia Giordano, A. Merlini, F. Capozzi, Giulio Ferrero, Cristina Tucciarello, Shahab Majidi, Simona Comparato, G. Mesiano, Elisabetta Liuzzi, F. Galvagno, Annamaria Massa, Silvia Brusco, V. Leuci, Chiara Riganti, S. Cutrupi, Michele De Bortoli, Lorenzo D’Ambrosio, Dario Sangiolo, G. Grignani, Y. Pignochino
          </td>
          <td>2025-09-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ovarian cancer represents a typically immune “cold” tumor, where obvious immunosuppression, spatial T-cell exclusion, and cellular dysfunction collectively limit immunotherapy effectiveness. Especially in high-grade serous ovarian carcinoma (HGSOC), the immune low-response state is driven by complex interactions among tumor-associated macrophages (TAMs), suppressive stromal networks, and the T-cell compartment (regulatory T cells, Tregs, and exhausted effector T cells). Emerging multi-omics technologies—particularly single-cell RNA sequencing and spatial transcriptomics—have showed the heterogeneity and spatial immune organization underlying this suppressed state. Here, we integrate these datasets to describe TAM phenotypes and spatial niches, T-cell exhaustion, Tregs accumulation, NK-cell dysfunction, and stromal barriers that enforce exclusion. We then derive phenotype-guided combination strategies to remodel the tumor microenvironment and improve responsiveness to immune checkpoint blockade. This synthesis provides a concise, multi-dimensional framework for precision immunotherapy and for overcoming resistance in immune-low ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880f6c405f773eb38fd078c68256074438417fa2" target='_blank'>
              Decoding immune low-response states in ovarian cancer: insights from single-cell and spatial transcriptomics for precision immunotherapy
              </a>
            </td>
          <td>
            Yalan Yan, Jiaan Lu, Huanyu Luo, Zizhang Wang, Ke Xu, Lexin Wang, Qin Wang
          </td>
          <td>2025-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P. Ji, Nancy R. Zhang
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) is the most common hematological malignancy. More than half of DLBCL patients achieve long‐term remission after treatment, but a third relapse after conventional Rituximab (R)‐based chemotherapy regimens, such as CHOP (cyclophosphamide, doxorubicin, vincristine and prednisone). Cancer cells are exposed to chronic replication stress, which impedes the duplication of their genome. Functional DNA repair pathways are therefore important for the survival of cancer cells. This dependence can be exploited therapeutically to hamper repair of the intrinsic DNA damage occurring during replication or to exacerbate DNA damage induced by chemotherapy. Using CRISPR‐Cas9 screening, we identified CHEK1, WEE1, ATR and RAD51 DNA repair factors as essential genes in DLBCL cells. According to these results, we investigated the effect of small molecules targeting DNA replication stress and DNA repair mechanisms, alone or in combination with the R‐CHOP genotoxic agents, cyclophosphamide and doxorubicin. Applying a standard threshold of 2 SDs below the IC50 of the genotoxic agent alone, a total of 3 synthetic lethal combinations have been identified including cyclophosphamide with CHK1/2 inhibitor, cyclophosphamide and ATR inhibitor and doxorubicin with DNAPK inhibitor. Co‐treatment with these molecules led to cell death, DNA damage induction and cell cycle arrest in DLBCL cells more efficiently than genotoxic agents alone. These data have been validated using primary DLBCL cells from patients. Our results open new perspectives for therapeutic approaches exploiting the synthetic lethality of genotoxic agents with DNA repair inhibitors to improve the therapeutic outcome of patients with DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61ea8280a7e78b4aa4b06c11d4f0a4eaa409f0ab" target='_blank'>
              Synthetic Lethal Combinations of DNA Repair Inhibitors and Genotoxic Agents to Target High‐Risk Diffuse Large B Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Ovejero, Julie Devin, Laura Alibert, Camille Soun, Yea-Lih Lin, Laure Dutrieux, M. Abouladze, Elvira Garcia de Paco, Ouissem Karmous Gadacha, A. Constantinou, Guillaume Cartron, C. Herbaux, O. Elemento, Philippe Pasero, Sandrine Roulland, J. Moreaux, C. Bret
          </td>
          <td>2025-08-23</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Complex chromosomal rearrangements (CCRs) are rare structural abnormalities involving at least three chromosomal breakpoints and often two or more chromosomes. Owing to their inherent genomic complexity, CCRs are frequently associated with abnormal phenotypes, including developmental delay, congenital anomalies, and infertility. In this study, we report four male patients, three of them with de novo rare structural chromosomal rearrangement detected through a combination of Giemsa-Trypsin (GTG) banding, fluorescence in situ hybridization (FISH), and high-resolution microarray techniques (SNP array and array CGH). Each of the four cases turned out to be of a different type: in addition to two exceptional CCRs, an inv dup del 18q and a cluster rearrangement involving the long arm of chromosome 4 were identified. Despite the limitations of the testing methods, we performed a detailed analysis of the relationship between the most detailed genotype data and the associated phenotype. Our study provides further valuable evidence that the use of molecular cytogenetic methods is of paramount importance even in cases with abnormal karyotypes detected by light microscopy, as high-resolution data may reveal unsuspected genomic complexity, which is essential for genetic counseling in these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/974abec08ecfb2c8d46a921b99de6ab00061bf03" target='_blank'>
              Uncovering Rare Structural Chromosomal Rearrangements: Insights from Molecular Cytogenetics
              </a>
            </td>
          <td>
            M. Czakó, A. Szabó, Á. Till, Anna Zsigmond, K. Hadzsiev
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background/Objectives: Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with poor prognosis. T cell exhaustion (TEX) is a key factor in tumor immune evasion and therapeutic resistance. In this study, we integrated single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq) data to characterize TEX-related transcriptional features in HCC. Methods: We first computed TEX scores for each sample using a curated 65-gene signature and classified them into high-TEX and low-TEX groups by the median score. Differentially expressed genes were identified separately in scRNA-seq and bulk RNA-seq data, then intersected to retain shared candidates. A 26-gene prognostic signature was derived from these candidates via univariate Cox and LASSO regression analysis. Results: The high-TEX group exhibited increased expression of immune checkpoint molecules and antigen presentation molecules, suggesting a tumor microenvironment that is more immunosuppressive but potentially more responsive to immunotherapy. Functional enrichment analysis and protein–protein interaction (PPI) network construction further validated the roles of these genes in immune regulation and tumor progression. Conclusions: This study provides a comprehensive characterization of the TEX landscape in HCC and identifies a robust gene signature associated with prognosis and immune infiltration. These findings highlight the potential of targeting TEX-related genes for personalized immunotherapeutic strategies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1a1103280627c731b735d281df63ab3e636c5d0" target='_blank'>
              Bulk and Single-Cell Transcriptomes Reveal Exhausted Signature in Prognosis of Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Ruixin Chun, Haisen Ni, Ziyi Zhao, Chunlong Zhang
          </td>
          <td>2025-08-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 CDKN2A/B homozygous deletion is among the most common genetic alterations in glioblastoma (GBM), often accompanied by losses of neighbouring tumour suppressors such as MTAP and PTPRD. These co-deletions create vulnerabilities that may be exploited for synthetic lethality-based therapies. While recent network analysis has demonstrated the potential of combination therapy to target such synthetic lethal nodes, the underlying patterns of co-deletion and their downstream molecular effects remain unclear. We conducted a multi-omics analysis to systematically assess these co-deletion patterns and their functional consequences in CDKN2A/B-deleted GBM.



 Copy number variation (CNV), whole exome sequencing (WES), mRNA expression, and reverse-phase protein array (RPPA) data from TCGA-GBM (n=63) and CPTAC (n=96) were analysed. Patients with signiﬁcant CDKN2A/B losses were identiﬁed using GISTIC2.0 and assessed for co-occurring alterations in MTAP, PTPRD, and EGFR and broader 9p21 losses using Fisher’s Exact test. Downstream mRNA and protein expression changes were evaluated with Mann-Whitney U tests.



 CDKN2A/B homozygous deletion was strongly associated with MTAP loss (TCGA-GBM: 74.6%; CPTAC: 83.0%), moderate PTPRD loss (TCGA-GBM: 55.6%; CPTAC: 30.8%), and moderate EGFR ampliﬁcation (TCGA-GBM: 55.6%; CPTAC: 60.4%). Complete 9p21 loss was common (TCGA-GBM: 54.9%; CPTAC: 35.4%) but deletion proportions varied within and across cohorts. CDKN2A/B-deleted patients had signiﬁcant reductions in MTAP and PTPRD mRNA and protein expression (p < 0.05) and increases in EGFR mRNA and protein expression (p < 0.05) com- pared to non-deleted patients.



 CDKN2A/B deletion is associated with broader 9p21 loss and functional changes in key synthetic lethal nodes, supporting its potential as a therapeutic biomarker. These results justify the development of synthetic lethality- based clinical trials targeting GBM with 9p21 loss. Future work will focus on functional validation of drug combinations and clinical trial design.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbcdf237ba9e5b742a8d73e7613b54470bb9dac1" target='_blank'>
              UNDERSTANDING PATTERNS OF CO-DELETION CDKN2A/B DELETED GBM TO SUPPORT A SYNTHETIC LETHALITY APPROACH TO TREATMENT
              </a>
            </td>
          <td>
            Tanishi Moitra, Michael Castro, Matt Williams
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer risk increases with age, and cellular senescence may be a major contributor to cellular carcinogenesis. Enormous efforts have been made to investigate the interrelation between aging and tumors, but little is known about the comparative features of normal aging, cellular senescence, and cancer at single-cell resolution. By integrating analyses of genomics, epigenomics, and bulk and single-cell transcriptomics, we revealed a directionally opposite transcriptional profile between cellular senescence and tumorigenesis at the single-cell level, which may be affected by epigenomic regulations. A total of 648 aging-dependent senescence-associated coregulated modules (SACMs), disproportionately affecting the reproductive systems of both females and males, were initially defined across 17 tissues. Single-cell analysis revealed that aging primarily affects endothelial cells, followed by T cells, epithelial cells, macrophages, and fibroblasts. Opposite directions of change in gene expression between aging and cancer can commonly be observed in endothelial, fibroblast, and epithelial cells, which may prompt the opposing patterns of gene expression between tissue aging and epithelial carcinoma at the bulk level. A similar pattern of expression can be observed in immune cells, which are characterized by decreased self-renewal with aging, but this pattern is reversed in epithelial carcinoma. Our study highlighted the role of senescence as a natural barrier against tumor formation and supported the idea that aging-related systemic environment changes create a protumorigenic milieu.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af4073352af61f280e6d1b0c8dc5f8cd7b466fe5" target='_blank'>
              Integrative Transcriptomic Analysis Decodes the Interplay Between Aging, Senescence, and Cancer.
              </a>
            </td>
          <td>
            Jiale Cai, Deng Wu, Dahua Xu, Fujin Liu, Peihu Li, Dehua Zheng, Zhizhou Xu, Meng Cao, Yutong Shen, Nihui Zhang, Guanghui Tian, Chengyi Fan, Bo Wang, Kongning Li, Xiaoman Bi
          </td>
          <td>2025-09-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Understanding tumor heterogeneity at the resolution of individualized cell–cell communication networks (CCCNs) remains a major computational challenge in precision oncology. Existing inference methods largely rely on population-level correlation and thus fail to capture patient-specific signaling patterns across diverse cell types. To address this limitation, we developed an integrative computational framework combining the nested hierarchical Dirichlet process (nHDP) model for identifying hierarchically structured gene expression modules, with instance-specific Greedy Fast Causal Inference (iGFCI) for inferring individualized CCCNs (iCCCNs) in colorectal cancer (CRC). Applied to large-scale single-cell RNA-seq data from over 625,000 cells, our model successfully decomposed complex gene expression modules GEMs, potentially representing the cell lineage and cellular signaling states, and uncovered iCCCNs across detailed cell subtypes. We further used TCGA bulk RNA-seq data with survival data to validate the clinical relevance of these individualized gene expression module causal interactions, demonstrating their potential as robust prognostic signatures in CRC. Finally, we used principled causal inference methods to search for ligand-receptor pairs that mediate cell-cell communication. This framework enables mechanistic insights into immune evasion. Our computational method represents a significant advance toward realizing personalized oncology, enabling precise patient stratification and identification of actionable biomarkers for improved therapeutic targeting in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab4e906c5fe00a93c967af7c5aabe8332c82ec34" target='_blank'>
              Inferring Personalized Cell-Cell Communication Networks in Colorectal Cancer with Individualized Causal Discovery
              </a>
            </td>
          <td>
            Aodong Qiu, Binfeng Lu, Gregory F. Cooper, Xinghua Lu, Lujia Chen
          </td>
          <td>2025-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BCL-2 is a central regulator of apoptosis and inhibits cell death by sequestering pro-apoptotic BH3 alpha-helices within a hydrophobic surface groove. While venetoclax, a BH3-mimetic drug, has transformed the treatment of BCL-2–driven malignancies, its efficacy is increasingly limited by acquired resistance mutations that disrupt small-molecule binding yet preserve anti-apoptotic function—reflecting a remarkable structural adaptation. Here, we employ hydrocarbon-stapled alpha-helices derived from the BAD BH3 motif as conformation-sensitive molecular probes to investigate this therapeutic challenge. The stapled peptides not only retain high-affinity binding to all BCL-2 variants but also show enhanced potency to select venetoclax-resistant mutants. Structural analyses, including X-ray crystallography and hydrogen-deuterium exchange mass spectrometry (HDX MS), demonstrate that these stapled helices restore native BH3 engagement by reversing the conformational consequences of resistance mutations. Notably, we identify a serendipitous interaction between the α3–α4 region of BCL-2 and hydrocarbon staple, which further compensates for altered groove conformation and contributes to mutant binding affinity. Together, these findings offer mechanistic insights into BCL-2 drug resistance and reveal a blueprint for designing next-generation inhibitors that overcome this clinically significant barrier to durable treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c989e0417fad14adefa4b6808df4b60e2e5666a7" target='_blank'>
              Structural insights into chemoresistance mutants of BCL-2 and their targeting by stapled BAD BH3 helices
              </a>
            </td>
          <td>
            Thomas M. DeAngelo, Utsarga Adhikary, Kyle J. Korshavn, H. Seo, Clara R. Brotzen-Smith, Christina M Camara, S. Dhe-Paganon, Gregory H. Bird, T. Wales, L. Walensky
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="


 Histone deacetylase inhibitors (HDACi’s) have multiple effects on cancer cell lines, including inhibition of cell growth in vitro, and of human tumor xenografts in vivo, and have had modest therapeutic efficacy in a subset of human malignancies. Previously, we have shown that chronic treatment of colon cancer cells with HDACI’s can result in adaptation, associated with marked decreases in cell growth, colony formation, and tumorigenicity, and extensive changes in gene expression, suggesting an attenuation of malignant phenotype and a possible novel approach to using these agents. Ewing Sarcoma (ES), likely derived from a very primitive mesenchymal-related cell, is an aggressive malignancy of children often resistant to therapy. ES cells classically contain the characteristic EWS-FLI1 translocation and fusion gene, known to dramatically alter histone modifications and epigenetic regulation. We hypothesized that chronic treatment of ES cell lines with HDACi’s will major changes in the program of ES cells gene expression that may alter their malignant phenotype. We treated the ES cell line A673, which contains the EWS-FLI fusion gene, with either stepwise increasing amounts of Entinostat (selective class I HDACi, up to 1.5µM) or single dose chronic treatment (1.5µM). We observed treatment-associated dramatic reductions in growth rates, colony formation, anchorage-independent growth, and major changes in gene expression (RNA sequencing, RT-qPCR) and chromatin organization (ATAC-seq). Gene expression analysis revealed hundreds of differentially expressed genes including changes associated with differentiation along both neural and mesenchymal pathways, consistent with observed changes in lineage-determining transcription factors of the Nkx, Sox, and Pax families. Other genes with changed expression levels included potential novel therapeutic targets such as the tyrosine kinase, RET, whose expression was reduced at both the RNA and protein levels, in association with reduced chromatin accessibility. We hypothesized that Ret may play a role in the malignant phenotype of ES and were interested in whether HDACi-induced reduction in Ret expression might sensitize A673 cells to Ret inhibitors. Co-treatment of A673 cells with Entinostat resulted in over two-fold increase in sensitivity to Cabozantinib, a tyrosine kinase inhibitor that inhibits a select group of tyrosine kinases including Reg. We are continuing to characterize the role of Ret as a potential therapeutic target, and as a mediator of the ES cell malignant phenotype through CRISPR-mediated gene.Acknowledgement: This work was supported by funding from the NJ Pediatric Hematology and Oncology Center of Excellence of the Rutgers Cancer Institute.



 Hsin-Ching Lin, Erica Raj, Luis Sanchez-Gonzalez, Aisling Byrne, Shin-Heng Chiou, Arnold B Rabson. Effects of chronic histone deacetylase inhibitor treatment on Ewing sarcoma cells highlight Ret as a potential therapeutic target [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43276da437d6f15c2c1a18ce683e6dea744bab69" target='_blank'>
              Abstract B014: Effects of chronic histone deacetylase inhibitor treatment on Ewing sarcoma cells highlight Ret as a potential therapeutic target
              </a>
            </td>
          <td>
            Hsin-Ching Lin, Erica Raj, L. Sánchez-González, Aisling Byrne, Shin-Heng Chiou, Arnold B. Rabson
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The mechanisms through which mutations in splicing factor genes drive clonal hematopoiesis (CH) and myeloid malignancies, and their close association with advanced age, remain poorly understood. Here we show that telomere maintenance plays an important role in this phenomenon. First, by studying 454,098 UK Biobank participants, we find that, unlike most CH subtypes, splicing-factor-mutant CH is more common in those with shorter genetically predicted telomeres, as is CH with mutations in PPM1D and the TERT gene promoter. We go on to show that telomere attrition becomes an instrument for clonal selection in advanced age, with splicing factor mutations ‘rescuing’ HSCs from critical telomere shortening. Our findings expose the lifelong influence of telomere maintenance on hematopoiesis and identify a potential shared mechanism through which different splicing factor mutations drive leukemogenesis. Understanding the mechanistic basis of these observations can open new therapeutic avenues against splicing-factor-mutant CH and hematological or other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4518f74e6ec990a2a9612e62e836380371ad0a19" target='_blank'>
              Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis
              </a>
            </td>
          <td>
            Matthew A. McLoughlin, Sruthi Cheloor Kovilakam, William G. Dunn, Muxin Gu, Jake Tobin, Y. Pershad, N. Williams, Daniel Leongamornlert, K. Dawson, Laura Bond, Ludovica Marando, Sean Wen, Rachael Wilson, Giampiero Valenzano, Vasiliki Symeonidou, Justyna Rak, Aristi Damaskou, M. Gozdecka, Xiaoxuan Liu, Clea Bárcena, J. Nomdedeu, Paul Costeas, Ioannis D. Dimitriou, E. Fiorillo, V. Orrù, José Guilherme de Almeida, Thomas McKerrell, Matthew Cullen, I. Mohorianu, T. Foukaneli, Alan J. Warren, Chi Wong, G. Follows, A. Godfrey, Emma J Gudgin, Francesco Cucca, Eoin McKinney, E. J. Baxter, M. Gerstung, J. Mitchell, D. Wiseman, Alexander G. Bick, M. Fabre, Pedro M Quiros, J. Nangalia, Siddhartha Kar, George S. Vassiliou
          </td>
          <td>2025-08-28</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Drug tolerant persisters (DTPs) drive cancer therapy resistance by temporarily evading drug action, allowing multiple routes to eventual permanent resistance. Despite clear evidence for DTPs, the timing of their emergence, proliferative nature, and how their population dynamics arise from measured single-cell kinetics remain poorly understood. Here we use time-lapse microscopy data from two cancer cell lines, integrating single-cell and population measurements, to develop a quantitative description of drug persistence. Contrary to the expectation that increasing levels of genotoxic stress should lead to slower times to division and faster times to death, we observe minor changes in the single-cell intermitotic and death time distributions upon increasing cisplatin concentration. Yet, population decay rates increase 3-fold, suggesting a surprising independence of the overall dynamics from the measured birth and death rates. To explain this phenomenon, we argue that the observed lineage correlations and concentration-dependent decay rates imply cell-state dependent fate choices made both pre and post-cisplatin as opposed to just post-drug birth/death rate-based competitive fate choices. We demonstrate that these cell-states, present in the ancestors of DTP and sensitive cells, exhibit no difference in cycling speed and are inherited across 2-3 cellular generations. A stochastic model implementing these rules simultaneously recapitulates the observed decay rates and cell-fate correlations, also explaining how pre-drug fate decisions are consistent with barcoding experiments where barcode diversity remains unchanged after drug administration. Our results provide a powerful perspective on drug tolerance based on general arguments, without requiring knowledge of the underlying molecular architecture of the heterogeneous cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b499c056c673d79d24c4d2c5acdf399c0d8012aa" target='_blank'>
              Inheritable cell-states shape drug-persister correlations and population dynamics in cancer cells
              </a>
            </td>
          <td>
            Anton Iyer, Adrián E. Granada, Shaon Chakrabarti
          </td>
          <td>2024-11-03</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioma represents the most prevalent primary tumors of the central nervous system, originating from glial cells. Cancer stem cells have the ability to extensively proliferate, self-renew, and form colonies, which contribute to tumorigenesis. Studies have found a population of cells within glioblastoma exhibiting characteristics similar to those of cancer cells, termed glioma stem cells (GSCs). GSCs have two distinct phenotypes: mesenchymal subtype (MES) and proneural subtype (PN). Despite the vital role of these subtypes in glioma biology, there is a significant lack of comprehensive reviews focusing on the regulatory mechanisms underlying each phenotype.This review integrates emerging insights into the regulatory mechanisms underlying GSCs plasticity, with dedicated analysis of novel pathways governing PN and MES phenotypes and their dynamic transition. By examining these critical elements, we aim to contribute to the development of novel therapeutic strategies in the ongoing fight against gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50da6e5130555aeaeba55f2b83f42650df5fbf68" target='_blank'>
              Phenotypic variations in glioma stem cells: regulatory mechanisms and implications for therapeutic strategies
              </a>
            </td>
          <td>
            Guofeng Tian, Yifu Song, Yaochuan Zhang, Liang Kan, Ana Hou, Sheng Han
          </td>
          <td>2025-09-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b348bc9a3e4abf3cb6086b89e0d2182bceec0bd" target='_blank'>
              NEK9-mediated Wnt signalling repressor TLE3 rewires Docetaxel resistance in cancer cells by inducing pyroptosis.
              </a>
            </td>
          <td>
            S. Ansari, Sibasish Mohanty, P. Mohapatra, Sudeshna Datta, Mamuni Swain, Rachna Rath, D. Muduly, Saroj K D Majumdar, R. Swain, Sunil K Raghav, Rupesh Dash
          </td>
          <td>2025-09-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81bcddaf13d081e97df3be63dd2c84368f07c4" target='_blank'>
              mTOR activity and metabolic reprogramming of CD8+ T cells is impaired under hypoxia and within the multiple myeloma bone marrow.
              </a>
            </td>
          <td>
            Taylor Fulton-Ward, N. Gudgeon, Isaac Thirlwell, E. Bishop, Bryan Marzullo, Hannah Giles, Graham McIlroy, Paul Ferguson, B. Kishore, Kate Rogers, Nuri Nuri Alfasi, Timothy Wong, Satnam Aytain, Daniel A. Tennant, G. Pratt, Sarah Dimeloe
          </td>
          <td>2025-09-12</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Overcoming drug resistance and targeting cancer stem cells remain challenges for curative cancer treatment. In particular, patients with chronic myeloid leukemia (CML) often require lifelong therapy with ABL1 tyrosine kinase inhibitors (TKIs), partly due to a persistent population of TKI-resistant leukemic stem cells (LSCs). Therefore, identifying specific pathways crucial for LSC maintenance is necessary. The Hedgehog (HH) pathway, especially the protein Smoothened (SMO), has been found to be essential for CML LSCs, but its role in TKI resistance is still largely unknown. We have now demonstrated that the expression of HH pathway genes SMO and GLI2 is increased in CD34+ CML stem/progenitor cells compared to healthy counterparts, and is higher in TKI-nonresponders than in responders by transcriptome profiling and qRT-PCR analysis. Interestingly, they are most highly expressed in LSCs compared to progenitors and mature cells in TKI-nonresponders. Inhibition of SMO through genetic knockdown or with a potent, selective SMO inhibitor, Glasdegib, reduces the survival of cells from nonresponder patients. Notably, SMO inhibition also sensitizes TKI-nonresponder stem/progenitor cells to Bostutinib, a second-generation TKI, both in vitro and in a patient-derived xenotransplantation (PDX) model. These findings present a promising therapeutic target and a model for curative combination therapies in stem-cell-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/158306749201eb349ae61ddb260f24663468ebb9" target='_blank'>
              Dual Targeting of Smoothened, a Key Regulator in the Hedgehog Pathway, and BCR-ABL1 Effectively Eradicates Drug-Insensitive Stem/Progenitor Cells in Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            Kelly A. Turner, Min Chen, K. Rothe, Donna L. Forrest, Xiaoyan Jiang
          </td>
          <td>2025-10-09</td>
          <td>Cells</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The study by Li et al. , “Single-cell transcriptome analysis reveals the association between histone lactylation and cisplatin resistance in bladder cancer”, investigated how histone lactylation contributes to cisplatin resistance in bladder cancer (BCa). Using high-resolution single-cell lineage tracing, the authors identified a distinct subpopulation of BCa cells with enhanced glycolytic metabolism that exhibited significant cisplatin resistance. Further analyses showed significant enrichment of histone H3 lysine 18 lactylation (H3K18la) at the promoter regions of the transcription factors YY1 and YBX1, which promoted their expression and facilitated the development of a drug-resistant phenotype. This study was the first to combine single-cell omics with lactylation modification mechanisms, providing new insights into strategies for overcoming cisplatin resistance in BCa. Nonetheless, potential limitations of the study should be carefully considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d10bc76434e826705d215ff116fbc298bccf9ac" target='_blank'>
              Histone H3K18 lactylation and cisplatin resistance in bladder cancer: insights from single-cell transcriptomic analysis
              </a>
            </td>
          <td>
            Zongchi Yang, Shaohua Chen
          </td>
          <td>2025-09-24</td>
          <td>Journal of Translational Genetics and Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Understanding the determinants of de novo mutation is critical for elucidating evolutionary processes and genetic disease susceptibility. But the interplay between life history, genomic architecture, and recombination remains poorly understood in non-model species. Domestic dogs, lacking the recombination regulator PRDM9 and subject to intense artificial selection, offer a unique system for dissecting factors that jointly influence mutation accumulation. Here, we leverage large-scale trio sequencing to unravel the determinants of de novo mutations across diverse dog breeds. Results By analyzing 390 trios from 43 breeds, we estimate a germline mutation rate of 4.89 × 10−9 per base pair per generation. Parental age, especially paternal age, strongly influences mutation rates, with a 1.5-fold greater paternal age effect in dogs compared to humans. Larger breeds exhibit elevated early-life mutations, aligning with accelerated developmental trajectories. Strikingly, CpG Islands in dogs exhibit a 2.6-fold higher mutation rate than the genomic average, unlike humans where no such increase occurs. We also find a tenfold hypermutated dog and suggest a unique maternal mechanism of MLH1-mediated germline instability during gametogenesis. Conclusions The unique mutational landscape in canids is determined by paternal age, body size, and CpG Islands recombination. Despite extensive breeding, germline mutation rates in dogs remain stable across breeds. The elevated mutation rate in CpG Islands due to recombination in the absence of PRDM9 underscores a distinct evolutionary mechanism in canids. These findings enhance our understanding of mutation dynamics, with implications for canine genetic diversity, disease susceptibility, and broader genomic studies in species lacking PRDM9. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03804-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a34d7a6083e4e2b09c979f8002d74afe9bdf558" target='_blank'>
              Determinants of de novo mutations in extended pedigrees of 43 dog breeds
              </a>
            </td>
          <td>
            Shao-Jie Zhang, Jilong Ma, Meritxell Riera, S. Besenbacher, J. Niskanen, Noora Salokorpi, S. Hundi, M. Hytönen, Tong Zhou, Gui-Mei Li, E. Ostrander, M. Schierup, H. Lohi, Guo-Dong Wang
          </td>
          <td>2025-09-25</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/870c85828d8ee9433f6ac03444541eaaaf010162" target='_blank'>
              The UAE Genome Program: Unique Genetic Insights from 43,608 Individuals
              </a>
            </td>
          <td>
            M. M. Mousa, M. M. Olbrich, I. I. Wohlers, A. A. Al aamri, A. H. Alsuwaidi, N. A. Marzouka, H. H. Alnaqbi, M. Alameri, D. D. Ruta, J. J. Alazazi, T. T. Magalhaes, J. J. Mafofo, J. J. Quilez, M. M. Allam, M. S. Mohamad, N. N. Drou, Y. Y. Idaghdour, R. R. Hamoudi, G. G. Tay, S. S. Ibrahim, F. F. Alkaabi, A. A. Al Mannaei, H. H. Alsafar
          </td>
          <td>2025-09-14</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer treatment is often complicated by the intricate dynamics of gene mutations, which
vary across different tumor sizes. Understanding these variations is essential for the development of targeted
therapies and the improvement of patient outcomes. In this study, we conducted a comprehensive analysis of gene
variant counts and mutation rates in breast cancer tumors, categorized by size into three groups: 0-49 mm, 50-
99 mm, and 100-149 mm. Genomic data were systematically extracted and analyzed to identify key genes with the
highest variant counts and to calculate mutation rates normalized per 1,000 observations. Data visualization
techniques were employed to elucidate trends in mutation frequencies across the different tumor sizes. Our
findings identified PARP1P1 as a gene with consistently high variant counts and mutation rates across all tumor
sizes, indicating its potential as a robust biomarker for cancer progression. Conversely, CCDC7, DLGAP1, and
SYN3 exhibited higher mutation rates predominantly in smaller tumors, suggesting their involvement in the early
stages of tumorigenesis. These results underscore the critical importance of gene-specific mutation patterns in
breast cancer, with PARP1P1 emerging as a gene of significant clinical relevance. Additionally, the sizedependent mutation patterns of CCDC7, DLGAP1, and SYN3 emphasize the need for early detection and the
development of targeted interventions. Future research should aim to validate these findings in larger patient
cohorts and investigate the functional implications of these mutations to further advance precision medicine in
breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07cd2288a49ab4f4df24f64b69bb548c18f6cb3" target='_blank'>
              Single Nucleotide Polymorphisms Reveal Tumor SizeDependent Mutation Patterns in Breast Cancer
              </a>
            </td>
          <td>
            Hongyu Gao Hongyu Gao
          </td>
          <td>2025-09-01</td>
          <td>Journal of Medical and Dental Science Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Autosomal monoallelic gene expression and asynchronous replication between alleles are well-established features of imprinted genes and genes regulated by allelic exclusion. Inactivation/Stability Centers (I/SCs) are recently described autosomal loci that exhibit epigenetic regulation of allelic expression and replication timing, with differences that can be comparable to those observed between the active and inactive X chromosomes 1. Here we characterize hundreds of autosomal loci with allele-specific epigenetic regulation of replication timing and gene expression, defining them as I/SCs. I/SCs are approximately 1 megabase in size and can contain both protein-coding and noncoding genes. In different single cell derived clones, these genes may be expressed from a single allele, the opposite allele, both alleles, or not expressed at all. This stochastic, yet mitotically stable, pattern indicates that the choice of which allele is expressed is independent of parent of origin and independent of the expression status of the other allele. Similarly, alleles within I/SCs show varying replication timing, either earlier or later, that is also independent of the other allele. Additionally, we find corresponding I/SCs in the mouse genome that display conserved synteny with human I/SCs. This allele-restricted regulation creates extensive cellular mosaicism through a stable epigenetic mechanism. This mosaicism impacts numerous dosage-sensitive genes associated with human diseases such as Parkinson, epilepsy, deafness, and impaired intellectual development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b5365a9e1403e23b11261427c35af7cc20c05ab" target='_blank'>
              Autosomal Allelic Inactivation: Variable Replication and Dosage Sensitivity
              </a>
            </td>
          <td>
            Michael B. Heskett, Athanasios E. Vouzas, Brian Johnstone, Krister P Freese, Phillip A. Yates, P. F. Copenhaver, Paul T. Spellman, David M. Gilbert, Mathew J. Thayer
          </td>
          <td>2025-08-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Although xenotransplantation has been revolutionized by the development of genome-edited pigs, it is still unknown whether these pigs and their offspring remain genomically stable. Here, we show that GGTA1-knockout (GTKO) pigs accumulated an average of 1205 genome-wide genetic mutations, and their filial 1 (F1) offspring contained an average of 18 de novo mutations compared with wild-type controls and their parents. The majority of mutations were in regions annotated as intergenic regions, without altering protein functions, and were not located at predicted off-target mutation sites. RNA-sequencing analysis and phenotype observations indicated that the accumulated mutations may have only a limited influence on GTKO pigs and most of the mutations in the GTKO pigs could be attributed to the electrotransfection of plasmids into cells. This is the first report that genetic mutations in genome-edited pigs are inherited stably by the next generation, providing a reference for the safe application and a standard approach to breed genome-edited pigs for xenotransplantation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/218c09acaaa59ca5db66362a81d2de17f5d053aa" target='_blank'>
              Genome-wide Genetic Mutations Accumulated in Pigs Genome-edited for Xenotransplantation and Their Filial Generation.
              </a>
            </td>
          <td>
            X. Huo, Xianhui Sun, Xiangyang Xing, Jing Lu, Jingjing Zhang, Yanyan Jiang, Xiao Zhu, Changlong Li, Jianyi Lv, Meng Guo, Lixue Cao, Xin Liu, Zhenwen Chen, Dengke Pan, Shunmin He, Chen Zhang, Xiaoyan Du
          </td>
          <td>2025-08-20</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Evolution of microbes under laboratory selection produces genetically diverse populations, owing to the continuous input of mutations and to competition among lineages. Whole-genome whole-population sequencing makes it possible to identify mutations arising in such populations, to use them to discern functional modules where adaptation occurs, and then map gene structure–function relationships. Here, we report on the use of this approach, adaptive genetics, to discover targets of selection and the mutational consequences thereof in E. coli evolving under chronic nutrient limitation. Results Replicate bacterial populations were cultured for ≥ 300 generations in glucose limited chemostats and sequenced every 50 generations at 1000X-coverage, enabling identification of mutations that rose to ≥ 1% frequency. Thirty-nine genes qualified as high value targets of selection, being mutated far more often than would be expected by chance. A majority of these encode regulatory proteins that control gene expression at the transcriptional (e.g., RpoS and OmpR), post-transcriptional (e.g., Hfq and ProQ), and post-translational (e.g., GatZ) levels. The downstream effects of these regulatory mutations likely impact not only acquisition and processing of limiting glucose, but also assembly of structural elements such as lipopolysaccharide, periplasmic glucans, and cell surface appendages such as flagella and fimbriae. Whether regulatory or structural in nature, recurrent mutations at high value targets tend to cluster at sites either known or predicted to be involved in RNA–protein or protein–protein interactions. Conclusions Our observations highlight the value of experimental evolution as a proving ground for inferences gathered from traditional molecular genetics. By coupling experimental evolution to whole-genome, whole-population sequencing, adaptive genetics makes it possible not only the genes whose mutation confers a selective advantage, but also to discover which residues in which genes are most likely to confer a particular type of selective advantage and why. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02331-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6356b2496b6b3e097c09287be315e34ce18f1a0c" target='_blank'>
              Adaptive genetics reveals constraints on protein structure/function by evolving E. coli under constant nutrient limitation
              </a>
            </td>
          <td>
            Katja Schwartz, M. Kinnersley, Charles Ross Lindsey, Gavin Sherlock, Frank Rosenzweig
          </td>
          <td>2025-08-20</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d07528eee7ff3d627e8787aab243a7521b7075a4" target='_blank'>
              Targeting p53-MDM2 pathway with novel triazole-oxazole hybrids: a fragment-based drug discovery approach for next-generation cancer therapies.
              </a>
            </td>
          <td>
            Apurva Prajapati, Hitesh Patel
          </td>
          <td>2025-09-23</td>
          <td>Molecular diversity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and immunologically "cold" cancer, with a 5-year survival rate of 13%. Despite the success of immunotherapy in other malignancies, its efficacy in PDAC remains limited due to complex immune evasion mechanisms and a pro-tumorigenic immune microenvironment. Several lines of evidence suggest that PDAC progresses from acinar-to-ductal metaplasia (ADM) to pancreatic intraepithelial neoplasia (PanIN) lesions to invasive cancer, accompanied by increasing immunosuppression. We hypothesize that understanding the spatial and temporal transitions of immune cell states during tumor evolution is key to uncovering mechanisms of immune dysfunction and guiding new therapeutic strategies. Using single-cell RNA sequencing (scRNA-seq) and multispectral imaging, we aim to uncover comprehensive, large-scale, and longitudinal changes in the immune microenvironment during PDAC progression. Here, we utilized the KrasG12D/+; Trp53R172H/+; Ptf1a-Cre (KPC) genetically engineered mouse model, and through longitudinal histological analysis of pancreas, we delineated the trajectory of disease progression, with PanIN I-II lesions appearing by 8–9 weeks, PanIN III by 11–12 weeks, and locally invasive PDAC by 16 weeks of age. To study the temporal evolution of immune cell-state transitions, we performed scRNA-seq on CD45+ enriched immune cells isolated from KPC mouse pancreas collected at 4, 8, 12, and 16 weeks using the 10x Genomics Chromium platform. To examine the spatial dynamics of immune cells, we performed multispectral imaging at defined time points using the Lunaphore COMET platform with a panel of over 15 antibodies, enabling simultaneous visualization of several markers on a single tissue section. In our analysis to date, we identified subsets of pro-tumorigenic macrophages emerging during PDAC development, along with a marked increase in myeloid-derived suppressor cells (MDSCs) in late-stage tumors. We also observed dynamic shifts in other immune cell populations evolving alongside tumor progression. Notably, we uncovered distinct subsets of monocytes, macrophages, and T cells exhibiting tumor-promoting potential, progressively enriched during the advanced stages of PDAC. Using multispectral imaging, we generated a comprehensive spatial map of the immune microenvironment across PDAC progression, capturing patterns of immune exclusion, infiltration, and cell–cell neighborhood interactions. Our study highlights the multi-layered immune dysfunction that evolves from early to late stages of PDAC - features that are challenging to capture in human samples. CD45+ enrichment improved the detection of rare and transient immune subsets by mitigating the masking effects caused by abundant cancer and stromal cells. Together, these findings provide an unbiased and detailed view of the evolving immune landscape in PDAC, laying the foundation for identifying microenvironmental and cell-intrinsic factors that either constrain or promote malignant progression, as well as for developing immune-based therapeutic strategies.



 Atul Verma, Doreen Klingler, Linda Bergmayr, Tanvi V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug. Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A106.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28e0f49d549158981070f5d866ac858fc605d13b" target='_blank'>
              Abstract A106: Tracing the immune landscape: Temporal and spatial remodeling during pancreatic cancer progression
              </a>
            </td>
          <td>
            Atul Verma, Doreen Klingler, Linda Bergmayr, T. V. Inamdar, Jonas Rosendahl, Michael Böttcher, Stefan Hüttelmaier, Patrick Michl, Philipp Jurmeister, Sebastian Krug
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Cancer remains the most lethal disease globally, despite the significant progress made in early screening, surgery, and therapeutic development in recent decades. Programmed cell death (PCD) is a genetically regulated process essential for eliminating aberrant cells, yet its dysregulation drives tumorigenesis and therapy resistance. In this review, we present a complete discovery timeline of them and comprehensively synthesize the roles and mechanisms of major PCD forms, such as apoptosis, necroptosis, autophagy, pyroptosis, ferroptosis, and cuproptosis, across diverse cancer types. We not only detail the molecular mechanisms, dual functions, and alterations of these PCD modalities in cancers, but also summarize their interconnections and intrinsic crosstalk. Furthermore, we comprehensively discuss how diverse therapies, including chemotherapy, radiotherapy, immunotherapy, targeted agents, and hormone therapy, engage and manipulate specific PCD pathways, revealing the involvement of PCD in cancer treatment mechanisms. This review integrates extensive preclinical and clinical evidence on PCD‐targeted therapies with an in‐depth focus on ferroptosis, including its regulatory networks and therapeutic relevance. Special emphasis is placed on prostate cancer, highlighting the PCD‐based translational opportunities in this common malignancy. Taken together, we provide novel insights into the complex interplay between PCD and cancer biology and offer a framework for developing precision oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c45427c1d2eb5ce79e949eeb0b16e4625cc2c1" target='_blank'>
              Programmed Cell Death in Cancer
              </a>
            </td>
          <td>
            Yuang Wei, William Hankey, Dongliang Xu, Fuwen Yuan
          </td>
          <td>2025-08-31</td>
          <td>MedComm</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Introduction The germline variations in the mitochondrial genome of esophageal cancer (EC) remain uncertain. Our study aimed to explore the distribution and pathogenicity of mitochondrial genome variations in EC, as well as to identify haplogroups associated with the development of EC. Methods We performed next-generation sequencing of the mitochondrial genomes from peripheral blood samples of 146 EC patients and 120 healthy controls. Variant annotation was performed using MitoMap, while pathogenicity prediction was conducted through tools such as MitoTip, SIFT, and PolyPhen2. Moreover, haplogroup classification was carried out using the Haplogrep3 platform. Results A total of 1299 mitochondrial variants were identified among 146 EC patients, including 171 novel (previously unreported) mutations. Compared with the healthy control group, the EC cohort exhibited a higher frequency of variants in genes such as ND2, COX1, COX2, 12S rRNA, and 16S rRNA. Three tRNA mutations (7496_T>C, 5771_A>G, and 5613_T>A) were predicted to be potentially pathogenic. Within the protein-coding regions, 14 variants were classified as deleterious based on predictions from 13 independent bioinformatic algorithms. Notably, mitochondrial haplogroup D4 was significantly associated with a decreased risk of developing EC. Furthermore, several mtDNA single-nucleotide polymorphisms (SNPs), including 302_A>AC, 1824_T>C, 1842_A>G, 3010_G>A, 8414_C>T, and 14668_C>T, showed significant associations with EC susceptibility. Conclusion We found that the number of variations in multiple regions of the mitochondrial genome in the EC population was higher than that in the control group. Additionally, several potentially pathogenic variants were identified, and haplogroup D4 was suggested as a potentially protective haplogroup against the development of EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf327fe5e0d4d48be3be7be1aba2802cb0e06f01" target='_blank'>
              Next-generation sequencing of mitochondrial DNA reveals pathogenic variants and protective haplogroup D4 in esophageal cancer
              </a>
            </td>
          <td>
            Xiucheng Jiang, Lan Shi, Mei Zhao, Cui Chen, Tao Tang, Simeng Ji, Bingbing Lv, Lihua Jia, Shuhan Duan, Jinyue Ma, Jiyu Pang, Bo Mu, Yongsheng Zhao, Junbao Yang
          </td>
          <td>2025-09-22</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Multiple myeloma (MM) is an incurable malignancy of plasma cells, with over 35,000 new cases diagnosed annually in the United States. Despite an expanding arsenal of approved therapies, nearly all patients relapse, and mechanisms underlying disease progression remain poorly understood. In particular, the role of long non-coding RNAs (lncRNAs) in MM progression and treatment response is largely unexplored. To address this gap, we performed transcriptome sequencing of newly diagnosed MM (NDMM) patient samples and compared individuals with short progression-free survival (PFS; <24 months) to those with prolonged PFS (>24 months) following standard first-line therapy. We identified 157 lncRNAs upregulated in patients with short PFS, and prioritized the most significantly upregulated transcript, LINC01432, for functional characterization. CRISPR-mediated knockdown of LINC01432 expression results in upregulation of genes associated with interferon-α/γ responses and increases apoptosis. Targeting LINC01432 with locked nucleic acid antisense oligonucleotides also induces apoptosis, which can be rescued by LINC01432 overexpression. Mechanistically, we discovered that LINC01432 binds the RNA-binding protein CELF2 directly. Transcriptomic analysis following depletion of either LINC01432 or CELF2 revealed 108 overlapping target genes, indicating that this lncRNA–protein complex regulates transcriptional programs governing immune activation, stress response, and cell survival. In summary, this study identified lncRNAs associated with NDMM and characterized LINC01432 as a critical regulator of MM cell survival, acting in complex with CELF2 to repress pro-apoptotic and immune response pathways. These findings highlight LINC01432 as a potential therapeutic target for overcoming resistance in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd632c0840577b467e9ee9d9ed820a2a7df922de" target='_blank'>
              Investigation of lncRNA expression in newly diagnosed multiple myeloma reveals a LINC01432-CELF2 axis as an inhibitor of apoptosis
              </a>
            </td>
          <td>
            Richa Mishra, Prasanth Thunuguntla, Dhanusha Duraiyan, Alani Perkin, Katelyn Bagwill, Savannah Gonzales, Catheryn Sizemore, Vanessa Brizuela, Jaiyana King, Steve Daly, Yoon Jae Chang, Mahdote Abebe, Yash Rajana, Kelly Wichmann, Christ Enyan, Shruthi Rangineni, M. Fiala, J. Fortier, R. Jayasinghe, Mark Schroeder, Li Ding, Ravi Vij, John DiPersio, Jessica M. Silva-Fisher
          </td>
          <td>2025-10-06</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a highly heterogeneous malignancy, with its tumor immune microenvironment (TIME) playing a crucial role in tumor progression, immune evasion, and treatment response. Despite insights from single-cell technologies, challenges such as small samples and inconsistent annotations have limited a comprehensive understanding. Thus, a large-scale, high-resolution single-cell atlas is needed to fully explore the heterogeneity of CRC TIME and guide personalized treatment. Methods We analyzed single-cell transcriptomic data from 100 CRC samples (371,223 cells). Immune, stromal, and epithelial cells in the TIME were clustered, annotated, and functionally analyzed. Transcription factor networks were inferred using SCENIC, differentiation states assessed with CytoTRACE, and cell–cell communication analyzed via CellPhoneDB. TIME subtypes were constructed from bulk RNA-seq data, followed by survival analysis. SCRN1 was further validated through in vitro experiments. Results This study presents a comprehensive single-cell CRC atlas, revealing significant heterogeneity in T cells, B cells, myeloid cells, fibroblasts, endothelial cells, and epithelial cells within the TIME. We defined two immune ecological subtypes: subtype 1 (enriched in metabolic and motility pathways with poor prognosis) and subtype 2 (enriched in immune response pathways, with better prognosis and greater immunotherapy potential). High SCRN1 expression in malignant epithelial cells correlated with poor prognosis and metastasis. In vitro experiments confirmed the role of SCRN1 in promoting CRC cell proliferation and migration. Conclusions This study provides a comprehensive characterization of the CRC TIME, links TIME subtypes to prognosis, and identifies SCRN1 as a potential therapeutic target, offering insights for CRC precision subtyping and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04173-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32c5be539fad2c0073ae6e62159eaef1e83a1ef" target='_blank'>
              The single-cell atlas of colorectal cancer reveals cellular heterogeneity and key molecular features of the tumor immune microenvironment
              </a>
            </td>
          <td>
            JunHu Li, ZhiQiang Liu, BoYu Zhang, Liuping You, Yuenan Huang
          </td>
          <td>2025-10-10</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer.
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-06</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a prevalent and malignant tumor of the digestive system, characterized by high incidence and mortality rates. This study aimed to investigate the heterogeneity of the tumor microenvironment (TME) and the involvement of immune cells in CRC. Single-cell RNA sequencing (scRNA-seq) data obtained from the Gene Expression Omnibus (GEO) database were used to analyze and identify six major cell types across normal, core, and border tumor samples. A total of 27414 cells from various regions of patients with CRC were selected for subsequent analyses. Cellular interaction analysis revealed that differential signaling pathways between the TME and normal tissues, with several pathways involving interactions between myeloid cells and epithelial cells. Myeloid cells were extracted and classified into six subtypes based on markers identified in the literature. Monocle3 revealed the trajectory of tumor-associated macrophages (TAMs) and identified genes associated with pseudotime. Single-Cell ENrichment analysis for Interpreting Cellular Heterogeneity (SCENIC) analysis identified specific regulons and target genes associated with TAMs. This study reanalyzed single-cell RNA-sequencing data and provided insights into the heterogeneity of the TME, particularly in relation to the role of TAMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e37ff9c0af32829265ad92932c051ab5305d1fd" target='_blank'>
              Single-cell RNA sequencing analysis reveals the heterogeneity and effect of TAMs in colorectal cancer.
              </a>
            </td>
          <td>
            Chengang Wang, Ying Qian, Mingsheng Zhang
          </td>
          <td>2025-08-20</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="SUMMARY Metastasis is an emergent continuum, driven by evolving reciprocal adaptations between continuously disseminating tumor cells (DTCs) and the specialized metastatic niches of distant organs. The interplay between intrinsic and niche-driven mechanisms that enables DTCs to survive and home to distant organs remains incompletely understood. Here, using MetTag, a single-cell barcoding and transcriptome profiling approach with time-stamped batch identifiers (BC.IDs), we mapped temporal, clonal dynamics of DTCs and the immune cell landscape across ovarian cancer metastatic niches. Deep sequencing of barcodes revealed preferred enrichment of early-disseminated clones across metastatic niches. Mechanistically, single-cell RNA sequencing (scRNA-seq) coupled with velocity analyses in ascites and metastasis-bearing omenta uncovered an emergent, distinct interferon-gamma (IFNγ) centric transcriptional trajectory, enriched among early seeding clones. Moreover, in vivo CRISPR/Cas9 screening of metastatic niche-specific signatures demonstrated that genes belonging to the ascites IFNγ signature, including Marco, Gbp2b, and Slfn1, are functionally important for peritoneal metastasis. Knockout of IFNγ receptor 1 (Ifngr1) in tumor cells significantly reduced metastatic burden and extended survival, underscoring the importance of tumor cell intrinsic IFNγ signaling in ovarian cancer metastasis. Furthermore, we identified that the tumor intrinsic IFNγ response and ascites-derived tumor-associated macrophages (TAMs) protect cancer cells from anoikis-mediated death within the IFNγ-rich ascites environment. Our study resolves temporal dynamics of disseminating tumor cells and highlights an ascites-driven, IFNγ program as a necessary pro-metastatic adaptation in the ovarian metastasis cascade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95e0d9496ced48f75294cbc20541bbba656bbf" target='_blank'>
              Temporal Clonal Tracing and Functional Perturbation Reveal Niche-Adaptive and Tumor-Intrinsic IFNγ Dependencies Driving Ovarian Cancer Metastasis
              </a>
            </td>
          <td>
            Emilija Aleksandrovic, Shaneann Fross, Samantha M. Golomb, Xiyu Liu, Zhuo Zhao, Nikitha M. Das, T. Reese, Wei Ma, Jacqueline Lopez, M. S. Stack, Min Zhao, Siyuan Zhang
          </td>
          <td>2025-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd9218b7e444fde0659e2b4793dd03d323ad206c" target='_blank'>
              Single-cell genetics identifies cell type-specific causal mechanisms in complex traits and diseases
              </a>
            </td>
          <td>
            A. Henry, A. Senabouth, R. Tyebally, B. Bowen, P. C. Allen, E. Spenceley, E. Sagi-Zsigmond, A. Cuomo, J. Fan, H. L. Huang, H. Tanudisastro, A. Xue, K. M. de Lange, G. Figtree, A. W. Hewitt, D. G. MacArthur, J. E. Powell
          </td>
          <td>2025-08-29</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0987fea6e48fd7cd3a2b34fbb805d81b9c4a798" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Nancy R Zhang, Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew Pardoll, Christina Jackson
          </td>
          <td>2025-09-24</td>
          <td>Research Square</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Treatments for high-grade serous ovarian cancer (HGSOC) are initially effective but most invariably fail. Although they can successfully suppress the bulk of the tumour cell population, residual cancer cells can enter alternative therapy-resistant cell fates highlighted by proliferative arrest. Understanding the nature of these fates and how cells may resume uncontrolled proliferation will lead to the development of new treatments for HGSOC. In this study, we examine the response of HGSOC cells to standard of care cisplatin chemotherapy and to the RNA Polymerase I transcription inhibitor CX-5461/Pidnarulex, two drugs that elicit a potent DNA damage response and growth arrest. Here, we identify that HGSOC cells exposed to these therapies show multiple hallmarks of therapy-induced senescence (TIS) and derive a core TIS gene expression signature irrespective of genetic background or senescence trigger. Given that TIS is a potentially escapable state, we have performed a focussed drug screen to identify drugs that eradicate senescent HGSOC cells. We identify that therapy-induced senescent HGSOC cells, including those with decreased sensitivity to senolytic drugs that inhibit the pro-survival protein BCL-XL, can be eliminated using drugs that induce ferroptosis, an iron-dependent form of cell death. Mechanistically, we demonstrate that senescent HGSOC cells have altered expression of regulators of iron metabolism leading to intracellular iron overload that underpins this targetable vulnerability. Together, we highlight elevated levels of iron as a TIS biomarker in HGSOC and the potential of inducing ferroptosis to eradicate residual HGSOC cells following initial therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3787e084ffe3e100d6ec83768df6b1bb600d9e4a" target='_blank'>
              Exploiting dysregulated iron homeostasis to eradicate persistent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Carmelo Cerra, Madeleine R. C. Tancock, N. Thio, Ada Koo, AnnRann Wong, Karla J Cowley, Swati Varshney, Madelynne O. Willis, Kaylene Simpson, D. Bowtell, E. Sanij, Elizabeth L Christie, Richard B. Pearson, Jian Kang, Keefe T. Chan
          </td>
          <td>2025-09-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Trisomy 21 (T21), resulting in Down Syndrome (DS), is the most prevalent chromosomal abnormality worldwide. While pulmonary disease is a major cause of morbidity and mortality in DS, the ontogeny of pulmonary complications remains poorly understood. We recently demonstrated that T21 lung anomalies, including airway branching and vascular lymphatic abnormalities, are initiated in utero. Here, we aimed to describe molecular changes at the single cell level in prenatal T21 lungs. Single cell RNA sequencing (scRNAseq) was used to generate transcriptomic profiles of individual human lung cells in tissue obtained from T21 (n=5) and non-T21 (n=4) prenatal lungs. Clustering of cells, marker identification, UMAP representation, and differential expression analysis were performed in Seurat. Cell type annotation and pathway analysis were annotated using Toppfun and a human fetal lung cell atlas. Spatial differences in cellular phenotypes were validated using immunofluorescent staining (IF) and fluorescent in situ hybridization (FISH). Our results detail changes in gene expression at the time of initiation of histopathological abnormalities in T21 prenatal lungs. Notably, we identify precocious differentiation of epithelial cells, widespread induction of key extracellular matrix molecules in mesenchymal cells and hyper-activation of IFN signaling in endothelial cells. This single cell dataset of T21 lungs greatly expands our understanding of antecedents to pulmonary complications and should facilitate efforts to mitigate respiratory disease in DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375170fedb13897af17ad3594d19e24e0cad2f55" target='_blank'>
              The Transcriptional Landscape of Developing Human Trisomy 21 Lungs.
              </a>
            </td>
          <td>
            S. Bhattacharya, A. Frauenpreis, C. Cherry, Gail Deutsch, I. Glass, Thomas J. Mariani, Denise Al Alam, Soula Danopoulos
          </td>
          <td>2025-09-25</td>
          <td>American journal of respiratory cell and molecular biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Cisplatin remains a standard first-line therapy for epithelial ovarian cancer; however, the development of chemoresistance leads to poor prognosis and high recurrence rates. Kaplan–Meier analysis of patient data (TCGA) revealed that cisplatin-sensitive tumors are associated with significantly improved overall survival, underscoring the need for strategies to overcome resistance in clinical settings. To investigate the underlying mechanisms, we conducted a comprehensive bioinformatics analysis of publicly available GEO datasets from cisplatin-treated ovarian cancer patients (n = 255). This analysis identified six key molecular drivers of resistance: ZBTB33 (Kaiso), pregnane X receptor (PXR), NF-κB, HER2 (ERBB2), P-glycoprotein (P-gp/ABCB1), and HIF1A. These targets were validated in ovarian tumor specimens via immunohistochemistry, confirming elevated expression in chemo-resistant disease. Additionally, the quantitative real-time PCR analysis confirms the transcriptional upregulation of the six resistance-associated genes in cisplatin-resistant SKOV3 and OVCAR-5 ovarian cancer cells, consistent with the IHC findings. The average fold change in mRNA transcripts ranged from 2.4 for P-glycoprotein (P-gp) to 5 for both NF-kB and Kaiso (ZBTB33). Although less well studied in ovarian cancer, Kaiso (ZBTB33) is known to regulate EMT and tumor invasion in other solid tumors. Functional studies using SKOV3 and OVCAR-5 cell lines demonstrated that knockdown of Kaiso (ZBTB33) via RNA interference significantly increased cisplatin-induced cell death, indicating a direct role in therapeutic resistance. Furthermore, we investigated the synergistic effects of combining SDA, a plant-based omega-3 fatty acid known to inhibit NF-κB, and cisplatin on the cell death of SKOV3 and OVCAR-5 cell lines, comparing the results to those of each compound used individually. Interestingly, co-treatment with stearidonic acid (SDA) synergistically enhanced the cytotoxicity of cisplatin at a lower dose in both cell models. These findings reveal a clinically relevant resistance signature and highlight the therapeutic potential of combinatorial strategies that target both transcriptional regulators (e.g., Kaiso) and inflammatory signaling (e.g., NF-κB). Dual targeting of these pathways may resensitize tumors to cisplatin and improve outcomes for patients with advanced ovarian cancer.



 Mahmoud Mansour, Sabrina Van Ginkel, Maaz Kamal Alata, Ibrahim Bani, Isra Elhussin. Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a654db1a318364cee4d66d47d1f28ac0d3c685d" target='_blank'>
              Abstract A025: Molecular signature of cisplatin resistance in ovarian cancer identifies therapeutic opportunities for re-sensitization
              </a>
            </td>
          <td>
            Mahmoud Mansour, Sabrina Van Ginkel, M. Alata, Ibrahim Bani, Isra Elhussin
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background High-dose chemotherapy for neuroblastoma is often intolerable for children, and its effectiveness is difficult to determine. Immunotherapy has become a popular research focus as a potential treatment. Therefore, identifying effective immune targets and drug synergistic chemotherapy against neuroblastoma is crucial. Methods We conducted proteomics exploratory analysis of urine from 12 neuroblastoma before and after chemotherapy. The immune-related differential proteins before and after chemotherapy were obtained through differential expression analysis and dynamic expression model analysis. The ESTIMATE and ssGSEA analyses used indirectly infer the characteristics of the immune microenvironment. Gene ontology and the Kyoto Encyclopedia of Genes and Genomes enrichment analysis were conducted to reveal the biological functions of the candidate genes. Results Immune analysis indicated that the immunosuppressive state in neuroblastoma patients caused by chemotherapy was closely related to CD8 T cells. Chemotherapy-associated differentially expressed proteins were significantly enriched in the mTOR signaling pathway. Neuroblastoma chemotherapy can significantly inhibit the expression of SEMA7A protein, and it is closely related to CD8T cell infiltration. Tetrachlorodibenzodioxin acting on SEMA7A was predicted using CTD databases. Conclusions This study combines proteomics and bioinformatics to predict and explore potential immune targets for synergistic chemotherapy against neuroblastoma, providing a new direction for clinical treatment that warrant further validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff0047968ad7104558f18bc99ba975d0d0c0dd6e" target='_blank'>
              The changes of immune function after neuroblastoma chemotherapy and the prediction of potential immune targets
              </a>
            </td>
          <td>
            Yunfei Ma, Yan Su, Jing Huang, Wen Zhao, Cheng Huang, Jing Qin, Shengjie You, Yan Hu, Xin Ni
          </td>
          <td>2025-08-20</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Purpose The alternative splicing of mRNA precursors allows one gene to yield multiple proteins with distinct functions. CDC-like kinases serve as pivotal regulators of alternative splicing. Control of protein expression also occurs at the level of DNA through histone methylation and demethylation. We investigated the activity of two CLK inhibitors, cirtuvivint and CC-671, and the LSD1 inhibitor iadademstat alone and in combination with anticancer drugs or investigational agents. Methods Well-characterized patient-derived cancer cell lines from the PDMR (https://pdmr.cancer.gov/models/database.htm) were used along with standard human cancer cell lines. Multi-cell type-tumor spheroids were grown from a ratio of 6:2.5:1.5 malignant cells, endothelial cells, and mesenchymal stem cells. Following three days of growth, the spheroids were exposed to the single agents or combinations at concentrations up to the clinical Cmax value for each agent, if known. After seven days of exposure, cell viability was assessed using the CellTiter-Glo 3D assay and spheroid volume was assessed by bright field imaging. Results Several of the targeted oncology drugs exhibited additive and greater-than-additive cytotoxicity when combined with a CLK inhibitor, or the LSD1 inhibitor. These agents included the XPO1 inhibitor, eltanexor, and the KRAS G12D specific inhibitor MRTX-1133 which had activity in tumor lines harboring the KRAS G12D mutation. LSD1 inhibition was effective with ubiquitin proteasome pathway inhibitors. Conclusion: These findings may provide guidance for development of clinical trial combination regimens including cirtuvivint, CC-671 or iadademstat. Full data sets are available on PubChem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd17fd73c746ccc133073188d85cd1b611794f0" target='_blank'>
              RNA processing kinase inhibitors and epigenetic inhibitors in combination with oncology drugs or investigational agents in multi-cell type patient-derived tumor cell line spheroids
              </a>
            </td>
          <td>
            Beverly A. Teicher, T. Dexheimer, Thomas Silvers, N. Coussens, Eric M Jones, Steven D. Gore, Mark W. Kunkel, James H. Doroshow
          </td>
          <td>2025-09-12</td>
          <td>Cancer Chemotherapy and Pharmacology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Resistance to programmed cell death is a defining hallmark of cancer and a persistent barrier to successful therapy. Dual-function proteins such as p53, Ras, HIF-1α, BNIP3, and NF-κB act as molecular switches that determine cell fate between apoptosis and survival. In tumors, these proteins are deregulated not only by intrinsic mutations but also by extrinsic signals from the tumor microenvironment (TME). This Mini Review critically analyzes previous therapeutic approaches, emphasizing overlooked mechanisms such as Ras-mediated suppression of p53. It proposes a sequential therapeutic strategy: first, dismantling TME adaptations (hypoxia, inflammation, protective autophagy); second, inhibiting oncogenic Ras signaling; and third, restoring p53 activity. The phased approach integrates biomarker-guided patient stratification, recognizes tumor–microenvironment co-evolution, and highlights how resistance evolves over time. Although the concept does not resolve all challenges, it outlines a rational framework for restoring apoptotic competence and provides a pathway for translational and clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ed9bca532a19e58792704a40e38539d4ea37e5" target='_blank'>
              Context-dependent rewiring of dual-function proteins in cancer: a sequential strategy to restore apoptosis
              </a>
            </td>
          <td>
            Leon Strzadala
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Whole-genome duplication (WGD) represents an evolutionarily conserved process occurring in prokaryotes, eukaryotes, and somatic mammalian tissues. While developmentally programmed WGD supports normal tissue regeneration, unscheduled WGD drives chromosomal instability and oncogenic progression in cancer. Recent studies have clarified dual roles of WGD across physiological homeostasis and disease pathogenesis. Here, we review the prevalence of WGD, the molecular mechanisms driving its major causes and its biological consequences. In addition, we highlight recent advancements in WGD detection, including both conventional cytogenetic techniques and newly developed high-throughput sequencing approaches. The integration of multi-omics and machine learning further improves ploidy analysis, particularly in cancer research. Together, these insights establish WGD as a critical regulator of development, regeneration, and disease and underscore the importance of emerging computational and sequencing tools for its precise characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ebbe235d853294a695ca1852f4aab18a1bed1ef" target='_blank'>
              The dual role of whole-genome duplication: biological mechanisms, functional consequences, and detection advances.
              </a>
            </td>
          <td>
            Yawei Song, Jiajie Yang, Shuheng Wu, Wei Wu
          </td>
          <td>2025-09-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pediatric tumors such as neuroblastoma are characterized by a genome-wide ‘transcriptional burden’, surmising the involvement of multiple alterations of gene expression. Search for master regulators of transcription whose inactivation is lethal for tumor cells identified the non-POU domain-containing octamer-binding protein (NONO), a member of the Drosophila Behavior/Human Splicing family known for the ability to form complexes with macromolecules. NONO emerges as an essential mechanism in normal neurogenesis as well as in tumor biology. In particular, NONO interactions with RNAs, largely with long non-coding MYCN transcripts, have been attributed to the aggressiveness of neuroblastoma. Broadening its significance beyond MYCN regulation, NONO guards a subset of transcription factors that comprise a core regulatory circuit, a self-sustained loop that maintains transcription. As a component of protein–protein complexes, NONO has been implicated in the control of cell cycle progression, double-strand DNA repair, and, generally, in cell survival. Altogether, the pro-oncogenic roles of NONO justify the need for its inactivation as a therapeutic strategy. However, considering NONO as a therapeutic target, its druggability is a challenge. Recent advances in the inactivation of NONO and downstream signaling with small molecular weight compounds make promising the development of pharmacological antagonists of NONO pathway(s) for neuroblastoma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8911b531a2ccdbdcdabe70eb37ce9df87687fde" target='_blank'>
              Saying “Yes” to NONO: A Therapeutic Target for Neuroblastoma and Beyond
              </a>
            </td>
          <td>
            Sofya S. Pogodaeva, O. O. Miletina, Nadezhda V. Antipova, Alexander A. Shtil, O. Kuchur
          </td>
          <td>2025-10-03</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="All genetic variation fueling evolution depends on mutations. Although mutations have been extensively studied for almost a century, until a decade ago the investigation of mutations was limited to population-level analysis. This constraint has hampered the exploration of cellular heterogeneity in mutation processes and its evolutionary implications. To overcome these limitations, quantitative visualization methods for studying mutations in the bacterium Escherichia coli at the single-cell level have been developed. These approaches offer the possibility of accessing a major source of mutations, i.e., DNA polymerase errors, and their fate, i.e., repair versus conversion to mutation. In addition, such methods allow for quantitative characterization of the effects of mutations on cell fitness. This article discusses insights into the mutation process derived from these new single-cell mutagenesis approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8609c459912b06b1118e54bf7a6c19294215dd3c" target='_blank'>
              Mutagenesis: Insights from Single-Cell, Real-Time Approaches.
              </a>
            </td>
          <td>
            Marina Elez, Chiara Enrico Bena, Lydia Robert
          </td>
          <td>2025-09-16</td>
          <td>Annual review of microbiology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R. Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Through natural or synthetic lineage barcodes, single-cell technologies now enable the joint measurement of molecular states and clonal identities, providing an unprecedented opportunity to study cell fate and dynamics. Yet, most computational methods for inferring cell development and differentiation rely exclusively on transcriptional similarity, overlooking the lineage information encoded by lineage barcodes. This limitation is exemplified by T cells, where subtle transcriptional differences mark divergent fates with distinct biological activity. Single-cell RNA and matched TCR sequencing is now ubiquitous in the analysis of clinical samples, where the TCR sequence provides an endogenous clonal barcode and could reveal clonal T cell responses. We present Clonotrace, a computational framework that jointly models gene expression and clonotype information to infer cell state transitions and fate biases with higher fidelity. While motivated by challenges in analyzing T cell populations, especially in the tumor microenvironment and immunotherapy settings, Clonotrace is broadly applicable to any lineage-barcoded single-cell dataset. Across diverse systems including T cells, hematopoietic differentiation, and cancer therapy resistance models, Clonotrace reveals differentiation hierarchies, distinguishes unipotent from multipotent states, and identifies candidate fate-determining genes driving lineage commitment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b1a84299cf4bd86e16928f5b0911b15d09c6d9" target='_blank'>
              Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling
              </a>
            </td>
          <td>
            Yuntian Fu, D. Mathew, Mingshuang Wang, Xinyi E. Chen, Kevin Z Lin, Dylan Schaff, Sydney M Shaffer, Drew M Pardoll, Christina Jackson, Nancy R Zhang
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7188c988056f790ffe3551531682c3604b9aecad" target='_blank'>
              Integrative Gene-Centric Analysis of Breast Cancer: High-Confidence Germline Predisposition Genes from Population-Scale Cohorts
              </a>
            </td>
          <td>
            R. Zucker, S. Schreiber, A. Stern, M. Linial
          </td>
          <td>2025-09-12</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although purine metabolism is one of the most impacted pathways in colorectal cancer (CRC), little is known about the role of equilibrative nucleoside transporter 2 (ENT2) in CRC development and its association with the altered purine metabolism pathway. This study aimed to determine the role of ENT2 in altered purine metabolism in the early and late stages of CRC using CRISPR/Cas9 gene editing tools and a variety of functional experiments. The expression of ENT2 was significantly higher (P < 0.001) in all CRC cell lines as compared to the normal colon cells. The two CRC cell lines with the highest ENT2 expression, the early stage HT29 cells and the late stage DLD1 cells, were knocked out (KO) using the CRISPR/Cas9 tool. The hypoxanthine (HPX) level and the xanthine oxidase (XO) activity were significantly higher in both HT29/KO and DLD1/KO single cell-derived clones (P < 0.01). The increase in HPX level and XO activity were associated with an elevation in the reactive oxygen species (ROS) level. These data suggest that the ENT2 KO elevated the ROS levels induced apoptosis and impaired the cell proliferation of the early stage of CRC cell line, i.e., HT29/KO clonal cells. In this context, targeting ENT2 gene might be a potential strategy in CRC treatment by increasing the production of ROS and hence, inducing the apoptosis pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f2bfbf1ad7ece3cdb77ba1d7bef452cab9f9d5" target='_blank'>
              CRISPR/Cas9 mediated ENT2 gene knockout altered purine catabolic pathway and induced apoptosis in colorectal cell lines
              </a>
            </td>
          <td>
            Safaa M Naes, S. Ab-Rahim, M. Mazlan, S. Syafruddin, M. A. Mohtar, Asmaa Y Abuhamad, Amirah Abdul Rahman
          </td>
          <td>2025-08-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="In BRAF-mutant melanoma cells treated with inhibitors of the kinases BRAF and MEK, a subset of cells rapidly and nongenetically adapts to escape drug-induced quiescence and reenters the cell cycle. Here, we investigated the mechanisms enabling this drug escape by computationally reconstructing single-cell lineages from time-lapse imaging data, linking dynamic signaling pathways to distinct cell-cycle fate outcomes. We found that reactivation of the MEK substrate ERK was necessary but not sufficient to drive escape; rather, the activity of the protein complex mTORC1 was also required to promote cell growth and protein synthesis in drug-treated cells destined for cell-cycle reentry. ERK and mTORC1 signaling converged to increase the abundance of cyclin D1 protein, a critical bottleneck for cell-cycle commitment under drug pressure. In cells in which endogenous cyclin D1 was fluorescently tagged using CRISPR, the subset that escaped drug treatment exhibited marked accumulation of cyclin D1 at least 15 hours before cell-cycle reentry, enabling early prediction of future drug escape. Cyclin D1 thus represents both an early biomarker and potential therapeutic target for suppressing drug escape in melanoma. We observed a similar mTORC1-driven mechanism underlying escape in lung cancer cells, but not colon cancer cells, highlighting partial generalizability across cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a13ba89953abcf7b10acb1b3494af7221bf52b0b" target='_blank'>
              MAPK and mTORC1 signaling converge to drive cyclin D1 protein production to enable cell cycle reentry in melanoma persister cells.
              </a>
            </td>
          <td>
            Varuna Nangia, Humza M. Ashraf, Nasreen Marikar, Victor J. Passanisi, Christopher R Ill, Sabrina L Spencer
          </td>
          <td>2025-09-02</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Targeted therapy has been established as a therapeutic option for the treatment of metastatic melanoma. Despite initially being very efficient, many tumors develop resistance to targeted therapy, leading to its failure. We previously demonstrated that the neural crest (NC)-associated gene ERRFI1 is highly expressed in metastatic melanoma and correlates with a bad prognosis. Here, we show that the expression of ERRFI1 was upregulated in melanoma and negatively correlated with the expression of melanocytic differentiation markers, such as MITF and TYR. Downregulation of ERRFI1 with the help of siRNA increased the susceptibility of melanoma cells toward BRAF inhibition (BRAFi) and resensitized BRAFi-resistant melanoma cells to BRAFi. Mass spectrometry-based proteomic analysis revealed that ERRFI1 silencing diminished the activation of the mitogen-activated protein kinase (MAPK) and AKT signaling pathways, which usually contribute to drug resistance. Furthermore, we show that miR-200c targeted the 3'UTR of ERRFI1 and reduced its expression, resulting in the resensitization of BRAFi-resistant melanoma cells to BRAFi. Our study results suggest that ERRFI1 could be a potential therapeutic target for the treatment of metastatic melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a645774843921b1750028307066a9681c4de577" target='_blank'>
              The neural crest-associated gene ERRFI1 is involved in melanoma progression and resistance toward targeted therapy.
              </a>
            </td>
          <td>
            Nina Wang, Qian Sun, Daniel Novák, Lei Zhu, Juliane Poelchen, Tamara Steinfass, Yiman Wang, V. Umansky, Jochen Utikal
          </td>
          <td>2025-10-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The clonal evolution model provides a framework for understanding the evolution of cancer cells. According to this model, cancer cells accumulate genetic mutations over time, and these mutations are passed down to their descendants, leading to genetic diversity within the tumor. Some of these mutations confer selective advantages, causing certain lineages of cancer cells (clones) to dominate and expand. However, this model is rooted in certain conceptual assumptions, which we propose to revisit by considering the potential involvement of retrotransposons in cancer initiation and progression. In recent years, it has become evident that transposable elements, particularly retrotransposons, play a significant role in driving cancer transformation and progression. We first review how current knowledge about retrotransposon activity aligns with the clonal evolution model by highlighting its ability to modulate cancer cell fitness. We then take a forward-looking perspective to explore additional ways retrotransposons may also influence clonal dynamics beyond the current model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb2629152d1c164e115163464d0e7599e1333a7" target='_blank'>
              Revisiting Clonal Evolution Through the Light of Retrotransposons.
              </a>
            </td>
          <td>
            Anaïs Lamoureux, Emilie Elvira-Matelot, Françoise Porteu, L. Laplane
          </td>
          <td>2025-10-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Nucleoside analogues are used as drugs and as labels in laboratory-based research. However, the effect of different nucleoside analogue mechanism(s) on cell sensitivity or mutagenesis is unclear. This is particularly important in cancer treatments where checkpoint proteins and DNA damage factors are often mutated. We tested six nucleoside analogues in fission yeast, Schizosaccharomyces pombe. We found that the mutations in the DNA replication checkpoint cause unique sensitivity profiles towards chemotherapeutic nucleoside analogues (gemcitabine, 5-fluorouracil, cytarabine) and the non-clinical analogue bromodeoxyuridine. Antiretroviral compounds, zidovudine and lamivudine, did not alter cell growth. We compared half-maximal inhibitory concentration (IC50) doses between checkpoint deficient yeast strains, examining culture growth and DNA mis-segregation. Intriguingly, gemcitabine and bromodeoxyuridine doses above the IC50 promoted better growth. Above each compound’s IC50 dose we saw that cells were insensitive to nucleoside analogue re-exposure, particularly in DNA replication checkpoint mutants (cds1∆, rad3∆). Thus, pairing nucleoside analogue use with personal genomics may inform drug choice, dose, and schedule. Finally, these data indicate that resistance may be predictable, informing clinical strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea9673674f99d856da45d7531d1ed654d61e6d4f" target='_blank'>
              Checkpoint-Dependent Sensitivities to Nucleoside Analogues Uncover Specific Patterns of Genomic Instability
              </a>
            </td>
          <td>
            Zainab Burhanuddin Kagalwala, Mohammed Ayan Chhipa, Zohreh Kianfard, Essam Karam, Sirasie P. Magalage, Sarah A. Sabatinos
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="INTRODUCTION
Multiple myeloma (MM) is an incurable clonal B-cell malignancy characterized by the accumulation of neoplastic plasma cells in the bone marrow (BM). Many pieces of evidence indicate that hypoxia promotes MM progression, but the underlying mechanisms are not well known.


METHODS
We analyzed gene expression profiles of 3 MM cell lines under hypoxia and the MMRF CoMMpass project. We validated the expression patterns of hypoxia-associated genes (HAGs) in CD138+ BM cells from MM patients at different stages. Single-cell RNA sequencing data were used to analyze the performance of HAGs in the BM microenvironment.


RESULTS
We identified 17 HAGs differentially expressed in three MM cell lines under hypoxia. While in the MMRF project, we identified 92 differentially expressed HAGs in newly diagnosed MM patients. MM cell lines and the MMRF project shared 9 HAGs, including ADM, BNIP3L, EGLN1, FAM162A, HMOX1, PDK1, PLOD1, STAT5B, and TFRC. Notably, 8 of them were significantly associated with the overall survival of MM patients, and 6 were significantly associated with the MM patient survival in the first year after diagnosis. Then, hypoxia pressure scores calculated using these genes displayed significant differences between MM patients and healthy individuals. Further, we validated the expression patterns of HAGs using another cohort data and performed qRT-PCR using our own samples, and the results confirmed severe hypoxia existed in plasma cells and other cell types of the BM microenvironment of MM patients compared to healthy individuals.


CONCLUSION
Taken together, our findings may contribute to the treatment and prognosis prediction of MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9473a008423522556ea8e825486e97240efc7ead" target='_blank'>
              Transcriptome Analysis Identifies Functional and Prognostic Hypoxia-Associated Genes in Multiple Myeloma.
              </a>
            </td>
          <td>
            Lijia Hou, Jing Zhang, Q. Ran, Zhongjun Li, Maoshan Chen
          </td>
          <td>2025-09-26</td>
          <td>International journal of laboratory hematology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Ionizing radiation can lead to breaks in the double-stranded structure of DNA. Although cells have mechanisms that can repair such breaks, misrejoining events can occur that lead to mutations in an individual’s genome. When mutations affect certain genes, such as oncogenes and tumor-suppressor genes, cancer can result. Although ionizing radiation is a known cause of cancer, it has been difficult to determine how much of an excess cancer risk results from low-dose radiation exposure. Radiation signatures that distinguish radiation-induced cancers from spontaneous cancers are thus useful for assessing cancer risks after exposure to low-dose radiation. Recent advances in high-throughput sequencing technologies have enabled the comprehensive profiling of mutations in individual cancer cells. Analyses of radiation-associated cancer genomes have identified an increase in small deletions and chromosome rearrangements, including large deletions, inversions, and translocations, which are likely to be formed by the misrejoining of DNA double-strand breaks. Among these, chromosome rearrangements leading to cancer-related mutational events are potentially useful as biomarkers and as therapeutic targets for treating radiation-induced cancer. This review summarizes research on radiation-induced mutational signatures found in animal and human cancers, consolidates current knowledge, and discusses the future perspectives in this research field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33c28820d79dcb3eabee840cf5431bbdeaec073" target='_blank'>
              Mutational Signatures in Radiation-Induced Cancer: A Review of Experimental Animal and Human Studies
              </a>
            </td>
          <td>
            K. Daino, C. Tsuruoka, Atsuko Ishikawa, S. Kakinuma, T. Imaoka
          </td>
          <td>2025-08-29</td>
          <td>Biology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="DNA base editing is increasingly used for human genetic modification, but methods for monitoring off-target editing are nascent. Here we present a simple model-independent workflow for identifying sites of off-target base-editing in relevant cell types on a genome-wide level. We report that sites of off-target editing by the ABE8e editor could be identified using an ABE8e derivative with restored DSB cleavage activity. This allows marking of enzyme-generated double-stranded (ds) DNA breaks by incorporation of dsDNA oligonucleotides that are transfected into primary target cells. DNA sequencing at sites of oligonucleotide incorporation reported both the genomic location of off-target cleavage and the extent of base-editing nearby. We present a platform combining this cellular (BEiGUIDE-Seq) and computational workflow (CRISPRito) to generate optimized amplicon panels for convenient monitoring of off-target base editing. This work introduces a generalizable strategy to evaluate off-target edits in patient-derived cells, addressing a critical safety gap for clinical base editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39cf7d75780ca966f04d4549c86bf384c245b3c1" target='_blank'>
              An integrated enzymatic and computational pipeline for quantifying off-target base-editing
              </a>
            </td>
          <td>
            Alexander G. McFarland, Soon-Keat Ooi, Davin Tafuri, Elahe Kamali, Natalie Cooper, Emma J Cook, Shantan Reddy, Danuta Jarocha, Nils Wellhausen, Saar Gill, A. Roche, J. Fraietta, Frederic D. Bushman, Friederike Herbst
          </td>
          <td>2025-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is an aggressive ovarian cancer with a heterogeneous tumor microenvironment (TME). Advances in single-cell RNA sequencing (scRNA-seq) and spatially-resolved transcriptomics have enabled the study of complex TME. This study explores connections between molecular subtypes described from bulk transcriptomes and spatial domains characterized by distinct gene expression in HGSC and their variability between patients. We explored both intra- and inter-tumor heterogeneity across 2D space and identified differing spatial patterns of gene expression pertaining to immune pathways and vasculature development. Functional characterization of tumor spaces revealed potentially shared cell states across molecular subtypes, while correlation analysis underscored subtype-specific spatial anti-colocalization between spots exhibiting antigen-presenting functions and B cell-mediated immunity. Lastly, we performed spatially-aware cell-cell communication analysis on the spatial samples and identified a molecular subtype specific difference in total signaling activity and heterogeneity in Midikine signaling between the differentiated subtype. Our results suggest that generating multiple tissue slices per patient might be necessary to enable comprehensive characterization of HGSC spatial transcriptomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12564a0c02e3362085c018e2a380154da3a01959" target='_blank'>
              Characterizing intra- and inter-tumor heterogeneity in Ovarian high-grade serous carcinoma subtypes using single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Weishan Li, Laurie Grieshober, Jason Gertz, Adriana Ivich, J. Doherty, Casey S. Greene, SC Hicks
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Method Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Results Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). Conclusion This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor immunity has emerged as a focal point for cancer research. Although tumor immunotherapy represents a promising approach to cancer treatment, its effectiveness is often hindered by the heterogeneity of the tumor microenvironment (TME) and immune escape mechanisms. In recent years, small non-coding RNAs (sncRNAs) have attracted increasing attention in tumor immunology due to their essential role in gene regulation. This review systematically examines the multifaceted regulatory functions of sncRNAs in tumor immunity, with a focus on six major subtypes: microRNA, siRNA, piRNA, snoRNA, tsRNA, and snRNA. The molecular mechanisms by which these sncRNAs reshape the TME are discussed, including their roles in modulating immune cell differentiation (e.g., T cell polarization, macrophage phenotype transition), regulating immune checkpoint expression (PD-1/PD-L1, CTLA-4, Tim-3, LAG-3), and influencing tumor antigen presentation. This review also explores the dynamic network through which sncRNAs contribute to tumor immune escape. Furthermore, this study highlights the clinical potential of sncRNAs as liquid biopsy biomarkers and their application prospects in therapeutic strategies, such as targeted silencing of immunosuppressive molecules via nano-delivery systems, combination treatments with radiotherapy and chemotherapy, and Chimeric Antigen Receptor T-cell (CAR-T) therapy. Despite current challenges, including limited delivery efficiency and off-target effects, emerging technologies like AI-assisted sequence design and organ-on-a-chip models present new opportunities for clinical translation. This comprehensive review provides a theoretical foundation and translational insights for elucidating the functional network of sncRNAs in tumor immunology and advancing precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7dcee9c888595b1a7bd3be0ebd8ff37c9ea4926" target='_blank'>
              Small non-coding RNAs: key regulatory factors and potential therapeutic targets in tumor immunity
              </a>
            </td>
          <td>
            Zihan Liu, Haotian Dong, Chengyuan Ye, Jianing Yan, Min Miao, Y. Shao
          </td>
          <td>2025-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Despite therapeutic advances, metastatic melanoma, and particularly brain metastasis (MBM), remains a lethal burden for patients. Existing single-cell studies offer a more detailed view of melanoma and its microenvironment, which is crucial to improve diagnosis and treatment. Results We here present a computational reanalysis of single-nucleus data comparing 15 MBM and 10 extracranial melanoma metastases (ECM), considering recent best practice recommendations. We used cell type-specific pseudobulking and omit imputation during patient integration to gain complementary insights. Interestingly, our analysis revealed high homogeneity in tumor cell expression profiles within and between MBM and ECM. However, MBM displayed even higher homogeneity but a more flexible energy metabolism, suggesting a specific metastatic adaptation to the putatively more restricted brain microenvironment. While tumor cells were homogeneous, the metastasis microenvironment, especially lymphocytes and related immune-tumor interaction pathways, exhibited greater divergence between MBM and ECM. Overall, this suggests that major differences between MBM and ECM are potentially driven by variations in their microenvironment. Finally, a comparison of single-cell data to previous bulk studies, including their deconvoluted putative cell types, showed significant differences, potentially causing divergent conclusions. Conclusion Our study contributed to refine the understanding of differences between MBM and ECM, suggesting these are potentially more influenced by their local microenvironments. Future research and therapies could possibly focus on the metabolic flexibility of melanoma brain metastases and patient-specific immune pathway alterations. Supplementary Information The online version contains supplementary material available at 10.1186/s13062-025-00691-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e755d874cd5c0ea8aa714bc1a90cc86adaa7c4ba" target='_blank'>
              Single-cell transcriptome analysis suggests cells of the tumor microenvironment as a major discriminator between brain and extracranial melanoma metastases
              </a>
            </td>
          <td>
            Konrad Grützmann, Michael Seifert
          </td>
          <td>2025-09-16</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genetic variants associated with autoimmune diseases are highly enriched within putative cis-regulatory regions of CD4+ T cells, suggesting that they could alter disease risk through changes in gene regulation. However, very few genetic variants have been shown to affect T cell gene expression or function. Here we tested >18,000 autoimmune disease-associated variants for allele-specific effects on expression using massively parallel reporter assays in primary human CD4+ T cells. We find 545 variants that modulate expression in an allele-specific manner (emVars). Primary T cell emVars greatly enrich for likely causal variants, are mediated by common upstream pathways and their putative target genes are highly enriched within a lymphocyte activation network. Using bulk and single-cell CRISPR-interference screens, we confirm that emVar-containing T cell cis-regulatory elements modulate both known and previously unappreciated target genes that regulate T cell proliferation, providing plausible mechanisms by which these variants alter autoimmune disease risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3e805b68e0ffd72ffd76d368fd9a6bf4ce73ef1" target='_blank'>
              Genetic and epigenetic screens in primary human T cells link candidate causal autoimmune variants to T cell networks
              </a>
            </td>
          <td>
            Ching-Huang Ho, Maxwell A. Dippel, Meghan S. McQuade, LeAnn P. Nguyen, Arpit Mishra, Stephan Pribitzer, Samantha Hardy, Harshpreet Chandok, Florence M. Chardon, Troy A. McDiarmid, Hannah A. DeBerg, Jane H. Buckner, Jay Shendure, Carl G. de Boer, Michael H. Guo, Ryan Tewhey, John P. Ray
          </td>
          <td>2025-09-18</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Ovarian cancer (OvCa) remains largely refractory to immune checkpoint blockade (ICB), owing to a profoundly immunosuppressive tumor microenvironment (TME). Single-cell RNA sequencing (scRNA-seq) of patient tumors has revealed a TME enriched in immunosuppressive macrophages and characterized by exclusion of T cells from the tumor core, features strongly associated with ICB resistance. However, the molecular mechanisms underlying the establishment of this immune landscape remain poorly understood. To identify regulators of OvCa immunity, we employed Perturb-map, a first-of-its-kind spatial functional genomics platform that enables multiplexed CRISPR/Cas9 knockout of candidate genes while preserving the spatial context of tumor–TME interactions in situ. We targeted 35 genes, prioritized through ligand–receptor interaction analysis of human OvCa scRNA-seq data, in an immunocompetent OvCa mouse model. Among these, IL-4 emerged as a key driver of ICB resistance. While IL-4 knockout did not affect tumor growth in untreated animals, it substantially remodeled the local TME and rendered tumors responsive to anti-PD-1 therapy. Unexpectedly, we found that both mouse and human ovarian cancer cells are a source of IL-4, a cytokine traditionally restricted to immune cells. Analysis of patient tumors revealed that tumor-associated macrophages exhibit a robust IL-4, induced transcriptional signature, implicating IL-4 in shaping the immunosuppressive TME in human disease, consistent with our preclinical findings (Cell, 2024). Furthermore, we observed that standard-of-care chemotherapy regimens, including carboplatin, paclitaxel, and cisplatin—induce a dose-dependent increase in IL-4 expression in both murine and human OvCa cell lines. This chemotherapy-induced IL-4 upregulation may contribute to immune evasion and recurrence following initial therapeutic response. Therapeutically, combined blockade of IL-4Rα and PD-1 significantly delayed tumor progression in vivo compared to monotherapy, suggesting that IL-4 signaling is both a necessary and targetable mechanism of immune resistance. These findings are clinically actionable, as Dupilumab, an FDA-approved anti-IL-4Rα antibody, is already in clinical use and well-tolerated. Our results uncover a previously unrecognized, chemotherapy-induced immunoregulatory axis in ovarian cancer and provide a strong rationale for therapeutic strategies that combine IL-4 pathway inhibition with ICB to overcome resistance and improve patient outcomes.



 Gurkan Mullaoglu, Hunter Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian Brown, Alessia Baccarini. Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/462188b4ccc770fdbf7b28c6a3fda711e0408972" target='_blank'>
              Abstract PR015: Tumor-derived IL-4 links chemotherapy to immunotherapy resistance in ovarian cancer
              </a>
            </td>
          <td>
            Gurkan Mullaoglu, Hunter T. Potak, Paula Guerrero Gonzalez, Gardner Davlin, Joshua Leinwand, Dmitriy Zamarin, Brian D. Brown, Alessia Baccarini
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Metastatic lung carcinoma poses considerable treatment difficulties. It exhibits cellular characteristics that differ from those of early-stage cancer. Immunotherapy demonstrates enhanced hope for patients with advanced and metastatic lung cancer. Tumor-associated macrophages (TAMs) may be a contributing factor that diminishes the effectiveness of immunotherapy. Secreted phosphoprotein 1 (SPP1)+ TAMs are considered to possess immunosuppressive characteristics, as their interaction with CD8+ T lymphocytes results in the exhaustion of these cells. Methods We analyzed single-cell RNA sequencing datasets of lung cancer metastases in order to examine the development of brain metastatic tumors. Furthermore, the cell–cell interactions between SPP1+ TAM cells and CD8+ T cells within primary and brain metastatic cancers were systematically compared. Additionally, we utilized a flow cytometer and immunofluorescence to demonstrate how SPP1 affects the function of CD8+ T cells. In vitro, we generated SPP1-overexpressing macrophages and performed qPCR, Western blot, and co-culture assays with or without anti-SPP1 or anti-A2AR treatment to evaluate immunosuppressive effects. Results The results demonstrated that the proportions of immune cells in metastatic brain tissues are lower, while the infiltration of macrophages is higher. SPP1+ TAMs contribute to immune suppression in lung cancer by limiting the activation of CD8+ T cells and cytokine production. Anti-SPP1 treatment positively impacts CD8+ T cell function, counteracting SPP1-induced dysfunction and facilitating the production of cytokines. SPP1 overexpression in macrophages enhanced their immunosuppressive phenotype by upregulating CD73 and cytokines such as IL-10 and TGF-β, leading to impaired CD8⁺ T cell function via A2AR signaling; notably, neutralization of SPP1 or A2AR successfully restored CD8⁺ T cell activity. Conclusions Our findings characterize the immunological environment of both primary and metastatic lung cancer, highlighting SPP1-mediated immune suppression as a potential therapeutic target to restore T cell responses. Preclinical data demonstrate that Anti-SPP1 antibodies can reverse T cell exhaustion and enhance immune responses in lung cancer. Supplementary Information The online version contains supplementary material available at 10.1007/s00262-025-04180-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659292b315a12f27348d32dba8f9ec9a4a03cedd" target='_blank'>
              Single-cell transcriptomic analyses provide insights into SPP1+ TAM-mediated immune suppression and CD8+ T cell dysfunction in lung cancer
              </a>
            </td>
          <td>
            Rahma Taher Almgrami, Tengyue Zhang, Qitai Zhao, Mingming You, Jinyan Liu, Yi Zhang
          </td>
          <td>2025-09-29</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer is the second leading cause of death worldwide, with an estimated 27.5 million new cases projected by 2040. Disruptions in cell cycle control cause DNA replication errors to accumulate during cell growth, leading to genomic instability and tumor development. Proteins that regulate cell cycle progression and checkpoint mechanisms are crucial targets for cancer therapy. NIMA-related kinases (NEKs) are a family of serine/threonine kinases involved in regulating various aspects of the cell cycle and mitotic checkpoints in humans. Among these, NEK10 is the most divergent member and has been associated with both cancer and ciliopathies, a group of disorders caused by defects in cilia structure or function. Despite its biological significance and distinctive domain architecture, the structural details of NEK10 remain largely unknown. To address this gap, we employed computational modeling techniques to predict the complete structure of the NEK10 protein. Our analysis revealed a catalytic domain flanked by two coiled-coil domains, armadillo repeats (ARM repeats), an ATP binding site, two putative ubiquitin-associated (UBA) domains, and a PEST sequence known to regulate protein degradation. Furthermore, we mapped a comprehensive interactome of NEK10, uncovering previously unreported interactions with the cancer-related proteins MAP3K1 and HSPB1. MAP3K1, a serine/threonine kinase and E3 ubiquitin ligase frequently mutated in cancers, interacts with the catalytic region of NEK10. The interaction with HSPB1, a molecular chaperone associated with poor cancer prognosis, is mediated by NEK10's ARM repeats. Our findings highlight a potential connection between NEK10, ciliogenesis, and cancer, suggesting an important role in cancer development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84bfc7864763ec8c742e96b5b16848a5dc021a94" target='_blank'>
              In Silico Analysis of Human NEK10 Reveals Novel Domain Architecture and Protein-Protein Interactions.
              </a>
            </td>
          <td>
            A. S. Eichner, Nathaniel Zimmerman, Avdar San, Shaneen Singh
          </td>
          <td>2025-10-03</td>
          <td>Proteins</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of “signatures”, one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines [1–3]. The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a “funnel”, through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0289ae32bcbca4304d80ec26b288975aaa0f5e9" target='_blank'>
              Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature
              </a>
            </td>
          <td>
            Natanael Spisak, Marc de Manuel, Molly Przeworski
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a threat to public health, with a global incidence and mortality of over 1.9 million and 0.9 million people, respectively. Anti-epidermal growth factor receptor (EGFR)-based treatment is recommended for CRC with wild-type rat sarcoma (RAS). However, after continuous treatment with this regimen, acquired resistance occurs, which hampers the prognosis of patients. The median progression-free survival of patients with metastatic CRC receiving first-line anti-EGFR-based treatment is 10–13 months. The widely recognized mechanisms that induce acquired anti-EGFR resistance are related to the activation of the RAS/RAF/MEK/ERK and PI3K/AKT/mTOR signaling pathways. In addition, novel mechanisms that induce acquired resistance, such as microsatellite stability/mismatch repair status, noncoding RNAs, the tumor microenvironment, exosome-mediated intracellular communication, and post-transcriptional modification, are being discovered. To improve personalized medication, biomarkers with predictive value for acquired anti-EGFR resistance are recognized from both tumoral samples and liquid biopsies. On the basis of the identified mechanisms, clinicians have developed several treatment strategies to cope with acquired anti-EGFR resistance. This review provides an overview of acquired anti-EGFR resistance in RAS wild-type CRC by summarizing the common genes and proteins, potential novel mechanisms, and predictive biomarkers related to acquired anti-EGFR resistance, as well as treatment strategies to address this resistance. This review may serve as a potential reference for exploring possible treatment strategies for acquired anti-EGFR resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce156c01d06746945c1d8cd107bf601f31353493" target='_blank'>
              An overview of mechanisms, biomarkers, and treatment strategies for acquired anti-EGFR resistance in RAS wild-type colorectal cancer
              </a>
            </td>
          <td>
            Chenxiang Ding, Ruoyu Yao, Yong Wang, Hongjie Liu, Chunyan Li, Hua Xie
          </td>
          <td>2025-09-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Mechanisms of defense against nucleic acids are central to maintaining genome integrity and have evolved to sense multiple nucleic acids alterations in structure and/or location within a cell. In cancer, dysregulation of DNA repair pathways and gene transcription are common and are associated with cellular stress and innate immune signaling, thus promoting immune surveillance. Subversion of such responses is often required for a cancer to evade immune control and progress to become clinically significant. One example is the epigenetic silencing of stimulator of interferon genes (STING), which is activated by cGAMP produced by the cyclic GMP-AMP synthase (cGAS) after sensing cytosolic DNA. Importantly, therapeutic interventions, such as radiation therapy, can leverage these hard-wired nucleic acids defense pathways to induce anti-tumor immune responses. Extensive preclinical data demonstrate that induction of interferon type I (IFN-I) in cancer cells and the tumor microenvironment by radiation promotes antigen presentation and recruitment of T cells and plays a central role in the ability of radiation to synergize with immune checkpoint inhibitors. In the clinic, however, such effects of radiation have not been consistently observed in unselected patient populations. This highlights the need for an improved understanding of the complex cross-talk between radiation-induced DNA damage response and innate immune signaling, and their alterations in cancer. We are currently investigating the interplay between cGAS/STING pathway and the ectonucleotidases that convert pro-inflammatory signals generated by radiation into immunosuppressive adenosine. The latter offers candidate actionable targets to improve radiation-induced immune activation. Overall, insights into the fundamental mechanisms that couple DNA damage with innate immune signaling will enable the fine tuning of therapeutic interventions that promote anti-tumor immune responses.



 Sandra Demaria. Therapeutic modulation of nucleic acids-induced immune activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Mechanisms of Cancer Immunity and Cancer-related Autoimmunity; 2025 Sep 24-27; Montreal, QC, Canada. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(9 Suppl):Abstract nr IA03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a77876b58141521d365b58e7010c04c24fa79e35" target='_blank'>
              Abstract IA03: Therapeutic modulation of nucleic acids-induced immune activation
              </a>
            </td>
          <td>
            Sandra Demaria
          </td>
          <td>2025-09-24</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Polymerase theta (POLQ) is essential for theta-mediated end joining (TMEJ), an error-prone DNA repair pathway that is hyperactive in homologous recombination-deficient (HRD) cancers such as high-grade serous ovarian cancer (HGSOC). Inhibiting POLQ is a promising therapeutic strategy, but the molecular determinants of sensitivity and downstream mechanisms remain incompletely defined.



 We assessed the growth-inhibitory activity of two structurally distinct POLQ inhibitors, novobiocin and ART558, across 18 well-characterized HGSOC cell lines. Genomic profiling data were integrated with drug response values (GR50s) to identify predictive biomarkers, including mutational status, HRD scores, and copy number signatures (CX1–CX17). Mechanistic assays included POLQ ATPase activity, cell cycle analysis (FUCCI), immunofluorescence for γH2AX and POLQ, apoptosis markers and DNA fiber assays.



 POLQ inhibitor sensitivity varied independently from olaparib or carboplatin response, indicating a distinct mode of action. Novobiocin sensitivity significantly correlated with CX3 and CX5 copy number signatures, which reflect impaired homologous recombination and replication stress. In contrast, resistant lines were enriched for CX14 and CX6, signatures linked to mitotic errors and telomere shortening. Sensitive cell lines showed increased DNA damage (γH2AX foci), POLQ accumulation on chromatin, and apoptosis, particularly in BRCA1/2-mutant backgrounds. Cell cycle profiling revealed impaired G1/S progression and S-phase depletion. DNA fiber assays demonstrated replication fork degradation upon POLQ inhibition, consistent with POLQ’s role in fork protection.



 We identify genomic features, specifically CX3 and CX5 copy number signatures as candidate biomarkers of POLQ inhibitor sensitivity in HGSOC. Mechanistically, POLQ inhibition induces replication stress, persistent DNA damage, and apoptosis in HR-deficient settings. These findings support the clinical development of POLQ inhibitors and rational patient stratification based on genome instability signatures.



 Aleksandra Kruglikov. POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe824b1195032b879fa6dbdb3ed12cc08d24058" target='_blank'>
              Abstract A037: POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response
              </a>
            </td>
          <td>
            Aleksandra Kruglikov
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The BRAF protein regulates cell growth and division through key signaling pathways. Mutations in BRAF, particularly the V600E variant, are frequently observed in colorectal cancer (CRC) and are associated with poor prognosis and therapeutic challenges. Tumors harboring certain BRAF mutations often exhibit primary resistance to BRAF inhibitor monotherapies. Over time, these tumors can also develop acquired resistance, further complicating treatment. In this study, we employed replica exchange molecular dynamics simulations combined with machine learning techniques to investigate the structural alterations induced by BRAF mutations and their contribution to drug resistance. Our analyses revealed that conformational changes in mutant BRAF proteins associated with dabrafenib residues psi494, phi600, phi644, phi663, psi675, and phi677 were sufficient for classifying drug-resistant vs. drug-sensitive variants. Similarly, for vemurafenib, residues psi450, phi484, phi495, phi518, psi622, and phi622 were the key residues that influence drug binding and resistance mechanisms. These residues are located in the N-lobe of CR3, which is responsible for ATP binding and the regulation of BRAF kinase activity. These findings offer deeper insights into the molecular basis of BRAF-driven resistance and provide predictive models for phenotypic outcomes of various BRAF mutations. The study underscores the importance of targeting specific BRAF variants for more effective, personalized therapeutic strategies in drug-resistant CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcf6858b6f2686bd3d3197f896e957a981c35dc" target='_blank'>
              Combining Molecular Dynamics and Machine Learning to Predict Drug Resistance Causing Variants of BRAF in Colorectal Cancer
              </a>
            </td>
          <td>
            Longsheng Xie, Christopher Lockhart, Dmitri K Klimov, M. S. Jafri
          </td>
          <td>2025-08-30</td>
          <td>Molecules</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genomic instability is a hallmark of aging and cancer. A key contributor to genomic instability includes alternative DNA structures, such as cruciform-forming inverted repeats (IRs). Short IRs (< 100 bps) are abundant in the human genome, mutagenic, and enriched at mutation hotspots in human cancer genomes. Using an innovative mutation-reporter mouse model, we showed that short IRs are mutagenic in vivo. Further, we found that aging exacerbates IR-induced genomic instability, as evidenced by increased mutation frequencies and altered spectra in the spleen and brain of mice harboring either a short IR or control B-DNA sequence at 2 and 24 months of age. These findings establish a link between aging and enhanced mutagenesis at short IRs, providing a unique in vivo platform to investigate age-related mechanisms of DNA structure-mediated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdc4812813b7511aa9fae9d1c0a6308731d4e5f" target='_blank'>
              Aging Increases Short, Inverted Repeat-Mediated Genomic Instability In Vivo.
              </a>
            </td>
          <td>
            Pooja Mandke, Pallavi Kompella, Guliang Wang, Karen M. Vasquez
          </td>
          <td>2025-09-30</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258c585d858690cd52a8f16499c3f52ebcd32f02" target='_blank'>
              Genetic regulation of cell type-specific chromatin accessibility shapes immune function and disease risk
              </a>
            </td>
          <td>
            A. Xue, J. Fan, O. A. Dong, H. L. Huang, P. C. Allen, E. Spenceley, E. Sagi-Zsigmond, B. Bowen, A. Cuomo, A. Henry, H. Tanudisastro, Z. Qiao, L. Chen, E. Ben-David, K. Farh, D. Neavin, A. S. Lee, A. Senabouth, C. Bartie, R. A. McCloy, V. Chin, K. M. de Lange, G. Figtree, A. W. Hewitt, D. G. MacArthur, J. E. Powell
          </td>
          <td>2025-08-29</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary This study found that RBMX is highly expressed in human osteosarcoma. Through animal models combined with single-cell transcriptome sequencing analysis, it was discovered that knockout of RBMX significantly enhanced the infiltration of immune cells in the tumor microenvironment, suggesting that intervening in the expression of RBMX may help make tumors more likely to respond to immunotherapy. This study reveals that CD8+T cells may be important cells regulated by RBMX. Further experiments revealed that RBMX regulates the expression of H2-K1 and THBS1, driving the depletion of CD8+T cells in osteosarcoma through cell communication. This study may provide potential targets for reshaping the immune microenvironment of osteosarcoma and improving the therapeutic effect of osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/703a197b1ec112a245ad6c288dc9ed70bf8339fb" target='_blank'>
              The RNA-Binding Protein RBMX Mediates the Immunosuppressive Microenvironment of Osteosarcoma by Regulating CD8+T Cells
              </a>
            </td>
          <td>
            Yu Qiu, Chao Pu, Chengguang Wang, Zhengxue Quan
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 BRCA1, BRCA2, and PALB2 mutations frequently involve shifting of the gene reading frame, resulting in the production of premature stop codons and truncated proteins. Frequently, the latter are subject to misfolding and proteasomal degradation. We have found that in response to prolonged PARP inhibitor (PARPi) treatment, multiple common pathogenic BRCA1 mutations use post-transcriptional mechanisms to produce truncated and hypomorphic protein isoforms. BRCA1 protein isoforms could induce PARPi resistance when expressed at sufficiently high levels. Moreover, we discovered several BRCA1 genetic alterations, such as secondary splice site mutations, and Alu-mediated gene rearrangements, which result in the production of protein isoforms that avoid misfolding and promote PARPi resistance. Finally, we are currently assessing the impact of PALB2 hypomorphic proteins on cancer initiation and will discuss recent results.



 Neil Johnson. BRCA mutations and protein products in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86b875969d2d788ca87a548ad50a77f9264de3b1" target='_blank'>
              Abstract IA022: BRCA mutations and protein products in cancer
              </a>
            </td>
          <td>
            Neil Johnson
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor-associated macrophages (TAMs) display remarkable functional heterogeneity, yet the molecular mechanisms driving their diverse phenotypes remain elusive. Using CRISPR screens in primary macrophages, we identified tumor-derived factors, including lactic acid, PGE2, and GM-CSF, as key modulators of TAM polarization. These factors interact synergistically and antagonistically to shape distinct TAM phenotypes that are highly conserved across human cancers. Mechanistically, lactic acid and PGE2 jointly induce angiogenic gene programs while suppressing GM-CSF-driven MHC-II expression at the chromatin level, creating mutually exclusive distributions of proangiogenic and MHC-II+ TAMs, which are differentially localized to specific spatial niches in the tumor microenvironment. Furthermore, we showed that shifting TAMs to an interferon-responsive phenotype, triggered by Adar inactivation, significantly promotes the infiltration of effector CD8+ T cells through specific receptor-ligand interactions. These findings uncover a conserved mechanism of TAM polarization and offer insights into therapeutic strategies for TAM reprogramming to potentiate cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cb6bdb7d04c92080704c1d38a2b724c54141d29" target='_blank'>
              Functional genetic screens reveal key pathways instructing the molecular phenotypes of tumor-associated macrophages.
              </a>
            </td>
          <td>
            Youxue Lu, Ce Luo, Lanxiang Huang, Genyi Wu, Lihan Zhong, Jieyu Chu, Fubing Wang, Zexian Zeng, Deng Pan
          </td>
          <td>2025-09-04</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 African Americans (AAs) have higher incidence rates of advanced lung cancer when compared to European Americans (EAs). DNA double-strand breaks (DNA DSBs) are the major cause of genomic instability that initiates cancer. Our previous work found AA lung cancer patients have more germline DNA DSB repair deficiencies than EAs, suggesting unrelated AAs may share genomic risk factors for developing lung cancer. We hypothesize African ancestry drives disease-causing alterations in DNA DSB repair pathway genes.



 ClinVar criteria were used to profile germline pathogenic variants (PVs) across the four major DNA DSB repair pathways: homologous recombination repair (HRR), classical non-homologous end joining (c-NHEJ), alternative end-joining (alt-EJ), and single-strand annealing (SSA). The discovery cohort consisted of populations of African (n=37,545) and European (n=622,057) descent in the Genome Aggregation Database. The validation cohort included study participants in the NIH All of Us Research Program (African, n=50,080) and (European, n=125,860). In silico RNA (SpliceA1, Pangolin, and PhyloP) and protein (VEP and CADD) annotation tools were used to predict functional consequences.



 In the African ancestry populations, 20 PVs across eight HRR genes, nine PVs across six c-NHEJ genes, six PVs across three alt-EJ genes, and 14 PVs across five SSA genes we discovered. In comparison, 115 PVs across 11 HRR genes, 52 PVs across eight c-NHEJ genes, 43 PVs across six alt-EJ genes, and 113 PVs across seven SSA genes were observed in the European ancestry populations. Interestingly, a RAD51 PV was exclusively enriched in African ancestry and not European ancestry populations for the HRR pathway. By contrast, NHEJ1, EXO1 and DCLRE1C PVs were uniquely present in those with European ancestry and not in individuals with African ancestry for the alt-EJ and SSA pathways, respectively. Frameshift mutations were the most common functional protein change in both populations.



 A total of 32 germline PVs across 13 DNA DSB repair genes were observed in African ancestry populations only, suggesting genetic ancestry may drive DNA repair deficiencies that increase susceptibility to developing lung cancer.



 Musinu Zakari, Egbenoma A. Aigboeghian, Khadijah A. Mitchell. Germline pathogenic variant profiling of DNA double-strand break repair pathways in populations with African and European genetic ancestry [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C077.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11a81bc23bab29ba6a3aa864a49c700d5fec40e" target='_blank'>
              Abstract C077: Germline pathogenic variant profiling of DNA double-strand break repair pathways in populations with African and European genetic ancestry
              </a>
            </td>
          <td>
            Musinu Zakari, Egbenoma A. Aigboeghian, Khadijah A. Mitchell
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2af718684499511487fc4ba4f7a1a788d70c340" target='_blank'>
              Somatic mutation and selection at population scale.
              </a>
            </td>
          <td>
            A. Lawson, Federico Abascal, Pantelis A. Nicola, Stefanie V. Lensing, A. Roberts, Georgios Kalantzis, Adrian Baez-Ortega, Natalia Brzozowska, J. E. El-sayed Moustafa, Dovile Vaitkute, Belma Jakupovic, A. Nessa, Sam Wadge, M. F. Österdahl, Anna L. Paterson, Doris M. Rassl, R. Alcantara, L. O’Neill, S. Widaa, Siobhan Austin-Guest, Matthew D. C. Neville, Moritz J Przybilla, Wei Cheng, M. Morra, Lucy Sykes, Matthew Mayho, Nicole Müller-Sienerth, N. Williams, Diana Alexander, Luke M. R. Harvey, Thomas Clarke, Alex Byrne, Jamie R Blundell, Matthew D Young, Krishnaa T. Mahbubani, K. Saeb-Parsy, Hilary C. Martin, Michael R Stratton, P. J. Campbell, R. Rahbari, K. Small, I. Martincorena
          </td>
          <td>2025-10-08</td>
          <td>Nature</td>
          <td>3</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8f1a53fb86f2b1b272e634808447e52e52e36" target='_blank'>
              Hotspots of human mutation point to clonal expansions in spermatogonia.
              </a>
            </td>
          <td>
            Vladimir Seplyarskiy, Mikhail A Moldovan, Evan Koch, Prathitha Kar, Matthew D. C. Neville, R. Rahbari, S. Sunyaev
          </td>
          <td>2025-10-08</td>
          <td>Nature</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8f50562c127ac79c0cdedb311535fc46568766" target='_blank'>
              Longitudinal single-cell analysis reveals treatment-resistant stem and mast cells with potential treatments for pediatric AML.
              </a>
            </td>
          <td>
            Denis Ohlstrom, Mojtaba Bakhtia, Hope L. Mumme, Marina E. Michaud, Alejandro De Janon Gutierrez, D. DeRyckere, Katherine E. Ferguson, Franklin Chien, William C. Pilcher, Sarthak Satpathy, Sean C Jordan, Douglas K Graham, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2025-09-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The heterogeneity of leukemic cells is the main cause of resistance to therapy in acute myeloid leukemia (AML). Consequently, innovative therapeutic approaches are critical to target a wide spectrum of leukemic clones, regardless of their genetic and non-genetic complexity. In this report, we leverage the vulnerability of AML cells to CDK6 to identify a combination therapy capable of targeting common biological processes shared by all leukemic cells, while sparing non-transformed cells. We demonstrate that the combined inhibition of CDK6 and LSD1 restores myeloid differentiation and depletes the leukemic progenitor compartment in AML samples. Mechanistically, this combination induces major changes in chromatin accessibility, leading to the transcription of differentiation genes and diminished LSC signatures. Remarkably, the combination is synergistic, induces durable changes in the cells, and is effective in PDX mouse models. While many AML samples exhibit only modest responses to LSD1 inhibition, co-targeting CDK6 restores the expected transcription response associated with LSD1 inhibition. Given the availability of clinical-grade CDK6 and LSD1 inhibitors, this combination holds significant potential for implementation in clinical settings through drug repositioning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cebb7efc870646ab91862d4a77c390cd760bc112" target='_blank'>
              Dual targeting of CDK6 and LSD1 is synergistic and overcomes differentiation blockade in AML
              </a>
            </td>
          <td>
            Lise M Brault, E. Voisset, Mathieu Desaunay, Antonia Boudet, Paraskevi Kousteridou, S. Letard, Nadine Carbuccia, A. Goubard, Rémy Castellano, Yves Collette, J. Vernerey, Isabelle Vigon, J. Pasquet, Patrice Dubreuil, Sophie Lopez, P. de Sepulveda
          </td>
          <td>2025-08-29</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="In humans, germline mutation rates are three- to four-fold higher in males than females for largely unknown reasons. We investigated whether transcription, a well-documented source of both DNA damage and repair in somatic tissues, is associated with sex differences in germline mutations. We used single-cell germline expression data from different stages of gametogenesis in males and females and phased de novo mutations from pedigrees. Focusing on protein-coding genes, we found no relationship between the male mutation rate and averaged gene expression levels during spermatogenesis despite evidence for transcription-asymmetry. Individual stages of spermatogenesis appear to vary in their contribution to mutation, however: expression levels in spermatogonial stem cells and sperm are significantly positively associated with paternal mutation rates, while those in primary spermatocytes are significantly negatively associated. Thus, transcription may have varying effects over male gametogenesis that are not visible from its cumulative effect on the total germline mutation rate. By contrast, the female mutation rate increases significantly with transcription levels averaged across different stages of folliculogenesis, with corresponding evidence for transcription-asymmetry. The difference between the sexes is also observed when using a smaller set of phased mutations from three-generation pedigrees and when relying on expression data from bulk gonadal tissues. These results indicate that transcription has distinct effects on the mutation rate in the two sexes, leading to more mutations in females but not males, in contrast to what one might expect from the overall paternal bias in germline mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f949d025880c67f108bb5128d039182864ec241e" target='_blank'>
              Sex differences in transcription-associated mutagenesis in the human germline
              </a>
            </td>
          <td>
            Minyoung J. Wyman, Ipsita Agarwal, Marc de Manuel, Natanael Spisak, Molly Przeworski
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most lethal gynecological malignancy. Therapeutic options remain limited for patients lacking predictive biomarkers, particularly those with BRCA wild-type tumors or those who have acquired resistance to both PARP inhibitors and platinum-based chemotherapy. Replication stress, TP53 mutations, and genomic instability characterize HGSOC. The cellular response to replication stress is primarily mediated by checkpoint kinases; however, this mechanism is frequently impaired in tumor cells. Consequently, cancer cells become increasingly dependent on the replication stress response (RSR) pathway for survival, and susceptible to therapies targeting the ATR-CHK1-WEE1 axis—a key regulator of genomic integrity. Inhibition of these checkpoint kinases can disrupt cell cycle control, inducing mitotic catastrophe and subsequent cancer cell death. Another defining feature of HGSOC is its immunosuppressive tumor microenvironment (TME), which has limited the efficacy of immune checkpoint inhibitors. Emerging evidence suggests that inhibition of the RSR pathway may not only exploit intrinsic tumor vulnerabilities but also modulate the TME to enhance anti-tumor immune responses. This provides rationale for combination approaches integrating RSR pathway inhibitors with innovative immune checkpoint blockade (ICB). This review examines the mechanistic rationale and therapeutic potential of such combinations, drawing on both preclinical and clinical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa92875288685395bcc326c756f9f278710481b" target='_blank'>
              Interplay of replication stress response and immune microenvironment in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Laura Venegas, Stephanie Lheureux
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="RAD50 is the central protein of the MRN complex and crucial in DNA double-strand break repair. RAD50 deficiency causes a genomic instability disorder characterized by microcephaly and stunted growth. Using lentiviral constructs, we investigated whether cancer-related RAD50 missense variants can complement the delayed damage response after exposure to the chemotherapeutic agent epirubicin and/or mitotic progression in RAD50-deficient fibroblasts. Eight missense variants, all capable of forming an MRN complex, supported the DNA damage response and mitotic features to different extents, indicating these functions are separable. Three variants showed both an impaired epirubicin response and slowed cell division in the likely pathogenic range. Assessing RAD50 missense variants with distinct functional readouts may help to further elucidate their differential roles in immunodeficiency and cancer and could improve therapeutic strategies. Impact statement RAD50 has a strong impact on DNA repair and cancer therapy. Here, we analyse RAD50 missense variants at four functional levels. Some variants showed an impaired epirubicin response and mitotic progression in the pathological range, while for others these endpoints were separable. Functional heterogeneity of RAD50 variants could contribute to clinical variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc8ea9e9e82f5d91c26df1ede413c6f7f658286" target='_blank'>
              RAD50 missense variants differentially affect the DNA damage response and mitotic progression.
              </a>
            </td>
          <td>
            Hanna Redeker, Swantje Kebel, Lea Völkening, Anna Vatselia, Louisa Weinhold, G. Asgedom, Axel Schambach, Detlev Schindler, T. Dörk, Kristine Bousset
          </td>
          <td>2025-10-01</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a highly heterogeneous malignancy characterized by significant variations in gene expression among distinct cell populations. Traditional sequencing technologies often obscure these differences, necessitating advanced approaches for a comprehensive understanding. Single-cell RNA sequencing (scRNA-seq) and Mendelian randomization (MR) analysis have emerged as powerful tools for unraveling the intricacies of PCa, enabling the identification of unique biological features at the individual cell level. In this study, droplet-based scRNA-seq analysis was performed on 4 PCa samples and 4 normal adjacent to tumor samples. The dataset, obtained from the NCBI GEO database, comprised 13,139 cells, facilitating the identification of diverse cell types and their respective gene expression patterns. Robust analysis pipelines, including pseudotemporal ordering, regional plot analysis, MR analysis, and comprehensive gene expression dynamics, were employed to investigate the tumor microenvironment, immune landscape, and potential causal associations with PCa susceptibility. Our scRNA-seq analysis revealed altered immune landscapes in PCa, underscoring the significance of T cells, particularly mucosal-associated invariant T (MAIT) cells. The study unveiled heterogeneity within the tumor microenvironment and identified key genes, such as ZFP36L2, associated with PCa susceptibility. MR analysis provided insights into the potential causal effects of MAIT cells on PCa, supported by analyses of gene expression dynamics and metabolic landscapes. ZFP36L2 emerged as a pivotal regulator, validated through various experimental approaches, including real-time quantitative polymerase chain reaction, western blot, and immunohistochemistry. Our findings underscore the significance of MAIT cells, presenting a novel diagnostic modality for advancing the diagnosis and treatment strategies of PCa. The identified genes, particularly ZFP36L2, offer potential avenues for precision medicine and targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/121d713d54f78ea6b18adcb73c968f4917204d87" target='_blank'>
              Decoding the association between mucosal-associated invariant T cells and prostate cancer via multi-omics analysis
              </a>
            </td>
          <td>
            Lichuan Lai, Rensheng Huang, Keke Si, Junming Huang, Lianli Yin
          </td>
          <td>2025-10-03</td>
          <td>Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background/Objectives: Multiple myeloma (MM) is a hematologic malignancy characterized by the clonal proliferation of abnormal plasma cells in bone marrow, predominantly affecting individuals over 65 years of age. Despite recent therapeutic advances, MM remains largely incurable due to frequent relapses and the emergence of drug resistance. MicroRNAs have emerged as key post-transcriptional regulators implicated in cancer progression, with miR-429 exhibiting context-dependent oncogenic or tumor-suppressive roles in various cancers. However, its function in MM has not been thoroughly investigated. Methods: miR-429 expression was evaluated in MM cells and patient samples by qRT-PCR. Functional effects were assessed through inhibition studies, proliferation/apoptosis assays, and co-culture with stromal cells. Results: In this study, we found that miR-429 expression is significantly downregulated in MM cell lines and primary malignant plasma cells compared to normal plasma cells. The functional inhibition of miR-429 in U266 cells led to a significant increase in cell proliferation without affecting spontaneous apoptosis, as confirmed in both MM cell lines and patient-derived plasma cells. Additionally, the inhibition of miR-429 in HS-5 stromal cells enhanced the proliferation of co-cultured MM cells, highlighting the role of the bone marrow microenvironment in disease progression. Conclusions: These findings suggest that miR-429 may act as a tumor suppressor by modulating MM cell proliferation. Although preliminary, our results support the need for further investigation into miR-429 as a potential biomarker or therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcbc38651a59954d239ad3d5c40304dac767702d" target='_blank'>
              miR-429 Acts as a Tumor Suppressor in Multiple Myeloma by Regulating Cell Proliferation
              </a>
            </td>
          <td>
            M. Abdelkarim, I. Limam, Emna Berred, Rachid Kharrat, Brigitte Sola, F. Ben Aissa-Fennira
          </td>
          <td>2025-08-26</td>
          <td>Hemato</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Multiple myeloma (MM) continues to be an incurable malignancy, even with recent therapeutic advancements. While epigenetic dysregulation at cis-regulatory elements is known to drive disease progression, the complete molecular mechanisms underlying these alterations are poorly understood. Using ATAC-seq analysis combined with computational footprinting of CD138+ cells from 55 MM patients, we depicted the dynamic changes in chromatin accessibility during disease progression and identified Nuclear Respiratory Factor 1 (NRF1) as a master regulator of vital MM survival pathways. We demonstrated that NRF1 maintains proteasome homeostasis by orchestrating the ubiquitination pathway, which is essential for MM cell survival. We discovered a novel enhancer element that physically interacts with the NRF1 promoter, sustaining its expression. Targeting this enhancer RNA reduced NRF1 levels and increased tumor cell sensitivity to bortezomib (BTZ), suggesting therapeutic potential. In xenograft models, we showed that antisense oligonucleotides (ASOs) targeting the NRF1 enhancer, either alone or combined with BTZ, significantly decreased tumor burden and improved survival. Our findings reveal a previously unknown NRF1-dependent mechanism regulating MM cell survival and present a promising therapeutic approach through manipulation of its regulatory network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cc8489d79911ba7bfacf778b92835e6e2c18c2c" target='_blank'>
              Nuclear respiratory factor 1 promotes cell survival in multiple myeloma under proteasome inhibition therapy.
              </a>
            </td>
          <td>
            T. Bruno, Maria Chiara Cappelletto, C. Cortile, S. Di Giovenale, Bruno Amadio, F. de Nicola, I. Falcone, Stefano Giuliani, Belinda Palermo, V. Catena, L. Ciuffreda, F. Cerruti, P. Cascio, Roberta Merola, Serena Masi, Valentina De Pascale, O. Annibali, Silvia Ferraro, S. Gumenyuk, Francesco Pisani, Francesco Marchesi, A. Mengarelli, Maurizio Fanciulli, G. Corleone
          </td>
          <td>2025-09-08</td>
          <td>Blood</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Chromatin serves to organize and compact the genome but also functions as a signaling hub for the dynamic regulation of transcriptional programs that control cell type specification. The historical discovery that several pro-differentiation anti-cancer agents target chromatin regulatory enzymes buoyed early interest in developing drugs that modulate chromatin structure and function. Chromatin-based drug discovery has since flourished alongside major advances in discovery chemistry and target selection, producing a rich collection of chemical probes, drugs, and drug candidates targeting chromatin regulatory processes. The substantial growth and maturity of this field over the last several decades provides an opportunity to reflect on the successes and failures associated with translating chromatin regulatory targets into anti-cancer drugs. Taking a target-centric perspective, we discuss the motivation for pursuing specific chromatin regulatory proteins and review the chemistries that enabled small molecule discovery and development. In so doing, we hope to evaluate the strength of these targets, the agents that prosecute them, and the prospects for future efforts in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47417d8f546abed9ecb1b94933a4b2ad03f57ce" target='_blank'>
              Chromatin Regulatory Targets for Anticancer Therapeutics.
              </a>
            </td>
          <td>
            Paige A. Barta, Trever R. Carter, Michael A. Erb
          </td>
          <td>2025-10-06</td>
          <td>Chemical reviews</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 The transcription factor SOX9 is associated with poor prognosis and therapy resistance in medulloblastoma, one of the most common malignant pediatric brain tumors. While SOX9 knockdown inhibits tumor recurrence in animal models, no direct SOX9 inhibitors are currently available for clinical use. SOX9 functions as a potent activator of a minimal COL2A1 enhancer that can work as a SOX9 specific response element (S9RE). In this study, we aimed to engineer S9RE to identify therapy-resistant tumor cells and further use it as a gene therapy to selectively target SOX9-expressing medulloblastoma.



 We employed lentiviral and adeno-associated virus (AAV) vectors, flow cytometry, immunohistochemistry, immunoblotting, in vitro cell models, and in vivo orthotopic mouse models to track and eliminate therapy-resistant medulloblastoma cells.



 We demonstrated that SOX9-active cells can be efficiently labeled using a reporter construct containing four tandem S9REs coupled to a minimal promoter (S9RE-minP) driving expression of enhanced green fluorescent protein (EGFP) and luciferase. Furthermore, S9RE-minP was used in an AAV vector to drive expression of herpes simplex virus thymidine kinase (HSV-TK), enabling a suicide gene therapy approach wherein ganciclovir selectively kills dividing SOX9-positive tumor cells.



 This AAV-based gene therapy approach, when combined with fractionated radiation, synergized with standard therapy to effectively eliminate SOX9-positive medulloblastoma cells in vitro and in vivo. These findings support the potential of SOX9-driven gene therapy as a targeted strategy to overcome therapy resistance in malignant pediatric brain tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567c890bcf48766298ba026810b43cd0d4d1b97d" target='_blank'>
              A cytotoxic gene therapy targeting SOX9-positive therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Tong Lin, S. Hutter, Elin Vikström, G. Rosén, Tobias Bergström, Karl O Holmberg, Miao Zhao, G. Giraud, F. Swartling
          </td>
          <td>2025-09-08</td>
          <td>Neuro-Oncology Pediatrics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/394995c0fcb10ae0448160919d2f62f9453d3170" target='_blank'>
              Comprehensive single-cell analysis reveals mast cells' roles in cancer immunity.
              </a>
            </td>
          <td>
            Hao Qian, Wenbo Wang, Jiaomeng Pan, Ming Zhao, Li Gao, Jia Zeng, Hang Gong, Guangyan Zhangyuan, Zi Li, Mao Zhang, Xia Shen, Yuanchi Weng, Xiaxing Deng, Yingying Huang
          </td>
          <td>2025-10-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a primary driver of cervical cancer. Integration of HPV into the human genome causes persistent expression of viral oncogenes E6 and E7, which promote carcinogenesis and disrupt host genomic function. However, the impact of integration on host gene expression remains incompletely understood. We used multimodal RNA sequencing, combining total RNA-seq and Cap Analysis of Gene Expression (CAGE), to clarify virus–host interactions after HPV integration. HPV-derived transcripts were detected in 17 of 20 clinical samples. In most specimens, transcriptional start sites (TSSs) showed predominant early promoter usage, and transcript patterns differed with detectable E4 RNA region. Notably, the high RNA expressions of E4 region and viral-human chimeric RNAs were mutually exclusive. Chimeric RNAs were identified in 13 of 17 samples, revealing 16 viral integration sites (ISs). CAGE data revealed two patterns of TSS upregulation centered on the ISs: a two-sided pattern (43.8%) and a one-sided pattern (31.3%). Total RNA-seq showed upregulation of 12 putative cancer-related genes near ISs, including MAGI1-AS1, HAS3, CASC8, BIRC2, and MMP12. These findings indicate that HPV integration drives transcriptional activation near ISs, enhancing expression of adjacent oncogenes. Our study deepens understanding of HPV-induced carcinogenesis and informs precision medicine strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48263c0732371718c3701a40772227c4abc1885c" target='_blank'>
              Elucidating Alterations in Viral and Human Gene Expression Due to Human Papillomavirus Integration by Using Multimodal RNA Sequencing
              </a>
            </td>
          <td>
            Kana Tamai, S. Kinjo, Ayumi Taguchi, K. Nagasaka, Daisuke Yoshimoto, Anh Quynh Duong, Yoko Yamamoto, Hitoshi Iuchi, M. Mori, K. Sone, Michiaki Hamada, Kei Kawana, Kazuho Ikeo, Yasushi Hirota, Yutaka Osuga
          </td>
          <td>2025-10-06</td>
          <td>Viruses</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Pediatric low-grade gliomas (pLGGs) are the most common brain tumors in children, comprising approximately 33% of all pediatric brain tumors. These tumors are typically driven by single somatic mutations in the MAPK pathway including genes such as BRAF, FGFR, and NF1. The majority of current therapies target MAPK pathway inhibition, which is often initially effective. However, high tumor rebound rates following treatment discontinuation underscore the urgent need for alternative therapeutic strategies. To identify novel MAPK independent vulnerabilities, we performed a whole genome CRISPR-Cas9 knockout screen in neural stem cells harboring pLGG mutations. Our screen revealed a strong dependence of BRAFV600E mutant cells on proteins involved in replication stress. We hypothesize BRAFV600E mutant cells experience elevated levels of replication fork stalling, making them more reliant on proteins that resolve replication stress. In this study, we are investigating how perturbation of proteins that prevent replication fork collapse and DNA damage affect the survival of BRAFV600E mutant cells. Our goal is to uncover novel therapeutic targets for pLGGs that are not reliant on MAPK inhibition.



 Joohee Lee, Jenna Robinson, Elisabeth Gonzalez, Kevin Zhou, David Root, Pratiti Bandopadhayay. Uncovering replication stress vulnerabilities in pediatric low grade gliomas using a genome wide CRISPR-Cas9 knockout screen [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04bf98a9ad932c12c72d4a95a40eb75073a085d" target='_blank'>
              Abstract B024: Uncovering replication stress vulnerabilities in pediatric low grade gliomas using a genome wide CRISPR-Cas9 knockout screen
              </a>
            </td>
          <td>
            Joohee Lee, Jenna Robinson, Elizabeth M. Gonzalez, Kevin N Zhou, David E. Root, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background: Tumor protein 53 gene (TP53) is a critical factor that controls different cell activities such as cell cycle, DNA repair mechanism, autophagy, apoptosis, and metabolism. The TP53 gene is the most commonly mutated gene, especially in the 4–8 exons region. This mutation enhances the development of many abnormalities, such as the initiation of different types of cancer. Aim: The main objective of this study was to design and evaluate the efficacy of three different primer sets that targeted the TP53 gene at the hotspot regions. Methods: To do that, twelve blood samples were collected from dogs belonging to the German Shepherd breed/K9 aged between 8–12 years. Then, the DNA extraction and polymerase chain reaction (PCR) took place by using the three primer sets, which were designed using SnapGene. The primer sets, namely, first primer, the second and the third targeted exons 5–9 located in the canine TP53 gene. In the following step, all the PCR products were sent for Sanger sequencing and then phylogenetic analysis. Results: Our findings indicated that the first primer set consistently showed higher amplification signal efficiency and reduced dimer formation compared with the second and third primer sets, respectively, with a 60ºC annealing temperature. In addition, all the sequenced samples aligned with the reference canine TP53 gene in the phylogenetic tree. Conclusion: This study offered the best TP53 primer design that targeted the hotspot regions of the canine TP53 gene for researchers who are interested in targeting such regions in this gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15b8d7f0216aab1094d22537bd4acce3a84b5f4" target='_blank'>
              Designated primers targeted canine TP53 gene hotspot regions
              </a>
            </td>
          <td>
            Fatimah Ali, A. Al-Zebeeby
          </td>
          <td>2025-08-31</td>
          <td>Open Veterinary Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>