<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-17 09:27:26 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>19</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Mutations and genome instability are hallmarks of cancer. The advanced Next Generation Sequencing(NGS) technology now enables rapid and cost-effective analysis for tumoral and normal genomic translational research. In this research, mRNA sequencing was applied to get the whole transcriptome data of mouse B16F10 melanoma and CT26 colon carcinoma cells. Up regulation genes and down regulation genes were identified in tumor cells with differentially expressed genes(DEGs) analysis. The DEGs were analyzed with GO, KEGG and Reactome pathway enrichment study. Results demonstrated the DEGs and enriched pathways were mainly associated with cell proliferation and biosynthetic processing. Further, gene fusion and alternative splicing analysis reveled fusion genes and skipped exons were extensive exist within the chromosomes of tumor cells. Together, the integrated DEGs and bioinformatics analysis in this article may shed light on further research of cancer treating targets and molecular mechanisms of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6e94aa8f9282678d155caf1a32035cd06536087" target='_blank'>
              Identification and analysis of DEGs of mouse tumor cells by integrated bioinformatics methods
              </a>
            </td>
          <td>
            Xizhong Jing, Baiyong Li, Yifei Wang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Copy number variants (CNVs) are an important source of genetic variation underlying rapid adaptation and genome evolution. Whereas point mutation rates vary with genomic location and local DNA features, the role of genome architecture in the formation and evolutionary dynamics of CNVs is poorly understood. Previously, we found the GAP1 gene in Saccharomyces cerevisiae undergoes frequent amplification and selection in glutamine-limitation. The gene is flanked by two long terminal repeats (LTRs) and proximate to an origin of DNA replication (autonomously replicating sequence, ARS), which likely promote rapid GAP1 CNV formation. To test the role of these genomic elements on CNV-mediated adaptive evolution, we evolved engineered strains lacking either the adjacent LTRs, ARS, or all elements in glutamine-limited chemostats. Using a CNV reporter system and neural network simulation-based inference (nnSBI) we quantified the formation rate and fitness effect of CNVs for each strain. Removal of local DNA elements significantly impacts the fitness effect of GAP1 CNVs and the rate of adaptation. In 177 CNV lineages, across all four strains, between 26% and 80% of all GAP1 CNVs are mediated by Origin Dependent Inverted Repeat Amplification (ODIRA) which results from template switching between the leading and lagging strand during DNA synthesis. In the absence of the local ARS, distal ones mediate CNV formation via ODIRA. In the absence of local LTRs, homologous recombination can mediate gene amplification following de novo retrotransposon events. Our study reveals that template switching during DNA replication is a prevalent source of adaptive CNVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/488ecf1ac91854034846373133d03867e6680f28" target='_blank'>
              Template switching during DNA replication is a prevalent source of adaptive gene amplification
              </a>
            </td>
          <td>
            Julie N. Chuong, Nadav Ben Nun, Ina Suresh, J. Matthews, Titir De, Grace Avecilla, Farah Abdul-Rahman, Nathan Brandt, Yoav Ram, David Gresham
          </td>
          <td>2025-02-03</td>
          <td>eLife</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Copy number variants (CNVs) are prevalent in both diploid and haploid genomes, with the latter containing a single copy of each gene. Studying CNVs in genomes from single or few cells is significantly advancing our knowledge in human disorders and disease susceptibility. Low-input including low-cell and single-cell sequencing data for haploid and diploid organisms generally displays shallow and highly non-uniform read counts resulting from the whole genome amplification steps that introduce amplification biases. In addition, haploid organisms typically possess relatively short genomes and require a higher degree of DNA amplification compared to diploid organisms. However, most CNV detection methods are specifically developed for diploid genomes without specific consideration of effects on haploid genomes. Challenges also reside in reference samples or normal controls which are used to provide baseline signals for defining copy number losses or gains. In traditional methods, references are usually pre-specified from cells that are assumed to be normal or disease-free. However, the use of pre-defined reference cells can bias results if common CNVs are present. Here, we present the development of a comprehensive statistical framework for data normalization and CNV detection in haploid single- or low-cell DNA sequencing data called HapCNV. The prominent advancement is the construction of a novel genomic location specific pseudo-reference that selects unbiased references using a preliminary cell clustering method. This approach effectively preserves common CNVs. Using simulations, we demonstrated that HapCNV outperformed existing methods by generating more accurate CNV detection, especially for short CNVs. Superior performance of HapCNV was also validated in detecting known CNVs in a real P. falciparum parasite dataset. In conclusion, HapCNV provides a novel and useful approach for CNV detection in haploid low-input sequencing datasets, with easy applicability to diploids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f40d70f46141697934edb1592beae132a61d8c33" target='_blank'>
              HapCNV: A Comprehensive Framework for CNV Detection in Low-input DNA Sequencing Data
              </a>
            </td>
          <td>
            Xuanxuan Yu, Fei Qin, Shiwei Liu, Noah Brown, Qing Lu, Guoshuai Cai, J. Guler, Feifei Xiao
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ex vivo drug response profiling is emerging as a valuable tool for identifying drug resistance mechanisms and advancing precision medicine in hematological cancers. However, the functional impact of dysregulation of the epigenome and transcriptome in this context remains poorly understood. In this study, we combined ex vivo drug sensitivity profiling with transcriptomic and epigenomic analyses in diagnostic samples from 597 pediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) patients. Ex vivo resistance to antimetabolites (e.g., cytarabine, thioguanine), glucocorticoids (e.g., dexamethasone, prednisolone), and doxorubicin was independently associated with reduced relapse-free survival (RFS, p < 0.05). Molecular profiling identified pre-treatment DNA methylation and gene expression patterns distinguishing resistant from sensitive cases, revealing key drug resistance signatures. These included aberrant expression of genes related to heme metabolism (e.g. ATPV06A) and KRAS signaling (e.g. ABCB1). Notably, we also observed atypical expression of genes usually restricted to T-cells and other immune cells (e.g., ITK) in resistant BCP-ALL cells. Our findings demonstrate that ex vivo drug response patterns are predictive of clinical outcomes and reflect intrinsic molecular states associated with drug tolerance. This integrative multi-omics approach highlights potential therapeutic targets and underscores the value of functional precision medicine in identifying treatment vulnerabilities in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b0bb7a98b1f284795f68f2172436eb0a2df655" target='_blank'>
              Ex Vivo Drug Responses and Molecular Profiles of 597 Pediatric Acute Lymphoblastic Leukemia Patients
              </a>
            </td>
          <td>
            Anna Pia Enblad, O. Krali, Henrik Gezelius, Anders Lundmark, Kristin Blom, Claes Andersson, J. Palle, B. Frost, S. Ryhänen, Trond Flægstad, Ó. Jónsson, Kjeld Schmiegelow, Arja Harila, Peter Nygren, Rolf Larsson, Gudmar Lönnerholm, J. Nordlund, Anna Pia
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and aggressively progressing malignancy that presents considerable clinical challenges.Although chemotherapy can effectively manage symptoms during the earlystages of SCCE, its long-term effectiveness is notably limited, with theunderlying mechanisms remaining largely undefined. In this study, weemployed single-cell RNA sequencing (scRNA-seq) to analyze SCCE samplesfrom a single patient both before and after chemotherapy treatment. Our analysisrevealed significant cellular plasticity and alterations in the tumormicroenvironment’s cellular composition. Notably, we observed an increase intumor cell diversity coupled with reductions in T cells, B cells, and myeloid-likecells. The pre-treatment samples predominantly featured carcinoma cells in amiddle transitional state, while post-treatment samples exhibited an expandedpresence of cells in terminal, initial-to-terminal (IniTerm), and universally alteredstates. Further analysis highlighted dynamic interactions between tumor cells andimmune cells, with significant changes detected in key signaling pathways, suchas TIGIT-PVR and MDK-SDC4. This study elucidates the complex dynamics of cellplasticity in SCCE following chemotherapy, providing new insights and identifyingpotential therapeutic targets to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae84357d27c8ef8315335bde0ededde5ffcf7d8" target='_blank'>
              Single-cell RNA sequencing elucidates cellular plasticity in esophageal small cell carcinoma following chemotherapy treatment
              </a>
            </td>
          <td>
            Qinkai Zhang, Ziyu Gao, Ru Qiu, Jizhao Cao, Chunxiao Zhang, Wei Qin, Meiling Yang, Xinyue Wang, Ciqiu Yang, Jie Li, Dongyang Yang
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="KRAS was long deemed undruggable until the discovery of the switch-II pocket facilitated the development of specific KRAS inhibitors. Despite their introduction into clinical practice, resistance mechanisms can limit their effectiveness. Initially, tumors rely on mutant KRAS, but as they progress, they may shift to alternative pathways, resulting in intrinsic resistance. This resistance can stem from mechanisms like epithelial-to-mesenchymal transition (EMT), YAP activation, or KEAP1 mutations. KRAS inhibition often triggers cellular rewiring to counteract therapeutic pressure. For instance, feedback reactivation of signaling pathways such as MAPK, mediated by receptor tyrosine kinases, supports tumor cell survival. Inhibiting KRAS disrupts protein homeostasis, but reactivation of MAPK or AKT can restore it, aiding tumor cell survival. KRAS inhibition also causes metabolic reprogramming and protein re-localization. The re-localization of E-cadherin and Scribble from the membrane to the cytosol causes YAP to translocate to the nucleus, where it drives MRAS transcription, leading to MAPK reactivation. Emerging evidence indicates that changes in cell identity, such as mucinous differentiation, shifts from alveolar type 2 to type 1 cells, or lineage switching from adenocarcinoma to squamous cell carcinoma, also contribute to resistance. In addition to these nongenetic mechanisms, secondary mutations in KRAS or alterations in upstream/downstream signaling proteins can cause acquired resistance. Secondary mutations in the switch-II pocket disrupt drug binding, and known oncogenic mutations affect drug efficacy. Overcoming these resistance mechanisms involves enhancing the efficacy of drugs targeting mutant KRAS, developing broad-spectrum inhibitors, combining therapies targeting multiple pathways, and integrating immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c1ae3fc98dfd6ef9151275fdf77ddcf612521a" target='_blank'>
              Mechanisms of Resistance to KRAS Inhibitors: Cancer Cells' Strategic Use of Normal Cellular Mechanisms to Adapt.
              </a>
            </td>
          <td>
            Noritaka Tanaka, Hiromichi Ebi
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy affecting women, manifesting as a heterogeneous disease with diverse molecular characteristics and clinical presentations. Recent studies have elucidated the role of epigenetic modifications in the pathogenesis of breast cancer, including drug resistance and efflux characteristics, offering potential new diagnostic and prognostic markers, treatment efficacy predictors, and therapeutic agents. Key modifications include DNA cytosine methylation and the covalent modification of histone proteins. Unlike genetic mutations, reprogramming the epigenetic landscape of the cancer epigenome is a promising targeted therapy for the treatment and reversal of drug resistance. Epidrugs, which target DNA methylation and histone modifications, can provide novel options for the treatment of breast cancer by reversing the acquired resistance to treatment. Currently, the most promising approach involves combination therapies consisting of epidrugs with immune checkpoint inhibitors. This review examines the aberrant epigenetic regulation of breast cancer initiation and progression, focusing on modifications related to estrogen signaling, drug resistance, cancer progression, and the epithelial–mesenchymal transition (EMT). It examines existing epigenetic drugs for treating breast cancer, including agents that modify DNA, inhibitors of histone acetyltransferases, histone deacetylases, histone methyltransferases, and histone demethyltransferases. It also delves into ongoing studies on combining epidrugs with other therapies and addresses the upcoming obstacles in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1d287b99832f39586fe0fd89b9b4c10e300e3c" target='_blank'>
              Cellular Epigenetic Targets and Epidrugs in Breast Cancer Therapy: Mechanisms, Challenges, and Future Perspectives
              </a>
            </td>
          <td>
            Ibrahim S. Alalhareth, Saleh M. Alyami, Ali H. Alshareef, Ahmed O. Ajeibi, M. A. Al Munjem, Ahmad A. Elfifi, Meshal M. Alsharif, S. A. Alzahrani, Mohammed A. Alqaad, Marwa B. Bakir, Basel A. Abdel-Wahab
          </td>
          <td>2025-02-03</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Despite efficacy of FLT3 and BCL2 inhibition in acute myeloid leukemia (AML), relapse limits survival. Mutation status and AML monocytic differentiation are implicated in resistance. On-treatment tumor evolution may select for genetically distinct clones or shifts in differentiation not resolvable by bulk sequencing. We performed multiomic single cell (SC) DNA/protein and RNA/protein profiling of patients treated on a clinical trial of the BCL2 inhibitor venetoclax and the FLT3 inhibitor gilteritinib (Ven/Git) to characterize immunophenotypic, transcriptional, and genetic clonal evolution on therapy. We found that while Ven/Gilt effectively eliminated FLT3 mutant clones, it selected for RAS mutations, RAS pathway activation and RAS-associated monocytic differentiation. In an in vitro model of monocytic differentiation associated with heightened RAS pathway activation, we demonstrated that MEK inhibition re-sensitized to Ven/Gilt. These data indicate RAS signaling is central to FLT3 and BCL2 inhibitor resistance, is tightly coupled to monocytic differentiation and can be overcome by RAS pathway inhibition. COI C.C.S. has provided educational talks for Astellas Pharma, served on advisory boards for Genentech/Abbvie and received research funding from Abbvie. B.C., Y.S. and J.H. are employees of Abbvie. M.S. and H.H. are or were previously employees of Genentech. Funding This work was supported in part by Abbvie. C.C.S. is a Leukemia & Lymphoma Society Scholar in Clinical Research and a Damon Runyon-Richard Lumsden Foundation Clinical Investigator supported (in part) by the Damon Runyon Cancer Research Foundation (CI-99–18). Statement of Significance: Mutational and non-mutational RAS signaling activation drives clonal selection, monocytic differentiation and treatment resistance to FLT3 and BCL2 inhibition in AML. MEK inhibition can resensitize resistant AML cells, suggesting therapeutic potential for combined FLT3, BCL2 and RAS pathway inhibition in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/044e6c5d837777d0153b7a0fb95ff71f6ce12ad5" target='_blank'>
              RAS pathway activation drives clonal selection and monocytic differentiation in FLT3 and BCL2 inhibitor resistance
              </a>
            </td>
          <td>
            Vanessa E. Kennedy, C. Peretz, Anushka Walia, Brenda Chyla, Yan Sun, Jason Hill, Elaine Tran, Andrew Koh, Timothy T. Ferng, Samantha M. Pintar, Matthew F. Jones, Bogdan Popescu, N. Murad, Ritu Roy, Adam Olshen, Sunil Joshi, E. Traer, Monique Dail, Habib Hamidi, Jessica Altman, Naval G Daver, M. Levis, James K. McCloskey, Alexander Perl, Catherine C Smith
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Epidermal growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) have demonstrated remarkable efficacy in treating non-small cell lung cancer (NSCLC), but acquired resistance greatly reduces efficacy and poses a significant challenge to patients. While numerous studies have investigated the mechanisms underlying EGFR-TKI resistance, its complexity and diversity make the existing understanding still incomplete. Traditional approaches frequently struggle to adequately reveal the process of drug resistance development through mean value analysis at the overall cellular level. In recent years, the rapid development of single-cell RNA sequencing technology has introduced a transformative method for analyzing gene expression changes within tumor cells at a single-cell resolution. It not only deepens our understanding of the tumor microenvironment and cellular heterogeneity associated with EGFR-TKI resistance but also identifies potential biomarkers of resistance. In this review, we highlight the critical role of single-cell RNA sequencing in lung cancer research, with a particular focus on its application to exploring the mechanisms of EGFR-TKI-acquired resistance in NSCLC. We emphasize its potential for elucidating the complexity of drug resistance mechanism and its promise in informing more precise and personalized treatment strategies. Ultimately, this approach aims to advance NSCLC treatment toward a new era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57a8423fcdeff9dbe3c23c8eb5bf687e8a733ce" target='_blank'>
              Single-Cell RNA Sequencing in Unraveling Acquired Resistance to EGFR-TKIs in Non-Small Cell Lung Cancer: New Perspectives
              </a>
            </td>
          <td>
            Lin Peng, Siyou Deng, Jinjie Li, Yujie Zhang, Li Zhang
          </td>
          <td>2025-02-11</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Resistance to endocrine therapy (ET) is common in estrogen receptor (ER) positive breast cancer. Multiple studies have demonstrated that upregulation of MAPK signaling pathways contributes to ET resistance. Herein we show that vandetanib treatment enhances sensitivity to ET in ET-sensitive and –resistant ER+ breast cancer models. Using a CRISPR knockout model, we demonstrate that vandetanib effects are partially mediated by RET receptor tyrosine kinase. Vandetanib treatment alters the gene expression program of ER+ breast cancer cells resulting in a less proliferative and more estrogen responsive Luminal-A like character. Tyrosine kinase network reprogramming was assessed using multiplexed kinase inhibitor beads-mass spectrometry (MIB/MS) assay to identify adaptive resistance mechanisms to vandetanib treatment, which identified upregulation of HER2 activity. Co-treatment to inhibit HER2 with lapatinib enhanced sensitivity to vandetanib, demonstrating biologic activity of HER2 upregulation. Finally, we use our operating room-to-laboratory assay that measures drug response in individual primary tumor cells in short term cultures to demonstrate conserved gene expression changes, including increased HER2 activity signatures, in vandetanib treated cells, and identify features associated with vandetanib response. These results support future investigation of RET targeting strategies considering reprogrammed networks, such as activated HER2, in patients with ET resistant ER+ breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/163cc944feb5c6e8c8bb5e07eb1e26b3c782d784" target='_blank'>
              Kinase Plasticity in Response to Vandetanib Enhances Sensitivity to Tamoxifen and Identifies Co-Treatment Strategies in Estrogen Receptor Positive Breast Cancer
              </a>
            </td>
          <td>
            Rasha T. Kakati, Austin A. Whitman, Santiago Haase, Attila T. Szenasi, Christine Hnc Thai, Elizabeth Brunk, Denis O. Okumu, Michael P. East, C. Perou, Gary L. Johnson, Philip M. Spanheimer
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>162</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Changes in the copy number of large genomic regions, termed copy number variations (CNVs), contribute to important phenotypes in many organisms. CNVs are readily identified using conventional approaches when present in a large fraction of the cell population. However, CNVs that are present in only a few genomes across a population are often overlooked but important; if beneficial under specific conditions, a de novo CNV that arises in a single genome can expand during selection to create a larger population of cells with novel characteristics. While the reach of single cell methods to study de novo CNVs is increasing, we continue to lack information about CNV dynamics in rapidly evolving microbial populations. Here, we investigated de novo CNVs in the genome of the Plasmodium parasite that causes human malaria. The highly AT-rich P. falciparum genome readily accumulates CNVs that facilitate rapid adaptation to new drugs and host environments. We employed a low-input genomics approach optimized for this unique genome as well as specialized computational tools to evaluate the de novo CNV rate both before and after the application of stress. We observed a significant increase in genome-wide de novo CNVs following treatment with a replication inhibitor. These stress-induced de novo CNVs encompassed genes that contribute to various cellular pathways and tended to be altered in clinical parasite genomes. This snapshot of CNV dynamics emphasizes the connection between replication stress, DNA repair, and CNV generation in this important microbial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c09296cb530f9bd4da8646286b7568688223753" target='_blank'>
              Replication stress increases de novo CNVs across the malaria parasite genome
              </a>
            </td>
          <td>
            Noah Brown, A. Luniewski, Xuanxuan Yu, Michelle Warthan, Shiwei Liu, Julia Zulawinska, Syed Ahmad, Molly Congdon, Webster Santos, Feifei Xiao, J. Guler
          </td>
          <td>2024-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4d7f1f1862a0b62b5b5df29d2416fddab0290c" target='_blank'>
              lncRNAs: the unexpected link between protein synthesis and cancer adaptation
              </a>
            </td>
          <td>
            Mila Gugnoni, Manoj Kumar Kashyap, Kishore K. Wary, A. Ciarrocchi
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Antimicrobial combination therapy is widely used to combat Mycobacterium tuberculosis (Mtb), yet resistance rates continue to rise. Mutator strains, with defects in DNA repair genes, are a known source of resistance in other bacterial infections, but their global role in Mtb remains unclear. Here, we provide a comprehensive overview of the contribution of single nucleotide polymorphisms (SNPs) in DNA Repair, Replication, and Recombination (3R) genes to resistance in Mtb. Through large-scale bioinformatics analysis of 53,589 whole-genomes, we identified 18 novel SNPs linked to genotypic drug resistance in 3R genes, covering 12.5% of clinical isolates/strains with available genome sequences. Notably, a number of the detected SNPs were positively selected during Mtb evolution. Experimental mutation frequency tests further indicated functional defects of these sequence variants in key DNA repair pathways. Our findings indicate that the role of mutability in 3R genes plays a significant role in Mtb resistance at the global level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8834cea19a0a1579b6f2abeb80d3493166b930f" target='_blank'>
              Genome wide analyses reveal the role of mutator phenotypes in the emergence of drug resistance in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            R. Zein-Eddine, A. Le Meur, S. Skouloubris, L. Jelsbak, G. Refrégier, H. Myllykallio
          </td>
          <td>2025-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0203e20c6bc4f26349dbeefa5213e0b374637b7e" target='_blank'>
              Oncofetal reprogramming drives phenotypic plasticity in WNT-dependent colorectal cancer
              </a>
            </td>
          <td>
            Slim Mzoughi, Megan Schwarz, Xuedi Wang, D. Demircioglu, Gulay Ulukaya, Kevin Mohammed, Habiba Zorgati, Denis Torre, Lewis E. Tomalin, Federico Di Tullio, Carlos Company, Yuliia Dramaretska, M. Leushacke, Bruno Giotti, T. Lannagan, D. Lozano-Ojalvo, P. Karras, Peter B Vermeulen, Dan Hasson, Robert Sebra, A. Tsankov, Owen J. Sansom, J. Marine, Nick Barker, G. Gargiulo, Ernesto Guccione
          </td>
          <td>2025-02-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="PAX3-FOXO1, an oncogenic transcription factor, drives a particularly aggressive subtype of rhabdomyosarcoma (RMS) by enforcing gene expression programs that support malignant cell states. Here we show that PAX3-FOXO1+ RMS cells exhibit altered pyrimidine metabolism and increased dependence on enzymes involved in de novo pyrimidine synthesis, including dihydrofolate reductase (DHFR). Consequently, PAX3-FOXO1+ cells display increased sensitivity to inhibition of DHFR by the chemotherapeutic drug methotrexate, and this dependence is rescued by provision of pyrimidine nucleotides. Methotrexate treatment mimics the metabolic and transcriptional impact of PAX3-FOXO1 silencing, reducing expression of genes related to PAX3-FOXO1-driven malignant cell states. Accordingly, methotrexate treatment slows growth of multiple PAX3-FOXO1+ tumor xenograft models, but not fusion-negative counterparts. Taken together, these data demonstrate that PAX3-FOXO1 induces cell states characterized by altered pyrimidine dependence and nominate methotrexate as an addition to the current therapeutic arsenal for treatment of these malignant pediatric tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50231dc8772bfbbe71ed944e599efde5187dc0b" target='_blank'>
              PAX3-FOXO1 drives targetable cell state-dependent metabolic vulnerabilities in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Katrina I. Paras, Julia S. Brunner, Jacob A. Boyer, Angela M. Montero, Benjamin T. Jackson, Sangita Chakraborty, Abigail Xie, Kristina Guillan, Armaan Siddiquee, Lourdes Pajuelo Torres, Joshua D. Rabinowitz, Andrew L Kung, Daoqi You, Filemon Dela Cruz, Lydia W. S. Finley
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b1219475c2c72e7d5e477e93702ece4b16c3cc" target='_blank'>
              A Genome-Wide CRISPR/Cas9 Screen Identifies Regulatory Genes for Stem Cell Aging.
              </a>
            </td>
          <td>
            Peng Su, Yi-liang Miao
          </td>
          <td>2024-12-21</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While the key aspects of genetic evolution and their clinical implications in clear cell renal-cell carcinoma (ccRCC) are well-documented, how genetic features co-evolve with the phenotype and tumor microenvironment (TME) remains elusive. Here, through joint genomic-transcriptomic analysis of 243 samples from 79 patients recruited to the TRACERx Renal study, we identify pervasive non-genetic intratumor heterogeneity, with over 40% not attributable to genetic alterations. By integrating tumor transcriptomes and phylogenetic structures, we observe convergent evolution to specific phenotypic traits, including cell proliferation, metabolic reprogramming and overexpression of putative cGAS-STING repressors amid high aneuploidy. We also uncover a co-evolution between the tumor and the T cell repertoire, as well as a longitudinal shift in the TME from an anti-tumor to an immunosuppressive state, linked to the acquisition of recurrently late ccRCC drivers 9p loss and SETD2 mutations. Our study reveals clinically-relevant and hitherto underappreciated non-genetic evolution patterns in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84f2fcb6ee92e2f103bced79a769e625785b29b" target='_blank'>
              Tracking non-genetic evolution from primary to metastatic ccRCC: TRACERx Renal.
              </a>
            </td>
          <td>
            Ángel Fernández-Sanromán, A. Fendler, B. J. Tan, A. Cattin, C. Spencer, Rachael Thompson, L. Au, Irene Lobon, H. Pallikonda, Alice Martin, F. Byrne, Antonia Franz, Anna Mikolajczak, Haseeb Rahman, Z. Tippu, S. T. Shepherd, Hugang Feng, Daqi Deng, Andrew Rowan, Lisa M. Pickering, A. J. Furness, Kate Young, David Nicol, S. Rudman, Tim O’Brien, K. Edmonds, Ashish Chandra, S. Hazell, Kevin Litchfield, G. Kassiotis, J. Larkin, S. Turajlic
          </td>
          <td>2025-01-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a complex and heterogeneous hematologic malignancy characterized by clonal evolution, genetic instability, and interactions with a supportive tumor microenvironment. These factors contribute to treatment resistance, disease progression, and significant variability in clinical outcomes among patients. This review explores the mechanisms underlying MM progression, including the genetic and epigenetic changes that drive clonal evolution, the role of the bone marrow microenvironment in supporting tumor growth and immune evasion, and the impact of genomic instability. We highlight the critical insights gained from single-cell technologies, such as single-cell transcriptomics, genomics, and multiomics, which have enabled a detailed understanding of MM heterogeneity at the cellular level, facilitating the identification of rare cell populations and mechanisms of drug resistance. Despite the promise of advanced technologies, MM remains an incurable disease and challenges remain in their clinical application, including high costs, data complexity, and the need for standardized bioinformatics and ethical considerations. This review emphasizes the importance of continued research and collaboration to address these challenges, ultimately aiming to enhance personalized treatment strategies and improve patient outcomes in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b4ede18b0be075e37d640173950681509758ae8" target='_blank'>
              Multiple Myeloma Insights from Single-Cell Analysis: Clonal Evolution, the Microenvironment, Therapy Evasion, and Clinical Implications
              </a>
            </td>
          <td>
            Sihong Li, Jiahui Liu, Madeline Peyton, Olivia Lazaro, Sean D. McCabe, Xiaoqing Huang, Yunlong Liu, Zanyu Shi, Zhiqi Zhang, Brian A. Walker, 
          </td>
          <td>2025-02-14</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a65325c104ad2755155bebd8b8e82939a9a6fc" target='_blank'>
              Transposon mediated functional genomic screening for BRAF inhibitor resistance reveals convergent Hippo and MAPK pathway activation events
              </a>
            </td>
          <td>
            Li Chen, Iulian Pruteanu-Malinici, A. Dastur, Xunqin Yin, Dennie Frederick, R. I. Sadreyev, Cyril H. Benes
          </td>
          <td>2025-01-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, , Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Cancer evolves continuously over time through a complex interplay of genetic, epigenetic, microenvironmental, and phenotypic changes. This dynamic behavior drives uncontrolled cell growth, metastasis, immune evasion, and therapy resistance, posing challenges for effective monitoring and treatment. However, today's data-driven research in oncology has primarily focused on cross-sectional analysis using data from a single modality, limiting the ability to fully characterize and interpret the disease's dynamic heterogeneity. Advances in multiscale data collection and computational methods now enable the discovery of longitudinal multimodal biomarkers for precision oncology. Longitudinal data reveal patterns of disease progression and treatment response that are not evident from single-timepoint data, enabling timely abnormality detection and dynamic treatment adaptation. Multimodal data integration offers complementary information from diverse sources for more precise risk assessment and targeting of cancer therapy. In this review, we survey methods of longitudinal and multimodal modeling, highlighting their synergy in providing multifaceted insights for personalized care tailored to the unique characteristics of a patient's cancer. We summarize the current challenges and future directions of longitudinal multimodal analysis in advancing precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a35a93de0536a7f3b11801216a34e22a7ecf3075" target='_blank'>
              Advancing Precision Oncology Through Modeling of Longitudinal and Multimodal Data
              </a>
            </td>
          <td>
            Luoting Zhuang, Stephen H. Park, Steven J. Skates, A.E. Prosper, Denise R. Aberle, William Hsu
          </td>
          <td>2025-02-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Structural variation is a major contributor to human diversity, adaptation, and disease. Simple structural variant (SV) types include deletions, insertions, duplications, inversions, and translocations, and SVs account for most of the variable bases between genomes. Complex structural variants (CSVs) that consist of one or more simple events in cis appear more frequently in diseases and cancers where DNA repair, apoptosis, and cell cycle checkpoints are compromised, although CSVs can also appear in germline genome sequences of healthy individuals. CSVs are often characterized by short tracts of homology or no homology, and while CSVs are more prevalent in complex regions that contain large repeats, smaller stretches of homology can also enable their formation across more unique loci. Long-read assemblies have increased the size of detectable SVs and expanded variant detection into more complex regions of the genome, and while they reconstruct CSVs, methods for identifying CSVs from assemblies is limited. Here, we have developed a new assembly-based approach to trace through complex loci rather than relying upon reference representations of alignments. We can now access CSVs in large complex segmental duplications, reveal structures that were previously unknown, and identify SV breakpoints with greater accuracy. We find 72 large CSVs per genome and 128 unique complex structures and CSVs in highly repetitive regions can now be detected including several distinct complex events in repetitive NBPF genes that was not previously callable with short-read or long-read CSV methods. This approach is implemented within a key assembly-based variant calling tool, PAV, and represents a substantial improvement identifying complex variants now ascertainable from contiguous genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbfc7a889c92ffa27e1ac1ac1e0ce4aa98dae5ef" target='_blank'>
              Large complex structural rearrangements in human genomes harbor cryptic structures
              </a>
            </td>
          <td>
            P. Audano, Carolyn A. Paisie, Christine R. Beck
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Autophagy is a critical cellular process that maintains homeostasis by recycling damaged or aberrant components. This process is orchestrated by a network of proteins that form autophagosomes, which engulf and degrade intracellular material. In cancer, autophagy plays a dual role: it suppresses tumor initiation in the early stages but supports tumor growth and survival in advanced stages. Chronic myeloid leukemia (CML), a hematological malignancy, is characterized by the Philadelphia chromosome, a chromosomal abnormality resulting from a translocation between chromosomes 9 and 22. Autophagy has emerged as a key factor in CML pathogenesis, promoting cancer cell survival and contributing to resistance against tyrosine kinase inhibitors (TKIs), the primary treatment for CML. Targeting autophagic pathways is being actively explored as a therapeutic approach to overcome drug resistance and enhance cancer cell death. Recent research highlights the intricate interplay between autophagy and CML progression, underscoring its role in disease biology and treatment outcomes. This review aims to provide a comprehensive analysis of the molecular and cellular mechanisms underlying CML, with a focus on the therapeutic potential of targeting autophagy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca600b2dcbd0c4958d816d4cfd021e7ec09edd48" target='_blank'>
              Autophagy and Cancer: Insights into Molecular Mechanisms and Therapeutic Approaches for Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            M. A. Kausar, Sadaf Anwar, Yusuf Saleem Khan, Ayman A. Saleh, Mai Ali Abdelfattah Ahmed, Simran Kaur, Naveed Iqbal, W. Siddiqui, M. Najm
          </td>
          <td>2025-02-02</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Metabolic reprogramming, particularly lipid metabolism, is a hallmark of cancer progression and a critical vulnerability in prostate cancer (PCa). Two pivotal studies, one by Lu et al. and the other by Dairo et al., have illuminated the roles of HOXB13, a transcriptional regulator, and fatty acid synthase (FASN), a key enzyme in lipid biosynthesis, in the metabolic dysregulation of PCa. The Lu et al. study highlights HOXB13’s role in androgen receptor (AR)-independent PCa, where it regulates lipid metabolism via epigenetic mechanisms involving histone deacetylase HDAC3. A G84E mutation in HOXB13 disrupts this regulation, leading to increased lipid synthesis and a pro-metastatic phenotype. The Dairo et al. study demonstrates that FASN hypomethylation, coupled with increased expression, drives lipid biosynthesis critical for tumor growth. Both studies establish a link between HOXB13 mutations and FASN dysregulation, underscoring their interplay in PCa biology. Therapeutically, pharmacological inhibition of FASN mitigates the aggressive features of HOXB13-deficient or mutant PCa, highlighting lipid metabolism as a promising target. Despite their strengths, including robust methodologies, limitations include reliance on preclinical models and the need for broader patient diversity. These studies collectively emphasize the potential of metabolic and epigenetic interventions for precision medicine in PCa, paving the way for novel therapies targeting lipid metabolism in patients with specific genetic and epigenetic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc58c2230658f37cc79a630362aeb84c369fdaa5" target='_blank'>
              Metabolic Reprogramming in Prostate Cancer: The Roles of HOXB13 and FASN in Tumor Progression and Therapy
              </a>
            </td>
          <td>
            Faisal Saeed, Lara Harik
          </td>
          <td>2025-01-14</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Comparative genomic studies of Marek’s disease virus (MDV) have suggested that attenuated and virulent strains share >98% sequence identity. However, these estimates fail to account for variation in regions of the MDV genome harboring tandem repeats. To resolve these loci and enable assessments of intrapopulation diversity, we used a PacBio Sequel II platform to sequence MDV strains CVI988/Rispens (attenuated), HPRS-B14 (virulent), Md5 (very virulent) and 675A (very virulent plus). This approach enabled us to identify patterns of variation in tandem repeat regions that are consistent with known phenotypic differences across these strains. We also found CVI988/Rispens variants showing a 4.3-kb deletion in the Unique Short (US) region, resulting in the loss of five genes. These findings support a potential link between MDV tandem repeats and phenotypic traits like virulence and attenuation, and demonstrate that DNA viruses can harbor high levels of intrapopulation diversity in tandem repeat regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9048b732c74cf2e0b72ef6607e53274d6fd6385" target='_blank'>
              High-Fidelity Long-Read Sequencing of an Avian Herpesvirus Reveals Extensive Intrapopulation Diversity in Tandem Repeat Regions
              </a>
            </td>
          <td>
            Alejandro Ortigas-Vásquez, Christopher D. Bowen, Daniel W. Renner, S. Baigent, Yaoyao Zhang, Yongxiu Yao, Venugopal Nair, Dave A. Kennedy, Moriah L. Szpara
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Background Genomic data is essential for clinical decision-making in precision oncology. Bioinformatic algorithms are widely used to analyze next-generation sequencing (NGS) data, but they face two major challenges. First, these pipelines are highly complex, involving multiple steps and the integration of various tools. Second, they generate features that are human-interpretable but often result in information loss by focusing only on predefined genetic properties. This limitation restricts the full potential of NGS data in biomarker extraction and slows the discovery of new biomarkers in precision oncology. Methods We propose an end-to-end deep learning (DL) approach for analyzing NGS data. Specifically, we developed a multiple instance learning DL framework that integrates somatic mutation sequences to predict two compound biomarkers: microsatellite instability (MSI) and homologous recombination deficiency (HRD). To achieve this, we utilized data from 3,184 cancer patients obtained from two public databases: The Cancer Genome Atlas (TCGA) and the Clinical Proteome Tumor Analysis Consortium (CPTAC). Results Our proposed deep learning method demonstrated high accuracy in identifying clinically relevant biomarkers. For predicting MSI status, the model achieved an accuracy of 0.98, a sensitivity of 0.95, and a specificity of 1.00 on an external validation cohort. For predicting HRD status, the model achieved an accuracy of 0.80, a sensitivity of 0.75, and a specificity of 0.86. Furthermore, the deep learning approach significantly outperformed traditional machine learning methods in both tasks (MSI accuracy, p-value = 5.11×10−18; HRD accuracy, p-value = 1.07×10−10). Using explainability techniques, we demonstrated that the model’s predictions are based on biologically meaningful features, aligning with key DNA damage repair mutation signatures. Conclusion We demonstrate that deep learning can identify patterns in unfiltered somatic mutations without the need for manual feature extraction. This approach enhances the detection of actionable targets and paves the way for developing NGS-based biomarkers using minimally processed data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad073d80b11f20c88ed32369a6393b6a5510e50a" target='_blank'>
              Deep Learning for Biomarker Discovery in Cancer Genomes
              </a>
            </td>
          <td>
            Michaela Unger, C. Loeffler, Laura Žigutytė, Srividhya Sainath, Tim Lenz, Julien Vibert, Andreas Mock, S. Fröhling, Trevor A. Graham, Zunamys I. Carrero, J. N. Kather
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cell cycle checkpoints are stringent quality control mechanisms that regulate cell cycle progression and division. Cancer cells often develop a dependency on the G2/M cell cycle checkpoint to facilitate DNA repair and resolve intrinsic or therapy-induced DNA damage. This dependency leads to therapy resistance, continuous cell division, and disease progression. Targeting G2/M checkpoints has been heavily pursued over the past two decades and has progressed into clinical studies. Recent genome-scale functional genomic studies have revealed that Myt1 kinase, an essential but previously overlooked molecule for the G2/M checkpoint, is a promising target for multiple types of cancers. In this work, we summarize the latest discoveries in molecular targeting of Myt1 kinase and discuss the challenges and limitations in expanding its clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17deb173d71c056d6283f804746aaf46a913ddbb" target='_blank'>
              Myt1 kinase: An Emerging Cell Cycle Regulator for Cancer Therapeutics.
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Javeria Zaheer, Diego Luis Ribeiro, Chunzhang Yang
          </td>
          <td>2025-01-16</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Objectives Unlike other diseases, cancer is not just a genome disease but should broadly be viewed as a disease of the cellular machinery. Therefore, integrative multifaceted approaches are crucial to understanding the complex nature of cancer biology. Bcl-2 (B-cell lymphoma 2), encoded by the human Bcl-2 gene, is a critical anti-apoptotic protein that regulates cell death pathways, primarily by inhibiting apoptosis. It plays a pivotal role in maintaining cellular homeostasis by preventing premature or excessive cell death. Genetic variations and dysregulation of Bcl-2 are particularly significant in cancer, as they disrupt the normal apoptotic machinery, enabling cancer cells to evade programmed cell death. Single nucleotide polymorphisms (SNPs) are considered viable diagnostic and therapeutic biomarkers for various cancers. Therefore, this study explores the association between SNPs in Bcl-2 and the structural, functional, protein-protein interactions (PPIs), drug binding and dynamic characteristics. Methods Comprehensive cross-validated bioinformatics tools and molecular dynamics (MD) simulations. Multiple sequence, genetic, structural and disease phenotype analyses were applied in this study. Results Analysis revealed that out of 130 mutations, approximately 8.5% of these mutations were classified as pathogenic. Furthermore, two particular variants, namely, Bcl-2G101V and Bcl-2F104L, were found to be the most deleterious across all analyses. Following 500 ns, MD simulations showed that these mutations caused a significant distortion in the protein conformational, protein-protein interactions (PPIs), and drug binding landscape compared to Bcl-2WT. Conclusion Despite being a predictive study, the findings presented in this report would offer a perspective insight for further experimental investigation, rational drug design, and cancer gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f67dee38c9954b65ace573045ef349b37771a6" target='_blank'>
              Integrative genomic analyses combined with molecular dynamics simulations reveal the impact of deleterious mutations of Bcl-2 gene on the apoptotic machinery and implications in carcinogenesis
              </a>
            </td>
          <td>
            G. Elamin, Zhichao Zhang, Depika Dwarka, K. Kasumbwe, John Mellem, Nompumelelo P. Mkhwanazi, P. Madlala, Mahmoud E. S. Soliman
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Genetic variation underpins evolutionary change, but accumulation of slightly deleterious mutations also increases mutation load. There are multiple factors affecting the extent of load such as population size and breeding system, yet other potential determinants remain unexplored. A common macromutation, whole-genome duplication (WGD) occurs broadly across Eukaryotes, yet we lack a clear understanding of how WGD impacts neutral and selective processes within a population. Using forward simulations and empirical analysis of 632 short- and 16 long-read sequenced individuals of Arabidopsis arenosa (23 diploid and 42 natural autotetraploid populations), we test for the effects of WGD on genome-wide diversity and mutation load. Our simulations show how genetic variation gradually rises in autotetraploids due to increase of mutational target size. Moreover, mutation load increases due to relaxed purifying selection when deleterious mutations are masked by additional chromosome copies. Empirical data confirm these patterns, showing significant increase in nucleotide diversity, ratios of non-synonymous to synonymous SNPs, and number of indels and large structural variants in A. arenosa autotetraploids. However, a rather modest increase in load proxies together with a broad distribution and niche of autotetraploids suggests load accumulation has not (yet) limited their successful expansion. Overall, we demonstrate a complex interplay between neutral processes and purifying selection in shaping genetic variation following WGD and highlight ploidy as an important determinant of genetic diversity and mutation load in natural populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86bbacd2db01892da50f5a93e5f2a705f35e9543" target='_blank'>
              Whole-genome duplication increases genetic diversity and load in outcrossing Arabidopsis
              </a>
            </td>
          <td>
            Jakub Vlček, Tuomas Hämälä, Cristina Vives Cobo, Emma Curran, G. Šrámková, T. Slotte, R. Schmickl, Levi Yant, Filip Kolář
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Targeted therapies like Venetoclax have increased the options available to acute myeloid leukemia (AML) patients, but survival remains poor due to drug resistance and disease relapse. We found that the translation initiation factor EIF4A1, which unwinds complex mRNA structures in the 5’ UTR of oncogenic transcripts, is highly expressed in AML stem- and progenitor-like cells. Inhibiting eIF4A with the small molecule Zotatifin reduces translation of transcripts related to the cell cycle and survival. This results in downregulation of AKT, STAT-5, and MCL-1 and underlies synergy of Zotatifin with Venetoclax. The drug combination promotes apoptosis across AML genotypes, while the effect on healthy blood cells is limited. Using in vivo relapsed and refractory AML patient-derived xenograft models, the combination significantly suppressed tumor burden and prolonged survival of xenografted mice. These results support eIF4A-mediated protein translation as a therapeutic target in AML. SIGNIFICANCE Despite advances in targeted therapies, the 5-year survival rate for acute myeloid leukemia remains around 32%. The efficacy of existing treatments may be improved with the addition of Zotatifin, an inhibitor targeting the translation initiation factor eIF4A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8672d4c71b8c253c15ee73780df3c698e781744" target='_blank'>
              Research Brief: Translation Initiation Represents an Acute Myeloid Leukemia Cell Vulnerability That Can Be Co-Targeted With BCL-2 Inhibition
              </a>
            </td>
          <td>
            Y. Lee, Jonathan D. Good, Noha Awais, Benjamin K. Eschle, Maia S. Lineberry, Jessica L. Teo, Sarah Bibeau, Morgan Wollman, George Karadimov, Nurefsan Sariipek, Jean Acosta, Jenny Noel, David Cucchi, E. DePasquale, Prafulla Gokhale, Fadi J. Najm, Richard M. Stone, Jacqueline S. Garcia, Andrew A. Lane, Peter van Galen
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Epigenetics plays a critical role relative to other branches of genetics, as it makes it possible to regulate gene expression without changing the nucleotide sequence of DNA molecules. This process allows cells to respond to external signals and adapt to  changes in  the  environment while keeping their genetic information intact. The  main mechanisms of  epigenetic regulation include DNA methylation, posttranslational modifications of  histones, chromatin remodeling and  regulation through non-coding RNAs. These processes play a key role in fundamental biological processes such as cellular differentiation, organismal development, and adaptation to environmental conditions.Disturbances in epigenetic mechanisms can lead to various pathologies, including cancer, neurological and  autoimmune diseases. Understanding the  mechanisms of  epigenetic regulation opens new avenues for  the  development of  targeted therapies that can correct abnormal epigenetic profiles without changing the DNA structure itself.In  recent years, the  development and  application of  innovative technologies, such  as  the  CRISPR/Cas9 genome editing system, have significantly expanded the ability to study epigenetic mechanisms and their relationship with diseases. These technologies allow not only a deeper understanding of epigenetic changes, but also the development of new therapeutic approaches, especially in the field of oncology. Research in  epigenetics is  also focusing on  the  interaction between epigenetic changes and the immune system, which opens new perspectives for the development of immunotherapies. The search for new markers of epigenetic disorders and therapeutic agents may lead to the development of individualized treatments that take into account the unique epigenetic profile of each patient.The role of epigenetic modifications in the development of diseases and the creation of new therapeutic strategies cannot be overestimated. Recent research in this area is revealing the potential of epigenetic approaches to treat a wide range of diseases, ushering in a new era in medicine where understanding and correcting epigenetic changes will be the key to effective treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d047bbd846068847a5bb8e163e24692ce5b5c48" target='_blank'>
              Epigenetic modulation in medicine: Regulation of gene expression in the context of pathogenesis and therapy
              </a>
            </td>
          <td>
            M. A. Omarov, A. R. Mulyukov, R. V. Khalitov, S. I. Safarov, G. U. Ayupova, O. N. Demianenko, V. V. Butenko, A. A. Safronova, R. R. Zainetdinova, O. Y. Ianturina, A. A. Miroshnikov
          </td>
          <td>2024-12-28</td>
          <td>Acta Biomedica Scientifica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In the field of RNA therapy, innovative approaches based on adenosine deaminases acting on RNA (ADAR)-mediated site-directed RNA editing (SDRE) have been established, providing an exciting opportunity for RNA therapeutics. ADAR1 and ADAR2 enzymes are accountable for the predominant form of RNA editing in humans, which involves the hydrolytic deamination of adenosine (A) to inosine (I). This inosine is subsequently interpreted as guanosine (G) by the translational and splicing machinery because of their structural similarity. Intriguingly, the novel SDRE system leverages this recoding ability of ADAR proteins to correct the pathogenic G to A nucleotide mutations through a short, engineered guide RNA (gRNA). Thus, ADAR-mediated SDRE is emerging as a powerful tool to manipulate the genetic information at the RNA level and correct disease-causing mutations without causing damage to the genome. Further it is emerging as a new instrument for personalized medicine, since treatments can be tailored to the unique genetic mutations present in an individual patient. In this short review, we aimed to described the main approached bases on ADARs activity, highlighting their advantages and disadvantages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4becd1d76f0fd4ff859476396dcb8dbd5450769" target='_blank'>
              ADAR Therapeutics as a New Tool for Personalized Medicine
              </a>
            </td>
          <td>
            Matteo Bertoli, Luca La Via, A. Barbon
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb6bea0ac205c045a3acd7f52f1841e20780dc8e" target='_blank'>
              Utilizing sc-linker to integrate single-cell RNA sequencing and human genetics to identify cell types and driver genes associated with non-small cell lung cancer
              </a>
            </td>
          <td>
            Yangfan Zhou, Liang Zhao, Meimei Cai, Doudou Luo, Y. Pang, Jianhao Chen, Qicong Luo, Qin Lin
          </td>
          <td>2025-01-23</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genetic studies of haematological cancers have pointed out the heterogeneity of leukaemia in its different subpopulations, with distinct mutations and characteristics, impacting the treatment response. Next-generation sequencing (NGS) and genome-wide analyses, as well as single-cell technologies, have offered unprecedented insights into the clonal heterogeneity within the same tumour. A key component of this heterogeneity that remains unexplored is the intracellular metabolome, a dynamic network that determines cell functions, signalling, epigenome regulation, immunity and inflammation. Understanding the metabolic diversities among cancer cells and their surrounding environments is therefore essential in unravelling the complexities of leukaemia and improving therapeutic strategies. Here, we describe the currently available methodologies and approaches to addressing the dynamic heterogeneity of leukaemia progression. In the second section, we focus on metabolic leukaemic vulnerabilities in acute myeloid leukaemia (AML) and acute lymphoblastic leukaemia (ALL). Lastly, we provide a comprehensive overview of the most interesting clinical trials designed to target these metabolic dependencies, highlighting their potential to advance therapeutic strategies in leukaemia treatment. The integration of multi-omics data for cancer identification with the metabolic states of tumour cells will enable a comprehensive “micro-to-macro” approach for the refinement of clinical practices and delivery of personalised therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cc7b64900de2dfae5390d10959b2a1e62e8bf1" target='_blank'>
              Unlocking the Heterogeneity in Acute Leukaemia: Dissection of Clonal Architecture and Metabolic Properties for Clinical Interventions
              </a>
            </td>
          <td>
            Martina Maria Capelletti, Orsola Montini, Emilio Ruini, Sarah Tettamanti, Angela Maria Savino, J. Sarno
          </td>
          <td>2024-12-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="CAR-T therapies utilizing T cells engineered with chimeric antigen receptors (CARs) have revolutionized the treatment of hematologic and immune-related malignancies. However, the anti-tumor capability of engineered CAR-T cells are often not persistent, which greatly dampen its clinical efficacy. To address this limitation, massive parallel genetic screens were widely used to identified novel targets and regulators that enhance T cell anti-tumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducting a pathway enrichment analysis. A T cell screen perturbation score (TPS) was employed to quantifies the impact of the gene on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease and T cell-derived cancers were analyzed to pick up gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 27 T-cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is made easily accessible through our webserver and allows users to interactively explore the impact of query genes on T cell function based on prior screen data and our TPS scoring module. TCPGdb is publicly accessible at http://tcpgdb.sidichenlab.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a099e229f2c35d38ee6b66c8422aec7d483d003c" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4ce21cb54a65498b2c72289ec6bd7eb2e7dc62" target='_blank'>
              Characterization of shared neoantigens landscape in Mismatch Repair Deficient Endometrial Cancer
              </a>
            </td>
          <td>
            E. De Paolis, C. Nero, E. Micarelli, Guido Leoni, A. Piermattei, R. Trozzi, Elisa Scarselli, A. D’Alise, Luciano Giacò, M. De Bonis, A. Preziosi, Gennaro Daniele, Diletta Piana, T. Pasciuto, Gianfranco Zannoni, Angelo Minucci, G. Scambia, Andrea Urbani, Francesco Fanfani
          </td>
          <td>2024-12-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Despite advancements in the treatment of advanced prostate cancer, including metastatic castration-resistant prostate cancer (mCRPC), the disease remains lethal, often developing resistance to conventional therapies such as androgen deprivation therapy (ADT) and chemotherapy. Therapy resistance poses a significant challenge, particularly in aggressive forms of prostate cancer. Data on TP53 mutations, especially gain-of-function (GOF) mutations, and their specific impact on prostate cancer metabolism remain limited, underscoring the need for a deeper understanding to uncover new therapeutic avenues. This study aimed to explore the metabolic consequences of TP53 GOF mutations (R175H and R273H) in prostate cancer progression using metabolomics analysis. Approximately 300 metabolites were significantly altered across PC3-GFP (KO), PC3-TP53WT (WT), and PC3-TP53 mutant (R175H & R273H) models. Key findings revealed dysregulation of essential metabolites, such as α-ketoglutarate and spermidine, which support cancer cell survival and progression. These insights not only enhance our understanding of the interplay between TP53 mutations and metabolic alterations but also identify potential biomarkers for prognosis and therapeutic targeting in therapy-resistant prostate cancer. In the TP53-R175H mutant, numerous essential metabolites were dysregulated (downregulated: glutamine, GAP/DHAP, and α-ketoglutarate; Upregulated: deoxyuridine, spermidine, and pseudouridine) when compared with the control (KO) thereby causing metabolic shifts that favor cancer cell survival. Conversely, metabolites such as GlaNAC/GlcNAC, adenosine, acetoacetic acid, acetylcholine, octopamine, itaconic acid, lactic acid, and NADP+ are all downregulated in the R273H mutant. While spermine, spermidine, N-acetyl-DL-serine, pseudouridine, deoxyguanosine, sucrose, and AMP/dGMP are upregulated by the R273H mutant but downregulated in GFP (KO). These findings showed that TP53 GOF mutations (R175H & R273H) cause metabolic alterations in numerous metabolites essential for tumor survival and progression. Furthermore, it may provide therapeutic benefits by identifying appropriate biomarkers that could be used as diagnostic and/or prognostic tools for prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6235a81e6d316a547376d1bb83acc9177dd3e97b" target='_blank'>
              Impact of TP53 Gain-of-Function Mutations on Metabolic Reprogramming in Prostate Cancer Progression
              </a>
            </td>
          <td>
            Farida Jazuli Usman, D. G. Gupta, S. Abdulkadir, Ibrahim Muhammad Usman, Adamu Jibrin Alhassan, Young A Yoo, Haris Jaafar Bello, Binta G. Kurfi, K. Babagana
          </td>
          <td>2025-01-10</td>
          <td>Asian Journal of Biochemistry, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9aa45e1efe02786e2e4534cf1ae408e65c7ffadc" target='_blank'>
              To describe the subsets of malignant epithelial cells in gastric cancer, their developmental trajectories and drug resistance characteristics
              </a>
            </td>
          <td>
            Tingting Xu, Tianying Zhang, Yan Sun, Sijia Wu
          </td>
          <td>2025-01-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="With the advancement of single-cell sequencing, we are now capable of analyzing the heterogeneity within tumors with increased precision, revealing the complex interactions between the tumor microenvironment and immune cell subgroups. Studies mentioned in the articles demonstrate significant cell heterogeneity in various cancers at genomic, transcriptomic, and proteomic levels. In particular, single-cell technologies aid in identifying cell-type-specific genetic lineage effects, for example, in the context of influenza infection and T cell surveillance. Current study focuses on the application of single-cell sequencing in cancer immunotherapy and the exploration of novel antigen identification and optimization of therapeutic strategies. The introduction of new strategies, such as the refinement of electrochemical immuno-sensors, synthesis of novel small molecule drugs, and optimized detection of genetic mutations, contributes collectively to the accurate identification of tumor markers and innovative development of therapeutic methods. The integration of treatment schemes and patient survival analysis further guides the implementation of personalized medicine. In summary, the application of single-cell sequencing and its integration with the optimization of therapeutic strategies promises revolutionary advancements in the field of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607f39c834cbc474d463e6168f60ce125e47f1f7" target='_blank'>
              Research On the Identification of Novel Antigens and The Optimization of Therapeutic Strategies in Cancer Immunotherapy Through Single-Cell Sequencing
              </a>
            </td>
          <td>
            Peixuan Li
          </td>
          <td>2024-12-24</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06ca40e33657af743c749690c1200f7526a655d0" target='_blank'>
              NKX6.3 modulation of mitotic dynamics and genomic stability in gastric carcinogenesis
              </a>
            </td>
          <td>
            J. H. Yoon, J. Kim, J. Eun, H. Ashktorab, Duane T. Smoot, S. Nam, Won Sang Park
          </td>
          <td>2025-01-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that plays a crucial role in regulating gene expression and maintaining genomic stability. With the advancement in sequencing technology, methylation studies have provided valuable insights into the diagnosis of rare diseases through the various identification of episignatures, epivariation, epioutliers, and allele-specific methylation. However, current methylation studies are not without limitations. This mini-review explores the current understanding of DNA methylation in rare diseases, highlighting the key mechanisms and diagnostic potential, and emphasizing the need for advanced methodologies and integrative approaches to enhance the understanding of disease progression and design more personable treatment for patients, given the nature of rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b0846fe63347313e7e0457ade21ff242a0edeb" target='_blank'>
              Expanding Upon Genomics in Rare Diseases: Epigenomic Insights
              </a>
            </td>
          <td>
            Jia W Tan, Emily J. Blake, Joseph D. Farris, Eric W. Klee
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/789273322c6b964dab5a55a3cbb9da30d7bda05f" target='_blank'>
              Combining Single-Cell RNA Sequencing and Mendelian Randomization to Explore Novel Drug Targets for Parkinson's Disease.
              </a>
            </td>
          <td>
            Xiaolong Wu, Kailiang Wang, Qinghua Li, Yuqing Zhang, Penghu Wei, Yongzhi Shan, Guoguang Zhao
          </td>
          <td>2025-01-31</td>
          <td>Molecular neurobiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Adaptation to challenging environmental conditions is crucial for the survival/fitness of all organisms. Alongside genetic mutations that provide adaptive potential during environmental challenges, epigenetic modifications offer dynamic, reversible, and rapid mechanisms for regulating gene expression in response to environmental changes in both evolution and daily life, without altering DNA sequences or relying on accidental favorable mutations. The widespread conservation of diverse epigenetic mechanisms - like DNA methylation, histone modifications, and RNA interference across diverse species, including plants - underscores their significance in evolutionary biology. Remarkably, environmentally induced epigenetic alterations are passed to daughter cells and inherited transgenerationally through germline cells, shaping offspring phenotypes while preserving adaptive epigenetic memory. Throughout anthropoid evolution, epigenetic modifications have played crucial roles in: i) suppressing transposable elements and viral genomes intruding into the host genome; ii) inactivating one of the X chromosomes in female cells to balance gene dosage; iii) genetic imprinting to ensure expression from one parental allele; iv) regulating functional alleles to compensate for dysfunctional ones; and v) modulating the epigenome and transcriptome in response to influence from the gut microbiome among other functions. Understanding the interplay between environmental factors and epigenetic processes may provide valuable insights into developmental plasticity, evolutionary dynamics, and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f7f5656f1a791fd7b047008e717c5695b95280c" target='_blank'>
              Epigenetics in evolution and adaptation to environmental challenges: pathways for disease prevention and treatment.
              </a>
            </td>
          <td>
            H. Abdolmaleky, S. Nohesara, Jin-Rong Zhou, S. Thiagalingam
          </td>
          <td>2025-02-13</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Introduction: Mutations in KEAP1 and NFE2L2 (NFR2) occur in approximately 20% of non-small cell lung cancer (NSCLC) patients and are associated with resistance to radiotherapy and systemic treatments. In previous research, we found that not all KEAP1 mutations found in NSCLC patients are pathogenic. These benign mutations exhibit a wild-type phenotype that does not confer therapy resistance. With only roughly 100 of 13,000 (<1%) potential single amino acid substitution mutations in KEAP1 characterized thus far, here we comprehensively evaluated all possible point mutations in KEAP1 using deep mutational scanning (DMS). Methods: The High-throughput Mutagenesis by Integrated TilEs (MITE) method was used to generate the KEAP1 DMS library. In short, the KEAP1 sequence was segmented into 21 variable regions (tiles), each comprising a 90-base pair variable region flanked by a 30-base pair constant region. Each codon position included the 20 possible natural amino acids plus a stop codon, for a total of 13,076 KEAP1 variants. The tiles were then assembled to ensure that each full-length KEAP1 construct contained only a single amino acid substitution. The KEAP1 DMS lentivirus library was transduced into H1299-KEAP1NULLcells, followed by selective pressure with H2O2, radiation, or vehicle control. The KEAP1 ORF was then amplified from surviving cell genomic DNA, and mutations were quantified using next-generation sequencing. Functional annotations were derived through mutation enrichment analysis. Results: Both H2O2 and radiation selective pressure of the KEAP1 DMS library induced a bimodal distribution of KEAP1 mutations, with cells harboring KEAP1 nonsense mutations being enriched and cells with silent mutations being depleted. By setting a threshold of 95% specificity for silent mutations to differentiate between benign and pathogenic mutations, we identified 94% of nonsense mutations and 88% of previously characterized pathogenic mutations as pathogenic. While most KEAP1 mutations were categorized as benign (64%), analysis of mutations in TCGA NSCLC patients showed pathogenic mutations were enriched (72%). Mapping mutation phenotypes onto a KEAP1 structural model revealed an enrichment of pathogenic mutations at critical functional sites, including the NRF2 binding pocket of the Kelch domain, the BTB homodimerization interface, and the hydrophobic core of the oxidative stress-sensing IVR domain. Conclusions: Through deep mutational scanning experiments, we generated a comprehensive list of pathogenic KEAP1 single amino acid substitutions. These results align with previously known biological functions and prior single amino acid substitutions characterized in the literature. These findings will facilitate the identification of benign and pathogenic KEAP1 mutations in clinical settings, enabling personalized radiotherapeutic and systemic therapy strategies for patients harboring KEAP1 mutations.
 Citation Format: Noah Kastelowitz, Shashank Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn. Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr BO23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f223327077f38ac09a18b5895edd5fce4b10c15a" target='_blank'>
              Abstract B023: Deep mutational scanning of KEAP1 to identify mutations resistant to radiation therapy and oxidative stress
              </a>
            </td>
          <td>
            Noah Kastelowitz, S. Shrishrimal, Soyeong Jun, Anni MY Zhang, Rui Wang, Ilayda Ilerten, Maximilian Diehn
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3834da21049e516e26734af02f73ef17c4698c" target='_blank'>
              Single cell RNA sequencing improves the next generation of approaches to AML treatment: challenges and perspectives
              </a>
            </td>
          <td>
            Zahra Khosroabadi, Samaneh Azaryar, Hassan Dianat-Moghadam, Zohreh Amoozgar, Mohammadreza Sharifi
          </td>
          <td>2025-01-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Stemness-associated cell states are linked to chemotherapy resistance in AML. We uncovered a direct mechanistic link between expression of the stem cell transcription factor GATA2 and drug resistance. The GATA-binding protein 2 (GATA2) plays a central role in blood stem cell generation and maintenance. We find substantial intra- and inter-patient variability in GATA2 expression across AML patient samples. GATA2 expression varies by molecular subtype and has been linked to outcome. In a murine model, KMT2A-MLL3 driven AML originating from a stem cell or immature progenitor cell population have higher Gata2 expression and are more resistant to the standard AML chemotherapy agent doxorubicin. Deletion of Gata2 resulted in more robust induction of p53 following exposure to doxorubicin. ChIP-Seq, RNA-Seq and functional studies revealed that GATA2 regulates the expression of RASSF4, a modulator of the p53 inhibitor MDM2. GATA2 and RASSF4 are anti-correlated in human cell lines and AML patient cell bulk and single cell expression datasets. Knockdown of Rassf4 in Gata2 low cells resulted in doxorubicin or nutlin-3 resistance. Conversely, overexpression of Rassf4 results in sensitization of cells expressing high levels of Gata2. Finally, doxorubicin and nutlin-3 are synergistic in Gata2-high murine AML, as well as AML patient samples. We discovered a previously unappreciated role for GATA2 in dampening p53-mediated apoptosis via transcriptional regulation of RASSF4, a modulator of MDM2. This role for GATA2 directly links the expression of a stemness associated transcription factor to chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149e4d0de77d706d929171b654f103344701ea6a" target='_blank'>
              GATA2 links stemness to chemotherapy resistance in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Fatemeh Alikarami, H. Xie, Simone S. Riedel, Haley Goodrow, D. Barrett, Leila Mahdavi, Alexandra Lenard, Changya Chen, T. Yamauchi, Etienne Danis, Zhendong Cao, Vu L. Tran, M. Jung, Yapeng Li, Hua Huang, Junwei Shi, Kai Tan, D. Teachey, E. Bresnick, Tobias Neff, Kathrin M Bernt
          </td>
          <td>2025-01-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is the most common soft tissue sarcoma in children, but the outcomes of high-grade RMS patients remain poor, underscoring the critical need for novel therapeutic strategies. Although metabolic pathways in RMS are incompletely characterized, emerging evidence suggests that metabolic adaptations in RMS resemble those in other malignancies. Here, we identify elevated cholesterol biosynthesis driven by the PROX1 transcription factor as a defining feature of RMS. Our findings demonstrate that the cholesterol biosynthesis pathway is essential for RMS cell growth, proliferation, and survival. Blocking this pathway through genetic or pharmacological inhibition of the key cholesterol biosynthesis enzymes significantly impairs RMS cell proliferation, halts cell cycle progression, and triggers apoptosis through activation of endoplasmic reticulum stress pathways. We furthermore validate the critical role of cholesterol biosynthesis in RMS progression in tumor xenograft models, demonstrating that silencing of the DHCR7 gene significantly suppresses tumor growth. Transcriptomic analysis revealed widespread downregulation of cell cycle-related genes following DHCR7 silencing, further supporting the role of cholesterol metabolism in cell cycle regulation. These results highlight the vulnerability of RMS cells to cholesterol biosynthesis inhibition and suggest that targeting this metabolic pathway as a promising therapeutic approach for improving RMS outcomes. Our findings provide a rationale for the development of novel therapies targeted to cholesterol biosynthesis in this aggressive cancer. Significance This study reveals that targeting cholesterol biosynthesis in rhabdomyosarcoma induces ER-stress, apoptosis and cell cycle arrest, highlighting a potential therapeutic strategy for treating this aggressive pediatric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1168b7a6afff81a3b6df61f48c435d90b22e7cf3" target='_blank'>
              Targeting de novo cholesterol synthesis in rhabdomyosarcoma induces cell cycle arrest and triggers apoptosis through ER stress-mediated pathways
              </a>
            </td>
          <td>
            N. Y. Gizaw, Kalle Kolari, Kari Alitalo, R. Kivelä
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Essentiality studies have traditionally focused on coding regions, often overlooking other small genetic regulatory elements. To address this, we obtained a high-resolution essentiality map at near-single-nucleotide precision of the genome-reduced bacterium Mycoplasma pneumoniae, combining transposon libraries containing promoter or terminator sequences. By integrating temporal transposon-sequencing data, we developed a novel method of essentiality assessment based on k-means unsupervised clustering, which provides dynamic and quantitative information on the fitness contribution of different genomic regions. We compared the insertion tolerance and persistence of the two engineered libraries, assessing the local impact of transcription and termination on cell fitness. Essentiality assessment at the local base-level revealed essential protein domains and small genomic regions that are either essential or inaccessible to transposon insertion. We also identified structural regions within essential genes that tolerate transposon disruptions, resulting in functionally split proteins. Overall, this study presents a nuanced view of gene essentiality, shifting from static and binary models to a more accurate perspective. Additionally, it provides valuable insights for genome engineering and enhances our understanding of the biology of genome-reduced cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0403d822930ef2b08e4a3eb27b77353feeed21e9" target='_blank'>
              Quantitative essentiality in a reduced genome: a functional, regulatory and structural fitness map
              </a>
            </td>
          <td>
            Samuel Miravet-Verde, Raul Burgos, Eva Garcia-Ramallo, Marc Weber, Luis Serrano
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genomic rearrangements play an important role in shaping genetic diversity, as they enable the generation of novel structural variations through specific genome manipulation tools. These variations contribute to phenotypic differences among individuals within a population, thereby serving as the foundation for natural selection and driving evolutionary processes. In recent years, Synthetic Chromosome Rearrangement and Modification by LoxP-mediated Evolution (SCRaMbLE) has emerged as a promising tool for studying genomic rearrangements. SCRaMbLE utilizes site-specific recombination mediated by loxPsym sites to induce targeted chromosomal rearrangements in yeast cells. In this review, we provide a comprehensive overview of recent advancements in optimization strategies of the SCRaMbLE system and discuss influential factors that affect its performance based on recent research findings. We demonstrate how the SCRaMbLE system can be employed for pathway engineering, phenotype improvement, genome minimization, and dissection of genotype-to-phenotype relationships. We highlight both the advantages and challenges associated with SCRaMbLE and envision its potential applications beyond yeast genetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0314619e822b8ab25ecabe0ad0985b3e4d4d9a40" target='_blank'>
              SCRaMbLE: a versatile tool for genome manipulation.
              </a>
            </td>
          <td>
            Li Cheng, Jielin Li, Yaojun Chen, Junbiao Dai
          </td>
          <td>2025-02-12</td>
          <td>Physiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58506d08da7a5b25a950b60c249aaba7b6397bbd" target='_blank'>
              Neutral drift upon threshold-like selection promotes variation in antibiotic resistance phenotype
              </a>
            </td>
          <td>
            A. N. Erdoğan, P. Dasmeh, Raymond D. Socha, Johnson Chen, Ben Life, Rachel Jun, Linda Kiritchkov, Dan Kehila, Adrian W. R. Serohijos, N. Tokuriki
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfaace030e74345b79142b5ae90516ba4841735" target='_blank'>
              High-throughput sequencing: a breakthrough in molecular diagnosis for precision medicine.
              </a>
            </td>
          <td>
            Dipali Dongare, Shaik Shireen Nishad, Sakshi Y Mastoli, S. Saraf, Nidhi Srivastava, Abhishek Dey
          </td>
          <td>2025-01-22</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The ribosome is one of the core macromolecules in the cell. The ribosomal RNAs (rRNA), which are essential components of the ribosome, are coded by the multi-copy ribosomal DNA (rDNA). Despite its highly conserved function, the rDNA displays substantial variation within all species analysed to date. This variation comprises both inter-individual differences in total copy number (CN) as well as inter- and intragenomic sequence variation in the form of single nucleotide variants (SNV) and insertions/deletions (INDELs) across rDNA copies. Whether germline variation of rDNA sequence associates with phenotypic traits in humans is, to date, unknown. Here, using the UK Biobank whole genome sequencing data, we first derive a high confidence list of rDNA-associated SNVs and INDELs that we validate in multiple ways. Using this list, we show that specific rDNA variants associate with several human traits. In particular, traits associated with body size appear enriched in variants within the Expansion Segment 15L region in the 28S rRNA. The strength of these associations does not diminish when accounting for the total rDNA CN of each individual. Our work represents the first large-scale association analysis of human traits with germline sequence variation in the rDNA, a source of human complex trait-relevant genetic variation that has thus far been largely ignored.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdee8f0782724daf5063853cc02bc21e85c5ca60" target='_blank'>
              Germline sequence variation within the ribosomal DNA is associated with human complex traits
              </a>
            </td>
          <td>
            Francisco Rodríguez-Algarra, Maia Cooper, F. Mardakheh, David M. Evans, V. Rakyan
          </td>
          <td>2025-02-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69cfefb50db5bb84559d86b5f736278ac09bb363" target='_blank'>
              LINC00626 drives tamoxifen resistance in breast cancer cells by interaction with UPF1
              </a>
            </td>
          <td>
            Hui Yuan, Lianbang Zhou, Wei Hu, Min Yang
          </td>
          <td>2025-01-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The study explores the role of KIF18A, a gene linked to whole-genome doubling (WGD) and chromosomal instability (CIN), in the metastatic progression of hepatocellular carcinoma (HCC). We found that KIF18A is a critical link between CIN and anoikis resistance, key factors in HCC metastasis. Through the bulk-seq analysis, we identified the pressures of anoikis and CIN faced by tumor cells during HCC metastasis and identified hub genes central to this process. Our results reveal that E2F activation during HCC progression leads to the transcription of KIF18A, promoting the survival and metastasis of CIN tumors. Overexpression of KIF18A leads to longer tumor life, more micronuclei, and increased survival and metastasis through non-canonical NF-kB activation. Deletion of KIF18A stabilizes the nuclear membrane of the micronucleus, silences cGAS-STING and PI3K-AKT pathways, and inhibits classical NF-kB. The study’s limitations include the need for further animal studies to validate the findings and explore the transient activation of the cGAS-STING pathway. Additionally, future research could focus on the potential therapeutic implications of targeting KIF18A in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae22aedbaa5a2bdc1f487c6a0f986379d3bb08c" target='_blank'>
              KIF18A in HCC Metastasis: Dual Role in Anoikis Resistance and Chromosome Instability
              </a>
            </td>
          <td>
            Changhao Kan, Weiyu Bai, Xinru Yang, Hayam Hamdy, Xianghui Yang, Xin Liu, Yan Lv, Junling Shen, Jianwei Sun
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epigenetics encompasses heritable and stable changes in gene expression caused by external chromosomal modifications, without altering the underlying DNA sequence. Epigenetic modifications, established during early development and maintained through successive cell divisions, play a critical role in regulating gene expression. Post-translational modifications (PTMs) are a key aspect of epigenetics and are essential for modulating protein functionality, as well as regulatory cellular processes, including proliferation, differentiation, metabolic pathways, and tumorigenic events. Among these, the small ubiquitin-related modifier (SUMOylation) system is a reversible PTM mechanism that alters target protein interaction surfaces through covalent binding to lysine residues, thereby influencing protein structure and function. Acute myeloid leukemia (AML) is a highly aggressive malignancy characterized by the clonal expansion of primitive hematopoietic stem cells of the myeloid lineage in the bone marrow. Despite recent advancements in therapeutic strategies and an improved understanding of leukemogenic pathways, patient outcomes remain poor, particularly in elderly populations. Consequently, efforts have focused on developing novel agents, including co-targeting specific mutations or integrating targeted therapies into combinatorial chemotherapeutic regimens. Emerging evidence suggests that SUMOylation plays a significant role in AML pathogenesis and treatment response, representing a promising therapeutic target for advanced disease cases. This review provides a brief analysis of the functional role of the SUMOylation system in AML and highlights its potential as a therapeutic target. We also discuss current knowledge gaps and propose directions for future research to advance precision medicine approaches for AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/734b8acfb1af3f3bde64169ef0104b00e64873e6" target='_blank'>
              Diagnostic and Therapeutic Implications of the SUMOylation Pathway in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Elena Chatzikalil, Konstantinos Arvanitakis, Filippos Filippatos, P. Diamantopoulos, T. Koufakis, Elena E Solomou
          </td>
          <td>2025-02-13</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>3</td>
          <td>1</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a heterogeneous and aggressive brain tumour that is invariably fatal despite maximal treatment. Genetic or transcriptomic ‘biomarkers’ could be used to stratify patients for treatments, however, pairing biomarkers with appropriate therapeutic ‘targets’ is challenging. Consequently, therapeutics have not yet been optimised for specific GBM patient subsets. Here we integrate genome-wide CRISPR/Cas9 knockout screening and genetic-annotation data for 60 distinct patient-derived, IDHwildtype, adult GBM cell lines, quantifying the essentiality of 15,145 genes. We describe a novel method using Targeted Learning, to estimate the effect size of GBM-relevant biomarkers on context-dependent gene essentiality (GBM-CoDE). We derive multiple target-biomarker pair hypotheses, which we release in an accessible platform to accelerate translation to biomarker-stratified clinical trials. Two of these (WWTR1 with EGFR mutation/amplification, and VRK1 with VRK2 expression suppression) have been validated in GBM, implying that our additional novel findings may be valid. Our method is readily translatable to other cancers of unmet need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1cf5308b3e1f097593cf8ac72787c4818c5257d" target='_blank'>
              Systematic identification of context-dependent gene-essentiality in Glioblastoma: The GBM-CoDE platform
              </a>
            </td>
          <td>
            Mitchell T Foster, Ailith Ewing, M. Frame, Paul M Brennan, A. Khamseh, S. Beentjes, Neil O. Carragher, Colin A. Semple
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="N6-methyladenosine is one of the most common and reversible post-transcriptional modifications in eukaryotes, and it is involved in alternative splicing and RNA transcription, degradation, and translation. It is well known that cancer cells acquire energy through metabolic reprogramming to exhibit various biological behaviors. Moreover, numerous studies have demonstrated that m6A induces cancer metabolic reprogramming by regulating the expression of core metabolic genes or by activating metabolic signaling pathways. Meanwhile, m6A modifications and related regulators are key targets in the regulation of immune effects. We further summarize how m6A modifications contribute to tumor metabolism, and how these events affect the tumor immune microenvironment, with a specific focus on different cell types. Finally, we focus on the specific applications of this field to tumor immunotherapy. We review the potential role of m6A in metabolic reprogramming of tumor immune microenvironment and its regulatory mechanism, with the aim of providing new targets for tumor metabolic regulation and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863477d0bd2f28e887ca2ce1de6b299ba12060bf" target='_blank'>
              N6-methyladenosine RNA methylation, a new hallmark of metabolic reprogramming in the immune microenvironment
              </a>
            </td>
          <td>
            Xiao-Yun Li, Lin Peng, Xue Yang, Jing Luo, Jianmei Wang, Kelin Mou, Huan Zhou, Yuhao Luo, Li Xiang
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Bladder cancer poses a significant threat to human health. In recent years, genetic circuit therapy has emerged as a novel alternative for precision tumor treatment, demonstrating promising potential for clinical application. Compared to traditional drugs, genetic circuits – typically carried by plasmids – offer advantages such as modularity, druggability, and shorter drug development cycles. These circuits can identify multiple molecular signals from tumors and integrate them through logic gates to specifically target tumor cells. Furthermore, by assembling effector modules, they can induce specific forms of cell death in tumor cells or alter malignant phenotypes, thereby reshaping the immune microenvironment to produce efficient, durable, and controllable antitumor effects. The urinary system serves as an ideal model for this new therapy due to its accessibility from the outside. Several genetic circuits have already been validated for the treatment of bladder cancer. This review outlined the effectiveness and potential value of genetic circuits in bladder cancer therapy. Objective: This review focused on the design principles of genetic circuits, their ability to recognize and convert signals, their therapeutic signal output, and the associated delivery vehicles. We also discussed the challenges and future prospects of genetic circuits as a novel form of “intelligent biotherapy.” Conclusion: The gene circuit can identify multiple signals, processing complex information, and generating multiple effects, thus providing a new approach for personalized treatment of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae601aa52d10ce092dba79e0fef244dcf8c04230" target='_blank'>
              Advancements in genetic circuits as part of intelligent biotherapy for the treatment of bladder cancer: A review
              </a>
            </td>
          <td>
            Chu Liu, Junlin Lu, Jiajian Lai, Kaiwen Jie, Tianxin Lin, Haifeng Wang, Xu Chen
          </td>
          <td>2025-02-04</td>
          <td>Bladder</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We lack tools to edit DNA sequences at scales necessary to study 99% of the human genome that is noncoding. To address this gap, we applied CRISPR prime editing to insert recombination handles into repetitive sequences, up to 1697 per cell line, which enables generating large-scale deletions, inversions, translocations, and circular DNA. Recombinase induction produced more than 100 stochastic megabase-sized rearrangements in each cell. We tracked these rearrangements over time to measure selection pressures, finding a preference for shorter variants that avoided essential genes. We characterized 29 clones with multiple rearrangements, finding an impact of deletions on expression of genes in the variant but not on nearby genes. This genome-scrambling strategy enables large deletions, sequence relocations, and the insertion of regulatory elements to explore genome dispensability and organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a90f87137d367e42fa5bf7ac26c12b1be111991" target='_blank'>
              Randomizing the human genome by engineering recombination between repeat elements.
              </a>
            </td>
          <td>
            Jonas Koeppel, Raphael Ferreira, T. Vanderstichele, Lisa M. Riedmayr, Elin Madli Peets, Gareth Girling, Juliane Weller, Pierre Murat, Fabio Giuseppe Liberante, Tom Ellis, George M. Church, Leopold Parts
          </td>
          <td>2025-01-31</td>
          <td>Science</td>
          <td>2</td>
          <td>8</td>
        </tr>

        <tr id="The biological process of aging is influenced by a complex interplay of genetic, environmental, and epigenetic factors. Recent advancements in the fields of epigenetics and senolytics offer promising avenues for understanding and addressing age-related diseases. Epigenetics refers to heritable changes in gene expression without altering the DNA sequence, with mechanisms like DNA methylation, histone modification, and non-coding RNA regulation playing critical roles in aging. Senolytics, a class of drugs targeting and eliminating senescent cells, address the accumulation of dysfunctional cells that contribute to tissue degradation and chronic inflammation through the senescence-associated secretory phenotype. This scoping review examines the intersection of epigenetic mechanisms and senolytic therapies in aging, focusing on their combined potential for therapeutic interventions. Senescent cells display distinct epigenetic signatures, such as DNA hypermethylation and histone modifications, which can be targeted to enhance senolytic efficacy. Epigenetic reprogramming strategies, such as induced pluripotent stem cells, may further complement senolytics by rejuvenating aged cells. Integrating epigenetic modulation with senolytic therapy offers a dual approach to improving healthspan and mitigating age-related pathologies. This narrative review underscores the need for continued research into the molecular mechanisms underlying these interactions and suggests future directions for therapeutic development, including clinical trials, biomarker discovery, and combination therapies that synergistically target aging processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99720a4dac2f698c148a925c719d60e5caa0c9f8" target='_blank'>
              The Intersection of Epigenetics and Senolytics in Mechanisms of Aging and Therapeutic Approaches
              </a>
            </td>
          <td>
            Daiana Burdusel, Thorsten R. Doeppner, R. Surugiu, D. Hermann, Denissa-Greta Olaru, Aurel Popa-Wagner
          </td>
          <td>2024-12-26</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Achieving targeted hypermutation of specific genomic sequences without affecting other regions remains a key challenge in continuous evolution. To address this, we evolved a T7 RNA polymerase (RNAP) mutant that synthesizes single-stranded DNA (ssDNA) instead of RNA in vivo, while still exclusively recognizing the T7 promoter. By increasing the error rate of the T7 RNAP mutant, it generates mutated ssDNA that recombines with homologous sequences in the genome, leading to targeted genomic hypermutation. This approach, termed T7 RNAP mutant-assisted continuous evolution (T7ACE), functions effectively in both typical prokaryotic and eukaryotic microorganisms (Escherichia coli and Saccharomyces cerevisiae), achieving targeted hypermutations at rates 2800- and 1200-fold higher than the genomic mutation rates, respectively. Using T7ACE, we successfully evolved an eight-fold increase in tigecycline resistance within 7 days and doubled the efficiency of a xylose utilization pathway in 10 days, demonstrating the efficiency and broad applicability of this single-component tool for continuous evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bff78f49477a4ea5070aa46ecc36f6c1f66021" target='_blank'>
              An evolved, orthogonal ssDNA generator for targeted hypermutation of multiple genomic loci
              </a>
            </td>
          <td>
            Weiran Chu, Rongzhen Tian, Yaxin Guo, Yaokang Wu, Fabian B H Rehm, Long Liu, Jianghua Li, Guocheng Du, Jian Chen, Yanfeng Liu
          </td>
          <td>2025-01-24</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Among the extensive genomic alterations in prostate cancer, the genomic deletion of PTEN stands out as one of the most consistently observed and confirmed alteration. PTEN loss in prostate tumors is primarily associated with cancer cell proliferation and survival through the activation of the PI3K-AKT-mTOR signalling pathway. However, its use as a robust biomarker in the clinical practice is hampered by the complex epigenetic, transcriptional and post-translational regulation. Here, we undertook an approach that combined in situ assessment of PTEN protein with transcriptional surrogates of its activity to gain insights into the downstream functional effects of PTEN loss in primary tumors. Our extensive bioinformatic analyses, including the integration with single-cell RNA-Seq approaches in a new clinical cohort, highlighted stroma remodeling as the major cancer cell-extrinsic process associated with PTEN loss. By applying similar computational strategy on the transcriptomic data generated from primary prostate tumors of genetically engineered Pten knock-out mouse models, we validated the causal role of Pten in the stromal reaction observed in clinical specimens. Mechanistically, we provide evidence for the activation of a paracrine program that encompasses enhanced TGF-β signalling and that is compatible with the secretome of PTEN-deficient senescent cancer cells. Our study provides relevant biological context to the cellular and molecular alterations unleashed upon PTEN protein loss.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2b98ad80d259dc7a130daab0e45ee7b8d7d990" target='_blank'>
              Transcriptional network analysis of PTEN protein-deficient prostate tumors reveals robust stromal reprogramming and signs of senescent paracrine communication
              </a>
            </td>
          <td>
            Ivana Rondon-Lorefice, Jose I. Lopez, Aitziber Ugalde-Olano, Maite Zufiaurre, I. Astobiza, Saioa Garcia-Longarte, Amaia Zabala, Sofía Rey, Aida Santos-Martín, Miguel Unda, Ana Loizaga-Iriarte, M. Graupera, Paolo Nuciforo, A. Carracedo, Isabel Mendizabal
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Plant mitochondrial and plastid genomes have exceptionally slow rates of sequence evolution, and recent work has identified an unusual member of the MutS gene family (“plant MSH1”) as being instrumental in preventing point mutations in these genomes. However, the eXects of disrupting MSH1-mediated DNA repair on “germline” mutation rates have not been quantified. Here, we used Arabidopsis thaliana mutation accumulation (MA) lines to measure mutation rates in msh1 mutants and matched wild type (WT) controls. We detected 124 single nucleotide variants (SNVs: 49 mitochondrial and 75 plastid) and 668 small insertions and deletions (indels: 258 mitochondrial and 410 plastid) in msh1 MA lines. In striking contrast, we did not find any organelle mutations in the WT MA lines, and reanalysis of data from a much larger WT MA experiment also failed to detect any variants. The observed number of SNVs in the msh1 MA lines corresponds to estimated mutation rates of 6.1ξ10-7 and 3.2 ξ10-6 per bp per generation in mitochondrial and plastid genomes, respectively. These rates exceed those of species known to have very high mitochondrial mutation rates (e.g., nematodes and fruit flies) by an order of magnitude or more and are on par with estimated rates in humans despite the generation times of A. thaliana being nearly 100-fold shorter. Therefore, disruption of a single plant-specific genetic factor in A. thaliana is suXicient to erase or even reverse the enormous diXerence in organelle mutation rates between plants and animals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d25e4691a96b49f18e1923e6d446e977ed7e35" target='_blank'>
              Flipping the switch on some of the slowest mutating genomes: Direct measurements of plant mitochondrial and plastid mutation rates in msh1 mutants
              </a>
            </td>
          <td>
            A. Broz, Mychaela M. Hodous, Yi Zou, Patricia C. Vail, Zhiqiang Wu, Daniel B. Sloan
          </td>
          <td>2025-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive malignancy that poses significant challenges due to high rates of relapse and resistance to treatment, particularly in older populations. While therapeutic advances have been made, survival outcomes remain suboptimal. The evolution of DNA and RNA sequencing technologies, including whole-genome sequencing (WGS), whole-exome sequencing (WES), and RNA sequencing (RNA-Seq), has significantly enhanced our understanding of AML at the molecular level. These technologies have led to the discovery of driver mutations and transcriptomic alterations critical for improving diagnosis, prognosis, and personalized therapy development. Furthermore, single-cell RNA sequencing (scRNA-Seq) has uncovered rare subpopulations of leukemia stem cells (LSCs) contributing to disease progression and relapse. However, widespread clinical integration of these tools remains limited by costs, data complexity, and ethical challenges. This review explores recent advancements in DNA/RNA sequencing in AML and highlights both the potential and limitations of these techniques in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e5d43f554c8fe98964b8130a7d1804ae561d5a" target='_blank'>
              Advances in DNA/RNA Sequencing and Their Applications in Acute Myeloid Leukemia (AML)
              </a>
            </td>
          <td>
            Fatimah Ahmed, Jiang Zhong
          </td>
          <td>2024-12-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We examine the complex relationship between genomic copy number variation (CNV) and gene expression, highlighting the relevance to cancer biology and other biological contexts. By tracing the history of genometranscriptome correlations, we emphasize the complexity and challenges in understanding these interactions, particularly within the heterogeneous landscape of human cancers. Recent advances in computational algorithms and high-throughput single-cell multi-omic sequencing technologies are discussed, demonstrating their potential to refine our understanding of cancer biology and their limitations. The integration of genomic and transcriptomic analyses, which offers novel insights into tumor evolution and heterogeneity as well as therapeutic strategies, is presented as a crucial approach for advancing cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fea1466a2396cc40442863f3364565e95fd6a58" target='_blank'>
              Exploring genome-transcriptome correlations in cancer.
              </a>
            </td>
          <td>
            M. Ronemus, Daniel Bradford, Zachary Laster, Siran Li
          </td>
          <td>2025-02-05</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, Graeme Lindsey, Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BRAF inhibitor-resistant melanomas (BRAFiR) acquire (epi)genetic and functional alterations that enable them to evade alternative treatments. Identifying these alterations is critical to advancing treatment strategies. Here, we explored the effect of Chk1 inhibition (Chk1i) on BRAFiR cells, revealing higher sensitivity compared to treatment-naïve cells both in vitro and in vivo. Using FUCCI-labeling and time-lapse microscopy, we show that S phase progression is required for Chk1i-induced cytotoxicity in BRAFiR cells, but not in treatment-naïve cells. Replication stress markers, including reduced BrdU incorporation and increased phospho-RPA and γH2AX, were exclusive to BRAFiR cells exposed to Chk1i. Untreated BRAFiR cells exhibited upregulated DNA replication genes, reduced progressing forks and increased origin firing, suggesting intrinsic replication changes. MAPK pathway reactivation in treatment-naïve cells mimicked BRAFiR traits, increasing sensitivity to Chk1i. These findings indicate that Chk1i exploits elevated replication stress specifically in BRAFiR, highlighting its therapeutic potential in overcoming MAPK inhibitor resistance in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5d8e2bacae8698be1cc41dc94d4d67c11c52a" target='_blank'>
              Resistance to BRAF inhibitors drives melanoma sensitivity to Chk1 inhibition
              </a>
            </td>
          <td>
            D. G. Carvalho, J. Kenski, Daniel A. Moreira, Matheus A. Rajão, O. Krijgsman, Carolina Furtado, M. Boroni, João P. B. Viola, D. Peeper, P. Possik
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Pancreatic cancer (PC), known as the "king of cancers," is characterized by an exceptionally low five-year survival rate, posing a formidable challenge to global public health. N6-methyladenosine (m6A) methylation is prevalent across various stages of eukaryotic RNA expression, including splicing, maturation, stability, translation, and localization, and represents a pivotal mechanism of epigenetic regulation. m6A methylation influences tumor initiation and progression by modulating post-transcriptional processes, playing a critical role in sustaining cancer cell stemness, promoting cell proliferation, and mediating drug resistance. Extensive research underscores the substantial contribution of m6A modifications to PC development. However, the multiplicity of m6A regulators and their intricate mechanisms of action complicate the landscape. This review aims to deepen the understanding of m6A's role in PC by delineating its involvement in four key areas of tumorigenesis: the hypoxic tumor microenvironment, metabolic reprogramming, immune microenvironment, and resistance mechanisms. Additionally, the review addresses the emerging frontier of m6A interactions with non-coding RNAs (ncRNAs), offering insights into the potential therapeutic and prognostic applications of m6A in the treatment and prognosis prediction of PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b08ca5467f445ec0f42c55b9835ad0ac4ba47" target='_blank'>
              The Mechanism and Latest Progress of m6A Methylation in the Progression of Pancreatic Cancer
              </a>
            </td>
          <td>
            Ze-Hao Liu, Peng Ma, Ying He, Yuejiao Zhang, Zuo Mou, Ting Fang, Wei Wang, Kaihuan Yu
          </td>
          <td>2025-01-13</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Based on the notion that hypomorphic germline genetic variants are linked to autoimmune diseases, we reasoned that novel targets for cancer immunotherapy might be identified through germline variants associated with greater T-cell infiltration into tumors. Here, we report that while investigating germline polymorphisms associated with a tumor immune gene signature, we identified PKCδ as a candidate. Genetic deletion of PKCδ in mice resulted in improved endogenous antitumor immunity and increased efficacy of anti-PD-L1. Single-cell RNA sequencing revealed myeloid cell expression of Prkcd, and PKCδ deletion caused a shift in macrophage gene expression from an M2-like to an M1-like phenotype. Conditional deletion of PKCδ in myeloid cells recapitulated improved tumor control that was augmented further with anti-PD-L1. Analysis of clinical samples confirmed an association between PRKCD variants and M1/M2 phenotype, as well as between a PKCδ KO-like gene signature and clinical benefit from anti-PD-1. Our results identify PKCδ as a candidate therapeutic target that modulates myeloid cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31ff12050b59b63f4c02e53380b59da37e0eb557" target='_blank'>
              PKCδ germline variants and genetic deletion in mice augment antitumor immunity through regulation of myeloid cells.
              </a>
            </td>
          <td>
            Kyle R Cron, A. Sivan, K. Aquino-Michaels, Andrea Ziblat, Emily F. Higgs, R.F. Sweis, Ruxandra Tonea, Seoho Lee, Thomas F Gajewski
          </td>
          <td>2025-01-14</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10−10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5′-TGT*−3′ motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential. IMPORTANCE Yarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica
              </a>
            </td>
          <td>
            Yuan-Ru Xiong, Yuan-Chun Fang, Min He, Keke Li, Lei Qi, Yang Sui, Ke Zhang, Xuechang Wu, Liang Meng, Ou Li, Dao-Qiong Zheng
          </td>
          <td>2024-12-23</td>
          <td>Applied and Environmental Microbiology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Alternative splicing (AS) is a mechanism that generates translational diversity within a genome. Equally important is the dynamic adaptability of the splicing machinery, which can give preference to one isoform over others encoded by a single gene. These isoform preferences change in response to the cell’s state and function. Particularly significant is the impact of physiological alternative splicing in T lymphocytes, where specific isoforms can enhance or reduce the cells’ reactivity to stimuli. This process makes splicing isoforms defining features of cell states, exemplified by CD45 splice isoforms, which characterize the transition from naïve to memory states. Two developments have accelerated the use of AS dynamics for therapeutic interventions: advancements in long-read RNA sequencing and progress in nucleic acid chemical modifications. Improved oligonucleotide stability has enabled their use in directing splicing to specific sites or modifying sequences to enhance or silence particular splicing events. This review highlights immune regulatory splicing patterns with potential significance for enhancing anticancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaaff8ba550e599aee86ffb2e27658c31cc7d97" target='_blank'>
              Alternative splicing of modulatory immune receptors in T lymphocytes: a newly identified and targetable mechanism for anticancer immunotherapy
              </a>
            </td>
          <td>
            Shay Tzaban, Ori Stern, Elad Zisman, G. Eisenberg, Shiri Klein, S. Frankenburg, Michal Lotem
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c49e06286441ee42660f276752afffa0be81c30" target='_blank'>
              Dysregulation of GAS5-miRNA-Mediated Signaling Pathways in Cancer Pathobiology: A Comprehensive Exploration of Pathways Influenced by this Axis.
              </a>
            </td>
          <td>
            Enwa Felix Oghenemaro, Abdulrahman Qais Khaleel, J. Rizaev, R. Roopashree, Muath Suliman, Syeda Wajida Kazmi, Ahmed Hjazi, Pranchal Rajput, Yasser Fakri Mustafa, M. Abosaoda
          </td>
          <td>2024-12-24</td>
          <td>Biochemical genetics</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Tetraploidy is a condition in which the entire set of chromosomes doubles, most often due to errors during cell division. Tetraploidy can lead to genomic instability and significant consequences, in particular metastasis and treatment failure in tumours, including radiotherapy. The development of new strategies to sensitise these cells to treatment is of great importance. In our study, we investigated the in vitro combination of chemical treatment with the kinase inhibitor SP600125 and irradiation on diploid versus metastatic tetraploid RKO colon cancer clones. We assessed mitochondrial transmembrane potential, cell cycle and subG1 population by flow cytometry and performed clonogenic assays to evaluate cell sensitivity. We found that the combination overcomes irradiation resistance in metastatic tetraploid clones. To identify the main pathway involved in cell sensitivity, we screened the Harvard Medical School KINOMEscan library and performed a gene ontology biological process analysis. We found that the major kinases inhibited by SP600125 were ANKK1, BIKE, IKKA, JNK1, MP2K3, MP2K4, MKNK2, MYLK, PLK4, RPS6KA4(Kin,Dom,1), MYLK4 and TTK, and the pathways involved in clone sensitivity were DNA damage repair, radiation resistance and apoptosis, through JNK pathway inhibition. Finally, our main finding was that combined treatment with SP600125 and radiotherapy reduced the resistance of metastatic tetraploid cells to treatment, essentially by inhibiting the JNK pathway. This result supports a promising anti-cancer strategy to overcome the resistance of tetraploid cancer cells to irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0380fc971abb59db901b3b88a1c597ae6b670f0" target='_blank'>
              JNK Inhibition Overcomes Resistance of Metastatic Tetraploid Cancer Cells to Irradiation-Induced Apoptosis
              </a>
            </td>
          <td>
            Mohamed Jemaà, N. Setti Boubaker, Nesrine Kerkeni, Stephan M. Huber
          </td>
          <td>2025-01-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125e72b0b5ffa707da90fa52c6a8e151fa428862" target='_blank'>
              TRACERx analysis identifies a role for FAT1 in regulating chromosomal instability and whole-genome doubling via Hippo signalling
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, Alastair Magness, K. Thol, Thomas A. Webber, Ming Jiang, R. Saunders, Yun-Hsin Liu, D. Biswas, Esther O. Ige, Birgit Aerne, E. Grönroos, Subramanian Venkatesan, Georgia Stavrou, Takahiro Karasaki, M. Al Bakir, M. Renshaw, Hang Xu, Deborah Schneider-Luftman, Natasha Sharma, Laura Tovini, M. Jamal-Hanjani, S. McClelland, K. Litchfield, Nicolai J. Birkbak, Michael Howell, N. Tapon, K. Fugger, N. Mcgranahan, J. Bartek, N. Kanu, C. Swanton
          </td>
          <td>2024-12-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e5ba64df098cc1a4c221cfe3104b836d138b7b9" target='_blank'>
              Genome-wide association studies are enriched for interacting genes
              </a>
            </td>
          <td>
            Peter T. Nguyen, Simon G. Coetzee, Irina Silacheva, Dennis J. Hazelett
          </td>
          <td>2025-01-15</td>
          <td>BioData Mining</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-08</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Significant progress in determining the molecular origins and resistance mechanisms of mantle cell lymphoma (MCL) has improved our understanding of the disease's clinical diversity. These factors greatly impact prognosis in MCL patients. Given the dynamic alterations in MCL clones and disease evolution, it is crucial to recognize high-risk prognostic factors at diagnosis and relapse. Clinical factors include a high MCL International Prognostic Index score with a high Ki-67 proliferation index; early disease progression within 24 months of first-line treatment; >3 prior lines of therapy at relapse; and an aggressive (blastoid or pleomorphic) histology. Molecular aberrations include dysregulated cyclin D1; an aberrant SOX11-CD70 axis; upregulated Musashi-2; MYC rearrangement; metabolic reprogramming; and epigenetic changes. Other factors contributing to high-risk MCL include an immune-depleted microenvironment and clone adaptability with complex chromosomal anomalies and somatic mutations in TP53, NSD2, CCND1, CDKN2A, BIRC3, SP140, KMT2D, NFkBIE, SMARCA4, and NOTCH2. Ultra-high-risk MCL is indicated by the coexistence of multiple high-risk prognostic factors in the relapse setting and can portend very short progression-free survival. As MCL treatments advance towards cellular therapies, resistance to anti-CD19 chimeric antigen receptor T-cell therapy is also observed. These findings necessitate revisiting the prognostic impact of high-risk factors, current management strategies, new bi- and tri-specific T-cell engagers, combination therapies, novel therapeutic targets, and next-generation clinical trials for high-risk MCL patients. This article provides a comprehensive update on recognizing and managing high-risk MCL, encompassing current practices and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03364deb8d8c1c44077292ebb1c2031c1c0a3c37" target='_blank'>
              High-Risk MCL: Recognition and Treatment.
              </a>
            </td>
          <td>
            P. Jain, Michael L Wang
          </td>
          <td>2024-12-30</td>
          <td>Blood</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gene duplication is the process of a gene copied via specific molecular mechanisms to form more duplicate genes. As an important approach to the origination of new genes, gene duplication contributes to around half of the genes in eukaryotic genomes, facilitating the adaptive evolution of species. Over the past fifty years, especially since entering the genomics era in the last two decades, there have been extensive and profound discussions on the mechanisms, evolutionary processes and forces behind the emergence of duplicate genes. Sequence similarity of duplicate genes often leads to functional redundancy, enhancing organismal robustness. Conversely, functional divergence can create novel functions and improve evolvability. In this review, we summarize the mechanism of gene duplication, the fate and the evolutionary models of duplicate genes. This article concludes by outlining how long-read sequencing technologies, gene editing, and various other high-throughput techniques will further advance our understanding of the role of duplicate genes in the genetics-development-evolution network.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11a8968d2fd67447569fbf8e6c7852ab9321d0f2" target='_blank'>
              Evolution by gene duplication: in the era of genomics.
              </a>
            </td>
          <td>
            Jie-Yu Shen, Tian-Han Su, Da-Qi Yu, , Yong-E Zhang
          </td>
          <td>2025-02-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Organisms regulate gene expression in response to environmental cues, a process known as plasticity, to adjust to changing environments. Research into natural variation and the evolution of plasticity frequently studies cis-regulatory elements with theory suggesting they are more important evolutionarily than trans-regulatory elements. Genome-wide association studies have supported this idea, observing a predominance of cis-loci affecting plasticity. However, studies in structured populations provide a contrasting image, raising questions about the genetic architecture of natural variation in plasticity. To circumvent potential statistical difficulties present in genome-wide association studies, we mapped loci underlying transcriptomic plasticity in response to salicylic acid using recombinant inbred lines generated from two random Arabidopsis thaliana accessions. We detected extensive transgressive segregation in the salicylic acid response, suggesting that plasticity to salicylate in Arabidopsis is polygenic. Most loci (>75%) underlying this variation act in trans, especially for loci influencing plasticity. Trans-acting loci were enriched in genome hotspots, with predominantly small effect sizes distributed across many genes. This could potentially explain their under-discovery in genome-wide association studies. This work reveals a potentially important role for trans-acting loci in plastic expression responses, with implications for understanding plant adaptation to different environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6242103d645ac263c51e56f8b61d6b0403cdeb3b" target='_blank'>
              Trans-regulatory loci shape natural variation of gene expression plasticity in Arabidopsis
              </a>
            </td>
          <td>
            Mariele Lensink, Grey Monroe, Dan Kliebenstein
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, M. Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, K. Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d173ca4b0414b7ce03e7c7ceb9b88bb092a887d" target='_blank'>
              Chemical inducers of proximity: precision tools for apoptosis in transcriptional regulation
              </a>
            </td>
          <td>
            Kaituo Huang
          </td>
          <td>2024-12-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Every protein progresses through a natural lifecycle from birth to maturation to death; this process is coordinated by the protein homeostasis system. Environmental or physiological conditions trigger pathways that maintain the homeostasis of the proteome. An open question is how these pathways are modulated to respond to the many stresses that an organism encounters during its lifetime. To address this question, we tested how the fitness landscape changes in response to environmental and genetic perturbations using directed and massively parallel transposon mutagenesis in Caulobacter crescentus. We developed a general computational pipeline for the analysis of gene-by-environment interactions in transposon mutagenesis experiments. This pipeline uses a combination of general linear models, statistical knockoffs, and a nonparametric Bayesian statistical model to identify essential genetic network components that are shared across environmental perturbations. This analysis allows us to quantify the similarity of proteotoxic environmental perturbations from the perspective of the fitness landscape. We find that essential genes vary more by genetic background than by environmental conditions, with limited overlap among mutant strains targeting different facets of the protein homeostasis system. We also identified 146 unique fitness determinants across different strains, with 19 genes common to at least two strains, showing varying resilience to proteotoxic stresses. Experiments exposing cells to a combination of genetic perturbations and dual environmental stressors show that perturbations that are quantitatively dissimilar from the perspective of the fitness landscape are likely to have a synergistic effect on the growth defect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d35fab11c10334ca080d2ea7ac3de9caeecc10" target='_blank'>
              Discovering genetic modulators of the protein homeostasis system through multilevel analysis
              </a>
            </td>
          <td>
            Vishal Sarsani, Berent Aldikacti, Tingting Zhao, Shai He, Peter Chien, Patrick Flaherty
          </td>
          <td>2024-12-23</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Purpose: Pancreatic cancer (PC) is an aggressive lethal tumor with an unmet need for novel therapeutic approaches. KRAS activating mutations occur in 90-95% of PC and contribute to tumor progression and resistance to therapy, including radiation. The mechanisms by which oncogenic KRAS promotes radiation resistance are critical to understand in order to identify novel therapies. Methods: We first analyzed the expression levels of DNA damage response and repair genes using Affymetrix RNA expression microarray in isogenic HCT116 and SW48 cells with KRAS wide-type and KRASG13D activating mutations to identify novel targets by which KRAS mutations may confer radiation resistance. We analyzed the expression of STN1 in pancreas normal and cancer tissues and assessed the correlation with PC clinical outcomes using TCGA dataset. Human tumor xenografts were generated to explore the role of STN1 on tumor growth in vivo. Radiation response was assessed through clonogenicity and gH2AX foci assays. Homologous recombination (HR) and non-homologous end joining (NHEJ) repair reporter assays, chromatin spreading assay, cell cycle analysis, mitotic catastrophe, Annexin-V assays were performed to investigate the mechanisms of radiation-induced cell death. Mass spectrometry analysis was performed to identify STN1 interacting proteins important in DNA damage response and further validated by immunoprecipitation and immunoblotting. Results: We find that KRAS activation increases STN1 expression to enhance DNA double strand break repair capacity in PC. STN1 is a component of the CST complex normally important for telomere duplication and maintenance. We find that STN1 is significantly upregulated in PC, especially in aggressive subtypes of PC, associates with KRAS oncogenic mutations, and correlates with poor patient clinical outcomes. Genetic silencing or pharmacologic inhibition of KRAS signaling decreases STN1 expression in PC cells, suggesting KRAS signaling positively regulates STN1 expression. Interestingly, STN1 depletion reduces tumor growth in a heterotopic model of KRAS mutant PC. Mechanistically, depletion of STN1 potentiates DNA damage, replication stress, and sensitizes PC cells to ionizing radiation independent of CTC1 and TEN1. In support of this finding, STN1 silencing reduces both HR and NHEJ repair of DSBs. Furthermore, knockdown of STN1 impairs cell cycle arrest at the G2/M phase in response to ionizing radiation, which is accompanied with increased mitotic catastrophe, radiation-induced apoptosis. Proteomic analysis reveals that STN1 physically interacts with many proteins important for DNA repair, replication and cell cycle progression, including ATM, DICER1, CEP164, and CEP250. Conclusion: Our findings have revealed a novel, potentially CST complex-independent role of STN1 in DSB repair after radiation. STN1 may function at one of the apical nodes in the DNA damage response pathway by interacting with ATM. Our findings suggest STN1 may be a promising target for improving genotoxic therapies in KRAS mutant cancers, including PC.
 Citation Format: Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, Sergio Corrales-Guerrero, Sindhu Nair, Joanna M. Karasinska, James T. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams.STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P011">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e95ed7e839e47ffa944834be9b3c39c5d903" target='_blank'>
              Abstract P011: STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer
              </a>
            </td>
          <td>
            Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, S. Corrales-Guerrero, Sindhu Nair, J. Karasinska, J. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Prostate cancer is the most common cancer among men worldwide, especially in those over 65, and is a leading cause of cancer-related mortality. The disease typically advances from an androgen-dependent state to castration-resistant prostate cancer (CRPC), which poses significant treatment challenges. The androgen receptor (AR) on the X chromosome is a central driver in this process, activating genes that govern proliferation and survival. Mutations and amplifications of the AR are closely associated with disease progression and treatment resistance. While traditional therapies such as androgen deprivation therapy (ADT) and AR antagonists like enzalutamide have been effective, resistance persists due to reactivation of AR signaling through mechanisms like ligand-independent activation. Recent research highlights the role of epigenetic modifications in enhancing AR activity and drug resistance. The tumor microenvironment, particularly interactions with cancer-associated fibroblasts (CAFs) and tumor-associated macrophages (TAMs), further complicates treatment by promoting aggressive tumor behavior and immune evasion. Future directions include developing next-generation AR antagonists, identifying AR-related biomarkers for personalized therapy, and exploring combinations with immune checkpoint inhibitors. Additionally, basal cell-lumen-derived organoids provide innovative models that can enhance understanding and treatment strategies in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cc795daea1ab497c79508ac8d30990e3cf3c4a" target='_blank'>
              Androgen receptor dynamics in prostate cancer: from disease progression to treatment resistance
              </a>
            </td>
          <td>
            Caihong Li, Dongkai Cheng, Pengya Li
          </td>
          <td>2025-02-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer poses a major global health issue, profoundly affecting both mortality and morbidity rates across the world. A key obstacle in understanding the pathogenesis of colorectal cancer lies in its high inter-patient and spatial heterogeneity, making standard treatments ineffective. Commonly, the study on colorectal cancer relies on bulk RNA sequencing, offering an average gene expression profile for a heterogenous cell population. However, this approach obscures the heterogeneous characteristics of the cancer cells and hinders a comprehensive understanding of the complex interplay among different cell populations. Recently, the advent of single-cell RNA sequencing has been revolutionary, enabling researchers to analyze individual immune cells and overcome the limitations of bulk RNA sequencing. Through single-cell RNA sequencing, researchers have gained insights into the dynamic nature of the immune response to cancer and potential targets for immunotherapy. In this review, we discuss the technical aspects of single-cell RNA sequencing, the application of single-cell RNA sequencing in cancer immunology, and the potential of single-cell RNA in clinical settings. We believe that harnessing singlecell RNA sequencing in cancer research holds immense potential to drive the development of personalized immunotherapies, aiming to improve patient outcomes in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac82763361480546b876aad89a46e0a8e233fa" target='_blank'>
              Single Cell RNA Sequencing in Colorectal Cancer Immunology: Recent Updates, Application, and Emerging Challenges.
              </a>
            </td>
          <td>
            Sabrina George, Nor Adzimah Johdi
          </td>
          <td>2025-02-12</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Retrotransposons, multi-copy sequences that propagate via copy-and-paste mechanisms involving an RNA intermediate, occupy large portions of all eukaryotic genomes. A great majority of their manifold copies remain silenced in somatic cells, nevertheless, some are transcribed, often in a tissue specific manner, and a small fraction retains its ability to mobilize. Retrotransposon expression or mobility are increasingly recognized to contribute to normal development and tissue homeostasis, as well as to aging and disease. While it is well characterized that retrotransposon sequences may provide cis regulatory elements for neighboring genes, how their own expression and mobility are achieved in different somatic contexts is not well understood. Here, using long-read DNA sequencing, we characterize somatic retrotransposition in the Drosophila intestine. We show that retroelement mobility does not change significantly upon aging and is limited to very few active sub-families of retrotransposons. Importantly, we identify a polymorphic donor locus of an endogenous LTR retroviral element rover, active in the intestinal tissue. We reveal that gut activity of the rover donor copy depends on its genomic environment. Without affecting local gene expression, the copy co-opts its upstream enhancer sequence, rich in transcription factor binding sites, for somatic expression. Further we show that escargot, a snail-type transcription factor critical for gut progenitor cell function, can drive transcriptional activity of the active rover copy. These data provide new insights into how locus-specific features allow active retrotransposons to produce functional transcripts and mobilize in a somatic lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e2218d35b486ab25b581d00df030a25e3f5364" target='_blank'>
              An endogenous retroviral element co-opts an upstream regulatory sequence to achieve somatic expression and mobility
              </a>
            </td>
          <td>
            Natalia Rubanova, Darshika Singh, Louis Barolle, Fabienne Chalvet, Sophie Netter, Mickael Poidevin, Nicolas Servant, Allison J. Bardin, Katarzyna Siudeja
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The understanding of genetic and epigenetic regulation of gene expression in endothelial and vascular smooth muscle cells remains fragmented with limited experimental validation. Methods Chromatin conformation (Micro-C), DNA methylation (RRBS), chromatin accessibility (ATAC-seq), and transcriptome profiles (RNA-seq) were mapped in human induced pluripotent stem cell (hiPSC)-derived, isogenic endothelial and vascular smooth muscle cells (iECs and iVSMCs). CTCF and RAD21 were depleted to assess the functional relevance of chromatin architecture, and genome editing was used to evaluate the allelic effect of a blood pressure-associated single nucleotide polymorphism (SNP). Results Significant correlations were identified between gene expression levels and chromatin interactions, chromatin accessibility, and DNA methylation in iECs and iVSMCs, with chromatin interactions showing the strongest association. Chromatin contact regions displayed distinct epigenetic landscapes depending on the types of regulatory element interactions involved. Perturbation of CTCF and RAD21 revealed their differential regulatory effects, particularly on the expression of genes overlapping chromatin contacts, with RAD21 exhibiting a broader regulatory impact. SNPs associated with several vascular traits were enriched in chromatin loops or accessible regions in iECs or iVSMCs. Precise genome editing demonstrated allele-dependent effects of SNP rs9833313 on the expression of SHOX2 located 247.4 kbp away but within the same chromatin loops as the SNP. Conclusion This study provides an extensive epigenetic landscape of vascular cells that may drive novel research on the role of genetic and epigenetic mechanisms of vascular function and disease as demonstrated by our targeted experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cca08a65eb2683798ce82f8cb05c4cd45763cf1" target='_blank'>
              Global and genetic regulation of gene expression in human endothelial and vascular smooth muscle cells
              </a>
            </td>
          <td>
            Pengyuan Liu, Atrayee Ray, Yong Liu, Manoj K. Mishra, Qiongzi Qiu, Rajan Pandey, Bhavika Therani, Jing Huang, Jiayi Ren, C. Stelloh, Xiaowen Bai, Andrew S Greene, Allen W Cowley, Aron M. Geurts, Sridhar Rao, Mingyu Liang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Increased stemness of cancer cells exacerbates radioresistance, thereby greatly limiting the efficacy of radiotherapy. In order to study the changes in cancer cell stemness during radiotherapy, we established a radioresistance model of human non-small cell lung cancer A549 cells and obtained A549 radioresistant cells (A549-RR). We sampled the cells at different time points during the modeling process and investigated the heterogeneity of each group of cells using single-cell sequencing. Cells in the early stages of fractionated irradiation were found to be significantly up-regulated in stemness, and a subpopulation of cells producing this response was screened and referred to as “radiation-induced stemness-responsive cancer cells”. They were undergoing stemness response, energy metabolism reprogramming, and progressively differentiating into cells with more diverse and malignant phenotypes in order to attenuate the killing effect of radiation. Furthermore, we demonstrated that such responses might be driven by the activation of the EGFR-Hippo signaling pathway axis, which also plays a crucial role in the development of radioresistance. Our study reveals the dynamic evolution of cell subpopulation in cancer cells during fractionated radiotherapy; the early stage of irradiation can determine the destiny of the radiation-induced stemness-responsive cancer cells. The activation of stemness-like phenotypes during the development of radioresistance is not the result of dose accumulation but occurs during the early stage of radiotherapy with relatively low-dose irradiation. The degree of the radiation-induced stemness response of cancer cells mediated by the EGFR-Hippo signaling pathway might be a potential predictor of the efficacy of radiotherapy and the development of radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6498ad18eccc13924e712513e028e4bc431ff4" target='_blank'>
              Single-Cell Sequencing Reveals the Role of Radiation-Induced Stemness-Responsive Cancer Cells in the Development of Radioresistance
              </a>
            </td>
          <td>
            Zheng Shi, Cuilan Hu, Jiadi Liu, Wei Cheng, Xiaohua Chen, Xiongxiong Liu, Yanyu Bao, Haidong Tian, Boyi Yu, Feifei Gao, Fei Ye, Xiaodong Jin, Chao Sun, Qiang Li
          </td>
          <td>2025-02-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) comprise a group of genetically heterogeneous hematological malignancies that result in the abnormal growth of leukemic cells and halt the maturation process of normal hematopoietic stem cells. Despite using molecular and cytogenetic risk classification to guide treatment decisions, most AML patients survive for less than five years. A deeper comprehension of the disease’s biology and the use of new, targeted therapy approaches could potentially increase cure rates. RAS oncogene mutations are common in AML patients, being observed in about 15–20% of AML cases. Despite extensive efforts to find targeted therapy for RAS-mutated AMLs, no effective and tolerable RAS inhibitor has received approval for use against AMLs. The frequency of RAS mutations increases in the context of AMLs’ chemoresistance; thus, novel anti-RAS strategies to overcome drug resistance and improve patients’ therapy responses and overall survival are the need of the hour. In this article, we aim to update the current knowledge on the role of RAS mutations and anti-RAS strategies in AML treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607005d4dc391e9d5c0e31b65105281e28950846" target='_blank'>
              Rat Sarcoma Virus Family Genes in Acute Myeloid Leukemia: Pathogenetic and Clinical Implications
              </a>
            </td>
          <td>
            Shaimaa Khattab, Adriatik Berisha, Natalia Baran, P. Piccaluga
          </td>
          <td>2025-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05abf56ac7febcd59ec911620962ed978222a49b" target='_blank'>
              Copy number amplification of FLAD1 promotes the progression of triple-negative breast cancer through lipid metabolism
              </a>
            </td>
          <td>
            Xiao-Qing Song, Tian-Jian Yu, Ou-Yang Yang, Jianlong Ding, Yizhou Jiang, Zhi-Ming Shao, Yi Xiao
          </td>
          <td>2025-02-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recurrent heterozygous missense variants in the highly conserved RNA-helicase DHX37, which is required for ribosome biogenesis, are a frequent cause of 46,XY sex-reversal or testis regression syndrome. How these missense variants specifically disrupt testis formation is unknown. Here, we demonstrate that mutant DHX37 proteins retain their ATPase activity and are not associated with stabilization of cellular β-catenin. Transfection of DHX37 p.R674Q mutant protein in an in-vitro cellular model recapitulating human Sertoli cell formation, showed a reduced activation of pro-testis genes compared to the WT protein. The expression of a DHX37 mutant protein in in-vitro derived human Sertoli-like cells (iSLCs) was also associated with global changes in gene expression, predicted to impact basic cellular functions. To define RNA transcripts interacting with either the WT or a mutant (p.R674Q) protein, we combined HyperTRIBE and single-cell full-length RNA-sequencing approaches using iSLCs. Gene ontology analysis indicated that transcripts targeted by WT DHX37 were primarily associated with cytoskeleton organization, including cell motility and cell adhesion. However, in contrast transcripts targeted by the mutated DHX37 protein, were not only associated with cytoskeleton organization but also with protein degradation and cell death. These data provide mechanistic framework that may explain how variants in the DHX37 protein can result in 46,XY sex-reversal through altered RNA networks that are required for the formation and maintenance of the supporting cell lineages of the human testis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc63e8ee6b25254fd0f501a68c12dac5001bf4c1" target='_blank'>
              Altered RNA-processing provides a mechanistic framework delineating human sex-reversal associated with pathogenic variants in the RNA-helicase DHX37
              </a>
            </td>
          <td>
            M. Elzaiat, Estelle Talouarn, Somboon Wankanit, Laurène Schlick, C. Eozenou, J. Bignon-Topalovic, Etienne Kornobis, P. Commere, Chloé Baum, Valérie Seffer, Ken McElreavey, A. Bashamboo
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Deletion of the long q arm of chromosome 22 (22qDEL) is the most frequently identified recurrent somatic copy number alteration observed in papillary thyroid carcinoma (PTC). Since its role in PTC is not fully understood, we conducted a pooled analysis of genomic characteristics and clinical correlates in 1094 primary tumors from four published PTC genomic studies. The majority of PTC cases with 22qDEL exhibited arm-level loss of heterozygosity (86%); nearly all PTC cases with 22qDEL had losses in 22q12 and 13, which together constitute 70% of the q arm. Our analysis confirmed that 22qDEL occurs more frequently with RAS point mutations (50.4%), particularly HRAS (70.3%), compared with other PTC drivers (9.3%), supporting the conclusion that 22qDEL is unlikely to be a solitary driver of PTC but possibly an important co-factor in carcinogenesis, particularly in PTCs with RAS driver mutations. Differential RNA expression analyses revealed downregulation of most genes located on chromosome 22 in cases with 22qDEL compared to those without 22qDEL. Many differentially expressed genes are drawn from immune response and regulation pathways. These findings highlight the value of further investigations into the contributions of 22qDEL events to PTC, perhaps mediated through immune perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0fd29e42888016677b1946cca20fdf0013b09" target='_blank'>
              Somatic copy number deletion of chromosome 22q in papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Olivia W Lee, D. Karyadi, Stephen W Hartley, Weyin Zhou, M. Machiela, Shahriar A. Zamani, L. Zurnadzhy, John N Weinstein, Young Joo Park, Jeong-Sun Seo, Gerry A Thomas, Tetiana I Bogdanova, M. Tronko, Lindsay M. Morton, S. Chanock
          </td>
          <td>2025-01-01</td>
          <td>European Thyroid Journal</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Type 2 diabetes mellitus (T2DM) is a chronic metabolic disorder characterized by hyperglycemia. Current therapeutic strategies primarily manage symptoms, leaving a substantial unmet need for curative interventions. This review explores the potential of CRISPR-dCas9-mediated CpG island editing as a promising therapeutic approach for T2DM. CpG islands, DNA regions enriched in cytosine-guanine dinucleotides, play a pivotal role in gene regulation. Their methylation status significantly influences gene expression. By targeting specific CpG islands within genes involved in glucose metabolism and insulin signaling, CRISPR-dCas9 can modulate gene expression and restore metabolic homeostasis. A particular focus is placed on the TXNIP gene, implicated in T2D pathogenesis. Reprogramming TXNIP expression using CRISPR-dCas9 offers potential therapeutic benefits, including protecting pancreatic beta cells, enhancing insulin sensitivity, and mitigating inflammation. While the potential of CRISPR-dCas9-mediated CpG island editing is clear and evident now, further steps are imperative to translate this approach into effective and safe therapies for T2DM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/383b7fe07ebbb6081b1bfffef778fbc335d6d9c3" target='_blank'>
              CRISPR-dCas9-mediated CpG island editing: A potential game-changer for diabetes treatment
              </a>
            </td>
          <td>
            K. K. Dutta
          </td>
          <td>2024-12-21</td>
          <td>Journal of Bioscience and Environment Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d5925d1e02fdeb24a29a2a8e41aed5142d614d" target='_blank'>
              Advances in A-to-I RNA editing in cancer
              </a>
            </td>
          <td>
            Yi Zhang, Lvyuan Li, Juana Jessica Mendoza, Dan Wang, Qijia Yan, Lei Shi, Zhaojian Gong, Zhaoyang Zeng, Pan Chen, Wei Xiong
          </td>
          <td>2024-12-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Spatial organization of the genome is fundamental for ensuring accurate gene expression. This process depends on the communication between gene promoters and distal cis-regulatory elements (CREs), which together make up 8% of the human genome and are supported by the chromatin structure. It is estimated that over 90% of disease-associated variants are located in the non-coding region of the genome and may affect CRE. For the cystic fibrosis transmembrane conductance regulator (CFTR) gene, a complete understanding of tissue-specific CFTR expression and regulation is missing, in particular in the pancreas. Mechanistic insights into tissue-specific expression may provide clarity on the clinical heterogeneity observed in Cystic Fibrosis and CFTR-related disorders. Methods To understand the role of 3D chromatin architecture in establishing tissue-specific expression of the CFTR gene, we mapped chromatin interactions via circular chromosome conformation capture (4C) and epigenomic regulation through H3K27ac and DNase Hypersensitive site I (DHS) in Capan-1 pancreatic cells. Candidate regulatory regions are validated by luciferase reporter assay and CRISPR-knock out. Results We identified active regulatory regions not only around the CFTR gene but also outside the topologically associating domain (TAD). By performing functional assays, we validated our targets and revealed a cooperative effect of the −44 kb, −35 kb, +15.6 kb and 37.7 kb regions, which share common predicted transcription factor (TF) motifs. Comparative 3D genomic analysis and functional assays using the Caco-2 intestinal cell line revealed the presence of tissue-specific CREs. Conclusion By studying the chromatin architecture of the CFTR locus in Capan-1 cells, we demonstrated the involvement of multiple CREs upstream and downstream of the CFTR gene. We also extend our analysis to compare intestinal and pancreatic cells and provide information on the tissue-specificity of CRE. These findings highlight the importance of expanding the search for causative variants beyond the gene coding sequence but also by considering the tissue-specific 3D genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9279897acf411b562291c89812e3d397609427" target='_blank'>
              New insights into the cis-regulation of the CFTR gene in pancreatic cells
              </a>
            </td>
          <td>
            Clara Blotas, A. Le Nabec, M. Collobert, Mattijs Bulcaen, Marianne S. Carlon, Claude Férec, S. Moisan
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Molecular targeted drugs are frequently used in precision cancer therapies. However, the prolonged administration of these agents can result in drug resistance. Cancer cells exhibiting defects in homologous recombination are particularly susceptible to genotoxic stress, such as that induced by PARP inhibitors (PARPi), resulting in synthetic lethality. Here, we show that sustained treatment of such cancer cells with PARPi selectively upregulates YAP1-2α, a minor isoform of YAP1. Elevated YAP1-2α heterodimerizes with TAZ to induce liquid-liquid phase separation (LLPS) in the nucleus. This LLPS-driven process generates nuclear condensates that activate a super-enhancer comprising YAP1-2α, TAZ, TEAD, and BRD4. The super-enhancer reconfigures transcriptional networks to enhance cancer stem-like malignant properties, thereby potentiating cell resilience and augmenting cell resistance to adverse conditions, including precision cancer therapies. Our study reveals the presence of a super-enhancer that promotes malignant tumor progression. Targeting the YAP1-2α/TAZ-induced super-enhancer is a promising strategy for overcoming resistance to anticancer modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0a7f11bb9d0a8943e8f9ed992501df776b3e17" target='_blank'>
              A super-enhancer actuated by heterodimerization of a specific YAP1 isoform, YAP1-2α, with TAZ endows cancer stemness and drug resistance
              </a>
            </td>
          <td>
            Takuya Ooki, Takeru Hayashi, Yui Yamamoto, Masanori Hatakeyama
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d077c0e04d5c20a662dd5e8ac2b16b1dbaf335" target='_blank'>
              Integrative pan-cancer genomic analysis highlights mitochondrial protein p32 as a potential therapeutic target in Myc-driven tumorigenesis.
              </a>
            </td>
          <td>
            Qiufen Bi, Jun Nie, Qiang Wu, Liang Sun, Shuang Zhu, Jin Bai, Yong Liu, Fang Huang, Keli Chai
          </td>
          <td>2025-02-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c10b0a74142199e8c7ea18f31540a06dfe3ffe" target='_blank'>
              Aneuploidy-driven gene expression profiling in human blastocysts: insights from RNA-Seq analysis.
              </a>
            </td>
          <td>
            M. Bazrgar, F. Khajehoseini, P. Eftekhari-Yazdi, M. Bakhtiarizadeh, H. Gourabi, Parishad Saei, R. Pazhoomand, S. Hosseinishenatal, R. Mohammadi
          </td>
          <td>2025-01-03</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24be616e06c75b57b6ca8151f14f1e70c2c9348f" target='_blank'>
              PIK3CA mutation fortifies molecular determinants for immune signaling in vascular cancers.
              </a>
            </td>
          <td>
            Donghee Lee, Emma C Kozurek, Md Abdullah, Ethan J Wong, Rong Li, Z. Liu, Hai Dang Nguyen, Erin Dickerson, Jong Hyuk Kim
          </td>
          <td>2024-12-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background/Objectives: The highly polymorphic Major Histocompatibility Complex (MHC) genomic region, located on the short arm of chromosome 6, is implicated genetically in Parkinson’s disease (PD), a progressive neurodegenerative disorder with motor and non-motor symptoms. Previously, we reported significant associations between SINE-VNTR-Alu (SVA) expression quantitative trait loci (eQTLs) and Human Leucocyte Antigen (HLA) class I genotypes in PD. In this study, we aimed to evaluate SVA associations and their regulatory effects on transposable element (TE) transcription in the MHC class I region. Methods: Transcriptome data from the peripheral blood cells of 1530 individuals in the Parkinson’s Progression Markers Initiative (PPMI) cohort were reanalyzed for TE and gene expression using publicly available bioinformatics tools, including Salmon and Matrix-eQTL. Results: Four structurally polymorphic SVAs regulated the transcription of 18 distinct clusters of 235 TE loci, comprising LINEs (33%), SINEs (19%), LTRs (35%), and ancient transposon DNA elements (12%) located near HLA genes. The transcribed TEs were predominantly short, with an average length of 445 nucleotides. The regulatory effects of these SVAs varied significantly in terms of TE types, numbers, and transcriptional activation or repression. The SVA-regulated TE RNAs in blood cells appear to function as enhancer-like elements, differentially influencing the expression of HLA class I genes, non-HLA genes, and noncoding RNAs. Conclusions: These findings highlight the roles of SVAs and their associated TEs in the complex regulatory networks governing coding and noncoding gene expression in the MHC class I region, with potential implications for immune function and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7af8d419a6f31034c910704dd0806ace1362d2" target='_blank'>
              The Transcription of Transposable Elements Differentially Regulated by SVAs in the Major Histocompatibility Complex Class I Region of a Parkinson’s Progression Markers Initiative Cohort
              </a>
            </td>
          <td>
            Jerzy K. Kulski, A. Pfaff, S. Kõks
          </td>
          <td>2025-01-06</td>
          <td>Journal of Molecular Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Abstract The molecular basis of adaptive evolution and cancer progression are both complex processes that share many striking similarities. The potential adaptive significance of environmentally-induced epigenetic changes is currently an area of great interest in both evolutionary and cancer biology. In the field of cancer biology intense effort has been focused on the contribution of stress-induced non-coding RNAs (ncRNAs) in the activation of epigenetic changes associated with elevated mutation rates and the acquisition of environmentally adaptive traits. Examples of this process are presented and combined with more recent findings demonstrating that stress-induced ncRNAs are transferable from somatic to germline cells leading to cross-generational inheritance of acquired adaptive traits. The fact that ncRNAs have been implicated in the transient adaptive response of various plants and animals to environmental stress is consistent with findings in cancer biology. Based on these collective observations, a general model as well as specific and testable hypotheses are proposed on how transient ncRNA-mediated adaptive responses may facilitate the transition to long-term biological adaptation in both cancer and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbfc2bb67d7c58db953ed76c627eb64e92c4eb6a" target='_blank'>
              Adaptive Significance of Non-coding RNAs: Insights from Cancer Biology
              </a>
            </td>
          <td>
            John F. McDonald
          </td>
          <td>2025-01-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5668e09864551d73b8ef36f697d610eddc6e8f2b" target='_blank'>
              MircoRNAs predict and modulate responses to chemotherapy in leukemic patients.
              </a>
            </td>
          <td>
            D. Ellakwa, Mohamed Ahmed Abdelmalek, Mostafa M Mostafa, Takwa E. Ellakwa, Al-Hassan Soliman Wadan
          </td>
          <td>2025-01-14</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="Copy number variations of the human CNTN6 gene, resulting from megabase-scale microdeletions or microduplications in the 3p26.3 region, are frequently implicated in neurodevelopmental disorders such as intellectual disability and developmental delay. However, duplication of the full-length human CNTN6 gene presents with variable penetrance, resulting in phenotypes that range from neurodevelopmental disorders to no visible pathologies, even within the same family. Previously, we obtained a set of induced pluripotent stem cell lines derived from a patient with a CNTN6 gene duplication and from two healthy donors. Our findings demonstrated that CNTN6 expression in neurons carrying the duplication was significantly reduced. Additionally, the expression from the CNTN6 duplicated allele was markedly lower compared to the wild-type allele. Here, we first introduce a system for correcting megabase-scale duplications in induced pluripotent stem cells and secondly analyze the impact of this correction on CNTN6 gene expression. We showed that the deletion of one copy of the CNTN6 duplication did not affect the expression levels of the remaining allele in the neuronal cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c9c747915f6b48c8d45b21bda1f171cd233c0f" target='_blank'>
              Targeted correction of megabase-scale CNTN6 duplication in induced pluripotent stem cells and impacts on gene expression
              </a>
            </td>
          <td>
            Maria Gridina, Polina Orlova, Oleg Serov
          </td>
          <td>2025-01-20</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7b4b6b0cd06f7bc05e01f01febfa31643b31b8" target='_blank'>
              Prevalent integration of genomic repetitive and regulatory elements and donor sequences at CRISPR-Cas9-induced breaks
              </a>
            </td>
          <td>
            C. Bi, Baolei Yuan, Yingzi Zhang, Mengge Wang, Yeteng Tian, Mo Li
          </td>
          <td>2025-01-20</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Mutations in KRAS are a dominant driver of pancreatic ductal adenocarcinoma (PDAC), with over 40% of PDAC patients presenting with KRASG12D mutations. The recent development of small molecule inhibitors targeting KRASG12D has enabled targeting of mutant KRAS signaling and suppression of PDAC; however, the contribution of the tumor microenvironment (TME) to the sustained therapeutic efficacy of KRASG12D inhibition and mechanism/s of resistance to KRASG12D suppression remain to be elucidated. Here, we employed spatial transcriptomics, single cell RNA sequencing, and CODEX-based spatial proteomics to evaluate cancer cell intrinsic and extrinsic responses to KRASG12D inhibition with MRTX1133. While KRASG12D inhibition initially increases CD11c+ cells with impactful T cell infiltration within proximity to cancer cells, long-term treatment with MRTX1133 resulted in reversal of the immune responses leading to KRASG12D therapy resistance promoted by CDK8, a multiprotein mediator complex associated kinase. CDK8 imparts resistance in part through induction of downstream CXCL2 chemokine secretion, inhibition of FAS expression, and remodeling of the TME to promote immune evasion. Targeting CDK8 by itself and in combination with αCTLA-4 immunotherapy overcomes resistance to KRASG12D inhibition with prolonged survival with translational implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31857c1855f90c586c4091c69153f057310e6e33" target='_blank'>
              CDK8 remodels the tumor microenvironment to resist the therapeutic efficacy of targeted KRASG12D inhibition in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Kathleen M. McAndrews, Krishnan K. Mahadevan, Bingrui Li, Amari M. Sockwell, Sami J. Morse, Patience J. Kelly, Sarah I. Patel, Michelle L. Kirtley, Barbara A. Moreno Diaz, Hengyu Lyu, Xunian Zhou, Hikaru Sugimoto, Lakshmi Kavitha Sthanam, Meagan R. Conner, Shreyasee V. Kumbhar, Kent A. Arian, Yasaman Barekatain, Francesca Paradiso, Paola A. Guerrero, V. Bernard, Navid Sobhani, Alondra N. Camacho-Acevedo, Kiara E. Bornes, Phuong Thao Tran, Anirban Maitra, T. Heffernan, Raghu Kalluri
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d042ce055ab273191af06fc94f1a5e070d7aee0d" target='_blank'>
              The emerging roles of aberrant alternative splicing in glioma
              </a>
            </td>
          <td>
            Reda Ben Mrid, Sara El Guendouzi, Marco Mineo, Rachid El Fatimy
          </td>
          <td>2025-02-06</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Bacterial genomic DNA inversions, which govern molecular phase-variations, provide the bacteria with functional plasticity and phenotypic diversity. These targeted rearrangements enable bacteria to respond to environmental challenges, such as bacteriophage predation, evading immune detection or gut colonization. This study investigated the short- and long-term effects of the lytic phage Barc2635 on the functional plasticity of Bacteroides fragilis, a gut commensal. Germ-free mice were colonized with B. fragilis and exposed to Barc2635 to identify genomic alterations driving phenotypic changes. Phage exposure triggered dynamic and prolonged bacterial responses, including significant shifts in phase-variable regions (PVRs), particularly in promoter orientations of polysaccharide biosynthesis loci. These shifts coincided with increased entropy in PVR inversion ratios, reflecting heightened genomic variability. In contrast, B. fragilis in control mice exhibited stable genomic configurations after gut adaptation. The phase-variable Type 1 restriction-modification system, which affects broad gene expression patterns, showed variability in both groups. However, phage-exposed bacteria displayed more restrained variability, suggesting phage-derived selection pressures. Our findings reveal that B. fragilis employs DNA inversions to adapt rapidly to phage exposure and colonization, leveraging genomic variability for resilience. This study emphasizes gut bacterial genomic and phenotypic plasticity upon exposure to the mammalian host and to bacteriophages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f81af1bcd2a68ada21239ed5d5a4363a88cc20" target='_blank'>
              Bacteriophage-driven DNA inversions shape bacterial functionality and long-term co-existence in Bacteroides fragilis
              </a>
            </td>
          <td>
            S. Carasso, Roni Keshet-David, Jia Zhang, Haitham Hajjo, Dana Kadosh-Kariti, T. Gefen, N. Geva-Zatorsky
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cytarabine, daunorubicin, and etoposide (ADE) have been the standard backbone of induction chemotherapy regimens for acute myeloid leukemia (AML) patients for over five decades. However, chemoresistance is still a major concern, and a significant proportion of AML becomes resistant to ADE treatment leading to relapse and poor survival. Therefore, there is a significant need to identify mechanisms mediating drug resistance to overcome chemoresistance. Herein, we performed genome-wide synthetic lethal CRISPR/Cas9 screens using araC, daunorubicin and etoposide. We further integrated significant findings from the CRISPR screen with outcome in pAML patients treated with standard ADE regimen on 3 independent clinical trials to identify drug response biomarkers of prognostic significance. We identified seveal mediators that would represent clinically and biologically significant genes for ADE treatment, such as BCL2, CLIP2, and VAV3, which are resistant genes with high expression associated with poor outcomes in pAML treated with ADE, and GRPEL1, HCFC1, and TAF10, which are sensitive genes with high expression showing beneficial outcomes. Notably, knockdowns of the BCL2, CLIP2, and VAV3 genes sensitize the ADE component in AML cell lines, suggesting that these genes should be further studied as potential therapeutic targets to overcome chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/979aaf7e330314833a544cf8bca5acd47b995421" target='_blank'>
              Genome-wide CRISPR screen identifies AraC-Dauno-Eto (ADE) response modulators that predicts outcome in pediatric AML.
              </a>
            </td>
          <td>
            Nam Nguyen, Roya Rafiee, P. K. Parcha, A. Tagmount, J. Rubnitz, Raul C Ribeiro, Xueyuan Cao, S. Pounds, Christopher Vulpe, J. Lamba
          </td>
          <td>2024-12-23</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The link between cancer maintenance and an ability to sustain continued growth through stresses conferred by the cancer state itself is growing. However, there are significant gaps in our understanding of how this stress is managed, particularly at the level of cancer initiating cells. Here, we identify proteins comprising the dynamic, stress-adaptive ribonucleoprotein complexes known as stress granules (SG) to be enriched among the factors essential for leukemic stem cell (LSC)-driven leukemic propagation. Focusing on core SG nucleator G3BP1, we dissect the role of SGs in human acute myeloid leukemia (AML), their targetability, and the mechanisms they govern to uncover a novel propensity for AML, and in particular LSC-enriched fractions, to prime the expression of SG components, form SGs with greater fidelity and to be reliant on their establishment and continued integrity for LSC maintenance. We further unveil the transcript and protein interactome of G3BP1 in the AML context and show that consolidated control of innate immune signaling, and apoptosis repression is executed through regional binding specificity of G3BP1 to highly structured 3’UTRs and cooperation with the RNA helicase UPF1 to mediate transcript decay in SGs. Altogether our findings advance novel fundamental principles of stress adaptation exploited in AML and LSCs that may extend to other cancers and uncover SGs as a novel axis for therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbccad0bc8af050d6702c70f684ec11d3f4f7bb0" target='_blank'>
              Stress Granules Underlie Acute Myeloid Leukemia Stem Cell Survival and Stress Adaptation
              </a>
            </td>
          <td>
            Amanda Tajik, Emily Tsao, Soheil Jahangiri, Brendon Seale, Brian A. Yee, Jack T. Naritomi, Zaldy Balde, S. Cathelin, Ava Keyvani Chahi, Lance Li, He Tian Tony Chen, Nicholas Wong, Lina Liu, Pratik Joshi, Steven Moreira, Curtis W. McCloskey, Shahbaz Khan, Katherine L. Rothamel, Helena Boutzen, Suraj Bansal, Andy G. X. Zeng, Stefan Aigner, Yu Lu, J. Dick, T. Kislinger, Rama Khokha, Mark D. Minden, A. Gingras, Gene W. Yeo, Kristin J Hope
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Precision medicine approaches to cancer treatment aim to exploit genomic alterations that are specific to individual patients to tailor therapeutic strategies. Yet, some targetable genes and pathways are essential for tumor cell viability even in the absence of direct genomic alterations. In underrepresented populations, the mutational landscape and determinants of response to existing therapies are poorly characterized because of limited inclusion in clinical trials and studies. One way to reveal tumor essential genes is with genetic screens. Most screens are conducted on cell lines that bear little resemblance to patient tumors, after years of culture under nonphysiologic conditions. To address this problem, we aimed to develop a CRISPR screening pipeline in three-dimensionally grown patient-derived tumor organoid (PDTO) models. A breast cancer PDTO biobank that focused on underrepresented populations, including West African patients, was established and used to conduct a negative-selection kinome-focused CRISPR screen to identify kinases essential for organoid growth and potential targets for combination therapy with EGFR or MEK inhibitors. The screen identified several previously unidentified kinase targets, and the combination of FGFR1 and EGFR inhibitors synergized to block organoid proliferation. Together, these data demonstrate the feasibility of CRISPR-based genetic screens in patient-derived tumor models, including PDTOs from underrepresented patients with cancer, and identify targets for cancer therapy. Significance: Generation of a breast cancer patient-derived tumor organoid biobank focused on underrepresented populations enabled kinome-focused CRISPR screening that identified essential kinases and potential targets for combination therapy with EGFR or MEK inhibitors. See related commentary by Trembath and Spanheimer, p. 407.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde16e03b04b6ab8e7612d1796bad37a03db0723" target='_blank'>
              Kinome-Focused CRISPR-Cas9 Screens in African Ancestry Patient-Derived Breast Cancer Organoids Identify Essential Kinases and Synergy of EGFR and FGFR1 Inhibition.
              </a>
            </td>
          <td>
            Florencia P. Madorsky Rowdo, Rachel Martini, Sarah Ackermann, Colin P Tang, Marvel Tranquille, Adriana Irizarry, Ilkay Us, Omar Alawa, Jenna E Moyer, M. Sigouros, John Nguyen, Majd Al Assaad, Esther Cheng, Paula S Ginter, J. Manohar, Brian Stonaker, Richard Boateng, J. Oppong, E. Adjei, B. Awuah, I. Kyei, F. Aitpillah, M. Adinku, K. Ankomah, E. Osei-Bonsu, K. Gyan, Syed A Hoda, Lisa A Newman, Juan Miguel Mosquera, A. Sboner, O. Elemento, Lukas E. Dow, Melissa B. Davis, M. L. Martin
          </td>
          <td>2025-02-01</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>67</td>
        </tr>

        <tr id="ABSTRACT Background Acute lymphocytic leukemia (ALL), characterized by uncontrolled growth of abnormal lymphocytes, predominantly affects children. Genetic analysis focusing on genes and microRNAs reveals important information about the biology of ALL, enabling accurate diagnosis and treatment. This study examines gene and microRNA expression in B cell ALL to improve early diagnosis and personalized treatment. Methods Bone marrow samples were collected from patients both before and after the induction phase of chemotherapy. Comprehensive diagnostic techniques including flow cytometry, molecular assays, real‐time PCR for common translocations, karyotyping, and complete blood count (CBC) analysis were employed. These methods were utilized to determine the type and risk assessment of ALL, identify specific gene and microRNA expressions, and measure blood cell counts. Results The study comprised 12 patients, all under the age of 18. Post‐treatment RT‐PCR analysis revealed significant reductions in the expression of the ABCB1 gene, miR‐129‐5p, and miR‐9‐5p following the induction phase of chemotherapy. Karyotype analysis indicated that two patients were hypodiploid; unfortunately, both of these patients did not survive. Conclusion MicroRNAs and ABC genes serve as predictive and prognostic biomarkers in Acute Lymphoblastic Leukemia (ALL) and should be carefully reconsidered. It is more accurate to state that while microRNAs and ABC genes may potentially influence treatment response in ALL, further research is crucial to fully understand their roles in determining treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d6c6bf058c1b51d999c0b567643ee6cccfac4f" target='_blank'>
              ATP‐Binding Cassette Transporter Genes and microRNAs in Pediatric B‐Cell ALL: Expression Insights
              </a>
            </td>
          <td>
            Reza Nekouian, Pooya Faranoush, Fatemeh Khesali, Parisa Shams, M. Foroughi‐Gilvaee, Negin Sadighnia, Dorsa Fallah Azad, MohammadAli Ehsani, Mohammad Faranoush
          </td>
          <td>2024-12-23</td>
          <td>Journal of Clinical Laboratory Analysis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93bae82e2a8f4a414d538a8210faf20fea3eff9d" target='_blank'>
              Molecular and pharmacological heterogeneity of ETV6::RUNX1 acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Zhenhua Li, Huanbin Zhao, Wenjian Yang, Maud Maillard, S. Yoshimura, Yu-Chih Hsiao, Xingyu Huang, Yoshihiro Gocho, L. Rowland, Anthony Brown, Landon Choi, K R Crews, C. Mullighan, Samuel Brady, Cheng Cheng, Ti-Cheng Chang, Gang Wu, M. Loh, A. E. J. Yeoh, Federico Antillon-Klussmann, S. Jeha, H. Inaba, Jiyang Yu, Ching-Hon Pui, Seth E. Karol, William E Evans, Jun J Yang
          </td>
          <td>2025-01-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="Currently, mankind is fiercely struggling with cancer. Recently, we have been winning the battle against cancer through precision medicine and accompanying diagnostic methods, and we are raising many hopes with blockbuster drugs. It would be even better if we could read the cancer nucleotide sequence, identify them in advance, and suggest treatments simultaneously. However, this may be an impossible dream because it takes a lot of time and effort to diagnose and ensure all the long gene sequences of cancer at once. Thus, victory will be even closer if a rapid and accurate diagnosis of the cancer-specific gene biomarkers that will soon be imprinted can be made. With the advent of nanotechnology, a new short cancer diagnostic toolkit has been proposed to achieve the goal. This review presents a small diagnostic device that detects genetic fragments (simply 'Gizmo') of certain cancers. The development of numerous diagnostic methods has focused on (1) directly detecting pre-selectively targeted genes using novel diagnostic systems, and (2) indirectly detecting substantial improvements in diagnostic sensitivity only through detection signal amplification without existing gene amplification steps. Our fight against cancer is not a dream, but the result of success, and it is assumed that victory will accelerate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eccae0a3a741529fc24b2b02ce5646b06a22df12" target='_blank'>
              Engineering Gizmos for Short Cancer Genetic Fragments Discrimination.
              </a>
            </td>
          <td>
            Jisoo Yuk, Jeonghun Kim, Sunghwan Jung, Soong Ho John Um
          </td>
          <td>2025-02-05</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT The discovery at the end of the 20th century of genes that induce cell death revolutionised the biocontaintment of genetically manipulated bacteria for environmental or agricultural applications. These bacterial ‘killer’ genes were then assayed for their potential to target and control malignant cells in human cancers. The identification of the bacteriomes in different human organs and tissues, coupled with the observation that bacteria tend to accumulate near tumours, has opened new avenues for anti‐cancer strategies. This progress, along with recent insights into how cancer cells evade immune response, has prompted innovative therapeutic approaches. Tumour microenvironments are typically nutrient‐rich, characterised by low oxygen tensions and very resistant to immune responses. Two recent studies in MBT highlight the promise of using Salmonella typhimurium and Escherichia coli as vectors in novel cancer treatments. Engineered S. typhimurium strains can generate adjuvant flagellin‐antigen complexes that function as in situ vaccines, hence increasing the immunogenic responses within tumour environment. Similarly, gut E. coli can be used as vectors to targert tumour cells in colon cancer, enabling both diagnostic applications and localised treatments. Both approaches hold significant potential to improve patient survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bba9ffcefc81fe9e0a146a68f1324d903a6bef1" target='_blank'>
              Bacteria as Precision Tools for Cancer Therapy
              </a>
            </td>
          <td>
            C. Michán, José Prados, Juan-Luis Ramos
          </td>
          <td>2025-01-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background/Objectives: Treatment with tyrosine kinase inhibitors (TKIs) in chronic myeloid leukemia (CML) has revolutionized disease management and has transformed CML from a life-threatening disease to a chronic condition for many patients. However, overcoming resistance, particularly related to leukemic stem cells (LSC) that can persist even when the bulk of the leukemic cells are eliminated, remains a significant challenge. Methods: K562 and KU812 cell lines were treated in vitro with the TKI Imatinib (IM). Gene expression, protein analysis, and metabolomic screening were conducted to investigate the ability of the drug to enhance stem cell (SC) features. Moreover, a gene ontology analysis was performed on different available datasets, to further consolidate our data. Results: 48 h of IM treatment can significantly increase the expression of genes related to SC self-renewal, particularly SOX2 and OCT 3/4. Interestingly, these modulations occur in cells that remain alive after drug treatment and that displayed features consistent with leukemia stem-like CML cells, suggesting that SC genes levels are crucial even in cell population survived upon TKI treatment. Moreover, after in silico analysis of available data, we observed an enrichment of SOX2/NANOG and OCT 3/4 signatures after TKI treatment, thus strengthening our results. Conclusions: Our results confirmed the relevance of LSC features after TKI treatment, highlighting the need for more effective and potentially curative strategies targeting LSCs to overcome resistance in CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28fb6d29d24fd74f86c83c9e97c902987037fdab" target='_blank'>
              Tyrosine Kinase Inhibitor Therapy Enhances Stem Cells Profile and May Contribute to Survival of Chronic Myeloid Leukemiastem Cells
              </a>
            </td>
          <td>
            Simone Rocco, A. Maglione, Valentina Schiavo, Alessandro Ferrando, Carmen Fava, D. Cilloni, Barbara Pergolizzi, C. Panuzzo
          </td>
          <td>2025-01-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74606bb02b8c93f71db20ba35bf801348005a77" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Yu-Jia Shiah, Takafumi N. Yamaguchi, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, A. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Currently, numerous associations between genetic polymorphisms and various diseases have been characterized through Genome-Wide Association Studies. The majority of clinically significant polymorphisms are localized in non-coding regions of the genome. While modern bioinformatic resources make it possible to predict molecular mechanisms that explain the influence of non-coding polymorphisms on gene expression, such hypotheses require experimental verification. This review discusses the methods for elucidating the molecular mechanisms underlying the dependence of disease pathogenesis on specific genetic variants within non-coding sequences. A particular focus is on the methods to identify the transcription factors with binding efficiency contingent upon polymorphic variations. Despite remarkable progress in the bioinformatic resources enabling the prediction of the impact of polymorphisms on disease pathogenesis, the imperative for experimental approaches to this inquiry still persists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0e300f57318f3e11948f7c4bd85a221591ff36c" target='_blank'>
              Methods for Functional Characterization of Genetic Polymorphisms of Non-Coding Regulatory Regions of the Human Genome (Review)
              </a>
            </td>
          <td>
            A. Uvarova, E. A. Tkachenko, E. M. Stasevich, E. A. Zheremyan, K. Korneev, D. V. Kuprash
          </td>
          <td>2024-12-25</td>
          <td>Biohimiâ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d1fd954b718bc92e2f3d26e9d0a2654e108d08" target='_blank'>
              Prostate cancer epigenetics - from pathophysiology to clinical application.
              </a>
            </td>
          <td>
            V. Constâncio, J. Lobo, J. P. Sequeira, Rui Henrique, Carmen Jerónimo
          </td>
          <td>2025-01-16</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Monosomy 7 and deletion 7q are common chromosomal abnormalities in myeloid malignancies, and they are associated with a poor prognosis. The mechanism underlying their acquisition remains elusive. We identified a cohort of 24 patients exhibiting clones with different chromosome 7 abnormalities, such as deletion 7q, unstable derivatives (ring chromosomes or ‘naked’ centromeres), and monosomy 7. We designated this group as having cytogenetic clonal evolution of chromosome 7 abnormalities (CCE7). In some cases, CCE7 correlated with disease progression, suggesting that deletions or other derivatives involving the q-arm of chromosome 7 may arise early in the disease course. These abnormalities may be transient but can potentially evolve into monosomy 7. Within the CCE7 group, telomere loss or shortening may contribute to chromosomal instability and the emergence of unstable derivatives, as the chromosome 7 derivatives displayed loss or rearrangement of subtelomeric regions. Moreover, we identified variants in genes implicated in telomere biology disorders and observed specific genetic mutation profiles associated with different chromosome 7 abnormalities. These findings shed light on a potential mechanism leading to monosomy 7 through the evolution of chromosome 7q abnormalities. Identifying patients at risk of developing monosomy 7, based on the presence of unstable derivatives with telomere loss or a specific mutation profile, could potentially enhance patient management and guide the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd97cfdef751d5eb8d87f19388c6ddb34eef62a2" target='_blank'>
              Emergence and Cytogenetic Clonal Evolution of Chromosome 7 Abnormalities in Myeloid Malignancies: Investigating the Role of Telomere Dysfunction
              </a>
            </td>
          <td>
            Carmen Baldazzi, L. Bandini, V. Robustelli, A. Patuelli, C. Venturi, Alessandra Grassi, G. Marzocchi, Angela Ielpo, V. Solli, M. Bochicchio, Stefania Paolini, Chiara Sartor, F. Zingarelli, Antonio Curti, E. Ottaviani, N. Testoni
          </td>
          <td>2025-01-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1622b2dfde4aac03266cbf9b4a4aa33c5ec0f83" target='_blank'>
              A TBX2-driven signaling switch from androgen receptor to glucocorticoid receptor confers therapeutic resistance in prostate cancer.
              </a>
            </td>
          <td>
            Sayanika Dutta, Hamed Khedmatgozar, G. Patel, Daniel Latour, Jonathan Welsh, Mainak Mustafi, Antonina Mitrofanova, Manisha Tripathi, S. Nandana
          </td>
          <td>2024-12-20</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Despite rapid advances in mapping genetic drivers and gene expression changes in hematopoietic stem cells (HSCs), there is a relative paucity of studies at the protein level. Here, we perform a deep, multi-omic characterization (epigenome, transcriptome and proteome) of HSCs carrying a loss-of-function mutation in Tet2, a key driver of increased self-renewal in blood cancers. Using state-of-the-art, multiplexed, low-input mass spectrometry (MS)-based proteomics, we profile wildtype (WT) and TET2-deficient (Tet2-/-) HSCs and show that the proteome captures previously unrecognized molecular processes which define the pre-leukemic HSC molecular landscape. Specifically, we obtain more accurate stratification of WT and Tet2-/- HSCs than transcriptomic approaches and identify extracellular matrix (ECM) molecules as novel points of dysregulation upon TET2 loss. HSC expansion assays using ECM-functionalized hydrogels confirm a selective effect on the expansion of Tet2-mutant HSCs. Taken together, our study represents a comprehensive molecular characterization of Tet2-mutant HSCs and identifies a previously unanticipated role of ECM molecules in regulating self-renewal of disease-driving HSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee30ff0c55cc7680e35335a72a060f31e9944948" target='_blank'>
              Selection of pre-leukemic hematopoietic stem cells driven by distinct extracellular matrix molecules
              </a>
            </td>
          <td>
            M. Jassinskaja, D. Bode, Monika Gonka, Theodoros I. Roumeliotis, Alexander J Hogg, Juan A Rubio Lara, Ellie Bennett, Joanna Milek, Bart Theeuwes, M. S. Vijayabaskar, Lilia Cabrera Cosme, James L C Che, Sandy MacDonald, Sophia Ahmed, Benjamin A. Hall, Grace Vasey, Helena Kooi, M. Belmonte, Mairi S. Shepherd, W. Brackenbury, I. Kucinski, Satoshi Yamazaki, Andrew N. Holding, Alyssa H Cull, Nicola K Wilson, B. Göttgens, J. Choudhary, David G. Kent
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Significance Understanding the multifaceted genetic disorder ataxia–telangiectasia (A-T) offers new insights into the functions of the ATM protein kinase, the loss of which underlies this disease. A-T exemplifies the connection between genome instability, cellular senescence, and aging. This study elucidates the pathways leading to a hallmark of A-T’s cellular phenotype: the premature senescence of primary fibroblasts derived from affected individuals. Our findings provide a mechanistic understanding of how inherited genome instability accelerates cellular senescence, offering potential explanations for several A-T symptoms. The work introduces murine primary lung fibroblasts growing under 3% oxygen as a valuable experimental system to explore the genome instability-senescence link and the molecular events leading to the neoplastic transformation of primary diploid cells without any directed manipulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d92deb9c47fa1db095e272225b8766e094ed2771" target='_blank'>
              The cGAS–STING, p38 MAPK, and p53 pathways link genome instability to accelerated cellular senescence in ATM-deficient murine lung fibroblasts
              </a>
            </td>
          <td>
            M. Haj, Yann Frey, Amit Levon, Avishai Maliah, Tal Ben-Yishay, Rachel Slutsky, Riham Smoom, Y. Tzfati, Uri Ben-David, Carmit Levy, Ran Elkon, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M. Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/265becd303d54899b33b805c287a30e2d9e20265" target='_blank'>
              Repeat expansion in a fragile X model is independent of double strand break repair mediated by Pol θ, RAD52, RAD54 or RAD54B
              </a>
            </td>
          <td>
            Bruce E Hayward, Geum-Yi Kim, Carson J Miller, Cai McCann, Megan G. Lowery, Richard Wood, K. Usdin
          </td>
          <td>2025-02-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 genome-editing technology has revolutionized human stem cell (HSC) research, offering novel therapeutic and diagnostic applications. HSCs play a crucial role in regenerative medicine and genetic therapies due to their ability to self-renew and differentiate into various blood cell lineages. The precise genome-editing capability of CRISPR/Cas9 allows for targeted gene modifications, enabling the correction of inherited disorders, disease modeling and the discovery of novel biomarkers. Despite significant advancements, challenges such as off-target effects, delivery efficiency, and ethical concerns persist, requiring further research and optimization.
Objective: This systematic review evaluates the therapeutic and diagnostic potential of CRISPR/Cas9 genome editing in HSCs, focusing on its efficacy in gene correction for hematologic disorders, disease modeling and biomarker discovery.
Methods: A systematic review was conducted following PRISMA guidelines, analyzing studies published between 2015 and 2024. Literature searches were performed in PubMed, Web of Science, and Scopus using MeSH-aligned keywords. The inclusion criteria encompassed peer-reviewed studies utilizing CRISPR/Cas9 for gene modification in HSCs for therapeutic and diagnostic applications. Exclusion criteria included studies that lacked experimental validation, did not focus on HSCs, or were non-English publications. Out of 85 initially retrieved studies, 40 met the inclusion criteria, and 15 were selected for final synthesis.
Results: CRISPR/Cas9 gene-editing strategies in HSCs were categorized as gene knockout (53%), gene activation (40%), and dual knockout/activation (7%). Hematological disorders, including sickle cell anemia and beta-thalassemia, accounted for 35% of studies, demonstrating up to 90% correction in β-globin mutations. Neurodegenerative diseases constituted 20% of studies, where knockout of amyloid precursor protein (APP) in Alzheimer’s models resulted in a 60% reduction in plaque accumulation. Muscular dystrophy studies (10%) showed 75% improvement in dystrophin expression through gene activation. High-throughput CRISPR screening was employed in 15% of studies for biomarker identification. Despite promising outcomes, off-target mutations were observed in 28% of studies, and viral vector-based delivery methods were used in 65%, raising safety concerns.
Conclusion: CRISPR/Cas9 genome editing in HSCs presents a ground-breaking approach for treating genetic disorders and enhancing precision medicine. Its potential to correct disease-causing mutations, model complex disorders, and identify novel therapeutic targets is substantial. However, challenges in delivery methods, long-term safety, and ethical considerations remain barriers to clinical translation. Future research should focus on optimizing high-fidelity Cas9 variants, improving non-viral delivery methods, and addressing ethical concerns to ensure the safe and effective application of CRISPR/Cas9 in regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d20400a815c119b74c31ff8ba0f49e856aea421b" target='_blank'>
              EVALUATION OF CRISPR/CAS9 GENOME-EDITING SYSTEM IN HUMAN STEM CELLS HSCS: THERAPEUTICS AND DIAGNOSTICS PROSPECTS
              </a>
            </td>
          <td>
            Ehsan ul Haq, Haseeb Khaliq, Ayesha Muddasser, Misha Aslam, Farwa Zafar, Naveera Mazhar, Amna Naheed Khan
          </td>
          <td>2025-01-20</td>
          <td>Insights-Journal of Health and Rehabilitation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0841d31556f69507a5db1a9efb2aa8d37fe31d82" target='_blank'>
              DEMETER DNA demethylase reshapes the global DNA methylation landscape and controls cell identity transition during plant regeneration
              </a>
            </td>
          <td>
            Seunga Lee, S. Bae, Yunji Jeon, Pil Joon Seo, Yeonhee Choi
          </td>
          <td>2024-12-23</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The concept of genetic code expansion (GCE) has revolutionized the field of chemical and synthetic biology, enabling the site-specific incorporation of noncanonical amino acids (ncAAs) into proteins, thus opening new avenues in research and applications across biology and medicine. In this review, we cover the principles of GCE, including the optimization of the aminoacyl-tRNA synthetase (aaRS)/tRNA system and the advancements in translation system engineering. Notable developments include the refinement of aaRS/tRNA pairs, enhancements in screening methods, and the biosynthesis of noncanonical amino acids. The applications of GCE technology span from synthetic biology, where it facilitates gene expression regulation and protein engineering, to medicine, with promising approaches in drug development, vaccine production, and gene editing. The review concludes with a perspective on the future of GCE, underscoring its potential to further expand the toolkit of biology and medicine. Through this comprehensive review, we aim to provide a detailed overview of the current state of GCE technology, its challenges, opportunities, and the frontier it represents in the expansion of the genetic code for novel biological research and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c8f20128f59397264338dcf03afc1ecef23ea9" target='_blank'>
              Genetic Code Expansion: Recent Developments and Emerging Applications
              </a>
            </td>
          <td>
            Yujia Huang, Pan Zhang, Haoyu Wang, Yan Chen, Tao Liu, Xiaozhou Luo
          </td>
          <td>2024-12-31</td>
          <td>Chemical Reviews</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Mitochondria play key roles in maintaining cell life and cell function, and their dysfunction can lead to cell damage. Mitochondrial ribosomal proteins (MRPs) are encoded by nuclear genes and are assembled within the mitochondria. MRPs are pivotal components of the mitochondrial ribosomes, which are responsible for translating 13 mitochondrial DNA-encoded proteins essential for the mitochondrial respiratory chain. Recent studies have underscored the importance of MRPs in cancer biology, revealing their altered expression patterns in various types of cancer and their potential as both prognostic biomarkers and therapeutic targets. Herein, we review the current knowledge regarding the multiple functions of MRPs in maintaining the structure of the mitochondrial ribosome and apoptosis, their implications for cancer susceptibility and progression, and the innovative strategies being developed to target MRPs and mitoribosome biogenesis in cancer therapy. This comprehensive overview aims to provide insights into the role of MRPs in cancer biology and highlight promising strategies for future precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ce4a5576b3adfb615ae7e49696b3d3c132a872" target='_blank'>
              Mitochondrial Ribosomal Proteins and Cancer
              </a>
            </td>
          <td>
            Huiyi Wu, Xiaowei Zhu, Huilin Zhou, Min Sha, Jun Ye, Hong-Jeng Yu
          </td>
          <td>2025-01-01</td>
          <td>Medicina</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Precursor B cell acute lymphoblastic leukemia (pre-B-ALL) arises as a result of precursor B cells acquiring driver mutations that lead to arrested differentiation and increased proliferation. Identification of driver mutations and understanding their biological function is critical to understanding pre-B-ALL development and advancing disease treatment. Using a mouse model of pre-B-ALL driven by deletion of genes encoding the related E26-transformation-specific (ETS) transcription factors PU.1 and Spi-B, we performed whole exome sequencing to identify secondary driver mutations. We identified recurrent variants in E26 transformation-specific transcription variant 5 (ETV5) resulting in R392P, V444I, and T505A amino acid changes. We found that the R392P and V444I variants altered the ability of ETV5 to bind to DNA using electrophoretic mobility shift assay. R392P and V444I variants did not activate a Dual-Specificity-Phosphatase 6 (DUSP6) reporter. In contrast, T505A ETV5 could interact with DNA and activate the DUSP6 promoter. To determine biological function, we forced expression of wild type, R392P, V444I, or T505A ETV5 in an interleukin-7-dependent pre-B cell line. Proliferation and apoptosis assays showed that T505A ETV5 conferred a proliferative advantage to pre-B cells. RNA sequencing showed that expression of ETV5 variants significantly altered gene expression in cultured cells. Through gene set enrichment analysis, T505A was suggested to downregulate the p53 pathway and the anti-proliferative protein, B cell translocation gene 2 (encoded by Btg2). In summary, these data suggest that ETV5 mutations play a role in pre-B-ALL by affecting proliferation and cell survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f29fab27dbb4d851eea7f525bf89bbde3e863a" target='_blank'>
              T505A variant of ETV5 promotes proliferation of precursor B cells in a mouse model of acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Joshua Yi, Michael Wu, Michaela L. Dowling, Allanna C. E. MacKenzie, James Iansavitchous, R. DeKoter
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Kirsten Rat Sarcoma Virus mutation and its complications have been a major concern. Limitations to drawing its accurate configuration, its complications; mutates and thermodynamic properties, have limited their effectiveness as a molecular diagnostic tool. To counteract these challenges, over the years a different set of sophisticated Immunotherapeutic approaches have been adopted such as Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR), Caspase Protein System (CAS), and Chimeric Antigen Receptor-T cell (CAR-T). Some of them have already managed approval of the Food and Drug Administration. One of the primary goals of CRISPR/CAS technology is to correct mutations within the genetic sequence, thereby potentially repairing defective genes like KRAS. Meanwhile, CAR-T immunotherapies capitalise on genetically modified T cells. In comparison to traditional therapies, immunotherapies can offer greater safety and precision. However, their efficacy is still questionable while treating solid tumours. Here authors have covered the various key targets such as p53, p21, WNT, and RAS, their success stories over the years and have proposed the importance of targets like KRAS oncogene to manage a better output. KRAS is a RAS family oncoprotein responsible for the development of 85% of different cancers. The missense mutations at the various KRAS hotspots can lead to the development of lung and Pancreatic Cancers. Some of the most recently conducted revolutionary studies synthesize CAR-NK, CAR-M, and CRISPR-Cas9/3/10/Novel CAS systems, as well as their prospective molecular combination to target the KRAS. The purpose of this study is to provide a present status of these therapies, limitations, clinical trials, and a proposal to sketch a combinatorial impact via the PRISMA model for 62 different manuscripts (2001-2024). This format aligns and organizes everything in Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines, providing a clear and systematic infographic that represents the process, including stages of selection, screening, and documentation.
 Citation Format: SDanish Kadir. Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14aedd9264aefb4bd305bff7eb38afd48f109be" target='_blank'>
              Abstract B023: Harnessing the dual therapeutic approach of CRISPR/CAS9 and CAR-T cell technologies in KRAS mutated solid tumors
              </a>
            </td>
          <td>
            SDanish Kadir
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Within cells multiple related transcription factors targeting the same sequences may co-exist, leading to potential regulatory cooperativity, redundancy or competition. Yet the differential roles and biological functions of co-targeting transcription factors is poorly understood. In melanoma, three highly-related transcription factors are co-expressed: The mTORC1-regulated TFEB and TFE3, that are key effectors of a wide range of metabolic and microenvironmental cues; and MITF, that controls melanoma phenotypic identity. Here we reveal the functional specialization of MITF, TFE3 and TFEB and their impact on cancer progression. Notably, although all bind the same sequences, each regulates radically different and frequently opposing gene expression programs to coordinate differentiation, metabolic reprogramming, protein synthesis, and expression of immune modulators. The results uncover a hierarchical cascade in which microenvironmental stresses, including glucose limitation, lead MITF, TFEB and TFE3 to drive distinct biologically important transcription programs that underpin phenotypic transitions in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8434656ba881322bb7b7747d875569336b1b3a2" target='_blank'>
              Functional specialization of MITF, TFEB and TFE3 drives radically distinct adaptive gene expression programs in melanoma
              </a>
            </td>
          <td>
            Diogo Dias, Erica Oliveira, Román Martí-Díaz, Sarah Andrews, Ana Chocarro-Calvo, Alice Bellini, Laura Mosteo, Yurena Vivas García, J. Chauhan, Linxin Li, J. M. García-Martínez, José Neptuno Rodríguez-López, S. Maria-Engler, C. García-Jiménez, Luis Sánchez-Del-Campo, Pakavarin Louphrasitthiphol, Colin R. Goding
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Colorectal cancer is a common and fatal disease that affects many people globally. CRC is classified as the third most prevalent cancer among males and the second most frequent cancer among females worldwide. The purpose of this article is to examine how personalized medicine might be used to treat colorectal cancer. The classification of colorectal cancer based on molecular profiling, including the detection of significant gene mutations, genomic instability, and gene dysregulation, is the main topic of this discussion. Advanced technologies and biomarkers are among the detection methods that are explored, demonstrating their potential for early diagnosis and precise prognosis. In addition, the essay explores the world of treatment possibilities by providing light on FDA-approved personalized medicine solutions that provide individualized and precise interventions based on patient characteristics. This article assesses targeted treatments like cetuximab and nivolumab, looks at the therapeutic usefulness of biomarkers like microsatellite instability (MSI) and circulating tumor DNA (ctDNA), and investigates new approaches to combat resistance. Through this, our review provides a thorough overview of personalized medicine in the context of colorectal cancer, ultimately highlighting its potential to revolutionize the field and improve patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b80ad33c609a3090e37d7111c49bfa155aa4a9e" target='_blank'>
              Personalized medicine in colorectal cancer: a comprehensive study of precision diagnosis and treatment.
              </a>
            </td>
          <td>
            Fatemeh Gila, Somayeh Khoddam, Zahra Jamali, Mohmmad Ghasemian, Shayan Shakeri, Zeinab Dehghan, Jafar Fallahi
          </td>
          <td>2025-02-09</td>
          <td>Personalized medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Ovarian cancer is a deadly gynecologic disease with frequent recurrence. Current treatments for patients include platinum-based therapy regimens with PARP inhibitors specific for homologous recombination–deficient high-grade serous ovarian cancers (HGSOC). Despite initial effectiveness, patients inevitably develop disease progression as tumor cells acquire resistance. Toward the development of new therapeutic avenues, we describe a gene amplification involving both heat shock factor 1 (HSF1) and MYC, wherein these two genes are co-amplified in more than 30% of patients with HGSOC. We further found that HSF1 and MYC transcriptional activities were highly correlated with human HGSOC tumors and cell lines, suggesting that they may cooperate in the disease. CUT&RUN sequencing for HSF1 and MYC revealed overlapping HSF1 and MYC binding throughout the genome. Moreover, the binding peaks of both transcription factors in HGSOC cells were nearly identical, and a protein–protein interaction between HSF1 and MYC was detected, supporting molecular cooperation. Supporting a functional cooperation of these two transcription factors, the growth of HGSOC cells with the co-amplification was dependent on both HSF1 and MYC. To identify a therapeutic target that could take advantage of this unique HSF1 and MYC dependency, polo-like kinase 1 (PLK1) was correlated with HSF1 and MYC in HGSOC specimens. Targeting PLK1 with volasertib revealed a greater than 200-fold increased potency in HSF1–MYC co-amplified HGSOC cells compared with those with wild-type HSF1 and MYC copy numbers. Although the success of volasertib and other PLK1 inhibitors in clinical trials has been modest, the current study suggests that targeting PLK1 using a precision medicine approach based on HSF1–MYC co-amplification as a biomarker in HGSOC would improve therapy response and patient outcomes. Significance: We show that HSF1 and MYC genes are co-amplified in more than 30% of HGSOC and demonstrate that HSF1 and MYC functionally cooperate to drive the growth of HGSOC cells. This work provides the foundation for HSF1 and MYC co-amplification as a biomarker for treatment efficacy of the polo-like kinase 1 inhibitor volasertib in HGSOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37b1725a2f0ac79cd573a1bb5f9e94eafaf83ac" target='_blank'>
              MYC and HSF1 Cooperate to Drive Sensitivity to Polo-like Kinase 1 Inhibitor Volasertib in High-grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Matthew O'Malley, Haddie DeHart, Bobby Walker, Vrushabh Ulhaskumar, Pranav Jothirajah, Haimanti Ray, Lisa M Landrum, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2025-01-20</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6666f30aff9bcd1de0a39ac7dc32869d01658a2" target='_blank'>
              Integrated analysis of single-cell and bulk-RNA sequencing for the cellular senescence in prognosis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Fengqiang Yu, Liangyu Zhang, Xun Zhang, Jianshen Zeng, Fancai Lai
          </td>
          <td>2025-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Philadelphia chromosome-like B-cell acute lymphoblastic leukemia (Ph-like ALL) is driven by genetic alterations that induce constitutive kinase signaling and is associated with chemoresistance and high relapse risk in children and adults. Preclinical studies in the most common CRLF2-rearranged/JAK pathway-activated Ph-like ALL subtype have shown variable responses to JAK inhibitor-based therapies, suggesting incomplete oncogene addiction and highlighting a need to elucidate alternative biologic dependencies and therapeutic vulnerabilities, while the ABL-class Ph-like ALL subtype appears preferentially sensitive to SRC/ABL- or PDGFRB-targeting inhibitors. Which patients may be responsive versus resistant to tyrosine kinase inhibitor (TKI)-based precision medicine approaches remains a critical knowledge gap. Using bulk and single-cell multiomics analyses, we profiled residual cells from CRLF2-rearranged or ABL1-rearranged Ph-like ALL patient-derived xenograft models treated in vivo with targeted inhibitors to identify TKI-resistant subpopulations and potential mechanisms of therapeutic escape. We detected a specific MYC dependency in Ph-like ALL cells and defined a new leukemia cell subpopulation with senescence-associated stem cell-like features regulated by AP-1 transcription factors. This dormant ALL subpopulation was effectively eradicated by dual pharmacologic inhibition of BCL-2 and JAK/STAT or SRC/ABL pathways, a clinically-relevant therapeutic strategy. Single cell-derived molecular signatures of this senescence and stem/progenitor-like subpopulation further predicted poor clinical outcomes associated with other high-risk genetic subtypes of childhood B-ALL and thus may have broader prognostic applicability beyond Ph-like ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b87bb41065d73fc1bc25a49641c54717955deab" target='_blank'>
              Targeting Senescent Stemlike Subpopulations in Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Yang-yang Ding, Jonathan H Sussman, Kellyn Madden, J. Loftus, Robert K Chen, C. D. Falkenstein, D. A. Barcenas Lopez, David A Hottman, Benjamin Mathier, Wenbao Yu, Jason Xu, Changya Chen, Chia-Hui Chen, Bing He, S. Bandyopadhyay, Zhan Zhang, DongGeun Lee, Hong Wang, Junmin Peng, Chi V. Dang, Kai Tan, S. Tasian
          </td>
          <td>2025-01-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="DNA methylation is an epigenetic mechanism of gene silencing crucial to the regulation of gene expression in normal physiological events such as differentiation and X-inactivation. However, aberrant silencing of regulatory genes can contribute to oncogenic transformation, further perpetuated due to the heritability of these changes down the cell line. Though aberrant DNA methylation is implicated across cancer types, epigenetic therapy with DNA methyltransferase 1 (DNMT1) depleting drugs is only approved in the treatment of hematological cancers. This limitation is due to the drugs’ high toxicity, a byproduct of their mechanism as nucleoside analogs. Identifying less toxic nucleoside analogs or developing non-nucleoside analogs is necessary to expand the application of epigenetic therapy. This review examines the existing nucleoside and non-nucleoside DNMT1-inhibitors at various stages of preclinical and clinical development, and overviews prospective applications of epigenetic therapy in solid tumors as monotherapy and combined therapy. The list of drugs reported in this review is non-comprehensive as it excludes primary research on drugs last tested prior to the FDA approval of azacitidine and drugs not tested or inviable in human cells. This is to limit the survey to studies that intend to improve upon the pharmacological profile of the approved drugs. Of particular importance are the recently developed DNMT1-inhibitor (DNMT1-i) GSK analogs and the advancements in protein modeling that have elucidated their mechanism to the greatest precision yet. Not yet discussed in length in secondary literature, these findings provide a clearer model for the development of more specific DNMT-is. Furthermore, evidence showing enhanced efficacy of DNMT1 inhibitors when combined with DNA damaging agents identifies epigenetic combination therapy as a pertinent focus of future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c56d0e33616064b81832853efdda25a0062176f" target='_blank'>
              Exploring Epigenetic Complexity in Regulation of Hematopoietic Stem Cells Niche: A Mechanistic Journey from Normal to Malignant Hematopoiesis.
              </a>
            </td>
          <td>
            N. A. Yusoff, Zariyantey Abd Hamid, Izatus Shima Taib, Siti Razila Abdul Razak, S. B. Budin
          </td>
          <td>2025-01-23</td>
          <td>Advances in experimental medicine and biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="MOTIVATION
The emergence of the "telomere-to-telomere" genomics brought the challenge of identifying segmental duplications (SDs) in complete genomes. It further opened a possibility for identifying the differences in SDs across individual human genomes and studying the SD evolution. These newly emerged challenges require algorithms for reconstructing SDs in the most complex genomic regions that evaded all previous attempts to analyze their architecture, such as rapidly-evolving immunoglobulin loci.


RESULTS
We describe the GenomeDecoder algorithm for inferring SDs and apply it to analyzing genomic architectures of various loci in primate genomes. Our analysis revealed that multiple duplications/deletions led to a rapid birth/death of immunoglobulin genes within the human population and large changes in genomic architecture of immunoglobulin loci across primate genomes. Comparison of immunoglobulin loci across primate genomes suggests that they are subjected to diversifying selection.


AVAILABILITY AND IMPLEMENTATION
GenomeDecoder is available at https://github.com/ZhangZhenmiao/GenomeDecoder. The software version and test data used in this paper is uploaded to https://doi.org/10.5281/zenodo.14753844.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e06706d15b6b81ace23df04147d261b78926995" target='_blank'>
              GenomeDecoder: Inferring Segmental Duplica-tions in Highly-Repetitive Genomic Regions.
              </a>
            </td>
          <td>
            Zhenmiao Zhang, Ishaan Gupta, Pavel Pevzner
          </td>
          <td>2025-02-05</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22203f23dc18f7e477336e6b95f3919f3954fe27" target='_blank'>
              Mechanisms and therapeutic implications of gene expression regulation by circRNA-protein interactions in cancer
              </a>
            </td>
          <td>
            Nan Zhang, Xinjia Wang, Yu Li, Yiwei Lu, Chengcheng Sheng, Yumeng Sun, Ningye Ma, Yisheng Jiao
          </td>
          <td>2025-01-17</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Productive infections of oncogenic human papillomaviruses (HPVs) are closely linked to the differentiation of host epithelial cells, a process that the virus manipulates to create conditions favorable to produce virion progeny. This viral interference involves altering the expression of numerous host genes. Among these, proprotein convertases (PCs) have emerged as potential oncogenes due to their central role in cellular functions. Using RT-qPCR, aberrant PC gene expression was detected across the progression from early HPV infection stages to cancer. These findings demonstrated a progressive disruption of normal PC expression profiles, with FURIN consistently downregulated and other PCs upregulated, transitioning from the episomal stage to neoplastic and carcinoma phenotypes. This pattern of dysregulation was distinct from the broader trends observed in a variety of cancer types through bioinformatic analysis of publicly available transcriptomic datasets, where FURIN expression was predominantly upregulated compared to other PCs. Further bioinformatic investigations revealed a correlation between PC gene expression and cancer phenotype diversity, suggesting a potential link between the loss of normal PC gene expression patterns and the progression of HPV infections toward malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f01d2dc341bbc570cc91f7701d59ca7b10ccc365" target='_blank'>
              Dysregulation of FURIN and Other Proprotein Convertase Genes in the Progression from HPV Infection to Cancer
              </a>
            </td>
          <td>
            Gonzalo Izaguirre, Natalia Zirou, Craig Meyers
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Post-translational modifications (PTMs) of proteins dynamically build the buffering and adapting interface between oncogenic mutations and environmental stressors, on the one hand, and cancer cell structure, functioning, and behavior. Aberrant PTMs can be considered as enabling characteristics of cancer as long as they orchestrate all malignant modifications and variability in the proteome of cancer cells, cancer-associated cells, and tumor microenvironment (TME). On the other hand, PTMs of proteins can enhance anticancer mechanisms in the tumoral ecosystem or sustain the beneficial effects of oncologic therapies through degradation or inactivation of carcinogenic proteins or/and activation of tumor-suppressor proteins. In this review, we summarized and analyzed a wide spectrum of PTMs of proteins involved in all regulatory mechanisms that drive tumorigenesis, genetic instability, epigenetic reprogramming, all events of the metastatic cascade, cytoskeleton and extracellular matrix (ECM) remodeling, angiogenesis, immune response, tumor-associated microbiome, and metabolism rewiring as the most important hallmarks of cancer. All cancer hallmarks develop due to PTMs of proteins, which modulate gene transcription, intracellular and extracellular signaling, protein size, activity, stability and localization, trafficking, secretion, intracellular protein degradation or half-life, and protein–protein interactions (PPIs). PTMs associated with cancer can be exploited to better understand the underlying molecular mechanisms of this heterogeneous and chameleonic disease, find new biomarkers of cancer progression and prognosis, personalize oncotherapies, and discover new targets for drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22300c2f6fc124353b32d558bbea365d47496670" target='_blank'>
              Post-Translational Modifications of Proteins Orchestrate All Hallmarks of Cancer
              </a>
            </td>
          <td>
            Pathea Bruno, Aneeta Arshad, Maria-Raluca Gogu, Natalie Waterman, Rylie Flack, Kimberly Dunn, Costel C. Darie, A. Neagu
          </td>
          <td>2025-01-01</td>
          <td>Life</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states.
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdfee082d103369a456ad016f2f970300d872d89" target='_blank'>
              Pan-cancer and multiomics: advanced strategies for diagnosis, prognosis, and therapy in the complex genetic and molecular universe of cancer.
              </a>
            </td>
          <td>
            Camilly Victória Campanharo, Lívia Valle Dos Santos Silveira, D. D. Meira, Matheus Correia Casotti, Lorena Souza Castro Altoé, I. Louro, André Felipe Monteiro Gonçalves, André Manhães Machado, Breno Sousa Paiva, Ester de Souza Inocencio, Fabio Victor Vieira Rocha, Fellipe Pesente, Giulia de Souza Cupertino de Castro, João Pedro Dos Santos da Paixão, José Henrique Borges Bourguignon, Júlia Salarini Carneiro, Juliana Ribeiro de Oliveira, Pâmela de Souza Freire, Sophia Bridi Zamprogno, Taissa dos Santos Uchiya, Thais de Paula Rezende, Vinícius de Pádua Sanders Medeiros
          </td>
          <td>2024-12-26</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1d734aed102ed089c1bedbb1c15be484c90edb1" target='_blank'>
              tRNA-derived small RNAs: their role in the mechanisms, biomarkers, and therapeutic strategies of colorectal cancer
              </a>
            </td>
          <td>
            Bo Zhang, Yanru Pan, Zhe Li, Kefeng Hu
          </td>
          <td>2025-01-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The DNA double-strand break (DSB) always occurs within genome. The pathways that repair the DSB are essential in keeping cell viability. This review will focus on the recent understanding of TMEJ, especially two key protein factors involved in the TMEJ repair pathway2, DNA polymerase theta (Polθ) and DNA polymerase delta (Polδ), and discuss the potential mechanism of choosing repair pathways when there is a DNA Double Strand Break. We dissect TMEJ's unique mechanism of action, including recognition of DNA ends, microhomology search, end pairing, and DNA synthesis, by focusing on the enzyme Polθ as the central player to understand the repair pathway choice. We also discuss how BRCA-mutated cancer uses the TMEJ repair pathway to facilitate breast cancer cells' growth3. This review aims to provide a comprehensive overview of TMEJ's cellular functions, regulatory mechanisms, and pivotal role in BRCA-mutated breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ba6a1c7fb204a897897b1b79890ca6faef600e" target='_blank'>
              Polymerase Theta-mediated DNA End-Joining Repair in the Maintenance of Genome Stability
              </a>
            </td>
          <td>
            Xiangyu Yan
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous gastrointestinal malignancy. Despite significant advances in molecular targeted therapies for CRC in recent years, the increase in the overall survival rates for CRC patients remains limited. Therefore, there is an urgent need to explore novel drug targets. Herein, we show that heparin binding growth factor (HDGF) is highly expressed in CRC, and that its overexpression is associated with a poor disease-free interval. Additionally, we reveal that HDGF knockout reduces proliferation, migration, and invasion, while enhancing apoptosis in CRC cells, thereby validating HDGF as a potential therapeutic target for CRC. Mechanistically, we found that HDGF modulates DNA damage response and, by recruiting C-terminal binding protein-interacting protein (CtIP), it facilitates homologous recombination repair to influence CRC drug sensitivity. Furthermore, we propose that HDGF may serve as a recognition protein for H3K36me3, participating in the repair of damaged transcriptionally active genes, thus maintaining genomic stability in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f394455983264a89c22ede32239ab6721086ed77" target='_blank'>
              HDGF Knockout Suppresses Colorectal Cancer Progression and Drug Resistance by Modulating the DNA Damage Response
              </a>
            </td>
          <td>
            Riya Su, Qin Wang, Qun Hu, Wendurige, Kexin Li, Changshan Wang, Liang Tao
          </td>
          <td>2025-02-14</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Motivation Human mitochondrial DNA (mtDNA) mutations are causally implicated in maternally inherited mitochondrial respiratory disorders; however, the role of somatic mtDNA mutations in both late-onset chronic diseases and cancer remains less clear. Although recent advances in mtDNA base editing technology have the potential to model and characterize many of these mutations, current editing approaches are complicated by the potential for multiple unintentional edits (bystanders) that are only identifiable through empirical ‘trial and error’, thereby sacrificing valuable time and effort towards suboptimal construct development. Results We developed MitoEdit, a novel tool that incorporates empirical base editor patterns to facilitate identification of optimal target windows and potential bystander edits. MitoEdit allows users to input DNA sequences in a text-based format, specifying the target base position and its desired modification. The program generates a list of candidate target windows with a predicted number of bystander edits and their functional impact, along with flanking nucleotide sequences designed to bind TALE (transcription activator-like effectors) array proteins. In silico evaluations indicate that MitoEdit can predict the majority of bystander edits, thereby reducing the number of constructs that need to be tested empirically. To the best of our knowledge, MitoEdit is the first tool to automate prediction of base edits. Availability and implementation MitoEdit is freely available at Kundu Lab GitHub (https://github.com/Kundu-Lab/mitoedit). Contact Corresponding email: Gang.Wu@stjude.org; Mondira.Kundu@stjude.org Supplementary information Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb09ec051ad08fe864f50953db873f84673f691" target='_blank'>
              MitoEdit: a pipeline for optimizing mtDNA base editing and predicting bystander effects
              </a>
            </td>
          <td>
            Devansh Shah, K. McCastlain, Ti-Cheng Chang, Gang Wu, M. Kundu
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Background Periodontal disease is a widespread inflammatory condition that compromises the supporting structures of the teeth, potentially resulting in tooth loss if left untreated. Despite advancements in therapeutic interventions and an enhanced understanding of its pathophysiology, emerging techniques such as single-cell RNA sequencing (scRNA-seq) and Mendelian randomization (MR) present new opportunities for precision medicine in the management of periodontal disease. Methods Data derived from the GSE152042 dataset underwent rigorous quality control, normalization, and dimensionality reduction using Seurat and the MonacoImmuneData framework. Marker genes were identified to delineate subgroups for subsequent analysis utilizing CellChat and ClusterProfilerR. MR analysis of the expression quantitative trait loci (eQTLs) for these genes was conducted to determine causal relationships with periodontal disease, leveraging data from the IEU Open GWAS project. Results Single-cell analysis revealed distinct immune cell subtypes and indicated an increased presence of myeloid dendritic cells (mDCs) in patients with periodontal disease. MR analysis identified twenty-six significant genes, with LIMA1 (LIM domain and actin-binding 1) demonstrating a robust causal association with the progression of periodontal disease. Gene ontology and Kyoto Encyclopedia of Genes and Genomes analyses highlighted crucial pathways involved in periodontal inflammation and tissue destruction. Visualization at the single-cell level elucidated the role of LIMA1 in disease progression, alongside differences in cell communication dynamics between LIMA1-positive and -negative populations. Conclusion This study underscores the utility of scRNA-seq and MR in elucidating essential factors in the pathogenesis of periodontal disease, thereby reinforcing the necessity for targeted therapeutic strategies. The identification of LIMA1 as a pivotal gene in periodontal disease progression opens new avenues for precision medicine approaches, potentially enhancing treatment efficacy and patient outcomes in periodontal management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5112127a205674f57ff2162ed2050e8725facc14" target='_blank'>
              Myeloid dendritic cells and periodontal disease association: integrated study of single-cell sequencing and Mendelian randomization analysis
              </a>
            </td>
          <td>
            ChengJi Shi, Xinyi Ou, XiaoXu Lei, Lijuan Huang, Shuhao Xu, Wei Li, Xi Zhao
          </td>
          <td>2025-01-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63030f8f4a5318cca9879e32ba13ec7a2af3ed4" target='_blank'>
              Chromosome mis-segregation triggers cell cycle arrest through a mechanosensitive nuclear envelope checkpoint
              </a>
            </td>
          <td>
            Solène Hervé, Andrea Scelfo, Gabriele Bersano Marchisio, Marine Grison, Kotryna Vaidžiulytė, Marie Dumont, Annapaola Angrisani, Adib Keikhosravi, Gianluca Pegoraro, Mathieu Deygas, Guilherme P. F. Nader, Anne-Sophie Macé, Matteo Gentili, Alice Williart, N. Manel, Matthieu Piel, Y.A. Miroshnikova, D. Fachinetti
          </td>
          <td>2025-01-08</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="PARP inhibitors (PARPi) have received regulatory approval for the treatment of several tumors, including prostate cancer (PCa), and demonstrate remarkable results in the treatment of castration-resistant prostate cancer (CRPC) patients characterized by defects in homologous recombination repair (HRR) genes. Preclinical studies showed that DNA repair genes (DRG) other than HRR genes may have therapeutic value in the context of PARPi. To this end, we performed multiple CRISPR/Cas9 screens in PCa cell lines using a custom sgRNA library targeting DRG combined with PARPi treatment. We identified DNA ligase 1 (LIG1), essential meiotic structure-specific endonuclease 1 (EME1), and Fanconi anemia core complex associated protein 24 (FAAP24) losses as PARPi sensitizers and assessed their frequencies from 3% to 6% among CRPC patients. We showed that concomitant inactivation of LIG1 and PARP induced replication stress and DNA double-strand breaks, ultimately leading to apoptosis. This synthetic lethality (SL) is conserved across multiple tumor types (e.g., lung, breast, and colorectal), and its applicability might be extended to LIG1-functional tumors through a pharmacological combinatorial approach. Importantly, the sensitivity of LIG1-deficient cells to PARPi was confirmed in vivo. Altogether, our results argue for the relevance of determining the status of LIG1 and potentially other non-HRR DRG for CRPC patient stratification and provide evidence to expand their therapeutic options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6742716d8078e4d21c99bceef70e0cdf901ca76" target='_blank'>
              CRISPR/Cas9 screens identify LIG1 as a sensitizer of PARP inhibitors in castration-resistant prostate cancer
              </a>
            </td>
          <td>
            Giulia Fracassi, F. Lorenzin, F. Orlando, Ubaldo Gioia, Giacomo D’Amato, A. S. Casaramona, T. Cantore, D. Prandi, F. Santer, Helmut Klocker, Fabrizio d'Adda di Fagagna, J. Mateo, F. Demichelis
          </td>
          <td>2024-12-24</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="OBJECTIVE
Hematological malignancies often involve chromosomal translocations and fusion genes that drive disease progression. While MTCP1 is well-known in T-cell prolymphocytic leukemia (T-PLL), its role in myeloid neoplasms is less understood. This report presents the first identification of the t(X;1)(q28;q21) translocation leading to the GATAD2B::MTCP1 fusion in acute myeloid leukemia (AML) transformed from chronic myelomonocytic leukemia (CMML).


METHODS
The karyotypes were described according to the International System for Human Cytogenetic Nomenclature 2009. We performed targeted next-generation sequencing (NGS) on a panel of 172 genes commonly mutated in hematological malignancies (Supplemental Table 1), using an Illumina platform. RNA sequencing was conducted on total RNA extracted from bone marrow, also using the Illumina platform. The GATAD2B::MTCP1 fusion gene was confirmed by reverse transcription-polymerase chain reaction (RT-PCR) and Sanger sequencing, with specific primers for the fusion transcript (GATAD2B-F: CCTCTTTTTTTCGACGCC; MTCP1-R: ACTGAGCACAACACTTACGC).


RESULTS
The GATAD2B::MTCP1 fusion results from a breakpoint on 1q21 within GATAD2B exon 1 and Xq28 within MTCP1 exon 2. The patient with the GATAD2B::MTCP1 fusion exhibited disease progression from CMML to AML. Despite achieving initial remission with venetoclax-based therapy and allo-HSCT, the patient relapsed and died.


CONCLUSIONS
We propose that the GATAD2B::MTCP1 fusion upregulates MTCP1 expression rather than generating a fusion protein, thereby contributing to transformation and relapse in AML. Further investigations are needed to elucidate the precise role of this fusion event in myeloid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b642c96f7753533d71b6085fbf58b5dbe294ae5" target='_blank'>
              Identification of t(X;1)(q28;q21) generating a novel GATAD2B::MTCP1 gene fusion in CMML and its persistence during progression to AML.
              </a>
            </td>
          <td>
            Yi-Zi Liu, Fenghong Zhang, Chun-Xiao Hou, Zhi-Yu Zhang, Yiyan Zhu, Qian Wang, Yu Chen, Suqun Chen
          </td>
          <td>2024-12-18</td>
          <td>Hematology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer characterized by the lack of estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2). Chemotherapy remains the primary treatment option, yet TNBC frequently develops resistance, leading to relapse and metastasis. Emerging evidence highlights the potential of combining DNA methylation inhibitors with immune checkpoint inhibitors (ICIs). DNA methylation contributes to immune escape by silencing immune-regulatory genes, thereby reducing the tumor’s visibility to immune cells. Reversing this epigenetic modification can reinvigorate immune surveillance and enhance the efficacy of immunotherapies. This review discusses the role of DNA methylation in TNBC progression and immune evasion, focusing on recent advances in combination therapies involving DNA methylation inhibitors and ICIs. We discuss the underlying mechanisms that enable these therapeutic synergies, preclinical and clinical evidence supporting the approach, and the challenges posed by tumor heterogeneity, drug resistance, and toxicity. Finally, we explore the potential for personalized treatment strategies incorporating multi-omics data to optimize therapeutic outcomes. The integration of epigenetic therapies and immunotherapy offers a promising avenue for improving survival in TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c1a1657cbbdb3d35b60d2224060b7074a6c9d9c" target='_blank'>
              DNA methylation and immune evasion in triple-negative breast cancer: challenges and therapeutic opportunities
              </a>
            </td>
          <td>
            Wen-yu Cai, , Yi-ran Fei, Rui Ye, , Dan Hu, Wan-wan Zhang, Ming-fei Xia, Xiao-xiao Yang
          </td>
          <td>2025-02-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="GC-biased gene conversion (gBGC) is a widespread evolutionary force associated with meiotic recombination that favours the accumulation of deleterious AT to GC substitutions in proteins, moving them away from their fitness optimum. In many mammals recombination hotspots have a rapid turnover, leading to episodic gBGC, with the accumulation of deleterious mutations stopping when the recombination hotspot dies. Selection is therefore expected to act to repair the damage caused by gBGC episodes through compensatory evolution. However, this process has never been studied or quantified so far. Here, we analysed the nucleotide substitution pattern in coding sequences of a highly diversified group of Murinae rodents. Using phylogenetic analyses of about 70,000 coding exons, we identified numerous exon-specific, lineage-specific gBGC episodes, characterised by a clustering of synonymous AT to GC substitutions and by an increasing rate of non-synonymous AT to GC substitutions, many of which are potentially deleterious. Analysing the molecular evolution of the affected exons in downstream lineages, we found evidence for pervasive compensatory evolution after deleterious gBGC episodes. Compensation appears to occur rapidly after the end of the episode, and to be driven by the standing genetic variation rather than new mutations. Our results demonstrate the impact of gBGC on the evolution of amino-acid sequences, and underline the key role of epistasis in protein adaptation. This study contributes to a growing body of literature emphasizing that adaptive mutations, which arise in response to environmental changes, are just one subset of beneficial mutations, alongside mutations resulting from oscillations around the fitness optimum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efad2b9652457c29bcba988b3cdd7f8c5cf16ca6" target='_blank'>
              Molecular re-adaptation : compensatory evolution following deleterious episodes of GC-biased gene conversion in rodents
              </a>
            </td>
          <td>
            Marie Riffis, N. Saclier, Nicolas Galtier
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Renowned as a highly invasive and lethal tumor derived from neural stem cells in the central nervous system, glioblastoma (GBM) exhibits substantial histopathological variation and genomic complexity, which drive its rapid progression and therapeutic resistance. Alterations in mitochondrial DNA (mtDNA) copy number (CN) serve a crucial role in GBM development and progression, affecting various aspects of tumor biology, including energy production, oxidative stress regulation and cellular adaptability. Fluctuations in mtDNA levels, whether elevated or diminished, can impair mitochondrial function, potentially disrupting oxidative phosphorylation and amplifying reactive oxygen species generation, thereby fueling tumor growth and influencing treatment responses. Understanding the mechanisms of mtDNA-CN variations, and their interplay with genetic and environmental elements in the tumor microenvironment, is essential for advancing diagnostic and therapeutic strategies. Targeting mtDNA alterations could strengthen treatment efficacy, mitigate resistance and ultimately enhance the prognosis of patients with this aggressive brain tumor. The present review summarizes the existing literature on mtDNA alterations, specifically emphasizing variations in mtDNA-CN and their association with GBM by surveying articles published between 1996 and 2024, sourced from databases such as Scopus, PubMed and Google Scholar. In addition, the review provides a brief overview of mitochondrial genome architecture, knowledge regarding the regulation of mtDNA integrity and CN, and how mitochondria significantly impact GBM tumorigenesis. This review further presents information on therapeutic approaches for restoring mtDNA-CN that contribute to optimized mitochondrial function and improved health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/608231a30fdc2050cf19ab35c688a4cbcbde93d3" target='_blank'>
              Mitochondrial DNA copy number alterations: Key players in the complexity of glioblastoma (Review)
              </a>
            </td>
          <td>
            Abdul Aziz Mohamed Yusoff, Siti Zulaikha Nashwa Mohd Khair, Siti Muslihah Abd Radzak
          </td>
          <td>2025-01-24</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="This study aimed to elucidate the transcriptomic signatures and dysregulated pathways associated with the autoimmune response in Cd38-/- mice compared to wild-type (WT) mice within the bm12 chronic graft-versus-host disease (cGVHD) lupus model. We conducted bulk RNA sequencing on peritoneal exudate cells (PECs) and spleen cells (SPC) at two and four weeks following adoptive cell transfer. We also analyzed cells from healthy, untreated mice. These analyses revealed a sustained upregulation of a transcriptional profile of purinergic receptors and ectonucleotidases in cGVHD WT PECs, which displayed a coordinated expression with several type I interferon-stimulated genes (ISGs) and with key molecules involved in the cyclic GMP-AMP synthase-stimulator of interferon genes (cGAS-STING) signaling pathway, two hallmarks in the lupus pathology. A second purinergic receptor transcriptomic profile, which included P2rx7 and P2rx4, showed a coordinated gene expression of the components of the NLRP3 inflammasome with its potential activators. These processes were transcriptionally less active in cGVHD Cd38-/- PECs than in WT PECs. We have also shown evidence of a distinct enrichment in pathways signatures that define processes such as Ca2+ ion homeostasis, cell division, phagosome, autophagy, senescence, cytokine/cytokine receptor interactions, Th17 and Th1/Th2 cell differentiation in Cd38-/- versus WT samples, which reflected the milder inflammatory and autoimmune response elicited in Cd38-/- mice relative to WT counterparts in response to the allogeneic challenge. Last, we have shown an intense metabolic reprogramming toward oxidative phosphorylation in PECs and SPC from cGVHD WT mice, which may reflect an increased cellular demand for oxygen consumption, in contrast to PECs and SPC from cGVHD Cd38-/- mice, which showed a short-lived metabolic effect at the transcriptomic level. Overall, these findings support the pro-inflammatory and immunomodulatory role of CD38 during the development of the cGVHD-lupus disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47a5fade16c476abb5f2145f8e83f353870ec91" target='_blank'>
              CD38 deficiency leads to a defective short-lived transcriptomic response to chronic graft-versus-host disease induction, involving purinergic signaling-related genes and distinct transcriptomic signatures associated with lupus
              </a>
            </td>
          <td>
            M. Zubiaur, L. Terrón-Camero, Fernando Gordillo-González, Eduardo Andrés-León, Alicia Barroso-del Jesús, Luz María Canet-Antequera, María M. Pérez Sánchez-Cañete, África Martínez-Blanco, M. Domínguez-Pantoja, María Botía-Sánchez, Sonia Pérez-Cabrera, Nerea Bello-Iglesias, A. Alcina, A. Abadía-Molina, F. Matesanz, Esther Zumaquero, Ramón Merino, Jaime Sancho
          </td>
          <td>2025-02-10</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a small, yet pivotal subpopulation of tumor cells that play significant roles in tumor initiation, progression and therapeutic resistance. Circular RNAs (circRNAs) are a distinct class of RNAs characterized by their closed-loop structures, lacking 5′ to 3′ends. There is growing evidence that circRNAs are integral to the development and regulation of CSCs. Aberrant expression of circRNAs in CSCs can contribute to oncogenic properties and drug resistance. Specifically, oncogenic circRNAs modulate CSC behavior via key signaling pathways, thereby promoting CSC self-renewal and maintenance, as well as tumor progression. This review summarizes the latest research on the functional roles and regulatory mechanisms of circRNAs in CSC behavior and discusses potential applications and challenges of targeting circRNAs in CSCs. Understanding the intricate interactions between circRNAs and CSCs may lead to novel therapeutic strategies that effectively combat treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01b87d0f590a46f3144a9c75dc4396105c17dca" target='_blank'>
              Circular RNAs in cancer stem cells: Insights into their roles and mechanisms (Review)
              </a>
            </td>
          <td>
            Lunyu Yang, Yuling Yi, Zhu Mei, Dongmei Huang, Sitian Tang, Liyi Hu, Ling Liu
          </td>
          <td>2025-01-23</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Horizontal gene transfer (HGT) is a major driver of bacterial evolution, allowing bacteria to acquire new traits from other organisms. It enables the transfer of advantageous traits, such as antibiotic resistance, which can rapidly spread across bacterial populations, affecting public health. Several studies have estimated how widespread HGT can be and suggested that environmental changes are associated with changes in the rate of HGT. It is often considered that environmental changes lead to an increase in HGT rates due to the adaptive benefits of HGT. However, previous studies have been limited to one or a few species and have often focused on specific genes of interest. Here, we systematically investigate the effect of environmental changes on the rate of HGT in a large number of bacterial genomes along a deep phylogeny. We extend the scope of use of existing state-of -the -art methods for HGT inference to infer the transition into new ecosystems and the gain of pathogenicity. Transitions to new environments are associated with a decrease in the rate of HGT, along with the strengthening of purifying selection. This decrease in HGT rates is consistent regardless of genomic relatedness between the donor and recipient. Our results are robust to the use of different methodological frameworks and have implications for understanding the evolution of bacteria and HGT rates in natural microbial communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509a20de1480698aa85c82ec7a8080e8f597bcfe" target='_blank'>
              Horizontal gene transfer rates decrease during transitions to new ecosystems
              </a>
            </td>
          <td>
            Swastik Mishra, Martin J. Lercher
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2962c853fd82d5be105d8daa9d66daafd430b25e" target='_blank'>
              Elongation factor 2 in cancer: a promising therapeutic target in protein translation
              </a>
            </td>
          <td>
            X. Jia, Chuntian Huang, Fangfang Liu, Zigang Dong, Kangdong Liu
          </td>
          <td>2024-12-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a819a83329e7d0a414c5918852d0411278218a5" target='_blank'>
              High CDC20 levels increase sensitivity of cancer cells to MPS1 inhibitors.
              </a>
            </td>
          <td>
            Siqi Zheng, Linoy Raz, Lin Zhou, Yael Cohen-Sharir, Ruifang Tian, M. Ippolito, Sara Gianotti, Ron Saad, René Wardenaar, Mathilde Broekhuis, Maria F Suarez Peredo Rodriguez, Soraya Wobben, A. van den Brink, Petra Bakker, S. Santaguida, F. Foijer, Uri Ben-David
          </td>
          <td>2025-01-21</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Inotuzumab Ozogamicin (InO) is an antibody-calicheamicin conjugate with striking efficacy in B-cell acute lymphoblastic leukemia (B-ALL). However, there is wide inter-patient variability in treatment response, and the genetic basis of this variation remains largely unknown. Using a genome-wide CRISPR screen, we discovered the loss of DNTT as a primary driver of InO resistance. Mechanistically, DNTT downregulation attenuated InO-induced DNA damage response, cell cycle arrest, and mitochondrial apoptotic priming, ultimately leading to leukemia resistance to InO. Ex vivo leukemia InO sensitivity was highly associated with DNTT expression in ALL blasts, with substantial intra-leukemia heterogeneity as revealed by scRNA sequencing. In B-ALL patients enrolled in the COG trial AALL1621, we observed consistent DNTT downregulation in residual blasts post-InO treatment. The selection of DNTT-low blasts by InO therapy was also recapitulated in vivo using patient-derived xenograft models. Collectively, our data indicate that DNTT is a key regulator of calicheamicin response in leukemia and thus a potential biomarker for individualizing InO therapy in B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d0d43dca72c75e6c736c5bab87e5881541ce7f" target='_blank'>
              DNTT-mediated DNA Damage Response Drives Inotuzumab Ozogamicin Resistance in B-cell Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Carolin S Escherich, Takaya Moriyama, Zhenhua Li, Yu-Chih Hsiao, Wenjian Yang, Yizhen Li, Noemi Reyes, Megan Walker, Amit Budhraja, Sheetal Bhatara, E. Diaz-Flores, Wendy Stock, E. Paietta, M. Konopleva, S. Kornblau, M. Litzow, H. Inaba, Ching-Hon Pui, J. Opferman, M. Loh, Jiyang Yu, Maureen O'Brien, William E Evans, Jun J Yang
          </td>
          <td>2024-12-27</td>
          <td>Blood</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="How nervous systems coordinate the generation of specific neuron types with gene expression plasticity and how these mechanisms impact cell type evolution is unknown. Here we use Caenorhabditis species to study neuron-type robustness, plasticity and evolution, using VC4 and VC5 cholinergic motoneurons as models. In C. elegans, we found that epigenetic silencing through histone 3 lysine 9 methylation (H3K9me) is necessary to suppress the expression of the serotonin reuptake gene mod-5/Sert and a serotonergic phenotype in these cells. In contrast, we observed that VC4 and VC5 neurons in the Angaria group of species of the Caenorhabditis genus have evolved an intense serotonergic staining. This phenotype is caused by the emergence of a new enhancer in the mod-5/Sert locus, which has been recruited to the ancestral neuron-type gene regulatory network. Enhancer transfer from C. angaria is sufficient to impose a constitutive serotonergic fate in C. elegans. Remarkably, acquiring this new trait modulates egg-laying responses to high levels of exogenous serotonin, which can be found in specific environments. Finally, we discovered that the repression of the serotonergic fate in C. elegans VC4 and VC5 neurons is indeed a plastic trait that can be adjusted in specific environmental growth conditions to elicit egg-laying behaviours similar to those observed in Angaria species. Our work identifies gene regulatory mechanisms that coordinate the generation of robust neuron-type-specific programs with plastic gene expression responses. These findings identify a gene regulatory framework underlying the evolution of neuron-type-specific features and the emergence of novel behaviours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2315a11171225e8baca8060e06a35244519fbc76" target='_blank'>
              Gene regulatory mechanisms underlying evolutionary adaptations of homologous neuronal cell types
              </a>
            </td>
          <td>
            Andrea Millán-Trejo, Carlos Mora-Martínez, Adrián Tarazona-Sánchez, Carla Lloret-Fernández, Rafael Alis, Antonio Jordán, A. Barrios, Nuria Flames
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba7e1fd60eab1467a858b79695c0c9e7c85c584" target='_blank'>
              Mapping chromatin remodelling in glioblastoma identifies epigenetic regulation of key molecular pathways and novel druggable targets
              </a>
            </td>
          <td>
            C. Vinel, James Boot, Weiwei Jin, N. Pomella, Alexandra Hadaway, C. Mein, N. Zabet, Silvia Marino
          </td>
          <td>2025-02-07</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Breast cancer is the most frequently diagnosed malignancy and a leading cause of cancer-related mortality among women worldwide. Although NADPH oxidase 4 (NOX4) has been implicated in various oncogenic processes, its exact function in breast cancer progression, metabolic reprogramming, and immune modulation remains unclear.We used murine 4T1 and EO771 breast cancer models to generate NOX4 knockout (KO) cell lines via CRISPR/Cas9. In vitro assays (cell proliferation, colony formation, wound healing, and Seahorse metabolic analyses) and in vivo orthotopic tumor studies assessed the impact of NOX4 loss. Transcriptomic changes were identified through RNA sequencing and gene set enrichment analysis. We performed MYC knockdown in NOX4 KO cells to investigate its mechanistic role. Flow cytometry characterized tumor-infiltrating immune cells. Finally, NOX4-overexpressing cells were tested for survival benefit and response to dual-checkpoint immunotherapy (anti-PD-1/anti-CTLA-4).NOX4 deletion accelerated tumor growth in vivo and enhanced proliferation, colony formation, and migratory capacity in vitro. Metabolic profiling showed that NOX4 KO cells had elevated glycolysis and fatty acid oxidation, along with increased mitochondrial mass. Transcriptomic and enrichment analyses revealed MYC pathway activation in NOX4 KO cells; suppressing MYC reversed these hyperproliferative and metabolic changes. Immunologically, NOX4 KO reduced CD8+ T cell infiltration and function, partially due to lowered CCL11/CCL5 levels, while PD-L1 expression was upregulated. In contrast, NOX4 overexpression improved survival in mice and synergized with checkpoint blockade, demonstrating a positive effect on anti-tumor immunity.These findings show that NOX4 constrains breast cancer aggressiveness by limiting MYC-driven metabolic adaptations and supporting CD8+ T cell-mediated immunity. Loss of NOX4 promotes a more malignant phenotype and dampens T cell responses, whereas its overexpression prolongs survival and enhances checkpoint inhibitor efficacy. Therapeutically targeting the NOX4–MYC axis and leveraging NOX4’s immunomodulatory capacity could offer promising strategies for breast cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9114ae3608de23e0dbc2cbea2c8b3a88bf3557" target='_blank'>
              NOX4 modulates breast cancer progression through cancer cell metabolic reprogramming and CD8+ T cell antitumor activity
              </a>
            </td>
          <td>
            Yingying Xiong, Y. Weng, Shan Zhu, Jian Qin, Jia Feng, Xiaopeng Jing, Chao Luo, Wei Gong, Rui Sun, Min Peng
          </td>
          <td>2025-02-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id=". The development of cancer involves an intri‑ cate mechanism, including oncogene activation and tumor suppressor gene inactivation, often driven by epigenetic altera‑ tions. Notably, these epigenetic modifications, such as DNA hypermethylation, are dynamic and reversible, and emerging research suggests that dietary factors can influence them. Recent findings have examined the intricate associations between dietary compounds and cancer development, with a particular focus on the mechanisms through which natural components affect epigenetic alterations. Natural compounds have gained significant attention for their potential in chemo‑ prevention and treatment due to their capacity to modulate DNA methylation patterns, particularly at promoter CpG islands. Chemotherapy and radiation therapy are effective cancer treatments and are often associated with severe adverse effects. As a result, there is increasing interest in exploring natural substances as alternative therapeutic options. The present review focuses on natural compounds that can reverse hypermethylation, providing a potentially safer approach for targeted cancer treatment. These bioactive agents, abundant in natural compounds, exhibit the potential to combat cancer by inhibiting metastasis, inducing cell cycle arrest and reversing DNA hypermethylation. The present review aimed to provide further in‑depth mechanistic insight into the mechanisms through which natural compounds, such as genistein, resvera‑ trol, quercetin and capsaicin modulate DNA hypermethylation in various types of cancer, including breast, cervical, prostate and neuroblastoma. By combining in silico , in vitro and in vivo approaches, it uniquely integrates computational and experimental evidence to demonstrate compound‑specific reversal of epigenetic marks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eaa3ae9852c9314b773d0fad7daf6e3a6798eb1" target='_blank'>
              Natural compounds as epigenetic modulators: Reversing DNA hypermethylation in cancer (Review)
              </a>
            </td>
          <td>
            Farheen Farheen, Kantimahanti Shreya, R. Mathur, A. Jha
          </td>
          <td>2024-12-30</td>
          <td>World Academy of Sciences Journal</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Tumor-specific antigens, also known as neoantigens, have potential utility in anti-cancer immunotherapy, including immune checkpoint blockade (ICB), neoantigen-specific T cell receptor-engineered T (TCR-T), chimeric antigen receptor T (CAR-T), and therapeutic cancer vaccines (TCVs). After recognizing presented neoantigens, the immune system becomes activated and triggers the death of tumor cells. Neoantigens may be derived from multiple origins, including somatic mutations (single nucleotide variants, insertion/deletions, and gene fusions), circular RNAs, alternative splicing, RNA editing, and polymorphic microbiome. An increasing amount of bioinformatics tools and algorithms are being developed to predict tumor neoantigens derived from different sources, which may require inputs from different multi-omics data. In addition, calculating the peptide-major histocompatibility complex (MHC) affinity can aid in selecting putative neoantigens, as high binding affinities facilitate antigen presentation. Based on these approaches and previous experiments, many resources were developed to reveal the landscape of tumor neoantigens across multiple cancer types. Herein, we summarized these tools, algorithms, and resources to provide an overview of computational analysis for neoantigen discovery and prioritization, as well as the future development of potential clinical utilities in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676b6cef04b3606a814455c2e9548b49b282ba61" target='_blank'>
              Characterization of Tumor Antigens from Multi-omics Data: Computational Approaches and Resources.
              </a>
            </td>
          <td>
            Yunzhe Wang, James Wengler, Yuzhu Fang, Joseph Zhou, Hang Ruan, Zhao Zhang, Leng Han
          </td>
          <td>2025-01-20</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Environmental factors are linked to aging and age-related diseases. Emerging evidence suggests that enhancing body's resistance to xenobiotics might be an anti-aging strategy. The constitutive androstane receptor (CAR) regulates drug-metabolizing enzymes and transporters, coordinating metabolism and immune responses to adapt to stress triggered by exogenous exposure. However, the impact of activating CAR on aging remains unknown. In this study, Caenorhabditis elegans (C. elegans), drug-induced premature aging mice, and senescence accelerated P8 (SAMP8) mice are used as models to explore the effects of CAR activation on lifespan and healthspan, along with the underlying mechanisms. The results showed that hCAR agonist CITCO and mCAR agonist TCPOBOP prolonged the lifespan and healthspan in model organism. The longevity effects of CITCO and TCPOBOP were attenuated in CAR homozygous nhr-8/daf-12 mutant C. elegans as well as CAR-/- mice. In C. elegans, CITCO activated both anti-stress and detoxification genes, and increased the resistance to environmental adversities. Additionally, the lifespan-extending and xenobiotic resistant effects of CITCO might be related to the regulation of age-related pathways. Furthermore, CITCO improved age-related neurodegeneration in C. elegans models. Taken together, the results suggest that the longevity effects of CAR agonists may be related to the enhancement of xenobiotic resistance of animals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960efc318b7833116a8e79bd8aa34344f0dc8fbe" target='_blank'>
              Activation of Nuclear Receptor CAR: A Pathway to Delay Aging through Enhanced Capacity for Xenobiotic Resistance.
              </a>
            </td>
          <td>
            Jing Yu, Xiaoyan Gao, Hang Shi, Li-jun Zhang, Wenlong Nie, Ruochen Zhang, Ming Fang, Ying Liu, Yingxuan Yan, Bingbing Fan, Chengyuan Wu, Cheng Huang, Shengjie Fan
          </td>
          <td>2025-01-31</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e4a1ddd31bb3520afde406f9d3fbb676ebddb0" target='_blank'>
              Bruton’s tyrosine kinase inhibition re-sensitizes multidrug-resistant DLBCL tumors driven by BCL10 gain-of-function mutants to venetoclax
              </a>
            </td>
          <td>
            C. Coughlin, Dhanvantri Chahar, M. Lekakis, Abdessamad A Youssfi, Lingxiao Li, Evan R. Roberts, N. C. Gallego, C. Volmar, O. Landgren, S. Brothers, Anthony J Griswold, Catalina Amador, Daniel Bilbao, F. Maura, Jonathan H. Schatz
          </td>
          <td>2025-02-02</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad33c5772a7c6f391f659ca36bbfaca426706c80" target='_blank'>
              Improving the odds of survival: transgenerational effects of infections.
              </a>
            </td>
          <td>
            V. Spanou, Theano Andriopoulou, E. Giamarellos‐Bourboulis, Mihai G Netea
          </td>
          <td>2025-01-22</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc6572745a00744a0ffe9484307c354e0370541" target='_blank'>
              Short tandem repeats delineate gene bodies across eukaryotes
              </a>
            </td>
          <td>
            William B. Reinar, A. Krabberød, V. O. Lalun, M. Butenko, K.S. Jakobsen
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="RAD18 is a conserved eukaryotic E3 ubiquitin ligase that promotes genome stability through multiple pathways. One of these is gap-filling DNA synthesis at active replication forks and in post-replicative DNA. RAD18 also regulates homologous recombination (HR) repair of DNA breaks; however, the current literature describing the contribution of RAD18 to HR in mammalian systems has not reached a consensus. To investigate this, we examined three independent RAD18-null human cell lines. Our analyses found that loss of RAD18 in HCT116, but neither hTERT RPE-1 nor DLD1 cell lines, resulted in elevated sister chromatid exchange, gene conversion, and gene targeting, i.e., HCT116 mutants were hyper-recombinogenic (hyper-rec). Interestingly, these phenotypes were linked to RAD18’s role in PCNA K164 ubiquitination, as HCT116 PCNAK164R/+ mutants were also hyper-rec, consistent with previous studies in rad18−/− and pcnaK164R avian DT40 cells. Importantly, the knockdown of UBC9 to prevent PCNA K164 SUMOylation did not affect hyper-recombination, strengthening the link between increased recombination and RAD18-catalyzed PCNA K164 ubiquitination, but not K164 SUMOylation. We propose that the hierarchy of post-replicative repair and HR, intrinsic to each cell type, dictates whether RAD18 is required for suppression of hyper-recombination and that this function is linked to PCNA K164 ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4d531e6f454e305c916fc9c53a0c286d620e" target='_blank'>
              Cell Type Specific Suppression of Hyper-Recombination by Human RAD18 Is Linked to Proliferating Cell Nuclear Antigen K164 Ubiquitination
              </a>
            </td>
          <td>
            Colette B. Rogers, W. Leung, Ryan M. Baxley, Rachel E. Kram, Liangjun Wang, Joseph P. Buytendorp, Khoi Le, D. Largaespada, Eric A. Hendrickson, A. Bielinsky
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b91b716a484954fac596a012c478c6ddaee7a8e" target='_blank'>
              HMGA2 overexpression with specific chromosomal abnormalities predominate in CALR and ASXL1 mutated myelofibrosis.
              </a>
            </td>
          <td>
            Shivani Handa, Christoph Schaniel, J. Tripodi, Daiva Ahire, Md Babu Mia, Sophie Klingborg, D. Tremblay, Bridget K Marcellino, R. Hoffman, V. Najfeld
          </td>
          <td>2024-12-23</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Variant histone H3.3 is thought to be critical for survival of many cells, since it is deposited in expressed genes, a feature different from core histones. For example, H3.3 deletion leads to embryonic lethality in mice. However, requirement of H3.3 in later stage of development has remained unclear. The aim of this work was to elucidate the role of H3.3 for development of myeloid lineage, important for innate immunity. We conditionally knocked out (cKO) the H3.3 genes in myeloid progenitor cells differentiating into bone marrow derived macrophages (BMDMs). Progenitor cells lacking H3.3 were defective in replication, suffered from extensive DNA damage, and underwent apoptosis. Surviving H3.3cKO cells expressed many interferon stimulated genes (ISGs) throughout differentiation. Further, H3.3cKO BMDMs possessed chromatin accessible sites, and histone posttranslational modifications consistent with the gene expression profiles, Accordingly, H3.3cKO BMDMs retained general nucleosomal structure genome wide. In summary, H3.3 is required for proliferation of myeloid progenitor cells, but is in large part dispensable for differentiation of BMDMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee58f7554d51c36d858c992137d0b639e970461a" target='_blank'>
              Histone H3.3 ensures cell proliferation and genomic stability during myeloid cell development
              </a>
            </td>
          <td>
            Sakshi Chauhan, A. Dey, Todd S Macfarlan, Keiko Ozato
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f316b25c9fd66656561befcef17f13bf1d2230" target='_blank'>
              Mcm5 mutation leads to silencing of Stat1-bcl2 which accelerating apoptosis of immature T lymphocytes with DNA damage
              </a>
            </td>
          <td>
            Min Liu, Yuanyuan Li, Zhilin Deng, Ke Zhang, Shuying Huang, Jiamin Xia, Yi Feng, Yundan Liang, Chengfu Sun, Xindong Liu, Shurong Li, B. Su, Yong Dong, Sizhou Huang
          </td>
          <td>2025-02-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="ABSTRACT Background CREB binding protein (CREBBP) is a key epigenetic regulator, altered in a fifth of relapsed cases of acute lymphoblastic leukemia (ALL). Selectively targeting epigenetic signaling may be an effective novel therapeutic approach to overcome drug resistance. Anti‐tumor effects have previously been demonstrated for GSK‐J4, a selective H3K27 histone demethylase inhibitor, in several animal models of cancers. Methods To characterize the effect of GSK‐J4, drug response profiling, CRISPR‐Dropout Screening, BH3 profiling and immunoblotting were carried out in ALL cell lines or patient derived samples. Results Here we provide evidence that GSK‐J4 downregulates cyclic AMP‐responsive element‐binding protein (CREB) and CREBBP in B‐cell precursor‐ALL cell lines and patient samples. High CREBBP expression in BCP‐ALL cell lines correlated with high GSK‐J4 sensitivity and low dexamethasone sensitivity. GSK‐J4 treatment also induced Bcl‐2 and Bcl‐XL dependency and apoptosis. Conclusions This study proposes H3K27 demethylase inhibition as a potential treatment strategy for patients with treatment‐resistant ALL, using CREBBP as a biomarker for drug response and combining GSK‐J4 with venetoclax and navitoclax as synergistic partners.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6268afa1f23d7dedcd6b2c8390d3c0a47ae3fe38" target='_blank'>
              Inhibiting H3K27 Demethylases Downregulates CREB‐CREBBP, Overcoming Resistance in Relapsed Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Juan Lazaro-Navarro, C. Alcon, M. Dorel, Lina Alasfar, L. Bastian, Claudia Baldus, K. Astrahantseff, M. Yaspo, J. Montero, Cornelia Eckert
          </td>
          <td>2025-01-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. The translocation t(1;19), encoding the TCF3‐PBX1 fusion, is associated with intermediate risk and central nervous system (CNS) infiltration at relapse. Using our previously generated TCF3‐PBX1 conditional knock‐in mice, we established a model to study relapsed clones after in vivo chemotherapy treatment, CNS infiltration, and clonal dynamic evolution of phenotypic diversity at the single cell‐level using next‐generation sequencing technologies and mass cytometry. Mice transplanted with TCF3‐PBX1 + leukemia cells and treated with vehicle succumbed to disease, whereas 40% of treated mice with prednisolone or daunorubicin survived. Bulk and single‐cell RNA sequencing of FACS‐sorted GFP+ cells from TCF3‐PBX1 + leukemias arising after chemotherapy treatment revealed that apoptosis, interleukin‐, and TGFβ‐signaling pathways were regulated in CNS‐infiltrating leukemic cells. Across tissues, upregulation of the MYC signaling pathway was detected in persisting leukemic cells and its downregulation by BRD3/4 inhibition increased sensitivity to chemotherapy. In TCF3‐PBX1+ leukemia cells collected after chemotherapy treatment, mass cytometry identified increased phosphorylation of STAT3/5 upon preBCR stimulation, which was susceptible to inhibition by the proteasome inhibitor bortezomib. In summary, we developed a TCF3‐PBX1+ ALL mouse model and characterized relapsed disease after in vivo chemotherapy and cell phenotype dependence on microenvironment. Transcriptomics and phospho‐proteomics revealed distinct pathways that may underlie chemotherapy resistance and might be suitable for pharmacological interventions in human ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278da44c640128811ffe9cd7b7d31669827014f" target='_blank'>
              Dynamic evolution of TCF3‐PBX1 leukemias at the single‐cell level under chemotherapy pressure
              </a>
            </td>
          <td>
            Mira Kusterer, M. Lahnalampi, M. Voutilainen, Alexandra Brand, S. Pennisi, J. Norona, Gaia Gentile, Heike Herzog, G. Greve, Michael Lübbert, M. Sipola, Emma Kaartinen, R. Sankowski, Marco Prinz, S. Killmer, M. S. Lago, Bertram Bengsch, Stepan Cysar, K. Aumann, Martin Werner, J. Duyster, O. Lohi, M. Heinäniemi, J. Duque-Afonso
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Background Immune response variation among sepsis patients leads to substantial clinical heterogeneity that makes treatment challenging. Bulk RNA-sequencing of circulating immune cells has been used in an attempt to resolve sepsis heterogeneity, and has facilitated classification of patients into different subtypes, or “endotypes”. The Sepsis Response Signature (SRS) classification comprises 3 endotypes: SRS1 represents an immunosuppressed transcriptional profile with high mortality; SRS2 a relatively immunocompetent response with low mortality, and SRS3 is a profile close to a normal immune response. However, because bulk RNA-seq does not distinguish the transcriptional contributions of different cell types, mechanistic insights from these classifications are lacking, thus limiting clinical utility. Single-cell RNA-sequencing (scRNA-seq) enables cell-specific transcriptomics and has the potential to demonstrate the cellular basis of sepsis endotypes. In this study, we performed scRNA-seq on blood from sepsis patients to investigate the differences in immune cell transcriptional states among SRS endotypes.Figure 1. Fractional abundance of monocyte substate 1 (MS1) is highest in the Sepsis Response Signature 1 (SRS1) endotype Fraction of MS1 cells relative to all peripheral blood mononuclear cells (PBMCs) in SRS1 (n= 10), SRS2 (n= 33), and SRS3 (n= 11) patients. Boxes show the median and inter-quartile range (IQR) for each endotype, with whiskers extending to 1.5x the IQR in either direction from the top or bottom quartile. Two-sided Wilcoxon rank-sum test with Benjamini-Hochberg correction used for comparisons; ***p<0.001, ****p<0.0001. Methods Mononuclear immune cells were isolated from the blood of sepsis patients (n=54) at hospital presentation and the transcriptomes of ∼1500 single cells per sample were analyzed. By aggregating gene counts for each patient, we generated pseudobulked transcriptional profiles to approximate whole blood gene expression. We then leveraged the SepstratifieR machine learning framework to stratify patients into SRS endotypes and compared the proportions of immune cell transcriptional states across the endotypes. Results A monocyte transcriptional state – monocyte substate 1 (MS1), was differentially expanded across SRS endotypes (Fig.1). MS1 abundance was significantly higher for patients in the SRS1 endotype compared to SRS2 (p< 0.001) or SRS3 (p< 0.0001). Conclusion Identifying cellular signals underlying endotypes may improve understanding of sepsis immunobiology and open paths to precision therapy. This study demonstrates the expansion of a monocyte transcriptional state in the SRS1 endotype. MS1 cells inhibit T cell activation and proliferation, suggesting a potential mechanistic explanation for the immunosuppressive state of SRS1 patients. Disclosures Michael R. Filbin, MD, Day Zero Diagnostics: Grant/Research Support|Quidel: Grant/Research Support">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bda1ab33c7fae8e980cd6b50f6a865d00d04a191" target='_blank'>
              P-2245. Single cell RNA-sequencing of immune cells demonstrates differential expansion of a monocyte transcriptional state among the Sepsis Response Signature (SRS) endotypes
              </a>
            </td>
          <td>
            P. Ankomah, Alyssa DuBois, Michael R. Filbin, N. Hacohen, R. Bhattacharyya
          </td>
          <td>2025-01-29</td>
          <td>Open Forum Infectious Diseases</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501a2344c548bcfc786460170b1feb2e0d711776" target='_blank'>
              Sperm-borne small non-coding RNAs: potential functions and mechanisms as epigenetic carriers
              </a>
            </td>
          <td>
            Muhammad Naveed, Zhaokang Shen, Jianqiang Bao
          </td>
          <td>2025-01-17</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4211b81d0b19e4c7f2e0cfa7c734b541dc3fa943" target='_blank'>
              Epigenetics in the modern era of crop improvements.
              </a>
            </td>
          <td>
            Yan Xue, Xiaofeng Cao, Xiangsong Chen, Xian Deng, Xing Wang Deng, Yong Ding, Aiwu Dong, Cheng-Guo Duan, Xiaofeng Fang, L. Gong, Zhizhong Gong, Xiaofeng Gu, Chongsheng He, Hang He, Shengbo He, Xin-Jian He, Yan He, Yuehui He, Guifang Jia, Danhua Jiang, Jianjun Jiang, Jinsheng Lai, Zhaobo Lang, Chenlong Li, Qing Li, Xingwang Li, Bao Liu, Bing Liu, Xiao Luo, Yijun Qi, Weiqiang Qian, Guodong Ren, Qingxin Song, Xianwei Song, Zhixi Tian, Jia-Wei Wang, Yuan Wang, Liang Wu, Zhe Wu, Rui Xia, Jun Xiao, Lin Xu, Zhengyi Xu, Wenhao Yan, Hongchun Yang, J. Zhai, Yijing Zhang, Yusheng Zhao, Xuehua Zhong, Dao-Xiu Zhou, Ming Zhou, Yue Zhou, Bo Zhu, Jian-Kang Zhu, Qikun Liu
          </td>
          <td>2025-01-08</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Approximately 97% of the human genome comprises non-coding sequences, with nearly half originating from transposable elements. Among these, retrotransposons represent a critical subclass that replicates via a "copy-and-paste" mechanism and significantly influences the regulation of host genomes. In both normal and pathological contexts, retrotransposons contribute a vast reservoir of regulatory elements that can modulate the expression of genes. If left unchecked, retrotransposons can substantially affect host transcriptional programs and genomic integrity. Therefore, various mechanisms, including epigenetic modifications, are employed to mitigate their potentially deleterious effects. In diseases such as cancers, the epigenome is often significantly reprogrammed, which can lead to retrotransposon dysregulation. Drawing insights from recent studies conducted in human and murine cells, this review examines how retrotransposons expand the complexity of mammalian genomes, describes the impact of their epigenetic dysregulation in cancer development, and highlights the potential of targeting these sequences for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) refer to a group of RNA molecules that exceed a length of 200 nucleotides and lack the ability to code proteins. Numerous studies suggest that lncRNAs significantly contribute to the onset and progression of various forms of cancers. A specific lncRNA, known as long non-coding RNA 01296 (LINC01296), is extensively expressed in human malignancies. The level of LINC01296 has been shown to correlate with the progression and prognosis of cancers. Moreover, numerous scientific investigations have provided evidence that the dysregulation of LINC01296 functioning as a competitive endogenous RNA (ceRNA) exerts a profound influence on various aspects of cancer cell behavior, including proliferation, apoptosis, invasion, metastasis, and cell cycle progression, by means of regulating target genes and signaling pathways. An increasing body of data strongly suggests that LINC01296 may serve as a valuable biomarker for predicting cancer prognosis and could represent a promising therapeutic target for cancer intervention. In this comprehensive review, we summarize the recent advancements in our understanding of the role, underlying mechanisms, and clinical significance of LINC01296 in malignant tumors. The findings suggest that LINC01296 may be both a reliable biomarker and a potential therapeutic target for cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcfc7646d19e3cb4d25d1197357b2a9f9017093" target='_blank'>
              The Emerging Role of Long Non-Coding RNA 01296 in Human Malignancies.
              </a>
            </td>
          <td>
            Lei Luo, Fan Yang, Xiaoping Fu, Tingting Yu, Wenqian Tang, Juan Xue
          </td>
          <td>2025-01-31</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10a2ab1ac270b68e0530d0e78e9d779c180238b" target='_blank'>
              Histone Modifications and DNA Methylation in Psoriasis: A Cellular Perspective.
              </a>
            </td>
          <td>
            Jing Pan, Siji Chen, Xianzhen Chen, Yinjing Song, Hao Cheng
          </td>
          <td>2025-01-27</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The role of somatic mitochondrial DNA (mtDNA) mutations in leukemogenesis remains poorly characterized. To determine the impact of somatic mtDNA mutations on this process, we assessed the leukemogenic potential of hematopoietic progenitor cells (HPCs) from mtDNA mutator mice (Polg D257A) with or without NMyc overexpression. We observed a higher incidence of spontaneous leukemogenesis in recipients transplanted with heterozygous Polg HPCs and a lower incidence of NMyc-driven leukemia in those with homozygous Polg HPCs compared to controls. Although mtDNA mutations in heterozygous and homozygous HPCs caused similar baseline impairments in mitochondrial function, only heterozygous HPCs responded to and supported altered metabolic demands associated with NMyc overexpression. Homozygous HPCs showed altered glucose utilization with pyruvate dehydrogenase inhibition due to increased phosphorylation, exacerbated by NMyc overexpression. The impaired growth of NMyc-expressing homozygous HPCs was partially rescued by inhibiting pyruvate dehydrogenase kinase, highlighting a relationship between mtDNA mutation burden and metabolic plasticity in leukemogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a692877215b062ddb45f64e78542319dbb82acdd" target='_blank'>
              Somatic mtDNA mutation burden shapes metabolic plasticity in leukemogenesis
              </a>
            </td>
          <td>
            Xiujie Li-Harms, Jingjun Lu, Yu Fukuda, John Lynch, Aditya Sheth, Gautam Pareek, Marcin M Kamiński, Hailey S Ross, Christopher W. Wright, Amber L Smith, Huiyun Wu, Yong-Dong Wang, Marc Valentine, Geoffrey Neale, Peter Vogel, S. Pounds, J. Schuetz, Min Ni, M. Kundu
          </td>
          <td>2025-01-01</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Glioblastoma (GBM) is described as a group of highly malignant primary brain tumors and stands as one of the most lethal malignancies. The genetic and cellular characteristics of GBM have been a focal point of ongoing research, revealing that it is a group of heterogeneous diseases with variations in RNA expression, DNA methylation, or cellular composition. Despite the wealth of molecular data available, the lack of transferable pre-clinic models has limited the application of this information to disease classification rather than treatment stratification. Transferring the patients' genetic information into clinical benefits and bridging the gap between detailed descriptions of GBM, genotype-phenotype associations, and treatment advancements remain significant challenges. In this context, we present an advanced human GBM organoid model, the Laboratory Engineered Glioblastoma Organoid (LEGO), and illustrate its use in studying the genotype-phenotype dependencies and screening potential drugs for GBM. Utilizing this model, we have identified lipid metabolism dysregulation as a critical milestone in GBM progression and discovered that the microsomal triglyceride transfer protein inhibitor Lomitapide shows promise as a potential treatment for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3c5689a723411e5e090a7db347716f4db71706a" target='_blank'>
              Laboratory-Engineered Glioblastoma Organoid Culture and Drug Screening.
              </a>
            </td>
          <td>
            Changwen Wang, Nadja Stöffler, Haikun Liu
          </td>
          <td>2025-01-10</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mutations drive evolution and genetic diversity, but the impact of transcription on germline mutagenesis remains poorly understood. Here, we identify a hypermutation phenomenon at transcription start sites in the human germline, spanning several hundred base pairs in both directions. We link this TSS mutational hotspot to divergent transcription, RNA polymerase II stalling, R-loops, and mitotic—but not meiotic—double-strand breaks, revealing a recombination-independent mechanism distinct from known processes. Notably, the hotspot is absent in de novo mutation data. We reconcile this by showing that TSS mutations are significantly enriched with early mosaic variants often filtered out in de novo mutation calls, indicating that the hotspot arises during early embryogenesis. Mutational signature analysis reinforces these findings and implicates alternative non-homologous end joining and maternal mutation clusters. Our study provides the first detailed description of a germline TSS mutation hotspot, with broad evolutionary and biomedical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3da1619adeabbe4ded6c11c45076e2a69d9a307" target='_blank'>
              Transcription start sites experience a high influx of heritable variants fuelled by early development
              </a>
            </td>
          <td>
            Miguel Cortés Guzmán, David Castellano, Claudia Serrano Colome, V. Seplyarskiy, Donate Weghorn
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The heterogeneity of the hematopoietic system was largely veiled by traditional bulk sequencing methods, which measure the averaged signals from mixed cellular populations. In contrast, single-cell sequencing has enabled the direct measurement of individual signals from each cell, significantly enhancing our ability to unveil such heterogeneity. Building on these advances, numerous single-cell multi-omics techniques have been developed into high-throughput, routinely accessible platforms, delineating the precise relationships among the different layers of the central dogma in molecular biology. These technologies have uncovered the intricate landscape of genetic clonality and transcriptional heterogeneity in both normal and malignant hematopoietic systems, highlighting their roles in differentiation, disease progression, and therapy resistance. This review aims to provide a brief overview of the principles of single-cell technologies, their historical development, and a subset of ever-expanding multi-omics tools, emphasizing the specific research questions that inspired their creation. Amidst the evolving landscape of single-cell multi-omics technologies, our main objective is to guide investigators in selecting the most suitable platforms for their research needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a517176c60f9a80290d19966f93c06f80af4cac7" target='_blank'>
              Emerging technologies of single-cell multi-omics.
              </a>
            </td>
          <td>
            Yi June Kim, Koichi Takahashi
          </td>
          <td>2025-02-06</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cellular transformation is initiated by a driver mutation, resulting in the abnormal expression of an oncogene or the loss of a tumor suppressor gene in a normal cell. This event leads to oncogenic stress (OS), describing a disruption of cellular homeostasis which activates stress response pathways and transcriptomic programs that can either promote tumor development or trigger intrinsic safeguards such as senescence or apoptosis. While the immunogenicity of senescent cells is well established, the immunogenic potential of cells at the early OS stage and the mechanisms by which they are sensed by the immune system remain unknown. We developed an original in vitro OS model using primary human mammary epithelial cells to investigate this process. Our results show that OS cells display an immunoactive secretome associated with OS (SOS), as well as immunogenic membrane ligands. Intrinsically, OS cells activate the PERK and IRE1 branches of the unfolded protein response (UPR) that regulate their immunogenic features. Extrinsically, we demonstrated that neutrophils can detect OS-induced immunogenic signals, leading to their recruitment and their activation in co-culture in vitro. We validated this observation in two spontaneous murine models of breast cancer, where the earliest preneoplastic stages exhibit both UPR activation in epithelial cells and neutrophil infiltration. Altogether, our work identifies OS as the first immunogenic event in carcinogenesis and neutrophils as the first immune cell involved in immunosurveillance, paving the way for potential therapeutic approaches intercepting early steps of tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8681caea4fe589945dd4b9b0bfc53ac917eba1" target='_blank'>
              Oncogenic Stress is a Novel Immunogenic Signal Driven by the Unfolded Protein Response and Detected by Neutrophils
              </a>
            </td>
          <td>
            Antoine Debiesse, Lyvia Moudombi, Aurélien Voissière, Leïla Lefèvre, M. Malfroy, Iryna Moskalevska, Manuela Pereira Abrantes, Hélène Vanacker, Margaux Hubert, Roxane Pommier, Cassandra Assaf, Emmanuelle Jacquet, Marie Laurent, Sara Belluco, N. Goutagny, Virginie Petrilli, Julien Cherfils-Vicini, Christophe Caux, N. Bendriss-Vermare, M. Michallet
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="We report the engineering of lipid nanoparticles (LNPs) to transport CRISPR/Cas9 payloads, including double-stranded DNA (dsDNA) donor templates, designed for homology directed repair (HDR)-mediated site-specific insertion of the cystic fibrosis transmembrane conductance regulator (CFTR) gene to correct cystic fibrosis (CF) in diseased airway epithelium. We screened various nanoparticle formulations, adjusting ratios of Cas9-encoding mRNA, single guide RNAs (sgRNAs), and dsDNA donor templates to optimize gene editing using human bronchial epithelial cells (16HBE14o-) harboring a CF-causing mutation (G542X). Populations of G542X cells edited via LNP delivery of CFTR donors achieved 3 – 3.5% gene integration and yielded comparable CFTR protein expression compared to normal 16HBE14o- controls. These edited populations exhibit restoration of CFTR-dependent Cl- current to ca. 80% of values measured in normal 16HBE14o- cell monolayers. This LNP platform adds capabilities for transporting large gene editing machinery to airway epithelial cells for genomic integration of entire genes, enabling therapeutic solutions that achieve correction of any CF-causing mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0097561835a41509ae48e38631c6aa471820400c" target='_blank'>
              Lipid Nanoparticles for the Delivery of CRISPR/Cas9 Machinery to Enable Site-Specific Integration of CFTR and Mutation-Agnostic Disease Rescue
              </a>
            </td>
          <td>
            Ruth A. Foley, P. Ayoub, Vrishti Sinha, C. Juett, A. Sanoyca, Emily C. Duggan, Lindsay Lathrop, Priyanka Bhatt, Kevin Coote, Beate Illek, B. Gomperts, Donald B. Kohn, S. Jonas
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumorigenesis and progression are complex process with multifarious molecules and related mechanisms, while most of them are still elusive. Most research into the mechanisms of cancer today are focusing on encoding RNAs. However, for the past few years as the emergence and development of epigenetics, a kind of new molecule: non-coding RNAs (ncRNAs), RNAs without protein-coding function, have gained prominence in recent scientific discussions. Since the first ncRNA was discovered in 1990s, extensive research has shown that ncRNAs significantly influence processes such as cell growth, differentiation, metabolic regulation, and programmed cell death at both transcriptional and post-transcriptional stages. By means of acting as the tumor suppression and oncogenesis, research has identified several ncRNAs with abnormal expression patterns in cancer cells, marking them as primary oncogenic factors or valuable targets in cancer treatment. This review focus on three kinds of ncRNA, MicroRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs), the molecule structure, generative mechanism, characteristic function and regulatory mechanism in tumor progression. In addition, we will discuss the application value and future expiration of the therapeutic that are focusing on ncRNA, that will open up a new idea and methods for the treatment of cancer for future investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026f448482a66bffaadbd5d2efa692a3ff825932" target='_blank'>
              Non-coding RNAs in Tumor Biology: Exploring miRNAs, lncRNAs, and circRNAs Roles and Therapeutic Potentials
              </a>
            </td>
          <td>
            Tianhao Zhang
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10399c5309e81eebce7e85588dd100b798fd5fa0" target='_blank'>
              Preclinical assessment of splicing modulation therapy for ABCA4 variant c.768G>T in Stargardt disease
              </a>
            </td>
          <td>
            D. Karjosukarso, Femke Bukkems, Lonneke Duijkers, Tomasz Z Tomkiewicz, Julia Kiefmann, Andrei Sarlea, Sander Bervoets, Irene Vázquez-Domínguez, L. Molday, Robert S. Molday, Mihai G Netea, C. Hoyng, A. Garanto, R. W. Collin
          </td>
          <td>2025-01-21</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="
 Epigenetic therapies that induce global DNA hypomethylation activate transcriptionally repressed transposable elements (TEs). The activation of a class of TEs known as human endogenous retroviruses (HERVs) can trigger an innate antiviral response in cells called “viral mimicry”. Viral mimicry induced by aberrantly expressed HERVs stimulates interferon signaling genes resulting in an adaptive immune response that can ultimately lead to immunogenic cell death. This heightened immune activity can enhance the efficacy of immune checkpoint inhibitor therapies. Identifying cancers that are susceptible to a viral mimicry response or treatment regimens that induce aberrant transcription of TEs has not yet been fully explored. Recent computational tools enable the measurement of TE signal from short-read RNA-seq data and several large-scale reprocessing pipelines have uniformly processed RNA-seq data of cancer cell lines treated with epigenetic and cytotoxic drugs. However, these large-scale pipelines omit the quantification of TEs. To address this gap and fully utilize publicly available data, we have developed a computational pipeline and database of TE expression for more than 7,000 samples from 220 cancer cell lines treated with over 700 cytotoxic and epigenetic drugs, yielding over 2,100 differential expression experiments. To illustrate the usefulness of this database, we ranked drugs by %TE activation and found that the top three targets were DNMT (Decitabine), CDK9 (MC180295), and Topoisomerase II (Doxorubicin). We enable the exploration of this dataset by implementing several meta-analysis methods and visualizations made available as an interactive web application. This database and web application can serve as a valuable resource for the cancer and epigenetics community by enabling hypothesis generation and providing insights into how specific drugs may induce TE expression.
 Citation Format: Gennaro H Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre Issa. TEDEdb: Transposable element differential expression database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="This research sought to explore the role of epigenetic changes in cancer initiation and progression, the therapeutic potential of epigenetic modulators, i.e., DNMT inhibitors and HDAC inhibitors, and the expression levels of non-coding RNAs, i.e., miRNAs, in controlling epigenetic changes in cancer cells. The research used a quantitative method, i.e., structured questionnaires and regression analysis to determine the role of epigenetic changes in cancer biology. The research validated that 65% of the respondents were familiar with DNA methylation, 60% with histone modification, and 55% with gene-editing technologies like CRISPR. The Chi-Square analysis validated the presence of significant disparity in awareness among demographic groups, with p-values of 0.02 and 0.05 for awareness of DNA methylation by age and awareness of histone modification by medical specialty, respectively. Regression analysis validated that DNMT inhibitors significantly caused cancer cell apoptosis (β = 0.55, p = 0.0002) and suppressed cancer cell migration (β = -0.30, p = 0.02), while HDAC inhibitors also caused apoptosis (β = 0.47, p = 0.005) and promoted migration inhibition (β = -0.25, p = 0.03). The Chi-Square test of expression of miRNAs further validated significant correlations between miR-21 expression and cancer type (χ² = 10.4, p = 0.02) and between miR-34a expression and DNA methylation (χ² = 12.1, p = 0.01). These results suggest the potential of epigenetic therapies and miRNA-based therapies in controlling cancer cell behavior and enhancing therapeutic efficacy. This study highlights epigenetic modifications in cancer, the potential of DNMT and HDAC inhibitors, and miRNAs, urging further research on optimization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/869a71d30138952ce61567951ce4b3e25c0e546d" target='_blank'>
              The Role of Epigenetic Modifications in Cancer Development and Progression: Potential Therapeutic Approaches
              </a>
            </td>
          <td>
            Imtiaz Ali Soomro, Quratulain Badar, Wajid Ali, Adil khan, Syed Rizvan Ali, Waqas Mahmood
          </td>
          <td>2024-12-31</td>
          <td>Indus Journal of Bioscience Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Aging is a process of systemic deterioration and the most significant risk factor for cancers. Clonal hematopoiesis (CH) commonly occurs with aging and links to higher mortality, leukemia risk, and cardiovascular diseases. Age-related CH involves the abnormal clonal expansion of hematopoietic stem cells (HSCs) bearing somatic mutations in genes frequently mutated in leukemia, including genes encoding epigenetic regulators, e.g., the DNA demethylase TET2. While such mutations are known to alter the HSC epigenome, the mechanisms through which these mutations drive HSC self-renewal and myeloid transformation preferentially in old age remain elusive. In this study, we hypothesize that aging and CH mutations cooperatively reshape the HSC transcriptomic landscape, enhancing HSC competitive advantage that facilitates clonal expansion. Using single-cell multi-omic analyses and flow cytometry, we demonstrated, for the first time, that Tet2 deficiency mitigates detrimental HSC aging processes at the transcriptomic, epigenomic, and cellular levels, thereby preserving HSC function and promoting age-dependent expansion. Age-related gene expression changes, including increased Eya4, Lars2, and Mical2 and decreased Cdk6, CD53, and Sox4, were mitigated in Tet2-deficient HSCs. These gene expression changes were confirmed by single-nucleus ATAC-seq. Chromatin accessibility analysis also revealed Tet2 deficiency predominantly mitigates epigenetic changes associated with HSC aging. Specifically, age-related increases in chromatin accessibility of binding motifs of transcription factors (TFs), such as E2f4, Ppard, Mycn, and Max, and decreases in binding motifs of chromatin accessibility for Gabpa, Runx3, Etv4, and Elf1, were also mitigated in Tet2-deficient HSCs. Furthermore, DNA methylation status of these TF binding motifs is linked to the regulation for the targets’ gene expression, and CTCF insulator binding during HSC aging are also mitigated by Tet2 deficiency. Together, Tet2 deficiency stabilizes TF-target gene pairs critical for maintaining HSC self-renewal during aging. In conclusion, our findings reveal that Tet2 deficiency significantly contributes to the mitigation of HSC aging and hijacks the HSC expansion strategy during HSC aging via epigenetic reprogramming, which contributes to age-related clonal hematopoiesis. By elucidating the transcriptomic and epigenomic alterations in Tet2-deficient HSCs, our study provides novel insights into how age and somatic mutations interact to promote the pathogenesis of age-related hematological diseases. These discoveries enhance our understanding of how aging and somatic mutations interact, offering insights into potential biomarkers and therapeutic targets for age-associated hematological disorders.
 Citation Format: Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer Trowbridge, Hideyuki Oguro, Sheng Li. Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4766c6ad595902085d51a487e2edbf0d4dfa6d" target='_blank'>
              Abstract A014: Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis
              </a>
            </td>
          <td>
            Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer J. Trowbridge, Hideyuki Oguro, Sheng Li
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: CRISPR/Cas9 technology has rapidly advanced as a pivotal tool in cancer research, particularly in the precision targeting required for both detecting and treating malignancies. Its high specificity and low off-target effects make it exceptionally effective in applications involving Human Papillomavirus (HPV) related diseases, most notably cervical cancer. This approach offers a refined methodology for the rapid detection of viral infections and provides a robust platform for the safe and effective treatment of diseases associated with viral infections through gene therapy. Purpose: Gene therapy, within this context, involves the strategic delivery of genetic material into target cells via a vector. This is followed by the meticulous modulation of gene expression, whether through correction, addition, or suppression, specifically honed to target tumor cells while sparing healthy cells. This dual capacity to diagnose and treat at such a precise level underscores the transformative potential of CRISPR/Cas9 in contemporary medical science, particularly in oncology and virology. Research Design: This article provides an overview of the advancements made in utilizing the CRISPR-Cas9 system as a research tool for HPV-related treatments while summarizing its application status in basic research, diagnosis, and treatment of HPV. Data Collection: Furthermore, it discusses the future prospects for this technology within emerging areas of HPV research and precision medicine in clinical practice, while highlighting technical challenges and potential directions for future development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69224d4406f31958e4148b5cda3be164a3c3d00" target='_blank'>
              Research Status of Clustered Regulary Interspaced Short Palindromic Repeats Technology in the Treatment of Human Papillomavirus (HPV) Infection Related Diseases
              </a>
            </td>
          <td>
            Minxue Tang
          </td>
          <td>2025-01-01</td>
          <td>Cancer Control : Journal of the Moffitt Cancer Center</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a multifactorial and heterogeneous disease, ranking among the most prevalent malignancies in men. In 2020, there were 1,414,259 new cases of PCa worldwide, accounting for 7.3% of all malignant tumors. The incidence rate of PCa ranks third, following breast cancer and lung cancer. Patients diagnosed with high-grade PCa frequently present with existing or developing metastases, complicating their treatment and resulting in poorer prognoses, particularly for those with bone metastases. Utilizing single-cell RNA sequencing (scRNA-seq), we identified specific malignant cell subtypes that are closely linked to high-grade PCa. By investigating the mechanisms that govern interactions within the tumor microenvironment (TME), we aim to offer new theoretical insights that can enhance the prevention, diagnosis, and treatment of PCa, ultimately striving to improve patient outcomes and quality of life. Methods Data on scRNA-seq was obtained from the GEO database. The gene ontology and gene set enrichment analysis were employed to analyze differential expression genes. Using inferCNV analysis to identify malignant epithelial cells. We subsequently employed Monocle, Cytotrace, and Slingshot packages to infer subtype differentiation trajectories. The cellular communication between malignant cell subtypes and other cells was predicted using the CellChat package. Furthermore, we employed pySCENIC to analyze and identify the regulatory networks of transcription factors (TFs) in malignant cell subtypes. The MDA PCa 2b and VCap cell lines were employed to validate the analysis results through cellular functional experiments. In addition, a risk scoring model was developed to assess the variation in clinical characteristics, prognosis, immune infiltration, immune checkpoint, and drug sensitivity. Results A malignant cell subtype in PCa with high expression of NEFH was identified through scRNA-seq analysis. This subtype was situated at the differentiation terminal, exhibited a higher level of malignancy, and exhibited characteristics that were more prone to advanced tumor lesions. In addition, our research underscored the intricate interactions that exist within the TME, particularly the interaction between PTN secreted by this subtype and fibroblasts via the NCL receptor. This interaction may be closely associated with cancer-associated fibroblasts and tumor progression. Subsequently, we determined that the NEFH+ malignant cell subtype was significantly correlated with the TF IRX4. This TF is linked to a worse prognosis in PCa and may affect disease progression by regulating gene transcription. Our conclusions were additionally verified through cellular experiments. Furthermore, the prognostic model we developed demonstrated satisfactory predictive performance, with gene sets from the high NmRS group facilitating tumor progression and deterioration. The analysis of immune infiltration was instrumental in the development of clinical intervention strategies and patient prognosis. Conclusion By examining the cellular heterogeneity of a unique NEFH+ malignant cell subtype within the PCa microenvironment, we were able to disclose their reciprocal interaction with disease progression. This offers a novel viewpoint on the diagnosis and treatment of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/850edd1c421faf9a184e4ab73c61663fd59239af" target='_blank'>
              Unveiling the NEFH+ malignant cell subtype: Insights from single-cell RNA sequencing in prostate cancer progression and tumor microenvironment interactions
              </a>
            </td>
          <td>
            Jieya Wang, Fu Zhao, Qiang Zhang, Zhoujie Sun, Zhikai Xiahou, Changzhong Wang, Yan Liu, Zongze Yu
          </td>
          <td>2024-12-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f4f25866d80529aa37b7f953a4255c06039896" target='_blank'>
              A Protocol for Detecting DNA Methylation Changes at CpG Sites of Stemness-Related Genes in Aging Stem Cells.
              </a>
            </td>
          <td>
            M. Hodjat, S. M. Daghighi, Mohammad Abdollahi
          </td>
          <td>2025-01-17</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b52af98ba891142904f59178c596ff51575258f" target='_blank'>
              Rapid and scalable personalized ASO screening in patient-derived organoids
              </a>
            </td>
          <td>
            John C Means, A. Martinez-Bengochea, D. Louiselle, Jacqelyn Nemechek, John M Perry, Emily G Farrow, T. Pastinen, Scott T Younger
          </td>
          <td>2025-01-22</td>
          <td>Nature</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="The DREAM (dimerization partner, RB-like, E2F, and multi-vulval class B) complex is an evolutionarily conserved transcriptional repression complex that coordinates nearly one thousand target genes, primarily associated with the cell cycle processes. The formation of the DREAM complex consequently inhibits cell cycle progression and induces cellular quiescence. Given its unique role in cell cycle control, the DREAM complex has gained significant interest across various physiological and pathological contexts, particularly in conditions marked by dysregulated cell cycles, such as cancer. However, the specific cancer types most significantly affected by alterations in the DREAM complex are yet to be determined. Moreover, the possibility of restoring or pharmacologically targeting the DREAM complex as a therapeutic intervention against cancer remains a relatively unexplored area of research and is currently under active investigation. In this review, we provide an overview of the latest advances in understanding the DREAM complex, focusing on its role in cancer. We also explore strategies for targeting the DREAM complex as a potential approach for cancer therapeutics. Advances in understanding the precise role of the DREAM complex in cancer, combined with ongoing efforts to develop targeted therapies, may pave the way for new options in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f83080159a75138829f6897856a0dcb6a27d73" target='_blank'>
              Emerging Role of the DREAM Complex in Cancer and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Ye-Jin Hwang, Moon Jong Kim
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations found in AML such as DNMT3A, TET2 and ASXL1 can be found in the peripheral blood of healthy adults - a phenomenon termed clonal hematopoiesis (CH). These mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile, later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy. We assembled a retrospective cohort of patients diagnosed with de novo AML at our institution that underwent genomic sequencing at diagnosis, remission and/or relapse (total n=182 patients). FLT3 and NPM1 mutations were generally eliminated at complete remission but subsequently reemerged upon relapse, whereas DNMT3A, TET2 and ASXL1 mutations often persisted through remission. CH-related mutations exhibited distinct constellations of co-occurring genetic alterations, with NPM1 and FLT3 mutations enriched in DNMT3Amut AML, while CBL and SRSF2 mutations were enriched in TET2mut and ASXL1mut AML, respectively. In the case of NPM1 and FLT3 mutations, these differences vanished at the time of complete remission yet readily reemerged upon relapse, indicating the reproducible nature of these genetic interactions. Thus, CH-associated mutations that likely precede malignant transformation subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0b511bf0bfbb32bc9fd1061ccf533beadb0c77" target='_blank'>
              Persistent post-remission clonal hematopoiesis shapes the relapse trajectories of acute myeloid leukemia.
              </a>
            </td>
          <td>
            Ryan D Chow, Priya Velu, Safoora Deihimi, Jonathan P Belman, Angela Youn, Nisargbhai Shah, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Robert L. Bowman
          </td>
          <td>2025-02-12</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Senescence is a tumor suppressor mechanism triggered by oncogene expression and chemotherapy treatment. It orchestrates a definitive cessation of cell proliferation through the activation of the p53-p21 and p16-Rb pathways, coupled with the compaction of proliferative genes within heterochromatin regions. Some cancer cells have the ability to elude this proliferative arrest but the signaling pathways involved in circumventing senescence remain to be characterized. We have recently described that malignant cells capable of evading senescence have an increased expression of specific tRNAs, such as tRNA-Leu-CAA and tRNA-Tyr-GTA, alongside the activation of their corresponding tRNA ligases, namely LARS and YARS. We have previously shown that YARS promotes senescence escape by activating proliferation and cell cycle genes but its functions during this proliferative arrest remain largely unknown. In this study, we have continued to characterize the functions of YARS, describing non-canonical transcriptional functions of the ligase. Our results show that YARS is present in the nucleus of proliferating and senescent cells and interacts with the Trim28 transcriptional regulator. Importantly, YARS binds to the LIN9 promoter, a critical member of the Dream complex responsible for regulating cell cycle gene transcription. The ligase facilitates the binding and the phosphorylation of the type II RNA polymerase and promotes the deposition of activating epigenetic marks on the LIN9 promoter. Consequently, during senescence escape, YARS activates LIN9 expression and both proteins are necessary to induce the proliferation of emergent cells. These results underscore unconventional transcriptional functions of YARS in activating LIN9 expression in proliferating cells and during senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c36e3380440225cd14c09aec61c163d6f628c872" target='_blank'>
              A non-canonical role for the tyrosyl tRNA synthetase: YARS regulates senescence induction and escape and controls the transcription of LIN9.
              </a>
            </td>
          <td>
            H. Coquelet, Geraldine Leman, Amine Maarouf, Coralie Petit, Bertrand Toutain, Cécile Henry, Alice Boissard, Catherine Guette, Eric Lelièvre, Pierre Vidi, Jordan Guillon, O. Coqueret
          </td>
          <td>2025-01-05</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Spermatogenesis is a complex process that can be disrupted by genetic and epigenetic changes, potentially leading to male infertility. Recent research has rapidly increased the number of protein coding mutations causally linked to impaired spermatogenesis in humans and mice. However, the role of non-coding mutations remains largely unexplored. As a case study to evaluate the effects of non-coding mutations on spermatogenesis, we first identified an evolutionarily conserved topologically associated domain (TAD) boundary near two genes with important roles in mammalian testis function: Dmrtb1 and Lrp8. We then used CRISPR-Cas9 to generate a mouse line where 26 kb of the boundary was removed including a strong and evolutionarily conserved CTCF binding site. ChIP-seq and Hi-C experiments confirmed the removal of the CTCF site and a resulting mild increase in the DNA-DNA interactions across the domain boundary. Mutant mice displayed significant changes in testis gene expression, higher frequency of histological abnormalities, a drop of 47-52% in efficiency of meiosis, a 15-18% reduction in efficiency of spermatogenesis, and consistently, a 12-28% decrease in daily sperm production compared to littermate controls. Despite these quantitative changes in testis function, mutant mice show no significant changes in fertility. This suggests that non-coding deletions affecting testis gene regulation may have smaller effects on fertility compared to coding mutations of the same genes. Our results demonstrate that disruption of a TAD boundary can have a negative impact on sperm production and highlight the importance of considering non-coding mutations in the analysis of patients with male infertility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15762b1599e500696a63ae395a5142469b79eb41" target='_blank'>
              Deletion of an evolutionarily conserved TAD boundary impacts spermatogenesis in mice†.
              </a>
            </td>
          <td>
            Ana C. Lima, M. Okhovat, Alexandra M. Stendahl, Ran Yang, Jake VanCampen, K. Nevonen, Jarod Herrera, Weiyu Li, L. Harshman, Lev M. Fedorov, Katinka A. Vigh-Conrad, N. Ahituv, Donald F. Conrad, Lucia Carbone
          </td>
          <td>2025-02-04</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="In renewing tissues, mutations conferring selective advantage may result in clonal expansions. In contrast to somatic tissues, mutations driving clonal expansions in spermatogonia (CES) are also transmitted to the next generation. This results in an effective increase of de novo mutation rate for CES drivers. CES was originally discovered through extreme recurrence of de novo mutations causing Apert syndrome. Here, we develop a systematic approach to discover CES drivers as hotspots of human de novo mutation. Our analysis of 54,715 trios ascertained for rare conditions, 6,065 control trios and population variation from 807,162 mostly healthy individuals identifies genes manifesting rates of de novo mutations inconsistent with plausible models of disease ascertainment. We propose 15 genes hypermutable at loss-of-function (LoF) sites as candidate CES drivers. Additional 17 genes feature hypermutable missense mutations at individual positions, suggesting CES acting through gain-of-function (GoF). Among candidates are 5 of 13 known CES drivers, 11 cancer drivers, and 16 members of major signaling pathways; notably, 12 genes show CES evidence in direct sperm sequencing. CES increases the average mutation rate ~16-fold for LoF genes in both control trios and sperm and ~500-fold for pooled GoF sites in sperm. Positive selection in the male germline elevates the prevalence of genetic disorders and increases polymorphism levels, masking the effect of negative selection in human populations. Despite the excess of mutations in disease cohorts for 15 LoF CES driver candidates, only 5 show clear evidence of disease causality, suggesting that CES may lead to false-positive disease associations">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebbede9fcc8090e566844fd84a8257e56652782d" target='_blank'>
              Cohort-level analysis of human de novo mutations points to drivers of clonal expansion in spermatogonia
              </a>
            </td>
          <td>
            V. Seplyarskiy, M. A. Moldovan, E. M. Koch, P. Kar, M. D. Neville, R. Rahbari, S. Sunyaev
          </td>
          <td>2025-01-05</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Head and Neck Squamous Cell Carcinoma (HNSCC) is the sixth most common cancer worldwide, and local failure of disease following treatment with radiation therapy remains a major challenge for improving patient outcomes. We therefore designed a CRISPR-Cas9 pooled genetic screen to identify signaling mechanisms that regulate HNSCC radiosensitivity. Using a kinome gRNA library (763 genes with 8 gRNAs each), Cal27 and Detroit562 cells were screened using fractionated exposure to ionizing radiation as a selection pressure. Deep sequencing results identified significant enrichment or depletion of gRNAs for DNA damage signaling (ATM and DNAPK) and NFKB signaling (IRAK4, IRAK1, IKKA, IKKB), pathways known to regulate cell survival after radiation. Surprisingly, the screen also identified significant depletion of Janus kinase signaling (JAK1 and TYK2) to be a cause of radiation resistance. Independent knockout (KO) of Janus Kinase 1 (JAK1) in both Cal27 and Detroit562 cells caused radioresistance in vitro and in vivo. JAK1 KO prolongs G2 cell cycle arrest in HNSCC after treatment with radiation or after thymidine synchronization. JAK1 KO cells also have enhanced Rad51 foci formation indicative of homologous recombination usage for DNA repair. Consistent with an enhanced mitotic arrest, JAK1 KO causes the appearance of cells with 4N and 8N DNA content after exposure to radiation as measured by flow cytometry with propidium iodide or following 5-ethynyl-2’deoxyuridine (EdU) labeling. HNSCC cells with JAK1 KO, or treated with the JAK1 specific inhibitor Abrocitinib, formed significantly fewer micronuclei after radiation treatment consistent with a reduction in mitotic errors. Consistent with these findings, live cell imaging demonstrated cells with JAK1 KO undergo fewer mitotic catastrophe events. Loss of JAK1 function did not alter CDK1, but delayed activation of Aurora Kinase A (AURKA) and Polo-like Kinase 1 (PLK1), identifying a cell cycle signaling pathway regulated by JAK1. Live cell imaging also showed a more than doubling of the time required from chromatin condensation to sister chromatid separation, suggesting an associated defect in kinetochore assembly. The effects of inhibiting Kif18a, a kinesin that regulates kinetochore tension, in combination with radiation was therefore tested. Kif18 inhibition significant radiosensitized JAK1 KO HNSCC in clonogenic survival and xenograft models. In summary we have elucidated a JAK1-dependent mechanism for the regulation of AURKA/PLK1 signaling, a protective G2 arrest, and radioresistance. Furthermore, we provide a mechanistic and targeted pharmacologic approach for overcoming therapeutic resistance and enhancing radiation therapy in HNSCC.
 Citation Format: Vanessa Kelley, Marta Baro, William Gasperi, Chatchai Phoomak, Hojin Lee, Joseph Contessa. CRISPR-Cas9 screening reveals a novel JAK1 dependent mechanism of radioresistance in head and neck squamous cell carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab6adeb5ebe92c5746446bc4ffdb501c4fd87e05" target='_blank'>
              Abstract B024: CRISPR-Cas9 screening reveals a novel JAK1 dependent mechanism of radioresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Vanessa Kelley, Marta Baro, William Gasperi, C. Phoomak, Hojin Lee, Joseph Contessa
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Metabolism is the name given to all of the chemical reactions in the cell involving thousands of proteins, including enzymes, receptors, and transporters. Inborn errors of metabolism (IEM) are caused by defects in the production and breakdown of proteins, fats, and carbohydrates. Micro ribonucleic acids (miRNAs) are short non-coding RNA molecules, ⁓19-25 nucleotides long, hairpin-shaped, produced from DNA. They play key roles in regulating gene expression of target mRNAs at both transcriptional and post-transcriptional levels. Altered expression of miRNAs has emerged as an additional molecular mechanism implicated in the pathogenesis of many diseases, this altered miRNA expression is also present in inherited metabolic diseases. A single miRNA can regulate targets associated with similar cellular processes and pathways, making miRNAs powerful therapeutics to restore the impaired cellular functions seen in disease phenotypes by enhancing the cellular response. Although the miRNA research field has advanced significantly in recent years, studies in IEM are still limited. Further research on miRNA expression specifically related to IEM may allow the identification of new biomarkers for the diagnosis, progression, and prognosis of diseases. In this review, the literature studies between miRNAs and IEMs, diagnosis, prognosis, follow-up, and treatment possibilities of miRNAs were analyzed. We present recent advances on miRNAs in IEM and an overview of current miRNA therapeutics for the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6176f135941afbcf0385bbe268cda37e8759c7b8" target='_blank'>
              Effects of miRNAs in inborn error of metabolism and treatment strategies.
              </a>
            </td>
          <td>
            Harun Bayrak, Parisa Sharafi, A. Ç. Özketen, Mustafa Kılıç
          </td>
          <td>2025-01-07</td>
          <td>Postgraduate medical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer therapies have attempted to target the transcription factor p53, a gene also described as the “guardian of the genome,” for decades. However, the approach has faced numerous barriers to clinical efficacy due to several factors: mutations in p53 occur in almost half of all human cancers, mutations are cancer-specific, and the associated genomic changes grant mutant p53 with oncogenic potential unique from that of wild-type p53. A host of new therapeutic agents have emerged that work to target mutant p53. These agents can broadly be classified into six categories: the viral approach, direct modifiers of the p53 pathway, epigenetic modifiers of the p53 pathway, synthetic lethal agents, structural reactivators, and immune activating vaccines. Even these strategies have been met with limited success. Bypassing p53 entirely may be the next avenue in cancer therapeutics to kill tumor cells regardless of p53’s mutation pattern.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b1fb68a093354533852b3fe76935ba3e2890db" target='_blank'>
              Side-stepping the guardian of the genome: current cancer therapeutics targeting mutant p53
              </a>
            </td>
          <td>
            I. Taritsa, Eric T. Fossel
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Lung cancer is a leading cause of cancer-related deaths worldwide. Its high incidence and poor prognosis demonstrate the need to investigate new therapies. The PI3K/AKT pathway is activated in carcinogenic processes such as invasion, proliferation, and drug resistance. MiR-21 is a microRNA overexpressed in numerous types of cancer and which activates PI3K/AKT pathway by down-regulating its main targets, PTEN and PDCD4. CRISPR is a revolutionary gene-editing technology that allows genes to be deleted. The aim of this study was to use CRISPR/Cas9 technology as an option to reduce carcinogenic and drug resistance processes by eliminating miR-21. Methods: CRISPR/Cas9 was used to knock out miR-21 (miR-21 KO) in A549 lung cancer cells and thus reverse the carcinogenic processes activated by miR-21 overexpression. Furthermore, the effect of miR-21 KO on drug resistance was studied, choosing the main chemotherapeutic agents used for the treatment of lung cancer: gemcitabine, carboplatin, paclitaxel, and oxaliplatin. Results: miR-21 KO A549 cells exhibited a reduction in proliferation, migration, and colony formation compared to A549 cells. In contrast, the expression of PTEN and PDCD4 increased in miR-21 KO A549 cells. Furthermore, miR-21 KO A549 cells showed a decrease in the IC50 of the drugs used for the treatment of lung cancer: gemcitabine, carboplatin, paclitaxel, and oxaliplatin. Conclusions: Based on these results, miR-21 knock-out using CRISPR/Cas could be a promising strategy for the treatment of lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bdbf106416fb114d36dc96f35e7c3a96445c3a3" target='_blank'>
              Exploring miR-21 Knock-Out Using CRISPR/Cas as a Treatment for Lung Cancer
              </a>
            </td>
          <td>
            Patricia Lara, A. Aguilar-González, Francisco Martín, C. Mesas, Javier Moreno, Ana R. Rama
          </td>
          <td>2025-01-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Background: Clonal hematopoiesis of indeterminate potential (CHIP) is characterized by somatic mutations (e.g. DNMT3A, TET2, and ASXL1I) in hematopoietic stem cells, and linked to age-related clonal expansions and inflammatory states. We previously showed that patients with CHIP receiving CAR-T cell therapy have a four-fold risk of developing clinically significant cytokine release syndrome (CRS) compared to those without CHIP (Transplant Cell Ther 2024). This prompted us to investigate the mechanisms through which CHIP mutations could contribute to CAR-T-related toxicities. Methods: We employed CRISPR-based gene editing to target CHIP-associated genes—DNMT3A, TET2, and ASXL1—in M1-like macrophages, which were differentiated from human monocytes in the presence of 20 ng/mL of GM-CSF. Gene perturbation was achieved with cell-penetrating phosphorothioated single-stranded DNA oligonucleotides to deliver the CRISPR-Cas9 and gRNA complex (PS-Cas9/gRNA). Editing was performed using multiple gRNAs for each gene: DNMT3A (2 gRNAs), TET2 (3 gRNAs), and ASXL1 (2 gRNAs), and decreased protein expression was confirmed post-editing. We co-cultured CS1 CAR-T cells targeting multiple myeloma (MM), MM.1S cells, and edited autologous macrophages at a 1:1:0.1 ratio for 24 hours to assess the inflammatory response. Cytokine secretion was quantified using the LEGENDplex™ Human Inflammation Panel 1 (13-plex). Results: Compared to non-edited macrophages, DNMT3A-, TET2-, and ASXL1-edited macrophages consistently secreted significantly higher levels of IL-6 (642, 920, 1338 pg/ml vs. 325 pg/ml respectively, P<0.05-0.001), MCP-1(90, 122, 151 pg/ml vs. 55 pg/ml, respectively, P<0.01-0.001), TNFα (2015, 1722, 1874 pg/ml vs. 1043 pg/ml, respectively P:ns - < 0.001), IL-1β (75, 99, 146 pg/ml vs. 38 pg/ml, respectively, P<0.05-0.001), and IL-8 (21, 19, 22 ng/ml vs. 15 ng/ml, respectively, P:ns- <0.001). Conclusions: These findings suggest CHIP mutations in human macrophages may exacerbate the inflammatory response in CAR-T therapy, highlighting a potential biomarker for stratifying patients at higher risk for toxicities such as CRS and guiding more personalized prophylactic interventions.
 Citation Format: Xiuli Wang, Scott R. Goldsmith, June-Wha Rhee, Stephen J. Forman, Chunyan Zhang, Saro Armenian. CRISPR-based investigation of CHIP mutations reveals their role in CAR-T therapy-related inflammatory toxicities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/622934218386f9dc97c13a0c545169d234cb7502" target='_blank'>
              Abstract A019: CRISPR-based investigation of CHIP mutations reveals their role in CAR-T therapy-related inflammatory toxicities
              </a>
            </td>
          <td>
            Xiuli Wang, S. Goldsmith, J. Rhee, Stephen J Forman, Chunyan Zhang, S. Armenian
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="To achieve replicative immortality, cancer cells must activate telomere maintenance mechanisms. In 10 to 15% of cancers, this is enabled by recombination-based alternative lengthening of telomeres pathways (ALT). ALT cells display several hallmarks including heterogeneous telomere length, extrachromosomal telomeric repeats, and ALT-associated PML bodies. ALT cells also have high telomeric replication stress (RS) enhanced by fork-stalling structures (R-loops and G4s) and altered chromatin states. In ALT cells, telomeric RS promotes telomere elongation but above a certain threshold becomes detrimental to cell survival. Manipulating RS at telomeres has thus been proposed as a therapeutic strategy against ALT cancers. Through analysis of genome-wide CRISPR fitness screens, we identified ALT-specific vulnerabilities and describe here our characterization of the roles of SUB1, a ssDNA-binding protein, in telomere stability. SUB1 depletion increases RS at ALT telomeres, profoundly impairing ALT cell growth without impacting telomerase-positive cells. During RS, SUB1 is recruited to stalled forks and ALT telomeres via its ssDNA-binding domain. This recruitment is potentiated by RPA depletion, suggesting that these factors may compete for ssDNA. The viability of ALT cells and their resilience toward RS also requires ssDNA binding by SUB1. SUB1 depletion accelerates cell death induced by FANCM depletion, triggering unsustainable levels of telomeric damage in ALT cells. Finally, combining SUB1 depletion with RS-inducing drugs rapidly induces replication catastrophe in ALT cells. Altogether, our work identifies SUB1 as an ALT susceptibility with roles in the mitigation of RS at ALT telomeres and suggests advanced therapeutic strategies for a host of still poorly managed cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6497c277f2efbf01c25fef9e39f419c5577c8b02" target='_blank'>
              The single-stranded DNA-binding factor SUB1/PC4 alleviates replication stress at telomeres and is a vulnerability of ALT cancer cells.
              </a>
            </td>
          <td>
            Jean-Christophe Dubois, Erin Bonnell, Amélie Filion, Julie Frion, Samuel Zimmer, Muhammad Riaz Khan, Gabriela M. Teplitz, Lisa Casimir, Élie Méthot, Isabelle Marois, Mouhamed Idrissou, Pierre-Étienne Jacques, R. Wellinger, Alexandre Maréchal
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 The lysine methyltransferase KMT2A/MLL1 plays a crucial role in normal development and hematopoiesis. Rearrangement of KMT2A results in the formation of a KMT2A fusion protein with aberrant activity, which is a well-characterized driver of leukemogenesis and is therapeutically targetable. In contrast to the widely studied KMT2A translocation, the molecular consequences of other KMT2A alterations, including KMT2A amplification, remain unknown. KMT2A amplification is detected in highly aggressive and treatment-resistant myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). It strongly associates with TP53 mutations and complex karyotypes. Unlike KMT2A rearrangement, KMT2A amplification is reported mostly in patients older than 65 and treated with alkylating agents. Here, we conducted a comprehensive molecular analysis of KMT2A-amplified MDS/AML by long-read nanopore sequencing. We have revealed the unique epigenetic landscape, including DNA methylation, histone modification, and chromatin accessibility, associated with KMT2A amplification. We also explored the potential of targeting KMT2A or KMT2A-medicated pathways to block leukemogenesis. These results provide a foundation for targeting the KMT2A complex in KMT2A-amplified disease, representing a promising first step toward treating MDS/AML associated with KMT2A amplification.
 Citation Format: Jayme Ogino, Yang Liu, Wentao Yang, Madina Sukhanova, Sheng Li, Yali Dou. Epigenetic dynamics in KMT2A-amplified leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245971e55eeef33c3dd5df76ff7015895c3cd83a" target='_blank'>
              Abstract A008: Epigenetic dynamics in KMT2A-amplified leukemia
              </a>
            </td>
          <td>
            Jayme Ogino, Yang Liu, Wentao Yang, M. Sukhanova, Sheng Li, Yali Dou
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cafa641073896bd378d089dc135baed57213473" target='_blank'>
              NLRP7 maintains the genomic stability during early human embryogenesis via mediating alternative splicing
              </a>
            </td>
          <td>
            Zhongliang Chen, Liangxia Jiang, Min Su, Qibing Zeng, Peng Luo, Liangzhao Chu
          </td>
          <td>2025-01-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 For the last thirty years an increasing amount of evidence has shown the relevance of epigenetics in cell biology and tissue physiology, being DNA methylation aberrations in cancer the flag-ship for the recognition of its disturbance in human diseases. From the candidate gene approaches, new powerful technologies such as comprehensive DNA methylation microarrays and whole genome bisulfite sequencing has recently emerged that have reinforced the notion of epigenetic disruption in the crossroad of many sickness. From the poster-boy case of MGMT hypermethylation in the prediction of alkylating drug response to the personalized treatment of leukemia with small molecules targeted to fusion proteins involving histone modifiers, the field has walked a long path. The current talk will focus in the epigenetic profiling, basically at the level of DNA methylation and histone modifications, that is starting to provide clinical value in the diagnosis, prognosis and prediction of response to drug therapies. For cancer, we have already a wide view of the undergoing DNA methylation events that expand beyond classical promoter CpG islands of tumor suppressor genes and we have a growing list of mutated chromatin remodeler genes that contributes to the tumorigenesis process. It is time to apply this knowledge in practical clinical situations like the diagnosis of cancers of unknown primary, the screening of malignancies in high-risk populations or a biomarker selection of the patients that should receive treatment with anticancer drugs, including immunotherapy. Beyond cancer, DNA methylation is starting to be recognized as playing a major role in infectious diseases, and in this regard, the present lecture will also address the epigenomic component of COVID-19. Most important, Epigenomics can be added to Genomics, Transcriptomics and Proteomics plus Bioinformatics to create a Multomics view of human diseases.
 Citation Format: Manel Esteller. DNA methylation in cancer: From knowledge to applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr IA007">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [3, 11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>