<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-01-19 09:44:36 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>50</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>86</td>
          <td>46</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Aggressive variant prostate cancer (AVPC) is a lethal subtype of prostate cancer characterized by its androgen independence, resistance to chemotherapy, and display of neuroendocrine features which can emerge either de novo or via transformation after a prior diagnosis of adenocarcinoma. The poor clinical outcomes in patients with AVPC are associated with its profound molecular heterogeneity. In this study, we analyzed 23 consecutive AVPC cases treated at a dedicated small-cell clinic (2017-2025) using clinicogenomic and transcriptomic profiling. Transformed AVPC exhibited significantly shorter overall survival times than de novo AVPC (11.8 vs 26.0 months, P < 0.001). Integrative genomic analyses identified residual androgen signaling in subsets of cases harboring neuroendocrine lineage programs, highlighting a decoupling of lineage identity and morphology. To facilitate mechanistic and pharmacologic studies, we established NCI-LYM-1, a patient-derived organoid/PDX from an AR-negative, ASCL1+/SYP+ lymph node metastasis, which faithfully recapitulates the donor tumor's molecular and phenotypic features. Short- and long-read whole-genome sequencing combined with optical genome mapping identified biallelic inactivation of PTEN, TP53, RB1 and BRCA2 as potential drivers, demonstrating clonal concordance with circulating tumor DNA from the original patient donor. Pathway and perturbation analyses suggested that NCI-LYM-1 harbored a strong dependency on apoptotic pathways, which was confirmed by in vitro organoid testing with the BCL-2/BCL-xL inhibitor navitoclax (IC50: 0.27 M) and the MCL-1 inhibitor AZD-5991 (IC50: 0.060 M). Overall, NCI&-LYM-1 recapitulates the clinical aggressiveness and heterogeneity of AVPC, providing a tractable platform to identify novel precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e71e79208c8d37fdc48221a6ff9b64fb28fe818c" target='_blank'>
              Molecular decoupling of lineage identity and morphology in aggressive variant prostate cancer
              </a>
            </td>
          <td>
            C. Li, , M. L. Abel, D. S. Vargas Solivan, K. Bhadresha, S. Kartal, S. Nichols, K. Parmar, J. Twohig, T. M. Truong, C. H. Chau, K. Kelly, W. Figg, A. Thomas, A. Sowalsky
          </td>
          <td>2026-01-09</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="High-risk neuroblastoma is a pediatric tumor that originates from immature sympathetic neuroblasts and is characterized by a low mutational burden. Nevertheless, it frequently presents with highly recurrent chromosomal imbalances, including MYCN amplification and gain of chromosome 17q. Highly proliferative cancers, such as neuroblastoma, exhibit significant DNA replication stress, rendering tumor cells dependent on ATR-CHK1 signalling and DNA damage repair pathways. We previously hypothesized that gene dosage effects resulting from 17q copy number alterations of non-mutated genes involved in replication stress resistance could offer novel therapeutic opportunities. To identify critical candidate genes and pathways driving high-risk neuroblastoma, we performed an integrated bioinformatics analysis, and identified the BRIP1 gene, encoding the FANCJ protein, as top-ranked 17q candidate. FANCJ is involved in multiple processes that alleviate replication stress. In the absence of specific FANCJ-targeting compounds, we evaluated the phenotypic and molecular effects of pharmacological inhibition of the MUS81 endonuclease, which functions downstream of FANCJ in restarting stalled replication forks. When combined with CHK1 inhibition, we observed synergistic effects on neuroblastoma cell growth and survival, supporting further development of on-target MUS81 inhibitors for in vivo preclinical testing and future clinical trials aimed at overcoming replication stress resistance in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab80b92dbc650636e5eb2f387901a782d6b51b0" target='_blank'>
              Synergistic inhibition of CHK1 and MUS81 to combat replication stress resistance in high-risk neuroblastoma
              </a>
            </td>
          <td>
            E. Hilgert, Christophe Van Neste, Sarah-Lee Bekaert, F. Martens, Suzanne Vanhauwaert, E. Sanders, P. Verstraelen, Martijn Risseeuw, E. M. Westerhout, Mark A Grannetia, Serge Van Calenbergh, Nadine Van Roy, W. D. De Vos, Rob W F Wolthuis, Frank Speleman, L. Depestel, K. Durinck
          </td>
          <td>2025-11-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Mantle cell lymphoma (MCL) responds to frontline therapy but is susceptible to relapse. While Bruton’s tyrosine kinase inhibitors (BTKi) achieve high response rates, most patients eventually experience disease progression. Predicting responses to subsequent treatments remains challenging due to the lack of an established platform. Heterogeneity in gene alterations and cellular pathways contribute to resistance, complicating treatment approaches. Here, we present a multi-modal profiling platform, targeting key pathways rather than focusing on singular DNA-associated lesions. We identified dysregulated signaling pathways by performing gene expression profiling on 20 MCL samples using a custom MCL MATCH gene set and analyzed the data with gene-set variation analysis. We also screened 22 therapeutics in vitro to assess their efficacy. Whole exome sequencing was conducted to identify prevalent and actionable mutations linked to enriched pathways. Finally, we tested selected therapeutics in patient-derived xenograft mouse models to predict potential response in corresponding patients. Based on our integrative profiling, we identified the top therapeutic candidates for three patients in this study. This integrative platform may help identify targeted therapies for BTKi-relapsed/refractory MCL patients in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3929c1302d66ca38df0d5f835571ca5c0772a36" target='_blank'>
              Integrative profiling strategies to guide personalized therapy in mantle cell lymphoma: a pilot study
              </a>
            </td>
          <td>
            Yang Liu, H. Hill, Yijing Li, J. McIntosh, V. Jiang, F. Yan, Yixin Yao, Yue Fei, Jared R Zhang, Lawrence Qu, Jun Yao, P. Jain, Kent Chen, Michael Wang
          </td>
          <td>2025-11-21</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee85598703608ccef540311d86a0a9e7d9f4dae2" target='_blank'>
              Genetic barcoding of individual cells links cancer evolutionary trajectories and prognostic outcomes
              </a>
            </td>
          <td>
            Candice Merle, I. Di Terlizzi, M. Huyghe, Danae Welboren, Wenjie Sun, Cécile Conrad, Leïla Perié, S. Rulands, Silvia Fre
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Oncogenic KRAS mutations drive metabolic reprogramming in pancreatic ductal adenocarcinoma (PDAC). Src-homology 2 domain-containing phosphatase 2 (SHP2) is essential for full KRAS activity, and promising dual SHP2/mitogen-activated protein kinase (MAPK) inhibition is currently being tested in clinical trials. Exploitable metabolic adaptations may contribute to invariably evolving resistance. To understand the metabolic changes induced by dual inhibition, we comprehensively tested human and murine PDAC cell lines, endogenous tumor models, and patient-derived organoids, which are representative of the full spectrum of PDAC molecular subtypes. We found that dual SHP2/mitogen-activated protein kinase kinase (MEK1/2) inhibition induces major alterations in mitochondrial mass and function, impacts reactive oxygen species (ROS) homeostasis and triggers lipid peroxidase dependency. Anabolic pathways, autophagy and glycolysis were also profoundly altered. However, most strikingly, mitochondrial remodeling was evident, persisting into a therapy-resistant state. The resulting vulnerability to the induction of ferroptotic cell death via the combination of vertical SHP2/MEK1/2 with glutathione peroxidase (GPX4) inhibition was largely independent of the PDAC molecular subtype and was confirmed with direct targeting of RAS. The triple combination of SHP2/MEK1/2 inhibition and the ferroptosis-inducing natural compound withaferin A suppressed tumor progression in an endogenous PDAC tumor model in vivo. Our study offers a metabolic leverage point to reinforce RAS pathway interference for targeted PDAC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ece64da9d6dcfbff96e0c83595844cf2b2f5547" target='_blank'>
              Vertical RAS pathway inhibition in pancreatic cancer drives therapeutically exploitable mitochondrial alterations
              </a>
            </td>
          <td>
            P. Hafner, S. Keller, Xun Chen, Asma Alrawashdeh, H. Jumaa, Friederike I. Nollmann, S. Besson, Judith Kemming, Oliver Gorka, T. Das, Bismark Appiah, Ariane Lehmann, M. Li, Petya Apostolova, Bertram Bengsch, R. Zeiser, S. Tholen, Oliver Schilling, Olaf Groß, Andreas Vlachos, U. Wittel, Dominik von Elverfeldt, W. Reichardt, M. Boerries, G. Andrieux, Guus Heynen, S. Fichtner-Feigl, Luciana Hannibal, D. Ruess
          </td>
          <td>2026-01-16</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e46bc8969d218f90b6869d6ad86ad7ec80c28324" target='_blank'>
              Loss of ESE3/EHF is sufficient to promote cell plasticity, transformation and androgen-independent status in the early stage of prostate carcinogenesis
              </a>
            </td>
          <td>
            D. Albino, G. Sandrini, C. Musumeci, Elisa Storelli, Giovanni Papa, Daniela Impellizzieri, A. Cacciatore, G. Cassanmagnago, Atik Balla, Simone Mosole, G. Civenni, Andrea Rinaldi, Carmen Jeronimo, Rui M. Henrique, Marco Bolis, C. Catapano, G. M. Carbone
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Drug tolerant persister cells (DTPs) refer to a transient drug-tolerance sub-population of cancer cells characteristics of phenotype plasticity and heterogeneity. This adaptive cell state is a critical transitional phase, standing on the crossroad that cancer cells reacquire drug sensitivity or enter into the permanent drug resistance. Emerging evidences indicate the epitranscriptomic regulations, particularly RNA methylations are the important mechanism underline post-transcriptional regulations of genes expression across all RNA species. RNA is integral to gene expression as messenger RNA (mRNA), transfer RNA (tRNA) and ribosomal RNA (rRNA), which play roles in transmitting information from DNA to the synthesis of functional proteins. Methylation modifications on these RNAs are prevalent and represent a well-recognized non-genetic mechanism, exerting multifaceted regulatory effects on nucleic acid metabolism, such as nucleotide precursor availability, RNA processing dynamics, sub-cellular localization, transcript stability and translational fidelity/ efficiency. This review systematically sorts out the relevant references, demonstrating recent advances on the knowledge of the patterns of methylation modifications on mRNA, tRNA and rRNA, and how these modifications drive the generation and development of DTPs, which hallmarks of epithelial-mesenchymal transition, metabolism shift and immune escape. And then clinical strategies are delineated, leveraging pharmacological modulators of RNA-modifying enzymes alongside non-pharmaceutical lifestyle advice, for the development of therapy strategies preventing DTPs-rooted tumor relapse in this anti-tumor armamentarium with cytotoxic reagents, targeted therapies and immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098e2b3a70bd6f86f19a2d17ea69012683c3e930" target='_blank'>
              Therapy-induced mRNA, rRNA and tRNA methylation alterations confer tolerance phenotype in tumor cells: mechanism and implications
              </a>
            </td>
          <td>
            Anfeng Jiang, Shujie Liu, Zhiyuan Li, Xiongzhou Zhang, Minghao Duan, Bin Li
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, Darya A Yurchenko, Zhanna G Markova, Tatiana V Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Genetic instability is a hallmark of cancer, often arising from mutations to DNA damage repair and response (DDR) genes. Classical genetic, biochemical, and structural approaches elucidated the foundational mechanisms of DDR pathways and provided a scientific understanding of their involvement in repair of lesions induced by broad classes of DNA-damaging agents (DDAs). However, given the chemical diversity of DDAs and resultant DNA lesions, along with the multitude of interconnected DDR factors, the chemogenomic landscape of DDA–DDR interactions remains incompletely mapped. To this end, we developed a DDR-targeted, CRISPR knockout screening approach and assessed relationships amongst 353 DNA repair genes and 15 DDAs in LN229 glioma cells. Within this dataset of 5295 DDR-related chemogenomic interactions, we identified many established interactions and discovered novel ones. For example, we observed a specific role of transcription-coupled nucleotide excision repair in the repair of adducts generated by monofunctional alkylating agents, a role for the Fanconi anemia pathway in addressing methyl lesions, overt differences in DSB repair following treatment with topoisomerase I versus II poisons, and repair dependencies associated with the imidazotetrazines temozolomide, mitozolomide, and KL-50. Future directions will continue to investigate the mechanisms of novel chemogenomic interactions that we have uncovered as well as work to identify chemogenomic interactions amenable to clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018770119a2496ee321bc85a8d0504634bbf54c9" target='_blank'>
              Targeted CRISPR knockout screening identifies known and novel chemogenomic interactions between DNA damaging agents and DNA repair genes
              </a>
            </td>
          <td>
            Collin D. Heer, James L. Elia, Vijay Menon, Spenser S Johnson, Sofia R. Arbelaez, Sam Friedman, Francesc Lopez-Giraldez, R. Sundaram, Seth B Herzon, R. Bindra, Susan E. Gueble
          </td>
          <td>2026-01-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Loss-of-function mutations in the TP53 gene are the most frequent genetic events in human malignancies. However, the extent to which missense TP53 mutations confer oncogenic gain-of-function (GOF) properties remains incompletely understood. Using genetically defined pancreatic cancer organoid models, we demonstrated that mutant p53 provides a selective growth advantage under acute environmental stress and drives molecular transition toward a basal-like state. Transcriptomic analyses showed that under stress conditions, organoids expressing mutant p53 adopted a basal-like gene expression program, a characteristic of a poor prognosis subtype of pancreatic cancer. Furthermore, the molecular subtype was reversed to the classical subtype once it returned to normal growth conditions, suggesting that transcriptional plasticity arises from acute changes in the tissue microenvironment. Given these context-dependent oncogenic properties, our findings suggest that targeting mutant p53 may be a promising therapeutic strategy for pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f58263187bbd03e5cadcf959b6bde70a5832e40c" target='_blank'>
              Mutant p53 regulates reversible subtype plasticity in pancreatic cancer
              </a>
            </td>
          <td>
            June-Ha Shin, Hwa-Ryeon Kim, Jae-Seok Roe
          </td>
          <td>2025-12-01</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The reproducibility of scientific research has been increasingly challenged in recent years. While extrinsic factors (e.g., cross-contamination, mycoplasma infection, serum variability) are well-studied, the role of intrinsic attributes like cell passage number remains underexplored. Using two tumor cell lines (ACHN and Renca), we employed RNA sequencing to analyze transcriptomic dynamics across passages (P3 to P39). Results revealed a nonlinear transcriptomic shift: mid-passage cells (P10/P11, P17) showed heightened activity in cell cycle, metabolism, and stress response, whereas low (P3) and high passages (P24/P39) exhibited stable and similar profiles. KEGG analysis indicated significant alterations in signaling, immune, and metabolic pathways during passaging. This study demonstrates that passage number shapes transcriptional landscapes and impacts experimental reproducibility. We advocate for strict passage control and explicit reporting of passage information to enhance reliability. These findings underscore the need for standardized cell culture practices to improve research reproducibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf829c80ee16d8593d7d0383b2985d0175ddbfa6" target='_blank'>
              Cell passage number drives transcriptomic drift as an overlooked factor in experimental reproducibility
              </a>
            </td>
          <td>
            Shuai Liu, Hai-juan Peng, Junyan Meng, Liyuan Zhuo, Shaoping Fu, Wei Yang
          </td>
          <td>2025-11-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/768d3fe55602ad3e6e64d51b9bc66ae2727e39e3" target='_blank'>
              TP53 and RB1 are predictive genetic biomarkers for sensitivity to cytarabine in gliomas
              </a>
            </td>
          <td>
            L. Laaniste, Joachim Luginbuehl, Mi Trinh, Rachel Watkins, Gregor Lueg, Grant Neilson, Marjolein Burgers, Kasia Kania, Regina Reynolds, Zoe Smith, Julia Kleniuk, Lauren Gould, Dorota Pregowska, Sophie Mathiesen, Michael Johnson, Louwai Muhammed
          </td>
          <td>2025-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity and plasticity. GBM tumors contain neoplastic cells within a fluid spectrum of diverse transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we develop scFOCAL (Single-Cell Framework for -Omics Connectivity and Analysis via L1000), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identify compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states, and leverage this capability to predict combination synergy. We validate the significance of these findings in vitro, ex vivo, and in vivo, and identify a combination of an OLIG2 inhibitor and Depatux-M for the treatment of GBM. Our studies suggest that scFOCAL identifies cell states that are sensitive and resistant to targeted therapies in GBM using a measure of cell and drug connectivity, which can be applied to identify new synergistic combinations. Glioblastoma is characterised by high levels of intratumoural heterogeneity and plasticity, hindering treatment. Here, the authors develop an analytical framework, scFOCAL, to predict the sensitivity of glioblastoma cell subpopulations to therapies based on reversal of disease transcriptional signatures to identify synergistic therapeutic combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef55e3b53c83b7969434910c5b4838c55ce9672" target='_blank'>
              Drug and single-cell gene expression integration identifies sensitive and resistant glioblastoma cell populations
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, R. Komotar, Macarena I. de la Fuente, Gregory Stein, Alexandre Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, Nagi G. Ayad
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5ef1091f544d26b38d75e4b4f01629d3fcc603" target='_blank'>
              VARS1 fuels prostate cancer evolution via codon-selective translational rewiring
              </a>
            </td>
          <td>
            Qinju He, Rongrong He, Yingying Li, Miaomiao Xu, Shuchao Ren, C. Zou, Zhenyu Li, Wenchao Li, Yuanzhen Zhang, Lanxin Hu, Bin Xu, Baobing Zhao, Zhipeng Zhou, Dingxiao Zhang
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19e1efee6c0706220f7b18885b5b5a4638edf605" target='_blank'>
              A single-nucleus multimodal framework reveals epigenomic priming of chemoresistant states in ovarian cancer
              </a>
            </td>
          <td>
            Y. Landais, J. Bertorello, M. Puerto, Agathe Kahn, B. Simon, A. Roehrig, Adeline Durand, M. Quatredeniers, C. Lamy, E. Laas, Nicolas Pouget, Simon Dumas, J. Féron, Virginie Fourchotte, C. Bonneau, Vincent Cockenpot, Marianne Delville, Magali Richard, M. Kamal, F. Lécuru, C. L. Tourneau, C. Vallot
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4fd78fdc6e7168376ba52f3c22a1f28b8371667" target='_blank'>
              Transcriptomic profiling of epigenetic regulators and metabolic reprogramming in human cholangiocarcinoma
              </a>
            </td>
          <td>
            Amaya López-Pascual, Jasmin Elurbide, Emiliana Valbuena-Goiricelaya, M. Latasa, Elena Anaya, Elena Adán-Villaescusa, Borja Castelló-Uribe, L. A. Martínez-Pérez, I. Uriarte, M. Arechederra, Sergio Ciordia, Fernando Corrales, Sona Frankova, Eva Sticová, Ondřej Fabián, L. Colyn, Patricia Inacio, Robert Arnes-Benito, J. Bayo, Meritxell Huch, C. Berasain, M. G. Fernández-Barrena, Matías A. Avila
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Colorectal cancer (CRC) progression and therapy resistance are driven by heterogeneous tumor cell populations and microenvironmental interactions. However, a comprehensive single-cell atlas across patients that captures this heterogeneity, and its clinical implications has been lacking. Such an atlas could reveal rare tumor subpopulations that underpin disease aggressiveness and offer new prognostic biomarkers or therapeutic targets. We integrated two high-quality single-cell RNA sequencing datasets from 29 CRC patients to construct a cellular atlas encompassing immune, stromal and epithelial compartments. Malignant epithelial cells were distinguished via inferCNV-based copy number alteration analysis and reclustered, yielding seven transcriptionally distinct malignant subpopulations. One malignant epithelial cluster, marked by high expression of the long non-coding RNA ELFN1-AS1, exhibited the highest stemness signature. From this stem-like cluster, we derived a four-gene prognostic model (RPL21, GAL, ELFN1-AS1, AIF1L). In the TCGA-COAD cohort, this model stratified patients into high-risk and low-risk groups with significantly different survival outcomes, with the high-risk group experiencing significantly worse survival. High-risk tumors were enriched for metabolic and translational pathways and displayed distinct immune and genomic features. AIF1L was identified as a hub gene within the signature, and its knockdown in CRC cells enhanced migration and invasion, functionally validating its role in tumor progression. Our study provides a high-resolution single-cell atlas of CRC and identifies a previously unrecognized stem-like tumor cell subpopulation with prognostic significance. These findings highlight novel prognostic biomarkers and suggest potential therapeutic targets (such as AIF1L) that could inform patient stratification and the development of targeted therapies for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3d35184693bc48dd0fe85be6bfc55a03d6e4e7" target='_blank'>
              Single-cell RNA sequencing defines prognostic subtypes and identifies AIF1L as a therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Gao Li, Dingxue Wang, Guo Chen
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms in this study, we perform single nuclei multiome analysis of matched primary and recurrent tumors. Therapy resistant Medulloblastoma demonstrates an expanded persistent progenitor population. Additionally, distinct chromatin landscapes link to altered transcription and correspond with metabolic reprogramming. In vivo modeling of radiation resistance exhibits similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying targetable pathways for MYC-driven medulloblastoma. MYC-driven medulloblastoma is highly aggressive and associated with poor survival. Here, the authors perform single nuclei multi-omic analysis of matched primary and recurrent tumours and identify potential resistance mechanisms and targetable pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fba49308e533a39e32c7d3dcd263f21e4a690d4" target='_blank'>
              Single-cell multi-omics identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce, Breauna Brunt, P. Bompada, A. Donson, Andrew Goodspeed, Kiara Smart, Nicholas Foreman, Nathan A. Dahl, Rajeev Vibhakar
          </td>
          <td>2025-11-25</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6260543e09115d78b28611e811c1654d5ea522" target='_blank'>
              Mutant ASXL1 Drives Transcriptional Activation and Repression in Human Hematopoiesis
              </a>
            </td>
          <td>
            , Aliya Quintal, Samantha J. Worme, Thai Nguyen, Zachary Schonrock, Mitsuhiro Tsuchiya, Sonia N. Acharya, Hanqian L Carlson, Itallia V. Pacentine, Shawn B Shrestha, Jommel Macaraeg, Randall Armstrong, James McGann, Sara Evans-Dutson, Theresa A. Lusardi, Brendan L. O’Connell, Andrew C. Adey, G. Yardımcı, Julia E. Maxson, Theodore P. Braun
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Prostate cancer is a highly heterogeneous disease, driven by genomic and transcriptional changes that impact disease progression and metastatic potential. The interplay between clonal evolution, transcriptional plasticity, and tumour microenvironment is, however, poorly understood. Here, we leverage and integrate single-nuclei RNA sequencing and whole-genome sequencing from 43 spatially distinct tumour samples from five patients with locally advanced prostate cancer to reconstruct clonal evolution trajectories and transcriptional changes driving metastasis at single-cell resolution. We find extensive clonal heterogeneity, including both monophyletic and polyphyletic metastatic dissemination, and ongoing clonal evolution in the primary tumour after metastatic spread. Metastatic seeding converges on disease trajectories involving both genomic and transcriptional changes, including androgen receptor independence and activation of estrogen-, WNT- and JAK-STAT- pathway activity, in spatially distinct areas. Our findings suggest an intricate interplay between clonal evolution and cellular plasticity driving metastatic seeding and point toward more integrative prognostic markers for improved patient management. The impact of tumour heterogeneity on metastatic potential in prostate cancer remains poorly understood. Here, the analysis of single nuclei RNA sequencing and whole-genome sequencing from samples from five patients suggests an interplay between clonal evolution and cellular plasticity driving metastatic seeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea384ac37d8c4c2acd2bd5fc9ab62cffd7ea81c" target='_blank'>
              Clonal evolution and transcriptional plasticity shape metastatic dissemination routes in prostate cancer
              </a>
            </td>
          <td>
            Migle Mikutenaite, Evdoxia Karadoulama, F. Favero, A. Locallo, Francisco German Rodriguez Gonzalez, Daria Kiriy, Tina Keshavarzian, K. Furlano, M. Graefen, Robert G. Bristow, Clarissa Gerhäuser, C. Plass, G. Sauter, R. Simon, M. Lupien, S. Minner, Thorsten Schlomm, Joachim Weischenfeldt
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Tumors are composed of cellular populations with distinct genotypes and phenotypes, which dynamically evolve over time and during treatment. This process is known as clonal evolution, and it is difficult to reveal fine-scale clonal structure with traditional bulk sequencing. Although single-cell genome sequencing could enable reconstruction of tumor clonal evolution, it remains technically challenging and the number of single cells profiled is generally insufficient due to high cost. To address this issue, we developed scClone, a computational toolkit that integrates variant detection and genotype inference for single-cell RNA-seq (scRNA-seq) and spatial transcriptomic data. It further provides interactive visualization of clonal structure and dynamic evolution. scClone addresses key limitations inherent to scRNA-seq, such as expression drop-out and allelic imbalance, and incorporates cell type or state annotation with mutational signature analysis to enable comprehensive profiling of tumor heterogeneity. scClone demonstrated robust performance across multiple datasets—generated from both full-length and fragmented RNA sequencing—by accurately reproducing mutation profiles and resolving clonal mixtures in myeloma, hepatocellular carcinoma and pancreatic cancer. Additionally, scClone has been applied to spatial transcriptomics, enabling the delineation of clonal structures within histological sections from ovarian cancer and cutaneous squamous cell carcinoma. In summary, our results demonstrate that scClone can extract genetic information from scRNA-seq datasets, thereby successfully establishing genotype–phenotype associations at the single-cell level and providing insights into the clonal evolution of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d470f131086238e06c10a786c9bfc7fb86a41116" target='_blank'>
              De Novo Detection of Clonal Structure and Evolution in Single-Cell and Spatial Transcriptomes
              </a>
            </td>
          <td>
            Shihao Bai, Xianbin Su, Ziyao Chen, Ze-Guang Han
          </td>
          <td>2025-11-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29acdf562d4b29c5b781b23b586561ef03a93de" target='_blank'>
              Pan-cancer analysis reveals genomic fidelity and evolution of patient-derived organoids
              </a>
            </td>
          <td>
            Linoy Raz, Tal Ben-Yishay, Haia Khoury, Ron Saad, Amjad Zoabi, Giulia Orlandi, S. Donzelli, Giovanni Blandino, Uri Ben-David
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous malignancy with a poor prognosis and limited response to current therapies. Cancer stem cells (CSCs) contribute to this heterogeneity, driving tumor progression, immune evasion, and therapeutic resistance. Epigenetic regulators have emerged as pivotal modulators of CSC phenotypes. The histone methyltransferase SETDB1, a key stemness factor in both embryonic and adult stem cells, catalyzes H3K9 trimethylation (H3K9me3) and acts as a major oncogenic driver in a wide range of solid and hematological malignancies. Yet, its specific role in maintaining stemness and contributing to therapeutic resistance in HCC remains poorly defined. Integrative analyses of bulk and single-cell transcriptomic datasets from public HCC cohorts were performed. Tumors were stratified by SETDB1 expression, followed by pathway enrichment, immune deconvolution, and stemness scoring. Associations with clinical outcomes, immune phenotypes, drug resistance signatures, and chromatin-binding partners (DNMT3A, TRIM28, HDAC1) were also explored. The role of SETDB1 was then examined in vitro, where a stem-like phenotype was induced in HCC cell lines under hypoxic conditions, and patterns of SETDB1 expression and stemness marker levels were assessed following paclitaxel treatment. SETDB1-low tumors were enriched in well-differentiated, immunologically active subclasses and were associated with increased predicted responsiveness to immune checkpoint blockade (ICB), sorafenib, and transarterial chemoembolization. Conversely, SETDB1-high tumors exhibited transcriptional features of dedifferentiation, elevated stemness scores, and enrichment in aggressive molecular subtypes. These tumors also displayed higher TP53 mutation rates, reduced immune infiltration, immune exclusion signatures, and activation of pathways linked to ferroptosis and ABC transporter dysfunction, consistent with resistance to ICB and conventional therapies. SETDB1 expression was also associated with hypoxia-related pathways, suggesting a role in maintaining CSC niches. In vitro, hypoxia-induced stemness coincided with increased SETDB1 levels, whereas paclitaxel treatment decreased both SETDB1 expression and stemness markers. A chromatin risk score incorporating SETDB1 and its partners predicted poor disease-free survival independently of clinical parameters. SETDB1 defines a stemness-enriched, immune-resistant HCC subtype associated with poor outcomes and therapeutic failure. Correlations between SETDB1 expression and stem-like features in vitro suggest a potential role in maintaining CSC phenotypes, supporting its relevance as a biomarker and candidate for epigenetic- and CSC-directed therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c393716a837e5b1ef0091b6385eb92c64be4ab4" target='_blank'>
              The epigenetic regulator SETDB1 as a key component of cancer stem cells and drug resistance in primary liver cancer
              </a>
            </td>
          <td>
            Maël Padelli, Christophe Desterke, Aurore Devocelle, Georges Uzan, Antoinette Lemoine, Julien Giron-Michel
          </td>
          <td>2026-01-06</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9934d0866a4f499711a0c6e0e0bd21bdab1c7f72" target='_blank'>
              Tracking ongoing chromosomal instability using single-cell whole-genome sequencing
              </a>
            </td>
          <td>
            Bárbara Hernando, Blas Chaves-Urbano, Maria Escobar-Rey, Marina Torres, Alice Cádiz, Ángel Fernández-Sanromán, Sara Barrambana, Carmen G. Lechuga, Carmen Guerra, Mariano Barbacid, David Gómez-Sánchez, F. Sánchez-Vega, P. Razavi, Maria Garcia-Perez, Patricia G. Santamaria, Geoffrey Macintyre
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="BAP1, a deubiquitinase with a ubiquitin C-terminal hydrolase domain, functions as a tumor suppressor involved in diverse cellular processes, including DNA repair, genome stability, and apoptosis. Inactivating mutations in BAP1—particularly missense and deletion variants—are recurrent across multiple cancers, with a high prevalence in clear cell renal cell carcinoma (ccRCC), mesothelioma, and uveal melanoma. Among these, the Glu31Lys mutation in ccRCC impairs BAP1’s enzymatic activity, protein stability, and DNA repair functions. Here, we investigated the physiological impact of this recurrent mutation using isogenic KMRC-20 ccRCC cell clones in which the Glu31Lys substitution was precisely corrected to wild-type glutamate via CRISPR-Cas9–mediated adenine base editing. BAP1 reactivation restored anchorage-dependent growth in KMRC-20 cells—a hallmark of non-transformed epithelial cells—and increased apoptosis under non-adherent conditions, indicating reinstated sensitivity to anoikis. Mechanistically, this phenotypic switch was accompanied by post-transcriptional downregulation of N-cadherin and β-catenin under anchorage-free conditions, implicating BAP1 in the regulation of adhesion- and Wnt-related survival pathways. Furthermore, transcriptomic profiling revealed broad gene expression changes upon BAP1 restoration, suggesting that these combined alterations contribute to the re-establishment of anchorage-dependent growth in KMRC-20 cells. These findings uncover a previously unrecognized role for BAP1 in suppressing anchorage-independent survival, providing new insights into BAP1-driven tumorigenesis and underscoring the therapeutic potential of precise gene editing to restore tumor suppressor function in ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80b7cb16b5faa6ded3efeeec8e097bd6263a5387" target='_blank'>
              Restoration of BAP1 activity via base editing suppresses anchorage-independent survival in kidney cancer
              </a>
            </td>
          <td>
            Daye Lee, Boram Lee, Jiyeon Lee, Jongbum Kwon
          </td>
          <td>2025-11-25</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="This study aims to explore the plastic changes in cell lineages during the progression of osteoarthritis (OA) and their relationship with dysregulation of signaling pathways and provide new molecular targets for precise treatment. Single-cell RNA sequencing (scRNA-seq) technology was utilized to perform high-resolution cell lineage analysis of OA patients. The mappings of distinct cell subpopulations were systematically constructed and revealed the changes in key cell types and their transformation trajectories throughout the progression of OA. Furthermore, KEGG and GO enrichment and pseudotime trajectory analysis were applied to elucidate the functional reprogramming of different cell types and the dynamic imbalance of their signaling networks in OA. Additionally, in vitro experiments were conducted to validate the biological functions of candidate genes in OA. Articular cartilage showed a transcriptional cellular heterogeneity in OA by scRNA-seq analysis; the annotated PreFC, FC, and PreHTC subsets accounted for the main part of OA samples. PreFC cells revealed transcription, signaling, and metabolic reprogramming in OA; pseudotime trajectory found that PreFC transformed to FC cells under the condition of hypoxia and metabolic reprogramming, while fibrosis and ECM degradation pathways showed intense upregulation in preHTC evolved from PreFC cells. HIF1A and ANGPTL4 were identified as key molecular regulators of OA progression, contributing to ECM degradation, inflammation, and apoptosis in chondrocytes, as confirmed through functional validation. The cellular trajectories of OA show significant plasticity changes which are influenced by the dysregulation of multiple signaling pathways. This research provides new insights into the pathological process of OA and offers potential targets for therapeutic strategies targeting these abnormal mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22f338ae69990e93e850949b4a71efc7421d155" target='_blank'>
              Lineage plasticity and signal dysregulation define the cellular trajectory of osteoarthritis progression
              </a>
            </td>
          <td>
            Guanghao Chen, Kundi Tai, Guoyu Dai
          </td>
          <td>2025-11-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="CRISPR-Cas9 genome editing enables precise genetic modifications by introducing targeted DNA double-strand breaks (DSBs). While Cas9-induced DSBs are known to cause unintended on-target mutations, their impact on the epigenetic landscape remains unexplored. Here, we investigate how Cas9-induced DSBs affect DNA methylation patterns in human embryonic stem cells (hESCs). We induce DSBs at differentially methylated regions of imprinted genomic loci and perform high-coverage, long-read native DNA sequencing to simultaneously obtain genetic variant and base-resolution methylation data in a haplotype-resolved manner. Our findings reveal that DSBs cause significant changes in DNA methylation at target sites through mechanisms including homologous recombination, large structural variations, or defective methylation maintenance during DNA repair. Notably, these epigenetic changes can occur either together with or independently of genetic alterations. Beyond imprinted loci, Cas9-induced DSBs significantly disrupt DNA methylation patterns of the MLH1 epimutation alleles in colorectal cancer cells, and hypermethylated heterochromatin loci in hESCs. Clonal analysis indicates that the aberrant methylation changes are stable during in vitro passaging. Intriguingly, significant changes in DNA methylation levels are also detected around endogenous deletions in unedited genomic regions, suggesting that methylation alterations are not unique to Cas9 nuclease activity but represent a general outcome of DSB repair in human cells. This study underscores the importance of assessing and mitigating unintended epigenetic consequences in genome editing applications, as such changes can profoundly affect gene regulation and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f286fe9b90d8b6d662cbbd6ec5480ee68e1fcc17" target='_blank'>
              CRISPR-Cas9-induced double-strand breaks disrupt maintenance of epigenetic information
              </a>
            </td>
          <td>
            Mengge Wang, Yingzi Zhang, C. Bi, Mo Li
          </td>
          <td>2025-12-01</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Phenotypic plasticity plays a key role in cancer progression and metastasis, enabling cancer cells to adapt and evolve, but precisely how distinct axes governing phenotypic plasticity interact to shape tumour progression and patient outcomes remains unclear. We investigated five major interconnected axes of plasticity in ER-positive (ER+) breast cancer: Metabolic Reprogramming, Epithelial-to-Mesenchymal Plasticity (EMP), Luminal–Basal (Lineage) Switching, Stemness, and Drug-resistance using network dynamics simulations, integrative bulk and single-cell transcriptomic analyses and patient survival analyses. We show that these axes are not independent but drive one another, forming two mutually inhibiting ‘teams’ of nodes enabling specified cellular behaviour. One team (favouring high glycolysis, stem-like, basal-like, mesenchymal/hybrid and tamoxifen-resistant phenotype) was found to be associated with aggressive progression and worse survival. On the other hand, the opposing team (favouring high oxidative phosphorylation, non-stem-like, luminal-like, epithelial and tamoxifen-sensitive phenotype) correlated with better outcomes. Importantly, altering one axis of plasticity often drove coordinated responses along other axes and vice versa. Our findings establish phenotypic plasticity in cancer as a coordinated, multi-axis dynamical process, thus suggesting novel strategies to disrupt systems-level reprogramming enabling metastasis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acaf4b9f2872701e5c44aad69b0ee83629b9228" target='_blank'>
              Interconnected axes of phenotypic plasticity drive coordinated cellular behaviour and worse clinical outcomes in breast cancer
              </a>
            </td>
          <td>
            Ritesh Kumar Meena, Yijia Fan, Soundharya Ramu, Anjaney J. Pandey, Abhinav Kannan, Yashvita Subramanian, Yukta Subramanian, J. T. George, M. Jolly
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09e91ffccc5754debd09b64e59ef74200817ed97" target='_blank'>
              Transcriptomic and epigenetic profiling of neuroblastoma states in response to RBM39 degrader.
              </a>
            </td>
          <td>
            Hongjian Jin, Jie Fang, Jason Myers, Shivendra Singh, Jun Yang
          </td>
          <td>2025-12-09</td>
          <td>Scientific data</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b82ec4f52026d1218d8a25c579cd530acc17905" target='_blank'>
              Widespread remodeling of the RNA editome underlies transcriptional and clinical heterogeneity in pediatric acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sara Rosenthal, Thomas Whisenant, Roman Sasik, Maria Rivera, Geena Il Defonso, Charles-Alexandre Roy, Sabina Enlund, R. Shraim, Kathleen M. Fisch, D. Teachey, Frida Holm, D. Kuo, Qingfei Jiang
          </td>
          <td>2025-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="We evaluate the effect of most FDA-approved drugs (>7,000 conditions) on double-strand DNA break repair pathways by analyzing mutational outcomes in human induced pluripotent stem cells. We identify drugs that can be repurposed as inhibitors and enhancers of repair outcomes attributed to non-homologous and microhomology-mediated end joining (NHEJ, MMEJ), and homology-directed repair (HDR). We also identify functions of the proteins estrogen receptor 2 (ESR2) and aldehyde oxidase 1 (AOX1), affecting several key DNA repair proteins, such as ATM and 53BP1. Silencing of ESR2 can have a synergistic effect on increasing HDR when combined with NHEJ inhibition (mean 4.6-fold increase). We further identify drugs that induce synthetic lethality when NHEJ or HDR is blocked and may therefore be candidates for precision medicine. We anticipate that the ability to modulate the DNA repair outcomes with clinically safe drugs will help disease modeling, gene therapy, chimeric antigen receptor immunotherapy, and cancer treatment. DNA repair pathways shape CRISPR editing outcomes. Here, authors identified FDA approved drugs that can be repurposed as repair modulators or to induce synthetic lethality, and uncovered new roles for ESR2 and AOX1 in DNA repair, enhancing editing and offering potential therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6eea628e8acb5e08c2cd2c0b97b3bdae99bd71" target='_blank'>
              Repurposing clinically safe drugs for DNA repair pathway choice in CRISPR genome editing and synthetic lethality
              </a>
            </td>
          <td>
            Dominik Macak, Philipp Kanis, S. Riesenberg
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d63cfbf90a2e544c2d24c8ab605e5d44e8ba24" target='_blank'>
              A Pan-Cancer Single-Cell Compendium of Intratumoural Heterogeneity
              </a>
            </td>
          <td>
            Ido Nofech-Mozes, Ashton Cook, Tom W. Ouellette, Sara Hafezi-Bakhtiari, Philip Awadalla, Sagi Abelson
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Motivation Understanding double-strand break (DSB) repair and its disruption is key to decipher genomic instability driving diseases such as cancer and reveal therapeutic avenues. Numerous genes have been linked to DSB repair for the first time in recent genome-wide perturbation studies assessing effects on mutational outcomes. However, the functional roles of most such genes remain poorly understood. Evidence from other studies shows that related genes similarly modulate the frequency of specific mutational outcomes following DSB repair, but analysis has largely ignored the multiplicity of gene functions and cross-talk between pathways. Here, we infer functional roles for candidate genes based on knockout mutational spectra by identifying mutational signatures shared with known genes and then mapping them to DSB repair functions. Signatures are identified using non-negative matrix factorization (NMF) to reflect functional multiplicity and cross-talk. Results We generated mutational spectra for mouse embryonic stem (mES) cells at three target sites across CRISPR knockouts of 307 known and 459 candidate DSB repair genes. Analysis using NMF revealed four mutational signatures associated with homology-directed repair (HDR), microhomology-mediated end-joining (MMEJ), and the initiation and ligation components of non-homologous end-joining (NHEJ). Signatures suggested that candidate genes Dbr1 and Hnrnpk could influence MMEJ and Fanconi anaemia (FA), and that loss of core NHEJ components (e.g. MRN complex or Ku proteins) could shift repair preference towards Ku-independent NHEJ. These findings demonstrate the utility of NMF for characterizing the contribution of genes to repair pathways and provide a foundation to discover new gene functions in DSB repair. Availability github.com/joanagoncalveslab/MuSpectraNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a5274e29358d261ef02cb6d42094a3acbdab40" target='_blank'>
              Signatures in CRISPR Mutational Spectra Reveal Role and Interplay of Genes in DNA Repair
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c23f53baca0dc3b384bd4e216c9b8503ff2be41" target='_blank'>
              Pooled single-cell CRISPRa/i screens for functional genomics in bacteria at scale
              </a>
            </td>
          <td>
            Jacob R. Brandner, Quoc Tran, Yujia Huang, Dmitry Sutormin, Karl D. Gaisser, Georg Seelig, J. Zalatan, James M. Carothers, A. Kuchina
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db26ccc6dc7e76ac6a63d600396d97f0ca6b89f2" target='_blank'>
              Passenger mutations link cellular origin and transcriptional identity in human lung adenocarcinomas
              </a>
            </td>
          <td>
            Sukanya Panja, Padmaja Mantri, K. E. Johnson, J. Andrade-Martínez, S. Yang, Aditya Deshpande, Huasong Tian, Shaham Beg, Kentaro Ohara, Alessandro Leal, Joel Rosiene, Marlon Stoeckius, Peter Smibert, William D. Travis, J. Mosquera, P. Polak, Marcin Imieliński
          </td>
          <td>2025-11-27</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Neuroblastoma (NB), the most common extracranial solid tumor in children, is characterized by remarkable cellular heterogeneity and clinical variability ranging from spontaneous regression to aggressive progression and relapse. Despite advances in multimodal therapies, including surgery, chemotherapy, radiotherapy, differentiation therapy, and immunotherapy—treatment resistance remains the principal barrier to improving survival in high-risk patients. Recent single-cell and spatial multi-omics studies have revolutionized our understanding of NB by revealing its developmental origins, lineage hierarchy, and adaptive evolution under therapeutic pressure. These technologies have delineated distinct cellular states along an adrenergic–mesenchymal continuum and uncovered the dynamic interplay between tumor cells and their microenvironment. Genetic instability, epigenetic reprogramming, and metabolic plasticity cooperate with immune and stromal remodeling to drive tumor persistence and relapse. At the molecular level, mechanisms such as MYCN-driven chromatin remodeling, super-enhancer reorganization, bypass signaling activation, quiescent persister programs, immune checkpoint engagement, and metabolic rewiring collectively enable therapeutic escape. Importantly, these processes are reversible, highlighting tumor plasticity as both a hallmark and a potential vulnerability of NB. Integrating single-cell transcriptomics, epigenomics, and spatial profiling provides an unprecedented framework to map resistance evolution, identify lineage-specific vulnerabilities, and guide rational combination strategies. Targeting epigenetic regulators, metabolic checkpoints, and immune suppressive networks in a temporally coordinated manner holds promise for converting NB from an adaptive to a controllable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/182327417f0bfd0539120b0fcc3161f6ade537ce" target='_blank'>
              Dissecting neuroblastoma heterogeneity through single-cell multi-omics: insights into development, immunity, and therapeutic resistance
              </a>
            </td>
          <td>
            Guoqian He, Sijia He, Xiao-yu Jing, Yi-Ling Dai, Xia Guo, Ju Gao, Wei Zhang
          </td>
          <td>2025-11-27</td>
          <td>Oncogene</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="The tumour suppressor TP53 (tumor protein p53) is a master regulator of cell cycle, DNA repair, and apoptosis, and its mutation is a hallmark of cancer, with individual mutations exerting distinct effects on tumour biology. Despite accounting for ~7% of all TP53 variants, splice site mutations remain the least studied class, and their functional and clinical consequences are poorly understood. We analyzed 25,058 TP53 variants (18,562 somatic; 6496 germline) to characterize the frequency, molecular impact, transcriptional effects, genomic instability, and clinical outcomes of splice mutations. These alterations showed distinct distributions and substitution patterns between germline and somatic contexts and were frequently associated with copy number alterations, reduced TP53 mRNA, and variable protein expression. Transcriptomic profiling identified two transcriptional phenotypes: one with global suppression of canonical p53 target genes and another with mixed activation and repression independent of tumour type. Genomic instability was elevated in a subset of splice-mutant tumours, correlating with increased relapse risk, while other splice mutations showed lower instability but divergent clinical outcomes, including unexpectedly poor prognoses. Our findings fill a critical knowledge gap, defining the biological and clinical spectrum of TP53 splice site mutations and highlighting their potential as prognostic biomarkers and therapeutic targets in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df743368c74812062cf79c30615a68f562fc982" target='_blank'>
              TP53 Splice Mutations Have Tumour-Independent Effects on Genomic Stability and Prognosis: An In Silico Study
              </a>
            </td>
          <td>
            Apeksha Arun Bhandarkar, Noah Ethan Kelly-Foleni, Debina Sarkar, Aaron Jeffs, T. Slatter, Antony Braithwaite, Sunali Y Mehta
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4393b9168f5ebc4a80f332a0eba92fab9bfeef3" target='_blank'>
              Single-Cell Multiomic Profiling Uncovers Radiation Dosage-Sensitive, Cluster-Specific Regulatory Dynamics in Glioblastoma
              </a>
            </td>
          <td>
            Khoi Huynh, Sara Nalina Barcik Weissman, Cheol Park, Maria L.G. Quiloan, Blake Sun Chang, Eric L Chang, Kelly Street, David Tran, A. Modrek
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Lynch syndrome (LS) is a colorectal cancer predisposition caused by an inherited heterozygous defect in any of four DNA mismatch repair (MMR) genes. MMR prevents nucleotide substitution mutations by correcting errors of replication opposite undamaged or subtly altered nucleotides. Here, we investigated whether dietary mutagens, which generally induce helix-distorting nucleotide lesions, affect LS-associated carcinogenesis. To this aim, we exposed mouse models of LS to 2-amino-1-methyl-6-phenylimidazo[4,5-b]pyridine (PhIP), a food-derived heterocyclic amine that selectively adducts guanines. PhIP exposure induced loss of the wild-type MMR allele in heterozygous intestinal stem cells, leading to MMR deficiency and to impaired DNA damage signalling associated with the clonal expansion of MMR-deficient intestinal stem cells. Whole-genome sequencing revealed that PhIP becomes significantly more mutagenic in intestinal stem cells when MMR is lost, inducing not only PhIP-guanine adduct-mediated C:G>A:T transversions but also a broader substitution spectrum that resembles the spontaneous mutational signature of LS-associated colorectal cancer. Thus, MMR corrects PhIP-induced misincorporations outside of adducted guanines. Chronic PhIP exposure of intestine-specific MMR-deficient mice induced adenocarcinomas with histopathological features of LS-associated CRC. This study implicates food-derived mutagens in multiple stages of LS-associated carcinogenesis, including allelic loss, and defective DNA damage signalling and compound hypermutagenesis in the resulting MMR-deficient cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19723251719a4b38892f07464455ad109fdcc89a" target='_blank'>
              A common food mutagen promotes intestinal carcinogenesis by multiple mechanisms in mouse models of Lynch syndrome
              </a>
            </td>
          <td>
            Emily Rayner, Anastasia Tsaalbi-Shtylik, Lemelinda Marques, Cedrick Agaser, Hailian Mei, D. Cats, Sophie van Zanen-Gerhardt, Daniela Salvatori, Niels de Wind
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Replicative immortality is a hallmark of cancer, driven by the activation of telomere maintenance mechanisms, that is yet to be therapeutically exploited. To expedite discoveries that will enable the development of therapeutics that target telomere maintenance mechanisms, this study provides a resource of telomere biology metrics for a pan-cancer panel of 976 cell lines. We generate proteomic data from data-independent-acquisition mass spectrometry for most of these cell lines and integrate pre-existing multi-omic, drug sensitivity, and molecular dependency data from CRISPR/Cas9 knock-out screens. The data illustrate a broad range and heterogeneity in telomere biology, including states that diverge from the binary model of telomere maintenance activation involving either telomerase or the Alternative Lengthening of Telomeres mechanism. Using the telomere biology metrics and multi-omic data, we derive proteomic and transcriptomic predictors of Alternative Lengthening of Telomeres and telomerase activity levels. Our investigations also reveal molecular vulnerabilities associated with the Alternative Lengthening of Telomeres mechanism and drug sensitivity correlating with telomerase activity levels. These findings illustrate opportunities for leveraging this resource to realize the potential for telomere biology-directed cancer therapeutics and companion diagnostics. The therapeutic relevance of telomere maintenance mechanisms in cancer, remains to be explored. Here, the authors integrate multi-omic data and functional readouts, generate a resource of telomere biology metrics and identify potential molecular vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc8e08e7ae63f123dc686a0aa51e8c03fa2c2c5" target='_blank'>
              Large-scale drug sensitivity, gene dependency, and proteogenomic analyses of telomere maintenance mechanisms in cancer cells
              </a>
            </td>
          <td>
            Yangxiu Wu, Zhaoxiang Cai, Dale Cross, Jane R. Noble, Kelsy Prest, Jamie Littleboy, Scott B. Cohen, Baylee Edlundh, Jennifer M. S. Koh, Ran Xu, Zainab Noor, Milad Bastami, Sara Valentini, Laura Richardson, S. Barthorpe, Nader Arymanesh, Phillip J. Robinson, P. Hains, Mathew J. Garnett, Qing Zhong, R. Reddel, Karen L MacKenzie
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background: Radiotherapy resistance in breast cancer remains a major clinical challenge. The key molecular determinants and cellular populations driving this resistance are not fully understood. Methods: A radiotherapy resistance (RR) gene panel was identified from TCGA-BRCA and GSE120798 cohorts. Single-cell and spatial transcriptomics characterized RRhigh epithelial cells (RRhighepi). A prognostic model, named SuperPC and StepCox-based Radiotherapy Resistance model (SSRR), was built via machine learning and Mendelian randomization. Functional roles of Prolyl 4-Hydroxylase Subunit Alpha 2 (P4HA2) were validated in vitro. Results: The RR gene panel was upregulated in tumors and enriched for cell cycle pathways. RRhighepi cells exhibited elevated stemness, activated cell cycle and metabolic programs, and enhanced DNA damage repair. RRhighepi represented a developmental origin and communicated with endothelial cells. The SSRR model stratified patients into high-risk groups with poorer survival and distinct therapeutic responses. P4HA2, a key model gene, was upregulated in multiple cancers. P4HA2 knockdown suppressed proliferation, invasion, and colony formation, and synergized with radiotherapy to reduce stemness and enhance DNA damage. WGCNA confirmed co-module membership of P4HA2 and the RR panel. Conclusions: This study, through multi-omics analysis, proposes a potential mechanistic model associated with radiotherapy resistance in breast cancer. P4HA2 is a potential therapeutic target that sensitizes breast cancer to radiotherapy. The RR gene panel and SSRR model provide insights into resistance mechanisms and prognostic stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/166d8bb1b89b913023588472f1dc07c1ab7e8bdf" target='_blank'>
              Single-cell and spatial transcriptomics reveal P4HA2-mediated radiotherapy resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Huimin Li, Junzhi Liu, Yuheng Jiao, Fengyu Xu, Shurui Wang, Qiang Tang
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transposable elements (TEs) are key agents of genome evolution across all domains of life. These mobile genetic elements can cause mutations through transposition or by promoting structural rearrangements. Stress conditions can amplify TE mobility, either by impairing TE suppression mechanisms or through stress-induced interactions between transcription factors and TE sequences, offering a route for rapid genetic change. As such, TEs represent an important source of adaptability within populations. To investigate the interplay between environmental stress and eukaryotic TE dynamics relevant to infectious disease, we examined how heat stress and host-mimicking medium (RPMI) affect TE mobility in the global human fungal pathogen Cryptococcus neoformans, using a collection of clinical and environmental isolates. Using a selection-based screen, we captured the mobilization of seven distinct mobile element families, encompassing diverse retrotransposons and DNA transposons, whose insertions conferred antifungal resistance. This includes a novel element, CNEST, which belongs to the CACTA, Mirage, Chapaev (CMC) supergroup. Heat stress at human body temperature (37°C) significantly increased the mobilization of a subset of these TEs, leading to higher rates of acquired antifungal resistance. Whole-genome assemblies revealed that, compared to retrotransposons, DNA transposons were hypomethylated and approximately uniformly distributed throughout the genome, features that may contribute to their frequent mobilization. We further assessed TE-driven genomic changes within hosts using serial isolates from patients with recurrent cryptococcal infections and from isolates passaged through mice. While we observed evidence of TE copy number changes near chromosome ends, we found no indication of TE-mediated alterations near gene-coding regions across any of the serial isolates. Finally, TE mobility was isolate- and strain-dependent, with significant variation even among clonally related strains collected from the same patient, emphasizing the role of genetic background in shaping TE activity. Together, these findings reveal a complex and dynamic relationship between environmental stress, genetic background, TE type-specific epigenetic regulation, and TE mobility, with important implications for adaptation and acquired antifungal resistance in C. neoformans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d02f41bc08d41c479d1ccb35a44f0d08249ddd1" target='_blank'>
              Temperature and genetic background drive mobilization of diverse transposable elements in a global human fungal pathogen
              </a>
            </td>
          <td>
            Anna I. Mackey, Vesper M. Fraunfelter, Samantha Shaltz, John McCormick, Callan Schroeder, John R. Perfect, Cédric Feschotte, Paul M. Magwene, Asiya Gusa
          </td>
          <td>2026-01-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Epigenetic drift is a key feature of aging and is associated with age-related diseases including cancer, yet the underlying molecular mechanisms remain unclear. Here, by analyzing DNA methylation and gene expression data from healthy and cancerous human colon samples, we identify an aging and colon cancer-associated DNA methylation (DNAm) drift. We find evidence that this drift is conserved in the mouse intestinal epithelium, where we demonstrate its origin within intestinal stem cells and identify its cell-intrinsic and non-mitotic characteristics, finding that its expansion is regulated via crypt clonality and fission. Mechanistically, we find that this drift is driven by age-related inflammation and reduced Wnt signaling, which dysregulate iron metabolism and impair TET activity. Despite CpG-level heterogeneity, we find that DNAm changes are consistent at the gene level, suggesting potential functionality. Our findings shed light on the epigenetic mechanisms of aging and provide a mechanistic basis for the hypermethylation observed in cancer. Krepelova and colleagues identify an aging- and colon cancer-associated DNA methylation drift in healthy and cancerous human colon samples, echoed in the mouse intestinal epithelium. They show it expands via intestinal cell clonality and is driven by dysregulated iron metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b850c3e1cece55c27c10ab9cc07854155c6241a0" target='_blank'>
              Iron homeostasis and cell clonality drive cancer-associated intestinal DNA methylation drift in aging
              </a>
            </td>
          <td>
            Anna Krepelova, Mahdi Rasa, Francesco Annunziata, Jing Lu, Chiara Giannuzzi, Omid Omrani, Elisabeth Wyart, P. Porporato, Ihab Ansari, Dor Bilenko, Y. Bergman, Francesco Neri
          </td>
          <td>2025-11-26</td>
          <td>Nature Aging</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Epithelial compartments play a central role in the progression and metastasis of colorectal cancer (CRC), yet the mechanisms underlying their transcriptional reprogramming across disease stages remain incompletely understood. We constructed comprehensive single-cell and spatial transcriptomic atlases encompassing normal colorectal tissue, primary CRC, and liver metastases (LMs), profiling over 200,000 high-quality cells. Specific coexpression modules and transcriptional regulators were identified and spatially mapped. Additionally, the clinical and functional roles of SCAND1 were investigated using western blot, RT-qPCR, and immunohistochemistry in clinical samples, as well as functional assays in CRC cell lines. Distinct coexpression modules were selectively enriched in normal epithelial cells and associated with vesicle transport, ion homeostasis, and barrier function, whereas LM-specific modules were linked to Wnt signalling, ribosome biogenesis, and cell cycle processes. High-CNV epithelial cells exhibited progressive activation of key transcription factors such as CEBPB, E2F, and the AP-1 family, with concurrent suppression of ARGLU1 and MTNR2L8, indicating coordinated remodelling of transcriptional and metabolic programs during metastasis. These changes were spatially validated and correlated with alterations in epithelial composition within the tumour core. Consistently, multiplex immunofluorescence (mIHC) confirmed the expression patterns of CEBPB, ARGLU1, S100A4 and FOSL1, across normal epithelium, CRC, and LM. Notably, SCAND1 was identified as a potential therapeutic target for malignant epithelial plasticity. We demonstrated for the first time that SCAND1 expression is significantly upregulated in CRC patients and is associated with superior diagnostic and prognostic value. Functional experiments revealed that SCAND1 promotes tumour cell proliferation, inhibits apoptosis, and enhances metastatic potential. Furthermore, co-culture of SCAND1-overexpressing tumour cells with T cells showed that SCAND1 overexpression attenuates T cell–mediated cytotoxicity, supporting an additional immune-evasive role. Our study elucidates the molecular logic of epithelial state transitions during CRC metastasis and underscores the promise of SCAND1 as a novel biomarker and therapeutic target in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/914a9adf8fcce69d1ec7290317d881a8d12ed794" target='_blank'>
              Dynamic remodelling of epithelial plasticity in colorectal cancer from single-cell and spatially resolved perspectives
              </a>
            </td>
          <td>
            Chengyuan Xu, Yun Pan, Siqi Zhang, Moubin Lin, Hailong Liu, Zicheng Yu
          </td>
          <td>2025-11-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: Achieving complete resection (R0) during secondary cytoreduction (SCR) is a pivotal prognostic factor for patients with platinum-sensitive relapsed ovarian cancer (PSROC). Methods: By integrating single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and functional assays on primary cells, we identified tumor microenvironment (TME) heterogeneity, particularly the presence of distinct subpopulations of cancer-associated fibroblasts (CAFs), as a critical determinant of surgical outcomes for SCR. Results: We characterized a previously unrecognized CAF-C7 subpopulation that was selectively enriched in tumors from non-R0 patients. Mechanistically, CAF-C7 promoted tumor cell migration, suppressed anoikis, and facilitated angiogenesis via the IGF1-IGF1R signaling axis, thereby contributing to multifocal recurrence and reducing the likelihood of complete resection. Importantly, the inhibition of the IGF1-IGF1R pathway effectively attenuated the pro-tumorigenic functions of CAF-C7. Conclusions: These findings uncover a novel cellular driver of unfavorable surgical outcomes in PSROC and suggest promising biomarkers and therapeutic targets for improving patient stratification and treatment in the context of SCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c69a7316901a4f26ec9560008414f1bd2cba29" target='_blank'>
              Single-Cell Analysis Links C7+ Cancer-Associated Fibroblasts with Incomplete Resection in Platinum-Sensitive Relapsed Ovarian Cancer
              </a>
            </td>
          <td>
            Longxia Li, Shangbing Gao, Yilizhati Maimaiti, Lifeng Lin, Xiaoxia Xing, Wei Wu, Yulian Chen, Mei-Chun Cai, Guanglei Zhuang, Rongyu Zang
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="CRISPR/Cas9 technologies provide unique capabilities for modeling disease and understanding gene-to-phenotype connections. In cultured cells, chemical-mediated control of Cas9 activity can limit off-target effects and enable mechanistic study of essential genes. However, widely-used Tet-On systems often show leaky Cas9 expression, leading to unintended edits, as well as weak activity upon induction. Leakiness can be problematic in the context of Cas9 nuclease activity, which may result in cumulative DNA damage and degradation of the target cell genome over time. To overcome these deficiencies, we have established transgenic platforms that minimize Cas9 functionality in the OFF-state along with maximized and uncompromised ON-state gene editing efficiency. By combining conditional destabilization and inhibition of Cas9, we have developed an all-in-one (one or multiple guide RNAs and Cas9) ultra-tight, Tet-inducible system with exceptional dynamic range (ON vs. OFF-state) across various cell lines and targets. As an alternative to Tet-mediated induction, we have created a Branaplam-regulated splice switch module for low-baseline and robust Cas9 activity control. Lastly, for circumstances where DNA damage needs to be avoided, we have constructed a dual-control, Tet-inducible CRISPRi module for tight and potent transcriptional silencing. This upgraded suite of inducible CRISPR systems has broad applications for numerous cell types and experimental conditions. Achieving tight Cas9 regulation without sacrificing activity remains difficult. Here, the authors design multi-level circuits combining anti-CRISPRs, splice sites, chemical induction, and degron control to enable ultra-high dynamic range and precise, on-demand genome editing across contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4cc5c8aa5f90027fd142622f22700e2cb43c085" target='_blank'>
              Chemically-inducible CRISPR/Cas9 circuits for ultra-high dynamic range gene perturbation
              </a>
            </td>
          <td>
            Rajini Srinivasan, Tao Sun, Azalia Sandles, Diana Wu, Liang Wang, Hetal Patel, Ryan Pabalate, Maram Bader, A. Heidersbach, Clark Ho, Shiqi Xie, Andrew Ng, Benjamin Haley
          </td>
          <td>2025-12-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Bladder cancer (BLCA) is the predominant type of urothelial carcinoma in urinary system, and resistance to cisplatin-based chemotherapy substantially worsens clinical outcomes, presenting a major therapeutic obstacle. In this study, we integrated genome-wide association study (GWAS) data with expression quantitative trait loci (eQTL) analyses, and applied Mendelian randomization (MR) to assess the causal effects of eQTLs from 19,942 genes on BLCA. By incorporating scRNA-seq data, our study also identifies differentially expressed genes (DEGs) in cisplatin-resistant BLCA cells and examined their causal associations with BLCA, aiming to elucidate genetic drivers of chemoresistance and tumor progression. Through this integrated approach, we identified the eQTL of the ARHGEF12 gene as a key mediator of cisplatin resistance. Bioinformatic analysis revealed that elevated ARHGEF12 expression was strongly associated with activation of the PI3K/Akt signaling pathway. To define ARHGEF12’s role in cisplatin resistance, we established a cisplatin-resistant UM-UC-3/DDP model. Silencing ARHGEF12 markedly reduced chemoresistance, increased apoptotic cell death, and induced pronounced morphological changes. Pharmacological modulation with the ROCK inhibitor Y-27632 and a rescue assay with the Akt activator SC79 supported a model in which ARHGEF12 drives chemoresistance via RhoA/ROCK-dependent activation of the PI3K/AKT axis. This study is the first to integrate MR with single-cell transcriptomics to explore the genetic contribution to cisplatin resistance in BLCA. Our results uncover a novel mechanistic role of ARHGEF12 in BLCA progression and chemoresistance and suggest it as a potential therapeutic target for precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/542b443be27492e921709c204aeb3560eab221c4" target='_blank'>
              Integrative multi-omics and single-cell transcriptomics reveal ARHGEF12 driving chemoresistance in bladder cancer
              </a>
            </td>
          <td>
            Kunyao Zhu, Zhejiao Zhang, Tinghao Li, Yan Sun, Junlong Zhu, Linfeng Wu, Xiaoyu Zhang, Hang Tong, Zijia Qin, Aimin Wang, Weiyang He
          </td>
          <td>2025-11-27</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lineage plasticity, the ability of cancer cells to alter their differentiated state through transcriptional and epigenetic reprogramming, has emerged as a key mechanism of therapeutic resistance across cancers. This adaptive process can manifest in multiple ways, including epithelial–mesenchymal transition, acquisition of stem-like features, and histological transformation, the most striking and clinically apparent example. In EGFR-mutant lung adenocarcinoma (LUAD), lineage plasticity is increasingly recognized as a prevalent mechanism of acquired resistance to tyrosine kinase inhibitors (TKIs). Among its visible manifestations, histologic transformation into small-cell lung cancer (SCLC) is the most frequent, while squamous transformation and other phenotypic shifts also occur. Transformed tumors typically retain the initiating EGFR mutation but lose EGFR dependence, acquire neuroendocrine features, and display aggressive clinical behavior with poor clinical outcomes compared with both de novo SCLC and non-transformed LUAD. Recent studies show that plasticity arises through combined genomic, transcriptomic, and epigenetic reprogramming, often foreshadowed by molecular alterations before overt histological change. Spatial and single-cell profiling reveal heterogeneous trajectories and intermediate states, while functional models and multi-omics approaches have begun to identify therapeutic vulnerabilities distinct from both de novo EGFR-mutated SCLC and classical EGFR-mutated LUAD. Thus, lineage plasticity, whether manifested as histologic transformation or through more subtle epigenetic reprogramming, represents a formidable resistance mechanism in NSCLC. Defining its molecular basis and temporal dynamics will be essential for early detection, prognostication, and the development of tailored therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99eda567a0f2b2f5e4b0555a0ae32377856dcb42" target='_blank'>
              Lineage Plasticity and Histologic Transformation in EGFR-TKI Resistant Lung Cancer
              </a>
            </td>
          <td>
            Li Yieng Eunice Lau, A. Skanderup, Aaron C. Tan
          </td>
          <td>2025-12-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Antibiotic resistance (AR) has long been interpreted through the lens of genetic mutations and horizontal gene transfer. Yet, mounting evidence suggests that epigenetic regulation, including DNA and RNA methylation, histone-like proteins, and small non-coding RNAs, plays a similarly critical role in bacterial adaptability. These reversible modifications reshape gene expression without altering the DNA sequence, enabling transient resistance, phenotypic heterogeneity, and biofilm persistence under antimicrobial stress. Advances in single-molecule sequencing and methylome mapping have uncovered diverse DNA methyltransferase systems that coordinate virulence, efflux, and stress responses. Such epigenetic circuits allow pathogens to survive antibiotic exposure, then revert to susceptibility once pressure subsides, complicating clinical treatment. Parallel advances in CRISPR-based technologies now enable direct manipulation of these regulatory layers. CRISPR interference (CRISPRi) and catalytically inactive dCas9-fused methyltransferases can silence or reactivate genes in a programmable, non-mutational manner, offering a new route to reverse resistance or sensitize pathogens. Integrating methylomic data with transcriptomic and proteomic profiles further reveals how epigenetic plasticity sustains antimicrobial tolerance across environments. This review traces the continuum from natural bacterial methylomes to engineered CRISPR-mediated epigenetic editing, outlining how this emerging interface could redefine antibiotic stewardship. Understanding and targeting these reversible, heritable mechanisms opens the door to precision antimicrobial strategies that restore the effectiveness of existing drugs while curbing the evolution of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5548701c56565e9287ebf40d194a1cd7f296a9ed" target='_blank'>
              From Methylomes to CRISPR Epigenetic Editing: New Paths in Antibiotic Resistance
              </a>
            </td>
          <td>
            Nada M. Nass, Kawther A. Zaher
          </td>
          <td>2025-12-01</td>
          <td>Pathogens</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fab0829fa668d3f68087f51374ae0be2ba943151" target='_blank'>
              Perturbational single-cell profiling of patient tumors defines lineage- and context-specific programs of innate immune resistance
              </a>
            </td>
          <td>
            C. Perry, A. Frey, Y. Fei, J. Wang, P. Costa, H. N. Amin, S. Ghadermarzi, D. Levine, W. Tong, A. Koda, M. Mackie, M. He, Y. Nie, K. Clulo, F. Ouerghi, J. Wei, T. Cordero Dumit, M. Yaskolko, M. Ding, A. Caldera, O. Kyrysyuk, L. Lum, J. W. Allen, W. Guo, A. Naqash, A. Elliot, A. Vanderwalde, M. Capelletti, T. Adeyelu, S. Barry, M. Hugaboom, A. Bacchiocchi, H. Kluger, M. Bosenberg, A. Iwasaki, D. Braun, J. Clune, D. van Dijk, K. Olino, J. Ishizuka
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Cancer cells precisely modulate replication stress to sustain genomic instability without triggering lethal DNA damage, yet regulators enabling this delicate balance remain largely unknown. Here, we identify N-methyl-D-aspartate receptor synaptonuclear signaling and neuronal migration factor (NSMF) as a novel and critical regulator of replication stress in colorectal cancer (CRC). NSMF expression is significantly elevated in CRC tissues and correlates closely with elevated replication stress. In ApcMin/+ mouse models, Nsmf knockout selectively induces replication-dependent DNA damage in tumor tissues, suppressing tumor growth and prolonging survival, without harming normal tissues. Mechanistically, NSMF deficiency impairs replication fork progression under stress conditions, resulting in DNA damage accumulation, growth arrest, and senescence. Conversely, NSMF overexpression provides resistance to oncogene-induced replication stress, enabling cancer cells to evade senescence and sustain proliferation. These findings establish NSMF as an essential safeguard against lethal replication stress and highlight its potential as a promising therapeutic target for CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75ccb80f68a23ffd6795b23a21df4811bf439f1c" target='_blank'>
              NSMF modulates replication stress to facilitate colorectal cancer progression
              </a>
            </td>
          <td>
            Kyeong Jin Shin, Yu Jin Lee, Gyuri Kim, Seongjun You, Kyung Lock Kim, Sabin Park, KyoungJun Lee, Eun A Lee, Do Hong Park, Hui Won Kim, Sanga Choi, Sung Min Jung, Jung-Hyun Kim, Jang Hyun Choi, Hyung-Sik Kim, Kyungjae Myung, Semin Lee, Seung-Jae Myung, Young Chan Chae
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy where traditional DNA sequence-based classification fails to fully explain clinical outcomes. This review explores the dynamic and multifaceted role of the epigenome as a critical driver of leukemogenesis, progression, and therapeutic resistance. We detail how disruptions in DNA methylation, histone modifications, chromatin architecture, and non-coding RNAs create a complex regulatory landscape that transcends genetic alterations. We highlight the emergence of single-cell multi-omics technologies, which are resolving intratumoral heterogeneity and uncovering the epigenetic signatures of leukemic stem cells. Furthermore, we discuss the significant translational potential of these discoveries, including the development of epigenetic biomarkers for refined risk stratification, minimal residual disease (MRD) monitoring, and predicting response to novel epigenetic therapies like menin and LSD1 inhibitors. Integrating these dynamic epigenetic layers into clinical decision-making promises to usher in a new era of precision medicine, ultimately improving prognostic accuracy and therapeutic outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3853f768df9c1d2f10b0fd3237fd7635474d26" target='_blank'>
              Beyond the DNA sequence: mapping the dynamic epigenetic landscape for risk stratification and therapeutic intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jiqian Xie, Hamed Soleimani Samarkhazan
          </td>
          <td>2025-12-16</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="C1orf50 encodes a small, evolutionarily conserved protein, the function of which remains unclear. Its significance across various human cancers, particularly its specific role in ovarian cancer within an immunogenomic context, is not yet fully understood. Utilizing The Cancer Genome Atlas and single-cell RNA sequencing (scRNA-seq) public datasets, we conducted a comprehensive profiling of C1orf50 across multiple cancer types, with a particular focus on ovarian cancer, to investigate its associations with copy-number status, genomic instability, tumor programs, and the immune microenvironment. Across cancer types, copy-number gain or amplification of C1orf50 was most frequent in ovarian cancer and closely tracked with higher messenger RNA levels. Higher C1orf50 expression was associated with a greater tumor mutational burden and homologous recombination deficiency, as indicated by gene-set patterns that suggested heightened cell-cycle and cellular stress responses accompanied by reduced oxidative phosphorylation, enrichment of regulatory T cells, and depletion of resting memory CD4 T cells. In ovarian cancer, focal events at chromosome 1p34.2 were accompanied by stepwise increases in C1orf50 expression by clinical stage and were linked to higher tumor mutational burden, homologous recombination deficiency, and greater loss of heterozygosity, together with more frequent gene alterations in BRCA1 or BRCA2. Immune composition clustered into profiles consistent with an immunosuppressive context in tumors with higher C1orf50 expression. The scRNA-seq data further revealed that cancer cells enhanced immune-suppressive interactions with various immune cell populations and diminished antigen-presentation signals. Analyses of genomic instability in ovarian cancer suggested mutational processes compatible with base-substitution patterns associated with cytidine deaminase activity and with insertion-deletion patterns characteristic of homologous recombination failure, while transcript-level patterns pointed to a broad downshift of canonical DNA repair activity with apparent compensatory adjustments in related pathways rather than a uniform change in any single pathway. The overexpression of C1orf50 characterizes an aggressive immunogenomic phenotype in ovarian cancer, distinguished by genomic instability, impaired DNA repair mechanisms, and extensive immunosuppression. These findings indicate that C1orf50 warrants consideration as a potential biomarker and a prospective target for therapeutic investigation. Furthermore, they advocate for the progression to prospective validation and functional studies to ascertain its clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aef46ec0fd0bf3582b841375c94b60ec801575fd" target='_blank'>
              Pan-cancer profiling links C1orf50 to DNA repair and immune modulation in ovarian cancer
              </a>
            </td>
          <td>
            Anna Rogachevskaya, Yusuke Otani, Akira Ohtsu, Vanessa D. Chin, Tirso Peña, Seiji Arai, Shinichi Toyooka, Atsushi Fujimura, Atsushi Tanaka
          </td>
          <td>2025-12-08</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) remains clinically challenging due to its molecular heterogeneity and poor outcomes, highlighting the urgent need for novel biomarkers and therapeutic targets. Purpose This study aims to identify and characterize the role of leucine-rich α-2-glycoprotein 1 (LRG1) in AML, evaluating its potential as both a prognostic biomarker and a therapeutic target. Methods We conducted an integrated basic and clinical investigation of LRG1 in AML. Methods included analysis of LRG1 expression in patient samples versus controls and pre- versus post-treatment, assessment of its clinical correlations with mutations and subtypes, and evaluation of its prognostic impact. Functional validation was performed using LRG1 knockdown models to assess effects on colony formation, apoptosis, and differentiation. Single-cell RNA profiling was utilized to identify LRG1-enriched cell populations and explore its role in microenvironmental crosstalk. Results Integrated analysis revealed significantly elevated LRG1 expression in AML patients compared to controls (P<0.001), with levels decreasing post-treatment (P<0.001). High LRG1 expression correlated with FLT3 mutations (P<0.01), M3-M5 AML subtypes (M0&M1&M2 VS M3, P<0.001; M0&M1&M2 VS M4, P<0.01; M0&M1&M2 VS M5, P<0.001; M3 VS M5, P<0.05; M4 VS M5, P<0.05), and worse survival (P<0.01). Functionally, LRG1 knockdown impaired colony formation (P<0.001), increased apoptosis (P<0.001), and disrupted differentiation (P<0.01). Single-cell profiling identified LRG1 enrichment in hematopoietic stem and progenitor cells (HSPCs) and myeloid progenitors, where it facilitated microenvironmental crosstalk via Macrophage Migration Inhibitory Factor (MIF), Galactoside-binding lectin (GALECTIN), and Cyclophilin A (CypA) signals. Conclusion Our findings establish LRG1 as a robust prognostic biomarker and a key functional regulator of AML maintenance through myeloid progenitor dysregulation, presenting it as a promising target for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c85b059f315aeac16b816e67ba97682c83e790" target='_blank'>
              LRG1 Drives AML Progression by Disrupting Myeloid Progenitor Regulation
              </a>
            </td>
          <td>
            Rongxia Guo, Peng Wu, Shuxin Yao, Fumei Liu
          </td>
          <td>2025-12-01</td>
          <td>Blood and Lymphatic Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains one of the deadliest primary brain tumors, with rapid recurrence and near-universal resistance to temozolomide (TMZ) limiting long-term survival. In this study, we identify a clinically actionable mechanism of resistance driven by the LMNA–PRKDC axis, which enhances DNA repair and tumor cell survival following TMZ treatment. Using patient-derived xenograft models of recurrent GBM, we demonstrate that resistant tumors exhibit elevated LMNA expression and increased physical interaction with PRKDC, a central regulator of non-homologous end joining (NHEJ). This interaction accelerates the repair of TMZ-induced DNA lesions, contributing to therapeutic failure. Proteomic profiling and targeted immunoprecipitation revealed a distinct LMNA–PRKDC–associated DNA repair complex. Inhibition of PRKDC with the ATP-competitive inhibitor KU57788 reversed resistance, restoring TMZ sensitivity and impairing tumor growth in vivo. Single-cell RNA sequencing of primary and recurrent GBM specimens further identified LMNA–PRKDC co-expression as a hallmark of treatment-resistant, glioma stem-like cell populations. Importantly, high LMNA–PRKDC expression was associated with inferior survival outcomes in GBM patient cohorts. These results establish the LMNA–PRKDC axis as a functional driver of TMZ resistance through enhanced DNA repair capacity in stem-like tumor subpopulations. Our findings support pharmacologic inhibition of PRKDC as a rational strategy to resensitize resistant GBM to standard chemotherapy and offer a foundation for future biomarker-driven clinical trials targeting DNA repair vulnerabilities in recurrent disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1056c18795c7e2da4201c08c63b3bd0b960f366" target='_blank'>
              LMNA-PRKDC axis enhances DNA repair and promotes chemoresistance in glioblastoma
              </a>
            </td>
          <td>
            Miranda R Saathoff, Rafał Chojak, Rebecca X Chen, Hasaan A Kazi, Umme H Faisal, Jack M. Shireman, Noah B Drewes, Cheol H. Park, Xuesong Fan, Sana A Khan, Irene Lazanyi, S. Baisiwala, C. D. James, C. Horbinski, Atique U. Ahmed
          </td>
          <td>2025-11-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Ribosomal DNA (rDNA) encodes the 18S, 5.8S, and 28S rRNA, accounting for ∼70% of cellular transcription. Despite its essential role and links to cancer and aging, quantifying rDNA instability in mammals remains challenging due to its repetitive organization and inherent heterogeneity. Here, we developed a murine rDNA FISH probe and genomic tools tailored for laboratory mouse strains. The results confirmed rDNA cluster locations, revealed substantial inter- and intra-strain as well as intercellular heterogeneity in rDNA organization within inbred mice and unstressed cells, and identified sources of spontaneous and replication-associated DNA double-strand breaks in the rDNA transcription termination region. Using mouse embryonic stem cells, we showed that BRCA1-mediated homologous recombination promotes rDNA instability, the non-homologous end joining factor XRCC1, but not Ku, suppresses intra-cluster deletions, and ATM kinase preserves rDNA cluster stability. Together, these findings establish a platform and tools for studying rDNA instability in animal models relevant to aging and cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377fa5f7328f7f52cd2825bb12b7a086a7b2a641" target='_blank'>
              Visualization and quantification of rDNA instabilities in mammalian cells and mouse models
              </a>
            </td>
          <td>
            Xiaolu Zhu, Wenxia Jiang, Wei Wu, Brian J. Lee, Demis Menolfi, Anthony Tubbs, Olivia M. Cupo, Eli Malkovskiy, Mattie Nester, Xiaobin S. Wang, Peter A Sims, Chyuan-Sheng V Lin, Lorraine Symington, Andre Nussenzweig, Brian McStay, Shan Zha
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT DNA surveillance mechanisms are crucial for maintaining genome stability and minimizing mutation rates. Mismatch repair (MMR) corrects replication errors that escape DNA-polymerase proofreading. In most organisms, MMR is orchestrated by MutS and MutL proteins. However, certain Archaea and Actinobacteria, including the major human pathogen Mycobacterium tuberculosis, lack these components. Instead, they appear to rely on the nuclease EndoMS/NucS, a structurally distinct enzyme governing a non-canonical MMR pathway. Since M. tuberculosis acquires drug resistance exclusively through chromosomal mutations, understanding its mutation rate regulation is critical. Nevertheless, the role of NucS in drug resistance evolution remains largely unexplored. We investigated NucS function in M. tuberculosis and uncovered a unique resistance dynamic distinct from other Actinobacteria. Deleting nucS altered the mutational spectrum but had minimal impact on the emergence of antibiotic-resistant mutants, contrasting sharply with other Actinobacteria where nucS loss dramatically increases mutation rates. This atypical behavior cannot be attributed solely to the R144S NucS polymorphism present in the H37Rv reference strain. Introducing the consensus NucS sequence in H37Rv produced only subtle changes in mutational spectrum without major effects on mutation rates. Analysis of 44,921 M. tuberculosis genomes revealed that most R144S-containing strains belong to the Euro-American sub-lineage 4.9, with no significant association with antibiotic resistance. However, nucS is under strong purifying selection, and R144S changes arose independently during M. tuberculosis evolution (homoplasy). Overall, our findings challenge the view that nucS serves as a genome stability guardian in M. tuberculosis and suggest additional mismatch repair mechanism(s) beyond NucS in this pathogen. IMPORTANCE DNA repair systems are crucial for maintaining the integrity of genetic information by scanning and correcting errors that arise during DNA replication. Most organisms use well-characterized proteins (MutS and MutL) for this task, but some bacteria and archaea, including most Actinobacteria, lack these conventional components. Instead, they employ an alternative enzyme, NucS, to fix replicative DNA errors. This alternative pathway is thought to limit harmful mutations and, in turn, reduce the likelihood of drug resistance development. In our study of Mycobacterium tuberculosis, the pathogen responsible for tuberculosis, we found that NucS plays only a minor role in preventing resistance-associated mutations. This unexpected result challenges current assumptions about DNA repair in this pathogen and points to the existence of other, yet unidentified mechanisms that safeguard its genome. Understanding these processes could open new avenues for therapeutic strategies against tuberculosis, a disease that remains a major global health threat. DNA repair systems are crucial for maintaining the integrity of genetic information by scanning and correcting errors that arise during DNA replication. Most organisms use well-characterized proteins (MutS and MutL) for this task, but some bacteria and archaea, including most Actinobacteria, lack these conventional components. Instead, they employ an alternative enzyme, NucS, to fix replicative DNA errors. This alternative pathway is thought to limit harmful mutations and, in turn, reduce the likelihood of drug resistance development. In our study of Mycobacterium tuberculosis, the pathogen responsible for tuberculosis, we found that NucS plays only a minor role in preventing resistance-associated mutations. This unexpected result challenges current assumptions about DNA repair in this pathogen and points to the existence of other, yet unidentified mechanisms that safeguard its genome. Understanding these processes could open new avenues for therapeutic strategies against tuberculosis, a disease that remains a major global health threat.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d734f5ee057101186e729eca6288974decb87e6d" target='_blank'>
              The unique role of nucS-mediated noncanonical mismatch repair in Mycobacterium tuberculosis resistance evolution
              </a>
            </td>
          <td>
            I. Martín-Blecua, Jorge Sastre-Domínguez, J. R. Valverde, Pablo García-Bravo, Ángel Ruiz-Enamorado, Rafael Prados-Rosales, Lahari Das, William R. Jacobs, Á. San Millán, Jesús Blázquez, Sonia Gullón
          </td>
          <td>2025-12-22</td>
          <td>mBio</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Medulloblastoma (MB) is one of the most prevalent malignant brain tumors in children, with tremendous cognitive and neuroendocrine disability among survivors. Group 3 MB (G3MB) has poor overall survival at <50%, high frequencies of metastases, and no targeted therapies. Amplification of MYC and activation of TGFβ signaling occur frequently in G3MB. Many tumors have no reported mutations, suggesting epigenetic drivers. We here describe novel humanized models for G3MB from human induced pluripotent stem cells (hiPSC). By transducing hiPSC-derived neuroepithelial stem cells (NESC), we determined that: 1) both MYC and TGFβ effectors drove tumors in vivo; 2) MYC/TGFβR1 in combination led to more aggressive tumors and resistance to clinical inhibitors of TGFβ, and 3) NESC-derived tumors clustered with human G3MB. To decipher mechanisms, we integrated RNA-sequencing with CUT&RUN (for MYC genomic localization and post-translational modification of histones). MYC-bound neural developmental genes were repressed in MYC/TGFβR1 co-driven lines. Gene signatures associated with the Polycomb Repressive Complex (PRC) demarcated with H3K27me3; the histone mark directly regulated by PRC. We identified JARID1B, a MYC binding partner and H3K4me3 demethylase, as a regulator of repressed neural genes. Primary G3MB also showed increased levels of H3K27me3 concurrent with higher expression of JARID1B. Knockdown of JARID1B in human G3MB cell lines reduced growth, supporting potential as a therapeutic target. We conclude that a MYC-TGFβ-JARID1B axis represses target genes to drive G3MB and present new humanized models for G3MB to understand epigenetic dysregulation in G3MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd58fafdd674bdec3d7247a90b13d164f167e167" target='_blank'>
              Human stem cell models for group 3 medulloblastoma uncover JARID1B as a regulator of the chromatin landscape
              </a>
            </td>
          <td>
            Zulekha A. Qadeer, Elise Hou, Dara Siegel, Samantha Westelman, B. Gudenas, Kyle S Smith, Wazim Mohammed Ismael, Linyu Wang, Mackenzie O Johnson, Shane Grele, Ronald Phua, Liam D. Hendrikse, Ekin Guney, Miller Huang, Alexandre Gaspar-Maia, P. Northcott, Michael D. Taylor, William A. Weiss
          </td>
          <td>2025-12-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Cervical cancer, as one of the most common types of malignant tumors in women, poses a significant threat to the health and survival of women worldwide. Tumor drug resistance has emerged as a critical bottleneck severely limiting the efficacy of current cervical cancer treatment strategies, making it imperative to delve deeper into the underlying mechanisms for improved therapeutic outcomes. Spatial transcriptomics and single-cell sequencing technologies were employed. Single-cell sequencing was used to uncover the heterogeneity in cervical cancer development. Pseudotime analysis, exploration of cell communication pathways, and spatial transcriptomics were further applied to investigate the roles of relevant molecules. This study reveals an abundance of PI3 + neoplastic (NEO) in tumors and SLC40A1 + NEO cells in HSIL samples. These cell populations may play crucial roles in the development of drug resistance, as they could potentially alter the tumor microenvironment and interfere with the efficacy of therapeutic agents. Pseudotime analysis, cell communication pathways, and spatial transcriptomics highlight the pivotal role of LGALS9 and its ligands, offering new insights for early diagnosis, molecular typing, prognosis evaluation, and personalized treatment of cervical cancer. Our findings reveal that high expression of LGALS9 suppresses T cell function, fosters a strongly immunosuppressive tumor microenvironment, and is associated with chemotherapy resistance, ineffective immunotherapy, and poor prognosis. LGALS9 may serve as a biomarker for predicting immunotherapy response. This knowledge has the potential to facilitate early diagnosis, precise molecular typing, accurate prognosis evaluation, and the development of personalized treatment strategies that can overcome tumor drug resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f662f4b181b3395cd096885302fa365b2511567e" target='_blank'>
              Exploring tumor heterogeneity and drug resistance in cervical cancer through single-cell and bulk transcriptomics
              </a>
            </td>
          <td>
            Yinxia Yang, Sirong Cheng, Yuxuan Zhong, Pengyu Sun, Xiaoru Deng, Bohui Zhou
          </td>
          <td>2025-11-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c60ea4aa70ca5fc55ee1757c090687b18cbeab5" target='_blank'>
              Construction of the cancer cell continuum reveals hybrid EMT state driving lung adenocarcinoma aggression.
              </a>
            </td>
          <td>
            Qiuhao Qu, Yixin Ma, Chao Huang, Botao Fa, Zhenni Liu, Yuhan Li, Shigao Yang, Zhengtao Xiao
          </td>
          <td>2025-11-25</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small molecules developed to target proteins or DNA may also bind RNA, but the extent and biological significance of such interactions among oncology drugs remain poorly defined. Here, we systematically profiled RNA interactions of a cohort of clinically approved anticancer agents and uncovered widespread RNA off-targeting. Cisplatin, a frontline chemotherapeutic agent for solid tumors, has emerged as a prominent RNA-binding drug. While the primary mechanism of action of cisplatin has been attributed to DNA damage-induced apoptosis, it has also been shown to bind RNA molecules. However, the extent of RNA binding in cancer cells and its functional relevance in platinum-based chemotherapy remained unknown. To map specific RNA targets of cisplatin in vivo, we developed PlatRNA-seq, a click-chemistry-enabled transcriptome-wide assay. Using this approach and integrated genomic, biophysical, and computational analysis, we show that cisplatin binding is enriched at guanine-rich regions of transcripts, with a pronounced affinity for RNA G-quadruplexes (rG4s) secondary structures. Cisplatin accumulates preferentially near the 5′ ends of transcripts associated with R-loop formation and RNA pol II stalling. Mechanistically, cisplatin binding to rG4s modulates their formation and stability. Importantly, we provide evidence that cisplatin-induced cytotoxicity is mediated in part through its binding to RNA, revealing a noncanonical RNA-based mechanism of action. Analysis of single-cell RNA-seq data from tumor biopsies of treatment-naïve ovarian cancer patients further shows that the expression of rG4-enriched cisplatin-RNA targets predicts platinum sensitivity, underscoring the prognostic and clinical relevance of drug-RNA interactions. Together, these results demonstrate that RNA off-targeting by small molecules is not passive but can modulate therapeutic outcomes and may be leveraged to overcome current limitations of chemotherapeutic agents. Our findings highlight the importance of systematically investigating RNA interactions of clinically used small molecules to better inform therapeutic and prognostic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4655d7bd6dd69864ccca69202d1967eb2da923e5" target='_blank'>
              Transcriptome-wide mapping reveals an RNA-dependent mechanism of platinum cancer drugs
              </a>
            </td>
          <td>
            Arun Krishnaraj, Xinrui Wei, Richi Thakral, Weiyi Guo, Siva Bala Subramaniyan, Xiaopei Zhang, Katelyn R. Alley, Anqi Feng, Andrew Hoy, Douglas Kung, Purushottam B. Tiwari, Aykut Üren, Rodrigo Maillard, Victoria J DeRose, Sreejith J Nair
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3e5486bf9671c3b3ce7043325de2e14e73fe8c9" target='_blank'>
              Connecting mitochondrial metabolism and mitotic fidelity to control vulnerability of high grade serous ovarian cancer patients to taxane-based chemotherapy
              </a>
            </td>
          <td>
            H. Moindjie, Morgane Morin, Evanthia Pangou, Diego Chianese, Cynthia Seiler, S. Rodrigues-Ferreira, Maria M. Haykal, Christine Péchoux, F. Dingli, D. Loew, Y. Kieffer, Géraldine Gentric, Patricia Pautier, Philippe Dessen, R. Argüello, Paolo Pinton, Fatima Mechta-Grigoriou, Catherine Brenner, Massimo Bonora, Clara Nahmias
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Type-II topoisomerases resolve topological stress in DNA through double-strand breaks. While topoisomerases are chemotherapy targets linked to therapy-related genotoxicity, TOP2B is uniquely positioned to influence mutagenesis through its activity in non-dividing cells and sensitivity to topoisomerase poisons. To investigate this, we generated DNA-binding maps of TOP2B, CTCF, and RAD21 in human cancer samples and analyzed these for driver mutations and mutational processes across 6500 whole cancer genomes. TOP2B-CTCF-RAD21 and TOP2B-RAD21 sites are enriched in somatic mutations and structural variants, particularly at sites with evolutionary conservation, high transcription and long-range chromatin interactions. TOP2B binds driver genes such as TP53, MYC, FOXA1, and VHL, and many frequently mutated non-coding regions. We show that one non-coding TOP2B-bound element at the non-coding RNA gene RMRP drives tumor initiation and growth in vivo. Our study highlights TOP2B as a safeguard of genome integrity and a marker of mutational processes and hotspots in cancer, underscoring implications for cancer genomics research. Topoisomerases resolve topological DNA stress via double-strand breaks and are established targets of cancer chemotherapies. Here, the authors link genomic binding of TOP2B with localized mutational processes in cancer genomes that include prominent driver genes and translocation hotspots.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5948f8a3f526ecfe0ce2c698b693d85da101b7" target='_blank'>
              Topoisomerase IIb binding delineates localized mutational processes and driver mutations in cancer genomes
              </a>
            </td>
          <td>
            Liis Uusküla-Reimand, Christian A. Lee, R. H. Oh, Zoe P. Klein, Nina Adler, Sana Akhtar Alvi, Ellen Langille, Elisa Pasini, Kevin C. L. Cheng, Evgenija Serafimova, Diala Abd-Rabbo, Huayun Hou, R. Tsai, Mamatha Bhat, D. Schramek, Michael D. Wilson, J. Reimand
          </td>
          <td>2025-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Type B1 and B2 thymomas are lymphocyte‐rich malignant tumours with few somatic mutations in protein‐coding regions of the nuclear genome; nonetheless, noncoding regions remain uncharacterized. Here, we developed a method to isolate pure thymoma cells from lymphocyte‐rich tissues, and then performed genome‐wide deep sequencing. The total number of somatic mutations was ~80 times higher in noncoding regions than in coding regions in type B12 thymomas (1,671.3 versus 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of the cases exhibited fewer than four nonsynonymous mutations in coding regions. A systematic noncoding analysis identified 405.0 mutations per case in cis‐regulatory elements and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Tumour‐specific/enriched mitochondrial heteroplasmic shift was observed in 90% of cases, with a significant proportion of mutations located in the D‐loop region. When tracing the evolutionary lineage of mtDNA mutation, the majority of cases can be explained by a linear evolutionary model. This suggests that positive selection may be operating on the mitochondrial genome during thymoma development. In summary, numerous noncoding mutations and mitochondrial heteroplasmic shift were detected in type B1 and B2 thymomas, some of which may be functional. Given the paucity of coding mutations observed in this disease entity, other factors such as disruption of the noncoding landscape and tumour‐specific/enriched mitochondrial heteroplasmic shift, may contribute to the development of thymoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/983bfc8a32a72d21e775036e5fe74d52ea0ed5cc" target='_blank'>
              Genome‐wide analysis of somatic noncoding mutation patterns and mitochondrial heteroplasmic shift in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Kaoru Watanabe, Juliet King, Emma Mclean, Susan Ndagire, Yoshihisa Takahashi, Masahiko Kuroda, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2025-12-04</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b81384b510909f679a401421e801231c6a36b6" target='_blank'>
              A tailored in vivo CRISPR screen identifies BAP1 as a potent tumor suppressor of soft tissue sarcoma
              </a>
            </td>
          <td>
            Jianguo Huang, Xingliang Liu, Warren Floyd, William Haugh, Zhaoyu Sun, Melissa J. Kasiewicz, Yaping Wu, Brian D. Piening, John T. Welle, Wesley K. Rosales, V. Rajamanickam, So Young Kim, Eric S. Xu, Lixia Luo, Yan Ma, Rutulkumar Patel, Ziqiang Zhang, B. Bernard, William L Redmond, Walter J. Urba, R. B. Bell, D. Kirsch
          </td>
          <td>2025-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor cell plasticity plays a key role in the progression of malignant neoplasms and therapy resistance, yet its spatial dynamics remain poorly understood. This study investigates the spatial distribution of epithelial–mesenchymal transition (EMT) and stemness, and their prognostic relevance in lung adenocarcinoma (LUAD). Single‐cell RNA sequencing was performed on 10 tumor fragments from three regions (core, core‐adjacent, and edge) of a single LUAD using a SURFSeq 5000 platform. Stemness potential was identified by CytoTRACE2, EMT by UCell and the Hallmark_epithelial_mesenchymal_transition signature, and tumor cells associated with disease prognosis by Scissor. LUAD displayed pronounced spatial heterogeneity of tumor cell plasticity. Stemness potential was heightened at the edge, while EMT was most prominent in the core. The correlation between EMT and stemness was slightly higher at the edge, though it remained negligible (R = 0.17, p = 5.5e−7). Higher EMT scores were observed in tumor cells associated with poor overall survival (r = 0.433, p < 2e−16), whereas stemness was prevalent in tumor cells associated with poor progression‐free survival (r = 0.251, p = 7.15e−6). Tumor cells associated with poor prognosis were enriched with the MYC‐targets v1 signature with PCBP1 and PA2G4 as the top contributing genes. Immunohistochemical data showed that PCBP1 and PA2G4 proteins predominantly localized within tumor cells of LUAD. Taken together, these findings highlight the spatial heterogeneity of tumor cell plasticity features in LUAD and uncover biomarkers associated with poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/badf7545e441ad3330c93dcdb325456db2158597" target='_blank'>
              Spatial Dynamics of Tumor Cell Plasticity in Lung Adenocarcinoma Revealed by Region‐Specific Single Cell Transcriptomics
              </a>
            </td>
          <td>
            A. Khozyainova, Maxim E. Menyailo, Vera G. Subrakova, Dmitry M. Loos, P. Iamshchikov, Anton A Fedorov, M. S. Tretyakova, Ustinia A Bokova, Evgeny O Rodionov, Sergey V Miller, Evgeny B Topolnitsky, D. V. Piankov, Sergei A Korostelev, Evgeny V. Denisov
          </td>
          <td>2025-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cb28bdcaf79e7d74e6f5d2dbc03edf13bb5c93" target='_blank'>
              SMAD4 loss drives cancer chromosomal instability through transcription-replication conflict-induced replication stress
              </a>
            </td>
          <td>
            Atmika Paul, Gargee Joshi, Philipp Jungk, Simranjeet Kaur, Shengyu Yao, Ioannis Tsirkas, N. Böhly, Maren Sitte, Gabriela Salinas, Stephan Hamperl, Maik Kschischo, Holger Bastians
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Personalized assessment of immunocompetence and DNA double-strand break (DSB) repair requires methods that are sensitive to genetic and molecular complexity beyond the well-known monogenic disorders. Inspired by decades of research using B cells to study DNA repair processes, here we present SWIBRID (SWItch junction Breakpoint Repertoire IDentification), a tool to systematically profile genomic junctions generated in vivo during antibody class switch recombination (CSR) in B cells. As CSR junctions reflect immune diversity and DNA repair proficiency, SWIBRID detects phenotypic manifestations of deficiencies via a highly scalable, blood-based PCR followed by long-read sequencing and bioinformatic analysis. We show that specific DNA repair defects, including cancer-associated mutations, exhibit distinct CSR junction patterns. Notably, SWIBRID distinguishes different types of DSB repair knockouts and identifies the respective genetic defect in cell lines. In 68 patients, we detect immunodeficiencies and DNA repair defects with high accuracy (area under the curve 0.99 and 0.84, respectively), and identify previously uncharacterized patient groups as well as patient-specific CSR junction signatures. With SWIBRID, we seek to advance the identification of pathogenic defects, support early diagnosis, and address molecular heterogeneity that drives variable clinical outcomes. Class switch recombination (CSR) is a process contributing to antibody diversity and generating characteristic genomic junctions that reflect DNA double-strand break repair efficiency. Here authors present a platform to profile the repertoire of CSR junction patterns; long-read-sequenced PCR products from blood-derived lymphocytes are analysed by a dedicated bioinformatic pipeline to accurately detect immunodeficiencies and DNA repair defects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba96a13f0bc51cc221f1e3cd935284f9970d0b5" target='_blank'>
              Recombination junctions from antibody isotype switching classify immune and DNA repair dysfunction
              </a>
            </td>
          <td>
            Clara Vázquez García, Benedikt Obermayer, Baerbel Keller, Mikhail Lebedin, C. Ratswohl, Hassan Abolhassani, Antonia Busse, M. Di Virgilio, Stephan Mathas, Dorothee Speiser, D. Beule, Q. Pan-Hammarström, K. Warnatz, K. de la Rosa
          </td>
          <td>2025-12-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Sequestosome 1 (p62/SQSTM1) is a multifunctional adaptor protein whose dysregulation promotes tumorigenesis through autophagy, metabolic reprogramming, and immune modulation. However, its role across cancer types and impact on the tumor immune microenvironment remain poorly defined. Here, we performed a comprehensive pan-cancer analysis to delineate the molecular and immunological landscape of p62 across human malignancies. TCGA analysis revealed rare mutations and limited prognostic impact of genomic alterations but marked transcriptomic upregulation in LIHC, LUAD, BRCA, KIRP, KICH, KIRC, and READ, correlating with poor survival, advanced stage, and higher T classification, particularly in BRCA and LUAD. Pathway analysis showed strong positive associations between p62 and metabolic adaptation and stress tolerance pathways, including oxidative phosphorylation, reactive oxygen species, and DNA repair, in most cancers. Notably, high p62 expression inversely correlated with immune cell infiltration in most epithelial cancers, such as BRCA, COAD, ESCA, HNSC, KIRC, LIHC, LUAD, LUSC, PAAD, PRAD, READ, THCA, while coinciding with elevated expression of immunosuppressive checkpoints such as PD-L1, B7-H3, EBAG9, PVR, and TGFB1, supporting a link between p62-driven metabolic remodeling and an immune-excluded tumor microenvironment. In contrast, GBM, LGG, OV, SARC, and TGCT showed positive correlations between p62 and immunoscore, with enrichment of interferon and pro-inflammatory pathways, reflecting a distinct immune-activated phenotype. Collectively, these findings identify p62 as a central regulator of tumor metabolism and immunity, suggesting that its context-dependent activity may dictate the balance between immune suppression and activation across cancers, and highlight two natural-product-derived PB1 inhibitors (ZINC70669789 and ZINC08877690) as promising candidates for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/716e845c24cbd3bb41e4b234ab94e5c67dc1b5b3" target='_blank'>
              Comprehensive pan-cancer analysis of p62 reveals its contribution to shaping tumor microenvironment and anti-tumor immunity
              </a>
            </td>
          <td>
            Zahra Nayeri, Elahe Tavakol, M. Rahmati, Marco Cordani, M. Djavaheri-Mergny, Vahid Shariati, M. Moosavi
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background: Glioblastoma multiforme (GBM) is a highly aggressive primary brain tumor characterized by rapid proliferation, profound invasiveness, and resistance to conventional therapies. Deubiquitinating enzymes (DUBs), crucial regulators of protein homeostasis, have recently been implicated in GBM pathogenesis. However, the specific DUBs that play central roles in GBM pathogenesis and their exact molecular mechanisms remain to be further elucidated. Methods: We systematically analyzed GBM datasets and clinical samples to identify differentially expressed DUBs. Functional experiments, including genetic manipulation, immunoprecipitation coupled mass spectrometry (IP-MS), comprehensive metabolic assays, mitochondrial function assessments, and orthotopic mouse models, were conducted. Results: Here, we identified PSMD14 as a protein significantly upregulated in GBM, with a close correlation to poor prognosis of patients. Mechanistic exploration revealed that PSMD14 stabilized IMPDH2, the rate-limiting enzyme of purine nucleotide biosynthesis, by selectively removing K48-linked polyubiquitin chains. When PSMD14 is inhibited genetically or pharmacologically, IMPDH2 stability diminishes, causing impaired nucleotide metabolism, mitochondrial dysfunction, increased DNA damage signaling, and reduced tumor malignancy. Importantly, these metabolic issues can be reversed by exogenous guanosine, highlighting the key role PSMD14 in metabolic regulation. In translational medicine, the PSMD14 inhibitor, Thiolutin, curbed GBM progression in vitro and in vivo by disrupting the de novo purine biosynthesis and resulting in mitochondrial fragmentation. Moreover, Thiolutin synergized with TMZ to overcome resistance and boost efficacy. This study reveals a new GBM metabolic axis and presents a promising PSMD14-targeting therapy. Conclusions: PSMD14–IMPDH2 axis serves as a crucial hub integrating post-translational modifications and metabolic homeostasis in GBM. Targeting PSMD14 enhances therapeutic sensitivity, presenting a promising strategy to overcome TMZ resistance and improve GBM treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96036dbb03725e6c9416ba601f587aa6ca12bc4a" target='_blank'>
              Coupling proteostasis and
 de novo
 purine biosynthesis of PSMD14 fuels glioblastoma progression and chemoresistance
              </a>
            </td>
          <td>
            Jiazheng Wang, Qun Cao, Zhikai Li, Xuxiu Lu, Zhuo Li, Chenghui Guo, Yuan Pan, Qing Zhang, Wenjie Li, Guo Xiang, Anjing Chen
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The rising rate of drug-resistant fungal infections and the emergence of fungal pathogens with intrinsic resistance phenotypes are a growing concern. The close evolutionary distance between mammals and fungi complicates the design of new antifungals and increases the chances of toxic off-target effects. As such, antifungal drug development usually focuses on fungal-specific proteins when considering potential new targets. Ideal drug targets should mediate essential cell processes and be highly sensitive to inhibition. Targeted gene repression can serve as a model for drug-mediated inhibition and for determining the dosage-sensitivity profile of genes of interest. In the fungal pathogen Candida albicans, classical approaches for gene repression can be labour-intensive and limited to one genetic background due to low throughput. Here, we adapt pooled CRISPRi screening in C. albicans for the first time and exploit this technique for large-scale functional genomic analysis. Through pooled CRISPRi screening, we test the repression sensitivity of over a hundred essential genes conserved in fungi but absent in humans, and successfully identify highly dosage-sensitive genes across multiple cell components and pathways. By extending our analysis to ten diverse environmental conditions, we show how the environment influences dosage-sensitivity profiles. Finally, we extend our experiments to two clinical drug-resistant C. albicans strain backgrounds and demonstrate that many of the fitness defects we observed are conserved in resistant clinical isolates. Together, our results highlight a set of genes that are highly dosage-sensitive across different genetic and environmental contexts, making them attractive targets for further investigation. By facilitating rapid, efficient large-scale functional genomics assays across diverse genetic backgrounds, CRISPRi pooled screening will open new frontiers in C. albicans biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/420f96e213065a56fe1298cd2a47d84efe8c11e0" target='_blank'>
              Pooled CRISPRi screening reveals fungal-specific vulnerabilities across environments and genetic backgrounds
              </a>
            </td>
          <td>
            Lauren F Wensing, Philippe C. Després, Desiree Francis, Meea Fogal, Anthony Hendriks, Nicholas C. Gervais, Clara Fikry, Abdul-Rahman Adamu Bukari, A. Gerstein, Christina A. Cuomo, Rebecca S. Shapiro
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5bdb6dd873fe4b6bf8ca46144bcdfa66e477ac0" target='_blank'>
              Beyond one-size-fits-all: single-cell transcriptomic signatures predict drug efficacy and reveal responder subgroups in endometriosis
              </a>
            </td>
          <td>
            R. Pérez-Moraga, C. Bafligil, S. Harden, Santiago García‐Martín, M. J. Jiménez-Santos, Ioanna Tiniakou, Sophie Ribeiro-Volturo, Line Gies, María Teresa Pérez Zaballos, Cristina Fernández-Molina
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive cancer with very poor outcomes. To identify additional drivers of leukemogenesis, we analyzed sequencing data from 1,727 unique individual patients with AML, which revealed mutations in ubiquitin ligase family genes in 11.2% of samples from adult patients with AML with mutual exclusivity. The SKP1/CUL1/F-box (SCF) E3 ubiquitin ligase complex gene, FBXO11, was the most significantly downregulated gene of the SCF complex in AML. We found that FBXO11 interacts with and catalyzes K63-linked ubiquitination of LONP1 in the cytosol, to promote LONP1 entry into mitochondria. We show that depletion of FBXO11 or LONP1 reduced mitochondrial respiration through impaired LONP1 chaperone activity to assemble electron transport chain Complex IV. Reduced mitochondrial respiration secondary to FBXO11 or LONP1 depletion imparted myeloid-biased stem cell properties in primary CD34+ hematopoietic stem and progenitor cells (HSPCs) in vitro. In a human xenograft model, depletion of FBXO11 cooperated with AML1-ETO and mutant KRASG12D to generate serially transplantable AML. Our findings suggest that reduced FBXO11 cooperates to initiate AML by priming HSPC for myeloid-biased self renewal through attenuation of LONP1-mediated regulation of mitochondrial respiration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dedec3ee340c133db6456fffff3680788db71dc" target='_blank'>
              Loss of FBXO11 establishes a stem cell program in acute myeloid leukemia by dysregulating LONP1
              </a>
            </td>
          <td>
            Hayle Kincross, Y. Mo, Xuan Wang, L. Chang, Gerben Duns, Franziska Mey, Jihong Jiang, Zurui Zhu, Naomi Isak, Harwood Kwan, Tammy Lau, T. R. Docking, P. Garg, Jessica Tran, Shane Colborne, Se-Wing Grace Cheng, Shujun Huang, Nadia Gharaee, E. Willie, J. Parker, Josh Bridgers, Davis Wood, Ramon I Klein Geltink, Gregg B Morin, Aly Karsan
          </td>
          <td>2025-11-25</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Large-scale genomic rearrangements in cancer cells can lead to the creation of fusion genes which alter the expression or function of the partner genes. Some fusion genes act as tumour drivers and the use of kinase inhibitors in patients with specific fusions have led to breakthroughs in cancer therapy. The large clinical and scientific interest in fusions has led to the development of software that use RNA sequencing data to identify fusion transcripts. Unfortunately, fusion transcript callers output many predictions which lack underlying genomic rearrangements and large datasets with validated fusions are scarce. This paper and the accompanying Fusions4U web application present a resource of validated and annotated gene fusions for 328 cell lines from the Cancer Cell Line Encyclopedia. Predicted fusion transcripts from Arriba and STAR-Fusion were analysed with our published validation pipeline that uses matched whole-genome sequencing data to identify discordantly mapped read pairs and candidate genomic breakpoints that support genuine fusion events. This resulted in 8,753 and 2,244 validated fusion transcripts for Arriba and STAR-Fusion predictions, respectively, with 1,596 fusions common to both. Additional layers of annotation include alternative splicing of fusion transcripts, kinases, microRNA host genes, genes in the COSMIC Cancer Gene Census, as well as known fusion gene pairs from the Mitelman Database of Chromosome Aberrations and Gene Fusions in Cancer and the TumorFusions dataset. We furthermore analysed information about fusion genes together with cell line data from the PRISM drug repurposing screening as an example of how this dataset can be used. This resource can be used to design experiments for functional studies and drug development, alone or in combination with publicly available information for the Cancer Cell Line Encyclopedia cell lines. The large collection of validated fusion transcripts with candidate genomic breakpoints can also be used in development and evaluation of bioinformatic tools for fusion transcript detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1751293449b5956386c8f2f0dbda4aebaafaddec" target='_blank'>
              Fusions4U: a resource of validated and annotated gene fusions in 328 cancer cell lines
              </a>
            </td>
          <td>
            Arianna Alamshahi, Helena Persson
          </td>
          <td>2025-12-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has traditionally been thought to develop through stepwise mutation of the APC tumour suppressor and other driver genes, coupled with expansion of positively selected clones. However, recent publications show that many premalignant lesions comprise multiple clones expressing different mutant APC proteins1, 2, 3–4. Here, by mediating transformation on different mouse backgrounds containing mutations in Kras or other common CRC driver genes, we establish that the presence of diverse priming events in the normal mouse intestinal epithelium can change the transformation and clonal-selection landscape, permitting the fixation of strong driver mutations in Apc and Ctnnb1 that are otherwise lost due to negative selection. These findings, combined with our demonstration of mutational patterns consistent with similar priming events in human CRC, suggest that the order in which driver mutations occur in intestinal epithelium can determine whether clones are positively or negatively selected and can shape subsequent tumour development. The order in which driver mutations of colorectal cancer occur in intestinal epithelium can determine whether clones are positively or negatively selected and can shape subsequent tumour development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85284f318689c76d83e7b6a1af9509e1766375cf" target='_blank'>
              Decay of driver mutations shapes the landscape of intestinal transformation
              </a>
            </td>
          <td>
            Filipe C. Lourenço, Iannish D. Sadien, K. Wong, Sam Adler, Ashley Sawle, Leonor Schubert Santana, Lee Hazelwood, Giada Giavara, Anna M Nicholson, Matthew Eldridge, N. Maka, Gerard Lynch, Stephen T McSorley, J. Edwards, Richard Kemp, D. Adams, Doug Winton
          </td>
          <td>2025-12-03</td>
          <td>Nature</td>
          <td>2</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277dcab08835ae1612852095fa2fa4e5142ac6cf" target='_blank'>
              Proximity of pre-mRNA 3′ end processing and transcription termination predicts enhanced gene expression
              </a>
            </td>
          <td>
            Agata Stępień, Deepshika Pulimamidi, Martyna Plens-Gałąska, Maxime Lalonde, Magda Kopczyńska, Hiroshi Kimura, S. Irshad, M. Gdula, Stephan Hamperl, Kinga Kamieniarz-Gdula
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f729ce791f41a9454d4b02d52f02d2067f40e331" target='_blank'>
              The role of chimeric FAM72B transcripts generated by intergenic mRNA trans-splicing in breast cancer
              </a>
            </td>
          <td>
            Gangotri Patra, Pok-Son Kim, Arne Kutzner, Klaus Heese
          </td>
          <td>2025-12-01</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Lynch syndrome (LS) is a genetic predisposition to mainly colorectal and endometrial cancer due to heterozygous disruptive germline mutations in the DNA mismatch-repair (MMR) genes MSH2, MSH6, MLH1, or PMS2. Beyond clearly pathogenic mutations, germline sequencing often reveals variants of uncertain significance (VUS), predominantly single base-pair alterations in coding or noncoding regions. These uncertain variants obstruct LS diagnosis, hampering personalized surveillance. To address this challenge, we developed a highly accurate functional assay that interrogates VUS pathogenicity in human cells. Building on a mouse-based cellular assay, we adapted oligonucleotide-directed mutation screening (ODMS) for human cells and introduced a refined approach named "coselection ODMS." To ensure physiological expression, the variant is introduced into the endogenous MMR gene by replication-coupled gene editing. Coselection ODMS demonstrated 100% accuracy in classifying 50 benign and 86 pathogenic variants spanning coding and noncoding regions in all four MMR genes. Among 109 patient-derived VUS, 51 were identified as deleterious for MMR function. Importantly, coselection ODMS delivered 100% concordant results in a clinical diagnostic laboratory. With >93% sensitivity and >92% specificity, coselection ODMS provides a highly reliable functional assay in the diagnosis of enigmatic LS variants, enabling risk assessment and personalized surveillance or treatment for affected families.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04046716ba1ab699ab383d6687b388d0e98d095b" target='_blank'>
              An accurate cellular assay to determine pathogenicity of coding and noncoding variants in Lynch syndrome genes.
              </a>
            </td>
          <td>
            Iris E. Glykofridis, Marleen Dekker, Chantal Stoepker, Thomas W. van Ravesteyn, Y. Tiersma, Cédric G van der Ham, Beaunelle de Bruijn, Salma Ebrahim, Renée X de Menezes, E. Kasteleijn, Frans Verheijen, T. V. van Ham, H. te Riele
          </td>
          <td>2026-01-16</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V. Semenov, Irina S. Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-08</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common primary liver cancer, characterized by high heterogeneity and poor prognosis. Chromatin remodeling regulates chromatin structure and function, and its dysregulation promotes tumor progression. Histone deacetylase 2 (HDAC2) is a key regulator involved in this process. This study links clinicopathological features with molecular mechanisms to elucidate the role of the chromatin-remodeling factor HDAC2 in hepatocellular carcinoma. Based on whole-slide images (WSIs) from TCGA-LIHC, pathological features were extracted by combining ResNet-50 and CellProfiler to build a prognostic model. SHAP was subsequently applied to interpret the most influential predictive features. Through integrative multi-omics analysis, HDAC2 was identified as a core gene, and its association with malignant phenotypes was validated using the Human Protein Atlas (HPA), bulk transcriptomic data, and single-cell datasets. Malignant cells were further identified with inferCNVpy, while CellChat and Vector were used to characterize intercellular communication and developmental trajectories. Finally, spatial transcriptomics integrated with the Spotlight and MISTy algorithms revealed the spatial distribution of HDAC2 and its interactions within the tumor microenvironment. The pro-tumor capability of HDAC2 was verified by both cell-based assays and nude mouse tumorigenicity experiments. From WSIs, 630 features were extracted with CellProfiler and 2,048 with ResNet-50. the top nine prognostic features were used to build a Lasso + GBM model selected from 101 machine learning algorithm combinations. SHAP analysis revealed the contribution of each feature. Correlation analysis with a curated chromatin remodeling gene set identified 49 genes significantly associated with these 9 features. For HDAC2, integrative analyses across bulk, single-cell, and spatial transcriptomics revealed its biological roles and underlying mechanisms. ROC curve analysis confirmed strong predictive performance, with average AUCs of 0.84 and 0.77 in the training and validation sets, respectively. HDAC2 knockdown reduced IL-1β/IL-6/TNF-α transcripts, lowered colony formation by ~ 55%, curbed invasion/migration by ~ 30–50% (P < 0.001), and trimmed xenograft weight by ~ 35% (P < 0.01), confirming that HDAC2 drives proliferation, motility and in-vivo tumorigenicity. HDAC2 drives hepatocellular carcinoma (HCC) progression by strengthening intercellular communication and forming a spatially organized oncogenic axis. Further analyses indicate that HDAC2-high cells can be regarded as differentiation origin points of the tumor and are significantly associated with poor prognosis. • HDAC2 is a key gene involved in chromatin remodeling that promotes HCC progression. • Integrative computational pathology and multi-omics analysis revealed HDAC2-related molecular mechanisms. • HDAC2 may serve as a potential prognostic biomarker and therapeutic target in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872eb7bdd212441a0452567d5163e9311d30ef05" target='_blank'>
              HDAC2-mediated chromatin remodeling drives hepatocellular carcinoma progression: an integrative analysis of computational pathology and multi-transcriptomics
              </a>
            </td>
          <td>
            Shiyang Yin, Xuancheng Zhou, Lai Jiang, Yuheng Gu, Jingxuan Dai, Haiqing Chen, Shengke Zhang, Jiaan Lu, Gang Huang, Yuxuan Jiang, Xun Sang, Kun Yuan, Hua Yang, Guanhu Yang, Shangke Huang, Hao Chi, Ke Xu
          </td>
          <td>2025-12-09</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2859fc19b9d23d91d42bdb280787178a4d820ac" target='_blank'>
              Rewiring the transcriptome: diagnostic and therapeutic implications of alternative splicing in solid cancers
              </a>
            </td>
          <td>
            Noura A. A. Ebrahim, T. Farghaly, Soliman M. A. Soliman
          </td>
          <td>2025-11-26</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Transposable elements (TEs) are repetitive DNA sequences typically silenced in normal tissues. Their dysregulation in cancer can significantly impact oncogene activation and tumorigenesis. However, due to their repetitive nature, TEs are often excluded from gene-centric single-cell analyses. Although recent advances have enabled single-cell TE quantification, a systematic resource for exploring single-cell TE dynamics in cancer has remained lacking. To address this, we developed TE-SCALE (https://ngdc.cncb.ac.cn/te-scale/), a single-cell database for integrative analysis and visualization of TE expression across human cancers. It incorporates over 1.3 million cells from 330 samples across 20 cancer types and 12 tissue origins. Built on our in-house pipeline scTEfinder, TE-SCALE provides a comprehensive pan-cancer TE expression atlas, enabling multi-scale exploration from tissue to cell population, supported by well-curated TE annotations. The platform offers three key analytical modules: differential TE expression, TE-gene co-expression network, and functional enrichment analysis. A user-friendly web interface supports flexible browsing, searching, analysis, and data download. Notably, TE-SCALE identifies tumor-specific TEs preferentially expressed in particular cancer types or disease states, underscoring their potential as biomarkers for diagnosis, monitoring, and immunotherapeutic targeting. Collectively, TE-SCALE provides an essential resource for decoding TE biology in cancer and expedites its translation into clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a20c719f57adedc7ed801a6aba2dc3cd3ae0ec" target='_blank'>
              TE-SCALE: a comprehensive database for exploring transposable element expression across human cancers at single-cell resolution
              </a>
            </td>
          <td>
            Xini Meng, Zhi Nie, Qifei Wang, Yiwen Hu, Yulan Deng, Na Ai, Zheng Huang, Yun Li, Yang Yuan, Jingfa Xiao, Jingyao Zeng, Guochao Li, Lan Jiang
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="Cervical cancer is among the prevalent gynecological malignancies affecting women worldwide. The diverse cellular populations and their intricate communication patterns within the tumor environment significantly influence cancer development and metastasis. Our research focused on elucidating the cellular diversity, intercellular signaling networks, and underlying molecular processes in cervical cancer tissues via single-cell transcriptomic approaches. Publicly available single-cell RNA sequencing (scRNA-seq) data for cervical cancer (GSE236738) were re-analysed to delineate the cellular composition and transcriptional programs of the tumour microenvironment. We applied UMAP and t-SNE algorithms for dimensionality reduction, alongside intercellular communication mapping, MIF pathway analysis, temporal trajectory modeling, and pathway enrichment studies, to comprehensively characterize the cellular architecture and molecular profiles within cervical cancer tissues. In addition, paired tumour and adjacent non-tumour cervical tissues from 20 patients were subjected to quantitative real-time PCR and enzyme-linked immunosorbent assay to confirm the transcriptional and protein-level expression of key genes identified by scRNA-seq. Analysis revealed substantial cellular and molecular diversity within cervical cancer specimens. Ten distinct cellular populations were characterized, including NK lymphocytes, epithelial components, and macrophage subsets, indicating pronounced cellular heterogeneity. Intercellular communication mapping revealed that stromal and epithelial populations are the primary coordinators of cell-to-cell signaling. MIF pathway activity peaked in cancer-associated fibroblasts, which exhibited distinct cell type distributions. Temporal analysis revealed dynamic cellular states and pinpointed critical regulatory elements, including ISG15. Pathway analysis emphasized the importance of neutrophil activity, mitochondrial respiration, and cell adhesion mechanisms in cancer progression. qPCR confirmed that ISG15 and TNFRSF18 were significantly upregulated in tumor tissues, while SDF4 was downregulated and NOC2L showed a moderate increase. ELISA results were consistent with transcript-level findings, demonstrating significant protein overexpression of ISG15 and TNFRSF18 in tumor tissues compared with controls. Our investigation comprehensively characterized the cellular diversity and molecular features of the cervical cancer microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7329b5a877ad7165acf8e0004372f128c5c7552" target='_blank'>
              Mapping cellular interactions and communication landscapes in cervical cancer via single-cell transcriptomics
              </a>
            </td>
          <td>
            Shaoli Zhuang, Xiaoxiao Deng, Furong Fu, Zhang Zhang
          </td>
          <td>2025-11-25</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a heterogeneous malignancy shaped by genetic alterations and immune microenvironmental interactions. While single-cell RNA sequencing (scRNA-seq) has uncovered diverse epithelial and T cell subsets, the spatial architecture of their crosstalk remains poorly understood. We integrated scRNA-seq and spatial transcriptomics data from 36 CRC patients to comprehensively characterize epithelial and T cell heterogeneity, differentiation dynamics, and intercellular communication networks. Data preprocessing and integration were conducted using Seurat and Harmony. Cell trajectory inference was performed via CytoTRACE and Slingshot. Ligand-receptor signaling was assessed using CellChat, while spatial mapping was achieved using CellTrek. Nine epithelial and eight T cell subpopulations were identified, each exhibiting distinct transcriptional states, CNV burdens, and pseudotime trajectories. CD8⁺ T_GZMK cells displayed tumor-specific activation signatures and engaged in enriched communication with epithelial subsets through CEACAM, APP, and MIF signaling pathways. Spatial transcriptomics confirmed the in situ colocalization of CD8⁺ T_GZMK cells with PTP4A3⁺ epithelial cells in tumor regions, revealing spatially organized epithelial–immune niches. These interactions suggest potential mechanisms of immune modulation and tumor progression. This study provides an integrative single-cell and spatial atlas of CRC, revealing structured epithelial–T cell communication and spatial architecture within the tumor microenvironment. Our findings offer novel insights into immune-epithelial crosstalk and identify signaling pathways that may serve as therapeutic targets or biomarkers for CRC precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c4f2ec4971acec7d7b9addcd8348e71b0ec1ff" target='_blank'>
              Decoding epithelial-T cell interactions in colorectal cancer through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Rentao Yu, Xinyu Lu, Zhuangzhuang Yue
          </td>
          <td>2025-11-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9932f70712f8bef30c70a5b989c4e744cf379f36" target='_blank'>
              Harnessing transcriptomics for discovery of natural products to overcome acquired cancer resistance.
              </a>
            </td>
          <td>
            Heerim Yeo, Sang-Yun Kim, Sang-Min Park
          </td>
          <td>2025-12-08</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PAAD) metastasis is driven by complex tumor–microenvironment interactions. Here, we integrated single-cell and bulk transcriptomic analyses of 104,855 cells from 10 patients to delineate the cellular and molecular landscape of primary versus metastatic PAAD. We identified metastasis-associated epithelial (LMO7⁺, TOP2A⁺, PIGR⁺), fibroblast (IGKC⁺, RGS5⁺), and M2-like macrophage (APOE⁺, CD14⁺, FOLR2⁺, SPP1⁺) subpopulations, validated via bulk deconvolution. Functional analyses revealed upregulated Wnt signaling, epithelial–mesenchymal transition, and angiogenesis in metastatic epithelial and fibroblast compartments. Intercellular communication analysis highlighted SPP1-mediated macrophage–epithelial/fibroblast crosstalk involving key receptor–ligand pairs, contributing to immune suppression and metastatic niche formation. Integrating gene expression and cell proportions, we developed a prognostic model with high predictive accuracy (C-index > 0.85), stratifying patients into risk groups with distinct immune landscapes. Furthermore, PTK6 was identified as a driver of PAAD proliferation, migration, and invasion. Collectively, our study elucidates TME-driven mechanisms of PAAD metastasis, identifies prognostic and therapeutic targets, and provides a framework for precision intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759c03413ef4b1106fa2a25c75713885026e6884" target='_blank'>
              Single-cell transcriptomic analysis reveals key cell types and pathogenic genes associated with metastasis in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Shijun Shen, Zhiqiang Li, Hong Yang, Xinwei Zhang, Gang Chen, Chengzhou Pa
          </td>
          <td>2025-11-29</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Cancer immune evasion is orchestrated by tumor‐intrinsic molecular constraints that remain incompletely defined. Here, we performed an in vivo genome‐wide clustered regularly interspaced short palindromic repeats (CRISPR) loss‐of‐function screen to catalogue gene regulatory determinants of immune evasion in cancer cells. We identify C9ORF50 as a novel splicing regulator whose inhibition profoundly sensitizes cancer to immune surveillance. Integrated multi‐omics profiling reveals this intrinsically disordered protein exhibits liquid–liquid phase separation properties and forms nuclear condensates that colocalize with spliceosome components. Genetic ablation correlates with intron retention in multiple spliceosome components and cytoplasmic accumulation of double‐stranded RNA, which is associated with type I interferon activation and enhances chemokine‐mediated T cell recruitment. As a result, C9ORF50 inhibition amplifies tumor cell immunogenicity, enhancing T cell infiltration in poorly infiltrated tumors. Clinically, elevated C9ORF50 expression correlates with poor survival and diminished lymphoid infiltration across malignancies. Therapeutic targeting of C9ORF50 using RNA interference enhances T cell infiltration and suppresses tumor growth. Our work identifies C9ORF50 as a candidate therapeutic target that modulates RNA splicing and tumor immunity, suggesting splicing regulation as a potential strategy to enhance immunotherapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4776a77ccb7e8b56c7236aff620b0308713e23cc" target='_blank'>
              Genome‐wide CRISPR screen reveals an uncharacterized spliceosome regulator as new candidate immunotherapy target
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Lulu Zhang, Fangzhou Liu, Yanhua Qi, Tao Hou, Ming Li, Sujuan Zhang, Yu Liu, Zhixiang Yuan, Jiali Liu, Yanming Hu, Jingyang Wang, C. Song, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Wenjun Mao, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-11-28</td>
          <td>iMeta</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Gene amplification is thought to be common in bacterial populations, providing a rapid and reversible mode of adaptation to diverse stresses, including the acquisition of antibiotic resistance. We previously showed that the opportunistic pathogen Staphylococcus aureus evolves resistance to the dual-targeting fluoroquinolone delafloxacin (DLX) that inhibits both the DNA gyrase and DNA topoisomerase IV via amplification of an efflux pump encoding gene sdrM. However, the pathways that control gene amplification, and consequently adaptive trajectories, remain understudied, especially in gram-positive bacteria like S. aureus. Here, we show that specific DNA repair and chromosomal separation proteins alter the frequency of gene amplification and selection of amplified regions in S. aureus. Through a screen of 40 mutants deficient in various DNA processes, we determined that while sdrM amplification was still the almost universal path to DLX resistance, other mutations that increased sdrM expression reduced the selection frequency of sdrM amplification, demonstrating the critical role of sdrM in DLX resistance. We found that similar to other bacteria, both sdrM amplification and loss of amplified gene copies required a functional RecA recombinase, but multiple other mutants in pathways required for amplification in other species still exhibited frequent sdrM amplification, suggesting that S. aureus may have alternate routes of gene amplification. Finally, loss of function mutants of the tyrosine recombinase XerC, that is known to play a role in chromosomal separation, were deficient for sdrM amplification, indicating that XerC is a novel modulator of gene amplification, or the maintenance or selection of amplified gene copies. Thus, our work sheds light on genetic factors that alter gene amplification-mediated evolutionary trajectories to antibiotic resistance in S. aureus and can potentially unlock mechanisms by which such evolution of resistance can be inhibited.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/875b466f28280308897b4f66650c929c02cc31e9" target='_blank'>
              Modulators of gene amplification alter evolution of antibiotic resistance in Staphylococcus aureus
              </a>
            </td>
          <td>
            K. P. T. Silva, Anthony M Martini, Anupama Khare
          </td>
          <td>2025-12-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1c6986a7f4aa4bd9fab34a1de917cf610f7b9a" target='_blank'>
              Interplay between mobile genetic elements drives a fusion-deletion life cycle of plasmids to fuel antimicrobial resistance
              </a>
            </td>
          <td>
            Thomas Ipoutcha, Yiqing Wang, Eduardo P. C. Rocha, José R. Penadés
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction Cancer origin patterns significantly influence cancer phenotypes and hallmark characteristics. Method In the present review, four distinct carcinogenesis trajectories that contribute to malignant transformation: mutator phenotype, chromosomal instability, dysmetabolism, and stemness, are explored. Results and Discussion In the mutator phenotype trajectory, deficiencies in DNA repair or synthesis systems lead to hypermutation and accumulation of oncogenic alterations. The chromosomal instability trajectory involves aneuploidy-induced copy number alterations in oncogenes and tumor suppressor genes. Dysmetabolic carcinogenesis is driven by the accumulation of oncometabolites due to alterations in metabolic genes. The stemness trajectory refers to the malignant transformation of cells possessing stem-like properties under oncogenic stimuli. Each trajectory independently promotes carcinogenesis and endows cancer cells with distinct characteristics. Notably, the primary oncogenic drivers in each trajectory can self-reinforce and form spontaneous-reinforcing loops that amplify oncogenic signals. Although crosstalk exists among trajectories, evidence suggests they are mutually exclusive during cancer origin. Therefore, targeting specific carcinogenesis trajectories and disrupting the self-reinforcing oncogenic loops may represent novel therapeutic strategies. Understanding carcinogenesis trajectories provides a framework for future cancer research and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c92c0ec788118fc8fac54bdd52f8df0eef75667c" target='_blank'>
              Carcinogenesis trajectories
              </a>
            </td>
          <td>
            Rui Wang, Zhaopeng Yan
          </td>
          <td>2025-12-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d72b955b3aea1e2e08b44ecc1d42ae2576e649d" target='_blank'>
              Ionizing Radiation Shapes Genome Evolution Through Nuclear Abnormalities That Trigger Delayed Proliferative Death
              </a>
            </td>
          <td>
            Mian Zhao, Bejo Presila, Cheng-Zhong Zhang, Spektor Alexander
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICI) have demonstrated prolonged efficacy in certain melanoma patients, yet a significant portion of patients do not experience clinical improvement, with the mechanisms underlying this resistance still not fully understood. Using established cell markers, we partitioned the single‐cell transcriptome into clusters, finding a notable link between NK cells and patient response to immunotherapy. We further identified four distinct subpopulations of NK cells, profiling marker gene sets and unique biological functions associated with each subpopulation. This analysis provides insights into the trajectory of NK cell development and differentiation, along with identifying the transcription factors driving these processes. The study pinpointed NK cluster 01 as pivotal in influencing patient sensitivity and prognosis during immunotherapy. Single‐cell transcriptome and spatial transcriptomics (ST) analysis revealed the proximity of NK cluster 01 cells to melanoma cells, hinting at a potential regulation of cell‐cell interaction via the IFN‐II signaling pathway network. ST analysis revealed the spatial arrangement and interaction of NK cluster 01 cells with melanoma cells. This study explores the feasibility of targeting NK cluster 01 cells with small molecule drugs via molecular docking, offering a promising approach to bolster the clinical utility of NK cell therapy. We comprehensively analyze the heterogeneity of NK cells within melanoma, elucidate the potential regulatory interactions between NK cells and other microenvironmental components, and establish a basis for the future clinical utilization of distinct NK cell subsets as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbdc1aaba32b7600dd3221ae61c5e31ee96bd3" target='_blank'>
              Integrating Single‐Cell and Spatial Transcriptomics Reveals NK Cell Subpopulations Associated With Immunotherapy for Melanoma
              </a>
            </td>
          <td>
            Zhicheng Hu, Yongfei Chen, Hao Yang, Qiuming Pan, Hongrui Li, Shuting Li, Junxi Wang, Yudi Huang, Guanglong Huang, Shanqiang Qu
          </td>
          <td>2025-12-01</td>
          <td>Smart Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The DNA Damage Repair (DDR) signaling pathway serves as a crucial molecular hub that regulates chemotherapy efficacy, offering significant translational value in the field of precision oncology. This review systematically analyzes the molecular mechanisms of five core DDR pathways (Homologous Recombination Repair, Non‐Homologous End Joining, Base Excision Repair, Nucleotide Excision Repair, and Mismatch Repair) in mediating chemotherapy resistance in tumors, and thoroughly elucidates the correlation between key molecular events—such as BRCA1/2 deficiency, MMR functional abnormalities, and Ataxia Telangiectasia Mutated/Ataxia Telangiectasia and Rad3‐related (ATM/ATR) signaling pathway dysregulation—and chemotherapy sensitivity. The DDR deficiency biomarker system established through the integration of multi‐omics data provides molecular classification tools for predicting the efficacy of platinum‐based drugs. This study focuses on the mechanism by which Poly ADP‐Ribose Polymerase inhibitors reverse Homologous Recombination‐Deficient tumor resistance through “synthetic lethality” effects while also revealing the synergistic anti‐tumor effects of ATM/ATR inhibitors in combination with chemotherapeutic agents. The research presents an innovative molecular synergy model between DDR regulation and Immune Checkpoint Blockade, confirming that tumor neoantigen release induced by DDR deficiency can enhance immunotherapy responses. This article also provides perspectives on multidimensional intervention strategies based on the DDR network, including the development of inhibitors targeting novel DDR targets, the establishment of DDR pathway functional assessment systems based on multidimensional biomarkers, and the investigation of synergistic paradigms between DDR and novel therapeutic modalities. Additionally, we explore the dynamic evolution mechanisms of DDR‐mediated chemotherapy resistance by analyzing the interactions between DDR and metabolic reprogramming, as well as other related processes. These breakthrough advances provide theoretical foundations and innovative directions for overcoming chemotherapy resistance and advancing personalized treatment, marking a new era in cancer therapy characterized by precision targeting of DDR pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/944e141ebbf24071dde5281d6930372251bae23d" target='_blank'>
              Translational Aspects of DNA Damage Repair in Optimizing Cancer Chemotherapy
              </a>
            </td>
          <td>
            Anqi Lin, Jinyue He, Aimin Jiang, Jian Zhang, Quan Cheng, Hengguo Zhang, Wenjun Mao, Peng Luo
          </td>
          <td>2025-12-01</td>
          <td>Advanced Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 EGFR-mutant non-small cell lung cancers (NSCLCs) initially respond to EGFR tyrosine kinase inhibitors (TKIs) such as osimertinib, but tumors eventually develop resistance. While genetic mutations explain many instances of resistance, an increasing body of evidence points to transcriptional and cell state changes as another route by which tumor cells evade therapy. The dynamics that shape these adaptive responses and the non-genetic mechanisms involved in facilitating them remain incompletely understood. We investigated transcriptional programs activated in EGFR-mutant LUAD cells following TKI treatment and observed robust induction of SOX2, a transcription factor involved in promoting stemness and lineage plasticity, within a day of drug exposure. In parallel, we investigated the role of APOBEC cytidine deaminases (APOBEC3A and APOBEC3B), which have been linked to mutagenic processes in cancer. While both enzymes have been implicated in drug resistance, the individual contributions of the two enzymes remain ill studied. Using competition assays with single gene knockouts of either APOBEC3A or APOBEC3B, we observed that loss of APOBEC3A leads to a more profound reduction in the survival of drug-tolerant cells following TKI therapy. These results highlight a functional role for APOBEC activity in sustaining early drug-tolerant populations under therapeutic pressure. Over the course of long-term osimertinib treatment, we also identified distinct subpopulations of cells with elevated ΔNp63, an isoform of TP63 that serves as a marker of squamous cell trans-differentiation. These ΔNp63-high cells emerge during the acquisition of drug resistance, and evidence suggests that this subpopulation may rely on CDK4, a cell cycle-regulating kinase, to sustain growth under therapeutic pressure. Pharmacological inhibition of CDK4 with palbociclib reduced the survival of ΔNp63-high cells, highlighting a potential therapeutic vulnerability in this proliferative subpopulation. Together, our findings demonstrate that adaptive resistance in EGFR-mutant NSCLC is driven by several non-genetic resistance mechanisms, including SOX2 induction, ΔNp63-driven squamous cell transdifferentiation, and APOBEC-mediated promotion of drug-tolerant persister cells.



 Sharan Srinivasan, Jessica N. Becerra, Nina M. Garcia, James V. Alvarez. SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a218a816c806ca36ea70bf5b2dadd14af3bb49f" target='_blank'>
              Abstract B019: SOX2 induction and ΔNp63-high subpopulations mark adaptive responses to EGFR inhibition in lung adenocarcinoma
              </a>
            </td>
          <td>
            Sharan Srinivasan, Jessica N Becerra, N. M. Garcia, James V. Alvarez
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Rare tumor cell populations can evade cancer therapies, ultimately driving drug resistance and relapse. These resilient populations, termed “persisters,” display distinct molecular states compared to the bulk tumor. While some persisters acquire genetic mutations that confer growth advantage, others undergo non-genetic remodeling that enhances cellular plasticity and fitness. Previous studies suggest that stochastic gene activation promotes persister survival (Shaffer et al., Nature 2017; Goyal et al., Nature 2023), yet the master regulators of this plasticity remain undefined. Using DNA barcoding, we demonstrate that pre-determined, non-stochastic persisters exist in KRAS-mutant non-small cell lung cancer (NSCLC) treated with the KRAS G12C inhibitor sotorasib. These clones repopulate the tumor without harboring recurrent genetic alterations. By isolating single clones destined to persist or go extinct, and profiling their transcriptomes and chromatin accessibility, we found that persister clones activate an interferon-γ (IFNγ) transcriptional signature accompanied by increased chromatin accessibility at sites enriched for the transcription factors IRF1 and ZNF384. Functionally, ectopic ZNF384 expression converted the entire cell population into a resistant state, while IRF1 knockout prevented persister formation. Cleavage Under Targets & Release Using Nuclease (CUT&RUN) assays of ZNF384 and IRF1 revealed their binding to promoter and enhancer regions, including loci regulating IFNγ-responsive genes, thereby contributing to persister plasticity and survival. Together, our study identifies IRF1 and ZNF384 as tumor-intrinsic regulators linking IFNγ signaling to transcriptional plasticity, providing mechanistic insight into the evolution of drug tolerance and resistance in KRAS-mutant NSCLC.



 Chendi Li, Anahita Nimbalkar, Christopher J. Graser, Melissa Vieira, Mohammed U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, Anne Y. Saiki, Paul E. Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata. Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B014.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331fb37a264b2ec60d8af95499de6058539d7fa9" target='_blank'>
              Abstract B014: Transcription factors governing the evolution of drug-tolerant persisters in KRAS-mutant non-small cell lung cancer
              </a>
            </td>
          <td>
            Chendi Li, Anahita Nimbalkar, Christopher J Graser, Melissa Vieira, M. U. Syed, Qian Qin, Barbara Karakyriakou, Sarah Clark, A. Y. Saiki, Paul E Hughes, Christopher Ott, Luca Pinello, Franziska Michor, Aaron N. Hata
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Complex chromosomal rearrangement (CCR) refers to a structural rearrangement involving at least two chromosomes or a minimum of three breakpoints. CCR may lead to intellectual disability, structural anomalies, infertility, and recurrent miscarriages. Chromosome karyotyping and chromosomal microarray analysis (CMA) are unable to detect complex chromosomal rearrangements. As multiple diagnostic approaches are available in clinical practice for detecting chromosomal structural abnormalities and copy number variations—each with its own advantages and limitations—selecting the appropriate testing method is crucial for effective clinical management. Optical genome mapping (OGM) is an advanced genomic technology that utilizes ultra-long single-molecule analysis to comprehensively detect chromosomal aberrations and structural variants at high resolution. Amniocentesis was performed for a 36-year-old multipara (advanced maternal age), with subsequent comprehensive fetal genetic analysis including chromosome karyotyping, CMA, and OGM. Family members underwent peripheral blood karyotyping and OGM. The fetal karyotype derived from amniotic fluid was 46,XN,?ins(18)(q21.2;p11.31p11.2). CMA demonstrated duplications of four segments and a deletion of one segment on chromosome 18. Therefore, OGM was performed on the fetal and family members to further elucidate the chromosomal structure. The fetus has derived CCRs on chromosome 18 of maternal origin. In contrast, both the mother and the second daughter, who carried the identical CCRs, were phenotypically normal. OGM is of significant importance in the diagnosis and characterization of CCRs. OGM plays a critical role in diagnosing complex chromosomal rearrangements and has proven to be invaluable in clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a05870664d5ee26da7943564ddcc7debed72e97b" target='_blank'>
              Optical genome mapping of a complex structural rearrangement family line on chromosome 18
              </a>
            </td>
          <td>
            Liyi Cai, Yuying Jiang, Na Zhang, Xinying Chen
          </td>
          <td>2025-12-01</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: Despite the use of immunotherapy in esophageal squamous cell carcinoma (ESCC), treatment failure occurs occasionally in patients, yet the underlying mechanisms remain poorly understood. Methods: We conducted large-scale single-cell RNA sequencing (scRNA-seq) data analysis, which integrated seven independent datasets from 192 ESCC patients to yield over 440,000 high-quality single cells, to systematically characterize the tumor microenvironment (TME) landscape during ESCC progression and immunotherapy response. Additionally, we performed high-resolution spatial transcriptomics (stRNA-seq) using the 10x Visium HD platform on paired pre- and post-treatment tissues from two patients (one immunotherapy responder and one non-responder), which enhanced the findings from the scRNA-seq data and mapped therapy-induced TME at the spatial level. Multiplex immunohistochemistry was employed based on seven patients to confirm distinct patterns of intercellular crosstalk underlying differential therapeutic outcomes. Results: In scRNA-seq data, we found that B lineage cells were reduced during ESCC progression but were enriched in immunotherapy-resistant patients. Further analysis of malignant ESCC cells suggested that immunotherapy resistance might be associated with a subpopulation of tumor cells exhibiting aberrantly elevated cholesterol biosynthesis. Cell communication analysis of scRNA-seq and stRNA-seq data collectively revealed that immunotherapy resistance was linked to cellular crosstalk between cholesterol-biosynthetic tumor cells and germinal center (GC) B cells within tertiary lymphoid structures. Notably, single-cell, spatial data, and multiplex immunohistochemistry demonstrated that cholesterol biosynthesis-associated ESCC cells express elevated levels of MIF. This disrupts GC reactions by competing with the CXCL12-CXCR4 signaling axis via MIF-CXCR4 interactions, thereby impairing B cell-mediated immunity. Conclusions: MIF+ tumor cells in GCs may be a biomarker for predicting immunotherapy resistance in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b8a1ee6ff72d48f3e16c31d484505ab587cac16" target='_blank'>
              MIF-expressing tumor cells mediate immunotherapeutic resistance in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Jing Song, Xiaomei Song, Yue Xie, Hong Guo, Yupeng Cun
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc0251706b1f215af0fc6511724feaeecccbbd1" target='_blank'>
              Wakhan: reconstruction of chromosome-scale copy number profiles of tumor genomes with long-read sequencing
              </a>
            </td>
          <td>
            T. Ahmad, A. Keskus, S. Aganezov, A. Goretsky, I. Rodriguez, B. Yoo, L. Lansdon, E. Repnikova, L. Zhang, Y. Liu, A. Donmez, A. Bryant, S. Tulsyan, J. Park, J. Gardner, B. McNulty, S. Sacco, J. Shetty, Y. Zhao, B. Tran, S. Malikić, C.-P. Day, K. Miga, B. Paten, C. Sahinalp, M. Farooqi, M. Dean, M. Kolmogorov
          </td>
          <td>2025-12-15</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee41f49d06aaeb019cdfc5399ce87f114ac10230" target='_blank'>
              Clinical Implications of SRPK1 Expression in Human Tumours: A Comprehensive Pan-Cancer Analysis Based on Multi-Omics Databases
              </a>
            </td>
          <td>
            Duygu Duzgun, S. Oltean
          </td>
          <td>2025-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) infects mainly CD4+ T lymphocytes and causes both malignant and inflammatory diseases: the aggressive malignancy known as Adult T-cell leukemia/lymphoma (ATL) and several chronic inflammatory syndromes. HTLV-1 infection is established by integration of proviral DNA (~9000 bp) into the host genome. In HTLV-1, two viral genes, tax and HBZ, play critical roles in viral transcription and promotion of T-cell proliferation, respectively. The present study was undertaken to test the hypothesis that the higher-order structure of proviral chromatin regulates its transcription on both the plus strand and the minus strand. ATAC-seq analysis identified an open chromatin region in the pol gene of proviral DNA, which we name IPOR, in many ATL cases. Using reporter assays, it was found that the sequence of IPOR suppresses the transcription of the plus strand and activates that of the minus strand. Binding motifs of Eomes and TEAD proteins were predicted in this region, and we confirmed recruitment of the transcription factors to their respective motifs by ChIP-qPCR. A mutant of IPOR which cannot bind the transcription factors weakened the transcriptional activating effects compared with the wild type, suggesting that the IPOR and those transcription factors suppress the 5' long terminal repeat (LTR) but activate the 3'LTR. In addition, a mutant HTLV-1 molecular clone, which possesses the IPOR mutant, produced a higher titer of virus than the wild type. RNA-seq analysis of HTLV-1-infected cell lines, in which Tax expression can be traced after induction, revealed that EOMES expression decreases during the tax transcriptional burst and resumes following termination of the burst. These findings suggested that the expression dynamics of Eomes affect the transient expression of Tax. The IPOR sequence appears to regulate the transcription from both LTRs, suppressing the 5'LTR but activating the 3'LTR. Recruitment of Eomes to the IPOR is likely to influence the expression of Tax and HBZ. Since both viral genes are involved in diverse mechanisms for viral replication, cellular proliferation, and immune regulation, the IPOR may play a role in fine-tuning the modes of viral persistence in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2f1e6ae01e0a8f9b5438f95ad7675208ffd69" target='_blank'>
              Intra-pol proviral open region of HTLV-1 controls the transcription from both long terminal repeats
              </a>
            </td>
          <td>
            Miyu Sonoda, Azusa Tanaka, Xueda Chen, Wenyi Zhang, Helen Kiik, S. Ramanayake, K. Nosaka, N. Takenouchi, Masanori Nakagawa, Akihiro Fujimoto, C. Bangham, Masao Matsuoka, Jun-ichirou Yasunaga
          </td>
          <td>2025-12-08</td>
          <td>Retrovirology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="TP53 mutations are among the worst prognostic factors in acute myeloid leukemia (AML), with affected patients facing relapse-free survival of just five-to-six months compared to TP53 wild-type patients. A major barrier to improving outcomes lies in the dearth of effective therapies, as TP53 mutant patients remain refractory to conventional cytotoxic chemotherapies, targeted therapies, and even allogeneic stem cell transplantation. In this review, we first summarize current clinical strategies and the major setbacks of p53 activators, MDM2/X regulators, and immunotherapy, highlighting the disconnect between promising pre-clinical studies and limited durable clinical responses. We next discuss the mechanisms of therapy resistance in TP53 mutant AML, with specific emphasis on dysfunction in the mitochondrial apoptotic pathway and clonal evolution of TP53 mutant hematopoietic stem cells. We then outline a roadmap for developing tailored therapies that may finally redefine prognosis for this high-risk patient population, including apoptotic activators, cell-cycle modulators, and immune- and metabolic-based therapies. We lastly call attention to new biomarker-driven approaches that can improve patient stratification and optimize identification of responders. By connecting mechanistic understanding with translational insights, this review underscores both the formidable challenges and the emerging opportunities in TP53 mutant AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2b2b9dab76dc863b017965cd1c9f9debef78076" target='_blank'>
              Pandora’s Box of AML: How TP53 Mutations Defy Therapy and Hint at New Hope
              </a>
            </td>
          <td>
            E. Olesinski, Shruti Bhatt
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Nucleotide substitutions are common in cancer cells, and they occur in both protein coding regions and non-coding regions (5′ and 3′ UTRs and introns). Although substitutions in non-coding regions have the potential to alter gene expression, it is the alteration of coding regions that affects protein function and has the most drastic effect on cellular transformation. Mutations in certain genes (e.g., TP53, KRAS) are common to nearly all cancers, but most cancers are characterized by specific gene mutation signatures. In this report, we investigated nucleotide substitution signatures in coding regions of the top 25 most frequently mutated genes in multiple human cancers. The goal was to examine whether unique nucleotide substitution biases are associated with various cancers. A pan-cancer analysis showed that the most altered nucleotide is guanine, which is biased towards G->A transitions. A per-cancer analysis identified ten cancers with biased substitutions in certain genes. Some of these biases were expected (e.g., KRAS in gastrointestinal cancers or JAK2 in blood cancers). Our analysis revealed biased signature substitutions in 17 genes, of which 14 were characterized as drivers and constituted a closely related set of cell cycle regulators. We conclude that nucleotide substitution biases contribute to specific alterations in cancer genes that produce cellular transformation. Principle component analysis of nucleotide substitutions shows that most cancers cluster together, meaning that they have similar nucleotide changes. However, certain cancers, most notably lung, pancreas, and blood cancers, can be differentiated from each other based on specific nucleotide signatures. Thus, nucleotide substitution patterns can be used to differentiate between some cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa7149b8cff77e69abfdaa8112e2209417108ee" target='_blank'>
              Nucleotide Substitution Biases in Related Cancer Driver Genes
              </a>
            </td>
          <td>
            Adam Khadre, Yifan Dou, Golrokh Mirzaei, Ruben C. Petreaca
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/362966d97ea996aa6c3590d2dcc1c33fc19cf534" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular and molecular changes in EGFR-positive lung adenocarcinoma before and after Furmonertinib treatment.
              </a>
            </td>
          <td>
            Jian Chen, Minghui Cai, Lifei Meng, Ze Hong, Wentao Hu
          </td>
          <td>2025-11-27</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, A. Feng, J. Richmon, D. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cellular plasticity, the ability of cells to dynamically alter their phenotypes, is a key driver of tumor evolution. This process is a hallmark of cancer which enables the acquisition of malignant traits, leading to metastasis, progression, and therapy resistance. It is governed by cell‐intrinsic factors, such as genomic instability and epigenetic reprogramming, and extrinsic stimuli from the tumor microenvironment. However, a unified framework is still needed to position plasticity as the central process that links these drivers to diverse cancer hallmarks. In this review, we first explore how plasticity enables key steps of tumor evolution, including tumorigenesis, metastasis driven by epithelial–mesenchymal plasticity (EMP), therapy resistance, and cancer stem cell (CSC) dynamics. We then summarize the intrinsic and extrinsic mechanisms that govern this adaptability. Finally, we discuss clinical advances in monitoring and targeting plasticity and highlight how new spatiotemporal technologies can address current research challenges. This review provides a framework positioning cellular plasticity as a central mechanism in cancer evolution, connecting its fundamental drivers to clinical translation. By synthesizing the latest advances, we offer perspectives for developing therapies that integrate prediction, monitoring, and targeting of plasticity to proactively guide cancer evolution toward manageable outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683e6aa0ee59731275431238ec93cc43ffca8292" target='_blank'>
              Tumor Cell Plasticity in Cancer: Signaling Pathways and Pharmaceutical Interventions
              </a>
            </td>
          <td>
            Shangwei Sun, Yunwei Sun, Ling Lan, S. Luan, Jin Zhou, Jiehui Deng, Yong Yuan, Zhong Wu
          </td>
          <td>2025-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Understanding DNA double-strand break (DSB) repair is crucial for the development of targeted anticancer therapies, yet the roles of many genes remain unclear. Recent studies show that disruption of known DSB repair genes can alter the sequence-specific distribution of mutations arising after DSB repair, suggesting that genome-wide perturbation screens could be leveraged to identify new DSB genes leading to distinct deviations from the expected wild-type distribution. Given the challenges of designing controls for a genome-wide screen, we explore the high gene throughput to forgo the use of traditional controls by reframing the analysis as an outlier detection problem, assuming that most genes have minimal influence on DSB repair outcomes. We propose MUSICiAn (Mutational Signature Catalogue Analysis), a compositional data analysis method that ranks gene perturbation impact on mutational spectra without controls by measuring deviations from the central tendency considering the distribution of all spectra. We show that MUSICiAn effectively estimates pseudo-controls for the Repair-seq screen, yielding 476 genes and 60 nontargeting controls. We further apply MUSICiAn to the first genome-wide screen of 18 406 genes with mutational spectra readout, MUSIC, reporting that MUSICiAn successfully recovers known DSB repair genes, highlights the spliceosome as a lesser-appreciated player, and reveals candidates for further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16a1d3962960ca885e8c3efafcd749c2130bf3c7" target='_blank'>
              MUSICiAn: genome-wide identification of genes involved in DNA repair via control-free mutational spectra analysis
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2026-01-06</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cabefaf605ac94e03143b5c4643d22a1076d5d" target='_blank'>
              miRNA-mediated cell-to-cell communications boost DNA repair during the Radioadaptative Response
              </a>
            </td>
          <td>
            María del Carmen Domínguez-Pérez, María Jesús Fernández-Ávila, Lourdes González-Vinceiro, L. Zannini, Héctor Peinado, Román González-Prieto, Néstor García-Rodríguez, Pablo Huertas
          </td>
          <td>2025-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding how chemotherapy perturbs cell cycle dynamics is critical for advancing cancer treatment. We develop a probabilistic, multi-generational framework based on a Bellman–Harris branching process to quantify treatment-induced shifts in tumor cell dynamics. The model incorporates key drug-responsive behaviors, including checkpoint activation, apoptosis, and checkpoint adaptation that propagates inherited DNA damage, enabling the characterization of heterogeneous survival outcomes after treatment. Biological parameters map directly onto DNA repair fidelity and cell-fate decisions, providing mechanistic insights beyond what is accessible from experiments alone. Dose-dependent extensions further allow exploration of treatment-induced perturbations. Model parameters were calibrated to empirical cell cycle measurements using the robust adaptive Metropolis algorithm. Global sensitivity analysis shows that scale parameters governing unfaithful DNA repair under G2/M- and S phase–specific agents exert major influence on model predictions, particularly at later time points. Across three chemotherapies, the framework reveals consistent dose-dependent alterations in cell cycle dynamics, with higher doses driving pronounced disruptions. Together, these results demonstrate how model-informed analyses can provide quantitative insight into treatment-induced cell cycle perturbations and support the refinement of therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/130ec8b5cedf3ce34952bca8649e510976ff8a38" target='_blank'>
              Probabilistic modeling of cell cycle dynamics in response to cell cycle targeting chemotherapy drugs to guide treatment strategies
              </a>
            </td>
          <td>
            Chenhui Ma, Evren Gurkan-Cavusoglu
          </td>
          <td>2025-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Head-and-neck squamous cell carcinoma (HNSCC) remains a lethal malignancy with stagnant survival despite advances in surgery, radiotherapy, and systemic therapy. Beyond cetuximab and PD-1 inhibitors, there are only a few targeted options, which benefit only a minority of patients, underscoring the need for new biomarkers and druggable dependencies. Genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) Cas9 screening now enables systematic, high-specificity investigation of gene function to reveal determinants of tumor proliferation, survival, and therapy response. Compared with RNA interference, CRISPR provides cleaner on-target knockout and more interpretable phenotypes, allowing efficient discovery of essential genes and synthetic-lethal interactions. Although the Cancer Dependency Map profiled 89 OSCC/HNSCC lines to nominate baseline dependencies, drug-perturbed states critical for understanding platinum resistance remain underexplored. Only a handful of HNSCC studies have applied genome-wide CRISPR cas9 screening: two mapped core essential genes; two mapped cisplatin resistance and radiation resistance; and others uncovered synthetic-lethal targets, including vulnerabilities to mTOR inhibition, EGFR inhibition, glutamine metabolism inhibition, and host determinants of oncolytic HSV-1 efficacy. This review synthesizes these findings, highlights methodological considerations (library design, coverage, and treatment duration), and integrates complementary functional data to prioritize targets for rational combinations. This review also provides information on the TCGA database and in vivo CRISPR screening that can accelerate precision therapeutics for patients with HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31c18994958d899b77bdbb4cbe1e355e7819ca3" target='_blank'>
              Target Discovery in Head-and-Neck Squamous Cell Carcinoma: Genome-Wide CRISPR Screens Illuminate Therapeutic Resistance and Actionable Dependencies
              </a>
            </td>
          <td>
            V. K. Vincent-Chong
          </td>
          <td>2025-12-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Senescence is a pleiotropic phenotype that alternatively suppresses or promotes cancer. The tumor suppressive roles are linked to clearance of damaged cells by the immune system, while cells with tumor promoting functions resist apoptosis, evade immune clearance and either persist and support the tumor microenvironment, or escape and proliferate. What generates these diverse populations is unclear. We investigated this in preneoplastic zebrafish livers where the epigenetic regulator, UHRF1, is overexpressed in hepatocytes. Double strand breaks, DNA methylation repatterning, retrotransposon expression, cell cycle withdrawal and activation of Atm and Tp53 dependent senescence were early responses to UHRF1 overexpression. This evolved to generate diverse populations of senescent cells, some which expressed immune and senescence signatures plus anti-apoptotic markers, and others co-expressed proliferative genes. The fate of these populations was dictated by UHRF1 levels and Tp53, as Tp53 loss enabled proliferation of cells with reduced UHRF1 expression but not in cells expressing high UHRF1. The senolytic Navitoclax targeted only a subset of senescent cells. Thus, the diversity of senescent cells driven by epigenetic changes can generate divergent outcomes. Short summary UHRF1 overexpression in zebrafish hepatocytes induces DNA damage leading to Atm–tp53–dependent senescence with UHRF1 levels dictating whether hepatocytes remain senescent, escape, or undergo senolytic elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b990cf440846339c231eccc76316041249b0f0d" target='_blank'>
              UHRF1 Overexpression Generates Distinct Senescent States with Different Tp53 Dependencies
              </a>
            </td>
          <td>
            Elena Magnani, Charlene Chen, F. Macchi, Bhavani P Madakashira, Tijana Randic, Ian McBain, Kirsten C. Sadler
          </td>
          <td>2025-11-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b4d870dc4ba763551ef81671814fc477f8046f" target='_blank'>
              CancerSTFormer enables multi-scale analysis of spot-resolution spatial transcriptomes and dissects gene and immune regulatory responses to targeted therapies
              </a>
            </td>
          <td>
            Benjamin S Strope, Dana Varghese, William Z Bowie, Stacy Wang, Qian Zhu
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Achieving precise and efficient integration of gene-sized DNA sequences into the human genome remains a major obstacle to gene therapy. Existing approaches depend on double-strand DNA breaks, which can lead to unintended genome alterations. Many monogenic diseases arise from diverse patient-specific mutations, making individualized correction impractical and underscoring the need for universal full-gene replacement strategies. We developed INsertion by Targeted Anchoring and Conditional Transposition (INTACT) to enable targeted insertion at genomic safe harbor loci. We engineered a mammalian transposase with mutations in its DNA-binding domain to reduce off-target integration. Site specificity was then restored by linking programmable sequence-specific DNA-binding proteins to the transposase. Systematic optimization of INTACT revealed key determinants of precision, including noncovalent linkage between the transposase and DNA-binding protein, strict spacing between the binding site and the TTAA insertion sequence, and linkage of the DNA-binding protein to an internal position within the transposase. On-target insertion was achieved across multiple loci, with optimized INTACT averaging 1.2 targeted insertions per cell. An off-target assay confirmed that DNA-binding domain mutations substantially reduced unwanted integration events to near-background levels. Our site-directed transposase enables precise, efficient genomic insertion of >4 kb DNA without double-strand breaks, offering a powerful new tool for genome engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dab1f9e5fe9c69a3e2717cbaf0dfc3ffa7c602b3" target='_blank'>
              Gene-sized DNA insertion at genomic safe harbors in human cells using a site-directed transposase
              </a>
            </td>
          <td>
            James E. Short, Lisa Sharek, Joshua F. Meckler, I. Stoytchev, Christopher T. Tran, Clara Errard, Brian E. Hew, Brandon E. Johnson, David F. Waller, Shanfu Xie, Joseph J. Higgins, Raymond Tabibiazar, Jesse B. Owens
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most common form of primary liver cancer and has limited therapeutic options. Epigenetic dysregulation plays a crucial role in hepatocarcinogenesis; however, its impact on cis-regulatory elements (CREs) and retrotransposons remains underexplored. Here, we investigated the epigenetic changes underlying the aberrant CRE and retrotransposon activity in HCC. We show that focal DNA hypomethylation of these elements is associated with transcriptional reprogramming. Notably, we uncover a dual regulatory mechanism for GPC3, a key diagnostic biomarker and immunotherapeutic target in HCC. This requires the concomitant reactivation of a fetal liver super-enhancer (SE) and DNA hypomethylation of CpG islands. Furthermore, DNA methylation loss drives cryptic activation of retrotransposons, some of which exhibit prognostic potential. Intriguingly, we identify a HERVE-int-derived long non-coding RNA that shows higher expression in patients with more aggressive tumors, poorer disease outcomes, and is correlated with molecular signatures associated with improved response to immunotherapy. Collectively, our findings reveal widespread epigenomic dysregulation of CREs and retrotransposons in HCC, highlighting new potential therapeutic strategies and biomarkers. Integrative epigenomic profiling of hepatocellular carcinoma reveals dual regulation of the immunotherapeutic target GPC3 and identifies retrotransposons as potential biomarkers for prognosis and immunotherapy response prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed2d6556ed99cbdaa0f49af5ea87b73a7cb6583" target='_blank'>
              Multi-omic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potential
              </a>
            </td>
          <td>
            Clooney C Y Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y J Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B S Lai, K. Yip, Alfred S Cheng, Danny Leung
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="R-loops consist of double-stranded DNA-RNA hybrids and a complementary DNA strand that is displaced from the duplex. R-loops play important role in numerous normal physiological processes, including DNA methylation, chromatin remodeling, RNA editing, replication, DNA repair, immunoglobulin class switching, and chromosome segregation during cell division. However, excessive or untimely formation of R-loops can lead to replicative collapse and subsequent DNA damage, resulting in genomic instability. One type of genomic rearrangements that is strongly associated with cancer malignancy is the extrachromosomal amplification of genes on circular DNA molecules (ecDNA). These molecules are relieved of hereditary constraints and conventional segregation laws and can endow cancer cells with the ability to rapidly change their genome, thereby accelerating tumor evolution and the development of therapy resistance. Multiple lines of evidence indicate that upregulated transcription of a gene can increase its susceptibility to amplification. Although the mechanisms underlying these processes are not yet fully understood, R-loops may play an important role in initiating gene amplification. In this review, we highlight the role of R-loops in replicative collapse, double-strand breaks, and DNA damage repair. We also provide examples of gene amplifications that is known to be induced by R-loops. Finally, we discuss amplification mechanisms in which involvement of R-loops has not yet been demonstrated, but appears highly likely.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d383d8d8b2013ac53fe539c01740ab644c592c14" target='_blank'>
              R-loops as a trigger for intra- and extrachromosomal DNA amplification in cancer
              </a>
            </td>
          <td>
            Tatyana F. Kovalenko, Amal Abdurazakov, Nadezhda V. Antipova, Mikhail I. Shakhparonov, M. Pavlyukov
          </td>
          <td>2025-12-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="


 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents, including doxorubicin, are common. Therefore, this project aims to characterize the evolution of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from the ATRT cell line, BT16, and RMC cell line, PEDS005T, utilizing a dose escalation approach. This resulted in DR cell lines that are 10-to-30-fold resistant compared to the parental cell line. Furthermore, we generated two independent DR cell lines from each parental cell line, enabling the capture of distinct evolutionary trajectories leading to resistance. We profiled the transcriptomes by bulk RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1, in 3 of 4 models. To assess whether ABCB1 upregulation alone was sufficient for DR, we treated them with an ABCB1 pharmacological inhibitor and found that ABCB1-upregulated DR cells were re-sensitized to doxorubicin, whereas an ABCB1-negative BT16 DR cell line remained resistant. To determine a shared mechanism of resistance between these models, we profiled the methylomes by Oxford Nanopore Technologies long-read sequencing. We observed global hypermethylation when compared to the parental counterparts. Treatment with a hypomethylating agent reduced cell viability up to 75%. Future studies will focus on identifying and therapeutically targeting these commonalities to overcome resistance regardless of evolutionary trajectory. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there are different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, reduced cell viability resulting from treatment with a hypomethylating agent suggests there are common vulnerabilities despite differing evolutionary trajectories.



 Katie T. Skinner, Benjamin P. Lee, Jessica S. Yoon, Andrew L. Hong. The evolution of doxorubicin resistance in SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B017.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ebc129836d1140b7ad59722551b94ec70c22ad5" target='_blank'>
              Abstract B017: The evolution of doxorubicin resistance in
 SMARCB1
 -deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P. Lee, Jessica S. Yoon, Andrew L. Hong
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite substantial progress in the field of bladder cancer management, the disease continues to represent an important health issue characterized by increased recurrence and progression rates. This is largely attributed to cancer stem cells (CSCs), a unique cell subpopulation capable of self-renewal, differentiation and resistance to conventional anti-cancer therapies. At the same time, our understanding of cancer biology has been revolutionized by the identification of non-coding RNAs (ncRNAs), a heterogeneous group of RNA molecules that do not translate into proteins yet function as pivotal regulators of gene expression. Emerging evidence demonstrates that ncRNAs modulate key hallmarks of CSCs, including self-renewal, epithelial–mesenchymal transition and drug resistance. This review investigates the intricate interplay between ncRNAs and the core signaling pathways that underlie bladder CSC biology. Unravelling the nexus between CSCs and ncRNAs is crucial for developing novel diagnostic biomarkers, better prognostic tools and innovative therapeutic strategies for patients with bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13091e7103acf49fcdc024b48e3b7a3f0f4f481d" target='_blank'>
              The Crossroads of Cancer Regulation: Discussing the Role of Non-Coding RNAs in Bladder Cancer Stem Cells
              </a>
            </td>
          <td>
            Alexandros Georgiou, Dimitrios Triantis, M. Goulielmaki, V. Zoumpourlis
          </td>
          <td>2025-12-11</td>
          <td>Uro</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Cancer represents a disease in which genetic alterations reassert the dominance of the cell cycle over all other cellular processes. From the earliest stages of evolution, the coupling of energy utilization with nucleotide and DNA synthesis established replication as the central driver of cellular behavior. In cancer, this evolutionary logic is replayed in reverse. Hyperactivation of the cell cycle drives hyperactivation of its metabolic core, while the loss or inactivation of tumor suppressor genes, many of which are cell-type specific, links this accelerated proliferation with altered cell fate. With virtually unlimited energy available, malignant cells amplify nucleotide production and DNA replication without restraint. This Perspective proposes that anti-metabolites, long-standing pillars of cancer therapy, can be redesigned to target the main components of DNA metabolism. By rationally combining these anti-metabolites into synergistic triads (three anti-metabolites, well selected, administered together), therapy may dismantle the metabolic foundations of cancer and achieve more durable control across tumor types. The combinations that could yield meaningful progress are outlined and discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67e9bbd295e09ee137dd6241889d264b14d2be13" target='_blank'>
              Targeting energy, nucleotide, and DNA synthesis in cancer
              </a>
            </td>
          <td>
            Ben Zion Vider
          </td>
          <td>2025-12-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a29ad767de6b1b60629bc324b66e333d5c7c4a2b" target='_blank'>
              A Human Genetics Framework for De-risking Gene Editing Targets for Hematopoietic Cell and Gene Therapy
              </a>
            </td>
          <td>
            A. Dhall, Alyssa Pyclik, Matthew H. Ung, Julia Etchin, Y. Keschner, Huanying Gary Ge, Tirtha Chakraborty, J. Lydeard
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90832a3d9b2f9576b36552899274c52bd58fcd98" target='_blank'>
              RBM8A confers oxaliplatin resistance in gastric cancer by maintaining EGFR mRNA stability
              </a>
            </td>
          <td>
            Yichun Ma, Ying Xiang, Mei Yang, Yanmei Zhu, Yani Pan, Nannan Zhang, Xueni Fu, Ping Jiang, Yinya Pan, Hai Wu, Qiong Yan, Wenjun Li, Hongji Tao, Shangtao Mao, Yufei Tao, Jiale Li, Linzhe Su, Guangtao Gao, Fangmei An, Zhangding Wang, Guifang Xu
          </td>
          <td>2025-12-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Gene expression regulation is a stochastic process that can be modified by mutations not only in a deterministic manner but also in a probabilistic way, for example by changing the extent of cell-to-cell variability in gene expression (also called “gene expression noise”) without necessarily changing the expected (mean) expression level. Such mutations can act either in cis (perturbing expression noise at their own locus) or in trans (perturbing expression noise of a gene located elsewhere in the genome). Although systematic studies have successfully analyzed the properties of cis-acting mutations modulating gene expression noise, less is known about the type of genetic changes that may alter expression noise in trans. Here, we applied genetic mapping on yeast strains (Saccharomyces cerevisiae) generated by random mutagenesis and identified three mutations (in chs1, yme2 and msh1 genes) that changed the expression noise of a reporter gene in the nuclear genome regulated by the yeast TDH3 promoter. Each of these mutations affected either extrinsic noise (variability due to cell-specific factors), intrinsic noise (inherent variability within each cell), or both. Surprisingly, all three mutations targeted proteins located outside of the nucleus: Yme2 and Msh1 being involved in the maintenance of mitochondrial genome integrity, and Chs1 being necessary for the repair of cell-wall defects in freshly-born daughter cells. Our results reveal that mitochondrial state can modulate the extent of intrinsic expression noise of eukaryotic nuclear genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648b0095152d2b4dfed796867db1f69280813729" target='_blank'>
              Trans-acting mutations reveal non-nuclear modulators of both intrinsic and extrinsic gene expression noise in a eukaryote
              </a>
            </td>
          <td>
            Nena Martin, Julie Kleine-Schultjann, Quentin Boussau, Agnès Dumont, Hélène Duplus-Bottin, Laurent Modolo, G. Yvert, P. Wittkopp, F. Duveau
          </td>
          <td>2025-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Globally, breast cancer (BC) is the most prevalent cancer in women, and drug resistance is a foremost challenge in BC management that contributes to 90% of BC-related deaths. Drug resistance arises as cellular adaptation during treatment, but the underlying mechanisms remain complex and are not yet fully understood. Research studies have shown that most anticancer drugs are effective against the mass of tumor cells rather than Cancer Stem Cells (CSCs). They have the ability to become dormant upon exposure to anticancer drugs and acquire drug resistance over a period of time after silently modifying themselves to resist the drug environment. The maintenance of the stem cell phenotype is mostly driven by stem cell transcription factors (e.g., KLFs, Myc, ZEB2, RUNX1, TWIST1, OCT4, NANOG, FOXO3, SOX2, E2Fs, etc.), which are mediated by epigenetic changes in tumor cells at many levels. These recognized factors/pathways have been demonstrated to be promising therapeutic targets. Therefore, knowledge of epigenetics in regulating cancer stem cells via stem cell transcription factors may be a promising solution for the reversal of therapy-resistant BC. In this review, we emphasize stem cell transcription factors, cancer stem cells, and epigenetic regulation as critical drivers of drug resistance in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bb0cff2c052008da468be3376723f26d8ad1c7" target='_blank'>
              Interplay between Epigenetic and Transcription Factors Mediating Drug Resistance via Stem Cells in Breast Cancer
              </a>
            </td>
          <td>
            Komal Barhela, A. Chaudhary, Richa Mishra, Nupur Nupur, Sri Krishna Jayadev Magani, H. Rudayni, Sanjay Kumar, Binayak Kumar
          </td>
          <td>2025-12-09</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying T-cell acute lymphoblastic leukemia (T-ALL) is essential for developing more effective therapeutic strategies. Despite therapeutic advances, the role of RNA-binding proteins in the pathogenesis of T-ALL remains poorly understood. Here, we investigate the RNA-binding protein Quaking (QKI), identifying it as a key regulator of splicing with tumor-suppressive properties in T-ALL. Through the analysis of two independent pediatric T-ALL cohorts, we demonstrate that QKI expression is frequently reduced in T-ALL, particularly within the HOXA subtype, and this reduction correlates with poor overall and event-free survival. Using T-ALL cell lines, we show that QKI depletion induces widespread splicing alterations, with numerous events corroborated in patient samples. Transcriptome profiling indicates that QKI downregulation leads to broad changes in gene expression, notably affecting pathways related to cell cycle progression, cholesterol homeostasis, and epithelial-mesenchymal transition. Functional assays demonstrate that QKI overexpression in T-ALL cells significantly reduces cell proliferation, induces G0/G1 cell cycle arrest, and limits leukemia progression and dissemination, ultimately improving survival in xenograft models. Together, these findings provide compelling evidence that QKI functions as a regulator of RNA splicing with tumor-suppressive activity in T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca5ec1e411749bba69b71a539ea37c30a49dc190" target='_blank'>
              QKI dysregulation induces extensive splicing changes in T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Bruno Palhais, Nitesh D Sharma, Igor Fijalkowski, T. Pieters, D. Deforce, F. Nieuwerburgh, Pieter Mestdagh, Panagiotis Ntziachristos, K. Matlawska-Wasowska, P. V. Vlierberghe
          </td>
          <td>2026-01-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="One of the most important changes in the transformation of normal cells into tumor cells is metabolism. In order to satisfy the more active proliferation, migration and metastasis of cancer cells, abnormal changes occur in various pathways and molecules involved in metabolism, which eventually lead to metabolic reprogramming of tumor cells. This process involves the uptake of nutrients and changes in major metabolic forms. As an important part of post-transcriptional epigenetics, RNA methylation modifications can regulate RNA processing and metabolism, while dynamically and reversibly influencing the expression of specific molecules, thereby ultimately affecting diverse biological processes and cellular phenotypes. In this review, various types of RNA methylation modifications involved in cancer are summarized. Subsequently, we systematically elucidate the mechanism of RNA modification for metabolic reprogramming in cancer, including glucose, lipid, amino acid and mitochondrial metabolism. Most importantly, we discuss in depth the clinical significance of RNA modification in metabolic targeted therapy and immunotherapy from mechanism to therapeutic application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e3fd9d374b6d8826edc1b140acc672b5b30712" target='_blank'>
              RNA Methylation in Cancer Metabolism: from Mechanisms to Therapeutic Opportunities
              </a>
            </td>
          <td>
            Zeyu Wu, Yuncan Xing, S. Mei, T. Xiao, Fangze Wei, Qian Liu
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) have identified over 60 autosomal risk loci associated with clear cell renal cell carcinoma (ccRCC), yet the functional mechanisms underlying these associations remain largely unclear. To establish connections between risk variants and their target genes, we applied the activity-by-contact (ABC) model, which integrates epigenomic data and Micro-C interactions, complemented with renal-specific quantitative trait loci, to predict enhancer-gene relationships. Our analyses implicate variation in hypoxia sensing, cell cycle regulation, and telomerase maintenance pathways as central mediators of ccRCC risk. These findings provide new insights into the molecular basis of ccRCC susceptibility and highlight potential therapeutic avenues for prevention and treatment. Integration of genome-wide association data with epigenomic and chromatin interaction data identifies target genes mediating clear cell renal cell carcinoma risk, implicating disrupted hypoxia signalling, cell cycle, and telomerase regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26c65e87cfb98c7e3f6c3d6a8a7f0754996d3cf" target='_blank'>
              Deciphering genetic susceptibility to clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Maria Mandelia, Philip J Law, Charlie Mills, M. Went, J. Vijayakrishnan, R. Houlston
          </td>
          <td>2025-12-01</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c863981147c795d5bd3bdf0f0373efc730fa086e" target='_blank'>
              Genetic identification of the RAS proteostatic machinery and its failure to regulate oncogenic variants
              </a>
            </td>
          <td>
            J. Bigenzahn, F. Kartnig, Manuela Vollert, V. Sedlyarov, Giulio Superti-Furga
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abedd58342641982b4fa326716bab2f81320dd" target='_blank'>
              Coordinate Up-regulation of Both TK1 and TS Expression in Cycling Cells: Mechanistic Basis and Implications for Dual-Targeted Cancer Therapy
              </a>
            </td>
          <td>
            Wanxing Wang, Jiawei Zhang, Guangyou Duan, Yiyao Zhang, Guofeng Xu, Zhi Cheng, Shunmei Chen, Xue Yao, Xin Li, Shan Gao
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Analysis of coding areas has long been used to study monogenic illnesses, but despite the extensive use of whole-exome sequencing (WES), up to half of suspected cases remain genetically unexplained. Variants outside coding areas can alter splicing, transcript stability, or gene regulation, compromising normal gene activity. These include mutations in noncoding RNAs, promoters, enhancers, deep intronic sequences, and untranslated regions (UTRs). Several well-known disorders have been linked to these mechanisms, including β-thalassemia caused by deep intronic mutations leading to aberrant splicing, familial hypercholesterolemia caused by promoter defects affecting LDLR expression, and inherited retinal diseases driven by noncoding variants influencing retinal gene regulation. These instances show that pathogenic variation is not limited to the exome and can have significant clinical implications. This review summarizes current understanding of noncoding and regulatory variants in monogenic diseases, discusses how they influence diagnosis and therapy, and highlights integrative approaches combining genomic, transcriptomic, and epigenomic data. Multi-layered research has increased diagnostic accuracy and unveiled new therapeutic potentials, although noncoding variations make the connection between genotype and phenotype more complex. Noncoding regions will need to be incorporated into standard diagnostic procedures to convert molecular insights into concrete therapeutic applications in the future. Predictive algorithms, patient-derived model systems, and functional validation testing will all help to simplify this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffd879f4fd9d85153dc94e05643890e26eb43d7" target='_blank'>
              Beyond the Exome: The Role of Noncoding and Regulatory Variants in Monogenic Diseases
              </a>
            </td>
          <td>
            Efthalia Moustakli, Nektaria Zagorianakou, Stylianos Makrydimas, Andreas Miltiadous, Alexandros T. Tzallas, G. Makrydimas
          </td>
          <td>2025-12-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The extensive use of next‐generation sequencing (NGS) multi‐gene panels and advanced analysis algorithms have led to the identification of numerous genetic variants associated with breast, ovarian, and pancreatic cancer. Copynumber variations (CNVs), defined as deletions and duplications of specific DNA regions, account for up to 10% of pathogenic variants and can affect any of the cancer‐predisposing genes. Despite this, CNVs’ contribution beyond BRCA1 and BRCA2 remains underexplored. This observational study analyzed data from 2949 patients, primarily affected by breast or ovarian cancer, who underwent NGS testing with a 22‐gene hereditary cancer panel between 2018 and 2023, with a focus on CNV results. In line with comparison studies, a total diagnostic yield of 14.8% was observed with pathogenic variants in BRCA1, BRCA2, CHEK2, ATM, and PALB2 accounting for most of positive findings. In contrast, CNVs were found in 1.4% of patients, displaying a peculiar distribution pattern. PALB2 exhibited the highest frequency of pathogenic CNVs (66.7%), representing 62.2% of all PALB2 pathogenic variants. Notably, 24 out of 28 PALB2 CNV carriers shared the deletion of Exon 11. Further investigations revealed identical breakpoints and common geographical origins, and moreover, the same haplotype for some of the families suggests a relatively recent founder effect. Simultaneous sequence and copy number analyses resulted in likely higher positive predictive value of the test and, more interestingly, disclosed an unforeseen single contribution of CNVs in PALB2 gene, confirming geography as a key factor in shaping human genetic variations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5409ce5039253d00afe3723c013d5dc9adb26b15" target='_blank'>
              Characterization of Copy Number Variants in Hereditary Cancer Patients Through NGS Shows a Distinctive PALB2 Contribution to the Diagnostic Yield
              </a>
            </td>
          <td>
            L. Bonamici, L. Artuso, Marco Marino, A. Toss, Diletta Sidoti, Elena Barbieri, M. Venturelli, I. Marchi, Chiara Pescucci, R. Manfredini, Laura Papi, M. Dominici, L. Cortesi, E. Tenedini, Enrico Tagliafico
          </td>
          <td>2026-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract CircleBase V2 (http://circlebase2.maolab.org) is a comprehensive update of our previously developed database, offering a significantly expanded and functionally enhanced resource for investigating extrachromosomal circular DNA (eccDNA) in human and mouse. This version introduces several key advancements: (i) a 12-fold increase in eccDNA data, comprising over 7.4 million entries from >300 types of cell lines and tissues, providing broader coverage across diverse physiological and disease contexts; (ii) inclusion of mouse eccDNAs, with >3.6 million entries from over 20 tissues, enabling robust cross-species comparative analyses and enhancing translational research with mouse models; (iii) a dedicated motif discovery module that systematically identifies junction-specific sequence patterns, shedding light on molecular mechanisms underlying eccDNA formation; and (iv) an improved functional scoring framework that integrates multi-omic annotations, including chromatin accessibility, regulatory elements, genetic variants, and chromatin interactions, to help prioritize functionally relevant eccDNAs. Additionally, CircleBase V2 features an upgraded user interface with enhanced visualization tools, supporting flexible querying, batch downloads, and integrative analyses. Collectively, these advancements establish CircleBase V2 as an indispensable platform for uncovering the biological roles and regulatory functions of eccDNAs in cancer, development, and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dd28d1f3215fd0927b181577ea1be3dbf04184" target='_blank'>
              CircleBase V2: an eccDNA annotation platform across cancers and species
              </a>
            </td>
          <td>
            Ling Wei, Leisheng Shi, Ning Wu, Hongyu Zhao, Zibaguli Wubulikasimu, Keyan Liu, Ming Deng, Fengbiao Mao, Xi Xiang, Hongsen Bi, Xiaolu Zhao
          </td>
          <td>2025-11-22</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Polyploidization is associated with progression of cancer, making cancer cells more dangerous. The common polyploid cancer cells constitute a considerable part of tumors (up to 56% in metastases). The giant polyploid cancer cells (PGCC), which appear under severe stress caused by treatment when the majority of cells die, present an enigmatic phenomenon both in fundamental and practical sense because they develop treatment resistance. Using transcriptome meta-analysis, we studied different types of polyploid cancer cells and found that in common polyploid cancer cells, the genes of unicellular (UC) origin and stemness are upregulated (compared to diploid cancer cells). At that, the upregulated UC genes show a higher local and global protein interactome centrality than the upregulated stemness genes, suggesting that the UC interactome attractor is a driving force behind this backward movement along the evodevo axis. Surprisingly, PGCC show the opposite picture. There occurs the suppression of UC and stemness genes with the upregulation of multicellular genes (especially those involved in intercellular communication), suggesting a reversal towards multicellular (MC) state. This effect is enhanced in PGCC's early progeny but diminished in the late progeny, indicating its transient nature. PGCC of different origin (breast, ovarian, prostate cancers), induced by different stresses (radiation or drugs with various mechanisms of action), show a similar behavior. The first principal component of transcriptome profiles, which is common for all cell types (initial cancer cells, PGCC, early and late progeny) and contains the major part of expression variance, is also directed along the gene evolutionary age axis. While the common polyploid cancer cells comply with the 'serial atavism' model of oncogenesis, PGCC present a unique phenomenon of the short-term return to multicellularity probably associated with collective acquisition of resistance to treatment. Our analysis revealed also the evolutionary origin of the main differences in gene expression, emphasizing the importance of gene age axis in transcriptome analyses. The deep evolutionary basis of variation in gene expression across and within cell types might become a general framework for interrelated problems of cell and cancer biology and regenerative medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f70142643bd7f756c6f81b5e68194318a9413f" target='_blank'>
              Polyploid giant cancer cells (PGCC): short-term return to multicellularity
              </a>
            </td>
          <td>
            Alexander E. Vinogradov, O. Anatskaya
          </td>
          <td>2025-11-24</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Molecular profiling of human primary cell types is essential for understanding human biology. We present a transcriptome and proteome map of 28 primary human cell types. Three major clusters of epithelial, endothelial, and mesenchymal cell types were observed in both the transcriptome and proteome levels along with the discovery of cell type enriched molecules including GRAP and C1orf116. The epithelial cell specific protein C1orf116 was further validated using immunohistochemistry across various human tissues. An exhaustive protein database search considering 39 post-translational modifications (PTMs) revealed novel insights into the PTM landscape including identification of understudied PTMs such as serine O-acetylation and histidine methylation. This also enabled comprehensive characterization of proteins with diverse PTMs. Interestingly, an unexpectedly higher frequency of dioxidation on tryptophan compared to methionine led to the identification of oxidative mitochondria complex subunit proteins. Further, a search strategy accounting for alternative translational start sites, splice junctions and translational readthrough refined genome annotation using proteomic evidence. For example, peptides from translational readthrough including extended sequence of LDHB and MDH1 were detected representing the first peptide-level evidence of these protein readthrough isoforms. Our comprehensive transcriptome and proteome data revealed cell type-specific molecular cues and heterogeneity, offering new insights into disease mechanisms often overlooked by tissue proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58bc593b57c9117036851d0c30cf78e5e938882b" target='_blank'>
              A transcriptomic and proteomic map of primary human cell types
              </a>
            </td>
          <td>
            Dong-Gi Mun, Anil K. Madugundu, S. Renuse, R. Nirujogi, Chan Hyun Na, Min-Sik Kim, Mayank Saraswat, Smrita Singh, M. G. Ramarajan, Shivani Tiwary, Jürgen Cox, Amol Prakash, Marc K. Halushka, Kathleen H. Burns, Richard K. Kandasamy, Akhilesh Pandey
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Chemotherapy resistance poses a significant challenge in the treatment of osteosarcoma. While DNA damage repair mechanisms play a crucial role in this resistance, effective intervention strategies remain limited. This study elucidates an intrinsic DNA damage repair mechanism in osteosarcoma cells treated with methotrexate (MTX) that can be effectively disrupted through the overexpression of ZFP36L1. Our findings indicate that ZFP36L1 expression is downregulated in MTX-resistant osteosarcoma cells. By overexpressing ZFP36L1 in osteosarcoma cell lines, we observed an increase in sensitivity to MTX. Further investigation revealed that the overexpression of ZFP36L1 reduces the efficiency of the DNA damage repair process, particularly by inhibiting the DCLRE1C-mediated non-homologous end joining (NHEJ) pathway. Through analysis of the 3′ untranslated region (3′UTR) of DCLRE1C mRNA, we identified 8 potential AU-rich elements (AREs) that bind to ZFP36L1. We demonstrated that ZFP36L1 directly interacts with DCLRE1C mRNA, leading to its degradation. In summary, decreased ZFP36L1 expression serves as an inherent mechanism enabling osteosarcoma to develop resistance to MTX therapy. These results highlight ZFP36L1 as a promising therapeutic target for overcoming MTX chemoresistance in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cea84915e514504007b9efa9609a988ba9924750" target='_blank'>
              Downregulation of ZFP36L1 contributes to methotrexate resistance in osteosarcoma through enhanced NHEJ DNA repair mechanisms
              </a>
            </td>
          <td>
            Jiahao Zhuang, M. Ma, Biao Yang, Yinliang Liu, R. Mi, Wen Yang, Yixuan Lu, Haoye Yu, Wangchang Wu, Yihui Song, Peng Wang, Hongyu Li
          </td>
          <td>2025-11-24</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Simple Summary Neuroblastoma (NB) is a childhood cancer that arises when certain nerve-related cells fail to mature properly and become malignant. Our study focuses on a gene called INSM1, which is normally active during early cell development but is abnormally elevated in NB tumors. We aimed to understand how INSM1 contributes to cancer’s aggressiveness and poor differentiation. We discovered that INSM1 keeps tumor cells in an immature, stem-like state and is closely linked to a chemical pathway called the methionine cycle, which affects gene regulation through epigenetic changes. Disrupting this cycle alters INSM1 activity and may help push cancer cells toward normal development. We also found that retinoic acid, a compound known to promote cell differentiation, reduces INSM1 and another cancer-driving gene, N-Myc. These findings suggest that targeting INSM1 and its metabolic regulation could be a promising strategy to treat NB by encouraging tumor cells to mature and lose their malignant properties. This research provides new insights into the biology of NB and may lead to more effective therapies for children affected by this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb4304aef088d414eab3a9ced76da697e7e9f2" target='_blank'>
              Regulation of INSM1 Gene Expression and Neuroendocrine Differentiation in High-Risk Neuroblastoma
              </a>
            </td>
          <td>
            Chiachen Chen, Siyuan Cheng, Xiuping Yu, Yisheng Lee, Michael S. Lan
          </td>
          <td>2025-12-22</td>
          <td>Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) represent a paradigm of cancer evolution, where spatial and temporal heterogeneity creates dynamic ecosystems that rapidly adapt to therapeutic pressures. Through multi-region multi-omics profiling of 45 patients across primary, irradiated, and recurrent tumors, we reconstructed evolutionary trajectories using an integrated analytical framework. DNA methylation (RRBS) was processed via Bismark, with CAMDAC deconvolution distinguishing cell type-specific signals from copy number alterations. Transcriptomic data were analyzed using Salmon and edgeR, while copy number landscapes were inferred through TitanCNA and InferCNV. Evolutionary dynamics were reconstructed using PRISM, enabling methylation-based phyloepigenome reconstruction to trace subclonal epigenetic evolution. Our analysis reveals that STS evolution follows a branching phylogenetic pattern, with early “trunk” alterations establishing the foundational epigenome and subsequent “branch” events driving radiation adaptation. PRISM-based methylation phylogenies demonstrated conserved evolutionary trajectories across patients, with epigenetic divergence increasing through treatment. At diagnosis, widespread methylome remodeling systematically rewires cellular identity, with convergent hypermethylation-mediated silencing of tumor suppressive networks (SMARCA4, DICER1, FOXO3) and hypomethylation-driven activation of oncogenic pathways (CCND1/2, CDK6). Across patients, these events were recurrent and enriched for functional gene expression changes, representing positive selection for survival advantages. Under radiotherapy selection pressure, the evolutionary dynamics shift from broad remodeling to targeted adaptation. The epigenome stabilizes, focusing on DNA repair (DDB2), stemness (ZBTB16), and proliferation (CCND1) pathways—evidence of directional selection for resistance mechanisms. Single-cell phylogenetics of >150,000 cells reveals that radiation selects for pre-existing subclones rather than generating novel populations, with overall ecosystem architecture maintained despite cellular turnover. Spatial-temporal mapping demonstrates that evolutionary trajectories vary dramatically both between patients and within individual tumors, generating multi-focal ecosystems where distinct subclones coexist, compete, and occasionally converge on shared adaptive solutions. This evolutionary framework explains radiotherapy resistance as the inevitable outcome of selection acting upon pre-existing heterogeneity. The convergent emergence of DNA repair enhancement, stem-like properties, and immunosuppressive niches across patients represents parallel evolution under common selective pressures. Together, these findings argue for the development of evolution-informed therapeutic strategies that anticipate and intercept adaptive trajectories, with the potential to overcome the formidable challenge of heterogeneity in treatment-resistant STS.



 Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, Raffi Avedian, Robert Steffner, David Mohler, Anusha Kalbasi, Matt van de Rijn, Gregory Charville, Everett Moding. Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9dbc7c928e0902865ecc73b6cf6d1b8b370047f" target='_blank'>
              Abstract B018: Decoding the evolutionary landscape of soft tissue sarcomas: From multiregion origins to therapy-driven adaptation
              </a>
            </td>
          <td>
            Shaghayegh Soudi, Nadia Silvia, Ajay Subramanian, Serey C L Nouth, Faith Ryu, Taryn Kaneko, Christin New, Deborah Kenney, R. Avedian, R. Steffner, David Mohler, A. Kalbasi, Matt van de Rijn, Gregory Charville, E. Moding
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The tumor suppressor RAD51D is essential for homologous recombination (HR). Pathogenic variants in RAD51D are associated with breast and ovarian cancers. However, most clinical missense variants are of unknown significance. We performed a multiplex assay of variant effect to test 6,888 RAD51D coding variants for loss-of-function. The resulting variant-to-function map perfectly separates known pathogenic and benign variants and is validated by orthogonal HR and biochemical assays across 70 clinical variants. Our screen shows that variants in the DNA-binding or ATPase core most severely compromise HR, and we identify the RAD51D-RAD51C interface within the BCDX2 complex as essential for regulating its ATPase activity. We hypothesize that, paradoxically, the primary function of RAD51D is to slow the ATPase activity of BCDX2, thereby allowing sufficient time and space for RAD51 filament assembly. Together, we identify hotspots of deleterious RAD51D variants and uncover the mechanisms by which variants compromise its biochemical functions. Highlights Used a multiplexed assay of functional effect (MAVE) to assess the functionality via olaparib sensitivity of 6,888 RAD51D coding variants, which can be used for variant classification Provided cellular functional analysis for 70 clinically-identified breast and ovarian cancer RAD51D variants Identified key regions and enzymatic activities of RAD51D critical for its function in the BCDX2 and the X3CDX2 complexes Determined mechanism of RAD51D-mediated regulation of BCDX2 ATPase activity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147f7b191823354d4eee4bfd6f57b8ab44617ab" target='_blank'>
              High-throughput mapping of 6,888 RAD51D variants identifies distinct biochemical functions needed for homologous recombination and olaparib response
              </a>
            </td>
          <td>
            Kristie E. Darrah, Shelby L. Hemker, Yashpal Rawal, Noah J. Goff, Phoebe S. Parker, Gayatri Ganesan, Caleb M. Stratton, Katherine Oppenheimer, Ella Roberts, Elena Glick, Nicole Banks, Arjun Kumar, Silvia Casadei, Matthew W Snyder, Katherine Nathanson, S. Domchek, Lea M. Starita, Shaun K. Olsen, Patrick Sung, J. Kitzman, Kara A. Bernstein
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/626532adf6dad0f909b1f0dfc7f2303a4960f3da" target='_blank'>
              Mapping the dynamic RNA binding proteome in human effector T cells identifies differentiation and cytotoxicity regulators
              </a>
            </td>
          <td>
            M. V. Lattanzio, Koos Rooijers, Kaspar Bresser, Antonia Bradarić, N. Kanagasabesan, Nikolina Šoštarić, Maia Nestor Martin, Nila H. Servaas, F. V. van Alphen, Arie J. Hoogendijk, M. Wolkers
          </td>
          <td>2025-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract The clonal evolution of breast cancer involves a complex dialogue between tumor cells and their environment. In this process, epigenetic mechanisms play a crucial role in regulating the cellular transcriptome without altering the underlying DNA sequence. Here, we provide an updated summary of three main epigenetic mechanisms: histone modifications, long non-coding RNAs (lncRNAs), and higher-order chromatin structures. Post-translational modifications of DNA or histones influence gene expression by altering chromatin accessibility and/or recruiting regulatory protein complexes. This process is dynamically regulated by enzymes that add or remove these marks, as well as by reader proteins that recognize them. Dysregulated expression or malfunction of these regulators creates an aberrant epigenetic landscape and gene expression profile, contributing to breast cancer initiation, metastasis, and drug resistance. Notably, the donor molecules for chromatin modifications are largely derived from intermediate metabolites shaped by environmental cues, highlighting the intricate crosstalk between epigenetic regulation and both cellular and systemic metabolic states. DNA and histone modifications are further interrelated with lncRNAs and higher-order chromatin architectures, which have been actively investigated in breast carcinogenesis. We also briefly introduce the role of epigenetics in other chromatin-associated events such as replication initiation. Aberrant replication initiation can drive gene duplication and genomic alterations resembling those observed in clinical breast cancer, endowing tumor cells with growth advantages and therapeutic resistance. Finally, we summarize emerging therapeutic strategies that target epigenetic vulnerabilities in breast cancer and discuss their current limitations and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502ad7aff3505dc7bf1c94f78c3fe440208426d" target='_blank'>
              Advances in the epigenetic regulation of breast cancer
              </a>
            </td>
          <td>
            Xiao Liu, Xin Wu, Jianan Ding, Xiaoying Qiao, Jing Liang
          </td>
          <td>2025-11-21</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bladder cancer is a highly heterogeneous malignant tumor of the urinary system with high recurrence rates, posing significant challenges in its diagnosis and treatment. Advances in multi-omics research have elucidated the molecular mechanisms underlying the pathogenesis and progression of bladder cancer, including driver gene mutations (e.g., FGFR3, TP53/RB1), dysregulation of signaling pathways (such as PI3K/AKT/mTOR and RAS-MAPK), epigenetic alterations, non-coding RNA networks, tumor microenvironment remodeling, and metabolic reprogramming. This review systematically summarizes recent progress in translational research bridging molecular mechanisms to breakthroughs in precision therapy, covering the clinical applications and challenges of FGFR inhibitors, immune checkpoint inhibitors, antibody-drug conjugates, and gene therapies. Translational efforts are increasingly relying on molecular subtyping to develop subtype-specific treatment strategies. Although significant advances have been made in precision therapy for bladder cancer, critical research gaps remain, including tumor heterogeneity, therapy resistance, and insufficient validation of biomarkers. Future research directions emphasize the potential of liquid biopsy for non-invasive diagnosis and dynamic monitoring, rational combination therapies, multi-omics data integration, and artificial intelligence in advancing personalized treatment, providing a systematic and forward-looking perspective on precision medicine in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14a6737ae3e58aea00cf12ac1c7b98be1125652" target='_blank'>
              Molecular mechanisms and translational advances in bladder cancer: from driver genes to precision therapy
              </a>
            </td>
          <td>
            Jinjie Xiang, Yuhui Luo, Baiyu Zhang, Kunbin Ke, Hao Li
          </td>
          <td>2025-12-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Interpreting protein-truncating variants predicted to escape nonsense-mediated decay (NMDe) based on the 50-bp rule is challenging due to their variable consequences. The Clinical Genome Resource, Cancer Genomics Consortium, and Variant Interpretation for Cancer Consortium recommendations focus on tumor suppressor genes where NMDe variants may result in loss-of-function. However, guidance for interpreting NMDe variants in oncogenes and dual-function genes remains limited. To address this gap, we screened the Catalogue of Somatic Mutations in Cancer, focusing on oncogenes and dual-function genes with at least 10 NMDe variants supported by published functional studies. This analysis prioritized 15 genes exhibiting two distinct NMDe patterns resulting in gene activation. The first pattern and gene examples involve NMDe variants causing loss of C-terminal regulatory regions that mediate protein inhibition and/or degradation, frequently manifesting as attenuated cell surface receptor internalization/degradation (e.g., CSF3R) or increased intracellular protein stability (e.g., CCND3). The second is driven exclusively by frameshift NMDe variants that generate novel peptide fragments that alter protein interactions (e.g., CALR). Interrogation of these genes in the St. Jude Children Research Hospital clinical genomics pediatric cohort identified 119 NMDe variants across 8 genes in ~3% (113/3,492) of unique patient samples, with 118 of these classified as likely oncogenic or higher (99 relating to the first pattern and 19 to the second). One variant was classified as of uncertain significance. Our data emphasize the need to integrate gene function, variant type, and effects on the C-terminus to comprehensively evaluate somatic NMDe variants and predict their consequences across adult and pediatric cancer cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27a7745ae6bf012b3eb019827bca6d4bb15b544" target='_blank'>
              Analysis and Interpretation of Somatic NMD-Escaping Variants in Oncogenes and Dual-Function Genes across Adult and Pediatric Cancer Cohorts
              </a>
            </td>
          <td>
            M. Eldomery, Karissa M. Dieseldorff Jones, Maria Namwanje, Runjun D Kumar, Jiaming Li, M. Wilkinson, Lu Wang, J. Klco, Li Tang, Jennifer L. Neary, Sharon E. Plon, P. R. Blackburn
          </td>
          <td>2025-11-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Resistance to anti-cancer drugs remains a major challenge in chemotherapy. Combination therapy using sensitizers has emerged as a promising strategy to restore drug sensitivity in resistant cancer cells. However, experimental screening of sensitizers is laborious and costly, highlighting the need for computational methods that enable systematic and efficient prediction. We developed SOCAR, a network-based computational framework that predicts sensitizer drugs by integrating transcriptome profiles with molecular interaction networks. SOCAR identifies resistance-associated genes and network modules, and quantifies each drug's potential to reverse these resistance mechanisms. Applied to 4,009 drugs, SOCAR accurately predicted candidate sensitizers for tamoxifen-resistant breast cancer (AUROC = 0.90). In vitro assays validated that all twelve top-ranked candidates significantly reduced cell viability (p < 0.005) when co-administered with tamoxifen. Furthermore, protein activity analyses showed that resistance-module proteins were markedly altered after acquiring resistance but were restored to normal levels following combined treatment (p < 0.05). Collectively, SOCAR provides a systems-level framework for discovering novel sensitizers and elucidating mechanisms of resistance reversal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab1acf5805cb5dd4152a1362e84cfaff70e9641d" target='_blank'>
              SOCAR: Network-based Computational Framework to Overcome Acquired Tamoxifen Resistance of MCF7 Cells.
              </a>
            </td>
          <td>
            Mijin Kwon, Young Min Woo, Woochang Hwang, Jong-Won Kim, Doheon Lee
          </td>
          <td>2025-12-05</td>
          <td>IEEE transactions on computational biology and bioinformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epigenetics encompasses heritable but reversible modifications of gene expression that occur without changes in the DNA sequence and involve mechanisms such as DNA and RNA methylation and histone modifications. These mechanisms modulate chromatin architecture, genome stability, and cellular responses to environmental stressors, and their dysregulation contributes to oncogenesis and cancer progression. In parallel, radiotherapy remains a cornerstone of cancer treatment; furthermore, ionizing radiation induces epigenetic modifications alongside direct DNA double-strand breaks and oxidative damage. Radiation-induced epigenetic changes, including global or locus-specific DNA methylation shifts (e.g., genes promoter CpG islets), histone acetylation and methylation imbalances, are increasingly recognized as key contributors to molecular radioresistance. These adaptive responses may enhance tumor cell survival, affect therapeutic efficacy, and promote metastasis. Understanding the interplay between radiation exposure and epigenetic remodeling opens new perspectives for precision oncology and diagnostics. Epigenetic biomarkers hold potential for predicting treatment response and prognosis, while epigenetic modifiers may sensitize tumors to radiation. This review summarizes current evidence on radiation-induced epigenetic mechanisms and evaluates their diagnostic, prognostic, and therapeutic implications in cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18126342fcb0010fdf4cce9fa40fdeeb69a2ac9b" target='_blank'>
              A New Look at the Role of Radiation-Related Epigenetic Mechanisms in Diagnosis and Anticancer Therapies
              </a>
            </td>
          <td>
            A. Olichwier, Magdalena Bruzgo-Grzybko, Izabela Suwda Kalita, Natalia Bielicka, Ewa Chabielska, A. Gromotowicz-Poplawska
          </td>
          <td>2025-11-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal malignancies of the female reproductive system, with its high mortality rate largely driven by chemotherapy resistance and disease recurrence. Ovarian cancer stem cells (OCSCs), a small subpopulation within ovarian tumors, are characterized by their capacity for self-renewal, differentiation, and tumorigenic growth. They are recognized as central drivers of tumor initiation, metastasis, drug resistance, and relapse. Mounting evidence in recent years has highlighted the pivotal role of metabolic reprogramming in sustaining OCSC stemness and therapeutic resistance. In this review, we reported the major metabolic pathways engaged by cancer stem cells (CSCs), including glucose metabolism (glycolysis, the tricarboxylic acid cycle, oxidative phosphorylation, and reactive oxygen species regulation), lipid metabolism, and amino acid metabolism. These pathways function to meet the bioenergetic and biosynthetic requirements of CSCs. Particular emphasis is placed on the metabolic plasticity of OCSCs, which can transform between a relatively inactive quiescent state and a highly proliferative active state. This adaptability allows OCSCs to respond dynamically to microenvironmental changes, facilitate ovarian cancer implantation and metastasis, and evade chemotherapeutic stress. We further analyze the molecular networks governing OCSC metabolism, including key signaling cascades and transcription factors. From a therapeutic perspective, we discuss the anti-diabetic drug metformin, which has demonstrated potential in targeting CSC metabolism in both preclinical models and clinical studies. Finally, we outline future research directions aimed at exploiting the metabolic vulnerabilities of OCSCs. We highlight that combination strategies targeting metabolism hold significant potential for overcoming treatment resistance and preventing ovarian cancer recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a98ef6b05ff0d2a035206d3941ef2e78eccd31c9" target='_blank'>
              The metabolic landscape of ovarian cancer stem cells: how do they survive?
              </a>
            </td>
          <td>
            Jixue Tan, Lin Tang, Qian Zhang
          </td>
          <td>2026-01-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small-cell lung carcinoma (SCLC) remains one of the most aggressive lung cancers and continues to pose a major challenge for precision oncology. Despite its morphological uniformity, SCLC exhibits marked molecular heterogeneity with recurrent, potentially targetable genomic alterations. Comprehensive profiling is often hindered by limited tissue availability and the need for rapid therapeutic intervention. We performed genomic profiling of 55 primary and metastatic SCLC samples using a 324-gene hybrid-capture next-generation sequencing panel. Consistent with prior reports, nearly all tumors exhibited biallelic TP53 and RB1 inactivation. Recurrent alterations involved the PI3K/Akt/mTOR pathway (62%), chromatin regulators (42%), and NOTCH signaling genes (15%). PTEN mutations were enriched in brain metastases. Frequent copy-number gains affected SOX2, NKX2-1, MYC-family genes, and CCNE1. Two novel recurrent amplifications of potential clinical significance were identified: TYRO3 (33%) and SDHA (13%). TYRO3, a TAM family receptor tyrosine kinase, and SDHA, a mitochondrial enzyme involved in succinate metabolism, may contribute to tumor progression and represent emerging therapeutic vulnerabilities. These findings underscore the genomic diversity of SCLC and highlight the potential utility of broad next-generation sequencing in uncovering new molecular targets for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c257de36865932aa9b4ccc1af139b71a5ba02b7" target='_blank'>
              Comprehensive Genomic Profiling of Small-Cell Lung Cancer Reveals Frequent Potentially Targetable Alterations
              </a>
            </td>
          <td>
            Dániel Schmalz, Zoltán Krabóth, Veronika Czoma, Péter Urbán, Attila Gyenesei, István Ruzsics, Veronika Sárosi, Á. Boronkai, Emőke Papp, B. Kajtár
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213457505483d2de87291e80a391cdd8fe7c3867" target='_blank'>
              Linking somatic mutations in cancer to the electronic properties of DNA
              </a>
            </td>
          <td>
            Benoît de Witte, Cyril Karamaoun, Pauline Hermans, Maxime Tarabichi, F. Pucci, Marianne Rooman
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c2bcfd8cc843d79fc1979b529c65ef07bebf80" target='_blank'>
              Targeted suppression of tumor growth by CREPT promoter-driven diphtheria toxin fragment A
              </a>
            </td>
          <td>
            Jingya Li, Mengdi Li, Wenchen Wang, Guancheng Jiang, Yinyin Wang, Jianqiu Sheng, Zhijie Chang, Jianqiu Sheng, Ming-yang Li, Fangli Ren
          </td>
          <td>2025-11-22</td>
          <td>Cancer Gene Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Cancer cells adapt to environmental and therapeutic pressures through karyotypic plasticity, including whole genome doubling, aneuploidy, and structural genome changes. Polyploid/polyaneuploid giant cancer cells (PGCCs/PACCs) represent an extreme state of this plasticity, arising through endoreplication as a common response to chemotherapy, irradiation, viral infection, hypoxia, or nutrient restriction. PGCCs act as drug-tolerant reservoirs that can later depolyploidize to repopulate tumors, underscoring their role in recurrence and therapy failure. Yet, how readily tumor cells access the PGCC state and how karyotype dynamics sculpt drug sensitivity remain poorly understood. To address this gap, we integrated (i) long-term evolution experiments (LTEEs), (ii) mathematical modeling, and (iii) computational analysis into a unified framework to define karyotype evolution as a predictive biomarker of therapy response. To study karyotype evolution under metabolic stress, we developed CLONEID, an LTEE platform that combines bright-field imaging, single-cell karyotyping and a neutral-drift framework distinguishing drift from selection. Its computer-vision module classifies PGCC versus proliferative states while PCA of other morphometrics revealed stress-specific phenotypes even without whole-genome doubling or overt ploidy change. LTEEs (>6 months) under glucose deprivation, phosphate restriction or hypoxia revealed stress- and ploidy-dependent trajectories. Glucose deprivation promoted whole-genome doubling whereas phosphate restriction and hypoxia drove chromosome loss toward near-diploid states. Strikingly, our in vivo studies confirmed hypoxia-induced ploidy reduction, which was predicted by LTEE trajectories within the 36 days and recapitulated by our model. Moreover, long-term adaptations also shifted therapy response. Glucose-deprived cultures gained resistance to gemcitabine and topotecan while phosphate-deprived cultures became more sensitive to taxanes and carboplatin. Furthermore, our functional assays showed that evolved cultures enhanced T-cell activation while control condition suppressed immunogenicity through tryptophan metabolism. To complement LTEEs, we built an ODE linear chain trick (LCT) model of the cell cycle, calibrated with isogenic diploid and tetraploid TNBC lines. Our model successfully captured ploidy-conditioned therapeutic responses. Specifically, gemcitabine, among 74 tested anticancer agents, eliminated near-diploid cells, but spared tetraploids that entered PGCC state and resumed proliferation after drug withdrawal. Overall, our findings identified that whole-genome doubling, polyploidy, and PGCC formation shape both long-term evolution and acute therapy responses. Integrating LTEEs with LCT modeling successfully links ploidy, karyotypic adaptation, and drug sensitivity. Our approach establishes the ground for “Drug–Karyotype” pairs as biomarkers and shows how metabolic niches sculpt KFLs, offering a path toward personalized, evolution-informed oncology.



 Vural Tagal, Jackson Cole, Richard J. Beck, Didem Ilter, Daria Myroshnychenko, Konstantin Martin, Andriy Marusyk, Ana P. Gomes, Noemi Andor. Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06671e4b962908638a954d43578035d059b71aa3" target='_blank'>
              Abstract B026: Modeling karyotype-driven adaptations to metabolic restrictions predicts therapeutic response and immunogenicity in cancer
              </a>
            </td>
          <td>
            Vural Tagal, Jackson Cole, Richard J Beck, Didem Ilter, D. Myroshnychenko, Konstantin Martin, A. Marusyk, Ana P. Gomes, Noemi Andor
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4f815c5ea5ff8465ffcedac66d5a28bbf1f8878" target='_blank'>
              Metabolic profiling of therapy-induced senescent cancer cells via TPEF, MALDI-MS, and RNA-sequencing.
              </a>
            </td>
          <td>
            S. Ghislanzoni, F. Padelli, Matteo Niero, A. Bertolotti, A. Belfiore, Simone Torelli, A. Bresci, Andrea Masella, Silvia Betti, Dario Polli, Luca Agnelli, I. Bongarzone
          </td>
          <td>2025-12-17</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ef5b0807e681dc02a817ada526d9c5fc0ddd59" target='_blank'>
              RegEvol: detection of directional selection in regulatory sequences through phenotypic predictions and phenotype-to-fitness functions
              </a>
            </td>
          <td>
            Alexandre Laverré, T. Latrille, Marc Robinson-Rechavi
          </td>
          <td>2025-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed06b707f53d2e1de85e352e5c2852fcf93afdb7" target='_blank'>
              Multiplexed CRISPR/Cas9 Editing of Tumor Suppressor Genes Recapitulates Molecular and Morphological Features of High-Risk Endometrial Cancer
              </a>
            </td>
          <td>
            Maria Vidal-Sabanés, R. Navaridas, Núria Bonifaci, Ada Gay-Rua, Damià Ortega-Peinado, Joaquim Egea, Mario Encinas, Xavier Matias-Guiu, David Llobet-Navás, Xavier Dolcet
          </td>
          <td>2025-12-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3de429a7817141d17bab7346b3911bc733e9f80d" target='_blank'>
              Integrated methylome analysis identifies an epigenetically silenced tumor suppressor RASA4 in small cell lung cancer.
              </a>
            </td>
          <td>
            Meng Fu, Qizhi Zhu, Jian Qi, Xiaopeng Zhang, Zhipeng Wang, Jialiang Wang, Xiang Huang, Zongtao Hu, Wei Wang, Bo Hong, Hongzhi Wang
          </td>
          <td>2025-12-31</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most common and lethal form of primary brain tumor, characterized by poor prognosis and limited treatment options, with a median survival of only 12 to 15 months post-diagnosis. Despite standard therapeutic protocols involving temozolomide (TMZ) and radiotherapy, effective treatment is challenging, underscoring the need to explore alternative therapeutic strategies. One promising approach is the inhibition of DNA damage response (DDR) pathways. RNaseH2, a key enzyme involved in ribonucleotide excision repair (RER), plays a critical role in maintaining genomic stability by removing ribonucleotides misincorporated into DNA. Thus, inhibition of RNaseH2 may enhance tumor mutational burden (TMB) and potentially transform immunogenically “cold” tumors into immunogenically “hot” tumors. In this study, a high-throughput screening (HTS) assay resulted in the identification of 52 potential RNaseH2 inhibitors from a library of 71227 compounds. We selected six of these inhibitors for further investigation, evaluating their effects both alone and in combination with TMZ in commercially available U87 MG wild-type and IDH1 mutant (IDH1 WT and MUT) glioma cells and patient-derived cells established from glioma organoids (GBO-PDC). This study highlights the therapeutic potential of RNaseH2 inhibition in combination with TMZ for GBM therapy, validated in patient-derived model, offering a promising avenue for treating this highly aggressive and yet uncurable cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e477cbaef78af07b0155030aaf5907db898c369d" target='_blank'>
              RNaseH2 inhibition potentiates temozolomide response in patient derived glioblastoma cells
              </a>
            </td>
          <td>
            Miroslava Kissova, Judit Martinez Segarra, T. S. Iveland, Marthe Vestvik, Erlend Ravlo, Wei Wang, Lars Hagen, N. Liabakk, María Cámara-Quílez, Miquel Arano Barenys, Ole Solheim, B. Hoff, E. Sundby, Magnar Bjørås, Geir Slupphaug, Torkild Visnes, Alessandro Brambilla
          </td>
          <td>2025-11-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="


 Untreated prostate tumors depend on androgen receptor (AR) for growth and thus are treated with hormonal therapy. However, resistance almost always emerges as tumors evolve into a castration-resistant state. The evolution of resistance can follow different paths, and castration-resistant prostate cancer (CRPC) exhibits multiple epigenomic subtypes: androgen receptor-dependent CRPC-AR, and lineage plastic subtypes CRPC-SCL (stem cell-like), CRPC-WNT (Wnt-dependent), and CRPC-NE (neuroendocrine). By transcriptomic profiling of tissue, and whole-genome sequencing (WGS) of tissue and cell-free DNA (cfDNA) from 500 patient samples, we relate genomic variants with epigenomic state. We annotate fractional contribution of each subtype for all patient samples using deconvolution approaches with a set of signature genes and accessible chromatin sites. We confirm that AR amplifications at the genomic level are associated with the emergence of CRPC-AR, and RB1 biallelic loss is associated with CRPC-NE. Importantly, we find chromosomal rearrangements in the YAP/TAZ pathway are associated with the presence of CRPC-SCL. In particular, we find complex rearrangements on chromosome 4, which are supported by patient-matched Hi-C data, and decrease promoter interactions of MOB1B, a YAP/TAZ pathway inhibitor, with its enhancers. Together, the genomic variants in the pathway can predict CRPC-SCL with 79% accuracy. By computing cancer cell fraction (CCF) of the genomic events, we find high concordance between the CCF of genomic events and the fraction of their associated epigenomic subtype. Thus, higher CCF of chr 4 rearrangements corresponds to higher tumor fraction of CRPC-SCL. Our study shows how genomic events enable transition to different CRPC states that exhibit differential therapeutic susceptibilities.



 Marjorie Roskes, Alexander Martinez-Fundichely, Sandra Cohen, Metin Balaban, Chen Khuan. Wong, Weiling Li, Tonatiuh A. Gonzalez, Anisha B. Tehim, Hao Xu, Shahd ElNaggar, Matthew Myers, Andrea Sboner, Benjamin J. Raphael, Yu Chen, Ekta Khurana. Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b27abd0fec4993dd28bd5604ff77de95b09d9b" target='_blank'>
              Abstract B013: Chromosomal rearrangements at the YAP/TAZ pathway genes are associated with heterogeneity and stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, Weiling Li, T. A. Gonzalez, A. Tehim, Hao Xu, S. ElNaggar, M. Myers, A. Sboner, B. Raphael, Yu Chen, E. Khurana
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412e59c514dd99d419339c7001f85dc5c6704bad" target='_blank'>
              Retroelements in thyroid cancer: epigenetic plasticity, dedifferentiation, and therapeutic opportunities.
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, B. Pessoa, G. D. A. Guardia, Juliana Moreira Marques, P. Galante, R. L. Batista
          </td>
          <td>2025-11-26</td>
          <td>Reviews in endocrine & metabolic disorders</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b68b7fa61f9edb29d61947a73ec839e1cdd7c8" target='_blank'>
              CRISPR-Cas9 technology: a breakthrough in cancer gene therapy
              </a>
            </td>
          <td>
            Z. I. Jameel
          </td>
          <td>2026-01-12</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genomic studies have revealed elevated damage and mutation rates in active transcription factor (TF) binding sites in UV-linked cancers. Previous investigations into the relationship between TF activity and UV DNA damage have primarily focused on select TFs or been done in aggregate across large cohorts of TFs at kilobase resolution. While collectively, there is evidence that TFs contribute to UV-induced mutagenesis by both enhancing initial damage formation and attenuating repair, there has yet to be a comprehensive characterization of these mechanisms on a per-TF basis. Using genome-wide maps of UV damage from human skin fibroblasts, we developed a scalable statistical framework to analyze TF-mediated mutagenic mechanisms across hundreds of TFs. We identify numerous previously unreported TFs that significantly enhance and / or inhibit damage formation in their binding sites. A systematic survey of TF-DNA complexes further revealed that positions of UV damage modulation coincide with TF-induced structural distortions that either protect or predispose DNA to photodimer formation. Additionally, we analyzed repair efficiency in TF binding sites with unprecedented resolution, identifying specific TFs and binding site positions likely to compete with repair factors. By comparing these results with skin cancer mutations, we distinguish mutation peaks driven by increased damage susceptibility versus attenuated repair, illustrating that TF-mediated mutagenesis is highly contextual and dependent on the TF, binding site position, and sequence context of the damaged locus. Our approach provides a robust statistical framework for elucidating mechanisms of mutagenic TF-binding and offers novel insights into the complex interplay between protein interactions, DNA damage, and repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87aa924971352d37048c9215f6b18e4948ab222" target='_blank'>
              High-throughput characterization of transcription factors that modulate UV damage formation and repair at single-nucleotide resolution
              </a>
            </td>
          <td>
            Raluca Gordân, Hana Wasserman, Bo Chi, Kaitlynne A. Bohm, M. Duan, Harshit Sahay, Alexias Safi, Gregory Crawford, Peng Mao, John J. Wyrick, Miles Pufall
          </td>
          <td>2025-12-10</td>
          <td>Research Square</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Summary SISTEM is a software package and mathematical framework for simulating tumor evolution and cell migrations at single-cell resolution. Unlike existing frameworks which simulate cancer cell populations under the neutral coalescent or using simple birth–death models, SISTEM simulates tumor populations under somatic clonal selection using an agent-based framework. SISTEM can generate mutation profiles, read counts, and DNA sequencing reads along with ground truth cell lineages and migration graphs under a number of easily customizable mutation and selection models. For improved realism, SISTEM allows for cell fitness to be driven by genomic events of various scales including single nucleotide variants, segmental gains and losses, whole-chromosomal and chromosome-arm aberrations, and whole-genome duplications. SISTEM also includes numerous migration models to simulate metastatic cancers, facilitating the exploration and evaluation of diverse migration patterns. Availability and implementation SISTEM is written in Python and is freely available open-source under GNU GPLv3 from: https://github.com/samsonweiner/sistem">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f6ea45c6bd4d361ab9e67581ccc6b02aecffc37" target='_blank'>
              SISTEM: simulation of tumor evolution, metastasis, and DNA-seq data under genotype-driven selection
              </a>
            </td>
          <td>
            Samson Weiner, M. Bansal
          </td>
          <td>2025-11-23</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="RNA modifications constitute a versatile and dynamic layer of post-transcriptional regulation that enables T lymphocytes to fine-tune gene expression programs in response to developmental, environmental, and pathogenic cues. Chemical marks such as N6-methyladenosine (m6A), 5-methylcytosine (m5C), and pseudouridine (Ψ) shape transcript stability, splicing, localization, and translation through coordinated actions of writer, reader, and eraser proteins. Emerging evidence reveals that these pathways orchestrate T cell lineage specification, activation thresholds, effector-memory balance, and immune tolerance, while their dysregulation contributes to infection, autoimmunity, malignancy, and graft rejection. Integrating findings across m6A and other epitranscriptomic marks—including m5C, Ψ, N7-methylguanosine (m7G), N1-methyladenosine (m1A), N4-acetylcytidine (ac4C), and N6-2'-O-methyladenosine (m6Am) —this review delineates how distinct RNA modifications converge on shared molecular circuits controlling transcriptional, metabolic, and signaling networks in T cell immunity. Aberrant modification patterns reshape cytokine profiles, mitochondrial metabolism, and antigen-driven responses, thereby influencing disease trajectories across diverse pathological contexts. Collectively, these insights establish RNA modification as a central regulatory axis linking transcriptomic plasticity to immune function and therapeutic responsiveness. We further highlight unresolved challenges—such as defining spatiotemporal modification landscapes and achieving selective pharmacological modulation—and propose integrative multi-omics and in vivo perturbation approaches to translate epitranscriptomic mechanisms into targeted immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31430e99c93ca7c4106fca21b4980e81ea35dc46" target='_blank'>
              RNA Modifications in T cell Immunity: Mechanisms, Disease Relevance, and Therapeutic Potential
              </a>
            </td>
          <td>
            Xinyuan Zhao, Meiyan Zou, Weiyao Feng, Nina Li, Xu Chen, Pei Lin, Zihao Zhou, Yunfan Lin, Li Cui
          </td>
          <td>2026-01-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a highly resistant brain tumor with a dismal prognosis despite standard treatments like temozolomide (TMZ) and radiation therapy (RT). This review delineates the molecular mechanisms driving this resistance to inform future therapeutic strategies. A narrative review was conducted using PubMed, synthesizing studies from the year 2000 up to March 2025 on DNA repair enzymes, TMZ and RT resistance pathways, anti-apoptosis mechanisms, genetic heterogeneity, and cell cycle dysregulation in GBM. Five key resistance mechanisms were identified: (1) DNA repair enzymes (e.g., MGMT, APNG) counteract TMZ-induced damage, regulated by NF-κB and CHD4; (2) TMZ and RT resistance depends on MGMT methylation and MMR functionality; (3) anti-apoptotic pathways (e.g., Bcl-2, PI3K/Akt) protect GBM cells, mitigated by TRAIL sensitization; (4) genetic mutations (EGFR, PTEN, TP53) confer adaptability; and (5) quiescent and proliferative GBM stem cells (GSCs) sustain recurrence. Novel therapies targeting HK2 and neural stem cells show promise. GBM resistance is multifaceted, requiring personalized, multi-targeted approaches. Future research should prioritize epigenetic modulation and combination therapies to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d28a9446444f9c597df4c24c1f9700021704068b" target='_blank'>
              Molecular Insights into Glioblastoma Resistance and Implications for Treatment Advancements: A Systematic Review
              </a>
            </td>
          <td>
            Kaustov Chakrabarti, Amanda M. Mohabir, Mahesh Nukala, Sara M. Safiullah, Tierney R. Woitas
          </td>
          <td>2025-11-21</td>
          <td>Scholarly Research In Progress</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Resistance to single-agent therapy in acute myeloid leukemia (AML) often arises from compensatory epigenetic circuits. We present a causal inference framework that treats therapy as an intervention (do-operator) and defines resistance as the deviation of dual perturbations from the additive expectation of monotherapies. By modeling the causal structure of the leukemia stem-cell epigenetic landscape that maintains identity and fitness, the framework shows that synergistic and antagonistic interactions can be inferred directly from single perturbations, suggesting interaction inference may be simpler than previously assumed. We applied this model to Perturb-seq of 16 epigenetic regulators in KMT2A-rearranged AML (>31,000 single-cell transcriptomes). From single-knockout profiles, the framework correctly predicted synergistic pairs (Menin+KAT6A, Menin+DOT1L) that enhanced differentiation and induced cell death, as well as an antagonistic pair (DOT1L + PCGF1) that conferred resistance. Predictions were validated by pharmacologic inhibition and bulk RNA-seq, confirming the model’s predictive accuracy. This work presents a mechanism-guided approach for rational prioritization of epigenetic drug combinations in AML and other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057014eca0b4705ea6ee698244a083b962878940" target='_blank'>
              Causal interaction inference of compensatory structures from single-cell perturb-seq in AML
              </a>
            </td>
          <td>
            Changde Cheng, Sajesan Aryal, Brittany Curtiss, Xinyue Zhou, Rui Lu
          </td>
          <td>2025-12-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26845d219b1c091d76174e4b4093ca359b51b378" target='_blank'>
              Benchmarking Chemical, Genetic, and Cell Line Encodings for Cancer Perturbation Response Prediction
              </a>
            </td>
          <td>
            Valentyna Zinchenko, Andreas Schlicker, Roman Kurilov, Alison Pouplin, Santiago Villalba, Marc Horlacher
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da24476f2b5b20f3dabd541d60d10f29d3bbfa00" target='_blank'>
              Rapid Autopsy Multi-Omic Analysis Identifies Divergent Evolutionary Trajectories and DNA Damage Resistance Mechanisms in FGFR2-Driven Cholangiocarcinoma
              </a>
            </td>
          <td>
            A. Sheel, A. Paruchuri, J. Reeser, M. Wing, R. Vella, E. Samorodnitsky, A. Smith, T. Dao, E. L. Hoskins, R. Bonneville, H. Z. Chen, C. Li, Z. Risch, P. Allenby, A. G. Freud, A. D. Fitzthum, W. Chen, S. Roychowdhury
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mutations drive evolution and genetic diversity, with the most consequential mutations occurring in coding exons and regulatory regions. However, the impact of transcription on germline mutagenesis remains poorly understood. Here, we identify a mutational hotspot at transcription start sites (TSSs) in the human germline, spanning several hundred base pairs in both directions. Notably, the hotspot is absent in de novo mutation data. We reconcile this by showing that TSS mutations are significantly enriched with early mosaic variants, many of which are excluded from de novo mutation calls, indicating that the hotspot partly arises during early embryogenesis. We associate the TSS mutational hotspot with divergent transcription, RNA polymerase II stalling, R-loops, and mitotic—but not meiotic—double-strand breaks, suggesting a recombination-independent mechanism distinct from known processes. Our findings are reinforced by mutational signature analysis, which highlights alternative double-strand break repair and transcription-associated mutagenesis. These insights reveal a germline mutational phenomenon with evolutionary and biomedical implications, particularly affecting genes linked to cancer and developmental phenotypes. The impact of transcription on germline mutagenesis remains poorly understood. Here, the authors identify a mutational hotspot at transcription start sites in the human germline that is significantly enriched with early mosaic variants; they also analyse the molecular mechanisms that could cause it and explore its potential consequences for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4de526e07a78fe794f2e553a66798f6d31a5db32" target='_blank'>
              Transcription start sites experience a high influx of heritable variants fueled by early development
              </a>
            </td>
          <td>
            Miguel Cortés Guzmán, David Castellano, Claudia Serrano Colome, Vladimir Seplyarskiy, Donate Weghorn
          </td>
          <td>2025-11-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Epitranscriptomic modifications represent a fundamental regulatory layer in cancer biology, with RNA methylation emerging as a pivotal mechanism governing transcriptomic dynamics. Among these, 5-methylcytosine (m5C) RNA methylation—a ubiquitous and conserved epitranscriptomic mark—has been identified across diverse RNA species, including mRNAs, rRNAs, tRNAs, and mitochondrial RNAs. Notably, the RNA m5C “writers”—enzymes responsible for installing this modification onto target RNAs—have emerged as central regulators of tumorigenesis, with NSUN (NOP2/Sun RNA methyltransferase) proteins playing a particularly pivotal role. We synthesize current knowledge of the cellular localization, substrate specificity, and biological functions of m5C-modifying enzymes, focusing predominantly on the NSUN family in the cancer context. We first dissect the spatiotemporal regulation patterns of NSUN proteins—from their nuclear roles in pre-mRNA processing to cytoplasmic functions in mRNA decay and translation—and their conserved methyltransferase domains that dictate target RNA recognition. This review further explores the molecular mechanisms by which NSUN proteins govern tumor progression, metastasis, and therapeutic responses, emphasizing their dual roles in both initiating oncogenic programs and maintaining cancer cell plasticity. Finally, we discuss the translational implications of targeting NSUN-mediated m5C pathways, highlighting small-molecule inhibitors designed against NSUN substrate specificity, combinatorial strategies with conventional chemotherapy or immunotherapy, and the promise of epitranscriptomic diagnostics and prognostic based on NSUN expression signatures. By positioning NSUN proteins as integral nodes in the RNA epigenomic network, this synthesis not only deepens our understanding of cancer pathogenesis but also identifies novel epitranscriptomic targets for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27c9e5c606d98501332df4288112a7acba40a71" target='_blank'>
              NSUNs-driven dysregulation: the next frontier in targeted cancer therapy?
              </a>
            </td>
          <td>
            Ying Zhao, Yuying Zhang, Haonan Zhou, Lei Zhang, Jinyu Guo, Yue Tan, Ting Wu, Yi Peng, Ying Che, Zhanlei Pei, Jun Li, Boshi Fu, Minjie Wei, Gang Wu, Xiaoyun Hu, Huizhe Wu
          </td>
          <td>2025-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Malignant tumors developing in the conditions of chronic hypoxia, unlike normal tissues, quickly acquire resistance to decreased oxygen level. Hypoxia-resistant cells in malignant disease obtain various features promoting their survival: they alter metabolism to aerobic glycolysis, activate angiogenesis-stimulating programs, and rearrange signaling cascades to adapt to hypoxia. At the same time, tumor cells become resistant to chemotherapy and radiation therapy and effectively colonize metastatic niches. 
This review analyzes signaling pathways which activate in the conditions of chronic hypoxia and underlie tumor adaptation to insufficient oxygen. Apart from the main pathway associated with activation of hypoxia-inducible factor 1α (HIF-1α), such cascades as STAT3, Snail cascade associated with epithelial-mesenchymal transition, NRF2 factor responsible for adaptation to reactive oxygen species, and stemness factors OCT4, SOX2 and NANOG are also considered. Effective killing of tumor cells requires simultaneous inhibition of several parts of signaling pathways because disruption of just one will lead to switching of signaling proteins to bypass the blocked one. The article lists various inhibitors of signaling pathways patricipating in tumor cell adaptation to chronic hypoxia and describes their mechanisms of action and targets. Development of successful strategies for treatment of hypoxia-resistant tumors requires identification of an effective combination of the above-mentioned and other inhibitors of the key signaling cascades participating in adaptation to hypoxia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21710e96836bbca03c72f6e2b505d714f5b359ca" target='_blank'>
              Rearrangement of signaling pathways and adaptation of tumor cells to hypoxia
              </a>
            </td>
          <td>
            O. E. Andreeva, E. I. Mikhaevich, M. Krasil'nikov
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Gastrointestinal (GI) cancers pose a significant global health burden, driven by complex molecular alterations and microenvironmental interactions. Advances in molecular pathogenesis have elucidated recurrent driver gene mutations, such as KRAS, TP53, and APC, alongside dysregulated signaling pathways including Wnt, RAS-MAPK, and PI3K-AKT, which collectively underpin tumor initiation and progression. Complementing genetic changes, epigenetic alterations—such as DNA hypermethylation, histone modifications, and regulatory non-coding RNAs—further contribute to malignant evolution by reshaping chromatin architecture and gene expression. These mechanisms not only promote uncontrolled proliferation but also reinforce therapeutic resistance by dynamically modifying the tumor microenvironment (TME). Molecular subtyping efforts, including The Cancer Genome Atlas (TCGA) classification for gastric cancer (GC) and the Consensus Molecular Subtypes (CMS) for colorectal cancer (CRC), have delineated disease heterogeneity, revealing distinct pathogenic pathways and enabling refined prognostic stratification. Such insights provide the biological rationale for diagnostic techniques and targeted interventions. For instance, anti-EGFR and anti-VEGF monoclonal antibodies disrupt oncogenic signaling and tumor angiogenesis, respectively, and have demonstrated substantial clinical efficacy in selected patient populations. In parallel, immunotherapy has emerged as a transformative modality in oncology. Immune checkpoint inhibitors targeting PD-1/PD-L1 and CTLA-4 reinvigorate antitumor immunity and have reshaped standard-of-care protocols for several GI malignancies. Beyond conventional immunotherapies, innovative strategies such as CAR-T cell therapy and neoantigen-based vaccines are being actively investigated. These approaches aim to overcome immune evasion mechanisms and enhance tumor-specific targeting, offering promise for patients with resistant or advanced disease. This review comprehensively analyzes the evolving molecular landscape of GI cancers and the corresponding development of targeted and immunotherapeutic agents. It highlights a balanced integration of mechanistic discovery and clinical translation, underscoring their synergistic roles in advancing precision oncology and improving survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5602f60be23d19b8d126a3c7ae3c86407dda38e" target='_blank'>
              Gastrointestinal cancer: molecular pathogenesis and targeted therapy
              </a>
            </td>
          <td>
            Yang Jin, Xiaobo He, Yanfeng Wu
          </td>
          <td>2025-12-01</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Gliomas, particularly glioblastomas (GBMs), exhibit pronounced chromosomal instability (CIN), contributing to aggressive behavior and therapy resistance. While next-generation sequencing (NGS) and SNP arrays reveal genomic alterations, they often miss rare subclones and complex structural variants. We employed Spectral Karyotyping (SKY) in patient-derived primary glioma cultures to directly visualize chromosomal abnormalities and developed two CIN metrics: Aneuploidy Score (AS) and Structural Abnormality Score (SS). Methods We analyzed 41 glioma samples (35 patients; 6 with paired primary/recurrent tumors) per 2021 WHO classification. Tumor cells were cultured and subjected to SKY to assess gains, losses, translocations, deletions, and rearrangements at single-cell resolution. Molecular markers (e. g., TP53, IDH, TERT promoter, MGMT methylation) were also evaluated. AS and SS were calculated and correlated with clinical and molecular features and survival. Results SKY revealed high intra- and intertumoral heterogeneity. GBMs showed frequent chromosome 7 gains, 10 losses, and Y chromosome loss in males. High SS correlated with shorter progression-free survival (PFS), higher WHO grade, recurrence, and prior treatment. TERT promoter wild-type and MGMT-methylated tumors had higher SS, suggesting reliance on alternative telomere maintenance or impaired repair. High AS was associated with TP53 mutation and independently predicted poor overall survival (OS) in newly diagnosed GBM. Conclusions AS and SS capture distinct aspects of glioma evolution. AS reflecting early aneuploid transformation and SS indicating therapy-induced structural alterations. Combined use of SKY-based cytogenetic profiling with molecular diagnostics may enhance risk stratification and guide treatment in gliomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4128906c3c6baadd56f3bed2cf74e59eb041415b" target='_blank'>
              10150-GEN-4 Exploring Chromosomal Instability in Patient-Derived Gliomas: Insights into Aneuploidy, Structural Abnormalities, and Tumor Evolution
              </a>
            </td>
          <td>
            Tetsuya Negoto
          </td>
          <td>2025-12-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Breast cancer is a leading cause of cancer-related mortality among women, with hormone receptor-positive (HR + ) disease representing the majority of cases. Despite advances in targeted therapies and antibody-drug conjugates, resistance to treatment remains a significant challenge, necessitating innovative therapeutic strategies. Epigenetic dysregulation, particularly involving histone acetylation, plays a crucial role in tumorigenesis and therapy resistance. Lysine acetyltransferase 6 (KAT6A/B), key regulators of chromatin structure and gene expression, have emerged as promising therapeutic targets in HR+ breast cancer. This review highlights the molecular mechanisms by which KAT6A/B modulate histone acetylation and gene regulation, emphasizing their roles in oncogenic pathways and cellular processes such as transcriptional activation, chromatin remodeling, and regulation of cellular senescence. We discuss preclinical evidence supporting the therapeutic potential of KAT6 inhibition and the challenges faced in epigenetic drug development, including the need for robust biomarker-driven approaches. Finally, we explore the prospects for integrating KAT6 inhibitors with existing therapies to overcome resistance and improve patient outcomes in breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/882962d718736529869b6bb59db5c43e6a4e7b49" target='_blank'>
              Exploring KAT6 as a therapeutic target in breast cancer: epigenetic approaches for precision medicine
              </a>
            </td>
          <td>
            Cyril Roussel-Simonin, Aranzazu Fernandez-Martinez, S. Postel-Vinay, F. Mosele, Alessandro Viansone, C. Massard, Adrien Mouren, Julien Vibert, B. Pistilli
          </td>
          <td>2025-12-01</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer.
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b30edb27f07cff97af34aacf8166230cf5b8614" target='_blank'>
              Unsupervised Identification of Cancer Attractor States through the Lens of Embryonic Origin and Cancer Hallmarks
              </a>
            </td>
          <td>
            P. Deshpande, Prabal Deb, Ibrahim Al Haddabi, B. Itkin
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human cells possess a complex network of regulatory systems——the DNA damage repair pathway (DDR), to deal with various DNA damage to prevent the inheritance of deleterious mutations. DDR perform its functions via cell cycle checkpoint activation. Mutations in the DDR gene are also the basis for tumor progression, metastasis, and therapeutic effect to treatments that cause DNA damage. Consequently, DDR has received high attention for its role in the therapy against cancer, and the related experiments have been carried out to further explore. Up to now, PARP inhibitors such as Olaparib, Rucaparib, Niraparib, Talazoparib, etc. have been approved for marketing and clinical use, other DDR inhibitors (DDRi) such as ATMi/ATRi/DNA-PKi, Chk1/2i, and WEE1i have partially entered clinical trials. Synthetic lethality has led to the exploration of combination of DDRi with other anti-tumor therapies, in particular, the combination of DDRi with immunotherapies such as chemoradiotherapy and immune checkpoint inhibitors (ICIs) has achieved potential efficacy. This article reviewed the DDR inhibitors that are currently in development or in clinical therapy, analyzed the limitations and future development prospects, and stated the mechanisms and clinical trials in antitumor and immunotherapy, to provide ideas for the subsequent development of DDRi and enhance the efficacy of antitumor immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9a6e3b57be514835a193a70ee6587199732e497" target='_blank'>
              Research progress of DNA damage repair (DDR) and DDR inhibitors in tumor immunotherapy
              </a>
            </td>
          <td>
            Qi Liu, Chunmei Zhang, Yixuan Gao, Dongmei Feng, Duo Deng, Yun Pan
          </td>
          <td>2025-12-01</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Osteosarcoma arises within heterogeneous tumor–immune ecosystems in which impaired antigen visibility—shaped by chromatin programs—limits immune surveillance and blunts responses to immunotherapy. Beyond structural defects in the antigen-processing pathway, Polycomb-mediated repression, DNA hypermethylation, and state-specific enhancer closure converge on the HLA class I/NLRC5/interferon axis to diminish peptide display. These constraints are context dependent, varying across malignant clones, differentiation states, and myeloid and T-cell niches. Traditional bulk assays obscure this complexity; single-cell ATAC-seq, integrated with single-cell and spatial transcriptomics, now resolves promoter–enhancer accessibility at HLA, NLRC5, and antigen-processing genes, distinguishes reversible repression from fixed lesions, and links microenvironmental stress to interferon competence. Translationally, epigenetic reprogramming—targeting Polycomb repressive complex 2 (PRC2), DNA methyltransferases (DNMTs), and complementary regulators (for example, LSD1, BET, CDK4/6, YAP/TEAD)—offers biomarker-guided avenues to restore antigen presentation, provided ecosystem-aware pharmacodynamic readouts track chromatin opening and antigen-presentation recovery across compartments. Despite encouraging preclinical evidence, efficacy will depend on clone selection, scheduling that preserves interferon signaling, and rational combinations with innate agonists and checkpoint blockade. This mini-review synthesizes epigenetic mechanisms of antigen-presentation failure in osteosarcoma and outlines how single-cell chromatin profiling can guide strategies to reinstate tumor antigen visibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f51ccd00bee57595f98289280452a3ec7548b15" target='_blank'>
              Epigenetic control of antigen presentation failure in osteosarcoma: from single-cell chromatin maps to therapeutic strategies
              </a>
            </td>
          <td>
            Yan He, Heng Wu
          </td>
          <td>2025-11-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a heterogeneous tumor, ranging from cases with spontaneous regression (MS stage) to high risk (HR) tumors. Resistance to therapy presents a major challenge in the treatment and management of HR-NB, contributing to a poor prognosis. In these patients, the resistance to conventional treatment is also related to non-genetic features, such as the cellular plasticity. Novel studies have demonstrated the presence of two distinct cell phenotypes: adrenergic (ADRN) and mesenchymal (MES), which reflect the heterogeneity of NB. MicroRNAs (miRNAs) are small, endogenous and non-coding RNAs with the ability to regulate gene expression and may have a crucial role in controlling cell plasticity. However, the role of miRNAs in NB plasticity has not been investigated yet. We investigated miRNA signature in NB cells subtypes (MES and ADRN) and in the extracellular vesicles (EVs) released by them, to identify potential MES related biomarkers. Differentially expressed miRNAs were identified by RT-qPCR and subjected to gene ontology, KEGG pathway, and protein-protein interaction network analyses. Candidate miRNAs were validated in plasma-derived EVs from NB patients. We identified miR-199a-3p as strongly upregulated in the MES cells subtype. Moreover, its expression levels were significantly higher in primary cell lines derived from HR patients compared to low-risk (LR) ones. This was confirmed by a bioinformatics analysis in patient tissue obtained from TARGET NB dataset. Protein-protein interaction analysis uncovered a complex network, with FN1, CD44, and YAP1, identified as key genes upregulated by miR-199a-3p, all of which are closely associated with the MES phenotype. Among the miRNAs significantly upregulated in EVs derived from MES cell lines, miR-584a-5p was significantly higher in EVs isolated from plasma of HR and L/Intermediate(I)R patients compared to MS. MiR-584-5p is typically considered a tumor suppressor; to support this role miR-584a-3p resulted significantly upregulated in L/IR tumor tissue, both in the INSS and COG classifications (TARGET NB database). Our findings identified specific miRNAs as MES phenotype related biomarkers. Further studies should investigate the potential impact of miRNAs on plasticity-related pathways in order to open new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b8406e9d606c161b955f6866c60bbec146aba97" target='_blank'>
              Identification of a miRNAs signature as potential biomarker of mesenchymal phenotype in neuroblastoma patients
              </a>
            </td>
          <td>
            Silvia Lampis, Alessandro Paolini, Virginia Di Paolo, A. Galardi, Salvatore Raieli, E. Miele, Lauriane Lemelle, F. Fabozzi, A. Serra, A. Mastronuzzi, M. A. De Ioris, Andrea Masotti, Franco Locatelli, A. Di Giannatale
          </td>
          <td>2025-11-26</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0127956ee657e81733c7622d2b48c0407a2cab1" target='_blank'>
              Cell heterogeneity contributes to the variable response of HIV-1 to latency reversing agents
              </a>
            </td>
          <td>
            Rachel Topno, Hussein Karaki, Flavia Mazzarda, Oriane Pourcelot, Manon Philippe, K. Zibara, Ovidiu Radulescu, Edouard Bertrand
          </td>
          <td>2025-11-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Introduction Glioblastoma (GBM) is a grade 4 brain tumor that originates in astrocytes. GBM has a high death rate and a very low median patient survival rate of 14–16 months, even with well-established treatment regimens. The majority of preclinical models assess the effectiveness of molecular leads on two-dimensional (2D) cell cultures, which may provide insight into toxicity against particular genotypes of GBM but do not provide insight into the mechanism of action of the therapeutic drug. Three-dimensional (3D) cultures present an attractive alternative due to their ability to closely model in vivo tumor-like conditions. Methods In the present study, we used a rotary cell culture technique to culture 3D cancer spheroids of the T98G cell line. Initially, we estimated the relative potency of histone deacetylase (HDAC) inhibitors, which are molecular leads currently in clinical trials as epigenetic therapy for GBM, on 2D and 3D cultures of T98G. We characterized the effect of the 3D half-maximal inhibitory concentrations (IC50) on spheroids using a live–dead assay to figure out which inhibitors inhibited cell viability in 3D the most. Finally, we checked the effects of the non-specific and specific inhibitors on tumor migration dynamics using an electric cell impedance sensing (ECIS) device with the help of two parameters—rate of migration (ROM) and late resistance (LR). Results and discussion Our results show that the specific HDAC-6 inhibitor Tubacin had a more potent anti-proliferative effect in both the cytotoxicity and live–dead assays. The non-specific inhibitor Vorinostat surprisingly promoted migration in the cells at its 2D IC50 value treatment, and none of the inhibitors was able to significantly decrease late resistance compared to untreated controls, indicating the need for the development of more potent HDAC inhibitors for monotherapy for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b65360a025f78e2e58d8871128571101edbb1c92" target='_blank'>
              Isoform-specific vs. pan-histone deacetylase inhibition as approaches for countering glioblastoma: an in vitro study
              </a>
            </td>
          <td>
            Ameya Joshi, Natasha Ratnapradipa, Jayce Hughes, Erik Moore, Andrew Ekpenyong, Surabhi Shukla
          </td>
          <td>2025-11-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="SMARCA4 deficiency plays a critical role in the oncogenesis of various aggressive tumors that exhibit resistance to conventional chemotherapy and radiotherapy, posing significant challenges to clinical management. The pivotal role of SMARCA4 deficiency in tumorigenesis suggests the need for a paradigm shift from traditional tumor origin-based approaches to novel tumor-agnostic strategies focused on molecular alterations associated with SMARCA4 deficiency. This review explores potential targetable molecular changes and emerging therapeutic strategies for SMARCA4-deficient tumors. Molecular alterations related to SMARCA4 deficiency involve impaired genomic stability, defects in DNA mismatch repair, and elevated tumor mutation burdens, all of which suggest potential sensitivity to immune checkpoint inhibitors (ICIs). Recent studies indicate that combining ICIs with chemotherapy or anti-angiogenic agents as first-line treatments may offer clinical benefits for SMARCA4-deficient tumors. Furthermore, SMARCA4 deficiency epigenetically affects chromatin accessibility, alters the distribution of Polycomb group proteins on chromatin, and modulates histone acetylation, highlighting the potential efficacy of epigenetic regulators such as EZH2 and HDAC inhibitors. In addition, synthetic lethality strategies targeting vulnerabilities in SMARCA4-deficient tumors are promising therapeutic approaches, including inhibitors of SMARCA2, CDK4/6, ATR, CHK1, PARP, and the oxidative phosphorylation pathway. Based on current clinical evidence, ICI-based combination therapies represent the most promising first-line regimens for SMARCA4-deficient tumors. Although a theoretical basis supports the potential of tumor-agnostic therapy as a promising strategy for these tumors, several challenges remain in clinical practice. These include heterogeneous therapeutic responses across tumor types, safety concerns associated with synthetic lethality-based agents, and the absence of any histology-agnostic approved therapy for SMARCA4-deficient tumors. The continued development of novel therapeutics and further large-scale clinical evaluations are essential to overcoming these barriers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ff8adc9ee03e886c2da7337f12fb44eb799835" target='_blank'>
              Tumor-agnostic therapy: a potential therapeutic approach for SMARCA4-deficient malignancies
              </a>
            </td>
          <td>
            Yang Liu, Zhi-Hui Liu, Qiong Zhang, Yin-Miao Bai, Xiao-Bo Guo, Hong-Chen Ji, Hong-Mei Zhang
          </td>
          <td>2025-12-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26a70cf29a8102e7c49d2b80d42e52230a68a64c" target='_blank'>
              Integrative multi-omics reveals PARP14 as a key IFNγ-regulated mediator of metastatic progression in Ewing sarcoma
              </a>
            </td>
          <td>
            S. Sánchez-Serra, M. Chicón-Bosch, Núria Martorell-Villanueva, Eduardo Candeal, Ignasi Jarne-Sanz, J. Bech-Serra, D. Giménez-Llorente, Manuel Gris-Lorente, P. Monaco, L. Santana-Viera, Silvia Mateo-Lozano, Jaume Mora, Mario Niepel, Carolina de la Torre Gomez, Rebeca Sanz-Pamplona, Ana Cuadrado, Ana Losada, R. López-Alemany, Ò. M. Tirado
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 Cancer stem cells (CSCs), first identified in acute myeloid leukemia with a CD34+/CD38- phenotype, are hypothesized to sustain clonal survival through self-renewal and resistance to chemotherapy. In B-cell acute lymphoblastic leukemia (B-ALL), stem-like properties can be reacquired through plastic transitions among stem-like, hematogone-like, and naïve B-cell–like states. This plasticity provides an evolutionary route by which chemotherapy-resistant subpopulations re-establish stem-like compartments, contributing to disease relapse under therapy. Such fluidity complicates efforts to target the stem-like compartment directly and underscores the need to understand how chemotherapy reshapes these transitions to drive resistance. We analyzed flow cytometry data from bone marrow (N=63) and peripheral blood (N=46) samples of B-ALL patients at diagnosis, post-induction, and in remission (N=38). Four major cell states are defined from stem-like, progenitor, and differentiated: 1) CD34+/CD38-, 2) CD34+/CD38+, 3) CD34-/CD38+, and 4) CD34-/CD38-. Patient-specific, continuous-time Markov chain modeling quantified transition rates, Qij between state i and j. In remission, differentiation (Q31) consistently exceeded dedifferentiation (Q13), reflecting normal lineage progression. In relapse, however, Q31 – Q13 was often negative with greater variance, indicating that resistant clones rebuilt stem-like compartments through dedifferentiation. This heterogeneity underscores distinct evolutionary mechanisms of resistance across patients. We next explore evolution-based interventions based on our model. Promoting differentiation (increasing Q31) transiently reduced stem-like fractions but failed to prevent rebound due to ongoing dedifferentiation. Inhibiting dedifferentiation (reducing Q13) produced sustained depletion of CD34+/CD38- cells and more effectively shifted relapse dynamics toward remission-like equilibria. Thus, targeting the dedifferentiation fraction emerges as a promising therapeutic strategy. To link transition dynamics with clinical outcomes, we applied mechanistic learning by integrating Markov-derived features with machine learning. Linear discriminant analysis (LDA) achieved high predictive accuracy for BCR::ABL1 status (80%), disease status (84% marrow, 89% blood), and minimal residual disease (71%). Principal component analysis highlighted the balance of differentiation and dedifferentiation as the most informative predictor. These results show that resistance evolves not only from persistence of stem-like cells but also from reciprocal transitions that replenish this compartment. Our findings suggest that therapy pressure influences cell transition landscapes in B-ALL, with relapse often associated with dedifferentiation-driven restoration of stem-like states. Mechanistic learning highlights stem-state reciprocity (in/out rate balance) as a potential predictive biomarker, while modeling indicates that inhibiting dedifferentiation may be effective to reduce resistant stem-cell like clones.



 Sadegh Marzban, Curtis Gravenmier, Ling Zhang, Jeffrey West. Cell state transitions drive the evolution of disease progression in B-lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr B035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37463b573a65ae5e34a361dd2f42a6408948633c" target='_blank'>
              Abstract B035: Cell state transitions drive the evolution of disease progression in B-lymphoblastic leukemia
              </a>
            </td>
          <td>
            S. Marzban, Curtis Gravenmier, Ling Zhang, J. West
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetic interactions can reveal gene function and identify cancer-relevant synthetic lethals, but systematic mapping in human cells is constrained by inefficient reagents, vast combinatorial search space, and diversity of cell types. Here, we leverage principles from yeast genetic networks to identify human gene modules enriched for genetic interactions. Using our Cas12a-based In4mer combinatorial knockout platform, we screen all pairwise interactions within receptor tyrosine kinase and DNA damage response modules across eight diverse cancer cell lines. We identify hundreds of unreported synthetic lethals, including a dense network within the protein glycosylation machinery, and confirm that interactions in 2D cell culture are maintained in more physiologically relevant models. Our targeted modules show up to 16-fold enrichment of interaction density, providing a scalable strategy for systematic interaction mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a137f8aa4f89541a558bf1cd83f7f679eee7b" target='_blank'>
              Functional modules predict cancer-relevant genetic interactions in mammalian cells
              </a>
            </td>
          <td>
            Chenchu Lin, Veronica Gheorghe, Juihsuan Chou, Sabriyeh Alibai, Subin Kim, N. Anvar, Yixin Xu, Xingdi Ma, Lori L. Wilson, Russell Moser, Christopher J. Kemp, Junjie Chen, Scott Kopetz, Traver Hart
          </td>
          <td>2025-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Iron enables tumor cells to maintain pro-tumoral functions including DNA synthesis and repair, drug resistance and metabolic processes such as oxidative phosphorylation and regulation of reactive oxygen species. To meet these demands, tumor cells rewire iron metabolism to increase iron uptake and use. Therefore, disrupting iron metabolism either by limiting availability or by exploiting iron accumulation to induce ferroptosis, might be a promising strategy for cancer therapy. Recent studies suggest that other cell populations in the tumor microenvironment, including immune cells and cancer-associated fibroblasts, depend on iron and can contribute to iron dysregulation in tumors. Here, we will discuss how iron-dependent pathways contribute to tumor development, with a focus on iron sulfur cluster proteins and heme and their effects on metabolism. In addition, we will describe the relevance of iron crosstalk within the tumor microenvironment in promoting tumor growth, metabolic reprogramming and immune evasion. Finally, we will explore the therapeutic potential of targeting iron-dependent processes beyond the scope of ferroptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/534987841cc5b057714d290f608f6acdcd5ef2ac" target='_blank'>
              Iron and metabolic rewiring in cancer
              </a>
            </td>
          <td>
            Marina Ciscar, César Rodríguez-Santana, Naiara Santana-Codina
          </td>
          <td>2026-01-09</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Malignant transformation of mature cystic teratoma (MTMCT) of the ovary is a rare but aggressive malignancy for which no standardized chemotherapy or effective targeted therapies currently exist. To identify therapeutic vulnerabilities in MTMCT, we performed a genome-wide CRISPR-Cas9 knockout screen using the MTMCT-derived NOSCC1 cell line. Two parallel selective pressures were applied: in vivo tumorigenicity in immunodeficient mice and cisplatin exposure in vitro. From this screen, 67 negatively selected genes were identified, among which SOD1 and NDUFB4 emerged as top candidates based on high basal expression levels and clinical relevance. Integration with spatial transcriptomic data from three independent MTMCT patient tumors further supported the prioritization of these targets. SOD1 was selected for further investigation due to the availability of known pharmacological inhibitors. Both siRNA-mediated knockdown and small-molecule inhibition of SOD1 using LCS-1 significantly suppressed MTMCT cell proliferation in vitro by inducing oxidative stress and impairing cell cycle progression. This antiproliferative effect was reversed by co-treatment with N-acetylcysteine, a reactive oxygen species scavenger. In vivo validation using patient-derived xenograft models demonstrated that oral administration of LCS-1 led to significant tumor growth suppression and increased expression of apoptotic and DNA damage markers, including cleaved caspase-3 and γH2AX. These findings establish SOD1 as a critical vulnerability in MTMCT and provide preclinical evidence supporting redox modulation as a therapeutic strategy for this highly chemoresistant and understudied ovarian cancer subtype. Our integrative approach combining functional genomics, spatial transcriptomics, and pharmacologic validation offers a framework for the discovery of novel targets in rare gynecologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e67036000a59a86b2e3dd28f5c317d050f6d45c" target='_blank'>
              In-Tumor CRISPR-Cas9 Knockout Screening and Novel Therapy Development for Malignant Transformation of Ovarian Teratoma.
              </a>
            </td>
          <td>
            S. Tamauchi, Kosuke Yoshida, Xinyuan Wang, Atsushi Nakagawa, Akira Yokoi, N. Yoshikawa, K. Niimi, Yusuke Yamamoto, Hiroaki Kajiyama
          </td>
          <td>2026-01-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, M. A. Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="(1) Background: The transcription factor p73 exists in multiple isoforms with divergent functions in cancer. While DNp73 promotes stemness, epithelial–mesenchymal transition (EMT), and metastasis, the tumor-suppressive isoform TAp73 can also switch to promoting cancer progression. How isoforms sharing the same DNA-binding domain produce divergent outcomes remains unclear. (2) Methods: Here, we performed CUT&RUN in combination with JASPAR, transcriptomics, proteomics, patient survival and gene expression data to map genome-wide and promoter-associated DNA-binding and coregulatory transcription factor (coTF) profiles of TAp73α and DNp73β in melanoma cells. (3) Results: Systematic screening for motif enrichment in cancer hallmark gene sets revealed TAp73- and DNp73-specific coTF repertoires with distinct functions. We identified a coregulator signature for EMT genes enriched for both isoforms that has tumor context-dependent effects on survival and correlates with unfavorable patient prognosis. Of these EMT-associated coTFs, PATZ1 was validated as a novel direct interactor of DNp73β. (4) Conclusions: Our results provide a comprehensive reference map of p73 isoform-specific binding and coregulator recruitment and establish a workflow to model their influence on cancer reprogramming with implications for AI-based individualized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d7357286e4d775a6aeae941e92117c88a7c586" target='_blank'>
              Integrative Multimodal Profiling of TAp73 and DNp73 Reveals Isoform-Specific Transcriptomic Coregulator Landscapes in Cancer Programs
              </a>
            </td>
          <td>
            Steffen Möller, A. Spitschak, Nico Murr, Brigitte M Pützer
          </td>
          <td>2025-12-31</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Nearly half of the human genome consists of transposable elements, among which endogenous retroviruses, remnants of ancient retroviral infections, represent some of the most evolutionarily intriguing due to their paradoxical functional duality. While research has documented functional ERV exaptation in key biological processes, these elements have also been associated with age-related diseases, particularly cancer. This apparent contradiction presents an evolutionary question: why would potentially disruptive elements persist in genomes over evolutionary time? Here we review the complex relationship between ERVs, aging and cancer to address this question. After reviewing the physiological roles of ERVs, we explore how the transcriptional activation of normally repressed ERVs may function as an evolutionary-conserved genomic surveillance system that, when triggered by cellular stressors, generates viral-like nucleic acids and proteins that activate pathways to potentially eliminate cancerous cells. Conversely, we discuss how cancer cells could appropriate ERV expression to distort cellular processes, promoting inflammation and senescence that ultimately facilitate tumor progression. Despite this duality, we advance a novel hypothesis that many ERVs have been exapted in mammalian genomes primarily as defense mechanisms against tumorigenesis. This evolutionary perspective provides a framework for understanding both the persistence of ERVs in our and other mammals’ genomes and their intriguing roles in cancer biology. Moreover, even after tumor development, ERVs can be exploited by immunotherapy due to their canonical function as regulators of the immune response, positioning them as emerging central elements in cancer treatment strategies. This work offers new insights into these endogenous retroviruses’ evolutionary significance and potential applications in cancer therapeutics and diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd2557a8bd78199bb0c2bedd92cbc9a68a2fb0" target='_blank'>
              Endogenous retroviruses in aging and cancer: from genomic defense to oncogenic activation
              </a>
            </td>
          <td>
            Gabriel Arantes dos Santos, Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Rafael Loch Batista, Pedro A. F. Galante
          </td>
          <td>2025-12-01</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung cancer, recognized as one of the most prevalent malignancies with the highest rates of incidence and mortality globally, presents a substantial challenge on a worldwide scale. This challenge is exacerbated by the disease’s difficulty in early detection, a pronounced rate of metastasis, and resistance to treatment, all of which contribute to elevated mortality rates. The tumor microenvironment (TME) plays a critical role in the sustenance and advancement of various solid tumors, including lung cancer. The intricate composition of the TME facilitates tumor proliferation, metastatic spread, and therapeutic resistance by supplying metabolic resources, fostering angiogenesis, and enabling immune evasion. Nonetheless, the regulatory frameworks operating within the TME remain poorly understood. An increasing body of evidence suggests that epigenetic regulation—encompassing mechanisms such as DNA methylation, histone modification, and the action of non-coding RNAs—is pivotal in the initiation and progression of lung cancer. Furthermore, epigenetic modifications significantly influence the functional dynamics of the tumor microenvironment, thereby impacting intercellular interactions and cellular behaviors within the TME, which in turn affects the trajectory of disease progression. This article aims to present the most recent advancements in research concerning the epigenetic regulation of tumor cell interactions with the TME in the context of lung cancer biology. Additionally, it examines the current implications of epigenetic regulation within the tumor microenvironment and its influence on lung cancer behavior. We also investigate the potential relevance and emerging therapeutic avenues presented by epigenetic regulation in the clinical diagnosis and treatment of lung cancer, aspiring to propose novel strategies to address existing treatment challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f9bbf3d9c9172ba516c36021afce36965e86d5" target='_blank'>
              Epigenetic regulation of the tumor microenvironment in lung cancer: mechanism insights and therapeutic prospects
              </a>
            </td>
          <td>
            Maoqin Yang, Xudong Lei, Dexiang Ren, Dakai Qin, Xiaojun Xia
          </td>
          <td>2025-11-28</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aef2d09241345ec730cec2612f4e7bed8b0baea" target='_blank'>
              DNA2 variant analysis supports the nuclease activity as a preferred therapeutic target
              </a>
            </td>
          <td>
            Katherine E. Baillie, Sijie Zhang, Veena Mathew, Troy Nations, Balakrishna Koneru, Peter C. Stirling
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tumorigenesis is closely related to defects in DNA damage response (DDR), which plays an important role in maintaining genomic integrity. The DDR process includes DNA damage recognition, cell cycle checkpoint pathway activation, and DNA damage repair. On the one hand, defects in DDR increase the instability of the cellular genome, which ultimately leads to tumorigenesis. On the other hand, tumor therapy can take advantage of DDR defects to induce apoptosis by causing damage to tumor cells, thus achieving therapeutic goals. Tumor immunotherapy is an emerging method of tumor treatment that attacks and kills cancer cells by inhibiting negative regulatory factors, enhancing the ability of immune cells to recognize antigens on the surface of tumor cells and activating the patient's own immune system. Recent research evidence suggests that DDR inhibition enhances antitumor immune responses and acts in synergy with immunotherapy. This review describes the different categories of DNA damage, their corresponding DNA damage repair pathways, the antitumor mechanisms of action targeting DDR inhibition, and the prominent contributions of these mechanisms to the immunotherapeutic response. By exploring recent advances in the interactions between targeted DDR and immunotherapy, this review provides novel therapeutic insights into the treatment of DNA damage-associated cancers as well as the development of combination immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ffe870d92fa7406cd9d4f5f65954a04fd0351a" target='_blank'>
              Implications of DNA damage response and immunotherapy in tumor therapy
              </a>
            </td>
          <td>
            Wenqin Li, Binghang Li, Chang Cai, Jie Chen, Yiqiu Li, Rongrui Liang, Jing Zhao
          </td>
          <td>2025-12-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The mechanisms underlying cancer cell drug resistance have not yet been fully elucidated. Numerous processes hinder effective tumor treatment, such as cancer cell epigenetic alterations, DNA repair, drug target modification, and epithelial–mesenchymal transition. Recent studies have highlighted the ability of cancer cells to transiently enter the resting phase, namely, the G0 phase of the cell cycle under treatment conditions; this is associated with an elevated level of cell death. Cells in the G0 phase are characterized by a specific phenotype, the exact features of which remain to be fully elucidated. Cell division and cell cycle dynamics are closely associated with cell adhesion, i.e., interaction with the extracellular matrix in normal cells, whereas cancer cells are characterized by the dys-coordination of these processes. However, the precise patterns of cell adhesion alterations in association with the cell cycle remain unclear. Knowledge on the interactions of cells in the G0 phase with the extracellular matrix during the entry, the exit, and the sustenance during the resting phase is limited. At the same time, focal adhesion-mediated molecules may be an attractive goal for cancer cells in the G0 phase, targeting to overcome tumor dissemination and drug resistance. Drug resistance and immune system response evasion are specific features of nonproliferating cancer cells that call for more detailed and accurate characterization of this population of malignant cells to optimize anticancer treatment and control tumor dissemination. The present review represents the recent data on the association of focal adhesion and cancer cell surveillance due to cell cycle alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16149b6070228042071985c6c4e07c2cd03bc235" target='_blank'>
              Focal Adhesion and Cancer Cell Cycle-Associated Resistance
              </a>
            </td>
          <td>
            T. Ruksha, N. Palkina, Ekaterina Z. Lapkina, Kangdong Liu
          </td>
          <td>2025-12-25</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Oxidative DNA damage is a major driver of genome instability and human disease. Among the various types of oxidative DNA base lesions, 8-oxo-7,8-dihydroguanine (8oxoG) is particularly prevalent due to guanine’s low oxidation potential and the abundance of guanine-rich (G-rich) sequences across the genome. Structure-forming repeat sequences, which are commonly G-rich, can adopt alternative DNA secondary structures that further expose nucleobases to oxidative damage. The base excision repair (BER) pathway is primarily responsible for the repair of 8oxoG lesions; however, the complex topologies and dynamic conformations formed by these repeat sequences present challenges for complete repair. Inefficient BER within these structures can lead to DNA strand breaks, mutations, and large chromosomal rearrangements, all of which are associated with human disease. Notably, structure-forming repeat sequences are often enriched at regulatory genomic regions, where BER can directly influence processes such as replication and transcription. This review summarizes current insights into BER activity within oxidatively damaged structure-forming repeat sequences and highlights how repair efficiency within these sequences impacts genome stability and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a9b8ec92f7ce4226378783f87443780b182e66" target='_blank'>
              Base excision repair within structure-forming repeat sequences and its impact on cancer and other diseases
              </a>
            </td>
          <td>
            Carson B Cohen, Millie C Coombes, Christopher P Merlo, Chantal A Kontor, Riaz Meah, Amy M Whitaker
          </td>
          <td>2025-12-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c517ba39588758cc3e5fedf674d78132e58c9969" target='_blank'>
              Identification of full-length circular RNAs linked with therapy resistance of pediatric cancers
              </a>
            </td>
          <td>
            C. Bessière, L. Babin, E. Andraos, J. M. Riepl, A. Szymansky, M. Lodrini, H. Deubzer, A. Eggert, C. Quivoron, C. Rigaud, V. Verge, S. Pyronnet, L. Lamant, F. Meggetto, C. Gaspin, S. Fuchs
          </td>
          <td>2025-12-04</td>
          <td>None</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d2c494a1cd657524f51f9577b21d3baa396ea7" target='_blank'>
              Spatiotemporal regulation of cell cycle states within the complex tumor microenvironment
              </a>
            </td>
          <td>
            Gianni Zanardelli, Olivier Tassy, Maulik K. Nariya, Nacho Molina
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea0c9bba6958723f9dc44615a55714dd8d76cc78" target='_blank'>
              Understanding KRAS: mechanisms of resistance and future therapeutic strategies
              </a>
            </td>
          <td>
            Rana Mahmoud Khashaba, Nada Ali Refaat, Sama Elsayed Elmorsy, Fatma Ibrahim Ali, Raghda W. Magar
          </td>
          <td>2025-11-27</td>
          <td>Discover Applied Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f8fae2bcd154582c2189d775e6e129baa231cb3" target='_blank'>
              A multi-omics analysis of human fibroblasts overexpressing an Alu transposon reveals widespread disruptions in aging-associated pathways.
              </a>
            </td>
          <td>
            Juan I. Bravo, Eyael Tewelde, Christina D. King, J. Bons, Samah Shah, Jacob Rose, Judith Campisi, Birgit Schilling, B. Benayoun
          </td>
          <td>2025-12-11</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="RNA modification, a prominent epigenetic mechanism, has been implicated in regulating RNA function, stability, processing, and interactions, including pseudouridylation, acetylation, and methylation. Recent evidence highlights that 5-methylcytosine (m5C) influences key cellular processes such as proliferation, differentiation, apoptosis, and stress responses by modulating RNA stability, translation, transcription, nuclear export, and cleavage. This review consolidates current insights into the role and mechanisms of m5C methylation across various tumor types, underscoring its pivotal involvement in post-transcriptional regulation and its profound effects on gene expression, cellular dynamics, and tumor biology. The mechanisms through which m5C methylation impacts tumor progression, including modulation of glucose and iron metabolism, as well as resistance to therapeutic agents, are also discussed. Finally, the review identifies critical future research avenues, focusing on elucidating the underlying mechanisms, developing targeted therapies, and advancing personalized medicine approaches to leverage m5C methylation in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/201f634747623dbdceebac9d2c8e4b2347b7edde" target='_blank'>
              RNA m5C methylation in cancer: mechanisms and biological impact
              </a>
            </td>
          <td>
            Zhenyu Guan, Wendong Li, Yuting He, Wenzhi Guo
          </td>
          <td>2025-11-21</td>
          <td>Oncogenesis</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, Ellen Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31260d36f9a7cbef928f0364f7e2575dba44afd5" target='_blank'>
              High-confidence structural predictions of extrachromosomal DNA with ecDNAInspector
              </a>
            </td>
          <td>
            Sophia J. Pribus, Yanding Zhao, Zhicheng Ma, Clemens Weiss, Aziz Khan, Kathleen E. Houlahan, Christina Curtis
          </td>
          <td>2025-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park, Md Rashedunnabi Akanda
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acbdbd54a52a897e14acda45d5d2ae5113465461" target='_blank'>
              Genomic characterization of colorectal tumors: insights into significantly mutated genes, pathways, and survival outcomes.
              </a>
            </td>
          <td>
            T. Harrison, Syed H Zaidi, Hang Yin, R. Steinfelder, C. Qu, E. Aglago, Sonja I. Berndt, Lisa A. Boardman, Hermann Brenner, Daniel D. Buchanan, P. Campbell, Yin Cao, A. T. Chan, S. Chanock, K. Doheny, David A. Drew, J. C. Figueiredo, Amy J. French, S. Gallinger, P. Georgeson, M. Giannakis, Ellen L. Goode, S. Gruber, A. Gsur, Marc J. Gunter, Sophia Harlid, M. Hoffmeister, Wen-Yi Huang, Meredith A J Hullar, J. R. Huyghe, Mark A Jenkins, Yi Lin, Victor Moreno, Neil Murphy, Polly A Newcomb, Christina C. Newton, J. Nowak, M. Obón-Santacana, Shuji Ogino, T. Shelford, Mingyang Song, Claire E. Thomas, A. Toland, T. Ugai, Caroline Y Um, B. van Guelpen, Quang M Trinh, Wei Sun, T. Hudson, L. Hsu, Ulrike Peters, Amanda I. Phipps
          </td>
          <td>2025-12-18</td>
          <td>BMC cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Lineage plasticity is a cancer hallmark that drives disease progression and treatment resistance1,2. Plasticity is often mediated by epigenetic mechanisms that may be reversible; however, there are few examples of such reversibility. In castration-resistant prostate cancer (CRPC), plasticity mediates resistance to androgen receptor (AR) inhibitors and progression from adenocarcinoma to aggressive subtypes, including neuroendocrine prostate cancer (CRPC-NE)3, 4–5. Here we show that plasticity-associated treatment resistance in CRPC can be reversed through the inhibition of NSD2, a histone methyltransferase6. NSD2 upregulation in CRPC-NE correlates with poor survival outcomes, and NSD2-mediated H3K36 dimethylation regulates enhancers of genes associated with neuroendocrine differentiation. In prostate tumour organoids established from genetically engineered mice7 that recapitulate the transdifferentiation to neuroendocrine states, and in human CRPC-NE organoids, CRISPR-mediated targeting of NSD2 reverts CRPC-NE to adenocarcinoma phenotypes. Moreover, a canonical AR program is upregulated and responses to the AR inhibitor enzalutamide are restored. Pharmacological inhibition of NSD2 with a first-in-class small molecule reverses plasticity and synergizes with enzalutamide to suppress growth and promote cell death in human patient-derived organoids of multiple CRPC subtypes in culture and in xenografts. Co-targeting of NSD2 and AR may represent a new therapeutic strategy for lethal forms of CRPC that are currently recalcitrant to treatment. Inhibition of the histone methyltransferase NSD2 and the androgen receptor in preclinical models can reverse lineage plasticity to suppress tumour growth and promote cell death in multiple subtypes of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31a72e85b6954f413f71412e13d4f1a6449106e" target='_blank'>
              NSD2 targeting reverses plasticity and drug resistance in prostate cancer
              </a>
            </td>
          <td>
            Jia J Li, A. Vasciaveo, Dimitris Karagiannis, Zhen Sun, Kristjan H. Gretarsson, Xiao Chen, O. Ouerfelli, F. Socciarelli, Ziv Frankenstein, Hanyang Dong, Min Zou, W. Yuan, Guangli Yang, Gabriel M Aizenman, Tania Pannellini, Xinjing Xu, H. Beltran, Yu Chen, Kevin L Gardner, Brian D. Robinson, J. D. De Bono, O. Gozani, Cory Abate-Shen, Mark A. Rubin, M. Loda, Charles L. Sawyers, Andrea Califano, Chao Lu, Michael M. Shen
          </td>
          <td>2025-11-26</td>
          <td>Nature</td>
          <td>2</td>
          <td>68</td>
        </tr>

        <tr id="Large parallel genetic screens have been used to identified targets and regulators that enhance T cell antitumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducted a pathway enrichment analysis. A T cell screen perturbation score (TPS) metric was employed to quantify the impact of a gene perturbation on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease cohort and T cell-derived cancer patients were incorporated to gain further insight on gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 35 T cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is accessible through our webserver (http://tcpgdb.sidichenlab.org/) and allows users to interactively explore the impact of query genes on T cell function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39e4489bcd5040b80166991d7202bfd2ac7d128" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators.
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2025-11-21</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Chromosome instability (CIN) remains among the most important problems in modern cancer research. In this study, we conducted a genome-wide RNAi screen to identify genes that contribute to CIN. To achieve this, we used a human artificial chromosome in a novel sensitized screen to measure CIN. We screened 18,658 genes for their roles in maintaining chromosomal stability and identified 834 candidates as potential CIN genes. A secondary RNAi screen identified 44 genes with the most pronounced CIN phenotypes. In guilt-by-association analysis using a published set of 8,498 proteins across a panel of 949 cancer cell lines, this cohort of 44 genes displayed a striking correlation with mitotic regulators. Furthermore, altered expression of these proteins was associated with a poor prognosis across multiple cancer types. Specifically, downregulation of AMY2B, ALAD, PDGFRA, PPIE, VEZ1, and TTC19 is associated with poor survival in two or more of the following: small cell lung cancer, lung adenocarcinoma, adrenocortical carcinoma, ovarian cancer, and breast cancer. The genes identified in this screen hold potential as prognostic markers for patient survival across several cancer types and could potentially serve as targets for the development of new therapeutic approaches aimed at mitigating CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48402b9da158c79dbc9aec979b9f8a854d30f0b3" target='_blank'>
              A genome-wide RNAi screen for novel CIN genes using human artificial chromosome.
              </a>
            </td>
          <td>
            Mikhail Liskovykh, N. Y. Kochanova, Chih-Yuan Chiang, A. Dhall, V. Aksenova, Yu-Chi Chen, William C Reinhold, M. Dasso, Anish Thomas, K. Cheng, Y. Pommier, William C. Earnshaw, Vladimir Larionov, N. Kouprina
          </td>
          <td>2025-12-01</td>
          <td>PNAS nexus</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d9d9e99d312aee7d3e8959371325610d5ce9af0" target='_blank'>
              MIF-mediated reprogramming of myeloid lineage within the glioma tumor microenvironment impacts the efficacy of immune stimulatory gene therapy
              </a>
            </td>
          <td>
            Ziwen Zhu, Noah C. Kanis, Anthony E. George, Michael Albdewi, Yingxiang Li, Anzar A. Mujeeb, Brandon L. McClellan, Gurveer Singh, Jialin Liu, W. Al-Holou, J. Heth, Joshua D. Welch, Justin D. Lathia, Pedro Lowenstein, Richard Bucala, Maria G. Castro
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ionizing radiation (IR) is used to treat more than half of cancer patients because it induces DNA double-strand breaks and triggers apoptosis. IR also damages other nucleic acids, lipids, proteins and cellular organelles, initiating additional complex cellular responses. Some of these responses are transient, while others can become permanent and lead to changes in cellular identity. This review focuses on cell fate plasticity, defined as the conversion of one cell type into another, during recovery after IR-induced damage. We recognize that this process likely occurs along a continuum and may be reversible. We will distinguish cell fate plasticity from molecular or phenotypic plasticity, such as epigenetic modifications, transcriptomic shifts, altered signalling, morphological changes or acquisition of migratory behaviour, all of which are clinically relevant but do not constitute a change in cell type for the purposes of this review. Importantly, cell fate plasticity can enable cancer cells to acquire stem-like properties, which has major implications for tumour progression and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcae9ab22121364b1772334438f8e401bf8921d7" target='_blank'>
              Radiation-induced cell fate plasticity.
              </a>
            </td>
          <td>
            Michael Shiferaw, T. T. Su
          </td>
          <td>2026-01-14</td>
          <td>Open biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aecbc95950c12cbf91340acad8d28e801a22a818" target='_blank'>
              tRNA-derived fragments (tRFs) as key non-coding players in the tumor transcriptome of colorectal cancer: introducing a new suspect responsible for the remaining unknowns of tumor pathology
              </a>
            </td>
          <td>
            Hamid Aria, Behnam Mansoori, Zahra Saadatian, Amin Moradi Hasan-Abad, Roumina Norouzi, Shayan Heidari, Masume Dehghani, Y. Mansoori, A. Daraei
          </td>
          <td>2025-12-01</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b02f1b354e2c642c1280c585ed8336d631e18" target='_blank'>
              ZMYM3 S464: a potential phospho-regulatory hub in epigenetic remodeling and oncogenesis
              </a>
            </td>
          <td>
            Apoorva Pai, Althaf Mahin, Samseera Ummar, Athira Perunelly Gopalakrishnan, Prathik Basthikoppa Shivamurthy, A. Rajeev, Rajesh Raju
          </td>
          <td>2026-01-06</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Neuroblastoma (NB) is the most common extracranial solid tumor in children and accounts for 15% of childhood cancer death. The nucleosome remodeling and deacetylase (NuRD) complex is a major chromatin remodeling complex that regulates chromatin accessibility and gene transcription. However, its role in the pathogenesis of neuroblastoma remains poorly understood.


METHODS
The genetic dependency and clinical significance of MBD3 in neuroblastoma was evaluated by analysis of public datasets. The function of MBD3 in neuroblastoma cell growth was evaluated by shRNA knockdown experiment. Cleavage under targets and tagmentation sequencing (CUT&Tag-seq), coupled with RNA-sequencing, was employed to explore the mechanisms involved in the epigenetic regulation executed by NuRD decommissioning following MBD3 deficiency.


RESULTS
Here we find that MBD3 is the most lineage-selective dependency among the non-enzymatic subunits of the NuRD complex in neuroblastoma. Knockdown of MBD3 induces cell cycle arrest and apoptosis, and inhibits neuroblastoma growth in vivo. Mechanistically, MBD3 deficiency leads to decommissioning of the NuRD complex and dissociation of the EZH2-PRC2 complex from chromatin, thereby orchestrating the epigenetic regulation of gene expression by modulating the balance between histone acetylation and methylation. NuRD decommissioning upon MBD3 deficiency selectively downregulates the expression of core regulatory transcription factors and upregulates a tumor suppressor SRCIN1, collectively suppressing neuroblastoma progression.


CONCLUSIONS
Our data identify MBD3 and the NuRD complex as potential therapeutic targets in neuroblastoma, highlighting the critical role of epigenetic regulation in tumor maintenance. Targeting this pathway may offer a novel strategy to selectively impair neuroblastoma cell survival and improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9509051ef6556e77505cd01c5bcf3d3b1c22a6b6" target='_blank'>
              MBD3 deficiency decommissions the NuRD complex and orchestrates the epigenetic regulation of gene expression to suppress neuroblastoma progression.
              </a>
            </td>
          <td>
            Chen Chen, Bize Guo, Siyan Wu, Qinfang Zhu, Hui Shi, Jinhu Wang, Qiang Shu, Ting Tao
          </td>
          <td>2026-01-07</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b89d2858599b4236f22769f3a4af65ec9fd4eb7" target='_blank'>
              Deciphering a common code: Unitary gene profile factor enabling Spontaneous Tumor Regression
              </a>
            </td>
          <td>
            Arushi Misra, B. Kumari, Prasun K. Roy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract V(D)J recombination, essential for adaptive immunity, is initiated by RAG1/2, which recognizes recombination signal sequences (RSSs) flanking gene segments at antigen receptor loci. RSSs comprise conserved heptamer and nonamer motifs separated by a 12/23-base spacer. While the first three heptamer nucleotides (5′-CAC) are strictly conserved, RSS promiscuity enables RAG “off-target” activity at RSS-like sequences, termed “cryptic RSSs” (cRSSs), contributing to chromosomal deletions and lymphoid tumorigenesis. Notably, the nonamer exhibits substantial sequence variability across physiological RSSs. In addition, many cRSSs lack a discernible nonamer-like sequence, limiting the ability to predict RAG targets based on RSS consensus. Using a high-throughput approach, we characterized here the nonamer properties supporting RAG-mediated recombination. While the consensus nonamer (5′-ACAAAAACC) exhibited strong functionality, numerous sequences significantly different from it promoted a high recombination rate. These functional nonamers balance two, somewhat opposing, features: affinity for the RAG nonamer-binding domain—primarily via a shared purine–A/T–purine motif at positions 4–6—and nucleosome repulsion. Moreover, nonamers of genomic cRSSs mimic canonical nonamers mainly through nucleosome-repelling sequences. This study provides a model for both physiological RAG activity and its off-target effects, enhancing our understanding of immune repertoire formation and the genetic basis of lymphoid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3be51b4ff1033c4b22dd2a4e43eac3aedc8d967" target='_blank'>
              The nonamer code for RAG-mediated recombination in vivo
              </a>
            </td>
          <td>
            Wessal Hanout-Chuiko, Sunil Pachakar, Rawan Foqara, Hadas Sibony-Benyamini, Shira Peretz, Nurit Edri, Moshe Dessau, Yaakov Maman
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9014a6b11423460ea6b5d8b30bbc873c704d2d2a" target='_blank'>
              A key regulator of missing-self innate immunity is polymorphic and under diversifying selection
              </a>
            </td>
          <td>
            Rocco F. Notarnicola, M. Herdegen-Radwan, Joanna Różańska-Wróbel, M. Konczal, Karolina Przesmycka, P. Kotlík, W. Babik, Jacek Radwan
          </td>
          <td>2025-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d20c38801bb8ed91cd62e4262382fdf5cac88f32" target='_blank'>
              Single-cell multiplex approaches deeply map ON-target CRISPR-genotoxicity and reveal its mitigation by palbociclib and long-term engraftment.
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Bergès, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, Christelle Debeissat, Philippe Brunet de la Grange, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2026-01-10</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has a high incidence and mortality rate globally, with approximately 40% of patients harboring Kirsten rat sarcoma viral oncogene homolog (KRAS) mutations. Patients with KRAS mutations exhibit poorer prognoses compared to those with KRAS wild-type tumors, often showing resistance to targeted therapies and chemotherapy. There is an acute need for innovative therapeutic strategies, particularly as KRASG12C inhibitors have been approved by the U.S. Food and Drug Administration (FDA) for second-line therapy in treating non-small cell lung cancer (NSCLC). Yet, their efficacy in CRC remains suboptimal due to the emergence of drug resistance. Current research on KRAS inhibitor resistance in CRC focuses mainly on single biological processes, leaving the complex interplay between cellular systems underexplored. This review synthesizes evidence across four key dimensions — metabolism, epigenetics, immunology, and proteostasis — to reveal multidimensional mechanisms of resistance. How these interconnected pathways work in synergy with signaling aberrations, genetic alterations, and translational modifications to reinforce resistance was investigated. The research also integrates recent advances in multi-targeted strategies, including combinations of KRAS inhibitors with metabolic or epigenetic modulators or immune checkpoint blockade. These findings provide potential strategies for overcoming resistance in KRAS-mutant CRC, addressing a critical gap in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de09a3f40e67f360dfd11b639fa71c10cdd13842" target='_blank'>
              Unravelling Resistance: Integrating Metabolism, Epigenetics, Immunology, and Proteostasis in Strategies against Kirsten Rat Sarcoma Viral Oncogene Homolog-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            Jingyi Li, Na Song, Rui Ma, Xiujuan Qu
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Transposable elements (TEs), commonly referred to as “jumping genes,” are discrete DNA fragments capable of mobilization within the genome, often creating new copies during transposition. Once considered non-functional “junk DNA,” TEs are now recognized as essential regulators of genome evolution, gene networks, and epigenetic processes. They contribute to genome innovation through gene duplication, exon shuffling, regulatory rewiring, and stress-responsive plasticity. Conversely, their uncontrolled activation can disrupt genetic integrity. Such aberrant TE activity has been associated with immune dysfunction, cancer, neurodegeneration, and reproductive disorders. 
In livestock, recent studies have identified TE-derived regulatory elements that influence growth, reproduction, immunity, and adaptation. Notably, TEs constitute approximately 45–60% of the genomes of major livestock species, including pigs, sheep, and cattle, highlighting their extensive regulatory impact. Advances in genome editing and artificial intelligence have facilitated their functional characterization. This allows a sharper focus on TE-driven regulation in animal genomes without repeatedly specifying the livestock context. 
Emerging technologies such as CRISPR–transposon systems, programmable epigenome editing, and AI-driven TE mapping offer new opportunities for precise genome engineering, functional genomics, and livestock breeding. This review consolidates current knowledge on TE classification, molecular mechanisms, evolutionary roles, disease associations, and biotechnological  applications, with emphasis on livestock genomics. It also outlines existing challenges, biosafety considerations, and future directions toward harnessing TEs for precision animal breeding and synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434f9971fc3a63cead1542771f75e88db7b4209d" target='_blank'>
              Jumping Genes (Transposable Elements): A Comprehensive Review of Their Molecular Mechanisms, Evolutionary Impact and Applications in Livestock and Biomedicine
              </a>
            </td>
          <td>
            Dhruv Bhatia, Prakashji Thakor, Pankti J Naik, Brijal Barot
          </td>
          <td>2025-12-22</td>
          <td>Journal of Experimental Agriculture International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract CRISPR-Cas-based live-cell imaging has rapidly become a central technology for studying genome dynamics with high specificity and flexibility. By coupling nuclease-deactivated Cas (dCas) with programmable guide RNAs, genomic loci can be tracked in living cells, providing direct insights into nuclear organization and chromatin behavior. While repetitive regions such as telomeres and centromeres are readily visualized, labeling non-repetitive loci remains more challenging due to weak signals and high background. Recent advances, including multicolor labeling strategies, innovative amplification systems based on dCas9 and single-guide RNA (sgRNA) engineering, and integration with novel fluorescent reporters, have markedly expanded the applicability of CRISPR imaging across the genome. These developments have expanded the multiplexing capacity of CRISPR imaging, improved signal-to-background ratios, and even enabled the visualization of non-repetitive genomic loci. Nonetheless, key challenges remain, including cellular toxicity, replication stress, and genomic instability associated with prolonged CRISPR expression. In this review, we summarize recent advances in CRISPR live-cell imaging and highlight key design trade-offs and biological constraints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29cd97e95bdd92ebe7f7918448904c9cf53cd83" target='_blank'>
              Illuminating the genome: emerging approaches in CRISPR-Cas live-cell imaging
              </a>
            </td>
          <td>
            Zhiguang Xiao, Yujie Sun
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This paper delves into the role of epigenetics in complementing and challenging Darwinian evolution theory within the field of evolution. Epigenetics has revealed new sources of genetic variation, with epigenetic variations such as DNA methylation and histone modifications capable of influencing gene expression without altering DNA sequences. These variations can spread and accumulate within populations, exhibiting reversibility and dynamism, enabling species to rapidly adapt to environmental changes. Additionally, epigenetics expands the scope of natural selection, revises the patterns of genetic information flow, and highlights the need for updated evolutionary predictive models. Researchers have proposed theoretical integration pathways such as the Extended Evolutionary Synthesis framework, multilevel selection theory, and gene-epigenome coevolution models, offering new perspectives and directions for evolutionary biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b730d3e4497777825de2252ebd00338b352de17" target='_blank'>
              Epigenetics Complements and Challenges Darwinian Evolution: From Gene Regulation to the Study of Species Adaptation
              </a>
            </td>
          <td>
            Meichen Zhou
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The study aimed to identify the clinically relevant gene variants in unresectable cutaneous melanoma samples from Ukrainian patients using NGS technology and to investigate some pharmacogenomic markers useful for the development of cancer treat- ment strategies.


MATERIALS AND METHODS
30 samples of unresectable cutaneous melanomas of various localizations and differen- tiation grades were analyzed. The Ion Torrent NGS technology of targeted gene sequencing (Custom AmpliSeq™ Cancer hotspot and Pharmacogenomic panels) was applied to identify the genetic alterations, which were classifi d using franklin by Gennox database and custom pharmacogenomic Ion Reporter Software pipeline.


RESULTS
A total of 148 different gene alterations were identifi d in 40 genes (SNVs, MNVs, INdELs) by the Cancer hotspot Panel, revealing the mutation patterns consistent with the international data. however, notable discrepancies exist, such as a high KRAS mutation rate (29.3%), predominantly in stage III tumors, suggesting their role in tumor aggressiveness and progression. We identifi d the frequent TP53 and BRAF mutations, with BRAF V600E being the most common, and observed a higher prevalence of BRAF mutations in females. TP53 mutations were prevalent (59.3%) and varied with age and sex, though their prognostic signifi ance requires further validation. A second key novel fi ding was the detection of FLT3 mutations in 22.2% of the samples, with a signifi antly higher prevalence in stage IV disease, suggesting a potential role of FLT3 in melanoma progression and warranting further investigation as a prognostic biomarker and a potential target for the existing FLT3 inhibitors. Ultimately, our pilot analysis of pharmacogenomic markers un- derscored their potential clinical utility in choosing personalized treatment decisions. for instance, the identifi ation of a patient with the rs35599367 (G/A) risk allele for adverse drug reactions underscores the value of using pharmacogenetic data to make a correct selection between approved therapies.


CONCLUSIONS
Our study presents the fi st comprehensive analysis of somatic mutations and pharmacogenomic markers in unresectable cutaneous melanoma tissue samples from Ukrainian patients. We found that while common somatic variants generally align with global trends, the mutational landscape in this cohort presents several unique features: a high KRAS mutation rate and its apparent stage-specifi prevalence, and a high FLT3 mutation rate, predominantly in stage IV tumors. further validation on a larger number of samples, as well as more exhaustive analysis employ- ing alternative methods, is necessary to substantiate these fi dings and to facilitate more effective melanoma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ade3027bdd0b58ffd4bfe0744885c0eb045394f1" target='_blank'>
              SOMATIC GENE VARIANTS IN UNRESECTABLE CUTANEOUS MELANOMA CELLS AND DETECTION OF PHARMACOGENOMIC MARKERS: INFLUENCE ON STRATEGY OF EFFECTIVE CANCER TREATMENT.
              </a>
            </td>
          <td>
            R. Gulkovskyi, G. Gerashchenko, A. Bezverkhiy, N. Melnichuk, O. Mankovska, T. Marchyshak, H. Karaman, M. Krotevych, S. Trokhymych, O. Ruban, Z. Tkachuk, M. Tukalo, V. Kashuba
          </td>
          <td>2025-12-30</td>
          <td>Experimental oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="TP53 mutation-driven gene expression programs define oncogenic phenotypes. While extensive studies have concentrated on the transcriptome and proteome, post-transcriptional processes, particularly translational variation, remain underexplored. This study presents a comprehensive analysis of the transcriptomics, translatiomics, and proteomics dynamics in the ovarian cancer cell line SKOV3, with a focus on the effects of p53 missense mutations (R175H, R273H, and Y220C) on gene dosage fluctuations. Despite clear transcriptional differences between wild-type and mutant p53, we find that extensive translational and post-translational buffering processes attenuate these discrepancies, yielding comparatively stable protein abundances. Moreover, we delineate that the relative contributions of transcription output, translation engagement, and protein stability collectively shape the final protein abundance in the context of p53 mutations. Clinical proteomic analysis of platinum-resistant ovarian cancer tissues reveals tumor-specific factors and acquired resistance pathways linked to p53 mutations. Our findings elucidate the multilayered regulatory landscape of p53 mutations and identify potential risk factors for platinum resistance associated with these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/765c815e9b25f95d1edceba753e2ba1a94641e62" target='_blank'>
              Multilayered Regulatory Dynamics of p53 Mutations and Platinum Resistance in Ovarian Cancer.
              </a>
            </td>
          <td>
            Liling Hu, Hanchen Zou, LvYing Peng, Fan Li, Danya Liu, Jiangli Lu, Yuying Li, Chris Zhiyi Zhang, Qiu-Hong Tian
          </td>
          <td>2025-12-16</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Oncofusions, cancer-causing chimeric proteins formed by chromosomal rearrangements, are key drivers in approximately 16% of all cancer cases. These fusion proteins are associated with a wide range of malignancies, including varieties of leukemia, lung cancer, and sarcoma. Thousands of unique oncofusions have been identified in recent years, yet most of these fusion proteins lack mechanistic annotation. Many oncofusions dysregulate chromatin architecture in complex ways that are not yet fully elucidated. Investigating these chromatin alterations is both labor-intensive and costly, limiting the feasibility of large-scale studies. This lack of mechanistic insight has hindered the development of targeted therapies, with only a handful of efficacious small-molecule inhibitors directed toward oncofusions currently in clinical trials or approved for patient use. To address this critical gap, we adapted our recently developed PROD-ATAC method into a high-throughput, scalable platform to systematically elucidate the effects of dozens of candidate inhibitory small molecules on the chromatin dysregulation profiles induced by a pooled library of diverse oncofusions. In the PROD-ATAC scheme, a pooled protein-coding variant library is expressed in a single disease-agnostic cell line, and scATAC-seq (single-cell assay for transposase accessible chromatin) catalogs each variant’s effect on the chromatin landscape. We applied PROD-ATAC to simultaneously assay the chromatin regulatory impact of >40 small molecule inhibitors against a panel of tens of oncofusions (including the majority of patient-identified fusion sequences causal for Ewing sarcoma) in a single assay, generating genome-wide chromatin accessibility datasets representative of >1,000 unique drug - fusion interactions. In this manner, we characterized several small molecules as previously unidentified inhibitors of multiple oncofusions. Certain inhibitors were found to abrogate the chromatin dysregulation caused by an oncofusion either globally or in a targeted cell signaling pathway-specific manner. By profiling multiple rare fusions causal for the same cancers, we identified separate instances of homogenous and heterogenous modes of chromatin dysregulation associated with distinct fusion proteins. In this work, PROD-ATAC chromatin accessibility readouts coupled the identification of novel oncofusion inhibitors with the characterization of each inhibitor’s impact on the epigenetic landscape of oncofusion-expressing cells.



 James E. Corban, Max Frenkel, Srivatsan Raman. High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff99b547f5b85ca4c657ff5fd20f9c13db3f972" target='_blank'>
              Abstract A024: High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions
              </a>
            </td>
          <td>
            James E. Corban, Max Frenkel, Srivatsan Raman
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Advances in single cell multi-omics technologies have allowed for investigation into genotype-immunophenotype relationships and dynamic clonal changes in cancer patient samples. These technologies provide rich insights into genetic profiling, drivers of disease progression, and subclonal dynamics. Increased adoption and throughput of these technologies, necessitate accessible computational tools for processing and analysis. Toward developing easy to use computational tools, we introduce the Single Cell DNA (scDNA) package that allows for rapid analysis of single-cell molecular profiling. Our platform aims to provide a diagnostic summary tool for sample quality control, representative subclonal architecture within a sample, demultiplexing functionality for multi-sample processing, copy number variation, and mutation trajectory analysis to identify order of subclonal mutation acquisition. We showcase a series of vignettes on hematopoiesis datasets for each aim that reflects recently deployed uses of scDNA. Additionally, we show scDNA provides a modular, user-friendly framework that readily feeds into other standard software pipelines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e9603df019d6ce21c68f981f038fde96385eec2" target='_blank'>
              scDNA: Single Cell DNA analysis software toolkit for subclonality discovery and assessment
              </a>
            </td>
          <td>
            Michael Bowman, Shreeya Gounder, Varsha Singh, Olga Shestova, Troy Robinson, Amy Zhang, Anushka Gandhi, Roopsha Bandopadhyay, Sheng F Cai, R. L. Levine, Saar I. Gill, Linde A. Miles, Robert L. Bowman
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genome-wide association studies (GWAS) of melanoma have identified numerous susceptibility loci. However, causal genes and variants underlying risk have yet to be established for most. It is becoming apparent that many functional variants underlying complex traits act via cis-regulation that may be context-specific, dependent on availability of specific transcription factors/complexes in specific cell types and cell-states. To characterize a risk locus on chromosome band 2q33.1 associated with melanoma, breast cancer, and keratinocyte cancers, we integrated fine-mapping, cell-type specific expression quantitative trait locus (eQTL) analysis, a massively parallel reporter assay, individual luciferase assays, and SNP-based proteomics. Integrated analysis implicates the presence of multiple functional variants lying primarily within a promoter for CASP8. A haplotype containing rs3769823 appeared have the largest effect on expression. Strikingly, both tumor/normal context and this risk-associated haplotype play critical roles in mediating allelic cis-regulatory activity. Quantitative mass spectrometry for rs3769823 identified both E4F1, a transcriptional repressor, and IRF2, a transcriptional activator, as binding preferentially to risk-associated rs3969823-A. The binding of these transcription factors was validated via EMSA, supershift, and chromatin immunoprecipitation (ChIP) assays. The relative levels of E4F1 and IRF2 differ by cell-type and play a role in mediating transcriptional activity in a cell-type specific manner. Our results indicate that the top credible causal set variant rs3769823 likely influences expression of CASP8 and FLACC1 in a cell-type specific manner and may be a relevant functional variant for multiple cancers associated with this locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0bd03d9d9be45c3f139097e5cecc84ff17cc6a" target='_blank'>
              Functional characterization of a multi-cancer risk locus on chromosome band 2q33.1 near CASP8
              </a>
            </td>
          <td>
            Hyunkyung Kong, Jiyeon Choi, Tongwu Zhang, Cathrin Gräwe, Mai Xu, Rohit Thakur, Hayley A. Sowards, Rebecca Hennessey, Andrew Vu, Jianxin Shi, T. Bishop, Julia Newton-Bishop, Jérémie Nsengimana, M. Iles, M. Landi, M. Vermeulen, M. Law, L. Amundadottir, Melanoma Meta-7 Analysis, Consortium, Kevin M. Brown
          </td>
          <td>2026-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Simple Summary ADP-ribosylation is a post-translational modification that plays a crucial role in DNA repair, transcription, replication, and overall cell fitness. The ADP-ribose cycle represents one of the most tightly regulated biochemical pathways in the cell, as precise control of mono- and poly-ADP-ribose levels, together with the coordinated activity of the enzymes that synthesize and recycle ADP-ribose to NAD+, is essential for cellular homeostasis. Moving beyond the traditional cancer framework focused exclusively on oncogenes and tumor suppressor genes, ADP-ribose can be viewed as a central signaling molecule that regulates proteins involved in tumor progression and adaptation to stress conditions within the tumor microenvironment. In this context, proteins whose roles in ADP-ribosylation are relatively less characterized, such as mono-ADP-ribosyltransferases (MARTs) and poly(ADP-ribose) glycohydrolases (PARGs), are emerging as important regulators of these processes. As their functions and biological implications continue to be elucidated, these enzymes should be considered promising prognostic markers and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90553f7630f85c06dd88b57a27b79fc887c602e0" target='_blank'>
              Targeting MARylation and DePARylation in Cancer Therapy: New Promising Therapeutic Opportunities
              </a>
            </td>
          <td>
            Vanesa Cabeza-Fernández, Francisco Javier Ríos-Sola, D. Martín-Oliva, Jerónimo Borrego-Pérez, F. J. Oliver, J. Yélamos, J. Rodríguez-Vargas
          </td>
          <td>2025-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b41a50acd50b88a1b8f2629c5c9719051133237" target='_blank'>
              The inhibition of de novo purine synthesis increases LAMP2 expression to preserve cell viability.
              </a>
            </td>
          <td>
            Angela De Cristofaro, Serena Castelli, Federica Felice, M. Ciriolo, Enrico Desideri
          </td>
          <td>2025-12-11</td>
          <td>Cell death discovery</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Gastric cancer remains a highly prevalent and accounts for a notable proportion of global cancer mortality. Both Intrinsic and exogenous agents can exacerbate reactive oxygen species (ROS) related oxidized DNA base lesions and single stranded DNA breaks (SSBs). Base excision repair (BER) serves as the primary defense mechanism for repairing DNA damage induced by oxidative stress. DNA polymerase beta (Pol β) plays a critical role in BER and non-homologous end joining repair pathways. The Pol β is the first perform gap-filling DNA synthesis by its polymerase activity and then cleave a 5′-deoxyribose-5-phosphate (dRP) moiety via its dRP lyase activity. Furthermore, defect in POLB promotes genetic liability of the cancer cells for different targeted and synthetic lethality-based treatment strategies. In this review, we have provided a potential example to illustrate the mechanistic insight how PARP1 inhibitor (Olaparib) induces replication associated double strand breaks in POLB deficient cells and DNA mediated innate immune signal activation that likely enhances immune based therapy. Based on our previously published data and the current recent findings, POLB status of the patient likely provide genetic indicators to stratify gastric cancer patient. Overall, in this review article, we presented a new direction to highlight the opportunity to exploit POLB genetic defect in cancer cells to enhance treatment response and to explore synergistic effect to target gastric cancer cells that harbor aberrant DNA polymerase beta function with immune based therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0501b8820aaf9cfdd67e2f7a15f0de284236eed" target='_blank'>
              Harnessing DNA polymerase beta defect enhances synthetic lethality and treatment response in gastric cancer cells: implication for immunotherapy
              </a>
            </td>
          <td>
            Aashirwad Shahi, Shengyuan Zhao, Dawit Kidane
          </td>
          <td>2026-01-06</td>
          <td>Journal of Pharmacy &amp; Pharmaceutical Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Introduction Transfer RNA-derived small RNAs (tsRNAs) have emerged as critical regulators in cancer biology, influencing gene expression, protein synthesis, and cellular signaling. Their unique expression patterns and functional diversity highlight their potential for diagnostic, prognostic, and therapeutic potential, particularly in hepatocellular carcinoma (HCC), a major cause of cancer-related mortality worldwide. Areas covered This review provides the first integrative overview of tsRNAs in HCC, encompassing their biogenesis, classification, molecular functions, and involvement in tumor hallmarks, including proliferation, metastasis, apoptosis, metabolism, angiogenesis, and immune modulation. We also summarize current advances in detection methods and databases and highlight the translational potential of tsRNAs as diagnostic, prognostic, and therapeutic targets. This work emphasizes unexplored dimensions of tsRNA-mediated regulation, connecting mechanistic insights with clinical applications. Expert opinion Although tsRNAs show great promise in HCC diagnosis, prognosis, and therapy, clinical translation remains hindered by gaps in mechanistic understanding, technical challenges in detection, and a lack of large-scale validation. Overcoming these limitations through standardized methodologies and multi-omics integration could unlock their full potential in precision cancer medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad865e85ec3c0a41a64171ec5b4682d2f6b506a" target='_blank'>
              Emerging role of transfer RNA-derived small RNAs (tsRNAs) in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Abdelhamid M. Abdelhamid, Tasneem Abaza, Waleed T Kotb, Adham H Mohamed, Mohamed Z. Gad, Ayman M Metwally, Cédric Coulouarn, R. A. Youness
          </td>
          <td>2025-12-02</td>
          <td>Expert Review of Molecular Diagnostics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a highly plastic subpopulation within tumors that drives tumor initiation, progression, metastasis, therapeutic resistance, and disease relapse. Although CSC traits were initially attributed to stable genetic hierarchies, growing evidence suggests that cancer stemness is a dynamic and reversible state regulated by precise molecular events at the protein level. This literature review aims to synthesize experimental evidence regarding the impact of single amino acid mutations and alterations in regulatory proteins on the stemness properties of cancer cells. A systematic literature search was conducted using PubMed and Google Scholar to identify original research articles published between 2020 and 2025 that examined protein-level mutations, post-translational modifications, epigenetic regulators, and chromatin remodeling factors in diverse cancer models. The reviewed studies consistently demonstrate that single amino acid substitutions in oncogenic, tumor suppressor, or regulatory proteins can profoundly affect protein conformation, stability, enzymatic activity, and protein–protein interactions. These alterations frequently result in sustained activation of key stemness-associated signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and STAT3, thereby enhancing self-renewal capacity, stem cell marker expression, tumor-initiating potential, and resistance to conventional therapies. In parallel, dysregulation of post-translational modification systems, such as phosphorylation, ubiquitination, acetylation, and methylation, promotes epigenetic reprogramming and reactivation of embryonic transcriptional programs that favor cellular plasticity. Moreover, protein-level alterations strengthen cancer cell interactions with the tumor microenvironment, increasing responsiveness to hypoxia, inflammatory signals, and metabolic stress. Collectively, these findings highlight protein-level regulation as a central determinant of cancer stem cell maintenance and plasticity, and suggest that effective therapeutic strategies should target aberrant protein stability, post-translational modification machinery, and stemness-related signaling networks to achieve durable clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f961c9b291a85047c8455458a035edcd22afcac" target='_blank'>
              IMPACT OF SINGLE AMINO ACID MUTATIONS OR ALTERATIONS IN REGULATORY PROTEINS ON THE STEMNESS PROPERTIES OF CANCER CELLS
              </a>
            </td>
          <td>
            Dewi Sartika, Irmatul Laili, Sri Utami
          </td>
          <td>2025-12-30</td>
          <td>Journal of Stem Cell Research and Tissue Engineering</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposons (TEs) are genetic elements that can change their positions within genome. They cause transcriptional activation or repression by inserting into regulatory elements of related genes or by modulating epigenetic modifications on regulatory elements. In addition, some transposons encode and express peptides or proteins that affect or disrupt the cellular biological functions. Recent studies have demonstrated that while host cells effectively control most transposons, certain harmful insertions can disrupt normal gene expression processes, potentially leading to diseases. In this mini-review, we discuss recent advances in understanding how transposons contribute to genomic instability. We hope to provide researchers and clinical practitioners with new insights into transposon biology and its potential implications for disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e746fb5f9c54cbee0068471cbc97060459d5187a" target='_blank'>
              Transposons disrupt genomic stability and trigger cancers
              </a>
            </td>
          <td>
            Weixia Dong, Menghui Li, Ping Li, Xiaoli Hou, Fei He, Shaoping Ji
          </td>
          <td>2025-12-12</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Maintaining genome integrity in germ cells is crucial for fertility and species survival. However, the DNA repair mechanisms that sustain genome integrity in primordial germ cells (PGCs), which cope with high levels of replication stress, remain largely unknown. While the TatD family of proteins, evolutionarily conserved nucleases, has been found to play roles in various DNA-related processes, their in vivo functions in vertebrates have yet to be fully elucidated. TATDN2 has recently been implicated in resolving R-loops and participating in the replication stress response in BRCA1-deficient cancer cells. Here we found that tatdn2 exhibits conserved expression in mitotic and early meiotic germ cells across teleosts and mammals. Using medaka fish as a model, we then showed that loss of tatdn2 leads to all-phenotypically male adults and infertility due to PGC depletion during mitotic proliferation. We further demonstrated that knockout of tatdn2 increases R-loop accumulation and DNA damage, subsequently triggering apoptosis in PGCs. These findings indicate that tatdn2 plays a critical role in DNA damage repair associated with R-loop resolution in mitotic PGCs. Our study provides novel insights into the physiological function of TATDN2 and the mechanisms of genome maintenance in PGCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aba7e41b7a154253da5adb67bf89677856b6ff" target='_blank'>
              Tatdn2 is required for DNA repair to safeguard genome stability in primordial germ cells
              </a>
            </td>
          <td>
            Hanqiao Shang, Puxuan Jiang, Zehui Wang, Zhaojun Shan, Yuze Chen, Ting Zhang, Yingshu Li, Qiang Tu
          </td>
          <td>2025-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is a common and highly lethal type of primary brain tumor in adults. Therapeutic failure is partly attributed to a fraction of Glioblastoma Stem Cells (GSCs) that show high levels of heterogeneity and plasticity. GSCs exist in a transcriptional gradient between two states: Developmental (Dev) and Injury Response (IR) in which IR-GSCs exhibit more invasive behaviors. While previous studies have identified fitness genes in GSCs, the genes required to establish and maintain the Dev and IR states remain poorly defined.



 To identify the regulators of the IR GSC state, we performed a phenotypic genome-wide CRISPR-Cas9 knockout screen in patient-derived GSCs based on cell surface expression of the IR marker CD44. Validation of EP300 in regulation of IR state was performed using CRISPR gene editing and A-458 inhibitor treatment. RNA-seq, CUT&RUN, invasion assays, sphere forming assays and a mouse GBM model were used to characterize the phenotypic consequences of EP300 perturbation.



 We found that perturbation of the histone acetyltransferase EP300 led to decreased CD44 cell surface expression, and loss IR transcriptional state identity through dysregulation of the epigenome. Functional studies demonstrated that this loss of state identity coincides with decreased self-renewal and invasion in GSCs and delayed tumor initiation and progression in a mouse GBM model.



 Collectively, our results establish EP300 as a key regulator of IR state identity in GSCs and provide a mechanistic basis for therapeutic targeting of aggressive cellular states in GBM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbc9095ec6b337591d9385380c51a0b5670513e" target='_blank'>
              Epigenetic maintenance of the Injury Response state in Glioblastoma Stem Cells
              </a>
            </td>
          <td>
            Fatemeh Molaei, G. Macleod, S. Haider, Aleczandria S. Tiffany, Frank M Oteng, Jacob M Berman, Arianna Skirzynska, Molly S. Shoichet, D. Schramek, Peter B. Dirks, Stéphane Angers
          </td>
          <td>2026-01-10</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebed292326a57b5e447307df550aebc051db233e" target='_blank'>
              RNA-guided STAT3 modification fine tunes the epigenetic and epitranscriptomic regulation of CD4 + T helper cell differentiation during non-small cell lung cancer (NSCLC)
              </a>
            </td>
          <td>
            Roshni Bibi, Melvin George, Koustav Sarkar
          </td>
          <td>2025-12-27</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The Methyl CpG Binding Protein 2 (MECP2) gene has copy number gain in a number of human cancers and functions as an oncogene through an unusual epigenetic mechanism. We explored the possibility that MECP2 might be a therapeutic target, and whether its epigenetic mode of action is amenable to specific therapy. Constitutively expressed or inducible lentiviral short hairpin RNA (shRNA) directed at MECP2 in human triple negative breast cancer (TNBC) cell lines with high or low level of MECP2 protein were grown as xenografts to assess oncogene addiction. We next evaluated the effect of DNA methylation inhibitors or histone deacetylase inhibitors on monolayer or soft agar growth of isogenic human mammary epithelial cells with or without MECP2 overexpression, and xenograft growth of MECP2-dependent or MECP2-independent human TNBC or lung cancer cell lines. We then investigated the mechanism of MECP2-induced activation of MAPK pathway and assessed the effect of drug treatment. Human cancer cell lines with MECP2 overexpression show MECP2 dependence, and epigenetic drugs are effective in these models. Activated RAS and other activators of the MAPK pathway caused resistance to these therapies, giving insight into their novel mode of action and demonstrating specificity. The kinase PAK3 is important for MECP2-mediated induction of MAPK pathway, is modulated by epigenetic drugs affecting MECP2 action as expected and may itself be a therapeutic target for MECP2-driven cancers. These preclinical studies show that tumors overexpressing MECP2 might benefit from epigenetic therapy targeting MECP2 function and demonstrate a novel mechanism of action for these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4802b6674f8b3e0a4e440436dd1fc69dc98c09c" target='_blank'>
              MECP2-Dependent Cancers Can Be Targeted by Epigenetic Drugs: A New Role for Epigenetic Cancer Therapy.
              </a>
            </td>
          <td>
            Manish Neupane, Mukesh Kumar, Malela A Mwamufiya, Colleen M Donnelly, Alsu Ibragimova, Brian Curcio, Inna Chervoneva, Irina Vasilevskaya, Daniel P Silver
          </td>
          <td>2025-12-11</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Persistent fetal gene expression in childhood neoplasms is usually explained by a maturation block originating in the prenatal phase. In contrast, reactivation of fetal genes in adult malignancies is considered a consequence of oncofetal reprogramming (OFR) and is associated with aggressive disease. By reconstructing epigenetic ontogeny in juvenile myelomonocytic leukemia (JMML), we identified a postnatal maturation state of JMML stem cells with high transcriptional plasticity indicative of OFR in high-risk disease. Similarly, post-natal activation of oncogenic signaling by inducible Ptpn11E76K mutation in mice, triggered molecular plasticity and reactivation of fetal gene expression. Integrative multi-omics analysis revealed aberrant CD52 expression as a feature of high-risk JMML stem cells. Anti-CD52 treatment depleted JMML stem cells and blocked disease propagation in xenograft models. Our results challenge the prevailing maturation-block model of pediatric leukemogenesis and establish RAS-associated stem-cell plasticity as a determinant of OFR and potential therapeutic vulnerabilities in high-risk JMML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c603c5b4c0adc417c6958428e24696c22fabe88d" target='_blank'>
              Molecular Plasticity Results in Oncofetal Reprogramming and Therapeutic Vulnerabilities in Juvenile Myelomonocytic Leukemia.
              </a>
            </td>
          <td>
            M. Hartmann, M. Schönung, J. Rajak, Valentin J. Maurer, Ling Hai, K. Bauer, M. Hakobyan, Sina Stäble, Jens Langstein, Laura Jardine, R. Roelz, S. Bohler, Eleonora Khabirova, Abdul-Habib Maag, Dominik Vonficht, D. Lebrecht, K. Bernt, Kai Tan, Changya Chen, Fatemeh Alikarami, Julia A. Meyer, Jun Wang, Tobias Boch, Viktoria Flore, P. Lutsik, M. Milsom, S. Raffel, Christian Buske, Simon Haas, M. Haniffa, Jan-Philipp Mallm, S. Behjati, M. Bonder, Stefan Frohling, Elliot Stieglitz, Charlotte M. Niemeyer, Joschka Hey, C. Flotho, C. Plass, M. Erlacher, Matthias Schlesner, D. Lipka
          </td>
          <td>2025-12-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Background/Aim: Prostate cancer features profound transcriptional dysregulation within the androgen receptor (AR) signaling axis. The pioneer factor FOXA1, which facilitates AR binding to chromatin, is recurrently altered in 10-40% of tumors. Recent studies classify FOXA1 mutations as Class 1 Wing 2 mutations, which enhance AR-dependent tumorigenesis, and Class 2 C-terminal truncations, which promote lineage plasticity and therapy resistance. The interplay of FOXA1 alterations with TMPRSS2-ERG fusions and PROX1 remains incompletely understood. Materials and Methods: Data from The Cancer Genome Atlas (TCGA) Prostate Adenocarcinoma (PRAD) cohort (n=492) were analyzed via UCSC Xena and cBioPortal. FOXA1 mutations were categorized following Eyunni et al. Copy number was assessed by log2(tumor/normal) ratios. Mutual exclusivity and co-occurrence were evaluated using Fisher’s exact test with false-discovery-rate correction. Associations between FOXA1 status and genomic instability were assessed using the fraction genome altered (FGA) metric. Results: FOXA1 was broadly expressed, with subsets showing elevation. Class 1 mutations localized to the Wing 2 region, while Class 2 truncations clustered in the C-terminal domain. Copy number changes were infrequent, indicating mutation-driven reprogramming as the main oncogenic mechanism. TMPRSS2 and ERG strongly co-occurred (log2 OR >3, q<0.001), whereas FOXA1 was mutually exclusive with both TMPRSS2 and ERG (q<0.001). Although FOXA1 alterations showed no significant Pearson correlation with FGA (r=−0.01, p=0.76), a moderate Spearman correlation (ρ=0.52, p<0.001) suggested enrichment in genomically unstable tumors. Conclusion: FOXA1 defines a major oncogenic axis in prostate cancer, distinct from TMPRSS2-ERG fusion and PROX1 induction. Class 1 and 2 FOXA1 mutations drive alternative transcriptional programs leading to therapy resistance, highlighting FOXA1 as a critical biomarker and target for chromatin-directed interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/962282da85200138085414df96f69e7ea38e7831" target='_blank'>
              FOXA1 Alterations in Prostate Cancer: Expression, Mutation Classes, and Copy Number Changes
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2026-01-01</td>
          <td>In Vivo</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS), the most common pediatric soft tissue sarcoma, exhibits marked clinical heterogeneity driven by poorly understood molecular mechanisms. Identifying the molecular characteristics of different RMS subtypes and the molecular pathways influencing the RMS treatment response and recurrence is an urgent clinical need. Here, we perform deep proteomic profiling of 19 RMS tumors (8 alveolar [ARMS], 11 embryonal [ERMS]) and matched normal tissues, integrating bioinformatics with functional validation to delineate subtype-specific pathways, therapy resistance drivers, and actionable targets. ARMS tumors are characterized by ubiquitination pathway activation (UBE2R2, UBE2J2), while ERMS exhibits spliceosome dysregulation. Chemo- and radio-resistant tumors both show significant enrichment in the ribosome pathway. Relapsed cases show phosphonate and phosphinate metabolism pathway enrichment, suggesting metabolism reliance. Unsupervised clustering reveals ribosome- and glycolysis-driven subtypes with distinct metabolic dependencies. Functional studies implicate MED18─a core component of the Mediator complex─in mediating therapy resistance possibly via promoting DNA damage repair. Our study establishes proteomics as a tool to decode RMS heterogeneity, proposing subtype-tailored strategies targeting ubiquitination, splicing, and metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864be8bb8da57a2925ca6df4d25ae9855e5c9074" target='_blank'>
              Proteomic Profiling Reveals Candidate Proteins and Pathways Associated with Chemo-Radio-Sensitivity and Relapse in Rhabdomyosarcoma.
              </a>
            </td>
          <td>
            Zhiyuan Zhou, Ying Ye, Wenbin Guan, Chuanying Zhu, Lu Wen
          </td>
          <td>2026-01-15</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Central nervous system (CNS) metastasis is a major driver of morbidity in metastatic breast cancer, yet the molecular determinants of CNS tropism remain incompletely defined. LYN, a Src-family kinase integrating receptor tyrosine kinase and integrin signaling, is a biologically plausible mediator of metastatic traits. We performed a retrospective, multi-study analysis of publicly available breast cancer cohorts aggregated in cBioPortal. After harmonization and de-duplication, LYN status was determinable in 5,947 invasive carcinoma of no special type (NST) tumors across 29 studies. The primary endpoint was CNS metastasis at any time (Yes/No), harmonized via a prespecified controlled vocabulary (case-insensitive substring mapping). Somatic LYN variants (coding SNVs/indels) were collapsed to patient-level classes (missense-only; truncating if any nonsense/frameshift/splice). Variants with resolvable positions were mapped to Src-family modules (SH4/Unique, SH3, SH2, SH2-kinase linker, kinase). Two-group comparisons used two-sided Fisher’s exact tests with exact 95% CIs; domain screens used omnibus χ² and Benjamini–Hochberg FDR control. A prespecified Firth logistic model evaluated truncating vs. missense within LYN-mutant tumors. Public cancer genomics repositories (cBioPortal); multi-institutional cohorts. 5,947 tumors across multi-study cohorts with LYN status available. None. Primary: ever-CNS metastasis (yes/no). Secondary: distribution of LYN variant classes and domains (SH4/Unique, SH3, SH2, linker, kinase). CNS metastasis occurred in 5/46 (10.9%) LYN-mutated tumors vs. 110/5,901 (1.9%) LYN wild-type tumors (odds ratio (OR) = 6.42; 95% confidence interval (CI), 2.49–16.56; p = 0.0018). The endpoint was captured as ever vs. never CNS involvement (event dates unavailable), precluding time-to-event inference. Within LYN-mutant cases, an exploratory domain analysis indicated that distributions differed by CNS status (omnibus χ² p ≈ 0.014); a one-versus-rest signal at the SH4/Unique N-terminus was nominally significant and borderline after false discovery rate (FDR) (unadjusted p ≈ 0.010; q ≈ 0.052; small in-domain n = 3). By mutation class, truncating vs. missense showed a higher CNS-positive proportion (28.6% vs. 7.9%) but did not reach significance (Fisher p = 0.166; alternatively framed OR = 4.22; exact 95% CI, 0.58–30.75; p = 0.182). Firth estimates were directionally consistent with wide profile CIs under sparse counts. Across pooled cohorts, LYN mutation is associated with increased odds of CNS metastasis, and domain context appears informative, with a small-sample, FDR-borderline enrichment at the SH4/Unique N-terminus. The truncating-class signal is exploratory given limited power. Signals by domain (notably SH4/Unique) are exploratory and require independent validation in larger, uniformly annotated datasets. Given small mutant denominators and ever-CNS endpoint capture, findings are hypothesis-generating and not actionable for risk stratification or treatment selection. Results motivate domain-aware annotation in future validation studies and mechanistic work. What is already known: CNS metastasis is a major clinical problem in breast cancer; molecular determinants of brain tropism remain incompletely defined. What this study adds: Across pooled cohorts, any LYN mutation is associated with higher odds of CNS metastasis; domain context (notably SH4/Unique) may matter, albeit with small counts and FDR-borderline evidence. How this might affect research/practice/policy: Encourages domain-aware LYN annotation and targeted mechanistic work on membrane targeting/BBB traversal; not ready for risk stratification or treatment selection. What is already known: CNS metastasis is a major clinical problem in breast cancer; molecular determinants of brain tropism remain incompletely defined. What this study adds: Across pooled cohorts, any LYN mutation is associated with higher odds of CNS metastasis; domain context (notably SH4/Unique) may matter, albeit with small counts and FDR-borderline evidence. How this might affect research/practice/policy: Encourages domain-aware LYN annotation and targeted mechanistic work on membrane targeting/BBB traversal; not ready for risk stratification or treatment selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e375e78de0e51d3450906c5e0322f113edd318" target='_blank'>
              LYN mutations in breast cancer: hypothesis-generating evidence for an association with central nervous system metastasis and domain-level insights
              </a>
            </td>
          <td>
            Elif Kardelen Çağdaş, Berkay Çağdaş
          </td>
          <td>2025-12-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Traditionally, scientists tend to approach cancer research in a reductionistic way: aiming at uncovering underlying, separate components in malignant processes. And indeed, great progress has been made by reducing the development of a tumor to single, specific genes and mutations. For instance, familial adenomatous polyposis (FAP) could be reduced to a germline mutation in the Adenomatous Polyposis Coli (APC) gene. The escape of tumor cells from immune surveillance could be reduced to the tumor expression of immune checkpoints, resulting in new approaches in tumor therapy by applying immune checkpoint inhibitors. However, a germline mutation in APC is not 1:1 related to colorectal cancer (CRC), and only some patients respond to immune checkpoint inhibitors. The point here is that biological systems, also comprising cancer, have properties that cannot be reduced to single components. The cooperation of the single components results in new, emergent properties. The outcome of an interaction in a complex network, like the immune system, depends on the many cell types involved and the numerous molecules that interact and activate or inhibit pathways. The way the composing elements are organized is a causal factor in itself for any emergent property. The rise of genomic analysis at the end of the previous century, enabling us to sequence a full genome at the DNA and RNA levels, has initiated an awareness of the need for ‘systems biology’: to consider a full system and how it is organized, in all of its aspects, to understand biological pathways and their outcomes. In this review, we outline the prospects and limitations of systems biology in cancer research and propose a causal framework that integrates upward and downward causation and multiple realizability to understand the emergent properties of tumors that determine the dynamics of tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bad26730d321f49013a748603a1b42d130f54226" target='_blank'>
              The Need for a Systems Biology Approach in Cancer Explained
              </a>
            </td>
          <td>
            Hehuan Zhu, Xi Zhang, E. Nazemalhosseini-Mojarad, J. Roelands, L. D. A. N. de Muynck, C. Ravensbergen, Rachel Hoorntje, I. Stouten, Marianne Hokland, A. L. Vahrmeijer, Rob A. E. M. Tollenaar, Edwin Koster, Peter J K Kuppen
          </td>
          <td>2025-12-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5df76d59f3a1b2aa9161ce2bf7becbe9cb09edf1" target='_blank'>
              Uncovering the role of LINE-1 in the evolution of lung adenocarcinoma.
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, P. Joubert, Jian Sang, John P McElderry, Michelle Antony, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J Colón-Matos, M. Miraftab, Monjoy Saha, Olivia W. Lee, Kristine M. Jones, N. E. Caporaso, M. Wong, Kin Chung Leung, C. A. Hsiung, Chih-Yi Chen, E. S. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, B. G. Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, R. Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin M. Brown, Nathaniel Rothman, S. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, D. Wedge, M. Landi
          </td>
          <td>2025-12-10</td>
          <td>Nature</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e8ef7d28e9d10b44890e1b7be7668548430edd" target='_blank'>
              CHEK2 loss endows chemotherapy resistance to hematopoietic stem cells.
              </a>
            </td>
          <td>
            Jing Zhou, Tianyuan Hu, Dian Li, Sanming Li, Minhua Li, Xiangguo Shi, Daisuke Nakada
          </td>
          <td>2026-01-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="During a lifetime, normal cells accumulate thousands of changes in their genome sequence. These changes, termed somatic mutations, have mostly been studied in the context of cancer, but their presence in normal tissues is ubiquitous and widespread. Somatic mutation accompanies the aging process and is influenced by genetic and environmental factors. Differently from gene expression or imaging data, which fluctuate over time, somatic variants are non-reversible marks in the genome and accumulate over time. This property can be exploited to track the history of a cell, from conception to old age, providing information that cannot be acquired via classical histological tissue inspection nor other types of omics data. Mutations can track embryonic development, measure how clones compete in a tissue over time, or report the mutational processes active in cells and tissues throughout life. We discuss selected examples and emphasize how somatic mutation analysis can enable expanding applications at the service of physiology and cell biology, as well as a deeper understanding of the aging process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e1754e49deb7eef5ac28d0fb31ba15c272ffd93" target='_blank'>
              Reconstructing the lifelong history of cells and tissues via somatic mutation analysis
              </a>
            </td>
          <td>
            Sipontina Faienza, J. P. Margaria, Irene Franco
          </td>
          <td>2025-12-01</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: Genomic integrity is crucial to the cellular life cycle, which involves a tightly regulated process where cells progress through specific phases to ensure that fully replicated, undamaged DNA is inherited by daughter cells. Any dysfunction in this process or unrepaired DNA damage leads to cell cycle arrest and programmed cell death. Cancer cells are known to exploit these mechanisms to continue dividing. Usually, DNA damage arrests replication, allowing the DNA Damage Response (DDR) pathway to activate, which repairs the DNA or bypasses the damage to support cell survival and preserve genome integrity. For DNA damage bypass or translesion synthesis (TLS), a group of low-fidelity polymerases perform error-prone DNA synthesis opposite damaged bases, where REV1 functions as the main scaffolding protein. Previously, we reported non-TLS functions of REV1, including its role in triggering DNA damage-dependent specific DNA metabolic processes. Methods and Results: In this study, we demonstrate that REV1 plays a significant role in cell cycle progression and that its loss causes arrest at the G2/M phase in flow cytometry analysis. This unexpected phenotype includes dysregulation of G2/M regulators, such as Cyclin B1 and tubulins, in REV1-deficient cells compared to controls, as quantified by Western blot. Additionally, phosphorylation of histone H3 at serine 28 was significantly reduced in these REV1-deficient cells. These G2/M arrest features were even more pronounced in REV1-deficient cells treated with the tubulin inhibitor colchicine. Conclusions: Overall, this study reveals a previously unrecognized link between REV1 TLS polymerase inhibition and the G2/M cell cycle arrest.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/349f4407e91b1d8708aa93b1aa069a7dcc5ae83b" target='_blank'>
              REV1 Loss Triggers a G2/M Cell-Cycle Arrest Through Dysregulation of Mitotic Regulators
              </a>
            </td>
          <td>
            Brailey Buntin, Madison Guyette, Vihit Gupta, Kanayo Ikeh, Sombodhi Bhattacharya, Erica N. Lamkin, Allison Lafuze, Roxana del Rio-Guerra, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-12-31</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745490de31a317b3b5c109c205a2bdf3115f7580" target='_blank'>
              Structural Maintenance of Chromosomes 5/6 complex dysfunction enables tumor mutagenesis
              </a>
            </td>
          <td>
            T. Tran, J. Fan, X. Zhao, A. M. Green
          </td>
          <td>2025-12-05</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Bladder cancer is a prevalent and lethal malignancy worldwide, with treatment options often limited by the development of chemoresistance, especially to cisplatin-based regimens. In this study, we identified ubiquitin-specific protease 54 (USP54) as a novel regulator of bladder cancer cell proliferation, stemness, and cisplatin resistance. USP54 expression was consistently downregulated in both clinical bladder cancer specimens and cell lines. Functional experiments demonstrated that USP54 overexpression inhibited cancer cell growth, attenuated CSC-like properties, and restored cisplatin sensitivity in resistant cells by promoting drug-induced apoptosis. Mechanistically, USP54 directly interacted with SIRT6 and facilitated its deubiquitination, leading to increased SIRT6 protein stability. This post-translational stabilization of SIRT6 was shown to mediate the tumor-suppressive functions of USP54 in bladder cancer progression and chemoresistance. Together, these results uncover the USP54-SIRT6 axis as a previously unrecognized regulatory pathway controlling stemness and cisplatin response in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94fe31451f9b92eb7a9c410e61c50d70da9b9403" target='_blank'>
              USP54 suppresses bladder cancer proliferation, stemness, and cisplatin resistance via deubiquitination-mediated stabilization of SIRT6.
              </a>
            </td>
          <td>
            Yi-nan Han, Shengxing Wang, Rurui Xia, Bang-fen Zhou, Shigong Huang, Wengui Xie, Shijie Guo
          </td>
          <td>2025-11-26</td>
          <td>Journal of chemotherapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Transcriptional reprogramming through induced proximity has emerged as a powerful strategy for modulating the expression of oncogenic and tumor-suppressive genes. Inspired by transcriptional reprogramming approaches such as transcriptional/epigenetic chemical inducers of proximity (TCIPs) that link BCL6 inhibitors to transcriptional regulators, we sought to develop covalent ligands that rewire BCL6 proximity to selectively suppress MYC transcriptional output while derepressing BCL6 target loci. Through a chemistry-driven and chemoproteomics-enabled design strategy, we generated a panel of BCL6-based electrophile-bearing hybrid ligands and identified a nondegradative molecular glue, ZD-1-186, that potently suppresses MYC and robustly induces CDKN1A (p21) in diffuse large B-cell lymphoma cells. ZD-1-186 downregulates MYC more effectively than BCL6 inhibitors or degraders, while strongly derepressing canonical BCL6 targets, including p21. Through BCL6 pulldown proteomics, ZD-1-186 induced a selective recruitment of the noncanonical BAF complex subunit BRD9 to BCL6 and covalently modified BRD9 at C288. Pharmacologic inhibition or genetic knockdown of BRD9 attenuated ZD-1-186-mediated MYC suppression and blunted p21 induction. Transcriptomic profiling of ZD-1-186 showed simultaneous derepression of BCL6-repressive loci and suppression of MYC transcriptional programs. These findings demonstrated that ZD-1-186 acted as a transcriptional rewiring glue, recruiting BRD9 to BCL6-repressive loci to activate tumor-suppressive transcription, while also potentially redirecting BCL6 to BRD9-bound oncogenic loci. Overall, our work provides a blueprint for the rational discovery and design of electrophile-enabled, nondegradative molecular glues for targeted transcriptional rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e1cf0470c87b88a6512470358d48dec81cbfa4" target='_blank'>
              Discovery of Non-Degradative Covalent Molecular Glues for Transcriptional Reprogramming
              </a>
            </td>
          <td>
            Tuong Nghi Duong, Edward Pandji, Qian Shao, Daniel K. Nomura
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Antibiotic resistance is a widespread problem that reduces the effectiveness of antibiotics, leading the rise of multidrug resistance (MDR). Efflux pump, biofilm formation and plasmid based gene transfer together form the basis of co-adaptive mechanisms providing fleeting tolerance resistant overtime. Genes encoding efflux systems are alternative efflux-pumping systems that remove some antimicrobial agents, so that over extremely low intracellular concentrations are required for the bacterium to survive in the biofilm matrix.These, in turn, contribute to the structural and functional coherence of biofilms, which serve as protected niches favoring horizontal gene transfer and the propagation of plasmid-mediated resistance genes. In this review, we combine recent data from molecular and omics-based studies to demonstrate how these systems are co-regulated to control resistance in bacteria, with an emphasis on cross-talk between regulatory pathways including quorum-sensing, stress-response signalling and global transcriptional regulators. Insights into this interplay can pave the way for innovative therapeutic strategies, such as efflux pump inhibitors, biofilm dispersal agents and plasmid-curing strategies. Comprehensive knowledge of these interrelated processes may influence the development of future antimicrobial agents and their use, and may reduce the emergence of pan-resistant species of bacterial pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18f46ff4958d5baeef3aee465cd5d3e79a160508" target='_blank'>
              An integrated view of bacterial resistance in the antibiotic era: efflux pumps biofilms and plasmids
              </a>
            </td>
          <td>
            Zahraa Mohammed
          </td>
          <td>2025-12-31</td>
          <td>Journal of Biomedicine and Biochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Approximately 10% of cancers achieve replicative immortality through a telomerase-independent mechanism of telomere maintenance, termed Alternative Lengthening of Telomeres (ALT). ALT is particularly prevalent in certain subtypes of malignant gliomas, such as IDH-mutant astrocytoma and pediatric glioblastoma, and frequently co-occurs with ATRX inactivating mutations. Although ALT is an adaptive mechanism through which cancer cells achieve proliferative immortality, the elevated levels of replication stress observed in ALT tumors constitute a potential therapeutic vulnerability.


METHODS
Leveraging CRISPR/Cas9 screening data from the Cancer Dependency Mapping Project, coupled with patient-derived cell lines and xenografts, we identified SMARCAL1 as a novel synthetic lethal vulnerability in ATRX-deficient glioma models that engage ALT. Using complementary molecular assays for DNA damage, telomere maintenance, and telomeric replication stress, we define the mechanisms underlying cytotoxicity induced by SMARCAL1 depletion in ALT-positive glioma cells.


RESULTS
Our data demonstrate the annealing helicase SMARCAL1 is a highly specific synthetical lethal vulnerability in cancers that use ALT. SMARCAL1 localizes to ALT-associated PML bodies in ALT-positive glioma cell lines, including IDH-mutant astrocytomas. SMARCAL1 depletion, via doxycycline-induced RNAi, led to a hyperactivation of the ALT phenotype, high levels of DNA double-strand breaks in G2 phase, and cell death via mitotic catastrophe. In mice bearing intracranial xenografts derived from high-grade IDH-mutant astrocytoma, inducible SMARCAL1 depletion prolonged animal survival.


CONCLUSIONS
Our findings demonstrate that the molecular processes orchestrating ALT-mediated telomere maintenance constitute a targetable synthetic lethal vulnerability that can be exploited by SMARCAL1 inhibition, thus supporting the future development of small molecule inhibitors of SMARCAL1 as anti-cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c97a1417417751706742470fa54409032d4aaa86" target='_blank'>
              SMARCAL1 is a targetable synthetic lethal therapeutic vulnerability in ATRX-deficient gliomas that use Alternative Lengthening of Telomeres.
              </a>
            </td>
          <td>
            Alexandrea Brown, Laura Strickland, Elise N Erman, Christopher J. Pirozzi, Justin T. Low, B. Diplas, Emiley A Gibson, Mariah Shobande, Taher Khambati, Marharyta Krylova, Heng Liu, R. McLendon, Zachary J. Reitman, S. Keir, Lee Zou, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2026-01-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Epitranscriptomic regulation, particularly N6-methyladenosine (m ⁶ A) RNA methylation, has emerged as a pivotal layer of gene expression control, complementing traditional epigenetic mechanisms. As the most prevalent internal modification in eukaryotic messenger RNA, m ⁶ A dynamically modulates RNA splicing, stability, translation, and localization through the coordinated actions of writer, eraser, and reader proteins. Recent studies have illuminated the critical role of m ⁶ A in kidney disease and renal fibrosis. Dysregulation of m ⁶ A machinery has been shown to influence key pathogenic pathways in kidney disease development. Moreover, m ⁶ A-modified non-coding RNAs further integrate epitranscriptomic signals into fibrogenic and immune regulatory networks. Therapeutically, targeting m ⁶ A pathways offers promising avenues, with small-molecule inhibitors, RNA-based therapeutics, and clustered regularly interspaced short palindromic repeats (CRISPR)-based m ⁶ A editing technologies under active investigation. However, challenges remain in achieving tissue-specific modulation and avoiding systemic toxicity. This review synthesizes current knowledge on m ⁶ A-mediated regulation in kidney disease, highlighting mechanistic insights, disease-specific alterations, and emerging therapeutic strategies. As the field of epitranscriptomics continues to evolve, m ⁶ A represents a compelling frontier for understanding renal pathophysiology and developing precision therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7572e2ad51a46ae61a256c5f951a93b67ca897db" target='_blank'>
              Epitranscriptomics in kidney disease: insights and therapeutic potential
              </a>
            </td>
          <td>
            Yu-Cheng Tsai, Ming-Chang Tsai, Shuo-Ming Ou, Chih-Ching Lin, Szu-yuan Li
          </td>
          <td>2025-12-12</td>
          <td>Kidney Research and Clinical Practice</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c2666cb6814b50c610dbbea4b7e0fe4f5eecb71" target='_blank'>
              Rapamycin Differentially Impacts Germline Stem Cell Quiescence Across Diverse Genetic Backgrounds of Drosophila Melanogaster
              </a>
            </td>
          <td>
            Sahiti Peddibhotla, Miriam Gonzaga, Tricia Zhang, Yasha Goel, Jun Sun, Benjamin R. Harrison, Daniel E. L. Promislow, H. Ruohola-Baker
          </td>
          <td>2025-12-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background/Objectives To address the unelucidated mechanisms of breakpoint formation in TFE3-rearranged renal cell carcinoma (TFE3-rRCC), this study characterizes breakpoint distribution within the TFE3 gene. We further explore how non-canonical DNA structures and their thermodynamic stability fluctuation may act as predisposing factors for the genomic instability driving these characteristic translocations. Methods TFE3 breakpoints were identified in a cohort of 31 TFE3-rRCC tumor samples. The chi-square test was used to assess the statistical significance of breakpoint clustering. To investigate potential structural determinants, we predicted the distribution of G-quadruplex-forming sequences and palindromic motifs. Moving beyond simple motif density, we calculated the local Gibbs free energy changes (ΔG) associated with DNA secondary structures using Mfold and RNAfold to model thermodynamic stability across the TFE3 gene. This thermodynamic stability fluctuation was quantified as the maximum absolute local change in folding free energy (|dΔG|). Finally, this correlation between thermodynamic stability fluctuation and breakpoint location was validated by analyzing the 13 most frequently rearranged genes reported in the COSMIC database. Results A significant breakpoint cluster was identified within intron 5 of TFE3, containing 23 of 31 breakpoints (74.19%; chi-square test, P < 0.05). While the simple density of G-quadruplex or palindromic motifs did not directly correlate with breakpoint locations, a strong association with local thermodynamic stability fluctuation was observed. The region within intron 5 exhibited the highest thermodynamic stability fluctuation. This result suggests that regions of high thermodynamic stability fluctuation are correlated with increased susceptibility to DNA breakage. This finding was corroborated in the COSMIC dataset, where breakpoints in 12 of the 13 most frequently rearranged genes were similarly located near peaks of high |dΔG|. Conclusion Our findings indicate that breakpoint events in TFE3-rRCC are non-randomly clustered within intron 5. This clustering correlates strongly with regions characterized by high thermodynamic stability fluctuation (|dΔG|) of potential non-canonical DNA secondary structures. The principle that elevated local thermodynamic stability fluctuation is a feature of breakpoint locations was supported by analysis of a broader set of oncogenes, suggesting that high local thermodynamic stability fluctuation is a common feature of translocation-prone regions in cancer, representing a plausible, though not proven, contributor to genomic fragility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba18c1a28435183414957442b6594dd9961a8358" target='_blank'>
              Mechanistic study of TFE3 breakage in TFE3-rearranged renal cell carcinoma: the perspective of non-canonical DNA structures and their stability
              </a>
            </td>
          <td>
            Xiaopo Zhang, Qinshuang Dang, Xinghe Pan, Zhenggen Deng, Yanhao Xu, W. Gan, Hongqian Guo
          </td>
          <td>2025-11-25</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Conventional cancer therapies, including radiation therapy and chemotherapy, rely on inflicting DNA damage, yet they inevitably affect normal cells, leading to severe adverse effects. The advent of precision chemotherapy exploiting tumor-specific DNA repair defects has validated the effectiveness of this approach. The first successful example is PARP inhibitors, which selectively kill homologous recombination (HR) defective cancers, such as familial breast cancer possessing HR deficiency due to BRCA gene mutations. However, the broader landscape of DNA maintenance-including DNA replication, repair, and checkpoint pathways-harbors numerous mutations in tumors that remain untargeted. Here, we propose repurposing chain-terminating nucleoside analogs (CTNAs) to target such cancers' vulnerabilities. CTNAs, long utilized as anti-cancers and anti-viral drugs, inhibit replication and thereby suppress growth, but their activity has never been systematically aligned with specific cancer mutations associated with DNA maintenance defects. Based on our recent studies, we demonstrate that CTNAs elicit synthetic lethality in cells deficient for distinct DNA maintenance systems, amplifying replication stress, leading to cell death. We highlight the spectrum of CTNA-induced lesions and repair pathways required for cellular tolerance. This framework presents a versatile "repair-defect-guided" chemotherapy that expands the clinical utility of CTNAs and improves therapeutic effect by reducing side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e63be07db0a569b7816efc4ad2f977abe094978" target='_blank'>
              Targeting Genome Maintenance Defects of Cancers Using Chain-Terminating Nucleoside Analogs.
              </a>
            </td>
          <td>
            Ryotaro Kawasumi, Rubaiat E Tabassum, Kouji Hirota
          </td>
          <td>2025-12-10</td>
          <td>Cancer science</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bf0d027edeb1c0ea3dba2055e8f587e64fbdfe5" target='_blank'>
              Antiplasmid systems: a novel strategy to combat antibiotic resistance.
              </a>
            </td>
          <td>
            Mohd Asim, Syed Ahmed Rizvi, Qazi Mohd Rizwanul Haq
          </td>
          <td>2025-12-19</td>
          <td>The Journal of antimicrobial chemotherapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 849


 Background:
 Expression of the L-type amino acid transporter 1 (LAT1; gene symbol:
 SLC7A5
 ) is significantly enriched across a wide variety of cancers, and has been associated with cancer progression and poor prognosis. Yet the relationship between LAT1 and known prognostic genomic alterations in genes such as
 TP53
 ,
 KRAS
 , and
 BRAF
 remains understudied. We evaluated
 SLC7A5
 expression within gastrointestinal (GI) tumors in the cBioPortal database and characterized its correlation to known genomic/transcriptomic markers of cancer prognosis.
 Methods:
 We used 30,993 samples to analyze expression of
 SLC7A5
 expression z-scores relative to diploid samples (RNA Seq V2 RSEM) across malignancies, excluding overlapping samples, from 88 cancer studies within cBioPortal. Sub-analysis were performed in hepatobiliary (HPB), colorectal (CRC), esophagogastric (EGC), and pancreatic cancers (PDAC), along with a combination of these cancer types (pan-GI-cancer). mRNA expression z-scores relative to diploid samples were stratified by
 SLC7A5
 expression quartiles for each sub-analysis. The log2 ratio (L2R) alteration frequency of genomic/transcriptomic markers between quartiles was evaluated within the pan-GI-cancer cohort and each subgroup using a one-way ANOVA test with FDR correction.
 Results:
 SLC7A5
 enrichment was significantly associated with worsened overall survival across all cancers (Hazard Ratio (HR) 1.361 95% Confidence Interval (CI) 1.282 - 1.445; p < 0.001). In the pan-GI-cancer cohort, tumors with high
 SLC7A5
 expression were enriched for genomic alterations in
 TP53
 (L2R 0.38),
 APC
 (L2R 0.98) and
 KRAS
 (L2R 0.61; q < 0.001 for all). In particular,
 KRAS
 alterations were enriched in CRCs and HPBs with high
 SLC7A5
 (L2R 0.38 and 3.05; q < 0.001), while
 TP53
 alterations were enriched in EGCs and HPBs (L2R 0.33 and 0.81; q < 0.001). Within
 KRAS
 mutant GI tumors,
 SLC7A5
 -high expression status trended towards worsened disease free survival (HR 1.143 95% CI 0.745 - 1.753; p = 0.527). Conversely, alterations in
 CDH1
 and
 SMAD4
 were significantly depleted in
 SLC7A5
 -high EGCs (L2R -2.69; q < 0.001) and PDACs (L2R -0.96, q < 0.001), respectively. Neither
 ERBB2
 (HER2) nor
 CD274
 (PD-L1) expression was correlated with
 SLC7A5
 expression status in any GI cancer.
 Conclusions:
 SLC7A5
 expression levels are correlated with mutations in poor prognostic genomic markers, such as
 TP53
 ,
 KRAS
 , and
 APC
 , in GI malignancies. For
 KRAS
 ,
 SLC7A5
 -high status appears to be an independent poor prognostic indicator, though larger data sets are necessary to confirm these findings. Future prospective analyses may provide further insight into the mechanisms by which LAT1 expression impacts GI cancer outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d59d2d758c259eaac9b38950626817b2ccb7196" target='_blank'>
              Characterization of LAT1-related gene expression and its impact on prognostic tumor mutations and gastrointestinal (GI) cancer clinical outcomes.
              </a>
            </td>
          <td>
            Ajay Prakash, Kelly Pum, Paolo Goffredo, Travis Gates, Justin H. Hwang, Ella Boytim, Emil Lou, Subbaya Subramanian
          </td>
          <td>2026-01-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background/Objectives: Current genomics research equates the genome with DNA sequence and treats the epigenome as a regulatory layer. This DNA-centric view obscures the fact that genomic identity arises through epigenomic processes. The objective of this article is to reinterpret published findings into a new theoretical framework: the EpG2 (Epigenome–Genome) system. Methods: This work develops a new conceptual framework by integrating published evidence from diverse domains—including enhancer biology, overlapping genomic functions, alternative coding frames, zygotic genome activation, and disease-associated loci—and reinterpreting these findings through the lens of epigenomic processes. Results: Evidence shows that enhancers emerge only through the interplay of sequence, transcription factors, and chromatin environment. At fertilization, paternal and maternal genomes remain separate, and a new genome emerges through coordinated epigenomic reprogramming or zygote genome emergence (ZGE). DNA sequence risk variants illustrate the concept of contextual risk alleles, whose effects shift across tissues and developmental stages as epigenomic contexts change. Conclusions: The EpG2 system reframes the genome as a processual, emergent entity generated and regulated by epigenomic processes, offering a paradigm for understanding genomic variation beyond DNA sequence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2af39b725984a653ec4ca7bda8e349ac7f79a7de" target='_blank'>
              Introducing the EpG2 System: Epigenomic Processes and the Emergent Genome
              </a>
            </td>
          <td>
            E. Ruiz-Narváez
          </td>
          <td>2025-12-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) driven by KMT2A rearrangements (KMT2A-r) is an aggressive hematologic malignancy with poor prognosis and a high incidence in infants. While KMT2A fusion proteins drive leukemogenesis through transcriptional dysregulation, recent discoveries have highlighted the pivotal role of non-coding RNAs (ncRNAs) in shaping the molecular and epigenetic landscape of this disease. These key regulators of gene expression influence chromatin dynamics, transcriptional activation, and post-transcriptional control. Circular RNAs (circRNAs) contribute to genome instability and facilitate chromosomal translocations, while some fusion-derived circRNAs (f-circRNAs) sustain oncogenic signaling and promote chemoresistance. Long non-coding RNAs (lncRNAs) orchestrate transcriptional programs that maintain leukemic stem cell properties and reinforce aberrant self-renewal pathways. MicroRNAs (miRNAs) modulate critical oncogenic networks by regulating KMT2A fusion transcripts and downstream effectors, thereby impacting drug resistance, apoptosis, and proliferation. Meanwhile, enhancer RNAs (eRNAs) fine-tune transcriptional activity and epigenetic regulation, influencing KMT2A target gene expression and chromatin accessibility. Collectively, these ncRNAs integrate into the complex regulatory circuits of KMT2A-r ALL, revealing their potential as biomarkers for disease classification, risk stratification, and treatment response prediction. Understanding their interplay with KMT2A fusion proteins not only provides new insights into leukemogenesis but also highlights promising opportunities for therapeutic intervention and precision medicine in this high-risk leukemia subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b46cc5750ec443e7b12863d8785ed1d738e7da" target='_blank'>
              The hidden regulators: Non-coding RNAs in KMT2A-rearranged acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Maria Augusta Poersch, Ana Carolina Rodrigues, Priscila Elias Ferreira Stricker, Alexandre Luiz Korte Azevedo, D. Bruschi, J. D. de Oliveira
          </td>
          <td>2025-12-08</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/131d5075adfc8a892d5a30b581d5ddea0c0afd87" target='_blank'>
              Single-cell transcriptomic analysis reveals cellular heterogeneity and prognostic subtypes in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhen Han, Yuan Gao, Yuxing Zhao, Jiahong Wu, Jia Chen, Jinling He, Hong Bai, Liang Qiao, Yali Ren, Lei Sun
          </td>
          <td>2025-12-22</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd29b86c5215e5ff6e7ae57bd7a699692db0273" target='_blank'>
              Tissue-specific differences of gene expression variance in mutation accumulation lines of mice.
              </a>
            </td>
          <td>
            Eugenio López-Cortegano, Jobran Chebib, Anika Jonas, Sven Künzel, Peter D Keightley, D. Tautz
          </td>
          <td>2025-12-27</td>
          <td>Heredity</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="
 CD8
 +
 T cells are central to effective antitumor immunity, yet in cancer, they often undergo progressive transcriptional, epigenetic, and metabolic reprogramming that leads to an exhausted state and limits current immunotherapy. A deeper understanding of the molecular mechanisms that govern CD8
 +
 T cell differentiation and function within the tumor microenvironment is essential to overcome this barrier. This review outlines the current knowledge of the transcriptional and epigenetic programs that shape T cell heterogeneity in cancer and chronic infection, with a focus on the formation and maintenance of exhausted T cell subsets. We highlight how T cell–intrinsic factors such as transcription factors and chromatin regulators and extrinsic factors such as nutrient availability converge to influence T cell fate decisions and function, as well as how these are affected in cancer. Finally, we discuss emerging therapeutic strategies aimed at reprogramming the epigenome to restore T cell function, offering new avenues to enhance the efficacy and durability of cancer immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f87bd08b9b6bacb32db98fab54d14e8c824e903" target='_blank'>
              Transcriptional and Epigenetic Regulation of T Cell States in Cancer
              </a>
            </td>
          <td>
            Shixin Ma, Diana C. Hargreaves, S. Kaech
          </td>
          <td>2026-01-16</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction. Bacteria are capable of integrating a portion of the genetic material of an invading infectious agent as a spacer into the CRISPR array. This process is called CRISPR adaptation and is necessary for bacteria to adequately respond to the same infectious agent in the future. The molecular mechanisms of CRISPR adaptation remain unclear, particularly the role of extra-CRISPR-Cas components in bacteria during prespacer generation and spacer acquisition. In this study, we developed an approach to identifying nucleases responsible for the formation of prespacer 3’-ends in vivo. Materials and methods. We generated a genetic construct based on the pBR322 plasmid vector with gRNAs targeting the genes encoding cellular exonucleases with 3’-5’ activity in E. coli. This genetic construct and the pdCas9 plasmid were transformed into KD403 cells. The level of primed CRISPR adaptation was assessed using PCR and sequencing. The structure of the 3' ends of the prespacers was assessed using the FragSeq short-fragment sequencing method using the Accel-NGS® 1S Plus DNA Library Kit, Swift biosciences (xGen™ ssDNA & Low-Input DNA Library Preparation Kit, IDT). Results. In the KD403 ΔrecB strain with suppressed nucleases, a reduction in primed adaptation was detected, but sequencing revealed no disruption of the 3’ ends of the spacer precursors. Conclusion. Our results contribute to the understanding of the molecular mechanisms of CRISPR adaptation, which may aid in the development of CRISPR-based approaches in medicine and agriculture in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c67f3795fd0e777318561920146c27024f0382d" target='_blank'>
              Studying the mechanisms of prespacer 3’-end formation during CRISPR adaptation via simultaneous gene repression using dCas9
              </a>
            </td>
          <td>
            I.A. Balazs, O. Musharova, Yunna Petrusenko, S. Vinogradova, K. Severinov
          </td>
          <td>2025-12-08</td>
          <td>Molecular Genetics, Microbiology and Virology (Russian)</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, R. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 A crucial adaptability trait of melanoma cells that persist under all selection pressures, including therapies, is their ability to efficiently switch from proliferation to quiescence. With the goal of improving therapeutic strategies for preventing melanoma recurrence, herein we describe an approach to modeling this trait in cell culture. Body imposes a variety of selection pressures and bottlenecks ranging from different components of immunity to metabolic challenges to enforce quiescence and influence cancer evolution. The rationale for the choice of model as cell culture is that it is ideal for modeling the realistic depth of quiescence and for evaluating therapeutic strategies that may inhibit highly abnormal but adaptable cancer cells. As it is not possible to replicate all such bottlenecks in cell culture, our approach is to apply as severe a bottleneck as possible, thus eliminating all proliferative (non-adaptable) cells, leaving behind rare cells surviving in deep quiescence. We hypothesize that if the quiescent cell state being modeled can survive a severe bottleneck, it can also survive other bottlenecks of similar or lower severity. A lack of glutamine proved to be a strong and prolonged selection pressure for the highly metastatic human melanoma cell line A375SM, killing more than 99% of cells and selecting rare cells based on their ability to survive in deep quiescence. After 4 weeks, cells gradually exited quiescence and proliferated indefinitely. Furthermore, we modeled even deeper quiescence lasting longer than 4 weeks by increasing the severity of the selection pressure using dialyzed serum in medium. The cells selected in this manner were much more resistant to paclitaxel than was their parental cell line. We obtained similar results with the highly metastatic mouse melanoma cell line B16-BL6. We conclude that our approach is suitable for modeling abnormal deep quiescence in melanoma that drives the disease, including recurrence and metastasis. The main strength of the approach is that, besides modeling genetic alterations, it incorporates suitable selection pressures and the time element that are important in cancer evolution. Although there is a limitation of lack of body-like microenvironments in cell culture, we believe that abnormal quiescent cancer cell state being modeled here is capable of overcoming restraints imposed by the body. We will also discuss our published and new observations with rare cells (0.01% in population) surviving in quiescence for several weeks in aggressive triple-negative breast cancer cell lines and their functional characteristics. Our results obtained with adaptable cells from SUM149 TNBC cell line, modeled as above for melanoma cells, suggest that they are stem-like quiescent cells. We will also discuss how we choose and evaluate therapies that would likely inhibit quiescent and slow-growing cancer cells prior to recurrence. This approach may significantly improve how we investigate deep intrinsic resistance and strategies to overcome it.



 Balraj Singh, Vanessa N. Sarli, Nikil Erry, Anthony Lucci. A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4de6eb92fbb6a26e62be45c5389b5a53529d5" target='_blank'>
              Abstract A010: A cell culture model of quiescent melanoma cell state that governs cancer evolution, therapy resistance, and recurrence/metastasis
              </a>
            </td>
          <td>
            Balraj Singh, V. Sarli, Nikil Erry, Anthony Lucci
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that influences the regulation of gene expression, genome stability, and plant adaptation to the environment. In plants, the most common forms of methylation occur in cytosines (5mC) and, more recently, also in adenines (6mA), both playing specific and complementary roles in the control of genes, transposable elements, and plant development. The mechanisms that control the addition, removal, and reading of these marks include methyltransferase proteins, glycosylases, and methylation recognition domains, which interact with transcriptional factors and histone modifiers. Methylation is involved in various biological processes such as cell differentiation, flowering, embryogenesis, seed dormancy, and responses to biotic and abiotic stresses. The discovery of heritable epialleles and their application in plant breeding opens new possibilities for selection based on epigenetic traits. Epigenetic editing with CRISPR/Cas9 has emerged as a tool for inducing targeted modifications without altering the DNA sequence, offering alternatives for non-transgenic plant biotechnology. Furthermore, methylation contributes to the phenomenon of heterosis, revealing a functional role in hybrid superiority. However, challenges persist regarding the stability of these marks and the legal regulation of epigenetically edited cultivars. This work reviews recent advances, highlighting the integration of plant epigenetics with genetic, precision editing, and adaptive responses in cultivated plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0115e8ca9083e3fba7bd10aaa44af2c0bf0d7bc" target='_blank'>
              DNA methylation: regulation of gene expression and applications in plant breeding
              </a>
            </td>
          <td>
            Helio Souza Junior, Adriana Gonela
          </td>
          <td>2025-12-03</td>
          <td>Agronomy Science and Biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cure rates for childhood malignancies using established therapy protocols have increased to an average of 80% but have reached a plateau. Moreover, survival rates are particularly low for some pediatric tumors-such as high-risk group 3 medulloblastomas, osteosarcomas, Ewing sarcomas, high-risk neuroblastomas, and high-grade gliomas-and dismal for patients with relapsed malignancies. A functional drug response profiling platform for pediatric solid and brain tumors has been established within the INFORM program to identify patient-specific vulnerabilities and biomarkers and to unravel molecular mechanisms associated with drug response profiles for clinical translation. In this study, we performed a multiomics analysis using drug sensitivity profiles, as well as genomic and transcriptomic data, of 81 pediatric solid tumor samples. The integrative analysis suggested two multiomics signatures associated with drug sensitivity. One signature distinguished neuroblastoma samples with sensitivity to navitoclax, a BCL2 family inhibitor. A second signature was specific to a subset of Wilms tumors harboring the SIX1 (Q177R) hotspot mutation that displayed high expression of MGAM, PTPN14, STAT4, and KDM2B and high sensitivity to MEK inhibitors. A patient-specific causal interaction network analysis suggested possible molecular interactions between MEK inhibitors and the SIX1 mutation in Wilms tumor samples. In conclusion, the integration of drug sensitivity profiling and multiomics data revealed potential biomarkers that may be associated with drug sensitivity in pediatric solid tumors. Patient-specific causal interaction network analysis further elucidated the interaction between inhibitors and signature biomarkers, providing insights that may inform clinical translation.


SIGNIFICANCE
The combination of multiomics analysis and drug sensitivity profiling identified two signatures related to drug sensitivity in pediatric solid tumors, contributing to the advancement of functional precision medicine and personalized treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI .">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea339e646dcb66c61feed8e7a43ccdda96232499" target='_blank'>
              Integrative Multiomics and Drug Sensitivity Profiling Reveal Potential Biomarkers and Therapeutic Strategies in Pediatric Solid Tumors.
              </a>
            </td>
          <td>
            Dina ElHarouni, Rosa Hernansaiz-Ballesteros, H. Peterziel, G. Balasubramanian, C. Previti, Kathrin Schramm, Mirjam Blattner-Johnson, Rolf Kabbe, Barbara C Jones, S. Oppermann, David T. W. Jones, S. Pfister, Olaf Witt, Julio Saez-Rodríguez, Ina Oehme, N. Jäger, Matthias Schlesner
          </td>
          <td>2025-12-19</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b7de6d8aeceaad9c8d57eb1d9c3d308c5a80e2" target='_blank'>
              Ribosome biogenesis as a potential therapeutic target in KRAS mutant colorectal cancer.
              </a>
            </td>
          <td>
            Yui Tanaka, M. Sakahara, H. Yamanaka, Yasuko Natsume, Daisuke Kusama, Kohei Kumegawa, Harunori Yoshikawa, Yuich Abe, K. Okabayashi, Shimpei Matui, Y. Kitagawa, Naohiko Koshikawa, H. Osumi, E. Shinozaki, Satoshi Nagayama, Jun Adachi, R. Maruyama, Ryoji Yao
          </td>
          <td>2025-12-27</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd2702367051da70ddb6fa2941183d106513bc" target='_blank'>
              The expanding roles of homologous recombination proteins in genome stability.
              </a>
            </td>
          <td>
            Lorenzo Sassi, Andrea Martinez Marroquin, Salli Waked, Alessandra Ardizzoia, Vincenzo Costanzo
          </td>
          <td>2026-01-03</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Normal somatic cells possess innate defense mechanisms that confer resistance to cancer-driving mutations, including cellular senescence activation upon oncogene overexpression. However, oncogenic mutations are detected in normal somatic tissues, suggesting that some cells are able to escape these defense mechanisms. We investigated whether all cells become senescent upon overexpressing oncogenic KRAS G12D in primary human keratinocytes and found that while most cells entered a non-dividing state consistent with oncogene-induced senescence, 10-30% escaped these defenses and continued dividing. Cellular barcoding revealed that sister cells typically shared the same fate upon oncogene overexpression, suggesting heritable, non-genetic differences determine a cell’s ability to escape senescence programs. Using single-cell RNA sequencing prior to oncogene overexpression, we identified transcriptional states associated with cells that ultimately escape senescence, thus marking a subpopulation of cells most susceptible to cancer-driving mutations. We also observed transcriptional variation amongst escaper cells that may interact with subsequent mutational events. Our work demonstrates that a cell's initial transcriptional state influences its response to cancer-driving mutations and suggests that non-genetic variation interacts with genetic mutations to drive somatic evolution and cancer progression, challenging the genetics-only paradigm of cancer initiation.



 Grant Kinsler, Arjun Raj. Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Cancer Evolution: The Dynamics of Progression and Persistence; 2025 Dec 4-6; Albuquerque, NM. Philadelphia (PA): AACR; Cancer Res 2025;85(23_Suppl):Abstract nr A001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95adbe47a20bb5a9feae6f4d099365519435f00f" target='_blank'>
              Abstract A001: Pre-existing non-genetic cellular states determine susceptibility to oncogenic transformation
              </a>
            </td>
          <td>
            Grant Kinsler, Arjun Raj
          </td>
          <td>2025-12-04</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 TFE3 rearranged renal cell carcinoma (TFE3-tRCC) is a rare subtype of renal cell carcinoma (RCC) harboring gene fusions that trigger oncogenic activation of TFE3 transcription factor. The frequency of this rare tumor is surprisingly high in children and young adults (20% to 75% of childhood RCCs) compared to older adults (ranges between 1% and 5%). Importantly, TFE3-tRCC can show aggressive behavior with rapid progression and little to no response to drugs commonly used for renal cancer treatment. In this project we aim to define gene expression programs of distinct TFE3 fusions to uncover molecular events driven by chimeric TFE3 and delineate their consequences for important cancer phenotypes. By identifying fusion-specific pathway alterations, we seek to propose novel therapeutic strategies tailored to the patients with these fusions. Using a cohort of 31 patients with suspected diagnosis of TFE3-tRCC, we have employed a novel customized RNA-based targeted Next generation Sequencing (NGS) panel to determine putative fusion partners of TFE3. Next, high throughput technologies like short read and long read RNA sequencing and ChIP sequencing were used to characterize the molecular landscape downstream of different fusion events. Finally, to explore the influence of the transcriptional diversity on therapeutic decisions, we aim to perform molecularly informed drug screening in our in vitro models followed by in vivo validation. With our newly developed targeted NGS panel, we have characterized our patient cohort and identified 7 distinct TFE3 fusion partners in 25 cases out of 31. The putative partners are ASPSCR1, LUC7L3, MED15, NonO, PRCC, RBM10, and SFPQ. Certain partners confer unique tumor morphologies, e.g., MED15::TFE3 fusion associates with cystic features. Our transcriptomic analysis shows tumors with same fusion tend to cluster together, driven by shared pathway upregulation (OXPHOS, lysosomal pathway, autophagy, proteasome etc.). Differential upregulation of pathways such as immune response, metabolic pathways, mitophagy, RNA splicing etc. highlight fusion specific identity, informing fusion driven molecular characteristics. Preliminary in vitro chromatin accessibility data suggests the potential involvement of the chimeric TFE3 protein in alternative splicing regulation, which is further consistent with splicing enrichment observed in our transcriptomic analysis. Different fusion partners for TFE3 likely impact the dysregulation of the chimeric protein and drives tumorigenesis. However, our limited understanding of the molecular landscape and clinical consequences of the different TFE3 fusions presents as the major obstacle towards development of targeted therapeutic strategies for this disease. In this scenario of unmet clinical needs, we seek to define the molecular features of TFE3-tRCC, thereby uncovering specific vulnerabilities of tumor cells that prompt the development of new targeted therapeutic strategies.



 Debleena Basu, Hella Bolck, Dorothea Rutishauser, Peter Leary, Daniela C. Garcia (s), Abdullah Kahraman, Niels Rupp, Chantal Pauli, Holger Moch. Development of a mechanistic understanding and novel therapeutic strategies for TFE3-rearranged renal cell carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2de5c3be38d1f64bf43ef4ec1ea40cff4dcd004" target='_blank'>
              Abstract A022: Development of a mechanistic understanding and novel therapeutic strategies for TFE3-rearranged renal cell carcinoma
              </a>
            </td>
          <td>
            Debleena Basu, H. Bolck, D. Rutishauser, Peter Leary, Daniela C. Garcia (s), Abdullah Kahraman, Niels J. Rupp, Chantal Pauli, Holger Moch
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA double-strand breaks (DSB) represent one of the most severe forms of genomic damage. Thus, cells have evolved a complex network of DSB repair pathways, including homologous recombination, classical and alternative end joining, and single-strand annealing, which are tightly regulated by genetic and epigenetic factors. The selection and efficiency of these pathways influence genome integrity, oncogenesis, and therapeutic response. This comprehensive review synthesizes recent findings on the genetic regulation of DSB repair, with emphasis on pathway-specific regulators, chromatin context, and post-translational modifications. Moreover, this review integrates primary research from mammalian systems, including CRISPR-based studies, proteomics, and imaging, with a focus on publications from 2020 to 2025. We discuss the role of key players, such as MRE11-RAD50-NBS1 (MRN), ataxia telangiectasia mutated (ATM), mediator tumor suppressor p53-binding protein 1 (53BP1), breast cancer type 1 susceptibility protein (BRCA1), anti-silencing function 1 (ASF1), ring finger protein (RNF)8/168, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and RAD51 recombinase (RAD51), in orchestrating the associated pathway choice. Epigenetic modifications, RNA-mediated mechanisms, and chromatin remodeling dynamically influence the efficiency and fidelity of repair. Particular attention is provided to emerging regulators, including thyroid hormone receptor interactor 13 (TRIP13), ubiquitin-like with plant homeodomain (PHD) and RING finger domains 1 (UHRF1), Shieldin, and polymerase theta. This review highlights novel insights into transcription-associated DSB repair, the interplay of replication stress with repair pathway engagement, and context-dependent synthetic lethality. We also examine implications for cancer biology, including therapy resistance and biomarker development. Ultimately, understanding the genetic regulation of DSB repair pathways can provide critical insights into genome stability maintenance and reveal new therapeutic opportunities in cancer. Future work should focus on pathway crosstalk, phase-specific regulation, and integrating repair modulation into personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940ba623f1d9b48456fecd2cfe27231108b1711a" target='_blank'>
              Genetic Regulation of DNA Double-Strand Breaks and Repair Pathways.
              </a>
            </td>
          <td>
            Lucián Zastko
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Urothelial carcinoma of the bladder (UBC) is a heterogeneous disease from the clinicopathologic and molecular standpoint and presents clinical management problems for its range in progression, frequent recurrence and complicated genetic/epigenetic architecture. Not only traditional genetic changes, but also epigenetic dysregulation (e.g., alterations of chromatin remodeling, histone modification and DNA methylation) has been increasingly recognized to play a pivotal role in the process of tumor initiation, progression, metastasis and resistance to therapy. It is a dissapointing result that mutations of the chromatin remodeler such as ARID1A, KMT2C, KMT2D, and histone modifying enzymes (EP300, CREBBP and EZH2) not only cause loss-of-nucleosome positioning in addition lead to lack of enhancer function and perturbation ofT program contributing to shaping up intratumoral heterogeneity as well as transcriptional plasticity resulting in failure in immunoevasion. These changes sculpt the tumor microenvironment and impact the response to chemotherapy, immunotherapy, and targeted therapy.Epigenetic biomarkers based on mutation profiles, chromatin accessibility and DNA methylation signatures present new non-invasive strategies that can be followed for early detection, monitoring of disease and stratification of patients usage; which are all available in urine or plasma. Therapeutically, inhibitors of EZH2, HDACs, BET proteins, DNMTs and certain chromatin remodeling factors are currently being heavily investigated in preclinical studies or clinical trials as monotherapies or rational combinations with ICIs, chemotherapy or targeted therapies.Advances in tech nologies, such as single- and spatial genomics combined with AI-facilitated multi-omics integration make it possible to realize high-resolution maps of tumor heterogeneity, enhancer d namics, and epigenetic vulnerabilities. This review aims to present the current data on UBC epigenetic modifications and their mechanistic, biomarker and therapeutic implications. By combining mechanistic understanding with translational perspectives, we outline future directions to move precision oncology forward, optimize our treatment approach and improve clinical outcomes for UBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c58b19d1ac81843818571ae3d302c08f8f24348" target='_blank'>
              Epigenetic Alterations and Chromatin Landscape Remodeling in Urothelial Bladder Cancer: Molecular Drivers, Biomarker Potential, and Emerging Therapeutic Strategies
              </a>
            </td>
          <td>
            Amr Abd-Elraheem Abdo, Ahmed Mohamed Kamel
          </td>
          <td>2025-12-10</td>
          <td>Annals of urologic oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The phenomenon of transgene silencing was first observed shortly after the generation of the initial transgenic plants. The vast body of experimental data accumulated since then constitutes an invaluable resource for dissecting the mechanisms of epigenetic gene regulation. Silencing operates at either the transcriptional (TGS) or post-transcriptional (PTGS) level and is predominantly mediated by small interfering RNAs (siRNAs). Although these two epigenetic pathways involve distinct sets of proteins and enzymes, they share fundamental mechanistic features: the generation of double-stranded RNA (dsRNA), its processing into siRNAs by DICER-LIKE (DCL) enzymes, and the assembly of an Argonaute-centered effector ribonucleoprotein complex (RISC). Guided by sequence-specific siRNAs, this complex identifies complementary target sequences with high precision. A comprehensive understanding of these regulatory pathways enables the targeted induction or suppression of specific plant genes. This review traces the history of experimental findings regarding the loss of recombinant gene activity in transformants and their progeny, which collectively established the foundation for elucidating the molecular mechanisms of transgene silencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ca1645109162b7bd31d62f8f845655fb6172721" target='_blank'>
              Gene Inactivation in Transgenic Plants—A Unique Model for Studying Epigenetic Regulation of Gene Expression
              </a>
            </td>
          <td>
            T. Marenkova, A. Zagorskaya, I. Deyneko, Elena V. Deineko
          </td>
          <td>2026-01-13</td>
          <td>Plants</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Type 2 Diabetes (T2D) is a complex disease that arises from interaction between genetic and environmental factors. The activation of transposable elements (TEs) and the production of circular extrachromosomal DNA (eccDNA) may represent genetic mechanisms involved in cellular aging, metabolic alterations, and T2D development through distinct pathways. Although the origin and characteristics of TEs and eccDNA differ substantially, eccDNA can in some cases be derived from TEs. This review summarizes the current understanding of these mechanisms and examines the reported associations between T2D and either TEs or eccDNAs. These findings highlight the significant involvement of these molecules in disease pathogenesis, particularly in relation to aging, and underscore their great potential as biomarkers and targets for T2D prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd8c580d6e27747cbde8711a7b98defd37bc9d7" target='_blank'>
              Extrachromosomal Circular DNA and Transposable Elements in Type 2 Diabetes
              </a>
            </td>
          <td>
            Celeste Moya-Valera, Alex Fernando Arita, Francisco Lara-Hernández, Ana-Bárbara García-García, Felipe Javier Chaves
          </td>
          <td>2025-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 The loss of chromosome 3p and the inactivation of the tumor suppressor gene von Hippel-Lindau (
 VHL
 ) were identified in clear cell renal cell carcinomas (ccRCC) over three decades ago. Since then, mutations in genes for the three chromatin modulators, polybromo 1 (PBRM1), SET domain-containing 2 (SETD2), and BRCA1-associated protein-1 (BAP1), have been recognized as common in ccRCC. Although these genomic alterations are central to understanding ccRCC’s development, other deregulated cellular processes are also prominent in these tumors. Metabolic reprogramming is a key hallmark of this disease, characterized by various changes linked to the stabilization of hypoxia-inducible factors (HIF), including increased aerobic glycolysis, elevated lipid levels, and glutamine dependence for cell survival. Additionally, HIF-α stabilization plays a crucial role in regulating the immune system, thereby enhancing CD8
 +
 T lymphocyte cytotoxicity. Immune checkpoint inhibitors (ICI) are now used as first-line treatments to target the often highly infiltrated tumor microenvironment of ccRCC. However, the effectiveness of ICI varies and is difficult to predict. Although emerging studies are beginning to provide insight, evidence suggests roles for PBRM1, SETD2, and BAP1 in metabolic regulation and in shaping the tumor immune microenvironment in ccRCC. Here, we review recent advances in this field and examine their impact on the management of ccRCC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d8124d8b083fe78630c316b5f74781f77ea65c" target='_blank'>
              The fall of the genome protectors triad: PBRM1, SETD2, and BAP1’s impact on metabolism and immunity in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Mathieu Johnson, Sandra Turcotte
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ab72daae241f6f66396542f56383a72c207669" target='_blank'>
              Rescue from pyridostatin-stabilized telomeric G-quadruplexes by DNA cleavage and microhomology-mediated end joining
              </a>
            </td>
          <td>
            Samah Matmati, Satyajeet Rao, Joachim Lingner
          </td>
          <td>2025-11-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6139c8ae9b321fbea21e5f26cd01d386682251c" target='_blank'>
              Familial colorectal cancer: search for novel predisposition genes.
              </a>
            </td>
          <td>
            A. Försti, Beiping Miao, Abhishek Kumar, Dagmara Dymerska Zaremba, Magdalena Marciniak, J. Lubiński, Kari Hemminki
          </td>
          <td>2025-12-30</td>
          <td>Human genomics</td>
          <td>0</td>
          <td>118</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [15, 0],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>