<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-06 09:26:20 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>14</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Mutations and genome instability are hallmarks of cancer. The advanced Next Generation Sequencing(NGS) technology now enables rapid and cost-effective analysis for tumoral and normal genomic translational research. In this research, mRNA sequencing was applied to get the whole transcriptome data of mouse B16F10 melanoma and CT26 colon carcinoma cells. Up regulation genes and down regulation genes were identified in tumor cells with differentially expressed genes(DEGs) analysis. The DEGs were analyzed with GO, KEGG and Reactome pathway enrichment study. Results demonstrated the DEGs and enriched pathways were mainly associated with cell proliferation and biosynthetic processing. Further, gene fusion and alternative splicing analysis reveled fusion genes and skipped exons were extensive exist within the chromosomes of tumor cells. Together, the integrated DEGs and bioinformatics analysis in this article may shed light on further research of cancer treating targets and molecular mechanisms of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6e94aa8f9282678d155caf1a32035cd06536087" target='_blank'>
              Identification and analysis of DEGs of mouse tumor cells by integrated bioinformatics methods
              </a>
            </td>
          <td>
            Xizhong Jing, Baiyong Li, Yifei Wang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Phenotypic plasticity is a hallmark of cancer and increasingly realized as a mechanism of resistance to androgen receptor (AR)-targeted therapy. Now that many prostate cancer (PCa) patients are treated upfront with AR-targeted agents, it's critical to identify actionable mechanisms that drive phenotypic plasticity, to prevent the emergence of resistance. We showed that loss of tristetraprolin (TTP, gene ZFP36) increased NF-κB activation, and was associated with higher rates of aggressive disease and early recurrence in primary PCa. We also examined the clinical and biological impact of ZFP36 loss with co-loss of PTEN, a known driver of PCa. Analysis of multiple independent primary PCa cohorts demonstrated that PTEN and ZFP36 co-loss was associated with increased recurrence risk. Engineering prostate-specific Zfp36 deletion in vivo, induced prostatic intraepithelial neoplasia, and, with Pten co-deletion, resulted in rapid progression to castration-resistant adenocarcinoma. Zfp36 loss altered the cell state driven by Pten loss, demonstrated by enrichment of EMT, inflammation, TNFα/NF-κB, IL6-JAK/STAT3 gene sets. Additionally, our work revealed that ZFP36 loss also induced enrichment of multiple gene sets involved in mononuclear cell migration, chemotaxis, and proliferation. Use of the NF-κB inhibitor, dimethylaminoparthenolide (DMAPT) induced marked therapeutic responses in tumors with PTEN and ZFP36 co-loss and reversed castration resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a8debd5a52234a8e6e2eecf0d0e1cd5849b25a" target='_blank'>
              Low tristetraprolin expression activates phenotypic plasticity and primes transition to lethal prostate cancer in mice.
              </a>
            </td>
          <td>
            Katherine L. Morel, Beatriz German, Anis A Hamid, Jagpreet S Nanda, S. Linder, A. Bergman, Henk van der Poel, Ingrid Hofland, E. Bekers, S. Trostel, Deborah L. Burkhart, S. Wilkinson, Anson T. Ku, Minhyung Kim, Jina Kim, Duanduan Ma, Jasmine T. Plummer, Sungyong You, Xiaofeng A. Su, W. Zwart, A. Sowalsky, Christopher J. Sweeney, Leigh Ellis
          </td>
          <td>2024-11-19</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The high plasticity of cells undergoing epithelial-mesenchymal transition (EMT) promotes increased tumor heterogeneity, and its interaction with tumor-associated stromal cells appears to contribute to developing a stemness phenotype. Cells with these characteristics exhibit increased resistance to chemotherapy and radiotherapy, leading to disease relapse and metastasis. Here, we discuss the activation of the Wnt/β-catenin pathway in promoting EMT and stemness within the context of cellular resistance to these therapies. We discuss whether EMT and cancer stem cells (CSCs) function in conjunction, independently, or if a link is connecting their development. We further propose that this pathway is necessary to establish a connection between these two phenotypes. And suggest that it could hinder the rise of CSCs from treatment-induced EMT cells when inhibited. Understanding this cellular phenomenon might allow the development of new targeted therapies to improve clinical responses, particularly in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6af1b3bc1ef6fcb118710249dd08cded2ddaa10" target='_blank'>
              Wnt/β-catenin pathway as a link between therapy resistance-driven epithelial-mesenchymal transition and stemness in colorectal cancer.
              </a>
            </td>
          <td>
            M. R. Rocha, Yuri Kelly Castillo-Medina, Bárbara Martins de Lima Coelho, Luidy Lucas Lopes Rios, J. Morgado-Díaz
          </td>
          <td>2024-12-20</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Aim: Resistance to hormonal and targeted therapies in breast cancer limits treatment efficacy. Epigenetic alterations, including changes mediated by DNA methyltransferases, play a key role in this process. Previously, we identified that resistance to tamoxifen and rapamycin is associated with the suppression of DNMT3A. This study aims to further explore the mechanisms underlying this suppression, with a focus on identifying NR6A1 as a novel regulatory factor. Methods: Acquisition of resistant breast cancer cell sublines, MTT-test, immunoblotting, transient transfection and reporter analysis, lentiviral infection, qRT-PCR, and analysis of methylation using bisulfite pyrosequencing. Results: Our findings indicate that the development of cross-resistance in breast cancer cells to hormonal and targeted therapies involves a shift in cell signaling to alternative AKT pathways, marked by a localized suppression of the NR6A1/DNMT3A axis and associated DNA methylation changes. We demonstrated the critical role of NR6A1 downregulation in resistance development. Additionally, we observed activation of Snail - a key regulator in the epithelial-mesenchymal transition - as a mediator of the effects of NR6A1 depletion, establishing a direct link between Snail expression and resistance formation. Conclusion: The coordinated suppression of NR6A1 and DNMT3A may contribute to sustaining the resistant phenotype in breast cancer cells. This pathway could serve as a predictive marker, helping guide the selection of optimal therapeutic strategies for breast cancer treatment in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d24584557816dae92ae33446c23b68005521f3e" target='_blank'>
              Breast cancer cell resistance to hormonal and targeted therapeutics is correlated with the inactivation of the NR6A1 axis
              </a>
            </td>
          <td>
            O. Andreeva, D. Sorokin, S. Vinokurova, P. Kopnin, N. V. Elkina, A. Katargin, R. S. Faskhutdinov, D. Salnikova, A. M. Scherbakov, M. Krasil'nikov
          </td>
          <td>2024-11-23</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Epitranscriptomics, the study of RNA modifications such as N6-methyladenosine (m6A), has emerged as a pivotal field in cancer research. These chemical modifications influence gene expression, protein translation and cellular behavior, driving critical processes like tumor initiation, progression and metastasis. Furthermore, RNA modifications contribute to cancer stem cell plasticity, promoting survival and therapy resistance. Treatment resistance, a major obstacle in cancer therapy, is often driven by aberrant RNA modifications that affect the stability of coding and non-coding RNAs, leading to enhanced DNA repair, drug efflux and immune evasion. As a result, targeting RNA-modifying enzymes has gained attention as a novel therapeutic strategy. Inhibitors of "writers," "erasers" and "readers" of these modifications are currently being explored to restore sensitivity to conventional therapies. This commentary discusses the emerging role of RNA modifications in cancer progression and treatment resistance, highlighting the potential for novel therapeutic interventions in combatting drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/275d0d235e86c5013c4a72496e49c927fb2f4445" target='_blank'>
              Epitranscriptomics in oncology: The double-edged role of RNA modifications in cancer and resistance
              </a>
            </td>
          <td>
            Harsh Nawal
          </td>
          <td>2024-11-15</td>
          <td>International Journal of Recent Innovations in Medicine and Clinical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a complex clonal disorder characterized by clinical, genetic, metabolomic, and epigenetic heterogeneity resulting in the uncontrolled proliferation of aberrant blood-forming precursor cells. Despite advancements in the understanding of the genetic, metabolic, and epigenetic landscape of AML, it remains a significant therapeutic challenge. Functional profiling techniques, such as BH3 profiling (BP), gene expression profiling (GEP), proteomics, metabolomics, drug sensitivity/resistance testing (DSRT), CRISPR/Cas9, and RNAi screens offer valuable insights into the functional behavior of leukemia cells. BP evaluates the mitochondrial response to pro-apoptotic BH3 peptides, determining a cell’s apoptotic threshold and its reliance on specific anti-apoptotic proteins. This knowledge can pinpoint vulnerabilities in the mitochondria-mediated apoptotic pathway in leukemia cells, potentially informing treatment strategies and predicting therapeutic responses. GEP, particularly RNA sequencing, evaluates the transcriptomic landscape and identifies gene expression alterations specific to AML subtypes. Proteomics and metabolomics, utilizing mass spectrometry and nuclear magnetic resonance (NMR), provide a detailed view of the active proteins and metabolic pathways in leukemia cells. DSRT involves exposing leukemia cells to a panel of chemotherapeutic and targeted agents to assess their sensitivity or resistance profiles and potentially guide personalized treatment strategies. CRISPR/Cas9 and RNAi screens enable systematic disruption of genes to ascertain their roles in leukemia cell survival and proliferation. These techniques facilitate precise disease subtyping, uncover novel biomarkers and therapeutic targets, and provide a deeper understanding of drug-resistance mechanisms. Recent studies utilizing functional profiling have identified specific mutations and gene signatures associated with aggressive AML subtypes, aberrant signaling pathways, and potential opportunities for drug repurposing. The integration of multi-omics approaches, advances in single-cell sequencing, and artificial intelligence is expected to refine the precision of functional profiling and ultimately improve patient outcomes in AML. This review highlights the diverse landscape of functional profiling methods and emphasizes their respective advantages and limitations. It highlights select successes in how these methods have further advanced our understanding of AML biology, identifies druggable targets that have improved outcomes, delineates challenges associated with these techniques, and provides a prospective view of the future where these techniques are likely to be increasingly incorporated into the routine care of patients with AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d60f51fb21da6cf836930cd187d2acd3d680d13" target='_blank'>
              Decoding Acute Myeloid Leukemia: A Clinician’s Guide to Functional Profiling
              </a>
            </td>
          <td>
            Prasad Iyer, S. Jasdanwala, Yuhan Wang, K. Bhatia, Shruti Bhatt
          </td>
          <td>2024-11-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Background: Osimertinib is currently the most common precision therapy for patients with non- small cell lung cancer (NSCLC) with EGFR mutations. While it prolongs progression-free survival, resistance eventually develops in many, leading to disease progression. Somatic NGS has identified resistance mutations in the EGFR pathway and other oncogenes, but these account for only 40-45% of cases, suggesting non-genetic factors, such as epigenetic changes, may be involved. This study investigates the epigenetic landscape at diagnosis and progression on first- line osimertinib, focusing on cases with and without canonical resistance mutations. Methods: We defined two cohorts from GuardantINFORM, a clinico-genomics database, restricting to only patients with a test on the Guardant Infinity platform, a comprehensive blood-based NGS assay, that analyzes DNA sequence variants and performs extensive methylation profiling (∼15Mb) in >20,000 differentially methylated regions (DMRs) assessed for sample-specific methylation patterns. (A) Diagnosis Cohort: Comprised of 34 NSCLC patients with blood samples collected within six days prior to the initiation of first-line osimertinib therapy. (B) Progression Cohort: Comprised of 36 patients who were on first-line osimertinib therapy, with blood samples collected within 60 days of discontinuation. The progression cohort was further stratified based on the presence or absence of known resistance mutations identified through somatic variant analysis by Guardant360. Gene set enrichment analysis was conducted on these methylation profiles using the Reactome database to define pathway-level differences between the cohorts. Results: Pathway enrichment analysis in the diagnosis and progression cohorts found expected biological processes in NSCLC such as extracellular matrix organization, G- protein coupled receptor (GPCR) signaling, and FGFR pathways. Additionally, the progression cohort lacking canonical resistance mutations (n=23) exhibited unique enrichment in two specific pathways: Glucose-dependent Insulinotropic Polypeptide and Physiological Factors. These pathways include genes such as GATA4, which has been implicated in the epithelial- mesenchymal transition (EMT)—a process associated with transforming to a small cell phenotype, a known mechanism of resistance to EGFR inhibition, including osimertinib. Conclusion: This study demonstrates the utility of epigenetic profiling via liquid biopsies in uncovering non-genetic mechanisms underlying osimertinib resistance in advanced NSCLC. Identification of unique epigenetic pathways in progressing patients without canonical resistance mutations highlights the potential for discovering novel biomarkers of resistance. Further investigation into these findings may advance our understanding of resistance mechanisms across various cancer therapies, particularly for patients without identified genomic resistance, and inform clinical decision-making and trial design.
 Citation Format: Aaron Hardin, Leslie Bucheit, Amar Das, Craig Eagel, Helmy Eltoukhy. Pathway repression in progression on osimertinib therapy in NSCLC via a comprehensive genomic and epigenomic circulating tumor DNA (ctDNA) assay [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B036.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9810af5aadd760f6c2c32f19cb5c026e7ca93" target='_blank'>
              Abstract B036: Pathway repression in progression on osimertinib therapy in NSCLC via a comprehensive genomic and epigenomic circulating tumor DNA (ctDNA) assay
              </a>
            </td>
          <td>
            Aaron Hardin, L. Bucheit, Amar Das, Craig Eagel, H. Eltoukhy
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cells exploit a mesenchymal-like transcriptional state (MLS) to survive drug treatments. Although the MLS is well characterized, few therapeutic vulnerabilities targeting this program have been identified. Here, we systematically identify the dependency network of mesenchymal-like cancers through an analysis of gene essentiality scores in ~800 cancer cell lines, nominating a poorly studied kinase, PKN2, as a top therapeutic target of the MLS. Co-essentiality relationships, biochemical experiments, and genomic analyses of patient tumors revealed that PKN2 promotes mesenchymal-like cancer growth through a PKN2-SAV1-TAZ signaling mechanism. Notably, pairing genetic PKN2 inhibition with clinically relevant targeted therapies against EGFR, KRAS, and BRAF oncogenes suppresses drug resistance by depleting mesenchymal-like drug-tolerant persister cells. These findings provide evidence that PKN2 is a core regulator of the Hippo tumor suppressor pathway and highlight the potential of PKN2 inhibition as a generalizable therapeutic strategy to overcome drug resistance driven by the MLS across cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc877056e57a8e2b137d870c6f49bfae37eaa22e" target='_blank'>
              PKN2 is a dependency of the mesenchymal-like cancer cell state.
              </a>
            </td>
          <td>
            Shane T. Killarney, Gabriel Mesa, Rachel Washart, Benjamin Mayro, Kerry Dillon, S. Wardell, M. Newlin, Min Lu, Areej Abu Rmaileh, Nicky Liu, Donald P McDonnell, A. Pendergast, Kris C Wood
          </td>
          <td>2024-11-19</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d50475b8002866903aae4806efbfc187f7cb121" target='_blank'>
              Canonical and noncanonical NOTCH signaling in the nongenetic resistance of cancer: distinct and concerted control.
              </a>
            </td>
          <td>
            Xianzhe Huang, Wenwei Chen, Yanyan Wang, Dmytro Shytikov, Yanwen Wang, Wangyi Zhu, Ruyi Chen, Yuwei He, Yanjia Yang, Wei Guo
          </td>
          <td>2025-01-02</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ex vivo drug response profiling is emerging as a valuable tool for identifying drug resistance mechanisms and advancing precision medicine in hematological cancers. However, the functional impact of dysregulation of the epigenome and transcriptome in this context remains poorly understood. In this study, we combined ex vivo drug sensitivity profiling with transcriptomic and epigenomic analyses in diagnostic samples from 597 pediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) patients. Ex vivo resistance to antimetabolites (e.g., cytarabine, thioguanine), glucocorticoids (e.g., dexamethasone, prednisolone), and doxorubicin was independently associated with reduced relapse-free survival (RFS, p < 0.05). Molecular profiling identified pre-treatment DNA methylation and gene expression patterns distinguishing resistant from sensitive cases, revealing key drug resistance signatures. These included aberrant expression of genes related to heme metabolism (e.g. ATPV06A) and KRAS signaling (e.g. ABCB1). Notably, we also observed atypical expression of genes usually restricted to T-cells and other immune cells (e.g., ITK) in resistant BCP-ALL cells. Our findings demonstrate that ex vivo drug response patterns are predictive of clinical outcomes and reflect intrinsic molecular states associated with drug tolerance. This integrative multi-omics approach highlights potential therapeutic targets and underscores the value of functional precision medicine in identifying treatment vulnerabilities in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b0bb7a98b1f284795f68f2172436eb0a2df655" target='_blank'>
              Ex Vivo Drug Responses and Molecular Profiles of 597 Pediatric Acute Lymphoblastic Leukemia Patients
              </a>
            </td>
          <td>
            Anna Pia Enblad, O. Krali, Henrik Gezelius, Anders Lundmark, Kristin Blom, Claes Andersson, J. Palle, B. Frost, S. Ryhänen, Trond Flægstad, Ó. Jónsson, Kjeld Schmiegelow, Arja Harila, Peter Nygren, Rolf Larsson, Gudmar Lönnerholm, J. Nordlund, Anna Pia
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="KRAS was long deemed undruggable until the discovery of the switch-II pocket facilitated the development of specific KRAS inhibitors. Despite their introduction into clinical practice, resistance mechanisms can limit their effectiveness. Initially, tumors rely on mutant KRAS, but as they progress, they may shift to alternative pathways, resulting in intrinsic resistance. This resistance can stem from mechanisms like epithelial-to-mesenchymal transition (EMT), YAP activation, or KEAP1 mutations. KRAS inhibition often triggers cellular rewiring to counteract therapeutic pressure. For instance, feedback reactivation of signaling pathways such as MAPK, mediated by receptor tyrosine kinases, supports tumor cell survival. Inhibiting KRAS disrupts protein homeostasis, but reactivation of MAPK or AKT can restore it, aiding tumor cell survival. KRAS inhibition also causes metabolic reprogramming and protein re-localization. The re-localization of E-cadherin and Scribble from the membrane to the cytosol causes YAP to translocate to the nucleus, where it drives MRAS transcription, leading to MAPK reactivation. Emerging evidence indicates that changes in cell identity, such as mucinous differentiation, shifts from alveolar type 2 to type 1 cells, or lineage switching from adenocarcinoma to squamous cell carcinoma, also contribute to resistance. In addition to these nongenetic mechanisms, secondary mutations in KRAS or alterations in upstream/downstream signaling proteins can cause acquired resistance. Secondary mutations in the switch-II pocket disrupt drug binding, and known oncogenic mutations affect drug efficacy. Overcoming these resistance mechanisms involves enhancing the efficacy of drugs targeting mutant KRAS, developing broad-spectrum inhibitors, combining therapies targeting multiple pathways, and integrating immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c1ae3fc98dfd6ef9151275fdf77ddcf612521a" target='_blank'>
              Mechanisms of Resistance to KRAS Inhibitors: Cancer Cells' Strategic Use of Normal Cellular Mechanisms to Adapt.
              </a>
            </td>
          <td>
            Noritaka Tanaka, Hiromichi Ebi
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bcc9e0c8d78b7e4a0d8c81e52724a7077db80c6" target='_blank'>
              Saturation profiling of drug-resistant genetic variants using prime editing.
              </a>
            </td>
          <td>
            Younggwang Kim, Hyeong-Cheol Oh, Seungho Lee, H. Kim
          </td>
          <td>2024-11-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53d714dd5d6b619d9b1b3610a6e67560b7be4c02" target='_blank'>
              RAS signaling in carcinogenesis, cancer therapy and resistance mechanisms
              </a>
            </td>
          <td>
            Xiaojuan Yang, Hong Wu
          </td>
          <td>2024-11-09</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e385b264839697ab3b154e181f427818a7fd2d8b" target='_blank'>
              Biological and genetic characterization of a newly established human primary multidrug-resistant distal cholangiocarcinoma cell line, CBC3T-6
              </a>
            </td>
          <td>
            Mingzhen Bai, Ruoshui Wang, Chongfei Huang, Ruyang Zhong, Ningzu Jiang, Wenkang Fu, Ningning Mi, Long Gao, Yuyao Jin, Haidong Ma, Jie Cao, Haiying Yu, Qiang Jing, Chao Zhang, Ping Yue, Yong Zhang, Yanyan Lin, Hengwei Zhang, Wenbo Meng
          </td>
          <td>2024-11-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2f707eacba8ac9d3c84de9541b146c76849cd9" target='_blank'>
              Exosome-delivered NR2F1-AS1 and NR2F1 drive phenotypic transition from dormancy to proliferation in treatment-resistant prostate cancer via stabilizing hormonal receptors
              </a>
            </td>
          <td>
            Wenbin Chen, Yiyou Mao, YiYuan Zhan, Wenfeng Li, Jun Wu, Xiangming Mao, Bin Xu, Fangpeng Shu
          </td>
          <td>2024-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/029b373b085751bc9a273803a2aea04199d3ea75" target='_blank'>
              Metabolic reprogramming and therapeutic resistance in primary and metastatic breast cancer
              </a>
            </td>
          <td>
            Shan Liu, Xingda Zhang, Wenzheng Wang, Xue Li, Xue Sun, Yuqian Zhao, Qi Wang, Yingpu Li, Fangjie Hu, He Ren
          </td>
          <td>2024-11-21</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3c33518592790fa9ef5b83d8e1c71cca6958d4e" target='_blank'>
              Murine colon cancer derived cells exhibit heterogeneous resistance profiles against an oncolytic virus
              </a>
            </td>
          <td>
            Alejandra Larrieux, Rafael Sanjuán
          </td>
          <td>2024-11-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aim: Mutations in the mitochondrial (mt) genome contribute to metabolic dysfunction and their accumulation relates to disease progression and resistance development in cancer cells. This study explores the mutational status of the mt genome of cisplatin-resistant vs. -sensitive testicular germ cell tumor (TGCT) cells and explores its association with their respiration parameters, expression of respiratory genes, and preferences for metabolic pathways to reveal new markers of therapy resistance in TGCTs.
 Methods: Using Illumina sequencing with Twist Enrichment Panel, the mutations of mt genomes of sensitive 2102EP, H12.1, NTERA-2, T-cam and resistant 2102EP Cis, H12.1 ODM, 1411HP, 1777NRpmet, NTERA-2 Cis and T-cam Cis cell lines were identified. The mt respiration of the cells was assessed using high-resolution respirometry method (O2k-respirometer Oroboros) and the differential expression profiles of mt respiratory genes were determined using RT-qPCR. Associated preferences for metabolic pathways were compared using Glycolysis/OXPHOS assay.
 Results: In resistant TGCT cells, new mutations in mt genes MT-ND1-6, MT-RNR, MT-CO1-3, MT-ATP6 , and MT-CYB were recognized. The respiratory rates of the 1777NRpmet cell line were the highest, while those of the 1411HP line the lowest; rates of the control and all other TGCT cell lines fell between these two lines. The statistically significant differences in gene expression of the respiratory genes were recorded only in NTERA-2 Cis and T-cam Cis cell lines. Sensitive cell lines NTERA-2 and 2102EP preferred oxidative phosphorylation (OXPHOS), while glycolysis was typical for resistant NTERA-2 Cis, 2102EP Cis and 1411HP cell lines. Metastatic 1777NRpmet cells seem to utilize both. An isogenic pair of cell lines H12.1 and H12.1ODM showed the opposite dependence, sensitive H12.1 preferring glycolysis, while resistant H12.1ODM OXPHOS.
 Conclusion: In summary, our study identified new mutations in mt genes of resistant TGCT cell lines that are associated with different mt respiration parameters, gene expression patterns and preferences for metabolic pathways, providing potential novel molecular biomarkers that distinguish the resistant TGCT phenotype or specify its histological classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec03801d589beba72991767df646f63f72e7a8" target='_blank'>
              Mitochondrial genome variability and metabolic alterations reveal new biomarkers of resistance in testicular germ cell tumors
              </a>
            </td>
          <td>
            Pavlina Kabelikova, Danica Ivovic, Z. Sumbalová, M. Karhánek, Lucia Tatayova, Martina Skopkova, Michal Cagalinec, V. Bruderová, J. Roska, Dana Jurkovicova
          </td>
          <td>2024-12-18</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In a comprehensive study to decipher the multi-layered response to the chemotherapeutic agent temozolomide (TMZ), we analyzed 427 genomes and determined mutational patterns in a collection of ∼40 isogenic DNA repair-deficient human TK6 lymphoblast cell lines. We first demonstrate that the spontaneous mutational background is very similar to the aging-associated mutational signature SBS40 and mainly caused by polymerase zeta-mediated translesion synthesis (TLS). MSH2-/- mismatch repair (MMR) knockout in conjunction with additional repair deficiencies uncovers cryptic mutational patterns. We next report how distinct mutational signatures are induced by TMZ upon sequential inactivation of DNA repair pathways, mirroring the acquisition of chemotherapy resistance by glioblastomas. The most toxic adduct induced by TMZ, O6-meG, is directly repaired by the O6-methylguanine-DNA methyltransferase (MGMT). In MGMT-/- cells, MMR leads to cell death and limits mutagenesis. MMR deficiency results in TMZ resistance, allowing the accumulation of ∼105 C > T substitutions corresponding to signature SBS11. Under these conditions, N3-methyladenine (3-meA), processed by base excision repair (BER), limits cell survival. Without BER, 3-meA is read through via error-prone TLS, causing T > A substitutions but not affecting survival. Blocking BER after abasic site formation results in large deletions and TMZ hypersensitization. Our findings reveal potential vulnerabilities of TMZ-resistant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdcf8efd6ed1b3006f0325ffb555aaabb03b08a5" target='_blank'>
              Comprehensive whole-genome sequencing reveals origins of mutational signatures associated with aging, mismatch repair deficiency and temozolomide chemotherapy.
              </a>
            </td>
          <td>
            Taejoo Hwang, Lukasz Karol Sitko, Ratih Khoirunnisa, Fernanda Navarro-Aguad, David M Samuel, Hajoong Park, Banyoon Cheon, Luthfiyyah Mutsnaini, Jaewoong Lee, Burçak Otlu, Shunichi Takeda, Semin Lee, Dmitri Ivanov, Anton Gartner
          </td>
          <td>2024-12-05</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ae77252749b1c986ef7ff79cc410ca86b264c4" target='_blank'>
              Comparison of baseline global gene expression profiles of prostate cancer cell lines LNCaP and DU145
              </a>
            </td>
          <td>
            Khalid Ahmed, Zhannur Omarova, Alisalman Sheikh, Gulzhan Abuova, K. Ghias, Syed Hani Abidi
          </td>
          <td>2024-12-31</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81cf24082e18b8a822679c8c6c602531a443fe70" target='_blank'>
              Nucleotide metabolism-associated drug resistance gene NDUFA4L2 promotes colon cancer progression and 5-FU resistance
              </a>
            </td>
          <td>
            Hongxin He, Shiyao ZHENG, Shangkun JIN, Weijie Huang, Enhao Wei, Shen Guan, Chunkang Yang
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Background: Prostate cancer is a prevalent malignancy in men, with treatment resistance posing a significant challenge to successful cure. Acquired Resistance to Therapy (ART) arises from two main components: (1) Tumor Microenvironment (TME) and (2) mutations or epigenetic changes at gene level within the tumor cells themselves. While most cancer therapies target the intrinsic biology of cancer cells, growing evidence indicates that these treatments can also harm the stromal cells within TME, potentially driving ART. The biology of ART may vary based on the types of treatments, cell types, and organ locations. Thus, this study aims to dissect ART components driven by TME-mediated cellular processes in a comprehensive manner. Hypothesis: We propose that cancer therapies induce stress responses in non-cancerous cells within the TME. These stress responses involve the secretion of cytokines, mediators, and growth factors, which we term Stress Response Secretory Programs (SRSPs). We further hypothesize that targeting SRSPs could overcome ART that helps to improve treatment specificity and efficacy. Materials and Methods: We employed comprehensive single-cell profiling of common metastatic sites included liver, prostate, lung, bone, lymph node and spleen in preclinical models to identify secreted proteins and dysregulated pathways caused by genotoxic therapies. In vitro, we evaluated treatment resistance in murine prostate epithelial cell lines (Myc-CAP) by exposing them to conditioned media (CM) from irradiated murine fibroblasts (F4M2), using cell growth assays. Results: Single-cell RNA sequencing of organ samples revealed significant expressions of SRSPs and increased inflammatory responses in liver and prostate fibroblasts as well as other stromal cells treated with carboplatin. In vitro, CM from irradiated F4M2 cells enhanced cell proliferation and survival of Myc-CAP cells in response to docetaxel treatment. Conclusion: Our findings highlight the crucial role of SRSPs in mediating ART through interactions within the TME, which alter tumor cell phenotypes. Targeting these SRSPs holds potential for overcoming treatment resistance and improving therapeutic outcomes, underscoring the need to consider TME complexity in cancer treatment strategies.
 Citation Format: Tony Lok Heng Chu, Armand Bankhead, Ilsa Coleman, Sander Frank, Peter S. Nelson, Tarana Arman. Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C033.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c940709273c954da8cb4bd8042d98f46649ce8ee" target='_blank'>
              Abstract C033: Identifying and targeting prostate cancer therapy resistance mechanisms driven by the tumor microenvironment
              </a>
            </td>
          <td>
            Tony Lok Heng Chu, Armand Bankhead, Ilsa M Coleman, Sander Frank, Peter S. Nelson, Tarana Arman
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Background Osteosarcoma is a malignant tumor originating from mesenchymal bone tissue, characterized by high malignancy and poor prognosis. Despite progress in comprehensive treatment approaches, the five-year survival rate remains largely unchanged, highlighting the need to clarify its underlying mechanisms and discover new therapeutic targets. Methods This study utilized RNA sequencing data from multiple public databases, encompassing osteosarcoma samples and healthy controls, along with single-cell RNA sequencing data. Various methods were utilized, such as differential expression analysis of genes, analysis of metabolic pathways, and weighted gene co-expression network analysis (WGCNA), to pinpoint crucial genes. Using this list of genes, we developed and validated a prognostic model that incorporated risk signatures, and we evaluated the effectiveness of the model through survival analysis, immune cell infiltration examination, and drug sensitivity evaluation. Results We analyzed gene expression and metabolic pathways in nine samples using single-cell sequencing data. Initially, we performed quality control and clustering, identifying 21 statistically significant cell subpopulations. Metabolic analyses of these subpopulations revealed heterogeneous activation of metabolic pathways. Focusing on the osteoblastic cell subpopulation, we further subdivided it into six groups and examined their gene expression and differentiation capabilities. Differential expression and enrichment analyses indicated that tumor tissues were enriched in cytoskeletal and structural pathways. Through WGCNA, we identified core genes negatively correlated with four highly activated metabolic pathways. Using osteosarcoma patient data, we developed a risk signature model that demonstrated robust prognostic predictions across three independent cohorts. Ultimately, we performed a thorough examination of the model, which encompassed clinical and pathological characteristics, enrichment analysis, pathways associated with cancer markers, and scores of immune infiltration, highlighting notable and complex disparities between high-risk and low-risk populations. Conclusion This research clarifies the molecular mechanisms and metabolic features associated with osteosarcoma and how they relate to patient outcomes, offering novel perspectives and approaches for targeted therapy and prognostic assessment in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3c754f2161a5b2138f06ad7cb757b73f892978d" target='_blank'>
              Exploring the heterogeneity of osteosarcoma cell characteristics and metabolic states and their association with clinical prognosis
              </a>
            </td>
          <td>
            Sen Qin, YaoFeng Hu, RuCui Deng, Zhe Wang
          </td>
          <td>2024-12-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study aimed to generate Car- and Pac-resistant cell lines from the human lung adenocarcinoma H1792 cell line, designated as H1792/Car and H1792/Pac, and perform transcriptome sequencing to identify potential targets. Common differentially expressed genes (Co-DEGs) in both resistant cell lines were identified, followed by hub gene identification. Online validation was conducted through GEPIA and Kaplan-Meier Plotter platforms, with experimental validation performed using real-time quantitative PCR (RT-qPCR). After six months, the H1792/Car and H1792/Pac cell lines exhibited a 10.7-fold and 5.6-fold increase in resistance to Car and Pac, respectively. Flow cytometry analysis demonstrated that both resistant cell lines were resistant to cell cycle arrest and apoptosis induced by Car or Pac. Transcriptomic sequencing identified 123 Co-DEGs, including 72 upregulated and 51 downregulated genes, consistently expressed in both H1792/Car and H1792/Pac cell lines. Among these, 13 hub genes were identified, with colony-stimulating factor 3 (CSF3) uniquely associated with post-progression survival (PPS) in adenocarcinoma patients undergoing chemotherapy. Notably, CSF3 expression was significantly elevated in both H1792/Car and H1792/Pac compared to parental cells. These findings underscore the value of drug-resistant models in uncovering critical biomarkers. CSF3 emerges as a promising guiding marker or potential molecular target for optimizing Car- and Pac-based doublet regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/319b42d1b63fc345fe5d5188434196b326a0a009" target='_blank'>
              Transcriptomic Profiling of Carboplatin- and Paclitaxel-Resistant Lung Adenocarcinoma Cells Reveals CSF3 as a Potential Biomarker for the Carboplatin Plus Paclitaxel Doublet Regimens.
              </a>
            </td>
          <td>
            Pritsana Raungrut, Suchanan Tanyapattrapong, Thipphanet Masjon, Saowanee Maungchanburi, P. Thongsuksai
          </td>
          <td>2024-12-11</td>
          <td>Current issues in molecular biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cell-cell fusion has been implicated in various physiological and pathological processes, including cancer progression. This study investigated the role of cell-cell fusion in non-small cell lung cancer (NSCLC), focusing on its contribution to chemoresistance and tumor evolution. By co-culturing drug-sensitive and drug-resistant NSCLC cell lines, we observed spontaneous cell-cell fusion events, particularly under gefitinib selection. These fused cells exhibited enhanced fitness and a higher degree of chemoresistance compared to parental lines across a panel of 12 chemotherapeutic agents. Further analysis, including fluorescence imaging and cell cycle analysis, confirmed nuclear fusion and increased DNA content in the fused cells. Bulk RNA sequencing revealed genomic heterogeneity in fused cells, including enrichment of gene sets associated with cell cycle progression and epithelial-mesenchymal transition, hallmarks of cancer malignancy. These findings demonstrate that cell-cell fusion contributes significantly to therapeutic resistance and the promotion of aggressive phenotypes in NSCLC, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f78ae5e6d6b40de2fc2ee9f6021e6a3e7d5997a6" target='_blank'>
              Cell-Cell Fusion in NSCLC Confers a Fitness Benefit Under Drug Selection
              </a>
            </td>
          <td>
            Paulameena V. Shultes, D. Tadele, A. Durmaz, D. Weaver, R. Barker-Clarke, Mina N. Dinh, Siqing Liu, Endalkachew A. Alemu, Simon Rayner, Jacob G. Scott
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a multi-clonal disease, existing as a milieu of clones with unique but related genotypes as initiating clones acquire subsequent mutations. However, bulk sequencing cannot fully capture AML clonal architecture or the clonal evolution that occurs as patients undergo therapy. To interrogate clonal evolution, we performed simultaneous single cell molecular profiling and immunophenotyping on 43 samples from 32 NPM1-mutant AML patients at different stages of disease. Here we show that diagnosis and relapsed AML samples display similar clonal architecture patterns, but signaling mutations can drive increased clonal diversity specifically at relapse. We uncovered unique genotype-immunophenotype relationships regardless of disease state, suggesting leukemic lineage trajectories can be hard-wired by the mutations present. Analysis of longitudinal samples from patients on therapy identified dynamic clonal, transcriptomic, and immunophenotypic changes. Our studies provide resolved understanding of leukemic clonal evolution and the relationships between genotype and cell state in leukemia biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ca51b5a9370b4c12038c29c1cdd25d0324a8d0" target='_blank'>
              Genotype-immunophenotype relationships in NPM1-mutant AML clonal evolution uncovered by single cell multiomic analysis
              </a>
            </td>
          <td>
            Morgan Drucker, Darren Lee, Xuan Zhang, Bailee N. Kain, Michael Bowman, D. Nicolet, Zhe Wang, R. Stone, K. Mrózek, Andrew J Carroll, D. Starczynowski, R. L. Levine, John C. Byrd, Ann-Kathrin Eisfeld, Nathan Salomonis, H. L. Grimes, Robert L Bowman, Linde A. Miles
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dac92d6b19ba0a7b037b162b322bc5d897b5e68" target='_blank'>
              Programmed cell death pathways in lung adenocarcinoma: illuminating tumor drug resistance and therapeutic opportunities through single-cell analysis
              </a>
            </td>
          <td>
            Long Li, Shancheng He
          </td>
          <td>2024-12-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01ace1a7b748e6be8dfa6d49964b342cccfe62af" target='_blank'>
              PCK1 as a target for cancer therapy: from metabolic reprogramming to immune microenvironment remodeling
              </a>
            </td>
          <td>
            Na Liu, Xiao-Ren Zhu, Chang-Ying Wu, Yuan-Yuan Liu, Minbin Chen, Jin-Hua Gu
          </td>
          <td>2024-11-22</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55b842a30320c99847f56c0cadef798436c42b45" target='_blank'>
              PFKFB3-dependent redox homeostasis and DNA repair support cell survival under EGFR-TKIs in non-small cell lung carcinoma
              </a>
            </td>
          <td>
            Nadiia Lypova, Susan M. Dougherty, Brian F. Clem, Jing Feng, Xinmin Yin, Xiang Zhang, Xiaohong Li, Jason Chesney, Y. Imbert-Fernandez
          </td>
          <td>2024-12-18</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/660334082207693e7b39f13fca8e1d97d42e1441" target='_blank'>
              Comprehensive review of drug resistance in mammalian cancer stem cells: implications for cancer therapy
              </a>
            </td>
          <td>
            B. Mengistu, Tirunesh Tsegaw, Yitayew Demessie, Kalkidan Getnet, A. Bitew, M. Kinde, Asnakew Mulaw Beirhun, Atsede solomon Mebratu, Yesuneh Tefera Mekasha, Melaku Getahun Feleke, Melkie Dagnaw Fenta
          </td>
          <td>2024-12-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb69ecb6e89b0ba1487649df7128f98c6062a08a" target='_blank'>
              Multimodal analysis of RNA sequencing data powers discovery of complex trait genetics
              </a>
            </td>
          <td>
            D. Munro, Nava Ehsan, Seyed Mehdi Esmaeili-Fard, Alexander Gusev, A. Palmer, P. Mohammadi
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f970e4e2fc5fa77ad8483f91874195b589fe0b" target='_blank'>
              Single-cell RNA-seq reveals diverse molecular signatures associated with Methotrexate resistance in primary central nervous system lymphoma cells.
              </a>
            </td>
          <td>
            Rosuke Osako, A. Hayano, Atsushi Kawaguchi, R. Yamanaka
          </td>
          <td>2024-12-05</td>
          <td>Journal of neuro-oncology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Pharmacological inhibitors of oncogenic signaling, such as drugs that target ALK kinase (ALKi) in ALK+ lung cancers, can induce strong and durable clinical responses. However, targeted therapies are not curative in advanced cancers and even strong responders eventually develop resistance. While therapy resistance is typically attributed to cell-intrinsic (epi)mutational characteristics of tumor cells, it can also result from interactions of tumor cells with the tumor microenvironment (TME). In contrast to the detailed elucidation of the molecular mediators of therapy resistance, our knowledge of the evolutionary dynamics underlying the resistance emergence, including the impact of TME, remains poorly defined. A key open question in understanding the evolutionary dynamics of therapy resistance is whether relapse results from an expansion of pre-existing therapy-resistant subpopulations or from a bona fide gradual evolutionary process. To elucidate this question, we integrated experimental mouse studies with mathematical modeling, interrogating therapeutic responses of therapy-naïve experimental xenograft ALK+ tumors, spiked-in with differentially labeled resistant cells. We found that ALKi treatment induced a rapid expansion of resistant cells, which drastically reduced the magnitude and duration of remissions. Surprisingly, our in silico analyses pointed to the existence of a strong positive ecological interaction between therapy-resistant and therapy-sensitive competitors. Using a combination of spatial analyses, experimental studies, and mathematical modeling, we found that this interaction was mediated by peristromal niches that protected therapy-sensitive tumor cells. Specifically, by limiting tumor regression, the therapy- induced expansion of resistant cells limited the loss of protective peristromal niches, while the subsequent resumption of tumor growth created new peristromal niches capable of supporting the survival and proliferation of therapy-sensitive cells. While this niche-mediated interaction had only a marginal impact on the transition from remission to relapse, enhanced survival of therapy-sensitive cells potentiated their ability to adapt to ALKi, leading to a higher diversity of resistance phenotypes. In summary, our study has revealed a new type of indirect, niche-mediated ecological interaction between therapy-resistant and therapy-sensitive cells. The rapid expansion of rare pre-existent resistant cells and fast transition to relapse challenge a common assumption of the pre-existence of therapy resistance. Finally, our results highlight the essentiality of TME considerations in understanding the evolutionary dynamic underlying the development of therapy resistance.
 Citation Format: Mark Robertson-Tessi, Bina Desai, Tatiana Miti, Pragya Kumar, Sagnik Yarlagadda, Rishi Shah, Robert Vander Velde, Daria Miroshnychenko, David Basanta, Alexander Anderson, Andriy Marusyk. Therapy-protective peristromal niches mediate positive ecological interaction between therapy-sensitive and therapy-resistant cells, altering the evolutionary dynamics of acquired targeted therapy resistance in lung cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/832b711609a909524835a20edd34f9feaf6c94a2" target='_blank'>
              Abstract B008: Therapy-protective peristromal niches mediate positive ecological interaction between therapy-sensitive and therapy-resistant cells, altering the evolutionary dynamics of acquired targeted therapy resistance in lung cancers
              </a>
            </td>
          <td>
            M. Robertson-Tessi, Bina Desai, Tatiana Miti, Pragya Kumar, Sagnik Yarlagadda, Rishi Shah, R. V. Velde, Daria Miroshnychenko, D. Basanta, Alexander R. A. Anderson, A. Marusyk
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The transcriptional heterogeneity of tumor microenvironment (TME) cells is a crucial factor driving the diversity of cellular response to drug treatment and resistance. Therefore, characterizing the cells associated with drug treatment and resistance will help us understand therapeutic mechanisms, discover new therapeutic targets and facilitate precision medicine. Here, we describe a database, scDrugAct (http://bio-bigdata.hrbmu.edu.cn/scDrugAct/), which aims to establish connections among drugs, genes and cells and dissect the impact of TME cellular heterogeneity on drug action and resistance at single-cell resolution. ScDrugAct is curated with drug-cell connections between 3838 223 cells across 34 cancer types and 13 857 drugs and identifies 17 274 drug perturbation/resistance-related genes and 276 559 associations between >10 000 drugs and 53 cell types. ScDrugAct also provides multiple flexible tools to retrieve and analyze connections among drugs, genes and cells; the distribution and developmental trajectories of drug-associated cells within the TME; functional features affecting the heterogeneity of cellular responses to drug perturbation and drug resistance; the cell-specific drug-related gene network; and drug-drug similarities. ScDrugAct serves as an important resource for investigating the impact of the cellular heterogeneity of the TME on drug therapies and can help researchers understand the mechanisms of action and resistance of drugs, as well as discover therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/043f2ce1e460a880b52e0d2bd7d4a4783b1e445d" target='_blank'>
              ScDrugAct: a comprehensive database to dissect tumor microenvironment cell heterogeneity contributing to drug action and resistance across human cancers.
              </a>
            </td>
          <td>
            Yanjun Xu, Yifang Zhang, Kaiyue Song, Jiaqi Liu, Rui Zhao, Xiaomeng Zhang, Liying Pei, Mengyue Li, Zhe Chen, Chunlong Zhang, Peng Wang, Feng Li
          </td>
          <td>2024-11-11</td>
          <td>Nucleic acids research</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="In cancer research, the term epigenetics was used in the 1970s in its modern sense encompassing non-genetic events modifying the chromatin state, mainly to oppose the emerging oncogene paradigm. However, starting from the establishment of this prominent concept, the importance of these epigenetic phenomena in cancer rarely led to questioning the causal role of genetic alterations. Only in the last 10 years, the accumulation of problematic data, better experimental technologies, and some ambitious models pushed the idea that epigenetics could be at least as important as genetics in early oncogenesis. Until this year, a direct demonstration of epigenetic oncogenesis was still lacking. Now, Parreno, Cavalli and colleagues, using a refined experimental model in the fruit fly Drosophila melanogaster, enforced the initiation of tumors solely by imposing a transient loss of Polycomb repression, leading to a purely epigenetic oncogenesis phenomenon. Despite a few caveats that we discuss, this pioneering work represents a major breakpoint in cancer research. We are led to consider the theoretical and conceptual implications on oncogenesis and to search for links between this artificial experimental model and naturally occurring processes, while revisiting cancer theories that were previously proposed as alternatives to the oncogene-centered paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f2f2c2f1a7eb766c5cfd45527bf643538142c9e" target='_blank'>
              Evidence of Epigenetic Oncogenesis: A Turning Point in Cancer Research.
              </a>
            </td>
          <td>
            Jean-Pascal Capp, Benoît Aliaga, Vera Pancaldi
          </td>
          <td>2024-11-21</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Despite the ability to induce strong clinical responses, therapies that target mutational drivers of oncogenic signaling are not curative in advanced cancers, as a subset of cancer cells is capable of surviving therapy and evolving resistance. While therapy resistance is commonly viewed as a cell-intrinsic phenomenon, multiple factors produced by tumors can also confer strong therapy resistance. In contrast to the detailed elucidation of molecular mediators of cell-intrinsic and stroma-mediated (SM) resistance, the relative importance of cell-intrinsic and SM resistance to in vivo therapeutic responses remains poorly defined. To address this gap of knowledge, we used a well-characterized experimental model of ALK+ lung cancer H3122 cell line that exemplifies a duality observed across common models of targetable cancers. Under standard stroma-free in vitro cultures, H3122 cells can survive under therapeutically relevant concentrations of ALK inhibitors (ALKi), eventually acquiring cell-intrinsic therapy resistance. At the same time, co-culture with stromal fibroblasts drastically reduces the sensitivity of H3122 cells to ALKi, indicating the relevance of SM resistance. To understand the impact of SM resistance on in vivo responses, we sought to identify the mechanism(s) responsible for the therapy-protective effects of stromal fibroblasts, subsequently interrogating the impact of the disruption of this mechanism on in vivo therapeutic responses. Our in vitro studies with a large panel of primary stromal fibroblast isolates identified the HGF-cMET axis as the major mechanism responsible for ALKi desensitization. Surprisingly, xenograft validation studies demonstrated a weak effect of HGF-cMET modulation. Histological analyses of tumor tissues revealed that this relative weakness is attributable to a strong, spatially limited HGF-independent component of SM resistance. Mechanistic follow-up demonstrated that the HGF-independent component of SM resistance integrates the effect of multiple well-known juxtacrine and paracrine-acting mechanisms. Whereas the multifactorial nature of stroma-mediated resistance prevented a clear assessment of its relative contribution to in vivo therapy responses, our spatial analyses indicate that SM resistance dominates the ability of tumors to avoid therapeutic elimination. Surprisingly, we found that SM resistance was also a substantial contributor to tumor relapse, indicating that in vivo therapy resistance can integrate both cell-intrinsic and cell-extrinsic effects. This duality of therapy resistance and the multifactorial underpinning of both cell-intrinsic and stroma-mediated resistance present a challenge to therapeutic strategies focused on identifying and targeting specific resistance mechanisms. Our studies suggest that this limitation can be overcome by shifting the therapeutic focus towards shared orthogonal therapeutic sensitivities of tumor cells within residual disease.
 Citation Format: Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara Henry, Viktoriya Marusyk, Chandler Gatenbee, Marilyn Bui, Jacob Scott, Philipp M. Altrock, Eric Haura, Alexander R.A. Anderson, David Basanta, Andriy Marusyk. Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943bc0d90e7f0f13d1f5d2038bd76c206372d768" target='_blank'>
              Abstract C035: Spatially limited stroma-mediated resistance potentiates targeted therapy resistance through an integration of multiple juxtacrine and paracrine mechanisms
              </a>
            </td>
          <td>
            Bina Desai, Tatiana Miti, Sandhya Prabhakaran, Daria Miroshnychenko, Menkara-Chinua N Henry, Viktoriya Marusyk, Chandler D. Gatenbee, Marilyn M Bui, Jacob G. Scott, Philipp M. Altrock, E. Haura, Alexander R. A. Anderson, D. Basanta, A. Marusyk
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90407b0bb1438492495e5a85f14d864b163e35fe" target='_blank'>
              Engineering structural variants to interrogate genome function.
              </a>
            </td>
          <td>
            Jonas Koeppel, Juliane Weller, T. Vanderstichele, Leopold Parts
          </td>
          <td>2024-11-12</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccf23b7479efffbb3dbb4f8c6a99c762eb0cdef2" target='_blank'>
              Multimodal scanning of genetic variants with base and prime editing.
              </a>
            </td>
          <td>
            Olivier Belli, Kyriaki Karava, Rick Farouni, Randall J. Platt
          </td>
          <td>2024-11-12</td>
          <td>Nature biotechnology</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="All forms of genetic variation originate from new mutations, making it crucial to understand their rates and mechanisms. Here, we use long-read PacBio sequencing to investigate de novo mutations that accumulated in 12 inbred mouse lines derived from three commonly used inbred strains (C3H, C57BL/6, and FVB) maintained for 8-15 generations in a mutation accumulation (MA) experiment. We built chromosome-level genome assemblies based on the MA line founders' genomes, and then employed a combination of read and assembly-based methods to call the complete spectrum of new mutations. On average, there are ~45 mutations per haploid genome per generation, about half of which (54%) are insertions and deletions shorter than 50 bp (indels). The remainder are single nucleotide mutations (SNMs, 44%) and large structural mutations (SMs, 2%). We found that the degree of DNA repetitiveness is positively correlated with SNM and indel rates, and that a substantial fraction of SMs can be explained by homology-dependent mechanisms associated with repeat sequences. Most (90%) indels can be attributed to microsatellite contractions and expansions, and there is a marked bias towards 4 bp indels. Among the different types of SMs, tandem repeat mutations have the highest mutation rate, followed by insertions of transposable elements (TEs). We uncover a rich landscape of active TEs, and notable differences in their spectrum among MA lines and strains, and a high rate of gene retroposition. Our study offers novel insights into mammalian genome evolution, and highlights the importance of repetitive elements in shaping genomic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4966070a9e9e2ca28b9c82696887c588bee4c86" target='_blank'>
              The rate and spectrum of new mutations in mice inferred by long-read sequencing.
              </a>
            </td>
          <td>
            Eugenio López-Cortegano, Jobran Chebib, Anika Jonas, Anastasia Vock, Sven Künzel, P. Keightley, D. Tautz
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="
 The concept of synthetic lethality has long held promise for the next generation of genetically targeted cancer therapies to follow the success of PARP inhibitors in the treatment of BRCA-mutant cancers. The classical definition of synthetic lethality arose in Drosophila with the observation that loss of either of two genes independently had little effect on cell viability but loss of both led to cell death. This definition has now been expanded to capture pairs of genes in which genetic alteration of one gene and pharmacological inhibition of the other leads to death of cancer cells while sparing the normal cells which lack the genetic alteration, leading to a broad therapeutic index. Functional genomics screening (using unbiased RNAi or CRISPR-based technology) has enabled the systematic discovery of novel synthetic lethal targets for drug discovery. The selective dependence of MTAP-deleted cells on PRMT5 is one of the strongest genetic interactions observed in early RNAi screens. Approximately 10-15% of all human cancer is MTAP-deleted, providing a large and diverse patient population. MTA-cooperative PRMT5 inhibitors, which bind preferentially in the presence of MTA, have been developed by scientists at Tango Therapeutics and elsewhere to leverage the synthetic lethal relationship between PRMT5 and MTAP loss. MTA-cooperative PRMT5 inhibitors inhibit PRMT5 in MTAP-deleted cancer cells and spare MTAP-expressing normal cells leading to clinically well-tolerated and efficacious therapies. MTA-cooperative PRMT5 inhibitors have broad therapeutic potential both as single agent and in combination with other therapies.
 Citation Format: Kimberly Briggs.Leveraging synthetic lethality for the development of novel cancer therapies. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539ee15239f2d3141b9b8d5ed5ea99f9aadfdf6f" target='_blank'>
              Abstract IA002: Leveraging synthetic lethality for the development of novel cancer therapies
              </a>
            </td>
          <td>
            Kimberly J Briggs
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="The emergence of drug resistance leading to cancer recurrence is one of the challenges in the treatment of cancer patients. Several mechanisms can lead to drug resistance, including epigenetic changes. Histone deacetylases (HDACs) play a key role in chromatin regulation through epigenetic mechanisms and are also involved in drug resistance. The control of histone acetylation and the accessibility of regulatory DNA sequences such as promoters, enhancers, and super-enhancers are known mechanisms by which HDACs influence gene expression. Other targets of HDACs that are not histones can also contribute to resistance. This review describes the contribution of HDACs to the mechanisms that, in some cases, may determine resistance to chemotherapy or other cancer treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4960e67938604ec9bdc889e3cde099b5e416cd0e" target='_blank'>
              HDAC-driven mechanisms in anticancer resistance: epigenetics and beyond
              </a>
            </td>
          <td>
            Martina Minisini, Martina Mascaro, Claudio Brancolini
          </td>
          <td>2024-11-20</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Gene therapy, which involves the delivery of genetic material into cells to correct an underlying genetic problem, has emerged as a promising approach for treating various conditions. To promote research in this rapidly evolving field, we developed the Gene Therapy Omnibus (GTO) (http://www.inbirg.com/gto/), a comprehensive resource containing detailed clinical trial data and molecular information related to gene therapy. The GTO includes 6333 clinical trial records and 3466 transcriptome profiles, with information on 614 altered genes and 22 types of gene therapy, including DNA therapies, RNA therapies and genetically-modified cell therapies. For each gene therapy product in a clinical trial, detailed information, such as altered gene name, structural components, indication, vector information, phase of the clinical trial, clinical outcomes and adverse effects, is provided when available. Additionally, 345 comparison datasets, including 29 single-cell RNA-sequencing datasets comprising information on both gene therapy and control samples, were established. Differential gene expression and downstream functional enrichment analyses were performed through standardized pipelines to elucidate the molecular alterations induced by gene therapy. The user-friendly interface of the GTO supports efficient data retrieval, visualization and analysis, making it an invaluable resource for researchers and clinicians performing clinical research on gene therapy and the underlying mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64873a6a2d0e61325df0c910d686a855ae83c490" target='_blank'>
              GTO: a comprehensive gene therapy omnibus.
              </a>
            </td>
          <td>
            Xuehang Meng, Yujia Du, Chang Liu, Zhaoyu Zhai, Jianbo Pan
          </td>
          <td>2024-11-14</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cell lines are the most common in-vitro models for the evaluation of anti-cancer drug sensitivities. Past studies have been conducted to decipher and characterize the pharmacogenomic feature of cell lines based on other omics data, such as genomic mutation data and whole-genome RNA sequencing (RNA-seq) profiles. In particular, proteomic data is also an essential component for the characterization of tumours. However, different from RNA-seq datasets rich in numerous transcriptome profiles of cancer cell lines and cell viability assay of drug responses, the pharmacogenomic protein quantifications are relatively scarce. With the availability of the recently enriched proteomic dataset ProCan-DepMapSanger, we systematically evaluated the interplays among genomic mutations, transcription, and protein expressions across cancer cell lines. In general, blood cancers have higher RNA-protein correlations than those in solid cancers. The differential expression analysis on protein data helped identify more expressional and functional impact of genomic mutations of cancer genes. We also integrated the proteomic map with drug molecular chemical features to construct a bi-modal machine learning model to infer the drug sensitivities of cancer cell lines. Our results demonstrated that protein quantifications can lead to better drug response prediction performance than the model trained on transcriptome profiles. In addition, integrating protein data with drug chemical features, represented as molecular graphs and learned by Graph Neural Network, outperformed the state-of-the-art model DeepOmicNet for drug response prediction in proteomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c89e8ade30b207677f590882fddb023a8cd320d3" target='_blank'>
              Drug Response Modeling across Cancers: Proteomics vs. Transcriptomics
              </a>
            </td>
          <td>
            Zetian Zheng, Lei Huang, Fuzhou Wang, Linjing Liu, Jixiang Yu, Weidun Xie, Xingjian Chen, Xiangtao Li, Ka-Chun Wong
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer.
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-02</td>
          <td>Cell research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Loss of function screens using shRNA (short hairpin RNA) and CRISPR (clustered regularly interspaced short palindromic repeats) are routinely used to identify genes that modulate responses of tumor cells to anti-cancer drugs. Here, by integrating GSEA (Gene Set Enrichment Analysis) and CMAP (Connectivity Map) analyses of multiple published shRNA screens, we identified a core set of pathways that affect responses to multiple drugs with diverse mechanisms of action. This suggests that these pathways represent “weak points” or “Achilles heels”, whose mild disturbance should make cancer cells vulnerable to a variety of treatments. These “weak points” include proteasome, protein synthesis, RNA splicing, RNA synthesis, cell cycle, Akt-mTOR, and tight junction-related pathways. Therefore, inhibitors of these pathways are expected to sensitize cancer cells to a variety of drugs. This hypothesis was tested by analyzing the diversity of drugs that synergize with FDA-approved inhibitors of the proteasome, RNA synthesis, and Akt-mTOR pathways. Indeed, the quantitative evaluation indicates that inhibitors of any of these signaling pathways can synergize with a more diverse set of pharmaceuticals, compared to compounds inhibiting targets distinct from the “weak points” pathways. Our findings described here imply that inhibitors of the “weak points” pathways should be considered as primary candidates in a search for synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68c29dcb0705107711d0d05473f868fd7dbb129" target='_blank'>
              Cancer Cell’s Achilles Heels: Considerations for Design of Anti-Cancer Drug Combinations
              </a>
            </td>
          <td>
            Valid Gahramanov, Frederick S. Vizeacoumar, Alain Morejon Morales, Keith Bonham, M. Sakharkar, Santosh Kumar, Franco J Vizeacoumar, A. Freywald, Michael Y. Sherman
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Sinonasal squamous cell carcinoma (SNSCC) is a rare and aggressive malignancy with limited treatment options, necessitating comprehensive molecular characterization to uncover actionable therapeutic targets. Here, we integrate bulk and single-cell transcriptomics, epigenomics, and DNA methylation profiling to construct a detailed molecular and cellular landscape of SNSCC. Our analysis identifies five transcriptionally and epigenetically distinct malignant subpopulations, including hypoxic (TC1) and proliferative (TC2) clusters, with TC1 showing significant association with poor clinical outcomes. We uncover a hypoxia-driven angiogenic axis, wherein TC1 cells secrete adrenomedullin (ADM), MIF, and VEGFA to activate endothelial tip cells, fostering tumor angiogenesis. Epigenetic reprogramming, characterized by alterations in DNA methylation and chromatin accessibility, underpins these transcriptional programs, with AP-1 transcription factors emerging as key regulators. Notably, ADM expression is epigenetically controlled, correlates with advanced clinical stage, and predicts reduced survival in head and neck cancer patients. This multi-omic study highlights the tumor heterogeneity and microenvironmental interactions driving SNSCC progression, revealing epigenetically regulated pathways as promising therapeutic targets for future intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0f1bcee9ecccbc4aae5987d7312ce58acb94e80" target='_blank'>
              Multi-omic profiling identifies malignant subpopulations and a hypoxia-driven angiogenic axis as therapeutic vulnerabilities in sinonasal squamous cell carcinoma
              </a>
            </td>
          <td>
            Chaelin You, Jaewoo Park, Jung Yeon Jang, Myeong S. Yu, Yoo-Sam Chung, Keunsoo Kang, Jihwan Park, Ji Heui Kim, Kyuho Kang
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite surgical resection, rigorous endoscopic surveillance, and immunotherapy with the Bacillus Calmette–Guérin (BCG) vaccine, 30% of high-risk bladder cancers recur, and 10% result in fatal outcomes within two years of diagnosis. The global shortage of BCG underscores the urgent need for alternative or complementary therapeutic strategies. To address this, we integrated transcriptomic profiling and targeted genomic sequencing to identify four consensus intrinsic subtypes of bladder cancer. Initially derived from bulk RNA profiling, these subtypes were further validated at the cellular and tissue-compartment levels using single-cell RNA sequencing and spatial transcriptomics. Notably, we identified a subtype of inflamed tumors with enhanced endogenous retroelement expression and increased commensal bacterial presence, which showed the highest responsiveness to BCG therapy. Additionally, we developed a machine learning-based model incorporating composite molecular features to predict recurrence risk, achieving a high accuracy (AUC = 0.90). Our findings establish a molecular precision framework for bladder cancer and nominate novel therapeutic targets to reduce reliance on BCG immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f0d9394677cb2bb295a5f2fe96a5d0c0fc0b84" target='_blank'>
              Genomic and transcriptomic profiling of high-risk bladder cancer reveals diverse molecular and microenvironment ecosystems
              </a>
            </td>
          <td>
            K. Meghani, Yanni Yu, N. Frydenlund, Erik Z. Li, B. Choy, S. Abdulkadir, J. Meeks
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Inactivation of disease alleles by allele-specific editing is a promising approach to treat dominant-negative genetic disorders, provided the causative gene is haplo-sufficient. We previously edited a dominant NEFL missense mutation with inactivating frameshifts and rescued disease-relevant phenotypes in induced pluripotent stem cell (iPSC)-derived motor neurons. However, a multitude of different NEFL missense mutations cause disease. Here, we addressed this challenge by targeting common single-nucleotide polymorphisms in cis with NEFL disease mutations for gene excision. We validated this haplotype editing approach for two different missense mutations and demonstrated its therapeutic potential in iPSC-motor neurons. Surprisingly, our analysis revealed that gene inversion, a frequent byproduct of excision editing, failed to reliably disrupt mutant allele expression. We deployed alternative strategies and novel molecular assays to increase therapeutic editing outcomes while maintaining specificity for the mutant allele. Finally, population genetics analysis demonstrated the power of haplotype editing to enable therapeutic development for the greatest number of patients. Our data serve as an important case study for many dominant genetic disorders amenable to this approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f234bdb78eaa4af9b83bb4b5e5fe79a48274ca" target='_blank'>
              Haplotype editing with CRISPR/Cas9 as a therapeutic approach for dominant-negative missense mutations in NEFL
              </a>
            </td>
          <td>
            Poorvi H. Dua, Bazilco M. J. Simon, Chiara B.E. Marley, Carissa M. Feliciano, Hannah L. Watry, Dylan Steury, Abin Abraham, Erin N. Gilbertson, Grace D Ramey, John A. Capra, Bruce R. Conklin, Luke M. Judge
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary KMT2A partial tandem duplication (PTD) involves intragenic duplications within the KMT2A gene and has been associated with acute myeloid leukemia (AML) and myelodysplastic syndrome (MDS). KMT2A PTD cannot be detected by conventional karyotyping or fluorescence in situ hybridization (FISH). In this study, we used optical genome mapping (OGM) to analyze the cytogenomic alterations in 1277 hematolymphoid neoplasms and identified KMT2A PTD exclusively in patients with myeloid neoplasms, including 35 (7%) with AML, 5 (2.2%) with MDS, and 5 (7.2%) with chronic myelomonocytic leukemia (CMML). Neoplasms with KMT2A PTD frequently exhibit a normal or non-complex karyotype and consistently harbor gene mutations involving epigenetic regulators, the FLT3/RAS signaling pathway, transcription factors, and spliceosome genes. Patients with KMT2A PTD are generally resistant to conventional chemotherapy, with the exception of those with de novo AML, which demonstrates a high remission rate. Patients with KMT2A PTD positive secondary AML and MDS had a poor outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce3f21aa3468dded682d3ad1d494e6eb3d9bfdb" target='_blank'>
              Detection of KMT2A Partial Tandem Duplication by Optical Genome Mapping in Myeloid Neoplasms: Associated Cytogenetics, Gene Mutations, Treatment Responses, and Patient Outcomes
              </a>
            </td>
          <td>
            Qing Wei, Shimin Hu, Jie Xu, Sanam Loghavi, Naval G Daver, G. Toruner, Wei Wang, L. J. Medeiros, Guilin Tang
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="KRAS is one of the most frequently mutated oncogenes in cancer and is involved in various tumorigenic processes. Among its diverse roles, KRAS mutations have been shown to upregulate macropinocytosis, a form of endocytosis that allows cells to engulf extracellular fluid and its contained solutes, subsequently supporting the increased need for cancer cells' growth and proliferation. The nutrient uptake function of macropinocytosis was initially described within the framework of KRAS-driven pancreatic tumors. Although constitutive macropinocytosis can be induced by activating mutations of proteins that are commonly found in cancer, KRAS-induced macropinocytosis constitutes a fundamental area of research due to the high occurrence of mutated KRAS in cancer (~1/3 of all cancers).  Furthermore, KRAS-mutated macropinocytosis not only contributes to tumorigenesis but also plays a critical role in developing resistance to treatments, as it was found to be implicated in Multi-Drug Resistance (MDR) in cancer cells. This mini-review aims to synthesize current knowledge of mechanisms of KRAS-mutated macropinocytosis briefly and examines the relationship between KRAS and macropinocytosis in the light of its role in cancer progression and drug resistance, highlighting therapeutic implications, targeting potential vulnerabilities and outlining clinical advancements in pertinent therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4b47f4761d206b1edf858bd1fc0d092897567" target='_blank'>
              The Role of KRAS in Macropinocytosis and its Implications of Multi-Drug Resistance in Cancer
              </a>
            </td>
          <td>
            F. Fenyvesi, Zeinab Ibrahim
          </td>
          <td>2024-12-23</td>
          <td>De Remediis</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Evolutionary turnover in non-coding regions has driven phenotypic divergence during past speciation events and continues to facilitate environmental adaptation through variants. We used a deep learning model to identify the substrates of regulatory turnover using genome wide mutations mimicking three evolutionary pathways: recent history (human-chimp substitutions), modern population (human population variation), and mutational susceptibility (random mutations). We observed enhancer turnover in approximately 6% of the whole genome, with more than 80% of the novel activity arising from repurposing of enhancers between cell-types. Frequency of turnover in a cell-type is remarkably similar across the three pathways, despite only ∼19% overlap in the source regions. The enhancers predisposed to turnover display reduced evolutionary constraints and are depleted in GWAS variants. Most of these occur within 100kb of a gene, with the highest turnover occurring near neurodevelopmental genes including CNTNAP2, NPAS3, and AUTS2. Flanking enhancers of these genes undergo high turnover irrespective of the mutational model pathway, suggesting a high plasticity in neurocognitive evolution. Based on susceptibility to random mutations, these enhancers were identified as vulnerable by nature and feature a higher abundance of cell type-specific transcription factor binding sites (TFBSs). Our findings suggest that enhancer repurposing within vulnerable loci drives regulatory innovation while keeping the core regulatory networks intact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bd45453df9a37a6cefa13a9308dee594ffe80f7" target='_blank'>
              Regulatory plasticity of the human genome
              </a>
            </td>
          <td>
            Jaya Srivastava, Ivan Ovcharenko
          </td>
          <td>2024-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tubo-ovarian high-grade serous carcinoma (HGSC) is the most lethal gynecological malignancy and frequently responds to platinum-based chemotherapy because of common genetic and somatic impairment of DNA damage repair (DDR) pathways. The mechanisms of clinical platinum resistance are diverse and poorly molecularly defined. Consequently, there are no biomarkers or medicines that improve patient outcomes. Herein we use single cell mass cytometry (CyTOF) to systematically evaluate the phosphorylation and abundance of proteins known to participate in the DNA damage response (DDR). Single cell analyses of highly characterized HGSC cell lines that phenocopy human patients show that cells with comparable levels of intranuclear platinum, a proxy for carboplatin uptake, undergo different cell fates. Unsupervised analyses revealed a continuum of DDR responses. Decompositional methods were used to identify eight distinct protein modules of carboplatin resistance and sensitivity at single cell resolution. CyTOF profiling of primary and secondary platinum-resistance patient models shows that a complex DDR sensitivity module is strongly associated with response, suggesting it as a potential tool to clinically characterize complex drug resistance phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1700b92ee74968435b1e4e6c96fff15f73c54ccf" target='_blank'>
              Coordinated protein modules define DNA damage responses to carboplatin at single cell resolution in human ovarian carcinoma models
              </a>
            </td>
          <td>
            Jacob S. Bedia, Ying-Wen Huang, Antonio Delgado Gonzalez, Veronica D. Gonzalez, I. Funingana, Zainab Rahil, Alyssa Mike, Alexis Lowber, M. Vias, Alan Ashworth, James D. Brenton, W. Fantl
          </td>
          <td>2024-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Targeted therapies like Venetoclax have increased the options available to acute myeloid leukemia (AML) patients, but survival remains poor due to drug resistance and disease relapse. We found that the translation initiation factor EIF4A1, which unwinds complex mRNA structures in the 5’ UTR of oncogenic transcripts, is highly expressed in AML stem- and progenitor-like cells. Inhibiting eIF4A with the small molecule Zotatifin reduces translation of transcripts related to the cell cycle and survival. This results in downregulation of AKT, STAT-5, and MCL-1 and underlies synergy of Zotatifin with Venetoclax. The drug combination promotes apoptosis across AML genotypes, while the effect on healthy blood cells is limited. Using in vivo relapsed and refractory AML patient-derived xenograft models, the combination significantly suppressed tumor burden and prolonged survival of xenografted mice. These results support eIF4A-mediated protein translation as a therapeutic target in AML. SIGNIFICANCE Despite advances in targeted therapies, the 5-year survival rate for acute myeloid leukemia remains around 32%. The efficacy of existing treatments may be improved with the addition of Zotatifin, an inhibitor targeting the translation initiation factor eIF4A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8672d4c71b8c253c15ee73780df3c698e781744" target='_blank'>
              Research Brief: Translation Initiation Represents an Acute Myeloid Leukemia Cell Vulnerability That Can Be Co-Targeted With BCL-2 Inhibition
              </a>
            </td>
          <td>
            Y. Lee, Jonathan D. Good, Noha Awais, Benjamin K. Eschle, Maia S. Lineberry, Jessica L. Teo, Sarah Bibeau, Morgan Wollman, George Karadimov, Nurefsan Sariipek, Jean Acosta, Jenny Noel, David Cucchi, E. DePasquale, Prafulla Gokhale, Fadi J. Najm, Richard M. Stone, Jacqueline S. Garcia, Andrew A. Lane, Peter van Galen
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Drug resistance is a major obstacle to the successful treatment of non-small cell lung cancer (NSCLC), the most common subtype of lung cancer. The purpose of this research was to look at possible treatment approaches and the processes behind drug resistance in non-small cell lung cancer.
Methods: From March 2022 to February 2024, a prospective observational research was carried out at Lady Reading Hospital in Peshawar. 128 NSCLC patients in all were enrolled. Next-generation sequencing, DNA methylation tests, and immunological profiling were used to examine the genetic, epigenetic, and environmental components linked to medication resistance. Multivariate logistic regression, t-tests, chi-square tests, and Kaplan-Meier survival analysis were among the statistical techniques used.
Results: T790M mutations contributed to resistance, with EGFR mutations found in 35% of patients. Ten percent of patients had MET amplifications, while seven percent of patients had ALK rearrangements. In resistant cancers, epigenetic changes such as hypoacetylation of histones and hypermethylation of CDKN2A were common. Immunosuppressive cells and cancer-associated fibroblasts were abundant in the tumor microenvironment. The median overall survival (OS) of patients with drug-resistant NSCLC was 14 months, a substantial decrease from the 22 months seen in patients without resistance (p < 0.001).
Conclusions: The research emphasizes how genetic, epigenetic, and microenvironmental variables all play a role in the complex process of medication resistance in non-small cell lung cancer. Molecular profiling and targeted medicines along with personalized treatment techniques are critical to overcome resistance and enhancing patient outcomes. Larger, more varied populations and the creation of cutting-edge treatment approaches should be the main topics of future study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96863261655834ce5a911e129e2f5a3a8ee64f44" target='_blank'>
              Understanding the Mechanisms of Drug Resistance in Non-Small Cell Lung Cancer Potential Therapeutic Approaches
              </a>
            </td>
          <td>
            Afnan Ali Shah, Waqar Ahmad Tahir, Owais Nawaz Khan, A. Abid, Yasar Umar Khan, Talha Mazhar
          </td>
          <td>2024-12-31</td>
          <td>Innovative Research in Applied, Biological and Chemical Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Somatic genomic rearrangements are hallmarks of cancer. Complex genomic rearrangements (CGRs) involving multiple intertwined structural alterations are often present in tumor genomes. CGRs frequently harbor oncogenic drivers, but their genomic architectures and etiologies are poorly understood. We used deep-coverage optical mapping technology to profile the genomic landscapes of normal lung tissues, benign pulmonary lesions, carcinoma in situ, and advanced carcinomas to examine the patterns of genome disorganization and instability in different stages of carcinogenesis in lung. Large rearrangements and CGRs were prevalent in the carcinomas. We developed omcplR to resolve the architecture of CGRs and predict their genesis from optical mapping data using genome-graph concept. We found that CGRs often arose from hierarchical combinations of multiple, localized simple structural variations, and harbored allelic heterogeneity at the affected loci. Rearrangement patterns and associated genomic signatures suggested that chromoanasynthesis was a likely prevalent mechanism driving complex genomic rearrangements. The early rearrangement junctions in intra-chromosomal CGRs were usually localized within the same chromatin domains, but the late junctions in advanced tumors had more heterogeneous contexts suggesting progressive organizational heterogeneity. The CGRs, especially the late events therein were under positive selection. A composite signature of genomic alterations including the CGRs captured the trajectory of progressive genomic disorganization and instability with carcinogenesis in lung and underscored the extent of genomic structural heterogeneity among the in-situ tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0b28163735146a5869b061b33efdd0283e9308" target='_blank'>
              Deconstructing evolutionary histories of complex genomic rearrangements in lung malignancies
              </a>
            </td>
          <td>
            Xiaoju Hu, Vignesh Venkat, Gregory R Riedlinger, Zhiyuan Shen, Jyoti Malhotra, Subhajyoti De
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aaea8ef96bcf757be9b672ce6a26fbe4a6d0c91" target='_blank'>
              Integration of single-cell sequencing and drug sensitivity profiling reveals an 11-gene prognostic model for liver cancer
              </a>
            </td>
          <td>
            Qunfang Zhou, Jingqiang Wu, Jiaxin Bei, Zixuan Zhai, Xiuzhen Chen, Wei Liang, Jing Meng, Mingyu Liu
          </td>
          <td>2024-11-25</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The P-glycoprotein efflux pump, encoded by the MDR1 gene, is an ATP-driven transporter capable of expelling a diverse array of compounds from cells. Overexpression of this protein is implicated in the multi-drug resistant phenotype observed in various cancers. Numerous studies have attempted to decipher the impact of genetic variants within MDR1 on P-glycoprotein expression, functional activity, and clinical outcomes in cancer patients. Among these, three specific single nucleotide polymorphisms—T1236C, T2677G, and T3435C - have been the focus of extensive research efforts, primarily through in vitro cell line models and clinical cohort analyses. However, the findings from these studies have been remarkably contradictory. In this study, we employ a computational, data-driven approach to systematically evaluate the effects of these three variants on principal stages of the gene expression process. Leveraging current knowledge of gene regulatory mechanisms, we elucidate potential mechanisms by which these variants could modulate P-glycoprotein levels and function. Our findings suggest that all three variants significantly change the mRNA folding in their vicinity. This change in mRNA structure is predicted to increase local translation elongation rates, but not to change the protein expression. Nonetheless, the increased translation rate near T3435C is predicted to affect the protein’s co-translational folding trajectory in the region of the second ATP binding domain. This potentially impacts P-glycoprotein conformation and function. Our study demonstrates the value of computational approaches in elucidating the functional consequences of genetic variants. This framework provides new insights into the molecular mechanisms of MDR1 variants and their potential impact on cancer prognosis and treatment resistance. Furthermore, we introduce an approach which can be systematically applied to identify mutations potentially affecting mRNA folding in pathology. We demonstrate the utility of this approach on both ClinVar and TCGA and identify hundreds of disease related variants that modify mRNA folding at essential positions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3513112218d83dd11f2ce2e7d98d7d9261c6c5e8" target='_blank'>
              Computational Analysis of MDR1 Variants Predicts Effect on Cancer Cells via their Effect on mRNA Folding
              </a>
            </td>
          <td>
            T. Gutman, T. Tuller
          </td>
          <td>2024-12-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Dysregulated epigenetic programs that restrict differentiation, reactivate fetal genes, and confer phenotypic plasticity are critical to colorectal cancer (CRC) development. By screening a small molecule library targeting epigenetic regulators using our dual reporter system, we found that inhibiting histone deacetylase (HDAC) 1/2 promotes CRC differentiation and anti-tumor activity. Comprehensive biochemical, chemical, and genetic experiments revealed that on-target blockade of the HDAC1/2 catalytic domain mediated the differentiated phenotype. Unbiased profiling of histone posttranslational modifications induced by HDAC1/2 inhibition nominated acetylation of specific histone lysine residues as potential regulators of differentiation. Genome-wide assessment of implicated marks indicated that H3K27ac gains at HDAC1/2-bound regions associated with open chromatin and upregulation of differentiation genes upon HDAC1/2 inhibition. Disrupting H3K27ac by degrading acetyltransferase EP300 rescued HDAC1/2 inhibitor-mediated differentiation of a patient-derived CRC model using single cell RNA-sequencing. Genetic screens revealed that DAPK3 contributes to CRC differentiation induced by HDAC1/2 inhibition. These results highlight the importance of specific chemically targetable histone modifications in governing cancer cell states and epigenetic reprogramming as a therapeutic strategy in CRC. BRIEF SUMMARY HDAC1/2 inhibition promotes colorectal cancer differentiation via gains in H3K27ac, which can be reversed by blocking its acetyltransferase EP300.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207eb16a60f11154e81c963033e896c8385daaa7" target='_blank'>
              Chemical perturbations impacting histone acetylation govern colorectal cancer differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Akhouri Kishore Raghawan, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, P. Sahgal, Daulet Aitymbayev, Jennifer S. Thalappillil, Malvina Papanastasiou, William Hawkins, Steven A. Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Understanding the molecular pathogenesis of MLL fusion oncoprotein (MLL-FP) leukaemia has spawned epigenetic therapies that have improved clinical outcomes in this often-incurable disease. Using genetic and pharmacological approaches, we define the individual and combined contribution of KAT6A, KAT6B and KAT7, in MLL-FP leukaemia. Whilst inhibition of KAT6A/B is efficacious in some pre-clinical models, simultaneous targeting of KAT7, with the novel inhibitor PF-9363, increases the therapeutic efficacy. KAT7 interacts with Menin and the MLL complex and is co-localised at chromatin to co-regulate the MLL-FP transcriptional program. Inhibition of KAT6/KAT7 provides an orthogonal route to targeting Menin to disable the transcriptional activity of MLL-FP. Consequently, combined inhibition rapidly evicts the MLL-FP from chromatin, potently represses oncogenic transcription and overcomes primary resistance to Menin inhibitors. Moreover, PF-9363 or genetic depletion of KAT7 can also overcome acquired genetic/non-genetic resistance to Menin inhibition. These data provide the molecular rationale for rapid clinical translation of combination therapy in MLL-FP leukaemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99055bf5d30190642cc04c1d1e10700ca75334b" target='_blank'>
              Catalytic inhibition of KAT6/KAT7 enhances the efficacy and overcomes primary and acquired resistance to Menin inhibitors in MLL leukaemia
              </a>
            </td>
          <td>
            Shellaina J. V. Gordon, Florian Perner, Laura MacPherson, Daniela V. Wenge, Wallace Bourgeois, Katie Fennell, Tabea Klaus, Jelena Petrovic, Jakub Horvath, Joan Cao, John J. Lapek, Sean Uryu, Jeffrey White, E. Lam, X. Mu, Yih-Chih Chan, Andrea Gillespie, Benjamin Blyth, M. Camerino, Ylyva E. Bozikis, Henrietta Holze, Kathy Knezevic, Jesse J Balic, P. Stupple, Ian P. Street, B. Monahan, Shikhar Sharma, Elanor N. Wainwright, Dane Vassiliadis, Thomas A. Paul, Scott A. Armstrong, Mark A Dawson
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc4ce21cb54a65498b2c72289ec6bd7eb2e7dc62" target='_blank'>
              Characterization of shared neoantigens landscape in Mismatch Repair Deficient Endometrial Cancer
              </a>
            </td>
          <td>
            E. De Paolis, C. Nero, E. Micarelli, Guido Leoni, A. Piermattei, R. Trozzi, Elisa Scarselli, A. D’Alise, Luciano Giacò, M. De Bonis, A. Preziosi, Gennaro Daniele, Diletta Piana, T. Pasciuto, Gianfranco Zannoni, Angelo Minucci, G. Scambia, Andrea Urbani, Francesco Fanfani
          </td>
          <td>2024-12-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Lung cancer is the leading cause of cancer mortality, and despite improvements in treatment, tumors typically respond incompletely and resume growth after acquisition of drug resistance. Recent completion of a neoadjuvant osimertinib Phase II trial treating patients with surgically resectable stage I-IIIA EGFR-mutated non-small cell lung cancer (EGFRm NSCLC) (NCT03433469) (Blakely et. Al, JCO 2024) has highlighted the importance of further identifying non-genomic mechanisms of persistence and resistance to targeted therapy. Here we analyzed these patient samples to identify transcriptionally regulated signaling patterns enriched at Residual Disease (RD) after neoadjuvant osimertinib treatment compared to treatment naïve samples (TN), as well as signaling patterns enriched at RD in patients who ultimately develop disease recurrence after surgical resection compared to those who remain disease-free. Spatially resolved transcriptomic sequencing was performed, using the 10X Genomics Visium platform, on 36 tissue sections (n=9 TN, n=19 RD, n=4 Progressive Disease (PD), n=4 Tumor Adjacent Normal), from 25 patients. TN and PD samples were from standard-of-care surgical resections. 18 of 19 RD samples were from NCT03433469. The Visium array spots that tile the sequencing capture area are 55 µm in diameter, and thus can contain an estimated 1-10 cells within each array spot. After quality control, 91,582 array spots remained for downstream analysis. We identified high confidence tumor array spots annotated both as “Cancer” by a board-certified pathologist and with tumor characteristic copy number variations. Through dimensionality reduction clustering, we identified 10 major array spot clusters, including the high confidence tumor array spot cluster. These clusters were further subset into 44 high resolution clusters of array spot niches, 32 of which were shared across at least nine patients. Because array spots are not single-cell resolution, these niches represent spots with similar mixed cell type composition and transcriptional signatures. We identified significant increases in the CCL14+ Endothelial Cell Niche, CXCL2+ Endothelial Cell Niche, Lymphocytes + Endothelial Niche, and CXCL12+ Fibroblast Niche in RD samples compared to TN. We identified a cell signaling program defined by spatially co-expressed ligand-receptor pairs associated with focal adhesion, enriched in PD samples. This same pattern was enriched in RD samples from patients who had recurrent disease after surgical resection compared to RD samples from patients who remained disease free. These data describe an increase in immune trafficking in RD samples, as well as identify an EGFR- alternative signaling pathway correlating with osimertinib resistance. Identification of spatially resolved, differential architectural organization and signaling patterns in RD and PD states will expand our understanding of targeted therapy resistance in EGFRm NSCLC, with the potential to identify additional clinically actionable targets.
 Citation Format: Whitney Tamaki, Daniel L. Kerr, Wei Wu, Grant Eilers, Anatoly Urisman, Yu-Ting Chou, Philippe Gui, Shigeki Nanjo, Johannes R. Kratz, David M. Jablons, Trever G. Bivona, Collin M. Blakely. Identification of cell-cell signaling programs across the tumor-tumor microenvironment ecosystem and associated with clinical status in neoadjuvant osimertinib treated patient samples revealed by spatial profiling [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00831e7253b0b6e70e43b596e9e7ff252fbadae" target='_blank'>
              Abstract C025: Identification of cell-cell signaling programs across the tumor-tumor microenvironment ecosystem and associated with clinical status in neoadjuvant osimertinib treated patient samples revealed by spatial profiling
              </a>
            </td>
          <td>
            W. Tamaki, D. Kerr, Wei Wu, Grant Eilers, A. Urisman, Yu-Ting Chou, Philippe Gui, Shigeki Nanjo, J. Kratz, D. Jablons, T. Bivona, Collin M Blakely
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Genetic studies of haematological cancers have pointed out the heterogeneity of leukaemia in its different subpopulations, with distinct mutations and characteristics, impacting the treatment response. Next-generation sequencing (NGS) and genome-wide analyses, as well as single-cell technologies, have offered unprecedented insights into the clonal heterogeneity within the same tumour. A key component of this heterogeneity that remains unexplored is the intracellular metabolome, a dynamic network that determines cell functions, signalling, epigenome regulation, immunity and inflammation. Understanding the metabolic diversities among cancer cells and their surrounding environments is therefore essential in unravelling the complexities of leukaemia and improving therapeutic strategies. Here, we describe the currently available methodologies and approaches to addressing the dynamic heterogeneity of leukaemia progression. In the second section, we focus on metabolic leukaemic vulnerabilities in acute myeloid leukaemia (AML) and acute lymphoblastic leukaemia (ALL). Lastly, we provide a comprehensive overview of the most interesting clinical trials designed to target these metabolic dependencies, highlighting their potential to advance therapeutic strategies in leukaemia treatment. The integration of multi-omics data for cancer identification with the metabolic states of tumour cells will enable a comprehensive “micro-to-macro” approach for the refinement of clinical practices and delivery of personalised therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cc7b64900de2dfae5390d10959b2a1e62e8bf1" target='_blank'>
              Unlocking the Heterogeneity in Acute Leukaemia: Dissection of Clonal Architecture and Metabolic Properties for Clinical Interventions
              </a>
            </td>
          <td>
            Martina Maria Capelletti, Orsola Montini, Emilio Ruini, Sarah Tettamanti, Angela Maria Savino, J. Sarno
          </td>
          <td>2024-12-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="BACKGROUND
Structural variation (SV), defined as balanced and unbalanced chromosomal rearrangements >1 kb, is a major contributor to germline and neoplastic disease. Large variants have historically been evaluated by chromosome analysis and now are commonly recognized by chromosomal microarray analysis (CMA). The increasing application of genome sequencing (GS) in the clinic and the relatively high incidence of chromosomal abnormalities in sick newborns and children highlights the need for accurate SV interpretation and reporting. In this review, we describe SV patterns of common cytogenetic abnormalities for laboratorians who review GS data.


CONTENT
GS has the potential to detect diverse chromosomal abnormalities and sequence breakpoint junctions to clarify variant structure. No single GS analysis pipeline can detect all SV, and visualization of sequence data is crucial to recognize specific patterns. Here we describe genomic signatures of translocations, inverted duplications adjacent to terminal deletions, recombinant chromosomes, marker chromosomes, ring chromosomes, isodicentric and isochromosomes, and mosaic aneuploidy. Distinguishing these more complex abnormalities from simple deletions and duplications is critical for phenotypic interpretation and recurrence risk recommendations.


SUMMARY
Unlike single-nucleotide variant calling, identification of chromosome rearrangements by GS requires further processing and multiple callers. SV databases have caveats and limitations depending on the platform (CMA vs sequencing) and resolution (exome vs genome). In the rapidly evolving era of clinical genomics, where a single test can identify both sequence and structural variants, optimal patient care stems from the integration of molecular and cytogenetic expertise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a2a826f1810ca4839038af6a42ce3daef01b34" target='_blank'>
              Structural Variation Interpretation in the Genome Sequencing Era: Lessons from Cytogenetics.
              </a>
            </td>
          <td>
            Lucilla Pizzo, M. K. Rudd
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-risk multiple myeloma (MM) is genomically unstable, comprised of heterogeneous populations of tumor cells that evolve over time. Light chain escape (LCE) is a clinical phenomenon observed when light chains rise separately from M-spike values, implying divergent tumor evolution. We sought to understand LCE by performing high depth transcriptomic and phenotypic studies. We performed single cell RNA sequencing and ex vivo drug sensitivity profiling on serial bone marrow biopsies from a patient with LCE at diagnosis, first relapse and relapsed/refractory timepoints. Single cell RNA sequencing uncovered distinct transcriptomic subpopulations with phenotypes that could be tracked separately by clinical serum light chain and M-spike values. Genes differentially expressed between subpopulations were assessed for generalizable effects on prognosis from the MMRF CoMMpass and GSE24080 datasets. Notably, the LCE subpopulation exhibited gene expression profile featuring prominent LAMP5-overexpression, which was associated with risk for osteolytic bone lesions. Ex vivo drug sensitivity testing displayed differential sensitivity of the subpopulations. Copy Number Variant inference showed that the transcriptomic subpopulation underlying LCE was related to a genetic subclone that evolved over time. Our findings illustrate that malignant subpopulations underly light chain escape in multiple myeloma. These studies implied that LCE and LAMP5 gene overexpression portends for increased risk of osteolytic bone disease and adverse prognosis, findings that were confirmed in the subset of patients in the CoMMpass database with light chain escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e33173187a2b1d3d0f9422813ed4bc1d3b6c7d8" target='_blank'>
              Single Cell RNA Sequencing Before and After Light Chain Escape Reveals Intra-patient Multiple Myeloma Subpopulations with Divergent Osteolytic Gene Expression.
              </a>
            </td>
          <td>
            Denis Ohlstrom, Z. Walker, Abhishek Pandey, Lorraine N Davis, Krysta Engel, Zenggang Pan, Peter A Forsberg, T. Mark, Austin E. Gillen, Daniel W. Sherbenou
          </td>
          <td>2024-12-19</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb40aeff67c4162245cfc687f8adca56b59144a" target='_blank'>
              Integrated molecular and functional characterization of the intrinsic apoptotic machinery identifies therapeutic vulnerabilities in glioma
              </a>
            </td>
          <td>
            E. Fernández, Wilson X Mai, Kai Song, Nicholas A. Bayley, Jiyoon Kim, Henan Zhu, Marissa S Pioso, Pauline Young, Cassidy Andrasz, Dimitri Cadet, Linda M Liau, Gang Li, William Yong, Fausto J Rodriguez, Scott J. Dixon, Andrew J Souers, Jingyi Jessica Li, Thomas Graeber, T. Cloughesy, David A. Nathanson
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Patients with epidermal growth factor receptor mutant nonsmall cell lung cancer (NSCLC) often fail to treat gefitinib because of secondary drug resistance. The development of tumor drug resistance is closely related to variations in cancer cell metabolism. Single-cell metabolomics analysis can provide unique information about tumor drug resistance. Herein, we constructed a platform to study the secondary resistance of tumor cells based on single-cell metabolomics (sSRTC-scM). A gefitinib-resistant NSCLC cell line (PC9GR) was constructed by increasing the dose step by step. The metabolic profiles of parental PC9 cells and PC9GR cells with different drug resistance levels were detected by intact living-cell electrolaunching ionization mass spectrometry at the single-cell level. The data were analyzed by statistical methods such as t-SNE, variance, volcano plot, heat map, and metabolic pathway analysis. Using this platform, we found that the metabolic fingerprints of PC9GR cells can evaluate drug resistance degrees. The metabolic fingerprints continue to be altered with the increase of drug resistance. We revealed 19 metabolic markers of secondary resistance by variance analysis and clarified that the glycerophospholipid metabolic pathway of PC9GR cells changed significantly. In addition, we found that with the increase in drug resistance levels, the heterogeneity of single-cell metabolism became greater and the number of cells with weak drug resistance gradually decreased. This phenomenon can be utilized to illustrate the drug resistance degrees of PC9GR cells. This study provides diagnostic markers for evaluating the drug resistance of tumors and gives new insight into overcoming the secondary resistance of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84899e62659612e5b6164a21695dadcc8005bf4e" target='_blank'>
              Single-Cell Mass Spectrometry Studies of Secondary Drug Resistance of Tumor Cells.
              </a>
            </td>
          <td>
            Guizhen Zhu, Wenmei Zhang, Yaoyao Zhao, Guangyun Wang, Hanyu Yuan, Guangsheng Guo, Xiayan Wang
          </td>
          <td>2024-12-20</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88d1d3e327b2658624144e333e3ed0c37f8a284" target='_blank'>
              CRISPR/Cas9-induced double-strand breaks in the huntingtin locus lead to CAG repeat contraction through DNA end resection and homology-mediated repair
              </a>
            </td>
          <td>
            P. Śledziński, Mateusz Nowaczyk, Marianna Śmiełowska, M. Olejniczak
          </td>
          <td>2024-12-03</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The MCF-7R breast cancer cell line, developed by treating the parental MCF-7 cells with increasing doses of doxorubicin, serves as a model for studying acquired multidrug resistance (MDR). MDR is a major challenge in cancer therapy, often driven by overexpression of the efflux pump P-glycoprotein (P-gp) and epigenetic modifications. While many P-gp inhibitors show promise in vitro, their nonspecific effects on the efflux pump limit in vivo application. Curcumin, a natural compound with pleiotropic action, is a nontoxic P-gp inhibitor capable of modulating multiple pathways. To explore curcumin’s molecular effects on MCF-7R cells, we analyzed the expression of genes involved in DNA methylation and transcription regulation, including ABCB1/MDR1. Reduced representation bisulfite sequencing further unveiled key epigenetic changes induced by curcumin. Our findings indicate that curcumin treatment not only modulates critical cellular processes, such as ribosome biogenesis and cytoskeletal dynamics, but also reverses the resistant phenotype, toward that of sensitive cells. This study highlights curcumin’s potential as an adjuvant therapy to overcome chemoresistance, offering new avenues for pharmacological strategies targeting epigenetic regulation to re-sensitize resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a25fe3bb8a31c48e3c18c599af20c640c2363916" target='_blank'>
              Epigenetic and Cellular Reprogramming of Doxorubicin-Resistant MCF-7 Cells Treated with Curcumin
              </a>
            </td>
          <td>
            P. Poma, S. Rigogliuso, M. Labbozzetta, Aldo Nicosia, Salvatore Costa, M. Ragusa, M. Notarbartolo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Multiple myeloma (MM) is an incurable blood malignancy characterized by the clonal expansion of plasma cells and the secretion of monoclonal immunoglobulins. High-risk MM, defined by specific cytogenetic abnormalities, poses significant therapeutic challenges and is associated with inferior survival outcomes compared to standard-risk disease. Although molecularly targeted therapies have shown efficacy in other hematologic malignancies, currently venetoclax is the only targeted therapy approved for MM (t(11;14)). However, chromosome 1q gains, amplifications, and 1p deletions are frequently observed in MM, and have been linked to drug resistance and poor patient prognosis. Accordingly, this review focuses on emerging MM precision therapies capable of targeting dysregulated genes within these regions. It addresses gene therapies, small molecule inhibitors and monoclonal antibodies currently under investigation to antagonize oncogenic drivers including MCL-1, BCL9, F11R, and CKS1B, all of which are implicated in cell survival, proliferation or drug resistance. In conclusion, the link between chromosome 1 abnormalities and high-risk disease in MM patients offers a compelling rationale to identify and explore therapeutic targeting of chromosome 1 gene products as a novel precision medicine approach for a poorly served patient population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7a3813cb3c40b387f3627b9866b104fa1b1b388" target='_blank'>
              Chromosome 1 Alterations in Multiple Myeloma: Considerations for Precision Therapy.
              </a>
            </td>
          <td>
            Niamh McAuley, Izabela Cymer, R. McAvera, Ann M Hopkins, S. Glavey
          </td>
          <td>2024-12-04</td>
          <td>European journal of haematology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Ionizing radiotherapy (RT) is a widely used palliative and curative treatment strategy for malignancies. In solid tumors, RT-induced double strand breaks lead to the accumulation of indels, and their repair by non-homologous end-joining has been linked to the ID8 mutational signature in resistant cells. However, the extent of RT-induced DNA damage in hematologic malignancies and its impact on their evolution and interplay with commonly used chemotherapies has not yet been explored. Here, we interrogated 580 whole genome sequencing (WGS) from patients with large B-cell lymphoma, multiple myeloma, and myeloid neoplasms and identified ID8 only in relapsed disease. Yet, it was detected after exposure to both RT and mutagenic chemotherapy (i.e., platinum). Using WGS of single-cell colonies derived from treated lymphoma cells, we revealed a dose-response relationship between RT and platinum and ID8. Finally, using ID8 as a genomic barcode we demonstrate that a single RT-resistant cell may seed systemic relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7be6715a8b86cbf647052fda12a7d5d6ddae614" target='_blank'>
              Mutagenic impact and evolutionary influence of radiotherapy in hematologic malignancies
              </a>
            </td>
          <td>
            B. Diamond, Dhanvantri Chahar, M. Jain, A. Poos, Michael Durante, B. Ziccheddu, Marcella Kaddoura, M. Papadimitriou, K. Maclachlan, T. Jelínek, Faith Davies, N. Figura, Gareth Morgan, Elias K Mai, Katja C. Weisel, Roland Fenk, Marc-Steffen Raab, S. Usmani, O. Landgren, Frederick L. Locke, Hartmut Goldschmidt, Jonathan H. Schatz, Niels Weinhold, F. Maura
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ef6a160eddfbbad312082d26af8e03b167d27f" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance.
              </a>
            </td>
          <td>
            Michael J Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, Sarah R. Amend, James B. Hicks
          </td>
          <td>2024-11-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96442cd68e41fa673e831da90c37e37f248ca02b" target='_blank'>
              A multiomic atlas identifies a treatment-resistant, bone marrow progenitor-like cell population in T cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Jason Xu, Changya Chen, Jonathan H Sussman, S. Yoshimura, T. Vincent, P. Pölönen, Jianzhong Hu, S. Bandyopadhyay, Omar Elghawy, Wenbao Yu, Joseph S. Tumulty, Chia-Hui Chen, Elizabeth Li, Caroline Diorio, R. Shraim, H. Newman, Lahari Uppuluri, A. Li, Gregory M. Chen, David W Wu, Yang-yang Ding, Jessica Xu, Damjan Karanfilovski, Tristan Lim, Miles Hsu, Anusha Thadi, Kyung Jin Ahn, Chi-Yun Wu, Jacqueline Peng, Yusha Sun, Alice Wang, Rushabh Mehta, David Frank, Lauren Meyer, M. Loh, E. Raetz, Zhiguo Chen, Brent L Wood, M. Devidas, K. Dunsmore, S. Winter, Ti-Cheng Chang, Gang Wu, S. Pounds, Nancy R. Zhang, William L Carroll, S P Hunger, Kathrin M Bernt, Jun J Yang, C. Mullighan, Kai Tan, D. Teachey
          </td>
          <td>2024-11-25</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>103</td>
        </tr>

        <tr id="Background Cis-regulatory elements (CREs) are regions of DNA that regulate the expression of nearby genes. Changes in these elements can lead to phenotypic variations and adaptations in different populations. However, the regulatory dynamics underlying the local adaptation of traits remain poorly understood in Chinese and Western pigs. By comparing the chromatin accessibility profiles of skeletal muscle, liver, and fat between these two pig populations, we aimed to identify key regulatory elements that could explain phenotypic differences observed between the two groups. Results Our results revealed that the genome-wide chromatin accessibility profiles were largely similar at a qualitative level within tissues. However, we also identified local regions that exhibited quantitative differences, most of which occurred in liver tissue. Interestingly, we found that most of the increased chromatin accessibility in the livers of Chinese pigs was associated with tissue-specific openness. Furthermore, we observed a positive correlation between the ATAC-seq signal at the transcript start site (TSS) and the expression levels of nearby genes. Motif enrichment analysis revealed NR2F1 as a key regulator in Chinese pigs. Differentially expressed genes (DEGs) in Chinese pigs showed enrichment for NR2F1 response targets. One of the genes regulated by NR2F1 in Chinese pigs, NPC1, harbored a high alternative allelic frequency in the intron region. Conclusion Overall, our study provides valuable insights into the regulatory dynamics underlying phenotypic variation in pigs. By elucidating the role of CREs in driving phenotypic variation, we can better understand the genetic basis of complex traits and potentially identify targets for genetic improvement in livestock breeding programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22c996fba3282093d3837a50036a64e70bbb887" target='_blank'>
              Analysis of cis-regulatory changes underlying phenotype divergence shaped by domestication in pigs
              </a>
            </td>
          <td>
            Chunpeng Liu, Na Ao, Yuwen Liang, Tingting Ma, Qishan Wang, Zhen Wang, Fen Wu, Zhenyang Zhang, Yifei Fang, Minghui Wang, Yuchun Pan, Jing Fu
          </td>
          <td>2024-11-08</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Resistance to BRAF and MAPK inhibitors is a significant challenge in melanoma treatment, driven by adaptive and acquired mechanisms that allow tumour cells to evade therapy. Here, we examined early signalling responses to single and combined BRAF and MAPK inhibition in a BRAFV600E, drug-sensitive melanoma cell line and a drug-resistant ARID1A-knockout (KO) derivative. ARID1A, frequently mutated in melanoma, is associated with resistance and immune evasion. Using an innovative systems biology approach that integrates transcriptomics, proteomics, phosphoproteomics, and functional kinomics through matrix factorization and network analysis, we identified key signalling alterations and resistance mechanisms. We found that ARID1A-KO cells exhibited transcriptional rewiring, sustaining MAPK1/3 and JNK activity post-treatment, bypassing feedback sensitivity observed in parental cells. This rewiring suppressed PRKD1 activation, increased JUN activity—a central resistance network node—and disrupted PKC dynamics through elevated basal RTKs (e.g., EGFR, ROS1) and Ephrin receptor activity post-treatment. ARID1A mutations also reduced HLA-related protein expression and enriched extracellular matrix components, potentially limiting immune infiltration and reducing immunotherapy efficacy. Our graph-theoretical multi-omics approach uncovered novel resistance-associated signalling pathways, identifying PRKD1, JUN, and NCK1 as critical nodes. While receptor activation redundancies complicate single-target therapies, they also present opportunities for combination strategies. This study highlights ARID1A’s role in reshaping signalling and immune interactions, offering new insights into melanoma resistance mechanisms. By identifying actionable targets, including JUN and immune pathways, we provide a foundation for developing integrated therapeutic strategies to overcome resistance in BRAF/MAPK inhibitor-treated melanoma. One sentence summary This study reveals how ARID1A-mediated transcriptional rewiring drives resistance to MAPK inhibitors in melanoma by altering signalling pathways, immune interactions, and receptor dynamics, highlighting potential targets for combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bdb990bb1257653b7e587028f560dd7ef66b129" target='_blank'>
              ARID1A-induced transcriptional reprogramming rewires signalling responses to drug treatment in melanoma
              </a>
            </td>
          <td>
            Charlie George Barker, Sumana Sharma, Ana Mafalda Santos, Konstantinos-Stylianos Nikolakopoulos, Athanassios D. Velentzas, Franziska Völlmy, A. Minia, V. Pliaka, M. Altelaar, Gavin J Wright, Leonidas G. Alexopoulos, D. Stravopodis, E. Petsalaki
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d907c1f70a7e162db16222c621c3aec6a63202a" target='_blank'>
              Integrated single-cell analysis reveals heterogeneity and therapeutic insights in osteosarcoma
              </a>
            </td>
          <td>
            Dongan He, Xiaoqian Che, Haiming Zhang, Jiandong Guo, Lei Cai, Jian Li, Jinxi Zhang, Xin Jin, Jianfeng Wang
          </td>
          <td>2024-11-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033d9d641f5cc9ef741b9f061b7709ec5c750d88" target='_blank'>
              Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER.
              </a>
            </td>
          <td>
            Olivia Lucas, S. Ward, Rija Zaidi, Abigail Bunkum, A. Frankell, David A Moore, M. Hill, Wing Kin Liu, Daniele Marinelli, E. Lim, S. Hessey, C. Naceur-Lombardelli, A. Rowan, Sukhveer Kaur Purewal-Mann, Haoran Zhai, M. Dietzen, Boyue Ding, G. Royle, Sam Aparicio, N. Mcgranahan, M. Jamal-Hanjani, N. Kanu, C. Swanton, S. Zaccaria
          </td>
          <td>2024-11-29</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a hematological malignancy characterized by the clonal proliferation of malignant plasma cells within the bone marrow. The disease’s complexity is underpinned by a variety of genetic and molecular abnormalities that drive its progression. Methods This review was conducted through a state-of-The-art literature search, primarily utilizing PubMed to gather peer-reviewed articles. We focused on the most comprehensive and cited studies to ensure a thorough understanding of the genetic and molecular landscapes of MM. Results We detail primary and secondary alterations such as translocations, hyperdiploidy, single nucleotide variants (SNVs), copy number alterations (CNAs), gene fusions, epigenetic modifications, non-coding RNAs, germline predisposing variants, and the influence of the tumor microenvironment (TME). Our analysis highlights the heterogeneity of MM and the challenges it poses in treatment and prognosis, emphasizing the distinction between driver mutations, which actively contribute to oncogenesis, and passenger mutations, which arise due to genomic instability and do not contribute to disease progression. Conclusion & Future Perspectives We report key controversies and challenges in defining the genetic drivers of MM, and examine their implications for future therapeutic strategies. We discuss the importance of systems biology approaches in understanding the dependencies and interactions among these alterations, particularly highlighting the impact of double and triple-hit scenarios on disease outcomes. By advancing our understanding of the molecular drivers and their interactions, this review sets the stage for novel therapeutic targets and strategies, ultimately aiming to improve clinical outcomes in MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec433a1270b41378c5571d49631b7d920ddc502b" target='_blank'>
              The Genetic and Molecular Drivers of Multiple Myeloma: Current Insights, Clinical Implications, and the Path Forward
              </a>
            </td>
          <td>
            Meghana Ram, Molly Fraser, Júnia Vieira dos Santos, R. Tasakis, Ariana Islam, Jannah Abo-Donia, Samir Parekh, A. Laganà
          </td>
          <td>2024-12-01</td>
          <td>Pharmacogenomics and Personalized Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Structural variants (SVs) drive gene expression in the human brain and are causative of many neurological conditions. However, most existing genetic studies have been based on short-read sequencing methods, which capture fewer than half of the SVs present in any one individual. Long-read sequencing (LRS) enhances our ability to detect disease-associated and functionally relevant structural variants (SVs); however, its application in large-scale genomic studies has been limited by challenges in sample preparation and high costs. Here, we leverage a new scalable wet-lab protocol and computational pipeline for whole-genome Oxford Nanopore Technologies sequencing and apply it to neurologically normal control samples from the North American Brain Expression Consortium (NABEC) (European ancestry) and Human Brain Collection Core (HBCC) (African or African admixed ancestry) cohorts. Through this work, we present a publicly available long-read resource from 351 human brain samples (median N50: 27 Kbp and at an average depth of ∼40x genome coverage). We discover approximately 234,905 SVs and produce locally phased assemblies that cover 95% of all protein-coding genes in GRCh38. Utilizing matched expression datasets for these samples, we apply quantitative trait locus (QTL) analyses and identify SVs that impact gene expression in post-mortem frontal cortex brain tissue. Further, we determine haplotype- specific methylation signatures at millions of CpGs and, with this data, identify cis-acting SVs. In summary, these results highlight that large-scale LRS can identify complex regulatory mechanisms in the brain that were inaccessible using previous approaches. We believe this new resource provides a critical step toward understanding the biological effects of genetic variation in the human brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6820702487ac1cdc10f0d17762d1b83d53ed3d9f" target='_blank'>
              Long-read sequencing of hundreds of diverse brains provides insight into the impact of structural variation on gene expression and DNA methylation
              </a>
            </td>
          <td>
            Kimberley J. Billingsley, Melissa Meredith, K. Daida, P. Álvarez Jerez, S. Negi, L. Malik, Rylee Genner, Abraham Moller, Xinchang Zheng, S. Gibson, Mira Mastoras, Breeana Baker, Cedric Kouam, Kimberly Paquette, Paige Jarreau, M. B. Makarious, A. Moore, Samantha Hong, Dan Vitale, Syed Shah, Jean Monlong, C. Pantazis, Mobin Asri, Kishwar Shafin, P. Carnevali, S. Marenco, Pavan K Auluck, Ajeet Mandal, K. Miga, A. Rhie, X. Reed, Jinhui Ding, Mark R. Cookson, Mike A. Nalls, Andrew B Singleton, Danny E Miller, Mark J. P. Chaisson, W. Timp, J. R. Gibbs, A. Phillippy, M. Kolmogorov, Miten Jain, F. Sedlazeck, B. Paten, C. Blauwendraat
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Determining the phenotypic effects of single nucleotide variants is critical for understanding the genome and interpreting clinical sequencing results. Base editors, including diversifying base editors that create C>N mutations, are potent tools for installing point mutations in mammalian genomes and studying their effect on cellular function. Numerous base editor options are available for such studies, but little information exists on how the composition of the editor (deaminase, recruitment method, and fusion architecture) affects editing. To address this knowledge gap, the effect of various design features, such as deaminase recruitment and delivery method (electroporation or lentiviral transduction), on editing was assessed across ∼200 synthetic target sites. The direct fusion of a hyperactive variant of activation-induced cytidine deaminase to the N-terminus of dCas9 (DivA-BE) produced the highest editing efficiency, ∼4-fold better than the previous CRISPR-X method. Additionally, DivA-BE mutagenized the DNA strand that anneals to the targeting sgRNA to create G>N mutations, which were absent when the deaminase was fused to the C-terminus of dCas9. The DivA-BE editors efficiently diversified their target sites, an ideal characteristic for discovering functional variants. These and other findings provide a comprehensive analysis of how design features influence the activity of several popular base editors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4879f4371c59083e5df3495887b2d3ce24001052" target='_blank'>
              Towards optimizing diversifying base editors for high-throughput studies of single- nucleotide variants
              </a>
            </td>
          <td>
            Carley I Schwartz, Nathan S. Abell, Amy Li, Aradhana, Josh Tycko, Alisa Truong, Stephen B. Montgomery, Gaelen T. Hess
          </td>
          <td>2024-11-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/557911aa9625d7bd95b52abd71f3273a6c915b9d" target='_blank'>
              Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in cancer.
              </a>
            </td>
          <td>
            Yang Yang, Michelle L Badura, Patrick C O'Leary, Henry Delavan, Troy M Robinson, Emily A Egusa, Xiaoming Zhong, Jason Swinderman, Haolong Li, Meng Zhang, Minkyu Kim, Alan Ashworth, Felix Y. Feng, Jonathan Chou, Lixing Yang
          </td>
          <td>2024-11-18</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Population-scale resources of genetic, molecular, and cellular information form the basis for understanding human genomes, charting the heritable basis of disease, and tracing the effects of mutations. Pooled perturbation assays applied to cellular models, probing the effect of many perturbations coupled with an scRNA-seq readout (Perturb-seq), are especially potent references for interpreting disease-linked mutations or gene expression changes. However, the utility of existing maps has been limited by the comprehensiveness of perturbations possible, and the relevance of their cell line context. Here, we present the first genome-scale CRISPR interference (CRISPRi) perturbation map with single-cell RNA sequencing readout across many human genetic backgrounds in human pluripotent cells. To do so, we establish large-scale CRISPRi screening in human induced pluripotent stem cells from healthy donors, using over 20,000 guide RNAs to target 7,226 genes across 34 cell lines from 26 genetic backgrounds, and gather expression data from nearly 2 million cells. We comprehensively map trans expression changes induced by the target knockdowns, which complement co-expression patterns in unperturbed cells and facilitate the functional annotation of target genes to biological processes and complexes. Consistency of targeting protein complex members point to protein complexes as a nexus for aggregating transcriptional variation, revealing novel interaction partners. We characterise variation in perturbation effects across donors, with expression quantitative trait loci linked to higher genetic modulation of perturbation effects but overall low replication of trans effects due to knockdown of their corresponding cis regulators. This study pioneers population-scale CRISPR perturbations with single cell readouts that will fuel foundation models for the future of effective modulation of cellular disease phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5800ff9ee76e49e133d6ead67bfba5c5d87d6239" target='_blank'>
              A genome-scale single cell CRISPRi map of trans gene regulation across human pluripotent stem cell lines
              </a>
            </td>
          <td>
            Claudia Feng, Elin Madli Peets, Yan Zhou, Luca Crepaldi, S. Usluer, Alistair S. Dunham, Jana M. Braunger, Jing Su, Magdalena E Strauss, Daniele Muraro, Kimberly Ai Xian Cheam, M. Bonder, Edgar Garriga Nogales, Sarah Cooper, Andrew Bassett, Steven Leonard, Yong Gu, Bo Fussing, David Burke, Leopold Parts, O. Stegle, B. Velten
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="This review explores articles concerning the experimental research cycle on genome instability in cell populations of highly malignant recurrent organotropic rhabdomyosarcoma RA- 2 in rats. Clonal analysis and cloning were pivotal components of this research, which relies on the frequency of cells with micronuclei and internuclear bridges to gauge the intensity of chromothripsis and break-fusion-bridge cycles. The efficacy of cloning, determined by these indicators, stemmed from the deliberate isolation of tumor stem cells, yielding clones in which chromothripsis activity and breakage-fusion-bridge cycles were sustained. Notably, it is plausible that the stem cells themselves, progenitors of these clones, harbor dicentric chromosomes and chromosomal fragments, contributing to the formation of "fatal micronuclei" in their karyotype. Cloning based on bridges and micronuclei has proven effective up to a certain threshold (15%-18%), reaffirming the predicted reproductive extinction of malignant cell populations under mutational pressure and genome chaos, as posited by the genetic theory of cell populations. Furthermore, this review highlights the potential of ergodic cancer therapy as a novel therapeutic strategy. Ergodic therapy offers promising prospects for late-stage and terminal malignant tumors, where conventional treatments may fall short due to advanced progression. Furthermore, by "enhancing chromothripsis" through the induction of additional micronuclei and bridges, ergodic cancer therapy seeks to increase genome chaos to a critical threshold, potentially halting malignant progression. This innovative approach presents opportunities to explore new drugs and targets for chromothripsis-based oncotherapy, addressing the pressing need for effective treatments in advanced stages of malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ee83cc51235034563b6307f2bcb528ce64b269" target='_blank'>
              Perspectives on Ergodic Cancer Therapy Derived from Cloning Genome Chaos via In Vivo Rhabdomyosarcoma RA-2 Models: a Narrative Review.
              </a>
            </td>
          <td>
            Sergey Shityakov, Natalia Lubinets, Viacheslav Kravtsov
          </td>
          <td>2024-11-08</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0297441e9d4a24ced84d1909df04d7019094986" target='_blank'>
              Functional screening reveals genetic dependencies and diverging cell cycle control in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Daniel J. Merk, F. Tsiami, S. Hirsch, B. Walter, L. Haeusser, Jens D. Maile, Aaron Stahl, M. Jarboui, Anna Lechado-Terradas, Franziska Klose, S. Babaei, Jakob Admard, N. Casadei, C. Roggia, Michael Spohn, J. Schittenhelm, Stephan Singer, U. Schüller, F. Piccioni, N. Persky, Manfred Claassen, M. Tatagiba, Philipp J. Kahle, D. Root, M. Templin, Ghazaleh Tabatabai
          </td>
          <td>2024-12-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a66c3b63a667066dce8a15c38a4f95ef3f07fdfa" target='_blank'>
              Molecular profiling reveals novel therapeutic targets and clonal evolution in ovarian clear cell carcinoma
              </a>
            </td>
          <td>
            Angel Chao, Chen-Yang Huang, Willie Yu, C. Lin, Hao Lin, Angel Chao, Cheng-Tao Lin, Hung-Hsueh Chou, Kuang-Gen Huang, Huei-Jean Huang, Ting-Chang Chang, Steven G. Rozen, Ren-Chin Wu, Chyong-Huey Lai
          </td>
          <td>2024-11-14</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Cancer research has historically focused on the Somatic Mutation Theory (SMT), viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking Cancer Evolution: From Genetic Mutations to Complex Information Systems in Tumor Reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e013dd21a317acebc6bbd04a30655def3c28bb68" target='_blank'>
              Decorin as a key marker of desmoplastic cancer-associated fibroblasts mediating first-line immune checkpoint blockade resistance in metastatic gastric cancer.
              </a>
            </td>
          <td>
            Ki Tae Kim, Min Hee Lee, Su-Jin Shin, In Cho, Jung Cheol Kuk, Jina Yun, Yoon Young Choi
          </td>
          <td>2024-11-09</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The study by Grossmann and colleagues uses single-nucleus RNA sequencing in a cohort of matched high-risk neuroblastoma primary tumor samples, obtained from the same patient at diagnosis and definitive surgery, to identify persister cells that survive induction chemotherapy. These persister cells utilize mechanisms of chemoresistance that are both tumor-intrinsic and tumor-extrinsic, are highly dependent on the original genetic profile of the tumor, and represent novel, patient-specific targets to precisely inhibit chemoresistance and disease recurrence. See related article by Grossmann et.al., p. 2387.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17960d164122a139c932c5adc28d8d07f55f0399" target='_blank'>
              Characterization of Persister Cells Provides Insights into Mechanisms of Therapy Resistance in Neuroblastoma.
              </a>
            </td>
          <td>
            Amber B. Wolf, C. P. Reynolds, Eveline Barbieri
          </td>
          <td>2024-12-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metabolic reprogramming from oxidative respiration to glycolysis is generally considered to be advantageous for tumor initiation and progression. However, we found that breast cancer cells forced to perform glycolysis acquired a vulnerability to PARP inhibitors. Small-molecule inhibition of mitochondrial respiration-using glyceollin I, metformin, or phenformin-induced overproduction of the oncometabolite lactate, which acidified the extracellular milieu and repressed the expression of homologous recombination (HR)-associated DNA repair genes. These serial events created so-called "BRCAness," in which cells exhibit an HR deficiency phenotype despite lacking germline mutations in HR genes such as BRCA1 and BRCA2, and, thus, sensitized the cancer cells to clinically available poly(ADP-ribose) polymerase inhibitors. The increase in lactate repressed HR-associated gene expression by decreasing histone acetylation. These effects were selective to breast cancer cells; normal epithelial cells retained HR proficiency and cell viability. These mechanistic insights into the BRCAness-prone properties of breast cancer cells support the therapeutic utility and cancer cell-specific potential of mitochondria-targeting drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d552f07898d1f01f3a3d3064ba56a79d2a0073d0" target='_blank'>
              Metabolically inducing defects in DNA repair sensitizes BRCA-wild-type cancer cells to replication stress.
              </a>
            </td>
          <td>
            Kenji Watanabe, Tatsuro Yamamoto, Tomoko Fujita, Shinjiro Hino, Yuko Hino, K. Yamazaki, Y. Ohashi, Shun Sakuraba, H. Kono, M. Nakao, Koji Ochiai, Shingo Dan, Noriko Saitoh
          </td>
          <td>2024-11-12</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Polyploidization is a driving force of wheat evolution and speciation, yet its impact on epigenetic regulation and gene expression remains unclear. Here, we constructed a high-resolution epigenetic landscape across leaves, spikes, and roots of hexaploidy wheat and its tetraploid and diploid relatives. Inter-species stable-expression genes exhibited conserved amino acid sequences under strong purifying selection, while dynamic-expression genes were linked to species-specific adaptation. During hexaploidization, dominant D-subgenome homoeolog expression was suppressed via reduced activating epigenetic signals, converging expression with the A and B subgenomes. Proximal chromatin regions near genes were more stable, whereas distal regions, particularly enhancer-like elements mediated by H3K27ac and H3K4me3, exhibit higher dynamism. Sequence variations in these enhancers lead to differential gene regulation, influencing traits such as spike development. For instance, the two haplotypes of dCRE region of TaDEP-B1 resulted in significant differences in its expression and spikelet numbers. We also observed a coevolution of transcription factors and their binding sites, particularly within the expanded ERF family, which regulates spike morphology. This study highlights the interplay between sequence variation and epigenetic modifications in shaping transcriptional regulation during wheat speciation, offering valuable insights for genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3eb492b176e5f2e28bdd73ac3853852b58a5d2c" target='_blank'>
              Reshaping epigenomic landscapes in facilitating the speciation of bread wheat
              </a>
            </td>
          <td>
            Zhaoheng Zhang, Xuelei Lin, Jingjing Yue, Yong-Chang Xu, Lingfeng Miao, Wenqiang Tang, Weilong Guo, Jun Xiao
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive malignancy that poses significant challenges due to high rates of relapse and resistance to treatment, particularly in older populations. While therapeutic advances have been made, survival outcomes remain suboptimal. The evolution of DNA and RNA sequencing technologies, including whole-genome sequencing (WGS), whole-exome sequencing (WES), and RNA sequencing (RNA-Seq), has significantly enhanced our understanding of AML at the molecular level. These technologies have led to the discovery of driver mutations and transcriptomic alterations critical for improving diagnosis, prognosis, and personalized therapy development. Furthermore, single-cell RNA sequencing (scRNA-Seq) has uncovered rare subpopulations of leukemia stem cells (LSCs) contributing to disease progression and relapse. However, widespread clinical integration of these tools remains limited by costs, data complexity, and ethical challenges. This review explores recent advancements in DNA/RNA sequencing in AML and highlights both the potential and limitations of these techniques in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e5d43f554c8fe98964b8130a7d1804ae561d5a" target='_blank'>
              Advances in DNA/RNA Sequencing and Their Applications in Acute Myeloid Leukemia (AML)
              </a>
            </td>
          <td>
            Fatimah Ahmed, Jiang Zhong
          </td>
          <td>2024-12-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Breast tumors comprise several sub-populations of cells, so-called “clones”, with specific cellular and molecular characteristics. Some cancer clones can be more ‘aggressive’ than others in particular tumor microenvironments (TMEs), contributing to metastatic progression and drug resistance. Understanding the specific genes/pathways deregulated in these aggressive cancer cells is essential for developing new tailored treatment strategies. To accomplish this, we optimized the use of a lentiviral-based barcoding technique (Berthelet, J., et.al., Sci Adv 2021, Serrano, A., et.al., Nat Rev Cancer 2022) where cancer cells from cell lines and patient-derived xenografts are tagged with optical barcodes and transplanted into the mammary fat pads of immunodeficient mice. Upon resection of tumors and collection of metastatic organs at the experimental endpoint, we analyzed clonal fate and fitness using flow cytometry and imaging. The subsequent molecular characterization of individual clones was determined by bulk and single-cell RNA sequencing. We also leverage the use of these barcoded models to investigate the mechanisms of innate and acquired resistance to Sacituzumab Govitecan (SG), a drug recently approved for the treatment of patients with advanced breast cancer. Our studies revealed unique gene signatures and pathways associated with preferential organ tropism (in the lungs, liver and brain TMEs) and SG resistance. This comprehensive analysis, currently validated using autopsy samples and patient databases, will support the design of new predictive markers and targeted therapies for metastatic breast cancer.
 Citation Format: Sreeja Reddy Gadipally, Jean Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, Caroline Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, Charafe Jauffret Emmanuelle, Melissa Davis, Robin L. Anderson, Luciano Martelotto, Belinda Yeo, Delphine Merino. Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/528d6539434045b291c3a0faa67fa6d3c29fbc80" target='_blank'>
              Abstract B005: Exploring breast cancer heterogeneity in the context of metastatic tumour microenvironments and therapy resistance
              </a>
            </td>
          <td>
            Sreeja Gadipally, J. Berthelet, Samuel C. Lee, Dharmesh D. Bhuva, Michael Roach, Farrah El-Saafin, C. Bell, Yunjian Wu, David Baloyan, Christphe Ginestier, C. Emmanuelle, Melissa J Davis, Robin L Anderson, Luciano Martelotto, B. Yeo, Delphine Merino
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Proteins that emerge de novo from noncoding DNA could negatively or positively influence cellular physiology in the sense of providing a possible adaptive advantage. Here, we employ two approaches to study such effects in a human cell line by expressing random sequences and mouse de novo genes that lack homologs in the human genome. We show that both approaches lead to differential growth effects of the cell clones dependent on the sequences they express. For the random sequences, 53% of the clones decreased in frequency, and about 8% increased in frequency in a joint growth experiment. Of the 14 mouse de novo genes tested in a similar joint growth experiment, 10 decreased, and 3 increased in frequency. When individually analysed, each mouse de novo gene triggers a unique transcriptomic response in the human cells, indicating mostly specific rather than generalized effects. Structural analysis of the de novo gene open reading frames (ORFs) reveals a range of intrinsic disorder scores and/or foldability into alpha-helices or beta sheets, but these do not correlate with their effects on the growth of the cells. Our results indicate that de novo evolved ORFs could easily become integrated into cellular regulatory pathways, since most interact with components of these pathways and could therefore become directly subject to positive selection if the general conditions allow this.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43878c4781a378a056438c19941008a1a26fd3df" target='_blank'>
              Expression of Random Sequences and de novo Evolved Genes From the Mouse in Human Cells Reveals Functional Diversity and Specificity
              </a>
            </td>
          <td>
            Silvia Aldrovandi, Johana Fajardo Castro, K. Ullrich, Amir Karger, Victor Luria, D. Tautz
          </td>
          <td>2024-12-01</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a321392f82837178158946451c3a5d2c2f02daa" target='_blank'>
              Genomic and transcriptomic profiling of radioresistant prostate and head and neck cancers implicate a BAHD1-dependent modification of DNA damage at the heterochromatin
              </a>
            </td>
          <td>
            Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, E. Tan, Kah M Tan, Dennis J J Poon, Pek L Chu, Dewi Susanti, Pei Ling Tai, Monica Ryu, James Proudfoot, E. Yeo, Khee Chee Soo, Melvin L. K. Chua
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Cancer initiation and progression are controlled not only by intrinsic factors, such as genetic mutations and mesenchymal trait acquisition, but also by dynamic spatiotemporal shifts within the tumor microenvironment. These elements induce plasticity, and enhance invasiveness, metastasis, and treatment resistance, while promoting immune evasion through the reprogramming of the immune microenvironment. However, the molecular mechanisms driving this reprogramming remain unclear. In this review, we summarize the recent pivotal findings on pancreatic cancer, which is a highly malignant disease, and overview the roles of plasticity, metabolic reprogramming, and the creation of an immunosuppressive microenvironment in disease progression. We also present our research on two crucial signaling pathways, i.e., the dynamic interplay between mutational load and the tumor microenvironment, and the acquisition of cancer plasticity via inflammatory responses and metabolic reprogramming. A comprehensive understanding of these processes is expected to transform therapeutic strategies and improve the outcomes of pancreatic cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d6b2b31d56874111eca289fe4cec7a0df598ab" target='_blank'>
              Plasticity and Tumor Microenvironment in Pancreatic Cancer: Genetic, Metabolic, and Immune Perspectives
              </a>
            </td>
          <td>
            A. Hashimoto, S. Hashimoto
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The complex and heterogeneous genomic landscape of multiple myeloma (MM) and many of its clinical and prognostic implications remains to be understood. In other cancers, such as breast cancer, using whole-exome sequencing (WES) and molecular signatures in clinical practice has revolutionized classification, prognostic prediction, and patient management. However, such integration is still in its early stages in MM. In this study, we analyzed WES data from 35 MM patients to identify potential mutational signatures and driver mutations correlated with clinical and cytogenetic characteristics. Our findings confirm the complex mutational spectrum and its impact on previously described ontogenetic and epigenetic pathways. They show TYW1 as a possible new potential driver gene and find no significant associations of mutational signatures with clinical findings. Further studies are needed to strengthen the role of mutational signatures in the clinical context of patients with MM to improve patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca1e2e137b31b6b87c597d6e1aab2922694e2f3" target='_blank'>
              Whole-Exome Sequencing, Mutational Signature Analysis, and Outcome in Multiple Myeloma—A Pilot Study
              </a>
            </td>
          <td>
            Lorenz Oelschläger, A. Künstner, Friederike Frey, T. Leitner, L. Leypoldt, N. Reimer, Niklas Gebauer, L. Bastian, Katja C. Weisel, Verena-Wilbeth Sailer, Christoph Röcken, Wolfram Klapper, Björn Konukiewitz, E. M. Murga Penas, Michael Forster, Natalie Schub, H. Ahmed, Jutta Kirfel, N. von Bubnoff, Hauke Busch, Cyrus Khandanpour
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, , Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cae0c3f9678085fee30003f0aae12e5e2f38618" target='_blank'>
              Multiple myeloma long-term survivors exhibit sustained immune alterations decades after first-line therapy
              </a>
            </td>
          <td>
            R. Lutz, F. Grünschläger, Malte Simon, M. Awwad, M. Bauer, S. Yousefian, N. Beumer, L. Jopp-Saile, Anastasia Sedlmeier, Llorenç Solé-Boldo, Bogdan Avanesyan, Dominik Vonficht, Patrick Stelmach, Georg Steinbuss, T. Boch, Simon Steiger, M. Baertsch, N. Prokoph, K. Rippe, B. Durie, Claudia Wickenhauser, A. Trumpp, Carsten Müller-Tidow, D. Huebschmann, Niels Weinhold, M. Raab, B. Brors, Hartmut Goldschmidt, Charles D. Imbusch, M. Hundemer, Simon Haas
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Multiple myeloma (MM), a clonal B-cell neoplasia, is an incurable and heterogeneous disease where survival ranges from a few months to more than 10 years. The clinical heterogeneity of MM arises from multiple genomic events that result in tumour development and progression. Recurring genomic abnormalities including cytogenetic abnormalities, gene mutations and abnormal gene expression profiles in myeloma cells have a strong prognostic power. With the advancement in technologies and the development of novel drugs, the prognostic factors and treatment paradigms of MM have been fast evolving over the past few years. Following the introduction of new highthroughput cytogenomic technologies such as array comparative genome hybridisation (aCGH) or single nucleotide polymorphism array (SNP array) and molecular techniques such as gene expression profiling (GEP) and massively parallel genomic sequencing, the prediction of survival in MM no longer solely depends on conventional cytogenetics and interphase fluorescence in situ hybridisation (iFISH) analysis findings. These new technologies enable screening for all possible chromosomal aberrations and other genomic alterations, identifying each aberration on a case-bycase basis and discovering new aberrations that are relevant in unraveling the tumor cells' complex biology. This in turn allows a better understanding of the disease complexity and heterogeneity. The objective of this review on the genetic architecture of MM is to discuss the latest developments on the cytogenetic/cytogenomic-based risk classification of MM that are currently in use and their prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f51a37b55e510ee5aeb00a67198a8c2f2e860c2a" target='_blank'>
              Genetic architecture of Multiple Myeloma and its prognostic implications - An updated review.
              </a>
            </td>
          <td>
            E. Foong, R. Ankathil
          </td>
          <td>2024-12-01</td>
          <td>The Malaysian journal of pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb9159d119b846b192af86b098eda853ace7fad" target='_blank'>
              Orthogonal and multiplexable genetic perturbations with an engineered prime editor and a diverse RNA array
              </a>
            </td>
          <td>
            Qichen Yuan, Hongzhi Zeng, Tyler C Daniel, Qingzhuo Liu, Yongjie Yang, Emmanuel C Osikpa, Qiaochu Yang, Advaith Peddi, Liliana M Abramson, Boyang Zhang, Yong Xu, Xue Gao
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancers exhibit a remarkable ability to develop resistance to a range of treatments, often resulting in relapse following first-line therapies and significantly worse outcomes for subsequent treatments. While our understanding of the mechanisms and dynamics of the emergence of resistance during cancer therapy continues to advance, many questions remain about which treatment strategies can minimize the probability that resistance will evolve, thereby improving long-term patient outcomes. In this study, we present an evolutionary simulation model of a clonal population of cells that can acquire resistance mutations to one or more treatments. We then leverage this model to examine the efficacy of a two-strike “extinction therapy” protocol—in which two treatments are applied sequentially in an effort to first contract the population to a vulnerable state and then push it to extinction—in comparison to that of a combination therapy protocol. We investigate the impact of parameters such as the timing of the switch between the two strikes, the rate of emergence of resistant mutations, the dose of the applied drugs, the presence of cross-resistance, and whether resistance is a binary or a quantitative trait. Our results indicate that the timing of switching from the first to the second strike has a marked effect on the likelihood of driving the population to extinction, and that extinction therapy outperforms combination therapy when cross-resistance is present. We conduct an in silico trial that reveals more detailed insight into when and why a second strike will succeed or fail. Finally, we demonstrate that modeling resistance as a quantitative rather than binary trait does not change our overall conclusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f738c3dddcab46b26ff2c97c3888fefb1e9d2858" target='_blank'>
              Evolutionary rescue model informs strategies for driving cancer cell populations to extinction
              </a>
            </td>
          <td>
            Amjad Dabi, Joel S. Brown, Robert A Gatenby, Corbin D. Jones, Daniel R. Schrider
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background: Arsenic trioxide (ATO) is an anticancer agent for treating acute promyelocytic leukemia (APL). However, 5–10% of patients fail to respond, developing relapsed/refractory disease. The aim of this study was to identify potential new therapeutic approaches for ATO-unresponsive APL by targeting the anti-apoptotic genes that contribute to drug resistance. Methods: RNA expression of dysregulated genes involved in the apoptotic pathway was analyzed by comparing ATO-resistant APL cell clones generated in our lab with the corresponding sensitive clones, at basal levels and after 48 h of treatment with ATO. Results: ATO-resistant APL cells showed upregulation of APAF1, BCL2, BIRC3, and NOL3 genes, while CD70 and IL10 genes were downregulated, compared to ATO-sensitive cells. Treatment with ATO strongly increased the expression of the anti-apoptotic genes BIRC3, NOL3, and BCL2A1 and significantly downregulated BCL2 in ATO-sensitive clones. Although all these genes can be relevant to ATO-resistance, we selected BCL2 and BIRC3 as druggable targets. A direct correlation between BCL2 expression and the sensitivity to the BCL2 inhibitor venetoclax was observed, indicating BCL2 as predictive biomarker of the response. Moreover, the combination of venetoclax with ATO exerted synergistic cytotoxic effects, thus reverting the resistance to ATO. APL treatment with SMAC mimetics such as LCL161 and xevinapant (inhibitors of BIRC3) was not as effective as the BCL2 inhibitor as a monotherapy but exerted synergistic effects in combination with ATO in cells with low BIRC expression. Conclusions: This study demonstrates the therapeutic potential of venetoclax in combination with ATO in vitro and strongly encourages further investigation of relapsed/refractory APL with high BCL2 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95090e68a4ade11845786f29702e9cd58361ca40" target='_blank'>
              Combination Treatment of Resistant Acute Promyelocytic Leukemia Cells with Arsenic Trioxide and Anti-Apoptotic Gene Inhibitors
              </a>
            </td>
          <td>
            Manuela Giansanti, T. Ottone, S. Travaglini, M. Voso, Grazia Graziani, Isabella Faraoni
          </td>
          <td>2024-11-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="The past 2 decades have witnessed extraordinary advances in our understanding of the genetic factors influencing inflammatory bowel disease (IBD), providing a foundation for the approaching era of genomic medicine. On behalf of the NIDDK IBD Genetics Consortium, we herein survey 11 grand challenges for the field as it embarks on the next 2 decades of research utilizing integrative genomic and systems biology approaches. These involve elucidation of the genetic architecture of IBD (how it compares across populations, the role of rare variants, and prospects of polygenic risk scores), in-depth cellular and molecular characterization (fine-mapping causal variants, cellular contributions to pathology, molecular pathways, interactions with environmental exposures, and advanced organoid models), and applications in personalized medicine (unmet medical needs, working toward molecular nosology, and precision therapeutics). We review recent advances in each of the 11 areas and pose challenges for the genetics and genomics communities of IBD researchers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/909f07dc90473856ce58964c4432f026f2398970" target='_blank'>
              Eleven Grand Challenges for Inflammatory Bowel Disease Genetics and Genomics.
              </a>
            </td>
          <td>
            Greg Gibson, J. Rioux, Judy H. Cho, T. Haritunians, Akshaya Thoutam, Maria T Abreu, S. Brant, Subramaniam Kugathasan, Jacob L McCauley, Mark Silverberg, D. McGovern
          </td>
          <td>2024-12-19</td>
          <td>Inflammatory bowel diseases</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Targeted treatment of advanced or metastatic urothelial carcinomas (UCs) requires the identification of druggable mutations. This study describes the development of a 3′ Rapid Amplification of cDNA Ends (3′ RACE)-based targeted RNA sequencing panel which accounts for the status of all genes relevant to UC treatment, namely, FGFR1-4, KRAS, NRAS, BRAF, ERBB2 (HER2), CD274 (PD-L1) and PIK3CA. FGFR2/3-activating point mutations or fusions were found in 54/233 (23.2%) tumors. FGFR3 rearrangements were identified in 11 patients, with eight of them being undetectable by commonly used PCR kits. In addition, one tumor contained a high-copy FGFR2 gene amplification accompanied by strong overexpression of the gene. Mutations in RAS/RAF genes were present in 30/233 (12.9%) UCs and were mutually exclusive with alterations affecting FGFR2/3 genes. On the contrary, activating events in the HER2 oncogene (point mutations and overexpression), as well as PIK3CA mutations, which were relatively common, occurred with similar frequencies in RAS/RAF- or FGFR2/3-positive vs. negative samples. High PD-L1 mRNA expression was associated with advanced disease stage and was not observed in tumors with increased HER2 mRNA expression or in UCs with evidence for FGFR2/3 activation. Three of the studied carcinomas had high-level microsatellite instability (MSI). Overall, more than half of the UCs had potentially druggable genetic alterations. The proposed NGS panel permits comprehensive and cost-efficient analysis of UC-specific molecular targets and may be considered in clinical routine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51df5895686bd4393d617e0cc57a4eeef5644462" target='_blank'>
              Use of 3′ Rapid Amplification of cDNA Ends (3′ RACE)-Based Targeted RNA Sequencing for Profiling of Druggable Genetic Alterations in Urothelial Carcinomas
              </a>
            </td>
          <td>
            N. Mitiushkina, V. Tiurin, Aleksandra A Anuskina, Natalia A. Bordovskaya, Ekaterina A. Nalivalkina, Darya M. Terina, M. V. Berkut, Anna D Shestakova, Maria V. Syomina, E. Kuligina, A. Togo, E. Imyanitov
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a hematological malignancy characterized by a block in differentiation and accelerated proliferation of myeloid progenitor cells. Genes encoding for epigenetic regulators are among the most frequent targets for mutations and structural variations in AML, giving rise to profound epigenetic heterogeneity between and within tumors. Deletions of chromosome 5q [del(5q)] are among the most common copy number alterations in AML and are associated with extremely poor clinical outcome and therapy resistance, however the mechanisms linking del(5q) to leukemic progression are not understood. Analyzing DNA methylation profiles from 477 elderly AML patients using DNA methylome deconvolution, we discovered that del(5q) AML is an epigenetically distinct subgroup characterized by a signature of DNA hypermethylation, which we propose may be linked to dysregulation of H3K9me1/2 and overexpression of the leukemic stem cell marker, DNMT3B. Interrogation of the minimally deleted 5q region highlighted the H3K9me1/2 demethylase KDM3B as a likely target for haploinsufficiency in this subgroup. Our data suggest that del(5q) AML should be reconsidered as an epigenetically dysregulated subgroup, driven by heterozygous loss of KDM3B, and that the resulting imbalance of H3K9me1/2 may contribute to the progression of these aggressive leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8284d22674e6348a422c0aa420226cc0452e3a13" target='_blank'>
              Acute Myeloid Leukemia with deletion 5q is an epigenetically distinct subgroup defined by heterozygous loss of KDM3B
              </a>
            </td>
          <td>
            Katherine Kelly, Linda Welte, Etienne Sollier, Anna Riedel, Fiona Brown-Burke, Michael Scherer, Harold Keer, Mohammad Azab, E. Jahn, H. Döhner, Konstanze Döhner, P. Lutsik, C. Plass
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The CRISPR/Cas genome editing approach in non-model organisms poses challenges that remain to be resolved. Here, we demonstrated a generalized roadmap for a de novo genome annotation approach applied to the non-model organism Macrobrachium rosenbergii. We also addressed the typical genome editing challenges arising from genetic variations, such as a high frequency of single nucleotide polymorphisms, differences in sex chromosomes, and repetitive sequences that can lead to off-target events. For the genome editing of M. rosenbergii, our laboratory recently adapted the CRISPR/Cas genome editing approach to embryos and the embryonic primary cell culture. In this continuation study, an annotation pipeline was trained to predict the gene models by leveraging the available genomic, transcriptomic, and proteomic data, and enabling accurate gene prediction and guide design for knock-outs. A next-generation sequencing analysis demonstrated a high frequency of genetic variations in genes on both autosomal and sex chromosomes, which have been shown to affect the accuracy of editing analyses. To enable future applications based on the CRISPR/Cas tool in non-model organisms, we also verified the reliability of editing efficiency and tracked off-target frequencies. Despite the lack of comprehensive information on non-model organisms, this study provides an example of the feasibility of selecting and editing specific genes with a high degree of certainty.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9f9b90d707bd226bf9a4714aba878076437ac1" target='_blank'>
              Roadmap and Considerations for Genome Editing in a Non-Model Organism: Genetic Variations and Off-Target Profiling
              </a>
            </td>
          <td>
            Hanin Wattad, Jonathan Molcho, Rivka Manor, S. Weil, E. Aflalo, V. Chalifa-Caspi, Amir Sagi
          </td>
          <td>2024-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary Adult B-cell acute lymphoblastic leukemia (B-ALL) is characterized by genetic heterogeneity and high relapse rate. Forty-four adult B-ALL patients were studied at both diagnosis and relapse by next-generation sequencing (NGS) in order to identify genetic alteration driving relapse. Four main genetic pathways leading to relapse in adults were identified: IKZF1plus genetic profile, RAS mutations and TP53 alterations in Philadelphia chromosome (Ph)-negative B-ALL and acquisition of ABL1 mutations in Ph-positive patients. In addition, three clonal evolution patterns were identified: the persistent clone trajectory (25%), the expanding clone trajectory (11%) and the therapy-boosted trajectory (48%). Our results reveal the presence of preferential biological pathways leading to relapse in adult B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54416c39b5f84bc7f01293f1564700b87fc9be40" target='_blank'>
              Preferential Genetic Pathways Lead to Relapses in Adult B-Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Josgrey Navas-Acosta, Alberto Hernández-Sánchez, Teresa González, Ángela Villaverde Ramiro, Sandra Santos, Cristina Miguel, J. Ribera, Isabe L Granada, M. Morgades, Ricardo Sánchez, E. Such, S. Barrena, Juana Ciudad, Julio Dávila, Natalia de Las Heras, Alfonso García-de Coca, J. Labrador, J. Queizán, Sandra Martín, Alberto Orfao, J. Ribera, Rocío Benito, J. Hernández-Rivas
          </td>
          <td>2024-12-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The development of sequencing technologies and their increased accessibility in clinical services and genetic laboratories have considerably accelerated the identification of genetic variants associated with rare diseases (RDs). Among these, Mendelian disorders of the epigenetic machinery (MDEM) are rare monogenic diseases characterized by the presence of mutations in genes encoding epigenetic regulators that play a key role in organismal development and cellular functions. Loss of function of these regulators is expected to lead to epigenetic modifications that profoundly affect genome expression and cellular identity. Disruptions in DNA methylation profiles have been documented in MDEMs, emerging as a useful diagnostic tool. The current challenge is to determine whether and how these epigenomic alterations drive the mechanisms underlying the clinical manifestations in patients suffering from this class of diseases. Studying MDEMs may therefore shed light on the important role of epigenetic information in health and disease, particularly the mechanisms involved in the development and understanding of complex pathologies, such as neurodevelopmental disorders and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b00b4062f8130453f8f77acda434d081dca0a7cd" target='_blank'>
              [Genetic diseases of epigenetic machinery].
              </a>
            </td>
          <td>
            Maud de Dieuleveult, Guillaume Velasco
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Supported by the literature, our initial hypothesis was that Estrogen Receptor alfa (ESR1) may function as a master regulator by influencing the expression of epithelial-to-mesenchymal transition (EMT)-related genes in cancer cells. To explore this further, we used the CRISPR/Cas9 gene editing system to create MCF-7 sublines with down-regulated ESR1 expression and analyzed its impact on EMT initiation. By applying two distinct types of gRNA for gene editing, we established six MCF-7 cell sublines with either nearly complete or partial down-regulation of the ESR1 isoforms. Unexpectedly, the data obtained revealed no discernible impact of ESR1 down-regulation on EMT manifestation as Western blot and Real-Time qPCR analysis of selected clones revealed no changes in EMT markers expression. We suggested that those of the ESR1 isoforms, the expression of which was not affected by gene editing, could be crucial for the initiation of EMT. The obtained cell models will be used further to evaluate the activity of ESR1 isoforms.

Keywords: CRISPR/Cas9, epithelial-to-mesenchymal transition, estrogen receptor alfa, MCF-7 cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1993f620feca20abe8e5a87884e89477496308d9" target='_blank'>
              Generation of the MCF-7 cell sublines with CRISPR/Cas9 mediated disruption of estrogen receptor alfa (ESR1) expression
              </a>
            </td>
          <td>
            L. Savinska, S. A. Kvitchenko, S. S. Palchevskyi, I. V. Kroupskaya, A. V. Mazov, O. M. Garifulin, V. V. Filonenko
          </td>
          <td>2024-12-12</td>
          <td>The Ukrainian Biochemical Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract. N6-methyladenosine (m6A), recognized as the most prevalent post-transcriptional modification in organisms, has been substantiated to exert a significant influence on the genetic mechanisms underlying breast cancer. To delve deeper into the disparities in expression, modification, and interactions between mRNA and m6A in breast cancer (BC), we conducted RNA-seq and MeRIP-Seq analyses on eight human breast tissue transcriptome samples acquired from the GEO database. combining two single omics and integrated analysis, the study unveiled the mRNA and m6A expression patterns, differential genes, and enriched pathway functional disparities in BC (on Chromosome 17): (1) A confidence list comprised of 53 differential genes was derived by intersecting the 72 identified differentially expressed mRNA genes and the 781 differentially modified m6A genes; (2) The three-way analysis revealed two distinct types of pathways related to BC: ABC transporter activity and Response to epidermal growth factor. Among these, the EGF pathways exhibited the closest association with the differential m6A modifications. Meanwhile, all three BC subtypes enriched in DisGeNET were all linked to EGF; (3) By integrating PPI and enrichment analysis, we selected target genes from the confidence gene list, which included ABCA6, ABCA8, and ABCA10, known to be involved in regulating ABC transporters, and ERBB2, a central hub gene in PPI with a pivotal role in the EGF pathway and Her2-positive BC subtype under m6A differential modification. Subsequent research may uncover RNA-binding proteins for these target genes and offer effective drug design targets for BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/184644e2cacda5f2067cd608875e2e198b6261ee" target='_blank'>
              Exploring the transcriptomic and m6a landscape of human chromosome 17 in breast cancer: A combined RNA-seq and MeRIP-Seq analysis
              </a>
            </td>
          <td>
            Mengmeng Wang
          </td>
          <td>2024-11-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Oncogene-driven non-small-cell lung cancer (NSCLC) is typically treated with targeted therapies to inhibit oncogene downstream signaling pathways and reduce tumor survival. The most common subtypes, EGFR and KRAS mutations, are amenable to these therapies; however, resistance often develops, leading to oncogene-independent metastases. This study explores non-oncogene addiction (NOA) as a novel strategy, targeting essential genes like ATR, which is involved in DNA damage response, and pyruvate dehydrogenase kinases (PDKs), which play a role in energy metabolism. Experiments were conducted using sensitive PC9 and the corresponding osimertinib-resistant cells (PC9/OR), namely EGFR-mutant H1975 and KRAS-mutant A549 cells treated with TKIs, alongside ATR and DCA inhibitors. The results indicated that combining these approaches could enhance efficacy compared to TKIs alone, suggesting a tailored strategy based on tumor subtype. This research underscores the potential of new therapeutic targets to improve treatment outcomes in patients with NSCLC compared to traditional TKI therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f046003a233e91305d9f9f261ac2302b860f60ce" target='_blank'>
              Combined Therapeutic Strategies Based on the Inhibition of Non-Oncogene Addiction to Improve Tumor Response in EGFR- and KRAS-Mutant Non-Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Luisa Amato, Daniela Omodei, Caterina De Rosa, Annalisa Ariano, S. Capaldo, Camilla Carmela Tufano, Rossella Buono, Cristina Terlizzi, Anna Nardelli, Vitale Del Vecchio, Rosanna Palumbo, C. Tuccillo, F. Morgillo, Federica Papaccio, Virginia Tirino, F. Iommelli, C. D. Della Corte, Viviana De Rosa
          </td>
          <td>2024-11-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d49c3b89849fb97fd894e63e495575849c7b7545" target='_blank'>
              A multilineage screen identifies actionable synthetic lethal interactions in human cancers.
              </a>
            </td>
          <td>
            Samson H. Fong, Brent M Kuenzi, Nicole M Mattson, John Lee, Kyle S. Sanchez, A. Bojorquez-Gomez, Kyle Ford, Brenton P. Munson, Katherine Licon, Sarah Bergendahl, John Paul Shen, J. Kreisberg, Prashant Mali, Jeffrey H Hager, Michael A White, T. Ideker
          </td>
          <td>2024-11-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="OBJECTIVE
Colorectal cancer (CRC) is a type of digestive system cancer. At the molecular level, some factors, including genetic and epigenetic factors, as well as various signaling pathways such as oxidative stress and inflammation, play an active role in the onset of CRC. Genetic and epigenetic mutations, particularly in oncogenes and tumor suppressor genes, occur during colorectal adenocarcinoma development as a result of a change in gastrointestinal epithelial cell proliferation and self-renewal rates. This study aimed to determine the genes and molecular mechanisms that play a role in the emergence of this disease by analyzing the CRC data.


MATERIAL AND METHODS
Microarray data selected for bioinformatics analysis is Gene Expression data stored with the code GSE110224 in the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus (GEO) database. Gene expression analysis, functional clustering analysis, enrichment analysis, and pathway analysis were performed using this data set.


RESULTS
Analysis of raw transcriptomic data revealed 1770 common DEGs in CRC. While the expression level of 769 of these genes increased, the expression level of 1001 genes decreased. A Protein-protein interaction (PPI) network was created from the first 25 genes with increased expression levels and 11 signature genes were identified. Increased expression of REG1A, MMP3, FOXQ1 and CEMIP genes and decreased expression of AQP8, CA1, CLDN8, PYY, CA4, CEACAM7 and SLC30A10 genes were observed.


CONCLUSIONS
This approach revealed a CRC-specific molecular profile and may provide some guidance for further investigation of potential biomarkers for diagnosis and prognosis prediction of CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23912ddf1ff258ca2ed29c33ca9a5048f7e47144" target='_blank'>
              Bioinformatics analysis of colorectal cancer transcriptomic data reveals novel prognostic signature and potential biomarker genes.
              </a>
            </td>
          <td>
            S. Dalkılıç, Lütfiye Kadıoğlu Dalkılıç, Lütfü Uygur, Mustafa Timurkaan, Barış Gültürk, Mustafa Kaplan
          </td>
          <td>2024-12-07</td>
          <td>Scandinavian journal of gastroenterology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81af71d49f35f4ca08b391cf52780be4b4e726a9" target='_blank'>
              Integrated single-cell analysis reveals distinct epigenetic-regulated cancer cell states and a heterogeneity-guided core signature in tamoxifen-resistant breast cancer
              </a>
            </td>
          <td>
            Kun Fang, A. Ohihoin, Tianxiang Liu, Lavanya Choppavarapu, Bakhtiyor Nosirov, Qianben Wang, Xue-Zhong Yu, S. Kamaraju, Gustavo W. Leone, Victor X Jin
          </td>
          <td>2024-11-18</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d04c42082a8c74f3b67d96a365b6e54394480b2" target='_blank'>
              Leveraging homologous recombination deficiency for sarcoma
              </a>
            </td>
          <td>
            Lara Planas-Paz, Chantal Pauli
          </td>
          <td>2024-11-13</td>
          <td>Pathologie (Heidelberg, Germany)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The association between RET and multiple endocrine neoplasia type 2 was established in 1993 and remains one of the very few oncogenes for which distinct phenotypes (medullary thyroid cancer or phaeochromocytoma) are associated with the same hot-spot variants occurring either in germline or somatic DNA. Somatic RET fusion events have also been described in several cancers, including papillary thyroid cancer, non-small cell lung cancer, breast cancer, salivary gland cancer and pancreatic cancer. Highly selective RET inhibitors have improved outcomes in RET-altered cancers and have been well tolerated. Nevertheless, primary and acquired drug resistance has been observed, arising from distinct genomic alterations either in RET (on target resistance) or via alternate oncogenic pathways (bypass resistance). The same mechanisms of resistance have been observed across multiple cancer types, which implies RET-altered cancers evolve away from RET addiction via stochastic subclonal events. Understanding these mechanisms is crucial for identifying therapeutic opportunities to overcome resistance. Successful treatment targeting bypass oncogenes has been reported in several instances, at least for short-term outcomes; in contrast, although several compounds have been reported to overcome on-target RET alterations, none have yet been translated into routine clinical practice and this remains an area of urgent clinical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4b80f3267e487a6f53e42db32a243722c94787" target='_blank'>
              Mechanisms of resistance to RET-directed therapies.
              </a>
            </td>
          <td>
            R. J. Clifton-Bligh
          </td>
          <td>2024-12-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Studies of the genetics of Alzheimer's disease (AD) have largely focused on single nucleotide variants and short insertions/deletions. However, much of the disease heritability has yet to be uncovered, suggesting that other forms of genetic variation promote substantial portions of genetic risk. Uncovering the genetic basis of AD can lead to new disease biomarkers and delineate disease mechanisms. As there are over one million short tandem repeats (STRs) in the genome and pathogenic expansions of STR cause over 30 neurologic diseases, it is important to ascertain whether STRs may also be implicated in AD risk.


METHOD
Here, we utilize PCR-free whole genome sequencing data from 2,981 individuals (1,489 AD case and 1,492 control individuals) to study the role of STRs in AD. We applied two cutting-edge algorithms, ExpansionHunter and gangSTR, to genotype 321,742 STR tracts across the genome using the whole genome sequencing data. For each STR, we identified individuals carrying an expansion defined as having an STR tract length that is an outlier from the population. We then calculated the number of STR expansions in each individual and tested for differences in aggregate burden of STR expansions in AD case versus control individuals.


RESULT
Individuals with AD had a 1.19-fold increase burden of STR expansions compared to controls (p = 8.27 × 10-3, two-sided Mann Whitney test). Individuals carrying > 30 STR expansions had 3.62-fold higher odds of having AD and had more advanced spread of tau neuropathology as measured by Braak staging. AD STR expansions were highly enriched within gene regulatory elements active in the hippocampus.


CONCLUSION
These results demonstrate that harboring a high burden of STR expansions in the genome increases risk of AD and that STR expansions may serve as a novel genomic biomarker of AD risk and neuropathologic progression. Our findings suggest that STRs may disrupt gene expression and underscore the pivotal role of genomic instability in the onset and progression of AD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c97ba104dd79c532efc58b5c587f4cc601f3d1fb" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            Michael H Guo, Wan-Ping Lee, Badri N Vardarajan, Gerard D. Schellenberg, Jennifer E. Phillips-Cremins
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Although immune checkpoint inhibitors (ICIs) have revolutionized melanoma treatment, many individuals either show no response or eventually develop acquired resistance. This variability could be linked to differences in the tumor immune microenvironment, pre-existing drug-resistant cells or treatment induced changes in melanoma cell states. Using single cell RNA sequencing (scRNA-seq) coupled with heritable barcodes, this study aims to lineage trace changes in melanoma transcriptional cell states and map changes in the tumor immune microenvironment to uncover if preexisting and/or therapy-induced melanoma transcriptional cell states lead to resistance to ICIs. In this study, we used YUMMER1.7 mouse melanoma cells which were transduced with barcodes under conditions that allow delivery of a single unique barcode to each cell, this served as a cell lineage tag. Each barcode is heritable and stably transcribed into RNA molecules so that individual barcoded cells can be matched with a gene expression profile from scRNA-seq outputs. The barcoded cells were subcutaneously injected into C57BL/6 mice, and once tumors were established, mice were treated with 3 cycles of anti-PD-1 and anti-CTLA-4 therapy. Lineage tracing and scRNA-seq analyses were performed on tumors harvested prior to treatment, early on treatment (Day 6), during minimal residual disease (Day 13) and upon relapse. Early on treatment (Day 6)- two tumour groups were harvested, those that responded well and others that only partially responded. Our results showed that during the early on treatment phase (Day 6) tumors that responded well and those that only partially responded displayed distinct barcodes. Additionally, prevalent barcoded cells identified at the minimal residual disease phase were also dominant upon relapse. Analysis of the transcriptional states is underway to determine if ICI therapy induces transcriptional heterogeneity in sensitive and tolerant cells and if a particular or several transcriptional states lead to relapse. This study will potentially identify predictive response biomarkers and vulnerabilities of ICI naïve and resistant cells that could be targeted to improve outcomes for melanoma patients. We will also identify neoadjuvant approaches, to remove the inherent heterogeneity within the tumor cell population, facilitating a more uniform sensitivity to ICIs.
 Citation Format: Reem Saleh, Riyaben Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen Sheppard. Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Tumor-body Interactions: The Roles of Micro- and Macroenvironment in Cancer; 2024 Nov 17-20; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(22_Suppl):Abstract nr C044.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2046528ce162da3c656f0facbde8bbcbfa6faf03" target='_blank'>
              Abstract C044: Single cell RNA-sequencing coupled with lineage tracing identifies novel clonal populations associated with immunotherapy resistance
              </a>
            </td>
          <td>
            Reem Saleh, Riyaben P. Patel, Dane Vassiliadis, Fayrouz Hammal, Benjamin Blyth, Xin Du, Katie Fennell, Mark A. Dawson, Grant A. McArthur, Karen E. Sheppard
          </td>
          <td>2024-11-17</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1f5a7ee68cb6b5638219d48a2b0e5c5d417966" target='_blank'>
              A complex systems approach to mosaic loss of the Y chromosome.
              </a>
            </td>
          <td>
            Xihan Guo, Xueqin Dai
          </td>
          <td>2024-12-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromatin interactions between cis-regulatory elements (CREs) and specific gene promoters offer insights into the mechanisms of gene activation and repression. This knowledge is crucial for evaluating the effects of non-coding variants and chromatin alterations in cell differentiation or disease. Experimental techniques Hi-C and single-cell Hi-C provide information on CRE-promoter interactions, but the high costs and labor requirements limit their widespread use. We introduce UniversalEPI, an attention-based neural network designed to predict CRE-promoter interactions up to 2 Mb apart across unseen cell types without model retraining and using only DNA sequence and chromatin accessibility data as input. In UniversalEPI, chromatin accessibility profiles with masked non-accessible regions and DNA sequence data get passed through several convolution layers to identify binding motifs of three key transcription factors ubiquitously involved in promoter activation and CRE-promoter looping; a transformer block then takes the information from the convolutional levels passed through stochastic gating to predict chromatin interactions. While being a relatively lightweight model because of its architecture, UniversalEPI outperforms the state-of-the-art methods in inferring CRE-promoter interactions in unseen cell types across human cancerous and non-cancerous datasets. UniversalEPI can thus also be used to incorporate pseudo-bulk chromatin accessibility data from single-cell ATAC-seq experiments to pinpoint cell subtype-specific interactions. We illustrate this ability of UniversalEPI by tracking the dynamics of CRE-promoter interactions during macrophage reprogramming and the transition between cancer transcriptional states in esophageal adenocarcinoma. UniversalEPI thus advances in silico 3D chromatin modeling, supporting studies of chromatin dynamics in organism development and impacts of non-coding variants in cell differentiation and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3fd81a2650d0cef2f3db9d2292eaf00de44ac0" target='_blank'>
              UniversalEPI: harnessing attention mechanisms to decode chromatin interactions in rare and unexplored cell types
              </a>
            </td>
          <td>
            Aayush Grover, Lin Zhang, Till Muser, Simeon Häfliger, Minjia Wang, Josephine Yates, E. V. Van Allen, F. Theis, Ignacio L Ibarra, E. Krymova, Valentina Boeva
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Significant progress in determining the molecular origins and resistance mechanisms of mantle cell lymphoma (MCL) has improved our understanding of the disease's clinical diversity. These factors greatly impact prognosis in MCL patients. Given the dynamic alterations in MCL clones and disease evolution, it is crucial to recognize high-risk prognostic factors at diagnosis and relapse. Clinical factors include a high MCL International Prognostic Index score with a high Ki-67 proliferation index; early disease progression within 24 months of first-line treatment; >3 prior lines of therapy at relapse; and an aggressive (blastoid or pleomorphic) histology. Molecular aberrations include dysregulated cyclin D1; an aberrant SOX11-CD70 axis; upregulated Musashi-2; MYC rearrangement; metabolic reprogramming; and epigenetic changes. Other factors contributing to high-risk MCL include an immune-depleted microenvironment and clone adaptability with complex chromosomal anomalies and somatic mutations in TP53, NSD2, CCND1, CDKN2A, BIRC3, SP140, KMT2D, NFkBIE, SMARCA4, and NOTCH2. Ultra-high-risk MCL is indicated by the coexistence of multiple high-risk prognostic factors in the relapse setting and can portend very short progression-free survival. As MCL treatments advance towards cellular therapies, resistance to anti-CD19 chimeric antigen receptor T-cell therapy is also observed. These findings necessitate revisiting the prognostic impact of high-risk factors, current management strategies, new bi- and tri-specific T-cell engagers, combination therapies, novel therapeutic targets, and next-generation clinical trials for high-risk MCL patients. This article provides a comprehensive update on recognizing and managing high-risk MCL, encompassing current practices and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03364deb8d8c1c44077292ebb1c2031c1c0a3c37" target='_blank'>
              High-Risk MCL: Recognition and Treatment
              </a>
            </td>
          <td>
            P. Jain, Michael L Wang
          </td>
          <td>2024-12-30</td>
          <td>Blood Journal</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Organisms regulate gene expression in response to environmental cues, a process known as plasticity, to adjust to changing environments. Research into natural variation and the evolution of plasticity frequently studies cis-regulatory elements with theory suggesting they are more important evolutionarily than trans-regulatory elements. Genome-wide association studies have supported this idea, observing a predominance of cis-loci affecting plasticity. However, studies in structured populations provide a contrasting image, raising questions about the genetic architecture of natural variation in plasticity. To circumvent potential statistical difficulties present in genome-wide association studies, we mapped loci underlying transcriptomic plasticity in response to salicylic acid using recombinant inbred lines generated from two random Arabidopsis thaliana accessions. We detected extensive transgressive segregation in the salicylic acid response, suggesting that plasticity to salicylate in Arabidopsis is polygenic. Most loci (>75%) underlying this variation act in trans, especially for loci influencing plasticity. Trans-acting loci were enriched in genome hotspots, with predominantly small effect sizes distributed across many genes. This could potentially explain their under-discovery in genome-wide association studies. This work reveals a potentially important role for trans-acting loci in plastic expression responses, with implications for understanding plant adaptation to different environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6242103d645ac263c51e56f8b61d6b0403cdeb3b" target='_blank'>
              Trans-regulatory loci shape natural variation of gene expression plasticity in Arabidopsis
              </a>
            </td>
          <td>
            Mariele Lensink, Grey Monroe, Dan Kliebenstein
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74432db7e7bddd7e5f3b4f353ed49acd74ffe66d" target='_blank'>
              Getting the right combination to break the epigenetic code.
              </a>
            </td>
          <td>
            Seda S. Tolu, A. Viny, Jennifer E Amengual, Barbara Pro, Susan E Bates
          </td>
          <td>2024-12-02</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d173ca4b0414b7ce03e7c7ceb9b88bb092a887d" target='_blank'>
              Chemical inducers of proximity: precision tools for apoptosis in transcriptional regulation
              </a>
            </td>
          <td>
            Kaituo Huang
          </td>
          <td>2024-12-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c220b52569d4d73c77f6958bccc6ea49d4592f" target='_blank'>
              Increased translation driven by non-canonical EZH2 creates a synthetic vulnerability in enzalutamide-resistant prostate cancer
              </a>
            </td>
          <td>
            Shankha S Chatterjee, J. Linares, Tania Cid-Diaz, A. Duran, Mohd Imran K Khan, Marta Osrodek, Nicholas J Brady, Miguel Reina-Campos, A. Marzio, V. B. Venkadakrishnan, Martin K. Bakht, F. Khani, Juan Miguel Mosquera, Brian D. Robinson, Jenna Moyer, O. Elemento, Andrew C. Hsieh, David W. Goodrich, D. Rickman, H. Beltran, J. Moscat, Maria T. Diaz-Meco
          </td>
          <td>2024-11-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Abstract Treatment with genotoxic agents, such as platinum compounds, is still the mainstay therapeutical approach for the majority of cancers. Our understanding of the mechanisms of action of these drugs is, however, imperfect and continuously evolving. Recent advances highlighted single-stranded DNA (ssDNA) gap accumulation as a potential determinant underlying cisplatin chemosensitivity, at least in some genetic backgrounds, such as BRCA mutations. Cisplatin-induced ssDNA gaps form upon restart of DNA synthesis downstream of cisplatin-induced lesions through repriming catalyzed by the PRIMPOL enzyme. Here, we show that PRIMPOL overexpression in otherwise wild-type cells results in accumulation of cisplatin-induced ssDNA gaps without sensitizing cells to cisplatin, suggesting that ssDNA gap accumulation does not confer cisplatin sensitivity in BRCA-proficient cells. To understand how ssDNA gaps may cause cellular sensitivity, we employed CRISPR-mediated genome-wide genetic screening to identify factors which enable the cytotoxicity of cisplatin-induced ssDNA gaps. We found that the helicase HELQ specifically suppresses cisplatin sensitivity in PRIMPOL-overexpressing cells, and this is associated with reduced ssDNA accumulation. We moreover identify RAD52 as a mediator of this pathway. RAD52 promotes ssDNA gap accumulation through a BRCA-mediated mechanism. Our work identified the HELQ-RAD52-BRCA axis as a regulator of ssDNA gap processing and cisplatin sensitization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d8640e7a773e07a338c2a6edff813977332e73d" target='_blank'>
              CRISPR knockout genome-wide screens identify the HELQ-RAD52 axis in regulating the repair of cisplatin-induced single-stranded DNA gaps
              </a>
            </td>
          <td>
            Lindsey M Pale, Jude B. Khatib, Alexandra Nusawardhana, Joshua Straka, Claudia M. Nicolae, George-Lucian Moldovan
          </td>
          <td>2024-11-12</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a myeloproliferative disorder caused by the reciprocal translocation of the BCR-ABL1 oncogene, which activates tyrosine kinase resulting in uncontrolled cell proliferation and apoptosis suppression. Although Imatinib (IM) is an effective treatment, IM resistance remains a significant concern. Therefore, microRNAs (miRNAs) have emerged as potential alternative therapies. These short, non-coding RNAs (20-22 nucleotides) regulate gene expression by binding to the 3' untranslated region (3'UTR) of target genes. The study aims to identify miRNAs linked to CML, determine miRNA’s target genes, construct a protein-protein interaction (PPI) network, and analyze pathways and gene ontology. A literature search using the keywords “microRNA”, and “chronic myeloid leukemia” yielded relevant papers, which were screened and categorized into three groups: 1) miRNA associated with TKI resistance, 2) miRNA as biomarkers or leukemogenesis, and 3) miRNA as therapy. Target genes for the identified miRNAs were determined using DIANA tools and miRTarBase. Gene ontology and pathway analysis were conducted using DAVID, while PPI and network visualization were performed using STRING, Cytoscape, and ClueGO. Thirteen miRNAs were selected, targeting 782 genes and forming 16 clusters. ClueGO identified clusters associated with key biological processes, including G1/S cell cycle transition, miRNA-mediated gene silencing, hematopoietic stem cell differentiation, and apoptosis regulation. Target genes are also significant in the CML pathway and other cancer pathways, such as p53, ErbB, FoxO, autophagy, apoptosis, VEGF, TNF, and microRNA. Notably, hsa-miR-16 emerged as the most promising therapeutic and biomarker candidate for CML, highlighting its role in critical pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082a3893f78953cb45d6e871eb9d7b48eccbc043" target='_blank'>
              Role of MicroRNAs in Chronic Myeloid Leukemia (CML) Treatment, Biomarkers, and Resistance to Tyrosine Kinase Inhibitor: A Bioinformatics-Based Analysis
              </a>
            </td>
          <td>
            Syarifah Faezah binti Syed Mohamad, Rabiatul Adawiyah Ramlee, M. H. Elias
          </td>
          <td>2024-11-21</td>
          <td>Journal of Tropical Life Science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Taxanes and platinum molecules, specifically paclitaxel and carboplatin, are widely used anticancer drugs that induce cell death and serve as first-line chemotherapy for various cancer types. Despite the efficient effect of both drugs on cancer cell proliferation, many tumours have innate resistance against paclitaxel and carboplatin, which leads to inefficient treatment and poor survival rates. Haploid human embryonic stem cells (hESCs) are a novel and robust platform for genetic screening. To gain a comprehensive view of genes that affect or regulate paclitaxel and carboplatin resistance, genome-wide loss-of-function screens in haploid hESCs were performed. Both paclitaxel and carboplatin screens have yielded selected plausible gene lists and pathways relevant to resistance prediction. The effects of mutations in selected genes on the resistance to the drugs were demonstrated. Based on the results, an algorithm that can predict resistance to paclitaxel or carboplatin was developed. Applying the algorithm to the DNA mutation profile of patients' tumours enabled the separation of sensitive versus resistant patients, thus, providing a prediction tool. As the anticancer drugs arsenal can offer alternatives in case of resistance to either paclitaxel or carboplatin, an early prediction can provide a significant advantage and should improve treatment. The algorithm assists this unmet need and helps predict whether a patient will respond to the treatment and may have an immediate clinically actionable application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a5886b4a974b52231aaaccc4e3210ef163722ed" target='_blank'>
              Predicting tumour resistance to paclitaxel and carboplatin utilising genome-wide screening in haploid human embryonic stem cells.
              </a>
            </td>
          <td>
            J. Nissenbaum, Emanuel Segal, Hagit Philip, R. Cashman, T. Golan‐lev, Benjamin E. Reubinoff, Adi Turjeman, O. Yanuka, Elyad Lezmi, O. Kopper, N. Benvenisty
          </td>
          <td>2024-11-10</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Trisomy of human chromosome 21 (T21) gives rise to Down syndrome (DS), the most frequent live-born autosomal aneuploidy. T21 triggers genome-wide transcriptomic alterations that result in multiple atypical phenotypes with highly variable penetrance and expressivity in individuals with DS. Many of these phenotypes, including atypical neurodevelopment, emerge prenatally. To enable in vitro analyses of the cellular and molecular mechanisms leading to the neurological alterations associated with T21, we created and characterized a panel of genomically diverse T21 and euploid induced pluripotent stem cells (iPSCs). We subsequently differentiated these iPSCs to generate a panel of neural progenitor cells (NPCs). Alongside characterizing genotype effects from T21, we found that T21 NPCs showed inter-individual variability in growth rates, oxidative stress, senescence characteristics, and gene and protein expression. Pathway enrichment analyses of T21 NPCs identified vesicular transport, DNA repair, and cellular response to stress pathways. These results demonstrate T21-associated variability at the cellular level and suggest that cell lines from individuals with DS should not solely be analyzed as a homogenous population. Examining large cohorts of genetically diverse samples may more fully reveal the effects of aneuploidy on transcriptomic and phenotypic characteristics in T21 cell types. A panel of genomically diverse T21 and euploid induced pluripotent stem cells (iPSCs) were created and subsequently differentiated into neural progenitor cells (NPCs). T21 NPCs showed reduced growth, increased oxidative stress, and inter-individual variability in gene and protein expression. This inter-individual variability suggests that studies with large cohorts of genetically diverse T21 samples may more fully reveal the effects of aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b76c08b2fe900818339a97ad9dde96559af8a93" target='_blank'>
              Analysis of genotype effects and inter-individual variability in iPSC-derived trisomy 21 neural progenitor cells.
              </a>
            </td>
          <td>
            Sarah E Lee, Laura L Baxter, Monica I Duran, Samuel D Morris, Iman A Mosley, Kevin A Fuentes, Jeroen L. A. Pennings, F. Guedj, Diana W. Bianchi
          </td>
          <td>2024-11-13</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/374fa109d0f98cd344d26f23bd8b463547823902" target='_blank'>
              MiR-363-3p induces tamoxifen resistance in breast cancer cells through PTEN modulation
              </a>
            </td>
          <td>
            Yaning Liang, Cuiyu Shi, Yu Wang, Bingjie Fan, Wei Song, Rong Shen
          </td>
          <td>2024-12-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Amplification of the programmed cell death-ligand 1 gene (CD274) is highly prevalent and associated with a high response rate to immune checkpoint inhibitors (ICIs) in lymphomas, and is also a potential biomarker for ICI treatment of solid tumors. However, the efficacy of ICIs for solid tumors with CD274 amplification identified by comprehensive genomic profiling (CGP) has been unclear. We here examined ICI efficacy for solid tumors with CD274 amplification identified by CGP in a national database. Methods We retrospectively analyzed data from the Center for Cancer Genomics and Advanced Therapeutics database containing 60,155 CGP test results for individuals with solid tumors. Only clinical data from patients treated with ICIs alone (not those undergoing concomitant therapy with molecularly targeted or cytotoxic chemotherapeutic agents) were evaluated. We matched 48 patients in the CD274 amplification-positive group with 170 patients in the CD274 amplification-negative group in a 1:4 ratio based on tumor type, histology, treatment, and age. Overall survival (OS), time to next treatment (TTNT), and response rate were evaluated as treatment outcomes in the two groups. Results OS was similar in the CD274-amplified and matched CD274-non-amplified groups (median of 22.1 vs 26.3 months, respectively; HR of 0.92 with a 95% CI of 0.55 to 1.54; p=0.075). TTNT tended to be longer in the CD274-amplified group than in the matched CD274-non-amplified group (median of 16.5 vs 14.0 months; HR of 0.63 with a 95% CI of 0.37 to 1.08; p=0.091). The objective response rate was 33.3% and 18.4% (difference of 14.9%, with a 95% CI of –0.2% to 31.6%), and the disease control rate was 63.9% and 41.1% (difference of 22.8%, with a 95% CI of 5.1% to 40.4%), in the CD274-amplified and matched CD274-non-amplified groups, respectively. Conclusions The number of patients with solid tumors positive for CD274 amplification in this analysis is the largest to date, and our results suggest that such gene amplification may be associated with the outcome of ICI treatment in such individuals. CD274 amplification identified by CGP may therefore be a predictor of ICI efficacy for solid tumors. Trial registration number UMIN000029779.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e69b62b239b4cbdbf362661323ab2f6c2f830f1f" target='_blank'>
              Evaluation of immune checkpoint inhibitor efficacy for solid tumors with CD274 (PD-L1 gene) amplification identified by comprehensive genomic profiling: retrospective study based on a nationwide database
              </a>
            </td>
          <td>
            Tomohiro Nakayama, Takayuki Takahama, Y. Chiba, Naoki Shiraishi, H. Kawakami, K. Yonesaka, Kazuhiko Nakagawa, H. Hayashi
          </td>
          <td>2024-12-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="In malignant hematological diseases, clonal genetic alterations, such as chromosomal aberrations and gene mutations, are responsible for the uncontrolled division of abnormal hemopoietic cells. The detection of clonal variants has not only diagnostic, but also prognostic and therapeutic significance. They enable risk-based differentiated treatment of patients and the use of targeted (genotype-specific) therapies. Chromosomal abnormalities can be identified with cytogenomic testing (karyotyping, fluorescent in situ hybridization - FISH, microarray). In chronic myeloid leukemia, myelodysplastic neoplasia and acute leukemias, chromosome analysis is a mandatory test at the time of diagnosis. In some lymphoid malignancies (chronic lymphocytic leukemia, multiple myeloma), instead of karyotyping, submicroscopic abnormalities and translocations are detected by FISH method. Despite the rapid spread of high-sensitivity new-generation sequencing techniques, cytogenetic studies are still essential in the routine diagnosis of malignant hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71423c394e32005e4978325b7a14267c802e6e62" target='_blank'>
              [The role of cytogenetic tests in the diagnosis of malignant hematologic diseases].
              </a>
            </td>
          <td>
            A. Ujfalusi
          </td>
          <td>2024-12-10</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia-telangiectasia-mutated) and ATR (Ataxia-telangiectasia-mutated and Rad3-related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure to UV-B, UV-C and protons induced the microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana.
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ddc2e4963aa356dbc3de332034a633ed4d7780" target='_blank'>
              Antisense oligonucleotides as a targeted therapeutic approach in model of acute myeloid leukemia.
              </a>
            </td>
          <td>
            D. Maceckova, L. Vaňková, J. Bufka, Petr Hošek, Jiří Moravec, Pavel Pitule
          </td>
          <td>2024-12-18</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The ability of cancer cells to alter their identity, known as lineage plasticity, is crucial for tumor progression and therapy resistance. In lung adenocarcinoma (LUAD), tumor progression is characterized by a gradual loss of lineage fidelity and the emergence of non-pulmonary identity programs. This can lead to hybrid-identity (hybrid-ID) states in which developmentally incompatible identity programs are co-activated within individual cells. However, the molecular mechanisms underlying these identity shifts remain incompletely understood. Here, we identify the gastrointestinal (GI) transcriptional regulator HNF4α as a critical driver of tumor growth and proliferation in KRAS-driven LUAD. In LUAD cells that express the lung lineage specifier NKX2-1, HNF4α can induce a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also forms an aberrant protein complex with NKX2-1, which disrupts NKX2-1 localization and dampens pulmonary identity within hybrid-ID LUAD. Sustained signaling through the RAS/MEK pathway is critical for maintaining the hybrid-ID state. Moreover, RAS/MEK inhibition augments NKX2-1 chromatin binding at pulmonary-specific genes and induces resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion enhances sensitivity to pharmacologic KRASG12D inhibition. Collectively, our data show that co-expression of opposing lineage specifiers leads to a hybrid identity state that can drive tumor progression and dictate response to targeted therapy in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207f2a257f041c211d79bb19f39ca01a49097c42" target='_blank'>
              Opposing lineage specifiers induce a pro-tumor hybrid-identity state in lung adenocarcinoma
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Rushmeen Tariq, Anna Waters, Kayla T. O'Toole, Eric L. Snyder
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BRCA1, a critical tumor suppressor gene, plays an essential role in maintaining genomic stability through its involvement in DNA double-strand break repair, particularly via homologous recombination. Loss or impairment of BRCA1 function disrupts this repair pathway, resulting in genomic instability and significantly increased susceptibility to breast and ovarian cancers. To elucidate the molecular mechanisms by which BRCA1 mutations contribute to tumorigenesis, we employed quantitative mass spectrometry-based proximity labeling and affinity purification to identify cancer-specific protein-protein interactions (PPIs). Our integrated omics and visualization analysis of interactors revealed that the BRCA1-Y1853ter mutant, through its interaction with BARD1, perturbs the interactome and impacts cellular processes within the cytoplasm and nucleoplasm. Structural data further corroborated these findings, showing enhanced binding between the mutant BRCA1 and specific interactors, suggesting an altered functional profile. Together, these observations raise the hypothesis that the BRCA1-Y1853ter mutant may exhibit gain-of-function characteristics, providing new insights into the molecular and cellular effects of mutations in the BRCA1 C-Terminal (BRCT) domain and their implications for the pathogenesis of breast and ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7081d3dc4a7fe94ebd83524e06e557bb05ca412" target='_blank'>
              Differential Remodeling of the BRCA1 Interactome by a Unique Minimal Clinical Truncation Mutation
              </a>
            </td>
          <td>
            Jingjing Chen, Mihaela E. Sardiu, M. Rekowski, Jules Nde, Zachary Clark, S. Anant, Michael P. Washburn, Roy A Jensen
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The mitochondrial genome, which encodes genes essential for respiration and cellular homeostasis, is the target of abundant and highly diverse somatic alterations in cancers. Somatic alterations to mitochondrial DNA (mtDNA) nearly always arise heteroplasmically, producing heterogeneous ensembles of mtDNA within single cells. Here, we review new insights derived from exponential increases in genomic sequencing data that have uncovered the nature of, selective pressure for, and functional consequences of cancer-associated mtDNA alterations. As many discoveries have been limited by their ability to determine cell-to-cell variation in mtDNA genotype, we describe a new generation of single-cell sequencing approaches that resolve otherwise indeterminate models of mtDNA heteroplasmy. In tandem with novel approaches for mtDNA editing and modeling of mutations, these advances foreshadow the quantitative dissection of dosage-dependent mtDNA phenotypes that underlie both tumor evolution and heterogeneous response to therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823dce23cbbddef91125605bfb3305bbb4cdca3d" target='_blank'>
              Single-Cell Technologies for Studying the Evolution and Function of Mitochondrial DNA Heteroplasmy in Cancer
              </a>
            </td>
          <td>
            Sonia Boscenco, Erin M. Cumming, Minsoo Kim, Caleb Lareau, Ed Reznik
          </td>
          <td>2024-11-12</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chronic antigenic stimulation is central to marginal zone lymphoma (MZL) development. While the pharmacological inhibition of the B-cell receptor (BCR) signaling is initially effective, secondary resistance frequently develops. We conducted a CRISPR interference (CRISPRi) screen investigating enhancer-associated long non-coding RNAs (elncRNAs) in MZL cells to identify transcripts crucial to shape their dependence on BCR pathway activation. We identified LOC730338, an elncRNA associated with A-to-I RNA editing, which we renamed ADARreg. Silencing ADARreg re-sensitized tumor cells to BCR pathway inhibition by modulating ADAR2 nuclear translocation and altering intronic RNA editing. The process preferentially affected genes that enhance immune responses, such as STING, IRF3, and p65. ADARreg knockdown also increased lymphoma cell sensitivity to NK cell-mediated cytotoxicity. Our results indicate that targeting elncRNAs like ADARreg represents a potential strategy to overcome drug resistance in lymphoma, also opening new therapeutic opportunities for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53b0f5b32756fe2cd1cc1438cd5631c98fcc3016" target='_blank'>
              Enhancer lncRNA LOC730338 limits the innate immune response against lymphoma cells, inhibiting ADAR2-dependent alternative transcription
              </a>
            </td>
          <td>
            Luciano Cascione, F. Guidetti, S. Ramnarayanan, Andrea Rinaldi, F. Spriano, Alex Zadro, C. Tarantelli, Nicolas Munz, A. Arribas, Rory Johnson, Francesco Bertoni, S. Napoli
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc92ee9eeeb11fdc0010923fa936390ffdc668a" target='_blank'>
              Multi-omic and single-cell profiling of chromothriptic medulloblastoma reveals genomic and transcriptomic consequences of genome instability
              </a>
            </td>
          <td>
            Petr Smirnov, M. Przybilla, Milena Simovic-Lorenz, R. G. Parra, H. Susak, Manasi Ratnaparkhe, John K. L. Wong, V. Körber, Jan-Philipp Mallm, George Philippos, M. Sill, Thorsten Kolb, Rithu Kumar, Nicola Casiraghi, K. Okonechnikov, David R. Ghasemi, K. Maass, K. Pajtler, Anna Jauch, A. Korshunov, Thomas Höfer, M. Zapatka, Stefan M Pfister, W. Huber, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>109</td>
        </tr>

        <tr id="Base editing is a CRISPR-based technology that enables high-throughput, nucleotide-level functional interrogation of the genome, which is essential for understanding the genetic basis of human disease and informing therapeutic development. Base editing screens have emerged as a powerful experimental approach, yet significant cell-to-cell variability in editing efficiency introduces noise that may obscure meaningful results. Here, we develop a co-selection method that enriches for cells with high base editing activity, substantially increasing editing efficiency at a target locus. We evaluate this activity-based selection method against a traditional screening approach by tiling guide RNAs across TP53, demonstrating its enhanced capacity to pinpoint specific mutations and protein regions of functional importance. We anticipate that this modular selection method will enhance the resolution of base editing screens across many applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08e9009471315625c4ef8a88426a3b56b32f40bd" target='_blank'>
              Activity-based selection for enhanced base editor mutational scanning
              </a>
            </td>
          <td>
            Eleanor G Kaplan, Ryan J Steger, Spencer T Shah, Laura M Drepanos, Audrey L. Griffith, Ganna Reint, John G Doench
          </td>
          <td>2024-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="BACKGROUND The mitotic DNA integrity checkpoint signaling pathway is potentially involved in cancers that regulate genomic stability where protein kinases play a pivotal role. 16 total protein kinase genes are involved in this pathway: ATM, BRSK1, CDK1, CDK2, CHEK1, CHEK2, MAP3K20, NEK11, PLK1, PLK2, PLK3, PRKDC, STK33, TAOK1, TAOK2, and TAOK3. This study aims to provide pan-cancer profiles of the protein kinases in mitotic DNA integrity checkpoint signaling gene set for potential prognostic and diagnostic purposes, as well as future potential therapeutic targets for cancer in a clinical setting. METHODS Multi-omic data was acquired for the 16 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of SNV, CNV, methylation, mRNA expression, pathway crosstalk, and microRNA regulation networks. RESULTS The SNV profile showed that most of these genes have a high SNV mutation frequency across some cancer types, such as UCEC and SKCM. The CNVs of some of these genes are associated with the survival of UCEC, KIRP, and LGG. BRCA, KIRC, LUAD, and STAD might be affected by the mRNA expression of these genes which might involve regulation of copy number, methylation, and miRNA. In addition, these genes also cross-talk with some known cancer pathways. CONCLUSION The protein kinases in mitotic DNA integrity checkpoint signaling may play a role in cancer development and, with adequate research, could potentially be developed as biomarkers for cancer diagnosis and prognosis. However, further efforts are necessary to validate their clinical value for diagnosis and prognosis and to develop practical applications in clinical settings. Nevertheless, these pan-cancer profiles offer a better overall understanding as well as useful information for future reference regarding mitotic DNA integrity checkpoint signaling in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bfd3e9f554799cf23b8ba08f42b425055efabcc" target='_blank'>
              Pan-Cancer Genetic Profiles of Mitotic DNA Integrity Checkpoint Protein Kinases
              </a>
            </td>
          <td>
            Ayana Meegol Rasteh, Hengrui Liu, Panpan Wang
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human T-cell leukemia virus type 1 (HTLV-1) broadly impacts host genes, affecting the infected cell population and inducing the development of a disease with a poor prognosis, adult T-cell leukemia-lymphoma (ATL). This study aimed to provide a comprehensive epigenomic characterization of the infected cell population and evaluated the transcriptome and chromatin structures of peripheral blood cells in HTLV-1-infected individuals using RNA sequencing (RNA-seq) and assay for transposase-accessible chromatin sequencing (ATAC-seq). The infected cells showed significant changes in gene expression patterns from the polyclonal stage and before ATL onset while demonstrating similarities to tumor-forming ATL cells. These similarities were a result of large-scale open chromatin changes, supporting the independent early formation of epigenomic aberrations as an underlying mechanism for later clonal propagation. This study also demonstrated that HTLV-1 Tax directly affects the host chromatin structure, thereby developing fundamental epigenomic characteristics. Several Tax target genes, including the RASGRP3-ERK pathway, were recognized, indicating an impact on signaling pathways. This genome-wide variability in chromatin structural property is a novel feature of HTLV-1 infection and may contribute to pathogenic mechanisms. In addition, it has crucial implications for better understanding the impact of HTLV-1 on the host genome and identifying novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f7c450fea8921cdae499b880a5d0560a4c49aa" target='_blank'>
              Rewired chromatin structure and epigenetic gene dysregulation during HTLV-1 infection to leukemogenesis.
              </a>
            </td>
          <td>
            Jun Mizuike, Kako Suzuki, Shu Tosaka, Yuta Kuze, Seiichiro Kobayashi, Makoto Nakashima, Koji Jimbo, Y. Nannya, Yutaka Suzuki, K. Uchimaru, Makoto Yamagishi
          </td>
          <td>2024-11-19</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Type 2 diabetes (T2D) is a chronic metabolic disorder marked by insulin resistance and relative insulin deficiency, affecting over 422 million people globally and projected to increase, making it a major public health concern. This condition is associated with severe complications such as retinopathy, nephropathy, cardiovascular diseases, and neuropathy, highlighting the need for a deeper understanding of its mechanisms to develop more effective prevention and treatment strategies. Transcriptomic analysis, particularly through RNA-seq, has provided valuable insights into the gene expression patterns in T2D, highlighting pathways involved in insulin signaling, metabolic regulation, and inflammation. This approach, including single-cell RNA sequencing, helps overcome the challenges of immune cell population heterogeneity, enabling the identification of distinct cell types and their specific roles in T2D progression. In this paper, we present a dataset of single-cell and bulk RNA sequencing results comparing expression patterns in blood cells between diabetes and healthy samples and describe some preliminary analysis of the dataset, highlighting differences in both gene expression and cell type composition and putting them in the context of existing research of T2D.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a51901dd5557db1a40b6d73e6a320f7468575a8" target='_blank'>
              Deciphering the Transcriptomic Landscape of Type 2 Diabetes: Insights from Bulk RNA Sequencing and Single-Cell Analysis
              </a>
            </td>
          <td>
            A. A. Tkachenko, Z. Tonyan, Y. A. Nasykhova, Y. A. Barbitoff, I. N. Renev, M. M. Danilova, A. Mikhailova, O. B. Glavnova, S. Chepanov, S. A. Selkov, N. V. Golovkin, M. E. Vlasova, A. S. Glotov
          </td>
          <td>2024-11-08</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9d497e065e31082024ffd65467aebd24c60b68f" target='_blank'>
              OXPHOS mediators in acute myeloid leukemia patients: Prognostic biomarkers and therapeutic targets for personalized medicine
              </a>
            </td>
          <td>
            A. K. Abdel-Aziz
          </td>
          <td>2024-11-12</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="INTRODUCTION
Acute myeloid leukemia (AML) is an aggressive and poor-prognosis blood cancer. Despite a low mutation burden compared to other cancers, AML is heterogenous and identifying robust therapeutic targets has been difficult. Genomic profiling has greatly advanced our understanding of AML, and has revealed targets for AML therapy. However, only 50% of AML patients have gene mutations that are currently druggable, and relapse rates remain high. The addition of proteomic profiling is emerging to address these challenges.


AREAS COVERED
Using references collected through Pubmed, we review recent studies that have combined genomic and proteomic profiling (i.e. proteogenomic profiling), as well as studies that have additionally integrated other omics approaches, such as phosphoproteomics. We highlight how proteogenomic profiling promises to deconvolve the cellular pathways driving leukemogenesis, uncover novel therapeutic targets, and identify biomarkers of response to novel and existing therapies.


EXPERT OPINION
Proteogenomic profiling is providing unparalleled insight into AML, and is beginning to identify robust biomarkers. Standardization of workflows will be required before mass spectrometry-based proteomic assays can be integrated into routine clinical use. However, the demonstrated ability to adapt signatures into biomarker panels that can be assayed by existing clinical workflows is enabling current clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00da146263cc574dc14279fe5c3bf5fbe1f370e6" target='_blank'>
              Proteogenomic profiling of acute myeloid leukemia to identify therapeutic targets.
              </a>
            </td>
          <td>
            Heather C Murray, Jonathan R. Sillar, Maddison Chambers, N. Verrills
          </td>
          <td>2024-11-22</td>
          <td>Expert review of proteomics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The development of DNA damage response (DDR)-directed therapies is a major area of clinical investigation, yet to date Poly (ADP-ribose) polymerase (PARP) inhibitors remain the only approved therapy in this space. Major challenges to DDR-targeted therapies in the post-PARPi era are the context dependency of DDR alterations and the presence of pre-existing resistance in this heavily pre-treated population. To that end, we used a contemporary platform to analyze pre-treatment circulating tumor DNA (ctDNA) samples from 173 patients enrolled onto two Phase 1/2 trials harboring pathogenic variants (PVs) in DDR genes. Baseline ctDNA analysis revealed a wealth of insights, including circulating tumor fraction estimation, impact of clonal hematopoiesis, PV allelic status, homologous recombination deficiency (HRD) signatures and presence of pre-existing resistance. HRD reversions were detected in 44% of evaluable patients and included large genomic rearrangements leading to deletion of whole or partial exons. We also discovered reversion of ATM in two patients previously treated with platinum chemotherapy, which has not previously been described. This study showcases the genomic complexity of DDR-altered tumors, revealed through baseline ctDNA profiling, an understanding of which is crucial for the future clinical development of novel DDR-directed therapies and combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ea40ca69f43c15a5bcd04ecbea4a02554caede3" target='_blank'>
              Comprehensive baseline ctDNA characterization in two biomarker-selected Phase 1/2 studies using genomic and methylation profiling
              </a>
            </td>
          <td>
            I. Silverman, J. Schonhoft, B. Herzberg, A. Yablonovitch, E. Lagow, S. Sethuraman, D. Ulanet, J. Yang, I. Kim, P. Basciano, M. Cecchini, E. Lee, S. Lheureux, E. Fontana, B. A. Carneiro, J. Reis-Filho, T. Yap, M. Zinda, E. Rosen, V. Rimkunas
          </td>
          <td>2024-11-11</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Expansions and contractions of tandem DNA repeats are a source of genetic variation in human populations and in human tissues: some expanded repeats cause inherited disorders, and some are also somatically unstable. We analyzed DNA sequence data, derived from the blood cells of >700,000 participants in UK Biobank and the All of Us Research Program, and developed new computational approaches to recognize, measure and learn from DNA-repeat instability at 15 highly polymorphic CAG-repeat loci. We found that expansion and contraction rates varied widely across these 15 loci, even for alleles of the same length; repeats at different loci also exhibited widely variable relative propensities to mutate in the germline versus the blood. The high somatic instability of TCF4 repeats enabled a genome-wide association analysis that identified seven loci at which inherited variants modulate TCF4 repeat instability in blood cells. Three of the implicated loci contained genes (MSH3, FAN1, and PMS2) that also modulate Huntington’s disease age-at-onset as well as somatic instability of the HTT repeat in blood; however, the specific genetic variants and their effects (instability-increasing or-decreasing) appeared to be tissue-specific and repeat-specific, suggesting that somatic mutation in different tissues—or of different repeats in the same tissue—proceeds independently and under the control of substantially different genetic variation. Additional modifier loci included DNA damage response genes ATAD5 and GADD45A. Analyzing DNA repeat expansions together with clinical data showed that inherited repeats in the 5’ UTR of the glutaminase (GLS) gene are associated with stage 5 chronic kidney disease (OR=14.0 [5.7–34.3]) and liver diseases (OR=3.0 [1.5–5.9]). These and other results point to the dynamics of DNA repeats in human populations and across the human lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63d6786cd91e6aa465caa485b8915ace9338470d" target='_blank'>
              Insights into the causes and consequences of DNA repeat expansions from 700,000 biobank participants
              </a>
            </td>
          <td>
            Margaux L.A. Hujoel, R. Handsaker, Nolan Kamitaki, Ronen E. Mukamel, Simone Rubinacci, P. Palamara, S. Mccarroll, Po-Ru Loh
          </td>
          <td>2024-11-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Fatty acid binding protein 7 (FABP7) is a multifunctional chaperone involved in lipid metabolism and signaling. It is primarily expressed in astrocytes and neural stem cells (NSCs), as well as their derived malignant glioma cells within the central nervous system. Despite growing evidence for FABP7’s tumor-intrinsic onco-metabolic functions, its mechanistic role in regulating the brain tumor immune microenvironment (TIME) and its impact on prognosis at the molecular level remain incompletely understood. Utilizing combined transcriptome profiling and pan-cancer analysis approaches, we report that FABP7 mediates the expression of multiple onco-immune drivers, collectively impacting tumor immunity and clinical outcomes across brain cancer subtypes. An analysis of a single-cell expression atlas revealed that FABP7 is predominantly expressed in the glial lineage and malignant cell populations in gliomas, with nuclear localization in their parental NSCs. Pathway and gene enrichment analysis of RNA sequencing data from wild-type (WT) and Fabp7-knockout (KO) mouse brains, alongside control (CTL) and FABP7-overexpressing (FABP7 OV) human astrocytes, revealed a more pronounced effect of FABP7 levels on multiple cancer-associated pathways. Notably, genes linked to brain cancer progression and tumor immunity (ENO1, MUC1, COL5A1, and IL11) were significantly downregulated (>2-fold) in KO brain tissue but were upregulated in FABP7 OV astrocytes. Furthermore, an analysis of data from The Cancer Genome Atlas (TCGA) showed robust correlations between the expression of these factors, as well as FABP7, and established glioma oncogenes (EGFR, BRAF, NF1, PDGFRA, IDH1), with stronger associations seen in low-grade glioma (LGG) than in glioblastoma (GBM). TIME profiling also revealed that the expression of FABP7 and the genes that it modulates was significantly associated with prognosis and survival, particularly in LGG patients, by influencing the infiltration of immunosuppressive cell populations within tumors. Overall, our findings suggest that FABP7 acts as an intracellular regulator of pro-tumor immunomodulatory genes, exerting a synergistic effect on the TIME and clinical outcomes in brain cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15939c78a79b5a83578829fc7d538cf232f24932" target='_blank'>
              Integrated Transcriptome Profiling and Pan-Cancer Analyses Reveal Oncogenic Networks and Tumor-Immune Modulatory Roles for FABP7 in Brain Cancers
              </a>
            </td>
          <td>
            Yool Lee, Carlos C. Flores, Micah Lefton, Sukanya Bhoumik, Yuji Owada, Jason R. Gerstner
          </td>
          <td>2024-11-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e682f9ff1366b8e5398704d90d9b219b2fb57501" target='_blank'>
              Whole genome sequencing of CRISPR/Cas9-engineered NF-κB reporter mice for validation and variant discovery
              </a>
            </td>
          <td>
            Guruswamy Mahesh, Erik W. Martin, Mohammad Aqdas, Kyu-Seon Oh, Myong-Hee Sung
          </td>
          <td>2024-11-13</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 The treatment of many solid tumors presents significant challenges with chemotherapy, radiation therapy, and the limited effectiveness of immunotherapy. Targeted therapy offers a promising approach, yet the lack of validated targets limits its applicability across the spectrum of solid tumors. Our past studies have revealed that an R-loop and mRNA-dependent DNA repair (RDDR) pathway, induced by damage at the transcribed regions of the genome, contributes to cell survival and drug resistance in cancer cells. This is particularly due to the high levels of transcription and DNA damage in these cells. We identified the RNA methyltransferase TRDMT1 as the primary modifier of mRNA methyl-5-cytosine in the RDDR pathway. Overcoming technical difficulties in monitoring RNA modifications, we developed the first TRDMT1 inhibitor (TRDMT1i) using in-house built cell-based and in vitro assays. We have identified highly potent and specific lead compounds that significantly reduce TRDMT1 activity at nanomolar concentrations. Our current lead TRDMT1 inhibitors meets most of the criteria that are required for optimal lead. Initial assessments involving the screening of normal and cancer cell lines, xenograft models, and patient specimens indicate that TRDMT1i sensitizes tumors with genomic instability when used as monotherapy. We aim to develop TRDMT1i as an investigational new drug (IND) for future clinical trials, initially targeting ovarian cancer as a monotherapy or in combination therapy for first-line maintenance and systematic therapy. Our goal is to pioneer the discovery of the RDDR pathway, leading to the development of innovative platforms and next-generation medications to revolutionize the targeting of mRNA modifications in cancer treatment.
 Citation Format: Li Lan. Targeting mRNA methyltransferase in RNA-dependent DNA repair in cancer therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5c058059942f92471fbbee99fbac2c5e418612" target='_blank'>
              Abstract PR001: Targeting mRNA methyltransferase in RNA-dependent DNA repair in cancer therapy
              </a>
            </td>
          <td>
            Li Lan
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Hepatocellular carcinoma (HCC) is one of the most lethal cancers, usually diagnosed at an advanced stage. Metabolic reprogramming plays a significant role in HCC progression, probably related to immune evasion, yet the key gene is unclear. In this study, six metabolism‐related genes with prognostic implications were screened. Correlation analysis between the key genes and immune cell subtypes was conducted, and a prominent gene strongly associated with immunosuppression, SLC16A3, was identified. Overexpression of SLC16A3 is associated with the loss of T‐cell function and might lead to the upregulation of several immunosuppressive proteins. Gene function enrichment analysis showed genes correlated with SLC16A3 primarily involved in cell adhesion. Single‐cell analysis showed that the SLC16A3 gene was mainly expressed in macrophages, especially some tumour‐promoting macrophages. Further analysis of spatial transcriptome data indicated that SLC16A3 was enriched at the tumour invasion front. The mIHC revealed that patients with high SLC16A3 expression exhibited significantly reduced infiltration of GZMB+ cells. And SLC16A3 inhibitors significantly suppressed the proliferation of HCC, while simultaneously enhancing T‐cell cytotoxicity and reducing exhaustion. These results reveal the phenomenon of immune escape mediated by metabolic reprogramming and suggest that SLC16A3 may serve as a novel target for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09067585cbbd1095414cbdd6644f6a4c19d71e0f" target='_blank'>
              Integrated Single‐Cell and Spatial Transcriptome Reveal Metabolic Gene SLC16A3 as a Key Regulator of Immune Suppression in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Qianlong Kang, Xiaomeng Yin, Zhenru Wu, Aiping Zheng, Lusi Feng, Xuelei Ma, Li Li
          </td>
          <td>2024-12-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Intron retention is a type of alternative splicing in which introns remain unspliced in mature RNA transcripts. In order to explore the landscape and consequences of genetically regulated intron retention, we perform an intron retention quantitative trait loci (irQTL) analysis in 49 human tissues across 838 individuals. We identify 8,624 unique intron retention events associated with genetic polymorphisms. 1,369 irQTLs (16%) are also associated with genome-wide association study (GWAS) traits, suggesting that genetically regulated intron retention may play a causal role in the regulation of these GWAS traits. 1,999 irQTLs (23%) colocalize with eQTLs to their respective gene, suggesting that intron retention and steady-state gene expression levels are regulated by shared sets of genetic variants. Through a colocalization analysis of irQTL:eQTL events, we demonstrate that irQTLs are sufficient to generate eQTLs when one of the alternatively spliced transcripts is preferentially targeted by the nonsense mediated decay (NMD) pathway. Surprisingly, for intron retention events whose potential NMD effects can be confidently predicted based on their positions within known gene annotations, we find that 58.8% (923/1570) of the colocalized irQTL and eQTL pairs show effect size directions that are discordant with the NMD model. Moreover, we find that irQTLs are significantly more likely to occur in the same gene with the same effect size direction as compared to exon skipping QTLs. Through mathematical modeling and analysis of experimental perturbation data, we provide evidence that eQTLs are able to generate irQTLs by altering the steady state ratios of spliced and unspliced transcripts, and we postulate that this mechanism may partially underlie the widespread intron retention observed previously in various biological conditions. Taken together, these results show that intron retention and steady state gene expression levels are closely intertwined to regulate phenotypic traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7207fff2c80c573e0b32d3cba5d95f5f23832382" target='_blank'>
              Origins and Implications of Intron Retention Quantitative Trait Loci in Human Tissues
              </a>
            </td>
          <td>
            Eddie Park, Yi Xing
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
The Cancer Genome Atlas identified four distinct molecular subtypes of endometrial cancer (EC): POLE mutated, mismatch repair deficient (dMMR), copy number low, and copy number high. The goal of this review is to summarize the profound clinical implications of molecular subtyping, particularly in guiding treatment decisions for dMMR and microsatellite instability high (MSI-H) EC.


RECENT FINDINGS
Clinical trials have demonstrated the remarkable efficacy of immunotherapy in dMMR/MSI-H EC tumors. Trials including GARNET, KEYNOTE-158, NRG GY-018, and RUBY have shown significant improvements in clinical outcomes for patients with advanced and recurrent disease, leading to FDA approvals for immunotherapy in both frontline and recurrent EC treatment settings.Building on these successes, recent studies, including DUO-E, are exploring combination therapies to enhance the efficacy of immunotherapy in EC. Simultaneously, trials including NRG GY-020, are investigating the potential benefits of immunotherapy in early-stage disease.


SUMMARY
Immunotherapy therapy has revolutionized the treatment of endometrial cancer in both upfront and recurrent settings, with molecular subtyping identifying patients most likely to benefit, especially those with dMMR/MSI-H tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/393fc2865caf4bb12ae0736a3a6d4578650e38fd" target='_blank'>
              The advent of immune checkpoint inhibition for the treatment of patients with primary advanced or recurrent dMMR/MSI high endometrial cancer in 2025.
              </a>
            </td>
          <td>
            Tarik Silk, Kari Hacker, W. Growdon, B. Pothuri
          </td>
          <td>2024-12-02</td>
          <td>Current opinion in obstetrics & gynecology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) patients that fail to achieve a complete metabolic response with frontline immunochemotherapy have a poor prognosis. Genomic profiling has led to a broader understanding of the molecular drivers in DLBCL, but it is unknown how well current classifiers identify patients that will experience primary treatment resistance (PTR). Using whole exome and RNA sequencing data from newly diagnosed DLBCL patients, we evaluated the genomic landscape of PTR and compared it to that of non‐PTR DLBCL. We found a significant increase in the frequency of TP53 (34% vs. 15%, p = 0.005) and ARID1A mutations (21% vs. 7%, p = 0.007) in PTR cases, with pathway analysis further demonstrating a downregulation of TP53 and an increase in chromatin modifying pathways. These results suggest that TP53 and ARID1A may be key mediators of PTR and important pathways contributing to the poor outcomes. We found that the current molecular classifiers were unable to identify PTR cases at diagnosis. However, our newly identified high‐risk signature identified 46% of PTR cases at diagnosis. Overall, these results contribute to our understanding of the genomic landscape of patients with primary treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dad9b8db809f309807b25afc6dc6fdb4c12b2eb" target='_blank'>
              Molecular Features of Diffuse Large B‐Cell Lymphoma Associated With Primary Treatment Resistance
              </a>
            </td>
          <td>
            A. M. Bock, K. Wenzl, J. Novak, M. Stokes, M. Hopper, J. Krull, A. Dropik, V. Sarangi, María Ortiz, Nicholas Stong, C. Huang, M. Maurer, Rebecca L. King, U. Farooq, Yucai Wang, T. Witzig, S. Ansell, T. Habermann, J. Cerhan, A. K. Gandhi, Grzegorz Nowakowski, A. Novak
          </td>
          <td>2024-11-29</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>127</td>
        </tr>

        <tr id="Therapies targeting mutated RAS, the most frequently mutated oncogene in human cancers, could benefit millions of patients. Recently approved RAS inhibitors represent a breakthrough, but are limited to a specific KRASG12C mutation and prone to resistance. Synthetic gene circuits offer a promising alternative by sensing and integrating cancer-specific biomolecular inputs, including mutated RAS, to selectively express therapeutic proteins in cancer cells. A key challenge for these circuits is achieving high cancer selectivity to prevent toxicity in healthy cells. To address this challenge, we present a novel approach combining multiple RAS sensors into RAS-targeting gene circuits, which allowed us to express an output protein in cells with mutated RAS with unprecedented selectivity. We implemented a modular design strategy and modelled the impact of individual circuit components on output expression. This enabled cell-line specific adaptation of the circuits to optimize selectivity and fine-tune expression. We further demonstrate the targeting capabilities of the circuits by employing them in different RAS-driven cancer cells. This work highlights the potential of synthetic gene circuits as a novel therapeutic strategy for RAS-driven cancers, advancing the application of synthetic biology in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51f32597c4aa4f2a9295599f20b89af450418867" target='_blank'>
              Synthetic gene circuits that selectively target RAS-driven cancers
              </a>
            </td>
          <td>
            Gabriel Senn, Leon Nissen, Y. Benenson
          </td>
          <td>2024-11-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cellular differentiation and homeostasis rely on complex mechanisms to control gene expression, enabling the different cell lineages of an organism to establish and then "memorize" different epigenetic states. The processes that control gene expression are centered on chromatin, a complex of DNA, histone proteins and RNA, whose structure is finely regulated. Targeted epigenomic engineering tools make it possible to interfere with and study these processes, revealing the logic of epigenetic memory mechanisms. This article reviews the main classes of targeted epigenome modification tools and illustrates how they can be used to better understand and modify the epigenome of cells, paving the way for potentially revolutionary therapeutic prospects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bf48ee21647af051d470ca1e8d40b7d4279ad6" target='_blank'>
              [Targeted epigenome engineering].
              </a>
            </td>
          <td>
            Hedvika Martin, Michel Wassef
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In recent years, gene editing technologies have rapidly evolved to enable precise and efficient genomic modification. These strategies serve as a crucial instrument in advancing our comprehension of genetics and treating genetic disorders. Of particular interest is the manipulation of large DNA fragments, notably the insertion of large fragments, which has emerged as a focal point of research in recent years. Nevertheless, the techniques employed to integrate larger gene fragments are frequently confronted with inefficiencies, off-target effects, and elevated costs. It is therefore imperative to develop efficient tools capable of precisely inserting kilobase-sized DNA fragments into mammalian genomes to support genetic engineering, gene therapy, and synthetic biology applications. This review provides a comprehensive overview of methods developed in the past five years for integrating large DNA fragments with a particular focus on burgeoning CRISPR-related technologies. We discuss the opportunities associated with homology-directed repair (HDR) and emerging CRISPR-transposase and CRISPR-recombinase strategies, highlighting their potential to revolutionize gene therapies for complex diseases. Additionally, we explore the challenges confronting these methodologies and outline potential future directions for their improvement with the overarching goal of facilitating the utilization and advancement of tools for large fragment gene editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05aaf5b1a8b083f9c6bd065129f6dea59d703d40" target='_blank'>
              CRISPR/Cas-Based Gene Editing Tools for Large DNA Fragment Integration.
              </a>
            </td>
          <td>
            Shuhan Yang, Guang Hu, Jianming Wang, Jie Song
          </td>
          <td>2024-12-16</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc282ab7c49fa5f1e3334e1737972fbc08f7f900" target='_blank'>
              Single cell analysis identified a basal cell transition state associated with the development and progression of bladder cancer
              </a>
            </td>
          <td>
            Yang Li, Pengjie Shi, Yuhong Ding, Zhipeng Yao, Lilong Liu, Junyi Hu, Zhenghao Liu, Jinxu Li, Ke Chen, Yaxin Hou
          </td>
          <td>2024-11-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Currently, numerous associations between genetic polymorphisms and various diseases have been characterized through Genome-Wide Association Studies. The majority of clinically significant polymorphisms are localized in non-coding regions of the genome. While modern bioinformatic resources make it possible to predict molecular mechanisms that explain the influence of non-coding polymorphisms on gene expression, such hypotheses require experimental verification. This review discusses the methods for elucidating the molecular mechanisms underlying the dependence of disease pathogenesis on specific genetic variants within non-coding sequences. A particular focus is on the methods to identify the transcription factors with binding efficiency contingent upon polymorphic variations. Despite remarkable progress in the bioinformatic resources enabling the prediction of the impact of polymorphisms on disease pathogenesis, the imperative for experimental approaches to this inquiry still persists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0e300f57318f3e11948f7c4bd85a221591ff36c" target='_blank'>
              Methods for Functional Characterization of Genetic Polymorphisms of Non-Coding Regulatory Regions of the Human Genome (Review)
              </a>
            </td>
          <td>
            A. Uvarova, E. A. Tkachenko, E. M. Stasevich, E. A. Zheremyan, K. Korneev, D. V. Kuprash
          </td>
          <td>2024-12-25</td>
          <td>Biohimiâ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Introduction Despite remarkable progress in Pediatric Acute Myeloid Leukemia (pAML) treatments, the relapsed disease remains difficult to treat, making it pertinent to identify novel biomarkers of prognostic/therapeutic significance. Material and methods Bone marrow samples from 21 pAML patients were analyzed using single cell RNA sequencing, functional assays with ARMH1 knockdown and overexpression were performed in leukemia cell lines to evaluate impact on proliferation and migration, and chemotherapy sensitivity. Mitochondrial function was assessed via Seahorse assay, ARMH1 interacting proteins were studied using co-immunoprecipitation. Bulk RNA-seq was performed on ARMH1knockdown and over expressing cell lines to evaluate the pathways and networks impacted by ARMH1. Results Our data shows that ARMH1, a novel cancer-associated gene, is highly expressed in the malignant blast cells of multiple pediatric hematologic malignancies, including AML, T/B-ALL, and T/B-MPAL. Notably, ARMH1 expression is significantly elevated in blast cells of patients who relapsed or have a high-risk cytogenetic profile (MLL) compared to standard-risk (RUNX1, inv (16)). ARMH1 expression is also significantly correlated with the pediatric leukemia stem cell score of 6 genes (LSC6) associated with poor outcomes. Perturbation of ARMH1 (knockdown and overexpression) in leukemia cell lines significantly impacted cell proliferation and migration. The RNA-sequencing analysis on multiple ARMH1 knockdown and overexpressing cell lines established an association with mitochondrial fatty acid synthesis and cell cycle pathways.The investigation of the mitochondrial matrix shows that pharmacological inhibition of a key enzyme in fatty acid synthesis regulation, CPT1A, resulted in ARMH1 downregulation. ARMH1 knockdown also led to a significant reduction in CPT1A and ATP production as well as Oxygen Consumption Rate. Our data indicates that downregulating ARMH1 impacts cell proliferation by reducing key cell cycle regulators such as CDCA7 and EZH2. Further, we also established that ARMH1 is a key physical interactant of EZH2, associated with multiple cancers. Conclusion Our findings underscore further evaluation of ARMH1 as a potential candidate for targeted therapies and stratification of aggressive pAML to improve outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9c3fb31bae52e61839b92c250ccd940a2b13b6" target='_blank'>
              ARMH1 is a novel marker associated with poor pediatric AML outcomes that affect the fatty acid synthesis and cell cycle pathways
              </a>
            </td>
          <td>
            Mojtaba Bakhtiari, Sean C Jordan, Hope L. Mumme, Richa Sharma, Mala Shanmugam, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Background: Multiple myeloma (MM), a B-cell neoplasm characterized by the infiltration of malignant plasma cells into the bone marrow, is the second most common blood malignancy in the United States. The molecular pathogenesis of MM involves both genetic and epigenetic alterations. While 60-75% of newly diagnosed patients achieve a complete response or better with current modern combination therapies, almost all patients eventually progress largely due to the persistence or re-emergence of minimal residual disease (MRD). Monitoring for MRD has become essential for evaluating treatment efficacy and predicting long-term outcomes in MM patients. We aimed to investigate whether epigenetic changes, particularly 5- hydroxymethylcytosines (5hmC), at diagnosis are associated with MRD status following treatment. Methods: Utilizing 5hmC-Seal, a highly sensitive chemical labeling technique, we profiled genome-wide 5hmC in circulating cell-free DNA (cfDNA) and bone marrow genomic DNA (gDNA) at baseline and after 8 cycles of treatment (C8) from newly diagnosed MM patients (n=25) enrolled in a clinical trial (NCT01816971). MRD testing was performed on bone marrow samples using the clonoSEQ next generation sequencing assay (limit of detection 10-6). Results: The 5hmC modification levels were summarized across various genomic features, revealing enrichment in gene bodies and enhancer regions, consistent with the known regulatory role of 5hmC. We ranked 5hmC-modified genes at baseline by their variability, and found that the number of overlapping genes between cfDNA and gDNA was significantly higher than expected by permutation analysis. Subsequently, we identified the top 100 most variable 5hmC- modified genes in gDNA, which exhibited a higher Pearson correlation with cfDNA within individuals (mean r=0.90) compared to between individuals (mean r=0.82; p<0.05), suggesting that cfDNA has the potential to reflect gDNA in capturing some MM-related variations. Among the 11 patients with MRD+ at C8, a genome-wide scan of 5hmC in cfDNA comparing baseline and C8 identified 1,614 differentially modified genes (p<0.05) after adjusting for age and sex. [BC1] Furthermore, we identified 459 differentially modified genes comparing baseline 5hmC levels between patients who were MRD+ (n=11) and MRD- (n= 14) at C8 (p<0.05), after adjusting for age and sex. These results highlight the differential epigenetic landscape associated with MRD status. Conclusions: Genome-wide 5hmC profiling of cfDNA from blood revealed a similar epigenetic landscape to gDNA from bone marrow in patients with MM, suggesting that cfDNA may offer a less invasive alternative for assessing epigenetic modifications. Moreover, this profiling identified distinct epigenetic changes associated with the achievement of MRD- negativity. These findings provide a foundation for further investigation into the molecular alterations linked to MM disease biology that could potentially lead to MRD negativity. Future directions will focus on identifying biomarkers that could inform personalized treatment strategies.
 Citation Format: Zhou Zhang, Bei Wang, Krissana Kowitwanich, John Cursio, Daniel Appelbaum, Chuan He, Wei Zhang, Benjamin Derman, Brian Chiu, Andrzej Jakubowiak. Genome-wide 5-hydroxymethylation mapping in cell-free DNA to characterize the epigenetic differences in minimal residual disease of multiple myeloma [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr B032.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3829eae7b79b3a46c73b54d2d250e97c8b4ba07a" target='_blank'>
              Abstract B032: Genome-wide 5-hydroxymethylation mapping in cell-free DNA to characterize the epigenetic differences in minimal residual disease of multiple myeloma
              </a>
            </td>
          <td>
            Zhou Zhang, Beibei Wang, Krissana Kowitwanich, John Cursio, Daniel Appelbaum, Chuan He, Wei Zhang, B. Derman, B. Chiu, A. Jakubowiak
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Cancer is a complex and multifaceted group of diseases characterized by highly intricate mechanisms of tumorigenesis and tumor progression, which complicates diagnosis, prognosis, and treatment. In recent years, targeted therapies have gained prominence by focusing on specific mutations and molecular features unique to each tumor type, offering more effective and personalized treatment options. However, it is equally critical to explore the genetic commonalities across different types of cancer, which has led to the rise of pan-cancer studies. These approaches help identify shared therapeutic targets across various tumor types, enabling the development of broader and potentially more widely applicable treatment strategies. This review aims to provide a comprehensive overview of key concepts related to tumors, including tumorigenesis processes, the tumor microenvironment, and the role of extracellular vesicles in tumor biology. Additionally, we explore the molecular interactions and mechanisms driving tumor progression, with a particular focus on the pan-cancer perspective. To achieve this, we conducted an in silico analysis using publicly available datasets, which facilitated the identification of both common and divergent genetic and molecular patterns across different tumor types. By integrating these diverse areas, this review offers a clearer and deeper understanding of the factors influencing tumorigenesis and highlights potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a5b3ea0ad38a337f0df54ee493320c36455e356" target='_blank'>
              Exploring the Complexity of Pan-Cancer: Gene Convergences and in silico Analyses
              </a>
            </td>
          <td>
            Leandro Teodoro, Ana Claudia Carreira, M.C. Sogayar
          </td>
          <td>2024-12-01</td>
          <td>Breast Cancer : Targets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff4ae0eaa9128342a0dedd747e19622d92c7f71" target='_blank'>
              FDFT1 maintains glioblastoma stem cells through activation of the Akt pathway
              </a>
            </td>
          <td>
            Hui Mo, Jiajia Shao, Zhun Li, Peiting Zeng, Xinke Yin, Yongsheng Huang, Peng Wang, Jianwei Liao
          </td>
          <td>2024-12-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce415c1d6cd65bcfca677d71376e494061ba037" target='_blank'>
              Genome-wide profiling of DNA repair proteins in single cells
              </a>
            </td>
          <td>
            Kim L. de Luca, Pim M. J. Rullens, Magdalena A. Karpinska, Sandra S. de Vries, Agnieszka Gacek-Matthews, L. Pongor, G. Legube, Joanna W. Jachowicz, A. M. Oudelaar, Jop Kind
          </td>
          <td>2024-11-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Mutations in the Nucleophosmin-1 (NPM1) gene are among the most common molecular aberrations in acute myeloid leukemia (AML). Various studies have established mutant NPM1 (mNPM1) as a faithful molecular measurable residual disease (MRD) marker with prognostic significance. Assessment of prognostic mNPM1 is included in the European LeukemiaNet (ELN) recommendations on MRD detection in AML. Due to recent advancements of promising drugs targeting mNPM1 AML, monitoring of mNPM1 MRD has gained interest, and is generally done by reverse transcriptase quantitative PCR (RT-qPCR). However, these RT-qPCR assays use cDNA as input, are based on gene expression levels of mNPM1, and are generally limited to specific mNPM1 gene variants. The main advantages of next-generation sequencing (NGS) using genomic DNA as input are stability, independence of gene expression levels, and the ability to detect any NPM1 variant in a single assay. Here we comprehensively investigated the applicability of NGS on DNA to detect mNPM1 MRD in a cohort of 119 (cDNA) and 310 (DNA) mNPM1 AML patients in complete remission after two cycles of induction chemotherapy. We demonstrate high correlations in levels and prognostic value between RT-qPCR/cDNA and NGS/DNA approaches, postulating NGS/DNA as an attractive alternative to RT-qPCR. We report that the 2% mNPM1/ABL1 threshold by RT-qPCR/cDNA corresponds to a NGS/DNA variant allele frequency (VAF) of 0.01%. The NGS/DNA threshold of >0.01% after two cycles of induction chemotherapy identifies significantly more AML patients with an increased relapse risk than current RT-qPCR/cDNA assays. The prognostic significance of mNPM1 MRD appears greatest in FLT3-ITD AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29543a49300e06642677b500be09c4ca9fc87fd1" target='_blank'>
              Advantages of a genomic DNA-based NGS assay for detection of mutant NPM1 measurable residual disease in AML.
              </a>
            </td>
          <td>
            C. Vonk, T. Grob, M. Rijken, F. Kavelaars, J. Konijnenburg, Gert J. Ossenkoppele, M. Manz, Yngvar Floisand, Bob Löwenberg, Peter J M Valk
          </td>
          <td>2024-12-05</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Cancer stem cells are characterized by self-renewal, clonal tumor initiation capacity, and treatment resistance, which play essential roles in the tumor progression of prostate cancer (PCa). In this study, we aim to explore the features of cancer stemness and characterize the expression of stem cell-related genes for PCa. Methods We downloaded RNA-seq data and related clinical information from The Cancer Genome Atlas (TCGA) database. The mRNA stemness index (mRNAsi) was analyzed for various clinical features, overall survival (OS), and disease-free survival (DFS), and a weighted gene co-expression network analysis (WGCNA) was performed to identify crucial gene modules and key genes, which may play a role in CSCs. The key gene functions were verified using multiple databases, including the TCGA and Gene Expression Omnibus database (GEO). Next, we explored the potential function of the modules and genes obtained using WGCNA using an enrichment analysis. Finally, we performed in vitro experiments for further verification. Results We found that mRNAsi were higher in PCa tissues than in normal tissues, and the mRNAsi were closely related to the clinical characteristics of PCa. A total of 16 key genes associated with the mRNAsi scores were identified by WGCNA analysis, including NCAPG, NEK2, DLGAP5, CENPA, CENPF, TPX2, GTSE1, KIF4A, NEIL3, CDC25C, UBE2C, CDCA5, MELK, SKA3, NUF2, and BIRC5. These genes were explicitly highly expressed in PCa across TCGA cancers and were validated in 3 independent GEO PCa datasets. The functional annotations of the key genes were linked with the cell proliferation processes. NUF2 may be a potential biomarker for PCa. In vitro experiments showed that knockdown NUF2 reduced the proliferation and migration of PCa cells. Conclusion The 16 key genes identified in this study significantly correlate with PCa stem cell characteristics and showed prognosis-oriented effects in PCa patients. Further, the NUF2 gene may be used as a drug target for treating PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc9466fa18820a99adb15c098ec91082dee0a3" target='_blank'>
              NUF2 is associated with cancer stem cell characteristics and a potential drug target for prostate cancer
              </a>
            </td>
          <td>
            Dongxu Zhang, Pu Liang, Qi Wang, Bowen Xia, Liqian Yu, Xiaopeng Hu
          </td>
          <td>2024-12-05</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genome and transcriptome engineering have emerged as powerful tools in modern biotechnology, driving advancements in precision medicine and novel therapeutics. In this review, we provide a comprehensive overview of the current methodologies, applications, and future directions in genome and transcriptome engineering. Through this, we aim to provide a guide for tool selection, critically analyzing the strengths, weaknesses, and best use cases of these tools to provide context on their suitability for various applications. We explore standard and recent developments in genome engineering, such as base editors and prime editing, and provide insight into tool selection for change of function (knockout, deletion, insertion, substitution) and change of expression (repression, activation) contexts. Advancements in transcriptome engineering are also explored, focusing on established technologies like antisense oligonucleotides (ASOs) and RNA interference (RNAi), as well as recent developments such as CRISPR-Cas13 and adenosine deaminases acting on RNA (ADAR). This review offers a comparison of different approaches to achieve similar biological goals, and consideration of high-throughput applications that enable the probing of a variety of targets. This review elucidates the transformative impact of genome and transcriptome engineering on biological research and clinical applications that will pave the way for future innovations in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186bfc2c36c7ab505b6c05a328b8e959555d7e19" target='_blank'>
              A Manual for Genome and Transcriptome Engineering.
              </a>
            </td>
          <td>
            Yesh Doctor, Milan Sanghvi, Prashant Mali
          </td>
          <td>2024-11-08</td>
          <td>IEEE reviews in biomedical engineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A significant number of castration-resistant prostate cancer (CRPC) evolve into a neuroendocrine (NE) subtype termed NEPC, leading to resistance to androgen receptor (AR) pathway inhibitors and poor clinical outcomes. Through Hi-C analyses of a panel of patient-derived xenograft tumors, here we report drastically different 3D chromatin architectures between NEPC and CRPC samples. Such chromatin re-organization was faithfully recapitulated in vitro on isogenic cells undergoing NE transformation (NET). Mechanistically, neural transcription factor (TF) NKX2-1 is selectively and highly expressed in NEPC tumors and is indispensable for NET across various models. NKX2-1 preferentially binds to gene promoters, but it interacts with chromatin-pioneering factors such as FOXA2 at enhancer elements through chromatin looping, further strengthening FOXA2 binding at NE enhancers. Conversely, FOXA2 mediates regional DNA demethylation, attributing to NE enhancer priming and inducing NKX2-1 expression, forming a feed-forward loop. Single-cell multiome analyses of isogenic cells over time-course NET cells identify individual cells amid luminal-to-NE transformation, exhibiting intermediate epigenetic and transcriptome states. Lastly, NKX2-1/FOXA2 interacts with, and recruits CBP/p300 proteins to activate NE enhancers, and pharmacological inhibitors of CBP/p300 effectively blunted NE gene expression and abolished NEPC tumor growth. Thus, our study reports a hierarchical network of TFs governed by NKX2-1 in regulating the 2D and 3D chromatin re-organization during NET and uncovers a promising therapeutic approach to eradicate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f60ea05e69028e7c615b1e7620f9deafcd0cdd" target='_blank'>
              Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, Juan Wang, Q. Jin, Peng Jin, Feng Yue, Martin G. Sanda, Victor Corces, Nicolas Altemose, Jonathan C. Zhao, Jindan Yu
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Divergently-paired genes (DPGs) represent one of the minimal co-transcriptional units (the rest include tandemly- and convergently-paired genes) of clustered genes; the former and the latter constitute greater than 10% and 75% of the total human genes, respectively. Our previous studies have shown that vertebrate DPGs are more conserved, both organizationally and functionally than invertebrates. Three critical questions remain to be addressed: (1) what are the conserved DPGs over vertebrate lineages, especially among mammals and primates? (2) being bidirectionally transcribed, to what extent do DPGs share their promoter sequences and how mechanistically and stringently are their co-expression regulated within the shared inter-TSS (transcription start site) sequence space? and (3) based on the recently released high-quality human genome assemblies, how do human-associated DPGs distribute over selected primate lineages and what are their possible functional consequences biologically? Our study begins by identifying 1399 human DPGs (12% of all human protein-coding genes), and presents findings from this analysis. First, 1136, 1118, 925, and 830 human DPGs are shared genetically with primates, mammals, avians, and fish, respectively. DPGs are not only functionally enriched toward direct protein-DNA interactions and cell cycle synchronization but also exhibit obvious lineage association, narrow in principle toward synchronization of certain core molecular mechanisms and cellular processes. Second, their inter-TSS distances and expression variables affect both co-expression strength and disparity between the two genes. Finally, our results based on a comparison among the primate DPGs reveal that the human-associated DPGs exhibit intensive diversification in co-expression, duplication, and definite involvement in neural development. Within humans, 55 and 357 DPGs are associated to the Chinese (YAO) and the European (CHM13) assemblies, respectively. Our results offer novel insights into comprehending the structure-function selection of gene clusters over evolutionary time scales, as well as a deeper understanding of the regulatory characteristics of co-expressed neighboring genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/556aa7ddf0138c889d129f4811923a16a04d7859" target='_blank'>
              Lineage-associated Human Divergently-Paired Genes (DPGs) Exhibit Regulatory Characteristics and Evolutionary Trends
              </a>
            </td>
          <td>
            Guangya Duan, Sisi Zhang, Bixia Tang, Jingfa Xiao, Zhang Zhang, Peng Cui, Jun Yu, Wenming Zhao
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer arises through an evolutionary process in which somatic mutations, including single nucleotide variants (SNVs) and copy number aberrations (CNAs), drive the development of a malignant, heterogeneous tumor. Reconstructing this evolutionary history from sequencing data is critical for understanding the order in which mutations are acquired and the dynamic interplay between different types of alterations. Advances in modern whole genome single-cell sequencing now enable the accurate inference of copy number profiles in individual cells. However, the low sequencing coverage of these low pass sequencing technologies poses a challenge for reliably inferring the presence or absence of SNVs within tumor cells, limiting the ability to simultaneously study the evolutionary relationships between SNVs and CNAs. In this work, we introduce a novel tumor phylogeny inference method, Pharming, that jointly infers the evolutionary histories of SNVs and CNAs. Our key insight is to leverage the high accuracy of copy number inference methods and the fact that SNVs co-occur in regions with CNAs in order to enable more precise tumor phylogeny reconstruction for both alteration types. We demonstrate via simulations that Pharming outperforms state-of-the-art single-modality tumor phylogeny inference methods. Additionally, we apply Pharming to a triple-negative breast cancer case, achieving high-resolution, joint reconstruction of CNA and SNV evolution, including the de novo detection of a clonal whole-genome duplication event. Thus, Pharming offers the potential for more comprehensive and detailed tumor phylogeny inference for high-throughput, low-coverage single-cell DNA sequencing technologies compared to existing approaches. Availability https://github.com/elkebir-group/Pharming">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d64f3b951403db46ae5a215e58c613919dd9b93c" target='_blank'>
              Pharming: Joint Clonal Tree Reconstruction of SNV and CNA Evolution from Single-cell DNA Sequencing of Tumors
              </a>
            </td>
          <td>
            Leah L. Weber, Anna Hart, Idoia Ochoa, M. El-Kebir
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The accumulation of germline mutations underpins population diversity and drives genetic evolution. Despite the availability of extensive phylogenetic data, the lack of suitable methodologies has hindered the comprehensive characterization of germline mutational processes across evolutionary trees. To address this, we develop a robust three-step methodology that extracts germline mutational processes from alignments of closely related species. First, we estimate regional, branch-specific trinucleotide mutational spectra from a multispecies alignment. Second, we extract mutational processes jointly across an evolutionary clade by analyzing mutation rate variation along the genome using Reciprocal Principal Components Analysis (RPCA). Finally we filter artifactual mutational signatures using DNA symmetry. Applying this method to five primate clades and a rodent outgroup clade revealed nine distinct mutational processes. Notably, five of these processes were consistently observed across all six groups. We identified underling biological mechanism for at least seven of the processes, highlighting phenomena such as biased gene conversion, bulky lesion resolution, and maternal mutagenesis. We validated identified processes using human and non-human polymorphism data. This study offers new insights into the biology and evolution of mutagenesis in primates and introduces a methodological toolkit to investigate mutational processes across phylogenies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08874cb963d779e2ed8115054d6ecf603a380a6a" target='_blank'>
              A Phylogeny-Based Approach to Discover Mutational Processes in Primates
              </a>
            </td>
          <td>
            James C. Kitch, Vladimir Seplyarskiy
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a4f47822586a1fd8d669e38036044aa6da804e" target='_blank'>
              Chromothripsis in cancer.
              </a>
            </td>
          <td>
            Milena Simovic-Lorenz, Aurélie Ernst
          </td>
          <td>2024-11-15</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="An important direction in cancer prevention is the development of novel strategies that can be used in the treatment by affecting the mechanisms of the regulatory pathways and genomic elements involved in malignant transformation. Strategy proposed in this study is to initiate cell death mechanisms in response to the depletion of the energy resources within a cancer cell due to the uncontrolled spread of mobile genetic elements throughout its genome. The calculation results obtained using a preliminary mathematical model that simulates cellular bioenergetic balance, taking into account the energy consumption for retrotransposition of the mobile elements such as LINE-1 and SINE, show that this scenario is plausible. Intracellular resources undergo a critical redistribution when affecting the genomic defense mechanisms and the LINE-1 transcription rate. This leads to a sharp increase in energy consumption for retrotransposon transcription that causes a significant decrease in the pool of free ATPs in the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05095e01db5773f04351486017740af52565a16e" target='_blank'>
              Control of the Activity of Mobile Elements in Cancer Cells as a Strategy for Anticancer Therapy
              </a>
            </td>
          <td>
            S. R. Pavlov, V. V. Gursky, M. Samsonova, A. Kanapin, A. A. Samsonova
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6b74de7021622cacaa229773949fb0310d8a9a" target='_blank'>
              A practice-oriented genome-profiling study for acute myeloid leukemia using the novel HANDLE system: HM-screen-JAPAN02.
              </a>
            </td>
          <td>
            H. Arai, N. Hosono, Sung-Gi Chi, Kentaro Fukushima, Daisuke Ikeda, Satoshi Iyama, Akihiko Gotoh, Takayuki Ikezoe, Chikashi Yoshida, Goichi Yoshimoto, Junya Kanda, Naoto Takahashi, E. Sakaida, Kensuke Usuki, Takahiro Yamauchi, Yosuke Minami
          </td>
          <td>2024-12-16</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f233ae172c7106f684ef62640304f7d51d73186" target='_blank'>
              Integrative mapping of human CD8+ T cells in inflammation and cancer.
              </a>
            </td>
          <td>
            Ziwei Xue, Lize Wu, Ruonan Tian, Bing Gao, Yu Zhao, Bing He, Di Sun, Bingkang Zhao, Yicheng Li, Kaixiang Zhu, Lie Wang, Jianhua Yao, Wanlu Liu, Linrong Lu
          </td>
          <td>2024-11-29</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The resistance of tumors to immune checkpoint inhibitors (ICI) may be intricately linked to cellular senescence, although definitive clinical validation remains elusive. In this study, comprehensive pan-cancer scRNA-seq analyses identify fibroblasts as exhibiting the most pronounced levels of cellular senescence among tumor-associated cell populations. To elucidate this phenomenon, a fibroblast senescence-associated transcriptomic signature (FSS), which correlated strongly with protumorigenic signaling pathways and immune dysregulation that fosters tumor progression, is developed. Leveraging the FSS, the machine learning (ML) framework demonstrates exceptional accuracy in predicting ICI response and survival outcomes, achieving superior area under curve (AUC) values across validation, testing, and in-house cohorts. Strikingly, FSS consistently outperforms established signatures in predictive robustness across diverse cancer subtypes. From an integrative analysis of 17 CRISPR/Cas9 libraries, CDC6 emerges as a pivotal biomarker for pan-cancer ICI response and prognostic stratification. Mechanistically, experimental evidence reveals that CDC6 in tumor cells orchestrates fibroblast senescence via TGF-β1 secretion and oxidative stress, subsequently reprogramming the tumor microenvironment and modulating ICI response. These findings underscore the translational potential of targeting fibroblast senescence as a novel therapeutic strategy to mitigate immune resistance and enhance antitumor efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83eb29659bd661ae208e4642398f8b3b7069e404" target='_blank'>
              A Distinguished Roadmap of Fibroblast Senescence in Predicting Immunotherapy Response and Prognosis Across Human Cancers.
              </a>
            </td>
          <td>
            Dongjie Chen, Pengyi Liu, Jiayu Lin, Longjun Zang, Yihao Liu, Shuyu Zhai, Xiongxiong Lu, Yuanchi Weng, Hongzhe Li
          </td>
          <td>2024-12-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA topoisomerases are essential molecular machines that manage DNA topology in the cell and play important roles in DNA replication and transcription. We find that knocking down the enzyme Topoisomerase Top2 or its mammalian homolog Top2b increases the life span of S. cerevisiae, C. elegans, and mice. Top2b reduction also extends the health span of mice and alleviates the pathologies of aging in multiple tissues. At the cellular/molecular level, Top2b reduction attenuates the major hallmarks of aging, such as cellular senescence, de-regulated nutrient sensing, epigenetic alteration, and lysosomal biogenesis. We observed that Top2b reduction significantly changes the epigenetic landscape in various mouse tissues toward those of the young animals, and differentially down-regulates genes with active promoter and high expression. Our observations suggest that Top2 reduction confers longevity effect across species via a conserved mechanism, and may be used as a novel therapeutic strategy for countering aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2158b0d6967f642c67566187862ea04490680967" target='_blank'>
              Reduction of DNA Topoisomerase Top2 reprograms the epigenetic landscape and extends health and life span across species
              </a>
            </td>
          <td>
            Man Zhu, Meng Ma, Lunan Luo, Feiyang Li, Jiashun Zhang, Yan Pan, Lu Yang, Ying Xiao, Ziteng Wang, Bo Xian, Yi Zheng, Hao Li, Jing Yang
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Acute lymphocytic leukemia (ALL), characterized by uncontrolled growth of abnormal lymphocytes, predominantly affects children. Genetic analysis focusing on genes and microRNAs reveals important information about the biology of ALL, enabling accurate diagnosis and treatment. This study examines gene and microRNA expression in B cell ALL to improve early diagnosis and personalized treatment.


METHODS
Bone marrow samples were collected from patients both before and after the induction phase of chemotherapy. Comprehensive diagnostic techniques including flow cytometry, molecular assays, real-time PCR for common translocations, karyotyping, and complete blood count (CBC) analysis were employed. These methods were utilized to determine the type and risk assessment of ALL, identify specific gene and microRNA expressions, and measure blood cell counts.


RESULTS
The study comprised 12 patients, all under the age of 18. Post-treatment RT-PCR analysis revealed significant reductions in the expression of the ABCB1 gene, miR-129-5p, and miR-9-5p following the induction phase of chemotherapy. Karyotype analysis indicated that two patients were hypodiploid; unfortunately, both of these patients did not survive.


CONCLUSION
MicroRNAs and ABC genes serve as predictive and prognostic biomarkers in Acute Lymphoblastic Leukemia (ALL) and should be carefully reconsidered. It is more accurate to state that while microRNAs and ABC genes may potentially influence treatment response in ALL, further research is crucial to fully understand their roles in determining treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d6c6bf058c1b51d999c0b567643ee6cccfac4f" target='_blank'>
              ATP-Binding Cassette Transporter Genes and microRNAs in Pediatric B-Cell ALL: Expression Insights.
              </a>
            </td>
          <td>
            Reza Nekouian, Pooya Faranoush, Fatemeh Khesali, Parisa Shams, M. Foroughi‐Gilvaee, Negin Sadighnia, Dorsa Fallah Azad, MohammadAli Ehsani, Mohammad Faranoush
          </td>
          <td>2024-12-23</td>
          <td>Journal of clinical laboratory analysis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc6bfc5e3065b396e263e4793dde2e7908cde4e" target='_blank'>
              CRISPR-StAR enables high-resolution genetic screening in complex in vivo models.
              </a>
            </td>
          <td>
            E. Uijttewaal, Joonsun Lee, Annika Charlotte Sell, Naomi Botay, Gintautas Vainorius, Maria Novatchkova, J. Baar, Jiaye Yang, Tobias Potzler, Sophie van der Leij, Christopher Lowden, Julia Sinner, Anais Elewaut, Milanka Gavrilovic, Anna Obenauf, Daniel Schramek, Ulrich Elling
          </td>
          <td>2024-12-16</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e4edf1ee1ccd4a382a2ceac11847327f800ad1" target='_blank'>
              Silencing of LINC01278 promotes sensitivity of non-small cell lung cancer cells to osimertinib by targeting miR-324-3p/ZFX axis.
              </a>
            </td>
          <td>
            Quan Lei, Ping Liu, Xinlei Guan, Li Liu, Wenjuan He
          </td>
          <td>2024-12-19</td>
          <td>Cytotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Over 100 modifications to RNA are known to occur in human cells, where they play critical roles in many aspects of normal cellular physiology, such as cell fate decisions and terminal differentiation, through effects on RNA biology such as protein and nucleic acid interactions. Our survey of human RNA-modifying enzymes suggests many are cancer essential enzymes with striking synthetic lethal profiles, including the DHX9 helicase. DHX9 is a multifunctional DExH-box RNA helicase which can unwind regions of double-stranded DNA and RNA helices but has a greater propensity for secondary structures such as DNA/RNA hybrids (R-loops) and DNA/RNA G-quadruplexes. DHX9 interacts with and regulates many proteins, including key members of DNA damage repair pathways. We have found that DHX9 knockdown is synthetic lethal in microsatellite high (MSI-H) or defective mismatch repair (dMMR) tumor models. A suite of assays was developed to identify and optimize potent and selective inhibitors of the DHX9 helicase, which recapitulate our findings with genetic tools. Profiling of DHX9 inhibitors across a broad panel of cancer cell lines reveals that tumor cells with mutations in the DNA damage repair genes BRCA1 and/or BRCA2 are also responsive to DHX9 inhibitor treatment in vitro and in vivo. DHX9 inhibition leads to increased RNA/DNA secondary structures such as R-loops and G-quadruplexes, resulting in subsequent DNA damage and increased replication stress, leading to cell cycle arrest and apoptosis. These results suggest that DHX9 inhibitors are a novel treatment modality for patients with defective DNA damage repair pathways such as dMMR and/or BRCA mutations.
 Citation Format: Serena J Silver. Inhibitors of DHX9 RNA helicase as synthetic lethal cancer therapeutics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr I003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9294cfb055883f4bc1decf8c1932ef149f57a0f" target='_blank'>
              Abstract I003: Inhibitors of DHX9 RNA helicase as synthetic lethal cancer therapeutics
              </a>
            </td>
          <td>
            Serena J Silver
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a02ec089cfd84f1709cb22a73645834678bb0f8d" target='_blank'>
              Quantifying the regulatory potential of genetic variants via a hybrid sequence-oriented model with SVEN
              </a>
            </td>
          <td>
            Yu Wang, Nan Liang, Ge Gao
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d182842a9c5b9ec9f888e5c17ac019a50f732fb3" target='_blank'>
              Beginning at the ends: telomere and telomere-based cancer therapeutics.
              </a>
            </td>
          <td>
            Zahra Sadr, Masoumeh Ghasemi, Soheyla Jafarpour, Reyhaneh Seyfi, Aida Ghasemi, Elham Boustanipour, H. Khorshid, Naeim Ehtesham
          </td>
          <td>2024-12-06</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b7ecb008360a3e79fcfe183be595899effbacee" target='_blank'>
              N6-methyladenosine RNA modification in head and neck squamous cell carcinoma (HNSCC): current status and future insights.
              </a>
            </td>
          <td>
            Pramodha Janakiraman, Jayasree Peroth Jayaprakash, Sridhanya Velayudham Muralidharan, Kumar Pranav Narayan, P. Khandelia
          </td>
          <td>2024-11-24</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Biological mechanisms are inherently dynamic, requiring precise and rapid gene manipulation for effective characterization. Traditional genetic perturbation tools such as siRNA and CRISPR knockout operate on timescales that render them unsuitable for exploring dynamic processes or studying essential genes, where chronic depletion can lead to cell death. Here, we compared four major inducible degron systems—dTAG, HaloPROTAC, and two auxin-inducible degron (AID) tools—in human pluripotent stem cells. We evaluated basal degradation levels, inducible degradation kinetics, and recovery dynamics for endogenously tagged genes. While the AID 2.0 system is the most efficient for rapid protein degradation, it exhibited higher basal degradation and slower recovery after ligand washout. To address these challenges, we applied directed protein evolution, incorporating base-editing-mediated mutagenesis and iterative functional selection and screening. We discovered novel OsTIR1 variants, including S210A, with significantly enhanced overall degron efficiency. The resulting system, designated as AID 3.0, demonstrates minimal basal degradation and rapid and effective target protein depletion and substantially rescues the cellular and molecular phenotypes due to basal degradation or slow target protein recovery in previous systems. We conclude that AID 3.0 represents a superior degron technology, offering a valuable tool for studying gene functions in dynamic biological contexts and exploring therapeutic applications. Additionally, the research strategy used here could be broadly applicable for improving other degron and biological tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a538e1075274bba3f3af7ca77c8e65e5a189520" target='_blank'>
              Comparative analysis and directed protein evolution yield an improved degron technology with minimal basal degradation, rapid inducible depletion, and faster recovery of target proteins
              </a>
            </td>
          <td>
            Mazhar Adli, De Xing, Tao Bai, Ozlem Neyisci, Seyedehzahra Paylakhi, Alexander Duval, Yasemin Tekin
          </td>
          <td>2024-11-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Discovered more than four decades ago, nonsense-mediated mRNA decay (NMD) plays a fundamental role in the regulation of gene expression and is a major contributor to numerous diseases. With advanced technologies, several novel approaches aim to directly circumvent the effects of disease-causing frameshift and nonsense mutations. Additional therapeutics aim to globally dampen the NMD pathway in diseases associated with pathway hyperactivation, one example being Fragile X Syndrome. In other cases, therapeutics have been designed to hijack or inhibit the cellular NMD machinery to either activate or obviate transcript-specific NMD by modulating pre-mRNA splicing. Here, we discuss promising approaches employed to regulate NMD for therapeutic purposes and highlight potential challenges in future clinical development. We are optimistic that the future of developing target-specific and global modulators of NMD (inhibitors as well as activators) is bright and will revolutionize the treatment of many genetic disorders, especially those with high unmet medical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8d79dce5cfe95746b7b429088778df04c8485b" target='_blank'>
              Exploring the therapeutic potential of modulating nonsense-mediated mRNA decay.
              </a>
            </td>
          <td>
            Mary McMahon, L. Maquat
          </td>
          <td>2024-12-12</td>
          <td>RNA</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaa35d32ecdf58360fe2158aa64cfa3b17ec1726" target='_blank'>
              Evolving cell states and oncogenic drivers during the progression of IDH-mutant gliomas.
              </a>
            </td>
          <td>
            Jingyi Wu, L. G. Gonzalez Castro, S. Battaglia, C. E. El Farran, J. D’Antonio, Tyler E. Miller, M. Suvà, B. Bernstein
          </td>
          <td>2024-11-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="A significant portion of human cancers utilize a recombination-based pathway, Alternative Lengthening of Telomeres (ALT), to extend telomeres. To gain further insights into this pathway, we developed a high-throughput imaging-based screen named TAILS (Telomeric ALT In situ Localization Screen), to identify genes that either promote or inhibit ALT activity. Screening over 1000 genes implicated in DNA transactions, TAILS revealed both well-established and novel ALT modulators. We have identified new factors that promote ALT, such as the nucleosome-remodeling factor CHD4 and the chromatin reader SGF29, as well as factors that suppress ALT, including the RNA helicases DDX39A/B, the replication factor TIMELESS, and components of the chromatin assembly factor CAF1. Our data indicate that defects in histone deposition significantly contribute to ALT-associated phenotypes. Based on these findings, we demonstrate that pharmacological treatments can be employed to either exacerbate or suppress ALT-associated phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b124646ba82a460ac6aa3c7cf35a789c6739919" target='_blank'>
              Identification of Novel Modulators of the ALT Pathway Through a Native FISH-Based Optical Screen
              </a>
            </td>
          <td>
            Benura Azeroglu, Simran Khurana, Shih-Chun Wang, Gianna M Tricola, Shalu Sharma, Camille Jubelin, Ylenia Cortolezzis, Gianluca Pegoraro, Kyle M. Miller, Travis H. Stracker, Eros Lazzerini Denchi
          </td>
          <td>2024-11-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The determination of the genetic subtypes of primary CNS lymphoma (PCNSL) and their relationship to differential chemo-immunotherapeutic response has not been established. There is a particular need for genomic biomarkers that identify patients with newly-diagnosed PCNSL at high risk of early progression and death. We applied targeted next-generation sequencing for detection of recurrent single nucleotide variants, copy number alterations and zygosity abnormalities in diagnostic specimens from 78 PCNSL patients treated with a standard methotrexate-based regimen, to identify prognostically significant molecular subgroups. All patients received induction immunochemotherapy and 44 proceeded to dose-intensive consolidation. Genomic aberrations at four loci were associated with 91% of lymphoma progression events and all 15 deaths: (1) chromosome 6p copy-neutral loss of heterozygosity (CN-LOH) or focal homozygous deletion (HD) at 6p21.3, and mutations of tumor suppressor genes (2) BTG1, (3) ETV6 and (4) TP53. Cox regression multivariate analysis demonstrated a high risk of progression in patients with aberrations at these loci. Genomic aberrations at these loci were also associated with significantly shorter survival. Lower expression of HLA-DR was associated with 6p CN-LOH/6p21.3 HD and inferior prognosis. These genomic aberrations identify a high-risk molecular subgroup that may inform risk stratification in PCNSL. Further elucidation of the mechanisms of therapeutic resistance associated with the high-risk genetic phenotype is requisite to facilitate precision medicine and progress in therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d443853490c421bb34b808e8720db40347672b" target='_blank'>
              Identification of Genomic Biomarkers of Disease Progression and Survival in Primary CNS Lymphoma.
              </a>
            </td>
          <td>
            H. Geng, Shirley Shaoyi Mo, Lingjing Chen, A. Ballapuram, Mazie Tsang, Ming Lu, A. Rauschecker, Kwun Wah Wen, W. P. Devine, D. A. Solomon, J. Rubenstein
          </td>
          <td>2024-11-13</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Glioblastoma (GBM) is uniformly lethal due to profound treatment resistance. Altered cellular metabolism is a key mediator of GBM treatment resistance. Uptake of the essential sulfur-containing amino acid methionine is drastically elevated in GBMs compared to normal cells, however, it is not known how this methionine is utilized or whether it relates to GBM treatment resistance. Here, we find that radiation acutely increases the levels of methionine-related metabolites in a variety of treatment-resistant GBM models. Stable isotope tracing studies further revealed that radiation acutely activates methionine to S-adenosyl methionine (SAM) conversion through an active signaling event mediated by the kinases of the DNA damage response. In vivo tumor SAM synthesis increases after radiation, while normal brain SAM production remains unchanged, indicating a tumor- specific metabolic alteration to radiation. Pharmacological and dietary strategies to block methionine to SAM conversion slowed DNA damage response and increased cell death following radiation in vitro. Mechanistically, these effects are due to depletion of DNA repair proteins and are reversed by SAM supplementation. These effects are selective to GBMs lacking the methionine salvage enzyme methylthioadenosine phosphorylase. Pharmacological inhibition of SAM synthesis hindered tumor growth in flank and orthotopic in vivo GBM models when combined with radiation. By contrast, methionine depletion does not reduce tumor SAM levels and fails to radiosensitize intracranial models, indicating depleting SAM, as opposed to simply lowering methionine, is critical for hindering tumor growth in intracranial models of GBM. These results highlight a new signaling link between DNA damage and SAM synthesis and define the metabolic fates of methionine in GBM in vivo. Inhibiting radiation-induced SAM synthesis slows DNA repair and augments radiation efficacy in GBM. Using MAT2A inhibitors to deplete SAM may selectively overcome treatment resistance in GBMs with defective methionine salvage while sparing normal brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11aa91d406c5f4b75a9a6b9dbcbaac218f6d1df0" target='_blank'>
              Reciprocal links between methionine metabolism, DNA repair and therapy resistance in glioblastoma
              </a>
            </td>
          <td>
            Navyateja Korimerla, Baharan Meghdadi, Isra Haq, Kari Wilder-Romans, Jie Xu, Nicole Becker, Ziqing Zhu, Peter Kalev, Nathan Qi, Charles Evans, Maureen Kachman, Zitong Zhao, Angelica Lin, A. Scott, Alexandra M O'Brien, Ayesha U. Kothari, Peter Sajjakulnukit, Li Zhang, Sravya Palavalasa, Erik R Peterson, Marc L Hyer, Katya Marjon, Taryn Sleger, Meredith A. Morgan, C. Lyssiotis, Everett M. Stone, S. Ferris, Theodore S. Lawrence, Deepak Nagrath, Weihua Zhou, Daniel R Wahl
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The explosion of next-generation sequencing technologies has allowed researchers to move from studying single genes, to thousands of genes, and thereby to also consider the relationships within gene networks. Like others, we are interested in understanding how developmental and evolutionary forces shape the expression of individual genes, as well as the interactions among genes. To this end, we characterized the effects of genetic background and developmental environment on brain gene coexpression in two parallel, independent evolutionary lineages of Trinidadian guppies (Poecilia reticulata). We asked whether connectivity patterns among genes differed based on genetic background and rearing environment, and whether a gene’s connectivity predicted its propensity for expression divergence. In pursuing these questions, we confronted the central challenge that standard approaches fail to control the Type I error and/or have low power in the presence of high dimensionality (i.e., large number of genes) and small sample size, as in many gene expression studies. Using our data as a case study, we detail central challenges, discuss sample size guidelines, and provide rigorous statistical approaches for exploring coexpression differences with small sample sizes. Using these approaches, we find evidence that coexpression relationships differ based on both genetic background and rearing environment. We report greater expression divergence in less connected genes and suggest this pattern may arise and be reinforced by selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffb7f6929569fe684cb5a3f56de734a873153a7a" target='_blank'>
              FLEXIBILITY IN GENE COEXPRESSION AT DEVELOPMENTAL AND EVOLUTIONARY TIMESCALES
              </a>
            </td>
          <td>
            Eva K. Fischer, Youngseok Song, Wen Zhou, K. Hoke
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a heterogeneous malignant disease with high relapse and mortality rates. To characterise the multiomics features of T-ALL, we conducted integrative analyses using single-cell RNA, TCR and chromatin accessibility sequencing on pre- and post-treatment peripheral blood and bone marrow samples of the same patients. We found that there is transcriptional rewiring of gene regulatory networks in T-ALL cells. Some transcription factors, such as TCF3 and KLF3, showed differences in activity and expression levels between T-ALL and normal T cells and were associated with the prognosis of T-ALL patients. Furthermore, we identified multiple malignant TCR clonotypes among the T-ALL cells, where the clonotypes consisted of distinct combinations of the same TCR α and β chain per patient. The T-ALL cells displayed clonotype-specific immature thymocyte cellular characteristics and response to chemotherapy. Remarkably, T-ALL cells with an orphan TCRβ chain displayed the strongest stemness and resistance to chemotherapy. Our study provided transcriptome and epigenome characterisation of T-ALL cells categorised by TCR clonotypes, which may be helpful for the development of novel predictive markers to evaluate treatment effectiveness for T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35386d641d4da0518c18007554d1299175192fd" target='_blank'>
              Single-Cell Multiomics Reveals TCR Clonotype-Specific Phenotype and Stemness Heterogeneity of T-ALL Cells.
              </a>
            </td>
          <td>
            Songnan Sui, Xiaolei Wei, Yue Zhu, Qiuyue Feng, Xianfeng Zha, Lipeng Mao, Boya Huang, Wen Lei, Guobing Chen, Huien Zhan, Huan Chen, Ru Feng, Chengwu Zeng, Yangqiu Li, Oscar Junhong Luo
          </td>
          <td>2024-12-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5f657e821743bf515a8df904c0de712f1e97ea" target='_blank'>
              Understanding the biological processes of kidney carcinogenesis: an integrative multi-omics approach
              </a>
            </td>
          <td>
            Ricardo Cortez Cardoso Penha, Alexandra Sexton Oates, S. Senkin, Hanla A. Park, Joshua Atkins, I. Holcatova, A. Hornakova, S. Savic, Simona Ognjanovic, Beata Świątkowska, J. Lissowska, D. Zaridze, A. Mukeria, V. Janout, Amélie Chabrier, Vincent Cahais, C. Cuenin, G. Scelo, M. Foll, Z. Herceg, Paul Brennan, K. Smith-Byrne, N. Alcala, James D. McKay
          </td>
          <td>2024-11-26</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="Background/Objectives: AML is an aggressive malignant disease characterized by aberrant proliferation and accumulation of immature blast cells in the patient’s bone marrow. Chemotherapeutic treatment can effectively induce remission and re-establish functional hematopoiesis. However, many patients experience chemoresistance-associated relapse and disease progression with a poor prognosis. The identification of molecular determinants of chemoresistance that could serve as potential targets for the therapeutic restoration of chemosensitivity has proven to be challenging. Methods: To address this, we have analyzed longitudinal changes in the expression of microRNAs during disease progression in a small set of four AML patients, combined with gene ontology (GO) pathway analysis and evaluation of gene expression data in patient databases. Results: MicroRNA profiling of bone marrow samples at diagnosis and after relapse revealed significant differential expression of a large number of microRNAs between the two time points. Subsequent GO pathway analysis identified 11 signal transduction pathways likely to be affected by the differential miRNA signatures. Exemplary validation of the FoxO signaling pathway by gene expression analysis confirmed significant upregulation of FOXO1 and the target genes GADD45 and SOD2. Conclusions: Here, we show how a microRNA-based pathway prediction strategy can be used to identify differentially regulated signaling pathways that represent potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ad3edfd6af9770fc155f2d3924cf9e30fae927" target='_blank'>
              MicroRNA Screening Reveals Upregulation of FoxO-Signaling in Relapsed Acute Myeloid Leukemia Patients
              </a>
            </td>
          <td>
            Paula Reichelt, S. Bernhart, U. Platzbecker, Michael Cross
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0540ae8ca06fd2f7603169d85504347af8c2f004" target='_blank'>
              Copy number alterations in pediatric B-cell precursor acute lymphoblastic leukemia patients and their association with patients' outcome.
              </a>
            </td>
          <td>
            Nesma E. Abdelfattah, Ghada M Elsayed, Amira H Soliman, Emad N Ebeid, M. E. El Ashry
          </td>
          <td>2024-11-26</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract BACKGROUND AND OBJECTIVES Comprehensive Genome Profiling (CGP) has been covered by insurance since June 2019. This advancement has enabled the identification of therapeutic target gene mutations, improving access to drugs through clinical trials and patient-requested therapy programs. However, it has been noted that the number of cases that actually lead to treatment remains low. This study aims to examine the current status of CGP for brain and spinal cord tumor patients, assessing its utility and challenges. METHODS We investigated 46 cases of central nervous system tumors that underwent CGP by April 2024. The focus was on the detection of MSI-H, TMB-H, NTRK fusion genes, and BRAF gene mutations, as well as the genetic mutations associated with recommended drugs, and examined the cases that led to actual treatment. RESULTS The median age of the patients was 50 years. The breakdown included 26 cases of adult-type diffuse glioma, 5 cases of diffuse midline glioma with H3 K27M alteration, 1 case of pilomyxoid astrocytoma, 2 cases of ependymal tumors, 3 cases of atypical teratoid/rhabdoid tumor (AT/RT), 3 cases of anaplastic meningioma, and 6 other cases. MSI-H and TMB-H were found in 1 case of glioblastoma, TMB-H alone in 3 cases, NTRK fusion gene in 1 case of spinal glioma, and BRAF V600E mutation in 1 case of glioblastoma. Thirty cases (65%) received some form of recommendation, with 6 cases (13%) receiving drug treatment. Only the spinal glioma with the NTRK fusion gene showed significant improvement with entrectinib. CONCLUSION While CGP is useful for identifying genetic abnormalities, its connection to actual treatment is limited. Future efforts should focus on developing new therapies and enhancing clinical trials, as well as exploring high-precision biomarkers and detailed analyses of the relationship between treatment history, clinical information, and genetic abnormalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14aeeca234b50505d917c580f12352399721b7e7" target='_blank'>
              10128- COT-15 CURRENT STATUS AND CHALLENGES OF COMPREHENSIVE GENOME PROFILING FOR CNS TUMORS
              </a>
            </td>
          <td>
            Y. Takahashi, Takahiro Ogawa, Ichita Taniyama, Takumi Yamanaka, N. Hashimoto
          </td>
          <td>2024-11-29</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e501c572b2c39ffe6df84d823c30df850a5f3204" target='_blank'>
              PU.1 eviction at lymphocyte-specific chromatin domains mediates glucocorticoid response in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Dominik Beck, Honghui Cao, Feng Tian, Yizhou Huang, Miao Jiang, Han Zhao, Xiaolu Tai, Wenqian Xu, Hansen J. Kosasih, David Kealy, Weiye Zhao, Samuel J. Taylor, Timothy A. Couttas, Gaoxian Song, Diego Chacon-Fajardo, Yashna Walia, Meng Wang, A. Dowle, Andrew N. Holding, Katherine S. Bridge, Chao Zhang, Jin Wang, Jian-Qing Mi, Richard B. Lock, Charles E de Bock, Duohui Jing
          </td>
          <td>2024-11-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is a hematologic tumor with poor prognosis and significant clinical heterogeneity. By integrating transcriptomic data, single-cell RNA sequencing data and independently collected RNA sequencing data this study aims to identify key genes in AML and establish a prognostic assessment model to improve the accuracy of prognostic prediction. Materials and methods We analyzed RNA-seq data from AML patients and combined it with single-cell RNA sequencing data to identify genes associated with AML prognosis. Key genes were screened by bioinformatics methods, and a prognostic assessment model was established based on these genes to validate their accuracy. Results The study identified eight key genes significantly associated with AML prognosis: SPATS2L, SPINK2, AREG, CLEC11A, HGF, IRF8, ARHGAP5, and CD34. The prognostic model constructed on the basis of these genes effectively differentiated between high-risk and low-risk patients and revealed differences in immune function and metabolic pathways of AML cells. Conclusion This study provides a new approach to AML prognostic assessment and reveals the role of key genes in AML. These genes may become new biomarkers and therapeutic targets that can help improve prognostic prediction and personalized treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fba51cf3073c5201e7804e4ac1e1ae1bb4b49f8" target='_blank'>
              Single-cell transcriptomics reveals heterogeneity and prognostic markers of myeloid precursor cells in acute myeloid leukemia
              </a>
            </td>
          <td>
            Guangfeng He, Lai Jiang, Xuancheng Zhou, Yuheng Gu, Jingyi Tang, Qiang Zhang, Qingwen Hu, Gang Huang, Ziye Zhuang, Xinrui Gao, Ke Xu, Yewei Xiao
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Microtubules play essential roles in diverse cellular processes and are important pharmacological targets for treating human disease. Here, we sought to identify cellular factors that modulate the sensitivity of cells to antimicrotubule drugs. We conducted a genome-wide CRISPR/Cas9-based functional genetics screen in human cells treated with the microtubule-destabilizing drug nocodazole or the microtubule-stabilizing drug paclitaxel. We further conducted a focused secondary screen to test drug sensitivity for ∼1,400 gene targets across two distinct human cell lines and to additionally test sensitivity to the KIF11 inhibitor, STLC. These screens defined gene targets whose loss enhances or suppresses sensitivity to antimicrotubule drugs. In addition to gene targets whose loss sensitized cells to multiple compounds, we observed cases of differential sensitivity to specific compounds and differing requirements between cell lines. Our downstream molecular analysis further revealed additional roles for established microtubule-associated proteins and identified new players in microtubule function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe92d3ab4eb0f4d2bf377007f003eebe7871111" target='_blank'>
              Functional genetics reveals modulators of antimicrotubule drug sensitivity.
              </a>
            </td>
          <td>
            Kuan-Chung Su, Elena Radul, Nolan K Maier, Mary-Jane Tsang, Claire Goul, Brittania Moodie, Océane Marescal, Heather R Keys, I. Cheeseman
          </td>
          <td>2024-11-21</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Mutagenic processes drive evolutionary progress, with ultraviolet (UV) radiation significantly affecting evolution. Despite extensive research on SOS response-mediated mutagenesis, UV-induced repair mechanisms remain complex, and their effects on cell survival and mutagenesis are not fully understood. We previously observed a near-perfect correlation between RecA-mediated SOS response and mutation levels in Escherichia coli following UV treatment. However, prolonged UV exposure caused transient non-culturability and impaired SOS-mediated mutagenesis. Using fluorescent reporters, flow cytometry, promoter-reporter assays, single-gene deletions, knockouts, and clonogenic assays, we found that excessive UV exposure disrupts cellular translation, reducing SOS gene expression, albeit with minimal impact on membrane permeability or reactive oxygen species levels. While our findings underline the abundance of repair mechanisms in E. coli cells, enabling them to compensate when specific genes are disrupted, they also highlighted the differential impacts of gene deletions on mutagenesis versus culturability, leading to three major outcomes: (i) Disruption of proteins involved in DNA polymerase for trans-lesion synthesis (UmuC and UmuD) or Holliday junction resolution (RuvC) results in significantly decreased mutagenesis levels while maintaining a transient non-culturability pattern after UV exposure. (ii) Disruption of proteins involved in homologous recombination (RecA and RecB) and nucleotide excision repair (UvrA) leads to both significantly reduced mutagenesis and a more severe transient non-culturability pattern after UV exposure, making these cells more sensitive to UV. (iii) Disruption of DNA Helicase II (UvrD), which functions in mismatch repair, does not affect mutagenesis levels from UV radiation but results in a very pronounced transient non-culturability pattern following UV exposure. Overall, our results further advance our understanding of bacterial adaptation mechanisms and the role of DNA repair pathways in shaping mutagenesis. Author Summary Ultraviolet (UV) radiation has been a significant force in driving genetic variation and adaptation throughout billions of years of evolution. By directly damaging DNA and triggering repair mechanisms, UV radiation is a powerful tool for studying mutagenesis. This study aims to provide new insights into the complex processes behind bacterial mutagenesis, a critical topic in microbiology and public health. Although the connection between the SOS response and DNA repair in UV-treated cells is well known, two key phenomena — UV-induced bacterial cell dormancy and mutagenesis — remain poorly understood. Our findings reveal a highly unusual, SOS response-dependent transient non-culturability in Escherichia coli cells following prolonged UV exposure. However, the downstream mechanisms of this phenomenon and its links to mutagenesis remain unclear. This study seeks to thoroughly investigate these phenomena, offering new insights into their underlying molecular processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8574f751e70ca946f8a54c7557e261ab9d00bd43" target='_blank'>
              UV-Induced DNA Repair Mechanisms and Their Effects on Mutagenesis and Culturability in Escherichia coli
              </a>
            </td>
          <td>
            Sreyashi Ghosh, Mehmet A. Orman
          </td>
          <td>2024-11-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The Single Nucleotide Polymorphism Database (dbSNP), established in 1998 by the National Center for Biotechnology Information (NCBI), has been a critical resource in genomics for cataloging small genetic variations. Originally focused on single nucleotide polymorphisms (SNPs), dbSNP has since expanded to include a variety of genetic variants, playing a key role in genome-wide association studies (GWAS), population genetics, pharmacogenomics, and cancer research. Over 25 years, dbSNP has grown to include more than 4.4 billion submitted SNPs and 1.1 billion unique reference SNPs, providing essential data for identifying disease-related genetic variants and studying human diversity. Integrating large-scale projects like 1000 Genomes, gnomAD, TOPMed, and ALFA has expanded dbSNP's catalog of human genetic variation, increasing its usefulness for research and clinical applications. Keeping up with advancements such as next-generation sequencing and cloud-based infrastructure, dbSNP remains a cornerstone of genetic research supporting continued discoveries in precision medicine and population genomics. DATABASE URL: https://www.ncbi.nlm.nih.gov/snp.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ad2237820ea0faf5485f1b743dafd0d5f1429b0" target='_blank'>
              The evolution of dbSNP: 25 years of impact in genomic research.
              </a>
            </td>
          <td>
            Lon Phan, Hua Zhang, Qiang Wang, Ricardo Villamarin, Tim Hefferon, Aravinthan Ramanathan, Brandi Kattman
          </td>
          <td>2024-11-12</td>
          <td>Nucleic acids research</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Aging constitutes complex and dynamic alterations in molecular and physiological processes and is associated with numerous disorders, in part due to increased genetic instability. The aging population is projected to double by 2050, underscoring the urgent need to better understand the relationships between aging and age-related disorders. Repetitive DNA elements are intrinsic sources of genetic instability and have been found to co-localize with mutation hotspots in human cancer genomes. In this study, we explored the relationship between aging and DNA repeat-mediated genetic instability in vivo using an H-DNA-forming mirror-repeat sequence from the cancer-associated human c-MYC gene. Utilizing a unique mutation-reporter mouse model, we observed tissue-specific effects of aging on H-DNA-induced genetic instability, with mutation frequencies increasing in spleen tissues and remaining unchanged in testis tissues. Analysis of the mutation spectra revealed large deletion mutations as the primary contributor to increasing H-DNA-induced mutations, supported by increased cleavage activity of H-DNA structures in aged spleen tissues. Our findings demonstrate that aging has distinct tissue-specific effects on repeat-mediated, cancer-associated mutations, providing insights into the complex relationship between aging and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8066191467ac61ad7ea1a2f5a55dbdea73e14d68" target='_blank'>
              Tissue-Specific Effects of Aging on Repeat-Mediated Mutation Hotspots In Vivo
              </a>
            </td>
          <td>
            Alexandra M. D’Amico, Tonia T. Li, Karen M. Vasquez
          </td>
          <td>2024-11-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15084ff422804e2f9ab2fbc7e8a0344ff82b8dbd" target='_blank'>
              High CD44 expression and enhanced E-selectin binding identified as biomarkers of chemoresistant leukemic cells in human T-ALL.
              </a>
            </td>
          <td>
            J. Calvo, Irina Naguibneva, Anthony Kypraios, Florian Gilain, B. Uzan, Baptiste Gaillard, Léa Bellenger, Laurent Renou, Christophe Antoniewski, Hélène Lapillonne, Arnaud Petit, P. Ballerini, Stéphane J. C. Mancini, Tony Marchand, Jean-François Peyron, Françoise Pflumio
          </td>
          <td>2024-11-24</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Mutations in the TP53 tumor suppressor genes are prevalent in aggressive cancers. Pharmacological reactivation of dysfunctional p53 due to mutations is a promising strategy for treating such cancers. Recently, a multifunctional proline- and glutamine-rich protein, PTB-associated splicing factor (PSF), was identified as a key driver of aggressive cancers. PSF promotes the expression of numerous oncogenes by modulating epigenetic and splicing mechanisms. We previously screened a small-molecule library and discovered compound No.10-3 as a potent PSF inhibitor. Here, we report the discovery of a No.10-3 analog, C-30, as a potent PSF inhibitor. Compared to No.10-3, C-30 treatment specifically suppressed the growth and induced apoptosis of mutant p53-bearing and therapy-resistant cancer cells. Interestingly, C-30 activated a set of p53-regulated genes in therapy-resistant cancer cells. A comprehensive analysis of PSF and p53 binding regions demonstrated a higher level of PSF-binding potential in mutant p53-expressing cancer cells around genomic regions identified as p53-binding peaks in p53-wild type cancer cells. Treatment of mutant p53-expressing cancer cells with C-30 decreases PSF binding around these sites, leading to activated histone acetylation. We further demonstrated that C-30 impaired tumor growth and increased the expression of p53-target genes in vivo. These results suggested that C-30 produces tumor-suppressive effects similar to the functional reactivation of p53, providing a rationale for the inhibition of PSF activity as a promising therapy against treatment-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a883a5ac34ea9851ff495a97f5000312eee25b0f" target='_blank'>
              Inhibition of PSF activity overcomes resistance to treatment in cancers harboring mutant p53.
              </a>
            </td>
          <td>
            Ken‐ichi Takayama, Tomohiro Sato, Teruki Honma, Minoru Yoshida, Satoshi Inoue
          </td>
          <td>2024-12-03</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Polyploidy, the result of whole genome duplication (WGD), is widespread across the tree of life and is often associated with speciation and adaptability. It is thought that adaptation in autopolyploids (within-species polyploids) may be facilitated by increased access to genetic variation. This variation may be sourced from gene flow with sister diploids and new access to other tetraploid lineages, as well as from increased mutational targets provided by doubled DNA content. Here, we deconstruct in detail the origins of haplotypes displaying the strongest selection signals in established, successful autopolyploids, Arabidopsis lyrata and Arabidopsis arenosa. We see strong signatures of selection in 17 genes implied in meiosis, cell cycle, and transcription across all four autotetraploid lineages present in our expanded sampling of 983 sequenced genomes. Most prominent in our results is the finding that the tetraploid-characteristic haplotypes with the most robust signals of selection were completely absent in all diploid sisters. In contrast, the fine-scaled variant 'mosaics' in the tetraploids originated from highly diverse evolutionary sources. These include widespread novel reassortments of trans-specific polymorphism from diploids, new mutations, and tetraploid-specific inter-species hybridization-a pattern that is in line with the broad-scale acquisition and reshuffling of potentially adaptive variation in tetraploids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77790d15f0fe6779a277cbbec309ea4248fac17f" target='_blank'>
              Polyploids broadly generate novel haplotypes from trans-specific variation in Arabidopsis arenosa and Arabidopsis lyrata.
              </a>
            </td>
          <td>
            Magdalena Bohutínská, Eliška Petříková, Tom R. Booker, Cristina Vives Cobo, Jakub Vlček, G. Šrámková, Alžběta Poupětová, Jakub Hojka, K. Marhold, Levi Yant, Filip Kolář, R. Schmickl
          </td>
          <td>2024-12-23</td>
          <td>PLoS genetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Adaptive radiations are rich laboratories for exploring, testing, and understanding key theories in evolution and ecology because they offer spectacular displays of speciation and ecological adaptation. Particular challenges to the study of adaptive radiation include high levels of species richness, rapid speciation, and gene flow between species. Over the last decade, high-throughput sequencing technologies and access to population genomic data have lessened these challenges by enabling the analysis of samples from many individual organisms at whole-genome scales. Here we review how population genomic data have facilitated our knowledge of adaptive radiation in five key areas: (1) phylogenetics, (2) hybridization, (3) timing and rates of diversification, (4) the genomic basis of trait evolution, and (5) the role of genome structure in divergence. We review current knowledge in each area, highlight outstanding questions, and focus on methods that facilitate detection of complex patterns in the divergence and demography of populations through time. It is clear that population genomic data are revolutionising the ability to reconstruct evolutionary history in rapidly diversifying clades. Additionally, studies are increasingly emphasising the central role of gene flow, re-use of standing genetic variation during adaptation, and structural genomic elements as facilitators of the speciation process in adaptive radiations. We highlight hybridization-and the hypothesized processes by which it shapes diversification-and questions seeking to bridge the divide between microevolutionary and macroevolutionary processes as rich areas for future study. Overall, access to population genomic data has facilitated an exciting era in adaptive radiation research, with implications for deeper understanding of fundamental evolutionary processes across the tree of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6567f401893572605f0ef83ca062582a85587de" target='_blank'>
              Population Genomics of Adaptive Radiation.
              </a>
            </td>
          <td>
            Lucia L Combrink, Jimena Golcher-Benavides, Alexander L Lewanski, J. Rick, William C Rosenthal, C. Wagner
          </td>
          <td>2024-12-24</td>
          <td>Molecular ecology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Motivation Many tumours show deficiencies in DNA damage response (DDR), which influence tumorigenesis and progression, but also expose vulnerabilities with therapeutic potential. Assessing which patients might benefit from DDR-targeting therapy requires knowledge of tumour DDR deficiency status, with mutational signatures reportedly better predictors than loss of function mutations in select genes. However, signatures are identified independently using unsupervised learning, and therefore not optimised to distinguish between different pathway or gene deficiencies. Results We propose SNMF, a supervised non-negative matrix factorisation that jointly optimises the learning of signatures: (1) shared across samples, and (2) predictive of DDR deficiency. We applied SNMF to mutation profiles of human induced pluripotent cell lines carrying gene knockouts linked to three DDR pathways. The SNMF model achieved high accuracy (0.971) and learned more complete signatures of the DDR status of a sample, further discerning distinct mechanisms within a pathway. Cell line SNMF signatures recapitulated tumour-derived COSMIC signatures and predicted DDR pathway deficiency of TCGA tumours with high recall, suggesting that SNMF-like models can leverage libraries of induced DDR deficiencies to decipher intricate DDR signatures underlying patient tumours. Availability https://github.com/joanagoncalveslab/SNMF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afb1aa490661811f25b1b0555092a4330771d6f8" target='_blank'>
              SNMF: Integrated Learning of Mutational Signatures and Prediction of DNA Repair Deficiencies
              </a>
            </td>
          <td>
            Sander Goossens, Y. Tepeli, Colm Seale, Joana P. Gonçalves
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance to temozolomide (TMZ) chemotherapy due to DNA mismatch repair (MMR) enzyme deficiency is a barrier to improving outcomes for IDH wild-type glioblastoma (GBM) patients. KL-50 is a new imidazotetrazine-based therapeutic designed to induce DNA interstrand cross links, and subsequent double-stranded breaks, in an MMR-independent manner in cells with O-6-Methylguanine-DNA Methyltransferase (MGMT) deficiency. Previous research showed its efficacy against LN229 glioma cells with MMR and MGMT knockdown. Its activity against patient-derived GBM that model post-TMZ recurrent tumors is unclear.


METHODS
We created MMR-deficient GBM patient-derived xenografts through exposure to TMZ, followed by treatment with additional TMZ or KL-50. We also generated isogenic, MSH6 knockout patient-derived GBM and tested them for sensitivity to TMZ and KL-50.


RESULTS
KL-50 extended the median survival of mice intracranially engrafted with either patient-derived TMZ-naïve GBM6 or TMZ-naïve GBM12 by 1.75-fold and 2.15-fold, respectively (p<0.0001). A low dose (4 Gy) of fractionated RT further extended the survival of KL-50 treated GBM12 mice (median survival=80 days for RT+ KL-50 vs. 71 days KL-50 alone, P=0.018). KL-50 also extended the median survival of mice engrafted with post-TMZ, MMR-deficient GBM6R-m185 (140 days for KL-50 vs. 37 days for vehicle, p<0.0001). MSH6-KO increased TMZ IC50 for GBM6 and GBM12 cultures by >5-fold and >12-fold for cell death and live cell count outputs, respectively. In contrast, MSH6-KO actually decreased KL-50 IC50 by 10-80%.


CONCLUSION
KL-50-based compounds are a promising new strategy for the treatment of MGMT-deficient, MMR-deficient GBM that recurs after frontline TMZ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3a0f339bf4219c2fc489415b10b402bba9dd1" target='_blank'>
              The novel DNA cross-linking agent KL-50 is active against patient-derived models of new and recurrent post-temozolomide mismatch repair-deficient glioblastoma.
              </a>
            </td>
          <td>
            Matthew McCord, Thomas Sears, Wenxia Wang, Rahul Chaliparambil, Shejuan An, Jann Sarkaria, C. James, Bruce Ruggeri, Susan E. Gueble, R. Bindra, C. Horbinski
          </td>
          <td>2024-12-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract RNA modification has emerged as a crucial area of research in epigenetics, significantly influencing tumor biology by regulating RNA metabolism. N‐acetyltransferase 10 (NAT10)‐mediated N4‐acetylcytidine (ac4C) modification, the sole known acetylation in eukaryotic RNA, influences cancer pathogenesis and progression. NAT10 is the only writer of ac4C and catalyzes acetyl transfer on targeted RNA, and ac4C helps to improve the stability and translational efficiency of ac4C‐modified RNA. NAT10 is highly expressed and associated with poor prognosis in pan‐cancers. Based on its molecular mechanism and biological functions, ac4C is a central factor in tumorigenesis, tumor progression, drug resistance, and tumor immune escape. Despite the increasing focus on ac4C, the specific regulatory mechanisms of ac4C in cancer remain elusive. The present review thoroughly analyzes the current knowledge on NAT10‐mediated ac4C modification in cancer, highlighting its broad regulatory influence on targeted gene expression and tumor biology. This review also summarizes the limitations and perspectives of current research on NAT10 and ac4C in cancer, to identify new therapeutic targets and advance cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc21647bfb08f2bead5c258ee67888a30cbd41b0" target='_blank'>
              The role and mechanism of NAT10‐mediated ac4C modification in tumor development and progression
              </a>
            </td>
          <td>
            Zhuoran Gu, Libin Zou, Xinjian Pan, Yang Yu, Yongqiang Liu, Zhijin Zhang, Ji Liu, Shiyu Mao, Junfeng Zhang, Changcheng Guo, Wei Li, Jiang Geng, Wentao Zhang, Xudong Yao, Bing Shen
          </td>
          <td>2024-12-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) is characterized by recurrent somatic mutations in epigenetic regulators, which stratify patients into clinically significant subgroups with distinct prognoses and treatment responses. However, the cell type-specific epigenetic landscape of RCC—broadly and in the context of these mutations—is incompletely understood. To investigate these open questions, we integrated single nucleus ATAC sequencing data from RCC tumors across four independent cohorts. In clear cell RCC tumors, we identified four shared malignant epigenetic programs related to angiogenesis, proximal tubule-like features, interferon (IFN) signaling, and one that lacked distinct genomic regions with increased accessibility. Among the mutated epigenetic regulators, BAP1 mutation exhibited the most significant impact on chromatin accessibility in tumor cells, and the associated epigenetic changes were linked to IFN response. We identify multiple potential sources of elevated IFN signaling in these lesions, such as increased immune infiltration and increased accessibility and expression of an IFN-associated ERV, ERV3-16A3_LTR. We find that the expression of ERV3-16A3_LTR may itself be a negative prognostic biomarker in ccRCC. Our findings highlight the convergence of malignant epigenetic programs across ccRCC tumors and suggest that BAP1 loss, potentially through ERV3-16A3_LTR dysregulation, is associated with an IFN response-high epigenetic program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f60a362700f5a511019adc2819b352bb15feca9" target='_blank'>
              Single-cell epigenetic profiling reveals an interferon response-high program associated with BAP1 deficiency in kidney cancer
              </a>
            </td>
          <td>
            Sabrina Y. Camp, M. X. He, Michael S. Cuoco, E. Saad, Erica Pimenta, Kevin Meli, Z. Bakouny, C. Labaki, Breanna M. Titchen, Yun Jee Kang, Jack Horst, Rachel Trowbridge, Erin Shannon, K. Helvie, A. Thorner, S. Vigneau, Angie Mayorga, Jahnavi Kodali, Hannah Lachmayr, Meredith Bemus, Jihye Park, T. Choueiri, Kevin Bi, E. V. Van Allen
          </td>
          <td>2024-11-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="When nature evolves a gene over eons at scale, it produces a diversity of homologous sequences with patterns of conservation and change that contain rich structural, functional, and historical information about the gene. However, natural gene diversity accumulates slowly and likely excludes large regions of functional sequence space, limiting the information that is encoded and extractable. We introduce upgraded orthogonal DNA replication (OrthoRep) systems that radically accelerate the evolution of chosen genes under selection in yeast. When applied to a maladapted biosynthetic enzyme, we obtained collections of extensively diverged sequences with patterns that revealed structural and environmental constraints shaping the enzyme's activity. Our upgraded OrthoRep systems should support the discovery of factors influencing gene evolution, uncover previously unknown regions of fitness landscapes, and find broad applications in biomolecular engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62df0e938cd90390057d65a16ab4d97002811d4a" target='_blank'>
              Continuous evolution of user-defined genes at 1 million times the genomic mutation rate.
              </a>
            </td>
          <td>
            Gordon Rix, Rory L. Williams, Vincent J. Hu, Aviv Spinner, A. Pisera, Debora S. Marks, Chang C. Liu
          </td>
          <td>2024-11-08</td>
          <td>Science</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Tumor immune microenvironment (TIME) plays a key role to understand how tumors respond to prostate cancer (PC) therapies and potential mechanisms of resistance. Previous research has suggested that specific genomic aberrations, such as microsatellite instability (MSI) or CDK12 bi-allelic loss can allow PC patients more likely to respond to immune checkpoint inhibitors (ICI) or other immune therapies. However, responses to these treatments remain highly variable even in selected patients. Thus, it is essential to obtain more information about tumor immune cells that infiltrate these tumors, and on their plasticity and interactions, in order to better understand the underlying biology to allow development of new therapeutic strategies. This review analyzes: 1) How interactions among immune cell populations and other cells infiltrating the tumor stroma can modulate the progression of PC, 2) How the standard therapies to treat PC (such as androgen deprivation therapy, new androgen-directed hormone therapy or chemotherapy) may influence the dynamic changes of the immunome and 3) What are the limitations in characterizing the immune landscape of the host´s response to tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a0720f599ad6eaa2ff25316e47f65e17e367b2" target='_blank'>
              Immunome profiling in prostate cancer: a guide for clinicians
              </a>
            </td>
          <td>
            Luis San-Jose Manso, A. Alfranca, Ignacio Moreno-Pérez, María Ruiz-Vico, Clara Velasco, P. Toquero, María Pacheco, Almudena Zapatero, Diego Aldave, Guillermo Celada, Eduardo Albers, M. D. Fenor de la Maza, Jorge García, Elena Castro, David Olmos, Ramón Colomer, N. Romero-Laorden
          </td>
          <td>2024-11-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration has been implicated in transforming HPV infection into cancer. To resolve genome dysregulation associated with HPV integration, we performed Oxford Nanopore long-read sequencing on 72 cervical cancer genomes from an Ugandan dataset that was previously characterized using short-read sequencing. We found recurrent structural rearrangement patterns at HPV integration events, which we categorized as: del(etion)-like, dup(lication)-like, translocation, multibreakpoint, or repeat region integrations. Integrations involving amplified HPV-human concatemers, particularly multibreakpoint events, frequently harbored heterogeneous forms and copy numbers of the viral genome. Transcriptionally active integrants were characterized by unmethylated regions in both the viral and human genomes downstream from the viral transcription start site, resulting in HPV-human fusion transcripts. In contrast, integrants without evidence of expression lacked consistent methylation patterns. Furthermore, whereas transcriptional dysregulation was limited to genes within 200 kilobases of an HPV integrant, dysregulation of the human epigenome in the form of allelic differentially methylated regions affected megabase expanses of the genome, irrespective of the integrant's transcriptional status. By elucidating the structural, epigenetic, and allele-specific impacts of HPV integration, we provide insight into the role of integrated HPV in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8457c68dc264c60760742af32b0cda2346039" target='_blank'>
              Rearrangements of viral and human genomes at human papillomavirus integration events and their allele-specific impacts on cancer genome regulation.
              </a>
            </td>
          <td>
            Vanessa L. Porter, Michelle Ng, Kieran O’Neill, Signe MacLennan, R. Corbett, Luka Culibrk, Zeid Hamadeh, Marissa Iden, Rachel Schmidt, Shirng-Wern Tsaih, C. Nakisige, Martin Origa, Jackson Orem, Glenn Chang, Jeremy Fan, K. Nip, Vahid Akbari, Simon K. Chan, James Hopkins, Richard A. Moore, E. Chuah, K. Mungall, A. Mungall, Inanҫ Birol, S. Jones, Janet S. Rader, M. Marra
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790f4e8c7c6b66bd0a095a2a61d1ede88c46207f" target='_blank'>
              DNA damage response inhibitors in cancer therapy: lessons from the past, current status and future implications.
              </a>
            </td>
          <td>
            Yvette Drew, F. Zenke, Nicola J Curtin
          </td>
          <td>2024-11-12</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Objective This study investigates the impact of Titin (TTN) gene mutations on radiotherapy sensitivity in rectum adenocarcinoma (READ) by examining changes in the tumour immune microenvironment. Methods Data on gene expression and mutations in READ were obtained from The Cancer Genome Atlas (TCGA) and International Cancer Genome Consortium (ICGC) databases. Bioinformatics analysis explored the correlation between TTN mutations and immune cell infiltration. In vitro, lentiviral vectors were used to assess TTN mutations' effects on ANKRD1 expression in two READ cell lines. ANKRD1 was overexpressed, and clonogenic assays evaluated radiotherapy sensitivity. Flow cytometry, immunofluorescence, and comet assays examined mutations' impact on cell cycle, apoptosis, and DNA damage response (DDR). An in vivo mouse model and formalin‐fixed paraffin‐embedded samples from locally advanced rectal cancer (LARC) patients before and after radiotherapy were analyzed, followed by prognostic evaluation. Results Bioinformatics revealed that TTN mutations increase radiation sensitivity in LARC by slowing cell proliferation, promoting apoptosis, and reducing DDR. TTN mutations also inhibit ANKRD1 expression via JUN disruption and enhance CD4/CD8 T‐cell infiltration, improving anti‐tumour immunity and outcomes. Observations from the clinical study showed a substantial decline in ANKRD1 expression levels alongside a notable surge in the counts of CD4+ and CD8+ T cells after undergoing radiotherapy. Patients with TTN mutations, low ANKRD1 expression, and high densities of CD4+ and CD8+ T cells had longer 3‐year disease‐free survival in READ. Conclusion Our findings reveal that TTN mutations can serve as biomarkers for enhanced radiotherapy sensitivity in READ. By altering the tumour's immune microenvironment, these mutations may provide a novel target for personalized radiotherapy strategies, potentially improving therapeutic outcomes in patients with READ. Highlights The association between TTN mutations and tumour mutation burden, as well as immune cell infiltration in READ, is examined. TTN mutations enhance the radiation sensitivity of READ cells and weaken DNA damage repair in response to radiation. TTN mutations increase the radiation sensitivity of READ cells by inhibiting ANKRD1. The infiltration of CD8+ and CD4+ T cells induced by TTN mutations is essential for anti‐tumour immunity. TTN mutations serve as a biomarker for the pathological response to preoperative radiotherapy in READ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c88046e7da64567208e76b83bd85143084796" target='_blank'>
              Titin gene mutations enhance radiotherapy efficacy via modulation of tumour immune microenvironment in rectum adenocarcinoma
              </a>
            </td>
          <td>
            Hengchang Liu, Jialiang Liu, Xu Guan, Zhixun Zhao, Pu Cheng, Haipeng Chen, Zheng Jiang, Xishan Wang
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Despite recent advances in the biology of IDH-wildtype glioblastoma, it remains a devastating disease with median survival of less than 2 years. However, the molecular underpinnings of the heterogeneous response to the current standard-of-care treatment regimen consisting of maximal safe resection, adjuvant radiation, and chemotherapy with temozolomide remain unknown.


METHODS
Comprehensive histopathologic, genomic, and epigenomic evaluation of paired initial and recurrent glioblastoma specimens from 106 patients was performed to investigate the molecular evolution and cellular phenotypes underlying differential treatment responses.


RESULTS
While TERT promoter mutation and CDKN2A homozygous deletion were early events during gliomagenesis shared by initial and recurrent tumors, most other recurrent genetic alterations (eg, EGFR, PTEN, and NF1) were commonly private to initial or recurrent tumors indicating acquisition later during clonal evolution. Furthermore, glioblastomas exhibited heterogeneous epigenomic evolution with subsets becoming more globally hypermethylated, hypomethylated, or remaining stable. Glioblastoma that underwent sarcomatous transformation had shorter interval to recurrence and were significantly enriched in NF1, TP53, and RB1 alterations and the mesenchymal epigenetic class. Patients who developed somatic hypermutation following temozolomide treatment had significantly longer interval to disease recurrence and prolonged overall survival, and increased methylation at 4 specific CpG sites in the promoter region of MGMT was significantly associated with this development of hypermutation. Finally, an epigenomic evolution signature incorporating change in DNA methylation levels across 347 critical CpG sites was developed that significantly correlated with clinical outcomes.


CONCLUSIONS
Glioblastoma undergoes heterogeneous genetic, epigenetic, and cellular evolution that underlies prognostically different treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8c70c5b87e0a8ebd1138c27ed60e44214d7859" target='_blank'>
              Longitudinal multimodal profiling of IDH-wildtype glioblastoma reveals the molecular evolution and cellular phenotypes underlying prognostically different treatment responses.
              </a>
            </td>
          <td>
            Calixto-Hope G Lucas, N. Al-Adli, Jacob S. Young, Rohit Gupta, R. Morshed, Jasper Wu, Ajay Ravindranathan, Anny Shai, N. Oberheim Bush, Jennie W Taylor, John de Groot, J. Villanueva‐Meyer, M. Pekmezci, Arie Perry, Andrew W Bollen, P. Theodosopoulos, M. Aghi, E. Chang, Shawn L Hervey-Jumper, D. Raleigh, Annette M Molinaro, Joseph F Costello, Aaron A. Diaz, Jennifer L. Clarke, N. Butowski, Joanna J. Phillips, Susan M Chang, Mitchell S. Berger, D. A. Solomon
          </td>
          <td>2024-11-19</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="OBJECTIVE
Hematological malignancies often involve chromosomal translocations and fusion genes that drive disease progression. While MTCP1 is well-known in T-cell prolymphocytic leukemia (T-PLL), its role in myeloid neoplasms is less understood. This report presents the first identification of the t(X;1)(q28;q21) translocation leading to the GATAD2B::MTCP1 fusion in acute myeloid leukemia (AML) transformed from chronic myelomonocytic leukemia (CMML).


METHODS
The karyotypes were described according to the International System for Human Cytogenetic Nomenclature 2009. We performed targeted next-generation sequencing (NGS) on a panel of 172 genes commonly mutated in hematological malignancies (Supplemental Table 1), using an Illumina platform. RNA sequencing was conducted on total RNA extracted from bone marrow, also using the Illumina platform. The GATAD2B::MTCP1 fusion gene was confirmed by reverse transcription-polymerase chain reaction (RT-PCR) and Sanger sequencing, with specific primers for the fusion transcript (GATAD2B-F: CCTCTTTTTTTCGACGCC; MTCP1-R: ACTGAGCACAACACTTACGC).


RESULTS
The GATAD2B::MTCP1 fusion results from a breakpoint on 1q21 within GATAD2B exon 1 and Xq28 within MTCP1 exon 2. The patient with the GATAD2B::MTCP1 fusion exhibited disease progression from CMML to AML. Despite achieving initial remission with venetoclax-based therapy and allo-HSCT, the patient relapsed and died.


CONCLUSIONS
We propose that the GATAD2B::MTCP1 fusion upregulates MTCP1 expression rather than generating a fusion protein, thereby contributing to transformation and relapse in AML. Further investigations are needed to elucidate the precise role of this fusion event in myeloid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b642c96f7753533d71b6085fbf58b5dbe294ae5" target='_blank'>
              Identification of t(X;1)(q28;q21) generating a novel GATAD2B::MTCP1 gene fusion in CMML and its persistence during progression to AML.
              </a>
            </td>
          <td>
            Yi-Zi Liu, Fenghong Zhang, Chun-Xiao Hou, Zhi-Yu Zhang, Yiyan Zhu, Qian Wang, Yu Chen, Suqun Chen
          </td>
          <td>2024-12-18</td>
          <td>Hematology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb360932204eba380459d47c649431d652466b65" target='_blank'>
              Deep mutation, insertion and deletion scanning across the Enterovirus A proteome reveals constraints shaping viral evolution.
              </a>
            </td>
          <td>
            William Bakhache, Walker Symonds-Orr, Lauren McCormick, Patrick T. Dolan
          </td>
          <td>2024-11-28</td>
          <td>Nature microbiology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Many potential oncology targets are difficult to modulate by conventional classes of agents. Interfering with these targets at the level of RNA is a novel approach for difficult-to target proteins, as well as for non-coding RNAs, emerging targets that are part of the “dark matter’ of the genome. Antisense oligonucleotide (ASO) therapeutics are short sequences of 12- 30 DNA/RNA residues, designed to bind the cognate transcribed RNA. The ASO is chemically modified to increase stability, improve binding affinity, optimize pharmacokinetic properties and reduce immunostimulatory potential. The hybridization of ASO to cognate RNA results in cleavage of bound RNA by RNAse H1 and subsequent degradation. Several ASOs are marketed for treatment of rare diseases, but to date none have been approved for oncology indications. The long-noncoding RNA (lncRNA) MALAT1 is an example of an oncology target uniquely suited for an ASO therapeutic. This lncRNA is not translated and resides mostly in the nucleus, limiting access for most therapeutic approaches. MALAT1 is highly upregulated in various cancers, appears to play a role in epithelial to mesenchymal transition, and is associated with disease progression in patients. Inhibition of MALAT1 in preclinical breast cancer models, either genetic or pharmacologic using a murine ASO, leads to changes in tumor architecture and reduction in metastasis. FLM-7523 is a 16-mer ASO designed to bind to human MALAT1. It is a generation 2.5 design based on the constrained ethyl (cEt) modification to the 3 residues on the 3’ and 5’ ends of the ASO, and administered in a saline solution without any delivery vehicle. Preclinical studies have shown that FLM-7523 effectively inhibits MALAT1 expression in in vitro and in vivo models, and GLP toxicology studies have been completed to support a First-in-Human clinical trial. The signal transduction molecule STAT3 is another oncology target of interest that has, to date, been difficult to modulate. Danvatirsen is a 16-mer generation 2.5 ASO targeting human STAT3 mRNA and is currently in clinical trials. Preclinical in vivo studies in murine tumor models have demonstrated a reduction in STAT3 mRNA and protein following treatment with a STAT3 targeting ASO, along with tumor growth inhibition. Combination treatments with immune checkpoint inhibitors, including anti-PD-1 and anti-PD-L1, demonstrate improved tumor growth inhibition relative to monotherapy. In clinical trials, tumor uptake of danvatirsen is observed in post-treatment biopsies, with corresponding reduction in STAT3 protein. Danvatirsen is generally well tolerated with the most common adverse events being increased liver function tests and decreased myeloid hematologic parameters, both of which are easily monitored, manageable, and reversible. Clinical responses have been observed with the most robust being in combination with a PD-L1 inhibitor in recurrent/metastatic head and neck squamous cell carcinoma. In conclusion, ASOs are novel therapies that can inhibit oncology targets not tractable by traditional approaches.
 Citation Format: Andrew Denker. Antisense Oligonucleotides as Therapeutics for Difficult-to-Drug Targets in Oncology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr I008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/583396bf2c1cdcdd8ba3437f476b656eed400380" target='_blank'>
              Abstract I008: Antisense Oligonucleotides as Therapeutics for Difficult-to-Drug Targets in Oncology
              </a>
            </td>
          <td>
            Andrew Denker
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Epilepsy has a genetic predisposition, yet causal factors and the dynamics of the immune environment in epilepsy are not fully understood.


METHODS
We analyzed peripheral blood samples from epilepsy patients, identifying key genes associated with epilepsy risk through Mendelian randomization, using eQTLGen and genome-wide association studies. The peripheral immune environment's composition in epilepsy was explored using CIBERSORT. An epilepsy mouse model was established to validated the expression of key genes at the transcriptomic and proteomic levels through single-cell analysis. Relevant pathways were verified. Finally, we developed a predictive model for antiepileptic drug response in epilepsy patients.


RESULTS
We found that CDC25B, DNMT1, GZMA, MTX1, and SSH2 expression decreases epilepsy risk, whereas FGD3, RAF1, and SH3BP5L increase it. Epilepsy patients exhibited an altered peripheral immune profile, notably with increased activated mast cells and decreased CD4 memory activated T cells and γδ T cells. Eight genes were significantly related to this immune environment. In the animal model, FGD3, SSH2, and DNMT1 were upregulated at both mRNA and protein levels in the hippocampus. FGD3 and SSH2 are specifically elevated in microglia and are primarily associated with actin regulation. The trained predictive model was deployed on an online platform.


CONCLUSIONS
This study elucidates key genes linked to epilepsy, delineates the epilepsy immune landscape, and highlights the interaction between these domains, providing insights into potential epilepsy mechanisms and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854bd21d1bc874faf39064245eb4080956ac6c09" target='_blank'>
              Integrated Mendelian Randomization and Single-Cell Transcriptomics Analysis Identifies Critical Blood Biomarkers and Potential Mechanisms in Epilepsy.
              </a>
            </td>
          <td>
            Jianwei Shi, Jing Xie, Yanfeng Yang, Bin Fu, Zuliang Ye, Ting Tang, Quanlei Liu, Jinkun Xu, Penghu Wei, Yongzhi Shan, Guoguang Zhao
          </td>
          <td>2025-01-01</td>
          <td>CNS neuroscience & therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background. The management of pediatric acute myeloid leukemia (AML) is based on the prognostic risk classification of initial leukemia. Targeted next-generation sequencing (NGS) is a reliable method used to identify recurrently mutated genes of pediatric AML and associated prognosis.
Methods. In this study, we retrospectively evaluated the prognostic, and therapeutic utility of a targeted NGS panel covering twenty-five genes, in 21 children with de novo and 8 with relapsed or secondary AML.  
Results. Variants were detected in 44.8% of patients, and 63.2% of them were in the signaling pathway genes. The number of variants per patient and diversity increased with age. The panel results affected hematopoietic stem cell transplantation decisions, especially in core binding factor AML, and allowed the categorization of diseases according to current classifications. Panel results also pointed out predisposition to germline leukemia to the extent of the panel coverage. No targeted therapy was used based on the variants, and none of the variants were used to monitor minimal residual disease.
Conclusions. Targeted NGS results, along with well-known genetic aberrations and treatment responses, can guide treatment modalities. The coverage of the routine panels should include proven mutations of childhood AML and germline leukemia predisposition genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4601dddbafa15e4c68929e64e4ba8e0770802f" target='_blank'>
              The importance of targeted next-generation sequencing based genomic profiling at diagnosis of childhood acute myeloid leukemia: a single center experience
              </a>
            </td>
          <td>
            D. Kaçar, B. Çavdarlı, Ayça Koca Yozgat, Melek Işık, F. Kurtipek, Fatma Tuba Yıldırım, T. Bayhan, Dilek Gürlek Gökçebay, Namık Yaşar Özbek, N. Yaralı
          </td>
          <td>2024-12-19</td>
          <td>The Turkish Journal of Pediatrics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Regulatory mutations, coding sequence alterations, and gene deletions and duplications are generally expected to have qualitatively different effects on fitness. We aim to ground this expectation within a theoretical framework using evolutionary simulations of gene regulatory networks (GRNs) controlling the expression of fitness-related genes. We examined the distribution of fitness effects as a function of the type of mutation and the topology of the gene network. Contrary to our expectation, the GRN topology had more influence on the effect of mutations than the type of mutation itself. In particular, the topology conditioned (i) the speed of adaptation, (ii) the distribution of fitness effects, and (iii) the degree of pleiotropy, acting as explanatory factor for all mutation types. All mutations had the potential to participate in adaptation, although their propensity to generate beneficial variants differed according to the network topology. In scalefree networks, the most common topology for biological networks, coding mutations were more pleiotropic and overrepresented in both beneficial and deleterious mutations, while regulatory mutations were more often neutral. However, this observation was not general, as this pattern was reversed in alternative networks. These results highlight the critical role of network topology in defining mutations’ contributions to adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416ce7026f77cde0b3183fa7cbeac65ad92bf503" target='_blank'>
              Gene network topology drives the mutational landscape of gene expression
              </a>
            </td>
          <td>
            Sylvain Pouzet, Arnaud Le Rouzic
          </td>
          <td>2024-11-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a genetic anemia caused by the production of an abnormal adult hemoglobin. The clinical severity is lessened by elevated fetal hemoglobin (HbF) production in adulthood. A promising therapy is the transplantation of autologous, hematopoietic stem/progenitor cells (HSPCs) treated with CRISPR/Cas9 to downregulate the HbF repressor BCL11A via generation of double strand breaks (DSBs) in the +58-kb erythroid-specific enhancer. Here, to further enhance HbF production without increasing the mutagenic load, we targeted both +58-kb and +55-kb BCL11A erythroid-specific enhancers using base editors. We systematically dissected DNA motifs recognized by the key transcriptional activators within these regions and identified the critical nucleotides required for activator binding. Multiplex base editing of these residues was efficient and safe and generated no or little DSBs and genomic rearrangements. We observed substantial HbF reactivation, exceeding the levels achieved using the CRISPR/Cas9 nuclease-based strategy, thus efficiently rescuing the sickling phenotype. Multiplex base editing was efficient in long-term repopulating HSPCs and resulted in potent HbF reactivation in vivo. In summary, these results show that multiplex base editing of BCL11A erythroid-specific enhancers is a safe and potent strategy for treating sickle cell disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949a30362791a74e8f68a2ef1865b96856027877" target='_blank'>
              Multiplex base editing of BCL11A regulatory elements to treat sickle cell disease
              </a>
            </td>
          <td>
            L. Fontana, P. Martinucci, S. Amistadi, T. Felix, M. Mombled, A. Tachtsidi, G. Corre, A. Chalumeau, G. Hardouin, J. Martin, O. Romano, M. Amendola, P. Antoniou, A. Miccio
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Phenotypic variation among individuals plays a key role in evolution, since variation provides the material on which natural selection can act. One important link between genetic and phenotypic variation is gene expression. As for other phenotypes, the range of accessible expression variation is limited and biased by different evolutionary and developmental constraints. Gene expression variability broadly refers to the tendency of a gene to vary in expression (i.e., between individuals or cells) due to stochastic fluctuations or differences in genetic, epigenetic, or environmental factors, separately from the differences between e.g. organs. Variability due to biomolecular stochasticity (transcriptional ‘noise’) and cell-to-cell heterogeneity has been well-studied in isogenic populations of unicellular organisms such as bacteria and yeasts. However, for more complex organisms with multiple cells, tissues, and organs sharing the same genetic background, the interplay between inter-individual expression variability, gene and organ function, and gene regulation remains an open question. In this study, we used highly multiplexed 3’-end Bulk RNA Barcoding and sequencing (BRB-seq) to generate transcriptome profiles spanning at least nine organs in outbred individuals of three ray-finned fish species: zebrafish, Northern pike, and spotted gar. For each condition, we measured expression variation per gene independent of mean expression level. We observed that lowly variable genes are enriched in cellular housekeeping functions whereas highly variable genes are enriched in stimulus-response functions. Furthermore, genes with highly variable expression between individuals evolve under weaker purifying selection at the coding sequence level, indicating that intra-species gene expression variability predicts inter-species protein sequence divergence. Genes that are broadly expressed across organs tend to be both highly expressed and lowly variable between individuals, whereas organ-biased genes are typically highly variable within their top organ of expression. For genes with organ-biased expression profiles, we inferred differences in selective pressure on gene regulation depending on their top organ. We found that genes with peak expression in the brain have low inter-individual expression variability across non-nervous organs, suggesting stabilizing selection on regulatory evolution of brain-biased genes. Conversely, liver-biased genes have highly variable expression across organs, implying weaker regulatory constraints. These patterns show that gene regulatory mechanisms evolved differently based on constraints on the primary organ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae3441fc7f69de97d18458cb71329f2fa4d201d" target='_blank'>
              Inter-individual gene expression variability implies stable regulation of brain-biased genes across organs
              </a>
            </td>
          <td>
            C. F. Bucao, Consolée Aletti, Sébastien Moretti, Andrew W. Thompson, Brett L. Racicot, Catherine Wilson, J. Bobe, Ingo Braasch, Y. Guiguen, John H. Postlethwait, Marc Robinson-Rechavi
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer.
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, , Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="
 Introduction: A major contributor to prostate cancer (PCa) mortality is the development of resistance to single agents targeting androgen receptor (AR) signaling. We have developed an unbiased shRNA-based negative selection screen to identify small RNAs (shRNAs and siRNAs) that inhibit PCa cell growth and survival (toxic sRNAs). Computational analyses of the most toxic sRNAs identified revealed that each can simultaneously target multiple known AR-coregulatory mRNAs. AR-coregulators modulate AR transcriptional response and are essential to AR signaling. We have proposed that the identified toxic sRNAs inhibit androgen signaling through seed-mediated targeting of androgen regulatory gene networks, resulting in inhibition of cell growth and viability. The goal of the current studies was to begin to analyze the broad therapeutic potential of these toxic sRNAs for PCa. We focused on analyzing i) their potential to thwart cell growth and/or viability when delivered to preclinical models of PCa, and ii) the targetome of the toxic sRNAs as a mean to uncover novel AR-coregulators that represent a targetable axis within androgen signaling. Methods: i) cationic lipid nanoparticles (LNPs), that efficiently deliver the toxic siRNAs to PCa cell lines, were used to deliver toxic siRNAs to organoids grown from PCa patient derived xenografts (PDX) models. 3D image analysis using the Ominer® software package was used to determine organoid count and size, the numbers of nuclei per organoid, and the fraction of dead cells to discriminate between cytotoxic and cytostatic effects. ii) to identify novel AR-coregulators, toxic siRNA direct targets were identified by sequencing of transcripts captured by biotinylated toxic siRNAs. Directly bound targets, were subjected to pathway enrichment and network analysis followed by identification of hub genes (Cytoscape). The effect of individually silencing potential AR-coregulators on the expression of androgen responsive genes was analyzed using Nanostring. Results and Conclusions: i) The effect of the toxic siRNAs on growth and viability of the PCa PDX-derived organoids will be presented and discussed as compelling preliminary data for therapeutic testing in more complex pre-clinical models (PCa PDXs). ii) Based on bioinformatic and downstream analysis of the direct targets of two toxic siRNAs, we selected 12 potentially novel AR- coregulators, with varied known functions including chromatin modifiers, transcriptional regulators, etc. Silencing of several of these potential AR-coregulators significantly affect expression of numerous AR responsive genes. Future experiments will focus on analyzing their interaction and functional association with the AR (i.e. effect of AR binding to chromatin). In conclusion, we have developed an assay that allowed the identification of toxic sRNAs for PCa cells and demonstrated their potential as therapeutics and as a tool for discovery of potential novel targets for PCa.
 Citation Format: Joshua M Corbin, Hamilton Young, Stefan Wilhelm, Adam S Asch, Chao Xu, Maria J Ruiz Echevarria. Identification of small RNAs that function as multi-target therapeutics and as tools for novel target discovery in prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr B026.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76597cc9479b4e5e95620afe5db4d33d3815b07" target='_blank'>
              Abstract B026: Identification of small RNAs that function as multi-target therapeutics and as tools for novel target discovery in prostate cancer
              </a>
            </td>
          <td>
            Joshua M. Corbin, Hamilton Young, Stefan Wilhelm, Adam S Asch, Chao Xu, Maria J. Ruiz Echevarría
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract We developed Del-read, an algorithm targeting medium-sized deletions (6–100 bp) in short-reads, which are challenging for current variant callers relying on alignment. Our focus was on Micro-Homolog mediated End Joining deletions (MMEJ-dels), prevalent in myeloid malignancies. MMEJ-dels follow a distinct pattern, occurring between two homologies, allowing us to generate a comprehensive list of MMEJ-dels in the exome. Using Del-read, we identified numerous novel germline and somatic MMEJ-dels in BEAT-AML and TCGA-breast datasets. Validation in 672 healthy individuals confirmed their presence. These novel MMEJ-dels were linked to genomic features associated with replication stress, like G-quadruplexes and minisatellite. Additionally, we observed a new category of MMEJ-dels with an imperfect-match at the flanking sequences of the homologies, suggesting a mechanism involving mispairing in homology alignment. We demonstrated robustness of the repair system despite CRISPR/Cas9-induced mismatches in the homologies. Further analysis of the canonical ASXL1 deletion revealed a diverse array of these imperfect-matches. This suggests a potentially more flexible and error-prone MMEJ repair system than previously understood. Our findings highlight Del-read's potential in uncovering previously undetected deletions and deepen our understanding of repair mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece74b68770d9549840bdc9ed51b1bf59c99a07" target='_blank'>
              Utilizing insights of DNA repair machinery to discover MMEJ deletions and novel mechanisms
              </a>
            </td>
          <td>
            Aditee Kadam, Shay Shilo, Hadas Naor, Alexander Wainstein, Yardena Brilon, T. Feldman, Mark D. Minden, N. Kaushansky, N. Chapal-Ilani, L. Shlush
          </td>
          <td>2024-11-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Aim Pancreatic carcinoma is an aggressive cancer that progresses without many symptoms. The difficulty of early diagnosis and an inadequate response to traditional treatments also cause the survival rate of pancreatic cancer to be low. Current research is focusing on methods of diagnosis and treatment, such as gene therapy, to increase survival rates. Small interfering ribonucleic acid (siRNA) has emerged as a promising advanced therapeutic strategy for cancer treatment. This study sought to silence the KRAS gene in the human pancreatic carcinoma cell line using a complex of small interfering ribonucleic acid (siRNA) and gold nanoparticles (AuNP). Methods In this study, 25 nM siRNA and gold nanoparticles at 0.5 mg/ml, 0.25 mg/ml, and 0.125 mg/ml concentrations were used to silence the KRAS gene in the CAPAN-1 cell line. Real-time PCR analysis, agarose gel electrophoresis, and double staining were carried out, and xCelligence real-time cell analysis (RTCA) was used to measure proliferation. Results The PCR analysis revealed crossing point (CP) values of actin beta (ACTB) ranging from 33.04 to 35.98, which was in the expected range for all samples. The interaction between the gold nanoparticle/siRNA complex in the double staining analysis revealed that the most effective concentration of gold nanoparticle was 0.125 mg/ml. The WST-1 technique showed that siRNA/AuPEI cells in application groups had a viability rate of over 90%, indicating no toxicity or side effects. The xCELLigence RTCA® showed that at hour 72, there was a significant difference in proliferation in the 0.5 mg/mL PEI/AuNP-siRNA, 0.25 mg/mL PEI/AuNP-siRNA, and 0.125 mg/mL PEI/AuNP-siRNA application groups compared to the control and siRNA groups (p < 0.05). By hour 96, all three groups were statistically different from the control and siRNA groups in terms of proliferation (p < 0.05). Conclusions The results of this analysis suggest that the AuPEI/siRNA complex can be effectively used to silence the target gene, but more studies are needed to verify these results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ee59cc0bcf1def023f7abe057418a49f245a10" target='_blank'>
              Investigation of the efficacy of siRNA-mediated KRAS gene silencing in pancreatic cancer therapy
              </a>
            </td>
          <td>
            Büşra Küçükekmekci, Fatma Azize Budak Yıldıran
          </td>
          <td>2024-11-12</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Direct targeting of the KRAS-G12C–mutant protein using covalent inhibitors (G12Ci) acts on human non–small cell lung cancer (NSCLC). However, drug resistance is an emerging concern in this approach. Here, we show that MRTX849, a covalent inhibitor targeting the KRAS-G12C mutation, leads to the reactivation of the mitogen-activated protein kinase signaling pathway in MRTX849-resistant NSCLC and pancreatic ductal adenocarcinoma. A genome-wide CRISPR screen revealed that the adenosine triphosphate binding cassette transporter ABCC1 mediates MRTX849 resistance. Functional studies demonstrated that the transcription factor JUN drives ABCC1 expression, resulting in multidrug resistance. An unbiased drug screen identified the tyrosine kinase inhibitor dasatinib that potentiates MRTX849 efficacy by inhibiting SRC-dependent JUN activation, avoiding multidrug resistance and tumor suppression in vitro as well as in suitable preclinical mouse models and patient-derived organoids. SRC inhibitors (DGY-06-116, dasatinib, and bosutinib) also exhibit synergistic effects with MRTX849 in eliminating various tumor cell lines carrying KRAS-G12C mutations. Thus, SRC inhibitors amplify the therapeutic utility of G12Ci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73cf84601cd445f6b977aa9f10f264a438ee1f0c" target='_blank'>
              SRC kinase drives multidrug resistance induced by KRAS-G12C inhibition
              </a>
            </td>
          <td>
            Xinxin Song, Zhuan Zhou, Ammar Elmezayen, Runliu Wu, Chunhua Yu, B. Gao, John D Minna, Kenneth D Westover, Herbert J. Zeh, Guido Kroemer, Lynn E. Heasley, Rui Kang, D. Tang
          </td>
          <td>2024-12-11</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc6572745a00744a0ffe9484307c354e0370541" target='_blank'>
              Short tandem repeats delineate gene bodies across eukaryotes
              </a>
            </td>
          <td>
            William B. Reinar, A. Krabberød, V. O. Lalun, M. Butenko, K.S. Jakobsen
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1f1a3134c1ae8bfb1d2f982a790258260dcc9c9" target='_blank'>
              Advances in epigenetic therapies for B-cell non-hodgkin lymphoma.
              </a>
            </td>
          <td>
            Weiwen Hu, Lanlan Zang, Xiaoxi Feng, Shuhui Zhuang, Liudi Chang, Yongjing Liu, Jinyan Huang, Yuanyuan Zhang
          </td>
          <td>2024-12-09</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Specific and biologically informed treatments for medulloblastoma, especially the highly lethal TP53 mutant SHH subgroup, remain elusive, where radiotherapy is the primary treatment option. Applying genome-wide CRISPR-Cas9 screening in combination with lethal doses of radiotherapy, we identified the main driver of radiation resistance in SHH medulloblastoma is loss of p53. A negative selection CRISPR-Cas9 screen across multiple models of Trp53-deficient SHH medulloblastoma revealed a strong synthetic lethal interaction between components of the non-homologous end-joining pathway and radiation, particularly DNA protein kinase (DNA-PK) and its binding partners. Both genetic and pharmacological perturbation of DNA-PK enhanced radiosensitivity in TP53-deficient SHH medulloblastoma, leading to cell death. In vivo treatment of somatic and germline TP53-mutant SHH medulloblastoma models with peposertib, a small-molecule inhibitor of DNA-PK, significantly improved survival when combined with radiotherapy, strongly supporting further clinical investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9142d0bf16c5263f737ec56456cdebe6a89b5da8" target='_blank'>
              Targeting synthetic lethality between non-homologous end joining and radiation in very high-risk SHH medulloblastoma
              </a>
            </td>
          <td>
            Alexandria DeCarlo, G. Macleod, Carolina Fernandes da Silva, Lily Shen, Lucas Aragao, Mariska Sie, Deborah Termini, Jon Magee, Brian L. Gudenas, Sneha Sukumaran, Frederic Charron, Richard Marcellus, R. Al-Awar, Ahmed Aman, U. Tabori, C. Nör, P. Northcott, Peter Dirks, Stéphane Angers, Vijay Ramaswamy
          </td>
          <td>2024-11-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Over the last three decades, human genetics has gone from dissecting high-penetrance Mendelian diseases to discovering the vast and complex genetic etiology of common human diseases. In tackling this complexity, scientists have discovered the importance of numerous genetic processes – most notably functional regulatory elements – in the development and progression of these diseases. Simultaneously, scientists have increasingly used multiplex assays of variant effect to systematically phenotype the cellular consequences of millions of genetic variants. In this article, we argue that the context of genetic variants – at all scales, from other genetic variants and gene regulation to cell biology to organismal environment – are critical components of how we can employ genomics to interpret these variants, and ultimately treat these diseases. We describe approaches to extend existing experimental assays and computational approaches to examine and quantify the importance of this context, including through causal analytic approaches. Having a unified understanding of the molecular, physiological, and environmental processes governing the interpretation of genetic variants is sorely needed for the field, and this perspective argues for feasible approaches by which the combined interpretation of cellular, animal, and epidemiological data can yield that knowledge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc2f3e163ee6e921d0ab8446acca00c0495f358b" target='_blank'>
              Understanding genetic variants in context
              </a>
            </td>
          <td>
            Nasa Sinnott-Armstrong, Stanley Fields, Fritz Roth, Lea M. Starita, Cole Trapnell, Judit Villen, Douglas M. Fowler, C. Queitsch
          </td>
          <td>2024-12-03</td>
          <td>eLife</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML), a malignant disease of the bone marrow, is characterized by the clonal expansion of myeloid progenitor cells and a block in differentiation. The high heterogeneity of AML significantly impedes the development of effective treatment strategies. Enhancer of zeste homolog 2 (EZH2), the catalytic subunit of the polycomb repressive complex 2 (PRC2), regulates the expression of downstream target genes through the trimethylation of lysine 27 on histone 3 (H3K27me3). Increasing evidence suggests that the dysregulation of EZH2 expression in various cancers is closely associated with tumorigenesis. In the review, we examine the role of EZH2 in AML, highlighting its crucial involvement in regulating stemness, proliferation, differentiation, immune response, drug resistance and recurrence. Furthermore, we summarize the application of EZH2 inhibitors in AML treatment and discuss their potential in combination with other therapeutic modalities. Therefore, targeting EZH2 may represent a novel and promising strategy for the treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c772976d3b58810f697b6b1b73c79162dca59db" target='_blank'>
              The epigenetic role of EZH2 in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jinyong Fang, Jingcheng Zhang, Lujian Zhu, Xiaoru Xin, Huixian Hu
          </td>
          <td>2024-12-06</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Variants in the hereditary cancer-associated BRCA1 and BRCA2 genes can alter RNA splicing, producing transcripts that encode internally truncated yet potentially functional proteins. However, few studies have quantitatively analyzed variant-specific splicing isoforms. Here, we investigated cells heterozygous and homozygous for the BRCA2:c.681+5G>C variant. Using droplet digital RT-PCR, we identified two variant-specific mRNA isoforms. The predominant transcript is out-of-frame, contains a premature termination codon, and is degraded via the nonsense-mediated mRNA decay pathway. In addition, we detected a novel minor isoform encoding an internally truncated protein lacking non-essential domains. Homozygous mutant cells expressed low levels of BRCA2 protein and were defective in DNA repair. Using CRISPR-Cas9 gene editing, we induced the production of in-frame transcripts in mutant cells, which resulted in increased protein expression, enhanced RAD51 focus formation, and reduced chromosomal breaks after exposure to genotoxic agents. Our findings highlight the therapeutic potential of splicing modulation to restore BRCA2 function in mutant cells, offering a promising strategy to prevent cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e5b85723a00f90d4b01abffc19718d9fe90498" target='_blank'>
              Genetic modulation of RNA splicing rescues BRCA2 function in mutant cells.
              </a>
            </td>
          <td>
            Beatriz Anjo Lima, Ana Carolina Pais, Juliette Dupont, Patrícia Dias, Noélia Custódio, A. B. Sousa, Maria Carmo-Fonseca, C. Carvalho
          </td>
          <td>2024-12-31</td>
          <td>Life science alliance</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Antisense oligonucleotides (ASOs) are promising drugs capable of modulating the protein expression of virtually any target with high specificity and high affinity through complementary base pairing. However, this requires a deep understanding of the target sequence and significant effort in designing the correct complementary drug. In addition, ASOs have been demonstrated to be well tolerated during their clinical use. Indeed, they are already used in many diseases due to pathogenic RNAs of known sequences and in several neurodegenerative diseases and metabolic diseases, for which they were given marketing authorizations (MAs) in Europe and the United States. Their use in oncology is gaining momentum with several identified targets, promising preclinical and clinical results, and recent market authorizations in the US. However, many challenges remain for their clinical use in cancer. It seems necessary to take a step back and review our knowledge of ASOs and their therapeutic uses in oncology. The objectives of this review are (i) to summarize the current state of the art of ASOs; (ii) to discuss the therapeutic use of ASOs in cancer; and (iii) to focus on ASO usage in glioblastoma, the challenges, and the perspective ahead.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffd9426049fd86843a3a9afb40054bcb7a65473" target='_blank'>
              Clinical Applications of Antisense Oligonucleotides in Cancer: A Focus on Glioblastoma
              </a>
            </td>
          <td>
            Alexandre Khuu, M. Verreault, Philippe Colin, Helene Tran, A. Idbaih
          </td>
          <td>2024-11-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="BACKGROUND
Real-time reverse transcription quantitative polymerase chain reaction (RT-qPCR) is a powerful tool for analysing target gene expression in biological samples. To achieve reliable results by RT-qPCR, the most stable reference genes must be selected for proper data normalisation, particularly when comparing cells of different types. We aimed to choose the least variable candidate reference genes among eight housekeeping genes tested within a set of human cancer cell lines (HeLa, MCF-7, SK-UT-1B, A549, A431, SK-BR-3), as well as four lines of normal, non-malignant mesenchymal stromal cells (MSCs) of different origins.


METHODS
The reference gene stability was evaluated using four algorithms (BestKeeper, NormFinder, geNorm and the comparative ΔCt method) and ranked with the RefFinder web-based tool.


RESULTS
We found increased variability in the housekeeping genes' expression in the cancer cell lines compared to that in normal MSCs. POP4 and GAPDH were identified as the most suitable reference genes in cancer cells, while 18S and B2M were the most suitable in MSCs. POP4 and EIF2B1 were shown to be the least variable genes when analysing normal and cancer cell lines together. Epidermal growth factor receptor (EGFR) mRNA relative expression was normalised by the three most stable or three least stable reference genes to demonstrate the reliability of reference genes validation.


CONCLUSION
We analysed and selected stable reference genes for RT-qPCR analysis in the wide panel of cancer cell lines and MSCs. The study provides a reliable tool for future research concerning the expression of genes involved in various intracellular signalling pathways and emphasises the need for careful selection of suitable references before analysing target gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc012565e424f177faf0ef0b8356c2d0c19811fd" target='_blank'>
              Evaluation of Reference Gene Stability for Investigations of Intracellular Signalling in Human Cancer and Non-Malignant Mesenchymal Stromal Cells.
              </a>
            </td>
          <td>
            V. Kosheverova, Alexander Schwarz, R. Kamentseva, Marianna Kharchenko, Elena Kornilova
          </td>
          <td>2024-12-25</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Uveal melanoma (UM), arising from melanocytes in the choroid, accounts for 3% to 5% of all melanocytic tumors and over 70% of intraocular malignancies. Despite effective local treatments, metastasis remains a significant challenge, with more than half of patients developing metastatic disease within ten years. Conventional therapies often yield poor outcomes, highlighting the urgent need for novel therapeutic strategies to enhance survival and prognosis for UM patients. Methods We conducted a detailed analysis of the GSE139829 dataset, focusing on scRNA-seq data from eight primary UM patients and three with metastatic disease. Through clustering and marker gene expression analyses, we identified distinct subtypes of UM tumor cells and examined their transcriptional, metabolic, and intercellular communication profiles. We developed a novel prognostic model, PCOLCE TCs Risk Score (PTRS), centered on the C5 PCOLCE+ tumor cells, which was validated through in vitro functional assays. Additionally, we performed immune infiltration and metabolic pathway analyses to elucidate tumor-immune interactions and their clinical significance. Results We identified eight distinct cell types in UM and classified tumor subpopulations into six subgroups. The C5 PCOLCE+ TCs subpopulation was highlighted as crucial in UM malignancy, demonstrating high differentiation potential and a significant role in tumor progression. CellChat analysis revealed substantial communication between C5 PCOLCE+ TCs and fibroblasts, suggesting their involvement in tumor growth and extracellular matrix remodeling. Metabolic pathway analysis indicated enhanced oxidative phosphorylation and glutathione metabolism in this subpopulation. Additionally, we developed a PTRS model based on C5 PCOLCE+ TCs, identifying CITED1 as a high-risk gene that promotes UM cell proliferation, invasion, and migration in vitro. Conclusion This study provides insights into UM metastasis via single-cell analysis, identifying C5 PCOLCE+ TCs as key malignancy drivers associated with oxidative phosphorylation and immune interactions. Our PTRS model highlights CITED1 as a high-risk gene that promotes UM cell proliferation, paving the way for new prognostic models and therapeutic targets to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960495014c1bd91ad32e09912185659c46495106" target='_blank'>
              Single-cell RNA sequencing reveals key molecular drivers and immune landscape in uveal melanoma: implications for targeted therapy and prognostic modeling
              </a>
            </td>
          <td>
            Zeyu Song, Wenwen Shao, Zhikai Xiahou, Yue Xu, Xiaofeng Zhang
          </td>
          <td>2024-11-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="INTRODUCTION
Leukemia is typically categorized into myeloid leukemia and lymphoblastic leukemia based on the origins of leukemic cell. Myeloid leukemia is a group of clonal malignancies characterized by the presence of increased immature myeloid cells in both bone marrow and peripheral blood. Of note, the aberrant expression of specific proteins or the generation of fusion proteins due to chromosomal abnormalities are well established drivers in various forms of myeloid leukemia. Therefore, these oncoproteins represent promising targets for drug development.


AREAS COVERED
In this review, we comprehensively discussed the pathogenesis of typical leukemia oncoproteins and current landscape of small molecule drugs targeting these oncogenic proteins. Additionally, we elucidated novel strategies, including proteolysis-targeting chimeras (PROTACs), hyperthermia and genomic editing, which specifically degrade oncogenic proteins in myeloid malignancies.


EXPERT OPINION
Although small molecule drugs have significantly improved the prognosis of oncoprotein driven myeloid leukemia patients, drug resistance due to the mutations in oncoproteins is still a great challenge in clinic. New approaches such as PROTACs, hyperthermia and genomic editing are considered promising approaches for the treatment of oncoprotein-driven leukemia, especially for drug resistant mutants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea5fe5c5cfd460bcc9a1a8e4861d057f9becb9a" target='_blank'>
              Therapeutic approaches targeting oncogenic proteins in myeloid leukemia: challenges and perspectives.
              </a>
            </td>
          <td>
            Xing Yi Yan, Yuan-yuan Kang, Ze Yan Zhang, Ping Huang, Chang Yang, Hua Naranmandura
          </td>
          <td>2024-12-16</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="DNMT3A and TET2 are epigenetic regulator genes commonly mutated in age-related clonal hematopoiesis (CH). Despite having opposed epigenetic functions, these mutations are associated with increased all-cause mortality and a low risk for progression to hematological neoplasms. While individual impacts on the epigenome have been described using different model systems, the phenotypic complexity in humans remains to be elucidated. Here we make use of a natural inflammatory response occurring during coronavirus disease 2019 (COVID-19), to understand the association of these mutations with inflammatory morbidity (acute respiratory distress syndrome-ARDS) and mortality. We demonstrate the age-independent, negative impact of DNMT3A mutant CH on COVID-19-related ARDS and mortality. Using single cell (sc-) proteogenomics we show that DNMT3A mutations involve myeloid and lymphoid lineage cells. Using single cell multiomics sequencing, we identify cell-specific gene expression changes associated with DNMT3A mutations, along with significant epigenomic deregulation affecting enhancer accessibility, resulting in overexpression of IL32, a proinflammatory cytokine that can result in inflammasome activation in monocytes and macrophages. Finally, we show with single cell resolution that the loss of function of DNMT3A is directly associated with increased chromatin accessibility in mutant cells. Hence, we demonstrate the negative prognostic impact of DNMT3Amt CH on COVID-19 related ARDS and mortality. DNMT3Amt CH in the context of COVID-19, was associated with inflammatory transcriptional priming, resulting in overexpression of IL32. This overexpression was secondary to increased chromatic accessibility, specific to DNMT3Amt CH cells. DNMT3Amt CH can thus serve as a potential biomarker for adverse outcomes in COVID-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be6f734acb174ae5d3944a4074bf5564fa474087" target='_blank'>
              Single cell multiomics reveal divergent effects of DNMT3A and TET2 mutant clonal hematopoiesis in inflammatory response.
              </a>
            </td>
          <td>
            Wazim Mohammed Ismail, Jenna A. Fernandez, M. Binder, T. Lasho, Minsuk Kim, Susan M. Geyer, Amelia Mazzone, C. Finke, A. Mangaonkar, Jeong-Heon Lee, Liguo Wang, Kwan Hyun Kim, Vernadette A. Simon, Fariborz Rakhshan Rohakthar, Amik Munankarmy, Seul Kee Byeon, S. Schwager, Jonathan J Harington, Melissa R. Snyder, Keith D Robertson, Akhilesh Pandey, Eric D. Wieben, Nicholas Chia, Alexandre Gaspar-Maia, Mirinal S Patnaik
          </td>
          <td>2024-12-04</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Standard treatments in hormone receptor-positive (HR+)/HER2-metastatic breast cancer (mBC) typically involve endocrine therapy (ET) combined with CDK4/6 inhibitors, yet resistance to ET remains a persistent challenge in advanced cases. A deeper knowledge of the use of liquid biopsy is crucial for the implementation of precision medicine in mBC with real-time treatment guidance. Our study assesses the prognostic value of PIK3CA and ESR1 mutations in DNA derived from extracellular vesicles (EV-DNA) in longitudinal plasma from 59 HR+/HER2-mBC patients previously exposed to aromatase inhibitors, with a comparative analysis against circulating tumor DNA (ctDNA). Mutations were evaluated by digital PCR. PIK3CA and ESR1 mutations were found in 22 and 25% of patients. Baseline ESR1 mutations in EV-DNA were associated with shorter progression-free survival (PFS) across the cohort, with the Y537S mutation showing a particularly strong impact on the outcome of fulvestrant-treated patients. In contrast, PIK3CA mutations in EV-DNA did not significantly correlate with PFS, whereas in ctDNA, they were linked to poor outcomes. Altogether, this study positions EV-DNA as a valuable biomarker alongside ctDNA, enriching the understanding of different analytes in liquid biopsy and supporting strategies for HR+/HER2-mBC in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81a494e51740ee0fffc692f62981d9635e146bbd" target='_blank'>
              Molecular Profiling of Endocrine Resistance in HR+/HER2-Metastatic Breast Cancer: Insights from Extracellular Vesicles-Derived DNA and ctDNA in Liquid Biopsies
              </a>
            </td>
          <td>
            Ana Martínez-Rodríguez, Jesús Fuentes-Antrás, Víctor Lorca, Alfonso López de Sá, P. Pérez-Segura, F. Moreno, J. García-Saenz, V. García-Barberán
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Vγ9Vδ2T cells have the unique ability to recognize a broad range of malignant transformed cells. The tumor targeting event involving BTN2A1 and BTN3A1 dimers on the tumor cell surface is critical, leading to full activation of the TCR. Although the molecular mechanisms governing TCR engagement and T cell activation are well-characterized, the role of Vγ9Vδ2 T cells in cancer immune surveillance remains to be fully elucidated, particularly the mechanisms that enable these cells to discriminate between healthy and malignant cells at an early stage of malignant transformation. We employed two independent, genetically engineered step-wise mutagenesis models of human colorectal and breast cancer that mimic the transformation steps leading to tumor formation. We demonstrate that various single oncogenic mutations introduced into healthy organoids or cells, are sufficient to upregulate surface expressed BTN2A1 and enable Vγ9Vδ2 TCR binding to tumor cells. However, full activation of T cells through a Vγ9Vδ2TCR required additional subsequent phosphorylation of juxtamembrane (JTM) amino acids of BTN3A1, leading to the activating heterodimerization of BTN2A1 and 3A1. Using a protein interactome mapping pipeline, we identified PHLDB2, SYNJ2 and CARMIL1 as key players in controlling these delicate dual surface dynamics of BTN2A1 and 3A1 during early transformation. This mode of action allowed Vγ9Vδ2TCR T cells to control tumors in vitro and in vivo, emphasizing the crucial role of these molecules from early mutagenesis, to advanced cancer stages, and highlighting the therapeutic potential of a Vγ9Vδ2TCR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae97086c70f2495b91617599cd421e999f208f8" target='_blank'>
              Sensitivity to Vγ9Vδ2TCR T cells is imprinted after single mutations during early oncogenesis
              </a>
            </td>
          <td>
            A. Cleven, A. Meringa, P. Brazda, Domenico Fascì, T. Koorman, Tineke Aarts, Inez Johanna, D. Beringer, P. Hernández-López, S. Heijhuurs, Tomohiro Mizutani, Sangho Lim, M. Huismans, Jochem Bernink, David Vargas Diaz, Wei Wu, Esther San Jose, Jelle Schipper, Nikos Tsakirakis, Lauren Hoorens van Heyningen, Annick Nouwens, Lucrezia Gatti, T. Straetemans, Hugo Snippert, Jeanine Roodhart, P. Derksen, J. Drost, M. Altelaar, Albert J. R. Heck, Hans Clevers, Juergen Kuball, Z. Sebestyén
          </td>
          <td>2024-11-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Background Diffuse midline gliomas (DMGs) with histone H3K27M mutations represent a devastating paediatric brain cancer characterized by abysmal prognosis and limited treatment options. The only approved treatment is radiotherapy (RT), but most of the tumours relapse with fatal consequences. In this study, we sought to investigate whether irradiation leads to senescence induction and explore the efficacy of senolytics against DMG. Methods We have characterised the senescent phenotype of five genetically heterogeneous H3K27M-altered human DMG cell lines, combining cellular and/or molecular approaches. The sensitivity of senescent cells to Bcl-xL inhibition has been demonstrated in dose/response curves in vitro and in a PDX model of DMG. Results Here, we show that ionizing radiation induces senescence and SASP responses in both TP53 mutant and wild-type H3K27M-altered human DMG cell lines. We identify Navitoclax as a potent senolytic agent that selectively targets senescent DMG cells into apoptosis by inhibiting Bcl-xL. Related compounds, such as a proteolysis-targeting chimera (PROTAC)-mediated Bcl-xL degradation and a galacto-conjugated form of Navitoclax also show an effective senolytic activity in senescent cancer cells. Finally, we show that a combination therapy of irradiation and Navitoclax results in reduced tumor burden and increased mouse survival in an orthotopic xenograft DMG model. Conclusion These results offer a rationale for further clinical development of senolytic therapies as part of multimodal treatment approaches for DMG patients/ Key Points Ionising irradiation induces senescence in human DMG cells independently of the p53 status. Bcl-xL inhibition results in apoptosis of human DMG senescent cells in synergy with irradiation. Combination of irradiation and BcL-xL inhibition reduces tumourigenesis in a PDX model of DMG. Importance of the Study H3K27M-altered DMG are devastating paediatric tumours with an abysmal prognosis. The only approved treatment is radiotherapy but this is palliative and tumours almost always relapse with fatal consequences for the patients. In this study, we show that radiotherapy results in senescence induction in five genomically heterogeneous human DMG cell lines. We identify that drugs targeting the anti-apoptotic protein Bcl-xL show a strong senolytic activity in conjunction with radiotherapy both in vitro in DMG cells and in vivo in a PDX model of H3K27M-altered DMG. Treatment with Bcl-xL inhibitor Navitoclax, or related compounds targeting Bcl-xL protein degradation or containing a galactose conjugated form of Navitoclax results in DMG cancer cell apoptosis. As several of these inhibitors are currently being tested in ongoing clinical trials against other diseases, our data support the use of Bcl-xL inhibition mediated senolytics as an adjuvant therapy to radiotherapy to potentially improve outcomes in this challenging disease setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b580291efecff13cf295c14d6b2926c855610e2" target='_blank'>
              Targeted inhibition of Bcl-xL following radiation reduces tumourigenesis in preclinical models of H3K27M-altered diffuse midline glioma
              </a>
            </td>
          <td>
            Ashley Vardon, S. Haston, Romain Guiho, Diana Carvahlo, Rebecca Carter, J. Boult, Daniel Gharai, Diwakar Santhakumar, Ielyaas Cloete, James Gray, John Apps, O. Ogunbiyi, Daohong Zhou, Guangrong Zheng, Ramón Martínez-Máñez, Mark Lythgoe, Laura Donovan, A. Carcaboso, Owen Williams, Jesus Gil, Thomas Jacques, Jamie Dean, D. Michod, Chris Jones, Darren Hargrave, J. Martínez-Barbera
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 Introduction:
 Our previous studies have shown that sustained activation of the DNA damage response (DDR) in cardiomyocytes leads to p53/p21 activation and cardiac dysfunction. Although the DDR generally involves molecules in DNA replication and repair pathways, the non-proliferative nature of cardiomyocytes suggests a cardio-specific DDR mechanism. However, our understanding of DDR in cardiomyocytes remains limited. Here, we aim to use CRISPR interference (CRISPRi) knockdown screens to identify genes critically involved in DDR regulation in human cardiomyocytes. We hypothesize that identifying these gene clusters may allow us to develop methods to prevent cardiac dysfunction by suppressing DDR in cardiomyocytes.


 Methods and Results:
 We established a human iPS cell line stably expressing dCas9-KRAB, which allows CRISPRi-mediated gene knockdown, and differentiated the cells into cardiomyocytes. The resulting human iPS cell-derived cardiomyocytes (hiPSCMs) showed the achievement of approximately 80% knockdown efficiency after gRNA transfection. We stimulated the hiPSCMs with H
 2
 O
 2
 and quantitatively evaluated the expression levels of the DDR markers γH2AX and p21 by immunostaining using the Operetta
 ®
 high content imaging system. The DDR markers showed a significant concentration-dependent increase in response to H
 2
 O
 2
 administration. For arrayed CRISPRi screening, we constructed a gRNA library targeting 437 DDR-related genes. Using this library, we knocked down each DDR-related gene in hiPSCMs followed by H
 2
 O
 2
 stimulation. We quantified the expression levels of DDR markers by calculating the fluorescence intensity ratios relative to control after gene knockdown, and standardized them to calculate Z scores for all 437 genes. The screening successfully revealed the differential impact of each gene knockdown on γH2AX and p21 expression. We identified 71 genes that significantly affected their expression (Z-score < -1 or > 1). Mapping these genes to DDR pathways highlighted the differential impact of gene knockdown within the same pathway, and stratified their importance in cardiomyocytes.


 Conclusions:
 Arrayed CRISPR screening using hiPSCMs revealed differential functional significance of DDR-related genes in cardiomyocytes, identifying 71 genes of particularly significant importance. These findings provide a critical understanding of the cardio-specific DDR pathway and important clues for establishing an appropriate method to suppress DDR in the failing heart.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/439d21c245bf6379b70f4b13da6c727542ea5aee" target='_blank'>
              Abstract 4143847: CRISPR screening identifies critical factors regulating DNA damage response in human cardiomyocytes under oxidative stress
              </a>
            </td>
          <td>
            Masayuki Kubota, Masamichi Ito, S. Nomura, Norihiko Takeda, I. Komuro
          </td>
          <td>2024-11-12</td>
          <td>Circulation</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Defining consequential differences in intestinal epithelial stem cells in healthy humans versus those with inflammatory bowel disease (Crohn’s disease and ulcerative colitis) is essential for the development of much needed therapies to restore the epithelial barrier and maintain its fidelity. Employing single cell transcriptomic/epigenomic approaches and colonoid models from children and adults with Crohn’s disease led us to identify an inflammatory secretory progenitor (ISP) cell state present almost exclusively in patients with Crohn’s disease compared to control subjects. ISPs exhibit gene expression profiles consistent with normal secretory progenitor cells but concomitantly express a suite of distinguishing pro-inflammatory genes. Mechanistically, ISPs exhibit open chromatin and gene expression of ISP signature genes. While these ISP-specific genes are not expressed in intestinal stem cells, their chromatin is accessible in Crohn’s disease stem cells suggesting that ISP genes are epigenetically poised in stem cells and are transcriptionally activated in ISP cells in the presence of inflammatory stimuli. Consistently, Crohn’s disease colonoids exhibit sustained ISP gene expression that can be elicited further with pro-inflammatory cytokines or via co-culture with pro-inflammatory macrophages. In summary, we define differences in the epithelial stem and progenitor compartment of patients with Crohn’s disease suggesting aberrant stem cell differentiation and inflammatory gene expression arises during disease. HIGHLIGHTS We identify an inflammatory secretory progenitor cell state in endoscopically non-inflamed tissue from pediatric and adult patients with Crohn’s disease. Crohn’s disease epithelial stem and progenitor cells display increased chromatin accessibility preceding the emergence of inflammatory secretory progenitor cell states. Crohn’s disease colonoids from non-inflamed regions contain inflammatory secretory progenitor cells in the absence of inflammatory stimuli Inflammatory secretory progenitor cells expand in colonoid culture in response to cytokine stimulation or co-culture with pro-inflammatory macrophages">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1c97ff6cf153588b934b86c33ad5324b1f6d33" target='_blank'>
              An epigenetic basis for sustained inflammatory epithelial progenitor cell states in Crohn’s disease
              </a>
            </td>
          <td>
            Tatiana A. Karakasheva, Clara Morral Martinez, Yusen Zhou, Jingya Qui, Xinyi E. Chen, Gloria E. Soto, Shaneice K. Nettleford, Olivia T. Hix, Daana M. Roach, Alyssa M. Laguerta, Anusha Thadi, Rachael M. Edwards, Daniel Aleynick, Sarah Weinbrom, Elizaveta Borodyanskaya, Oliver Pickering, Marykate Fulton, Chia- Chen, Bella V. Peterson, Erik B. Hagen, Ian P. Yannuzzi, Zainab Haider, Zvi Cramer, Maire C Conrad, Ning Li, Meenakshi Bewtra, Yasin Uzun, Kai Tan, J. Kelsen, Andy J. Minn, Christopher Joachim Lengner, Kathryn E. Hamilton
          </td>
          <td>2024-11-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="In this data paper we present a study of RNA expression in association with the disease course of acute myeloid leukemia (AML). We have previously identified aldehyde dehydrogenase genes ALDH1A1 and ALDH2 as prospective actionable targets in AML. ALDH1A1 is expected to have key functions in leukemia stem-like cells that are prone to a dormant state in terms of metabolic activity and proliferation. Cells with a higher activity of metabolism as a whole and in mitochondria in particular, are likely to generate a higher abundance of formaldehyde and acetaldehyde. Cell survival necessitates removal of formaldehyde and acetaldehyde, which is to a substantial degree the function of ALDH2. AML cells with a mutant NPM1 gene permit higher MYC activity which would lead to increased metabolic activity. Extended MYC activity is allowed by the mutant NPM1 protein, compared to NPM1 wild-type. Here, we show that survival analysis in patients with NPM1 mutant AML yields a higher hazard ratio for ALDH2 RNA than for ALDH1A1, which is not the case for patients with wild-type NPM1. This result is consistent with the difference in enzymatic function between ALDH2 and ALDH1A1, the latter of which is not suited for small aldehydes, especially formaldehyde. This should open the door to the examination of ALDH2 inhibitors such as the clinically approved disulfiram, for the treatment of NPM1 mutant AML that proves refractory to ALDH1A1 inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3bb3ed3bfd23dfd95787be2d92e76b3566389ea" target='_blank'>
              RNA expression data suggest ALDH2 inhibition as a concept for killing acute myeloid leukemia cells that are resistant to ALDH1 inhibitors: the example of NPM1 mutant AML
              </a>
            </td>
          <td>
            G. Dancik, S. Vlahopoulos
          </td>
          <td>2024-11-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="For long, genome-wide coverage has been used as a measure of sequencing quality and quantity, but the biology hidden beneath has not been fully exploited. Here we performed comparative analyses on genome-wide coverage profiles between nucleus genome DNA (gDNA) samples from the 1000 Genomes Project (n=3,202) and cell-free DNA (cfDNA) samples from healthy controls (n=113) or cancer patients (n=362). Regardless of sample type, we observed an overall conserved landscape with coverage segmentation, where similar levels of coverage were shared among adjacent windows of genome positions. Besides GC-content, we identified protein-coding gene density and nucleosome density as major factors affecting the coverage of gDNA and cfDNA, respectively. Differential coverage of cfDNA vs gDNA was found in immune-receptor loci, intergenic regions and non-coding genes, reflecting distinct genome activities in different cell types. A further rise in coverage at non-coding genes/intergenic regions and a further drop of coverage at protein-coding genes/genic regions within cancer cfDNA samples suggested a relative loss of contribution by normal cells. Importantly, we observed the distinctive convergence of coverage in cancer-derived cfDNA, with the extent of convergence positively correlated to stages. Based on the findings we developed and validated an outlier-detection approach for cfDNA-based cancer screening without the need of cancer samples for training. The method achieved 97% sensitivity on pediatric sarcomas (n=241) and 44% sensitivity on early-stage lung cancers (n=36) with >90% specificity for condition-matched tasks, 100% sensitivity on late-stage cancers (n=85) for condition-unmatched tasks, outperforming current benchmarks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58763bc618f0e854c5267409a63aba70151b9e8e" target='_blank'>
              Coverage landscape of the human genome in nucleus DNA and cell-free DNA
              </a>
            </td>
          <td>
            Jiaqi Luo, Shuaicheng Li
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery.
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-03</td>
          <td>Cell research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Alternative polyadenylation (APA) is markedly dysregulated across cancer types. Although this phenomenon is well established and associated with altered patient prognosis, the functional importance of most dysregulated polyadenylation to cancer phenotypes remains unknown, as does the potential therapeutic impact of dysregulated APA. Recent work by our lab and others to address this gap has systematically identified specific APA events that alter key tumor phenotypes, including malignant cell growth and tumor recognition by cytotoxic T cells. Motivated by these studies, we sought to identify small molecules that modulate APA for therapeutic intent. We created a sensitive reporter of APA and used this reporter to conduct a molecular phenotypic reporter screen for APA-altering compounds. This screen revealed a diversity of compounds that altered APA transcriptome-wide. Phenotypic characterization of the strongest hit emerging from this screen revealed that it strongly induced cell differentiation and death of myeloid leukemia cells, with negligible toxicity for normal hematopoietic stem cells. Overall, these results highlight the untapped but promising therapeutic potential of targeting APA in cancer.
 Citation Format: Toshihiro Banjo, Satoshi Kaito, Austin M Gabel, Omar Abdel-Wahab, Robert K Bradley. Alternative polyadenylation as a therapeutic target in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr I002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a59f242b855efb6a0b407734c15c83806a10c213" target='_blank'>
              Abstract I002: Alternative polyadenylation as a therapeutic target in cancer
              </a>
            </td>
          <td>
            Toshihiro Banjo, Satoshi Kaito, Austin M. Gabel, Omar Abdel-Wahab, Robert K Bradley
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50ed629581e0a6379aef290f044fad2c2797280b" target='_blank'>
              Proteomic profiling of gliomas unveils immune and metabolism-driven subtypes with implications for anti-nucleotide metabolism therapy
              </a>
            </td>
          <td>
            Jinsen Zhang, Rui Sun, Yingying Lyu, Chaxian Liu, Ying Liu, Yuan Feng, Minjie Fu, Peter Jih Cheng Wong, Zunguo Du, Tianming Qiu, Yi Zhang, Dongxiao Zhuang, Zhiyong Qin, Yu Yao, Wei Zhu, Tiannan Guo, Wei Hua, Hui Yang, Ying Mao
          </td>
          <td>2024-11-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4d7fc87f26192c026401538b3ff0f4f71ccbc54" target='_blank'>
              The PI3K-AKT-mTOR axis persists as a therapeutic dependency in KRASG12D-driven non-small cell lung cancer
              </a>
            </td>
          <td>
            W. McDaid, L. Wilson, H. Adderley, A. Martinez-Lopez, M. Baker, J. Searle, L. Ginn, T. Budden, M. Aldea, A. Marinello, J. Aredo, A. Viros, B. Besse, H. Wakelee, F. Blackhall, S. Castillo-Lluva, C. Lindsay, A. Malliri
          </td>
          <td>2024-11-12</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d24c4e944e46fe6552525ff683c3906cad05e1d" target='_blank'>
              KDELR1 regulates chondrosarcoma drug resistance and malignant behavior through Intergrin-Hippo-YAP1 axis
              </a>
            </td>
          <td>
            Huabin Yin, Dongjie Jiang, Yongai Li, Wenjun Chen, Jie Zhang, Xinghai Yang, Jinbo Hu, Haifeng Wei
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetic processes play key roles in regulating gene expression, genome stability, and metabolic output in organisms across the tree of life. Yet, the role epigenetics plays in regulating genomes and behaviors remains underdeveloped for microalgae, particularly as new species are identified and characterized. This is likely due to the cumbersome nature and species-dependent attributes of epigenetic wet-lab methodologies, which preclude the rapid identification of epigenetic modifications and modulators. However, there is extraordinary conservation of epigenetic processes from budding yeast to humans; in many cases, one may infer how behavior and function are epigenetically regulated in novel species by simply identifying epigenetic modulators, or the proteins responsible for conferring epigenetic modifications. To this end, we have developed a graphical software package, titled PERCEPTIVE (pipeline for the prediction of epigenetic modulators in novel species). This novel platform solely uses the genomic sequence of an organism, and preexisting information from model species, to predict the epigenetic modulators and associated modifications in a novel species. Predictions are presented to the user in a graphical interface, which provides literature-based interpretation of results, enabling users to quickly understand potential epigenetic processes in their species of interest and plan follow-up experiments. To test PERCEPTIVE, we predicted epigenetic modulators in several feedstock candidate algae species. To validate these predictions, wet-lab studies were performed, including mass spectrometry; these results underscore the high accuracy of PERCEPTIVE predictions. Overall, PERCEPTIVE represents a powerful tool for the research and manipulation of algal species, which does not require a priori knowledge of epigenetics and is accessible to a broad set of investigators.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb34ee3f1ae2f57ecfe1bc910338d94272b5ae12" target='_blank'>
              PERCEPTIVE: an R Shiny pipeline for the prediction of epigenetic modulators in novel species
              </a>
            </td>
          <td>
            Eric M Small, Christina R. Steadman
          </td>
          <td>2024-11-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction: Metastatic castration-resistant prostate cancer (mCRPC) is a heterogeneous disease which can be classified into clinically relevant subtypes based on the expression of genes, such as the androgen receptor (AR) and neuroendocrine markers. Neuroendocrine prostate cancer (NEPC), characterized by gain of stem-like and neuroendocrine features and lack of AR expression is a clinically aggressive variant. Due to the lack of adequate biomarkers, NEPC is usually detected at a very advanced stage. There is mounting evidence that molecular subtype changes seen in NEPC are enforced by widespread epigenetic alterations, in particular DNA methylation changes. In this study, we aim to devise a novel DNA methylation-based assay for molecular subtyping and disease monitoring from cell-free DNA (cfDNA). Methods: We analyzed genome wide methylation patterns in 56 prostate cancer patient-derived xenograft (PDX) and 128 mCRPC tumors using array- and sequencing-based assays. We integrated DNA methylation at promoters, gene bodies and transcription factor binding site (TFBS) to determine the landscape of methylation alterations at key lineage specific genes. Using whole genome methylation derived from tissue with matched expression data we developed a deep learning framework to predict gene expression directly from tissue or cfDNA. Using key marker genes, the model was used to discern tumor molecular phenotypes from tissue and cfDNA in three independent cohorts of mCRPC patients using whole genome bisulfite sequencing and low-pass Enzymatic Methyl-Seq (EM-seq). Results: We observed a tight association between promoter, gene body and TFBS methylation with gene expression. Inferring gene expression from methylation for lineage specific markers such as AR, KLK3, ASCL1, INSM1, SRRM4 and DLL3 we classified molecular subtypes from both tissue and cfDNA. Additionally, for AR and ASCL1, we identified core sets of TFBSs whose differential methylation allowed for accurate assay-independent molecular subtype quantification. Applying the optimized quantitative model to mCRPC patients who underwent comprehensive tissue sampling by rapid autopsy we observed accurate subtype classification from both tissue samples and cfDNA for all cases. A similar analytical performance was observed in additional clinical mCRPC cohorts with cfDNA. Conclusion: Whole-genome methylation analysis of cfDNA allows for the prediction of gene expression patterns in tumor tissues, enabling non-invasive tumor subclassification and assessment of therapeutic targets.
 Citation Format: Mohamed Adil, Brian Hanratty, Pallabi Mustafi, Chitvan Mittal, Helen Richards, Ilsa Coleman, Radhika Patel, Anna-Lisa Doebley, Robert Patton, Eden Cruikshank, Patricia Galipeau, Ruth Dumpit, Martine Roudier, Jin-Yih Low, Navonil Sarkar, Robert Montgomery, Eva Corey, Colm Morrissey, Peter Nelson, Gavin Ha, Michael Haffner. Advance prostate cancer detection through epigenomic profiling of cell-free DNA [abstract]. In: Proceedings of the AACR Special Conference: Liquid Biopsy: From Discovery to Clinical Implementation; 2024 Nov 13-16; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2024;30(21_Suppl):Abstract nr PR011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d75835ad73ce4e1b3a9a9511ee1b494fe2c5ff" target='_blank'>
              Abstract PR011: Advance prostate cancer detection through epigenomic profiling of cell-free DNA
              </a>
            </td>
          <td>
            M. Adil, B. Hanratty, Pallabi Mustafi, Chitvan Mittal, Helen Richards, Ilsa Coleman, Radhika A Patel, Anna-Lisa Doebley, Robert Patton, Eden Cruikshank, Patricia Galipeau, R. Dumpit, Martine P. Roudier, J. Low, N. Sarkar, R. Montgomery, E. Corey, C. Morrissey, Peter S Nelson, Gavin Ha, Michael C Haffner
          </td>
          <td>2024-11-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91c207d0383664748707c83abd8bdd12dcda845" target='_blank'>
              TET2 mutation in acute myeloid leukemia: biology, clinical significance, and therapeutic insights
              </a>
            </td>
          <td>
            Qiang Gao, Kefeng Shen, Min Xiao
          </td>
          <td>2024-11-09</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA), highly amplified circular DNA, is widespread in human cancers and serves as a primary location for oncogene amplification. Due to its dynamic structural rearrangement and asymmetric inheritance during cell division, ecDNA increases tumor genome complexity, contributing to drug resistance and shortened patient survival. Here, we show that ecDNA-mediated genomic amplification is highly associated with transcript fusion events, which are prevalent in cancer and often lead to tumor development. We discovered that transcripts with the highest fusion events originate from genes amplified on ecDNA across ecDNA(+) cell lines. Notably, PVT1 (Plasmacytoma Variant Translocation 1) long noncoding RNA, which is known to be a hotspot for chromosomal translocation, is the most frequently fused RNA species in MYC/PVT1-amplified ecDNA(+) cancer cells. Exon 1 of PVT1 is the predominant fusion partner and confers RNA stability, increasing the RNA abundance of the partner oncogene. Using a model cell line with PVT1-MYC fusion on ecDNA, we found that RNA expression of PVT1-MYC increases in vivo in an ecDNA-dependent manner, while canonical MYC does not show a prominent increase. Additionally, ectopic expression of PVT1- MYC in MYC-depleted cancer cells provides higher rescue efficiency than canonical MYC. In contrast, a PVT1-MYC mutant that is unable to enhance RNA stability fails to rescue, highlighting the functional significance of PVT1-mediated RNA stabilization in cancer. This study unveils the link between PVT1 fusion and ecDNA in cancer, providing insights for developing diagnostic and therapeutic approaches tailored to target ecDNA.
 Citation Format: Hyerim Yi, Shu Zhang, Matthew G Jones, Julia A Belk, Jason Swinderman, Quanming Shi, Ellis J Curtis, Vishnu P Kanakaveti, Paul S Mischel, Howard Y Chang. PVT1 fusion on extrachromosomal DNA (ecDNA) increases oncogene RNA stability and cancer cell growth [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: RNAs as Drivers, Targets, and Therapeutics in Cancer; 2024 Nov 14-17; Bellevue, Washington. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(11_Suppl):Abstract nr PR014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b099680e1d89fbf4dffca6c72e1bc67243614d39" target='_blank'>
              Abstract PR014: PVT1 fusion on extrachromosomal DNA (ecDNA) increases oncogene RNA stability and cancer cell growth
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Matthew G Jones, J. Belk, Jason Swinderman, Quanming Shi, Ellis J Curtis, Vishnu P Kanakaveti, P. Mischel, Howard Y. Chang
          </td>
          <td>2024-11-14</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>25</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [12, 1],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>