<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-11-03 09:32:43 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>63</td>
          <td>45</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Platinum-based chemotherapy resistance remains a critical barrier in colorectal cancer (CRC) treatment. In this study, cytogenetic karyotyping was combined with transcriptomic profiling (RNA-seq) to elucidate resistance mechanisms by comparing two CRC cell lines: oxaliplatin-sensitive HCT116 and its resistant derivative HCT116 oxpl-R. Karyotyping unveiled tetraploidization and extensive genomic rearrangements in resistant cells, accompanied by prominent transcriptomic changes: 1807 differentially expressed genes (1216 upregulated and 519 downregulated). Pathway enrichment highlighted altered redox homeostasis and metabolic adaptation. Specifically, HCT116 oxpl-R cells exhibited elevated reactive oxygen species (ROS) production and enhanced energy metabolism. These findings establish a direct association between structural genomic alterations, transcriptional rewiring, and functional phenotypes in platinum resistance, providing a framework for targeting metabolic vulnerabilities in refractory CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df1c5397ea3bf691ad1c8a6e4e4cf327eb00a1d" target='_blank'>
              Transcriptomic and Cytogenetic Analysis of Oxaliplatin-Resistant Colorectal Adenocarcinoma HCT116 Cells to Identify Markers Associated with Platinum Resistance
              </a>
            </td>
          <td>
            A. Morshneva, O. Gnedina, Ksenia Fedotova, Natalija Yartseva, Nikolay Aksenov, M. Igotti
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquired therapy resistance is a dynamic process associated with early epigenetic modifications reshaping gene transcription across multiple cellular pathways, ultimately giving rise to drug-tolerant persister (DTP) cells. Unraveling the mechanisms that sustain DTP cell survival and drive their evolution into stable drug-resistant cells (DRC) is crucial for developing targeted therapies. Cisplatin-tolerant and cisplatin-resistant models were established using liver and gastric cancer cell lines for the first time to explore these mechanisms. Our investigation centered on the distinct epigenetic landscapes of DTP and DRC cells following cisplatin exposure. RNA sequencing revealed that DTP cells exhibit downregulation of pathways involved in cell cycle regulation, DNA replication, transcription, and chromatin maintenance while upregulating those associated with cell–cell communication and cytokine signaling. Interestingly, these transcriptional changes revert in the DRC state, suggesting a high degree of plasticity during the DTP phase. Furthermore, DTP cells have elevated levels of heterochromatin markers, H3K9me3, H3K27me3, and HP1α, along with their methyltransferases, G9a and Ezh2. Knockdown studies and inhibition of the enzyme activity of these modifiers showed suppression of DTP cell emergence. Valproic acid (VPA), a phase III candidate, was further assessed in vivo, where its sequential administration with cisplatin significantly reduced tumor burden versus cisplatin alone. These findings highlight the therapeutic promise of targeting epigenetic modifications to pre-sensitize cancer cells to chemotherapy, thereby restricting the survival advantage of DTP cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01966-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/649db90cb654c2e41c431aeb4e6b9016419e7a71" target='_blank'>
              The synergistic action of HDAC inhibitor with cisplatin impedes survival and proliferation of drug-tolerant persister in gastric and liver cancer cells
              </a>
            </td>
          <td>
            Anjali Singh, Abhiram Natu, Flevia Anthony, Hemalatha Muthu, B. Khade, Duane T. Smoot, Hassan Ashktorab, Sanjay Gupta
          </td>
          <td>2025-10-08</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5ee3b9a577273caee9d178991970953321a673" target='_blank'>
              Integrated Genomic Profiling Reveals Mechanisms of Broad Drug Resistance and Opportunities for Phenotypic Reprogramming in Cancer Cells
              </a>
            </td>
          <td>
            Ian Mersich, Brian S. J. Blagg, Aktar Ali
          </td>
          <td>2025-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Glioblastoma (GBM), the most common primary malignant brain tumor in adults, remains uniformly fatal due to the lack of effective targeted therapies. The epidermal growth factor receptor (EGFR) is the most frequently altered receptor tyrosine kinase oncogene in GBM with most alterations impacting the receptor ectodomain function, including gene amplification, mutation, rearrangement, and splicing site changes, which occur in approximately 50% of GBM tumors. Depatuxizumab mafodotin (Depatux-M; ABT-414), an antibody-drug conjugate composed of an EGFR-specific antibody (ABT-806) that recognizes the EGFR ectodomain linked to the cytotoxic agent monomethyl auristatin F, initially showed clinical promise. However, it failed to improve survival in phase III trials, highlighting an urgent need to understand mechanisms of resistance. Methods We generated in vivo ABT-414 resistant GBM models using patient-derived xenografts (PDXs) and performed genomics and transcriptomic profiling, including whole exome sequencing, bulk RNA sequencing, and single-cell RNA sequencing. Results ABT-414-resistant tumors exhibited transcriptional reprogramming characterized by upregulation of synaptic and developmental gene networks and downregulation of biosynthetic processes, indicative of a plastic, therapy-adaptive state. Whole-exome sequencing revealed novel mutations exclusive to resistant tumors, including a recurrent TEK (TIE2) S466I point mutation present in all ABT-414 resistant GBM12 PDX tumors. Functional validation demonstrated that ectopic expression of TEK S466I and TEK WT in PDX models reduced EGFR levels, suggesting a novel feedback mechanism linking TEK signaling to EGFR downregulation and contributes to resistance. Conclusion Our findings demonstrate that resistance to ABT-414 arises through both adaptive transcriptional remodeling and newly acquired genetic alterations. TEK-mediated suppression of EGFR represents a previously unrecognized mechanism of resistance, with potential implications for overcoming antibody-drug conjugate failure in GBM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07216-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2178575c5331764ee967eeeb61b931918d25454a" target='_blank'>
              Glioblastoma resistance to EGFR antibody-drug conjugate is driven by transcriptional reprogramming and TEK-induced EGFR suppression
              </a>
            </td>
          <td>
            M. Blomquist, T. Noviello, C. Sereduk, Dustin Grief, F. Caruso, Ritin Sharma, K. Garcia-Mansfield, P. Pirrotte, J. Kloss, F. D’Angelo, B. Marin, K. Porath, Sonia Jain, Benjamin Rabichow, S. Ensign, A. Iavarone, Antonella Paladino, J. Sarkaria, Joseph C Loftus, M. Ceccarelli, Nhan L Tran
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3565fa0fad40d6b64564b524def98cd2cdcfa949" target='_blank'>
              Metabolic plasticity drives mechanisms of PARP inhibitor persistence in ovarian cancer
              </a>
            </td>
          <td>
            Adriana Del Pino Herrera, Joon-Hyun Song, Karla Torres-Arciga, Susoma Halder, Kadin El Bakkouri, Mehdi Damaghi, Meghan C. Ferrall-Fairbanks
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Reprogramming often yields heterogeneous cell fates, yet the underlying mechanisms remain poorly understood. To address this, we developed single-cell transcription factor sequencing (scTF-seq), a single-cell technique that induces barcoded, doxycycline-inducible TF overexpression and quantifies TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells, scTF-seq generated a gain-of-function atlas for 384 mouse TFs, identifying key regulators of lineage specification, cell cycle control and their interplay. Leveraging single-cell resolution, we uncovered how TF dose shapes reprogramming heterogeneity, revealing both dose-dependent and stochastic cell state transitions. We classified TFs into low-capacity and high-capacity groups, with the latter further subdivided by dose sensitivity. Combinatorial scTF-seq demonstrated that TF interactions can shift from synergistic to antagonistic depending on the relative dose. Altogether, scTF-seq enables the dissection of TF function, dose and cell fate control, providing a high-resolution framework to understand and predict reprogramming outcomes, advancing gene regulation research and the design of cell engineering strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793bb57e9ee8be0a292f9fe822d8b2b3a9781a8c" target='_blank'>
              Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq
              </a>
            </td>
          <td>
            Wangjie Liu, Wouter Saelens, Pernille Y Rainer, Marjan Biočanin, V. Gardeux, Antoni J. Gralak, Guido van Mierlo, Angelika Gebhart, Julie Russeil, Tingdang Liu, Wanze Chen, Bart Deplancke
          </td>
          <td>2025-10-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e73a0dfd6d309117f8f8d19fdfdd72a4ff9c34bc" target='_blank'>
              Long-read single-cell genome, transcriptome and open chromatin profiling links genotype to phenotypes
              </a>
            </td>
          <td>
            Alexandra Pančíková, Ruben Cools, Marios Eftychiou, Margo Aertgeerts, Joris Vande Velde, Heidi Segers, Jan Cools, L. Harbers, J. Demeulemeester
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The cellular plasticity of neuroblastoma is defined by a mixture of two major cell states, adrenergic and mesenchymal, which may contribute to therapy resistance. However, how neuroblastoma cells switch cellular states during therapy remains largely unknown, and how to eradicate neuroblastoma regardless of its cell state is a clinical challenge. To better understand the cellular plasticity of neuroblastoma in chemoresistance, we define the transcriptomic and epigenetic map of adrenergic and mesenchymal types of neuroblastomas using human and murine models treated with indisulam, a selective RBM39 degrader. We show that cancer cells not only undergo a bidirectional switch between adrenergic and mesenchymal states, but also acquire additional cellular states, reminiscent of the developmental pliancy of neural crest cells. These cell state alterations are coupled with epigenetic reprogramming and dependency switching of cell state–specific transcription factors, epigenetic modifiers, and targetable kinases. Through targeting RNA splicing, indisulam induces an inflammatory tumor microenvironment and enhances the anticancer activity of natural killer cells. The combination of indisulam with anti-GD2 immunotherapy results in a durable, complete response in high-risk transgenic neuroblastoma models, providing an innovative, rational therapeutic approach to eradicate tumor cells regardless of their potential to switch cell states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8bfb0658591ed4d276e6df844046df62c0e809" target='_blank'>
              RBM39 degrader invigorates innate immunity to eradicate neuroblastoma despite cancer cell plasticity
              </a>
            </td>
          <td>
            Shivendra Singh, Jie Fang, Hongjian Jin, Lee-Ann Van de Velde, Andrew Cortes, Jiani Chen, Sivaraman Natarajan, E. Poon, Qiong Wu, Christopher L. Morton, Mary A. Woolard, Waise Quarni, Jacob A Steele, Jon P Connelly, Liusheng He, Rebecca Thorne, Gregory Turner, Thomas Confer, Melissa Johnson, W. Caufield, Burgess Freeman, Timothy Lockey, A. Murphy, P. J. Murray, Takashi Owa, S. Pruett-Miller, Ruoning Wang, L. Chesler, Julie R. Park, A. Davidoff, John Easton, Xiang Chen, Paul G. Thomas, Jun Yang
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b95fbb2531e397416e257c3c0be1df4a9857f2e4" target='_blank'>
              Integrated single-cell whole genome sequencing and spatial transcriptomics reveal latent intra-tumoral heterogeneity in ovarian cancer
              </a>
            </td>
          <td>
            Rania Bassiouni, Yuxin Jin, L. Gibbs, Jing Qian, Solomon O. Rotimi, Heather Miller, Michelle G. Webb, S. Rajpara, Javier Arias-Stella, David W Craig, Lynda D. Roman, John D Carpten
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b1867c4003902c154bf44ef7de285c3bf97bccb" target='_blank'>
              Selective mRNA translation determines adaptative mutability of melanoma cells to anti-BRAF/MEK combination therapy
              </a>
            </td>
          <td>
            Lucilla Fabbri, Lucie Lagadec, Eva Guérin, Hélène Lecourt, Dorothée Baille, Laetitia Besse, Cédric Messaoudi, Laurent Désaubry, Hussein Abou-Hamdan, Sévérine Roy, Bérangère Lombard, D. Loew, Jean Yves Scoazec, Caroline Robert, S. Vagner
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a hematologic malignancy characterized by the accumulation of myeloid blasts in the bone marrow. Despite the availability of potential curative treatments, patients frequently experience unfavorable outcomes. One crucial aspect contributing to relapse is the plasticity of leukemic clones, which enables them to switch between active proliferation and dormancy. The adaptability of AML underscores the need for novel therapies targeting AML-specific proteins. To address this, genome-wide CRISPR screens can be utilized to identify cancer entity-specific vulnerabilities. Leveraging publicly available functional genomics datasets and comparing AML with non-AML cancer cell lines, we identified a significant dependency on the cell cycle-regulating gene MYBL2 in AML. We describe MYBL2 as a key driver of AML cell growth and proliferation, highlighting its established role as a cell cycle regulator. Also, our findings uncover its previously unrecognized function as an inhibitor of cellular senescence. A knockdown of MYBL2 induces cell cycle arrest in the G2/M phase with subsequent induction of apoptosis in vitro, and reduces leukemic burden in a patient-derived xenograft (PDX) model in vivo. Interestingly, some AML cells evade apoptosis and enter a senescent-like phenotype upon MYBL2-knockdown, which is reversible upon re-expression of MYBL2. Finally, analyses of clinical data from two publicly available patient cohorts demonstrate a lower probability of survival in patients with higher MYBL2 expression, further hinting at the potential relevance of MYBL2 in AML. In conclusion, our findings demonstrate the essential role of MYBL2 in AML, governing the balance between cell proliferation, cell survival and senescence, ultimately influencing the fate of AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bab0ba7401319632cc10fcda395ad12a7cf902e" target='_blank'>
              Cell cycle regulator MYBL2 is a distinct vulnerability in acute myeloid leukemia
              </a>
            </td>
          <td>
            Sandra Küchler, Silke Brilloff, Silvia Schäfer, Elahe Rahimian, Vida Kufrin, Shraddha S Peri, J. Musa, T. Grünewald, Denis M Schewe, C. Ball, M. Bornhäuser, H. Glimm, Marius Bill, A. A. Wurm
          </td>
          <td>2025-10-20</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Doxorubicin (DOX) is a widely used chemotherapeutic agent, but its efficacy is often limited by cancer cell resistance. Although multiple DOX resistance mechanisms have been characterized, the global transcriptomic alterations underlying this phenomenon remain poorly understood. The aim of this work was to determine whether a common transcriptional response associated with DOX desensitization exists across tumor cells of different origins and to identify the core elements of this response. Methods: We performed an integrated bioinformatics analysis, including: analysis of independent transcriptomic datasets (comparing DOX-resistant neuroblastoma, breast, and cervical carcinoma cells to their DOX-sensitive counterparts), functional annotation of differentially expressed genes, reconstruction and topology analysis of gene networks, text mining, and survival analysis. The findings were validated through in vitro functional tests, RT-PCR, and analysis of the Cancer Therapeutics Response Portal and The Cancer Genome Atlas. Results: We showed that DOX resistance in cancer cells is associated with cytoskeletal reorganization, modulation of cell adhesion, cholesterol biosynthesis, and dysregulation of mTORC1, Wnt, and Gβγ signaling pathways. Network analysis identified a conserved regulome of 37 resistance-linked genes, with GJA1, SEH1L, TCF3, TUBA4A, and ZYX emerging as central hubs (mean degree: 8.7–19.7; mean fold change: 2.4–21.3). Experimental validation in DOX-resistant KB-8-5 cervical carcinoma cells and their sensitive counterparts (KB-3-1) confirmed enhanced cellular adhesion and reduced intracellular cholesterol levels associated with chemoresistance, supporting our in silico findings. A detailed follow-up analysis verified the upregulation of these hub genes in chemoresistant cells and their correlation with poor clinical outcomes across multiple cancer types. Conclusions: This integrative analysis identifies conserved transcriptomic signatures of DOX resistance and highlights hub genes GJA1, SEH1L, TCF3, TUBA4A, and ZYX with potential as predictive biomarkers and therapeutic targets. Targeting these pathways may help overcome chemoresistance and improve treatment outcomes in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2b259a82bfdbeb3c8cb64a5419e6a90ac8b1c5" target='_blank'>
              Novel Cross-Cancer Hub Genes in Doxorubicin Resistance Identified by Transcriptional Mapping
              </a>
            </td>
          <td>
            A. D. Moralev, O. Markov, M. Zenkova, A. Markov
          </td>
          <td>2025-10-16</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e72f57d35ee3dac82999a8b9759c889a0508af" target='_blank'>
              Genetic and transcriptomic alterations underlying aggressiveness in 5-fluorouracil-resistant HCT116 cells.
              </a>
            </td>
          <td>
            Pornchai Sooksaen, A. Thim-uam, Ratsada Praphasawat, Chutamas Thepmalee, Kanokkarn Phromnoi, Damratsamon Surangkul, Surisak Prasarnpun, Chakkraphong Khonthun
          </td>
          <td>2025-10-10</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acd8a87b52ce435726d0713cb35d9987d7862c73" target='_blank'>
              The two sides of resistance: aggressiveness and mitotic instability as the Achilles’ heel of Osimertinib-resistant NSCLC
              </a>
            </td>
          <td>
            P. Armanetti, E. Cabrera San Millán, R. Mercatelli, Chiara Cavallini, Alice Chiodi, E. Mosca, Arianna Consiglio, Nicola Losito, Luca Menichetti, E. Bramanti, Azhar Ali, Elena Levantini, Giorgia Maroni
          </td>
          <td>2025-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer drug resistance remains a major barrier to durable treatment success, often leading to relapse despite advances in precision oncology. While combination therapies are being increasingly investigated, such as chemotherapy with small molecule inhibitors, predicting drug response and identifying rational drug combinations based on resistance mechanisms remain major challenges. Therefore, a proteome-wide, single-gene overexpression screening platform is essential for guiding rational therapy selection. Here, we present BOGO (Bxb1-landing pad human ORFeome-integrated system for a proteome-wide Gene Overexpression), a robust, scalable, and reproducible screening platform that enables single-copy, site-specific integration and overexpression of ∼19,000 human ORFs across cancer cell models. Using BOGO, we identified drug-specific response drivers for 16 chemotherapeutic agents and integrated clinical datasets to uncover proliferation and resistance-associated genes with prognostic potential. Drug response similarity networks revealed both shared and unique mechanisms, highlighting key pathways such as autophagy, apoptosis, and Wnt signaling, and notable resistance-associated genes including BCL2, POLD2, and TRADD. In particular, we proposed a synergistic combination of the BCL2 family inhibitor ABT-263 (Navitoclax®) and the DNA analog TAS-102 (Lonsurf®), which revealed that lysosomal modulation is a key mechanism driving DNA analog resistance. This combination therapy selectively enhanced cytotoxicity in colorec tal and pancreatic cancer cells in vitro, and demonstrated therapeutic benefit in vivo in both cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models. Together, these findings establish BOGO as a powerful gene overexpression perturbation platform for systematically identifying chemoresistance and chemosensitization drivers, and for discovering rational combination therapies. Its scalability and reproducibility position BOGO as a broadly applicable tool for functional genomics and therapeutic discovery beyond cancer resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb42d8c647bdd3aef172a4ebd8dbd580fe7fe538" target='_blank'>
              BOGO: A Proteome-Wide Gene Overexpression Platform for Discovering Rational Cancer Combination Therapies
              </a>
            </td>
          <td>
            Kyeong Beom Jo, M. Alruwaili, Da-Eun Kim, Yongjun Koh, Hyeyeon Kim, Kwontae You, Ji-Sun Kim, Saba Sane, Yanqi Guo, Jacob P. Wright, H. Lim, Maricris N. Naranjo, Atina G. Coté, Frederick P. Roth, David E. Hill, Jung-Hyun Choi, Hunsang Lee, Kenneth A. Matreyek, K. Farh, Jong-Eun Park, Hyunkyung Kim, Andrei V. Bakin, Dae-Kyum Kim
          </td>
          <td>2025-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Based on our previous study, we have identified ZNF180, a zinc finger protein, as a pro-tumorigenic regulator in primary melanoma and a marker for poor prognosis. Herein, we report that ZNF180-regulated pathway, hence ZNF180-regulome, underlies resistance towards immune checkpoint inhibitions (ICIs). Methods To investigate regulatory roles of ZNF180 to confer these immune suppressive phenotypes, we performed ZNF180 knock-down in melanoma cells in vitro with different genetic backgrounds, namely A375 (BRAF-mutant) and SKMEL147 (NRAS-mutant) cells, and performed RNA- and ATAC-sequencing. We performed integrative analysis of RNA- and ATAC-sequencing data with publicly available sequencing data from ICI-treated cohorts to construct comprehensive model of ZNF180-regulome and its impacts on immune microenvironment. Further, we performed ZNF180 silencing in immune competent Yumm1.7 murine model to confirm the changes in immune microenvironments. Results ZNF180-regulome was predictive of ICI responses in independent bulk sequencing cohorts, and ZNF180+ tumors persisted after the therapy with immune-suppressive features such as MHC-I loss and CD155 expressions, the primary ligand to TIGIT inhibitory receptor. Further, ZNF180 silencing revealed its regulations on AP-1 transcription factors to drive melanoma reprogramming towards de-differentiated MITFlowAXLhigh cells, an established melanoma subtypes associated with recurrence and ICI resistance. In tandem, we observed that ZNF180+ tumor neighborhood significantly excluded with CD4 T-cells in metastatic tumor, and its silencing in immune competent murine model increased CD4 helper T-cell infiltrations with significant tumor regression in vivo. Conclusion Collectively, these results indicate ZNF180 is a tumor intrinsic regulator of melanoma plasticity to drive de-differentiated phenotypes with immune-suppressive features including loss of immunogenicity, T-cell inhibitory signals through TIGIT/CD155 checkpoint and exclusion of CD4 helper T-cells. As ZNF180-regulome manifests in non-metastatic melanoma in contrast to the current focus of standard-of-care ICI on the metastatic disease, these results establish ZNF180-regulome as a biomarker and novel therapeutic avenue for early-stage, non-metastatic melanoma to intervene ICI resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31baf3f08584db6bc1b15c0af412ba0fb9282ea1" target='_blank'>
              ZNF180 modulates tumor intrinsic immunotherapy resistance in melanoma through driving plasticity
              </a>
            </td>
          <td>
            Won-Min Song, Sree Karani Kondapuram, Xianxiao Zhou, Shu-hsia Chen, Praveen Agrawal
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="This study explores the molecular impact of cisplatin, Actemra (tocilizumab), remdesivir, and SARS-CoV-2 infection on colorectal cancer (CRC) SW-480 cells using RNA sequencing (RNA-Seq). Differential gene expression analysis revealed treatment-specific transcriptional changes. Cisplatin led to the downregulation of genes involved in lipid metabolism and focal adhesion, while remdesivir upregulated chromatin remodeling pathways. SARS-CoV-2 infection altered cytokine signaling, particularly IL-17 and TNF-α. Principal component and hierarchical clustering analyses confirmed distinct gene expression profiles across treatments. ELISA assays validated the reduction of ACE2 and CD147 protein levels following drug treatment. MTT assays demonstrated remdesivir’s cytotoxicity at high doses. These findings highlight potential drug-gene interactions and suggest that combining transcriptomic profiling with functional assays may guide personalized therapeutic strategies for CRC patients, especially those co-infected with SARS-CoV-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5eae066da47093f217085c9697d40cc942624f2a" target='_blank'>
              RNA sequencing reveals transcriptional signatures of drug response and SARS-CoV-2 interaction in colorectal cancer cells
              </a>
            </td>
          <td>
            Alaa K. Hameed, Ifad K. Abd Al-Shibly, Rana A. Ghaleb
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hypoxia is a well-recognized clinical feature of solid tumors, including lung cancer, and is associated with poor prognosis due to its role in promoting resistance to chemotherapy and radiotherapy. To investigate the cellular consequences of hypoxia, we cultured A549 lung adenocarcinoma cells under 1% O2 and examined their growth, cell cycle distribution, and redox status. Hypoxia significantly reduces cell proliferation and induced G1 phase cell cycle arrest, suggesting a cytostatic effect. Biochemical analysis showed a 2.64- and 2.04-fold increase in total and mitochondrial reactive oxygen species (ROS) levels, respectively, along with an elevated total thiol levels under hypoxic conditions compared to normoxia. To assess the reversibility of the hypoxic response, we performed a reciprocal oxygen exposure experiment where cells initially grown under hypoxia were re-exposed to normoxia, and vice versa. To explore the underlying molecular mechanism, we analyzed transcriptomic datasets (GEO accession: GSE48134 and GSE42416) which revealed that hypoxia downregulated key genes involved in energy metabolism (e.g., PDK4, G6PD), cell cycle progression (e.g., CCND1, CDK2), and redox regulation (e.g., GCLM, TXNRD1, NQO1, GCLC). Further, few of redox-related genes were validated by RT-PCR in A549 cells cultured under hypoxia and normoxia for 24 h. Importantly, cyclic hypoxia (intermittent hypoxia-reoxygenation) conditions showed partial restoration of some of these transcripts, supporting the transient nature of hypoxic stress, consistent with our in vitro observations. Furthermore, transcriptome profiles from adenocarcinoma patients (GEO accession: GSE30979) also match our cell line observations. Thus, our results clearly show that hypoxia causes a temporary cell cycle arrest in lung cancer cells, which is reversible when oxygen is restored. Supplementary Information The online version contains supplementary material available at 10.1007/s12032-025-03058-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e77bbcdb23f2ddac965c8e06176945c4fad76ded" target='_blank'>
              Hypoxia-drives reversible cell cycle arrest in lung cancer cells via modulation of cellular redox and gene expression
              </a>
            </td>
          <td>
            D. Maurya, Varshita Mehta, Babita Singh
          </td>
          <td>2025-09-29</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance limits the therapeutic efficacy of osimertinib in lung adenocarcinoma (LUAD). Redox homeostasis is crucial for LUAD progression. However, how redox imbalance interacts with the tumor microenvironment (TME) to drive osimertinib resistance (OR) remains unclear.


METHODS
The single-cell RNA sequencing (scRNA-seq) data GSE243562 were combined with the Cancer Genome Atlas (TCGA)-LUAD transcriptomes to map the TME cell population heterogeneity in osimertinib-resistant LUAD. Through univariate Cox regression and least absolute shrinkage and selection operator (LASSO) regularization, a prognostic signature founded on redox-related genes (RRGs) was built. Therapeutic compounds targeting these signature genes were prioritized by molecular docking. Their expression patterns were subsequently validated in vitro.


RESULTS
Cancer-associated fibroblasts (CAFs) were central hubs in the TME of osimertinib-resistant LUAD, exhibiting enhanced intercellular communication. Computational profiling identified 10 differentially expressed RRGs, predominantly enriched in CAFs. Using a six-gene signature comprising AGER, CYP2J2, FMO2, HSPA1B, SOD3, and VASN, we categorized LUAD patients into separate risk categories. High-risk patients showed significantly reduced survival, an immunosuppressive status, and a higher tumor mutation burden (p < 0.05). The overexpression of these six genes was confirmed in OR cells. Critically, inhibiting SOD3 restored osimertinib sensitivity in vitro (p < 0.05). Clinically, SOD3 expression was lower in patients sensitive to third-generation epidermal growth factor receptor-tyrosine kinase inhibitors (EGFR-TKIs) than in those with resistant disease.


CONCLUSIONS
Targeting CAFs represents a promising strategy to overcome osimertinib resistance. Our six-gene redox signature offers a clinical framework for patient risk stratification and novel therapeutic strategy design. Future work will explore these targets to develop new treatments for LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1eb1b2e535167099275ea725fe5cce81f5d8d82" target='_blank'>
              Integrated Analysis of Single-cell and Bulk-RNA Sequencing Data to Identify Redox-related Genes as Prognostic Biomarkers and Therapeutic Targets of Lung Adenocarcinoma With Osimertinib Resistance.
              </a>
            </td>
          <td>
            Chen Fang, Chaoxing Liu, Rong Qi, Jiabin Ding, Ting Luo, Feng Yu, Guohua Zhang, Chao Shi, Daya Luo, Feng Qiu
          </td>
          <td>2025-09-25</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a501c277b7efc44209952f58d1318feb42e211" target='_blank'>
              YAP1 drives aggressive and therapy resistant state in melanoma through reprogramming the chromatin and regulating immune evasive programs
              </a>
            </td>
          <td>
            Nana Chen, Gracia M. Bonilla, Abdullah AI Emran, Brian Lin, Haymanti Bhanot, Yang Sun, Avinash Waghray, Mateo Useche, David Liu, Genevieve M Boland, Ivana L de la Serna, V. Sancisi, Martin Sattler, Jayaraj Rajagopal, Xu Wu, Ruslan Sadreyev, David E Fisher, S. Saladi
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract BRCA2 mutations contribute to the pathogenesis and treatment sensitivity of a subset of ovarian, breast, prostate, and pancreatic cancers. When these cancers become therapy resistant, secondary mutations that restore the BRCA2 open reading frame are found in half the cases, but other causes of resistance remain incompletely understood. Here, we identified translational readthrough of a premature termination codon (PTC) as a cause of resistance to poly(ADP-ribose) polymerase inhibitors (PARPis) and cisplatin in cells derived from the BRCA2-mutated ovarian cancer line PEO1 by PARPi selection. Despite persistence of the signature 4965C > G (p.Y1655X) BRCA2 mutation, low-level expression of full-length BRCA2 protein was detectable in these cells by immunoblotting and tandem mass spectrometry. Either BRCA2 knockdown or gene interruption 5′ or 3′ to the PTC restored treatment sensitivity, implicating BRCA2 in the resistance. Reporter assays demonstrated UAG-selective readthrough in the resistant clones but not parental cells. Moreover, custom searching of global proteomic data indicated readthrough of stop codons, particularly UAGs, in additional proteins in the resistant clones. Finally, multi-omic analysis identified multiple changes in the nonsense-mediated decay and termination machineries that favor readthrough. Accordingly, the present results identify PTC readthrough as a potential mechanism of drug resistance in cells with BRCA2 nonsense mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5f812b097b8ef89a4b208e38861c4246f71092" target='_blank'>
              Codon specific readthrough as a mechanism of BRCA2 restoration in acquired PARP inhibitor and chemotherapy resistance
              </a>
            </td>
          <td>
            Cordelia McGehee, X Wei Meng, Xinyan Wu, Cristina Correia, A. Venkatachalam, K. Flatten, Kevin L. Peterson, , Olivia K. Rossman, Clara Wong, R. Hurley, Jill M. Wagner, Paula A. Schneider, Haiming Dai, Guy G Poirier, Andrea E. Wahner Hendrickson, Elizabeth M. Swisher, Akhilesh Pandey, Hu Li, Scott H. Kaufmann
          </td>
          <td>2025-10-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bd95123f481a9cef05baf3ace1238c5ad7fb915" target='_blank'>
              A telomere-to-telomere map of somatic mutation burden and functional impact in cancer
              </a>
            </td>
          <td>
            Min-Hwan Sohn, Danilo Dubocanin, Mitchell R. Vollger, Youngjun Kwon, Anna Minkina, Katherine M. Munson, Samuel FM Hart, Jane E. Ranchalis, Nancy L. Parmalee, Adriana E. Sedeño-Cortés, Jeffrey Ou, N. Y. Au, Stephanie Bohaczuk, Brianne Carroll, C. D. Frazar, William T. Harvey, Kendra Hoekzema, Meng-Fan Huang, Caitlin N Jacques, Dana M. Jensen, J. T. Kolar, R. Lee, Jiadong Lin, Kelsey Loy, Taralynn Mack, Yizi Mao, Meranda M. Pham, E. Ryke, Josh Smith, Lila M. Sutherlin, Elliott G Swanson, Jeffrey M. Weiss, SMaHT Assembly, C. Carvalho, Tim H. H. Coorens, Kelley Harris, Chia-Lin Wei, E. Eichler, Nicolas Altemose, James T. Bennett, Andrew B. Stergachis
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Treatments for high-grade serous ovarian cancer (HGSOC) are initially effective but most invariably fail. Although they can successfully suppress the bulk of the tumour cell population, residual cancer cells can enter alternative therapy-resistant cell fates highlighted by proliferative arrest. Understanding the nature of these fates and how cells may resume uncontrolled proliferation will lead to the development of new treatments for HGSOC. In this study, we examine the response of HGSOC cells to standard of care cisplatin chemotherapy and to the RNA Polymerase I transcription inhibitor CX-5461/Pidnarulex, two drugs that elicit a potent DNA damage response and growth arrest. Here, we identify that HGSOC cells exposed to these therapies show multiple hallmarks of therapy-induced senescence (TIS) and derive a core TIS gene expression signature irrespective of genetic background or senescence trigger. Given that TIS is a potentially escapable state, we have performed a focussed drug screen to identify drugs that eradicate senescent HGSOC cells. We identify that therapy-induced senescent HGSOC cells, including those with decreased sensitivity to senolytic drugs that inhibit the pro-survival protein BCL-XL, can be eliminated using drugs that induce ferroptosis, an iron-dependent form of cell death. Mechanistically, we demonstrate that senescent HGSOC cells have altered expression of regulators of iron metabolism leading to intracellular iron overload that underpins this targetable vulnerability. Together, we highlight elevated levels of iron as a TIS biomarker in HGSOC and the potential of inducing ferroptosis to eradicate residual HGSOC cells following initial therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3787e084ffe3e100d6ec83768df6b1bb600d9e4a" target='_blank'>
              Exploiting dysregulated iron homeostasis to eradicate persistent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Carmelo Cerra, Madeleine R. C. Tancock, N. Thio, Ada Koo, AnnRann Wong, Karla J Cowley, Swati Varshney, Madelynne O. Willis, Kaylene Simpson, D. Bowtell, E. Sanij, Elizabeth L Christie, Richard B. Pearson, Jian Kang, Keefe T. Chan
          </td>
          <td>2025-09-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43d57534bc0d9d0ab660dd7d7cddbf910c13f4aa" target='_blank'>
              Strain-resolved CRISPRi-Seq reveals conserved antibiotic vulnerabilities in Staphylococcus aureus
              </a>
            </td>
          <td>
            Maria-Vittoria Mazzuoli, Jessica Burnier, Albane Schmid, Louise Martin, Axel B. Janssen, Clément Gallay, A. Zinkernagel, J. Veening
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43797c7834d20e5589904d0168347625fb0cad6e" target='_blank'>
              Unraveling cellular dynamic changes in tumor evolution induced by long-term low dose-rate radiation.
              </a>
            </td>
          <td>
            Wanshi Li, Weiwei Pei, Yiwei Wang, Jing Nie, Guang Hu, Yongsheng Zhang, Zhifei Cao, Zaozao Chen, Keyan Miao, Yuqi Chen, Likai Shi, Yingchu Dai, , Hailong Pei, C. Ye, Yasser F Ali, Wentao Hu, Wenying Yan, Guangming Zhou
          </td>
          <td>2025-09-05</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3002080effb079c5d4239d8ceb95fa35b9194a90" target='_blank'>
              Phenotypic plasticity including drug efflux drives reversible irinotecan resistance in LIM1215 colorectal cancer cells.
              </a>
            </td>
          <td>
            Tugce Di̇lber, Gizem Damla Yalcin, Kübra Yılmaz, Ahmet Acar
          </td>
          <td>2025-10-28</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cells frequently reprogramme their metabolism to support growth and survival, making metabolic pathways attractive targets for therapy. In this study, we investigated the metabolic effects of three kinase inhibitors and their synergistic combinations in the gastric cancer cell line AGS using genome-scale metabolic models and transcriptomic profiling. We applied the tasks inferred from the differential expression (TIDE) algorithm to infer pathway activity changes in the different conditions. We also explored a variant of TIDE that uses task-essential genes to infer metabolic task changes, providing a complementary perspective to the original algorithm. Our results revealed widespread down-regulation of biosynthetic pathways, particularly in amino acid and nucleotide metabolism. Combinatorial treatments induced condition-specific metabolic alterations, including strong synergistic effects in the PI3Ki–MEKi condition affecting ornithine and polyamine biosynthesis. These metabolic shifts provide insight into drug synergy mechanisms and highlight potential therapeutic vulnerabilities. To support reproducibility, we developed an open-source Python package, MTEApy, implementing both TIDE frameworks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3017a5eb28e4bc7906f6c3abd507b3d2f862ad" target='_blank'>
              Constraint based modeling of drug induced metabolic changes in a cancer cell line
              </a>
            </td>
          <td>
            Xavier Benedicto, Åsmund Flobak, M. Ponce-de-León, Alfonso Valencia
          </td>
          <td>2025-10-06</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aim: Urinary bladder cancer (UBC) often develops chemoresistance, reducing treatment effectiveness. This study aimed to investigate diverse molecular mechanisms underlying acquired resistance by establishing and characterizing a comprehensive panel of UBC cell lines resistant to common chemotherapeutics. Methods: Fifteen UBC cell lines were examined: three parental lines (RT-112, TCC-SUP, UMUC-3) and twelve derived sublines adapted to cisplatin, vinblastine, or gemcitabine. Drug sensitivity was assessed using the SRB assay. Resistance mechanisms were explored via quantitative real-time PCR (targeting genes including ABCB1, dCK, hENT1, ECHDC1, TUBB3), Western blotting (assessing proteins such as p21, Cyclin B, and Mcl-1), and biochemical assessment of glutathione levels and redox state. Results: The adapted sublines exhibited distinct resistance profiles and cross-resistance patterns. Gene expression and protein analyses revealed drug- and lineage-specific alterations, involving factors such as p21, Cyclin B, and Mcl-1. Changes in glutathione metabolism were also associated with resistance. Notably, no single, universal mechanism accounted for resistance across the entire panel. Conclusion: UBC cells develop diverse, context-dependent adaptive strategies to resist cisplatin, vinblastine, and gemcitabine. These findings highlight the complexity of chemoresistance mechanisms. The characterized cell line panel represents a valuable resource for future studies aimed at understanding and overcoming drug resistance in bladder cancer, suggesting that personalized therapeutic approaches may be necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fbd01421411fd7eb4380d44e0e070090cf7648" target='_blank'>
              Distinct adaptive strategies to cisplatin, vinblastine and gemcitabine in a panel of chemoresistant bladder cancer cell lines
              </a>
            </td>
          <td>
            Monika Cuprych-Belter, Agnieszka Łupicka-Słowik, Artur Anisiewicz, Martin Michaelis, J. Cinatl jr., Mateusz Psurski
          </td>
          <td>2025-09-12</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5c397ac071eb956e6c0c3d19f024d33e098a95" target='_blank'>
              The Breakome of BRCA1 and BRCA2 Pathway Mutation Carriers Reveals Early Processes in Breast Oncogenesis
              </a>
            </td>
          <td>
            Sara Oster Flayshman, Osama Hidmi, Jackelyn A. Alva-Ornelas, Jonathan Monin, Mark A. LaBarge, Victoria L Seewaldt, Y. Drier, Rami I. Aqeilan
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="KRAS mutations represent the most common driver genetic alterations in multiple malignant tumors, particularly exerting well-defined oncogenic driver roles in non-small cell lung cancer, colorectal cancer, and pancreatic ductal adenocarcinoma. The recent success of KRAS G12C mutation-specific allosteric inhibitors marks a therapeutic milestone in KRAS-targeted oncology. However, the rapid emergence of drug resistance in clinical applications has significantly limited the durable efficacy of these agents. The resistance mechanisms exhibit profound complexity, encompassing multidimensional pathways such as secondary/co-occurring mutations, compensatory reprogramming of signaling pathways, cellular lineage plasticity, and immune evasion within the tumor microenvironment. Studies have demonstrated that the identification of resistance-associated biomarkers is not only of critical clinical value for predicting treatment response and early warning of resistance but also provides a key entry point for dissecting resistance mechanisms. This review systematically summarizes the latest research advances in KRAS-mutant inhibitor resistance biomarkers, with a focus on the analysis of resistance molecular mechanisms, discovery of predictive biomarkers, current limitations and challenges, and exploration of biomarker-based combination therapy strategies. By integrating basic research and clinical data, we highlight the breakthroughs required to realize the application value of resistance biomarkers and prospects future research directions and priorities, aiming to provide a biomarker-related theoretical framework for precision therapy of KRAS-mutant tumors and accelerate the clinical translation of resistance-overcoming strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dedd636638a7ce054ecf1cb2befdf53d765c7e0b" target='_blank'>
              Advances in biomarkers of resistance to KRAS mutation-targeted inhibitors
              </a>
            </td>
          <td>
            Lihao Wang, Dandan Wei, Shanshan Li, Shiqing Jiang
          </td>
          <td>2025-10-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype; however, clinically approved prognostic biomarkers and therapeutic options remain limited. This study aimed to investigate tumor dependency genes to identify novel therapeutic targets for TNBC.Tumor dependency genes for TNBC were identified using the The Cancer Dependency Map (DEPMAP) database. The TCGA-BRCA dataset was utilized to analyze the expression, survival associations, and pathway enrichment of these genes. Single-cell datasets were employed to explore cellular trajectories and biological functions within tumor dependency gene-associated cell subpopulations. Genomic sequencing was used to investigate the somatic mutational landscape influencing the infiltration abundance of the tumor dependency-associated subpopulation. The METABRIC dataset assessed the impact of the tumor dependency-associated subpopulation on radiotherapy, chemotherapy, and combination therapy outcomes. Potential drugs were identified using the Connectivity Map (CMAP). Colony formation experiment and the CCK-8 experiment were performed to validate the biological function of gene.Four tumor dependency genes (TDGs) were identified. These genes were highly expressed in TNBC and associated with poor prognosis. Enrichment analysis revealed their significant involvement in cell cycle-related pathways. Single-cell analysis demonstrated that the tumor dependency-associated subpopulation (TDAS), defined by these four genes, resided at the differentiation terminus of epithelial/tumor cells and was linked to energy metabolism and cell proliferation pathways. Crucially, patients with high TDAS infiltration abundance were found to be unsuitable for surgery alone and should receive combined radiotherapy or chemotherapy. Potential therapeutic agents targeting the TDAS were screened. And in vitro experiments confirmed the cell proliferation role of candidate genes.This study identifies four potential TNBC biomarkers for assessing TDAS abundance, providing novel insights and strategies for personalized TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04c8aa1f75b0d7e0c49aabd5f19b6bf7360c1cf" target='_blank'>
              Integrating single-cell transcriptomics and whole-genome CRISPR CAS9 screen identifies a cell cluster associated with tumor dependency in triple-negative breast cancer
              </a>
            </td>
          <td>
            Su Liu, Shuo Wang, Guixin Wang, Yingxi Li, Zhigang Zhao, Yao Tian, Jun Cao
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, T.-C. Chang, Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Antimycin A, an antifungal agent that inhibits mitochondrial respiration, provides a useful model for studying resistance mechanisms. Antifungal resistance is an escalating clinical concern with limited treatment options available. To understand the molecular mechanisms of antimycin A-resistance, a genetically stable, antimycin A-resistant Saccharomyces cerevisiae strain was successfully developed for the first time through an evolutionary engineering strategy, based on long-term systematic application of gradually increasing antimycin A-stress in repetitive batch cultures without prior chemical mutagenesis. Comparative whole genome re-sequencing analysis of the evolved strain ant905-9 revealed two missense mutations in PDR1 and PRP8 genes involved in pleiotropic drug resistance and RNA splicing, respectively. Using CRISPR/Cas9 genome editing tools, the identified mutations were introduced individually and together into the reference strain, and it was confirmed that the Pdr1p.M732R mutation alone confers antimycin A-resistance in S. cerevisiae. Comparative transcriptomic analysis of the reverse engineered Pdr1p.M732R strain showed alterations in PDR (pleiotropic drug response), transmembrane transport, vesicular trafficking and autophagy pathways. Our results highlight the potential key role of PDR1 in antifungal drug resistance. This study provides new insights into mitochondrial drug resistance and the adaptive potential of yeast under respiratory stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b896127a479ec689ce1d1e9d511ec8bcaa5efae" target='_blank'>
              Evolutionary Engineering and Molecular Characterization of an Antimycin A-Resistant Saccharomyces cerevisiae Strain: the Key Role of Pleiotropic Drug Resistance (PDR1).
              </a>
            </td>
          <td>
            A. Topaloğlu, C. Holyavkin, Ö. Esen, O. Morkoç, K. Persson, C. Geijer, Z. P. Çakar
          </td>
          <td>2025-10-17</td>
          <td>FEMS yeast research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) holds promise for animal cloning but remains limited by low efficiency and phenotypic abnormalities, often attributed to incomplete nuclear reprogramming. This study presents an integrative genomic and epigenomic analysis of cloned buffaloes and their respective donors using long-read Oxford Nanopore sequencing. Our results showed a high degree of genomic similarity between clones and donors, with most variations located in non-coding regions and structural variants (SV) distributions highly correlated at the chromosomal level. Gene and protein level overlap of SV-affected loci revealed 70.9–73.3% gene-level and 69.7–72.5% protein-level similarity. Despite this genetic similarity, DNA methylation analysis identified differentially methylated regions (DMRs), particularly in intergenic and promoter regions. Clones exhibited slightly lower CpG methylation than the donors. The DMRs in donor vs. clone comparisons indicated higher hypomethylated regions than hypermethylated regions. Functional enrichment of DMR-associated genes highlighted pathways linked to mitochondrial function, oxidative phosphorylation, and reproductive processes. Although clones showed moderate genome-wide methylation correlation with donors, key differences in methylation suggest incomplete epigenetic reprogramming. Despite these epigenetic differences, all clones were phenotypically normal and healthy into adulthood. This study offers the first comprehensive SV and methylome profile of SCNT-derived buffaloes and emphasizes the role of epigenetic mechanisms in clone development and health, providing valuable insights to enhance cloning efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed0c9146dce33964d38cc27a7e6aa2903443aa1" target='_blank'>
              Unravelling High Nuclear Genomic Similarity and Mitochondria Linked Epigenetic Divergence in SCNT Derived Buffalo Clones via Long-Read Nanopore Genome Sequencing
              </a>
            </td>
          <td>
            Meeti Punetha, Dharmendra Kumar, Satish Kumar, Bhavya Maggo, Priya Dahiya, Pradeep Kumar, Rakesh K. Sharma, Yash Pal, P. S. Yadav
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Advanced sarcomas have a poor prognosis and limited therapeutic options. Disease recurrence is caused by persistent cells that survive drug treatments. The alkylating agent trabectedin, when combined with the poly (ADP-ribose) polymerase 1 (PARP1) inhibitor olaparib, exhibits variable antitumor effects in advanced sarcomas. In this study, we demonstrate that the expression of the transcription factor OCT4 is upregulated in persistent cells that survive treatment with trabectedin and olaparib, through the cGAS-STING-IRF3-IFNβ pathway. This route also leads to the upregulation of natural killer (NK) and cytokine-induced killer (CIK) lymphocyte activating ligands. These molecular events enhance the antitumor efficacy of immunotherapy with NK and CIK cells, targeting both the bulk population and residual drug-tolerant cells. In conclusion, the activation of the cGAS-STING pathway has a double-edged effect, enriching the OCT4+ persistent cell population while increasing the expression of NK/CIK ligands. The addition of olaparib to trabectedin potentiates the cGAS-STING pathway activation and the upregulation of NKG2DLs, while simultaneously counteracting the OCT4 overexpression. Therefore, sequential treatment with trabectedin and olaparib followed by NK/CIK immunotherapy represents a promising strategy against advanced sarcomas and warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506de56eb87cad4e65432310520f88b9e3f761bd" target='_blank'>
              The cGAS-STING pathway is a master regulator of OCT4 expression in persistent sarcoma cells and enhances cellular immunotherapy with NK and CIK lymphocytes
              </a>
            </td>
          <td>
            Giorgia Giordano, A. Merlini, F. Capozzi, Giulio Ferrero, Cristina Tucciarello, Shahab Majidi, Simona Comparato, G. Mesiano, Elisabetta Liuzzi, F. Galvagno, Annamaria Massa, Silvia Brusco, V. Leuci, Chiara Riganti, S. Cutrupi, Michele De Bortoli, Lorenzo D’Ambrosio, Dario Sangiolo, G. Grignani, Y. Pignochino
          </td>
          <td>2025-09-23</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e5023d83ec2a5ceab2bac53b66c2b863c5e8ad9" target='_blank'>
              Non-centromeric CENP-A epigenetically regulates epithelial-mesenchymal plasticity and heterogeneity in human cells
              </a>
            </td>
          <td>
            Charlène Renaud-Pageot, Daniele Capocefalo, Sébastien Lemaire, Audrey Forest, Laura Cantini, G. Almouzni
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Chemotherapy response variability in cancer patients necessitates novel strategies targeting chemoresistant populations. While combinatorial regimens show promise through synergistic pharmacological interactions, traditional pathway enrichment methods relying on static gene sets fail to capture drug-induced dynamic transcriptional perturbations. Methods To address this challenge, we developed the Pathway-Responsive Gene Sets (PRGS) framework to systematically identify chemoresistance-associated pathways and guide therapeutic intervention. Comparative evaluation of three computational strategies (GSEA-like method, Hypergeometric test-based method, Bates test-based method) revealed that the GSEA-like methodology exhibited superior performance, enabling precise identification of drug-induced pathway dysregulation. Results Key experimental findings demonstrated PRGS’s superiority over conventional Pathway Member Gene Sets (PMGS), exhibiting statistical independence (p < 0.0001) and enhanced detection of chemotherapy-driven pathway dysregulation. Application of PRGS to the GDSC dataset identified 8 resistance-associated pathways. Screening of agents targeting these pathways yielded candidates with predicted anti-resistance activity. An in vitro cellular experiment demonstrated that the bortezomib-bleomycin combination exhibited synergistic cytotoxicity (IDAcomboScore = 0.014) in T47D cells, highlighting the potential of PRGS-guided therapeutic strategies. Discussion This study establishes a PRGS-based methodological framework that integrates genomic perturbations with precision oncology, demonstrating its capacity to decode resistance mechanisms and guide therapeutic development through dynamic pathway analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/814bf51ab320b8d56ddbcfefe1cd9a9aa4f449e3" target='_blank'>
              Drug discovery for chemotherapeutic resistance based on pathway-responsive gene sets and its application in breast cancer
              </a>
            </td>
          <td>
            Dehua Feng, Jingwen Hao, Lingxu Li, Jian Chen, Xinying Liu, Ruijie Zhang, Huirui Han, Tianyi Li, Xuefeng Wang, Xia Li, Lei Yu, Bing Li, Jin Li, Limei Wang
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9628d3e5281a106e55cc014d5bb9cff1f576bc47" target='_blank'>
              Dosage responses of aneuploid autosomal chromosomes
              </a>
            </td>
          <td>
            Natali Papanicolaou, Sebastian Wettersten, Guilherme Maia, Antonio Lentini, B. Reinius
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d62c6090571c67ea1645ed1df6849f539cf798b" target='_blank'>
              Molecular and cellular dynamics of measurable residual disease progression in myelodysplastic syndromes
              </a>
            </td>
          <td>
            Yu-Hsiang Chen, Timothy Bi, Minhang Xu, Jaehyun Lee, , Mengchuan Zhuang, A. Velarde, D. McCurry, Hui Yang, G. Al-Atrash, Gabriela Rondon, E. Shpall, F. Jelloul, F. N. Cervoni-Curet, Sassine Ghanem, Celine Kerros, Priya Koppikar, J. Molldrem, G. Garcia-Manero, Ankit Patel, Pavan Bachireddy
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Although genome sequencings have recognized many cancer-associated genes in hepatocellular carcinoma (HCC), distinguishing their functional effect remains challenging. Leveraging on a genome-wide CRISPR knockout (KO) screening, we uncovered spliceosome factors as major survival essential genes in HCC and up-regulations of ferroptosis suppressors [particularly glutamate-cysteine ligase catalytic subunit (GCLC)] in lenvatinib resistance. Our KO screen in patient-derived HCC organoid showed splicing factor 3b subunit 4 (SF3B4) to be top-ranked, conferring prosurvival signal in HCC organoid and driving tumorigenic potentials in both hepatic progenitor organoids and hydrodynamic tail vein injection HCC murine model. The combined RNA immunoprecipitation sequencing, long-read isoform sequencing, and transcriptome revealed characteristic splicing landscape regulated by SF3B4 and identified T-box transcription factor 3 (TBX3) variant TBX3+2a as a potent downstream effector. Our findings highlighted vital roles of SF3B4 in HCC cell survival and tumor progression, and the phenomenon of ferroptosis resistance in patients unresponsive to first-line agent lenvatinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee6f7f5b50ee5eb699e0b5d0e4977d78a15779e" target='_blank'>
              Genome-wide CRISPR screen identifies splicing factor SF3B4 in driving hepatocellular carcinoma
              </a>
            </td>
          <td>
            Yue Guo, Mingjing Xu, Hua Xue, Xiaofan Ding, Alissa M. Wong, Na Lin, Dandan Pu, A. M. Wong, Xin Wang, Hui Zhao, Nathalie Wong
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Resistance to platinum-based drugs and PARP inhibitors (PARPi) is the leading cause of treatment failure in epithelial ovarian cancer (EOC). This study aimed to identify resistance mechanisms shared by both. Using bioinformatic analyses, EOC tissues, primary tumor cells and organoids, and chemoresistant cell lines, we identified lymphoid enhancer-binding factor 1 (LEF1) as a candidate, whose expression was increased in both platinum-resistant and PARPi-resistant tumors. Moreover, LEF1 deficiency increased EOC cell sensitivity to cisplatin and PARPi in vitro and in vivo. Mechanistically, LEF1 knockdown promoted double-strand breaks and significantly impaired both homologous recombination and nonhomologous end joining by directly downregulating the transcription of PARP1 and NUMA1. In addition, the LEF1 inhibitor niclosamide increased ovarian cancer sensitivity to Cisplatin and PARPi in patient-derived organoids and Niraparib-resistant cell lines. These findings indicate that LEF1 is a potential therapeutic target for overcoming resistance to chemotherapy based on platinum and PARPi in EOC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/648ee47786050552f00a0c31f0d21f6a61f09642" target='_blank'>
              LEF1 confers resistance to DNA-damaging chemotherapies through upregulation of PARP1 and NUMA1 in ovarian cancer
              </a>
            </td>
          <td>
            Lin Huang, Mengna Zhu, Mengqing Chen, F. Ying, Qiulei Wu, Pan Liu, Wenhan Li, Guoqing Li, Yuewen Gao, Shuyan Yi, Wenwen Wang, Yiping Wen, Si Sun, Jing Cai, Man Xiao
          </td>
          <td>2025-09-10</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc219a64ec3d466c88c4dfd11649123286e2032b" target='_blank'>
              Cross-Species Insights from ART-D to Uncover Evolutionarily Conserved Oncogenic Mechanisms
              </a>
            </td>
          <td>
            Yifan Guo, Jiadong Zheng, Yixin Wu, Kun Zhao, Dian Lv, Wenhan Liu, Mengling Wang, Jin-Yu Lu, Wenyan Xu, Xianping Wang, Xianjue Ma
          </td>
          <td>2025-10-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9ca9b266d6e3d48dee0fe7d9f0b768bc883a83" target='_blank'>
              Functional interrogation uncovers a critical role for a high-plasticity cell state in lung adenocarcinoma
              </a>
            </td>
          <td>
            , , Jonathan Rub, Klavdija Krause, Emma Brown, Gary Guzman, Zeda Zhang, Hannah C Styers, Griffin Hartmann, Zhuxuan Li, Xueqian Zhuang, Scott W. Lowe, D. Betel, Yan Yan, Tuomas Tammela
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Ovarian cancer (OC) is lethal due to near universal development of resistance to platinum-based chemotherapy. Metabolic adaptations can play a pivotal role in therapy resistance. Here, we aimed to identify key metabolic pathways that regulate platinum response and represent potential therapeutic targets. Transcriptomic and metabolomic analyses in cisplatin sensitive and resistant ovarian cancer cells identified enrichment of pyrimidine metabolism related to upregulated de novo pyrimidine synthesis. 15N-glutamine flux analysis confirmed increased de novo pyrimidine synthesis in cisplatin resistant cells. Targeting this pathway using brequinar (BRQ), an inhibitor of the key enzyme dihydroorotate dehydrogenase (DHODH), decreased cell viability, delayed G2/M cell cycle progression, and altered expression of genes related to mitochondrial electron transport in resistant cells. Under basal conditions, cisplatin resistant cells had a lower oxygen consumption rate (OCR) and spare respiratory capacity (SRC) than sensitive cells. BRQ suppressed OCR in both sensitive and resistant but only inhibited SRC in resistant cells. In cell line-derived and patient-derived xenograft models, BRQ attenuated the growth of cisplatin resistant ovarian tumors and enhanced the inhibitory effects of carboplatin. Together, these results identify metabolic reprogramming in cisplatin resistant ovarian cancer that induces an acquired dependency on de novo pyrimidine synthesis, which can be targeted to sensitize tumors to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d708b6c5cad44de1ded3e0fd0aa39875db711a4c" target='_blank'>
              Metabolic Dependency on De Novo Pyrimidine Synthesis is a Targetable Vulnerability in Platinum Resistant Ovarian Cancer.
              </a>
            </td>
          <td>
            H. Cardenas, Yinu Wang, Guangyuan Zhao, Delan Xingyue Hao, A. Isac, Vanessa Hernandez, Ujin Kim, Wenan Qiang, Hao F. Zhang, Daniela Matei
          </td>
          <td>2025-10-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer.
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-06</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4375fd9e9cdc4563ab3b711dfac8983eeba0ca" target='_blank'>
              Genome-Wide CRISPR/Cas9 Screening Reveals Lipid Metabolism and Inflammatory Signalling as Modulators of Ganoderic Acid DM Cytotoxicity
              </a>
            </td>
          <td>
            Nimo Abdullah, Jacqueline Lewis, Prakash Arumugam
          </td>
          <td>2025-10-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/573448152ca34657743158301dc6d5b323113cd1" target='_blank'>
              TP53-mutant AML with ribosomal gene loss exhibits impaired protein translation and sensitivity to HSP90 inhibition
              </a>
            </td>
          <td>
            Jean-François Spinella, J. Chagraoui, I. Boivin, Guillaume Richard Carpentier, Céline Moison, N. Mayotte, François Béliveau, Vincent-Philippe Lavallée, J. Hébert, Guy Sauvageau
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear.We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells.High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells.Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0d2028082631983321af009576821353c38710" target='_blank'>
              Pharmacological Induction of Irreversible Senescence in Drug-Tolerant Persister Cells Prevents Tumor Relapse
              </a>
            </td>
          <td>
            Bowen Wang, Yun Zhi, Pengqi Wang, Wenbo Guo, Jin Gu, Hanqing He, Kezhang He, Wei Zhou, Ting Wang, Nana Huang, Haixia Yang, Dan Wang, Tianhua Ma, Sheng Ding
          </td>
          <td>2025-10-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Multidrug resistance is a significant barrier in cancer therapy largely due to poorly understood regulatory mechanisms. Here we reveal that certain anticancer drugs can bind to newly synthesized proteins prior to reaching their canonical targets, resulting in various forms of protein damage. This binding disrupts protein functions, particularly those of mitochondrial proteins, resulting in substantial cytotoxicity. The protein damage is further exacerbated by mitochondrial reactive oxygen species generated as a consequence of the initial damage, creating a positive feedback loop. In response, cancer cells rapidly initiate a chain of events, which we term the Protein Damage Response (PDR). This includes damage recognition primarily mediated by protein ubiquitination and subsequent damage clearance via the proteasome system. Notably, patients with advanced, drug-resistant metastatic breast or colon cancers exhibit elevated proteasome activity. In an effort to predict drug resistance, we developed a sensitive kit for detecting proteasome levels, enabling the identification and subtyping of patients with high proteasome activity to support tailored therapeutic strategies. Using a three-dimensional tumor slice culture-based drug sensitivity assay and an investigator-initiated clinical trial, we demonstrate that three clinically approved proteasome inhibitors effectively overcome multidrug resistance in colon and breast cancer patients with elevated proteasome activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1785188ad7f7984f485f66a5779a54c95bdf57ed" target='_blank'>
              Suppressing protein damage response to overcome multidrug resistance in cancer therapy
              </a>
            </td>
          <td>
            Fangyuan Shao, Zongjie Li, Hao Xiao, Yujun Chen, Yuheng Zhang, Ling Li, Yuzhong Peng, Xinyi Li, Yuxing Hou, Bo Li, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Xi Han, Jiaxin Yao, Cuiting Zhang, Lijian Wang, Haifeng Li, N. Shao, Kai Miao, Xiaoling Xu, Yanxia Shi, Changhua Zhang, Jun Yan, Ying Lin, Chu-Xia Deng
          </td>
          <td>2025-09-30</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The persistent residual tumor cells that survive after chemotherapy are a major cause of treatment failure, but their survival mechanisms remain largely elusive. These cancer cells are typically characterized by a quiescent state with suppressed activity of MYC and MTOR. We observed that the MYC-suppressed persistent triple-negative breast cancer (TNBC) cells are metabolically flexible and can upregulate mitochondrial oxidative phosphorylation (OXPHOS) genes and respiratory function (“OXPHOS-high” cell state) in response to DNA-damaging anthracyclines such as doxorubicin, but not to taxanes. The elevated biomass and respiratory function of mitochondria in OXPHOS-high persistent cancer cells were associated with mitochondrial elongation and remodeling, suggestive of increased mitochondrial fusion. A genome-wide CRISPR editing screen in doxorubicin-persistent OXPHOS-high TNBC cells revealed the BCL-XL gene as the top survival dependency in these quiescent tumor cells, but not in their untreated proliferating counterparts. Quiescent OXPHOS-high TNBC cells were highly sensitive to BCL-XL inhibitors, but not to inhibitors of BCL2 and MCL1. Interestingly, inhibition of BCL-XL in doxorubicin-persistent OXPHOS-high TNBC cells rapidly abrogated mitochondrial elongation and respiratory function, followed by caspase 3/7 activation and cell death. The platelet-sparing proteolysis-targeted chimera (PROTAC) BCL-XL degrader DT2216 enhanced the efficacy of doxorubicin against TNBC xenografts in vivo without induction of thrombocytopenia that is often observed with the first-generation BCL-XL inhibitors, supporting the development of this combinatorial treatment strategy for eliminating dormant tumor cells that persist after treatment with anthracycline-based chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfb7ad9e0cb28b546f19e574e42e36713a6ee110" target='_blank'>
              Quiescent OXPHOS-High Triple-Negative Breast Cancer Cells That Persist After Chemotherapy Depend on BCL-XL for Survival
              </a>
            </td>
          <td>
            Slawomir Andrzejewski, Marie Winter, Leandro Encarnacao Garcia, O. Akinrinmade, Francisco Madeira Marques, Emmanouil Zacharioudakis, Anna Skwarska, J. Aguirre-Ghiso, Marina Konopleva, Guangrong Zheng, Susan A. Fineberg, Daohong Zhou, E. Gavathiotis, Tao Wang, Eugen Dhimolea
          </td>
          <td>2025-10-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df773895d5d3bb014e2dd4dc6276d432f2994491" target='_blank'>
              YAP1 defines an emergent, plastic population of relapsed small cell lung cancer
              </a>
            </td>
          <td>
            C. Stewart, K. Ramkumar, Runsheng Wang, Yuanxin Xi, Alexa J. Halliday, L. Diao, Qi Wang, Alejandra G Serrano, Sarah M. Groves, Simon Heeke, Azusa Tanimoto, L. Kaiser, Whitney E Lewis, Mukulika Bose, Pedro Da Rocha, L. Karacosta, Vito Quaranta, Jing Wang, Julie George, Luisa Maren, Solis Soto, Bingnan Zhang, J. Heymach, L. Byers, C. Gay
          </td>
          <td>2025-10-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da08fc2c51aefcac002abe7af73df18eb83fa23" target='_blank'>
              Integrative proteo-genomic profiling uncovers key biomarkers of lapatinib resistance in HER2-positive breast cancer.
              </a>
            </td>
          <td>
            J. Steggall, V. Rajeeve, N. Al-Subaie, A. Naeem, A. Ikram, A. Naeem, A. Hayat
          </td>
          <td>2025-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20106bc7c27e870a9fd1562421c545c6ddaef656" target='_blank'>
              Multi-omics-driven kinetic modeling reveals metabolic vulnerabilities and differential drug-response dynamics in ovarian cancer
              </a>
            </td>
          <td>
            Ilias Toumpe, M. Masid, V. Hatzimanikatis, L. Miskovic
          </td>
          <td>2025-10-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Dysregulation of early B cell lymphopoiesis—the process guiding cellular immunity development—can lead to malignancy, making it crucial to understand its regulatory mechanisms. We generated a multiomics resource comprising paired chromatin accessibility and gene expression profiles across eight human B cell precursor populations, providing a detailed characterization of early human B cell development. Integrative analysis revealed highly cell type–specific regulatory elements and enabled the reconstruction of the gene regulatory network governing differentiation. We identified putative candidate regulons, such as ELK3, enriched in pro–B cells and potentially involved in cell cycle progression. Regulons from bulk data were projected onto single-cell data, validating their activity and refining the regulatory landscape. This resource enabled identification of active regulatory programs and transformation-associated states in B cell acute lymphoblastic leukemia. The publicly available atlas provides a valuable resource for understanding B cell development and disease, supporting future efforts to decode regulatory programs in immunity and hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72948d6bed4b92194fb9e2b0dbf1593531ef8b66" target='_blank'>
              Uncovering the regulatory landscape of early human B cell lymphopoiesis and its implications in the pathogenesis of B-ALL
              </a>
            </td>
          <td>
            Núria Planell, Xabier Martínez-de-Morentin, Daniel Mouzo, D. Lara-Astiaso, A. Vilas-Zornoza, P. S. Martín-Uriz, D. Alignani, B. Paiva, A. Maillo, Aleksandra Kurowska, Nerea Berastegui, P. García-Olloqui, Arantxa Urdangarin, Peri Noori, A. Ortega-Legarreta, Mikel Hernaez, V. Lagani, N. Kiani, Matthias Merkenschlager, T. Ezponda, J. Martín-Subero, Ricardo N. Ramirez, Jesper N. Tegnér, F. Prósper, D. Gómez-Cabrero
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65a53aeb0c15e601e15d7f66e640ef0193a0c8f9" target='_blank'>
              Lineage origin and microenvironment shape neuroblastoma transcriptional state and plasticity
              </a>
            </td>
          <td>
            Nora Fresmann, Julia Köppke, A. Gauert, Anis Senoussi, Pedro Olivares-Chauvet, Marie Schott, Lennart Höfer, A. Henssen, N. Rajewsky, B. Spanjaard, Anja I. H. Hagemann, J. Junker
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c57f7b414b240c72f02d9113a840fdad776a055" target='_blank'>
              BRCA2-deficiency Causes Global Transcriptomic Alterations in Endothelial Cells
              </a>
            </td>
          <td>
            Sepideh Nikfarjam, Aman Singh, Shuhan Bu, Hien C. Nguyen, Krishna K. Singh
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185da472d5e884ad916a2af2cce445855a3e882" target='_blank'>
              Single-cell lineage trajectory defines CDK inhibitor-sensitive cells-of-origin in esophageal squamous cell cancer
              </a>
            </td>
          <td>
            Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The recurrence of clinically advanced cancers is an evolutionary consequence of standard-of-care chemotherapies generally administered at maximum tolerated doses to kill as many cancer cells as possible. The inevitable appearance of resistance raises the possibility of shifting treatment goals from complete tumor eradication to long-term disease control. The latter approach is employed by adaptive therapy, which aims to inhibit the evolutionary dynamics governing the spread of resistant tumor phenotypes. Adaptive therapy changes focus from the cancer cells that are responsive to therapy to those that are resistant and ultimately govern outcome. This therapeutic approach retains a pool of sensitive cancer cells to compete with the therapy-resistant ones through dynamic dose modulation and/or timing. Thus, fluctuations of treatment-sensitive cells are used to control the resistant population and prolong tumor control with existing therapy agents. Here, we explore non-genetic mechanisms of resistance, including the protective role of the tumor stroma, the epithelial-to-mesenchymal transition, the overexpression of drug efflux pumps, and the extracellular vesicle-mediated transfer of them. These mechanisms can increase the size of the resistant population at the expense of the sensitive one, reducing the ability of adaptive therapy to force tumor evolution into controllable cycles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd5596bbafd3812efcd1d8c0a4bfdf21b8096e0" target='_blank'>
              Adaptive cancer therapy: can non-genetic factors become its achilles heel?
              </a>
            </td>
          <td>
            Gábor Valcz, Robert A. Gatenby, B. Újvári, E. Buzás, Béla Molnár
          </td>
          <td>2025-09-22</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In Mycobacterium tuberculosis, a prevalent and deadly pathogen, resistance to antibiotics evolves primarily through non-synonymous mutations in proteins. Sequence-based analyses are currently used to understand the genetic basis of antibiotic resistance, either via genotype-phenotype association, or via signals of convergent evolution. These methods focus on primary sequence and usually neglect other biological signals such as protein structural information. We hypothesize that integrating the structural context of mutations improves the prediction of effects on function and phenotype. We curate high confidence structural annotations for the M. tuberculosis proteome from 1,371 crystallography and 2,316 AlphaFold predictions, and combine the structures with mutations from over 31,000 clinical M. tuberculosis isolates. We demonstrate that mutations in proteins known to cause resistance are clustered in 3D space, even in proteins where inactivating mutations at any position are thought to cause resistance. We develop a statistic to search the M. tuberculosis proteome for signal of clustered non-synonymous mutations, finding over 450 proteins that display this signal, many of which have a known relationship with antibiotic resistance. Innovatively, we show that a supervised classifier trained on structure features alone has an F1 score of 94.6% at classifying mutations as resistance-conferring. This work demonstrates that protein structure provides useful information for categorizing which variants may cause antibiotic resistance, even when the majority of structures are AI-predicted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/736eca4109e1243533868a8b023c33d1b51f37d5" target='_blank'>
              The structural context of mutations in proteins predicts their effect on antibiotic resistance
              </a>
            </td>
          <td>
            Anna G Green, Mahbuba Tasmin, R. Vargas, M. Farhat
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Tongue squamous cell carcinoma (TSCC) is a common oral cancer that has a high propensity for recurrence and metastasis. Therefore, TSCC has a 50% 5-year survival rate. Platinum-based chemotherapy is an effective treatment for squamous cell carcinoma, however, chemotherapy resistance remains a major issue. Therefore, innovative and effective drug combinations are needed to improve TSCC patient prognosis. Methods In this study, we conducted an in vitro CRISPR/Cas9 library screen using two TSCC cell lines (Tscca and Cal27) to identify specific genes that, when inhibited, synergize with cisplatin to effectively suppress tumor growth. Results We identified STK19 as a potential drug target. Inhibition of STK19 enhances the response of TSCC to cisplatin. Through genetic and pharmacological methods, it has been demonstrated that reducing STK19 activity enhances cisplatin-induced DNA damage. The mechanism involves the depletion of MGMT with STK19 inhibition, leading to conditional lethality and synergistic reduction of tumors in vivo when combined with cisplatin. Overall, in this study, unbiased genetic testing was used to successfully identify synthetic lethal drug combinations for TSCC. Conclusion STK19 was identified as a promising target that could enhance the killing effects of cisplatin on tongue squamous carcinoma cells, offering a novel therapeutic option for individuals who are insensitive to conventional treatment methods. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07156-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381730f5ba586c5f8338553be785fcaa929b30ab" target='_blank'>
              CRISPR/Cas9 library screening reveals that STK19 has synergistic antitumor effects when combined with cisplatin on tongue squamous cell carcinoma
              </a>
            </td>
          <td>
            Chunyan Li, Wen Peng, Zhaoming Zhong, Chao Zhang, Xuemin Wang, Rujia Qin, Qian Lei, Jialing Lv, Feineng Liu, Yinghong Zhao, Zengbo Lv, Chongxin Li, Shixuan Yang, Haixia Zhang, Zhi Tao, Chuanzheng Sun
          </td>
          <td>2025-10-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/098344733a7758eefb37e7a0275fbd5613565b00" target='_blank'>
              Genomic and Epigenetic Interplay Drives Adaptive Fusion via Reproduction Trade-Off
              </a>
            </td>
          <td>
            Yang Chen, Lin Bi, Xiaoli Fan, Shuang Liang, Chengyuan Li, Yihong Wang, Sophie G. Martin, Gaowen Liu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Abstract Resistance to targeted cancer therapies is a significant barrier to favorable treatment outcomes. Malignant cells can tolerate and resist drug treatments due to their biological flexibility. Specifically, slow-cycling drug-resistant cells may achieve permanent resistance to the treatment or restore sensitivity upon cessation of therapy. Enhancing cancer treatment methodologies necessitates a deeper understanding of the adaptability of tumor cells. Drug resistance and cellular heterogeneity are closely associated with cancer cell adaptability. Alterations in cellular signaling, interactions with the tumor microenvironment, and genetic and epigenetic alterations are all implicated. Analyzing these pathways will enhance our understanding of how cancer cells evolve and evade treatment. Two effective strategies to address cancer cell adaptability are to target specific biological pathways and to employ combination therapies. The progression of cancer therapy methodologies relies on comprehending and exploring the concept of cancer cell adaptability. Understanding tumor heterogeneity and drug resistance necessitates identifying the cellular, molecular, and genetic processes that govern cancer cell plasticity. This understanding enables the development of more personalized and effective cancer therapies, leading to improved treatment outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/714e22df09170482d5d0d601474075162f529a30" target='_blank'>
              Cancer cell plasticity and therapeutic resistance: mechanisms, crosstalk, and translational perspectives
              </a>
            </td>
          <td>
            Saeid Ghorbian
          </td>
          <td>2025-09-26</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Phenotypic plasticity is a key mechanism of metastatic progression and cancer therapy resistance. This hallmark of human malignancies is enabled by highly conserved epigenetic mechanisms that control gene expression. Functional alterations in DNA methylation and histone post-translational modifications have been extensively described as drivers of metastatic dissemination and therapy resistance. Pharmacological inhibitors of epigenetic enzymes can revert these alterations, thereby stopping cancer progression and counteracting the emergence of resistant clones. Despite promising pre-clinical evidence, the clinical implementation of epigenetic therapies in solid cancers has led to disappointing results. Several factors can explain these challenges, including the lack of rational combinations. Notably, response to epigenetic treatments can be heterogeneous and short-lived. A liquid biopsy technology that allows the measure of specific epigenetic alterations enables patient selection and therapy monitoring, leading to the development of precision epigenetic therapies. In this review, we discuss the state of the art of this emerging treatment modality, and we identify key challenges that need to be overcome to reach the full potential of this new therapeutic concept.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/876ad36ab7595e359c53da7e87448ed401afa714" target='_blank'>
              Precision epigenetic therapies in oncology
              </a>
            </td>
          <td>
            Arundhati Chaudhary, Kayleigh J A Orchard, F. Salani, Theodora Partsou, Mark Eccleston, Guido Bocci, A. Italiano, Francesco Crea
          </td>
          <td>2025-09-19</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34b22389536004a2e5e3234be8036e5614f2a3ae" target='_blank'>
              A blueprint for mutation-defined hallmark vulnerabilities across human cancers
              </a>
            </td>
          <td>
            Ran Xu, Rebecca San Gil, Xin Tracy Liu, Yanfei Qi, Derek Tran, , , Lenka Munoz, Graham J. Mann, Yuchen Feng
          </td>
          <td>2025-10-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a93b5a70ed710c9aacd9cdfba3788cae5ad699" target='_blank'>
              Large-scale single-cell phylogenetic mapping of clonal evolution in the human aging esophagus
              </a>
            </td>
          <td>
            Tamara Prieto, Dennis J. Yuan, John Zinno, Clayton Hughes, Nicholas Midler, Sheng Kao, Jani Huuhtanen, Ramya Raviram, Foteini Fotopoulou, Neil Ruthen, Srinivas Rajagopalan, Joshua S. Schiffman, Andrew R. D’Avino, Sang-Ho Yoon, Jesus Sotelo, Nathaniel D. Omans, Noelle Wheeler, Alejandro Garces, Barun Pradhan, A. P. Cheng, N. Robine, Catherine Potenski, Katharine Godfrey, N. Kakiuchi, Akira Yokoyama, Seishi Ogawa, Julian Abrams, Ivan Raimondi, D. Landau
          </td>
          <td>2025-10-13</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb9b509dc6cd6165a216f7f2331a61922945369" target='_blank'>
              Pooled CRISPR screens identify genes and non-coding genomic regions that regulate red blood cell density
              </a>
            </td>
          <td>
            Nicolas Brosseau, Thomas Pincez, Ken Sin Lo, M. Beaudoin, Guillaume Lettre
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf3f4a8479510c49ff95777ff501eb3f2838138" target='_blank'>
              Epigenetic priming promotes tyrosine kinase inhibitor resistance and oncogene amplification.
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B Jensen, Ning Sun, Nelli Khudaverdyan, Ting Zhao, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, G. Wang, A. Tackett, Yinsheng Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Z Xiao
          </td>
          <td>2025-10-23</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e127ecdb0d709635e11801397bb592a991279b49" target='_blank'>
              Locus-specific transcriptional regulation of transposable elements by p53
              </a>
            </td>
          <td>
            Julia M. Freewoman, Andrew J. Rosato, Thomas M. Russell, Feng Cui
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df2588e9805448486d7a055c5c952eb019840ed" target='_blank'>
              Analysis of Naturally Occurring Somatic Insertions in the Human Genome
              </a>
            </td>
          <td>
            Chih-Lin Hsieh, Zarko Manojlovic, Timothy Okitsu, C. Okitsu, Jordan Wlodarczyk, Nick Shillingford, Ramzi Bawab, Yong Hwee Eddie Loh, Michael R. Lieber
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole-genome sequencing to study the coding and noncoding genomes (promoters, enhancers, and noncoding–nonenhancer regions) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region is preserved despite treatment-induced genetic bottlenecks. Compared with other noncoding regions, enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find that recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer, including KLF5 and TP63. Transcriptional expression based on RNA sequencing data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared with cancers without enhancer mutations, suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well. Significance: Recurrent somatic mutations in enhancer regions that control pancreatic lineage genes represent a previously unrecognized source of genetic alteration in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Noncoding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Human genome resequencing typically involves mapping reads to a reference genome to call variants; however, this approach suffers from both technical and reference biases, leaving many duplicated and structurally polymorphic regions of the genome unmapped. Consequently, existing variant benchmarks, generated by the same methods, fail to assess these complex regions. To address this limitation, we present a telomere-to-telomere genome benchmark that achieves near-perfect accuracy (i.e. no detectable errors) across 99.4% of the complete, diploid HG002 genome. This benchmark adds 701.4 Mb of autosomal sequence and both sex chromosomes (216.8 Mb), totaling 15.3% of the genome that was absent from prior benchmarks. We also provide a diploid annotation of genes, transposable elements, segmental duplications, and satellite repeats, including 39,144 protein-coding genes across both haplotypes. To facilitate application of the benchmark, we developed tools for measuring the accuracy of sequencing reads, phased variant call sets, and genome assemblies against a diploid reference. Genome-wide analyses show that state-of-the-art de novo assembly methods resolve 2–7% more sequence and outperform variant calling accuracy by an order of magnitude, yielding just one error per 100 kb across 99.9% of the benchmark regions. Adoption of genome-based benchmarking is expected to accelerate the development of cost-effective methods for complete genome sequencing, expanding the reach of genomic medicine to the entire genome and enabling a new era of personalized genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc9eb566f5ca36d1f2d3bccbe8022cfc8c0b963a" target='_blank'>
              A complete diploid human genome benchmark for personalized genomics
              </a>
            </td>
          <td>
            Nancy F. Hansen, Nathan Dwarshuis, Hyun Joo Ji, A. Rhie, Hailey Loucks, Glennis A. Logsdon, Mitchell R. Vollger, Jessica M. Storer, Juhyun Kim, Eleni Adam, Nicolas Altemose, D. Antipov, Mobin Asri, Sofia Barreira, Stephanie C. Bohaczuk, Andrey V. Bzikadze, S. A. Carioscia, Andrew Carroll, Kuan-Hao Chao, Yanan Chu, Arun Das, Peter Ebert, A. English, M. Fleharty, Laura E. Fleming, G. Formenti, A. Guarracino, Gabrielle Hartley, Katharine M. Jenike, Jenna Kalleberg, Yu Kang, Robert King, Josipa Lipovac, Mira Mastoras, Matthew W. Mitchell, S. Negi, Nathan D. Olson, Keisuke K. Oshima, L. F. Paulin, Brandon D. Pickett, David Porubsky, Jane E. Ranchalis, D. Ranjan, M. Rautiainen, Harold Riethman, Robert D Schnabel, F. Sedlazeck, Kishwar Shafin, M. Šikić, Steven J. Solar, Alex Sweeten, W. Timp, Justin Wagner, DongAhn Yoo, Ying Zhou, Erik Garrison, E. Eichler, Michael C Schatz, Andrew B. Stergachis, Rachel J. O’Neill, K. Miga, Steven L. Salzberg, S. Koren, J. Zook, A. Phillippy
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>5</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa96a0707b7c6c78fe5bd4b2a45727093019bee" target='_blank'>
              Targeting the crosstalk of metabolism reprogramming and replication stress: novel strategy to combat cancer.
              </a>
            </td>
          <td>
            Wenbo Liu, Xiangyan Jiang, Yong Ma, Huiguo Qing, Yingcun Bao, Zuoyi Jiao
          </td>
          <td>2025-09-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Epithelioid hemangioendothelioma (EHE) is a difficult to treat vascular sarcoma defined by TAZ-CAMTA1 (TC) or YAP-TFE3 (YT) fusion proteins. Human cell lines needed to further understand the pathogenesis of EHE have been lacking. Herein, we describe a method to generate EHE extended primary cell cultures. An integrated multi –omic and functional approach was used to characterize these cultures. The cell cultures, relatively homogenous by single cell RNA-Seq, demonstrated established characteristics of EHE including increased proliferation, anchorage independent growth, as well as the overall gene expression profile and secondary genetic alterations seen in EHE. Whole genome sequencing (WGS) identified links to epigenetic modifying complexes, metabolic processes, and pointed to the importance of the extracellular matrix (ECM) in these tumors. Bulk RNA-Seq demonstrated upregulation of pathways including PI3K-Akt signaling, ECM/ECM receptor interaction, and the Hippo signaling pathway. Development of these extended primary cell cultures allowed for single-cell profiling which demonstrated different cell compartments within the cultures. Furthermore, the cultures served as a therapeutic platform to test the efficacy of TEAD inhibitors in vitro. Overall, the development of primary EHE cell cultures will aid in the mechanistic understanding of this sarcoma and serve as a model system to test new therapeutic approaches. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-19453-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4728cebf1ce6a2c3cd2884745cce7881e7bc18e9" target='_blank'>
              EHE cell cultures are a platform for mechanistic and therapeutic investigation
              </a>
            </td>
          <td>
            Nicholas Scalora, Gillian DeWane, Yuliia Drebot, Ali A. Khan, Souradip Sinha, Krishnendu Ghosh, Denise Robinson, Patricia Cogswell, Andrew M Bellizzi, Anthony N. Snow, Patrick Breheny, M. Chimenti, M. Tanas
          </td>
          <td>2025-10-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ultra-accurate detection of rare somatic mutations is critical for understanding mutational processes in human disease, aging, and environmental exposures, yet current methods are limited by error rates, restricted genome coverage, and high DNA input. We present UDSeq, a duplex sequencing protocol combining random fragmentation, efficient UMI ligation, and quantitative input control to achieve near-complete genome/exome representation from as little as 100 pg DNA. Benchmarking in human sperm estimates a UDSeq error rate of ∼2.5×10-9 per base pair. UDSeq captures mutational signatures from heterogeneous populations without clonal expansion, reproduces exposure-specific patterns in cell lines and rodent models, and enables cross-species profiling. Compared with prior duplex methods, UDSeq yields up to fourfold more usable duplex molecules, improves library conversion, and remains cost-effective. We include a step-by-step protocol with quality-control checkpoints for fragment size, ligation yield, library conversion, and duplication rate. UDSeq provides a scalable, low-input platform for accurate profiling of somatic mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a25aa13b255931a4c667fbbd3b80aa9966be011" target='_blank'>
              A Universal Duplex Sequencing Approach for Accurate Detection of Somatic Mutations
              </a>
            </td>
          <td>
            Shuvro P. Nandi, Yuhe Cheng, Shams Al-azzam, Safa Saeed, Audrey Kristin, Nadia Sunico, Isabella Stuewe, Zichen Jiang, Luka Culibrk, Maria Zhivagui, Xiaoxu Yang, Rachel Wise, Foster C. Jacobs, Bérénice Chavanel, M. Korenjak, Mia Petljak, S. Balbo, Laurie G Hudson, Ke Jian Liu, Jiri Zavadil, Joseph G. Gleeson, Ludmil B. Alexandrov
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3948f28751f8205d2389f7d1df87013dbea193d3" target='_blank'>
              RUNX1T1-HDAC Reprogramming of the HOX Code Signaling Drives a Targetable Pan-Cancer Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, Catherine Yijia Zhang, Xiao Jin, Yaru Xu, I. Y. Kim, Ping Mu
          </td>
          <td>2025-10-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Isocitrate dehydrogenase 1 (IDH1) mutations are key drivers of glioma biology, influencing tumor aggressiveness and treatment response. To elucidate their molecular impact, we performed proteome analysis on patient-derived (PD) and U87MG glioma cell models with either mutant or wild-type IDH1. We quantified over 6000 protein groups per model, identifying 1594 differentially expressed proteins in PD-AS (IDH1MUT) vs. PD-GB (IDH1WT) and 904 in U87MUT vs. U87WT. Both IDH1MUT models exhibited enhanced MHC antigen presentation and interferon signaling, indicative of an altered immune microenvironment. However, metabolic alterations were model-dependent: PD-AS cells shifted toward glycolysis and purine salvage, while U87MUT cells retained oxidative phosphorylation, potentially due to D2-hydroxyglutarate (2OHG)-mediated HIF1A stabilization. We also observed a predominance of downregulated DNA repair proteins in IDH1MUT models, particularly those involved in homologous recombination. In contrast, RB1 and ASMTL were strongly upregulated in both IDH1MUT models, implicating them in DNA repair and cellular stress responses. We also found distinct expression patterns of proteins regulating histone methylation in IDH1MUT cells, favoring increased methylation of H3K4, H3K9, and H3K36. A key driver of this may be the upregulation of SETD2 in PD-AS, an H3K4 and H3K36 trimethyltransferase linked to the recruitment of HIF1A as well as DNA mismatch repair proteins. This study uncovers candidate biomarkers and pathways relevant to glioma progression and therapeutic targeting, but also underscores the complexity of predicting glioma pathogenesis and treatment responses based on IDH1 mutation status. While proteome profiling provides valuable insights, a comprehensive understanding of IDH1MUT gliomas will likely require integrative multi-omics approaches, including DNA/RNA methylation profiling, histone and protein post-translational modification analyses, and targeted DNA damage and repair assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2740eea5fa4be364de22267706ccdb117ba04ffc" target='_blank'>
              Proteomic Analysis Uncovers Enhanced Inflammatory Phenotype and Distinct Metabolic Changes in IDH1 Mutant Glioma Cells
              </a>
            </td>
          <td>
            Sigrid Ravn Berg, Alessandro Brambilla, Lars Hagen, Animesh Sharma, C. Vågbø, N. Liabakk, Miroslava Kissova, Miquel Arano Barenys, Magnar Bjørås, S. Torp, Geir Slupphaug
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7c296f01ad5b58abb7169e8898ff697ed61fde" target='_blank'>
              Pan-cancer analysis of patient-specific gene regulatory landscapes identifies recurrent PD-1 pathway dysregulation linked to outcome
              </a>
            </td>
          <td>
            T. Belova, Ping-Han Hsieh, Ladislav Hovan, Daniel Osorio, M. Kuijjer
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Gliomas make up almost half of primary central nervous system tumors. Despite advancements in surgery and neuro-oncology, developing an effective treatment remains challenging. The protein Growth Arrest and DNA-Damage-Inducible, Gamma (GADD45G) is crucial for key cellular functions like DNA repair, genomic stability, and apoptosis. While GADD45G dysregulation has been found in various cancers, its role in glioma is still unclear. Methods We analyzed unified pan-cancer datasets (TCGA, TARGET, GTEx) from UCSC Xena and integrated glioma data from CGGA and GEO (GSE108476). Prognostic value was assessed via multivariate Cox regression and Kaplan-Meier survival analysis using Gliovis. Single-cell RNA-seq data (GSE103224, GSE138794, GSE173278) were processed with Seurat (R 4.2.2), with Harmony for batch correction and UMAP for visualization. Malignant subclusters were annotated using marker genes. Functional enrichment and cell-type proportion estimation were conducted. Single-cell analysis revealed GADD45G expression patterns and identified its top correlated genes in malignant glioblastoma cells. Overexpression of GADD45G was performed to investigate its impact on cell function. Western blot analysis was used to examine the role of GADD45G in glioma cell invasion and migration. Results Through comprehensive analysis across multiple datasets, it was found that GADD45G expression is higher in glioma patients compared to normal individuals, and its expression is generally higher in lower-grade gliomas than in glioblastoma. Cox regression analysis indicated that GADD45G has a protective effect. Survival curves further demonstrated that elevated GADD45G levels are associated with improved overall survival in patients. In this study, we identified four highly heterogeneous GBM cell subpopulations using single-cell data. The MES-like cells was significantly associated with poor prognosis. Spearman correlation analysis revealed the correlation between GADD45G and VIM. Further experiments revealed that GADD45G modulates glioma cell invasion and migration, potentially through its effects on EMT-like phenotypic features. Conclusion GADD45G expression is significantly associated with glioma outcomes and may serve as a promising biomarker for prognosis evaluation. Its involvement in regulating EMT-like phenotypic traits further highlights its therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a082642c3a677a6ad712ab650d25ebba716c4d15" target='_blank'>
              GADD45G as a novel prognostic biomarker and therapeutic target in glioma: integrative analysis of bulk and single-cell RNA sequencing
              </a>
            </td>
          <td>
            Cong Shen, Haiping Cai, Cheng-liang Mao, Jiahao Yang, Kai Tang
          </td>
          <td>2025-09-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is a fundamental epigenetic process that balances X-linked gene expression between females and males by silencing one X chromosome in female cells. Variability or skewing of XCI can influence the clinical presentation of X-linked disorders. Bain type X-linked intellectual disability syndrome (MRXSB), caused by mutations in the X-linked HNRNPH2 gene, is characterized by intellectual disability, developmental delay, and neurological abnormalities. In female patients, XCI heterogeneity complicates disease modeling and therapeutic development. Induced pluripotent stem cells (iPSCs) offer a unique platform to study patient-specific disease mechanisms, but the dynamics of XCI during iPSC reprogramming, maintenance, and differentiation are not fully understood. In this study, we generated 12 iPSC clones from fibroblasts of a female MRXSB patient heterozygous for the HNRNPH2 c.340C > T mutation. Four clones expressed the mutant HNRNPH2 allele and eight expressed the wild-type allele, indicating X chromosome reactivation (XCR) followed by random XCI during reprogramming. Importantly, these XCI patterns remained stable during long-term iPSC propagation and subsequent differentiation into the three germ layers and neural stem cells. Our findings provide new insights into XCI and XCR dynamics in the context of X-linked neurodevelopmental disorders and emphasize the importance of careful clone selection for accurate disease modeling using iPSC-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6b0bf32c5498df56453ec6772d7356b34b8b65" target='_blank'>
              Reactivation of Human X-Linked Gene and Stable X-Chromosome Inactivation Observed in Generation and Differentiation of iPSCs from a Female Patient with HNRNPH2 Mutation
              </a>
            </td>
          <td>
            Guibin Chen, Alexander Rodriguez-Lopez, Darawalee Wangsa, Richa Madan Lomash, Xiuli Huang, Catherine Z. Chen, Rodney A. Bowling, Neda Ghousifam, Courtney J. Banks, Kerstin A. Hurd, Jizhong Zou, Wei Zheng
          </td>
          <td>2025-09-23</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03559-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5143128d7c214ae6fd617f30d5f2500e3f9594d8" target='_blank'>
              A Unifying Mechanism for Shared Splicing Aberrations in Splicing Factor Mutant Cancers
              </a>
            </td>
          <td>
            Prajwal C. Boddu, Rahul Roy, Stephen Hutter, Francis Baumgartner, Wenxue Li, Gabriele Todisco, F. Ficara, M. G. Della Porta, Yansheng Liu, T. Haferlach, Manoj M. Pillai
          </td>
          <td>2025-10-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Treatment-induced neuroendocrine prostate cancer (NEPC) represents a lethal evolution of prostate adenocarcinoma under androgen receptor pathway inhibition, posing a significant clinical challenge. In a recent landmark study, Wang et al. introduced an innovative internal Z-score based approach to comprehensively characterize the transcription factor (TF) landscape in prostate cancer progression, uncovering distinct TF profiles associated with adenocarcinoma and NEPC lineages. Notably, the study proposes a three-phase model of NEPC transdifferentiation—comprising de-differentiation, dormancy, and re-differentiation—revealing dynamic shifts in TF expression that underpin lineage plasticity and therapeutic resistance. This commentary critically evaluates the methodological advancements, the functional significance of the identified TF signatures, and the broader implications of these findings for developing novel therapeutic strategies. By delineating the molecular events driving the transition from androgen receptor (AR)-dependent adenocarcinoma to treatment-resistant NEPC, this work underscores the potential of targeting early and dormant phases of transdifferentiation to improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f432a498dcbb709202897ddd04a1b59a52e28fa2" target='_blank'>
              Transcription Factor Dynamics in Neuroendocrine Prostate Cancer Development
              </a>
            </td>
          <td>
            Yu Wang, Yuzhuo Wang
          </td>
          <td>2025-10-24</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="


 Acquired treatment resistance in homologous recombination (HR) deficient high-grade serous ovarian cancer (HGSOC) commonly involves restoration of HR, often through secondary, somatic reversion mutations in HR genes. Our recent analysis of HR deficient, end-stage HGSOC found that most resistance mechanisms, including reversion mutations, are subclonal. The aim of this study is to investigate how the subclonal, reversion-negative cells persist during treatment. We are investigating single cell sequencing approaches that enable genotyping of individual cells, such that the phenotype of cells without reversions can be studied. In a pilot experiment we have used two HGSOC cell lines with known reversion mutations. To genotype individual cells and thus overcome the limitations of standard single-cell RNA sequencing (scRNA-seq), we combined short-read whole-transcriptome scRNA-seq with long-read, targeted single-cell cDNA sequencing on an Oxford Nanopore MinION using a 44-gene hybridization capture panel. A custom bioinformatics pipeline, incorporating Flexiplex for multiallelic codon resolution, annotated cell barcodes with their HR driver, reversion and TP53 mutations. Genotypes were then transferred to the short-read dataset following outlier and doublet filtering, and transcriptomic differences between cells with and without reversions were identified through differential gene expression analysis. We have repeated this protocol on a patient ascites sample. Prior short-read DNA sequencing from the cell lines and matched tumor samples defined the driver BRCA1 mutation, reversion events, pathogenic TP53 mutations, and whether there was loss of heterozygosity (LOH) at the BRCA1 loci. We successfully genotyped BRCA1 driver and reversion alleles in both cell lines in the long-read data. Rare cells heterozygous wildtype for the BRCA1 driver mutation were detected, suggesting either back-reversion events or subclones that escaped LOH - cells that were undetectable by bulk short-read methods. Cell genotypes were mapped to the short-read whole transcriptome data, and transcriptional programs that may support cell survival were identified in the non-reverted cells. Our approach demonstrates the utility of unbiased long-read sequencing for detecting BRCA1/2 reversions at single-cell resolution, including potential back-reversion subclones that are challenging to identify through bulk sequencing approaches. The differential gene expression results have identified potential resistance mechanisms. Ongoing work includes incorporation of fusion detection, structural-variant calling, and splice isoform analysis in our bioinformatics pipeline to further elucidate resistance pathways. We anticipate this work will result in an increased understanding of the resistance mechanisms that arise in HGSOC patients with HR deficiency, including identification of subclonal resistance mechanisms that cannot be detected through bulk sequencing approaches.



 Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne Willis, Joy Hendley, AOCS Study Group, Timothy Semple, Alicia Oshlack, Elizabeth L. Christie. Characterizing resistance at single cell resolution in HR deficient HGSOC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc7b444080721cb95cb0b3c57900c2232de880b" target='_blank'>
              Abstract PR019: Characterizing resistance at single cell resolution in HR deficient HGSOC
              </a>
            </td>
          <td>
            Lauren Tjoeka, Stuart Bencraig, David Yoannidis, Madelynne O. Willis, J. Hendley, Aocs Study Group, T. Semple, Alicia Oshlack, Elizabeth L. Christie
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b535784b00dc4f23ce87d1f10a7f009abae40d" target='_blank'>
              Extensive and differential platinum chemotherapy mutagenesis in children
              </a>
            </td>
          <td>
            Anna Wenger, H. Lee-Six, Manas Dave, M. Layeghifard, A. Lawson, Federico Abascal, Pantelis A. Nicola, T. D. Treger, Toochi Ogbonnah, Conor Parks, Thomas R. W. Oliver, Jonathan Kennedy, A. Hodder, Nathaniel D Anderson, Felipe Luz Torres Silva, Mi K Trinh, Thomas Dowe, Marwo Habarwaa, James J. Sun, Sergio Assia-Zamora, M. Cortes-Cerisuelo, W. Jassem, Charlotte Town, Anil Dhawan, Vandana Jain, Karin Straathof, M. Deheragoda, I. Martincorena, Liina Palm, J. C. Hutchinson, Tim H H Coorens, Claire Trayers, Nigel Heaton, A. Shlien, Yoh Zen, Foad J. Rouhani, Sam Behjati
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Nucleoside analogues are used as drugs and as labels in lab-based research. However, the effect of different nucleoside analogue mechanism(s) on cell sensitivity or mutagenesis is unclear. This is particularly important in cancer treatments where checkpoint proteins and DNA damage factors are often mutated. We tested 6 nucleoside analogues in the fission yeast, Schizosaccharomyces pombe. We found that the mutations in the DNA replication checkpoint cause unique sensitivity profiles towards chemotherapeutic nucleoside analogues (gemcitabine, 5-fluorouracil, cytarabine) and the non-clinical analogue bromodeoxyuridine. Antiretroviral compounds, zidovudine and lamivudine, did not alter cell growth. We compared half-maximal inhibitory concentration (IC50) doses between checkpoint deficient yeast strains, examining culture growth and DNA mis-segregation. Intriguingly, gemcitabine and bromodeoxyuridine doses above the IC50 promoted better growth. Above each compound’s IC50 dose we saw that cells were insensitive to nucleoside analogue re-exposure, particularly in DNA replication checkpoint mutants (cds1Δ, rad3Δ). Thus, pairing nucleoside analogue use with personal genomics may inform drug choice, dose, and schedule. Finally, these data indicate that resistance may be predictable, informing clinical strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea9673674f99d856da45d7531d1ed654d61e6d4f" target='_blank'>
              Checkpoint-Dependent Sensitivities to Nucleoside Analogues Uncover Specific Patterns of Genomic Instability
              </a>
            </td>
          <td>
            Zainab Burhanuddin Kagalwala, Mohammed Ayan Chhipa, Zohreh Kianfard, Essam Karam, Sirasie P. Magalage, Sarah A. Sabatinos
          </td>
          <td>2025-07-14</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 CDK4/6 inhibitors exhibit therapeutic potential in several tumor entities. Their clinical efficacy in glioblastoma might be limited by intrinsic and acquired resistance mechanisms. Understanding underlying genetic and molecular factors of this resistance might inform the design of effective therapeutic strategies. Genome-wide CRISPR-Cas9 screens offer a powerful approach to systematically identify resistance drivers and synthetic lethal interactions, paving the way for rational combination therapies.



 We conducted genome-wide CRISPR-Cas9 knockout and activation screens in human glioma cell lines LN229 and GS9 under CDK4/6 inhibition using Brunello and Calabrese libraries. Top hits were validated using functional assays. Furthermore, we conducted bulk RNA sequencing of treated and untreated cells for transcriptomic insights into drug response modifiers and molecular pathways involved.



 Functional genomics identified Ambra1 and CCNE1 as key drivers of resistance. Both Ambra1 knockout and CCNE1 overexpression, conferred resistance across both long-term and stem-like glioma models. Additionally, knockout of CHEK1 and FAM122A uncovered synthetic lethal interactions with CDK4/6 inhibition. Functional validation confirmed that glioma cells rely on CHEK1 or FAM122A for cell cycle progression, with cell line specific sensitivity. Transcriptomic analyses consistently highlighted alterations in pathways associated with cell cycle regulation, DNA replication, and the Fanconi anemia pathway. Of note, several differentially expressed genes overlapped with screen hits, supporting the robustness of our findings. Notably, pharmacological inhibition of CHEK1, both in vitro and ex vivo, enhanced sensitivity to CDK4/6 inhibition in a sequence-dependent manner, indicating a critical role of the sequence of drug administration.



 Our functional genomics approach uncovers mediators of acquired resistance and synthetic lethality in the context to CDK4/6 inhibitors in glioma. Co-targeting of CHEK1 or FAM122A with CDK4/6 inhibitors represents a promising therapeutic strategy to overcome resistance and improve the therapeutic efficacy. Ongoing in vivo studies using xenograft and syngeneic glioma models aim to validate the survival benefit of these combinations and to elucidate their impact on the tumor microenvironment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfaeb6b136b57493744dd9b571df497dca29c55b" target='_blank'>
              OS05.8.A DISSECTING RESISTANCE TO CDK4/6 INHIBITION IN EXPERIMENTAL GLIOMA VIA GENOME-WIDE CRISPR-CAS9 SCREENING
              </a>
            </td>
          <td>
            S. Surender, L. Haeusser, L. Kuhlburger, F. Tsiami, L. Maise, S. Nahnsen, S. Beck, D. Merk, G. Tabatabai
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e33fb7e42f73e65912640c7f54217de318d7753d" target='_blank'>
              Base-editing a single missense mutation in A20 enhances CAR-T cell efficacy
              </a>
            </td>
          <td>
            Adam Blaisdell, Stefanie Bachl, Luis R. Sandoval, Carter Ching, C. Bowman, Nupura Kale, Manu Prabandham, Morgan Diolaiti, Claire Havig, Rommel Advincula, Nika Lenci, Zhongmei Li, Emily Yamashita, Charlotte H. Wang, Shimin Zhang, Qi Liu, Philip Achacoso, Dorothea Stibor, Inger Øynebråten, Jin Seo, Alan Ashworth, Alex Marson, C. Ye, B. Malynn, Justin Eyquem, J. Carnevale, Averil Ma
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Microsatellite-stable metastatic colorectal cancer (MSS mCRC) is currently treated with chemotherapy and targeted agents based on RAS and BRAF mutational status. Although these therapies offer initial benefit, most patients rapidly develop resistance, with fewer than 20% remaining progression-free at two years. This review aims to synthesize emerging evidence on the metabolic mechanisms driving treatment resistance in MSS mCRC, with a particular focus on the immune-metabolic signature (IMMETCOLS) classification. We conducted a comprehensive review of preclinical models, transcriptomic datasets, and clinical trial results addressing metabolic adaptations to chemotherapy and targeted therapies in MSS mCRC. The IMMETCOLS framework defines three metabolic subtypes—IMC1, IMC2, and IMC3—each associated with distinct resistance mechanisms. IMC1 exhibits glycolysis and transforming growth factor-β (TGF-β)-dependent signaling enriched in inflammatory fibroblasts, conferring resistance to chemotherapy. IMC2 relies on oxidative phosphorylation and glutamine metabolism, supporting antioxidant defenses and resistance to both cytotoxic agents and anti-EGFR therapies. IMC3 demonstrates lactate-fueled respiration and pentose phosphate pathway activation, contributing to redox balance, DNA repair, and resistance to targeted therapies such as anti-BRAF or KRAS inhibitors. All subtypes display metabolic plasticity under therapeutic pressure. Emerging clinical data support tailoring targeted therapy combinations based on IMMETCOLS subtype, particularly in BRAF- and HER2-positive populations. Understanding subtype-specific metabolic rewiring in MSS mCRC offers novel opportunities to overcome drug resistance. Targeting the metabolic vulnerabilities defined by the IMMETCOLS signature may improve response durability and inform precision treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/662ed873ba87c81960ffe96e54fa075a4697d04f" target='_blank'>
              Metabolic plasticity drives specific mechanisms of chemotherapy and targeted therapy resistance in metastatic colorectal cancer
              </a>
            </td>
          <td>
            Mariam Rojas, Malena Manzi, S. Madurga, Fernando Enrique García Velásquez, Maira Alejandra Romero, Silvia Marín, Marta Cascante, Joan Maurel
          </td>
          <td>2025-09-23</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b62a398f1d31342054fedad92398759dd5022c8" target='_blank'>
              CRlSPR/Cas9 screening revealed BlRC6-AS1/BlRC6 mediates abiraterone resistance via NHEJ pathway-dependent A20 degradation in prostate cancer
              </a>
            </td>
          <td>
            Lan Li, Xue-Ni An, Yong-Tong Ruan, Rong Yang, Peng Li, Xiao-Yu Wu, , Ping Gao, 
          </td>
          <td>2025-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14bb77c980017aee37cb31324f45fa9fe45acb96" target='_blank'>
              Moonlighting Role of Meiotic SYCP1 in Breast Cancer: A Chromatin-Bound Regulator of DNA Repair, Transcription, and Drug Resistance
              </a>
            </td>
          <td>
            LC Brennan, OV Grinchuk, M. Pachon, IF Sou, C. Fawcett, CG Nogueira, M. Guthrie, AD Bates, M. Hine, A. Thomaz, Z. Hu, AB Fielding, OR Davies, W-W Tee, UL McClurg
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite extensive prior research on prostate cancer (PCa) transcriptomics, the molecular mechanisms underlying the disease’s progression, particularly in the castration-resistant or metastatic stages, remain incompletely understood. The majority of recent research has concentrated on bulk RNA sequencing, which could mask the variation found in tumor microenvironments. This study aims to address this gap by integrating single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing with weighted gene co-expression network analysis (WGCNA) to investigate the molecular mechanisms of PCa at a higher resolution. In order to further individualized treatment plans for PCa, we aim to discover important genes and signaling pathways that could be used as therapeutic targets. We first preprocessed expression profile data from prostate cancer tissue samples, selecting 9,809 high-quality cells from a dataset. Following batch correction with Harmony and dimensionality reduction with principal component analysis (PCA), we used the Louvain clustering algorithm to divide the cells into discrete subtypes. The clusters were then visualized using t-SNE. This resulted in 16 cellular subtypes categorized into five major cell types: epithelial cells, monocytes, endothelial cells, CD8 + T-cells, and fibroblasts. Analysis of receptor-ligand pairs uncovered significant interactions between monocytes and both tumor cells and endothelial cells. Applying the high-dimensional WGCNA (hdWGCNA) method to construct a gene co-expression network, we detected seven gene modules, four of which were highly expressed in tumor cell subtypes and contained 380 key genes. Combining pathway analysis, we ultimately screened six key genes: CNPY2, CPE, DPP4, IDH1, NIPSNAP3A, and WNK4. We used Cox univariate regression and least absolute shrinkage and selection operator (lasso) regression techniques to build a prognostic prediction model that included these six important genes based on clinical data gathered from PCa patients. The prognostic prediction model constructed in this study demonstrated excellent predictive performance in both the training set and an external validation set, with the high-risk group showing significantly lower overall survival (OS) than the low-risk group. Furthermore, there was a substantial correlation found between risk scores and several immune-related gene sets, chemotherapeutic drug sensitivity, and tumor immune infiltration. High- and low-risk groups exhibited significant differences in immune cell content, immune factor levels, and immune dysfunction. Further analysis revealed significant correlations between the expression levels of model genes and multiple disease-related genes. Through Gene Set Variation Analysis (GSVA) and Gene Set Enrichment Analysis (GSEA), we uncovered perturbations in multiple signaling pathways in high- and low-risk groups, potentially impacting the prognosis of PCa patients. This study uncovers key genes and signaling pathways in the prostate cancer tumor microenvironment, particularly genes such as CNPY2, CPE, DPP4, IDH1, NIPSNAP3A and WNK4, which have potential as therapeutic targets. Our findings provide new insights into personalized treatment strategies for PCa and warrant further clinical validation in the future. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15682-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779fa642ce8a3880c524211b95657271b91e6425" target='_blank'>
              Integrative analysis of molecular mechanisms in prostate cancer via single-cell RNA sequencing and weighted gene co-expression network analysis
              </a>
            </td>
          <td>
            Jing Zhai, Yizhou Wang, Yu Zhang, Wenhui Zhu, Xinyu Xu, Yu Peng, Guanxiong Ding
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Polymerase theta (POLQ) is essential for theta-mediated end joining (TMEJ), an error-prone DNA repair pathway that is hyperactive in homologous recombination-deficient (HRD) cancers such as high-grade serous ovarian cancer (HGSOC). Inhibiting POLQ is a promising therapeutic strategy, but the molecular determinants of sensitivity and downstream mechanisms remain incompletely defined.



 We assessed the growth-inhibitory activity of two structurally distinct POLQ inhibitors, novobiocin and ART558, across 18 well-characterized HGSOC cell lines. Genomic profiling data were integrated with drug response values (GR50s) to identify predictive biomarkers, including mutational status, HRD scores, and copy number signatures (CX1–CX17). Mechanistic assays included POLQ ATPase activity, cell cycle analysis (FUCCI), immunofluorescence for γH2AX and POLQ, apoptosis markers and DNA fiber assays.



 POLQ inhibitor sensitivity varied independently from olaparib or carboplatin response, indicating a distinct mode of action. Novobiocin sensitivity significantly correlated with CX3 and CX5 copy number signatures, which reflect impaired homologous recombination and replication stress. In contrast, resistant lines were enriched for CX14 and CX6, signatures linked to mitotic errors and telomere shortening. Sensitive cell lines showed increased DNA damage (γH2AX foci), POLQ accumulation on chromatin, and apoptosis, particularly in BRCA1/2-mutant backgrounds. Cell cycle profiling revealed impaired G1/S progression and S-phase depletion. DNA fiber assays demonstrated replication fork degradation upon POLQ inhibition, consistent with POLQ’s role in fork protection.



 We identify genomic features, specifically CX3 and CX5 copy number signatures as candidate biomarkers of POLQ inhibitor sensitivity in HGSOC. Mechanistically, POLQ inhibition induces replication stress, persistent DNA damage, and apoptosis in HR-deficient settings. These findings support the clinical development of POLQ inhibitors and rational patient stratification based on genome instability signatures.



 Aleksandra Kruglikov. POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe824b1195032b879fa6dbdb3ed12cc08d24058" target='_blank'>
              Abstract A037: POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response
              </a>
            </td>
          <td>
            Aleksandra Kruglikov
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f04f35397873ebcf0a05890d5a35b22af43957" target='_blank'>
              LncRNA LINC00941 Links Oncogenic KRAS Signaling to Aggressiveness and Chemoresistance in Pancreatic Cancer
              </a>
            </td>
          <td>
            Thalita Bueno Corrêa, Gabriel Lucas da Fonseca, Rauni Borges Marques, S. M. Gomes-Filho, Lutero Hasenkamp, Y. Magalhães, L. Russo, N. Hoch, F. Forti, Elena Levantini, E. M. Reis, D. S. Bassères
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Glioma, the most prevalent primary brain tumor, exhibits dysregulated ribosome biogenesis closely linked to malignant behavior. However, the role of ribosome biogenesis in glioma and prognosis remains incompletely understood. This study aimed to construct a molecular signature based on ribosome biogenesis-related genes to predict patient survival and therapeutic response in glioma. Methods Utilizing The Cancer Genome Atlas (TCGA) glioma cohort data, we constructed a ribosome biogenesis-related genes (RBRGs) signature using LASSO regression and multivariate Cox analyses, and subsequently validating its prognostic value in independent cohorts. We systematically evaluated the signature’s associations with clinicopathological features, tumor immunity, genomic instability, tumor stemness, and therapeutic sensitivity. The oncogenic role of the key gene UTP20 was experimentally validated in U87 and U251 glioma cell lines through MTS, colony formation, and transwell assays. Results We established a four-gene RBRGs signature (NOP10, UTP20, SHQ1, and PIH1D2). Elevated RBRGs score significantly correlated with shortened overall survival and adverse clinical characteristics, including advanced age, high WHO grade, IDH wild-type status, and absence of 1p/19q codeletion. A nomogram incorporating the RBRGs score demonstrated excellent predictive performance (C-index = 0.841). RBRGs-associated genes were enriched in immune regulatory pathways. The high-risk group exhibited increased infiltration of immunosuppressive cells (macrophages, myeloid-derived suppressor cells [MDSCs], and cancer-associated fibroblasts [CAFs]), upregulation of immunosuppressive checkpoints, and resistance to immunotherapy. Furthermore, the RBRGs signature correlated with genomic alterations, heterogeneity, tumor stemness, and therapeutic sensitivity. Crucially, UTP20 knockdown significantly suppressed glioma cell proliferation and invasion in vitro. Conclusion The RBRGs signature was successfully developed and validated as an independent prognostic biomarker and predictor of therapeutic response in glioma, highlighting its extensive association with tumor heterogeneity. Furthermore, this study identified UTP20 as a key oncogenic driver that facilitates glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9861e316b5d2145ed1a2316d77e1536ce5d1fb6d" target='_blank'>
              Ribosome biogenesis-related gene signature predicts prognosis and immune landscape in glioma and identifies UTP20 as a therapeutic target
              </a>
            </td>
          <td>
            Yadan Li, Xiaolong Tang, Wenhui Zhao, Xiuwen Si, Yanfang Cui, Yanbin Dong, Yongshuo Liu
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d2cfb28da058ae0d8b29e85290850287448ab79" target='_blank'>
              Transition between cell states of sensitivity reveals molecular vulnerability of drug-tolerant cells.
              </a>
            </td>
          <td>
            L. Peyre, Marielle Péré, Mickael Meyer, Benjamin Bian, Marina Moureau-Barbato, Walid Djema, B. Mari, Georges Vassaux, Jérémie Roux
          </td>
          <td>2025-10-01</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Purpose The alternative splicing of mRNA precursors allows one gene to yield multiple proteins with distinct functions. CDC-like kinases serve as pivotal regulators of alternative splicing. Control of protein expression also occurs at the level of DNA through histone methylation and demethylation. We investigated the activity of two CLK inhibitors, cirtuvivint and CC-671, and the LSD1 inhibitor iadademstat alone and in combination with anticancer drugs or investigational agents. Methods Well-characterized patient-derived cancer cell lines from the PDMR (https://pdmr.cancer.gov/models/database.htm) were used along with standard human cancer cell lines. Multi-cell type-tumor spheroids were grown from a ratio of 6:2.5:1.5 malignant cells, endothelial cells, and mesenchymal stem cells. Following three days of growth, the spheroids were exposed to the single agents or combinations at concentrations up to the clinical Cmax value for each agent, if known. After seven days of exposure, cell viability was assessed using the CellTiter-Glo 3D assay and spheroid volume was assessed by bright field imaging. Results Several of the targeted oncology drugs exhibited additive and greater-than-additive cytotoxicity when combined with a CLK inhibitor, or the LSD1 inhibitor. These agents included the XPO1 inhibitor, eltanexor, and the KRAS G12D specific inhibitor MRTX-1133 which had activity in tumor lines harboring the KRAS G12D mutation. LSD1 inhibition was effective with ubiquitin proteasome pathway inhibitors. Conclusion: These findings may provide guidance for development of clinical trial combination regimens including cirtuvivint, CC-671 or iadademstat. Full data sets are available on PubChem.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd17fd73c746ccc133073188d85cd1b611794f0" target='_blank'>
              RNA processing kinase inhibitors and epigenetic inhibitors in combination with oncology drugs or investigational agents in multi-cell type patient-derived tumor cell line spheroids
              </a>
            </td>
          <td>
            Beverly A. Teicher, T. Dexheimer, Thomas Silvers, N. Coussens, Eric M Jones, Steven D. Gore, Mark W. Kunkel, James H. Doroshow
          </td>
          <td>2025-09-12</td>
          <td>Cancer Chemotherapy and Pharmacology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcb21ab8df09d5daf66bd07fdf4abe868e528e74" target='_blank'>
              RIPPLET: Mutation-Only Gene and Pathway Profiling for Precision Oncology
              </a>
            </td>
          <td>
            Fabius Wiesmann, Cassandra Litchfield, M. Mayoux, Tumor Profiler Consortium, A. Wicki, Burkhard Becher, V. Koelzer, H. Moch, Marija Buljan, Sònia Tugues, Bettina Sobottka
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment. Methods We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks. Results SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes. Conclusion Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            X. Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Estrogen receptor alpha (ERα)-positive (ER+) breast cancers are driven by 17β-estradiol (E2) binding to ERα, which transcriptionally regulates downstream target genes. Although microarrays and conventional RNA sequencing have identified E2 target genes, pre-designed probes and short read lengths are limited in accurately capturing complex transcript structures. Long-read RNA sequencing offers a solution by spanning entire transcripts, providing a more complete view of the transcriptome. Here, we employed nanopore long-read direct RNA sequencing (DRS) complemented with 3’-end sequencing, in vitro experiments, and deep learning-based protein modeling to explore the intricate landscape of E2-responsive transcriptome and protein level implications. Our analysis revealed a range of E2-responsive non-coding and coding isoforms, including intronically polyadenylated (IPA) mRNAs. One of these IPA isoforms was detected for TLE1 (Transducin-like enhancer protein 1), which positively assists ERα-chromatin interactions for a subset of E2 target genes. The IPA isoform produces a C-terminus truncated protein, lacking the WDR interaction domain, but retains dimerization/tetramerization capacity through its intact N-terminus Q-domain. Structural modeling and protein-based assays confirmed the truncated protein’s dimerization potential and nuclear localization. Functional assays showed that overexpression of truncated TLE1 reduces the E2-induced upregulation of GREB1, an E2-responsive gene, thereby disrupting transcriptional regulation. Importantly, a lower IPA isoform ratio is associated with worse survival in ER+ patients, highlighting clinical relevance. Our study uncovers new layers of complexity in the E2-regulated transcriptome, providing insights into truncated proteins. These findings contribute to a deeper understanding of gene regulation and may help the development of new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0bac48baf3ce0314e9eaa5b424b9e568bd79fa" target='_blank'>
              E2-regulated transcriptome complexity revealed by long-read direct RNA sequencing: from isoform discovery to truncated proteins
              </a>
            </td>
          <td>
            Didem Naz Dioken, Ibrahim Ozgul, Irem Yilmazbilek, Elanur Almeric, I. C. Eroglu, Mustafa Cicek, Utku Cem Yilmaz, Deniz Karagozoglu, Esra Çiçek, E. Karaca, Tolga Can, A. E. Erson-Bensan
          </td>
          <td>2025-08-02</td>
          <td>RNA Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35317e7a4cee78e23641ba79f278922203278977" target='_blank'>
              A chemical-genetic interaction between PAF1 and ENL/AF9 YEATS inhibition
              </a>
            </td>
          <td>
            Paige A. Barta, Leopold Garnar-Wortzel, T. Bishop, Rachel E Hayward, Lauren M. Hargis, James B. Shaum, H. Kwok, Brian B Liau, Benjamin F. Cravatt, Michael A. Erb
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f58efafad79666afbd80c8e6304e3d7db77f3c" target='_blank'>
              Whole genome CRISPR knockout screen reveals ID3 as a key regulator of myeloma cell survival via TCF3 and c-MYC
              </a>
            </td>
          <td>
            Clara Andersson-Rusch, Md Abu Hanif, Ingrid Quist-Løkken, P. Sætrom, Miłosz Roliński, Johanna Fredrikke Nordstrand Møen, Naiara Campos Roman, K. Misund, V. Beisvåg, P. A. Aas, Barbara van Loon, T. Holien
          </td>
          <td>2025-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c454f3d7e351cc169def09b1bb6a18830063a225" target='_blank'>
              PAN-CANCER ANALYSES IDENTIFY ONCOGENIC DRIVERS, EXPRESSION SIGNATURES, AND THERAPEUTIC VULNERABILITIES IN RHO GTPase PATHWAY GENES
              </a>
            </td>
          <td>
            Rubén Fernández, L. F. Lorenzo-Martín, Víctor Quesada, Xosé R. Bustelo
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e980a76e192e63af260762d40719bac841d722f3" target='_blank'>
              REV1 inhibition enhances trinucleotide repeat mutagenesis
              </a>
            </td>
          <td>
            Ava Siegel, Daniel Almstead, Naveen Kothandaraman, Jessica Reich, Erica N. Lamkin, Josh A Victor, Aarzoo Grover, Kanayo Ikeh, Hannah Koval, Andrew Crompton, Hongjun Jang, Hyejin Lee, Roxana Del Rio Guerra, Dmitry M Korzhnev, M. Kyle Hadden, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-14</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d545356a888be0cc54d6000ee5eedd795258580" target='_blank'>
              SpliceUp: Predicting SF3B1 mutations in individual cells via aberrant splice site activation from scRNA-seq data
              </a>
            </td>
          <td>
            Maksim Kholmatov, K. D. Prummel, B. Guezguez, Eva Hellström-Lindberg, Pedro L. Moura, Judith B. Zaugg
          </td>
          <td>2025-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a leading cause of global cancer-related mortality, necessitating the identification of novel therapeutic targets. Integrating genetic and transcriptomic data may reveal key molecular drivers of CRC progression and treatment opportunities. Methods We performed a multiomics analysis combining genome-wide association study (GWAS) data (p < 1e-6) and RNA-seq data from the TCGA. Differential expression analysis (Limma) identified 24 consistently dysregulated genes (17 mRNAs, 7 lncRNAs) in CRC. Survival analysis was used to evaluate their prognostic impact on overall survival (OS), relapse-free survival (RFS), and post progression survival (PPS). Drug‒gene interactions were explored via Enrichr, and virtual screening (PubChem) prioritized high-affinity compounds that target PYGL, a metabolic regulator. Results Integration of GWAS and RNA-seq revealed that 24 CRC-associated genes, including PYGL, SMAD7, and TCF7L2, are involved in tumor metabolism and Wnt/TCF signaling. Survival analysis revealed that five genes (CDKN2B, BOC, METRNL, etc.) were significantly correlated with OS, RFS, and PPS. Ten small-molecule candidates targeting PYGL exhibited high binding affinity, suggesting their therapeutic potential. Conclusion This study identified CRC-linked genes through GWASs and transcriptomics, highlighting their prognostic and druggable relevance. Computational drug repurposing pinpoints PYGL inhibitors as promising candidates, offering a translational framework for CRC therapy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b980affee0b92f2b13dcf77cc903fad2021f3f0b" target='_blank'>
              Integrative GWAS and RNA-Seq analysis for target identification and virtual drug screening in colorectal cancer
              </a>
            </td>
          <td>
            Qinghui Liu, Yiyang Lei, Zixuan Liu, Jiale Han
          </td>
          <td>2025-10-27</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dynamic monitoring of epidermal growth factor receptor (EGFR) mutations is essential for the early identification of resistance and treatment adaptation. Single-cell heterogeneity analysis is crucial for precision cancer medicine, yet sensitive and specific detection methods for individual tumor cells remain challenging. Here, we develop a PCR-CRISPR/Cas12a platform enhanced by the incorporation of mismatched base in crRNA at specific site for single-cell point mutation detection. This platform demonstrated high specificity and sensitivity, detecting point mutation at a frequency of 0.1% and in as low as 1.02 ng of genomic DNA, which represents an improvement over the amplification-refractory mutation system PCR (ARMS-PCR). Notably, the accuracy of the platform is highly consistent with next-generation sequencing (NGS), as evidenced by Kappa test values surpassing 0.9. By utilizing a conical-pore membrane with optimized porosity for single circulating tumor cell (CTC) enrichment, our platform enables point mutations detection in individual tumor cells, offering potential enhancements in precision and reliability for EGFR mutation analysis. This novel methodology holds potential for more accurate and personalized cancer treatment strategies. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s00604-025-07569-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10dcb7758993fbdaa68c9e94ed0b90e6468f9a4b" target='_blank'>
              Mismatch-introduced crRNA guided PCR-CRISPR/Cas12a platform improves EGFR point mutation detection in single tumor cell
              </a>
            </td>
          <td>
            Meng Wu, Fang Wang, Yu Wang, Ya-Xian Wu, Bo-Yu Tian, Xin-Yuan Ou, Qin-Chen Xu, Xiao-Yan Wu, Chao Han, Wan-Li Liu, Shan Xing
          </td>
          <td>2025-10-11</td>
          <td>Mikrochimica Acta</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Synthetic lethal interactions are attractive therapeutic candidates as they enable selective targeting of cancer cells in which somatic alterations have disrupted one member of a synthetic lethal gene pair while leaving normal tissues untouched, thus minimising off-target toxicity. Despite this potential, the number of well-established and validated synthetic lethal gene pairs is modest. Results We generate a dual-guide CRISPR/Cas9 Library and analyse 472 predicted synthetic lethal pairs in 27 cancer cell Lines from melanoma, pancreatic and lung cancer Lineages. We report a robust collection of 117 genetic interactions within and across cancer types and explore their candidacy as therapeutic targets. We show that SLC25A28 is an attractive target since its synthetic lethal paralog partner SLC25A37 is homozygously deleted pan-cancer. We generate knockout mice for Slc25a28 revealing that, except for cataracts in some mice, these animals are normal; suggesting inhibition of SLC25A28 is unlikely to be associated with profound toxicity. Conclusions We provide and validate an extensive collection of synthetic lethal interactions across cancer types. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03737-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d43269824feb99e7e4f4f573a2dcaf7ac03c77" target='_blank'>
              A compendium of synthetic lethal gene pairs defined by extensive combinatorial pan-cancer CRISPR screening
              </a>
            </td>
          <td>
            V. Harle, Victoria Offord, Birkan Gökbağ, Lazaros Fotopoulos, Thomas Williams, Diana Alexander, Ishan Mehta, N. Thompson, R. Olvera-León, Stefan Peidli, Vivek Iyer, Emanuel Gonçalves, Narod Kebabci, Barbara De Kegel, J. van de Haar, Lang Li, Colm J. Ryan, David J. Adams
          </td>
          <td>2025-09-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Cancer can recur when a subset of tumor cells, termed persister cells, survive therapy and re-enter the cell cycle. The cellular lineages that give rise to persister cells and the mechanisms that confer the persister state remain poorly understood. Through single-cell multiomic profiling (snRNA-seq and snATAC-seq) on a cohort of (1) non-malignant fallopian tube, (2) treatment-naïve, and (3) neoadjuvant chemotherapy (NACT)-treated high-grade serous ovarian cancer (HGSOC) patient samples, we identified an epigenetic signature that defines the chemotherapy-tolerant persister state. The changes in chromatin accessibility characterizing the signature were identified in residual NACT tumors but are also present in treatment-naive samples from patients who later developed resistance. Furthermore, this epigenetic signature independently predicted chemotherapy response in patient-derived xenograft models of HGSOC and in a separate patient cohort. Cells enriched in the persister state arose from multiple lineages and displayed activation of oncogenic pathways, including altered stress responses, epithelial to mesenchymal transition, and changes to the cell cycle promoting quiescence. Finally, we identified a subset of genes that are epigenetically primed for expression before treatment and are upregulated after treatment. These findings suggest that an intrinsic epigenetic program primes tumor cells towards chemotherapy tolerance and reveal new vulnerabilities that could be exploited to delay or prevent cancer recurrence.



 Mihai G. Dumbrava, Wazim M. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. John Weroha, Alexandre Gaspar-Maia. Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a450af46a8e3f0b0c1a2127ce5282e5d4428e5a8" target='_blank'>
              Abstract PR008: Single cell multiomic analysis of high-grade serous ovarian carcinoma reveals an intrinsic epigenetic program that primes chemotherapy tolerance in persister cells
              </a>
            </td>
          <td>
            Mihai G. Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohammed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Genetic instability is a hallmark of cancer cells. Homologous recombination (HR) plays a pivotal role in maintaining genome stability through its DNA repair and replication fork escort functions. Therefore, HR is classified as a tumour suppressor pathway. Consistently, many HR genes are mutated in cancer, especially in hereditary breast and ovarian cancer. However, although RAD51 controls the central steps of HR, no RAD51 mutations are associated with cancer predisposition, constituting the “RAD51 paradox”. One of the potential explanations for the “RAD51 paradox” is that mutations affecting mediator/accessory genes (such as BRCA1 or BRCA2) in cancer result in the absence of RAD51 on damaged DNA, leaving access to alternative exclusively mutagenic repair processes, such as single-strand annealing (SSA) or alternative end-joining (A-EJ), which can rescue some cell viability but also increase genetic instability. This raises the question of whether cancer predisposition actually results from HR deficiency itself or from alternative, nonconservative repair pathways. One study assessing this question in a mouse model revealed that decreasing RAD51 HR activity without stimulating SSA or A-EJ in vivo not only does not favour tumorigenesis but rather protects against it. These data suggest that RAD51-controlled HR is not a tumour suppressor but rather favours tumour progression. Cancer cells are highly proliferative, actively replicating their genomes, and are therefore subjected to high replication stress; pathways enabling them to cope with this massive replication stress, such as HR, should help them survive and proliferate, contrary to the belief dogma that HR acts as a tumour suppressor pathway. We propose that HR/RAD51, through its essential role in overcoming replication stress, should facilitate cancer progression as soon as early pretumorigenic hyperplasia states that trigger an active replication program, challenging commonly accepted views.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5daecb61cbcd3e72a82377afb24da2ece1dd449" target='_blank'>
              Establishment of human glioblastoma cell culture collection
              </a>
            </td>
          <td>
            Hélèna L. Denis, Jaëlle Méroné, V. Watters, V. Fort, Gabriel Khelifi, Mikalie Lavoie, Line Berthiaume, Félix Rondeau, Émy Beaumont, Karine Michaud, M. D'Astous, S. Saikali, Marc-Étienne Huot, Étienne Audet-Walsh, Maxime Richer, Samer M. I. Hussein
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 The formation of cancer and responsiveness to therapy is governed by a constantly evolving balance between somatic variation and dynamic epigenetic regulation. This evolution has been recognized as a significant obstacle challenging the development of durable cancer therapies. Thus, targeting epigenetic reprogramming in human cancers is a key focus for the design of new drugs to potentially deliver robust and durable anti-tumor activity. Lysine acetyltransferases (KATs) are a promising class of epigenetic targets that operate within interchangeable multi-subunit complexes to regulate cell cycle, pluripotency, and lineage determination. Within this class, KAT6A and KAT7 have been recognized as mechanistically intertwined modulators of stem cell identity and lineage commitment through partially overlapping control of chromatin organization specified by histone H3 acetylation of lysine 23 (H3K23) and lysine 14 (H3K14). Multiple lines of orthogonal evidence point to a “paralog” genetic relationship between these enzymes to support cancer cell autonomous survival in lineage-enriched CCLE subsets. Moreover, recent preclinical findings in pediatric and adult leukemia indicate the need to inhibit both KAT6 and KAT7 to deliver durable anti-tumor responses. These observations prompted evaluation of concomitant inhibition of KAT6 and KAT7 as a therapeutic approach in biomarker-defined solid cancers. Here, we report an equipotent KAT6/KAT7 dual inhibitor with high selectivity versus the structurally similar KAT family members KAT5/KAT8. We observed synergistic impact on cell viability and pharmacodynamic (PD) modulation of H3K23/H3K14 acetylation in biomarker positive breast, lung, and ovarian cancers with dual KAT6/7 inhibition versus selective inhibition of KAT6 or KAT7. Quantitative transcriptomic and epigenomic profiling of cancer cell responses to dual KAT6/7i revealed global inhibition of H3K23 and H3K14 acetylation as well as local suppression of H3K27ac-dependent enhancer elements at gene promoters controlling lineage specification, stemness, and WNT signaling. In vitro single cell RNA sequencing and protein expression analysis of ER+ PDX derived breast cancer cells indicated that dual KAT6/7 inhibition can overcome KAT6i associated adaptive drug resistance, downregulating drug tolerant persister cell states and metabolic adaptation. In vivo efficacy studies in biomarker positive breast and lung CDX models confirmed enhanced tumor growth inhibition and PD modulation for dual KAT6/7 versus selective KAT6 inhibition. Notably, dual KAT6/7 inhibition promoted durable and stable anti-tumor responses in multiple standard-of-care resistant and ESR1-mutant breast cancer PDXs that could not be achieved with KAT6 inhibition alone. In vivo bulk transcriptomics analysis of dual KAT6/7 treated PDX tumor samples highlighted disruption of lineage identity and metabolic rewiring. These observations indicate equipotent inhibition of KAT7 and KAT6A/B has the potential to deliver deep and durable anti-tumor activity in biomarker-selected patient populations.



 Manav Gupta, Scott R. Tyler, Nour Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, Kelly S. Trego, Mason Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, Jay Prakash. Jain, Michael E . Dalziel, Julie Deichert, Sunjay Sethi, Ivan G. Shabalin, Rebeca Choy, Diana M. Munoz, Christian R. Frey, Peter Teriete, Josh Taygerly, Yuchen Bai, Richard Zang, Xin Linghu, Claire L. Neilan, Paul A. Barsanti, Michael A . White, Brian T. Jones. Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7cc963d6c0afb39c048f1457ab9b594f9c49864" target='_blank'>
              Abstract C022: Dual KAT6/7 inhibition delivers robust monotherapy anti-tumor activity in biomarker selected indications and disrupts emergence of drug-tolerant persister cell populations
              </a>
            </td>
          <td>
            Manav Gupta, Scott R. Tyler, N. Ghaddar, Pranav Gupta, Steven L. Spivak, Katelyn N. Lukas, Carl Schultz, Anjali Bisaria, Zabrisky Roland, K. Trego, Mason J. Appel, Parker Y. Jameson, Jacob Burch-Konda, Brittany Cruzan, Angelica Gonzalez-Sanchez, Monah Khalaj, J. Jain, Michael E. Dalziel, Julie Deichert, Sunjay Sethi, I. Shabalin, Rebeca M. Choy, Diana M. Muñoz, Christian R. Frey, P. Teriete, J. Taygerly, Yuchen Bai, Richard Zang, Linghu Xin, Claire L. Neilan, P. Barsanti, Michael A. White, 
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b68316c5391afa0d45aa42e226e37099bf84d45" target='_blank'>
              Tissue specificity and chromosomal alterations shape divergent immune programs in HRD tumors
              </a>
            </td>
          <td>
            D. Gulhan, David Barras, Marco Mina, Eleonora Ghisoni, Yoo-Na Kim, V. Viswanadham, Hu Jin, Florian Huber, K. Homicsko, M. Bassani-Sternberg, G. Ciriello, Peter J. Park, Denarda Dangaj Laniti
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Genetic variation at thousands of short tandem repeats (STRs), which consist of consecutive repeated sequences of 1-6bp, has been statistically associated with gene expression and other molecular phenotypes in humans. However, the causality and regulatory mechanisms for most of these STRs remains unknown. Massively parallel reporter assays (MPRA) enable testing the regulatory activity of a large number of synthesized variants, but have not been applied to STRs due to experimental and computational challenges. Here, we optimized an MPRA framework based on random barcoding to study the impact of variation in repeat copy number on expression. We first performed an MPRA on sequences derived from 30,516 promoter-proximal STR loci along with up to 152bp of genomic context, testing 3-4 variants with differing repeat copy numbers for each locus in HEK293T cells. We identified 1,366 loci with significant associations between repeat copy number and expression, which were enriched for positive effect sizes (P=2.08e-110). We then designed a second MPRA in which we performed deeper perturbations, including systematic manipulation of the repeat unit sequence, orientation, and copy number, with 200-300 perturbations for each of the 300 loci with the strongest signals. Our results revealed that the repeat unit sequence is the primary driver of differences in the relationship between copy number and expression across loci, whereas orientation and flanking sequence have weaker effects, primarily for AT-rich repeat units. The high resolution of these perturbations enabled us to detect non-linear effects, most notably for AAAC/GTTT repeats, which emerge only beyond a certain copy number threshold. Finally, we observed that a subset of STRs in our library show expression levels that are tightly linked with predicted DNA secondary structure formation. We repeated our perturbation MPRA in HeLa S3 cells under wildtype and RNase H1 knockdown conditions, which, via reduction in RNase H1 activity, are expected to hinder resolution of R-loops. This demonstrated that associations between copy number and expression at G-quadruplex-forming CCCCG/CGGGG repeats are particularly sensitive to loss of RNase H1, providing support for an R-loop mediated mechanism for these repeats. Altogether, we establish STRs as a critical component of the non-coding regulatory grammar and provide a framework for understanding how this dynamic form of genetic variation shapes gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad23e384b94840a5506d6ec02839f2c2721fcd3" target='_blank'>
              Systematic evaluation of the impact of promoter proximal short tandem repeats on expression
              </a>
            </td>
          <td>
            Xuan Zhang, Lingzhi Zhang, Ellice Wang, Susan Benton, Eduardo Modolo, Mikhail Maksimov, Sharona Shleizer-Burko, Qingqing Gong, Catherine Wang, M. Lamkin, E. Mendenhall, Melissa Gymrek, A. Goren
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/679fc499895ef9321451ab5cf3bc0bc1060162ad" target='_blank'>
              DNA shape and epigenomics distinguish the mechanistic origin of human genomic structural variations
              </a>
            </td>
          <td>
            Nadejda B. Boev, Mark B. Gerstein, Sushant Kumar
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Adenoid cystic carcinoma of the salivary gland (SGACC) is a highly aggressive malignancy characterized by poor patient survival outcomes. While several studies have analyzed the transcriptome of the salivary gland at the bulk and single-cell level, no spatial transcriptomic analyses of this tissue have been published. Most of the existing publications on SGACC have predominantly relied on bulk and single cell RNA sequencing approaches, which do not resolve the spatially localized transcriptional heterogeneity nor have the resolution for defining molecular markers within tumor subpopulations. SGACC is clinically notable for the presence of multiple tumor clones, distinct spatial phenotypes, and its indolent yet invasive nature coupled with a high propensity for distant metastasis. These features may reflect co-expression of tumor-associated markers across diverse cellular niches, and a resultant biological complexity which causes standard treatment such as surgical resection, radiation therapy, and chemotherapy to be largely ineffective in significantly improving long-term survival, and highlights the need for more precise, targeted therapeutic strategies. Herein, we analyzed single cell (n = 4) and high-resolution spatial transcriptomics samples (n = 5) to characterize cancer cell populations in MYB- and non-MYB-expressing cell states, delineated gene expression signatures, and identified critical molecular interactions specific to SGACC. We used Visum HD to obtain spatial transcriptomics data at 2µm squared high resolution. This allowed a multi-omics approach comprising single cell and spatial transcriptomic methods to enable the discovery of novel transcriptional signatures and microenvironmental features not captured by conventional methods. Spatial mapping revealed marked cellular heterogeneity and demonstrated how tissue environments influence cellular transcriptomics. To tumor heterogeneity, we focused on tumorigenic cell populations, profiled plasma and T cell enrichment within the tumor microenvironment and identified key pathways and transcriptional drivers including the MYB-NFIB fusion underlying the tumor cluster formation. Our findings indicate an upregulation of genes involved in extracellular matrix remodeling, autophagy, and reactive stromal cell populations. We further found evidence of partial epithelial-mesenchymal transition (P-EMT) programming within MYB-expressing tumor clusters. Pathway analysis revealed that mutations in the spatial query sample prominently affect the PI3K-AKT and IL-17 signaling pathways, together with a downregulation of canonical Wnt signaling in some regions of the tissue architecture adjacent to immune cells. Collectively, these results underscore the complex regulatory landscape of SGACC and offer insights into its cellular dynamics and possible therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d936540aedb35ad533bb62a60043e3d08a9252f" target='_blank'>
              Gene-Expression Programs in Salivary Gland Adenoid Cystic Carcinoma Analyzed Using Single-Cell and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Ifeoma Ebinumoliseh, Gopikrishnan Bijukumar, Kendall Hoff, K. Brayer, Elaine L. Bearer, Scott A. Ness, Jeremy S. Edwards
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88aadf318bf49329614cbd65f8b3f0ff419cae30" target='_blank'>
              Pan-cancer analysis reveals context-dependent roles of LINE-1 ORF1p in immune regulation and copy number alterations
              </a>
            </td>
          <td>
            E. Kaparos, Wenjing Zhang, Ryusei Miyanaga, Wilson H. McKerrow, J. Boeke, Teresa Davoli, P. Mita, Kelly Ruggles, David Fenyö
          </td>
          <td>2025-10-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Radiotherapy resistance remains a major clinical challenge, largely driven by tumors’ ability to dynamically adapt through complex molecular networks. Critically, the ubiquitin system has emerged as a critical regulator of this resistance. This review examines how the ubiquitin system orchestrates radiotherapy resistance through spatiotemporal control of DNA repair fidelity, metabolic reprogramming, and immune evasion. We explore how the ubiquitin code, defined by its chain topology diversity (such as K48-linked proteolysis versus K63-mediated signaling) and crosstalk with phosphorylation, SUMOylation, and acetylation, generates diverse resistance mechanisms. These mechanisms, however, also present vulnerabilities exploitable for radio-sensitization. Notably, monoubiquitylation of both histone and non-histone protein collaboratively modulates chromatin dynamics and DNA damage responses to maintain genome integrity during radiation. Furthermore, ubiquitination critically regulates caner metabolism, reprogramming processes such as ferroptosis susceptibility, hypoxia adaptation, and nutrient flux, thereby creating targetable vulnerabilities for radio-sensitization. While targeting key E3 ligases and deubiquitinases (DUBs) shows preclinical promise, clinical translation faces obstacles including functional redundancy, unintended on-target toxicity, and adaptive tumor responses. Distinct from other post-translational modifications (PTMs), the ubiquitin system offers unique clinical advantages: its dynamic reversibility, chain topology diversity, and recent breakthroughs in targeted degradation (e.g., PROTACs) enable precise disruption of radioresistance networks. By integrating these mechanistic insights with biomarker-guided therapeutic strategies, ubiquitin-targeting agents are emerging as fundamental components of next-generation radiotherapy protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f21d3a6716308e53e33306aeb9e0bc2d5a80f91" target='_blank'>
              Decoding the ubiquitin network: molecular mechanisms and therapeutic vulnerabilities for precision radio-sensitization in cancer
              </a>
            </td>
          <td>
            Fu-Ying Zhao, Li-Ye Shi, Lu-Jiao Yu, Si-Qi Wang, Chu-Yao Sun, Na Zhang, Hua-Qin Wang
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Platinum-based chemotherapeutic drugs induce intra- and interstrand purine crosslinks that causes DNA damage and tumor cell death. While these drugs are front-line treatments for multiple human cancers – including high-grade serous ovarian cancer (HGSOC) – platinum resistance and tumor recurrence remains a major clinical challenge. Here, we report findings from both a genome-wide CRISPRi screen and an RNA-binding protein (RBP)-focused CRISPRi screen aimed at identifying modulators of sensitivity and resistance of HGSOC cells to two widely used platinum agents, cisplatin and oxaliplatin, as well as to an microtubule-targeting taxane. Using an in vitro treatment regimen designed to mimics clinical treatment protocols, we found that loss of genes in the Fanconi anemia (FA) and homologous recombination (HR) DNA repair pathways markedly increased the killing of p53-deficient HGSOC cells following treatment with cisplatin or oxaliplatin. In contrast, knockdown of genes in the nucleotide excision repair (NER) pathway—despite its known role in repairing platinum-induced intrastrand crosslinks, particularly in G1—had a significantly lesser effect. These results suggest that platinum-induced cytotoxicity in HGSOC is primarily driven by replication stress, which is exacerbated by disruption of error-free DNA repair pathways active during the S- and G2 phases of the cell cycle. Our screens also identified several known and novel RBPs as potent sensitizers to both platinum agents and taxanes. Depletion of these RBPs resulted in substantial activation of interferon signaling, a hallmark of innate immune activation and cellular stress responses. Among them, the RNA-editing enzyme ADAR1 emerged as a key modulator of drug resistance. ADAR1 catalyzes adenosine-to-inosine RNA editing, thereby altering mRNA coding potential, secondary structure, and subcellular localization. Additionally, ADAR1 has been implicated in regulating the integrated stress response (ISR), a signaling network that enables cellular adaptation to environmental stress. As part of this stress adaptation, we observed that platinum and taxol treatment caused a translational shift from canonical cap-dependent translation mediated by eIF4G1 to a cap-independent mechanism reliant on eIF4G2, supporting selective translation of stress-responsive mRNAs. Together, our findings suggest that ADAR1-dependent suppression of interferon signaling and activation of the ISR contributes to resistance to both platinum-based chemotherapy and microtubule-targeting agents, which together comprise the current standard of care for HGSOC.



 Yi Wen Kong, Erika D. Handly, Robin A. Lu, Molly A. Bird, Jack Schaeffer, Samkyu P. Yaffe, Michael B. Yaffe. CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr PR009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/912428b4f14100051b63d5819642408cb6578a72" target='_blank'>
              Abstract PR009: CRISPRi screening reveals RNA editing and the integrated stress response as major determinants of platinum and taxol sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs), particularly the HERV-K family, are increasingly recognized for their roles in cancer biology, yet the function of ERVK3-1 (HERVK_3p21.31) in liver hepatocellular carcinoma (LIHC) remains largely unexplored. Methods We analyzed transcriptomic data from the TCGA-LIHC cohort to identify differentially expressed genes (DEGs) between high and low ERVK3-1 expression groups, followed by functional enrichment analyses (GO, KEGG, GSEA), protein-protein interaction (PPI) network construction, and hub gene identification. The immunological relevance of ERVK3-1 was assessed through TIP immune cycle analysis, single-cell RNA sequencing datasets, and correlation with immune checkpoint expression. Immunotherapy responsiveness was evaluated using TIDE and TCIA databases. Results High ERVK3-1 expression was associated with enrichment in metabolic and oxidative stress-related pathways, while low expression correlated with cell cycle and DNA replication. PPI analysis revealed mitosis-related hub genes (e.g., CCNB1, CDK1). ERVK3-1 expression promoted early immune cell recruitment but was inversely correlated with later stages of the cancer immunity cycle, including immune infiltration and T-cell killing. Single-cell data showed high ERVK3-1 expression in immunosuppressive subsets, alongside positive associations with inhibitory immune checkpoints (e.g., PD-1, CTLA-4, TIM-3). High ERVK3-1 expression also correlated with greater immune evasion and reduced immunotherapy responsiveness. Conclusions ERVK3-1 plays a multifaceted role in LIHC progression, contributing to metabolic reprogramming, immune suppression, and resistance to immunotherapy. These findings highlight ERVK3-1 as a potential prognostic biomarker and therapeutic target in liver cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12985-025-02928-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186dc1e93c9e9bba5bd5b29915ff6bacc720805a" target='_blank'>
              Human endogenous retrovirus ERVK3-1 characterizes a metabolically active and immunosuppressive subtype of liver cancer
              </a>
            </td>
          <td>
            Xiaofen Wen, Shulv Weng, Minna Chen, Danxia Lin, Wenwu Xue, De Zeng, Jiaxin Shen
          </td>
          <td>2025-10-01</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CDK7 has emerged as a cancer target because of its pivotal roles in cell cycle progression and transcription. Several CDK7 inhibitors (CDK7i) are now in clinical evaluation. Identifying patients most likely to respond to treatment and early detection of tumour evolution towards resistance are necessary for optimal implementation of cancer therapies. Continuous culturing of prostate cancer cells with Samuraciclib, a non-covalent ATP-competitive CDK7i, led to outgrowth of resistant cells. These were characterised by the acquisition of a single base change in the CDK7 gene, Asp97 to Asn (D97N). Mutant cells were resistant to other non-covalent CDK7i but remained sensitive to covalent CDK7i. Cryo-EM structure and kinase ligand affinity determinations revealed reduced affinity of the CDK7-D97N mutant for non-covalent CDK7i. Remarkably, Asp97 is absolutely conserved in human CDKs, inferring its importance for the activities of all CDKs. Consistent with this, mutation of the homologous residue in CDK12 (D819N) or CDK4 (D99N) promoted resistance to drugs that inhibit these CDKs. Our findings reveal a general mechanism for acquired resistance with obvious implications for patients treated with CDK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f4e4fc33eb2e8814d38d33c359427c463ea1f89" target='_blank'>
              Resistance to CDK7 inhibitors directed by acquired mutation of a conserved residue in cancer cells
              </a>
            </td>
          <td>
            Chun-Fui Lai, Victoria I. Cushing, Ellen Olden, C. Bevan, R. C. Coombes, Basil J. Greber, L. Buluwela, Simak Ali
          </td>
          <td>2025-09-08</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Glioblastoma remains one of the most treatment-refractory solid tumors due to rapid adaptive resistance across multiple convergent molecular pathways. In particular, persistent activation of the PI3K-AKT axis combined with mesenchymal transition (EMT) reprogramming contributes to rapid failure of standard therapeutic regimens, including temozolomide, radiation, and immunotherapeutics. To address this clinical gap, we developed a synthetic RNA-guided dual-targeting system designed to simultaneously suppress both canonical PI3K-AKT activation and mesenchymal escape signatures, using strand-engineered molecules optimized for combinatorial silencing. Using patient-derived glioblastoma stem cell lines and isogenic resistance models, we performed transcriptomic profiling to identify co-expressed resistance-driving nodes. Computational integration of CRISPR perturbation datasets, RNA interference screens, and synthetic lethality maps enabled selection of targetable PI3K-AKT intermediates and EMT regulators including SOX2, TWIST1, and ZEB1. Synthetic RNA molecules were designed to co-suppress both vertical (PI3K-AKT) and horizontal (EMT plasticity) resistance axes, with in vitro validation demonstrating >70% knockdown efficiency and significant synergistic reduction in proliferation (p<0.001) across diverse resistance phenotypes. Mechanistic assays revealed reversal of mesenchymal signatures with restored epithelial markers (E-cadherin upregulation; vimentin downregulation) alongside AKT dephosphorylation. Combination treatment with low-dose temozolomide further amplified cell death, suggesting sensitization to standard chemotherapy. In vivo murine xenograft models exhibited >60% reduction in tumor volume compared to controls (p<0.01), without evidence of off-target cytotoxicity or systemic toxicity. This study demonstrates the feasibility of multi-axis synthetic RNA-guided combinatorial targeting to suppress glioblastoma therapeutic resistance. These findings support future clinical translation of modular RNA-based therapeutics capable of overcoming the heterogeneity and plasticity that characterize high-grade gliomas.



 Shivi Kumar. Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/746db514f096cab5d3002088013773d14b8c8b84" target='_blank'>
              Abstract A036: Synthetic RNA-guided combinatorial targeting of glioblastoma adaptive resistance: Dual suppression of PI3K-AKT and mesenchymal transition escape pathways
              </a>
            </td>
          <td>
            Shivi Kumar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is a multifaceted disease characterized by uncontrolled cell division resulting from substantial disruptions of normal biological processes. Central to its development is cellular transformation, which involves a dynamic sequence of events including chromosomal translocations, genetic mutations, abnormal DNA methylation, post-translational protein modifications, and other genetic and epigenetic alterations. These changes compromise physiological regulatory mechanisms and contribute to accelerated tumor growth. A critical factor in this process is the dysregulation of transcription factors (TFs) which regulate gene expression and DNA transcription. Dysregulation of TFs initiates a cascade of biochemical events, such as abnormal DNA replication, that further enhance cell proliferation and increase genomic instability. This microenvironment not only sustains tumor growth but also promotes the accumulation of somatic mutations, thereby fueling tumor evolution and heterogeneity. In this study, we employed an in silico approach to identify TFs regulating 622 key genes whose mutations are implicated in carcinogenesis. Transcriptional regulatory networks were analyzed through bioinformatics methods to elucidate molecular pathways involved in cancer development. A thorough understanding of these processes may help to clarify the function of dysregulated TFs and facilitate the development of novel therapeutic approaches designed to make cancer treatments personalized and efficacious.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020eeb95aab1edda2cdedffd56c0eb8b8019a670" target='_blank'>
              From Transcription Factors Dysregulation to Malignancy: In Silico Reconstruction of Cancer’s Foundational Drivers—The Eternity Triangle
              </a>
            </td>
          <td>
            A. Cammarota, A. Carrizzo, M. De Marco, N. Bukvic, Francesco Jacopo Romano, Alessandra Rosati, M. Chetta
          </td>
          <td>2025-10-12</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 DHX9, a multi-functional RNA helicase, plays important roles in maintaining genome stability by unwinding various nucleic acid secondary structures that may lead to replication stress and DNA damage if left unresolved. DHX9 is overexpressed in multiple cancers, with selective vulnerability to DHX9 inhibition noted in tumors characterized by genomic instability and high replicative stress. ATX-559 is an orally bioavailable, small-molecule inhibitor of DHX9 helicase activity currently in Phase 1 clinical development. ATX-559 induces robust anti-tumor activity in a variety of preclinical models with genomic instability, including those with microsatellite instability-high (MSI-H) and/or deficient mismatch repair (dMMR), BRCA alterations, and homologous recombination repair deficiency (HRD).



 ATX-559 anti-proliferative activity was tested in a multi-indication cell panel screen. Triple negative breast cancer (TNBC) and colorectal cancer (CRC) cells were used to evaluate ATX-559 mechanistic activity using immunofluorescence imaging, dot blot, and western blot assays. ATX-559 was dosed orally BID to evaluate its anti-tumor activity in CRC and TNBC cell line-derived xenograft (CDX) models, as well as a panel of 64 patient-derived xenograft (PDX) models including TNBC, Ovarian, CRC, endometrial, and gastric cancers. qPCR was used to evaluate the target engagement PD biomarker, circBRIP1.



 ATX-559 treatment leads to increased accumulation of R-loops and G-quadruplexes resulting in increased replicative stress, subsequent DNA damage, and cell death selectively in cancer models with genomic instability and replicative stress. ATX-559 sensitivity was enriched in dMMR/MSI-H cell lines including CRC. Sensitivity was also enriched in BRCA altered TNBC and HRD ovarian cancer cell lines. ATX-559 is well tolerated in vivo, resulting in robust dose-dependent tumor growth inhibition and regression selectively observed in CRC dMMR/MSI-H and BRCA altered TNBC CDX models, along with a well correlated PK/PD/efficacy relationship. Additionally, ATX-559 demonstrated enriched sensitivity in a large panel of BRCA altered TNBC (75%), dMMR/MSI-H (68%), and HRD ovarian (55%) PDX models.



 Together, these findings represent preclinical validation of DHX9 as a selective therapeutic in cancers with genomic instability and replication stress. ATX-559 is currently in a first-in-human, Phase 1/2, open-label, dose-escalation and expansion study to evaluate the safety profile of ATX-559 and to determine the recommended phase 2 dose (RP2D) and preliminary antitumor activity in patients with locally advanced or metastatic solid tumors, and molecularly defined cancers (NCT06625515).



 Sunaina Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen Lynes, Stuart Ince, Jason A. Sager, Serena J. Silver. ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2a8e9dfeed1b9e094182590e0697b62580f888" target='_blank'>
              Abstract B045: ATX-559, a first-in-class, clinical-stage DHX9 inhibitor, as a targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            S. Nayak, Jennifer Castro, Monique Laidlaw, Priya Rajaratnam, Maureen M. Lynes, Stuart Ince, Jason A. Sager, Serena J Silver
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: During cancer development, tumor cells accumulate somatic mutations, which could generate tumor-specific neoantigens. The aberrant protein can be recognized by the immune system as no-self, triggering an immune response against cells expressing this aberrant protein which could mediate tumor control or rejection. Since the expression of this mutated protein is exclusive to tumor cells, great efforts are being made to identify neoantigens of relevance in the development of new cancer treatment strategies. In comparison to adulthood tumors, pediatric malignancies present fewer mutations and thus fewer potential neoantigens. Acute lymphoblastic leukemia (ALL) is the most common pediatric malignancy worldwide that can be benefited by the identification of neoantigens for immunotherapy approaches, the landscape of neoantigens in ALL is not well known, therefore the aim of our study was to identify potential neoantigens in ALL pediatric patients. Methods: To identify neoantigens in ALL, whole-exome sequencing of matched tumor-normal cells from pediatric cases was performed, with these data HLA-I alleles predicted and somatic mutations identified to propose potential neoantigens based on binding affinity of mutated peptide-HLA-I. Results: We found a strong correlation between tumor mutational burden (TMB) and neoantigen load (p < 0.001) but no correlation with prognosis. Furthermore, TMB and neoantigens were greater in ALL patients with at least one mutated DNA mismatch repair gene (p < 0.001). Also, differences between B- and T-cell ALL were found but statistical significance did not remain after permutation. Conclusions: The presence of neoantigens in pediatric cases with ALL makes the neoantigen-based immunotherapy a promising new strategy for the treatment of this malignancy, especially for patients with relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/924894c8892c6596749a9249cd4cf79014fb16ed" target='_blank'>
              Somatic Mutations in DNA Mismatch Repair Genes, Mutation Rate and Neoantigen Load in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Diana Karen Mendiola-Soto, L. Gómez-Romero, J. C. Núñez-Enríquez, J. Flores-Lujano, E. Jiménez‐Hernández, A. Medina‐Sanson, V. Bekker-Méndez, M. Mata-Rocha, M. Pérez-Saldívar, D. A. Duarte-Rodríguez, J. R. Torres-Nava, J. G. Peñaloza‐González, L. V. Flores-Villegas, R. Amador-Sánchez, M. Velázquez-Aviña, J. A. Martín-Trejo, L. E. Merino-Pasaye, K. Solís-Labastida, R. M. Espinosa-Elizondo, C. Pérez-Amado, D. May-Hau, O. Sepúlveda-Robles, H. Rosas-Vargas, J. Mejía‐Aranguré, S. Jiménez-Morales
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming.
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite the initial responses to the tyrosine kinase inhibitor (TKI) for cancer therapy, many patients often relapse with no curative regimens available. Further, the ability to target therapeutic agents to cancer cells with appropriate doses remains challenging in the clinic, especially for leukemia. Here, we show that naïve CML cells are dynamically heterogeneous in colony formation. Larger clones expand while smaller ones diminish and eventually disappear. Compared to resistant cells, parental populations, including CD44+ stem cells, form a greater number of larger, solid spheroids. Upregulation of fat mass and obesity associated protein (FTO), an RNA N6-methyladenosine demethylase, and stem cell markers (e.g., CD44, CD133, CD25) is more obvious in resistant cells compared to parental cells. FTO inhibitors (e.g., CS1, FB23-2) appreciably impair the growth of resistant cells either alone or in combination with nilotinib. FTO protein expression is unexpectedly upregulated by CS1 or FB23-2 treatment in multiple leukemia cell lines. We then constructed RNA nanoparticles encapsulating FTO siRNAs and conjugated with anti-CD133 RNA aptamers. We showed that, compared to negative control, these nanoparticles were taken up much more efficiently by resistant cells that highly express CD133. Treatment with the CD133-guided FTO siRNA nanoparticles efficiently silenced FTO expression in resistant cells, which leads to a significant reduction in their colony and spheroid formation. These findings offer new insights into cancer drug resistance and advance the application of RNA nanotechnology for treating leukemia. The research provides a foundation for developing novel, targeted therapies for resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/610e20fd12fcf52179a62018bf638a0c7e862c87" target='_blank'>
              CD133-Guided RNA Nanoparticle Delivery of FTO siRNA Impairs Leukemia Resistance to Tyrosine Kinase Inhibitor Therapy
              </a>
            </td>
          <td>
            Huiqin Bian, Changli Zhou, Hiroaki Koyama, Wen Gao, Sicheng Bian, Tao Cheng, Xiaonan Han, William Tse, Shujun Liu, Alexander Miron
          </td>
          <td>2025-10-15</td>
          <td>RNA NanoMed</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany B. Johnson, Christopher Markovic, C. Fronick, Robert S. Fulton, Michael P. Meers, DH Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The field of oncology has been extensively studied to design more effective and efficient treatments. This review explores the advanced techniques that are transforming our comprehension of cancer and its constituents. Specifically, it highlights the signaling pathways that drive tumor progression, angiogenesis, and resistance to therapy, as well as the modern approaches used to identify and characterize these pathways within the tumor microenvironment (TME). Key pathways discussed in this review include vascular endothelial growth factor (VEGF), programmed cell death protein 1/programmed cell death ligand 1 (PD-1/PD-L1), cytotoxic T-lymphocyte-associated protein 4 (CTLA-4), and various extracellular matrix (ECM) pathways. Conventional methods of diagnosis have yielded sufficient knowledge but have failed to reveal the heterogeneity that exists within the TME, resulting in gaps in our understanding of the cellular interaction and spatial dynamics. Single-cell sequencing (SCS) and spatial transcriptomics (ST) are effective tools that can enable the dissection of the TME with the resolution capacity of a single cell. SCS allows the capture of the unique genetic and transcriptomic profiles of individual cells along with rare cell types and new therapeutic targets. ST complements this by providing a spatial map of gene expression, showing the gene expression profiles within the tumor tissue at specific sites with good accuracy. By mapping gene expression patterns at a single cell level and correlating them with the spatial locations, researchers can uncover the intricate networks and microenvironmental influences that contribute to tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfe2d5d0e8233b17d2256f6ca476fecb056c29a" target='_blank'>
              Decoding the Tumor Microenvironment: Insights and New Targets from Single-Cell Sequencing and Spatial Transcriptomics
              </a>
            </td>
          <td>
            Shriya Pattabiram, Prakash Gangadaran, Sanjana Dhayalan, Gargii Chatterjee, Danyal Reyaz, Kruthika Prakash, Raksa Arun, R. Rajendran, B. Ahn, K. N. Aruljothi
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="This article provides a comprehensive overview of targeted and epigenetic therapies for acute myeloid leukemia (AML), highlighting their role in advancing treatment strategies. Recognizing that AML arises from complex genetic and epigenetic alterations, a deeper understanding of its pathogenesis has paved the way for precision medicine. The article reviews key targeted therapies, including IDH inhibitors (e.g., ivosidenib, enasidenib) for IDH mutations, FLT3 inhibitors (e.g., midostaurin, gilteritinib) for FLT3 alterations, BCL2 inhibitors (e.g., venetoclax) impacting apoptosis, and emerging Menin inhibitors for specific subtypes like KMT2A-rearranged or NPM1-mutant AML. It also discusses epigenetic therapies that target reversible changes, such as DNMT inhibitors (e.g., azacitidine, decitabine) for DNA methylation and HDAC inhibitors for histone modifications, along with other promising agents under development targeting epigenetic regulators. Furthermore, the article explores gene therapy as a significant future direction, detailing advancements in gene editing (CRISPR/Cas9) and strategies designed to shield healthy hematopoietic cells from targeted toxicity (e.g., CD33, CD123 editing) or to enhance the anti-leukemic function of immune cells (e.g., CAR-T, TCR therapy). The future perspective emphasizes the importance of developing synergistic combination therapies and leveraging molecular insights for truly personalized treatment approaches to overcome therapeutic challenges and improve long-term outcomes for AML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1f1a005625d793e87f3fac27421531cd23f4756" target='_blank'>
              Targeted and epigenetic therapies for acute myeloid leukemia treatment
              </a>
            </td>
          <td>
            Marzieh Shokoohi, Mobin Ghaderi, Milad Mokarami, Homaira Arian, Sara Mohammadnabi, Arastoo Farokhi
          </td>
          <td>2025-09-26</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Histone acetylation is widely assumed to directly instruct gene activation. Among acetylated residues, H4K16ac is one of the most abundant modifications, conserved across all eukaryotes. Despite its established role in X-chromosome hyperactivation in Drosophila, its function in mammalian cells has remained elusive. Here, we show that in human somatic cells, H4K16ac does not substantially affect gene expression, but instead controls the spatiotemporal program of genome replication. By combining a meta-analysis of public datasets and perturbation experiments designed to minimize confounding effects, we found that H4K16ac is neither associated with nor required for transcriptional activity. Rather, H4K16ac depletion resulted in premature replication of heterochromatic regions and widespread alterations in replication timing across the genome. These defects were driven by the aberrant activation of cryptic replication origins at long terminal repeats—repetitive elements typically marked by H4K16ac and whose sequence context resembles that of canonical origins in euchromatic regions. Our findings reveal an unexpected role for one of the most prevalent chromatin modifications and uncover a new regulatory mechanism that safeguards genome replication fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d31c25c7ef175d7c8a34dda15d005ef97496d8" target='_blank'>
              Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression
              </a>
            </td>
          <td>
            Marta Milan, Valeria Runfola, Manthan Patel, Lucia Falbo, R. Noberini, C. Soriani, S. Rodighiero, Tiziana Bonaldi, Vincenzo Costanzo, Madapura M. Pradeepa, Paola Scaffidi
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 KRAS is a major oncogenic driver in pancreatic ductal adenocarcinoma (PDAC), mutationally activated in approximately 90% of cases. Mutations in this oncogene have been associated with more aggressive disease, poorer outcomes, and have remained hard to drug for over three decades. Recently developed small molecule inhibitors of KRAS (KRASi) have shown promising efficacy in advanced, previously treated PDAC patients but ultimately, most patients develop acquired resistance. Approximately half of these resistance cases do not present a putative genomic driver. In preclinical studies using genetically engineered mouse models of PDAC, we and others have demonstrated that cell state identity along the classical/epithelial-mesenchymal axis is a key determinant of response and resistance to KRASi. Notably, acute KRASi treatment initially induces a strong and selective bottlenecking of the malignant population in vivo, resulting in tumors that are predominantly classical with depleted mesenchymal characteristics. While these findings highlight a potent cell state-selective effect of acute KRASi treatment on PDAC cells, the underlying mechanisms driving the drug-induced cell state selection and plasticity remain poorly characterized. These mechanisms may reveal novel therapeutic targets for developing combination therapies that improve therapeutic responses. To comprehensively map and modulate epithelial-mesenchymal (E-M) cell state plasticity in response to KRASi treatment, we performed CRISPRi Perturb-seq (single-cell gene expression readout) with lineage tracing on a patient-derived PDAC cell line treated with the RAS(ON) multi-selective inhibitor RMC-7977. We captured 734,092 high-quality single-cell transcriptomic profiles and 156,119 unique clones across 60 genetic perturbations, including E- and M-specific transcription factors (TF) inferred to influence E-M plasticity from previous single-cell lineage tracing experiments. To optimally model clonal growth and transition rates, we profiled cells across a time series at day 6, day 15 and day 22, after 1 week of DMSO or KRASi treatment. We developed a hierarchical generative probabilistic model to jointly infer E↔M transition rates and cell state-specific growth rates, while capturing the effects of perturbations and KRASi treatment on state dynamics. Analysis of longitudinally tracked clones revealed that KRASi treatment specifically reduced the M-state growth rate while increasing the probability of M→E transitions compared to DMSO. Notably, we identified several TFs, displaying KRASi-dependent changes in state transition dynamics, either promoting (ELF3, MEIS2 among others) or decreasing (ZNF281, IRF9, among others) probabilities of M→E transitions when compared to non-targeting controls. These preclinical findings suggest that perturbation of these factors may be used as a paradigm to prevent cell state plasticity upon acute KRASi treatment and could be used to homogenize tumor populations towards a treatment-sensitive state.



 Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al'Khafaji, Kyle E. Evans, Mehrtash Babadi, Thouis R. Jones, Elisa Donnard, David T. Ting, Nir Hacohen, Dana Pe'er, Eric S. Lander, Andrew J. Aguirre, Arnav Mehta. Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B120.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/095259190df0afbd227ee5a60c0db74dfea61a15" target='_blank'>
              Abstract B120: Mapping and modulating epithelial-mesenchymal plasticity under RAS(ON) multi-selective inhibition in PDAC through lineage tracing and Perturb-seq
              </a>
            </td>
          <td>
            Julien Dilly, Mike Bogaev, Lynn Bi, Abigail Collins, Martin Jankowiak, Aziz Al’Khafaji, Kyle E Evans, M. Babadi, Thouis R. Jones, Elisa Donnard, David T Ting, N. Hacohen, D. Pe’er, Eric S. Lander, A. Aguirre, Arnav Mehta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28967441eb1f46f1a9e983b8df32cfb979b71b23" target='_blank'>
              Chromatin and gene-regulatory dynamics of human pulmogenesis by single cell multiomic sequencing
              </a>
            </td>
          <td>
            Samuel H. Kim, Soon Il Higashino, Lea C. Steffes, Li Li, Anca M. Pașca, Maya Kumar, Christin S. Kuo, Mark A. Krasnow, William J. Greenleaf
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) remains a major clinical challenge in hematologic oncology, characterized by a continuous evolution of molecular drivers that shape its heterogeneity across the age spectrum. Pediatric B-ALL is generally associated with high cure rates, while adult forms of the disease are often more aggressive and less responsive to treatment. This review examines the age-specific genetic and epigenetic landscapes that contribute to this disparity, revealing how the nature and timing of molecular alterations point to fundamentally different leukemogenic processes. Favorable genetic aberrations, such as ETV6::RUNX1 and hyperdiploidy, are predominant in children, whereas adults more frequently present with high-risk features, including BCR::ABL1 fusions and IKZF1 deletions. Epigenetic distinctions are similarly age-dependent, involving divergent patterns of DNA methylation, histone modifications, and non-coding RNA expression. For example, pediatric B-ALL frequently harbors mutations in epigenetic regulators like SETD2 and CREBBP, while adult B-ALL is more commonly affected by alterations in TET2 and IDH1/2. These molecular differences are not only prognostic but also mechanistic, reflecting distinct developmental trajectories and vulnerabilities. Understanding these age-driven transitions is essential for improving risk stratification and developing precision therapies tailored to the unique biology of B-ALL across the lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb81ee6e89e6609c04144954df008c19fbd5a755" target='_blank'>
              Age-Driven Genetic and Epigenetic Heterogeneity in B-ALL
              </a>
            </td>
          <td>
            Yoana Veselinova, M. Esteller, Gerardo Ferrer
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT The era of targeted therapies has significantly advanced our understanding of cancer growth and metastasis. Intrinsic or acquired drug resistance remains a major challenge, rendering clinically overt metastatic disease incurable in most patients. This review first examines key clinical trials and their primary outcomes involving targeted therapies in the most common advanced solid tumors, along with the main mechanisms underlying drug resistance. Recently, micrometastatic disease has emerged as a novel focus of investigation aimed at definitively curing advanced solid tumors. Accordingly, this review explores the biology of micrometastases, current challenges in their detection and monitoring, and the main strategies proposed to prevent their progression. The potential roles of nanotechnology and artificial intelligence‐driven predictive models are also discussed. Furthermore, we highlight specific characteristics of micrometastatic disease that favor immune modulation, and we evaluate the effectiveness of an immunotherapy regimen that inhibits immune suppression. The lead time provided by serum tumor markers, used experimentally to better track the progression of otherwise undetectable micrometastatic disease, also forms the mechanistic basis for a novel protocol we propose to prevent relapse in high‐risk cancer patients. This innovative protocol holds scientific relevance being supported by an appropriate mathematical model and ready for immediate application in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d0a2d7b2866fae7f5ad7e2d51fefa760b9024b2" target='_blank'>
              Targeted Therapies in the Most Common Advanced Solid Tumors, Drug Resistance, and Counteracting Progressive Micrometastatic Disease: The Next Frontier of Research
              </a>
            </td>
          <td>
            Andrea Nicolini, Paola Ferrari, Roberto Silvestri, Dario A. Bini
          </td>
          <td>2025-09-27</td>
          <td>MedComm</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Cancers develop and evolve by overcoming a variety of selective pressures including cell-intrinsic growth-inhibitory mechanisms, microenvironmental stresses, and therapeutic interventions. In addition to mutational mechanisms, cancer cell adaptation via transcriptional plasticity remains an important but poorly understood driver of cancer cell fitness. For example, ctDNA analysis of disease progression samples after KRAS inhibitor (KRASi) treatment identified genomically-defined mechanisms of resistance in only approximately 50% of patients, suggesting that non-genomic mechanisms play an equally important role. Thus, there is a pressing need to devise systematic and scalable methods to determine essential drivers of prognostically-relevant pancreatic cancer cell states, and to engineer state-relevant models that can be applied to discover new therapeutic approaches that overcome these modes of resistance. Here, we present a strategy to systematically identify transcriptional drivers of inflammation- and therapy-resistant cell states in pancreatic cancer. We have established a robust screening pipeline to 1) over-express human transcription factor (TF) isoforms in a pooled format within cancer models; 2) perform positive-selection screens under a variety of selective pressures to define transcriptional mechanisms of inflammation tolerance and KRASi resistance; 3) perform single-cell RNA sequencing to define resistance phenotypes; and 4) conduct mechanistic studies to examine the downstream signaling networks that facilitate cancer cell growth in the setting of these selective pressures. We uncover TFs and states that drive resistance in a tissue- and perturbation-specific manner, as well as states that are robust to multiple different selective pressures. A subset of TFs and associated gene programs induce tolerance to chronic interferon exposure in our screens and are reflected in vivo in the setting of immunotherapy resistance. We also identify distinct TF families that drive resistance to pharmacologic KRAS inhibition. Notably, cancer cell states that confer resistance to KRAS inhibition also exhibit cross-resistance to other MAPK pathway inhibitors and to microenvironmental signals, suggesting that cell state adaption may contribute to increased cancer cell fitness and multi-drug resistance as cancers progress. More broadly, our strategy enables the systematic identification and mapping of cell states that drive cancer cell fitness to in vivo clinical atlas datasets, providing a framework for interpretation of clinical phenotypes and predictive modeling of cell state evolution. Furthermore, the cell state-defined models developed here provide a substrate for new therapeutic discovery using cancer cell states as biomarkers, a key next step to developing combination strategies that overcome therapeutic resistance and improve outcomes for pancreatic cancer patients.



 Yuzhou Evelyn. Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava Amini, Peter S. Winter, Srivatsan Raghavan. Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B103.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1dd25ee2338a106f19e6d004e36d07be5c2aef6" target='_blank'>
              Abstract B103: Defining cancer cell states that drive inflammation tolerance and KRAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andew Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c038a3eb980cd4c3376d3d5fe3b978d2d8815e95" target='_blank'>
              A Machine Learning Approach Reveals CRISPR-Cas I-F as a Genomic Marker of Antibiotic Susceptibility in Uropathogenic E. coli
              </a>
            </td>
          <td>
            Alexandra M Young, P. Humburg, Fang Liu, Michael C. Wehrhahn, A. Tay, Stephen M. Riordan, Li Zhang
          </td>
          <td>2025-10-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune checkpoint inhibitors have transformed treatment for several cancers, yet clinical trials of programmed cell death protein 1 (PD-1) blockade in glioblastoma (GBM) have consistently failed to show therapeutic benefit. While some studies have reported treatment-related transcriptional changes, particularly in T cells, findings remain limited and inconsistent. The aim of this study was to investigate changes in tumor cells and tumor-associated macrophages (TAMs) after PD-1 blockade in recurrent GBM using spatial transcriptomics. We performed Digital Spatial Profiling (GeoMx, NanoString) on FFPE tumor samples from 26 patients with matched primary and recurrent IDH-wildtype GBM, including 16 patients who received neoadjuvant nivolumab at recurrence. Tumor (SOX2⁺) and TAM (IBA1⁺) segments were selected for targeted spatial analysis. Following quality control and filtering, transcriptomic profiles were compared between nivolumab-treated and untreated recurrent tumors. PD-1 blockade did not induce detectable gene expression changes in either tumor cells or TAMs. There were no significant differences in global expression profiles or in more targeted analyses of malignant cell states, cell cycle activity, interferon signaling, or myeloid transcriptional programs. These results consistently indicate that neoadjuvant PD-1 blockade does not elicit measurable responses at the spatial transcriptomic level in tumor cells or TAMs in recurrent GBM. These findings align with the lack of clinical benefit observed in trials and highlight the need for alternative strategies to improve immunotherapy outcomes in GBM. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02937-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6af3945d99a9f12a27763251dbd874f6e32349" target='_blank'>
              Spatial transcriptomic analysis reveals lack of response to PD-1 blockade in recurrent glioblastoma
              </a>
            </td>
          <td>
            S. Artzi, M. N. Klausen, D. Harwood, S. Michaelsen, S. Maarup, A. Locallo, V. Fougner, N. Bager, N. Hammouda, D. Nørøxe, B. Hasselbalch, U. Lassen, J. Weischenfeldt, B. Kristensen
          </td>
          <td>2025-09-17</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Reprogramming gene expression at the translational level drives intestinal tumorigenesis. Codon decoding during translation elongation relies on tRNA modifications, while their pathological relevance in colorectal cancer remains to be elucidated. Here, we show that AlkB homolog 8 (ALKBH8), a uridine 34 (U34) tRNA methyltransferase, is a direct target of Wnt/β-catenin and is upregulated in colorectal cancer. Genetic ablation of ALKBH8 inhibits the development of intestinal tumors in Apcmin/+, azoxymethane/dextran sulfate sodium (AOM/DSS), and xenograft models. Loss of ALKBH8 induces ribosome pausing at adenine-ending codons, impairing the translation elongation of mRNAs enriched with these codons. Specifically, ALKBH8 regulates the translation of KRAS proto-oncogene in a codon-dependent manner. Rescue experiments demonstrate that the methyltransferase activity of ALKBH8 is required for its translation-promoting function. Together, our findings reveal ALKBH8-dependent mRNA translation as a critical mediator of intestinal tumorigenesis, underscoring its potential as a promising target for colorectal cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/741e7e4c4eea499624351c733df3953216fc634e" target='_blank'>
              ALKBH8-mediated codon-specific translation promotes colorectal tumorigenesis
              </a>
            </td>
          <td>
            Yu Qian, Canlan Wu, Saisai Wei, Sujun Yan, Junxuan Peng, Lei Yu, Yunyi Gao, Jingyu Hou, Wentao Yu, Zhanghui Chen, Jun Zhang, Xiangwei Gao
          </td>
          <td>2025-10-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db525943b0ddbcca73f6e3d8fd119fef42f8fb23" target='_blank'>
              Aberrant inheritance of extrachromosomal DNA amplifications promotes cancer evolution
              </a>
            </td>
          <td>
            Shir Marom, Inbar Lifshits Dayan, Venkata Narasimha Kadali, Mădălina Giurgiu-Kraljič, Gabriela Koifman, Karen Hakeny, Madhuri Chaurasia, Orléna Benamozig, R. Nevo, I. Azuri, Julia Ryvkin, R. Rotkopf, Gil Stelzer, Nino Oniashvili, Jacques Mardoukh, Sarah Pollock, Nika Iremadze, Z. Shipony, Meital Kupervaser, N. Wigoda, Dena Leshkowitz, Z. Elazar, Esther Berko, A. Henssen, Ofer Shoshani
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Introduction Osteosarcoma (OS) is the most common bone tumor, characterized by a high incidence, rapid progression, and frequent metastases. The implementation of chemotherapy has made important progress, while the necrosis rate is limited and the survival rates remain unsatisfactory, therefore novel approaches are needed. Methods We used proteomic analysis to characterize the molecular landscape of patients exhibiting different levels of chemotherapy-induced necrosis. Results Patients with low necrosis rate (≤70%) showed distinct expression patterns, with significant upregulation of proteins involved in DNA replication, metabolism, and mitochondrial pathway. The Runx1-related signaling pathway was also identified as potentially involved in disease progression. Remarkably, Mitochondrial Ribosomal Protein L4 (MRPL4) and Macrophage Erythroblast Attacher, E3 Ubiquitin Ligase (MEMA) were identified as hub proteins in MEGENA analysis and the public database. By integrating with immunohistochemistry, the higher expression level was verified in samples of OS patients compared to those of healthy people. Discussion Overall, our project improves the knowledge of the expression pattern with different necrosis rates of OS samples, and the findings of MRPL4 and MAEA indicate the potential role in chemoresistance and provide new targets for the therapeutic strategy for OS patients with a low necrosis rate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/807bdc16249a2e2214890d47ae8318eb43958f1d" target='_blank'>
              Identification of MAEA protein as a potential target for chemoresistance in osteosarcoma using bioinformatics and proteomic analysis
              </a>
            </td>
          <td>
            Chen Zhang, Ruizhen Wang, Xin Yi, Wannian Wang, Jing Yang, Lihua Zhang, Guibin Wang, Wei Wang
          </td>
          <td>2025-09-08</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background BRD9 is involved in multiple physiological and pathological pathways, yet its functional role and molecular mechanisms in gastric cancer (GC) remain largely unexplored. Addressing this knowledge gap is critical given the persistent global mortality burden of GC and the limited efficacy of current therapeutic strategies. Methods BRD9 expression in GC patients was systematically analyzed using immunohistochemical (IHC) assays and transcriptomic datasets. Comprehensive functional validation, employing cellular and murine tumor models, elucidated BRD9’s role in GC progression. Molecular pathways underlying BRD9-mediated gastric carcinogenesis were delineated through integrated approaches, including RNA sequencing, co-immunoprecipitation (co-IP), subcellular fractionation, and luciferase reporter assays. Results BRD9 was significantly overexpressed in GC and associated with poor patient prognosis. Functionally, BRD9 promoted GC cell proliferation and enhanced DNA damage repair capacity. Mechanistically, elevated BRD9 expression inhibited p53 nuclear translocation via direct binding, subsequently activating the E2F transcription factor family. Notably, we identified that E2F1 directly bound to and transactivated the BRD9 promoter, establishing a positive feedback loop that sustains BRD9 expression. Additionally, BRD9 knockdown sensitized GC cells to cisplatin and oxaliplatin treatment. Conclusions These findings highlight the critical role of BRD9 in GC progression and its therapeutic potential. The BRD9-p53-E2F1 axis acts as a crucial regulator of GC cell proliferation and DNA damage response. Targeting BRD9 pharmacologically could be a novel therapeutic approach to enhance chemotherapy efficacy and improve treatment outcomes in GC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02449-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dadff4be7ebdf814a21760a8db20acee07e5dfa" target='_blank'>
              BRD9-p53-E2F1 circuit orchestrates cell growth and DNA damage repair in gastric cancer
              </a>
            </td>
          <td>
            Qingqing Zhou, Qi Wang, Yantao Duan, Chi Zhang, Tengfei Liu, Hengrui Liu, Mindi Zhao, Zhihuang Hu
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="OBJECTIVE
Large-scale CRISPR screens have identified essential genes across cancer cell lines, but links between tumor functional properties and specific dependencies require investigation to reveal the mechanisms underlying dependencies and broaden understanding of targeted therapy.


METHODS
We selected 47 breast cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) with multi-omics data including gene dependency; somatic mutations; copy number alterations; and transcriptomic, proteomic, metabolomic, and methylation data. We established a dependency marker association (DMA) analytic pipeline by using linear regression modeling to assess associations between 3,874 representative gene dependencies and multi-omics markers. Additionally, we conducted non-negative matrix factorization clustering, to stratify breast cancer cell lines according to gene dependency features, and investigated cluster-specific DMAs.


RESULTS
We interpreted valuable DMAs according to two primary aspects. First, dependencies associated with gain-of-function alterations revealed addiction to lactate transporter SLC16A3, thus suggesting a promising therapeutic target. Second, dependencies associated with loss-of-function alterations included synthetic lethality (SL), collateral SL, and prioritized metabolic SL, encompassing paralog SL (e.g., IMPDH1 and IMPDH2), single pathway SL (e.g., GFPT1 and UAP1), and alternative pathway SL (e.g., GPI and PGD). DMA analysis of the two clusters with divergent dependency signatures demonstrated that cluster1 cell lines exhibited extensive metabolism with mitochondrial protein dependencies, whereas cluster2 displays enhanced cell signaling, and reliance on DNA replication and membrane organelle regulators.


CONCLUSIONS
We established a DMA analysis pipeline linking the gene dependencies of breast cancer cell lines to multi-omics characteristics, thus elucidating the underpinnings of tumor dependencies and offering a valuable resource for developing novel precision treatment strategies incorporating relevant markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/753908ccf816b6d23f787d4f37906f4b14d28c45" target='_blank'>
              Comprehensive investigation of the molecular basis of cancer dependencies suggests therapeutic options for breast cancer.
              </a>
            </td>
          <td>
            Rui Ding, Zhimin Shao, Tianjian Yu
          </td>
          <td>2025-10-23</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract BRCA1 (breast cancer-associated protein 1) plays a central role in homologous recombination (HR) through interactions with multiple proteins across its various domains. The C-terminal BRCT domains bind HR regulators such as ABRAXAS1, CtIP, and BRIP1, each contributing to distinct, sometimes opposing, functions. While pathogenic mutations frequently cluster within the canonical BRCT phospho-binding pocket, the broader mutational landscape and its functional consequences remain poorly understood. Here, we used a site-saturation mutagenesis library of the BRCT domains to test >4,000 single-residue variants for their ability to bind ABRAXAS1 and CtIP. Using a yeast two-hybrid screen, we systematically assessed these interactions and validated key findings in mammalian cells. The resulting interaction map identified previously uncharacterized residues critical for partner binding and demonstrated their detrimental impact on HR. Importantly, we identified separation-of-function mutations that selectively disrupt individual protein interactions, enabling a more detailed analysis of each partner’s contribution to HR. Functional assays suggested that disruption of CtIP binding had the most pronounced impact on HR. Furthermore, integration of our data with clinical variant data revealed a strong correlation between loss of protein binding and pathogenicity, highlighting the potential utility of our interaction map for clinical variant interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c047e85b483d99dee8271604efee8cc044f9d733" target='_blank'>
              Profiling BRCA1-BRCT interactions and their functional relevance at amino acid resolution
              </a>
            </td>
          <td>
            Venda Mangkusaputra, A. G. Murachelli, Zhengzhou Yu, Annouche den Hollander, R. Menafra, Anne Schreuder, S. Kloet, T. Sixma, Sylvie M. Noordermeer
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Glioblastoma (GBM), as one of the most common and aggressive primary brain tumors, pose significant challenges in diagnosis and treatment, highlighting the urgent need for the identification of novel biomarkers, NFKBIE, which may play a critical role in tumor progression, immune modulation, and therapeutic response. We investigated the role of NFKBIE in GBM through both bioinformatics analysis and experimental validation. Initially, bioinformatics analysis based on public databases revealed significant differential expression of NFKBIE across various cancer types, particularly in GBM, where elevated expression was correlated with poor patient prognosis, including overall survival, disease-specific survival, and progression-free survival. Furthermore, genetic alterations in NFKBIE, such as copy number variations (CNVs) and tumor mutational burden (TMB), were significantly associated with tumor progression. In the experimental validation phase, we utilized small interfering RNA (siRNA) technology to silence NFKBIE expression in GBM cell lines, including U87 and T98G. Functional assays were then performed to assess the impact of NFKBIE knockdown. Cell proliferation assays, including CCK-8 and clonogenic assays, demonstrated that silencing NFKBIE significantly inhibited the proliferation of GBM cells. Migration and invasion assays, including wound healing and Transwell assays, further confirmed that NFKBIE knockdown markedly suppressed the migratory and invasive abilities of GBM cells. Additionally, apoptosis assays, such as TUNEL staining, revealed that NFKBIE silencing significantly promoted apoptosis in GBM cells. Western blot analysis confirmed the changes in the expression of key markers associated with proliferation, stemness, migration, invasion, and apoptosis. Furthermore, in a mouse xenograft model, NFKBIE knockdown significantly slowed tumor growth. Histological analysis of the excised tumors confirmed that reduced NFKBIE expression was closely associated with the inhibition of tumor growth. Notably, NFKBIE silencing also led to the downregulation of components of the Hedgehog signaling pathway, suggesting that NFKBIE may regulate tumor progression through modulation of this critical pathway. In conclusion, high NFKBIE expression is strongly associated with GBM progression and poor prognosis, and targeting its expression may offer a promising therapeutic strategy for GBM treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14775-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567a01090acb470e17dfe52d5931751c0e81fc63" target='_blank'>
              NFKBIE as a prognostic biomarker and therapeutic target for GBM: role in Hedgehog signaling activation
              </a>
            </td>
          <td>
            Feng Mo, Yang Li, Ce Dong, Lijun Yang, Xiaoli Ma, Baogen Pan, Z. Fan
          </td>
          <td>2025-10-06</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cells arise from the cumulative acquisition of genetic and epigenetic alterations that affect vital cellular functions. Genomic instability results from deficiencies in protective mechanisms, such as cell cycle checkpoints, DNA replication control, or DNA repair. Claspin integrates a group of crucial proteins that maintain genome integrity. It participates in key cellular events such as DNA damage checkpoint activation, DNA replication, replication stress responses, DNA repair, epigenetic memory, and apoptosis. Given its crucial functions, a role for Claspin in cancer is not a surprise. Indeed, there is a considerable body of evidence linking Claspin deregulation with cancer. For instance, over-expression of Claspin and Tim promoted the survival of cancer cells by enabling adaptation to oncogene-induced replication stress. In addition, Claspin gene (CLSPN) mutations that affect checkpoint regulation have been identified in cancer patients, suggesting that they may contribute to cancer development. Changes in Claspin expression levels may be used as a prognostic marker in several types of cancer. Finally, several therapy-resistance signaling pathways seem to converge onto Claspin’s stabilization, turning Claspin into an attractive target for chemo- and radio-sensitization. In this review, we will focus on the role of Claspin in cancer and ways in which Claspin can be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86699ba9ba454ef5c75f676effbf8547c5ac5e0c" target='_blank'>
              Claspin and Cancer: Where Are We Now?
              </a>
            </td>
          <td>
            D. Azenha, Teresa C. Martins
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aadf2778bea4e8d563036eda473ff737cd085e3" target='_blank'>
              Gene-sized DNA insertion at genomic safe harbors in human cells using a site-directed transposase
              </a>
            </td>
          <td>
            James E Short, Lisa Sharek, Joshua F. Meckler, I. Stoytchev, Christopher T Tran, Clara Errard, Brian E Hew, Brandon E Johnson, David F Waller, Shanfu Xie, Joseph J Higgins, Raymond Tabibiazar, Jesse B Owens
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9095797e23ad8944a80f33c6e2aa42554f1acad9" target='_blank'>
              Uncovering Cellular Interactome Drivers of Immune Checkpoint Inhibitor Response in Advanced Melanoma Patients
              </a>
            </td>
          <td>
            Shay Ladd, Anne M. Talkington, Mary O’Sullivan, R. Barnes, Remziye E. Wessel, Gabriel F. Hanson, Sepideh Dolatshahi
          </td>
          <td>2025-09-08</td>
          <td>Cellular and Molecular Bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8768ee203fcd258706f2498c5fcf4ab4120a2eda" target='_blank'>
              Somatic evolution of prostate cancer: mutation, selection, and epistasis across disease stages
              </a>
            </td>
          <td>
            Moein Rajaei, Alexander Yang, Kira A. Glasmacher, Christopher N. Cross, Nic Fisk, Elizabeth B. Perry, Jeff Mandell, S. Gaffney, Takafumi N. Yamaguchi, J. Livingstone, Jose Costa, Peter Humphrey, Vincent L. Cannataro, P. Boutros, Jeffrey P. Townsend
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Understanding tumor heterogeneity is essential for advancing cancer treatment. Clonal reconstruction methods play a pivotal role in deciphering this heterogeneity. Our goal is to develop a clonal reconstruction approach that is clinically applicable, easy to implement, and capable of delivering both high-speed performance and excellent reconstruction accuracy. Results We present MyClone, a probabilistic method designed to reconstruct the clonal composition of tumors using deep sequencing genomic data. MyClone processes read counts and copy number information of single nucleotide variants derived from deep sequencing data, enabling it to determine the mutational composition of clones and the cancer cell fractions of these mutations. Compared to existing clonal reconstruction methods, MyClone enhances clustering accuracy and cancer cell fraction prediction when applied to deep-targeted sequencing data and bulk tumor sequencing data with deep sequencing coverage. Additionally, MyClone achieves a substantial improvement in computational speed. We rigorously validated MyClone’s performance using both simulated and real clinical datasets and applied it to analyze a circulating tumor DNA sequencing dataset from 139 metastatic breast cancer patients. In this analysis, we explored mechanisms of drug resistance in metastatic breast cancer and identified 10 mutated genes potentially associated with drug resistance or sensitivity. Conclusions For deeply sequenced data, MyClone outperforms existing methods on both targeted sequencing and bulk tumor data. Its high computational efficiency and reconstruction accuracy position MyClone as a promising tool for broad clinical application in cancer treatment. The source code is publicly available at https://github.com/Hansen0413/myclone-code. Supplementary Information The online version contains supplementary material available at 10.1186/s12859-025-06252-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008b5a4c97ae8500318c69c4b98363d3ad251446" target='_blank'>
              MyClone: rapid and precise reconstruction of clonal population structures for tumors
              </a>
            </td>
          <td>
            Qihan Guo, Dan Lv, Bo Lan, Haili Qian, Fei Ma, Ting Chen
          </td>
          <td>2025-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53813423a746c2c5a950b8780d63e2566846931" target='_blank'>
              Modeling response to the KRAS-G12C inhibitor AZD4625 in KRASG12C NSCLC patient-derived xenografts reveals insights into primary resistance mechanisms.
              </a>
            </td>
          <td>
            J. Rosen, P. Cao, Nhu-An Pham, Matthew Waas, Q. Li, Katrina Hueniken, Mutian Wang, R. Navab, Leanne Wybenga-Groot, N. Radulovich, Michael Niedbala, A. Koers, Sarah Ross, Michael F Moran, A. Sacher, M. Tsao
          </td>
          <td>2025-10-11</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="This study aimed to identify novel prognostic biomarkers and therapeutic strategies by focusing on drug resistance-related phenotypes. We integrated multi-omics data from various databases encompassing sequencing, clinical information, resistance-related gene sets, and genomic alteration data. Single-sample gene set enrichment analysis (ssGSEA) was employed to calculate resistance scores for individual samples, which were subsequently applied in survival analysis. Furthermore, we utilized machine learning algorithms to develop a robust prognostic model validated across multiple independent datasets. Our findings revealed 12 genes consistently linked to PAAD prognosis in diverse datasets. Pathway enrichment analysis indicated that the high-risk group was enriched in pathways associated with systemic lupus erythematosus and the cell cycle, while the low-risk group showed significant enrichment in neuroactive ligand-receptor interaction pathways. Additionally, immune cell infiltration analysis exhibited substantial differences between risk groups, with the high-risk cohort presenting lower levels of activated CD8 + T cells but higher levels of regulatory T cells. The random survival forest (RSF) model demonstrated superior predictive performance, achieving a concordance index of 0.634 and time-dependent ROC AUCs of 0.973, 0.978, and 0.996 at 1, 2, and 3 years, respectively. In conclusion, this study identifies 12 critical drug resistance genes in PAAD and highlights the associated immune differences in patient risk, paving the way for targeted immunotherapy research to improve therapeutic strategies against this formidable disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9036a9dfcaada4e83527a56e65b2bda1ef503c41" target='_blank'>
              To Develop Prognostic Markers Related to Drug Resistance in Pancreatic Cancer Patients Based on Multiple Machine Learning Methods
              </a>
            </td>
          <td>
            Song Xu, Chuanmin Deng, Z. Man, Song Yang, Ming Xu
          </td>
          <td>2025-10-10</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Tunneling nanotubes (TNTs) are dynamic membranous protrusions that are critical for intercellular communication in cancer, yet their transient nature and lack of definitive molecular markers pose challenges for systematic study. To address this issue gap in knowledge, we used bulk and single-cell transcriptomic sequencing approaches to identify differentially expressed genes in ovarian cancer cells enriched for TNTs to identify molecular pathways and genes potentially implicated in TNT formation, maintenance and function. We used edgeR and Seurat to perform differential expression analysis on ovarian cancer cells grown in conditions favoring upregulation of TNT formation and further performed single-cell sequencing of the same cell type isolated via spatiotemporal genomic analysis (SaGA) and fluorescence-activated cell sorting (FACS). We found 771, 32 and 279 genes upregulated in bulk sequenced TNT forming SKOV3, COV362 and ES-2 ovarian carcinoma cell lines, respectively. Gene families that repeatedly came up across all three of the cell lines included carbonic anhydrases, matrix metalloproteinases, SERPIN (Serine Protease Inhibitor-30), and SNORA (Small Nucleolar RNAs). We narrowed down candidate genes of interest to 23 differentially expressed genes (DEGs), many of which are involved in cellular stress response, organization of the extracellular matrix or intracellular cytoskeleton, cell adhesion, and other pathways. Further analyzing the single cell sequenced ES-2 cells, we found the differential expression of KISS1 gene, a metastasis suppressor implicated in cytoskeletal organization, significantly upregulated in TNT-enriched cells, highlighting its potential role in TNT formation. Other genes, such as EPHB1, MMP3, and WNT7B, were also upregulated in TNT-enriched cells, suggesting their involvement in regulating cell adhesion, migration, and differentiation. Conversely, we observed downregulation of BGN, a gene associated with promoting metastatic potential and NFkB signaling in cancer cells, as well as SYNGR1, a gene traditionally linked to synaptic transmission but also involved in vesicle exocytosis, which may influence TNT formation.



 Rojina Sapkota. RNA-seq analysis to uncover drivers of cancer cells forming tunneling nanotubes for intercellular communication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb5389ea72947e09856895a8de975c87a724ed10" target='_blank'>
              Abstract B015: RNA-seq analysis to uncover drivers of cancer cells forming tunneling nanotubes for intercellular communication
              </a>
            </td>
          <td>
            Rojina Sapkota
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40935d544b5f896144e035981022bfa9b8389a70" target='_blank'>
              Investigation of Double-Stranded DNA Donors and CRISPR-Cas9 RNP for Universal Correction of Mutations Causing Cystic Fibrosis in Human Airway Cells
              </a>
            </td>
          <td>
            Vrishti Sinha, P. Ayoub, C. Juett, Lindsay E. Lathrop, Ruth A. Foley, Ruby A. Sims, Joseph D. Long, Emily C. Duggan, Neil Fernandes, Beate Illek, B. Gomperts, S. Jonas, Donald B. Kohn
          </td>
          <td>2025-09-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Ovarian cancer remains the most lethal gynecologic malignancy, with five-year survival rates below 50% largely due to late-stage diagnosis and the development of drug resistance. The advent of precision medicine offers new opportunities to improve patient outcomes. To date, molecular profiling has identified key therapeutic targets in ovarian cancer, including homologous recombination deficiency (HRD), BRCA mutations, and mismatch repair defects. PARP inhibitors have demonstrated significant clinical benefit in patients with HRD-positive tumors. Also, folate receptor alpha (FRa) targeting antibody drug conjugate, mirvetuximab soravtansine has demonstrated improved overall survival in platinum resistant ovarian cancer expressing high levels of FRa. To exploit DNA replication vulnerabilities in ovarian cancer cells, inhibitors of cell cycle regulators ATR and CHK1, have been studied in ovarian cancer preclinical and clinical settings. DHX9 helicase inhibition represents another novel approach, targeting RNA/DNA helicase activity critical for transcriptional regulation and DNA repair processes in cancer cells. Additional emerging targets include KIF18A that regulates mitotic progression and microtubule dynamics, which have demonstrated selective activity in chromosomally instable high-grade serous ovarian cancer cells. Novel therapeutic targets such as DHX9 helicase and KIF18A, offer new opportunities for patients with limited treatment options. Future directions include expanding biomarker testing, developing combination therapeutic strategies that exploit multiple vulnerabilities simultaneously, and addressing resistance mechanisms.



 Jung-Min Lee. Advancing precision therapy through molecular profiling and targeting DNA replication vulnerabilities in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c03eea4f891eb2bca142fb8a480f45ccc5bf2a" target='_blank'>
              Abstract IA018: Advancing precision therapy through molecular profiling and targeting DNA replication vulnerabilities in ovarian cancer
              </a>
            </td>
          <td>
            Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Glioblastoma (GBM) is marked by profound spatial heterogeneity. This complexity fuels tumor progression and therapy resistance. Although recent spatial transcriptomics (ST) studies have defined transcriptional niches, or metaprograms, the systems level regulatory logic linking upstream signaling, transcriptional programs, and therapeutic vulnerabilities remains unresolved. Here, we introduce an integrated, multi layered framework that unifies transcription factor and pathway activity inference (STAN and SPAN) with ligand/receptor and drug target mapping, thereby constructing a systems-level regulatory atlas of GBM across 26 tumors. This atlas uncovers spatial niches organized around transcriptional hubs and signaling programs, including hypoxic mesenchymal regions coordinated by HIF1A and SOX2 with mTORC1 signaling, macrophage rich areas driven by STAT3 mediated immune modulation, and astrocyte-like states shaped by lipid metabolic regulators such as SREBF2. By coupling this regulatory atlas to drug target predictions, we identify niche specific therapeutic vulnerabilities, such as VEGF blockade in HIF1A high regions and MAO-B inhibition in mesenchymal states. Finally, we provide an open, interactive web resource to make these data broadly accessible. This work provides a generalizable blueprint for linking spatial regulation to therapeutic hypotheses across diverse human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a40d513760c094d00d7ea814fe1f34729bf0c35" target='_blank'>
              A Multi-Layered Atlas of Spatial Regulatory Programs and Therapeutic Vulnerabilities in Glioblastoma
              </a>
            </td>
          <td>
            Linan Zhang, Matthew Lu, Xiaojun Ma, H. Osmanbeyoglu
          </td>
          <td>2025-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) remains a formidable therapeutic challenge, with high relapse rates rooted not only in genetic heterogeneity, but also in dynamic and reversible epigenetic dysregulation that drive resistance to conventional and targeted small-molecule therapies. Although small-molecule inhibitors targeting epigenetic regulators have demonstrated preclinical efficacy and initial clinical activity, therapeutic gains are often limited by transient responses, intra- and inter-patient heterogeneity, and an incomplete understanding of predictive biomarkers. Besides, clinical outcomes have often been less robust than preclinical data suggested, partly due to the complexity of AMLs genetic and epigenetic heterogeneity, the limitations of model systems, and intrinsic resistance mechanisms. Combination approaches, such as pairing epigenetic agents with chemotherapy, immunotherapy, or other targeted drugs are under investigation. However, clinical efficacy remains inconsistent and requires a mechanistic rationale to avoid antagonism. To move forward, it is essential to delineate specific epigenetic dependencies in resistance-prone AML subtypes, design rational, biomarker-guided combination regimens, and enhance specificity and pharmacodynamic profiles of epigenetic inhibitors. Such an integrated strategy holds promise for elevating the effectiveness of epigenetic therapies in AML. Continued research is essential to refine epigenetic-based precision medicine and ultimately improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1118798f14d99fa1bf83fc376d07fc1df6c5c" target='_blank'>
              Epigenetic mechanisms of drug resistance in acute myeloid leukemia: advances in small-molecule targeted therapy
              </a>
            </td>
          <td>
            Jialing Tang, Xibo Ye, Zhangyang Xu, Tianjiao Min, Zherun Tian, Li Ma, Ying He, Beier Jiang
          </td>
          <td>2025-10-20</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute leukemias are highly aggressive hematologic malignancies that demand intensive chemotherapy regimens. However, drug toxicity remains a major barrier to treatment success and patient survival. In this context, pharmacogenomics offers a promising strategy by identifying single-nucleotide variants (SNVs) that influence drug metabolism, efficacy, and toxicity, ultimately impacting treatment outcomes. This study analyzed data from the ClinPGx/PharmGKB database to identify clinically annotated variants related to chemotherapy response in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL). A total of 24 variants were curated for AML and 57 for ALL. Among these, nonsynonymous variants were most frequent in ALL (31.6%), while synonymous variants predominated in AML (33.3%). Although traditionally considered neutral, synonymous and intronic variants may influence gene expression through regulatory or splicing mechanisms. The analysis revealed clinically significant variants associated with chemotherapy response, particularly in the ABCB1 gene, observed in 12.5% of AML and 10.5% of ALL cases. Several variants, particularly TPMT, NUDT15, ABCC1, SLC28A3, and RARG, were associated with severe adverse effects such as myelotoxicity, mucositis, cardiotoxicity, and hepatotoxicity. This study reinforces the importance of genetic variants in modulating the therapeutic response and toxicity to chemotherapy drugs in acute leukemias. Analysis of ClinPGx/PharmGKB data emphasizes ABCB1 as a potential resistance marker and supports pre-treatment genotyping of genes like TPMT and NUDT15 to prevent severe toxicities. Future advances should include the expansion of pharmacogenetic studies in underrepresented populations and the clinical validation of new markers in prospective trials, aiming to consolidate precision medicine as a routine part of the therapeutic management of acute leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41d398f2f78483d0f180028688c26b519950cb78" target='_blank'>
              Pharmacogenomics Applied to Acute Leukemias: Identifying Clinically Relevant Genetic Variants
              </a>
            </td>
          <td>
            Flávia Melo Cunha de Pinho Pessoa, Isabelle Magalhães Farias, B. Nogueira, C. B. Machado, Igor Valentim Barreto, Anna Karolyna da Costa Machado, Guilherme Passos de Morais, Leidivan Sousa da Cunha, Deivide de Sousa Oliveira, André Pontes Thé, Rodrigo Monteiro Ribeiro, Patrícia Maria Pontes Thé, Manoel Odorico de Moraes Filho, Maria Elisabete Amaral de Moraes, Caroline Aquino Moreira-Nunes
          </td>
          <td>2025-10-22</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba74e2d71d8617d061a7a165c8a86d93142799c4" target='_blank'>
              Accurate characterization of CRISPR-Cas9 genome editing outcomes and mosaicism with near-perfect long reads
              </a>
            </td>
          <td>
            Ida Höijer, R. van Schendel, A. Emmanouilidou, Rebecka Östlund, I. Bunikis, M. Tijsterman, M. den Hoed, Adam Ameur
          </td>
          <td>2025-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Topoisomerase 1 (Top1) removes transcription-related helical torsions and thus plays an important role in preventing genome instability instigated by the formation of non-canonical DNA secondary structures. The genetically tractable Saccharomyces cerevisiae model proved effective in defining the critical function of Top1 to prevent recombination and chromosomal rearrangement at G4-forming genomic loci and studying the human cancer-associated Top1 mutants through the expression of analogous yeast mutants. We previously showed that cleavage-defective Top1 mutants strongly elevate the rate of recombination at G4 DNA, which involves binding to G4 DNA and interaction with the protein nucleolin (Nsr1 in yeast). Here, we further explored the mechanism of genome instability induced by the yeast Top1Y740* mutant, analogous to the human Top1W765Stop mutant conferring resistance to CPT. We show that yTop1Y740* elevates duplications as well as recombination specifically at G4-forming sequences. Interestingly, SUMOylation of yTop1Y740*, which does not affect the G4 DNA-binding or Nsr1-interaction by this mutant, is necessary for such elevated G4-specific genome instability. Many tumors with mutations at the C-terminal residues of Top1, particularly W765, have significantly high G4-associated mutations, underscoring the importance of further investigation into how SUMOylation affects the function of these Top1 mutants at G4 DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/414ccc1f67726dc580ae95a50a7d8e620a487775" target='_blank'>
              Mutations and Recombination at G4 DNA-Forming Sequences Exacerbated by CPT-Resistant Mutant Topoisomerase 1 Is Dependent on SUMOylation
              </a>
            </td>
          <td>
            Shivani Singh, Xinji Zhu, Nayun Kim
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="BACKGROUND
Diffuse midline gliomas (DMGs), particularly diffuse intrinsic pontine gliomas (DIPGs), are aggressive pediatric brain tumors with a median survival of less than 12 months. Notably, approximately 70% of these tumors harbor P53 pathway alterations, including TP53 or PPM1D mutations. Identifying precision therapies for this subset is crucial. This study aims to employ transcriptomics-guided high-throughput drug screening to identify effective treatments for DIPG/DMG with P53 pathway alterations.


METHODS
Transcriptomic profiling of 98 patient samples containing 31 DIPGs revealed key activated pathways. Patient-derived cell lines were subjected to high-throughput screening using a cell cycle-targeting drug library. Lead candidates were validated both in vitro and in orthotopic xenograft models, while combination therapies were assessed for their ability to overcome TP53-mutant resistance.


RESULTS
Transcriptomic analysis revealed activation of P53 and cell cycle pathways in DIPGs. High-throughput drug screening identified SN-38, a topoisomerase I inhibitor, as selectively targeting TP53 wild-type tumor cells by inducing G2 arrest and apoptosis. TP53 knockdown abolished SN-38's efficacy, while PPM1D knockdown enhanced sensitivity, confirming a TP53-dependent mechanism. Conversely, TP53-mutated cells exhibited SN-38 resistance via ATR pathway activation. Combining SN-38 with the ATR inhibitor AZ20 restored apoptosis and suppressed TP53-mutant tumor growth in vitro and in vivo.


CONCLUSIONS
Guided by transcriptomic profiling, this study utilized high-throughput drug screening to identify SN-38 as a potential therapy for TP53 wild-type DIPG/DMG, while the SN-38 and AZ20 combination was effective against TP53-mutant tumors. This approach provides a promising treatment strategy for this malignancy and establishes a novel paradigm for drug screening in DIPG/DMG.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a7ee270c7115fa9b3b5a7437784e7892bea4467" target='_blank'>
              Transcriptomics-Guided High-Throughput Drug Screening Identifies Potent Therapies for P53 Pathway Alterated DIPG/DMG.
              </a>
            </td>
          <td>
            Zhuan-Xin Jiang, Luyang Xie, Hang Zhou, Yibo Geng, Xiong Xiao, Tian Li, Yuxuan Deng, Mingxin Zhang, Shaobo Shan, Cheng Xu, Liwei Zhang
          </td>
          <td>2025-09-17</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epigenetic dysregulation has been increasingly understood to be a key factor in the development of myeloid malignancies, often acting alongside genetic alterations to disrupt normal hematopoiesis. The inherent reversibility of epigenetic changes has provided an excellent opportunity for therapeutic intervention. Hypomethylating agents (HMA) preferentially alter gene expression in heavily methylated malignant myeloid cells by inhibiting DNA methyltransferases thereby altering gene expression. They have since become foundation therapies in myelodysplastic syndrome (MDS) and in older patients with acute myeloid leukemia (AML). However, complete responses to HMA monotherapy are limited and usually non-durable. In recent years, novel approaches have been sought to overcome resistance and expand the role of epigenetic therapies in MDS and AML as well as in less well-studied myeloid malignancies such as myeloproliferative neoplasms (MPNs) and MDS/MPN overlap syndromes. Combination regimens that synergistically pair HMAs or other epigenetically active therapeutics with agents targeting apoptosis, cellular metabolism, and immune evasion have also shown early promise in improving patient outcomes. Oral formulations of HMAs have made maintenance strategies more convenient and tolerable for patients, with demonstrated benefits in AML and ongoing investigations in MDS and post-transplant settings. This review will explore these novel therapeutic strategies in the treatment of myeloid malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14b616f7da39ac33cb77f5ad7d9b809d8c14dc8a" target='_blank'>
              Novel approaches to the use of hypomethylating agents in myeloid malignancies.
              </a>
            </td>
          <td>
            Justin Cheng, Casey O’Connell
          </td>
          <td>2025-09-25</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/466091ce59415bf8e8d644d95fdb8949a53fb96b" target='_blank'>
              Cancer-associated fibroblasts drive metabolic heterogeneity in KRAS-mutant colorectal cancer cells
              </a>
            </td>
          <td>
            Elizabeth Elton, Niki Tavakoli, Handan Cetin, Stacey D. Finley
          </td>
          <td>2025-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Chromosomal rearrangements that generate novel fusion genes are a hallmark of acute myeloid leukemia (AML). Depletion experiments in cell line models have suggested that their continued expression is required for maintaining their leukemic phenotype and that fusion genes therefore represent ideal cancer-specific therapeutic targets. However, to which extent this result holds true for the different stages of hematopoietic development in primary cells and whether therapeutic agents can be efficiently delivered to those cells is still unclear. In this study, we demonstrate that primary AML cells harboring the chromosomal translocation t(8;21) are critically dependent on the corresponding fusion gene, RUNX1::RUNX1T1, to suppress differentiation and maintain stemness. Silencing RUNX1::RUNX1T1 expression using siRNA-loaded lipid nanoparticles induces substantial changes in chromatin accessibility, thereby redirecting the leukemia-associated transcriptional network towards a myeloid differentiation program. Single-cell analyses reveal that this transcriptional reprogramming is associated with the depletion of immature stem and progenitor-like cell populations, accompanied by an expansion of granulocytic and eosinophilic/mast cell-like populations with impaired self-renewal capacity. These findings underscore the essential role of RUNX1::RUNX1T1 in sustaining AML and highlight the therapeutic potential of targeting fusion gene expression in primary AML cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5469845f04d542fe9db7d0ec4a2180ccb6b7c697" target='_blank'>
              Fusion Gene Depletion Eliminates Stemness and Induces Bidirectional Differentiation of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Polina K Derevyanko, L. Swart, L. D. Mata Casimiro, Anita van Oort, M. Du Plessis, Luca van den Brink, Minoo Ashtiani, C. Zwaan, A. Krippner-Heidenreich, Constanze Bonifer, Raymond Schiffelers, Josef Vormoor, S. Kellaway, O. Heidenreich
          </td>
          <td>2025-09-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9752e134e3e7566b741e6bb0305e92d0896d6a" target='_blank'>
              Timing the onset of homologous recombination deficiency before cancer diagnosis
              </a>
            </td>
          <td>
            Michail Andreopoulos, Muchun Niu, Yang Zhang, V. Viswanadham, D. Gulhan, Hu Jin, Felipe Batalini, Gerburg Wulf, Chenghang Zong, Peter J. Park, Dominik Glodzik
          </td>
          <td>2025-10-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="A population of cells within a tumor can be described as antifragile (the opposite of fragile) if they derive a benefit from fluctuations or perturbations in environmental conditions induced by treatment. We hypothesized that treatment fluctuations may either promote (antifragile tumor) or inhibit (fragile tumor) the evolution of treatment resistance to targeted therapies. Analysis of the convexity of dose response curves provides a direct prediction of response to prescribed fluctuations in treatment. Theory predicts that continuous treatment protocols (i.e. zero prescribed fluctuations in treatment) will outperform uneven protocols when dose response is convex. We apply this theory to predict in vivo response to targeted therapy and validate these predictions by experimentally testing high/low intermittent dosing (uneven dosing) and continuous dosing (even dosing) schedules. Convexity (and its inverse, concavity) explains two phenomena: dose response is a convex (fragile) function but resistance onset rate is a concave (antifragile) function. Thus, we design, propose, and validate alternative treatment schedules which maximize response while maintaining prolonged sensitivity to treatment. These analyses provide supporting evidence that fluctuations alter the evolutionary trajectory of tumors in response to targeted therapy, even without altering the cumulative dose. This insight is used to design alternative, switching treatment protocols to limit the evolution of resistance, a process which we call evolutionary antifragile therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8cf7a63052092f4dae35998f972484b69a62e3" target='_blank'>
              Evolutionary antifragile therapy
              </a>
            </td>
          <td>
            J. West, Bina Desai, M. Strobl, J. Gallaher, M. Robertson-Tessi, A. Marusyk, Alexander R. A. Anderson
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 Mutational activation of KRAS is critical for driving the tumorigenic growth of pancreatic ductal adenocarcinoma (PDAC). Recent clinical observations with the approved KRASG12C selective inhibitors sotorasib and adagrasib support the therapeutic value of targeting KRAS in PDAC. However, long-term efficacy is hindered by both primary and treatment-associated acquired resistance, and it is likely applicable to other types of KRAS inhibitors targeting different mutant alleles. To define the mechanisms behind resistance to KRAS inhibition, we performed a CRISPR loss-of-function screen to identify genes modulating the sensitivity of PDAC cells to KRAS inhibition. The screen identified loss of KEAP1 as a significant driver of resistance to inhibitors targeting KRAS. KEAP1 negatively regulates the transcription factor NRF2, which orchestrates transcription of genes that drive the anti-oxidative stress response and promote cell survival. We verified that knockout of KEAP1 drove NRF2-dependent resistance to KRAS inhibition. To elucidate the mechanism of KEAP1 loss-mediated resistance, RNA-sequencing analyses in KEAP1-depleted cancer cells revealed that KEAP1 loss protected cancer cells from oxidative stress and cell cycle arrest induced by KRAS inhibition. We then established a PDAC KEAP1-deficient (PKD) gene signature, which was found prevalent in patients and models insensitive to KRAS inhibitor treatment. This gene set was distinct from transcriptional changes associated with other transcriptional signatures driving resistance to KRAS inhibitors (ERK, MYC and YAP/TAZ-TEAD). Finally, analysis of KEAP1-loss associated genes also identified the activation of glutamine metabolism, and we found the combination treatment with the glutamine antagonist DRP-104 enhanced KRAS inhibitor-mediated growth suppression in PDAC patient-derived organoids. In summary, our studies have established the molecular mechanisms underlying KEAP1-NRF2 pathway-driven resistance to KRAS inhibition and validated a potential therapeutic strategy to overcome this resistance.



 Wen-Hsuan Chang, Addison G. Stamey, Runying Yang, Ryan Robb, Jeffery A. Klomp, Clint A. Stalnecker, Andrew M. Waters, Antje Schaefer, Kirsten L. Bryant, Adrienne D. Cox, Channing J. Der. Defining a molecular basis and therapeutic approach for KEAP1-NRF2 mediated resistance to KRAS inhibition in KRAS-mutant pancreatic ductal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B083.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e978c547c01981533894a87c73996da5913ebc3" target='_blank'>
              Abstract B083: Defining a molecular basis and therapeutic approach for KEAP1-NRF2 mediated resistance to KRAS inhibition in KRAS-mutant pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Wen-Hsuan Chang, Addison G Stamey, Run-Wei Yang, Ryan Robb, Jeffery A. Klomp, Clint A. Stalnecker, Andrew M. Waters, Antje Schaefer, Kirsten L. Bryant, Adrienne D. Cox, C. Der
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="Metastatic breast cancer (MBC) remains the primary cause of mortality in breast cancer patients, driven by tumor heterogeneity, cellular evolution, and therapy-resistant clones. Traditional bulk transcriptomics, although informative, fail to capture rare subpopulations and context-specific gene expression, which are crucial for understanding disease progression. Single-cell transcriptomics (SCT) has emerged as a transformative approach, enabling high-resolution analysis of individual cells to reveal tumor composition, lineage dynamics, and transcriptional plasticity. This review highlights how SCT reshapes our understanding of MBC by mapping tumor evolution, identifying cancer stem-like cells, and characterizing states of epithelial-mesenchymal transition. We explore how SCT reveals clonal and spatial heterogeneity, and how tumor microenvironment components, including immune, stromal, and endothelial cells, interact with cancer cells to support immune evasion and the formation of a metastatic niche. SCT also uncovers mechanisms of therapeutic resistance, including transcriptional reprogramming and the survival of drug-tolerant subpopulations. Integrating SCT with spatial transcriptomics and multi-omics platforms offers a comprehensive view of the MBC ecosystem and may uncover novel therapeutic targets. We further discuss the translational potential of SCT for biomarker discovery, liquid biopsy development, and precision oncology. We address current technical challenges and future directions for clinical application. SCT is poised to transform MBC research and guide next-generation therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf97dd9ea39bc843d16cb57d14bc4af2cfce4592" target='_blank'>
              Single-cell transcriptomics in metastatic breast cancer: mapping tumor evolution and therapeutic resistance
              </a>
            </td>
          <td>
            Xu Han, Xin Li, Ling Bai, Gangling Zhang
          </td>
          <td>2025-10-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hematologic malignancies, including leukemia, lymphoma, and multiple myeloma, represent a diverse group of blood cancers driven by complex oncogenic pathways and genetic aberrations. Despite advances in treatment, resistance and relapse remain significant clinical challenges. This review presents an integrative approach to blood cancer therapy, highlighting the convergence of precision medicine, CRISPR-based gene editing, immunotherapy (including CAR-T cells and immune checkpoint inhibitors), and targeted modulation of amino acid metabolism. We investigate the molecular pathways driving tumour development and resistance, highlighting how multi-omic profiling facilitates personalised therapy approaches. Recent breakthroughs in reprogramming immune responses and editing driver mutations have revolutionized therapeutic paradigms. Additionally, the metabolic dependencies of cancer cells, particularly in amino acid biosynthesis and catabolism, offer novel vulnerabilities. This meticulous synthesis outlines a framework for translational techniques that integrate molecular targeting with immunological and metabolic regulation to improve treatment effectiveness and surmount therapeutic resistance in haematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/602acd2f85cdfeb967bf671ca25b0e041a71b90f" target='_blank'>
              Deciphering and targeting oncogenic pathways through integrated approaches and amino acid metabolism in hematologic malignancies
              </a>
            </td>
          <td>
            Farhan Ikhtiar, Adil Jamal, Syed M Safeer Mehdi Bokhari
          </td>
          <td>2025-09-16</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00ef580d1c2e048fccb2889a7c92151af7fd5d7" target='_blank'>
              Probability Distribution for Rare Neutral Mutations in Cancers and Application to Dynamic Precision Medicine of Cancer
              </a>
            </td>
          <td>
            Wei He, Matthew D. McCoy, Chen-Hsiang Yeang, Rebecca B. Riggins, Robert A. Beckman
          </td>
          <td>2025-10-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chemoresistance is a major obstacle in cancer treatment, and it often results in treatment failure and disease progression. Among the plethora of factors contributing to chemoresistance, the transcription factor Krüppel-like factor 4 (KLF4) has emerged as a pivotal player. This review discusses the role of KLF4 in orchestrating various mechanisms underlying cancer chemoresistance. KLF4, originally identified as a regulator of cell differentiation and proliferation, has recently gained attention for its role in modulating cellular responses to chemotherapeutic agents. Through complex regulatory networks, KLF4 modulates the process of drug efflux, DNA repair, apoptotic signaling, tumor heterogeneity, and cancer cell stemness, leading to the development of cancer chemoresistance. Additionally, tissue or cell types specific post-translational modification (PTM) of KLF4 plays a significant role in the development of cancer chemoresistance. The review explores emerging possibilities and available information that can be utilised to understand the mechanism of chemoresistance mediated by KLF4 in cancer. In conclusion, understanding the complex mechanisms through which KLF4 orchestrates cancer chemoresistance opens promising avenues for developing more effective therapeutic interventions to combat treatment-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e91a9cb65a0f14dcdac5b83f903eda917dba7f" target='_blank'>
              KLF4 in cancer chemoresistance: molecular mechanisms and therapeutic implications
              </a>
            </td>
          <td>
            Suresh Singh Yadav, Punita Kalia, Navneet Kaur, Sourav Sharma, Shambhavi Thakur, Soni Kumari, R. R. Nair
          </td>
          <td>2025-09-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de278a01c7636edead18a0789115129a52c7207e" target='_blank'>
              Centromere instability links genome damage to immune activation in systemic sclerosis
              </a>
            </td>
          <td>
            Azait Imtiaz, Mohammad Waseem, Hudson O’Neill, Bo-Ruei Chen, Wioletta Czaja, Rafael Contreras-Galindo
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bb8a507427a3e0cfe9ea50f66dabfc18ac940f2" target='_blank'>
              Cancer Metastasis: Therapeutic Challenges and Opportunities.
              </a>
            </td>
          <td>
            Sri Sathya Sandilya Garemilla, Manisha Choudhary Kadambala, S. C. Gampa, Swetha Chinthala, S. Garimella
          </td>
          <td>2025-10-16</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most lethal gynecological malignancy. Therapeutic options remain limited for patients lacking predictive biomarkers, particularly those with BRCA wild-type tumors or those who have acquired resistance to both PARP inhibitors and platinum-based chemotherapy. Replication stress, TP53 mutations, and genomic instability characterize HGSOC. The cellular response to replication stress is primarily mediated by checkpoint kinases; however, this mechanism is frequently impaired in tumor cells. Consequently, cancer cells become increasingly dependent on the replication stress response (RSR) pathway for survival, and susceptible to therapies targeting the ATR-CHK1-WEE1 axis—a key regulator of genomic integrity. Inhibition of these checkpoint kinases can disrupt cell cycle control, inducing mitotic catastrophe and subsequent cancer cell death. Another defining feature of HGSOC is its immunosuppressive tumor microenvironment (TME), which has limited the efficacy of immune checkpoint inhibitors. Emerging evidence suggests that inhibition of the RSR pathway may not only exploit intrinsic tumor vulnerabilities but also modulate the TME to enhance anti-tumor immune responses. This provides rationale for combination approaches integrating RSR pathway inhibitors with innovative immune checkpoint blockade (ICB). This review examines the mechanistic rationale and therapeutic potential of such combinations, drawing on both preclinical and clinical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa92875288685395bcc326c756f9f278710481b" target='_blank'>
              Interplay of replication stress response and immune microenvironment in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Laura Venegas, Stephanie Lheureux
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the second leading cause of cancer death worldwide, largely due to the limited efficacy of current therapies in advanced stages of the disease. Most cases of HCC develop in the setting of chronic liver disease, particularly cirrhosis, where ongoing cycles of inflammation, hepatocyte death and regeneration foster the gradual accumulation of genetic and epigenetic alterations that promote malignant transformation. These molecular changes contribute to the high degree of tumour heterogeneity observed in HCC, a major factor underlying resistance to current treatments. As a result, sustained clinical responses to existing therapies, such as tyrosine kinase inhibitors, anti-angiogenic agents and immune checkpoint inhibitors, remain uncommon. In this context, a growing body of evidence has identified epigenetic dysregulation as a key driver of tumour progression and therapeutic resistance, highlighting a new frontier for intervention. This review provides clinicians and researchers with a comprehensive overview of the emerging field of epigenetic therapies in HCC, summarising results from both completed and ongoing clinical trials involving the so-called ‘epidrugs’. Importantly, we discuss how targeting epigenetic mechanisms may not only suppress tumour growth but also enhance the effectiveness of current therapies by reversing resistance pathways. By translating complex molecular insights into tangible therapeutic strategies, epigenetics is poised to reshape the future of HCC management, offering renewed hope for more durable and personalised treatment responses in a disease where progress is urgently needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca914bc3e6f4b196e21231ee5479dd3a9a6ded8" target='_blank'>
              Epigenetic therapies in hepatocellular carcinoma: emerging clinical tools and applications
              </a>
            </td>
          <td>
            Barbara Bueloni, Maite G Fernandez-Barrena, Esteban J. Fiore, Matías A. Avila, J. Bayo, Guillermo Mazzolini
          </td>
          <td>2025-09-23</td>
          <td>Gut</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction Precise genome editing is a critical focus in gene therapy, and the CRISPR-Cas9 system has become a powerful and versatile tool for this purpose. However, a significant limitation of the CRISPR-Cas9 system is its low homologous recombination rate, which can impede the restoration of normal gene function. To address some of these challenges, advanced gene-editing technologies, such as base editors and prime editors have been developed. Here, we explored whether Cas9-specific single-stranded DNA (ssDNA) aptamers could enhance the PE2 system’s functionality. Methods Systematic evolution of ligands by exponential enrichment (SELEX) was utilized to isolate high-affinity Cas9-specific ssDNA aptamers. Molecular docking simulations were subsequently performed to characterize the binding interactions between these aptamers and the PE2 protein. PE2 editing efficiency was quantitatively assessed using flow cytometry and Sanger sequencing. In bladder cancer cell lines, p53 mutation repair was evaluated by quantitative PCR and Western blot analysis, while cellular responses were examined through proliferation (CCK-8) and apoptosis assays. Results Molecular docking analysis revealed the interaction sites between SELEX-screened Cas9-specific aptamers and the PE2 protein. The incorporation of these aptamers significantly enhanced PE2 editing efficiency. In bladder cancer cells, the aptamer-PE2 complex effectively restored p53 function, leading to suppressed cellular proliferation and enhanced apoptosis rates. Discussion Our study demonstrates that Cas9-specific aptamers can effectively enhance prime editing efficiency. This provide new insights into the modulation of prime editing and hold potential for improving its clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10045515c2dba7d4dc48bd78fde644b501b3d21d" target='_blank'>
              Screening and characterization of DNA aptamers that modulate prime editing
              </a>
            </td>
          <td>
            Mingxia Wang, Xia Wu, Xinbo Huang, Jing Ye, Yaoting Gui
          </td>
          <td>2025-09-09</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55be761462c41a8fdd2d1a07cb3c86165b3561a4" target='_blank'>
              Systematic identification of genomic non-response biomarkers to cancer therapies
              </a>
            </td>
          <td>
            Joseph Usset, J. de Ligt, Sophie Roerink, P. Roepman, E. Cuppen, Francisco Martínez-Jiménez
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="DNA mismatch repair (MMR) preserves genomic integrity by correcting replication errors. Deficiency in MMR (MMRd) results in microsatellite instability, increased tumor mutational burden, neoantigen generation, and activation of immune response. In this review, we first outlined how MMR loss promotes immune activation and responsiveness to immune checkpoint blockade (ICB), establishing MMRd status as the first tumor-agnostic biomarker for ICB therapy. Subsequently, we summarized the compelling evidence that defines MMR status as a dynamic, context-dependent process influenced by environmental and therapeutic pressures, rather than a fixed, binary trait. Accordingly, we discussed the implications of spatial and temporal heterogeneity of MMR status for both the diagnosis and treatment of cancer, the differential response of MMRd tumors to ICB, as well as the occasional benefits observed in MMR-proficient (MMRp) immune-cold cancers. We then explored strategies to exploit MMR dynamics and mimic MMRd-like phenotypes through alkylating agents, pharmacological MMR inhibition, and stress-mediated modulation, with the aim of sensitizing refractory tumors to immunotherapy. Finally, we reported emerging therapeutic opportunities in MMRd tumors, including Werner helicase inhibition, nonsense-mediated decay blockade, and neoantigen-targeted vaccines. Altogether, reframing MMR as a dynamic and targetable axis may broaden immunotherapy applicability and advance precision immune oncology across different tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6face00bfd040e4e182ff419f940662d44199ca5" target='_blank'>
              Mismatch Repair as a Dynamic and Clinically Actionable Vulnerability in Cancer.
              </a>
            </td>
          <td>
            Eleonora Piumatti, P. P. Vitiello, V. Amodio, A. Bardelli, Giovanni Germano
          </td>
          <td>2025-09-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d93739961372e0101853c9916cb4256544b480" target='_blank'>
              SenLect: a genetically encoded system to purify senescent cells
              </a>
            </td>
          <td>
            Hourieh Tousian, Rachael A Protzman, T. Sargeant, Julian M. Carosi
          </td>
          <td>2025-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Chemotherapy is a foundational element in cancer therapy; however, the development of resistance to antineoplastic agents presents a significant challenge to achieving enduring therapeutic triumphs. The acquisition of drug resistance by malignant cells is the result of a complex interplay between inherent and acquired mechanisms. This phenomenon leads to a reduction in pharmacological effectiveness and contributes to treatment failure. This review investigates the primary mechanisms contributing to chemoresistance in specific malignancies, namely lung, pancreatic, and thyroid cancers, along with non-Hodgkin lymphoma. The primary focus of this research is on genetic modifications, interactions within the tumor microenvironment, and cellular adaptations at the molecular level. The present study places particular emphasis on the contributions of drug efflux pumps, DNA repair pathways, epithelial-mesenchymal transition (EMT), microRNAs, and the suppression of apoptosis in mediating this resistance. In addition, we examine promising approaches to counteract drug resistance, encompassing multi-agent regimens, inhibitors targeting specific molecular pathways, and innovative therapeutic modalities. A comprehensive understanding of these fundamental mechanisms is imperative for the development of sophisticated therapeutic interventions that not only improve patient outcomes but also effectively address resistance. The resistance pathways and prospective translational approaches discussed are drawn from both preclinical and clinical investigations, providing a thorough perspective. Keywords: The following terms are relevant to the study: chemoresistance, tumor microenvironment, molecular pathways, targeted therapy, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ec567f4501baf205432eef42dce8eb9f17d9a8c" target='_blank'>
              Mechanisms of Chemoresistance in Solid and Hematologic Malignancies: Challenges and Future Perspectives
              </a>
            </td>
          <td>
            Badiaa Batlamous, Boutaina Elgharbaoui, Imane Bensalim, Mohamed Khalis
          </td>
          <td>2025-09-24</td>
          <td>Middle Eastern Cancer and Oncology Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for patients with cancer. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer and established that radiation-resistant tumor cells have enhanced metastatic capacity. Resistance to radiation increases the expression of integrin β3 (ITGB3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGB3 transcripts. Specifically, the RNA binding protein heterogeneous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs that sponge the family of let-7 microRNAs that target ITGB3. Collectively, our findings identify a mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8064b7e84c02e1178a653876a56f4f95d2d1c28" target='_blank'>
              Resistance to radiation enhances metastasis by altering RNA metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, Christi A. Silva, Rui Li, Brendan A Pacheco, L. J. Zhu, William A Flavahan, A. Mercurio
          </td>
          <td>2025-10-17</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Through most of medical history, treatments for metastatic cancers were ineffective, and rapid patient death was inevitable. Over the past five decades, a worldwide drug development effort has introduced a remarkable range of new cancer treatment strategies and agents so that virtually all metastatic cancers have one or more effective therapeutic options to prolong life. Yet most metastatic cancers remain fatal, and increasingly, the proximate cause of death is evolution. Local or systemic therapies applied to large, heterogeneous cancer populations elicit complex short- and long-term adaptive responses. Cells already possessing the molecular machinery of resistance obtain a stepwise fitness benefit relative to treatment-sensitive cells, allowing increased proliferation. Cells, otherwise sensitive to the treatment, may survive when in epigenetic states resistant to the treatment-induced death pathway or microenvironmental conditions that reduce drug delivery/efficacy, followed by a transition to "hard-wired" resistance allowing proliferation. These dynamics, enabled by the vast information content of the human genome, can produce diverse adaptive strategies in response to virtually all available treatments. Thus, oncology is rapidly approaching an era in which patient death is caused not by the absence of effective therapies but rather by eco-evolutionary dynamics that defeat initially successful treatments. Emerging evidence suggests that explicit integration of evolutionary principles to control or eliminate resistant populations can improve outcomes. In this issue of Cancer Research, Hockings and colleagues present an important evolutionary strategy to delay or prevent the evolution of resistance in ovarian cancer, with broad potential application. See related article by Hockings et al., p. 3503.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa08f9c64d9497ce73be1df571e19d588bc63780" target='_blank'>
              The Approaching Darwinian Era in Oncology.
              </a>
            </td>
          <td>
            Robert A. Gatenby
          </td>
          <td>2025-09-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human leukocyte antigen (HLA) polymorphism hinders off-the-shelf cell therapies. We developed REWRITE, a modular platform for iterative, scar-minimized genome writing of synthetic constructs >100 kb in human pluripotent stem cells (hPSCs). Using REWRITE, we deleted 105–209 kb of the HLA locus and installed synthetic 24 kb or 100 kb HLA haplotypes, and a 62 kb antigen-processing locus. This uncovered a persistent, heritable “epigenetic ghost” — an active state lingering despite genetic removal — whose resolution to a silenced default state is driven by native intergenic DNA. These loci restored inducible expression in key lineages, sparing cells from NK-mediated killing and establishing HLA-matched T-cell tolerance, enabling off-the-shelf cell therapies. REWRITE facilitates extensible programming of multigenic functions in allogeneic human cells — from immune design to genome architecture discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3662bdeb8a587004f28d0c02911c74aa245dde2b" target='_blank'>
              Human Genome REWRITE for Off-the-Shelf Stem Cells Reveals an “Epigenetic Ghost”
              </a>
            </td>
          <td>
            Serena F. Generoso, Sarah Levovitz, Susanna Jaramillo, Minjoo Kim, Sumanth Dara, Shean Fu Phen, Bryan Yi, Tomoki Yanagi, Thomas L. DesMarais, Neta Agmon, Megan S. Hogan, Leslie A. Mitchell, David M. Truong
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common neoplasm that carries the risk of transformation into Richter’s syndrome (RS), a highly aggressive B cell lymphoma with poor prognosis. Limited availability of animal models and cell lines hinders understanding of transformation mechanisms. Addressing this gap, we established the first in silico dynamic model of the disease. Our methodology integrates mathematical logic modeling with experimental data to identify disease drivers, mechanisms, and potential therapeutic targets. We validated the model by comparing the model’s readout with experimental data from different biological levels, such as single-cell RNA sequencing analyses and a CLL/RS patient formalin-fixed paraffin-embedded (FFPE) tissue cohort. Our analyses identified BMI1 proto-oncogene and TP53 loss as key RS progression regulators. In addition, we performed an in silico target screening to identify promising target combinations in a personalized fashion. Through the synergy of mathematical modeling with experimental readouts, our model provides a complementary approach to investigate the process of CLL transformation to RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b0084b1cbd03c2692e1100a97e2d57fe76c34d4" target='_blank'>
              CLL to Richter syndrome: Integrating network strategies with experiments elucidating disease drivers and personalized therapies
              </a>
            </td>
          <td>
            Julia Maier, J. Schwab, Silke D. Werle, Ralf Marienfeld, S. Stilgenbauer, Peter Möller, Nensi Ikonomi, Hans A. Kestler
          </td>
          <td>2025-09-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The combination of 5-fluorouracil and irinotecan (FOLFIRI) remains a standard-of-care treatment for metastatic colorectal cancer (mCRC), yet benefits only about half of patients. Using patient-derived xenografts (PDXs), we investigated the biological underpinnings of this heterogeneous response. FOLFIRI-resistant models showed transcriptional upregulation of innate immunity and mitochondrial metabolism genes, together with reduced expression of the DNA polymerase POLD1. Sensitive counterparts exhibited a BRCAness-like phenotype with genomic scars of homologous recombination (HR) deficiency, not caused by genetic or epigenetic loss of HR genes but by low abundance of the RAD51 recombinase. In tumoroids, forced RAD51 overexpression attenuated HR deficiency-related scars and chemotherapy-induced damage, while HR inhibition through ATM blockade enhanced drug sensitivity. The predictive relevance of key response determinants was validated in clinical samples. This work illuminates functional, non-genetic facets of BRCAness in mCRC and introduces actionable biomarkers and targets, offering prospects to improve clinical decision-making and broaden therapeutic options for chemorefractory patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb783603c147c9c964fbfcc1a8e05ff7e8e4445" target='_blank'>
              The molecular and functional landscape of resistance to FOLFIRI chemotherapy in metastatic colorectal cancer.
              </a>
            </td>
          <td>
            Marco Avolio, S. Leto, F. Sassi, B. Lupo, Elena Grassi, Irene Catalano, E. Zanella, V. Vurchio, F. Cottino, Petros K Tsantoulis, L. Lazzari, Paolo Luraghi, Martina Ferri, Francesco Galimi, E. Berrino, S. Bellomo, Marco Viviani, A. Sogari, G. Mauri, F. Tosi, F. Cruciani, A. Sartore-Bianchi, S. Siena, Felice Borghi, V. Torri, E. Élez, J. Tabernero, Maria Nieva, C. Montagut, N. Tarazona, Andrés Cervantes, Sabine Tejpar, A. Bardelli, C. Marchiò, S. Marsoni, Andrea Bertotti, L. Trusolino
          </td>
          <td>2025-09-22</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Simple Summary Neuroblastoma is a dangerous childhood cancer where many genes are in the “on” position, driving an uncontrolled tumor growth. A multifunctional non-POU domain-containing octamer-binding protein (NONO) acts as a master regulator of these genes. Therefore, NONO emerges as a key culprit of the disease’s aggressiveness and resistance to therapy. This review explores how targeting NONO could offer new therapeutic strategies. We discuss the challenges of drug development and highlight recent development of chemical tools to inhibit NONO function. Successful targeting of NONO is expected to provide powerful new treatments for children with high-risk neuroblastoma, potentially overcoming the limitations of current therapies. Abstract Pediatric tumors such as neuroblastoma are characterized by a genome-wide ‘transcriptional burden’, surmising the involvement of multiple alterations of gene expression. Search for master regulators of transcription whose inactivation is lethal for tumor cells identified the non-POU domain-containing octamer-binding protein (NONO), a member of the Drosophila Behavior/Human Splicing family known for the ability to form complexes with macromolecules. NONO emerges as an essential mechanism in normal neurogenesis as well as in tumor biology. In particular, NONO interactions with RNAs, largely with long non-coding MYCN transcripts, have been attributed to the aggressiveness of neuroblastoma. Broadening its significance beyond MYCN regulation, NONO guards a subset of transcription factors that comprise a core regulatory circuit, a self-sustained loop that maintains transcription. As a component of protein–protein complexes, NONO has been implicated in the control of cell cycle progression, double-strand DNA repair, and, generally, in cell survival. Altogether, the pro-oncogenic roles of NONO justify the need for its inactivation as a therapeutic strategy. However, considering NONO as a therapeutic target, its druggability is a challenge. Recent advances in the inactivation of NONO and downstream signaling with small molecular weight compounds make promising the development of pharmacological antagonists of NONO pathway(s) for neuroblastoma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8911b531a2ccdbdcdabe70eb37ce9df87687fde" target='_blank'>
              Saying “Yes” to NONO: A Therapeutic Target for Neuroblastoma and Beyond
              </a>
            </td>
          <td>
            Sofya S. Pogodaeva, O. O. Miletina, Nadezhda V. Antipova, Alexander A. Shtil, O. Kuchur
          </td>
          <td>2025-10-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Ribosomal proteins (RPs) are essential structural and functional components of the ribosome, and their disruption during embryogenesis generally results in embryonic lethality. Nevertheless, cancer genomes frequently harbor somatic missense mutations and gene deletions in RP genes in patterns that suggest selective advantages, often linked to inactivation of the tumor suppressor protein p53 pathway. This review discusses the landscape of RP mutations in cancer and their mechanistic consequences. RP mutations are detected across multiple malignancies, including glioblastoma, melanoma, T-cell acute lymphoblastic leukemia, as well as in congenital ribosomopathies, such as Diamond-Blackfan anemia, which confer an elevated lifetime risk of developing cancer. Cancer-associated RP mutations disturb ribosome homeostasis, compromise translational fidelity, and trigger proteotoxic and ribosomal stress, yet without halting tumor growth. Some RP mutants occupy structurally sensitive positions within the ribosome, altering mRNA selectivity and quality control. In turn, cancer cells may adapt through compensatory mechanisms, including the upregulation of RP paralogs, activation of proteostasis regulators, and rewiring of stress response pathways. Rather than a loss-of-function event, an RP mutation may create a persistent ribosomal disequilibrium that fundamentally alters cellular functions. Such changes in a cancer cell could generate interesting therapeutic vulnerabilities and targets, such as dependence on stress signaling, proteasome activity, or RP paralog expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8cf8b6873dcfd04105fc54f804a07dc87de411" target='_blank'>
              Toward an understanding of cancer-associated ribosomal protein mutations
              </a>
            </td>
          <td>
            Mikael S. Lindström
          </td>
          <td>2025-10-03</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations in RNA-binding proteins are increasingly identified in cancers through tumor sequencing and are correlated with disease progression, therapy response, and overall patient outcomes, underscoring the need to study them. Here, we focus on the RNA-binding protein Poly-C binding protein 1 (PCBP1), which binds target RNAs through K-homology (KH) domains to regulate RNA fate. PCBP1 is a tumor suppressor gene and hotspot missense mutations at leucine residues 100 and 102 are observed in colorectal cancer (CRC). PCBP1 mutations have been recurrently reported in CRC genome-wide mutation studies and are associated with poor clinical outcomes; however, their effects on PCBP1 expression and function remain largely unexplored. We show that cancer-associated mutations substituting leucine 100 and 102 with glutamine, proline, or arginine destabilize PCBP1, leading to increased protein turnover. The L100/L102 residues occur at the interface of the RNA-binding KH1 and KH2 domains, and our molecular dynamics simulations show that mutations at these residues disrupt the secondary structure of PCBP1. Additionally, these mutants display increased cytoplasmic localization. Importantly, mutant PCBP1 physically interacts with wild type PCBP1 and suppresses its expression through a dominant-negative mechanism. Together, our data demonstrate that CRC-associated PCBP1 mutations destabilize the protein and act as dominant-negative variants, revealing a novel mechanism of tumor suppressor inactivation in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a7a507c9272e7ca6986d3ffe683c348245fff39" target='_blank'>
              Colorectal cancer-associated PCBP1 mutations disrupt protein stability in a dominant negative manner
              </a>
            </td>
          <td>
            Paige V. Blinkiewicz, Nicole C. Hoenes, Myra X. Afzal, Yihong Liu, Ulysses J. Hill, B. Bjarnadóttir, T. E. Burgin, P. Malaney
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Recent technological breakthroughs have enabled multidimensional phenotyping, with unprecedented single-cell resolution and genome-wide coverage, across multiple osteoarthritis (OA)-relevant tissues, such as articular cartilage, synovium, infrapatellar fat pad, and subchondral bone. The majority of the single nucleotide variations (SNVs) that have been associated with OA are located in non-protein coding regions and confer risk for disease by altering the expression level, instead of the amino acid sequence of the gene product. These data have shaped the concept of OA as a polygenic disease, where genetic factors disrupt the chromatin landscape in disease-relevant cells, leading to aberrant expression of effector genes. Pharmacologic manipulation of the OA-driving epigenetic landscape has recently emerged as an attractive path for the development of disease-modifying drugs. Novel clustered regulatory interspaced short palindromic repeats (CRISPR)-based technologies provide opportunities for precise epigenetic editing at the desired genomic regions and may allow a targeted transcriptional regulation of disease-relevant genes in disease-relevant cells. The aim of the present narrative review is to summarize the emerging data on the role of epigenetic factors and chromatin structure as calibrators of the risk for developing OA and to discuss the opportunities and challenges arising from the use of chromatin landscape to guide drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb518b2e515941622a9daa713244c8885ed8ed56" target='_blank'>
              The Articular Chromatin Landscape in Osteoarthritis
              </a>
            </td>
          <td>
            G. Kalliolias, E. Basdra, Athanasios G. Papavassiliou
          </td>
          <td>2025-10-15</td>
          <td>Cells</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Anticancer drug resistance remains a significant challenge to the efficacy of cancer treatment, with DNA repair enzymes contributing to this resistance. We hypothesized that thymine DNA glycosylases (TDGs) may be involved in anticancer drug resistance given their dual function of DNA repair and demethylation as well as investigated their possible involvement in the induction of β-catenin in SNUC5 cells resistant to 5-fluorouracil (SNUC5/5-FUR) and oxaliplatin (SNUC5/OXTR). The expression of TDG and phospho-β-catenin increased in both resistant cell types when compared to that in SNUC5 cells. Moreover, knockdown of TDG significantly suppressed phospho-β-catenin expression in both resistant cell types, resulting in enhanced sensitivity to anticancer drugs. TDG binding to the β-catenin promoter was stronger in both resistant cell types than in SNUC5 cells, showing a decreased methylation pattern in the CpG islands of the β-catenin promoter. Furthermore, another DNA demethylase, ten-eleven translocation 1 (TET1), showed the same pattern as TDG in both resistant cell types. Additionally, TDG significantly interacted more with TET1 in both resistant cell types than in SNUC5 cells, enhancing binding to the same locus in the β-catenin promoter. These findings suggest that TDG may be a promising target molecule for overcoming drug resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def655ba0cc554974966a66e65dd66380f445b21" target='_blank'>
              Involvement of the DNA Demethylase Thymine DNA Glycosylase in Anticancer Drug Resistance.
              </a>
            </td>
          <td>
            K. Kang, M. Piao, Herath Mudiyanselage Maheshika Madhuwanthi Senavirathna, Hye-Jin Boo, Sang-Pil Yoon, Joo Mi Yi, Jinny Park, J. Hyun
          </td>
          <td>2025-10-15</td>
          <td>Biomolecules & therapeutics</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/889c401af57de3889b95cf28e228d76bc1fc7c6d" target='_blank'>
              Repeated losses of self-fertility shaped heterozygosity and polyploidy in yeast evolution
              </a>
            </td>
          <td>
            Nina Vittorelli, C. Gómez-Muñoz, Irina Andriushchenko, Louis Ollivier, Nicolas Agier, Stéphane Delmas, Yann Corbeau, Guillaume Achaz, M. Cosentino Lagomarsino, Gianni Liti, Bertrand Llorente, G. Fischer
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f80032618ad0bd974f697fa45a97d919565be754" target='_blank'>
              Selective degradation of oncogenic extrachromosomal DNA by type I-E CRISPR
              </a>
            </td>
          <td>
            Ensieh Poursani, Vu v. Pham, A. Ehteda, Orazio Vittorio
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Posttranslational modifications (PTMs) on histones play critical roles in cellular processes, including gene expression and tumorigenesis. However, the regulatory mechanisms and functional consequences of newly identified lysine acylation modifications in cancer therapy remain to be elucidated. Here, we established diverse preclinical tumor models resistant to epithelial growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) and focused on histone lysine crotonylation (Kcr), as it exhibited more pronounced alterations compared to other acylations. Next, we identified acyl-CoA synthetase short-chain family member 2 as a key regulator responsible for the resistance-associated decrease in crotonylation levels. Furthermore, integrated crotonylomic, transcriptomic, and epigenomic profiling, supplemented by gene manipulation studies, revealed that EGFR-TKI resistance resulted from transcriptional suppression of HNF1A and activation of the PI3K/AKT signaling pathway, which were regulated by reduced histone H3 lysine 56 crotonylation. Importantly, through pharmacological screening, we identified a histone decrotonylase inhibitor that enhanced EGFR-TKI sensitivity by activating epigenetic reprogramming through the selective upregulation of histone Kcr levels across multiple models in vitro and in vivo. Collectively, our findings uncover a previously unrecognized epigenetic mechanism driven by crotonylation that contributes to EGFR-TKI resistance, highlighting the potential of modulating crotonylation as a novel therapeutic strategy to enhance the efficacy of EGFR-TKIs in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9119d6e3343091d67a219621bccb08e3e1171344" target='_blank'>
              Activation of epigenetic reprogramming via crotonylation overcomes resistance to EGFR-TKI therapy in lung cancer.
              </a>
            </td>
          <td>
            Sihong Chen, Mengyan Zhong, Xiaoyuan Wang, Youwei Su, Yingxi Zhao, Simeng Wang, Shougeng Liu, Yongshan Zhao, Yang Zhang, Xinyu Fan, Zongang Liu, Lina Jia, Xuefei Bao, Wei Cui, Jingyu Yang, Chun-Hui Wu, Guoliang Chen, Lihui Wang
          </td>
          <td>2025-09-30</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third leading cause of cancer-related deaths. Integrating cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present a comparative mRNA-seq analysis of tissue samples from 32 CRC patients, pairing tumors with adjacent healthy tissues. Differential expression gene (DEG) analysis revealed dysregulated metabolic programs. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2, which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potential of the Xc- system, we analyzed gene perturbations from CRISPR screens across various cell types and used functional assays in five primary patient-derived organoid models. We identified a previously uncharacterized cell surface protein signature predicting chemotherapy resistance and highlighted the causality and potential of pharmacological blockage of ferroptosis as a promising avenue for cancer therapy. Redox homeostasis, ion/amino acid transporters, and regulators of neuronal survival and differentiation were pathways associated with these co-dependent genes in patient specimens. This study highlights several potential clinical targets for CRC therapy and promotes the use of patient-derived organoids oids to functionally validate in silico predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af3aa92ab85625c6d369fc2c084fc1ee49ca6118" target='_blank'>
              Patient-specific pharmacogenomics demonstrates xCT as predictive therapeutic target in colon cancer with possible implications in tumor connectivity.
              </a>
            </td>
          <td>
            M. Strecker, Keren Zohar, Martin Böttcher, T. Wartmann, H. Freudenstein, Maximilian Doelling, M. Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, D. Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. Kahlert
          </td>
          <td>2025-09-24</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Despite decades of progress in ovarian cancer treatment, therapeutic resistance remains a persistent and lethal challenge. Emerging evidence suggests polyploid giant cancer cells (PGCCs) as key drivers of this resistance. PGCCs are distinguished by multinucleation or a single oversized nucleus containing multiple chromosome sets. Polyploid cell populations are present in premalignant lesions and expand with disease progression and therapeutic stress. PGCCs demonstrate heightened resistance to treatment relative to non-PGCCs and can regenerate non-PGCCs via budding once therapeutic pressure is alleviated, leading to relapse. Supported by extensive preclinical and clinical evidence, we hypothesize that elimination of PGCCs may reduce therapeutic resistance in ovarian cancer. Given their significant role in resistance and relapse, PGCCs remain largely untargeted due to a lack of scalable detection methods. Standard viability assays such as MTT, XTT, and ATP-based screens fail to capture the rare but critical PGCC subpopulation. Current gold-standard methods, fluorescence-activated cell sorting (FACS) coupled with manual microscopy, while accurate, are time-intensive and unsuitable for high-throughput drug screening. Thus, a rapid and accurate technique for studying PGCCs is essential. In this pilot study, we have developed a high-throughput single-cell morphological analysis pipeline capable of precisely quantifying PGCC numbers and proportions. The pipeline accurately quantifies PGCCs based on nuclear and cellular features, including size and DNA content. Leveraging this platform, we screened a library of 2,726 FDA-approved compounds to identify agents that effectively kill PGCCs. Hits from this screen were validated in 3D spheroid cultures, revealing compounds with potent anti-PGCC activity. Complementary transcriptomic analysis (RNA-Seq) of flow-sorted PGCCs versus non-PGCCs unveiled profound dysregulation of the FOXM1 signaling axis, implicating this pathway as a key regulator of PGCC formation and survival. Consistent with this, the high-throughput screen identified Thiostrepton, a known FOXM1 inhibitor, as a selective anti-PGCC agent. Across multiple ovarian cancer cell lines, FOXM1 inhibitors outperformed standard-of-care agents (Paclitaxel, Cisplatin, Olaparib) in eliminating PGCCs. siRNA knockdown of FOXM1 significantly reduced PGCC numbers, particularly under Paclitaxel stress. Moreover, FOXM1 inhibition triggered apoptosis in PGCCs, an effect reversible with the pan-caspase inhibitor Z-VAD-FMK. Together, these findings underscore PGCCs as a therapeutically actionable cell state driving resistance in ovarian cancer. Our integrative single-cell morphological and transcriptomic platform enables rapid discovery of anti-PGCC agents and establishes FOXM1 inhibition as a compelling strategy to eradicate this resilient subpopulation. This pipeline offers broad utility for overcoming therapy resistance across diverse malignancies.



 Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen. Combating ovarian cancer resistance via polyploid cancer cell targeted screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A023.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62f27ef6b0280aa103e9f9c96579bbfa985df28c" target='_blank'>
              Abstract A023: Combating ovarian cancer resistance via polyploid cancer cell targeted screening
              </a>
            </td>
          <td>
            Hsiao-Chun Chen, Yuan Zhang, Yushu Ma, Yu-Chih Chen
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background EME1, a critical DNA repair endonuclease, has emerged as a potential oncogene implicated in genome instability and cancer progression. However, its pan-cancer roles, prognostic significance, immune interactions, and therapeutic targeting remain underexplored. Methods We conducted a comprehensive pan-cancer analysis integrating multi-omics data from public databases, including TIMER2.0, GEPIA2, TISIDB, and cBioPortal, to evaluate EME1 expression, genetic alterations, and their association with clinical outcomes, immune infiltration, and molecular pathways. Virtual screening of 3180 FDA-approved drugs and molecular dynamics (MD) simulations were employed to identify and validate potential EME1 inhibitors. Results EME1 was significantly overexpressed in various human cancers and positively associated with advanced tumor grade and stage. High EME1 expression and mutations were linked to poor overall and disease-free survival. Immunogenomic profiling revealed strong positive correlations between EME1 and myeloid-derived suppressor cells (MDSCs), alongside a negative association with endothelial cell function, suggesting immunosuppressive roles. Machine learning models based on EME1-associated genes demonstrated high predictive accuracy for liver hepatocellular carcinoma (AUC > 0.90). Virtual screening identified eight promising drug candidates, including Everolimus and Dioscin, with strong binding affinities. MD simulations confirmed the stability of these interactions, particularly for Dioscin. Conclusion This study reveals the multifaceted oncogenic roles of EME1 in tumor progression, immune evasion, and prognosis. It proposes EME1 as a promising biomarker and therapeutic target across multiple cancer types. The identified drug candidates warrant further in vitro and in vivo validation for potential repurposing in EME1-targeted cancer therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03631-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2914997c524dd90bc8f262d9d5d651e9af6b9f8" target='_blank'>
              Oncogenic EME1 promotes tumor progression and immune modulation in human cancers with therapeutic targeting potential
              </a>
            </td>
          <td>
            Muhammad Alaa Eldeen, Abdelrahman Mostafa, Farag Mamdouh, Waleed K. Abdulsahib, D. Alshaya, Eman Fayad, Hassan M. Otifi, Hesham M. Hassan, Mohammed A. Alshehri, A. Althobaiti, Ghadi Alsharif, Mohamed A. Soltan
          </td>
          <td>2025-10-13</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="The heterogeneity and dynamic evolution of the tumor microenvironment (TME) are key drivers of tumor progression, immune evasion, and therapeutic resistance. Traditional transcriptomic approaches are limited by their reliance on population-averaged data and the lack of spatial context, making it challenging to fully dissect the complex interactions among tumor cells, immune infiltrates, and stromal components within the TME. In recent years, the synergistic advancement of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has revolutionized our ability to construct high-resolution cellular atlases and delineate spatially organized functional modules within the TME. These technologies have enabled the identification of novel cell subtypes, cell-cell communication pathways, and spatial niches associated with disease progression and therapy response. This review systematically compiles recent technological breakthroughs in the integration of single-cell and spatial multi-omics, highlights key discoveries in TME research, and discusses their transformative impact on the development of precision oncology. Additionally, it addresses current technical limitations and explores emerging strategies and future directions for advancing this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a0c36ae523e1a4fc23b5c8bc8013f692b5b483" target='_blank'>
              Refining Tumor Microenvironment Insights via Single-Cell and Spatial Transcriptomics for Precision Medicine
              </a>
            </td>
          <td>
            Xianbing Wang
          </td>
          <td>2025-09-05</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Resistance to programmed cell death is a defining hallmark of cancer and a persistent barrier to successful therapy. Dual-function proteins such as p53, Ras, HIF-1α, BNIP3, and NF-κB act as molecular switches that determine cell fate between apoptosis and survival. In tumors, these proteins are deregulated not only by intrinsic mutations but also by extrinsic signals from the tumor microenvironment (TME). This Mini Review critically analyzes previous therapeutic approaches, emphasizing overlooked mechanisms such as Ras-mediated suppression of p53. It proposes a sequential therapeutic strategy: first, dismantling TME adaptations (hypoxia, inflammation, protective autophagy); second, inhibiting oncogenic Ras signaling; and third, restoring p53 activity. The phased approach integrates biomarker-guided patient stratification, recognizes tumor–microenvironment co-evolution, and highlights how resistance evolves over time. Although the concept does not resolve all challenges, it outlines a rational framework for restoring apoptotic competence and provides a pathway for translational and clinical testing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79ed9bca532a19e58792704a40e38539d4ea37e5" target='_blank'>
              Context-dependent rewiring of dual-function proteins in cancer: a sequential strategy to restore apoptosis
              </a>
            </td>
          <td>
            Leon Strzadala
          </td>
          <td>2025-10-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434dbd23a412e8c919acd85ad46264f7a360e2f6" target='_blank'>
              Long-read sequencing reveals telomere inheritance patterns from human trios
              </a>
            </td>
          <td>
            Yuxin Zhou, David R. Lougheed, Warren A. Cheung, I. Thiffault, T. Pastinen, Guillaume Bourque
          </td>
          <td>2025-10-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Aim: Cisplatin resistance in head and neck squamous cell carcinoma (HNSCC) is thought to involve both reduced drug uptake and altered molecular responses. However, the relative contribution of these mechanisms remains unclear. Methods: Two HNSCC cell lines with differing sensitivity (HNO97 and HNO41) were analyzed using cytotoxicity assays, atomic absorption spectroscopy-based quantification of intracellular cisplatin, caspase 3/7 assays, Western blotting, polymerase chain reaction (PCR)-based transcriptomic analysis of DNA damage response and cell cycle arrest pathways, and RNA-seq data from The Cancer Genome Atlas (TCGA) to characterize the resistance phenotype. Results: HNO97 (IC50 = 440 µM) was 7.6-fold more resistant to cisplatin than HNO41 (IC50 = 57.8 µM; P = 0.0286). After quantifying intracellular uptake (pg Pt/µg protein) and normalizing cytotoxicity to intracellular drug levels, HNO97 (IC50 = 778.9 pg Pt/µg protein) remained 5-fold more resistant than HNO41 (IC50 = 153.5 pg Pt/µg protein), indicating only a partial reduction in resistance (33% decrease, from 7.6-fold to 5-fold; P = 0.0286). At cisplatin concentrations yielding comparable intracellular exposure (HNO97: 440 µM; HNO41: 196 µM; both ≈ 725 pg Pt/µg protein), caspase 3/7 activation and induction of CDKN1A, GADD45A, GADD45G, and PPP1R15A were weaker in HNO97 than in HNO41. Notably, baseline expression of these genes was significantly higher in HNO97. In the TCGA cohort, multivariate analysis showed that high FANCD2 expression was associated with unfavorable recurrence-free survival in platinum-treated patients (hazard ratio = 4.0; P = 0.011), but not in those who did not receive platinum chemotherapy. Conclusion: Cisplatin resistance in HNSCC appears to be driven primarily by molecular mechanisms involving DNA damage response and cell cycle arrest pathways, rather than poor drug uptake.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba27336a14cf094e2bc0e409273ca8370a971da1" target='_blank'>
              Cisplatin resistance in head and neck squamous cell carcinoma is linked to DNA damage response and cell cycle arrest transcriptomics rather than poor drug uptake
              </a>
            </td>
          <td>
            Ketaki Sandu, R. Warta, Uddipta Biswas, Wang Zhang, Patrick Michl, Christel Herold-Mende, Johanna Weiss, Dirk Theile
          </td>
          <td>2025-09-19</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cisplatin resistance presents a significant challenge in cancer therapy, emphasizing the necessity for identifying new regulatory elements that influence drug response. Recent research has revealed the importance of long noncoding RNAs (lncRNAs) in chemotherapy resistance, with LINC02381 identified as a potential regulatory factor. Through an in-depth bioinformatics analysis, we investigated the impact of LINC02381 on cisplatin resistance in ovarian cancer across various datasets. By conducting differential expression analysis, survival analysis, gene set enrichment analysis (GSEA), and constructing protein-protein interaction (PPI) networks, we identified key pathways associated with LINC02381 expression. The results indicated that the altered expression of LINC02381 in patients treated with cisplatin was associated with reduced survival. Functional studies and correlation analyses further demonstrated that this LncRNA influences critical pathways and genes related to apoptosis, efflux, DNA repair, and EMT. Lastly, through an examination of its interactions with microRNA and protein networks, we identified LINC02381 as a ceRNA implicated in cisplatin resistance. Our findings suggest that LINC02381 may influence cisplatin sensitivity in ovarian cancer and establish a basis for further experimental validation, including molecular assays or in vivo analyses, and suggest the potential therapeutic targeting of LINC02381 to combat chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a8ae366b9c19996905f6b9e1ed9f603948832f" target='_blank'>
              The Role of LINC02381 in Modulating Cisplatin Resistance in Ovarian Cancer: A Bioinformatics Approach.
              </a>
            </td>
          <td>
            Zeinab Karbalaei Pazoki, B. M. Soltani, Mostafa Hosseini, Shiva Irani
          </td>
          <td>2025-09-16</td>
          <td>Iranian journal of allergy, asthma, and immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Mutational activation of the KRAS oncogene in pancreatic ductal adenocarcinoma (PDAC) plays a crucial role in tumor initiation, progression, and alterations of cellular programs that support PDAC growth. Mutant KRAS alters the metabolic state of PDAC and has been linked to upregulation of glycolysis and alterations in the nutrient scavenging pathways autophagy and macropinocytosis. Our lab determined that genetic suppression or inhibition of the RAS pathway unexpectedly resulted in upregulation of autophagy. Additionally, concurrent inhibition of the RAS-MAPK pathway and autophagy synergistically improved anti-tumor efficacy in both PDAC cell lines, organoids, and PDX tumor models. To begin to address the temporal regulation of metabolic signaling following RAS inhibition, we examined the sustainability of upregulated autophagic flux following prolonged (168 h) RAS inhibition. Preliminary data indicates that autophagy upregulation following KRAS inhibition is transient and returns to basal levels following prolonged inhibitor treatment. Given the limited durability of autophagic induction upon RAS inhibition, we sought to determine whether other nutrient scavenging pathways that culminate in the lysosome were altered following prolonged RAS inhibitor treatment. Macropinocytosis is upregulated in PDAC and is RAS-dependent. We monitored macropinocytic uptake following direct inhibition of RAS, and found that following an initial decrease in macropinocytosis, PDAC cells exhibited a compensatory increase in macropinocytotic uptake and utilization at later timepoints. Based on this data, we have begun to interrogate the metabolic phenotypes associated with cells subjected to acute (24-72 h) and prolonged (168 h) RAS inhibitor treatment. Furthermore, we have also derived PDAC cell lines that are resistant to RAS inhibition via prolonged inhibitor dose escalation. We found that RAS inhibitor resistant cell lines have heterogenous capacities for autophagic flux and retain similar sensitivity to autophagy inhibitors. Conversely, resistant cells display persistently increased macropinocytosis, suggesting that that scavenging pathway compensates for metabolic requirements of growth under sustained RAS inhibition. Future studies are aimed at investigating the molecular underpinnings driving upregulated macropinocytosis and more broadly profiling the metabolic landscape of RAS-inhibitor resistant PDAC models.



 Sarah E. Ackermann, Ryan Robb, Khalilah E. Taylor, Runying Yang, Kirsten L. Bryant. Assessing metabolic requirements for RAS inhibitor resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509f9b16a108ea5a5b93f12330dee22b394bce19" target='_blank'>
              Abstract A006: Assessing metabolic requirements for RAS inhibitor resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Sarah E Ackermann, Ryan Robb, Khalilah E Taylor, Run-Wei Yang, Kirsten L. Bryant
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background/Objectives: Liquid biopsy using cfDNA has emerged as a promising, minimally invasive alternative to traditional tissue biopsy for detecting cancer-associated mutations. However, the extremely low proportion of mutant DNA in cfDNA poses a major challenge for accurate detection, especially when using conventional sequencing methods. To address this limitation, we sought to develop a highly sensitive diagnostic strategy to selectively enrich rare mutant sequences and improve the detection of clinically important mutations in patients with NSCLC. Methods: We established a CRISPR/Cas12a-based diagnostic system designed to selectively cleave WT DNA, thereby increasing the relative abundance of mutant DNA in cfDNA samples. Following Cas12a-mediated WT cleavage, the remaining DNA was subjected to PCR amplification for mutation identification. The system was applied to plasma cfDNA from blood samples of 48 NSCLC patients to evaluate its ability to detect two major EGFR mutations: L858R and exon 19 deletion. Results: The CRISPR/Cas12a-based diagnostic system effectively identified low-frequency EGFR mutations in cfDNA. Specifically, all 7 L858R-positive samples and 6 out of 11 samples harboring exon 19 deletions—previously validated through tissue biopsy—were successfully detected. This demonstrated a high degree of concordance between our liquid biopsy approach and conventional diagnostic methods. Conclusions: Our findings highlight the potential of the CRISPR/Cas12a-based mutation enrichment system as a powerful tool for detecting rare oncogenic mutations in liquid biopsy samples. This technique enhances diagnostic sensitivity and could be broadly applicable for the non-invasive detection of various genetic alterations in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/068b8935c94cdd53865ee3f28471c9d10867ede9" target='_blank'>
              CRISPR with a Double Mismatch Guide RNA Enhances Detection Sensitivity for Low-Frequency Single-Base EGFR Mutation in Circulating Cell-Free DNA of Lung Cancer Patients
              </a>
            </td>
          <td>
            K. W. Been, Seunghun Kang, Taegeun Bae, Sumin Hong, Garyeong Kim, J. Hur, Woochang Hwang, Boksoon Chang
          </td>
          <td>2025-10-16</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the second leading cause of cancer deaths, with a five-year survival rate of 65%. Oxaliplatin was the first platinum drug shown to improve CRC patient outcomes and is now a common adjuvant therapy for advanced disease, yet 90% of patients develop resistance. Oxaliplatin was developed as a third-generation derivative of cisplatin, but recent evidence points to divergent modes of action. Genome-wide CRISPR activation and knockout screens were conducted to identify genetic changes that confer resistance to oxaliplatin in two CRC cell lines with distinct molecular backgrounds (SW620 and RKO). Guide RNAs corresponding to the neutral amino acid transporter SLC43A1 (LAT3) were the most significantly enriched in knockout screens and depleted in activation screens, suggesting a potential role for LAT3 in modulating oxaliplatin resistance. In vitro CRISPR knockout and overexpression of LAT3 in SW620 and RKO cell lines confirm increased resistance or sensitivity to oxaliplatin, respectively, and this correlated with levels of intracellular accumulation of oxaliplatin and DNA damage, demonstrating that enhanced LAT3-mediated uptake of oxaliplatin can exert its expected mechanism of action and induce cytotoxicity. Further studies to investigate the specificity of this mechanism of resistance to oxaliplatin in CRC and other cancers are ongoing to clarify the molecular basis of oxaliplatin resistance. These findings suggest a novel use for LAT3 as a potential biomarker and predictor of oxaliplatin sensitivity and can provide insights into the interplay between essential nutrient uptake and drug transport.



 Nisha R. Pawar, Heidi M. Wade, Allison V. Mitchell, Connor P. Jewell, Lisa M. Jenkins, Robert W. Robey, Michael M. Gottesman. A whole genomic CRISPR-Cas9 screen identifies the LAT3 (SLC43A1) neutral amino acid transporter as a major determinant of oxaliplatin resistance in colorectal cancer cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb9e1d72d03ca8da071d5c00ee84e7d75c353cf" target='_blank'>
              Abstract A039: A whole genomic CRISPR-Cas9 screen identifies the LAT3 (SLC43A1) neutral amino acid transporter as a major determinant of oxaliplatin resistance in colorectal cancer cells
              </a>
            </td>
          <td>
            Nisha R Pawar, Heidi M Wade, Allison V. Mitchell, Connor P. Jewell, Lisa M. Jenkins, Robert W. Robey, Michael M. Gottesman
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee655efaca68829593073257505b746a95fc7049" target='_blank'>
              Non-canonical DNA in bird telomere-to-telomere genomes
              </a>
            </td>
          <td>
            Linnéa Smeds, Simona Secomandi, Chul Lee, Francesca Chiaromonte, Erich D. Jarvis, G. Formenti, Kateryna D. Makova
          </td>
          <td>2025-10-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="This review analyzed approximately 115 peer-reviewed studies published between 2010 and 2025, focusing on molecular subtyping and circulating tumor DNA (ctDNA)-guided approaches in Diffuse Large B-Cell Lymphoma (DLBCL). Evidence was synthesized from retrospective cohorts, prospective clinical trials, and translational studies, highlighting how molecular heterogeneity, clonal evolution, and the tumor microenvironment complicate classification and treatment. While molecular subtypes such as MCD, BN2, EZB, A53, and ST2 have improved prognostication, their routine use in clinical practice remains limited due to cost, complexity, and restricted access to sequencing platforms. Tumor-informed ctDNA assays show promise for minimal residual disease (MRD) monitoring and adaptive therapy, yet their predictive power for CAR-T therapy, bispecific antibodies, and checkpoint inhibitors is still incompletely understood. Overall, the literature converges on the need for integrated strategies combining ctDNA, molecular subtyping, and immune microenvironment analysis to personalize frontline therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5768d4ecc136da89bacb2b8af1b5cdad6bfb931" target='_blank'>
              Personalized Frontline Therapy in Diffuse Large B-Cell Lymphoma: Integrating Circulating Tumor DNA for Real-Time Adaptive Treatment Stratification
              </a>
            </td>
          <td>
            Vasisht Karri, Samir Dalia
          </td>
          <td>2025-10-17</td>
          <td>Lymphatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) remains a major global health burden, contributing significantly to morbidity and mortality worldwide. Despite advances in understanding the molecular landscape of CRC and the development of targeted therapies, there is still a critical need for novel treatment strategies that address specific molecular vulnerabilities in CRC. To address this challenge, we developed a robust and scalable platform for target discovery, centered on identifying synthetic lethal (SL) interactions linked to key oncogenic drivers such as APC, KRAS, and SMAD4. Using colorectal cancer (CRC) as a model, we integrated genomic and transcriptomic analyses across a diverse set of biologically relevant systems, including human intestinal stem cell (hISC)-derived isogenic cancer models that accurately replicate tumor genotypes, patient-derived cells (PDCs), and clinical tumor specimens. At the heart of our platform is CRISPR-based functional screening, enabling systematic interrogation of gene dependencies. To benchmark our engineered models, we applied machine learning to RNA sequencing data, comparing them against a large cohort of clinically annotated CRC transcriptomes. This approach enabled precise molecular subtyping, detailed characterization of gene expression programs, and the identification of novel, tractable vulnerabilities specific to genetically defined subgroups. Candidate targets were prioritized based on therapeutic relevance and selective efficacy in cancer cells, while sparing untransformed hISCs. Our findings not only advance early-stage target identification and validation but also support the initiation of precision drug discovery programs focused on CRC subtypes driven by APC, KRAS, and SMAD4 mutations. Moreover, the versatility of our platform enables its application to other malignancies. In summary, we present an integrated, high-resolution framework for uncovering novel molecular dependencies in CRC, providing a foundation for precision oncology and future therapeutic innovation.



 Eliza Zimoląg, Joanna Szuszkiewicz, Marcin Serocki, Marcin Duleba, Oleksii Bryzghalov, Izabella Więckowska, Marcelina Chmiel, Barbara Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, Michał Mikula, Rafał Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan. Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a3fb0b893ce7123a2e4b7841128bf8eca7bc83" target='_blank'>
              Abstract B050: Identification of novel molecular vulnerabilities in colorectal cancer through integrated transcriptomic profiling and functional genomics
              </a>
            </td>
          <td>
            E. Zimoląg, J. Szuszkiewicz, M. Serocki, M. Dulęba, O. Bryzghalov, Izabella Więckowska, M. Chmiel, B. Lipert, Katarzyna Sarad, Julia Wirkijowska, Szymon Worszyło, Dariusz Klonski, M. Mikula, Rafal Dziadziuszko, Andrew Thomason, Krzysztof Brzózka, Andrzej Mazan
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="M2-type pyruvate kinase (PKM2) serves as the key rate-limiting enzyme in aerobic glycolysis within tumor cells, where its aberrantly high expression in numerous human malignancies facilitates tumor progression by enhancing glycolytic flux through diverse signaling pathways. Beyond its metabolic function, extensive studies have established PKM2 as a critical non-metabolic signaling regulator implicated in multiple oncogenic processes, including tumor proliferation, invasion, migration, immune evasion, and resistance to chemotherapy. The elucidation of PKM2-mediated oncogenic pathways has spurred the development of targeted therapeutic strategies, positioning PKM2 as a promising target in cancer therapy. However, comprehensive reviews addressing the relationship between PKM2 and tumorigenesis remain limited. This review systematically examines the biological functions of PKM2, the signaling mechanisms through which it exerts its effects in malignant tumors, and the latest advances in the development of PKM2-targeted therapeutics, offering insights into potential directions for future drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00425408db2494b0cf2536b5348cfea19b9b7329" target='_blank'>
              Targeting PKM2 in cancer therapeutics: mechanistic advances and translational opportunities
              </a>
            </td>
          <td>
            Lin Liu, Junyi Wang, Libo Liang, Chunhua Chen, Meng Xie, Wencheng Tang
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) is the most common malignant primary brain tumor in adults. Cadherin-3 (or P-cadherin) is a cell adhesion molecule encoded by the CDH3 gene, which we previously identified as a key contributor to GBM aggressiveness. Despite its functional effects, the underlying mechanisms through which Cadherin-3 affects GBM remain unclear. This study aims to elucidate the molecular, cellular, and metabolic processes mediated by CDH3 that drive GBM.



 RNA sequencing was performed in two GBM models, including a patient-derived culture and a cell line, genetically manipulated to express differential levels of CDH3. Metabolic profiling was conducted via Seahorse analyses and extracellular glucose and lactate quantifications by HPLC. Lactate levels were further measured in the interstitial fluid of subcutaneous xenograft tumors. Protein expression of metabolism-associated markers was evaluated through western blot and immunohistochemistry in GBM cells and in in vivo tumors. The Cancer Genome Atlas (TCGA) data was used to study correlations between CDH3 and metabolic gene signatures, and associations with patient survival.



 Transcriptomic analyses revealed that CDH3 expression is associated with multiple oncogenic pathways, including those related to metabolism. Findings from independent GBM cell models indicate that Cadherin-3 influences energy metabolism in a cell context-dependent manner, affecting the balance between glycolysis and oxidative phosphorylation. In GBM patient samples, CDH3 expression is positively correlated with glycolytic/plurimetabolic (GPM)-related genes, and inversely correlated with mitochondrial (MTC) signature genes. Notably, high CDH3 expression combined with the GPM signature defines a subgroup of patients with particularly poor survival outcomes.



 These findings suggest Cadherin-3 as a key regulator of GBM metabolism, shaping distinct energetic profiles depending on the cellular context. Furthermore, its association with the GPM signature identifies a subset of GBM patients with worse prognosis, underscoring the potential for CDH3 as both a biomarker for patient stratification and a possible therapeutic target in GBM.



 Financial support was provided by FCT - Fundação para a Ciência e a Tecnologia - https: //doi.org/10.54499/2022.04859.PTDC, PD/BDE/143154/2019, COVID/BDE/153298/2023, co-funded by MCTES, FSE, UE, and NORTE 2020; and by National funds through FCT - https: //doi.org/10.54499/UIDB/50026/2020, https: //doi.org/10.54499/UIDP/50026/2020, and https: //doi.org/10.54499/LA/P/0050/2020, and by the projects NORTE-01-0145-FEDER-000039, and NORTE-01-0145-FEDER-000055, supported by NORTE 2020, under the PORTUGAL 2020 Partnership Agreement through the European Regional Development Fund (ERDF).
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86af8fd649dc07baf379d8b4dbce9b39b94fac35" target='_blank'>
              P01.23.A CADHERIN-3-MEDIATED METABOLIC REPROGRAMMING AND PROGNOSTIC STRATIFICATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            E. Martins, C. Gonçalves, J. Peixoto, J. M. Ferreira, S. Queirós, A. Longatto-Filho, P. M. Costa, N. S. Osório, S. Granja, B. Costa
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Kyinno provides TMD8 cell lines with endogenous BTK resistance mutations (e.g., C481S, L528W, V416L, T474I, A428D) under native regulation, enabling detailed studies of covalent and non-covalent inhibitor responses. These models faithfully recapitulate clinical escape alleles, supporting both mechanistic investigations and drug-resistance profiling.



 High-fidelity CRISPR/Cas9 RNP electroporation with 200-bp ssODN donors and proprietary HDR enhancers routinely yields >80 % BTK knock-in efficiency. Following optimized limiting-dilution cloning, each clone undergoes genomic PCR, Sanger sequencing, and RT-PCR to confirm on-target integration, allelic zygosity.



 Mutant TMD8 lines display broad IC50 shifts: A428D and L528W confer >100-fold resistance to both ibrutinib and pirtobrutinib, C481S remains nanomolar to pirtobrutinib, whereas T474I/V416L stay responsive to zanubrutinib; all variants expect A428D retain low-nanomolar BGB-16673 and NX-5948 susceptibility. In vivo, C481F tumors show modest ibrutinib delay, whereas L528W xenografts resist ibrutinib yet regress under daily NX-5948 without overt weight loss. Proliferation assays mirror these drug-response patterns across the panel.



 Kyinno’s BTK-mutant lines accelerate high-throughput screening of small-molecule inhibitors and targeted degraders, delineate resistance circuitry via isogenic comparisons, and improve translational relevance by faithfully modeling patient-derived escape variants. This platform supports rapid go/no-go decision-making in early drug discovery and biomarker identification for next-generation BTK therapies.



 Jinying B. Ning, feng B. hao, yue B. huang, Thomas A. Strong. BTK-EndoMut Platform: Engineered Resistance Models Accelerating BTK-Targeted Drug Discovery [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr B099.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723b954bdd0f1f0d06162f08a8fe3aac141048aa" target='_blank'>
              Abstract B099: BTK-EndoMut Platform: Engineered Resistance Models Accelerating BTK-Targeted Drug Discovery
              </a>
            </td>
          <td>
            Jinying B. Ning, feng B. hao, yue B. huang, Thomas A. Strong
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BCL-2 is a central regulator of apoptosis and inhibits cell death by sequestering pro-apoptotic BH3 alpha-helices within a hydrophobic surface groove. While venetoclax, a BH3-mimetic drug, has transformed the treatment of BCL-2–driven malignancies, its efficacy is increasingly limited by acquired resistance mutations that disrupt small-molecule binding yet preserve anti-apoptotic function—reflecting a remarkable structural adaptation. Here, we employ hydrocarbon-stapled alpha-helices derived from the BAD BH3 motif as conformation-sensitive molecular probes to investigate this therapeutic challenge. The stapled peptides not only retain high-affinity binding to all BCL-2 variants but also show enhanced potency to select venetoclax-resistant mutants. Structural analyses, including X-ray crystallography and hydrogen-deuterium exchange mass spectrometry (HDX MS), demonstrate that these stapled helices restore native BH3 engagement by reversing the conformational consequences of resistance mutations. Notably, we identify a serendipitous interaction between the α3–α4 region of BCL-2 and hydrocarbon staple, which further compensates for altered groove conformation and contributes to mutant binding affinity. Together, these findings offer mechanistic insights into BCL-2 drug resistance and reveal a blueprint for designing next-generation inhibitors that overcome this clinically significant barrier to durable treatment responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c989e0417fad14adefa4b6808df4b60e2e5666a7" target='_blank'>
              Structural insights into chemoresistance mutants of BCL-2 and their targeting by stapled BAD BH3 helices
              </a>
            </td>
          <td>
            Thomas M. DeAngelo, Utsarga Adhikary, Kyle J. Korshavn, H. Seo, Clara R. Brotzen-Smith, Christina M Camara, S. Dhe-Paganon, Gregory H. Bird, T. Wales, L. Walensky
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Radiotherapy (RT) is a crucial treatment for hepatocellular carcinoma (HCC); however, resistance to radiation remains a major challenge to its effectiveness. Circular RNAs (circRNAs), traditionally considered stable non-coding RNAs, have recently gained attention for their potential to encode proteins or peptides that play crucial roles in HCC radioresistance. In this study, circSNX25 (has_circ_0004874) was identified as a novel circRNA with coding potential related to radiation resistance in HCC through circRNA sequencing, Northern blotting, and mass spectrometry (MS). We demonstrated that SNX25-215, a novel protein encoded by circSNX25, rather than the parental circSNX25 itself, promotes resistance to radiotherapy in HCC cells both in vitro and in vivo. Co-immunoprecipitation (co-IP) and molecular docking revealed that amino acids H207 and E214 of SNX25-215 are critical for binding to Golgi to ER traffic protein 4 homolog (GET4). This interaction inhibits the binding of BCL2-associated athanogene 6 (BAG6) to GET4, thereby exposing the nuclear localization sequence (NLS) of BAG6 and facilitating its nuclear translocation. This, in turn, enhances DNA damage repair, ultimately increasing resistance to ionizing radiation (IR) in HCC cells. Importantly, elevated levels of SNX25-215 lead to nuclear localization of BAG6, endowing HCC cells with radioresistant activity, which is further supported by clinical evidence. Our findings highlight the potential of circSNX25 as a prognostic biomarker and a therapeutic target for overcoming radioresistance in HCC. This study provides deeper insights into the roles that circRNA-encoded proteins play in HCC radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16daa4d6e57dc11c1f72d49cb0e599242e4afab2" target='_blank'>
              Circular SNX25 encoded radioresistance augmenter facilitates DNA damage repair in hepatocellular carcinoma by targeting BAG6-GET4 interaction
              </a>
            </td>
          <td>
            Shuping Li, Ying Gao, Yuhao Tang, Aoran Dong, Shuifang Hu, Ruizhi Wang, Xiaoyong Gong, Zhenwei Peng
          </td>
          <td>2025-10-21</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a60db9a10da32fa1ae50df98fedcabbcf5befcb" target='_blank'>
              Mutation bias in driver genes reveals the distribution of effects of oncogenic mutations
              </a>
            </td>
          <td>
            Anastasia V. Stolyarova, G. Bazykin
          </td>
          <td>2025-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The disruption of the epigenetic mechanisms of gene expression regulation due to the emergence of pathogenic variants in genes-encoding elements of epigenetic machinery leads to the development of chromatinopathies. This group of hereditary diseases includes 179 syndromes, some of which present with overlapping phenotypes. Despite the variety of approaches to molecular diagnostics of chromatinopathies, it is not always possible to establish the molecular diagnosis by traditional methods; thus, the issue of optimizing diagnostic algorithms remains relevant. One of the most rapidly expanding areas of post-genomic molecular diagnostics is episignature detection, which relies on genome-wide DNA methylation analysis. This article aims to represent an original approach to indirect diagnostics of chromatinopathies on the example of Rubinstein–Taybi syndrome 1, which is based on the analysis of the methylation level of a limited set of loci designed to reproduce its classic episignature. In the current study, we apply two methods of targeted quantitative analysis of DNA methylation, which are relatively accessible and can be integrated into diagnostic practice. We demonstrate that Rubinstein–Taybi syndrome 1 episignature may be successfully reduced to a single locus of human genome, and that quantitative bisulfite DNA methylation analysis at this locus allows accurate identification of the Rubinstein–Taybi syndrome 1 patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3537a5166624f1652437f23c675d33be54fbf812" target='_blank'>
              DNA Methylation at a Single Locus of Human Genome Accurately Recapitulates Episignature of CREBBP-Related Rubinstein–Taybi Syndrome
              </a>
            </td>
          <td>
            Olga A. Zemlianaia, A. Kalinkin, A. Tanas, Anna V. Efremova, I. Volodin, O. Ismagilova, Anton S. Smirnov, D. Zaletaev, Marina V Nemtsova, S. Kutsev, V. Strelnikov
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Epstein-Barr virus (EBV)+ and EBV- primary CNS lymphomas (PCNSL) carry distinct mutational landscapes, but their transcriptional and epigenetic profiles have not been integrated and compared. This precludes further insights into pathobiology and molecular differences, relevant for classification and targeted therapy.



 23 EBV- and 15 EBV+ PCNSL, histologically classified as diffuse large B-cell lymphomas, were subjected to RNA-Sequencing and EPIC methylation arrays. Unsupervised clustering analyses were performed. Differentially expressed and differentially methylated genes were identified and integrated. Patient-derived cell lines were used to validate findings and identify novel targeted therapies.



 Two distinct transcriptional clusters were found, which separated EBV- and EBV+ PCNSL (p < 0.0001). The EBV+transcriptional signature contained genes (GPR15, FCER2/CD23, SLAMF1/CD150) closely regulated by EBV oncogenes in B-cells. Pathway enrichment analysis uncovered enhanced B-cell receptor (BCR) and WNT/beta-catenin signaling in EBV- lymphomas, whereas Interleukin-10, NOTCH, and viral life cycle pathways were upregulated in EBV+ PCNSL. Correspondingly, BCR-associated SYK kinase activity was enriched in EBV- tumors while JAK2 was overrepresented in EBV+ PCNSL. Epigenetic profiling revealed reduced global promoter methylation in EBV+ PCNSL. Two methylation clusters were recognized, which separated EBV- and EBV+ PCNSL (p < 0.0001). Epigenetic profiles were distinct from 2,788 other brain tumor and non-malignant reference samples. Promoter region hypermethylation of CD79B, a BCR subunit critical for sustained proliferation in EBV- disease, highly correlated (R = -0.7) with its transcriptional downregulation in EBV+ PCNSL. EBV- patient-derived PCNSL cell lines were particularly sensitive to BTK inhibition whereas EBV+ cell lines could be specifically targeted using JAK/STAT inhibitors.



 EBV+ and EBV- PCNSL harbor distinct transcriptional and epigenetic profiles, corroborating them as distinctive biological subtypes. Uncovered differences provide novel insights into their pathobiology, may guide molecular diagnostics and targeted therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3aa2b999ec4779ab9cae5ba7f248fce9520caa2" target='_blank'>
              P14.10.A DISTINCT EPIGENETIC AND TRANSCRIPTIONAL PROFILES OF EPSTEIN-BARR VIRUS (EBV) POSITIVE AND NEGATIVE PRIMARY CNS LYMPHOMAS
              </a>
            </td>
          <td>
            L. Hai, D. Friedel, F. Hinz, D. Hoffmann, S. Doubrovinskaia, H. Rohdjess, K. Weidenauer, E. Denisova, G. Scheffler, T. Kessler, A. Kourtesakis, C. Herold-Mende, O. Henegariu, J. Baehring, J. Dietrich, B. Brors, W. Wick, F. Sahm, L. Kaulen
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Chromosomal inversions shape evolution and are implicated in human disease, yet their effects on genomic variation and health outcomes remain poorly understood. We analyze genome-wide human inversion polymorphisms, contrasting single-event and recurrent loci. Inversion recurrence is validated using structured-coalescent simulations. We show that single-event inversions evolve in near-complete isolation: inverted haplotypes show ∼16-fold lower diversity and strong differentiation from direct haplotypes (median FST = 0.33). By contrast, recurrent inversions maintain gene flow, resulting in similar diversity across orientations and ∼4-fold lower differentiation. We further find marked differences in coding sequence conservation between single-event and recurrent inversions. Using the NIH All of Us biobank, we impute inversions and identify four inversions with significant disease associations. Notably, the 17q21 inversion is associated with reduced risk of cognitive decline (OR=0.919) and breast cancer (OR=0.910) but with increased obesity risk (OR=1.097), consistent with pleiotropic selection. These findings establish inversions as major drivers of human genetic diversity and disease, with evolutionary outcomes critically dependent on recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/219c7d990ca63e41240349f098c120c6ba543f70" target='_blank'>
              Genome-wide diversity of chromosomal inversions and their disease relationships
              </a>
            </td>
          <td>
            Scott C. Sauers, Bailey Huepenbecker, PingHsun Hsieh
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Poly(ADP-ribose) glycohydrolase (PARG), the primary de-PARylating enzyme, plays a critical role in resolving DNA damage. PARG inhibitors (PARGi) are currently in clinical development for patients with tumors exhibiting homologous recombination deficiency (HRD). PARGi induces toxic single-stranded DNA gaps (ssGAPs) in ovarian cancer models by exacerbating S-phase PARylation and causing Okazaki fragment processing (OFP) defects (https://www.pnas.org/doi/10.1073/pnas.2413954121). Similarly, inhibition of USP1, a deubiquitinating enzyme, promotes the accumulation of ssGAPs during replication in BRCA1-deficient cells. Notably, cellular sensitivity to either PARGi or USP1 inhibitor (USP1i) correlates strongly with ssGAP formation, identifying ssGAPs as a predictive biomarker of drug response. Recently CRISPR screens with USP1 inhibitors have identified OFP genes (e.g., PARG, PARP1, FEN1) as USP1i sensitizers. Given the clinical challenges associated with USP1i monotherapy, there is increasing interest in rational combination strategies that maintain efficacy while potentially improving tolerability. Here we show that co-targeting PARG and USP1 induces a novel synergistic interaction in ovarian cancer models by amplifying replication stress and DNA damage. USP1i augmented the PARGi-induced PARylation at DNA replication forks, leading to a greater increase in ssGAPs gaps, compared to treatment with either single agent alone. This dual inhibition led to accumulation of unresolved replication intermediates, culminating in mitotic catastrophe, as demonstrated by increased micronuclei formation and chromosomal aberrations. Importantly, the PARGi and USP1i combination also enhanced tumor cell killing in a subset of PARP-inhibitor-resistant BRCA1-deficient tumors. Mechanistically we identified RAD18-mediated PCNA ubiquitination as a key driver of synergistic lethality. Loss of RAD18 rescued fork-associated PARylation, reduced replication stress, and suppressed ssGAPs thereby mitigating the combined effects of USP1 and PARG inhibition and causing drug resistance. Consistently, USP1 knockout cells exhibited increased sensitivity to PARGi , which was reversed by RAD18 depletion, highlighting RAD18’s essential role in translesion synthesis-mediated ssGAP filling and combination-induced lethality. Notably, USP1i and PARGi also synergized with other ssGAP-inducing agents (e.g., FEN1 inhibitors), suggesting broader combinatorial potential. Finally, we observed synergistic activity in specific HRD patient derived ovarian cancer organoids (PDOs). Our data provides novel mechanistic insights into the therapeutic potential of combining these inhibitors, offering preclinical evidence to support its advancement for treating tumors with and without HRD and addressing PARPi resistance. Ovarian cancer PDOs enable rapid sensitivity testing, with ssGAP accumulation serving as a potential predictive biomarker. This combination strategy holds promise for enhancing treatment efficacy while reducing toxicity, paving the way for more targeted and effective therapeutic options.



 Ramya Ravindranathan, Nicholas William. Ashton, Alexandre Andre. Da Costa, Ozge Somuncu, Ben Lamarre, Sirisha Mukkavalli, Kalindi Parmar, Geoffrey Shapiro, Alan D'Andrea. Synergistic ovarian cancer killing by combination of PARG and USP1 inhibitors: Role of single stranded DNA gaps as a predictive biomarker [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c44d105eb035ae20f8668c449943eff1251b5f67" target='_blank'>
              Abstract A010: Synergistic ovarian cancer killing by combination of PARG and USP1 inhibitors: Role of single stranded DNA gaps as a predictive biomarker
              </a>
            </td>
          <td>
            Ramya Ravindranathan, N. Ashton, A. A. D. Da Costa, Ozge Somuncu, Benjamin P Lamarre, S. Mukkavalli, K. Parmar, Geoffrey I. Shapiro, Alan D. D'Andrea
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 BCOR and its homologue BCORL1 are core components of the non-canonical polycomb repressor complex (PRC) 1.1, playing a central role in epigenetic gene silencing and transcriptional repression. Genomic alterations involving BCOR/L1, primarily BCOR internal tandem duplications (ITD) or BCOR/L1 fusions, occur in central nervous system (CNS) tumors with distinct molecular characteristics and poor clinical outcome. Thus far, the impact of these BCOR/L1-alterations on PRC1.1 functionality in CNS tumors remains elusive. Therefore, a better understanding of the underlying biology is essential to develop improved therapies



 By analyzing bulk, single-cell RNA and chromatin immunoprecipitation (ChIP) sequencing data in tissue samples, we explored whether BCOR-ITD or BCOR/L1 fusions share molecular mechanisms and target gene activation patterns in brain tumors. The functionality and targetable vulnerabilities of identified BCOR/L1 targets were confirmed in-vitro and in-vivo using patient-derived and transgenic cell as well as xenograft models.



 BCOR-altered CNS tumors harbor a unique transcriptomic gene signature, indicative of impaired PRC1.1 repressive functions. This dysregulation leads to activation of distinct BCOR-mediated (FGF/FGFR, PDGFR) and PRC1.1 target genes (GATA6). ChIP-Sequencing analyses confirmed recruitment of activating histone marks (H3K27ac), despite binding of KDM2B and mutant BCOR/L1 to the respective target loci. In parallel, PRC1/2-mediated repressive histone modifications, H3K27me3 and H2AK119ub, were absent. Single-cell RNA sequencing further uncovered a distinct tumor cell population enriched for GATA6 and FGF/FGFR expression. Finally, the dependency on receptor tyrosine kinase signaling activity, identified in BCOR/L1-altered tumors, was confirmed by drug screening, revealing FGFR/PDGFR inhibitors as highly active compounds. The multikinase inhibitor nintedanib not only showed anti-cancer activity in a BCORL1-fused PDX model, but also clinical activity in two patients affected by progressive BCOR/L1-driven CNS tumors.



 We deciphered key oncogenic pathways driven by BCOR/BCORL1 alterations in CNS tumors and identified potential targeted therapeutics, which may be leveraged for improved therapy of patients suffering from these aggressive malignancies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75b0af93bf5bc077fbcbc7bfdc9baedb941faee0" target='_blank'>
              P10.23.B UNCOVERING THERAPEUTIC VULNERABILITIES IN PEDIATRIC BCOR/L1-DRIVEN CNS TUMORS THROUGH MULTI-OMICS PROFILING
              </a>
            </td>
          <td>
            D. Lötsch-Gojo, L. Gabler-Pamer, K. Bruckner, D. Kirchhofer, J. LaBelle, A. Beck, S. Wang, D. Al Tukmachi, J. Quack, S. Madlener, D. Senfter, A. Federico, C. Dorfer, T. Eder, F. Grebien, A. Tauziède-Espariat, J. Pahnke, J. Bunt, V. Hovestadt, J. Schueler, C. Haberler, M. Filbin, M. Kool, W. Berger, J. Gojo
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Rolling circle amplification (RCA) has emerged as a highly versatile and robust isothermal amplification technology, offering exceptional sensitivity, specificity, and scalability for next-generation molecular diagnostics and multi-omics research. Its ability to generate long, repetitive DNA sequences with high fidelity has made it a pivotal tool in disease diagnostics, genomic analysis, and spatial transcriptome profiling. Recent advancements have expanded RCA into various formats, including solution-phase, solid-phase, hydrogel-based, and digital RCA, enhancing its analytical performance and adaptability across diverse biological applications. The integration of RCA with CRISPR-based detection, biosensors, and nanoparticle-assisted signal amplification has significantly improved its diagnostic potential, enabling ultrasensitive detection of nucleic acids, proteins, extracellular vesicles, and single cells. In addition to diagnostics, RCA has profoundly impacted genomic research, particularly in extrachromosomal circular DNA sequencing, offering insights into genome instability and cancer evolution. In spatial transcriptomics, RCA-based in situ RNA sequencing and hybridization have enabled high-resolution gene expression mapping in tissue architecture, providing invaluable insights into cellular heterogeneity and disease microenvironments. This review comprehensively explores the fundamental principles, design modifications, and advanced signal amplification strategies of RCA, with a focus on its applications in molecular diagnostics, genome analysis, and spatial transcriptome profiling. Additionally, it addresses key challenges, including reaction specificity, automation, and cost-effectiveness, and discusses future directions for integrating RCA into high-throughput molecular diagnostics, biomarker discovery, and single-cell and spatial multi-omics platforms. Through continuous innovation, RCA is poised to play a transformative role in precision medicine, next-generation sequencing, and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/576501d76c4b29ed935e06956faf76592e2b944c" target='_blank'>
              Rolling circle amplification for next-generation molecular diagnostics, genome analysis, and spatial transcriptome profiling.
              </a>
            </td>
          <td>
            Jueun Han, Yejin Song, Jihee Lee, Khulan Nyamzaya, Eunjung Kim
          </td>
          <td>2025-09-08</td>
          <td>Nanoscale</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, characterized by cells that exhibit 'monster traits' such as uncontrolled proliferation, invasiveness, resistance to apoptosis, and immune evasion. Traditional therapies often target single hallmarks of cancer, leading to limited success and eventual resistance. This paper proposes a novel, integrative therapeutic framework inspired by evolutionary adaptations in species with remarkable cancer resistance, such as elephants (Loxodonta Africana). Elephants possess enhanced TP53-mediated cancer surveillance and DNA repair mechanisms, which contribute to their extremely low cancer incidence. We hypothesize that by synthetically recapitulating these 'anti-monster' factors—such as augmented p53 activity, enhanced DNA repair, and controlled inflammation—in human cells, we can reprogram malignant phenotypes towards benignity. Our proposed strategy combines synthetic biology, gene therapy (including CRISPR/Cas systems), and multi-targeted pharmacological approaches to simultaneously address multiple cancer hallmarks. By leveraging advanced *in vitro* organoid and *in vivo* models, this research aims to pioneer a new paradigm in oncology that moves beyond cytotoxic elimination towards cellular reprogramming and durable cancer prevention, potentially yielding a significant reduction in lifetime cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c541ffbed17a7ff84c98c50ab50d257ba1c2dd4" target='_blank'>
              Engineering Cancer Resistance: A Novel Framework Inspired by Evolutionary Adaptations
              </a>
            </td>
          <td>
            Hafiz Muhammadismail Azher Syed
          </td>
          <td>2025-09-29</td>
          <td>Wah Academia Journal of Health and Nutrition</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Telomere homeostasis serves as a key regulatory mechanism linking aging and cancer. While telomere attrition imposes a proliferative barrier by inducing cellular senescence, abnormal telomere elongation circumvents this constraint, thereby granting malignant cells unlimited replicative capacity. This study systematically explores the causal relationship between telomere length and cancer risk, with the goal of elucidating the molecular pathways involved in telomere-driven tumorigenesis. Method Mendelian randomization (MR) was employed to establish a causal link between telomere length and pan-cancer susceptibility. Multiple MR models were employed to ensure the robustness of the results. Telomere-associated genes were identified through SNPense analysis, followed by Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment. Key gene regulatory networks were identified and visualized using the MCODE and cytoHubba algorithms. The expression profile of TERT across diverse cancer types was analyzed using TCGA datasets, and its diagnostic potential was evaluated via receiver operating characteristic (ROC) curve analysis. Correlations between TERT expression and immune cell infiltration were further explored. Results Longer telomere length was significantly associated with an increased risk of 33 cancer types (IVW OR = 1.27–1.41, all P < 0.001). A total of 143 telomere-related genes were identified, with functional enrichment highlighting their involvement in genome integrity and telomere maintenance. TERT emerged as the most influential hub gene (MCC score = 169). Transcriptomic analyses from TCGA demonstrated widespread TERT overexpression in 16 cancers (e.g., CHOL, LIHC, LUAD), a finding corroborated by RT-qPCR validation. TERT exhibited high diagnostic performance (AUC > 0.85 in 16 cancers, peaking at AUC = 0.97 in LUSC). Immune infiltration analysis revealed a positive correlation with Th2 cells (r = 0.42) but negative correlations with dendritic cells (r = − 0.38) and macrophages (r = − 0.31). Conclusion This study proposes a comprehensive framework linking telomere length regulation to cancer progression through the TERT axis. Telomere dysfunction contributes to tumorigenesis via two key mechanisms: promoting genomic instability and altering the immune microenvironment. These findings offer new insights into telomere-driven oncogenesis and lay a conceptual foundation for precision diagnostics and targeted therapies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fbec04f31e44d8506f3f60bd214523d80d9aa3" target='_blank'>
              TERT links telomere length to cancer risk by integrating genomic instability and immune modulation
              </a>
            </td>
          <td>
            Jingwei Zhao, Jiayun Zhu, Kangle Zhu, Mingyang Wang, Wei Gong, Jiandong Wang, Shilei Liu
          </td>
          <td>2025-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cardiac myxoma, the most common primary heart tumor, remains poorly understood at the molecular level. Here, we combined single-nucleus RNA sequencing, third-generation transcriptomics, and untargeted metabolomics to dissect its origin and pathology. Single-cell analyses demonstrate an endothelial origin driven by aberrant endothelial-to-mesenchymal transition (EndMT), with pseudotime and RNA-velocity tracing a continuum from endothelial-like to mesenchymal-like and metabolically active states. We identify two distinct myxoma subtypes: Subtype 1, marked by MAPK/WNT/EGFR pathway activation, and Subtype 2, characterized by ribosomal and oxidative phosphorylation signatures alongside immune-evasive programs. Third-generation data highlight extracellular matrix remodeling and endothelial signaling, while metabolomics reveal dysregulated purine, nicotinic acid, and nicotinamide metabolism. Notably, MET–PTK2 signaling emerges as a potential driver of tumor initiation and progression. These integrated findings define the cellular architecture and metabolic adaptations of cardiac myxoma and lay the foundation for future interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0702495e091268eccb0b45edf83576207a5dbb0c" target='_blank'>
              Multi-omics data reveal the origin of cardiac myxoma
              </a>
            </td>
          <td>
            Shengzhong Liu, Wanfeng Zhang, Huajun Sun, Chenqing Zheng, Keli Huang, Chengming Fan, Rensheng Lai, Mingzhu Yin, Jie Lan, Xiushan Wu, Longke Ran, Xiaoping Li
          </td>
          <td>2025-09-26</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Paclitaxel is a widely used chemotherapeutic agent that plays a vital role in the treatment of various cancers, including breast, ovarian, and lung cancers. Despite its success, the development of resistance significantly limits its long-term efficacy. Paclitaxel resistance involves a range of molecular mechanisms, including alterations in drug transport, mutations in β-tubulin, and activation of pro-survival signaling pathways. Recent studies have highlighted the crucial role of circular RNAs (circRNAs) in mediating paclitaxel resistance. CircRNAs are characterized by their stable, covalently closed structures, which protect them from exonuclease degradation. They regulate drug resistance through various mechanisms, including modulating transcription factors and influencing cellular processes such as apoptosis, immune response, and cancer stem cell dynamics by sponging microRNAs. This review focuses on the mechanisms by which circRNAs contribute to paclitaxel resistance and discusses their dual roles as both oncogenes and tumor suppressors. Furthermore, we explore the potential of circRNAs as novel therapeutic targets to overcome paclitaxel resistance. By targeting specific circRNAs or restoring tumor-suppressive circRNAs, it may be possible to enhance paclitaxel sensitivity and improve treatment outcomes. Future research is essential to further understand the role of circRNAs in paclitaxel resistance and to develop effective circRNA-targeted therapies for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b2a64750b1e25583cb541b2ce40bf7626415eaa" target='_blank'>
              The dual role of circular RNAs in paclitaxel resistance: insights into molecular mechanisms and clinical implications.
              </a>
            </td>
          <td>
            Ehsan Maghrebi-Ghojogh, O. Q. B. Allela, Karar H. Alfarttoosi, Sada Ghalib Taher, Mariem Alwan, M. Jawad, Hiba Mushtaq, Mitra Zareanshahraki, Hossein Gandomkar
          </td>
          <td>2025-10-17</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="LINE-1 retrotransposition is common in human cancers and rearrangements at insertion sites can contribute to cancer-driving oncogene amplifications and promote genome instability. However, the mechanisms underlying rearrangements of L1 retrotransposition intermediates are poorly understood. To address this gap, we developed GFP-based recombination reporter assays to study the formation of L1 retrotransposition-mediated rearrangements. Using these reporters combined with long-read sequencing approaches, we find that L1 retrotransposition intermediates can recombine with distal DNA breaks to generate chromosomal rearrangements. We also find that two distinct L1 insertion intermediates can recombine with each other to generate chromosomal rearrangements. Both types of rearrangements depend on L1-encoded ORF2p endonuclease and reverse transcriptase activities. Using these reporters, we discover that L1 retrotransposition-mediated rearrangements are robustly induced when the recombining sequences share homology and that their formation requires the homologous recombination factor BRCA1. Given the repetitive nature of our genome, these findings highlight the risk of L1 insertion intermediates becoming substrates for aberrant recombination and promoting genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb644439b03063ebcc926aea5075b191a155a103" target='_blank'>
              L1 insertion intermediates recombine with one another or with DNA breaks to form genome rearrangements
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Jupiter C. Kalinowski, Cheuk-Ting Law, Phillip Schofield, Aidan Burn, Kathleen H. Burns
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56a3d728c459f75967661fea1d9baa92b4ef2796" target='_blank'>
              Age distinguishes selection from causation in cancer genomes
              </a>
            </td>
          <td>
            David Cheek, Martin Blohmer, Martin A. Nowak, Tibor Antal, Kamila Naxerova
          </td>
          <td>2025-10-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The estimation of the in vitro potency of anticancer agents against tumor cell lines in tissue culture is known to be fraught with a high degree of variability. While quality control of experimental conditions has been demonstrated by others to reduce this variability, other sources of variability remain even after standardization of experimental conditions. In this poster, we assess two potential mechanisms for this residual variability: phenotypic stochasticity in response and heritable resistance to treatment.



 We developed three agent-based mathematical models representing different potential scenarios for cellular responses to anticancer therapy: (1) all cells have an equal probability of survival under treatment (phenotypic stochasticity) and future division; (2) sensitive cells deterministically die under treatment while resistant cells survive and may divide to produce resistant offspring (heritable resistance); (3) sensitive subclones have a greater probability of death under treatment compared to resistant subclones (both mechanisms). We implemented these models in MATLAB, running simulated in vitro experiments consisting of large (106) or small (500 or 5000) populations of virtual cells under treatment. The final size of these virtual cell populations serves as the readout of the virtual experiments. To assess the effect of phenotypic stochasticity and heritable resistance on variation in experimental outcomes, we simulated these experiments 10,000 times and report the coefficient of variation (CV).



 Variability in response to treatment was low under most simulated conditions. In the first model simulating phenotypic stochasticity alone, high coefficients of variation in assay outcome were observed only when the probability of survival under treatment was less than 10%. In simulations of deterministic heritable survival under treatment, the coefficient of variation was generally low. However, simulations incorporating both mechanisms exhibited substantial variability in experimental outcome, consistent with the observed phenomenon.



 Stochastic cellular response to treatment or heritable binary resistance alone are insufficient explanations for the variation in in vitro experimental results. On the other hand, the high degree of variation in experimental outcomes is consistent with a combination of heritable resistance and phenotypic stochasticity. Our findings are consistent with a role for Darwinian evolution in the response to anticancer treatment for cancer cells in tissue culture.



 Alexander L. Young, Liam Noronha, Madison Stoddard, T Ryan. Gregory, Arijit Chakravarty. Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of in vitro potency of anticancer agents [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C073.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a885f4ecedc46db5c7868acd715fccb8dde4f83" target='_blank'>
              Abstract C073: Heritable resistance and phenotypic stochasticity are likely drivers of irreproducibility in experimental determination of
 in vitro
 potency of anticancer agents
              </a>
            </td>
          <td>
            Alexander L. Young, Liam Noronha, M. Stoddard, T. R. Gregory, Arijit Chakravarty
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S. Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="BACKGROUND AND PURPOSE
In order to significantly improve the therapeutic treatment of breast cancer, the exploration of underlying genetic and molecular differences is absolutely necessary. Here, our goal was to integrate mutational status of entire pathways to reveal molecular pathway interactions determining survival.


EXPERIMENTAL APPROACH
A comprehensive analysis of breast cancer mutations was conducted by integrating data from three distinct databases with a total of 4586 samples encompassing over 25,000 genes. For each gene, we filtered mutations that disruptively affect the protein structure. Cox proportional hazard regression was employed to link altered pathways to outcome. We also identified the co-occurring and mutually exclusive disruptive mutations.


KEY RESULTS
We identified 17 genes, the mutation status of which alone seriously affects relapse-free survival. The three most significant genes were TP53 (HR: 2.04, p: 4.65 × 10-33), CARD11 (HR: 2.59, p: 1.54 × 10-5) and PIK3R1 (HR: 2.27, p: 3.66 × 10-5). The five most significant biological processes and KEGG pathways affecting relapse-free survival include negative regulation of cell population proliferation, positive regulation of DNA-templated transcription, protein stabilisation, and MicroRNAs in cancer, hepatocellular carcinoma, and breast cancer. Co-mutation and mutual exclusivity analysis identified significant enrichment in 241 gene pairs. Finally, we also established an online platform to enable future analysis of the established cohort for any selected pathway.


CONCLUSIONS AND IMPLICATIONS
We assembled a comprehensive database of breast cancer samples and used this cohort to identify cancer-specific disruptive mutation signatures linked to altered survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/919db15729dadcc0b973aea2e0ec780f64f60ace" target='_blank'>
              Pathway-level mutational signatures predict breast cancer outcomes and reveal therapeutic targets.
              </a>
            </td>
          <td>
            Máté Posta, Balázs Győrffy
          </td>
          <td>2025-10-07</td>
          <td>British journal of pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we find that EZH2 regulates a multilineage cell state dependent on the activation of translation and regulation of RNA maturation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration. Moreover, we elucidated that the regulation of cellular state transition by EZH2 inhibition requires activation of the RNA degrader, Tristetraprolin (TTP). Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype.



 Beatriz German, Leigh Ellis. EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e13a2ff65980383cdcce076c99675421b3f51723" target='_blank'>
              Abstract C033: EZH2 regulates cell identity in neuroendocrine-like prostate cancer by driving alternate transcription via activation of translation and RNA processing regulation
              </a>
            </td>
          <td>
            Beatriz German, Leigh Ellis
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transposable elements (TEs), once regarded as genomic “junk,” are now recognized as powerful regulators of gene expression, genome stability, and innate immunity. In the context of neurodegeneration, particularly Amyotrophic Lateral Sclerosis (ALS), accumulating evidence implicates TEs as active contributors to disease pathogenesis. ALS is a fatal motor neuron disease with both sporadic and familial forms, linked to genetic, epigenetic, and environmental factors. While coding mutations explain a subset of cases, advances in long-read sequencing and epigenomic profiling have unveiled the profound influence of non-coding regions—especially retrotransposons such as LINE-1, Alu, and SVA—on ALS onset and progression. TEs may act through multiple mechanisms: generating somatic mutations, disrupting chromatin architecture, modulating transcriptional networks, and triggering sterile inflammation via innate immune pathways like cGAS-STING. Their activity is normally repressed by epigenetic regulators, including DNA methylation, histone modifications, and RNA interference pathways; however, these controls are compromised in ALS. Taken together, these insights underscore the translational potential of targeting transposable elements in ALS, both as a source of novel biomarkers for patient stratification and disease monitoring, and as therapeutic targets whose modulation may slow neurodegeneration and inflammation. This review synthesizes the current knowledge of TE biology in ALS; integrates findings across molecular, cellular, and systems levels; and explores the therapeutic potential of targeting TEs as modulators of neurodegeneration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69ff26f0b7276e91620b33b746e298b203260717" target='_blank'>
              The Other Side of the Same Coin: Beyond the Coding Region in Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            Paola Ruffo, Benedetta Perrone, Francesco Perrone, F. De Amicis, Rodolfo Iuliano, Cecilia Bucci, Angela Messina, F. Conforti
          </td>
          <td>2025-10-18</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 SETD2 is a tumor suppressor gene responsible for trimethylation of H3K36, crucial for DNA mismatch repair and homologous recombination. Novel synthetic lethality agents, ATR inhibitors (ATRi), have demonstrated anti-tumor activity in preclinical models and early clinical trials for pan-tumors. SETD2 and H3K36 deficient cells have been demonstrated to display Micro Satellite Instability (MSI) with increased tumor mutational burden (TMB). Deleterious SETD2 alterations occur in 5-10% of brain tumor patients with glioblastoma, IDH wild type, WHO grade 4, (GBM). These patients have a poor prognosis. This study aims to evaluate targeted treatment strategies in central nervous system (CNS) tumors, by investigating the potential of ATRi for tumors harboring deleterious SETD2 alterations and immuno-therapy for tumors with increased mutational burden, with the objective of improving patient outcomes.



 This study analyzes genomic profiles from 434 patients diagnosed with central nervous system (CNS) tumors between January 1st, 2019, and February 28th, 2025, enrolled in the prospective Neurogenome and CoPPO studies. Rigshospitalet has one of the largest single-institution datasets with comprehensive data on SETD2 alterations derived from whole exome/genome sequencing including structural variants, copy number variations, and RNA expression.



 The study cohort included 434 patients with CNS tumors (166 female, 268 male) with a median age at diagnosis of 59 years (interquartile range: 49-66 years). GBM was the predominant tumor type (86%), followed by astrocytoma, IDH mutated (mut), WHO grade 2-4 (12%) and oligodendroglioma, IDH-mut, 1p/19q co-deletion, WHO-grade 2-3 (1.2%). Other CNS tumors were diagnosed in 6 (1.4%) patients. Deleterious SETD2 alterations were found in 13 (3%) patients: 11 with GBM, 1 astrocytoma IDH mut, and 1 oligodendroglioma. Twelve patients had mutations: 6 premature stop codons, 5 frameshift mutations and 1 splice site variant, and one case with biallelic SETD2 loss. Data on tumor mutational burden (TMB) was assessable for 12 SETD2-deficient patients, none exceeding 10mut/mb; mean TMB was 3.3 mut/mb vs. 6.2 mut/mb in the SETD2-wildtype (wt) group. No SETD2-deficient patients had received temozolomide (TMZ) treatment prior to sampling. By contrast, some samples in the SETD2-wt cohort were obtained at relapse, which may have inflated TMB due to TMZ treatment. COSMIC signatures were available for 10 SETD2-deficient patients with none having indications of MSI based on COSMIC signature evaluation.



 Loss of SETD2 function in primary CNS-tumors is relatively rare and was predominantly driven by mutations, with copy number analysis revealing only 1 biallelic deletion in 434 patients. SETD2 alterations was primarily seen in GBM patients. The SETD2 deficient patients did not exhibit indications of MSI or high TMB. However, the impact of patient relapse samples with TMZ induced mutations in the SETD2-wt cohort is being explored further together with pan-tumor markers for DNA damage repair.



 Tine Østergaard, Christoffer L. Halvorsen, Dorte S. Nørøxe, Vincent N. Fougner, Iben Spanggaard, Kristoffer S. Rohrberg, Martin Højgaard. Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in SETD2 without associated increase in mutational burden [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5615e3697959292b283838db14154a56c4e82cf6" target='_blank'>
              Abstract C037: Comprehensive genomic profiling in central nervous system tumors reveals low frequency of deleterious alterations in
 SETD2
 without associated increase in mutational burden
              </a>
            </td>
          <td>
            Tine Østergaard, Christoffer L. Halvorsen, D. Nørøxe, V. Fougner, I. Spanggaard, Kristoffer S. Rohrberg, M. Højgaard
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Cancer remains a leading cause of death for children and adolescents, underscoring the need for therapeutic advances. The PRISM assay enables rapid profiling of emerging anti-cancer agents by barcoding and pooling >900 genetically and functionally characterized cancer cell lines. The PRISM assay reveals not only drug potency and selectivity, but also genomic correlates of sensitivity and predictive biomarkers. To better represent pediatric, hematopoietic, and rare cancers in our cell collection, we expanded the PRISM panel in collaboration with the Pediatric Cancer Dependencies Accelerator project by adding 109 new cell lines, including 21 pediatric models, 60 hematopoietic lines, and 14 previously unrepresented subtypes. In many pediatric cancers, epigenetic proteins are frequently altered and are promising therapeutic targets; therefore, we screened clinically relevant compounds targeting epigenetic proteins at 8 concentrations for 5 days. The SMARCA2/4 inhibitor, BRM/BRG1 ATP Inhibitor-1, and the BRD9 degrader, CFT8634, showed strong activity in atypical teratoid rhabdoid tumor (ATRT) models. In contrast, the SMARCA2-selective degrader, ACBI2, was less active in ATRT, despite similar potency. Further work will investigate the mechanistic basis of this differential activity. As some small molecules developed against epigenetic targets may require a longer incubation period to enact their cytotoxic effects, we developed a 10-day PRISM assay, twice as long as our standard 5-day assay. We tested the EZH2 inhibitor, mevrometostat, in both the 5-day and 10-day assay and observed greater potency, selectivity, and stronger correlations with EZH2 dependency in the 10-day assay. Finally, given the prevalence and utility of combination treatments for pediatric cancers, we also wanted to probe the drug combination landscape. We screened a collection of combinations in a 5x5 dose matrix format, focusing on inhibitors targeting DNA repair proteins (ATR, ATM, WEE1, CHEK1, PolTheta, USP1, DNA-PK, Topo1) in combination with talazoparib or gemcitabine. We observed synergy in pediatric subtypes for many but not all of the combinations tested. We identified selective synergies in pediatric subtypes, including Ewing sarcoma with talazoparib-based combinations and B-ALL with low-dose talazoparib + ATM inhibitor AZD1390. Further analysis of this combination dataset will refine our understanding of specific contexts in which we observe synergy and reveal biomarkers of synergistic effects. Together, these advances position PRISM to accelerate pediatric cancer research by expanding disease-relevant models and enabling long-duration single-agent screening and matrix combination screening at scale. All data will eventually be made publicly available on depmap.org, providing a resource for the pediatric cancer research community to explore novel therapeutic opportunities.



 Colleen T. Harrington, Antonella Masciotti, Ursula Widocki, Ashish Bino George, Blanche Ip, Tenzin Sangpo, Li Wang, Laura Doherty, Aydin Golabi, Rachael Barry, Emily Reeves, John Davis, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. Defining the epigenetic drug landscape and rational combinations for pediatric cancer using the PRISM screening assay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A026.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b814cec2f64b38043334e1b8daf5cb66a0a058be" target='_blank'>
              Abstract A026: Defining the epigenetic drug landscape and rational combinations for pediatric cancer using the PRISM screening assay
              </a>
            </td>
          <td>
            Colleen T. Harrington, Antonella Masciotti, Ursula Widocki, Ashish Bino George, Blanche C. Ip, Tenzing Sangpo, Li Wang, Laura Doherty, Aydin Golabi, Rachael Barry, Emily Reeves, John F. Davis, Melissa Ronan, M. Rees, Jennifer Roth
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Traditional pooled genetic dependency screens, such as DepMap, have proven powerful for identifying essential genes in cancer cell lines. However, these pooled formats are prone to the bystander effect, where secreted factors from wild-type cells support the survival of knockout (KO) cells within the same culture, potentially masking dependencies on specific secreted proteins. To address this limitation, we performed a well-by-well CRISPR screen targeting genes encoding secreted factors in the pancreatic ductal adenocarcinoma (PDAC) cell line HPAFII. This arrayed format eliminates cross-feeding and enables precise assessment of cell-intrinsic dependencies on targeted proteins and could thus expose hidden vulnerabilities. CellTiter-Glo (CTG) viability assays were used to assess gene lethality following target gene KOs. Furthermore, RNA sequencing verified effective disruption of target genes, establishing the specificity and reliability of the screening workflow. We observed strong growth inhibition in EFNA1 (Ephrin-A1) KO samples compared to lipofectamine-treated controls. However, this inhibitory effect was missed and underrepresented in the conventional pooled format. EFNA1 is a ligand for EPH receptors, involved in cell-cell communication, migration, and proliferation signaling. Differential gene expression analysis comparing EFNA1 KO to lipofectamine-treated controls revealed enriched pathways of oxidative phosphorylation, E2F targets, MYC targets, and DNA repair in EFNA1 KO samples. This result supports our observation of lethality in EFNA1 KO samples. It suggests that EFNA1 contributes to cell-cell communication and is critical for HPAFII cell survival. Its KO triggers stress pathways, including E2F, MYC, OXPHOS, and DNA repair, and acts as a suppressor of excessive proliferative programs. In conclusion, we identified EFNA1 as a critical non-cell-autonomous essential gene through an arrayed CRISPR screen, with its loss showing lethality to HPAFII cells exclusively in isolated settings, likely as a result of disrupted cell-cell interactions. Defining such dependencies may facilitate the discovery of directly druggable and more tumor-selective therapeutic strategies.



 Xingyi Shi, David Ruddy, Millicent Gabriel, Michelle Piquet, Kris Muskiewicz, Joel Wagner, Joshua Korn, Cory Johannessen. Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr C035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddd6ce939e1194e9cb9b823d2bb2e7b39fb1e35f" target='_blank'>
              Abstract C035: Arrayed CRISPR Screening Reveals EFNA1 as a Non-Cell-Autonomous Essential Gene in Pancreatic Cancer Cells
              </a>
            </td>
          <td>
            Xingyi Shi, David A Ruddy, Millicent Gabriel, M. Piquet, Kris Muskiewicz, Joel P Wagner, Joshua Korn, Cory Johannessen
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In humans, germline mutation rates are three- to four-fold higher in males than females for largely unknown reasons. We investigated whether transcription, a well-documented source of both DNA damage and repair in somatic tissues, is associated with sex differences in germline mutations. We used single-cell germline expression data from different stages of gametogenesis in males and females and phased de novo mutations from pedigrees. Focusing on protein-coding genes, we found no relationship between the male mutation rate and averaged gene expression levels during spermatogenesis despite evidence for transcription-asymmetry. Individual stages of spermatogenesis appear to vary in their contribution to mutation, however: expression levels in spermatogonial stem cells and sperm are significantly positively associated with paternal mutation rates, while those in primary spermatocytes are significantly negatively associated. Thus, transcription may have varying effects over male gametogenesis that are not visible from its cumulative effect on the total germline mutation rate. By contrast, the female mutation rate increases significantly with transcription levels averaged across different stages of folliculogenesis, with corresponding evidence for transcription-asymmetry. The difference between the sexes is also observed when using a smaller set of phased mutations from three-generation pedigrees and when relying on expression data from bulk gonadal tissues. These results indicate that transcription has distinct effects on the mutation rate in the two sexes, leading to more mutations in females but not males, in contrast to what one might expect from the overall paternal bias in germline mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f949d025880c67f108bb5128d039182864ec241e" target='_blank'>
              Sex differences in transcription-associated mutagenesis in the human germline
              </a>
            </td>
          <td>
            Minyoung J. Wyman, Ipsita Agarwal, Marc de Manuel, Natanael Spisak, Molly Przeworski
          </td>
          <td>2025-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Targeted therapy has been established as a therapeutic option for the treatment of metastatic melanoma. Despite initially being very efficient, many tumors develop resistance to targeted therapy, leading to its failure. We previously demonstrated that the neural crest (NC)-associated gene ERRFI1 is highly expressed in metastatic melanoma and correlates with a bad prognosis. Here, we show that the expression of ERRFI1 was upregulated in melanoma and negatively correlated with the expression of melanocytic differentiation markers, such as MITF and TYR. Downregulation of ERRFI1 with the help of siRNA increased the susceptibility of melanoma cells toward BRAF inhibition (BRAFi) and resensitized BRAFi-resistant melanoma cells to BRAFi. Mass spectrometry-based proteomic analysis revealed that ERRFI1 silencing diminished the activation of the mitogen-activated protein kinase (MAPK) and AKT signaling pathways, which usually contribute to drug resistance. Furthermore, we show that miR-200c targeted the 3'UTR of ERRFI1 and reduced its expression, resulting in the resensitization of BRAFi-resistant melanoma cells to BRAFi. Our study results suggest that ERRFI1 could be a potential therapeutic target for the treatment of metastatic melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a645774843921b1750028307066a9681c4de577" target='_blank'>
              The neural crest-associated gene ERRFI1 is involved in melanoma progression and resistance toward targeted therapy.
              </a>
            </td>
          <td>
            Nina Wang, Qian Sun, Daniel Novák, Lei Zhu, Juliane Poelchen, Tamara Steinfass, Yiman Wang, V. Umansky, Jochen Utikal
          </td>
          <td>2025-10-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Introduction Ovarian Cancer remains a significant global health concern, with high mortality rates, largely due to late-stage diagnosis and limited treatment options. These extrinsic factors are driven or exacerbated by intrinsic mechanisms such as persistent activation or upregulation of Signal Transducer and Activator of Transcription 3 (STAT3). STAT3 promotes tumor growth, inhibits apoptosis, accelerates angiogenesis and metastasis, facilitates immune evasion, and contributes to chemoresistance. Consequently, STAT3 activation fosters an aggressive ovarian cancer phenotype, contributing to treatment failure, poor prognosis and low survival rates, highlighting the urgent need for novel, safe, effective and affordable STAT3-targeted therapeutic strategies. In this study, we developed a novel double-stranded DNA minicircle (mcDNA) inhibitor, designed to act as a decoy for STAT3, preventing its binding to target gene promoters. Methods Utilizing the SKOV3 ovarian cancer cell line, we evaluated the effects of our inhibitor in vitro on cell viability through MTS assay, its apoptotic and necrotic effects using flow cytometry and the expression modulation of downstream STAT3-regulated genes, assayed through RT-qPCR and Western blot analysis. Results We demonstrate that anti-STAT3 mcDNA significantly reduces the viability of SKOV3 cells at low nanomolar concentrations, while sparing the control group. The effect observed was dose-dependent. Mechanistically, anti-STAT3 mcDNA induces apoptosis and necrosis in treated cells, also revealing a certain dose-dependency, while also decreasing cell proliferation. Finally, our inhibitor significantly downregulates STAT3-dependent anti-apoptotic genes MCL1 and PIM1. Conclusion These findings suggest that anti-STAT3 mcDNA is a promising, effective and specific candidate for targeted STAT3 inhibition in SKOV3 ovarian cancer cells, warranting further validation in ovarian cancer, in vivo exploration and potential application in other types of malignancies, where STAT3 acts as an oncogenic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298edc0f21db1280384ca9a0eea5b54ca57a8bc1" target='_blank'>
              Breaking the cancer code: a novel DNA minicircle to disable STAT3 in ovarian cancer cells SKOV3
              </a>
            </td>
          <td>
            Adina-Gabriela Vasilescu, Andrei-Mihai Vasilescu, Livia E. Sima, Natalia Baran, S. Szedlacsek
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Chromosome instability (CIN) and Genetic instability are hallmark features of malignant tumor cells, contributing to tumour progression, drug resistance and metastasis. KIF18A, a member of the kinesin superfamily of motor proteins, is essential for preserving the structural integrity of the bipolar spindle apparatus during cell division. Recent studies have demonstrated that genetic depletion of KIF18A impacts the chromosome alignment, elongation of mitotic spindles, and activates spindle assembly checkpoint in chromosomal instable tumour cells ultimately leading to cell death, while having negligible impact towards the survival of normal cells. These findings suggest that targeting KIF18A represents a promising therapeutic approach for selectively inhibiting the growth of chromosomally instable tumours, while sparing the normal division of euploid cells. We have designed novel macrocyclic KIF18A inhibitors based on our structural understanding from Cryo-EM structures of initial acyclic binders. Structure guided SAR led to the identification of highly potent and selective compounds with optimized properties. Lead compounds exhibited significant inhibition of KIF18A ATPase activity and demonstrated robust cellular anti-proliferative potency in chromosomally instable (CIN-high) high-grade serous ovarian carcinoma cells (HGSOC, OVCAR-3) while retaining a good selectivity over the CIN-low MCF7 cells. This in vitro potency was supported by a significant and dose-dependent upregulation of phospho-Histone H3 and gamma-H2AX, consistent with the induction of mitotic arrest and DNA damage. Further, the lead compounds exhibited encouraging drug-like properties, including acceptable in vitro ADMET profile and excellent in vivo exposure with low clearance resulting in good oral bioavailability in rodents. Additionally, these compounds showed significant anti-tumor efficacy that was associated with robust induction of phospho-histone H3 in the OVCAR3 CDX model. The macrocyclic lead compound is being extensively evaluated in advanced in vitro and in vivo assays to support further development.



 Kiran Aithal B, Sandip Sengupta, Raghavendra N R, Shailesh Vishwanath. Jadhav, Aditya Kiran. Gatta, Balasubramanyam D, Suresh D. Babu, Gokulakrishnan R, Srinivasu P, Samiulla D S, Amith A, Aravind A B, Girish C. Daginakatte, Kavitha Nellore, Kishore Narayanan, Chandrasekhar Abbineni, Murali Ramachandra, Susanta Samajdar. Discovery and preclinical characterization of novel macrocyclic KIF18A Inhibitors for treatment of chromosomally instable tumors [abstract]. In: Proceedings of the AACR-NCI-EORTC International Conference on Molecular Targets and Cancer Therapeutics; 2025 Oct 22-26; Boston, MA. Philadelphia (PA): AACR; Mol Cancer Ther 2025;24(10 Suppl):Abstract nr A030.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab5f82e03ba5520dcf239a8a55ec749b43cc168" target='_blank'>
              Abstract A030: Discovery and preclinical characterization of novel macrocyclic KIF18A Inhibitors for treatment of chromosomally instable tumors
              </a>
            </td>
          <td>
            Kiran Aithal B, Sandip Sengupta, Raghavendra N R, Shailesh V. Jadhav, Aditya Kiran. Gatta, Balasubramanyam D, Suresh D. Babu, Gokulakrishnan R, Srinivasu P, S. S, A. A., A. B, G. Daginakatte, K. Nellore, Kishore Narayanan, C. Abbineni, Murali Ramachandra, S. Samajdar
          </td>
          <td>2025-10-22</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 TP53, often referred to as the guardian of the genome, is one of the most frequently mutated genes in human cancers. High grade serous ovarian cancer (HGSOC) has a TP53 mutation rate of up to 96%. Given its multi-faceted role in human cancers, there have been many efforts to therapeutically target mutant p53. APR-246 is a small molecule drug that is proposed to bind mutant p53 and promote its thermodynamic stabilization, allowing the tumor suppressor to regain wild-type-like conformation and functionality. However, the mechanism of this drug has remained a contentious topic, as it has been shown to act via various p53-independent mechanisms, including modulation of the redox system. To address this, we have performed a systematic functional genetic screening for the identification of APR-246 rescuable p53 mutants. Our approach leverages a pooled TP53 mutagenesis library containing ∼8,000 amino acid variants, spanning the entirety of the TP53 coding sequence. Cell based screenings against sub-lethal doses of APR-246 in two ovarian cancer cell lines, SKOV3 and OVCAR5, has revealed that sensitivity to this drug is cell-line dependent rather than mutant p53-dependent. Our results suggest that p53-independent factors such as intrinsic resistance to ferroptosis may play a more significant role in response to APR-246. We propose that resistance to ferroptosis provides cellular conditions which enable identification of p53 mutants that are sensitive to APR-246. This is evidenced by our results which show that genetic screen in OVCAR5, which is more intrinsically resistant to ferroptosis, uncovers a greater number of potentially sensitive mutants, including R175H, a mutant largely recognized as rescuable by APR-246. Further, we have identified GPX4, an anti-ferroptosis gene, as a key determinant of APR-246 sensitivity. Indeed, we demonstrate synergy between APR-246 and GPX4 inhibitor, ML-210. These studies support the hypothesis that APR-246 efficacy is largely attributable to p53-independent mechanisms and that attenuation of these off-target effect may reveal potentially sensitive p53 mutants.



 Anais Saunders, Caili Tong, Anthony N. Karnezis, Gary S. Leiserowitz, Jeremy R. Chien. Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7beb6941bcf2a1d1e3d25e6917a37bc8794a91d" target='_blank'>
              Abstract A046: Functional genetic screen of TP53 mutants reveals APR-246 sensitivity is mediated by p53 independent effects
              </a>
            </td>
          <td>
            Anais Saunders, Caili Tong, A. Karnezis, G. Leiserowitz, Jeremy Chien
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="


 Pancreatic cancer is a lethal disease that has a five-year survival rate of ∼13%. Pancreatic cancer is marked by a number of significant alterations in its dense tumor microenvironment and metabolism. Our analyses of differential gene expression in multiple datasets point to HDAC1 as a consistently upregulated gene in pancreatic cancer and one that is significantly correlated with multiple metabolic pathways. Using CRISPR/Cas-9 gene editing, we generated HDAC1-knockout cell lines. Our current findings show when HDAC1 is deleted, pancreatic cancer cell lines display alternate growth phenotypes, differentially express metabolic targets, and have global metabolomics changes. Further, Seahorse analysis revealed a switch to oxidative phosphorylation in a subset of HDAC1 knockout cells. This distinct response to HDAC1 deletion demonstrates the importance of understanding the nuanced characteristics in order to effectively target metabolism in pancreatic cancer based on different patient subgroups.



 Don-Gerard B. Conde, Evan J. Zhou, Maximilian Farma, Soren Jensen, Laiba Sheikh, Matthew Cheung, Chiamaka J. Ezeh, Harshita N. Khedkar, Zeribe C. Nwosu. Deletion of HDAC1 demonstrates alternative metabolic response in pancreatic cancer [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr B135.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01b243a07ac38b6a04c17b406fa31db8c75f6446" target='_blank'>
              Abstract B135: Deletion of HDAC1 demonstrates alternative metabolic response in pancreatic cancer
              </a>
            </td>
          <td>
            Don-Gerard Conde, Evan J. Zhou, Maximilian Farma, Soren Jensen, Laiba Sheikh, Matthew Cheung, Chiamaka J. Ezeh, H. N. Khedkar, Z. Nwosu
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="


 Acute lymphoblastic leukemia (ALL) is the leading cause of cancer-related mortality in pediatrics. Between 15-30% of patients relapse, and current treatments are associated with acute and long-term toxicity. The immune component of the tumor microenvironment is an under-explored factor that could play a role in the establishment or progression of the disease, as well as modulate response to chemo- and immunotherapies. We characterized the molecular profile of leukemias from 32 patients with pediatric ALL treated under the ALLIC-GATLA 2010 clinical protocol in Argentina by transcriptome sequencing (Illumina). We estimated the immune component through digital cytometry (MIXTURE) and a cytolytic score based on the expression of five genes. We searched for single nucleotide variants (SNV) in the transcriptome (CTAT-Mutations) that could alter the interaction with immune cells in the tumor microenvironment. For this, we analyzed 843 genes from Reactome collections associated with the immune system. Variants were filtered and annotated using publicly available germline and somatic variant databases. The group of patients with higher abundance of cytotoxic cells (Z-score>1) was associated with an increased risk of relapse/death (HR=4.5; p=0.05), and higher levels of CX3CR1 (q-val<0.05) and TIM-3 (q-val=0.06). Analysis of an independent high-risk cohort (TARGET B-ALL phase II, N=88), confirmed that a higher cytolytic score was associated with the risk of relapse in an univariate (HR=2.66, p=0.001) and multivariate model including cytolytic score (p<0.006), protocol (p=0,07), ETV6::RUNX1 (p:>0.05) and TCF3::PBX1 status (p<0.001). Cytolytic score was not associated with molecular subtype, age, sex or early response to treatment (p>0.05). For SNV analysis, we developed a threshold value for the QUAL parameter to select for high-confidence variants, establishing a specificity of 95%. Eleven variants were found in nine patients, being 8/11 classified by Franklin as TIER3 in CEP290, UBE3B, CPNE3, DUSP4, DAPP1, RNF135, TBC1D10C and TLR1; 2/11 as TIER1 in TP53, KRAS; and 1/11 as TIER4 in BTN3A3. Patient harboring variants in these genes were significantly associated with a lower GSVA score in the Reactome dataset “MHC_Class_II_Antigen_Presentation” (q-val<0.01). We found a variant in UBE3B, located in the active site of the E3 ubiquitin ligase, and associated with a low GSVA score in the Reactome “Antigen presentation through MHC class I” pathway. A novel variant in CPNE3 gene was found, located in a highly conserved residue and predicted to be deleterious by multiple tools. Conclusion: the abundance of cytotoxic cells was independent of molecular subtype and associated with a worse prognosis in independent cohorts. Further analysis of the variants found in this study has the potential to identify novel mechanisms modulating the interaction between the immune system and leukemic cells in ALL.



 Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, Pablo Sanchis, Maria Cecilia Riccheri, Geraldine Gueron, Javier Cotignola, Maria Sol Ruiz. Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A040-PR008.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8f83b4eeb02fdcb7483db9e8c77e60a6f1fd7e" target='_blank'>
              Abstract A040-PR008: Characterization of cytotoxic cell abundance and genetic variants in immune related genes in pediatric acute lymphoblastic leukemia patients
              </a>
            </td>
          <td>
            Ignacio Gomez Mercado, Daniel Avendaño, Sabrina Ledesma Bazan, P. Sanchis, M. C. Riccheri, G. Gueron, J. Cotignola, Maria Sol Ruiz
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer cells can acquire a reversible, dormant drug-tolerant persister state mimicking embryonic diapause to evade therapy pressure. Deciphering the precise mechanisms driving cancer cells into or out of a diapause-like persister cell-state could provide strategies to overcome resistance. Here, we showed that following chemotherapy, diverse therapeutic agents converge on WNT pathway activation to induce a de novo diapause-like cell state across various triple-negative breast cancer (TNBC) cell line, xenograft, and patient-derived organoid models. Among early persister cells, only transcriptionally WNT-active persisters exhibited the transcriptional and functional characteristics typical of diapause-like cells, including a negative correlation with MYC transcriptional activity and reversible restricted proliferation. The WNT signaling pathway functioned as both an inducer and biomarker of the diapause-like early persister cell state in both parental (chemo-naïve) and chemotherapy-treated cells. Entry into and exit from the diapause-like early persister cell state was triggered by the transcriptional upregulation of components essential for canonical WNT ligand secretion. A combinatorial treatment strategy inhibiting WNT ligand secretion alongside chemotherapy effectively targeted the early mechanisms underlying the acquisition and enrichment of a diapause-like cell phenotype. These findings reveal WNT pathway activation as an early event that leads to a reversible diapause-like persister state and highlight the potential of targeting this axis to prevent the development of drug-resistant populations before they are firmly established.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9bf626a72b9d5ae7b70a19b1f05585d0584500" target='_blank'>
              Chemotherapy-Treated Breast Cancer Cells Activate the WNT Signaling Pathway to Enter a Diapause-Like Early Persister State.
              </a>
            </td>
          <td>
            Youssef El Laithy, Willy Antoni Abreu de Oliveira, A. Pabba, Alessandra Qualizza, Gwenny Cosemans, Paula Garcia-Diaz, François Richard, Paraskevi Athanasouli, Carla Rios Luci, Wout De Wispelaere, Larissa Mourao, Siân Hamer, Stijn Moens, A. de Jaime-Soguero, M. Baietti, S. Hutten, Jos Jonkers, Stephen-John Sammut, S. Soenen, C. Scheele, Alejandra Bruna, C. Desmedt, D. Annibali, F. Lluis
          </td>
          <td>2025-10-21</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="BACKGROUND
Head and neck squamous cell carcinoma (HNSCC) is a malignancy with poor prognosis and survival. While epidermal growth factor receptor (EGFR) is a known driver of HNSCC, EGFR inhibitors show limited efficacy as monotherapies, suggesting that effective combination therapies are needed.


METHODS
We performed GeCKO and Kinase CRISPR library screens to identify candidate knockouts that increase EGFR inhibitor sensitivity in resistant HNSCC cells. We validated a candidate using the CellTiter-Glo viability assay following RNA interference and investigated the mechanisms using apoptosis and cell cycle flow cytometry analysis.


RESULTS
Kinase CRISPR library screens identified that PIK3C2A gene downregulation enhanced EGFR inhibitor sensitivity in resistant HNSCC cells. Viability assays further validated that PIK3C2A knockdown overcame EGFR inhibitor resistance in HNSCC cells.


CONCLUSIONS
Our data suggests that PIK3C2A is a novel mediator of EGFR inhibitor resistance in HNSCC. Future studies are required to determine the mechanisms by which PIK3C2A drives EGFR inhibitor resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7734f4490eb7420a85bafc6685737b3d4a1310e4" target='_blank'>
              CRISPR Screens Identify PIK3C2A as a Novel Mediator of EGFR Inhibitor Resistance in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Jiayu Wang, M. Ludwig, Aditi Kulkarni, Andrew C Birkeland, Nicole L. Michmerhuizen, E. Gensterblum-Miller, Jingyi Zhai, Hui Jiang, P. Swiecicki, Marisa Buchakjian, M. H. Heft Neal, S. Chinn, M. Spector, J. C. Brenner
          </td>
          <td>2025-09-23</td>
          <td>Head & neck</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, with genetic alterations such as single nucleotide polymorphisms (SNPs) playing a critical role in tumor progression and therapy resistance. Non-synonymous SNPs (nsSNPs) in AKT2, a key kinase in the PI3K/AKT signaling pathway, can impact protein structure and function, leading to reduced efficacy of targeted cancer therapies. This study employs computational approaches to investigate the structural and functional consequences of nsSNPs in the AKT2 and their impact on inhibitor interactions. Three structurally and functionally significant nsSNPs (Y265N, R274H, and R467W) were identified where only R274H and R467W were associated with reduced inhibitor binding. R274H, and R467Wwere found to disrupt key molecular mechanisms, including metal binding, loss of allosteric sites, and alterations in post-translational modifications. Molecular docking revealed that R274H, in kinase domain, disrupts key hydrogen bonds with THR292 and GLU279, leading to more flexible binding pocket and significantly reduced binding affinity for Capivasertib and Ipatasertib. Similarly, R467W, in AGC-kinase C-terminal domain, causes the loss of hydrogen bonds with THR292, ASN280, and GLU279, leading to decreased binding affinity for Akt1/Akt2-IN-1, Capivasertib, and Ipatasertib inhibitors. MD simulations further demonstrated that R274H and R467W caused substantial structural deviations and increased residue flexibility, with R467W exhibiting the most pronounced destabilizing effect. These findings suggest that these mutations may contribute to inhibitor resistance by weakening inhibitor interactions and destabilizing the protein-inhibitor complex. This study underscores the importance of genetic screening in optimizing cancer treatment and highlights the need for mutation-specific therapeutic strategies targeting AKT2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eae9bad202a55fa2b73207b09b4b791549de243" target='_blank'>
              Computational evaluation of AKT2 mutations reveals R274H and R467W as potential drivers of protein instability and inhibitor resistance in cancer therapy
              </a>
            </td>
          <td>
            Sadia Afrin Runa, Mahafujul Islam Quadery Tonmoy, Md Ashiqul Islam, Md Aminul Islam
          </td>
          <td>2025-10-27</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Chemotherapy remains a central strategy in cancer treatment; however, its effectiveness is often diminished by the emergence of drug resistance, which contributes to over 90% of cancer-related deaths. Traditionally, resistance has been linked to genetic mutations and altered drug metabolism. Recent findings, however, highlight the critical role of extracellular vesicles (EVs) in mediating intercellular communication and facilitating adaptive resistance. Objective: This review examines the involvement of tumour-derived EVs in chemotherapy resistance, focusing on their cargo microRNAs (miRNAs), proteins, and lipids and their impact on drug efflux, apoptosis evasion, and immune suppression. Key Findings: EVs facilitate resistance through various mechanisms. For instance, EVs transfer ATP-binding cassette (ABC) transporters such as P-glycoprotein, enhancing drug efflux and promoting multidrug resistance. Oncogenic miRNAs (e.g., miR-21, miR-155) within EVs suppress pro-apoptotic genes (PTEN, CASP3), impeding cell death. Moreover, EVs contribute to tumour microenvironment remodelling by activating cancer-associated fibroblasts and carrying PD-L1 to suppress T-cell responses. Clinical Relevance: EV-derived biomarkers including circulating miRNAs (miR-192, miR-484, miR-205) and DNA mutations (KRAS, TP53, EGFR) offer potential for non-invasive monitoring. Therapeutic strategies such as EV inhibition (e.g., GW4869, Rab27a inhibitors) and engineered EVs for targeted delivery are promising. Conclusion: EV-targeted approaches may transform personalised cancer therapy. Nonetheless, overcoming challenges related to EV heterogeneity, standardisation, and specificity is essential for clinical translation. Future work should prioritise tumour-selective EV inhibition and AI-based biomarker discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ed368d419373eb421b0bc96362c09a7f1f3c08b" target='_blank'>
              Tumour-Derived Extracellular Vesicles in Chemotherapy Resistance: Molecular Pathways, Clinical Implications and Therapeutic Opportunities
              </a>
            </td>
          <td>
            , , , Mukund Pache, A. Darekar
          </td>
          <td>2025-10-04</td>
          <td>Asian Journal of Research in Pharmaceutical Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f7f4234a610f853443cb0ed073e79f12207d8b" target='_blank'>
              Multiple patient-derived glioblastoma models reveal synthetic lethality through concurrent PI3K and CDK4/6 inhibition by blocking trans-active cooperation.
              </a>
            </td>
          <td>
            Jing Zhang, Xu Chen, Meng Cheng, Jingzhe Wang, Yueyao Wu, Han Xie, Chunyu Zhang, Honghao Wang, Ying Pang, Tongjie Ji, Yuntong Yang, Junyu Yang, Siyi Xu, Zhigang Wang, Qi Wang, Min Liu, Chunlong Zhong
          </td>
          <td>2025-09-18</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="BACKGROUND
Ageing is a general, intrinsic and progressively deleterious process that affects all cells, tissues and organs albeit at different extent and rate in each individual. The complexity and universality of its phenotypic manifestations suggest a multifactorial origin. The autosomal recessive disorder Werner syndrome likely represents a segmental progeroid disorder since patients show several, but not all phenotypes of premature ageing.


SUMMARY
Proliferative senescence of diploid cells in culture provided a model system in which ageing can be studied experimentally. Cultures of cells from patients with Werner syndrome experienced an extreme form of proliferative senescence and a clonal succession of translocations, known as variegated translocation mosaicism. In addition, Werner syndrome cells showed spontaneous deletion formation and a prolongation of and arrest in the S phase of the cell cycle. The WRN protein harbors a helicase, an exonuclease and a RecQ interaction domain. With the latter the WRN protein may interact with NBS1, RPA, MRE11, TREX1, MUTYH, POT1, TRF1, FEN-1, PAPRP-1, p97/VCP, TRF2, DNA polymerase(beta), Ku76/80, EXO-1, NEIL1, and p53, which are key to DNA damage response pathways including canonical NHEJ, homologous recombination, base excision repair and telomere maintenance. The WRN exonuclease domain is a target of WRNIP1 binding, which links WRN to resolution of stalled replication due to collision with transcription and the ATM-mediated cell cycle checkpoint. . Patients with an incomplete complement of Werner syndrome phenotypes, called atypical Werner syndrome patients, were found to carry variants in LMNA, POLD1, SPRTN, MDM2, CTC1, SAMHD1.


KEY MESSAGES
These findings broaden the genotypic landscape and the phenotypic spectrum of Werner syndrome. In this review potential molecular mechanisms underlying genomic instability in Werner syndrome, including chromothripsis due to asynchronous S phase traverse and telomere crises followed by bridge fusion breakage cycles are discussed. The participation of WRN in multiple gene networks is consistent with the multifactorial nature of ageing in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c57a230c575d7a7f034afedf23d1a5bec7bbac" target='_blank'>
              Molecular mechanisms of proliferative senescence and genomic instability in Werner syndrome and the WRN gene network.
              </a>
            </td>
          <td>
            Martin Poot
          </td>
          <td>2025-09-19</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) mutations drive resistance and poor prognosis, underscoring the need for more effective therapies. The oxidative drug therapy combining arsenic trioxide (ATO) and D-vitamin C (D-VC) has demonstrated promising efficacy by targeting mitochondrial functions and depleting antioxidant defences to induce apoptosis in CRC cells. ATO and D-VC create a hostile environment for cancer cells by simultaneously targeting mitochondrial metabolism and redox homeostasis, reducing their ability to adapt and survive. This study evaluated the cytotoxic effects of ATO/D-VC in 2D cell cultures and 3D cell models, known as clusteroids, generated from CRC cell lines HCT116 and SW620. In the 2D cultures, the ATO/D-VC combination significantly reduced cell proliferation to 40-60% and viability to below 30% of control levels. In contrast, clusteroids showed a more limited response, with proliferation reduced to 60-80% and viability to 80-90%, highlighting the impact of the extracellular matrix (ECM) and cell-cell interactions in limiting drug diffusion within structured tumour microenvironments. To overcome these drug diffusion barriers, ATO and D-VC were individually encapsulated in poloxamer-stabilized shellac-based nanoparticles (NPs) surface functionalized with Savinase, a protease known to degrade ECM components. The cell viability and cell proliferation assays demonstrated that nanoparticle-mediated delivery significantly enhanced treatment efficacy in clusteroids. Dual treatment of Savinase-coated ATO and D-VC loaded NPs caused pronounced disruption of clusteroid morphology and substantially reduced both viability and proliferation to approximately 30-40% of untreated control levels. Compared to the free drug and uncoated nanoparticle formulations, the Savinase-functionalized nanoparticle formulation achieved nearly twice the reduction in viability and proliferation, indicating a marked improvement in therapeutic effect. Unloaded Savinase-coated nanoparticles showed minimal impact, underscoring their biocompatibility. This approach demonstrates the potential of protease-functionalized nanoparticles to enhance the oxidative drug delivery and efficacy in CRC tumours and could potentially allow targeting the therapeutic resistance in other solid tumours with dense ECM barriers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f5bdfb3097c61efebeecce6a1c806793e42de5b" target='_blank'>
              Savinase-functionalised oxidative drug-loaded nanocarriers enhance the treatment of solid colorectal tumours in a 3D cell culture model.
              </a>
            </td>
          <td>
            Anel G Mun, Nuriya Nurlankyzy, Saule Kalmagambetova, Aidos Baumuratov, D. Sarbassov, V. N. Paunov, Agata N. Burska
          </td>
          <td>2025-10-16</td>
          <td>Journal of materials chemistry. B</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c1083dc843cbe4881a3eab94e321d4cc603b08" target='_blank'>
              H3.3 De Novo Mutations Alter Lysine 36 Methylation via Distinct Mechanisms
              </a>
            </td>
          <td>
            Shahir M. Morcos, S. Sarvan, Alejandro Saettone, Cassandra J. Wong, Samuel Chau Duy Tam Vo, Ayala Milo, Omar H. Bayoumy, Rajesh Angireddy, Giovanni L. Burke, Jack F Greenblatt, Nada Jabado, Carol C.L. Chen, Elizabeth Bhoj, A. Gingras, Jean-François Couture, Eric I. Campos
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Neutrophils are the major populations of white blood cells and have been reported to facilitate cancer metastasis. Meanwhile, emerging evidence has recently suggested the anti-cancer role of neutrophils. Our previous study revealed that CB-839 and 5-FU-treated colorectal cancer (CRC) tumors recruited neutrophils and induced neutrophil extracellular traps (NETs). Cathepsin G (CTSG), which is released during NET formation, enters CRC cells through the receptor for advanced glycation end products (RAGE) and cleaves 14-3-3ε to promote apoptosis. However, the detailed mechanism underlying CTSG’s anti-tumor function remains less studied. In this study, we report that CTSG enters CRC cells through RAGE-mediated endocytosis. Knocking out RAGE or inhibiting endocytosis blocks CTSG from entering CRC cells and attenuates CTSG-induced apoptosis. Furthermore, the clathrin coat assembly complex and SNARE proteins were enriched in an arrayed CRISPR/Cas9 screening targeting human membrane trafficking genes. Knocking out SNARE protein STX1A prevents the spread of CTSG in CRC cells and the induction of cleaved PARP. A pooled genome-wide CRISPR/Cas9 screening further identifies the role of CDK1 in the NET-induced killing of CRC cells. Inhibiting CDK1 protected CRC cells from killing by CTSG. Our study reveals novel mechanisms by which CTSG enters and kills CRC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e75bfdb76d91a9f85d365cd5f61987432f80a06" target='_blank'>
              CRISPR/Cas9 screenings reveal the role of STX1A and CDK1 in Cathepsin G entering and killing colorectal cancer cells
              </a>
            </td>
          <td>
            Yuxiang Wang, V. Rozen, Trang Dinh, He Li, Yamu Li, Zhenghe Wang
          </td>
          <td>2025-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Intratumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by the coexistence of classical epithelial, basal, and mesenchymal cancer cell states. Targeting cancer cell plasticity—the ability of tumor cells to adapt to extrinsic pressures via non-genetic mechanisms—has emerged as a promising therapeutic strategy. In parallel, the immunosuppressive tumor microenvironment remains a major barrier to effective treatment in PDAC. Yet, whether specific cell states possess distinct functional properties that drive key tumor behaviors—such as phenotypic plasticity or immune evasion—remains poorly understood. Using genetically engineered mouse models driven by oncogenic Kras G12D and loss of Trp53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We find that the KPC model harbors a basal state, which harbors transcriptional signature that aligns with the "basal" signature in human PDAC and is linked to poor prognosis and chemoresistance. The basal state co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. To functionally interrogate the basal state in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the basal state has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the basal state is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the basal state over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we find that the basal state is a key driver of therapeutic resistance: rather than being eliminated by treatment, it persists through phenotypic adaptation in response to both KRAS-targeted therapies and cytotoxic chemotherapy. Co-elimination of the basal state alongside these therapies leads to deeper and more durable responses, underscoring its functional role in tumor persistence under therapy. In parallel, we observe that the basal state plays a central role in enforcing immunosuppression in PDAC tumors. Basal state ablation rapidly reshapes the tumor immune microenvironment, and we observe loss of cancer-associated fibroblasts, polarization of PDAC-resident macrophages toward a pro-inflammatory phenotype, and increased infiltration of T and NK cells. Cytokine profiling of basal-ablated tumors revealed a dramatic reduction in multiple immune-modulatory and immunosuppressive cytokines. Taken together, our data indicate that PDAC harbors a striking dependency on the basal state for both therapeutic resistance and immune evasion, and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.



 Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C. Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Zeynep C. Tarcan, Olca Basturk, Rona Yaeger, James G. Christensen, Yan Yan, Elisa de Stanchina, Dana Pe'er, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B108.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6879ea5062e58b2315581060e5992acf02efdbb" target='_blank'>
              Abstract B108: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Kate Ryan, Samuel Rose, Hannah C Styers, Nikhita Pasnuri, Jonathan Rub, Jung Yun Kim, Ashlyn Moore, Jill Hallin, Z. Tarcan, O. Basturk, R. Yaeger, James G Christensen, Yan Yan, E. de Stanchina, D. Pe’er, Tuomas Tammela
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2927a64ad6f25a642a97a344c961d20f78a6873" target='_blank'>
              Neuroblastoma-associated ALK variants have distinct cellular and biochemical activities
              </a>
            </td>
          <td>
            Anna M. Wulf, Stuart Lutimba, Dylan Cameron, Jade Desjardins, Tanya J. Shaw, Leone Rossetti, Mohammad A. Mansour, Karen J. Liu
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0007d62f2b8c21bcd3de104015c7a577a0669693" target='_blank'>
              Defining treatment-resistant brain cancer: Genetic screening to identify oncogene-driven immunomodulation and therapy resistance.
              </a>
            </td>
          <td>
            Zachary David Myers, S. Matosevic
          </td>
          <td>2025-10-26</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06e0ec6da10ec76bbd69b6353780f9b000cfc6a" target='_blank'>
              Overexpression of the TPX2 gene is associated with enhanced tumor malignancy in lung adenocarcinoma and identification of marine natural inhibitors of the Aurora kinase A-TPX2 protein complex
              </a>
            </td>
          <td>
            Md Imtiaz, Partha Biswas, Md. Mohaimenul Islam Tareq, Md Hasan Jafre Shovon, Sadia Jannat Tauhida, Md. Nazmul Hasan Zilani, Md Nazmul Hasan
          </td>
          <td>2025-10-17</td>
          <td>In Silico Pharmacology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="In this work, we investigate the population dynamics of tumor cells under therapeutic pressure. Although drug treatment initially induces a reduction in tumor burden, treatment failure frequently occurs over time due to the emergence of drug resistance, ultimately leading to cancer recurrence. To model this process, we employ a two-type branching process with state-dependent growth rates. The model assumes an initial tumor population composed predominantly of drug-sensitive cells, with a small subpopulation of resistant cells. Sensitive cells may acquire resistance through mutation, which is coupled to a change in cellular fitness. Furthermore, the growth rates of resistant cells are modulated by the overall tumor burden. Using stochastic differential equation techniques, we establish a functional law of large numbers for the scaled populations of sensitive cells, resistant cells, and the initial resistant clone. We then define the stochastic recurrence time as the first time the total tumor population regrows to its initial size following treatment. For this recurrence time, as well as for measures of clonal diversity and the size of the largest resistant clone at recurrence, we derive corresponding law of large number limits. These asymptotic results provide a theoretical foundation for constructing statistically consistent estimators for key biological parameters, including the cellular growth rates, the mutation rate, and the initial fraction of resistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9167a7b57d302ec6999f1b5950893c5788d31753" target='_blank'>
              Parameter Estimation in Recurrent Tumor Evolution with Finite Carrying Capacity
              </a>
            </td>
          <td>
            K. Leder, Zicheng Wang, Xuanming Zhang
          </td>
          <td>2025-10-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Disclosure: M. Jabeen: None. X. Dai: None. J. Zhu: None. P. Hergenrother: SAB Systems Oncology. D.J. Shapiro: None. Classically, the development of resistance to therapies for estrogen receptor (ER) positivebreast cancer, is thought to often involve a single mutational change in ER. In contrast,detailed time course analysis of the process by which ER positive breast cancer cellsbecome resistant to the promising small molecule anticancer drug, ErSO, shows that theacquisition of resistance is a dynamic multi-step process in which cells initially acquireresistance through relatively high frequency events that result in poor growth and cellfitness. These cells are rapidly and progressively replaced by resistant cells with other,apparently lower frequency, changes until resistant cells acquire near wild-type growthrates. We identified the ErSO compounds that work by inducing lethal hyperactivation ofthe normally tumor protective anticipatory unfolded protein response (a-UPR). In ERpositive breast cancer cells key steps in ErSO action are initial activation through anER:Src:PLC complex, resulting in calcium release from endoplasmic reticulum stores,activation of the TRPM4 sodium channel, leading to cell swelling, plasma membranerupture and necrotic cell death. In orthotopic breast and ovarian cancer mousexenografts, ErSO often induced complete regression without recurrence of large primarytumors and most metastases and was effective in PDOs and fresh patient tumor samples.In cell culture and PDOs, we generated more than 10 clonal ErSO-resistant lines.Unexpectedly the resistant clones did not arise from a large pool of non-dividing persistercells and instead derived from a few single survivor cells with initial very slow proliferation,leading to clonal outgrowth. Surprisingly, the initial resistance event in most of the cloneswas massive overexpression of progesterone receptor (PR). PR was known to bind Srctightly, enabling it to sequester Src away from the ErSO activated ER:Src:PLC complex.But overexpression of PR is toxic to cells, and resulted in very slow growing resistantcells. We found that within weeks, these slow-growing clones lost overexpression of PRand engaged other mechanisms that enabled both ErSO-resistance and more robustproliferation. From multiple cycles of RNA-seq done on more than 10 clonal lines asresistance progressed, we obtained snapshots of the evolution of resistance. We did notobserve down-regulator of the intensively studied regulator of plasma membrane rupture,NINJ1. Instead, down-regulation of the TRPM4 sodium channel, other transporters andsystems that might enhance cell survival progressively evolved. These data suggest thatthe evolution of resistance to anticancer agents may often be a dynamic multi-stepprocess rather than a single rare event and that there may be different, potentiallytargetable, vulnerabilities in cancer cells as they progressively evolve resistance toanticancer therapies. Presentation: Saturday, July 12, 2025">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1cc113504dfa0f1063d26be01aa07b7174beef" target='_blank'>
              SAT-302 Acquired Resistance to Breast Cancer Therapy is a Dynamic Multi-step Evolutionary Process
              </a>
            </td>
          <td>
            M. Jabeen, Xinyi Dai, Junyao Zhu, Paul Hergenrother, D. Shapiro
          </td>
          <td>2025-10-01</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is a deadly disease affecting millions of people worldwide. Circulating tumor cells (CTCs) represent a critical link between primary malignancies and metastasis, acting as key players in cancer dissemination, progression, and recurrence. Although rare, CTCs offer a valuable, non-invasive window into tumor biology and the evolution of disease in patients. CTCs can exist as single cells in the circulation, but some are shed and travel in larger groups, referred to as CTC clusters. These clusters possess a greater oncogenic potential compared to individual CTCs. In this review, we aim to provide insight into the dynamic biological processes underlying CTC generation, biology, and survival, with a focus on epithelial-to-mesenchymal transition (EMT) and beyond like cancer stem cells (CSCs), cellular plasticity, and senescence. A crucial aspect of CTC biology is EMT, a process that imparts cancer cells with increased motility, invasiveness, resistance to apoptosis, and the ability to intravasate and evade the immune system. Beyond EMT the cancer cells show further plasticity, allowing epithelial tumor cells to adopt mesenchymal or hybrid phenotypes, which enables adaptation to a changing microenvironment and enhances therapy resistance. Moreover, a subset of cancer cells can acquire stem cell-like properties, including self-renewal and tumor-initiating capacity. EMT, along with processes such as dedifferentiation, contributes to the generation of cancer stem cells. In recent years, studies have also highlighted the complex and paradoxical role of senescence in CTC biology. While senescence typically results in permanent cell cycle arrest, in cancer cells it may be reversible and can promote tumor cell dormancy, immune evasion, and metastatic reactivation. By exploring the connections between CTCs, EMT, CSCs, plasticity, and senescence, we aim to shed light on the unique biology of CTCs, their metastatic potential, and their contributions to tumor heterogeneity. We hope that a better understanding of these processes will help advance the development of novel biomarkers and therapeutic targets for solid tumors beyond EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a99a639294b1ce88e2f796dfc138b9bd8dbb8609" target='_blank'>
              Circulating tumor cells: indicators of cancer progression, plasticity and utility for therapies
              </a>
            </td>
          <td>
            Tamás Richárd Linkner, Z. Nagy, Alexandra Kalmár, Eszter Farkas, Fruzsina Bányai, Nikolett Szakállas, István Takács, Béla Molnár
          </td>
          <td>2025-10-20</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Rhabdomyosarcoma (RMS) is the most frequent soft tissue sarcoma of childhood. The fusion positive (FP) subtype expressing the chimeric transcriptional factor PAX3-FOXO1, the driver of malignancy, has limited treatment options due to resistance to treatments. As a pediatric cancer, RMS has a low mutational burden with epigenetic transcriptional deregulations playing key roles. Thus, targeting the basic transcriptional machinery including catalytic effectors of transcription is an emerging anti-cancer approach. The transcriptional cyclin-dependent kinase CDK12 regulates the transcription elongation phase, particularly of the DNA damage response (DDR) genes, by phosphorylating Ser2 (pSer) on RNA PolII. In Ewing sarcoma, CDK12 inhibition leads to a DNA repair deficient phenotype and high sensitivity to DNA damaging agents. PAX3-FOXO1 RMS cells display elevated levels of replication stress highly relying to DDR for survival. Therefore, in this study we evaluate the impact of CDK12 inhibition on PAX3-FOXO1 RMS tumorigenic features. We found that CDK12 mRNA levels were highly up-regulated in RMS primary samples compared to healthy control muscles, with one of the highest expression among the adult and pediatric tumors analyzed. Then, we treated PAX3-FOXO1 cell lines with THZ531, an inhibitor of CDK12 and its paralog CDK13. Six hours of THZ531 treatment at nanomolar concentrations decreased pSer2 of RNA PolII and reduced PAX3-FOXO1 transcripts and protein levels. The drug also reduced cell survival in 2D and 3D culture conditions. THZ531-treated cells accumulated in G2/M phase and underwent PARP- and Casp 3/7-dependent apoptosis, increased markers of DNA damage associated to DNA double-strand breaks (neutral comet assay), and reduced the levels of the DDR players ATM, ATR, RAD51, BRCA1 and BRCA2. Similar results were obtained after treatment with another CDK12/13i with Cyclin K degrader ability, SR-4835, and with a CDK12-specific degrader, BSJ-4-116, suggesting on-target effects. The three drugs showed similar low nanomolar concentrations effective on PAX3-FOXO1 RMS organoids. Preliminary RNAseq data in two PAX3-FOXO1 RMS cell lines showed that THZ531 downregulated only a low number of genes, common to the two cell lines. The most downregulated gene families were those of DDR pathways. Moreover, a negative correlation between gene length and gene expression levels was found. CDK12 KO in two RMS cell lines affected cell survival in 2D and 3D conditions compared to control cells suggesting on-target activity of the drugs. Phosphoproteomics analysis after THZ531 treatment showed a significant decrease of a number of serines and threonine phosphopeptides also involving a subset on DDR proteins. THZ531 treatment resulted synergic in vitro with Olaparib (PARPi) and with JQ1 (BRD4i). Interestingly, BRD4 is needed for PAX3-FOXO1 activity on oncogenic super-enhancers. Analysis of nascent RNAs and mapping of RNA PolII to identify direct targets of CDK12 is ongoing. Supported by Associazione Italiana per la Ricerca sul Cancro (AIRC) IG #27794 grant to RR



 Marika Attili, Lucrezia D'Archivio, Riccardo Mazzocchi, Francesca Antonella Aiello, Matteo Cassandri, Silvia Pomella, Adam D. Durbin, Francesco Marampon, Kim PJ Schellekens, Michael T Meister, Jarno Drost, Concetta Quintarelli, Simone Sidoli, Michele Ceribelli, Craig J Thomas, Franco Locatelli, Biagio De Angelis, Kristy R Stengel, Rossella Rota. CDK12 pharmacological targeting promotes cell death by abrogating DNA damage response, impairs PAX3-FOXO1 expression and synergizes with BRD4 and PARP inhibitors in fusion positive rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd33df041a8224c723527c37b828d7c14637d9c" target='_blank'>
              Abstract B015: CDK12 pharmacological targeting promotes cell death by abrogating DNA damage response, impairs PAX3-FOXO1 expression and synergizes with BRD4 and PARP inhibitors in fusion positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            Marika Attili, Lucrezia D'Archivio, Riccardo Mazzocchi, F. Aiello, Matteo Cassandri, S. Pomella, A. Durbin, Francesco Marampon, Kim PJ Schellekens, Michael T. Meister, J. Drost, C. Quintarelli, Simone Sidoli, Michele Ceribelli, Craig J Thomas, Franco Locatelli, B. De Angelis, Kristy Stengel, R. Rota
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Chromatin serves to organize and compact the genome but also functions as a signaling hub for the dynamic regulation of transcriptional programs that control cell type specification. The historical discovery that several pro-differentiation anti-cancer agents target chromatin regulatory enzymes buoyed early interest in developing drugs that modulate chromatin structure and function. Chromatin-based drug discovery has since flourished alongside major advances in discovery chemistry and target selection, producing a rich collection of chemical probes, drugs, and drug candidates targeting chromatin regulatory processes. The substantial growth and maturity of this field over the last several decades provides an opportunity to reflect on the successes and failures associated with translating chromatin regulatory targets into anti-cancer drugs. Taking a target-centric perspective, we discuss the motivation for pursuing specific chromatin regulatory proteins and review the chemistries that enabled small molecule discovery and development. In so doing, we hope to evaluate the strength of these targets, the agents that prosecute them, and the prospects for future efforts in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a47417d8f546abed9ecb1b94933a4b2ad03f57ce" target='_blank'>
              Chromatin Regulatory Targets for Anticancer Therapeutics.
              </a>
            </td>
          <td>
            Paige A. Barta, Trever R. Carter, Michael A. Erb
          </td>
          <td>2025-10-06</td>
          <td>Chemical reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Despite multi-modal treatment intensification, nearly half of children with high-risk neuroblastoma (NBL) still succumb to their disease, underscoring the urgent need for novel therapeutic strategies with potent and durable anti-tumor activity. As in many pediatric cancers, gene expression signatures defining NBL cell state are under epigenetic control. Through epigenetic rewiring of the enhancer landscape, treatment with retinoic acid leads to suppression of oncogenic MYCN expression and subsequently to growth arrest and differentiation. However, the epigenetic and antiproliferative effects of retinoic acid are completely reversible upon drug withdrawal, leading to rapid tumor cell proliferation. Here, we sought to identify epigenetic modifiers that enhance the antiproliferative effects of retinoids in NBL. We conducted a screen with 452 compounds targeting epigenetic modifiers to identify synergistic growth inhibitors. We identified PF-9363, a first-in-class-inhibitor of the histone acetyltransferases KAT6A/B, as synergistically inhibiting NBL growth in combination with retinoic acid in NBL. Importantly, this growth suppression persisted beyond retinoid withdrawal, in stark contrast to retinoid monotherapy. Moreover, the KAT6A/B inhibitor PF-9363 synergizes with RA through sustained downregulation of MYCN and the transcription factors PHOX2B and GATA3, caused by gene expression silencing through PRC2-dependent deposition of H3K27me3 across their regulatory enhancers, extending the antiproliferative and differentiating effects of retinoic acid. In NSG xenograft models of NBL, PF-9363 alone demonstrated significant anti-tumor activity and tumor volumes were significantly reduced compared to vehicle treated mice. Moreover, combination treatment with retinoic acid and PF-9363 led to the most profound and sustained tumor growth suppression, lasting beyond retinoid withdrawal, an effect not observed with either agent alone, where tumors resumed rapid growth following drug removal. Reflecting the epigenetic reprogramming of these tumor cells, expression of GD2 was significantly upregulated in GD2low NBL cells by treatment with retinoic acid plus PF-9363 both in vitro and in vivo, thereby enhancing the efficacy of dinutuximab and anti-GD2 CAR T cell therapy. In conclusion, as KAT6A/B have recently emerged as druggable epigenetic targets in adult cancers—with early-phase clinical trials demonstrating favorable safety profiles and durable responses—our studies highlight KAT6A and KAT6B as promising novel therapeutic targets to enhance the efficacy of differentiation therapy and GD2 immunotherapy in NBL.



 Nina Weichert-Leahey, Alla Berezovskaya Berezovskaya, Mark W. Zimmerman, Francesca Alvarez-Calderon, Marlana Winschel, Silvi Salhotra Salhotra, Nathaniel Mabe Mabe, Ulrike Gerdemann, Kimberly Stegmaier, Adam D. Durbin, Derek A. Oldridge, Brian J. Abraham, A. Thomas Look. KAT6A and KAT6B are therapeutic targets to enhance the efficacy of differentiation therapy and GD2 immunotherapy in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A020-PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3567721ac2b118d5c3f6cec8514027fe0b1cf669" target='_blank'>
              Abstract A020-PR004: KAT6A and KAT6B are therapeutic targets to enhance the efficacy of differentiation therapy and GD2 immunotherapy in neuroblastoma
              </a>
            </td>
          <td>
            Nina Weichert-Leahey, Alla Berezovskaya, Mark W. Zimmerman, Francesca Alvarez-Calderon, Marlana B Winschel, S. Salhotra, Nathaniel Mabe Mabe, Ulrike Gerdemann, K. Stegmaier, Adam D. Durbin, Derek A. Oldridge, Brian J. Abraham, A. Look
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Immune evasion is a hallmark of cancer development and poses an important impediment to the effectiveness of immune checkpoint inhibitors (ICIs). Cancer cells take advantage of heterogeneous intrinsic and extrinsic pathways to circumvent immune detection, such as metabolic remodeling (e.g., increased glycolysis, activation of IDO1), genomic mutation (e.g., JAK/STAT, β-catenin), and epigenetic suppression of immune-regulatory genes. Concurrently, TME promotes immune suppression through Tregs, MDSCs, TAMs, and fibroblast-mediated extracellular matrix remodeling. Hypoxia and cytokine dysregulation also undermine antigen presentation and T-cell functionality. These immunoevasion strategies form the foundation of both native (innate) and adaptive resistance to ICIs, while recent evidence places emphasis on microbiota composition being able to modify therapeutic response. The PD-1/PD-L1 pathway remains the focus of ICI therapy, but PD-L1 expression is limited by spatial, temporal, and technical heterogeneity. Beyond PD-L1, integrated biomarker approaches including tumor mutational burden (TMB), microsatellite instability (MSI), IFN-γ gene signatures, and circulating tumor DNA (ctDNA) have arisen to further inform patient stratification. Emerging therapeutic technologies—e.g., dual checkpoint blockade, engineered cytokines, personalized neoantigen vaccines, and adoptive T cell therapy (CAR-T, TCR-T)—are designed to overcome resistance and maximize clinical efficacy. Integration of multi-omics and AI-based models provides additional precision in the tailoring of immunotherapy. This review integrates existing knowledge of immune escape and resistance, highlighting dynamic biomarker development and combinatorial approaches for next-generation personalized cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6fa22f99aea24aa51b0cd66f7e9b27adb85e90" target='_blank'>
              Immune Evasion and Resistance in Cancer Progression: Overcoming Checkpoint Inhibition Challenges with Personalized Immunotherapy Guided by PD-L1 Expression
              </a>
            </td>
          <td>
            Archana Venkatesan, Karthick Sivanatham
          </td>
          <td>2025-10-21</td>
          <td>Journal of Analytical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The tumor microenvironment (TME) is composed of a diverse and dynamic spectrum of cell types, cellular activities, and cell–cell interactions (CCI). Understanding the complex CCI within the TME is critical for advancing cancer treatment strategies, including modulating or predicting drug responses. Recent advances in omics technologies, including spatial transcriptomics and proteomics, have allowed improved mapping of CCI within the TME. The integration of omics insights from different platforms may facilitate the identification of novel biomarkers and therapeutic targets. This review discusses the latest computational methods for inferring CCIs from different omics data and various CCI and drug databases, emphasizing their applications in predicting drug responses. We also comprehensively summarize recent patents, clinical trials, and publications that leverage these cellular interactions to refine cancer treatment approaches. We believe that the integration of these CCI-focused technologies can improve personalized therapy for cancer patients, thereby optimizing treatment outcomes and paving the way for next-generation precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb307e16bbb2fc4780518da53d8301520562b427" target='_blank'>
              Cell–cell interactions as predictive and prognostic markers for drug responses in cancer
              </a>
            </td>
          <td>
            Xuehan Lu, Xiao Tan, Eun Ju Kim, Xinnan Jin, Meg L. Donovan, Jazmina L. Gonzalez Cruz, Zherui Xiong, Maria Reyes Becerra de los Reyes Becerra Perez, Jialei Gong, J. Monkman, Divya Agrawal, A. Kulasinghe, Quan Nguyen, Z. Tuong
          </td>
          <td>2025-10-08</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer remains a leading cause of mortality worldwide, with rising incidence and death rates continuing to rise. While conventional treatments such as surgery, radiotherapy, and chemotherapy form the backbone of cancer care, they are often limited by adverse effects, recurrence risk, and incomplete tumor eradication. Tumor immunotherapy—particularly immune checkpoint inhibitors and chimeric antigen receptor (CAR) T cell therapy—has emerged as a transformative approach by activating and reprogramming anti-tumor immune responses. Despite these advances, significant challenges persist, including limited response rates to checkpoint inhibitors, the immunosuppressive nature of the tumor microenvironment (TME), and resistance mechanisms employed by tumor cells. Growing evidence suggests that immune cell senescence is a critical contributor to TME-driven immunosuppression. Senescent immune cells exhibit functional decline, elevated expression of inhibitory immune checkpoint molecules, and increased secretion of pro-inflammatory cytokines, collectively impairing anti-tumor immunity and reducing the efficacy of immunotherapy. This review highlights the role of immune cell senescence in shaping the immunosuppressive TME and driving resistance to immunotherapy. It further discusses emerging therapeutic strategies that combine immunotherapy with senescence-targeting interventions, aiming to provide novel insights into the development of more effective cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56cdaa455cdfae646c5bce11b54ada6d03f2699c" target='_blank'>
              Senescent immune cells in the tumor microenvironment: emerging insights into cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Duolun Gao, Peiyan Kan, Yanjie He, Siyu Sun, Lei Tang, Fan Yang
          </td>
          <td>2025-10-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2af718684499511487fc4ba4f7a1a788d70c340" target='_blank'>
              Somatic mutation and selection at population scale.
              </a>
            </td>
          <td>
            A. Lawson, Federico Abascal, Pantelis A. Nicola, Stefanie V. Lensing, A. Roberts, Georgios Kalantzis, Adrian Baez-Ortega, Natalia Brzozowska, J. E. El-sayed Moustafa, Dovile Vaitkute, Belma Jakupovic, A. Nessa, Sam Wadge, M. F. Österdahl, Anna L. Paterson, Doris M. Rassl, R. Alcantara, L. O’Neill, S. Widaa, Siobhan Austin-Guest, Matthew D. C. Neville, Moritz J Przybilla, Wei Cheng, M. Morra, Lucy Sykes, Matthew Mayho, Nicole Müller-Sienerth, N. Williams, Diana Alexander, Luke M. R. Harvey, Thomas Clarke, Alex Byrne, Jamie R Blundell, Matthew D Young, Krishnaa T. Mahbubani, K. Saeb-Parsy, Hilary C. Martin, Michael R Stratton, P. J. Campbell, R. Rahbari, K. Small, I. Martincorena
          </td>
          <td>2025-10-08</td>
          <td>Nature</td>
          <td>4</td>
          <td>90</td>
        </tr>

        <tr id="


 Gliosarcoma, a rare and aggressive variant of IDH-wildtype glioblastoma, is characterized by striking intra-tumoral heterogeneity with alternating glial and mesenchymal differentiation. The mesenchymal component is often enriched at recurrence and associated with therapy resistance, suggesting treatment-driven clonal selection. Despite its poor prognosis and distinct histology, gliosarcoma remains underexplored at the molecular level, with no systematic profiling of its cellular or spatial architecture - limiting progress toward targeted therapies.



 Single-nucleus RNA sequencing (snRNA-seq) was performed on eight FFPE gliosarcoma samples, complemented by spatial transcriptomic profiling (10X Visium) in four matched cases. Multimodal integration enabled high-resolution mapping of malignant and non-malignant compartments. To enhance spatial context extraction and to model tumor-microenvironment interactions, advanced computational approaches, including graph neural networks (GNNs) and convolutional neural networks (CNNs), are being developed.



 Integration of snRNA-seq and spatial transcriptomic data revealed a distinct population of stromal-like malignant cells enriched for mesenchymal signatures. Compared to conventional glioblastoma, gliosarcoma samples exhibited a reduced presence of neural progenitor cell (NPC-like) and oligodendrocyte progenitor cell (OPC-like) states, suggesting early dominance or selective expansion of the mesenchymal compartment. Spatial annotation of the tumor microenvironment revealed immune-rich niches populated by disease-associated microglia (DIMs), damage-associated macrophages (DAMs), and macrophages (MACs). Preliminary spatial modeling indicates potential interactions between immune population and mesenchymal-like tumor cells that may support therapy-resistant niches.



 This ongoing study integrates single-nucleus and spatial transcriptomic profiling to characterize mesenchymal transition and resistance in gliosarcoma. Early findings highlight a distinct mesenchymal-like malignant population and associated immune environments that may drive resistance. Further computational modeling aims to uncover key cellular interactions and molecular programs, that could inform novel therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47d9bc10df09a19173093cb2520a4af4dc46d0d5" target='_blank'>
              P14.25.B DECODING MORPHO-MOLECULAR DIVERSITY IN GLIOSARCOMA AND INSIGHTS INTO RESISTANCE USING SPATIAL TRANSCRIPTOMICS
              </a>
            </td>
          <td>
            D. Savran, N. Schoebe, N. Berghaus, D. Reuss, M. Ratliff, C. Herold-Mende, S. Krieg, W. Wick, A. von Deimling, D. Heiland, F. Sahm, P. Sievers
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>