<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-03 09:26:02 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>17</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Phenotypic plasticity, the capacity of cells to transition between distinct phenotypic and lineage states over time, is a genetically and epigenetically encoded trait essential for normal development and adult tissue homeostasis. In cancer, phenotypic plasticity programs can be deployed aberrantly to enable disease progression and acquired therapeutic resistance. Cancer phenotypic plasticity is a current barrier to achieving cures for advanced cancers using available molecularly targeted therapies. This review summarizes the complex and interconnected molecular pathways implicated in phenotypic plasticity, both in the context of normal tissue homeostasis and cancer. Molecular pathways convergent between these contexts are highlighted while pathways enabling plasticity are distinguished from those that specify the phenotype of already plastic cells. Key unresolved questions in the field are discussed along with emerging technologies that may be used to help answer them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91d0b45e5336c08fda7f8b88d683482dae5405ff" target='_blank'>
              The molecular determinants of phenotypic plasticity in homeostasis and neoplasia
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2024-12-13</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Nongenetic plasticity has emerged as a key driver of cancer drug resistance, yet its precise origins, nature, and consequences are not fully clarified. This review examines technological, computational, and conceptual developments in the nongenetic determinants of drug resistance. We begin by proposing refined definitions of cellular state, fate, and plasticity. We subsequently contextualize the findings from multimodal approaches to investigate plasticity, highlighting how new single-cell lineage-tracing methods provide opportunities for quantitatively capturing state transitions, their timescales and heritability, and how they contribute to resistance mechanisms. We also draw parallels with concepts from developmental biology and microbial persistence research. Next, we cover the role that computational approaches have played in revealing the otherwise latent patterns of heterogeneity that underlie plasticity from complex datasets. We conclude by emphasizing the need for standardized terminology in this rapidly evolving field and the path from bench to bedside.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38fadd15ca85570a489d3058bda72d96573ce1d" target='_blank'>
              Navigating Nongenetic Plasticity in Cancer Drug Resistance
              </a>
            </td>
          <td>
            Hanxiao Sun, Nitu Kumari, Madeline E. Melzer, Vadim Backman, Yogesh Goyal
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Mutations and genome instability are hallmarks of cancer. The advanced Next Generation Sequencing(NGS) technology now enables rapid and cost-effective analysis for tumoral and normal genomic translational research. In this research, mRNA sequencing was applied to get the whole transcriptome data of mouse B16F10 melanoma and CT26 colon carcinoma cells. Up regulation genes and down regulation genes were identified in tumor cells with differentially expressed genes(DEGs) analysis. The DEGs were analyzed with GO, KEGG and Reactome pathway enrichment study. Results demonstrated the DEGs and enriched pathways were mainly associated with cell proliferation and biosynthetic processing. Further, gene fusion and alternative splicing analysis reveled fusion genes and skipped exons were extensive exist within the chromosomes of tumor cells. Together, the integrated DEGs and bioinformatics analysis in this article may shed light on further research of cancer treating targets and molecular mechanisms of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6e94aa8f9282678d155caf1a32035cd06536087" target='_blank'>
              Identification and analysis of DEGs of mouse tumor cells by integrated bioinformatics methods
              </a>
            </td>
          <td>
            Xizhong Jing, Baiyong Li, Yifei Wang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying rapid adaptation to stress is a fundamental question in evolutionary biology. A common mechanism is copy number changes, which can affect single genes, larger genomic regions, or even entire chromosomes in the case of eukaryotes (aneuploidy). Bacterial species with genomes comprising more than one chromosome are common, yet the role of adaptive copy number changes affecting entire chromosomes is unknown. Here, by using a bacterium with two chromosomes (chrI and chrII) as model system and combining evolution experiments in the presence of severe nutrient stress with whole-genome sequencing, we show that chromosomal copy number changes can drive rapid adaptation. We detected increased copy number of the entire chrII in almost half of the sequenced clones, isolated from twelve independent populations. In most cases, copy number increases were driven by mutations in the replication protein of chrII. A higher chrII copy number is likely to be beneficial because chrII codes for the operon responsible for metabolizing D-gluconic acid, the only carbon source present in the medium. Our results revealed a novel mechanism for rapid adaptation in bacteria. Given the prevalence of bacteria with secondary chromosomes, and the enrichment of secondary chromosomes in genes associated with interaction with the environment, this may be an important mechanism for adapting to novel environments that has been overlooked.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e1750352edfd93ae418d5a02e63e98887bb1a2" target='_blank'>
              Chromosomal plasticity can drive rapid adaptation in bacteria
              </a>
            </td>
          <td>
            Luis Vega-Cabrera, O. Skovgaard, Macarena Toll-Riera
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4bbd7ae49e10933acda8e20cbd7e3bb473c50d" target='_blank'>
              Endosomal pH is an evolutionarily conserved driver of phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            Hari Prasad, H. Bv, A. R. Subbalakshmi, S. Mandal, M. Jolly, Sandhya S. Visweswariah
          </td>
          <td>2024-12-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d50475b8002866903aae4806efbfc187f7cb121" target='_blank'>
              Canonical and noncanonical NOTCH signaling in the nongenetic resistance of cancer: distinct and concerted control.
              </a>
            </td>
          <td>
            Xianzhe Huang, Wenwei Chen, Yanyan Wang, Dmytro Shytikov, Yanwen Wang, Wangyi Zhu, Ruyi Chen, Yuwei He, Yanjia Yang, Wei Guo
          </td>
          <td>2025-01-02</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ex vivo drug response profiling is emerging as a valuable tool for identifying drug resistance mechanisms and advancing precision medicine in hematological cancers. However, the functional impact of dysregulation of the epigenome and transcriptome in this context remains poorly understood. In this study, we combined ex vivo drug sensitivity profiling with transcriptomic and epigenomic analyses in diagnostic samples from 597 pediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) patients. Ex vivo resistance to antimetabolites (e.g., cytarabine, thioguanine), glucocorticoids (e.g., dexamethasone, prednisolone), and doxorubicin was independently associated with reduced relapse-free survival (RFS, p < 0.05). Molecular profiling identified pre-treatment DNA methylation and gene expression patterns distinguishing resistant from sensitive cases, revealing key drug resistance signatures. These included aberrant expression of genes related to heme metabolism (e.g. ATPV06A) and KRAS signaling (e.g. ABCB1). Notably, we also observed atypical expression of genes usually restricted to T-cells and other immune cells (e.g., ITK) in resistant BCP-ALL cells. Our findings demonstrate that ex vivo drug response patterns are predictive of clinical outcomes and reflect intrinsic molecular states associated with drug tolerance. This integrative multi-omics approach highlights potential therapeutic targets and underscores the value of functional precision medicine in identifying treatment vulnerabilities in pediatric ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b0bb7a98b1f284795f68f2172436eb0a2df655" target='_blank'>
              Ex Vivo Drug Responses and Molecular Profiles of 597 Pediatric Acute Lymphoblastic Leukemia Patients
              </a>
            </td>
          <td>
            Anna Pia Enblad, O. Krali, Henrik Gezelius, Anders Lundmark, Kristin Blom, Claes Andersson, J. Palle, B. Frost, S. Ryhänen, Trond Flægstad, Ó. Jónsson, Kjeld Schmiegelow, Arja Harila, Peter Nygren, Rolf Larsson, Gudmar Lönnerholm, J. Nordlund, Anna Pia
          </td>
          <td>2024-12-20</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The molecular non-genetic changes of resistance to sotorasib are currently uncertain. The aim of this study was to generate a sotorasib-resistant cell line via selective pressure and systematically examine the molecular and phenotypic alterations caused by resistance.Mutant NCI-H358 (KRASG12C) were exposed to incremental doses (2–512 nM) of sotorasib. Then, resistant clones were separated by single-cell sorting. Proliferation was analyzed in real-time by xCELLigence; protein profiles were quantified by protein arrays; and mRNA expression profile was measured using the PanCancer Pathways panel by NanoString. In silico analyses were conducted from a database comprising patient-derived xenograft (PDX) models and cell lines resistant to sotorasib. AKT and p38. The synergistic effect of combining AKT, p38, and EGFR inhibitors was assessed using the SynergyFinder platform. Additionally, AKT and p38 genes were silenced using esiRNA.Sotorasib-resistant H358-R cell line displayed markers of the mesenchymal-epithelial transition and loss of cell adhesion. Were identified 30 overexpressed genes in the resistance model, implicating in signaling pathways that leads to AKT activation and heightened protein expression levels of phosphorylated AKT and p38. To identify potential therapeutic strategies for overcoming sotorasib resistance, we investigated the combination of AKT and p38 inhibitors. Notably, combined inhibition of AKT (MK2206) and p38 (adezmapimod) restored sensitivity to sotorasib in resistant cell lines, as did silencing AKT expression.These findings underscore the importance of adaptive mechanisms in sotorasib resistance in NSCLC cells contributing by EMT activation and demonstrates synergic combination with AKT and p38 inhibitors to restore sotorasib sensitivity in KRASG12C cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8d2b84647241a84761deba81b95c6e8effb443d" target='_blank'>
              Sotorasib resistance triggers epithelial-mesenchymal transition and activates AKT and P38-mediated signaling
              </a>
            </td>
          <td>
            R. A. Megid, G. G. Ribeiro, I. Gomes, A. C. Laus, L. Ferro Leal, Luciane Sussuchi da Silva, A. A. Ariwoola, J. M. Dias, R. Reis, Renato Jose da Silva-Oliveira
          </td>
          <td>2025-01-30</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="KRAS was long deemed undruggable until the discovery of the switch-II pocket facilitated the development of specific KRAS inhibitors. Despite their introduction into clinical practice, resistance mechanisms can limit their effectiveness. Initially, tumors rely on mutant KRAS, but as they progress, they may shift to alternative pathways, resulting in intrinsic resistance. This resistance can stem from mechanisms like epithelial-to-mesenchymal transition (EMT), YAP activation, or KEAP1 mutations. KRAS inhibition often triggers cellular rewiring to counteract therapeutic pressure. For instance, feedback reactivation of signaling pathways such as MAPK, mediated by receptor tyrosine kinases, supports tumor cell survival. Inhibiting KRAS disrupts protein homeostasis, but reactivation of MAPK or AKT can restore it, aiding tumor cell survival. KRAS inhibition also causes metabolic reprogramming and protein re-localization. The re-localization of E-cadherin and Scribble from the membrane to the cytosol causes YAP to translocate to the nucleus, where it drives MRAS transcription, leading to MAPK reactivation. Emerging evidence indicates that changes in cell identity, such as mucinous differentiation, shifts from alveolar type 2 to type 1 cells, or lineage switching from adenocarcinoma to squamous cell carcinoma, also contribute to resistance. In addition to these nongenetic mechanisms, secondary mutations in KRAS or alterations in upstream/downstream signaling proteins can cause acquired resistance. Secondary mutations in the switch-II pocket disrupt drug binding, and known oncogenic mutations affect drug efficacy. Overcoming these resistance mechanisms involves enhancing the efficacy of drugs targeting mutant KRAS, developing broad-spectrum inhibitors, combining therapies targeting multiple pathways, and integrating immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83c1ae3fc98dfd6ef9151275fdf77ddcf612521a" target='_blank'>
              Mechanisms of Resistance to KRAS Inhibitors: Cancer Cells' Strategic Use of Normal Cellular Mechanisms to Adapt.
              </a>
            </td>
          <td>
            Noritaka Tanaka, Hiromichi Ebi
          </td>
          <td>2024-12-27</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad9d32195a3dad4b1106ede5f2fdac86e111eb66" target='_blank'>
              Cellular and molecular aspects of drug resistance in cancers.
              </a>
            </td>
          <td>
            Rahaman Shaik, M. S. Malik, Sreevani Basavaraju, Jihan Qurban, Fatimah M M Al-Subhi, Sathvika Badampudi, Jagruthi Peddapaka, Azeeza Shaik, Ahmad Abd‐El‐Aziz, Z. Moussa, Saleh A. Ahmed
          </td>
          <td>2024-12-09</td>
          <td>Daru : journal of Faculty of Pharmacy, Tehran University of Medical Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Changes in the copy number of large genomic regions, termed copy number variations (CNVs), contribute to important phenotypes in many organisms. CNVs are readily identified using conventional approaches when present in a large fraction of the cell population. However, CNVs that are present in only a few genomes across a population are often overlooked but important; if beneficial under specific conditions, a de novo CNV that arises in a single genome can expand during selection to create a larger population of cells with novel characteristics. While the reach of single cell methods to study de novo CNVs is increasing, we continue to lack information about CNV dynamics in rapidly evolving microbial populations. Here, we investigated de novo CNVs in the genome of the Plasmodium parasite that causes human malaria. The highly AT-rich P. falciparum genome readily accumulates CNVs that facilitate rapid adaptation to new drugs and host environments. We employed a low-input genomics approach optimized for this unique genome as well as specialized computational tools to evaluate the de novo CNV rate both before and after the application of stress. We observed a significant increase in genome-wide de novo CNVs following treatment with a replication inhibitor. These stress-induced de novo CNVs encompassed genes that contribute to various cellular pathways and tended to be altered in clinical parasite genomes. This snapshot of CNV dynamics emphasizes the connection between replication stress, DNA repair, and CNV generation in this important microbial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c09296cb530f9bd4da8646286b7568688223753" target='_blank'>
              Replication stress increases de novo CNVs across the malaria parasite genome
              </a>
            </td>
          <td>
            Noah Brown, A. Luniewski, Xuanxuan Yu, Michelle Warthan, Shiwei Liu, Julia Zulawinska, Syed Ahmad, Molly Congdon, Webster Santos, Feifei Xiao, J. Guler
          </td>
          <td>2024-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The success of cancer therapy has been significantly hampered by various mechanisms of therapeutic resistance. Chief among these mechanisms is the presence of clonal heterogeneity within an individual tumor mass. The introduction of the concept of cancer stem cells (CSCs)—a rare and immature subpopulation with tumorigenic potential that contributes to intratumoral heterogeneity—has deepened our understanding of drug resistance. Given the characteristics of CSCs, such as increased drug‐efflux activity, enhanced DNA‐repair capacity, high metabolic plasticity, adaptability to oxidative stress, and/or upregulated detoxifying aldehyde dehydrogenase (ALDH) enzymes, CSCs have been recognized as a theoretical reservoir for resistant diseases. Implicit in this recognition is the possibility that CSC‐targeted therapeutic strategies might offer a breakthrough in overcoming drug resistance in cancer patients. Herein, we summarize the generation of CSCs and our current understanding of the mechanisms underlying CSC‐mediated therapeutic resistance. This extended knowledge has progressively been translated into novel anticancer therapeutic strategies and significantly enriched the available options for combination treatments, all of which are anticipated to improve clinical outcomes for patients experiencing CSC‐related relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/467b9483651e22480e8aee88bab2b5c598d31695" target='_blank'>
              Unraveling the underlying mechanisms of cancer stem cells in therapeutic resistance for optimizing treatment strategies
              </a>
            </td>
          <td>
            Yunhan Tan, Siyuan Qin, Zhe Zhang, Yongen Liu, Li Zhou, Bo Li, Edouard C. Nice, Yuanyuan Zhang, Jing Jing
          </td>
          <td>2025-01-10</td>
          <td>MedComm – Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Aim: Mutations in the mitochondrial (mt) genome contribute to metabolic dysfunction and their accumulation relates to disease progression and resistance development in cancer cells. This study explores the mutational status of the mt genome of cisplatin-resistant vs. -sensitive testicular germ cell tumor (TGCT) cells and explores its association with their respiration parameters, expression of respiratory genes, and preferences for metabolic pathways to reveal new markers of therapy resistance in TGCTs. Methods: Using Illumina sequencing with Twist Enrichment Panel, the mutations of mt genomes of sensitive 2102EP, H12.1, NTERA-2, T-cam and resistant 2102EP Cis, H12.1 ODM, 1411HP, 1777NRpmet, NTERA-2 Cis and T-cam Cis cell lines were identified. The mt respiration of the cells was assessed using high-resolution respirometry method (O2k-respirometer Oroboros) and the differential expression profiles of mt respiratory genes were determined using RT-qPCR. Associated preferences for metabolic pathways were compared using Glycolysis/OXPHOS assay. Results: In resistant TGCT cells, new mutations in mt genes MT-ND1-6, MT-RNR, MT-CO1-3, MT-ATP6, and MT-CYB were recognized. The respiratory rates of the 1777NRpmet cell line were the highest, while those of the 1411HP line the lowest; rates of the control and all other TGCT cell lines fell between these two lines. The statistically significant differences in gene expression of the respiratory genes were recorded only in NTERA-2 Cis and T-cam Cis cell lines. Sensitive cell lines NTERA-2 and 2102EP preferred oxidative phosphorylation (OXPHOS), while glycolysis was typical for resistant NTERA-2 Cis, 2102EP Cis and 1411HP cell lines. Metastatic 1777NRpmet cells seem to utilize both. An isogenic pair of cell lines H12.1 and H12.1ODM showed the opposite dependence, sensitive H12.1 preferring glycolysis, while resistant H12.1ODM OXPHOS. Conclusion: In summary, our study identified new mutations in mt genes of resistant TGCT cell lines that are associated with different mt respiration parameters, gene expression patterns and preferences for metabolic pathways, providing potential novel molecular biomarkers that distinguish the resistant TGCT phenotype or specify its histological classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec03801d589beba72991767df646f63f72e7a8" target='_blank'>
              Mitochondrial genome variability and metabolic alterations reveal new biomarkers of resistance in testicular germ cell tumors
              </a>
            </td>
          <td>
            Pavlina Kabelikova, Danica Ivovic, Z. Sumbalová, M. Karhánek, Lucia Tatayova, Martina Skopkova, Michal Cagalinec, V. Bruderová, J. Roska, Dana Jurkovicova
          </td>
          <td>2024-12-18</td>
          <td>Cancer Drug Resistance</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Recent studies indicate that the development of drug resistance and increased invasiveness in melanoma is largely driven by transcriptional plasticity rather than canonical coding mutations. Understanding the mechanisms behind cell identity shifts in oncogenic transformation and cancer progression is crucial for advancing our understanding of melanoma and other aggressive cancers. While distinct melanoma phenotypic states have been well characterized, the processes and transcriptional controls that enable cells to shift between these states remain largely unknown. In this study, we initially leverage the well-established zebrafish melanoma model as a high-throughput system to dissect and analyze transcriptional control elements that are hijacked by melanoma. We identify key characteristics of these elements, making them translatable to human enhancer identification despite the lack of direct sequence conservation. Building on our identification of a zebrafish sox10 enhancer necessary for melanoma initiation, we extend these findings to human melanoma, identifying two human upstream enhancer elements that are critical for full SOX10 expression. Stable biallelic deletion of these enhancers using CRISPR-Cas9 induces a distinct phenotype shift across multiple human melanoma cell lines from a melanocytic phenotype towards an undifferentiated phenotype and is also characterized by an increase in drug resistance that mirrors clinical data including an upregulation of NTRK1, a tyrosine kinase, and potential therapeutic target. These results provide new insights into the transcriptional regulation of SOX10 in human melanoma and underscore the role of individual enhancer elements and potentially NTRK1 in driving melanoma phenotype plasticity and drug resistance. Our work lays the groundwork for future gene-based and combination kinase-inhibitor therapies targeting SOX10 regulation and NTRK1 as a potential avenue for enhancing the efficacy of current melanoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99954db143d577a902929c4be9a53feb2b61c4ba" target='_blank'>
              Specific SOX10 enhancer elements modulate phenotype plasticity and drug resistance in melanoma
              </a>
            </td>
          <td>
            Sophia “Noah” DeGeorgia, Charles K Kaufman
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c2f707eacba8ac9d3c84de9541b146c76849cd9" target='_blank'>
              Exosome-delivered NR2F1-AS1 and NR2F1 drive phenotypic transition from dormancy to proliferation in treatment-resistant prostate cancer via stabilizing hormonal receptors
              </a>
            </td>
          <td>
            Wenbin Chen, Yiyou Mao, YiYuan Zhan, Wenfeng Li, Jun Wu, Xiangming Mao, Bin Xu, Fangpeng Shu
          </td>
          <td>2024-12-18</td>
          <td>Journal of Nanobiotechnology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Early proteomic adaptations in cells serve as an initial line of defense, allowing populations to cope with environmental changes before long-term genetic alterations occur. Therefore, they are important in determining the trajectory taken by populations experiencing stress. Using a representative set of genes and DNA barcoding in , we examine how two facets of these adaptations, gene expression and fitness, are distributed within populations experiencing different types and levels of abiotic stress. Together, our data reveal two fundamentally distinct modes of population-level stress response in yeast. The first mode is complex and exploratory, in which individual members adjust the expression of a large part of their genes, seemingly searching phenotypic space for a solution. This leads to phenotypic and fitness heterogeneity within the population. The second mode is degenerate, with all individuals adopting similar gene-expression levels and obtaining uniform fitness. The first mode is employed when a prevalent stress is encountered, and the second under a rare one. This hints at a role for historical encounters in phenotypic plasticity.




 Published by the American Physical Society
 2025


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6a478c7f0333501510b429436dc82e2028e47df" target='_blank'>
              Variability in Protein Expression and Fitness Under Stress: Distinct Modes of Early Adaptation of Populations
              </a>
            </td>
          <td>
            Maor Knafo, Shahar Rezenman, R. Nevo, Ivgeni Tsigalnitski, Ziv Reich, R. Kapon
          </td>
          <td>2025-01-30</td>
          <td>PRX Life</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Genetic variation leads to phenotypic variability in pluripotent stem cells that presents challenges for regenerative medicine. Although recent studies have investigated the impact of genetic variation on pluripotency maintenance and differentiation capacity, less is known about how genetic variants affecting the pluripotent state influence gene regulation in later stages of development. Here, we characterized expression of more than 12,000 genes in 127 donor-matched Diversity Outbred (DO) mouse embryonic stem cell (mESC) and neural progenitor cell (mNPC) lines. Quantitative trait locus (QTL) mapping identified 2,947 expression QTL (eQTL) unique to DO mNPCs and 1,113 eQTL observed in both mNPCs and mESCs with highly concordant allele effects. We mapped three eQTL hotspots on Chromosomes (Chrs) 1, 10, and 11 that were unique to mNPCs. Target genes of the Chr 1 hotspot were overrepresented for those involved in mRNA processing, DNA repair, chromatin organization, protein degradation, and cell cycle. Mediation analysis of the Chr 1 hotspot identified Rnf152 as the best candidate mediator expressed in mNPCs, while cross-cell type mediation using mESC gene expression along with partial correlation analysis strongly implicated genetic variant(s) affecting Pign expression in the mESC state as regulating the mNPC Chr 1 eQTL hotspot. Together these findings highlight that many local eQTL confer similar effects on gene expression in multiple cell states; distant eQTL in DO mNPCs are numerous and largely unique to that cell state, with many co-localizing to mNPC-specific hotspots; and mediation analysis across cell types suggests that expression of Pign early in development (mESCs) shapes the transcriptome of the more specialized mNPC state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4ad28abad19bc73c3c14d84322dd6bba9b2687" target='_blank'>
              Cross cell-type systems genetics reveals the influence of eQTL at multiple points in the developmental trajectory of mouse neural progenitor cells
              </a>
            </td>
          <td>
            Selcan Aydin, Daniel A. Skelly, Hannah Dewey, J. M. Mahoney, Ted Choi, L. Reinholdt, Christopher L. Baker, Steven C. Munger
          </td>
          <td>2025-01-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background/Aim Bile tract cancer (BTC) is a malignant tumor with a poor prognosis. Recent studies have reported the heterogeneity of the genomic background and gene alterations in BTC, but its genetic heterogeneity and molecular profiles remain poorly understood. Whole-genome sequencing may enable the identification of novel actionable gene mutations involved in BTC carcinogenesis, malignant progression, and treatment resistance. Patients and Methods We performed whole-genome sequencing of six BTC samples to elucidate its genetic heterogeneity and identify novel actionable gene mutations. Somatic mutations, structural variations, copy number alterations, and their associations with clinical factors were analyzed. Results The average number of somatic mutations detected in each case was 53,705, with SNVs accounting for most of these mutations (85.02%). None of the 331 mutations related to BTC in The Cancer Genome Atlas (TCGA) database were found in the mutations identified in our study. A higher prevalence of gene mutations was observed in samples without vascular invasion than in those with vascular invasion. Several genes with differences in mutation accumulation between groups were identified, including ADAMTS7, AHNAK2, and CAPN10. Conclusion Our study provides novel insights into the genomic landscape of BTC and highlights the potential of whole-genome sequencing analysis to identify actionable gene mutations and understand the molecular mechanisms underlying this malignancy. The high mutational burden, structural variations, and copy number alterations observed in BTC samples in this study underscore the genetic complexity and heterogeneity of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34204b05b4cfe3c5397ce50291dbbf864d51918d" target='_blank'>
              Whole-genome Sequencing Analysis of Bile Tract Cancer Reveals Mutation Characteristics and Potential Biomarkers
              </a>
            </td>
          <td>
            T. Kokuryo, M. Sunagawa, J. Yamaguchi, Taisuke Baba, S. Kawakatsu, N. Watanabe, S. Onoe, T. Mizuno, T. Ebata
          </td>
          <td>2024-12-27</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Bladder cancer is one of the most common malignancies of the urinary system and has always presented great challenges in treatment due to its intricate biological features and high recurrence rates. Although great developments were achieved in immunotherapy and targeted therapies within the last decade, therapeutic outcomes for a great number of patients remain unsatisfactory, particularly as to long-term efficacy. Review discusses the molecular mechanisms developed during the process of bladder cancer progression: genetic and epigenetic alterations, dynamics of the tumor microenvironment (TME), and dysregulation and abnormal activation of various signaling pathways—all contributing to therapeutic resistance. It is genetic mutation, especially in both low- and high-grade tumors, that, alongside epigenetic modifications, plays a considerable role in tumor aggressiveness and drug resistance. TME, comprising cancer-associated fibroblasts (CAFs), immunosuppressive cells, and different components of the extracellular matrix (ECM), orchestrates a setting that fosters tumor growth and immune evasion and confers resistance on any therapeutic regime that might be used. The review also provides an overview of PI3K/AKT and MAPK signaling pathways in the progression of bladder cancer and the development of targeted therapies against them. Further, it discusses the challenges and mechanisms of resistance to immunotherapy, including those involving immune checkpoint inhibitors. Other promising approaches include the development of new therapeutic strategies that target not only the signaling pathways but also immune checkpoints in combination therapies. This review aims to contribute to the elaboration of more effective and personalized treatment strategies by fully understanding the underlying mechanisms involved in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ee79f09b15911cd50c65c34f5cba7cd83105949" target='_blank'>
              Redefining bladder cancer treatment: innovations in overcoming drug resistance and immune evasion
              </a>
            </td>
          <td>
            Peng Huang, Jie Wang, Zongze Yu, Jiaan Lu, Zhou Sun, Zhigui Chen
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="With the rapid advances in cancer research, the list of variants and genes that drive human diseases is constantly expanding. Moreover, the FDA has approved more cancer therapies that incorporate a broader set of genomic features than simple gene variants such as Tumor Mutation Burden (TMB), microsatellite instability status (MSI), and fusion events in gene families such as the NTRK receptors. These features currently require multiple testing methods (IHC/FISH/etc.). With the cost of NGS testing dropping, it is now possible to envision an NGS assay able to reliably detect these features without the need for additional testing. The EIPM multidisciplinary team has developed EXaCT-2: a whole exome sequencing (WES) assay that gives the coverage of a targeted assay on cancer genes and the breadth to detect copy number events, cancer-related fusions, and viruses, which can facilitate diagnostic and therapeutic decisions for cancer patients. We evaluated EXaCT-2 on 250 matched tumor/normal pairs and compared its performance with orthogonally validated results. We show the assay achieves the expected coverage of critical cancer genes, provides a better characterization of somatic copy number alterations, detects common cancer rearrangements and viruses, and enables the accurate estimation of global molecular metrics, such as tumor mutational burden and microsatellite instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd2c47d3b761e9f5ea55d7bc3672fcf26321970f" target='_blank'>
              EXaCT-2: An augmented and customizable oncology-focused whole exome sequencing platform
              </a>
            </td>
          <td>
            P. H. Waltman, P. Chandra, K. W. Eng, D. C. Wilkes, H. Park, C. Pabon, T. Kane, P. Delpe, E. Fernandez, K. Gorski, B. Bhinder, M. Simi, A. King, N. Greco, J. Manohar, J. M. Tang, P. Zisimopoulos, M. A. Assaad, T. Teneyck, D. Roberts, J. Monge, F. Demichelis, W. Tam, M. M. Ouseph, Alexandros Sigaras, H. Beltran, H. Rennert, N. Lindeman, W. Song, J. Solomon, J. M. Mosquera, R. Kim, J. Catalano, D. Hassane, M. Sigouros, O. Elemento, A. Alonso, A. Sboner
          </td>
          <td>2024-12-08</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Breast cancer is the most prevalent cancer and the leading cause of cancer-related mortality among women. Recent breakthroughs in breast cancer therapeutics have significantly enhanced outcomes for hormone receptor-positive and HER2-negative subtypes. However, the emergence of drug resistance, particularly in triple-negative breast cancer, presents a formidable challenge. The intricate interplay of genetic and metabolic diversity within breast cancer cells is pivotal to its development. By reprogramming metabolic pathways, cancer cells can adapt and thrive, meeting the demands of survival, growth, and invasion. These metabolic shifts also play a key role in the development of resistance to conventional therapies. This review explores the genetic and metabolic complexities of breast cancer, emphasizing the diverse subtypes and their unique profiles. We examine how genetic variations and metabolic alterations contribute to breast cancer development and progression, influencing both treatment efficacy and resistance. By integrating insights into the genetic background and metabolic reprogramming of breast cancer subtypes, this review aims to highlight the genetic variations and metabolic alterations that contribute to the pathogenesis of breast cancer, with a vision of advancing more precise and effective targeted therapies as well as discovering of novel diagnostic and prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8d2c71ef9a50520221d0171aab11b20229faa4" target='_blank'>
              Genetic heterogeneity and metabolic reprogramming in breast cancer
              </a>
            </td>
          <td>
            Yarkın Dolaş, Ayşe Buruş, B. Çeltikçi
          </td>
          <td>2024-12-29</td>
          <td>Acta Medica</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62b1219475c2c72e7d5e477e93702ece4b16c3cc" target='_blank'>
              A Genome-Wide CRISPR/Cas9 Screen Identifies Regulatory Genes for Stem Cell Aging.
              </a>
            </td>
          <td>
            Peng Su, Yi-liang Miao
          </td>
          <td>2024-12-21</td>
          <td>Methods in molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent developments in single-cell multi-omics technologies have provided the ability to identify diverse cell types and decipher key components of the tumor microenvironment (TME), leading to important advancements toward a much deeper understanding of how tumor microenvironment heterogeneity contributes to cancer progression and therapeutic resistance. These technologies are able to integrate data from molecular genomic, transcriptomic, proteomics, and metabolomics studies of cells at a single-cell resolution scale that give rise to the full cellular and molecular complexity in the TME. Understanding the complex and sometimes reciprocal relationships among cancer cells, CAFs, immune cells, and ECs has led to novel insights into their immense heterogeneity in functions, which can have important consequences on tumor behavior. In-depth studies have uncovered immune evasion mechanisms, including the exhaustion of T cells and metabolic reprogramming in response to hypoxia from cancer cells. Single-cell multi-omics also revealed resistance mechanisms, such as stromal cell-secreted factors and physical barriers in the extracellular matrix. Future studies examining specific metabolic pathways and targeting approaches to reduce the heterogeneity in the TME will likely lead to better outcomes with immunotherapies, drug delivery, etc., for cancer treatments. Future studies will incorporate multi-omics data, spatial relationships in tumor micro-environments, and their translation into personalized cancer therapies. This review emphasizes how single-cell multi-omics can provide insights into the cellular and molecular heterogeneity of the TME, revealing immune evasion mechanisms, metabolic reprogramming, and stromal cell influences. These insights aim to guide the development of personalized and targeted cancer therapies, highlighting the role of TME diversity in shaping tumor behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a6a26f346274035b31fd9f5e6d93385cd1777b" target='_blank'>
              Leveraging Single-Cell Multi-Omics to Decode Tumor Microenvironment Diversity and Therapeutic Resistance
              </a>
            </td>
          <td>
            Hussein Sabit, B. Arneth, Timothy M. Pawlik, Shaimaa Abdel-Ghany, Aysha Ghazy, Rawan M. Abdelazeem, A. Alqosaibi, I. Al-Dhuayan, J. Almulhim, N. A. Alrabiah, Ahmed Hashash
          </td>
          <td>2025-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Developing resistance to androgen receptor (AR) signaling inhibitors is a significant challenge in the treatment of castration-resistant prostate cancer. Prolonged use of inhibitors like enzalutamide can cause prostate cancer cells to undergo lineage reprogramming, transitioning to neuroendocrine subtypes that no longer rely on AR signaling. These neuroendocrine subtypes are among the most aggressive forms of prostate cancer. During this process of lineage plasticity, cancer cells experience extensive transcriptional rewiring and acquire stem-like properties characterized by increased stemness. Research has shown that prostate cancer cells gain these stem-like traits through the expression of stem cell-associated proteins such as NANOG, particularly under stable and accumulating conditions. The post-translational modification of NANOG at specific sites is critical for maintaining its stability, which in turn enhances the tumorigenic potential of the cells. This review discusses the mechanisms by which NANOG phosphorylation promotes stemness and lineage plasticity in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d23842d78f2e67e6ef1863e8dd082e1ac56f105a" target='_blank'>
              Molecular mechanisms driving lineage plasticity in prostate cancer: NANOG and beyond
              </a>
            </td>
          <td>
            Hamed Maasoumyhaghighi, Mansoureh Nouri, Jinghui Liu, Xiaoqi Liu
          </td>
          <td>2024-12-27</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a65325c104ad2755155bebd8b8e82939a9a6fc" target='_blank'>
              Transposon mediated functional genomic screening for BRAF inhibitor resistance reveals convergent Hippo and MAPK pathway activation events
              </a>
            </td>
          <td>
            Li Chen, Iulian Pruteanu-Malinici, A. Dastur, Xunqin Yin, Dennie Frederick, R. I. Sadreyev, Cyril H. Benes
          </td>
          <td>2025-01-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="BACKGROUND
Genetic testing has traditionally been divided into molecular genetics and cytogenetics, originally driven by the use of different assays and their associated limitations. Cytogenetic technologies such as karyotyping, fluorescent in situ hybridization or chromosomal microarrays are used to detect large "megabase level" copy number variants and other structural variants such as inversions or translocations. In contrast, molecular methodologies are heavily biased toward subgenic "small variants" such as single nucleotide variants, insertions/deletions, and targeted detection of intragenic, exon level deletions or duplications. The boundaries between these approaches are now increasingly blurred as next-generation sequencing technologies and their use for genome-wide analysis are used by both disciplines, therefore eliminating the historic and somewhat artificial separation driven by variant type.


CONTENT
This review discusses the history of genomic nomenclature across both fields, summarizes implementation challenges for the clinical genetics community, and identifies key considerations for enabling a seamless connection of the stakeholders that consume variant descriptions.


SUMMARY
Standardization is naturally a lengthy and complex process that requires consensus building between different stakeholders. Developing a standard that not only fits the multitude of needs across the entities that consume genetic variant information but also works equally well for all genetic variant types is an ambitious goal that calls for revisiting this vision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3059d272054cb1568f7b2a0448e22020a17460e8" target='_blank'>
              Standardization of Genomic Nomenclature across a Diverse Ecosystem of Stakeholders: Evolution and Challenges.
              </a>
            </td>
          <td>
            Laura K Conlin, Melissa J Landrum, Robert R Freimuth, Birgit Funke
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/660334082207693e7b39f13fca8e1d97d42e1441" target='_blank'>
              Comprehensive review of drug resistance in mammalian cancer stem cells: implications for cancer therapy
              </a>
            </td>
          <td>
            B. Mengistu, Tirunesh Tsegaw, Yitayew Demessie, Kalkidan Getnet, A. Bitew, M. Kinde, Asnakew Mulaw Beirhun, Atsede solomon Mebratu, Yesuneh Tefera Mekasha, Melaku Getahun Feleke, Melkie Dagnaw Fenta
          </td>
          <td>2024-12-18</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86f4e5647ecb2016c81a38b053dec4bd9af04b68" target='_blank'>
              Insights into genetic aberrations and signalling pathway interactions in chronic lymphocytic leukemia: from pathogenesis to treatment strategies
              </a>
            </td>
          <td>
            A. Sobczyńska-Konefał, Monika Jasek, Lidia Karabon, E. Jaskuła
          </td>
          <td>2024-12-28</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Structural variation is a major contributor to human diversity, adaptation, and disease. Simple structural variant (SV) types include deletions, insertions, duplications, inversions, and translocations, and SVs account for most of the variable bases between genomes. Complex structural variants (CSVs) that consist of one or more simple events in cis appear more frequently in diseases and cancers where DNA repair, apoptosis, and cell cycle checkpoints are compromised, although CSVs can also appear in germline genome sequences of healthy individuals. CSVs are often characterized by short tracts of homology or no homology, and while CSVs are more prevalent in complex regions that contain large repeats, smaller stretches of homology can also enable their formation across more unique loci. Long-read assemblies have increased the size of detectable SVs and expanded variant detection into more complex regions of the genome, and while they reconstruct CSVs, methods for identifying CSVs from assemblies is limited. Here, we have developed a new assembly-based approach to trace through complex loci rather than relying upon reference representations of alignments. We can now access CSVs in large complex segmental duplications, reveal structures that were previously unknown, and identify SV breakpoints with greater accuracy. We find 72 large CSVs per genome and 128 unique complex structures and CSVs in highly repetitive regions can now be detected including several distinct complex events in repetitive NBPF genes that was not previously callable with short-read or long-read CSV methods. This approach is implemented within a key assembly-based variant calling tool, PAV, and represents a substantial improvement identifying complex variants now ascertainable from contiguous genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbfc7a889c92ffa27e1ac1ac1e0ce4aa98dae5ef" target='_blank'>
              Large complex structural rearrangements in human genomes harbor cryptic structures
              </a>
            </td>
          <td>
            P. Audano, Carolyn A. Paisie, Christine R. Beck
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Breast cancer (BC) is considered as the most frequent cancer among women and a significant contributor to mortality. The CRISPR/Cas9 gene-editing tool has promising applications for BC drug resistance. It is a unique and creative approach that has lately attracted attention and can be used to fix gene alterations related to multidrug resistance. Recent research has effectively investigated and targeted particular genes linked to BC treatment resistance using CRISPR/Cas9 gene editing, including those linked to hormone receptor signaling, drug efflux transporters, and DNA repair pathways. The CRISPR/Cas9 technology's selective disruption or mutation of these genes provides valuable information about their role in resistance and paves the path for cutting-edge treatment options. CRISPR/Cas9 gene editing can overcome BC treatment resistance by identifying crucial genetic variables and revealing new therapeutic targets Despite the advantages, there are limitations in the study on CRISPR/Cas9-based gene editing for BC treatment resistance, for example, off-target effects and the improvement of delivery techniques are still major issues. Successful clinical translation depends on methods to improve the specificity and effectiveness of CRISPR/Cas9 editing and to solve these constraints. This review aims to explore the possibility of CRISPR/Cas9 gene editing as an innovative method of combating BC medication resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4162c264e1c96e6bc11bcded7869da58b56f7ba" target='_blank'>
              Molecular Mechanisms of Breast Cancer Drug Resistance and CRISPR/Cas9 Strategies to Overcome
              </a>
            </td>
          <td>
            B. Hussen, Bnar Saleh Ismael, Saman S. Abdulla, Noor Haval Jamal, S. A. Mustafa, Zana Baqi Najmalddin, Mohammed Fatih Rasul
          </td>
          <td>2024-12-30</td>
          <td>BioMed Target Journal</td>
          <td>1</td>
          <td>27</td>
        </tr>

        <tr id="Metabolic reprogramming, particularly lipid metabolism, is a hallmark of cancer progression and a critical vulnerability in prostate cancer (PCa). Two pivotal studies, one by Lu et al. and the other by Dairo et al., have illuminated the roles of HOXB13, a transcriptional regulator, and fatty acid synthase (FASN), a key enzyme in lipid biosynthesis, in the metabolic dysregulation of PCa. The Lu et al. study highlights HOXB13’s role in androgen receptor (AR)-independent PCa, where it regulates lipid metabolism via epigenetic mechanisms involving histone deacetylase HDAC3. A G84E mutation in HOXB13 disrupts this regulation, leading to increased lipid synthesis and a pro-metastatic phenotype. The Dairo et al. study demonstrates that FASN hypomethylation, coupled with increased expression, drives lipid biosynthesis critical for tumor growth. Both studies establish a link between HOXB13 mutations and FASN dysregulation, underscoring their interplay in PCa biology. Therapeutically, pharmacological inhibition of FASN mitigates the aggressive features of HOXB13-deficient or mutant PCa, highlighting lipid metabolism as a promising target. Despite their strengths, including robust methodologies, limitations include reliance on preclinical models and the need for broader patient diversity. These studies collectively emphasize the potential of metabolic and epigenetic interventions for precision medicine in PCa, paving the way for novel therapies targeting lipid metabolism in patients with specific genetic and epigenetic profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc58c2230658f37cc79a630362aeb84c369fdaa5" target='_blank'>
              Metabolic Reprogramming in Prostate Cancer: The Roles of HOXB13 and FASN in Tumor Progression and Therapy
              </a>
            </td>
          <td>
            Faisal Saeed, Lara Harik
          </td>
          <td>2025-01-14</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to randomly generated nucleotide barcodes. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple-negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3ba3313bee111d463805c5f2fb966ea7f8b649e" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) play a central role in tumor progression, recurrence, and resistance to conventional therapies, making them a critical focus in oncology research. This review provides a comprehensive analysis of CSC biology, emphasizing their self-renewal, differentiation, and dynamic interactions with the tumor microenvironment (TME). Key signaling pathways, including Wnt, Notch, and Hedgehog, are discussed in detail to highlight their potential as therapeutic targets. Current methodologies for isolating CSCs are critically examined, addressing their advantages and limitations in advancing precision medicine. Emerging technologies, such as CRISPR/Cas9 and single-cell sequencing, are explored for their transformative potential in unraveling CSC heterogeneity and informing therapeutic strategies. The review also underscores the pivotal role of the TME in supporting CSC survival, promoting metastasis, and contributing to therapeutic resistance. Challenges arising from CSC-driven tumor heterogeneity and dormancy are analyzed, along with strategies to mitigate these barriers, including novel therapeutics and targeted approaches. Ethical considerations and the integration of artificial intelligence in designing CSC-specific therapies are discussed as essential elements of future research. The manuscript advocates for a multi-disciplinary approach that combines innovative technologies, advanced therapeutics, and collaborative research to address the complexities of CSCs. By bridging existing gaps in knowledge and fostering advancements in personalized medicine, this review aims to guide the development of more effective cancer treatment strategies, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99151138c6fc7abf05753313b772a7f6497b1128" target='_blank'>
              Deciphering the Role of Cancer Stem Cells: Drivers of Tumor Evolution, Therapeutic Resistance, and Precision Medicine Strategies
              </a>
            </td>
          <td>
            Mohamed El-Tanani, S. Rabbani, S. Satyam, I. Rangraze, A. Wali, Yahia El-Tanani, Alaa A. A. Aljabali
          </td>
          <td>2025-01-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Tumor heterogeneity is a significant factor influencing cancer treatment effectiveness and can arise from genetic, epigenetic, and phenotypic variations among cancer cells. Understanding how tumor heterogeneity impacts tumor evolution and therapy response can lead to more effective treatments and improved patient outcomes. Traditional bulk genomic approaches fail to provide insights into cellular-level events, whereas single-cell RNA sequencing (scRNA-seq) offers transcriptomic analysis at the individual cell level, advancing our understanding of tumor growth, progression, and drug response. However, implementing single-cell approaches in clinical trials involves challenges, such as obtaining high-quality cells, technical variability, and the need for complex computational analysis. Effective implementation of single-cell genomics in clinical trials requires a collaborative “Team Medicine” approach, leveraging shared resources, expertise, and workflows. Here, we describe key technical considerations in implementing the collection of research biopsies and lessons learned from integrating scRNA-seq into City of Hope’s clinical trial design, highlighting collaborative efforts between computational and clinical teams across breast, brain, and ovarian cancer studies to understand the composition, phenotypic state, and underlying resistance mechanisms within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e501cb60571ffc07f7277f4aa067130d5278a3" target='_blank'>
              Single-Cell Transcriptomics Sheds Light on Tumor Evolution: Perspectives from City of Hope’s Clinical Trial Teams
              </a>
            </td>
          <td>
            Patrick A. Cosgrove, Andrea H. Bild, T. Dellinger, B. Badie, J. Portnow, Aritro Nath
          </td>
          <td>2024-12-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Despite applying conventional predictive methodologies to obtain genomic insights, predicting drug sensitivity for healthcare organizations in the USA remains a daunting challenge. Cancer is a highly dynamic, adaptive disease tumor cells have repeatedly shown the capability to evolve mechanisms whereby therapeutic interventions can be evaded. Besides, one genomic alteration seldom predicts drug sensitivity. This research project aimed to address the challenges of predicting drug sensitivity by leveraging the GDSC dataset, an extensive resource connecting genomic profiles of cancer cell lines with their sensitivity to a wide range of anti-cancer drugs. This research's key focus was identifying robust genomic markers, including any specific mutations, gene expression patterns, or epigenetic modifications associated with drug sensitivity or resistance. Advanced machine learning and statistical methods were utilized by the predictive models to analyze complex relations that may exist between different genomic alterations and their drug sensitivity. The dataset used for this research project was derived from the Kaggle website. This dataset was compiled by the research project Genomics of Drug Sensitivity in Cancer collaboration between the Sanger Institute in the United Kingdom and the Massachusetts General Hospital Cancer Center in the United States. In their investigation, there was a massive screening of human cancer cell lines with a wide range of anti-cancer drugs. Data collection was performed by large-scale screening of diverse anti-cancer drugs against human cancer cell lines of various types. Cell viability was measured using the Cell-Titer-Glo assay following 72 hours of drug treatment. Several machine learning models were deployed, namely, Random Forest, Linear Regression, and XG-Boost, which exhibited specific strengths. Specific performance metrics used included MSE, RMSE, MAE, and R². As the statistics indicate, among the three models, Random Forest stands out and performs the best on this dataset across all metrics. A smaller value of MAE, MSE, and RMSE signifies that it provided the best forecast for the target variable. It also gave the highest R-squared value. Application of drug sensitivity prediction analysis in cancer can provide an overview of the mechanisms that underlie both tumor response and resistance by investigating the model predictions. The proposed predictive models have the potential to make significant impacts on clinical decision-making in cancer therapy. Predictive models derive informed decisions regarding a patient's risk of recurrence of disease, their response to certain therapies, and their prognosis based on complex clinical and genomic data. 
Keywords: Genomic Predictors; Drug Sensitivity; Genomic Markers; Personalized medicine; Machine Learning; Random Forest Algorithm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3597d32cedd5f2f5690112a3136db0246ef249" target='_blank'>
              Genomic predictors of drug sensitivity in cancer: Integrating genomic data for personalized medicine in the USA
              </a>
            </td>
          <td>
            Laxmi Pant, Abdullah Al Mukaddim, Md khalilor Rahman, Abdullah Al Sayeed, Md Sazzad Hossain, MD Tushar Khan, Adib Ahmed
          </td>
          <td>2024-12-09</td>
          <td>Computer Science &amp; IT Research Journal</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Background Cellular plasticity, involving loss of lineage determination and emergence of hybrid cell states, plays a pivotal role in non-small cell lung cancer (NSCLC) disease progression and therapy resistance. However, the full spectrum of atypical states generated in human NSCLC and the pathways that regulate them are yet to be fully elucidated. Here we examine the role of developmental programmes, alveogenesis and branching morphogenesis (BM), in regulating phenotypic diversity in NSCLC. Methods Transcriptomic analysis of epithelial cells isolated from murine lungs at different stages of organogenesis were used to derive gene signatures for developmental programmes. Bulk tissue transcriptomic datasets from human NSCLC and non-neoplastic control samples were used to identify whether developmental programmes were associated with molecular, morphological, and clinical parameters. Single-cell RNA-sequencing was used to identify malignant cell states in human NSCLC (n = 16,621 epithelial cells from 72 samples) and protein level validation of these states was carried out using multiplexed immunohistochemistry (n = 40). Results Mutually antagonistic regulation of alveogenesis and BM was found to account for a significant proportion of transcriptomic variance in human NSCLC bulk tissue datasets. BM activation was associated with poor overall survival rates in five independent lung adenocarcinoma (LUAD) cohorts (p=2.04e-13); and was significantly prognostic for resistance to tyrosine kinase inhibitors (TKIs; p=0.003) and immune checkpoint blockade (ICBs; p=0.014), in pre-treatment biopsies. Single-cell RNA-sequencing analysis revealed that malignant LUAD cells with loss of alveolar lineage fidelity predominantly acquired inflamed or basal-like cellular states, which were variably persistent in samples from TKI and ICB recurrence. Conclusions Our results show LUAD tumours undergo reversion from an alveogenic to branching morphogenic phenotype during disease progression, generating inflamed or basal-like cell states that are variably persistent following TKI or ICB treatments. These findings identify prognostic biomarkers for therapy response and underscore the role of different cell states in resistance to multiple treatment modalities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/390cd095a97603dc458c49ab8aaa4b76e0155529" target='_blank'>
              Developmental programmes drive cellular plasticity, disease progression and therapy resistance in lung adenocarcinoma
              </a>
            </td>
          <td>
            Kamila J Bienkowska, Stephany Gallardo Y, Nur S Zainal, Matthew Ellis, Maria-Antoinette Lopez, Judith Austine, Sai Pittla, S. Chee, A. Alzetani, Emily C Shaw, Christian H Ottensmeier, Gareth J Thomas, Christopher Hanley
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is one of the deadliest human cancers, with the overall 5-year survival rate stagnating in recent decades due to the lack of innovative treatment approaches. Apart from the recently Food and Drug Administration-approved epidermal growth factor receptor inhibitor and immune checkpoint inhibitor, alternative therapeutic strategies that target epigenetic abnormalities, an emerging cancer hallmark, remain to be fully explored. A pathological epigenetic landscape, characterized by widespread reprogramming of chromatin modifications such as DNA methylation and histone modifications, which drives transcription deregulation and genome reorganization, has been extensively documented in numerous cancers, including HNSCC. Growing evidence indicates that these frequent epigenomic alterations play pivotal roles in regulating malignant transformation, promoting metastasis and invasion, and reshaping the tumor microenvironment. Furthermore, these epigenetic changes also present unique vulnerabilities that open new avenues for identifying novel prognostic biomarkers and developing targeted antitumor therapies. In this review, we summarize recent discoveries of epigenetic dysregulations in HNSCC, with a focus on deregulated chromatin modifications, which include aberrant DNA methylation, oncohistone H3 lysine 36 to methionine (H3K36M) mutation, as well as recurrent mutations or altered expression of chromatin-modifying enzymes such as NSD1, EZH2, and KMT2C/D. Importantly, we discuss the various molecular mechanisms underlying the contributions of these epigenetic alterations to HNSCC development, particularly their involvement in deregulated cell proliferation and cell death, metabolic reprogramming, tumor immune evasion, and phenotypic plasticity. Finally, we conclude by highlighting the translational and clinical implications of targeting the epigenetic machinery, which offers promising prospects for overcoming resistance to conventional radiotherapy/chemotherapy and enhancing the response to immunotherapy in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f6940a95157f857134f2791aea649c5fe14cbd" target='_blank'>
              Targeting Epigenetic Dysregulations in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Y. Li, C. Lu
          </td>
          <td>2024-12-19</td>
          <td>Journal of dental research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 The concept of synthetic lethality has long held promise for the next generation of genetically targeted cancer therapies to follow the success of PARP inhibitors in the treatment of BRCA-mutant cancers. The classical definition of synthetic lethality arose in Drosophila with the observation that loss of either of two genes independently had little effect on cell viability but loss of both led to cell death. This definition has now been expanded to capture pairs of genes in which genetic alteration of one gene and pharmacological inhibition of the other leads to death of cancer cells while sparing the normal cells which lack the genetic alteration, leading to a broad therapeutic index. Functional genomics screening (using unbiased RNAi or CRISPR-based technology) has enabled the systematic discovery of novel synthetic lethal targets for drug discovery. The selective dependence of MTAP-deleted cells on PRMT5 is one of the strongest genetic interactions observed in early RNAi screens. Approximately 10-15% of all human cancer is MTAP-deleted, providing a large and diverse patient population. MTA-cooperative PRMT5 inhibitors, which bind preferentially in the presence of MTA, have been developed by scientists at Tango Therapeutics and elsewhere to leverage the synthetic lethal relationship between PRMT5 and MTAP loss. MTA-cooperative PRMT5 inhibitors inhibit PRMT5 in MTAP-deleted cancer cells and spare MTAP-expressing normal cells leading to clinically well-tolerated and efficacious therapies. MTA-cooperative PRMT5 inhibitors have broad therapeutic potential both as single agent and in combination with other therapies.
 Citation Format: Kimberly Briggs.Leveraging synthetic lethality for the development of novel cancer therapies. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539ee15239f2d3141b9b8d5ed5ea99f9aadfdf6f" target='_blank'>
              Abstract IA002: Leveraging synthetic lethality for the development of novel cancer therapies
              </a>
            </td>
          <td>
            Kimberly J. Briggs
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both in vitro and in vivo. Here, we apply GENEVA to study the KRAS G12C inhibitors, recapitulating known properties of these drugs and uncovering a previously unknown role for mitochondrial activation in cell death induced by KRAS inhibition. We demonstrate that this finding can be leveraged for the development of combination therapies with greater efficacy. Finally, we show that the application of GENEVA with in vivo mouse models revealed epithelial to mesenchymal transition (EMT) as a key mechanism for resistance to KRAS G12C inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198b25a5af544d70c25d15829b150e55c9e084e8" target='_blank'>
              Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations
              </a>
            </td>
          <td>
            Johnny X. Yu, Jung Min Suh, Katerina D. Popova, Kristle Garcia, Tanvi Joshi, Bruce Culbertson, Jessica B. Spinelli, Vishvak Subramanyam, Kevin Lou, K. Shokat, Jonathan Weissman, Hani Goodarzi
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Omics technologies represent the emergence of a new paradigm in biomedical research and clinical practice and now enable us to understand human development and the molecular basis of developmental disorders[1]. Technologies such as genomics, transcriptomics, proteomics, metabolomics, and emerging fields such as epigenomics and microbiomics are transforming the revolution in our ability to decode the mysteries of health and disease in developmental medicine (a field of study that examines and treats children with congenital and developmental conditions)[2].
Genomics: 
Genomics is at the core of developmental medicine, the study of an organism’s full genetic material[3]. Next-generation sequencing (NGS), whole genome sequencing (WGS), and whole exome sequencing (WES) have made dramatic improvements allowing for the better detection of genetic anomalies linked to developmental disorders. Mutations, including single nucleotide polymorphisms (SNPs), copy number variations (CNVs), and structural rearrangements, have been linked to conditions such as autism spectrum disorders (ASD), congenital heart defects, and neurodevelopmental delays [4].
Genomic breakthroughs give early, precise diagnoses for use in personalized therapeutic strategies. Early detection and treatment of inherited metabolic disorders by genetic screening in newborns and when neonates have developed clinical disorders are other examples of early detection and treatment. In addition, new disease pathways have been uncovered, and these can be targeted[5].
Transcriptomics: 
Genomics is when you get the blueprint and transcriptomics is when you get how and when to express the blueprint across developmental stages and tissues. RNA sequencing (RNA-seq) has established itself as a powerful tool for understanding temporal and spatial gene expression patterns and their molecular basis in developmental disorders. For instance, transcriptomic analysis has identified aberrant gene expression that is related to neurodevelopmental conditions, including epilepsy and cerebral palsy[6].
Single-cell RNA sequencing (scRNA-seq) is revolutionizing developmental biology and the ability to study gene expression at single-cell resolution. This technology is especially relevant in developmental medicine where cell lineage specification and cell-cell interactions determine organogenesis. For example, siRNA-seq has shown the developmental trajectories of neuronal and immune cells and the mechanisms of brain malformations [7].
Proteomics and Metabolomics: 
Analysis of the complete set of proteins and metabolites by proteomics and metabolomics, respectively, provides functional context to genomic and transcriptomic variations. Alterations in protein expression and post-translational modifications were demonstrated in proteomic studies in Duchenne muscular dystrophy (DMD) and mitochondrial disorders[8].In contrast, metabolomics discovers biochemical perturbations that support developmental disorders. Metabolic profiling is used for the diagnosis and monitoring of inborn errors of metabolism such as phenylketonuria (PKU). Further, metabolomics has been successfully integrated into noninvasive liquid biopsies, such as blood or saliva, to comprehensively monitor biomarkers of disease progression and treatment efficacy in pediatric populations.[9].
Epigenomics:
Heritable modifications to DNA and histones that control gene activity while leaving the underlying genetic code intact are the province of epigenomics. Environmental factors such as maternal health, nutrition, and toxins have a pivotal role in the development of these modifications.
Several developmental disorders including Beckwith Wiedemann syndrome and Rett syndrome have been linked to DNA methylation and histone modifications. Prenatal stressors and environmental exposures are emerging as contributors to the shaping of epigenetic patterns during critical developmental windows, with lifelong health consequences[10].
Microbiomics: 
Gut microbiota and the human microbiome have been regularly targeted for their function in early development, immune system maturation, and neurodevelopment. The gut brain axis is disrupted in developmental disorders, e.g. autism spectrum disorders and GI dysregulation in children.
High-throughput sequencing-based microbiomics and multi-omics integration provide new insights into the impact of microbial communities on host metabolism, immune functioning, and cognitive development. Promising clinical applications include targeted probiotic therapies and dietary interventions to restore microbiome balance[11].
Multi-Omics Integration: Towards Precision Medicine
It is possible to transform developmental medicine through the integration of multi-omics data (genomics, transcriptomics, proteomics, metabolomics, and epigenomics). Multi-omics approaches construct comprehensive molecular networks to gain a systems-level understanding of disease pathogenesis and developmental biology.
Liquid biopsies are a major tool for the implementation of multi-omics analyses in clinical practice. These non-invasive samples enable the simultaneous analysis of circulating DNA, RNA, proteins, and metabolites, offering real-time views of disease progression and therapeutic responses. In pediatric care, early intervention is key to improving outcomes making such precision approaches particularly relevant[12].
Challenges and future directions:
While omics technologies have the potential to be transformative to developmental medicine, several hurdles prevent their full integration into this field. High costs, complexity of data analysis, and the requirement for standard protocols for sample collection and bioinformatics workflows are included among these. Moreover, whereas omics discoveries need to be validated in large diverse cohorts for translation into actionable clinical outcomes, qualitative cost analyses reveal that they potentially present the most opportunities for translating research findings to clinical practice.
There’s much we can do to solve these problems with artificial intelligence (AI) and machine learning. Tools powered by AI can analyze complex multi-omics datasets and identify biomarkers, predict disease trajectories, and identify new therapeutic targets with unprecedented precision and speed[13].
CONCLUSION
Omics technologies are revolutionizing developmental medicine, allowing for a depth understanding of the genetic molecular, and environmental causes of health and disease. By empowering early diagnosis, precision therapies, and a greater understanding of developmental disorders, these tools are enabling improved clinical outcomes and healthier lives for children going forward.
To realize this potential, it will be necessary to sustain investment in research, technology, and clinical integration. As omics-driven discoveries, precision medicine, and technological innovation continue to converge, we look forward to a bright future where omics will transform the landscape of pediatric care by allowing us to diagnose, treat, and prevent developmental disorders in the next generation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/336212168e8299a91c199a321715e38e6c6bde36" target='_blank'>
              The Impact of Omics Technologies on Advancing Developmental Medicine
              </a>
            </td>
          <td>
            Naveed Shuja
          </td>
          <td>2024-12-18</td>
          <td>DEVELOPMENTAL MEDICO-LIFE-SCIENCES</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic cancer is the result of mutations in crucial genetic markers like KRAS and TP53 that make treatment challenging. This article discusses how CRISPR Cas9 technology can be combined with these markers to create treatments. CRISPR allows for the alteration or repair of these mutations, with the aim of restoring gene function or blocking cancer-causing pathways. For instance, CRISPR has the potential to fix mutations in TP53 or CDKN2A genes and restore SMAD4 signaling or target the KRAS oncogene in the body’s cells. However promising, it may be that CRISPR encounters obstacles like unintentional effects and challenges in effectively delivering it to pancreatic tumor cells. Furthermore, ethical concerns, especially related to the editing of the germline, need consideration. As techniques based on CRISPR advance, there is a chance for them to transform the treatment landscape for cancer by offering personalized therapies. More studies are needed to enhance how treatments are administered accurately and safely through methods and targeted testing for effectiveness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d180ed2014a91157721c0376d7e2ca0a9fe65e1" target='_blank'>
              Integrating CRISPR Technology with Key Genetic Markers in Pancreatic Cancer: A New Frontier in Targeted Therapies
              </a>
            </td>
          <td>
            Dimitrios Stefanoudakis
          </td>
          <td>2025-01-06</td>
          <td>SynBio</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="MAPK pathway inhibitors (MAPKi) are increasingly used in the treatment of advanced colorectal cancer, but often produce short-lived responses in patients. Although acquired resistance by de novo mutations in tumors have been found to reduce response in some patients, additional mechanisms underlying the limited response durability of MAPK targeting therapy remain unknown. Here, we denote new contributory tumor biology and provide insight on the impact of tumor plasticity on therapy response. Analysis of MAPKi treated patients revealed activation of stemness programs and increased ASCL2 expression, which are associated with poor outcomes. Greater ASCL2 with MAPKi treatment was also seen in patient-derived CRC models, independent of driver mutations. We find ASCL2 denotes a distinct cell population, arising from phenotypic plasticity, with a proliferative, stem-like phenotype, and decreased sensitivity to MAPKi therapy, which were named adaptive plasticity tumor (APT) cells. MAPK pathway suppression induces the APT phenotype in cells, resulting in APT cell enrichment in tumors and limiting therapy response in preclinical and clinical data. APT cell depletion improved MAPKi treatment efficacy and extended MAPKi response durability in mice. These findings uncover a cellular program that mitigates the impact of MAPKi therapies and highlights the importance of addressing tumor plasticity to improve clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f697113c6b546482f0f42def83352f20ea919c" target='_blank'>
              Adaptive Plasticity Tumor Cells Modulate MAPK-Targeting Therapy Response in Colorectal Cancer
              </a>
            </td>
          <td>
            O. Villarreal, Yixin Xu, Ha Tran, Annette A. Machado, Dionne Prescod, Amanda Anderson, R. Minelli, Mike Peoples, Alejandro Hernandez Martinez, H. Lee, Chi Wut Wong, Natalie W Fowlkes, Preeti Kanikarla, Alexey Sorokin, J. Alshenaifi, Oluwadara Coker, Kangyu Lin, Chris Bristow, A. Viale, J. P. Shen, C. Parseghian, Joseph R Marszalek, Ryan Corcoran, S. Kopetz
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e564005ed56626c695c387291b72229a4171b45" target='_blank'>
              Synchronized long-read genome, methylome, epigenome and transcriptome profiling resolve a Mendelian condition.
              </a>
            </td>
          <td>
            Mitchell R. Vollger, Jonas Korlach, K. Eldred, Elliott G. Swanson, Jason G. Underwood, Stephanie C. Bohaczuk, Yizi Mao, Y. H. Cheng, Jane Ranchalis, Elizabeth E. Blue, Ulrike Schwarze, Katherine M. Munson, Christopher T. Saunders, Aaron M. Wenger, Aimee Allworth, S. Chanprasert, Brittney L. Duerden, Ian A. Glass, M. Horike‐Pyne, Michelle Kim, K. Leppig, Ian J. McLaughlin, Jessica Ogawa, E. A. Rosenthal, Sam Sheppeard, Stephanie M Sherman, Samuel Strohbehn, Amy L Yuen, Andrew W Stacey, Thomas A Reh, Peter H. Byers, M J Bamshad, F. Hisama, Gail P. Jarvik, Yasemin Sancak, Katrina M. Dipple, A. Stergachis
          </td>
          <td>2025-01-29</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Genomic data is essential for clinical decision-making in precision oncology. Bioinformatic algorithms are widely used to analyze next-generation sequencing (NGS) data, but they face two major challenges. First, these pipelines are highly complex, involving multiple steps and the integration of various tools. Second, they generate features that are human-interpretable but often result in information loss by focusing only on predefined genetic properties. This limitation restricts the full potential of NGS data in biomarker extraction and slows the discovery of new biomarkers in precision oncology. Methods We propose an end-to-end deep learning (DL) approach for analyzing NGS data. Specifically, we developed a multiple instance learning DL framework that integrates somatic mutation sequences to predict two compound biomarkers: microsatellite instability (MSI) and homologous recombination deficiency (HRD). To achieve this, we utilized data from 3,184 cancer patients obtained from two public databases: The Cancer Genome Atlas (TCGA) and the Clinical Proteome Tumor Analysis Consortium (CPTAC). Results Our proposed deep learning method demonstrated high accuracy in identifying clinically relevant biomarkers. For predicting MSI status, the model achieved an accuracy of 0.98, a sensitivity of 0.95, and a specificity of 1.00 on an external validation cohort. For predicting HRD status, the model achieved an accuracy of 0.80, a sensitivity of 0.75, and a specificity of 0.86. Furthermore, the deep learning approach significantly outperformed traditional machine learning methods in both tasks (MSI accuracy, p-value = 5.11×10−18; HRD accuracy, p-value = 1.07×10−10). Using explainability techniques, we demonstrated that the model’s predictions are based on biologically meaningful features, aligning with key DNA damage repair mutation signatures. Conclusion We demonstrate that deep learning can identify patterns in unfiltered somatic mutations without the need for manual feature extraction. This approach enhances the detection of actionable targets and paves the way for developing NGS-based biomarkers using minimally processed data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad073d80b11f20c88ed32369a6393b6a5510e50a" target='_blank'>
              Deep Learning for Biomarker Discovery in Cancer Genomes
              </a>
            </td>
          <td>
            Michaela Unger, C. Loeffler, L. Žigutytė, Srividhya Sainath, Tim Lenz, Julien Vibert, Andreas Mock, S. Fröhling, Trevor A. Graham, Zunamys I. Carrero, J. N. Kather
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Loss of function screens using shRNA (short hairpin RNA) and CRISPR (clustered regularly interspaced short palindromic repeats) are routinely used to identify genes that modulate responses of tumor cells to anti-cancer drugs. Here, by integrating GSEA (Gene Set Enrichment Analysis) and CMAP (Connectivity Map) analyses of multiple published shRNA screens, we identified a core set of pathways that affect responses to multiple drugs with diverse mechanisms of action. This suggests that these pathways represent “weak points” or “Achilles heels”, whose mild disturbance should make cancer cells vulnerable to a variety of treatments. These “weak points” include proteasome, protein synthesis, RNA splicing, RNA synthesis, cell cycle, Akt-mTOR, and tight junction-related pathways. Therefore, inhibitors of these pathways are expected to sensitize cancer cells to a variety of drugs. This hypothesis was tested by analyzing the diversity of drugs that synergize with FDA-approved inhibitors of the proteasome, RNA synthesis, and Akt-mTOR pathways. Indeed, the quantitative evaluation indicates that inhibitors of any of these signaling pathways can synergize with a more diverse set of pharmaceuticals, compared to compounds inhibiting targets distinct from the “weak points” pathways. Our findings described here imply that inhibitors of the “weak points” pathways should be considered as primary candidates in a search for synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68c29dcb0705107711d0d05473f868fd7dbb129" target='_blank'>
              Cancer Cell’s Achilles Heels: Considerations for Design of Anti-Cancer Drug Combinations
              </a>
            </td>
          <td>
            Valid Gahramanov, Frederick S. Vizeacoumar, Alain Morejon Morales, Keith Bonham, M. Sakharkar, Santosh Kumar, Franco J Vizeacoumar, A. Freywald, Michael Y. Sherman
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Despite its relatively recent emergence, single-cell sequencing has cemented its place in scientific research. It has grown exponentially in less than two decades since its start, with broad impact in the biological sciences. The blood represents an attractive system for early development and application of single-cell technologies. As a result, single-cell analyses in blood and leukemia have led the way in describing how cellular heterogeneity affects cancer progression. In this review, we discuss the technological and conceptual advances brought by single-cell genomics, ranging from genetic evolution and differentiation states that mediate drug resistance to the complex interactions required for immunotherapy responses. These high-resolution insights are starting to enter clinical assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fa592467bc471ca2cb2ea46b2951c1d4aa2628" target='_blank'>
              Insights into Leukemia from Single-Cell Genomics
              </a>
            </td>
          <td>
            Nurefsan Sariipek, Peter van Galen
          </td>
          <td>2024-12-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef73b59f7e05ebe6c2a352df52b047a092cb9e6" target='_blank'>
              State of the art CRISPR-based strategies for cancer diagnostics and treatment
              </a>
            </td>
          <td>
            E. Di Carlo, Carlo Sorrentino
          </td>
          <td>2024-12-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Prostate cancer is one of the most prevalent malignant tumors in men, particularly in regions with a high Human Development Index. While the long-term survival rate for localized prostate cancer is relatively high, the mortality rate remains significantly elevated once the disease progresses to advanced stages, even with various intensive treatment modalities. The primary obstacle to curing advanced prostate cancer is the absence of comprehensive treatment strategies that effectively target the highly heterogeneous tumors at both genetic and molecular levels. Prostate cancer development is a complex, multigenic, and multistep process that involves numerous gene mutations, alteration in gene expression, and changes in signaling pathways. Key genetic and pathway alterations include the amplification and/or mutation of the androgen receptor, the loss of Rb, PTEN, and p53, the activation of the WNT signaling pathway, and the amplification of the MYC oncogene. This review summarizes the mechanisms by which these genes influence the progression of prostate cancer and highlights the interactions between multiple genes and their relationship with prostate cancer. Additionally, we reviewed the current state of treatments targeting these genes and signaling pathways, providing a comprehensive overview of therapeutic approaches in the context of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38fab9adcd074d173268fdd423c799e2e23c962b" target='_blank'>
              Interactions between key genes and pathways in prostate cancer progression and therapy resistance
              </a>
            </td>
          <td>
            Fan Wu, Heng Zhang, Miaomiao Hao
          </td>
          <td>2025-01-23</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="FLT3 mutations are among the most common genetic alterations in acute myeloid leukemia (AML) and are associated with poor prognosis. Significant advancements have been made in developing FLT3 inhibitors (FLT3Is), such as quizartinib, which have improved treatment outcomes in both newly diagnosed and relapsed/refractory AML. Resistance to FLT3Is remains a major clinical challenge, driven by diverse mechanisms including FLT3 point mutations, cellular escape pathways, and the influence of the bone marrow microenvironment. Sustained STAT5 phosphorylation, AXL upregulation, and CXCR4 signaling have been identified as key factors in FLT3I resistance. Additionally, metabolic adaptations have been shown to support the survival of FLT3I-resistant cells. Ongoing clinical trials are investigating various combination regimens, including quizartinib with chemotherapy, Bcl-2 inhibitors, hypomethylating agents, and immune-modulatory drugs, with promising preliminary results. The European LeukemiaNet 2022 guidelines recommend incorporating FLT3Is into treatment regimens; however, questions remain regarding the best timing for the administration of each FLT3I. Additional studies are required to determine the optimal FLT3I-based combinations, reduce resistance emergence, and improve outcomes. This review highlights the current state of FLT3I therapy, ongoing challenges with resistance, and future directions in optimizing treatment for FLT3-mutated AML, focusing on quizartinib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3878f024ad9767de696002db14bad327a4d61cfe" target='_blank'>
              Advances and Challenges in Quizartinib-Based FLT3 Inhibition for Acute Myeloid Leukemia: Mechanisms of Resistance and Prospective Combination Therapies.
              </a>
            </td>
          <td>
            A. Bruzzese, E. Martino, Caterina Labanca, F. Mendicino, E. Lucia, Virginia Olivito, Teresa Rossi, A. Neri, F. Morabito, E. Vigna, Massimo Gentile
          </td>
          <td>2025-01-06</td>
          <td>European journal of haematology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Gene therapy has long been a cornerstone in the treatment of rare diseases and genetic disorders, offering targeted solutions to conditions once considered untreatable. As the field advances, its transformative potential is now expanding into oncology, where personalized therapies address the genetic and immune-related complexities of cancer. This review highlights innovative therapeutic strategies, including gene replacement, gene silencing, oncolytic virotherapy, CAR-T cell therapy, and CRISPR-Cas9 gene editing, with a focus on their application in both hematologic malignancies and solid tumors. CRISPR-Cas9, a revolutionary tool in precision medicine, enables precise editing of cancer-driving mutations, enhancing immune responses and disrupting tumor growth mechanisms. Additionally, emerging approaches target ferroptosis—a regulated, iron-dependent form of cell death—offering new possibilities for selectively inducing tumor cell death in resistant cancers. Despite significant breakthroughs, challenges such as tumor heterogeneity, immune evasion, and the immunosuppressive tumor microenvironment (TME) remain. To overcome these barriers, novel approaches like dual-targeting, armored CAR-T cells, and combination therapies with immune checkpoint inhibitors and ferroptosis inducers are being explored. Additionally, the rise of allogeneic “off-the-shelf” CAR-T therapies offers scalable and more accessible treatment options. The regulatory landscape is evolving to accommodate these advancements, with frameworks like RMAT (Regenerative Medicine Advanced Therapy) in the U.S. and ATMP (Advanced Therapy Medicinal Products) in Europe fast-tracking the approval of gene therapies. However, ethical considerations surrounding CRISPR-based gene editing—such as off-target effects, germline editing, and ensuring equitable access—remain at the forefront, requiring ongoing ethical oversight. Advances in non-viral delivery systems, such as lipid nanoparticles (LNPs) and exosomes, are improving the safety and efficacy of gene therapies. By integrating these innovations with combination therapies and addressing regulatory and ethical concerns, gene therapy is poised to revolutionize cancer treatment, providing durable, effective, and personalized solutions for both hematologic and solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a9f2e1247ab5090aa522e48935aeedefea03c5" target='_blank'>
              Enhancing precision in cancer treatment: the role of gene therapy and immune modulation in oncology
              </a>
            </td>
          <td>
            Emile Youssef, Brandon Fletcher, Dannelle Palmer
          </td>
          <td>2025-01-13</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Targeted therapies like Venetoclax have increased the options available to acute myeloid leukemia (AML) patients, but survival remains poor due to drug resistance and disease relapse. We found that the translation initiation factor EIF4A1, which unwinds complex mRNA structures in the 5’ UTR of oncogenic transcripts, is highly expressed in AML stem- and progenitor-like cells. Inhibiting eIF4A with the small molecule Zotatifin reduces translation of transcripts related to the cell cycle and survival. This results in downregulation of AKT, STAT-5, and MCL-1 and underlies synergy of Zotatifin with Venetoclax. The drug combination promotes apoptosis across AML genotypes, while the effect on healthy blood cells is limited. Using in vivo relapsed and refractory AML patient-derived xenograft models, the combination significantly suppressed tumor burden and prolonged survival of xenografted mice. These results support eIF4A-mediated protein translation as a therapeutic target in AML. SIGNIFICANCE Despite advances in targeted therapies, the 5-year survival rate for acute myeloid leukemia remains around 32%. The efficacy of existing treatments may be improved with the addition of Zotatifin, an inhibitor targeting the translation initiation factor eIF4A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8672d4c71b8c253c15ee73780df3c698e781744" target='_blank'>
              Research Brief: Translation Initiation Represents an Acute Myeloid Leukemia Cell Vulnerability That Can Be Co-Targeted With BCL-2 Inhibition
              </a>
            </td>
          <td>
            Y. Lee, Jonathan D. Good, Noha Awais, Benjamin K. Eschle, Maia S. Lineberry, Jessica L. Teo, Sarah Bibeau, Morgan Wollman, George Karadimov, Nurefsan Sariipek, Jean Acosta, Jenny Noel, David Cucchi, E. DePasquale, Prafulla Gokhale, Fadi J. Najm, Richard M. Stone, Jacqueline S. Garcia, Andrew A. Lane, Peter van Galen
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Short tandem repeats (STRs) have been reported to influence gene expression across various human tissues. While STR variations are enriched in colorectal (CRC), stomach (STAD) and endometrial (UCEC) cancers, particularly in microsatellite instable (MSI) tumors, their functional effects and regulatory mechanisms on gene expression remain poorly understood across these cancer types. Results Here, we leverage whole-exome sequencing and gene expression data to identify STRs for which repeat lengths are associated with the expression of nearby genes (eSTRs) in CRC, STAD and UCEC tumors. Our analyses reveal that tumor STR profiles effectively capture both MSI phenotype and population structure. While most eSTRs are cancer-specific, shared eSTRs across multiple cancers exhibit consistent effects on gene expression. Notably, coding-region eSTRs identified in all three cancer types show positive correlations with nearby gene expression. We further validate the functional effects of eSTRs by demonstrating associations between somatic eSTR mutations and gene expression changes during the transition from normal to tumor tissues, suggesting their potential roles in tumorigenesis. Combined with DNA methylation data, we perform the first quantitative analysis of the interplay between STR variations and DNA methylation in tumors. We identify eSTRs where repeat lengths are associated with methylation levels of nearby CpG sites (meSTRs) and show that over 70% of eSTRs are significantly linked to local DNA methylation. Importantly, the effects of meSTRs on DNA methylation remain consistent across cancer types. Conclusions Overall, our findings enhance the understanding of how functional STR variations influence gene expression and DNA methylation. Our study highlights shared regulatory mechanisms of STRs across multiple cancers, offering a foundation for future research into their broader implications in tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a625ce20d0d6f7f915320a2751a41cca4a94608" target='_blank'>
              Multicancer analyses of short tandem repeat variations reveal shared gene regulatory mechanisms
              </a>
            </td>
          <td>
            Feifei Xia, M. Verbiest, Oxana Lundström, Tugce Bilgin Sonay, M. Baudis, Maria Anisimova
          </td>
          <td>2025-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, J. B. Jensen, Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>68</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are a rich source of genetic variation, but are difficult to genotype. While specialized repeat variant callers exist, they typically assume a euploid human genome. This means recent findings regarding phenotypic effects of STR variants in human health and disease cannot be readily extended to polyploid organisms or cancer, which is characterised by copy number alterations (CNAs). Here we present ConSTRain, a novel STR variant caller that explicitly accounts for the copy number of loci in its genotyping approach. We benchmark ConSTRain using a euploid human 100X whole genome sequencing sample where it calls STR allele lengths for over 1.7 × 106 loci in under 20 minutes with an accuracy of 98.28%. Subsequently, we show that ConSTRain resolves complex STR genotypes in an artificial trisomy 21 sample and a polyploid Dwarf Cavendish banana harbouring a large duplication. Finally, we analyse a microsatellite instable colorectal cancer tumoroid, where ConSTRain tackles CNAs and whole-genome duplications. ConSTRain is the first STR variant caller that allows for the investigation of repeats affected by CNAs, aneuploidies, and polyploid genomes. This unlocks the investigation of STRs across a wide range of contexts and organisms where they previously could not be easily studied.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42abd302650eb39af080a99a7d76a61bf128e039" target='_blank'>
              Genotyping Short Tandem Repeats Across Copy Number Alterations, Aneuploidies, and Polyploid Organisms
              </a>
            </td>
          <td>
            M. Verbiest, Elena Grassi, Andrea Bertotti, Maria Anisimova
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epigenetics plays a critical role relative to other branches of genetics, as it makes it possible to regulate gene expression without changing the nucleotide sequence of DNA molecules. This process allows cells to respond to external signals and adapt to  changes in  the  environment while keeping their genetic information intact. The  main mechanisms of  epigenetic regulation include DNA methylation, posttranslational modifications of  histones, chromatin remodeling and  regulation through non-coding RNAs. These processes play a key role in fundamental biological processes such as cellular differentiation, organismal development, and adaptation to environmental conditions.Disturbances in epigenetic mechanisms can lead to various pathologies, including cancer, neurological and  autoimmune diseases. Understanding the  mechanisms of  epigenetic regulation opens new avenues for  the  development of  targeted therapies that can correct abnormal epigenetic profiles without changing the DNA structure itself.In  recent years, the  development and  application of  innovative technologies, such  as  the  CRISPR/Cas9 genome editing system, have significantly expanded the ability to study epigenetic mechanisms and their relationship with diseases. These technologies allow not only a deeper understanding of epigenetic changes, but also the development of new therapeutic approaches, especially in the field of oncology. Research in  epigenetics is  also focusing on  the  interaction between epigenetic changes and the immune system, which opens new perspectives for the development of immunotherapies. The search for new markers of epigenetic disorders and therapeutic agents may lead to the development of individualized treatments that take into account the unique epigenetic profile of each patient.The role of epigenetic modifications in the development of diseases and the creation of new therapeutic strategies cannot be overestimated. Recent research in this area is revealing the potential of epigenetic approaches to treat a wide range of diseases, ushering in a new era in medicine where understanding and correcting epigenetic changes will be the key to effective treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d047bbd846068847a5bb8e163e24692ce5b5c48" target='_blank'>
              Epigenetic modulation in medicine: Regulation of gene expression in the context of pathogenesis and therapy
              </a>
            </td>
          <td>
            M. A. Omarov, A. R. Mulyukov, R. V. Khalitov, S. I. Safarov, G. U. Ayupova, O. N. Demianenko, V. V. Butenko, A. A. Safronova, R. R. Zainetdinova, O. Y. Ianturina, A. A. Miroshnikov
          </td>
          <td>2024-12-28</td>
          <td>Acta Biomedica Scientifica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40243d83674d54172095c39c31dd740a67953065" target='_blank'>
              Comparative transcriptomics suggests a highly species-specific nature of the phenotypic plasticity associated with the outbreaks of the two main pest locusts
              </a>
            </td>
          <td>
            N. Bakkali, S. Saadi, A. Badih, M. Bakkali
          </td>
          <td>2024-12-19</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dysregulated epigenetic programs that restrict differentiation, reactivate fetal genes, and confer phenotypic plasticity are critical to colorectal cancer (CRC) development. By screening a small molecule library targeting epigenetic regulators using our dual reporter system, we found that inhibiting histone deacetylase (HDAC) 1/2 promotes CRC differentiation and anti-tumor activity. Comprehensive biochemical, chemical, and genetic experiments revealed that on-target blockade of the HDAC1/2 catalytic domain mediated the differentiated phenotype. Unbiased profiling of histone posttranslational modifications induced by HDAC1/2 inhibition nominated acetylation of specific histone lysine residues as potential regulators of differentiation. Genome-wide assessment of implicated marks indicated that H3K27ac gains at HDAC1/2-bound regions associated with open chromatin and upregulation of differentiation genes upon HDAC1/2 inhibition. Disrupting H3K27ac by degrading acetyltransferase EP300 rescued HDAC1/2 inhibitor-mediated differentiation of a patient-derived CRC model using single cell RNA-sequencing. Genetic screens revealed that DAPK3 contributes to CRC differentiation induced by HDAC1/2 inhibition. These results highlight the importance of specific chemically targetable histone modifications in governing cancer cell states and epigenetic reprogramming as a therapeutic strategy in CRC. BRIEF SUMMARY HDAC1/2 inhibition promotes colorectal cancer differentiation via gains in H3K27ac, which can be reversed by blocking its acetyltransferase EP300.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207eb16a60f11154e81c963033e896c8385daaa7" target='_blank'>
              Chemical perturbations impacting histone acetylation govern colorectal cancer differentiation
              </a>
            </td>
          <td>
            Pornlada Likasitwatanakul, Zhixin Li, Paul Doan, S. Spisák, Akhouri Kishore Raghawan, Qi Liu, Priscilla Liow, Sunwoo Lee, David Chen, Pratyusha Bala, P. Sahgal, Daulet Aitymbayev, Jennifer S. Thalappillil, Malvina Papanastasiou, William Hawkins, Steven A. Carr, Haeseong Park, James M. Cleary, Jun Qi, Nilay S. Sethi
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Gene fusions are nucleotide sequences formed due to errors in replication and transcription control. These errors, resulting from chromosomal translocation, transcriptional errors or trans-splicing, vary from cell to cell. The identification of fusions has become critical as key biomarkers for disease diagnosis and therapy in various cancers, significantly influencing modern medicine. Chimeric Transcripts and RNA-Sequencing database version 8.0 (ChiTaRS 8.0; http://biosrv.org/chitars) is a specialized repository for human chimeric transcripts, containing 47 445 curated RNA transcripts and over 100 000 chimeric sequences in humans. This updated database provides unique information on 1055 chimeric breakpoints derived from public datasets using chromosome conformation capture techniques (the Hi-C datasets). It also includes an expanded list of gene fusions that are potential drug targets, and chimeric breakpoints across 934 cell lines, positioning ChiTaRS 8.0 as a valuable resource for testing personalized cancer therapies. By utilizing text mining on a curated selection of disease-specific RNA-sequencing data from public datasets, as well as patient blood and plasma samples, we have identified novel chimeras—particularly in diseases such as oral squamous cell carcinoma and glioblastoma—now catalogued in ChiTaRS. Thus, ChiTaRS 8.0 serves as an enhanced fusion transcript repository that incorporates insights into the functional landscape of chimeras in cancers and other complex diseases, based on liquid biopsy results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733a293484c1701290a845d632917cdf236590b" target='_blank'>
              ChiTaRS 8.0: the comprehensive database of chimeric transcripts and RNA-seq data with applications in liquid biopsy
              </a>
            </td>
          <td>
            Dylan DSouza, Lihi Bik, Olawumi Giwa, Shahaf Cohen, Hilit Levy Barazany, T. Siegal, M. Frenkel-Morgenstern
          </td>
          <td>2024-12-16</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Genetic studies of haematological cancers have pointed out the heterogeneity of leukaemia in its different subpopulations, with distinct mutations and characteristics, impacting the treatment response. Next-generation sequencing (NGS) and genome-wide analyses, as well as single-cell technologies, have offered unprecedented insights into the clonal heterogeneity within the same tumour. A key component of this heterogeneity that remains unexplored is the intracellular metabolome, a dynamic network that determines cell functions, signalling, epigenome regulation, immunity and inflammation. Understanding the metabolic diversities among cancer cells and their surrounding environments is therefore essential in unravelling the complexities of leukaemia and improving therapeutic strategies. Here, we describe the currently available methodologies and approaches to addressing the dynamic heterogeneity of leukaemia progression. In the second section, we focus on metabolic leukaemic vulnerabilities in acute myeloid leukaemia (AML) and acute lymphoblastic leukaemia (ALL). Lastly, we provide a comprehensive overview of the most interesting clinical trials designed to target these metabolic dependencies, highlighting their potential to advance therapeutic strategies in leukaemia treatment. The integration of multi-omics data for cancer identification with the metabolic states of tumour cells will enable a comprehensive “micro-to-macro” approach for the refinement of clinical practices and delivery of personalised therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0cc7b64900de2dfae5390d10959b2a1e62e8bf1" target='_blank'>
              Unlocking the Heterogeneity in Acute Leukaemia: Dissection of Clonal Architecture and Metabolic Properties for Clinical Interventions
              </a>
            </td>
          <td>
            Martina Maria Capelletti, Orsola Montini, Emilio Ruini, Sarah Tettamanti, Angela Maria Savino, J. Sarno
          </td>
          <td>2024-12-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="CAR-T therapies utilizing T cells engineered with chimeric antigen receptors (CARs) have revolutionized the treatment of hematologic and immune-related malignancies. However, the anti-tumor capability of engineered CAR-T cells are often not persistent, which greatly dampen its clinical efficacy. To address this limitation, massive parallel genetic screens were widely used to identified novel targets and regulators that enhance T cell anti-tumor capability and persistence in tumor microenvironment. We hypothesized that by combining the pooled screen data from multiple independent genetic screens we could provide a systematic, comprehensive, and robust analysis of the effect of gene perturbation on T-cell based immunotherapies. After collecting data from previously published T cell screens, including CRISPR-based and ORF-based screens, through Gene Expression Omnibus (GEO), we reprocessed the gene hits summary and conducting a pathway enrichment analysis. A T cell screen perturbation score (TPS) was employed to quantifies the impact of the gene on T cell function. Additionally, gene expression data (both bulk RNA level and single-cell RNA level) from autoimmune disease and T cell-derived cancers were analyzed to pick up gene perturbations that potentially augment T cell proliferation. We integrated all data and analysis on 27 T-cells screens into our state-of-the-art T cell perturbation genomics database (TCPGdb), which is made easily accessible through our webserver and allows users to interactively explore the impact of query genes on T cell function based on prior screen data and our TPS scoring module. TCPGdb is publicly accessible at http://tcpgdb.sidichenlab.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a099e229f2c35d38ee6b66c8422aec7d483d003c" target='_blank'>
              TCPGdb: A comprehensive T cell perturbation genomics database for identification of critical T cell regulators
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Kaiyuan Tang, Nipun Verma, Xinxin Zhu, Di Feng, James Cai, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Despite advancements in the treatment of advanced prostate cancer, including metastatic castration-resistant prostate cancer (mCRPC), the disease remains lethal, often developing resistance to conventional therapies such as androgen deprivation therapy (ADT) and chemotherapy. Therapy resistance poses a significant challenge, particularly in aggressive forms of prostate cancer. Data on TP53 mutations, especially gain-of-function (GOF) mutations, and their specific impact on prostate cancer metabolism remain limited, underscoring the need for a deeper understanding to uncover new therapeutic avenues. This study aimed to explore the metabolic consequences of TP53 GOF mutations (R175H and R273H) in prostate cancer progression using metabolomics analysis. Approximately 300 metabolites were significantly altered across PC3-GFP (KO), PC3-TP53WT (WT), and PC3-TP53 mutant (R175H & R273H) models. Key findings revealed dysregulation of essential metabolites, such as α-ketoglutarate and spermidine, which support cancer cell survival and progression. These insights not only enhance our understanding of the interplay between TP53 mutations and metabolic alterations but also identify potential biomarkers for prognosis and therapeutic targeting in therapy-resistant prostate cancer. In the TP53-R175H mutant, numerous essential metabolites were dysregulated (downregulated: glutamine, GAP/DHAP, and α-ketoglutarate; Upregulated: deoxyuridine, spermidine, and pseudouridine) when compared with the control (KO) thereby causing metabolic shifts that favor cancer cell survival. Conversely, metabolites such as GlaNAC/GlcNAC, adenosine, acetoacetic acid, acetylcholine, octopamine, itaconic acid, lactic acid, and NADP+ are all downregulated in the R273H mutant. While spermine, spermidine, N-acetyl-DL-serine, pseudouridine, deoxyguanosine, sucrose, and AMP/dGMP are upregulated by the R273H mutant but downregulated in GFP (KO). These findings showed that TP53 GOF mutations (R175H & R273H) cause metabolic alterations in numerous metabolites essential for tumor survival and progression. Furthermore, it may provide therapeutic benefits by identifying appropriate biomarkers that could be used as diagnostic and/or prognostic tools for prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6235a81e6d316a547376d1bb83acc9177dd3e97b" target='_blank'>
              Impact of TP53 Gain-of-Function Mutations on Metabolic Reprogramming in Prostate Cancer Progression
              </a>
            </td>
          <td>
            Farida Jazuli Usman, D. G. Gupta, S. Abdulkadir, Ibrahim Muhammad Usman, Adamu Jibrin Alhassan, Young A Yoo, Haris Jaafar Bello, Binta G. Kurfi, K. Babagana
          </td>
          <td>2025-01-10</td>
          <td>Asian Journal of Biochemistry, Genetics and Molecular Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="With the advancement of single-cell sequencing, we are now capable of analyzing the heterogeneity within tumors with increased precision, revealing the complex interactions between the tumor microenvironment and immune cell subgroups. Studies mentioned in the articles demonstrate significant cell heterogeneity in various cancers at genomic, transcriptomic, and proteomic levels. In particular, single-cell technologies aid in identifying cell-type-specific genetic lineage effects, for example, in the context of influenza infection and T cell surveillance. Current study focuses on the application of single-cell sequencing in cancer immunotherapy and the exploration of novel antigen identification and optimization of therapeutic strategies. The introduction of new strategies, such as the refinement of electrochemical immuno-sensors, synthesis of novel small molecule drugs, and optimized detection of genetic mutations, contributes collectively to the accurate identification of tumor markers and innovative development of therapeutic methods. The integration of treatment schemes and patient survival analysis further guides the implementation of personalized medicine. In summary, the application of single-cell sequencing and its integration with the optimization of therapeutic strategies promises revolutionary advancements in the field of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607f39c834cbc474d463e6168f60ce125e47f1f7" target='_blank'>
              Research On the Identification of Novel Antigens and The Optimization of Therapeutic Strategies in Cancer Immunotherapy Through Single-Cell Sequencing
              </a>
            </td>
          <td>
            Peixuan Li
          </td>
          <td>2024-12-24</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Patients with epidermal growth factor receptor mutant nonsmall cell lung cancer (NSCLC) often fail to treat gefitinib because of secondary drug resistance. The development of tumor drug resistance is closely related to variations in cancer cell metabolism. Single-cell metabolomics analysis can provide unique information about tumor drug resistance. Herein, we constructed a platform to study the secondary resistance of tumor cells based on single-cell metabolomics (sSRTC-scM). A gefitinib-resistant NSCLC cell line (PC9GR) was constructed by increasing the dose step by step. The metabolic profiles of parental PC9 cells and PC9GR cells with different drug resistance levels were detected by intact living-cell electrolaunching ionization mass spectrometry at the single-cell level. The data were analyzed by statistical methods such as t-SNE, variance, volcano plot, heat map, and metabolic pathway analysis. Using this platform, we found that the metabolic fingerprints of PC9GR cells can evaluate drug resistance degrees. The metabolic fingerprints continue to be altered with the increase of drug resistance. We revealed 19 metabolic markers of secondary resistance by variance analysis and clarified that the glycerophospholipid metabolic pathway of PC9GR cells changed significantly. In addition, we found that with the increase in drug resistance levels, the heterogeneity of single-cell metabolism became greater and the number of cells with weak drug resistance gradually decreased. This phenomenon can be utilized to illustrate the drug resistance degrees of PC9GR cells. This study provides diagnostic markers for evaluating the drug resistance of tumors and gives new insight into overcoming the secondary resistance of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84899e62659612e5b6164a21695dadcc8005bf4e" target='_blank'>
              Single-Cell Mass Spectrometry Studies of Secondary Drug Resistance of Tumor Cells.
              </a>
            </td>
          <td>
            Guizhen Zhu, Wenmei Zhang, Yaoyao Zhao, Guangyun Wang, Hanyu Yuan, Guangsheng Guo, Xiayan Wang
          </td>
          <td>2024-12-20</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Most tumors initially respond to treatment, yet refractory clones subsequently develop owing to resistance mechanisms associated with cancer cell plasticity and heterogeneity. Methods: We used a chemical biology approach to identify protein targets in cancer cells exhibiting diverse driver mutations and representing models of tumor lineage plasticity and therapy resistance. An unbiased screen of a drug library was performed against cancer cells followed by synthesis of chemical analogs of the most effective drug. The cancer subtype target range of the leading drug was determined by PRISM analysis of over 900 cancer cell lines at the Broad Institute, MA. RNA-sequencing and enrichment analysis of differentially expressed genes, as well as computational molecular modeling and pull-down with biotinylated small molecules were used to identify and validate RPS6KB1 (p70S6K or S6K1) as an essential target. Genetic restoration was used to test the functional role of S6K1 in cell culture and xenograft models. Results: We identified a novel derivative of the antihistamine drug ebastine, designated Super-ebastine (Super-EBS), that inhibited the viability of cancer cells representing diverse KRAS and EGFR driver mutations and models of plasticity and treatment resistance. Interestingly, PRISM analysis indicated that over 95% of the diverse cancer cell lines tested were sensitive to Super-EBS and the predicted target was the serine/threonine kinase S6K1. S6K1 is upregulated in various cancers relative to counterpart normal/benign tissues and phosphorylated-S6K1 predicts poor prognosis for cancer patients. We noted that inhibition of S6K1 phosphorylation was necessary for tumor cell growth inhibition, and restoration of phospho-S6K1 rendered tumor cells resistant to Super-EBS. Inhibition of S6K1 phosphorylation by Super-EBS induced caspase-2 dependent apoptosis via inhibition of the Cdc42/Rac-1/p-PAK1 pathway that led to actin depolymerization and caspase-2 activation. The essential role of S6K1 in the action of Super-EBS was recapitulated in xenografts, and knockout of S6K1 abrogated tumor growth in mice. Conclusion: S6K1 is a therapeutic vulnerability in tumors exhibiting intrinsic and/or acquired resistance to treatment">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e4047ca455e4b4317a09e2a2f99cd529981915d" target='_blank'>
              S6K1 is a Targetable Vulnerability in Tumors Exhibiting Plasticity and Therapy Resistance
              </a>
            </td>
          <td>
            Saptadwipa Ganguly, R. Burikhanov, Vitaliy M. Sviripa, Sally Ellingson, Jieyun Jiang, Christian M. Gosser, David Orren, Eva M. Goellner, G.G. Shenoy, Mahadev Rao, John A. D’Orazio, C. F. Brainson, Chang-Guo Zhan, Peter H. Spielmann, David S. Watt, Vivek M. Rangnekar
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that plays a crucial role in regulating gene expression and maintaining genomic stability. With the advancement in sequencing technology, methylation studies have provided valuable insights into the diagnosis of rare diseases through the various identification of episignatures, epivariation, epioutliers, and allele-specific methylation. However, current methylation studies are not without limitations. This mini-review explores the current understanding of DNA methylation in rare diseases, highlighting the key mechanisms and diagnostic potential, and emphasizing the need for advanced methodologies and integrative approaches to enhance the understanding of disease progression and design more personable treatment for patients, given the nature of rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b0846fe63347313e7e0457ade21ff242a0edeb" target='_blank'>
              Expanding Upon Genomics in Rare Diseases: Epigenomic Insights
              </a>
            </td>
          <td>
            Jia W Tan, Emily J. Blake, Joseph D. Farris, Eric W. Klee
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast Cancer (BrCa) exhibits a high phenotypic heterogeneity, leading to the emergence of aggressive clones and the development of drug resistance. Considering the BrCa heterogeneity and that metabolic reprogramming is a cancer hallmark, we selected seven BrCa cell lines with diverse subtypes to provide their comprehensive metabolome characterization: five lines commonly used (SK-Br-3, T-47D, MCF-7, MDA-MB-436, and MDA-MB-231), and two patient-derived xenografts (Hbcx39 and Hbcx9). We characterized their endometabolomes using 1H-NMR spectroscopy. We found distinct metabolite profiles, with certain metabolites being common but differentially accumulated across the selected BrCa cell lines. High levels of glycine, lactate, glutamate, and formate, metabolites known to promote invasion and metastasis, were detected in all BrCa cells. In our experiment setting were identified unique metabolites to specific cell lines: xanthine and 2-oxoglutarate in SK-Br-3, 2-oxobutyrate in T-47D, cystathionine and glucose-1-phosphate in MCF-7, NAD+ in MDA-MB-436, isocitrate in MDA-MB-231, and NADP+ in Hbcx9. The unique and enriched metabolites enabled us to identify the metabolic pathways modulated in a cell-line-specific manner, which may represent potential candidate targets for therapeutic intervention. We believe this study may contribute to the functional characterization of BrCa cells and assist in selecting appropriate cell lines for drug-response studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4c21e05117b322ca21c171873276565e0f755d" target='_blank'>
              Metabolic Profiling of Breast Cancer Cell Lines: Unique and Shared Metabolites
              </a>
            </td>
          <td>
            Mariana Gallo, Elena Ferrari, F. Brugnoli, Anna Terrazzan, Pietro Ancona, Stefano Volinia, V. Bertagnolo, Carlo M. Bergamini, A. Spisni, T. Pertinhez, Nicoletta Bianchi
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The MCF-7R breast cancer cell line, developed by treating the parental MCF-7 cells with increasing doses of doxorubicin, serves as a model for studying acquired multidrug resistance (MDR). MDR is a major challenge in cancer therapy, often driven by overexpression of the efflux pump P-glycoprotein (P-gp) and epigenetic modifications. While many P-gp inhibitors show promise in vitro, their nonspecific effects on the efflux pump limit in vivo application. Curcumin, a natural compound with pleiotropic action, is a nontoxic P-gp inhibitor capable of modulating multiple pathways. To explore curcumin’s molecular effects on MCF-7R cells, we analyzed the expression of genes involved in DNA methylation and transcription regulation, including ABCB1/MDR1. Reduced representation bisulfite sequencing further unveiled key epigenetic changes induced by curcumin. Our findings indicate that curcumin treatment not only modulates critical cellular processes, such as ribosome biogenesis and cytoskeletal dynamics, but also reverses the resistant phenotype, toward that of sensitive cells. This study highlights curcumin’s potential as an adjuvant therapy to overcome chemoresistance, offering new avenues for pharmacological strategies targeting epigenetic regulation to re-sensitize resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a25fe3bb8a31c48e3c18c599af20c640c2363916" target='_blank'>
              Epigenetic and Cellular Reprogramming of Doxorubicin-Resistant MCF-7 Cells Treated with Curcumin
              </a>
            </td>
          <td>
            P. Poma, S. Rigogliuso, M. Labbozzetta, Aldo Nicosia, Salvatore Costa, M. Ragusa, M. Notarbartolo
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Although somatic mutations are fundamentally important to human biology, disease, and aging, many outstanding questions remain about their rates, spectrum, and determinants in apparently healthy tissues. Here, we performed high-coverage exome sequencing on 265 samples from 14 GTEx donors sampled for a median of 17.5 tissues per donor (spanning 46 total tissues). Using a novel probabilistic method tailored to the unique structure of our data, we identified 8,470 somatic variants. We leverage our compendium of somatic mutations to quantify the burden of deleterious somatic variants among tissues and individuals, identify molecular features such as chromatin accessibility that exhibit significantly elevated somatic mutation rates, provide novel biological insights into mutational mechanisms, and infer developmental trajectories based on patterns of multi-tissue somatic mosaicism. Our data provides a high-resolution portrait of somatic mutations across genes, tissues, and individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93698e2f53feff8e8bfbf3cca400ebd477df43e0" target='_blank'>
              Landscape of human protein-coding somatic mutations across tissues and individuals
              </a>
            </td>
          <td>
            Huixin Xu, Rob Bierman, Dayna Akey, Cooper Koers, Troy Comi, Claire McWhite, Joshua M. Akey
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapy resistance still constitutes a common hurdle in the treatment of many human cancers and is a major reason for treatment failure and patient relapse, concomitantly with a dismal prognosis. In addition to “intrinsic resistance”, e.g., acquired by random mutations, cancer cells typically escape from certain treatments (“acquired resistance”) by a large variety of means, including suppression of apoptosis and other cell death pathways via upregulation of anti-apoptotic factors or through inhibition of tumor-suppressive proteins. Therefore, ideally, the tumor-cell-restricted induction of apoptosis is still considered a promising avenue for the development of novel, tumor (re)sensitizing therapies. A growing body of evidence has highlighted the multifaceted role of tripartite motif 25 (TRIM25) in controlling different aspects of tumorigenesis, including chemotherapeutic drug resistance. Accordingly, overexpression of TRIM25 is observed in many tumors and frequently correlates with a poor patient survival. In addition to its originally described function in antiviral innate immune response, TRIM25 can play critical yet context-dependent roles in apoptotic- and non-apoptotic-regulated cell death pathways, including pyroposis, necroptosis, ferroptosis, and autophagy. The review summarizes current knowledge of molecular mechanisms by which TRIM25 can interfere with different cell death modalities and thereby affect the success of currently used chemotherapeutics. A better understanding of the complex repertoire of cell death modulatory effects by TRIM25 is an essential prerequisite for validating TRIM25 as a potential target for future anticancer therapy to surmount the high failure rate of currently used chemotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb1e40b262aff1ec5083769b832df042944b012" target='_blank'>
              TRIM25: A Global Player of Cell Death Pathways and Promising Target of Tumor-Sensitizing Therapies
              </a>
            </td>
          <td>
            W. Eberhardt, Usman Nasrullah, Josef Pfeilschifter
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d406237f2977fc66d322658b801ab000ef5018c5" target='_blank'>
              KEAP1 mutations as key crucial prognostic biomarkers for resistance to KRAS-G12C inhibitors
              </a>
            </td>
          <td>
            Linyan Tian, Chengming Liu, Sufei Zheng, Huiyang Shi, Fang Wei, Wenxin Jiang, Yucheng Dong, Haiyan Xu, Enzhi Yin, N. Sun, Jie He, Yan Wang
          </td>
          <td>2025-01-17</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Resistance to BRAF and MAPK inhibitors is a significant challenge in melanoma treatment, driven by adaptive and acquired mechanisms that allow tumour cells to evade therapy. Here, we examined early signalling responses to single and combined BRAF and MAPK inhibition in a BRAFV600E, drug-sensitive melanoma cell line and a drug-resistant ARID1A-knockout (KO) derivative. ARID1A, frequently mutated in melanoma, is associated with resistance and immune evasion. Using an innovative systems biology approach that integrates transcriptomics, proteomics, phosphoproteomics, and functional kinomics through matrix factorization and network analysis, we identified key signalling alterations and resistance mechanisms. We found that ARID1A-KO cells exhibited transcriptional rewiring, sustaining MAPK1/3 and JNK activity post-treatment, bypassing feedback sensitivity observed in parental cells. This rewiring suppressed PRKD1 activation, increased JUN activity—a central resistance network node—and disrupted PKC dynamics through elevated basal RTKs (e.g., EGFR, ROS1) and Ephrin receptor activity post-treatment. ARID1A mutations also reduced HLA-related protein expression and enriched extracellular matrix components, potentially limiting immune infiltration and reducing immunotherapy efficacy. Our graph-theoretical multi-omics approach uncovered novel resistance-associated signalling pathways, identifying PRKD1, JUN, and NCK1 as critical nodes. While receptor activation redundancies complicate single-target therapies, they also present opportunities for combination strategies. This study highlights ARID1A’s role in reshaping signalling and immune interactions, offering new insights into melanoma resistance mechanisms. By identifying actionable targets, including JUN and immune pathways, we provide a foundation for developing integrated therapeutic strategies to overcome resistance in BRAF/MAPK inhibitor-treated melanoma. One sentence summary This study reveals how ARID1A-mediated transcriptional rewiring drives resistance to MAPK inhibitors in melanoma by altering signalling pathways, immune interactions, and receptor dynamics, highlighting potential targets for combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bdb990bb1257653b7e587028f560dd7ef66b129" target='_blank'>
              ARID1A-induced transcriptional reprogramming rewires signalling responses to drug treatment in melanoma
              </a>
            </td>
          <td>
            Charlie George Barker, Sumana Sharma, Ana Mafalda Santos, Konstantinos-Stylianos Nikolakopoulos, Athanassios D. Velentzas, Franziska Völlmy, A. Minia, V. Pliaka, M. Altelaar, Gavin J Wright, Leonidas G. Alexopoulos, D. Stravopodis, E. Petsalaki
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33b574995e584f7a063a425a60100a1abcb82e" target='_blank'>
              Identification of a novel TOP2B::AFF2 fusion gene in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tao Liu, Yang Wang, Xi-Zhou An, Jiaqi Liu, Yuqin Wu, Yan Xiang, Yong-Jie Zhang, Lan Huang, Jia-Cheng Li, Yu-Zhuo-Pu Li, Jie Yu
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a602005403e19b0fe4fd439cfa9748d3ee799d93" target='_blank'>
              Identification of cancer cell-intrinsic biomarkers associated with tumor progression and characterization of SFTA3 as a tumor suppressor in lung adenocarcinomas
              </a>
            </td>
          <td>
            Yu Zhao, Chengcheng Zhou, Ling Zuo, Haoming Yan, Yuhan Gu, Hong Liu, Guiping Yu, Xiaorong Zhou
          </td>
          <td>2025-01-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e3834da21049e516e26734af02f73ef17c4698c" target='_blank'>
              Single cell RNA sequencing improves the next generation of approaches to AML treatment: challenges and perspectives
              </a>
            </td>
          <td>
            Zahra Khosroabadi, Samaneh Azaryar, Hassan Dianat-Moghadam, Zohreh Amoozgar, Mohammadreza Sharifi
          </td>
          <td>2025-01-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Stemness-associated cell states are linked to chemotherapy resistance in AML. We uncovered a direct mechanistic link between expression of the stem cell transcription factor GATA2 and drug resistance. The GATA-binding protein 2 (GATA2) plays a central role in blood stem cell generation and maintenance. We find substantial intra- and inter-patient variability in GATA2 expression across AML patient samples. GATA2 expression varies by molecular subtype and has been linked to outcome. In a murine model, KMT2A-MLL3 driven AML originating from a stem cell or immature progenitor cell population have higher Gata2 expression and are more resistant to the standard AML chemotherapy agent doxorubicin. Deletion of Gata2 resulted in more robust induction of p53 following exposure to doxorubicin. ChIP-Seq, RNA-Seq and functional studies revealed that GATA2 regulates the expression of RASSF4, a modulator of the p53 inhibitor MDM2. GATA2 and RASSF4 are anti-correlated in human cell lines and AML patient cell bulk and single cell expression datasets. Knockdown of Rassf4 in Gata2 low cells resulted in doxorubicin or nutlin-3 resistance. Conversely, overexpression of Rassf4 results in sensitization of cells expressing high levels of Gata2. Finally, doxorubicin and nutlin-3 are synergistic in Gata2-high murine AML, as well as AML patient samples. We discovered a previously unappreciated role for GATA2 in dampening p53-mediated apoptosis via transcriptional regulation of RASSF4, a modulator of MDM2. This role for GATA2 directly links the expression of a stemness associated transcription factor to chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149e4d0de77d706d929171b654f103344701ea6a" target='_blank'>
              GATA2 links stemness to chemotherapy resistance in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Fatemeh Alikarami, H. Xie, Simone S. Riedel, Haley Goodrow, D. Barrett, Leila Mahdavi, Alexandra Lenard, Changya Chen, T. Yamauchi, Etienne Danis, Zhendong Cao, Vu L. Tran, M. Jung, Yapeng Li, Hua Huang, Junwei Shi, Kai Tan, D. Teachey, E. Bresnick, Tobias Neff, Kathrin M Bernt
          </td>
          <td>2025-01-22</td>
          <td>Blood</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The integration of genetic engineering into pharmacological research is transforming the landscape of precision medicine, offering the potential to revolutionize healthcare. Precision medicine focuses on tailoring treatments to individuals based on genetic, environmental, and lifestyle factors, moving beyond the traditional “one-size-fits-all” approach. This editorial highlights the pivotal role of genetic engineering in advancing precision medicine, exploring its applications in drug discovery, pharmacogenomics, and innovative therapies such as gene and cell treatments. Tools like CRISPR-Cas9 and advancements in omics technologies have accelerated the development of personalized therapies and enhanced our understanding of disease mechanisms. Despite these breakthroughs, challenges persist. Technical hurdles like off-target effects, ethical concerns surrounding germline editing, and the high costs of these technologies must be addressed. Opportunities, however, abound—ranging from artificial intelligence-driven innovations to collaborative research initiatives, which promise to streamline the development of genetic engineering applications. This convergence of disciplines is not only reshaping drug development but also expanding the therapeutic arsenal to tackle complex diseases. With sustained investment, interdisciplinary collaboration, and an emphasis on equitable access, the promise of precision medicine—treatments tailored to each individual’s genetic blueprint—is rapidly becoming a reality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1aafe7cc772e95c4f6fdbb53558e414c2fc61c" target='_blank'>
              The future of precision medicine: Integrating genetic engineering into pharmacological research
              </a>
            </td>
          <td>
            Md. Mosharraf Hossen
          </td>
          <td>2025-01-26</td>
          <td>Journal of Pharmacology and Medicine Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this review, we explore the application of next-generation sequencing in liver cancer research, highlighting its potential in modern oncology. Liver cancer, particularly hepatocellular carcinoma, is driven by a complex interplay of genetic, epigenetic, and environmental factors. Key genetic alterations, such as mutations in TERT, TP53, and CTNNB1, alongside epigenetic modifications such as DNA methylation and histone remodeling, disrupt regulatory pathways and promote tumorigenesis. Environmental factors, including viral infections, alcohol consumption, and metabolic disorders such as nonalcoholic fatty liver disease, enhance hepatocarcinogenesis. The tumor microenvironment plays a pivotal role in liver cancer progression and therapy resistance, with immune cell infiltration, fibrosis, and angiogenesis supporting cancer cell survival. Advances in immune checkpoint inhibitors and chimeric antigen receptor T-cell therapies have shown potential, but the unique immunosuppressive milieu in liver cancer presents challenges. Dysregulation in pathways such as Wnt/β-catenin underscores the need for targeted therapeutic strategies. Next-generation sequencing is accelerating the identification of genetic and epigenetic alterations, enabling more precise diagnosis and personalized treatment plans. A deeper understanding of these molecular mechanisms is essential for advancing early detection and developing effective therapies against liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf941fd49f418f7109e7e4bbca5152a8ddaa54e7" target='_blank'>
              Advances and challenges in molecular understanding, early detection, and targeted treatment of liver cancer
              </a>
            </td>
          <td>
            Ji Shi, Xu Zhu, Jun-Bo Yang
          </td>
          <td>2025-01-27</td>
          <td>World Journal of Hepatology</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/070c22030bfb86f8cf3bc0c2063075e7b4258116" target='_blank'>
              Identification of the clinical and genetic characteristics of gliomas with gene fusions by integrated genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            Guo-zhong Yi, Hua-yang Zhang, Tian-shi Que, Shan-qiang Qu, Zhi-yong Li, Song-tao Qi, Wen-yan Feng, Guang-long Huang
          </td>
          <td>2025-01-23</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58506d08da7a5b25a950b60c249aaba7b6397bbd" target='_blank'>
              Neutral drift upon threshold-like selection promotes variation in antibiotic resistance phenotype
              </a>
            </td>
          <td>
            A. N. Erdoğan, P. Dasmeh, Raymond D. Socha, Johnson Chen, Ben Life, Rachel Jun, Linda Kiritchkov, Dan Kehila, Adrian W. R. Serohijos, N. Tokuriki
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfaace030e74345b79142b5ae90516ba4841735" target='_blank'>
              High-throughput sequencing: a breakthrough in molecular diagnosis for precision medicine.
              </a>
            </td>
          <td>
            Dipali Dongare, Shaik Shireen Nishad, Sakshi Y Mastoli, S. Saraf, Nidhi Srivastava, Abhishek Dey
          </td>
          <td>2025-01-22</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae5f6bb3e804b17358c04ea87f466f1ee8583c6" target='_blank'>
              Shedding light on DNA methylation and its clinical implications: the impact of long-read-based nanopore technology
              </a>
            </td>
          <td>
            Alexandra Chera, Mircea Stancu-Cretu, N. Zabet, Octavian Bucur
          </td>
          <td>2024-12-30</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Objectives Unlike other diseases, cancer is not just a genome disease but should broadly be viewed as a disease of the cellular machinery. Therefore, integrative multifaceted approaches are crucial to understanding the complex nature of cancer biology. Bcl-2 (B-cell lymphoma 2), encoded by the human BCL-2 gene, is an anti-apoptotic molecule that plays a key role in apoptosis and genetic variation of Bcl-2 proteins and is vital in disrupting the apoptotic machinery. Single nucleotide polymorphisms (SNPs) are considered viable diagnostic and therapeutic biomarkers for various cancers. Therefore, this study explores the association between SNPs in Bcl-2 and the structural, functional, protein-protein interactions (PPIs), drug binding and dynamic characteristics. Methods Comprehensive cross-validated bioinformatics tools and molecular dynamics (MD) simulations. Multiple sequence, genetic, structural and disease phenotype analyses were applied in this study. Results Analysis revealed that out of 130 mutations, approximately 8.5% of these mutations were classified as pathogenic. Furthermore, two particular variants, namely, Bcl-2G101V and Bcl-2F104L, were found to be the most deleterious across all analyses. Following 500 ns, MD simulations showed that these mutations caused a significant distortion in the protein conformational, protein-protein interactions (PPIs), and drug binding landscape compared to Bcl-2WT. Conclusion Despite being a predictive study, the findings presented in this report would offer a perspective insight for further experimental investigation, rational drug design, and cancer gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f67dee38c9954b65ace573045ef349b37771a6" target='_blank'>
              Integrative genomic analyses combined with molecular dynamics simulations reveal the impact of deleterious mutations of Bcl-2 gene on the apoptotic machinery and implications in carcinogenesis
              </a>
            </td>
          <td>
            G. Elamin, Zhichao Zhang, Depika Dwarka, K. Kasumbwe, John Mellem, Nompumelelo P. Mkhwanazi, P. Madlala, Mahmoud E. S. Soliman
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background In the last few years, there has been increasing evidence in the literature for promoter hypermethylation of various host genes following infection with various pathogens. Therefore, in the present study, the role of epigenetic reprogramming of host cells through DNA methylation was evaluated which indicates that the disease progression takes place through changes in host immune response related genes. Objective To elucidate the differential genome-wide DNA methylation profile of peripheral blood mononuclear cells in pulmonary tuberculosis (PTB) patients. Materials and methods To decipher the methylation profile of PBMCs in PTB patients whole genome bisulphite sequencing was performed using 4 DNA samples from each study group i.e. PTB, healthy controls and diseased controls. Briefly, the samples were subjected to bisulphite conversion followed by library preparation and whole genome bisulphite sequencing. The data then were analysed for differential methylation analysis followed by gene set enrichment analysis to select DMRs for validation by sanger sequencing. Further the GEOR2 online server was used to investigate the mRNA expression of selected genes associated with hypermethylated DMRs in promoter region. Results Differential methylation region analysis was performed for the following comparisons. TB v/s Diseased (Hypermethylated=1756; Hypomethylated=1886), TB v/s Healthy( Hypermethylated=2203; Hypomethylated=2466), Diseased v/s Healthy ( Hypermethylated=1656; Hypomethylated=3936). Further gene set enrichment analysis showed that the hypermetylated regions belonged to genes that are involved in immune responses mainly T cell functioning and T cell mediated immune processes. There were 8 DMR belonging to TAF8,FZD5, HLA-DRB, MIR483, PVRIG, SH2B2, ZAP70, TNFRSF13C simultaneously (n=10 from each study group). mRNA expression analysis with GEOR2 (Datasets used n= 12) revealed significant downregulation of TNFRSF13C, ZAP70, PVRIG Among the selected genes associated with hypermethylated DMRs in promoter region. (e) Conclusion The altered differential methylation profile of PBMCs from TB patients shows that on onset the active disease there occurs aberrant methylation of expression regulating regions that in turn results in decreased T cell functions and hence suboptimal immune responses in TB">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d347b75a8073b0a8b0b0587d9118035bcdf0c73f" target='_blank'>
              Epigenetic Modulation of Host Immunity related genes in Pulmonary Tuberculosis: A Comprehensive Analysis of DNA Methylation Profiles in Peripheral Blood Mononuclear Cells
              </a>
            </td>
          <td>
            Ankit Kumar, Sumedha Sharma, Jyotdeep Kaur, Arnab Pal, A. N. Aggarwal, I. Verma
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5a9f2f7d6e562bbd4a865f9dd5dc95d52e44f1" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics reveals resistance mechanisms in breast cancer
              </a>
            </td>
          <td>
            Nathan D Maulding, Jun Zou, Wei Zhou, C. Metcalfe, Josh Stuart, Xin Ye, M. Hafner
          </td>
          <td>2025-01-10</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Oncogenes are typically overexpressed in tumor tissues and often linked to poor prognosis. However, recent advancements in bioinformatics have revealed that many highly expressed genes in tumors are associated with better patient outcomes. These genes, which act as tumor suppressors, are referred to as “paradoxical genes.” Analyzing The Cancer Genome Atlas (TCGA) confirmed the widespread presence of paradoxical genes, and KEGG analysis revealed their role in regulating tumor metabolism. Mechanistically, discrepancies between gene and protein expression-affected by pre- and post-transcriptional modifications-may drive this phenomenon. Mechanisms like upstream open reading frames and alternative splicing contribute to these inconsistencies. Many paradoxical genes modulate the tumor immune microenvironment, exerting tumor-suppressive effects. Further analysis shows that the stage- and tumor-specific expression of these genes, along with their environmental sensitivity, influence their dual roles in various signaling pathways. These findings highlight the importance of paradoxical genes in resisting tumor progression and maintaining cellular homeostasis, offering new avenues for targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab8741264f274ad63584e4f394278cc1295e34" target='_blank'>
              Discovery of paradoxical genes: reevaluating the prognostic impact of overexpressed genes in cancer
              </a>
            </td>
          <td>
            Dequan Liu, Lei Liu, Xiangyu Che, Guangzhen Wu
          </td>
          <td>2025-01-22</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da415d9954dd04158cfcfaa83dc1880ca38c0523" target='_blank'>
              Single-cell and spatial transcriptomics reveal SPP1-CD44 signaling drives primary resistance to immune checkpoint inhibitors in RCC
              </a>
            </td>
          <td>
            Junfeng Zhang, Qingyan Peng, Jin Fan, Fuzhong Liu, Hongbo Chen, X. Bi, Shuai Yuan, Wei Jiang, Ting Pan, Kailing Li, Sihai Tan, Peng Chen
          </td>
          <td>2024-12-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Human cancers are heterogeneous. Their genomes evolve from genetically diverse germlines in complex and dynamic environments, including exposure to potential carcinogens. This heterogeneity of humans, our environmental exposures, and subsequent tumours makes it challenging to understand the extent to which cancer evolution is predictable. Addressing this limitation, we re-ran early tumour evolution hundreds of times in diverse, inbred mouse strains, capturing genetic variation comparable to and beyond that found in human populations. The sex, environment, and carcinogenic exposures were all controlled and tumours comprehensively profiled with whole genome and transcriptome sequencing. Within a strain, there was a high degree of consistency in the mutational landscape, a limited range of driver mutations, and all strains converged on the acquisition of a MAPK activating mutation with similar transcriptional disruption of that pathway. Despite these similarities in the phenotypic state of tumours, different strains took markedly divergent paths to reach that state. This included pronounced biases in the precise driver mutations, the strain specific occurrence of whole genome duplication, and differences in subclonal selection that reflected both cancer susceptibility and tumour growth rate. These results show that interactions between the germline genome and the environment are highly deterministic for the trajectory of tumour genome evolution, and even modest genetic divergence can substantially alter selection pressures during cancer development, influencing both cancer risk and the biology of the tumour that develops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caebf07ac0a60cd920fb19088ba70b1c3f57d1ba" target='_blank'>
              Genetic background sets the trajectory of cancer evolution
              </a>
            </td>
          <td>
            Sarah J. Aitken, Frances Connor, Christine Feig, Tim F. Rayner, M. Lukk, Juliet Luft, Stuart Aitken, Claudia Arnedo-Pac, James F. Hayes, Michael D. Nicholson, Vasavi Sundaram, Jan C. Verburg, John Connelly, Craig J. Anderson, Mikaela Behm, Susan Campbell, Maëlle Daunesse, Ailith Ewing, Vera B. Kaiser, Elissavet Kentepozidou, O. Pich, A. Redmond, Javier Santoyo-Lopez, Inés Sentís, Lana Talmane, Paul Flicek, Núria López-Bigas, Colin A. Semple, Martin S. Taylor, D. T. Odom
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Abstract High-risk multiple myeloma is genomically unstable, comprising heterogeneous populations of tumor cells that evolve over time. Light chain escape (LCE) is a clinical phenomenon observed when light chains rise separately from M-spike values, which implies divergent tumor evolution. We sought to understand LCE by performing high-depth transcriptomic and phenotypic studies. We performed single-cell RNA-sequencing (scRNA-seq) and ex vivo drug sensitivity profiling on serial bone marrow biopsies from a patient with LCE at diagnosis, first relapse, and relapsed/refractory timepoints. scRNA-seq revealed distinct transcriptomic subpopulations with phenotypes that could be tracked separately by clinical serum light chain and M-spike values. Genes differentially expressed between subpopulations were assessed for generalizable effects on prognosis from the Multiple Myeloma Research Foundation CoMMpass and GSE24080 datasets. Notably, the LCE subpopulation exhibited gene expression profile featuring prominent LAMP5 overexpression, which was associated with risk for osteolytic bone lesions. Ex vivo drug sensitivity testing displayed differential sensitivity of the subpopulations. Copy number variant inference showed that the transcriptomic subpopulation underlying LCE was related to a genetic subclone that evolved over time. Our findings illustrate that malignant subpopulations underly LCE in multiple myeloma. These studies imply that LCE and LAMP5 gene overexpression portends for increased risk of osteolytic bone disease and adverse prognosis, findings that were confirmed in the subset of patients from the CoMMpass database with LCE. Significance: scRNA-seq was used to study a patient with high-risk multiple myeloma featuring LCE. LCE was rooted in a transcriptomic subpopulation that corresponded to a genetic subclone and established novel links between LCE and LAMP5 overexpression to osteolysis and prognosis, validated in RNA-seq databases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e33173187a2b1d3d0f9422813ed4bc1d3b6c7d8" target='_blank'>
              Single-Cell RNA Sequencing before and after Light Chain Escape Reveals Intrapatient Multiple Myeloma Subpopulations with Divergent Osteolytic Gene Expression
              </a>
            </td>
          <td>
            Denis Ohlstrom, Z. Walker, Abhishek Pandey, Lorraine N Davis, Krysta Engel, Zenggang Pan, Peter A Forsberg, T. Mark, Austin E. Gillen, Daniel W. Sherbenou
          </td>
          <td>2024-12-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract Cancer research has historically focused on the somatic mutation theory, viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information-processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking cancer evolution: from genetic mutations to complex information systems in tumor reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Cancer remains a significant global health challenge due to its genetic complexity and the diverse mutations, epigenetic modifications, and dysregulated signaling pathways involved. This complexity complicates effective treatment strategies. The CRISPR system, particularly CRISPR - Cas13, presents a promising therapeutic tool, offering precise RNA targeting with high accuracy and sensitivity. This paper examines the potential of Cas 13 in cancer therapy, focusing on four key areas: the mechanism of Cas13, Cas13  applications in cancer treatment, challenges and limitations. The results of the study showed that CRISPR-Cas13 can not only be used as a potential cancer diagnostic tool, but also regulate abnormal mRNA mutations and reduce cancer risk. By knocking down oncogenic lncRNA, CRISPR-Cas13 can inhibit the proliferation and migration of cancer cells. CRISPR-Cas13 has the potential to reduce cancer incidence and reduce cancer mortality through early diagnosis and intervention. In the future, technical optimization to address the off-target effects of Cas13 will be the key to its application in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64e7fa238661b59668d9ff9e403d41b35f3f94d6" target='_blank'>
              The Potential and Future Perspectives of CRISPR-Cas 13 Technology in Cancer Therapy Application
              </a>
            </td>
          <td>
            Yuhe Chen
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
The majority of patients with prostate cancer (PCa) exhibit intrinsic resistance to immune checkpoint blockade (ICB) following radiotherapy (RT). This resistance is generally attributed to the limited antigen presentation of heterogeneous cells within tumors. Here, we aimed to isolate and characterize these diverse subgroups of tumor post-RT to understand the molecular mechanisms of their resistance to ICB.


METHODS
Single-cell RNA-sequencing (scRNA-seq) was used to profile senescent cancer cell clusters induced by RT in LNCaP cells. The expression and phosphorylation levels of ataxia telangiectasia and Rad3-related protein (ATR) were assessed by immunohistochemistry in clinical samples from patients with or without RT. Co-immunoprecipitation, mutagenesis, and Western blotting were used to measure the interactions between proteins. Xenograft experiments were performed to assess the tumor immune response in the mice.


RESULTS
We identified a subset of PCa cells that exhibited resistance to RT, characterized by a reduced antigen presentation capability, which enhanced their ability to evade immune detection and resist cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) blockade. scRNA-seq revealed that the senescent state was a transient phase of PCa cells post-RT, particularly in CTLA-4 blockade treatment-resistant cells. This state was marked by increased cytosolic ATR level. Cytosolic ATR phosphorylated CD86 in its cytosolic domain and enhanced the interaction between CD86 and its E3 ligase MARCH1 through electrostatic attraction. Depletion or inhibition of Atr increased the sensitivity to immune attack and improved responses to anti-Ctla-4 antibody treatment in a mouse model.


CONCLUSIONS
Our findings indicate that the activation of cytosolic ATR, which is associated with cellular senescence, impedes the effectiveness of combined RT and ICB treatments. This discovery may provide valuable insights for improving the efficacy of combined RT and ICB therapies in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54111804377a9c0bbc1679bc34e9106d5b8ef9e5" target='_blank'>
              Radiotherapy-resistant prostate cancer cells escape immune checkpoint blockade through the senescence-related ataxia telangiectasia and Rad3-related protein.
              </a>
            </td>
          <td>
            Chenyi Shao, Yingyi Zhang, Hang Li, Jiajia Chen, Ting Huang, Jiaze Li, Simeng Wen, Sen Wang, Saijun Fan, Yu Zhao
          </td>
          <td>2024-12-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive malignancy that poses significant challenges due to high rates of relapse and resistance to treatment, particularly in older populations. While therapeutic advances have been made, survival outcomes remain suboptimal. The evolution of DNA and RNA sequencing technologies, including whole-genome sequencing (WGS), whole-exome sequencing (WES), and RNA sequencing (RNA-Seq), has significantly enhanced our understanding of AML at the molecular level. These technologies have led to the discovery of driver mutations and transcriptomic alterations critical for improving diagnosis, prognosis, and personalized therapy development. Furthermore, single-cell RNA sequencing (scRNA-Seq) has uncovered rare subpopulations of leukemia stem cells (LSCs) contributing to disease progression and relapse. However, widespread clinical integration of these tools remains limited by costs, data complexity, and ethical challenges. This review explores recent advancements in DNA/RNA sequencing in AML and highlights both the potential and limitations of these techniques in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e5d43f554c8fe98964b8130a7d1804ae561d5a" target='_blank'>
              Advances in DNA/RNA Sequencing and Their Applications in Acute Myeloid Leukemia (AML)
              </a>
            </td>
          <td>
            Fatimah Ahmed, Jiang Zhong
          </td>
          <td>2024-12-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is a leading cause of cancer-related deaths globally. The heterogeneity of the tumor microenvironment significantly influences patient prognosis, while the diversity of tumor cells shapes its unique characteristics. A comprehensive analysis of the molecular profile of tumor cells is crucial for identifying novel molecular targets for drug sensitivity analysis and for uncovering the pathophysiological mechanisms underlying CRC. Methods We utilized single-cell RNA sequencing technology to analyze 13 tissue samples from 4 CRC patients, identifying key cell types within the tumor microenvironment. Intercellular communication was assessed using CellChat, and a risk score model was developed based on eight prognostic genes to enhance patient stratification for immunotherapeutic approaches. Additionally, in vitro experiments were performed on DLX2, a gene strongly associated with poor prognosis, to validate its potential role as a therapeutic target in CRC progression. Results Eight major cell types were identified across the tissue samples. Within the tumor cell population, seven distinct subtypes were recognized, with the C0 FXYD5+ tumor cells subtype being significantly linked to cancer progression and poor prognosis. CellChat analysis indicated extensive communication among tumor cells, fibroblasts, and immune cells, underscoring the complexity of the tumor microenvironment. The risk score model demonstrated high accuracy in predicting 1-, 3-, and 5-year survival rates in CRC patients. Enrichment analysis revealed that the C0 FXYD5+ tumor cell subtype exhibited increased energy metabolism, protein synthesis, and oxidative phosphorylation, contributing to its aggressive behavior. In vitro experiments confirmed DLX2 as a critical gene associated with poor prognosis, suggesting its viability as a target for improving drug sensitivity. Conclusion In summary, this study advances our understanding of CRC progression by identifying critical tumor subtypes, molecular pathways, and prognostic markers that can inform innovative strategies for predicting and enhancing drug sensitivity. These findings hold promise for optimizing immunotherapeutic approaches and developing new targeted therapies, ultimately aiming to improve patient outcomes in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8647de1e0192ab64820f770599e15f3edf4689c3" target='_blank'>
              Innovative strategies to optimise colorectal cancer immunotherapy through molecular mechanism insights
              </a>
            </td>
          <td>
            Quanjun Lin, Zhiqiang Wang, Jue Wang, Ming Xu, Xinyi Zhang, Peng Sun, Yihang Yuan
          </td>
          <td>2024-12-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Purpose Papillary renal cell carcinoma (pRCC), the second most common subtype of renal cancer, exhibits heterogeneity in molecular features and response to targeted therapies, including antiangiogenic drugs. Discovering molecular biomarkers able to stratify pRCC patients into clinically relevant subgroups and understanding the underlying mechanisms are urgently needed to advance precision medicine. Here, we molecularly dissect a large pRCC series through the expression of the targets of hypoxia inducible factors (HIF), master regulators of angiogenesis, metabolic reprogramming and immune microenvironment. Experimental Design We merged and analyzed multi-omic and clinical data of 346 patients derived from two pRCC series (TCGA-KIRP and a Spanish metastatic series). Altered pathways and differences in tumor microenvironment were identified through tumor transcriptomic analyses. Tumor metabolome analysis was performed in selected cases. Results Molecular revision of driver mutations classified 302 patients as pRCC, while uncovering misclassified cases. Analysis of HIF targets gene expression identified a subset of pRCC tumors with increased HIF activity (31%; “HIF-active”). These tumors were characterized by high hypoxia scores, increased angiogenesis, low expression of Krebs cycle genes and mitochondrial activity, high immune infiltration and increased epithelial-mesenchymal transition (P<0.0001; each feature) and they were associated with increased metastasis risk and worse overall survival (P<0.005). Metabolome analyses revealed that HIF-active tumors accumulated L-2-hydroxyglutarate (L-2-HG), an oncometabolite that leads to pseudohypoxia by inhibiting HIF prolyl hydroxylases and preventing HIFα degradation. L-2-HG-accumulation agreed with diminished L2HGDH expression, associated with losing one copy of the gene and low expression of PPARGC1A, which regulates its transcription. Conclusions HIF-pathway overactivation defines a pseudohypoxic pRCC molecular subgroup characterized by high angiogenesis, immune infiltration and aggressive features. These findings advance our understanding of pRCC diversity, revealing potential clues for the variable response of patients to targeted therapies and paving the way to more personalized pRCC treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b17c0ff397a2a18624c6810af2df85c7a525f01" target='_blank'>
              Identification of a novel papillary renal cell carcinoma molecular subtype characterized by HIF-pathway over-activation
              </a>
            </td>
          <td>
            Javier de Nicolás-Hernández, Alicia Arenas, Carlos Valdivia, María Santos, Javier Lanillos, Nicole Bechmann, M. Peitzsch, Eduardo Gil-Vilariño, María Monteagudo, Sara Mellid, C. Montero‐Conde, A. Cascón, L. Leandro-García, G. de Velasco, Ignacio Durán, Javier Puente, J. García-Donás, Eduardo Caleiras, Mercedes Robledo, Cristina Rodríguez-Antona
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5889adf5def7b284cf9f7d22530eab68401de07f" target='_blank'>
              PlexinD1 is a driver and a therapeutic target in advanced prostate cancer.
              </a>
            </td>
          <td>
            Jing Wei, Jing Wang, Wen Guan, Jingjing Li, Tianjie Pu, E. Corey, Tzu-Ping Lin, Allen C Gao, Boyang Jason Wu
          </td>
          <td>2025-01-02</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most prevalent primary malignant brain tumor, remains challenging to treat due to extensive inter- and intra-tumor heterogeneity. This variability demands combination treatments to improve therapeutic outcomes. A significant obstacle in treating GBM is the expression of O6-methylguanine-DNA methyltransferase, a DNA repair enzyme that reduces the efficacy of the standard alkylating agent, temozolomide, in about 50% of patients. This underscores the need for novel, more targeted therapies. Our study investigates the metabolic–epigenetic impact of combining SN-38, a novel topoisomerase inhibitor inducing DNA double-strand breaks, with rabusertib, a checkpoint kinase 1 inhibitor. We identified this synergistic combination through high-throughput drug screening across a panel of GBM cell lines using a cancer drug library combined with SN-38. A secondary metabolic screening with the PEDS algorithm demonstrated a synergistic modulation of purine, one-carbon, and redox metabolism. Furthermore, the combined treatment led to the significant depletion of epigenetically relevant metabolites such as 5-methyl-cytosine, acetyl-lysine, and trimethyl-lysine. Reduced intermediates of the glutathione cycle indicated increased cellular stress following combinatorial treatment. Overall, the combination of SN-38 and rabusertib synergistically disrupts metabolites associated with epigenetic adaptations, leading to cytotoxicity independent of O6-methylguanine-DNA methyltransferase status, thereby underpinning this combination as a promising candidate for combinatorial therapy in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ed32abd0bdaa8ba2724fd42a8df45084943d6e1" target='_blank'>
              Targeting Metabolic and Epigenetic Vulnerabilities in Glioblastoma with SN-38 and Rabusertib Combination Therapy
              </a>
            </td>
          <td>
            J. Chiou, Valeria Impedovo, Yen Bao Huynh, Ruggiero Gorgoglione, Luiz O. F. Penalva, A. Lodi, Andrew J. Brenner, Stefano Tiziani
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, Graeme Lindsey, Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BRAF inhibitor-resistant melanomas (BRAFiR) acquire (epi)genetic and functional alterations that enable them to evade alternative treatments. Identifying these alterations is critical to advancing treatment strategies. Here, we explored the effect of Chk1 inhibition (Chk1i) on BRAFiR cells, revealing higher sensitivity compared to treatment-naïve cells both in vitro and in vivo. Using FUCCI-labeling and time-lapse microscopy, we show that S phase progression is required for Chk1i-induced cytotoxicity in BRAFiR cells, but not in treatment-naïve cells. Replication stress markers, including reduced BrdU incorporation and increased phospho-RPA and γH2AX, were exclusive to BRAFiR cells exposed to Chk1i. Untreated BRAFiR cells exhibited upregulated DNA replication genes, reduced progressing forks and increased origin firing, suggesting intrinsic replication changes. MAPK pathway reactivation in treatment-naïve cells mimicked BRAFiR traits, increasing sensitivity to Chk1i. These findings indicate that Chk1i exploits elevated replication stress specifically in BRAFiR, highlighting its therapeutic potential in overcoming MAPK inhibitor resistance in melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b5d8e2bacae8698be1cc41dc94d4d67c11c52a" target='_blank'>
              Resistance to BRAF inhibitors drives melanoma sensitivity to Chk1 inhibition
              </a>
            </td>
          <td>
            D. G. Carvalho, J. Kenski, Daniel A. Moreira, Matheus A. Rajão, O. Krijgsman, Carolina Furtado, M. Boroni, João P. B. Viola, D. Peeper, P. Possik
          </td>
          <td>2025-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract Significant advances in the development of new cancer therapies have given rise to multiple novel therapeutic options in chemotherapy, radiotherapy, immunotherapy, and targeted therapies. Although the development of resistance is often reported along with temporary disease remission, there is often tumor recurrence of an even more aggressive nature. Resistance to currently available anticancer drugs results in poor overall and disease‐free survival rates for cancer patients. There are multiple mechanisms through which tumor cells develop resistance to therapeutic agents. To date, efforts to overcome resistance have only achieved limited success. Epitranscriptomics, especially related to m6A RNA modification dysregulation in cancer, is an emerging mechanism for cancer therapy resistance. Here, recent studies regarding the contributions of m6A modification and its regulatory proteins to the development of resistance to different cancer therapies are comprehensively reviewed. The promise and potential limitations of targeting these entities to overcome resistance to various anticancer therapies are also discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/477760476e17db056d04688fc6e1891cd8c161d2" target='_blank'>
              Epitranscriptomic RNA m6A Modification in Cancer Therapy Resistance: Challenges and Unrealized Opportunities
              </a>
            </td>
          <td>
            M. Uddin, Zhishan Wang, Chengfeng Yang
          </td>
          <td>2024-12-11</td>
          <td>Advanced Science</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="The CRISPR/Cas system has emerged as a transformative genome-editing technology, enabling precise modifications of genetic material. This paper explores the potential applications of CRISPR/Cas technology in the treatment of gliomas, particularly glioblastoma (GBM), the most aggressive and lethal brain tumor. With over 3,000 genes linked to disease-causing mutations, CRISPR/Cas offers a novel approach to address the underlying genetic alterations in cancer. Recent studies demonstrate the efficacy of CRISPR/Cas12a in knocking out miR-21, a microRNA inversely correlated with glioblastoma survival, resulting in reduced tumor growth in mouse models. Furthermore, we introduce a novel "genome shredding" approach using CRISPR/Cas9 to target repetitive DNA sequences in GBM, leading to extensive cell death, independent of the tumor's genetic profile. Despite these promising advancements, challenges remain in optimizing CRISPR's efficacy and delivery mechanisms to improve patient outcomes. This review highlights the significant potential of CRISPR/Cas systems in revolutionizing glioma therapy while addressing the need for continued research to mitigate off-target effects and enhance therapeutic delivery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/500664bef54808a00d4dd7947aa8294abce819eb" target='_blank'>
              CRISPR/Cas gene editing and its application in the treatment of Gliomas in the brain
              </a>
            </td>
          <td>
            Raymond Sun
          </td>
          <td>2024-12-24</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic cancer (PC), known as the "king of cancers," is characterized by an exceptionally low five-year survival rate, posing a formidable challenge to global public health. N6-methyladenosine (m6A) methylation is prevalent across various stages of eukaryotic RNA expression, including splicing, maturation, stability, translation, and localization, and represents a pivotal mechanism of epigenetic regulation. m6A methylation influences tumor initiation and progression by modulating post-transcriptional processes, playing a critical role in sustaining cancer cell stemness, promoting cell proliferation, and mediating drug resistance. Extensive research underscores the substantial contribution of m6A modifications to PC development. However, the multiplicity of m6A regulators and their intricate mechanisms of action complicate the landscape. This review aims to deepen the understanding of m6A's role in PC by delineating its involvement in four key areas of tumorigenesis: the hypoxic tumor microenvironment, metabolic reprogramming, immune microenvironment, and resistance mechanisms. Additionally, the review addresses the emerging frontier of m6A interactions with non-coding RNAs (ncRNAs), offering insights into the potential therapeutic and prognostic applications of m6A in the treatment and prognosis prediction of PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b08ca5467f445ec0f42c55b9835ad0ac4ba47" target='_blank'>
              The Mechanism and Latest Progress of m6A Methylation in the Progression of Pancreatic Cancer
              </a>
            </td>
          <td>
            , Peng Ma, Ying He, , Zuo Mou, Ting Fang, Wei Wang, Kaihuan Yu
          </td>
          <td>2025-01-13</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell-free gene expression (CFE) systems empower synthetic biologists to build biological molecules and processes outside of living intact cells. The foundational principle is that precise, complex biomolecular transformations can be conducted in purified enzyme or crude cell lysate systems. This concept circumvents mechanisms that have evolved to facilitate species survival, bypasses limitations on molecular transport across the cell wall, and provides a significant departure from traditional, cell-based processes that rely on microscopic cellular “reactors.” In addition, cell-free systems are inherently distributable through freeze-drying, which allows simple distribution before rehydration at the point-of-use. Furthermore, as cell-free systems are nonliving, they provide built-in safeguards for biocontainment without the constraints attendant on genetically modified organisms. These features have led to a significant increase in the development and use of CFE systems over the past two decades. Here, we discuss recent advances in CFE systems and highlight how they are transforming efforts to build cells, control genetic networks, and manufacture biobased products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66dfd8e3013123535721c036d0642b5350410cbb" target='_blank'>
              Cell-Free Gene Expression: Methods and Applications
              </a>
            </td>
          <td>
            Andrew Hunt, Blake J. Rasor, Kosuke Seki, Holly M Ekas, Katherine F. Warfel, Ashty S. Karim, M. Jewett
          </td>
          <td>2024-12-19</td>
          <td>Chemical Reviews</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Metabolic rewiring is required for cancer cells to survive in harsh microenvironments and is considered to be a hallmark of cancer. Specific metabolic adaptations are required for a tumor to become invasive and metastatic. Cell division and metabolism are inherently interconnected, and several cell cycle modulators directly regulate metabolism. Here, we report that TBK1, which is a noncanonical IKK kinase with known roles in cell cycle regulation and TLR signaling, affects cellular metabolism in cancer cells. While TBK1 is reported to be overexpressed in several cancers and its enhanced protein level correlates with poor prognosis, the underlying molecular mechanism involved in the tumor-promoting role of TBK1 is not fully understood. In this study, we show a novel role of TBK1 in regulating cancer cell metabolism using combined metabolomics, transcriptomics, and pharmacological approaches. We find that TBK1 mediates the regulation of nucleotide and energy metabolism through aldo-keto reductase B10 (AKRB10) and thymidine phosphorylase (TYMP) genes, suggesting that this TBK1-mediated metabolic rewiring contributes to its oncogenic function. In addition, we find that TBK1 inhibitors can act synergistically with AKRB10 and TYMP inhibitors to reduce cell viability. These findings raise the possibility that combining these inhibitors might be beneficial in combating cancers that show elevated levels of TBK1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a02a736e9562fec8efc444bbd368f2506195967" target='_blank'>
              TBK1 Reprograms Metabolism in Breast Cancer: An Integrated Omics Approach.
              </a>
            </td>
          <td>
            M. Maan, Neha Jaiswal, Min Liu, Harold I Saavedra, S. Chellappan, Mainak Dutta
          </td>
          <td>2024-12-13</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a278e50ce1c75eb976a4bb58887b3881b23ccb24" target='_blank'>
              Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations
              </a>
            </td>
          <td>
            Julianne S Funk, Maria Klimovich, Daniel Drangenstein, Ole Pielhoop, Pascal Hunold, Anna Borowek, Maxim Noeparast, Evangelos Pavlakis, M. Neumann, Dimitrios-Ilias Balourdas, Katharina Kochhan, N. Merle, Imke Bullwinkel, M. Wanzel, Sabrina Elmshäuser, J. Teply-Szymanski, Andrea Nist, Tara Procida, Marek Bartkuhn, Katharina Humpert, M. Mernberger, R. Savai, Thierry Soussi, A. Joerger, Thorsten Stiewe
          </td>
          <td>2025-01-07</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cancer, a prominent global health concern, involves the uncontrolled growth and dissemination of abnormal cells, forming tumors that can infiltrate nearby tissues and metastasize to distant sites. Early and precise tumor detection and treatments are crucial to impede their spread and mitigate disease progression. This study presents the development of a scoring system that harnesses single-cell RNA sequencing (scRNA-seq) data to predict the suitability of solid tumors, specifically glioblastoma (GBM) and lymphoma, for advanced cell therapies such as CAR-T, NK cell, and TIL therapies. By integrating key cellular factorsincluding tumor-infiltrating cells, immunosuppressive cells, and immune escape markersthis system provides a quantitative assessment of a tumors potential responsiveness to these therapies. Validation with patient-derived samples has demonstrated the system's potential to improve precision in therapy selection, offering a promising tool for personalized oncology. This research not only aims to enhance the effectiveness of cell therapies for cancer patients but also lays the groundwork for applying similar methodologies to other cancer types, advancing the integration of precision medicine in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1aa40086c8127d386e515ff58d140655d2ef491d" target='_blank'>
              A Scoring System that Leverages Single Cell RNA-Sequencing Data to Predict Cell Therapy Outcome in Solid Tumor Treatment
              </a>
            </td>
          <td>
            Yuwen Li
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c49e06286441ee42660f276752afffa0be81c30" target='_blank'>
              Dysregulation of GAS5-miRNA-Mediated Signaling Pathways in Cancer Pathobiology: A Comprehensive Exploration of Pathways Influenced by this Axis.
              </a>
            </td>
          <td>
            Enwa Felix Oghenemaro, Abdulrahman Qais Khaleel, J. Rizaev, R. Roopashree, Muath Suliman, Syeda Wajida Kazmi, Ahmed Hjazi, Pranchal Rajput, Yasser Fakri Mustafa, M. Abosaoda
          </td>
          <td>2024-12-24</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer therapies have attempted to target the transcription factor p53, a gene also described as the “guardian of the genome,” for decades. However, the approach has faced numerous barriers to clinical efficacy due to several factors: mutations in p53 occur in almost half of all human cancers, mutations are cancer-specific, and the associated genomic changes grant mutant p53 with oncogenic potential unique from that of wild-type p53. A host of new therapeutic agents have emerged that work to target mutant p53. These agents can broadly be classified into six categories: the viral approach, direct modifiers of the p53 pathway, epigenetic modifiers of the p53 pathway, synthetic lethal agents, structural reactivators, and immune activating vaccines. Even these strategies have been met with limited success. Bypassing p53 entirely may be the next avenue in cancer therapeutics to kill tumor cells regardless of p53’s mutation pattern.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b1fb68a093354533852b3fe76935ba3e2890db" target='_blank'>
              Side-stepping the guardian of the genome: current cancer therapeutics targeting mutant p53
              </a>
            </td>
          <td>
            I. Taritsa, Eric T. Fossel
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tetraploidy is a condition in which the entire set of chromosomes doubles, most often due to errors during cell division. Tetraploidy can lead to genomic instability and significant consequences, in particular metastasis and treatment failure in tumours, including radiotherapy. The development of new strategies to sensitise these cells to treatment is of great importance. In our study, we investigated the in vitro combination of chemical treatment with the kinase inhibitor SP600125 and irradiation on diploid versus metastatic tetraploid RKO colon cancer clones. We assessed mitochondrial transmembrane potential, cell cycle and subG1 population by flow cytometry and performed clonogenic assays to evaluate cell sensitivity. We found that the combination overcomes irradiation resistance in metastatic tetraploid clones. To identify the main pathway involved in cell sensitivity, we screened the Harvard Medical School KINOMEscan library and performed a gene ontology biological process analysis. We found that the major kinases inhibited by SP600125 were ANKK1, BIKE, IKKA, JNK1, MP2K3, MP2K4, MKNK2, MYLK, PLK4, RPS6KA4(Kin,Dom,1), MYLK4 and TTK, and the pathways involved in clone sensitivity were DNA damage repair, radiation resistance and apoptosis, through JNK pathway inhibition. Finally, our main finding was that combined treatment with SP600125 and radiotherapy reduced the resistance of metastatic tetraploid cells to treatment, essentially by inhibiting the JNK pathway. This result supports a promising anti-cancer strategy to overcome the resistance of tetraploid cancer cells to irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0380fc971abb59db901b3b88a1c597ae6b670f0" target='_blank'>
              JNK Inhibition Overcomes Resistance of Metastatic Tetraploid Cancer Cells to Irradiation-Induced Apoptosis
              </a>
            </td>
          <td>
            Mohamed Jemaà, N. Setti Boubaker, Nesrine Kerkeni, Stephan M. Huber
          </td>
          <td>2025-01-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68732e730bf76384988cc8a9370de7f622451ad" target='_blank'>
              Targeting KRAS: from metabolic regulation to cancer treatment
              </a>
            </td>
          <td>
            Yanyan Shi, Huiling Zheng, Tianzhen Wang, Shengpu Zhou, Shiqing Zhao, Mo Li, Baoshan Cao
          </td>
          <td>2025-01-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BCR::ABL1 drives chronic myeloid leukemia (CML) disease and treatment, as revealed by the success of tyrosine kinase inhibitor (TKI) therapy. However, additional poorly characterized molecular pathways, acting as BCR::ABL1 independent mechanisms, play crucial roles in CML, contributing to leukemic stem cells (LSCs) persistence, TKI resistance and disease progression. Here, by combining high sensitive mass spectrometry (MS)-based phosphoproteomics with the SignalingProfiler pipeline, we obtained two signaling maps offering a comprehensive description of the BCR::ABL1 dependent and independent pro-survival signalling mechanisms. We leveraged these maps to unbiasedly and systematically discover therapeutic vulnerabilities, by implementing the Druggability Score computational algorithm. By this strategy, and in combination with in vitro and in ex vivo functional assays, we show a crucial role of acquired FLT3-dependency in resistant CML models. In conclusion, we reposition FLT3, one of the most frequently mutated drivers of acute leukemia, as a potential therapeutic target for TKI resistant CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e945b75e65b40ef8d68e265cfc17bc10434dceea" target='_blank'>
              Phosphoproteomics reveals novel BCR::ABL1-independent mechanisms of resistance in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Valeria Bica, Veronica Venafra, Giorgia Massacci, Simone Graziosi, Sara Gualdi, G. Minnella, F. Sorà, P. Chiusolo, Maria Elsa Brunetti, G. Napolitano, M. Breccia, Dimitrios Mougiakakos, Martin Böettcher, Thomas Fischer, Livia Perfetto, F. Sacco
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f79d34836731e3bc98f26657fe4be3cd3ea9eb09" target='_blank'>
              The systemic evolutionary theory of the origin of cancer (SETOC): an update
              </a>
            </td>
          <td>
            Antonio Mazzocca, Giovanni Ferraro, Giovanni Misciagna
          </td>
          <td>2025-01-14</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e5ba64df098cc1a4c221cfe3104b836d138b7b9" target='_blank'>
              Genome-wide association studies are enriched for interacting genes
              </a>
            </td>
          <td>
            Peter T. Nguyen, Simon G. Coetzee, Irina Silacheva, Dennis J. Hazelett
          </td>
          <td>2025-01-15</td>
          <td>BioData Mining</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Introduction Metastatic cancer affects millions of people worldwide annually and is the leading cause of cancer-related deaths. Most patients with metastatic disease are not eligible for surgical resection, and current therapeutic regimens have varying success rates, some with 5-year survival rates below 5%. Here, we test the hypothesis that metastatic cancer can be genetically targeted by exploiting single base substitution mutations unique to individual cells that occur as part of normal aging prior to transformation. These mutations are targetable because ~10% of them form novel tumor-specific “NGG” protospacer adjacent motif (PAM) sites targetable by CRISPR-Cas9. Methods Whole genome sequencing was performed on five rapid autopsy cases of patient-matched primary tumor, normal and metastatic tissue from pancreatic ductal adenocarcinoma decedents. CRISPR-Cas9 PAM targets were determined by bioinformatic tumor-normal subtraction for each patient and verified in metastatic samples by high-depth capture-based sequencing. Results We found that 90% of PAM targets were maintained between primary carcinomas and metastases overall. We identified rules that predict PAM loss or retention, where PAMs located in heterozygous regions in the primary tumor can be lost in metastases (private LOH), but PAMs occurring in regions of loss of heterozygosity (LOH) in the primary tumor were universally conserved in metastases. Conclusions Regions of truncal LOH are strongly retained in the presence of genetic instability and, therefore, represent genetic vulnerabilities in pancreatic adenocarcinomas. A CRISPR-based gene therapy approach targeting these regions may be a novel way to genetically target metastatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada3070ea15ff6901b235b693417e61df449621f" target='_blank'>
              Islands of genomic stability in the face of genetically unstable metastatic cancer
              </a>
            </td>
          <td>
            Kirsten Bowland, Jiaying Lai, Alyza M. Skaist, Yan Zhang, S. Teh, Nicholas J. Roberts, Elizabeth Thompson, Sarah Wheelan, Ralph H. Hruban, Rachel Karchin, Matthew H. Bailey, Christine A. Iacobuzio-Donahue, J. Eshleman
          </td>
          <td>2024-12-19</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1f5a7ee68cb6b5638219d48a2b0e5c5d417966" target='_blank'>
              A complex systems approach to mosaic loss of the Y chromosome.
              </a>
            </td>
          <td>
            Xihan Guo, Xueqin Dai
          </td>
          <td>2024-12-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb46016667edbc4343a8e993830903869bc95403" target='_blank'>
              The mutational landscape of Staphylococcus aureus during colonisation
              </a>
            </td>
          <td>
            Francesc Coll, B. Blane, Katherine L. Bellis, M. Matuszewska, Toska Wonfor, Dorota Jamrozy, M. Toleman, Joan A Geoghegan, Julian Parkhill, R. Massey, Sharon J. Peacock, E. Harrison
          </td>
          <td>2025-01-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. AbstractThis study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2) and triple negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0340ce98aabd282b36902293b1006accf1c9547b" target='_blank'>
              Investigating the genomic landscape of mouse models of breast cancer metastasis
              </a>
            </td>
          <td>
            Christina Ross, Karol Szczepanek, Jack Sanford, Tinghu Qiu, Larry Pearce, Kent W. Hunter
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Organisms regulate gene expression in response to environmental cues, a process known as plasticity, to adjust to changing environments. Research into natural variation and the evolution of plasticity frequently studies cis-regulatory elements with theory suggesting they are more important evolutionarily than trans-regulatory elements. Genome-wide association studies have supported this idea, observing a predominance of cis-loci affecting plasticity. However, studies in structured populations provide a contrasting image, raising questions about the genetic architecture of natural variation in plasticity. To circumvent potential statistical difficulties present in genome-wide association studies, we mapped loci underlying transcriptomic plasticity in response to salicylic acid using recombinant inbred lines generated from two random Arabidopsis thaliana accessions. We detected extensive transgressive segregation in the salicylic acid response, suggesting that plasticity to salicylate in Arabidopsis is polygenic. Most loci (>75%) underlying this variation act in trans, especially for loci influencing plasticity. Trans-acting loci were enriched in genome hotspots, with predominantly small effect sizes distributed across many genes. This could potentially explain their under-discovery in genome-wide association studies. This work reveals a potentially important role for trans-acting loci in plastic expression responses, with implications for understanding plant adaptation to different environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6242103d645ac263c51e56f8b61d6b0403cdeb3b" target='_blank'>
              Trans-regulatory loci shape natural variation of gene expression plasticity in Arabidopsis
              </a>
            </td>
          <td>
            Mariele Lensink, Grey Monroe, Dan Kliebenstein
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3899cef9922a83c1043ba54cdf83bc829243942" target='_blank'>
              The role of genetic diversity, epigenetic regulation, and sex-based differences in HIV cure research: a comprehensive review
              </a>
            </td>
          <td>
            Punitha Letchumanan, Kumitaa Theva Das
          </td>
          <td>2025-01-03</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d173ca4b0414b7ce03e7c7ceb9b88bb092a887d" target='_blank'>
              Chemical inducers of proximity: precision tools for apoptosis in transcriptional regulation
              </a>
            </td>
          <td>
            Kaituo Huang
          </td>
          <td>2024-12-20</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Chemotherapy is a potent tool against cancer, but drug resistance remains a major obstacle. To combat this, understanding the molecular mechanisms behind resistance in cancer cells and the protein expression changes driving these mechanisms is crucial. Targeting the Ubiquitin‐Proteasome System (UPS) has proven effective in treating multiple myeloma and shows promise for solid tumours. Despite initial success with the proteasome inhibitor bortezomib, acquired resistance soon after treatment poses a significant challenge to its efficacy. In this study, we explored proteins potentially involved in acquired resistance to bortezomib using label‐free nLC–MS/MS proteomic analysis. The investigation revealed 299 proteins with notable differences in expression levels in the bortezomib‐resistant PC3 prostate cancer cell line. Using bioinformatics tools, we illustrated the top 10 gene ontology (GO) processes [e.g., translational initiation (p = 5.964E‐10), CRD‐mediated mRNA stabilisation (p = 1.636E‐5), and hydrogen ion transmembrane transport (p = 6.46E‐5)] and the top 20 KEGG [e.g., metabolic pathways (p = 7.601E‐13), biosynthesis of amino acids (p = 3.834E‐12), and chemical carcinogenesis—reactive oxygen species (p = 1.891E‐4)] and REACTOME [e.g., metabolism (p = 4.182E‐21), translation (p = 9.484E‐18), and Nonsense‐Mediated Decay (NMD) (p = 1.829E‐8)] pathways in the PC3‐resistant cells. We further refined our results by comparing them with globally validated TCGA datasets. We correlated the 299 proteins identified through proteomic analysis with tumour aggressiveness and resistance by comparing them with the TCGA nodal metastasis N0 vs. N1 datasets using the UALCAN portal and identified 37 proteins consistent with our results. We believe that a combination of bortezomib with chemotherapeutics targeting these proteins could be effective in overcoming the resistance developed against bortezomib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fb7e9b77b4e308aef46ff7c96b68009d3d9b402" target='_blank'>
              Comparative Analysis of Acquired Resistance to Bortezomib in Prostate Cancer Cells Using Proteomic and Bioinformatic Tools
              </a>
            </td>
          <td>
            Semih Seker, B. Sahin, A. Yerlikaya
          </td>
          <td>2025-01-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background The goal of precision oncology is to find effective therapeutics for every patient. Through the inclusion of emerging therapeutics in a high‐throughput drug screening platform, our functional genomics pipeline inverts the common paradigm to identify patient populations that are likely to benefit from novel therapeutic strategies. Approach Utilizing drug screening data across a panel of 46 cancer cell lines from 11 tumor lineages, we identified an ovarian cancer‐specific sensitivity to the first‐in‐class CRL4 inhibitors KH‐4‐43 and 33‐11. CRL4 (i.e., Cullin‐4 RING E3 ubiquitin ligase) is known to be dysregulated in a variety of cancer contexts, making it an attractive therapeutic target. Unlike proteasome inhibitors that are associated with broad toxicity, CRL4 inhibition offers the potential for tumor‐specific effects. Results We observed that CRL4 inhibition negatively regulates core gene signatures that are upregulated in ovarian tumors and significantly slowed tumor growth as compared to the standard of care, cisplatin, in OVCAR8 xenografts. Building on this, we performed combination drug screening in conjunction with proteomic and transcriptomic profiling to identify ways to improve the antitumor effects of CRL4 inhibition in ovarian cancer models. CRL4 inhibition consistently resulted in activation of the mitogen‐activated protein kinase (MAPK) signaling cascade at both the transcriptomic and protein levels, suggesting that survival signaling is induced in response to CRL4 inhibition. These observations were concordant with the results of the combination drug screens in seven ovarian cancer cell lines that showed CRL4 inhibition cooperates with MEK inhibition. Preclinical studies in OVCAR8 and A2780 xenografts confirmed the therapeutic potential of the combination of KH‐4‐43 and trametinib, which extended overall survival and slowed tumor progression relative to either single agent or the standard of care. Conclusions Together, these data demonstrate the prospective utility of functional modeling pipelines for therapeutic development and underscore the clinical potential of CRL4 inhibition in the ovarian cancer context. Highlights A precision medicine pipeline identifies ovarian cancer sensitivity to CRL4 inhibitors. CRL4 inhibition induces activation of MAPK signalling as identified by RNA sequencing, proteomics, and phosphoproteomics. Inhibitor combinations that target both CRL4 and this CRL4 inhibitor‐induced survival signalling enhance ovarian cancer sensitivity to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261c6bcbc9392bde91ee7e96491c8582460ba6dd" target='_blank'>
              Functional genomics pipeline identifies CRL4 inhibition for the treatment of ovarian cancer
              </a>
            </td>
          <td>
            S. Claridge, Shalini Nath, Anneliese Baum, Richard Farias, J. Cavallo, Nile M Rizvi, Lamberto De Boni, Eric Park, Genesis Lara Granados, Matthew Hauesgen, Ruben Fernandez-Rodriguez, Eda N Kozan, E. Kanshin, Khoi Q Huynh, Peng-Jen Chen, Kenneth Wu, Beatrix Ueberheide, Juan Miguel Mosquera, Fred R Hirsch, Robert J. DeVita, O. Elemento, Chantal Pauli, Zhen-Qiang Pan, Benjamin D Hopkins
          </td>
          <td>2025-01-24</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Every protein progresses through a natural lifecycle from birth to maturation to death; this process is coordinated by the protein homeostasis system. Environmental or physiological conditions trigger pathways that maintain the homeostasis of the proteome. An open question is how these pathways are modulated to respond to the many stresses that an organism encounters during its lifetime. To address this question, we tested how the fitness landscape changes in response to environmental and genetic perturbations using directed and massively parallel transposon mutagenesis in Caulobacter crescentus. We developed a general computational pipeline for the analysis of gene-by-environment interactions in transposon mutagenesis experiments. This pipeline uses a combination of general linear models, statistical knockoffs, and a nonparametric Bayesian statistical model to identify essential genetic network components that are shared across environmental perturbations. This analysis allows us to quantify the similarity of proteotoxic environmental perturbations from the perspective of the fitness landscape. We find that essential genes vary more by genetic background than by environmental conditions, with limited overlap among mutant strains targeting different facets of the protein homeostasis system. We also identified 146 unique fitness determinants across different strains, with 19 genes common to at least two strains, showing varying resilience to proteotoxic stresses. Experiments exposing cells to a combination of genetic perturbations and dual environmental stressors show that perturbations that are quantitatively dissimilar from the perspective of the fitness landscape are likely to have a synergistic effect on the growth defect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d35fab11c10334ca080d2ea7ac3de9caeecc10" target='_blank'>
              Discovering genetic modulators of the protein homeostasis system through multilevel analysis
              </a>
            </td>
          <td>
            Vishal Sarsani, Berent Aldikacti, Tingting Zhao, Shai He, Peter Chien, Patrick Flaherty
          </td>
          <td>2024-12-23</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bc121c6fca52318c6e934753748ac46a988822d" target='_blank'>
              Integrating single-cell RNA-Seq and bulk RNA-Seq data to explore the key role of fatty acid metabolism in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hua Dai, Xin Tao, Yuansen Shu, Fanrong Liu, Xiaoping Cheng, Xiushen Li, Bairui Shu, Hongcheng Luo, XuXiang Chen, Zhaorui Cheng
          </td>
          <td>2025-01-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ebbd60130879f07ccf62c91d033d068147d46a" target='_blank'>
              A safe haven for cancer cells: tumor plus stroma control by DYRK1B.
              </a>
            </td>
          <td>
            Miriam Ems, Anna Brichkina, Matthias Lauth
          </td>
          <td>2025-01-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Background/Objectives: Colorectal cancer (CRC) remains a leading cause of cancer mortality globally, with KRAS mutations occurring in 30–40% of cases, contributing to poor prognosis and resistance to anti-EGFR therapy. This review explores the biological significance, clinical implications, and therapeutic targeting of KRAS mutations in CRC. Methods: A comprehensive analysis of the existing literature and clinical trials was performed, highlighting the role of KRAS mutations in CRC pathogenesis, their impact on prognosis, and recent advancements in targeted therapies. Specific attention was given to emerging therapeutic strategies and resistance mechanisms. Results: KRAS mutations drive tumor progression through persistent activation of MAPK/ERK and PI3K/AKT signaling pathways. These mutations influence the tumor microenvironment, cancer stem cell formation, macropinocytosis, and cell competition. KRAS-mutant CRC exhibits poor responsiveness to anti-EGFR monoclonal antibodies and demonstrates primary and acquired resistance to KRAS inhibitors. Recent breakthroughs include the development of KRAS G12C inhibitors (sotorasib and adagrasib) and promising agents targeting G12D mutations. However, response rates in CRC remain suboptimal compared to other cancers, necessitating combination therapies and novel approaches, such as vaccines, nucleic acid-based therapeutics, and macropinocytosis inhibitors. Conclusions: KRAS mutations are central to CRC pathogenesis and present a significant therapeutic challenge. Advances in KRAS-targeted therapies offer hope for improved outcomes, but resistance mechanisms and organ-specific differences limit efficacy. Continued efforts in personalized treatment strategies and translational research are critical for overcoming these challenges and improving patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2803fdbec8d915a47df49692d27cd1e6edb65e91" target='_blank'>
              The Role of KRAS Mutations in Colorectal Cancer: Biological Insights, Clinical Implications, and Future Therapeutic Perspectives
              </a>
            </td>
          <td>
            Mitsunobu Takeda, Shoma Yoshida, Takuya Inoue, Y. Sekido, Tsuyoshi Hata, Atsushi Hamabe, T. Ogino, N. Miyoshi, Mamoru Uemura, Hirofumi Yamamoto, Y. Doki, H. Eguchi
          </td>
          <td>2025-01-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="The phenomenon of adaptive mutation in bacteria presents several challenges to classical models of evolution, particularly regarding the observed coordination of mutation patterns across populations. Here, we examine statistical evidence from evolved Escherichia coli populations showing mutation enrichment up to 256-fold above background rates in key metabolic genes, with remarkable temporal stability that appears to transcend known bacterial communication mechanisms. We propose a theoretical framework suggesting that bacteria may utilize quantum coherent oscillations, potentially mediated through synchronized membrane potential fluctuations, to achieve this degree of coordination in adaptive mutation. Building on Sprouffske et al.’s findings that high mutation rates can limit adaptive evolution in E. coli, we examined mutation patterns in four key metabolic genes (pykF, topA, cspC, and rpoC) previously identified as targets of selection. While Sprouffske found that extremely high mutation rates impaired adaptation, our analysis reveals that these genes show non-random enrichment patterns (p < 1.76×10-34 for pykF) that maintain remarkable temporal stability across multiple timepoints. Recent advances in quantum biology have demonstrated sustained quantum coherence in biological systems, we present a model for how quantum-mediated bacterial signaling could potentially contribute to adaptive mutation. The framework makes several experimentally testable predictions about mutation patterns, population dynamics, and coherence times in bacterial populations under stress conditions. While substantial experimental validation remains necessary, this work provides specific approaches for investigating potential quantum contributions to bacterial adaptation, with implications for understanding evolutionary mechanisms and bacterial stress responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c15202aa7837996110092bdd84ad8961fcfb080a" target='_blank'>
              Could Quantum-Mediated Bacterial Signaling Explain Adaptive Mutation?
              </a>
            </td>
          <td>
            Patrick Ross
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) comprise a group of genetically heterogeneous hematological malignancies that result in the abnormal growth of leukemic cells and halt the maturation process of normal hematopoietic stem cells. Despite using molecular and cytogenetic risk classification to guide treatment decisions, most AML patients survive for less than five years. A deeper comprehension of the disease’s biology and the use of new, targeted therapy approaches could potentially increase cure rates. RAS oncogene mutations are common in AML patients, being observed in about 15–20% of AML cases. Despite extensive efforts to find targeted therapy for RAS-mutated AMLs, no effective and tolerable RAS inhibitor has received approval for use against AMLs. The frequency of RAS mutations increases in the context of AMLs’ chemoresistance; thus, novel anti-RAS strategies to overcome drug resistance and improve patients’ therapy responses and overall survival are the need of the hour. In this article, we aim to update the current knowledge on the role of RAS mutations and anti-RAS strategies in AML treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607005d4dc391e9d5c0e31b65105281e28950846" target='_blank'>
              Rat Sarcoma Virus Family Genes in Acute Myeloid Leukemia: Pathogenetic and Clinical Implications
              </a>
            </td>
          <td>
            Shaimaa Khattab, Adriatik Berisha, Natalia Baran, P. Piccaluga
          </td>
          <td>2025-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are post-transcriptional regulators involved in key biological processes. Lineage-specific losses of multiple miRNA families during evolution are rare. If they occur, they often coincide with significant changes in gene regulation with consequences extending to body plan modifications. Recently, 15 mammalian miRNA families were found missing in the Eumuroidea, including mouse and rat. It remains unknown to what extent this has affected the gene regulatory network in this lineage. Here, we shed light on the implications of these losses. We integrate in silico characterizations of these miRNAs with differential gene expression analyses in human and mouse inducible pluripotent stem cells (iPSCs) overexpressing two representatives, Mir-197 and Mir-769. This revealed that partly the same genes are downregulated in response to the miRNA overexpression in mouse and human. We dated the emergence of the miRNA target sites in the corresponding transcripts to before the mammalian divergence. Therefore, the response to miRNA overexpression in mouse iPSCs likely represents remnants of an ancient gene regulatory network that existed already in the last common ancestor of human and mouse. This provides strong evidence that the miRNA loss disrupted established regulatory connections in Eumuroidea. Notably, we detected an accompanying loss of at least 37 transcription factors in in this lineage. Thus, Eumuroidea have modified their gene regulatory networks on two levels, transcriptionally and post-transcriptionally. It will now be interesting to precisely chart the differences in gene regulation and how this affects the use of mouse and rat as model systems for human disease. Author summary The phenotypic specifics of a species are jointly determined by its sets of structural and regulatory genes, where the latter control gene expression in a spatial and temporal manner. Micro-RNAs (miRNAs) are short regulatory genes and alter gene expression post-transcriptionally. Over evolutionary timescales, they are rarely lost in larger numbers on individual lineages, highlighting their functional relevance. The model species mouse and rat, together with their relatives in the Eumuroidea lack 15 miRNA families that are otherwise largely conserved throughout the mammals. If, and to what extent, the corresponding gene loss has affected gene regulation in the last common ancestor of Eumuroidea is unknown. Here, we provide evidence that the corresponding loss of miRNA genes is not compensated by other genes. We trace remnants of the ancestral regulatory network under control of two miRNAs in mouse, providing first time evidence that the miRNA loss in the Eumuroidea must have had consequences for gene expression regulation. We further show that protein-coding genes that have been co-lost with the 15 miRNAs are enriched for transcription factors. Jointly, this reveals that the diversification of Eumuroidea during mammalian evolution was accompanied by a change in gene expression regulation on the transcriptional and post-transcriptional level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f814f43dd7def60b542ffaccd394458b9eb13d6e" target='_blank'>
              Concerted loss of microRNAs uncovers multi-level restructuring of gene regulation in rodents
              </a>
            </td>
          <td>
            Felix Langschied, M. Leisegang, Stefan Günther, F. Hahner, Ralf Peter Brandes, Ingo Ebersberger
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genomic analysis has played a significant role in the identification of driver mutations that are linked to disease progression and response to drug treatment in ovarian cancer. A prominent example is the stratification of epithelial ovarian cancer (EOC) patients with homologous recombination deficiency (HRD) characterized by mutations in DNA damage repair genes such as BRCA1/2 for treatment with PARP inhibitors. However, recent studies have shown that some epithelial ovarian tumors respond to PARP inhibitors irrespective of their HRD or BRCA mutation status. An exclusive focus on the genome overlooks the significant insight that can be gained from other biological analytes, including proteins, which carry out cellular functions. Proteogenomics is the integration of genomics, transcriptomics, epigenomics and proteomics data. This review paper provides novel insight into the role of proteogenomics as an analytical approach to identify predictive biomarkers of drug treatment response in epithelial ovarian cancer. Proteogenomic analysis can facilitate the identification of predictive biomarkers of drug treatment response, consequently greatly improving the stratification of patients with EOC for treatment towards a goal of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1220cfea040dac3859c8764ba1d49996aa4caab" target='_blank'>
              Opportunities for predictive proteogenomic biomarkers of drug treatment sensitivity in epithelial ovarian cancer
              </a>
            </td>
          <td>
            T. Philips, Britt K. Erickson, Stefani N. Thomas
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ddc2e4963aa356dbc3de332034a633ed4d7780" target='_blank'>
              Antisense oligonucleotides as a targeted therapeutic approach in model of acute myeloid leukemia.
              </a>
            </td>
          <td>
            D. Maceckova, L. Vaňková, J. Bufka, Petr Hošek, Jiří Moravec, Pavel Pitule
          </td>
          <td>2024-12-18</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background. Colorectal carcinomas (CRCs) are seldom eradicated by cytotoxic chemotherapy. Cancer cells with stem-like functional properties, often referred to as “cancer stem cells” (CSCs), display preferential resistance to several anti-tumor agents used in cancer chemotherapy, but the molecular mechanisms underpinning their selective survival remain only partially understood. Methods. In this study, we used Transcription Factor Target Genes (TFTG) enrichment analysis to identify transcriptional regulators (activators or repressors) that undergo preferential activation by chemotherapy in CRC cells with a “bottom-of-the-crypt” phenotype (EPCAM+/CD44+/CD166+; CSC-enriched) as compared to CRC cells with a “top-of-the-crypt” phenotype (EPCAM+/CD44neg/CD166neg; CSC-depleted). The two cell populations were purified in parallel by fluorescence-activated cell sorting (FACS) from a patient-derived xenograft (PDX) line representative of a moderately differentiated human CRC, following in vivo chemotherapy with irinotecan (CPT-11). The transcriptional regulators identified as differentially activated were tested for differential expression in normal vs. cancer tissues, and in cell populations enriched in stem/progenitor cell-types as compared to differentiated lineages (goblet cells, enterocytes) in the mouse colon epithelium. Finally, the top candidate was tested for mechanistic contribution to drug-resistance by selective down-regulation using short-hairpin RNAs (shRNAs). Results. Our analysis identified E2F4 and TFDP1, two core components of the DREAM transcriptional repression complex, as transcriptional modulators preferentially activated by irinotecan in EPCAM+/CD44+/CD166+ as compared to EPCAM+/CD44neg/CD166neg cancer cells. The expression levels of both genes (E2F4, TFDP1) were found up-regulated in CRCs as compared to human normal colon tissues, and in a sub-population of mouse colon epithelial cells enriched in stem/progenitor elements (Epcam+/Cd44+/Cd66alow/Kitneg) as compared to other sub-populations enriched in either goblet cells (Epcam+/Cd44+/Cd66alow/Kit+) or enterocytes (Epcam+/Cd44neg/Cd66ahigh). Most importantly, E2F4 down-regulation using shRNAs dramatically enhanced the sensitivity of human CRCs to in vivo treatment with irinotecan, across three independent PDX models. Conclusions. Our data identified E2F4 and the DREAM repressor complex as critical regulators of human CRC resistance to irinotecan, and as candidate targets for the development of chemo-sensitizing agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a3c8fee4cf42a4e76a7d265977fa7fcfba7541" target='_blank'>
              The E2F4 transcriptional repressor is a key mechanistic regulator of colon cancer resistance to irinotecan (CPT-11)
              </a>
            </td>
          <td>
            Junichi Matsubara, Y. F. Li, Sanjay Koul, Junko Mukohyama, Luis E. Valencia Salazar, Taichi Isobe, D. Qian, Michael F. Clarke, Debashis Sahoo, R. Altman, Piero Dalerba
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Questions in cancer have engaged systems biologists for decades. During that time, the quantity of molecular data has exploded, but the need for abstractions, formal models, and simplifying insights has remained the same. This review brings together classic breakthroughs and recent findings in the field of cancer systems biology, focusing on cancer-cell pathways for tumorigenesis and therapeutic response. Cancer cells mutate and transduce information from their environment to alter gene expression, metabolism, and phenotypic states. Understanding the molecular architectures that make each of these steps possible is a long-term goal of cancer systems biology pursued by iterating between quantitative models and experiments. We argue that such iteration is the best path to deploying targeted therapies intelligently so that each patient receives the maximum benefit for their cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8544584d0cff558bd248c81e02267385e425b55" target='_blank'>
              Systems Biology of the Cancer Cell.
              </a>
            </td>
          <td>
            Kevin A. Janes, M. Lazzara
          </td>
          <td>2024-12-17</td>
          <td>Annual review of biomedical engineering</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d5925d1e02fdeb24a29a2a8e41aed5142d614d" target='_blank'>
              Advances in A-to-I RNA editing in cancer
              </a>
            </td>
          <td>
            Yi Zhang, Lvyuan Li, Juana Jessica Mendoza, Dan Wang, Qijia Yan, Lei Shi, Zhaojian Gong, Zhaoyang Zeng, Pan Chen, Wei Xiong
          </td>
          <td>2024-12-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a genetically heterogeneous malignancy characterized by the clonal expansion of myeloid precursor cells. Advances in genomic profiling have enhanced our understanding of AML pathogenesis, leading to the identification of recurrent mutations, including TP53, FLT3, MUC4, RAS, and IDH1/2. These mutations significantly influence treatment response and prognosis, with TP53 mutations conferring poor outcomes and resistance to conventional therapies. Despite the advent of venetoclax-based regimens, resistance mechanisms persist, necessitating the development of novel therapeutic strategies. This study aims to investigate the efficacy of drug combinations for the treatment of AML using both in vitro AML cell lines and an in vivo zebrafish embryo xenograft model. Specifically, we focus on two drug combinations; the pan-RAF inhibitor LY3009120 combined with the mTOR inhibitor sapanisertib (designated as LS), and the JAK1/2 inhibitor ruxolitinib combined with the ERK inhibitor ulixertinib (designated as RU). The study integrates real-time cell viability assays, xenograft imaging, and genomic analyses to assess drug efficacy and explore correlations between treatment responses and mutational profiles, particularly TP53, FLT3, and MUC4 mutations. Both combinations, LS and RU, demonstrated superior efficacy compared to venetoclax-based treatments in reducing cell viability across AML cell lines. The LS combination showed significant reductions in cell viability in MOLM16 and SKM cells, while RU exhibited comparable efficacy with lower toxicity profiles. In zebrafish embryos, the LS combination effectively inhibited the proliferation of xenografted human AML cells, as evidenced by decreased fluorescence signals, indicating cell death. The RU combination also disrupted survival signaling pathways, showing promise as a therapeutic strategy. Furthermore, a correlation was identified between drug response and mutational profiles, with TP53, FLT3 and MUC4 mutations significantly influencing sensitivity to the LS and RU combinations. These findings support the further development of LS and RU as effective alternatives to current clinical regimens, with implications for personalized AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e928f03f37069a717618c39ee33259e9551d5324" target='_blank'>
              Targeting AML Resistance with LY3009120-Sapanisertib and Ruxolitinib-Ulixertinib Combinations Demonstrate Superior Efficacy in FLT3, TP53, and MUC4 mutations
              </a>
            </td>
          <td>
            Elham Gholizadeh, Ehsan Zangene, A. Parsons, M. Kontro, Caroline A. Heckman, Mohieddin Jafari
          </td>
          <td>2024-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Spatial organization of the genome is fundamental for ensuring accurate gene expression. This process depends on the communication between gene promoters and distal cis-regulatory elements (CREs), which together make up 8% of the human genome and are supported by the chromatin structure. It is estimated that over 90% of disease-associated variants are located in the non-coding region of the genome and may affect CRE. For the cystic fibrosis transmembrane conductance regulator (CFTR) gene, a complete understanding of tissue-specific CFTR expression and regulation is missing, in particular in the pancreas. Mechanistic insights into tissue-specific expression may provide clarity on the clinical heterogeneity observed in Cystic Fibrosis and CFTR-related disorders. Methods To understand the role of 3D chromatin architecture in establishing tissue-specific expression of the CFTR gene, we mapped chromatin interactions via circular chromosome conformation capture (4C) and epigenomic regulation through H3K27ac and DNase Hypersensitive site I (DHS) in Capan-1 pancreatic cells. Candidate regulatory regions are validated by luciferase reporter assay and CRISPR-knock out. Results We identified active regulatory regions not only around the CFTR gene but also outside the topologically associating domain (TAD). By performing functional assays, we validated our targets and revealed a cooperative effect of the −44 kb, −35 kb, +15.6 kb and 37.7 kb regions, which share common predicted transcription factor (TF) motifs. Comparative 3D genomic analysis and functional assays using the Caco-2 intestinal cell line revealed the presence of tissue-specific CREs. Conclusion By studying the chromatin architecture of the CFTR locus in Capan-1 cells, we demonstrated the involvement of multiple CREs upstream and downstream of the CFTR gene. We also extend our analysis to compare intestinal and pancreatic cells and provide information on the tissue-specificity of CRE. These findings highlight the importance of expanding the search for causative variants beyond the gene coding sequence but also by considering the tissue-specific 3D genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9279897acf411b562291c89812e3d397609427" target='_blank'>
              New insights into the cis-regulation of the CFTR gene in pancreatic cells
              </a>
            </td>
          <td>
            Clara Blotas, A. Le Nabec, M. Collobert, Mattijs Bulcaen, Marianne S. Carlon, Claude Férec, S. Moisan
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, lacking the expression of estrogen receptor (ER), progesterone receptor (PR), and HER2/neu. The absence of these receptors limits therapeutic options necessitating the exploration of novel treatment strategies. Epigenetic modifications, which include DNA methylation, histone modifications, and microRNA (miRNA) regulation, play a pivotal role in TNBC pathogenesis and represent promising therapeutic targets. This review delves into the therapeutic potential of epigenetic interventions in TNBC, with a focus on DNA methylation, histone modifications, and miRNA therapeutics. We examine the role of DNA methylation in gene silencing within TNBC and the development of DNA methylation inhibitors designed to reactivate silenced tumor suppressor genes. Histone modifications, through histone deacetylation and acetylation in particular, are critical in regulating gene expression. We explore the efficacy of histone deacetylase inhibitors (HDACi), which have shown promise in reversing aberrant histone deacetylation patterns, thereby restoring normal gene function, and suppressing tumor growth. Furthermore, the review highlights the dual role of miRNAs in TNBC as both oncogenes and tumor suppressors and discusses the therapeutic potential of miRNA mimics and inhibitors in modulating these regulatory molecules to inhibit cancer progression. By integrating these epigenetic therapies, we propose a multifaceted approach to target the underlying epigenetic mechanisms that drive TNBC progression. The synergistic use of DNA methylation inhibitors, HDACi, and the miRNA-based therapies offers a promising avenue for personalized treatment strategies, aiming to enhance the clinical outcome for patients with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f782a5505b5b2b26e898e78e4e01384ddeb62d3" target='_blank'>
              Unlocking the epigenetic code: new insights into triple-negative breast cancer
              </a>
            </td>
          <td>
            G. Mahendran, Ann Dharshika, Ricardo Romero-Moreno, Nadeeshika Wickramarachchige Dona, S. H. G. S. Sarasija, Sumeth Perera, Gayathri N. Silva, Fang Yang, Guochun Zhang, Wickramarachchige Dona, Perera Sarasija, Silva. This
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5361264ba7fa5b03a05e9caae598d056bebbf71c" target='_blank'>
              Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer.
              </a>
            </td>
          <td>
            Yilv Yan, Dongqing Sun, Junjie Hu, Yue Chen, Liangdong Sun, Huansha Yu, Yicheng Xiong, Zhida Huang, Haoran Xia, Xinsheng Zhu, D. Bian, Fenghuan Sun, Likun Hou, Chunyan Wu, Orion R Fan, Haiyang Hu, An Zeng, Lele Zhang, Yi Eve Sun, Chenfei Wang, Peng Zhang
          </td>
          <td>2024-12-10</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="
 Introduction: The molecular pathways involved in the response to radiation therapy (RT) in pancreatic ductal adenocarcinoma (PDAC) remain poorly understood. Attempts to combine RT with immunotherapies in PDAC have yielded inconsistent outcomes, highlighting a critical gap in our understanding of immune dynamics and mechanisms influencing RT response. Our study aims to uncover molecular pathways and cellular subtypes associated with radioresponse. To do so, we developed a novel cell-cell interaction platform, "SpaCCI", to map ligand-receptor (LR) interactions, essential for understanding cellular communication within the TME and its contribution to therapeutic resistance. Methods: Single-cell RNA sequencing (scRNA-seq) was performed on 16 matched pre- and post-treatment PDAC biopsies from a Phase I/II dose-escalation clinical trial of stereotactic body radiation therapy. Additionally, spatial transcriptomics (ST) was conducted on resected PDAC (20 treated with neoadjuvant chemoradiation, 8 untreated). We integrated scRNA-seq and ST datasets using cell type decomposition and developed SpaCCI to enhance LR interaction detection in tissues with complex cell mixtures. Results: ScRNA-seq identified five molecular subpopulations of cancer cells associated to RT response, with PDAC_3 enriched post-RT and marked by genes involved in lipid metabolism and ferroptosis defense. Post-RT depleted subpopulations, such as PDAC_0 and PDAC_prolif, saw upregulation of pathways related to rapid proliferation and oxidative phosphorylation. In a murine model, PDAC_3 was similarly enriched in post-RT tumors, with persister cell lines from these tumors showing overexpression of ferroptosis defense genes and resistance to ferroptosis inducers including Erastin and RSL3. ST analysis also revealed spatial immune changes post-RT, with increased cytotoxic and effector memory T cells balanced by regulatory T cells and M2-like macrophages, indicating immune escape. NK and memory B cells correlated with improved survival post-RT, while plasma IgG cells localized outside the tumor bed, suggesting compartmentalized immune responses. Using SpaCCI, we observed differential enrichment of ligand-receptor interactions in RT-treated tumors, particularly among lipid-associated tumor macrophages, cancer cells, myofibroblast-like cancer-associated fibroblasts, and endothelial cells. Persister PDAC_3 subpopulations showed enriched LR SEMA3 signaling with CAFs, myeloid cells, and endothelial cells consistent with axon guidance pathways. Conclusions: This study presents a molecular and cellular atlas of PDAC response to RT, revealing persister cells populations with adaptations in lipid metabolism and ferroptosis defense that could be targeted to enhance RT efficacy. ST highlighted complex immune dynamics within the TME demonstrating a balance between inflammatory and immunosuppressive populations. Our findings provide a foundation for combinatorial therapeutic strategies and suggest that axon guidance pathways may also play an important role in the response to RT in PDAC.
 Citation Format: Vincent Bernard, Li-Ting Ku, Tianyu Wang, Kimal Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, Manoop S. Bhutani, Paola A. Guerrero, Ethan B. Ludmir, Albert C. Koong, Cullen M. Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay.Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P018">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd18fd5a66ac51a4015b7b7d9504296b9a16919" target='_blank'>
              Abstract P018: Mapping cellular and microenvironmental determinants of persister populations in pancreatic cancer following neoadjuvant chemoradiation
              </a>
            </td>
          <td>
            Vincent Bernard, Li-Ting Ku, Tianyu Wang, K. Rajapakshe, Dadi Jiang, Galia Jacobson, Jimin Min, Ching-Wei Tzeng, M. Bhutani, Paola A Guerrero, E. Ludmir, Albert C Koong, Cullen M Taniguchi, Anirban Maitra, Ziyi Li, Eugene J. Koay
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11afec2e2fbad72ef42b2a5b045ab6400e1d976a" target='_blank'>
              Genome-wide CRISPR-Cas9 screening identifies ITGA8 responsible for abivertinib sensitivity in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Xuan-Guang Li, Guangsheng Zhu, Pei-Jun Cao, Hua Huang, Yu-Hao Chen, Chen Chen, Pei-Jie Chen, Di Wu, Chen Ding, Zi-He Zhang, Rui-Hao Zhang, Zi-Xuan Hu, Wen-Hao Zhao, Ming-Hui Liu, Yong-Wen Li, Hongyu Liu, Jun Chen
          </td>
          <td>2025-01-14</td>
          <td>Acta pharmacologica Sinica</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24be616e06c75b57b6ca8151f14f1e70c2c9348f" target='_blank'>
              PIK3CA mutation fortifies molecular determinants for immune signaling in vascular cancers.
              </a>
            </td>
          <td>
            Donghee Lee, Emma C Kozurek, Md Abdullah, Ethan J Wong, Rong Li, Z. Liu, Hai Dang Nguyen, Erin Dickerson, Jong Hyuk Kim
          </td>
          <td>2024-12-21</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5489315ebaf863ded4b8dae32edbc65785a23ea3" target='_blank'>
              Bioinformatics analysis of mitochondrial metabolism-related genes demonstrates their importance in renal cell carcinoma
              </a>
            </td>
          <td>
            Mohsen Ahmadi, Maryam Motallebinezhad, P. Mousavi, Amir Hossein Miladipour, Shamim Fooladgar, S. Ghafouri-Fard, Seyed Amirhossein Fazeli
          </td>
          <td>2025-01-09</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract The molecular basis of adaptive evolution and cancer progression are both complex processes that share many striking similarities. The potential adaptive significance of environmentally-induced epigenetic changes is currently an area of great interest in both evolutionary and cancer biology. In the field of cancer biology intense effort has been focused on the contribution of stress-induced non-coding RNAs (ncRNAs) in the activation of epigenetic changes associated with elevated mutation rates and the acquisition of environmentally adaptive traits. Examples of this process are presented and combined with more recent findings demonstrating that stress-induced ncRNAs are transferable from somatic to germline cells leading to cross-generational inheritance of acquired adaptive traits. The fact that ncRNAs have been implicated in the transient adaptive response of various plants and animals to environmental stress is consistent with findings in cancer biology. Based on these collective observations, a general model as well as specific and testable hypotheses are proposed on how transient ncRNA-mediated adaptive responses may facilitate the transition to long-term biological adaptation in both cancer and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbfc2bb67d7c58db953ed76c627eb64e92c4eb6a" target='_blank'>
              Adaptive Significance of Non-coding RNAs: Insights from Cancer Biology
              </a>
            </td>
          <td>
            John F. McDonald
          </td>
          <td>2025-01-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/188a74494b56066cf4346e450a3d56405973c270" target='_blank'>
              Single-cell multi-omics analysis reveals candidate therapeutic drugs and key transcription factor specifically for the mesenchymal subtype of glioblastoma
              </a>
            </td>
          <td>
            Yufan Yang, Ziyuan Liu, Yerong Wei, Shuai He, Ancheng Gu, Zhiyong Li, Jianlong Li, Zhongyuan Xu, B. Cen
          </td>
          <td>2024-12-20</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8feadfbb33b4fde036009823f20cf7ee9457518a" target='_blank'>
              BET inhibition revealed varying MYC dependency mechanisms independent of gene alterations in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Loris Delrieu, Sieme Hamaidia, Emilie Montaut, Andrea Garcìa-Sandoval, Camille Teste, Patricia Betton-Fraisse, Thierry Bonnefoix, S. Carras, Rémy Gressin, Christine Lefebvre, Jérôme Govin, A. Emadali
          </td>
          <td>2024-12-19</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b72e954c6c11c764347d1edffb32b16bc9b9834" target='_blank'>
              Transcriptome and epigenome dynamics of the clonal heterogeneity of human induced pluripotent stem cells for cardiac differentiation
              </a>
            </td>
          <td>
            Jihye Yun, Jaemin So, Seunghee Jeong, Jiye Jang, Soyoung Han, Junseok Jeon, Kyungho Lee, Hye Ryoun Jang, Jaecheol Lee
          </td>
          <td>2024-12-11</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5668e09864551d73b8ef36f697d610eddc6e8f2b" target='_blank'>
              MircoRNAs predict and modulate responses to chemotherapy in leukemic patients.
              </a>
            </td>
          <td>
            D. Ellakwa, Mohamed Ahmed Abdelmalek, Mostafa M Mostafa, Takwa E. Ellakwa, Al-Hassan Soliman Wadan
          </td>
          <td>2025-01-14</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>2</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cytarabine, daunorubicin, and etoposide (ADE) have been the standard backbone of induction chemotherapy regimens for acute myeloid leukemia (AML) patients for over five decades. However, chemoresistance is still a major concern, and a significant proportion of AML becomes resistant to ADE treatment leading to relapse and poor survival. Therefore, there is a significant need to identify mechanisms mediating drug resistance to overcome chemoresistance. Herein, we performed genome-wide synthetic lethal CRISPR/Cas9 screens using araC, daunorubicin and etoposide. We further integrated significant findings from the CRISPR screen with outcome in pAML patients treated with standard ADE regimen on 3 independent clinical trials to identify drug response biomarkers of prognostic significance. We identified seveal mediators that would represent clinically and biologically significant genes for ADE treatment, such as BCL2, CLIP2, and VAV3, which are resistant genes with high expression associated with poor outcomes in pAML treated with ADE, and GRPEL1, HCFC1, and TAF10, which are sensitive genes with high expression showing beneficial outcomes. Notably, knockdowns of the BCL2, CLIP2, and VAV3 genes sensitize the ADE component in AML cell lines, suggesting that these genes should be further studied as potential therapeutic targets to overcome chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/979aaf7e330314833a544cf8bca5acd47b995421" target='_blank'>
              Genome-wide CRISPR screen identifies AraC-Dauno-Eto (ADE) response modulators that predicts outcome in pediatric AML.
              </a>
            </td>
          <td>
            Nam Nguyen, Roya Rafiee, P. K. Parcha, A. Tagmount, J. Rubnitz, Raul C Ribeiro, Xueyuan Cao, S. Pounds, Christopher Vulpe, J. Lamba
          </td>
          <td>2024-12-23</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="ABSTRACT Background SET domain‐containing protein 4 (SETD4) is a histone methyltransferase that has been shown to modulate cell proliferation, differentiation, and inflammatory responses by regulating histone H4 trimethylation (H4K20me3). Previous reports have demonstrated its function in the quiescence of cancer stem cells as well as drug resistance in several cancers. A limited number of systematic studies have examined SETD4's role in the tumor microenvironment, pathogenesis, prognosis, and therapeutic response. Methods Utilizing The Cancer Genome Atlas database, and other publicly accessible platforms, we comprehensively analyzed SETD4 gene expression, methylation patterns, and prognostic significance. Furthermore, we investigated its association with cancer‐related pathways, the immune microenvironment, immunotherapy markers, and drug resistance signatures of chemotherapy. Additionally, qRT‐PCR was performed to validate SETD4 expression in clinical specimens. Results The expression of SETD4 was abnormal across a variety of cancer types and the expression of SETD4 in colorectal cancer tissues was verified in clinical specimens. The upregulation of SETD4 may be a prognostic risk factor predicting poor overall survival and progression‐free survival. The analysis revealed that the mRNA level of SETD4 was modulated by promoter methylation, and patients with lower methylation levels showed shorter survival times. Pathway analysis showed that SETD4 influenced several key cell cycle pathways, including the G2M checkpoint, and mitotic spindle pathways. In addition, SETD4 negatively affects immune cell infiltration in most cancers, including B cells, CD8 T cells, and macrophages. The correlation between SETD4 and cancer stemness as well as homologous recombination deficiency varied across tumor types, suggesting that SETD4 may play a multifaceted role in tumor resistance. Notably, we identified several potential agents targeting SETD4. Conclusions This study demonstrates that SETD4 is an immune‐oncogenic molecule in multiple cancers, with the potential to be a diagnosis, prognosis, and targeted therapy marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f76f7155098d30f047c5ee80fc1fcf8c03b367a" target='_blank'>
              Identification of SETD4 as an Onco‐Immunological Biomarker Encompassing the Tumor Microenvironment, Prognoses, and Therapeutic Responses in Various Human Cancers
              </a>
            </td>
          <td>
            Yuyun Zhong, Ruiqi Wang, Zijie Huang, Zhaoting Hu, Bin Peng, Bin Chen, Liyue Sun
          </td>
          <td>2025-01-01</td>
          <td>Immunity, Inflammation and Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) accumulate somatic mutations over time, some conferring a fitness advantage that can lead to clonal hematopoiesis (CH). Mutations in DNMT3A, particularly at hotspot R882, are the most prevalent in CH and carry an increased risk of acute myeloid leukemia (AML). Although DNMT3A R882 mutations are linked to global DNA hypomethylation, the mechanisms underlying their selective advantage remain unclear. Here, we show that Dnmt3a-R882H mutant HSCs exhibit resilience under inflammatory and genotoxic stress. During IL-1β-induced emergency granulopoiesis, Dnmt3a R882H/+ HSCs uncouple increased proliferation from stem cell exhaustion. In contrast, wild-type HSCs rapidly progress to terminal differentiation. We link this phenotype to a delayed activation of the p53-p21-DREAM axis, that allows mutant HSCs to avoid attrition, despite increased replication stress. Similarly, mutant HSCs exhibit delayed Trp53 activation following irradiation, but eventually recover a physiological Trp53 response. Analysis of patient data reveals shared phenotypic features between DNMT3A and monoallelic TP53 mutations in CH and myeloid neoplasms, highlighting potential functional similarities. Collectively, these findings suggest that the expansion of DNMT3A-mutant clones is affected by impaired TP53 signaling, which confers resilience against stressors. Therapeutic strategies targeting inflammatory pathways or the p53-p21-DREAM axis may reduce DNMT3A-CH expansion and/or progression and its associated risks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e56fa3aad088d2f09d1437903fb4b00ec728238c" target='_blank'>
              Delayed Trp53 activation protects Dnmt3a-mutant hematopoietic stem cells from inflammatory attrition
              </a>
            </td>
          <td>
            Ludovica Marando, G. Giotopoulos, Pedro Madrigal, Dhoyazan M A Azazi, Ryan Asby, M. E. Calleja-Cervantes, M. Gu, David Lara-Astiaso, Nicola K Wilson, R. Hannah, Jaana Bagri, Emily F. Calderbank, E. Laurenti, E. Bernard, Sarah J Horton, Berthold Göttgens, E. Papaemmanuil, Shuchi Agrawal-Singh, M. Gozdecka, G. Vassiliou, B. Huntly
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a clonal disease with recurrent genetic alterations that affect epigenetic states. However, the implications of epigenetic dysregulation in disease progression remain unclear. Here, we interrogated single-cell and clonal level chromatin accessibility of bone marrow samples from 28 pAML patients representing multiple subtypes using mtscATAC-seq, which revealed distinct differentiation hierarchies and abnormal chromatin accessibility in a subtype-specific manner. Innate immune signaling was commonly enhanced across subtypes and related to improved advantage of clonal competition and unfavorable prognosis, with further reinforcement in a relapse-associated leukemia stem cell-like population. We identified a panel of 31 innate immunity related genes to improve the risk classification of pAML patients. By comparing paired diagnosis and post-chemotherapy relapse samples, we showed that primitive cells significantly reduced MHC class II signaling, suggesting an immune evasion mechanism to facilitate their expansion at relapse. Key regulators orchestrating cell cycle dysregulation were identified to contribute to pAML relapse in drug-resistant clones. Our work establishes the single-cell chromatin accessibility landscape at clonal resolution and reveals the critical involvement of epigenetic disruption, offering insights into classification and targeted therapies of pAML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993d1844c9aff473d0effdd0e0a49f56c123ecc" target='_blank'>
              Single-cell epigenetic and clonal analysis decodes disease progression in pediatric acute myeloid leukemia.
              </a>
            </td>
          <td>
            Boyu Cui, Lanlan Ai, Minghui Lei, Yongjuan Duan, Chao Tang, Jingliao Zhang, Yan Gao, Xuan Li, Caiying Zhu, Yingchi Zhang, Xiaofan Zhu, Tomoya Isobe, Wenyu Yang, Berthold Gottgens, Ping Zhu
          </td>
          <td>2024-12-11</td>
          <td>Blood</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14ca816cc15851962002a66601f774cc9384fd1d" target='_blank'>
              Comprehensive genome-scale CRISPR knockout screening of CHO cells
              </a>
            </td>
          <td>
            Sungwook Shin, Su Hyun Kim, Aghiles Gasselin, Gyun Min Lee, Jae Seong Lee
          </td>
          <td>2025-01-15</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="We conducted an investigation into the characteristics of single-cell differentiation data in gliomas, with a focus on developing DAPK1-based prognostic markers to predict patient outcomes. Dysregulated expression of DAPK1 has been associated with the invasive behavior of various malignancies, including gliomas. However, the precise role and underlying mechanisms of DAPK1 in gliomas remain inadequately understood.We performed analyses on RNA-seq and microarray datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO), in addition to single-cell RNA sequencing (scRNA-seq) data from glioma patients available in GEO. Utilizing the Seurat R package, we identified gene clusters associated with survival from the scRNA-seq data. Prognostic models were developed using LASSO and stepwise regression algorithms. Furthermore, we assessed the predictive potential of these genes within the immune microenvironment and their relevance in immunotherapy contexts.Our scRNA-seq data analysis revealed 32 distinct cell clusters corresponding to 10 cell types. Through dimensionality reduction and clustering, we identified three glial cell subpopulations based on their differentiation trajectories. DAPK1, serving as a marker gene for the terminal subpopulation, exhibited an association with poor prognosis.DAPK1-based prognostic models show promise for accurately predicting outcomes in glioblastoma and glioma. An in-depth examination of DAPK1’s specific mechanisms in glioblastoma could elucidate its role in immunotherapy response. Targeting the DAPK1 gene may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af0fd67f17577e9d1b4649b5fe0ecd703f94585d" target='_blank'>
              Single-cell sequencing uncovers the mechanistic role of DAPK1 in glioma and its diagnostic and prognostic implications
              </a>
            </td>
          <td>
            , Yan-Yu Ding, Si-Guo Zhao, Jie-Hui Zhao, Yu Gu, Dong-Hui Chen, Fan Zhang, Wen-Ming Hong
          </td>
          <td>2025-01-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Background Acute lymphocytic leukemia (ALL), characterized by uncontrolled growth of abnormal lymphocytes, predominantly affects children. Genetic analysis focusing on genes and microRNAs reveals important information about the biology of ALL, enabling accurate diagnosis and treatment. This study examines gene and microRNA expression in B cell ALL to improve early diagnosis and personalized treatment. Methods Bone marrow samples were collected from patients both before and after the induction phase of chemotherapy. Comprehensive diagnostic techniques including flow cytometry, molecular assays, real‐time PCR for common translocations, karyotyping, and complete blood count (CBC) analysis were employed. These methods were utilized to determine the type and risk assessment of ALL, identify specific gene and microRNA expressions, and measure blood cell counts. Results The study comprised 12 patients, all under the age of 18. Post‐treatment RT‐PCR analysis revealed significant reductions in the expression of the ABCB1 gene, miR‐129‐5p, and miR‐9‐5p following the induction phase of chemotherapy. Karyotype analysis indicated that two patients were hypodiploid; unfortunately, both of these patients did not survive. Conclusion MicroRNAs and ABC genes serve as predictive and prognostic biomarkers in Acute Lymphoblastic Leukemia (ALL) and should be carefully reconsidered. It is more accurate to state that while microRNAs and ABC genes may potentially influence treatment response in ALL, further research is crucial to fully understand their roles in determining treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d6c6bf058c1b51d999c0b567643ee6cccfac4f" target='_blank'>
              ATP‐Binding Cassette Transporter Genes and microRNAs in Pediatric B‐Cell ALL: Expression Insights
              </a>
            </td>
          <td>
            Reza Nekouian, Pooya Faranoush, Fatemeh Khesali, Parisa Shams, M. Foroughi‐Gilvaee, Negin Sadighnia, Dorsa Fallah Azad, MohammadAli Ehsani, Mohammad Faranoush
          </td>
          <td>2024-12-23</td>
          <td>Journal of Clinical Laboratory Analysis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT The discovery at the end of the 20th century of genes that induce cell death revolutionised the biocontaintment of genetically manipulated bacteria for environmental or agricultural applications. These bacterial ‘killer’ genes were then assayed for their potential to target and control malignant cells in human cancers. The identification of the bacteriomes in different human organs and tissues, coupled with the observation that bacteria tend to accumulate near tumours, has opened new avenues for anti‐cancer strategies. This progress, along with recent insights into how cancer cells evade immune response, has prompted innovative therapeutic approaches. Tumour microenvironments are typically nutrient‐rich, characterised by low oxygen tensions and very resistant to immune responses. Two recent studies in MBT highlight the promise of using Salmonella typhimurium and Escherichia coli as vectors in novel cancer treatments. Engineered S. typhimurium strains can generate adjuvant flagellin‐antigen complexes that function as in situ vaccines, hence increasing the immunogenic responses within tumour environment. Similarly, gut E. coli can be used as vectors to targert tumour cells in colon cancer, enabling both diagnostic applications and localised treatments. Both approaches hold significant potential to improve patient survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bba9ffcefc81fe9e0a146a68f1324d903a6bef1" target='_blank'>
              Bacteria as Precision Tools for Cancer Therapy
              </a>
            </td>
          <td>
            C. Michán, José Prados, Juan-Luis Ramos
          </td>
          <td>2025-01-01</td>
          <td>Microbial Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Cellular differentiation is controlled by intricate layers of gene regulation, involving the modulation of gene expression by various transcriptional regulators. Due to the complexity of gene regulation, identifying master regulators across the differentiation trajectory has been a longstanding challenge. To tackle this problem, a computational framework, single‐cell Boolean network inference and control (BENEIN), is presented. Applying BENEIN to human large intestinal single‐cell transcriptome data, MYB, HDAC2, and FOXA2 are identified as the master regulators whose inhibition induces enterocyte differentiation. It is found that simultaneous knockdown of these master regulators can revert colorectal cancer cells into normal‐like enterocytes by synergistically inducing differentiation and suppressing malignancy, which is validated by in vitro and in vivo experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19c7fb2686fa44d6df149bd4d9108d57834cc43d" target='_blank'>
              Control of Cellular Differentiation Trajectories for Cancer Reversion
              </a>
            </td>
          <td>
            Jeong-Ryeol Gong, Chun-Kyung Lee, Hoon-Min Kim, Juhee Kim, Jaeog Jeon, Sunmin Park, Kwang-Hyun Cho
          </td>
          <td>2024-12-11</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="DNA methylation is an important epigenetic mechanism that helps define and maintain cellular functions. It is influenced by many factors, including environmental exposures, genotype, cell type, sex, and aging. Since age is the primary risk factor for developing neurodegenerative diseases, it is important to determine if aging-related DNA methylation is retained when cells are reprogrammed to an induced Pluripotent Stem Cell (iPSC) state. Here, we selected peripheral blood mononuclear cells (PBMCs; n = 99) from a cohort of diverse and healthy individuals enrolled in the Genetic and Epigenetic Signatures of Translational Aging Laboratory Testing (GESTALT) study to convert to iPSCs. After reprogramming we evaluated the resulting iPSCs for DNA methylation signatures to determine if they reflect the confounding factors of age and environmental factors. We used genome-wide DNA methylation arrays in both cell types to show that the epigenetic clock is largely reset to an early methylation age after conversion of PBMCs to iPSCs. We further examined the epigenetic age of each cell type using an Epigenome-wide Association Study (EWAS). Finally, we identified a set of methylation Quantitative Trait Loci (methQTL) in each cell type. Our results show that age-related DNA methylation is largely reset in iPSCs, and each cell type has a unique set of methylation sites that are genetically influenced. Graphical Abstract Highlights Generation of a population-level set of iPSC lines from healthy individuals across the lifespan Aging-related features were reset based on epigenetic markers of cytosine methylation and telomere length By comparing methQTLs in iPSC vs. their donor PBMCs, we find that detection of methQTLs reflect biological functions of different cell types">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/399ecd8f3690914bdcf9c8d820f3abd8488f2bee" target='_blank'>
              Characterization of DNA methylation in PBMCs and donor-matched iPSCs shows methylation is reset during stem cell reprogramming
              </a>
            </td>
          <td>
            X. Reed, Cory A. Weller, S. Sáez-Atienzar, A. Beilina, Sultana Solaiman, Makayla K Portley, Mary Kaileh, Roshni Roy, Jinhui Ding, A. Moore, D. T. Whitaker, Bryan J. Traynor, J. R. Gibbs, Sonja W. Scholz, Mark R. Cookson
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background/Objectives: Treatment with tyrosine kinase inhibitors (TKIs) in chronic myeloid leukemia (CML) has revolutionized disease management and has transformed CML from a life-threatening disease to a chronic condition for many patients. However, overcoming resistance, particularly related to leukemic stem cells (LSC) that can persist even when the bulk of the leukemic cells are eliminated, remains a significant challenge. Methods: K562 and KU812 cell lines were treated in vitro with the TKI Imatinib (IM). Gene expression, protein analysis, and metabolomic screening were conducted to investigate the ability of the drug to enhance stem cell (SC) features. Moreover, a gene ontology analysis was performed on different available datasets, to further consolidate our data. Results: 48 h of IM treatment can significantly increase the expression of genes related to SC self-renewal, particularly SOX2 and OCT 3/4. Interestingly, these modulations occur in cells that remain alive after drug treatment and that displayed features consistent with leukemia stem-like CML cells, suggesting that SC genes levels are crucial even in cell population survived upon TKI treatment. Moreover, after in silico analysis of available data, we observed an enrichment of SOX2/NANOG and OCT 3/4 signatures after TKI treatment, thus strengthening our results. Conclusions: Our results confirmed the relevance of LSC features after TKI treatment, highlighting the need for more effective and potentially curative strategies targeting LSCs to overcome resistance in CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28fb6d29d24fd74f86c83c9e97c902987037fdab" target='_blank'>
              Tyrosine Kinase Inhibitor Therapy Enhances Stem Cells Profile and May Contribute to Survival of Chronic Myeloid Leukemiastem Cells
              </a>
            </td>
          <td>
            Simone Rocco, A. Maglione, Valentina Schiavo, Alessandro Ferrando, Carmen Fava, D. Cilloni, Barbara Pergolizzi, C. Panuzzo
          </td>
          <td>2025-01-01</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Currently, numerous associations between genetic polymorphisms and various diseases have been characterized through Genome-Wide Association Studies. The majority of clinically significant polymorphisms are localized in non-coding regions of the genome. While modern bioinformatic resources make it possible to predict molecular mechanisms that explain the influence of non-coding polymorphisms on gene expression, such hypotheses require experimental verification. This review discusses the methods for elucidating the molecular mechanisms underlying the dependence of disease pathogenesis on specific genetic variants within non-coding sequences. A particular focus is on the methods to identify the transcription factors with binding efficiency contingent upon polymorphic variations. Despite remarkable progress in the bioinformatic resources enabling the prediction of the impact of polymorphisms on disease pathogenesis, the imperative for experimental approaches to this inquiry still persists.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0e300f57318f3e11948f7c4bd85a221591ff36c" target='_blank'>
              Methods for Functional Characterization of Genetic Polymorphisms of Non-Coding Regulatory Regions of the Human Genome (Review)
              </a>
            </td>
          <td>
            A. Uvarova, E. A. Tkachenko, E. M. Stasevich, E. A. Zheremyan, K. Korneev, D. V. Kuprash
          </td>
          <td>2024-12-25</td>
          <td>Biohimiâ</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d1fd954b718bc92e2f3d26e9d0a2654e108d08" target='_blank'>
              Prostate cancer epigenetics - from pathophysiology to clinical application.
              </a>
            </td>
          <td>
            V. Constâncio, J. Lobo, J. P. Sequeira, Rui Henrique, Carmen Jerónimo
          </td>
          <td>2025-01-16</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103b75e315d591fa4510717dd10f578400ede160" target='_blank'>
              Central control of dynamic gene circuits governs T cell rest and activation
              </a>
            </td>
          <td>
            Maya M. Arce, Jennifer M. Umhoefer, Nadia Arang, Sivakanthan Kasinathan, Jacob W. Freimer, Zachary Steinhart, Haolin Shen, Minh T N Pham, Mineto Ota, Anika Wadhera, Rama Dajani, Dmytro Dorovskyi, Y. Chen, Qi Liu, Yuan Zhou, D. Swaney, K. Obernier, B. Shy, J. Carnevale, Ansuman T. Satpathy, N. Krogan, Jonathan K. Pritchard, Alex Marson
          </td>
          <td>2024-12-11</td>
          <td>Nature</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Aggressive variant prostate cancer (AVPC) is characterized by a molecular signature involving combined defects in TP53, RB1, and/or PTEN (AVPC-TSGs), identifiable through immunohistochemistry or genomic analysis. The reported prevalence of AVPC-TSG alterations varies widely, reflecting differences in assay sensitivity, treatment pressure, and disease stage evolution. Although robust clinical evidence is still emerging, the study of AVPC-TSG alterations in prostate cancer (PCa) is promising. Alterations in TP53, RB1, and PTEN, as well as the combined loss of AVPC-TSGs, may have significant implications for prognosis and treatment. These biomarkers might help predict responses to various therapies, including hormonal treatments, cytotoxic agents, radiotherapy, and targeted therapies. Understanding the impact of these molecular alterations in patients with PCa is crucial for personalized management. In this review, we provide a comprehensive overview of the emerging prognostic and predictive roles of AVPC-TSG alterations across PCa stages. Moreover, we discuss the implications of different methods used for detecting AVPC-TSG alterations and summarize factors influencing their prevalence. As our comprehension of the genomic landscape of PCa disease deepens, incorporating genomic profiling into clinical decision making will become increasingly important for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc76168bc6cc3f716d4ad9c939472b94406c6f4f" target='_blank'>
              The Emerging Predictive and Prognostic Role of Aggressive-Variant-Associated Tumor Suppressor Genes Across Prostate Cancer Stages
              </a>
            </td>
          <td>
            Martino Pedrani, Jessica Barizzi, Giuseppe Salfi, Alessandro Nepote, Irene Testi, Sara Merler, Luis Castelo-Branco, R. P. Mestre, Fabio Turco, Luigi Tortola, Jean-Philippe P. Theurillat, S. Gillessen, U. Vogl
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma.
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff4ae0eaa9128342a0dedd747e19622d92c7f71" target='_blank'>
              FDFT1 maintains glioblastoma stem cells through activation of the Akt pathway
              </a>
            </td>
          <td>
            Hui Mo, Jiajia Shao, Zhun Li, Peiting Zeng, Xinke Yin, Yongsheng Huang, Peng Wang, Jianwei Liao
          </td>
          <td>2024-12-20</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cells display distinct, recurrent phenotypic cell states. Metastatic spreading correlates with tumor cell state evolution. However, the molecular mechanisms underlying metastasis remain elusive. Here, we demonstrate that the quantitative dosage of oncogenic KRAS drives lung adenocarcinoma progression and metastasis via the integration of external signaling and pioneer transcription factor dynamics into qualitative cell states. Combining mouse models, in vivo CRISPR activation screens, and fate mapping, we show that even mild transcriptional amplification of KRAS significantly fuels tumor progression and metastasis. Chromatin profiling and transcriptomics reveal that high and low KRAS dosages supersede and integrate inflammatory and TGFβ signaling to dictate mouse cancer cell states. Patient data show that KRAS dosages correlate with distinct survival outcomes, transcription factor activity, and cell states. Direct KRAS inhibition in xenografts limits the KRAS-high “proliferative” cell state but spares a minimal residual state mimicking the KRAS-low “ciliated-like” state. Thus, oncogenic KRAS dosage fuels tumor heterogeneity at the cell state level and drives a bimodal tumor evolution during metastasis, with implications for prognosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/688e49a0ef57c8cb9ef7efb0f9083ef2faa0f479" target='_blank'>
              Transcriptional Dosage of Oncogenic KRAS Drives Lung Adenocarcinoma Cell States, Progression and Metastasis
              </a>
            </td>
          <td>
            M. Serresi, Ali Osman Çetin, Yuliia Dramaretska, Sonia Kertalli, M. Schmitt, Heike Naumann, Maria Zschummel, Marie Liesse-Labat, Lucas F. Maciel, Jeroen Declercq, J. Marine, G. Gargiulo
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Epigenetic dysregulation is a hallmark of Acute Myeloid Leukemia (AML), with mutations in DNA Methyltransferases (e.g., DNMT3A) being frequent and promising therapeutic targets. DNMTs form complexes with Histone Methyltransferases (HMTs), driving gene silencing loop via chromatin methylation crosstalk. However, potential connections between this DNMTs/HMTs cooperative activity and oncogenic requirements across the AML mutational spectrum remain poorly understood. Here, we demonstrate that AMLs carrying DNMT3A and Nucleophosmin (NPM1) mutations exhibit a specific epigenetic vulnerability toward a complex formed by DNMTs and G9a, a specific histone H3 Lysine 9 Methyltransferase (H3K9-HMT). Dual inhibition of DNMT/G9a restores differentiation, reduces tumor growth, and spares healthy progenitors compared to standard hypomethylating agents. Mechanistically, DNMT/G9a regulates NPM1 stability, inhibits HOXA9/MEIS1 activity, and triggers interferons (IFN) response via viral mimicry pathways by modulating hypermethylated retrotransposons. Collectively, our data unravel specific epigenetic vulnerabilities within the complex AML mutational landscape and provide a compelling rationale for the design of personalized epigenetic therapies with enhanced efficacy and safer clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33522a862f647024962f40881508708e0269adea" target='_blank'>
              DNMT/G9a Complex Inhibition Uncovers Epigenetic Vulnerabilities and Induces IFN-Response in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Angelique Schönefeld, Costanza Zanetti, Eric C. Schmitt, Kevin Woods, Ashley Westerback, Sidney M. Mitsch, Frank Rühle, Simge Kelekçi, Lisa Fol, Viral Shah, Miriam Sanchez-Saez, Sophia Thevissen, Alexander Schäffer, Ulrike Regina Schmits, D. Sasca, Thomas Kindler, Catherine Wölfel, Clarissa Feuerstein-Akgoz, Clarissa Gerhäuser, P. Lutsik, J. Rivière, J. Hecker, Katharina S Götze, U. Platzbecker, H. Echchannaoui, Matthias Theobald, C. Plass, Jan Padeken, H. Medyouf, B. Guezguez
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, , M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Colorectal cancer (CRC) has increased at an alarming rate amongst younger (< 50 years) individuals. Such early-onset colorectal cancer (EOCRC) has been particularly notable within the Hispanic/Latino population. Yet, this population has not been sufficiently profiled in terms of two critical elements of CRC -- the MYC proto-oncogene and WNT signaling pathway. Here, we performed a comprehensive multi-omics analysis on 30 early-onset and 37 late-onset CRC ([≥] 50 years) samples from Hispanic/Latino patients. Our analysis included DNA exome sequencing for somatic mutations, somatic copy number alterations, and global and local genetic similarity. Using RNA sequencing, we also assessed differential gene expression, cellular pathways, and gene fusions. We then compared our findings from early-onset Hispanic/Latino patient samples with publicly available data from Non-Hispanic White cohorts. Across all early-onset patients, which had a median 1000 Genomes Project Peruvian-in-Lima-like (1KG-PEL-like) genetic similarity proportion of 60%, we identified 41 WNT pathway genes with significant mutations. Six important examples were APC, TCF7L2, DKK1, DKK2, FZD10, and LRP5. Notably, patients with mutations in DKK1 and DKK2 had the highest 1KG-PEL-like proportion (79%). When we compared the Hispanic/Latino cohort to the Non-Hispanic White cohorts, four of these key genes -- DKK1, DKK2, FZD10, and LRP5 -- were significant in both risk association analyses and differential gene expression. Interestingly, early-onset tumors (vs. late-onset) exhibited distinct somatic copy number alterations and gene expression profiles; the differences included MYC and drug-targetable WNT pathway genes. We also identified a novel WNT gene fusion, RSPO3, in early-onset tumors; it was associated with enhanced WNT signaling. This integrative analysis underscores the distinct molecular features of EOCRC cancer in the Hispanic/Latino population; reveals potential avenues for tailored precision medicine therapies; and emphasizes the importance of multi-omics approaches in studying colorectal carcinogenesis. We expect this data to help contribute towards reducing cancer health disparities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62d0668a8b4dd0379f87ebaa20ad3d4dabb63bc8" target='_blank'>
              Assessment of MYC Gene and WNT Pathway Alterations in Early-Onset Colorectal Cancer Among Hispanic/Latino Patients Using Integrated Multi-Omics Approaches
              </a>
            </td>
          <td>
            F. Carranza, B. Waldrup, Y. Yin, Y. Amzaleg, M. Postel, D. Craig, J. Carpten, B. Salhia, D. Hernandez, N. Gutierrez, C. Ricker, J. Culver, C. Chavez, M. Stern, L. Baezconde-Garbanati, H.-J. Lenz, E. Velazquez-Villarreal
          </td>
          <td>2024-12-08</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba737d66daf461c8c86cfa467ed3f95f6246b1d2" target='_blank'>
              Targeting BRCA1-deficient PARP inhibitor-resistant cells with nickases reveals nick resection as a cancer vulnerability.
              </a>
            </td>
          <td>
            Jenna M Whalen, Jillian Earley, C. Wisniewski, A. Mercurio, Sharon B Cantor
          </td>
          <td>2025-01-21</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A significant number of castration-resistant prostate cancer (CRPC) evolve into a neuroendocrine (NE) subtype termed NEPC, leading to resistance to androgen receptor (AR) pathway inhibitors and poor clinical outcomes. Through Hi-C analyses of a panel of patient-derived xenograft tumors, here we report drastically different 3D chromatin architectures between NEPC and CRPC samples. Such chromatin re-organization was faithfully recapitulated in vitro on isogenic cells undergoing NE transformation (NET). Mechanistically, neural transcription factor (TF) NKX2-1 is selectively and highly expressed in NEPC tumors and is indispensable for NET across various models. NKX2-1 preferentially binds to gene promoters, but it interacts with chromatin-pioneering factors such as FOXA2 at enhancer elements through chromatin looping, further strengthening FOXA2 binding at NE enhancers. Conversely, FOXA2 mediates regional DNA demethylation, attributing to NE enhancer priming and inducing NKX2-1 expression, forming a feed-forward loop. Single-cell multiome analyses of isogenic cells over time-course NET cells identify individual cells amid luminal-to-NE transformation, exhibiting intermediate epigenetic and transcriptome states. Lastly, NKX2-1/FOXA2 interacts with, and recruits CBP/p300 proteins to activate NE enhancers, and pharmacological inhibitors of CBP/p300 effectively blunted NE gene expression and abolished NEPC tumor growth. Thus, our study reports a hierarchical network of TFs governed by NKX2-1 in regulating the 2D and 3D chromatin re-organization during NET and uncovers a promising therapeutic approach to eradicate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f60ea05e69028e7c615b1e7620f9deafcd0cdd" target='_blank'>
              Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, Juan Wang, Q. Jin, Peng Jin, Feng Yue, Martin G. Sanda, Victor Corces, Nicolas Altemose, Jonathan C. Zhao, Jindan Yu
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is the most prevalent type of malignant epithelial neoplasm that affects the oral cavity. It has been a significant health concern in many countries for a long time since it was usually treated with surgery, radiation, and/or chemotherapy. Drug resistance is the primary issue in patient populations and scientific research, which promotes OSCC tumour cell invasion and migration. Thus, identifying highly specific therapeutic targets could be the potential approach for more successful treatment of OSCC. It is still challenging to understand the genetic causes of oral carcinogenesis due to its highly varied clinic-pathological parameters. It is important to remember that signaling channels and complexes that affect chromatin accessibility control gene expression, which in turn affects cell development and differentiation. Histones undergo post-translational alteration to give this platform. Understanding the processes of gene regulation through histone methylation and its modifications could enhance the early detection, prognostic prediction, and therapy of OSCC. To be properly used as a therapeutic target, histone methylation in OSCC requires more investigation. This review details the dysregulated histone methylation and the modifying enzymes linked to the development and aetiology of OSCC. Furthermore, the part that lysine methylation plays in cell migration, chemo-resistance, and OSCC invasion is also investigated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ac8fe27c84ad3ea1843eef61f7ad80802b8dbb" target='_blank'>
              Molecular insight into histone methylation as a novel target for oral squamous cell carcinoma: future hope in personalised medicine
              </a>
            </td>
          <td>
            P. Rajendran, Monisha Prasad, Enas M. Ali, Ramya Sekar, Abdullah M AlZahrani, M. Karobari, M. A. M. Genena, B. Abdallah
          </td>
          <td>2025-01-27</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Colon cancer is the second most common cause of cancer death worldwide. Despite advances in the development of new molecular strategies for stratifying patients with colon cancer, many of these patients do not respond adequately to the standard of care. While previous studies have focused on the development of prognostic gene expression signatures, the exploration of predictive signatures to inform treatment decisions remains incomplete. In this study, we leveraged public gene expression datasets to design and experimentally validate a 37-gene expression signature for prognosis in colon cancer patients. We obtained a C-index of 0.732 (0.610-0.853) in four independent studies. Specifically, we discovered that the signature is associated with the mitotic phase of the cell cycle. Furthermore, the signature identified a population of colon cancer patients sensitive to tubulin inhibitor drugs. In particular, we validated in vitro and in vivo the efficacy of paclitaxel, a commonly used tubulin inhibitor in breast cancer treatment, in patient-derived preclinical models. These results highlight the importance of incorporating gene expression signatures to identify new therapeutic options for colon cancer treatment. Furthermore, the identification of alternative treatment options with potentially improved efficacy holds promise for the development of new clinical trials, and reshapes the biomarker-based treatment strategy for second line and refractory colon cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec4c73cdc7ad79204cf105e9ddd7970b91e50e10" target='_blank'>
              Transcriptomic signature reveals sensitivity to tubulin inhibitors in colon cancer
              </a>
            </td>
          <td>
            J. Liñares-Blanco, I. Chicote, Javier Ros, Anna M Alcántara, Jordi Martínez-Quintanilla, E. Élez, Héctor G Palmer, Carlos Fernandez-Lozano, Jose A Seoane
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="An important direction in cancer prevention is the development of novel strategies that can be used in the treatment by affecting the mechanisms of the regulatory pathways and genomic elements involved in malignant transformation. Strategy proposed in this study is to initiate cell death mechanisms in response to the depletion of the energy resources within a cancer cell due to the uncontrolled spread of mobile genetic elements throughout its genome. The calculation results obtained using a preliminary mathematical model that simulates cellular bioenergetic balance, taking into account the energy consumption for retrotransposition of the mobile elements such as LINE-1 and SINE, show that this scenario is plausible. Intracellular resources undergo a critical redistribution when affecting the genomic defense mechanisms and the LINE-1 transcription rate. This leads to a sharp increase in energy consumption for retrotransposon transcription that causes a significant decrease in the pool of free ATPs in the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05095e01db5773f04351486017740af52565a16e" target='_blank'>
              Control of the Activity of Mobile Elements in Cancer Cells as a Strategy for Anticancer Therapy
              </a>
            </td>
          <td>
            S. R. Pavlov, V. V. Gursky, M. Samsonova, A. Kanapin, A. A. Samsonova
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e31d8028a34805d9bd64c4990406de1eae29e1a1" target='_blank'>
              Age-related genomic alterations and chemotherapy sensitivity in osteosarcoma: insights from cancer genome profiling analyses
              </a>
            </td>
          <td>
            Hidetatsu Outani, Masachika Ikegami, Yoshinori Imura, Shou Nakai, Haruna Takami, Yuki Kotani, A. Inoue, S. Okada
          </td>
          <td>2024-12-17</td>
          <td>International Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c580172e55394858107b5db74bf7f577e6d7f63" target='_blank'>
              DNMT1 prolonged absence is a tunable cellular stress that triggers cell proliferation arrest to protect from major DNA methylation loss
              </a>
            </td>
          <td>
            Salvatore Martino, Serena Gargano, P. Carollo, A. Di Leonardo, V. Barra
          </td>
          <td>2024-12-18</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc6bfc5e3065b396e263e4793dde2e7908cde4e" target='_blank'>
              CRISPR-StAR enables high-resolution genetic screening in complex in vivo models.
              </a>
            </td>
          <td>
            E. Uijttewaal, Joonsun Lee, Annika Charlotte Sell, Naomi Botay, Gintautas Vainorius, Maria Novatchkova, J. Baar, Jiaye Yang, Tobias Potzler, Sophie van der Leij, Christopher Lowden, Julia Sinner, Anais Elewaut, Milanka Gavrilovic, Anna Obenauf, Daniel Schramek, Ulrich Elling
          </td>
          <td>2024-12-16</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Understanding the role of cancer stemness in predicting breast cancer (BRCA) response to radiotherapy is crucial for optimizing treatment outcomes. This study developed a stemness-based signature to identify BRCA patients who are likely to benefit from radiotherapy.Gene expression data for BRCA patients were obtained from the TCGA and METABRIC databases, including 920 TCGA-BRCA and 1980 METABRIC-BRCA patients. Univariate and multivariate Cox regression analyses were used to construct a radiosensitivity signature. Immune cell infiltration and pathway enrichment analyses were conducted using ESTIMATE and GSVA methods. The TIDE algorithm and the pRRophetic platform were employed to predict responses to radiotherapy. Radioresistant BRCA cells were examined using a colony formation assay. Key genes identified in the radiosensitivity signature were validated in vitro by qRT-PCR.By analyzing gene expression data from 920 BRCA samples, we identified a set of 267 stemness-related genes between high and low mRNAsi groups. Based on these genes, a radiosensitivity signature comprising two stemness-related genes (EMILIN1 and CYP4Z1) was constructed, stratifying patients into radiosensitive (RS) and radioresistant (RR) groups. Radiotherapy within the RS group significantly improved prognosis compared to non-radiotherapy patients. This signature was further validated in the METABRIC dataset. Notably, patients in the RS group also exhibited a significantly better response to immunotherapy compared to the RR group. We established a radioresistant BRCA cell line using the MCF-7 breast cancer cell line. A radioresistant breast cancer cell line (MCF-7/IR) was established by progressive exposure to increasing radiation doses. Comparative clonogenic and CCK8 assays demonstrated a radioresistant phenotype in the MCF-7/IR compared to MCF-7. In vitro studies utilizing both the MCF-7/IR and MCF-7 cell lines validated the expression of two radiosensitivity genes.This study identified a stemness-related gene signature predictive of radiosensitivity in breast cancer. This signature may guide personalized treatment strategies and inform the development of novel radiosensitizing agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57f78c9a265d652ec7795b69f721a283a62407ae" target='_blank'>
              Stemness-related gene signatures as a predictive tool for breast cancer radiosensitivity
              </a>
            </td>
          <td>
            Jinzhi Lai, Rongfu Huang, Jingshan Huang
          </td>
          <td>2025-01-31</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d0967e94330094078ab60dc71040acbf79f1f7c" target='_blank'>
              Targeting LMO2-induced autocrine FLT3 signaling to overcome chemoresistance in early T-cell precursor acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            C. Tremblay, J. Saw, Feng Yan, Jacqueline A Boyle, Ovini Amarasinghe, Shokoufeh Abdollahi, Anh Vo, Benjamin J Shields, Chelsea Mayoh, H. McCalmont, K. Evans, Anna Steiner, Kevin Parsons, M. McCormack, David R Powell, Nick Wong, Stephen M. Jane, Richard B Lock, David J Curtis
          </td>
          <td>2025-01-23</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d182842a9c5b9ec9f888e5c17ac019a50f732fb3" target='_blank'>
              Beginning at the ends: telomere and telomere-based cancer therapeutics.
              </a>
            </td>
          <td>
            Zahra Sadr, Masoumeh Ghasemi, Soheyla Jafarpour, Reyhaneh Seyfi, Aida Ghasemi, Elham Boustanipour, H. Khorshid, Naeim Ehtesham
          </td>
          <td>2024-12-06</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6666f30aff9bcd1de0a39ac7dc32869d01658a2" target='_blank'>
              Integrated analysis of single-cell and bulk-RNA sequencing for the cellular senescence in prognosis of lung adenocarcinoma
              </a>
            </td>
          <td>
            Fengqiang Yu, Liangyu Zhang, Xun Zhang, Jianshen Zeng, Fancai Lai
          </td>
          <td>2025-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Discovered more than four decades ago, nonsense-mediated mRNA decay (NMD) plays a fundamental role in the regulation of gene expression and is a major contributor to numerous diseases. With advanced technologies, several novel approaches aim to directly circumvent the effects of disease-causing frameshift and nonsense mutations. Additional therapeutics aim to globally dampen the NMD pathway in diseases associated with pathway hyperactivation, one example being Fragile X Syndrome. In other cases, therapeutics have been designed to hijack or inhibit the cellular NMD machinery to either activate or obviate transcript-specific NMD by modulating pre-mRNA splicing. Here, we discuss promising approaches employed to regulate NMD for therapeutic purposes and highlight potential challenges in future clinical development. We are optimistic that the future of developing target-specific and global modulators of NMD (inhibitors as well as activators) is bright and will revolutionize the treatment of many genetic disorders, especially those with high unmet medical need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8d79dce5cfe95746b7b429088778df04c8485b" target='_blank'>
              Exploring the therapeutic potential of modulating nonsense-mediated mRNA decay.
              </a>
            </td>
          <td>
            Mary McMahon, L. Maquat
          </td>
          <td>2024-12-12</td>
          <td>RNA</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Philadelphia chromosome-like B-cell acute lymphoblastic leukemia (Ph-like ALL) is driven by genetic alterations that induce constitutive kinase signaling and is associated with chemoresistance and high relapse risk in children and adults. Preclinical studies in the most common CRLF2-rearranged/JAK pathway-activated Ph-like ALL subtype have shown variable responses to JAK inhibitor-based therapies, suggesting incomplete oncogene addiction and highlighting a need to elucidate alternative biologic dependencies and therapeutic vulnerabilities, while the ABL-class Ph-like ALL subtype appears preferentially sensitive to SRC/ABL- or PDGFRB-targeting inhibitors. Which patients may be responsive versus resistant to tyrosine kinase inhibitor (TKI)-based precision medicine approaches remains a critical knowledge gap. Using bulk and single-cell multiomics analyses, we profiled residual cells from CRLF2-rearranged or ABL1-rearranged Ph-like ALL patient-derived xenograft models treated in vivo with targeted inhibitors to identify TKI-resistant subpopulations and potential mechanisms of therapeutic escape. We detected a specific MYC dependency in Ph-like ALL cells and defined a new leukemia cell subpopulation with senescence-associated stem cell-like features regulated by AP-1 transcription factors. This dormant ALL subpopulation was effectively eradicated by dual pharmacologic inhibition of BCL-2 and JAK/STAT or SRC/ABL pathways, a clinically-relevant therapeutic strategy. Single cell-derived molecular signatures of this senescence and stem/progenitor-like subpopulation further predicted poor clinical outcomes associated with other high-risk genetic subtypes of childhood B-ALL and thus may have broader prognostic applicability beyond Ph-like ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b87bb41065d73fc1bc25a49641c54717955deab" target='_blank'>
              Targeting Senescent Stemlike Subpopulations in Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Yang-yang Ding, Jonathan H Sussman, Kellyn Madden, J. Loftus, Robert K Chen, C. D. Falkenstein, D. A. Barcenas Lopez, David A Hottman, Benjamin Mathier, Wenbao Yu, Jason Xu, Changya Chen, Chia-Hui Chen, Bing He, S. Bandyopadhyay, Zhan Zhang, DongGeun Lee, Hong Wang, Junmin Peng, Chi V. Dang, Kai Tan, S. Tasian
          </td>
          <td>2025-01-07</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma multiforme (GBM) is one of the deadliest and most heterogeneous forms of brain cancer, characterized by its resistance to conventional therapies. Within GBM, a subpopulation of slow-cycling cells, often linked to quiescence and stemness, plays a crucial role in treatment resistance and tumor recurrence. This study aimed to identify novel biomarkers associated with these slow-cycling GBM cells.


METHODS
We utilized The Cancer Genome Atlas (TCGA)-GBM dataset and presented the reproducible bioinformatics analysis for our results.


RESULTS
Our analysis highlighted Membrane-Associated Protein 17 (MAP17) as strongly associated with the slow-cycling phenotype. We found that the protein cargo MAP17 expression is related to mesenchymal signatures and stem cell-related pathways. Also, MAP17 was linked to a distinct metabolic profile, characterized by significant enrichment in pathways related to folate, zinc, and fatty acids. Moreover, the immune cell distribution analysis revealed that MAP17 correlates with key molecular immune processes, including interferon-gamma (IFN-γ) signaling and antigen presentation, as well as immunosuppressive cells like myeloid-derived suppressor cells (MDSCs) and macrophages. MAP17-high tumors also showed elevated expression of several immune checkpoint inhibitors, indicating an immunosuppressive microenvironment.


CONCLUSION
These findings provide insight into the role of MAP17 in quiescence, stemness, and immune evasion, positioning it as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15fc0b51c9baf70fb9322e4160d56c406b168bfc" target='_blank'>
              Glioblastoma Stem Cells: MAP17 as a Novel Predictive Biomarker and Therapeutic Target Associated with Quiescence and Immune Evasion.
              </a>
            </td>
          <td>
            S. Aghamiri, Rada Amin
          </td>
          <td>2025-01-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer, a complex and heterogeneous disease, continues to be a major global health concern. Despite advancements in diagnostics and therapeutics, the aggressive nature of certain cancers remain a significant challenge, necessitating a deeper understanding of the underlying molecular mechanisms driving their severity and progression. Cancer severity and progression depend on cellular properties such as cell migration, cell division, cell shape changes, and intracellular transport, all of which are driven by dynamic cellular microtubules. Dynamic properties of microtubules, in turn, are regulated by an array of proteins that influence their stability and growth. Among these regulators, End Binding (EB) proteins stand out as critical orchestrators of microtubule dynamics at their growing plus ends. Beyond their fundamental role in normal cellular functions, recent research has uncovered compelling evidence linking EB proteins to the pathogenesis of various diseases, including cancer progression. As the field of cancer research advances, the clinical implication of EB proteins role in cancer severity and aggressiveness become increasingly evident. This review aims to comprehensively explore the role of microtubule-associated EB proteins in influencing the severity and aggressiveness of cancer. We also discuss the potential significance of EB as a clinical biomarker for cancer diagnosis and prognosis and as a target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8a843dc90163f1664e05266a959467a1810e6f" target='_blank'>
              End Binding Proteins: Drivers of Cancer Progression.
              </a>
            </td>
          <td>
            Dhakshmi Sasankan, Renu Mohan
          </td>
          <td>2024-12-19</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Cell cycle dysregulation and the corresponding metabolic reprogramming play significant roles in tumor development and progression. CDK9, a kinase that regulates gene transcription and cell cycle, also induces oncogene transcription and abnormal cell cycle in AML cells. The function of CDK9 for gene regulation in AML cells requires further exploration. In this study, we knocked down the CDK9 to investigate its effects on the growth and survival of AML cells. Through RNA-seq analysis, we identified that in U937 cells CDK9 regulates numerous genes involved in proliferation and apoptosis, including mTOR, SREBF1, and Bcl-2. Furthermore, our results demonstrated that both CDK9 and FASN are crucial for the proliferation and survival of Kasumi-1 and U937 cells. Mechanistically, MCL1, c-Myc, and Akt/mTOR/SREBF1 may be critical factors and pathways in the combined therapy of NVP-2 and Orlistat. In summary, our study revealed that CDK9 and FASN are vital for maintaining AML cell survival and proliferation. Treatment with NVP-2 and Orlistat may be a promising clinical candidate for patients with AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/985b606d62bac6448e6bc281a807101bb710d0ae" target='_blank'>
              NVP-2, in combination with Orlistat, represents a promising therapeutic strategy for acute myeloid leukemia
              </a>
            </td>
          <td>
            Qing Zhu, Jia Cheng, Yuqing Gao, Zimu Zhang, Jian Pan, Xin Su, Danhong Fei, Linbo Cai, Juanjuan Yu, Yanling Chen, Wanyan Jiao, Di Wu, Xiaolu Li, Peifang Xiao
          </td>
          <td>2025-01-12</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda44ec94f8b435093cf034210fe1f5d7c4827e5" target='_blank'>
              Targeting MYC for the treatment of breast cancer: use of the novel MYC-GSPT1 degrader, GT19630.
              </a>
            </td>
          <td>
            Minhong Tang, J. Crown, Michael J Duffy
          </td>
          <td>2025-01-28</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The explosion of next-generation sequencing technologies has allowed researchers to move from studying single genes, to thousands of genes, and thereby to also consider the relationships within gene networks. Like others, we are interested in understanding how developmental and evolutionary forces shape the expression of individual genes, as well as the interactions among genes. To this end, we characterized the effects of genetic background and developmental environment on brain gene coexpression in two parallel, independent evolutionary lineages of Trinidadian guppies (Poecilia reticulata). We asked whether connectivity patterns among genes differed based on genetic background and rearing environment, and whether a gene’s connectivity predicted its propensity for expression divergence. In pursuing these questions, we confronted the central challenge that standard approaches fail to control the Type I error and/or have low power in the presence of high dimensionality (i.e., large number of genes) and small sample size, as in many gene expression studies. Using our data as a case study, we detail central challenges, discuss sample size guidelines, and provide rigorous statistical approaches for exploring coexpression differences with small sample sizes. Using these approaches, we find evidence that coexpression relationships differ based on both genetic background and rearing environment. We report greater expression divergence in less connected genes and suggest this pattern may arise and be reinforced by selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffb7f6929569fe684cb5a3f56de734a873153a7a" target='_blank'>
              FLEXIBILITY IN GENE COEXPRESSION AT DEVELOPMENTAL AND EVOLUTIONARY TIMESCALES
              </a>
            </td>
          <td>
            Eva K. Fischer, Youngseok Song, Wen Zhou, K. Hoke
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a heterogeneous malignant disease with high relapse and mortality rates. To characterise the multiomics features of T-ALL, we conducted integrative analyses using single-cell RNA, TCR and chromatin accessibility sequencing on pre- and post-treatment peripheral blood and bone marrow samples of the same patients. We found that there is transcriptional rewiring of gene regulatory networks in T-ALL cells. Some transcription factors, such as TCF3 and KLF3, showed differences in activity and expression levels between T-ALL and normal T cells and were associated with the prognosis of T-ALL patients. Furthermore, we identified multiple malignant TCR clonotypes among the T-ALL cells, where the clonotypes consisted of distinct combinations of the same TCR α and β chain per patient. The T-ALL cells displayed clonotype-specific immature thymocyte cellular characteristics and response to chemotherapy. Remarkably, T-ALL cells with an orphan TCRβ chain displayed the strongest stemness and resistance to chemotherapy. Our study provided transcriptome and epigenome characterisation of T-ALL cells categorised by TCR clonotypes, which may be helpful for the development of novel predictive markers to evaluate treatment effectiveness for T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35386d641d4da0518c18007554d1299175192fd" target='_blank'>
              Single-Cell Multiomics Reveals TCR Clonotype-Specific Phenotype and Stemness Heterogeneity of T-ALL Cells.
              </a>
            </td>
          <td>
            Songnan Sui, Xiaolei Wei, Yue Zhu, Qiuyue Feng, Xianfeng Zha, Lipeng Mao, Boya Huang, Wen Lei, Guobing Chen, Huien Zhan, Huan Chen, Ru Feng, Chengwu Zeng, Yangqiu Li, Oscar Junhong Luo
          </td>
          <td>2024-12-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="INTRODUCTION
Approximately one-third of all AML patients have a mutation in the Fms-like tyrosine kinase 3 (FLT3) gene, which is associated with a poor prognosis in these individuals. The 2017 approval of midostaurin, the first FLT3 inhibitor, spurred extensive development of more potent and selective inhibitors with an improved safety profile.


AREAS COVERED
This review analyzes patent inventions for the treatment of AML using FLT3 inhibitors, covering developments from the earliest to the most recent, disclosed in 2024. Our search using the global Espacenet database identified numerous compounds with low nanomolar inhibitory concentrations against FLT3-ITD and FLT3-TKD mutants. These compounds have shown promise in preclinical studies. Co-inhibition strategies and combinatorial therapies to overcome resistance and enhance anti-leukemic efficacy are also discussed.


EXPERT OPINION
Recent patents highlight advances in the field of FLT3 inhibitors with a focus on overcoming resistance, improving selectivity and potency. Future strategies may include third-generation inhibitors such as type III allosteric inhibitors, irreversible inhibitors, or PROTACs. Personalized medicine approaches utilizing genetic profiling to tailor therapies are emphasized. Exploration of novel combination regimens with emerging therapies like CAR T-cell therapy, immune checkpoint inhibitors and small molecules targeting critical AML pathways is ongoing to further enhance anti-leukemic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a10fb2e6d07356cabc94588ceb638fa353df881c" target='_blank'>
              Strategies for the treatment of acute myeloid leukemia with FLT3 mutations: a patent review.
              </a>
            </td>
          <td>
            L. Górecki, E. Řezníčková, V. Kryštof, Martina Rezacova, Martina Ceckova, J. Korábečný
          </td>
          <td>2024-12-24</td>
          <td>Expert opinion on therapeutic patents</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background/Objectives: AML is an aggressive malignant disease characterized by aberrant proliferation and accumulation of immature blast cells in the patient’s bone marrow. Chemotherapeutic treatment can effectively induce remission and re-establish functional hematopoiesis. However, many patients experience chemoresistance-associated relapse and disease progression with a poor prognosis. The identification of molecular determinants of chemoresistance that could serve as potential targets for the therapeutic restoration of chemosensitivity has proven to be challenging. Methods: To address this, we have analyzed longitudinal changes in the expression of microRNAs during disease progression in a small set of four AML patients, combined with gene ontology (GO) pathway analysis and evaluation of gene expression data in patient databases. Results: MicroRNA profiling of bone marrow samples at diagnosis and after relapse revealed significant differential expression of a large number of microRNAs between the two time points. Subsequent GO pathway analysis identified 11 signal transduction pathways likely to be affected by the differential miRNA signatures. Exemplary validation of the FoxO signaling pathway by gene expression analysis confirmed significant upregulation of FOXO1 and the target genes GADD45 and SOD2. Conclusions: Here, we show how a microRNA-based pathway prediction strategy can be used to identify differentially regulated signaling pathways that represent potential targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ad3edfd6af9770fc155f2d3924cf9e30fae927" target='_blank'>
              MicroRNA Screening Reveals Upregulation of FoxO-Signaling in Relapsed Acute Myeloid Leukemia Patients
              </a>
            </td>
          <td>
            Paula Reichelt, S. Bernhart, U. Platzbecker, Michael Cross
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9efc51b7707d06e4363cf205a25f12d35ff74015" target='_blank'>
              Coupling cellular drug-target engagement to downstream pharmacology with CeTEAM
              </a>
            </td>
          <td>
            Nicholas C. K. Valerie, Kumar Sanjiv, O. Mortusewicz, S. Zhang, Seher Alam, Maria J. Pires, Hannah Stigsdotter, A. Rasti, M. Langelier, D. Rehling, Adam Throup, Oryn Purewal-Sidhu, M. Desroses, Jacob Onireti, Prasad Wakchaure, I. Almlöf, J. Boström, Luka Bevc, Giorgia Benzi, P. Stenmark, J. Pascal, T. Helleday, B. Page, M. Altun
          </td>
          <td>2024-12-06</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>82</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d64147ddd85cdd3e6fab3569173028554ae2a51c" target='_blank'>
              Detection of early relapse in multiple myeloma patients
              </a>
            </td>
          <td>
            Tereza Růžičková, M. Vlachová, L. Pečinka, Monika Brychtová, M. Vecera, Lenka Radová, Simona Ševčíková, Marie Jarošová, J. Havel, Luděk Pour, Sabina Ševčíková
          </td>
          <td>2025-01-29</td>
          <td>Cell Division</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="INTRODUCTION
Acute Myeloid Leukemia is a heterogeneous hematological malignancy characterized by the uncontrolled proliferation of abnormal myeloid cells. Besides several other genetic abnormalities developed in AML, FLT3 mutations are significant due to their worse prognostic impacts and therapeutic resistance. As a result, these mutations enable AML cells to develop mechanisms for evading immune surveillance.


AREAS COVERED
This review discusses the ways of immune escape of FLT3-mutated AML cells. A literature search was conducted on PubMed, Scopus, and Web of Science databases, covering articles published between 2010 and 2024 with related keywords. The discussion covers AML cells' downregulation of immune recognition markers, expression of immune checkpoint proteins, and establishment of an immunosuppressive tumor microenvironment. Specific attention is given to small extracellular vesicles and their participation in immune escape. The focus is on exosome-mediated pathways and possible combination therapies.


EXPERT OPINION
FLT3 mutations in AML represent a formidable therapeutic challenge due to their crucial role in immune evasion. Exosomes are major players in these processes. Combination therapies targeting the exosome pathway could significantly improve these patients' immune recognition and overall outcomes. Understanding the underlying mechanisms, including targeted therapies, will be required to transcend existing therapeutic limitations and push newer strategies in treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282d1e496e2d0d1f77a779d23677b8f4e81093e1" target='_blank'>
              FLT3-mutated AML: Immune Evasion Through Exosome-Mediated Mechanisms and innovative combination therapies targeting Immune escape.
              </a>
            </td>
          <td>
            J. S. Mohamed, Waleed K Abdulsahib, Dilfuza Ashurova, Gaurav Sanghvi, S. Ballal, Rsk Sharma, Piyus Kumar Pathak, Shankhyan Aman, Abhinav Kumar, Fadhil Feez Sead, Mvnl Chaitanya
          </td>
          <td>2025-01-30</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Productive infections of oncogenic human papillomaviruses (HPVs) are closely linked to the differentiation of host epithelial cells, a process that the virus manipulates to create conditions favorable to produce virion progeny. This viral interference involves altering the expression of numerous host genes. Among these, proprotein convertases (PCs) have emerged as potential oncogenes due to their central role in cellular functions. Using RT-qPCR, aberrant PC gene expression was detected across the progression from early HPV infection stages to cancer. These findings demonstrated a progressive disruption of normal PC expression profiles, with FURIN consistently downregulated and other PCs upregulated, transitioning from the episomal stage to neoplastic and carcinoma phenotypes. This pattern of dysregulation was distinct from the broader trends observed in a variety of cancer types through bioinformatic analysis of publicly available transcriptomic datasets, where FURIN expression was predominantly upregulated compared to other PCs. Further bioinformatic investigations revealed a correlation between PC gene expression and cancer phenotype diversity, suggesting a potential link between the loss of normal PC gene expression patterns and the progression of HPV infections toward malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f01d2dc341bbc570cc91f7701d59ca7b10ccc365" target='_blank'>
              Dysregulation of FURIN and Other Proprotein Convertase Genes in the Progression from HPV Infection to Cancer
              </a>
            </td>
          <td>
            Gonzalo Izaguirre, Natalia Zirou, Craig Meyers
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="HIV-1 persists as a global health challenge due to its ability to establish latent reservoirs that evade eradication by antiretroviral therapy (ART). This study introduces a high-diversity barcoded HIV-1 library combined with a novel peptide barcode system to enable single-cell multi-dimensional analysis of viral integration, transcription, splicing, and translation of distinct viral mRNA isoform. Our method revealed over 80 distinct HIV-1 splicing isoforms and uncovered significant temporal variation in mRNA diversity, driven by integration site and chromatin context. By linking HIV-1 5’UTR with peptide barcoding, we quantified the translation efficiency of alternative 5′ UTRs and demonstrated correlations with abundance of each viral mRNA isoform. Additionally, we showed that latency reversal agents, such as SAHA and Bryostatin, selectively reactivate proviruses depending on their genomic and chromatin context and leads to distinct viral RNA splicing program. This platform revealed the diversity of HIV-infected cells at levels of transcripts as well as at proteins, and subsequently the fate determination. The detailed understanding of the stochastic diversity and dynamic nature of HIV-1 latency, inform the development of targeted therapies to eliminate latent reservoirs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b93e98c68b3422ec0ed93ef0f0fe8270c85d96" target='_blank'>
              Extreme high-diversity barcoded HIV library enables single cell multi-omics analysis of viral expression and fate determination
              </a>
            </td>
          <td>
            Tian-hao Zhang, Li Sheng, Xiangzhi Meng, Wenjin Lai, Yushen Du, Yuan Shi, Jing Wen, Masakazu Kamata, S. Chanda, Irvin S. Y. Chen, Ren Sun
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Acute myeloid leukemia (AML) is a hematologic tumor with poor prognosis and significant clinical heterogeneity. By integrating transcriptomic data, single-cell RNA sequencing data and independently collected RNA sequencing data this study aims to identify key genes in AML and establish a prognostic assessment model to improve the accuracy of prognostic prediction. Materials and methods We analyzed RNA-seq data from AML patients and combined it with single-cell RNA sequencing data to identify genes associated with AML prognosis. Key genes were screened by bioinformatics methods, and a prognostic assessment model was established based on these genes to validate their accuracy. Results The study identified eight key genes significantly associated with AML prognosis: SPATS2L, SPINK2, AREG, CLEC11A, HGF, IRF8, ARHGAP5, and CD34. The prognostic model constructed on the basis of these genes effectively differentiated between high-risk and low-risk patients and revealed differences in immune function and metabolic pathways of AML cells. Conclusion This study provides a new approach to AML prognostic assessment and reveals the role of key genes in AML. These genes may become new biomarkers and therapeutic targets that can help improve prognostic prediction and personalized treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fba51cf3073c5201e7804e4ac1e1ae1bb4b49f8" target='_blank'>
              Single-cell transcriptomics reveals heterogeneity and prognostic markers of myeloid precursor cells in acute myeloid leukemia
              </a>
            </td>
          <td>
            Guangfeng He, Lai Jiang, Xuancheng Zhou, Yuheng Gu, Jingyi Tang, Qiang Zhang, Qingwen Hu, Gang Huang, Ziye Zhuang, Xinrui Gao, Ke Xu, Yewei Xiao
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87eccc46e0a8b0d1964c263841b403eaf8af0f71" target='_blank'>
              CXCR4 promotes tumor stemness maintenance and CDK4/6 inhibitors resistance in ER-positive breast cancer
              </a>
            </td>
          <td>
            Qiangfeng Shi, Wang Yang, Yiye Ouyang, Yujie Liu, Zijie Cai
          </td>
          <td>2025-01-28</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states.
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Zachary Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P. Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdfee082d103369a456ad016f2f970300d872d89" target='_blank'>
              Pan-cancer and multiomics: advanced strategies for diagnosis, prognosis, and therapy in the complex genetic and molecular universe of cancer.
              </a>
            </td>
          <td>
            Camilly Victória Campanharo, Lívia Valle Dos Santos Silveira, D. D. Meira, Matheus Correia Casotti, Lorena Souza Castro Altoé, I. Louro, André Felipe Monteiro Gonçalves, André Manhães Machado, Breno Sousa Paiva, Ester de Souza Inocencio, Fabio Victor Vieira Rocha, Fellipe Pesente, Giulia de Souza Cupertino de Castro, João Pedro Dos Santos da Paixão, José Henrique Borges Bourguignon, Júlia Salarini Carneiro, Juliana Ribeiro de Oliveira, Pâmela de Souza Freire, Sophia Bridi Zamprogno, Taissa dos Santos Uchiya, Thais de Paula Rezende, Vinícius de Pádua Sanders Medeiros
          </td>
          <td>2024-12-26</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Motivation Allele-specific methylation (ASM) refers to differential DNA methylation patterns between two alleles at a given locus. This phenomenon is often driven by genetic variants, such as single nucleotide variants (SNVs), which influence methylation in cis by affecting transcription factor or methylation regulator binding, leading to allele-specific differences. Understanding ASM is critical for elucidating gene regulation, as it impacts gene expression and contributes to normal biological variation as well as disease processes, including cancer [1] and autoimmune disorders [2],[3]. Another key driver of ASM is genomic imprinting, an epigenetic mechanism in which gene expression is regulated in a parent-of-origin-specific manner. Imprinted regions, marked during gametogenesis and maintained through cell divisions, are essential for growth, development, and metabolism. Dysregulation of imprinting is associated with developmental and metabolic disorders, such as Prader-Willi and Angelman syndromes, and certain cancers. Detecting ASM across the genome remains challenging due to its tissue- and cell-specific nature and the technical difficulty of phasing reads to assign methylation patterns to specific alleles. Current ASM detection pipelines (e.g., [4])) often require phasing via genetic variants, a computationally intensive process that is limited in regions with low heterozygosity. Results To address these limitations, we developed asms (Allele-Specific Methylation Scanner), a tool designed to detect ASM directly from methylation data without the need for prior phasing. asms offers a faster and complementary approach to uncovering the regulatory effects of ASM, particularly in genomic regions where genetic variants or imprinting play a critical role. For demonstration purposes we leverage the fact that reads generated by Oxford Nanopore (ONT) technology measure sequence and methylation status at once, but the same software can be used with other sequencing technologies. asms can check thousands of loci in a short time. The initial list of loci to examine can be given by the user, or generated by asms through a genomic scan. The asms cluster subcommand separates the reads based on methylation. If phasing results are available, asms can use them to verify whether distinct alleles correspond to distinct base modifications patterns. We benchmark our software using publicly available Ashkenazi trio data [5]. Implementation and availability asms is implemented in rust and python. The software is available at https://github.com/ecmra/asms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7dfc00aae1ea08fcc56a90d449bd2018337af0" target='_blank'>
              ASMS: finding allele specific methylation in human genomes without phasing
              </a>
            </td>
          <td>
            E. Raineri, Miguel Ángel Esteve Marco, Anna Esteve Codina
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Clinical medicine and biological sciences make extensive use of the gene editing method symbolized by CRISPR-Cas9. With the rapid development of molecular biology technology in recent years, the CRISPR-Cas systemwhich stands for clustered regularly interspaced short palindromic repeatshas become a potent tool for gene editing. Due to its high efficiency, precision, and flexibility, it has surpassed last two generations in its application within the field of molecular biology. This article provides a detailed overview of the development history, working mechanism, and applications of this technology. The CRISPR-Cas system achieves specific site modifications by introducing insertions, deletions, or single base substitutions at particular genomic sites. It has made significant contributions to research and treatment in areas such as the therapies of tumors and genetic diseases. However, challenges such as off-target effects need to be addressed before its practical clinical utility can be rigorously validated through further research and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ea967093b688e91d8c17ab505cb62e377bf9fa" target='_blank'>
              Progress in human diseases treatment CRISPR system
              </a>
            </td>
          <td>
            Yuanrong Wu
          </td>
          <td>2024-12-26</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="A major challenge in the stem cell biology field is the ability to produce fully functional cells from induced pluripotent stem cells (iPSCs) that are a valuable resource for cell therapy, drug screening, and disease modelling. Here, we developed a novel inducible CRISPR-mediated activation strategy (iCRISPRa) to drive the expression of multiple endogenous transcription factors (TFs) important for in vitro cell fate and differentiation of iPSCs to haematopoietic progenitor cells. This work has identified a key role for IGFBP2 in developing haematopoietic progenitors. We first identified nine candidate TFs that we predicted to be involved in blood cell emergence during development, then generated tagged gRNAs directed to the transcriptional start site of these TFs that could also be detected during single-cell RNA sequencing (scRNAseq). iCRISPRa activation of these endogenous TFs resulted in a significant expansion of arterial-fated endothelial cells expressing high levels of IGFBP2, and our analysis indicated that IGFBP2 is involved in the remodelling of metabolic activity during in vitro endothelial to haematopoietic transition. As well as providing fundamental new insights into the mechanisms of haematopoietic differentiation, the broader applicability of iCRISPRa provides a valuable tool for studying dynamic processes in development and for recapitulating abnormal phenotypes characterised by ectopic activation of specific endogenous gene expression in a wide range of systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1ef294ac361f4462935c56ea7ed2968fcd856ba" target='_blank'>
              A novel human pluripotent stem cell gene activation system identifies IGFBP2 as a mediator in the production of haematopoietic progenitors in vitro
              </a>
            </td>
          <td>
            P. Petazzi, T. Ventura, Francesca Paola Luongo, Heather McClafferty, Alisha May, H. Taylor, Mike Shipston, Nicola Romanò, Lesley M. Forrester, Pablo Menéndez, A. Fidanza
          </td>
          <td>2024-12-23</td>
          <td>eLife</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="BACKGROUND
Acquired resistance to temozolomide (TMZ) chemotherapy due to DNA mismatch repair (MMR) enzyme deficiency is a barrier to improving outcomes for IDH wild-type glioblastoma (GBM) patients. KL-50 is a new imidazotetrazine-based therapeutic designed to induce DNA interstrand cross links, and subsequent double-stranded breaks, in an MMR-independent manner in cells with O-6-Methylguanine-DNA Methyltransferase (MGMT) deficiency. Previous research showed its efficacy against LN229 glioma cells with MMR and MGMT knockdown. Its activity against patient-derived GBM that model post-TMZ recurrent tumors is unclear.


METHODS
We created MMR-deficient GBM patient-derived xenografts through exposure to TMZ, followed by treatment with additional TMZ or KL-50. We also generated isogenic, MSH6 knockout patient-derived GBM and tested them for sensitivity to TMZ and KL-50.


RESULTS
KL-50 extended the median survival of mice intracranially engrafted with either patient-derived TMZ-naïve GBM6 or TMZ-naïve GBM12 by 1.75-fold and 2.15-fold, respectively (p<0.0001). A low dose (4 Gy) of fractionated RT further extended the survival of KL-50 treated GBM12 mice (median survival=80 days for RT+ KL-50 vs. 71 days KL-50 alone, P=0.018). KL-50 also extended the median survival of mice engrafted with post-TMZ, MMR-deficient GBM6R-m185 (140 days for KL-50 vs. 37 days for vehicle, p<0.0001). MSH6-KO increased TMZ IC50 for GBM6 and GBM12 cultures by >5-fold and >12-fold for cell death and live cell count outputs, respectively. In contrast, MSH6-KO actually decreased KL-50 IC50 by 10-80%.


CONCLUSION
KL-50-based compounds are a promising new strategy for the treatment of MGMT-deficient, MMR-deficient GBM that recurs after frontline TMZ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03f3a0f339bf4219c2fc489415b10b402bba9dd1" target='_blank'>
              The novel DNA cross-linking agent KL-50 is active against patient-derived models of new and recurrent post-temozolomide mismatch repair-deficient glioblastoma.
              </a>
            </td>
          <td>
            Matthew McCord, Thomas Sears, Wenxia Wang, Rahul Chaliparambil, Shejuan An, Jann Sarkaria, C. James, Bruce Ruggeri, Susan E. Gueble, R. Bindra, C. Horbinski
          </td>
          <td>2024-12-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9558944d48747d281f714408c8c756ade188a3" target='_blank'>
              Genetic and epigenetic characterization of sarcoma stem cells across subtypes identifies EZH2 as a therapeutic target
              </a>
            </td>
          <td>
            Edmond O'Donnell, Maria Muñoz, Ryan Davis, Jessica Bergonio, R. L. Randall, Clifford Tepper, Janai R. Carr-Ascher
          </td>
          <td>2025-01-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="CRISPR-Cas9, awarded the 2020 Nobel Prize in Chemistry for its discovery by Jennifer Doudna and Emmanuelle Charpentier, has revolutionized gene editing due to its versatility, precision, and efficiency. Unlike previous gene editing tools such as Zinc Finger Proteins (ZFP) and TALENs, CRISPR-Cas9 allows targeted editing of any DNA sequence with minimal off-target effects. This article explores the potential of CRISPR-Cas9 technology to edit telomeresprotective DNA structures at chromosome endsand related genes to address telomere-associated diseases and aging mechanisms. Telomeres, consisting of repetitive DNA sequences and associated proteins, are crucial in protecting chromosomes and regulating cellular aging, with their progressive shortening linked to cancer and other age-related diseases. This article mainly discusses the molecular mechanisms of telomere maintenance, the feasibility of using CRISPR-Cas9 for telomere length control, and its implications for cancer, stem cell research, and telomere-associated diseases. While promising, further research is needed to fully understand the roles of regulatory pathways and non-genetic factors in telomere dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c900abb6d0862046c94ca8ef8e27867352e34ebe" target='_blank'>
              Applications of CRISPR-Cas9 Technology in Editing Telomeres and Their Related Control Genes
              </a>
            </td>
          <td>
            Tianye Liu
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Periodontal disease is a widespread inflammatory condition that compromises the supporting structures of the teeth, potentially resulting in tooth loss if left untreated. Despite advancements in therapeutic interventions and an enhanced understanding of its pathophysiology, emerging techniques such as single-cell RNA sequencing (scRNA-seq) and Mendelian randomization (MR) present new opportunities for precision medicine in the management of periodontal disease. Methods Data derived from the GSE152042 dataset underwent rigorous quality control, normalization, and dimensionality reduction using Seurat and the MonacoImmuneData framework. Marker genes were identified to delineate subgroups for subsequent analysis utilizing CellChat and ClusterProfilerR. MR analysis of the expression quantitative trait loci (eQTLs) for these genes was conducted to determine causal relationships with periodontal disease, leveraging data from the IEU Open GWAS project. Results Single-cell analysis revealed distinct immune cell subtypes and indicated an increased presence of myeloid dendritic cells (mDCs) in patients with periodontal disease. MR analysis identified twenty-six significant genes, with LIMA1 (LIM domain and actin-binding 1) demonstrating a robust causal association with the progression of periodontal disease. Gene ontology and Kyoto Encyclopedia of Genes and Genomes analyses highlighted crucial pathways involved in periodontal inflammation and tissue destruction. Visualization at the single-cell level elucidated the role of LIMA1 in disease progression, alongside differences in cell communication dynamics between LIMA1-positive and -negative populations. Conclusion This study underscores the utility of scRNA-seq and MR in elucidating essential factors in the pathogenesis of periodontal disease, thereby reinforcing the necessity for targeted therapeutic strategies. The identification of LIMA1 as a pivotal gene in periodontal disease progression opens new avenues for precision medicine approaches, potentially enhancing treatment efficacy and patient outcomes in periodontal management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5112127a205674f57ff2162ed2050e8725facc14" target='_blank'>
              Myeloid dendritic cells and periodontal disease association: integrated study of single-cell sequencing and Mendelian randomization analysis
              </a>
            </td>
          <td>
            ChengJi Shi, Xinyi Ou, XiaoXu Lei, Lijuan Huang, Shuhao Xu, Wei Li, Xi Zhao
          </td>
          <td>2025-01-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Background: RAD51C, a critical member of the RAD51 paralog family, is essential for homologous recombination (HR)-mediated DNA repair, a pathway crucial for maintaining genomic stability. Mutations in RAD51C have been linked to cancer susceptibility, particularly in breast and ovarian cancers, where impaired DNA repair mechanisms contribute to genomic instability and tumor progression. Despite its clinical significance, the functional impact of specific RAD51C variants remains poorly understood, necessitating a comprehensive investigation into their biological implications. 
Methods: This study classified RAD51C gene variants into damaging and tolerant categories using computational prediction tools, including SIFT, PolyPhen, CADD, MetaLR, and Mutation Assessor. Variants were prioritized based on consensus scores and classified as high-confidence damaging variants. Correlation and agreement among tools were analyzed to refine predictions. Principal Component Analysis (PCA) and clustering methods were employed to group variants based on prediction patterns. Protein-protein interaction (PPI) networks and pathway enrichment analyses were conducted to contextualize damaging variants within broader biological systems, with a focus on their roles in HR, DNA repair, and cellular processes. 
Results: A total of 2526 variants were analyzed, with damaging variants showing consistent patterns across tools. Consensus scores highlighted 302 high-confidence damaging variants, which were associated with disrupted biological processes, including double-strand break repair via homologous recombination, telomere maintenance, and regulation of cell cycle checkpoints. PPI analysis revealed an interconnected network with 11 nodes and 54 edges, with a clustering coefficient of 0.982, indicating tightly coordinated interactions among DNA repair proteins. Pathway enrichment analyses identified significant associations with homologous recombination (FDR = 2.55E-17) and the Fanconi anemia pathway (FDR = 2.96E-06). 
Conclusion: This study provides a comprehensive framework for assessing the functional impacts of RAD51C variants by integrating computational predictions with biological analyses. The findings underscore the importance of RAD51C in HR and DNA repair pathways, offering insights into its role in genomic stability and cancer progression. These results can inform the prioritization of variants for experimental validation and guide therapeutic strategies targeting DNA repair deficiencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16240c1097223735006d241b04503aff291d133b" target='_blank'>
              Statistical Analysis of Gene Variants for Homologous Recombination Pathways of DNA Repair leading to Cancer Susceptibility
              </a>
            </td>
          <td>
            Usha Adiga, B. Jyoti, P. Reddemma, Alfred J. Augustine, S. Vasishta
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Statistics in Medical Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5cabef889301b036a508a4127fd265270ae7d40" target='_blank'>
              DNA hypermethylation of tumor suppressor genes among oral squamous cell carcinoma patients: a prominent diagnostic biomarker.
              </a>
            </td>
          <td>
            Nistha Agarwal, A. Jha
          </td>
          <td>2024-12-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b542469fcebc2361e69bec6d43fd73238a87bdc" target='_blank'>
              Abstract A009: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gene editing with CRISPR-Cas9 technology revolutionizes modern medicine by enabling precise DNA modifications. Initially discovered as a bacterial defense mechanism, the Cas9 enzyme, guided by RNA, can target and cut specific DNA sites, allowing for gene editing. Applications include genetic engineering, functional studies, and potential treatments for genetic diseases like cancer. Notably, in 2023, CRISPR-Cas9 was approved for treating sickle cell anaemia with significant results, despite challenges like long-term security and high costs. CRISPR-Cas9 technology allows for correcting genetic defects, treating diseases, and improving agricultural crops. It can regulate gene transcription through the CRISPRi system, using an inactive Cas9 to interfere with gene expression without permanently altering DNA. This gene-editing tool shows promise in gene therapy, potentially curing diseases like HIV-1, sickle cell disease, and haemophilia B. However, challenges include off-target mutations and efficient delivery of CRISPR/Cas9. Precise target site selection and dosage control are crucial, with tools like CasOT helping identify and prevent unwanted mutations. CRISPR-Cas9 requires a PAM sequence to function, narrowing its targets in the genome but increasing specificity. Production of gRNA faces challenges due to mRNA processing, with alternatives like the artificial gene RGR showing promise. Efficient delivery methods are still needed, with current techniques involving DNA and RNA injection. Future applications include treating genetic diseases and agricultural improvements, with ongoing research essential for overcoming challenges and ensuring safety and accuracy. Global collaboration is vital for the ethical use of this technology.
Contribution to evidence-based healthcare: CRISPR therapy represents an exciting frontier in genetic medicine, allowing for precise gene editing and opening up new possibilities for treating incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0ce03e8f26830137fb7f68ada78482dfe3faf6" target='_blank'>
              CRISPR therapy: A revolutionary breakthrough in genetic medicine
              </a>
            </td>
          <td>
            Geneci Da Silva Barreto, Jair Brito do Nascimento, Karin Cristina Santos de Almeida, José Carlos Marcolino Neto, Samuel Lucas Ferreira Luz da Silva
          </td>
          <td>2024-12-16</td>
          <td>International Healthcare Review (online)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="BACKGROUND
The management of pediatric acute myeloid leukemia (AML) is based on the prognostic risk classification of initial leukemia. Targeted next-generation sequencing (NGS) is a reliable method used to identify recurrently mutated genes of pediatric AML and associated prognosis.


METHODS
In this study, we retrospectively evaluated the prognostic, and therapeutic utility of a targeted NGS panel covering twenty-five genes, in 21 children with de novo and 8 with relapsed or secondary AML.   Results. Variants were detected in 44.8% of patients, and 63.2% of them were in the signaling pathway genes. The number of variants per patient and diversity increased with age. The panel results affected hematopoietic stem cell transplantation decisions, especially in core binding factor AML, and allowed the categorization of diseases according to current classifications. Panel results also pointed out predisposition to germline leukemia to the extent of the panel coverage. No targeted therapy was used based on the variants, and none of the variants were used to monitor minimal residual disease.


CONCLUSIONS
Targeted NGS results, along with well-known genetic aberrations and treatment responses, can guide treatment modalities. The coverage of the routine panels should include proven mutations of childhood AML and germline leukemia predisposition genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4601dddbafa15e4c68929e64e4ba8e0770802f" target='_blank'>
              The importance of targeted next-generation sequencing based genomic profiling in the diagnosis of childhood acute myeloid leukemia: a single center experience.
              </a>
            </td>
          <td>
            D. Kaçar, B. Çavdarlı, Ayça Koca Yozgat, Melek Işık, F. Kurtipek, Fatma Tuba Yıldırım, T. Bayhan, Dilek Gürlek Gökçebay, Namık Yaşar Özbek, N. Yaralı
          </td>
          <td>2024-12-19</td>
          <td>The Turkish journal of pediatrics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) represent a small, yet pivotal subpopulation of tumor cells that play significant roles in tumor initiation, progression and therapeutic resistance. Circular RNAs (circRNAs) are a distinct class of RNAs characterized by their closed-loop structures, lacking 5′ to 3′ends. There is growing evidence that circRNAs are integral to the development and regulation of CSCs. Aberrant expression of circRNAs in CSCs can contribute to oncogenic properties and drug resistance. Specifically, oncogenic circRNAs modulate CSC behavior via key signaling pathways, thereby promoting CSC self-renewal and maintenance, as well as tumor progression. This review summarizes the latest research on the functional roles and regulatory mechanisms of circRNAs in CSC behavior and discusses potential applications and challenges of targeting circRNAs in CSCs. Understanding the intricate interactions between circRNAs and CSCs may lead to novel therapeutic strategies that effectively combat treatment resistance and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01b87d0f590a46f3144a9c75dc4396105c17dca" target='_blank'>
              Circular RNAs in cancer stem cells: Insights into their roles and mechanisms (Review)
              </a>
            </td>
          <td>
            Lunyu Yang, Yuling Yi, Zhu Mei, Dongmei Huang, Sitian Tang, Liyi Hu, Ling Liu
          </td>
          <td>2025-01-23</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Horizontal gene transfer (HGT) is a major driver of bacterial evolution, allowing bacteria to acquire new traits from other organisms. It enables the transfer of advantageous traits, such as antibiotic resistance, which can rapidly spread across bacterial populations, affecting public health. Several studies have estimated how widespread HGT can be and suggested that environmental changes are associated with changes in the rate of HGT. It is often considered that environmental changes lead to an increase in HGT rates due to the adaptive benefits of HGT. However, previous studies have been limited to one or a few species and have often focused on specific genes of interest. Here, we systematically investigate the effect of environmental changes on the rate of HGT in a large number of bacterial genomes along a deep phylogeny. We extend the scope of use of existing state-of -the -art methods for HGT inference to infer the transition into new ecosystems and the gain of pathogenicity. Transitions to new environments are associated with a decrease in the rate of HGT, along with the strengthening of purifying selection. This decrease in HGT rates is consistent regardless of genomic relatedness between the donor and recipient. Our results are robust to the use of different methodological frameworks and have implications for understanding the evolution of bacteria and HGT rates in natural microbial communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509a20de1480698aa85c82ec7a8080e8f597bcfe" target='_blank'>
              Horizontal gene transfer rates decrease during transitions to new ecosystems
              </a>
            </td>
          <td>
            Swastik Mishra, Martin J. Lercher
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2962c853fd82d5be105d8daa9d66daafd430b25e" target='_blank'>
              Elongation factor 2 in cancer: a promising therapeutic target in protein translation
              </a>
            </td>
          <td>
            X. Jia, Chuntian Huang, Fangfang Liu, Zigang Dong, Kangdong Liu
          </td>
          <td>2024-12-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The CRISPR/Cas9 system represents an adaptive immunological defense mechanism observed in bacteria and archaea, which has evolved to protect these organisms from persistent viral and plasmid assaults. This system is comprised of clustered regularly interspaced short palindromic repeats (CRISPR) and CRISPR-associated (Cas) proteins. By modifying the simplest type II CRISPR system and introducing a special small guide RNA (sgRNA) and Cas9 endonuclease into the cell, it is possible to cut double-stranded DNA at a specific location and realize knockout or knock-in. CRISPR/Cas9 system has many applications in biological and scientific research fields due to its efficient gene editing function, which makes up for the shortcomings of traditional transgenic technology, such as random integration and genetic instability, and has a broad development prospect. This paper comprehensively discusses the CRISPR/Cas9 technology’s applications in drug discovery and the drug development, such as screening and targeted editing of functional genes, screening and validating drug targets, fabricating animal bioreactors, constructing animal models, and treating genetic diseases, etc. It also discusses the risks and prospects of CRISPR/Cas9 technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/248b556c3f98dd8379f35db93ac781813dd8bf7b" target='_blank'>
              Advances in Gene Editing Technology in Biomedicine
              </a>
            </td>
          <td>
            Yuhan Wang
          </td>
          <td>2024-12-31</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a genetic anemia caused by the production of an abnormal adult hemoglobin. The clinical severity is lessened by elevated fetal hemoglobin (HbF) production in adulthood. A promising therapy is the transplantation of autologous, hematopoietic stem/progenitor cells (HSPCs) treated with CRISPR/Cas9 to downregulate the HbF repressor BCL11A via generation of double strand breaks (DSBs) in the +58-kb erythroid-specific enhancer. Here, to further enhance HbF production without increasing the mutagenic load, we targeted both +58-kb and +55-kb BCL11A erythroid-specific enhancers using base editors. We systematically dissected DNA motifs recognized by the key transcriptional activators within these regions and identified the critical nucleotides required for activator binding. Multiplex base editing of these residues was efficient and safe and generated no or little DSBs and genomic rearrangements. We observed substantial HbF reactivation, exceeding the levels achieved using the CRISPR/Cas9 nuclease-based strategy, thus efficiently rescuing the sickling phenotype. Multiplex base editing was efficient in long-term repopulating HSPCs and resulted in potent HbF reactivation in vivo. In summary, these results show that multiplex base editing of BCL11A erythroid-specific enhancers is a safe and potent strategy for treating sickle cell disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/949a30362791a74e8f68a2ef1865b96856027877" target='_blank'>
              Multiplex base editing of BCL11A regulatory elements to treat sickle cell disease
              </a>
            </td>
          <td>
            L. Fontana, P. Martinucci, S. Amistadi, T. Felix, M. Mombled, A. Tachtsidi, G. Corre, A. Chalumeau, G. Hardouin, J. Martin, O. Romano, M. Amendola, P. Antoniou, A. Miccio
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers.
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aa136c947da96c060fe9e8b5c25af5b195959fd" target='_blank'>
              Developing novel Lin28 inhibitors by computer aided drug design
              </a>
            </td>
          <td>
            Victor M Matias-Barrios, Mariia Radaeva, Graciella Rosellinny, Qiongqiong Jia, Ning Xie, Monica T L Villanueva, Hanadi Ibrahim, Jason Smith, Martin Gleave, N. Lallous, Suzana K Straus, Artem Cherkasov, Xuesen Dong
          </td>
          <td>2025-01-12</td>
          <td>Cell Death Discovery</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Inadequate response to androgen deprivation therapy (ADT) frequently arises in prostate cancer, driven by cellular mechanisms that remain poorly understood. Here, we integrated single-cell RNA sequencing, single-cell multiomics, and spatial transcriptomics to define the transcriptional, epigenetic, and spatial basis of cell identity and castration response in the mouse prostate. Leveraging these data along with a meta-analysis of human prostates and prostate cancer, we identified cellular orthologs and key determinants of ADT response and resistance. Our findings reveal that mouse prostates harbor lobe-specific luminal epithelial cell types distinguished by unique gene regulatory modules and anatomically defined androgen-responsive transcriptional programs, indicative of divergent developmental origins. Androgen-insensitive, stem-like epithelial populations - resembling human club and hillock cells - are notably enriched in the urethra and ventral prostate but are rare in other lobes. Within the ventral prostate, we also uncovered two additional androgen-responsive luminal epithelial cell types, marked by Pbsn or Spink1 expression, which align with human luminal subsets and may define the origin of distinct prostate cancer subtypes. Castration profoundly reshaped luminal epithelial transcriptomes, with castration-resistant luminal epithelial cells activating stress-responsive and stemness programs. These transcriptional signatures are enriched in tumor cells from ADT-treated and castration-resistant prostate cancer patients, underscoring their likely role in driving treatment resistance. Collectively, our comprehensive cellular atlas of the mouse prostate illuminates the importance of lobe-specific contexts for prostate cancer modeling and reveals potential therapeutic targets to counter castration resistance. Significance Statement Androgen deprivation therapy is a mainstay in prostate cancer treatment, yet many patients eventually develop castration-resistant disease—a lethal progression driven by poorly understood cellular mechanisms. Our study provides a comprehensive cellular map of the prostate, identifying key determinants of normal organization and castration-induced remodeling. By pinpointing the cell types and molecular programs that confer ADT responsiveness or resistance, our findings offer new directions for prostate cancer modeling and pave the way toward novel therapeutic strategies aimed at enhancing ADT efficacy and preventing the emergence of castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47e36c2b5c9a5688d815f8fbf4d281fd5d7c15cd" target='_blank'>
              Cellular cartography reveals mouse prostate organization and determinants of castration resistance
              </a>
            </td>
          <td>
            Han-Byoul Cho, Yuping Zhang, J. Tien, R. Mannan, Jie Luo, S. Narayanan, Somnath Mahapatra, Jing Hu, G. Shelley, Gabriel Cruz, Miriam Shahine, Lisha Wang, Fengyun Su, Rui Wang, Xuhong Cao, S. Dhanasekaran, Evan T. Keller, S. Pitchiaya, A. Chinnaiyan
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="Clinical risk scores in chronic myeloid leukemia (CML) are inadequate for identifying chronic phase (CP) patients at high-risk of blast crisis (BC) progression. The lack of accurate predictive tests hamper timely interventions, including stem cell transplants, that are more effective in early disease. By interrogating a single cell atlas of primary imatinib resistance for a BC-like gene expression signature, we identify aberrant CD42A+ megakaryocytic and CD10+CD19+ lymphoid progenitor expansion, as well as STAT1- and IFNγ-related inflammatory programs, as consistent features in the bone marrow and peripheral blood of CP patients at high-risk of BC transformation. We develop multi-color flow cytometry-based tests (MFC) to detect these features in CP patients at the time of diagnosis. Validating our MFC panels on a combined Australia-Singapore cohort comprising 28 CP patients, including 14 who underwent transformation, we demonstrate their ability to detect 100% of CP patients who transform with no false positives. Our findings highlight small populations of inflamed hematopoietic stem and progenitor cells, present at CP diagnosis, as powerful harbingers of future BC transformation. The ability of MFC panels to detect these cells at diagnosis support their inclusion as accurate risk-assessment tools to improve management of high-risk patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/695b8c7eb368f693a97922122a5ffdc4d6cd3154" target='_blank'>
              Single-cell transcriptomics-guided development of flow cytometric tests predicting chronic myeloid leukemia blast crisis transformation at chronic phase diagnosis
              </a>
            </td>
          <td>
            Mengge Yu, Vaidehi Krishnan, Pavithra Shyamsundar, K. Lee, Florian Schmidt, Xi Ren, Charles T.H. Chuah, Shyam Prabhakar, V. Saunders, D. Yeung, N. Shanmuganathan, Agnes S. M. Yong, Timothy P. Hughes, S. T. Ong
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The 5,000 to 8,000 monogenic diseases are inherited disorders leading to mutations in a single gene. These diseases usually appear in childhood and sometimes lead to morbidity or premature death. Although treatments for such diseases exist, gene therapy is considered an effective and targeted method and has been used in clinics for monogenic diseases since 1989. Monogenic diseases are good candidates for novel therapeutic technologies like gene editing approaches to repair gene mutations. Clustered regularly interspaced short palindromic repeats (CRISPR)-based systems, the pioneer and effective gene editing tool, are utilized for ex vivo and in vivo treatment of monogenic diseases. The current review provides an overview of recent therapeutic applications of CRISPR-based gene editing in monogenic diseases in in vivo and ex vivo models. Furthermore, this review consolidates strategies aimed at providing new treatment options with gene therapy, thereby serving as a valuable reference for advancing the treatment landscape for patients with monogenic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0a8862e584d090d20f1389daab0e1853e5ef72" target='_blank'>
              CRISPR/Cas System: A Powerful Strategy to Improve Monogenic Human Diseases as Therapeutic Delivery; Current Applications and Challenges.
              </a>
            </td>
          <td>
            Fatemeh Saberi, Zeinab Yousefi-Najafabadi, F. Shams, Zeinab Dehghan, Sepideh Ahmadi, Tayyebeh Pilehchi, Effat Noori, Zeinab Esmaeelzadeh, Maryam Bazgiri, Rezvan Mohammadi, Farzaneh Khani, M. Sameni, Parisa Moradbeigi, G. Kardar, Mohammad Salehi, Yong Teng, V. Jajarmi
          </td>
          <td>2025-01-07</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Originally derived from the immune system of bacteria, CRISPR-Cas9 technology has rapidly developed into a revolutionary gene editing tool that is widely used in molecular biology, drug development, and gene therapy. This technology not only provides a new treatment approach for genetic diseases such as sickle cell anemia and Duchenne muscular dystrophy, but also brings breakthrough progress in cancer treatment by enhancing chimeric antigen receptor T cell (CAR-T) therapy. CRISPR-Cas9 has also shown great potential in the drug discovery stage, especially in gene screening, development of disease models, and drug resistance research. However, with the rapid development of CRISPR technology, the scientific community still needs to solve challenges such as potential off-target effects in gene editing, inefficient delivery systems, and long-term safety. In addition, the ethical issues of gene editing have also sparked widespread discussion, especially how to ensure fair access when treating genetic diseases. Despite many challenges, CRISPR-Cas9 technology still shows great application prospects in the future of drug development and gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ad9a53620cc6ea8c99c67cb73104e6fa93ba3d" target='_blank'>
              The Pharmaceutical Applications of CRISPR-Cas9 and Its Development Trends
              </a>
            </td>
          <td>
            Yu Sang
          </td>
          <td>2024-12-31</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="CRISPR/Cas9 technologies provide unique capabilities for modeling disease and understanding gene-to-phenotype connections. In cultured cells, chemical-mediated control of Cas9 activity can limit off-target effects and enable mechanistic study of essential genes. However, widely-used Tet-On systems often show “leaky” Cas9 expression, leading to unintended edits, as well as weak activity upon induction. Leakiness can be distinctly problematic in the context of Cas9 nuclease activity, which may result in cumulative DNA damage and degradation of the target cell genome over time. To overcome these deficiencies, we established transgenic platforms that minimize Cas9 functionality in the off-state along with maximized and uncompromised on-state gene editing efficiency. By combining conditional destabilization and inhibition of Cas9, we developed an all-in-one (one or multiple guide RNAs and Cas9) ultra-tight, Tet-inducible system with exceptional dynamic range (on vs. off-state) across various cell lines and targets. As an alternative to Tet-mediated induction, we created a branaplam-regulated splice switch module for low-baseline and robust Cas9 activity control. Lastly, for circumstances where DNA damage needs to be avoided, we constructed a dual-control, Tet-inducible CRISPRi module for tight and potent transcriptional silencing. This upgraded suite of inducible CRISPR systems has broad applications for numerous cell types and experimental conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67f861c2ce0a102cbb4cb3e559a5affcf75ca92" target='_blank'>
              Chemically-inducible CRISPR/Cas9 circuits for ultra-high dynamic range gene perturbation
              </a>
            </td>
          <td>
            Rajini Srinivasan, Tao Sun, Azalia Sandles, Diana Wu, Liang Wang, A. Heidersbach, Clark Ho, Shiqi Xie, Andrew Ng, Benjamin Haley
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="BACKGROUND
 Colorectal cancer (CRC) is a prevalent malignant tumor characterized by a high mortality rate, with significant challenges persisting in the identification and management of its metastatic stage. The role of checkpoint kinase 1 (CHEK1), a cell cycle checkpoint kinase, in CRC has not been fully clarified. We hypothesize that the upregulation of CHEK1 may enhance the proliferation of CRC cells, indicating its potential as a novel therapeutic target for CRC therapy.
 AIM
 To investigate the expression and function of CHEK1 in CRC, this study utilizes single-cell RNA sequencing and tissue microarray data.
 METHODS
 Single-cell RNA sequencing technology was employed to analyze CRC cells from the GSE144735 dataset, and immunohistochemistry was conducted to confirm the expression of CHEK1 in CRC and adjacent tissues. We also integrated mRNA expression data from multiple public databases to assess global CHEK1 expression in CRC. Molecular docking experiments were performed to explore the interaction between CHEK1 and the potential drug nitidine chloride (NC), as well as to investigate the influence of CHEK1 on CRC cell proliferation.
 RESULTS
 We found comparatively elevated CHEK1 expression in the malignant epithelial cells of CRC, with marked upregulation of its mRNA levels in CRC tissues. Immunohistochemical analysis further confirmed the high expression of CHEK1 in CRC tissues, and the receiver operating characteristic curve demonstrated high accuracy (area under the curve = 0.964) for CHEK1 as a biomarker. Analysis of global datasets indicated a statistically significant overexpression of CHEK1 in CRC (standard mean difference = 1.81, P < 0.01), with summary receiver operating characteristic analysis yielding sensitivity and specificity values of 0.83 and 0.88, respectively. Molecular docking studies indicated that NC specifically targeted CHEK1, while clustered regularly interspaced short palindromic repeats knockout experiments demonstrated that CHEK1 promoted CRC cell proliferation.
 CONCLUSION
 Upregulation of CHEK1 promotes CRC cell proliferation. However, the dataset's diversity is limited, requiring further investigation into its specific mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a566889c9fc5fe520488439e6857e1a25ba38f4b" target='_blank'>
              Checkpoint kinase 1 in colorectal cancer: Upregulation of expression and promotion of cell proliferation
              </a>
            </td>
          <td>
            Yu-yan Pang, Zu-Yuan Chen, Da-Tong Zeng, Dong-Ming Li, Qi Li, Wan-Ying Huang, Bin Li, Jia-Yuan Luo, Bang-Teng Chi, Qiu Huang, Zhen-Bo Feng, Rong-Quan He
          </td>
          <td>2025-03-24</td>
          <td>World Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Fragment based novel drug identification and its validation through use of molecular dynamics and simulations.Comparing primary microcephaly genes with glioblastoma expression profiles reveals potential oncogenes, with proteins that support growth and survival in neural stem/progenitor cells likely retaining critical roles in glioblastoma. Identifying such proteins in familial and congenital microcephalic disorders offers promising targets for brain tumor therapy. Among these, KIF11, a kinesin motor protein (KSP), stands out as a significant oncogene. Expression analyses across various cancer types, including glioblastoma, demonstrate its overexpression in brain tumor patients. Using a targeted fragment-based drug discovery approach, we explored alternative small molecule inhibitors for KIF11. Existing drugs, such as ispinesib, are limited by side effects and multidrug resistance. Through molecular docking and simulations, we identified three candidate drug fragments. Further analysis confirmed that Mol-121026 exhibits a more stable interaction with KIF11 compared to ispinesib. Detailed analyses indicate that Mol-121026 binds to the same active site as the reference drug, effectively inhibiting KIF11's mechano-chemical activity. Importantly, Mol-121026, a derivative of 3-phenyl-1H-pyrazol-5-carboxylic acid, offers a promising alternative due to its lower molecular complexity, ability to target allosteric sites, and potential for optimization into a potent and effective drug candidate. Our findings identified Mol-121026 as a top candidate with a docking score of -10.2 kcal/mol and MM/GBSA binding energy of -19.10 kcal/mol. Molecular dynamics simulations revealed stable interactions with key residues GLU116 and GLU118, supporting its potential as a promising KIF11 inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9b86eac4f8c7f39109905d3ffc984c8589c2d9" target='_blank'>
              Fragment-Based Drug Discovery of KIF11 Inhibitors for Glioblastoma Treatment: Molecular Insights and Therapeutic Potential.
              </a>
            </td>
          <td>
            Q. Naseer, Xuexian Cao, Deng Yimai, Muhammad Ajmal Khan, Shengxia Chen
          </td>
          <td>2025-01-22</td>
          <td>Drug research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad33c5772a7c6f391f659ca36bbfaca426706c80" target='_blank'>
              Improving the odds of survival: transgenerational effects of infections.
              </a>
            </td>
          <td>
            V. Spanou, Theano Andriopoulou, E. Giamarellos‐Bourboulis, Mihai G. Netea
          </td>
          <td>2025-01-22</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Epigenetic therapies are emerging for pediatric cancers. Due to the relatively low mutational burden in pediatric tumors, epigenetic dysregulation and differentiation blockade is a hallmark of oncogenesis in some childhood cancers. By targeting epigenetic regulators that maintain tumor cells in a primitive developmental state, epigenetic therapies may induce differentiation. The most well-studied and clinically advanced epigenetic-targeted therapies include azacitidine and decitabine, which inhibit DNA methylation through competitive inhibition of the enzymatic activity of the DNA methyltransferase family enzymes. These DNA hypomethylating agents are Food and Drug Administration (FDA) approved for hematologic malignancies. The discovery that DNA hypermethylation occurs in patients with isocitrate dehydrogenase (IDH) mutations has led to the development and FDA approval of IDH inhibitors for hematologic and solid tumors. Epigenetic dysregulation in pediatric tumors is also driven by changes in the "histone code" that either promote oncogene expression or repress tumor suppressors. Cancers whose chromatin landscape is characterized by such aberrant histone posttranslational modifications may be amenable to targeted therapies that inhibit the chromatin-modifying enzymes that read, write, and erase these histone modifications. Small molecules that inhibit the enzymatic activity of histone deacetylases, acetyltransferases, and methyltransferases have been approved for the treatment of some adult cancers, and these agents are currently under investigation in various pediatric tumors. Chromatin regulatory complexes can be hijacked by oncogenic fusion proteins that are produced by chromosomal translocations, which are common drivers in pediatric cancer. Small molecules that disrupt oncogenic fusion protein activity and their associated chromatin complexes have demonstrated remarkable promise, and this approach has become the standard treatment for a subset of leukemias driven by the PML-RARA oncogenic fusion protein. A deeper understanding of the mechanisms that drive epigenetic dysregulation in pediatric cancer may hold the key to future success in this field, as the landscape of druggable epigenetic targets is also expanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a807fa68d0895be1a6925ed8f18dcca64f6453e" target='_blank'>
              Epigenetic Therapies.
              </a>
            </td>
          <td>
            Wallace Bourgeois, Scott A Armstrong, Emily B. Heikamp
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f23f58ff8f0c69ee80ac960c4ff630df39788c5" target='_blank'>
              LGR5: An emerging therapeutic target for cancer metastasis and chemotherapy resistance
              </a>
            </td>
          <td>
            W. Wang, N. Lokman, Simon C Barry, M. Oehler, C. Ricciardelli
          </td>
          <td>2025-01-17</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1f1a3134c1ae8bfb1d2f982a790258260dcc9c9" target='_blank'>
              Advances in epigenetic therapies for B-cell non-hodgkin lymphoma.
              </a>
            </td>
          <td>
            Weiwen Hu, Lanlan Zang, Xiaoxi Feng, Shuhui Zhuang, Liudi Chang, Yongjing Liu, Jinyan Huang, Yuanyuan Zhang
          </td>
          <td>2024-12-09</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematological malignancy; it is the most common acute leukemia in adults. AML prognosis is often poor, and relapse often occurs after initial remission. Recurrent genetic abnormalities underlying this disease and the presence of leukemic stem cells complicate disease treatment. However, the complex metabolic reprogramming that enables the unrestrained cell growth seen in these cells may also be their Achilles’ heel. In these cells, mitophagy operates as a double-edged sword. On one hand, it provides a source of building blocks for further cell division and serves as a method for removing damaged organelles, promoting cell survival. However, the profound metabolic changes to mitochondria also render these organelles more sensitive to damage and place them precariously close to excess mitophagic activation. This review discusses the dual role mitophagy plays in AML survival, the importance of targeting mitophagy to treat AML, and current progress in the area. The discovery and mechanism of action of multiple compounds that were used to inhibit or stimulate mitophagy and their effects on AML survival are also described. Further, we explore the combination strategy of mitophagy-targeting compounds with existing and/or novel chemotherapeutics to eradicate AML and discuss strategies to uncover new drug targets and novel mitochondria-targeting drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e8f6126e540840e2cf28df2eaa4c78cffc015ff" target='_blank'>
              Mitochondrial abnormalities as a target of intervention in acute myeloid leukemia
              </a>
            </td>
          <td>
            Elissa Tjahjono, Megan R. Daneman, Bernadetta Meika, Alexey V. Revtovich, N. V. Kirienko
          </td>
          <td>2025-01-20</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML), a malignant disease of the bone marrow, is characterized by the clonal expansion of myeloid progenitor cells and a block in differentiation. The high heterogeneity of AML significantly impedes the development of effective treatment strategies. Enhancer of zeste homolog 2 (EZH2), the catalytic subunit of the polycomb repressive complex 2 (PRC2), regulates the expression of downstream target genes through the trimethylation of lysine 27 on histone 3 (H3K27me3). Increasing evidence suggests that the dysregulation of EZH2 expression in various cancers is closely associated with tumorigenesis. In the review, we examine the role of EZH2 in AML, highlighting its crucial involvement in regulating stemness, proliferation, differentiation, immune response, drug resistance and recurrence. Furthermore, we summarize the application of EZH2 inhibitors in AML treatment and discuss their potential in combination with other therapeutic modalities. Therefore, targeting EZH2 may represent a novel and promising strategy for the treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c772976d3b58810f697b6b1b73c79162dca59db" target='_blank'>
              The epigenetic role of EZH2 in acute myeloid leukemia
              </a>
            </td>
          <td>
            Jinyong Fang, Jingcheng Zhang, Lujian Zhu, Xiaoru Xin, Huixian Hu
          </td>
          <td>2024-12-06</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Massively parallel sequencing (MPS) of nucleic acids has been a transformative technology for basic and applied genomic science, increasing efficiencies and decreasing costs to enable studies of unprecedented scope and impact. In clinical settings, these technological and scientific advances have led to the development of tests that are increasingly fast, comprehensive, and more frequently employed. Practitioners of genomic medicine have applied these tools across clinical settings, including diagnosis of inherited disorders and cancers and infectious disease detection and surveillance. In recent years, the commercial marketplace for MPS sequencers and reagents has been dominated by a few companies. The growing demand for sequencing has led to the recent emergence of several new sequencing platforms with techniques that may provide alternatives or improvements to existing workflows or allow the adoption of sequencing workflows in new settings. Clinical genomics laboratories will evaluate these platforms from a unique perspective, focusing on how technological advancements can improve patient care.


CONTENT
This review describes short-read sequencing platforms provided by Illumina, Element Biosciences, MGI, PacBio, Singular Genomics, Thermo Fisher Scientific, and Ultima Genomics. This review discusses their innovative approaches, principles, workflows, and applications.


SUMMARY
This review aims to inform laboratory geneticists, clinicians, and researchers about emerging short-read technologies and their applications in clinical genomics. By highlighting their principles and potential contributions, we aim to assist laboratories in selecting suitable solutions for their sequencing needs considering key factors such as applications, throughput, and integration with existing laboratory workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc19721b678ba2d4cedee69c6b99d021887e0026" target='_blank'>
              Innovations in Short-Read Sequencing Technologies and Their Applications to Clinical Genomics.
              </a>
            </td>
          <td>
            Katarzyna Polonis, Joseph H. Blommel, Andrew E O Hughes, DH Spencer, Joseph A Thompson, Molly C Schroeder
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mammalian genomes contain millions of regulatory elements that control the complex patterns of gene expression. Previously, The ENCODE consortium mapped biochemical signals across many cell types and tissues and integrated these data to develop a Registry of 0.9 million human and 300 thousand mouse candidate cis-Regulatory Elements (cCREs) annotated with potential functions1. We have expanded the Registry to include 2.35 million human and 927 thousand mouse cCREs, leveraging new ENCODE datasets and enhanced computational methods. This expanded Registry covers hundreds of unique cell and tissue types, providing a comprehensive understanding of gene regulation. Functional characterization data from assays like STARR-seq, MPRA, CRISPR perturbation, and transgenic mouse assays now cover over 90% of human cCREs, revealing complex regulatory functions. We identified thousands of novel silencer cCREs and demonstrated their dual enhancer/silencer roles in different cellular contexts. Integrating the Registry with other ENCODE annotations facilitates genetic variation interpretation and trait-associated gene identification, exemplified by discovering KLF1 as a novel causal gene for red blood cell traits. This expanded Registry is a valuable resource for studying the regulatory genome and its impact on health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99941d31f540b8743cd6f7ea4d2eb9891068be3e" target='_blank'>
              An Expanded Registry of Candidate cis-Regulatory Elements for Studying Transcriptional Regulation
              </a>
            </td>
          <td>
            Jill E. Moore, H. Pratt, K. Fan, Nishigandha Phalke, Jonathan Fisher, Shaimae I. Elhajjajy, Gregory Andrews, Mingshi Gao, Nicole Shedd, Yu Fu, Matthew C Lacadie, Jair Meza, Mohit Ganna, Eva Choudhury, Ross Swofford, Nina P. Farrell, Anusri Pampari, Vivekanandan Ramalingam, Fairlie Reese, Beatrice Borsari, Michelle Yu, Eve Wattenberg, M. Ruiz-Romero, Milad Razavi-Mohseni, Jinrui Xu, Timur Galeev, Michael A. Beer, R. Guigó, Mark B. Gerstein, Jesse M. Engreitz, Mats Ljungman, Timothy E. Reddy, M. Snyder, Charles B. Epstein, Elizabeth Gaskell, Bradley E. Bernstein, D. Dickel, Axel Visel, Len A. Pennacchio, A. Mortazavi, A. Kundaje, Zhiping Weng
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>125</td>
        </tr>

        <tr id="The p53 protein has attracted huge research interest over several decades due to its role as one of the most important tumor suppressors in mammals, which orchestrates a synchronous response from normal cells in the body to various forms of stress. The diverse cellular activities of the p53 protein are regulated mainly via its post-translational modifications (PTMs). PTMs affect p53 on several levels: at the level of the assembly of tetrameric complexes on DNA to transactivate its target genes, at the level of the assembly of tetrameric complexes on DNA to transactivate its target genes; at the level of proteolysis in the absence of stress; and on the contrary, at the level of augmented protein stability in response to stress signals. Disruptions in these regulatory mechanisms can lead to deviations from normal cellular function, boosting tumor initiation and progression. Conversely, targeted interventions in these pathways could prove beneficial for the development of antitumor therapies. Advancing our understanding of p53 modifiers and the proteins involved in its regulation equips researchers with an expanded toolkit for studying cellular processes and for developing biologically active molecules that influence p53-mediated responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd3fe1959718caec99cd867c8f3aa0f229e81022" target='_blank'>
              p53: The Multifaceted Roles of Covalent Modifications in Cancer
              </a>
            </td>
          <td>
            Tatiana A. Grigoreva, Angelina A. Romanova, V. Tribulovich, Nikolay B. Pestov, Ruslan A Oganov, Diana K. Kovaleva, T. Korneenko, N. A. Barlev
          </td>
          <td>2024-12-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Background Chromosomal instability (CIN), a hallmark of cancer, is commonly linked to poor prognosis in high‐grade prostate cancer (PCa). Paradoxically, excessively high levels of CIN may impair cancer cell viability. Consequently, understanding how tumours adapt to CIN is critical for identifying novel therapeutic targets. Methods Bioinformatic analyses were conducted to identify genes overexpressed in PCa tissues using The Cancer Genome Atlas (TCGA) and GEO datasets. Western blotting and immunohistochemistry assays were applied to determine the expression levels of euchromatic histone lysine methyltransferase 2 (EHMT2), pT232‐Aurora B and Cullin 3 (CUL3). The proliferation of cells was measured through CCK‐8 tests, clonogenesis and subcutaneous xenografts of human PCa cells in BALB/c nude mice. Live cell imaging, immunofluorescence (IF) and flow cytometry were used to confirm the role of EHMT2 in PCa cell mitosis. Co‐immunoprecipitation, Western blotting and IF assays further elucidated the underlying molecular mechanisms. Results EHMT2 was highly expressed in metastatic PCa tissues exhibiting elevated CIN and was strongly associated with adverse clinical outcomes in patients with PCa. Silencing EHMT2 impaired cell division, inducing G2/M‐phase arrest and mitotic catastrophe in PCa cells. Mechanistically, EHMT2 is indispensable to ensure the full activation of Aurora B through centromeric R‐loop‐driven ATR–CHK1 pathway, with EHMT2 protein expression peaking during the G2/M‐phase. Moreover, CUL3 was identified as a binding partner of EHMT2, mediating its polyubiquitination and destabilising its protein levels. Conclusions This study reveals a CUL3–EHMT2–Aurora B regulatory axis that safeguards accurate chromosome segregation in PCa cells, supporting the potential therapeutic application of EHMT2 inhibitors. Key points Euchromatic histone lysine methyltransferase 2 (EHMT2) is overexpressed in advanced prostate cancer, restraining catastrophic chromosomal instability (CIN) and enhancing cell fitness. EHMT2 functions via the centromeric R‐loop‐driven ATR–CHK1–Aurora B pathway to promote chromosomal stability. EHMT2 confers enzalutamide resistance via activating Aurora B. Cullin 3 (CUL3) promotes EHMT2 destabilisation via deubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc5ece350ce05d9ec1131335e290684b93468c8" target='_blank'>
              EHMT2‐mediated R‐loop formation promotes the malignant progression of prostate cancer via activating Aurora B
              </a>
            </td>
          <td>
            Yuyang Zhang, Mingqin Su, Yiming Chen, Li Cui, Wei Xia, Renfang Xu, Dong Xue, Xiansheng Zhang, Xingliang Feng
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a highly invasive and highly lethal malignant tumor. Due to the difficulty in early diagnosis and limited treatment options, the 5-year survival rate is less than 10%. Traditional treatments such as surgery, chemotherapy, and radiotherapy have limited effects on PDAC, which is mainly attributed to its unique tumor microenvironment (TME), common gene mutations (such as KRAS, and TP53), and immunosuppressive properties. In recent years, with the advancement of molecular biology and genomics technologies, the molecular mechanisms of PDAC have been studied more deeply, revealing the gene mutations and regulatory networks associated with its pathogenesis. In particular, KRAS mutations have become an important research direction for targeted therapy. However, the effect of immunotherapy in PDAC is limited by the immune escape characteristics of TME. This article systematically summarizes the key gene mutations in PDAC and their regulatory mechanisms, including epigenetic regulation, microRNA, histone modification and other influencing factors. CRISPR-Cas9 shows great potential in correcting gene mutations and reshaping TME. In addition, combining immunotherapy with targeted drug delivery (such as nanotechnology) can improve the precision of treatment and reduce side effects. These advances provide a comprehensive reference for future personalized and effective treatment options for PDAC">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1538e4df893f0e562277858358833ccdc8a4f65" target='_blank'>
              Genetic Mutation and Regulation in Pancreatic Ductal Adenocarcinoma Cancer: Developing Relevant Molecular Biomarkers
              </a>
            </td>
          <td>
            Xiaomi Ai
          </td>
          <td>2025-01-15</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The prevalent intra- and intertumoral heterogeneity results in undesirable prognosis and therapy failure of pancreatic cancer, potentially resulting from cellular senescence. Herein, integrated analysis of bulk and single-cell RNA-seq profiling was conducted to characterize senescence-based heterogeneity in pancreatic cancer. Publicly available bulk and single-cell RNA sequencing from pancreatic cancer patients were gathered from TCGA-PAAD, PACA-AU, PACA-CA, and GSE154778 datasets. The activity of three senescence-related pathways (cell cycle, DNA repair, and inflammation) was scored utilizing ssGSEA algorithm. A series of functional verifications of crucial genes were accomplished in patient tissue and pancreatic cancer cells. Based upon them, unsupervised clustering analysis was executed to classify pancreatic cancer samples into distinct senescence-based clusters at the bulk and single-cell levels. For single-cell transcriptome profiling, cell clustering and annotation were implemented, and malignant cells were recognized utilizing infercnv algorithm. Two senescence-based clusters were established and highly reproducible at the bulk level, with the heterogeneity in prognosis, clinicopathological features, genomic CNVs, oncogenic pathway activity, immune microenvironment and immune checkpoints. Senescence-relevant gene CHGA, UBE2C and MCM10 were proved to correlate with the migration and prognosis of pancreatic cancer. At the single-cell level, seven cell types were annotated, comprising ductal cells 1, ductal cells 2, fibroblasts, macrophages, T cells, stellate cells, and endothelial cells. The senescence-based classification was also proven at the single-cell level. Ductal cells were classified as malignant cells and non-malignant cells. In the tumor microenvironment of malignant cells, hypoxia and angiogenesis affected senescent phenotype. The heterogeneity in senescence was also observed between and within cell types. Altogether, our findings unveil that cellular senescence contributes to intra- and intertumoral heterogeneity in pancreatic cancer, which might facilitate the development of therapeutics and precision therapy in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2742fe352ef25d18eeda7d59fe2c6a8314ffc13" target='_blank'>
              Deciphering the senescence-based tumoral heterogeneity and characteristics in pancreatic cancer: Results from parallel bulk and single-cell transcriptome data.
              </a>
            </td>
          <td>
            Yeting Lu, Shuo Han, Jing Hu, Kaiji Lv, Y. Ruan, Gong Cheng, Jing Zhang, Xiang Wu, Zeming Weng, Xinhua Zhou
          </td>
          <td>2025-01-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="INTRODUCTION
Leukemia is typically categorized into myeloid leukemia and lymphoblastic leukemia based on the origins of leukemic cell. Myeloid leukemia is a group of clonal malignancies characterized by the presence of increased immature myeloid cells in both bone marrow and peripheral blood. Of note, the aberrant expression of specific proteins or the generation of fusion proteins due to chromosomal abnormalities are well established drivers in various forms of myeloid leukemia. Therefore, these oncoproteins represent promising targets for drug development.


AREAS COVERED
In this review, we comprehensively discussed the pathogenesis of typical leukemia oncoproteins and current landscape of small molecule drugs targeting these oncogenic proteins. Additionally, we elucidated novel strategies, including proteolysis-targeting chimeras (PROTACs), hyperthermia and genomic editing, which specifically degrade oncogenic proteins in myeloid malignancies.


EXPERT OPINION
Although small molecule drugs have significantly improved the prognosis of oncoprotein driven myeloid leukemia patients, drug resistance due to the mutations in oncoproteins is still a great challenge in clinic. New approaches such as PROTACs, hyperthermia and genomic editing are considered promising approaches for the treatment of oncoprotein-driven leukemia, especially for drug resistant mutants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea5fe5c5cfd460bcc9a1a8e4861d057f9becb9a" target='_blank'>
              Therapeutic approaches targeting oncogenic proteins in myeloid leukemia: challenges and perspectives.
              </a>
            </td>
          <td>
            Xing Yi Yan, Yuan-yuan Kang, Ze Yan Zhang, Ping Huang, Chang Yang, Hua Naranmandura
          </td>
          <td>2024-12-16</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Simple Summary KTMA2 gene rearrangements (KMT2Ar) frequently occur in acute leukemias (ALs) and portray an aggressive clinical phenotype, a high relapse risk, and a dismal prognosis. In these subsets of AL, the binding of the protein Menin (MEN1) to KMT2A fusion proteins drives the activation of a transcriptional pathway, altering the transcription of various genomic entities, particularly HOX/MEIS1 genes, which are crucial in the development of the leukemic phenotype. In this regard, KMT2Ar is prevalent in acute myeloid leukemia (AML), acute lymphoblastic leukemia, and some rarer AL subtypes. In addition, nucleophosmin-1 (NPM1m) mutations induce a gene expression profile similar to other KMT2Ar ALs, specifically in NPM1m AML. Therefore, inhibiting the interaction between MEN1 and KMT2Ar and NPM1m proteins induces apoptosis and promotes the differentiation of leukemic cells. In this regard, some targeted agents, such as MEN1 inhibitors, have been developed and are in an advanced phase of clinical investigation with encouraging clinical results. They have recently received their first formal recognition from the U.S. and European drug regulatory agencies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47f6d0074705ca17dc267107710154dd96466e76" target='_blank'>
              Menin Inhibitors: New Targeted Therapies for Specific Genetic Subtypes of Difficult-to-Treat Acute Leukemias
              </a>
            </td>
          <td>
            P. Niscola, Valentina Gianfelici, Marco Giovannini, D. Piccioni, Carla Mazzone, P. Fabritiis
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Background CREB binding protein (CREBBP) is a key epigenetic regulator, altered in a fifth of relapsed cases of acute lymphoblastic leukemia (ALL). Selectively targeting epigenetic signaling may be an effective novel therapeutic approach to overcome drug resistance. Anti‐tumor effects have previously been demonstrated for GSK‐J4, a selective H3K27 histone demethylase inhibitor, in several animal models of cancers. Methods To characterize the effect of GSK‐J4, drug response profiling, CRISPR‐Dropout Screening, BH3 profiling and immunoblotting were carried out in ALL cell lines or patient derived samples. Results Here we provide evidence that GSK‐J4 downregulates cyclic AMP‐responsive element‐binding protein (CREB) and CREBBP in B‐cell precursor‐ALL cell lines and patient samples. High CREBBP expression in BCP‐ALL cell lines correlated with high GSK‐J4 sensitivity and low dexamethasone sensitivity. GSK‐J4 treatment also induced Bcl‐2 and Bcl‐XL dependency and apoptosis. Conclusions This study proposes H3K27 demethylase inhibition as a potential treatment strategy for patients with treatment‐resistant ALL, using CREBBP as a biomarker for drug response and combining GSK‐J4 with venetoclax and navitoclax as synergistic partners.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6268afa1f23d7dedcd6b2c8390d3c0a47ae3fe38" target='_blank'>
              Inhibiting H3K27 Demethylases Downregulates CREB‐CREBBP, Overcoming Resistance in Relapsed Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Juan Lazaro-Navarro, C. Alcon, M. Dorel, Lina Alasfar, L. Bastian, Claudia Baldus, K. Astrahantseff, M. Yaspo, J. Montero, Cornelia Eckert
          </td>
          <td>2025-01-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/501a2344c548bcfc786460170b1feb2e0d711776" target='_blank'>
              Sperm-borne small non-coding RNAs: potential functions and mechanisms as epigenetic carriers
              </a>
            </td>
          <td>
            Muhammad Naveed, Zhaokang Shen, Jianqiang Bao
          </td>
          <td>2025-01-17</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4211b81d0b19e4c7f2e0cfa7c734b541dc3fa943" target='_blank'>
              Epigenetics in the modern era of crop improvements.
              </a>
            </td>
          <td>
            Yan Xue, Xiaofeng Cao, Xiangsong Chen, Xian Deng, Xing Wang Deng, Yong Ding, Aiwu Dong, Cheng-Guo Duan, Xiaofeng Fang, L. Gong, Zhizhong Gong, Xiaofeng Gu, Chongsheng He, Hang He, Shengbo He, Xin-Jian He, Yan He, Yuehui He, Guifang Jia, Danhua Jiang, Jianjun Jiang, Jinsheng Lai, Zhaobo Lang, Chenlong Li, Qing Li, Xingwang Li, Bao Liu, Bing Liu, Xiao Luo, Yijun Qi, Weiqiang Qian, Guodong Ren, Qingxin Song, Xianwei Song, Zhixi Tian, Jia-Wei Wang, Yuan Wang, Liang Wu, Zhe Wu, Rui Xia, Jun Xiao, Lin Xu, Zhengyi Xu, Wenhao Yan, Hongchun Yang, J. Zhai, Yijing Zhang, Yusheng Zhao, Xuehua Zhong, Dao-Xiu Zhou, Ming Zhou, Yue Zhou, Bo Zhu, Jian-Kang Zhu, Qikun Liu
          </td>
          <td>2025-01-08</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background Diffuse midline gliomas (DMGs) with histone H3K27M mutations represent a devastating paediatric brain cancer characterized by abysmal prognosis and limited treatment options. The only approved treatment is radiotherapy (RT), but most of the tumours relapse with fatal consequences. In this study, we sought to investigate whether irradiation leads to senescence induction and explore the efficacy of senolytics against DMG. Methods We have characterised the senescent phenotype of five genetically heterogeneous H3K27M-altered human DMG cell lines, combining cellular and/or molecular approaches. The sensitivity of senescent cells to Bcl-xL inhibition has been demonstrated in dose/response curves in vitro and in a PDX model of DMG. Results Here, we show that ionizing radiation induces senescence and SASP responses in both TP53 mutant and wild-type H3K27M-altered human DMG cell lines. We identify Navitoclax as a potent senolytic agent that selectively targets senescent DMG cells into apoptosis by inhibiting Bcl-xL. Related compounds, such as a proteolysis-targeting chimera (PROTAC)-mediated Bcl-xL degradation and a galacto-conjugated form of Navitoclax also show an effective senolytic activity in senescent cancer cells. Finally, we show that a combination therapy of irradiation and Navitoclax results in reduced tumor burden and increased mouse survival in an orthotopic xenograft DMG model. Conclusion These results offer a rationale for further clinical development of senolytic therapies as part of multimodal treatment approaches for DMG patients/ Key Points Ionising irradiation induces senescence in human DMG cells independently of the p53 status. Bcl-xL inhibition results in apoptosis of human DMG senescent cells in synergy with irradiation. Combination of irradiation and BcL-xL inhibition reduces tumourigenesis in a PDX model of DMG. Importance of the Study H3K27M-altered DMG are devastating paediatric tumours with an abysmal prognosis. The only approved treatment is radiotherapy but this is palliative and tumours almost always relapse with fatal consequences for the patients. In this study, we show that radiotherapy results in senescence induction in five genomically heterogeneous human DMG cell lines. We identify that drugs targeting the anti-apoptotic protein Bcl-xL show a strong senolytic activity in conjunction with radiotherapy both in vitro in DMG cells and in vivo in a PDX model of H3K27M-altered DMG. Treatment with Bcl-xL inhibitor Navitoclax, or related compounds targeting Bcl-xL protein degradation or containing a galactose conjugated form of Navitoclax results in DMG cancer cell apoptosis. As several of these inhibitors are currently being tested in ongoing clinical trials against other diseases, our data support the use of Bcl-xL inhibition mediated senolytics as an adjuvant therapy to radiotherapy to potentially improve outcomes in this challenging disease setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b580291efecff13cf295c14d6b2926c855610e2" target='_blank'>
              Targeted inhibition of Bcl-xL following radiation reduces tumourigenesis in preclinical models of H3K27M-altered diffuse midline glioma
              </a>
            </td>
          <td>
            Ashley Vardon, S. Haston, Romain Guiho, Diana Carvahlo, Rebecca Carter, J. Boult, Daniel Gharai, Diwakar Santhakumar, Ielyaas Cloete, James Gray, John Apps, O. Ogunbiyi, Daohong Zhou, Guangrong Zheng, Ramón Martínez-Máñez, Mark Lythgoe, Laura Donovan, A. Carcaboso, Owen Williams, Jesus Gil, Thomas Jacques, Jamie Dean, D. Michod, Chris Jones, Darren Hargrave, J. Martínez-Barbera
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [9, 6],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>