<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-05-12 09:30:06 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>33</td>
          <td>43</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d685ee84bfeabbf621f354633519c29df6bcad68" target='_blank'>
              Abstract 4399: Systematic interrogation of cell states that drive inflammation tolerance and therapy resistance in cancer
              </a>
            </td>
          <td>
            Srivatsan Raghavan, Evelyn Y. Tong, Aswanth H. Mahalingam, Zixin Chen, Jacob Smigiel, Tsukasa Shibue, Andrew W. Navia, Alex K. Shalek, Lorin Crawford, Ava P. Amini, Peter S. Winter
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4abf519afad0178ba59d0cd17f3f428071bf92cb" target='_blank'>
              Abstract 2674: Cellular plasticity facilitates therapy resistance through enhancing adaptability
              </a>
            </td>
          <td>
            Alicia Bjornberg, Xierali Aobuli, M. Froid, R. Barker-Clarke, Jeffrey Maltas, Jacob G. Scott, Berkley Gryder, David Bassanta, Alexander Anderson, Virginia Turati, A. Marusyk
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faabf5f01dce9d4fd060f035893412511b51d610" target='_blank'>
              Abstract 4426: Chromatin constraints underpin adeno-to-squamous cell state transitions during acquired resistance to KRAS inhibition
              </a>
            </td>
          <td>
            Ayushi S. Patel, Samuel Greene, Arianne Parvaresh-Rizi, Gustavo S. França, K. Wong, Itai Yanai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e19dad7b139ef16ca0124619b3b70027a193368b" target='_blank'>
              Uncovering Convergent Cell State Dynamics Across Divergent Genetic Perturbations Through Single-Cell High-Content CRISPR Screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Andrew Liao, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-05-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96dda01278fdb738773a58b49aca47aa0308c83d" target='_blank'>
              Abstract 5541: Dissecting chemotherapy resistance development in triple negative breast cancer through single cell multiome and lineage tracing
              </a>
            </td>
          <td>
            Chufan Zhang, Chu Pan, D. Cescon, M. Lupien
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bca1af35819fcfb6f9d7674ff4443096f1ec742" target='_blank'>
              Single-Cell Pharmacogenomic Landscapes of Epigenetic Therapy in Gastric Cancer
              </a>
            </td>
          <td>
            Chang Xu, Haoran Ma, T. Sheng, M. Lee, S. A. A. Abdul Ghani, Laura Perlaza-Jiménez, K. Huang, Shen Kiat Lim, Supriya Srivastava, Xuewen Ong, S. Tay, S. Ho, Angie Tan, Feng Zhu, Vincenzo Nasca, G. Randon, Hassan Ashktorab, A. S. Cheng, A. Jeyasekharan, Shang Li, Ming Teh, Raghav Sundar, David R. Powell, Joseph Rosenbluh, F. Pietrantonio, W. Yong, J. So, Patrick Tan
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eed8fdb3c30cedec701229d2292e4e2a7ab7ba3" target='_blank'>
              Abstract 6356: Multiscale high-content imaging to identify molecular and morphological signatures in drug-resistant cancer cells
              </a>
            </td>
          <td>
            Yaxin Yang, Jianquan Xu, Chaojie Zhang, Hongqiang Ma, Sarah J. Hainer, Yang Liu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Identifying genetic dependencies in human colon cancer could help identify effective treatment strategies. Genome-wide CRISPR-Cas9 dropout screens have the potential to reveal genetic dependencies, some of which could be exploited as therapeutic targets using existing drugs. In this study, we comprehensively characterized genetic dependencies present in a colon cancer organoid avatar, and validated tumor-specific selectivity of select pharmacologic agents. We conducted a genome-wide CRISPR dropout screen to elucidate the genetic dependencies that interacted with select driver somatic mutations. We found distinct genetic dependencies that interacted with WNT, MAPK, PI3K, TP53, and mismatch repair pathways and validated targets that could be exploited as treatments for this specific subtype of colon cancer. These findings demonstrate the utility of functional genomic screening in the context of personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f08c27268b9a856764ce30d936babdccabacfc" target='_blank'>
              Comprehensive genomic dependency landscape of a human colon cancer organoid
              </a>
            </td>
          <td>
            Sanam Khalili, Atefeh Mohseninia, Changlong Liu, Carolyn E. Banister, Paige Heine, M. Khazan, Sidney E. Morrison, P. Gokare, Glenn S Cowley, Barbara A. Weir, David Pocalyko, Kurtis E. Bachman, Phillip J. Buckhaults
          </td>
          <td>2025-03-14</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178582b0b4cac0c55049a52434a209bd8b7613e6" target='_blank'>
              Quantitative measurement of phenotype dynamics during cancer drug resistance evolution using genetic barcoding
              </a>
            </td>
          <td>
            F. J. Whiting, M. Mossner, C. Gabbutt, C. Kimberley, C. P. Barnes, A. Baker, Erika Yara, A. Sottoriva, Richard A. Nichols, Trevor A. Graham
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43efa757ce011d3db682c5cad92c0307f3b656c7" target='_blank'>
              The Role of Phenotypic Plasticity in Adaptation to Treatment and Prospective Plasticity Drivers in Hepatoblastoma
              </a>
            </td>
          <td>
            Emilia Chen, Christie English, Alejandro Allo Anido, Siân Hamer, Stefano Cairo, Alejandra Bruna
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9151033e1d6669563430e41bef627c6f841c5064" target='_blank'>
              Abstract 5551: From persistence to resistance: insights into the mechanisms of drug tolerance in cancer therapy using patient-derived organoids
              </a>
            </td>
          <td>
            Yasmine Abouleila, Timo Voskuilen, M. Doorn, Roel Verkerk, Joris Maas, C. Veríssimo, Sylvia F. Boj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eecdb6f080369d4f144052961e7b7ab0cd6c99ec" target='_blank'>
              Abstract 539: Secretory EMT fuels ERBB3 axis activation in resistance to EGFR inhibitors
              </a>
            </td>
          <td>
            A. Morselli, Donatella Romaniello, Chiara Miroglio, Martina Mazzeschi, M. Sgarzi, F. Pagano, C. Girone, Francesca Ambrosi, W. Kothalawala, Gyorffy Balazs, A. De Giglio, Michelangelo Fiorentino, Andrea Ardizzoni, Yosef Yarden, M. Lauriola
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Chemotherapy remains a prevalent strategy in cancer therapy; however, the emergence of drug resistance poses a considerable challenge to its efficacy. Most drug resistance arises from the accumulation of genetic mutations in a minority of resistant cells. The mechanisms underlying the emergence and progression of cancer resistance from these minority-resistant cells (MRCs) remain poorly understood. This study employs force-induced remnant magnetization spectroscopy (FIRMS) alongside various biological investigations to reveal the mechanical pathways for MRCs fostering drug resistance and tumor progression. The findings show that minority Paclitaxel-resistant cancer cells have enhanced mechanical properties. These cells can transmit high-intensity forces to surrounding sensitive cells (SCs) through the force transducer, Merlin. This force transmission facilitates the assimilation of surrounding SCs, subsequently strengthening the contraction and adhesion of tumor cells. This process is termed "mechano-assimilation," which accelerates the development of drug resistance and tumor progression. Interestingly, disturbances and reductions of mechano-assimilation within tumors can restore sensitivity to Paclitaxel both in vitro and in vivo. This study provides preliminary evidence highlighting the contribution of MRCs to the development of drug resistance and malignancy, mediated through mechanical interactions. It also establishes a foundation for future research focused on integrating mechanical factors into innovative cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f46cb96deeea2cf4ed683e45346444a026b4be6" target='_blank'>
              Mechanisms and Therapeutic Strategies for Minority Cell-Induced Paclitaxel Resistance and Tumor Progression Mediated by Mechanical Forces.
              </a>
            </td>
          <td>
            Xueyan Feng, Di Zhang, Guoxun Wang, Liwei Lu, Feng Feng, Xiuyu Wang, Chanchan Yu, Yahong Chai, Jin Zhang, Wenchao Li, Jing Liu, Hongxia Sun, Li Yao
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d56e1d2945b307c87e3e5dbfa94e2c69fe76398" target='_blank'>
              Multi-Omic Profiling Reveals Epigenetic Drivers of Immunotherapy Resistance in Multiple Myeloma
              </a>
            </td>
          <td>
            L. Haertle, Natalia Buenache Cuenda, Francisco J. Villena González, Isabel Espejo Díaz, Paula Lazaro del Campo, J. M. Rosa-Rosa, Rafael Alberto, Alonso Fernandez, Maria Nieves Lopez Muñoz, J. Sánchez-Pina, A. García-Ortiz, A. Valeri, S. Barrio, Fátima Al-Shahrour, Florian Bassermann, J. Martínez-López, Di Domenico, T. Domenico
          </td>
          <td>2025-04-19</td>
          <td>None</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Simple Summary Cancer has consistently posed a significant challenge on a global scale, with rising incidence and mortality. Despite the tremendous advancements in cancer therapeutics throughout the years, chemotherapy remains the current standard treatment in cancer management. As a result, chemotherapy resistance among cancer patients has emerged as a major clinical challenge, causing unsatisfactory responses to the treatment. Chemotherapy resistance is closely related to the dysregulation of a normal cellular process called ‘pre-mRNA Alternative Splicing’, where one gene can generate various protein forms with different structures and functions. A comprehensive understanding of dysregulated alternative splicing mechanisms is essential to propose novel strategies in cancer research aimed at identifying new and improved methods to maximise treatment efficacy and overcome chemotherapy resistance. Therefore, this review highlights the complexity of chemotherapy resistance and the potential roles of alternative splicing in overcoming resistance to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c315ff50467ff3617156ad8cf2b1b2d1ebcf6f" target='_blank'>
              Aberrant Splicing as a Mechanism for Resistance to Cancer Therapies
              </a>
            </td>
          <td>
            Duygu Duzgun, S. Oltean
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df82979a5771df69f4d6ba63039b4660f6ec17f" target='_blank'>
              Abstract 5065: Single-cell transcriptomic profiling of longitudinal patient tumors and PDX models reveals resistance mechanisms in ALK- and EGFR-mutant LUAD
              </a>
            </td>
          <td>
            Naomi Mfonfu, F. Braye, Galina Boldina, Matteo Cesaroni, Luc Fibroulet, Sergey I. Nikolaev, Andrey A. Yurchenko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863308a92ce0800145d29b567e0b6c910bc5cae2" target='_blank'>
              Abstract 6494: Genetically characterized canine tumoral cell lines are relevant models to test the efficacy of new targeted therapies: Example of anti-MDM2 targeted therapy
              </a>
            </td>
          <td>
            P. Montagne, U. Jarry, Caroline Confais, É. Cadieu, Aline Primot, Nicolaï Hoffman, Armel Houël, Rémy Le Guével, T. Derrien, Catherine André, Jérome Abadie, C. D. la Rochelle, B. Hédan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/221d45fdda028b3fdec5da9d6b92cc4847a8b483" target='_blank'>
              Understanding the Role of Toggle Genes in Chronic Lymphocytic Leukemia Proliferation
              </a>
            </td>
          <td>
            Olga Sirbu, Gunjan Agarwal, Alessandro Giuliani, Kumar Selvarajoo
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7191d1be8112623ab8fc75b326b24bf447563f16" target='_blank'>
              Abstract 1257: Response to primary therapy predicts tumor evolution dynamics at relapse in ovarian cancer
              </a>
            </td>
          <td>
            J. Oikkonen, Giulia Micoli, Susanna Holmström, Kari Lavikka, M. T. Nguyen, Yilin Li, G. Marchi, A. Rajavuori, Heidi Koskela, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Cancer is heterogeneous and variability in drug sensitivity is widely documented across cancer types. Adaptive therapy is an emerging modality of cancer treatment that leverages this drug resistance heterogeneity to improve therapeutic outcomes. Current standard treatments typically eliminate a large fraction of drug-sensitive cells, leading to drug-resistant relapse due to competitive release. Adaptive therapy aims to retain some drug-sensitive cells, thereby limiting resistant cell growth by ecological competition. While early clinical trials of such a strategy have shown promise, optimisation of adaptive therapy is a subject of active study. Current methods largely assume cell phenotypes to remain constant, even though cell-state transitions could permit drug-sensitive and -resistant phenotypes to interchange and thus escape therapy. We address this gap using a deterministic model of population growth, in which sensitive and resistant cells grow under competition and undergo cell-state transitions. Based on the model’s steady-state behaviour and temporal dynamics, we identify distinct balances of competition and phenotypic transitions that are suitable for effective adaptive versus constant dose therapy. Our data indicate that under adaptive therapy, models with cell-state transitions show a higher frequency of fluctuations than those without, suggesting that the balance between ecological competition and phenotypic transitions could determine population-level dynamical properties. Our analyses also identify key limitations of applying phenomenological models in clinical practice for therapy design and implementation, particularly when cell-state transitions are involved. These findings provide an overall perspective on the relevance of phenotypic plasticity for emerging cancer treatment strategies using population dynamics as an investigation framework. Significance Statement Drug-sensitive and -resistant cancer cells can compete with each other within the same tumour, and adaptive therapy exploits this competition to control overall tumour growth. The fact that sensitive and resistant cell types can switch phenotypes complicates the implementation of adaptive therapy. Our ODE-based theoretical ecology framework shows that asymmetric competition favouring sensitive cells generally benefits therapy outcomes, while phenotypic plasticity is usually detrimental. Our model also provides cell population-level indicators that can help predict the underlying balance between competition and plasticity. Phenomenological models are limited by whether their parameters can be reliably identified given scarce data, and our study illuminates the scope of such models in understanding cancer population dynamics and the need for mechanistic modelling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5674ed15d6d0f458d4dbdaa153d7277526c19b0b" target='_blank'>
              Impacts of competition and phenotypic plasticity on the viability of adaptive therapy
              </a>
            </td>
          <td>
            B. Vibishan, Paras Jain, Vedant Sharma, K. Hari, Claus Kadelka, Jason T. George, M. Jolly
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Summary We demonstrate that carcinogenesis and multi-therapy resistance in multiple myeloma (MM)—a treatable yet incurable plasma cell malignancy—are driven by epigenetic dysregulation. In this new paradigm, genomic and cytogenetic events unlock epigenetic plasticity, reshaping MM cell biology to evade tumor microenvironment constraints and therapeutic pressure. These conclusions are derived from a newly assembled cohort of nearly 1,000 patients, spanning premalignant to late-stage refractory MM, comprehensively characterized at molecular and clinical levels. Our findings provide a unifying framework to explain inter-patient genomic heterogeneity and the emergence of therapy resistance in sequential samples without new genomic alterations. In conclusion, we propose targeting epigenetic plasticity-mediated plasma cell evasion as a promising therapeutic strategy in MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19a7c4379c96f4e14d05eb168255be930f2fe17c" target='_blank'>
              Epigenetic Plasticity Drives Carcinogenesis and Multi-Therapy Resistance in Multiple Myeloma
              </a>
            </td>
          <td>
            A. Silva, Rafael R. Canevarolo, P. Sudalagunta, M. Meads, Maria Silva, Xiaohong Zhao, Dario Magaletti, R. Alugubelli, Gabriel De Avila, Erez Persi, F. Maura, Elissa Bell, Ryan T Bishop, Christopher Cubitt, S. Sansil, Wei Zhang, Jamie K Teer, Mingxiang Teng, Sean J Yoder, E. Siegel, Bijal Shah, T. Nishihori, Lori Hazlehurst, Conor C Lynch, O. Landgren, Oliver Hampton, Robert Gatenby, Daniel Sullivan, J. Brayer, William S. Dalton, John L Cleveland, Melissa Alsina, Rachid C Baz, K. Shain
          </td>
          <td>2025-04-15</td>
          <td>Research Square</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a113f78cc11aa5cfca10446523fbea0eb55d404" target='_blank'>
              Abstract 1512: EGR1 regulates phenotypic plasticity to promote breast cancer progression upon metabolic stress through elaborating HERPUD1-mediated unfolded protein response
              </a>
            </td>
          <td>
            Yi-Zhen Wu, Ge-Jyun Liu, Shan-Ting Lei, Wei-Huan Zeng, Mei-Ju Juan, Yu-Shu Liu, Yong-Han Su, David K. Ann, Ching-Ying Kuo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Clinical analyses demonstrated treatment naive DDR status was associated with different responses to frontline chemotherapy. Using longitudinal liquid biopsies, we found that DDR Intermediate and High tumors exhibited subtype switching and coincident emergence of heterogenous phenotypes following frontline treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes and may be associated with different chemotherapy responses and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02291-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671674f412412e8378b84a598f7d2a87a83c70ce" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2025-03-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Background Therapy-induced senescence (TIS) is considered a permanent cell cycle arrest following DNA-damaging treatments; however, its irreversibility has recently been challenged. Here, we demonstrate that escape from TIS is universal across breast cancer cells. Moreover, TIS provides a reversible drug resistance mechanism that ensures the survival of the population, and could contribute to relapse. Methods TIS was induced in four different breast cancer cell line with high-dose chemotherapy and cultured until cells escaped TIS. Parental, TIS and repopulating cells were analyzed by bulk and single-cell RNA sequencing and surface proteomics. A genetically engineered mouse model of triple-negative breast cancer was used to prove why current senolytics cannot overcome TIS in tumors. Results Screening the toxicity of a diverse panel of FDA-approved anticancer drugs revealed that TIS meditates resistance to half of these compounds, despite their distinct mechanism of action. Bulk and single-cell RNA sequencing, along with surface proteome analysis, showed that while parental and repopulating cells are almost identical, TIS cells are significantly different from both, highlighting their transient nature. Furthermore, investigating dozens of known drug resistance mechanisms offered no explanation for this unique drug resistance pattern. Additionally, TIS cells expressed a gene set associated with immune evasion and a potential KRAS-driven escape mechanism from TIS. Conclusion Our results reveal that TIS, as a transient drug resistance mechanism, could contribute to overcome the immune response and to relapse by reverting to a proliferative stage. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02310-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838695afba129f01c7b6f860f5a9601961169be" target='_blank'>
              Therapy-induced senescence is a transient drug resistance mechanism in breast cancer
              </a>
            </td>
          <td>
            Eszter Bajtai, Csaba Kiss, Éva Bakos, Tamás Langó, Anna Lovrics, É. Schád, V. Tisza, Károly Hegedűs, Péter Fürjes, Zoltán Szabó, Gábor E. Tusnády, G. Szakács, Á. Tantos, S. Spisák, J. Tóvári, András Füredi
          </td>
          <td>2025-05-01</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/217101cddf06b5ca15708fdd50f15de9f63e9eb9" target='_blank'>
              Abstract 5092: Characterization of mismatch repair status in colorectal cancer using single cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Dimitri Joseph, Alexandra Guillaume
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbcd5ab9fc14637f7ac25a5d726846fbbce055ae" target='_blank'>
              Abstract 2396: Gene fusion and variant-aware isoform detection with functional prediction from long-read RNA sequencing
              </a>
            </td>
          <td>
            Colette Felton, Andrea Galvez, L. Sayles, Christopher Vollmers, Alejandro Sweet-Cordero, Angela N. Brooks
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor subclones refer to distinct cell populations within the same tumor that possess different genetic characteristics. They play a crucial role in understanding tumor heterogeneity, evolution, and therapeutic resistance. The formation of tumor subclones is driven by several key mechanisms, including the inherent genetic instability of tumor cells, which facilitates the accumulation of novel mutations; selective pressures from the tumor microenvironment and therapeutic interventions, which promote the expansion of certain subclones; and epigenetic modifications, such as DNA methylation and histone modifications, which alter gene expression patterns. Major methodologies for studying tumor subclones include single-cell sequencing, liquid biopsy, and spatial transcriptomics, which provide insights into clonal architecture and dynamic evolution. Beyond their direct involvement in tumor growth and invasion, subclones significantly contribute to tumor heterogeneity, immune evasion, and treatment resistance. Thus, an in-depth investigation of tumor subclones not only aids in guiding personalized precision therapy, overcoming drug resistance, and identifying novel therapeutic targets, but also enhances our ability to predict recurrence and metastasis risks while elucidating the mechanisms underlying tumor heterogeneity. The integration of artificial intelligence, big data analytics, and multi-omics technologies is expected to further advance research in tumor subclones, paving the way for novel strategies in cancer diagnosis and treatment. This review aims to provide a comprehensive overview of tumor subclone formation mechanisms, evolutionary models, analytical methods, and clinical implications, offering insights into precision oncology and future translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1d4c21265c412fc5d4c0775f9aa0ecd36b6c8eb" target='_blank'>
              Advances in tumor subclone formation and mechanisms of growth and invasion
              </a>
            </td>
          <td>
            Yuhong Zhang, Weidong Wang
          </td>
          <td>2025-04-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive variant and androgen receptor (AR)-independent castration resistant prostate cancers (CRPC) represent the most significant diagnostic and therapeutic challenges in prostate cancer. This study examined a case of simultaneous progression of both adenocarcinoma and squamous tumors from the same common origin. Using whole-genome and transcriptome sequencing from 17 samples collected over >6 years, we established the clonal relationship of all samples, defined shared complex structural variants, and demonstrated both divergent and convergent evolution at AR. Squamous CRPC-associated circulating tumor DNA was identified at clinical progression prior to biopsy detection of any squamous differentiation. Dynamic changes in the detection rate of histology-specific clones in circulation reflected histology-specific sensitivity to treatment. This dataset serves as an illustration of non-neuroendocrine transdifferentiation and highlights the importance of serial sampling at progression in CRPC for the detection of emergent non-adenocarcinoma histologies with implications for the treatment of lineage plasticity and transdifferentiation in metastatic CRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65b072299240230cb903f6cbde3bd730a8642b8" target='_blank'>
              A complex phylogeny of lineage plasticity in metastatic castration resistant prostate cancer
              </a>
            </td>
          <td>
            J. Nauseef, Timothy R. Chu, William F Hooper, Alicia Alonso, Ali Oku, Heather Geiger, Zoe R. Goldstein, Minita J. Shah, M. Sigouros, J. Manohar, Zoe Steinsnyder, Lara Winterkorn, Brian D. Robinson, A. Sboner, H. Beltran, O. Elemento, I. Hajirasouliha, Marcin Imielinski, D. Nanus, S. Tagawa, N. Robine, Juan Miguel Mosquera
          </td>
          <td>2025-03-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f680cbd3c45701162ce05be4c984e82cf3cb95" target='_blank'>
              Abstract 1258: Deciphering evolutionary dynamics and transcriptomic adaptions under therapy in small cell lung cancer
              </a>
            </td>
          <td>
            L. Kaiser, Marcel Schmiel, Christian Müller, Maria Cartolano, M. Peifer, Roman K. Thomas, Julie George
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9567c766da45eca4af083f72109f47e2c4c4d5f4" target='_blank'>
              Abstract 5130: DNA methylation profiling reveals epigenetic dynamics across epithelial-mesenchymal transition states in the triple-negative breast tumor microenvironment
              </a>
            </td>
          <td>
            Hanxu Lu, F. Kolling, Lucas A Salas, D. Pattabiraman, Todd W Miller, Brock C. Christensen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c9f0f37888749fdd1b5acd90887f2484e915ec3" target='_blank'>
              Abstract 5553: Acquisition of adaptive and reversible drug-tolerance state to cabozantinib in renal cell carcinoma cell lines
              </a>
            </td>
          <td>
            A. Zaccagnino, Hiresh Ayoubian, Kerstin Junker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Bone metastases represent frequent and severe complications in various cancers, notably impacting prognosis and quality of life. This review article delves into the genetic and epigenetic mechanisms underpinning drug resistance in bone metastases, a key challenge in effective cancer treatment. The development of drug resistance in cancer can manifest as either intrinsic or acquired, with genetic heterogeneity playing a pivotal role. Intrinsic resistance is often due to pre-existing mutations, while acquired resistance evolves through genetic and epigenetic alterations during treatment. These alterations include mutations in driver genes like TP53 and RB1, epigenetic modifications such as DNA methylation and histone changes, and pathway alterations, notably involving RANK-RANKL signaling and the PI3K/AKT/mTOR cascade. Recent studies underline the significance of the tumor microenvironment in fostering drug resistance, with components such as cancer-associated fibroblasts and hypoxia playing crucial roles. The interactions between metastatic cancer cells and the bone microenvironment facilitate survival and the proliferation of drug-resistant clones. This review highlights the necessity of understanding these complex interactions to develop targeted therapies that can overcome resistance and improve treatment outcomes. Current therapeutic strategies and future directions are discussed, emphasizing the integration of genomic profiling and targeted interventions in managing bone metastases. The evolving landscape of genetic research, including the application of next-generation sequencing and CRISPR technology, offers promising avenues for novel and more effective therapeutic strategies. This comprehensive exploration aims to provide insights into the molecular intricacies of drug resistance in bone metastases, paving the way for improved clinical management and patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed3b250a10d84b32555ed5529bedbf646475e86e" target='_blank'>
              The genetic architecture of bone metastases: unveiling the role of epigenetic and genetic modifications in drug resistance
              </a>
            </td>
          <td>
            Ahmad Dawalibi, Mohamad Bakir, Khalid S. Mohammad
          </td>
          <td>2025-04-22</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a3bde6f2ebc33e7239f55d9b0ddb6732733f31" target='_blank'>
              Abstract 4190: CRISPR-CAS9 library screening identifies novel vulnerabilities in CRC drug-tolerant persister cells
              </a>
            </td>
          <td>
            Martina Miotto, S. Lamba, G. Grasso, G. Corti, Julie Bonetto, A. Sogari, A. Bardelli, Mariangela Russo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Triple-negative breast cancer, characterized by aggressive growth and high intratumor heterogeneity, presents a significant clinical challenge. Here, we use a lineage-tracing system, ClonMapper, which couples heritable clonal identifying tags with single-cell RNA-sequencing (scRNA-seq), to better elucidate the response to doxorubicin in a model of TNBC. We demonstrate that, while there is a dose-dependent reduction in overall clonal diversity, there is no pre-existing resistance signature among surviving clones. Separately, we found the existence of two transcriptomically distinct clonal subpopulations that remain through the course of treatment. Among clones persisting across multiple samples we identified divergent phenotypes, suggesting a response to treament independent of clonal identity. Finally, a subset of clones harbor novel changes in expression following treatment. The clone and sample specific responses to treatment identified herein highlight the need for better personalized treatment strategies to overcome tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88360a99c9dba219540b793cdb094157ffa2ec6a" target='_blank'>
              Lineage Tracing Reveals Clone-Specific Responses to Doxorubicin in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Daylin Morgan, Andrea L. Gardner, Amy Brock
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc3b5fa458a00b7ac6aa3d0d25d390376657ca" target='_blank'>
              Abstract 5544: Identification, isolation and molecular characterization of drug-resistant sub-populations of pancreatic cancer cells
              </a>
            </td>
          <td>
            S. Choudhury, Shixiong Wang, Y. Akimov, Katarina Willoch, Biswajyoti Sahu, Alfonso Urbanucci, Thomas Fleischer, T. Aittokallio
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076e8ebfb600d70faa36a964146ab32ad3cd766a" target='_blank'>
              Screening and application of unstable genetically resistant strains in fission yeast.
              </a>
            </td>
          <td>
            Lu-Feng Dan, Yi-Wen Chu, Xin-Rong Wang, Xiang-Wei He
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd635bb806c9651d9badaf4a27cb3a0268b9273e" target='_blank'>
              Synergistic Targeting of EGFR, ESR1, BCL2, and TP53 Pathways: A Multi-Pronged Approach for Advanced Breast Cancer Therapy.
              </a>
            </td>
          <td>
            Harneet Marwah, H. K. Dewangan
          </td>
          <td>2025-04-24</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John P. McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank J. Colón-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea7a11f970c3515d6da1945980a0668b04d9e68" target='_blank'>
              Abstract 4189: CRISPR-edited cell lines targeting key molecular pathways: Accelerating the development of synthetic lethal therapeutics beyond PARP inhibitors
              </a>
            </td>
          <td>
            Yue Huang, Yunpeng Zhai, Yan Feng, Xinyue Jia, Xiaohuan Zhang, Jinying Ning, F‐J Hao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cd0035313cd95f13c0ad0b1db10655d699eb842" target='_blank'>
              Abstract 2991: Systematic discovery of regulators of adoptive T-cell therapeutic resistance using in vivo CRISPR screening
              </a>
            </td>
          <td>
            Chenlin Zhao, Sharif U. Ahmed, Xiyue Hu, R. S. Atwal, Zongjie Wang, Sophie Xiao, S. Kelley
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a1ac1da292bdfa7cb083f4691cc613492b11ea" target='_blank'>
              Pan-cancer Analysis Identified Ectopic RUNX1T1 Associated with Lineage Plasticity
              </a>
            </td>
          <td>
            Yuyin Jiang, Siyuan Cheng, I. Kim, Su Deng, Ping Mu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Development of next generation sequencing technologies allows to identify a large number of genetic landscape types in various cancers including breast cancer. Frequent genetic abnormalities identified using whole genome sequencing are point mutations (missense, nonsense mutations), deletions, insertions, which usually lead to activation of protooncogenes and inactivation of tumor suppressor genes. Genome sequencing of malignant tumors allowed, on one hand, to identify driver mutations in carcinogenic genes in different organs, and on the other – to use mutated genes for targeted therapy. Study of biological functions of these genes from the point of view of their contribution to carcinogenesis allows to better understand its mechanism. In this review, signaling cascades of breast cancer with identified mutated genes – targets for therapy – are analyzed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d647ad1e76680b2e0c017bae0688ab2ee0a334" target='_blank'>
              Signaling cascades are targets for breast cancer therapy in the light of genome – wide sequencing data
              </a>
            </td>
          <td>
            L. F. Gulyaeva, M. Filipenko, N. E. Kushlinskii
          </td>
          <td>2025-04-13</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf51075c6ced2d719b16f81522dd0e475d5f9454" target='_blank'>
              Abstract 4605: Exploring resistance mechanisms in cancer using patient-derived organoids reveals replication stress and EMT as potential biomarkers
              </a>
            </td>
          <td>
            Ruben M. Drews, Finnian Firth, Paul R. Barber, Anna Pasto, Tony T. Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6afa5982640193cfc266d2068be1bacf5082da2a" target='_blank'>
              Abstract 2648: ZFP36L2 orchestrates stress adaptive plasticity during injury repair and metastasis
              </a>
            </td>
          <td>
            Qingwen Jiang, Manisha Raghavan, Britney Forsyth, Aileen Rodriguez, Cyrus Tam, Sasha Balkaran, S. Hartner, Morgan Lallo, Ahmed Mahmoud, Andrew Moorman, Jura Pintar, D. Pe’er, R. Koche, Quaid Morris, Joseph Chan, K. Ganesh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background Survival of patients with metastatic castration-resistant prostate cancer (mCRPC) depends on the site of metastatic dissemination. Methods Patients with mCRPC were prospectively included in the CPCT-02 metastatic site biopsy study. We evaluated whole genome sequencing (WGS) of 378 mCRPC metastases to understand the genetic traits that affect metastatic site distribution. Results Our findings revealed that RB1, PIK3CA, JAK1, RNF43, and TP53 mutations are the most frequent genetic determinants associated with site selectivity for metastatic outgrowth. Furthermore, we explored mutations in the non-coding genome and found that androgen receptor (AR) chromatin binding sites implicated in metastatic prostate cancer differ in mutation frequencies between metastatic sites, converging on pathways that impact DNA repair. Notably, liver and visceral metastases have a higher tumor mutational load (TML) than bone and lymph node metastases, independent of genetic traits associated with neuroendocrine differentiation. We found that TML is strongly associated with DNA mismatch repair (MMR)-deficiency features in these organs. Conclusions Our results revealed gene mutations that are significantly associated with metastatic site selectivity and that frequencies of non-coding mutations at AR chromatin binding sites differ between metastatic sites. Immunotherapeutics are thus far unsuccessful in unselected mCRPC patients. We found a higher TML in liver and visceral metastases compared to bone and lymph node metastases. As immunotherapeutics response is associated with mutational burden, these findings may assist in selecting mCRPC patients for immunotherapy treatment based on organs affected by metastatic disease. Trial registration number NCT01855477. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01445-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc8017838415dd6bdeb817e19ffdcf7da3197701" target='_blank'>
              Whole genome sequencing of 378 prostate cancer metastases reveals tissue selectivity for mismatch deficiency with potential therapeutic implications
              </a>
            </td>
          <td>
            Daniel J Vis, S. Palit, Marie Corradi, Edwin Cuppen, Niven Mehra, M. Lolkema, L. Wessels, M. S. van der Heijden, W. Zwart, A. Bergman
          </td>
          <td>2025-03-20</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b08a5b6171c80002da0a1517c54be1999c70c6" target='_blank'>
              MED12 dysregulation: insights into cancer and therapeutic resistance.
              </a>
            </td>
          <td>
            R. Bhole, Jagruti Shinkar, Sonali Labhade, Pawan N Karwa, Harshad S. Kapare
          </td>
          <td>2025-03-19</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728e53382096a103c9861e4f20d349955371bb" target='_blank'>
              Abstract 2726: A circadian regulator drives tumor lineage plasticity through chromatin bivalency resolution
              </a>
            </td>
          <td>
            Yatian Yang, Xiong zhang, hongye zou, E. Corey, Ronald M. Evans, Amina Zoubeidi, Hongwu Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b4672987b5633b49ff2536422175c3660e10c3a" target='_blank'>
              Identification of a shared persistence program in triple-negative breast cancer across treatments and patients
              </a>
            </td>
          <td>
            Léa Baudre, Grégoire Jouault, Pacôme Prompsy, Mélissa Saichi, Sarah Gastineau, Christophe Huret, Laura Sourd, A. Dahmani, Elodie Montaudon, F. Dingli, D. Loew, Elisabetta Marangoni, J. Marsolier, C. Vallot
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9d728476d907142dd31d1c87ffc2d2c3f782be" target='_blank'>
              Abstract 1248: Single-cell RNAseq revealed multiple resistance mechanisms in patient-derived xenograft model of rectal cancer during treatment
              </a>
            </td>
          <td>
            Yuwei Song, Regina K. Irwin, Karin M. Hardiman, Zechen Chong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca6f30d3b9e64a4de714f44e69384d7201b4a8d" target='_blank'>
              Abstract 440: Dual epigenetic targeting of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Kyounghyun Kim, Ponmari Guruvaiya
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc2b803c7ac7b99f1e6df1e4aac1c92682211a5" target='_blank'>
              Abstract 5557: Deciphering the interplay of β-catenin/YAP signaling and alveolar lineage plasticity during targeted therapy in non-small cell lung cancer
              </a>
            </td>
          <td>
            Yu-Ting Chou, Wei Wu, D. Kerr, V. Olivas, Macey Slota, T. Bivona
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background Recent genomic analyses of poor prognostic and relapsed leukemia have revealed the involvement of diverse gene mutations in treatment resistance. These gene mutations are classified into two groups: mutations involving resistance to specific agents such as the BCR::ABL1 fusion gene mutations (typically T315I mutation) in tyrosine kinase inhibitor (TKI) resistance and those involving the resistance to diverse therapeutic modalities such as the TP53 gene mutations. In the latter type, although their associations with drug resistance have been clinically demonstrated, the direct association with resistance to each therapeutic modality remains to be fully elucidated. To overcome treatment resistance induced by these gene mutations, appropriate leukemic cellular models are urgently required. Using the cytidine base editing (CBE) system, we introduced two types of mutations through C-to-T transition into human leukemia cell lines and evaluated their significance in the drug sensitivities. Methods We applied the CBE system to introduce the T315I (ACT to ATT) mutation of the BCR::ABL1 fusion gene in a human Philadelphia chromosome-positive leukemia cell line and to introduce the T125M (ACG to ATG) mutation of the TP53 gene in a human B-cell precursor acute lymphoblastic leukemia (BCP-ALL) cell line. Results We first confirmed an introduction of the T315I mutation in one of four BCR::ABL1 alleles as a result of base editing in the obtained TKI-resistant subline. We also identified the additional C-to-T transition at adjacent codon 314 (ATC), which resulted in a silent mutation, in the same allele. We next confirmed that the obtained subline acquired the T125M mutation of the TP53 gene without additional C-to-T transition. In the T125M subline, transcriptional activities of the p53 protein were disrupted and the sensitivities to diverse chemotherapeutic drugs and irradiation were reduced. Conclusion Our observations demonstrated the utility of the CBE system for introducing specific nucleotide transitions into human leukemia cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb1ade1d81a455d4e9f820780a2721c7705b4dbb" target='_blank'>
              Utility of the Base Editing System for Introducing Drug-Resistant Gene Mutations Into Human Leukemia Cellular Models
              </a>
            </td>
          <td>
            Thao Nguyen, Minori Tamai, Shinichi Fujisawa, Akiko Nagamachi, K. Kagami, Chiaki Komatsu, Shotaro Iwamoto, Toshiya Inaba, Takanori Teshima, Takeshi Inukai
          </td>
          <td>2025-04-01</td>
          <td>Cureus</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01f5cfa8e398963dd5157942872b501bf3ce27d8" target='_blank'>
              Abstract 442: Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance
              </a>
            </td>
          <td>
            Manav Gupta, Pranav Gupta, Scott R. Tyler, N. Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, I. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel L. Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, J. Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca M. Choy, Richard Zang, Linghu Xin, Yuchen Bai, P. Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, J. Taygerly, Jacob Burch-Konda, P. Barsanti, Michael A. White, Brian T. Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dec962af7d1ee05246dee3d378f9152a2d6c3e" target='_blank'>
              Genetic variants in NUDT15 gene their clinical implications in cancer therapy.
              </a>
            </td>
          <td>
            Y. Jarrar, Maria Ghishan, Fatima Khirfan, Nancy Hakooz
          </td>
          <td>2025-04-14</td>
          <td>Drug metabolism and personalized therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematological malignancy. Patients with wild-type FLT3 relapsed or refractory (R/R) AML face significant therapeutic challenges due to the persistent lack of effective treatments. A comprehensive understanding of the mechanisms underlying chemotherapy resistance is needed to the development of effective treatment strategies. Therefore, we investigated the molecular mechanisms underlying cytarabine (Ara-C) resistance and daunorubicin (DNR) tolerance in Ara-C-resistant RHI-1 cells derived from the wild-type FLT3 AML cell line SHI-1. Quantitative analysis of intracellular drug concentrations, proteomics, and phosphoproteomics showed that DNR resistance in Ara-C-resistant RHI-1 cells is driven by metabolic remodeling toward mitochondrial metabolism, upregulation of DNA repair pathways, and enhanced reactive oxygen species (ROS) detoxification rather than reduced drug uptake. Moreover, targeting these compensatory mechanisms, particularly the OXPHOS complex I proteins, significantly improved the efficacy of both Ara-C and DNR. Conclusively, these findings highlight mitochondrial metabolism and DNA repair as critical factors in chemotherapy resistance and offer valuable insights into potential therapeutic targets for enhancing treatment outcomes in patients with wild-type FLT3 R/R AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49a8e37a3ebb622ad11d194102e12d1292163e3" target='_blank'>
              Mechanisms of chemotherapy failure in refractory/relapsed acute myeloid leukemia: the role of cytarabine resistance and mitochondrial metabolism
              </a>
            </td>
          <td>
            Soo Yeon Chae, Se-Young Jang, Jinhui Kim, Sehyun Hwang, D. Malani, O. Kallioniemi, S. Yun, Jong-Seo Kim, Hugh I Kim
          </td>
          <td>2025-04-23</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ac37cc5cb3a2b919750ef1ac7fd1c5b0996b8dd" target='_blank'>
              Abstract 2487: LINMAP: A framework for comprehensive cell division lineage reconstruction via molecular barcoding and machine learning
              </a>
            </td>
          <td>
            Lin Wang, Zheng Yin, Wenjuan Dong, J. Sheng, Li Yang, Kun Han, Dingcheng Gao, Stephen T. Wong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce9d10ba962b31f50d67da52456440cdb7f8eb0c" target='_blank'>
              Abstract 3880: Functional dissection of the highly plastic basal cell state in pancreatic cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Kate Ryan, Jung Yun Kim, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G. Christensen, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37bbf9a9279b9ea882191a6ab180a2f1195c66ee" target='_blank'>
              Abstract 1246: Driver and signaling cascade in therapy pressure driven cancer cell plasticity
              </a>
            </td>
          <td>
            Poorvi Subramanian, Sreenidhi Mohanvelu, Sivasubramani Narayanan, Afsana Parveen Jahir Hussain, Sabir Salim, Madhi Oli Ramamurthy, Loganayaki Periyasamy, N. Aravindan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7a11c8b2dc4731f11b9133f359e7ac89041b4a" target='_blank'>
              APOBEC3 activity promotes the survival and evolution of drug-tolerant persister cells during EGFR inhibitor resistance in lung cancer.
              </a>
            </td>
          <td>
            N. M. G. Garcia, Jessica N Becerra, Sharan Srinivasan, Brock J McKinney, Ashley V DiMarco, Feinan Wu, Matthew Fitzgibbon, James V. Alvarez
          </td>
          <td>2025-05-05</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef1937cc4baf82f21bbced66813abdd14ff16f89" target='_blank'>
              Application and research of genomic optical mapping technology in disease diagnosis.
              </a>
            </td>
          <td>
            Jing Quan, Yan-Qun Xiao, Da-Ru Lu, Yun Bao
          </td>
          <td>2025-04-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7826ec8e0e8cb2ddf7c9f777c6ebf6c3f9a237fa" target='_blank'>
              Advancing chronic myeloid leukemia research with next-generation sequencing: potential benefits, limitations, and future clinical integration.
              </a>
            </td>
          <td>
            Henry Sutanto, Laras Pratiwi, P. Romadhon, S. U. Y. Bintoro
          </td>
          <td>2025-04-21</td>
          <td>Human genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a11b087a0a9fc0566d0ad6c1ba6f886b3c8f91ce" target='_blank'>
              Abstract 6385: Heterogeneity in sensitivity to second-line inhibitors of therapy-resistant melanoma
              </a>
            </td>
          <td>
            Gianna T. Busch, Ryan H. Boe, Jingxin Li, Pavithran T. Ravindran, Arjun Raj
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2d4cdd26b00d23c9f2af55b08e343fcfbd17a4" target='_blank'>
              Abstract 1469: Deciphering mechanisms of intrinsic radioresistance in breast cancer: Insights from transcriptomics and CRISPRi screens
              </a>
            </td>
          <td>
            B. McBean, Priyanka S. Rana, Benjamin Hauk, A. Michmerhuizen, S. Lichtman-Mikol, Reine Abou Zeidane, Vesna Markovic, K. Wilder-Romans, Mingfang Tao, Alan P. Boyle, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9213a2dd4a654c0cc05ece63777d6864d7e262c3" target='_blank'>
              Role of Histone Deacetylases in Drug-Resistant Melanoma: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            B. Kalal
          </td>
          <td>2025-04-21</td>
          <td>Kinases and Phosphatases</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acaaf06b5fb2fb1345caa571c60786539fe457bf" target='_blank'>
              Somatic TP53 Mutations Drive T and NK Cell Dysfunction in AML and Can be Rescued by Reactivating Wild Type p53
              </a>
            </td>
          <td>
            Li Li, M. Muftuoglu, Edward Ayoub, Jiangxing Lv, Mahesh Basyal, A. Bidikian, Ran Zhao, Menon Prashant, Poo Yee Mak, Rupesh K. Kesharwani, Yuki Nishida, G. Issa, N. Varadarajan, B. Carter, Michael Andreeff
          </td>
          <td>2025-04-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The use of CRISPR to knockdown or knockout genes is a powerful tool for understanding the specific role of a gene in disease development. However, it can cause many unanticipated changes to the transcriptome that are not detected by DNA amplification and Sanger sequencing of the target site. Various RNA-sequencing techniques can be used to identify these changes and effectively gauge the full impact of the CRISPR knockout, thereby providing a means of selecting appropriate clones for further experimentation. Background/Objectives: RNA-seq data from 4 CRISPR knockout experiments were analyzed and techniques developed to both confirm the success of the CRISPR modifications and identify potential issues. Methods: A broad-based analysis of RNA-sequencing data identified many CRISPR-based changes not identified by PCR amplification of DNA around the CRISPR target site. These changes included an inter-chromosomal fusion event, exon skipping, chromosomal truncation, and the unintentional transcriptional modification and amplification of a neighboring gene. Conclusions: The inadvertent modifications identified by the evaluation of 4 CRISPR experiments highlight the value of using RNA-seq to identify transcriptional changes to cells altered by CRISPR, many of which cannot be recognized by evaluating DNA alone. Specific guidelines are presented for designing and analyzing CRISPR experiments using RNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/220a9a7ebf2665e1a3e4bd65d2ca79919ea760ce" target='_blank'>
              Techniques for Validating CRISPR Changes Using RNA-Sequencing Data
              </a>
            </td>
          <td>
            S. Rathe, Tracy A. Marko, Elizabeth N. Edwards, Paige Hazelton Ridder, Jyotika Varshney, Kyle B. Williams, James E. Johnson, B. Moriarity, D. A. Largaespada
          </td>
          <td>2025-03-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf13a264e1cd78c0d16de17ca799be490bf7e832" target='_blank'>
              Challenges and Pathways in Regulating Next-Gen Biological Therapies.
              </a>
            </td>
          <td>
            Surendra Agrawal, Sunita Vaidya, Jitendra Patel, P. Jirvankar, Pravina Gurjar
          </td>
          <td>2025-04-23</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Heterogeneity is inherent to living organisms and it determines cell fate and phenotypic variability. Despite its ubiquity, the underlying molecular mechanisms and the genetic basis linking genotype to-phenotype heterogeneity remain a central challenge. Here we construct a yeast knockout library with a clone and genotype RNA barcoding structure suitable for genome-scale analyses to generate a high-resolution single-cell yeast transcriptome atlas of 3500 mutants under control and stress conditions. We find that transcriptional heterogeneity reflects the coordinated expression of specific gene programs, generating a continuous of cell states that can be responsive to external insults. Cell state plasticity can be genetically modulated with mutants that act as state attractors and disruption of state homeostasis results in decreased adaptive fitness. Leveraging on intra-genetic variability, we establish that regulators of transcriptional heterogeneity are functionally diverse and influenced by the environment. Our multimodal perturbation-based single-cell Genotype-to-Transcriptome Atlas in yeast provides insights into organism-level responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89465a1373d3260110cc521e62c1032cd2c67144" target='_blank'>
              A single-cell resolved genotype-phenotype map using genome-wide genetic and environmental perturbations
              </a>
            </td>
          <td>
            Mariona Nadal-Ribelles, Carme Solé, Anna Díez-Villanueva, Camille Stephan-Otto Attolini, Yaima Matas, Lars M. Steinmetz, Eulàlia de Nadal, F. Posas
          </td>
          <td>2025-03-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c8de13c6fd974bf26d03a422a6ee56b54180c3" target='_blank'>
              Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, F. Cidre-Aranaz, A. Wenninger-Weinzierl, M. Zylka, Karla Queiroz, D. Kurek, A. Sastre, Javier Alonso, Martin Distel, Anna Obenauf, T. G. Grünewald, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eae4df06f51107c25979631d45f7a492cc54031" target='_blank'>
              Extrachromosomal Circular DNA in Cancer: Mechanisms and Clinical Applications.
              </a>
            </td>
          <td>
            Jiajia Li, Peng Luo, Zhengrui Li, Qi Wang, Xufeng Huang, Keliang Wang, Ruo Wang, Runzhi Chen
          </td>
          <td>2025-04-29</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In recent years, synthetic lethality has become an important theme in the field of targeted cancer therapy. Synthetic lethality refers to simultaneous defects in two or more genes leading to cell death, whereas defects in any single gene do not lead to cell death. Taking advantage of the genetic vulnerability that exists within cancer cells, it theoretically has no negative impact on healthy cells and has fewer side effects than non-specific chemotherapy. Currently, targeted cancer therapies focus on inhibiting key pathways in cancer. However, it has been found that over-activation of oncogenic-related signaling pathways can also induce cancer cell death, which is a major breakthrough in the new field of targeted therapies. In this review, we summarize the conventional gene targets in synthetic lethality (PARP, ATR, ATM, WEE1, PRMT) and provide an in-depth analysis of their latest potential mechanisms. We explore the impact of over-activation of pathways such as PI3K/AKT, MAPK, and WNT on cancer cell survival, and present the technical challenges of current research. Important theoretical foundations and insights are provided for the application of synthetic lethal strategies in cancer therapy, as well as future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a12057bb33fb82ecea65af89cddf52c4a9e3cd" target='_blank'>
              Perspectives on cancer therapy—synthetic lethal precision medicine strategies, molecular mechanisms, therapeutic targets and current technical challenges
              </a>
            </td>
          <td>
            Shixuan Peng, Mengle Long, Qisheng Chen, Zhijian Yin, Chang Zeng, Wanyong Zhang, Qingyang Wen, Xinwen Zhang, Weiqi Ke, Yongjun Wu
          </td>
          <td>2025-04-16</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Drug resistance results in poor outcomes for cancer patients. Adaptive therapy is a potential strategy to address drug resistance that exploits competitive interactions between sensitive and resistant subclones. Here, we showed that adapting carboplatin dose according to tumor response (adaptive therapy) significantly prolonged survival of murine ovarian cancer models compared to standard carboplatin dosing, without increasing mean daily drug dose or toxicity. Platinum resistant ovarian cancer cells exhibited diminished fitness when drug was absent in vitro and in vivo, which caused selective decline of resistant populations due to reduced proliferation and increased apoptosis. Conversely fitter, sensitive cells re-grew when drug was withdrawn. Using a bioinformatics pipeline that exploits copy number changes to quantify the emergence of treatment resistance, analysis of cell-free DNA obtained longitudinally from ovarian cancer patients during treatment showed subclonal selection through therapy, and measurements of resistant population growth correlated strongly with disease burden. These preclinical findings pave the way for future clinical testing of personalized adaptive therapy regimens tailored to the evolution of carboplatin resistance in individual ovarian cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872bfbc7aed3e8f94862c1f84bd0f838583a3b38" target='_blank'>
              Adaptive Therapy Exploits Fitness Deficits in Chemotherapy-Resistant Ovarian Cancer to Achieve Long-Term Tumor Control.
              </a>
            </td>
          <td>
            H. Hockings, E. Lakatos, Weini Huang, M. Mossner, Mohammed A Khan, Nikolina Bakali, Jacqueline McDermott, Kane Smith, Ann-Marie Baker, Trevor A. Graham, M. Lockley
          </td>
          <td>2025-04-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0da8bbe26d7b85bc1bd170b1562ddebfb5b49ba9" target='_blank'>
              Abstract 6382: DNA damage repair pathways enable a drug-tolerant persister state in RET-fusion NSCLC and precedes TKI resistance
              </a>
            </td>
          <td>
            F. Uddin, Samuel Tischfield, R. Somwar, Alexander Lim, Hyung Jun Woo, Christina J Falcon, Barbara P. Mello, Dennis Kinyua, Harsha Sridhar, P. Manoj, Hong Zhong, Juan Qiu, E. de Stanchina, M. Donoghue, Marc Ladanyi, E. Redin, Charles M. Rudin, Á. Quintanal-Villalonga, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a58e72b48369a6d6279bf45ff8835cc869b1ce39" target='_blank'>
              CRISPR screens in the context of immune selection identify CHD1 and MAP3K7 as mediators of cancer immunotherapy resistance
              </a>
            </td>
          <td>
            Alex Watterson, G. Picco, Vivien Veninga, Youhani Samarakoon, C. M. Cattaneo, Sara F Vieira, Emre Karakoc, S. Bhosle, Thomas W Battaglia, Sarah Consonni, Timotheus Y. F. Halim, E. Voest, Mathew J. Garnett, Matthew A. Coelho
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d1574fe2e02ef3f4ee02ae939fc0c878b2510b" target='_blank'>
              Abstract 2392: The use of transcriptomic analysis to identify novel drug targets for DSA drug combination studies in acute myeloid leukemia
              </a>
            </td>
          <td>
            William A Chen, Valery Filippov, Jie Fang, Pedro Ochoa, C. Casiano, Kristopher E Boyle, S. Dovat, Hongjian Jin, Jun Yang, Olivia L Francis-Boyle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13bcea3579836571808ac96de5e2e45ad09f6441" target='_blank'>
              Abstract 6418: Non-genetic drug adaptation to MAPK pathway inhibitors in melanoma
              </a>
            </td>
          <td>
            Varuna Nangia, Humza Ashraf, Nasreen Marikar, Victor J. Passanisi, Christopher Ryland Ill, Timothy E. Hoffman, Sabrina L. Spencer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dead0a4e6712a90fb575b926eb0a4af6b736d577" target='_blank'>
              Abstract 1259: Studying the role of chromatin accessibility in lineage plasticity of Rb1-deficient EGFR-mutant lung adenocarcinoma
              </a>
            </td>
          <td>
            Bradley Balk, David W. Goodrich
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a504b18b01f5f033738e84592b503d61c5582a0" target='_blank'>
              A tripartite cell-free translation system to study mammalian translation.
              </a>
            </td>
          <td>
            F. S. Arendrup, Kasper L Andersen, Anders H. Lund
          </td>
          <td>2025-04-16</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43978659ce38ef19fbe0b48f8cd99b848f7e9b3" target='_blank'>
              Abstract 7490: Uncovering rare cellular populations in esophageal preneoplasia through genetic perturbation tracking and scRNA-seq
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Jae-Il Park
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In this mini-review, we explore the advancements in genome-wide DNA methylation profiling, tracing the evolution from traditional methods such as methylation arrays and whole-genome bisulfite sequencing to the cutting-edge single-molecule profiling enabled by long-read sequencing (LRS) technologies. We highlight how LRS is transforming clinical and translational research, particularly by its ability to simultaneously measure genetic and epigenetic information, providing a more comprehensive understanding of complex disease mechanisms. We discuss current challenges and future directions in the field, emphasizing the need for innovative computational tools and robust, reproducible approaches to fully harness the capabilities of LRS in molecular diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ce2682b6582f6c12c8ed84aeeb14fe7a917240c" target='_blank'>
              Evolution of genome-wide methylation profiling technologies
              </a>
            </td>
          <td>
            Carolina Montano, W. Timp
          </td>
          <td>2025-04-01</td>
          <td>Genome Research</td>
          <td>1</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60dc48b94558a55ad5ff2dbcdf85170057c59b44" target='_blank'>
              Natural resistance to cancers in long-lived mammals: genomic mechanisms and experimental evidence to explain Peto's paradox.
              </a>
            </td>
          <td>
            Linxia Sun, Zhikang Xu, Mengqi Shuai, Chengxu Li, Guang Yang, Shixia Xu
          </td>
          <td>2025-03-21</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Background Lynch syndrome (LS), caused by germline pathogenic variants in the mismatch repair genes, leads to high rates of frameshift-derived neopeptide (FSDN) expression due to microsatellite instability (MSI). While colorectal cancer (CRC) prevention is effective, most LS-related tumors lack such strategies. Cancer vaccines targeting FSDNs offer a promising approach for immune interception in LS. This study aimed to identify and validate LS-related FSDNs to develop vaccines for cancer prevention. Methods We identified LS-related coding MS mutations and predicted FSDN with high coverage on common Human Leukocyte Antigen (HLA)-I and II alleles. We validated FSDN-associated mutations in colorectal adenomas (CrAD), endometrial cancers (EC), and CRC samples from patients with LS, non-LS tumors, and cell lines. Immunogenicity was assessed through interferon (IFN)-γ enzyme-linked immunospot and flow cytometry analysis of tissue-infiltrating lymphocytes (TILs) from LS carriers. Results We prioritized 53 HLA-I and 45 HLA-II FSDNs in MSI tumors using in silico predictions. Validation revealed 86.7% of FSDN-associated mutations present in LS-CRC samples, with a median of 7.67 (6.5–9) mutations in CrADs and 6.02 (2–10) in CRCs. Sequencing of CrAD and EC samples showed 95% and 77.5% of predicted FSDN-associated mutations, respectively. MSI cancer cell lines transcribed 69.8% of FSDNs. Immunogenicity assays showed that 71% of potential FSDNs elicited IFN-γ responses, with a median of 7.37 (1–10.75) HLA-I and 6 (2–5.75) HLA-II FSDNs per patient. After prioritizing 24 FSDN, in a cohort of 19 LS-derived samples (4 CrAD and 15 normal mucosa), 52% (10/19) demonstrated T-cell reactivity to an HLA-I neoantigen pool. CD8+CD137+ activation markers increased significantly (p=0.037) over time and peptide-specific cells were detected by pentamer staining. Conclusions Our predicted FSDN set has optimal coverage among LS carriers and can induce IFN-γ inflammatory responses in LS-derived TILs, offering an opportunity for vaccine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4e379b9e444406c0c4ab08355c1d7dbd836a26" target='_blank'>
              Discovery and validation of frameshift-derived neopeptides in Lynch syndrome: paving the way for novel cancer prevention strategies
              </a>
            </td>
          <td>
            Cristina Bayó, Giancarlo Castellano, Fátima Marín, J. Castillo-Iturra, T. Ocaña, Hardeep Kumari, M. Pellisé, Leticia Moreira, L. Rivero, M. Daca-Alvarez, Oswaldo Ortíz, S. Carballal, Rebeca Moreira, Júlia Canet-Hermida, Marta Pineda, Capella Gabriel, Georgina Flórez-Grau, Manel Juan, D. Benitez-Ribas, Francesc Balaguer
          </td>
          <td>2025-04-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Establishing DNA as the genetic material, deciphering the genetic code and defining the double helical structure of the genome provided a foundation for pursuing a mechanistic understanding of gene expression that is operative for biological control and pathology. Equally important are a series of discoveries that have established epigenetic mechanisms, defined as the control of gene expression by non-DNA-encoded regulation, mediated by an evolving histone code that supports chromatin states, as complementary and obligatory determinants for biological activity. Evidence is accruing for pivotal contributions by epigenetic parameters of control to gene expression that is required for development and differentiation as well as for transient and sustained responsiveness to physiological regulatory cues that support cell structure, function, cell survival, tissue repair and remodeling. Epigenetic control is emerging as a decisive contributor to the regulatory compliance for the highly integrative processes that balance phenotypic function and tumor suppression with aberrant gene expression that accompanies the onset and progression of disease, strikingly illustrated with tumorigenesis and skeletal disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be4c2c01eadaece3705383df803332330abc2445" target='_blank'>
              Reflections and perspectives on epigenetically mediated biological control: compromises in cancer and skeletal pathology
              </a>
            </td>
          <td>
            Gary S. Stein
          </td>
          <td>2025-04-11</td>
          <td>Academia Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d08d157066987d58c8cc7f8f1b1b2c36ec37eca2" target='_blank'>
              Abstract 5521: Integrative analysis of acquired drug resistance to KRAS(G12D) inhibitors
              </a>
            </td>
          <td>
            Ines Pulido Endrino, Laura C. Gunder, Malek G. Massad, Julian Carretero, Takeshi Shimamura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A recent study published in Nature by Maya M. Arce and colleagues unveils the role of central gene circuits in governing the delicate balance between T cell rest and T cell activation. 1 This work bridges fundamental molecular biology with ﬁ ndings in preclinical models, providing insights into context-speci ﬁ c gene regulation and potential therapeutic targets for immune modulation. The immune system relies on highly specialized cells to maintain homeostasis and respond to external threats. Among these, the CD4 + T cell compartment exhibits remarkable versatility, including T cell subsets with almost antagonistic functions, and that are capable of toggling between resting (quiescent) and activated states depending on environmental cues. The IL-2 receptor alpha subunit (IL-2R α or CD25) serves as a critical marker for T cell activation, with distinct expression pro ﬁ les in regulatory T cells (Treg) and effector T cells (Teff), characterized by either its constitutive or transient upregulation upon stimulation, respectively. Despite its importance, the trans-regulatory networks orchestrating these state-speci ﬁ c responses remain incompletely understood. Therefore, screening interventions that modulate IL-2R α expression can serve as a means of identifying genes involved in T cell activation, offering valuable insights by comparing the regulatory mechanisms between Teff and Treg cells. The primary objective of this study was to identify and characterize the regulators of IL-2R α expression across different T cell states. To this end, the authors conducted a series of CRISPR screens in primary human Treg and Teff cells to uncover context-dependent regulators and explore their role in maintaining resting and activated states. Using a designed library of 6,000 sgRNAs targeting over a thousand genes coding for transcription factors and chromatin modi ﬁ ers in resting and stimulated Teff and Treg, they mapped dynamic gene circuits that">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2782135135ae4e2ddd37866f821d202eefe30bbe" target='_blank'>
              From CRISPR screens to circuits: identifying key regulators in T cell activation and state transitions
              </a>
            </td>
          <td>
            Maria Silvia Roman Azcona, Toni Cathomen, C. Mussolino
          </td>
          <td>2025-04-07</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a21c97e108e69ab358170dee175b96d9645c472" target='_blank'>
              Cancer Dependency Map: Past and Present
              </a>
            </td>
          <td>
            Rand Arafeh, T. Shibue, Francisca Vazquez, William C Hahn
          </td>
          <td>2025-04-11</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/572fd6e1dc6d845d6923c7249ae200e2136415b1" target='_blank'>
              Genome-wide transcriptomic response of whole blood to radiation
              </a>
            </td>
          <td>
            Ahmed Salah, D. Wollschläger, Maurizio Callari, H. Schmidberger, Federico Marini, S. Zahnreich
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eab0fc38af5c962f28721cc54ecd1853477def0" target='_blank'>
              Single cell multi-omic whole genome sequencing and chromatin accessibility profiling reveals genome-epigenome coevolution in colorectal cancer
              </a>
            </td>
          <td>
            M. Mossner, Chloé Colson, Qingli Guo, Freddie Whiting, Heather E. Grant, Amanda Stafford, Peter Kyle, Daniella Donato-Brown, Jamie Murphy, A. Sottoriva, Ann-Marie Baker, Trevor A. Graham
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background About 40% of relapsed or non-responder tumors exhibit therapeutic resistance in the absence of a clear genetic cause, suggesting a pivotal role of intracellular communication. A deeper understanding of signaling pathways rewiring occurring in resistant cells is crucial to propose alternative effective strategies for cancer patients. Methods To achieve this goal, we developed a novel multi-step strategy, which integrates high sensitive mass spectrometry-based phosphoproteomics with network-based analysis. This strategy builds context-specific networks recapitulating the signaling rewiring upon drug treatment in therapy-resistant and sensitive cells. Results We applied this strategy to elucidate the BCR::ABL1-independent mechanisms that drive relapse upon therapy discontinuation in chronic myeloid leukemia (CML) patients. We built a signaling map, detailing - from receptor to key phenotypes - the molecular mechanisms implicated in the control of proliferation, DNA damage response and inflammation of therapy-resistant cells. In-depth analysis of this map uncovered novel therapeutic vulnerabilities. Functional validation in patient-derived leukemic stem cells revealed a crucial role of acquired FLT3-dependency and its underlying molecular mechanism. Conclusions In conclusion, our study presents a novel generally applicable strategy and the reposition of FLT3, one of the most frequently mutated drivers of acute leukemia, as a potential therapeutic target for CML relapsed patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02185-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff332b1ad425972fa586920b8d83716a55d5548" target='_blank'>
              A network-based approach to overcome BCR::ABL1-independent resistance in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Valeria Bica, Veronica Venafra, Giorgia Massacci, Simone Graziosi, Sara Gualdi, G. Minnella, F. Sorà, P. Chiusolo, Maria Elsa Brunetti, G. Napolitano, Massimo Breccia, Dimitrios Mougiakakos, Martin Böttcher, Thomas Fischer, Livia Perfetto, Francesca Sacco
          </td>
          <td>2025-04-10</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="Background & Aims Patients with colorectal cancer have heterogeneous clinical responses to chemotherapy, although clinical guidelines advise little variability in treatment selection based on molecular tumor features. Precision oncology research typically utilizes patient-derived tumor organoids (PDTO) to predict clinical outcomes, but such efforts are often not directed towards identification of molecular factors underlying differential responses to therapy. Methods Bulk RNA-sequencing was performed on treatment-naive PDTOs, and gene expression data was combined to drug sensitivity data to identify transcriptomic features associated with low in vitro sensitivity to chemotherapy. Whole-exome sequencing was performed on primary tumors to infer the somatic mutations of PDTOs and used to identify somatic mutations associated with differential in vitro drug responses. Publicly available gene expression and drug sensitivity data sets were used to validate the results. RNA interference was used for functional validation. Results PDTOs with low chemosensitivity had high JAK-STAT pathway activity resulting from high expression of interferon-stimulated genes. Evidence from single-cell RNA-sequencing confirmed chemotherapy-induced expression of interferon-stimulated genes in epithelial cells of cancers with partial response. EPSTI1 knockdown decreased cancer cell viability and sensitized cells to chemotherapy. Conclusions Sustained interferon signaling in epithelial cancer cells contributes to incomplete pathologic response in colorectal cancer. The findings highlight the potential of JAK-STAT inhibition or TRAIL pathway activation to enhance chemotherapy efficacy. Future studies investigating pharmacologic modulation of these pathways in preclinical CRC models are needed to determine their viability as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5ab87f4ae646ab5e983158fb38a2489ac5894d6" target='_blank'>
              Sustained epithelial interferon signaling modulates incomplete pathologic response in colorectal cancer
              </a>
            </td>
          <td>
            E. Erkan, Emmi K. Hämäläinen, Julia Kolikova, Kornelia Kuc, K. Ojala, Meiju Kukkonen, Ismaïl Hermelo, S. Koskensalo, T. Tarvainen, Alli Leppä, I. Keränen, C. Haapamäki, E. Karjalainen, Monica Carpelan-Holmström, L. Renkonen-Sinisalo, Laura E Koskenvuo, P. Puolakkainen, J. Mecklin, Elina Virolainen, M. Nykter, L. Aaltonen, Anna H. Lepistö, Ari Ristimäki, Toni T Seppälä
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51a1965c162640161a9b607e38387b4fb02d284d" target='_blank'>
              Abstract 2104: Epigenetic, transcriptomic and genetic analyses of HR+/HER2- primary and metastatic breast cancer results in cohort stratification by aggressiveness and reveals heterogeneous mechanisms of endocrine therapy resistance
              </a>
            </td>
          <td>
            Jomar Sangalang, Tony Sourisseau, V. Scott, Diep Tran, Nusaibah Ibrahimi, A. Tran-Dien, Hakim Benkirane, N. Servant, Fabrice André, Sergey I. Nikolaev
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7822df60288084d13eeb46f92b42204c2503de4" target='_blank'>
              Abstract 4420: Identification of a novel marker gene predicting recurrence after chemotherapy with DNA-damaging agents by using drug-resistant gastric cancer organoids
              </a>
            </td>
          <td>
            Kenji Harada, Naoya Sakamoto, Shingo Sakashita, M. Kojima, Kazuaki Tanabe, Hideki Ohdan, K. Shitara, Takahiro Kinoshita, Genichiro Ishii, Wataru Yasui, Atsushi Ochiai, Shumpei Ishikawa
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00d329440f52845fe5e04ed56db29d5167fcf05d" target='_blank'>
              Abstract 6199: Interlab study supporting hereditary cancer testing in an NGS-focused landscape
              </a>
            </td>
          <td>
            Dana J. Ruminski Lowe, Deepika Philkana, K. Nahlik, Melissa Berenger, Andrew Anfora, Yves Konigshofer, Catherine Huang, R. Garlick, B. Anekella
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daad54b0fcb7c5c704adea069e134ecf7c7a856b" target='_blank'>
              Abstract 1252: Evolution of transcriptomic and epigenomic intra-tumor heterogeneity in high-grade serous ovarian cancer with chemotherapy
              </a>
            </td>
          <td>
            Yuna Landais, Juliette Bertorello, Marta Puerto, Baptiste Simon, Amélie Roehrig, Adeline Durand, Marceau Quatredeniers, C. Lamy, Fabrice Lécuru, Christophe Le Tourneau, Céline Vallot
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) harboring BCR/ABL-T315I mutation has been a challenging obstacle for targeted therapy due to the acquired resistance to tyrosine kinase inhibitor (TKI)-based therapy. Thus, it is especially urgent to investigate more effective therapeutic targets to overcome T315I-induced resistance. Here, we reported that BCR/ABL-T315I mutant CML cells possessed a long-term proliferative capacity and tolerance to metabolic stress. Importantly, we also found that selenoamino acid metabolism was increased in the bone marrows of BCR/ABL-T315I patients compared with non-T315I patients by GSEA from RNA-Seq data. Indeed, GPX1 was highly expressed in T315I mutant cells, while knockout of GPX1 significantly suppressed cell proliferation and triggered apoptosis under glucose-deprived condition. GPX1 knockout showed decreased cell metabolism signaling as well as mitochondrial gene expression by RNA-Seq. Mechanistically, GPX1 maintained mitochondrial activity and oxygen consumption rate (OCR), retaining mitochondrial redox homeostasis and oxidative phosphorylation (OXPHOS). Additionally, mercaptosuccinic acid (MSA), a GPX inhibitor, inhibited CML colony formation and induced cell apoptosis under glucose-free condition. Therefore, GPX1 is a promising therapeutic target to overcome drug resistance induced by the T315I mutation, which provides a novel approach for BCR/ABL-T315I CML treatment by disturbing mitochondrial OXPHOS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e9003b9be433ceb088a877bba6ada889eb7e9e1" target='_blank'>
              GPX1 confers resistance to metabolic stress in BCR/ABL-T315I mutant chronic myeloid leukemia cells
              </a>
            </td>
          <td>
            Jun-Dan Wang, Jin-Xing Wang, , Na Xu, Ling Zhang, Jia-Jun Liu, Rui Gao, Z. Long
          </td>
          <td>2025-05-09</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT ALK fusions, such as the classic EML4‐ALK, are known drivers of lung cancer and effective therapeutic targets. However, variant ALK fusions, including intergenic fusions like LOC388942‐ALK (LA), have been detected in increasing numbers of patients, with their roles in tumorigenesis and ALK inhibitor resistance remaining unclear. Using CRISPR/Cas9, we generated the LA fusion in A549 and H441 cells, confirming elevated ALK expression via qRT‐PCR and immunohistochemistry (IHC) staining. Functional analyses showed that LA significantly promoted tumor growth in vitro and in vivo while conferring increased resistance to alectinib. RNA‐seq revealed upregulation of the FOS pathway in LA tumors, identifying FOS as a potential therapeutic target. Subsequently, we demonstrated that FOS disruption and inhibition sensitized LA tumors to treatment. RNA‐seq profiling demonstrated that FOS depletion in LOC388942‐ALK tumor significantly downregulated multiple oncogenic pathways related to cell cycle progression, DNA replication fidelity, and extracellular matrix remodeling, suggesting a pivotal role of FOS in maintaining tumor growth. These findings establish LOC388942‐ALK as a novel oncogenic driver in lung cancer, highlighting its role in tumor growth and ALK inhibitor resistance. Targeting FOS may provide a promising therapeutic strategy for tumors harboring this intergenic fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/459d6322ba2d77e523434dfb3f269f42f6deaabb" target='_blank'>
              Identifying the Intergenic ALK Fusion LOC388942‐ALK as a Driver of Non–Small Cell Lung Cancer
              </a>
            </td>
          <td>
            X. Zhai, Manli Wang, Qi Zhang, Donglin Li, Yanmou Wu, Zuoyu Liang, Jiewei Liu, Weiya Wang, Yu Liu, Guowei Che, Qinghua Zhou, Chong Chen
          </td>
          <td>2025-03-27</td>
          <td>MedComm</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a common and highly lethal type of primary brain tumor in adults. Therapeutic failure is partly attributed to a fraction of Glioblastoma Stem Cells (GSCs) that show high levels of heterogeneity and plasticity. GSCs exist in a transcriptional gradient between two states: Developmental (Dev) and Injury Response (IR) in which IR-GSCs exhibit more invasive behaviors. While previous studies have identified fitness genes in GSCs, the genes required to establish and maintain the Dev and IR states remain poorly defined. To identify the regulators of the IR GSC state, we performed a phenotypic genome-wide CRISPR-Cas9 knockout (KO) screen in patient-derived GSCs based on cell surface expression of the IR marker CD44. Notably, we found that perturbations of the histone acetyltransferase EP300 in IR GSCs led to decreased CD44 cell surface expression, significant downregulation of gene expression signatures associated with the IR state, and to decreased self-renewal and invasion. Furthermore, genetic targeting of Ep300 in a mouse GBM model delayed tumor initiation and/or progression. Collectively, our results establish EP300 as a regulator of the IR state in GSCs and provide a mechanistic basis for its therapeutic targeting in GBM. Significance A genome-wide phenotypic CRISPR-Cas9 screen in a patient-derived Glioblastoma Stem Cell line identified the genes required to maintain the Injury-Response cellular state, with a focus on the histone acetyl transferase gene EP300. This study suggests how therapeutic targeting of cellular state could reduce the aggressiveness of GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72194daa3f94b0757cee54e56b558726a1fc4778" target='_blank'>
              Identification of Injury-Response State Determinants in Glioblastoma Stem Cells
              </a>
            </td>
          <td>
            Fatemeh Molaei, G. Macleod, Shahan Haider, Aleczandria S. Tiffany, Frank M Oteng, Jacob M. Berman, Arianna Skirzynska, Molly S. Shoichet, Daniel Schramek, Peter B. Dirks, Stéphane Angers
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Lynch Syndrome (LS) is an autosomal dominant disease characterized by germline heterozygous mutations in DNA mismatch repair (MMR) genes. High-risk LS patients may proceed to colorectal cancer (CRC). However, the drivers or biomarkers of LS benign colon tissue approaching malignant CRC are not completely understood. The similarity and difference between LS-related and nonLS-related CRC are also not well interrogated (LS-CRC vs. nonLS-CRC). This study aimed to understand the cellular changes during malignant transition in LS. Methods Single-cell RNA sequencing (scRNA-seq) was used to analyze paired biopsy samples from 3 patients with LS (cancer tissues vs. adjacent normal tissues, n=3). Single-nuclear RNA sequencing (snRNA-seq) was used to analyze a frozen biopsy sample from a patient with LS (n=1). scRNA-seq or snRNA-seq datasets from CRC were downloaded from the open source. Integrative computational analysis was performed to conclude the distinct pattern in the single-cell atlas. Immuno-histo-fluorescence staining (IHF) were also performed for three key markers. Results In the single-cell atlas, we observed an increase in primitive cancer stem-cell-like cells with high expression of a general cancer biomarker CarcinoEmbryonic Antigen-related Cell Adhesion Molecule 5 (CEACAM5) in the epithelium of the LS. Infiltration of immune cells and DNA repair biological activity are dramatically increased in LS carcinoma. The burden in LS is fundamentally elevated compared to that in CRC or healthy donors when the mutations are in the coding-sequence-wide range. Furthermore, T cell and macrophage-related tumor immunity is readily mobilized in carcinomas compared to paraCArcinomas. Conclusions This study provides single-cell transcriptomic resource using affected tissues from patients with Lynch Syndrome and describes an integrative profile covering the alterations (cancer stem cell markers, mutation burden, and tumor immunity) during the malignant transition from healthy to Lynch Syndrome and to colorectal cancer at the single-cell level. Highlights and Figure/Table Index (Take-Home Messages) Colon tissues from HD (n=4), patients with LS (n=6) and with CRC (n=3) were collected for snRNA-seq or scRNA-seq analysis; Diagnosis of the 6 LS patients are clearly supported by our pedigree documentations (Fig. 1 and Supp. Fig. 1) and briefed in Table 1; Among the 6 patients with LS, 1 for snRNA-seq analysis (a carcinoma tissue only), 3 for scRNA-seq analysis (3 paired para-carcinoma and carcinoma tissues) and 3 for experimental validations; (Fig. 1) Following studies are focused on the comparisons between HD, LS and CRC; In addition, comparisons between paired carcinoma and para-carcinoma tissues from patients with LS were also performed; (Fig. 2-7) When HD and LS are compared, a cancer stem cell marker CEACAM5 is readily detected by snRNA-seq analysis; (Fig. 5) When carcinoma and para-carcinoma from LS patients are compared, increased immune cell infiltration of and enhanced DNA repair activity in the malignant tissues is readily detected by scRNA-seq analysis; in addition, three upregulated markers in LS carcinoma (BACE2, GRPC5A and OLFM4) were identified and validated by immunohistoinfloresnce; (Fig. 3 and Fig. 4) SComatic algorithm-based mutation calling analysis suggests a comparable mutation burden between carcinoma and para-carcinoma from LS patients, but such mutation burdens are grossly greater than that from HD or that from patients with CRC; (Fig. 6)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863c44a229dbcaa552459faeccde9844a06701be" target='_blank'>
              Tracing the stemness and malignant transition in a heritable colorectal cancer Lynch Syndrome by single-cell RNA-seq analysis
              </a>
            </td>
          <td>
            Junfeng Xu, Yuhang Li, Zhiqin Wang, Qianru Li, Aijun Liu, Jianqiu Sheng, Ge Dong, Lang Yang, Zhigang Cai
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Germline variants in the HSD3B1 gene influence response to hormone therapies. This study examined the “adrenal-permissive” allele (c.1100 C) and its impact on various somatic features in prostate tumors. Upon inferring HSD3B1 variant status from somatic DNA sequencing data in a cohort of 6550 primary and metastatic prostate tumors, we found that tumors harboring the adrenal-permissive CC genotype, as compared to the adrenal-restrictive AA genotype, had limited somatic changes in their exomes or transcriptomes. However, significant differences were observed when we examined immune regulatory pathways and immune cell populations. This was accompanied by differential gene expression of immunoglobulins, MHC class I and II molecules, and druggable cell surface genes. These features appeared to deviate across different metastatic biopsy sites. Altogether, while tumor-intrinsic features are not regulated by germline HSD3B1 c.1100 alleles, these genotypes instead may impact cell–cell interactions and the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5953257502a6a5fc55756fab20f962c1af1666a8" target='_blank'>
              The Influence of the Germline HSD3B1 Adrenal-Permissive Allele (c.1100 C) on the Somatic Alteration Landscape, the Transcriptome, and Immune Cell Infiltration in Prostate Cancer
              </a>
            </td>
          <td>
            Samuel Kellen, Allison Makovec, Carly D Miller, Shayan Nazari, Andrew Elliott, Aiden Deacon, Emily John, Nikitha Vobugari, N. Agarwal, Rana R McKay, Pedro C. Barata, Charles J. Ryan, Nima Sharifi, Justin H. Hwang, E. S. Antonarakis
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Recent research has increasingly highlighted alterations in the proto-oncogene MET, whose abnormal activation has been implicated in multiple cancers. MET encodes c-MET, a receptor tyrosine kinase critical for cellular growth, survival, and migration. Aberrant c-MET signaling, driven by mutations or gene amplification, promotes proliferation and invasion, contributing to tumorigenesis. Scope of the Review: While MET mutations are most often observed in non-small cell lung cancer (NSCLC), they also occur in other malignancies, including breast and gastric cancers. This review highlights key MET alterations, such as gene amplification, gene fusions, and exon 14 skipping deletions, and examines their prevalence across various tumor types. Major Conclusions: We discuss the clinical significance of c-MET as a therapeutic target and identify gaps in knowledge that could inform the development of alternative treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00b01ff6f2bb032a063ee8283cbebc4ff3348662" target='_blank'>
              Targeting c-MET Alterations in Cancer: A Review of Genetic Drivers and Therapeutic Implications
              </a>
            </td>
          <td>
            Michelle Ji, Shridar Ganesan, Bing Xia, Yanying Huo
          </td>
          <td>2025-04-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32e4d3a24ff91016f9e42d7a576edaca46c0ba68" target='_blank'>
              Abstract 5339: Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer
              </a>
            </td>
          <td>
            Yuki Matsunaga, Emma Aleksandrovic, H. M. Patel, D. Sudhan, Khushi Ahuja, D. Ye, Chang-Ching Lin, Lei Guo, Cynthia Ma, Jeon Lee, Siyuan Zhang, Carlos L. Arteaga, A. Hanker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16cb06677804f48c2ac85f36bdf345c2b4df6264" target='_blank'>
              Abstract 431: Novel WRN drug resistant CDX models
              </a>
            </td>
          <td>
            Haiting Dai, Tiejun Bing, Huayi Qu, Huawei Pan, Wen-Jen Yu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654c576507b39c0969a5f79ea0daae0b983731a4" target='_blank'>
              Riding the Wave of Progress: Examining the Current Landscape and Future Potential of MicroRNAs in Cancer Gene Therapy.
              </a>
            </td>
          <td>
            Saleem Ahmad, Hanif Khan, Njoka Irene Muthoni, Mohammed Alissa, Awaji Y. Safhi, F. Y. Sabei, Osama Abdulaziz, Khadijah Hassan Khan, Essam H Ibrahim, S. Khan
          </td>
          <td>2025-03-24</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Adenosine deaminases acting on RNA (ADAR) enzymes constitute a natural cellular mechanism that induces A-to-I(G) editing, introducing genetic changes at the RNA level. Recently, interest in the endogenous-ADAR editor has emerged for correcting genetic mutations, consisting of a programmed oligonucleotide that attracts the native ADAR, thereby offering opportunities for medical therapy. Here, we systematically chart the scope of cancer mutations that endogenous-ADAR can correct. First, analyzing germline single nucleotide variants in cancer predisposition genes, we find that endogenous-ADAR can revert a fifth of them, reducing the risk of cancer development later in life. Second, examining somatic mutations across various cancer types, we find that it has the potential to correct at least one driver mutation in over a third of the samples, suggesting a promising future treatment strategy. We also highlight key driver mutations that are amenable to endogenous-ADAR, and are thus of special clinical interest. As using endogenous-ADAR entails delivering relatively small payloads, the prospects of delivering endogenous-ADAR to various cancers seem promising. We expect that the large scope of correctable mutations that are systematically charted here for the first time will pave the way for a new era of cancer treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a024daf32faffd9eaf4adfd22fdca5d40c18746f" target='_blank'>
              A systematic evaluation of the therapeutic potential of endogenous-ADAR editors in cancer prevention and treatment
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Ariel Dadush, Sumeet Patiyal, P. Rajagopal, Gulzar Daya, Shay Ben-Aroya, A. Schäffer, Eli Eisenberg, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-02</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9adf9806a26a89ffa7e78b13710a5e6c2dc2cb20" target='_blank'>
              Abstract 5548: Single-cell transcriptome analysis reveals the role of myofibroblastic cancer-associated fibroblasts in chemotherapy resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Jiahao Guo, Samuele Cancellieri, Liangru Fei, T. Aittokallio, Biswajyoti Sahu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8354bb2cbdecdc87317ac3be8bce15b2494319c2" target='_blank'>
              Emerging Roles of ACTL6A as an Oncogenic Hub: Transcriptional Regulation and Beyond.
              </a>
            </td>
          <td>
            Kelvin K Tsai, Lin-Hsin Cheng, Chung-Chi Hsu, Pei-Ming Yang, Chin-Pin Chuu
          </td>
          <td>2025-04-07</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3aba9f5d2cf8bd1341acacd8ce88421b1a228c70" target='_blank'>
              Abstract 5547: High resolution target identification by unbiased forward genetics
              </a>
            </td>
          <td>
            Moritz Horn, Georg Michlits, Andreia Rocha, Christoph Stelzer, Josef M. Penninger
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In vivo genetic diversifiers have previously enabled efficient searches of genetic variant fitness landscapes for continuous directed evolution. However, existing genomic diversification modalities for mammalian genomic loci exclusively rely on deaminases to generate transition mutations within target loci, forfeiting access to most missense mutations. Here, we engineer CRISPR-guided error-prone DNA polymerases (EvolvR) to diversify all four nucleotides within genomic loci in mammalian cells. We demonstrate that EvolvR generates both transition and transversion mutations throughout a mutation window of at least 40 bp and implement EvolvR to evolve previously unreported drug-resistant MAP2K1 variants via substitutions not achievable with deaminases. Moreover, we discover that the nickase’s mismatch tolerance limits EvolvR’s mutation window and substitution biases in a gRNA-specific fashion. To compensate for gRNA-to-gRNA variability in mutagenesis, we maximize the number of gRNA target sequences by incorporating a PAM-flexible nickase into EvolvR. Finally, we find a strong correlation between predicted free energy changes underlying R-loop formation and EvolvR’s performance using a given gRNA. The EvolvR system diversifies all four nucleotides to enable the evolution of mammalian cells, while nuclease and gRNA-specific properties underlying nickase fidelity can be engineered to further enhance EvolvR’s mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4485500132f714892fcf79f15fa49a1b187bbe0" target='_blank'>
              Nickase fidelity drives EvolvR-mediated diversification in mammalian cells
              </a>
            </td>
          <td>
            Juan E Hurtado, Adam J. Schieferecke, Shakked O Halperin, John Guan, Dylan Aidlen, David V. Schaffer, J. Dueber
          </td>
          <td>2025-04-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/335901b6f2e363bcd4c769050615cefdadb15cb6" target='_blank'>
              Mechanistic Insights into Proteomic Mutation-Phenotype Linkages from Tiling Mutagenesis Screens
              </a>
            </td>
          <td>
            Wei He, Jen-Wei Huang, Yalong Wang, Samuel B. Hayward, G. Leuzzi, Rongjie Fu, Shuyue Wang, Alina Vaitsiankova, Mark T. Bedford, Raphaël Guérois, Alberto Ciccia, Han Xu
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Radiotherapy is a cornerstone of colorectal cancer (CRC) treatment; however, its therapeutic efficacy is often compromised by both intrinsic and acquired resistance in CRC cells. This study employed small interfering RNA (siRNA) technology to elucidate the functional roles of BAMBI, GADD34, NFKBIA, and NFKBID in CRC cell lines SW480 and HCT116. We assessed their impact on key cellular processes and radiation sensitivity. Gene silencing of all four target genes significantly suppressed CRC cell proliferation, migration, and invasion. Moreover, siRNA-mediated knockdown enhanced radiation sensitivity, as evidenced by a substantial increase in apoptosis and a marked reduction in cell viability compared with controls. These findings suggest that BAMBI, GADD34, NFKBIA, and NFKBID serve as critical regulators of CRC progression and radiation resistance. Overall, this study provides a mechanistic foundation for further exploration into the pathways underlying radiation resistance and underscores the potential for developing personalized radiotherapy strategies guided by molecular profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b02786eb68733b852aeed62e7695e4393e02cbb" target='_blank'>
              Mechanistic Insights into Radiation Resistance in Colorectal Cancer: Gene Exploration Study
              </a>
            </td>
          <td>
            Beamjun Park, Soohyeon Lee, Inyoung Jo, Donghyun Kang, Taewan Kim, Jaesung Ryu, Hyejeong Kong, Moojun Baek, Taesung Ahn
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Transposons are DNA sequences able to move or copy themselves to other genomic locations leading to insertional mutagenesis. Although transposon-derived sequences account for half of the human genome, most elements are no longer transposition competent. Moreover, transposons are normally repressed through epigenetic silencing in healthy adult tissues but become derepressed in several human cancers, with high activity detected in colorectal cancer. Their impact on non-malignant and malignant tissue as well as the differences between somatic and germline retrotransposition remain poorly understood. With new sequencing technologies, including long read sequencing, we can access intricacies of retrotransposition, such as insertion sequence details and nested repeats, that have been previously challenging to characterize. Results In this study, we investigate somatic and germline retrotransposition by analyzing long read sequencing from 56 colorectal cancers and 112 uterine leiomyomas. We identified 1495 somatic insertions in colorectal samples, while striking lack of insertions was detected in uterine leiomyomas. Our findings highlight differences between somatic and germline events, such as transposon type distribution, insertion length, and target site preference. Leveraging long-read sequencing, we provide an in-depth analysis of the twin-priming phenomenon, detecting it across transposable element types that remain active in humans, including Alus. Additionally, we detect an abundance of germline transposons in repetitive DNA, along with a relationship between replication timing and insertion target site. Conclusions Our study reveals a stark contrast in somatic transposon activity between colorectal cancers and uterine leiomyomas, and highlights differences between somatic and germline transposition. This suggests potentially different conditions in malignant and non-malignant tissues, as well as in germline and somatic tissues, which could be involved in the transposition process. Long-read sequencing provided important insights into transposon behavior, allowing detailed examination of structural features such as twin priming and nested elements. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00357-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6e6a5d11920586374ddcd1ad544515162f4af5" target='_blank'>
              Structural features of somatic and germline retrotransposition events in humans
              </a>
            </td>
          <td>
            Päivi Nummi, Tatiana Cajuso, T. Norri, Aurora Taira, Heli Kuisma, Niko Välimäki, A. Lepistö, L. Renkonen-Sinisalo, S. Koskensalo, T. Seppälä, Ari Ristimäki, Kyösti Tahkola, Anne Mattila, J. Böhm, J. Mecklin, Emma Siili, A. Pasanen, O. Heikinheimo, Ralf Bützow, A. Karhu, K. Burns, Kimmo Palin, Lauri A Aaltonen
          </td>
          <td>2025-04-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="ABSTRACT Background Genetic testing and sequencing technologies offer a comprehensive understanding of cancer genetics, providing rapid and cost‐effective solutions. In particular, these advanced technologies play an important role in assessing the complexities of the rare cancer types affecting several systems including the bone, endocrine, digestive, vascular, and soft tissue. This review will explore how genetic testing and sequencing technologies have contributed to the identification of biomarkers across several rare cancer types in diagnostic, therapeutic, and prognostic stages, thereby advancing PM. Methods A comprehensive literature search was conducted across PubMed (MEDLINE), EMBASE, and Web of Science using keywords related to sequencing technologies, genetic testing, and cancer. There were no restrictions on language, methodology, age, or publication date. Both primary and secondary research involving humans or animals were considered. Results In practice, fluorescence in situ hybridization, karyotype, microarrays and other genetic tests are mainly applied to identify specific genetic alterations and mutations associated with cancer progression. Sequencing technologies, such as next generation sequencing, polymerase chain reaction, whole genome or exome sequencing, enable the rapid analysis of millions of DNA fragments. These techniques assess genome structure, genetic changes, gene expression profiles, and epigenetic variations. Consequently, they help detect main intrinsic markers that are crucial for personalizing diagnosis, treatment options, and prognostic assessments, leading to better patient prognosis. This highlights why these methods are now considered as primary tools in rare cancer research. However, these methods still face multiple limitations, including false positive results, limited precision, and high costs. Conclusion Genetic testing and sequencing technologies have significantly advanced the field of rare cancer research by enabling the identification of key biomarkers for precision diagnosis, treatment, and prognosis. Despite existing limitations, their integration into clinical and research fields continues to improve the development of personalized medicine strategies for rare and complex cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44e6c2844fd1beb610939541608801e2d5fb55e3" target='_blank'>
              Advancing Precision Medicine: The Role of Genetic Testing and Sequencing Technologies in Identifying Biological Markers for Rare Cancers
              </a>
            </td>
          <td>
            Joviana Farhat, Lara Alzyoud, M. Alwahsh, A. Acharjee, B. Al-Omari
          </td>
          <td>2025-04-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc24f6cab28b67012c2dbe001093787b08c40ec" target='_blank'>
              Abstract 6713: RB1 and TP53 restrict lineage plasticity in prostate cancer by regulating chromatin accessibility
              </a>
            </td>
          <td>
            Mauricio Rodrigo Flores, David W. Goodrich, Justine J Jacobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba9a2666f037bf98ff78850cacfb5af10e4de653" target='_blank'>
              Abstract 1338: System biological approach identifies heterochromatin modification as an important mechanism of acquired cancer radioresistance
              </a>
            </td>
          <td>
            Chaw Yee Beh, Sharon Pei Yi Chan, Celestia Pei Xuan Yeo, Boon Hao Hong, Evelyn Mui Cheng Tan, Dewi Susanti, Pei Ling Tai, Eugenia Li Ling Yeo, Khee Chee Soo, Edward Kai-Hua Chow, Melvin Lee Kiang Chua
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3847b0cc02af732174f2aba89f84c63f9e5ea11" target='_blank'>
              Proof-of-Concept of A DNA-Based Recording System for High-Throughput Functional Gene Screening
              </a>
            </td>
          <td>
            Shoichi Kato, Atsushi Ikemoto, Jun Isayama, Tetsuya Takimoto, Hideyuki Saya, Ken-ichi Hamada
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gene fusions are an important driver of cancer and require rapid and accurate detection to guide clinical decisions. However, the performance characteristics of whole transcriptome sequencing (WTS) for the detection of gene fusions have not been thoroughly investigated. Methods We developed a novel WTS-based assay for the detection of gene fusions, MET exon 14 skipping and EGFR VIII alterations in clinical samples. Results We defined a DV200 value ≥ 30% as the threshold for RNA degradation, RNA input, fusion expression and number of mapped reads greater than 100 ng, 40 copies/ng and 80 Mb for optimal sensitivity of the WTS assay. Our assay successfully identified 62 out of 63 known gene fusions, achieving a sensitivity of 98.4%. The specificity of the assay was 100%, as no fusions were detected in the 21 fusion-negative samples. Good repeatability and reproducibility were observed in replicates, except for the TPM3::NTRK1 fusion, which was expressed below the threshold. Of all fusions identified in 101 NSCLC samples, 68.9% (20/29) were potentially actionable, compared to 20% in pan-cancer samples. In addition to actionable fusions, we also identified many fusions with potential diagnostic and prognostic value in pan-cancer. Conclusions We have developed a novel WTS assay with high sensitivity, specificity, repeatability and reproducibility. This assay can identify potentially actionable gene fusions and provides valuable insights into the fusion landscape in various cancers, which may help guide treatment decisions and aid in diagnosis and prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14186-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3600e83dedfe21ec35614866625aa600eb36e48e" target='_blank'>
              Development and application of a whole transcriptome sequencing assay for the detection of gene fusions in clinical cancer specimens
              </a>
            </td>
          <td>
            Songchen Zhao, Xinhua Du, Yan Zhang, Jing Bai, Lu Meng, Xuefei Li, Jiaxin Ma, Heyu Sheng, Xiaorui Fu, Yanfang Guan, Yuting Yi, Ling Yang, Xuefeng Xia, Xin Yi, Xinxin Tan, Caicun Zhou
          </td>
          <td>2025-05-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Cancer arises as an abnormality in genomic control, either due to the activation of oncogenes or loss of functioning tumor-suppressor genes. Oncogenes result in an uncontrolled increase in cell proliferation; tumor-suppressor genes act as a brake, leading the cell cycle to stop and promote cell death when necessary. This review seeks to tease out the complex interactions between these genes, throwing light on key signaling pathways such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, all of which play a role in the progression of the tumor. We discuss major genetic theories, especially Knudson’s two-hit hypothesis, and how modern technologies like next-generation sequencing have advanced our understanding of cancer genetics. We also underscore the most current therapeutic approaches involving oncogenes and the reactivation of tumor-suppressor gene function, with challenges including drug resistance and heterogeneity of the tumor. In this regard, understanding the balance between oncogenes and tumor-suppressor genes is critical in developing effective and personalized cancer treatment, thereby bettering patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b575dc8351428b4d22b9c3382531e97fc82f1bcf" target='_blank'>
              Abstract 545: RFX5 knockdown restores sensitivity to cellular cyclophosphamide in triple-negative breast cancer cell lines
              </a>
            </td>
          <td>
            Elizabeth Katherine Molchan, Kathleen Streeks, Alyssa Ohaegbulam, Mohammed Gbadamosi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, Markus Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/919d6be56b49f799484f0a056c28802836f4c288" target='_blank'>
              Abstract 2771: CRISPR/Cas9 essentiality screens identify stand-alone and IRF4-linked genetic vulnerabilities in melanoma cells
              </a>
            </td>
          <td>
            U. Sobhiafshar, N. T. Emre
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892d32d1ea696e84aa4f250bf0edeb5b5584501c" target='_blank'>
              CRISPRi perturbation screens and eQTLs provide complementary and distinct insights into GWAS target genes
              </a>
            </td>
          <td>
            Sam Ghatan, Jasper Panten, Winona Oliveros, Neville E. Sanjana, John A Morris, Tuuli Lappalainen
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24e80e0a03a79befa516d29031109ab14c8ea3b4" target='_blank'>
              Signature Gene Mutations in Colorectal Cancer: Potential Neoantigens for Cancer Vaccines
              </a>
            </td>
          <td>
            Jaegoo Yoon, Haeun Moon, Yuna Jeon, Soo-Cheon Choe, Hyunho Yoon
          </td>
          <td>2025-05-09</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="It is well established that hematologic malignancies are often considerably radiosensitive, which enables usage of far lower doses of therapeutic radiotherapy. This review summarizes the currently known genomic landscape of hematologic malignancies, particularly as it relates to radiosensitivity and the field of radiation oncology. By tracing the historical development of the modern understanding of radiosensitivity, we focus on the discovery and implications of pivotal mutated genes in hematologic malignancies such as TP53, ATM, and other genes critical to DNA repair pathways. These genetic insights have contributed significantly to the advancement of personalized medicine, aiming to enhance treatment precision and outcomes, and there is an opportunity to extend these insights to personalized radiotherapy. We explore the transition from early discoveries to the current efforts in integrating comprehensive genomic data into clinical practice. Specific examples from Hodgkin lymphoma, non-Hodgkin lymphoma, and plasma cell neoplasms illustrate how genetic mutations could influence radiosensitivity and impact subsequent radiotherapeutic response. Despite the advancements, challenges remain in translating these genetic insights into routine clinical practice, particularly due to the heterogeneity of alterations and the complex interactions within cancer signaling pathways. We emphasize the potential of radiogenomics to address these challenges by identifying genetic markers that predict radiotherapy response and toxicity, thereby refining treatment strategies. The need for robust decision support systems, standardized protocols, and ongoing education for healthcare providers is critical to the successful integration of genomic data into radiation therapy. As research continues to validate genetic markers and explore novel therapeutic combinations, the promise of personalized radiotherapy becomes increasingly attainable, offering the potential to significantly improve outcomes for patients with hematologic malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e274aa44601f7df10924839e8b158bfac4f727a9" target='_blank'>
              The art of war: using genetic insights to understand and harness radiation sensitivity in hematologic malignancies
              </a>
            </td>
          <td>
            N. Wijetunga, J. Yahalom, B. Imber, Thomas FitzGerald, Lara Hilal, Ji Hyun Hong, Jonathan Leeman
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532359eb783a00c7e5a0af7f6a8c8009cbbb7c55" target='_blank'>
              Abstract 2758: Methylome plasticity as a biomarker of treatment response in small cell lung cancer
              </a>
            </td>
          <td>
            D. Sacdalan, Sami Ul Haq, L. Zhan, J. J. Li, V. Philip, Scott V. Bratman, Geoffrey Liu, Benjamin H. Lok
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320798304ad2c6aa4cbcc689f8af4488470a00c4" target='_blank'>
              Abstract 1475: HDAC8 as a critical regulator of cancer cell genome stability: therapeutic implications
              </a>
            </td>
          <td>
            Zih-Yao Yu, Wei-Jan Huang, Stanton Gerson, Sung-Bau Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most prevalent malignancies worldwide. With the increasing incidence of CRC, there is an urgent need for effective strategies for early diagnosis and treatment. Circadian rhythm, a natural biological clock, regulates various physiological processes, and its disruption has been implicated in the onset and progression of cancer. However, the specific roles of circadian rhythm-related genes (CRDGs) in CRC remain unclear. Methods In this study, we analyzed the expression patterns of CRDGs in CRC using single-cell RNA sequencing (scRNA-seq) data and bulk RNA sequencing data from the GSE178318 dataset. We constructed a CRC prognostic model based on CRD scores. Additionally, we explored the potential mechanisms of CRDGs in tumor progression through weighted gene co-expression network analysis (WGCNA) and gene set enrichment analysis (GSEA), and assessed their impact on the response to immune checkpoint inhibitors. Results The analysis revealed that CRDGs were significantly upregulated in liver metastasis samples compared to primary CRC samples and were closely associated with several metabolic and immune-related pathways. The prognostic model based on CRD scores indicated that higher CRD scores were associated with poorer outcomes in immunotherapy. These findings were further validated in multiple datasets, underscoring the potential of CRDGs as prognostic indicators in CRC. Conclusion This study systematically reveals, for the first time, the expression characteristics of CRDGs in CRC and their relationship with tumor progression and response to immunotherapy. CRDGs may serve as effective prognostic biomarkers and therapeutic targets, offering new strategies for the personalized treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85b7699b45efdb0dd0f3e56012bf04dabfd7ec73" target='_blank'>
              Integration of scRNA-Seq and Bulk RNA-Seq Identifies Circadian Rhythm Disruption-Related Genes Associated with Prognosis and Drug Resistance in Colorectal Cancer Patients
              </a>
            </td>
          <td>
            Yong Tao, Jun Li, Jianhui Pan, Qing Wang, Ru Ke, D. Yuan, Hongbiao Wu, Yuepeng Cao, Lei Zhao
          </td>
          <td>2025-04-01</td>
          <td>ImmunoTargets and Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="EGFR is a proto-oncogene that is mutationally activated in a variety of cancers. Small molecule inhibitors targeting EGFR can be effective in slowing the progression of disease, and in some settings these drugs even cause dramatic tumor regression. However, responses to EGFR inhibitors are rarely durable, and the mechanisms contributing to response variation remain unclear. In particular, several distinct mechanisms have been proposed for how EGFR inhibition activates cell death, and a consensus has yet to emerge. In this study, we use functional genomics with specialized analyses to infer how genetic perturbations effect the drug-induced death rate. Our data clarify that inhibition of PI3K signaling drives the lethality of EGFR inhibition. Inhibition of other pathways downstream of EGFR, including the RAS-MAPK pathway, promote growth suppression, but not the lethal effects of EGFR inhibitors. Taken together, our study reveals the first “reference map” for the genome-wide genetic dependencies of lethality for EGFR inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ecf934bb91c2c002402ace642323761657b953a" target='_blank'>
              Genome-wide profiling identifies the genetic dependencies of cell death following EGFR inhibition
              </a>
            </td>
          <td>
            Sydney A Porto, Gavin A. Birdsall, Nicholas W. Harper, Megan E. Honeywell, Michael J. Lee
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c037285ee33cfd3dd5fa1bd1a95f79d26559f4b5" target='_blank'>
              Abstract 813: In vitro and in vivo modelling of drug tolerance and minimal residual disease to discover effective therapeutic combination strategies for EGFR-mutated lung cancer
              </a>
            </td>
          <td>
            Kristyna A. Gorospe, Ming Li, A. Arivajiagane, Nhu-An Pham, R. Navab, Kelsie L. Thu, M S Tsao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08272a17f03be6497869a3622bf7e66968e9d5bd" target='_blank'>
              Abstract 219: Uncovering synergistic and antagonistic interactions in epigenetic therapeutics using perturb-seq
              </a>
            </td>
          <td>
            Changde Cheng, Rui Lu, Sajesan Aryal, Doug A. Welsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55dbe0088f239e1966f0ecbd4169616a69ee9add" target='_blank'>
              Abstract 5673: Identification of potential therapeutic targets by in vivo CRISPR screening on xenograft models
              </a>
            </td>
          <td>
            Ce Zhong, Lu Bai, Tongtong Wang, Yiying Shao, Lingyun Zhang, Lu Zhang, Jingqi Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma, the most common and aggressive primary malignant brain tumor, is characterized by a high rate of recurrence, disability, and lethality. Therefore, there is a pressing need to develop more effective prognostic biomarkers and treatment approaches for glioblastoma. Lactylation, an emerging form of protein post-translational modification, has been closely associated with lactate, a metabolite of glycolysis. Since the initial identification of lactylation sites in core histones in 2019, accumulating evidence has shown the critical role that lactylation plays in glioblastoma development, assessment of poor clinical prognosis, and immunosuppression, which provides a fresh angle for investigating the connection between metabolic reprogramming and epigenetic plasticity in glioblastoma cells. The objective of this paper is to present an overview of the metabolic and epigenetic roles of lactylation in the expanding field of glioblastoma research and explore the practical value of developing novel treatment plans combining targeted therapy and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4de8e5b16e7caa0a9241085fd7820d5813bb6d8" target='_blank'>
              Lactylation in Glioblastoma: A Novel Epigenetic Modifier Bridging Epigenetic Plasticity and Metabolic Reprogramming
              </a>
            </td>
          <td>
            Qingya Qiu, Hui Deng, Ping Song, Yushu Liu, Mengxian Zhang
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Recent research has identified a potential protective effect of higher numbers of circulating lymphocytes on colorectal cancer (CRC) development. However, the importance of different lymphocyte subtypes and activation states in CRC development and the biological pathways driving this relationship remain poorly understood and warrant further investigation. Specifically, CD4+ T cells -- a highly dynamic lymphocyte subtype -- undergo remodelling upon activation to induce the expression of genes critical for their effector function. Previous studies investigating their role in CRC risk have used bulk tissue, limiting our current understanding of the role of these cells to static, non-dynamic relationships only. Methods Here, we combined two genetic epidemiological methods -- Mendelian randomisation (MR) and genetic colocalisation -- to evaluate evidence for causal relationships of gene expression on CRC risk across multiple CD4+ T cell subtypes and activation stage. Genetic proxies were obtained from single-cell transcriptomic data, allowing us to investigate the causal effect of expression of 1,805 genes across five CD4+ T cell activation states on CRC risk (78,473 cases; 107,143 controls). We repeated analyses stratified by CRC anatomical subsites and sex, and performed a sensitivity analysis to evaluate whether the observed effect estimates were likely to be CD4+ T cell-specific. Results We identified six genes with evidence (FDR-P<0.05 in MR analyses and H4>0.8 in genetic colocalisation analyses) for a causal role of CD4+ T cell expression in CRC development -- FADS2, FHL3, HLA-DRB1, HLA-DRB5, RPL28, and TMEM258 . We observed differences in causal estimates of gene expression on CRC risk across different CD4+ T cell subtypes and activation timepoints, as well as CRC anatomical subsites and sex. However, our sensitivity analysis revealed that the genetic proxies used to instrument gene expression in CD4+ T cells also act as eQTLs in other tissues, highlighting the challenges of using genetic proxies to instrument tissue-specific expression changes. Conclusions Our study demonstrates the importance of capturing the dynamic nature of CD4+ T cells in understanding disease risk, and prioritises genes for further investigation in cancer prevention research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3854c3b38125ddae1dd0d632aa93ab78f073f33f" target='_blank'>
              Transcriptome-wide Mendelian randomisation exploring dynamic CD4+ T cell gene expression in colorectal cancer development
              </a>
            </td>
          <td>
            Benedita Deslandes, Xueyan Wu, Matthew A. Lee, L. J. Goudswaard, Gareth W Jones, A. Gsur, A. Lindblom, Shuji Ogino, Veronika Vymetálková, A. Wolk, Anna H. Wu, J. Huyghe, Ulrike Peters, Amanda I. Phipps, Claire E Thomas, Rish K Pai, Robert C Grant, Daniel D Buchanan, J. Yarmolinsky, Marc J. Gunter, Jie Zheng, E. Hazelwood, E. E. Vincent
          </td>
          <td>2025-04-17</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cfdfff54622d4227a75b1cb40c4cc408256a9de" target='_blank'>
              DNA Damage Response and Repair Genes and Anthracycline-Induced Cardiomyopathy in Childhood Cancer Survivors: A Report From the Children's Oncology Group and the Childhood Cancer Survivor Study.
              </a>
            </td>
          <td>
            Xuexia Wang, Purnima Singh, Romina B Cejas, Liting Zhou, Noha Sharafeldin, P. Trainor, Wendy Landier, Changde Cheng, L. Hageman, Fan Wang, Y. Sapkota, Yutaka Yasui, M. Hudson, Eric J Chow, S. Armenian, Joseph P Neglia, D. S. Hawkins, JillP Ginsberg, P. Burridge, Gregory T Armstrong, Smita Bhatia
          </td>
          <td>2025-03-28</td>
          <td>Circulation. Genomic and precision medicine</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e9ca7a5e65c5dcd9a8e77355be009d71f35f7d" target='_blank'>
              Abstract 1473: POLQ facilitates annealing mediated long deletions at chromosomal break in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Manisha Jalan, Erika S. Buechelmaier, Jonathan Rafailov, Yingjie Zhu, Xin Pei, Sebastian Brylka, Joan Gomez-Aguilar, Niamh McDermott, N. Riaz, Marcin Imieliński, Simon N. Powell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1834a1d0667a17db6a7427294528dfb9ae37c" target='_blank'>
              Abstract 4358: Single nucleotide polymorphisms located in PK/PD pathway genes of standard AML chemotherapy regimen predicts outcome in pediatric AML
              </a>
            </td>
          <td>
            Aneesha Nath, Richard J Marrero, Xueyuan Cao, J. Rubnitz, S. Pounds, J. Lamba
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ddc9edbf2bf1f968d70d06263a337e58c74158" target='_blank'>
              Abstract 4498: Drug screens identify new therapeutic targets that synergize with EZH1/2 inhibition in adult T-cell leukemia/lymphoma
              </a>
            </td>
          <td>
            Noor B Ghanem, C. Ip, Kit I Tong, Mehran Bakhtiari, Michael Y. He, A. Schimmer, R. Kridel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="How do cancer cells respond to their environment, and what are the key regulators behind their behaviour? While immune cell reprogramming in the tumour microenvironment (TME) has been extensively studied, the dynamic regulatory changes within cancer cells in response to interactions with immune cells remain poorly understood. In Chronic Lymphocytic Leukaemia (CLL), this knowledge gap limits our ability to fully grasp the disease progression and to design effective, personalised interventions. To tackle this, we combine time-series transcriptomics with data-driven gene regulatory network (GRN) inference to uncover the temporal regulatory mechanisms driving CLL cell behaviour within a reconstituted in vitro TME. Using cultures of peripheral blood from CLL patients or of purified patient-derived CLL cells, we profile gene expression across five time points spanning 14 days under these experimental conditions. By inferring GRNs from transcription factor activity, we capture patient-specific and temporally resolved regulatory interactions that highlight how immune signals shape cancer cell phenotypic changes. Our network analysis reveals distinct gene modules associated with critical processes such as cytokine signalling, metabolic reprogramming and differentiation, hallmarks of immune-cancer cell interaction. Intriguingly, we found that while the presence of immune cells in the environment significantly alters CLL cell activation, their survival trajectories are predominantly governed by intrinsic features. This study not only offers mechanistic insights into how immune cell presence influences CLL cell fate but also presents a robust computational framework for integrating time-series transcriptomics with GRN inference, which can then be used to study the long-term behaviour of the CLL cells through dynamical modelling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46422bf8177dcfdbb03604f9382eef5f720cbd11" target='_blank'>
              Time-series RNA-Seq and data-driven network inference unveil dynamics of cell activation, survival and crosstalk in Chronic Lymphocytic Leukaemia in vitro models
              </a>
            </td>
          <td>
            Malvina Marku, Hugo Chenel, J. Bordenave, Marcelo Hurtado, Marcin Domagala, Flavien Raynal, M. Poupot, Loïc Ysebaert, Andrei Zinovyev, Vera Pancaldi
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80e1849f5d2a4bf9cdd4d6129184c9b1ac64a97" target='_blank'>
              Abstract 5413: Temporal regulation of c-Myc in endocycling cancer cells facilitates mitotic bypass in response to chemotherapy
              </a>
            </td>
          <td>
            Michael A. Loycano, Kenneth J. Pienta, Sarah R Amend
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5088d4df327912ec43b2d65ab6fa5deace3c7" target='_blank'>
              Abstract 5549: Identification of genes associated with drug persistence/resistance using CRISPR screening and scRNA-seq of ALK rearranged lung cancer
              </a>
            </td>
          <td>
            Ryohei Katayama, Takahiro Utsumi, Nobuyuki Kondo, K. Uchibori, Makoto Nishio
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Long Intergenic Non-Protein Coding RNA 1123 (LINC01123), located on human chromosome 2q13, is a pivotal factor in tumorigenesis, exerting multifaceted oncogenic effects. Its expression strongly correlates with clinicopathological features, patient survival, and disease progression. In vivo and in vitro experiments further demonstrate that LINC01123 influences diverse cellular processes, including proliferation, apoptosis, viability, migration, invasion, stemness, and tumor growth. Notably, it also regulates metabolic reprogramming, immune escape, and tumor cell resistance to treatment. LINC01123 is regulated by multiple transcription factors and participates in gene regulation through protein interactions and competitive endogenous RNA (ceRNA) networks, thereby modulating cancer-promoting effects. This work systematically elucidates its primary functions and molecular mechanisms driving cancer initiation and progression, suggesting that LINC01123 might serve as a novel potential oncogenic driver and biomarker in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca43f615285dd30f7033e316e2f7208e0ef3058" target='_blank'>
              A novel cancer-associated lncRNA, LINC01123, participates in tumor progression, metabolism, immune escape, and resistance
              </a>
            </td>
          <td>
            Qiang Liu, He Huang, Shuwen Zhang, Fangteng Liu, Ting Lou
          </td>
          <td>2025-04-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631764abf56c15adfa48bcb7f3a761557f103393" target='_blank'>
              Abstract 444: ASXL1 mutations in AML are associated with a distinct epigenetic state that results in vulnerabilities to epigenetic-targeted agents
              </a>
            </td>
          <td>
            Christopher P. Mill, W. Fiskus, Christine E. Birdwell, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, K. Sasaki, S. Loghavi, T. Kadia, Naval G Daver, C. Dinardo, Lu Wang, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce10ef6d68c6903f3dcfc0aa9cd8d7cbbe833212" target='_blank'>
              Abstract 2992: Uncovering metabolic dysregulation and therapeutic targets in aneuploid models of squamous cell carcinoma
              </a>
            </td>
          <td>
            Nadja Zhakula, Sejal Jain, Laura Byron, Joan J. Castellano, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genetic diseases are serious health issues worldwide, caused by changes in gene structure due to mutations in single or multiple genes. Currently, no definitive treatments or medicines are available to cure these disorders. Gene therapy, a medical approach that utilizes genetic material, offers potential for preventing and treating diseases by addressing their underlying causes rather than relying on traditional methods like medication and surgery. This technique has shown promise in certain conditions by altering genetic material, enabling the body to produce desirable proteins or medications. However, gene therapy remains unavailable for most diseases and is still largely in the experimental stage for many conditions. This review investigates the current and future status of gene therapy not only for genetic disorders but also for life-threatening diseases such as cancer, cardiovascular disease (CVD), and diabetes. Additionally, the applications and limitations of gene therapy were evaluated. A review of recent research concluded that while rare or inherited genetic disorders could potentially be prevented or cured by precisely targeting their root causes, significant challenges remain. Rather than solely managing blood disorders, retinal diseases, and neuromuscular conditions, gene therapy can potentially eliminate these conditions. Furthermore, gene-based therapies offer expansive possibilities beyond rare genetic disorders, paving the way for broader therapeutic applications in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52a192a46bd071e41e3a160b10b231a04116a32" target='_blank'>
              Advancement in Gene therapy: Current and Future of Gene Therapy
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-03-17</td>
          <td>Kashmir Journal of Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8bc96152bbbb1bdeac5241e7bd6de510c434bdc" target='_blank'>
              Abstract 454: Leveraging cell cycle to enhance differentiation therapy in acute myeloid leukemia
              </a>
            </td>
          <td>
            Riccardo Valzelli, Kourosh Hayatigolkhatmi, Amir Hosseini, E. Ceccacci, Giorgia Parodi, A. Bachi, S. Minucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c34b2c44db5e2413af90c06943f2d16b821f4c" target='_blank'>
              Abstract 6621: Multi-modal profiling reveals the role of somatic structural variations in renal cell carcinoma
              </a>
            </td>
          <td>
            Yuwei Zhang, Kapur B. Dhami, Yize Li, Cody Weimholt, Wagma Caravan, John M Herndon, R. Jayasinghe, Song Cao, Melissa A. Reimers, Russell Pachynski, Feng Chen, Li Ding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5520257da55eb20975e2b76db4a10f023d4be1" target='_blank'>
              Abstract 5543: High resolution lineage tracing for the study of cancer drug persistence
              </a>
            </td>
          <td>
            Binyanim Zhitomirsky, Jideofor Ezike, B. Danysh, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1950d0d983eeb2c05d4af1f02b47efd0ad374c" target='_blank'>
              Theoretical investigation of AKT1 mutations in breast cancer: a computational approach to structural and functional insights.
              </a>
            </td>
          <td>
            Balu Kamaraj, G. C
          </td>
          <td>2025-05-09</td>
          <td>Journal of computer-aided molecular design</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e8e473fd0d41652458c66d5982b1c03c7dc5d7e" target='_blank'>
              Abstract 3898: Integrating structural variants and single nucleotide variants to uncover evolutionary trajectories in melanoma
              </a>
            </td>
          <td>
            Xiwen Cui, A. Keskus, S. Malikić, Yuelin Liu, Anton Goretsky, Chi-Ping Day, F. Mehrabadi, M. Kolmogorov, Glenn Merlino, S. C. Sahinalp
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Trastuzumab emtansine (T-DM1) is an antibody–drug conjugate that is an effective therapy for HER2-positive breast cancer; however, its efficacy is limited by drug resistance. While multiple mechanisms of resistance have been proposed, these are not yet well understood. Greater understanding of T-DM1 sensitivity and resistance could provide new combination strategies to overcome resistance or predictive biomarkers to guide therapy. Methods We have conducted CRISPR/Cas9 functional genomics modifier screens in HER2-positive breast cancer cell lines to allow for unbiased discovery of T-DM1 sensitivity and resistance genes. Whole-genome knockout screens were carried out in MDA-MB-361 and MDA-MB-453 cells treated with T-DM1 and its payload cytotoxin DM1. Hits were validated in secondary T-DM1 screens using a focused single-guide RNA (sgRNA) library and subsequently by individual gene knockout. Results The whole-genome CRISPR screens with T-DM1 and DM1 identified 599 genes as potential modifiers of T-DM1 sensitivity and resistance. Of these, 17 genes were significantly enriched and 3 genes depleted at P < 0.001 in either or both MDA-MB-361 and MDA-MB-453 libraries in the secondary screens. Among the top hits, were known T-DM1 sensitivity genes ERBB2 and SLC46A3, in addition to negative regulators of mTOR complex 1: TSC1 and TSC2. MDA-MB-453 clones with knockout of TSC1 or partial knockout of TSC2 were more resistant to T-DM1 than wild type cells in competition growth assays and to T-DM1 and other HER2 targeting therapies (T-DXd, lapatinib and neratinib) in growth inhibition assays, and had increased internalisation of T-DM1 at 6 h. T-DM1 and the mTOR inhibitor everolimus demonstrated synergistic activity at inhibiting cell proliferation at multiple T-DM1 concentrations across four HER2-positive breast cancer cell lines. Conclusions Our CRISPR screening approach with T-DM1 in HER2-positive breast cancer cell lines identified genes not previously implicated in T-DM1 sensitivity or resistance, including TSC1 and TSC2. These genes may inform new strategies to enhance T-DM1 therapy in the clinic. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02000-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dee2394ef1c6b351b41594d320afdc6e8fce82eb" target='_blank'>
              CRISPR screens with trastuzumab emtansine in HER2-positive breast cancer cell lines reveal new insights into drug resistance
              </a>
            </td>
          <td>
            Barbara A Lipert, K. Siemens, Aziza Khan, R. Airey, Gech Heng Dam, Man Lu, Marcella Flinterman, Queenie Yong, T. Lee, Francis W Hunter, Stephen M F Jamieson
          </td>
          <td>2025-03-31</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac8d5d5cb28b197c707cf32dafe8a0eca6770a3b" target='_blank'>
              Analysis of mutually exclusive expression in cancer cells identifies a previously unknown intergenic regulatory paradigm.
              </a>
            </td>
          <td>
            Ling Tang, Yuzhe Hu, Chao Wang, Wenling Han, Pingzhang Wang
          </td>
          <td>2025-04-04</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f39bd0ddc99a69058c299e72e0d1b4296584ccd0" target='_blank'>
              Therapeutic targeting of chromatin alterations in leukemia and solid tumors.
              </a>
            </td>
          <td>
            Florian Perner, Tobias Berg, D. Sasca, Sophie-Luise Mersiowsky, Jayant Y. Gadrey, Johanna Thomas, Michael W M Kühn, Michael Lübbert
          </td>
          <td>2025-04-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T. Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Copy number variations (CNVs), as an important structural variant in genomes, are widely present in plants, affecting their phenotype and adaptability. In recent years, CNV research has not only focused on changes in gene copy numbers but has also been linked to complex mechanisms such as genome rearrangements, transposon activity, and environmental adaptation. The advancement in sequencing technologies has made the detection and analysis of CNVs more efficient, not only revealing their crucial roles in plant disease resistance, adaptability, and growth development, but also demonstrating broad application potential in crop improvement, particularly in selective breeding and genomic selection. By studying CNV changes during the domestication process, researchers have gradually recognized the important role of CNVs in plant domestication and evolution. This article reviews the formation mechanisms of CNVs in plants, methods for their detection, their relationship with plant traits, and their applications in crop improvement. It emphasizes future research directions involving the integration of multi-omics to provide new perspectives on the structure and function of plant genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0396c8ecd922291ec579ba57375e978767ea2182" target='_blank'>
              A Systematic Review of the Advances and New Insights into Copy Number Variations in Plant Genomes
              </a>
            </td>
          <td>
            Saimire Silaiyiman, Jiaxuan Liu, Jiaxin Wu, Lejun Ouyang, Zheng Cao, Chao Shen
          </td>
          <td>2025-05-06</td>
          <td>Plants</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f22aef6f2b3fccea5561ffb5ccb8e4b8470eaaa" target='_blank'>
              Hi-C for genome-wide detection of enhancer-hijacking rearrangements in routine lymphoid cancer biopsies
              </a>
            </td>
          <td>
            Jamin Wu, Shih-Chun A. Chu, Jang Cho, Misha Movahed-Ezazi, Kristyn Galbraith, C. Fang, Yiying Yang, Chanel Schroff, Kristin Sikkink, Michelle Perez-Arreola, Logan Van Meter, Savanna Gemus, Jon-Matthew Belton, Valentina Nardi, Abner Louissant, Dennis Shasha, A. Tsirigos, Noah A Brown, Tatyana Gindin, Marcin P. Cieslik, Minji Kim, Anthony D Schmitt, Matija Snuderl, Russell J H Ryan
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Abstract Genes and genetic variants associated with human disease are continually being discovered, but validating their causative roles and mechanisms remains a significant challenge. CRISPR/Cas9 genome editing in model organisms like zebrafish can enable phenotypic characterization of founder generation (F0) knockouts (Crispants), but existing approaches are not amenable to high-throughput genetic screening due to high variability, cost, and low phenotype penetrance. To overcome these challenges, here we provide guide RNA (gRNA) selection rules that enable high phenotypic penetrance of up to three simultaneous knockouts in F0 animals following injection of 1–2 gRNAs per gene. We demonstrate a strong transcriptomic overlap in our F0 knockouts and stable knockout lines that take several months to generate. We systematically evaluated this approach across 324 gRNAs targeting 125 genes and demonstrated its utility in studying epistasis, characterizing paralogous genes, and validating human disease gene phenotypes across multiple tissues. Applying our approach in a high-throughput manner, we screened and identified 10 novel neurodevelopmental disorders and 50 hearing genes not previously studied in zebrafish. Altogether, our approach achieves high phenotypic penetrance using low numbers of gRNAs per gene in F0 zebrafish, offering a robust pipeline for rapidly characterizing candidate human disease genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d442e32931385bdcd94b5a33eaea8e6b64c86c8" target='_blank'>
              Optimizing gRNA selection for high-penetrance F0 CRISPR screening for interrogating disease gene function
              </a>
            </td>
          <td>
            Sheng-Jia Lin, Kevin Huang, C. Petree, Wei Qin, Pratishtha Varshney, Gaurav K Varshney
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43d3beaaac78a42fdc6dae5c6e9c46a2476c5705" target='_blank'>
              Abstract 6709: NF1 mutations in lung adenocarcinoma preclinical models and potential targeted therapies: The crucial role of the RAS-MAPK pathway
              </a>
            </td>
          <td>
            J. Giraud, Doriane Gorret, Manuela Ye, Dominique Lallemand, William C. Reinhold, Didier Decaudin, Ingrid Laurendeau, E. Pasmant, Camille Tlemsani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7430000b0cae106ff398286a8c011c0d35dbd7" target='_blank'>
              Loss-of-Function genetic Screen Unveils Synergistic Efficacy of PARG Inhibition with Combined 5-Fluorouracil and Irinotecan Treatment in Colorectal Cancer
              </a>
            </td>
          <td>
            Cristina Queralt, Cristina Moreta-Moraleda, Marta Costa, Ferran Grau-Leal, J. Diesch, Carla Vendrell-Ayats, Eva Musulén, C. Bugés, José Luis Manzano, Sara Cabrero, Johannes Zuber, M. Buschbeck, Sonia Forcales, E. Martínez-Balibrea
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f09c779b15647c7ef62bbebbbbd96b9880a1d08" target='_blank'>
              PROX1: a key regulator of hepatocyte identity and tumorigenesis.
              </a>
            </td>
          <td>
            T. K. Lee, Stephianie Ma
          </td>
          <td>2025-03-31</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Lynch-like syndrome (LLS) presents very similar clinicopathological characteristics to Lynch syndrome (LS) but the mechanism for cancer predisposition remains unknown. The present study aims to investigate the causal mechanism of LLS by a comprehensive genetic and epigenetic approach. Thirty-two LLS and 34 LS patients with colorectal cancer (CRC) fitting the Amsterdam and Bethesda criteria were included, along with 29 CRC sporadic patients, and analyzed for the presence of pathogenic variants in 94 genes associated with hereditary tumors. The cohorts were also characterized for the methylation profile and examined through a sample group analysis and a Stochastic Epigenetic Mutations (SEMs) analysis in comparison with 29 age-matched healthy controls. The multigene panel analysis revealed the presence of pathogenic variants in non-mismatch repair (MMR) genes and three variants classified as pathogenic/likely pathogenic possibly predisposing to LLS. The epigenetic analysis showed epivariations targeting genes associated with LS or DNA repair, most of them associated with the Fanconi Anemia pathway, which could explain the susceptibility to cancer. Our results highlight the need for using extended genetic and epigenetic analyses to understand the causal mechanism of LLS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49b3bcb3d908521020f7834061654c58ccb871fd" target='_blank'>
              Comprehensive genetic and epigenetic characterization of Lynch-like syndrome patients.
              </a>
            </td>
          <td>
            F. Pirini, L. Calzari, G. Tedaldi, M. Tebaldi, Valentina Zampiga, Ilaria Cangini, R. Danesi, M. Ravegnani, Valentina Arcangeli, A. Passardi, E. Petracci, S. Bravaccini, G. Marisi, A. Viel, D. Barana, M. Pedroni, L. Roncucci, D. Calistri, Davide Gentilini
          </td>
          <td>2025-04-21</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01701fc1080f2a22b516da6a31e565685ef5a558" target='_blank'>
              μSeq: Universal mutation rate quantification via deep sequencing of a single clonal expansion
              </a>
            </td>
          <td>
            Simone Pompei, Alberto Geroldi, Pietro Rivetti, Elena Grassi, V. Vurchio, Giorgio Tallarico, Giorgio Corti, Lorenzo Tattini, Gianni Liti, Andrea Bertotti, M. Lagomarsino
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64abcfa81e97e4ccd6aa92a436d6301cbd6a137c" target='_blank'>
              CRISPR-Cas9: Transforming Functional Genomics, Precision Medicine, and Drug Development - Opportunities, Challenges, and Future Directions.
              </a>
            </td>
          <td>
            Zuber Khan, Mumtaz, Sumedha Gupta, Sidharth Mehan, Tarun Sharma, Manjeet Kumar, Pankaj Kumar Maurya, Arun Kumar Sharma, Ghanshyam Das Gupta, A. Narula
          </td>
          <td>2025-03-21</td>
          <td>Current gene therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7504c8566a459436aca1443282d3b91fd8a63ab8" target='_blank'>
              MOLECULAR BIOLOGY OF CANCER AND NEW TOOLS IN ONCOLOGY
              </a>
            </td>
          <td>
            Mauricio Zuluaga Zuluaga, Santiago Zuluaga, Dr. Avrina Kartika Ririe, Nolka J Omana, Njoku Chinyere Lizzy, Dr Nyiayu Fauziah Kurniawaty, Sp.PA
          </td>
          <td>2025-04-30</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="NTRK1, NTRK2, and NTRK3 gene fusions are rare oncogenic driver alterations found in diverse tumor types of adults and children. They are clinically important biomarkers as tumors harboring these genomic alterations have high response rates to targeted therapy. Routine testing for NTRK fusions and treatment with TRK inhibitors has been recommended in multiple tumor types; however, differences between testing technologies used for detecting NTRK fusions can result in variable likelihoods of identification.To assess the prevalence of NTRK fusions in a real-world standard-of-care setting, we analyzed data from 19,591 FFPE samples encompassing 35 solid tumor types submitted for comprehensive genomic profiling (CGP) as part of routine clinical care. CGP testing included DNA hybrid-capture sequencing to detect small variants, copy number alterations, microsatellite instability (MSI), and tumor mutational burden (TMB). RNA hybrid-capture sequencing was concurrently performed to detect fusions and splice variants. Detected NTRK fusions were categorized as oncogenic, likely oncogenic, or variant of unknown significance (VUS) based on the fusion partner, orientation, and breakpoint position.73 oncogenic or likely oncogenic NTRK fusions were identified in 69 unique tumor specimens across 19 tumor types for a total cohort prevalence of 0.35%. Tumor types with the highest NTRK fusion prevalence included glioblastoma (1.91%), small intestine (1.32%), and head and neck (0.95%) tumors with other solid tumor types ranging from 0.19% (uterine) to 0.63% (breast). We identified diverse intra and inter-chromosomal partner genes for NTRK fusions across all tumor types. Most NTRK fusions were detected in only one tumor specimen, though some recurrent fusions were noted with ETV6, TPM3, LMNA, EML4, TPR, PEAR1, IRF2BP2, and KANK1 fusion partners. Most NTRK fusions were mutually exclusive from other genomic driver alterations, however, almost a third of tumor specimens (29%) contained at least one co-occurring genomic driver, which may affect treatment decisions.The high prevalence of oncogenic and likely oncogenic NTRK fusions detected in our analysis suggests that RNA hybrid-capture-based sequencing for fusion detection is a highly sensitive method for identifying clinically meaningful known and novel NTRK fusions, which may be missed with other detection methods, directly impacting therapeutic options and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b25c599c72376da134e4b30a3445743d8254f163" target='_blank'>
              RNA hybrid-capture next-generation sequencing has high sensitivity in identifying known and less characterized oncogenic and likely oncogenic NTRK fusions in a real-world standard-of-care setting
              </a>
            </td>
          <td>
            Zachary D. Wallen, Marni B Tierno, Erica Schnettler, Alison Roos, Michelle F. Green, Kobina Amoah, R. Previs, S. Hastings, S. Pabla, Taylor J. Jensen, Brian Caveney, Marcia Eisenberg, P. Sathyan, Shakti H. Ramkissoon, Eric A Severson
          </td>
          <td>2025-05-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Gene fusions represent important oncogenic driver mutations resulting in aberrant cellular signaling. In up to 17% of all solid tumors at least one gene fusion can be identified. Precision therapy targeting fusion gene signaling has demonstrated effective clinical benefit. Advancements in clinically relevant next-generation sequencing and bioinformatic techniques have enabled expansion of therapeutic opportunity to subpopulations of patients with fusion gene expression. Clinically, tyrosine inhibitors have shown efficacy in treating fusion gene expressing cancers. Fusion genes are also clonal mutations, meaning it is a personal cancer target involving all cancer cells of that patient, not just a subpopulation of cancer cells within the cancer mass. Thus, both fusion signal disruption and immune signal targeting are effective therapeutic directions. This review discusses fusion gene targeting, therapeutic resistance, and molecular biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a39d77dcb6dc4121a7fa7e989f8a588d3f3ba74" target='_blank'>
              Clinical characterization and therapeutic targeting of fusion genes in oncology
              </a>
            </td>
          <td>
            Susan Morand, Lauren Rager, Daniel Craig, Alexander Nemunaitis, Khalil Choucair, Donald Rao, L. Stanbery, Richard C Phinney, Adam Walter, Maurizio Ghisoli, John Nemunaitis
          </td>
          <td>2025-03-24</td>
          <td>Future Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c29e60f63dffafbff9d001a2d3645ac87336a0cf" target='_blank'>
              G6PC3 promotes genome maintenance and is a candidate mammary tumor suppressor.
              </a>
            </td>
          <td>
            Xin Li, Maria Rossing, Ana Moisés da Silva, Muthiah Bose, Thorkell Gudjónsson, Jan Benada, J. Thatte, Jens Vilstrup Johansen, J. Börcsök, H. van der Gulden, Ji-Ying Song, Renée Menezes, Asma Tajik, Lucía Sena, Zoltan Szallasi, Morten Frödin, J. Jonkers, F. C. Nielsen, C. S. Sørensen
          </td>
          <td>2025-04-22</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5fa7d80af5b43992f24ebd211378eba4c94dea" target='_blank'>
              Abstract 503: Tracking evolution of transcriptional landscapes during colorectal cancer development and progression
              </a>
            </td>
          <td>
            Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, J. Edwards, Owen J. Sansom, Philip D Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common cancers worldwide, with KRAS mutations occurring in approximately 40% of cases. These mutations drive tumorigenesis through the constitutive activation of key signaling pathways, such as RAS-RAF-MEK-ERK (MAPK) and PI3K-AKT-mTOR, contributing to therapeutic resistance and poor prognosis. Advances in molecular biology have led to significant breakthroughs, including the development of KRAS G12C inhibitors, such as sotorasib and adagrasib, which have shown promise in clinical trials. However, their efficacy is limited to a small subset of KRAS-mutant CRC, and resistance mechanisms often emerge through compensatory pathway activation. Combination strategies, including KRAS inhibitors with anti-EGFR agents, have been explored in trials like KRYSTAL-1 and CodeBreaK 300. Emerging research highlights the role of the tumor microenvironment in immune evasion and therapeutic resistance, offering opportunities for novel immunotherapy approaches, including KRAS neoantigen vaccines and adoptive T-cell therapy. Despite these advancements, challenges such as intratumoral heterogeneity, limited immune infiltration, and non-G12C KRAS mutations remain significant hurdles. This review provides a comprehensive overview of the molecular mechanisms, current advances and challenges, and future prospects in the management of KRAS-mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21dad5695932a1ea3976fffa19e82d922b6f2098" target='_blank'>
              Targeting the KRAS Oncogene for Patients with Metastatic Colorectal Cancer
              </a>
            </td>
          <td>
            Ruoyu Miao, James Yu, Richard D Kim
          </td>
          <td>2025-04-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0db92d7678fd17061debda8f9cf53b805a41693" target='_blank'>
              Abstract 7145: The pan-cancer landscape of RIT1 genetic alterations
              </a>
            </td>
          <td>
            Matteo Repetto, P. Selenica, S. Yang, B. Weigelt, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7ab372dd4cae4e9e87fbdd2a61f62f73ea8595e" target='_blank'>
              Abstract 2973: Discovery of novel synthetic lethal targets for effective and safe colorectal cancer therapies
              </a>
            </td>
          <td>
            Andrzej Mazan, M. Dulęba, E. Zimoląg, J. Szuszkiewicz, M. Serocki, Joanna Krawczyk, O. Bryzghalov, Agata Stachowicz-Wałaszek, Izabella Wieckowska, M. Chmiel, A. Kurzejamski, B. Lipert, Katarzyna Sarad, M. Mikula, Rafal Dziadziuszko, T. Rzymski, Andrew Thomason, K. Brzózka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc63f157070e0fd70ef6717ad423137ac275d75" target='_blank'>
              Stochastically Emergent Tumors offer in vivo whole genome interrogation of cancer evolution from non-malignant precursors
              </a>
            </td>
          <td>
            Ruhollah Moussavi-Baygi, Matthew J. Ryan, Woogwang Sim, Samuel B. Hoelscher, V. Luga, Arun A. Chandrakumar, Lore Hoes, Junghwa Cha, Young Sun Lee, Katelyn Herm, Ben Doron, C.K. Cornelia Ding, Bradley A. Stohr, Peng Jin, Tejasveeta V. Nadkarni, Xiangyi Fang, Melita Haryono, An Nguyen, Wouter Karthaus, C.L. Sawyers, Felix Y. Feng, Hani Goodarzi, Rohit Bose
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease. Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy. Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction. Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Recent advances in screening programs and the development of innovative therapeutic strategies have significantly improved the clinical outcomes of cancer patients. However, many patients still experience treatment failure, primarily due to inherent or acquired drug resistance mechanisms. This challenge underscores the urgent need for novel therapeutic targets for the effective treatment of malignancies, as well as cancer-specific biomarkers to enhance early diagnosis and guide interventions. Epigenetic mechanisms, including DNA methylation, have recently garnered growing interest as key regulators of gene expression under both physiological and pathological conditions. Although epigenetic dysregulations are reliable tumor hallmarks, DNA methylation is still not routinely integrated into clinical practice, highlighting the need for further research to translate preclinical findings from the bench to the bedside. On these bases, the present review aims to illustrate the state of the art regarding the role of DNA methylation in cancer, describing the technologies currently available for DNA methylation profiling. Furthermore, the latest evidence on the application of DNA methylation hotspots in cancer diagnosis and prognosis, as well as the impact of epidrugs in cancer care, is discussed to provide a comprehensive overview of the potential clinical relevance of DNA methylation in advancing personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7000e596c1c5da0f5cbf0989fd9d5a1a4286bc4e" target='_blank'>
              Recent advances on gene-related DNA methylation in cancer diagnosis, prognosis, and treatment: a clinical perspective
              </a>
            </td>
          <td>
            Alessandro Lavoro, Daria Ricci, Giuseppe Gattuso, Federica Longo, Graziana Spoto, Anastasia Cristina Venera Vitale, Maria Chiara Giuliana, L. Falzone, Massimo Libra, S. Candido
          </td>
          <td>2025-05-05</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2db16c30d79726e181227a7646a42de7a4c218" target='_blank'>
              Abstract 829: ITGAV is a targetable vulnerability in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Gilbert K. Salloum
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/642babaec15e8d22494c611094dc188788c6a876" target='_blank'>
              Abstract 3807: Validation of a functional genomics screen based on lentiviral integration site analysis reveals single-gene disruptions enhancing CAR19 T-cell effectiveness in preclinical models
              </a>
            </td>
          <td>
            Beatriz Coutinho de Oliveira, Saaurav Bari, Jessica Kerr, Victoria Putzbach, Y. Phoon, J. Fraietta, Wei-Ting Hwang, Frederic D. Bushman, J. Melenhorst
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195e5c9d321296053fe8059a5c1ae5b1f465a2aa" target='_blank'>
              Abstract 5054: Understanding the genetic heterogeneity within clonal spheroids and its interaction with the tumor microenvironment
              </a>
            </td>
          <td>
            Anshu Yadav, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbe96f8609d981f1d51f682dd9dde5af13de525a" target='_blank'>
              Abstract 5533: Drug tolerance mechanisms in patient-derived xenograft models with epidermal growth factor receptor mutated tumors treated with tyrosine kinase inhibitors
              </a>
            </td>
          <td>
            A. Arivajiagane, Q. Li, Nhu-An Pham, Ming Li, Katrina Hueniken, Takamasa Koga, N. Radulovich, M S Tsao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339780e1e052f73bd7a155d38cbe11ed7059d53c" target='_blank'>
              Abstract 1253: Single-cell analysis reveals intricate subclonal heterogeneity and evolution in response to standard of care treatments in glioblastoma
              </a>
            </td>
          <td>
            Yueshan Zhao, Xiaoyu Lu, Verah Nyarige, Junfei Zhao, Gonzalo Lopez, Ann Forslund, Maria Ortiz-Estevez, Jorge Benitez-Hernandez, Wei Zhang, C. Fontanillo, Kai Wang, Ha Dang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/700ea58a4c1d36566c271fc146a1aaff828a43d8" target='_blank'>
              Abstract 3652: Celleporter: a foundation model of cell and tissue biology with application to patient stratification and target discovery
              </a>
            </td>
          <td>
            Yubin Xie, Eshed Margalit, Tyler Van Hensbergen, Dexter Antonio, Jake Schmidt, Yu (Phoebe) Guo, Jérémie Decalf, Maxime Dhainaut, Lucas Cavalcante, Hargita Kaplan, Rodney Collins, Francis Fernandez, Rob Schiemann, Eric Siefkas, Michela Meister, Joy Tea, Carl Ebeling, Anastasia Mavropoulos, Nicole Snell, Shafique Virani, Ronald Alfa, Lacey J Padrón, Jacob Rinaldi, Daniel Bear
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1f9135b22221853a957c30c494e32f6d68e606" target='_blank'>
              Abstract LB016: Targeting ATM modulates oncogenic pathways and amplifies chemotherapy efficacy in small cell lung cancer
              </a>
            </td>
          <td>
            Debdatta Halder, Utsav Sen, V. Jethalia, Subhamoy Chakraborty, Yosuke Dotsu, Andrew Elliot, A. Vanderwalde, D. Demircioglu, D. Hasson, Triparna Sen
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a8992e140899a62dade76814a52a30a531da19" target='_blank'>
              Abstract 5026: Discovery of oncogenic mediator genes in rectal cancer chemotherapy response using gene expression data from matched tumor and patient-derived organoids
              </a>
            </td>
          <td>
            Hanchen Huang, W. Zambare, Chao Wu, A. Colaprico, J. Alvarez, Min Jung Kim, A. Bercz, P. Bleu, Lily Wang, P. Paty, P. Romesser, J. J. Smith, X. Chen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1d772bceb37d658c289fadfd2de9d117e82a8a" target='_blank'>
              Abstract 4103: MLL4/COMPASS dysfunction in cancer and treatment
              </a>
            </td>
          <td>
            Zibo Zhao, Sarah Gold, Yuki Aoi, K. Meghani, Luke St John, Rukkia Liaqat, Carolyn Moloney, Yanni Yu, Jun Qian, I. Ben-Sahra, Rintaro Hashizume, D. Mahalingam, Joshua Meeks, A. Shilatifard
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular senescence, a stable state of cell cycle arrest induced by various stressors or genomic damage, is recognized as a hallmark of cancer. It exerts a context-dependent dual role in cancer initiation and progression, functioning as a tumor suppressor and promoter. The complexity of senescence in cancer arises from its mechanistic diversity, potential reversibility, and heterogeneity. A key mediator of these effects is the senescence-associated secretory phenotype (SASP), a repertoire of bioactive molecules that influence tumor microenvironment (TME) remodeling, modulate cancer cell behavior, and contribute to therapeutic resistance. Given its intricate role in cancer biology, senescence presents both challenges and opportunities for therapeutic intervention. Strategies targeting senescence pathways, including senescence-inducing therapies and senolytic approaches, offer promising avenues for cancer treatment. This review provides a comprehensive analysis of the regulatory mechanisms governing cellular senescence in tumors. We also discuss emerging strategies to modulate senescence, highlighting novel therapeutic opportunities. A deeper understanding of these processes is essential for developing precision therapies and improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286d205296f9a0a8d06235700589eef3dccbbc34" target='_blank'>
              Regulation of cellular senescence in tumor progression and therapeutic targeting: mechanisms and pathways
              </a>
            </td>
          <td>
            Bowei Liu, Zhigang Peng, Hao Zhang, Nan Zhang, Zao-bin Liu, Zhiwei Xia, Shaorong Huang, Peng Luo, Quan Cheng
          </td>
          <td>2025-04-02</td>
          <td>Molecular Cancer</td>
          <td>2</td>
          <td>21</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>Nature</td>
          <td>2</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b855a13f4583cca6b74a017150a15f6334f81973" target='_blank'>
              Abstract 2773: The Gundersen Precision Oncology Cohort: a biospecimen resource for mutation characterization and target discovery
              </a>
            </td>
          <td>
            Peter Feiszt, Paraic A. Kenny
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Somatic‐type malignancies (SMs) arising in germ cell tumors (GCTs) are aggressive neoplasms resistant to systemic treatment. Most are diagnosed in metastatic sites after chemotherapy; however, they have also been well‐documented in primary testicular GCTs. Historically, SMs were thought to originate in components of teratoma that acquire molecular alterations equivalent to those that characterize their true somatic counterparts. However, recent studies have shown that SMs typically lack the hallmark molecular alterations seen in similar somatic tumors. Additionally, clinicopathologic and molecular data suggest that a subset may derive from yolk sac tumor (YST) rather than teratoma. In this study, we evaluated the relationship between conventional histological types of GCTs and SMs by comparing expression of microRNA (miR)‐371–373 and genomic methylation profiles. A total of 96 samples (including multiple paired conventional GCT–SM samples from individual tumors) were assessed for miR‐371–373 expression by RT‐qPCR and genomic DNA methylation using a clinically validated assay. Expression of miR‐371–373 was higher in conventional GCTs than in SMs (considered as a single category encompassing all histological subtypes). However, miR‐371–373 expression was heterogeneous among SMs, with significantly higher levels in sarcomatoid YST (SYST) and glandular neoplasms than in other SMs. Genomic DNA methylation analysis showed that SMs (considered as a single category) did not form a distinct cluster. Instead, they grouped into multiple clusters that did not show perfect correspondence with histology and often included conventional GCTs. Genome‐wide methylation assessment showed a higher abundance of hypermethylated regions in SMs than in conventional GCTs. Analysis of paired conventional GCT and ‘somatic‐type’ components that did not meet size criteria for SMs dissected from individual tumors demonstrated separation according to histology, suggesting that epigenetic processes play a role in the transition from conventional GCT to ‘somatic‐type’ phenotypes. Gene‐level and pathway‐level analyses identified MAPK/RAS signaling, mitosis/proliferation, differentiation towards neural tissue/neuroectoderm, epithelial‐to‐mesenchymal transition, and DNA repair as key differentially regulated processes in components with somatic‐type histology, suggesting mechanisms of progression from conventional to ‘somatic’ phenotypes in GCT. These results support the hypothesis that a subset of SMs derive from YST and suggest that some subtypes (such as SYST) may represent ‘intermediate’ phenotypes. Additionally, analysis of differentially methylated promoter regions in SM identified genes and biologic processess that may underlie 'somatic tranformation' in GCTs. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175bd3895700f513d14429236b145a33de18651c" target='_blank'>
              MicroRNA‐371–373 cluster and methylome analysis suggests that a subset of ‘somatic‐type’ malignancies arising in germ cell tumors may originate in yolk sac tumor components
              </a>
            </td>
          <td>
            João Lobo, Nuno Tiago Tavares, Diana Fonseca, Carmen Jerónimo, Rui Henrique, Nicolas Wyvekens, Yiying Yang, Matija Snuderl, Fiona Maclean, Jennifer B. Gordetsky, C. Fletcher, Michelle S Hirsch, Jason L. Hornick, M. T. Idrees, Katrina Collins, L. Warmke, Thomas Ulbright, Andres M Acosta
          </td>
          <td>2025-03-28</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Despite advances, Glioblastoma (GBM) treatment remains challenging due to its rapid progression and resistance to therapies. Objectives This study aimed to investigate the role of anoikis—a mechanism by which cells evade programmed cell death upon detachment from the extracellular matrix—in GBM progression and prognosis. Methods Utilizing single-cell sequencing and bulk-transcriptome sequencing data from TCGA, GEO, and CGGA databases, we performed comprehensive bioinformatics analyses. We identified anoikis-related genes, constructed a prognostic model using 101 machine learning algorithms, and validated its clinical utility across multiple cohorts.Finally, we also verified the expression of model genes and the function of key gene in clinical samples and cell lines. Results Single-cell sequencing revealed heterogeneous expression of anoikis-related genes across distinct cell populations within GBM. MES-like Malignant cells and Myeloids exhibited higher enrichment of these genes, implicating their role in anoikis resistance and tumor aggressiveness. The prognostic model, based on identified genes, effectively stratified patients into high-risk and low-risk groups, demonstrating significant differences in survival outcomes. Mutation and tumor microenvironment analyses highlighted distinct genetic landscapes and immune cell infiltration patterns associated with different risk groups. SLC43A3 emerged as a key gene, showing significant upregulation in tumor tissues and correlating with poor prognosis in GBM. Conclusion This study provides insights into the molecular mechanisms of anoikis resistance in GBM, underscoring its critical role in tumor progression and patient prognosis. The developed prognostic model offers a promising tool for personalized treatment strategies and warrants further exploration of targeted therapies to improve outcomes for GBM patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03752-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90776aa9e4e44d828b89f11f0b82b3f7faa8093b" target='_blank'>
              Unraveling anoikis in glioblastoma: insights from single-cell sequencing and prognostic modeling
              </a>
            </td>
          <td>
            Q. Tang, Chenfeng Ma, Jiaheng Xie, Qixiang Zhang, Bingtao Zhang, Weiqi Bian, Qingyu Lu, Zeyu Wan, Wei Wu
          </td>
          <td>2025-03-26</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5750dddf572726400a2fad0203c2ec7fc9717955" target='_blank'>
              Abstract 478: Engineering viruses for the next generation targeted cancer therapy
              </a>
            </td>
          <td>
            Ikeda Trashi, Jeremiah Gassensmith
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ca665dc53860039a134586f0a4361eb2c643769" target='_blank'>
              Abstract 5645: Targeting transcription-replication conflict for selective chemotherapy in glioblastoma
              </a>
            </td>
          <td>
            Long Gu, Robert J. Hickey, L. Malkas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2116c40f11199361c9cb93d0a62e0a019f2b44da" target='_blank'>
              Abstract 4681: Multiomic insights into mechanisms of androgen receptor-driven radioresistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Benjamin Hauk, B. McBean, A. Michmerhuizen, Douglas Gurdak, Savannah Tidmore, Michelle Paulsen, L. Lerner, C. Ward, Kassidy M. Jungles, Daniel Spratt, Lori J. Pierce, Mats Ljungman, C. Speers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4afa94be560741f52b5090eabfe589892d769b54" target='_blank'>
              Modulating populational variance of methyl-guanine methyl transferase expression through miR-181d degradation: a novel mechanism of temozolomide resistance
              </a>
            </td>
          <td>
            Gatikrushna Singh, Shilpi Singh, Iteeshree Mohapatra, Stefan Kim, Mayur Sharma, Johnny Akers, Thien Nguyen, Eric Wong, Margot Martinez Moreno, E. Kokkoli, Shobha Vasudevan, Sean E Lawler, W. El-Deiry, Z. Gokaslan, Clark C. Chen
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f17818e89e418eef53189bf0e536334d7fee6f" target='_blank'>
              Abstract 5042: The Paipu pipeline for pan-mammalian cancer genomics
              </a>
            </td>
          <td>
            Bria S. Smith, Geesa Daluwatumulle, Leslie A. Smith, James A. Cahill, Kiley Graim
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ca3a1f08d426b3a13684391aa11a6fb1c8556a" target='_blank'>
              Abstract LB443: Unlocking cell cycle dynamics: Gene regulation of CRISPR-Cas9 and siRNA phenotypes using cell painting
              </a>
            </td>
          <td>
            Alexander Schreiner, A. Foitzik, Josien Levenga, Karin Boettcher, Barbara Sonnenberg, Z. Strezoska, Joe Trask
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a9e6faf45fc758543cd3dfb6d7c7bf6cfe5d5f" target='_blank'>
              Abstract 6892: Ribonucleotide reductase is a potential therapeutic target in colorectal cancer
              </a>
            </td>
          <td>
            Federica Invrea, Consalvo Petti, C. Isella, E. Médico
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, F. Vizeacoumar, Yue Zhang, L. Kyrylenko, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are short non-coding RNAs, which perform a key role in cellular differentiation and development. Most human diseases, particularly cancer, are linked to miRNA functional dysregulation implicated in the expression of tumor-suppressive or oncogenic targets. Cancer hallmarks such as continued proliferative signaling, dodging growth suppressors, invasion and metastasis, triggering angiogenesis, and avoiding cell death have all been demonstrated to be affected by dysregulated miRNAs. Thus, for the treatment of different cancer types, the detection and quantification of this type of RNA is significant. The classical and current methods of RNA detection, including northern blotting, reverse transcription-quantitative PCR, rolling circle amplification and next-generation sequencing, may be effective but differ in efficiency and accuracy. Furthermore, these approaches are expensive, and require special instrumentation and expertise. Thus, researchers are constantly looking for more innovative approaches for miRNA detection, which can be advantageous in all aspects. In this regard, an RNA manipulation tool known as the CRISPR and CRISPR-associated sequence 13 (CRISPR/Cas13) system has been found to be more advantageous in miRNA detection. The Cas13-based miRNA detection approach is cost effective and requires no special instrumentation or expertise. However, more research and validation are required to confirm the growing body of CRISPR/Cas13-based research that has identified miRNAs as possible cancer biomarkers for diagnosis and prognosis, and as targets for treatment. In the present review, current updates regarding miRNA biogenesis, structural and functional aspects, and miRNA dysregulation during cancer are described. In addition, novel approaches using the CRISPR/Cas13 system as a next-generation tool for miRNA detection are discussed. Furthermore, challenges and prospects of CRISPR/Cas13-based miRNA detection approaches are described.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e7cad934b860ca66c2574dc37104aa0e67005c2" target='_blank'>
              Current updates regarding biogenesis, functions and dysregulation of microRNAs in cancer: Innovative approaches for detection using CRISPR/Cas13-based platforms (Review)
              </a>
            </td>
          <td>
            Abdulaziz A. Aloliqi, Abdullah M. Alnuqaydan, Aqel Albutti, Basmah F. Alharbi, A. Rahmani, Amjad Ali Khan
          </td>
          <td>2025-04-11</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a605a688e58842e081be2262f0d61629173ea95c" target='_blank'>
              Abstract 1263: Reconstructing small cell lung cancer evolution using deep whole genome sequencing of circulating tumor DNA
              </a>
            </td>
          <td>
            Benjamin B. Morris, Zhihui Zhang, Andrew R. Lynch, Simon Heeke, Kyle Concannon, C. Stewart, Runsheng Wang, Alexa Halliday, W. Rinsurongkawong, Vadeerat Rinsurongkawong, J. J. Lee, Jianjun Zhang, Don L Gibbons, A. Vaporciyan, C. Gay, Hai T. Tran, J. Heymach, L. Byers, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6b6f2b84b8146ac33e92413473a3aba6e98338" target='_blank'>
              Abstract 1508: Therapy resistant cancer cells containing stress granules display signatures of oxidative stress
              </a>
            </td>
          <td>
            Louis T. A. Rolle, Luke V. Loftus, Kenneth J. Pienta, Sarah R Amend
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9365edb5d9a7621f7c2fa255adc2d1f2aebf2bfd" target='_blank'>
              Abstract 5334: Proliferative signaling through the ras-erk pathway can be inhibited by the overexpression of dual-specific phosphatase genes in collaboration with other negative regulators of cell cycle progression in the acute myeloid cell line HL-60
              </a>
            </td>
          <td>
            Michael Roberts, Amelia Harper, Kishan Mangru, Katy Meta, Sophia Kovatsis, J. Forrester, Grace Crossland, Cuong Nguyen, S. Bahadur
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3eb942377bafe8f7e43907ef0e3522eb17cee58" target='_blank'>
              1333 THY1 Upregulates Glioma Stem Cell Phenotype and Induces Treatment Resistance in Glioblastoma
              </a>
            </td>
          <td>
            Arushi Tripathy, S. Syed, V. Ravikumar, Dah-Luen Huang, A. Rehemtulla, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3d68234ed3b90b54b46e31e83216b5a77d6d67" target='_blank'>
              Abstract 5055: Efficient detection of chromosomal instability in diverse cancers using a targeted sequencing approach
              </a>
            </td>
          <td>
            H. Bao, Rui Liu, Haimeng Tang, Ningyou Li, Min Wu, Xunbiao Liu, Zhili Chang, Xue Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c906c5a3862aab5cd84c1716303e1ffc00c0344c" target='_blank'>
              NSD3: A Promising Target for Cancer Therapy.
              </a>
            </td>
          <td>
            Ting Huang, Bowen Zhang, Yifan Yang, Qiong Lin, Genbao Shao
          </td>
          <td>2025-03-26</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/434c5eb06ef2c6750531e761d704cb04ab799fe6" target='_blank'>
              Management of T-cell malignancies: Bench-to-bedside targeting of epigenetic biology.
              </a>
            </td>
          <td>
            Ariana Sabzevari, J. Ung, Jeffrey W. Craig, K. Jayappa, Ipsita Pal, D. Feith, Thomas P Loughran, Owen A O'Connor
          </td>
          <td>2025-04-15</td>
          <td>CA: a cancer journal for clinicians</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b4e4b812f53d84fe2b91abe7b8dd8fa8cfa234" target='_blank'>
              Abstract 1514: CRISPRi screening reveals RNA editing as a major determinant of platinum sensitivity in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Yi Wen Kong, E. Handly, Robin A. Lu, Molly A. Bird, Jack W. Schaeffer, Samkyu P. Yaffe, Michael B Yaffe
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c16f756ddb575cdf26b92c703a6e6c80ae8425f" target='_blank'>
              Abstract 6358: Single cell transcriptional dynamics in the HCMI cancer model collection
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Pérez-Mayoral, Megan J. Stine, Eva Y. Tonsing-Carter, James M. Clinton, The Hcmi Network, Peter W. Laird, Calvin J. Kuo, O. Elemento, David L. Spector, Andrew D Cherniack, Kyle Elrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Andrew McPherson, M. Garnett, D. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse S. Boehm
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c882aec893125b1655a88dcecde1b32dd7df1e2c" target='_blank'>
              Abstract 4094: Allosteric chromatin activity transition mechanisms drive epigenetic dosage compensation in essential genes triggered by oncogene activation in TRACERx
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, Xiaohong Liu, Georgia Stavrou, Nana E. Mensah, E. L. Cadieux, Özgen Deniz, M. Jamal-Hanjani, Stephan Beck, Jonas Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f4f8607796131836a1e01cc0a05131a1e222c3" target='_blank'>
              Abstract 5280: A strategic framework for modeling abnormal quiescence that drives cancer evolution, and for preclinical development of therapies to prevent recurrence and metastasis in high-risk patients
              </a>
            </td>
          <td>
            Balraj Singh, Anthony Lucci
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objective. Unlike genetic changes, epigenetic aberrations in prostate cancer can be reversed under the influence of a chemical agent. This fact makes the study of epigenetic changes an important object as potential therapeutic targets. Material and methods. PubMed, Medline, eLibrary.ru databases were analyzed for the keywords: epigenetic prostate cancer, lineage plasticity, neuroendocrine differentiation. For this literature review, 84 relevant publications were selected. The review included studies from 1982 to 2024. Results. The most widely studied epigenetic mutations are DNA hypo- and hypermethylation, histone variability (methylation and acetylation), and neuroendocrine differentiation. Conclusion. The study of the genomic landscape can reveal new opportunities for improving the diagnosis and therapy of castration-resistant prostate cancer (CRPC), which is a potentially lethal form of the disease. It is important not only to search for new biomarkers to identify genetic disorders, but also to study the optimal therapy for advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6160d8f350a3cb980348d4d0719c54ff7d5b6123" target='_blank'>
              Epigenetic abnormalities and neuroendocrine differentiation in prostate cancer
              </a>
            </td>
          <td>
            G. Kovchenko, A. Sivkov, L. N. Lyubchenko, A. Kaprin
          </td>
          <td>2025-03-18</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="This study presents a comprehensive genetic characterization of the H9c2 cell line, a widely used model for cardiac myoblast research. We established a short tandem repeat (STR) profile for H9c2 that is useful to confirm the identity and stability of the cell line. Additionally, we prepared H9c2 metaphase chromosomes and performed karyotyping and molecular cytogenetics to further investigate chromosomal characteristics. The genetic analysis showed that H9c2 cells exhibit chromosomal instability, which may impact experimental reproducibility and data interpretation. Next-generation sequencing (NGS) was performed to analyze the transcriptome, revealing gene expression patterns relevant to cardiac biology. Western blot analysis further validated the expression levels of selected cardiac genes identified through NGS. Additionally, Phalloidin staining was used to visualize cytoskeletal organization, highlighting the morphological features of these cardiac myoblasts. Our findings collectively support that H9c2 cells are a reliable model for studying cardiac myoblast biology, despite some genetic alterations identified resembling sarcoma cells. The list of genes identified through NGS analysis, coupled with our comprehensive genetic analysis, will serve as a valuable resource for future studies utilizing this cell line in cardiovascular medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1647a813c18457ce232ec6ab89eefeaf1dce3b9a" target='_blank'>
              Genetic and Molecular Characterization of H9c2 Rat Myoblast Cell Line
              </a>
            </td>
          <td>
            Thomas Liehr, Stefanie Kankel, Katharina S. Hardt, E. Buhl, H. Noels, Diandra T. Keller, Sarah K. Schröder-Lange, Ralf Weiskirchen
          </td>
          <td>2025-03-28</td>
          <td>Cells</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/973aff9217cd2f6bd7d72f2139353fd552c0fa4e" target='_blank'>
              SARE: an optimized method for detection functional regulatory elements in genome
              </a>
            </td>
          <td>
            Mohammadjavad Hosseinpoor
          </td>
          <td>2025-04-16</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The leading cause of human pregnancy loss is aneuploidy, often tracing to errors in chromosome segregation during female meiosis. While abnormal crossover recombination is known to confer risk for aneuploidy, limited data have hindered understanding of the potential shared genetic basis of these key molecular phenotypes. To address this gap, we performed retrospective analysis of preimplantation genetic testing data from 139,416 in vitro fertilized embryos from 22,850 sets of biological parents. By tracing transmission of haplotypes, we identified 3,656,198 crossovers, as well as 92,485 aneuploid chromosomes. Counts of crossovers were lower in aneuploid versus euploid embryos, consistent with their role in chromosome pairing and segregation. Our analyses further revealed that a common haplotype spanning the meiotic cohesin SMC1B is significantly associated with both crossover count and maternal meiotic aneuploidy, with evidence supporting a non-coding cis-regulatory mechanism. Transcriptome- and phenome-wide association tests also implicated variation in the synaptonemal complex component C14orf39 and crossover-regulating ubiquitin ligases CCNB1IP1 and RNF212 in meiotic aneuploidy risk. More broadly, recombination and aneuploidy possess a partially shared genetic basis that also overlaps with reproductive aging traits. Our findings highlight the dual role of recombination in generating genetic diversity, while ensuring meiotic fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed8e001dc646667003aedc35613a33f0517e9a4" target='_blank'>
              Common variation in meiosis genes shapes human recombination phenotypes and aneuploidy risk
              </a>
            </td>
          <td>
            S. A. Carioscia, Arjun Biddanda, Margaret R. Starostik, Xiaona Tang, Eva R Hoffmann, Zachary Demko, R. McCoy
          </td>
          <td>2025-04-04</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de37ada758a8c9054780c41940fecac62f6f55b3" target='_blank'>
              Abstract 2665: Single-cell analysis and cytokines modulation of intra-tumor subpopulations in colorectal cancer tumoroids
              </a>
            </td>
          <td>
            S. J. Fletcher, Irene Catalano, Elena Grassi, Sofia Borgato, Barbara Peracino, Luca Primo, L. Trusolino, A. Bertotti, A. Puliafito
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8df54e46e7183e7945f80187fc6745c6cb3b628" target='_blank'>
              Abstract 4632: Molecular determinants to neoadjuvant chemotherapy resistance in breast cancer patients: Insights from a Latin-American cohort
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos A. Orozco, R. Parra-Medina, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Combita
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5392c951f2de30ee4a1335350f2dfe3bff927f11" target='_blank'>
              Circular RNAs: driving forces behind chemoresistance and immune evasion in bladder cancer.
              </a>
            </td>
          <td>
            M. Saadh, Wael Sheet Hussein, Ali Fawzi Al-Hussainy, A. K. Bishoyi, M. Rekha, Mayank Kundlas, V. Kavitha, Zafar Aminov, Sada Ghalib Taher, Mariem Alwan, M. Jawad, Hiba Mushtaq
          </td>
          <td>2025-03-25</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Overexpression of the H3K36 histone methyltransferase NSD2 in t(4;14) multiple myeloma (MM) is an early, oncogenic event, and understanding its impact on genomic organisation and expression is relevant to understanding MM biology. We performed epigenetic, transcriptional and phenotypic profiling of the t(4;14) KMS11 myeloma cell line and its isogenic translocation knock out (TKO) to characterise the sequelae of NSD2 overexpression. We found a marked global impact of NSD2 on gene expression and DNA organisation implicating cell identity genes; notably the early lymphocyte regulator, LAIR1 and MM cell surface markers, including CD38, a classical marker of plasma cells which was reduced in TKO cells. Plasma cell transcription factors such as PRDM1, IRF4 and XBP1 were unaffected, suggesting a downstream direct gene effect of NSD2 on cell identity. Changes in cell surface markers suggest an altered surface immunophenotype. Our findings suggest a role for NSD2 in maintaining MM cell identity, with potential implications for future therapeutic strategies based on targeting of NSD2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3900ea300ad687a680fc22372f0a01df92719adc" target='_blank'>
              NSD2-epigenomic reprogramming and maintenance of plasma cell phenotype in t(4;14) myeloma
              </a>
            </td>
          <td>
            Andrea Gunnell, Scott T. Kimber, R. Houlston, Martin Kaiser
          </td>
          <td>2025-03-21</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cf64c5137ce4675e3a801e6fb302671875782c" target='_blank'>
              Using Cancer Profiles to Identify Synthetic Lethal Therapeutic Targets and Predictive Biomarkers in Cancer Gene Dependency Data
              </a>
            </td>
          <td>
            Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2025-04-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/140dbaae467c306b2efa10ba8bbd45efee843365" target='_blank'>
              Abstract 6595: Lineage markers and driver oncogenes coordinate through gene expression subtypes of lung adenocarcinoma
              </a>
            </td>
          <td>
            Minjeong Kim, W. Lamlertthon, Heejoon Jo, Yan Cui, Hyo Young Choi, M. Wilkerson, Liza Makowski, D. N. Hayes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Chromatin modulators, like Polycomb group proteins, are key epigenetic regulators of gene expression and are frequently mutated in cancers. In adult stem cells, epigenetic regulation maintains their identity and controls their differentiation during homeostasis or aging, but its direct role in tumorigenesis remains unclear. This work explores the function of Polycomb Repressive Complex l (PRCl) in adult Drosophila intestinal stem cells (ISCs). Disrupting core PRCl components in ISCs induces the formation of small cell clusters devoid of intestinal markers, a novel phenotype linked to premature mortality under stress. Notably, transient PRCl loss is sufficient for cluster formation. These clusters overproliferate and exhibit immortality in serial transplantations, leading to their designation as tumor-initiating intestinal cells (TIICs). While JAK/STAT signaling promotes TIIC formation, the TOLL/IMD immune pathways restrict their expansion independently of cell death. Altogether, our results highlight PRCl as an epigenetic tumor suppressor in adult intestinal stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde642ec7f4f1326cc387e14ebb63c69e66e1638" target='_blank'>
              A tumor-suppressive role of the PRC1 Polycomb epigenetic complex in the maintenance of adult Drosophila intestinal stem cell identity
              </a>
            </td>
          <td>
            Aurélia Joly, Ana-Maria Popmihaylova, Corinne Rancurel, Julie Soltys, L. Fritsch, Rihab Loudhaief, Armel Gallet, Edan Foley, Anne-Marie Martinez, Raphaël Rousset
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e607ac4077bd0584584c78f861a3a32f882b386" target='_blank'>
              Chemical Strategies to Modulate and Manipulate RNA Epigenetic Modifications.
              </a>
            </td>
          <td>
            Liang Cheng
          </td>
          <td>2025-03-18</td>
          <td>Accounts of chemical research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba26ff57f58d982ea6db2c2b425446ece055593" target='_blank'>
              Exploring the Prognostic and Predictive Impact of Genomic Loss of Heterozygosity and Homologous Recombination Deficiency Alterations in Patients With Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            R. Moretto, M. Germani, M. Carullo, V. Conca, A. Minelli, M. Giordano, Rossella Bruno, D. Rossini, Eleonora Gusmaroli, M. D. De Grandis, C. Antoniotti, L. Salvatore, A. Passardi, S. Tamberi, M. Scartozzi, F. Pietrantonio, S. Lonardi, Clara Ugolini, G. Masi, C. Cremolini
          </td>
          <td>2025-04-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e73b3c04c7a8aff09920f9f47c81555c9e458661" target='_blank'>
              Predicting Treatment Outcomes from Adaptive Therapy — A New Mathematical Biomarker
              </a>
            </td>
          <td>
            Kit Gallagher, Maximillian Strobl, P. Maini, Alexander R. A. Anderson
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a24afa888bf591284938fd750888dc7dfa3335e" target='_blank'>
              Abstract 2846: Association of cyclin E1 expression with genomic instability in ovarian cancer
              </a>
            </td>
          <td>
            Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, M. Poi, Sharon Wu, E. Missiglia, Bettina Bisig, K. Homicsko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Histone modifications help control gene activity and are important for maintaining the health of spermatogonial stem cells (SSCs), which are necessary for male fertility. In this study, we used advanced genetic analysis to identify key genes related to SSC function and aging. We discovered 2509 genes that exhibited differential expression in SSCs compared to other cells, with important genes including KDM5B, SCML2, SIN3A, and ASXL3 playing roles in how DNA is organized and read. Further analysis showed that these genes play essential roles in maintaining DNA structure and regulating gene activity. We also identified gene groups associated with SSC aging and discovered potential signals that help SSCs remain active and proliferate. These findings could contribute to developing new methods to protect male fertility and advance SSC research and treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27053da94cb913a8fb23edd13a7dab3ef09f1fee" target='_blank'>
              Integration of Microarray and Single-Cell RNA-Seq Data and Machine Learning Allows the Identification of Key Histone Modification Gene Changes in Spermatogonial Stem Cells
              </a>
            </td>
          <td>
            Ali Shakeri Abroudi, Hossein Azizi, Melika Djamali, Ali Qorbanee, Thomas Skutella
          </td>
          <td>2025-04-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7d798ffee04bc73cc38a17396f4a99f40e9d7c9" target='_blank'>
              Dysfunctional mismatch repair in patients with early triple-negative breast cancer.
              </a>
            </td>
          <td>
            María Muñiz-Castrillo, Noel Blaya Boluda, E. García-Torralba, P. Jiménez-Fonseca, Carmen González Del Rey, Milagros Balbín, Ginés Luengo-Gil, F. Ayala de la Peña, Emilio Esteban González, A. Carmona-Bayonas
          </td>
          <td>2025-05-04</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83d1334544559641fc60de6e87647100af4874" target='_blank'>
              Abstract 3454: Charting single-cell metabolism co-evolution during metastasis
              </a>
            </td>
          <td>
            Yingcheng Charles Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eb9f7e3e34a8c78dbc563d40d2ad93cdb90edab" target='_blank'>
              Engineered protein circuits for cancer therapy
              </a>
            </td>
          <td>
            Andrew C. Lu, Lukas Moeller, Stephen Moore, Shiyu Xia, Kevin Ho, Evan Zhang, Mark W. Budde, Haley Larson, Ali Ahmed Diaz, Bo Gu, James M. Linton, Leslie Klock, Michael J. Flynn, Xiaojing J. Gao, D. Siegwart, Hao Zhu, M. Elowitz
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) typically develops in the context of chronic liver disease, where prolonged hepatocyte exposure to inflammation drives the synergistic accumulation of genetic and epigenetic alterations. Epigenetic regulation encompasses multiple mechanisms that govern the transcription machinery accessibility to DNA. This process is regulated by the addition and removal of covalent marks on chromatin, which can either affect DNA-histone interactions or serve as scaffolds for other proteins, among other mechanisms. Recent research has revealed that epigenetic alterations can disrupt chromatin homeostasis, redirecting transcriptional regulation to favour cancer-promoting states. Consequently, these alterations play a pivotal role in the acquisition of cancer hallmarks and provide insights into several biological processes involved in hepatocarcinogenesis. This review highlights the key epigenetic mechanisms underlying the development, progression and dissemination of HCC, with a particular focus on DNA methylation and histone post-translational modifications. This knowledge is relevant for guiding the development of innovative therapeutic approaches based on epigenetic modulators.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab1d7979671681364e95ca7a0ffe9920c08634f1" target='_blank'>
              Epigenetic mechanisms involved in hepatocellular carcinoma development and progression
              </a>
            </td>
          <td>
            Barbara Bueloni, Maite Garcia Fernandez de Barrena, Matias Antonio Avila, J. Bayo, Guillermo Mazzolini
          </td>
          <td>2025-05-01</td>
          <td>eGastroenterology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262a87481193ac96806ab02306fdc71e5a95bc29" target='_blank'>
              Abstract 273: Decipher actionable tumor suppressor defects in prostate cancer for precision medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d478b727dab85de298dfc2ae721a0a16fcf9bb42" target='_blank'>
              Abstract 4154: Cancer cells exhibit novel metabolic vulnerabilities when exposed to low lipid environments
              </a>
            </td>
          <td>
            Diya L. Ramesh, Keene L. Abbott, Raphael Ferreira, M. V. Vander Heiden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba50a4897b94e23b9a56ce785c4accfcfef0e23" target='_blank'>
              Abstract 655: Spatiotemporally deciphering the response dynamics of micro-satellite stable rectal cancer to neoadjuvant short-course radiotherapy and PD-1 blockade
              </a>
            </td>
          <td>
            Jingwen Wang, Zhen Zhang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7188b7f43eeddcafd2d8d2c5bfae450202c1e2b1" target='_blank'>
              Abstract 4694: (R)evolutionary approach to predicting individual response to radiation therapy in breast cancer
              </a>
            </td>
          <td>
            Naheel Khatri, Raafat Chalar, Saiful Samad, Alexander Stessin, Mehdi Damaghi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0431d2b0b8dd854cb5fd756e0cd0875bd2691218" target='_blank'>
              CRISPR/Cas9-mediated generation of two isogenic CEP290-mutated iPSC lines
              </a>
            </td>
          <td>
            Joana Figueiro-Silva, Melanie Eschment, Michelle Mennel, Affef Abidi, B. Oneda, Anita Rauch, R. Bachmann-Gagescu
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7175fb6cb31082cc299c7c39736754f0576dcb29" target='_blank'>
              Abstract 4662: Tracking gene expression in the tumor microenvironment throughout breast cancer progression
              </a>
            </td>
          <td>
            Ola Sayed Ahmed, heba Morsy, H. Ateya, Morkoss Medhat, Abdel Rahman Zekri
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb67a4446cf1d38e1dfaa6a69287fa69917e1c7" target='_blank'>
              Abstract 2735: The role of ca2 in regulating 3d chromatin structure in tamoxifen resistant breast cancer
              </a>
            </td>
          <td>
            Lavanya Choppavarapu, Kun Fang, Tianxiang Liu, A. Ohihoin, Victor X. Jin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30bcf504297ff27c6be8dd475bedf1618228bd60" target='_blank'>
              Abstract 5052: Large-scale genomic analysis of mutation signatures reveals pathway alternations linked to breast cancer prognosis
              </a>
            </td>
          <td>
            Máté Posta, Balázs Győrffy
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c26f8d443588f8353e49604dbb91d4086b38223b" target='_blank'>
              High GLP1R gene copy numbers associated with microsatellite instability for multiple cancers and with better survival probabilities for glioblastoma.
              </a>
            </td>
          <td>
            Teresa M Thomas, Michael T Aboujaoude, Vayda R. Barker, Mallika Varkhedi, Rahul Jain, G. Blanck
          </td>
          <td>2025-04-10</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Missense mutations in the BCR::ABL1 kinase domain are found in approximately 12–80% of patients with chronic myeloid leukemia (CML). Clinically significant mutations include T315I, M244V, Y253H/F, E255K/V, V299L, and F359V. The aim of this study was to create a diagnostic system for rapid and inexpensive detection of the above mutations. We used genomic DNA and RNA from peripheral blood and bone marrow cells of 57 patients with a Ph-positive CML diagnosis established in the chronic phase. We have developed a method to detect mutations in the BCR::ABL1 gene based on allele-specific real-time polymerase chain reaction (AS-PCR). In parallel, we analyzed the RNA sequence of the protein kinase domain of the same samples by next-generation sequencing (NGS) covering the points of putative mutations. In this work, we compared the results obtained by both methods for mutation detection and variant allele frequency (VAF) estimation of mutated vs. normal alleles. The sensitivity and specificity of our diagnostic system were also evaluated. It was found that AS-PCR gives reliable results at VAF up to 0.01%. AS-PCR has high sensitivity and may serve as an alternative for the more time-consuming NGS in some cases, as well as for monitoring CML treatment and for analyzing archival material.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc35b97b7ff1497c87471f2609685fa153e61f6d" target='_blank'>
              Allele-Specific PCR for Detection of Missense Mutations in the Chimeric BCR::ABL1 Gene Causing Failure of Tyrosine Kinase Inhibitor Therapy in CML Patients
              </a>
            </td>
          <td>
            Anastasia Skripkina, I. Fevraleva, E. Kuzmina, B. Biderman, E. Stepanova, E. Chelysheva, Anna Turkina, A. Sudarikov
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Epigenetics and epigenomics are captivating fields of molecular biology, dedicated to the exploration of heritable alterations in gene expression and cellular phenotypes, which transpire devoid of any discernible modifications to the fundamental DNA sequence. This intricate regulatory apparatus encompasses multiple mechanisms, prominently featuring DNA methylation, histone modifications, and the involvement of non-coding RNA molecules in pivotal roles. To achieve a comprehensive grasp of these diverse mechanisms, it is imperative to conduct research employing animal models as proxies for human studies. Since experimental animal models like mice and rats struggle to replicate the diverse environmental conditions experienced by humans, this review focuses on comparing common epigenetic alterations in naturally occurring tumors in canine models, which share the human environment, with those in humans. Through this, we emphasize the importance of an epigenetic regulation in the comparative medical approach to a deeper understanding of cancers and further development of cancer treatments. Additionally, we elucidate epigenetic modifications pertinent to specific developmental stages, the ageing process, and the progression of various diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bfc3c89b9e9910bb56cc9d2d52428ce2d674ca" target='_blank'>
              Comparative epigenomics to clinical trials in human breast cancer and canine mammary tumor
              </a>
            </td>
          <td>
            Su-Jin Jeong, Kang-Hoon Lee, Je-Yoel Cho
          </td>
          <td>2025-03-19</td>
          <td>Animal Cells and Systems</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b960e7fd1bc9cd524e9bf3da8cda4cbc23e6133e" target='_blank'>
              Abstract 3800: Physiological nutrient levels reveal nucleotide salvage as a dependency in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            R. Elbashir, Keene L. Abbott, Ahmed Ali, Diya L. Ramesh, Michelle Wu, Brian T. Do, Anya Shevzov-Zebrun, Tenzin Kunchok, Millenia Waite, Wontaek Chung, Chelsea Zhang, S. Sivanand, Azucena Ramos, Jacob A. Hansen, Raphael Ferreira, Alexander Muir, Michael H. Hemann, M. V. Vander Heiden
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea10556f7e26587dbcdc0c9ad631944d5c7f3c43" target='_blank'>
              Abstract 3824: Investigating the role of allelic imbalance in pancreatic cancer aggressiveness, survival, and therapeutic resistance
              </a>
            </td>
          <td>
            Enrico Gurreri, Li Zhang, Hania Khan, Lisa Maria Mustachio, Ningping Feng, T. Heffernan, Joseph R Marszalek, Basturk Olca, David Klimstra, Stefano Sioletic, A. Sgambato, G. Tortora, Giulio F. Draetta, Luigi Perelli, G. Genovese
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790c5332ba21df7288f6cc769801a6e13199e6ab" target='_blank'>
              Evolution of robust cell differentiation mechanisms under epigenetic feedback
              </a>
            </td>
          <td>
            Davey Plugers, Kunihiko Kaneko
          </td>
          <td>2025-03-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed706044b2c945f86bdba6c75c9b7f8653518a59" target='_blank'>
              Abstract 3068: The clonal nature of FGFR3-TACC3 in glioma: Mechanism discovery and preclinical study
              </a>
            </td>
          <td>
            Yiming Li, Tao Li, Xiude Ren, Jianshen Liang, Jikang Fan, Xisen Wang, Sheng-ping Yu, Wei Zhang, Xuejun Yang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67f7c0a88fb3654ac3464cd672c3b86897a3782a" target='_blank'>
              KPT-330, A New Candidate Drug for Targeting NOTCH1 Overexpression in T-cell Acute Lymphoblastic Leukemia, An In Vitro and Silico Study.
              </a>
            </td>
          <td>
            Fariha Naz, Saima Ejaz, Tayyaba Wali, Atiqa Nudrat, Mian Abdur Rehaman Arif, Muhammad Qandeel Waheed, Fatma Hussain, Saadiya Zia
          </td>
          <td>2025-04-11</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c687713915048ef2b6e851768a7161399616a610" target='_blank'>
              Abstract 6416: Molecular dynamics driving phenotypic divergence among KRAS mutants in pancreatic tumorigenesis
              </a>
            </td>
          <td>
            David J. Falvo, A. Grimont, R. Chandwani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3217dae134c4119cc94589ae992593aeb93f60c2" target='_blank'>
              KRAS mutations in colorectal cancer: impacts on tumor microenvironment and therapeutic implications.
              </a>
            </td>
          <td>
            Anita Emami, Pouya Mahdavi Sharif, Nima Rezaei
          </td>
          <td>2025-05-04</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17eeae7cebe3a01a8802644a89b867255b13f516" target='_blank'>
              Abstract 7043: HMGA1 Epigenetic Regulators Drive Relapse in Pediatric B-Cell Leukemia By Amplifying ETV5 and Stemness Networks
              </a>
            </td>
          <td>
            Linda S. Resar, Jung-Hyun Kim, Andrei Thomas-Tikonenko, S. Tasian
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03e8bbeec84df302b9adf6fafec7b5584f0a9749" target='_blank'>
              Advancement of Gene Therapy in Cancer
              </a>
            </td>
          <td>
            Kahkashan Sultana, Ayesha Fatima
          </td>
          <td>2025-03-15</td>
          <td>Journal of Advanced Scientific Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal alterations are frequent events in lung cancer progression. Although gains and losses of chromosomal position have been reported, the association between copy number alteration and lung cancer patient survival has not been extensively investigated. In this study, we performed a meta-analysis of public cBioPortal datasets spanning 25 lung cancer studies to identify putative cancer driver genes with copy number alterations associated with overall patient survival. Ten copy-number altered genes enriched in deceased lung cancer patients were identified. Seven of these putative driver genes were located in the 7p11.2 chromosomal location, and two were in the 9p21.3 cytoband. Among these genes, the mitochondrial ribosomal protein S17 (MRPS17) amplification was significantly associated with a lower patient survival rate (P = 1.47e-7). To investigate the functional role of MRPS17, small interfering RNA-mediated knockdown was performed in two non-small cell lung cancer cell lines, A549 and NCI-H460. MRPS17 knockdown significantly reduced cell proliferation, migration, invasion, and anchorage-independent growth in both cell lines. Furthermore, knockdown of MRPS17 decreased the activation of the phosphatidylinositol 3-kinase/protein kinase B signaling pathway, suggesting its role in driving lung cancer progression through this critical oncogenic pathway. Our findings highlight MRPS17 as a potential cancer therapy target and a prognostic biomarker that may improve the survival rates of lung cancer patients. Future studies should explore its inhibition as a therapeutic strategy as well as elucidate its molecular mechanisms in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1c0b37271452f619e3684f3550b14101ddebda" target='_blank'>
              Mitochondrial Ribosomal Protein S17 Silencing Inhibits Proliferation and Invasiveness of Lung Cancer Cells
              </a>
            </td>
          <td>
            Woo Rin Lee, Kook Sun Ha
          </td>
          <td>2025-03-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/accef4dd1efc826cc2507a7542b8ca2e3603156d" target='_blank'>
              Cell-type-specific transposable element demethylation and TAD remodeling in the aging mouse brain
              </a>
            </td>
          <td>
            Qiurui Zeng, Wei Tian, Amit Klein, Anna Bartlett, Hanqing Liu, Joseph R. Nery, R. Castanon, Julia K. Osteen, Nicholas D. Johnson, Wenliang Wang, Wubin Ding, Huaming Chen, Jordan Altshul, M. Kenworthy, Cynthia Valadon, William Owens, Zhanghao Wu, Maria Luisa Amaral, Yuru Song, Cindy Tatiana Báez-Becerra, Silvia Cho, Chumo Chen, Jackson Willier, Stella Cao, Jon Rink, Jasper Lee, Ariana Barcoma, Jessica Arzavala, Nora Emerson, Yuancheng Ryan Lu, Bing Ren, M. Behrens, J. Ecker
          </td>
          <td>2025-04-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa91244e52369ba86bd81040f0cf520d88eb9d9f" target='_blank'>
              Expansion of a bacterial operon during cancer treatment ameliorates fluoropyrimidine toxicity.
              </a>
            </td>
          <td>
            Kai R Trepka, Wesley A Kidder, Than S. Kyaw, Taylor Halsey, Christine A Olson, Edwin F Ortega, Cecilia Noecker, Vaibhav Upadhyay, Dalila Stanfield, Paige Steiding, Benjamin G. H. Guthrie, Peter Spanogiannopoulos, Darren Dumlao, Jessie A. Turnbaugh, Matthew D. Stachler, Erin L. Van Blarigan, A. Venook, C. Atreya, P. Turnbaugh
          </td>
          <td>2025-04-16</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77cbbc2ef4270ebb636894a741346e8433adaa6" target='_blank'>
              Abstract 2552: TET2-mutant clonal hematopoiesis enrichment in the tumor microenvironment is an actionable driver of treatment resistance in solid tumors: Thyroid cancer as a paradigm
              </a>
            </td>
          <td>
            Pablo Sánchez Vela, Vera Tiedje, Julie L. Yang, B. Untch, Laura Boucai, Aaron J Stonestrom, Alberto Bueno Costa, Sebastià Franch Expósito, Avi Srivastava, M. Kerpelev, Jillian Greenberg, Matthew Wereski, Amanda Kulick, Kevin Chen, Tianyue Qin, Soo-Yeon Im, Anthony R Martinez Benitez, Raquel Pluvinet, Merve Sahin, K. Menghrajani, Gnana P. Krishnamoorthy, E. de Stanchina, A. Zehir, Rahul Satija, J. Knauf, R. Bowman, M. Esteller, Sean M Devlin, Michael F. Berger, R. Koche, Ross L. Levine, James A. Fagin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e04034cb839d11700bfc365e9345aa1519c5848" target='_blank'>
              Abstract 6712: Epigenetic reactivation of a tumor suppressor program in leukemia
              </a>
            </td>
          <td>
            Eric Wang, Cuijuan Han, Alexander Calderon, I. Aifantis, A. Tsirigos, Abimbo Lawal, S. H. Rajendran, Sadik Karma, Hussein A. Abbas, M. Y. Yassouf, Zhiping Zhang, Pedro Miura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Objectives/Goals: Large-scale tumor sequencing efforts have led to annotations of novel cancer hotspot mutations that may underlie driver or cooperative function. We have sought to define the molecular consequences of such hotspots associated with pediatric DICER1 syndrome cancers, with the ultimate goal of revealing novel targets that may inform new standards of care. Methods/Study Population: We have performed genomic analysis to identify tumor types (in TCGA and MSK-IMPACT patient data) for which mutations in the Dicer1 gene (encoding Dicer protein) emerge as the dominant signature of driver function. As Dicer is a critical RNA processing factor responsible for the generation of microRNAs, which are posttranscriptional gene regulatory molecules, we have modeled these mutations in human embryonic stem cells in order to study the direct effects on miRNAs and their target genes in an isogenic background. In addition to providing the required setting for unambiguous attribution of function to specific mutations, clonal human ES cells offer an opportunity for modeling of both developmental and cancer requirements associated with altered Dicer function. Results/Anticipated Results: Through generation of genomics and functional datasets from matched genotypes in Dicer mutated human ES cells, we have identified specific alterations in miRNAs and their effects on target genes. Unexpectedly, we found direct evidence for both loss of function and gain of function attributable to Dicer mutations. In addition, through integrated analysis of genomic data from tumor sequencing datasets and our human ES cell models, we have identified potential miRNA and target gene alterations that underlie tumorigenic potential, nominating gene candidates for targeted therapy in DICER1 syndrome. Direct mouse modeling of such candidate gene targets has revealed evidence for driver function of identified miRNA and their targets. Discussion/Significance of Impact: DICER1 syndrome cancers comprise a wide variety of rare pediatric tumor types. Presently, we still lack an effective standard of care. Furthermore, the previous lack of molecular profiling precluded targeted therapy opportunities. Our precise knock-in modeling of Dicer hotspots and deep profiling of relevant tumors now provide candidate targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597fce8f39319f3d39a09a21bf59ed3021208522" target='_blank'>
              518 Modeling of cancer mutations found in pediatric DICER1 syndrome informs novel therapeutic targeting strategies
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert W. Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9124f513a579068358157bc6335b2150cb3dde2" target='_blank'>
              Targeting processive transcription for Myc-driven circuitry in medulloblastoma
              </a>
            </td>
          <td>
            Lays Martin Sobral, Faye Walker, Krishnakumar Madhavan, Elizabeth Janko, Sahiti Donthula, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Mary M. Haag, Billie J. Carstens, Natalie J Serkova, Nicholas K Foreman, S. Venkataraman, Bethany L. Veo, Rajeev Vibhakar, Nathan A Dahl
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06e5c087c7af55ddb110acf4864affc68be63a67" target='_blank'>
              Abstract 4065: Unveiling drivers of esophageal adenocarcinoma with CRISPR pooled screen and single-cell RNA-sequencing
              </a>
            </td>
          <td>
            Ebtihal H. Mustafa, Julia V. Milne, Ka Meng Wu, Katherine Papastratos, N. Thio, Wayne Phillips, Nicholas Clemons
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235dd3b879fd0da050d8e59db442b8291733b6d" target='_blank'>
              Abstract 2391: Comprehensive quantification and annotation of viral gene integration and expression across thousands of human cancer models
              </a>
            </td>
          <td>
            Sowmya Iyer, Moore van Tienen, Phillipp Trollman, Yao He, William R. Sellers, Catarina D. Campbell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd321a5ebdb5267d14caeaf74b23869909c920a" target='_blank'>
              Restoration of tumor suppressor protein with enhanced activity using engineered tRNAs to induce tumor regression
              </a>
            </td>
          <td>
            Bangyan Kong, Zhetao Zheng, Zeyuan Mi, Zhiyang Dou, Yuelin Yang, Yuxuan Shen, Lian-Hua Dong, Jingru Lv, Ying Li, Qing Liang, Songlin Tian, Jiayu Wang, Hua Qiao, Qing Xia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fc3716db0c11759429e639a63e62e0f3409b05" target='_blank'>
              Abstract 5088: Single cell-based elucidation and pharmacologic targeting of master regulator dependencies in small cell lung cancer (SCLC) subpopulations
              </a>
            </td>
          <td>
            Lucas ZhongMing Hu, Anish Thomas, Andrea Califano
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Coupling single-cell transcriptomics with tumor-initiating cell enrichment identified IFN response gene expression not previously reported in bulk RNA-sequencing–derived signatures and proposed IFN/STAT1 signaling as a candidate therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49032c731c7f43ad3396d95549c8d805e477fd3" target='_blank'>
              Lineage Tracing and Single-Cell RNA Sequencing Reveal a Common Transcriptional State in Breast Cancer Tumor–Initiating Cells Characterized by IFN/STAT1 Activity
              </a>
            </td>
          <td>
            Eric P Souto, Ping Gong, John D. Landua, R. Srinivasan, Abhinaya Ganesan, L. Dobrolecki, S. C. Purdy, Xingxin Pan, Michael Zeosky, Anna Chung, S. S. Yi, H. Ford, Michael T. Lewis, Connie Cepko, Dr. Kenneth L. Scott, Dr. Michael Davidson
          </td>
          <td>2025-02-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The mouse genome is transcribed at different rates in both directions from the newly formed genome after fertilization. During embryonic genomic activation (EGA/ZGA), the first RNA metabolism creates heterogeneity between blastomeres. Indeed, ZGA-dependent maternal RNA degradation is crucial to regulate gene expression and enable the initiation and acquisition of full developmental competence. Subsequently, from the new genome, in addition to mRNAs, a wide range of regulatory ncRNAs are also transcribed. Regulatory ncRNAs (non-coding RNAs) have profoundly influenced fields ranging from developmental biology to RNA-mediated non-Mendelian inheritance, exhibiting sequence-specific functions. To date, the database cataloging ncRNA is not exhaustive, but their high sequence diversity, length and low expression level can vary within the same genome depending on environmental conditions, making understanding their functions often ambiguous. Indeed, during transcription control, cellular RNA content varies continuously. This phenomenon is observed in genetically identical organisms studied—bacteria, flies, plants and mammals—due to changes in transcription rates, and therefore, it impacts cellular memory. Importantly, experimental data regarding the simple modification of RNAs levels by microinjection into fertilized mouse eggs suggest that they certainly play a driving role in establishing and transmitting newly formed expression information. The idea here is that, even in a stable genome, transcripts can vary rapidly and significantly in response to environmental changes, initiated by transcriptional variations in the genome, thus altering cellular memory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768d936b723e7adb7dcc685d117648dc4b4eb5c" target='_blank'>
              RNA-Mediated Non-Mendelian Inheritance in Mice: The Power of Memory
              </a>
            </td>
          <td>
            M. Rassoulzadegan
          </td>
          <td>2025-04-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [2, 5],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>