<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-23 09:31:30 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>43</td>
          <td>43</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Cancer cells that survive therapeutic drug pressure are a significant cause of disease relapse and progression, impeding curative cancer treatment. Drug-triggered Darwinian selection and the emergence of subclones harbouring specific mutations that confer resistance have been well documented and extensively studied. However, these genetic alterations, while important, do not fully explain clinical observations where some patients, after a drug holiday, regain sensitivity to the same treatment despite previous disease progression. This phenomenon highlights the possibility that drug resistance may not solely rely on genetic mutations but could also involve reversible, non-genetic mechanisms. Recent studies have highlighted the existence of drug-tolerant persister cells (DTPs), a subpopulation of cancer cells that can survive short-term therapeutic pressure without acquiring resistance-associated genetic alterations. These cells exhibit a temporary yet reversible tolerance to the initial treatment while also acquiring cross-tolerance to other anti-cancer therapies. The presence of DTPs underscores a dynamic and complex plasticity in tumours, wherein cancer cells can utilise epigenetic rewiring, metabolic reprogramming, and specific signalling pathways to transit between drug-tolerant and drug-sensitive states to adapt to environmental pressures. Furthermore, this adaptive resilience enables DTPs to act as a reservoir for the development of genetically stable resistance, resulting in cancer therapy failure and eventual relapse. In this mini-review, we examine recent evidence on DTPs to provide an overview of their characteristics, development, and survival mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f885f194fb8254e0480e9608eccdddbd6818a86e" target='_blank'>
              Targeting Cancer Drug-Tolerant Persister Cells in Minimal Residual Disease
              </a>
            </td>
          <td>
            Wei Zhang, Sorayut Chattrakarn, Fuhui Chen, Haobin Chai, Mariana Maranga, Jingwei Zhang
          </td>
          <td>2025-05-27</td>
          <td>International Journal of Drug Discovery and Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e19dad7b139ef16ca0124619b3b70027a193368b" target='_blank'>
              Uncovering Convergent Cell State Dynamics Across Divergent Genetic Perturbations Through Single-Cell High-Content CRISPR Screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Andrew Liao, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-05-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, Muhammad T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/998e8a8abe04f7f2be63264d103e59a69474a153" target='_blank'>
              Adaptation by stochastic tuning of gene expression in mammalian cells
              </a>
            </td>
          <td>
            Amir Momen-Roknabadi, P. Oikonomou, Saeed Tavazoie
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e2397ca3adf8845c9715ac4636c6b9da727b026" target='_blank'>
              Epigenetic dynamics and molecular mechanisms in oncogenesis, tumor progression, and therapy resistance.
              </a>
            </td>
          <td>
            Mohammed Kaleem, Lubna Azmi, N. Shahzad, Murtada Taha, Shiv Kumar, Md. Ali Mujtaba, Abdulaziz Ali H Hazazi, A. Kayali
          </td>
          <td>2025-05-13</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Melanoma, a highly aggressive form of skin cancer, presents a significant clinical challenge. The identification of BRAFV600E mutations as a driver in roughly half of all melanoma cases has led to the development of targeted therapies, exemplified by Vemurafenib (Vem), which inhibits overactivated MAPK signaling. However, frequent relapses remain a major obstacle. To investigate the complex mechanisms driving drug resistance in melanoma, we developed PerturbFate, a scalable single-cell combinatorial-indexing strategy that integrates multi-modal profiling with CRISPR perturbations. This approach simultaneously captures newly synthesized and pre-existing transcriptomes, chromatin accessibility, and CRISPR sgRNA identity from individual cells. Using this platform, we profiled BRAFV600E melanoma cells (A375) after knocking down each of 143 potential drug resistance candidate genes, both with and without Vem treatment, in a pooled manner. We identified extensive phenotypic state transitions in melanoma cells under genetic and chemical perturbations, revealing that an undifferentiated state is associated with enhanced drug resistance, while a more benign, Vem-responsive melanocytic state exists at the opposite end of the phenotypic spectrum. Temporal gene regulatory network analyses revealed that AP-1 transcription factors (TFs), particularly FOSL1, pioneer other diverse TFs to drive the melanoma dedifferentiation. In contrast, SOX10 is the key determinant of the melanocytic state. We identified MAPK pathway, which primarily promotes proliferation, and Hippo/YAP signaling, which predominantly drives the undifferentiated state transition, as convergent key pathways mediating drug resistance. Their synergistic interplay, executed through the collaborative activation of downstream TFs (e.g., FOSL1, RREB1, SMADs, KLF5)-mediated gene expression, is a critical driver of Vem resistance. To elucidate the mechanisms underlying the observation that perturbation of genes encoding the Mediator Complex components (e.g., MED12, CCNC, MED13, MED19, MED15, MED24) induced autonomous dedifferentiation and/or resistance to Vem in melanoma, we established linkages between their distinct biochemical properties and heterogeneous perturbation responses. We also identified the convergence of dysregulated inflammation, potentially mediated by VEGFC and interleukins (ILs), with Hippo/YAP-mediated transcriptional responses in driving melanoma dedifferentiation and Vem resistance following perturbation of mediator complex genes. Altogether, our study introduces a novel single-cell multi-modal CRISPR perturbation platform, which we applied to comprehensively chart the responses of melanoma cells to the perturbation of all potential Vem resistance candidate genes. These findings pave the way for the identification of precise targets for combination therapies with enhanced efficacy against melanoma drug resistance.


 Zihan Xu, Ziyu Lu, Aileen Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao. Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Karyotyping, SNP arrays, fluorescence in situ hybridisation and next generation sequencing techniques have greatly improved our understanding of the genetic drivers of acute lymphoblastic leukemia (ALL). In the past years, another layer of genetic data has been added by the study of mutational signatures, patterns of somatic mutations that represent specific mutational mechanisms. Mutational signatures can give insight in tumor development, but also reveal mutagenic side-effects of treatment in relapse samples. Multiple treatment-related mutational signatures have been detected in relapsed ALL that could play a role in therapy resistance and relapse development. In fact, multiple pathogenic driver mutations have been attributed to these treatment-related mutational processes, including a recurrent TP53 mutation. Studies in childhood ALL revealed that thiopurine exposure is the most common source of therapy-related mutagenicity in ALL and presents differently when patients are DNA mismatch repair deficient. Thiopurine-induced DNA damage indicates that leukemic cells were able to survive thiopurine exposure. This could be due to metabolic defects, acquired mutations that induce thiopurine resistance during treatment or resistance to drugs synergizing with thiopurines. In this review, we discuss the types and prevalence of treatment-related mutational signatures in ALL, and explore mechanisms of thiopurine cytotoxicity and mutagenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f4deac881b3b7a4007820d8b07d9cc82f4007b" target='_blank'>
              Treatment-related mutagenic processes in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Cédric G van der Ham, F. V. van Leeuwen, Roland P Kuiper
          </td>
          <td>2025-05-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Osimertinib, a third-generation EGFR tyrosine kinase inhibitor (EGFR-TKI), has dramatically transformed the treatment landscape for patients with EGFR-mutant NSCLC. However, the long-term success of this therapy is often compromised by the onset of acquired resistance, with non-genetic mechanisms increasingly recognized as pivotal contributors. Here, we exploit a multi-omics approach to profile genome-wide chromatin accessibility and transcriptional landscapes between drug sensitive and resistant EGFR-mutant cells. Our findings reveal a robust concordance between epigenetic regulome and transcriptomic changes that characterize the osimertinib resistant state. Through CRISPR-based functional genomics screen targeting epigenetic regulators and transcription factors, we uncover a critical regulatory network featuring key members of the NuRD and PRC2 complexes that mediate resistance. Most critically, our screen identifies FOSL1 and JUN, two subunits of the AP-1 transcription factor within this network, as the most significant hits. Mechanistically, we demonstrate that cis-regulatory elements exhibiting altered chromatin accessibility in the resistant state are enriched for cognate AP-1 binding motifs, enabling AP-1 to orchestrate a gene expression program that underpins the druggable MEK/ERK signaling axis, potentially enhancing cell viability and fitness of resistant cells. Importantly, genetic depletion or pharmacological inhibition of AP-1 reinstates cellular and molecular sensitivity, and reverts resistance-associated phenotypes, such as epithelial-to-mesenchymal transition, upon anti-EGFR rechallenge by repressing AKT and ERK signaling. These findings provide novel insights into the epigenetic and transcriptional control of osimertinib resistance in EGFR-mutant NSCLC, highlighting AP-1 as a targetable vulnerability of resistance-related hallmarks and offering a promising avenue for developing resistance reversal strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b98b05c64d117a31d26404e164fc0fbed4525adc" target='_blank'>
              Integrative multi-omics identifies AP-1 transcription factor as a targetable mediator of acquired osimertinib resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            B. Dayanc, Sude Eris, Nazife Ege Gulfirat, Gulden Ozden-Yilmaz, E. Cakiroglu, Ozlem Silan Coskun Deniz, G. Karakülah, Serap Erkek-Ozhan, Serif Senturk
          </td>
          <td>2025-05-25</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83b5ca4a7a4ab998ed122fece632645d240cae73" target='_blank'>
              Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics
              </a>
            </td>
          <td>
            Manu Saraswat, Laura Rueda-Gensini, Elisa Heinzelmann, Tannia Gracia, Fani Memi, Grant de Jong, Jannes Straub, Cedar Schloo, D. Hoffmann, E. Jung, Tim Kindinger, Bettina Weigel, Bryce Lim, S. Weil, Oliver Gould, Richard Mair, Katharina Mikulik, Martin Rohbeck, W. Wick, F. Winkler, O. Bayraktar, O. Stegle, Moritz Mall
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/503cca6deb2a91fe468e2c781a6fa64738c62b22" target='_blank'>
              Drug resistance and tumor heterogeneity: cells and ensembles
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-05-31</td>
          <td>Biophysical Reviews</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="EGFR-mutant lung adenocarcinoma (LUAD) represents 20% of all non-small cell lung carcinomas, with most patients presenting with incurable metastatic disease. Treatment with mutant-selective EGFR tyrosine kinase inhibitors (TKIs) results in initial tumor reduction, yet nearly all patients eventually relapse. The mechanisms driving drug resistance are incompletely understood, creating significant barriers to curing metastatic disease. We integrated clinical genomic and single-nuclei RNA (snRNA) sequencing from a cohort of 62 EGFR-mutant LUAD patients treated with the third generation EGFR TKI, osimertinib, and compared treatment-naïve (TN), minimal residual disease (MRD), and progressive disease (PD) tumors. We found that disease progression is associated with a marked decrease in alveolar lineage fidelity, coincident with reduced MAPK signaling and adenocarcinoma identity. PD tumors with sustained MAPK pathway activity, such as those with EGFR or MET amplifications, tended to retain adenocarcinoma identity. In contrast, MAPK-low tumors were more likely to undergo histological transformation to squamous or neuroendocrine lineages. Remarkably, we observed rare tumor cell populations prior to treatment that were poorly differentiated, in some cases with neuroendocrine or squamous features. At progression, these histologically divergent tumor cells increased in prevalence, both in cases with overt histological transformation, and in others with sub-clinical histological plasticity. These findings suggest that pre-existing capacity for histologic plasticity may be a substrate for therapy induced selection. Taken together, our results illuminate genomically encoded MAPK signaling and lineage plasticity as complementary mechanisms of acquired resistance to EGFR TKI in lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd63209ef4aad2643f26aad61884d9a8966d7bd4" target='_blank'>
              Complementary modes of resistance to EGFR TKI in lung adenocarcinoma through MAPK activation and cellular plasticity
              </a>
            </td>
          <td>
            Matthew Zatzman, Á. Quintanal-Villalonga, Sohrab Salehi, Nicholas Ceglia, J. Lee, Amanda Pupo, C. Falcon, Nicole Rusk, I. Masilionis, O. Chaudhary, P. Manoj, R. Chaligné, Andrew McPherson, C. Rudin, Sohrab P. Shah, H. Yu
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Newcastle disease virus (NDV) is a promising oncolytic agent with a non-segmented, negative-sense single-stranded RNA (ssRNA) genome of approximately 15 kb. While NDV selectively replicates in and lyses a wide range of human cancer cells, a subset of these cells develop persistent infections, potentially compromising the therapeutic efficacy of NDV-based treatments. To investigate the molecular basis of persistent infection, we performed transcriptome profiling of TCCSUP bladder cancer cells persistently infected with the NDV AF2240 strain. Deep sequencing using Illumina HiSeq 2000 was conducted in triplicate, and the resulting viral and host reads were separated and analyzed. Using Integrative Genomic Viewer (IGV) software, we identified several nucleotide variants linked to persistence. Specifically, nucleotide alterations included a deletion at 359A→C and a substitution at 1653C→T within the nucleoprotein (NP) gene, as well as an insertion at 3338C→T in the matrix protein (M) gene. Additionally, a GGG base insertion was detected at position 2290 in the phosphoprotein (P) gene. Crucially, we observed truncations in the hemagglutinin-neuraminidase (HN) gene (nt 8263–8390) and the large polymerase (L) gene (nt 6203–6342). These mutations and truncations suggest significant disruptions in viral replication, assembly, and host cell attachment, potentially facilitating viral persistence in bladder cancer cells. Understanding these genomic alterations provides valuable insights into the mechanisms driving viral persistence and could inform strategies to optimize the oncolytic efficacy of NDV in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46c1e75575001a56bcabf1b3e8302abfbe59f19d" target='_blank'>
              Genomic alterations in persistently infecting oncolytic Newcastle disease virus reveal mechanisms of viral persistence in bladder cancer cells
              </a>
            </td>
          <td>
            U. Ahmad, D. Chau, SC Chan, K. Yusoff, A. Veerakumarasivam, S. Abdullah
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="
 Both intrinsic and acquired resistance limit the depth and duration of response to all clinically evaluated anti-KRAS therapies. Genetic resistance mechanisms, identified in nearly half of relapsed patients, include acquisition of mutations that effectively restore KRAS-dependent activities. Reported non-genetic mechanisms include signaling and transcriptional activities that support epithelial-to-mesenchymal transition (EMT) or transition to cancer subtypes less responsive to KRAS inhibition. We recently reported MYC amplification and TEAD-dependent transcriptional activation by YAP as drivers of resistance to the multi-RAS inhibitor RMC-7977 (RASmulti), the preclinical analog of RMC-6236. Importantly, we showed that cells resistant to RASmulti that harbor amplified MYC could be re-sensitized to RAS inhibition by co-treatment with TEAD inhibitors. These studies motivated us to define the MYC- and TEAD-dependent transcriptional profiles in KRAS-mutant pancreatic adenocarcinoma (PDAC) cells. We knocked down MYC using siRNA in a panel of eight human PDAC cell lines to derive the MYC-dependent transcriptome. We found 3,010 MYC-dependent genes, 36% of which support oncogenic KRAS signaling through ERK. To determine the TEAD-dependent transcriptome, we treated a panel of three sets of paired naïve and RASmulti-resistant KPC cell lines, derived from an autochthonous mouse model of PDAC, and of six human KRAS-mutant PDAC cell lines with the TEAD inhibitor IAG933, with RMC-7977, or with the combination. We found 3,865 TEAD-dependent genes in mouse cells and 3,590 TEAD-dependent genes in human cells. We found co-regulated RAS, MYC, and TEAD gene transcriptional networks enriched for essential genes that support cell cycle progression. We also found distinct gene clusters regulated by each protein that support distinct molecular functions, such as protein biosynthesis, RHO GTPases, and immune signaling. Importantly, combined inhibition of TEAD and RAS led to a more robust downregulation of essential genes than either single treatment. Proteomic profiling of PDAC cell lines with acquired resistance to RMC-7977 showed upregulation of TEAD-dependent gene products, supporting TEAD activation as a non-genetic mechanism of acquired resistance. Together, our findings expand the role of MYC and TEAD in supporting PDAC cell growth and resistance to RAS inhibition and provide rationale for further evaluation of combination therapeutic strategies to limit resistance.


 Brandon L. Mouery, Priya S. Hibshman, A Cole Edwards, Ryan D. Mouery, Runying Yang, Jeff A. Klomp, Angie L. Mordant, Thomas S. Webb, Aurora Cabrera, Laura E. Herring, Adrienne D. Cox, Timour Baslan, Channing J. Der, Clint A. Stalnecker. Determination of the MYC- and TEAD-dependent transcriptome in pancreatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB288.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0847dafec64111106a227406998f89b030109af3" target='_blank'>
              Abstract LB288: Determination of the MYC- and TEAD-dependent transcriptome in pancreatic cancer
              </a>
            </td>
          <td>
            Brandon L. Mouery, Priya S. Hibshman, A. C. Edwards, Ryan D. Mouery, Run-Wei Yang, Jeffrey A. Klomp, Angie L Mordant, Thomas S. Webb, Aurora Cabrera, Laura E. Herring, Adrienne D. Cox, T. Baslan, C. Der, Clint A. Stalnecker
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Locally advanced esophageal adenocarcinoma remains difficult to treat and the ecological and evolutionary dynamics responsible for resistance and recurrence are incompletely understood. Here, we performed longitudinal multiomic analysis of patients with esophageal adenocarcinoma in the MEMORI trial. Multi-region multi-timepoint whole-exome and paired transcriptome sequencing was performed on 27 patients before, during and after neoadjuvant treatment. We found major transcriptomic changes during treatment with upregulation of immune, stromal and oncogenic pathways. Genetic data revealed that clonal sweeps through treatment were rare. Imaging mass cytometry and T cell receptor sequencing revealed remodeling of the tumor microenvironment during treatment. The presence of genetic immune escape, a less-cytotoxic T cell phenotype and a lack of clonal T cell expansions were linked to poor treatment response. In summary, there were widespread transcriptional and environmental changes through treatment, with limited clonal replacement, suggestive of phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286e0cfb0095dd021b03614405167d3960808941" target='_blank'>
              Evolutionary and immune microenvironment dynamics during neoadjuvant treatment of esophageal adenocarcinoma
              </a>
            </td>
          <td>
            M. Barroux, J. Househam, E. Lakatos, T. Ronel, Ann-Marie Baker, Henrike Salié, M. Mossner, Kane Smith, C. Kimberley, S. Nowinski, Alison Berner, V. Gunasri, Martin Borgmann, S. Liffers, Marnix Jansen, G. Caravagna, K. Steiger, J. Slotta-Huspenina, Wilko Weichert, Luis Zapata, Eleftheria Giota, Sylvie Lorenzen, M. Alberstmeier, Benjamin M Chain, H. Friess, B. Bengsch, Roland M. Schmid, J. Siveke, M. Quante, Trevor A. Graham
          </td>
          <td>2025-05-14</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Next-generation sequencing technology enables uniform and impartial assessment of cancer diagnoses and prognosis. However, such studies are mostly type-specific, and capturing shared genomic abnormalities responsible for neoplastic transformation and progression is a challenging task. Pan-cancer analysis offers insights into the shared and unique molecular mechanisms driving cancer. We conducted an integrated gene-expression analysis using 10,629 samples from 30 distinct cancer types characterized by The Cancer Genome Atlas (TCGA). A gene co-expression network was constructed and genes overlapping between the selected modules and Differentially Expressed Genes (DEGs) were designated as genes of interest. Following a comprehensive literature review, ATP binding cassette subfamily A member 10 (ABCA10) and ATP binding cassette subfamily B member 5 (ABCB5) were selected as key candidates for downstream analysis due to the absence of systematic pan-cancer analysis of these genes. This study presents a unique contribution as the first comprehensive pan-cancer analysis of ABCA10 and ABCB5, highlighting their roles in tumor biology and clinical outcomes. We employed a variety of bioinformatics tools to explore the role of these genes across different tumors. Our research demonstrated that ABCA10 shows reduced expression, while ABCB5 displays variable expression patterns across tumors, indicating their opposing roles and flexible functions in pan-cancer. In many cancer patients, these expression patterns are correlated with worse survival outcomes. Furthermore, immunotherapy responses and immune infiltration across a variety of tumor types are associated with the expression levels of both ABCA10 and ABCB5. These results imply that ABCA10 and ABCB5 could serve as valuable predictive markers and potential therapeutic targets across various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f1be3a2286bbe9d4105e8c1625c6913bc7d7bf" target='_blank'>
              Integrated pan-cancer analysis revealed therapeutic targets in the ABC transporter protein family
              </a>
            </td>
          <td>
            M. B. E. Masood, Iqra Shafique, Muhammad Inam Rafique, Ayesha Iman, Ariba Abbasi, Mehak Rafiq, Uzma Habib
          </td>
          <td>2025-05-30</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Cancer progression and therapeutic resistance are propelled by the remarkable plasticity of signaling networks, which dynamically rewire under selective pressures to maintain proliferation, enable immune evasion and promote metastasis. Despite advances in precision oncology, the dynamic crosstalk between tumor cells, non-coding genomes and the microenvironment continues to undermine treatment efficacy. This call for submissions, Revolutionizing Cancer Treatment: Navigating the Intricate Landscape of Cellular Signaling Networks, seeks cutting-edge research that dissects these adaptive mechanisms through innovative technologies - from single-cell multi-omics and spatial transcriptomics to AI-powered network modeling. We welcome studies leveraging physiomimetic models (e.g., organoids, 3D-bioprinted ecosystems) to decode tumor heterogeneity, as well as translational work targeting emergent vulnerabilities at the intersection of epigenetics, metabolic reprogramming and stromal interactions. By integrating systems biology with computational and experimental approaches, this collection aims to catalyze the design of adaptive therapies that outmaneuver cancer's evolutionary resilience.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba13faeaf95516c4bb5cbd68aeea6df92e61f127" target='_blank'>
              Revolutionizing cancer treatment: Navigating the intricate landscape of cellular signaling networks.
              </a>
            </td>
          <td>
            Hao Zhang
          </td>
          <td>2025-05-26</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e20582


 Background:
 Despite advances in therapies, the development of drug resistance in non-small cell lung cancer (NSCLC) poses a significant challenge. The emergence of neoantigen-based therapies offer a promising new frontier which enhances treatment efficacy and reduces associated costs.
 Methods:
 A total of 24 patients diagnosed with NSCLC were enrolled in this study. Primary tumor tissues and peripheral blood samples were collected from each patient. Paired whole exome and transcriptomic sequencing data were performed to identify somatic mutations and to quantify their respective expression levels. Somatic single nucleotide variants (SNVs) and small insertions or deletions (inDels) were detected using Strelka algorithm from whole exome sequencing (WES) data. Neoantigen prediction focused on human leukocyte antigens (HLA) type 1. HLA alleles were predicted using WES data encoding adjacent non-cancerous tissue samples, identifying four alleles that were present in at least 50% of the patients. Neoantigens were considered potentially immunogenic if their predicted median IC50 binding scores were ≤500nM while expressed transcripts per million (TPM) exceeds 1 in tumor samples.
 Results:
 Using systematic bioinformatics pipeline, we analyzed 24 NSCLC samples with matched tumor tissues and peripheral blood, and detected the 4677 mutations involving 2442 genes, including 2819 missense SNVs, 941 InDels, 229 gene fusion and 688 RNA alternative splicing events, 94.7% of which (4428 mutations involving in 2247 genes) were common with those reported in the TCGA-LUAD dataset. The average number of mutations was 194.8 in NSCLCs. We found that C>T/G>A transitions/ transversions were dominant. Missense mutations were the most frequent types of somatic mutation in the coding sequence regions. 98% of mutated genes were not detected simultaneously with DNA and RNA mutations.
 ADCK5
 ,
 CD151
 and
 EIF3K
 were identified as potential new driver genes. Genomic variation profiling identified 2213 potential neoantigens with a median IC50 binding score of ≤500nM and a transcript expression level of >1 TPM. Among these, 41 were listed in databases TSNAdb, IEDB, and CTDatabase. A total of 14 genes harboring identical mutations were detected, with the identical mutations observed in over 50% of the patient cohort. Notably, mutations in 14 genes were either gene fusions or RNA alternative splicing events, and core antigenic sequences generated were found to be identical in the majority of patients. Of particular interest is the RPRD2 gene, where an RNA exitrons site was detected in all patient samples, along with the identification of the same neoantigen core sequence.
 Conclusions:
 Our analysis suggests that gene fusions and RNA splicing variations are more likely than point mutations to generate shared antigens, underscoring their importance for developing targeted immunotherapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1725e99e0357b9a0ce8d33b25d7c68d81214eed0" target='_blank'>
              Comprehensive genomic alterations identification and potential shared neoantigens for non-small cell lung cancer immunotherapy.
              </a>
            </td>
          <td>
            Muchun Zhu, Qing Zhao Ruan, Boyue Zhang, Weiqiang Yin, Run Sheng Ruan
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Therapy-induced senescence (TIS) is considered a permanent cell cycle arrest following DNA-damaging treatments; however, its irreversibility has recently been challenged. Here, we demonstrate that escape from TIS is universal across breast cancer cells. Moreover, TIS provides a reversible drug resistance mechanism that ensures the survival of the population, and could contribute to relapse. Methods TIS was induced in four different breast cancer cell line with high-dose chemotherapy and cultured until cells escaped TIS. Parental, TIS and repopulating cells were analyzed by bulk and single-cell RNA sequencing and surface proteomics. A genetically engineered mouse model of triple-negative breast cancer was used to prove why current senolytics cannot overcome TIS in tumors. Results Screening the toxicity of a diverse panel of FDA-approved anticancer drugs revealed that TIS meditates resistance to half of these compounds, despite their distinct mechanism of action. Bulk and single-cell RNA sequencing, along with surface proteome analysis, showed that while parental and repopulating cells are almost identical, TIS cells are significantly different from both, highlighting their transient nature. Furthermore, investigating dozens of known drug resistance mechanisms offered no explanation for this unique drug resistance pattern. Additionally, TIS cells expressed a gene set associated with immune evasion and a potential KRAS-driven escape mechanism from TIS. Conclusion Our results reveal that TIS, as a transient drug resistance mechanism, could contribute to overcome the immune response and to relapse by reverting to a proliferative stage. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02310-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9838695afba129f01c7b6f860f5a9601961169be" target='_blank'>
              Therapy-induced senescence is a transient drug resistance mechanism in breast cancer
              </a>
            </td>
          <td>
            Eszter Bajtai, Csaba Kiss, Éva Bakos, Tamás Langó, Anna Lovrics, É. Schád, V. Tisza, Károly Hegedűs, Péter Fürjes, Zoltán Szabó, Gábor E. Tusnády, G. Szakács, Á. Tantos, S. Spisák, J. Tóvári, András Füredi
          </td>
          <td>2025-05-01</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5321242314d73047e5ba0494889b2b9e2dbad68d" target='_blank'>
              Drug tolerance and persistence to EGFR inhibitor treatment are mediated by an ILK-SFK-YAP signaling axis in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Rocky Shi, D. Farnsworth, C.A. Febres-Aldana, Justine L M Chow, R. Sheena, T. Atwal, Juan Luis Gomez Marti, Samantha Li, Kiersten N. Thomas, Che-Min Lee, S. Awrey, P. McDonald, R. Somwar, S. Dedhar, Marc Ladanyi, K. Bennewith, William W Lockwood
          </td>
          <td>2025-05-31</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Simple Summary In a DNA sequence, there are some regions where a set of DNA bases is repeated right next to others (in tandem), called short tandem repeats (STRs). Sometimes, there are repeats of only one base (e.g., AAAAA or TTTTT). Mutations in such segments are not well studied. This study focused on these repetitive DNA changes in non-melanoma skin cancer (NMSC) across many cancer-related genes. The study identified changes that were only found in the cancerous tissue, not in normal skin. These mutations are associated with significant dysregulations of known cancer-related gene pathways. The findings suggest that these mutations can help identify NMSC patients who might benefit from personalized treatments, like hedgehog (Hh) inhibitors, immune checkpoint inhibitors (ICIs), and other precision therapies. This is the first study to find a new way of identifying and possibly treating skin cancer by looking at specific, previously ignored, repetitive patterns in tumor DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1bbd605d35a66620bdf5359a085c4a80fd24fe" target='_blank'>
              Short Tandem Repeat (STR) Somatic Mutation in Non-Melanoma Skin Cancer (NMSC): Association with Transcriptomic Profile and Potential Implications for Therapy
              </a>
            </td>
          <td>
            M. Kibriya, Armando Almazan, Maria Argos, Tariqul Islam, Christopher R. Shea, Habibul Ahsan, F. Jasmine
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Colon cancer (CC) is one of the most common and lethal cancers worldwide, with rising incidence rates in both developed and developing countries. Although advances in treatments such as surgery, chemotherapy, and targeted therapies have been made, prognosis for advanced colon cancer, particularly with metastasis, remains poor. Recent studies highlight the significant role of post-transcriptional modifications like acetylation in cancer biology, affecting processes like gene transcription, metabolism, and tumor progression. Methods This study applied multi-omics analyses, including single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and Mendelian randomization. Data were obtained from public datasets like GSE132465, UCSC Xena, and GeneCards. We focused on acetylation-related genes, specifically NAT10 and GNE, using scoring methods, cell–cell interaction models, and survival analyses to investigate their role in colon cancer development, metastasis, and immune evasion. Results This study identifies that NAT10 is highly expressed in epithelial cells of colorectal cancer (CC) and is closely associated with tumor progression and metastasis. Single-cell RNA sequencing analysis revealed that NAT10-positive epithelial cells exhibited strong interactions with myeloid cells and T cells, with significant differences in cell–cell communication (p < 0.05). Based-on-summary-data Mendelian randomization (SMR) analysis further supports a causal relationship between NAT10 and colorectal cancer. In the MR analysis, a significant positive correlation was observed between NAT10 and colorectal cancer risk using summary data from genome-wide association studies (GWAS) and expression quantitative trait loci (eQTL) studies (β_SMR = 0.004, p_SMR = 0.041, p_HEIDI = 0.737). These findings suggest that NAT10 may serve as a pathogenic factor in colorectal cancer development, providing additional genetic evidence that links this acetylation-related gene to colorectal cancer. Survival analysis further demonstrated that NAT10-positive epithelial cells are associated with poorer prognosis. In the TCGA dataset, patients with NAT10-positive epithelial cells exhibited a significantly shorter disease-free survival (DFS) (p = 0.012). Unlike GNE-positive cells, NAT10-positive epithelial cells exhibited immune escape characteristics, and TIDE analysis indicated that NAT10-positive epithelial cells were associated with a lower response to immune checkpoint blockade therapy (p = 1.3e−5), suggesting that they may impair the efficacy of immunotherapy by promoting immune evasion. In contrast, GNE was also significantly expressed in epithelial cells of colorectal cancer, but its role differs from that of NAT10. GNE-positive epithelial cells demonstrated strong communication with immune cells, particularly in interactions between myeloid cells and T cells through receptor-ligand pairs. Despite the important role of GNE-positive epithelial cells in the tumor microenvironment, their association with immune escape is weaker compared to NAT10. Survival analysis revealed that GNE-positive epithelial cells were associated with a better prognosis (p = 0.015). In the TCGA dataset, patients with GNE-positive epithelial cells displayed longer disease-free survival (DFS), contrary to the results from the SMR analysis. Conclusions Leveraging SMR and multi-omics analysis, this study highlights the significant role of acetylation-related genes, particularly NAT10, in colon cancer. The findings suggest that acetylation modifications in epithelial cells contribute to immune evasion and cancer progression. NAT10 could serve as a promising biomarker and therapeutic target for early diagnosis and targeted therapy, offering new avenues for improving colon cancer treatment and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37348c834755949c2631ebe0e9fc050634d0c0ce" target='_blank'>
              Deciphering the role of acetylation-related gene NAT10 in colon cancer progression and immune evasion: implications for overcoming drug resistance
              </a>
            </td>
          <td>
            Xuancheng Zhou, Xun Sang, Lai Jiang, Shengke Zhang, Chenglu Jiang, Yuheng Gu, Yiping Fu, Guanhu Yang, Jieying Zhang, Hao Chi, Binbin Wang, Xiaolin Zhong
          </td>
          <td>2025-05-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a devastating malignancy with a 5-year overall survival (OS) rate of approximately 12%. More than 90% of PDAC patients harbor oncogenic mutations in the Kirsten rat sarcoma viral homolog (KRAS) gene. MRTX1133 (MRTX), a novel inhibitor of KRASG12D (the most common KRAS mutation found in pancreatic and colon cancers) has shown promise as a therapeutic agent. To address reported resistance to MRTX, we adapted our anti-leukemia co-targeting strategy and evaluated a combination of MRTX and Bedaquiline (BED), an FDA-approved inhibitor of mitochondrial ATP production, in in vitro human PDAC models. The combination of MRTX and BED demonstrated enhanced cytotoxic effects by disrupting all 11 genes within the DNA helicase family (CMG complex: CDC45-MCM-GINS), which are essential for initiating DNA replication and regulating cell cycle progression. Notably, real-world data analysis from Caris Life Sciences and NCI-TCGA database revealed that low transcriptomic expression of the DNA helicase CMG complex was significantly associated with prolonged survival (e.g., low CDC45 expression and low GINS2 expression with greater than 8 months longer overall survival) in PDAC patients with KRASG12 mutations (N = 9,717; P < 0.00001). However, this combination therapy also triggered strong pro-survival nuclear reprogramming. This effect was mediated by significant genetic activation of an NFκB2-DDIT (DNA damage-induced transcript) axis, which supported tumor chromosomal integrity and DNA repair mechanisms. To overcome NFκB2-driven resistance mechanisms, we explored a triple-targeting strategy that addresses metabolic and genomic plasticity in addition to actively intercepting cell division. This approach combines MRTX1133, Bedaquiline, and the NFκB2 inhibitor SN52, offering a novel therapeutic avenue to treat aggressive pancreatic cancer and potentially improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00669-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecf14d3f8785e3a0d9d03809564ba14faf0a6c7" target='_blank'>
              Targeting DNA helicase CMG complex and NFκB2-driven drug-resistant transcriptional axis to effectively treat KRASG12D-mutated pancreatic cancer
              </a>
            </td>
          <td>
            Jeffrey Xiao, Joshua Kim, Brandon Park, David J. Baylink, Cedric Kwon, Victoria Tran, Scott Lee, Kevin Codorniz, Laren D Tan, Pamela Lobo Moreno, Amy Schill-Depew, S. Mirshahidi, David de Semir, Diana Hanna, Kiran Naqvi, H. Cao, Chien-Shing Chen, Joanne Xiu, H. Lenz, Hamid Mirshahidi, Mark E. Reeves, Yi Xu
          </td>
          <td>2025-05-26</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/129265cf9a8d9036012e909d0315f3224ece47d6" target='_blank'>
              The epigenetic regulator TRIM24 controls melanoma cell dedifferentiation and resistance to treatment in melanoma
              </a>
            </td>
          <td>
            Simon Durand, Félix Boivin, Roxane M. Pommier, L. Barbollat-Boutrand, Maxime Grimont, Félix Pham, Marion Dufeu, Eric Cumunel, Raphaël Schneider, Anthony Ferrari, Anaïs Eberhardt, Stéphane Dalle, J. Caramel
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Pediatric acute myeloid leukemias (AML) are aggressive malignancies characterized by rapid progression, therapy resistance, and poor outcomes, frequently originating during fetal development or early infancy. Unlike adult leukemias, which generally arise from a decades-long accumulation of genetic mutations, pediatric leukemias emerge following genetic alterations occurring prenatally or shortly after birth. However, the biological mechanisms underlying this age-dependent susceptibility and aggressiveness remain largely unexplained. To address this knowledge gap, we developed a novel humanized model of pediatric AML driven by the NUP98::NSD1 fusion oncogene, clinically associated with chemotherapy resistance and dismal prognosis, particularly when accompanied by mutations in the transcription factor WT1. Using CRISPR/Cas9 genome engineering to induce endogenous expression of this fusion oncogene directly in human primary hematopoietic stem and progenitor cells (HSPCs) isolated from distinct developmental stages, we demonstrate that leukemia initiation, lineage specification, and therapeutic response critically depend on the developmental stage of the initiating cells. We found that fetal-derived HSPCs readily underwent leukemia transformation upon expression of the NUP98::NSD1 fusion, especially when co-occurring with WT1 mutations. In contrast, postnatal HSPCs exhibited progressively diminished leukemia transformation potential. Single-cell RNA-seq and ATAC-seq analyses uncovered distinct transcriptional and epigenetic landscapes associated with developmental stage-specific leukemia initiation, leukemia stem cell hierarchies, and therapeutic vulnerabilities between fetal and postnatal-origin leukemias. Clinically, NUP98-rearranged AML exhibiting fetal-like transcriptional programs correlated strongly with worse patient outcomes. Furthermore, fetal-derived leukemias displayed unique metabolic dependencies, contributing to enhanced therapy resistance. Leveraging these insights, we identified and therapeutically targeted vulnerabilities including oxidative phosphorylation and anti-apoptotic mechanisms. Indeed, therapeutic strategies exploiting these developmental-stage-specific susceptibilities effectively overcame resistance in combination therapies and significantly reduced leukemia burden in our preclinical and patient-derived xenograft models. Together, our findings reveal an unappreciated developmental dimension in leukemia biology, illustrating how developmental context critically shapes leukemia potential, lineage fate decisions, and therapeutic response - even among leukemias with identical oncogenic mutations. This work strongly supports the development of age-specific precision oncology strategies to overcome therapy resistance and improve survival outcomes in pediatric patients with aggressive fusion-driven leukemias.


 Elvin Wagenblast. Developmental dependency and metabolic vulnerabilities in NUP98-rearranged pediatric leukemia [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY34-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a04310544d53b394a40ce917fc33a338d122cf3" target='_blank'>
              Abstract SY34-01: Developmental dependency and metabolic vulnerabilities in NUP98-rearranged pediatric leukemia
              </a>
            </td>
          <td>
            Elvin Wagenblast
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Mechanisms for resistance to cytotoxic cancer drugs are dependent on dynamic changes in the biochemistry of cellular pathways, information which is hard to obtain at the systems level. Here we use a deep functional proteomics implementation of the Cellular Thermal Shift Assay to reveal a range of induced biochemical responses to gemcitabine in resistant and sensitive diffuse large B cell lymphoma cell lines. Initial responses in both, gemcitabine resistant and sensitive cells, reflect known targeted effects by gemcitabine on ribonucleotide reductase and DNA damage responses. However, later responses diverge dramatically where sensitive cells show induction of characteristic CETSA signals for early apoptosis, while resistant cells reveal biochemical modulations reflecting transition through a distinct DNA-damage signaling state, including opening of cell cycle checkpoints and induction of translesion DNA synthesis programs, allowing bypass of damaged DNA-adducts. The results also show the induction of a protein ensemble, labeled the Auxiliary DNA Damage Repair, likely supporting DNA replication at damaged sites that can be attenuated in resistant cells by an ATR inhibitor, thus re-establishing gemcitabine sensitivity and demonstrating ATR as a key signaling node of this response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3932b7cf1d620393ace8a6959b61b212c07c27aa" target='_blank'>
              MS CETSA deep functional proteomics uncovers DNA repair programs leading to gemcitabine resistance
              </a>
            </td>
          <td>
            Ying Yu Liang, Khalidah Khalid, Hai Van Le, Hui Min Vivian Teo, Mindaugas Raitelaitis, Marc-Antoine Gerault, Jane Jia Hui Lee, Jiawen Lyu, Allison Chan, A. D. Jeyasekharan, Wai Leong Tam, Pär Nordlund, N. Prabhu
          </td>
          <td>2025-05-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd6992085eb1aec63c000f8c350d79dda9f01e87" target='_blank'>
              A computational strategy to uncover fusion genes in prostate cancer cell lines
              </a>
            </td>
          <td>
            Rebecca A. Morgan, Gary Hardiman
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Reversible alterations at DNA sequence or epigenetic levels can result in phenotypes that are unstably inherited. The reversibility of these inheritable changes might be uniquely beneficial for adaption to possible fluctuations in environment. However, unstable changes are always ignored for the genetic instability in traditional studies, especially in the cause of drug resistance. In this study, we conduct a specific genetic screen in fission yeast using rapamycin (+caffeine) and obtain 173 resistant isolates. In contrast to the common strategy of isolating stable genetic mutants, we passage the cell culture with rapamycin resistance on drug free condition and test the resistance of offspring every five days, and obtain 14 strains that exhibit unstable resistance to rapamycin (the drug resistance is lost randomly among the cell progenies without drug selection pressure). Further studies show that the unstable genetic resistance of some strains is regulated by reversible DNA sequence alterationat the ssp1 gene locus. This study provides new insights and relevant scientific basis for the regulatory mechanism of unstable drug resistance in the process of rapamycin as a clinical anti-tumor drug, and a new possible target for solving the problem of drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/076e8ebfb600d70faa36a964146ab32ad3cd766a" target='_blank'>
              Screening and application of unstable genetically resistant strains in fission yeast.
              </a>
            </td>
          <td>
            Lu-Feng Dan, Yi-Wen Chu, Xin-Rong Wang, Xiang-Wei He
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178f28f36aaf5c0de57a93091858bdbe9ce93e72" target='_blank'>
              Tumor Microenvironment in Ovarian Cancer through Spatial Transcriptomics and Identification of Key Gene Expression Profiles
              </a>
            </td>
          <td>
            K. K. Kırboğa, E. Küçüksille, M. Rudrapal, E. Aktaş, R. Achar, Gouri Deshpande, V. Stupin, E. Silina
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Breast cancer pathogenesis is closely linked to the mechanisms of cellular reprogramming, the process of converting differentiated cells into undifferentiated or different cell types. This reprogramming is achieved through the orchestrated expression of transcription factors which alter the cell’s existing chromatin landscape and establish a transcriptome corresponding to the reprogrammed cell state. Pioneer factors, a type of transcription factor, initiate cell development and cell fate transition by binding to specific DNA motifs within nucleosomes. This binding leads to the recruitment of co-factors, such as nucleosome remodelers, leading to chromatin opening and subsequent activation of specific genes essential for reprogrammed cell function. Improper engagement of these factors can lead to unanticipated or harmful gene expression, potentially contributing to various human diseases, including cancer. GATA3, a pioneer factor, is essential for normal mammary gland development and differentiation. In breast cancer cells, GATA3 has been shown to suppress tumor growth by inducing mesenchymal-to-epithelial transition (MET), the reverse process of epithelial-to-mesenchymal transition (EMT). Interestingly, although mutations in GATA3 are frequent and considered driver mutations, breast cancer patients carrying GATA3 mutations are generally associated with better patient survival. The roles of these mutations remain largely unexplored. M294R and M294K are hot spot missense mutations found in luminal breast cancer patients. These mutations occur in the DNA-binding domain of GATA3, which is crucial for its function as a transcription factor. For functional characterization of these mutants, we utilized MDA-MB-231 mesenchymal cells, a highly aggressive and invasive breast cancer cell line to model mesenchymal-to-epithelial transition (MET). Since ectopic expression of GATA3 induces MET in this cell line, these cells provide an excellent system to observe phenotypic and molecular alterations caused by its missense mutations M294R/K. To quantitatively measure changes in GATA3 mobility and its DNA binding dynamics in living cells, we utilized Fluorescence Recovery After Photobleaching (FRAP) assay. Additionally, to further elucidate the crucial role of GATA3, genomics analyses and phenotypic assays were conducted. Functional characterization of M294R/K mutants in MDA-MB-231 breast cancer cells demonstrated distinct phenotypic and molecular alterations compared to wild-type GATA3. While wild-type GATA3 induced MET, mutant-expressing cells exhibited differential responses. Remarkably, mutants derived from breast cancer patients displayed increased chromatin binding, leading to a tighter chromatin structure and differential gene expression of mesenchymal or epithelial markers. Altered cellular phenotypes such as cell migration are also observed in M294R/K expressed cells. Our finding underscores the importance of precise regulation of pioneer factor-chromatin interactions in cellular reprogramming processes. Understanding the molecular mechanisms underlying GATA3 mutations and their effects on cellular reprogramming in breast cancer provides valuable insights into the roles of GATA3 in breast cancer progression.
 Citation Format: Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku. Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-09.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ca95880c3c403f49e5578c8d1dc155a7b3a608" target='_blank'>
              Abstract P5-06-09: Evaluating Functional Impacts of GATA3 Mutations in Breast Cancer Using Mesenchymal-to-Epithelial Transition Model
              </a>
            </td>
          <td>
            Aerica Nagornyuk, Mika Saotome, Nobuki Hida, Alexander Samardzic, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. Results We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of diYerentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, diYerential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and diYerentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by diYerential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be suYicient for gene regulation. Conclusions Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS ‒ The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops ‒ Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis ‒ Chromatin domain boundaries are weakened during cancer progression ‒ Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops ‒ Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression ‒ Prominent changes in chromatin loops occur at a small subset of diYerentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Targeted therapy interventions using tyrosine kinase inhibitors (TKIs) provide encouraging treatment responses in patients with ALK-rearranged lung adenocarcinomas, yet resistance occurs almost inevitably. In addition to tumor cell-intrinsic resistance mechanisms, accumulating evidence suggests that cancer-associated fibroblasts (CAFs) within the tumor microenvironment contribute to therapy resistance. This study aimed to investigate CAF-driven molecular networks that shape the therapeutic susceptibility of ALK-driven lung adenocarcinoma cells. Methods Three-dimensional (3D) spheroid co-cultures comprising ALK-rearranged lung adenocarcinoma cells and CAFs were utilized to model the tumor microenvironment. Single-cell RNA sequencing was performed to uncover transcriptional differences between TKI-treated homotypic and heterotypic spheroids. Functional assays assessed the effects of CAF-conditioned medium and CAF-secreted factors on tumor cell survival, proliferation, lipid metabolism, and downstream AKT signaling. The therapeutic potential of targeting metabolic vulnerabilities was evaluated using pharmacological inhibition of lipid metabolism and by ferroptosis induction. Results CAFs significantly diminished the apoptotic response of lung tumor cells to ALK inhibitors while simultaneously enhancing their proliferative capacity. Single-cell RNA sequencing identified lipogenesis-associated genes as a key transcriptional difference between TKI-treated homotypic and heterotypic lung tumor spheroids. CAF-conditioned medium and the CAF-secreted factors HGF and NRG1 activated AKT signaling in 3D-cultured ALK-rearranged lung tumor cells, leading to increased de novo lipogenesis and suppression of lipid peroxidation. These metabolic adaptations were critical for promoting tumor cell survival and fostering therapy resistance. Notably, both dual inhibition of ALK and the lipid-regulatory factor SREBP-1, as well as co-treatment with ferroptosis inducers such as erastin or RSL3, effectively disrupted the CAF-driven metabolic-supportive niche and restored sensitivity of resistant lung tumor spheroids to ALK inhibition. Conclusions This study highlights a critical role for CAFs in mediating resistance to ALK-TKIs by reprogramming lipid metabolism in ALK-rearranged lung cancer cells. It suggests that targeting these metabolic vulnerabilities, particularly through inhibition of lipid metabolism or induction of ferroptosis, could provide a novel therapeutic approach to overcome resistance and improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40170-025-00400-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a02a6efa6497a2bcee01f9f0743c25e6ad43b8" target='_blank'>
              Cancer-associated fibroblasts promote drug resistance in ALK-driven lung adenocarcinoma cells by upregulating lipid biosynthesis
              </a>
            </td>
          <td>
            Ann-Kathrin Daum, Lisa Schlicker, M. Schneider, T. Muley, Ursula Klingmüller, Almut Schulze, Michael Thomas, P. Christopoulos, Holger Sültmann
          </td>
          <td>2025-06-16</td>
          <td>Cancer & Metabolism</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Colon cancer accounts for nearly 10% of all cancer diagnoses annually. Microsatellite instability-high (MSI-H) colon cancer is a particularly aggressive subtype of colon cancer that is known to have a significant number of genetic mutations. Microsatellite instability (MSI) refers to genetic hypermutability caused by the dysfunction of the DNA mismatch repair (MMR) system, which leads to errors in repetitive DNA sequences and is a hallmark of certain cancers, including MSI-H colon cancer. MSI-H colon cancer acts on the DNA mismatch repair system causing an accumulation of mutations in microsatellite regions of the DNA. These mutations have been linked with increased tumorigenesis and decreased response to conventional forms of chemotherapy for colon cancer. However, this increased tumor burden results in significant production of neoantigens which makes these tumors immunogenic and therefore perfect candidates for pairing with immunotherapy. As a result, adjuvant immunotherapy in MSI-H colon cancer has become a burgeoning field of research, and synthesizing information regarding the efficacy of these immunotherapies is the goal of this literature review. Pembrolizumab, a monoclonal antibody targeting programmed cell death protein 1 (PD-1), was among the first to receive approval for the treatment of MSI-H colon cancer and was observed to have significant efficacy versus traditional chemotherapy with statistically significant improvements in metrics such as progression-free survival (PFS), overall response (OR), and disease-free survival (DFS). Similarly, Nivolumab, another PD-1 inhibitor, and the combination of nivolumab and ipilimumab (a cytotoxic T-lymphocyte-associated protein 4 (CTLA-4) inhibitor) were granted Food and Drug Administration (FDA) approval for the treatment of MSI-H colon cancer and have also shown the ability to outperform traditional chemotherapy with a higher overall response rate (ORR), mean PFS, and overall survival rates. Off-label uses of existing adjuvant immunotherapies such as Atezolizumab (a programmed death-ligand 1 (PD-L1) inhibitor) and Durvalumab (a PD-L1 inhibitor) have shown promise with preliminary results showing higher DFS and recurrence-free survival but need further data collection. Emerging immunotherapy targets such as lymphocyte activation gene 3 (LAG-3) and T-cell immunoreceptor with immunoglobin and Immunoreceptor Tyrosine-based Inhibitory Motif domains (TIGIT) inhibitors show preliminary promise and may one day become a part of the approach to the treatment of MSI-H colon cancer. This review highlights current advancements, challenges, and emerging trends in the application of immunotherapy for MSI-H colon cancer, with a focus on improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b2777f5f00d4055a2cc08d7a7e5099fddff082" target='_blank'>
              Adjuvant Immunotherapy in Microsatellite Instability-High Colon Cancer: A Literature Review on Efficacy, Challenges, and Future Directions.
              </a>
            </td>
          <td>
            Vasisht Karri, Santosh Nimkar, Samir Dalia
          </td>
          <td>2025-05-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract APOBEC mutagenesis is one of the most common endogenous sources of mutations in human cancer and is a major source of genetic intratumor heterogeneity. High levels of APOBEC mutagenesis are associated with poor prognosis and aggressive disease across diverse cancers, but the mechanistic and functional impacts of APOBEC mutagenesis on tumor evolution and therapy resistance remain relatively unexplored. To address this, we investigated the contribution of APOBEC mutagenesis to acquired therapy resistance in a model of EGFR-mutant non–small cell lung cancer. We find that inhibition of EGFR in lung cancer cells leads to a rapid and pronounced induction of APOBEC3 expression and activity. Functionally, APOBEC expression promotes the survival of drug-tolerant persister cells (DTP) following EGFR inhibition. Constitutive expression of APOBEC3B alters the evolutionary trajectory of acquired resistance to the EGFR inhibitor gefitinib, making it more likely that resistance arises through de novo acquisition of the T790M gatekeeper mutation and squamous transdifferentiation during the DTP state. APOBEC3B expression is associated with increased expression of the squamous cell transcription factor ΔNp63 and squamous cell transdifferentiation in gefitinib-resistant cells. Knockout of p63 in gefitinib-resistant cells reduces the expression of the ΔNp63 target genes IL-1α/β and sensitizes these cells to the third-generation EGFR inhibitor osimertinib. These results suggest that APOBEC activity promotes acquired resistance by facilitating evolution and transdifferentiation in DTPs and that approaches to target ΔNp63 in gefitinib-resistant lung cancers may have therapeutic benefit. Significance: APOBEC mutagenesis is a common source of genetic heterogeneity in cancer, and APOBEC mutational signatures are enriched in metastatic and drug-resistant tumors. However, the mechanisms through which APOBEC3 enzymes promote tumor evolution remain unknown. In this study, we show that APOBEC3 activity contributes to the development of therapy-resistant cancer cells by promoting evolution of DTP cells. These findings offer insights into the role of APOBEC mutagenesis in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e7a11c8b2dc4731f11b9133f359e7ac89041b4a" target='_blank'>
              APOBEC3 Activity Promotes the Survival and Evolution of Drug-Tolerant Persister Cells during EGFR Inhibitor Resistance in Lung Cancer
              </a>
            </td>
          <td>
            N. M. G. Garcia, Jessica N Becerra, Sharan Srinivasan, Brock J McKinney, Ashley V DiMarco, Feinan Wu, Matthew Fitzgibbon, James V. Alvarez
          </td>
          <td>2025-05-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Radiotherapy (RT) resistance remains a substantial challenge in cancer therapy. Although physical factors are optimizing, the biological mechanisms for RT resistance are still elusive. Herein, we explored potential reasons for this difficult problem by generating RT‐resistant models for in vitro and in vivo experiments. We found that abnormal spindle‐like microcephaly‐associated protein (ASPM) was highly expressed in RT‐resistant samples and significantly correlated with disease advance in lung adenocarcinoma. Mechanistically, ASPM helps RT‐resistant cells to evade spindle checkpoint surveillance and complete cell division after irradiation through destruction of microtubule stability, with subsequent increases in chromosome mis‐segregation and deteriorating chromosomal stability during mitosis. Depletion of ASPM stabilized microtubules and significantly decreased chromosome mis‐segregation, restoring the sensitivity of RT‐resistant cells to radiation. We further found, with bioinformatics analysis, amino acid sequence 963–1263 of ASPM as a potential new drug target for overcoming RT resistance and identified 9 drug pockets within this domain for clinical translation. Our findings suggest that ASPM is a key regulator with an important role in promoting RT resistance in non‐small cell lung cancer, and that suppressing or blocking its expression could be worth exploring as therapy for a variety of RT‐resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3da5053fb0b4450ac57e5e9d12e55ad706abecf" target='_blank'>
              ASPM Induces Radiotherapy Resistance by Disrupting Microtubule Stability Leading to Chromosome Malsegregation in Non‐Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Tao Zhong, Ning Liu, Juan Wang, Songbo Xie, Lisheng Liu, Minglei Wang, Fei Wu, Xiaozheng Chen, Changyan Xiao, Xiaoxiao Gongye, Meng Wu, Liewei Wen, Jinming Yu, Dawei Chen
          </td>
          <td>2025-05-07</td>
          <td>Exploration</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly aggressive intraocular malignancy with limited therapeutic options for metastatic disease. Existing transgenic UM mouse models inadequately recapitulate human disease progression, while transplant models lack immune competence for studying the tumor immune microenvironment and therapeutic interventions. To address these limitations, we developed a genetically engineered mouse model incorporating stepwise genetic alterations implicated in human UM progression. Spatiotemporally controlled expression of mutant GNAQQ209L from the endogenous locus induced choroidal nevi with limited penetrance. Concomitant BAP1 deletion enhanced nevus formation, while further MYC activation led to fully penetrant intraocular tumors with metastatic potential. Single-cell RNA sequencing revealed malignant cells segregated into Melanocytic and Neural Crest-like subpopulations characterized by distinct transcriptional and biosynthetic programs. Trajectory analyses inferred dedifferentiation from the Melanocytic toward the Neural Crest-like state during tumor progression. Comparison to human UM revealed commonalities with highly aggressive Class 2 UM, including gene expression signatures and copy number gains affecting genes that map to human chromosome 8q beyond the activated MYC allele, suggesting cooperative effects of multiple drivers in this chromosomal region. The tumor microenvironment featured immunosuppressive macrophage populations and exhausted T cells, closely resembling human UM. This physiologically relevant, immune-competent model provides a platform for investigating UM biology, functionally characterizing candidate driver genes, and developing immune-based therapeutic strategies. SIGNIFICANCE STATEMENT We developed a mouse model that resembles the genetic progression and phenotypic plasticity of human UM. This spatially controlled model confirms the critical role of driver mutations in GNAQ and BAP1, proposes MYC as a promoter of malignant transformation in coordination with other chromosome 8q genes, and reveals UM progression through distinct cellular states. This model offers an urgently needed preclinical platform for understanding the immunogenomics of UM and for testing immune and targeted treatments for this lethal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35df183db757e5d1e691fe22c0b60fdf40bee0e5" target='_blank'>
              A multi-step immune-competent genetic mouse model reveals phenotypic plasticity in uveal melanoma
              </a>
            </td>
          <td>
            Xiaonan Xu, Xiaoxian Liu, J. Dollar, Xiao Liu, Neel Jasani, Benjamin Posorske, S. Sriramareddy, Vinesh Jarajapu, J. Kuznetsoff, John Sinard, Richard L. Bennett, Jonathan D. Licht, K. Smalley, J. W. Harbour, Xiaoqing Yu, F. Karreth
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Acquired genetic alterations drive resistance to endocrine and targeted therapies in metastatic breast cancer; however, the underlying processes engendering these alterations are largely uncharacterized. To identify the underlying mutational processes, we utilized a clinically annotated cohort of 3,880 patient samples with tumor-normal sequencing. Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) enzymes were prevalent and enriched in post-treatment hormone receptor-positive cancers. These signatures correlated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor therapy in hormone receptor-positive metastatic breast cancer. Whole-genome sequencing of breast cancer models and paired primary-metastatic samples demonstrated that active APOBEC3 mutagenesis promoted therapy resistance through characteristic alterations such as RB1 loss. Evidence of APOBEC3 activity in pretreatment samples illustrated its pervasive role in breast cancer evolution. These studies reveal APOBEC3 mutagenesis to be a frequent mediator of therapy resistance in breast cancer and highlight its potential as a biomarker and target for overcoming resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65dfedea9c7ed384b972c98e352341985ea58fa" target='_blank'>
              APOBEC3 mutagenesis drives therapy resistance in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Hong Shao, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Yanjun Chen, Isabella Vegas, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, G. Curigliano, B. Weigelt, N. Riaz, Simon N. Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-05-16</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Lymphomas are highly heterogeneous malignancies characterized by diverse genetic, epigenetic, and phenotypic profiles, which complicate diagnosis, prognosis, and treatment. Traditional bulk sequencing methods mask the complexity of intratumoral variation, often overlooking rare subclones that may drive disease progression and therapeutic resistance. Single-cell genomics has emerged as a transformative approach to decipher tumor heterogeneity at unprecedented resolution. This technology enables the dissection of individual cellular populations within lymphomas, offering insights into clonal evolution, transcriptional diversity, and microenvironmental interactions. By applying single-cell RNA sequencing (scRNA-seq), chromatin accessibility assays (scATAC-seq), and single-cell DNA sequencing (scDNA-seq), researchers can unravel lineage relationships, identify resistant subpopulations, and track dynamic changes in response to therapy. In lymphomas such as diffuse large B-cell lymphoma (DLBCL), follicular lymphoma (FL), and mantle cell lymphoma (MCL), single-cell approaches have revealed distinct malignant and non-malignant cell states that correlate with treatment outcomes. Moreover, integrating single-cell data with spatial transcriptomics and immune profiling enhances the understanding of the tumor microenvironment, including immune evasion mechanisms. These insights can inform personalized treatment strategies, identify novel therapeutic targets, and enable early detection of relapse. Despite technical challenges such as data complexity, sample viability, and cost, the application of single-cell genomics in lymphoma research is rapidly advancing. Future directions include multi-omics integration, real-time patient monitoring, and clinical translation of predictive biomarkers. This review underscores the pivotal role of single-cell genomics in resolving tumor heterogeneity and predicting therapeutic resistance, positioning it as a cornerstone for next-generation precision oncology in lymphomas,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/438856abbab164e715372bfe3f2a6c07df9594ec" target='_blank'>
              Role of mutational signatures and clonality assessments in tailoring targeted therapies for lymphoma
              </a>
            </td>
          <td>
            Andoh Sheikh Atta-ullah, Uche Philip, Sampson Janice Candy
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bab738cfa40844711ac785ce352b652355efc09b" target='_blank'>
              Precision oncology: transforming cancer care through personalized medicine.
              </a>
            </td>
          <td>
            Rituraj, Rashmi Saxena Pal, Jenny Wahlang, Y. Pal, Mvnl Chaitanya, Sumedha Saxena
          </td>
          <td>2025-06-09</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite the advances in single cell whole genome sequencing, the ability to detect structural variations from mitotic crossovers remains challenging, while sister chromatid exchanges remain undetectable. Here we describe sci-L3-Strand-seq, a combinatorial indexing method with linear amplification for DNA template strand sequencing that cost-effectively scales to millions of single cells. We provide a computational framework to fully leverage the throughput, as well as the relatively sparse but multifaceted genotype information within each cell that includes strandedness, digital counting of copy numbers, and haplotype-aware chromosome segmentation, to systematically distinguish seven possible types of mitotic crossover outcomes and resulting genome instability. We showcase the power of sci-L3-Strand-seq by quantifying the rates of error-free and mutational crossovers in thousands of cells, enabling us to explore enrichment patterns of genomic and epigenomic features. The throughput of sci-L3-Strand-seq also gave us the ability to measure subtle phenotypes, opening the door for future large mutational screens. Furthermore, mapping clonal lineages provided insights into the temporal order of certain genome instability events, showcasing the potential to dissect cancer evolution. Altogether, we show the wide applicability of sci-L3-Strand-seq to the study of DNA repair, structural variations, and genome instability in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/867f71c31d23d231530c8f668476d966ca7500c8" target='_blank'>
              High-throughput mapping of spontaneous mitotic crossover and genome instability events with sci-L3-Strand-seq
              </a>
            </td>
          <td>
            Peter Chovanec, Trevor Ridgley, Yi Yin
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Acquired resistance to molecularly targeted therapies remains a formidable challenge in the treatment of cancer, despite significant advancements over the last several decades. We critically evaluate the evolving landscape of resistance mechanisms to targeted cancer therapies, with a focus on the genetic, molecular, and environmental contributors across a variety of malignancies. Intrinsic mechanisms such as mutations, drug and drug target modifications, and, notably, the activation of the mitogen-activated protein kinase (MAPK) and phosphoinositide 3-kinase (PI3K)/Akt pathways are mechanisms different malignancies use to combat therapeutic effectiveness. Furthermore, extrinsic alterations to the tumor microenvironment contribute to therapeutic resistance. We highlight similarities and differences in mechanisms across a wide spectrum of cancers including hematologic malignancies, non-small cell lung cancer, gastrointestinal, breast, and prostate cancers, pancreatic, ovarian, endometrial, and intracranial gliomas. Emerging strategies to overcome resistance, including multi-targeted approaches, combination therapies, and exploitation of synthetic lethality, are all critically discussed. We advocate for a nuanced understanding of resistance mechanisms as a cornerstone for developing future therapeutic strategies, emphasizing the necessity for integrated approaches that encompass genomic insights and precision medicine to outpace the dynamic and complex nature of cancer evolution and therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25d12537bfda4f2fdd2edf55635a9d858058bddd" target='_blank'>
              Acquired resistance to molecularly targeted therapies for cancer
              </a>
            </td>
          <td>
            Nolan Stubbs, Tyler S. Roady, Maximilian P. Schwermann, Elias O. Eteshola, William J MacDonald, Connor Purcell, Dinara Ryspayeva, Nataliia Verovkina, Vida Tajiknia, Maryam Ghandali, Viva Voong, Alexis J. Lannigan, A. Raufi, S. Lawler, Sheldon L. Holder, B. Carneiro, Liang Cheng, Howard P. Safran, Stephanie L. Graff, Don S. Dizon, S. Mani, A. Seyhan, Robert W. Sobol, Eric T. Wong, Clark C. Chen, Z. Gokaslan, Martin S. Taylor, Brian M. Rivers, Wafik S. El-Deiry
          </td>
          <td>2025-06-05</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Glioblastoma (GBM) remains the most common and lethal adult malignant primary brain cancer with few treatment options. A significant issue hindering GBM therapeutic development is intratumor heterogeneity. GBM tumors contain neoplastic cells within a spectrum of different transcriptional states. Identifying effective therapeutics requires a platform that predicts the differential sensitivity and resistance of these states to various treatments. Here, we developed a novel framework, ISOSCELES (Inferred cell Sensitivity Operating on the integration of Single-Cell Expression and L1000 Expression Signatures), to quantify the cellular drug sensitivity and resistance landscape. Using single-cell RNA sequencing of newly diagnosed and recurrent GBM tumors, we identified compounds from the LINCS L1000 database with transcriptional response signatures selectively discordant with distinct GBM cell states. We validated the significance of these findings in vitro, ex vivo, and in vivo, and identified a novel combination of an OLIG2 inhibitor and Depatux-M for GBM. Our studies suggest that ISOSCELES identifies cell states sensitive and resistant to targeted therapies in GBM and that it can be applied to identify new synergistic combinations. Highlights Integration of GBM single-cell RNA sequencing data with L1000-derived drug response signatures facilitates clustering of tumor cells and small molecules on cell-drug connectivity. Cell-drug connectivity predicts the identities of drug-sensitive and resistant cell states. In silico perturbation analysis using cell-drug connectivity predicts drug-induced changes in the cell-drug connectivity landscape in vivo. In silico perturbation analysis to predict drug-induced changes in the tumor cell-drug connectivity landscape predicts drug combinations that synergize in vivo to extend survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25613f3a85e6691e66bbf8705e07ffcebf8c6cbf" target='_blank'>
              DRUG AND SINGLE-CELL GENE EXPRESSION INTEGRATION IDENTIFIES SENSITIVE AND RESISTANT GLIOBLASTOMA CELL POPULATIONS
              </a>
            </td>
          <td>
            Robert K. Suter, Anna M. Jermakowicz, Rithvik Veeramachaneni, Matthew D’Antuono, Longwei Zhang, Rishika Chowdary, Simon Kaeppeli, Madison Sharp, Pravallika Palwai, Vasileios Stathias, Grace Baker, Luz Ruiz, Winston Walters, Maria Cepero, Danielle M. Burgenske, Edward B Reilly, Anatol Oleksijew, Mark G. Anderson, S. Williams, Michael E. Ivan, Ricardo J Komotar, M. I. De La Fuente, Gregory Stein, Alexander Wojcinski, Santosh Kesari, J. Sarkaria, Stephan C. Schürer, N. Ayad
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>75</td>
        </tr>

        <tr id="Background/Objectives: TIGD1 (Trigger Transposable Element Derived 1) is a recently identified oncogene with largely unexplored biological functions. Emerging evidence suggests its involvement in multiple cellular processes across cancer types. This study aimed to perform a comprehensive pan-cancer analysis of TIGD1 to evaluate its expression patterns, diagnostic utility, prognostic value, and association with immunotherapy response and drug resistance. Methods: Transcriptomic and clinical data from TCGA and GTEx were analyzed using various bioinformatic tools. Expression profiling, survival analysis, immune correlation studies, gene set enrichment, single-cell sequencing, and drug sensitivity assessments were performed. Results: TIGD1 was found to be significantly upregulated in various tumor types, with notably high expression in colon adenocarcinoma. Elevated TIGD1 expression was associated with poor prognosis in several cancers. TIGD1 levels correlated with key features of the tumor immune microenvironment, including immune checkpoint gene expression, TMB, and MSI, suggesting a role in modulating anti-tumor immunity. GSEA and single-cell analyses implicated TIGD1 in oncogenic signaling pathways. Furthermore, high TIGD1 expression was linked to resistance to several therapeutic agents, including Zoledronate, Dasatinib, and BLU-667. Conclusions: TIGD1 may serve as a promising diagnostic and prognostic biomarker, particularly in colon, gastric, liver, and lung cancers. Its strong associations with immune modulation and therapy resistance highlight its potential as a novel target for precision oncology and immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1163400219b2ecb1878714587256ba612b794cd1" target='_blank'>
              Beyond Transposons: TIGD1 as a Pan-Cancer Biomarker and Immune Modulator
              </a>
            </td>
          <td>
            Merve Gülşen Bal Albayrak, Tuğcan Korak, Gurler Akpinar, Murat Kasap
          </td>
          <td>2025-05-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The integrity of the genome is maintained by mismatch repair (MMR) proteins that recognize and repair base mismatches and insertion/deletion errors generated during DNA replication and recombination. A defective MMR system results in genome-wide instability and the progressive accumulation of mutations. Tumors exhibiting deficient MMR (dMMR) and/or high levels of microsatellite instability (termed “microsatellite instability high”, or MSI-H) have been shown to possess fundamental differences in clinical, pathological, and molecular characteristics, distinguishing them from their “microsatellite stable” (MSS) counterparts. Molecularly, they are defined by a high mutational burden, genetic instability, and a distinctive immune profile. Their distinct genetic and immunological profiles have made dMMR/MSI-H tumors particularly amenable to treatment with immune checkpoint inhibitors (ICIs). The ongoing development of biomarker-driven therapies and the evaluation of novel combinations of immune-based therapies, with or without the use of conventional cytotoxic treatment regimens, continue to refine treatment strategies with the goals of maximizing therapeutic efficacy and survival outcomes in this distinct patient population. Moreover, the resultant knowledge of the mechanisms by which these features are suspected to render these tumors more responsive, overall, to immunotherapy may provide information regarding the potential optimization of this therapeutic approach in tumors with proficient MMR (pMMR)/MSS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0727d40a1aa873459400ba4134581be209441f5d" target='_blank'>
              Deficient Mismatch Repair and Microsatellite Instability in Solid Tumors
              </a>
            </td>
          <td>
            Joy A Awosika, James L. Gulley, Danielle M. Pastor
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cisplatin-based chemotherapy is a cornerstone in treating cervical cancer, yet the efficacy is frequently limited by the rapid onset of drug resistance, a major challenge in clinical management. To investigate this, we employed HPV16+ human cervix squamous carcinoma cells, SiHa (CIS/S), and their cisplatin-resistant subline (CIS/R) as a model. Using DIA-based proteomics, we identified 5152 protein groups and over 50,000 peptides with a global FDR <1%. Comparative analysis revealed 123 differentially expressed proteins. Gene Set Enrichment Analysis (GSEA) highlighted proteins involved in DNA damage, metabolism, and repair pathways (RFC4, RFC3, RFC2, DUT, DDX54, CDCA8, CDK7, CHAF1B, and GTF2F1), suggesting a role in developing acquired cisplatin resistance. Pathways related to mitotic spindle assembly and P53 signaling were found to be perturbed in resistant cells. Next, we screened a library of approx. 240 FDA-approved drugs against three protein targets and found four small-molecular ligands as potential hits for further in vitro validation. Cabozantinib and sorafenib gave us positive results in terms of increasing the cisplatin sensitivity of CIS/R cells. In conclusion, our findings provide insights into the molecular mechanisms underpinning cisplatin resistance in cervical cancer and propose novel strategies for combating this resistance through targeted therapies and drug repurposing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7839b1b461ca00a00e3a5b30fffff61171a6534" target='_blank'>
              Investigating Cisplatin Resistance in Squamous Cervical Cancer: Proteomic Insights into DNA Repair Pathways and Omics-Based Drug Repurposing.
              </a>
            </td>
          <td>
            Amrita Mukherjee, Sayan Manna, Avinash Singh, Adrija Ray, Sanjeeva Srivastava
          </td>
          <td>2025-04-29</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Lineage plasticity is critical for tumor progression and therapy resistance, but the molecular mechanisms underlying cell identity shifts in cancer remain poorly understood. In lung adenocarcinoma (LUAD), the loss of pulmonary lineage fidelity and acquisition of alternate identity programs converge on hybrid identity (hybrid ID) states, which are postulated to be key intermediates in LUAD evolution and are characterized by the coactivation of developmentally incompatible identity programs within individual cells. Here, we uncover a previously unrecognized role for the gastrointestinal transcriptional regulator HNF4α in driving tumor growth and hybrid ID states in LUAD. In LUAD cells expressing the lung lineage specifier NKX2-1, HNF4α induces a GI/liver-like state by directly binding and activating its canonical targets. HNF4α also disrupts NKX2-1 genomic localization and dampens pulmonary identity within hybrid ID LUAD. We show that this hybrid ID state is maintained by sustained RAS/MEK signaling. Inhibition of the RAS/MEK signaling cascade augments NKX2-1 chromatin binding at pulmonary-specific genes and induces drug resistance-associated pulmonary signatures. Finally, we demonstrate that HNF4α depletion sensitizes LUAD cells to KRASG12D inhibition. Collectively, our data show that coexpression of opposing lineage specifiers is a novel mechanism of identity dysregulation in LUAD that influences both tumor progression and response to targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e47763b8688fbd0471d5f59c0d98d63dd2cf51e" target='_blank'>
              Opposing lineage specifiers induce a protumor hybrid identity state in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Gabriela Fort, Henry Arnold, S. Camolotto, Kayla O'Toole, Rushmeen Tariq, Anna Waters, Katherine Gillis, Eric L. Snyder
          </td>
          <td>2025-06-12</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a dynamic and versatile genomic element with key roles in physiological regulation and disease pathology. This review synthesizes current knowledge on eccDNA, covering its classification, biogenesis, detection methods, biological functions, and clinical implications. Once considered rare anomalies, eccDNAs are now recognized as major drivers of oncogene amplification, genomic plasticity, and therapeutic resistance, particularly in cancer. EccDNA subtypes such as microDNA, double minutes, and ecDNA are defined by their structural, genomic, and pathological features. EccDNAs originate through diverse mechanisms including DNA repair, chromothripsis, breakage fusion bridge cycles, and apoptosis, occurring in both normal and stressed cells. Advances in long-read and single-cell sequencing, CRISPR-based synthesis, and computational tools have improved detection and functional analysis. Functionally, eccDNAs contribute to transcriptional amplification, activate immune responses through cGAS-STING signaling, and facilitate intercellular communication. They are found across a range of tissues and disease states—including cancer, cardiovascular, neurological, autoimmune, and metabolic disorders—and serve as both biomarkers and regulatory elements. We introduce the concept of the stress selection theory, which proposes eccDNA as an adaptive reservoir that enhances cellular fitness in response to environmental and therapeutic pressures. Despite growing insights, challenges remain in understanding tissue-specific roles, achieving clinical translation, and standardizing methodologies. Emerging tools in multi-omics, spatial biology, and artificial intelligence are expected to drive future breakthroughs in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31cb2df8f26047d209cf38790b651f7df3921fd" target='_blank'>
              Extrachromosomal circular DNA drives dynamic genome plasticity: emerging roles in disease progression and clinical potential
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-05-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 The cancer cell cycle possesses an incredible capacity to adapt or change in response to oncogenic mutations or drug treatment (cell cycle “plasticity”) to generate a diversity of solutions to the cell cycle (distinct cell cycle “programs”) that differ in key regulatory effectors and mechanisms. To study the extent and role of cell cycle plasticity in tumor progression and drug resistance, we developed cell cycle mapping - a single-cell proteomic imaging platform that combines highly multiplexed immunofluorescence with manifold learning to visualize cell cycle plasticity among and within tumors. Applying cell cycle mapping to a large library of hormone receptor (HR)+ breast tumors, we observed 4 distinct cell cycle programs driving tumor growth. These programs differed in the expression/activity of key cyclins and CDKs. In many cases, we observed multiple cell cycle programs within a single tumor correlating with differences in the cellular composition of the local tumor environment. These results also highlight the challenge of targeting the cancer cell cycle (e.g. with CDK4/6 inhibitors), given the heterogeneity of cell cycle programs actin within a single tumor. To further explore the role of cell cycle plasticity in targeted therapy, we used a combination of cell cycle mapping and time-lapse imaging to reveal how the breast cancer cell cycle responds to a panel of FDA-approved drugs targeting a range of cell cycle effectors, including multiple CDK inhibitors. We observed clear heterogeneity in the mechanisms of cell cycle adaptation executed across a panel of cell lines modeling distinct molecular (HR+, HER2+, basal) and histological subtypes (invasive ductal carcinoma, invasive lobular carcinoma) of breast cancer. Altered expression/activity of key cell cycle regulators provides a new “path” through the cell cycle, a molecular detour that allows cells to continue proliferating in the presence of drug to drive therapeutic resistance.


 Yian Yang, Thu Pham, Janet McLaughlin, Steffi Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert. Cell cycle plasticity drives tumor progression and drug resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY22-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05724e131a84cae0882ab8214c7527b7f572cfbc" target='_blank'>
              Abstract SY22-02: Cell cycle plasticity drives tumor progression and drug resistance
              </a>
            </td>
          <td>
            Yian Yang, Thu Pham, Janet McLaughlin, S. Oesterreich, Adrian Lee, Katarzyna Kedziora, Wayne Stallaert
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, Reva K Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423474067572f39ac510a609277d4e54dd172eb6" target='_blank'>
              YAP signaling promotes resistance to MEK and AKT inhibition in NF1-related MPNSTs
              </a>
            </td>
          <td>
            L. E. McGee, J. Grit, Curt J. Essenburg, Sophia Agrusa, Lisa Turner, K. Becker, Anwesha Dey, Jennifer E. Klomp, Jeffrey A. Klomp, Marie Adams, Ian Beddows, Emily Wolfrom, Daisy Fu, A. Hirbe, Carrie R. Graveel, Matthew R. Steensma
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The genomes of all organisms encode diverse functional elements, including thousands of genes and essential noncoding regions for gene regulation and genome organization. Systematic perturbation of these elements is crucial to understanding their roles and how their disruption impacts cellular function. Genetic perturbation approaches, which disrupt gene expression or function, provide valuable insights by linking genetic changes to observable phenotypes. However, perturbing individual genomic elements one at a time is impractical. Genetic screens overcome this limitation by enabling the simultaneous perturbation of numerous genomic elements within a single experiment. Traditionally, these screens relied on simple, high-throughput readouts such as cell fitness, differentiation, or one-dimensional fluorescence. However, recent advancements have introduced powerful technologies that combine genetic screens with image-based and single-cell sequencing readouts, allowing researchers to study how perturbations affect complex cellular phenotypes on a genome-wide scale. These innovations, alongside the development of CRISPR-Cas technologies, have significantly enhanced the precision, efficiency, and scalability of genetic screening approaches. In this review, we discuss the genetic screens performed in kinetoplastid parasites to date, emphasizing their application to both coding and noncoding regions of the genome. Furthermore, we explore how integrating image-based and single-cell sequencing technologies with genetic screens holds the potential to deliver unprecedented insights into cellular function and regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc5825d146111335de4ea8d326bda5fc9057913a" target='_blank'>
              Next generation genetic screens in kinetoplastids.
              </a>
            </td>
          <td>
            James Budzak, T. Siegel
          </td>
          <td>2025-06-06</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Transcription factors play a central role in cancer growth, progression, and metastasis, and contribute to intratumor phenotypic plasticity which enable drug tolerance and cancer relapse. Changes in the regulatory activities of transcription factors in cancer may not always be detected by investigating mutational signatures or differential expression of the transcription factor, as done in traditional analysis. In addition, past studies have focused on the activities of transcription factors in tumors as a whole and thus, have not fully captured the heterogeneity in gene regulation among different cell types within the tumor microenvironment. In this work, through an analysis of the transitions in regulatory network architecture and gene regulation dynamics, we identify the central transcription factors associated with lung adenocarcinoma progression. The gene NR2F1, associated with neurodevelopment and cancer dormancy, emerge as a key transcription factor in the progression of lung adenocarcinoma. We further identify transcription factors that are only active in cancer samples and uncover how changes in gene regulation dynamics influence intratumor heterogeneity. Taken together, our work elucidates the transitions in gene regulatory network during cancer progression, identifies central transcription factors in this process, and reveals the complex regulatory changes cooccurring in different cell types within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a902ef3e0da1388bf4b1861117a26aab25af6066" target='_blank'>
              Gene regulatory network transitions reveal the central transcription factors in lung adenocarcinoma progression
              </a>
            </td>
          <td>
            Upasana Ray, Adarsh Singh, Debabrata Samanta, Riddhiman Dhar
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="RNA–DNA differences (RDDs) challenge the traditional view of RNA as a faithful copy of DNA, arising through RNA editing, transcriptional errors, and oxidative damage. Reactive oxygen species (ROS) play a central role, inducing lesions like 8-oxo-guanine that compromise transcription and translation, leading to dysfunctional proteins. This review explores the biochemical basis of RDDs, their exacerbation under oxidative stress, and their dual roles in cellular adaptation and disease. RDDs contribute to genomic instability and are implicated in cancers, neurodegenerative disorders, and autoimmune diseases, while also driving phenotypic diversity. Drawing on terrestrial and spaceflight studies, we highlight the intersection of oxidative stress, RDD formation, and cellular dysfunction, proposing innovative mitigation approaches. Advancements in RDD detection and quantification, along with ROS management therapies, offer new avenues to restore cellular homeostasis and promote resilience. By positioning RDDs as a hallmark of genomic entropy, this review underscores the limits of biological adaptation. Furthermore, the prevalence of guanine-rich codons in antioxidant genes increases their susceptibility to ROS-induced oxidative lesions, linking redox stress, genomic instability, and constrained adaptation. These insights have profound implications for understanding aging, disease progression, and adaptive mechanisms in both terrestrial and space environments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a77198569606ddd08867d623b16c1b892ae7d5b" target='_blank'>
              RNA–DNA Differences: Mechanisms, Oxidative Stress, Transcriptional Fidelity, and Health Implications
              </a>
            </td>
          <td>
            Viktor Stolc, Ondrej Preto, M. Karhánek, Friedemann Freund, Yuri Griko, David J. Loftus, Maurice M. Ohayon
          </td>
          <td>2025-04-30</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2222f4aa3745d3827861310375ad2ef6308f3da" target='_blank'>
              Mapping genetic modifiers of epimutation rates reveals a punctuated-equilibrium model of CG methylome evolution
              </a>
            </td>
          <td>
            Zhilin Zhang, Wilma Wanney, Yangyang Xu, J. Zicola, Angela M. Hancock, Robert J. Schmitz, F. Johannes
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="CRISPR has become a revolutionary tool for cancer biology and therapy with unprecedented precision for genome editing. Besides cancer, CRISPR is also under research for possible use in the treatment of viral diseases, such as Bovine Viral Diarrhea Virus (BVDV), a serious livestock problem. Through enabling gene modifications to be made to individual genes, CRISPR is used in identifying oncogenic drivers, characterizing tumor suppressor networks, and designing innovative therapeutics. This review focuses on the various uses of CRISPR for cancer studies with special attention to methods like gene knockout, gene activation, and base editing that each has a great potential for correcting oncogenic mutations and restoring tumor suppressor activity. Moreover, CRISPR's capacity to regulate viral replication, as that of BVDV, signifies its dual role in oncology as well as in viral pathogenesis. Interventions based on CRISPR such as chimeric antigen receptor (CAR) T cell therapy and synthetic lethality are essentially revolutionizing cancer therapy by improving immune responses and capitalizing on the specificity that is characteristic of cancer cells. However, off-target effects, tumor heterogeneity, and ethical dilemmas remain onerous challenges for clinical application. CRISPR delivery systems, despite playing a central role in advancing cancer therapies, also have prospects in optimizing the effectiveness of treatment against viral pathogens like BVDV. Advances in delivery systems using nanoparticles and viral vectors are mitigating against these challenges and improving efficacy and specificity of CRISPR reagent in vivo. In addition to this, continued progress in new technologies such as prime editing and base editing is predicted to improve precision and efficacy of CRISPR-based therapy. Since CRISPR technology is continuously developing, CRISPR's potential to treat both cancer and viral infections, like BVDV, simultaneously will be instrumental in precision medicine. This review highlights the revolutionary potential of CRISPR to revolutionize cancer treatment paradigms and brings hope for more effective and individualized therapy. With potential developments to emergent and potentiated CRISPR tools and reagents, it is predicted that they will play a center stage role in precision oncology and promote better patient results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d3748e9db2d0a3114958d899e7fef2d33921ff" target='_blank'>
              CRISPR in Cancer Biology and Bovine Viral Diarrhea Therapy: Precision Genome Editing at the Frontier of Oncology and Viral Pathogenesis
              </a>
            </td>
          <td>
            Jindong Gao, Mengdi Zhang, Linjia Gao, Lei Kuang, Tuniyazi. Maimaiti, Ling Zhao, Qinghua Shang, Changmin Hu
          </td>
          <td>2025-05-19</td>
          <td>Bio Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genetically identical bacterial cells often exhibit heterogeneous responses to antibiotics − some survive, others die. Here, we show that this heterogeneity propagates across generations to give rise to phenotypic resistance. Using real-time single-cell tracking, we exposed Escherichia coli to the β-lactam cefsulodin at its clinical breakpoint concentration and analyzed cell fate within genealogical trees statistically. Cell survival was strongly correlated among family members, driving the selective enrichment of robust lineages within an otherwise susceptible population. Our genealogical population model identified heritable phenotypic resistance as a key factor underlying this enrichment, which was validated experimentally. Comparing enrichment dynamics between the wild-type and a tolC knock-out strain, deficient in multidrug efflux, uncovered nuanced changes that increased the intergenerational memory of phenotypic resistance. Our findings provide evidence for heritable phenotypic resistance and demonstrate how its propagation through cell-to-cell heterogeneity enables the survival of minority cells within isogenic populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6715bbe58fe26a64981336caf37053a9a8306b82" target='_blank'>
              Lineage-dependent variations in single-cell antibiotic susceptibility reveal the selective inheritance of phenotypic resistance in bacteria
              </a>
            </td>
          <td>
            Wesley Stine, Tatsuya Akiyama, David S. Weiss, Minsu Kim
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4eb70f970ca5d62a053191099abf0f49d770733" target='_blank'>
              Heterogeneous therapy-resistant cancer cells have distinct and exploitable drug sensitivity profiles
              </a>
            </td>
          <td>
            Gianna T. Busch, Ryan H. Boe, Jingxin Li, Robert F. Gruener, Miles Arnett, Pavithran T. Ravindran, M. Herlyn, R. S. Huang, Arjun Raj
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="The Extrachromosomal DNA (ecDNA) in cancer biology has emerged as a critical driver, enhancing the knowledge about tumorigenesis, progression and treatment. The current review article explores the multifaceted role of Extrachromosomal DNA, focusing on its ability to amplify oncogenes, enhance tumor heterogenicity, and support cancer cell survival and understanding therapeutics pressure. Moreover, the unique transcriptional and replication dynamics of ecDNA with linking the uneven segregation during cell division, create a significant environment for the understanding of genetic diversity and adaptive evolution in tumors. In clinics the ecDNA holds significant potential as a biomarker for cancer, diagnosis, treatment, therapeutic monitoring and progenies. The higher level of ecDNA correlates with poor clinical outcomes and resistance to therapies, emphasizing its prognostic value. The current therapeutic techniques targeting ecDNA specific vulnerabilities, such as disrupting replication stress pathways or silencing the transcriptional activity, represents promising ways for cancer treatment This review also highlights the potential of circulating ecDNA in liquid biopsies as a non-invasive diagnostic and monitoring tool. Despite these advances, challenges such as standardizing detection methods and understanding ecDNA diverse roles across cancer types remain. Addressing these gaps will unlock new opportunities in precision oncology, making ecDNA a cornerstone of future cancer diagnostics and therapeutics.
Bangladesh J Medicine 2025; 36(2): 82-91">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc548fd944fe0c719676cbeb264f28cf2ce64996" target='_blank'>
              The Multifaceted Role of Extrachromosomal DNA in Cancer: Functions and Clinical Implications
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-05-05</td>
          <td>Bangladesh Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chemotherapeutic resistance is a major obstacle to chemotherapeutic failure. Cancer cell resistance involves several mechanisms, including epithelial-to-mesenchymal transition (EMT), signaling pathway bypass, drug efflux activation, and impairment of drug entry. P-glycoproteins (P-gp) are an efflux transporter that pumps chemotherapeutic drugs out of cancer cells, resulting in chemotherapeutic resistance. Several types of long noncoding RNA (lncRNAs) have been identified in resistant cancer cells, including ODRUL, MALAT1, and ANRIL. The high expression level of ODRUL is related to the induction of ATP-binding cassette (ABC) gene expression, resulting in the emergence of doxorubicin resistance in osteosarcoma. lncRNAs are observed to be regulators of drug transporters in cancer cells such as MALAT1 and ANRIL. Targeting P-gp expression using natural products is a new strategy to overcome cancer cell resistance and improve the sensitivity of resistant cells toward chemotherapies. This review validates the inhibitory effects of natural products on P-gp expression and activity using in silico molecular docking. In silico analysis showed that Delphinidin and Asparagoside-f are the most significant natural product inhibitors of p-glycoprotein-1. These inhibitors can reverse multi-drug resistance and induce the sensitivity of resistant cancer cells toward chemotherapy based on in silico molecular docking. It is important to validate that pre-elementary docking can be confirmed using in vitro and in vivo experimental data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcff0bc9887f04614dc5ee21157374eef0c56686" target='_blank'>
              Discovery of drug transporter inhibitors tied to long noncoding RNA in resistant cancer cells; a computational model -in silico- study
              </a>
            </td>
          <td>
            Mohanad Diab, Amel Hamdi, Feras N. Al-Obeidat, Wael Hafez, Ivan Cherrez-Ojeda, Muneir Gador, Gowhar Rashid, S. F. Elkhazin, Mahmad Anwar Ibrahim, Tarek Farag Ismail, S. S. Alkafaas
          </td>
          <td>2025-04-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4397c5f80ce07fe2ffd98add28275e15180327b" target='_blank'>
              Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A
              </a>
            </td>
          <td>
            Charlotte K Brierley, B. Yip, G. Orlando, J. Wen, S. Wen, H. Goyal, M. Levine, G. M. Jakobsdottir, Avraam Tapinos, A. Cornish, A. Rodriguez-Romera, A. Rodriguez-Meira, M. Bashton, A. Hamblin, SA Clark, Joseph C Hamley, Olivia Fox, M. Giurgiu, Jennifer O'Sullivan, L. Murphy, A. Adamo, Aude-Anais Olijnik, A. Cotton, Emily Hendrix, S. Narina, S. Pruett-Miller, A. Enshaei, Claire N Harrison, Mark Drummond, S. Knapper, A. Tefferi, I. Antony-Debré, James O. J. Davies, A. Henssen, S. Thongjuea, David C Wedge, S. Constantinescu, E. Papaemmanuil, B. Psaila, John D. Crispino, Adam J Mead
          </td>
          <td>2025-06-09</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="The DNA damage response comprises a set of imperfect pathways that maintain cell survival following exposure to DNA damaging agents. Cancers frequently exhibit DNA repair pathway alterations that contribute to their intrinsic genome instability. This, in part, facilitates a therapeutic window for many chemotherapeutic agents whose mechanisms of action often converge at the generation of a double-strand DNA break. The development of therapy resistance occurs through countless molecular mechanisms that promote tolerance to DNA damage, often by preventing break formation or increasing repair capacity. This review broadly discusses the DNA damaging mechanisms of action for different classes of chemotherapeutics, how avoidance and repair of double-strand breaks can promote resistance, and strategic directions for counteracting therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77a69284f3f1f8221d2572829a8005590b029476" target='_blank'>
              DNA repair and the contribution to chemotherapy resistance
              </a>
            </td>
          <td>
            K. Nešić, Phoebe Parker, Elizabeth M. Swisher, John J Krais
          </td>
          <td>2025-05-26</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In the progression from Inflammatory Bowel Disease to associated cancer, the clonal mutational landscape shifts from selection of mutations in inflammatory genes to selection of cancer-driver mutations1–4. How prevalence and expansion of either type of mutated clones could be impacted by the cellular environment in which they arise, and how this affects the neoplastic outcome of colitis is unknown. Here, we combine in vivo lineage tracing, in-silico modelling, mutational profiling and spatial transcriptomics in a mouse model of colitis-associated tumorigenesis to capture clone fates associated with chronic inflammation. We identify epithelial- and immune-enriched neighbourhoods and propose a model in which establishment of a reparative tissue environment facilitates tumours initiation by promoting the selection and expansion of pro-oncogenic clones, reducing the span of inflammation-resistant neighbourhoods containing non-oncogenic clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b08041bfbc2e608a3daf26d2af9f4a68fced7b6" target='_blank'>
              Paired mutation calling and spatial transcriptomics identify cellular neighbourhoods dictating the neoplastic outcome of colitis
              </a>
            </td>
          <td>
            Doug Winton, Elisa B. Moutin, Linus Chang, Giada Giavara, Shenay Mehmed, Mathilde Colombé, Filipe C Lourenço, Maria-Nefeli Skoufou-Papoutsaki, Richard Kemp, Philippe Gascard, T. Tlsty, David Tourigny
          </td>
          <td>2025-05-08</td>
          <td>Research Square</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d7251ffb7f87367d6e05c4f72a28b57671f1431" target='_blank'>
              Senescence-Associated Chromatin Rewiring Promotes Inflammation and Transposable Element Activation
              </a>
            </td>
          <td>
            A. Dalgarno, Shane A. Evans, M. Kelsey, Thomas A. Nunez, Azucena Rocha, Kelly Clark, J. Sedivy, Nicola Neretti
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute over half of the human genome and have played a profound role in genome evolution. While most TEs have lost the ability to transpose, many retain functional elements that serve as drivers of genome innovation, including the emergence of novel genes and regulatory elements. Recent advances in experimental and bioinformatic methods have provided new insights into their roles in human biology, both in health and disease. In this review, we discuss the multifaceted roles of TEs in haematopoiesis, highlighting their contributions to both normal and pathological contexts. TEs influence gene regulation by reshaping gene-regulatory networks, modulating transcriptional activity, and creating novel regulatory elements. These activities play key roles in maintaining normal haematopoietic processes and supporting cellular regeneration. However, in haematological malignancies, TE reactivation can disrupt genomic integrity, induce structural variations, and dysregulate transcriptional programmes, thereby driving oncogenesis. By examining the impact of TE activity on genome regulation and variation, we highlight their pivotal roles in both normal haematopoietic processes and haematological cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae81192284f686304fb88058cdc41885e0ae02d1" target='_blank'>
              Transposable elements as genome regulators in normal and malignant haematopoiesis
              </a>
            </td>
          <td>
            D. Prokopov, Hale Tunbak, Eve Leddy, Bryce Drylie, Francesco Camera, Özgen Deniz
          </td>
          <td>2025-05-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6f92731e4153548f5adf8b4c57b2dcf0276aa2" target='_blank'>
              Single-cell transcriptomics uncovers that cell-to-cell communication controls p53 activation
              </a>
            </td>
          <td>
            L. Castellano, Mark E Robinson, Aleksandra Dabrowska, Niklas Feldhahn, S. Ottaviani, Justin Stebbing
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, Oren Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background Non-small cell lung cancer (NSCLC) is a significant global health challenge, with 2% of cases fuelled by RET rearrangements. RET inhibitors (RETi) have revolutionized treatment for these patients, but resistance remains an important clinical challenge limiting therapy effectiveness. This study investigated the mechanisms underlying resistance to RETi. Methods NSCLC cells were exposed to increasing doses of RETi (pralsetinib/BLU-667 and selpercatinib/LOXO-292) to generate resistant cells. RNA-Sequencing analysis identified differentially expressed genes in resistant versus sensitive cells, followed by in vitro and in vivo functional assays to explore novel therapeutic strategies. Additionally, tumor biopsies from RET-rearranged NSCLC patients who exhibited cancer progression on RET inhibitor therapy were analyzed. Results RNA-sequencing analysis revealed the upregulation of the EGFR signaling pathway and hyperactivation of AP1 complex members in resistant cells compared to sensitive cells. Silencing of EGFR and AP1 complex members significantly reversed drug resistance, whereas EGFR overexpression reduced the sensitivity of parental Lc2/AD cells to RET inhibitors. Furthermore, the combination of RET and EGFR inhibitors showed synergistic antitumor activity in vitro and hindered tumor growth in mouse models with resistant cell xenografts. Notably, we observed a significant increase in EGFR expression in tumor biopsies from NSCLC patients treated with RET inhibitors who experienced disease progression, further validating the clinical relevance of our findings. Conclusions This study elucidates EGFR's role in mediating resistance to RET inhibitors in NSCLC patients. These findings offer insights into therapeutic adaptation and explore personalized combinations of RET and EGFR inhibitors for improved clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03392-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a7ddba9885e41061d39a28d4851cc532e72ca2d" target='_blank'>
              AP1-mediated reprogramming of EGFR expression triggers resistance to BLU-667 and LOXO-292 in RET-rearranged tumors
              </a>
            </td>
          <td>
            Daniela Esposito, C. M. Ascione, S. Belli, F. Napolitano, A. Servetto, Felice Pepe, U. Malapelle, A. Iaccarino, G. Troncone, Diletta Barone, Emilio Bria, Roberto Ferrara, Daniele Lorenzini, G. Lo Russo, Maria-Rosa Ghigna, A. Marinello, M. Aldea, B. Besse, Luigi Formisano, Roberto Bianco
          </td>
          <td>2025-05-22</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4bfa20b6a77bb42c88c8a026bd97ffdcef4f25" target='_blank'>
              Epimutations driven by RNAi or heterochromatin evoke antifungal drug resistance in human fungal pathogens
              </a>
            </td>
          <td>
            Ye-Eun Son, Joseph Heitman
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ea5280910649de8b63e584d7975f22a13b00b4" target='_blank'>
              Characterization of Somatic Mutations in Human tRNA Genes Reveals Tumor-Specific Mutational Loads, and the Generation of tRNA Variants that Alter the Genetic Code
              </a>
            </td>
          <td>
            Marina Murillo-Recio, Marina Salvadores, Adrián Gabriel Torres, Lina Tsapanou, Aina Vaquer Picó, F. Supek, L. R. de Pouplana
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c29c85da72e04b5001274eb1b2f06a493d988d5" target='_blank'>
              Reprogramming of Cellular Plasticity via ETS and MYC Core-regulatory Circuits During Response to MAPK Inhibition in BRAF-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            H. Lee, Zhao Zheng, Alexey Sorokin, Chi Wut Wong, Stefania Napolitano, S. Chowdhury, Preeti Kanikarla, , Veena Kochat, Christopher Bristow, Sanjana Srinivasan, Mike Peoples, Emre Arslan, J. Alshenaifi, O. Villarreal, V. Morris, J. P. Shen, F. Meric-Bernstam, Abhinav K. Jain, Natalie W Fowlkes, Amanda Anderson, D. Menter, A. Saw, Kunal Rai, S. Kopetz
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Osteosarcoma (OS), the most prevalent primary bone malignancy in adolescents, is typically treated with cisplatin-based chemotherapy. However, the development of cisplatin resistance often leads to relapse or metastasis, significantly impairing therapeutic efficacy. To tackle this issue, patient-derived osteosarcoma organoids (OSOs) is established that accurately reflect the cellular composition and heterogeneity of the original tumors, as validated by single-cell RNA sequencing, bulk RNA sequencing, and histopathology analysis. Cisplatin resistance is successfully induced in these OSOs, creating a clinically relevant model for investigating chemoresistance. Utilizing RNA sequencing in cisplatin-resistance OSOs and CRISPR screening in OS cell line, ERCC6 is identified as a pivotal regulator of cisplatin resistance. Knockdown of ERCC6 markedly enhanced cisplatin sensitivity in vitro and in vivo. Mechanistically, ERCC6 interacts with HNRNPM, influencing the PI3K/AKT signaling pathway and alternative splicing of pre-mRNA for BAX. Notably, the knockdown of ERCC6 and HNRNPM increased expression of full-length BAX and reduced skipping of exon 2, thus promoting apoptosis. This exon skipping in BAX results in a frameshift and introduces a premature stop codon (TGA) within the BH3 domain. These findings underscore the utility of OSOs in elucidating resistance mechanisms and highlight ERCC6 and HNRNPM as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b1d547a41283aa903c6904283e207311a6e9007" target='_blank'>
              A Dual Approach with Organoid and CRISPR Screening Reveals ERCC6 as a Determinant of Cisplatin Resistance in Osteosarcoma.
              </a>
            </td>
          <td>
            Ruiling Xu, Sai Zhu, Wenchao Zhang, Haodong Xu, Chao Tu, Honghui Wang, Lu Wang, Na He, Tang Liu, Xiaoning Guo, Xiaolei Ren, Zhihong Li
          </td>
          <td>2025-06-06</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Guanine-rich DNA sequences have a propensity to form G-quadruplex structures. These structures play several important biological roles and are potential targets for anticancer drugs. However, no G-quadruplex-stabilizing agent has yet been approved for clinical use. Given that G-quadruplex stabilization is quite promising as a mechanism for novel anticancer therapies, it is crucial to elucidate its effects on healthy human cells. In our study, we modeled a potential human treatment using G4 -stabilizing agents and analyzed their effects on genome integrity, transcriptomic changes, and mitochondrial function focusing on non-cancerous cells to predict potential side effects of such treatments. We found that G-quadruplex stabilization does not compromise genome integrity. However, it can induce persistent alterations in the transcriptomic profile of human cells, including genes encoded on the mitochondrial genome. Notably, certain G-quadruplex-stabilizing agents triggered mitophagy in both human cells and Drosophila melanogaster. In summary, our findings indicate that while G-quadruplex stabilization does not cause genome instability, it may pose potential risks due to its long-term effects on transcription and its ability to induce mitophagy. Therefore, we recommend that all potential drug candidates be thoroughly evaluated for their ability to induce mitophagy and to promote cancer formation in animal models prior to clinical trials. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-04791-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277b0abab64e6f6a40e0606dd3b857caf1865195" target='_blank'>
              The effect of prolonged G-quadruplex stabilization on the functions of human cells
              </a>
            </td>
          <td>
            Nargis Karatayeva, Lili Hegedűs, Arindam Bhattacharjee, Eszter Németh, Ádám Póti, Lorinc Pongor, Gábor Juhász, David Szuts, Peter Burkovics
          </td>
          <td>2025-06-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA damage is a key factor in many human disorders, including cancer, chronic inflammation, and early aging. Genes involved in DNA damage repair (DDR) not only maintain genomic stability but also contribute to immune responses by regulating the expression of antimicrobial peptides and ligands that activate immune receptors. Various components of the DNA damage response (DDR), including DNA damage sensors, transducer kinases, and effector proteins, are capable of activating diverse immunological signaling pathways. While DDR gene mutations are common in cancers, their effects on immune characteristics are not well understood. We investigated how mutations in DDR genes influence the expression of immune regulatory genes, including immune stimulators, inhibitors, and genes related to the major histocompatibility complex (MHC) pathway. Using gene expression data from The Cancer Genome Atlas (TCGA) and mutation data from cBioPortal, we analyzed 264 DDR-related genes and 66 immune regulatory genes. These genes were clustered and categorized using Metascape, an integrative bioinformatics tool that applies enrichment-based analysis to group functionally related genes into clusters. The clustered genes were further validated through a literature review and the GeneCards database. We scored the change in immune regulatory gene expression in response to DDR gene mutations to identify differentially expressed immune stimulators, inhibitors, and MHC-related genes. Our analysis revealed positive and negative correlations between DDR gene mutations and the expression of immune modulators. These findings could help guide future cancer treatments based on biomarkers and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-99965-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5ec8884f344e1ff8037b8e2ddb589a5a344f4" target='_blank'>
              Pancancer analysis of DNA damage repair gene mutations and their impact on immune regulatory gene expression
              </a>
            </td>
          <td>
            Kanchana Yadav, Trishala Das, Andrew M Lynn
          </td>
          <td>2025-05-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3ab807859eab4d5f327f3a532083765a9ef4e89" target='_blank'>
              High-throughput single-cell CRISPRi screens stratify neurodevelopmental functions of schizophrenia-associated genes
              </a>
            </td>
          <td>
            Umut Yildiz, A. Claringbould, Mikael Marttinen, Víctor Campos-Fornés, Mantha Lamprousi, Manu Saraswat, Mathias Saver, Daria Bunina, Michael W. Dorrity, Judith B. Zaugg, K. Noh
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer is a complex genetic disease driven by somatic and germline mutations that alter critical cellular functions. With the advancement of Next-Generation Sequencing (NGS), researchers can now identify these mutations at high throughput; however, interpreting their biological and clinical relevance requires robust functional annotation. This study investigates the mutational landscape of prostate, breast, and pancreatic cancers using public RNA-seq datasets and an open-source bioinformatics pipeline. Key oncogenic drivers and tumor suppressor genes, including BRCA1, BRCA2, TP53, and KRAS, were analyzed using tools such as FastQC, Bowtie2, Samtools, FreeBayes, and SnpEff. Our findings highlight the predominance of missense mutations and frequent transition substitutions such as C→T and G→A, with silent and nonsense mutations also contributing to disease mechanisms. These insights emphasize the utility of integrative computational tools in variant annotation and their potential to enhance cancer diagnostics, prognostics, and targeted therapy selection in precision oncology">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0c74326956f2ce86b7c2f753a11ac5532d41d2" target='_blank'>
              Functional Annotation of Cancer Variants: Tools And Techniques
              </a>
            </td>
          <td>
            Alka Manisha, Jyoti Prakash, Ruchi yadav, Rachna Chaturvedi, Akanksha Pandey
          </td>
          <td>2025-05-14</td>
          <td>International Journal of Advanced Research in Science, Communication and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Homologous recombination deficiency (HRD) is associated with sensitivity to platinum-based chemotherapy and PARP inhibitors in BRCA-associated cancers, including ovarian, breast, prostate, and pancreatic cancers. This study explores HRD and homologous recombination repair (HRR) gene alterations in a pan-cancer cohort to guide precision oncology. Methods Clinical and genomic data from 9,262 patients with 17 solid tumor types were analyzed using the OncoScreenTM Plus kit. HRD scores, biallelic HRR and tumor suppressor gene alterations, and their clinical correlations were evaluated. Results HRD scores varied across cancer types, all showing a long tail in distribution. The prevalence of pathogenic alterations in pan-cancer HRR was 21.3%, with 13.7% of the cases having an HRD score ≥42. HRD-related events (LOH, LST, and TAI) exhibited similarities and cancer-specific patterns at the chromosomal arm level. Biallelic loss of HRR genes, especially BRCA1, BRCA2, RAD51D, RAD51 C, and PPP2R2 A was linked to higher HRD scores in BRCA-associated cancers, while BARD1, RAD51D, RAD54L, BRCA1, and MRE11 were associated with elevated HRD scores in in other cancer types (non-BRCA cancers). TP53 biallelic alterations, with or without HRR alterations, were linked to increased HRD scores. Higher HRD scores were associated with late-stage, older, metastatic, PD-L1 positive, non-MSI-H/non-POLE samples were correlated with genomic instability indexes, such as structural chromosomal instability (SCIN), weighted genome instability index (WGII), and whole-genome doubling (WGD). Conclusions This is the largest pan-cancer HRD study in an Asian population, providing insights for future HRD testing and targeted therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14267-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e5299d0b32b34fcf922057e19238ea118c6064" target='_blank'>
              Pan-cancer analysis of homologous recombination deficiency and homologous recombination repair–associated gene alterations in solid tumors from a large Asian cohort
              </a>
            </td>
          <td>
            Lili Ren, Runsi Yao, Ting Hou, Chenglin Liu, Fei Zhao, Xiaojun Chen, Zhou Zhang, Yan Huang
          </td>
          <td>2025-05-26</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc1dc109a3a473a99af4a77b675644d925cf4104" target='_blank'>
              Oncogenic and tumor-suppressive forces converge on a progenitor-orchestrated niche to shape early tumorigenesis
              </a>
            </td>
          <td>
            José Reyes, Isabella S. Del Priore, Andrea C. Chaikovsky, Nikhita Pasnuri, Ahmed M. Elhossiny, Tobias Krause, Andrew Moorman, Catherine Snopkowski, Meril Takizawa, Cassandra Burdziak, Nalin Ratnayeke, Ignas Masillioni, Yu-jui Ho, R. Chaligné, P. Romesser, A. Filliol, T. Nawy, John P. Morris, Zhen Zhao, M. P. di Magliano, Direna Alonso-Curbelo, D. Pe’er, Scott W. Lowe
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26fe076a51867bc46bb00941ab4e3aefb2a22ac5" target='_blank'>
              A novel Hox gene promoter fuels the evolution of adaptive phenotypic plasticity
              </a>
            </td>
          <td>
            Shen Tian, Bonnie Lee, T. Banerjee, S. Murugesan, Antónia Monteiro
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Colorectal carcinogenesis involves dynamic interactions between genetic susceptibility and cellular heterogeneity, yet current studies rarely disentangle causal genes from passive associations. While GWAS have mapped numerous risk loci, only a minority colocalize with eQTL/pQTL. A multi-omics framework combining single-cell transcriptomics, transcriptomics, proteomics, and MR is urgently needed to resolve cell-type-specific drivers of colorectal cancer pathogenesis. Methods We integrated GWAS data, eQTL data, pQTL data, and single-cell RNA sequencing differential gene expression profiles from public databases. Subsequent batch Two-sample Mendelian randomization and further SMR analysis aimed to identify key genes in the pathogenesis of colorectal cancer. Results Cluster analysis identified 4909 DEGs across various cell types. We discovered that 428 DEGs had a causal association with colorectal cancer through eQTL, of which 38 genes met the FDR statistical standards, and four of these genes (CTSF, PCSK7, LYZ, LMAN2L) also had causal associations through pQTL. SMR analysis confirmed the reliability of PCSK7 as a disease target. Conclusion By integrating single-cell data, transcriptomic data, proteomic data and GWAS data for MR analysis, we identified CTSF, PCSK7, LYZ, LMAN2L as potential targets for colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02636-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19765189a791bce76c8d451a55c97d45a6c9279a" target='_blank'>
              Integrating single-cell with transcriptome-proteome Mendelian randomization reveals colorectal cancer targets
              </a>
            </td>
          <td>
            Song Wang, Xin Yao, Shenshen Li, Shanshan Wang, Xuyu Huang, Jing Zhou, Xiao Li, Jieying Wen, Weixuan Lan, Yunsi Huang, Hao Li, Yunlong Sun, Xiaoqian Zhao, Qiaoling Chen, Xuedong Han, Ziming Zhu, Xinyue Zhang, Tao Zhang
          </td>
          <td>2025-05-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 4116


 Background:
 Biliary tract cancers (BTC) are rare, highly aggressive malignancies with limited treatment options, leading to consistently poor outcomes. An improved understanding of BTC tumor evolution could inform enhanced screening strategies, identify useful prognostic markers, and discover novel therapeutic targets.
 Methods:
 We performed whole genome and transcriptome sequencing (WGTS) at high depth ( > 80X) on a prospective cohort of BTC tumors. After detecting mutations and mutational signatures, we developed two novel methods for driver identification and evolutionary reconstruction. First, we created new oncogene and tumor suppressor-specific models that integrate copy number profiles, structural variant breakends, and expression changes to distinguish structural drivers from neutral chromosomal rearrangements. Second, we applied population genetics techniques to fit demographic models to tumor allele frequencies across 20 paired primary and metastatic samples.
 Results:
 We analyzed 130 tumor samples from 110 patients, representing the largest BTC whole-genome cohort to date. We identified hypermutated tumors (50-150 mutations/mb) with distinct etiologies, including mismatch repair deficiency, platinum exposure, tobacco use, and aristolochic acid-related damage, the latter of which was associated with response to immunotherapy. Ourintegrated-driver approach identified an association between selection on
 RAD23A
 and an increased structural variant load, resulting in a tandem-duplicator-like mutational phenotype. This also revealed an underappreciated impact of
 SMAD4
 in BTC, which is inactivated through multiple mutation types in 12%. Notably,
 BAP1
 mutations occurred in 24% of cases, including 4% that were inactivated through deletion of the
 BAP1
 promoter that lowered transcript expression, a previously undescribed mechanism. By pairing clinical data to the genomics, we observed a co-occurrence of
 BAP1
 mutations and
 FGFR2
 fusions in small-duct, mass-forming intrahepatic cholangiocarcinomas. These mutations were mutually exclusive with
 TP53
 mutations, which were enriched in patients with primary sclerosing cholangitis. Finally, our novel method for subclonal population reconstruction on paired samples illuminated recent tumor evolutionary dynamics and identified an
 ARID1B
 fusion with a potential role in metastasis.
 Conclusions:
 This study uncovered novel genomic mechanisms underlying the evolutionary origins of BTC beyond those previously identified with exome and panel sequencing, highlighting the value of WGTS. These results highlight the complex genomic heterogeneity of BTC, with potential implications for precision therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bbdd5f85cf47a722dc40e8dbc7f432d73cb4b4" target='_blank'>
              Whole-genome sequencing of biliary tract cancer: Uncovering the genomic origins of evolutionary trajectories.
              </a>
            </td>
          <td>
            Felix Beaudry, Nicholas Light, Duhan Yendi, A. Zorigtbaatar, Maggie Hildebrand, A. Dodd, Roxana Bucur, Julie M. Wilson, Enrique Sanz Garcia, Xin Wang, Erica S. Tsang, G. O’Kane, Steven Gallinger, Arndt Vogel, Jennifer J Knox, F. Notta, G. Sapisochin, Robert C Grant
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Constitutional mismatch repair deficiency (CMMRD) is a rare and highly aggressive cancer predisposition syndrome caused by biallelic germline mutations in mismatch repair genes. This condition is characterized by early-onset malignancies across multiple organ systems, including central nervous system tumors, hematological cancers, and gastrointestinal malignancies. CMMRD-associated tumors exhibit hypermutation and microsatellite instability, resulting in a high tumor mutation burden and rendering these malignancies responsive to immune checkpoint inhibitors (ICIs). ICIs targeting programmed cell death protein-1 and programmed cell death ligand 1 have demonstrated remarkable efficacy, particularly in hypermutated tumors, providing durable responses and improving survival outcomes. Advances in genetic and molecular diagnostics have enhanced the ability to identify CMMRD early, allowing for the implementation of comprehensive surveillance programs and improved management strategies. A multidisciplinary and individualized approach is essential for managing CMMRD patients. This review underscores the importance of early diagnosis, surveillance, and emerging therapeutic approaches to improve outcomes and quality of life for individuals and families affected by this devastating syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf20191865645f39f87803bf02b422fa40cafb91" target='_blank'>
              Constitutional Mismatch Repair Deficiency, the Most Aggressive Cancer Predisposition Syndrome : Clinical Presentation, Surveillance, and Management
              </a>
            </td>
          <td>
            Eungu Kang, Jin Kyung Suh, Sang-Dae Kim
          </td>
          <td>2025-04-29</td>
          <td>Journal of Korean Neurosurgical Society</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction and Objective: Genetic variation and its interaction with environmental cues are crucial in the etiology of metabolic diseases, which are strongly linked to adipocyte dysfunction and are highly cell state- and context-specific.
 Methods: Here, we leverage a population-scale biobank (CellGenBank) to conduct pooled natural genetic variation screens in primary human adipocyte villages for single-nucleus transcriptomic and chromatin accessibility profiling under various disease-relevant stimuli.
 Results: Processing 338k nuclei from 118 donors allowed us to identified key cell states characterized by canonical marker genes, including quiescent (PDGFRA) and proliferative (PDGFRA, CDK1, AURKB) adipose tissue-derived mesenchymal stem cells (AMSCs), structural Wnt-regulated adipose tissue-resident (SWAT; DCN, PLAC9, APOD) cells and adipogenic (ADIPOQ, PLIN1) cells, with distinct transcriptional responses to stimuli. By leveraging individual polygenic risk scores, we identified correlations between disease risk and shifts in cell state proportions. These states have been also mapped to metabolic disease relevant traits using single-cell heritability analysis, with body mass index highly enriched in AMSCs, whereas traits informative of metabolic health show strong enrichment in adipogenic cells and a subpopulation of SWAT cells. Furthermore, genome-wide eQTL mapping revealed hundreds of context- and state-specific eQTLs that were enriched in predicted gene-enhancer/promoter regions, linking these variants to their functional transcripts and chromatin accessibility.
 Conclusion: Collectively, the human adipocyte village approach coupled with single-nucleus functional genomics enabled the discovery of genetic mechanisms underlying metabolic diseases, highlighting its high potential as a tool in the development of effective therapeutic targets. (*Yi Huang and Joaquin Perez-Schindler contribute to this work equally)


 Y. Huang: None. J. Perez-Schindler: None. S. Datta: None. B. Min: None. N. Nambrath: None. M. Murali: None. H. Dashti: None. B. Sharma: None. S. SinghPoma: None. P. Kubitz: None. W. Qiu: None. R. Andersson: None. T. Jones: None. M. Claussnitzer: None.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f1a482f8b4cba32761087321e7d04e821730314" target='_blank'>
              2126-LB: Mapping Functional Genetic Variants and Disease Heritability on Gene Regulatory Programs in Human Adipocyte Villages under Metabolic Disease–Relevant Stimuli
              </a>
            </td>
          <td>
            YI Huang, Joaquín Pérez-Schindler, Sanchari Datta, Bellis Min, Nivedita Nambrath, Mayank Murali, Hesam Dashti, Bandana Sharma, Sunita Singhpoma, Phil Kubitz, Wei-Lin Qiu, Robin Andersson, Thouis Jones, Melina Claussnitzer
          </td>
          <td>2025-06-13</td>
          <td>Diabetes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations in the p53 gene are frequently observed in various cancers, prompting the initiation of efforts to restore p53 function as a therapeutic approach several decades ago. Nevertheless, only a limited number of drug development initiatives have progressed to late-stage clinical trials, and to date, no p53-targeted therapies have received approval in the USA or Europe. This situation can be attributed primarily to the characteristics of p53 as a nuclear transcription factor, which lacks the conventional features associated with drug targets and has historically been considered “undruggable”. In recent years, however, several promising strategies have emerged, including the enhanced iterations of previous approaches and novel techniques aimed at targeting proteins that have traditionally been considered undruggable. There is a growing interest in small molecules that can restore the tumor-suppressive functions of mutant p53 proteins, and the development of drugs specifically designed for particular p53 mutation types is currently underway. Other approaches aim to deplete mutant p53 or exploit vulnerabilities associated with its expression. Additionally, genetic therapy strategy and approaches have rekindled interest. Advances in mutant p53 biology, compound mechanisms, treatment modalities, and nanotechnology have opened up new avenues for p53-based therapies. However, significant challenges remain in clinical development. This review reassesses the progress in targeting p53-mutant cancers, discusses the obstacles in translating these approaches into effective therapies, and highlights p53-based therapies via nanotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811541704b1e89bbef8a11450dee5f1b59768d70" target='_blank'>
              Discovery of Drugs Targeting Mutant p53 and Progress in Nano-Enabled Therapeutic Strategy for p53-Mutated Cancers
              </a>
            </td>
          <td>
            Na Zhang, Zhiyuan Jing, Jie Song, Qiyue Liang, Yuxue Xu, Zhaowei Xu, Longping Wen, Pengfei Wei
          </td>
          <td>2025-05-26</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="As a predominant oncogenic driver in non-small cell lung cancer (NSCLC), EGFR frequently undergoes amplification or mutation, with EGFR-tyrosine kinase inhibitors (EGFR-TKIs) like gefitinib and erlotinib constituting frontline therapy for advanced EGFR-mutant cases. However, both primary and acquired resistance significantly limit clinical efficacy. Here, we revealed that glutathione metabolic pathway controlled by glutathione peroxidase GPX2 was abnormally activated in gefitinib-resistant A549 and HCC827-GR cell lines. Mechanistically, GPX2 triggers Hedgehog signaling activation through releasing GLI transcriptional regulator, promoting cancer stem cell (CSC) characteristics and TKI resistance. Notably, N6-methyladenosine (m6A) modification on GPX2 mRNA mediated by METTL14 diminished its stability. In vivo, GPX2 deletion constrained glutathione metabolism and boosted the effectiveness of TKI in cell line-derived xenograft models. Collectively, these findings demonstrate that GPX2 serves as a positive regulator of both primary and acquired EGFR-TKI resistance and could be a promising therapeutic target for precise treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50746db76f6a5070ba253127ed90d322b08f1eb2" target='_blank'>
              N6-methyladenosine-modified GPX2 impacts cancer cell stemness and TKI resistance through regulating of redox metabolism
              </a>
            </td>
          <td>
            Xu Yang, Long Yu, Miaomiao Shao, Huiling Yang, Kangwei Qi, Gaofei He, Lanxin Wang, Di Kong, Jianxin Gu, Xiaolin Xu, Lan Wang
          </td>
          <td>2025-06-18</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 e22507


 Background:
 Currently,Breast Cancer (BC) is not considered a mismatch repair (MMR)-deficiency (MMRd)–spectrum tumor. The BC patients carrying constitutional MMR pathogenic variants (PVs) are not eligible for dedicated screening programs and risk-reducing strategies, which are addressed only to the BC individual carriers of specific Homologous Recombination Repair (HRR) variants. The MMRd status, as well as the high levels of microsatellite instability (MSI-H) that occur upon loss of the expression of MMR genes, may have direct therapeutic implications for the patients, who may benefit from the treatment with immune-checkpoint inhibitors (ICIs).
 Methods:
 This was an observational, hospital-based, cohort study to investigate the prevalence of germline MMR PVs in BC patients who were appropriate candidates for hereditary breast and/or ovarian cancer (HBOC) predisposition evaluation. BC patients were tested through NGS-based multigene panel testing including MMR-associated genes, according to clinical established criteria.
 Results:
 Between January 2016 and December 2024, 6.020 BC patients were referred for genetic counseling and HBOC predisposition evaluation. In the cohort of consecutive 5.595 BC patients tested with multigene panel testing, a subgroup of 26 patients resulted carriers of germline MMR PVs. Interestingly, 6 of 26 MMR-mutated patients (23.1%) had also a personal bilateral BC history. The median age at BC diagnosis was 55 (range 31-82). Mutation prevalence showed that
 PMS2
 was the most mutated MMR gene (10 PVs, 38.5%), followed by
 MLH1
 (6 PVs, 23.1%),
 EPCAM
 (5 PVs, 19.2%),
 MSH2
 (3 PVs, 11.5%), and
 MSH6
 (2 PVs, 7.7%). Notably, in the MMR
 -
 mutated cohort, 73.1% of BC patients (19/26) reported a family history of HRR-associated tumors (BC, ovarian/prostate and/or pancreatic tumors), predominantly BC (78.9%). Conversely, first or second-degree family members with a colorectal or endometrial cancer diagnosis, that represents the main Lynch Syndrome-associate tumors, were reported only for 3 patients (11.5%). The genetic characteristics and the clinical behavior of BC in MMR-,
 BRCA
 - and HRR non-
 BRCA
 -carriers were also collected. The variant of uncertain significance (VUS) prevalence was described.
 Conclusions:
 Our results suggested that, in women who met hereditary BC testing criteria but missed the established Lynch Syndrome clinical or testing criteria, the use of panel testing including MMR genes could lead to the identification of a subgroup of BC patients who are carriers of MMR constitutional PVs, with potential important preventive and therapeutic implications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dea6cd0e8c7a31e9b9298c77292b06af1383e9ba" target='_blank'>
              Germline evaluation of mismatch repair (MMR) genes in breast cancer patients missing Lynch syndrome criteria: Clinical behavior, preventive paths and therapeutic implications.
              </a>
            </td>
          <td>
            L. Incorvaia, L. Cortesi, T. D. Bazan Russo, Clarissa Mujacic, Emma Zattarin, A. Perez, Elena Barbieri, Maria Concetta Vitale, I. Marchi, V. Gristina, L. Insalaco, A. Carreca, C. Brando, N. Barraco, A. Galvano, D. Fanale, G. Badalamenti, V. Bazan, I. Carreca, A. Russo
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="OBJECTIVE
To identify exonic markers that could improve analytic performance characteristics of next-generation sequencing (NGS) in detecting mismatch repair deficiency (dMMR) using colorectal cancer (CRC) as a model.


METHODS
Coding sequences of a target NGS panel (~1.13 megabase) were compared between dMMR CRC and mismatch repair-proficient (pMMR) CRC in a training cohort (41 dMMR CRCs and 213 pMMR CRCs) and a validation cohort (33 dMMR CRCs and 307 pMMR CRCs) with documented mismatch repair status by immunohistochemical and/or microsatellite instability assays.


RESULTS
The dMMR CRC cases showed significantly higher insertion/deletion (indel) mutations within exonic homopolymers (homo-indels), occurring predominantly within longer repeats of 5 to 10 nucleotides (92%, P < .0001), rather than shorter repeats of 2 to 4 nucleotides seen in pMMR CRC (62%). Homo-indels in dMMR CRC were not random. Hotspot loci were consistent between the training and validation cohorts. The dMMR defined by indels within homopolymers of 5 or more nucleotides, homopolymers of 7 or more nucleotides, or a panel of hotspots all showed 100% sensitivity and specificity with a range of cutoffs.


CONCLUSIONS
We propose that this approach allows one to identify highly sensitive and specific markers for detecting dMMR CRC by NGS alone. Further studies are warranted to test whether these markers are applicable to non-CRC neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26edad68bf6dd00a6af6b5832569dd252b00f30f" target='_blank'>
              Highly sensitive and specific markers for detection of mismatch repair deficiency by next-generation sequencing.
              </a>
            </td>
          <td>
            Ming-Tseh Lin, Eric S Christenson, Aparna Pallavajjala, James R. Eshleman
          </td>
          <td>2025-05-03</td>
          <td>American journal of clinical pathology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Although EGFR tyrosine kinase inhibitors (EGFR-TKIs) are effective for EGFR-mutant lung adenocarcinoma (LUAD), resistance inevitably develops through diverse mechanisms, including secondary genetic mutations, amplifications and as-yet undefined processes. To comprehensively unravel the mechanisms of EGFR-TKI resistance, we establish a biobank of patient-derived EGFR-mutant lung cancer organoids, encompassing cases previously treated with EGFR-TKIs. Through comprehensive molecular profiling including single-cell analysis, here we identify a subgroup of EGFR-TKI-resistant LUAD organoids that lacks known resistance-related genetic lesions and instead exhibits a basal-shift phenotype characterized by the hybrid expression of LUAD- and squamous cell carcinoma-related genes. Prospective gene engineering demonstrates that NKX2-1 knockout induces the basal-shift transformation along with EGFR-target therapy resistance. Basal-shift LUADs frequently harbor CDKN2A/B loss and are sensitive to CDK4/6 inhibitors. Our EGFR-mutant lung cancer organoid library not only offers a valuable resource for lung cancer research but also provides insights into molecular underpinnings of EGFR-TKI resistance, facilitating the development of therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/254ae3285f00635c9a4bf18409c42e63470ffca9" target='_blank'>
              Basal-shift transformation leads to EGFR therapy-resistance in human lung adenocarcinoma
              </a>
            </td>
          <td>
            T. Shinozaki, Kazuhiro Togasaki, Junko Hamamoto, A. Mitsuishi, T. Fukushima, K. Sugihara, T. Ebisudani, Masahiko Okada, Ayaka Saito, Lisa Shigematsu, H. Takaoka, Fumimaro Ito, Keiko Ohgino, Kota Ishioka, Kageaki Watanabe, T. Hishima, Yutaka Kurebayashi, K. Emoto, H. Terai, S. Ikemura, I. Kawada, Keisuke Asakura, Tomoyuki Hishida, H. Asamura, Yuki Ohta, Sirirat Takahashi, Mayumi Oda, Megumu Saito, M. Matano, Kenzo Soejima, Masayuki Fujii, Koichi Fukunaga, Hiroyuki Yasuda, Toshiro Sato
          </td>
          <td>2025-05-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Abstract Adenosine deaminases acting on RNA (ADAR) enzymes constitute a natural cellular mechanism that induces A-to-I(G) editing, introducing genetic changes at the RNA level. Recently, interest in the endogenous-ADAR editor has emerged for correcting genetic mutations, consisting of a programmed oligonucleotide that attracts the native ADAR, thereby offering opportunities for medical therapy. Here, we systematically chart the scope of cancer mutations that endogenous-ADAR can correct. First, analyzing germline single nucleotide variants in cancer predisposition genes, we find that endogenous-ADAR can revert a fifth of them, reducing the risk of cancer development later in life. Second, examining somatic mutations across various cancer types, we find that it has the potential to correct at least one driver mutation in over a third of the samples, suggesting a promising future treatment strategy. We also highlight key driver mutations that are amenable to endogenous-ADAR, and are thus of special clinical interest. As using endogenous-ADAR entails delivering relatively small payloads, the prospects of delivering endogenous-ADAR to various cancers seem promising. We expect that the large scope of correctable mutations that are systematically charted here for the first time will pave the way for a new era of cancer treatment options.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a024daf32faffd9eaf4adfd22fdca5d40c18746f" target='_blank'>
              A systematic evaluation of the therapeutic potential of endogenous-ADAR editors in cancer prevention and treatment
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Ariel Dadush, Sumeet Patiyal, P. Rajagopal, Gulzar Daya, Shay Ben-Aroya, A. Schäffer, Eli Eisenberg, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-02</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="SETD2, a frequently mutated epigenetic tumor suppressor in acute leukemia, is associated with chemotherapy resistance and poor patient outcomes. To explore potential therapeutics for SETD2-mutant leukemia, we employed an integrated approach combining computational prediction with epigenetic compound library screening. This approach identified G9a inhibitors as promising candidates, capable of reversing gene expression signatures associated with Setd2 deficiency and selectively inhibiting SETD2-deficient cells. RNA-sequencing analysis revealed that G9a inhibitor significantly downregulated Myc and Myc-regulated genes involved in translation, DNA replication, and G1/S transition in Setd2-mutant cells. Further chromatin immunoprecipitation sequencing analysis showed that G9a inhibition reduced H3K9me2 levels at the long non-coding RNA Mir100hg locus, coinciding with specific upregulation of the embedded microRNA let-7a-2 in Setd2-mutant cells. Given the established role of let-7a in MYC suppression, these findings suggest a potential mechanism by which G9a inhibitors induce MYC downregulation in SETD2-mutant leukemia. Additionally, correlation analysis between computational prediction and phenotypic outcomes highlighted the MYC signature as a key predictor of drug efficacy. Collectively, our study identifies G9a inhibitors as a promising therapeutic avenue for SETD2-mutant leukemia and provides novel insights into refining drug prediction strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd7dcd6f0a91145058e060a1de0576521a64c51a" target='_blank'>
              Integrated Computational and Functional Screening Identify G9a Inhibitors for SETD2-Mutant Leukemia.
              </a>
            </td>
          <td>
            Ya Zhang, Mengfang Xia, Zhenyi Yi, Pinpin Sui, Xudong He, Liping Wang, Qiyi Chen, Hong-Hu Zhu, Gang Huang, Qian-Fei Wang
          </td>
          <td>2025-04-29</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="


 BRAF and MEK inhibitors are standard treatments for patients with BRAF V600E-mutated melanoma. However, resistance to these therapies poses a significant challenge. Understanding the molecular mechanisms underlying this resistance is critical for the development of more effective treatment strategies.



 To investigate the role of histone deacetylase 8 (HDAC8) in drug resistance in melanoma and evaluate the potential of HDAC8 degradation as a therapeutic strategy.



 We utilized overexpression vector to examine the effects of HDAC8 in melanoma cells. The proliferative, migratory, and drug sensitivity phenotypes were assessed in HDAC8-overexpressing melanoma cells. Additionally, we tested YX862, a selective HDAC8 degrader, to explore its effects on HDAC8 activity and downstream cellular processes. Acetylation of SMC3, a cohesin complex protein, was measured, and genome-wide HiC analysis was employed to assess chromatin structural changes.



 HDAC8 overexpression promoted a more proliferative and migratory phenotype in melanoma cells. Furthermore, HDAC8-overexpressing cells exhibited reduced sensitivity to BRAF and MEK inhibitors. YX862 treatment specifically degraded HDAC8, leading to increased acetylation of SMC3. Genome-wide HiC analysis revealed that changes in SMC3 acetylation altered the three-dimensional structure of chromatin, potentially contributing to drug resistance.



 These findings highlight the critical role of HDAC8 in melanoma progression and drug resistance. HDAC8 degraders like YX862 may offer a promising therapeutic approach to overcome resistance and inhibit melanoma progression by modulating key epigenetic and cellular pathways.



 Kanchan Gupta, Richard L. Bennett, Michael F. Emmons, Yufeng Xiao, Guangrong Zheng, Keiran S. Smalley, Jonathan D. Licht. HDAC8 over-expression promotes drug resistance and alters chromatin architecture in melanoma cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB419.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/babbc9f6ed9c4e4469fe6161951a6c9c5723cbc6" target='_blank'>
              Abstract LB419: HDAC8 over-expression promotes drug resistance and alters chromatin architecture in melanoma cells
              </a>
            </td>
          <td>
            Kanchan Gupta, Richard L Bennett, M. Emmons, Yufeng Xiao, Guangrong Zheng, K. Smalley, Jonathan D. Licht
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdeb410a20ddd7cc5b7551e87eaa4c74d76779b7" target='_blank'>
              A genome-scale CRISPR deletion screen in Chinese Hamster Ovary cells reveals essential regions of the coding and non-coding genome
              </a>
            </td>
          <td>
            Federico De Marco, Ivy Rose Sebastian, Antonino Napoleone, Alexander Molin, Markus Riedl, Nina Bydlinski, K. Motheramgari, Mohamed Hussein, Lovro Kramer, Thomas Kelly, Thomas Jostock, N. Borth
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Genomic instability has been increasingly recognized over the past decade as a fundamental driver of cancer initiation and progression, largely owing to its association with specific genes and cellular mechanisms that offer therapeutic potential. However, a comprehensive molecular framework that captures the interconnected processes underlying this phenomenon remains elusive. In this study, we focused on polo-like kinase 1 (PLK1), a key cell cycle regulator frequently overexpressed in diverse human tumors, to reconstruct a regulatory network that consolidates pre-existing biological knowledge exclusively related to pathways involved in genome stability maintenance and cancer. The resulting model integrates nine biological processes, 1030 reactions, and 716 molecular species to form a literature-supported network in which PLK1 serves as a central regulatory node. However, rather than depicting an isolated PLK1-centric system, this network reflects a broader and more complex architecture of interrelated genomic instability mechanisms. As expected, the simulations reproduced known behaviors associated with PLK1 dysregulation, reinforcing the well-established role of the kinase in genome destabilization. Importantly, this model also enables the exploration of additional, less-characterized dynamics, including the potential involvement of genes such as kif2c, incenp, and other regulators of chromosomal segregation and DNA repair, which appear to contribute to instability events downstream of PLK1. While these findings are grounded in mechanistic simulations and require further experimental validation, gene expression and survival analyses across tumor types support their clinical relevance by linking them to poor prognosis in specific cancers. Overall, the model provides a systemic and adaptable foundation for studying PLK1-related genomic instability, enabling both the reinforcement of known mechanisms and discovery of candidate genes and circuits that may drive tumorigenesis through compromised genome integrity across distinct cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1225ffe2d0c6c775628165cdb9bef1963448726" target='_blank'>
              A First-in-Class Therapy Targeting Transcription-Replication Conflicts Drives Synthetic Lethality in Homologous Recombination Deficient Cancers
              </a>
            </td>
          <td>
            Yi Chieh Lim, Soak-Kuan Lai, Ru Han Yang, Yong Kang Chua, Rémi Patouret, Vaishnervi Manoj, Colin Chun Xuan Soh, Benjamin Lebeau, Ning Cham, Wei Meng, Sze Keong Tey, Szu Ling Yeap, Songjing Zhang, Jessie Yong Xing Woon, Zahin Ali, Marek Mutwil, Pritpal Singh, M. Fullwood, Linda LD Zhong, Xue Ming Dong, Chiba Shunsuke, Cheng-Gee Koh, H. Li
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed7953efc8b662a0f9dd292cd306d987c8eb0ba8" target='_blank'>
              Breast Cancer Clustering Integrating Complete Gene Expression Profiles and Genetic Ancestry
              </a>
            </td>
          <td>
            Johanna Stepanian, Alejandro Mejía-Garcia, Carlos A. Orozco, Jorge Duitama
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Objective DDX17, an ATP-dependent RNA/DNA helicase, is implicated in the regulation of RNA metabolism and has been linked to tumorigenesis and metastasis in various cancers. While studies have explored the role of DDX17 in specific cancers, further research is needed to understand its mechanisms across different cancer types. Methods We leveraged several public databases, including TIMER, The Cancer Genome Atlas (TCGA), and the Genotype-Tissue Expression (GTEx), to investigate DDX17 mRNA expression across 33 types of tumors. The GEPIA2 database was utilized to assess the impact of DDX17 on overall survival (OS) and disease-free survival (DFS) in patients with these tumors. Additionally, we employed cBioPortal to examine DDX17 gene alterations in various tumor tissues. Further analysis was conducted using the R language to explore the correlation between DDX17 and a range of clinical features, including tumor microenvironment (TME), immune regulatory genes, immune checkpoints, tumor mutational burden (TMB), microsatellite instability (MSI), DNA methylation, RNA methylation, and drug sensitivity. Gene Set Enrichment Analysis (GSEA) was also applied to elucidate the molecular mechanisms mediated by DDX17. Results DDX17 showed significant differences in expression between cancer and normal tissues. Expression of DDX17 was associated with patient prognosis, TMB, MSI, and drug sensitivity in certain cancers. DDX17 is additionally involved in modulating immune functions and influencing the tumor microenvironment. Further analysis of DDX17 mutation sites and types showed that the mutation frequency was highest in endometrial cancer and the major mutations of DDX17 were missense mutations. Conclusion These findings indicated that DDX17 may be considered a potential prognostic biomarker and a promising target for novel immunotherapeutic approaches in cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02955-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c500003015dd54739438713ee9cd7419ce63da8" target='_blank'>
              Prognostic and immunological landscape of DDX17 in pan-cancer analysis: a comprehensive study
              </a>
            </td>
          <td>
            Yu Lei, Xin He, Peng Chen, Yiping Qin, Bin Ge, Yan Liu, Pu Li, Xing Wei
          </td>
          <td>2025-06-17</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Advances in cancer biology have been achieved by the identification of oncogenes and tumor suppressor genes through the remarkable progression of next-generation sequencing. New techniques, such as single-cell analysis, help uncover cancer progression and heterogeneity. Reverse genetic screenings, including methods like random mutagenesis via retroviruses, transposons, RNA interference, and CRISPR, are useful for exploring gene functions and their roles in cancer. Especially in random mutagenesis, CRISPR screening and its modifications have recently emerged as powerful tools due to their comprehensiveness and simplicity in inducing genetic mutations. Initially, CRISPR screening focused on analyzing biological phenotypes in a cell population. It has since evolved to incorporate advanced techniques, such as combining single-cell and spatial analyses. These developments enable the investigation of cell-cell and spatial interactions, which more closely mimic In Vivo microenvironments, offering deeper insights into complex biological processes. These approaches allow for the identification of essential genes involved in cancer survival, drug resistance, and tumorigenesis. Together, these technologies are advancing cancer research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC), the third most common cancer worldwide, is one of the deadliest cancers. CRC is known as a cold tumor, characterized by a low immune response that makes it difficult for immune cells to infiltrate and exhibits strong resistance to immunotherapy with checkpoint inhibition. This restricted response is largely attributed to signature gene mutations including mismatch repair (MMR) genes, KRAS, BRAF, APC, and TP53, which are also the main oncogenes in CRC. Mutated signature genes continuously upregulate abnormal signaling pathways, leading to excessive proliferation, cancer progression, and metastasis. Furthermore, it reorganizes the tumor microenvironment (TME) by recruiting immunosuppressive cells. However, the mutation can produce neoantigens that can provoke an immune response, making it a potential target for immunotherapy. In particular, cancer vaccines that leverage the strong neoantigenic properties of these mutations are considered promising for overcoming immune resistance and eliciting anti-tumor responses. In this review, we will describe signature gene mutations in CRC and focus on cancer vaccines targeting these mutations as potential therapies for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24e80e0a03a79befa516d29031109ab14c8ea3b4" target='_blank'>
              Signature Gene Mutations in Colorectal Cancer: Potential Neoantigens for Cancer Vaccines
              </a>
            </td>
          <td>
            Jaegoo Yoon, Haeun Moon, Yuna Jeon, Soo-Cheon Choe, Hyunho Yoon
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3847b0cc02af732174f2aba89f84c63f9e5ea11" target='_blank'>
              Proof-of-Concept of A DNA-Based Recording System for High-Throughput Functional Gene Screening
              </a>
            </td>
          <td>
            Shoichi Kato, Atsushi Ikemoto, Jun Isayama, Tetsuya Takimoto, Hideyuki Saya, Ken-ichi Hamada
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Mutational single base substitution (SBS) and insertion/deletion (indel; ID) signatures are characteristic patterns of somatic mutations in cancer that may reflect underlying etiologic factors or pathogenetic mechanisms, 1 for example, deamination of 5 ‐ methylcytosine to thymine (SBS1), activity of the APOBEC family of cytidine deaminases (SBS2, SBS13), DNA damage caused by reactive oxygen species (ROS; SBS18), and defects of DNA replication and repair (ID1) (https://cancer.sanger. ac.uk/cosmic/signatures). In recent years, sequencing studies of B ‐ cell precursor acute lymphoblastic leukemias (BCP ALL) have identified SBS signatures of etiological/pathogenetic importance. 2 – 8 However, much remains to be ascertained, such as ID signatures and differences in signatures between major and minor clones. We performed whole genome sequencing (WGS) of 84 pediatric BCP ALLs with high hyperdiploidy (HeH; n = 23), ETV6 :: RUNX1 ( n = 23), TCF3 :: PBX1 ( n = 9), and B ‐ other ( n = 29) to identify SBS/ID signatures and to explore differences between major clones and minor subclones. In addition, WGS and RNA ‐ seq were used to detect fusion genes, single gene rearrangements, single ‐ nucleotide variants">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7030b71d700aea527ada3adaf24955243fbb85cd" target='_blank'>
              Mutational signatures and kataegis in pediatric B‐cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Rebeqa Gunnarsson, Minjun Yang, Andrea Biloglav, Kristina B Lundin-Ström, H. Lilljebjörn, A. Castor, T. Fioretos, Linda Olsson-Arvidsson, K. Paulsson, B. Johansson
          </td>
          <td>2025-05-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Mechanisms for resistance to cytotoxic cancer drugs are dependent on dynamic changes in the biochemistry of cellular pathways, information which is hard to obtain at the systems level. We are using a deep functional proteomics implementation of CETSA (Cellular Thermal Shift Assay) revealing a range of induced biochemical responses to DNA-damaging drugs in resistant and sensitive cell lines. The Cellular Thermal Shift Assay (CETSA) was originally developed in our lab for accessing drug target engagement in intact cells (Martinez Molina, 2013), and is now widely used for this application in academia and industry. However, CETSA can also detect modulations of cellular biochemistry (Dai, 2018), e.g. effects on downstream-of-target pathways induced by cancer drugs (Liang 2021, Ramos 2024), although this application of CETSA remains relatively unexplored. Using proteome-wide CETSA we have studied the time dependent response to more than 10 DNA damaging drugs some in both resistant and sensitive cells. Responses often reflect some known pathway modulations but also novel components of drug mechanism of action and resistance. As an example, gemcitabine rapidly induced binding to ribonucleotide reductase and CETSA shifts reflecting DNA damage responses. However, after 3-8 hours the responses diverge dramatically where sensitive cells show induction of characteristic CETSA signals for early apoptosis (Ramos, 2024), while resistant cells reveal biochemical modulations reflecting transition through a distinct DNA-damage signaling state, including opening of cell cycle checkpoints and induction of translesion DNA synthesis (TLS) programs, allowing bypass of damaged DNA-adducts. The data also reveal the induction of a new program, labeled the Auxiliary DNA Damage Repair (ADDR) protein ensemble likely supporting DNA replication at damaged sites. We show that the ADDR response can be attenuated in resistant cells by an ATR inhibitor re-establishing gemcitabine sensitivity and demonstrate ATR as a key signaling node of this response. The ADDR response is also seen for several other DNA-damaging drugs (eg cisplatin and cladribine). Prominent biochemical responses are typically seen in several other cellular pathways, where ribosomal biogenesis stands out as frequently affected process for a number of these drugs (Liang, 2021). Together our data illustrate that CETSA can yield highly valuable information on modulation of downstream-of-target pathway biochemistry in cancer drug action and resistance. Proteins discovered in some specific pathway also constitute new candidate drug targets for attenuate resistance responses, as well as potential clinical biomarkers. - Martinez Molina et al. Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay. Science, 2013 341(6141):84-7. - Dai et al. Modulation of Protein-Interaction States through the Cell Cycle. Cell, 2018 173(6):1481-1494. - Liang et al. CETSA interaction proteomics define specific RNA modification pathways as key components of fluorouracil-based cancer drug toxicity. Cell Chemical Biology, 2021, S2451-9456(21)00306-8. - Ramos et al. Proteome-wide CETSA reveals diverse apoptosis-inducing mechanisms converging on an initial apoptosis effector stage focused at the peripheral region of the nucleus. Cell Reports, 2024 43(10):114784.


 Mindaugas Raitelaitis, Marc-Antoine Gerault, Smaranda Bacanu, Ying Yu Liang, Khalidah Khalid, Hai Van Le, Hui Min Vivian Teo, Jia Hui Jane Lee, Jiawen Lyu, Allison Chan, Anand Jeyasekharan, Sara Lööf, Wai Leong Tam, Nayana Prabhu, Pär Nordlund. MS-CETSA deep functional proteomics uncovers a new induced DNA-repair programs leading to resistance to DNA-damaging drugs [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB088.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6fae3dd08f79bd09723c73641aec5030e746484" target='_blank'>
              Abstract LB088: MS-CETSA deep functional proteomics uncovers a new induced DNA-repair programs leading to resistance to DNA-damaging drugs
              </a>
            </td>
          <td>
            Mindaugas Raitelaitis, Marc-Antoine Gerault, S. Bacanu, Ying Yu Liang, Khalidah Khalid, Hai Van Le, Hui Min Vivian Teo, Jia Hui Jane Lee, Jiawen Lyu, Allison Chan, Anand D. Jeyasekharan, Sara Lööf, Wai Leong Tam, N. Prabhu, Pär Nordlund
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-13</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16855fcdba1a65e13a1196ac0779031e3a7e82b5" target='_blank'>
              DNA repair under heat: DNA Polymerase λ modulates heat stress-induced mutagenesis in plants
              </a>
            </td>
          <td>
            Clair M. Wootan, John Lutterman, Nathan Springer, Xiaosa Xu, Feng Zhang
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d7a0ecc45c91a460045023891d09c9a49fcad6" target='_blank'>
              Identification and Validation of a Previously Missed Mutational Signature in Colorectal Cancer
              </a>
            </td>
          <td>
            Mariya Kazachkova, Burçak Otlu, M. Díaz-Gay, Ammal Abbasi, Sarah Moody, Sandra Perdomo, David C Wedge, Paul Brennan, Michael R Stratton, Ludmil B. Alexandrov
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892d32d1ea696e84aa4f250bf0edeb5b5584501c" target='_blank'>
              CRISPRi perturbation screens and eQTLs provide complementary and distinct insights into GWAS target genes
              </a>
            </td>
          <td>
            Sam Ghatan, Jasper Panten, Winona Oliveros, Neville E. Sanjana, John A Morris, Tuuli Lappalainen
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05df4b99ec2d4ca5afa53e36017d7bbb5db0a668" target='_blank'>
              Functional Genomic Screens Reveal RBBP4 as a Key Regulator of Cell Cycle Progression in TMZ-Resistant Glioblastoma
              </a>
            </td>
          <td>
            Ezgi Yagmur Kala, Filiz Senbabaoglu Aksu, Erdem Ercan, Altar Ozbıyık, Ahmet Cingöz, Ozlem Yedier-Bayram, A. C. Aksu, Ayse Derya Cavga, E. Yilmaz, I. Kok, A. Humeyra, Dur Karasayar, Ibrahim Kulac, Hamzah Syed, M. Philpott, AP Cribbs, T. Bagci-Onder
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Microsatellite instability (MSI) is a hallmark of mismatch repair (MMR) deficiency and characterizes a distinct subset of colorectal cancers (CRC). In parallel, telomere length dynamics have emerged as important contributors to genomic stability and tumorigenesis. However, the relationship between MSI status, MMR protein expression, and telomere maintenance remains poorly defined. This study aimed to investigate the association between MSI status and telomere length in CRC cell lines and to evaluate the expression of key MMR proteins (MLH1, MSH2, MSH6, PMS2) to elucidate molecular differences between MSI and microsatellite stable (MSS) phenotypes. A panel of CRC cell lines with known MSI and MSS statuses was used. Telomere length was quantified using real-time quantitative PCR (qPCR) based on the T/S ratio method. MSI status was confirmed via PCR using mononucleotide repeat markers. Western blotting was performed to assess protein expressions of MLH1, MSH2, MSH6, and PMS2. β-actin served as a loading control. qPCR analysis revealed that MSI cell lines exhibited significantly longer telomeres compared to MSS lines (P < 0.05). Western blot results showed reduced or absent expression of MLH1 and PMS2 in MSI cell lines, confirming MMR deficiency. In contrast, MSS cell lines maintained normal expression of all tested MMR proteins. These findings suggest a link between defective MMR function and altered telomere dynamics in MSI-CRC. MSI CRC cell lines exhibit telomere elongation and loss of key MMR proteins, highlighting distinct molecular features compared to MSS counterparts. These insights may inform future strategies for personalized CRC diagnostics and therapeutics, particularly in the context of telomere-targeted or immunomodulatory treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceeae7fc57ed1314ec1ac7e38395f2ecdc9bde9a" target='_blank'>
              Telomere Elongation and Mismatch Repair Deficiency Distinguish MSI Colorectal Cancer Cell Lines
              </a>
            </td>
          <td>
            Rajasekhar Moka, Shreya Upadhya
          </td>
          <td>2025-05-29</td>
          <td>Cancer Research Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 10504


 Background:
 Lynch syndrome (LS), caused by germline pathogenic variants (gPVs) in DNA mismatch repair genes (
 MLH1, MSH2, MSH6, PMS2
 ), is characterized by microsatellite instability (MSI)/mismatch repair deficiency (MMRd) of associated tumors. The impact of the underlying gPV on the prognosis of LS patients with advanced cancer receiving ICB has not been elucidated.
 Methods:
 Consecutive cancer-affected LS pts consented to tumor-normal DNA sequencing via MSK-IMPACT (NCT01775072), from 01/2014-01/2024 with advanced solid tumors and receiving ≥1 ICB dose were identified. In cases of multiple cancers, only the first one was considered. MSI and MMRd was assessed via NGS-derived MSISensor score or IHC staining, respectively. Radiological response rates and survival were assessed.
 Results:
 Among 186 LS patients, 132 had an ICB treated advanced tumor with the following prevalence of gPV:
 MSH2
 41% (N=54),
 MLH1
 28%

 (N=37),
 PMS2
 17% (N=23), and
 MSH6
 14% (N=18). The most represented tumor types were colorectal (35%, N=46), urothelial (14%, N=18), pancreatic/biliary (13%, N=17), upper gastro-intestinal (11%, N=15), endometrial (11%, N=15). Objective response rate was 58%, including 35% with radiological complete response (CR) with higher CR rates in g
 MLH1
 and g
 MSH2
 patients (Table). Median follow-up was 53 months [range:0.3-116.3]. Median overall survival (mOS) / Progression free survival (mPFS) were significantly prolonged in
 non-gPMS2
 patients vs. g
 PMS2
 patients (Table). MMR IHC was available for 107 (80%) tumors identifying 88% (n=94) as MMRd and 12% (n=13) as MMRp tumors. 69% (9/13) of MMRp tumors were in
 PMS2
 patients with 8 being both MMRp and MSS suggesting a sporadic origin rather than LS-related cancer. Stratifying by MMRd and MSI status, mOS was similar in patients with MMRd/MSI and MMRd/MSS tumors (109 months vs not reached (NR); HR=0.86; 95%CI [0.35-2.13]), but significantly diminished in in pts with MMRp as compared to MMRd tumors (20 vs 109 months, HR=0.27; 95%CI [0.77-0.96]). In g
 PMS2
 patients with MMRd tumors, median OS was 27.7 months.
 Conclusions:
 ICB treatment results in high CR rates in LS patients. The shorter mOS amongst LS patients with
 gPMS2
 compared to other LS genes is partially explained by the higher prevalence of sporadic MMRp tumors. Even in the context of an MMRd tumor, patients with g
 PMS2
 have worse outcomes than LS patients with other gPVs.




 Treatment
 Total
 Complete Response% (n)
 Objective Response Rate% (n)
 Disease Control rate% (n)
 mPFS (months)
 mOS (months)





 MSH2

 54
 46% (25)
 65% (35)
 89% (48)
 45
 109



 MLH1

 37
 41% (15)
 65% (24)
 81% (30)
 50
 NR



 PMS2

 23
 9% (2)
 30% (7)
 74% (17)
 11
 20



 MSH6

 18
 22% (4)
 61% (11)
 78% (14)
 109
 NR



 MLH1/MSH2/MSH6

 109
 40% (44)
 64% (70)
 84% (92)
 52
 109



 Non-PMS2
 vs PMS2

 -
 OR=7.195%CI [1.7-31.6]P=0.003
 OR=4.195%CI [1.5-10.3]P=0.005
 OR=1.995%CI [0.64-5.6]P=0.24
 HR=0.21 95%CI [0.13-0.35].P<0.0001
 HR=0.19 95%CI [0.10-0.35]P=0.0005




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21fda5f8e2362a55474e61f5ad84ed0422e3cafb" target='_blank'>
              Gene-specific outcomes in patients with Lynch syndrome treated by immune checkpoint blockade for advanced cancer.
              </a>
            </td>
          <td>
            V. Randrian, O. Artz, Mohammad Abbass, Emily Harrold, Michael B Foote, A. Cercek, D. Mandelker, Ying L. Liu, M. Carlo, Alicia J Latham, Y. Murciano-Goroff, J. Shia, Luis A. Diaz, B. Rousseau, Z. Stadler
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="In cancer genomics, there is the realization that some very frequently mutated genes like TP53 or KRAS are extremely hard to approach from a therapeutic perspective. In the realm of cancer treatment, the development of agents capable of transforming these alterations into actionable targets would be like capturing unicorns. These unicorns of oncology-drugs targeting undruggable targets that span various cancer types-offer an exciting opportunity for the development of histology-agnostic therapies. Among these, recent developments in targeting TP53 mutations, KRAS alterations, and MTAP loss mark pivotal advancements for innovative treatment strategies. TP53 mutations, which occur in nearly every cancer type, are central to tumorigenesis and contribute to resistance against conventional therapies. KRAS mutations, especially in cancers like pancreatic and lung cancers, are notoriously difficult to target but are now yielding promising therapeutic avenues. Similarly, MTAP is frequently deleted in cancers such as lung, pancreatic, and mesothelioma and plays a critical role in cellular metabolism and epigenetics, making its loss a significant vulnerability in tumor cells. Recent developments in drug discovery provide a broad foundation for the clinical testing of drugs that target common mechanisms across diverse tumor types, offering hope for a more universal approach to cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bc931554d7166b2b731a598949019062e38b61" target='_blank'>
              Capturing Unicorns: Targeting Cancers With TP53 Mutations, RAS Alterations Beyond G12C, and MTAP Loss-No Target Is Out of the Realm of Possibility.
              </a>
            </td>
          <td>
            Pooja A. Shah, K. Wiman, K. Cichowski, J. Rodon Ahnert
          </td>
          <td>2025-06-01</td>
          <td>American Society of Clinical Oncology educational book. American Society of Clinical Oncology. Annual Meeting</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="
 8602


 Background:
 MTAP
 loss leads to methylthioadenosine (MTA) accumulation, disrupting downstream metabolic pathways. Targeting synthetic lethal interactions with
 MTAP
 loss offers a promising therapeutic strategy. The DNA damage response (DDR) pathway, essential for genomic stability, is commonly dysregulated in lung cancer, impacting treatment response and prognosis. Emerging evidence has highlighted potential association between
 MTAP
 loss and mutations in DDR genes, suggesting that investigating their relationship is critical for uncovering disease mechanisms and identifying novel biomarkers and therapeutic targets.
 Methods:
 Hybridization capture sequencing with StarPanel NGS Assay (1,326 genes) was conducted on 2,258 Chinese lung cancer patients' tumor and matched peripheral blood samples. Then somatic and germline mutations, and TMB values of each sample were obtained. Based on a depth-based algorithm, differences between tissue and control samples in CNVs from the gene level were analyzed. Besides, we utilized shifts in germline heterozygous SNPs within the gene region to assist in determining if the loss was homozygous or heterozygous.
 Results:
 Homozygous and heterozygous deletions of
 MTAP
 were detected in 11.07% and 6.95% of all samples. Notably, 61.67% of these samples exhibited co-loss of
 MTAP
 ,
 CDKN2A
 and
 CDKN2B
 . The median TMB was significantly higher in samples with
 MTAP
 loss (3.85 mut/Mb) compared to those with intact
 MTAP
 (2.56 mut/Mb). In samples with homozygous
 MTAP
 loss, the top somatically co-altered genes were
 EGFR
 (76.00%),
 TP53
 (57.60%) and
 CDKN2A
 (52.80%). Gain-of-function (GoF) mutations were most prevalent in
 EGFR
 (56.80%),
 KRAS
 (12.00%) and
 MDM2
 (10.00%), while Loss-of-function (LoF) mutations were most common in
 TP53
 (50.40%),
 RBM10
 (11.60%) and
 PTEN
 (7.60%). Notably, LoF mutations in
 MTAP
 -loss samples showed a higher prevalence of DDR genes compared to
 MTAP
 -intact samples, including
 RAD50
 (2.40% vs. 0.00%,
 p
 < .0001) and
 POLQ
 (2.40% vs. 0.00%,
 p
 < .0001). Enrichment analysis further revealed LoF mutations unique to
 MTAP
 -loss samples were significantly enriched in the DDR pathway (
 p
 < .0001). For top germline pathogenic mutations, two DDR genes,
 RECQL4
 (0.80% vs. 0.38%,
 p
 = 0.29) and
 BRCA2
 (0.40% vs. 0.43%,
 p
 = 1), showed no significant differences between
 MTAP
 -loss and
 MTAP
 -intact samples.
 Conclusions:
 Potential association exists between
 MTAP
 loss and DDR pathway dysregulation, which may impact tumorigenic processes and therapeutic vulnerabilities. The enrichment of DDR-related LoF mutations indicates
 MTAP
 loss could exacerbate genomic instability by impairing DNA damage repair mechanisms, thereby increasing TMB and driving cancer progression with a distinct molecular profile. These vulnerabilities are primarily driven by somatic mutations, providing a rationale for exploring personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305ad8ea528f00d8a04f7506987172e9d5a5207d" target='_blank'>
              Dysregulation of DNA damage repair in lung cancer driven by
 MTAP
 loss: Mechanistic insights and target discovery.
              </a>
            </td>
          <td>
            Bo Jiang, W. Liao, Xin Wang, Hongliang Hui, Yan-Gui Lin, Linjie Qiu, Dan Li, Min Luo, Haoran Miao, Fan Qiu, Yiqian Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 The Cancer Dependency Map (DepMap) is an ongoing initiative to identify and explore synthetic lethal targets, by systematically linking tumor molecular alterations to gene dependencies and drug sensitivities. The resource used by scientists worldwide has significantly contributed to both basic and translational cancer research and has provided the community with a landscape view of synthetic lethal targets. Numerous targets identified through DepMap are actively under investigation, with several already advancing into clinical trials, highlighting their clinical relevance and potential. For instance, we recently discovered a synthetic lethal interaction involving the PELO-HBS1L and SKI complex, both involved in mRNA quality control. PELO is a promising synthetic lethal target in cancers with SKI complex deficiencies, such as those with FOCAD deletions or high microsatellite instability. Despite the progress, much work remains to fully exploit cancer dependencies. We have integrated next-generation patient-derived models, including organoids and neurospheres grown in 3D to better represent various cancer subtypes. Additionally, we used di-genic screening libraries to uncover synthetic lethal dependencies previously masked due to paralog redundancy. We are also actively expanding multi-omics datasets and building computational features that will enable the identification of novel molecular links to synthetic lethal dependencies. DepMap continues to grow with the ultimate goal of creating a comprehensive and detailed landscape of cancer vulnerabilities to accelerate precision cancer medicine.


 Francisca Vazquez. Mapping synthetic lethal interactions at scale: A functional genomics approach to cancer therapeutic discovery [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY14-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a986ae0bdffed64d9a4eefeb701fd4d7309816c" target='_blank'>
              Abstract SY14-02: Mapping synthetic lethal interactions at scale: A functional genomics approach to cancer therapeutic discovery
              </a>
            </td>
          <td>
            Francisca Vazquez
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Colorectal cancer is a common malignant tumor in the gastrointestinal tract, and the mechanisms of its occurrence, development, and metastasis have always been the focus of the medical community’s attention. The study of CRC genetic mechanisms began with the identification of oncogenes or tumor suppressor genes and their key pathways. With further research, researchers gradually realized that single genes or pathways alone could not explain the occurrence, development, and metastasis of CRC. The development of bulk sequencing technology has helped us to analyze the occurrence, development, and metastasis mechanisms of CRC from a multi-gene, multi-pathway, and multi-dimensional perspective, but it has not brought significant benefits to the clinical treatment of tumors. The main reason for this is that bulk sequencing technology relies on homogeneous cell grouping and cannot capture the heterogeneity between cells within the tumor and the interactions within the tumor microenvironment. The development of single-cell technology has made it possible to study the mechanisms of heterogeneity between cells within CRC and the interaction within the tumor microenvironment. This review discusses the mechanisms of CRC occurrence and development in three stages: traditional molecular biology level of single gene, bulk sequencing, and single-cell sequencing. These results show that the occurrence of CRC is the result of complex interactions between genetic and non-genetic factors in somatic cell evolution, where the heterogeneity between cells within the tumor and the tumor microenvironment are crucial for CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ab28824b4d67fc04f291dd14952095fef1aa0aa" target='_blank'>
              Research progress of colorectal cancer in genomic and transcriptomic at multi-level
              </a>
            </td>
          <td>
            Qinglian Wen, Shuangyan Han, Yongxia Cui
          </td>
          <td>2025-05-30</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Eliciting DNA damage in tumor cells continues to be one of the most successful strategies against cancer. This is the case for classical chemotherapy drugs and radiotherapy. In the modern era of personalized medicine, this strategy tries to identify specific vulnerabilities found in each patient’s tumor, to inflict DNA damage in certain cell contexts that end up in massive cancer cell death. Cells rely on multiple DNA repair pathways to fix DNA damage, but cancer cells frequently exhibit defects in these pathways, many times being tolerant to the damage. Key vulnerabilities, such as BRCA1/BRCA2 mutations, have been exploited with PARP inhibitors, leveraging synthetic lethality to selectively kill tumor cells and improving patients’ survival. In the DNA damage response (DDR) network, kinases ATM, ATR, Chk1, and Chk2 coordinate DNA repair, cell cycle arrest, and apoptosis. Inhibiting these proteins enhances tumor sensitivity to DNA-damaging therapies, especially in DDR-deficient cancers. Several small-molecule inhibitors targeting ATM/Chk2 or ATR/Chk1 are currently being tested in preclinical and/or clinical settings, showing promise in cancer models and patients. Additionally, pharmacological blockade of ATM/Chk2 and ATR/Chk1 axes enhances the effects of immunotherapy by increasing tumor immunogenicity, promoting T-cell infiltration and activating immune responses. Combining ATM/Chk2- or ATR/Chk1-targeting drugs with conventional chemotherapy, radiotherapy or immune checkpoint inhibitors offers a compelling strategy to improve treatment efficacy, overcome resistance, and enhance patients’ survival in modern oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24f87b378053b6c4aa81e36eb090c1658bdb9c58" target='_blank'>
              Cancer Vulnerabilities Through Targeting the ATR/Chk1 and ATM/Chk2 Axes in the Context of DNA Damage
              </a>
            </td>
          <td>
            Anell Fernandez, Maider Artola, Sergio Leon, N. Otegui, Aroa Jimeno, Diego Serrano, Alfonso Calvo
          </td>
          <td>2025-05-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Tumors are characterized by global changes in epigenetic modifications such as DNA methylation and histone modifications that are functionally linked to tumor progression. Accordingly, several drugs targeting the epigenome have been proposed for cancer therapy, notably, histone deacetylase inhibitors (HDACi) such as vorinostat and DNA methyltransferase inhibitors (DNMTi) such as zebularine. However, a fundamental challenge with such approaches is the lack of genomic specificity, i.e., the transcriptional changes at different genomic loci can be highly variable, thus making it difficult to predict the consequences on the global transcriptome and drug response. For instance, treatment with DNMTi may upregulate the expression of not only a tumor suppressor but also an oncogene, leading to unintended adverse effect. Methods Given the pre-treatment transcriptome and epigenomic profile of a sample, we assessed the extent of predictability of locus-specific changes in gene expression upon treatment with HDACi using machine learning. Results We found that in two cell lines (HCT116 treated with Largazole at eight doses and RH4 treated with Entinostat at 1µM) where the appropriate data (pre-treatment transcriptome and epigenome as well as post-treatment transcriptome) is available, our model distinguished the post-treatment up versus downregulated genes with high accuracy (up to ROC of 0.89). Furthermore, a model trained on one cell line is applicable to another cell line suggesting generalizability of the model. Conclusions Here we present a first assessment of the predictability of genome-wide transcriptomic changes upon treatment with HDACi. Lack of appropriate omics data from clinical trials of epigenetic drugs currently hampers the assessment of applicability of our approach in clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a8c2bc7236dcb12e89963dabb5f55fdfd61138" target='_blank'>
              Predicting gene expression changes upon epigenomic drug treatment
              </a>
            </td>
          <td>
            P. Agrawal, Vishaka Gopalan, M. A. Rumi, Sridhar Hannenhalli
          </td>
          <td>2025-05-02</td>
          <td>F1000Research</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="
 Synthetic lethality, a phenomenon in which alterations in two or more genes together induce cell death while changes in either gene alone do not, provides a promising approach for cancer treatment. Indeed, various inhibitors targeting synthetic lethal interactions are approved by the Food and Drug Administration or in clinical trials. Seeking to nominate additional synthetic lethal targets, we analyzed large-scale CRISPR knockout screening data and identified Pelota mRNA surveillance and ribosome rescue factor (PELO) as a promising therapeutic target for two independent and frequently observed molecular subtypes of cancer: biallelic deletion of chromosomal region 9p21.3 (9p21.3-/-) and microsatellite instability-high (MSI-H).
 Homozygous deletion of chromosomal region 9p21.3 is one of the most frequently observed somatic copy number alterations in human cancers, occurring in approximately 15% of all cancers. Many 9p21.3-/- cancers are associated with poor clinical outcomes including subsets of glioblastoma, mesothelioma, urothelial, pancreatic, esophageal, and non-small cell lung cancers. The primary driver of 9p21.3-/- is thought to be loss of the tumor suppressor cyclin-dependent kinase inhibitor 2A. Additionally, studies have linked the loss of the interferon gene cluster on 9p21.3 to immune evasion and primary resistance to immune checkpoint inhibitors. Extensive deletions of chromosomal regions can provide opportunities for cancer treatment. For instance, PRMT5 and MAT2A were found to be promising targets in 9p21.3-/- cancers due to synthetic lethal interactions with deletions of the 9p21.3 gene methylthioadenosine phosphorylase. While clinical trials are underway examining PRMT5 and MAT2A inhibitors, the frequency and poor outcomes of many 9p21.3−/− cancers underscores the urgent need for additional therapies to treat this diverse group of cancers. Using large-scale functional genomic datasets, we identified PELO as the top preferential dependency in 9p21.3-/- cell lines.
 We also observed that some cell lines with intact 9p21.3 (9p21.3+) were dependent on PELO for survival. Examining other genomic features, we found that these cells were characterized as MSI-H, a hypermutable state observed in subsets of colon, endometrial, gastric, and ovarian cancers. Indeed, when we compared 9p21.3+/MSI-H and 9p21.3+/microsatellite stable (MSS) cell lines, we found that PELO scored as a strong preferential dependency in MSI-H cell lines, second only to the previously described synthetic lethal target Werner helicase (WRN).
 To validate these findings, we interrogated the viability effects of PELO knockdown across a panel of cell lines representing 9p21.3+/MSS, 9p21.3-/-/MSS, or 9p21.3+/MSI-H. CRISPR interference (CRISPRi)-mediated PELO knockdown with three distinct guide RNAs impaired the viability of 9p21.3-/-/MSS cell lines and 9p21.3+/MSI-H cell lines, but not 9p21.3+/MSS cell lines. We also sought to validate PELO dependency in tumor organoid models for which pre-existing CRISPR data were unavailable, and therefore, we could only predict their dependence on PELO based on the two identified biomarkers. We demonstrated that 9p21.3-/- or MSI-H tumor organoids were preferentially dependent on PELO as compared to 9p21.3+/MSS models. We also sought to evaluate the effect of PELO knockdown in tumor maintenance in vivo. Leveraging a doxycycline-inducible CRISPRi system to suppress PELO expression, we showed that PELO knockdown significantly shrank 9p21.3-/- tumors grown on the flanks of nude mice.
 We next pursued the culprit lesion in 9p21.3-/- cancers that conferred PELO dependency. Hypothesizing that the loss of a 9p21.3 gene confers PELO dependency, we performed a focused loss-of-function CRISPR/Cas12a screen targeting 9p21.3 genes in the context of PELO knockdown or control conditions. We identified the mRNA surveillance gene FOCAD as a 9p21.3 gene whose knockout lead to preferential impairment in cells with PELO depletion. This led us to hypothesize that FOCAD deletions were conferring PELO dependency. In focused validation, we showed that FOCAD deletion sensitizes cell lines to PELO knockdown. Furthermore, we discovered that restoring FOCAD rescued 9p21.3-/- cells from PELO depletion. Together, these data demonstrated FOCAD loss was both necessary and sufficient for PELO dependency in the 9p21.3-/- context.
 Next, we sought to determine the mechanistic basis for PELO dependency in MSI-H cancers. We hypothesized that a mutation associated with MSI-H sensitizes cells to PELO loss. We observed that deletions in the splicing acceptor site at intron 29 of the superkiller complex (SKIc) member tetratricopeptide repeat domain 37 (TTC37) was the most correlated microsatellite mutation with PELO dependency. Thymidine deletions in this splicing acceptor site were predictive of decreased TTC37 protein levels in cell lines. These data led us to hypothesize that functional loss of TTC37 is the culprit lesion that renders MSI-H cancers dependent on PELO for survival. Evaluating this hypothesis, we found that TTC37 knockout sensitized cells to PELO depletion. We also showed that exogenous expression of TTC37, but not control, cDNA rescued the viability of MSI-H cell lines from PELO knockdown. Taken together, these data demonstrated that functional TTC37 impairment is sufficient and necessary for PELO dependency in the MSI-H setting and could serve as a refined predictive biomarker for PELO dependency.
 We next sought to investigate how FOCAD loss and TTC37 mutations converge to confer PELO dependency. TTC37 interacts with Ski2-like RNA helicase (SKIV2L) and WD Repeat Containing Protein 61 (WDR61) to form the SKIc, which promotes the 3’-5’ exosome degradation of mRNA from stalled ribosomes. We examined the Cancer Cell Line Encyclopedia proteomics dataset and demonstrated positive correlations between FOCAD, TTC37, and SKIV2L. Consistent with these datasets, we observed decreased TTC37 and SKIV2L protein levels in 9p21.3-/- cell lines and MSI-H cell lines. Based on these results, we asked if FOCAD was required and/or sufficient to maintain TTC37 and SKIV2L protein stability. We expressed FOCAD cDNA in 9p21.3-/- cells and observed increased TTC37 and SKIV2L protein levels. Furthermore, FOCAD knockout reduced SKIV2L and TTC37 protein levels in 9p21.3+/MSS cells. Together, these data support FOCAD as a critical regulator of SKIV2L and TTC37 stability. We hypothesized that FOCAD deletions and TTC37 mutations were capturing SKIc loss, thereby increasing reliance on PELO for survival. We reasoned that if functional loss of the SKIc was responsible for PELO dependency, SKIV2L deletion would also confer increased PELO dependency. We knocked SKIV2L out and observed that loss of SKIV2L sensitized cells to PELO loss. These data support our hypothesis that loss of SKIc function confers increased dependence on PELO for survival.
 To assess how SKIc-deficient cancer cells respond to PELO suppression, we performed gene expressing profiling. We found that SKIc-deficient cells had a robust transcriptional response with gene set enrichment analyses demonstrating upregulation of the unfolded protein response, a signaling network responding to aggregated unfolded or misfolded proteins. We validated these findings by demonstrating that PELO depletion preferentially upregulated C/ebp homologous protein (CHOP) expression and induced X-BOX binding protein (XPB1) splicing, markers of UPR activation, in the SKIc-deficient setting.
 Together, our observations reveal that MSI-H associated mutations and large 9p21.3 deletions involving TTC37 and FOCAD, respectively, independently impair the SKIc and confer a synthetic lethal relationship with PELO. Since MSI-H and large 9p21.3 deletions are frequently observed in patients, a PELO-based therapeutic could have broad implications for clinical oncology.


 Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel Lubicki, Ashir Borah, Nicholas Lofaso, Sohani Das Sharma, Riya V. Kishen, Joshua Dempster, Francisca Vazquez, Edmond M. Chan. Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY14-03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The TP53 gene encodes the critical tumor suppressor p53, which is involved in the regulation of cell cycle control, apoptosis, and genomic stability. In cancer biology, p53 holds an elevated position because high mutation rates in the TP53 gene across human cancers lead to abnormal p53 protein. This would allow limitless replication of genetically damaged cells with malignant potential and so facilitate tumorigenesis. Normally referred to as the “guardian of the genome,” this protein assumes a central role in different cellular functions spanning metabolism through cell cycle checkpoints and apoptosis to DNA repair mechanisms. Its significance extends beyond tumor suppression because it controls metabolic pathways that are central to maintaining homeostasis under normal physiological conditions. The normal functions of p53 as a tumor suppressor are abrogated by mutations; indeed, many such mutant forms act as oncogenes themselves when expressed in cancer cells. The review paper presents information from recent scientific investigations on the chemical structure of p53 protein and its mutations with a discussion on implications for cancer treatment. It also integrates various research findings to clarify the clinical effects of the p53 gene, especially in breast cancer and myelodysplastic syndromes (MDS), and looks into changing treatment plans that focus on p53 problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c0adda59bbabd91047006215c78f72a09302aad" target='_blank'>
              Structure, Function and Clinical Significance of P53 Protein: A Review Article
              </a>
            </td>
          <td>
            Farah mumtaz, Hadeel Nasser, Ruqaya Yahya Abd AL-Shaheed, Ali A. Al-fahham
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma patients show poor prognosis due in part to poor response to standard-of-care therapies. Hence, there is an urgent need to identify novel therapeutic strategies. Oncogenic Kras mutations are a key driver of PDAC tumorigenesis. Novel Kras-targeted therapies have demonstrated improved efficacies in preclinical models and are currently under trial in the clinic. However, resistance has been observed against these therapies. Metabolic alterations play a key role in regulating therapy responsiveness. Notably, Krasis a key regulator of metabolic reprogramming in pancreatic cancer. Thus, we examined the metabolic reprogramming that occurs upon KRAS signaling inhibition via targeting MEK, ERK, or KRASG12D. We utilized metabolomic, lipidomic, and isotope-tracing approaches to investigate the metabolic alterations that may allow cells to escape Kras signaling inhibition. Our findings reveal that inhibition of MEK, ERK, or KRASG12Dresults in enhanced lipid turnover and fatty acid oxidation while suppressing glycolysis, glucose oxidation, and glutamine metabolism in PDAC cells. We confirmed the increased lipid turnover by kinetic flux analysis of stable-isotope-labeled palmitate that showed increased breakdown of palmitate into acetyl-CoA and the TCA cycle metabolites in cells treated with inhibitors of Kras signaling. Notably, we observed that the increased lipid turnover and fatty acid oxidation are mediated by lipophagy, rather than cytosolic lipolysis. The increased lipophagy was driven by enhanced nuclear translocation and transcriptional activity of Transcription factor EB (TFEB), a member of the MiT/TFE family of basic helix-loop-helix leucine zipper transcription factors. TFEB regulated multiple lipid catabolism and lipophagy-related genes. Inhibiting fatty acid oxidation by genetic perturbations or by utilizing pharmacological agents exhibited synergistic anti-tumor activity when combined with MEK/ERK/KRASG12D inhibitors in pancreatic cancer cell lines and organoids. This combination therapy also decreased tumor burden and improved overall survival in orthotopic cell line injection and patient-derived xenograft (PDX) mouse models. These results provide novel mechanistic insights into the metabolic adaptations underlying resistance to Kras signaling inhibition. These studies highlight fatty acid oxidation as a critical metabolic vulnerability in PDAC upon KRAS signaling inhibition, offering a promising therapeutic target toenhance the efficacy of KRAS pathway inhibitors. The identified novel combinationspresent an opportunity to improve patient outcomes.


 Pankaj Kumar Singh. Targeting the metabolic basis of resistance to Kras signaling inhibition in pancreatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY13-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/928df874c983f8437edd504f81de1181538485ee" target='_blank'>
              Abstract SY13-01: Targeting the metabolic basis of resistance to Kras signaling inhibition in pancreatic cancer
              </a>
            </td>
          <td>
            P. K. Singh
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Breast cancer (BC) is the leading cause of cancer-related deaths in women. It is molecularly and clinically heterogeneous, driven by distinct genetic and epigenetic alterations, making prognosis prediction and treatment selection challenging. Identifying novel markers for BC progression and therapeutic targets remains an urgent need. Although growing research has highlighted the role of epigenetic mechanisms, particularly cancer-associated histone mutations (onco-histone), in regulating BC pathogenesis and its hallmarks, including stemness and drug resistance, the role of histone variants—the naturally derived sequence variants—remains largely unknown, with the exception of H3.3. The newly identified, mammalian-specific histone variant H2A.J has been shown to be highly enriched in human luminal BC and implicated in a stem cell signature correlated with metastatic risk in primary BC. However, it is unclear what role H2A.J plays in BC and how it may function in other BC subtypes. Using cell lines of multiple BC subtypes and patient-derived xenograft (PDX) tumor microarrays, our studies have discovered that H2A.J is present in human triple-negative BC (TNBC), with significantly elevated levels in metastatic TNBC compared to the primary counterpart. Our unpublished results show that H2A.J is the only histone variant that strongly and specifically interacts with the tumor suppressor, retinoblastoma 1 (RB1) in metastatic TNBC cells and luminal BC. Our data also demonstrate that RB1 protein level is significantly correlated with H2A.J in estrogen receptor-positive (ER+) luminal BC PDX samples. Our preliminary results suggest that H2A.J chromatin occupancy affects RB1 recruitment to chromatin, RB1-bound chromatin targets, and ultimately RB1-dependent phenotypes. We will discuss how H2A.J regulates RB1-bound chromatin loci, and hence, the canonical and non-canonical functions of RB1 in tumorigenesis. Our bulk RNA-sequencing and western blotting data show that knocking down (KD) H2A.J interferes with G1/S cell cycle progression and results in the upregulation of estrogen response, apoptosis, and mTOR signaling. Notably, the H2A.J KD-transcriptome signature strongly correlates with that of CDK4/6 inhibitor treatment. Our data also suggest H2A.J's role in maintaining cell lineage fidelity. Overall, our results suggest that H2A.J regulates cellular heterogeneity, stemness, and metastasis in BC. The outcome of our research will not only advance our understanding of the underlying mechanisms leading to BC progression and metastasis but also offer new opportunities for utilizing H2A.J as a target for diagnostic and therapeutic interventions in BC.
 Citation Format: Rana Gbyli, Tyler Jensen, Joseph Balowski, Yao Li, Xiang Zhang, Qin Yan, Andrew Xiao. Harnessing the Power of the Variant: Breast Cancer's Strategic Incorporation of the Histone H2A.J [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e6e23951bd3e66f36e177a58521abd15ef2dc11" target='_blank'>
              Abstract P5-06-22: Harnessing the Power of the Variant: Breast Cancer's Strategic Incorporation of the Histone H2A.J
              </a>
            </td>
          <td>
            Rana Gbyli, Tyler Jensen, Joseph Balowski, Yao Li, Xiang Zhang, Qin Yan, Andrew Xiao
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell-cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti-tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non-neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations.
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of internal medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/701bc66d5a290edadb747fc58b7aca8807e79e86" target='_blank'>
              Mutational signature catalogue (MUSIC) of DNA double-strand break repair
              </a>
            </td>
          <td>
            Marco Barazas, R. van Schendel, M. Tijsterman
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Inherited familial tumors are linked to distinct germline mutations that result in different syndromic phenotypes, increasing cancer risk in patients. Recent findings have unveiled new evidence of genotype-phenotype correlations and highlighted the essential role of biomolecular and immunohistochemical (IHC) analyses in pinpointing predictive markers for immune-therapy responses. Lynch Syndrome (LS), recently identified as four unique hereditary cancer syndromes, is defined by specific germline mutations in Mismatch Repair Genes MLH1, MSH2, MSH6, PMS2. It features distinctive characteristics such as the early development of visceral tumors (primarily colorectal cancer, endometrial, ovarian, urothelial) and a high incidence of synchronous and metachronous cancers. LS cancers exhibit microsatellite instability (MSI). MSI was first used as an indicator of a genetic predisposition, but it is now recognized as an essential predictive marker for therapy response to immune checkpoint inhibitors. These findings resulted in the approval of immune checkpoint inhibitors, such as anti-programmed cell death 1 (anti-PD1) or anti-programmed cell death ligand 1 (anti-PD-L1) by regulatory bodies solely based on MSI status, independent of the type of cancer. In this case, a transition to a universal LS molecular screening method for all newly diagnosed colorectal and endometrial cancers has been effectively promoted. This shift in viewpoint will necessitate a thorough examination of present guidelines, encompassing recommendations on the optimal timing and approach for the implementation of biomolecular and IHC analyses to evaluate MSI status in additional cancers associated with LS. In this review, we offer a clinical summary of the primary MSI-H/dMMR cancers linked to LS and present a detailed description of immunotherapy, recently introduced for some neoplastic entities but increasingly extended to other tumors. We additionally emphasize the absence of existing molecular screening for rarer LS-associated tumors, like sebaceous cancers, which could gain the most from universal IHC screening and contemporary immuno-therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95bdc52534450d884880cfc38f5af0b788372b88" target='_blank'>
              Hereditary Micro‑Satellite Instable cancers associated with Lynch Syndrome: predictive biomarkers and novel immuno-therapeutic approaches.
              </a>
            </td>
          <td>
            G. Ponti, F. Gelsomino, Also Tomasi, T. Ozben, L. Losi, M. Manfredini
          </td>
          <td>2025-06-01</td>
          <td>Critical reviews in clinical laboratory sciences</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background T-cell acute lymphoblastic leukemia (T-ALL) is a relatively rare hematological malignancy, characterized by the uncontrolled proliferation of immature T lymphoblasts and associated with a generally unfavorable prognosis. Our previous research has demonstrated that decreased mitochondrial activity is associated with the aggressiveness of T-ALL tumors. However, the mechanisms underlying this phenomenon and its contribution to treatment resistance remain largely elusive. Methods We have built up the largest known T-ALL tumor bank, with a median follow-up of 32 months, including our transcriptomic data from 79 newly sequenced tumors that adds to the 54 publicly accessible samples. Computational analyses and a series of functional assays were performed to investigate the molecular links between altered mitochondrial activity and drug resistance. Results The transcriptomic analysis revealed that down-regulation of mitochondrial activity is a potent driver of ABCB1 activation, a gene strongly associated with multidrug resistance. In tumors with low mitochondrial activity, the impaired fatty acids β-oxidation leads to intracellular lipid accumulation, which is directly involved in ABCB1 activation. Indeed, our data show that lipid neo-synthesis and accumulation promotes the activation of lipogenic transcription factors, liver X receptors (LXRs), which act as drivers of ABCB1 expression. Tumor data analyses confirmed that high ABCB1 expression in tumour samples is indeed associated with reduced mitochondrial gene expression, lipid droplet enrichment, increased tumour aggressiveness, and significantly shorter patient survival. Conclusions Our study demonstrates that reduced mitochondrial activity drives multidrug resistance in adult T-ALL via lipid-mediated activation of ABCB1. These findings enhance our understanding of the biology of aggressive T-ALL and provide insight into mechanisms of resistance to conventional chemotherapy. Consequently, we propose that targeting de novo lipogenesis and restricting dietary fats, such as caprylic acid, may help overcome treatment resistance in patients with T-ALL exhibiting low mitochondrial activity. Trial registration The clinical trial was registered under the identifiers ChiCTR-ONRC-14004968 and ChiCTR2000031553 at ClinicalTrials.gov. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06423-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17840f9a8a2001064a43216b8500d420bfe553ae" target='_blank'>
              Mitochondrial dysfunction fuels drug resistance in adult T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Shanshan Guo, Ekaterina Bourova-Flin, Sophie Rousseaux, Florent Chuffart, Lijun Peng, Duohui Jing, Jian-Qing Mi, Saadi Khochbin, Jin Wang
          </td>
          <td>2025-05-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Elucidating the role of somatic mutations in cancer, healthy tissues, and aging depends on methods that can accurately characterize somatic mosaicism across different cell types, as well as assay their impact on cellular function. Current technologies to study cell-type-specific somatic mutations within tissues are low-throughput. We developed Duplex-Multiome, incorporating duplex consensus sequencing to accurately identify somatic single-nucleotide variants (sSNV) from the same nucleus simultaneously analyzed for single-nucleus ATAC-seq (snATAC-seq) and RNA-seq (snRNA-seq). By introducing strand-tagging into the construction of snATAC-seq libraries, duplex sequencing reduces sequencing error by >10,000-fold while eliminating artifactual mutational signatures. When applied to 98%/2% mixed cell lines, Duplex-Multiome identified sSNVs present in 2% of cells with 92% precision and accurately captured known sSNV mutational spectra, while revealing unexpected subclonal lineages. Duplex-Multiome of > 51,400 nuclei from postmortem brain tissue captured sSNV burdens and spectra across all major brain cell types and subtypes, including those difficult to assay by single-cell whole-genome sequencing (scWGS). This revealed for the first time that diverse neuronal and glial cell types show distinct rates and patterns of age-related mutation, while also directly discovering developmental cell lineage relationships. Duplex-Multiome identified clonal sSNVs occurring at increased rates in glia of certain aged brains, as well as clonal sSNVs that correlated with changes in expression of nearby genes, in both neurotypical and autism spectrum disorder (ASD) individuals, directly demonstrating that somatic mutagenesis can contribute to gene expression phenotypes. Duplex-Multiome can be easily adopted into the 10X Multiome protocol and will bridge somatic mosaicism to a wide range of phenotypic readouts across cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41cd39016a4b2c876a2210fd9227cfa83893338" target='_blank'>
              Unraveling Tyrosine-Kinase Inhibitor Resistance in NSCLC Cells via Same-cell Measurement of RNA, Protein, and Morphological Responses
              </a>
            </td>
          <td>
            Vivian Dien, Connor Thompson, Tristin Rammel, Jaime Roquero, Marielle Krivit, Grace Yeo, Daniel Honigfort, Pazinah Bhadha, Colin Brown-Greeves, Mariam Dawood, Keerthana Elango, Hyukjin J. Kwon, Tyler Lopez, Alvin Lo, Meghna Manjunath, Pin Ren, Ben Song, Mitch Sundkamp, Ramreddy Tippana, Wilfred Van Ijcken, Jennifer Wong, Jane Yao, Da Zhou, Sinan Arslan, Matthew Kellinger, Amirali Kia, S. Kruglyak, Michael Previte, Molly He
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Insufficient eradication of cancer cells and survival of drug tolerant clones are major relapse driving forces. Underlying molecular mechanisms comprise activated pro-survival and anti-apoptotic signaling leading to insufficient apoptosis and drug resistance. The identification of programmed cell death pathways alternative to apoptosis opens up for possibilities to antagonize apoptosis escape routes. We have earlier shown that acute lymphoblastic leukemia (ALL) harbours a distinct propensity to undergo cell death by RIPK1-dependent necroptosis, activated by small molecule second mitochondria-derived activators of caspase (SMAC) mimetics. Despite demonstrated safety and tolerability of SMAC mimetics in clinical trials, their efficacy as single agent appears still limited, highlighting the need for combinatorial treatments. Here, we investigate so far unexplored regulatory mechanisms of necroptosis and identify targets for interference to augment the necroptotic anti-leukemia response. Ex vivo drug response profiling in a model of the bone marrow microenvironment reveals powerful synergy of necroptosis induction with histone deacetylase inhibition. Subsequent transcriptome analysis and functional in vivo CRISPR screening identify gene regulatory circuitries through the master transcription regulators SP1, p300 and HDAC2 to drive necroptosis. While deletion of SP1 or p300 confers resistance to necroptosis, loss of HDAC2 sensitizes to RIPK1-dependent cell death by SMAC mimetics. Consequently, our data inform strong in vivo anti-leukemic activity of combinatorial necroptosis induction and HDAC inhibition in patient-derived human leukemia models. Thus, transcriptional dependency of necroptosis activation is a key regulatory mechanism that identifies novel targets for interference, pointing out a strategy to exploit alternative non-apoptotic cell death pathways to eradicate resistant disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e50b0632f13f3cfa99a142440642fa7ca4167d68" target='_blank'>
              Transcriptional remodeling shapes therapeutic vulnerability to necroptosis in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Anna Saorin, Anna Dehler, Bartimée Galvan, F. Steffen, Marine Ray, Dong Lu, Xin Yu, James Kim, Aneta Drakul, Samanta Kisele, Jin Wang, Jean-Pierre Bourquin, B. Bornhauser
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Bacteria with similar genomes can exhibit different phenotypes due to alternative gene expression patterns. In this study, we analysed four antibiotic-resistant Staphylococcus aureus hospital isolates using transcriptomics, PacBio genome sequencing, and methylomics analyses. Transcriptomic data were obtained from cultures exposed to gentamicin, the iodine-alanine complex CC-196, and their combination. We observed strain-specific expression patterns of core and accessory genes that remained stable under antimicrobial stress – a phenomenon we term the Clonal Gene Expression Stability (CGES) that is the main discovery of the paper. An involvement of epigenetic mechanisms in stabilization of the CGES was hypothesized and statistically verified. Canonical methylation patterns controlled by type I restriction-modification systems accounted for ~ 10% of epigenetically modified adenine residues, whereas multiple non-canonically modified adenines were distributed sporadically due to imperfect DNA targeting by methyltransferases. Protein-coding sequences were characterized by a significantly lower frequency of modified nucleotides. Epigenetic modifications near transcription start codons showed a statistically significant negative association with gene expression levels. While the role of epigenetic modifications in gene regulation remains debatable, variations in non-canonical modification patterns may serve as markers of CGES.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/312d8b17fcc90be9c2950c359123ee70cd25237f" target='_blank'>
              Epigenetic background of lineage-specific gene expression landscapes of four Staphylococcus aureus hospital isolates
              </a>
            </td>
          <td>
            I. Korotetskiy, S. Shilov, T. Kuznetsova, N. Zubenko, L. Ivanova, O. Reva
          </td>
          <td>2025-05-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1950d0d983eeb2c05d4af1f02b47efd0ad374c" target='_blank'>
              Theoretical investigation of AKT1 mutations in breast cancer: a computational approach to structural and functional insights
              </a>
            </td>
          <td>
            Balu Kamaraj, G. C
          </td>
          <td>2025-05-09</td>
          <td>Journal of computer-aided molecular design</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758af464be2f5fe707681ad668cca7d960076b75" target='_blank'>
              Targetable BIRC5 dependency in therapy-resistant TP53 mutated acute myeloid leukemia
              </a>
            </td>
          <td>
            Ahmed M Mamdouh, Fang Qi Lim, Yang Mi, E. Olesinski, Cheryl Gek Teng Chan, S. Jasdanwala, Xiao Xian Lin, Yichen Wang, Jordan Yong Ming Tan, K. Bhatia, Valeriia Sapozhnikova, Chuqi Wang, Aarthi Nivasini Mahesh, Daniel Tan En Liang, Nishtha Chitkara, P. Mertins, L. Hogdal, Brandon D Brown, Torsten Haferlach, Camille Lobry, Coleman Lindsley, Alexandre Puissant, Han Kiat Ho, Sanjukta Das, A. Letai, S. Kornblau, J. Krönke, Edward Ayoub, Koji Itahana, Michael Andreeff, Shruti Bhatt
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Homologous recombination (HR) and mismatch repair (MMR) act as guardians of the human genome, and defects in HR or MMR are causative in at least a quarter of all malignant tumors. Although these DNA repair-deficient tumors are eligible for effective targeted therapies, fully reliable diagnostic strategies based on functional assay have yet to be established, potentially limiting safe and proper application of the molecular targeted drugs. Here we show that transient transfection of artificial DNA substrates enables ultrarapid detection of HR and MMR. This finding led us to develop a diagnostic strategy that can determine the cellular HR/MMR status within one day without the need for control cells or tissues. Notably, the accuracy of this method allowed the discovery of a pathogenic RAD51D mutation, which was missed by existing companion diagnostic tests. Our methods, termed HR eye and MMR eye, are applicable to frozen tumor tissues and roughly predict the response to therapy. Overall, the findings presented here could pave the way for accurately assessing malignant tumors with functional defects in HR or MMR, a step forward in accelerating precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9695c321b4adb2f24badedc1d3f1003b2ca44567" target='_blank'>
              HR eye & MMR eye: one-day assessment of DNA repair-defective tumors eligible for targeted therapy
              </a>
            </td>
          <td>
            Shinta Saito, Shingo Kato, Usaki Arai, Atsuki En, Jun Tsunezumi, Taichi Mizushima, K. Tateishi, N. Adachi
          </td>
          <td>2025-05-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Chromodomain Helicase DNA-binding protein 1-Like (CHD1L) is a chromatin remodeling enzyme increasingly recognized as an oncogenic factor promoting tumor progression and metastatic potential by orchestrating transcriptional programs that drive epithelial-mesenchymal transition (EMT), cytoskeletal remodeling, and metastatic dissemination. In parallel, CHD1L has emerged as a master regulator of tumor cell survival by regulating DNA damage response and repair and enforcing G1 cell cycle progression. Furthermore, CHD1L plays a key role in immune evasion pathways by regulating signaling cascades and by suppressing both apoptotic and non-apoptotic cell death. In particular, CHD1L is a key suppressor of PARthanatos, a caspase-independent mechanism triggered by poly(ADP-ribose) (PAR) polymer fragmentation and apoptosis-inducing factor (AIF) activation. By regulating SPOCK1, MDM2, and TCTP, CHD1L further supports survival under cellular stress. Its overexpression correlates with metastasis, therapy resistance, and poor prognosis across many solid tumors. This review covers CHD1L’s structure, oncogenic functions, and developmental origins, and highlights emerging therapeutic strategies that target CHD1L as a druggable vulnerability in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6296c3dbcbe7b1d1919c8debad6aee4011adbed2" target='_blank'>
              CHD1L in cancer and beyond: structure, oncogenic functions, and therapeutic potential
              </a>
            </td>
          <td>
            Sophia Clune, Paul Awolade, Hector Esquer, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2025-05-30</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Transcribed conserved non-coding elements (TCNEs), which are non-coding genomic elements that can regulate vital gene expression, play an unclear role in the development of severe diseases mainly associated with carcinogenesis. Currently, there are no mature tools for the identification of TCNEs. To compensate for the lack of a systematic interpretation of the functional characterization and regulatory mechanisms of TCNE spatiotemporal activities, we developed a flexible pipeline, called captureTCNE, to depict the landscape of TCNEs and applied it to our breast cancer cohort (SEU-BRCA). Meanwhile, we investigated the genome-wide characteristics of TCNEs and unraveled that TCNEs harbor enhancer-like chromatin signatures as well as participate in the transcriptional machinery to regulate essential genes or architect biological regulatory networks of breast cancer. Specifically, the TCNE transcripts could recruit RBPs, such as ENOX1 and PTBP1, which are involved in gene expression regulation, to participate in the formation of regulatory networks and the association with altered splicing patterns. In particular, the presence of a non-classical secondary structure, called RNA G-quadruplex, on TCNE transcripts contributed to the recruitment of RBPs associated with subtype-specific transcriptional processes related to the estrogen response in breast cancer. Ultimately, we also analyzed the mutational signatures of variant-containing TCNEs and discerned twenty-one genes as essential components of the regulatory mechanism of TCNEs in breast cancer. Our study provides an effective TCNE identification pipeline and insights into the regulatory mechanisms of TCNEs in breast cancer, contributing to further knowledge of TCNEs and the emergence of innovative therapeutic strategies for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa57a71626db4f88ef44a82587dc582edcdfe08" target='_blank'>
              Dissecting the Emerging Regulatory and Mechanistic Paradigms of Transcribed Conserved Non-Coding Elements in Breast Cancer
              </a>
            </td>
          <td>
            Wenyong Zhu, Hao Huang, Qiong Li, Yu Gu, Rong Zhang, Huiling Shu, Yunqi Zhao, Hongde Liu, Xiao Sun
          </td>
          <td>2025-04-27</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c635cae84c7bdc8795ab539e7b406a70e03c5a4" target='_blank'>
              A transient mutational burst occurs during yeast colony development.
              </a>
            </td>
          <td>
            Nicolas Agier, Nina Vittorelli, Louis Ollivier, Frédéric Chaux, Alexandre Gillet-Markowska, Samuel O'Donnell, Fanny Pouyet, G. Fischer, Stéphane Delmas
          </td>
          <td>2025-06-09</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T. Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The CRISPR-Cas9 system has emerged as a transformative tool in genetic engineering, enabling scientists to edit genomes with unprecedented precision, efficiency, and simplicity. Since its adaptation from a prokaryotic immune mechanism to a gene-editing platform, CRISPR-Cas9 has rapidly progressed from bench research to translational and clinical applications. This paper presents a comprehensive review of the most recent advancements in CRISPR-Cas9 technology, particularly focusing on innovations that enhance the accuracy, versatility, and therapeutic utility of gene editing. Key breakthroughs include the development of base editing and prime editing technologies, which allow for single-nucleotide substitutions and complex DNA insertions or deletions without creating double-stranded breaks. These tools have significantly expanded the scope of precise genetic modifications and minimized genotoxicity. 
Moreover, the engineering of high-fidelity Cas9 variants—such as eSpCas9, SpCas9-HF1, and HypaCas9—has markedly reduced off-target activity, thereby addressing one of the major limitations in clinical genome editing. Innovations in delivery strategies, including the use of viral vectors (e.g., AAV, lentivirus), lipid nanoparticles, and electroporation of ribonucleoprotein complexes, have improved the in vivo and ex vivo application of CRISPR-Cas9 for therapeutic purposes. Notable clinical progress has been observed in the treatment of genetic disorders such as sickle cell disease, β-thalassemia, and hereditary blindness, with several patients experiencing durable therapeutic benefits. Simultaneously, CRISPR-based immunotherapies are being explored for cancers and viral infections, such as HIV, marking a shift toward more personalized and targeted treatments. 
Despite these advances, challenges remain, including immune responses to Cas9 proteins, potential mosaicism in edited cells, ethical concerns regarding germline editing, and regulatory hurdles that govern clinical translation. Furthermore, the integration of computational tools, including machine learning and deep learning algorithms, has become critical for optimizing guide RNA design, predicting off-target effects, and refining editing outcomes. This integration of biotechnology and computational sciences is propelling the CRISPR field into a new era of precision medicine. 
In conclusion, recent advancements in CRISPR-Cas9 technology are not only pushing the boundaries of what is technically feasible in genome editing but are also laying the groundwork for safe, effective, and personalized therapeutic applications. As the technology continues to mature, it holds the promise to redefine the future of medicine, particularly in the treatment and potential cure of a wide range of genetic and complex diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c179a051e549adc925ccce1b7d2a4b7dcb7a485" target='_blank'>
              Recent Advancements in CRISPR-Cas9 Technology for Precision Gene Editing and Therapeutic Applications
              </a>
            </td>
          <td>
            Dr. D.Jayarajan, Dr. Amit Kumar Dutta, Dr. T.Deborah Paripuranam, Mrs. M.Krishnaveni, Dr S.Bhavya
          </td>
          <td>2025-05-23</td>
          <td>Journal of Neonatal Surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15180


 Background:
 Pancreatic cancer has apoor prognosis and limited therapeutic options. While
 K-RAS
 mutations are prevalent in this malignancy, a subset of patients harboring wild-type
 K-RAS
 presents distinct clinical behavior. Gene fusions have been discerned in pancreatic cancer and are promising candidates for administering targeted therapies.
 Methods:
 We conducted a retrospective analysis of 926 pancreatic cancer patients of which 165 (18%) carried a wild-type
 K-RAS
 gene, by next generation sequencing (NGS) or polymerase chain reaction (PCR) testing. Tumors were analyzed with a clinical-grade hybrid capture-based RNA-seq clinical fusionome test, evaluating gene fusions between 1104 genes or to other known or novel partners as well as gene expression transcript analysis. Single nucleotide variants/insertions deletions (SNV/Indels) were tested with a clinical-grade NGS assay targeting > 300 cancer genes.
 Results:
 Gene fusions were detected in 32% of
 K-RAS
 wild-type pancreatic cancers, with 75% involving known oncogenic, clinically actionable genes including
 FGFR
 (two
 FGFR2
 and one
 FGFR1
 fusion), two
 NTRK3
 and one each of
 RAF1, RET, ROS1
 and
 BRAF
 fusions. Deleterious SNVs were frequently found in both fusion-positive and fusion-negative
 K-RAS
 wild-type pancreatic cancers. Transcriptomic expression revealed additional potentially druggable targets, e.g.,
 ROS1
 and
 FGFR3
 overexpression in a tumor harboring an
 NTRK3
 fusion,
 BRAF
 overexpression in the context of an otherwise non-actionable
 ETV6
 fusion, and
 MET
 overexpression in a
 TES::TFEC
 -fusion bearing cancer.
 Conclusions:
 Druggable fusions are frequent in
 K-RAS
 wild-type pancreatic cancers. Moreover, RNA fusion assays as well as transcriptomic expression testing can identify candidate personalized therapies for patients with
 K-RAS
 wild-type pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723fc53e1c67e373fda744cec67f0720b9c9b122" target='_blank'>
              The cancer fusionome and overexpressed druggable oncogenic signals in
 K-RAS
 wild-type pancreatic cancer.
              </a>
            </td>
          <td>
            Michal Krawczyk, Roisin Puentes, Nathan D Montgomery, Derek D Lyle, R. Kurzrock, F. Lopez-Diaz
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemotherapy. However, PDAC with germline BRCA mutations, which lead to homologous recombination (HR) deficiency (HRD), demonstrated an increased sensitivity to platinum-based chemotherapy regimens. This increased chemosensitivity was also seen in PDACs with germline or somatic mutations in the DNA double-strand damage response (DDR) genes beyond canonical HR genes such as BRCA1, BRCA2, and PALB2. However, there are no consensus methods to determine HRD status; and neither is there a well-defined list of HR-DDR genes. In addition, how HRD and/or HR-DDR gene mutation status impacts the tumor immune microenvironment including tumor mutation burden (TMB), neoantigen load, T cell receptor (TCR) repertoire, and effector T cell infiltration is unknown. Thus, in this study, we developed a new method to categorize PDACs into HRD-positive and HRD-negative subgroups by using results from whole exome sequencing, whole genome sequencing, or both into consideration. We classified a cohort of 89 PDACs into HRD-positive (n = 18) and HRD-negative (n = 69) tumors. HR-DDR gene variants were identified more frequently in HRD-positive PDACs than HRD-negative PDACs, with RAD51B, BRCA2 and ATM alterations most frequently identified in HRD-positive PDACs. Notably, TMB and neoantigen load was significantly higher in HRD-positive PDACs compared to HRD-negative tumors. Interestingly, HRD-positive PDACs, PDACs with high tumor mutational burden, and PDAC with high neoantigen load were all associated with lower CD8 + T lymphocyte infiltration and T cell clonal diversity, suggesting a mechanism of resistance to immune checkpoint inhibitors (ICIs). Therefore, this study suggests that treatments to enhance effector T cell infiltration and T cell clonal diversity may overcome resistance to ICI-based immunotherapy in HRD-positive PDACs. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00673-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76f00cebbe1db7a6bb20d7d6bd25ec9f1ccbba1" target='_blank'>
              Homologous recombination-DNA damage response defects increase TMB and neoantigen load, but not effector T cell density and clonal diversity in pancreatic cancer
              </a>
            </td>
          <td>
            Mengyue Lei, Jessica Gai, T. McPhaul, Huijuan Luo, Penghui Lin, Dongbin Liu, Michael Pishvaian, Nicholas J Roberts, Kui Wu, Jin He, Lei Zheng
          </td>
          <td>2025-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e17020


 Background:
 Testicular germ cell tumors (TGCT) are the most common malignancy in young adult men, with a globally increasing incidence. While these tumors are highly curable due to their exceptional sensitivity to cisplatin (cDDP), a proportion of patients develops a resistance to this chemotherapy, which results in disease progression and patient death. The cause of cisplatin resistance in TGCT has not been elucidated yet and is supposed to be multifactorial.
 Methods:
 We conducted a CRISPR Knockout (KO) screening in a TGCT cell line (NCCIT) using a lentiviral sgRNAs library (Addgene #101926) targeting 3015 genes involved in cell cycle regulation and cancer development. The screening population was cultured with or without cDDP (1.25 uM) for 7 days. Samples of treated and control cells were harvested before and after cDDP exposure, the sgRNA cassettes were sequenced by massive parallel sequencing, and the differences in sgRNA abundance were determined using the MAGeCK/MAGeCK-Flute pipeline.
 Results:
 The CRISPR-KO screening data analysis identified 59 gene KOs that were significantly positively selected and 7 gene KOs that were significantly negatively selected after cDDP exposure. Among the top positive hits were genes involved in the regulation of apoptosis (
 BMF, CDIP1
 and
 BAX
 ), epigenetic modifiers (
 HDAC8, SSX2
 ), G-protein regulators (
 ADCY8, GNG10
 ), and others (e.g.
 MAGEA2, ACADM, LAMP1
 ) — inactivation of these genes potentially contributes to the development of cDDP resistance. Among the top negative hits were genes implicated in DNA damage repair (
 FANCB
 ,
 ERCC5
 ), sphingolipid metabolism (
 KDSR
 ), and cytokinesis (
 SPECC1L
 ) — inactivation of these genes may increase sensitivity to cDDP.
 Conclusions:
 The results highlight several potential mechanisms of cDPP resistance in TGCT. Some of these genes and pathways are well known and have been previously described in association with platinum resistance in malignant tumors, while others — such as the G-protein regulators, sphingolipid synthesis, and
 SPECC1L
 — are novel in this context and may reflect the unique biology of TGCT progression. SPECC1L is involved in spindle organization and microtubule stabilization; its paralog SPECC1 is highly expressed in testis. Polyenoic sphingolipids are essential for the completion of meiosis in male germ cells and spermatozoa production.

 ADCY8 controls the production of cAMP, which is a key signaling molecule in spermatogenesis. They have also been described in association with malignant proliferation and progression. The identified hits will be validated in subsequent experiments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff267218117b8545c90f6a863002128909b0591f" target='_blank'>
              Use of CRISPR-KO screening to identify novel genes and pathways potentially contributing to cisplatin resistance in testicular germ cell tumors.
              </a>
            </td>
          <td>
            Laura Matouskova, Kristyna Soffrova, V. Bakardjieva-Mihaylova, J. Stuchlý, L. Boublikova, Karolina Skvarova
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="
 Background: Combinations of endocrine therapy and CDK4/6 inhibitors (CDK4/6i) used in metastatic breast cancer (MBC) result in heterogenous clinical benefit with subsets of patients having disease control ranging anywhere from a few months to more than 5 years. Understanding the mechanisms that account for these differences can enable new pharmacologic approaches to convert more of these cancers into the long-term control subset. Methods: To elucidate the causal factors that underlie the observed clinical variability, we have utilized multi-omic analyses of patient samples as well as experimental modeling using a diverse cohort of ER+ breast cancers. For clinicogenomic analyses, we analyzed 467 patients with ER+/ HER2- MBC treated with first-line CDK4/6i at our institution (MSK) with detailed clinical follow annotation and follow up. For genomic/transcriptomic studies, we analyzed single cell RNA sequencing data, whole genome sequencing data, and Ki67 score from the neoadjuvant CDK4/6i containing arms in the FELINE trial (NCT02712723). Patient derived xenografts, patient derived organoids, isogenic cell lines were used to define the underlying mechanisms. Live cell imaging assays were performed by FUCCI cell cycle sensor to assess cell cycle transitions. Results: Of 467 patients from MSK cohort, 129 (27.6%) had pre-CDK4/6i loss-of-function variants in TP53, corresponding to short PFS (median PFS= 8.7 months, 95% CI: 6.6–10.3; HR = 2.12, 95% CI: 1.68 to 2.68; p = 2.64e-10). This finding was validated in two large independent clinical cohorts outside MSK. In patient derived and isogenic cell lines, p53 loss had no impact on initial drug response but promoted long-term cell outgrowth and prevented emergence of senescence phenotypes. Following cells in detailed time courses revealed the propensity of p53 mutant but not wild type cells to reenter the cell cycle after initial arrest by CDK4/6 inhibition. Gene expression analysis of RNA seq data revealed a significant positive association between TP53 mutation status and DREAM complex genes, assessed by GSEA (NES= 1.51, p = 1e-4) In models, deletion of DREAM complex component, p130-Rbl2 was found to fully recapitulate the effects of p53 loss: promoting long-term outgrowth from CDK4/6i, blocking therapy induced senescence, and promoting cell cycle re-entry. To validate these results, we analyzed the multi-omic and histologic characteristics of serial samples from the FELINE trial. Compared with WT cases, baseline TP53 LOF had a higher amount of proliferating cells at the end of treatment, measured by Ki67 > 10% (4.5% of WT cases vs. 60% of mutant cases, respectively, OR = 26.8, p = 0.014), the per cell proliferation score, measured by ssGSEA (log2FC: 0.2 vs. 2.8, respectively) as well as a higher pseudobulk expression of DREAM complex related genes (GSEA: NES = 2.0, q = 3.7e-4). Finally, to identify strategies that might facilitate stable cell cycle control of p53KO tumors, we investigated which targets of p53 regulate DREAM and identified p21. Given role of p21 in regulating both CDK4/6 and CDK2 we investigated whether CDK2 inhibitors would provide additional benefit to CDK4/6i. Addition of CDK2i was able to fully restore efficacy in p53KO lines as well as a patient derived model with TP53 mutation enabling irreversible cell cycle arrest and long-term growth suppression. Conclusion: Wild type p53 facilitates DREAM complex assembly in response to CDK4/6i in ER+ breast cancer, underlying long-term disease control. p53 loss enables cells to reenter the cell cycle and facilitates the development of CDK4/6i resistance. Combined inhibition of CDK2 with CDK4/6 represents a promising therapeutic strategy for overcoming loss of p53 and enabling long-term antitumor efficacy.
 Citation Format: Rei Kudo, Anton Safonov, Enrico Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, Edaise M. da Silva, Qing Li, Marie Will, Atsushi Fushimi, Harikrishna Nakshatri, Andrew Koff, Britta Weigelt, Qamar J. Khan, Pedram Razavi, Sarat Chandarlapaty. DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS13-04.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c291e911332efcfe859d324b671af4bba597c2" target='_blank'>
              Abstract PS13-04: DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition
              </a>
            </td>
          <td>
            Rei Kudo, A. Safonov, E. Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, E. D. da Silva, Qing Li, Marie Will, A. Fushimi, H. Nakshatri, Andrew Koff, B. Weigelt, Qamar J Khan, P. Razavi, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Canine lymphoma, a phenotypically and genetically heterogeneous disease, represents a significant proportion of canine cancers. We present a large-scale study of 238 dogs with lymphoma to better understand the genetic landscape of canine lymphoma, as well as the relationship to clinical outcomes. Using a targeted next-generation sequencing panel comprising 308 genes, we screened somatic and germline mutations in matched tumor and normal samples. Our findings revealed key associations between genetic alterations and lymphoma subtypes, with certain somatic variants linked to significant differences in response to common chemotherapy regimens. Recurrent mutations in genes such as KMT2C, KMT2D, NOTCH2, TRAF3, CCND1, ARID1A, CREBBP, and TP53 were observed, with TRAF3 mutations standing out for their significant association with prolonged progression-free survival and overall survival in B-cell lymphomas. In contrast, mutations in PIK3CD and CREBBP were associated with inferior outcomes in T-cell lymphomas, highlighting the immunophenotype-specific impact of genetic alterations on treatment responses. These findings support the integration of comprehensive genomic profiling in planning treatment strategies and optimizing clinical outcomes in canine lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e07f223413194847051e584c8f37b09d9f47c90d" target='_blank'>
              Identification of novel genetic mutations for the treatment prognostication of canine lymphoma
              </a>
            </td>
          <td>
            Josephine Tsang, Qi-Jing Yap, Sheena Kapoor, Jerry Cromarty, Sushmita Sen, Minji Kim, George Courcoubetis, Suhyeon Cho, Deanna Swartzfager, Stanley Park, Sungwon Lim, Ilona Holcomb, Jamin Koo
          </td>
          <td>2025-06-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary The presence of certain DNA damage/repair errors together can cause cancer cells to die, offering hope for new treatments. Mutational signatures show how DNA damage/repair errors happened by looking at the types and amounts of mutations. We studied mutational signatures that occur together or separately, called mutational signature interactions, to find which combinations of DNA repair errors help cancer cells thrive and which ones harm them. We analyzed mutational signatures and their interactions in 1014 breast cancer genomes. HRd and NHEJd were found together with APOBEC, while they did not overlap with the ROS mutational signature. Mutational signature interactions were distinct in various breast cancer types. Non-HRd TNBC tumors showed a unique ROS signature, while HRd TNBC tumors did not have an ROS signature. These analyses show that mutational signatures and their interactions could help identify promising synthetic lethal targets in DNA repair systems, improve how patients are grouped and point to potential treatments. Abstract Background: Defective DNA repair systems result in the accumulation of mutations, loss of genomic integrity, and eventually cancer. Following initial malignant transformation due to specific DNA damage and defective DNA repair, cancer cells become reliant upon other DNA repair pathways for their survival. The co-occurrence of specific repair deficiencies brings catastrophic outcomes such as cell death for cancer cells and thus holds promise as a potential therapeutic strategy. Exploring the co-occurrence and mutual exclusivity of mutational signatures provides valuable knowledge regarding combinations of defective repair pathways that are cooperative and confer selective advantage to cancer cells and those that are detrimental and cannot be tolerated by them. Methods: Taking advantage of mutational signature profiling, we analyzed whole-genome sequences of 1014 breast cancers to reveal the underlying mutational processes and their interrelationships. Results: We found an inverse relationship between deficiencies of homologous recombination (HRd) and non-homologous end joining (NHEJd) with reactive oxygen species (ROS). Moreover, HRd and NHEJd co-occurred with APOBEC but were mutually exclusive with mismatch repair deficiency (MMRd) and ROS. Our analysis revealed that SBS8 and SBS39 signatures of currently unknown etiology correlate with NHEJd. ID1 and ID2 signatures co-occur with ROS and have mutual exclusivity with HRd, SBS8, SBS39 and NHEJd. The ID4 signature, with currently unknown etiology, has mutual exclusivity with HRd and NHEJd and co-occurred with ROS. On the other hand, the ID15 signature, with currently unknown etiology, co-occurred with SBS8, SBS39, HRd, NHEJd and DBS2, while having an inverse relationship with MMRd and ROS. Comparing the mutational signatures of HRd and non-HRd TNBC genomes reveals the unique presence of ROS signatures in non-HRd tumors and the lack of ROS signature in HRd tumors. Conclusion: Taken together, these analyses indicate the possible application of mutation signatures and their interactions in advancing patient stratification and suggest appropriate therapies targeting the make-up of individual tumors’ mutational processes. Ultimately, this information provides the opportunity to discover promising synthetic lethal candidates targeting DNA repair systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/575ec445e704ca863d5507eaa78e755caa175d07" target='_blank'>
              The Use of Mutational Signatures to Decipher the Inter-Relationship of Reactive Oxygen Species with Homologous Recombination and Non-Homologous End-Joining Deficiencies as Well as Their Effects on APOBEC Mutagenesis in Breast Cancer
              </a>
            </td>
          <td>
            Amir Farmanbar, Robert Kneller, Sanaz Firouzi
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="As many as 90% of human pancreatic ductal adenocarcinoma (PDAC) tumors harbor gain-of-function mutations in the KRAS oncogene. Recently, inhibitors of the most common KRAS mutation, KRASG12D, have entered the clinical arena. However, early evidence suggests that as monotherapy, KRASG12D inhibitors such as MRTX1133 at best provide brief periods of disease stabilization. Hence, there is a growing interest in understanding the mechanisms through which tumors acquire resistance to KRAS inhibition. In the present study, we generated in vitro models of MRTX1133 resistance and subjected parental and drug-resistant cell lines to RNA sequencing. This suggested that MRTX1133-resistant tumor cells undergo a global shift toward histone acetylation. Inhibition of the histone acetyltransferase EP300 reversed the drug-resistant phenotype in vitro, which subsequent RNA sequencing experiments determined was associated with the suppression of pro-survival FOSL1 signaling. Accordingly, siFOSL1 reversed the MRTX1133-resistant phenotype with similar effects on pro-survival signaling. Given the lack of clinically useful EP300 or FOSL1 inhibitors, we next explored whether inhibitors of the acetylation scanning BET proteins would be similarly effective. The addition of BET inhibitors re-sensitized several resistant cell lines to MRTX1133 and impaired FOSL1-mediated survival signaling in vitro. In murine models of MRTX1133-resistant PDAC, BET inhibition cooperated with MRTX1133 to markedly extend overall survival. As BET inhibitors are currently under clinical testing, the combination of MRTX1133 and BET inhibitors warrants further investigation, particularly in tumors that have developed resistance to KRAS inhibition. SIGNIFICANCE Here, we demonstrate that BET inhibition is effective in PDAC with acquired resistance to KRAS inhibitors. As BET inhibitors are under clinical testing, the combination of KRAS and BET inhibitors warrants consideration in PDAC patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31571cc4d771e5d8cbcfb30a223ba711a12ce0b" target='_blank'>
              Resistance to the KRASG12D Inhibitor MRTX1133 is Associated with Increased Sensitivity to BET Inhibition
              </a>
            </td>
          <td>
            Daniel R. Principe, Jeffrey H Becker, Anastasia E. Metropulos, Alejandra M Marinelarena, Thao D Pham, A. Aissa, H. Munshi
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Abstract Repeats are the most diverse and dynamic but also the least well-understood component of microbial genomes. For all we know, repeat-associated mutations such as duplications, deletions, inversions and gene conversion might be as common as point mutations, but because of short-read myopia and methodological bias, they have received much less attention. Long-read DNA sequencing opens the perspective of resolving repeats and systematically investigating the mutations they induce. For this study, we assembled the genomes of 16 closely related strains of the bacterial pathogen Mycobacterium tuberculosis from Pacific Biosciences HiFi reads, with the aim of characterizing the full spectrum of DNA polymorphisms. We found that complete and accurate genomes can be assembled from HiFi reads, with read size being the main limitation in the presence of duplications. By combining a reference-free pangenome graph with extensive repeat annotation, we identified 110 variants, 58 of which could be assigned to repeat-associated mutational mechanisms such as strand slippage and homologous recombination. Whilst recombination events were less frequent than point mutations, they affected large regions and introduced multiple variants at once, as shown by three gene conversion events and a duplication of 7.3 kb that involved ppe18 and ppe57, two genes possibly involved in immune subversion. The vast majority of variants were present in single isolates, such that phylogenetic resolution was only marginally increased when estimating a tree from complete genomes. Our study shows that the contribution of repeat-associated mechanisms of mutation can be similar to that of point mutations at the microevolutionary scale of an outbreak. A large reservoir of unstudied genetic variation in this ‘monomorphic’ bacterial pathogen awaits investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48cccc87585756169c7b4f284da57f120538805b" target='_blank'>
              Gene conversion and duplication contribute to genetic variation in an outbreak of Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Christoph Stritt, Michelle Reitsma, Ana Maria Garcia Marin, G. Goig, A. Dötsch, S. Borrell, C. Beisel, Iñaki Comas, D. Brites, S. Gagneux
          </td>
          <td>2025-05-01</td>
          <td>Microbial Genomics</td>
          <td>1</td>
          <td>83</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686514f6a242593983edc4eedbd2ea02d17cd369" target='_blank'>
              REpair of heterozygous Mutations independent of Exogenous Donor template with high efficiency (REMEDY) using allele specific CRISPR targeting and HDR enhancers
              </a>
            </td>
          <td>
            Y. Sezgin, Genesis Snyder, Noushin Saljoughian, Colin Maguire, Ebru Erzurumluoglu Gokalp, Devi Jaganathan, Eleonora S D'Ambrosio, B. Ozes, Gregory Wheeler, Ben Kelly, Mark E. Hester, Zarife Sahenk, S. Vaidyanathan, A. Rashnonejad, Jerry Mendell, Nizar Y. Saad, Roshini S. Abraham, Juhi Bagaitkar, Susan D. Reynolds, Allison Bradbury, M. N. Kararoudi
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Maintaining genome integrity is crucial for the proper functioning and development of organisms. One intriguing aspect of genome integrity is the formation and function of neocentromeres at non-centromeric sites. CENP-A, a centromere-specific protein, is essential for centromere identification and function. However, in many cancers, CENP-A is often found to be ectopically misplaced when overexpressed. Moreover, CENP-A deposition at the centromere depends on the transcription of centromeric non-coding RNAs. Consequently, ectopic CENP-A is found at transcriptionally active and frequent breakpoint regions. To further explore ectopic CENP-A localization, we previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing a non-centromeric oncogenic lncRNA, PCAT2, which was capable of recruiting CENP-A to its transcription site. In this work, we tracked cells carrying this stable transgene to understand the longevity of the induced ectopic CENP-A site at the chromosome that harbors it. Our findings revealed that the induced epigenetic memory was eventually lost due to the suppression of the transgene through competing epigenetic silencing mechanisms. This epigenetic restoration naturally reversed the ectopic CENP-A level to its previous levels at the engineered site. These data suggest that cells may have evolved failsafe mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription, unless otherwise favored by selection involving multiple components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5efe50139c531598a2ac6d2663615d0ad3cf36f2" target='_blank'>
              Combined genomic and phenotypic classification of inherited and acquired genetic disease with long-read sequencing
              </a>
            </td>
          <td>
            Y. Fontaine, S. Aradhya, S. Ji, E. Masle-Farquhar, I. Bosi, J. Forkgen, Z. Qiao, I. Stevanovski, A. M. Reis, M. Rapadas, D. Suan, P. Hsu, B. Wainstein, A. W. Hewitt, G. A. Figtree, I. W. Deveson, O. M. Siggs
          </td>
          <td>2025-05-09</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Induced pluripotent stem cell technology has significantly advanced regenerative medicine, providing an invaluable platform for modeling neurodevelopmental disorders and facilitating the development of novel therapeutic strategies. This article discusses the potential of induced pluripotent stem cells to unravel the cellular mechanisms underlying neurodevelopmental disorders, which, despite their genetic diversity, share common pathological features. Traditional models have struggled to replicate human-specific phenotypes, whereas induced pluripotent stem cell-based models, including two-dimensional cultures and three-dimensional organoids, offer more accurate representations of neural development and disease. The article explores advances in reprogramming and differentiation protocols that have enabled the generation of patient-specific induced pluripotent stem cell models, while acknowledging the challenges that persist, such as genomic instability and reprogramming inefficiencies. Additionally, the integration of CRISPR/Cas9 gene editing and patient-derived models has led to precision therapies targeting specific genetic mutations, including small molecules, gene editing, and antisense oligonucleotides. By combining in vitro and in vivo approaches, induced pluripotent stem cell-induced pluripotent stem cell models have expanded our understanding of neurodevelopmental disorder mechanisms, including aging, sex differences, and epigenetic regulation. However, challenges in model reproducibility and physiological complexity remain, and the article emphasizes efforts to address these limitations through optimized differentiation protocols, robust quality control, and ethical sourcing. Ultimately, induced pluripotent stem cells hold transformative potential for elucidating neurodevelopmental disorder pathogenesis and advancing personalized therapies for these complex disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9e960f8ed1a3b0a93c7188fe8abe97aa6e400e7" target='_blank'>
              Induced pluripotent stem cell models for advancing neurodevelopmental disorder research and regenerative medicine: a narrative review
              </a>
            </td>
          <td>
            Moawiah M. Naffaa
          </td>
          <td>2025-06-11</td>
          <td>Regenerative Medicine Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The CRISPR-Cas9 system has been harnessed and repurposed into a powerful genome editing tool. By leveraging this technology, researchers can precisely cut, paste, and even rewrite DNA sequences within living cells. Nevertheless, the application of CRISPR screen technology goes far beyond mere experimentation. It serves as a pivotal tool in the fight against genetic diseases, systematically dissecting complex genetic landscapes, empowering researchers to unravel the molecular mechanisms underlying biological phenomena, and enabling scientists to identify and target the root causes of illnesses such as cancer, cystic fibrosis, and sickle cell anemia. Among all, cancer poses a formidable challenge for medicine, spurring eradication efforts. Radiotherapy, as a traditional treatment, yields results but has limitations. It eradicates cancer cells but also damages healthy tissues, causing adverse effects that reduce quality of life. Additionally, not all cancer cells respond to radiotherapy, and some may develop resistance, worsening the condition. To address this, a comprehensive whole-genome CRISPR screen technology is introduced, as it enables the efficient identification of radiosensitive and radioresistant genes, thereby advancing the field of cancer research and treatment. A genome-wide CRISPR screen was conducted in lung adenocarcinoma cells exposed to irradiation following the described protocol, through which both radioresistance- and radiosensitivity-associated genes were identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0b084e039e9bd637d4e61bfb3817e38d13b7fe" target='_blank'>
              Genome-Wide CRISPR Screen for Unveiling Radiosensitive and Radioresistant Genes.
              </a>
            </td>
          <td>
            Yu Yuan, Ziyu Jiang, Yuxin Zeng, Jiawen Tang, Jiang Luo, Conghua Xie, Yan Gong
          </td>
          <td>2025-05-23</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Dear Editor , Predicting a patient ’ s response to cancer therapies and identifying additional molecular targets is crucial for advancing cancer treatment. BRAF V600E -speci ﬁ c inhibitors have demonstrated promising initial clinical results in solid tumors; however, side effects and drug resistance are common, and for most patients, response to BRAF V600E inhibitor therapy is transient. Treatment with dabrafenib (Dab), a selective BRAF V600E inhibitor, in combination with trametinib (Tra), a MEK1/2 inhibitor, is effective for BRAF V600E+ unresectable or metastatic solid tumors. However, resistance mechanisms and activation of alternative proliferation pathways can still bypass BRAF/MEK dual blockade. 1 To comprehensively identify genes that regulate resistance to MAPK inhibitors, we selected two well-characterized human thyroid cancer (TC) cell lines, validated for preclinical studies, namely K1 (GLAG-66) and 8505c. Both cell lines are BRAF V600E - positive, but each has its own distinct clinical characteristics and genomic alterations. 2 K1 and 8505c cell lines stably expressing spCas9 were transduced with a second lentiviral vector containing the sgRNA cassette (Fig. 1a). To study combination Dab+Tra (D + T) therapy in these cell lines, we ﬁ rst determined their inhibitory concentration dosage curves and selected a combination dosage of 10 μ M Dab + 10 nM Tra for the screen. The Brunello CRISPR knockout lentiviral genome-wide library was used to identify genes whose loss confers resistance or sensitization to MAPK inhibitors. We analyzed and determined the statistical signi ﬁ cance of each sgRNA change. Among the top scorers, with a FDR below 5% for both cell lines, we found genes known to cause resistance against MAPK inhibitors ( NF1 , CUL3 , NF2 , and MED12 ), 3">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c82496d631807bef56d8a2c9a3bd893fa002732a" target='_blank'>
              Loss of the aryl hydrocarbon receptor promotes cancer cell resistance to BRAFV600E targeted therapies
              </a>
            </td>
          <td>
            Mourad Zerfaoui, Dharmeshkumar Patel, Yueqi Zhang, Raymond F. Schinazi, Y. Errami
          </td>
          <td>2025-05-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The clustered regularly interspaced short palindromic repeats (CRISPR)/Cas system is a powerful genome editing tool that has revolutionized research. Single nucleotide polymorphisms (SNPs) are the most common form of genetic variation in humans. Only a subset of these SNPs has been shown to be linked to genetic diseases, while the biological relevance of the majority remains unclear. Investigating these variants of unknown significance could provide valuable insights into their roles in biological processes, disease susceptibility, and treatment responses. While CRISPR/Cas has emerged as a transformative technology, its ability to make single nucleotide substitutions remains a significant limitation. Other techniques in single nucleotide editing, such as base editing and prime editing, offer promising possibilities to complement CRISPR/Cas systems, though they also have their own limitations. Hence, alternative approaches are necessary to overcome the limitations of CRISPR. Here, to improve the feasibility of generating single base edits in the genome, we provide a protocol that introduces a multiple expression and dual selection (MEDS) system, which, alongside CRISPR, utilizes the opposing roles of cytosine deaminase/uracil phosphoribosyltransferase (CD/UPRT) for negative selection and neomycin phosphotransferase II (NPT II) for positive selection. As a proof of concept and to demonstrate feasibility of the method, we used MEDS, along with traditional CRISPR-Cas9, to generate sickle hemoglobin by introducing a point mutation (A → T) in the sixth codon of the hemoglobin beta gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea49db880f24cd1e00dae98c95c6cafbd8936b9" target='_blank'>
              CRISPR genome editing using a combined positive and negative selection system
              </a>
            </td>
          <td>
            I. Sharma, Kerisa Hall, Shannon Moonah
          </td>
          <td>2025-05-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/918b316170d23091ae769d68a6cb5ef4225d1640" target='_blank'>
              Mechanisms of resistance to active state selective tri-complex RAS inhibitors
              </a>
            </td>
          <td>
            Ben Sang, Ling Feng Ye, Feng Hu, Yasin Pourfarjam, Antonio Cuevas-Navarro, Shijie Fan, Zheng Fu, Aaliyah Washington, Diego J. Rodriguez, Alberto Vides, Sumit Kar, Ethan Ahler, Kevin K. Lin, A. Hegde, J. Smith, B. Wolpin, S. Punekar, A. Spira, I. Garrido-Laguna, David S. Hong, A. Dar, R. Yaeger, K. Arbour, P. Lito
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="ARID1A, a key subunit of the SWI/SNF chromatin remodeling complex, is frequently mutated in cancers. However, effective clinical treatments for patients with this mutation are limited, highlighting a demand for new therapeutic strategies. Here, we establish Werner syndrome ATP-dependent helicase (WRN) as a critical vulnerability target in ARID1A-mutated cancers. Upon genetic and pharmacological inhibition of WRN, ARID1A-mutated cells had defective Chk1-mediated DNA damage signaling, resulting in compensatory Chk2 activation, leading to G1-phase arrest and apoptosis, whereas ARID1A-proficient cells underwent Chk1-dependent G2/M arrest. Additional p21 inhibition, combined with WRN suppression, drove G1-arrested cells to re-enter the cell cycle, triggering mitotic catastrophe. The anti-tumor efficacy of WRN inhibition was validated using in vivo cell lines and patient-derived xenograft mouse models. Our findings define WRN as a selective therapeutic target in ARID1A-mutated cancers and suggest a combinatorial strategy of WRN and p21 inhibition as a therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ee45b7e29b6454158a945bb535aaf35dcedcd93" target='_blank'>
              Differential DNA damage response to WRN inhibition identifies a targetable vulnerability in ARID1A-mutated cancers
              </a>
            </td>
          <td>
            Jiwon Kim, Jaeik Oh, Dongjun Jang, Seungjae Shin, Soo Jin Lee, Sang Eun Lee, Yoojin Yang, Dohee Kim, Ae-Jin Choi, Hae-Rim Jung, Yumi Oh, Sung-yup Cho
          </td>
          <td>2025-05-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumour microenvironment is composed of a complex cellular network involving cancer, stromal and immune cells in dynamic interactions. A large proportion of this network relies on direct physical interactions between cells, which may impact patient responses to clinical therapy. Doublets in scRNA-seq are usually excluded from analysis. However, they may represent directly interacting cells. To decipher the physical interaction landscape in relation to clinical prognosis, we inferred a physical cell–cell interaction (PCI) network from ‘biological’ doublets in a scRNA-seq dataset of approximately 18,000 cells, obtained from 7 treatment-naive ovarian cancer patients. Focusing on cancer-stromal PCIs, we uncovered molecular interaction networks and transcriptional landscapes that stratified patients in respect to their clinical responses to standard therapy. Good responders featured PCIs involving immune cells interacting with other cell types including cancer cells. Poor responders lacked immune cell interactions, but showed a high enrichment of cancer-stromal PCIs. To explore the molecular differences between cancer-stromal PCIs between responders and non-responders, we identified correlating gene signatures. We constructed ligand-receptor interaction networks and identified associated downstream pathways. The reconstruction of gene regulatory networks and trajectory analysis revealed distinct transcription factor (TF) clusters and gene modules that separated doublet cells by clinical outcomes. Our results indicate (i) that transcriptional changes resulting from PCIs predict the response of ovarian cancer patients to standard therapy, (ii) that immune reactivity of the host against the tumour enhances the efficacy of therapy, and (iii) that cancer-stromal cell interaction can have a dual effect either supporting or inhibiting therapy responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641e58c95ba92d526c7831c22b5ab2944de7c20b" target='_blank'>
              Direct cell interactions potentially regulate transcriptional programmes that control the responses of high grade serous ovarian cancer patients to therapy
              </a>
            </td>
          <td>
            S. Hameed, Walter Kolch, Donal J. Brennan, Vadim Zhernovkov
          </td>
          <td>2025-04-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c2d54e4671afd8b01fe843ed17126ffeb50bcf" target='_blank'>
              Genetic, epigenetic, and molecular determinants of multiple myeloma and precursor plasma cell disorders: a pathophysiological overview.
              </a>
            </td>
          <td>
            Henry Sutanto, Debi Yulia Sandra, Ardea Safira, Galih Januar Adytia, Alief Waitupu, P. Z. Romadhon
          </td>
          <td>2025-06-03</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, J. Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="This review article aims to provide a comprehensive overview of genetic mutations and alterations in prostate cancer, highlighting their roles in disease development and progression. The scope of the review encompasses common genetic mutations, such as those in the PTEN and TP53 genes, as well as structural alterations like the TMPRSS2-ERG fusion. It also discusses the impact of these genetic changes on cellular functions, including cell cycle regulation and signaling pathways. Major findings indicate that genetic mutations contribute to the uncontrolled proliferation of prostate cells and influence treatment responses, emphasizing the importance of molecular profiling in guiding personalized therapy. Additionally, the review explores the significance of the tumor microenvironment and epigenetic modifications in prostate cancer progression. The insights gained from this review underscore the necessity for ongoing research into genetic alterations to improve diagnostic methods and therapeutic strategies, ultimately enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58a7054831337d3623636983abbff338a1c3f2ff" target='_blank'>
              Genetic Mutations in Prostate Cancer: Insights and Implications
              </a>
            </td>
          <td>
            Chinenye Mary Okorochi, Frederick Olusegun Akinbo, Gerald Ikechi Eze, Kehinde Elizabeth Olukoya, Enoch Chibuike Okorochi, Austin Ifeanyi Obiozor, Uzochukwu Chimdindu Ibe, Simon Enenche Iyalla, Macpherson Ifeanyi Ukachukwu
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d271ccc22e71dc47432dac9f4b41d18e3a1dd713" target='_blank'>
              Linking regulatory variants to target genes by integrating single-cell multiome methods and genomic distance.
              </a>
            </td>
          <td>
            E. Dorans, K. Jagadeesh, K. Dey, A. L. Price
          </td>
          <td>2025-06-12</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27a802d056910b5b0c0dd0ae061c28d9e6b15a90" target='_blank'>
              Lamin A/C Deficiency Drives Genomic Instability and Poor Survival in Small-Cell Lung Cancer through Increased R-loop Accumulation
              </a>
            </td>
          <td>
            Christopher W. Schultz, Sourav Saha, A. Dhall, Yang Zhang, Parth Desai, L. Pongor, D. Scheiblin, Valentin Magidson, Yilun Sun, Christophe E Redon, Suresh Kumar, Manan Krishnamurthy, Henrique B. Dias, V. Aksenova, Elizabeth Giordano, N. Takahashi, Michael Nirula, Mohit Arora, Chiori Tabe, M. Thomas, Rajesh Kumar, Yasuhiro Arakawa, Ukhyun Jo, Beverly A. Teicher, M. Aladjem, Stephen Lockett, M. Dasso, Y. Pommier, A. Sharma, Anish Thomas
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background: Rhabdoid tumours (RTs) are aggressive neoplasms most often characterised by biallelic loss of the SMARCB1 gene, encoding a core subunit of the SWI/SNF chromatin-remodelling complex. Despite their relative genetic stability, RTs exhibit a highly malignant phenotype and poor prognosis. Methods: This review explores the mechanisms underlying SMARCB1 aberrations, their role in driving hallmarks of cancer, and emerging therapeutic strategies for RTs. Ongoing clinical trials listed on ClinicalTrials were reviewed to evaluate the translational potential of targeted therapies in SMARCB1-deficient rhabdoid tumours. Results: Loss of SMARCB1 drives multiple cancer hallmarks by disrupting key regulatory pathways. It promotes unchecked cell proliferation through alterations in p16INK4a and Myc signalling. SMARCB1-deficient tumours possess immune-evading capabilities via PD-L1 overexpression and immune checkpoint activation. SMARCB1 deficiency also alters cellular energetics. The nucleotide biosynthesis pathway has been demonstrated to be upregulated in RT organoids, as shown by increased levels of pathway metabolites. Enzymes of the mevalonate pathway such as HMG-CoA reductase and mevalonate kinase are also dysregulated. Targeting glutathione metabolism with eprenetapopt may induce oxidative stress and apoptosis. Widespread epigenetic aberrations, including increased EZH2 activity, are being targeted with inhibitors such as tazemetostat. Conclusions: SMARCB1 loss is a central driver of cancer hallmarks in RTs, enabling proliferation, immune evasion, metabolic reprogramming, and epigenetic dysregulation. Future horizons in RT treatment include immunotherapies, epigenetic modifiers, and gene therapies. The synergy and optimal timing of targeted therapy with conventional treatment requires further characterisation for clinical translation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e0bef453aa1a3eef3841216ea0de0de9cb63f28" target='_blank'>
              SMARCB1 Deficiency as a Driver of the Hallmarks of Cancer in Rhabdoid Tumours: Novel Insights into Dysregulated Energy Metabolism, Emerging Targets, and Ongoing Clinical Trials
              </a>
            </td>
          <td>
            Abdul L. Shakerdi, Graham P. Pidgeon
          </td>
          <td>2025-05-01</td>
          <td>Metabolites</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer that can be caused by DNA replication stress. Copy number variation (CNV) is a type of genomic instability that has been associated with both tumorigenesis and drug resistance, but how these structural variants form in response to replication stress is not fully understood. Here, we established a direct repeat genetic reporter in Saccharomyces cerevisiae to detect recombination events that result in either a duplication or a deletion. Using this system, we measured recombination resulting from site-specific replication fork stalling initiated by Tus binding to an array of Ter sites. We found that a Tus/Ter fork block downstream of direct repeats induced CNV by a mechanism involving the Mph1 translocase, Exo1-catalyzed end resection and Rad51-dependent strand invasion. While the Slx4 scaffold protein and its nuclease-binding partner, Rad1-Rad10, were shown to be required for duplications, we found that they suppress deletion formation in this context. These opposing functions suggest that both recombination products arise through a large loop heteroduplex intermediate that is cleaved by Rad1-Rad10 in a manner that promotes duplications and eliminates deletions. Taken together, these studies give insight into the mechanisms governing CNV in the context of replication fork stalling, which may ultimately provide a better understanding of how replication stress contributes to cancer and other diseases characterized by genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cd288ac45c9138adb03e430b097a3550c9e6c0" target='_blank'>
              The Slx4-Rad1-Rad10 nuclease differentially regulates deletions and duplications induced by a replication fork barrier
              </a>
            </td>
          <td>
            Marina K Triplett, Iffat Ahmed, Swathi Shekharan, L. Symington
          </td>
          <td>2025-05-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Autophagy suppresses tumorigenesis in normal cells, but in established tumors, it can promote tumor progression, particularly by enhancing resistance to stress. However, the mechanism underlying this tumor-promoting function remains unclear. To investigate this, we adopted an interdisciplinary approach combining database analysis with experimental validation. Specifically, by classifying the autophagy-related genes using AutoML analysis on their expression patterns in the COSMIC database, we identified an autophagy subnetwork that correlated with the PLK1-RAD9A axis, a pathway we had previously linked to genotoxic resistance. Cell-based experiments confirmed that autophagy enhanced PLK1 expression at both the transcriptional and translational levels, facilitating genotoxic resistance. Notably, in stressed S-phase cells, we found that PLK1 expression levels varied among individual cells, yet overall cell population acquired genotoxin resistance. The genotoxin resistance in the cell population with heterogeneous PLK1 expression was driven by autophagy by facilitating the secretion of currently unidentified factors, likely by switching funtion of RAD9A from DNA checkpoint to substance secretion. Together our data demonstrate that intra-tumor heterogeneity contributes to the malignant features of tumors through an autophagy-PLK-RAD9A axis that promotes intercellular communication via secretion. (180 words).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c385cb8f12e65d2ac980560afdb3e17f79dfea" target='_blank'>
              Autophagy promotes the acquisition of genotoxic resistance in cancer cells in a paracrine manner via upregulation of PLK1-RAD9A axis.
              </a>
            </td>
          <td>
            M. Ikura, Takuma Shiraki, T. Ikura, Kanji Furuya
          </td>
          <td>2025-05-29</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="One of the strongest signatures of aging is an accumulation of mutant mitochondrial DNA (mtDNA) heteroplasmy. Here we investigate the mechanism underlying this phenomenon by calling mtDNA sequence, abundance, and heteroplasmic variation in human blood using whole genome sequences from ∼750,000 individuals. Our analyses reveal a simple, two-step mechanism: first, individual cells randomly accumulate low levels of “cryptic” mtDNA mutations; then, when a cell clone proliferates, the cryptic mtDNA variants are carried as passenger mutations and become detectable in whole blood. Four lines of evidence support this model: (1) the mutational spectrum of age-accumulating mtDNA variants is consistent with a well-established model of mtDNA replication errors, (2) these mutations are found primarily at low levels of heteroplasmy and do not show evidence of positive selection, (3) high mtDNA mutation burden tends to co-occur in samples harboring somatic driver mutations for clonal hematopoiesis (CH), and (4) nuclear GWAS reveals that germline variants predisposing to CH (such as those near TERT, TCL1A, and SMC4) also increase mtDNA mutation burden. We propose that the high copy number and high mutation rate of mtDNA make it a particularly sensitive blood-based marker of CH. Importantly, our work helps to mechanistically unify three prominent signatures of aging: common germline variants in TERT, clonal hematopoiesis, and observed mtDNA mutation accrual.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a832134834abc3268ec7af8e86bace5b876111" target='_blank'>
              Mechanism of age-related accumulation of mitochondrial DNA mutations in human blood
              </a>
            </td>
          <td>
            Rahul Gupta, Timothy J. Durham, Grant Chau, M. Uddin, Wenhan Lu, K. Karczewski, D. Howrigan, P. Natarajan, Wei Zhou, Benjamin M. Neale, V. Mootha
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Gastric cancer (GC) has a high incidence and mortality rate globally. Double minutes (DMs) are extrachromosomal circular chromosomes that carry amplified oncogenes or drug resistance genes, and they are closely associated with tumorigenesis and drug resistance. To investigate the role and regulatory mechanisms of double minutes in the malignant progression of gastric cancer, we utilize bioinformatics methods to analyze genomic copy number variation data from the CCLE cell line database and TCGA solid tumor database. We analyze and summarize the genomic features of tumor samples with double minutes. Based on these features, we classify gastric cancer samples. We use the FANTOM5 database, R package "GenomicRanges", Bedtools, and MEME SUITE to analyze the functions of the enhancers on double minutes and their regulated target genes in gastric cancer. Next, we apply the CIBERSORT package and the GDSC drug database to analyze immune cell infiltration and drug resistance in gastric cancer samples with and without double minutes from TCGA. The results show that the genome with a copy number greater than 10 and genomic fragments longer than 50 kb play a significant role. The regulatory role of double minutes enhancers affects drug resistance and tumor immunity, with disruptions in the enhancer-transcription factor-target gene regulatory loops also linked to tumor immunity. Furthermore, the target genes regulated by double minutes enhancers not only alter the expression of immune-related genes but also contribute to increased drug resistance to common chemotherapy agents in gastric cancer samples. High expression of drug resistance-related target genes in gastric cancer samples with double minutes is closely associated with poor prognosis. This study provides new insights for the treatment of gastric cancer patients with double minutes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7809d316520a84fae49c8004779285724bae28b" target='_blank'>
              Analysis of regulatory mechanisms of enhancers in gastric cancer with double minute chromosomes based on bioinformatics.
              </a>
            </td>
          <td>
            Meng-Ting An, Guan-Lin Guo, Jie Wu, Wen-Jing Sun, Xue-Yuan Jia
          </td>
          <td>2025-05-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Significance Aneuploidy is a major cause of birth defects, a hallmark of cancer, and a reflection of the dosage sensitivity of genes involved in quantitative traits. Most studies on how aneuploidy affects gene expression have examined single genomic regions at a time. Here, we examined combination trisomies, combination monosomies, and combinations of trisomy for one region and monosomy for another. The effects on global gene expression were found to be cumulative, rebalanced, or in some cases multiplicative. Most aneuploidies alter the total transcriptome size. The results have implications for the stoichiometric control of gene expression, the progression of aneuploidy in cancer, the evolution of sex chromosomes, the role of the triploid block in polyploid evolution, and the control of quantitative traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/644edaec4bdaa378dc76678350b3c6e4610cba95" target='_blank'>
              Global modulation of gene expression and transcriptome size in aneuploid combinations of maize
              </a>
            </td>
          <td>
            Hua Yang, Vincent A. Brennan, Zhi Gao, Jian Liu, Frimpong Boadu, Jianlin Cheng, J. Birchler
          </td>
          <td>2025-05-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Molecular profiling of DNA and RNA from pediatric cancers by next-generation sequencing has been demonstrated to improve diagnosis and prognosis and to identify somatic alterations indicating vulnerability to targeted therapies. Hence, much like in the treatment of adult cancers, molecular profiling is now routinely utilized in clinical workflows for pediatric cancers as a companion to conventional pathology diagnosis. Many variants of unknown significance identified through DNA profiling are being characterized by saturation genome editing, enabled by CRISPR editing technology and clever functional assays. Newer technologies and analytics are revealing additional structural complexity around cancer drivers and gene fusions in pediatric cancer DNA. Similarly, computational methods such as rare variant association studies and polygenic risk scoring are being used to identify novel cancer susceptibility. Together, these advances are expanding our understanding of pediatric cancer's complexity and fueling the development of emerging methods such as liquid biopsy-based monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae78e85702756670907089a42238ceb27fd06f7" target='_blank'>
              Pediatric Cancer Genetics and Genomics.
              </a>
            </td>
          <td>
            Elaine R. Mardis
          </td>
          <td>2025-05-28</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Mitochondria are highly dynamic organelles that can exhibit a range of phenotypes from highly fragmented to highly networked. Many cancer cells exhibit fragmented mitochondrial networks, and this phenotype supports metabolic reprogramming to promote tumor growth. We previously demonstrated that in Ras mutant tumors, ERK mediated phosphorylation of the mitochondrial fission GTPase Drp1 at S616 leads to an increase in mitochondrial fission activity that is responsible for the fragmented phenotype. We and others have demonstrated that Drp1-mediated mitochondrial fission plays a tumor-promoting role in pancreatic and other Ras-driven cancers, but its role in therapeutic resistance is unknown. The MAPK pathway is the most critical effector pathway downstream of mutant Ras. Despite this, targeting this pathway directly in Ras-driven pancreatic cancer has been unsuccessful due to the development of resistance. In the present study we have generated a panel of patient-derived pancreatic cancer cell lines resistant to the MEK inhibitor trametinib and analyzed their mitochondrial phenotypes. We find that trametinib-resistant pancreatic cancer cells exhibit increased expression and phosphorylation of Drp1 compared to sensitive counterparts, even in the presence of trametinib. Furthermore, quantitative analysis of mitochondrial structure reveals that trametinib-induced mitochondrial restructuring is inhibited in resistant cells. Intriguingly, we have identified CDK4/6 as novel kinase of Drp1. Inhibition of CDK4/6 activity is sufficient to block Drp1 phosphorylation in resistant cells, suggesting that activation of these kinases is driving the reactivation of mitochondrial fission in the absence of MAPK signaling. Importantly, CRISPR-mediated deletion of Drp1 from resistant cells either partially re-sensitizes the cells to trametinib (1 line) or leads to growth inhibition independent of drug treatment (3 lines). These data highlight the importance of Drp1 reactivation in maintaining the growth of trametinib resistant tumor cells. Furthermore, we have identified c-MYC oncoprotein is reactivated in trametinib resistant pancreatic cancer cells and mediates Drp1S616 phosphorylation through CDK4/6 kinases. Together, these findings point to the importance of mitochondrial structure in drug resistance and suggest that inhibition of mitochondrial fission might be a promising therapeutic strategy to combat resistance to MAPK and RAS inhibitors.


 Salma Sharmin, Jennifer A. Kashatus, Sara J Adair, Todd W Bauer, David F Kashatus. Reactivation of Drp1S616 phosphorylation by c-MYC-CDK4/6 signaling axis plays a functional role in resistance to MEK inhibition in pancreatic cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB283.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0285decb175fa812aedf3fc7188a16ac9c685df9" target='_blank'>
              Abstract LB283: Reactivation of Drp1S616 phosphorylation by c-MYC-CDK4/6 signaling axis plays a functional role in resistance to MEK inhibition in pancreatic cancer cells
              </a>
            </td>
          <td>
            Salma Sharmin, Jennifer A. Kashatus, S. Adair, Todd W. Bauer, David F. Kashatus
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Recent advances in screening programs and the development of innovative therapeutic strategies have significantly improved the clinical outcomes of cancer patients. However, many patients still experience treatment failure, primarily due to inherent or acquired drug resistance mechanisms. This challenge underscores the urgent need for novel therapeutic targets for the effective treatment of malignancies, as well as cancer-specific biomarkers to enhance early diagnosis and guide interventions. Epigenetic mechanisms, including DNA methylation, have recently garnered growing interest as key regulators of gene expression under both physiological and pathological conditions. Although epigenetic dysregulations are reliable tumor hallmarks, DNA methylation is still not routinely integrated into clinical practice, highlighting the need for further research to translate preclinical findings from the bench to the bedside. On these bases, the present review aims to illustrate the state of the art regarding the role of DNA methylation in cancer, describing the technologies currently available for DNA methylation profiling. Furthermore, the latest evidence on the application of DNA methylation hotspots in cancer diagnosis and prognosis, as well as the impact of epidrugs in cancer care, is discussed to provide a comprehensive overview of the potential clinical relevance of DNA methylation in advancing personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7000e596c1c5da0f5cbf0989fd9d5a1a4286bc4e" target='_blank'>
              Recent advances on gene-related DNA methylation in cancer diagnosis, prognosis, and treatment: a clinical perspective
              </a>
            </td>
          <td>
            Alessandro Lavoro, Daria Ricci, Giuseppe Gattuso, Federica Longo, Graziana Spoto, Anastasia Cristina Venera Vitale, Maria Chiara Giuliana, L. Falzone, Massimo Libra, S. Candido
          </td>
          <td>2025-05-05</td>
          <td>Clinical Epigenetics</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="
 CRISPR screens are a powerful discovery engine for biological research and have been extensively used for drug target discovery and validation. Pooled screens are advantageous because they can easily be performed at genome-scale when linked to read-outs such as cellular fitness or viability. However, these types of screens often provide limited insights into complex molecular mechanisms. Arrayed screens, on the other hand, can be combined with information-rich read-outs, but tend to be challenging to scale and costly. Perturb-Seq/CROP-Seq screens, in which the effects of thousands of CRISPR perturbations can be studied using single-cell RNA sequencing (scRNA-Seq), offer an attractive alternative as they combine CRISPR perturbations in a pooled fashion with a high-content readout.
 Here, we present a genome-wide CROP-Seq screen to identify genes involved in the polarization of THP-1-derived macrophages. M0 macrophages were differentiated into the M1 subtype that secretes proinflammatory cytokines, phagocytizes microbes, and initiates an immune response. We used a compressed CROP-Seq paradigm in which cells were loaded with multiple sgRNAs to increase throughput, and the phenotypic outcomes were subsequently deconvolved. Macrophage polarization was assessed by scRNA-Seq, focusing on a pre-defined signature of 395 mRNAs. Our experiment identified key regulators of M1 polarization, including TLR4, INFGR1, JAK1 and TYK2. In addition, CITE-Seq was incorporated into the screening workflow as macrophages are known to orchestrate immune responses to microenvironmental stimuli sensed by a complex set of surface receptors.
 Overall, this experiment establishes the feasibility of genome-scale CROP-Seq screens and thus paves the way towards utilizing this technology for unbiased drug target discovery.


 Daniela Flehberger, Katarina Lalatovic, Sumit Pawar, Henrik Schmidt, Lukas Badertscher, Tilmann Buerckstuemmer. A compressed genome-wide CROP-Seq screen to dissect the polarization of THP-1-derived macrophages [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb0009e770a8281c6370f663944711cca95affd" target='_blank'>
              Abstract LB043: A compressed genome-wide CROP-Seq screen to dissect the polarization of THP-1-derived macrophages
              </a>
            </td>
          <td>
            Daniela Flehberger, Katarina Lalatovic, Sumit Pawar, Henrik Schmidt, Lukas Badertscher, Tilmann Buerckstuemmer
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a leading cause of cancer-related mortality worldwide, highlighting the need for a deeper understanding of the genetic mechanisms driving its development and progression. Identifying genetic mutations that affect key molecular pathways is crucial for advancing CRC diagnosis, prognosis, and treatment. Patient-derived xenograft (PDX) models are essential tools in precision medicine and preclinical research, aiding in the development of personalized therapeutic strategies. In this study, a comparative analysis was conducted on the most frequently mutated genes—APC, TP53, KRAS, BRAF, NRAS, and ERBB2—using data from publicly available databases (n = 7894) and models from University Medicine Rostock (n = 139). The aim of this study was to evaluate the accuracy of these models in reflecting the mutational landscape observed in patient-derived samples, with a focus on both individual mutations and co-occurring mutational patterns. Our comparative analysis demonstrated that while the ranking of individual mutations remained consistent, their overall frequencies were slightly lower in the PDX models. Interestingly, we observed a notably higher prevalence of BRAF mutations in the PDX cohort. When examining co-occurring mutations, TP53 and APC mutations—both individually and in combination with other alterations—were the most frequent in both datasets. While the PDX models showed a greater prevalence of single mutations and a slightly higher proportion of tumors without detectable mutations compared to the public dataset, these findings present valuable insights into CRC’s mutational landscape. The discrepancies highlight important considerations, such as selective engraftment bias favoring more aggressive tumors, differences in sample size between the two cohorts, and potential bottleneck effects during PDX engraftment. Understanding these factors can help refine the use of PDX models in CRC research, enhancing their potential for more accurate and relevant applications in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fae3b3f38766eb5d24f530ee4f1034c40e5da445" target='_blank'>
              Mutational Patterns in Colorectal Cancer: Do PDX Models Retain the Heterogeneity of the Original Tumor?
              </a>
            </td>
          <td>
            Maria El Hage, Zhaoran Su, Michael Linnebacher
          </td>
          <td>2025-05-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Enhancers, as distal cis-regulatory elements in the genome, have a pivotal influence on orchestrating precise gene expression. Enhancer RNAs (eRNAs), transcribed from active enhancer regions, are increasingly recognized as key regulators of transcription. N6-methyladenosine (m6A), the most plentiful internal modification in eukaryotic mRNAs, has garnered significant research interest in recent years. With advancements in high-throughput sequencing technologies, it has been established that m6A modifications are also present on eRNAs. An accumulative body of evidence demonstrates that aberrant enhancers, eRNAs, and m6A modifications are intimately connected with carcinoma onset, progression, invasion, metastasis, treatment response, drug resistance, and prognosis. However, the underlying molecular mechanisms governing m6A modification of eRNAs in cancer remain elusive. Here, we review and synthesize current understanding of the regulatory roles of enhancers, eRNAs, and m6A modifications in cancer. Furthermore, we investigate the possible roles of eRNAs m6A modification in tumorigenesis based on existing literature, offering novel perspectives and directions for future research on epigenetic regulatory mechanisms in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6966deaca0f1174a84788609926a02bf29bca084" target='_blank'>
              Biological roles of enhancer RNA m6A modification and its implications in cancer
              </a>
            </td>
          <td>
            Yangyang Han, Jingqi Sun, Minghui Yao, L. Miao, Mengjia Li
          </td>
          <td>2025-05-30</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 3577


 Background:
 HRD is linked to sensitivity to platinum-based chemotherapy and poly (ADP-ribose) polymerase inhibitors across various tumor types. However, the presence of HRD, driver events (such as pathogenic mutations in HR genes or other genomic alterations), and its clinical relevance in mCRC remain underexplored.
 Methods:
 We performed the VHIO-300 test, an ISO15189 accredited custom NGS panel profiling over 450 genes (including HR-related genes:
 ATM, BRCA1, BRCA2, BRIP1, CHEK2 or PALB2
 ) on 356 Stage IV mCRC patients (corresponding to 247 primary colorectal and 109 to metastatic samples) enrolled in the Vall d’Hebron Institute of Oncology’s Molecular Prescreening Program from June 2021 to December 2024. All samples had a tumor cellularity > 40% as per pathologist evaluation
 .
 An HR score (sHR) based on genome-wide copy number alterations (CNA) and loss of heterozygosity (LOH) patterns is generated. After cross-validation with Myriad MyChoice, sHR ≥ 56 was established based on a cohort of ovarian tumors and used to identify HRD in mCRC tumor samples.
 Results:
 HRD prevalence was 3.4% in our mCRC cohort (12/356), but, much higher in metastatic lesions, (6.4%, 7/109) than in primary samples (2.03%, 5/247) (p = 0.05). In fact, the median sHR between primary (21) vs. metastases (30) in CRC was significantly different (p < 0.01). Regarding HR gene status, HR-mutated samples were not significantly within the HRD group (p = 0.27) and only 6.5% (2/31) were HRD. Noteworthy,
 BRCA2
 exhibits a frameshift deletion in a homopolymer stretch, that is a frequent hotspot in microsatellite instable (MSI) tumors, but this event was not found to be associated with HRD. In fact, all HRD tumors (n = 12) were microsatellite stable (MSS). Other frequent alterations in mCRC were studied and
 BRAF
 mutations were found to be present in 42% of HRD tumors (p < 0.01). Inversely, HRD was rare in
 KRAS
 -mutated samples (0.7%; 1/140) and, in fact, highly correlated with non-HRD status (p = 0.02), especially the G12 mutation (p < 0.01). Interestingly, CNA profiles also revealed a strong association between the
 BCL2L1
 loci gain and HRD (p < 0.01). Clinically, HRD was not significantly associated with prognostic value nor clinical benefit to oxaliplatin-based combinations. However, the limited sample size and heterogeneous treatment lines restricted robust statistical analysis.
 Conclusions:
 This study identified a small, yet significant subset of mCRC that displays HRD. sHR and HRD rates were higher in metastatic lesions vs primary tumors, indicating HR scarring could be accumulating over time in some mCRC patients. No clear association between pathogenic HR gene mutations and HRD were found, suggesting the involvement of alternative molecular mechanisms in this process. Frequent co-occurring events, such as
 BRAF
 mutations or
 BCL2L1
 gains could be drivers in CRC HRD, and shape, eventually, new therapeutic options for these patients in the metastatic setting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14cea3ee627505991df2a2a9cb9438dafa79fb0" target='_blank'>
              Drivers of homologous recombination deficiency (HRD) in metastatic colorectal cancer (mCRC).
              </a>
            </td>
          <td>
            Paula Romero Lozano, Javier Ros Montañá, Eduardo García-Galea, Maria Vila-Casadesús, R. Comas, Ágatha Martín, Jenifer González-Zorelle, Paolo Nuciforo, H. G. Pálmer, J. Tabernero, E. Élez, Ana Vivancos-Prellezo
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="
 Chemoresistance is a significant challenge in the field of oncology and is the leading cause of treatment failure in breast cancer. Cancer cells could adapt to hypoxia microenvironment through metabolic reprogramming, while developing a strong drug resistance phenotype at the same time. However, the mechanism under which is still largely unknown. Efforts need to be done to identify novel targets and treatment modalities to address the complex issue of chemoresistance in breast cancer. In this study, we set our sights on an upregulated tRNA-derived fragment, tRF-1432, while we screened through high-throughput sequencing in breast cancer hypoxia models, especially in drug-resistant breast cancer cells/tissues. tRFs are a new category of small non-coding RNAs that occur when pre-tRNAs or mature tRNAs are selectively sheared by enzymes under certain pressure, and which play crucial roles in tumorigenesis. Further investigation on the biological roles and clinical value of tRF-1432 has been done. Overexpression of tRF-1432 is positively correlated with tumor cell proliferation and anti-apoptosis ability, and significantly increased chemoresistance of breast cancer in vitro and in vivo. Immunoprecipitation experiments confirmed the direct bond between tRF-1432 and RBMS1, knockdown of RBMS1 may deprive the tumor-promoting and chemoresistance properties of tRF-1432. Furthermore, in tRF-1432 overexpression breast cancer cells we observed a significant decrease of CCL3, a gene which could induce the transition in the phenotype of macrophages into a proinflammatory one. As a crucial member of tumor microenvironment (TME), macrophages participate in the mediation of therapeutic responses. Silencing tRF-1432 could facilitate M1 macrophage polarization and enhance the response to doxorubicin chemosensitivity in breast cancer. Mechanismly, tRF-1432 decreased the stability of CCL3-mRNA with the cooperation of RNA binding protein RBMS1, thereby reduced M1 macrophage polarization and promote the progression and chemoresistance breast cancer via CCL3-CCR5 signaling. In conclusion, our study clarified the role of hypoxia-induced tRF-1432 in breast cancer development and chemoresistance, and elucidate the molecular mechanism of which, aiming to provide a novel target for reversing chemoresistance in breast cancer.
 Citation Format: Yuhan Dai, Shuhan Zhao, Huilin Chen, Yue Huang, Yiqin Xia, Jinhui Peng, Jiangdong Jin, Yifan Wu, Shui Wang, Yangyang Cui, Hui Xie. tRF-1432–a novel target for reversing chemoresistance in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f41503b57a7cfe308d1aaa0daf08fd39b991514" target='_blank'>
              Abstract P3-02-17: tRF-1432–a novel target for reversing chemoresistance in breast cancer
              </a>
            </td>
          <td>
            Yuhan Dai, Shuhan Zhao, Huilin Chen, Yue Huang, Yiqin Xia, Jinhui Peng, Jiangdong Jin, Yifan Wu, Shui Wang, Yangyang Cui, Hui Xie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Kirsten rat sarcoma viral oncogene homolog (KRAS) is the most frequently mutated member of the RAS family of small GTPases (RAS). It affects about one-fifth of cancer cases. The tumor microenvironment (TME) is a multifaceted network of immune cells, metabolites, microbiota, stromal components, and extracellular matrix. It creates a dynamic ecosystem that supports malignant initiation, progression, and therapy resistance through bidirectional crosstalk with tumor cells. Emerging evidence reveals distinct TME landscapes shaped by wild-type versus oncogenic KRAS variants. Additionally, TME rewiring occurs during KRAS-targeted therapies. Deciphering these KRAS-dependent TME architectures and their therapeutic vulnerabilities represents a critical frontier for precision oncology. This review synthesizes key milestones and persistent challenges in KRAS inhibitor development. And it systematically evaluates how KRAS mutations orchestrated immunosuppressive niches, metabolic symbiosis, stromal remodeling, and microbiome dysbiosis, supported by mechanistic insights from preclinical and clinical studies. It further explores therapeutic opportunities arising from targeting TME interactions, including rational combinations of KRAS inhibitors with immune checkpoint blockade, metabolic agents, or microbiota-modulating strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2718254ccab31c0317c655b38e3998d0c1a62d" target='_blank'>
              Mechanistic Foundations of KRAS-Driven Tumor Ecosystems: Integrating Crosstalk among Immune, Metabolic, Microbial, and Stromal Microenvironment.
              </a>
            </td>
          <td>
            Jiayao Ma, Shenao Fu, Jun Tan, Ying Han, Yihong Chen, Xiangying Deng, Hong Shen, Shan Zeng, Yinghui Peng, Changjing Cai
          </td>
          <td>2025-06-09</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c1fa3b6b4a3a16edf1c0f2cbb051f95cae4239" target='_blank'>
              Gene regulatory networks linked to GABA signalling emerge as relevant for glioblastoma pathogenesis
              </a>
            </td>
          <td>
            Itzel Nissen, Soran Dakhel, Chaitali Chakraborty, A. H. Nygaard, A. Kirkeby, A. Hörnblad, Cemal Erdem, Silvia Remeseiro
          </td>
          <td>2025-05-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Focal gene amplification serves as an oncogenic driver during tumorigenesis and is a hallmark of many forms of cancer. Oncogene amplifications promote genomic instability, which is integral to cancer cell survival and evolution. However, focal gene amplification potentially affords an opportunity for therapeutic exploitation. As a proof-of-concept, we leverage CRISPR-Cas9 nickase to selectively promote cancer cell death in MYCN-amplified neuroblastoma in a gene amplification-dependent manner. Our analysis demonstrates that CRISPR-Cas9 nickase can generate a lethal number of highly toxic, replication-dependent double-strand breaks in cells harboring amplified loci. Furthermore, we demonstrate that Cas9 nickase—mediated toxicity can be modulated in combination with small molecule inhibitors targeting key regulators of the DNA-damage response or cell death pathways. Importantly, our findings in MYCN-amplified neuroblastoma translate to other cancer types with distinct oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426fa411ebfe0306ef5fb3727f439d058ec49944" target='_blank'>
              Selective targeting of genome amplifications and repeat elements by CRISPR-Cas9 nickases to promote cancer cell death
              </a>
            </td>
          <td>
            Matthew B Hanlon, Jason M Shohet, S. Wolfe
          </td>
          <td>2025-06-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Colorectal cancer (CRC) ranks among the top causes of cancer-related fatalities globally. Recent progress in genomics, proteomics, and bioinformatics has greatly improved our comprehension of the molecular underpinnings of CRC, paving the way for targeted therapies and immunotherapies. Nonetheless, obstacles such as tumor heterogeneity and drug resistance persist, hindering advancements in treatment efficacy. In this context, the integration of artificial intelligence (AI) and organoid technology presents promising new avenues. AI can analyze genetic and clinical data to forecast disease risk, prognosis, and treatment responses, thereby expediting drug development and tailoring treatment plans. Organoids replicate the genetic traits and biological behaviors of tumors, acting as platforms for drug testing and the formulation of personalized treatment approaches. Despite notable strides in CRC research and treatment - from genetic insights to therapeutic innovations - numerous challenges endure, including the intricate tumor microenvironment, tumor heterogeneity, adverse effects of immunotherapies, issues related to AI data quality and privacy, and the need for standardization in organoid culture. Future initiatives should concentrate on clarifying the pathogenesis of CRC, refining AI algorithms and organoid models, and creating more effective therapeutic strategies to alleviate the global impact of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a09e7a2013b89bc3cd3dc50a12185b7bd0c09d" target='_blank'>
              Recent advances and challenges in colorectal cancer: From molecular research to treatment
              </a>
            </td>
          <td>
            Gao-Xiu Qi, Rui-Xia Zhao, Chen Gao, Zeng-Yan Ma, Shang Wang, Jing Xu
          </td>
          <td>2025-06-07</td>
          <td>World Journal of Gastroenterology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Osteosarcoma (OS) presents significant treatment challenges due to chemoresistance. This study explores the molecular mechanisms underlying chemoresistance in OS, focusing on the novel fusion gene RPS27-RPS24. Using single-cell multi-omics techniques, we identified a significant upregulation of RPS27-RPS24 in chemoresistant OS cells. Our analyses revealed that RPS27-RPS24 enhances glutaminase (GLS)-mediated glutamine metabolism and inhibits copper-induced cell death, thereby promoting chemoresistance. In vitro experiments with adriamycin-resistant (ADMR) OS cells confirmed that overexpression of RPS27-RPS24 leads to increased cell viability and proliferation under chemotherapy. In vivo studies further validated these findings, demonstrating that targeting glutamine metabolism can reverse chemoresistance. Our results suggest that the RPS27-RPS24 fusion gene plays a critical role in OS chemoresistance through metabolic reprogramming, providing a potential therapeutic target for improving OS treatment outcomes. The application of multiple analytical techniques in this study (as shown in the upper image) and the hypothesized mechanism (as shown in the lower image).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe8559842f0f521ca0cecc77e49311c45066c19f" target='_blank'>
              Single-cell multi-omics elucidates the role of RPS27-RPS24 fusion gene in osteosarcoma chemoresistance and metabolic regulation
              </a>
            </td>
          <td>
            Zhiwei Tao, Pingan Zou, Zhengxu Yang, Tao Xiong, Zhi Deng, Qincan Chen
          </td>
          <td>2025-04-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Small cell lung cancer (SCLC) is among the most aggressive and lethal lung malignancies, accounting for approximately 14% of all lung cancer cases. Characterized by the loss of TP53 and RB1 in most cases, SCLC exhibits high genomic instability. ATM kinase, a critical DNA damage repair (DDR) protein of the PIKK family, is highly expressed in cancers. High ATM expression correlates with resistance to ATR and PARP inhibitors, and ATM inhibitors improves therapeutic outcomes in DDR-deficient cancers. Despite this, the mechanistic underpinnings of ATM inhibition and its potential therapeutic applications in SCLC remain poorly understood.



 We employed a comprehensive approach that included the analysis of next generation sequencing (NGS) data, including RNA-sequencing from approximately 1,000 SCLC clinical samples, representing one of the largest and most comprehensive panels of cases to date. Additionally, we utilized an IC50 panel of two ATM inhibitors across a diverse panel of SCLC cell lines. Bulk RNA sequencing and mass spectrometry data were used to delineate the molecular effects of ATM inhibition. Functional assays, including western blot, qRT-PCR, cell cycle, and apoptosis analyses, were conducted to explore cell death pathways under ATM inhibition.



 ATM expression was significantly elevated in SCLC clinical specimens, particularly in tumors with high YAP1 expression, as well as those with high co-expression of YAP1, ASCL1, NEUROD1, and/or POU2F3. Sensitivity to ATM inhibition was greater in SCLC cell lines than in non-small cell lung cancer (NSCLC), with vulnerability linked to the unfolded protein response and oxidative phosphorylation pathways. Transcriptomic and proteomic profiling revealed that ATM inhibition led to the downregulation of several oncogenic signaling pathways, including MYC and PI3K/AKT, and an upregulation of DNA damage response genes. ATM inhibition also increased intracellular oxidative stress, promoting cell death through apoptosis. Importantly, ATM inhibition synergized with standard chemotherapy agents, significantly enhancing therapeutic efficacy in preclinical models.



 Our findings highlight ATM as a critical therapeutic target in SCLC. ATM inhibition rewires the SCLC transcriptome and proteome, downregulating oncogenic pathways and enhancing the effectiveness of existing chemotherapy. This novel role of ATM provides a strong rationale for developing combination therapies targeting DDR and associated oncogenic pathways, paving the way for improved drug development and patient outcomes in SCLC.



 Debdatta Halder, Utsav Sen, Vrinda Jethalia, Subhamoy Chakraborty, Yosuke Dotsu, Andrew Elliot, Ari Vanderwalde, Deniz Demircioglu, Dan Hasson, Triparna Sen. Targeting ATM modulates oncogenic pathways and amplifies chemotherapy efficacy in small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1f9135b22221853a957c30c494e32f6d68e606" target='_blank'>
              Abstract LB016: Targeting ATM modulates oncogenic pathways and amplifies chemotherapy efficacy in small cell lung cancer
              </a>
            </td>
          <td>
            Debdatta Halder, Utsav Sen, V. Jethalia, Subhamoy Chakraborty, Yosuke Dotsu, Andrew Elliot, A. Vanderwalde, D. Demircioglu, D. Hasson, Triparna Sen
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Endocrine resistance is frequently encountered in the clinic through a variety of mechanisms such as NF1 loss that induce alternative survival pathways and suppress estrogen responsiveness. While targeting the induced pathways such as the RAS-MAPK pathway can have antitumor effects, it can also incur toxicities. To identify novel and potentially more specific therapeutic vulnerabilities in this context, we performed CRISPR/Cas9 screens in wildtype and NF1 knockout isogenic ER+ models and identified NR2F2, an orphan nuclear receptor, to be essential specifically in NF1 loss cells. Our in vitro and in vivo results revealed that NR2F2 is a critical regulator of the estrogen signaling pathway, largely playing a repressive role. We found that NF1 loss could upregulate NR2F2 expression level in ER+ breast cancer cells through the activation of the MAPK pathway. NR2F2 overexpression by itself conferred endocrine resistance, while genetic knockout and pharmacologic inhibition of NR2F2 sensitized ER+ breast cancer cells to endocrine therapies and attenuated resistance across multiple endocrine refractory tumor models including those driven by NF1 loss, ARID1A loss, and PTEN loss. These results reveal upregulation of NR2F2 to be a recurrent mechanism whereby hormone responsiveness is suppressed. At a mechanistic level, investigation into chromatin accessibility and transcription revealed that altered NR2F2 expression modulates genome-wide chromatin accessibility including ER-regulated loci and modulates ER transcriptional activity through its direct interactions with ER and its transcriptional coregulators. Collectively, our data unveil a critical mechanism whereby endocrine resistance occurs through the repression of estrogen responsiveness and highlight a new therapeutic approach to restore endocrine response through pharmacologic inhibition of NR2F2.
 Citation Format: Yanyan Cai, Peihua Zhao, Fan Wu, Huiyong Zhao, Hong Shao, Antonio Marra, Payal Patel, Elizabeth O’Connell, Emma Fink, Matthew M Miele, Zhuoning Li, Elisa De Stanchina, Emiliano Cocco, Pedram Razavi, Eneda Toska, Sean W Fanning, Guotai Xu, Anna A Sablina, Sarat Chandarlapaty. Targeting NR2F2 overcomes multiple forms of endocrine resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-03-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97507505295680ec94e45659684f5586171f75f5" target='_blank'>
              Abstract P2-03-17: Targeting NR2F2 overcomes multiple forms of endocrine resistance
              </a>
            </td>
          <td>
            Yanyan Cai, P. Zhao, Fan Wu, Huiyong Zhao, Hong Shao, Antonio Marra, Payal Patel, Elizabeth O'Connell, Emma C. Fink, Matthew M. Miele, Zhuoning Li, Elisa de Stanchina, Emiliano Cocco, P. Razavi, Eneda Toska, S. Fanning, Guotai Xu, A. Sablina, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Abstract Ageing is a gradual biological process marked by a decline in physiological function, increasing susceptibility to disease, and mortality. Transposable elements (TEs) are repetitive DNA sequences capable of moving within the genome and thus potentially inducing mutations and disrupting normal cellular functions. Their mobile nature contributes to genomic variation, as transposition events can alter gene expression, chromosome structure, and the epigenetic landscape. To mitigate TE-induced damage, cells rely on epigenetic mechanisms, such as DNA methylation, histone modifications, and small RNAs, to repress TE activity. However, these silencing mechanisms become less effective with age, leading to increased TE activation. This review explores the dual role of TEs as both a cause and consequence of ageing, suggesting a complex relationship between TEs and the ageing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca3b56d174d42a8f2b0813e61df3277c11215871" target='_blank'>
              Exploring the Relationship of Transposable Elements and Ageing: Causes and Consequences
              </a>
            </td>
          <td>
            Miriam Merenciano, Anaïs Larue, Chloé Garambois, William Vilas Boas Nunes, Cristina Vieira
          </td>
          <td>2025-05-15</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cytosine base editors enable programmable and efficient genome editing using an intermediate featuring a U•G mismatch across from a DNA nick. This intermediate facilitates two major outcomes, C•G to T•A and C•G to G•C point mutations, and it is not currently well-understood which DNA repair factors are involved. Here, we couple reporters for cytosine base editing activity with knockdown of 2015 DNA processing genes to identify genes involved in these two outcomes. Our data suggest that mismatch repair factors facilitate C•G to T•A outcomes, while C•G to G•C outcomes are mediated by RFWD3, an E3 ubiquitin ligase. We also propose that XPF, a 3’-flap endonuclease, and LIG3, a DNA ligase, are involved in repairing the intermediate back to the original C•G base pair. Our results demonstrate that competition and collaboration among different DNA repair pathways shape cytosine base editing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc2996866bb9608e216151dc0142fedfafc78c6d" target='_blank'>
              Elucidating the genetic mechanisms governing cytosine base editing outcomes through CRISPRi screens
              </a>
            </td>
          <td>
            Sifeng Gu, Z. Bodai, Rachel A Anderson, Hei Yu Annika So, Quinn T. Cowan, Alexis C. Komor
          </td>
          <td>2025-05-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In breast cancer related to the BRCA1 mutation, luminal progenitor cells are believed to be the cells of origin, yet how these cells transform into invasive cancer cells remain poorly understood. Here, we combine single-cell epigenomic and transcriptomic data to reconstitute sequences of events in luminal cells that lead to tumorigenesis. Upon deletion of Trp53 and Brca1, we find that luminal progenitors display an extensive epigenomic disorder associated with a loss of cell identity. These cells then progress to tumor formation through a partial epithelial-to-mesenchymal transition, orchestrated by Snail and the timely activation of immunosuppressive and FGF signaling with their microenvironment. In human samples, pre-tumoral changes can be detected in early stage, basal-like tumors, which rarely recur, as well as in normal-like mammary glands of BRCA1 mutation carriers who have had cancer. Our study fills critical gaps in our understanding of BRCA1-driven tumorigenesis, opening perspectives for the early monitoring of individuals with high cancer risk. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02331-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f69d9b899da12be47c2084b992747c9f0654fa" target='_blank'>
              Epigenomic disorder and partial EMT impair luminal progenitor integrity in Brca1-associated breast tumorigenesis
              </a>
            </td>
          <td>
            Camille Landragin, Mélissa Saichi, Marthe Laisné, Adeline Durand, Pacôme Prompsy, Renaud Leclère, Jérémy Mesple, Kyra Borgman, Amandine Trouchet, Marisa M Faraldo, A. Chiche, A. Vincent-Salomon, Hélène Salmon, Céline Vallot
          </td>
          <td>2025-04-28</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Chinese hamster ovary (CHO) cells are the primary host for biopharmaceutical production. To meet increasing demands for productivity, quality, and complex molecule expression, genetic engineering, particularly clustered regularly interspaced short palindromic repeats (CRISPR)‐mediated gene knockout (KO), is widely used to optimize host cell performance. However, systematic screening of KO targets remains challenging due to the labor‐intensive process of generating and evaluating individual clones. In this study, we present a robust, high‐throughput CRISPR workflow using stable KO pools in CHO cells. These pools maintain genetic stability for over 6 weeks, including in multiplexed configurations targeting up to seven genes simultaneously. Compared to clonal approaches, KO pools reduce variability caused by clonal heterogeneity and better reflect the host cell population phenotype. We demonstrate the utility of this approach by reproducing the beneficial phenotypic effects of fibronectin 1 (FN1) KO, specifically prolonged culture duration and improved late‐stage viability in fed‐batch processes. This workflow enables efficient identification and evaluation of promising KO targets without the need to generate and test large numbers of clones. Overall, screening throughput is increased 2.5‐fold and timelines are compressed from 9 to 5 weeks. This provides a scalable, efficient alternative to traditional clonal screening, accelerating discovery for CHO cell line engineering for biopharmaceutical development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6144e6cb50fe1bc2be0960973cca20d4b825428" target='_blank'>
              Utilizing Stable Gene‐Edited Knockout Pools for Genetic Screening and Engineering in Chinese Hamster Ovary Cells
              </a>
            </td>
          <td>
            J. Marzluf, Kirchmeier Daniela, Jennifer Klein, Christoph Zehe, Ann-Cathrin Leroux
          </td>
          <td>2025-05-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Do individual cancer cells learn to become therapy resistant?
 Do cells remember their past experiences? DNA is one way to encode these past experiences, but cannot generally be altered within a lifetime. How, then, can cells learn? We present data showing that cancer cells, in the context of therapy resistance, remember their past, and this past can influence their future. We find the molecular mechanism by which such regulatory flexibility is achieved. We think this mechanism may open new avenues for treating therapy-resistant tumors.


 Arjun Raj. Do individual cancer cells learn to become therapy resistant [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY41-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb5dda59dfa4344afba6cf797618f999a60a61b" target='_blank'>
              Abstract SY41-01: Do individual cancer cells learn to become therapy resistant
              </a>
            </td>
          <td>
            Arjun Raj
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 e14570


 Background:
 Immune checkpoint blockades (ICBs) showed little efficacy in pediatric solid tumors. The reconfiguration of gene regulation in cancers is a major factor in tumor immune microenvironment (TiME) remodeling and ICB resistance. Hypothesis: A subset of pediatric tumors may exhibit an epigenetically altered TiME, potentially responsible for ICB resistance. Objective: classify the TiME of pediatric extracranial solid tumors and describe the gene regulatory networks (GRNs) of immune landscapes.
 Methods:
 We studied bulk tumor gene expression and DNA-methylation from 184 pediatric extracranial solid tumors. Similarity network fusion (SNFtool) individualized TiME phenotypes by dual-omic clustering of immune genes. The clusters were compared by differential analysis for gene expression and methylation. We studied the relationship of enhancer methylation to gene expression (ELMER package) to infer methylation reprogramming. An enrichment study of regulatory binding regions (ReMapEnrich) identified the master regulators that define the GRNs unique to each phenotype.
 Results:
 SNF clustering identified 3 phenotypes with 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 exhibited low expression of immune genes (“cold” phenotype), cl2 overexpressed immune genes (“hot” phenotype), and cl3 featured global hypermethylation (epigenetically “altered” phenotype). Different tumor types were present across all phenotypes, but Ewing sarcoma was more frequent in altered, osteosarcoma and neuroblastoma in hot, and Wilms tumor in cold phenotype (
 p
 <0.05). Both hot and altered phenotypes overexpressed immune checkpoints (
 CD274
 ,
 PDCD1
 ), T-cell activator chemokines (
 CXCL9
 ,
 CXCL10
 ) and pro-inflammatory pathways, central to ICB sensitivity. However, only hot tumors overexpressed genes and pathways essential for antigen-presenting machinery and immune recognition. Methylation regulation was responsible for derepressing MHC-II genes and their regulators in hot tumors. In contrast, gene silencing in altered phenotype relied on hypermethylation/downregulation of the CIITA cofactor domain, the master control of MHC-II genes, and other immune-specific master regulators crucial for anti-tumor immunity and ICB sensitivity. The regulatory landscape of immune gene repression in the altered phenotype involved binding regions specific to the MYC/MAX network and the polycomb-group protein PCR2 (
 i.e.,
 EZH2 and SUZ12).
 Conclusions:
 We demonstrated that GRN reconfiguration participates in TiME reshaping in pediatric extracranial solid tumors. A subset of tumors with epigenetically altered immune phenotype has an immune recognition capacity repressed by MYC/MAX and PCR2 complexes. Future studies should investigate specific inhibitors to reprogram the GRN to foster immune recognition and ICB sensitivity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b6a52d83f8f6dace4bda8bd4ea08fd3f518537c" target='_blank'>
              Dual-omic analysis to reveal gene regulatory network of immune landscape in childhood solid tumors and implications for immunotherapy.
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Mona Patoughi, Thibault Mallevaey, Lara Herrmann, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Noel Raynal, Vincent-Philippe Lavallée, Sonia Cellot, T. Tran, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9cc09a30e02fe7681c9c9c4146bae9713fa5e17" target='_blank'>
              Integrative CRISPR Screens and RNA-Omics Discover an Essential Role for PUF60-3’ Splice Site Interactions in Cancer Progression
              </a>
            </td>
          <td>
            Alexandra T. Tankka, J. Einstein, Cathy Zhou, Vivian N. Pham, Yuhan Zhang, Jack T. Naritomi, Grady G. Nguyen, Orel Mizrahi, Mark Perelis, Joseph Sarsam, Frederick E. Tan, D. Kaufman, Corina E. Antal, Gene W Yeo
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) modification, one of the most prevalent RNA epigenetic modifications in eukaryotes, constitutes over 60% of all RNA methylation modifications. This dynamic modification regulates RNA processing, maturation, nucleocytoplasmic transport, translation efficiency, phase separation, and stability, thereby linking its dysregulation to diverse physiological and pathological processes. METTL3, a core catalytic component of the methyltransferase complex responsible for m6A deposition, is frequently dysregulated in diseases, including colorectal cancer (CRC). Although METTL3’s involvement in CRC pathogenesis has been documented, its precise molecular mechanisms and functional roles remain incompletely understood. METTL3 mediates CRC progression-encompassing proliferation, invasion, drug resistance, and metabolic reprogramming-through m6A-dependent modulation of both coding RNAs and noncoding RNAs. Its regulatory effects are primarily attributed to interactions with key signaling pathways at multiple stages of CRC development. Emerging evidence highlights METTL3 as a promising biomarker for CRC diagnosis and prognosis, as well as a potential therapeutic target. By synthesizing recent advances in METTL3 research within CRC, this review provides critical insights into novel strategies for clinical diagnosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ccdd33e7855e192e72bc49bfec42dd0313d9f3f" target='_blank'>
              Switching from messenger RNAs to noncoding RNAs, METTL3 is a novel colorectal cancer diagnosis and treatment target
              </a>
            </td>
          <td>
            Junnan Liao, Wen-Juan Ni, Ping-Hui Wu, Ya-Dong Yang, Ying Yang, Wen Long, Mei-Zhen Xie, Xiu-Zhi Zhu, Fuhua Xie, Xiao-Min Leng
          </td>
          <td>2025-05-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Melanoma brain metastases (MBM) are a leading cause of death in patients with advanced disease. MBM treatment relay on targeted and immunotherapy and on stereotactic radiosurgery as gold standard. Life expectancy has improved significantly with these therapies however, targeted therapy is short lived and only about half of the patients respond to immunotherapy, while radiation is limited by melanoma cells intrinsic resistance to DNA damage. New therapeutic approaches are therefore needed to treat MBM. Here we investigate a new role of Notch1 in genomic instability and demonstrate that blockade of both Notch1 and the DNA repair factor ChK1 causes extensive DNA damage and tumor cell death increasing survival in MBM bearing mice. Methods Anti-Notch1 (anti-N1) was previously described. Prexaserib, a ChK1 inhibitor, is currently in clinical trials. K457 and A375 melanoma cells were used. RNA sequencing was performed in K457 cells treated with anti-N1 and Gene Set Enrichment Analysis performed. DNA damage was evaluated by a DNA fiber assay to assess replication fork speed; and γH2AX foci count and neutral comet assay to quantify double strand breaks. Cell survival was evaluated by trypan blue and a colony formation assay. Luciferase expressing A375 cells were orthotopically inoculated in the right cerebral cortex of athymic nude mice, for in vivo evaluation of a therapy with anti-N1 and prexasertib. Survival was assessed by Kaplan-Meyer survival curves and significance assessed by a Log-rank test. Results Notch1 blockade caused genomic instability by reducing histone availability, leading to DNA replication stress and DNA damage. This in turn, resulted in the activation of the DNA Damage Response pathway ATR/ChK1 to counter the damage. Co-inhibition of Notch1, via anti-N1, and ChK1, via prexasertib (prex), exacerbated DNA damage increasing melanoma cell death. Importantly, combination anti-N1/prex significantly improved survival of mice bearing MBMs. Conclusions A therapy with anti-N1/prexasertib could represent a novel treatment strategy, alone or in combination with current treatment regimens, for melanoma brain metastases. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03411-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a360dee32c6a48d005c164a15d89bdb2679317f" target='_blank'>
              Co-inhibition of Notch1 and ChK1 triggers genomic instability and melanoma cell death increasing the lifespan of mice bearing melanoma brain metastasis
              </a>
            </td>
          <td>
            Varsha Thakur, Vijay S Thakur, Dazhi Wang, Juliano Tiburcio de Freitas, Anna Bianchi, Luis Nivelo, Oliver Umland, S. Welford, Barbara Bedogni
          </td>
          <td>2025-05-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Research on myeloid neoplasms, a field that has been driving scientific advances in cancer for over 50 years, has yielded many discoveries that have fundamentally reshaped our understanding of cancer biology. These insights, often the product of leukemia research, have been instrumental in developing more mechanism-based treatments in the early 2000s [1]. Recognizing epigenetic dysregulation as a common disease mechanism in myeloid cancers has been groundbreaking regarding recent treatment developments that exploit chromatin-based oncogenic mechanisms. In the case of acute myeloid leukemia (AML), sequencing studies aimed at assessing the complement of genetic alterations demonstrated that more than 60% of AML cases harbored disease-driving mutations in epigenetic regulators. This high prevalence underscores the importance of epigenetic dysregulation in AML pathogenesis [2, 3]. Chromatin regulators commonly control disease-specific transcriptional programs, making them attractive therapeutic targets to manipulate neoplastic gene expression programs, particularly in myeloid neoplasms. Several drugs targeting epigenetic mechanisms and exploiting myeloid disease-specific dependencies have recently been approved for treating myeloid neoplasms. Many additional drugs are currently being investigated in clinical trials, and numerous new compound developments are being studied in preclinical studies. This manuscript will review (1) chromatin-based disease mechanisms, such as DNA methylation, chromatin regulatory complexes, and histone modifications, currently investigated for therapeutic exploitation in myeloid malignancies, and (2) therapeutic developments already approved or investigated for treating these diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9903badc85679a15da9a5a2e6c4c1cc4ffc6496" target='_blank'>
              The evolving landscape of epigenetic target molecules and therapies in myeloid cancers: focus on acute myeloid leukemia and myeloproliferative neoplasms.
              </a>
            </td>
          <td>
            Michael W M Kühn, Naveen Pemmaraju, F. Heidel
          </td>
          <td>2025-05-15</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Non-replicative herpes simplex virus type 1 (nrHSV-1) vectors are promising delivery vehicles for gene therapy due to their large DNA payload capacity and ability to infect a broad range of cell types. However, the genomic deletions made to generate such nrHSV-1 vectors can result in undersized genomes that trigger genomic instability—including rearrangements and size extensions—compromising their therapeutic potential. This study investigates the stabilization of undersized nrHSV-1 vectors through the insertion of stuffer DNA segments. We assess genomic stability, productivity, toxicity, and transgene expression in vitro and in vivo. Our findings demonstrate that nrHSV-1 can accommodate variations in genome size up to 5–6% and highlight the importance of maintaining a genome size close to that of the wild-type HSV-1 for enhanced genomic stability and sustained transgene expression without adverse effects. This strategy offers a promising approach for optimizing nrHSV-1 vectors for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63984927847424424cc3892babc095b69c28460a" target='_blank'>
              Genomic Size Is Critical to Guarantee the Genomic Stability of Non-Replicative HSV1 Vectors
              </a>
            </td>
          <td>
            Justine Basset, Alexandra Seraffin, Julien Ratelade, Yohann Dickx, Tomasz Benedyk, Grzegorz Sarek, Teddy Jégu, Alberto L. Epstein
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Potassium ions (K+) within the tumor microenvironment, along with dysregulation of K+ channels, play critical roles in supporting cancer cell survival and preventing their elimination. Directly monitoring changes in K+ homeostasis within cancer cells is invaluable for understanding these processes. However, achieving high selectivity over other biological metal ions, a detection dynamic range that aligns with intracellular K+ levels, and broad accessibility to research laboratories remain technically challenging for current K+ imaging probes. In this study, we report the in vitro selection of the first K+-specific RNA-cleaving DNAzyme and the development of a K+-specific DNAzyme fluorescent sensor with exceptional selectivity, achieving over 1000-fold selectivity against Na+ and more than 100-fold selectivity over other major biologically relevant metal ions. This sensor has an apparent dissociation constant (105 mM) that is close to the intracellular level of K+, and it has a broad detection range from 21 to 200 mM K+. Using this tool, we reveal a progressive decline in intracellular K+ levels in breast cancer cells with more advanced progression states. Moreover, we demonstrate that elevated extracellular K+ levels interfere with the efficacy of anticancer compounds like ML133 and Amiodarone, suggesting an underappreciated role of microenvironmental K+ in chemoresistance. Notably, blocking the Kir2.1 channel activity restored treatment sensitivity, presenting a potential strategy to overcome chemoresistance in aggressive cancers. These findings underscore the role of K+ homeostasis in tumor progression and support further exploration of ion-channel-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b15a5b2af9bd939cc77bcb6b3ee460a1e0556b12" target='_blank'>
              Decoding Potassium Homeostasis in Cancer Metastasis and Drug Resistance: Insights from a Highly Selective DNAzyme-Based Intracellular K+ Sensor.
              </a>
            </td>
          <td>
            Zhenglin Yang, Xiangli Shao, Yuting Wu, Aritra Roy, Elijah Garcia, Annie Farrell, Shreestika Pradhan, Weijie Guo, Heather Gan, Zeynep Korkmaz, Emily Adams, Yi Lu
          </td>
          <td>2025-05-14</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mutation spectra vary across genetic and environmental contexts, leading to differences between and within species. Most research on mutation spectrum has focused on the trinucleotide (3-mer) mutation types in mammals, limiting the breadth and depth of variation surveyed. In this study, we use whole-genome resequencing data across 108 eukaryotic species – including mammals, fish, plants, and invertebrates – to characterize pentanucleotide (5-mer) non-coding mutation spectra using a Bayesian approach. Our findings reveal cytosine transition mutability at CpG and (among plants) at CHG sites as the main drivers of variation in mutation spectra across eukaryotes, correlating strongly with genomic CpG and CHG depletion. However, despite the influence of methylation on CpG mutability, genome-wide average CpG methylation levels do not predict CpG transition rates across species and CHG methylation does not predict CHG transition rate, indicating unknown genetic or environmental factors influencing mutation rates at methylated cytosines. Together, our results illustrate the pivotal role of mutagenesis in shaping genome composition across eukaryotes and highlight a gap in knowledge about the mechanisms governing mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840be1a2fe72dda5c8b163dbfd7199ef76a61434" target='_blank'>
              Methylation-associated mutagenesis underlies variation in the mutation spectrum across eukaryotes
              </a>
            </td>
          <td>
            Fabián Ramos-Almodóvar, Ziyue Gao, Benjamin F. Voight, I. Mathieson
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a400876e8f32c63411441923e6bbb8ebd5d239" target='_blank'>
              Chromosome-Specific Aneuploidy Engineering via dCas9-Induced Centromeric Chromatin Relaxation
              </a>
            </td>
          <td>
            Hugang Feng, Daqi Deng, Rashmi Dahiya, Libin Wang, Jingkun Zeng, Benjy Jek Yang Tan, F. Byrne, Scott T C Shepherd, Jiahao Wang, Sarah C. Johnson, Alison Harrod, Karen A. Lane, Annika Fendler, A. Cattin, Z. Tippu, Meilun Nie, Yiming Zhao, Ruijia Wang, Wei Ai, Omar Bouricha, Taja Barber, Yuliia Dovga, Yihan Xu, Yimeng Xu, Liani G. Devito, Lyn Healy, Zhen Sun, Weiming Shen, Hongye Wang, Eugénie S. Lim, Sam O'Toole, Scott Akker, William M. Drake, Haojie Jin, Jessica A. Downs, Sarah E. McClelland, J. Diffley, Peter Ly, S. Turajlic
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="
 In cancer research, perturbing gene function is crucial for understanding disease mechanisms and developing treatments. Techniques such as RNA interference and CRISPR-knockout, -interference, or -activation are commonly used. In this recent study we compared the phenotypic effects of cellular morphology by knocking out or knocking down key cell cycle regulators, Aurora kinase B (AURKB), Geminin (GMNN), and Polo-like kinase 1 (PLK1), which are known key critical proteins involved in cell cycle regulation, and their dysfunction linked to cancer progression. In this in vitro osteosarcoma cell line (U 2-OS) model, we inhibited these proteins using CRISPR-Cas9 for gene knockout or siRNA for mRNA knockdown in a comparative study not previous reported. Employing Cell Painting methodology (6 fluorescent probes, high content screening (HCS) confocal imaging, and multivariant analysis) we imaged and profiled the morphological phenotypes following gene perturbations. Principal component analysis (PCA) analysis of hundreds of HCS image phenotypic feature measurements revealed significant morphological changes and reduced cell numbers from AURKB, GMNN, and PLK1 inhibition in U 2-OS cells caused significant phenotypic changes and reduced cell numbers suggesting alteration of cell cycle progression. Interestingly, in the GMNN inhibition, the siRNA-treated cells showed a higher percentage of large nuclei compared to small nuclei from CRISPR-treated cells. These findings suggest siRNA-mediated knockdown has a faster inhibitory effect on GMNN than CRISPR-treated conditions. This study demonstrates the complementary nature of CRISPR-Cas9 and siRNA gene modulation and underscores the importance of orthogonal validation for robust genetic studies with key cell cycle regulators.


 Alexander Schreiner, Angelika Foitzik, Josien Levenga, Karin Boettcher, Barbara Sonnenberg, Žaklina Strezoska, Joe Trask. Unlocking cell cycle dynamics: Gene regulation of CRISPR-Cas9 and siRNA phenotypes using cell painting [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB443.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18ca3a1f08d426b3a13684391aa11a6fb1c8556a" target='_blank'>
              Abstract LB443: Unlocking cell cycle dynamics: Gene regulation of CRISPR-Cas9 and siRNA phenotypes using cell painting
              </a>
            </td>
          <td>
            Alexander Schreiner, A. Foitzik, J. Levenga, Karin Boettcher, Barbara Sonnenberg, Z. Strezoska, Joe Trask
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genotoxic stress-induced stem cell maldifferentiation (GSMD) integrates DNA damage responses with loss of stemness and lineage-specific differentiation to prevent damaged stem cell propagation. However, molecular mechanisms governing GSMD remain unclear. Here, we identify the p53-induced long non-coding RNA LOC644656 as a key regulator of GSMD in human embryonic stem cells. LOC644656 accumulates in the nucleus upon DNA damage, disrupting pluripotency by interacting directly with POU5F1 and KDM1A/LSD1-NuRD complexes, repressing stemness genes, and activating TGF-β signaling. Additionally, LOC644656 mitigates DNA damage by binding DNA-PKcs and modulating the DNA damage response. In cancer, elevated LOC644656 correlates with poor patient survival and enhanced chemoresistance. Our findings demonstrate that LOC644656 mediates stemness suppression and resistance to genotoxic stress by coordinating DNA damage signaling and differentiation pathways. Thus, LOC644656 represents a potential therapeutic target for overcoming chemoresistance and advancing stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70a2bd396273c74fb23a4471cceb78f9c794e8d" target='_blank'>
              p53-inducible lncRNA LOC644656 causes genotoxic stress-induced stem cell maldifferentiation and cancer chemoresistance
              </a>
            </td>
          <td>
            Ai Tamura, Kazuyuki Yamagata, Takashi Kono, Masanori Fujimoto, Takahiro Fuchigami, Motoi Nishimura, Masataka Yokoyama, Akitoshi Nakayama, N. Hashimoto, I. Sakuma, Nobuyuki Mitsukawa, Yusuke Kawashima, O. Ohara, S. Motohashi, Eiryo Kawakami, Takashi Miki, Atsushi Onodera, Tomoaki Tanaka
          </td>
          <td>2025-05-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 9520


 Background:
 HRDsig can aid in identifying tumors with DNA repair deficiencies, guiding PARP inhibitor (PARPi) use. It expands treatment options beyond breast cancer gene (BRCA) mutations enabling personalized therapy. Combining PARPi with targeted therapy or immunotherapy shows promise in overcoming resistance and improving outcomes for treatment-resistant ACM.
 Methods:
 Formalin-fixed paraffin-embedded (FFPE) primary tumors and metastatic biopsies from 9,576 ACM cases were analyzed using hybrid capture-based comprehensive genomic profiling (CGP), evaluating all classes of genomic alterations (GA). Central pathology review was conducted for all cases. Microsatellite instability-high (MSIH) status, tumor mutational burden (TMB), genomic ancestry, and mutational signatures were derived from sequencing data. HRDsig was assessed using copy number changes and genomic scars. PD-L1 expression was determined via immunohistochemistry [Dako 22C3; tumor proportion score (TPS)]. Statistical comparisons were made using Fisher’s exact test with Benjamini-Hochberg correction.
 Results:
 Among 9,576 ACM cases, 198 (2.1%) were HRDsig positive (HRDsig+). HRDsig+ cases when compared to HRDsig negative (HRDsig-) were older (median age: 69 vs. 67 years; p=0.034), more often female (49.5% vs. 36.2%; p=0.001), and had more GA per tumor (median: 7 vs. 6; p=0.026). HRDsig+ cases were more often of African (5.6% vs. 1.2%; p<0.0001) or American ancestry (8.1% vs. 4.1%; p=0.026) and less often European (84.3% vs. 94.0%; p<0.0001). GA more common in HRDsig+ included
 IGF1R
 (5.1% vs. 1.3%; p=0.003),
 KIT
 (11.6% vs. 5.6%; p=0.004),
 KRAS
 (7.1% vs. 2.7%; p=0.005),
 NF1
 (38.4% vs. 21.4%; p<0.0001),
 RAD21
 (6.7% vs. 3.0%; p=0.037), and
 TP53
 (40.4% vs. 24.3%; p<0.0001). HRDsig- cases exhibited higher TMB (median: 13.8 vs. 6.1; p<0.0001), more frequent TMB >10 mutations/Mb (60.9% vs. 39.9%; p<0.0001), and more UV light exposure trinucleotide signatures (57.3% vs. 36.4%; p<0.0001). GA more common in HRDsig- cases included
 BRAF
 (including V600E) (44.6% vs. 23.2%; p<0.0001),
 CDKN2A
 (49.2% vs. 36.9%; p=0.003),
 NRAS
 (28.0% vs. 11.6%; p<0.0001), and
 TERT
 (74.4% vs. 29.8%; p<0.0001). In both positive and negative groups, PD-L1 low-level expression (1-49% TPS) was comparable (43.3%-39.7%), while BRCA1/2 alterations (<2.0%) and MSIH status (0.0%-0.1%) remained rare.
 Conclusions:
 HRDsig+ status is rare in ACM but more frequent in non-white genomic ancestries. HRDsig+ ACM cases are less likely to have BRAF GA, more likely to have KIT GA and lower TMB levels. These findings may guide the future development of clinical trials employing combination therapies and PARPi in ACM.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/162f9c9205bbddc1331c24e0a0b790056a2c6e2c" target='_blank'>
              Homologous recombination deficiency signature (HRDsig) in advanced cutaneous melanoma (ACM): A genomic landscape study.
              </a>
            </td>
          <td>
            Nimisha Srivastava, D. Pavlick, E. Sokol, O. Gjoerup, J. Elvin, Ryon P. Graf, Gerald Li, J. Quintanilha, Sanchit Panda, J. Ross, Jade Homsi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Since its advent about ten years ago, the CRISPR-Cas9 system has been frequently used in biomedical applications. It has advanced various fields, and CRISPR-Cas9-based therapeutics have shown promising results in the treatment of specific hematological diseases. Furthermore, CRISPR gene editing technologies have revolutionized cancer research by enabling a broad range of genetic perturbations, including genetic knockouts and precise single nucleotide changes. This perspective focuses on the state-of-the-art methodology of CRISPR knock-ins to engineer immune cells. Since this technique relies on homology-directed repair (HDR) of double-strand breaks (DSBs) induced by the Cas9 enzyme, it can be used to introduce specific mutations into the target genome. Therefore, this methodology offers a valuable opportunity to functionally study specific mutations and to uncover their impacts not only on overall cell functions but also on the mechanisms behind cancer-related alterations in common signaling pathways. This article highlights CRISPR knock-in strategies, protocols, and applications in cancer and immune research, with a focus on diffuse large B cell lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15e1b343be2ebd40cfad2b5fa562b6f9cb4962e" target='_blank'>
              Strategies for CRISPR-based knock-ins in primary human B cells and lymphoma cell lines
              </a>
            </td>
          <td>
            Sophie Lund, Chun Gong, Xin Yu, Louis M. Staudt, Daniel J Hodson, Sebastian Scheich
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Despite major advances in genetic screening technology, a formal approach for quantifying gene function remains underdeveloped, thereby limiting the utility of these techniques in deciphering the complex behavior of human cells. In this study, we leverage information theory with a perturbational analysis of replicator dynamics to characterize functional drivers of selection in pooled CRISPR screens. Our approach challenges established methods for CRISPR screen analysis, while offering additional insight into selection dynamics through the Kullback-Leibler divergence (DKL) and cumulants of the fitness distribution. By modeling fluctuations in gene-fitness effects as a linear response to environmental perturbations, we derive a geometric measure for genomic information content based on a second-order approximation of the DKL. Our analysis reveals that functional information—encoded (or shared) between genes—can be quantified by analyzing the directions corresponding to maximal conditional selection within the space of decomposed gene-environment interactions. This geometric representation offers several advantages for the functional analysis of the human genome and its network architecture. Moreover, by constraining the space to cell-type-specific fluctuations, we uncover developmental and tissue-specific functional signatures. These findings represent significant progress in the dynamic analysis of gene function and in the functional wiring of the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4334ca7a2aab75bdf008ae60416f93b19bfd20e8" target='_blank'>
              Perturbational fitness analysis of CRISPR screens uncovers information-theoretic relation between gene function and selection
              </a>
            </td>
          <td>
            A.N. Andersen, N. Chica, L. Piechaczyk, S. Nakken, M. Zucknick, J. Enserink
          </td>
          <td>2025-05-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cell heterogeneity is a universal feature of life. Although biological processes affected by cell-to-cell variation are manifold, from developmental plasticity to tumour heterogeneity and differential drug responses, the sources of cell heterogeneity remain largely unclear1,2. Mutational and epigenetic signatures from cancer (epi)genomics are powerful for deducing processes that shaped cancer genome evolution3–5. However, retrospective analyses face difficulties in resolving how cellular heterogeneity emerges and is propagated to subsequent cell generations. Here, we used multigenerational single-cell tracking based on endogenously labelled proteins and custom-designed computational tools to elucidate how oncogenic perturbations induce sister cell asymmetry and phenotypic heterogeneity. Dual CRISPR-based genome editing enabled simultaneous tracking of DNA replication patterns and heritable endogenous DNA lesions. Cell lineage trees of up to four generations were tracked in asynchronously growing cells, and time-resolved lineage analyses were combined with end-point measurements of cell cycle and DNA damage markers through iterative staining. Besides revealing replication and repair dynamics, damage inheritance and emergence of sister cell heterogeneity across multiple cell generations, through combination with single-cell transcriptomics, we delineate how common oncogenic events trigger multiple routes towards polyploidization with distinct outcomes for genome integrity. Our study provides a framework to dissect phenotypic plasticity at the single-cell level and sheds light onto cellular processes that may resemble early events during cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="Over the past half-century of the global effort against cancer, the vast majority of investigations in both tumor basic research and clinical practice have centered on the "somatic mutation" theory, such as in molecular classification, individualized precision medicine strategies, gene therapy approaches, the development of neoantigen-based tumor vaccines, and advancements in sequencing technologies. Even in the extensively studied tumor microenvironment (including tumor immunity), which has garnered significant attention in recent years, the underlying mechanisms frequently revert to specific genes and mutations within tumor cells or microenvironmental cells as the primary driving forces. However, despite the dominance of the "somatic mutation" paradigm, truly effective approaches for curing cancer in clinical settings remain elusive. Undoubtedly, if the prevailing "somatic mutation theory" continues to monopolize cancer research, meaningful progress in understanding and treating cancer will likely remain frustratingly out of reach. At this critical juncture in the evolution of cancer research, a comprehensive re-evaluation of cancer not only is necessary but also imperative, highlighting the urgent need for a profound transformation in our conceptual framework. This article systematically elucidates the novel perspective offered by the "tumor system" for comprehending the essence of cancer, the foundational principles of "tumor ecology" and their potential applications in treatment, and explores in depth the theoretical framework and research significance of the emerging field of "ecological pathology". Beyond merely advocating for the abandonment of the currently dominant linear reductionist paradigm of cancer, this commentary strives to construct a pragmatic and systematically structured framework to guide the trajectory of the "post-genomic revolution in oncology" and the "tumor ecological philosophy", ultimately fostering the realization of the overarching societal goal of eradicating cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbba50cb169d2a51d829cb21c736013cc01b71a8" target='_blank'>
              [Rethinking cancer].
              </a>
            </td>
          <td>
            W. R. Luo
          </td>
          <td>2025-05-28</td>
          <td>Zhonghua zhong liu za zhi [Chinese journal of oncology]</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

Recent advances in CRISPR–Cas9–based genome editing have revolutionized cancer
immunotherapy by enabling precise and efficient T-cell engineering. This review focuses on how
CRISPR-mediated modifications enhance T-cell functionality, highlighting key applications, such
as CAR-T cells and TCR-engineered T-cell therapies. By selectively targeting immune-regulating
genes, CRISPR empowers T cells to recognize and eliminate tumor cells with greater specificity
and reduce off-target effects. This precision opens the door to personalized treatment strategies, allowing
T cells to be tailored to each patient’s unique tumor profile. Furthermore, the ability to
rapidly and systematically edit multiple gene targets has accelerated the development of more potent
and durable T-cell therapies, overcoming some of the current limitations in cancer immunotherapy.
Overall, CRISPR’s transformative potential of CRISPR provides a platform for next-
generation patient-centric cancer treatments, paving the way for improved clinical outcomes and
expanded therapeutic options.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcb955649c8f44c414ba7f048441db68da50f99" target='_blank'>
              Innovation and Implication of CRISPR-Mediated T-cell Engineering in Cancer Immunotherapy: A New Insight
              </a>
            </td>
          <td>
            Syeda Areej Imran, H. Sultan, Muhammad Zubair, S. Ghazanfar, Anas Islam, Syed Mustafizur Rahaman, Sumel Ashique
          </td>
          <td>2025-06-03</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aa2044a37bf408a983f6463cafdba0b09b74c94" target='_blank'>
              A resource and computational approach for quantifying gene editing allelism at single-cell resolution
              </a>
            </td>
          <td>
            Matthew Ung, Gabriella Angelini, Ruijia Wang, Alyssa Pyclik, J. Lydeard, J. Ferrucio, Michelle I. Lin, Tirtha Chakraborty, Huanying Gary Ge
          </td>
          <td>2025-06-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0036830d7ca0d3dd2d206e6880d31ee221a0279" target='_blank'>
              Advances in single-cell DNA sequencing enable insights into human somatic mosaicism.
              </a>
            </td>
          <td>
            Diane D. Shao, Andrea J. Kriz, Daniel A. Snellings, Zinan Zhou, Yifan Zhao, Liz Enyenihi, Christopher A. Walsh
          </td>
          <td>2025-04-25</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0631bffe60efc9e321a6ea4a06deb1f550391138" target='_blank'>
              Clonal effects of the Ras oncogene revealed by somatic mutagenesis in a Drosophila cancer model
              </a>
            </td>
          <td>
            Takuya Akiyama, M. Gibson
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Gemcitabine (GEM) is used as a first-line therapy for patients diagnosed with any stage of pancreatic cancer (PC); however, patient survival is poor because of GEM resistance. Thus, new approaches to overcome GEM resistance in PC are urgently needed. Here, we aimed to establish an in vivo drug-resistant PC model and identify genes involved in GEM resistance. We focused on one of these factors, CITED4, and elucidated its mechanisms of action in GEM resistance in PC. Methods L3.6pl, a GEM-sensitive PC cell line, was orthotopically injected into the pancreas of BALB/c nude mice to establish a GEM-resistant PC animal model. Transcriptomic data from control or GEM-resistant tumor-derived cells were analyzed. GEM resistance was evaluated using cell viability, clonogenicity, and apoptosis assays. An apoptosis array was used to identify genes downstream of CITED4. A CITED4 knockout-mediated GEM sensitivity assay was performed in an orthotopic xenograft mouse model using PANC-1 cells, which are GEM-resistant cells. Results From the RNA sequencing data of isolated GEM-resistant PC cells and The Cancer Genome Atlas dataset, 15 GEM resistance-related genes were found to be upregulated, including CITED4, the gene encoding a type of CBP/p300-interacting transactivator implicated in several cancers. CITED4 knockdown in drug-resistant cells reduced cell proliferation and migration but increased apoptosis. To identify the molecular mechanism underlying CITED4-mediated induction of GEM resistance, alterations in Baculoviral IAP Repeat Containing 2 (BIRC2) levels were observed using an apoptosis array. BIRC2 expression was downregulated following CITED4 knockdown in GEM-resistant PC cell lines. Furthermore, chromatin immunoprecipitation and promoter assays showed that BIRC2 was directly regulated by CITED4. Consistent with the CITED-knockdown experiments, silencing of BIRC2 increased the sensitivity of L3.6pl-GEM-resistant and PANC-1 cell lines to GEM. Furthermore, CITED4 knockout using the CRISPR-Cas9 system in PANC-1 cells increased the sensitivity to GEM in orthotopic mice. Moreover, elevated CITED4 and BIRC2 expression levels were associated with poorer outcomes in human PC clinical samples. Conclusions Collectively, these results indicate that CITED4 regulates GEM resistance via inhibition of apoptosis by upregulating BIRC2 expression in PC cells. Therefore, CITED4 may serve as a valuable diagnostic marker and therapeutic target for GEM-resistant PC. Supplementary Information The online version contains supplementary material available at 10.1186/s12929-025-01140-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996f9fcf7ebe507f3a153f5e021e2017337d0d54" target='_blank'>
              Gemcitabine resistance by CITED4 upregulation via the regulation of BIRC2 expression in pancreatic cancer
              </a>
            </td>
          <td>
            Eun-Jeong Jeong, Yuna Roh, Eunsun Jung, Jin-Seong Hwang, Taesang Son, Hyun Seung Ban, Tae-Su Han, Young-Kug Choo, Jang-Seong Kim
          </td>
          <td>2025-05-19</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Somatic mosaicism is an important mechanism of immune dysregulation and explains a growing proportion of the 70% of patients with suspected IEI who lack a molecular diagnosis. Identifying disease-causing somatic mutations is challenging due to the presence of such variants in a small subset of cells, leading to a low variant allele fraction (VAF).
 We developed a custom 71-gene capture-based panel for high-depth sequencing of genes known or hypothesized to be associated with dominant immune dysregulation. Sequencing was performed with samples from 223 patients with immune dysregulation phenotypes and 96 currently healthy individuals. Mean age was 24.5 years for patients and 17.8 years for healthy individuals. 52.5% of patients and 40.6% of healthy individuals were biologic males. Coverage across coding regions of targeted genes had an average depth of 5752×. Our custom analysis pipeline utilized the union of multiple variant callers to call somatic variants.
 We identified 36 somatic variants in 21 genes from 28 patients (12.6% of patients) and 6 healthy controls (6.25% of healthy). To validate our assay, an independent library was sequenced for 60 patients and 18 healthy controls, including samples harboring 27 somatic variants. Droplet digital PCR (ddPCR) was used as an orthogonal method to validate 17 variants. All tested variants were confirmed as somatic. Genes with somatic mosaicism in >1 affected patient included FAS, STAT3, CARD11, NRAS, TNFAIP3, NLRP3, and PIK3CD. Three variants were previously described to cause somatic immune dysregulation. Immune cell populations from 7 patients with variants in FAS, NRAS, TNFAIP3, and UBA1 were purified and ddPCR performed, which identified enrichment of somatic variants in different cell types. This included somatic variants in FAS that had a VAF of 5-7% in whole blood but 21-28% in DNTs, all in individuals with a clinical phenotype consistent with ALPS.
 Our studies demonstrate the utility of high-depth targeted sequencing of suspected IEI patients to identify pathogenic and potentially pathogenic somatic variants, including low VAF variants directly from whole blood. Ongoing studies include expanding our gene panel, in vitro functional validation of novel variants, and utilizing single-cell technologies to identify the functional impact of somatic variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c50140965edf1ee7a8dc5d1ad463533f2e158f1a" target='_blank'>
              Deep Sequencing Reveals Somatic Mosaicism in Genes Relevant to Genetic Errors of Immunity in a Cohort of Patients with Immune Dysregulation
              </a>
            </td>
          <td>
            Elizabeth G Schmitz, Alexander Paul, Rajarshi Ghosh, Nermina Saucier, Ana Kolicheski, Sammy Risma, Tiphanie Vogel, Eric Allenspach, David Hoytema van Konijnenburg, C. Platt, Megan Elkins, Jolan Walter, Katie Lewis, Adriana de Jesus, Raphaela Goldbach-Mansky, J. Durkee-Shock, Jeffrey Cohen, Irini Sereti, Andrea Lisco, Ottavia Delmonte, Luigi Notarangelo, V. Rao, Morgan Similuk, Steven Holland, M. Griffith, O. Griffith, Magdalena A. Walkiewicz, Megan Cooper
          </td>
          <td>2025-04-25</td>
          <td>Journal of Human Immunity</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The transcription factor PAX5 is a major target of genetic alterations in human B-cell precursor acute lymphoblastic leukemia (B-ALL). Among the alterations, the P80R mutation affecting the DNA-binding domain represents the most frequent PAX5 point mutation in B-ALL. In contrast to other somatic PAX5 mutations, PAX5P80R defines a distinct B-ALL subtype characterized by a unique transcriptional program. Here, we aimed to develop a model to elucidate the mechanism by which PAX5P80R perturbs normal B-cell differentiation and the oncogenic relays involved in PAX5P80R-driven malignant progression. A retroviral complementation approach of Pax5-deficient murine fetal liver cells demonstrated at the functional and molecular levels that PAX5P80R failed to rescue definitive B-cell commitment but maintained the repression of T-cell development. Moreover, PAX5P80R eventually led to clonal B-ALL transformation after transplantation through the acquisition of secondary mutations in genes involved in the JAK/STAT and RAS/MAPK pathways. Finally, transcriptomic analyses combined with pharmacological investigation revealed ectopic activation of HIF2α as a common feature of B-ALL and identified acriflavine as a potent drug against B-ALL. Hence, this study provides a strategy to model the multistep process of B-ALL and sheds light on the biological mechanism by which the PAX5P80R mutation leads to leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0667740b0e5a9a582710e0b5a8d5562613a3f309" target='_blank'>
              Modeling the PAX5P80R Mutation Reveals HIF2α Activation as a Common Feature and Therapeutic Target in B-Cell Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            Manon Bayet, Vincent Fregona, Mathieu Bouttier, Clémence Rouzier, Jérémy Bigot, Laura A Jamrog, Sylvie Hebrard, N. Prade, Stéphanie Lagarde, Christine Didier, Stéphanie Gachet, M. Passet, Laetitia Largeaud, Marlène Pasquet, A. Khamlichi, E. Clappier, Eric Delabesse, C. Broccardo, Bastien Gerby
          </td>
          <td>2025-05-12</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="CRISPR technology has revolutionized genome editing by enabling precise, permanent modifications to genetic material. To circumvent the irreversible alterations associated with traditional CRISPR methods and facilitate research on both essential and nonessential genes, CRISPR interference or inhibition (CRISPRi) and CRISPR activation (CRISPRa) were developed. The gene-silencing approach leverages an inactivated Cas effector protein paired with guide RNA to obstruct transcription initiation or elongation, while the gene-activation approach exploits the programmability of CRISPR to activate gene expression. Recent advances in CRISPRi technology, in combination with other technologies (e.g., biosensing, sequencing), have significantly expanded its applications, allowing for genome-wide high-throughput screening (HTS) to identify genetic determinants of phenotypes. These screening strategies have been applied in biomedicine, industry, and basic research. This review explores the CRISPR regulation mechanisms, offers an overview of the workflow for genome-wide CRISPR-based regulation for screens, and highlights its superior suitability for HTS across biomedical and industrial applications. Finally, we discuss the limitations of current CRISPRi/a HTS screening methods and envision future directions in CRISPR-mediated HTS research, considering its potential for broader application across diverse fields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3860c88a16ddc6c01ca95906456a41db1aa0ff54" target='_blank'>
              CRISPR-Based Regulation for High-Throughput Screening.
              </a>
            </td>
          <td>
            Lingling Jiao, Qi Zhou, Dongchang Sun
          </td>
          <td>2025-05-22</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e17631


 Background:
 Poly (ADP-ribose) polymerase inhibitors (PARPi) has been studied to show positive efficacy in endometrial cancer (EC). However, rational candidates for precision treatment with PARPi need to be further explored. Recently, it has been reported that tumors with homologous recombination deficiency (HRD) are sensitive to PARPi. Here, we aimed to assess the relationship between HRD and molecular subgroups in EC.
 Methods:
 HRD statues and gene characteristics of 372 EC patients were retrospectively collected from August, 2022 to August, 2024. HRD score was calculated as the sum of loss of heterozygosity (LOH), telomeric allelic imbalance (TAI), and large-scale state transitions (LST) scores based on the next-generation sequencing (NGS) data.
 Results:
 Of the EC that underwent NGS testing, there were 5.38% (20/372) patients with
 BRCA1/2
 mutations, and the top two
 BRCA1/2
 mutated subgroups were microsatellite instability-high (MSI-H) (2.96%) and copy number low (CN-L) (1.88%) subgroups. However, there were 10.75% (40/372) patients with positive HRD, which is largely restricted to copy number high (CN-H) (4.57%), CN-L (2.96%) and MSI-H (2.96%) subgroup. Interestingly, based on HRD score with the cutoff of 30, there were mostly CN-H and CN-L subgroups and almost no
 POLE
 mutation and MSI-H subgroups in non
 BRCA1/2
 mutation and positive HRD patients.
 Conclusions:
 BRCA1/2
 mutations and positive HRD were seen in 5.38% and 10.75% EC patients, respectively. HRD score testing, based on LOH, TAI, and LST, may help select patients benefiting from PARPi with particular reference to CN-H and CN-L subgroups.


 Prevalence of HRD statues in different molecular subgroups of EC.





 HRD
 Non-HRD




 Molecular Subgroups
 All

 BRCA1/2
 mutation

 HRD score≥30




 POLE
 mut

 1(0.27%)
 1(0.27%)
 0(0%)
 8(2.15%)


 MSI-H
 11(2.96%)
 11(2.96%)
 0(0%)
 64(17.20%)


 CN-H
 17(4.57%)
 1(0.27%)
 16(4.30%)
 64(17.20%)


 CN-L
 11(2.96%)
 7(1.88%)
 4(1.08%)
 196(52.69%)






 HRD: Homologous recombination deficiency; EC: endometrial carcinomas;
 POLE
 mut:
 POLE
 mutations; MSI-H: microsatellite instability-high; CN-L: copy number low; CN-H: copy number high.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66ff21a16e023a00431cd6937437785071b41ecc" target='_blank'>
              Prevalence of homologous recombination deficiency in different molecular subgroups of endometrial carcinomas.
              </a>
            </td>
          <td>
            Yang Rao, Wenwen Zhang, Na Zhang, Han Han, Yong Xiao, Ping Liu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>165</td>
        </tr>

        <tr id="
 Somatic mutations in the small GTPase K-Ras are responsible for approximately 30% of human cancers and are generally associated with poor response to standard therapies. The mutations in K-Ras were identified in the early 1980’s but it took almost four decades for the first drugs targeting the protein to be approved. The long delay was a result of the apparent lack of a drug binding pocket on K-Ras despite detailed structural and biochemical characterization of the protein, leading to it being referred to as “undruggable.” This view changed when a chemical strategy based on targeting the K-Ras (G12C) mutant common in lung cancer was successful in identifying a druggable pocket in 2013. This finding led eight years later to the approval of two new medicines for patients with K-Ras (G12C) lung cancer, sotorasib and adagrasib. One current challenge in K-Ras drug development is development of agents to inhibit the more frequent K-Ras (G12D) and K-Ras (G12V) versions found in colon and pancreatic cancer. The other challenge is drug resistance which is faced by all targeted cancer therapies including the new K-Ras (G12C) directed drugs. In this lecture I will provide a perspective on challenges and opportunities of newly developed K-Ras targeting drugs.


 Kevan M. Shokat. Overcoming the undruggable nature of the most common human oncogene, K-Ras [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr PL02-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28a737e7facd64110f2da3e1fda75d42f81d817c" target='_blank'>
              Abstract PL02-01: Overcoming the undruggable nature of the most common human oncogene, K-Ras
              </a>
            </td>
          <td>
            K. Shokat
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>133</td>
        </tr>

        <tr id="The advent of genome-editing technologies, particularly the RNA-guided the clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated system (Cas) 9, which originates from prokaryotic CRISPR/Cas adaptive immune mechanisms, has revolutionized molecular biology. Renowned for its simplicity, cost-effectiveness, and capacity for multiplexed gene editing, CRISPR/Cas9 has emerged as the most versatile and widely adopted genome-editing platform. Its applications span fundamental research, biotechnology, medicine, and therapeutics. This review highlights recent advancements in CRISPR-based technologies, focusing on CRISPR/Cas9, CRISPR/Cas12a, and CRISPR/Cas12f. It emphasizes precision editing methods like base editing and prime editing, which enable targeted nucleotide changes without double-strand breaks. The specificity of these tools, including on-target accuracy and off-target risks, is critically evaluated. Additionally, recent preclinical and clinical efforts to treat diseases such as cancer and sickle cell disease using CRISPR are summarized. Finally, the challenges and future directions of CRISPR-mediated gene therapy are discussed, emphasizing its potential to integrate with other molecular approaches to address unmet medical needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d5091bbb50a58b99c0d0d29fe316c867de14c3f" target='_blank'>
              Progress, Applications and Prospects of CRISPR-Based Genome Editing Technology in Gene Therapy for Cancer and Sickle Cell Disease.
              </a>
            </td>
          <td>
            Sha-Sha Zang, Ruirui Zhang, Jia-Run Zhang, Xi Zhang, Jun Li
          </td>
          <td>2025-05-12</td>
          <td>Human gene therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 In the United States, breast cancer is one of the leading causes of cancer death in women. Triple-negative breast cancer (TNBC) is the most aggressive of all molecular subtypes of breast cancer. It is highly heterogeneous and difficult to target and treat due to its lack of targetable receptors. Our aim is to identify new molecular targets that could aid the treatment of TNBC cases and reduce cytotoxicity of the current therapeutic regimen. The TP53 gene, a known tumor suppressor is the most frequently mutated gene in triple-negative breast cancer occurring in >80 % of TNBC cases. Statistical pathway analysis from RNA-Seq and ChIP-Seq done on the MCF10A mammary epithelial cell lines expressing the 10 most clinically prevalent p53 mutants identified the Hippo pathway to be dysregulated when associated with aggressive cellular phenotypes. Upon suppression of the upstream Hippo tumor suppressor pathway, the activation of a series of genes promoting cancer hallmark phenotypes ensues. A cell-based TEAD reporter assay showed transcriptional activity of TEAD to be higher in MCF10A cells expressing more aggressive p53 mutants compared to the cell lines with less aggressive mutants. Knockdown of the TAZ protein decreased the invasiveness of the invasive p53 mutant expressing cell lines. We hypothesized that, in addition to neo-morphic activities of different p53 mutants, the heterogeneity in clinical phenotypes observed in TNBC is due to functional interactions of the driver mutation in p53 with other functionally important co-existing mutations and dysregulated pathways, called ‘co-drivers’. To identify potential functionally important ‘co-driver’ mutations that cause cancer progression and heterogeneity in clinical phenotypes, a genome-wide screen was performed using lentiviral CRISPR screen library on the invasive MCF10A-p53-R273C-MycOE cell line. In vivo experiments showed that injection of the library-transduced cells yielded tumor in mice. PCR and whole-exome sequencing of the tumor showed frameshift mutations in the enriched EVC2 (EvC Ciliary Complex Subunit 2), NR1D1 (Nuclear Receptor 1D1), and ARAF (a Ras-activating RAF family kinase) genes. Confirming the hits from screening, knockout of these genes in R273C-MycOE cells also showed increased cell invasion. In parallel, in vitro screens carried out using CRISR knock-out library on the least aggressive MCF10A-p53-Y234C cells identified potential co-drivers. Knockout of GCNT1 and FGD6 genes in MCF10A-p53-Y234C cells increased cell invasion and induced dephosphorylation and activation of TAZ. TEAD activity was also found to be higher in these knock-out cells compared to the Y234C cells. Knockdown of the TAZ protein showed decreased TEAD activity in the FGD6 and GCNT1 knock-out cells and substantially decreased cell invasion. We have carried out functional characterization of the TP53 mutations in TNBC and confirmed the cellular phenotypes for specific p53 mutants obtained from the in vitro and in vivo screens. We are validating these identified ‘co-driver’ genes and targets to discover the molecular mechanisms affecting phenotypic heterogeneity of the p53 mutants and investigating the mechanisms through which Hippo pathway drives the progression of TNBC, as well as assessing the pathway as a target for potential therapy for TNBC with specific TP53 mutations.
 Citation Format: Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura Gonzalez-Malerva, Duatin Grief, Lydia Sakala, Jin Park, Joshua LaBaer. Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f9063c7f21ea86d31d2953e319cbe0d8301969" target='_blank'>
              Abstract P2-11-20: Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches
              </a>
            </td>
          <td>
            Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura González‐Malerva, Duatin Grief, Lydia Sakala, Jin G Park, Joshua LaBaer
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Immune checkpoint blockade (ICB) therapy has shown promise in treating advanced colorectal cancer, particularly in patients with microsatellite instability-high (MSI-H) tumors. However, only a subset of these patients responds favorably, highlighting the need for strategies to improve immunotherapy efficacy. Methods To identify potential regulators of immunotherapy response, we conducted a comprehensive analysis of colorectal cancer datasets from The Cancer Genome Atlas (TCGA). We performed multi-omics analyses and functional assays in both human and murine colorectal cancer cell lines. Additionally, we evaluated tumor growth and immune cell infiltration using syngeneic mouse models. Results Our analysis revealed that RNA binding motif protein 15 (RBM15) is highly expressed in colorectal cancer and correlates with poor patient prognosis. Functional studies demonstrated that RBM15 loss led to increased expression of fumarate hydratase (FH). This led to decreased levels of fumarate, a metabolite known to suppress anti-tumor immune responses. In vivo, RBM15 depletion significantly delayed tumor progression and enhanced CD8⁺ T cell infiltration and activation in the tumor microenvironment. Discussion These findings identify RBM15 as a negative regulator of anti-tumor immunity in colorectal cancer. Targeting RBM15 may represent a novel strategy to boost immune responsiveness and improve outcomes for patients undergoing immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6fd5bb65d4751738b1ded96c2f1862a85aeb651" target='_blank'>
              RBM15-mediated metabolic reprogramming boosts immune response in colorectal cancer
              </a>
            </td>
          <td>
            Chen Wang, Mengyan Chen, Panyu Chen, Jinlu Han, Hong Hu, Jiong Chen, Qiong Wu, De Zhao, Tongshuai Wang, Jingyi Zhou, Qi Li, Runkai Zhou, Yugang Wen, Jing Yang, Min Shi, Yugang Wang
          </td>
          <td>2025-04-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE Despite the significance of immune checkpoint inhibitors (ICIs) in solid tumor treatment, identifying ICI-sensitive populations remains a challenge. Mutations in DNA damage response (DDR) pathway genes are increasingly linked to favorable ICI responses; however, selection criteria for specific DDR pathway biomarkers remain elusive. METHODS Data of patients with cancer who received ICIs and non-ICI therapy were extracted from public databases. Synthetic lethal (SL) gene combinations were selected from the SynLethDB database. Kaplan-Meier analysis was conducted to investigate the correlation between SL gene co-mutations in DDR pathways and prognosis. Immune infiltration analysis was performed using the Timer online tool based on DNA and RNA sequences obtained from The Cancer Genome Atlas database. RESULTS Patients with synthetic lethal co-mutations in DDR pathways exhibited significantly extended overall survival (HRhazard ratio [HR], 0.76 [95% CI, 0.62 to 0.92]; P = .0061), progression-free survival (HR, 0.33 [95% CI, 0.18 to 0.60]; P = .0003), and higher objective response rates (66.7% v 21.0%; P = .0009) after receiving ICIs. Conversely, DDR SL co-mut+ patients receiving non-ICI treatment presented with markedly shortened survival (HR, 1.35 [95% CI, 1.16 to 1.56]; P < .0001). High-frequency DDR SL co-mutation pairs were enriched in the checkpoint factor pathway. TP53-ATM emerged as a common combination, with TP53-ATM co-mut+ patients receiving more survival benefits from ICI (HR, 0.63 [95% CI, 0.41 to 0.99]; P = .045). Immune infiltration analysis demonstrated altered immune reactivity in TP53-ATM co-mut+ tumors, with higher levels of CD4+ T cells, plasma cells, macrophages, natural killer cells, and myeloid dendritic cells but lower levels of CD8+ T cells. CONCLUSION Our study identified SL co-mutations in the DDR pathway as promising biomarkers for ICI efficacy. Specifically, the TP53-ATM co-mutation status can be applied in identifying ICI-responsive patients. SL co-mutations in DDR pathways correlate with modification of the tumor immune microenvironment, shaping a favorable niche for ICIs to stimulate immune responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bc22e4c2353cc65d9354b43ba05083ae13037b8" target='_blank'>
              Synthetic Lethal Co-Mutations in DNA Damage Response Pathways Predict Response to Immunotherapy in Pan-Cancer
              </a>
            </td>
          <td>
            Lincheng Zhang, Yue Wang, Rong Qiao, Yao Zhang, Huiping Qiang, Huawei Du, Fengcai Wen, Miao Liu, Yan Zhou, Wei Nie, Hua Zhong
          </td>
          <td>2025-05-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Currently, there are no targeted therapies for triple-negative breast cancer (TNBC). 1 While radiation therapy plays a crucial role in breast cancer treatment, its effectiveness against TNBC is hindered by acquired resistance to radiation over time. This highlights the need for more sensitive and specific therapeutic approaches. 2 ALDH1A1 is associated with stemness and poor outcomes in TNBC and is a potential new target for treatment. ALDH1A1 contributes to chemoresistance in TNBC3, and we propose that it also promotes radiation resistance through both its antioxidant properties and regulation of cancer stemness. Previous studies have indicated that the radiation resistance observed in TNBC is linked to increased levels of ALDH1A14, suggesting it plays a protective role against radiation-induced cell death. In this study, we did a CRISPR/Cas9 knock out (KO) of ALDH1A1 in SUM 159 cells and derived single cell clones to screen for the effects of ALDH1A1 KO. Confirmation of ALDH1A1 KO was done at the genomic level by Sanger sequencing and at the protein level by western blotting analysis. The response of ALDH1A1 knockout clonal derived cells to radiation was analyzed using colony forming units (CFU), proliferation assays (Alamar blue), and Annexin V/PI staining. All the ALDH1A1 knockout clones analyzed showed enhanced radiation response with reduced proliferation, and colony formation units post-radiation, as compared to the parental cells. Furthermore, tumor initiation was measured in the parental cells and ALDH1A1 KO clones using the organoid culture system platform. Overall, ALDH1A1 KO clonal derived cells formed fewer tumoroids post-radiation, as measured by cell proliferation assays in a 3D culture system. Together these findings highlight the critical relationship between ALDH1A1 expression and radiation sensitivity, emphasizing the potential of ALDH1A1 as a target for improved treatment of TNBC.
 Citation Format: Grace Aijayi, Shirin R. Modarai, Aihui Ma, Lynn M. Opdenaker, Jennifer Sims Mourtada. CRISPR/Cas9 knockout of ALDH1A1 increases radiation response in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/addcc6f0bdb6fd3863e3c5a752c6092ba78d179c" target='_blank'>
              Abstract P3-02-25: CRISPR/Cas9 knockout of ALDH1A1 increases radiation response in triple-negative breast cancer
              </a>
            </td>
          <td>
            Grace Aijayi, Shirin R. Modarai, Aihui Ma, Lynn M. Opdenaker, J. Mourtada
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686f795b4b66276294de9ae162a3558934271a82" target='_blank'>
              A CRISPR-CAS9 high throughput machine-learning platform for modulation of genes involved in Parkinson’s disease-associated PINK1-mitophagy in iPSC-derived dopaminergic neurons
              </a>
            </td>
          <td>
            M. Soutar, Benedetta Carbone, Petya Kindalova, Rahil Mehrizi, Fernanda Martins Lopes, Nathaniel Lam, Alice Rockliffe, Toni Braybrook, Matthew Taylor, Cuong Nguyen, Fiona Ducotterd, Alastair D. Reith, Lisa Mohamet, Helene Plun-Favreau
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7d798ffee04bc73cc38a17396f4a99f40e9d7c9" target='_blank'>
              Dysfunctional mismatch repair in patients with early triple-negative breast cancer.
              </a>
            </td>
          <td>
            María Muñiz-Castrillo, Noel Blaya Boluda, E. García-Torralba, P. Jiménez-Fonseca, Carmen González Del Rey, Milagros Balbín, Ginés Luengo-Gil, F. Ayala de la Peña, Emilio Esteban González, A. Carmona-Bayonas
          </td>
          <td>2025-05-04</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The biological mechanisms underlying the cooperation between germline genetic variants and somatic mutations during carcinogenesis are rarely elucidated. Here, characterizing isogenic prostate cancer cell lines, we dissected the interplay between a germline variant at the 7p14.3 locus (rs1376350, G>A) and early recurrent prostate cancer-specific mutation in the Speckle-Type POZ protein (SPOP) gene across human prostate adenocarcinomas. The transcriptomes of multiple edited models pointed to Gli3 and the Hedgehog signaling pathway in a genotype-specific manner, while SPOP mutation and AR stimulation promote Gli3 accumulation in the full-length (FL) transcriptionally active form. This, in turn, triggers the cell-autonomous production of steroids that prostate cancer relies on, in line with the enhanced responsiveness of SPOP-mutated prostate cancer to androgen deprivation therapy. These data demonstrate that germline variants dictate men's prostate cancer somatic evolution and suggest opportunities to jointly model germline-somatic tandems to help untangle the complexity of human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6ef3ec565535edd5c8feb3d331621aab63b409b" target='_blank'>
              Germline-somatic liaison dictates cancer subtype via de novo steroid biosynthesis.
              </a>
            </td>
          <td>
            P. Gasperini, A. Alaimo, Blerta Stringa, Yoon-Mi Chung, Yari Ciani, F. Lorenzin, Giulia Fracassi, Y. Zekri, F. Orlando, Orsetta Quaini, Sebastian Gregoricchio, G. Petris, Antonio Casini, Christopher E Barbieri, W. Zwart, A. Cereseto, Nima Sharifi, Andrea Lunardi, F. Demichelis
          </td>
          <td>2025-06-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, A. Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The ubiquitin–proteasome system enables post-transcriptional protein modification and is a major pathway for the degradation of most of them in eukaryotic cells. Among these, the ubiquitin-specific protease (USP) family is the most extensively studied. As an important member of the USP family, ubiquitin-specific protease 39 (USP39) plays an essential role in RNA splicing and protein regulation. This review comprehensively summarizes the structural characteristics and molecular functions of USP39, emphasizing its pivotal role in the regulation of cellular processes. Dysregulation of USP39 is closely associated with the progression of various cancers through mechanisms such as immune evasion, modulation of oncogenic signaling pathways, and altered RNA splicing. These processes impact key aspects of cancer biology, including proliferation, metastasis, and therapy resistance, underscoring the broad implications of USP39 in tumor progression. Recent studies position USP39 as a promising target for cancer treatment. Future research should explore its upstream regulatory networks, develop small-molecule inhibitors, and evaluate its potential for precision oncology. This review integrates the latest insight into USP39, providing a foundation for its clinical application in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d101a1675c0ce8425b854017f981230f8a30dc4b" target='_blank'>
              Exploring the cancerous nexus: the pivotal and diverse roles of USP39 in cancer development
              </a>
            </td>
          <td>
            Yujing Chen, Jingyi Zhang, Jinfeng Yang, Jiawei Zhao, Xiaotong Guo, Juzheng Zhang, Jinfeng Gan, Weijia Zhao, Siqi Chen, Xinwen Zhang, Yi Lin, Jiamin Jin
          </td>
          <td>2025-05-10</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42c09d4105986a237e810f7afea7abc78ee847e1" target='_blank'>
              RNase H1 levels dramatically affect mitochondrial genome maintenance with little impact on nuclear R-loops in murine B cells
              </a>
            </td>
          <td>
            K. Sakhuja, Stella R. Hartono, L. Sanz, Eriko Ijiri, Caitlin Darling, Louis Dye, James Iben, H. Chon, S. Cerritelli, Robert J. Crouch, F. Chédin
          </td>
          <td>2025-04-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Somatic mitochondrial DNA (mtDNA) mutations are commonly observed in tumors, yet their functional role in pediatric cancers remains largely unexplored. The unique biology of mtDNA—including its heteroplasmic nature, where mutant and wild-type mtDNA coexist—poses challenges in understanding its contribution to disease progression. To address this, we utilized an integrative approach combining bulk whole-genome sequencing, single-cell analysis, and experimental modeling. Analysis of 637 matched tumor and germline samples from the Pediatric Cancer Genome Project revealed an enrichment of functional-impact mtDNA variants in specific pediatric leukemia subtypes. Single-cell sequencing of diagnostic leukemia samples demonstrated that somatic mtDNA mutations can emerge early in leukemogenesis and undergo positive selection, achieving intermediate heteroplasmy—a "sweet spot" balancing mitochondrial dysfunction and cellular fitness. Systems biology analyses linked these heteroplasmic mutations to functional heterogeneity and metabolic adaptability within leukemic cells. To directly investigate the role of mtDNA mutations in leukemogenesis, we studied hematopoietic progenitor cells (HPCs) with varying mtDNA mutation burdens using wild-type, heterozygous, and homozygous mtDNA mutator mice (Polg D257A). Heterozygous Polg HPCs exhibited a higher incidence of spontaneous leukemogenesis and supported oncogene-driven transformation, whereas homozygous Polg HPCs showed impaired leukemogenesis due to disrupted glucose metabolism and pyruvate dehydrogenase inhibition. These findings highlight the role of intermediate heteroplasmy in balancing mitochondrial dysfunction and cellular fitness, enabling leukemic cells to maintain metabolic adaptability. Targeting this balance may uncover novel therapeutic strategies to disrupt the metabolic plasticity driving pediatric leukemia progression.


 Mondira Kundu. Exploring the significance of somatic mtDNA mutations in the context of acute leukemia [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY27-02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f70cae7350af93e8189b0a48977a508877000d4" target='_blank'>
              Abstract SY27-02: Exploring the significance of somatic mtDNA mutations in the context of acute leukemia
              </a>
            </td>
          <td>
            M. Kundu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancers with BRCA1 or BRCA2 mutations are defective in repair of DNA double-strand breaks by homologous recombination, resulting in compensatory error-prone repair that causes genomic instability. Poly(ADP-ribose) polymerase inhibitors (PARPi) are FDA-approved to treat homologous recombination-defective cancers, inducing therapy responses by synthetic lethality. PARPis increase micronuclei formation and cytosolic double-stranded DNA accumulation, activating stimulator of interferon genes (STING). Activation of STING can mediate anticancer innate immune responses by increasing T-cell infiltration into the tumor microenvironment. However, PARPi responses are not durable, and therapy resistance ensues with limited therapeutic options available for these patients. Using PARPi-sensitive and -resistant patient-derived xenografts and mouse-derived allografts, Pedretti and colleagues show in this issue of Cancer Research that the PARPi olaparib in combination with the next-generation STING agonist diABZI can overcome PARPi resistance in a manner dependent on NK cell function in the tumor microenvironment. Their study highlights a novel component of the STING-dependent innate immune response repertoire required for fighting PARPi-resistant cancer. Potent and specific next-generation STING agonists are being tested in the clinic in solid and liquid tumors, indicating a resurgence of these drugs after a long period of modest clinical activity, with a special focus on combination therapy strategies to fight therapy-resistant cancer. See related article by Pedretti et al., p. 1888.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759717dee22ef6b61851d2be7d6d7403771e9974" target='_blank'>
              Combining STING Agonists with PARP Inhibitors Mounts an NK-Dependent Defense against Therapy-Resistant Breast Cancer.
              </a>
            </td>
          <td>
            Zahra Gohari, Lora Stojanovic, F. Rassool
          </td>
          <td>2025-05-15</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52c3844fc0c5bb373680e6bacbba142f0e508fa" target='_blank'>
              Forecasting oncogene amplification and tumour suppressor deletion
              </a>
            </td>
          <td>
            Bárbara Hernando, Ángel Fernández-Sanromán, Alice Cádiz, Patricia G. Santamaria, David Gómez-Sánchez, Maria Escobar-Rey, Blas Chaves-Urbano, Joseph Thompson, Marina Torres, G. R. de Garibay, Vera Adradas, Eva Álvarez, Maxime Tarabichi, T. Lesluyes, Juan Manuel Coya, Jon Zugazagoitia, Luis Paz-Ares, Geoff Macintyre
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="


 Werner syndrome helicase (WRN) is a RecQ helicase family member that plays an important role in genome stability, primarily through its involvement in DNA repair processes such as nonhomologous end-joining and homologous recombination. Upregulation of WRN in mismatch repair (MMR)-deficient or microsatellite instability-high (MSI-H) tumor cells facilitates DNA damage repair and, thus, cell survival. In these cells, WRN inhibition induces double-strand DNA breaks, leading to cell death. Therefore, WRN has emerged as a novel, promising synthetic lethality target in MMR-deficient/MSI-H solid tumors, particularly endometrial and colorectal cancers, which exhibit MMR-deficiency/MSI-H in approximately 20%-30% and 5%-15% of cases, respectively. GSK4418959, a WRNi, is a distinct, oral small molecule that uniquely binds to an allosteric site in the WRN helicase domain, exhibiting synthetic lethality in MMR-deficient/MSI-H tumor cells. It showed potent antitumor activity in in vitro and in vivo preclinical models by selectively inducing DNA damage and, consequently, death of MSI-H cancer cells.



 The SYLVER first-in-human, open-label, multicenter, phase 1/2 trial (NCT06710847) opened on December 9, 2024 and aims to assess safety, tolerability, pharmacokinetics, pharmacodynamics, and preliminary efficacy of GSK4418959, as monotherapy or in combination with a programmed cell death protein-1 (PD-1) inhibitor. The trial comprises WRNi monotherapy dose escalation (part 1), WRNi monotherapy dose expansion (part 2), and dose escalation of WRNi + PD-1 inhibitor (part 3). Key eligibility criteria include age ≥18 years, an advanced MMR-deficient/MSI-H solid tumor, an ECOG PS score of 0-1, and having exhausted standard of care (parts 1 and 3) or received 1-3 lines of systemic therapy, including ≥1 immune checkpoint inhibitor therapy (part 2). Patients who have not recovered from treatment-associated adverse events, have uncontrolled hypertension, or have received prior treatment with a WRNi are ineligible. Primary endpoints are safety during the dose-limiting toxicity observation period (ie, first 21 days; parts 1 and 3) and objective response rate (part 2). Secondary endpoints include overall safety (parts 1-3), pharmacokinetics (parts 1-3), progression-free survival (part 2), and duration of response (part 2). Tumor responses will be assessed per RECIST 1.1. Planned enrollment is approximately 133 patients in the US, Australia, Japan, South Korea, Spain, Belgium and the Netherlands, with potential to expand to additional countries for later stages of the study.



 This study (NCT06710847) is sponsored by GSK.



 NCT06710847
 Third-party medical writing support: Writing and editorial support, funded and coordinated by GSK, was provided by Tafara T.R. Kunota, PhD, and Mary C. Wiggin of Ashfield MedComms, an Inizio company.



 Vivek Samnotra, Natalia Ramos-Hernandez, Thomas Faitg, Meenakshi Srinivasan, Tania Roy, Andrew Liu, Tomasz Zaremba, Clare Holford, Rachel Smith, Jason T. Henry, Jeanne Tie, Noboru Yamamoto, Gul Serbest. First-in-human, phase 1/2 study of GSK4418959, an oral Werner DNA helicase inhibitor, alone or combined with other anticancer agents in adults with advanced mismatch repair-deficient or microsatellite instability-high solid tumors (SYLVER) [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr CT194.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2d8faa11ff0e47828029568767559cbff4cf0c6" target='_blank'>
              Abstract CT194: First-in-human, phase 1/2 study of GSK4418959, an oral Werner DNA helicase inhibitor, alone or combined with other anticancer agents in adults with advanced mismatch repair-deficient or microsatellite instability-high solid tumors (SYLVER)
              </a>
            </td>
          <td>
            Vivek Samnotra, Natalia Ramos-Hernandez, Thomas Faitg, Meenakshi Srinivasan, Tania Roy, Andrew Liu, Tomasz Zaremba, Clare Holford, Rachel Smith, Jason T Henry, Jeanne Tie, N. Yamamoto, Gul Y. Serbest
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 The impairment of DNA mismatch repair (MMR) function occurs in various cancers, including colorectal (15%), gastric (22%), endometrial (20-30%) and ovarian (10-15%) cancers. MMR-deficient (dMMR) cancers are characterized by a high mutational load and frequent insertion and/or deletion events in repetitive DNA sequences, a condition known as microsatellite instability-high (MSI-H). The Werner syndrome protein (WRN) is essential for the survival of cancer cells exhibiting microsatellite instability (MSI) and represents a synthetic lethal target for MSI-H/dMMR cancers. We have identified YF087, a highly selective, reversible and potent WRN inhibitor that has demonstrated robust activity in both in vitro anti-proliferation assays and in vivo cancer xenograft models. YF087 exhibited an IC50 of 8.1 nM in the WRN helicase enzymatic assay and an IC50 of 42 nM in the anti-proliferation assay in SW48 (MSI-H) colorectal cancer cell. YF087 exhibits excellent selectivity against another helicase BLM at the enzymatic level and is highly selective for MSI-H cells compared to microsatellite stable (MSS) cell in the anti-proliferation assay. YF087 is more effective than both WRN reversible inhibitor HRO761 and irreversible inhibitor RG6457 in the SW48 cellular anti-proliferation assay in the presence of 50% human serum. In the HCT116 colorectal cancer xenograft model, 5 mg/kg, 15 mg/kg and 45 mg/kg YF087 oral and daily treatment for 22 days induced regression rates of 32%, 43% and 57%, respectively. Tumors continue to regress after the treatment ends and remained tumor-free status for more than two months after dose termination at the 45 mg/kg dose. YF087 also exhibits superior in vivo anti-tumor efficacy in other MSI-H/dMMR cancer xenograft models upon oral and daily treatment and maintained longer tumor regression after termination of treatment vs HRO761 and RG6457. In patient derived MSI-H gastrointestinal cancer models resistant to anti-PD-1 therapy, treatment with YF087 resulted in tumor inhibition or regression. YF087 also possesses favorable preclinical ADME profiles suitable for clinical development. Based on these findings, YF087 is currently in IND-enabling studies to support phase 1 clinical trial in MSI-H/dMMR cancer patients.


 Yaolin Wang, Rongfeng Liu, Yanxin Yu, Hong Yang, Jifang Weng, Yan Weng, Hong Mei, David Dai. YF087 is a potent and selective inhibitor of WRN, specifically targeting MSI-H cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB187.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fb1304f7cbde247e546624ce764f580b19d5f83" target='_blank'>
              Abstract LB187: YF087 is a potent and selective inhibitor of WRN, specifically targeting MSI-H cancer cells
              </a>
            </td>
          <td>
            Yaolin Wang, Rongfeng Liu, Yanxin Yu, Hong Yang, Jifang Weng, Yan Weng, Hong Mei, David Dai
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e13538


 Background:
 Understanding the molecular profile of cancer-associated genes is crucial in oncology, as it informs targeted therapies and improves patient outcomes. Tumor suppressor genes inhibit the activation of oncogenic signals, while oncogenes promote them. Ras proteins, encoded by KRAS, NRAS, and HRAS genes, are involved in cell growth, differentiation, and survival pathways. This study aims to investigate the frequency of KRAS, NRAS, and HRAS mutations in solid tumors treated at a single academic institution between January 2017 and May 2024. By analyzing these mutations, we seek to enhance our understanding of their distribution across different cancer types in Venezuela, providing valuable insights for personalized cancer treatment strategies.
 Methods:
 We conducted a retrospective, cross-sectional, analytical cohort study. The study included 175 patients, with 179 molecular genetic sequencing studies and 20 RT-PCR analyses. We examined mutations in codons 12, 13, 59, 61, 117, and 146 of the KRAS, NRAS, and HRAS genes.
 Results:
 We identified 43 mutations in the RAS gene family: 37 in KRAS, 5 in NRAS, and 1 in HRAS. Lung cancer accounted for 14 of those mutations, with 13 in KRAS and 1 in HRAS. Colorectal cancer showed 19 mutations, with 16 in KRAS and 3 in NRAS. Other KRAS mutations (13.9%) were found in breast, endometrial, pancreatic cancers, and thymoma. We also recorded 2 NRAS mutations in melanoma and 1 in pancreatic cancer.
 Conclusions:
 This study represents the largest collection of KRAS, NRAS, and HRAS mutations reported in Venezuela.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d60941cc22a3a39b9e5850c1cc1b772e65a8cda" target='_blank'>
              Frequency of RAS family gene mutations in metastatic solid tumors in Venezuela: Experience at a large academic center.
              </a>
            </td>
          <td>
            Guillermo Borga, Oscar A. Sucre Astorga, Santiago Sucre, C. Sucre
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db3f634648b2c3c7a12699cca388efeca2e26cd" target='_blank'>
              Comprehensive Measurement of Inter-Individual Variation in DNA Repair Capacity in Healthy Individuals
              </a>
            </td>
          <td>
            T. Zhai, P. Mazzucato, C. Ricciardi, D. C. Christiani, , L. D. Samson, I. A. Chaim, Z. D. Nagel
          </td>
          <td>2025-06-16</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Constitutional mismatch repair deficiency (CMMRD), caused by bi-allelic germline variants in mismatch repair (MMR) genes, is associated with high cancer incidence early in life. A better understanding of mutational processes driving sequential CMMRD tumors can advance optimal treatment. Here, we describe a genomic characterization on a representative collection of CMMRD-associated tumors consisting of 41 tumors from 17 individuals. Mutational patterns in these tumors appear to be influenced by multiple factors, including the affected MMR gene and tumor type. Somatic polymerase proofreading mutations, commonly present in brain tumors, are also found in a T-cell lymphoblastic lymphoma displaying associated mutational patterns. We show prominent mutational patterns in two second primary hematological malignancies after temozolomide treatment. Furthermore, an indel signature, characterized by one-base pair cytosine insertions in cytosine homopolymers, is found in 54% of tumors. In conclusion, analysis of sequential CMMRD tumors reveals diverse mutational patterns influenced by the affected MMR gene, tumor type and treatment history.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c61f9eb1036d6b9dbdcb4d99fcc534a6fcd3f8" target='_blank'>
              Unraveling mutagenic processes influencing the tumor mutational patterns of individuals with constitutional mismatch repair deficiency
              </a>
            </td>
          <td>
            Dilys D Weijers, Snežana Hinić, E. Kroeze, M. Gorris, G. Schreibelt, Sjors Middelkamp, Arjen R Mensenkamp, R. Bladergroen, K. Verrijp, N. Hoogerbrugge, Pieter Wesseling, Rachel S van der Post, Jan L C Loeffen, C. Gidding, M. V. van Kouwen, I. J. D. de Vries, R. van Boxtel, R. D. de Voer, M. Jongmans, Roland P Kuiper
          </td>
          <td>2025-05-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) affects genomic stability and has potential as a biomarker for the effectiveness of poly (ADP-ribose) polymerase (PARP) inhibitors and immune checkpoint inhibitors. However, the clinical and molecular profile of HRD in non-small cell lung cancer (NSCLC), particularly in the Chinese population, remains poorly characterized. Based on the next-generation sequencing data of 158 Chinese NSCLC patients, we analyzed the HRD scores of mutations in homologous recombination repair (HRR) genes and dissected the correlation between HRD state and programmed death-ligand 1 (PD-L1) expression. Alterations in HRR genes were observed in 8.9% of the patients, with ATM and BRCA2 being the most commonly affected genes. HRD-high (HRD-H) status was significantly associated with advanced disease stage (≥III) and lung squamous cell carcinoma (LUSC). Transcriptomic analysis revealed distinct gene expression profiles between HRD-H and HRD-low (HRD-L) subgroups, with HRD-H tumors exhibiting predominantly downregulated genes. While EGFR mutations occurred at similar frequencies across HRD status, TP53 mutations were significantly enriched in HRD-H cases. HRD-H status correlated with higher PD-L1 positivity in NSCLC overall, but not within the lung adenocarcinoma (LUAD) subgroup in our cohort. The Cancer genome atlas analysis showed higher PD-L1 protein expression in HRD-H LUAD, but not in LUSC. Kyoto Encyclopedia of Genes and Genomes analysis identified enrichment of complement and coagulation cascades, ABC transporters, and bile secretion pathways in HRD-H tumors, suggesting links to immune evasion and drug resistance. This study elucidates the genomic landscape of HRD in Chinese NSCLC patients and provides insights into its potential clinical utility for therapeutic targeting. Our findings suggest that integrated HRD scoring may guide the application of PARP inhibitors and immunotherapy in specific NSCLC patient subgroups. Further prospective clinical studies are needed to validate the predictive value of HRD scoring in NSCLC treatment and to optimize patient selection strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c403957972723c45a13a1636514a41ce8c5b8d3" target='_blank'>
              Genomic alterations of homologous recombination deficiency in Chinese NSCLC patients
              </a>
            </td>
          <td>
            Shuang Xiang, Changqiong Shen, Chun Huang, Ya-ting Yang, Jing Guo, Yi Liu, Mingzhu Yin, Song Duan
          </td>
          <td>2025-06-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e275fe9c31757491fc33c855fc96ec1cae73962" target='_blank'>
              Offspring effect of maternal preconception exposure to metabolic disruptors
              </a>
            </td>
          <td>
            Carlos Díaz-Castillo, Stephanie R. Aguiar, Raquel Chamorro-Garcia
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human exonuclease 1 (EXO1), a member of the structure-specific nuclease family, plays a critical role in maintaining genome stability by processing DNA double-strand breaks (DSBs), nicks, and replication intermediates during DNA replication and repair. As its exonuclease activity is essential for homologous recombination (HR) and replication fork processing, EXO1 has emerged as a compelling therapeutic target, especially in cancers marked by heightened DNA damage and replication stress. Through high-throughput screening of 45,000 compounds, we identified seven distinct chemical scaffolds that demonstrated effective and selective inhibition of EXO1. Representative compounds from two of the most potent scaffolds, C200 and F684, underwent a comprehensive docking analysis and subsequent site-directed mutagenesis studies to evaluate their binding mechanisms. Biochemical assays further validated their potent and selective inhibition of the EXO1 nuclease activity. Tumor cell profiling experiments revealed that these inhibitors exploit synthetic lethality in BRCA1-deficient cells, emphasizing their specificity and therapeutic potential for targeting genetically HR-deficient (HRD) cancers driven by deleterious mutations in HR genes like BRCA1/2. Mechanistically, EXO1 inhibition suppressed DNA end resection, stimulated the accumulation of DNA double-strand breaks, and triggered S-phase PARylation, effectively disrupting DNA repair pathways that are essential for cancer cell survival. These findings establish EXO1 inhibitors as promising candidates for the treatment of HRD cancers and lay the groundwork for the further optimization and development of these compounds as targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bbdf0d10f263cc553f41d2d920cc6308756913" target='_blank'>
              Discovery and Characterization of Small Molecule Inhibitors Targeting Exonuclease 1 for Homologous Recombination-Deficient Cancer Therapy.
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Chen Wang, Lingzi Ma, Megan Luo, J. Jossart, John J. Perry, David Kwon, Zhe Wang, Xinyu Pei, Changxian Shen, Yingying Wang, Mian Zhou, Holly Yin, David Horne, A. Nussenzweig, Li Zheng, Binghui Shen
          </td>
          <td>2025-05-16</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 The fatal Diffuse Midline Gliomas (DMG) are characterized by an undruggable H3K27M mutation in H3.1 or H3.3. K27M impairs normal development by stalling of differentiation. As replication timing influences gene expression, cell fate, and cellular response to therapeutics, we undertook a multi-omics approach (Repli-seq, cell cycle RNA-seq, single cell RNA-seq) in DMG cells (H3.1K27M and H3.3K27M subgroups) and tumors in comparison to normal brain to gain an appreciation for targetable pathways in DMG. DMG cells presented differential replication timing in each subgroup, which, in turn, correlated with significant differential gene expression including the cholesterol biosynthesis pathway. Consistent with these findings, DMG tumors presented high replication stress and cholesterol biosynthesis pathway signatures. Moreover, DMG cells were specifically vulnerable to an inhibitor of the cholesterol pathway and to a replication stress therapy, singly and in combination. In conclusion, this combinatorial genomics approach revealed insights into therapeutic opportunities for this incurable disease.


 Anastasia E. Hains, Joana G. Marques, Kashish Chetal, Tsunetoshi Nakatani, Andreas Ettinger, Carlos A. O Biagi, Adriana Gonzalez-Sandoval, Renjitha Pillai, Alberto Gatto, Maria-Elena Torres-Padilla, Ruslan I. Sadreyev, Mariella G. Filbin, Capucine Van Rechem. Diffuse midline gliomas: From cell cycle to therapeutic opportunities [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY23-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4a376ae55f874e4964cbc6c1df2c78ad21d8c3" target='_blank'>
              Abstract SY23-01: Diffuse midline gliomas: From cell cycle to therapeutic opportunities
              </a>
            </td>
          <td>
            Anastasia E. Hains, Joana G. Marques, Kashish Chetal, Tsunetoshi Nakatani, Andreas Ettinger, Carlos A. O Biagi, Adriana Gonzalez-Sandoval, Renjitha Pillai, Alberto Gatto, M. Torres-Padilla, Ruslan I. Sadreyev, Mariella G. Filbin, Capucine Van Rechem
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Genome maintenance is of the utmost importance in stem cells, as mutations can be propagated and cause defects in derivative tissues. Many stem cell types display low mutation rates, with embryonic stem cells (ESCs) being a notable example. The bases for this property are unclear but may be achieved by optimization of various processes including high-fidelity DNA repair, cell cycle checkpoint controls, and hypersensitivity to genotoxic insults that trigger cell death. Here, we investigate the mechanisms underlying the unique responses of mouse ESCs (mESCs) to replication stress (RS) using an array of small molecule inhibitors and genotoxins. We find that whereas mESCs survive under acute RS in an ATR- and CHK1-dependent manner similar to somatic cells, they lack a strong G2/M checkpoint and fail to repair DNA crosslinks in the absence of ATR signaling. Despite the lack of a strong G2/M checkpoint, mESCs maintain a spindle assembly checkpoint (SAC). We posit that mESCs preferentially repair DNA crosslinks in S phase via homology-directed mechanisms, and cells that fail to complete repair before mitosis undergo mitotic catastrophe and cell death. These findings shed light on mutation avoidance mechanisms in ESCs that may extend to other stem cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca127bc92b2f97069c67210cae368db947e036c" target='_blank'>
              Embryonic Stem Cell-Specific Responses to DNA Replication Stress
              </a>
            </td>
          <td>
            Ryan C. James, Jerry K. Wang, Siddhanth R. Bhatt, Sophie E. Shadid, Daryl J. Phuong, J. Schimenti
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Tumor suppressor genes are frequently targeted by mutations introducing premature termination codons (PTC) in the protein coding sequence, both in sporadic cancers and in the germline of patients with cancer predisposition syndromes. These mutations have a high pathogenic impact since they generate C‐terminal truncated proteins with altered stability and function. In addition, PTC mutations trigger transcript degradation by nonsense‐mediated mRNA decay. Suppression of PTC by translational readthrough restores protein biosynthesis and stabilizes the PTC‐targeted mRNA, making a suitable therapeutic approach the reconstitution of active full‐length tumor suppressor proteins by pharmacologically‐induced translational readthrough. Here, we review the recent advances in small molecule pharmacological induction of translational readthrough of disease‐associated PTC from tumor suppressor genes, and discuss the therapeutic potential of translational readthrough in specific groups of patients with hereditary syndromic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c0c11615ee99ac8075d00fb3fd428aa4a1973d7" target='_blank'>
              Therapeutic Potential of Translational Readthrough at Disease‐Associated Premature Termination Codons From Tumor Suppressor Genes
              </a>
            </td>
          <td>
            Leire Torices, C. Nunes-Xavier, R. Pulido
          </td>
          <td>2025-05-01</td>
          <td>Iubmb Life</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Blood cancers are one of the most common cancers worldwide. These diseases stem from defects in blood components having cytogenetic aberrations and genetic mutations. There have been vast improvements in terms of treatment options and survival outcomes. Nevertheless, due to the clonal nature and heterogeneity of the diseases, the number of cases reported exhibit a rising pattern due to chemoresistance and disease relapse thus posing a healthcare burden. Therefore, the need for more specific forms of targeted therapies is ever-present.


RECENT FINDINGS
CRISPR has emerged as a key player and is the epitome of gene editing technology in this post genomic era. In line with the current trend, numerous studies in blood cancer research have extensively utilized CRISPR-based applications to understand the functional genomics of hematologic malignancies and identify potential therapeutic targets for development of novel therapeutic applications.


SUMMARY
The importance of comprehending the utilities of state-of-the-art technologies such as CRISPR for studying hematologic malignancies has never been more apparent and timelier. Therefore, this review attempts to scrutinize the versatility of CRISPR applications which range from functional genomics to immunotherapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215dcf4713fb2244aa818902ca452469fccb308c" target='_blank'>
              Multifaceted roles of CRISPR technology in blood cancer research.
              </a>
            </td>
          <td>
            Maheswaran Solayappan, Adam Azlan, Kang Zi Khor, Mot Yee Yik, Aswani Jaishanker, Thevendran Ramesh, Mohamed Saleem, N. Yusoff, E. Moses
          </td>
          <td>2025-06-04</td>
          <td>Current opinion in hematology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 In solid malignancies, T cells functionality is closely intertwined with metabolic reprogramming within the tumor microenvironment (TME), which supports the increased energy request of cancer cells and suppresses the effector function and cytokines production of the immune infiltrate. In primary colorectal cancer (CRCs) and related liver metastases, we previously revealed extensive transcriptional remodeling across tumors, being metabolic pathways one of the major drivers of this variance. Among these, the adenosine (Ado) pathway has been shown to impact on the functionality of T cells infiltrating primary and metastatic colorectal cancer. Accordingly, upon T cell receptor (TCR) engagement, T cells release a substantial amount of adenosine triphosphate (ATP) in the TME, which is converted into Ado by CD39 and CD73. While ATP is necessary for T cells’ survival and activation, Ado dampens T cells anti-tumor activity. We previously demonstrated that the genetic disruption of CD39, which has the dual role of maintaining high levels of ATP while inhibiting Ado production, improves the activity of TCR-edited (TCRED) T cells in the adoptive T cell therapy (ACT) context. However, the mechanisms by which CD39 affects T cell behavior is not fully elucidated. Since CD39 activity depends on ATP availability, determined by the balance between catabolic and anabolic pathways, we investigated whether changes in T cells’ energetic metabolism could occur in TCRED T cells after CD39 disruption. By exploiting the Seahorse assay, we compared TCREDCD39COMPHER2 T cells and TCREDCD39KOHER2 T cells’ energetic profiles upon antigen-specific stimulation. We observed a reduced oxygen consumption rate and extracellular acidification rate in TCREDCD39KOHER2 T cells as well as a limited production of ATP, comparable to unstimulated cells. This suggests a metabolic adaptation of TCREDCD39KOHER2 cells towards glycolysis, a less efficient but oxygen-independent ATP production pathway. To investigate the metabolic adaptation of TCREDCD39KOHER2 T cells in a hostile TME, we challenged them in the absence of glucose and oxygen; we noticed a strongly diminished production of reactive oxygen species by TCREDCD39KOHER2 T cells, with the absence of oxygen being the most influencing factor, suggesting less reliance of these cells on mitochondrial activity. To gain a deeper knowledge of how the mitochondrial activity changes upon CD39 disruption, we performed morphological analyses by transmission electron microscopy and observed a rearrangement of TCREDCD39KOHER2 T cells’ mitochondrial cristae upon antigen-specific stimulation. Overall, we linked metabolic reprogramming in TCREDCD39KOHER2 T cells to improved anti-tumor performances. Single-cell RNA sequencing analysis is ongoing to understand the specific pathways involved in this metabolic shift and how can be harnessed to potentiate ACT strategies.


 Alessia Potenza, Laura Conte, Chiara Balestrieri, Martina Spiga, Lorenzo Da Dalt, Fabrizia Bonacina, Paolo Monti, Danilo Norata, Eliana Ruggiero, Chiara Bonini. CD39 disruption boosts TCR-edited T cells by metabolic reprogramming [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB361.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bd9437dc2e0ed80a3857872f18bcdf9620b5f6c" target='_blank'>
              Abstract LB361: CD39 disruption boosts TCR-edited T cells by metabolic reprogramming
              </a>
            </td>
          <td>
            A. Potenza, Laura Conte, Chiara Balestrieri, Martina Spiga, L. Da Dalt, F. Bonacina, Paolo Monti, Danilo Norata, Eliana Ruggiero, Chiara Bonini
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Jose Antonio Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="This study analyzed the genetics of classical Hodgkin lymphoma (cHL) by using circulating tumor DNA (ctDNA). Two genetic subtypes were identified, differing in genetic instability mechanisms: one subtype (64% of cases) showed a higher mutation load and a higher fraction of mutations associated with activation-induced cytidine deaminase and microsatellite instability signatures, while the other subtype (36% of cases) exhibited chromosomal instability with more somatic copy number alterations. Whole-genome duplication was more common in cHL compared to other B-cell tumors and emerged as a prognostic biomarker for patients undergoing ABVD-based therapy. Non-coding regulatory mutations, similar to those in diffuse large B-cell lymphoma, were highly prevalent in 86% of cHL. A recurrent somatic expression quantitative trait locus (seQTL) involving the BCL6 gene was found in 30% of cases. The seQTL of BCL6 aligned with accessible chromatin and increased H3K27 acetylation in cHL, disrupted PRDM1 binding, and co-occurred with BCL6 expression in cHL cells. Weak to strong expression of BCL6 was observed in 68% of cases and BCL6 expression associated with gene repression similarly in cHL and germinal center B cells. After BCL6 degradation, the core set of genes directly bound and regulated by BCL6 was derepressed in cHL and proliferation was impaired. The number and clonality of neoantigens was associated with tumor microenvironment type and response to checkpoint blockade. Finally, ctDNA analysis was suggested as a tool to distinguish ambiguous PET/CT-positive lesions post-treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc33ff26bbe2b930345064a0a4545ff56f194772" target='_blank'>
              A Comprehensive Genetic Study of Classical Hodgkin Lymphoma Using Circulating Tumor DNA.
              </a>
            </td>
          <td>
            M. Pirosa, A. Bruscaggin, L. Terzi di Bergamo, M. Salehi, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, G. A. Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Ilaria Romano, S. Annunziata, F. Bergesio, R. Boldorini, E. Borsatti, P. Bulian, Eleonora Calabretta, S. Chauvie, Francesco Corrado, S. Crisci, M. Cuzzocrea, R. De Filippi, Bernhard Gerber, M. Kurlapski, L. M. Larocca, Elisabetta Merlo, A. Rinaldi, M. Rodari, G. Romanowicz, G. Sacchetti, A. Stathis, G. Stussi, I. Zangrilli, A. Pinto, L. Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, A. Tzankov, Carmelo Carlo-Stella, G. Gaidano, L. Ceriani, Emanuele Zucca, Davide Rossi
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a6b3a707fdcdfce06b9c16b1eadf99680a52469" target='_blank'>
              DNA extrusion size determines pathway choice during CAG repeat expansion
              </a>
            </td>
          <td>
            M. Bhatia, A. Phadte, Anna Lakhina, A. R. Monte Carlo, S. Barndt, A. Pluciennik
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosomal 11q13.3 amplification is the most common gene copy-number variation event in head and neck squamous cell carcinoma (HNSCC) that corresponds with poor prognosis. Although cyclin D1, a G1/S phase cell-cycle regulatory protein at this locus, is considered as a key driver of malignant progression, further exploration is needed to develop more effective targets for cases with this amplification. Using CRISPR-based gene knockout screening of genes located in chr11q13.3, we found that loss of the gene encoding the Fas-associated death domain (FADD) protein, a well-recognized adapter to caspase-8 that induces cell apoptosis, significantly reduced cancer cell proliferation. FADD expression was elevated in chr11q13.3-amplified tumors and correlated with poor prognosis. RNA sequencing, mass spectrometry, and proteomics analyses revealed a direct relationship between FADD and the DNA helicase MCM5 in the S phase. FADD and cyclin D1 acted at different stages of the cell cycle to synergistically induce proliferation, and caspase-8 deficiency was required for the oncogenic activity of FADD. In a patient-derived xenograft model with chr11q13.3 amplification, combined administration of the DNA helicase complex inhibitor and CDK4/6 inhibitor effectively curtailed tumor growth. Overall, this study identified a nonclassic oncogenic role for FADD in mediating tumor progression in HNSCC and provided a feasible treatment option for patients with chr11q13.3 amplification. Significance: FADD promotes progression of tumors with chr11q13.3 amplification by binding to the DNA helicase complex, which can be targeted in combination with cyclin D1 as a viable therapeutic strategy for HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, E. Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Aging is a process of systemic deterioration and the most significant risk factor for cancers. Clonal hematopoiesis (CH) commonly occurs with aging and links to higher mortality, leukemia risk, and cardiovascular diseases. Age-related CH involves the abnormal clonal expansion of hematopoietic stem cells (HSCs) bearing somatic mutations in genes frequently mutated in leukemia, including genes encoding epigenetic regulators, e.g., the DNA demethylase TET2. While such mutations are known to alter the HSC epigenome, the mechanisms through which these mutations drive HSC self-renewal and myeloid transformation preferentially in old age remain elusive.
 In this study, we hypothesize that aging and CH mutations cooperatively reshape the HSC transcriptomic landscape, enhancing HSC competitive advantage that facilitates clonal expansion. Using single-cell multi-omic analyses and flow cytometry for Mx1-cre mouse models, we demonstrated, for the first time, that Tet2 deficiency mitigates detrimental HSC aging processes at the transcriptomic, epigenomic, and cellular levels, thereby promoting age-dependent expansion.
 Age-related gene expression changes, including increased Eya4, Lars2, and Mical2 and decreased Cdk6, CD53, and Sox4, were mitigated in Tet2-deficient HSCs, with a log2 fold-change cutoff as 0.5 and FDR<20%. These gene expression changes were confirmed by single-nucleus ATAC-seq. Chromatin accessibility analysis also revealed Tet2 deficiency predominantly mitigates epigenetic changes associated with HSC aging. Specifically, age-related increase in chromatin accessibility of binding motifs of transcription factors (TFs), such as E2f4, Ppard, Mycn, and Max, and decrease in binding motifs of chromatin accessibility for Gabpa, Runx3, Etv4, and Elf1, were also mitigated in Tet2-deficient HSCs. Furthermore, DNA methylation status of these TF binding motifs is linked to the regulation for the targets’ gene expression, and CTCF insulator binding during HSC aging are also mitigated by Tet2 deficiency. Together, Tet2 deficiency stabilizes TF-target gene pairs critical for maintaining HSC self-renewal during aging.
 In conclusion, our findings reveal that Tet2 deficiency significantly contributes to the mitigation of HSC aging and hijacks the HSC expansion strategy during HSC aging via epigenetic reprogramming, which contributes to age-related CH. By elucidating the transcriptomic and epigenomic alterations in Tet2-deficient HSCs, our study provides novel insights into how age and somatic mutations interact to promote the pathogenesis of age-related hematological diseases. These discoveries enhance our understanding of how aging and somatic mutations interact, offering insights into potential biomarkers and therapeutic targets for age-associated hematological disorders.


 Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina Mujica, Ming Xu, Eric M. Pietras, James DeGregori, Jennifer J. Trowbridge, Hideyuki Oguro, Sheng Li. Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB101.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab96ea3da2d8da3f9378cdc0608c7c70ccd45b06" target='_blank'>
              Abstract LB101: Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis
              </a>
            </td>
          <td>
            Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Ming Xu, Eric M Pietras, James DeGregori, Jennifer J Trowbridge, Hideyuki Oguro, Sheng Li
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immune evasion is a hallmark of gliomas, yet the genetic mechanisms by which tumors escape immune surveillance remain incompletely understood. In this study, we systematically examined the presence of somatic mutations in human leukocyte antigen (HLA) genes and genes encoding proteins involved in antigen-presentation across isocitrate dehydrogenase wild-type (IDHwt) and mutant (IDHmut) gliomas using targeted next-generation sequencing (NGS). To address the challenges associated with detecting somatic mutations in these highly polymorphic and complex regions of the genome, we applied a combination of short-read and long-read sequencing techniques, extended the genetic region of interest (exons and introns), and applied a tailored bioinformatics analysis pipeline, which enabled an accurate evaluation of comprehensive sequencing data. Our analysis identified mutations in HLA class II and non-classical HLA genes as well as genes associated with antigen presentation, such as TAP1/2 and B2M. Three-dimensional modeling of individual mutations simulated the potential impact of somatic mutations in TAP1 and B2M on the encoded protein configuration. The presence of somatic mutations supports the role of antigen-presenting genes in the pathophysiology and potential immune escape of gliomas. Our data demonstrated an increased frequency of such mutations in recurrent glioblastoma (GBM), potentially resulting from a positive selection or mutagenic enrichment of tumor cells during tumor progression. Taken together, this research generates new insights and hypotheses for the functional analysis and optimization of immunotherapy strategies for gliomas, which may guide personalized treatment paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/720b53545edf3b58403eb323f878fcac355077a2" target='_blank'>
              Somatic mutations in HLA class genes and antigen presenting molecules in malignant glioma.
              </a>
            </td>
          <td>
            Sara C. Schulte, Wolfgang Peter, G. Rosenberger, Moritz Schäfer, Cecile L Maire, A. Rünger, A. Ryba, K. Riecken, K. Fita, Jakob Matschke, Nuray Akyüz, Judith Dierlamm, G. Klau, F. Ricklefs, J. Gempt, Manfred Westphal, K. Lamszus, A. Dilthey, M. Mohme
          </td>
          <td>2025-05-05</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Simple Summary Triple-negative breast cancer (TNBC) is an aggressive and hard-to-treat form of breast cancer that has limited treatment options. In TNBC, changes in DNA (mutations) associated with the cancer, frequently occur in the TP53 gene. The TP53 gene normally helps cells repair damage or self-destruct when they are not functioning properly; however when mutated, TP53 cannot function properly. In this study, we found that different types of TP53 mutations, affect how cancer cells manage calcium—an important signal that controls how cells grow and die. Some of these mutations made the cancer cells less likely to die under stress, which may explain why they can resist certain treatments. To address this problem, we tested a TP53 reactivator that helps restore the normal functions of TP53. This TP53 reactivator increased calcium levels in the cancer cells and made them more responsive to treatment. Our findings help explain why some cancers are more aggressive and suggest a new approach to treatment by targeting both the mutant TP53 and the disrupted calcium signals. This could lead to better therapies and improved outcomes for people with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd38d72cf5acc6bf0f18946326e613c63624e9b4" target='_blank'>
              TP53 Mutation-Specific Dysregulation of Store-Operated Calcium Entry and Apoptotic Sensitivity in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            Kaneez E. Rabab, Paul J. Buchanan, Grace Colley, Anita White, Aisling Murphy, Chloe McCormack, Alexander J. Eustace
          </td>
          <td>2025-05-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) constitutes a heterogeneous hematological malignancy, often correlated with disruptions in various signaling pathways, chromosomal deletions, and gene mutations. A comprehensive grasp of CLL pathogenesis and its associated risk factors remains vital for the development of more efficacious treatment strategies. Although non-coding RNAs (ncRNAs) do not encode proteins, they possess substantial regulatory influence over target genes. These ncRNAs govern a multitude of target genes implicated in the pathogenesis of CLL. Furthermore, some ncRNAs serve as prognostic markers in CLL and might contribute to overcoming chemotherapy resistance. This review determines the association between ncRNAs and the molecular mechanisms driving the onset and advancement of CLL, with particular emphasis on the roles of these ncRNAs in modulating signaling pathways, encompassing NF-κB/PI3K-AKT/TNF and P53 in CLL. It also underscores their relevance as significant biomarkers and their potential as therapeutic targets in clinical CLL settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b84b37ca243b8728f07f5d45468a2ea6eb55d49" target='_blank'>
              The Role of MicroRNAs, CircRNAs, and LncRNAs in Chronic Lymphocytic Leukemia.
              </a>
            </td>
          <td>
            Nan Chen, Yaping Zhang, Yongning Jiang, Guangcan Gao, Jianyong Li, W. Shi
          </td>
          <td>2025-04-25</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary KRAS gene mutations are key drivers of many cancers, but how these mutations alter cellular processes to promote tumor growth remains unclear. This study investigates how specific KRAS mutations disrupt protein interactions to rewire cancer cell metabolism. We developed a novel approach combining protein labeling and advanced analysis to compare the interaction partners of normal KRAS with three common cancer-causing mutants (G12C, G12D, and G12V). Our findings reveal that these mutations dramatically change KRAS’s ability to bind proteins controlling insulin signaling, energy production, and stress responses. These altered interactions force cells into abnormal metabolic states that accelerate tumor growth and help evade immune defenses. Notably, all three mutations weaken connections with a protein responsible for breaking down KRAS while strengthening ties to another protein that drives cell growth. By mapping these mutation-specific changes, we explain how different KRAS variants promote cancer and identify new targets for precision therapies. This work provides crucial insights for developing treatments that specifically counteract the effects of particular KRAS mutations, offering hope for more effective anti-cancer strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062a3fc4849de0614d1c46e8b851d9474a3f38cf" target='_blank'>
              Mapping the Interactome of KRAS and Its G12C/D/V Mutants by Integrating TurboID Proximity Labeling with Quantitative Proteomics
              </a>
            </td>
          <td>
            Jiangwei Song, Busong Wang, Mingjie Zou, Haiyuan Zhou, Yibing Ding, Wei Ren, Lei Fang, Jingzi Zhang
          </td>
          <td>2025-04-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Radiation therapy causes long-term skeletal muscle atrophy and fibrosis in juvenile cancer survivors. The mechanisms responsible for the skeletal muscle late effects of radiation therapy are not well-understood and have prevented the development of effective treatments. Using single-cell RNA sequencing (scRNA-seq), we characterize cellular dynamics and communication in a murine model of therapeutic radiation at 24-hours and 56-days post-irradiation (post-IR). We detected changes in muscle stem (satellite) cells (MuSCs) characterized by an acute preservation of committed MuSCs and long-term relative depletion of deep quiescent MuSCs. A conserved senescence Cdkn1a signature was observed in all muscle-resident cells post-IR. Genes related to fibroblast proliferation were up-regulated and a fibrotic and senescent transcriptome persisted in Fibro-adipogenic progenitors (FAPs) post-IR. Intercellular communication analysis revealed FAPs as the primary contributor of extracellular matrix (ECM) and target of monocyte/macrophage-derived TGF-β signalling post-IR through TGF-βR2 on FAPs. Together, our findings provide insights into the potential mechanisms and intercellular communication responsible for radiation-induced muscle atrophy and fibrosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/697ad09c5bedb5b01070a9a0761c75e4c69a9ec3" target='_blank'>
              Single-cell transcriptomic analysis reveals alterations to cellular dynamics and paracrine signalling in radiation-induced muscle pathology.
              </a>
            </td>
          <td>
            Nicolás Collao, E. Johannsen, J. Just, Michael De Lisio
          </td>
          <td>2025-05-02</td>
          <td>American journal of physiology. Cell physiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="RNA-binding proteins (RBPs) regulate gene expression at the post-transcriptional level and are important factors in cancer progression and response to various therapeutic strategies. Radioresistance, an obstacle caused due to various intrinsic and extrinsic factors, remains a major hindrance in the treatment of cancer and could lead to tumor recurrence. Though research is being conducted on the cause and association of radioresistance with various cellular and environmental factors, there remains much to be explored and discovered. The roles of several RNA-binding proteins in tumor progression and metastasis are well documented. In addition, recent studies suggest the connection between Cancer Stem Cells (CSCs) and chemoresistance. We and others have extensively studied the regulatory role of RBPs in regulating CSCs. Resistance to radiation therapy and the involvement of RBPs in this process is under-studied. In this review, we have provided an updated compilation of the significant role played by RBPs in radioresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c698b18a36a4473d26968a3adb42677b721c32" target='_blank'>
              RNA Binding Proteins are Pivotal Regulators of Cancer Radioresistance and Potential Targets for Preventing Tumor Recurrence.
              </a>
            </td>
          <td>
            Vasanth Kanth Thasma Loganathbabu, Srisri Satishkartik, Vanshikaa Karthikeyan, Sayantani Chattopadhyay, Shriya P, Harin N Ganesh, Kirubakaran Rangasamy, Satish Ramalingam, ArulJothi Nagarajan
          </td>
          <td>2025-04-29</td>
          <td>Current radiopharmaceuticals</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7668e6be2f122a26f394a92f23a38b2f39f2818" target='_blank'>
              Genome Editing in Gynecological Oncology: The Emerging Role of CRISPR/Cas9 in Precision Cancer Therapy.
              </a>
            </td>
          <td>
            Naina Kumar
          </td>
          <td>2025-05-20</td>
          <td>Therapeutic innovation & regulatory science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abfdae0d9888d7167ae3464de1089da14e882abe" target='_blank'>
              Multi-cellular phenotypic dynamics during the progression of breast tumors
              </a>
            </td>
          <td>
            Louise Gsell, Spencer S. Watson, Iva Sutevski, Matteo Massara, Klara Soukup, Alper Eroglu, Jeff E Mold, Antony Cougnoux, Johanna A. Joyce, Jean Hausser
          </td>
          <td>2025-04-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Programmable gene integration in human cells has the potential to enable mutation-agnostic treatments for loss-of-function genetic diseases and facilitate many applications in the life sciences. CRISPR-associated transposases (CASTs) catalyze RNA-guided DNA integration but thus far demonstrate minimal activity in human cells. Using phage-assisted continuous evolution (PACE), we generated CAST variants with >200-fold average improved integration activity. The evolved CAST system (evoCAST) achieves ~10 to 30% integration efficiencies of kilobase-size DNA cargoes in human cells across 14 tested genomic target sites, including safe harbor loci, sites used for immunotherapy, and genes implicated in loss-of-function diseases, with undetected indels and low levels of off-target integration. Collectively, our findings establish a platform for the laboratory evolution of CASTs and advance a versatile system for programmable gene integration in living systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc7a114e9413b1138dda6d46ec6c355a9e8561f" target='_blank'>
              Programmable gene insertion in human cells with a laboratory-evolved CRISPR-associated transposase.
              </a>
            </td>
          <td>
            Isaac P. Witte, George D. Lampe, Simon Eitzinger, Shannon M Miller, Kiara N Berríos, A. McElroy, Rebeca T. King, Olivia G Stringham, Diego R. Gelsinger, P. Vo, Albert T. Chen, Jakub Tolar, Mark J. Osborn, S. H. Sternberg, David R. Liu
          </td>
          <td>2025-05-15</td>
          <td>Science</td>
          <td>2</td>
          <td>33</td>
        </tr>

        <tr id="
 6528


 Background:
 NPM1
 GA characterize a clinically important subset of AML cases which relapse in more than 50% of treated patients despite being generally sensitive to conventional chemotherapy regimens. In AML, the interactions between GA in
 NPM1
 ,
 KMT2A
 and menin protein have been linked to leukemogenesis and represent new potential targets for anti-tumor therapies.
 Methods:
 4,206 cases of AML underwent comprehensive genomic profiling from 2019 through 2024, using the FoundationOne Heme combined hybrid capture based DNA and RNA sequencing assay. All classes of relevant GA were evaluated. The tumor mutation burden (TMB), homologous recombination deficiency signature (HRDsig) and microsatellite Stability (MSS) status were determined from the sequenced data.
 Results:
 633 (15.1%) of the 4,206 AML featured
 NPM1
 GA (NPM1mut). Short variant mutations were found in >99% of the NPM1mut AML with the W288fs*12 frameshift base substitution accounting for 92.4% of cases. An
 NPM1
 -
 MLF1
 fusion was identified in 1.3% of NPM1mut cases. The NPM1mut+ were more frequently associated with female patients (53.4% vs 41.5%; p<.0001) and had a slightly higher median age compared to the NPM1 wild type (NPM1wt) AML patients (62yrs vs 60yrs; p<.0001). Majority of patients (>60%) were from European decent. There were greater NPM1 GA in patients with European (77.1% vs 68.5%; p<.0001) and lower with African ancestry (9.2% vs 10.2%; p<.0001). MSI High (0% in both groups) status, HRDsig+ (0—0.1%) and elevated TMB (median < 1 mutation/Mb) were extremely uncommon in both groups. GA more frequent in NPM1mut AML compared to the NPM1wt AML cohort included
 DNMT3A
 (39.2% vs 12.6%; p<.0001),
 FLT3
 (54.5% vs 14.7%; p<.0001)
 ,
 IDH1
 (16.1% vs 5.6%; p<.0001),
 IDH2
 (19.0% vs 9.0%; p<.0001),
 TET2
 (23.4% vs 13.5%; p<.0001) and
 WT1
 (12.5% vs 9.4% p=.02). GA more frequent in NPM1wt AML included
 ASXL1
 (17.1% vs 3.6%; p,.0001),
 BCOR
 (7.5% vs 1.6%; p<.0001),
 KMT2A
 (14.7% vs 0.2%; p<.0001),
 RUNX1
 (22.5% vs 1.9%; p-,.0001),
 STAG2
 (6.9% vs 1.6%; p<.0001) and
 TP53
 (19.1% vs 4.1%; p<.0001).
 Conclusions:
 The development of menin inhibitors has recently identified GA in
 NPM1
 as a promising target of therapy for AML patients. Other therapy targets in AML such as
 FLT3
 and
 IDH1/2
 are more frequently identified in NPM1mut than NPM1wt AML, while
 KMT2A
 is more frequently identified in NPM1wt AML. This genomic landscape study reveals significant differences in important GA associated with AML in NPM1mut and NPM1wt cases which may enrich our understanding of the molecular profile in AML and identify additional targets for therapy.


 Pathogenic genomic alterations in NPM1mut and NPM1wt AML.





 NPM1wt AML (n=3573)
 NPM1mut AML (n=633)
 P-value





 ASXL1
 17.1%
 3.6%
 <.0001



 CEBPA
 6.4%
 8.2%
 NS



 DNMT3A
 12.6%
 39.2%
 <.0001



 FLT3
 14.7%
 54.5%
 <.0001



 IDH1
 5.6%
 16.1%
 <.0001



 IDH2
 9.4%
 19.0%
 <.0001



 KMT2A
 14.7%
 0.2%
 <.0001



 RUNX1
 22.5%
 1.9%
 <.0001



 TET2
 13.5%
 23.4%
 <.0001



 TP53
 19.1%
 4.1%
 <.0001





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f4a065a163ca2b1d9ef8d78ae544e2bc89a03fb" target='_blank'>
              Nucleophosmin (
 NPM1
 ) genomic alterations (GA) in acute myeloid leukemia (AML): A genomic landscape study.
              </a>
            </td>
          <td>
            Osama Batayneh, Alexandra Goodman, Devashish Desai, D. Pavlick, Chelsea Marcus, Caleb Ho, J. Ross, Teresa Gentile, Zheng Zhou, Krishna Ghimire
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Metastatic prostate cancer (mPC) is a clinically and molecularly heterogeneous disease. While there is increasing recognition of diverse tumor phenotypes across patients, less is known about the molecular and phenotypic heterogeneity present within an individual. In this study, we aimed to define the patterns, extent, and consequences of inter- and intra-tumoral heterogeneity in lethal prostate cancer. By combining and integrating in situ tissue-based and sequencing approaches, we analyzed over 630 tumor samples from 52 mPC patients. Our efforts revealed phenotypic heterogeneity at the patient, metastasis, and cellular levels. We observed that intra-patient, inter-tumoral molecular subtype heterogeneity was common in mPC and showed associations with genomic and clinical features. Additionally, cellular proliferation rates varied within a given patient across molecular subtypes and anatomic sites. Single-cell sequencing studies revealed features of morphologically and molecularly divergent tumor cell populations within a single metastatic site. These data provide a deeper insight into the complex patterns of tumoral heterogeneity in mPC with implications for clinical management and the future development of diagnostic and therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb925628a7789a4c45633ccff7c788aadf4bd5ff" target='_blank'>
              Patterns of intra- and inter-tumor phenotypic heterogeneity in lethal prostate cancer.
              </a>
            </td>
          <td>
            Martine P. Roudier, Roman Gulati, Erolcan Sayar, Radhika A Patel, Micah Tratt, Helen Richards, Paloma Cejas, Miguel Muñoz Gomez, Xintao Qiu, Yingtian Xie, B. Hanratty, Samir Zaidi, Jimmy L. Zhao, M. Adil, Chitvan Mittal, Yibai Zhao, R. Dumpit, Ilsa Coleman, J. Low, Thomas W Persse, Patricia Galipeau, John K. Lee, Maria S Tretiakova, Meagan Chambers, F. Vakar‐López, Lawrence D. True, Marie E Perrone, Hung-Ming Lam, Lori Kollath, Chien-Kuang C. Ding, Stephanie Harmon, Heather H. Cheng, Evan Y. Yu, R. Montgomery, Jessica E. Hawley, , Eva Corey, M. Schweizer, Manu Setty, G. Ha, Charles Sawyers, C. Morrissey, Henry Long, Peter S Nelson, Michael C. Haffner
          </td>
          <td>2025-06-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8533a84bf23dfa93f9b981af3fd2fcfd03e25b" target='_blank'>
              AURKA inhibition amplifies DNA replication stress to foster WEE1 kinase dependency and synergistic antitumor effects with WEE1 inhibition in cancers
              </a>
            </td>
          <td>
            Jong Woo Lee, Jackie Shi, Julian Barrantes, Pratima Chaurasia, Sundong Kim, S. Cruz-Gomez, Cindy Yang, Jianlei Gu, Hongyu Zhao, Dejian Zhao, Benjamin Schiff, Ansley Roche, Elizabeth B. Perry, Jeffrey P. Townsend, Gary M. Kupfer, Katerina Politi, E. Golemis, Barbara Burtness
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background CRISPR interference (CRISPRi), the repurposing of the RNA-guided endonuclease dCas9 as a programmable transcriptional repressor, allows highly specific repression (knockdown) of gene expression. CRISPRi platforms can often have incomplete knockdown, performance variability across cell lines and gene targets, and inconsistencies dependent on the guide RNA sequence employed. Results Here, we explore the combination of novel repressor domains with strong Krüppel-associated box (KRAB) repressors, screening > 100 bipartite and tripartite fusion proteins for their ability to reduce gene expression as CRISPRi effectors. We show that these novel repressor fusions have reduced dependence on guide RNA sequences, better slow cell growth when used to knock down expression of essential genes, and function in either fusion or scaffold modalities. Furthermore, we isolate and characterize a particularly effective CRISPRi platform, dCas9-ZIM3(KRAB)-MeCP2(t), which shows improved gene repression of endogenous targets both at the transcript and protein level across several cell lines and when deployed in genome-wide screens. Conclusions We posit that these novel repressor fusions can enhance the reproducibility and utility of CRISPRi in mammalian cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03640-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/541b174ba857f7a16f7cb6e09012bb05165641ba" target='_blank'>
              Engineering novel CRISPRi repressors for highly efficient mammalian gene regulation
              </a>
            </td>
          <td>
            Andrew J Kristof, Krithika Karunakaran, Christopher Allen, Paula Mizote, Sophie Briggs, Zixin Jian, Patrick Nash, J. Blazeck
          </td>
          <td>2025-06-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 e17084


 Background:
 Six-Transmembrane Epithelial Antigen of the Prostate 1 (STEAP1) is a transmembrane protein frequently overexpressed in prostate cancer (PC). Antibody-based therapies targeting STEAP1 have shown promising preclinical and clinical outcomes. Despite its widespread and PC-specific overexpression, the molecular regulation and biological function of STEAP1 remain poorly understood. Elucidating the mechanisms that govern STEAP1 expression and its role in PC progression could provide valuable insights into disease mechanisms, improve patient selection for STEAP1-targeted therapies, and inform combination or sequential treatment strategies.
 Methods:
 To investigate STEAP1 regulation and function, we first compared its expression to established PC driver genes and observed a correlation between STEAP1 and the androgen receptor (AR) as well as several AR-regulated genes. Using pathway analyses in the Decipher (localized prostate cancer) and West Coast Dream Team (WCDT, metastatic prostate cancer) clinical cohorts, we examined dysregulated pathways in patients with high STEAP1 expression. Additionally, we performed a FACS-based whole-genome CRISPR interference (CRISPRi) screen in C4-2B cells using an anti-STEAP1 antibody to identify regulators by comparing STEAP1-high and STEAP1-low expressing subsets.
 Results:
 In the WCDT dataset, STEAP1 was highly expressed in AR-positive metastatic prostate cancer subtypes, irrespective of neuroendocrine differentiation, but was significantly lower in AR-negative subtypes. STEAP1 expression strongly correlated with several AR-regulated genes, including STEAP2, KLK2, NKX3-1, KLK3, ARHGAP6, SLC45A3, and PMEPA1. Pathway analyses in both localized and metastatic PC cohorts revealed dysregulated metabolic pathways, particularly lipid metabolism, in STEAP1-high patients. The CRISPRi screen identified several potential targetable regulators of STEAP1, including MYC, SWI/SNF chromatin remodeling complexes.
 Conclusions:
 Our study establishes a strong association between STEAP1 and AR-driven metastatic castration-resistant prostate cancer subtypes. Furthermore, patient data suggest that STEAP1 is linked to dysregulated metabolic pathways, particularly those involving lipid metabolism. The identification of STEAP1 regulators through the CRISPRi screen provides valuable insights into its regulation and biological function, offering potential avenues for developing combination therapies, strategies to prevent treatment resistance, and improved patient stratification for STEAP1-targeted treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e2f4b4ddfd1356bdf18bff71926fd89045fb54e" target='_blank'>
              FACS-based whole-genome CRISPRi screen to uncover STEAP1 regulators in prostate cancer.
              </a>
            </td>
          <td>
            Shan Zhu, Mikhail Dias, Yuxin Yang, Emma Dolan, Thaidy Moreno Rodriguez, Tamilla Nechiporuk, Jun Zhu, Derek Xiang, Tianyi Liu, Tatyanah Farsh, Meng Zhang, Gabriel Eades, Stanley Leung, Lisa N Chesner, Peter Bruno, M. Cooperberg, David A Quigley, F. Feng, Haolong Li, Carissa E. Chu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>82</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>