<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-28 09:36:42 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="DHFR gene amplification is commonly present in methotrexate (MTX)-resistant colon cancer cells and acute lymphoblastic leukemia. In this study, we proposed an integrative framework to characterize the amplified region by using a combination of single-molecule real-time sequencing, next-generation optical mapping, and chromosome conformation capture (Hi-C). We identified an amplification unit spanning 11 genes, from the DHFR gene to the ATP6AP1L gene position, with high adjusted interaction frequencies on chromosome 5 (~2.2 Mbp) and a twenty-fold tandemly amplified region, and novel inversions at the start and end positions of the amplified region as well as frameshift insertions in most of the MSH and MLH genes were detected. These mutations might stimulate chromosomal breakage and cause the dysregulation of mismatch repair. Characterizing the tandem gene-amplified unit may be critical for identifying the mechanisms that trigger genomic rearrangements. These findings may provide new insight into the mechanisms underlying the amplification process and the evolution of drug resistance. Sequencing a large region of DNA containing many surplus copies of genes linked to drug resistance in colon cancer cells may illuminate how these genomic rearrangements arise. Such regions of gene amplification are highly repetitive, making them impossible to sequence using ordinary methods, and little is known about how they are generated. Using advanced methods, Jeong-Sun Seo at Seoul National University Bundang Hospital in South Korea and co-workers sequenced a region of gene amplification in colon cancer cells. The amplified region was approximately 20 times the length of that in healthy cells and contained many copies of an eleven-gene segment, including a gene implicated in drug resistance. The region also contained mutations in chromosomal repair genes which would disrupt repair pathways. These results illuminate the genetic changes that lead to gene amplification and drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>5</td>
          <td>49</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S. França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>46</td>
          <td>44</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744c450e0fe105bb7eff34d3f9f47cc3b73c8c0c" target='_blank'>
              A patient-derived organoid model captures fetal-like plasticity in colorectal cancer.
              </a>
            </td>
          <td>
            Liang Xiong, Ying Xu, Zhaoya Gao, Jingyi Shi, Yunfan Wang, Xiaodong Wang, Wensheng Huang, Ming Li, Longteng Wang, Jun Xu, Cheng Li, Jin Gu, Hongkui Deng, Molong Qu
          </td>
          <td>2025-07-04</td>
          <td>Cell research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, M. T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67b3f2125e256fc001c7ec119739d1e135a5d03" target='_blank'>
              A Mathematical Model of Persister Cell Plasticity and Its Impact on Adaptive Cancer Therapy
              </a>
            </td>
          <td>
            Adithya Mathews, D. Basanta
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9621f1d2872733a662aa856b54a67a500948b496" target='_blank'>
              Therapeutic targeting of mismatch repair-deficient cancers.
              </a>
            </td>
          <td>
            Paul Johannet, B. Rousseau, Carol Aghajanian, Mike Foote, Luis A. Diaz
          </td>
          <td>2025-07-10</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/503cca6deb2a91fe468e2c781a6fa64738c62b22" target='_blank'>
              Drug resistance and tumor heterogeneity: cells and ensembles
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Hyunbum Jang
          </td>
          <td>2025-05-31</td>
          <td>Biophysical Reviews</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) can lead to metastasis and high mortality rates known for its aggressive and invasive properties. Currently, primary treatment options of surgical resection, chemotherapy and radiotherapy have many therapeutic limitations for OSCC patients due to its dynamic evolutionary pathways and the development of resistance to conventional therapies. Moreover, previous studies fail to emphasize the roles of cellular heterogeneity, drug resistance, and evolutionary trajectories in OSCC. This review explores the intricate tumor microenvironment landscape of OSCC, focusing on the cellular heterogeneity, drug resistance, and evolutionary trajectories as well as genetic, epigenetic, and environmental risk factors contributing to the OSCC progression. Tumor heterogeneity arises from environmental exposures (e.g., tobacco, HPV infection, dietary carcinogens) that drive clonal evolution, creating subpopulations of cells with distinct mutational profiles and therapeutic vulnerabilities. Recent advances in in the precision medicine and combination therapy of OSCC paves the way for innovative therapeutic strategies, such as targeting molecular subclones through real-time monitoring and leveraging computational models to predict treatment response. By recognizing tumor heterogeneity as both a driver of therapeutic resistance and a therapeutic target, precision medicine frameworks can integrate environmental risk factor data, molecular profiling, and early detection tools to optimize outcomes. This review underscores the necessity for a multidisciplinary approach to understand and combat the complexity of OSCC, proposing directions for future research to enhance diagnosis and treatment efficacy. Graphical Abstract This review highlights how research into cellular heterogeneity, drug resistance, and evolutionary trajectories in oral squamous cell carcinoma is revolutionizing our understanding and treatment of the disease, emphasizing personalized approaches that could significantly enhance patient care in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/463cb6e3ce6f98a2e93f47745076dd9434c6558f" target='_blank'>
              Oral squamous cell carcinoma: Insights into cellular heterogeneity, drug resistance, and evolutionary trajectories
              </a>
            </td>
          <td>
            Yang Zhou, Liyin Wang, Minghua Liu, Hongfang Jiang, Yan Wu
          </td>
          <td>2025-06-12</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Anticancer chemotherapy is an essential part of cancer treatment, but the emergence of resistance remains a major hurdle. Metabolic reprogramming is a notable phenotype associated with the acquisition of drug resistance. Here, we develop a computational framework that predicts metabolic gene targets capable of reverting the metabolic state of drug-resistant cells to that of drug-sensitive parental cells, thereby sensitizing the resistant cells. The computational framework performs single-gene knockout simulation of genome-scale metabolic models that predicts genome-wide metabolic flux distribution in drug-resistant cells, and clusters the resulting knockout flux data using uniform manifold approximation and projection, followed by k-means clustering. From the clustering analysis, knockout genes that lead to the flux data near that of drug-sensitive cells are considered drug sensitization targets. This computational approach is demonstrated using doxorubicin- and paclitaxel-resistant MCF7 breast cancer cells. Drug sensitization targets are further refined based on proteome and metabolome data, which generate GOT1 for doxorubicin-resistant MCF7, GPI for paclitaxel-resistant MCF7, and SLC1A5 as a common target. These targets are experimentally validated where treating drug-resistant cancer cells with small-molecule inhibitors results in increased sensitivity of drug-resistant cells to doxorubicin or paclitaxel. The applicability of the developed framework is further demonstrated using drug-resistant triple-negative breast cancer cells. Taken together, the computational framework predicts drug sensitization targets in an intuitive and cost-efficient manner and can be applied to overcome drug-resistant cells associated with various cancers and other metabolic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7052bad828f26940bd1911c2893c8896c16d13b8" target='_blank'>
              Genome-scale knockout simulation and clustering analysis of drug-resistant breast cancer cells reveal drug sensitization targets.
              </a>
            </td>
          <td>
            JinA Lim, Hae Deok Jung, S. Park, Moonhyeon Jeon, Da Sol Kim, Ryeongeun Cho, Dohyun Han, Han Suk Ryu, Yoo-Ree Kim, Hyun Uk Kim
          </td>
          <td>2025-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Next-generation sequencing technology enables uniform and impartial assessment of cancer diagnoses and prognosis. However, such studies are mostly type-specific, and capturing shared genomic abnormalities responsible for neoplastic transformation and progression is a challenging task. Pan-cancer analysis offers insights into the shared and unique molecular mechanisms driving cancer. We conducted an integrated gene-expression analysis using 10,629 samples from 30 distinct cancer types characterized by The Cancer Genome Atlas (TCGA). A gene co-expression network was constructed and genes overlapping between the selected modules and Differentially Expressed Genes (DEGs) were designated as genes of interest. Following a comprehensive literature review, ATP binding cassette subfamily A member 10 (ABCA10) and ATP binding cassette subfamily B member 5 (ABCB5) were selected as key candidates for downstream analysis due to the absence of systematic pan-cancer analysis of these genes. This study presents a unique contribution as the first comprehensive pan-cancer analysis of ABCA10 and ABCB5, highlighting their roles in tumor biology and clinical outcomes. We employed a variety of bioinformatics tools to explore the role of these genes across different tumors. Our research demonstrated that ABCA10 shows reduced expression, while ABCB5 displays variable expression patterns across tumors, indicating their opposing roles and flexible functions in pan-cancer. In many cancer patients, these expression patterns are correlated with worse survival outcomes. Furthermore, immunotherapy responses and immune infiltration across a variety of tumor types are associated with the expression levels of both ABCA10 and ABCB5. These results imply that ABCA10 and ABCB5 could serve as valuable predictive markers and potential therapeutic targets across various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75f1be3a2286bbe9d4105e8c1625c6913bc7d7bf" target='_blank'>
              Integrated pan-cancer analysis revealed therapeutic targets in the ABC transporter protein family
              </a>
            </td>
          <td>
            M. B. E. Masood, Iqra Shafique, Muhammad Inam Rafique, Ayesha Iman, Ariba Abbasi, Mehak Rafiq, Uzma Habib
          </td>
          <td>2025-05-30</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771ef0b5061a9c5213ccca9fb3801ba5c0ea8357" target='_blank'>
              Whole Exome Sequencing and Single-Cell DNA Sequencing for Assessment of Clonal Heterogeneity and Evolution in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Sadiksha Adhikari, S. Eldfors, Esa Pitkänen, Markus Vähä-Koskela, C. Heckman
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="In ~70% of patients with pancreatic ductal adenocarcinoma, the TP53 gene acquires gain-of-function (GOF) mutations leading to rapid disease progression. Specifically, missense p53 (misp53) GOF mutations associate with therapy resistance and worse clinical outcomes. However, the molecular functions of distinct misp53 mutants in plasticity and therapy response remain unclear. Integrating multicenter patient data and multi-omics, we report that the misp53R273H/C mutant is associated with cell cycle progression and a basal-like state compared to the misp53R248W/Q mutant. Loss of misp53R273H/C decreased tumor growth and liver metastasis while prolonging survival in preclinical models. We found that misp53R273H/C specifically regulated the Rb/DREAM axis involved in cell cycle regulation. Notably, a clinical CDK4/6 inhibitor reduced misp53R273H/C mutant expression. However, it triggered MAPK/ERK-mediated resistance mechanisms, enhancing cell survival and resistance to CDK4/6 inhibitors. Combining MAPK/ERK and CDK4/6 inhibitors reduced misp53R273H/C-associated oncogenic functions. Thus, distinct misp53 mutants show unique cell-intrinsic plasticity, therapeutic vulnerabilities, and resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8366ad2cf60feed0070f18be69dfac6ab275f732" target='_blank'>
              TP53 missense–specific transcriptional plasticity drives resistance against cell cycle inhibitors in pancreatic cancer
              </a>
            </td>
          <td>
            Laura Urbach, Lena Wieland, Frederike Penz, Rebecca Diya Samuel, Stefan Küffer, Lukas Klein, Christof Lenz, Ulrich Sax, Michael Ghadimi, Ramona Schulz-Heddergott, E. Hessmann, V. Ellenrieder, Nelson Dusetti, Shiv K. Singh
          </td>
          <td>2025-07-04</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 8624


 Background:
 Lung adenocarcinoma (LUAD), the most common lung cancer subtype, has poor survival rates and limited treatment options. Among its molecular drivers, loss of methylthioadenosine phosphorylase (MTAP) is linked to aggressive development and poor outcomes, yet its downstream effects remain unclear. MTAP is essential for purine biosynthesis and S-adenosyl methionine (SAM) production, a key methyl donor for epigenetic regulation. MTAP-loss leads to methylthioadenosine (MTA) accumulation, which inhibits methyltransferase activity and disrupts epigenetic control. However, the impact of MTAP deficiency on transcriptional and metabolic programming in LUAD is still largely unknown, highlighting a critical gap for targeted therapy. This study examines these molecular consequences to identify potential therapeutic vulnerabilities in MTAP-deficient LUAD.
 Methods:
 RNA-sequencing data for 510 LUAD samples were obtained from The Cancer Genome Atlas (TCGA) via cBioPortal. Samples were classified as MTAP-loss (n=64) or MTAP-normal (n=446) based on mutational data. Differential gene expression was performed using the DESeq2 package for R and the Benjamini-Hochberg procedure was used to control for false discovery rate. DNA methylation (Illumina Human Methylation 450k) was compared between MTAP-loss (n=46) and MTAP-normal (n=404) cohorts. Significantly altered probes were mapped to differentially expressed genes (DEGs). Pathway and gene ontology (GO) analyses were conducted using KEGG, GO, and EnrichR to identify dysregulated pathways.
 Results:
 DNA methylation analysis revealed 581 hypomethylated and only 51 hypermethylated probes in MTAP-loss samples, underscoring a global hypomethylation phenotype likely driven by MTA accumulation. We identified 343 differentially expressed, hypomethylated genes in MTAP-loss LUAD samples. Of these, upregulated genes were highly enriched in mitochondrial function and stress response pathways, whereas downregulated genes were linked to cell differentiation and developmental processes, suggesting an epigenetically driven metabolic reprogramming. Notably, these alterations may confer heightened cellular survival and adaptability under stress, while curtailing normal differentiation programs.
 Conclusions:
 Our findings indicate that MTAP loss in LUAD leads to a coordinated shift in DNA methylation and gene expression, promoting survival-focused metabolic and stress responses at the expense of normal regulatory pathways. These results highlight novel vulnerabilities in MTAP-deficient tumors and suggest potential targets for precision therapies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bac3c5fbe15b9c30b43fa429b943e30d1102b5e9" target='_blank'>
              Epigenetic and transcriptional consequences of MTAP-loss in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Swati Pothukuchi, Jihao Xu, Ssu-Wei Hsu, Ching-Hsien Chen
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Molecular genetic diagnoses are critical to prevention and treatment of inherited polyposis and colorectal cancer. 19 genes responsible for these conditions are known, but many severely affected patients and families remain unsolved. Cryptic intronic variants that alter splicing of these genes and incomplete characterisation of recessive predisposition contribute to these diagnostic gaps. Methods Adaptive sampling long-read DNA sequencing targeted to 19 colon cancer genes, paired with direct long-read RNA whole-transcriptome sequencing, was undertaken for four patients referred for deficiency of mismatch repair proteins and/or familial polyposis, for whom multigene panel testing yielded negative or uncertain germline results. Results Genetic diagnoses were obtained for all four patients. Each patient carried a cryptic intronic germline variant in a different colon cancer gene. The variants abrogated splicing by various mechanisms, all leading to loss of gene function. Patient 1 was heterozygous for intronic insertion into MSH2 of an Alu element, leading to extension of transcription into the affected intron and a stop. Patient 2 was heterozygous for deep intronic insertion into APC of a Long Interspersed Nuclear Element (LINE), creating a pseudoexon and a stop. Patient 3 was compound heterozygous at MLH3, including a cryptic intronic substitution leading to exon skipping and a stop. Patient 4 was compound heterozygous at MUTYH, including a deep intronic deletion yielding an extremely short intron and transcriptional loss of an exon encoding a critical protein domain. Conclusion Paired long-read DNA and RNA sequencing can enhance diagnostic yield through detection of cryptic intronic variants that impact cancer predisposition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5066c2b7365798ed65ee7d8872b1b51cf8054789" target='_blank'>
              Long-read DNA and RNA sequencing for inherited polyposis and colorectal cancer: cryptic intronic variants and multiple mutational mechanisms
              </a>
            </td>
          <td>
            A. Jacobson, Amal AbuRayyan, S. Gulsuner, Haley Slater, Yagiz Anasiz, Sirajummuneer M Ahmad, Ming K. Lee, Jessica B Mandell, Emily J Rettner, E. Konnick, Colin C Pritchard, M. King, T. Walsh, Brian H Shirts
          </td>
          <td>2025-06-26</td>
          <td>Journal of Medical Genetics</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eaa0126e7f9afd60d57ab46895cddf6ea4cbe3" target='_blank'>
              Pangenome-based identification of cryptic pathogenic variants in undiagnosed rare disease patients
              </a>
            </td>
          <td>
            S. S. Jang, S. Kim, S. Lee, S. Kim, J. Moon, J. Kim, J. Chae
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Despite extensive research efforts, glioblastoma (GBM) remains a deadly disease with poor prognosis. Although previous studies have identified various cell states within GBM tumors, the molecular mechanism underlying adaptive GBM cell plasticity induced by conventional therapy remains unclear.


METHODS
We used fluorescent reporters for proneural (PN) and mesenchymal (MES) subtypes to monitor GBM cell plasticity in real-time across multiple patient-derived cell lines. This approach revealed cells that concurrently expressed both proneural and mesenchymal markers. To investigate this unique hybrid population, we implemented a comprehensive methodological approach encompassing bulk and single-cell RNA sequencing, single-cell ChIP sequencing, nuclear proteomics, high-resolution imaging, orthotopic mouse models, clinical dataset analysis, and pharmacological and genetic techniques. This multifaceted strategy allowed us to gain functional and molecular insights into this distinct cellular population.


RESULTS
We showed that these hybrid cells are increased by conventional therapies, and are resistant to these therapies. At the molecular level, hybrid cells display significant alterations in chromatin structure and nuclear protein composition, elevated transcriptional activity, Myc activation, and improved transport between the nucleus and cytoplasm. Genetic and pharmaceutical inhibition of the nuclear import/export shuttling machinery, increased in hybrid cells, effectively suppressed adaptive GBM cell plasticity and hybrid identity, thereby enhancing the sensitivity of GBM cells to therapies.


CONCLUSION
Our results indicate that GBM hybrid cells play a crucial role in chemoradiation resistance. The nuclear transport machinery presents a potential therapeutic target for hybrid cells, offering a way to counteract the typical resistance to treatment observed in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0e616d882e7bd8159604ae82c04f4049ee875a" target='_blank'>
              Proneural-Mesenchymal hybrid glioblastoma cells are resistant to therapy and dependent on nuclear import.
              </a>
            </td>
          <td>
            G. Bourmeau, O. Anezo, Jeremy Raymond, A. Ballestín, C. Pichol-Thievend, J. Reveilles, Adrien Thomas, Lin Wang, Melanie Miranda, Eve Moutaux, S. Liva, V. Ribecco, L. Besse, F. Dingli, D. Loew, Céline Vallot, G. Gargiulo, V. Ravi, Kevin Joseph, G. Seano
          </td>
          <td>2025-07-08</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="This study investigated whether therapeutic X-ray doses can affect the expression of mismatch repair (MMR) genes and proteins using Lynch syndrome-associated human colorectal cancer cell lines. MMR-deficient cell lines (HCT116, SW48, LoVo) and an MMR-proficient control cell line (HT29) were exposed to X rays [a 2 Gy dose or 2 Gy daily for five consecutive days (10 Gy)]. Reverse transcription quantitative real-time PCR (RT-qPCR) and Western blotting were used to detect the radiation-induced changes in the expression of RNAs and proteins, respectively. RT-qPCR revealed that MLH1 and MSH6 genes were stably expressed regardless of the MMR status of the cell line and the radiation dose. In contrast, the MSH2 gene was either up-regulated or down-regulated after 2 Gy or 10 Gy or both. The expression of PMS2 increased after 10 Gy irradiation in all MMR-deficient cell lines, even though the data were not statistically significant compared to other doses, except for the LoVo cell line. Protein expression analysed using Western blotting demonstrated that MLH1 protein expression was stable, whereas the expression of MSH2 was significantly affected by radiation exposure in both MLH1-deficient cell lines. No correlation between the expression of RNA and protein could be identified. In conclusion, radiation may have significantly differential effects on MMR RNA and protein expression when different cell lines, doses, and specific genes are considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1b8835db8d3ae7872bbc2b8269596b668e92380" target='_blank'>
              The Effect of X rays on the Expression of Mismatch Repair Genes and Proteins in Lynch Syndrome Associated Human Colorectal Cancer Cell Lines.
              </a>
            </td>
          <td>
            Mingzhu Sun, L. Cruz-Garcia, Danny Freestone, Kevin Monahan, C. Badie, Yannick Comoglio, Hannah Mancey, J. Moquet, Stephen Barnard
          </td>
          <td>2025-06-20</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Activation of oncogenes, such as through mutations in Kirsten rat sarcoma viral oncogene homolog (KRAS), triggers profound disruptions in cellular homeostasis that set off a cascade of stress responses. These responses enable cells to cope with the array of challenges encountered during tumorigenesis by activating defense mechanisms that promote adaptation and survival. Key components of this oncogenic stress response include heat shock proteins, the ubiquitin-proteasome system, autophagy, nuclear factor erythroid 2-related factor 2–antioxidant response element signaling, DNA damage response proteins, p53, redox-regulating proteins, and stress granules. This review concentrates on KRAS-driven oncogenic transformation, as KRAS mutations are among the most common in human cancers, accounting for over 90% of pancreatic ductal adenocarcinoma cases, around 30% of lung cancers, and approximately 50% of colorectal cancers. We examine the intricate molecular interplay between oncogenic stress and the associated cellular defense mechanisms, emphasizing the key molecular events that follow KRAS activation. Importantly, the very pathways that allow cancer cells to adapt to oncogenic stress also offer novel therapeutic opportunities. By selectively targeting pivotal regulators within these stress response pathways, we can potentially disrupt the survival mechanisms of cancer cells. This strategy not only promises to enhance the effectiveness of existing treatments but also paves the way for the development of innovative therapies designed to combat tumor progression. In essence, exploiting oncogenic stress responses represents an original and promising therapeutic approach in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a19b0d4dfe04d02b6a9051c867a7e87fe82159" target='_blank'>
              Oncogenic stress response mechanisms as new therapeutic targets in cancer treatment: A review
              </a>
            </td>
          <td>
            J. Iovanna, Matias Estaras, Daniel Grasso, Martin E. Fernández Zapico, J. L. Neira, P. Santofimia-Castaño
          </td>
          <td>2025-06-13</td>
          <td>Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Stem cell plasticity is crucial for rapidly proliferating tissues, such as the intestine, to cope with microenvironmental stress and maintain homeostasis. This framework may be employed by cancer stem cells (CSCs) to interact with the tumor microenvironment (TME) and give rise to drug resistance, tumor recurrence, and metastasis. Here, we consider colorectal cancer (CRC) as an example to review the technological and conceptual advances in microenvironmental regulation of cancer stemness. Some unique features of CRC, including reprogramming of stromal cells, bacterial niche alteration, and metabolic interplay, were summarized to illustrate how the TME affects the plasticity of CSCs. The key pathways and molecular mechanisms involved in metastasis and drug resistance are reviewed to obtain a deeper understanding of how CSCs may be manipulated for better outcomes in CRC treatment. These findings advance our knowledge of tumor heterogeneity and may provide insights into the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e85fc5484d5fb7be8f2f1af589f1c3b9e327f305" target='_blank'>
              Microenvironmental Regulation of Colorectal Cancer Stemness: Technological Advances, Molecular Basis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Hui Zhao, Qilin Li, Jichao Qin
          </td>
          <td>2025-06-26</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The evolution of techniques used to identify structural variants (SVs) and copy number variants (CNVs) in genomes have seen significant development in the last decade. With the growing use of more technologies including chromosomal microarray, genome sequencing and genome mapping in clinical cytogenetics laboratories, reporting the frequency of SVs and CNVs has increased the complexity of genomic results. In conventional testing (e.g. karyotype or FISH) individual cells are analyzed and abnormalities are reported at the single cell level directly as a proportion of the analyzed cells. Whereas for bulk genome assays structural and sequence changes are often reported as variant allele frequencies and fractional copy number states. The International System of Cytogenomic Nomenclature (ISCN) recommends converting these values into a “proportion of the sample”, which requires different calculations and underlying assumptions based on the data type. This review illustrates how the different methods of interpreting and reporting data are performed and identifies challenges in the conversion of these values to a proportion of the sample. We stress the need for careful interpretation of data with consideration for factors that may alter how proportions are reported including overlapping SVs and CNVs or regions with acquired homozygosity. We also demonstrate, using validation data of SVs and CNVs tested by multiple techniques how results are largely consistent across methodologies, but can show dramatic differences in rare circumstances. This review focuses on illustrating many of the challenges with aligning reporting using different techniques and their underlying assumptions. As hematologic disease classifications start to incorporate numeric limits (e.g. VAF defining thresholds), it is important for laboratory geneticists, pathologists and clinicians to appreciate the differences in methodologies, potential pitfalls and the nuances when comparing bulk genome analyses to the more conventional single cell techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883b8047db9fe9c01ef1b4982d92e6a1387b4be0" target='_blank'>
              What the VAF? A guide to the interpretation of variant allele fraction, percent mosaicism, and copy number in cancer
              </a>
            </td>
          <td>
            Adam C Smith, Hubert Tsui, Sila Usta, Jose-Mario Capo-Chichi
          </td>
          <td>2025-07-08</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Targeted therapy has markedly improved outcomes for patients with non-small cell lung cancer (NSCLC). However, the emergence of drug resistance remains a major clinical challenge, limiting long-term treatment efficacy. Clustered regularly interspaced short palindromic repeats (CRISPR)/Cas9, a revolutionary gene-editing technology, offers precise and efficient genetic modifications, providing new insights into the mechanisms of drug resistance in NSCLC. The present review explored the application of CRISPR/Cas9 in overcoming resistance associated with key oncogenic drivers, including EGFR, KRAS, ALK, ROS1, MET and BRAF. It summarized recent advances in CRISPR-based strategies to reverse resistance, enhance targeted therapy effectiveness and develop potential therapeutic interventions. Additionally, it discussed current limitations, including off-target effects, delivery challenges and ethical concerns, while highlighting future directions for clinical translation. Using CRISPR/Cas9 technology may pave the way for novel, personalized treatment approaches in NSCLC, ultimately improving patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ddd19a50ff46be19254c0572146995de6cd1223" target='_blank'>
              Harnessing CRISPR/Cas9 to overcome targeted therapy resistance in non-small cell lung cancer: Advances and challenges (Review)
              </a>
            </td>
          <td>
            Jianting Du, Xian Gong, Renjie Huang, Bin Zheng, Chun Chen, Zhang Yang
          </td>
          <td>2025-07-09</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Acute PM2.5 exposure has been implicated in lung cancer progression, yet its impact on genetically distinct NSCLC cells remains underexplored. This study investigates how mutation-specific transcriptional responses influence susceptibility to PM2.5-induced oncogenic alterations, focusing on A549 and NCI-H1975 cells. This provides preliminary insight into the transcriptomic effects of acute PM2.5 exposure in NSCLC cells with distinct oncogenic mutations (A549 and NCI-H1975), serving as a guide for understanding mutation-specific responses to environmental stress. Cells were exposed to PM2.5 (200 µg/mL, 24 h), followed by RNA sequencing and analysis. Gene ontology and pathway enrichment analyses were conducted to identify key molecular alterations associated with tumour progression. NCI-H1975 cells exhibited a stronger transcriptional response, with a higher fold change in differentially expressed genes (DEGs), indicating greater PM2.5 susceptibility. Upregulated genes were linked to oxidative stress, carcinogen activation, metabolic reprogramming, and therapy resistance, reinforcing tumour survival under PM2.5 stress. Conversely, the downregulation of tumour suppressor genes suggests immune suppression and potential immunotherapy resistance. This study reveals that acute PM2.5 exposure induces mutation-specific transcriptomic alterations in NSCLC, with EGFR-mutant cells exhibiting heightened oxidative stress, metabolic shifts, and immune evasion. The upregulation of key genes highlights the profound molecular impact of short-term exposure, paving the way for future studies on pollution-driven oncogenic mechanisms and resistance pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a6ed301518d88cdb3fb1cbceb5ace0ecec33a5" target='_blank'>
              Acute PM2.5 Exposure in Distinct NSCLC Cell Lines Reveals Strong Oxidative Stress and Therapy Resistance Signatures Through Transcriptomic Analysis
              </a>
            </td>
          <td>
            A. Panya, Saruda Thongyim, Pachara Sattayawat, Sahutchai Inwongwan
          </td>
          <td>2025-06-01</td>
          <td>Toxics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec7c3cc61289a3954a3ddd8b996ed6f9c80850e" target='_blank'>
              The adaptive molecular landscape of reprogrammed telomeric sequences
              </a>
            </td>
          <td>
            Melania D’Angiolo, Benjamin P. Barré, Sakshi Khaiwal, J. Muenzner, Johan Hallin, Matteo De Chiara, Nicolò Tellini, J. Warringer, M. Ralser, Eric Gilson, Gianni Liti
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 6519


 Background:
 Acute Myeloid Leukemia (AML) is an aggressive hematologic malignancy marked by abnormal proliferation of myeloid progenitor cells, often leading to poor outcomes and high relapse rates. While the genetic underpinnings of AML are well-documented, the role of immune dysregulation in its progression remains underexplored. This study integrates whole exome sequencing (WES) and transcriptome analysis to identify genetic mutations, transcriptional alterations, and immune regulatory disruptions, with a focus on their contributions to hematopoiesis and immune dysfunction.
 Methods:
 WES and RNA sequencing were performed on 10 AML patient samples to investigate somatic mutations, differential gene expression, and immune-related pathways. WES data were analyzed to detect mutations in AML-associated genes, while transcriptomic analysis compared gene expression profiles between AML cells and normal hematopoietic stem cells (HSCs). Bioinformatic tools, including differential expression analysis, Gene Set Enrichment Analysis (GSEA), and pathway mapping, were employed to identify key regulatory networks. Single-cell RNA sequencing (scRNA-seq) was conducted to assess cellular heterogeneity and differentiation dynamics, with a focus on immune cell subsets and pathways involved in immune evasion.
 Results:
 Analysis revealed recurrent mutations and dysregulation in genes critical to hematopoiesis, apoptosis, and immune regulation, including
 RUNX1, FLT3, CEBPA, TP53, WT1, GATA2,
 and
 TET2
 . Transcriptomic profiling highlighted distinct gene expression patterns in AML cells, with significant disruption in cell cycle control, differentiation, and apoptosis. Dysregulated immune pathways, such as IL-7R and PD-1, were identified as key contributors to immune cell activation impairment and immune tolerance. scRNA-seq data provided insights into the cellular heterogeneity of AML, uncovering altered lineage differentiation and immune subset composition, which may facilitate immune evasion and disease progression.
 Conclusions:
 This integrative analysis illuminates the interplay between genetic mutations, transcriptional dysregulation, and immune dysfunction in AML. The findings underscore the pivotal role of immune pathways, such as IL-7R and PD-1, in AML pathogenesis, presenting them as potential therapeutic targets. By linking genetic and immune alterations, this study advances our understanding of AML biology and highlights the need for therapies addressing both genetic and immune dysfunctions to improve clinical outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0560b8cde9abe934906f2d0949eedbe36b4b07c6" target='_blank'>
              Decoding immune dysregulation in AML: Insights from integrated genomic and transcriptomic analysis.
              </a>
            </td>
          <td>
            Harsh Goel, Pranay Tanwar
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, R. Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b2aba01b3716bf8fef6d8f3f3f6e3783dfc01cf" target='_blank'>
              Recent advances in novel targeting mechanisms for colorectal cancer.
              </a>
            </td>
          <td>
            Zhiping Li, Hailing Wang, Yan Liu, Ahequeli Gemingnuer, Yinan Wang, Xin Meng
          </td>
          <td>2025-07-11</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background/Objectives: Aneuploidy is near-ubiquitous in cancer and can decrease chemotherapy efficacy while also sensitizing cells to other drugs. Methods: To systematically identify treatment strategies that target aneuploid cancers, data were integrated from The Cancer Genome Atlas (TCGA; 10,967 samples, 16,948 aneuploidy events) and the Broad Institute’s Profiling Relative Inhibition Simultaneously in Mixtures (PRISM) screen of 578 cancer cell lines and 4518 compounds. Results: Our analyses uncovered 37,720 significant positive and negative associations linking specific aneuploidies and treatments with patient prognosis or cell viability. Within TCGA data, 22 treatments correlated with improved 5-year survival for specific aneuploid cancers, whereas 46 were linked to worse outcomes. A complementary analysis of PRISM identified 17,946 compound–aneuploidy associations and 16,189 mechanism of action (MOA)–aneuploidy associations. Pathway-altering compounds that selectively reduce viability in cells with aneuploidy profiles were discovered, including an unexpectedly prominent number of glucocorticoid receptor agonists. Conclusions: This integrated dataset provides a resource for designing therapeutic decision hypotheses, identifying drug-repurposing opportunities, and informing future studies aimed at targeting aneuploidy-induced vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811f8e93329a18a966c2c968f4a1e0431dca5e53" target='_blank'>
              Half the Chromosome It Used to Be: Identifying Cancer Treatments Targeting Aneuploid Losses
              </a>
            </td>
          <td>
            Andrew O. Disharoon, Joe R. Delaney
          </td>
          <td>2025-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2537688c884004a06a6382cdb89bfdc63b85ffff" target='_blank'>
              CRISPR-Based Gene Dependency Screens reveal Mechanism of BRAF Inhibitor Resistance in Anaplastic Thyroid Cancer
              </a>
            </td>
          <td>
            Shawn Noronha, Yue Liu, Gaga Geneti, Haojian Li, Xiaolin Wu, David Sun, Vaibhavi Gujar, Takashi Furusawa, Alexei Lobanov, Maggie Cam, Lipika R. Pal, N. Nair, Chi-Ping Day, E. Ruppin, Chandrayee Gosh, Jiangnan Hu, Suresh Kumar, T. Andresson, King Chan, Maura O’Neill, Raj Chari, Y. Pommier, J. Rivero, Urbain Weyemi, E. Kebebew, Myriem Boufraqech
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Genome-wide association studies have suggested numerous colorectal cancer (CRC) susceptibility genes, but their causality and therapeutic potential remain unclear. To prioritise causal associations between gene expression/splicing and CRC risk (52,775 cases; 45,940 controls), we perform a transcriptome-wide association study (TWAS) across six tissues with Mendelian randomisation and colocalisation, integrating sex- and anatomical subsite-specific analyses. Here we reveal 37 genes with robust causal links to CRC risk, ten of which have not previously been reported by TWAS. Most likely causal genes with evidence of cancer cell dependency show elevated expression linked to risk, suggesting therapeutic potential. Notably, SEMA4D, encoding a protein targeted by an investigational CRC therapy, emerges as a key risk gene. We also identify a female-specific association with CRC risk for CCM2 expression and subsite-specific associations, including LAMC1 with rectal cancer risk. These findings offer valuable insights into CRC molecular mechanisms and support promising therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b470fb91f2d0d137327a26167f01f560bda9b57e" target='_blank'>
              Multi-tissue expression and splicing data prioritise anatomical subsite- and sex-specific colorectal cancer susceptibility genes
              </a>
            </td>
          <td>
            E. Hazelwood, D. Canson, Benedita Deslandes, Xuemin Wang, P. F. Kho, D. Legge, Andrei-Emil Constantinescu, Matthew A Lee, D. T. Bishop, Andrew T. Chan, S. Gruber, J. Hampe, L. Le Marchand, Michael O Woods, Rish K. Pai, S. Schmit, J. C. Figueiredo, Wei Zheng, J. R. Huyghe, Neil Murphy, Marc J. Gunter, T. Richardson, V. L. Whitehall, E. E. Vincent, Dylan M Glubb, Tracy A. O’Mara
          </td>
          <td>2025-05-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Despite advancements in cancer therapy, clinical options remain limited and therapeutic resistance inevitably develops. In melanoma, the second most commonly mutated driver oncogene is neuroblastoma rat sarcoma viral oncogene homolog (NRAS) which remains undruggable by small molecular agents. For those treated with BRAF/MEK combination therapy, in the setting of V600 BRAF mutations, or immunotherapies regardless of somatic genetic make-up, resistance eventually develops in the majority of cases. Developing rational combination therapies, as suggested by preclinical studies involving genetic manipulation of resistance mediators, necessitates a substantial expansion of druggable targets. Novel chemistry strategies, such as chemoproteomics platforms and chemical inducer of proximity (CIP) agents, offer promising solutions. Chemoproteomics enables the rapid identification of ligands to guide medicinal chemistry, while CIP agents alter the expression levels of key proteins. Using oncogene (such as NRAS) and p53 as pivotal resistance mediators, and the lineage-specific transcription factor SRY-box transcription factor 10 [SOX10]) in melanoma as illustrative examples, we demonstrate how these technologies can be leveraged to rapidly expand the druggable target pool and overcome resistance mechanisms, ultimately paving the way for regimens that produce deeper and more durable responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525c67efe3ba67b1849a4b3d8417e86cfd7d04fc" target='_blank'>
              The Rise of New Targets in Melanoma.
              </a>
            </td>
          <td>
            Xue Bai, Keith T Flaherty
          </td>
          <td>2025-06-26</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are repetitive DNA sequences that contribute to genetic diversity and play a significant role in disease susceptibility. The human genome contains approximately 1.5 million STR loci, collectively covering around 3% of the total sequence. Certain repeat expansions can significantly impact cellular function by altering protein synthesis, impairing DNA repair, and leading to neurodegenerative and neuromuscular diseases. Traditional short-read sequencing struggles to accurately characterize STRs due to its limited read length, which limits the ability to resolve repeat expansions, increases mapping errors, and reduces sensitivity for detecting large insertions or interruptions. This review examines how long-read sequencing technologies, particularly Oxford Nanopore and PacBio, overcome these limitations by enabling direct sequencing of full STR regions with improved accuracy. We discuss challenges in sequencing, bioinformatics workflows, and the latest computational tools for STR detection. Additionally, we highlight the strengths and limitations of different methods, providing deeper insight into the future of STR genotyping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bca970ab6c190594afe0092773766d77d84cc3b" target='_blank'>
              Get ready for short tandem repeats analysis using long reads-the challenges and the state of the art
              </a>
            </td>
          <td>
            Marija Chaushevska, Karmele Alapont-Celaya, Anne Kristine Schack, Lukasz Krych, M. C. Garrido Navas, Anastasia Krithara, Gjorgji Madjarov
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Identifying genetic mutations contributing to solid tumors by altering the biological pathways related to tumor formation and development is essential for the development of targeted therapies. This study aimed to identify commonly mutated genes and altered pathways in canine solid tumors. Four dogs with different types of naturally occurring neoplasias (urothelial carcinoma, adenocarcinoma, rhabdomyosarcoma, and chondrosarcoma) were randomly selected and classified into carcinoma and sarcoma groups based on histopathological findings. Tumor tissues were analyzed using whole-genome sequencing, and significant variants shared within each tumor group were identified. Gene set enrichment analyses were conducted to compare the biological and functional pathways altered by the mutations in each carcinoma and sarcoma group. Forty-three and fifty-eight genes were identified in the carcinoma and sarcoma groups, respectively. Distinctions between the two tumor groups were noted for mutations related to tumor metastatic function. Mutations were identified in genes encoding cell adhesion molecules in the carcinoma group, whereas significant variations in extracellular matrix-related molecules were evident in the sarcoma group. This study revealed mutations and modified pathways associated with immune and tumor metastatic functions in canine carcinoma and sarcoma, indicating their significant relevance to the development and progression of each tumor group. Additionally, the distinctions indicated that different therapeutic approaches were required for each tumor group.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ab56a6166113f6d5cb839f0773a111660d3e21" target='_blank'>
              Identifying functional roles and pathways of shared mutations in canine solid tumors by whole-genome sequencing
              </a>
            </td>
          <td>
            YeSeul Jeon, Hyeona Bae, Seung-Wan Woo, Jaemin Kim, Dohyeon Yu
          </td>
          <td>2025-05-30</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Globally, gastric cancer (GC) stands as the fifth most prevalent form of malignant neoplasm and represents a significant contributor to mortality associated with oncological conditions. Despite advancements in therapeutic strategies for GC, the outcomes for patients with advanced stages of the disease continue to be unfavorable, largely due to tumor heterogeneity and the challenges posed by resistance to therapeutic agents. Metabolic reprogramming is pivotal in driving the advancement of GC, contributing to the development of resistance to pharmacological treatments and facilitating the cancer’s ability to evade immune surveillance. Developing multi-target comprehensive treatment strategies by integrating tumor microenvironment (TME) modulation holds promise for significantly improving therapeutic efficacy.The study analyzed GC and identified key cell subtypes by integrating data derived from single-cell RNA-sequencing (scRNA-seq) alongside spatial transcriptomics information. Cell type identification was accomplished using the tool of Seurat, and the spatial distribution of cell types was revealed through the Robust Cell Type Decomposition technique. CellChat was used to analyze the interactions between key cell subtypes and other cells, and the “StLearn” package was employed to investigate spatial cell communication in depth. Additionally, the functional role of the key molecule ELK4 was validated through in vitro experiments.This research utilized scRNA-seq combined with spatial transcriptomics to comprehensively analyze GC, identifying the C1 NDUFAB1+ subtype, which exhibited high proliferative activity, metabolic reprogramming capabilities, and immune evasion properties. It was found that the C1 NDUFAB1+ subtype closely interacted with fibroblasts and pericytes via the PARs signaling pathway. Additionally, in vitro experiments confirmed that knockdown of ELK4 substantially curbed tumor cell proliferation, migration, and invasion.This study revealed the main significance of the C1 NDUFAB1+ subtype in GC, elucidating its core mechanisms in tumor progression, metabolic reprogramming, and immune evasion. ELK4 was identified as a key regulatory factor that markedly enhanced the proliferation, migratory capacity, and invasive potential of tumor cells, while changes in the TME were a driving force behind immune suppression and drug resistance. The findings underscored the importance of developing specific therapeutic targets, targeting metabolic reprogramming, and overcoming immune evasion, providing new theoretical foundations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94360dc80f5310630bd93066b812e27d59cfa9fb" target='_blank'>
              Integrative single-cell and spatial transcriptomics uncover ELK4-mediated mechanisms in NDUFAB1+ tumor cells driving gastric cancer progression, metabolic reprogramming, and immune evasion
              </a>
            </td>
          <td>
            Yuwei Sun, Wenyang Nie, Zhikai Xiahou, Xiaojing Wang, Wenjia Liu, Zongkai Liu, Zhiheng Lin, Zhaidong Liu
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d135048b3a2d9e0d80a5651add7d7f82d46f7f4a" target='_blank'>
              Innate immunity and the NF-κB pathway control prostate stem cell plasticity, reprogramming and tumor initiation.
              </a>
            </td>
          <td>
            Chen Jiang, Yura Song, S. Rorive, Justine Allard, Elisavet Tika, Zahra Zahedi, C. Dubois, Isabelle Salmon, Alejandro Sifrim, Cédric Blanpain
          </td>
          <td>2025-06-23</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The emergence of CRISPR-Cas9 technology has transformed functional genomics, offering unmatched opportunities to dissect and understand biological pathways and identify novel therapeutic targets in cancer. Breast cancer is a complex, heterogeneous disease and remains a major cause of morbidity and mortality in women, particularly when diagnosed at advanced or metastatic stages where effective treatments are limited. High-throughput CRISPR screening is undoubtedly a powerful tool to discover novel drug targets, uncover synthetic lethal interactions, and identify vulnerabilities in cancer. This review focuses on advances in our understanding of breast cancer developed through CRISPR-based screening technology, particularly in identifying drivers of breast cancer progression, growth, and metastasis, as well as in identifying potential new therapeutic targets and combination therapies. We discuss recent discoveries, current challenges, and limitations of this approach and explore how advancements in CRISPR technology could have a profound impact on the future of breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66815ded8a01a2e4c316009cc71e7951021a0e06" target='_blank'>
              CRISPR screening approaches in breast cancer research
              </a>
            </td>
          <td>
            Mark Samuels, Simoni Besta, A. L. Betrán, Reza Shirazi Nia, Xiaohong Xie, Xidong Gu, Qijin Shu, Georgios Giamas
          </td>
          <td>2025-07-12</td>
          <td>Cancer Metastasis Reviews</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background The heterogeneity of colorectal cancer (CRC) and its complex immune microenvironment pose significant challenges for treatment. Understanding the cellular composition and dynamic changes is essential for uncovering mechanisms of tumour progression. Methods To investigate the cellular heterogeneity and immune microenvironment of CRC, identifying critical subpopulations, functional pathways, and prognostic biomarkers, single-cell transcriptomic data from 41 CRC samples across four datasets were integrated. Bioinformatic analyses identified cellular subpopulations, cell communication networks, and prognostic biomarkers. The expression patterns, clinical significance and biological function of TUBB were validated in vitro. Results A distinct epithelial subpopulation with proliferative and invasive features was identified, promoting tumour progression by resisting apoptosis and remodelling the extracellular matrix. ActMono, a terminal state of myeloid cells, was enriched in tumours and linked to disease progression. Cell communication analysis highlighted galectin signalling in immune regulation. A prognostic model (CRS) based on secretory immune cell-related genes identified TUBB as a key molecule influencing the cell cycle and extracellular matrix remodelling, with its expression patterns, clinical significance and biological effects validated in vitro. Conclusion This study reveals critical subpopulations, signalling pathways, and biomarkers in CRC, providing insights into tumour progression and potential therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e4b3d5d75559211564431db384708f82801552" target='_blank'>
              Single-cell sequencing reveals dysregulated cell type perturbations and critical mediator communication remodelling in colorectal cancer
              </a>
            </td>
          <td>
            Chengyuan Xu, Siqi Zhang, Zhouyu Zhang, Luoqiu Zhang, Bin Sun, Zicheng Yu, Hailong Liu
          </td>
          <td>2025-06-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6f92731e4153548f5adf8b4c57b2dcf0276aa2" target='_blank'>
              Single-cell transcriptomics uncovers that cell-to-cell communication controls p53 activation
              </a>
            </td>
          <td>
            L. Castellano, Mark E Robinson, Aleksandra Dabrowska, Niklas Feldhahn, S. Ottaviani, Justin Stebbing
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a09fe0116b674341564f20d065726804460dc81" target='_blank'>
              Identification of hub genes and potential molecular mechanisms of tumor immune microenvironment-related radiotherapy sensitivity in locally advanced cervical cancer.
              </a>
            </td>
          <td>
            Yichen Zan, Qing Liu, Yue Yin, Chongxin Guo, Jing Xu, Junhan Gao, Guanglu Dong
          </td>
          <td>2025-06-30</td>
          <td>Reproductive sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna C Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Proliferating cell nuclear antigen (PCNA) is a central regulator of numerous cellular processes impacting DNA metabolism and genetic integrity, including DNA damage, cell cycle progression, and transcriptional regulation. PCNA aberrations manifest in different disease phenotypes, including neoplastic proliferation, chromatin disassembly, genomic instability, and impaired DNA repair. Although the role of PCNA in control of DNA homeostasis has been extensively studied in some cancers, its pan-cancer relevance in tumorigenesis, immune interactions, and therapeutic resistance remains underexplored. Here, we undertook a comprehensive analysis of publicly available databases to assess the relationship between PCNA expression and the immunological, survival, genetic, functional, and drug sensitivity profiles across multiple cancer types. PCNA mRNA levels were altered in across cancers and associated with altered cancer signaling networks constituting Wnt, Hippo, and mTOR pathways. Increased transcript levels were associated with poor overall survival in some cancers, including ACC, CESC, LGG, and THYM. Amplification was the predominant genetic alteration of PCNA in multiple tumors. In several tumors, upregulation of PCNA was linked with differences in tumor-infiltrating lymphocytes, and specific immune-inhibitors, and chemokines. Moreover, elevated PCNA expression was linked with increased sensitivity to several drugs, particularly to Navitoclax, NPK76-II-72-1, and Ciclopirox across cancers. Our study orients the first comprehensive pan-cancer analysis of PCNA, uncovering its prognostic significance and altered expression across various cancers through multiomics data. Unlike previous studies, tumor-specific genetic alterations, such as amplification and hypomethylation, and the paradoxical immune microenvironment linked to PCNA were explored, suggesting potential immune evasion mechanisms. Additionally, new therapeutic avenues reveal PCNA’s relationship with drug sensitivity to agents like Navitoclax and Ciclopirox, providing invaluable insights for pharmacological interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3571f2ad7f72f9a390320cd46deb0b77021902" target='_blank'>
              PCNA in Pan-Cancer: A Prognostic Biomarker Unveiled Through a Data-Driven, Multidimensional Analysis of Transcriptomics, Immunity, and Functional Profiling
              </a>
            </td>
          <td>
            Depanshi Pandit, Ravindranath Bilachi Sanganabasappa, Amardeep Dhillon, Sanjiban Chakrabarty
          </td>
          <td>2025-06-26</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ea5280910649de8b63e584d7975f22a13b00b4" target='_blank'>
              Characterization of Somatic Mutations in Human tRNA Genes Reveals Tumor-Specific Mutational Loads, and the Generation of tRNA Variants that Alter the Genetic Code
              </a>
            </td>
          <td>
            Marina Murillo-Recio, Marina Salvadores, Adrián Gabriel Torres, Lina Tsapanou, Aina Vaquer Picó, F. Supek, L. R. de Pouplana
          </td>
          <td>2025-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c29c85da72e04b5001274eb1b2f06a493d988d5" target='_blank'>
              Reprogramming of Cellular Plasticity via ETS and MYC Core-regulatory Circuits During Response to MAPK Inhibition in BRAF-mutant Colorectal Cancer
              </a>
            </td>
          <td>
            H. Lee, Zhao Zheng, Alexey Sorokin, Chi Wut Wong, Stefania Napolitano, S. Chowdhury, Preeti Kanikarla, A. Singh, Veena Kochat, Christopher Bristow, Sanjana Srinivasan, Mike Peoples, Emre Arslan, J. Alshenaifi, O. Villarreal, V. Morris, J. P. Shen, F. Meric-Bernstam, Abhinav K. Jain, Natalie W Fowlkes, Amanda Anderson, D. Menter, A. Saw, Kunal Rai, S. Kopetz
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Leukemia, a group of malignant blood disorders, arises from the uncontrolled proliferation of abnormal white blood cells. Genetic mutations play a critical role in the initiation and progression of leukemia. This review aims to provide an overview of the genetic landscape of leukemia, focusing on the most common genetic alterations and their clinical implications. A literature search was conducted to identify relevant studies on genetic mutations in leukemia. The identified studies were critically appraised to assess their methodological quality. The present review highlights the key genetic alterations associated with different types of leukemia, including chromosomal translocations, point mutations, and gene copy number variations. These genetic abnormalities can impact disease prognosis, treatment response, and overall patient survival. A comprehensive understanding of the genetic basis of leukemia is essential for accurate diagnosis, prognostication, and the development of targeted therapies. Future research should focus on identifying novel genetic markers, elucidating the underlying mechanisms of leukemogenesis, and developing innovative therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e966f7c7f21cf8c9b127f6afb40f6c702d2c3" target='_blank'>
              A Review Article: Evaluation of the Frequency of Genetic Mutations in Leukemia.
              </a>
            </td>
          <td>
            A. Hamadi
          </td>
          <td>2025-06-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy is defined as the loss or gain of a whole chromosome or its region. Even at early stages of development, it usually leads to fatal consequences, including developmental defects/abnormalities and death. For a long time, it was believed that the disruption of gene balance results in pronounced effects at both the cellular and organismal levels, adversely affecting organism formation. It has been shown that the gene imbalance resulting from aneuploidy leads to proteotoxic and metabolic stress within the cell, reduced cell proliferation, genomic instability, oxidative stress, etc. However, some organisms have exhibited tolerance to aneuploidies, which may even confer adaptive advantages, such as antibiotic resistance in pathogenic fungal strains. A significant factor likely lies in the complexity of the tissue and organ organization of specific species. Polyploid organisms are generally more tolerant of aneuploidy, particularly those that have recently undergone whole-genome duplication. This review places special emphasis on the examination of sex chromosome aneuploidies in humans. In addition to primary effects, or cis effects (changes in the quantity of the transcripts of genes located on the aneuploid chromosome), aneuploidy can induce secondary or trans effects (changes in the expression levels of genes located on other chromosomes). The results of recent studies have prompted a reevaluation of the impact of aneuploidy on the structural-functional organization of the genome, transcriptome, and proteome of both the cell and the entire organism. Despite the fact that, in the cases of aneuploidy, the expression levels for most genes correlate with their altered copy numbers in the cell, there have been instances of dosage compensation, where the transcript levels of genes located on the aneuploid chromosome remained unchanged. The review presents findings from recent studies focused on compensatory mechanisms of dosage compensation that modify gene product quantities at post-transcriptional and post-translational levels, alleviating the negative effects of aneuploidy on cellular homeostasis. It also discusses the influence of extrachromosomal elements on the spatial organization of the genome and the changes in gene expression patterns resulting from their presence. Additionally, the review specifically examines cases of segmental aneuploidy and changes in copy number variants (CNVs) in the genome. Not only the implications of their composition are considered, but also their localization within the chromosome and in various compartments of the interphase nucleus. Addressing these questions could significantly contribute to enhancing cytogenomic diagnostics and establishing a necessary database for accurate interpretation of identified cases of segmental aneuploidy and CNVs in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271a442ac782b355db581186dfcccbd9f3d50dd8" target='_blank'>
              From cytogenetics to proteogenomics: new horizons in the study of aneuploidies
              </a>
            </td>
          <td>
            K. Zadesenets, N. B. Rubtsov
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Guanine-rich DNA sequences have a propensity to form G-quadruplex structures. These structures play several important biological roles and are potential targets for anticancer drugs. However, no G-quadruplex-stabilizing agent has yet been approved for clinical use. Given that G-quadruplex stabilization is quite promising as a mechanism for novel anticancer therapies, it is crucial to elucidate its effects on healthy human cells. In our study, we modeled a potential human treatment using G4 -stabilizing agents and analyzed their effects on genome integrity, transcriptomic changes, and mitochondrial function focusing on non-cancerous cells to predict potential side effects of such treatments. We found that G-quadruplex stabilization does not compromise genome integrity. However, it can induce persistent alterations in the transcriptomic profile of human cells, including genes encoded on the mitochondrial genome. Notably, certain G-quadruplex-stabilizing agents triggered mitophagy in both human cells and Drosophila melanogaster. In summary, our findings indicate that while G-quadruplex stabilization does not cause genome instability, it may pose potential risks due to its long-term effects on transcription and its ability to induce mitophagy. Therefore, we recommend that all potential drug candidates be thoroughly evaluated for their ability to induce mitophagy and to promote cancer formation in animal models prior to clinical trials. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-04791-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277b0abab64e6f6a40e0606dd3b857caf1865195" target='_blank'>
              The effect of prolonged G-quadruplex stabilization on the functions of human cells
              </a>
            </td>
          <td>
            Nargis Karatayeva, Lili Hegedűs, Arindam Bhattacharjee, Eszter Németh, Ádám Póti, Lorinc Pongor, Gábor Juhász, David Szuts, Peter Burkovics
          </td>
          <td>2025-06-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer progression remains a significant clinical challenge. Phenotypic adaptation by tumour cells results in disease heterogeneity, which drives treatment resistance and immune escape. T-cell immunotherapy, while effective at treating some cancer subtypes, can also fail due to limits on tumour immunogenicity or T-cell recognition. For example, one potential contributor to immune escape involves the density and alignment of the extracellular matrix (ECM) surrounding tumours, also known as tumour-associated collagen signature (TACS). However, the specific mechanisms by which aligned fibres contribute to decreased patient survival rates have not yet been decoupled. Here, we developed EVO-ACT (EVOlutionary agent-based cancer T-cell interaction), a two-dimensional agent-based modelling framework designed to investigate how different TACS architectures impact tumour evolution and dynamic interactions with CD8[Formula: see text] T cells. Our results highlight that TACS-driven modulation of T-cell dynamics, combined with phenotypic adaptation, such as epithelial-to-mesenchymal transition, underlies differences in tumour immunogenicity and the application of our model can successfully recapitulate clinically observed breast cancer survival trends.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195f55d30b50b8b9b55b6c384b5267c2b8395881" target='_blank'>
              Distinct evolutionary patterns of tumour-immune escape and elimination determined by extracellular matrix architectures.
              </a>
            </td>
          <td>
            Yijia Fan, Jason T. George
          </td>
          <td>2025-07-01</td>
          <td>Journal of the Royal Society, Interface</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Gastric cancer remains a leading cause of cancer-related morbidity and mortality worldwide. The genetic factors contributing to gastric cancer progression and resistance to therapies, particularly radiotherapy, are not fully understood. TP53, ATM, RAD51, and BAX are genes involved in DNA repair, apoptosis, and response to stress. The aim of this study was to investigate the expression patterns of these genes in gastric cancer, their potential role in radiotherapy resistance, and their diagnostic value. Methodology Gene expression levels of TP53, ATM, RAD51, and BAX were assessed using RT-qPCR across 9 gastric cancer cell lines and 6 normal control cell lines. Additionally, protein expression was confirmed via IHC and TCGA dataset analysis. Methylation levels of these genes were evaluated in gastric cancer tissues using the GSCA database. Mutational analysis was conducted using cBioPortal, and survival analysis was performed using Kaplan-Meier and meta-analysis. The radiotherapy resistance study was carried out by knocking down TP53, RAD51, and BAX in AGS and MKN-45 gastric cancer cell lines, followed by expression analysis, colony formation, and wound healing assays. Results The expression of TP53, RAD51, and BAX was significantly upregulated, while ATM was downregulated in gastric cancer cell lines compared to normal controls. All four genes demonstrated good discriminatory power (AUC = 1) in distinguishing gastric cancer from normal samples. Methylation analysis revealed significant hypomethylation of TP53, RAD51, and BAX, and hypermethylation of ATM in gastric cancer tissues. Mutational analysis showed that TP53 was altered in 88% of gastric cancer samples, while ATM, RAD51, and BAX exhibited lower mutation rates. Survival analysis suggested that elevated expression of TP53, RAD51, and BAX may be linked to poorer survival outcomes, while reduced ATM expression appeared to associate with decreased overall survival. However, these associations require further validation through additional studies. Knockdown of TP53, RAD51, and BAX in AGS and MKN-45 cells resulted in significantly reduced cell proliferation and slower wound healing, highlighting their role in radiotherapy resistance. Conclusion The TP53, RAD51, and BAX genes are significantly involved in gastric cancer progression and resistance to radiotherapy. Their expression and mutation status provide valuable diagnostic and prognostic information. Supplementary Information The online version contains supplementary material available at 10.1186/s41065-025-00496-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8363e00ef3c38472bf63112f24092628b9ff0683" target='_blank'>
              In silico and in vitro assessment of TP53, ATM, RAD51, and BAX genes in gastric cancer and their contribution to radiotherapy resistance
              </a>
            </td>
          <td>
            Junwei Zhang, Pengtao He
          </td>
          <td>2025-07-12</td>
          <td>Hereditas</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background The human genome contains approximately 3,200 near full-length autonomous human endogenous retroviral (HERV) genomes distributed across the 23 chromosomes. These autonomous HERV proviral genomes include long terminal repeats (LTRs) capable of promoting RNA transcription. In quiescent cells, most HERV loci remain transcriptionally silent. However, environmental changes, such as epigenetic remodeling of chromatin, can activate these silenced loci. Methods To study HERV reactivation, we previously analyzed autonomous HERV expression patterns in monocytes isolated from peripheral blood mononuclear cells (PBMCs) identified in single-cell RNA sequencing (scRNA-seq) databases using the Azimuth application. We developed a Window-based HERV Alignment (WHA) method, which analyzes aligned DNA sequences using sequential, non-overlapping windows of defined lengths. Samples were scored as positive (>= 9 good/usable windows) or negative (<= 8 good/usable windows). Results Using WHA, we established a control set from 31 normal individuals, with fewer than 8 windows at selected HERV loci. We analyzed scRNA-seq data from three studies of hospitalized COVID-19 patients and found distinct HERV expression patterns in monocytes. Unique patterns were also found in patients with influenza, Dengue virus, or sepsis. We next examined HERV expression at early (<7 days) and late (>14 days) timepoints post COVID-19 recovery and detected HERV loci in both groups. Analyzing 12 patients with post-acute sequelae of COVID-19 (PASC), we identified three HERV loci expressed in all patients. Some loci showed amplified numbers of good/usable windows, indicating longer transcripts and greater sequence depth. The most amplified locus was located within an intron of JAKMIP2, which, along with neighboring host genes, also showed increased transcription. Conclusion Previous studies have shown that viral infections, including COVID-19, influenza, and Dengue virus, as well as sepsis, can induce innate immune memory in monocytes through epigenetic remodeling of hematopoietic stem and myeloid precursor cells. The identification of co-amplified HERV loci and neighboring host gene transcripts in monocytes from PASC patients suggests expansion of epigenetically remodeled myeloid progenitors. The identification of these HERV-host gene patterns provides a foundation needed to understand the clinical features of patients with PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890c566bb6e206114c5472e39818cdcd77102fc" target='_blank'>
              Amplification of select autonomous HERV loci and surrounding host gene transcription in monocytes from patients with post-acute sequelae of COVID-19
              </a>
            </td>
          <td>
            Hyunmin Koo, Casey D. Morrow
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78884b796044723309cb646cb50e973d65185744" target='_blank'>
              A proteogenomic approach to identifying a gene signature associated with HOXB13 G84E carrier status in prostate cancer tumours
              </a>
            </td>
          <td>
            K. Raspin, Z. Noor, A. T. Aref, J. Marthick, C.-h. Jung, S. Donovan, S. Williams, P. Hains, P. J. Robinson, R. L. Balleine, Q. Zhong, R. Reddel, L. M. FitzGerald, J. L. Dickinson
          </td>
          <td>2025-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Somatic cell reprogramming into human induced pluripotent stem cells entails significant intracellular changes, including modifications in mitochondrial metabolism and a decrease in mitochondrial DNA copy number. However, the mechanisms underlying this decrease in mitochondrial DNA copy number during reprogramming remain unclear. Here we aimed to elucidate these underlying mechanisms. Through a meta-analysis of several RNA sequencing datasets, we identified genes responsible for the decrease in mitochondrial DNA. We investigated the functions of these identified genes and assessed their regulatory mechanisms. In particular, the expression of the thymidine kinase 2 gene (TK2), located in the mitochondria and required for mitochondrial DNA synthesis, is decreased in human pluripotent stem cells as compared with its expression in somatic cells. TK2 was significantly downregulated during reprogramming and markedly upregulated during differentiation. Collectively, this decrease in TK2 levels induces a decrease in mitochondrial DNA copy number and contributes to shaping the metabolic characteristics of human pluripotent stem cells. However, contrary to our expectations, treatment with a TK2 inhibitor impaired somatic cell reprogramming. These results suggest that decreased TK2 expression may result from metabolic conversion during somatic cell reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afec2f521a0fb9cff12157659a7e8ef543de136a" target='_blank'>
              Changes in mitochondrial thymidine metabolism and mtDNA copy number during induced pluripotency
              </a>
            </td>
          <td>
            Hyun Kyu Kim, Yena Song, Minji Kye, Byeongho Yu, Hyungkyu Choi, Sung-Hwan Moon, M. Lee
          </td>
          <td>2025-06-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21690ca2e89ea2f16a09c71f18f6490b7208397c" target='_blank'>
              High Throughput Screening Identifies Small Molecules that Synergize with MRTX1133 Against Acquired Resistant KRAS G12D Mutated CRC
              </a>
            </td>
          <td>
            Natalie Thielen, Ning Wei, Emiko Nagai, Edward Chu, S. Kitamura, Chaoyuan Kuang
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 4116


 Background:
 Biliary tract cancers (BTC) are rare, highly aggressive malignancies with limited treatment options, leading to consistently poor outcomes. An improved understanding of BTC tumor evolution could inform enhanced screening strategies, identify useful prognostic markers, and discover novel therapeutic targets.
 Methods:
 We performed whole genome and transcriptome sequencing (WGTS) at high depth ( > 80X) on a prospective cohort of BTC tumors. After detecting mutations and mutational signatures, we developed two novel methods for driver identification and evolutionary reconstruction. First, we created new oncogene and tumor suppressor-specific models that integrate copy number profiles, structural variant breakends, and expression changes to distinguish structural drivers from neutral chromosomal rearrangements. Second, we applied population genetics techniques to fit demographic models to tumor allele frequencies across 20 paired primary and metastatic samples.
 Results:
 We analyzed 130 tumor samples from 110 patients, representing the largest BTC whole-genome cohort to date. We identified hypermutated tumors (50-150 mutations/mb) with distinct etiologies, including mismatch repair deficiency, platinum exposure, tobacco use, and aristolochic acid-related damage, the latter of which was associated with response to immunotherapy. Ourintegrated-driver approach identified an association between selection on
 RAD23A
 and an increased structural variant load, resulting in a tandem-duplicator-like mutational phenotype. This also revealed an underappreciated impact of
 SMAD4
 in BTC, which is inactivated through multiple mutation types in 12%. Notably,
 BAP1
 mutations occurred in 24% of cases, including 4% that were inactivated through deletion of the
 BAP1
 promoter that lowered transcript expression, a previously undescribed mechanism. By pairing clinical data to the genomics, we observed a co-occurrence of
 BAP1
 mutations and
 FGFR2
 fusions in small-duct, mass-forming intrahepatic cholangiocarcinomas. These mutations were mutually exclusive with
 TP53
 mutations, which were enriched in patients with primary sclerosing cholangitis. Finally, our novel method for subclonal population reconstruction on paired samples illuminated recent tumor evolutionary dynamics and identified an
 ARID1B
 fusion with a potential role in metastasis.
 Conclusions:
 This study uncovered novel genomic mechanisms underlying the evolutionary origins of BTC beyond those previously identified with exome and panel sequencing, highlighting the value of WGTS. These results highlight the complex genomic heterogeneity of BTC, with potential implications for precision therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bbdd5f85cf47a722dc40e8dbc7f432d73cb4b4" target='_blank'>
              Whole-genome sequencing of biliary tract cancer: Uncovering the genomic origins of evolutionary trajectories.
              </a>
            </td>
          <td>
            Felix Beaudry, Nicholas Light, Duhan Yendi, A. Zorigtbaatar, Maggie Hildebrand, A. Dodd, Roxana Bucur, Julie M. Wilson, Enrique Sanz Garcia, Xin Wang, Erica S. Tsang, G. O’Kane, S. Gallinger, Arndt Vogel, Jennifer J Knox, F. Notta, G. Sapisochin, Robert C Grant
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="As a predominant oncogenic driver in non-small cell lung cancer (NSCLC), EGFR frequently undergoes amplification or mutation, with EGFR-tyrosine kinase inhibitors (EGFR-TKIs) like gefitinib and erlotinib constituting frontline therapy for advanced EGFR-mutant cases. However, both primary and acquired resistance significantly limit clinical efficacy. Here, we revealed that glutathione metabolic pathway controlled by glutathione peroxidase GPX2 was abnormally activated in gefitinib-resistant A549 and HCC827-GR cell lines. Mechanistically, GPX2 triggers Hedgehog signaling activation through releasing GLI transcriptional regulator, promoting cancer stem cell (CSC) characteristics and TKI resistance. Notably, N6-methyladenosine (m6A) modification on GPX2 mRNA mediated by METTL14 diminished its stability. In vivo, GPX2 deletion constrained glutathione metabolism and boosted the effectiveness of TKI in cell line-derived xenograft models. Collectively, these findings demonstrate that GPX2 serves as a positive regulator of both primary and acquired EGFR-TKI resistance and could be a promising therapeutic target for precise treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50746db76f6a5070ba253127ed90d322b08f1eb2" target='_blank'>
              N6-methyladenosine-modified GPX2 impacts cancer cell stemness and TKI resistance through regulating of redox metabolism
              </a>
            </td>
          <td>
            Xu Yang, Long Yu, Miaomiao Shao, Huiling Yang, Kangwei Qi, Gaofei He, Lanxin Wang, Di Kong, Jianxin Gu, Xiaolin Xu, Lanxin Wang
          </td>
          <td>2025-06-18</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/196ee85ccabe285a00ef4209f93a8dd533568146" target='_blank'>
              Identifying rare spontaneous mutations through wildtype E. coli population sequencing
              </a>
            </td>
          <td>
            R. Green, M. Bawn, A. Angus-Whiteoak, M. Jago, FJ Whelan, M. Lagator, N. Hall, CG Knight, R. Krašovec
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The circadian clock governs daily rhythms in numerous physiological processes through precise regulation of gene expression and biochemical functions. Dysregulation of the circadian rhythm has been implicated in carcinogenesis and cancer progression. However, the mechanisms by which the circadian clock influences cancer phenotype and chemotherapy resistance, particularly in cholangiocarcinoma (CCA), remain poorly understood. Using cell lines established from primary CCA and metastatic ascites of two male patients, we manipulated core clock genes (BMAL1, PER2, and NR1D1) to evaluate their effects on circadian rhythms. We analyzed alterations in circadian phenotypes at dynamic and single time points and assessed their impact on cancer-related phenotypic changes, including proliferation, apoptosis, cell cycle regulation, migration, invasion, and the expression of epithelial-to-mesenchymal transition (EMT) and cancer stem cell markers. Additionally, we examined the impact of circadian disruption on gemcitabine sensitivity. Genetic deletion of BMAL1, PER2, and NR1D1 disrupted circadian rhythm and significantly altered cancer phenotypes. Notably, BMAL1 and NR1D1 impairment exacerbated cell migration, invasion, and EMT activation in CCA cells. BMAL1 loss also induced gemcitabine resistance. In contrast, PER2 repression enhanced chemosensitivity and inhibited metastasis. The modulation of the circadian gene triggered phenotypic changes in CCA cells, indicating a crucial involvement of core-clock components in the pathological mechanisms hastening bile duct cancer malignancy. Our findings advance the understanding of regulating CCA malignancy and may offer a novel target for its treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63df1b948313a08b9d2b74bdfc742ccab84a19a9" target='_blank'>
              Dysfunction of core clock genes regulates malignant phenotype and gemcitabine sensitivity of cholangiocarcinoma cells.
              </a>
            </td>
          <td>
            Yin Li, Aimin Zheng, Yangang Cui, Tianbao Liu
          </td>
          <td>2025-06-17</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) accounts for twelve percent of all breast cancer cases, with a survival rate around ten percent lower than ER+/PR+ positive breast cancers. There are limited therapeutic options as these tumors do not respond to hormonal therapy or HER2-targeted treatments. We hypothesized that new insights into pathogenic mechanisms in TNBC can be obtained from studying epigenetic alterations through Hi-C (genome-wide chromosome conformation capture) data analysis.We developed a computational strategy that captured key properties of chromatin conformation while incorporating statistical measures of interaction significance. This model addresses limitations in Hi-C data analysis without relying on predefined features like TADs and compartments. We applied this model to Hi-C and RNA-seq data from TNBC patients, representing the data as multilayer networks to identify genome-wide properties of the TNBC 3D genome.Our network-based analysis revealed distinct chromatin interaction patterns in TNBC compared to healthy contralateral controls. Hi-C data can distinguish interaction patterns related to diseased phenotypes or interaction patterns with potential to exert regulatory effects instead of incidental contacts, but some apparently random interactions may also support important genome regulatory activities.Our findings demonstrate that network-based Hi-C analysis can capture the genome-wide complexity of chromatin interactions in TNBC. This integrative approach provides new insights into the epigenetic mechanisms underlying TNBC pathogenesis and contributes to the advancement of analysis methods for future investigations into novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2cde03b116eaa9a9ec374af95ff1672e86aa8b" target='_blank'>
              Integration of chromosome conformation and gene expression networks reveals regulatory mechanisms in triple negative breast cancer
              </a>
            </td>
          <td>
            Helena Reyes-Gopar, Keila Adonai Pérez-Fuentes, M. Bendall, Enrique Hernández-Lemus
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Hypoxia, characterized by a reduction in tissue oxygen levels, is a hallmark of many solid tumors and affects a range of cellular processes, including DNA repair. In low-oxygen conditions, cancer cells often suppress key DNA repair pathways such as homologous recombination (HR), leading to the accumulation of DNA damage and increased genomic instability. These changes not only drive tumor progression but also contribute to resistance against conventional therapies. Hypoxia significantly reduces the effectiveness of oxygen-dependent treatments, including radiotherapy and many chemotherapeutic agents. To address this limitation, bioreductive drugs have been developed that become selectively activated in hypoxic environments, providing targeted cytotoxic effects within oxygen-deprived tumor regions. Additionally, the rapid growth of tumors often results in disorganized and inefficient vasculature, further impairing the delivery of oxygen and therapeutic agents. This review explores the molecular mechanisms by which hypoxia disrupts DNA repair and contributes to treatment resistance. It also presents emerging therapeutic strategies aimed at targeting the hypoxic tumor microenvironment to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34398b4aa06a09890c501084c80149cba79331e4" target='_blank'>
              Influence of Hypoxia on Tumor Heterogeneity, DNA Repair, and Cancer Therapy: From Molecular Insights to Therapeutic Strategies.
              </a>
            </td>
          <td>
            Dominika Kunachowicz, Paulina Tomecka, Mikołaj Sędzik, Jarosław Kalinin, Jacek Kuźnicki, Nina Rembiałkowska
          </td>
          <td>2025-07-10</td>
          <td>Cells</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed7953efc8b662a0f9dd292cd306d987c8eb0ba8" target='_blank'>
              Breast Cancer Clustering Integrating Complete Gene Expression Profiles and Genetic Ancestry
              </a>
            </td>
          <td>
            Johanna Stepanian, Alejandro Mejía-Garcia, Carlos A. Orozco, Jorge Duitama
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c58caf03a35382e82dc43ac8f2167d8ee1ee67" target='_blank'>
              X-chromosome upregulation operates on a gene-by-gene basis at RNA and protein levels
              </a>
            </td>
          <td>
            Ryan N. Allsop, Jeffrey Boeren, Beatrice F. Tan, Sarra Merzouk, Suresh Poovanthingal, W. V. van Ijcken, Jeroen Demmers, Hegias Mira Bontenbal, C. Gontan, J. Gribnau, V. Pasque
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Essential transcription and cell cycle progression controllers are CDKs, whose dysregulation is a defining trait of many human cancers. CDKs have grown to be very crucial therapeutic targets in cancer. Although traditional CDK inhibitors have demonstrated therapeutic efficacy, they frequently encounter limitations due to resistance mechanisms and off-target effects. Recent developments in targeted protein degradation, like proteolysis-targeting chimeras (PROTACs) and molecular glues, offer creative ways to destroy CDK proteins specifically. These techniques reduce scaffolding activities and slow down kinase activity, hence more completely blocking oncogenic CDK signaling. This paper highlights the clinical and preclinical developments of PROTACs and molecular glues, investigates the current CDK-targeting therapeutic landscape, and studies the molecular basis of CDK dysregulation in cancer. We also address their benefits over conventional inhibitors, current issues, and possibilities for inclusion into precision oncology. These new approaches taken together represent a change in CDK-targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1262893392840549bf93c0b9f7515437c79c1d7" target='_blank'>
              Targeting CDKs in cancer therapy: advances in PROTACs and molecular glues
              </a>
            </td>
          <td>
            Hany E. Marei, Khaled Bedair, Anwarul Hasan, Layla Al-mansoori, Alice Gaiba, Andrea Morrione, C. Cenciarelli, Antonio Giordano
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 e18058


 Background:
 HNCs are a diverse group of malignancies characterized by genetic, epigenetic, and proteomic alterations that drive tumor progression, treatment resistance, and poor outcomes.
 Methods:
 Molecular profiling of 273 HNC samples was conducted using a semiconductor-based next-generation sequencing (NGS) platform at Datar Cancer Genetics. A subset of 106 cases underwent targeted transcriptomic analysis of 20,802 genes. Additionally, PD-L1 (22C3) IHC, microsatellite instability (MSI), and tumor mutational burden (TMB) were analyzed in selected samples.
 Results:
 Transcriptomic analysis highlighted significant alterations in critical cellular pathways, including cell cycle control, apoptosis, DNA repair mechanisms, and transcription regulation. Among the most dysregulated genes were
 DLGAP5
 ,
 CASP14
 ,
 BCL2
 , and
 JUN
 . Copy number profiling in a subset (N=74) revealed significant chromosomal instability, with 34% of cases showing losses on specific chromosome arms and 9% exhibiting gains, notably in regions such as
 4p, 19p
 and
 1p.
 Mutations were predominantly observed in tumor suppressor genes, with a lower frequency in oncogenes. Frequently altered genes include
 TP53
 (63%),
 CDKN2A
 (19%),
 PIK3CA
 (12%),
 TERT
 (15%),
 HRAS
 (6%) and
 EGFR
 (1%).
 HRAS
 mutations offer potential for targeted therapy with tipifarnib, a farnesyltransferase inhibitor.
 EGFR
 alterations, though rare, have proven challenging to effectively target with
 EGFR
 -specific therapies such as cetuximab or tyrosine kinase inhibitors, underscoring the critical need for innovative strategies to exploit this target. Amplifications were observed in
 MYC
 (9%),
 CCND1
 (8%),
 FGF19
 (6%),
 FGF3
 (4%) and
 FGF4
 (4%). Targetable amplifications were seen in
 EGFR
 (6%),
 ERBB2
 (2%) and
 MET
 (1%), offering potential opportunities for personalized therapies.
 MET
 amplifications are also linked to aggressive tumor behavior. Fusion was a relatively less common event, observed in 9.5% cases, including one case each of targetable
 BRAF-MRPS33
 and
 FGFR1-PLAG1
 fusion. High TMB (≥10 muts/mb) was seen in 21% cases (Median 12, range 0-28). None of the tumors showed MSI-high status (n=63). Higher PD-L1 positivity was observed in TMB high samples (76% vs 59%), with PD-L1 22-C3 CPS ≥1 in 66% (50/ 76) and PD-L1 28-8 TPS ≥1 in 46% (36/ 78).
 Conclusions:
 This study underscores the genetic heterogeneity of HNCs, pointing to deregulated pathways involved in cell cycle control, apoptosis, and DNA repair as promising targets for future therapeutic development and personalized treatment strategies. Also this study points towards a subset of patients who may derive a greater benefit with immunotherapy, based on TMB/PDL1.


 ESCAT TIER distribution of the molecular alterations detected in head and neck cohort (N=273).




 TIERs
 Total Cases
 % occurrence




 IC
 6
 2%


 IIA
 21
 8%


 IIIA
 85
 31%


 IIIA
 1
 0%


 IIIB
 57
 21%


 IVA
 180
 66%


 IVB
 71
 26%


 X
 142
 52%




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204052c6edd84c3f9c00c6d7bada097fe56677e" target='_blank'>
              Molecular profiling of head and neck cancers (HNCs): Genetic heterogeneity, immune landscape, and therapeutic targets.
              </a>
            </td>
          <td>
            R. K. Choudhary, S. Limaye, Padman Vamadevan, Vivek Agarwala, Vikas Goswami, H. P, Vijay Anand Reddy, Niyati Krunal Shah, Darshit Kalpeshkumar Shah, Shivam Shingla, Dhanashri Ahire, Vinayak Rao, R. Gosavi, Priyanka Desale, R. Datar, D. Patil
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neoantigen-specific T cells play a major role in cancer immunotherapy. Mutated proteins derived from non-synonymous mutations in tumor genomic DNA are the major source of neoantigens. It is conceivable that non-synonymous mutations found in tumor mitochondrial DNA (mtDNA) can also generate neoantigens that can be recognized by T cell receptors (TCRs). However, no tumor mitochondrial mutation-specific TCRs have been reported. As a proof-of-concept study, we obtained 3,798 non-synonymous single-nucleotide mutations across 38 tumor types through The Cancer Mitochondria Atlas project. These mutations were subjected to an epitope prediction algorithm to predict binding affinities for HLA-A*0201. The top 300 candidate peptides were synthesized and screened against 10 healthy donors with homozygous HLA-A*0201 alleles. Mutation-reactive T cells were subjected to single-cell sequencing to identify TCR sequences, followed by validations. Here, four MT-ND2 (A103T) mutation-specific TCRs were isolated from a donor. These TCRs interacted with HLA-A*0201-restricted IMMAMTMKL peptide. Additionally, through a single-cell sequencing approach, we demonstrated that a non-synonymous mutation, MT-CO1 (V274I), could be detected in nearly all tumor cells in a colorectal tumor specimen, whereas other mutations in 2 out of 4 tumors were detected in subclonal populations. This study suggests that isolating tumor mitochondrial mutation-specific TCRs is possible, but some biological barriers need to be considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1e1b3cf669d6e64ff59e594e8300039fb1835d" target='_blank'>
              Isolation of mitochondrial mutation-specific T cell receptors.
              </a>
            </td>
          <td>
            Catherine Kirkpatrick, Shanshan Hao, Charles M Quick, Steven R Post, Yuet-Kin Leung, L. Burdine, Yong-Chen William Lu
          </td>
          <td>2025-06-30</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Castration-resistant prostate cancer (CRPC) remains the leading cause of mortality among prostate cancer patients. While second-generation androgen receptor (AR) pathway-targeted therapies, such as Abiraterone, have significantly improved survival outcomes, resistance to these treatments ultimately emerges, posing a critical challenge. Understanding the mechanisms underlying Abiraterone resistance is essential for developing strategies to enhance patient outcomes. In this study, a super-enhancer (SE)-driven transcriptional regulatory circuit is identified involving BCL6, NFIB, and SMAD3 that facilitates Abiraterone resistance in CRPC. Through comprehensive analyses of SE expression profiles in Abiraterone-resistant CRPC cells and their parental counterparts, it is revealed that this circuit plays a pivotal role in resistance progression. Mechanistically, BCL6, NFIB, and SMAD3 synergistically remodel the transcriptional landscape of resistant CRPC cells, driving resistance by regulating cholesterol biosynthesis and cell cycle pathways. The findings provide critical insights into the transcriptional dysregulation underlying Abiraterone resistance and highlight potential therapeutic strategies to counteract treatment resistance in CRPC, ultimately aiming to improve patient survival and quality of life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5333f40f3a2272e85aae404e2fe1d3f4d247124b" target='_blank'>
              A Super-Enhancer-Driven Transcriptional Regulatory Circuit Underlying Abiraterone Resistance in Castration-Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Liling Jiang, Jiamin Wang, Guanjie Peng, Haichuan Zhang, Jinxin Fang, Yingyin Gao, Enzhe Lou, Yangzhou Liu, Wa Ding, Bingyuan Liu, Qiong Mao, Lizhen Jiang, Aochu Liu, Xinyue Li, Shiwen Hu, Qiaomin Ma, Yueyuan Zheng, Zhigang Zhao, Xianping Shi
          </td>
          <td>2025-06-05</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Triple negative breast cancer (TNBC) remains one of the most aggressive and therapeutically challenging subtypes of BC due to its lack of targeted treatment options and high plasticity. Increasing evidence suggests that epigenetic mechanisms, including DNA methylation, histone modifications, and non-coding RNA regulation, play a pivotal role in facilitating TNBC phenotype by influencing the expression of genes involved in subtype reprogramming. Modulating these epigenetic marks offers a promising strategy to drive TNBC subtype conversion toward less aggressive forms that are more responsive to treatment, thereby enhancing therapeutic efficacy. By targeting enzymes such as DNA methyltransferases and histone modifiers using epigenetic drugs (epidrugs), TNBC cells can be re-sensitized to hormone therapy, chemotherapy, and targeted treatments. This review delves into the role of epigenetic modulation in harnessing the plasticity of TNBC to drive its conversion into subtypes with better prognoses and discusses problems and limitations associated with the use of epidrugs in this context. By promoting subtype conversion, epidrugs offer a promising strategy for providing more personalized and effective therapies for TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ada35a90892ec0be34375adfcf309b46149ff70" target='_blank'>
              Epigenetic Therapeutics: Reprogramming Triple-Negative Breast Cancer into Responsive Subtypes.
              </a>
            </td>
          <td>
            Wafaa S. Ramadan, Aya Mudhafar Al-Azawi, L. Lozon, Rawan R. Kawaf, Ragheb Alsheikh Zein, Yahia El-Gharib, Raafat El-Awady
          </td>
          <td>2025-07-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Sex differences in breast cancer (BC) lead to distinct clinical and molecular characteristics. Sex-associated heterogeneity relies intricately on the oncogenic properties of cancer cells and multicellular interactions in tumor microenvironments. Here we conducted a multiomic analysis encompassing single-cell RNA sequencing, spatial transcriptomics and large-scale histological analysis, combined with genetic and pharmacological perturbations to systematically develop a high-resolution and spatially resolved map of intratumoral expression heterogeneity in male breast cancer (MBC). Among the malignant cells, we identify 9 consensus meta-programs (MPs, for example, estrogen response and interferon response), each consisting of dozens of genes that are coordinately upregulated in subpopulations of cells within MBC. Of these, MP5 exhibits significant male bias, characterized by aberrant activation of the estrogen receptor α (ERα) signaling pathway, inducing endocrine therapy resistance in MBC. Bidirectional interactions between MP5 malignant cells and immunosuppressive cancer-associated fibroblasts form an immune exclusionary microenvironment mediated by LRBA. Collectively, our study highlights the endocrine therapy resistant and immunosuppressive role of LRBA in MBC, and these findings may provide insights into the molecular features of MBC and facilitate the development of sex-specifical treatment strategies.
 Citation Format: Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, Jingyan Xue, Jiong Wu. Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3df96734a9b8fda8be55365f44c01373b9990e46" target='_blank'>
              Abstract P3-02-26: Sex-biased cancer cells induce endocrine therapy resistance and immune suppression in male breast cancer
              </a>
            </td>
          <td>
            Zhishuang Gao, Zehao Wang, Yue Zhou, Xiaoting Chen, J. Xue, Jiong Wu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a clonal malignancy of myeloid progenitor cells that demonstrates highly variable responses to current regimens, highlighting the need for precision medicine. However, reliable biomarkers for precision medicine treatment remain elusive due to cellular heterogeneity. Conventional Models based on bulk RNA sequencing and ex vivo assays often fail to capture the intricate molecular pathways and gene networks that underlie treatment response and resistance. Here, NetAML, a novel network-based precision medicine platform that systematically develops 87 drug sensitivity prediction models for 87 clinical drugs using ex vivo drug responses from 520 AML patients with RNA-Seq is presented. The approach leverages network-based analysis and machine learning to identify biologically interpretable gene signatures that capture the complex molecular interactions driving differential drug responses. Notably, the signature genes derived from the models reveal distinct cellular mechanisms. For instance, the co-expression of C19ORF59 and FLT3 is associated with resistance to FLT3 inhibitors. In summary, NetAML offers a powerful strategy for personalized AML treatment by constructing drug-specific models, identifying clinically actionable biomarkers, and supporting the development of optimized, patient-specific therapeutic regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b29cce61ad93b17c3f11650bbf2734b68498fe27" target='_blank'>
              A Network-Driven Framework for Drug Response Precision Prediction of Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Yinyin Wang, Rui Liu, Yinnan Zhang, Xiang Luo, Chengzhuang Yu, Shentong Fang, Ninghua Tan, Jing Tang
          </td>
          <td>2025-07-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d7a0ecc45c91a460045023891d09c9a49fcad6" target='_blank'>
              Identification and Validation of a Previously Missed Mutational Signature in Colorectal Cancer
              </a>
            </td>
          <td>
            Mariya Kazachkova, Burçak Otlu, M. Díaz-Gay, Ammal Abbasi, Sarah Moody, Sandra Perdomo, David C Wedge, Paul Brennan, Michael R Stratton, Ludmil B. Alexandrov
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Lynch syndrome (LS) is the most common inherited disorder predisposing individuals to colorectal cancer (CRC). It results from germline defects in DNA mismatch repair (MMR) genes, which are critical for maintaining genomic integrity. Here, we demonstrate that the expression of the MMR genes MLH1 and PMS2 are significantly reduced in colon tumor tissues from a proband with CRC, potentially resulting from inherited mutations in the MLH1 gene in LS cases. We identified a previously unreported in-frame deletion mutation in the MLH1 gene, classified as a variant of uncertain significance (VUS) due to its undefined role in oncogenesis. The majority of functionally inactive mutants were located in the residues Phe614 to Lys617, which form crucial hydrogen bonds. Taken together, our data reveals a correlation between this mutation and increased susceptibility to LS-associated tumors. The study offers a valuable insight for evaluating cancer susceptibility in carriers of MLH1 mutants, potentially elucidating the functional roles of MLH1 in oncogenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14554-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a32f988e02c1ad79822d81fa1f876d387930af3" target='_blank'>
              An in-frame deletion mutation in MLH1 drives Lynch syndrome-associated colorectal cancer
              </a>
            </td>
          <td>
            Xiaojuan Lin, Ce Bian, Dongni Liang, Lin Li, Qingqing Tang, Qingli Li
          </td>
          <td>2025-07-11</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Oncogenic retroviruses are known for their pathogenesis via insertional mutagenesis, in which the presence of a provirus and its transcriptional control elements alter the expression of a nearby or surrounding host gene. There are reports of proviral integration driving oncogenesis in people with HIV and the use of HIV-derived vectors for gene therapy has raised concern about oncogenic side effects. To study this issue, we used an in vitro primary human CD4 + T cell infection model developed in our laboratory to identify HIV-1 integration sites that might influence cell proliferation or survival. Combining integration site analysis and bulk RNA sequencing, we found that an upregulated STAT3 signature due to proviral insertional mutagenesis was associated with persistent HIV-infected CD4 + T cells. HIV+ persistent cells also expressed a STAT3-related anti-apoptotic and cytotoxic phenotype that resembles that of HIV-associated T cell lymphomas. HIV insertional mutagenesis of STAT3 and expression of its downstream targets provides a model of HIV-associated T cell lymphomas that can be used to further determine the oncogenic drivers of HIV-associated lymphomas, both AIDS- and gene therapy-associated, and, potentially, to evaluate therapeutics against these HIV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309400b908a0604bb16da7990788bd142c72eac7" target='_blank'>
              In vitro HIV DNA integration in STAT3 drives T cell persistence—A model of HIV-associated T cell lymphoma
              </a>
            </td>
          <td>
            M. Rist, M. Kaku, John M. Coffin
          </td>
          <td>2025-07-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aberrant DNA methylation is a hallmark of colorectal cancer (CRC), contributing to tumor progression through the silencing of tumor suppressor genes and activation of oncogenes. Indicaxanthin (IND), a dietary betalain pigment from Opuntia ficus indica, has shown antiproliferative effects in CRC models, yet its epigenetic impact remains unexplored. In this study, we investigated the effects of IND on the methylome of Caco-2 cells using Reduced Representation Bisulfite Sequencing (RRBS). IND induced a global hypermethylation profile, particularly at gene promoters and CpG islands. Among the differentially methylated genes, 60% were protein-coding, and 10% encoded transcription factors, including PAX5 and TFAP4, both hypermethylated at active enhancers. Functional enrichment analysis revealed pathways beyond canonical intestinal functions, suggesting altered cell identity and plasticity. Transcription factor targets (SOX10, NFKB1, AHR, ARNT) were significantly enriched among the affected genes, several of which are involved in transdifferentiation processes. Methylation changes also indicated potential reprogramming toward epithelial cell types from pulmonary or neuroectodermal origin. Moreover, IND induced selective hypomethylation of Alu elements on chromosome 21 and hypermethylation of rDNA loci, hinting at suppressed ribosomal biogenesis. Overall, these findings highlight the epigenetic remodeling potential of IND and its possible role in modulating cell fate and metabolism in CRC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aad87b105620f84af351d2656d30853c48a91d89" target='_blank'>
              Epigenetic Remodeling of Regulatory Regions by Indicaxanthin Suggests a Shift in Cell Identity Programs in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            M. Ragusa, C. Gentile, Aldo Nicosia, Salvatore Costa, Sara Volpes, Laura Greco, F. Naselli, Fabio Caradonna
          </td>
          <td>2025-06-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba6449e97e98af13aca8bca121a01effd920c5b" target='_blank'>
              Function-specific epistasis shapes evolutionary trajectories towards antibiotic resistance
              </a>
            </td>
          <td>
            G. Petrungaro, Theresa Fink, Booshini Fernando, Gerrit Ansmann, T. Bollenbach
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Cancer development and progression are driven by the accumulation of somatic genetic alterations, which occur in a specific temporal order. However, how the order of mutations impacts cancer phenotypes of solid tumors remains poorly understood. To address this, we developed a novel computational framework, IMOP-Cancer (Identifying Mutation Order Pairs in Cancer), to identify mutation gene pairs whose order influences cancer phenotypes. We applied IMOP-Cancer to The Cancer Genome Atlas-Lung Adenocarcinoma (TCGA-LUAD) cohort and identified 446 key mutation order pairs, with 34 pairs significantly associated with prognosis. Mutation order impacts cancer phenotypes, as demonstrated by CSMD3 and PTPRD (tumor proliferation) and TP53 and NAV3 (immune modulation), with effects validated in four independent datasets. We further presented the impact of mutation pairs on cancer phenotypes through case studies in the TCGA cohorts of bladder urothelial carcinoma (BLCA), and colon adenocarcinoma (COAD). We extended this analysis to 33 cancer cohorts from TCGA portal, identifying 106 034 critical mutation pairs across 17 cancers, with 3036 pairs co-occurring in multiple cancers. Shared mutation pairs across cancers also showed distinct effects on cancer phenotype. Our study highlights the importance of mutation order in cancer progression and diversity, offering new insights into the temporal dynamics of co-occurring mutations and paving the way for personalized treatment strategies and improved diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f348ae5004fc1076373867ea177acfe998124027" target='_blank'>
              IMOP-Cancer: identifying mutation order pairs impacting cancer phenotypes
              </a>
            </td>
          <td>
            Yijing Zhang, Shaobo Kang, Renjie Dou, Wanmei Zhang, Yuanyuan Liu, Yang Wu, Dongxue Li, Fangfang Fan, Yanyan Ping
          </td>
          <td>2025-07-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Allele-specific expression (ASE) of somatic mutations can be caused by cis-activation of the mutant allele or silencing of the wildtype allele and has been investigated by examining the enrichment of mutant allele in RNA relative to DNA. Here we show that this mutation-based approach can be confounded by gene expression differences in tumor and normal cells which co-exist in most bulk tumor samples. We modeled mutant allele expression by incorporating tumor/normal expression difference, mutant allele dosage, tumor purity, and nonsense-mediated decay (NMD) efficiency, projecting that the enrichment can occur without ASE. This confounding effect is exacerbated with low tumor purity and is dependent on mutant allele dosage for NMD-sensitive mutations. The model was validated empirically using somatic insertions/deletions (indels) from 9,101 the Cancer Genome Atlas (TCGA) samples, showing three-fold higher enrichment in driver genes vs. non-drivers. To explore if the enriched mutant allele expression can be leveraged to complement DNA-based somatic mutation detection, we performed de novo somatic indel calling using TCGA RNA-Seq, increasing the TCGA driver indel repertoire by ∼14%, mostly in samples with lower tumor purity. Our study not only revealed confounding factors in ASE analysis but also demonstrated the utility of leveraging RNA-Seq for driver mutation analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/224205d6db5d08579b24b382ff5353dc292b7523" target='_blank'>
              Confounding factors in assessing the enriched expression of somatic mutant allele in bulk tumor samples
              </a>
            </td>
          <td>
            Kohei Hagiwara, A. Thrasher, Nadezhda V. Terekhanova, Jinghui Zhang
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In cancer genomics, there is the realization that some very frequently mutated genes like TP53 or KRAS are extremely hard to approach from a therapeutic perspective. In the realm of cancer treatment, the development of agents capable of transforming these alterations into actionable targets would be like capturing unicorns. These unicorns of oncology-drugs targeting undruggable targets that span various cancer types-offer an exciting opportunity for the development of histology-agnostic therapies. Among these, recent developments in targeting TP53 mutations, KRAS alterations, and MTAP loss mark pivotal advancements for innovative treatment strategies. TP53 mutations, which occur in nearly every cancer type, are central to tumorigenesis and contribute to resistance against conventional therapies. KRAS mutations, especially in cancers like pancreatic and lung cancers, are notoriously difficult to target but are now yielding promising therapeutic avenues. Similarly, MTAP is frequently deleted in cancers such as lung, pancreatic, and mesothelioma and plays a critical role in cellular metabolism and epigenetics, making its loss a significant vulnerability in tumor cells. Recent developments in drug discovery provide a broad foundation for the clinical testing of drugs that target common mechanisms across diverse tumor types, offering hope for a more universal approach to cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bc931554d7166b2b731a598949019062e38b61" target='_blank'>
              Capturing Unicorns: Targeting Cancers With TP53 Mutations, RAS Alterations Beyond G12C, and MTAP Loss-No Target Is Out of the Realm of Possibility.
              </a>
            </td>
          <td>
            Pooja A. Shah, K. Wiman, K. Cichowski, J. Rodon Ahnert
          </td>
          <td>2025-06-01</td>
          <td>American Society of Clinical Oncology educational book. American Society of Clinical Oncology. Annual Meeting</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="
 8602


 Background:
 MTAP
 loss leads to methylthioadenosine (MTA) accumulation, disrupting downstream metabolic pathways. Targeting synthetic lethal interactions with
 MTAP
 loss offers a promising therapeutic strategy. The DNA damage response (DDR) pathway, essential for genomic stability, is commonly dysregulated in lung cancer, impacting treatment response and prognosis. Emerging evidence has highlighted potential association between
 MTAP
 loss and mutations in DDR genes, suggesting that investigating their relationship is critical for uncovering disease mechanisms and identifying novel biomarkers and therapeutic targets.
 Methods:
 Hybridization capture sequencing with StarPanel NGS Assay (1,326 genes) was conducted on 2,258 Chinese lung cancer patients' tumor and matched peripheral blood samples. Then somatic and germline mutations, and TMB values of each sample were obtained. Based on a depth-based algorithm, differences between tissue and control samples in CNVs from the gene level were analyzed. Besides, we utilized shifts in germline heterozygous SNPs within the gene region to assist in determining if the loss was homozygous or heterozygous.
 Results:
 Homozygous and heterozygous deletions of
 MTAP
 were detected in 11.07% and 6.95% of all samples. Notably, 61.67% of these samples exhibited co-loss of
 MTAP
 ,
 CDKN2A
 and
 CDKN2B
 . The median TMB was significantly higher in samples with
 MTAP
 loss (3.85 mut/Mb) compared to those with intact
 MTAP
 (2.56 mut/Mb). In samples with homozygous
 MTAP
 loss, the top somatically co-altered genes were
 EGFR
 (76.00%),
 TP53
 (57.60%) and
 CDKN2A
 (52.80%). Gain-of-function (GoF) mutations were most prevalent in
 EGFR
 (56.80%),
 KRAS
 (12.00%) and
 MDM2
 (10.00%), while Loss-of-function (LoF) mutations were most common in
 TP53
 (50.40%),
 RBM10
 (11.60%) and
 PTEN
 (7.60%). Notably, LoF mutations in
 MTAP
 -loss samples showed a higher prevalence of DDR genes compared to
 MTAP
 -intact samples, including
 RAD50
 (2.40% vs. 0.00%,
 p
 < .0001) and
 POLQ
 (2.40% vs. 0.00%,
 p
 < .0001). Enrichment analysis further revealed LoF mutations unique to
 MTAP
 -loss samples were significantly enriched in the DDR pathway (
 p
 < .0001). For top germline pathogenic mutations, two DDR genes,
 RECQL4
 (0.80% vs. 0.38%,
 p
 = 0.29) and
 BRCA2
 (0.40% vs. 0.43%,
 p
 = 1), showed no significant differences between
 MTAP
 -loss and
 MTAP
 -intact samples.
 Conclusions:
 Potential association exists between
 MTAP
 loss and DDR pathway dysregulation, which may impact tumorigenic processes and therapeutic vulnerabilities. The enrichment of DDR-related LoF mutations indicates
 MTAP
 loss could exacerbate genomic instability by impairing DNA damage repair mechanisms, thereby increasing TMB and driving cancer progression with a distinct molecular profile. These vulnerabilities are primarily driven by somatic mutations, providing a rationale for exploring personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305ad8ea528f00d8a04f7506987172e9d5a5207d" target='_blank'>
              Dysregulation of DNA damage repair in lung cancer driven by
 MTAP
 loss: Mechanistic insights and target discovery.
              </a>
            </td>
          <td>
            Bo Jiang, W. Liao, Xin Wang, Hongliang Hui, Yan-Gui Lin, Linjie Qiu, Dan Li, Min Luo, Haoran Miao, Fan Qiu, Yiqian Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU-023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T > C in KKU-023, m.9493G > A in KKU-055, m.9172C > A in KKU-100, m.15024G > C in KKU-213A, m.12994G > A in KKU-452, and m.13406G > A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92494ba8290eb1c5fd7c0d865dc1a82ce3683d1" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines.
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, Poramate Klanrit, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, Hasaya Dokduang, W. Loilome
          </td>
          <td>2025-06-09</td>
          <td>PloS one</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a deadly disease known for its genetic heterogeneity. LTR12C is an endogenous retrovirus-derived regulator of pro-apoptotic genes and is normally silenced by epigenetic regulation. In this study, we found that the treatment of two glioblastoma cell lines, T98-G and U87-MG, with DNA methyltransferase (DNMT) and histone deacetylase (HDAC) inhibitors activated LTR12C expression. Combined treatment with these epigenetic drugs exerted a synergistic action on the LTR12C activation in both cell lines, while treatment with each drug as a single agent had a far weaker effect. A strong induction of the expression of the TP63 gene was seen in both cell lines, with the pro-apoptotic isoform GTA-p63 accounting for most of this increase. Coherently, downstream targets of p63, such as p21 and PUMA, were also induced by the combined treatment. Furthermore, we observed a significant reduction in the GBM cell growth and viability following the dual DNMT/HDAC inhibition. These findings reveal that the reactivation of LTR12C expression has the potential to modulate survival pathways in glioblastoma and provide information regarding possible epigenetic mechanisms that can be used to treat this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b329500c4414f15e1eee2caec6a0bfaed4d1e7f7" target='_blank'>
              Functional Characterization of LTR12C as Regulators of Germ-Cell-Associated TA-p63 in U87-MG and T98-G In Vitro Models
              </a>
            </td>
          <td>
            Lucia Meola, Sohum Rajesh Shetty, A. Peschiaroli, Claudio Sette, Camilla Bernardini
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8860f8e38f6028b324d356789271bd3f9604af9d" target='_blank'>
              Transcriptional Biomarkers in the Diagnosis of Genetic Disorders: Opportunities, Challenges, and Prospects for Application.
              </a>
            </td>
          <td>
            Lidia N. Nefedova, Tatiana N. Krasnova
          </td>
          <td>2025-06-01</td>
          <td>Biochemistry. Biokhimiia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Elucidating the role of somatic mutations in cancer, healthy tissues, and aging depends on methods that can accurately characterize somatic mosaicism across different cell types, as well as assay their impact on cellular function. Current technologies to study cell-type-specific somatic mutations within tissues are low-throughput. We developed Duplex-Multiome, incorporating duplex consensus sequencing to accurately identify somatic single-nucleotide variants (sSNV) from the same nucleus simultaneously analyzed for single-nucleus ATAC-seq (snATAC-seq) and RNA-seq (snRNA-seq). By introducing strand-tagging into the construction of snATAC-seq libraries, duplex sequencing reduces sequencing error by >10,000-fold while eliminating artifactual mutational signatures. When applied to 98%/2% mixed cell lines, Duplex-Multiome identified sSNVs present in 2% of cells with 92% precision and accurately captured known sSNV mutational spectra, while revealing unexpected subclonal lineages. Duplex-Multiome of > 51,400 nuclei from postmortem brain tissue captured sSNV burdens and spectra across all major brain cell types and subtypes, including those difficult to assay by single-cell whole-genome sequencing (scWGS). This revealed for the first time that diverse neuronal and glial cell types show distinct rates and patterns of age-related mutation, while also directly discovering developmental cell lineage relationships. Duplex-Multiome identified clonal sSNVs occurring at increased rates in glia of certain aged brains, as well as clonal sSNVs that correlated with changes in expression of nearby genes, in both neurotypical and autism spectrum disorder (ASD) individuals, directly demonstrating that somatic mutagenesis can contribute to gene expression phenotypes. Duplex-Multiome can be easily adopted into the 10X Multiome protocol and will bridge somatic mosaicism to a wide range of phenotypic readouts across cell types and tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Although genetic variants associated with bladder cancer (BCa) risk have been identified through hypothesis-driven and genome-wide association studies, a systematic understanding of BCa genetic susceptibility at the gene and pathway levels remains to be achieved.



 In this 2-stage functional genomics study, we used 5 independent tools for genome-wide gene mapping and ranking based on BCa genome-wide association studies summary statistics, followed by a meta-analysis of gene-level significance p values, to obtain a consensus gene ranking in terms of association with BCa. Subsequently, we performed preranked gene-set enrichment analysis to identify the functional pathways involved in BCa genetic susceptibility. Joint analysis with gene-set enrichment analysis, based on somatic alteration frequency, was performed to explore the pathway-level relationships between genetic susceptibility and somatic alterations in BCa.



 Other than the well-known BCa genes (such as FGFR3, MYC, TERT, CCNE1, and TP63), we additionally prioritized a set of novel genes likely to be genetically implicated in BCa development, including SETD2, a possible tumor suppressor gene involved in chromatin remodeling. We further demonstrated convergence between genetic associations and somatic alterations at both the gene (eg, FGFR3 and TERT) and pathway levels (eg, cell cycle and chromatin modification), as well as functional ontologies specifically implicated in germline predisposition to BCa (eg, CD8/TCR signaling, immune checkpoints, and cytokine signaling).



 We identified several novel genes associated with BCa and demonstrated that genetic variants contribute to the development of BCa by affecting antitumor immunity, response to toxic exposure, and RNA and protein homeostasis and synergizing with somatic alterations in various cancer-related pathways.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff12d27723662d274dee934c75d998f194be948" target='_blank'>
              Gene and pathway analysis of genome-wide genetic associations of bladder cancer
              </a>
            </td>
          <td>
            Mingjun Shi, Xiangyu Meng, Xuan Xu, Qiaoli Wang
          </td>
          <td>2025-06-05</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76dc4a293cd6dc10ea69cf397e4b517d730cdc8e" target='_blank'>
              Synergistic role of Alu and core duplicon sequences in driving genomic instability at the disease-associated 16p12.3–p13.11 region
              </a>
            </td>
          <td>
            Stefania Fornezza, Giuliana Giannuzzi
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromothripsis, a phenomenon of massive genomic rearrangements, introduces extensive mutations in critical genes, affecting cell survival and apoptosis. Among these genes, the BCL2 (B-cell lymphoma 2) gene, which plays a crucial antiapoptotic role in cancer cells, is often subjected to significant alterations. Here, we present a computational pipeline to model and analyze the structural and functional impacts of chromothripsis-induced Single-Nucleotide Polymorphisms (SNPs) within the BCL2 gene. This pipeline integrates mutation simulation, homology modeling, and protein inter-action analysis to evaluate the stability and apoptotic potential of BCL2 mutations. These results indicate that chromothripsis-induced mutations can destabilize the Bcl-2 protein, thereby disrupting its binding affinity with apoptotic regulators, such as Bax. These findings support the potential of ergodic anticancer therapy to exploit such mutations, facilitating the apoptosis of cancer cells. Our computational model offers a novel in silico approach for understanding mutation-driven alterations in cancer biology, aiding the development of therapeutic strategies targeting apoptotic pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8abc547c111e4d9c763f84a8f9da96c16d5a06c5" target='_blank'>
              Breaking Boundaries in Cancer Therapy: Harnessing Chromothripsis- Induced Mutations for Targeted BCL2 Protein Destabilization.
              </a>
            </td>
          <td>
            Sergey Shityakov, Michael Nosonovsky, Ekaterina V. Skorb, Viacheslav Kravtsov
          </td>
          <td>2025-06-02</td>
          <td>Current drug discovery technologies</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The DNA repair protein RAD18 activates ‘Y-family’ Trans-Lesion Synthesis (TLS) DNA polymerases that are DNA damage-tolerant and potentially error-prone. RAD18 is also frequently overexpressed and pathologically activated in cancer cells. However, the extent to which RAD18 shapes cancer genomes and impacts tumorigenesis is unclear. Therefore, we tested the effect of Rad18 status on chemically-induced and oncogene-driven tumorigenesis. In a chemically-induced oral carcinogenesis model, acute (2-16 days) 4NQO-treatment induces expression of Rad18 and TLS polymerase mRNAs in mouse oral epithelial cells prior to emergence of oral squamous cell carcinomas (OSCCs). Chronic (8 week) 4NQO-treatment leads to onset of oral tumors that is accelerated in Rad18-/- mice when compared with Rad18+/+ animals. Analysis of OSCC genomes reveals increased levels of G(C)>T(A) transversions in Rad18-/- tumors when compared with Rad18+/+. Therefore, Rad18 promotes error-free bypass of 4NQO-induced DNA lesions and suppresses 4NQO-induced oral carcinogenesis. In a KrasG12D-induced lung carcinogenesis model, Rad18-deficiency did not affect rates or incidence of oncogene-induced lung tumors or mutations. Taken together, we demonstrate that Rad18 has context-specific tumor-suppressive activity. Given the prevalence of 4NQO-like environmental exposures, RAD18 is highly likely to shape human cancer genomes and perhaps influence other aspects of the tumorigenic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318ddf85ddf5e7011f464ee8a8d2e2b51144fcdd" target='_blank'>
              DIFFERENTIAL ROLES OF RAD18 IN REPRESSING CARCINOGEN- AND ONCOGENE-DRIVEN MUTAGENESIS IN VIVO
              </a>
            </td>
          <td>
            J. R. Anand, Bethany Wagner, Jitong Lou, Qisheng Gu, Yang Yang, Gaith Droby, Di Wu, Akankshya Jena, Jialiu Xie, Yuri Fedoriw, Bernard E Weissman, Scott E. Williams, Cyrus Vaziri
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 e15180


 Background:
 Pancreatic cancer has apoor prognosis and limited therapeutic options. While
 K-RAS
 mutations are prevalent in this malignancy, a subset of patients harboring wild-type
 K-RAS
 presents distinct clinical behavior. Gene fusions have been discerned in pancreatic cancer and are promising candidates for administering targeted therapies.
 Methods:
 We conducted a retrospective analysis of 926 pancreatic cancer patients of which 165 (18%) carried a wild-type
 K-RAS
 gene, by next generation sequencing (NGS) or polymerase chain reaction (PCR) testing. Tumors were analyzed with a clinical-grade hybrid capture-based RNA-seq clinical fusionome test, evaluating gene fusions between 1104 genes or to other known or novel partners as well as gene expression transcript analysis. Single nucleotide variants/insertions deletions (SNV/Indels) were tested with a clinical-grade NGS assay targeting > 300 cancer genes.
 Results:
 Gene fusions were detected in 32% of
 K-RAS
 wild-type pancreatic cancers, with 75% involving known oncogenic, clinically actionable genes including
 FGFR
 (two
 FGFR2
 and one
 FGFR1
 fusion), two
 NTRK3
 and one each of
 RAF1, RET, ROS1
 and
 BRAF
 fusions. Deleterious SNVs were frequently found in both fusion-positive and fusion-negative
 K-RAS
 wild-type pancreatic cancers. Transcriptomic expression revealed additional potentially druggable targets, e.g.,
 ROS1
 and
 FGFR3
 overexpression in a tumor harboring an
 NTRK3
 fusion,
 BRAF
 overexpression in the context of an otherwise non-actionable
 ETV6
 fusion, and
 MET
 overexpression in a
 TES::TFEC
 -fusion bearing cancer.
 Conclusions:
 Druggable fusions are frequent in
 K-RAS
 wild-type pancreatic cancers. Moreover, RNA fusion assays as well as transcriptomic expression testing can identify candidate personalized therapies for patients with
 K-RAS
 wild-type pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723fc53e1c67e373fda744cec67f0720b9c9b122" target='_blank'>
              The cancer fusionome and overexpressed druggable oncogenic signals in
 K-RAS
 wild-type pancreatic cancer.
              </a>
            </td>
          <td>
            Michal Krawczyk, Roisin Puentes, Nathan D Montgomery, Derek D Lyle, R. Kurzrock, F. Lopez-Diaz
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemotherapy. However, PDAC with germline BRCA mutations, which lead to homologous recombination (HR) deficiency (HRD), demonstrated an increased sensitivity to platinum-based chemotherapy regimens. This increased chemosensitivity was also seen in PDACs with germline or somatic mutations in the DNA double-strand damage response (DDR) genes beyond canonical HR genes such as BRCA1, BRCA2, and PALB2. However, there are no consensus methods to determine HRD status; and neither is there a well-defined list of HR-DDR genes. In addition, how HRD and/or HR-DDR gene mutation status impacts the tumor immune microenvironment including tumor mutation burden (TMB), neoantigen load, T cell receptor (TCR) repertoire, and effector T cell infiltration is unknown. Thus, in this study, we developed a new method to categorize PDACs into HRD-positive and HRD-negative subgroups by using results from whole exome sequencing, whole genome sequencing, or both into consideration. We classified a cohort of 89 PDACs into HRD-positive (n = 18) and HRD-negative (n = 69) tumors. HR-DDR gene variants were identified more frequently in HRD-positive PDACs than HRD-negative PDACs, with RAD51B, BRCA2 and ATM alterations most frequently identified in HRD-positive PDACs. Notably, TMB and neoantigen load was significantly higher in HRD-positive PDACs compared to HRD-negative tumors. Interestingly, HRD-positive PDACs, PDACs with high tumor mutational burden, and PDAC with high neoantigen load were all associated with lower CD8 + T lymphocyte infiltration and T cell clonal diversity, suggesting a mechanism of resistance to immune checkpoint inhibitors (ICIs). Therefore, this study suggests that treatments to enhance effector T cell infiltration and T cell clonal diversity may overcome resistance to ICI-based immunotherapy in HRD-positive PDACs. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00673-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76f00cebbe1db7a6bb20d7d6bd25ec9f1ccbba1" target='_blank'>
              Homologous recombination-DNA damage response defects increase TMB and neoantigen load, but not effector T cell density and clonal diversity in pancreatic cancer
              </a>
            </td>
          <td>
            Mengyue Lei, Jessica Gai, T. McPhaul, Huijuan Luo, Penghui Lin, Dongbin Liu, Michael Pishvaian, Nicholas J Roberts, Kui Wu, Jin He, Lei Zheng
          </td>
          <td>2025-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e17020


 Background:
 Testicular germ cell tumors (TGCT) are the most common malignancy in young adult men, with a globally increasing incidence. While these tumors are highly curable due to their exceptional sensitivity to cisplatin (cDDP), a proportion of patients develops a resistance to this chemotherapy, which results in disease progression and patient death. The cause of cisplatin resistance in TGCT has not been elucidated yet and is supposed to be multifactorial.
 Methods:
 We conducted a CRISPR Knockout (KO) screening in a TGCT cell line (NCCIT) using a lentiviral sgRNAs library (Addgene #101926) targeting 3015 genes involved in cell cycle regulation and cancer development. The screening population was cultured with or without cDDP (1.25 uM) for 7 days. Samples of treated and control cells were harvested before and after cDDP exposure, the sgRNA cassettes were sequenced by massive parallel sequencing, and the differences in sgRNA abundance were determined using the MAGeCK/MAGeCK-Flute pipeline.
 Results:
 The CRISPR-KO screening data analysis identified 59 gene KOs that were significantly positively selected and 7 gene KOs that were significantly negatively selected after cDDP exposure. Among the top positive hits were genes involved in the regulation of apoptosis (
 BMF, CDIP1
 and
 BAX
 ), epigenetic modifiers (
 HDAC8, SSX2
 ), G-protein regulators (
 ADCY8, GNG10
 ), and others (e.g.
 MAGEA2, ACADM, LAMP1
 ) — inactivation of these genes potentially contributes to the development of cDDP resistance. Among the top negative hits were genes implicated in DNA damage repair (
 FANCB
 ,
 ERCC5
 ), sphingolipid metabolism (
 KDSR
 ), and cytokinesis (
 SPECC1L
 ) — inactivation of these genes may increase sensitivity to cDDP.
 Conclusions:
 The results highlight several potential mechanisms of cDPP resistance in TGCT. Some of these genes and pathways are well known and have been previously described in association with platinum resistance in malignant tumors, while others — such as the G-protein regulators, sphingolipid synthesis, and
 SPECC1L
 — are novel in this context and may reflect the unique biology of TGCT progression. SPECC1L is involved in spindle organization and microtubule stabilization; its paralog SPECC1 is highly expressed in testis. Polyenoic sphingolipids are essential for the completion of meiosis in male germ cells and spermatozoa production.

 ADCY8 controls the production of cAMP, which is a key signaling molecule in spermatogenesis. They have also been described in association with malignant proliferation and progression. The identified hits will be validated in subsequent experiments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff267218117b8545c90f6a863002128909b0591f" target='_blank'>
              Use of CRISPR-KO screening to identify novel genes and pathways potentially contributing to cisplatin resistance in testicular germ cell tumors.
              </a>
            </td>
          <td>
            Laura Matouskova, Kristyna Soffrova, V. Bakardjieva-Mihaylova, J. Stuchlý, L. Boublikova, Karolina Skvarova
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5b8cce1b576a2635e4230be0b09f8a3f1a1a138" target='_blank'>
              Metabolite correlation networks reveal complex phenotypes of adaptation-driving mutations
              </a>
            </td>
          <td>
            Sharon Samuel, Weronika Jasinska, Ezra Sternlicht, Matvey Nikelshparg, D. Kleiner, S. Pilosof, Shimon Bershtein
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Breast cancer (BC) is a highly prevalent malignancy in women and is often resistant to available therapies, calling for urgent investigation of the molecular mechanisms underlying its pathogenesis and progression. BC is thought to result from a complex interplay between genetic and environmental factors. Among key factors, chronic stress has been associated with worse cancer outcomes and can profoundly impact the epigenome. However, both stress and BC phenotypes are complex and heterogeneous, making studies that examine their molecular links challenging. Despite their heterogeneity, stressors trigger a neuroendocrine response that in humans culminates in the release of cortisol, a highly lipophilic hormone that traverses essentially every cell and induces widespread genomic effects. Modeling such effects at the epigenetic level, here we examine whether cellular DNA methylation (DNAm) markers of chronic stress – derived from human fibroblasts undergoing prolonged exposure to physiological stress cortisol levels – distinguish BC phenotypes in two independent human cohorts. Our results show that methylomic signatures of stress are consistently higher in tumor samples as compared to normal samples and in more advanced tumor stages and grades. Follow-up analyses further identify specific DNAm sites driving these associations, which are significantly enriched for cell adhesion pathways in both cohorts. These findings provide insights into the molecular mechanisms linking stress with BC and a proof-of-concept for utilizing cell model-derived disease biomarkers in environmental epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1693e248c789481400e0b6452a28ce9652ad34f" target='_blank'>
              Cell-Derived Epigenomic Markers of Chronic Stress Distinguish Breast Cancer Phenotypes
              </a>
            </td>
          <td>
            Amrita Acharya, Anthony S. Zannas
          </td>
          <td>2025-06-01</td>
          <td>The Yale Journal of Biology and Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains one of the most lethal malignancies, largely due to its pronounced cellular heterogeneity, therapy resistance, and metastatic propensity, traits driven by cancer stem cells (CSCs). Once thought to be a rare, static subpopulation, CSCs are now recognized as a dynamic, plastic cell state influenced by both intrinsic and extrinsic factors. In this review, we explore the “inner” (genetic, cellular, and metabolic) and “outer” (microenvironmental and macroenvironmental) determinants that govern CSC identity, emergence, and evolution in PDAC. We focus on how acinar and ductal cell plasticity, in combination with oncogenic mutations (e.g., KRAS, TP53), contribute to tumor initiation and CSC generation, how genomic instability further fuels CSC heterogeneity and adaptability, and we highlight how micro and macro TME properties, metabolism and external factors regulate CSC behavior afterwards. Altogether, CSCs exist at the nexus of genetic chaos and environmental cues. Understanding this complex interplay is key to developing therapies that not only eradicate CSCs but also prevent their reemergence. While still far from clinical application, disrupting CSC-supportive networks should be a research and clinical focus for improving outcomes for PDAC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b08e2a3f4aedf4a7111928c29cee984777f2e319" target='_blank'>
              Inner and Outer Determinants of Pancreatic Cancer Stem Cells
              </a>
            </td>
          <td>
            J. C. López-Gil, Diego Navarro, B. Sainz
          </td>
          <td>2025-06-16</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e20051


 Background:
 Histone lactylation, a newly identified epigenetic modification, has been extensively implicated in the initiation and progression of tumors. However, the precise role of histone lactylation in modulating tumor cell responses to T cell-mediated cytotoxicity remains insufficiently explored.
 Methods:
 To elucidate the adaptive changes tumor cells undergo in response to T cell-induced cytotoxicity within the tumor microenvironment (TME), we performed a comprehensive, genome-wide CRISPR-Cas9 screen. This approach facilitated the identification of genetic alterations in lung cancer cells co-cultured with cytotoxic T lymphocytes (CTLs) that promote their evasion of T cell-mediated destruction. Complementary RNA sequencing (RNA-seq) and chromatin immunoprecipitation sequencing (ChIP-seq) analyses were employed to investigate the molecular mechanisms by which tumor cells modulate their response to T cell-mediated killing, particularly following the deletion of the monocarboxylate transporter 1 (MCT1).
 Results:
 MCT1 deletion significantly heightened the cancer-cell susceptibility to CTL-mediated killing. Furthermore, the loss of MCT1 led to mitochondrial dysfunction within the tumor cells. Mechanistically, MCT1 knockout resulted in a reduction of histone H3K18 lactylation and downregulation of ferridoxin reductase (FDXR), which disrupted mitochondrial homeostasis and precipitated mitochondrial dysfunction. Notably, the combination of MCT1 inhibition with immune checkpoint blockade (ICB) therapy substantially enhanced ICB efficacy, prolonged survival in both lung cancer and malignant pleural effusion (MPE) murine models, and alleviated the immunosuppressive milieu within the TME.
 Conclusions:
 Our findings reveal a novel role for lactate metabolism in modulating mitochondrial dysfunction and anti-tumor immunity, with lactylation-mediated regulation of mitochondrial homeostasis serving as a critical mechanism. These results provide new insights into the epigenetic regulation of tumorigenesis and highlight potential immune-metabolic strategies for targeted lactate metabolism therapies in the treatment of lung cancer and MPE.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7369949ad290ca8642cb013b1fc65d9e0567a5" target='_blank'>
              MCT1-mediated histone lactylation modulates lung cancer cell sensitivity to T-cell–mediated killing by influencing mitochondrial dysfunction.
              </a>
            </td>
          <td>
            Q. Xue, W. Peng, Qiong Zhou
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BRAF and MEK inhibitors (BRAF/MEKi) have radically changed the treatment landscape of advanced BRAF mutation-positive tumours. However, limited efficacy and emergence of drug resistance are major barriers for successful treatments. Here, by using relevant preclinical models, we find that Connexin43 (Cx43), a protein that plays a role in cell-to-cell communication, enhances the effectiveness of BRAF/MEKi by recruiting DNA repair complexes to lamin-associated domains and promoting persistent DNA damage and cellular senescence. The nuclear compartmentalization promoted by Cx43 contributes to genome instability and synthetic lethality caused by excessive DNA damage, which could provide a therapeutic approach for these tumours to overcome drug resistance. Based on these findings, we designed a drug combination using small extracellular vesicles (sEVs) to deliver the full-Cx43 in combination with the BRAF/MEKi. This study reveals Cx43 as a regulator of DNA repair and BRAF/MEKi response, highlighting the therapeutic potential that this approach could eventually have in the clinic to overcome the limitations of current therapies and improve treatment outcomes for patients with advanced BRAF mutant tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1e4426f618e52ca064c1a54a223e0ac95626bea" target='_blank'>
              Cx43 enhances response to BRAF/MEK inhibitors by reducing DNA repair capacity
              </a>
            </td>
          <td>
            A. Varela-Vázquez, A. Guitián-Caamaño, P. Carpintero-Fernández, Alexander Carneiro-Figueira, Vanesa Álvarez, M. Varela-Eirín, Teresa Calleja-Chucla, S. B. Bravo-López, Anxo Vidal, Juan Sendón-Lago, Marina Rodríguez-Candela Mateos, José R Caeiro, Victoria Sanz-Moreno, Trond Aasen, Miguel G. Blanco, Guadalupe Sabio, María Quindós, Carmen Rivas, David Santamaría, Carlos Fernandez-Lozano, E. Fonseca, Pablo Huertas, Berta Sánchez-Laorden, Constance Alabert, M. Mayán
          </td>
          <td>2025-07-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686514f6a242593983edc4eedbd2ea02d17cd369" target='_blank'>
              REpair of heterozygous Mutations independent of Exogenous Donor template with high efficiency (REMEDY) using allele specific CRISPR targeting and HDR enhancers
              </a>
            </td>
          <td>
            Y. Sezgin, Genesis Snyder, Noushin Saljoughian, Colin Maguire, Ebru Erzurumluoglu Gokalp, Devi Jaganathan, Eleonora S D'Ambrosio, B. Ozes, Gregory Wheeler, Ben Kelly, Mark E. Hester, Zarife Sahenk, S. Vaidyanathan, A. Rashnonejad, Jerry Mendell, Nizar Y. Saad, Roshini S. Abraham, Juhi Bagaitkar, Susan D. Reynolds, Allison Bradbury, M. N. Kararoudi
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2c94a8a149ed8b89b155e50f8c13949352a8f13" target='_blank'>
              Molecular dynamics driving phenotypic divergence among KRAS mutants in pancreatic tumorigenesis
              </a>
            </td>
          <td>
            A. Grimont, David J. Falvo, Whitney J. Sisso, P. Zumbo, Christopher W. Chan, Francisco Santos, Grace Pan, Megan Cleveland, Tomer M. Yaron, Alexa Osterhoudt, Yinuo Meng, Maria Paz Zafra, William B. Fall, André F. Rendeiro, Erika Hissong, R. Yantiss, D. Betel, Mark A. Magnuson, Steven D. Leach, Anil K. Rustgi, Lukas E. Dow, R. Chandwani
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b9da8860ba14d09c8dc1fb69eaa54dde193b6ba" target='_blank'>
              Exon-skipping and genetic compensation due to biallelic mutations in the neurodevelopmental disease gene LNPK
              </a>
            </td>
          <td>
            Rose M. Doss, Sara A. Wirth, Jonathan W. Pitsch, Caroline M. Dias, A. Gropman, Martin W. Breuss
          </td>
          <td>2025-06-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba8d29d07cea737ebc62462b3266da22e085b82c" target='_blank'>
              LIN28A-Dependent Kinome and Phosphoproteome Reprogramming Promotes Imatinib Resistance
              </a>
            </td>
          <td>
            O. Hovey, Mallory I. Frederick, Quan M. Quach, Jenica H. Kakadia, Alyssa Wu, Kyle Yang, Tingting Wu, Xiang Ruan, Tomonori Kaneko, C. Voss, Ilka U. Heinemann, S. S. Li
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef9a4a9c46c279f0d7bcc3c37222b01d2f23045" target='_blank'>
              Minimal activation of the p53 DNA damage response by a modular cytosine base editor enables effective multiplexed gene knockout in induced pluripotent stem cells
              </a>
            </td>
          <td>
            R. Blassberg, Olga Mielczarek, J. Stombaugh, H. Okkenhaug, Alice Abreu Torres, Paul Russell, John Lambourne, Immacolata Porreca
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: SARS-CoV-2, which drove the 2019 COVID-19 pandemic, continues to engender inquiries into the role of host genetic factors in disease susceptibility. Despite the identification of over 1,000 genes potentially associated with SARS-CoV-2 and COVID-19, the mechanisms connecting genetic variants to phenotype remain elusive. To shed light on these mechanisms, we undertook an integrated analysis, merging data from whole-genome association analyses of COVID-19 with methylome and transcriptomic. Methods: Study includes African American adults from the GENE-FORECAST study, encompassing 371 individuals with whole genome sequencing (WGS), 203 with DNA methylation, and 321 with RNA sequencing (RNA-Seq) of blood. 53.3% of participants reported COVID-19. Significant loci associated with COVID-19 were examined within the framework of methylation quantitative trait loci (mQTL) which located near the gene-of-original (cis-mQTL) and expression quantitative trait loci (eQTL) which located near the gene-of-origin (cis-eQTL), enabling analysis to assess mediators between genetic variants and COVID-19 status. Results: Our analysis identified four intronic variants and confirmed a missense variant, rs1052067, in PMF1 associated with COVID-19. Causal mediation analysis revealed that the combination of genetic variants within PMF1, epigenomics, and transcriptomics mapped four pathways influencing COVID-19 status. These pathways include: rs9659072 -> DNAm at chr1:156285845 (annotated to TMEM79) -> ENSG00000198715:13 (annotated to GLMP); rs12083543 -> DNAm at chr1:155951748 (ARHGEF2) -> ENSG00000198715:13 (GLMP); rs1052067 -> DNAm at chr1:155951748 (ARHGEF2)-> ENSG00000198715:13 (GLMP); rs1543294 -> ENSG00000198715:13 (GLMP) -> DNAm at chr1:156077518 (MEX3A). Conclusions: Through integrated multi-omics analyses, we identified genetic variants whose effects on COVID-19 susceptibility are mediated by changes in DNA methylation and mRNA expression. These findings offer insights into potential mechanistic pathways that merit further exploration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00c7bc8e6839adbc276042c4324afad16196de83" target='_blank'>
              Multi-omics analyses of the complex interplay between genetic variants, DNA methylation, and gene expression in COVID-19.
              </a>
            </td>
          <td>
            Guanjie Chen, Lisa DeRoo, Gabriel Goodney, A. Doumatey, Jie Zhou, A. Adeyemo, C. Rotimi, Amadou Gaye
          </td>
          <td>2025-07-01</td>
          <td>American journal of physiology. Heart and circulatory physiology</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Introduction: Anal squamous cell carcinoma (ASCC) is a rare but increasingly common gastrointestinal malignancy, mainly associated with oncogenic human papillomaviruses (HPVs). The role of non-coding RNAs (ncRNAs) in tumorigenesis is recognized, but the impact of viral ncRNAs on host gene expression remains unclear. Methods: We re-analyzed total RNA-Seq data from 70 anal biopsies: 31 low-grade squamous intraepithelial lesions (LGSIL), 16 high-grade SIL (HGSIL), and 23 ASCC cases. Microbial composition was assessed taxonomically. Novel viral miRNAs were predicted using vsRNAfinder and linked to host targets using TargetScan and expression correlation analyses. Results: Microbial profiling revealed significant differences in abundance, with Alphapapillomaviruses types 9, 10, and 14 enriched across lesion grades. We identified 90 novel viral miRNAs and 177 significant anti-correlated miRNA–mRNA interactions. Target genes were enriched in pathways related to cell cycle, epithelial–mesenchymal transition, lipid metabolism, immune modulation, and viral replication. Discussion: Our findings suggest that HPV-derived miRNAs, including those from low-risk types, may contribute to neoplastic transformation by modulating host regulatory networks. Conclusion: This study highlights viral miRNAs as potential drivers of HPV-related anal cancer and supports their utility as early biomarkers and therapeutic targets in ASCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b911fe59aaa1647471be1a4e931c53c959630dc5" target='_blank'>
              Human Papillomavirus-Encoded microRNAs as Regulators of Human Gene Expression in Anal Squamous Cell Carcinoma: A Meta-Transcriptomics Study
              </a>
            </td>
          <td>
            Daniel J. García, Marco A. Pulpillo-Berrocal, José L. Ruiz, E. Andrés-León, L. Terrón-Camero
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Circular RNA (CircRNA) is a single-stranded RNA arising from back splicing. CircRNAs interact with mRNA, miRNA, and proteins. These interactions regulate various life processes, including transcription, translation, cancer progression, anticancer drug resistance, and metabolism. Due to a lack of cap and poly(A) tails, circRNAs show exceptional stability and resistance to RNase degradation. CircRNAs exhibit dysregulated expression patterns in various cancers and influence cancer progression. Stability and regulatory roles in cancer progression make circRNAs reliable biomarkers and targets for the development of anticancer therapeutics. The dysregulated expression of circRNAs is associated with resistance to anticancer drugs. Enhanced glycolysis by circRNAs leads to resistance to anticancer drugs. CircRNAs have been known to regulate the response to chemotherapy drugs and immune checkpoint inhibitors. Exogenous circRNAs can encode antigens that can induce both innate and adaptive immunity. CircRNA vaccines on lipid nanoparticles have been shown to enhance the sensitivity of cancer patients to immune checkpoint inhibitors. In this review, we summarize the roles and mechanisms of circRNAs in anticancer drug resistance and glycolysis. This review discusses clinical applications of circRNA vaccines to overcome anticancer drug resistance and enhance the efficacy of immune checkpoint inhibitors. The advantages and disadvantages of circRNA vaccines are also discussed. Overall, this review stresses the potential value of circRNAs as new therapeutic targets and diagnostic/prognostic biomarkers for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/874575a6c9ec93c6a8a39ef176c30450fe144ed8" target='_blank'>
              Circular RNAs as Targets for Developing Anticancer Therapeutics.
              </a>
            </td>
          <td>
            Jaewhoon Jeoung, Wonho Kim, Hyein Jo, Dooil Jeoung
          </td>
          <td>2025-07-18</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differentially alternative spliced transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways, and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes, including vFLIP, vCyclin, and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells and significantly inhibited KSHV-driven proliferation, cellular transformation, and tumorigenesis. Consistently, FAM50A knockdown suppressed the proliferation of PEL cells. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. IMPORTANCE Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies. Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b22cecd1973d138799718526caad5582306174" target='_blank'>
              KSHV reprograms host RNA splicing via FAM50A to activate STAT3 and drive oncogenic cellular transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Ling Ding, Karla Paniagua, Xian Wang, Yufei Huang, Mario A. Flores, Shou‐Jiang Gao
          </td>
          <td>2025-06-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, Dariusz Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="All germlines—including those of humans, protists, cancer, and metazoans—are capable of proliferating through asymmetric cell division, giving rise to committed stem cells. Their common evolutionarily roots trace back to the hypoxic germline of the last common ancestor of amoebozoans, metazoans, and fungi, referred to as the Ur-germline. Consequently, all modern germlines that produce stem cells—including the cancer stemgermline—retain physiological characteristics of this ancestral Ur-germline. Stress, particularly hyperoxic germline conditions, can irreversibly damage the DNA repair genes in the stemgermline, leading to genome dysfunction, mitotic arrest (senescence), and loss of key functions such as stemness and asymmetric cell division. In most cases, genomically compromised senescent cells in humans and animals undergo apoptotic senescence and are eliminated. However, a minority of dysfunctional cells undergo restorative senescence, initiating a process of unicellularization and genome reconstruction. During unicellularization, genes associated with multicellular functions are downregulated, while ancient unicellular stemgermline genes are upregulated. Cells exiting restorative senescence follow a genome repair program via characteristic unicellular mechanisms, such as hyperpolyploidization and depolyploidization. This process gives rise to progenitor cells that establish a cancer stemgermline and a unicellular cancer cell system. Briefly, restorative senescence escapers restore genomic architecture, function, and molecular integrity using evolutionary mechanisms inherited from the Ur-germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e904557b06689862f566644873a1f68fbe6aba" target='_blank'>
              Non-genetic mechanisms in cancer evolution: senescence, unicellularization, and cycles of stemness recovery
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2025-06-25</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 e23551


 Background:
 Sarcomas are rare, heterogeneous, and aggressive malignancies with poor prognosis, driven by dysregulated proliferation pathways (e.g., CDK4/6, c-Myc) and impaired tumor suppression mechanisms (e.g., CDKN2A/p16INK4a-RB, p14ARF-MDM2-p53). These genetic alterations result in uncontrolled growth, metastasis, and therapy resistance. Emerging therapies targeting these pathways, such as CDK4/6 inhibitors and MDM2 antagonists, show promise. Furthermore, genomic testing can provide insights into mesenchymal stem cell-derived sarcomagenesis and oncogenic microenvironmental interactions providing new opportunities for precision-based treatments. This study investigates molecular pathways in aggressive sarcomas which could assist in expanding targeted treatment options beyond the current common genetic fusions.
 Methods:
 Retrospectively, we investigated the comprehensive genomic profiling of 10 sarcoma patients including subtypes leiomyosarcoma(n=2), ewing sarcoma(n=1), metastatic angiosarcoma(n=2), extraskeletal myxoid chondrosarcoma(n=1), pleomorphic liposarcoma(n=1), retroperitoneal leiomyosarcoma(n=1), soft tissue sarcoma(n=1) and differentiated spindle cell sarcoma(n=1) at different disease stages including progressive, metastatic, and recurrent disease types. Next Generation Sequencing (NGS) test was performed using OncoIndx Assay (Shafi et al., 2024).
 Results:
 The study cohort comprised 10 sarcoma patients, including males (n=6) and females (n=4). Disease stages included progressive (n=4), metastatic (n=3), and recurrent (n=3), as identified through PET scans and histopathological examination (HPE). Recent therapeutic regimen predominantly included chemotherapy (Eribulin, Palliative chemo, Trabectedin, Gem + Docetaxel), combined therapy (Paclitaxel + Pembro, Pazopanib + DCT), and surgery. Primary genomic findings highlighted the impairment of two critical pathways including cell proliferation (n=3/10) and tumor suppression (n=4/10). Further, no genomic alteration was detected in three patients despite aggressive disease as per PET/HPE findings. In addition, genomic findings from three patients also exhibited a high tumor mutation burden (TMB) thereby expanding their possibilities to immunotherapy.
 Conclusions:
 This study highlights the utility of comprehensive genomic profiling in broadening the scope of treatment by highlighting the exact drivers for and disease aggressiveness in sarcomas. The presence of genomic alterations in cell proliferation and tumor suppression pathways could also be early indicators of disease aggressiveness if closely monitored. With only a few selected fusions currently crucial in sarcomas, this study extends the molecular repertoire for practice. Also, additional analysis like RNA sequencing might be more beneficial in such scenarios.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff451ede2084f04e820f94a7adab6b6822648014" target='_blank'>
              Cellular proliferation and tumor suppression pathways as potential targets in clinically aggressive sarcomas.
              </a>
            </td>
          <td>
            Jyoti Bajpai, D. Kishan, M. Basavalingegowda, K. Hariramani, Aarthi Ramesh, Sandhya Iyer, Kaivallya Dasu, Bhagwat Jadhav, Sangeeta Prajapati, Alian D'Souza, Prakruti Singh, A. Vasudevan, A. Bharde, M. Uttarwar, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Traditional approaches for capturing genomic alterations and DNA methylation require separate assays, complicating clinical workflows and limiting sample utilization, particularly with low-input materials like cell-free DNA. To address these challenges, we introduce a streamlined approach combining mutation and methylation profiling via mutation-protective strand synthesis with modified deoxycytidine triphosphates, demonstrating high concordance with standard enzymatic methyl-seq and DNA-seq in both whole-genome sequencing of cell lines and targeted sequencing of clinical samples. In potential clinical contexts, incorporating multi-omics information with this approach modestly improve circulating tumor DNA (ctDNA) detection by ~12% in pre-treatment lung cancer patients (N = 26) while preserving specificity in healthy controls (N = 13), and reveal relationships between homologous recombination repair (HRR) gene function and homologous recombination deficiency (HRD) mediated by promoter methylation-driven biallelic loss of HRR genes in gynecologic cancer patients (N = 27). For practical convenience, this method was also implemented on qPCR platform with high performance (0.5% limit of detection). With its adaptability and potential utility in ctDNA detection and treatment, this approach holds promise for advancing clinical diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461cfd126a8923a87cd09690f854d762a0619664" target='_blank'>
              A DNA alteration and methylation co-detection method for clinical purpose
              </a>
            </td>
          <td>
            Jiyan Yu, Chunhe Yang, Xintao Zhu, Zhankun Wang, Boping Xu, Ye Cai, Jingbo Zhao, Ruijian Guo, Wuzhou Yuan, Jianqing Wang, Bohao Dong, Frank Ron Zheng, Shuang Yang
          </td>
          <td>2025-06-04</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d73cdbb862170214d86c7235fa4d0f318184be5" target='_blank'>
              Genome-wide CRISPR screens identify PTGES3 as a novel AR modulator
              </a>
            </td>
          <td>
            Haolong Li, James E. Melnyk, Becky Xu Hua Fu, Raunak Shrestha, Meng Zhang, M. Sjöström, S. Feng, Jasmine A. Anderson, Wanting Han, Lisa N Chesner, Hyun Jin Shin, Tatyanah Farsh, Humberto J. Suarez, Seema Nath, Jonathan Chou, Rajdeep Das, Emily A Egusa, Jun Zhu, Aidan Winters, Ashutosh Maheshwari, Junjie T Hua, M. Alshalalfa, William S. Chen, Marsha Calvert, E. Davicioni, Audrey Kishishita, Abhilash Barpanda, Tianyi Liu, Arun P. Wiita, B. Stohr, Javed Siddiqui, Bo Huang, Eric J. Small, K. Shokat, Peter S. Nelson, David A Quigley, Elizabeth V. Wasmuth, Luke A. Gilbert, Felix Y. Feng
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc486e9a397957b4d882241501f31a8794a03818" target='_blank'>
              Precisely defining disease variant effects in CRISPR-edited single cells.
              </a>
            </td>
          <td>
            Y. Baglaenko, Zepeng Mu, Michelle Curtis, Hafsa M Mire, Vidyashree Jayanthi, Majd Al Suqri, Cassidy Liu, Ryan Agnew, A. Nathan, A. Mah-Som, David R. Liu, Gregory A. Newby, Soumya Raychaudhuri
          </td>
          <td>2025-07-23</td>
          <td>Nature</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer is a growing concern around the world with more than 20 million new cases confirmed and counting. This is partly caused by the overuse of Chemotherapeutic drugs against Cancer, leading to its potency. Cancer starts with an accumulation of genetic mutations in the genome, and the ignorance to cell checks by CD8 and NK cells. The Cancer cell utilizes its hallmark capabilities and uses one of its cells to metastasizes in other areas of the body either through the perineural, circulatory, or lymphatic system. p53 is a tumor suppressor gene which repairs the mutations of the cell by conducting cell cycle arrest and inducing apoptosis. However, cancer cells can increase the expression of SB100 which can inhibit p53. One solution is to combine PRIMA-1 and CDDP which restores p53 and it’s wild type functions to induce apoptosis. This effect is increased with the use of adenoviral DN-Akt since it could increase p53 phosphorylation at Ser15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/928e7ed5dc7a58f8574294e13f4f74e2d872a648" target='_blank'>
              Evolving Resistance: The Interplay of p53, Chemoresistance, and Metastasis in Cancer's Journey
              </a>
            </td>
          <td>
            Zenith Kwong
          </td>
          <td>2025-06-17</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Epitranscriptomics has emerged as a rapidly evolving field that focused on studying post-transcriptional RNA modifications and their role in spatiotemporal regulation of gene expression. N6-methyladenosine (m6A) and 5-methylcytosine (m5C) represent the most extensively studied modifications on mRNAs. These reversible modifications, mediated by ‘writer,’ ‘eraser,’ and ‘reader’ proteins, dynamically fine-tune mRNA stability, splicing, and translation. Growing evidence links their dysregulation to pathological states, including cancer progression and metastasis, where their aberrant deposition on oncogenes or tumor suppressors alters cellular signaling and therapeutic responses. In the current study, we present a detailed analysis of the m5C epitranscriptomic landscape across distinct breast cancer molecular subtypes. Using CRISPR/Cas9, we confirm NSUN2 as a key m5C writer in human mRNAs. NSUN2 loss was validated by targeted sequencing and Western blotting. Furthermore, we demonstrate the regulatory effects of NSUN2 on its canonical mRNA targets, revealing its critical role in maintaining proper gene expression networks. Our findings strongly suggest that additional m5C writers contribute to m5C methylation machinery. Additionally, we assessed the functional impact of NSUN2 depletion on mRNAs harboring m5C sites using mRNA stability assays. Furthermore, our analysis revealed distinct m5C methylation patterns among breast cancer subtypes, highlighting unique m5C signatures associated with the disease. Notably, we identified specific hypomethylated and hypermethylated m5C sites in each breast cancer cell line, representing a universal m5C methylation signature for breast cancer. Our study constitutes the first comprehensive m5C epitranscriptomic atlas in human breast cancer and paves the way for future research aimed at developing targeted therapeutic interventions that leverage the m5C methylation landscape. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01648-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c7ab96a995706a16195a85c4f63f01b36d8d6b" target='_blank'>
              Exploring the m5C epitranscriptome of mRNAs in breast cancer cells through genome engineering and long-read sequencing approaches
              </a>
            </td>
          <td>
            Konstantina Athanasopoulou, P. Adamopoulos, P. Tsiakanikas, Andreas Scorilas
          </td>
          <td>2025-06-25</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="This review article aims to provide a comprehensive overview of genetic mutations and alterations in prostate cancer, highlighting their roles in disease development and progression. The scope of the review encompasses common genetic mutations, such as those in the PTEN and TP53 genes, as well as structural alterations like the TMPRSS2-ERG fusion. It also discusses the impact of these genetic changes on cellular functions, including cell cycle regulation and signaling pathways. Major findings indicate that genetic mutations contribute to the uncontrolled proliferation of prostate cells and influence treatment responses, emphasizing the importance of molecular profiling in guiding personalized therapy. Additionally, the review explores the significance of the tumor microenvironment and epigenetic modifications in prostate cancer progression. The insights gained from this review underscore the necessity for ongoing research into genetic alterations to improve diagnostic methods and therapeutic strategies, ultimately enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58a7054831337d3623636983abbff338a1c3f2ff" target='_blank'>
              Genetic Mutations in Prostate Cancer: Insights and Implications
              </a>
            </td>
          <td>
            Chinenye Mary Okorochi, Frederick Olusegun Akinbo, Gerald Ikechi Eze, Kehinde Elizabeth Olukoya, Enoch Chibuike Okorochi, Austin Ifeanyi Obiozor, Uzochukwu Chimdindu Ibe, Simon Enenche Iyalla, Macpherson Ifeanyi Ukachukwu
          </td>
          <td>2025-05-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d271ccc22e71dc47432dac9f4b41d18e3a1dd713" target='_blank'>
              Linking regulatory variants to target genes by integrating single-cell multiome methods and genomic distance.
              </a>
            </td>
          <td>
            E. Dorans, K. Jagadeesh, K. Dey, A. L. Price
          </td>
          <td>2025-06-12</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer remains a significant global health threat, characterized by high morbidity and mortality rates. Despite advancements, numerous challenges persist in cancer treatment, underscoring the urgent need for novel therapeutic strategies. As a burgeoning area of cancer research, increasing evidence highlights the critical role of 5-methylcytosine (m5C) methylation in tumorigenesis, progression, and clinical application. In this review, we systematically review the biological mechanism and clinical translational potential of m5C RNA methylation in cancer and analyze in detail the dynamic interactions between m5C regulatory proteins (writer proteins, eraser proteins, and reader proteins). We focus on the role of m5C regulatory proteins in regulating oncogenic signaling pathways, metabolic reprogramming, and immune microenvironment remodeling in different tumor types, while also exploring its clinical potential as a diagnostic marker, prognostic indicator, and target for therapy. This review not only offers a comprehensive analysis of the molecular mechanisms and tumor regulatory networks associated with m5C modification but also focuses on the clinical translational potential of m5C methylation in cancer, aiming to provide novel insights into precision cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07aa242baf6069f9d71bd233815a9263406948fe" target='_blank'>
              m5C RNA methylation in cancer: from biological mechanism to clinical perspectives
              </a>
            </td>
          <td>
            Fang Yuan, Yuxuan Xia, Zhen Huang, Huihui Gao, Yuqi Bian, Xiangqin Zhou
          </td>
          <td>2025-06-21</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Cervical cancer (CC) remains a significant global health challenge despite advancements in screening, HPV vaccination, and therapeutic strategies. Tumor heterogeneity, driven by epigenetic modifications, affects immune evasion, metastasis, and treatment response. Cancer-associated fibroblasts (CAFs) play a crucial role in CC progression and therapy resistance. Single-cell sequencing offers new insights but remains underutilized in CC research. This study integrates single-cell RNA sequencing (scRNA-seq), spatial transcriptomics, and deconvolution analysis to identify key genes and immunotherapy targets. By constructing a prognostic model and exploring the immune microenvironment, we aim to provide novel insights into CC pathogenesis and potential therapeutic strategies. Methods We utilized scRNA-seq, spatial transcriptomics, deconvolution analysis, and pseudotime trajectory mapping to delineate fibroblast subtypes within the tumor immune microenvironment (TIME) of CC. Functional annotations, differential gene expression profiling, cell–cell communication pathways, and transcription factor networks were systematically analyzed. A prognostic model based on bulk RNA-seq data was constructed and validated through survival analysis, with correlations to immune microenvironment characteristics. Functional experiments investigated the role of SDC1, a critical mediator of fibroblast-tumor crosstalk. Additionally, Fibroblast–tumor cell co-culture systems and functional assays were employed to investigate the paracrine role of SDC1. The CAF MYH11⁺ subpopulation was isolated via fluorescence-activated cell sorting (FACS). Multiplex immunofluorescence and immunohistochemical analyses were performed on both cultured cells and human cervical cancer tissue samples to characterize the spatial distribution and dynamic remodeling of MYH11 during stromal reorganization. Results Six distinct fibroblast subtypes were identified, including the C0 MYH11 + fibroblasts, which exhibited unique roles in stemness maintenance, metabolic activity, and immune regulation. Spatial and functional analyses revealed that the C0 subtype is central to tumor-fibroblast interactions, particularly through the MDK-SDC1 signaling axis. The prognostic model incorporating fibroblast-specific markers demonstrated robust predictive power for patient survival outcomes. Additionally, in vitro SDC1 knockdown significantly inhibited CC cell proliferation, migration, and invasion. Fibroblasts show spatially regulated heterogeneity, with activation markers enriched in the tumor zone and MYH11 highest in normal zones, indicating dynamic stromal remodeling. C0 MYH11 + CAF Promotes Tumor Cell Proliferation, Migration, and Inhibits Apoptosis via Soluble SDC1. Conclusion Our results illustrate, in some ways, the possible immunomodulatory and tumor supporting roles of CAFs in CC TIME and highlight the possibility that the MDK-SDC1 pathway is a promising therapeutic target. This study not only promotes a partially new understanding of temporal heterogeneity in CC, but also provides a possible reference base for the development of new biomarkers and immunotherapy approaches to improve clinical outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03432-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d1387a936b1ef6d7e045f6c7bcff60d568a6f7" target='_blank'>
              Deciphering the tumor immune microenvironment: single-cell and spatial transcriptomic insights into cervical cancer fibroblasts
              </a>
            </td>
          <td>
            Zhiheng Lin, Youwei Zhou, Zhenran Liu, Wenyang Nie, Hengjie Cao, Shengnan Li, Xuanling Li, Lijun Zhu, Guangyao Lin, Yanyu Ding, Yi Jiang, Zuxi Gu, Lianwei Xu, Zhijie Zhao, Hua-Bao Cai
          </td>
          <td>2025-07-05</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d94bb593ee184379443d2fb409c739871b84fe" target='_blank'>
              tugMedi: simulator of cancer-cell evolution for personalized medicine based on the genomic data of patients
              </a>
            </td>
          <td>
            Iurii S. Nagornov, E. Furukawa, M. Nagai, S. Yagishita, Tatsuhiro Shibata, M. Kato
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Cancer treatment options including small molecules, antibody-drug conjugates (ADCs), CAR T-cells, and combination therapies are rapidly expanding to improve cancer patient survival. However, efficacy of targeted therapies is limited by interpatient and intratumoral heterogeneity in breast cancer, and may be further exacerbated by dysregulated tumor microenvironments. For example, studies suggested that changes in tumor and breast tissue microenvironment in obesity (e.g. inflammation, increased growth factors) could lead to worsened prognosis that could be related to therapy resistance. Systematic characterization of the cellular effects of emerging drug candidates, particularly across different tumor microenvironments, is challenging, as it needs to be surveyed in diverse cancer contexts beyond just a handful of cancer cell lines. To overcome this challenge, we created PRISM (Profiling Relative Inhibition Simultaneously in Mixtures), an effective drug discovery screening platform with over 900 genetically distinct cancer cell lines of diverse lineages, to rapidly test therapeutics in a high-throughput manner. These cell lines, including 33 different breast cancer cell lines covering diverse cell types and subgroups, carry a specific 24 nucleotide “barcode” tag and have rich baseline genomic and functional characterization. They are first pooled into 20-25 cell pools of different lineages with similar growth rates, then subsequently cultured together with or without drugs. The relative barcode quantity read out by NextGen sequencing determines cell line responses to drugs. This pooled approach significantly reduces the time and the resources required for drug screening. Importantly, utilizing the rich cell line characterization to interpret viability profiles enables the identification of drivers of differential sensitivity and potential biomarkers of compound response. Furthermore, we created a breast cancer cell specific pool to investigate how media conditions mirroring different tumor microenvironments [e.g. media filtration, addition of insulin (0.01 mg/mL) or EGF (0.01 ug/mL)] can impact drug effects on breast cancer cells. In the PRISM 900+ cell panel treated with ADCs conjugated to different payloads that target HER2, there was a clear association between cell line sensitivity and ERBB2 (encoding HER2) gene expression, with the strongest effects on viability observed in a subset of cell lines overexpressing ERBB2. Upon further comparison of biomarker profiles across different ADCs, we established relationships between payload resistance and ABCB1 efflux pumps emerged for a subset of ADC payloads. Using the breast cancer cell specific pool, our data showed that estrogen receptor (ER) degraders fulvestrant and vepdegestrant (ARV-471) had significantly greater effects in reducing viability of ER+ versus ER- breast cancer cell lines. Media filtering reduced the drug effects of fulvestrant but not ARV-471, suggesting that ARV-471 efficacy may be less sensitive to the tumor microenvironment than fulvestrant. Lapatinib (HER2/neu and EGF receptor inhibitor) was more effective in killing HER2+ than HER2- negative breast cancer cells, with cells cultured in EGF-spiked media being more sensitive to lapatinib, whereas media filtration diminished its effects. We did not observe breast cancer subgroup specific killing by alpelisib (PI3K inhibitor) or olaparib (PARP inhibitor), but breast cancer cells became less sensitive to alpelisib when cultured in either insulin- or EGF-spiked media, and less sensitive to olaparib in EGF-spiked media. Overall, our study highlights how the PRISM 900+ cell panel can provide insights into drug specificity, cancer subtype selectivity, and to uncover clinically relevant targets. The breast cancer specific pool can further increase the throughput of understanding how drug responses may be altered in different tumor microenvironments in breast cancer.
 Citation Format: Blanche C. Ip, Jillian N. Eskra, Tenzin Sangpo, Ashish Bino George, Melissa Ronan, Matthew G. Rees, Jennifer A. Roth. PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-10-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5ec03b1ee18de91282ff62b13d5cb24a1236003" target='_blank'>
              Abstract P4-10-26: PRISM high-throughput screening to elucidate how tumor microenvironments modulate drug responses of breast cancer cells
              </a>
            </td>
          <td>
            Blanche C. Ip, Jillian N. Eskra, Tenzing Sangpo, Ashish Bino George, M. Ronan, M. Rees, Jennifer A. Roth
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Tumor protein p53 target gene 1 (TP53TG1) is a significant long non-coding RNA (lncRNA) located on human chromosome 7q21.12, exhibiting dysregulated expression in several cancer tissues, and operates both as an oncogene and a tumor suppressor. TP53TG1 modulates gene expression by several mechanisms, including “competitive RNA” activity, competitive miRNA binding, control of critical signaling pathways, and facilitation of epigenetic changes. TP53TG1 affects tumor proliferation, migration, invasion, and apoptosis through intricate mechanisms, modifies tumor cells resistance to therapies, influences the tumor immune microenvironment and metabolic reprogramming, and may present various avenues for clinical therapeutic intervention. Notwithstanding considerable advancements in comprehending the molecular function of TP53TG1, some challenges persist, especially regarding the efficient and targeted administration of TP53TG1-based therapeutics. Advanced RNA delivery technologies, including lipid nanoparticles (LNPs), sphingomyelin-based nanosystems, and polymer-based nanocarriers, have demonstrated the capacity to surmount these obstacles. This review aims to clarify the expression pattern of TP53TG1 in 10 malignant tumors, its biological relevance, its potential as a therapeutic target, and its importance in clinical applications, thereby offering a novel perspective on molecular targeting in cancer. In the future, integrating TP53TG1 with conventional medications may augment efficacy, offer personalized treatments, and eventually increase the prognosis for patients with aggressive or drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f226208fa68047280df52857f7f84b24287ce57b" target='_blank'>
              Targeting TP53TG1: a promising prognostic biomarker and therapeutic target for personalized cancer therapy
              </a>
            </td>
          <td>
            Haodong He
          </td>
          <td>2025-07-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 International efforts focused on discovery of biological targets in inflammatory breast cancer (IBC) have suggested that aberrant inter-cellular relationship instead of genomic aberrations is the key driver of IBC. To identify such IBC-specific aberrations, we generated single nucleus chromatin accessibility and transcriptome atlas of IBCs (9,628 nuclei from three donors) and compared IBC atlas with a similar atlas of the healthy breast (81,735 nuclei from 92 donors). We recently reported single nucleus atlas of breast tissues of healthy women of diverse genetic ancestry (Nature Medicine in press). In that report, we described markers that identify three major epithelial cell subtypes [Basal-myoepithelial (BM), luminal adaptive secretory precursor (LASP), luminal hormone sensing (LHS)], two endothelial cell subtypes, two adipocyte subtypes, fibroblasts, macrophages, and T cells of the healthy breast. We also showed that LHS cells are the likely cells-of-origin of Luminal A, Luminal B and HER2+ breast cancers. Unlike these breast cancer subtypes, IBC does not appear to have a cell-of-origin, as epithelial cells in IBCs shared gene expression pattern across BM, LASP and LHS cells. Similar results were obtained when comparison was restricted to ancestry-specific healthy breast atlas. However, individual gene level expression differences affecting specific signaling pathways were observed in epithelial cells of IBCs. LASP, and LHS cells of IBCs overexpressed GPR137C, a positive regulator of mTORC1 signaling, and HS6ST3, an enzyme required for heparan sulfate synthesis. LASP and LHS cells of IBCs displayed activation of mitotic and matrix metalloproteinase signaling, respectively. Endothelial cells of IBCs, which showed significant gene expression differences compared to endothelial cells of the healthy breast, displayed enhanced integrin cell surface interaction but loss of cell junction organization. Most critically, BM, LASP, LHS, and endothelial cells of IBCs compared to their counterparts in the healthy breast showed downregulation of TEX14, an inter-cellular bridge forming factor in germ cells and a regulator of mitosis. Downregulation of TEX14 expression in these cells was accompanied with changes in the chromatin accessibility patterns of this gene. These results reinforce the notion that defective inter-cellular communication is a hallmark of IBC and TEX14 expression levels may serve as a biomarker of this defect. We suggest that greater attention to vascular biology and vascular-epithelial cell communication has to be given for better understanding of IBC biology and therapeutic targeting.
 Citation Format: Harikrishna Nakshatri, Poornima Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu. Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-05-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4562fd16c140b8e77bf1f62e51f30b753052c4ad" target='_blank'>
              Abstract P3-05-23: Single nucleus chromatin accessibility and transcriptome analyses reveal aberrant inter-cellular communication in inflammatory breast cancer
              </a>
            </td>
          <td>
            H. Nakshatri, P. Bhat-Nakshatri, Cihat Erdogan, Hongyu Gao, Yunlong Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Stem-cell aging leads to a progressive decline in self-renewal and differentiation. How changes in chromatin architecture shape the gene-expression program underlying this loss of function remains incompletely understood. Here, we integrate transcriptomic, epigenomic, and Hi-C data from young and in-vitro-aged human mesenchymal stem cells (MSCs) to map super-enhancer–promoter (SE–promoter) loops and trace how they rewire during aging. SE target genes are enriched for Gene Ontology terms central to MSC identity and are disproportionately represented among age-regulated transcripts, suggesting that altered SE activity contributes to functional decline. YY1 is highly enriched at both promoters and SEs in young cells but is depleted from these loci in old cells. Loss of YY1 coincides with weakened Hi-C contacts, and YY1 knockdown in young MSCs recapitulates age-associated expression changes, especially among SE targets. Together, our results highlight YY1 as a key stabilizer of SE–promoter looping and gene-expression homeostasis during stem-cell aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1b7945aec29a964bdbfcf02143406273c34eb8e" target='_blank'>
              Disruption of YY1-mediated super-enhancer–promoter looping drives transcriptomic changes during mammalian stem-cell aging
              </a>
            </td>
          <td>
            Weiwei Dang, Luyang Sun, Brenna S. McCauley, Haiying Liu, Xueqian Chen
          </td>
          <td>2025-06-06</td>
          <td>Research Square</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Kirsten rat sarcoma viral oncogene homolog (KRAS) is the most frequently mutated member of the RAS family of small GTPases (RAS). It affects about one-fifth of cancer cases. The tumor microenvironment (TME) is a multifaceted network of immune cells, metabolites, microbiota, stromal components, and extracellular matrix. It creates a dynamic ecosystem that supports malignant initiation, progression, and therapy resistance through bidirectional crosstalk with tumor cells. Emerging evidence reveals distinct TME landscapes shaped by wild-type versus oncogenic KRAS variants. Additionally, TME rewiring occurs during KRAS-targeted therapies. Deciphering these KRAS-dependent TME architectures and their therapeutic vulnerabilities represents a critical frontier for precision oncology. This review synthesizes key milestones and persistent challenges in KRAS inhibitor development. And it systematically evaluates how KRAS mutations orchestrated immunosuppressive niches, metabolic symbiosis, stromal remodeling, and microbiome dysbiosis, supported by mechanistic insights from preclinical and clinical studies. It further explores therapeutic opportunities arising from targeting TME interactions, including rational combinations of KRAS inhibitors with immune checkpoint blockade, metabolic agents, or microbiota-modulating strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a2718254ccab31c0317c655b38e3998d0c1a62d" target='_blank'>
              Mechanistic Foundations of KRAS-Driven Tumor Ecosystems: Integrating Crosstalk among Immune, Metabolic, Microbial, and Stromal Microenvironment.
              </a>
            </td>
          <td>
            Jiayao Ma, Shenao Fu, Jun Tan, Ying Han, Yihong Chen, Xiangying Deng, Hong Shen, Shan Zeng, Yinghui Peng, Changjing Cai
          </td>
          <td>2025-06-09</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Focal gene amplification serves as an oncogenic driver during tumorigenesis and is a hallmark of many forms of cancer. Oncogene amplifications promote genomic instability, which is integral to cancer cell survival and evolution. However, focal gene amplification potentially affords an opportunity for therapeutic exploitation. As a proof-of-concept, we leverage CRISPR-Cas9 nickase to selectively promote cancer cell death in MYCN-amplified neuroblastoma in a gene amplification-dependent manner. Our analysis demonstrates that CRISPR-Cas9 nickase can generate a lethal number of highly toxic, replication-dependent double-strand breaks in cells harboring amplified loci. Furthermore, we demonstrate that Cas9 nickase—mediated toxicity can be modulated in combination with small molecule inhibitors targeting key regulators of the DNA-damage response or cell death pathways. Importantly, our findings in MYCN-amplified neuroblastoma translate to other cancer types with distinct oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426fa411ebfe0306ef5fb3727f439d058ec49944" target='_blank'>
              Selective targeting of genome amplifications and repeat elements by CRISPR-Cas9 nickases to promote cancer cell death
              </a>
            </td>
          <td>
            Matthew B Hanlon, Jason M Shohet, S. Wolfe
          </td>
          <td>2025-06-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aebba9c32cce818d2e97fcfa124c22eff76b4f3" target='_blank'>
              LSH-mediated resolution of R-loops mitigates transcription-replication conflicts to preserve genomic stability in prostate cancer cells
              </a>
            </td>
          <td>
            Kai Ni, Yu Zhang, Kun Zheng, Jianwen Huang, Nuorong Xiong, Rong Liang, Yingxiao Chen, Honglin Jin, Youlong Hai, Wang Li, Jiayi Ma, Ranxing Yang, Guanggai Xia, Yueqing Bai, Lujie Song, Xiaoyong Hu, Zijian Tang, Qiang Fu
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d38264c518ae5531d6fad77480962e1b2608093d" target='_blank'>
              Inhibition of the minor spliceosome restricts the growth of a broad spectrum of cancers.
              </a>
            </td>
          <td>
            Karen Doggett, Kimberly J Morgan, Anouk M Olthof, S. Mieruszynski, Benjamin B Williams, A. Garnham, M. Milevskiy, L. Whitehead, Janine Coates, M. Buchert, R. J. O'Donoghue, T. Hall, Tracy L Putoczki, Matthias Ernst, Kate D Sutherland, R. Kanadia, Joan K Heath
          </td>
          <td>2025-07-07</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5870c7c37807f788d6b6da82817760500b862c7f" target='_blank'>
              Genetic modifiers of somatic expansion and clinical phenotypes in Huntington's disease highlight shared and tissue-specific effects.
              </a>
            </td>
          <td>
            Jong-Min Lee, Zachariah L. McLean, Kevin Correia, Jun Wan Shin, Sujin Lee, Jae-Hyun Jang, Yukyeong Lee, Kyung-Hee Kim, Doo Eun Choi, Jeffrey D. Long, D. Lucente, Ihn Sik Seong, Ricardo Mouro Pinto, James V. Giordano, J. Mysore, Jacqueline Siciliano, Emanuela Elezi, Jayla Ruliera, T. Gillis, V. Wheeler, M.E Macdonald, J. Gusella, Anna Gatseva, Marc Ciosi, Vilija Lomeikaite, Hossameldin Loay, Darren G. Monckton, Christopher Wills, Tom Massey, Lesley Jones, P. Holmans, Seung Kwak, Cristina Sampaio, Michael Orth, G. B. Landwehrmeyer, J. Paulsen, E. R. Dorsey, Richard H. Myers
          </td>
          <td>2025-06-09</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>127</td>
        </tr>

        <tr id="As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6381edb3d3b2ef9cddaaae818d10d5852bbd41f" target='_blank'>
              Detecting and quantifying clonal selection in somatic stem cells
              </a>
            </td>
          <td>
            Verena Körber, N. A. Jakobsen, N. Ansari-Pour, Rachel Moore, Nina Claudino, M. Metzner, Eva Thielecke, Franziska Esau, B. Usukhbayar, Mirian Angulo Salazar, Simon Newman, B. Kendrick, Adrian H Taylor, Rasheed Afinowi-Luitz, R. Gundle, B. Watkins, Kim Wheway, D. Beazley, S. Dakin, Antony Palmer, Andrew J. Carr, Paresh Vyas, Thomas Höfer
          </td>
          <td>2025-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer represents a significant global health hazard marked by elevated morbidity and mortality rates. Furthermore, the majority of tumor therapies encounter challenges, including metastasis, recurrence, and drug resistance. Consequently, it is essential to identify a specific and efficient tumor treatment approach. In recent years, the ongoing investigation and comprehension of tumors have led to significant attention towards cancer stem cells (CSCs). CSCs can facilitate tumor progression via self-renewal, differentiation capabilities, and multidrug resistance. Their function as a fundamental contributor to tumor heterogeneity, drug resistance, recurrence, and metastasis has emerged as a significant focus in cancer therapy research. In recent years, microRNAs (miRNAs) have been identified as crucial post-transcriptional regulators in biological processes, including chemosensitivity, self-renewal, apoptosis, invasion, and metastasis of cancer stem cells (CSCs). This paper systematically reviews the molecular mechanisms through which miRNAs influence the characteristics of cancer stem cells by targeting essential genes (e.g., SOX2, EGFR, c-Met) and modulating signaling pathways, including Wnt/β-catenin, Notch, Hedgehog, and PI3K/Akt. Furthermore, we investigated the viability of miRNAs as non-invasive biomarkers for cancer diagnosis and prognosis evaluation, examined the similarities and attributes of pivotal miRNAs in modulating cancer stem cell functionality, and deliberated on therapeutic approaches stemming from miRNA regulation of cancer stem cell activity. We anticipate that this research will yield novel insights into targeted cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c085b3dd2e8cad808f86e5932aff3c84c64bec4" target='_blank'>
              Regulatory Functions of microRNAs in Cancer Stem Cells: Mechanism, Facts, and Perspectives.
              </a>
            </td>
          <td>
            Xingmei Mao, Sixue Peng, Yan Lu, Linjiang Song
          </td>
          <td>2025-07-14</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction: Colorectal cancer (CRC) is a heterogeneous and multifactorial malignancy driven by a series of genetic and epigenetic alterations. In this field, telomere/telomerase dysfunction contributes to CRC carcinogenesis by impairing genomic stability and cellular replication. Objective: This study aimed to evaluate genetic and epigenetic alterations in CRC by examining mutation rates in the human telomerase reverse transcriptase (hTERT) promoter region, relative telomere length (RTL), hTERT gene expression, and DNA methylation in the TERT hypermethylated oncological region (THOR). Methods: A total of 45 CRC and 34 adjacent normal tissue samples from Moroccan patients were analyzed using molecular approaches, such as Sanger sequencing, quantitative PCR (qPCR), reverse transcription qPCR (RT-qPCR), and methylation-specific PCR (MSP). Results: No mutations in the hTERT promoter region were identified. However, hypermethylation in the THOR region was reported in 82.2% of CRC samples and 79.4% of adjacent normal tissues. High hTERT expression was detected in 50% of CRC patients. In addition, telomere length was significantly shorter (p=0.002) in cancerous tissues (1.41 [1.36 – 1.43]) compared to normal mucosa (1.559 [1.46 – 1.63]), with an RTL ratio less than 1 (0.90 [0.86 – 0.95]). No significant differences were found between clinicopathological features and hTERT expression, THOR methylation, or RTL, except for a significant correlation between THOR hypermethylation and smaller tumor size (p=0.017) and between THOR methylation and RTL in CRC tissues (p=0.034). Conclusion: These results suggest that telomere lengthening is crucial for CRC initiation and progression, and cancer cells tend to shorten telomeres to maintain the chromosomal instability (CIN) required for tumor progression. Further research is needed to elucidate the mechanisms underlying telomere shortening in CRC and understand the role of telomerase/telomere complex in CRC initiation and progression, which could provide new diagnostic, prognostic, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e32cd6246bc493b71aa8671eea1af0f3ca4e1f7" target='_blank'>
              Exploring genetic and epigenetic alterations in hTERT gene in colorectal cancer
              </a>
            </td>
          <td>
            Meryem Jafari, Boutaina Addoum, A. Laraqui, Mohammed Oukabli, Ahmed Bounaim, Abdelmounim Ait Ali, Sidi Mohammed Bouchantouf, Noureddine Njoumi, Belkouchi Abdelkader, Fatima Zohra BenMoula, Hicham El Annaz, Khalid Ennibi, I. Chaoui, Youssef Bakri, Mohammed Attaleb
          </td>
          <td>2025-06-03</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Akt (v-akt murine thymoma virus oncogene homologue) is a well-known serine-threonine kinase that functions as a central node in various important signal cascades involved in cellular maintenance. Akt has also been implicated in oncogenic malignancies as evidenced by protein overexpression, activation and somatic aberration of components in the phosphoinositide-3 kinase-Akt pathway. As such, Akt is a potential target in cancer therapy. Akt is frequently activated in human cancer tissues not only due to aberrant upstream signaling, but also by genetic mutations in AKT itself. This leads to the aberrant activation of pathways downstream of Akt that regulate cell-cycle progression and metabolism as well as activation of transcription factors that promote oncogenesis. In this review, we summarize previous research on Akt, including the molecular mechanisms underlying Akt signal transduction, as well as its physiologic roles and the pathologic consequences when dysregulated. We also discuss the roles of dysregulated protein overexpression/activation, increases in gene copy number, single nucleotide polymorphisms and the network of non-coding RNAs that regulate this pathway, with a particular focus on lung carcinomas. Finally, we discuss strategies that might lead to more effective targeting of Akt for clinical cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85d3fe246854d31780a784ab845d685bc6cf0479" target='_blank'>
              AKT: A Central Node in Complex Signaling Cascades.
              </a>
            </td>
          <td>
            Kentaro Minegishi, Yoh Dobashi, Emi Kimura, Akiteru Goto
          </td>
          <td>2025-05-30</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic modifications are heritable, reversible alterations that causally reshape chromatin architecture and thereby influence DNA repair without changing nucleotide sequence. DNA methylation, histone modifications and non-coding RNAs profoundly influence DNA repair mechanisms and genomic stability. Aberrant epigenetic patterns in cancer compromise DNA damage recognition and repair, therefore impairing homologous recombination (HR), non-homologous end joining (NHEJ), and base excision repair (BER) by suppressing key repair genes and lowering access to repair sites. Then it is dissected how loss-of-function mutations in Switch/Sucrose non-fermentable, imitation switch and CHD (Chromodomain helicase DNA-binding) chromatin-remodeling complexes impair nucleosome repositioning, preventing effective damage sensing and assembly of repair machinery. Non-coding RNAs contribute to epigenetic silencing at DNA break sites, exacerbating repair deficiencies. This review evaluates recent advances concerning epigenetic dysfunction and DNA repair impairment. It is also highlighted that nanoparticle-mediated delivery strategies are designed to overcome pharmacologic resistance. It is presented how epigenetic dysregulation of DNA repair can guide more effective and drug-resistant cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d922eca60c4fcba055a3683ed299a04149ee715f" target='_blank'>
              Epigenetic Dysregulation in Cancer: Implications for Gene Expression and DNA Repair-Associated Pathways
              </a>
            </td>
          <td>
            Nina Rembiałkowska, Katarzyna Rekiel, Piotr Urbanowicz, Mateusz Mamala, Karolina Marczuk, Maria Wojtaszek, Marta Żywica, Eivina Radzevičiūtė-Valčiukė, V. Novickij, J. Kulbacka
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is a major global health challenge, with an estimated 19.3 million new cases by 2025, largely due to lifestyle changes, environmental factors, and increased life expectancy. Despite advances in conventional therapies, issues such as drug resistance, systemic toxicity, and relapse continue to hinder long‐term survival rates. One underexplored molecular mechanism potentially contributing to cancer progression is the enzyme Human tRNA‐dihydrouridine synthase 2 (hDus2). hDus2 plays a pivotal role in tRNA modification by converting uridine residues to dihydrouridine, enhancing tRNA structural flexibility and translational efficiency. Elevated dihydrouridine levels in tRNA have been correlated with cancer progression, highlighting hDus2 as a potential therapeutic target. In this study, we investigated the inhibition of hDus2 as a novel strategy for cancer therapy. Using in silico techniques, we screened a library of FDA‐approved drugs against the 3D structure of hDus2. Top candidate compounds identified through virtual screening with the Glide module were further evaluated using molecular dynamics simulations. Parameters such as RMSD, RMSF, Rg, SASA, and PCA were analyzed. Additionally, binding free energy calculations were performed to validate the stability and interaction strength of the hDus2 ligand complexes. Our results identified DB09050, a cephalosporin antibiotic used for treating complicated intra‐abdominal infections, and DB00650, a folate analog, as promising hDus2 inhibitors. These findings pave the way for further development of anticancer therapeutics post‐experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3a55c7e0fe1d71c7c054d254cdfb39eedafd941" target='_blank'>
              Repurposing of FDA‐Approved Drugs Against Leukemia by Targeting hDus2
              </a>
            </td>
          <td>
            Anshuman Chandra, Monika Mukhija, Kanika Devi, Susmita Chaudhuri, P. Yadav, Vijay Kumar Goel
          </td>
          <td>2025-06-01</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, Sophia Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 e14570


 Background:
 Immune checkpoint blockades (ICBs) showed little efficacy in pediatric solid tumors. The reconfiguration of gene regulation in cancers is a major factor in tumor immune microenvironment (TiME) remodeling and ICB resistance. Hypothesis: A subset of pediatric tumors may exhibit an epigenetically altered TiME, potentially responsible for ICB resistance. Objective: classify the TiME of pediatric extracranial solid tumors and describe the gene regulatory networks (GRNs) of immune landscapes.
 Methods:
 We studied bulk tumor gene expression and DNA-methylation from 184 pediatric extracranial solid tumors. Similarity network fusion (SNFtool) individualized TiME phenotypes by dual-omic clustering of immune genes. The clusters were compared by differential analysis for gene expression and methylation. We studied the relationship of enhancer methylation to gene expression (ELMER package) to infer methylation reprogramming. An enrichment study of regulatory binding regions (ReMapEnrich) identified the master regulators that define the GRNs unique to each phenotype.
 Results:
 SNF clustering identified 3 phenotypes with 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 exhibited low expression of immune genes (“cold” phenotype), cl2 overexpressed immune genes (“hot” phenotype), and cl3 featured global hypermethylation (epigenetically “altered” phenotype). Different tumor types were present across all phenotypes, but Ewing sarcoma was more frequent in altered, osteosarcoma and neuroblastoma in hot, and Wilms tumor in cold phenotype (
 p
 <0.05). Both hot and altered phenotypes overexpressed immune checkpoints (
 CD274
 ,
 PDCD1
 ), T-cell activator chemokines (
 CXCL9
 ,
 CXCL10
 ) and pro-inflammatory pathways, central to ICB sensitivity. However, only hot tumors overexpressed genes and pathways essential for antigen-presenting machinery and immune recognition. Methylation regulation was responsible for derepressing MHC-II genes and their regulators in hot tumors. In contrast, gene silencing in altered phenotype relied on hypermethylation/downregulation of the CIITA cofactor domain, the master control of MHC-II genes, and other immune-specific master regulators crucial for anti-tumor immunity and ICB sensitivity. The regulatory landscape of immune gene repression in the altered phenotype involved binding regions specific to the MYC/MAX network and the polycomb-group protein PCR2 (
 i.e.,
 EZH2 and SUZ12).
 Conclusions:
 We demonstrated that GRN reconfiguration participates in TiME reshaping in pediatric extracranial solid tumors. A subset of tumors with epigenetically altered immune phenotype has an immune recognition capacity repressed by MYC/MAX and PCR2 complexes. Future studies should investigate specific inhibitors to reprogram the GRN to foster immune recognition and ICB sensitivity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b6a52d83f8f6dace4bda8bd4ea08fd3f518537c" target='_blank'>
              Dual-omic analysis to reveal gene regulatory network of immune landscape in childhood solid tumors and implications for immunotherapy.
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Mona Patoughi, Thibault Mallevaey, Lara Herrmann, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Noel Raynal, Vincent-Philippe Lavallée, Sonia Cellot, T. Tran, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fb2bae8a5c0fba39a07b5a8201b6d06e217381d" target='_blank'>
              Lineage memory shapes viral resistance barriers in human skin
              </a>
            </td>
          <td>
            Laura C. Van Eyndhoven, Grant Kinsler, Jingchao Zhang, Aoife O’Farrell, Rim Abderrahim, Apurv Srivastav, Catherine G. Triandafillou, Todd M. Greco, Kenneth S. Zaret, I. Cristea, Megan H. Orzalli, N. Drayman, Abhyudai Singh, Arjun Raj
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background Targeted sequencing is critical in cancer diagnosis, treatment selection, and monitoring. However, the effectiveness of these methods for reflecting whole-exome sequencing (WES)-level mutational signatures remains unclear. Therefore, we addressed this issue through simulation-based analysis to clarify how well targeted sequencing can reproduce WES-level mutational signatures. Methods We compared the correlation and similarity of mutational signatures between whole-exome sequencing-level mutation data and downsampled data for gene sets targeted by each sequencing method in 13 cancer types. Additionally, a similarity analysis of the mutational signatures was conducted using randomly downsampled data. Results The comparison between whole-exome sequencing and targeted sequencing showed a low correlation based on Pearson’s correlation coefficient but a high similarity based on the Dice similarity index. As a result of the downsampling of data with cancer-related genes and whole genes to evaluate similarity, the cancer-related gene random set showed high similarity when 200–400 genes were selected. However, the whole-genome random set required 2–3 times as many genes as the cancer-related gene random set to show high similarity. Among cancer types, colorectal and lung cancers demonstrated high similarity with fewer downsampled genes, whereas breast and prostate cancers required more downsampled genes to achieve high similarity. Conclusion This study demonstrated that current clinically used targeted sequencing methods can reflect whole-exome sequencing-level mutational signatures. This suggests that considering the cancer type and average number of gene mutations in each patient when selecting targeted sequencing methods can lead to more effective treatment choices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89cf22b23dffb414f1a7e7cc4a7de2334cfe0a1e" target='_blank'>
              Evaluation of potential of targeted sequencing through mutational signature simulation
              </a>
            </td>
          <td>
            Keisuke Kodama, Yiwei Ling, Hiroshi Ichikawa, Toshifumi Wakai, Shujiro Okuda
          </td>
          <td>2025-06-25</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Pancreatic cancer (PC) is a highly malignant digestive system cancer that is difficult to diagnose early and is highly resistant to conventional treatments. Recent studies on epigenetics have provided new insights into the early diagnosis of PC and treatments including immunotherapy. Epigenetic changes can alter gene expression without altering the DNA sequence, involving mechanisms such as DNA methylation, histone modification, and the abnormal expression of noncoding RNAs. These mechanisms play crucial roles in the occurrence, development, and therapeutic response of tumors. This review summarizes progress in epigenetic research on PC and discusses the epigenetic characteristics, underlying mechanisms, early diagnosis, prognostic evaluation, treatment strategies, and future challenges for PC. Strategies targeting epigenetic changes may restore the expression of tumor suppressor genes and regulate the immune microenvironment and may be combined with immune checkpoint inhibitors as a new approach for the precise treatment of PC. However, there are challenges associated with the clinical application of epigenetic therapy, such as the development of resistance and adverse side effects, and challenges in target selection, so new epigenetic drugs and combination treatment strategies are needed. With the development of precision medicine, the integration of epigenetics and clinical medicine research has yielded broadly applicable findings. Interdisciplinary cooperation and advancements in clinical trials will further promote the application of epigenetics in PC research, providing more effective treatment options for patients. In the future, the combined application of epigenetic therapy and immunotherapy, the design of personalized treatment plans, and the development of technologies such as liquid biopsy will likely change the treatment landscape of PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4149d90c094734cc78b9c3853bc13232b1e107" target='_blank'>
              Advances in pancreatic cancer epigenetics: From the mechanism to the clinic
              </a>
            </td>
          <td>
            Jia-Dan Zhou, Hong-Zhang Shen
          </td>
          <td>2025-07-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac106c26ebd96b1e15958a8ddb1a880a0c7c51a9" target='_blank'>
              Intratumoral Heterogeneity of Vimentin Modulates Nuclear Mechanotransduction, DNA Damage Response and Cancer Cell Survival
              </a>
            </td>
          <td>
            Infante Elvira, Terriac Emmanuel, Gelin Matthieu, Siegfried Hugo, Pereira David, Roca Vanessa, Varet Hugo, Khalilian Sara, Rietveld Reinier, Asnacios Atef, van Bodegraven Emma J, Etienne-Manneville Sandrine
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Phosphoglycerate dehydrogenase (PHGDH) is the rate-limiting enzyme in the de novo Serine synthesis pathway (SSP), a highly regulated pathway overexpressed in several tumors. Specifically, PHGDH expression is dynamically regulated during different stages of tumor progression, promoting cancer aggressiveness. Previously, we demonstrated that high Serine (Ser) availability, obtained by increased exogenous uptake or increased PHGDH expression, supports 5-Fluorouracil (5-FU) resistance in colorectal cancer (CRC). Beyond its metabolic role in sustaining Ser biosynthesis, different “non-enzymatic roles” for PHGDH have recently been identified. The present study aims to investigate non-enzymatic mechanisms through which PHGDH regulates 5-FU response in CRC. Methods Overexpression and gene silencing approaches have been used to modulate PHGDH expression in human CRC cell lines to investigate the role of this enzyme in 5-FU cellular response. Identified mechanisms have been validated in selected 5-FU resistant cell lines, CRC patients-derived tumor tissue samples, and patients-derived 3D organoids. Transcriptomic analysis was performed on wild-type and PHGDH-silenced cell lines, allowing the identification of pathways responsible for PHGDH-mediated 5-FU resistance. The relevance of identified genes was validated in vitro and in vivo in a CRC xenograft model. Results PHGDH expression is highly variable among CRC tissues and patient-derived 3D organoids. A retrospective analysis of CRC patients highlighted a correlation between PHGDH expression and therapy response. Coherently, the modulation of PHGDH expression by gene silencing/overexpression affects 5-FU sensitivity in CRC cell lines. Transcriptomic analysis on CRC cell lines stably silenced for PHGDH evidenced down regulation in Hedgehog (HH) pathway. Accordingly, in vitro and in vivo studies demonstrated that the combined treatment of 5-FU and HH pathway inhibitors strongly hinders CRC cell survival and tumor growth in CRC xenograft models. Conclusions PHGDH sustains 5-FU resistance in CRC by mediating the upregulation of the HH signaling; targeting the here identified PHGDH-HH axis increases 5-FU susceptibility in different CRC models suggesting the 5-FU/HH-inhibitors combinatorial therapeutic strategy as a valid approach to counteract drug resistance in CRC. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03447-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e4a03b9a2a33e6b19be4d9c774178fc59d033" target='_blank'>
              PHGDH drives 5-FU chemoresistance in colorectal cancer through the Hedgehog signaling
              </a>
            </td>
          <td>
            Caterina Mancini, Giulia Lori, Gianluca Mattei, Marta Iozzo, Dayana Desideri, Fabio Cianchi, Laura Fortuna, Federico Passagnoli, D. Massi, Filippo Ugolini, Luca Messerini, Salvatore Piscuoglio, Antonio Pezone, Francesca Magherini, A. Biagioni, T. Lottini, Demetra Zambardino, G. Truglio, Elena Petricci, Alberto Magi, A. Arcangeli, Luisa Maresca, Barbara Stecca, E. Pranzini, M.L. Taddei
          </td>
          <td>2025-07-10</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background The three-prime repair exonuclease 1, TREX1, degrades cytosolic DNA to prevent aberrant immune activation. Its inactivation results in DNA accumulation in the cytosol and induction of the cGAS-STING DNA sensing pathway, interferon signaling, and inflammation. Germline pathogenic TREX1 mutations are known to lead to hereditary autoimmune and autoinflammatory disorders, whereas the consequences of TREX1 mutations in cancer remain poorly understood. Results To assess the importance of human TREX1 amino acid variants, we analyzed protein sequences of the functional TREX1b isoform from 168 mammalian species and integrated available data on TREX1 sequence and copy number alterations in hereditary autoimmune and autoinflammatory disorders, cancer, and in human populations. While the entire TREX1b protein was conserved in placental mammals, egg-laying mammals and marsupials had their own unique C-terminal regions, with each predicted to contain a transmembrane domain. We modeled human TREX1 variants occurring in autoimmune disease and cancer samples at 12 protein positions to evaluate their predicted impact on protein stability and function. Conclusions Our findings provide novel insight into the role of TREX1 molecular variation in cancer, where genetic or epigenetic loss of TREX1 activity may improve susceptibility to treatment. However, TREX1 gene deletion in tumors was associated with unfavorable patient outcomes, most likely due its frequent co-occurrence with the loss of the entire 3p chromosomal arm, which contains known cancer-related genes. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00785-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53a6c0f4b15cdd23d8488729c5a843f6b9a322d6" target='_blank'>
              Prevalence and impact of molecular variation in the three-prime repair exonuclease 1 TREX1 and its implications for oncology
              </a>
            </td>
          <td>
            Marwa Shekfeh, Mariam M. Konaté, Julia Krushkal
          </td>
          <td>2025-06-28</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The Mismatch Repair (MMR) pathway, a crucial tumor suppressor mechanism, is dysregulated across cancer types. We previously identified the loss of the MutL complex of the MMR pathway, as a key factor inducing resistance to endocrine therapy in Estrogen Receptor positive (ER+) breast cancer patients. Preliminary data previously generated in the lab supports a role for MutL loss in promoting metastasis of ER+ breast cancer. A mammary-specific knockout mouse model for MLH1, a principal component of MutL, created in the Haricharan lab develops aggressive mammary tumors that metastasize to multiple organs. To unveil the mechanism by which MLH1 loss triggers metastasis, an unbiased RPPA and RNA-Seq was conducted which revealed cytokine secretion in the immune system as a distinctive signature of MLH1- ER+ breast cancer cells. The tumor secretome, particularly cytokines, can manipulate macrophage plasticity to skew polarization states towards either an anti- or pro- metastatic phenotype. Secreted cytokine levels altered by MLH1 loss were identified in multiple ER+ breast cancer cells lines from concentrated conditioned media. Additionally, in vivo experiments with xenograft MLH1- ER+ tumors in nude mice, revealed an elevated number of CD206 expressing tumor-promoting M2 macrophages in the tumor microenvironment. Furthermore, MLH1- ER+ breast cancer cells grown in the presence of macrophages exhibit an increase in their migratory capacity. Our preliminary data strongly indicate that loss of MLH1 alters the ER+ breast cancer secretome and alters macrophage maturation, suggesting a role for MLH1 in immunomodulation supporting metastatic progression. Investigation of this new role will uncover key tumor-intrinsic drivers of “cold” tumor immune microenvironment profiles and present unique opportunities for therapeutic interventions to convert them into “hot” profiles, thereby improving ER+ breast cancer patient outcomes.
 Citation Format: Megha Raghunathan, Aloran Mazumder, Sabrina Carrell, Daniel Lozano, Nindo Punturi, Svasti Haricharan. MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-14.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f9ff414c291cb2d359a6c911b717a2e6733c88" target='_blank'>
              Abstract P1-07-14: MutL loss in ER+ breast cancer promotes pro-tumorigenic macrophage polarization that aids migration capacity
              </a>
            </td>
          <td>
            M. Raghunathan, A. Mazumder, Sabrina Carrell, D. Lozano, Nindo B Punturi, S. Haricharan
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mutation spectra vary across genetic and environmental contexts, leading to differences between and within species. Most research on mutation spectrum has focused on the trinucleotide (3-mer) mutation types in mammals, limiting the breadth and depth of variation surveyed. In this study, we use whole-genome resequencing data across 108 eukaryotic species – including mammals, fish, plants, and invertebrates – to characterize pentanucleotide (5-mer) non-coding mutation spectra using a Bayesian approach. Our findings reveal cytosine transition mutability at CpG and (among plants) at CHG sites as the main drivers of variation in mutation spectra across eukaryotes, correlating strongly with genomic CpG and CHG depletion. However, despite the influence of methylation on CpG mutability, genome-wide average CpG methylation levels do not predict CpG transition rates across species and CHG methylation does not predict CHG transition rate, indicating unknown genetic or environmental factors influencing mutation rates at methylated cytosines. Together, our results illustrate the pivotal role of mutagenesis in shaping genome composition across eukaryotes and highlight a gap in knowledge about the mechanisms governing mutation rates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/840be1a2fe72dda5c8b163dbfd7199ef76a61434" target='_blank'>
              Methylation-associated mutagenesis underlies variation in the mutation spectrum across eukaryotes
              </a>
            </td>
          <td>
            Fabián Ramos-Almodóvar, Ziyue Gao, Benjamin F. Voight, I. Mathieson
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Objective. This study describes mutations of genes that stimulate and regulate cell growth, programmed cell death, DNA repair, and cell growth suppression in a boy with osteosarcoma. 
Case Report. We report a case of bone sarcoma in a 9-year-old boy with possible familial predisposition. In our patient, only a subset of tumor cells expressed the ATRX protein, which is known to control the expression of several genome regions. The function of the p53 protein, which acts as a transcription factor that regulates the DNA damage repair response, cell cycle progression, and apoptosis pathways, is lost in 40-50% of malignant cells. Retinoblastoma was positive in the predominant subset of tumor cells. Deletion is found on chromosome 9, cytoband 9p21.3, where the genes for CDKN2A and CDKN2B are located. Neoplastic cells were SATB2-positive in a substantial subset, with nu- clear staining. The SATB2 protein is a DNA-binding protein involved in transcriptional regulation and chromatin remodeling. Chromosomal losses of 8p and 19q11-q13.43 were also found. These regions contain several tumor suppressor genes, including NKX3.1, whose reduced expression correlates with 8p loss in high-grade tumors. Although there was no known cancer syn- drome in the family, the maternal grandfather had a similar tumor requiring amputation. 
Conclusion. Chromosomal instability is a hallmark of osteosarcoma and is characterized by heterogeneous and extensive genetic complexity. Various numerical and structural genomic rearrangements have been described in cancer cells. However, there is little consistent genetic change to understand the etiopathogenesis of this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec206c4dcb70e35f0aed0eba180bbda6edcf22d" target='_blank'>
              Genetic Analysis of Osteosarcoma Cells in a 9-year-old Boy: Genes Involved in Cell Cycle Control
              </a>
            </td>
          <td>
            Dragan Jovanovic, Alwajih Tariq, Sara Dlugos
          </td>
          <td>2025-06-20</td>
          <td>Acta Medica Academica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aggressive and therapy-resistant glioblastoma is among the most lethal malignant tumors in humans. Complete surgical resection is often unachievable; therefore, combination chemoradiotherapy is used to target tumor cells residual beyond the resection margin. This approach induces DNA damage in tumor cells and activates the apoptosis pathway. Unfortunately, recurrence remains a major clinical challenge, frequently manifesting as more aggressive and treatmentresistant glioblastoma phenotypes. The DNA repair and damage response (DDR) pathways are critical for maintaining genome stability. While defects in these mechanisms contribute to oncogenesis, they also make tumor cells vulnerable to DNA-damaging therapy, as the cells become dependent on residual repair capacity. It is of paramount importance to understand the molecular components of these mechanisms and to identify potential therapeutic/pharmacological targets for improving outcomes in glioblastoma patients. A subpopulation of stem-like cells, designated as glioblastoma cancer stem cells (CSCs), has been identified as a critical factor in the initiation, maintenance, and recurrence of tumors. These cells exhibit therapy resistance due to enhanced DNA repair capacity. In addition, emerging evidence suggests a link between carbohydrate metabolism and DNA repair pathways, thereby revealing novel therapeutic vulnerabilities in glioblastoma. This review examines current strategies targeting DNA repair mechanisms in glioblastoma. We present a synopsis of recent advancements in research concerning the mechanisms and factors involved in the elimination of DNA damage induced by ionizing radiation and temozolomide (TMZ). Furthermore, we explore the potential of DNA repair pathway inhibitors under investigation in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8edb7bc241371338e935ca9559370edd806d8316" target='_blank'>
              DNA Damage and Repair in Glioblastoma: Emerging Therapeutic Perspectives
              </a>
            </td>
          <td>
            I. Gareev, O. Beylerli, S. Roumiantsev
          </td>
          <td>2025-07-01</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT Cancer‐associated fibroblasts (CAFs) are functionally diverse stromal regulators that orchestrate tumor progression, metastasis, and therapy resistance through dynamic crosstalk within the tumor microenvironment (TME). Recent advances in single‐cell multiomics and spatial transcriptomics have identified conserved CAF subtypes with distinct molecular signatures, spatial distributions, and context‐dependent roles, highlighting their dual capacity to promote immunosuppression or restrain tumor growth. However, therapeutic strategies struggle to reconcile this functional duality, hindering clinical translation. This review systematically categorizes CAF subtypes by origin, biomarkers, and TME‐specific functions, focusing on their roles in chemoresistance, maintenance of stemness, and formation of immunosuppressive niches. We evaluate emerging targeting approaches, including selective depletion of tumor‐promoting subsets (e.g., fibroblast activation protein+ CAFs), epigenetic reprogramming toward antitumor phenotypes, and inhibition of CXCL12/CXCR4 or transforming growth factor‐beta signaling pathways. Spatial multiomics‐driven combinatorial therapies, such as the synergistic use of CAFs and immune checkpoint inhibitors, are highlighted as strategies to overcome microenvironment‐driven resistance. By integrating CAF biology with translational advances, this work provides a roadmap for developing subtype‐specific biomarkers and precision stromal therapies, directly informing efforts to disrupt tumor‐stroma coevolution. Key concepts include spatial transcriptomics, stromal reprogramming, and tumor‐stroma coevolution, offering actionable insights for both mechanistic research and clinical innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d8f3a8f9589a88ea165cf34934eda0905c9005" target='_blank'>
              Cancer‐Associated Fibroblasts: Heterogeneity, Cancer Pathogenesis, and Therapeutic Targets
              </a>
            </td>
          <td>
            Yutao Li, Qingyun Liu, Xilin Jing, Yuqi Wang, Xiaohua Jia, Xing Yang, Kezhong Chen
          </td>
          <td>2025-07-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In persons living with HIV (PWH), non-AIDS-related tumors, including colorectal cancer (CRC), have become major health concerns worldwide since the introduction of highly active antiretroviral therapy. To date, no study has addressed the underlying molecular mechanisms in PWH with CRC. To explore the impact of PWH with CRC, we sequenced total RNA and DNA from individuals with HIV-negative and PWH formalin-fixed paraffin-embedded (FFPE) CRC for transcriptome and genome analyses. We performed RNA and DNA extraction from FFPE samples, library preparation, total RNA sequencing, and whole-genome sequencing. A total of 1,705 genes were found to be differentially expressed genes (DEGs), including 1,121 upregulated DEGs and 584 downregulated DEGs, in PWH compared with HIV-negative CRC. Functional pathway analysis revealed that the DEGs were enriched mainly in infectious and immune diseases and various metabolic processes. The immune infiltration results revealed that the numbers of activated dendritic cells (aDCs), natural killer T cells (NKT cells), and T follicular helper cells (Tfh cells) were greater and that the number of memory B cells was lower in patients with CRC than in PWH. Twelve hub genes involved in interferon-stimulated genes (ISGs)-IFI44, MX1, OAS1, OAS3, BST2, IFIT1, FGF2, EGF, CCL3, CCL4, SHH, and PPARG-are positively related to aDC, NKT, Tfh, and memory B cells. We found highly analogous insertions, deletions, and functional annotations of the detected single nucleotide polymorphisms and indel mutations in PWH and patients with HIV-negative CRC. This study provides new insights into crucial ISGs, immune infiltration, immune variants, and pathways involved in CRC with HIV infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cccc5f8e56af8b89c04e12b2c557c686201dd5f" target='_blank'>
              Whole-Genome and RNA Sequencing Reveal Novel Insights into the Pathogenesis of Colorectal Cancer in Persons Living with HIV.
              </a>
            </td>
          <td>
            Yuxue Gao, Pengxiang Yang, Yuanyue Guan, Qiqi Ning, Jing Chang, Dexi Chen, F. Wei, Yulin Zhang, Yuening Zhang
          </td>
          <td>2025-06-30</td>
          <td>AIDS research and human retroviruses</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 e22604


 Background:
 The success of PARP inhibitors in BRCA1/2 mutant tumors represents the convergence of germline and somatic molecular profiling as part of routine management of clinical cancer care. Other predictive molecular alterations with matched precision oncology drugs in the germline setting have also emerged. The expansion of germline genetic testing to guide therapy selection has increasingly shifted the burden of germline cancer risk assessment to the point-of-care oncologists who now have the additional challenge of ensuring appropriate interpretation of genetic test results. We therefore sought to expand OncoKB, MSK’s FDA-recognized precision oncology knowledge base, to incorporate annotations for germline variants by partnering with MSK’s Clinical Genetics and Diagnostic Molecular Genetics (DMG) services to provide clinicians and researchers worldwide with an automated, evidence-based tool for interpreting the pathogenicity and clinical actionability of germline variants.
 Methods:
 We reviewed data compiled by MSK’s DMG for over 2,600 unique germline variants in known cancer susceptibility genes. For each variant, we standardized information on pathogenicity, penetrance, mechanism of inheritance, and associated inherited syndromes to enable integration into OncoKB. Using the OncoKB therapeutic levels of evidence framework, we assigned clinical actionability to individual germline variants based on supporting evidence as to whether the variant is predictive of response to standard care or investigational targeted therapy. Additionally, we expanded the OncoKB API to include endpoints for germline variants, enabling the programmatic annotation of germline sequencing reports and research cohorts.
 Results:
 To date, OncoKB has incorporated information on 2631 unique germline variants across 91 genes implicated in germline cancer predisposition and detected by MSK-IMPACT. 24 genes across >10 cancer types are considered Level 1 standard care biomarkers in the germline setting. To date, more than 3400 patients in the MSK-IMPACT cohort have germline alterations that can be annotated with germline OncoKB.
 Conclusions:
 OncoKB’s new annotation of germline variants provides a novel and easily accessible tool for clinicians who utilize germline genetic testing to programmatically determine the effect of the variant based on MSK clinical expertise. Public release of this data is expected in 2025.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f63c0ca8de2d3faffe2962c69f406fbdcff953" target='_blank'>
              OncoKB: Automated annotation of the pathogenicity and treatment implications of germline mutations in cancer.
              </a>
            </td>
          <td>
            Moriah H. Nissan, Nikita Mehta, Hongxin Zhang, Amanda Dhaneshwar, Margaret Sheehan, Y. Kemel, Calvin Lu, S. Suehnholz, Nicole Fernandez, Hyeonjin Park, Yirong Li, Ciyu Yang, Bree Martin, Y. Murciano-Goroff, Lauren Banaszak, D. Solit, N. Schultz, Z. Stadler, D. Mandelker, D. Chakravarty
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>126</td>
        </tr>

        <tr id="Pathogenic variants of the RB1 gene have commonly been found in many cancer types, including retinoblastoma. Nonsense mutations are the most common mutation type in retinoblastoma; however, few cell lines mimic nonsense mutations in the RB1 gene that are commonly observed in patients. Here, we established retinoblastoma-like cell lines carrying mono- and bi-allelic nonsense mutations in the RB1 gene. We introduced the R552X mutation using target activation-induced cytidine deaminase base editing and successfully constructed cell lines carrying mono- and bi-allelic mutations. The model cell lines showed decreased RB1 expression at both the mRNA and protein levels, and increased cell proliferation. Furthermore, we rescued the nonsense mutation in the RB1 gene in model cell lines by converting stop codon 552 to tryptophan using an adenine base editor. This approach may be applicable for establishing cell lines with pathogenic variants found in patients and suggests a strong potential for the application of gene editing as a therapeutic strategy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10600-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea3a4888b8a55d32192a68005fc9979ffdbc161" target='_blank'>
              Establishment and rescue of fibroblast cell lines carrying a nonsense mutation of RB1 by CRISPR-based base editing
              </a>
            </td>
          <td>
            Youngri Jung, Eunhee Seo, Soyeon Yang, Sangsu Bae, Jeong Hun Kim, Hyeon-Ki Jang, Dong Hyun Jo
          </td>
          <td>2025-07-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 e15133


 Background:
 Leucine-Rich Repeat-Containing Protein 15 (LRRC15), a TGFβ-regulated antigen, is prominently expressed in mesenchymal-derived cancer cells and cancer-associated fibroblasts (CAFs) within aggressive, immune-excluded, and treatment-resistant tumors. In response to these challenges, we developed an LRRC15-targeted radioimmunotheranostic compound designed for molecularly specific, non-invasive imaging and therapy. To fully leverage this technology, our goal was to identify the genomic drivers of heterogeneous TGFβ-induced LRRC15 expression across various tumor microenvironments and to assess the diagnostic and therapeutic efficacy of LRRC15-radioimmunotheranostics.
 Methods:
 We utilized an integrated approach that combined high-throughput functional compound screening, single-cell RNA sequencing (scRNAseq), and small interfering RNA (siRNA) knockdown assays to identify genes influencing TGFβ-LRRC15 activity. The relationship of these genes with clinical outcomes was explored across various tumor cohorts. To facilitate clinical translation of LRRC15-targeted radiotheranostics, we investigated the therapeutic impact of [177Lu]-LRRC15 radioimmunotherapy (RIT) on TGFβ-LRRC15 activity through transcriptomic analyses in multiple murine tumor models at various post-treatment time points.
 Results:
 MMP2
 ,
 SPARC
 ,
 TGFβR2
 , and
 WNT5B
 were identified as key genes to the TGFβ-mediated upregulation of LRRC15. Their expression, especially within CAFs, correlated with patient responses to immunotherapy and the progression of aggressive tumors. [177Lu]-LRRC15 RIT effectively targeted TGFβ-driven LRRC15-positive CAFs and cancer cells, inhibiting tumor progression in murine models. Post-treatment transcriptomic analyses indicated a significant reduction in pro-tumorigenic signatures and disruption of the TGFβ-LRRC15 axis associated with immune resistance.
 Conclusions:
 Understanding the molecular pathways and genomic determinants that regulate biomarker dynamics is crucial for refining targeted diagnostic and therapeutic strategies. We identified
 TGFβR2
 ,
 MMP2
 ,
 SPARC
 , and
 WNT5B
 as pivotal drivers of LRRC15 upregulation in response to TGFβ signaling. Together, these markers, along with LRRC15, predict patient prognosis and response to immunotherapy in immune-excluded tumors. Our results highlight the potential of LRRC15-radiotheranostics to non-invasively detect, monitor, and target TGFβ-driven, immunotherapy-resistant tumors, demonstrating their substantial clinical and translational value.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17994795c0c4dd07c7e95b07b1ff40f44d106d8d" target='_blank'>
              LRRC15-radioimmunotheranostics: Targeting TGFβ-driven resistance in immune-excluded solid tumors.
              </a>
            </td>
          <td>
            H. D. S. Ulmert, Mohamed Altai, Claire M. Storey, Michael Cheng, Katharina Lückerath, S. Subudhi, Ken Herrmann, Wahed Zedan, Julie Tran, Julie Park, Smiths S. Lueong, Marija Trajkovic- Arsic, B. A. Siddiqui, Constance Yuen, Laurent Bentolila, Johannes Czernin, Jens T. Siveke, Daniel L J Thorek, Xia Yang, R. Damoiseaux
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background Gastric cancer (GC) is a prevalent malignant tumor globally, posing a significant threat to human health. The histone code reader Spindlin1 (SPIN1) has been implicated in tumorigenesis and tumor progression, however, the exact molecular mechanisms underlying these processes remain incompletely understood. The biological function and regulatory mechanisms of SPIN1 in GC remain ambiguous. This study aims to investigate the regulatory mechanisms of SPIN1 in the pathogenesis and progression of GC, as well as to identify genes closely associated with SPIN1 and potential biomarkers. Methods Gene expression profiles from 375 patients diagnosed with gastric cancer (GC) and 32 control subjects were obtained from the TCGA-STAD database. Our study examined the relationships between SPIN1 expression and various factors including tumor progression, clinical stage, survival status, immune microenvironment and drug sensitivity within the cohort of 375 GC patients and 32 controls. Furthermore, we investigated the interplay between m6A and 5 mC regulators in influencing the expression of SPIN1 in GC, and identified genes with significant correlations with SPIN1 through Spearman correlation analysis. Results A significantly elevated expression of SPIN1 was found in 375 GC patients compared to 32 control subjects. SPIN1 expression was positively correlated with EMT score and angiogenesis score. Cell proliferation-related gene sets (myogenesis, mitotic spindle and G2M checkpoint) were all significantly associated with the high SPIN1 GC group. Eosinophils was associated with high expression of SPIN1. A total of 21 checkpoints were associated with SPIN1 expression. Low SPIN1 expression group could benefit from Axitinib, Cytarabine, Pazopanib and Sunitinib. Most m6A regulators and a subset of m5C regulators were positively associated with SPIN1. Finally, we screened the 10 genes with the strongest correlation with SPIN1, among which CDH11 and SLC8A1 were associated with the prognosis of GC. Conclusion In conclusion, our study has provided valuable insights into the pivotal role of SPIN1 in GC development, elucidating its potential molecular mechanisms and establishing it as a promising therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/331e134e048d7607cc8b86285c9b6db9b7b6e085" target='_blank'>
              Gene expression profiling of SPIN1 in gastric cancer: insights into tumorigenesis and potential therapeutic targets
              </a>
            </td>
          <td>
            Bei-Bei Lv, Lei Cai, Yao Xiao, Rui-Han Wang, Xiao-Yan Lin
          </td>
          <td>2025-06-11</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14aba9861491c1dd938896b3ff280930aaa89511" target='_blank'>
              HiFIseek: gene-specific enrichment of high-impact mutations in associated genomic regions
              </a>
            </td>
          <td>
            Rafael Riudavets Puig, I. S. Brorson, Denise G. O’Mahony, Vessela N. Kristensen, Anthony Mathelier
          </td>
          <td>2025-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="

Recent advances in CRISPR–Cas9–based genome editing have revolutionized cancer
immunotherapy by enabling precise and efficient T-cell engineering. This review focuses on how
CRISPR-mediated modifications enhance T-cell functionality, highlighting key applications, such
as CAR-T cells and TCR-engineered T-cell therapies. By selectively targeting immune-regulating
genes, CRISPR empowers T cells to recognize and eliminate tumor cells with greater specificity
and reduce off-target effects. This precision opens the door to personalized treatment strategies, allowing
T cells to be tailored to each patient’s unique tumor profile. Furthermore, the ability to
rapidly and systematically edit multiple gene targets has accelerated the development of more potent
and durable T-cell therapies, overcoming some of the current limitations in cancer immunotherapy.
Overall, CRISPR’s transformative potential of CRISPR provides a platform for next-
generation patient-centric cancer treatments, paving the way for improved clinical outcomes and
expanded therapeutic options.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcb955649c8f44c414ba7f048441db68da50f99" target='_blank'>
              Innovation and Implication of CRISPR-Mediated T-cell Engineering in Cancer Immunotherapy: A New Insight
              </a>
            </td>
          <td>
            Syeda Areej Imran, H. Sultan, Muhammad Zubair, S. Ghazanfar, Anas Islam, Syed Mustafizur Rahaman, Sumel Ashique
          </td>
          <td>2025-06-03</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dcbc0047fe628bc7d0d1f2c63478bc9274ca9ad" target='_blank'>
              Punctuated mutagenesis promotes multi-step evolutionary adaptation in human cancers
              </a>
            </td>
          <td>
            Christopher J Graser, Wenbo Wu, Cole Christini, Mia Petljak, Franziska Michor
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer immunotherapies have remodeled the way many different forms of cancer are treated by, leveraging the immune system to recognize and attack malignant cells. These therapies have demonstrated durable responses in cancers affecting the blood, bone marrow, and lymph nodes, solidifying their role as a cornerstone of modern oncology. However, despite their success, current immunotherapy tools and techniques require further refinement.Regulatory non-coding RNAs (ncRNAs), actively transcribed by the mammalian genome, play crucial roles in regulating a wide range of cellular processes. They are key players in epigenetic mechanisms that govern differentiation, development, and the progression of invasive carcinogenesis. Emerging evidence suggests that regulatory ncRNAs are also pivotal in cancer immunity, acting as drivers of immune responses in the development of metastatic tumors. These ncRNAs influence the tumor microenvironment (TME), modulate the immune system, and affect the efficacy of immune checkpoint inhibitors and T cell therapies by regulating various signaling pathways.This review article explores the diagnostic and therapeutic potential of ncRNAs in cancer immunotherapy, highlighting their role in modulating carcinogenesis and antitumor immunity. By understanding the influence of regulatory ncRNAs, we can pave the way for the development of prognostic biomarkers and more effective tumor immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5583bab6f08d24dffa1738f78e75a7bde850afb" target='_blank'>
              Recent Advances in Cancer Immunotherapy Aided by Regulatory Non-Coding RNA: A Review.
              </a>
            </td>
          <td>
            D. Baishya, Arpita Barman, B. Rabha, Kaushik Kumar Bharadwaj, Joyeeta Talukdar
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54707acfce1cdea645a84eff273df6515c31448f" target='_blank'>
              Regulatory roles of ncRNAs in gastric cancer progression: a comprehensive insight into key signaling pathways.
              </a>
            </td>
          <td>
            Tahereh Barati, Zohreh Mirzaei, Fatemeh Abarghooi Kahaki, Hamed Afkhami
          </td>
          <td>2025-07-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Intrahepatic cholangiocarcinoma (ICC) is characterized by high malignancy, and its global incidence is predicted to continue to increase over the past decades. However, the mechanisms underlying ICC pathogenesis and progression remain unclear.


APPROACH AND RESULTS
The training cohort consisted of single-cell sequencing of 12 treatment-naïve ICC samples and spatial transcriptomics of four ICC samples. The validation cohort comprised of RNA-seq data from 87 ICC tumor samples. Finally, we validated our findings via multiplex immunofluorescence, organoids, and mice models both in vivo and in vitro. We found significant heterogeneity within the tumor microenvironment (TME) of ICC patients. ICC cells were classified into five molecular subtypes, and we revealed that aspartate beta-hydroxylase (ASPH) was a marker gene for invasion subtypes. We then selected cepharanthine (CEP) as an ASPH inhibitor that effectively suppressed tumor progression. Regarding the ICC lymphatic metastasis mechanism, we found that tumor cells in N1 lymph nodes highly expressed tumor-specific MHC-II molecules but lacked co-stimulatory factors CD80/CD86, inducing a state of anergy in CD4+ T cells, which might facilitate ICC immune evasion.


CONCLUSIONS
The TME of ICC was heterogeneous. ASPH markedly enhanced ICC invasion The ASPH inhibitor CEP significantly inhibits ICC progression and may serve as a targeted therapeutic drug for ICC. Tumor cells in N1 lymph nodes demonstrate high expression of tumor-specific MHC-II molecules, but silencing of co-stimulatory factors such as CD80/CD86 induces CD4+ T cells into an anergic state. Our study indicated that ASPH and MHC-II may serve as novel therapeutic targets for ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2012ee56dc72ff219a9edcce829b5a1b74d80284" target='_blank'>
              Tumor microenvironment heterogeneity and progression mechanisms in intrahepatic cholangiocarcinoma: A study based on single-cell and spatial transcriptomic sequencing.
              </a>
            </td>
          <td>
            Fengwei Li, Yao Li, Lishan Wang, Lei Xu, Hui Xue, Wenxin Wei, Yong Xia, Lei Wang, Feng Shen, Kui Wang
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive malignancy with a poor prognosis that is driven primarily by oncogenic KRAS mutations present in > 90% of cases. KRAS mutations, particularly the G12D mutation which dominates in PDAC, fuel tumor initiation, progression, and immune evasion, thereby contributing to therapy resistance. Nevertheless, KRAS has long been considered "undruggable" due to its structure. Recent advances have spurred transformative progress in direct KRAS inhibition. While FDA-approved mutation-specific and pan-KRAS inhibitors show limited efficacy in PDAC, emerging agents (MRTX1133 and RMC-9805) have demonstrated preclinical promise. However, resistance remains a critical hurdle and is driven by pathway reactivation, secondary mutations, and metabolic adaptations. Alternative strategies targeting upstream regulators (SHP2 and SOS1) aim to block KRAS activation and associated resistance mechanisms. Preclinical studies have also highlighted synergistic benefits of combining KRAS inhibitors with MEK, PI3K, or CDK4/6 inhibitors, which are now undergoing clinical evaluation. Immunotherapies, including KRAS-targeted vaccines and adoptive T-cell therapies, have further expanded the therapeutic landscape of enhancing KRAS-targeted therapies in PDAC. The molecular basis of KRAS-driven PDAC, current inhibitors, resistance mechanisms, and innovative strategies are discussed herein to address treatment barriers. Opportunities to improve clinical outcomes are underscored in this challenging malignancy by integrating insights from preclinical and clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8cfbe6080878693ce57a6e261fa2c4f9ddf499" target='_blank'>
              Drugging the 'undruggable' KRAS: breakthroughs, challenges, and opportunities in pancreatic cancer.
              </a>
            </td>
          <td>
            Nawaz Khan, Umar Raza, Syed Aqib Ali Zaidi, Muhadaisi Nuer, Kayisaier Abudurousuli, Yipaerguli Paerhati, Alifeiye Aikebaier, Wenting Zhou
          </td>
          <td>2025-07-07</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Recurrent somatic mutations in the key spliceosome component, SF3B1, have been identified at various frequencies across several cancer types. The most common hotspot mutation is the K700E missense mutation, and while its effects on splicing have been well characterised at the molecular level, the mis-spliced genes that contribute to cancer progression and/or dictate responses to therapy are still unclear. Here, we used we use cell line modelling to assess the impact of the SF3B1K700E mutation on the cellular response to various apoptosis-inducing agents. Our data suggest that the SF3B1K700E mutation leads to reduced cFLIP levels, along with defects in the splicing and translation of BCL2, causing a shift in the balance of pro- and anti-apoptotic genes and proteins, which confers greater sensitivity to the bivalent SMAC mimetic, BV-6. As such, BV-6 may represent a therapeutic opportunity for patients with SF3B1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78c6127f794d73c9e1a0f7e0fc6e3deedb616b30" target='_blank'>
              The cancer-associated SF3B1K700E spliceosome mutation confers enhanced sensitivity to BV-6-induced cytotoxicity
              </a>
            </td>
          <td>
            Lydia E Roets, J. Blayney, Hayley McMillan, Patrick J Preston, Alexander Mutch, Ken I Mills, K. Savage, K. Lappin
          </td>
          <td>2025-07-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fda3215ef96d1797670c5063bd5db4df44e918" target='_blank'>
              Genetic variants in oncogenic miRNA and 3' untranslated region of tumor suppressor genes: emerging insight into cancer genetics.
              </a>
            </td>
          <td>
            Anita Choudhary, Anil Kumar, A. Munshi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="


 To investigate whether synthetic embryos (blastoids) derived from three different human stem-cell lines mimic human blastocyst molecular dynamics, including chromosome instability and cellular composition.



 Although human blastoids were derived from different stem cell lines, they exhibit highly similar transcriptomes. Their genome appears to be more stable than human embryos.



 Blastoids—stem cell-derived models of human blastocysts—provide a scalable and ethically-justified model to study early embryonic development. Different stem cell sources influence blastoid differentiation, but their specific impact remains unclear. While blastoids resemble human embryos in both transcriptional and morphological aspects, their genomic stability is largely unexplored. Single-cell RNA-seq and DNA-seq studies have shown that aneuploidy, a hallmark of early development, affects >80% of human blastocysts. Understanding whether blastoids exhibit similar rate of chromosomal instability and how aneuploid cells may be corrected is crucial for assessing their relevance in modeling early human development and advancing reproductive medicine and developmental biology.



 Three different stem cell lines, including human naïve embryonic (HNES1, H9) and induced pluripotent stem cells (niPSC HDF75) were used to generate blastoids. From each blastoid, we isolated 8-16 cells for single cell sequencing. We aimed to (i) investigate how starting cell cultures influence resultant blastoid molecular composition, (ii) determine whether blastoids recapitulate the high levels of chromosome instability observed in preimplantation human embryos, and (iii) characterize abnormal cells within blastoid.



 We collected initial stem cells (Day 0) and blastoid cells 72 hours after trophectoderm (TE) versus inner cell mass (ICM) induction (Day 3). We profiled their genome and transcriptome using single cell G&T-seq, performed copy number profiling, trajectory and differential expression analyses. Additionally, we identified copy number variations from published blastoid single-cell RNA-seq data using the inferCNV algorithm. This allowed us to compare blastoids derived from different stem cell sources, specifically HNES1 and niPSC.



 Transcriptome analysis of Day 3 blastoid cells (n = 155), derived from NHES1 and niPSC, revealed high similarity between them. However, single-cell genome profiling uncovered a mix of genomic stability and imbalance. While 43% of cells displayed chaotic copy number alterations, 57% showed more stable karyotypes, with 38% consistently exhibiting segmental gains on chromosomes 5 and 19. The transcriptomic cell scoring analysis revealed that cells with more disbalanced profiles were predominantly in S-phase or G2/M-phase. In contrast, genomically stable cells were mostly in G1-phase. Notably, copy number inference from public single-cell transcriptomic data of blastoids (n = 120 randomly selected cells; 30 cells per timepoint across 4 timepoints) revealed slight copy number gains, including chromosome 19, at 60 hours post-differentiation, followed by a sharp increase in copy number gains and losses at 96 hours post-differentiation.



 A limitation of our study is that a limited number of cells (5-8) were analyzed per single blastoid.



 Our findings highlight that chromosome instability also emerges in blastoids, though at rates differing from human preimplantation development. However, analyzing genomic instability in these models can help to refine and optimize them to better mimic human development.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb157ed086d09d3d17646e65718ea01bd71ec8a" target='_blank'>
              O-205 Single-cell multi-omics to study genome plasticity in synthetic models of early human development
              </a>
            </td>
          <td>
            D. Obukhova, D. Luijkx, J. Paulissen, A. Swinnen, W. H. Loneus, M. Drüsedau, W. van Dijk, M. Bergamelli, E. Vrij, H. Brunner, A. Salumets, M. Zamani Esteki
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="
 In the United States, breast cancer is one of the leading causes of cancer death in women. Triple-negative breast cancer (TNBC) is the most aggressive of all molecular subtypes of breast cancer. It is highly heterogeneous and difficult to target and treat due to its lack of targetable receptors. Our aim is to identify new molecular targets that could aid the treatment of TNBC cases and reduce cytotoxicity of the current therapeutic regimen. The TP53 gene, a known tumor suppressor is the most frequently mutated gene in triple-negative breast cancer occurring in >80 % of TNBC cases. Statistical pathway analysis from RNA-Seq and ChIP-Seq done on the MCF10A mammary epithelial cell lines expressing the 10 most clinically prevalent p53 mutants identified the Hippo pathway to be dysregulated when associated with aggressive cellular phenotypes. Upon suppression of the upstream Hippo tumor suppressor pathway, the activation of a series of genes promoting cancer hallmark phenotypes ensues. A cell-based TEAD reporter assay showed transcriptional activity of TEAD to be higher in MCF10A cells expressing more aggressive p53 mutants compared to the cell lines with less aggressive mutants. Knockdown of the TAZ protein decreased the invasiveness of the invasive p53 mutant expressing cell lines. We hypothesized that, in addition to neo-morphic activities of different p53 mutants, the heterogeneity in clinical phenotypes observed in TNBC is due to functional interactions of the driver mutation in p53 with other functionally important co-existing mutations and dysregulated pathways, called ‘co-drivers’. To identify potential functionally important ‘co-driver’ mutations that cause cancer progression and heterogeneity in clinical phenotypes, a genome-wide screen was performed using lentiviral CRISPR screen library on the invasive MCF10A-p53-R273C-MycOE cell line. In vivo experiments showed that injection of the library-transduced cells yielded tumor in mice. PCR and whole-exome sequencing of the tumor showed frameshift mutations in the enriched EVC2 (EvC Ciliary Complex Subunit 2), NR1D1 (Nuclear Receptor 1D1), and ARAF (a Ras-activating RAF family kinase) genes. Confirming the hits from screening, knockout of these genes in R273C-MycOE cells also showed increased cell invasion. In parallel, in vitro screens carried out using CRISR knock-out library on the least aggressive MCF10A-p53-Y234C cells identified potential co-drivers. Knockout of GCNT1 and FGD6 genes in MCF10A-p53-Y234C cells increased cell invasion and induced dephosphorylation and activation of TAZ. TEAD activity was also found to be higher in these knock-out cells compared to the Y234C cells. Knockdown of the TAZ protein showed decreased TEAD activity in the FGD6 and GCNT1 knock-out cells and substantially decreased cell invasion. We have carried out functional characterization of the TP53 mutations in TNBC and confirmed the cellular phenotypes for specific p53 mutants obtained from the in vitro and in vivo screens. We are validating these identified ‘co-driver’ genes and targets to discover the molecular mechanisms affecting phenotypic heterogeneity of the p53 mutants and investigating the mechanisms through which Hippo pathway drives the progression of TNBC, as well as assessing the pathway as a target for potential therapy for TNBC with specific TP53 mutations.
 Citation Format: Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura Gonzalez-Malerva, Duatin Grief, Lydia Sakala, Jin Park, Joshua LaBaer. Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f9063c7f21ea86d31d2953e319cbe0d8301969" target='_blank'>
              Abstract P2-11-20: Discovering Co-driver Genes and Pathways of Mutant TP53 in Breast Cancer by CRISPR Screening and Multi-omics Approaches
              </a>
            </td>
          <td>
            Lilian Nwachukwu, Anasuya Pal, Yining Zheng, Laura González‐Malerva, Duatin Grief, Lydia Sakala, Jin G Park, Joshua LaBaer
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8648dc1482068a18c20a79db3017f33c1028502b" target='_blank'>
              Context-dependent regulatory variants in Alzheimer’s disease
              </a>
            </td>
          <td>
            Ziheng Chen, Yaxuan Liu, Ashley R. Brown, Heather H. Sestili, Easwaran Ramamurthy, Xushen Xiong, D. Prokopenko, BaDoi N. Phan, Lahari Gadey, Peinan Hu, Li-Huei Tsai, Lars Bertram, Winston Hide, Rudolph E. Tanzi, M. Kellis, Andreas R. Pfenning
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 How can we improve the safety and efficiency of germline gene correction using CRISPR/Cas9 in early mouse embryos?



 We observed a higher frequency of desirable gene editing events when using crRNA XT for CRISPR/Cas9-mediated gene correction during fertilization of mouse oocytes.



 CRISPR/Cas9 induces double-stranded DNA breaks, which can be repaired via homology-directed repair (HDR) using a repair template to achieve precise gene editing. Undesired events may occur, including non-homologous end joining, leading to error-prone repair causing insertions or deletions (indels), unrepaired DNA breaks resulting in large deletions or partial chromosomal loss, or HDR using the untargeted parental chromosome (gene conversion), causing loss of heterozygosity.
 HDR frequencies in early embryos are typically much lower compared to gene editing outcomes in somatic cell lines. In this study, we aimed to improve HDR efficiency by using chemically modified crRNA optimized for high nuclease environments.



 To achieve gene correction with reduced mosaicism, we co-injected CRISPR/Cas9 along with mutant C57BL/6J mouse sperm (carrying a homozygous Plcz1 deletion, causing failed fertilization after ICSI) into wild-type B6D2F1 and DBA/2 mouse oocytes for subsequent developmental and detailed genetic analysis, respectively.
 We analysed 59 embryos injected with crRNA XT and 60 embryos injected with classical crRNA. The repair template used contained synonymous variants, which, if detected, indicate successful gene correction through HDR with template use.



 We injected CRISPR/Cas9 ribonucleoprotein complexes, designed for correcting the Plcz1 deletion, simultaneously with mutant mouse sperm into wild-type mouse oocytes during piezo-assisted ICSI, followed by artificial activation. We compared the use of crRNA XT, containing additional modifications for improved functional stability, to classical crRNA. After four days of in vitro culture, DNA was extracted and used for targeted sequencing, whole genome amplification, and copy number variation (CNV) analysis to evaluate the events following gene editing.



 After gene correction using classical crRNA, targeted sequencing revealed that the deletion was undetectable in 8 out of 60 embryos (13.3%). However, none of these embryos contained the synonymous variants which would indicate HDR with the provided repair template. In contrast, using crRNA XT, the deletion was no longer detected in 22 out of 59 embryos (37.3%). The synonymous variants were found in 50% of the embryos with undetectable deletions, suggesting a remarkable increase in HDR rates with the repair template. These results indicate that crRNA XT, with its enhanced stability, leads to a remarkable improvement in the frequency of desired gene editing events.
 To explain further the events in which the deletion is no longer detected without incorporation of the repair template’s synonymous variants, we performed whole-genome amplification and copy number variation (CNV) analysis using shallow whole-genome sequencing. In the embryos edited using classical crRNA, we observed partial chromosomal loss of chromosome 6 (carrying the Plcz1 gene) in 1 out of 7 analysed embryos and aneuploidy with partial chromosomal loss of chromosome 6 in 1 out of 7 analysed embryos. CNV analysis of embryos edited with crRNA XT is still ongoing to complete our comparison.



 The resolution of the shallow whole-genome sequencing is limited to 5 Mbp. To detect larger deletions smaller than this threshold, we will perform long-read sequencing.



 Targeted sequencing of embryos undergoing early gene correction for the Plcz1 deletion revealed that the use of crRNA XT, which offers greater stability, significantly enhanced the frequency of HDR with the repair template compared to classical crRNA. These findings contribute to a deeper understanding of germline gene editing events.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797fea7f3b3a355f785c9bfcca4bad7300fab085" target='_blank'>
              P-804 Improving CRISPR/Cas9 gene correction in early mouse embryos using RNase-resistant crRNA XT
              </a>
            </td>
          <td>
            S. Jooris, A. Rybouchkin, F. Jalali, J. Parrington, F. Vanden Meerschaut, T. Hochepied, K. Claes, D. Stoop, B. Heindryckx, B. Menten
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Glioblastoma is the most aggressive type of brain tumor and is associated with a poor prognosis. First-line treatment is surgical resection followed by radiotherapy and temozolomide-based chemotherapy. However, the duration of treatment with temozolomide is limited due to both its toxicity and the development of drug resistance. The prognostic and predictive factor for response to temozolomide is the methylation status of the MGMT promoter. Indeed, loss of MGMT promoter methylation is a major cause of chemoresistance. However, the development of drug resistance is not only associated with changes in MGMT methylation. The entire epigenome changes and acquires specific properties necessary for tumor progression. Methods To study epigenetic alterations associated with temozolomide exposure, we generated a TMZ-resistant cell model. We investigated epigenetic alterations in the cell model and in a cohort of patients with recurrent glioblastoma using genome-wide methylome approaches (Epic Arrays 850 k). Results We investigated the epigenetic changes associated with temozolomide exposure. Therefore, we generated a TMZ-resistant cell model and studied the epigenetic features acquired after selective temozolomide pressure. Our next step was to investigate the epigenomic differences between primary and recurrent tumors in a small cohort of patients. Finally, we performed a cross-analysis between the epigenetic differences found in TMZ-resistant cells and recurrent glioblastomas to identify common signatures that could be used to guide future resistance-overcoming studies. Conclusions Temozolomide induces significant epigenetic changes in glioblastoma, which may contribute to treatment resistance and increased tumor aggressiveness. The results suggest that further research into DNA methylation changes associated with TMZ resistance is crucial. The use of primary tumor cells in resistance models may help identify strategies to overcome chemoresistance in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06767-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/862b534a5409db78fa687bac68e4b9867edb5de0" target='_blank'>
              DNA methylation remodeling in temozolomide resistant recurrent glioblastoma: comparing epigenetic dynamics in vitro and in vivo
              </a>
            </td>
          <td>
            Michela Buonaiuto, Mariella Cuomo, Davide Costabile, Federica Trio, Sara Ferraro, O. Affinito, Alberto De Bellis, Maria Laura Del Basso De Caro, Roberta Visconti, Lorenzo Chiariotti, Giuseppe Catapano, R. Della Monica
          </td>
          <td>2025-07-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The biological mechanisms underlying the cooperation between germline genetic variants and somatic mutations during carcinogenesis are rarely elucidated. Here, characterizing isogenic prostate cancer cell lines, we dissected the interplay between a germline variant at the 7p14.3 locus (rs1376350, G>A) and early recurrent prostate cancer-specific mutation in the Speckle-Type POZ protein (SPOP) gene across human prostate adenocarcinomas. The transcriptomes of multiple edited models pointed to Gli3 and the Hedgehog signaling pathway in a genotype-specific manner, while SPOP mutation and AR stimulation promote Gli3 accumulation in the full-length (FL) transcriptionally active form. This, in turn, triggers the cell-autonomous production of steroids that prostate cancer relies on, in line with the enhanced responsiveness of SPOP-mutated prostate cancer to androgen deprivation therapy. These data demonstrate that germline variants dictate men's prostate cancer somatic evolution and suggest opportunities to jointly model germline-somatic tandems to help untangle the complexity of human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6ef3ec565535edd5c8feb3d331621aab63b409b" target='_blank'>
              Germline-somatic liaison dictates cancer subtype via de novo steroid biosynthesis.
              </a>
            </td>
          <td>
            P. Gasperini, A. Alaimo, Blerta Stringa, Yoon-Mi Chung, Yari Ciani, F. Lorenzin, Giulia Fracassi, Y. Zekri, F. Orlando, Orsetta Quaini, Sebastian Gregoricchio, G. Petris, Antonio Casini, Christopher E Barbieri, W. Zwart, A. Cereseto, Nima Sharifi, Andrea Lunardi, F. Demichelis
          </td>
          <td>2025-06-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ead10baece9545ae1e3801dde43ce3d766e518" target='_blank'>
              METTL5 Blockade Enhances Anti-Tumor Immune Response via Inducing Neoantigen Generation
              </a>
            </td>
          <td>
            Yangyi Zhang, Xiaoyan Shi, Yuci Wang, Ruiqi Wang, Yanlan Cao, Hao Chen
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The PIK3R1 gene encodes the class IA PI3K regulatory subunit p85α, which is frequently altered in cancer. PIK3R1 functions as a tumor suppressor by stabilizing and inhibiting the catalytic activity of p110, and it directly binds to and enhances the activity of the PTEN lipid phosphatase. Aberrations in the PIK3R1 gene are associated with poor prognosis in cancer; available data underscore the significant role of PIK3R1 mutations in mediating tumorigenesis by promoting the signaling of the PI3K/AKT/mTOR pathway. Moreover, copy number variations, driver mutations, and epigenetic alterations in PIK3R1 contribute to tumorigenesis and progression through distinct mechanisms. This article reviews the cancer-promoting effects of PIK3R1 gene aberrations across major cancer types and elucidates their underlying mechanisms. It also discusses the targeted therapies for related aberrations, aiming to provide a comprehensive understanding of the dynamic interplay of PIK3R1 in cancer, thereby advancing precision medicine and the development of targeted interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f1f74c89d744eac6c65e6a4ebb2236a33792d6e" target='_blank'>
              Cancer-Associated PIK3R1 Genetic Aberrations and Precision Medicine
              </a>
            </td>
          <td>
            Huan Xie, Yirong Li, Xinran Li
          </td>
          <td>2025-06-12</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1d0252edfafbc65dacd73e410e1f6e287ac0ef7" target='_blank'>
              Pathway Sculptor for Compact and Versatile Combinatorial Genetic Perturbation
              </a>
            </td>
          <td>
            Bo Gu, James M. Linton, Brice Graham Hendrickson, Hengyu Li, Ron Hadas, Gal Manella, Jan Gregrowicz, M. Elowitz
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e861449b1c1d4d8b590227b0b752be56a345c9e3" target='_blank'>
              In-Silico Characterization of TP53 Splice Mutations in Somatic and Germline Tumours
              </a>
            </td>
          <td>
            Apeksha Arun Bhandarkar, Noah Ethan Kelly-Foleni, Debina Sarkar, Aaron Jeffs, Tania Slatter, Anthony Braithwaite, Sunali Y Mehta
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Over 7000 rare diseases have been described, collectively affecting 350 million people worldwide. Most of these conditions result from nonsense mutations, representing approximately 10% of all genetic mutations associated with human inherited diseases. Nonsense mutations convert a sense codon into a premature termination codon (PTC), leading to premature translation termination and the production of truncated, nonfunctional proteins. This results in a loss‐of‐function phenotype in many genetic disorders, contributing to the disease's severity and progression. The molecular mechanisms of PTC formation involve various genetic alterations, including single‐nucleotide changes, frameshifts, and splicing mutations. The nonsense‐mediated mRNA decay (NMD) pathway degrades mRNAs containing premature termination codons (PTCs). In contrast, 25% of PTC mRNAs, depending on the PTC position and cellular context, can evade NMD, resulting in the synthesis of truncated proteins. A termination codon during translation is essential for proper protein synthesis, and translational readthrough—a process in which the ribosome bypasses the PTC and reaches the natural stop codon—may restore some level of protein function. The effectiveness of readthrough depends on the surrounding genetic context and the type of amino acid incorporated at the PTC position. This review aims to explore the molecular characteristics of nonsense‐related diseases (NRDs), including cystic fibrosis, hemophilia, Fabry disease, choroideremia, Usher syndrome, Shwachman‐Diamond syndrome, and certain hereditary neuropathies and cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c11ad4ab5cb6198324925028453027503cc9a711" target='_blank'>
              Nonsense Mutations in Rare and Ultra‐Rare Human Disorders: An Overview
              </a>
            </td>
          <td>
            Emanuele Vitale, Davide Ricci, Federica Corrao, I. Fiduccia, I. Cruciata, P. Carollo, A. Branchini, L. Lentini, I. Pibiri
          </td>
          <td>2025-06-01</td>
          <td>Iubmb Life</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The ATP-hydrolytic ectoenzyme ENPP1 has been implicated in the metastasis and recurrence in triple-negative breast cancer (TNBC), primarily by contributing to tumor cell survival and treatment resistance. However, the precise mechanisms remain unclear. In a model of local recurrence (LR), circulating tumor cells (CTC) engrafting in the post-resection tumor bed developed a radioresistant phenotype linked to an ENPP1+-gene signature which was also identified in TNBC patients, suggesting ENPP1´s role in genome integrity. Blockade of ENPP1 using a permeable ENPP1 inhibitor (AVA-NP-695) reduced radioresistance, mechanistically attributed to decreased homologous recombination (HR) resulting in persistent DNA damage, as evidenced by enhanced tail moment and sustained γH2AX formation. This impaired DNA damage repair (DDR) sensitized tumor cells to ionizing radiation (IR). Notably, several DDR inhibitors (i) (including PARPi and ATMi) showed the highest synergy score in a targeted pharmacological screening. In vivo, dual ENPP1/ATM inhibition heightened radiosensitivity, compromised tumor cell survival and enhanced STING-TBK1 signaling by preventing ENPP1-mediated cGAMP hydrolysis. This resulted in robust innate and long-lasting adaptive antitumor immune memory responses, leading to significant tumor regression. Remarkably, combined treatment post-IR reduced spontaneous metastasis and local recurrence, and induced abscopal effects that impacted distant tumor spread in orthotopic tumor models. Thus, these findings position ENPP1 as a critical link between genome integrity and immunosuppression, offering promising translational opportunities for treating local or distant dissemination in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34af6ea74bf5effdace3557bda27f7680f31a271" target='_blank'>
              Dual ENPP1/ATM depletion blunts DNA damage repair boosting radioimmune efficacy to abrogate triple-negative breast cancer
              </a>
            </td>
          <td>
            Borja Ruiz-Fernández de Córdoba, K. Valencia, C. Welch, Haritz Moreno, S. Martínez-Canarias, C. Zandueta, Eduardo Gómez, Alfonso Calvo, N. Otegui, M. Echepare, Ignacio Garzón, D. Ajona, David Lara-Astiaso, Elisabet Guruceaga, Laura Guembe, Ruben Pio, I. Melero, Silve Vicent, Fernando Pastor, R. Martinez-Monge, F. Lecanda
          </td>
          <td>2025-06-13</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 e13538


 Background:
 Understanding the molecular profile of cancer-associated genes is crucial in oncology, as it informs targeted therapies and improves patient outcomes. Tumor suppressor genes inhibit the activation of oncogenic signals, while oncogenes promote them. Ras proteins, encoded by KRAS, NRAS, and HRAS genes, are involved in cell growth, differentiation, and survival pathways. This study aims to investigate the frequency of KRAS, NRAS, and HRAS mutations in solid tumors treated at a single academic institution between January 2017 and May 2024. By analyzing these mutations, we seek to enhance our understanding of their distribution across different cancer types in Venezuela, providing valuable insights for personalized cancer treatment strategies.
 Methods:
 We conducted a retrospective, cross-sectional, analytical cohort study. The study included 175 patients, with 179 molecular genetic sequencing studies and 20 RT-PCR analyses. We examined mutations in codons 12, 13, 59, 61, 117, and 146 of the KRAS, NRAS, and HRAS genes.
 Results:
 We identified 43 mutations in the RAS gene family: 37 in KRAS, 5 in NRAS, and 1 in HRAS. Lung cancer accounted for 14 of those mutations, with 13 in KRAS and 1 in HRAS. Colorectal cancer showed 19 mutations, with 16 in KRAS and 3 in NRAS. Other KRAS mutations (13.9%) were found in breast, endometrial, pancreatic cancers, and thymoma. We also recorded 2 NRAS mutations in melanoma and 1 in pancreatic cancer.
 Conclusions:
 This study represents the largest collection of KRAS, NRAS, and HRAS mutations reported in Venezuela.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d60941cc22a3a39b9e5850c1cc1b772e65a8cda" target='_blank'>
              Frequency of RAS family gene mutations in metastatic solid tumors in Venezuela: Experience at a large academic center.
              </a>
            </td>
          <td>
            Guillermo Borga, Oscar A. Sucre Astorga, Santiago Sucre, C. Sucre
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="By offering a precise, effective, and adaptable method for genome editing, CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) technology has completely transformed genetic biotechnology. Through a biochemical lens that highlights the molecular complexities behind its methods and discoveries, this review explores the complex field of CRISPR cloning and its emerging frontiers. We examine the molecular underpinnings of target identification, protospacer adjacent motif (PAM) specificity, and Cas nuclease activation, starting with the groundbreaking discovery of CRISPR-Cas systems in bacterial adaptive immunity. Advances in CRISPR-based cloning techniques, such as the creation of synthetic guide RNAs, Cas variants with specific functions, and high-fidelity and base-editing systems that reduce off-target effects, are also examined in the study. The molecular dynamics of DNA-RNA-protein interactions during gene targeting, activation of DNA repair pathways, and epigenetic state regulation are given special attention. We investigate cutting-edge uses beyond traditional genome editing, including RNA-targeting Cas proteins (like Cas13), CRISPR-based diagnostics and treatments, CRISPR interference (CRISPRi), and CRISPR activation (CRISPRa). Additionally covered is the incorporation of CRISPR with other biochemical technologies, such as gene drives, programmable transcription factors, and synthetic biology circuits. The field's increasing biochemical sophistication is highlighted by new developments like CRISPR-nanoparticle conjugates for targeted delivery and CRISPR screening platforms for functional genomics. In the end, this review emphasizes how a thorough biochemical comprehension of CRISPR systems is essential to maximizing their potential in environmental engineering, biotechnology, medicine, and agriculture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf5fde83fcfa701e2f38df12fc05920c20f45e1" target='_blank'>
              Exploring CRISPR Cloning and Beyond Through a Biochemical Lens in Genetic Biotechnology
              </a>
            </td>
          <td>
            Mariam Mariam, Khizra Ikhlaq, Zulaikha Abid, Fatima Sahar Uraf Maryam, Syed Qasim Mehmood, Ahtasham Ahmad, Sohrab Anwar Abbasi, Imran Ali Sani, Muhammad Adeel
          </td>
          <td>2025-07-02</td>
          <td>Scholars Academic Journal of Biosciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Rare genetic DNA repair deficiency syndromes can lead to immunodeficiency, neurological disorders, and cancer. In the general population, inter-individual variation in DNA repair capacity (DRC) influences susceptibility to cancer and several age-related diseases. Genome wide association studies and functional analyses show that defects in multiple DNA repair pathways jointly increase disease risk, but previous technologies did not permit comprehensive analyses of DNA repair in populations. To overcome these limitations, we used fluorescence multiplex host cell reactivation (FM-HCR) assays that directly quantify DRC across six major DNA repair pathways. We assessed DRC in phytohemagglutinin-stimulated primary lymphocytes from 56 healthy individuals and validated assay reproducibility in 10 individuals with up to five independent blood draws. We furthermore developed generalized analytical pipelines for systematically adjusting for batch effects and both experimental and biological confounders. Our results reveal significant inter-individual variation in DRC for each of 10 reporter assays that measure the efficiency of distinct repair processes. Our data also demonstrate that correlations between the activities of different DNA repair pathways are relatively weak. This finding suggests that each pathway may independently influence susceptibility to the health effects of DNA damage. We furthermore developed a pipeline for analyzing comet repair kinetics and related our new functional data to previously reported comet assay data for the same individuals. Our pioneering analysis underscores the sensitivity of FM-HCR assays for detecting subtle biological differences between individuals and establishes standardized methodologies for population studies. Our findings and open source analytical tools advance precision medicine by enabling comprehensive exploration of genetic, demographic, clinical, and lifestyle factors and supporting targeted interventions to enhance DNA repair and maintain genomic integrity, thereby promoting personalized healthcare and disease prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db3f634648b2c3c7a12699cca388efeca2e26cd" target='_blank'>
              Comprehensive Measurement of Inter-Individual Variation in DNA Repair Capacity in Healthy Individuals
              </a>
            </td>
          <td>
            Ting Zhai, Patrizia Mazzucato, Catherine Ricciardi, David C. Christiani, Liming Liang, Leona D Samson, I. A. Chaim, Z. Nagel
          </td>
          <td>2025-06-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="FOXA1 is altered in 10 to 40% of prostate cancers, yet its oncogenic mechanisms remain uncharacterized in vivo. We developed knock-in mouse models representing distinct classes of FOXA1 mutations. Histopathological and multi-omic analyses of prostate tissues and organoids revealed that Class 1 mutations, in conjunction with p53 inactivation, drive androgen-dependent adenocarcinomas through co-activation of mTORC1/2 and oncogenic AR signaling stemming from chimeric AR-half enhancers. In contrast, Class 2 mutations induce intra-luminal plasticity by reprogramming differentiated luminal cells into a progenitor-like state through activation of KLF5 and AP-1 neo-enhancer circuitries, which enables enhanced survival and proliferation even under castrate androgen levels. Our findings establish FOXA1 as a multifaceted oncogene, with distinct mutational classes divergently evolving to drive prostate tumorigenesis or therapy-resistant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877a995b94079d1a62d50951659ac85f4910e75e" target='_blank'>
              Divergent FOXA1 mutations drive prostate tumorigenesis and therapy-resistant cellular plasticity.
              </a>
            </td>
          <td>
            Sanjana Eyunni, R. Mannan, Yuping Zhang, E. Young, Qiuyang Zhang, Jie Luo, Matthew Pang, Somnath Mahapatra, J. Tien, James M. George, Mustapha Jaber, Hamzah Hakkani, Sandra E Carson, Abigail J Todd, Noshad Hosseini, M. Gondal, Ryan J. Rebernick, Xuhong Cao, Fengyun Su, Rui Wang, Rohit Mehra, Jing Li, Marcin P. Cieslik, A. Chinnaiyan, A. Parolia
          </td>
          <td>2025-06-26</td>
          <td>Science</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Blood cancers are one of the most common cancers worldwide. These diseases stem from defects in blood components having cytogenetic aberrations and genetic mutations. There have been vast improvements in terms of treatment options and survival outcomes. Nevertheless, due to the clonal nature and heterogeneity of the diseases, the number of cases reported exhibit a rising pattern due to chemoresistance and disease relapse thus posing a healthcare burden. Therefore, the need for more specific forms of targeted therapies is ever-present.


RECENT FINDINGS
CRISPR has emerged as a key player and is the epitome of gene editing technology in this post genomic era. In line with the current trend, numerous studies in blood cancer research have extensively utilized CRISPR-based applications to understand the functional genomics of hematologic malignancies and identify potential therapeutic targets for development of novel therapeutic applications.


SUMMARY
The importance of comprehending the utilities of state-of-the-art technologies such as CRISPR for studying hematologic malignancies has never been more apparent and timelier. Therefore, this review attempts to scrutinize the versatility of CRISPR applications which range from functional genomics to immunotherapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215dcf4713fb2244aa818902ca452469fccb308c" target='_blank'>
              Multifaceted roles of CRISPR technology in blood cancer research.
              </a>
            </td>
          <td>
            Maheswaran Solayappan, Adam Azlan, Kang Zi Khor, Mot Yee Yik, Aswani Jaishanker, Thevendran Ramesh, Mohamed Saleem, N. Yusoff, E. Moses
          </td>
          <td>2025-06-04</td>
          <td>Current opinion in hematology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer genomics has advanced a lot; a main problem is still turning molecular findings into treatments due to the variety of cells in each tumor and gaps in knowing certain pathway functions. Predicting whether a treatment will be effective is difficult right now which means that patient outcomes are often less than ideal. This research tackled the issue by studying whether looking at many genes and oncogenic pathways at the same time helps identify which patients may show cancer progression or treatment resistance. For this study, we looked at the genes, proteins and other molecules from a total of 1,500 patients in six major types of cancer (breast, colorectal, lung, melanoma, ovarian and prostate). We used genomic variant calling, analysis of gene pathways and detailed study of proteins with advanced bioinformatics and machine learning tools. The study found that TP53 mutations (OR: 2.14, 95% CI: 1.72-2.66, p < 0.001) and the PI3K/AKT/mTOR pathway being activated (OR: 1.89, 95% CI: 1.51-2.37, p = 0.003) were strongly linked to cancer not responding to treatment and negative outcomes. Tumors with a high number of mutations (≥ 10/Mb) reacted much better to immunotherapy (OR: 3.02, p < 0.001). Three subgroups of tumors were found through unsupervised analysis and these showed different chances of success: tumors with many mutations and high immune activity performed best (39.1% complete response), but those with mutations in PI3K and TP53 had the worst prognosis (28.5% progressive disease). Patients whose tumors were active in the PI3K/AKT pathway had progression-free survival of 8.2 months, significantly less (log-rank p < 0.001) than those whose tumors were not active. Evidence shows that integrative molecular profiling is better at predicting outcomes than the traditional method of classifying tumors by looking at slides. Study results show which molecular subgroups are most important to oncology and help link their treatment response to which therapy is best,">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e766178e1eebb59cae34bf9ab40a3c9e125bd68f" target='_blank'>
              Integrative Molecular Profiling of Oncogenic Pathways and Genetic Mutations in Cancer Progression and Therapeutic Response
              </a>
            </td>
          <td>
            Javeria Taj, Muhammad Noman Ajmal, Tayyaba Arshad, Afifa Dawood, Ali Abbas, Misbah Hafeez, Hamza Rafeeq, Abdul Malik, Aiman Nishat, Mujahid Hussain
          </td>
          <td>2025-06-20</td>
          <td>Scholars Academic Journal of Biosciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0e7e32e3545d727024253e68ada05454a1fe358" target='_blank'>
              The role of ASXL1, SRSF2, and EZH2 mutations in chromatin dysregulation of myelodysplastic neoplasia and acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hosang Yu, Junshik Hong, Dong-Yeop Shin, Chul-Hwan Lee
          </td>
          <td>2025-06-25</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Previous studies majorly focused on the role of driver gene variation in the occurrence, development and treatment of glioma. Here, we revealed the effect of signaling pathway abnormality by accumulated genetic mutations on patient survival and explored potential mechanism in glioma. We found that abnormalities of three signaling pathways, including autophagy-animal, focal adhesion and neuroactive ligand-receptor interaction, were associated with progression-free survival (PFS) in glioma. Based on the three pathways, we constructed and validated a biological pathway score (BPS) model to predict patient prognosis. Both PFS and overall survival (OS) were poorer in glioma patients with high BPS. In the glioma patients with high BPS, mutation frequencies of PTEN, TTN and EGFR were significantly increased. Immune checkpoints (PD-1, PD-L1 and TIM-3) were strikingly expressed in the high BPS patients with glioma, despite observed high levels of infiltrating immune cells. Differential expression genes in the high BPS patients were enriched in proliferation-related pathways. These results revealed potential reasons of poor prognosis in the high BPS patients with glioma. In addition, we found that low BPS patients with glioma might benefit from sevoflurane during surgery anesthesia. Overall, this study exhibited that the abnormalities of PFS-related signaling pathways by genetic mutations impacted survival of patients and revealed potential mechanism of poor PFS in glioma. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-12346-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c3a6594417935ac589e785911e02c60fd97e52" target='_blank'>
              Mutational accumulation drives pathway dysregulation to predict glioma patient survival
              </a>
            </td>
          <td>
            Mengjiao Wan, Caiping Zhuang, Wei Li, Rongrong Yan, Wen Chen, Jing Cheng, Chengjin Yue, Lin Chen, Yingyao Quan
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="To systematically investigate the molecular and pathological mechanisms of enhancer RNA (eRNA)–mediated transcriptional regulation in glioma recurrence and progression, transcriptomic, regulatome, and genomic data were integrated to analyze eRNA behavior in lower-grade gliomas (stages II/III) and glioblastomas (stage IV). Most eRNAs exhibited dynamic expression during glioma progression, regulated by master transcription factors (TFs) and affected by genomic mutations. The constructed perturbed TF-mediated eRNA-promoter regulatory landscape revealed that rewiring eRNA-promoter networks contributed to glioma malignancy. Drug response–related eRNAs associated with poor prognosis were identified, highlighting their clinical potential. Overall, integrative analysis highlights the critical role of eRNA-mediated regulatory rewiring in glioma progression, providing valuable insights into transcriptional mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b9a681aad2033d6a1c3522f91909b48b06e614" target='_blank'>
              Enhancer RNA–mediated transcriptional regulatory programs reveal the malignant progression of glioma
              </a>
            </td>
          <td>
            Yingying Ma, Shuyuan Wang, Yijie Jia, Shijie Wang, Wendian Liu, Jiaxin Yu, Jun Wang, Jingkai Geng, Jiayi Zhao, Zheng Zhao, Juan Xu, Hui Liu
          </td>
          <td>2025-06-06</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Personalized immunotherapy of pancreatic ductal adenocarcinoma (PDAC) through T-cell mediated targeting of tumor-specific, mutanome-encoded neoepitopes may offer new opportunities to combat this disease, in particular by countering recurrence after primary tumor resection. However, the sensitive and accurate calling of somatic mutations in PDAC tissue samples is compromised by the low tumor cell content. Moreover, the repertoire of immunogenic neoepitopes in PDAC is limited due to the low mutational load of the majority of these tumors. Methods We developed a workflow involving the combined analysis of next-generation DNA and RNA sequencing data from matched pairs of primary tumor samples and patient-derived xenograft models towards the enhanced detection of driver mutations as well as single nucleotide variants encoding potentially immunogenic T-cell neoepitopes. Subsequently, we immunized HLA/human T-cell receptor (TCR) locus-transgenic mice with synthetic peptides representing candidate neoepitopes, and molecularly cloned the genes encoding TCRs targeting these epitopes. Result Application of our pipeline resulted in the identification of greater numbers of non-synonymous mutations encoding candidate neoepitopes with increased confidence. Furthermore, we provide proof of concept for the successful isolation of HLA-restricted TCRs from humanized mice immunized with different neoepitopes, several of which would not have been selected based on mutanome analysis of PDAC tissue samples alone. These TCRs mediate specific T-cell reactivity against the tumor cells in which the corresponding mutations were identified. Conclusion Enhanced mutanome analysis and candidate neoepitope selection increase the likelihood of identifying therapeutically relevant neoepitopes, and thereby support the optimization of personalized immunotherapy for PDAC and other poorly immunogenic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74726107b1853e699ed8135ce18a77b1befecac" target='_blank'>
              Enhanced mutanome analysis towards the induction of neoepitope-reactive T-cell responses for personalized immunotherapy of pancreatic cancer
              </a>
            </td>
          <td>
            M. Volkmar, Dana Hoser, Claudia Lauenstein, Janina Rebmann, Agnes Hotz-Wagenblatt, Jan Rieger, I. Poschke, Jonas P. Becker, Angelika B. Riemer, M. Sprick, A. Trumpp, Oliver Strobel, Thomas Blankenstein, G. Willimsky, R. Offringa
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, E. Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Transposable elements (TEs) are scattered across mammalian genomes. Silencing of TEs prevents harmful effects caused by either global activation leading to genome instability or insertional mutations disturbing gene transcription. However, whether the activation of a TE can cause disease without directly affecting gene expression is largely unknown. Here we show that a TE insertion can adopt nearby regulatory activity, resulting in the production of cell-type-specific viral-like particles (VLPs) that affect embryo formation. Failure to silence an LTR retrotransposon inserted upstream of the Fgf8 gene results in their co-expression during mouse development. VLP assembly in the Fgf8-expressing cells of the developing limb triggers apoptotic cell death, resulting in a limb malformation resembling human ectrodactyly. The phenotype can be rescued by mutating the retrotransposon coding sequence, thus preventing its full endogenous retroviral cycle. Our findings illustrate that TE insertions can be incorporated into the local genomic regulatory landscape and that VLP production in post-implantation embryos can cause developmental defects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01aef940fe3e91fdc0987fa45fa3309b39304aae" target='_blank'>
              Enhancer adoption by an LTR retrotransposon generates viral-like particles, causing developmental limb phenotypes
              </a>
            </td>
          <td>
            Juliane Glaser, Giulia Cova, B. Fauler, C. Prada-Medina, Virginie Stanislas, M. H. Phan, R. Schöpflin, Yasmin Aktas, Martin Franke, G. Andrey, Natalia Bartzoka, C. Paliou, Verena Laupert, W. Chan, L. Wittler, T. Mielke, Stefan Mundlos
          </td>
          <td>2025-07-09</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Background: Hormone Receptor Positive (HR+) breast cancer is predominantly managed with endocrine therapies that target the Estrogen Receptor Alpha (ERα), a key mediator of estrogen's role in promoting tumor growth. However, the emergence of endocrine resistance poses a significant clinical challenge, limiting the efficacy of these treatments and impeding optimal patient outcomes. Understanding the mechanisms behind ERα's importance and the development of resistance is crucial for advancing therapeutic strategies against HR+ breast cancer. Methods: In our study, we utilized the GEO database to identify ADP-Ribosylation Factor-Like GTPase 3 (ARL3) as a gene associated with tamoxifen resistance in HR+ breast cancer. We constructed cell lines with ARL3 knockout and overexpression to assess the impact of ARL3 on cell proliferation, migration, and tamoxifen sensitivity. RNA sequencing (RNA-seq) and liquid chromatography-mass spectrometry (LC/MS) were employed to demonstrate the influence of ARL3 knockout on downstream ERα pathways. Furthermore, co-immunoprecipitation (CO-IP), immunofluorescence and Western blot (WB) assays confirmed the role of ARL3 in the ubiquitination and degradation pathway of ERα. Results: In our analysis of TCGA breast cancer subtypes, we also observed that ARL3 is highly expressed in luminal breast cancer. Additionally, our collection of breast cancer tissue mRNA and immunohistochemistry data revealed that ARL3 expression is elevated in HR+ breast cancer, indicating a correlation with estrogen receptors. We found that ARL3 promotes the proliferation and migration of HR+ breast cancer. Furthermore, overexpression of ARL3 was associated with reduced sensitivity to tamoxifen, and in vivo tumorigenesis in nude mice showed that tumor volume and weight were significantly decreased compared to the control group. To investigate the underlying mechanisms, RNA-seq analysis of ARL3 knockout and control groups revealed a pronounced inhibition of the MYC pathway. Western blot (WB) validation confirmed that ARL3 knockout suppresses the expression of downstream pathways of ERα. Our data suggest that the association between ARL3 and ERα primarily occurs in the cytoplasm, mainly binding to the ligand-binding domain (LBD) of ERα, which is consistent with ARL3 not being recruited to the promoters occupied by ERα. We also discovered that the ARL3/ERα cascade promotes mitochondrial autophagy to enhance mitochondrial oxidative phosphorylation. In summary, our study establishes a non-genomic mechanism whereby ARL3 controls the transcription of estrogen-dependent genes associated with breast cancer cell proliferation by stabilizing ERα levels. Conclusion: In our study, we posit that ADP-ribosylation factor-like GTPase 3 (ARL3) exerts a pivotal influence on the proliferative capacity, metastatic potential, and responsiveness to endocrine therapies in hormone receptor-positive (HR+) breast cancer. ARL3 modulates the stability of estrogen receptor alpha (ERα), thereby regulating the activation of downstream signaling cascades and the mitochondrial functionality within neoplastic cells. Our findings suggest that ARL3 represents a promising therapeutic target for HR+ breast cancer. We envision that in-depth investigation into the molecular underpinnings of ARL3's role could pave the way for the development of innovative pharmacological agents capable of overcoming endocrine resistance in patients, ultimately enhancing their clinical outcomes and survival rates.
 Citation Format: Han Li, Yang Liu, Zehao Cai, Dan Shu, Yang Peng, Kang Li, Shengchun Liu. ARL3 Promotes Hormone Receptor Positive Breast Cancer Progression and Tamoxifen Resistance through ERα Stablization [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-04-10.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aec8ed2ad5da8e6d551e26dd1027baa5c75f898" target='_blank'>
              Abstract P2-04-10: ARL3 Promotes Hormone Receptor Positive Breast Cancer Progression and Tamoxifen Resistance through ERα Stablization
              </a>
            </td>
          <td>
            Han Li, Yang Liu, Zehao Cai, Dan-hua Shu, Yang Peng, Kang Li, Shengchun Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 4530


 Background:
 There is an unmet need to characterize molecular drivers of resistance to ICIs in the RCC tumor microenvironment, especially in cell-type-specific contexts. To address this, we identified coherent gene programs in each cell type using scRNA-seq data to uncover an immune cell phenotype associated with ICI resistance.
 Methods:
 We performed per-cell-type weighted coexpression network analysis on scRNA-seq data of 443,337 cells from 70 RCC tumor samples, mostly from patients (pts) who received ICI (S. Kashima, ASCO, 2024), to construct mutually co-expressed gene sets (modules) and their cell-level expression. We compared module expression scores between responders (R, complete or partial response) and non-responders (NR, progressive disease) with the Wilcoxon rank-sum test plus FDR correction, and ran tests with permuted labels to rule out statistical artifacts. For validation, we performed this analysis on another scRNA-seq cohort from a phase II trial (HCRN GU16-260; NCT03117309), and further, we used CoxPH survival analysis on module signature scores computed by GSVA to assess the prognostic role of module expression in bulk RNA-seq samples from the IMmotion 150, CheckMate 009/010/025, and Javelin 101 trials, after stratifying them into immune-high (top 50%) and immune-low (bottom 50%) groups according to CIBERSORTx-inferred total immune infiltration.
 Results:
 Analysis of scRNA-seq (two independent datasets) from ICI-treated RCC tumors revealed a module of robustly co-expressed ribosomal and translation-associated genes that was significantly upregulated in NR in immune cells (but not tumor or stromal cells), including macrophages, CD4+ and CD8+ T, NK, and B cells (p-values ranging from 1e-20 to 1e-200). Analysis of an independent scRNA-seq cohort (HCRN GU16-260) also yielded a module of mutually co-expressed ribosomal proteins only in immune cells that was upregulated in NR (immune p < 1e-20, non-immune p > .05). Finally, validation in large-scale bulk RNA-seq data from clinical trials shows that for patients receiving any ICI-based therapy (nivolumab, avelumab + axitinib, or atezolizumab +/- bevacizumab), module signature scores were significantly associated with worse PFS only in the CoxPH analysis of immune-high samples (HR = 1.65, 95% CI, 1.18–2.3, p < .005), while immune-low samples showed no effect (HR = 0.93, 95% CI 0.71–1.25, p > .5), concordant with findings from the scRNA-seq analysis. Those receiving a TKI only (sunitinib) exhibited no association, regardless of immune infiltration.
 Conclusions:
 Though individual ribosomal proteins have been found to be prognostic for RCC, a cell-type-specific module-level analysis elucidated a link between a coherent translation program in immune cells and resistance and poor PFS for pts receiving ICI specifically, contributing to a model of ICI-resistant molecular phenotypes in RCC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20a73336f21776051cca93529e3500b9abff72ac" target='_blank'>
              Co-expression network-based analysis of gene programs contributing to immune checkpoint inhibitor (ICI) resistance in renal cell carcinoma (RCC).
              </a>
            </td>
          <td>
            Ro Malik, Rishabh Rout, S. Kashima, E. Saad, Harry Kane, Valisha Shah, M. Hugaboom, Zhaochen Ye, N. Schindler, A. Dighe, Maxine Sun, G. M. Lee, Wenxin Xu, S. Signoretti, B. McGregor, Rana R McKay, Michael B. Atkins, Eliezer M Van Allen, T. Choueiri, D. Braun
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Cancer remains one of the most formidable global health challenges due to its complex pathogenesis, late-stage diagnosis, and limited therapeutic options for many subtypes. Despite advancements in reducing cancer-related mortality through lifestyle modifications and early interventions, it continues to rank among the top ten causes of death worldwide. A growing body of evidence highlights the critical role of non-coding RNAs, particularly microRNAs (miRNAs), in cancer biology. MiRNAs are short, non-coding RNA molecules approximately 19–25 nucleotides in length that regulate gene expression at the post-transcriptional level. Among these, miRNA-1303 has garnered increasing attention due to its aberrant expression across a variety of malignancies and its involvement in diverse cellular processes, including cell proliferation, apoptosis, migration, invasion, and chemoresistance. In this review, we provide a comprehensive overview of miRNA-1303, emphasizing its dysregulation in different cancer types such as colorectal, breast, prostate, and lung cancers. We explore the molecular mechanisms and signaling pathways modulated by miRNA-1303, including its interaction with oncogenes, tumor suppressors, and key cellular networks like PI3K/AKT, Wnt/β-catenin, and MAPK pathways. We also discuss its functional role in tumor development and progression, supported by both in vitro and in vivo studies. Furthermore, we evaluate the potential of miRNA-1303 as a clinically relevant biomarker for early cancer detection, prognosis prediction, and treatment monitoring. Its expression patterns have been correlated with tumor stage, metastasis, and patient survival, suggesting promising utility in clinical decision-making. In addition, we explore the emerging therapeutic strategies targeting miRNA-1303, including miRNA mimics, inhibitors, and delivery systems designed to modulate its expression in cancer cells. By synthesizing current findings, this review underscores the clinical significance of miRNA-1303 in oncology and proposes future directions for its application in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc4113503577ac3e30314feefc6d7bf89e1993d" target='_blank'>
              MicroRNA-1303 in cancer pathogenesis and therapy: clinical implications for biomarker development and targeted treatment strategies
              </a>
            </td>
          <td>
            Lingzi Zheng, Ling Hu, B. Badehnoosh
          </td>
          <td>2025-07-11</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="While somatic variants are well-characterized drivers of tumor evolution, their influence on cellular fitness in nonmalignant contexts remains understudied. We identified a mosaic synonymous variant (m.7076A > G) in the mitochondrial DNA (mtDNA)-encoded cytochrome c-oxidase subunit 1 (MT-CO1, p.Gly391=), present at homoplasmy in 47% of immune cells from a healthy donor. Single-cell multiomics revealed strong, lineage-specific selection against the m.7076G allele in CD8+ effector memory T cells, but not other T cell subsets, mirroring patterns of purifying selection of pathogenic mtDNA alleles. The limited anticodon diversity of mitochondrial tRNAs forces m.7076G translation to rely on wobble pairing, unlike the Watson-Crick-Franklin pairing used for m.7076A. Mitochondrial ribosome profiling confirmed stalled translation of the m.7076G allele. Functional analyses demonstrated that the elevated translational and metabolic demands of short-lived effector T cells (SLECs) amplify dependence on MT-CO1, driving this selective pressure. These findings suggest that synonymous variants can alter codon syntax, impacting mitochondrial physiology in a cell type-specific manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77e5c9949a4e7b360acc964029cfc77cf08593d" target='_blank'>
              Cell type-specific purifying selection of synonymous mitochondrial DNA variation.
              </a>
            </td>
          <td>
            Caleb A. Lareau, Patrick Maschmeyer, Yajie Yin, Jacob C. Gutierrez, R. Dhindsa, Anne-Sophie Gribling-Burrer, Sebastian Zielinski, Yu-Hsin Hsieh, Lena Nitsch, Veronika Dimitrova, Benan N. Nalbant, Frank A Buquicchio, Tsion Abay, Robert R. Stickels, Jacob C. Ulirsch, Patrick K. Yan, Fangyi Wang, Zhuang Miao, Katalin D. Sandor, Bence Daniel, Vincent Liu, Paul-Lennard Mendez, Petra Knaus, Manpreet Meyer, William J. Greenleaf, Anshul Kundaje, R. Smyth, Mathias Munschauer, Leif S. Ludwig, Ansuman T. Satpathy
          </td>
          <td>2025-07-29</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Background Resveratrol (RSV) is one of the most studied and used biomolecules, for which many pharmacological effects targeting multiple tissues have been described. However, a common underlying mechanism driving its full pharmacological activity has not been described in detail. G-quadruplexes (G4s) are non-canonical nucleic acid structures found in regulatory genomic locations and involved in controlling gene transcription, telomere maintenance, or genome stability, among others. This study provides a genome-wide characterization of RSV G4-binding properties, explaining its multi-target traits. Methods Immunofluorescence assays using a nucleolar and a G4-specific antibody were used to characterize RSV cellular effects on the nucleolus and G4 stabilization. DNA damage and cell cycle analyses were performed via western blot and flow cytometry. Breaks lLbeling In Situ and Sequencing (BLISS) was used to map double strand breaks (DSB) in response to treatment, and identify G4s targeted by RSV. mRNA sequencing was used to identify changes at the transcriptional level upon treatment and relate them to a direct targeting of G4s. Biophysical assays (circular dichroism, ultraviolet–visible [UV–Vis] titration, differential scanning calorimetry, and nuclear magnetic resonance) were used to characterize RSV–G4 interactions. Lastly, luciferase-based transcription assays were performed to confirm RSV–G4 interaction in vitro and its direct influence on gene expression. Results In a cellular context, RSV treatment showed classic G4-ligand effects, such as nucleolar disassembly, inhibition of RNA polymerase I, DNA damage, and cell cycle arrest. RSV was shown to stabilize cellular G4s, which accumulated around double strand breaks in the promoters of differentially expressed genes. Upon treatment, G4 stabilization triggered DNA damage and controlled gene expression. The interaction between RSV and target G4s was confirmed in vitro by biophysical assays and through luciferase-based transcription assays. Conclusions A G4-dependent mode of action was demonstrated as the main mechanism underlying RSV pleiotropic effects, along with the identification of target genes and G4s. This in-depth analysis of the mode of action of RSV will be helpful to improve its therapeutic potential in a wide variety of health scenarios. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00747-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309b9a5bcda358645827e337bc4418daea949d2f" target='_blank'>
              In-depth analysis of the mode of action of resveratrol: genome-wide characterization of G-quadruplex binding properties
              </a>
            </td>
          <td>
            Ana Soriano-Lerma, Victoria Sánchez-Martín, Javier Murciano-Calles, Matilde Ortiz-González, Maria J. Tello-Lopez, Virginia Pérez-Carrasco, Ángel Linde-Rodríguez, Inmaculada Ramírez-Macías, Irene Gómez-Pìnto, Inmaculada López-Aliaga, M. Soriano, Jose A. Garcia Salcedo
          </td>
          <td>2025-06-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 e17569


 Background:
 Molecular profiling of tumors using Next Generation Sequencing (NGS) enables the detection of genetic alterations associated with response to targeted therapies, as well as the assessment of genomic scars indicative of homologous recombination deficiency (HRD). HRD serves as a critical biomarker for predicting response to PARP inhibitors. In this study, the efficiency of a 1021-gene NGS panel was evaluated for HRD status analysis in comparison with validated methodologies in ovarian cancer tissues.
 Methods:
 A total of 43 tumor samples from 34 patients with known HRD status, based on validated Myriad and OncoScan methodologies, were analyzed. The NGS-based analysis (Gene+) included sequencing of 1021 genes and evaluation of genomic instability. Genomic instability was assessed using three key biological parameters: LOH (Loss of Heterozygosity), TAI (Telomeric Allelic Imbalance), and LST (Large-Scale Transitions). Key evaluation criteria included HRD score concordance, BRCA1/2 mutation detection, and methodology reproducibility.
 Results:
 HRD status showed a concordance rate of 97.1% (33/34) between the 1021+HRD panel and the validated Myriad test (Table 1). Discrepancies were observed in one sample, with an HRD score of 43 based on the 1021-gene panel, compared to 35 (Myriad), suggesting potential differences near the cutoff threshold. Sensitivity for BRCA1/2 mutation detection was 100%. Additionally, reproducibility of HRD detection was 100% across 12 repeated samples. Positive HRD scores remained consistent even at low TCC levels (≥20%), with an agreement rate of 95.7% (22/23).
 Conclusions:
 The new NGS methodology demonstrated high concordance with validated methodologies for HRD detection, robust BRCA mutation identification, and excellent reproducibility. These findings support its clinical utility for guiding therapeutic decisions with PARP inhibitors.


 Concordance between the validated NGS method and the 1021-gene pPanel.






 Validated HRD Test





 HRD +


 HRD -


 PPV


 NPV


 OPA


 r





 1021+HRD

 HRD +

 24
 1
 96.00%
 100%
 97.06%
 0.9295


 HRD -
 0
 9
 p< .00001




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a8eda4aba208d5d448ccbe5c5479df09a876f7a" target='_blank'>
              Evaluation of a 1021-gene NGS panel for HRD detection in clinical samples.
              </a>
            </td>
          <td>
            E. Fountzilas, E. Papadopoulou, V. Metaxa-Mariatou, Stella Maxouri, Aikaterini Tsantikidi, Chrysiida Chatzigiannidou-Florou, Konstantinos Papazisis, Christos Papadimitriou, T. Floros, M. Liontos, A. Bokas, Eleni Timotheadou, Anastasios L. Boutis, Prokopios Dimopoulos, Xiaorui Fu, Yun Xing, Xunmei Zheng, Xinhua Du, G. Nasioulas
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 e14571


 Background:
 Homozygous deletion of the MTAP occurs in 10-15% of all human cancers. MTAP catalyzes the conversion of MTA (S-adenosylmethionine) into methionine, playing a role in cellular energy metabolism and protein synthesis. Currently, numerous PRMT5 inhibitor drugs are being developed. Accurately detecting the status of MTAP is particularly important. Until recently, fluorescence in situ hybridization (FISH) served as the gold standard for detection of 9p21 loss. However, there is an increasing demand for NGS-based of MTAP deletion detection due to its ease of application and cost-effectiveness. Therefore, we have developed a panel-based detection method that can accurately determine deletion status of MTAP.
 Methods:
 In this study, we utilized a NGS panel that covers more than 2000 SNP loci within the genome. The sequencing depth was greater than 500X. To obtain the absolute copy number of the tumor, an algorithm was employed to correct the log2Ratio for each bin by utilizing the offset between tumor purity and log2Ratio. To validate the accuracy of this method, clinical samples from 41 cases were selected for simultaneous FISH testing and the NGS panel detection mentioned above. An additional 1260 clinical samples were selected for NGS testing to confirm the incidence of all exon deletions and partial loss.
 Results:
 All samples were successfully detected. Among 41 cases, FISH and NGS accurately identified 28 homozygous deletions and 11 heterozygous deletions; 2 samples incorrectly identified. The overall consistency in status determination reached 95% (39/41). Using the above algorithm to evaluation additional samples, 17% of the deletions can be accurately identified, including 60.7% of homozygous deletion. The detection rates of MTAP homozygous deletion in different cancer types are similar to the TCGA data. Specifically, it was observed that non-small cell lung cancer had a frequency of 16.7%, head and neck cancer of 17.3%, urothelial carcinoma of 40.4%, colorectal of 3.0%, pancreatic of 18.0%, glioma of 21.0%, and esophageal of 19.5%. The algorithm can accurately categorize all identified deletions into two groups: 13.8% complete gene deletions and 3.2% partial exon deletions. The multiple exon loss frequcies inclued exon 5-8 loss in 2.8% , exon 6-8 loss in 1.4%, exon 7-8 loss in 4.7% , exon 8 loss in 7.5% and other less common multi-exon losses.
 Conclusions:
 The NGS-based method established in this study for detecting MTAP deletion demonstrates a high consistency of 95% with FISH. Furthermore, it shows superior performance to FISH in detecting partial exon loss. This research method holds potential for application in companion diagnostics for drug therapy.


 Frequency of homozygous deletion in different cancers.




 Cancer
 Complete exons loss
 Partial exons loss




 NSCLC
 15.5%
 1.2%


 HNC
 5.8%
 11.5%


 UC
 38.6%
 1.8%


 CRC
 0.7%
 2.3%


 PAAD
 14.0%
 4.0%


 GBM
 14.3%
 6.7%


 ESCA
 15.2%
 4.3%




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b320e33960c69348661dfe1c7049c3f5b27cc56b" target='_blank'>
              The performance of MTAP homozygous deletion detection based on next-generation sequencing technology.
              </a>
            </td>
          <td>
            Lu Meng, Yan Zhang, Xinhua Du, Shicheng Feng, Heyu Sheng, Yun Xing, Yadi Cheng, Yanfang Guan, Xin Yi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 8523


 Background:
 Driver-negative lung adenocarcinoma (DN-LUAD) without actionable genomic alterations (AGAs) has a poor prognosis. A deeper understanding of its molecular background, particularly across the entire genome, is crucial for improving risk assessment and therapeutic development.
 Methods:
 We performed deep whole-genome sequencing (WGS; tumor depth: 100–120x, normal depth: 30x, paired-end reads: 150 bp) and RNA sequencing using fresh-frozen tissues from four institutions. DN-LUAD was defined as LUAD without known actionable mutations or gene fusions. We analyzed somatic mutations, copy number alterations (CNAs), and structural variations (SVs). Tumors were classified as whole-genome doubling (WGD) if more than half of the autosomal tumor genome showed at least two copies in the major copy numbers of somatic cells. HRDetect score ≥0.7 was used to determine the presence of homologous recombination DNA repair deficiency (HRD) (Davies H, et al. Nat Med. 2017). Gene Set Enrichment Analysis (GSEA) was used for pathway analysis.
 Results:
 Among the 745 patients (pts), 517 were classified as having DN-LUAD. WGS identified AGAs undetected by whole-exome sequencing (WES) in 33 (4.4%) pts (
 EGFR
 13,
 KRAS
 7,
 BRAF
 5,
 ERBB2
 4,
 MET
 4, and
 HRAS
 1). The frequently observed genomic alterations in DN-LUAD are shown in the Table. WGD was observed in 62.1% (n = 321) of DN-LUADs and was significantly associated with higher TMB, CNA, and SV burden. Tumor suppressor gene (TSG) mutations in
 TP53
 ,
 STK11
 , and
 KEAP1
 , as well as
 CDKN2A
 copy number loss and SVs, were significantly more frequent in WGD pts. HRD was identified in 19 DN-LUAD pts (3.7%), with a significantly higher frequency in the WGD group (WGD vs. without WGD; 4.98% vs. 1.5%, p = 0.016). GSEA showed significant (q < 0.0001) upregulation of cell cycle pathways (E2F targets, G2M checkpoint, and MYC targets) and downregulation of immune pathways (allograft rejection and interferon-gamma response) in DN-LUADs with WGD.
 Conclusions:
 This largest-ever WGS study identified AGAs undetectable by WES and uncovered a subgroup of DN-LUAD characterized by increased genomic instability driven by multiple TSG alterations associated with WGD, which was also more likely to exhibit HRD. WGD was associated with the upregulation of cell cycle pathways and the downregulation of immune pathways. These findings highlight the critical role of WGS in elucidating the pathogenesis of DN-LUAD.





 AllN = 517
 WGDn = 321
 Without WGDn = 196

 p





 Median TMB (Mutations/Mb)
 6.6
 10.2
 3.0
 <0.0001


 Median No. of CNAs
 101
 117
 77
 <0.0001


 Median No. of SVs
 185
 244
 103
 <0.0001



 TP53

 381 (74)
 277 (86)
 104 (53)
 <0.0001



 SMARCA4

 59 (11)
 43 (13)
 16 (8)
 0.094



 STK11

 56 (11)
 42 (13)
 14 (7)
 0.049



 CDKN2A

 44 (9)
 33 (10)
 11 (6)
 0.092



 KEAP1

 40 (8)
 34 (11)
 6 (3)
 0.003



 CDKN2A
 loss

 106 (21)
 78 (24)
 28 (14)
 0.009



 MET
 amplification

 44 (9)
 34 (11)
 10 (5)
 0.045



 CDKN2A

 135 (26)
 95 (30)
 40 (20)
 0.028



 FHIT

 125 (24)
 99 (31)
 26 (13)
 <0.0001




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b5bbd4f2e96da01654006c7177d2c691759a6f" target='_blank'>
              Whole-genome and transcriptome landscape of actionable driver-negative lung adenocarcinoma.
              </a>
            </td>
          <td>
            Masahiro Torasawa, K. Shiraishi, Akifumi Mochizuki, Shingo Matsumoto, Junko Hamamoto, Hirokazu Matsushita, Issei Imoto, Hiroyuki Yasuda, Kazuhisa Takahashi, Yuichi Shiraishi, Masahiro Tsuboi, Koichi Goto, Shun-Ichi Watanabe, R. Hamamoto, T. Kohno
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The problem of interpretation of the genetic data from patients with inherited cardiovascular diseases still remains relevant. To date, the clinical significance of approximately 40 % of variants in genes associated with inherited cardiovascular diseases is uncertain, which requires new approaches to the assessment of their pathogenetic contribution. A combination of the induced pluripotent stem cell (iPSC) technology and editing the iPSC genome with CRISPR/Cas9 is thought to be the most promising tool for clarifying variant pathogenicity. A variant of unknown significance in MYH7, p.Met659Ile (c.1977G>A), was previously identified in several genetic screenings of hypertrophic cardiomyopathy patients. In this study, the single nucleotide substitution was corrected with CRISPR/Cas9 in iPSCs generated from a carrier of the variant. As a result, two iPSC lines (ICGi019-B-1 and ICGi019-B-2) were generated and characterized using a standard set of methods. The iPSC lines with the corrected p.Met659Ile (c.1977G>A) variant in MYH7 possessed a morphology characteristic of human pluripotent cells, expressed markers of the pluripotent state (the OCT4, SOX2, NANOG transcription factors and SSEA-4 surface antigen), were able to give rise to derivatives of three germ layers during spontaneous differentiation, and retained a normal karyotype (46,XY). No CRISPR/Cas9 off-target activity was found in the ICGi019-B-1 and ICGi019-B-2 iPSC lines. The maintenance of the pluripotent state and normal karyotype and the absence of CRISPR/Cas9 off-target activity in the iPSC lines with the corrected p.Met659Ile (c.1977G>A) variant in MYH7 allow using the iPSC lines as an isogenic control for further studies of the variant pathogenicity and its impact on the hypertrophic cardiomyopathy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cb56531c5e4ea40f0f8c8f05402c5581e5ae495" target='_blank'>
              Generation of the ICGi019-B-1 and ICGi019-B-2 lines via correction of the p.Met659Ile (c.1977G>A) variant in MYH7 of patient-specific induced pluripotent stem cells using CRISPR/Cas9
              </a>
            </td>
          <td>
            A. E. Shulgina, S. Pavlova, J. Minina, S. Zakian, E. V. Dementyeva
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="MYC is an aberrantly regulated transcription factor implicated in approximately 70% of human tumors, where it extensively modulates signaling pathways associated with cancer progression. Inactivating MYC has been shown to inhibit tumor growth and even induce sustained tumor regression across various cancer types, a phenomenon referred to as oncogene addiction. However, in vitro studies reveal that the knockout or knockdown of MYC in numerous tumor cell lines does not necessarily result in cell death, despite these tumors exhibiting MYC addiction in vivo. This discrepancy suggests that the unique tumor microenvironment in vivo may play a critical role in facilitating MYC addiction in cancer cells. MYC is also widely acknowledged for its role in mediating the immune evasion of tumor cells. Nevertheless, due to the extensive regulation of cellular gene expression by MYC and the incomplete understanding of the mechanisms underlying tumor immune escape, the precise pathways through which MYC influences tumor immune evasion remain inadequately elucidated. Recent years have seen the identification of novel tumor immune escape mechanisms, some of which have been demonstrated to be directly or indirectly regulated by MYC. For instance, MYC may contribute to immune evasion by modulating the expression of argininosuccinate synthetase 1 (ASS1), a key enzyme involved in arginine biosynthesis. Herein, in this study, we explore some novel potential mechanisms through which MYC facilitates the immune evasion of tumor cells, alongside a combined therapeutic approach targeting MYC and employing immunotherapy based on this mechanism. Furthermore, we suggest that targeting proteins interacting with MYC to modulate its expression and function may serve as an alternative strategy to direct MYC targeting, thereby expediting the clinical translation of combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61614de2defb80c90945b2feff409bc535c7b7f8" target='_blank'>
              Beyond the Limit: MYC Mediates Tumor Immune Escape
              </a>
            </td>
          <td>
            Zhongyang Hong, Sitong Ming, Xin Luan, Zhe Sun, Weidong Zhang
          </td>
          <td>2025-06-29</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H. Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a significant cause of male mortality worldwide. Since metastases are the underlying cause of lethality, identifying markers for metastatic potential would be highly valuable. To address this issue, we set out to identify protein-coding genes with metastasis-specific expression changes in PCa. We employed a previously reported unique cohort consisting of metastases from castration-resistant prostate cancer (mCRPC) and matching primary tumors. Our comprehensive gene expression analyses identified 85 differentially expressed genes (DEGs) associated specifically with mCRPC, comprising 63 upregulated and 22 downregulated genes. Investigation of the transcription factors (TFs), such as the androgen receptor and its co-regulators FOXA1 and HOXB13, known to be important in prostate tumorigenesis, revealed their involvement in the differential expression of these genes. Furthermore, we identified enriched binding sites for nine TFs, namely EZH2, SUZ12, TLE3, TP63, CBX7, RNF2, SP140, JARID2, and CBX8, in the regulatory regions of the DEGs. Analysis of progression-free survival of prostatectomy-treated men highlighted 16 DEGs with significant prognostic value. Of these, three genes (FRMPD1, TMEM18, and ZNHIT3) were independent prognostic markers of biochemical recurrence. TMEM18 has putative androgen receptor-binding sites in its promoter region, and analysis of LNCaP cells following stimulation with dihydrotestosterone revealed a significant upregulation of TMEM18, confirming the androgen regulation of the gene. Furthermore, we confirmed the prognostic significance of TMEM18 expression at the protein level with immunohistochemistry (IHC) in a primary PCa tumor cohort. In conclusion, we identified 85 mCRPC-associated genes and showed that TMEM18 has prognostic value in early PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1055b33e266f652109e7e688c5916a5b1312690" target='_blank'>
              Identification of protein-coding genes associated with metastatic prostate cancer
              </a>
            </td>
          <td>
            M. Sattari, Hanna E. Rauhala, Leena Latonen, William B. Isaacs, M. Nykter, G. Bova, J. Kesseli, T. Visakorpi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-Related Cancer</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 6528


 Background:
 NPM1
 GA characterize a clinically important subset of AML cases which relapse in more than 50% of treated patients despite being generally sensitive to conventional chemotherapy regimens. In AML, the interactions between GA in
 NPM1
 ,
 KMT2A
 and menin protein have been linked to leukemogenesis and represent new potential targets for anti-tumor therapies.
 Methods:
 4,206 cases of AML underwent comprehensive genomic profiling from 2019 through 2024, using the FoundationOne Heme combined hybrid capture based DNA and RNA sequencing assay. All classes of relevant GA were evaluated. The tumor mutation burden (TMB), homologous recombination deficiency signature (HRDsig) and microsatellite Stability (MSS) status were determined from the sequenced data.
 Results:
 633 (15.1%) of the 4,206 AML featured
 NPM1
 GA (NPM1mut). Short variant mutations were found in >99% of the NPM1mut AML with the W288fs*12 frameshift base substitution accounting for 92.4% of cases. An
 NPM1
 -
 MLF1
 fusion was identified in 1.3% of NPM1mut cases. The NPM1mut+ were more frequently associated with female patients (53.4% vs 41.5%; p<.0001) and had a slightly higher median age compared to the NPM1 wild type (NPM1wt) AML patients (62yrs vs 60yrs; p<.0001). Majority of patients (>60%) were from European decent. There were greater NPM1 GA in patients with European (77.1% vs 68.5%; p<.0001) and lower with African ancestry (9.2% vs 10.2%; p<.0001). MSI High (0% in both groups) status, HRDsig+ (0—0.1%) and elevated TMB (median < 1 mutation/Mb) were extremely uncommon in both groups. GA more frequent in NPM1mut AML compared to the NPM1wt AML cohort included
 DNMT3A
 (39.2% vs 12.6%; p<.0001),
 FLT3
 (54.5% vs 14.7%; p<.0001)
 ,
 IDH1
 (16.1% vs 5.6%; p<.0001),
 IDH2
 (19.0% vs 9.0%; p<.0001),
 TET2
 (23.4% vs 13.5%; p<.0001) and
 WT1
 (12.5% vs 9.4% p=.02). GA more frequent in NPM1wt AML included
 ASXL1
 (17.1% vs 3.6%; p,.0001),
 BCOR
 (7.5% vs 1.6%; p<.0001),
 KMT2A
 (14.7% vs 0.2%; p<.0001),
 RUNX1
 (22.5% vs 1.9%; p-,.0001),
 STAG2
 (6.9% vs 1.6%; p<.0001) and
 TP53
 (19.1% vs 4.1%; p<.0001).
 Conclusions:
 The development of menin inhibitors has recently identified GA in
 NPM1
 as a promising target of therapy for AML patients. Other therapy targets in AML such as
 FLT3
 and
 IDH1/2
 are more frequently identified in NPM1mut than NPM1wt AML, while
 KMT2A
 is more frequently identified in NPM1wt AML. This genomic landscape study reveals significant differences in important GA associated with AML in NPM1mut and NPM1wt cases which may enrich our understanding of the molecular profile in AML and identify additional targets for therapy.


 Pathogenic genomic alterations in NPM1mut and NPM1wt AML.





 NPM1wt AML (n=3573)
 NPM1mut AML (n=633)
 P-value





 ASXL1
 17.1%
 3.6%
 <.0001



 CEBPA
 6.4%
 8.2%
 NS



 DNMT3A
 12.6%
 39.2%
 <.0001



 FLT3
 14.7%
 54.5%
 <.0001



 IDH1
 5.6%
 16.1%
 <.0001



 IDH2
 9.4%
 19.0%
 <.0001



 KMT2A
 14.7%
 0.2%
 <.0001



 RUNX1
 22.5%
 1.9%
 <.0001



 TET2
 13.5%
 23.4%
 <.0001



 TP53
 19.1%
 4.1%
 <.0001





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f4a065a163ca2b1d9ef8d78ae544e2bc89a03fb" target='_blank'>
              Nucleophosmin (
 NPM1
 ) genomic alterations (GA) in acute myeloid leukemia (AML): A genomic landscape study.
              </a>
            </td>
          <td>
            Osama Batayneh, Alexandra Goodman, Devashish Desai, D. Pavlick, Chelsea Marcus, Caleb Ho, J. Ross, Teresa Gentile, Zheng Zhou, Krishna Ghimire
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background CRISPR interference (CRISPRi), the repurposing of the RNA-guided endonuclease dCas9 as a programmable transcriptional repressor, allows highly specific repression (knockdown) of gene expression. CRISPRi platforms can often have incomplete knockdown, performance variability across cell lines and gene targets, and inconsistencies dependent on the guide RNA sequence employed. Results Here, we explore the combination of novel repressor domains with strong Krüppel-associated box (KRAB) repressors, screening > 100 bipartite and tripartite fusion proteins for their ability to reduce gene expression as CRISPRi effectors. We show that these novel repressor fusions have reduced dependence on guide RNA sequences, better slow cell growth when used to knock down expression of essential genes, and function in either fusion or scaffold modalities. Furthermore, we isolate and characterize a particularly effective CRISPRi platform, dCas9-ZIM3(KRAB)-MeCP2(t), which shows improved gene repression of endogenous targets both at the transcript and protein level across several cell lines and when deployed in genome-wide screens. Conclusions We posit that these novel repressor fusions can enhance the reproducibility and utility of CRISPRi in mammalian cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03640-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/541b174ba857f7a16f7cb6e09012bb05165641ba" target='_blank'>
              Engineering novel CRISPRi repressors for highly efficient mammalian gene regulation
              </a>
            </td>
          <td>
            Andrew J Kristof, Krithika Karunakaran, Christopher Allen, Paula Mizote, Sophie Briggs, Zixin Jian, Patrick Nash, J. Blazeck
          </td>
          <td>2025-06-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lung cancer is among the leading causes of cancer-related death in the U.S. Cigarette smoking remains the leading risk factor for lung cancer and can cause somatic mutations in the critical tumor suppressor gene TP53, among others. Mutations in TP53 not only cause loss of wild-type function but may also introduce oncogenic, gain-of-function (GOF) properties. The frequency of missense mutations at residues p.V157 and p.R158 in p53 increases dramatically in lung cancer relative to other cancers. These p53 mutants exhibit both loss of wild-type p53 function and GOF properties, including broadly rewiring gene expression programs in lung cancer cells. Several pan-cancer hotspot mutations in p53 impart GOF activities that reprogram cellular metabolism. To refine our understanding of the GOF properties conferred by the lung-enriched p53 mutants, the cellular metabolism of cells containing mutant p53 (V157F) was investigated. Untargeted metabolomics revealed that glutathione metabolism is among the top altered metabolic pathways related to mutant p53 (V157F). p53 mutants V157F and R158L provided resistance to oxidative stress induced by both menadione and cigarette smoke extract. The cell death experienced in the absence of mutant p53 (V157F; R158L) was due to the increase in reactive oxygen species (ROS). These findings suggest that the lung-enriched mutations in p53 (V157F; R158L) confer lung cancer cells with resistance to ROS, and ROS accumulation in the absence of mutant p53 causes cell death.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce8d3955759f3ec29e6914d7584360c523775fbf" target='_blank'>
              Somatic p53 mutations that are markedly overrepresented in lung cancer confer resistance to ROS-induced cell death.
              </a>
            </td>
          <td>
            Mason A Tracewell, Jonathan Karlin, Samantha M. Barnada, Elizabeth L McDuffie, Charles P Scott, J. Barta, Steven B. McMahon
          </td>
          <td>2025-06-03</td>
          <td>Carcinogenesis</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 e17084


 Background:
 Six-Transmembrane Epithelial Antigen of the Prostate 1 (STEAP1) is a transmembrane protein frequently overexpressed in prostate cancer (PC). Antibody-based therapies targeting STEAP1 have shown promising preclinical and clinical outcomes. Despite its widespread and PC-specific overexpression, the molecular regulation and biological function of STEAP1 remain poorly understood. Elucidating the mechanisms that govern STEAP1 expression and its role in PC progression could provide valuable insights into disease mechanisms, improve patient selection for STEAP1-targeted therapies, and inform combination or sequential treatment strategies.
 Methods:
 To investigate STEAP1 regulation and function, we first compared its expression to established PC driver genes and observed a correlation between STEAP1 and the androgen receptor (AR) as well as several AR-regulated genes. Using pathway analyses in the Decipher (localized prostate cancer) and West Coast Dream Team (WCDT, metastatic prostate cancer) clinical cohorts, we examined dysregulated pathways in patients with high STEAP1 expression. Additionally, we performed a FACS-based whole-genome CRISPR interference (CRISPRi) screen in C4-2B cells using an anti-STEAP1 antibody to identify regulators by comparing STEAP1-high and STEAP1-low expressing subsets.
 Results:
 In the WCDT dataset, STEAP1 was highly expressed in AR-positive metastatic prostate cancer subtypes, irrespective of neuroendocrine differentiation, but was significantly lower in AR-negative subtypes. STEAP1 expression strongly correlated with several AR-regulated genes, including STEAP2, KLK2, NKX3-1, KLK3, ARHGAP6, SLC45A3, and PMEPA1. Pathway analyses in both localized and metastatic PC cohorts revealed dysregulated metabolic pathways, particularly lipid metabolism, in STEAP1-high patients. The CRISPRi screen identified several potential targetable regulators of STEAP1, including MYC, SWI/SNF chromatin remodeling complexes.
 Conclusions:
 Our study establishes a strong association between STEAP1 and AR-driven metastatic castration-resistant prostate cancer subtypes. Furthermore, patient data suggest that STEAP1 is linked to dysregulated metabolic pathways, particularly those involving lipid metabolism. The identification of STEAP1 regulators through the CRISPRi screen provides valuable insights into its regulation and biological function, offering potential avenues for developing combination therapies, strategies to prevent treatment resistance, and improved patient stratification for STEAP1-targeted treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e2f4b4ddfd1356bdf18bff71926fd89045fb54e" target='_blank'>
              FACS-based whole-genome CRISPRi screen to uncover STEAP1 regulators in prostate cancer.
              </a>
            </td>
          <td>
            Shan Zhu, Mikhail Dias, Yuxin Yang, Emma Dolan, Thaidy Moreno Rodriguez, Tamilla Nechiporuk, Jun Zhu, Derek Xiang, Tianyi Liu, Tatyanah Farsh, Meng Zhang, Gabriel Eades, Stanley Leung, Lisa N Chesner, Peter Bruno, M. Cooperberg, David A Quigley, F. Feng, Haolong Li, Carissa E. Chu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="
 5038


 Background:
 Antitumor immunity has become a critical focus in improving Penile squamous cell carcinoma (PSCC) patient outcomes. Among various immune features, tertiary lymphoid structures (TLS) have emerged as crucial players in tumor immunity. Nevertheless, the specific role of B cells within TLS and their interaction with naive T cells in PSCC remain largely unclear. Therefore, this study aims to elucidate the role of CD74⁺ B cells in TLS and their impact on immune regulation in PSCC.
 Methods:
 Spatial transcriptomics produced 519,608 cellular profiles from tumor samples of 18 patients. Additionally, scRNA-seq on tissues from 21 patients yielded 150,540 cells. Bulk RNA-seq was performed on tissues from 55 patients. TLS were identified and visualized using the DBSCAN algorithm, which clusters B cells, T cells, and DCs. Survival analyses were conducted to examine the association between TLS density and patient outcomes. Moreover, immune activity and cell infiltration were assessed via AUCell and CIBERSORT, while pseudotime analysis was used to explore dynamic gene expression changes during cell development. Additionally, pathway enrichment analysis identified key signaling pathways, and intercellular signaling was analyzed through cell communication models.
 Results:
 TLS, accurately identified using the DBSCAN algorithm, were significantly associated with improved prognosis in PSCC patients (Internal cohort, n = 152, P < 0.01; External cohort, n=63, P < 0.05). The spatial distribution, a high density of B cells was observed within the TLS regions. Notably, CD74⁺ B cells were enriched within TLS, particularly during early developmental stages. Mapped the spatially in situ developmental trajectory of CD74⁺ B cells and uncovered the dynamic changes in gene expression throughout their maturation process, we observed that CD74⁺ B cells within the TLS of PSCC patients predominantly exhibited features of early developmental stages. In line with this, scRNA-seq data further validated these observations. These cells, through their HLA molecules, interacted with CD4/CD8 ligands on naive T cells, thereby activating critical transcription factors such as NFKB1, NFKB2, NFATC1, NFATC2, FOS, and RUNX1. This interaction ultimately amplified immune responses within the tumor microenvironment. Furthermore, patients with higher CD74⁺ B cell expression exhibited better responses to immunotherapy (pCR : P < 0.01and CR: P < 0.01) and underscoring the therapeutic relevance of these cells.
 Conclusions:
 By activating naive T cells through antigen presentation, CD74⁺ B cells within TLS significantly enhance local immune responses in PSCC. Thus, CD74⁺ B cells not only serve as a promising biomarker but also represent a potential therapeutic target, providing novel insights into the immunological mechanisms underlying PSCC progression and response to immunotherapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f359d9e9ac8454c440cb7ea9056b23fcf8d8cd2" target='_blank'>
              Spatial transcriptional dynamics of CD74⁺ B cells in tertiary lymphoid structures and effects on immune evolution in penile squamous cell carcinoma.
              </a>
            </td>
          <td>
            Zaishang Li, Ting Xue, Bonan Chen, Hui Han
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Bladder cancer remains a significant global health challenge with a high mortality rate despite advancements in treatment modalities. Metabolic alterations serve as crucial contributors to cancer progression, particularly influencing tumor aggressiveness and patient outcomes. Therefore, this study aimed to identify and characterize metabolic hubs associated with disease progression and tumor aggressiveness in bladder cancer. Methods DNA methylation, mRNA expression and protein expression, along with clinical data for bladder cancer patients were retrieved from TCGA database. Differentially expressed metabolic hubs among tumor aggressiveness groups and between early vs advanced stage tumors were identified using ANOVA and Student’s t-test respectively, whereas survival association of metabolic genes was assessed using an R code. Pathway enrichment, network construction, random walk, transcription factor prediction and gene set enrichment analyses were conducted using DAVID, Cytoscape, Java, ChEA3 and GSEA tools respectively. Validation of the identified gene signature was performed using NCBI GEO datasets. Results Through a metabolism-targeted differential expression and survival analysis-based approach, we identified 105 metabolic genes, whose expression patterns correlated with tumor aggressiveness and clinical outcomes in bladder cancer patients. Subsequent network construction and random walk analysis refined this list to a seven-gene metabolic signature (Metab-GS), comprising both oncogenic (ALDH1B1, ALDH1L2, CHSY1, CSGALNACT2, GPX8) and tumor suppressors (FBP1, HPGD) hubs. Upstream analysis identified epigenetic modifications, particularly DNA hypermethylation of tumor suppressor metabolic hubs and reduced USF2-NuRD complex activity-driven increased expression of oncogenic metabolic hubs, contributing to glycolytic shift and extracellular matrix remodeling, and establishing an inflammatory tumor microenvironment. Lastly, validation of our findings in multiple independent GEO datasets confirmed that high Metab-GS scores are associated with tumor aggressiveness and progression, advanced disease stage, metastatic spread, disease recurrence, and poor overall and cancer-specific survival in bladder cancer patients. Conclusion Overall, a seven-gene metabolic signature predicts tumor aggressiveness and poor prognosis in bladder cancer patients, underscoring the potential of targeting the epigenetic dysregulation-induced metabolic reprogramming as a therapeutic strategy for aggressive bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2256df5a86a275fdcbea3d8f093b97a8cc84427d" target='_blank'>
              Epigenetic dysregulation-induced metabolic reprogramming fuels tumor progression in bladder cancer
              </a>
            </td>
          <td>
            Jian Zhang, Xiaosong Fan, Xu Xu, Yichao Han, Weixing Yu, Bicheng Yang, Yanling Chen, Shaolin Zhang
          </td>
          <td>2025-06-23</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="An individual’s genetics contributes to their risk of developing amyotrophic lateral sclerosis (ALS); however, there is still a large proportion of the heritability of ALS to be understood. Part of this missing heritability may lie in complex variants, such as the long interspersed element 1 (L1) retrotransposon, which have yet to be evaluated. The majority of L1 insertions in the human genome are no longer able to retrotranspose, but to date 279 retrotransposition-competent (RC) L1s have been reported. Many RC-L1s are polymorphic for their presence/absence; therefore, each individual will have a different number and complement of RC-L1s. These elements have been hypothesized to be involved in disease processes by multiple mechanisms such as somatic mutation by retrotransposition, the triggering of neuroinflammation and DNA damage. We hypothesize that L1s may influence disease development either through their effects on endogenous genes or through the properties that enable them to retrotranspose. Whole genome sequencing data from the New York Genome Center ALS consortium were used to characterize L1 variation identifying 2,803 polymorphic L1 elements and association analysis was performed in European individuals (ALS/ALS with other neurological disorder (ALSND) n = 2,653, controls n = 320). There were no individual L1 elements associated with disease, but we did identify a significant increase in the number of RC-L1s in ALS/ALSND genomes (p = 0.01) and the presence of ≥46 RC-L1s showed the most significant association (OR = 1.09 (1.02–1.16), p = 0.01) with disease. Analysis of individual L1s and their association with age at onset and survival identified one L1 whose presence was significantly associated with a lower age at onset (52.7 years) compared to homozygous absent individuals (59.2 years) (padj = 0.009). Our study has identified novel genetic factors for both disease risk and age at onset in ALS providing further evidence for the role of L1 retrotransposons in neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/215cd8618896e86063f645cabe583a6274cdedb5" target='_blank'>
              Retrotransposition-competent L1s are increased in the genomes of individuals with amyotrophic lateral sclerosis
              </a>
            </td>
          <td>
            A. Pfaff, Sulev Kõks
          </td>
          <td>2025-06-03</td>
          <td>Experimental Biology and Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d072dee701b15034f00be3c11a965c7ef7b90a4" target='_blank'>
              Clonal memory of cell division in humans diverges between healthy haematopoiesis and acute myeloid leukaemia
              </a>
            </td>
          <td>
            A. Donada, G. Hermange, T. Tocci, A. Midoun, G. Prevedello, L. Hadj Abed, D. Dupré, W. Sun, I. Milo, S. Tenreira Bento, C. Pospori, A. Innes, C. Willekens, J. Vargaftig, D. Michonneau, C. Lo Celso, N. Servant, KR Duffy, H. Isambert, PH Cournède, L. Laplane, L. Perie
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 APOBEC3 (A3) enzymes play a pivotal role in mutagenesis across various cancer types. These enzymes, which are single-stranded DNA cytosine deaminases, convert cytosine to uracil (C-to-U) as part of innate antiviral immune responses. Aberrant expression of A3 enzymes in several cancers leads to DNA damage, mutagenesis, and genomic instability. In breast cancer, in particular, APOBEC3B (A3B) is the major endogenous mutator that contributes to tumor evolution and resistance to treatment (1–4). DNA C-to-U deamination events result in characteristic C-to-T transitions and C-to-G transversions, and genomic uracils can also be processed into single- or double-stranded DNA breaks and larger chromosomal aberrations. Consequently, tumors with elevated A3B levels endure chronic genotoxic stress and may exhibit increased sensitivity to inhibitors targeting specific DNA repair pathways. Previous research from our laboratory identified DNA uracil glycosylase 2 (UNG2), a critical initiator of the base excision repair (BER) pathway, as a synthetic lethal partner with A3B (5). Genetic disruption of UNG2, combined with high A3B expression, led to cell death (5). This finding suggests that other DNA repair proteins could potentially serve as synthetic lethal partners with A3B under conditions of elevated A3B-induced DNA damage. To test this hypothesis, we used pharmacological and genetic inhibition of DNA damage repair proteins in isogenic cell line models for A3B. Specifically, we quantified cancer cell viability following treatment with various commercially available DNA damage response inhibitors, in the presence or absence of A3B. Additionally, we performed CRISPR screens using a guide RNA library targeting 237 DNA damage repair and response genes in a doxycycline-inducible TREX-293-A3Bi-eGFP cell line. Cells expressing or lacking A3B were harvested for DNA extraction and sequencing at different time points. By comparing guide RNA abundance between doxycycline-treated cells and untreated cells, we identified dropout guides that disrupt genes, creating potential synthetic lethal combinations with A3B. The results from both pharmacological and genetic inhibition approaches will be presented.
 Selected References:
 1. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).
 2. Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv 2, e1601737 (2016).
 3. Bertucci, F. et al. Genomic characterization of metastatic breast cancers. Nature 569, 560–564 (2019).
 4. Venkatesan, S. et al. Induction of APOBEC3 exacerbates DNA replication stress and chromosomal instability in early breast and lung cancer evolution. Cancer Discov (2021) doi:10.1158/2159-8290.CD-20-0725.
 5. Serebrenik, A. A. et al. The deaminase APOBEC3B triggers the death of cells lacking uracil DNA glycosylase. Proc Natl Acad Sci USA 116, 22158–22163 (2019).
 Citation Format: Bojana Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, Reuben S. Harris. APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e021a93ee84b96a73fc8c1738f94674803feaf" target='_blank'>
              Abstract P5-06-26: APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs
              </a>
            </td>
          <td>
            B. Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="GEN1 is implicated in DNA damage repair, as are several other breast cancer susceptibility genes, and is included in several comprehensive next-generation sequencing (NGS) testing panels. To investigate the possible association of GEN1 variants with breast cancer risk, we sequenced this gene in 617 Polish women with hereditary breast cancer (HBC) and 300 Polish cancer-free controls. No protein-truncating variants were detected in the conserved part of GEN1 (first 480 codons). Two frameshift variants were detected in the last exon of GEN1: c.2515_2519delAAGTT (p.Lys839Glufs*2) and c.1929_1932delAAAG (p.Lys645Cysfs*29). The p.Lys839Glufs*2 variant was detected in 21.1% of 617 HBC cases and 18.4% of 300 controls (p = 0.38). The p.Lys645Cysfs*29 variant was rare, seen in 0.6% of 617 HBC cases and 0.3% of 300 controls. The variant was then detected in 38 (0.24%) of 15,930 unselected breast cancer cases and 8 (0.17%) of 4702 cancer-free female controls from Poland (OR = 1.40, p = 0.49). Clinical characteristics of breast tumors in the 38 carriers of p.Lys645Cysfs*29 and 15,892 non-carriers were similar. Survival was similar among variant carriers and non-carriers (the age-adjusted HR = 0.87, p = 0.76). The wild-type GEN1 allele was retained in all five breast cancers of carriers of p.Lys645Cysfs*29. No cancer type was more frequent in the relatives of 35 p.Lys645Cysfs*29 variant carriers compared to the relatives of 14,592 non-carriers. We conclude that GEN1 is unlikely to be a high or moderate-risk breast cancer susceptibility gene. Our study has clinical implications for genetic counseling and suggests that GEN1 changes should be reclassified as variants of uncertain significance (VUS) when they are detected in clinical testing panels.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01181fd107e903d29456b2281a104bc160955086" target='_blank'>
              Analysis of GEN1 as a Breast Cancer Susceptibility Gene in Polish Women
              </a>
            </td>
          <td>
            Katarzyna Gliniewicz, D. Wokołorczyk, W. Kluźniak, Klaudia Stempa, T. Huzarski, H. Rudnicka, A. Jakubowska, M. Szwiec, J. Jarkiewicz-Tretyn, M. Cechowska, P. Domagała, T. Dębniak, M. Lener, J. Gronwald, J. Lubiński, Steven A. Narod, Mohammad R. Akbari, C. Cybulski
          </td>
          <td>2025-06-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="
 3143


 Background:
 HIF2 alpha inhibitors (HIF2i) have been recently approved for VHL-related tumors, especially clear cell renal cell carcinoma (ccRCC). Alterations in genes that regulate HIF signaling may modulate response to HIF2i and may help identify tumor types that could benefit from new therapeutic approaches targeting HIF activity. Here, we describe the landscape of these alterations across different cancer types.
 Methods:
 This study included patients (pts) from the TCGA PanCancer Atlas and GENIE Cohort v17.0, across all cancer types. All pts underwent Next Generation Sequencing or OncoPanel analysis of their tumors. Alterations in genes related to the HIF pathway (
 VHL
 ,
 EPAS1
 ,
 EGLN1
 , and
 EGLN2
 ) and citric acid cycle (
 SDHA
 ,
 SDHB
 ,
 SDHC
 ,
 SDHD
 ,
 SDHAF1
 ,
 FH
 ,
 IDH1
 ,
 IDH2
 , and
 MDH2
 ) were screened, and only driver mutations, identified by OncoKB, CIViC Variants, My Cancer Genome, and the available literature, were included. Pts with a mutation in at least one gene were included. Mutation frequencies across different cancer types were analyzed for each cohort, then pooled together.
 Results:
 We identified a total of 10,953 and 167,073 pts on TCGA and GENIE, respectively. Mutation frequencies for different cancer types are shown in the Table. Among the mutated cases,
 IDH1
 was the most altered gene in glioma (90.20% [95% Confidence Interval (CI): 90.19; 90.21]), hepatobiliary cancers (75.48% [75.40; 75.56]) and melanoma (55.75% [55.57; 55.92]), and the second most common in leukemia (39.86% [39.68; 40.04]), while
 IDH2
 was the most altered in leukemia (57.52% [57.34; 57.70]), and the second most common in glioma (5.25% [5.24; 5.25]), and hepatobiliary cancers (19.10% [19.04; 19.17]).
 SDHA
 was the second most altered gene in melanoma (12.95% [12.87; 13.03]). In RCC,
 VHL
 was the most mutated gene (92.79% [92.78; 92.79]), followed by
 FH
 (2.41% [2.40; 2.41]).
 EPAS1
 was mutated in 64.19% of cases with driver mutations in miscellaneous neuroepithelial tumors (MNET) ([57.07; 71.31]), but not in RCC. Among the altered cases in pheochromocytoma,
 SDHB
 was the most commonly altered gene (39.71% [35.31; 44.11], followed by
 VHL
 (18.41% [15.92; 20.91]).
 SDHB
 was also the second most common gene to be mutated in MNET (23.12% [19.37; 26.86]).
 Conclusions:
 In this analysis, mutations in HIF-regulating genes were detected in multiple cancers, and were not limited to those studied in the context of HIF inhibitors. Further research is required to elucidate whether these gene alterations sensitize tumors to HIF inhibition.


 Pooled mutation frequencies for all considered genes in different cancer types.




 Cancer Type
 N
 % Pooled Mutation Frequency of any gene
 95% CI




 RCC
 3662
 39.76
 39.74; 39.77


 Glioma
 12657
 23.67
 23.67; 23.67


 Leukemia
 1857
 15.49
 15.47; 15.50


 Hepatobiliary Cancer
 4234
 11.18
 11.18; 11.19


 MNET
 253
 8.61
 8.54; 8.68


 Pheochromocytoma
 198
 5.63
 5.56; 5.69


 Melanoma
 6589
 4.34
 4.34; 4.34




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b30602d5706dbaf2b3bb125abc8e340c39322ae" target='_blank'>
              Landscape of genomic alterations in genes implicated in the regulation of hypoxia inducible factor (HIF) signaling: A pooled analysis of two pan-cancer cohorts.
              </a>
            </td>
          <td>
            Wassim Daoud Khatoun, Jad El Masri, E. Saad, Marc Eid, M. Machaalani, Mustafa Jamal Saleh, R. Nawfal, Karl Semaan, R. El Hajj Chehade, Clara Steiner, E. Yekeduz, Liliana Ascione, C. Labaki, R. Saliby, T. El Zarif, S. Signoretti, D. McDermott, Eliezer M Van Allen, T. Choueiri
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6483fac3f504fbebfb3a60d1e21f332ce1ac0267" target='_blank'>
              Epigenetic reprogramming of mast and cancer cells modifies tumor-promoting cytokine networks.
              </a>
            </td>
          <td>
            Alejandro Schcolnik-Cabrera, Marisol Ramírez-Yautentzi, Rodolfo Soria-Castro, R. Chacón-Salinas, Rocío Morales-Bárcenas, G. Domínguez-Gómez, A. González-Fierro, Alfonso Dueñas-González, Alma D Chávez-Blanco
          </td>
          <td>2025-07-24</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are abundant in the human genome with approximately 300,000 embedded in gene introns, exons, and untranslated regions. High penetrance STR variants cause human diseases such as Myotonic dystrophy, Baratela-Scott syndrome, and various ataxias. The possibility that STRs contribute to polygenic disease is supported by recent high-powered datasets that link STRs to more subtle effects on gene expression. Indeed, STR variants can induce Z-DNA and H-DNA folding; alter nucleosome positioning; and change the spacing of DNA binding sites. On the other hand, little is known about how STR variants affect RNA secondary structure and accessibility. These factors could affect rates of splicing, nuclear export, and translation. We hypothesize that effects on RNA structure can be predicted using computational tools and associated with gene expression using DNA and RNA sequencing data. We test this hypothesis using data from the 1000 Genomes Project and ViennaRNA. We identify 17,255 transcribed STRs that affect RNA folding (fSTRs); 356 are possibly associated with gene expression. We characterize fSTRs by repeat motif, length, and gene level annotation. Transcribed fSTR variants tend to affect RNA multiloops and external loops. Effects on RNA accessibility depends on the repeat motif: a surprising result that is checked against simulation. These results shed light on how transcribed STRs affect RNA structure and pave the way for experimental validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ac5e9e6ba805badefe346bba5943b6b1c9b8c40" target='_blank'>
              Short tandem repeat variants are possibly associated with RNA secondary structure and gene expression
              </a>
            </td>
          <td>
            Nick Kinney, Dikshya Pathak, Emma Evans, Paola Arias
          </td>
          <td>2025-06-18</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 4154


 Background:
 Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with a dismal prognosis despite advances in treatment. PDAC is characterized by a dense stromal environment with suppressed anti-tumor immunity that contributes to treatment resistance. Oncogenic
 KRAS
 mutations are present in > 90% of PDACs and are known to play a role in modulating the tumor immune microenvironment (TME). In this study, we examined the differences in TME components between
 KRAS
 -mutant (m) and
 KRAS
 -wildtype (wt) PDACs.
 Methods:
 Comprehensive genomic and immune profiling (CGIP), including the RNA-seq based gene expression assessment of 395 immune-associated genes, was performed on 311 PDAC patient samples. Gene expression signatures of tumor immunogenicity (TIGS) and cell proliferation were calculated by averaging the gene expression ranks of 161 immune-associated genes and 10 proliferation genes, respectively. The normalized gene expression rank of 22 immune checkpoint genes and 17 cancer testis antigen (CTA) genes were also calculated. DNA-seq was used to identify
 KRAS
 mutations and to calculated tumor mutational burden (TMB). Continuous variables were compared between subgroups using the Wilcoxon Rank-Sum test and categorical variables were compared between groups using the Chi Squared test. For statistical significance, p < 0.05 was required.
 Results:
 The cohort consisted of 311 PDAC patient samples, comprising 159 females (51.1%) and 152 males (48.9%) with a median age at testing of 69.3 years (38.7-92.6). A total of 264 specimens (84.9%) exhibited a
 KRAS
 mutation, with the most common being G12D and G12V while 47 (15.1%) had
 KRAS
 -wt tumors. No difference in TIGS, CP, or TMB was observed between
 KRAS
 -wt and
 KRAS
 -m tumors.
 KRAS
 -wt tumors exhibited greater expression of 12 of 17 tested CTAs than
 KRAS
 -m tumors, including GAGE13 (p = 0.004) NY-ESO-1 (p = 0.01), MAGEA3 (p = 0.01), and LAGE1A (p = 0.01). Analysis of the gene expression of 22 immune checkpoint genes showed no difference for most of the genes, though there was higher expression of PD-1 (p = 0.04) and CD27 (p = 0.03) seen in
 KRAS
 -wt compared to
 KRAS
 -m tumors.
 Conclusions:
 KRAS
 -wt PDACs exhibited greater expression of cancer testis antigen genes compared to
 KRAS
 -m tumors, suggesting potential therapeutic susceptibility to immunotherapy and adoptive cell therapies leveraging the expression of CTAs as targets. Assessment of
 KRAS
 status and immunotherapy susceptibility may support future clinical trial selections for therapies targeting the complex interplay of genomic and immune components of pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ae6d1fe40ee2b5375e715aee2512cfe9d76d6fb" target='_blank'>
              Unveiling the differences in tumor immune microenvironment between
 KRAS
 -wildtype and
 KRAS
 -mutant pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            Heidi Ko, S. Pabla, Michelle F Green, M. Senosain, Hardik Parikh, Erik Van Roey, Shuang Gao, Yamuna Pulivendula, P. DePietro, Stuti Kaushalkumar Patel, Kyle C. Strickland, R. Previs, Erin Newburn, Kamal S Saini, Alicia Dillard, Eric A Severson, Brian Caveney, Taylor J. Jensen, Shakti H. Ramkissoon, R. Seager
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Unchecked growth in three-dimensions (3D) in culture is a key feature of immortalized cells on the path to malignant transformation and hence a potential target phenotype for prevention. Also, expression of genes driving this process, but not that of 2D growth, would likely be more specific to cancer development and their inhibition would be less toxic to normal cells, many of which can grow in 2D but rarely in 3D culture. To define such genes, we compared CRISPR depletion screens performed in HBLAK, a spontaneously immortalized, non-tumorigenic human urothelial cell line, grown in 2D to those in 3D. Using the CRISPR Bassik DTKP (drug target kinase phosphatase) deletion library targeting 2,333 genes, we identified 85 genes which were specifically lost in 3D cultures. Prioritizing hits to those associated with bladder cancer in patients provided us with a set of 11 genes. Only one gene, MAPK1 remained relevant if a human pan-cancer criteria was applied. Single gene in vitro validation confirmed that MAPK1 inhibition was specific to 3D growth. We also found that MAPK1 depletion led to significant growth reductions in human tumor xenografts in vivo. Inhibition of MAPK1 by Ulixertinib, an orally active MAPK1 inhibitor, led to human bladder cancer growth inhibition in both 3D in vitro and in vivo models. In summary, screening for genes specifically driving 3D growth in immortalized cells may provide targets for both prevention and early therapy in bladder and other cancers while potentially limiting therapeutic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c39ec53d27180d1be55595d4ca83d125348b0422" target='_blank'>
              Genes driving three-dimensional growth of immortalized cells and cancer
              </a>
            </td>
          <td>
            Mukta Basu, Jin-Fen Xiao, Saravana Kumar Kailasam Mani, Fangyuan Qu, Yongqi Lin, Jason Duex, Huihui Ye, Vanessa Neang, Dan Theodorescu
          </td>
          <td>2025-06-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Mutational signatures are increasingly used to understand the mechanisms causing cancer. However, their important applications in predicting prognosis and stratifying patients for therapy are hampered by inaccurate inference of the various featureless, dense trinucleotide mutational spectra, which are often confounded with one another. One of them is the homologous recombination deficiency (HRd)-associated signature SBS3, relevant because of its association with prognosis in ovarian and breast cancer and because of its potential as a biomarker for synthetic lethality therapies. Methods Here, we highlight strong benefits of a multimodal approach for mutational signature extraction, applied on top of standard bioinformatic pipelines. By jointly operating on single-base substitution (SBS) and indel (ID) spectra, this method enables accurate identification of various DNA repair deficiency signatures and patient survival prediction. Results Across four different cohorts of whole-genome sequenced high-grade serous ovarian cancers (HGSOC), the multimodal SBS + ID approach correctly distinguished the commonly confused signatures SBS3, SBS5, SBS8, SBS39, and SBS40. Importantly, we robustly identified two different multimodal SBS3 signatures, m-SBS3a and m-SBS3b, with distinct patterns in the indel spectrum. Multimodal SBS3b signature was strongly predictive of longer survival in ovarian cancer patients, replicating across four cohorts, with effect sizes greatly exceeding other genetic markers. Our m-SBS3 also predicted survival in platinum-treated patients with various cancer types, and moreover, the SBS + ID joint inference was successfully applied to mismatch repair-deficient colorectal cancer and immunotherapy response, supporting a general utility of the multimodal mutational signatures approach. Conclusions Overall, combining SBS and ID mutations improves detection of HR deficiency-associated signatures and reveals distinct SBS3 subtypes with prognostic value. This multimodal approach outperforms existing markers and is readily applicable to therapy stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01497-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01c11401f870f4aa8e6266e2fe8e3ab75a0da51" target='_blank'>
              Joint inference of mutational signatures from indels and single-nucleotide substitutions reveals prognostic impact of DNA repair deficiencies
              </a>
            </td>
          <td>
            Patricia Ferrer-Torres, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-07-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Loss-of-function mutations in genome maintenance genes fuel tumorigenesis through increased genomic instability. A subset of these tumor suppressors are challenging to identify due to context dependency, including functional interactions with other genes and pathways. Here, we searched for potential causal genes that impact tumor development and/or progression in breast cancer through functional-genetic screening of candidate genes. MYH4, encoding a class II myosin, emerged as a top hit impacting genomic stability. We show that MYH4 suppresses DNA replication stress by promoting replication licensing and replication fork progression. Moreover, we observed a strong synergistic relationship among class II myosins in suppressing replication-associated DNA damage. Genomic analysis of Pan-Cancer Analysis of Whole Genomes project breast cancer samples revealed frequent concomitant loss of TP53 with MYH4 and class II myosins on chromosome 17p. Notably, Myh4 disruption accelerated mouse mammary tumorigenesis in a Trp53-deficient background. In conclusion, our results suggest an unanticipated function of MYH4 in p53-mediated tumor suppression that can explain their combined loss in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64806f05c07b8d24d35ea0d2ce3f2fd4c89aa895" target='_blank'>
              The class II myosin MYH4 safeguards genome integrity and suppresses tumor progression
              </a>
            </td>
          <td>
            J. Thatte, Ana Moisés da Silva, Judit Börcsök, Thorkell Gudjónsson, Jan Benada, Xin Li, Muthiah Bose, H. van der Gulden, Ji-Ying Song, Renée Menezes, Elena Martín-Doncel, Luis Toledo, Valdemaras Petrosius, Cord Brakebusch, J. Jonkers, F. C. Nielsen, Maria Rossing, C. S. Sørensen
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c9d768db6c320ae3141a12d174f6fdda8a0e39f" target='_blank'>
              Modified hTERT promoters-driven purine nucleoside phosphorylase-gene therapy in association with chemo- and targeted therapy in the context of ovarian cancer.
              </a>
            </td>
          <td>
            Quoc Manh Nguyen, Pierre-François Dupré, Mathieu Berchel, Marylène Lévêque, Sylvie Choblet-Thery, F. d’Arbonneau, T. Montier
          </td>
          <td>2025-06-27</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutational signatures of single-base substitutions (SBSs) characterize somatic mutation processes which contribute to cancer development and progression. However, current mutational signatures do not distinguish the two independent steps that generate SBSs: the initial DNA damage followed by erroneous repair. To address this modelling gap we developed DAMUTA, a hierarchical Bayesian probabilistic model that infers separate signatures for each process, and captures their sample-specific interaction. We applied DAMUTA to 18,974 pan-cancer whole genome sequencing mutation catalogues from 23 cancer types and show that tissue-specificity in mutation patterns is driven largely by variability in damage processes. We also show that misrepair processes are predictive of DNA damage response deficiencies. Unlike existing approaches, DAMUTA distinguishes damage from misrepair contributions, and we demonstrate significant improvements over a mutational-burden baseline or signatures from the COSMIC database. Our analysis reveals a shared pan-cancer pattern of early clonal transition-mutations which shifts to a more uniform substitution pattern consistent with increased reliance on translesion synthesis for damage tolerance. DAMUTA thus generates a compact set of signatures which resolves redundancies of current signature models, disentangles the effects of DNA damage and misrepair processes, and facilitates improved stratification of tumours, while providing a framework towards a unified pan-cancer model of the cellular response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3964588c1648b53d4e30a576d480451079a470ad" target='_blank'>
              Damage and Misrepair Signatures: Compact Representations of Pan-cancer Mutational Processes
              </a>
            </td>
          <td>
            Caitlin F. Harrigan, Kieran R. Campbell, Quaid Morris, Tyler Funnell
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2fb143b1900a1c66ed78148b97787d2aa07d158" target='_blank'>
              Investigation of a germline variant of uncertain significance (MRE11:c.1138C > T) identified by exome sequencing in a cancer-affected individual and co-segregation analysis in affected and unaffected family members.
              </a>
            </td>
          <td>
            Mohammadreza Saberiyan, Mohammad Reza Seyedtaghia, Elham Mojodi, A. Nejatizadeh
          </td>
          <td>2025-05-31</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Background: Patients exhibiting exceptional response to targeted treatments represent a unique opportunity towards identifying predictive biomarkers of response. The introduction of edits into the genome using Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) creates cancer models that can unravel the mechanisms driving tumorigenesis and response to drug agents. The study aimed to identify candidate molecular alterations associated with exceptional response to CDK inhibitors (CDKi) and validate their effects using genome editing. Methods: Patients with ER-positive, HER2-negative advanced breast cancer treated with CDKi and endocrine combination therapy at Departments of Oncology-affiliated with the Hellenic Cooperative Oncology Group (HeCOG), were included. Exceptional response was defined as either complete response (CR) to the treatment regimen or partial response (PR) for 3 years, per physician’s assessment. Formalin-fixed paraffin-embedded (FFPE) tumor tissue samples, obtained prior to CDKi administration from these patients were retrospectively collected. Whole-exome sequencing (WES) was performed to analyze tumor mutational landscape of exceptional responders to CDKi. Using CRISPR-mediated genome editing, a breast cancer cell model harboring a patient-specific SNP and the negative control were prepared, and subsequently evaluated for their responsiveness to palbociclib. Transcriptomic and proteomic analyses were conducted to uncover alterations in network interactions, between the two cell lines, elucidating the exceptional response to CDKi. Results: Overall, 12 women identified as exceptional responders, initiated CDKi treatment from March 2017 to February 2020; median age at diagnosis was 46 (range, 37 to 78). All patients had received the CDKi as first-line treatment; most commonly palbociclib (10 patients, 83.3%). Based on physician assessment, 50% of patients achieved CR with the combination therapy. After a median follow up of 65 months, 10 patients are still alive, and 7 are still receiving treatment with CDKi in combination with endocrine treatment. WES results with adequate mapped reads and acceptable technical characteristics in both tumor and control blood samples were obtained from 6 of 12 patients. A set of germline and somatic single nucleotide polymorphisms (SNPs) commonly shared in this cohort was identified. The focus was on MINDY1, a member of the MINDY family of deubiquitinases (DUBs), known as a deubiquitylase of ERα in breast cancer, the common SNP among all 6 patients. Utilizing CRISPR-based genome editing, a breast cancer cell model harboring a specific MINDY1 SNP was engineered, and their response to palbociclib was subsequently evaluated. Depletion of MINDY1 transcripts through RNA interference with short-hairpin RNA, increased sensitivity to palbociclib in both cell lines, confirming that MINDY1 protein is involved in molecular pathways related to palbociclib response. The functional role of MINDY1 rs771205 was also investigated by combining transcriptomic and proteomic data. Our data demonstrated that the MINDY1 rs771205 SNP, which causes an amino acid substitution, while it does not impact the respective transcript and protein levels, alters the MINDY1 protein’s interaction network. Conclusions: The Rs771205 single mutation alters the MINDY1 protein’s interaction network, sensitizing cells to palbociclib. Our study sheds new light on the underlying mechanisms driving exceptional response in patients with ER-positive, HER2-negative advanced breast cancer treated with CDKi and endocrine combination therapy , demonstrating that it is an opportune moment to invest attention and resources towards probing the underlying mechanisms of exceptional response in cancer patients.
 Citation Format: Eleni Balla, Yi Li, Kyriaki Papadopoulou, John T. Nguyen, Zikun Zhou, Anna Koumarianou, Eleni Galani, Rania Romanidou, George Fountzilas, Evangelia Razis, Angelos Koutras, Leonidas Bleris, Elena Fountzilas. MINDY1 role and exceptional response to CDK inhibitors in metastatic breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-03-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea823fc49d305c67d0c88bebfe1f1411182d2dc" target='_blank'>
              Abstract P4-03-26: MINDY1 role and exceptional response to CDK inhibitors in metastatic breast cancer
              </a>
            </td>
          <td>
            Eleni Balla, Yi Li, K. Papadopoulou, John T. Nguyen, Zikun Zhou, Anna Koumarianou, E. Galani, Rania Romanidou, G. Fountzilas, E. Razis, A. Koutras, L. Bleris, E. Fountzilas
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Objective: To undertake a literature review on the development of cervical cancer, emphasizing epidemiological data, human papillomavirus (HPV) infection in carcinogenesis, and the role of the XRCC1 gene in DNA repair, in order to provide valuable information for the medical and scientific community. The goal is to provide a valuable resource for clinical practice. Bibliographic Review: Cervical cancer ranks as the fourth most common malignancy worldwide. Despite known environmental, behavioral, and genetic risk factors, their management remains inadequate. Persistent infection with oncogenic HPV is a well-established risk factor; however, not all infected individuals develop cervical cancer, suggesting the involvement of additional mechanisms. Among these, polymorphisms in the XRCC1 gene have gained attention. XRCC1 encodes a scaffold protein essential for DNA repair coordination and cell cycle regulation. Three main XRCC1 polymorphisms—Arg194Trp, Arg280His, and Arg399Gln—are associated with impaired DNA repair, genomic instability, and increased cervical cancer susceptibility. Final Considerations: A deeper understanding of the genetic and cellular mechanisms underlying cervical cancer may facilitate the development of targeted medical interventions for women, contributing to improved quality of life reduced mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607876d22f013a85473f817b1c9f164dd5e62623" target='_blank'>
              Cervical cancer and XRCC1 gene polymorphisms
              </a>
            </td>
          <td>
            Andréia Michelle Alves Cunha de Alcântara, Ivan Barbosa Barros, Israel Santos da Silva, Karla Roberta Lemos Maul de Souza, Ivan Barbosa Barros, M. Maia, Paulo Roberto Eleutério de Souza
          </td>
          <td>2025-05-30</td>
          <td>Revista Eletrônica Acervo Saúde</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) plays a critical role in cell growth and survival. PIK3CA somatic mutations are linked to the PIK3CA-related overgrowth spectrum (PROS) syndrome. Cells from those patients appeared to be associated with a moderate but significant radiosensitivity. Mutations or amplifications in this gene are common in breast, colorectal, and lung cancers. Alterations in the PIK3CA gene, including amplification and mutation, are common in cancer, but their influence on radiotherapy is not yet fully understood. This report reveals a potential association between PIK3CA gene modifications and radiosensitivity (p < 0.05) deducted from 8 established colorectal cancer cell lines (HT-29, DLD-1, HCT-116, SW480, HCT-15, Colo-320, and LoVo). Meanwhile gene amplification (> 2) seems to be linked to increased radiation sensitivity, mutations appear to be associated with increased radioresistance in colorectal cancer cells. Leveraging this relationship, PIK3CA amplification and mutations could act as biomarker to pinpoint patients who might benefit from more personalized radiotherapy regimens. However, this association is preliminary and hypothesis-generating in view of the limited number of cases. Further studies are needed to confirm this conclusion. By uncovering the distinct mechanistic effects of these PIK3CA alterations on radiosensitivity phenotype in both normal and cancerous cells, researchers can lay the groundwork for tailored radiotherapy strategies in colorectal cancer. This insight could enhance treatment effectiveness while reducing side effects, ultimately leading to improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d2f05a3aac70c0b974b05ec72c5f8dff7113c5" target='_blank'>
              Effects of PIK3CA Gene Modifications on Radiosensitivity in Colorectal Cancer Cells.
              </a>
            </td>
          <td>
            Ghazi Alsbeih, K. Al-Hadyan
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor protein p53 (TP53) is a crucial regulator of genomic integrity, frequently mutated in more than half of all human cancers. These mutations predominantly target the DNA-binding domain (DBD), impairing p53's interaction with DNA and its tumor-suppressive functions. To elucidate the structural and functional consequences of p53 mutations, we investigated 148 missense variants located within its DNA-binding interface using cutting-edge computational approaches. We employed AlphaFold3 (AF3) to predict p53-DNA complex structures, integrating these predictions with molecular dynamics (MD) and force-guided pulling simulations to assess mutation-induced changes in structural stability and DNA-binding properties. Moreover, we compared the results of our study with experimental in vitro enrichment scores (RFS) and Combined Annotation Dependent Depletion (CADD). We identified a moderate negative correlation between plDDT and the pathogenicity of mutant variants, suggesting that mutations causing more significant alterations in the protein tertiary structure have a greater negative impact on cellular function. Moreover, we identified two possible structural mechanisms through which mutations can impair the p53 functionality. Specifically, some mutations, such as R248P and N239S, reduce the binding affinity of the p53-DNA complex, whereas others, such as C238Y and P278R, enhance affinity but compromise the structural stability of the complex. Furthermore, we uncovered mutations with potential rescuing effects, such as E285A and M243T, which preserved structural stability and enhanced the DNA-binding ability. Our findings provide a comprehensive framework for understanding the molecular mechanisms underlying p53 mutations and their role in cancer pathogenesis. This study highlights the value of integrative computational approaches in investigating protein-nucleic acid interactions, providing critical insights that can guide the development of therapeutic strategies targeting p53 mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ab5ce93ebd4a1fbc747fc251d769b1eab00ffa3" target='_blank'>
              Investigation of Structural Mechanisms Underlying p53 Dysfunction Caused by 148 Missense Mutations Using AlphaFold3 and Molecular Dynamics Simulations.
              </a>
            </td>
          <td>
            Khondamir R Rustamov, J. I. Razzokov, A. Baev
          </td>
          <td>2025-06-04</td>
          <td>Journal of chemical information and modeling</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2968d582c4d09600f6a6aab6083cc272395bfdb6" target='_blank'>
              MicroRNA-driven organ-specific metastasis in triple-negative breast cancer: biogenesis, mechanisms, and therapeutic approaches.
              </a>
            </td>
          <td>
            Deepshikha Rathore, Sweny Jain, Shanaya Patel, Nandani Dharwal, N. Shukla, Chirag Desai, Jigna Shah, Heena V. Dave
          </td>
          <td>2025-06-28</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CHROMR is a primate-specific long noncoding RNA with emerging roles in homeostasis and pathophysiology. Elevated blood levels of CHROMR have been observed in patients with cardiovascular disease and several cancers, where it is correlated with poor clinical outcomes. Like many lncRNAs, CHROMR accumulates in both the nucleus and the cytoplasm, and it assumes distinct functions in each of these cellular compartments. In the nucleus, CHROMR sequesters a transcriptional repressor complex to activate interferon-stimulated gene expression and antiviral immunity. In the cytoplasm, CHROMR competitively inhibits microRNAs involved in cholesterol efflux and cell cycle regulation, thereby impacting gene pathways involved in reverse cholesterol transport, HDL biogenesis, and tumor growth. In this review, we detail the multifaceted functions of CHROMR in cholesterol metabolism, innate immunity, and cancer progression. We also explore the potential molecular mechanisms that govern its expression and dynamic subcellular localization, which may be key to its functional versatility. Advancing our understanding of the regulatory networks and cellular environments that shape CHROMR activity will be critical for assessing its promise as a therapeutic target and diagnostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e434fd00ac8710c07c05ad122a8af40ef9af54ab" target='_blank'>
              The Multifaceted Roles of CHROMR in Innate Immunity, Cancer, and Cholesterol Homeostasis
              </a>
            </td>
          <td>
            Emma R. Blaustein, Coen van Solingen
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract The homologous recombination (HR) system repairs DNA double-strand breaks produced by the DNA damage response, which is a complex signaling pathway consisting of the key proteins BRCA1/2 and other DNA repair proteins, such as the ATM, PALB2, BARD1, RAD51, and Fanconi anemia proteins. Mutations and epigenetic alterations in HR-related genes may lead to homologous recombination deficiency (HRD), resulting in genomic instability and contributing to the development of certain solid tumors. The biological significance and molecular mechanism of BRCA1/2 mutation-related HRD are well understood, but the relationships of other HR-related genes and their variant forms with HRD have not been sufficiently studied. These genes exhibit multiple forms of variation, including one or more HR genes, germline or somatic mutations, monoallelic or biallelic variants, and not all variants present HRD. Therefore, HRD is usually defined as HR-related gene variation, but recent studies have shown that defining it as the combined score of loss of heterozygosity, LST and TAI, known as the HRD score, can more accurately assess genomic instability. In patients with HRD, platinum-based therapy and poly ADP-ribose polymerase enzyme inhibitor (PARPi) have been shown to perform well in ovarian, breast, and prostate cancers. For gastrointestinal cancer (GI cancer), HRD has been relatively well studied in pancreatic cancer, but its role in other cancers has rarely been reported. Herein, we review the pathogenesis and predictive value of HRD, including the use of platinum drugs, PARPi, and immunotherapy, in digestive system tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b86ad0d0eb3efaff60321645af70d194c1741a2" target='_blank'>
              The prognostic and predictive value of homologous recombination deficiency in gastrointestinal cancer
              </a>
            </td>
          <td>
            Beibei Jiang, Yile Shang, Xiang Zhang, Wenguang He, Hanju Hua, Feng Ye, Xile Zhou, Yandong Li, Weixiang Zhong, Wei-wei Jiang, Guosheng Wu
          </td>
          <td>2025-06-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="PURPOSE AlphaMissense is a new artificial intelligence–based approach to predicting the pathogenicity of missense variants. However, whether its predictions can be directly applied to clinical decision making remains unclear. This study aimed to evaluate the accuracy of AlphaMissense predictions for DNA damage repair (DDR) genes using genomic and clinical characteristics. METHODS Sequencing data from 56,965 patients with cancer who underwent Memorial Sloan Kettering–Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT) testing between April 2015 and March 2023 and data from The Cancer Genome Atlas (TCGA)/Pan-Cancer Analysis of Whole Genomes (PCAWG) were analyzed. AlphaMissense pathogenicity was evaluated in six commonly mutated DDR genes. Missense mutations were classified into three categories on the basis of AlphaMissense and OncoKB: known pathogenic, newly identified pathogenic by AlphaMissense, or benign. RESULTS In the MSK-IMPACT cohort, 1,182 (17.5%) of 6,743 unique DDR gene missense mutations were newly identified as pathogenic. In breast, ovarian, pancreatic, and prostate cancers, homologous recombination (HR)–deficiency signatures were more common in tumors with new pathogenic missense mutations in BRCA1/2, PALB2, and RAD51C than in benign missense mutations (66.7% v 35.2%, P = .021). In the TCGA/PCAWG data set, HR-deficiency signatures were also more frequent in tumors with new pathogenic mutations than in wild-type (46.2% v 19.2%, P = .036). For tumors with POLE missense mutations, there were no significant differences in tumor mutation burden and POLE-associated signatures between new pathogenic and benign mutations. Patients with new pathogenic ATM missense mutations had fewer TP53 mutations (30.5% v 54.6%, P < .001) and showed improved irradiated tumor control (hazard ratio, 0.58 [95% CI, 0.35 to 0.95]; P = .03) compared with those with benign missense mutations. CONCLUSION Our findings suggest that AlphaMissense can help identify previously unknown pathogenic DDR gene mutations, but its accuracy is gene-dependent. AlphaMissense prediction still requires additional confirmation with clinical and functional validation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03400cbb538705acb05e22ae6ecc28eab38ddec" target='_blank'>
              AlphaMissense for Identifying Pathogenic Missense Mutations in DNA Damage Repair Genes in Cancer
              </a>
            </td>
          <td>
            Shu Yazaki, Xin Pei, Simon N. Powell, Atif J. Khan, J. Setton, N. Riaz
          </td>
          <td>2025-06-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31620c8deab30d15f98e2b8cf7eeeade98239959" target='_blank'>
              LINC01410 in cancer: a comprehensive review of its oncogenic role, regulatory mechanisms, and clinical implications.
              </a>
            </td>
          <td>
            Saman Morovat, Pejman Morovat, Ashkan Kalantary-Charvadeh, Marzieh Mojbafan, S. Teimourian
          </td>
          <td>2025-07-12</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc739b29aa041a82d43e91798c3e41caf831aa6b" target='_blank'>
              Enhancer Poising Enables Pathogenic Gene Activation by Noncoding Variants
              </a>
            </td>
          <td>
            Ethan W. Hollingsworth, Zhuoxin Chen, Cindy X. Chen, Sandra H. Jacinto, Taryn A Liu, Evgeny Z. Kvon
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Significance Here, we address a major limitation in the effectiveness of poly(ADP-ribose) polymerase inhibitors (PARPi) in breast cancer gene (BRCA)-mutant prostate cancer treatment: Only ~50% of patients respond despite clear genomic evidence of defective homologous recombination. Prior efforts to study PARPi resistance in prostate cancer have been plagued by the lack of suitable cell lines. We overcame this challenge using primary prostate organoids coupled with genome-wide CRISPR screening. The key finding is that loss-of-function mutations in the DNA prereplication complex (pre-RC) confer PARPi resistance. These genes map to chromosomal regions frequently lost in prostate cancer and could therefore serve as potential biomarkers of treatment response. Pharmacologic inhibition of geminin, a negative regulator of the pre-RC, can restore PARPi sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e95a3ba5883a1720490b54e4008aa7d7b0986c0b" target='_blank'>
              BRCA2 reversion mutation–independent resistance to PARP inhibition through impaired DNA prereplication complex function
              </a>
            </td>
          <td>
            Kyrie Pappas, Matteo Ferrari, Perianne Smith, S. Nandakumar, Zahra Khan, Serina B. Young, Justin Laclair, M. V. Russo, Emmet Huang-Hobbs, Nikolaus Schultz, W. Abida, Wouter Karthaus, M. Jasin, Charles L Sawyers
          </td>
          <td>2025-06-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>100</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99ef5f46003666cce9bfbffdcaf4920a36fe552" target='_blank'>
              RNA G-quadruplexes: emerging regulators of gene expression and therapeutic targets.
              </a>
            </td>
          <td>
            Zukela Ruzi, Wei Zha, Huang Yuan Yuan, Jiaorui Liu
          </td>
          <td>2025-07-03</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5f43f79f58977259121120764f54b519011ee5e" target='_blank'>
              RBM5 recruiting MGC32805 in a sandwich mode and inducing ΔFAS neoantigen and triggering FAS properties switch: implication in colorectal cancer.
              </a>
            </td>
          <td>
            Huizhe Wu, Xiaoyun Hu, Yilin Wang, Xianglong Zhu, Qing Zhao, Ying Zhao, Wanlin Cui, Mingrong Zhang, Haishan Zhao, Kai Li, Minjie Wei
          </td>
          <td>2025-06-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Cellular reprogramming is a fundamental process in specific developmental stages and is also invaluable for disease modeling. It involves the conversion of differentiated cells into undifferentiated or specialized cell types. This transformation relies on coordinated action of transcription factors or chemical stimuli, which reshapes the cell's chromatin structure and establishes a different gene expression profile. Among these transcription factors, pioneer factors play a critical role, initiating cellular reprogramming and guiding cell fate transitions by selectively binding to specific DNA motifs within nucleosomes at inactive chromatin. The GATA3 protein serves as a pioneer transcription factor essential in mammary gland development and is involved in breast tumor development. GATA3 facilitates the cellular reprogramming of mesenchymal breast cancer cells into epithelial breast cancer cells. Despite the established role of pioneer factors in chromatin remodeling and cell fate regulation, the precise molecular mechanisms that govern their selective binding to chromatin are still not fully understood. Using the mesenchymal-to-epithelial transition (MET) model, we aimed to understand how GATA3 selectively binds to genes that are essential for successful cellular reprogramming. Utilizing a combination of machine learning and experimental approaches, we investigated GATA3's binding affinity to DNA and nucleosome substrates. Machine learning analysis identified two distinct zinc-finger domains which are required for effective GATA3 nucleosome binding. The results of this deep learning analysis were used to design a series of GATA3 mutant cell lines and DNA substrates. The experimental approach involved two strategies: first, analyzing the binding affinity of wild-type GATA3 to DNA substrates designed based on machine learning predictions, and second, examining the nucleosome binding affinity of GATA3 mutants, including those predicted by computational modeling and those derived from breast cancer patients. Electrophoretic mobility shift assays confirmed the differential binding affinities of GATA3 mutants, indicating the importance of specific motifs and zinc finger domains. Our findings highlight the significance of DNA sequence specificity in modulating pioneer transcription factor activity and chromatin accessibility. Mutational analysis revealed distinct roles for each zinc finger domain, with mutations affecting DNA binding affinity and chromatin accessibility. This study provides insights into the molecular mechanisms governing GATA3-mediated chromatin remodeling during MET. By characterizing the interaction between DNA/nucleosome binding affinity and chromatin accessibility, we expand our understanding of pioneer transcription factor function in breast cancer progression.
 Citation Format: Jill Goodman, Mika Saotome, Motoki Takaku. Nucleosome binding affinity of the pioneer factor GATA3 influences chromatin reprogramming in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c4cf68bc1ac895d343e311d0ea750e083006c2b" target='_blank'>
              Abstract P5-06-15: Nucleosome binding affinity of the pioneer factor GATA3 influences chromatin reprogramming in breast cancer
              </a>
            </td>
          <td>
            Jill Goodman, Mika Saotome, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/532ae00d954ab26c4423bce9c0c68dd5069e2e84" target='_blank'>
              Targeting miR-32-5p suppresses c-MYC-driven proliferation and induces apoptosis in MCF-7 breast cancer cells.
              </a>
            </td>
          <td>
            Ahmed I Khoder, I. H. El-Sayed, Yasser B M Ali
          </td>
          <td>2025-07-25</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/982238bfa802534a4f1e695ccfc316aaa317bc9c" target='_blank'>
              Phased epigenetic fragility and stability accelerate the genetic evolution of B-cells
              </a>
            </td>
          <td>
            Mark Y Xiang, Haripriya Vaidehi Narayanan, Vaibhava Kesarwani, Tiffany Wang, Alexander Hoffmann
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="<jats:p>
            <jats:bold>e15094</jats:bold>
          </jats:p>
          <jats:p>
            <jats:bold>Background:</jats:bold>
            Comprehensive genomic profiling (CGP) is essential for identifying actionable and prognostic biomarkers in oncology practice. This study investigates oncogenic alterations, key driver mutations, and co-mutation patterns in Indian cancer patients. By targeting hallmark pathways driving disease progression, it highlights the potential of combinatorial therapies over single-biomarker approaches to address cancer's complex evolution.
            <jats:bold>Methods:</jats:bold>
            A retrospective analysis was conducted on FFPE tumor samples from 3,740 Indian cancer patients across 37 malignancies. Comprehensive genomic profiling (CGP) was performed using next-generation sequencing (NGS) with whole exome sequencing and a custom-designed "Functional Cancer Exome" panel targeting 1,212 cancer-related genes. Variants were annotated using multiple databases, with functional and co-mutation analyses providing key insights into oncogenic mechanisms and mutation patterns.
            <jats:bold>Results:</jats:bold>
            A standard variant filtration strategy identified 2,795 pathogenic mutations, focusing on clinically relevant alterations across cancer types. The analysis highlights the prevalence of pathogenic mutations in 1075 pivotal genes, with
            <jats:italic toggle="yes">TP53</jats:italic>
            (34.8%),
            <jats:italic toggle="yes">KRAS</jats:italic>
            (15.3%), and
            <jats:italic toggle="yes">PIK3CA</jats:italic>
            (10.3%) being most prevalent. When unique genes identified in this study were mapped onto hallmark pathways associated with cancer etiology, we arrived at 13 genes(
            <jats:italic toggle="yes">MYC, IDH1, SOD1, CD36, EGFR,</jats:italic>
            and
            <jats:italic toggle="yes">GCH1</jats:italic>
            among others) that drive cross-talk across at least five distinct pathways, acting as master regulators of disease progression. We also looked at the functional categories of co-mutated genes compared with hallmark properties in table 1.
            <jats:bold>Conclusions:</jats:bold>
            One biomarker/gene and one drug combination may not work during disease evolution. The hub genes identified emphasize the need for combinatorial therapeutic approaches targeting multiple pathways to disrupt crosstalk driving disease progression. Future research trials integrating clinical validation will be essential to translate these therapeutic insights into effective personalized treatments.
            <jats:table-wrap id="table-511756-001" orientation="portrait" position="anchor">
              <jats:table content-type="mtgabstract">
                <jats:thead content-type="mtgabstract">
                  <jats:tr>
                    <jats:th colspan="1" content-type="border-bottom align-left" rowspan="1"/>
                    <jats:th colspan="1" content-type="border-bottom" rowspan="1">TSG</jats:th>
                    <jats:th colspan="1" content-type="border-bottom" rowspan="1">Oncogenes</jats:th>
                    <jats:th colspan="1" content-type="border-bottom" rowspan="1">Translocated Cancer Genes</jats:th>
                    <jats:th colspan="1" content-type="border-bottom" rowspan="1">Kinases</jats:th>
                    <jats:th colspan="1" content-type="border-bottom" rowspan="1">Development Marker</jats:th>
                    <jats:th colspan="1" content-type="border-bottom" rowspan="1">Homeodomain</jats:th>
                    <jats:th colspan="1" content-type="border-bottom" rowspan="1">TF</jats:th>
                    <jats:th colspan="1" content-type="border-bottom" rowspan="1">Cytokine and Growth Factor</jats:th>
                  </jats:tr>
                </jats:thead>
                <jats:tbody>
                  <jats:tr>
                    <jats:td colspan="1" content-type="row-heading" rowspan="1">TSG</jats:td>
                    <jats:td colspan="1" rowspan="1">59</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">1</jats:td>
                    <jats:td colspan="1" rowspan="1">4</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">1</jats:td>
                    <jats:td colspan="1" rowspan="1">12</jats:td>
                    <jats:td colspan="1" rowspan="1">1</jats:td>
                  </jats:tr>
                  <jats:tr>
                    <jats:td colspan="1" content-type="row-heading" rowspan="1">Oncogenes</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">58</jats:td>
                    <jats:td colspan="1" rowspan="1">39</jats:td>
                    <jats:td colspan="1" rowspan="1">16</jats:td>
                    <jats:td colspan="1" rowspan="1">9</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">19</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                  </jats:tr>
                  <jats:tr>
                    <jats:td colspan="1" content-type="row-heading" rowspan="1">Translocated Cancer Genes</jats:td>
                    <jats:td colspan="1" rowspan="1">1</jats:td>
                    <jats:td colspan="1" rowspan="1">39</jats:td>
                    <jats:td colspan="1" rowspan="1">41</jats:td>
                    <jats:td colspan="1" rowspan="1">9</jats:td>
                    <jats:td colspan="1" rowspan="1">5</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">19</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                  </jats:tr>
                  <jats:tr>
                    <jats:td colspan="1" content-type="row-heading" rowspan="1">Kinases</jats:td>
                    <jats:td colspan="1" rowspan="1">4</jats:td>
                    <jats:td colspan="1" rowspan="1">16</jats:td>
                    <jats:td colspan="1" rowspan="1">9</jats:td>
                    <jats:td colspan="1" rowspan="1">57</jats:td>
                    <jats:td colspan="1" rowspan="1">12</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                  </jats:tr>
                  <jats:tr>
                    <jats:td colspan="1" content-type="row-heading" rowspan="1">Development Marker</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">9</jats:td>
                    <jats:td colspan="1" rowspan="1">5</jats:td>
                    <jats:td colspan="1" rowspan="1">12</jats:td>
                    <jats:td colspan="1" rowspan="1">36</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">3</jats:td>
                  </jats:tr>
                  <jats:tr>
                    <jats:td colspan="1" content-type="row-heading" rowspan="1">Homeodomain</jats:td>
                    <jats:td colspan="1" rowspan="1">1</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">23</jats:td>
                    <jats:td colspan="1" rowspan="1">23</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                  </jats:tr>
                  <jats:tr>
                    <jats:td colspan="1" content-type="row-heading" rowspan="1">TF</jats:td>
                    <jats:td colspan="1" rowspan="1">12</jats:td>
                    <jats:td colspan="1" rowspan="1">19</jats:td>
                    <jats:td colspan="1" rowspan="1">19</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">23</jats:td>
                    <jats:td colspan="1" rowspan="1">102</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                  </jats:tr>
                  <jats:tr>
                    <jats:td colspan="1" content-type="row-heading" rowspan="1">Cytokine and Growth Factor</jats:td>
                    <jats:td colspan="1" rowspan="1">1</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">2</jats:td>
                    <jats:td colspan="1" rowspan="1">3</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">0</jats:td>
                    <jats:td colspan="1" rowspan="1">21</jats:td>
                  </jats:tr>
                </jats:tbody>
              </jats:table>
            </jats:table-wrap>
          </jats:p>">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9d2f0103c62cbb2bcca185d669141429a7560fa" target='_blank'>
              The Indian cancer genome landscape: Pathogenic alterations and emerging therapeutic targets.
              </a>
            </td>
          <td>
            Satyajit Khuntia, Nilesh Mukherjee, V. J, Sreekanth S.P., Jinumary John, N. Tanwar, S. Nayak, V. Sarathy, R. Thirumalairaj, Ghanashyam Biswas, V. P. Gangadharan, K. Prabash, V. Patil, Ramakant Deshpande, P. Chirmade, K. Rishi, H. Goswami, Giridharan Periyasamy, V. Veldore
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transposable elements (TEs) are abundant in the human genome, and they provide the source for genetic and functional diversity. Previous studies have suggested that TEs are repressed by DNA methylation and chromatin modifications. Here through integrating transcriptome and 3D genome architecture studies, we showed that haploinsufficient loss of NIPBL selectively activates alternative promoters (altPs) at the long terminal repeats (LTRs) of the TE subclasses. This activation occurs through the reorganization of topologically associating domain (TAD) hierarchical structures and the recruitment of proximal enhancers. These observations indicate that TAD hierarchy restricts transcriptional activation of LTRs that already possess open chromatin features. Perturbation of hierarchical chromatin topology can lead to co-option of LTRs as functional altPs, driving aberrant transcriptional activation of oncogenes. These data uncovered a new layer of regulatory mechanisms of TE expression and posit TAD hierarchy dysregulation as a new mechanism for altP-mediated oncogene activation and transcriptional diversity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13b5cab876cec5cb4938bafcc68be31f6630d5cb" target='_blank'>
              Disruption of TAD hierarchy promotes LTR co-option in cancer
              </a>
            </td>
          <td>
            Elissa W P Wong, Merve Sahin, Rui Yang, UkJin Lee, Dan Li, Y. Zhan, Rohan Misra, Fanny Tomas, Nawaf Alomran, A. Polyzos, Cindy J. Lee, Tuan Trieu, A. Martínez-Fundichely, Thomas Wiesner, Andrew Rosowicz, Shuyuan Cheng, Christina Liu, Morgan Lallo, A. Shoushtari, T. Merghoub, Pierre-Jacques Hamard, R. Koche, Ekta Khurana, E. Apostolou, Deyou Zheng, Yu Chen, Christina S. Leslie, Ping Chi
          </td>
          <td>2025-06-30</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="PIK3CA and KRAS are among the most frequently mutated oncogenes and often co-mutated in colorectal cancers. Understanding how KRAS codon-specific mutations affect cross-talks between the PI3K and MAPK pathways and response to targeted therapies, such as the p110α-specific inhibitor inavolisib (GDC-0077), is critical for advancing precision oncology. Focusing on colorectal PIK3CA+KRAS co-mutated models, we found that KRAS G12D-mutated cells were more sensitive to inavolisib than models with KRAS G13D, or other MAPK pathway mutations, even though the PI3K and MAPK pathways were active in both genotypes. In most co-mutated models, regardless of the type of KRAS alteration, combination of inavolisib with MAPK pathway inhibitors showed synergy in vitro and in vivo. Our work highlights how specific codon substitutions in KRAS differentially toggle pathway activity and alter sensitivity to inavolisib, which could inform whether patients would benefit more from single-agent inavolisib or combination with MAPK pathway inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e70c098958b33ecc57b400045bf2df06bb8439fa" target='_blank'>
              KRAS codon-specific mutations differentially toggle PI3K pathway signaling and alter sensitivity to inavolisib (GDC-0077).
              </a>
            </td>
          <td>
            Kyung W Song, Christy C Ong, Eva Lin, Jeff Lau, Nicole M Sodir, Dexter Jin, Katherine E. Hutchinson, Shiqi Xie, Jenille Tan, Yuxin Liang, Z. Modrusan, Scott E Martin, Danilo Maddalo, M. Hafner, Anwesha Dey
          </td>
          <td>2025-07-08</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="Recent studies have revealed that polyclonality—where multiple distinct subclones cooperate during early tumor development—is a critical feature of tumor evolution, as demonstrated by Sadien et al. and Lu et al. in Nature (October 2024). These findings show that early polyclonal interactions can overcome fitness barriers, ultimately transitioning to monoclonality as dominant clones emerge. Understanding and targeting these interclonal dynamics offers new therapeutic opportunities. In this perspective, we outline how computational modeling and artificial intelligence (AI) tools can provide deeper insights into tumor polyclonality and identify actionable therapeutic strategies. By applying ligand–receptor interaction analysis, clonal trajectory reconstruction, network and pathway modeling, and spatial analysis, researchers can prioritize communication hubs, evolutionary bottlenecks, and microenvironmental niches that sustain tumor progression. These approaches, when integrated with experimental validation, offer a translational pathway from foundational discoveries to personalized cancer treatments aimed at disrupting cooperative subclonal ecosystems and preventing malignant progression. We commend the recent Nature publications, “Polyclonality overcomes fitness barriers in Apc-driven tumorigenesis” by Sadien et al. [1] and “Polyclonal-to-monoclonal transition in colorectal precancerous evolution” by Lu et al. [2], both featured on 2024 October 30. These groundbreaking studies employed distinct lineage tracing methods to investigate the origins and evolutionary dynamics of colorectal and intestinal tumorigenesis. Despite their different approaches, both studies reached convergent conclusions: Polyclonality plays a pivotal role in the early stages of tumor development, providing critical insights into how diverse cellular populations collaborate to overcome fitness barriers and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6580f8163812203e6792c0b0ae89c7c42827a3" target='_blank'>
              Artificial Intelligence-Powered Insights into Polyclonality and Tumor Evolution
              </a>
            </td>
          <td>
            Hong Zhao, T. Ideker, Stephen T C Wong
          </td>
          <td>2025-06-13</td>
          <td>Research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="
 8622


 Background:
 Epidermal growth factor receptor (EGFR)-activating mutations (Del19 or L858R) are oncogenic drivers of non-small cell lung cancer (NSCLC). Most patients treated with tyrosine kinase inhibitors (TKIs) will eventually develop resistance mutations including the T790M gatekeeper mutation. Osimertinib, a third-generation covalent TKI, is efficacious against the T790M resistance mutation and prevents its onset. However, treatment with Osimertinib inevitably induces additional mutations, especially the C797S mutation, as well as various off-target resistance mechanisms. To date, there are no approved therapies capable of overcoming mutational or non-mutational resistance to third-generation EGFR TKIs.
 Methods:
 We have characterized the efficacy of two novel fourth-generation EGFR inhibitors, CCM-205 and CCM-308, which are potent against both mutational and non-mutational tumor resistance to Osimertinib. Enzymatic binding affinities were determined by KdELECT assay. Cell-Titer-Glo (CTG) assay was used to assess cytotoxicity (cellular IC
 50
 ) of CCM-205 and CCM-308
 in vitro
 on EGFR triple mutant and other Osimertinib-resistant cell lines, with comparison to both Osimertinib and fourth-generation EGFR inhibitor BLU-945.
 In vivo
 tumor growth inhibition (TGI) was determined in Osimertinib-resistant xenografts including the triple mutant PC9-DTC (Del19/T790M/C797S) model.
 Results:
 In Ba/F3 EGFR DTC and LTC cells, CCM-205 / CCM-308 inhibit proliferation with IC
 50
 s of 137 nM / 40 nM and 198 nM / 61 nM respectively, while Osimertinib antiproliferation is limited to 1.225 µM and 1.562 µM respectively. Moreover, CCM-205 / CCM-308 spare Ba/F3 EGFR WT better than Osimertinib with IC
 50
 s of 577 nM / 294 nM (182 nM for Osimertinib). CCM-205 / CCM-308 also bind tightly to double mutants targeted by Osimertinib with K
 d
 s

 for EGFR LT (L858R/T790M) of 4.9 nM / 1.2 nM. CCM-205 and CCM-308 are more potent against PC9 DTC cells (IC
 50
 s: 1.02 µM and 220 nM, respectively) than Osimertinib (4.10 µM) and are comparable to BLU-945 (559 nM). In addition, CCM-205 / CCM-308 are highly potent against Osimertinib-resistant PC9 (IC
 50
 s: 728 nM / 321 nM) and H1975 (IC
 50
 s: 1.716 µM / 684 nM) cell lines generated through 8-week treatment with 1 µM Osimertinib (Osimertinib IC
 50
 = 3.81 µM and 4.70 µM, respectively), while EGFR-specific fourth-generation inhibitors targeting C797S such as BLU-945 lose potency (IC
 50
 s: 7.96 µM and > 10 µM, respectively). In the PC9-DTC xenograft, CCM-205 completely inhibited tumor growth and induced tumor regression (> 100% TGI) exceeding that of BLU-945, while the tumor was resistant to Osimertinib (< 20% TGI), when agents were delivered orally at similar fractions of their maximum tolerated doses (MTDs).
 Conclusions:
 Novel fourth-generation EGFR inhibitors have been developed that can potentially overcome both on-target and off-target resistance in NSCLC and have potential clinical applications.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fa72281124a34cbe69e7cd414c25408817fca4b" target='_blank'>
              Novel potent and selective fourth-generation inhibitors targeting EGFR for NSCLC therapy.
              </a>
            </td>
          <td>
            G. Errasti, Thomas Delacroix, Kalpana Ghoshal, Robert Lee, Anisha Ghosh, Raj Chakrabarti
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncogenic mutations of Ras are among the most common genetic alterations in human cancer, with an estimated disease burden of >3 million new patients per year worldwide. Despite widespread appreciation of the importance of Ras in cancer, direct binding ligands, which block downstream signaling, were not reported until 2013 due to the lack of obvious drug binding pockets in the protein. The clinically approved K-Ras inhibitors are mutant-selective as they rely on covalent recognition of the highly nucleophilic somatic cysteine residue of K-Ras(G12C). Recent preclinical reports of noncovalent K-Ras binding inhibitors have emerged, which lack mutant specificity and exhibit varying degrees of biochemical preference for mutant K-Ras over the wild-type. An adjacent glycine-13 mutation, p. G13C, particularly abundant in lung, colorectal, and pancreatic cancer, has not been targeted with an approved therapeutic molecule. Here, we report a series of targeted electrophiles designed to covalently modify Cys13 in K-Ras(G13C), overcoming the structural challenge posed by its shifted position relative to Cys12 in K-Ras(G12C). These inhibitors effectively alkylate K-Ras(G13C) in both GDP- and GTP-bound states, block effector interactions, and suppress the growth of K-Ras(G13C)-mutation cancer cell lines. Our findings expand the landscape of covalent K-Ras inhibitors beyond G12 mutations, providing a new therapeutic strategy for K-Ras(G13C)-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7630138c5c420d8de4571d182b52233bbb51cc61" target='_blank'>
              Distal Covalent Targeting Suppresses Signaling of Oncogenic K-Ras(G13C) in Cancer Cells.
              </a>
            </td>
          <td>
            Qinheng Zheng, Tianfang Shen, Julius Pampel, K. Shokat
          </td>
          <td>2025-07-09</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Burkitt Lymphoma (BL), an aggressive B-cell lymphoma driven by MYC translocations, requires intensive chemotherapy treatments which deliver high effectiveness yet increase future risks of developing secondary malignancies. We have previously shown that DDX3X, an RNA helicase frequently mutated in BL, is essential for B cell lymphomagenesis in mice.To assess if DDX3X could therefore represent a promising therapeutic target for BL, we tested two DDX3X inhibitors, the well characterized RK-33 and the more potent newly developed C1, in three BL cell lines (CA46, Raji, Daudi). We found that the 3 cell lines exhibited differential sensitivities to the drugs in vitro, with Daudi being the most susceptible and Raji the most resistant. In vivo, RK-33 treatment in a xenograft BL model reduced tumor progression in all cell lines, albeit with variable efficacy compared to the clinical drug Pevonedistat, and again with the Daudi cells being the most responsive to the treatment. Transcriptomic and proteomic analyses indicated that RK-33-mediated inhibition of DDX3X, and DDX3X ablation through siRNA affects oxidative phosphorylation among other pathways and leads to an increase of intracellular reactive oxygen species (ROS). A CRISPR chemogenomic screen to identify synthetic lethalities linked to RK-33 implicated enzymes of the glutathione synthesis pathway and the Keap1-Nrf2-ARE pathway. We therefore tested the inhibition of the glutathione pathway with buthionine sulfoximine and showed that it reduced the CC50 of RK-33 in BL cells lines.Our findings not only support DDX3X as a therapeutic target in BL but also provide evidence for a combinatorial treatment strategy to improve the efficacy of current treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af8e47b46f9074e8482f490db8d4008cd6713d8" target='_blank'>
              Inhibiting the RNA helicase DDX3X in Burkitt lymphoma induces oxydative stress and impedes tumor progression in xenografts
              </a>
            </td>
          <td>
            H. Beauchemin, Zeinab Dalloul, Eva-Maria Piskor, Virginie Calderon, Andrew Chatr-Aryamontri, Thierry Bertomeu, T. Möröy
          </td>
          <td>2025-07-23</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24cfd9851764715ead654d1c47914a25b505dff" target='_blank'>
              Die Vermessung des Genoms
              </a>
            </td>
          <td>
            Clemens Steinek, H. Leonhardt
          </td>
          <td>2025-06-01</td>
          <td>BIOspektrum</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="HSD3B1 encodes an enzyme that catalyzes the conversion of adrenal precursors into potent sex steroids. A common germline variant (c.1100C) enhances this effect and is linked to breast cancer (BC) progression. As the HSD3B1 genotypes contribute to differences in local and adrenal steroid production, their transcriptional and phenotypic effects on cancers influenced by hormonal signaling such as BC and endometrial cancer (EC)—particularly in relation to menopausal status—remain unclear. We analyzed BC and EC sequenced from patients that received diagnostic tests in oncology clinics, and we determined the germline HSD3B1 c.1100 genotype (AA, AC, CC) from tumor DNA sequencing by using variant allele frequency, with inferred menopausal status assumed by age at molecular profiling. Whole-transcriptome RNA sequencing and gene set enrichment analysis showed that adrenal-permissive homozygous (CC) tumors in premenopausal ER + BC were enriched for hormone-related pathways, including Estrogen Response Early (NES ≈ +1.8). In premenopausal triple-negative BC, adrenal-restrictive homozygous (AA) tumors exhibited the elevated expression of immune and epithelial genes and the increased prevalence of MED12 alterations (AA 0.25% vs. CC 8%, p < 0.01). In endometrioid EC, CC tumors demonstrated the suppression of immune and proliferative pathways. Postmenopausal cases had higher progesterone receptor IHC positivity (AA 75% vs. CC 83%, p < 0.05) and numerically more frequent ESR1 copy number gains (AA 2.0% vs. CC 4.0%). Results highlight context-specific associations between germline HSD3B1 genotypes and tumor biology in BC and EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83fbb8500f0f6e5e010ceeaf4c87e837f593cf33" target='_blank'>
              HSD3B1 (c.1100C) Genotype Is Associated with Distinct Tumoral and Clinical Outcomes in Breast and Endometrial Cancers
              </a>
            </td>
          <td>
            Nikitha Vobugari, Allison Makovec, Samuel Kellen, Shayan S. Nazari, Andrew Elliott, Devin Schmeck, Aiden Deacon, Gabriella von Dohlen, Emily John, Pedro C. Barata, N. Agarwal, Melissa A. Geller, Britt K. Erickson, George W Sledge, J. Ostrander, Rana R McKay, Charles J. Ryan, Nima Sharifi, Emmanuel S. Antonarakis, Justin H. Hwang
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="While switch-like gene expression (“on” in some individuals and “off” in others) has been linked to biological variation and disease susceptibility, a systematic analysis across tissues is lacking. Here, we analyze genomes, transcriptomes, and methylomes from 943 individuals across 27 tissues, identifying 473 switch-like genes. The identified genes are enriched for associations with cancers and immune, metabolic, and skin diseases. Only 40 (8.5%) switch-like genes show genetically controlled switch-like expression in all tissues, i.e., universally switch-like expression. The rest show switch-like expression in specific tissues. Methylation analysis suggests that genetically driven epigenetic silencing explains the universally switch-like pattern, whereas hormone-driven epigenetic modification likely underlies the tissue-specific pattern. Notably, tissue-specific switch-like genes tend to be switched on or off in unison within individuals, driven by tissue-specific master regulators. In the vagina, we identified seven concordantly switched-off genes linked to vaginal atrophy in females. Experimental analysis of vaginal tissues shows that low estrogen levels lead to decreased epithelial thickness and ALOX12 expression. We propose that switched-off driver genes in basal and parabasal epithelia suppress cell proliferation, leading to epithelial thinning and vaginal atrophy. Our findings underscore the implications of switch-like genes for diagnostic and personalized therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25a8bc142a1fa908c8d231b3fb99478b1a7a0c89" target='_blank'>
              Switch-like gene expression modulates disease risk
              </a>
            </td>
          <td>
            Alber Aqil, Yanyan Li, Zhiliang Wang, Saiful Islam, Madison Russell, T. Kallak, M. Saitou, Omer Gokcumen, Naoki Masuda
          </td>
          <td>2025-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Alternative splicing (AS) dysregulation is increasingly recognized as a critical factor in cancer progression and drug response. However, precisely detecting and characterizing complex splicing events, particularly those involving microexons, remains technically challenging. Ribosomal Protein 24 (RPS24), which contains three microexons (3, 18, and 22 bp), serves as an ideal model for studying complex AS regulation in cancer. We developed a high-resolution detection method for RPS24 microexon variations and investigate their relationship with KRAS proto-oncogene, GTPase (KRAS) inhibition in lung adenocarcinoma (LUAD) to identify potential biomarkers for KRAS-targeted inhibitors. We established an integrated methodological approach combining RNA-seq analysis with fragment analysis to detect RPS24 AS patterns. Using this method, we analyzed RPS24 AS across a panel of lung cancer cell lines and examined AS changes in KRAS-mutant cell lines following treatment with KRAS inhibitors. Our method successfully characterized distinct RPS24 AS isoform compositions across lung cancer cell lines, demonstrating high accuracy in detecting 3 bp variations. In KRAS-mutant cell lines, we observed a consistent upregulation of the 3 bp-containing isoform following KRAS inhibition, indicating a specific correlation with treatment response. This study provides a robust methodology for analyzing complex AS events and supports the RPS24 3 bp-containing isoform as a potential biomarker for KRAS inhibitor response in LUAD. These findings offer new insights into the molecular mechanisms of KRAS inhibitor therapy and strategies for monitoring treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21ce2d06805a31042a8c5858c7a0eac61b833f6" target='_blank'>
              High-resolution detection of RPS24 microexon variations reveals novel splicing patterns in response to KRAS-targeted therapy in lung adenocarcinoma
              </a>
            </td>
          <td>
            D. Nam, Bin Tian, Jiyeon Park, Yeun-Jun Chung
          </td>
          <td>2025-06-10</td>
          <td>BMB Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 2519


 Background:
 Tumor heterogeneity and low mutational burden limits the availability of effective immunotherapy targets. Aberrant RNA-splicing (neojunctions) represents an underexplored yet promising source of neoantigens. To address this, we developed a neoantigen discovery platform (SNIPP) that characterizes a novel class of clonally-expressed, splicing-derived neoantigens. Furthermore, we validated the immunogenicity of these neoantigens by identifying specific TCRs that drive CD8+ T-cell-mediated tumor killing.
 Methods:
 SNIPP identified public neojunctions by analyzing TCGA RNA-seq data, selecting neojunctions with a positive sample rate (PSR) > 10% and filtering out those found in GTEx normal tissue RNA-seq data (PSR < 1%) across 12 cancer types. To characterize intratumorally conserved neojunctions, we performed maximally-distanced multi-site biopsies (
 n
 = 535) within glioma patients (
 n
 = 56) and generated RNA-seq data for each intratumoral site. Two independent algorithms were utilized to predict peptide processing likelihood and HLA-binding affinity of splicing-derived neoantigen candidates. Neoantigen-specific TCR sequences were identified via
 in vitro
 sensitization of PBMCs and subsequent 10x V(D)J scRNA-seq. These TCRs were transduced into CD8+ T-cells, which were tested downstream for immunogenicity and cytotoxicity against glioma cell lines.
 Results:
 Our pipeline identified 789 public neojunctions, including 32 neojunctions concurrently detected in transcriptomic and proteomic glioma datasets and confidently predicted to be presented by HLA-A*02:01. IVS and subsequent 10x V(D)J scRNA-seq identified TCR clonotypes reactive against neojunctions in
 RPL22
 (
 n
 = 7) and
 GNAS
 (
 n
 = 1), with the latter exhibiting high intratumoral conservation (detected in > 90% of spatially-mapped biopsies across 17/56 patients (26.78%)). TCR-transduced CD8+ T-cells recognized and were immunogenically activated and demonstrated cytotoxicity against endogenously processed and presented neoantigens in GBM and melanoma lines. Additionally, IDH1-mutant oligodendrogliomas exhibited significantly higher neojunction expression compared to IDH1-mutant astrocytomas and IDH1wt subtypes. Differential gene expression analysis (DESeq2) revealed reduced expression of splicing factors in oligodendrogliomas, attributed to their specific co-deletion of chromosomes 1p and 19q. CRISPRi-mediated knockdown of these splicing factors (e.g. SF3A3, SNRPD2) in IDH1wt glioma cells resulted in significantly increased expression of corresponding neojunctions.
 Conclusions:
 Our study highlights a novel class of neoantigens derived from tumor-wide aberrant RNA splicing. The SNIPP platform effectively identifies public intratumorally-conserved neojunctions with strong therapeutic potential. Furthermore, elevated neojunction expression in oligodendroglioma underscores the mechanistic link between dysregulated splicing factor expression and RNA splicing abnormalities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476976fc09e028193d5d0a3658a6d5d31763d4d8" target='_blank'>
              Tumor-wide RNA splicing aberrations and their potential as therapeutic neoantigen targets.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O. Stevers, Iñaki Etxeberria, T. Nejo, Maggie Colton Cove, Lee-may Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Aaron A. Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideho Okada
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e7385113aeec5ed3d01c2aa1c0786f66a695de" target='_blank'>
              Mammalian epigenomic conservation of promoters and enhancers in the heart associates with trait-associated variation and impacts cardiomyocyte phenotypes
              </a>
            </td>
          <td>
            S. Frost, Diego Fernandez-Aroca, Elise Parey, Adrian Rodriguez-Gonzalez, Daniel Pavon-Heredia, Diego Villar
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The identification of recurrent oncogenic drivers has enabled targeted therapeutic strategies for subsets of non-small cell lung carcinoma (NSCLC) and other malignancies. Oncogenic fusions involving the RAS-activating guanine exchange factor (GEF) RASGRF1 are reported in multiple tumors, but their prevalence and genetic heterogeneity remain undefined. Here, we query RNA-seq data from a real-world database of diverse human malignancies and identify 40 tumors with rearrangements involving RASGRF1 or the related RASGRF2 predicted to generate chimeric proteins. Half of these fusions occur in NSCLC, pancreatic cancer, and melanoma and are enriched in tumors without other established driver alterations. A subset of RASGRF fusions contains transmembrane partners, and membrane localization enhances RAS activation and transforming activity. Loss of N-terminal PH1 and DH domains in RASGRF fusions also promotes transformation. Although some fusions lack the PH1 but not the DH domain, our functional assays indicate that loss of the PH1 domain alone is insufficient to drive cellular transformation. Our findings provide insights about the tissue distribution, structural diversity, and oncogenic mechanisms of RASGRF fusions. As cell models driven by these fusions are sensitive to MAPK pathway inhibition, oncogenic RASGRF fusions may represent a therapeutic target in rare molecular subsets of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bb9a75bf2438a8363d43341f77587497989fb9b" target='_blank'>
              Genetic and oncogenic features of RASGRF fusions
              </a>
            </td>
          <td>
            Sreya Das, Daniel S Lenchner, Ellen Jaeger, Lisa Hunihan, Dana F DeSantis, Stamatina Fragkogianni, Karyn Ronski, Frederick H Wilson
          </td>
          <td>2025-07-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chemotherapy resistance poses a significant challenge in the initial treatment of acute myeloid leukemia (AML). Growth differentiation factor 15 (GDF-15) has been shown to play a critical role in cancer progression; however, the potential mechanisms by which GDF-15 contributes to AML progression and chemotherapy resistance remain unclear. We found that M2 macrophages secrete high levels of GDF-15, promoting resistance of AML cells to mitoxantrone (MTX). Furthermore, we demonstrated that MTX induces downregulation of the SLC7A11/GPX4 signaling axis in AML cells, mediating ferroptosis. GDF-15 enhances the expression of the SLC7A11/GPX4 axis, thereby inhibiting ferroptosis in AML cells and contributing to drug resistance. In addition, GDF-15 mitigates the decline in mitochondrial membrane potential and mitochondrial quality induced by MTX. In vivo experiments indicate that blocking GDF-15 effectively enhances the sensitivity of AML cells to mitoxantrone by reducing the expression of the SLC7A11/GPX4 axis. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-02787-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad6ac9dcd9a8baf8afdd764796e6efddfc19e74" target='_blank'>
              GDF-15 upregulates the SLC7A11/GPX4 signaling axis and promotes mitoxantrone resistance in AML cells
              </a>
            </td>
          <td>
            Qian-wei Lu, Yang Liao
          </td>
          <td>2025-06-21</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gliomas are the most common and lethal malignant primary brain tumors in adults, associated with the highest number of years of potential life lost. The latest WHO classification for central nervous system tumors highlighted the need for new biomarkers of diagnosis, prognosis, and response to therapy. The PI3K/Akt signaling pathway is clearly implicated in tumorigenesis, being one of the most frequently altered pathways in cancer. Activating PI3KCA mutations are oncogenic and can influence both prognosis and treatment response in various tumor types. In gliomas, however, studies have reported inconsistent PIK3CA mutational frequencies, ranging from 0% to 30%. Furthermore, the impact of these alterations on glioma diagnosis, prognosis, and therapy response remains unclear. Current evidence suggests that PIK3CA mutations may represent early and constitutive events in glioma development, associated with worse glioblastoma prognoses, earlier recurrences, and widespread disease. Among these, the hotspot mutation H1047R has been particularly associated with a more aggressive phenotype while also modulating the neuronal microenvironment. In this review, we examine the clinical relevance of PIK3CA mutations across different cancers, with a particular focus on their emerging role in glioma. Moreover, we also discuss the therapeutic potential and challenges of targeting PIK3CA mutations in the context of glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6d6f1fbe9d06843c9afa611b81f785472f82b9" target='_blank'>
              PIK3CA Mutations: Are They a Relevant Target in Adult Diffuse Gliomas?
              </a>
            </td>
          <td>
            A. Tomás, M. Pojo
          </td>
          <td>2025-05-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genetic lesions drive cancer development and progression, and understanding their origins will reveal the mechanisms of carcinogenesis. We showed that murine FAM72A promotes mutagenic DNA repair during antibody maturation by acting as a substrate adaptor of the CTLHMKLN1 E3 ligase to induce the proteasome degradation of Uracil DNA glycosylase 2 (UNG2), a pivotal enzyme of the base excision repair. In humans, the FAM72 gene has expanded to include four paralogues named FAM72A-D. Bioinformatic studies suggested that the human FAM72 genes are overexpressed in a broad range of cancers. However, the functional roles of FAM72A-D in human biology and cancer are unknown. Here, we show that FAM72 family members are minimally expressed in most healthy tissues except for thymus, and that FAM72A, B and D are overexpressed in primary tumorigenic tissues. Human FAM72 expression inversely correlates with UNG2 protein level in human cell lines and primary tumorigenic tissues suggesting that human FAM72 promotes UNG2 degradation. However, only FAM72A is able to bind to and induce UNG2 degradation in human cells. Our results suggest that the ability of FAM72A to induce UNG2 degradation contributes to neoplasia in a variety of cancer types by promoting mutagenic repair of genomic dUs. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07723-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369de028cb26608e44c32340a1371447a2c75ba9" target='_blank'>
              FAM72A promotes UNG2 degradation and mutagenesis in human cancer cells
              </a>
            </td>
          <td>
            Yuqing Feng, P. Barbulescu, Chetan K Chana, Melika Shirdarreh, Hong Yang, Lizhen Wu, Sami Mamand, Mohammad Kashem, Amin Zia, Ming Han, Julissa Tsao, Trevor Pugh, David Cescon, David G. Schatz, Frank Sicheri, Alberto Martin, R. Pezo
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42ec56e20e2974783416562dededf866b32de1c4" target='_blank'>
              Research progress on cancer-related epigenetic switches.
              </a>
            </td>
          <td>
            Chunyu Yu, Jie Sun, Jinlong Tong, Ziqing Xu, Qijia Zhang
          </td>
          <td>2025-06-13</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3f47763920024ffd71f9b24d0bcffb5c1ed193c" target='_blank'>
              Retrospective and multifactorial single-cell profiling reveals sequential chromatin reorganization during X inactivation.
              </a>
            </td>
          <td>
            Samy Kefalopoulou, Pim M. J. Rullens, Kim L. de Luca, Sandra S. de Vries, T. Korthout, Alexander van Oudenaarden, Peter Zeller, Jop Kind
          </td>
          <td>2025-07-10</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617aa65bc58f5944ef8ed06dc69bcc6987f45d4a" target='_blank'>
              Emerging roles and clinical perspectives of long noncoding RNAs in prostate cancer.
              </a>
            </td>
          <td>
            Rajarethinam Kumar, Saradhadevi Muthukrishnan
          </td>
          <td>2025-06-27</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="All biological processes, from embryonic development to cancer, are tightly controlled by the interactions between genetics and epigenetics. An array of epigenetic modifications, such as DNA methylation, histone/chromatin modifications, and noncoding RNA-mediated targeting, are essential to regulate the heritable changes that occur during multiple cellular processes. A failure in proper regulation results in inappropriate gene expression that ultimately leads to pathological states. Groundbreaking advances in genomics and transcriptomics have revealed the potential involvement of epigenetics in various physiological and pathological states. The promising clinical and preclinical results shown by epigenetics drugs further underscore the central role of epigenetics in multiple human diseases, including cancer. AT rich interaction domain (ARID)-containing proteins are a family of evolutionarily conserved DNA binding proteins that regulate epigenetic modifications. Genome sequencing has revealed the existence of 15 ARID family proteins that are divided into 7 subfamilies based on their sequence and domain homology. Although the ARID family of proteins are implicated in cell growth, development, differentiation, and cancer, the diverse biological functions of many family members remain to be elucidated. Here, we focus on ARID4B to summarize its prominent role in embryonic stem cell differentiation and human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72fba835b7a96d97fab7eb6f68f890cb10aeda31" target='_blank'>
              ARID4B: An Orchestrator from Stem Cell Fate to Carcinogenesis
              </a>
            </td>
          <td>
            R. R. K. Kandy, M. Arumugam, Mukesh Pratap Yadav, Bibhuti Bhusan Mishra, Jyotika Sharma
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="A cancer is said to be hormone-dependent when hormones influence the carcinogenesis process. Studies suggest a possible protective effect of hormone treatment in colorectal cancer. Given the hormone treatment, a thorough study of CYP17A1 is necessary. This study aims to assess the penetrance of CYP17A1 in the development of colorectal cancer. This study involved 24 colorectal cancer patients and 24 controls. For each sample, DNA extraction was performed, followed by CYP17A1 gene amplification and Sanger sequencing. The nature and position of mutations were identified using Mutation Surveyor version 5.1.2. DnaSP version 5.10, MEGA version 7.014, and the Arlequin program version 3.1 were used to highlight the parameters of variability, differentiation, and demogenetic evolution of the study population. The results revealed heterozygous mutations in the CYP17A1 gene and a substitution at the c.-34T>C promoter region. The absence of nonsynonymous mutations was revealed by very low genetic variability in the cancer population compared with controls. There was also little genetic differentiation and distance between the two populations. The frequent c.-34T>C mutation suggests that this polymorphism may modulate the transcriptional activity of CYP17A1 and consequently the hormones under this gene's control, and therefore colorectal tumor growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ea4b0f263eb567df369a8a5a5af8293415099bc" target='_blank'>
              Genetic alterations of CYP17A1 in the occurrence of colorectal cancer in Senegal
              </a>
            </td>
          <td>
            Anna Ndong, Bineta Kénémé, Yacine Seye, Fatimata Mbaye, M. Sembene
          </td>
          <td>2025-06-12</td>
          <td>International journal of health sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Oncogenic viruses typically manipulate host cellular mechanisms to drive tumorigenesis. They exploit pioneering transcription factors to modify gene expression, enabling uncontrolled proliferation. These viruses alter chromatin accessibility and induce chromatin remodelling, disrupting DNA repair and promoting viral genome integration. Additionally, epigenetic reprogramming through mechanisms like DNA methylation and histone modifications silences tumor suppressor genes and activates oncogenes. Understanding these mechanisms is critical for identifying more improved therapeutic targets, improving diagnostics, and predicting disease progression. Advances in this field can guide the development of innovative treatments and early detection tools. This comprehensive review synthesizes existing knowledge on the contributions of oncogenic viruses such as hepatitis B virus (HBV), hepatitis C virus (HCV), human papillomavirus (HPV), and human T-cell leukaemia virus type 1 (HTLV-1), Epstein–Barr virus (EBV), human herpesvirus 8 (HHV-8), and Merkel cell polyomavirus (MCV) to cancer development, highlighting their therapeutic relevance and driving forward research in viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b6df879c24dbe78b1949e0a2207cf7cabf08ee" target='_blank'>
              The role of pioneering transcription factors, chromatin accessibility and epigenetic reprogramming in oncogenic viruses
              </a>
            </td>
          <td>
            M. Kgatle, S. Mbambara, Leon Khoza, Olalekan Fadebi, T. Mashamba-Thompson, M. Sathekge
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The KRAS (Kirsten rat sarcoma viral oncogene homolog) gene mutation is commonly found in colorectal, lung, and pancreatic carcinomas. Unfortunately, blocking KRAS straight away has proven to be challenging. PROTACs (Proteolysis Targeting Chimeras), a class of bifunctional molecules, are designed to break down proteins, offering a unique strategy to target KRAS and overcome the limitations of traditional inhibition. This review discusses PROTACs targeting KRAS mutations in cancer, highlighting major findings, current limitations, and future perspectives. To achieve this, we thoroughly analyzed literature sourced from reputable databases, including PubMed, Google Scholar, and ScienceDirect. Various relevant articles were obtained from the reference section of the selected papers. PROTACs successfully induce the degradation of mutant KRAS in cell lines, leading to a decrease in cell viability compared to control groups. PROTAC treatment results in the suppression of downstream signalling pathways associated with KRAS, such as the MAPK and PI3K/AKT pathways. Animal studies demonstrate the ability of the PROTAC to effectively target KRAS-mutant tumors, inhibiting tumour growth without significant toxicities. New advances in this field can lead to cancer treatments that specifically target KRAS-mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd32e9291d35493d6856af19e795518346dbb5ff" target='_blank'>
              Innovative Therapies for Oncogenic KRAS Mutations: Precision Strategies with PROTACs in Cancer Treatment.
              </a>
            </td>
          <td>
            Praveen Halagali, Himanshu Sharma, Mahalaxmi Rathnanand, Vamshi Krishna Tippavajhala
          </td>
          <td>2025-06-04</td>
          <td>Anti-cancer agents in medicinal chemistry</td>
          <td>0</td>
          <td>12</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [2],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>